/** zh localization */
define({
    // curated begin ----------------------------------------------------------------------------------
    "Built-in references were created using default options": "使用默认选项创建了内置引用。",
    "If your genome of interest is not listed, contact the Galaxy team": "如果您感兴趣的基因组未列出，请与Galaxy团队联系（微信：usegalaxy ）。",
    "Select this option for compatibility with MultiQC (--new-summary)": "选择此选项以实现与MultiQC的兼容性（--new-summary）",
    "Output alignment summary to a file in addition to stderr. (--summary-file)": "将比对摘要输出到文件以及标准错误输出（--summary-file）",
    "Current password": "当前密码",
    "New password": "新密码",
    "Please provide a value for this option.": "请为该选项提供一个值。",
    "Collapse/Expand": "折叠/展开",

    "Show Sections": "显示分组",
    "Hide Sections": "隐藏分组",

    "Address": "通讯地址",

    "All workflows": "所有流程",
    "China": "中国",
    "search datasets": "搜索数据集",
    "History Size": "历史记录大小",
    "Manage Preferred History Storage": "管理首选历史存储",
    "Show active": "显示活动的",
    "Include deleted": "包括删除的",
    "Include hidden": "包括隐藏的",
    "Select All": "全选",
    "Clear selection": "清空选择",

    "Add to Favorites": "添加到收藏夹",
    "Remove from Favorites": "从收藏中移除",
    "Clear": "清除",
    "Show favorites": "显示收藏夹",
    "Show panel options": "显示面板选项",

    // masthead
    Workflow: "流程",
    Visualize: "可视化",
    Data: "数据",
    Admin: "管理员",
    Help: "帮助",
    User: "用户",
    // masthead tooltip
    "Tools and Current History": "工具和当前历史",
    "Chain tools into workflows": "将工具链接到流程",
    "Visualize datasets": "可视化数据集",
    "Access resources": "访问资源",
    "Administer this Galaxy": "管理您的 Galaxy",
    "Support, contact, and community": "支持、联系和社区",
    "Account and saved data": " 账户和保存的数据",
    "See Galaxy Training Materials": "查看 Galaxy 培训材料",
    "Enable/Disable Window Manager": "启用/禁用窗口管理器",
    "Show/Hide Window Manager": "显示/隐藏窗口管理器",

    // Data menu
    "Data Libraries": "数据库",
    Datasets: "数据集",
    Histories: "历史",
    Pages: "页面",
    Visualization: "可视化",
    Workflows: "流程",
    "Workflow Invocatios": "流程调用",

    // Help menu
    "Galaxy Help": "帮助",
    Support: "支持",
    Videos: "视频",
    "Community Hub": "社区",
    "How to Cite Galaxy": "引用",
    "Interactive Tours": "交互式引导",
    About: "关于",

    // User menu
    "Signed in as": "登录为",
    Notifications: "通知",
    Preferences: "偏好",
    "Sign Out": "退出登录",

    // Activity bar
    Upload: "上传",
    Tools: "工具",
    //Workflows: "流程",
    "Workflow Invocations": "流程调用",
    //Visualize: "可视化",
    //Histories: "历史",
    "History Multiview": "历史多视图",
    //Datasets: "数据集",
    //Notifications: "通知",
    Settings: "设置",
    // Activity bar tooltip
    "Download from URL or upload files from disk": "从网址下载或从磁盘上传文件",
    "Search and run tools": "搜索并运行工具",
    "Search and run workflows": "搜索并运行流程",
    "Show all workflow runs": "显示所有流程调用",
    "Show all histories": "显示所有历史记录",
    "Select histories to show in History Multiview": "选择在历史多视图中显示的历史记录",
    "Show all datasets": "显示所有数据集",
    "You have no unread notifications": "您没有未读通知",
    "Edit preferences": "编辑偏好",
    // Activity bar setting
    "Customize Activity Bar": "自定义活动栏",
    "User Preferences": "用户偏好",
    "Displays currently active interactive tools (ITs), if these are enabled by the administrator.": "显示当前活动的交互式工具（ITs），如果这些工具已被管理员启用。",
    "Opens a data dialog, allowing uploads from URL, pasted content or disk.": "打开一个数据对话框，允许从URL、粘贴的内容或磁盘上传。",
    "Displays the tool panel to search and access all available tools.": "显示用于搜索和访问所有可用工具的工具面板。",
    "Displays a panel to search and access workflows.": "显示一个搜索和访问工作流的面板。",
    "Displays all workflow runs.": "显示所有工作流运行。",
    "Displays the list of available visualizations.": "显示可用可视化列表。",
    "Displays the list of all histories.": "显示所有历史记录的列表。",
    "Displays the history selector panel and opens History Multiview in the center panel.": "显示历史选择器面板，并在中心面板中打开历史多视图。",
    "Displays all of your datasets across all histories.": "显示所有历史记录中的所有数据集。",
    "Displays this dataset.": "显示这个数据集。",

    // Upload
    // ---------------------------------------------------------------------------- upload-view
    "Upload Data": "上传数据",
    "Upload from Disk or Web": "从磁盘或网络上传",
    "Download from web or upload from disk": "从网页获取或者从本地磁盘上传数据",
    Regular: "常规数据",
    Composite: "复合数据",
    Collection: "集合数据",
    "Rule-based": "基于规则",
    "Type (set all):": "类型（全部设置）：",
    "Reference (set all):": "参考基因组（全部设置）：",
    "Composite Type:": "复合类型：",
    "Reference:": "参考基因组：",
    "Collection:": "集合：",
    "Upload type:": "上传类型：",
    "There is no description available for this file extension.": "这种文件扩展名没有可用的描述。",
    "This system will try to detect the file type automatically. If your file is not detected properly as one of the known formats, it most likely means that it has some format problems (e.g., different number of columns on different rows). You can still coerce the system to set your data to the format you think it should be.  You can also upload compressed files, which will automatically be decompressed.": "尝试自动检测文件类型。如果您的文件没有被正确地识别为已知格式之一，很可能意味着它存在一些格式问题（例如，不同行的列数不同）。您仍然可以强制系统将您的数据设置为您认为它应该是的格式。您也可以上传压缩文件，这些文件将自动被解压缩。",

    // ---------------------------------------------------------------------------- default-row
    //"Upload configuration": "上传配置",
    "Download data from the web by entering URLs (one per line) or directly paste content.": "通过输入网址（每行一个）从网络下载数据，或者直接粘贴内容。",
    "You added": "您已将",
    "file(s) to the queue. Add more files or click 'Start' to proceed.": "个文件添加到队列中。添加更多文件或点击 '开始' 以继续。",
    // ---------------------------------------------------------------------------- default-view
    "Choose local files": "选择本地文件",
    "Choose remote files": "选择远程文件",
    "Paste/Fetch data": "粘贴/获取数据",
    Start: "开始",
    Pause: "暂停",
    Reset: "重置",
    Close: "关闭",
    Cancel: "取消",
    Build: "构建",
    "Drop files here": "拖动文件到这里",
    // ---------------------------------------------------------------------------- composite-row
    Select: "选择",
    "Select a composite type": "选择一种复合类型",
    // ---------------------------------------------------------------------------- rule-based
    "Insert tabular source data to extract collection files and metadata.": "插入表格源数据以提取集合文件和元数据。",
    "Insert tabular source data here.": "在这里插入表格源数据。",
    "Paste data": "粘贴数据",
    "Choose dataset": "选择数据集",

    // History
    History: "历史",
    "Create new history": "创建新的历史",
    "Switch to history": "切换历史",
    "History options": "历史选项",
    "You have": "您有",
    "histories.": "个历史记录",
    "Show Histories Side-by-Side": "并列显示历史",
    "Resume Paused Jobs": "恢复暂停的任务",
    "Copy this History": "复制历史",
    "Delete this History": "删除历史",
    "Export Tool Citations": "导出工具引用",
    "Export History to File": "导出历史到文件",
    "Archive History": "归档历史",
    "Extract Workflow": "提取工作流程",
    "Show Invocations": "显示调用",
    "Share or Publish": "分享或发布",
    "Set Permissions": "设置权限",
    "Make Private": "设为私有",

    "With entire history...": "对整个历史进行……",
    "Copy Datasets": "复制数据集",
    "Unhide All Hidden Content": "显示所有隐藏内容",
    "Delete All Hidden Content": "删除所有隐藏内容",
    "Purge All Deleted Content": "清除所有已删除内容",

    "of": "/",
    "selected": "已选择",
    "With": "已选择",
    "selected...": "个数据集",
    Hide: "隐藏",
    Delete: "删除",
    "Delete (permanently)": "清除（永久删除)",
    "Build Dataset List": "构建数据集列表",
    "Build Dataset Pair": "构建配对数据集",
    "Build List of Dataset Pairs": "构建配对数据集列表",
    "Build Collection from Rules": "通过规则构建集合",
    "Change Database/Build": "改变数据库/构建",
    "Change data type": "改变数据类型",
    "Add tags": "添加标签",
    "Remove tags": "移除标签",

    // User-preferences
    "User preferences": "用户偏好",
    "You are signed in as": "您以",
    "and you are using": "的身份登录，目前已使用",
    "of disk space.": "磁盘空间。",
    "If this is more than expected, please visit the": "如果这比预期的要多，请访问",
    "to free up disk space.": "以释放磁盘空间。",
    "Your disk quota is:": "您的磁盘配额为",
    "Manage Information": "管理信息",
    "Edit your email, addresses and custom parameters or change your public name.": "编辑您的电子邮件、地址和自定义参数，或更改您的账号。",
    "Change Password": "更改密码",
    "Allows you to change your login credentials.": "允许您更改登录凭证。",
    "Manage Toolbox Filters": "管理工具箱过滤器",
    "Customize your Toolbox by displaying or omitting sets of Tools.": "通过显示或删除工具来自定义您的工具箱。",
    "Set Dataset Permissions for New Histories": "为新的历史设置数据集权限",
    "Grant others default access to newly created histories. Changes made here will only affect histories created after these settings have been stored.": "授予他人对新创建的历史记录的默认访问权限。此处所做的更改仅会影响在这些设置被保存后创建的历史记录。",
    "Manage API Key": "管理 API 密钥",
    "Access your current API key or create a new one.": "访问您当前的 API 密钥或创建一个新的。",
    "Manage Notifications": "管理通知",
    "Manage your notification settings.": "管理您的通知设置。",
    "Manage Cloud Authorization": "管理云端授权",
    "Add or modify the configuration that grants Galaxy to access your cloud-based resources.": "添加或修改配置，以允许Galaxy访问您的云端资源。",
    "Manage Custom Builds": "管理自定义构建",
    "Add or remove custom builds using history datasets.": "使用历史数据集添加或删除自定义构建。",
    "Click here to show or hide the activity bar while it is in beta. In a future release, this will be the default behavior.": "点击此处以显示或隐藏活动栏，目前它仍处于测试阶段。在未来的版本中，这将变成默认行为。",
    "Pick a Color Theme": "选择一个颜色主题",
    "Click here to change the user interface color theme.": "点击这里更改用户界面的颜色主题。",
    "Text": "文本",
    "Hover": "悬停",
    "Active": "活动",
    "Make All Data Private": "使所有数据私有",
    "Click here to make all data private.": "点击这里使所有数据私有。",
    "Preferred Storage Location": "首选存储位置",
    "Select a Preferred storage location for the outputs of new jobs.": "为新任务的输出选择一个首选存储位置。",
    "Delete Account": "删除账户",
    "Delete your account on this Galaxy server.": "删除您在本Galaxy服务器上的账户。",
    "Sign out": "退出登录",
    "Click here to sign out of all sessions.": "点击这里退出登录。",
    "Do you want to continue and sign out of all active sessions?": "您是否要继续并注销所有活动会话？",

    // User Preferences - managment information
    "Configuration option for your search results": "搜索结果的配置选项",
    "Maximum number of search results": "最大搜索结果数",
    "Do you want to be able to re-use previously run jobs ?": "您是否希望能够重复使用之前运行过的任务？",
    "Do you want to be able to re-use  equivalent jobs ?": "你想要能够重复使用等价的工作吗？",
    "Prefered language": "首选语言",
    "Your Invenio RDM Account": "您的 Invenio 研究数据管理账户",
    "Personal Token to publish records to Invenio RDM": "个人令牌，用于发布记录到 Invenio RDM（Invenio 研究数据管理器）",
    "Public name to publish records (formatted as \"Lastname, Firstname\")": "公开发布记录的姓名（格式为“姓氏，名字”）",
    "Whether to make new draft records publicly accessible (Yes) or restricted (No).": "是否将新草稿记录设为公开可访问（是）或受限（否）。",

    // Storage dashboard
    "Storage Dashboard": "存储仪表板",
    "Here you can see an overview of your disk usage status.": "在这里，可以看到您的磁盘使用情况概览。",
    "You've got": "您总共有",
    "of total disk quota": "的磁盘配额",
    "This is the maximum disk space that you can use across all your storage sources. Unlimited storage sources are not taken into account": "这是您可以在所有存储源中使用的最大磁盘空间。不受限制的存储源不计算在内。",
    "storage source": "存储源",
    "% of disk quota used": "% 已使用磁盘配额",
    Refresh: "刷新",
    "Recalculating disk usage... this may take some time, please check back later.": "正在重新计算磁盘使用情况...这可能需要一些时间，请稍后再查看。",
    "Is your usage more than expected?": "您的使用量是否超出预期？",
    "Find out what is eating up your space and learn how to easily and safely free up some of it.": "找出是什么占用了您的空间，并了解如何轻松且安全地释放其中的一部分空间。",
    "Free up disk usage": "释放磁盘使用空间",
    "Visually explore your disk usage": "可视化地探索您的磁盘使用情况",
    "Want to know what histories or datasets take up the most space in your account? Here you can explore your disk usage in a visual way.": "想知道哪些历史记录或数据集占用了您账户中的大部分空间吗？在这里，您可以以可视化的方式探索您的磁盘使用情况。",
    "Explore now": "立即探索",

    // Register and log in
    "Create a Galaxy account": "创建一个Galaxy账户",
    "Email address": "邮箱地址",
    "Password": "密码",
    "Confirm password": "确认密码",
    "Public name": "账号名称",
    "Subscribe to mailing list": "订阅邮件列表",
    "Your public name is an identifier that will be used to generate addresses for information you share publicly. Public names must be at least three characters in length and contain only lower-case letters, numbers, dots, underscores, and dashes ('.', '_', '-').": "您的账号是一个标识符，将用于为您公开共享的信息生成地址。账号长度至少为3个字符，只能包含小写字母、数字、点、下划线和横线。（“.”，“_”，“-”）。",
    'Your public name is an identifier that will be used to generate addresses for information you share publicly. Public names must be at least three characters in length and contain only lower-case letters, numbers, dots, underscores, and dashes (".", "_", "-").': "您的账号是一个标识符，将用于为您公开共享的信息生成地址。账号长度至少为3个字符，只能包含小写字母、数字、点、下划线和横线。（“.”，“_”，“-”）。",
    "Create": "创建",
    "Already have an account?": "已经有了一个账户？",
    "Log in here.": "请登录。",

    "Welcome to Galaxy, please log in": "欢迎来到 Galaxy，请登录",
    "Public Name or Email Address": "账号或邮箱",
    "Forgot password?": "忘记密码？",
    "Click here to reset your password.": "请点击此处重置您的密码。",
    "Don't have an account?": "还没有账号？",
    "Register here.": "这里注册。",

    // Additional Options
    "Additional Options": "邮件通知",
    "Email notification": "任务完成后是否发送邮件通知?",
    "Yes": "是",
    "No": "否",
    "Send an email notification when the job completes.": "您在使用过程中碰到任何问题，欢迎联系Galaxy团队获得支持，微信：usegalaxy，QQ群：925694514",

    // Other
    "Requirements": "依赖",
    "Localization": "本地化",
    "Administration": "管理",

    // curated end ----------------------------------------------------------------------------------


    // APPEND Text From Locale Auto Translated By Kimi
    "Log in or Register": "登录或注册",
    "Using": "使用",
    "Shared Data": "共享数据",
    "Published Histories": "发布历史",
    "Published Workflows": "发布工作流",
    "Published Visualizations": "发布可视化",
    "Published Pages": "发布页面",
    "New Track Browser": "新轨道浏览器",
    "Saved Visualizations": "保存的可视化",
    "Mailing Lists": "邮件列表",
    "Wiki": "维基",
    "Register": "注册",
    "Custom Builds": "自定义构建",
    "Logout": "退出",
    "Saved Histories": "保存的历史记录",
    "Saved Datasets": "保存的数据集",
    "Saved Pages": "保存的页面",
    "Histories shared with me": "与我分享的历史",
    "Visualizations": "可视化",
    "Log in or register a new account": "登录或注册新账户",
    "Access published resources": "访问已发布的资源",
    "Analysis home view": "分析主页视图",
    "Log in for details.": "登录以获取详细信息。",
    "Click for details.": "点击查看详情。",
    "Toggle Advanced Search": "高级搜索切换",
    "Clear Search (esc)": "清除搜索（esc）",
    "Interactive Tools": "交互式工具",
    "Show active interactive tools": "显示活动交互式工具",
    "View your dataset": "查看您的数据集",
    "History Actions": "历史操作",
    "History Lists": "历史列表",
    "Histories Shared with Me": "与我共享的历史",
    "Current History": "当前历史",
    "Show Structure": "显示结构",
    "Delete Permanently": "永久删除",
    "Dataset Actions": "数据集操作",
    "Dataset Security": "数据集安全",
    "Collapse Expanded Datasets": "合并扩展数据集",
    "Unhide Hidden Datasets": "显示隐藏数据集",
    "Delete Hidden Datasets": "删除隐藏的数据集",
    "Purge Deleted Datasets": "清除已删除的数据集",
    "Downloads": "下载",
    "Other Actions": "其他操作",
    "Import from File": "从文件导入",
    "Webhooks": "Webhooks",
    "This history is empty": "这段历史是空的",
    "No matching datasets found": "未找到匹配的数据集",
    "An error occurred while getting updates from the server": "在从服务器获取更新时发生了一个错误",
    "Please contact a Galaxy administrator if the problem persists": "如果问题持续存在，请联系Galaxy管理员。",
    "An error was encountered while <% where %>": "遇到了一个错误，位于 <% where %>",
    "You are currently viewing a deleted history!": "您正在查看已删除的历史记录！",
    "You are over your disk quota": "您的磁盘配额已超。",
    "Tool execution is on hold until your disk usage drops below your allocated quota": "工具执行将暂停，直到您的磁盘使用量降至分配的配额以下。",
    "All": "所有",
    "None": "无",
    "For all selected": "对于所有选定的",
    "Edit history tags": "编辑历史标签",
    "Edit history annotation": "编辑历史注释",
    "Click to rename history": "点击重命名历史",
    "Operations on multiple datasets": "多数据集上的操作",
    "Hide datasets": "隐藏数据集",
    "Unhide datasets": "显示数据集",
    "Delete datasets": "删除数据集",
    "Undelete datasets": "恢复数据集",
    "Permanently delete datasets": "永久删除数据集",
    "This will permanently remove the data in your datasets. Are you sure?": "这将永久删除您的数据集中的数据。您确定吗？",
    "Dataset": "数据集",
    "Annotation": "注释",
    "This history is empty. Click 'Get Data' on the left tool menu to start": "这个历史记录是空的。点击左侧工具菜单上的“获取数据”开始。",
    "You must be logged in to create histories": "您必须登录才能创建历史记录",
    "You can <% loadYourOwn %> or <% externalSource %>": "您可以<% loadYourOwn %>或<% externalSource %>",
    "You can": "你可以",
    " or": "或",
    "load your own data": "加载你自己的数据",
    "Include Deleted Datasets": "包含已删除的数据集",
    "Include Hidden Datasets": "包含隐藏数据集",
    "Unable to purge dataset": "无法清除数据集",
    "Cannot display datasets removed from disk": "无法显示已从磁盘中移除的数据集",
    "This dataset must finish uploading before it can be viewed": "这个数据集必须在可以查看之前完成上传",
    "This dataset is not yet viewable": "这个数据集目前还无法查看",
    "View data": "查看数据",
    "Download dataset": "下载数据集",
    "Additional files": "附加文件",
    "View details": "查看详情",
    "This is a new dataset and not all of its data are available yet": "这是一个新数据集，其数据尚未全部可用。",
    "You do not have permission to view this dataset": "您没有权限查看此数据集",
    "The job creating this dataset was cancelled before completion": "创建此数据集的任务在完成之前被取消了",
    "This job is waiting to run": "这个任务正在等待运行",
    "This dataset is currently uploading": "这个数据集目前正在上传",
    "Metadata is being auto-detected": "元数据正在自动检测",
    "This job is currently running": "这个任务目前正在运行",
    "This job is paused. Use the \"Resume Paused Jobs\" in the history menu to resume": "这个任务已暂停。使用历史菜单中的“恢复暂停的任务”来恢复任务。",
    "An error occurred with this dataset": "出现了一个与这个数据集相关的错误",
    "No data": "无数据",
    "An error occurred setting the metadata for this dataset": "在为这个数据集设置元数据时发生了一个错误",
    "There was an error getting the data for this dataset": "获取此数据集的数据时发生了错误。",
    "This dataset has been deleted and removed from disk": "此数据集已被删除并从磁盘中移除",
    "This dataset has been deleted": "这个数据集已被删除",
    "This dataset has been hidden": "这个数据集已经被隐藏",
    "Edit attributes": "编辑属性",
    "Cannot edit attributes of datasets removed from disk": "无法编辑从磁盘中移除的数据集的属性",
    "Undelete dataset to edit attributes": "恢复数据集以编辑属性",
    "This dataset must finish uploading before it can be edited": "这个数据集必须在可以编辑之前完成上传",
    "This dataset is not yet editable": "这个数据集目前还不可编辑",
    "Dataset is already deleted": "数据集已被删除",
    "View or report this error": "查看或报告此错误",
    "Run this job again": "再次运行此作业",
    "Visualize in": "在中可视化",
    "Undelete it": "恢复它",
    "Permanently remove it from disk": "永久地从磁盘上删除它",
    "Unhide it": "显示它",
    "You may be able to": "您可能能够",
    "set it manually or retry auto-detection": "手动设置或重试自动检测",
    "Edit dataset tags": "编辑数据集标签",
    "Edit dataset annotation": "编辑数据集注释",
    "Tool Help": "工具帮助",
    "Reset passwords": "重置密码",
    "Search Tool Shed": "搜索工具库",
    "Monitor installing repositories": "监控安装仓库",
    "Manage installed tools": "管理已安装的工具",
    "Reset metadata": "重置元数据",
    "Download local tool": "下载本地工具",
    "Tool lineage": "工具谱系",
    "Reload a tool's configuration": "重新加载工具的配置",
    "View Tool Error Logs": "查看工具错误日志",
    "Manage Display Whitelist": "管理显示白名单",
    "Manage Tool Dependencies": "管理工具依赖",
    "Users": "用户",
    "Groups": "组",
    "API keys": "API密钥",
    "Impersonate a user": "模拟用户",
    "Quotas": "配额",
    "Roles": "角色",
    "Local data": "本地数据",
    "Form Definitions": "表单定义",
    "Edit annotation": "编辑注释",
    "Subscript": "下标",
    "Superscript": "上标",
    "Trackster": "Trackster",
    "Any additional comments you can provide regarding what you were doing at the time of the bug.": "您能提供任何关于出现错误时您正在做什么的额外评论吗？",
    "Your email address": "你的电子邮件地址",
    "Report": "报告",
    "Error Report": "错误报告",
    "Dataset details": "数据集详情",
    "Save permissions.": "保存权限。",
    "Change the datatype to a new type.": "将数据类型更改为新类型。",
    "Convert the datatype to a new format.": "将数据类型转换为新格式。",
    "Save attributes of the dataset.": "保存数据集的属性。",
    "Save permissions": "保存权限",
    "Manage dataset permissions": "管理数据集权限",
    "Change datatype": "更改数据类型",
    "Convert datatype": "转换数据类型",
    "Convert to new format": "转换为新格式",
    "Import into History": "导入到历史记录",
    "Deleting selected items": "删除选定的项目",
    "Please select folders or files": "请选择文件夹或文件",
    "Please enter paths to import": "请输入导入的路径",
    "Adding datasets from your history": "从您的历史记录中添加数据集",
    "Create New Folder": "创建新文件夹",
    "Create New Library": "创建新库",
    "Tours": "图尔",
    "Manage Activity Bar": "管理活动栏",
    "Associate OpenIDs with your account.": "将OpenIDs与您的账户关联。",
    "Enable or disable the communication feature to chat with other users.": "启用或禁用与其他用户聊天的通信功能。",
    "Manage OpenIDs": "管理 OpenID",
    "Change communication settings": "更改通信设置",
    "Configured Galaxy Tool Sheds": "配置好的Galaxy工具仓库",
    "Repository Installation Queue": "仓库安装队列",
    "Repository Status": "仓库状态",
    "Workflows Missing Tools": "工作流缺少工具",
    "Share": "共享",
    "Workflow submission failed": "工作流提交失败",
    "Run workflow": "运行工作流",
    "Job submission failed": "作业提交失败",
    "Execute": "执行",
    "Tool request failed": "工具请求失败",
    "Get Data": "获取数据",
    "GENERAL TEXT TOOLS": "通用文本工具",
    "Copy and insert individual steps": "复制并插入单独步骤",
    "Warning": "警告",
    "An email notification will be sent when the job has completed.": "当作业完成时，将发送电子邮件通知。",
    "Add a step label.": "添加一个步骤标签。",
    "Assign columns": "分配列",
    "Delete this repeat block": "删除这个重复块",
    "placeholder": "占位符",
    "Repeat": "重复",
    "Browse GenomeSpace": "浏览基因组空间",
    "Browse": "浏览",
    "Error": "错误",
    "Create a collection of paired datasets": "创建一组配对数据集的集合",
    "View all histories": "查看所有历史记录",
    "Refresh history": "刷新历史",
    "View error logs": "查看错误日志",
    "View lineage": "查看谱系",
    "Manage dependencies": "管理依赖",
    "Manage whitelist": "管理白名单",
    "Manage metadata": "管理元数据",
    "Manage tools": "管理工具",
    "Monitor installation": "监控安装",
    "Install new tools": "安装新工具",
    "Tool Management": "工具管理",
    "Forms": "表单",
    "User Management": "用户管理",
    "Manage jobs": "管理作业",
    "Display applications": "显示应用程序",
    "Data tables": "数据表",
    "Data types": "数据类型",
    "Server": "服务器",
    "Could Not Save": "无法保存",
    "Saving...": "保存中...",
    "Add tracks": "添加轨迹",
    "New Visualization": "新可视化",
    "Add Data to Saved Visualization": "添加数据到保存的可视化",
    "Close visualization": "关闭可视化",
    "Circster": "Circster",
    "Bookmarks": "书签",
    "Add group": "添加组",
    "Remove parameter from tree": "从树中移除参数",
    "Add parameter to tree": "添加参数到树",
    "Remove": "移除",
    "Select datasets for new tracks": "选择新轨迹的数据集",
    "Libraries": "库",
    "Zoom out": "缩小",
    "Zoom in": "放大",
    "Phyloviz Help": "Phyloviz 帮助",
    "Save visualization": "保存可视化",
    "PhyloViz Settings": "PhyloViz 设置",
    "Title": "标题",
    "Filtering Dataset": "过滤数据集",
    "Filter Dataset": "过滤数据集",
    "Show individual tracks": "显示单独的轨道",
    "Trackster Error": "Trackster错误",
    "Tool parameter space visualization": "工具参数空间可视化",
    "Tool": "工具",
    "Set as overview": "设置为概览",
    "Set display mode": "设置显示模式",
    "Filters": "过滤器",
    "Show composite track": "显示复合轨道",
    "Edit settings": "编辑设置",
    "Test title": "测试标题",
    "Test Title": "测试标题",
    "Test button": "测试按钮",
    "title": "标题",
    "Create new Build": "创建新的构建",
    "Delete custom build.": "删除自定义构建。",
    "Provide the data source.": "提供数据来源。",
    "Next in History": "历史记录中的下一个",
    "Previous in History": "历史记录上一个",
    "Chat online": "在线聊天",
    "Multiple collections": "多个集合",
    "Dataset collections": "数据集集合",
    "Dataset collection": "数据集收集",
    "Multiple datasets": "多个数据集",
    "Single dataset": "单个数据集",
    "tool tooltip": "工具提示",
    "Analyze Data": "分析数据",
    "Account registration or login": "账户注册或登录",
    "Create New": "创建新记录",
    "Copy History": "复制历史",
    "Manage Allowlist": "管理允许列表",
    "Edit dataset attributes": "编辑数据集属性",
    "Manage custom builds": "管理自定义构建",
    "Manage Toolbox filters": "管理工具箱过滤器",
    "Set dataset permissions for new histories": "为新的历史记录集设置数据集权限",
    "Configured Tool Sheds": "配置好的工具仓库",
    "Repository Queue": "存储库队列",
    "Upload configuration": "上传配置",
    "FTP files": "FTP文件",
    "Choose FTP file": "选择FTP文件",
    "Choose FTP files": "选择FTP文件",
    "Manage allowlist": "管理允许列表",
    "Terms and Conditions": "条款和条件",
    "Active InteractiveTools": "活动交互工具",
    "(empty)": "（空）",
    "Copy": "复制",
    "Beta Features": "Beta特性",
    "Use Beta History Panel": "使用Beta历史面板",
    "search all datasets": "搜索所有数据集",
    "search histories": "搜索历史记录",
    "Search tips": "搜索技巧",
    "clear search": "清除搜索",
    "Rename history...": "重命名历史...",
    "Search Workflows": "搜索工作流",
    "Updated": "更新了",
    "Sharing": "分享",
    "Bookmarked": "已收藏",
    "Import": "导入",
    "Copy link": "复制链接",
    "Visualize this data": "可视化这些数据",
    "Manage": "管理",
    "This will change the datatype of the existing dataset but not modify its contents. Use this if Galaxy has incorrectly guessed the type of your dataset.": "这将更改现有数据集的数据类型，但不会修改其内容。如果Galaxy错误地猜测了您数据集的类型，请使用此功能。",
    "Info": "信息",
    "Add an annotation or notes to a dataset; annotations are available when a history is viewed.": "为数据集添加注释或笔记；在查看历史记录时，注释是可用的。",
    "Manage Third-Party Identities": "管理第三方身份",
    "Connect or disconnect access to your third-party identities.": "连接或断开对您第三方身份的访问。",
    "Add or modify the configuration that grants Galaxy to access your cloud-based resources": "添加或修改配置，以允许Galaxy访问您的基于云的资源。",
    "Enable notifications": "启用通知",
    "Allow push and tab notifcations on job completion. To disable, revoke the site notification privilege in your browser.": "允许在作业完成时推送和显示通知。要禁用，请在您的浏览器中撤销站点通知权限。",
    "You are using": "您正在使用",
    "of disk space in this Galaxy instance.": "在这个Galaxy实例中的磁盘空间。",
    "Is your usage more than expected? See the": "你的使用量超出预期了吗？请参阅",
    "documentation": "文档",
    "for tips on how to find all of the data in your account.": "对于如何找到您账户中的所有数据的提示。",
    "If you change your email address you will receive an activation link in the new mailbox and you have to activate your account by visiting it.": "如果您更改了电子邮件地址，您将收到一个激活链接到新的邮箱，并且您需要通过访问该链接来激活您的账户。",
    "Insert address": "插入地址",
    "Do you want to be able to re-use equivalent jobs ?": "您是否希望能够重用等价的作业？",
    "If you select yes, you will be able to select for each tool and workflow run if you would like to use this feature.": "如果您选择是，您将能够为每个工具和工作流程运行选择是否使用此功能。",
    "Use distributed compute resources": "使用分布式计算资源",
    "Remote resource id": "远程资源ID",
    "default - Galaxy will decide where to put your jobs": "默认 - Galaxy将决定在哪里放置您的作业",
    "Your EGA (European Genome Archive) Account": "您的欧洲基因组档案库（EGA）账户",
    "Username": "用户名",
    "Your Dropbox Account": "你的Dropbox账号",
    "Dropbox access token": "Dropbox 访问令牌",
    "Your B2DROP Account": "您的B2DROP账户",
    "Your OMERO instance connection details": "您的OMERO实例连接详情",
    "Your ENA Webin account details": "你的ENA Webin账户详情",
    "ENA Webin ID": "ENA Webin 编号",
    "Your CDS API Key": "你的CDS API密钥",
    "Key": "关键",
    "Search datasets": "搜索数据集",
    "Search Tool Shed (Beta)": "搜索工具仓库（测试版）",
    "Form definitions": "表单定义",
    "Tools and Tool Shed": "工具和工具箱",
    "Your workflows": "你的工作流程",
    // APPEND Text From Vue Auto Translated By Kimi
    "No matching activities found.": "未找到匹配的活动。",
    "Edit annotation...": "编辑注释...",
    ".": ".",
    "At least one element is needed for the collection. You may need to": "至少需要一个元素来创建集合。你可能需要",
    "Clear selected": "清除选定",
    "Click": "点击",
    "Click the": "点击",
    "Collections cannot have duplicated names. The following list names are duplicated:": "集合不能有重复的名称。以下列出的名称重复了：",
    "Create list": "创建列表",
    "De-select all selected datasets": "取消选择所有选中的数据集",
    "Discard": "丢弃",
    "Discard elements from the final created list by clicking on the": "从最终创建的列表中通过点击来丢弃元素",
    "Galaxy could not be reached and may be updating.  Try again in a few minutes.": "Galaxy 无法连接，可能正在更新。请几分钟后再试。",
    "No datasets were selected": "没有选择数据集",
    "No elements left. Would you like to": "没有剩余元素。您想要",
    "Once your collection is complete, enter a": "一旦你的收藏完成，输入一个",
    "Please fix these duplicates and try again.": "请修复这些重复项，然后再次尝试。",
    "Rename elements in the list by clicking on": "在列表中通过点击重命名元素",
    "Reorder the list by clicking and dragging elements. Select multiple elements by clicking on": "通过点击并拖动元素来重新排序列表。通过点击选择多个元素。",
    "Sort datasets by name": "按名称对数据集进行排序",
    "The following selections could not be included due to problems:": "由于存在问题，以下选择无法包括在内：",
    "Undo all reordering and discards": "撤销所有重新排序和丢弃的操作",
    "and click": "并点击",
    "and reselect new elements.": "并重新选择新元素。",
    "and you can then move those selected by dragging the entire group. Deselect them by clicking them again or by clicking the": "然后你可以通过拖动整个组来移动那些被选中的。再次点击它们或点击取消选择。",
    "button to exit the interface.": "退出界面的按钮。",
    "button.": "按钮。",
    "cancel": "取消",
    "has been deleted or purged": "已被删除或清除",
    "has errored, is paused, or is not accessible": "出现错误，已暂停，或无法访问",
    "is a collection, this is not allowed": "这是一个集合，这是不允许的。",
    "link.": "链接。",
    "name": "名称",
    "start over": "重新开始",
    "the existing name": "现有的名称",
    "them": "它们",
    "to begin again as if you had just opened the interface.": "重新开始，就像你刚刚打开了界面一样。",
    "to sort datasets alphabetically.": "对数据集进行按字母顺序排序。",
    "Click to rename": "点击重命名",
    "Remove this dataset from the list": "从列表中移除这个数据集",
    "Exactly two elements needed for the collection. You may need to": "确切地需要两个元素来收集。你可能需要",
    "No datasets were selected.": "没有选择数据集。",
    "Swap": "交换",
    "Two (and only two) elements are needed for the pair. You may need to": "需要两个（且仅有两个）元素来组成配对。你可能需要",
    "forward": "正向",
    "link to make your forward dataset the reverse and the reverse dataset forward": "链接，使您的正向数据集变成反向，反向数据集变成正向。",
    "reverse": "反转",
    "Unpair": "未配对",
    ". (Note: you do not have to pair all unpaired datasets to finish.)": "（注意：您不必将所有未配对的数据集配对才能完成。）",
    "At least two elements are needed for the collection. You may need to": "至少需要两个元素来收集。你可能需要",
    "Auto-pair": "自动配对",
    "Choose from common filters": "选择常见过滤器",
    "Choosing from a list of preset filters by clicking the arrow beside the filter input.": "通过点击过滤器输入旁边的箭头从预设过滤器列表中选择。",
    "Clear Filters": "清除过滤器",
    "Clear filters link": "清除筛选器链接",
    "Clearing the filters by clicking the": "通过点击来清除过滤器",
    "Click a dataset in the": "点击一个数据集在",
    "Click one of the Pair these datasets buttons in the middle column to pair the datasets in a particular row.": "点击中间列中的一个“配对这些数据集”按钮，以在特定行中配对数据集。",
    "Could not automatically create any pairs from the given dataset names. You may want to choose or enter different filters and try auto-pairing again.": "无法自动从给定的数据集名称创建任何配对。您可能需要选择或输入不同的过滤器，然后再次尝试自动配对。",
    "Drag to change": "拖动以更改",
    "Entering partial dataset names in either the": "在任一选项中输入部分数据集名称",
    "Entering regular expressions to match dataset names. See:": "输入正则表达式以匹配数据集名称。参见：",
    "Filter text": "过滤文本",
    "MDN's JavaScript Regular Expression Tutorial": "MDN的JavaScript正则表达式教程",
    "No datasets were found matching the current filters.": "没有找到与当前筛选器匹配的数据集。",
    "Note: forward slashes (\\\\) are not needed.": "注意：不需要使用正斜杠（\\）。",
    "Pair these datasets": "配对这些数据集",
    "Paired datasets are shown in the": "配对数据集显示在",
    "Remove file extensions from pair names?": "移除配对名称中的文件扩展名？",
    "To pair datasets, you can:": "要配对数据集，你可以：",
    "To unpair individual dataset pairs, click the": "要解对单个数据集对，请点击",
    "Unpair all": "取消配对",
    "Unpaired datasets are shown in the": "未配对的数据集显示在",
    "Use this box to filter elements, using simple matching or regular expressions.": "使用此框筛选元素，可以使用简单匹配或正则表达式。",
    "You can filter what is shown in the unpaired sections by:": "您可以通过以下方式过滤未配对部分中显示的内容：",
    "You can include or remove the file extensions (e.g. \".fastq\") from your pair names by toggling the": "您可以通过切换来包含或移除配对名称中的文件扩展名（例如“.fastq”）。",
    "control.": "控制。",
    "filtered out": "过滤掉",
    "forward column": "前向列",
    "forward filter": "前向过滤",
    "link to unpair all pairs.": "链接以取消所有配对。",
    "or": "或者",
    "paired section": "配对部分",
    "pairs": "配对",
    "reverse column": "反转列",
    "reverse filter": "反向筛选",
    "to have your datasets automatically paired based on name.": "以名称自动配对您的数据集。",
    "to select it then click a dataset in the": "选择它，然后点击一个数据集",
    "unpair buttons (": "成对按钮（",
    "unpaired forward": "未配对的正向",
    "unpaired reverse": "未配对的反向",
    "unpaired section": "非配对部分",
    "New Type": "新类型",
    "Save": "保存",
    "Close and show more help": "关闭并显示更多帮助",
    "Create a <i>single</> pair": "创建一个<i>单一</i>的配对",
    "Create collection": "创建集合",
    "Enter a name for your new collection": "输入一个名称，为您的新集合命名",
    "Expand or Close Help": "展开或关闭帮助",
    "Hide original elements?": "隐藏原始元素？",
    "Name:": "名称：",
    "Convert": "转换",
    "Database/Build": "数据库/构建",
    "Datatypes": "数据类型",
    "Edit Collection Attributes": "编辑集合属性",
    "Convert Collection": "转换集合",
    "Converter Tool:": "转换工具：",
    "Export": "导出",
    "Give the exported file a name.": "给导出的文件命名。",
    "Name": "名称",
    "Create new record": "创建新记录",
    "Export to existing draft record": "导出到现有的草稿记录",
    "Export to existing record": "导出到现有记录",
    "Export to new record": "导出到新记录",
    "Export to this record": "导出到此记录",
    "File name": "文件名",
    "Give the new record a name or title.": "给新记录命名或标题。",
    "Record name": "记录名称",
    "draft record has been created in the repository.": "草稿记录已在仓库中创建。",
    "Search": "搜索",
    "Assign Datatype": "分配数据类型",
    "Attributes": "属性",
    "Auto-detect": "自动检测",
    "Convert to Datatype": "转换为数据类型",
    "Create Dataset": "创建数据集",
    "Edit Dataset Attributes": "编辑数据集属性",
    "Permissions": "权限",
    "You must be logged in to receive emails": "您必须登录才能接收电子邮件。",
    "Link copied to your clipboard": "链接已复制到您的剪贴板",
    "No options available.": "没有可用的选项。",
    "Select / Deselect all": "选择 / 取消选择全部",
    "Dataset will be run against *all* other datasets.": "数据集将与所有其他数据集进行比对。",
    "Datasets will be run in matched order with other datasets.": "数据集将与其他数据集按匹配顺序运行。",
    "Linked:": "链接：",
    "This is a batch mode input field. Individual jobs will be triggered for each dataset.": "这是一个批量模式输入字段。每个数据集将触发单独的作业。",
    "Unlinked:": "未链接：",
    "Select / Deselect All": "全选/全不选",
    "Import History": "导入历史",
    "No entries found.": "未找到条目。",
    "Not available.": "不可用。",
    "Nothing found with:": "没有找到与以下内容相关的信息：",
    "Create Page": "创建页面",
    "Create Visualization": "创建可视化",
    "Amount of items in history": "历史记录中的项目数量",
    "Are you sure you want to restore this (purged) history? Please note that this will not restore the datasets associated with this history. If you want to fully recover it, you can import a copy from the export record instead.": "你确定要恢复这个（已清除）的历史记录吗？请注意，这不会恢复与此历史记录相关联的数据集。如果你想完全恢复它，你可以从导出记录中导入一个副本。",
    "Are you sure you want to restore this history? This will move the history back to your active histories.": "你确定要恢复这个历史记录吗？这将把历史记录移回你的活动历史记录中。",
    "History Import Failed": "历史导入失败",
    "History Restore Failed": "历史恢复失败",
    "History Restored": "历史恢复",
    "Import a new copy of this history from the associated export record": "从相关导出记录中导入此历史的新副本",
    "Importing History in background...": "在后台导入历史记录...",
    "Last edited/archived": "最后编辑/归档",
    "Published": "发布",
    "Snapshot available": "快照可用",
    "This history has an associated export record containing a snapshot of the history that can be used to import a copy of the history.": "这个历史记录有一个相关的导出记录，其中包含了一个历史记录的快照，可以用来导入这个历史记录的副本。",
    "This history is public.": "这段历史是公开的。",
    "Unarchive this history and move it back to your active histories": "解压缩这个历史记录，然后将其移回您的活动历史记录中。",
    "View this history": "查看这个历史记录",
    "items": "项目",
    "database": "数据库",
    "format": "格式",
    "Download Dataset": "下载数据集",
    "last edited": "最近编辑",
    "You can load your own data": "你可以加载你自己的数据",
    "get data from an external source": "从外部来源获取数据",
    "This history has been deleted": "这段历史已被删除",
    "You are over your disk quota. Tool execution is on hold until your disk usage drops below your allocated quota.": "您的磁盘配额已超限。在您的磁盘使用量降至分配的配额以下之前，工具执行将暂停。",
    "Export Citations for all Tools used in this History": "导出此历史记录中使用的所有工具的引用",
    "Export and Download History as a File": "导出并下载历史记录为文件",
    "Log in to": "登录到",
    "Permanently Delete History": "永久删除历史记录",
    "Resume all Paused Jobs in this History": "恢复此历史记录中所有暂停的作业",
    "This will make all the data in this history private (excluding library datasets), and will set permissions such that all new data is created as private. Any datasets within that are currently shared will need to be re-shared or published. Are you sure you want to do this?": "这将使这个历史记录中的所有数据变为私有（不包括库数据集），并将权限设置为创建的所有新数据都是私有的。其中任何当前已共享的数据集都需要重新共享或发布。您确定要这样做吗？",
    "Yes, permanently delete this history.": "是的，永久删除这个历史记录。",
    "Really delete all hidden datasets?": "真的要删除所有隐藏的数据集吗？",
    "Really permanently delete all deleted datasets?": "确实永久删除所有已删除的数据集？",
    "Really unhide all hidden datasets?": "真的要取消隐藏所有隐藏的数据集吗？",
    "Warning, this operation cannot be undone.": "警告，此操作无法撤销。",
    "The operation failed for the following reasons:": "操作失败的原因如下：",
    "Undelete": "恢复删除",
    "Unhide": "隐藏",
    "Last edited": "上次编辑",
    "Load More": "加载更多",
    "Open in center panel": "在中心面板中打开",
    "Open in multi-view": "在多视图中打开",
    "Set as current history": "设置为当前历史",
    "Loading History...": "正在加载历史...",
    "Edit": "编辑",
    "Log in to Rename History": "登录到重命名历史",
    "Not Editable": "不可编辑",
    "Change Selected": "更改选定",
    "Click a history to switch to it": "点击历史记录以切换到它",
    "Click here to reset selection": "点击此处重置选择",
    "Filter histories": "过滤历史记录",
    "Currently showing 4 most recently updated histories": "当前显示4个最近更新的历史",
    "No History found.": "未找到历史记录。",
    "Open modal to select/deselect histories": "打开模态框以选择/取消选择历史记录",
    "Recent": "最近",
    "Search datasets and collections in selected histories": "在选定的历史记录中搜索数据集和集合",
    "Show 4 most recently updated histories instead": "显示最近更新的4个历史记录",
    "Collection copied to new history": "集合已复制到新的历史记录中",
    "Create and pin new history": "创建并固定新的历史记录",
    "Create new history with this item": "创建一个包含此条目的新历史记录",
    "Dataset copied to new history": "数据集已复制到新的历史记录中",
    "Select histories": "选择历史记录",
    "Deleted": "已删除",
    "Purged": "清除",
    "Description": "描述",
    "Library": "库",
    "Synopsis": "摘要",
    "exclude restricted": "排除受限的",
    "include deleted": "包含已删除的",
    "per page": "每页",
    "total": "总数",
    "Created": "创建了",
    "Details": "详情",
    "Failed to retrieve library details.": "无法检索库详细信息。",
    "Folder": "文件夹",
    "Location Details": "位置详情",
    "Login": "登录",
    "Or, register with email": "或者，使用电子邮件注册",
    "↓ Scroll to review ↓": "向下滚动以查看",
    "1 history pinned. Click here to reset": "1 条历史记录已固定。点击此处重置",
    "Currently showing 4 most recently updated histories in Multiview": "当前在多视图中显示了4个最近更新的历史记录",
    "Open History Multiview": "打开历史多视图",
    "Reset selection to show 4 most recently updated histories instead": "重置选择以显示最近更新的4个历史记录",
    "Select histories to pin to Multiview": "选择要固定到多视图的历史记录",
    "Data Managers": "数据管理器",
    "search tools": "搜索工具",
    "Advanced Tool Search": "高级工具搜索",
    "Assign Another Column": "分配另一列",
    "From Column": "从列",
    "Select a column": "选择一列",
    "Edit column definition": "编辑列定义",
    "Remove this column definition": "删除这个列定义",
    "Enter a regular expression.": "输入一个正则表达式。",
    "Regular Expression": "正则表达式",
    "Regular Expressions": "正则表达式",
    "Apply": "应用",
    "Edit this rule.": "编辑这个规则。",
    "Remove this rule.": "删除这个规则。",
    "Recently used rules": "最近使用的规则",
    "Add nametag for name": "为名称添加名牌",
    "Advanced Option: View and or edit the JSON representation of the rules to apply to this tabular data": "高级选项：查看或编辑应用于此表格数据的规则的JSON表示",
    "Apply changes to rule source and return to rule preview": "应用规则源的更改并返回到规则预览",
    "Apply these column definitions and return to rules preview": "应用这些列定义并返回到规则预览",
    "Button is disabled due to validation errors. Please see alerts above.": "按钮因验证错误而禁用。请查看上面的警报。",
    "By default columns will be sorted lexicographically, check this option if the columns are numeric values and should be sorted as numbers": "默认情况下，列将按字典顺序排序，如果列是数值类型并且应该按数字排序，请勾选此选项。",
    "Close this modal and do not create any collections": "关闭这个模态框，不要创建任何集合",
    "Close this modal and do not upload any datasets": "关闭这个模态框，不要上传任何数据集",
    "Column": "列",
    "Create new collection from specified rules and datasets": "根据指定的规则和数据集创建新的集合",
    "Datasets submitted to Galaxy for creation, this dialog will close when dataset creation is complete. You may close this dialog at any time, but you will not be informed of errors with dataset creation and you may have to refresh your history manually to view new datasets once complete.": "提交到Galaxy以创建数据集，当数据集创建完成时，此对话框将关闭。您可以随时关闭此对话框，但您将不会收到有关数据集创建错误的信息，您可能需要手动刷新您的历史记录，以在完成后查看新数据集。",
    "Default": "默认",
    "Dismiss this error and return to the rule builder to try again with new rules": "忽略此错误并返回到规则构建器，尝试使用新的规则再次进行操作",
    "Filter": "过滤",
    "For": "对于",
    "Galaxy is waiting for collection creation, this dialog will close when this is complete. You may close this dialog at any time, but you will not be informed of errors with collection creation and you may have to refresh your history manually to view new collections once complete.": "Galaxy 正在等待创建集合，当此过程完成时，此对话框将关闭。您可以随时关闭此对话框，但您将不会收到有关集合创建错误的信息，并且您可能需要手动刷新您的历史记录，以查看完成后的新集合。",
    "General rules to apply": "一般规则的应用",
    "Genome": "基因组",
    "Hide original elements": "隐藏原始元素",
    "History ID (hid)": "历史记录 ID（hid）",
    "Invert filter.": "反转过滤。",
    "List Identifier": "列出标识符",
    "Number of Groups": "组数",
    "Numeric sorting.": "数值排序。",
    "Okay": "好的",
    "Paired Identifier": "配对标识符",
    "Path": "路径",
    "Prefix or suffix length": "前缀或后缀长度",
    "Remove column definition assignment": "移除列定义分配",
    "Remove rows matching supplied value": "删除匹配指定值的行",
    "Remove rows not matching the specified regular expression at specified column": "删除不符合指定列中指定正则表达式的行",
    "Remove rows that have non-empty values at specified column": "删除在指定列具有非空值的行",
    "Replacement Expression": "替代表达式",
    "Reset text area to current set of rules": "重置文本区域以当前规则集",
    "Rules": "规则",
    "Rules that filter rows from the data": "从数据中筛选行的规则",
    "Rules that generate new columns": "生成新列的规则",
    "Save rules and return to tool form": "保存规则并返回工具表单",
    "Starting from": "从",
    "Stop editing rules and dismiss changes": "停止编辑规则并放弃更改",
    "Tags": "标签",
    "These issues must be resolved to proceed:": "这些问题必须解决才能继续：",
    "Type": "类型",
    "URL": "URL",
    "URL (deferred)": "URL（延迟）",
    "Upload collection using specified rules": "使用指定规则上传集合",
    "Value": "值",
    "greater than": "大于",
    "greater than or equal to": "大于或等于",
    "less than": "小于",
    "less than or equal to": "小于或等于",
    "Copy Link": "复制链接",
    "Copy Tool ID": "复制工具ID",
    "Download": "下载",
    "See in Tool Shed": "查看工具箱",
    "Selected": "选择",
    "Switch to": "切换到",
    "Citations": "引用",
    "License": "许可",
    "References": "参考文献",
    "Run Tool": "运行工具",
    "Tool Parameters": "工具参数",
    "View Tool source": "查看工具源码",
    "Tutorials": "教程",
    "clear search (esc)": "清除搜索（Esc）",
    "search tours": "搜索旅行路线",
    "Choose local file": "选择本地文件",
    "Choose remote file": "选择远程文件",
    "Queue will pause after completing the current file...": "队列将在完成当前文件后暂停...",
    "Import FTP files": "导入FTP文件",
    "An API key will allow you to access via web API. Please note that this key acts as an alternate means to access your account and should be treated with the same care as your login password.": "一个API密钥将允许您通过Web API访问。请注意，此密钥作为访问您的帐户的替代方式，应像对待您的登录密码一样小心对待。",
    "Create a new key": "创建一个新的密钥",
    "Are you sure you want to delete this key?": "你确定要删除这个键吗？",
    "Failed to access disk usage details.": "无法访问磁盘使用详情。",
    "No items to clear": "没有项目需要清除",
    "Review and clear": "审查并清除",
    "Congratulations!": "恭喜！",
    "Freeing up some space...": "释放一些空间...",
    "Reason": "原因",
    "Something went wrong...": "出了些问题...",
    "Sorry, some items couldn't be cleared": "抱歉，一些项目无法清除",
    "I understand that once I delete the items, they cannot be recovered.": "我明白一旦我删除了这些项目，它们就无法恢复。",
    "Permanently delete": "永久删除",
    "To free up account space, review and select items to be permanently deleted here.": "为了释放账户空间，在这里审查并选择要永久删除的项目。",
    "Back to Dashboard": "返回仪表板",
    "Learn more": "了解更多",
    "Manage your account storage": "管理您的账户存储",
    "The storage manager only shows elements that count towards your disk quota.": "存储管理器仅显示计入您的磁盘配额的元素。",
    "This feature is currently in Beta, if you find any issues please report them": "这个功能目前处于测试阶段，如果你发现任何问题，请报告它们",
    "All your storage sources have unlimited disk space. Enjoy!": "您所有的存储源都有无限的磁盘空间。享受吧！",
    "Active vs Archived vs Deleted Total Size": "活动 vs 归档 vs 删除 总大小",
    "An error occurred while permanently deleting the history.": "在永久删除历史记录时发生了一个错误。",
    "An error occurred while undeleting the history.": "在恢复历史记录时发生了一个错误。",
    "Are you sure you want to permanently delete this history? This action cannot be undone.": "你确定要永久删除这个历史记录吗？这个操作无法撤销。",
    "History permanently deleted successfully.": "历史记录已成功永久删除。",
    "History undeleted successfully.": "历史记录已成功恢复。",
    "Loading your storage data. This may take a while...": "正在加载您的存储数据。这可能需要一些时间...",
    "Permanently delete history?": "永久删除历史记录？",
    "This graph shows the total size taken by your histories, split between active, archived and deleted histories.": "这个图表展示了您的历史记录所占用的总大小，分为活动、归档和已删除的历史记录。",
    "Active vs Deleted Total Size": "活动与已删除的总大小",
    "An error occurred while permanently deleting the dataset.": "在永久删除数据集时发生了一个错误。",
    "An error occurred while undeleting the dataset.": "在恢复数据集时发生了一个错误。",
    "Are you sure you want to permanently delete this dataset? This action cannot be undone.": "您确定要永久删除这个数据集吗？这个操作无法撤销。",
    "Dataset permanently deleted successfully.": "数据集已成功永久删除。",
    "Dataset undeleted successfully.": "数据集已成功恢复。",
    "Permanently delete dataset?": "永久删除数据集？",
    "This graph shows the total size of your datasets in this history, split between active and deleted datasets.": "这个图表显示了您在这个历史记录中的数据集的总大小，分为活动和已删除的数据集。",
    "Set this history as current": "将此历史记录设置为当前记录",
    "You can manage notifications channels and preferences here.": "你可以在这里管理通知渠道和偏好设置。",
    "You can manage push notifications preferences here.": "你可以在这里管理推送通知的偏好设置。",
    "Contribute variants to Beacon": "贡献变体到Beacon",
    "Manage Beacon": "管理信标",
    "All of your histories and datasets have been made private. If you'd like to make all *future* histories private please use the": "您所有的历史记录和数据集都已设为私有。如果您希望将所有*未来的*历史记录也设为私有，请使用",
    "interface": "接口",
    "Select a dataset to visualize:": "选择一个数据集进行可视化：",
    "There is no suitable dataset in your current history which can be visualized with this plugin.": "当前历史记录中没有合适的数据集可以被这个插件可视化。",
    "search visualizations": "搜索可视化",
    "No tool recommendations": "没有工具推荐",
    "Tool recommendations": "工具推荐",
    "Click 'Save' to save your changes and proceed, 'Don't Save' to discard them and proceed, or 'Cancel' to return to the editor.": "点击“保存”以保存更改并继续，点击“不保存”以放弃更改并继续，或者点击“取消”返回编辑器。",
    "Discard changes and proceed": "放弃更改并继续",
    "Do not run proceed and return to editor": "不要继续执行，返回到编辑器",
    "Don't Save": "不保存",
    "Save changes and proceed": "保存更改并继续",
    "You have unsaved changes. Do you want to save them before proceeding?": "您有未保存的更改。在继续之前您要保存它们吗？",
    "Submit": "提交",
    "View workflow invocations": "查看工作流调用",
    "Rename": "重命名",
    "Run": "运行",
    "Create Broadcast": "创建广播",
    "Update Broadcast": "更新广播",
    "Broadcasts are notifications that are displayed to all users. They can be scheduled to be published at a specific time and can be set to expire at a specific time. They can also be marked as urgent to make them stand out.": "广播是通知，它们会显示给所有用户。它们可以被安排在特定时间发布，并且可以设置在特定时间过期。它们还可以被标记为紧急，以使它们脱颖而出。",
    "Once you send the notification, it will be sent to all the recipients and cannot be edited or deleted.": "一旦您发送通知，它将发送给所有收件人，并且无法编辑或删除。",
    "Send Notification": "发送通知",
    // APPEND Text From XML-Lable Auto Translated By Kimi
    "Source for the reference genome": "参考基因组的来源",
    "Select a reference genome": "选择一个参考基因组",
    "Select the reference genome": "选择参考基因组",
    "Is this a single or paired library": "这是一个单端还是配对的文库",
    "FASTA/Q file": "FASTA/Q文件",
    "Specify strand information": "指定链信息",
    "FASTA/Q file #1": "FASTA/Q 文件 #1",
    "FASTA/Q file #2": "FASTA/Q 文件 #2",
    "Paired Collection": "配对集合",
    "Interleaved FASTA/Q file": "交错的FASTA/Q文件",
    "Summary Options": "摘要选项",
    "Output alignment summary in a more machine-friendly style.": "以更机器友好的风格输出对齐摘要。",
    "Print alignment summary to a file.": "将比对摘要打印到文件。",
    "Advanced Options": "高级选项",
    "Input options": "输入选项",
    "Skip the first N reads or pairs in the input": "跳过输入中的前N个读段或配对",
    "Align the first N reads or read pairs from the input (after the first N reads or pairs have been skipped), then stop": "对输入的前N个读段或读段对进行比对（跳过前N个读段或对之后），然后停止",
    "Trim 5' end": "修剪5'端",
    "Trim 3' end": "修剪3'端",
    "Select quality score encoding": "选择质量分数编码",
    "Convert input qualities from Solexa (which can be negative) to Phred (which can't). This scheme was used in older Illumina GA Pipeline versions (prior to 1.3)": "将输入的质量值从Solexa（可以是负数）转换为Phred（不能是负数）。这种方案在旧版的Illumina GA Pipeline（1.3之前）中使用。",
    "Are quality values provided as space separated integers?": "质量值是否以空格分隔的整数提供？",
    "Alignment options": "对齐选项",
    "Function governing the maximum number of ambiguous characters (usually Ns and/or .s) allowed in a read as a function of read length": "控制读取中允许的最大模糊字符数（通常是 N 和/或 .）的函数，作为读取长度的函数。",
    "Constant term (B)": "常数项 (B)",
    "Coefficient (A)": "系数（A）",
    "Ignore quality values": "忽略质量值",
    "Skip forward strand of reference": "跳过参考序列的正向链",
    "Skip reverse strand of reference": "跳过参考序列的反向链",
    "Scoring options": "评分选项",
    "Maximum mismatch penalty": "最大错配惩罚",
    "Minimum mismatch penalty": "最小错配惩罚",
    "Maximum soft-clipping penalty": "最大软裁剪惩罚",
    "Minimum soft-clipping penalty": "最小软剪切惩罚",
    "Allow soft-clipping": "允许软裁剪",
    "Ambiguous read penalty": "模糊读取惩罚",
    "Read gap open penalty": "读取间隙开放惩罚",
    "Read gap extend penalty": "读取间隙延伸惩罚",
    "Reference gap open penalty": "参考间隙开放惩罚",
    "Reference gap extend penalty": "参考间隙延伸惩罚",
    "Function governing the minimum alignment score needed for an alignment to be considered \"valid\" (i.e. good enough to report)": "控制最小比对得分的函数，以确定一个比对是否被认为是“有效”的（即足够好以报告）",
    "Spliced alignment options": "拼接比对选项",
    "Penalty for canonical splice sites": "规范剪接位点的惩罚",
    "Penalty for non-canonical splice sites": "非典型剪接位点的惩罚",
    "Penalty function for long introns with canonical splice sites": "具有规范剪接位点的长内含子的惩罚函数",
    "Penalty function for long introns with non-canonical splice sites": "长内含子的非经典剪接位点的惩罚函数",
    "Minimum intron length": "最小内含子长度",
    "Maximum intron length": "最大内含子长度",
    "Disable spliced alignment": "禁用剪接对齐",
    "Minimum fragment length for valid paired-end alignments": "有效配对端对齐的最小片段长度",
    "Maximum fragment length for valid paired-end alignments": "有效配对末端对齐的最大片段长度",
    "GTF file with known splice sites": "已知剪接位点的GTF文件",
    "Transcriptome assembly reporting": "转录组组装报告",
    "Disable automatic template length adjustment for RNA-seq reads": "禁用RNA-seq读取的自动模板长度调整",
    "reports a list of novel splice sites": "报告一系列新的剪接位点",
    "Reporting options": "报告选项",
    "Primary alignments": "主要比对",
    "Output options": "输出选项",
    "Write unaligned reads (in fastq format) to separate file(s)": "将未比对的读取（以fastq格式）写入单独的文件中",
    "Write aligned reads (in fastq format) to separate file(s)": "将对齐的读取（以fastq格式）写入单独的文件中",
    "SAM options": "SAM 选项",
    "Suppress SAM records for reads that failed to align.": "抑制未能比对的读取的SAM记录。",
    "Edit Read Group IDs": "编辑读取组ID",
    "Read Group ID": "读取组ID",
    "Read Groups text": "读取组文本",
    "Read Group text": "读取组文本",
    "Add/remove 'chr' from reference names in alignment files": "在比对文件中添加/删除参考序列名称中的'chr'",
    "When printing secondary alignments, HISAT2 by default will write out the SEQ and QUAL strings. Specifying this option causes HISAT2 to print an asterisk in those fields instead.": "当打印次要比对时，HISAT2 默认会输出 SEQ 和 QUAL 字符串。指定这个选项会导致 HISAT2 在这些字段中打印一个星号 (*) 代替。",
    "Other options": "其他选项",
    "Use this number as the seed for pseudo-random number generator": "使用这个数字作为伪随机数生成器的种子",
    "Re-initialize the pseudo-random generator for each read using the current time": "为每次读取重新初始化伪随机数生成器，使用当前时间。",
    "Alignment file (maf, xmfa, stockholm)": "比对文件（maf、xmfa、stockholm）",
    "Bin Size": "二进制大小",
    "Links file": "链接文件",
    "Link End Behaviour": "链接端行为",
    "Output Style": "输出样式",
    "Use number of links rather than sum of ends": "使用链接数而不是端点总和",
    "Calculate log10 of the values.": "计算值的以10为底的对数。",
    "Normalize": "标准化",
    "Max Gap": "最大间隙",
    "Minimum Bundle Membership": "最小捆绑成员数",
    "Minimum Bundle Extent": "最小束范围",
    "Minimum Bundle Size": "最小包大小",
    "Minimum Bundle Identity": "最小包体身份",
    "Karyotype": "核型",
    "Reference Genome Source": "参考基因组来源",
    "Choose your Circos karyotype": "选择你的 Circos 核型图",
    "Source Genome Build": "源基因组构建",
    "Source FASTA Sequence": "源FASTA序列",
    "Karyotype Configuration": "核型配置",
    "Sequence Lengths": "序列长度",
    "Ideogram": "染色体图",
    "Limit/Filter and Order Chromosomes": "限制/过滤并排序染色体",
    "Reverse these Chromosomes": "反转这些染色体",
    "Chromosome units": "染色体单元",
    "Spacing Between Ideograms (in chromosome units)": "染色体单位间距",
    "Spacing between specific ideograms": "特定染色体图之间的间距",
    "Chromosome": "染色体",
    "Spacing between these two ideograms (in chromosome units)": "这两个图示之间的间距（以染色体单位计）",
    "Radius": "半径",
    "Thickness": "厚度",
    "Angle Offset": "角度偏移",
    "Angle orientation": "角度方向",
    "Labels": "标签",
    "Show Label": "显示标签",
    "Label Font Size": "标签字体大小",
    "Parallel": "并行",
    "Cytogenic Bands": "细胞遗传带",
    "Show Cytogenic Bands?": "显示染色体带？",
    "Fill Bands": "填充带",
    "Band Transparency": "条带透明度",
    "Band Stroke Thickness": "带状笔触厚度",
    "Band Stroke Color": "条带笔触颜色",
    "General": "通用",
    "Plot Background": "绘制背景",
    "Background Color": "背景颜色",
    "Image Radius": "图像半径",
    "Load additional colour profiles": "加载额外的颜色配置文件",
    "2D Data Tracks": "二维数据轨迹",
    "2D Data Plot": "二维数据图",
    "Outside Radius": "外部半径",
    "Inside Radius": "内部半径",
    "Plot Type": "绘图类型",
    "Histogram Data Source": "直方图数据源",
    "Plot Format Specific Options": "绘图格式特定选项",
    "Fill Color": "填充颜色",
    "Fill underneath the histogram": "填充直方图下方",
    "Join non-abutting Bins": "合并非相邻的箱子",
    "Heatmap Data Source": "热图数据源",
    "Color Scale": "颜色尺度",
    "Advanced color entry": "高级颜色输入",
    "Color Mapping": "颜色映射",
    "Scale Log Base": "比例对数基",
    "Highlight Data Source": "高亮数据源",
    "Line Data Source": "线数据源",
    "Scatter Plot Data Source": "散点图数据源",
    "Glyph": "图形",
    "Glyph Size": "图形大小",
    "Tile Data Source": "平铺数据来源",
    "Layers": "层",
    "Padding": "填充",
    "Margin": "边缘",
    "Overflow Behavior": "溢出行为",
    "Layer Overflow Color": "层溢出颜色",
    "Text Data Source": "文本数据源",
    "Label Size": "标签大小",
    "Show Link": "显示链接",
    "Link thickness": "链接厚度",
    "Link Color": "链接颜色",
    "Link dimensions": "链接维度",
    "Snuggle Labels": "Snuggle Labels",
    "Max snuggle Distance": "最大依偎距离",
    "Snuggle Sampling": "Snuggle Sampling",
    "Snuggle Tolerance": "紧密容忍度",
    "Snuggle Link Overlap Test": "Snuggle 链接重叠测试",
    "Snuggle Link Overlap Tolerance": "Snuggle 链接重叠容差",
    "Snuggle refine": "Snuggle refine",
    "Connector Data Source": "连接器数据源",
    "Connector dimensions": "连接器尺寸",
    "Orient Inwards": "朝向内侧",
    "Link Tracks": "链接轨迹",
    "Link Data": "链接数据",
    "Link Data Source": "链接数据源",
    "Link Type": "链接类型",
    "Twist Ribbon": "扭曲丝带",
    "Ribbon background pattern": "彩带背景图案",
    "Bezier Radius": "贝塞尔半径",
    "Advanced Settings": "高级设置",
    "Crest": "Crest",
    "Bezier Radius Purity": "贝塞尔半径纯度",
    "Perturb links?": "扰乱链接？",
    "Crest min": "Crest 最小值",
    "Crest max": "Crest 最大",
    "Bezier Radius min": "贝塞尔半径最小值",
    "Bezier Radius max": "贝塞尔半径最大值",
    "Bezier Radius Purity min": "贝塞尔半径纯度最小",
    "Bezier Radius Purity max": "贝塞尔半径纯度最大",
    "Scaling": "缩放",
    "Zoomed Region": "缩放区域",
    "End": "结束",
    "Scale": "比例",
    "Ticks": "蜱虫",
    "Skip first label": "跳过第一个标签",
    "Skip last label": "跳过最后一个标签",
    "Tick Group": "蜱虫组",
    "Tick Spacing": "刻度间距",
    "Tick Size": "刻度大小",
    "Show Tick Labels": "显示刻度标签",
    "Label Offset": "标签偏移",
    "Label Format": "标签格式",
    "Show unit in tick labels": "显示刻度标签中的单位",
    "Outputs": "输出",
    "PNG": "PNG",
    "SVG": "SVG（Scalable Vector Graphics）是一种使用XML来描述二维图形的语言，在生物信息学中常用于图形化展示数据。",
    "Configuration Archive": "配置归档",
    "Source for reference genome": "参考基因组来源",
    "Window size": "窗口大小",
    "1D/2D Data track": "一维/二维数据轨道",
    "Bin size": "箱大小",
    "Summarization method": "摘要方法",
    "Data file": "数据文件",
    "BigWig files": "大波形文件",
    "Input table": "输入表格",
    "Output PNG": "输出PNG",
    "Output SVG": "输出 SVG",
    "Output Configuration Archive": "输出配置归档",
    "Ticks & Labels": "刻度与标签",
    "Segment Labels": "段标签",
    "Show label": "显示标签",
    "Show parallel to rack": "显示与机架平行",
    "Font Size": "字体大小",
    "Color": "颜色",
    "Tick Labels": "刻度标签",
    "Data Format": "数据格式",
    "BED File (BED6+ only)": "BED 文件（仅限 BED6+）",
    "GFF3 File": "GFF3 文件",
    "GFF3 Attribute to pull value from": "GFF3 属性值提取",
    "Select input data file :": "选择输入数据文件：",
    "Select predicted data file :": "选择预测数据文件：",
    "Select trained model :": "选择训练好的模型：",
    "Track": "轨道",
    "Visibility": "可见性",
    "Numerical column for x axis": "数值列作为x轴",
    "Numerical column for y axis": "数值列作为y轴",
    "Plot title": "图标题",
    "Label for x axis": "X轴标签",
    "Label for y axis": "y轴标签",
    "Number of breaks (bars)": "断裂数（条）",
    "Include smoothed density": "包含平滑密度",
    "Plot as frequency (counts)": "绘制为频率（计数）",
    "Alignment File": "比对文件",
    "Reference Sequence": "参考序列",
    "Annotations": "注释",
    "Annotation Style": "注释风格",
    "Species": "物种",
    "Exons File": "外显子文件",
    "Highlights File": "高亮文件",
    "Underlays File": "底层文件",
    "Repeats File": "重复序列文件",
    "Links File": "链接文件",
    "Full Sequence Name": "完整序列名称",
    "Offset": "偏移量",
    "Choose Warnings to Suppress": "选择要抑制的警告",
    "Plot Title": "绘图标题",
    "Output File Type": "输出文件类型",
    "Series": "序列",
    "Header in first line?": "首行包含标题？",
    "Column for x axis": "X轴列",
    "Column for y axis": "Y轴列",
    "Series Type": "系列类型",
    "Line Type": "行类型",
    "Line Color": "线条颜色",
    "Line Width": "线宽",
    "Point Type": "点类型",
    "Point Color": "点颜色",
    "Point Scale": "点刻度",
    "Will you select a reference genome from your history or use a built-in genome?": "您将从历史记录中选择一个参考基因组，还是使用内置的基因组？",
    "Reference genome": "参考基因组",
    "BAM alignment": "BAM比对",
    "Image dimensions": "图像尺寸",
    "Output format": "输出格式",
    "Advanced options": "高级选项",
    "Count overlapping bases with mate for paired-end": "计算配对末端的重叠碱基",
    "Only keep chromosomes matching this regular expression.": "只保留与这个正则表达式匹配的染色体。",
    "Minimum mapping quality": "最小映射质量",
    "y-axis (depth) limit of the plot": "y轴（深度）的绘图限制",
    "Clip the plot at the y-axis limit": "将图表在 y 轴限制处剪裁",
    "Skip chromosomes shorter than": "跳过长度小于",
    "How to bin the coverage under one pixel": "如何将覆盖度整合到一个像素以下",
    "Plot coverage using points instead of areas": "使用点而不是区域绘制覆盖度",
    "Skip chromosomes matching this regular expression": "跳过与此正则表达式匹配的染色体",
    "Filter by sample": "按样本过滤",
    "Limit to these read groups": "限制这些读组",
    "How to interpret identifiers in --samples": "如何解释 --samples 中的标识符",
    "JBrowse Data Directory": "JBrowse 数据目录",
    "Reference genome to display": "参考基因组以显示",
    "Output JBrowse": "输出 JBrowse",
    "Genetic Code": "遗传密码",
    "JBrowse-in-Galaxy Action": "JBrowse-in-Galaxy 操作",
    "Previous JBrowse Instance": "之前的JBrowse实例",
    "Track Group": "轨道组",
    "Track Category": "轨道类别",
    "Annotation Track": "注释轨迹",
    "Track Options": "轨道选项",
    "Track Type": "轨道类型",
    "BlastXML Track Data": "BlastXML 轨道数据",
    "Features used in Blast Search": "在Blast搜索中使用的特征",
    "Minimum Gap Size": "最小间隔大小",
    "Is this a protein blast search?": "这是一个蛋白质BLAST搜索吗？",
    "Index this track": "索引这个轨迹",
    "description": "描述",
    "SNP Track Data": "SNP 轨道数据",
    "GFF/GFF3/BED Track Data": "GFF/GFF3/BED 轨道数据",
    "match/match_part data": "匹配/匹配部分 数据",
    "This is match/match_part data": "这是匹配/部分匹配数据",
    "Match Part Feature Type": "匹配部分特征类型",
    "JBrowse Track Type [Advanced]": "JBrowse 轨道类型 [高级]",
    "product,name,id": "产品，名称，编号",
    "BAM Track Data": "BAM 轨道数据",
    "Autogenerate SNP Track": "自动生成SNP轨迹",
    "Maximum size of BAM chunks": "BAM块的最大大小",
    "BigWig Track Data": "大波形轨道数据",
    "Use XYPlot": "使用 XYPlot",
    "Show variance band": "显示方差带",
    "Track Scaling": "轨道缩放",
    "Track minimum": "轨道最小值",
    "Track maximum": "轨道最大值",
    "Visual Scaling": "视觉缩放",
    "Use MultiBigWig Plugin": "使用MultiBigWig插件",
    "REST Endpoint": "REST端点",
    "Track Label": "轨道标签",
    "SPARQL Server URL": "SPARQL服务器URL",
    "SPARQL Query": "SPARQL查询",
    "Plugins": "插件",
    "BlastView": "BlastView",
    "Combo Track Selector": "组合轨道选择器",
    "GC Content": "GC含量",
    "Region of the genome to plot": "基因组绘图区域",
    "Configure figure parameters": "配置图形参数",
    "Font size for the labels of the plot": "图表标签的字体大小",
    "Resolution (dpi)": "分辨率（每英寸点数）",
    "Figure width in cm": "图宽（以厘米为单位）",
    "Width in centimeters of the plotting (central) part": "绘图（中心）部分的宽度（以厘米为单位）",
    "Figure height in centimeters": "图高（以厘米为单位）",
    "Proportion of space dedicated to track labels": "空间分配给轨迹标签的比例",
    "Horizontal alignment of track labels": "轨道标签的水平对齐",
    "Use a decreasing x axis": "使用一个递减的x轴",
    "Include tracks in your plot": "在您的图表中包含轨迹",
    "Choose style of the track": "选择轨道的风格",
    "Color map to use for the heatmap": "用于热图的颜色映射",
    "Depth": "深度",
    "Plot the transformed value": "绘制转换后的值",
    "height of the heatmap": "热图的高度",
    "Show masked bins": "显示掩蔽的二进制文件",
    "Boundaries file": "边界文件",
    "Scale factor": "缩放因子",
    "Rasterize the heatmap": "将热图栅格化",
    "Define color:": "定义颜色：",
    "Color of track": "轨道颜色",
    "Color map to use for the scores": "用于分数的颜色图",
    "width of lines": "线宽",
    "height": "高度",
    "Plot labels": "绘制标签",
    "Put all labels inside the plotted region": "将所有标签放置在绘制区域内",
    "Allow to put labels in the right margin": "允许将标签放在右边距",
    "Fontsize": "字体大小",
    "Fontstyle": "字体样式",
    "display to use": "要使用的显示器",
    "Bed style": "Bed风格",
    "Include the arrowhead in the bed interval:": "在bed区间中包含箭头头：",
    "Size of arrowhead relative to the plotted region": "箭头大小相对于绘制区域的比例",
    "Configure other bed parameters": "配置其他bed参数",
    "interval between 2 arrows on introns in UCSC style": "在UCSC风格中，内含子上两个箭头之间的间隔",
    "arrow length": "箭头长度",
    "Representation of the link:": "链接的表示：",
    "ylim": "y轴限制",
    "Use summit information to make the curve.": "使用峰值信息来绘制曲线。",
    "Show visualization of data range": "显示数据范围的可视化",
    "Factor to change the width of the curve.": "改变曲线宽度的因子。",
    "Plot a vertical bar at the summit position.": "在峰顶位置绘制一个垂直条形。",
    "Plot labels (name, p-val, q-val)": "绘制标签（名称，p值，q值）",
    "Configure bigwig parameters": "配置大波形图参数",
    "Summary method": "摘要方法",
    "Number of bins": "箱数",
    "Compute an operation on the file": "在文件上计算一个操作",
    "operation to compute on the file(s)": "对文件进行计算的操作",
    "second_file (bedgraph format)": "第二个文件（bedgraph格式）",
    "NAN's to zeros": "将NAN转换为零",
    "Use middle: Instead of using start and end, use the middle of intervals (useful for 4C-seq tracks).": "使用中间值：而不是使用起始和结束，使用区间的中间值（对于4C-seq轨迹很有用）。",
    "Display a grid on the y-axis": "在 y 轴上显示一个网格",
    "type of plotting": "绘图类型",
    "Plot an horizontal line at 0 when type is lines": "当类型为线条时，在0处绘制一条水平线。",
    "Position of the score with respect to bin start and end": "得分相对于区间起始和结束的位置",
    "Color of the individual lines": "个体线条的颜色",
    "Color of the summary line": "摘要行的颜色",
    "colormap": "颜色映射",
    "Line Style": "线条样式",
    "color of the vline": "vline颜色",
    "transparency (alpha) of the vline": "vline 的透明度（alpha）",
    "Plot the vline above or behind other tracks": "在其他轨道上方或后方绘制垂直线",
    "color of the highlight": "高亮颜色",
    "transparency (alpha) of the highlight": "高亮部分的透明度（alpha）",
    "width of the edge of the highlight": "高亮边缘的宽度",
    "color of the edge of the highlight": "高亮边缘的颜色",
    "Plot the highlight above or behind other tracks": "在其他轨道上方或后方绘制高亮显示",
    "Values where horizontal lines should be plotted separated by comma": "在逗号分隔的地方绘制水平线",
    "Color of line(s)": "线条的颜色",
    "height of the Hlines track": "Hlines轨道的高度",
    "Height of the spacer.": "间隔的高度。",
    "Fontsize of the x-axis.": "x轴的字体大小。",
    "chromosome name and coordinates position relative to the axis": "染色体名称和相对于轴的坐标位置",
    "Color of the scale bar": "比例尺的颜色",
    "level of transparency (alpha)": "透明度级别（alpha）",
    "Set the scalebar position": "设置比例尺位置",
    "coordinate where the scale bar should begin.": "刻度条应该开始的坐标。",
    "coordinate where the scale bar should end.": "标尺应该结束的坐标。",
    "coordinate where the scale bar should be plotted (center)": "刻度条应该绘制的坐标（中心）",
    "size of the scale bar": "比例尺的大小",
    "where the size of the scale bar should appear relative to the bar": "其中比例尺大小应该相对于条形图的位置出现",
    "Fontsize of the size of the scale bar.": "刻度条大小的字体大小。",
    "height of the Scale bar track": "比例尺轨迹的高度",
    "height of the Fasta track": "Fasta 轨道的高度",
    "Reference": "参考序列",
    "Color when bases are identical": "当碱基相同时的颜色",
    "Color when bases are different": "当碱基不同时的颜色",
    "Color when bases corresponds to gap": "当碱基对应到间隔时的颜色",
    "[Optional] The order you want your species (space separated)": "[可选] 您希望物种的顺序（用空格分隔）",
    "[Optional] The label you want for the above species (space separated)": "[可选] 您想要为上述物种设置的标签（以空格分隔）",
    "Display only the species specified above": "仅显示上述指定的物种",
    "Display the sequence of the reference": "显示参考序列",
    "Image output format": "图像输出格式",
    "Newick file": "新克文件",
    "Display scale bar": "显示比例尺",
    "Unit name for the scale bar": "比例尺的单位名称",
    "Leaf font": "叶形字体",
    "Leaf font-size": "叶字体大小",
    "Branch support": "分支支持",
    "color": "颜色",
    "font-size": "字体大小",
    "move": "移动",
    "Branch length": "分支长度",
    "Image width": "图像宽度",
    "Draw a radial tree": "绘制一个径向树",
    "Choose an output format": "选择一个输出格式",
    "Graphical Fragment Assembly": "图形化片段组装",
    "Image height": "图像高度",
    "Node name labels?": "节点名称标签？",
    "Node length labels?": "节点长度标签？",
    "Font size": "字体大小",
    "Node width": "节点宽度",
    "Produce jpg, png or svg file?": "生成 jpg、png 或 svg 文件？",
    "Output the information in a single tab-delimited line starting with the graph file": "输出信息以单行制表符分隔的文本开始，以图形文件为首",
    "Select input files for which to produce intersections": "选择要生成交集的输入文件",
    "Use this metric for the heatmap": "使用此指标进行热图分析",
    "Reference genome ID": "参考基因组ID",
    "Select the type of correlation": "选择相关性类型",
    "Heatmap plot type": "热图图类型",
    "Show the diagonal values in the heatmap ?": "在热图中显示对角线值？",
    "Boxplot color": "箱线图颜色",
    "Label font size": "标签字体大小",
    "Heatmap main title": "热图主标题",
    "Output the intervene matrix file?": "输出干预矩阵文件？",
    "Select the order of intersections of sets": "选择集合交集的顺序",
    "Number of top intersections to show in plot": "在图中显示的顶级交叉点数量",
    "Show empty intersections in plot?": "在图表中显示空交集？",
    "Show intersection sizes above bars ?": "显示条形图上方的交集大小？",
    "Color of the main bar plot": "主条形图的颜色",
    "Color of the set size bar plot": "集合大小条形图的颜色",
    "The y-axis label of the intersection size bars": "交集大小条形图的y轴标签",
    "The x-axis label of the set size bars": "集合大小条形图的x轴标签",
    "Number of pathways to plot": "要绘制的通路数量",
    "KEGG pathway": "KEGG通路",
    "KEGG pathways": "KEGG 通路",
    "Does the file have header (a first line with column names)?": "文件是否有标题行（第一行包含列名）？",
    "Species to use": "使用物种",
    "Provide a gene data file?": "提供一份基因数据文件？",
    "Gene data": "基因数据",
    "Does the file have header (a first line with sample names)?": "文件是否有标题（第一行包含样本名称）？",
    "Format for gene data": "基因数据格式",
    "Provide a compound data file?": "提供一个复合数据文件？",
    "Compound data": "化合物数据",
    "Format for compound data": "化合物数据格式",
    "Should multiple states of gene or compond data be intergrated and plotted in the same graph?": "应该将基因或化合物数据的多个状态集成并绘制在同一张图中吗？",
    "Are the gene and compound data paired?": "基因和化合物数据是否配对？",
    "Output Options": "输出选项",
    "Output for pathway": "通路输出",
    "Plot on same layer?": "在同一层上绘制？",
    "Map the NULL gene or compound data to pathway?": "将NULL基因或复合数据映射到通路上？",
    "Split node groups into individual nodes?": "将节点组拆分为单独的节点？",
    "Expand multiple-gene nodes into single-gene nodes?": "将多基因节点扩展为单基因节点？",
    "Position of pathview signature": "pathview 特征的定位",
    "Input file": "输入文件",
    "Use a LEfSe output file as input?": "使用LEfSe输出文件作为输入？",
    "Input file corresponding to LEfSe output file": "输入文件对应于LEfSe输出文件",
    "List which levels should be annotated in the tree": "列出树中应注释的级别",
    "List which levels should use the external legend for the annotation": "列出应该使用外部图例进行注释的级别",
    "List which levels should be highlight with a shaded background": "列出应该用阴影背景突出显示的级别",
    "List of the clades that should be highlight with a shaded background": "应该用阴影背景突出显示的类群列表",
    "List of color to use for the shaded background": "用于阴影背景的颜色列表",
    "Title of the GraPhlAn plot": "GraPhlAn 绘图的标题",
    "Title font size": "标题字体大小",
    "Default size for clades that are not found as biomarkers": "未作为生物标志物发现的类群的默认大小",
    "Minimum value of clades that are biomarkers": "作为生物标记的最小类群值",
    "Maximum value of clades that are biomarkers": "作为生物标记的最大类群值",
    "Default font size": "默认字体大小",
    "Minimum font size": "最小字体大小",
    "Maximum font size": "最大字体大小",
    "Font size for the annotation legend": "注释图例的字体大小",
    "Minimun abundance value for a clade to be annotated": "一个类群被注释的最小丰度值",
    "Number of clades to highlight": "要突出显示的类群数量",
    "Minimum number of biomarkers to extract": "最少提取的生物标记物数量",
    "Row number containing the names of the features": "包含特征名称的行号",
    "Row number containing the names of the samples": "包含样本名称的行号",
    "Row number to use as metadata": "作为元数据使用的行号",
    "Row number to skip from the input file": "输入文件中要跳过的行数",
    "Percentile of sample value distribution for sample selection": "样本值分布的百分位数用于样本选择",
    "Percentile of feature value distribution for sample selection": "样本选择的特征值分布百分位数",
    "Number of top samples to select": "选择的顶级样本数量",
    "Number of top features to select": "选择的顶级特征数量",
    "Implicit or explicit full ID list?": "隐式或显式完整ID列表？",
    "Full dataset (with all identifiers)": "完整数据集（包含所有标识符）",
    "Sets": "设置",
    "Members of set": "集合成员",
    "Caption for set": "集合的标题",
    "Select table data file": "选择表格数据文件",
    "The mode of column selection": "列选择模式",
    "Select the columns for dimentions:": "选择用于维度的列：",
    "Type the column names": "输入列名",
    "Cast all the dimensions into categorical datatype?": "将所有维度转换为分类数据类型？",
    "Select a column containg the values for coloring:": "选择一个包含用于上色值的列：",
    "FASTA file": "FASTA 文件",
    "Pixel height per nucleotide": "每个核苷酸的像素高度",
    "Pixel width per nucleotide": "每个核苷酸的像素宽度",
    "A nucleotide: specify RGB value or select color?": "一个核苷酸：指定RGB值还是选择颜色？",
    "RGB": "RGB",
    "T nucleotide: specify RGB value or select color?": "T核苷酸：指定RGB值还是选择颜色？",
    "G nucleotide: specify RGB value or select color?": "G核苷酸：指定RGB值还是选择颜色？",
    "C nucleotide: specify RGB value or select color?": "C核苷酸：指定RGB值还是选择颜色？",
    "N nucleotide: specify RGB value or select color?": "N 核苷酸：指定 RGB 值还是选择颜色？",
    "Input tree": "输入树",
    "Dpi of the output image": "输出图像的分辨率",
    "Size of the output image": "输出图像的大小",
    "Distance between the most external graphical element and the border of the image": "图像最外层图形元素与边框之间的距离",
    "Annotation file": "注释文件",
    "Tabular input dataset": "表格输入数据集",
    "Seed value for reproducibiity": "用于可重复性的种子值",
    "perplexity": "困惑度",
    "theta": "θ",
    "The column with the ID to use in grouping (or just an ID if plotting invdividuals)": "用于分组的列的ID（或者仅仅是一个ID，如果是绘制个体的话）",
    "The column that contains the first numerical data": "包含第一个数值数据的列",
    "Advanced - log transformation": "高级 - 对数转换",
    "Legend options": "图例选项",
    "Select input file": "选择输入文件",
    "is this output from MOTHUR?": "这是来自MOTHUR的输出吗？",
    "Species A": "物种 A",
    "Species B": "物种 B",
    "MUMmer search algorithm": "MUMmer 搜索算法",
    "Backbone Line Type": "骨架线类型",
    "Backbone Line Color": "骨架线颜色",
    "Backbone Line Width": "骨架线宽度",
    "Include legend?": "包括图例？",
    "Legend title": "图例标题",
    "Legend position": "图例位置",
    "Chromosome Column": "染色体列",
    "Start Column": "起始列",
    "End Column": "结束列",
    "Legend text": "图例文本",
    "BAM file": "BAM文件",
    "Lower bp limit (optional)": "较低碱基对限制（可选）",
    "Upper bp limit (optional)": "上限碱基对数（可选）",
    "Input Pretext map file": "输入预文本映射文件",
    "Output image format": "输出图像格式",
    "JPEG quality factor": "JPEG 质量因子",
    "Output image resolution": "输出图像分辨率",
    "Color Map": "颜色图",
    "Sequence specification string": "序列规范字符串",
    "Min Texels": "最小纹理元素",
    "Show grid?": "显示网格？",
    "Grid size": "网格尺寸",
    "Grid color": "网格颜色",
    "Print Sequence names": "打印序列名称",
    "expression data": "表达数据",
    "Input column separator": "输入列分隔符",
    "Gene sets Pathways": "基因集 通路",
    "Do you have non cancer transcriptomes in your data set ?": "您有非癌症转录组数据集吗？",
    "Sample status": "样本状态",
    "Throw away components leading to low stability of sampling noise": "丢弃导致采样噪声稳定性低的组分",
    "Number of runs to determine stability": "确定稳定性的运行次数",
    "Minimum of standard deviation to filter out low variable genes": "最小标准差，用于筛选出变异性低的基因",
    "Minimum of gene expression to filter out low variable genes": "基因表达的最小值，用于筛选出变异性低的基因",
    "Cluster samples on heatmap": "在热图上聚类样本",
    "Cluster pathways on heatmap": "在热图上聚类途径",
    "Show sample labels on heatmap": "在热图上显示样本标签",
    "Cluster pathway labels on heatmap": "在热图上聚类途径标签",
    "Return log file of pathifier": "返回Pathifier的日志文件",
    "Return Pathifier S4 object": "返回 Pathifier S4 对象",
    "Proband VCF file": "先证者 VCF 文件",
    "Proband BAM file": "先证者BAM文件",
    "Proband sex": "先证性別",
    "Is the proband affected?": "先证者受影响吗？",
    "Proband sample name": "先证者样本名称",
    "Single/Trio analysis": "单样本/三人组分析",
    "Father input": "父本输入",
    "Father VCF file": "父本 VCF 文件",
    "Father BAM file": "父BAM文件",
    "Is the father affected?": "父亲受影响了吗？",
    "Father sample name": "父亲样本名称",
    "Mother input": "母体输入",
    "Mother VCF file": "母体 VCF 文件",
    "Mother BAM file": "母 BAM 文件",
    "Is the mother affected?": "母亲受影响了吗？",
    "Mother sample name": "母本样本名称",
    "Genes of interest": "感兴趣的基因",
    "Select reference genome version": "选择参考基因组版本",
    "Annotated Genome File (Gbk)": "注释基因组文件（GBK）",
    "Plot Width": "绘图宽度",
    "Box Label": "盒子标签",
    "Select Custom Region": "选择自定义区域",
    "Start Zoom": "开始 Zoom",
    "End Zoom": "结束缩放",
    "Multi Line Plot": "多行图",
    "Nucleotides Per Line": "每行核苷酸数",
    "Common Feature(s) to EXCLUDE": "排除的常见特征",
    "Extra Feature(s) to EXCLUDE": "额外功能（要排除的）",
    "Common Feature(s) label(s) to EXCLUDE": "排除的常见特征（标签）",
    "Extra Feature(s) label(s) to EXCLUDE": "排除的额外特征标签",
    "Common Product names to EXCLUDE from labeling": "排除标签中的常见产品名称",
    "Extra Product(s) name label(s) to EXCLUDE": "排除额外产物（名）标签",
    "Name ordering": "名称排序",
    "Title of Graph": "图表标题",
    "Label algorithm dictates label placement": "标签算法决定标签放置",
    "Feature Colors": "特征颜色",
    "Feature Color Options": "特征颜色选项",
    "Feature Name": "特征名称",
    "Feature Color": "特征颜色",
    "Product Name Colors": "产品名称颜色",
    "Product Color Options": "产品颜色选项",
    "Product Name": "产品名称",
    "Product Color": "产品颜色",
    "Enable data clustering": "启用数据聚类",
    "Heatmap colorscheme": "热图颜色方案",
    "Input in tabular format": "输入为表格格式",
    "Indicate quartiles of your data": "指示你的数据的四分位数",
    "Violin color options": "小提琴颜色选项",
    "Violin border options": "小提琴边框选项",
    "Axis scaling": "轴刻度缩放",
    "minimal range of y-axis": "y轴的最小范围",
    "maximal range of y-axis": "y轴的最大范围",
    "Column to plot on x-axis": "x轴上的列",
    "Column to plot on y-axis": "在 y 轴上绘制的列",
    "Type of plot": "图表类型",
    "Plotting multiple groups": "绘制多个组",
    "column differentiating the different groups": "区分不同组的列",
    "Reverse color scheme": "反转颜色方案",
    "Axis title options": "轴标题选项",
    "Axis text options": "轴文字选项",
    "Plot title options": "绘图标题选项",
    "Grid lines": "网格线",
    "Input should have column headers - these will be the columns that are plotted": "输入应包含列标题 - 这些将是绘制的列",
    "relative line width": "相对线宽",
    "Bin width for plotting": "绘图的箱宽",
    "Plot counts or density": "绘制计数或密度",
    "Faceting": "分面",
    "Coloring groups": "着色群组",
    "Select table": "选择表格",
    "Select input dataset options": "选择输入数据集选项",
    "Sample names orientation": "样本名称方向",
    "Select column, for row names": "选择列，作为行名",
    "Select groups": "选择组",
    "Group": "组",
    "Group name": "组名",
    "Columns for group": "组的列",
    "Change colors": "更改颜色",
    "Pick color": "选择颜色",
    "Select type of ellipses": "选择椭圆类型",
    "Select plot layout": "选择绘图布局",
    "Input should have column headers - these will be the columns that are plotted and row names.": "输入应包含列标题 - 这些将是绘制的列和行名。",
    "key title": "关键标题",
    "Data transformation": "数据转换",
    "Compute z-scores prior to clustering": "在聚类之前计算z分数",
    "Scale data on the plot (after clustering)": "在绘图（聚类后）上对数据进行缩放",
    "Clustering columns and rows": "聚类列和行",
    "Distance method": "距离方法",
    "Clustering method": "聚类方法",
    "Labeling columns and rows": "标记列和行",
    "Type of colormap to use": "使用的颜色映射类型",
    "Name of the colormap": "颜色图名称",
    "First color": "第一个颜色",
    "Second color": "第二种颜色",
    "Third color": "第三种颜色",
    "Short read data from your current history": "当前历史中的短读数据",
    "Contaminant list": "污染列表",
    "Submodule and Limit specifing file": "子模块和限制指定文件",
    "BED file": "BED文件",
    "Minimum size of N tract": "N串联的最小尺寸",
    "Penalty for a non-N": "非N的惩罚",
    "Print gaps only, no sequence": "仅打印间隔，不打印序列",
    "Quality values": "质量值",
    "# start positions in a window": "# 窗口中的起始位置",
    "Treat lowercases as masked": "将小写字母视为掩蔽",
    "Input FASTA/Q file #1": "输入FASTA/Q文件 #1",
    "Input FASTA/Q file #2": "输入FASTA/Q文件 #2",
    "Quality threshold": "质量阈值",
    "Take intersection": "取交集",
    "Convert to lowercase when one of the input base is N": "当输入碱基之一为N时，转换为小写。",
    "Pick a random allele from het": "从杂合子中随机选择一个等位基因",
    "Suppress hets in the input": "在输入中抑制hets",
    "Input SNP file": "输入SNP文件",
    "RNG seed": "随机数生成器种子",
    "Subsample (decimal fraction or number)": "子采样（小数分数或数字）",
    "Enable 1-pass mode": "启用单遍模式",
    "Mask bases with quality lower than INT": "遮蔽质量低于整数(INT)的碱基",
    "Mask bases with quality higher than INT": "遮蔽质量高于INT的碱基",
    "Masked bases converted to CHAR; 0 for lowercase": "掩码碱基转换为CHAR；小写字母为0",
    "Number of residues per line; 0 for 2^32-1": "每行残基数；0表示2^32-1",
    "Quality shift: ASCII-INT gives base quality": "质量偏移：ASCII-INT 提供碱基质量",
    "Random seed": "随机种子",
    "Sample fraction of sequences": "序列的样本分数",
    "Mask regions in BED or name list file": "在BED文件或名称列表文件中屏蔽区域",
    "Drop sequences with length shorter than INT": "删除长度小于整数INT的序列",
    "Mask complement region": "掩蔽互补区域",
    "Reverse complement": "反向互补",
    "Force FASTA output (discard quality)": "强制输出FASTA格式（忽略质量）",
    "Drop comments at the header lines": "删除头部行的注释",
    "Drop sequences containing ambiguous bases": "删除含有模糊碱基的序列",
    "Output the 2n-1 reads only": "输出2n-1个读取结果",
    "Output the 2n reads only": "输出只有2n的读取",
    "Select source of sequence choices": "选择序列选择的来源",
    "Input BED file": "输入 BED 文件",
    "Input ID list": "输入ID列表",
    "TAB delimited output": "制表符分隔的输出",
    "Sequence line length": "序列行长度",
    "Telomere repeat to search for. Default is vertebrate": "要搜索的端粒重复序列。默认是脊椎动物。",
    "Penalty for a non-repeat": "非重复的惩罚",
    "Maximum drop": "最大下降",
    "Minimum score": "最小分数",
    "Print scoring": "打印得分",
    "Mode for trimming FASTQ File": "FASTQ文件修剪模式",
    "Error rate threshold": "错误率阈值",
    "Maximally trim down to INT bp": "最大修剪至INT个碱基对",
    "Trim INT bp from left": "从左侧修剪INT个碱基对",
    "Trim INT bp from right": "从右侧修剪 INT 个碱基对",
    "Sequence file to be filtered": "待过滤的序列文件",
    "SAM/BAM mapping of those sequences": "SAM/BAM 序列比对",
    "Output mapped reads, unmapped reads, or both?": "输出映射的读段、未映射的读段，还是两者都输出？",
    "Paired read treatment": "配对读取处理",
    "Is this library paired- or single-end?": "这个库是成对末端（双端测序）的还是单末端（单端测序）的？",
    "FASTQ file": "FASTQ 文件",
    "FASTQ collection": "FASTQ 集合",
    "Apply filter treatments?": "应用过滤处理？",
    "Filter sequence based on their length?": "根据序列长度过滤序列？",
    "Filter too small sequences?": "过滤太小的序列？",
    "Minimum length threshold to conserve sequences": "最小长度阈值以保留序列",
    "Filter too big sequences?": "过滤过大的序列？",
    "Maximal length threshold to conserve sequences": "最大长度阈值以保留序列",
    "Filter sequences based on quality score?": "根据质量分数过滤序列？",
    "Filter sequences based on their minimum score?": "根据它们的最小分数过滤序列？",
    "Minimum score threshold to conserve sequences": "最小分数阈值以保留序列",
    "Filter sequences based on their maximum score?": "根据它们的最大得分过滤序列？",
    "Maximum score threshold to conserve sequences": "最大得分阈值以保留序列",
    "Filter sequences based on their mean score?": "根据它们的平均得分过滤序列？",
    "Filter sequences based with too small mean score?": "基于平均分数过小的序列进行过滤？",
    "Minimum mean score threshold to conserve sequences": "最小均分阈值以保留序列",
    "Filter sequences based with too high mean score?": "根据平均得分过高来过滤序列？",
    "Maximum mean score threshold to conserve sequences": "最大均值分数阈值以保留序列",
    "Filter sequences based on their base content?": "根据它们的碱基含量过滤序列？",
    "Filter sequences based on their GC percentage?": "根据它们的GC百分比过滤序列？",
    "Filter sequences with too small GC percentage?": "过滤GC百分比过低的序列？",
    "Minimal GC percentage threshold to conserve sequences": "最小GC百分比阈值以保留序列",
    "Filter sequences with too high GC percentage?": "过滤掉GC百分比过高的序列？",
    "Maximal GC percentage threshold to conserve sequences": "最大GC百分比阈值以保留序列",
    "Filter sequences based on their number of N bases?": "根据N碱基的数量过滤序列？",
    "Maximal N number threshold to conserve sequences": "最大N个数阈值以保留序列",
    "Filter sequences based on their percentage of N bases?": "根据N碱基的百分比过滤序列？",
    "Maximal N percentage threshold to conserve sequences": "最大N百分比阈值以保留序列",
    "Filter sequences with characters other than A, T, C, G and N?": "过滤掉含有除A、T、C、G和N之外字符的序列？",
    "Filter sequences based on their complexity?": "根据序列的复杂性过滤序列？",
    "Method to filter low complexity sequences": "低复杂性序列过滤方法",
    "Threshold value used to filter sequences by sequence complexity": "用于根据序列复杂性过滤序列的阈值",
    "Apply trimming treatments?": "应用修剪处理？",
    "Trim all sequences from the 3'-end to a defined length?": "将所有序列从3'端修剪到一个定义的长度？",
    "Length of sequences after trimming": "修剪后序列的长度",
    "Trim all sequences from the ends?": "是否从序列的两端进行修剪？",
    "Trim sequences by a defined number of positions?": "根据定义的位置数量修剪序列？",
    "Trim sequences at the 5'-end by a defined number of positions?": "在5'端修剪一定数量的位置序列？",
    "Number of positions to trim on 5'-end": "5'端修剪的位置数",
    "Trim sequences at the 3'-end by a defined number of positions?": "在3'端通过定义的位数修剪序列？",
    "Number of positions to trim on 3'-end": "3'端修剪的位置数",
    "Trim sequences by a defined percentage of read length?": "根据读取长度的一定百分比修剪序列？",
    "Trim sequences at the 5'-end by a defined percentage of read length?": "在5'端按定义的读取长度百分比修剪序列？",
    "Percentage of positions to trim on 5'-end": "5'端要修剪的碱基位置百分比",
    "Trim sequences at the 3'-end by a defined percentage of read length?": "在读取长度的一定百分比处修剪3'端序列？",
    "Percentage of positions to trim on 3'-end": "3'端要修剪的碱基位置百分比",
    "Trim tails?": "修剪尾部？",
    "Trim poly-A/T tail?": "修剪多聚腺苷酸/胸腺嘧啶尾巴？",
    "Trim poly-A/T tail at the 5'-end?": "修剪5'端的聚A/T尾吗？",
    "Minimum length of poly-A/T to trim at the 5'-end": "在5'端修剪的最小长度为聚腺苷酸/胸腺嘧啶",
    "Trim poly-A/T tail at the 3'-end?": "修剪3'端的多聚腺苷酸/胸腺嘧啶尾？",
    "Trim poly-N tail?": "修剪多聚N尾端？",
    "Trim poly-N tail at the 5'-end?": "修剪5'端的多聚N尾？",
    "Minimum length of poly-N to trim at the 5'-end": "在5'端修剪的最小长度为聚N",
    "Trim poly-N tail at the 3'-end?": "修剪3'端的多聚N尾吗？",
    "Trim sequence by quality score?": "根据质量分数修剪序列？",
    "Trim sequence by quality score from the 5'-end?": "从5'端根据质量分数修剪序列？",
    "Quality score threshold to trim positions": "修剪位置的质量分数阈值",
    "Trim sequence by quality score from the 3'-end?": "从3'端根据质量分数修剪序列？",
    "Type of quality score calculation to use": "要使用的测序质量评分计算类型",
    "Rule tu use to compare quality score to calculated value": "用于比较质量分数与计算值的规则",
    "Size of the sliding window used to calculated quality score by type": "滑动窗口的大小用于按类型计算质量分数",
    "Step size used to move the sliding window": "滑动窗口移动时使用的步长",
    "Choose the reference type": "选择参考类型",
    "Select one or more fasta file": "选择一个或多个fasta文件",
    "Select one or more keywords": "选择一个或多个关键词",
    "Kmer length used for finding contaminants": "用于查找污染物的Kmer长度",
    "Look for reverse-complements of kmers in addition to forward kmers?": "寻找k-mers的反向互补序列以及正向k-mers？",
    "Treat the middle base of a kmer as a wildcard to increase sensitivity in the presence of errors?": "将kmer中间的碱基视为通配符以提高在存在错误时的敏感性？",
    "Reads need at least this many matching kmers to be considered as matching the reference": "读取至少需要这个数量的匹配k-mers才能被认为是与参考匹配的",
    "A read needs at least this fraction of its total kmers to hit a ref in order to be considered a match": "一个读取至少需要其总kmer的这个分数与参考序列匹配，才能被认为是匹配的",
    "A read needs at least this fraction of its total bases to be covered by ref kmers to be considered a match": "一个读取需要至少这个比例的其总碱基被参考kmer覆盖才能被认为是匹配的",
    "Maximum Hamming distance for ref kmers (subs only)": "参考kmer的最大汉明距离（仅替换）",
    "Hamming distance for query kmers": "查询k-mer的汉明距离",
    "Maximum edit distance from ref kmers (subs and indels)": "最大编辑距离（替换和插入/删除）从参考k-mers",
    "Do not match kmers comntaining N?": "不要匹配包含N的kmers？",
    "Trim failed reads to 1bp instead of discarding them?": "将失败的读取修剪到1个碱基而不是丢弃它们？",
    "Associate read with sequence sharing most kmers if multiple matches?": "如果多个匹配，将读取与共享最多k-mer的序列关联？",
    "Don't do kmer-based operations on read 1?": "不要对读1进行k-mer基础的操作？",
    "Don't do kmer-based operations on read 2?": "不要在读取2上执行k-mer基础操作？",
    "Specify outputs": "指定输出",
    "Output statistics?": "输出统计信息？",
    "Specify statistics outputs": "指定统计输出",
    "Output histograms?": "输出直方图？",
    "Reads": "读取",
    "Quality scores": "质量分数",
    "Minimal quality score": "最小质量分数",
    "Minimal length of contiguous segment": "连续片段的最小长度",
    "Select technology": "选择技术",
    "Low quality bases in homopolymers": "同聚体中的低质量碱基",
    "Restrict length of each read to": "限制每个读取的长度为",
    "Quality score file": "质量分数文件",
    "Sequences": "序列",
    "Compute length for these sequences": "计算这些序列的长度",
    "How many title characters to keep?": "要保留多少标题字符？",
    "Strip fasta description from header?": "从标题中去除fasta描述？",
    "FASTA alignment": "FASTA 比对",
    "Define Base Offsets as": "定义基础偏移量为",
    "Offset from 5' end": "5'端的偏移量",
    "Offset from 3' end": "从3'端的偏移量",
    "Keep reads with zero length": "保留长度为零的读取序列",
    "FASTQ File": "FASTQ 文件",
    "Trim ends": "修剪末端",
    "Step size": "步长",
    "Maximum number of bases to exclude from the window during aggregation": "聚合窗口期间排除的最大碱基数量",
    "Aggregate action for window": "窗口聚合操作",
    "Trim until aggregate score is": "修剪直到聚合分数是",
    "Quality score": "质量分数",
    "FASTQ file to convert": "FASTQ 文件转换",
    "How many columns to divide title string into?": "标题字符串要分成多少列？",
    "FASTA File": "FASTA 文件",
    "Quality Score File": "质量分数文件",
    "Force Quality Score encoding": "强制质量分数编码",
    "Minimum size": "最小尺寸",
    "Maximum size": "最大尺寸",
    "Minimum quality": "最小质量",
    "Maximum quality": "最大质量",
    "Maximum number of bases allowed outside of quality range": "允许质量范围外的最大碱基数量",
    "This is paired end data": "这是成对末端数据",
    "Quality Filter on a Range of Bases": "在一系列碱基上的质量过滤",
    "Aggregate read score for specified range": "聚合指定范围的读取分数",
    "Keep read when aggregate score is": "当聚合分数是时保留读取",
    "File to groom": "要整理的文件",
    "Input FASTQ quality scores type": "输入 FASTQ 质量分数类型",
    "Output FASTQ quality scores type": "输出 FASTQ 质量分数类型",
    "Summarize input data": "总结输入数据",
    "Fix inconsistent identifiers": "修正不一致的标识符",
    "Match Reads": "匹配读取",
    "Match Reads by": "匹配读取",
    "Identifier Match Type": "标识符匹配类型",
    "Match by": "匹配方式",
    "Sequence Match Type": "序列匹配类型",
    "Quality Match Type": "质量匹配类型",
    "Manipulate Reads": "操作读取",
    "Manipulate Reads on": "操作读取数据",
    "Identifier Manipulation Type": "标识符操作类型",
    "From": "从",
    "To": "为了",
    "Sequence Manipulation Type": "序列操作类型",
    "New adapter": "新适配器",
    "Quality Manipulation Type": "质量操作类型",
    "Miscellaneous Manipulation Type": "杂项操作类型",
    "File to mask": "文件遮罩",
    "Mask input with": "用掩码输入",
    "When score is": "当得分是",
    "FASTQ reads": "FASTQ 读取",
    "Type of paired-end datasets": "配对末端数据集的类型",
    "Left-hand mates": "左侧配对",
    "Right-hand mates": "右手配对",
    "Paired-end reads collection": "配对末端读取集合",
    "Left-hand Reads": "左侧序列",
    "Right-hand Reads": "右向读取",
    "FASTQ Header Style": "FASTQ 头部样式",
    "Bases to insert between joined reads": "插入连接的读取之间的碱基",
    "Rename sequence identifiers to": "将序列标识符重命名为",
    "First base to keep": "首先保留碱基",
    "Last base to keep": "保留的最后一个碱基",
    "Discard sequences with unknown (N) bases": "丢弃含有未知（N）碱基的序列",
    "Rename sequence names in output file (reduces file size)": "在输出文件中重命名序列名称（减小文件大小）",
    "Compress output FASTA": "压缩输出FASTA",
    "Statistics report file": "统计报告文件",
    "Desired output format": "期望的输出格式",
    "Quality cut-off value": "质量截断值",
    "Percent of bases in sequence that must have quality equal to / higher than cut-off value": "序列中质量等于或高于截止值的碱基百分比",
    "New width for nucleotides strings": "核苷酸字符串的新宽度",
    "FastQ data": "FastQ 数据",
    "Bin scores": "二进制分数",
    "Score types to show": "显示分数类型",
    "Maximum read length": "最大读取长度",
    "Single-end or paired reads": "单端或配对的读取",
    "Select paired collection(s)": "选择配对的集合（或多个）",
    "Overrepresented Sequence Analysis": "过表达序列分析",
    "Enable overrepresented analysis": "启用过度表达分析",
    "Overrepresentation sampling": "过采样",
    "Filter Options": "过滤选项",
    "Quality filtering options": "质量过滤选项",
    "Disable quality filtering": "禁用质量过滤",
    "Qualified quality phred": "合格的质量分值（Phred）",
    "Unqualified percent limit": "不合格百分比限制",
    "N base limit": "N碱基限制",
    "Length filtering options": "长度筛选选项",
    "Disable length filtering": "禁用长度过滤",
    "Length required": "长度要求",
    "Maximum length": "最大长度",
    "Low complexity filtering options": "低复杂性过滤选项",
    "Enable low complexity filter": "启用低复杂性过滤",
    "Complexity threshold": "复杂性阈值",
    "Read Modification Options": "读取修改选项",
    "PolyG tail trimming": "PolyG尾部修剪",
    "PolyX tail trimming": "PolyX尾部修剪",
    "PolyX minimum length": "PolyX 最小长度",
    "UMI processing": "UMI处理",
    "Enable unique molecular identifer": "启用独特的分子标识符",
    "UMI location": "UMI位置",
    "UMI length": "UMI长度",
    "UMI prefix": "UMI前缀",
    "Per read cutting by quality options": "每个读取按质量切割选项",
    "Cut by quality in front (5')": "在前面（5'）处按质量切割",
    "Cut by quality in tail (3')": "在尾部（3'）按质量切割",
    "Cutting window size": "切割窗口大小",
    "Cutting mean quality": "切割平均质量",
    "Base correction by overlap analysis options": "重叠分析的碱基校正选项",
    "Enable base correction": "启用碱基校正",
    "Output HTML report": "输出HTML报告",
    "Output JSON report": "输出 JSON 报告",
    "Fasta file": "FASTA文件",
    "Minimal length": "最小长度",
    "Reads in FASTQ format": "读取FASTQ格式的数据",
    "Remove N bp from the 3' end": "从3'末端移除N个碱基对",
    "Select a paired collection": "选择一个配对集合",
    "Advanced settings": "高级设置",
    "Trim low-quality ends from reads in addition to adapter removal (Enter phred quality score threshold)": "修剪低质量的序列末端，除了去除接头（输入Phred质量分数阈值）",
    "Overlap with adapter sequence required to trim a sequence": "需要与适配器序列重叠以修剪序列",
    "Maximum allowed error rate": "最大允许误差率",
    "Discard reads that became shorter than length N": "丢弃长度小于N的读段",
    "Instructs Trim Galore! to remove N bp from the 5' end of read 1": "指示 Trim Galore! 从读取1的5'端移除N个碱基对",
    "Instructs Trim Galore! to remove N bp from the 5' end of read 2 (Only for paired-end reads)": "指示 Trim Galore! 从配对末端读取的第二个读取的 5' 端移除 N 个碱基对（仅适用于配对末端读取）",
    "Generate a report file": "生成一个报告文件",
    "specify if you would like to retain unpaired reads": "指定是否保留未配对的读取",
    "Unpaired single-end read length cutoff needed for read 1 to be written": "未配对的单端读取长度截止值，需要用于写入读取1",
    "Unpaired single-end read length cutoff needed for read 2 to be written": "未配对的单端读取长度截止值需要为读取2编写",
    "RRBS specific settings": "RRBS特定设置",
    "Specifies that the input file was an MspI digested RRBS sample": "指定输入文件是经过MspI酶切的RRBS样本",
    "Screen quality-trimmed sequences for 'CAA' or 'CGA' at the start of the read and, if found, removes the first two basepairs": "筛选掉序列开头为'CAA'或'CGA'的序列，并在发现这些序列时，移除前两个碱基对。",
    "Trimming settings": "修剪设置",
    "Hard-trimm 5' ends": "硬修剪5'端",
    "Hard-trimm 3' ends": "硬修剪3'端",
    "Mouse epigenetic clock mode": "小鼠表观遗传时钟模式",
    "Remove polyA tails": "去除polyA尾",
    "Reads to deduplicate in SAM or BAM format": "在SAM或BAM格式中去重的读取",
    "Output a matrix of genes and cells, instead of a flat file": "输出一个基因和细胞的矩阵，而不是一个平面文件。",
    "Prepend a label to all column headers": "在所有列标题前添加标签",
    "Label to Prepend": "标签前缀",
    "Allow unpaired reads": "允许未配对的读取",
    "Allowlist of accepted barcodes": "接受的条形码允许列表",
    "Denylist of accepted barcodes": "拒绝接受的条形码列表",
    "Apply correction to cell barcodes?": "应用细胞条形码校正？",
    "Enable quality filter?": "启用质量过滤器？",
    "Phred score threshold": "Phred 分数阈值",
    "Mask UMI bases below threshold": "遮蔽低于阈值的UMI碱基",
    "Single-end or paired-end reads?": "单端或双端测序？",
    "Single-end FASTQ reads": "单端 FASTQ 读取",
    "Paired-end interleaved FASTQ reads": "配对端交错的FASTQ读取",
    "Output only one file with all reads": "输出一个包含所有读取的文件",
    "Paired-end forward strand FASTQ reads": "成对末端正链FASTQ读取",
    "Paired-end reverse strand FASTQ reads": "配对末端反向链FASTQ读取",
    "Paired-end FASTQ reads as paired collection": "配对末端的FASTQ读取作为配对集合",
    "Length threshold": "长度阈值",
    "Don't do 5' trimming": "不要进行5'端修剪",
    "Truncate sequences with Ns at first N position": "在第一个N的位置截断含有N的序列",
    "Reads to group in SAM or BAM format": "在SAM或BAM格式中对读段进行分组",
    "Output a flatfile describing the read groups": "输出一个描述读组的平面文件",
    "Count reads or UMIs": "计数读取或UMIs",
    "Method for detection of knee": "检测膝盖的方法",
    "Use the first N reads to automatically identify the true cell barcodes": "使用前N个读取来自动识别真实的细胞条形码",
    "Don't select a threshold": "不要选择阈值",
    "Detect and correct CBs above the threshold": "检测并校正高于阈值的CBs",
    "Cell parameters": "细胞参数",
    "Specify the number of cell barcodes to accept": "指定要接受的细胞条形码数量",
    "Prior expectation on the upper limit on the number of cells sequenced": "对测序细胞数量上限的先验期望",
    "Hamming distance for correction of barcodes to whitelist barcodes. Set to zero to generate no error correcting metrics": "汉明距离用于校正条形码以匹配白名单条形码。设置为零以不生成任何错误校正指标。",
    "Input data structure": "输入数据结构",
    "Forward reads": "正向序列",
    "Reverse reads": "反向读取",
    "Paired reads": "配对读取",
    "Minimum overlap": "最小重叠",
    "Maximum overlap": "最大重叠",
    "Maximum mismatch density": "最大错配密度",
    "Combine read pairs in both orientations": "结合两个方向的读对",
    "Output a text rendering of the histogram": "输出直方图的文本渲染",
    "Save FLASH log file": "保存FLASH日志文件",
    "Select the format of your input data": "选择您的输入数据格式",
    "Choose the sam file on which you want to do quality control": "选择您想要进行质量控制的sam文件",
    "Choose the bam file on which you want to do quality control": "选择您想要进行质量控制的BAM文件",
    "Enter the experiment name": "输入实验名称",
    "Enter the number of cores to run the mapping quality control": "输入用于运行映射质量控制的核心数",
    "Select the species": "选择物种",
    "Enter the Ensembl database version": "输入Ensembl数据库版本",
    "Perform mQC with unique mapped reads only": "仅使用唯一映射的读段执行 mQC。",
    "Choose the style of the RPF phase plot": "选择 RPF 相位图的风格",
    "Which mapper is used to generate your SAM file?": "你使用的是哪个比对器来生成你的SAM文件？",
    "Only use alignments that mapped fewer times than this value": "只使用映射次数少于该值的比对",
    "Enter the minimum RPF length for gene distribution construction": "输入用于基因分布构建的最小RPF长度",
    "Enter the maximum RPF length for gene distribution construction": "输入用于基因分布构建的最大RPF长度",
    "The way P-site offsets were determined": "P位点偏移的确定方式",
    "Choose a text file with the offsets. Each line should be structured as RPF length - offset, seperated by a tab or a space": "选择一个包含偏移量的文本文件。每一行的结构应该是 RPF 长度 - 偏移量，用制表符或空格分隔",
    "Enter the minimum RPF length for plastid offset calculation": "输入用于叶绿体偏移计算的最小RPF长度",
    "Enter the maximum RPF length for plastid offset calculation": "输入最大RPF长度以计算质体偏移",
    "FASTA or Multi-FASTA file": "FASTA或多序列FASTA文件",
    "Estimated genome size": "估计的基因组大小",
    "Generate gap stats": "生成间隙统计",
    "Input FASTQ": "输入 FASTQ",
    "Output thresholds": "输出阈值",
    "Total bases": "总碱基数",
    "Keep percentage": "保持百分比",
    "Max. length": "最大长度",
    "Min. length": "最小长度",
    "Min. mean quality": "最小均值质量",
    "Min. window quality": "最小窗口质量",
    "External references": "外部参考",
    "Reference assembly": "参考组装",
    "Reference Illumina read": "参考Illumina读取",
    "Score weights": "得分权重",
    "Weight length score": "体重长度得分",
    "Weight mean quality": "权重平均质量",
    "Weight window quality": "权重窗口质量",
    "Read manipulation": "读取操作",
    "Trim non-k-mer-matching": "修剪非k-mer匹配的部分",
    "Split at nr. of bases": "在碱基数目处拆分",
    "Other": "其他",
    "Sliding window size": "滑动窗口大小",
    "Minimum sequence length (after clipping, sequences shorter than this length will be discarded": "最小序列长度（修剪后，短于此长度的序列将被丢弃）",
    "Source": "源",
    "Enter custom clipping sequence": "输入自定义剪切序列",
    "Choose Adapter": "选择接头",
    "Enter a non-zero value to keep the adapter sequence and x bases that follow it": "输入一个非零值以保留适配器序列及其后跟随的x个碱基",
    "Single-end or Paired-end reads?": "单端或双端测序读取？",
    "FASTQ/A file": "FASTQ/A 文件",
    "FASTQ/A file #1": "FASTQ/A 文件 #1",
    "FASTQ/A file #2": "FASTQ/A 文件 #2",
    "Adapter Options": "适配器选项",
    "What to do if a match is found": "如果找到匹配项该怎么办",
    "Maximum error rate": "最大错误率",
    "Do not allow indels (Use ONLY with anchored 5' (front) adapters).": "不允许插入或缺失（仅与锚定的5'端（前端）适配器一起使用）。",
    "Match times": "匹配次数",
    "Minimum overlap length": "最小重叠长度",
    "Match wilcards in reads": "匹配读取中的通配符",
    "Match wilcards in adapters": "匹配适配器中的通配符",
    "Look for adapters in the reverse complement": "在反向互补中查找接头",
    "Discard Trimmed Reads": "丢弃修剪后的读取",
    "Discard Untrimmed Reads": "丢弃未修剪的读段",
    "Minimum length (R1)": "最小长度（R1）",
    "Maximum length (R1)": "最大长度（R1）",
    "Max N": "最大 N",
    "Pair filter": "配对过滤器",
    "Max expected errors": "最大预期错误",
    "Max average expected errors": "最大平均预期误差",
    "Discard CASAVA filtering": "丢弃CASAVA过滤",
    "Cut bases from reads before adapter trimming": "在适配体修剪之前剪切读取的碱基",
    "Quality cutoff": "质量截断",
    "NextSeq trimming": "NextSeq 修剪",
    "Trim Ns": "修剪N碱基",
    "Trim poly-A tails": "修剪聚腺苷酸尾部",
    "Strip suffix": "去除后缀",
    "Shortening reads to a fixed length": "将读取缩短到固定长度",
    "Length": "长度",
    "End at which to apply the slice": "应用切片的结束位置",
    "Length tag": "长度标签",
    "Rename reads": "重命名读取",
    "Change negative quality values to zero": "将负的质量值改为零",
    "Outputs selector": "输出选择器",
    "Statistics text file": "统计文本文件",
    "Barcodes to use": "要使用的条形码",
    "Library to split": "库分割",
    "Barcodes found at": "在找到的条形码",
    "Characters to strip from the end of the sequence id before matching": "在匹配之前，从序列ID的末尾删除的字符",
    "Select index read file": "选择索引读取文件",
    "Number of allowed mismatches": "允许的不匹配数",
    "Number of allowed barcodes nucleotide deletions": "允许的条形码核苷酸缺失数",
    "Alignments in BED format": "BED格式的比对结果",
    "Set position one nt downstream of 3'-end as crosslinked nucleotide": "将3'端下游一个核苷酸的位置设为交联核苷酸",
    "Split by:": "分割依据：",
    "Number of chunks": "块数",
    "Size of chunks (bp)": "块大小（碱基对数）",
    "Specify advanced parameters": "指定高级参数",
    "The maximum edit distance permitted when aligning the paired reads": "在对配对的读取进行比对时允许的最大编辑距离",
    "An arbitrary number of extra bases to trim from the ends of reads": "从读取序列的两端修剪任意数量的额外碱基",
    "Trim this number of bases from the start of each sequence": "从每个序列的起始位置修剪这些碱基的数量",
    "Use the reverse complement of this number of from the beginning of the reverse read to align reads": "使用从反向读取的开始处的这个数字的反向互补序列来对齐读取",
    "Raw read data from your current history": "当前历史记录中的原始读取数据",
    "Adapter list": "适配器列表",
    "Disable grouping of bases for reads >50bp": "禁用对大于50个碱基的读取进行碱基分组",
    "Lower limit on the length of the sequence to be shown in the report": "报告中显示序列长度的下限",
    "Length of Kmer to look for": "要查找的Kmer长度",
    "Barcoded sequences": "条形码序列",
    "Pattern for extracting barcode nucleotides": "提取条形码核苷酸的模式",
    "Alignments in SAM or BAM format": "SAM或BAM格式的比对结果",
    "BED6 file containing alignments": "包含对齐信息的BED6文件",
    "FASTQ barcode library": "FASTQ 条形码库",
    "Threshold for spurious event removal": "杂散事件移除的阈值",
    "Remove this many nucleotides": "移除这么多核苷酸",
    "Barcode read position (BPOS)": "条形码读取位置（BPOS）",
    "Add matched barcode at the end of the read header (ADD)": "在读取头的末尾添加匹配的条形码（ADD）",
    "Select SAM/BAM dataset": "选择 SAM/BAM 数据集",
    "If true do not write duplicates to the output file instead of writing them with appropriate flags set": "如果为真，则不要将重复项写入输出文件，而是将适当的标志设置为已写入",
    "Assume the input file is already sorted": "假设输入文件已经排序",
    "Do you have a predefined list of UMIs": "您是否有预定义的UMI列表？",
    "UMI file": "UMI文件",
    "Barcode file": "条形码文件",
    "Unique Molecular Identifier location": "唯一分子标识符位置",
    "Where to find the UMIs in the read name": "在读取名称中在哪里可以找到UMIs（唯一分子标识符）",
    "Number of maximum mismatches to consider two Unique Molecular Identifiers (UMIs) similar": "考虑两个独特的分子标识符（UMIs）相似时，最大不匹配的数量",
    "Maximum number of Ns a UMI can contain": "一个UMI可以包含的最大Ns数量",
    "Character to split up the header": "用于分隔标题的字符",
    "Should barcode information be removed from read names in the output BAM": "输出BAM文件中是否应去除读名字中的条形码信息",
    "Unique Molecular Identifier location for trimming": "UMI剪切位置",
    "Where to find the UMIs in the read name that should be removed from the header": "在读取名称中找到应该从标题中删除的UMIs的位置",
    "Comment": "注释",
    "Add this comment to BAM dataset": "将此注释添加到BAM数据集",
    "The scoring strategy for choosing the non-duplicate among candidates": "选择候选重复项中的非重复项的评分策略",
    "Regular expression that can be used to parse read names in the incoming SAM/BAM dataset": "用于解析传入的SAM/BAM数据集中读取名称的正则表达式",
    "The maximum offset between two duplicte clusters in order to consider them optical duplicates": "两个重复簇之间的最大偏移量，以便将它们视为光学重复品。",
    "Select validation stringency": "选择验证的严格程度",
    "Do your reads contain Unique Molecular Identifiers(UMIs)": "你的测序数据是否包含独特的分子标识符（UMIs）",
    "Output UMI related statistics files?": "输出UMI相关统计文件？",
    "Run in batch mode?": "在批处理模式下运行？",
    "Input FASTA File(s)": "输入FASTA文件",
    "How are sequences judged to be unique?": "序列如何被判断为唯一？",
    "Accession Parsing Regex": "访问号解析正则表达式",
    "Aligned protein file": "对齐的蛋白质文件",
    "Genetic code": "遗传密码",
    "Unaligned nucleotide sequences": "未比对的核苷酸序列",
    "Do you want k or K?": "您想要 k 还是 K？",
    "type in 3 parameters": "输入3个参数",
    "type in 2 parameters": "输入两个参数",
    "Maximum number of correction for within a kmer length window": "在kmer长度窗口内的最大校正数",
    "Whether or not to allow trimming": "是否允许修剪",
    "Whether or not to discard unfixable reads. Will lose paired-end matching when discarding": "是否丢弃无法修复的读取。丢弃时会丢失配对末端匹配",
    "Whether or not to ignore the quality socre": "是否忽略质量分数",
    "Quality for the bases corrected to the specified score": "将指定分数校正后的碱基质量",
    "Sequentialize the sampling stage, output the same result with different runs": "序列化采样阶段，不同运行输出相同的结果",
    "Dataset type": "数据集类型",
    "Read 1 Fastq": "读取1个Fastq文件",
    "Read 2 Fastq": "读取2个Fastq",
    "FASTQ Paired Dataset": "FASTQ配对数据集",
    "Split read ids on this character": "在该字符上拆分读取ID",
    "Maximum percentage difference between matching segments": "匹配片段之间最大百分比差异",
    "Minimum length of matching segements": "匹配片段的最小长度",
    "Output verbose stitch length report": "输出详细的拼接长度报告",
    "Fasta file to split": "将FASTA文件分割",
    "Split mode": "分割模式",
    "Number of chunks to split into": "要分成的块数",
    "Specify target sequences": "指定目标序列",
    "Target sequence": "目标序列",
    "Include specific intervals": "包括特定的区间",
    "Exclude specific intervals": "排除特定区间",
    "Tool mode": "工具模式",
    "SAK input file": "SAK 输入文件",
    "Generates the initial set of paths": "生成初始路径集",
    "Sort sequences": "排序序列",
    "Homopolymer compression": "同聚物压缩",
    "Report mode": "报告模式",
    "Feature for reporting sizes": "报告大小的功能",
    "BED coordinares feature": "BED坐标特征",
    "Expected genome size": "预期的基因组大小",
    "Report actual sequence": "报告实际序列",
    "Thousands separator in output": "输出中的千位分隔符",
    "Tabular-format output": "表格格式输出",
    "Input AGP file": "输入 AGP 文件",
    "FastQ file": "FastQ 文件",
    "FastQ file R1 (left)": "FastQ文件 R1（左侧）",
    "FastQ file R2 (right)": "FastQ 文件 R2（右）",
    "Filter uncorrectable reads": "过滤无法校正的读取",
    "Additional options": "额外选项",
    "kmer length": "kmer长度",
    "max correction within k-bp window": "在k-bp窗口内的最大校正",
    "estimate weak kmer count": "估算弱k-mer计数",
    "expected number of kmers": "期望的k-mer数量",
    "Name of file that contains the forward paired-end reads": "包含正向配对末端读取的文件名",
    "Name of file that contains the reverse paired-end reads": "包含反向配对端读取的文件名",
    "Specify a p-value for the statistical test": "指定统计检验的p值",
    "Minimum overlap size": "最小重叠大小",
    "Maximum possible length of the assembled sequences": "组装序列的最大可能长度",
    "Minimum possible length of the assembled sequences": "组装序列的最小可能长度",
    "Minimum length of reads after trimming the low quality part": "修剪低质量部分后读取的最小长度",
    "Quality score threshold for trimming the low quality part of a read": "修剪低质量读取部分的质量分数阈值",
    "Maximal proportion of uncalled bases in a read": "未调用碱基在读取中的最大比例",
    "Specify  the upper bound for the resulting quality score": "指定生成的质量分数的上限",
    "Type of statistical test": "统计检验类型",
    "Disable empirical base frequencies": "禁用经验基础频率",
    "Use N base if uncertain": "如果不确定，使用N个碱基",
    "Scoring method": "评分方法",
    "Output files": "输出文件",
    "Input reads in FASTQ format": "输入FASTQ格式的读取序列",
    "Output all statistics?": "输出所有统计信息？",
    "Output base name of input file?": "输出输入文件的基本名称？",
    "Skip errors?": "跳过错误？",
    "Output machine-friendly tabular format?": "输出机器友好的表格格式？",
    "Output alphabet letters?": "输出字母表中的字母？",
    "Output average quality of the read?": "输出读段的平均质量？",
    "Output GC content?": "输出GC含量？",
    "Output GC-Skew?": "输出GC-Skew？",
    "Output header line?": "输出标题行？",
    "Output sequence length?": "输出序列长度？",
    "Output only names (no sequences and qualities)?": "仅输出名称（不包括序列和质量）？",
    "Eliminate qualities from output?": "从输出中消除特性？",
    "Output id instead of full head?": "输出ID而不是完整的头信息？",
    "Output md5 hash of sequence?": "输出序列的md5哈希值？",
    "Choose the category of the reads files to be analyzed": "选择要分析的读取文件的类别",
    "Read file": "读取文件",
    "Forward read file": "正向读取文件",
    "Reverse read file": "反向读取文件",
    "Collection of fastq paired read files": "快速q配对读取文件集合",
    "Interleaved read file": "交错读取文件",
    "FASTQ Options": "FASTQ选项",
    "Maximum Phred score expected in input files and used when writing output files": "输入文件中预期的最大Phred分数，并在写入输出文件时使用",
    "FASTQ Trimming Options": "FASTQ 修剪选项",
    "Adapter sequence expected to be found in mate 1 reads, specified in read direction": "在配对1的读取中预期找到的接头序列，按照读取方向指定",
    "Adapter sequence expected to be found in mate 2 reads, specified in read direction": "在配对的第2个读取中预期会找到的接头序列，以读取方向指定",
    "Tabular file containing adapter sequences": "包含接头序列的表格文件",
    "Trim reads only if the overlap between read and the adapter is at least this number of bases long": "仅当读段与适配器之间的重叠部分至少有这么多碱基长时才修剪读段",
    "The fraction of mismatches allowed in the aligned region": "允许在对齐区域中的不匹配比例",
    "Slip the alignment by this number of bases in the 5' end to allow for missing bases in the 5' end of the read": "将比对在5'端滑动这么多个碱基，以允许读取的5'端缺失碱基",
    "Trim the 5' of reads by a fixed amount after removing adapters, but before carrying out quality based trimming?": "在移除适配器后，但在进行基于质量的修剪之前，通过固定数量修剪读取的5'端。",
    "Value for trimming the 5' of mate1 reads": "对 mate1 读取的 5' 端进行修剪的值",
    "Trim the 3' of reads by a fixed amount after removing adapters, but before carrying out quality based trimming?": "在去除接头后，但在进行基于质量的修剪之前，通过固定数量修剪读取的3'端。",
    "Value for trimming the 3' of mate1 reads": "修剪 mate1 读取的 3' 端的值",
    "Trim consecutive Ns from the 5' and 3' termini?": "从5'和3'末端修剪连续的Ns？",
    "Discard reads containing more than this number of ambiguous bases ('N') after trimming": "丢弃在修剪后含有多于这个数字的模糊碱基（'N'）的读段",
    "Trim consecutive stretches of low quality bases (threshold set by --minquality) from the 5' and 3' termini?": "从5'和3'末端修剪连续的低质量碱基（阈值由--minquality设置）？",
    "Perform sliding window approach to quality base-trimming inspired by sickle?": "执行滑动窗口方法来进行受sickle启发的质量基修剪。",
    "Threshold for trimming low quality bases using --trimqualities and --trimwindows": "使用 --trimqualities 和 --trimwindows 修剪低质量碱基的阈值",
    "Preserve (i.e., do not trim by --trimns, --trimqualities, and --trimwindows) bases at the 5p?": "保留（即，通过 --trimns、--trimqualities 和 --trimwindows 不进行修剪）5'端的碱基吗？",
    "Reads shorter than this length are discarded following trimming": "修剪后，比这个长度短的读段将被丢弃",
    "Reads longer than this length are discarded following trimming": "读取长度超过此长度的序列在修剪后将被丢弃",
    "FASTQ Merging Options": "FASTQ 合并选项",
    "Output complete reads with an 'M_' name prefix and trimmed reads with an 'MT_' name prefix?": "输出完整读取的序列，其名称前缀为'M_'，修剪后的读取序列，其名称前缀为'MT_'。",
    "Minimum overlap between mate 1 and mate 2 before the reads are collapsed into one when collapsing paired-end reads, or when attempting to identify complete template sequences in single-end mode": "在将配对末端读取合并为一个读取时，或在单端模式下尝试识别完整的模板序列时，配对1和配对2之间的最小重叠",
    "Collapse deterministically?": "确定性地合并？",
    "Collapse conservatively?": "保守地合并？",
    "Optionally select one or more of the following additional outputs": "可选择性地选择以下一个或多个附加输出",
    "Pattern mode": "模式模式",
    "Pattern/motif sequence": "模式/基序序列",
    "Pattern/motifs are regular expressions": "模式/基序是正则表达式",
    "Pattern/motif file": "模式/基序文件",
    "Sequence type": "序列类型",
    "Output mode": "输出模式",
    "Lenth of the sequence to validate": "序列长度验证",
    "Circular genome": "环状基因组",
    "Pattern/motif contains degenerate bases": "模式/基序包含简并碱基",
    "Do not show matched sequences": "不要显示匹配的序列",
    "Ignore case": "忽略大小写",
    "Maximum mismatch": "最大不匹配",
    "Non-greedy mode": "非贪婪模式",
    "Only search on positive strand": "仅在正链上搜索",
    "FM-index": "FM索引",
    "FASTA head is NCBI stype": "FASTA 头是 NCBI 风格",
    "Regular expression": "正则表达式",
    "Match case": "匹配大小写",
    "Do not search the reverse complement": "不要搜索反向互补序列",
    "Maximum length of the match to report": "最大匹配长度报告",
    "Space separated list of fasta sequences to search": "用空格分隔的fasta序列列表进行搜索",
    "Quality and trimming options": "质量和修剪选项",
    "Select option for quality trimming ends before mapping": "选择在映射前对末端进行质量修剪的选项",
    "Trim regions with average quality below this value": "修剪平均质量低于此值的区域",
    "Don't trim reads to be shorter than this value": "不要将读取修剪到短于这个值",
    "Use quality scores when determining which read kmers to use as seeds?": "在确定使用哪些读取的kmer作为种子时，是否使用质量分数？",
    "Merging parameters": "合并参数",
    "Error-correct the overlapping part, but don't merge.": "校正重叠部分的错误，但不要合并。",
    "Trim all non-overlapping portions, leaving only consensus sequence. By default, only sequence to the right of the overlap (adapter sequence) is trimmed.": "修剪所有不重叠的部分，只留下共识序列。默认情况下，只修剪重叠部分右侧的序列（适配器序列）。",
    "Minimum insert size to merge reads": "最小插入大小以合并读取",
    "Minimum number of overlapping bases to allow merging": "允许合并的最小重叠碱基数",
    "Ignore bases with quality below this": "忽略质量低于此值的碱基",
    "Cap output quality scores at this": "将输出的质量分数限制在这个值",
    "Increase the minimum overlap requirement for low-complexity reads": "增加低复杂性序列的最小重叠要求",
    "Ban overlaps with over this many times the expected number of errors.": "Ban 与预期错误次数的重叠超过了这么多倍。",
    "Probability filter to disallow improbable overlaps.": "概率过滤器，用于排除不可能的重叠。",
    "Ban overlaps that create kmers with count below this value": "禁止创建计数低于此值的kmer的重叠区域",
    "Take quality factors into account": "考虑质量因素",
    "Provide adapter sequences to improve accuracy?": "提供接头序列以提高准确性？",
    "Left adapter sequence": "左适配器序列",
    "Right adapter sequence": "右侧接头序列",
    "Evaluate overlaps via...": "评估重叠部分通过...",
    "Max error rate; higher increases merge rate.": "最大错误率；提高会增加合并率。",
    "Lower increases merge rate; min is 1.": "降低增加合并率；最小值为1。",
    "Lower increases merge rate; min is 0.": "降低增加合并率；最小是0。",
    "Maximum mismatches allowed in overlapping region.": "在重叠区域允许的最大错配数。",
    "Cutoff for second-best overlap ratio.": "次优重叠比的截断值。",
    "The best overlap must have at least 'margin' fewer mismatches than the second best.": "最佳重叠必须至少比第二佳重叠少'margin'个不匹配。",
    "Do not allow more than this many mismatches.": "不允许有超过这么多的不匹配。",
    "Require the answer from flat mode and ratio mode to agree, reducing false positives if both are enabled.": "要求平面模式和比率模式的答案一致，如果两者都启用，则减少误报。",
    "Results": "结果",
    "Which tool was used generate logs?": "哪个工具被用来生成日志？",
    "Output of BAMtools": "BAMtools的输出",
    "Bismark output": "Bismark输出",
    "Type of Bismark output?": "Bismark输出类型？",
    "Output of Bowtie 2": "Bowtie 2的输出",
    "Output of BUSCO": "BUSCO的输出",
    "Output of Cutadapt": "Cutadapt的输出",
    "deepTools output": "deepTools 输出",
    "deepTool used to generate the output?": "DeepTool 用于生成输出？",
    "Output of fastp": "fastp 的输出",
    "FastQC output": "FastQC输出",
    "Type of FastQC output?": "FastQC输出类型？",
    "Output of FeatureCounts": "FeatureCounts的输出",
    "Output of Flexbar": "Flexbar的输出",
    "GATK output": "GATK输出",
    "Type of GATK output?": "GATK输出类型？",
    "Output of HiCExplorer": "HiCExplorer的输出",
    "Output of HISAT2": "HISAT2的输出",
    "Output of HTSeq": "HTSeq的输出",
    "Output of Kallisto": "Kallisto的输出",
    "Output of MACS2": "MACS2的输出",
    "Picard output": "Picard输出",
    "Type of Picard output?": "Picard输出类型？",
    "Output of Prokka": "Prokka的输出",
    "Output of PycoQC": "PycoQC的输出",
    "Output of Qualimap BamQC": "Qualimap BamQC的输出",
    "Output of Quast": "Quast的输出",
    "RSeQC output": "RSeQC输出",
    "Type of RSeQC output?": "RSeQC输出类型？",
    "RSeQC BAM stat: output file": "RSeQC BAM 统计：输出文件",
    "RSeQC gene body coverage:  stats file": "RSeQC基因体覆盖度：统计文件",
    "RSeQC infer experiment: configuration output": "RSeQC推断实验：配置输出",
    "RSeQC inner distance: frequency file": "RSeQC内距离：频率文件",
    "RSeQC junction annotation: stats file": "RSeQC 接头注释：统计文件",
    "RSeQC junction saturation: junction saturation plot Rscript file": "RSeQC 结合饱和：结合饱和图 Rscript 文件",
    "RSeQC read distribution: stats output": "RSeQC 读取分布：统计输出",
    "RSeQC read duplication: positions XLS file": "RSeQC 重复读取：位置 XLS 文件",
    "RSeQC read GC output": "RSeQC 读取 GC 输出",
    "Output of Samblaster": "Samblaster的输出",
    "Samtools output": "Samtools输出",
    "Type of Samtools output?": "Samtools输出类型？",
    "Samtools stats output": "Samtools 统计输出",
    "Samtools flagstat output": "Samtools 标志统计输出",
    "Samtools idxstats output": "Samtools idxstats 输出",
    "Samtools rmdup output": "Samtools rmdup 输出",
    "Output of Slamdunk Alleyoop": "Slamdunk Alleyoop的输出",
    "Output of SnpEff": "SnpEff的输出",
    "Log file of SortMeRNA": "SortMeRNA的日志文件",
    "STAR output": "STAR输出",
    "Type of STAR output?": "STAR输出类型？",
    "STAR log output": "STAR日志输出",
    "STAR gene count output": "STAR基因计数输出",
    "Output of TopHat2": "TopHat2的输出",
    "Output of Trimmomatic": "Trimmomatic的输出",
    "VCFTools output": "VCFTools 输出",
    "Type of VCFTools output?": "VCFTools 输出类型？",
    "VCFTools relatedness2 output": "VCFTools 相关性2 输出",
    "VCFTools TsTV count output": "VCFTools TsTV计数输出",
    "VCFTools TsTV qual output": "VCFTools TsTV 质量输出",
    "VCFTools TsTV summary output": "VCFTools TsTV 汇总输出",
    "The plot type to visualise the data with": "可视化数据的图表类型",
    "Section name": "小节名称",
    "X axis label": "X轴标签",
    "Y axis label": "Y轴标签",
    "Data files": "数据文件",
    "Report title": "报告标题",
    "Custom comment": "自定义评论",
    "Use only flat plots (non-interactive images)": "只使用平面图（非交互式图像）",
    "Output the multiQC plots raw data?": "输出 multiQC 图表的原始数据？",
    "Output the multiQC log file?": "输出 multiQC 日志文件？",
    "Single reads or paired reads": "单端读取或双端读取",
    "FASTQ file: forward reads": "FASTQ 文件：正向读取",
    "FASTQ file: reverse reads": "FASTQ 文件：反向读取",
    "Determine quality scoring": "确定质量评分",
    "Skip check for duplicated readnames": "跳过检查重复的读名",
    "Disallow internal adaptor ocurrences": "不允许内部适配体出现",
    "Match wildcards": "匹配通配符",
    "Specify a minimum/maximum length for reverse reads (R2)": "指定反向读取（R2）的最小/最大长度",
    "Minimum length (R2)": "最小长度（R2）",
    "Maximum length (R2)": "最大长度（R2）",
    "Mode options": "模式选项",
    "Select mode": "选择模式",
    "Write kmers and counts as fasta.": "将k-mer序列及其计数以fasta格式写出。",
    "Dump kmers with at least this depth.": "转储至少具有此深度的k-mers。",
    "Choose the mode": "选择模式",
    "Database": "数据库",
    "Input file (Fasta file)": "输入文件（Fasta文件）",
    "Taxonomy entry": "分类学条目",
    "Taxonomic identifier": "分类学标识符",
    "Species binomial name": "物种双名法名称",
    "Taxonomic identifier(s) to exclude": "要排除的分类标识符（一个或多个）",
    "Additional contaminants to identify": "需要识别的额外污染物",
    "Split fasta sequences on N-runs of length at least 10": "将长度至少为10的N-串联序列拆分",
    "BLAST-div of the tax-id": "BLAST-物种分类ID的划分",
    "Database location": "数据库位置",
    "Select Action report": "选择操作报告",
    "Minimumm sequence length to keep": "最小序列长度以保留",
    "First Read Input": "首次读取输入",
    "Second Read Input": "第二读输入",
    "Merge Reads": "合并读取",
    "General Arguments (Optional)": "通用参数（可选）",
    "First Read Discarded FASTQ Filename": "首读丢弃的 FASTQ 文件名",
    "Second Read Discarded FASTQ Filename": "第二读丢弃的FASTQ文件名",
    "Input Sequence is in Phred+64 Format": "输入序列采用Phred+64格式",
    "Quality Score Cutoff": "质量分数截断",
    "Minimum Length of Reads": "最小读取长度",
    "Additional Adapter/Primer Trimming Arguments": "额外的接头/引物修剪参数",
    "Adapter Sequence A": "适配器序列 A",
    "Adapter Sequence B": "适配器序列 B",
    "Minimum Overall Base Pair Overlap with Adapter": "最小整体碱基对与适配器重叠",
    "Maximum Fraction of Good Quality Mismatching Bases": "最大分数的优质不匹配碱基",
    "Minimum Fraction of Matching Bases": "最小匹配碱基分数",
    "Adapter Alignment Band-width": "适配器对齐带宽",
    "Adapter Alignment Gap-Open": "适配器对齐间隙-开启",
    "Adapter Alignment Gap-Extension": "适配器对齐间隙-扩展",
    "Adapter Alignment Gap-End": "适配器对齐间隙-末端",
    "Minimum Local Alignment Score Cutoff": "最小局部比对得分截断值",
    "Read Alignment Band-width": "读取对齐带宽",
    "Read Alignment Gap-Open": "读取比对间隙-开放",
    "Read Alignment Gap-Extension": "读取比对间隙-扩展",
    "Read Alignment Gap-End": "读取比对间隙-末端",
    "Read Alignment Maximum Fraction Gap Cutoff": "读取对齐最大分数间隙截止值",
    "Optional Arguments for Merging": "可选参数用于合并",
    "Maximum Quality Score in Output": "输出中的最大质量分数",
    "Print Overhang When Adapters Are Present and Stripped": "当存在并已剥离适配器时打印悬挂端",
    "Minimum Overall Base Pair Overlap": "最小整体碱基对重叠",
    "Single or Paired-end reads": "单端或配对末端读取",
    "Select first set of reads": "选择第一组读段",
    "Select second set of reads": "选择第二组读取",
    "Select fasta/fastq dataset": "选择fasta/fastq数据集",
    "Subsample reads based on": "基于...对读取进行子采样",
    "Specify genome size in": "指定基因组大小",
    "Genome size to calculate coverage with respect to": "基因组大小，用于计算覆盖度",
    "The desired coverage to sub-sample the reads to": "所需的覆盖度以对读取进行子采样",
    "Specify number of bases in": "指定碱基数量",
    "Explicitly set the number of bases required": "明确设定所需的碱基数量",
    "Random seed to use": "随机种子用于",
    "Input FASTQ file": "输入 FASTQ 文件",
    "Input FASTQ file (R1/first of pair)": "输入FASTQ文件（R1/第一对）",
    "Input FASTQ file (R2/second of pair)": "输入FASTQ文件（配对中的第二个/R2）",
    "Select FASTQ dataset collection with R1/R2 pair": "选择具有R1/R2配对的FASTQ数据集集合",
    "Perform initial ILLUMINACLIP step?": "执行初始的ILLUMINACLIP步骤？",
    "Select standard adapter sequences or provide custom?": "选择标准适配器序列或提供自定义的？",
    "Adapter sequences to use": "要使用的接头序列",
    "Custom adapter sequences in fasta format": "自定义的接头序列在fasta格式中",
    "Maximum mismatch count which will still allow a full match to be performed": "允许的最大不匹配数，仍然可以执行完整匹配",
    "How accurate the match between the two 'adapter ligated' reads must be for PE palindrome read alignment": "两个“接头连接”的配对回文读取对齐的匹配精度必须有多高？",
    "How accurate the match between any adapter etc. sequence must be against a read": "任何适配器等序列与读取之间的匹配必须有多准确",
    "Minimum length of adapter that needs to be detected (PE specific/palindrome mode)": "需要检测到的接头最小长度（特定于成对模式/回文模式）",
    "Always keep both reads (PE specific/palindrome mode)?": "始终保留两个读取（成对特定/回文模式）？",
    "Trimmomatic Operation": "Trimmomatic操作",
    "Select Trimmomatic operation to perform": "选择要执行的 Trimmomatic 操作",
    "Number of bases to average across": "平均覆盖的碱基数量",
    "Average quality required": "平均质量要求",
    "Minimum length of reads to be kept": "保留的读取的最小长度",
    "Minimum quality required to keep a base": "保持碱基所需的最小质量",
    "Number of bases to keep from the start of the read": "从读取的起始处保留的碱基数量",
    "Number of bases to remove from the start of the read": "从读取序列的起始处移除的碱基数量",
    "Minimum average quality required to keep a read": "保持读取所需的最小平均质量",
    "Target read length": "目标读取长度",
    "Strictness": "严格性",
    "Quality score encoding": "质量得分编码",
    "Output trimlog file?": "输出修剪日志文件？",
    "Output trimmomatic log messages?": "输出 Trimmomatic 日志信息？",
    "Enter mean, standard deviation, max, and min for insert lengths.": "输入插入长度的均值、标准差、最大值和最小值。",
    "Select fastq dataset": "选择fastq数据集",
    "Select analysis mode": "选择分析模式",
    "Set algorithmic options?": "设置算法选项？",
    "Minimum seed length": "最小种子长度",
    "Band width for banded alignment": "带状比对的带宽",
    "Off-diagonal X-dropoff": "非对角X下降",
    "Look for internal seeds inside a seed longer than -k * THIS VALUE": "寻找比 -k * 该值 还长的种子内部的种子",
    "Seed occurrence for the 3rd round seeding": "第三轮种子出现",
    "Skip seeds with more than that many occurrences": "跳过出现次数超过该数值的种子",
    "Drop chains shorter than this fraction of the longest overlapping chain": "删除比最长重叠链短于此分数的链",
    "Discard a chain if seeded bases shorter than THIS VALUE": "丢弃如果种子碱基短于此值的链",
    "Perform at most this many rounds of mate rescues for each read": "对每个读取执行最多这么多轮的配偶救援",
    "Skip mate rescue": "跳过配偶救援",
    "Skip pairing; mate rescue performed unless -S also in use": "跳过配对；除非同时使用-S，否则执行配偶救援",
    "Discard full-length exact matches": "丢弃完全匹配的完整序列",
    "Set scoring options?": "设置评分选项？",
    "Score for a sequence match": "序列匹配的得分",
    "Penalty for a mismatch": "不匹配的惩罚",
    "Gap open penalties for deletions and insertions": "删除和插入的间隙开放惩罚",
    "Gap extension penalties; a gap of size k cost '-O + -E*k'. If two numbers are specified, the first is the penalty of extending a deletion and the second for extending an insertion": "间隙延伸惩罚；大小为k的间隙成本为'-O + -E*k'。如果指定了两个数字，第一个是延伸删除的惩罚，第二个是延伸插入的惩罚。",
    "Penalties for 5'-end and 3'-end clipping": "5'端和3'端剪切的惩罚",
    "Penalty for an unpaired read pair": "未配对的读对的惩罚",
    "Set input/output options": "设置输入/输出选项",
    "Minimum score to output": "输出的最低分数",
    "If there are less than THIS VALUE hits with score >80% of the max score, output them all in the XA tag": "如果得分大于最大得分的80%的命中数少于该值，则在XA标签中输出它们全部",
    "Output all alignments for single-ends or unpaired paired-ends": "输出所有单端或未配对的配对末端的比对结果",
    "Append FASTA/FASTQ comment to BAM output": "将FASTA/FASTQ注释添加到BAM输出",
    "Output the reference FASTA header in the XR tag": "在XR标签中输出参考FASTA标题",
    "Use soft clipping for supplementary alignments": "使用软剪切进行补充比对",
    "Mark shorter split hits of a chimeric alignment in the FLAG field as 'secondary alignment' instead of 'supplementary alignment'": "在 FLAG 字段中，将较短的分裂命中的嵌合对齐标记为“次要对齐”，而不是“补充对齐”。",
    "Select input type": "选择输入类型",
    "Select BAM dataset": "选择BAM数据集",
    "maximum edit distance if the value is integer, or the fraction of missing alignments given 2% uniform base error rate if float. In the latter case, the maximum edit distance is automatically chosen for different read lengths.": "最大编辑距离，如果值为整数，则为最大编辑距离；如果为浮点数，则表示在2%的均匀碱基错误率下缺失对齐的比例。在后一种情况下，不同读长的最大编辑距离将自动选择。",
    "maximum number or gap openings": "最大间隔开口数",
    "maximum number of gap extensions": "最大间隔扩展数",
    "do not put an indel within this many bp towards the ends": "不要在两端这么多碱基对范围内放置插入或缺失",
    "maximum occurrences for extending a long deletion": "最大出现次数用于扩展一个长删除",
    "seed length": "种子长度",
    "maximum differences in the seed": "种子中的最大差异",
    "maximum entries in the queue": "队列中的最大条目数",
    "mismatch penalty": "错配惩罚",
    "gap open penalty": "间隙开放惩罚",
    "gap extension penalty": "间隙延伸惩罚",
    "stop searching when there are more than this value of equally best hits": "当有超过这个值的同样最佳命中时停止搜索",
    "quality threshold for read trimming down to 35bp": "读取修剪的质量阈值降低到35bp",
    "length of barcode": "条形码长度",
    "log-scaled gap penalty for long deletions": "对长删除的对数缩放间隙惩罚",
    "Megablast XML output": "Megablast XML输出",
    "Compare these sequences": "比较这些序列",
    "against target database": "针对目标数据库",
    "using word size": "使用单词大小",
    "report hits above this identity (-perc_identity)": "报告高于这个相似度的命中（-perc_identity）",
    "set expectation value cutoff (-evalue)": "设置期望值截止值（-evalue）",
    "Filter out low complexity regions? (-dust)": "过滤掉低复杂性区域？（-dust）",
    "Select a genome reference from your history or a built-in index?": "从您的历史记录中选择一个基因组参考，或者选择一个内置索引？",
    "Select a genome": "选择一个基因组",
    "Is this library mate-paired?": "这个库是配对末端测序的吗？",
    "FASTQ": "FASTQ",
    "First read in pair": "第一读段配对",
    "Second read in pair": "配对中的第二个读段",
    "FASTQ paired dataset": "FASTQ配对数据集",
    "Read group": "读取组",
    "Align sequencing reads in": "对测序读取进行比对",
    "Against reference sequences that are": "针对参考序列",
    "Using reference genome": "使用参考基因组",
    "Select a reference dataset": "选择一个参考数据集",
    "Linker file": "连接文件",
    "Select a base quality score 454 dataset": "选择一个基础质量得分454数据集",
    "Do you want to modify the reference name?": "您想要修改参考序列名称吗？",
    "Enter name for the Reference sequence": "输入参考序列的名称",
    "Select QUERY sequence(s)": "选择查询序列",
    "Use lastz_32": "使用 lastz_32",
    "Where to look": "在哪里查找",
    "which strand to search": "搜索哪个链",
    "Perform a self-alignment: the target sequence is also the query.": "执行自我比对：目标序列也是查询序列。",
    "Inhibit the re-creation of mirror-image alignments.": "禁止重建镜像对齐。",
    "Set HSP limit": "设置高得分片段（HSP）限制",
    "Discard queries that have more than N HSPs": "丢弃具有超过N个HSP的查询",
    " Withhold warnings for queries that exceed the limit set above": "对于超过上述设置限制的查询，不显示警告。",
    "Keep queries that exceed the limit and supress warnings": "保留超出限制的查询并抑制警告",
    "For queries that have more than N HSPs, discard any HSPs that score below the Nth best.": "对于有超过N个HSP的查询，丢弃那些得分低于第N好的HSP。",
    "Set ratio of aligned bases to query length": "设置比对碱基与查询序列长度的比率",
    "Stop processing gapped alignments for a query/strand if its ratio of aligned bases to query length exceeds": "如果查询/链的对齐碱基与查询长度之比超过，则停止处理该查询/链的间隙对齐。",
    "Warnings for queries that exceed the limit are witheld": "超过限制的查询警告被保留",
    "Same as above but any alignments discovered for this query/strand, before it exceeds the threshold, are reported.": "与上述相同，但在超过阈值之前，报告了针对此查询/链的任何比对。",
    "Scoring": "打分",
    "Read the substitution scores and gap penalties (and possibly other options) from a scoring file (see help below for file fomat description).": "从评分文件中读取替换分数和间隔惩罚（以及可能的其他选项）（见下面的文件格式描述帮助）。",
    "Set the score values for a match and mismatch": "将匹配和不匹配的得分值设置好",
    "Score value for a match": "匹配的得分值",
    "Score value for a mismatch": "不匹配的分数值",
    "Set the score penalties for opening and extending a gap": "设置打开和延伸间隙的分数惩罚",
    "Gap opening penalty": "间隔开放惩罚",
    "Gap extension penalty": "间隙延伸惩罚",
    "Treat each N in the input sequences as an ambiguous nucleotide": "将输入序列中的每个N视为一个模糊的核苷酸",
    "Treat each of the IUPAC-IUB ambiguity codes (B, D, H, K, M, R, S, V, W, and Y, as well as N) in the input sequences as a completely ambiguous nucleotide.": "将输入序列中的每个IUPAC-IUB模糊代码（B、D、H、K、M、R、S、V、W和Y，以及N）视为完全模糊的核苷酸。",
    "Seeding": "播种",
    "Select seed type": "选择种子类型",
    "Select seed pattern": "选择种子模式",
    "Set seed length": "设置种子长度",
    "Specify an arbitrary pattern": "指定一个任意模式",
    "Allow transitions": "允许转换",
    "Filter seeds": "过滤种子",
    "Number of transitions": "转换数",
    "Number of matches": "匹配数",
    "HSPs (Gap-free extension)": "高得分片段（无间隙延伸）",
    "Perform gap-free extension of seeds to HSPs": "执行无间隔扩展种子到HSPs",
    "Skip the gap-free extension stage": "跳过无间隔延伸阶段",
    "Select HSP finding method": "选择HSP查找方法",
    "Find HSPs using the exact match extension method with the given length threshold": "使用给定的长度阈值，通过精确匹配扩展方法查找HSP（高得分片段）",
    "Specify number of mismatches": "指定不匹配的数量",
    "Specify length threshold": "指定长度阈值",
    "Find HSPs using the x-drop extension method with this threshold": "使用x-drop扩展方法找到HSPs（高得分片段），并使用此阈值。",
    "Set the score threshold for the x-drop extension method": "设置 x-drop 延伸方法的分数阈值",
    "Set an adaptive score threshold for the x-drop extension method": "为x-drop扩展方法设置一个自适应分数阈值",
    "Adjust for entropy when qualifying HSPs in the x-drop extension method": "在x-drop扩展方法中，对高得分区段（HSPs）进行筛选时，考虑熵的影响。",
    "Adjust for entropy when qualifying HSPs in the x-drop extension method, and report (to stderr) any HSPs that are discarded as a result.": "在x-drop扩展方法中，考虑熵对HSPs（高得分对）的鉴定进行调整，并报告（到标准错误）因此而丢弃的任何HSPs。",
    "Don't adjust for entropy when qualifying HSPs": "在鉴定高得分片段（HSPs）时，不要考虑熵的影响。",
    "Chaining": "连锁",
    "Perform chaining of HSPs with no penalties": "执行无惩罚的HSPs（高得分片段）连接",
    "Perform chaining with penalties": "执行带有惩罚的链式操作",
    "Penalty for diagonal in dynamic programming matrix": "动态规划矩阵中对角线上的惩罚",
    "Penalty for anti-diagonal in dynamic programming matrix": "动态规划矩阵中对角线惩罚",
    "Gapped extension": "带隙延伸",
    "Perform gapped extension of HSPs": "执行HSPs的带间隔扩展",
    "Skip the gapped extension stage": "跳过间隙延伸阶段",
    "Set the threshold for terminating gapped extension": "设置终止带间隔延伸的阈值",
    "If y-drop extension encounters the end of the sequence, extend the alignment to the end of the sequence rather than trimming it back to the location giving the maximum score": "如果y-drop扩展遇到序列的末端，将比对扩展到序列的末端，而不是将其修剪回给出最大分数的位置",
    "Set the threshold for gapped extension": "设置间隙延伸的阈值",
    "Revert to handling bounding alignments the way they were handled in BLASTZ.": "恢复处理边界对齐的方式，就像在BLASTZ中处理的那样。",
    "Filtering": "过滤",
    "Filter alignments by percent identity": "根据相似度百分比过滤比对结果",
    "Minimum identity": "最小相似性",
    "Maximum identity (optional)": "最大相似性（可选）",
    "Filter alignments by continuity": "过滤连续性对齐",
    "Minimum continuity": "最小连续性",
    "Maximum continuity (optional)": "最大连续性（可选）",
    "Filter alignments by coverage": "过滤对齐结果以获取覆盖度信息",
    "Minimum coverage": "最小覆盖度",
    "Maximum coverage (optional)": "最大覆盖度（可选）",
    "Filter alignments by how many bases match": "根据匹配的碱基数量过滤比对结果",
    "Filter alignments by how many bases match expressed as percentage": "根据匹配碱基的百分比过滤比对结果",
    "Filter alignments by the number of mismatches": "根据不匹配数过滤比对结果",
    "Filter alignments by the number of gaps": "根据间隔数过滤比对结果",
    "Filter alignments by the number of gap columns": "根据间隔列的数量过滤比对结果",
    "Do not output a trivial self-alignment block if the target and query sequences are identical": "如果目标序列和查询序列相同，不要输出一个平凡的自我比对块",
    "Interpolation": "插值",
    "Perform additional alignment between the gapped alignment blocks, using (presumably) more sensitive alignment parameters": "在有间隙的对齐块之间执行额外的对齐，使用（很可能）更敏感的对齐参数",
    "Output": "输出",
    "Specify the output format": "指定输出格式",
    "Select a BAM flavor to output": "选择一个BAM格式输出",
    "Select which fields to include": "选择要包含的字段",
    "Seleat MAF flavor": "选择 MAF 风味",
    "Create a dotplot representation of alignments?": "创建一个点图表示对齐。",
    "Control file for inference": "推断控制文件",
    "Will you select a reference genome from your history or use a built-in index?": "您将从历史记录中选择一个参考基因组，还是使用内置索引？",
    "Choose whether to use Default options for building indices or to Set your own": "选择是否使用默认选项来构建索引，或者设置您自己的选项",
    "Choose to use automatic or specified behavior for some parameters (-a)": "选择对某些参数使用自动或指定的行为（-a）",
    "Whether or not to use a packed representation for DNA strings (--packed)": "是否使用DNA字符串的打包表示（--packed）",
    "Maximum number of suffixes allowed in a block (--bmax)": "一个块中允许的最大后缀数（--bmax）",
    "Maximum number of suffixes allowed in a block as a fraction of the length of the reference (--bmaxdivn)": "在参考序列长度的分数中允许的后缀最大数量（--bmaxdivn）",
    "The period for the difference-cover sample (--dcv)": "差集覆盖样本周期（--dcv）",
    "Whether or not to disable the use of the difference-cover sample (--nodc)": "是否禁用差异覆盖样本的使用（--nodc）",
    "Whether or not to build the part of the reference index used only in paired-end alignment (-r)": "是否构建仅在配对端对齐中使用的参考索引部分（-r）",
    "The indexer will mark every 2^n Burrows-Wheeler rows with their corresponding location on the genome (-o)": "索引器将在基因组上标记每2^n个Burrows-Wheeler行及其对应的位置（-o）",
    "The size of the ftab lookup table used to calculate an initial Burrows-Wheeler range with respect to the first n characters of the query (-t)": "用于计算查询的前n个字符的初始Burrows-Wheeler范围的ftab查找表的大小（-t）",
    "Whether or not to convert Ns in the reference sequence to As (--ntoa)": "是否将参考序列中的N转换为A（--ntoa）",
    "Endianness to use when serializing integers to the index file (--big/--little)": "在将整数序列化到索引文件时使用的字节顺序（--big/--little）",
    "Seed for the pseudorandom number generator (--seed)": "伪随机数生成器的种子（--seed）",
    "Bowtie settings to use": "Bowtie 设置使用",
    "Skip the first n reads (-s)": "跳过前 n 个读取（-s）",
    "Only align the first n reads (-u)": "只对前n个读取进行比对（-u）",
    "Trim n bases from high-quality (left) end of each read before alignment (-5)": "在比对之前，从每个读段的高质量（左侧）端修剪n个碱基（-5）",
    "Trim n bases from low-quality (right) end of each read before alignment (-3)": "在比对之前，从每个读段的低质量（右侧）端修剪n个碱基（-3）",
    "Alignment mode": "对齐模式",
    "Maximum number of mismatches permitted in the seed (-n)": "种子中允许的最大不匹配数（-n）",
    "Maximum permitted total of quality values at all mismatched read positions (-e)": "允许所有不匹配的读取位置的质量值总和的最大值（-e）",
    "Seed length (-l)": "种子长度（-l）",
    "Whether or not to round to the nearest 10 and saturating at 30 (--nomaqround)": "是否四舍五入到最近的10并限制在30（--nomaqround）",
    "Maximum number of mismatches (-v)": "最大不匹配数（-v）",
    "Choose whether or not to attempt to align against the forward reference strand (--nofw)": "选择是否尝试对正向参考链进行比对（--nofw）",
    "Choose whether or not to attempt to align against the reverse-complement reference strand (--norc)": "选择是否尝试对反向互补的参考链进行比对（--norc）",
    "Whether or not to make Bowtie guarantee that reported singleton alignments are 'best' in terms of stratum and in terms of the quality values at the mismatched positions (--best)": "是否要求Bowtie确保报告的单例比对是“最佳”的，无论是在层级上还是在不匹配位置的质量值上（--best）",
    "Whether or not to try as hard as possible to find valid alignments when they exist (-y)": "是否尽可能努力地寻找有效对齐（-y）",
    "Maximum number of backtracks permitted when aligning a read (--maxbts)": "在对齐读取时允许的最大回溯数（--maxbts）",
    "Whether or not to report only those alignments that fall in the best stratum if many valid alignments exist and are reportable (--strata)": "是否仅报告那些在最佳层级中存在的对齐，如果存在多个有效且可报告的对齐（--strata）",
    "Whether or not to report all valid alignments per read (-a)": "是否报告每个读取的所有有效对齐（-a）",
    "Report up to n valid alignments per read (-k)": "报告每个读取最多n个有效对齐（-k）",
    "Suppress all alignments for a read if more than n reportable alignments exist (-m)": "如果一个读取有超过n个可报告的比对，则抑制该读取的所有比对（-m）",
    "Write all reads with a number of valid alignments exceeding the limit set with the -m option to a file (--max)": "将所有有效比对数量超过使用 -m 选项设置的限制的读取写入到一个文件中（--max）",
    "Write all reads that could not be aligned to a file (--un)": "将所有无法与文件对齐的读取写入文件（--un）",
    "Override the offrate of the index to n (-o)": "覆盖索引的偏移率到 n（-o）",
    "Seed for pseudo-random number generator (--seed)": "伪随机数生成器的种子（--seed）",
    "Forward FASTQ file": "正向FASTQ文件",
    "Reverse FASTQ file": "反转 FASTQ 文件",
    "Maximum insert size for valid paired-end alignments (-X)": "有效配对末端对齐的最大插入大小（-X）",
    "The upstream/downstream mate orientation for valid paired-end alignment against the forward reference strand (--fr/--rf/--ff)": "对于有效的配对末端比对，相对于正向参考链，配对末端的方向是上游/下游（--fr/--rf/--ff）",
    "Skip the first n pairs (-s)": "跳过前n对（-s）",
    "Only align the first n pairs (-u)": "仅对前n对序列进行比对（-u）",
    "Minimum insert size for valid paired-end alignments (-I)": "有效的配对末端比对的最小插入大小（-I）",
    "Maximum number of attempts Bowtie will make to match an alignment for one mate with an alignment for the opposite mate (--pairtries)": "Bowtie 尝试为一对配对序列中的一个序列匹配对齐时，最多尝试的次数（--pairtries）",
    "Whether or not to report all valid alignments per pair (-a)": "是否报告每对的所有有效对齐（-a）",
    "Report up to n valid alignments per pair (-k)": "报告每对最多n个有效对齐（-k）",
    "Suppress all alignments for a pair if more than n reportable alignments exist (-m)": "如果一对中有超过n个可报告的比对，则抑制所有比对（-m）",
    "Save the bowtie mapping statistics to the history": "将bowtie比对统计信息保存到历史记录中",
    "Suppress the header in the output SAM file (--sam-nohead)": "在输出的SAM文件中不包含头部信息（--sam-nohead）",
    "Use the following dataset as the reference sequence": "使用以下数据集作为参考序列",
    "Select a profile of preset options": "选择一组预设选项的配置文件",
    "Indexing options": "索引选项",
    "Use homopolymer-compressed k-mer ?": "使用同聚物压缩的k-mer？",
    "K-mer size": "K-mer大小",
    "Minimizer window size": "最小化窗口大小",
    "Split index for every N input gigabases": "将每N个输入吉基对的索引拆分",
    "Mapping options": "映射选项",
    "Retain at most INT secondary alignments": "保留最多INT个次要比对",
    "Max fragment length for PE alignment": "对于成对末端（PE）比对的最大片段长度",
    "Filter out top FLOAT fraction of repetitive minimizers": "过滤掉重复最小子的顶部 FLOAT 分数",
    "Specify an interval of k-mer occurrences": "指定一个k-mer出现次数的区间",
    "K-mer ocurrence lower limit": "K-mer 出现次数下限",
    "K-mer ocurrence upper limit": "K-mer出现上限",
    "Force minimap2 to always use k-mers occuring this many times or fewer": "强制 minimap2 始终使用出现次数不超过该值的 k-mers",
    "Discard a query minimizer": "丢弃一个查询最小化器",
    "Stop chain enlongation if there are no minimizers in INT-bp": "如果INT-bp内没有最小化器，则停止链的延伸。",
    "Bandwidth used in chaining and DP-based alignment": "在链式比较和基于动态规划的比对中使用的带宽",
    "Minimal number of minimizers on a chain": "链上最小最小数",
    "Minimal chaining score (matching bases minus log gap penalty)": "最小链式得分（匹配碱基减去对数间隙惩罚）",
    "Maximum seed skips during chaining": "最大种子跳跃数在链化过程中",
    "Maximum number of partial chains checked during chaining": "在链式处理过程中检查的最大部分链数量",
    "Skip self and dual mappings (for the all-vs-all mode)": "跳过自身和双映射（对于全部对全部模式）",
    "Min secondary-to-primary score ratio": "次要与主要得分比率最小",
    "Fine control the removal of redundant hits": "精细控制去除冗余命中",
    "Customize spliced alignment mode?": "自定义剪接对齐模式？",
    "Maximum allowed gap on the reference": "参考序列上允许的最大间隙",
    "Cost of non-canonical (non-GT-AG) splicing": "非典型剪接（非GT-AG）的代价",
    "How to find GT-AG": "如何找到GT-AG",
    "Assume conserved flanking region of splice sites?": "假设剪接位点的保守侧翼区域？",
    "Use previously annotated splice sites to guide the alignment?": "使用先前注释的剪接位点来指导比对？",
    "Dataset with annotated genes or introns": "带有注释基因或内含子的数据集",
    "Annotated splice site bonus": "注释剪接位点奖励",
    "Gap open penalties for deletions": "删除的间隔开放惩罚",
    "Gap open penalties for insertions": "插入的间隔开放惩罚",
    "Gap extension penalty for extending an insertion; if left empty uses the value specified for Gap extension penalties above": "插入扩展的间隙延伸惩罚；如果留空，则使用上面指定的间隙延伸惩罚值",
    "Z-drop threshold for truncating an alignment": "Z-drop 阈值用于截断比对",
    "Z-drop threshold for reverse-complementing the query": "Z-drop阈值用于反转补全查询",
    "Minimal peak DP alignment score": "最小峰值DP（动态规划）比对分数",
    "Filter seeds towards the ends of chains before performing base-level alignment?": "在执行碱基水平比对之前，是否要过滤掉链末端的种子？",
    "Set advanced output options": "设置高级输出选项",
    "Select an output format": "选择一个输出格式",
    "Don't output base quality": "不输出碱基质量",
    "Write CIGAR with >65535 ops to the CG tag": "将CIGAR写入CG标签，操作数大于65535",
    "Minibatch size for mapping (in megabyte)": "映射的Minibatch大小（以兆字节为单位）",
    "Output cs tag?": "输出cs标签？",
    "Generate CIGAR": "生成CIGAR",
    "Write =/X CIGAR operators": "写入 =/X CIGAR 操作符",
    "Use soft clipping for supplementary alignments ?": "使用软剪切来处理补充对齐吗？",
    "Custom or built-in reference genome": "自定义或内置参考基因组",
    "Reference genome with annotation": "参考基因组和注释",
    "Type of single-cell RNA-seq": "单细胞RNA测序类型",
    "RNA-Seq Cell Barcode Whitelist": "RNA-Seq 细胞条形码白名单",
    "Configure Chemistry Options": "配置化学选项",
    "Cell Barcode Start Base": "细胞条形码起始碱基",
    "Cell Barcode Length": "细胞条形码长度",
    "UMI Start Base": "UMI起始碱基",
    "UMI Length": "UMI长度",
    "Barcode and cDNA on the same mate\\?": "条形码和cDNA在同一配对上？",
    "Barcode sequence is a part of": "条形码序列是一部分",
    "Number of bases to clip (=CB+UMI+adapter)": "要剪切的碱基数量（=CB+UMI+适配器）",
    "Barcode Size is same size of the Read": "条形码大小与读取大小相同",
    "UMI deduplication (collapsing) algorithm": "UMI去重（合并）算法",
    "Type of UMI filtering": "UMI 过滤类型",
    "Matching the Cell Barcodes to the WhiteList": "匹配细胞条形码到白名单",
    "Cell barcode whitelist information": "细胞条形码白名单信息",
    "Start anchor base for cell barcode": "起始锚定基础，用于细胞条形码",
    "0-based position of the CB start with respect to the anchor base": "相对于锚定碱基的CB起始位置的0-based位置",
    "End anchor base for cell barcode": "细胞条形码的末端锚定碱基",
    "0-based position of the CB end with respect to the anchor base": "以锚定碱基为参考的CB端的0-based位置",
    "Start anchor base for UMI": "开始锚定基础用于UMI（唯一分子标识符）",
    "0-based position of the UMI start with respect to the anchor base": "与锚定碱基相比，UMI起始位置的0-based位置",
    "End anchor base for UMI": "UMI的末端锚定碱基",
    "0-based position of the UMI end with respect to the anchor base": "相对于锚定碱基的UMI末端的0-based位置",
    "File containing cell IDs of the samples. One ID per line in order of samples in the above collection.": "包含样本细胞ID的文件。每个ID占一行，按上述集合中样本的顺序排列。",
    "Strandedness of Library": "图书馆的链特异性",
    "Collect UMI counts for these genomic features": "收集这些基因组特征的UMI计数",
    "Cell filtering type and parameters": "细胞过滤类型和参数",
    "Number of expected cells": "预期细胞数",
    "Robust maximum percentile for UMI count": "稳健的最大百分位数用于UMI计数",
    "Maximum to minimum ratio for UMI count": "UMI计数的最大与最小比率",
    "Output raw matrix in addition to filtered one": "输出原始矩阵以及过滤后的矩阵",
    "Minimum number of barcodes (used as partition parameter for ambient estimation)": "最小条形码数量（用作环境估计的分区参数）",
    "Maximum number of barcodes (used as partition parameter for ambient estimation)": "最大条形码数量（用作环境估计的分区参数）",
    "Consider at least these many UMIs per barcode after initial cell calling": "考虑在初始细胞调用后，每个条形码至少有这么多UMIs",
    "Minimum UMI:median ratio after initial cell calling": "最小UMI：初始细胞鉴定后的中位数比率",
    "Number of extra barcodes after initial cell calling": "初始细胞筛选后额外的条形码数量",
    "Maximum adjusted p-value for determining a barcode as non-ambient": "确定一个条形码为非环境因素的最大调整后的p值",
    "Number of log likelihood simulations": "对数似然模拟次数",
    "Number of top cells to report sorted by UMI count": "按UMI计数报告的顶级细胞数量",
    "Field 3 in the Genes output.": "基因输出中的第三个字段。",
    "Read alignment tags to include in the BAM output": "将 BAM 输出中包含的读取对齐标签",
    "Output global gene count": "输出全局基因计数",
    "Output unmapped reads in the BAM": "输出未比对的BAM中的读取结果",
    "Single-end or paired-end reads": "单端或双端测序读段",
    "RNA-Seq FASTQ/FASTA file": "RNA测序 快速质量/快速文件",
    "RNA-Seq FASTQ/FASTA file, forward reads": "RNA测序 FASTQ/FASTA 文件，正向读取",
    "RNA-Seq FASTQ/FASTA file, reverse reads": "RNA-Seq 快速测序 FASTQ/FASTA 文件，反向读取",
    "RNA-Seq FASTQ/FASTA paired reads": "RNA-Seq 配对读取的 FASTQ/FASTA",
    "Reference genome with or without an annotation": "参考基因组，带或不带注释",
    "Build index with or without known splice junctions annotation": "使用或不使用已知剪接连接注释构建索引",
    "Use 2-pass mapping for more sensitive novel splice junction discovery": "使用两轮映射以更敏感地发现新的剪接连接点",
    "Number of reads to map in the first pass": "第一次映射要映射的读取数",
    "Pregenerated splice junctions datasets of your samples": "预生成的样本剪接连接数据集",
    "Report chimeric alignments?": "报告嵌合对齐？",
    "BAM output format specification": "BAM输出格式规范",
    "HI tag values should be": "HI标签值应该是",
    "Output filter criteria": "输出筛选标准",
    "Exclude the following records from the BAM output": "从BAM输出中排除以下记录",
    "Would you like to set additional output filters?": "您想设置额外的输出过滤器吗？",
    "Would you like to keep only reads that contain junctions that passed filtering?": "您是否只想保留包含已通过过滤的连接点的读取？",
    "Score range below the maximum score for multimapping alignments": "多映射对齐的最大分数范围",
    "Maximum number of alignments to output a read's alignment results, plus 1": "输出一个读段的比对结果的最大比对数，加1",
    "Maximum number of mismatches to output an alignment, plus 1": "输出一个比对时允许的最大不匹配数，加1",
    "Maximum ratio of mismatches to mapped length": "最大错配率与映射长度比",
    "Maximum ratio of mismatches to read length": "最大错配比例与读取长度",
    "Minimum alignment score": "最小比对分数",
    "Minimum alignment score, normalized to read length": "最小比对得分，标准化到读取长度",
    "Minimum number of matched bases": "最少匹配碱基数量",
    "Minimum number of matched bases, normalized to read length": "匹配碱基的最小数量，标准化到读取长度",
    "Maximum number of multimapping alignments to output for a read": "对于一个读取，输出的最大多映射对齐数",
    "Calculation method for TLEN": "TLEN的计算方法",
    "Algorithmic settings": "算法设置",
    "Configure seed, alignment and limits options": "配置种子、对齐和限制选项",
    "Seed parameters": "种子参数",
    "Search start point through the read": "搜索读取的起始点",
    "Search start point through the read, normalized to read length": "通过读取，将搜索起始点标准化为读取长度",
    "Maximum length of seeds": "种子的最大长度",
    "Maximum number of mappings to use a piece in stitching": "最大映射数，用于拼接中的片段",
    "Maximum number of seeds per read": "每个读取的最大种子数",
    "Maximum number of seeds per window": "每个窗口的最大种子数",
    "Maximum number of one seed loci per window": "每个窗口中一个种子位点的最大数量",
    "Alignment parameters": "对齐参数",
    "Minimum intron size": "最小内含子大小",
    "Maximum intron size": "最大内含子大小",
    "Maximum gap between two mates": "两个配对之间的最大间隙",
    "Minimum overhang for spliced alignments": "拼接对齐的最小悬垂",
    "Maximum number of mismatches for stitching of the splice junctions (-1: no limit)": "拼接剪接位点时允许的最大不匹配数（-1：无限制）",
    "Non-canonical motifs": "非典型基序",
    "GT/AG and CT/AC motif": "GT/AG和CT/AC基序",
    "GC/AG and CT/GC motif": "GC/AG和CT/GC基序",
    "AT/AC and GT/AT motif": "AT/AC和GT/AT基序",
    "Minimum overhang for annotated spliced alignments": "注释剪接对齐的最小悬挂长度",
    "Minimum mapped length for a read mate that is spliced": "外接读段的最小比对长度",
    "Minimum mapped length for a read mate that is spliced, normalized to mate length": "对于被剪接的读段配对，映射长度的最小值，已根据配对长度进行标准化",
    "Maximum number of windows per read": "每个读取的最大窗口数",
    "Maximum number of transcripts per window": "每个窗口中转录本的最大数量",
    "Maximum number of different alignments per read to consider": "每个读取要考虑的不同对齐的最大数量",
    "type of read ends alignment": "读取端点对齐类型",
    "minimum number of overlap bases to trigger mates merging and realignment": "触发配对合并和重新对齐的最小重叠碱基数",
    "maximum proportion of mismatched bases in the overlap area": "重叠区域中不匹配碱基的最大比例",
    "Chimeric alignment parameters": "嵌合体比对参数",
    "Minimum length of chimeric segment": "嵌合片段的最小长度",
    "Minimum total (summed) score of chimeric segments": "嵌合段的最小总（累加）分数",
    "Maximum difference of chimeric score from read length": "最大与读长相关的嵌合分数差异",
    "Minimum difference between the best chimeric score and the next one": "最佳嵌合分数与次佳分数之间的最小差异",
    "Penalty for a non-GT/AG chimeric junction": "非GT/AG嵌合连接处的惩罚",
    "Minimum overhang for a chimeric junction": "嵌合连接的最小悬垂长度",
    "Maximum gap in the read sequence between chimeric segments": "在读取序列中嵌合段之间的最大间隙",
    "Discard chimeric alignments with Ns in the genome sequence around the chimeric junction": "丢弃在基因组序列中嵌合连接处周围有N的嵌合对齐",
    "Maximum number of multi-alignments for the main chimeric segment.": "主嵌合体段的最大多重比对数量。",
    "Maximum number of chimeric multi-alignments": "最大数量的嵌合多重比对",
    "Score range for multi-mapping chimeras": "多映射嵌合体的得分范围",
    "Performance tweaks / Troubleshooting": "性能调整 / 故障排除",
    "Number of genome bins for coordinate-sorting": "基因组箱数用于坐标排序",
    "Maximum number of loci anchors are allowed to map to": "允许的最大定位点锚点映射数量",
    "Submit a Bismark mapping report": "提交一个Bismark比对报告",
    "Additional reports to include in the HTML page (optional)": "要在HTML页面中包含的额外报告（可选）",
    "Submit the corresponding Bismark deduplication report": "提交相应的 Bismark 去重报告",
    "Submit the corresponding Bismark splitting report": "提交相应的Bismark拆分报告",
    "Submit the corresponding Bismark M-bias report": "提交相应的 Bismark M-偏差报告",
    "Submit the corresponding Bismark nucleotide report": "提交相应的Bismark核苷酸报告",
    "Create a separate logfile, otherwise logs are added to the dataset info.": "创建一个独立的日志文件，否则日志会被添加到数据集信息中。",
    "Select Reference Genome (bismark indexes)": "选择参考基因组（Bismark索引）",
    "Select Reference Genome (fasta)": "选择参考基因组（fasta格式）",
    "FASTQ/FASTA file": "FASTQ/FASTA 文件",
    "Paired End Pairs": "配对末端对",
    "Mate pair 1": "配对1",
    "Mate pair 2": "配对伙伴 2",
    "Minimum insert size for valid paired-end alignments": "有效配对末端比对的最小插入大小",
    "Maximum insert size for valid paired-end alignments": "有效的配对末端对齐的最大插入大小",
    "Sort BAM file by chromosomal position (not compatibile with methylation extractor)": "按染色体位置对BAM文件进行排序（不兼容甲基化提取器）",
    "Bismark settings to use": "Bismark 设置使用",
    "Number of mismatches to be allowed in a seed alignment during multiseed alignment": "在多种子比对过程中，允许种子比对中出现的错误数量",
    "Length of the seed substrings to align during multiseed alignment": "在多种子比对过程中，用于比对的种子子字符串的长度",
    "How many consecutive seed extension attempts can fail before Bowtie 2 moves on": "在Bowtie 2继续之前，可以有多少连续的种子扩展尝试失败",
    "Maximum number of times Bowtie 2 will re-seed reads with repetitive seeds": "Bowtie 2重新播种具有重复种子的读取的最大次数",
    "Only aligns the first N reads or read pairs from the input": "仅对输入中的前N个读取或读取对进行比对",
    "Skip (i.e. do not align) the first N reads or read pairs from the input": "跳过（即不比对）输入中的前N个读取或读取对",
    "Set a function governing the minimum alignment score needed for an alignment to be considered `valid` (i.e. good enough to report)": "设置一个函数，用于确定对齐得分的最小值，以便将对齐视为“有效”的（即足够好以报告）。",
    "Disable looking for discordant alignments if it cannot find any concordant alignments (only for paired-end reads)": "如果找不到一致的比对，则禁用寻找不一致比对（仅适用于成对末端读取）",
    "Disable Bowtie 2's behaviour to try to find alignments for the individual mates (only for paired-end reads": "禁用Bowtie 2尝试为单个配对（仅针对成对末端读取）寻找对齐的行为",
    "Write ambiguous reads to an extra output file": "将模糊的读取写入一个额外的输出文件",
    "Write all reads that could not be aligned to a file": "将无法比对到文件中的所有读取写入文件",
    "Write the bismark output and summary information to an extra file": "将Bismark的输出和摘要信息写入到一个额外的文件中",
    "Offer all report files concatenated in one file (Mapping Report)": "将所有报告文件合并到一个文件中（映射报告）",
    "The sequencing library was constructed in a non strand-specific manner, alignments to all four bisulfite strands will be reported.": "测序文库以非链特异性方式构建，将报告对所有四个亚硫酸盐链的比对。",
    "Treat input data as PBAT-Seq libraries": "将输入数据视为PBAT-Seq文库",
    "Submit the resulting bam/sam file from Bismark bisulfite mapper": "提交 Bismark 亚硫酸盐测序映射器生成的 bam/sam 文件",
    "SAM/BAM file from Bismark bisulfite mapper": "Bismark 亚硫酸盐测序映射器生成的 SAM/BAM 文件",
    "Ignore the first N bp from the 5’ end of single-end read when processing the methylation call string.": "在处理甲基化调用字符串时，忽略单端读取5'端的前N个碱基对。",
    "Ignore the last N bp from the 3' end of single-end read when processing the methylation call string.": "在处理甲基化调用字符串时，忽略单端读取3'端的最后N个碱基对。",
    "Ignore the first N bp from the 5’ end of Read 1 when processing the methylation call string.": "在处理甲基化调用字符串时，忽略来自读取1的5'端的前N个碱基对。",
    "Ignore the last N bp from the 3' end of Read 1 when processing the methylation call string.": "在处理甲基化调用字符串时，忽略来自读取1的3'端的最后N个碱基对。",
    "Ignore the first N bp from the 5' end of Read 2 of paired-end sequencing results": "忽略配对端测序结果中第二条读段（Read 2）5'端的前N个碱基对",
    "Ignore the last N bp from the 3' end of Read 2 of paired-end sequencing results": "忽略配对端测序结果中第二条读段（Read 2）的3'端的N个碱基",
    "This option avoids scoring overlapping methylation calls twice, in case of overlapping read one and read two": "这个选项避免了对重叠的甲基化调用进行两次评分，以防在一个读取中重叠的读取一和读取二",
    "Short methylation summary output (Splitting Report)": "短甲基化总结输出（拆分报告）",
    "Methylation proportion report for each possible position in the read (Mbias Report)": "甲基化比例报告，用于报告读取中每个可能位置的甲基化比例（Mbias报告）",
    "Genome-wide methylation report for all cytosines in the genome (Cytosine Report)": "全基因组甲基化报告，用于报告基因组中所有胞嘧啶的甲基化情况（胞嘧啶报告）",
    "Reports Cpg Context only": "仅报告 CpG 上下文",
    "Generate Bismark coverage report": "生成 Bismark 覆盖报告",
    "Advanced output settings": "高级输出设置",
    "Merge all four possible strand-specific methylation info into context-dependent output files": "将所有四种可能的链特异性甲基化信息合并到依赖于上下文的输出文件中",
    "Merge all non-CpG contexts into one file": "将所有非CpG上下文合并到一个文件中",
    "Single or paired reads": "单端或配对读取",
    "Collection of paired reads": "成对序列的集合",
    "Forward strand": "正向链",
    "Reverse strand": "反向链",
    "Input sequences": "输入序列",
    "Kmer Size": "Kmer大小",
    "p-value": "P值",
    "Exhaustive Mode": "穷尽模式",
    "Decontaminate": "去污染",
    "Do not allow insertions in assembly": "不允许在组装中插入",
    "Consensus sequence has 'n' instead of gaps": "共识序列中使用'n'代替空缺",
    "Output assembly matrix": "输出组装矩阵",
    "Print all best mappings": "打印所有最佳映射",
    "Minimum phred score": "最小Phred分数",
    "Cut a constant number of nucleotides from the 5 prime": "从5'端切割固定数量的核苷酸",
    "Only count kmers": "仅计算k-mers",
    "Minimum percent identity": "最小百分比一致性",
    "Force end to end mapping": "强制端到端映射",
    "Minimum depth at base": "最小深度在基座",
    "Score for match": "匹配得分",
    "Penalty for mismatch": "错配的惩罚",
    "Penalty for gap opening": "间隔开放的惩罚",
    "Penalty for gap extension": "间隔惩罚",
    "Reward for pairing reads": "配对读取的奖励",
    "Set CGE penalties and rewards": "设置 CGE 惩罚和奖励",
    "Use output from earlier STAR run or let STAR Fusion control running STAR": "使用之前STAR运行的输出，或让STAR Fusion控制运行STAR",
    "Chimeric junction file from STAR (with STAR-Fusion settings)": "来自STAR（带有STAR-Fusion设置）的嵌合连接文件",
    "left.fq file": "左侧的.fq文件",
    "right.fq file (actually optional, but highly recommended)": "正确的.fq文件（实际上是可选的，但强烈推荐）",
    "Source for sequence to search": "要搜索的序列来源",
    "Genome to search": "要搜索的基因组",
    "Select the reference genome (FASTA file)": "选择参考基因组（FASTA文件）",
    "Gene model (gff3,gtf) file for splice junctions and fusion gene detection": "用于剪接连接和融合基因检测的基因模型（gff3、gtf）文件",
    "Result of BLAST+-blastn of the reference fasta sequence with itself": "BLAST+-blastn 对参考fasta序列与自身进行比对的结果",
    "Settings to use": "使用设置",
    "minimum number of junction-spanning reads required.": "所需的最小连接跨越读取数。",
    "minimum fusion support = (#junction_reads + #spanning_frags)": "最小融合支持度 = (#连接点读段数 + #跨越片段数)",
    "maximum number of partners allowed for a given fusion": "允许给定融合的最大伴侣数",
    "minimum of 3 junction reads required if breakpoint lacks involvement of only reference junctions": "如果断点只涉及参考连接点，则至少需要3个连接点读取",
    "10% of the dominant isoform junction support": "10%的主要异构体连接支持",
    "non-ref junctions within 5 are merged into single calls": "非参考连接点在5个碱基范围内合并为单个调用",
    "E-value threshold for blast searches": "BLAST搜索的E值阈值",
    "Is this single or paired library": "这是单个还是配对的文库",
    "Interleaved FASTQ file": "交错的FASTQ文件",
    "Select reference genome": "选择参考基因组",
    "Do you want to use presets?": "您想要使用预设吗？",
    "Do you want to tweak input options?": "您想要调整输入选项吗？",
    "Skip (i.e. do not align) the first that many reads or pairs in the input": "跳过（即不进行比对）输入中的前那么多的读取或配对。",
    "Align the first that many reads or read pairs from the input (after the -s/--skip reads or pairs have been skipped), then stop": "对输入的前那么多的读取或配对进行比对（在跳过-s/--skip读取或配对之后），然后停止。",
    "Trim that many bases from 5' (left) end of each read before alignment": "在比对之前，从每个读取序列的5'（左侧）端修剪这么多碱基",
    "Trim that many bases from 3' (right) end of each read before alignment": "在比对之前，从每个读取序列的3'端（右侧）修剪这么多碱基。",
    "Quality values are represented in the read input file as space-separated ASCII integers, e.g., 40 40 30 40..., rather than ASCII characters, e.g., II?I.... Integers are treated as being on the Phred quality scale unless --solexa-quals is also specified": "质量值在读取输入文件中以空格分隔的ASCII整数值表示，例如，40 40 30 40...，而不是ASCII字符，例如，II?I....。整数被视为Phred质量标度，除非同时指定了--solexa-quals",
    "Do you want to tweak alignment options?": "您想要调整对齐选项吗？",
    "Set the number of mismatches to be allowed in a seed alignment during multiseed alignment (see `Multiseed alignment` section of help below)": "在多种子比对过程中设置允许的种子比对中的不匹配数目（参见下面帮助中的`多种子比对`部分）",
    "Sets the length of the seed substrings to align during multiseed alignment (see `Multiseed alignment` section of help below)": "设置在多种子比对期间要对齐的种子子字符串的长度（请参阅下面的“多种子比对”部分的帮助）。",
    "Set a function governing the interval between seed substrings to use during multiseed alignment (see `Multiseed alignment` section of help below). Also see description of this option below in the help section": "设置一个函数，该函数控制在使用多种子对齐（参见下面帮助中的`多种子对齐`部分）期间使用的种子子字符串之间的间隔。还可以在帮助部分的下面看到此选项的描述。",
    "Set a function governing the maximum number of ambiguous characters (usually `N`s and/or `.`s) allowed in a read as a function of read length": "设置一个函数，该函数控制允许在一个读取中出现的模糊字符（通常是`N`和/或`.`）的最大数量，作为读取长度的函数。",
    "Pad dynamic programming problems by that many columns on either side to allow gaps": "在动态规划问题的两侧各填充相应数量的列，以允许存在间隙。",
    "Disallow gaps within that many positions of the beginning or end of the read": "不允许在读取的开始或结束的这么多位置内出现间隙",
    "When calculating a mismatch penalty, always consider the quality value at the mismatched position to be the highest possible, regardless of the actual value": "在计算不匹配的惩罚时，始终要考虑不匹配位置的质量值是最高的可能值，而不考虑实际的值",
    "Do not attempt to align unpaired reads to the forward (Watson) reference strand": "不要尝试将未配对的读取对齐到正向（Watson）参考链",
    "Do not attempt to align unpaired reads to the reverse (Crick) reference strand": "不要尝试将未配对的读取序列与反向（克里克）参考链进行比对",
    "Prevent searching for 1-mismatch end-to-end alignments before using the multiseed heuristic (see `Multiseed alignment` section of help below)": "在使用多种子启发式方法（见下面帮助中的“多种子对齐”部分）之前，防止搜索1个错配的端到端对齐",
    "Select between `--local` and `--end-to-end` alignment modes": "选择 `--local` 和 `--end-to-end` 对齐模式",
    "Do you want to tweak scoring options?": "您想要调整打分选项吗？",
    "Set the match bonus": "设置匹配奖励",
    "Set the maximum (`MX`) and minimum (`MN`) mismatch penalties, both integers": "设置最大（`MX`）和最小（`MN`）错配惩罚，两者都是整数",
    "Sets penalty for positions where the read, reference, or both, contain an ambiguous character such as `N`": "设置当读取、参考或两者都包含一个模糊字符（例如 'N'）时的惩罚",
    "Set the read gap opening penalty": "设置读取间隙开放的惩罚值",
    "Set the read gap extension penalty": "设置读取间隙延伸的惩罚分数",
    "Set the reference gap opening penalty": "设置参考间隙开放惩罚",
    "Set the reference gap extension penalty": "设置参考间隙延伸惩罚",
    "Do you want to use -a or -k options": "您想要使用 -a 或 -k 选项吗？",
    "Searches for at most that many distinct, valid alignments for each read": "为每个读取搜索最多那么多不同的、有效的对齐",
    "Do you want to tweak effort options?": "你想要调整努力选项吗？",
    "Attempt that many consecutive seed extension attempts to `fail` before Bowtie 2 moves on, using the alignments found so far": "尝试那么多连续的种子扩展尝试去“失败”，然后Bowtie 2根据到目前为止找到的比对继续进行。",
    "Set the maximum number of times Bowtie 2 will `re-seed` reads with repetitive seeds": "设置Bowtie 2在遇到重复种子时重新生成种子的最大次数。",
    "Do you want to tweak Other Options?": "您想要调整其他选项吗？",
    "Do you want to tweak SAM/BAM Options?": "您想要调整SAM/BAM选项吗？",
    "Would you like the output to be a SAM file": "您希望输出为SAM文件吗？",
    "Suppress SAM records for reads that failed to align": "抑制那些未能比对的读取的SAM记录",
    "Suppress SEQ and QUAL strings for secondary alignments": "抑制次要比对的SEQ和QUAL字符串",
    "Suppress standard behavior of truncating readname at first whitespace at the expense of generating non-standard SAM": "抑制在第一个空白处截断读名字的标准行为，以生成非标准SAM文件为代价。",
    "Use '='/'X', instead of 'M,' to specify matches/mismatches in SAM record.": "在SAM记录中，使用'='或'X'代替'M'来指定匹配/不匹配。",
    " Exclude soft-clipped bases when reporting TLEN": "在报告 TLEN 时排除软剪切碱基",
    "Reorder output to reflect order of the input file": "将输出重新排序以反映输入文件的顺序",
    "Save the bowtie2 mapping statistics to the history": "将bowtie2比对统计信息保存到历史记录中",
    "Is this library paired-end?": "这个文库是成对末端的吗？",
    "Reads in FASTQ/FASTA files": "读取FASTQ/FASTA文件",
    "Reads from first strand": "来自第一链的读取",
    "Reads from second strand": "来自第二链的读取",
    "Maximum size of the inserts (paired end)": "插入片段（成对末端）的最大大小",
    "Detect split/spliced reads": "检测到的剪接/拼接读段",
    "Min coverage for spliced transcripts": "拼接转录本的最小覆盖度",
    "Min length of a spliced fragment": "拼接片段的最小长度",
    "Report spliced alignment with score greater than this scale times the score": "报告剪接对齐，如果得分大于这个比例乘以得分",
    "max evalue for splits": "分裂的最大E值",
    "Bisulfite mapping": "亚硫酸盐测序映射",
    "Minimum size of queries": "查询的最小尺寸",
    "Maximum number of alignments that will be reported": "最大报告的比对数",
    "Min percentage of matches per read in semi-global alignment": "每个读段在半全局比对中的匹配最小百分比",
    "Hits to report?": "报告命中数？",
    "add 5' adapter": "添加5'接头",
    "add 3' adapter": "添加3'接头",
    "clipping accuracy": "剪切精度",
    "Clip polyA tail": "剪切 polyA 尾部",
    "Autoclip unknown 3prime adapter": "自动剪切未知的3'端适配器",
    "Enable hard clipping": "启用硬裁剪",
    "Sorts the output by chromsome and position": "按照染色体和位置对输出进行排序",
    "search seeds initially with n differences": "使用 n 个差异最初搜索种子",
    "search seeds with jump size": "搜索种子，跳跃大小",
    "max evalue": "最大E值",
    "maximum width of a suffix array interval, i.e. a query seed will be omitted if it matches more than n times": "后缀数组区间的最大宽度，即如果查询种子匹配超过n次，则该查询种子将被省略",
    "Output unmatched reads": "输出不匹配的读段",
    "reference sequence": "参考序列",
    "reads": "读取",
    "Pre-set parameters (-x)": "预设参数（-x）",
    "Pre-set parameters to increase sensitivity for different sequencing technologies. (-x)": "预设参数以提高不同测序技术的灵敏度。(-x)",
    "For split alignment, take alignment with smallest coordinate as primary": "对于拆分比对，将坐标最小的比对作为主比对",
    "Don't lower MAPQ for split alignment": "不要降低拆分比对的MAPQ值",
    "BAM sorting mode": "BAM排序模式",
    "Is this single-end or paired-end data?": "这是单端数据还是双端数据？",
    "RNA-Seq FASTQ file": "RNA测序FASTQ文件",
    "RNA-Seq FASTQ file, forward reads": "RNA-Seq FASTQ 文件，正向读取",
    "RNA-Seq FASTQ file, reverse reads": "RNA-Seq FASTQ文件，反向读取",
    "RNA-Seq FASTQ paired reads": "RNA-Seq 配对读取的 FASTQ",
    "TopHat settings to use": "TopHat 使用的设置",
    "Max realign edit distance": "最大重新比对编辑距离",
    "Max edit distance": "最大编辑距离",
    "Library Type": "文库类型",
    "Final read mismatches": "最终读取不匹配",
    "Use bowtie -n mode": "使用 bowtie 的 -n 模式",
    "Anchor length (at least 3)": "锚定长度（至少3）",
    "Maximum number of mismatches that can appear in the anchor region of spliced alignment": "在剪接对齐的锚定区域中可以出现的最大不匹配数",
    "The minimum intron length": "最小内含子长度",
    "The maximum intron length": "最大内含子长度",
    "Maximum number of alignments to be allowed": "最大允许的比对数量",
    "Minimum intron length that may be found during split-segment (default) search": "在分割-片段（默认）搜索中可能找到的最小内含子长度",
    "Maximum intron length that may be found during split-segment (default) search": "在进行分裂段（默认）搜索时可能发现的最大内含子长度",
    "Number of mismatches allowed in each segment alignment for reads mapped independently": "在独立比对每个片段时，允许的每个片段比对中的不匹配数目",
    "Minimum length of read segments": "最小读取片段长度",
    "Output unmapped reads": "输出未比对的读段",
    "Use coverage-based search for junctions": "使用基于覆盖度的搜索来寻找接头",
    "Minimum intron length that may be found during coverage search": "在覆盖度搜索期间可能发现的最小内含子长度",
    "Maximum intron length that may be found during coverage search": "在覆盖度搜索期间可能发现的最大内含子长度",
    "Use Microexon Search": "使用Microexon搜索",
    "Do Fusion Search": "执行融合搜索",
    "Anchor Length": "锚点长度",
    "Minimum Distance": "最小距离",
    "Read Mismatches": "读取不匹配",
    "Multireads": "多重读取",
    "Multipairs": "多重配对",
    "--fusion-ignore-chromosomes; Ignore some chromosomes such as chrM when detecting fusion break points": "--fusion-ignore-chromosomes; 在检测融合断点时忽略一些染色体，例如 chrM",
    "Set Bowtie2 settings": "设置 Bowtie2 参数",
    "Use Preset options": "使用预设选项",
    "Preset option": "预设选项",
    "Specify read group?": "指定读取组？",
    "Read group identiﬁer (ID). Each @RG line must have a unique ID. The value of ID is used in the RG tags of alignment records. Must be unique among all read groups in header section.": "读取组标识符（ID）。每个@RG行必须有一个唯一的ID。ID的值用于对齐记录的RG标签中。在头部部分的所有读取组中必须是唯一的。",
    "Library name (LB)": "库名称（LB）",
    "Platform/technology used to produce the reads (PL)": "用于生成读取的平台/技术（PL）",
    "Sample (SM)": "样本（SM）",
    "How many rows get marked during annotation of some or all of the Burrows-Wheeler rows (-o)": "在对一些或全部的Burrows-Wheeler行进行注释（-o）期间，有多少行被标记？",
    "The size of the lookup table used to calculate an initial Burrows-Wheeler range with respect to the first n characters of the query (-t)": "用于计算查询的第一个n个字符的初始Burrows-Wheeler范围的查找表的大小（-t）",
    "Number of first bases of the reference sequence to index (--cutoff)": "参考序列要索引的起始碱基数量（--cutoff）",
    "Maximum permitted total of quality values at mismatched read positions (-e)": "允许的最大错误匹配位置的质量值总和（-e）",
    "Number of mismatches for SOAP-like alignment policy (-v)": "SOAP-like 对齐策略中不匹配的数量（-v）",
    "SNP penalty (ratio of SNPs per base in the subject genome) (--snpphred)": "SNP惩罚（目标基因组中每个碱基的SNP比例）(--snpphred)",
    "Ratio of SNPs per base (estimated ratio for colorspace alignments) (--snpfrac)": "每个碱基的SNP比率（对颜色空间比对的估计比率）（--snpfrac）",
    "Keep the extreme-ends nucleotides and qualities rather than trimming them (--col-keepends)": "保留两端的核苷酸和质量分数，而不是修剪它们（--col-keepends）",
    "Choose whether or not to attempt to align the forward reference strand (--nofw)": "选择是否尝试对正向参考链进行比对（--nofw）",
    "Choose whether or not to align against the reverse-complement reference strand (--norc)": "选择是否对反向互补的参考链进行比对（--norc）",
    "Report up to n valid arguments per pair (-k)": "报告每对中有效参数的数量（-k）",
    "Suppress the header in the output SAM file": "在输出的SAM文件中不包含头部信息",
    "High frequency k-mers dataset": "高频k-mers数据集",
    "Turn off SV-aware mode": "关闭结构变异感知模式",
    "Force winnowmap to always use k-mers occuring this many times or fewer": "强制 winnowmap 始终使用出现次数或更少的 k-mers",
    "how to find GT-AG": "如何找到GT-AG",
    "minimal peak DP alignment score": "最小峰值DP（动态规划）比对分数",
    "don't output base quality": "不要输出碱基质量",
    "write CIGAR with >65535 ops to the CG tag": "将大于65535个操作的CIGAR写入CG标签",
    "minibatch size for mapping (in megabyte)": "映射的微批次大小（以兆字节为单位）",
    "write =/X CIGAR operators": "写入 =/X CIGAR 操作符",
    "use soft clipping for supplementary alignments ?": "使用软剪切来处理补充对齐吗？",
    "Input reads": "输入序列",
    "Save this reference to your history": "将此参考保存到您的历史记录中",
    "Reference sequences": "参考序列",
    "Source for the background": "背景来源",
    "Select sequences from data table": "从数据表中选择序列",
    "Select dataset with sequences": "选择包含序列的数据集",
    "Discard sequences that are insufficiently identical to the reference": "丢弃与参考序列相似度不足的序列",
    "Alphabet to use for alignment": "用于比对的字母表",
    "Parametrize using score matrix": "使用评分矩阵进行参数化",
    "Output discarded sequences to a separate dataset": "将丢弃的序列输出到一个单独的数据集",
    "Also try to align against reverse complement of reference": "也尝试将参考序列的反向互补序列进行比对。",
    "Include reference as first sequence in aligned BAM": "在对齐的BAM文件中将参考序列作为第一个序列包含进去",
    "Process this number of input bases in each batch regardless of nThreads (for reproducibility)": "在每个批次中处理这么多输入碱基，而不考虑nThreads（为了可重复性）",
    "Maximum indel length": "最大插入或删除长度",
    "Do not allow indels longer than the specified maximum indel length?": "不允许插入或缺失（indels）的长度超过指定的最大插入或缺失长度？",
    "Look this far for read-end deletions with anchors shorter than K using brute force": "使用蛮力查找锚点长度小于K的末端缺失",
    "Approximate minimum alignment identity to look for": "近似最小比对一致性以寻找",
    "Minimum number of seed hits required for candidate sites": "候选位点所需的最小种子命中数",
    "Use local (rather than global) alignments?": "使用局部（而非全局）比对？",
    "Allow only perfect mappings?": "仅允许完全映射？",
    "Allow only perfect and semiperfect (perfect except for N's in the reference) mappings?": "仅允许完全匹配和半完全匹配（除了参考序列中的N之外完全匹配）的映射？",
    "Specify behavior on ambiguously-mapped reads with multiple top-scoring mapping locations": "指定对于具有多个最高得分映射位置的不明确映射读取的行为",
    "Map paired reads to the same strand?": "将配对的读取映射到同一链上？",
    "Forbid pairing of reads without correct strand orientation?": "禁止配对没有正确链方向的读段？",
    "Mark the read with the lower mapping quality as unmapped if a read pair is mapped with an inappropriate insert size or orientation?": "如果一个测序对的插入大小或方向不合适，将映射质量较低的读标记为未映射。",
    "Treat unpaired reads as unmapped?": "将未配对的读取视为未映射？",
    "Reverse complement both reads prior to mapping (for LMP outward-facing libraries)?": "在映射之前对两个读取进行反向互补（对于LMP向外的库）？",
    "Reverse complement read2 prior to mapping?": "在映射之前对读段2进行反向互补？",
    "Maximum allowed distance between paired reads": "配对读取之间允许的最大距离",
    "Maximum average insert size for trying to rescue paired reads": "尝试恢复配对读取时的最大平均插入大小",
    "Maximum mismatches allowed in a rescued read": "允许在被拯救的读取中的最大不匹配数",
    "Initial average distance between paired reads": "配对读段之间的初始平均距离",
    "Run in deterministic mode?": "在确定性模式下运行？",
    "Restrict alignment band to this fraction of read length": "将比对带宽限制为读取长度的这个分数",
    "Set the bandwidth directly to this fraction of read length": "将带宽直接设置为读取长度的这个分数",
    "Maximum number of alignments per read to output": "每个读取输出的最大比对数",
    "Discard low-information kmers that occur often?": "丢弃经常出现的低信息量kmer吗？",
    "Fraction of kmers to ignore": "忽略的k-mer分数",
    "Use a greedy algorithm to discard the least-useful kmers on a per-read basis?": "使用贪婪算法在每个读取的基础上丢弃最不有用的kmers？",
    "Potential mapping sites must have at least this many consecutive exact matches": "潜在的映射位点必须至少有这么多连续的完全匹配",
    "Undo trimming after mapping?": "映射后撤销修剪？",
    "Set to a positive number 1-50 to generate fake quality strings for fasta input reads": "将正数1-50设置为生成fasta输入读取的假质量字符串。",
    "Keep going, rather than crashing, if a read has out-of-range quality values?": "如果一个读取具有超出范围的质量值，继续进行而不是崩溃？",
    "Do not map reads with average quality below this value": "不要映射平均质量低于这个值的读取",
    "Calculate maq from this many initial bases": "计算从这么多初始碱基的maq",
    "Output secondary alignments?": "输出次级比对？",
    "Maximum number of total alignments to output per read": "每个读取输出的最大总比对数",
    "Output only secondary alignments with score of at least this fraction of primary": "仅输出得分至少为主得分这一分数的次要比对",
    "Only output secondary alignments for ambiguously-mapped reads?": "只输出对于映射不明确的读取的次要比对？",
    "Generate cigar strings more quickly?": "更快地生成CIGAR字符串？",
    "Truncate read and ref names at the first whitespace, assuming that the remainder is a comment or description?": "截断读取和参考名称，在第一个空格处，假设其余部分是注释或描述？",
    "Output statistics in machine-friendly (key=value) format?": "输出统计信息为机器友好的（键=值）格式？",
    "Output the total number of unmapped reads and bases?": "输出未比对的读取数量和碱基总数。",
    "Rename reads based on their mapped insert size?": "根据它们的映射插入大小重命名读取？",
    "Post-filtering options": "后过滤选项",
    "Specify exact minimum identity allowed for alignments to be output": "指定允许输出比对的确切最小一致性",
    "Ban alignments with more than this many substitutions": "禁止具有超过这个数量替换的比对",
    "Ban alignments with more than this many insertions": "禁止对齐中插入超过此数量的序列",
    "Ban alignments with more than this many deletions": "禁止包含超过此数量删除的比对",
    "Ban alignments with more than this many indels": "禁止对齐中包含超过此数量的插入或缺失",
    "Ban alignments with more than this many edits": "禁止对齐，如果编辑次数超过这个数量",
    "Ban alignments with an insertion longer than this": "禁止比这个更长的插入序列的比对",
    "Ban alignments with a deletion longer than this": "禁止对齐中包含超过此长度的删除",
    "Ban alignments with more than this many ns (includes nocall, noref, and off scaffold ends)": "禁止对齐包含超过这个数量的“ns”（包括未调用、无参考和支架末端之外）的对齐结果。",
    "Set alignment mode": "设置比对模式",
    "Minimum mapped read length in basepairs": "最小映射读长（以碱基对为单位）",
    "Output at maximum N alignments for each read, 0 means no maximum": "对于每个读取，输出最多N个比对，0表示没有最大限制",
    "Diploid mode": "二倍体模式",
    "VCF file with diploid SNVs and InDels": "双倍体单核苷酸变异和插入/缺失变异的 VCF 文件",
    "Transform alignments, spliced junctions and quantifications back to the reference coordinates": "将比对结果、剪接连接位点和定量数据转换回参考坐标",
    "Select SAM/BAM dataset or dataset collection": "选择 SAM/BAM 数据集或数据集集合",
    "Load picard dictionary file from": "从加载Picard字典文件",
    "Use dictionary from the list": "使用列表中的字典",
    "Use the following dataset to create dictionary": "使用以下数据集创建字典",
    "Select coordinate dataset or dataset collection": "选择坐标数据集或数据集集合",
    "Load reference genome from": "从...加载参考基因组",
    "The level(s) at which to accumulate metrics": "要累积指标的级别（们）",
    "Input file contains Bisulphite sequenced reads": "输入文件包含亚硫酸盐测序的读取",
    "Adapter": "适配器",
    "Use this adaptor sequence": "使用这个适配器序列",
    "Larger paired end reads and inter-chromosomal pairs considered chimeric": "较大的配对末端读取和染色体间配对被认为是嵌合体的",
    "Use the folloing dataset as the reference sequence": "使用以下数据集作为参考序列",
    "Calculate the base distribution over aligned reads only": "计算仅对齐的读取中的碱基分布",
    "Calculate the base distribution over PF (passing filtering) reads only": "计算仅在通过过滤（PF）的读取上的碱基分布",
    "The size of windows on the genome that are used to bin reads": "基因组上用于分bin读取的窗口大小",
    "For summary metrics, exclude GC windows that include less than this fraction of the genome": "对于汇总指标，排除包含少于该分数的基因组的GC窗口",
    "An interval list file that contains the locations of the baits used": "一个包含用于捕获的间隔列表文件",
    "An interval list file that contains the locations of the targets": "一个区间列表文件，包含了目标的位置",
    "Minimum mapping quality for a read to contribute coverage": "用于贡献覆盖的最小映射质量",
    "Minimum base quality for a base to contribute coverage": "用于贡献覆盖度的最小碱基质量",
    "Treat bases with coverage exceeding this value as if they had coverage at this value": "将覆盖度超过这个值的碱基当作覆盖度为这个值来处理",
    "Clip overlapping reads": "剪切重叠的读取",
    "Generate mean, sd and plots by trimming the data down to MEDIAN + DEVIATIONS*MEDIAN_ABSOLUTE_DEVIATION": "生成均值、标准差和图表，通过将数据修剪到中位数 + 偏差 * 中位数绝对偏差",
    "Explicitly sets the Histogram width, overriding automatic truncation of Histogram tail": "显式设置直方图宽度，覆盖直方图尾部的自动截断",
    "When generating the Histogram, discard any data categories (out of FR, TANDEM, RF) that have fewer than this percentage of overall reads": "在生成直方图时，丢弃任何（FR、TANDEM、RF）数据类别，如果这些类别的读段占总体读段的百分比少于这个百分比值。",
    "Load gene annotation from": "从...加载基因注释",
    "Gene annotation (GTF/GFF3)": "基因注释（GTF/GFF3）",
    "Gene annotations in refFlat form": "基因注释以refFlat格式",
    "Location of rRNA sequences in genome, in interval_list format": "基因组中rRNA序列的位置，以区间列表格式",
    "What is the RNA-seq library strand specificity": "RNA-seq文库的链特异性是什么",
    "When calculating coverage based values use only use transcripts of this length or greater": "在计算基于覆盖度的值时，只使用这个长度或更大的转录本",
    "Sequences to ignore": "序列忽略",
    "Ignore reads matching this sequence": "忽略匹配此序列的读取",
    "This percentage of the length of a fragment must overlap one of the ribosomal intervals for a read or read pair to be considered rRNA.": "片段长度的这一百分比必须与核糖体区间之一重叠，才能将读取或读取对视为rRNA。",
    "SAM/BAM Input file": "SAM/BAM 输入文件",
    "How many nucleotides (context) on each side to consider": "每个侧边考虑多少个核苷酸（上下文）",
    "Contexts to print in the detailed tables, separated by comma (avoid spaces)": "在详细表格中打印的上下文，用逗号分隔（避免空格）",
    "Minimum base quality score": "最小碱基质量分数",
    "Maximum insert size": "最大插入片段大小",
    "Minimum insert size": "最小插入片段大小",
    "Assume SAM/BAM is sorted": "假设SAM/BAM文件已排序",
    "Include duplicates": "包括重复项",
    "Include unpaired reads": "包含未配对的读取",
    "Probability (between 0 and 1) that any given read will be kept": "任意给定的读取保留的概率（介于0和1之间）",
    "Random seed value": "随机种子值",
    "The minimum number of bases at the starts of reads that must be identical for reads to be grouped together for duplicate detection": "用于重复检测的读取序列起始处必须相同的最小碱基数目",
    "The maximum rate of differences between two reads to call them identical": "两个读取之间的最大差异率，以将它们称为相同",
    "The minimum mean quality of the bases in a read pair for the read to be analyzed": "在分析读段对时，读段中碱基的最小平均质量",
    "Do not process self-similar groups that are this many times over the mean expected group size": "不要处理与平均预期组大小相比有这么多倍的自相似组",
    "What is your input data": "你的输入数据是什么",
    "Input fastq file for single end data": "输入单端数据的fastq文件",
    "Input fastq file for the first read in paired end data": "输入配对端数据中第一读的fastq文件",
    "Input fastq file for the second read of paired end data": "输入配对端数据的第二读的fastq文件",
    "FASTQ paired dataset collection": "FASTQ配对数据集集合",
    "Read group name": "读取组名",
    "Sample name": "样本名称",
    "The library name": "库名称",
    "The platform unit (often run_barcode.lane)": "平台单元（通常为 run_barcode.lane）",
    "The platform type (e.g. illumina, 454)": "平台类型（例如 illumina, 454）",
    "The sequencing center from which the data originated": "数据来源的测序中心",
    "Predicted median insert size, to insert into the read group header": "预测的中位数插入大小，插入到读取组头信息中",
    "Comment to include in the output dataset's header": "输出数据集标题中要包含的注释",
    "Optional description information": "可选描述信息",
    "Run date": "运行日期",
    "Minimum quality allowed in the input fastq": "输入fastq中允许的最小质量",
    "If true and this is an unpaired fastq any occurance of '/1' will be removed from the end of a read name": "如果为真，并且这是一个未配对的fastq文件，那么将从读取名称的末尾移除任何出现的'/1'。",
    "Allow (and ignore) empty lines": "允许（并忽略）空行",
    "Select filtering type": "选择过滤类型",
    "Dataset containing read names that will be INCLUDED in the output": "包含将包含在输出中的读取名称的数据集",
    "Dataset containing read names that will be EXCLUDED in the output": "数据集包含将在输出中排除的读取名称",
    "Adds the mate CIGAR tag (MC) if true, does not if false": "如果为真，则添加配对CIGAR标签（MC），如果为假，则不添加",
    "Assume that the input file is QUERYNAME sorted": "假设输入文件是按QUERYNAME排序的",
    "Regular expression that can be used in unusual situations to parse non-standard read names in the incoming SAM/BAM dataset": "在非标准读取名称的传入SAM/BAM数据集中，可用于解析的正则表达式",
    "Barcode Tag": "条形码标签",
    "The minimum distance to buffer records to account for clipping on the 5' end of the records": "缓冲记录以考虑记录5'端截断的最小距离",
    "Skip record pairs with no mate cigar and include them in the output": "跳过没有配对的CIGAR记录对，并将它们包含在输出中",
    "SAM/BAM dataset or dataset collection": "SAM/BAM 数据集或数据集集合",
    "If set to true, calculate mean quality over aligned reads only": "如果设置为true，则仅计算对齐读取的平均质量",
    "If set to true calculate mean quality over reads passing quality filter": "如果设置为真，则计算通过质量过滤的读取的平均质量",
    "Selected unaligned SAM or BAM with original reads": "选择未对齐的SAM或BAM文件，包含原始读取数据",
    "What type of aligned data do you have?": "您拥有哪种类型的比对数据？",
    "Aligned SAM or BAM dataset": "对齐的SAM或BAM数据集",
    "SAM or BAM dataset(s) with alignment data": "SAM或BAM数据集（们），包含比对数据",
    "Aligned SAM or BAM dataset for Read 1": "对齐的SAM或BAM数据集，用于读取1",
    "SAM or BAM dataset(s) with alignment data for Read1": "SAM或BAM数据集，包含Read1的比对数据",
    "Aligned SAM or BAM dataset for Read 2": "对齐的SAM或BAM数据集，用于第二读段",
    "SAM or BAM dataset(s) with alignment data for Read2": "带有Read2比对数据的SAM或BAM数据集",
    "Aligned SAM or BAM dataset for Single Reads": "对齐的SAM或BAM数据集，用于单读序列",
    "Whether to clip adapters where identified": "是否在识别到的地方剪切接头",
    "Whether the data is from bisulfite sequencing (used when caculating the NM tag)": "是否数据来自亚硫酸盐测序（用于计算NM标签时）",
    "Whether to output only aligned reads": "是否只输出对齐的读取",
    "The maximum number of insertions or deletions permitted for an alignment to be included": "允许包含对齐的最大插入或删除数",
    "Retain the following alignment attribute": "保留以下对齐属性",
    "Reserved alignment attributes (tags starting with X, Y, or Z) that should be brought over from the alignment data when merging": "保留的比对属性（以X、Y或Z开头的标签），在合并时应该从比对数据中带入",
    "Remove the following alignment attribute": "移除以下对齐属性",
    "Attributes from the alignment record that should be removed when merging.": "在合并时应该移除的来自比对记录的属性。",
    "The number of bases trimmed from the beginning of read 1 prior to alignment": "在比对之前从读取1的开头修剪的碱基数量",
    "The number of bases trimmed from the beginning of read 2 prior to alignment": "在比对之前从读取2的开头修剪的碱基数量",
    "The expected orientation of proper read pairs": "正确的配对读取的预期方向",
    "Use the aligner's idea of what a proper pair is rather than computing in this program": "使用比对工具对正确配对的定义，而不是在这个程序中计算",
    "Strategy for selecting primary alignment when the aligner has provided more than one alignment for a pair or fragments": "选择主要比对的策略，当比对器为一对或片段提供了多个比对时",
    "For paired reads, soft clip the 3' end of each read if necessary so that it does not extend past the 5' end of its mate": "对于配对的读取，如果必要，将每个读取的3'端进行软裁剪，以确保它不会延伸到其配对读取的5'端之外。",
    "If false, do not write secondary alignments to output": "如果为假，则不将次级比对写入输出",
    "Merge the sequence dictionaries of the datasets being merged": "合并正在合并的数据集的序列字典",
    "FASTA dataset or dataset collection": "FASTA 数据集或数据集集合",
    "The line length to be used for the output fasta file": "输出的fasta文件使用的行长",
    "Truncate sequence names at first whitespace": "在第一个空白处截断序列名称",
    "If set to true, include quality for no-call bases in the distribution": "如果设置为真，则在分布中包括未调用碱基的质量",
    "If true, then allows only a partial overlap of the BAM contigs with the new reference sequence contigs": "如果为真，则只允许BAM序列与新参考序列的部分重叠",
    "If true, then permits mapping from a read contig to a new reference contig with the same name but a different length": "如果为真，则允许将读取的序列片段映射到具有相同名称但长度不同的新参考序列片段上",
    "Select SAM/BAM dataset or dataset collection (header recepient dataset)": "选择 SAM/BAM 数据集或数据集集合（头信息接收数据集）",
    "SAM/BAM dataset from which Header will be read (header source dataset)": "从该SAM/BAM数据集中读取头部信息（头部来源数据集）",
    "Restore original qualities from the OQ field to the QUAL field if available": "如果可用，从OQ字段恢复原始质量到QUAL字段",
    "The maximum number of records to examine to determine if we can exit early and not output, given that there are a no original base qualities (if we are to restore) and mate cigars exist. Set to 0 to never skip the dataset": "我们最多检查的记录数，以确定是否可以提前退出而不输出，假设我们有原始的碱基质量（如果我们要恢复）和配对的cigar存在。设置为0以从不跳过数据集。",
    "Remove duplicate read flags from all reads": "去除所有读取中的重复读取标志",
    "Remove all alignment information from the file": "从文件中移除所有比对信息",
    "Clear attribute": "清除属性",
    "When removing alignment information, specify optional tags to remove (e.g., XM)": "在移除比对信息时，指定要移除的可选标签（例如，XM）",
    "Discard reads in order to produce a consistent output BAM": "丢弃读段以产生一致的输出BAM",
    "If SANITIZE=true and higher than MAX_DISCARD_FRACTION reads are discarded due to sanitization then the program will exit with an Exception instead of exiting cleanly": "如果 SANITIZE=true 并且由于净化而丢弃的读段数量超过 MAX_DISCARD_FRACTION，则程序将抛出异常而不是正常退出",
    "The sample alias to use in the reverted output file. This will override the existing sample alias in the file and is used only if all the read groups in the input file have the same sample alias": "在还原输出文件中使用的样本别名。这将覆盖文件中现有的样本别名，并且仅在输入文件中的所有读取组具有相同的样本别名时使用",
    "The library name to use in the reverted output file. This will override the existing sample alias in the file and is used only if all the read groups in the input file have the same sample alias": "在还原输出文件中使用的库名称。这将覆盖文件中现有的样本别名，并且仅在输入文件中的所有读取组具有相同的样本别名时使用。",
    "The sort order to create the reverted output file with": "创建反向输出文件的排序顺序",
    "Re-reverse bases and qualities of reads with negative strand flag set before writing them to fastq": "在将读取的序列写入fastq文件之前，重新反转带有负链标志设置的读取的碱基和质量分数。",
    "Include non-PF reads from the SAM/BAM dataset into the output FASTQ": "将 SAM/BAM 数据集中的非 PF（Proper Pair，正确配对）的读取包含在输出的 FASTQ 文件中。",
    "The attribute that stores the position at which the SAM/BAM record should be clipped": "存储SAM/BAM记录应该被剪切的位置的属性",
    "The action that should be taken with clipped reads: 'X' means the reads and qualities should be trimmed at the clipped position; 'N' means the bases should be changed to Ns in the clipped region; and any integer means that the base qualities should be set to that value in the clipped region": "剪切读取的操作：'X'表示在剪切位置处修剪读取和质量；'N'表示在剪切区域中将碱基更改为N；任何整数表示在剪切区域中将碱基质量设置为该值",
    "The number of bases to trim from the beginning of read 1": "从读1的开头修剪的碱基数量",
    "The maximum number of bases to write from read 1 after trimming": "从读1修剪后写入的最大碱基数目",
    "The number of bases to trim from the beginning of read 2": "从读取2的开头修剪的碱基数量",
    "The maximum number of bases to write from read 2 after trimming": "从读2修剪后写入的最大碱基数量",
    "If true, include non-primary alignments in the output": "如果为真，则在输出中包含非主要比对",
    "Sort order": "排序顺序",
    "Select output mode": "选择输出模式",
    "Select validation error types to ignore": "选择要忽略的验证错误类型",
    "The maximum number of lines output in verbose mode": "在详细模式下输出的最大行数",
    "If true, only report errors and ignore warnings": "如果为真，则只报告错误并忽略警告",
    "If true and input is a BAM file with an index file, also validates the index": "如果为真，并且输入是带有索引文件的BAM文件，还将验证索引",
    "Whether the SAM or BAM file consists of bisulfite sequenced reads": "SAM或BAM文件是否包含亚硫酸盐测序的读取",
    "Select quality encoding scheme": "选择质量编码方案",
    "Do you want to output a fastq file per read group (two fastq files per read group if the group is paired)": "您是否希望为每个读组输出一个fastq文件（如果读组是成对的，则每个读组输出两个fastq文件）？",
    "Will generate an interleaved fastq if paired, each line will have /1 or /2 to describe which end it came from": "如果配对，则会生成交错的fastq文件，每行都会有/1或/2来描述它来自哪个端",
    "SAM or BAM file to filter": "SAM或BAM文件筛选",
    "Header in output": "输出中包含标题",
    "Minimum MAPQ quality score": "最小MAPQ质量分数",
    "Filter on bitwise flag": "过滤位标志",
    "Only output alignments with all of these flag bits set": "仅输出设置了所有这些标志位的比对",
    "Skip alignments with any of these flag bits set": "跳过设置了这些标志位的任何比对",
    "Select alignments from Library": "选择来自库的比对",
    "Select alignments from Read Group": "从读取组选择比对",
    "Output alignments overlapping the regions in the BED FILE": "输出与BED文件中区域重叠的比对结果",
    "Use inverse selection": "使用反向选择",
    "Select regions (only used when the input is in BAM format)": "选择区域（仅在输入为BAM格式时使用）",
    "Select the output format": "选择输出格式",
    "BAM File to Convert": "BAM 文件转换",
    "Genetic intervals (in BED format)": "遗传区间（以BED格式）",
    "hard clip": "硬剪切",
    "only clip reads that match bed file strand annotation": "仅剪辑与bed文件链表注释匹配的读取",
    "clip both ends of reads (false = 5' only)": "剪切两端的读段（false = 仅5'端）",
    "don't write excluded reads to output (default = write all)": "不将排除的读段写入输出（默认 = 写入全部）",
    "Min Read length": "最小读取长度",
    "Tolerance": "容差",
    "BAM File": "BAM文件",
    "Is this paired-end or single end data": "这是成对末端数据还是单末端数据",
    "Treat as single-end": "视为单端",
    "Choose a pileup file to condense:": "选择一个pileup文件进行压缩：",
    "which contains:": "包含：",
    "Which base do you want to concatenate": "您想要连接哪个碱基",
    "Select column with sequence name": "选择包含序列名称的列",
    "Select column with base location": "选择带有碱基位置的列",
    "Select column with base to concatenate": "选择要连接的列",
    "Select column with coverage": "选择覆盖度列",
    "Do not report bases with coverage less than:": "不要报告覆盖度小于的碱基：",
    "Select dataset": "选择数据集",
    "which contains": "包含",
    "Select column with reference base": "选择包含参考基的列",
    "Select column with read bases": "选择包含读取碱基的列",
    "Select column with base qualities": "选择包含碱基质量的列",
    "Select coordinate column": "选择坐标列",
    "Do not consider read bases with quality lower than": "不考虑质量低于的读取碱基",
    "Do not report positions with coverage lower than": "不要报告覆盖度低于",
    "Only report variants?": "只报告变异？",
    "Convert coordinates to intervals?": "将坐标转换为区间？",
    "Print total number of differences?": "打印差异总数？",
    "Print quality and base string?": "打印质量和碱基字符串？",
    "Select dataset to convert": "选择要转换的数据集",
    "Print all?": "打印全部？",
    "Files to merge": "要合并的文件",
    "Merge all component bam file headers into the merged bam file": "将所有组件BAM文件的头部合并到合并后的BAM文件中",
    "Select the BAM file to generate the pileup file for": "选择BAM文件以生成pileup文件",
    "Whether or not to print the mapping quality as the last column": "是否在最后一列打印映射质量",
    "Whether or not to print only output pileup lines containing indels": "是否只打印包含插入或缺失的输出堆积行",
    "Where to cap mapping quality": "在哪里设置映射质量的上限",
    "Call consensus according to MAQ model?": "调用基于MAQ模型的共识？",
    "Theta parameter (error dependency coefficient) in the MAQ consensus calling model": "Theta 参数（错误依赖系数）在 MAQ 共识调用模型中",
    "Number of haplotypes in the sample": "样本中的单倍型数量",
    "Expected fraction of differences between a pair of haplotypes": "预期的单倍型对之间的差异分数",
    "Phred probability of an indel in sequencing/prep": "测序/准备中插入/缺失的Phred概率",
    "BAM File to report statistics of": "BAM文件报告统计信息",
    "BAM file(s)": "BAM文件（们）",
    "Set advanced options": "设置高级选项",
    "Set filter by flags": "设置过滤标志",
    "Require": "要求",
    "Exclude": "排除",
    "Select regions to call": "选择要调用的区域",
    "skip unlisted positions (chr pos) or regions": "跳过未列出的位置（染色体 位置）或区域",
    "region in which pileup is generated": "生成pileup的区域",
    "Select read groups to exclude": "选择要排除的测序组",
    "Text file": "文本文件",
    "Read groups": "读取组",
    "Disable read-pair overlap detection": "禁用配对读取重叠检测",
    "Do not discard anomalous read pairs": "不要丢弃异常的读段对",
    "Disable BAQ (per-Base Alignment Quality), see below": "禁用BAQ（每碱基对齐质量），详见下文",
    "Coefficient for downgrading mapping quality for reads containing excessive mismatches": "针对含有过多不匹配的读取，降低映射质量的系数",
    "max per-file depth; avoids excessive memory usage": "最大每个文件的深度；避免过度使用内存",
    "Recalculate BAQ on the fly": "实时重新计算BAQ",
    "Minimum mapping quality for an alignment to be used": "用于一个比对的最小映射质量",
    "Minimum base quality for a base to be considered": "考虑一个碱基的最小碱基质量",
    "quality is in the Illumina-1.3+ encoding": "质量是在Illumina-1.3+编码中",
    "Output base positions on reads": "输出读取的碱基位置",
    "Output mapping quality": "输出映射质量",
    "Output an extra column containing comma-separated read names. (--output-QNAME)": "输出一个额外的列，包含逗号分隔的读名字。(--output-QNAME)",
    "Output absolutely all positions": "输出所有位置",
    "Add tags to output": "在输出中添加标签",
    "Select BAM dataset to slice": "选择BAM数据集进行切片",
    "How do you want to slice your dataset?": "您想要如何切分您的数据集？",
    "Select references (chromosomes and contigs) you would like to restrict bam to": "选择您希望将BAM文件限制在哪些参考序列（染色体和基因组片段）上",
    "Regions": "区域",
    "Enter START coordinate (1-based)": "输入起始坐标（1-based） START：开始",
    "Enter END coordinate": "输入末端坐标",
    "SAM/BAM/CRAM data set": "SAM/BAM/CRAM 数据集",
    "What would you like to look at?": "您想要查看什么？",
    "What would you like to have reported?": "您希望报告什么？",
    "Include SAM header in the output?": "在输出中包含SAM头文件？",
    "Configure filters": "配置过滤器",
    "Filter by regions": "按区域过滤",
    "Filter by intervals in a bed file": "在bed文件中通过区间过滤",
    "Filter by readgroup": "根据读组过滤",
    "Filter by read group": "根据读取组过滤",
    "Filter by read groups in file": "根据文件中的读组进行过滤",
    "Filter by quality": "按质量过滤",
    "Filter by library": "按库过滤",
    "Filter by number of CIGAR bases consuming query sequence": "根据消耗查询序列的CIGAR碱基数量进行过滤",
    "Require that these flags are set": "要求设置这些标志",
    "Exclude reads with any of the following flags set": "排除设置了以下任意标志的读取",
    "Exclude reads with all of the following flags set": "排除所有设置了以下所有标志的读取",
    "Filter by tag": "按标签筛选",
    "Filter by readnames": "根据读名字过滤",
    "Configure subsampling": "配置子采样",
    "Subsample alignment": "子样本对齐",
    "Downsampling factor": "下采样因子",
    "Target # of reads": "目标读取数",
    "Produce extra dataset with dropped reads?": "生成一个丢弃了的读取数据集？",
    "Produce extra dataset with retained reads?": "生成包含保留下来的读取序列的额外数据集？",
    "Select source dataset": "选择源数据集",
    "Select target dataset": "选择目标数据集",
    "Keep the new header unmodified?": "保持新标题不变？",
    "Mapped reads input dataset": "映射后的读取输入数据集",
    "Reference genome regions to calculate mapping statistics for": "参考基因组区域，用于计算映射统计信息",
    "Dataset specifying regions": "数据集指定区域",
    "Invert regions": "反转区域",
    "Generate per-base coverage output": "生成每碱基覆盖度输出",
    "Skip duplicate reads": "跳过重复的读取",
    "Settings affecting specific plots": "影响特定图形的设置",
    "Number of bins to use in across-reference plots": "在参考基因组图上使用的箱数",
    "Draw chromosome limits": "绘制染色体界限",
    "Plot expected GC-content distribution of the following reference genome": "绘制以下参考基因组的预期GC含量分布图",
    "Homopolymer size": "同聚物大小",
    "Report samples": "报告样本",
    "BAM QC data to combine": "BAM质检数据合并",
    "Single-sample BAM QC data": "单样本BAM质量控制数据",
    "Name to use for this dataset": "为此数据集使用的名称",
    "Name of the group": "组名",
    "Data associated with this group": "与这个组相关联的数据",
    "BAM dataset(s) to filter": "BAM数据集（们）待过滤",
    "Select BAM manipulation": "选择BAM操作",
    "Choose the source for the reference list": "选择参考列表的来源",
    "Using reference file": "使用参考文件",
    "Print quality scores?": "打印质量分数？",
    "Do not print header": "不要打印标题",
    "Number of random alignments to grab": "随机对齐数",
    "Random number generator seed": "随机数生成器种子",
    "Keep duplicates marked": "保留重复标记",
    "Keep base qualities": "保持碱基质量",
    "Bed file": "BED文件",
    "5' to 3' Tag Shift (bp)": "5'至3'标签偏移（碱基对）",
    "Bin Size (bp)": "Bin 大小（碱基对）",
    "Set Tags to be equal?": "设置标签相等？",
    "Reads to examine": "要检查的序列",
    "Require proper PE?": "需要正确的成对末端（PE）？",
    "Combine strands?": "合并链？",
    "Composite smoothing window (bp)": "复合平滑窗口（碱基对）",
    "Output frequencies heatmap?": "输出频率热图？",
    "Alignment": "比对",
    "Remove duplicate reads": "去除重复的读段",
    "Mark supplementary reads of duplicates as duplicates": "将重复的补充读取标记为重复项",
    "Clear previous duplicate settings and tags.": "清除之前的重复设置和标签。",
    "Expected maximum read length of INT bases. (default 300, min=0)": "预期最大读取长度为INT个碱基。（默认为300，最小为0）",
    "Optical distance (if set, marks with dt tag, min=1)": "光学距离（如果设置，则用dt标签标记，最小值为1）",
    "Duplicate decision method for paired reads.": "配对读取的重复判定方法。",
    "Include quality check failed reads.": "包含质量检查失败的读段。",
    "Print basic statistics": "打印基本统计信息",
    "Reference FASTA file": "参考FASTA文件",
    "Alignments in BAM format": "BAM格式的比对结果",
    "Merge only reads overlapping the specified regions in the BED file": "仅合并覆盖BED文件中指定区域的读取",
    "Merge files in a region": "合并区域内的文件",
    "File to take @headers from": "从文件中获取头部信息",
    "Make @RG headers unique": "使 @RG 头信息唯一",
    "Make @PG headers unique": "使 @PG 头信息唯一",
    "random seed": "随机种子",
    "BAM or SAM file to convert": "BAM或SAM文件转换",
    "Default quality if none is given": "默认质量，如果没有给出的话",
    "Output quality in the OQ tag if available": "如果可用，在OQ标签中输出质量",
    "add Illumina Casava 1.8 format entry to header (eg 1:N:0:ATCACG)": "在头部添加Illumina Casava 1.8格式的条目（例如 1:N:0:ATCACG）",
    "outputs": "输出",
    "Copy RG/BC/QT tags to output header": "将 RG/BC/QT 标签复制到输出标题",
    "copy arbitrary tags to the FASTA header line": "将任意标签复制到FASTA头行",
    "Omit or append read numbers": "忽略或追加读取编号",
    "Write index read files": "写入索引读取文件",
    "write first index reads to extra data set": "将第一个索引的读取写入额外的数据集",
    "write second index reads to extra data set": "将第二个索引的读取写入到额外的数据集中",
    "Index format: How to parse barcode and quality tags": "索引格式：如何解析条形码和质量标签",
    "Barcode tag [default: BC]": "条形码标签 [默认: BC]",
    "Quality tag [default: QT]": "质量标签 [默认: QT]",
    "Select alignment": "选择比对",
    "Remove secondary and unmapped reads": "删除次要和未映射的读取",
    "Disable FR proper pair check": "禁用 FR 正确配对检查",
    "Add template cigar ct tag": "添加模板cigar ct标签",
    "Add ms (mate score) tags": "添加 ms（配偶分数）标签",
    "Set coverage distribution": "设置覆盖度分布",
    "Maximum coverage": "最大覆盖度",
    "Coverage step": "覆盖度步骤",
    "Exclude reads marked as duplicates": "排除标记为重复的读取",
    "Desired output files": "期望输出文件",
    "Filter by SAM flags": "根据SAM标志过滤",
    "Size of GC-depth bins": "GC深度区间的大小",
    "Limit to a specific read group name": "限制到特定的读组名称",
    "Minimum read length to generate statistics for": "生成统计信息所需的最小读取长度",
    "Report only the main part of inserts": "只报告插入序列的主要部分",
    "BWA trim parameter": "BWA修剪参数",
    "Suppress absence of insertions": "抑制缺失插入",
    "Remove overlaps of paired-end reads from coverage and base count computations": "从覆盖度和碱基计数计算中移除配对末端读取的重叠部分",
    "Only bases with coverage above this value will be included in the target percentage computation": "只有覆盖度高于这个值的碱基才会被包括在目标百分比计算中",
    "Output alignments overlapping the regions in the BED file": "输出与BED文件中区域重叠的比对结果",
    "Replace header in output file": "替换输出文件中的标题",
    "Minimum MAPQ": "最小 MAPQ",
    "Exclude deletions and ref skips": "排除删除和参考跳过",
    "Depth threshold": "深度阈值",
    "Additional Filter Options": "额外的过滤选项",
    "Require that these flags are set.": "要求设置这些标志。",
    "Exclude reads with any of the following flags set (the default set is UNMAP,SECONDARY,QCFAIL,DUP or 0x704).": "排除设置了以下任意标志的读取（默认设置是 UNMAP,SECONDARY,QCFAIL,DUP 或 0x704）。",
    "Only report depth in a specified region": "仅在指定区域报告深度",
    "Genomic intervals (in BED format)": "基因组区间（以BED格式表示）",
    "Output all positions": "输出所有位置",
    "Ignore reads shorter than": "忽略短于",
    "Limits the maximum number of considered alignments starting at any sequence position to INT. Note that, the reported depths values might be larger than INT. If 0, depth is set to the maximum integer value, effectively removing any depth limit. (default = 8000)": "将任何序列位置开始考虑的比对的最大数量限制为整数INT。请注意，报告的深度值可能会大于INT。如果为0，则深度设置为最大整数值，有效地移除任何深度限制。（默认值 = 8000）",
    "Only count reads with base quality greater than": "只计算碱基质量大于的读段",
    "Only count reads with mapping quality greater than": "仅计算映射质量大于的读段",
    "Include reads with deletions in depth computation.": "包含删除的读取在深度计算中。",
    "For the overlapping section of a read pair, count only the bases of a single read.": "对于配对序列的重叠部分，只计算单个序列的碱基。",
    "Print a file header to file.": "打印文件头部到文件。",
    "Primary sort key": "主排序键",
    "Alignment tag": "对齐标签",
    "Secondary sort key": "次级排序键",
    "Minhash collation": "Minhash 校对",
    "Are you pooling bam files?": "你正在合并BAM文件吗？",
    "BAM files to be pooled": "BAM文件要被合并",
    "Minimum read length": "最小读取长度",
    "Minimum base quality": "最小碱基质量",
    "Show specified region": "显示指定区域",
    "Histogram": "直方图",
    "Number of bins in histogram": "直方图的箱数",
    "Remove reads that are smaller than": "移除小于特定大小的读段",
    "Remove reads that are larger than": "移除大于的读段",
    "Keep only mapped reads": "只保留已映射的读段",
    "Keep only unmapped reads": "只保留未映射的读取序列",
    "Keep only properly paired reads": "只保留正确配对的读段",
    "Discard properly paired reads": "丢弃正确配对的读段",
    "Remove reads that match the mask": "移除与掩码匹配的读取序列",
    "Remove reads that have the same sequence": "移除具有相同序列的读段",
    "Remove reads that start at the same position": "移除起始位置相同的读段",
    "Remove reads with that many mismatches": "移除具有那么多不匹配的读取",
    "Remove secondary alignment reads": "去除次要比对的读取",
    "Remove reads that do not pass the quality control": "移除未通过质量控制的读段",
    "Remove reads that are marked as PCR dupicates": "删除被标记为PCR重复的读段",
    "Remove reads that are in any of the regions": "移除位于任何区域中的读取",
    "Remove reads that are NOT any of the regions": "删除不在任何区域中的读取序列",
    "Strand information from BED file is ignored": "从BED文件中忽略链信息",
    "Exclude reads NOT mapped to a reference": "排除未映射到参考序列的读段",
    "Exclude reads mapped to a particular reference": "排除映射到特定参考序列的读取",
    "Filter by maximum mismatch ratio": "按最大错配比率过滤",
    "Condition": "条件",
    "Select BAM property to filter on": "选择要过滤的BAM属性",
    "Filter on this alignment flag": "过滤这个比对标志",
    "Filter on this CIGAR string": "过滤这个CIGAR字符串",
    "Filter on insert size": "过滤插入片段大小",
    "Select reads marked as duplicates": "选择标记为重复的读取",
    "Select reads failing QC": "选择未通过质量控制的读取",
    "Select first mate in a read pair": "选择读段对中的第一个成员",
    "Selected mapped reads": "选择的映射读段",
    "Select reads with mapped mate": "选择有配对映射的读取",
    "Select reads with mate on the reverse strand": "选择配对在反向链上的读取",
    "Select paired reads": "选择配对的读取",
    "Select BAM records for primary alignments": "选择BAM记录以进行主要比对",
    "Select properly paired reads": "选择正确配对的读取",
    "Select reads in the reverse strand only": "选择仅在反向链上的读取",
    "Select second mate in a read pair": "选择读对中的第二个配偶",
    "Filter on read mapping quality (phred scale)": "过滤读取映射质量（phred分数）",
    "Filter on the position of the mate": "过滤配对的位置",
    "Filter on reference name for the mate": "根据参考序列名称过滤配对序列",
    "Filter on read name": "过滤读取名称",
    "Filter on the position of the read": "根据读取位置进行过滤",
    "Filter on a sequence motif": "根据序列模式进行过滤",
    "Filter on the reference name for the read": "根据参考名称过滤读取",
    "Filter on a particular tag": "过滤特定标签",
    "Would you like to set rules?": "您想要设置规则吗？",
    "Enter rules here": "在这里输入规则",
    "BAM file to recalculate": "BAM文件重新计算",
    "Do you also want BAQ (Base Alignment Quality) scores to be calculated?": "您是否也需要计算 BAQ（基础比对质量）分数？",
    "Use BAQ to cap read base qualities": "使用BAQ来限制读取碱基质量",
    "Extended BAQ for better sensitivity": "扩展 BAQ 以提高灵敏度",
    "Change identical bases to '='": "将相同的碱基更改为'='",
    "Coefficient to cap mapping quality of poorly mapped reads": "用于限制质量较差的读取映射质量的系数",
    "Input BAM (or CRAM)": "输入BAM（或CRAM）",
    "Output per base depth": "每碱基深度输出",
    "Compute depth by region": "计算区域深度",
    "Window size for region calculation": "区域计算的窗口大小",
    "BED file specifying regions": "指定区域的BED文件",
    "Restrict depth calculations to chromosome": "限制深度计算到染色体",
    "Exclude reads with these bits set in FLAG": "排除在FLAG中设置了这些位的读取",
    "Only include reads with these bits set in FLAG": "仅包含在FLAG中设置了这些位的读取",
    "Disable fast mode": "禁用快速模式",
    "Specify thresholds for output when using region output": "在使用区域输出时指定输出阈值",
    "Use median depth of region": "使用区域的中位数深度",
    "Read depth thresholds for depth-defined regions": "深度定义区域的读取深度阈值",
    "Minimum depth for depth-defined region": "深度定义区域的最小深度",
    "Depth-defined region name": "深度定义区域名称",
    "Select dataset to phase": "选择数据集进行分相",
    "Phase parameters": "相位参数",
    "Maximum length for local phasing": "局部相位分析的最大长度",
    "Minimum Phred-scaled level of detail to call a heterozygote": "调用杂合子所需的最小Phred缩放细节水平",
    "Minimum base quality to be used in het calling": "在杂合性（het）调用中使用的最低碱基质量",
    "Read depth": "读取深度",
    "Do not attempt to fix chimeric reads": "不要尝试修复嵌合读段",
    "Drop reads with ambiguous phase": "丢弃具有不明确阶段的读取",
    "Choose file type:": "选择文件类型：",
    "Input file in BAM format": "输入文件为BAM格式",
    "Ignore read quality?": "忽略读取质量？",
    "Bam files were not generated with paired-end reads?": "Bam 文件没有使用成对末端读取生成？",
    "Input file in FASTA format": "输入文件为FASTA格式",
    "Input file in FASTQ format": "输入文件为FASTQ格式",
    "List of fastq files are not paired-end reads?": "fastq 文件列表不是成对的末端读取？",
    "Ignore readnames?": "忽略读名字？",
    "BAM dataset to split by single_end/paired_end": "BAM数据集按单端/双端分离",
    "BAM dataset to split by reference": "BAM 数据集按参考序列拆分",
    "BAM dataset to split by mapped/unmapped": "BAM数据集按映射/未映射拆分",
    "BAM dataset to split by tag value": "BAM 数据集按标签值拆分",
    "Enter tag name here": "在这里输入标签名称",
    "Command": "命令",
    "mpileup file": "mpileup 文件",
    "min-coverage": "最小覆盖度",
    "min-reads2": "最小读取数2",
    "min-avg-qual": "最小-平均质量",
    "min-var-freq": "最小变异频率",
    "min-freq-for-hom": "最小频率用于同质性",
    "strand-filter": "链特异性过滤",
    "vcf-sample-list": "vcf样本列表",
    "variants": "变异体",
    "normal mpileup file": "正常 mpileup 文件",
    "tumor mpileup file": "肿瘤 mpileup 文件",
    "min-coverage-normal": "最小覆盖度-正常",
    "min-coverage-tumor": "最小覆盖度-肿瘤",
    "normal-purity": "正常纯度",
    "tumor-purity": "肿瘤纯度",
    "somatic-p-value": "体细胞-P值",
    "validation": "验证",
    "Input BAM 1": "输入 BAM 1",
    "Input BAM 2": "输入BAM文件 2",
    "max base pair difference between possibly matching records": "最大碱基对差异，可能匹配的记录之间",
    "only print the fields that differ": "只打印不同的字段",
    "BAM fields to diff": "BAM字段差异",
    "diff the flags.": "比较标志。",
    "diff the mapping qualities.": "比较映射质量。",
    "diff the mate chrom/pos.": "比较配对的染色体/位置。",
    "diff the insert sizes.": "比较插入大小。",
    "diff the sequence bases.": "比较序列碱基。",
    "diff the base qualities.": "比较碱基质量。",
    "do not diff the the cigars.": "不要对CIGAR字符串进行差异比较。",
    "do not diff the positions.": "不要对位置进行差异比较。",
    "Tags to diff": "标签差异",
    "diff the specified Tags formatted as Tag:Type,Tag:Type,Tag:Type...": "比较指定的标签，格式为 Tag:Type,Tag:Type,Tag:Type...",
    "Select a sam or bam file.": "选择一个sam或bam文件。",
    "Split BAM dataset(s) by": "将BAM数据集按以下方式分割：",
    "Select dataset to filter": "选择数据集进行过滤",
    "Flag": "标志位",
    "Set the states for this flag": "设置此标志的状态",
    "SAM file to convert": "SAM文件转换",
    "Select SAM or BAM file on which to clip overlapping read pairs": "选择SAM或BAM文件，以便对配对的重叠读取进行剪切。",
    "Enter a tag in which to store the original CIGAR": "输入一个标签，用于存储原始的CIGAR字符串",
    "Output statistics on the overlaps?": "输出重叠部分的统计信息？",
    "Only output overlapping read pairs?": "仅输出重叠的配对序列？",
    "Enter an integer representation of a flag to skip records with any of the specified flags set": "输入一个整数表示的标记，以跳过设置任何指定标记的记录",
    "Mark records that would be completely clipped as unmapped?": "将完全被剪切的记录标记为未映射？",
    "Header options": "标题选项",
    "Input SAM file": "输入 SAM 文件",
    "5' end window (n)": "5'端窗口（n）",
    "3' end window (m)": "3'端窗口（m）",
    "Save discarded reads in additional SAM file?": "在额外的SAM文件中保存丢弃的读取？",
    "Reads with a smaller insert size than this are ignored": "忽略插入大小比这个小的读取",
    "Reads with a larger insert size than this are ignored": "忽略插入大小比这个更大的读取",
    "BED/VCF/GFF file": "BED/VCF/GFF 文件",
    "Specify names for each file": "为每个文件指定名称",
    "Add BED files and names": "添加 BED 文件和名称",
    "Name of the file": "文件名",
    "Report the count of features followed by the % coverage for each annotation file": "报告每个注释文件的特征计数，然后是每个注释文件的%覆盖率。",
    "Convert the following BAM file to BED": "将以下BAM文件转换为BED文件",
    "What type of BED output would you like": "您想要哪种类型的BED输出",
    "Use alignment's edit-distance for BED score": "使用比对的编辑距离为BED打分",
    "Use other NUMERIC BAM alignment tag as the BED score": "使用其他数值型BAM比对标签作为BED分数",
    "Convert the following BED12 file to BED6": "将以下 BED12 文件转换为 BED6 文件",
    "Convert the following BED file to BAM": "将以下BED文件转换为BAM文件",
    "Indicate that the input BED file is in BED12 (a.k.a 'blocked' BED) format": "表明输入的BED文件是BED12（也称为“blocked” BED）格式",
    "Set a mapping quality (SAM MAPQ field) value for all BED entries": "为所有BED条目设置一个映射质量（SAM MAPQ字段）值",
    "overlap intervals in this BED/VCF/GFF file?": "在这个BED/VCF/GFF文件中重叠的区间？",
    "How ties for closest feature should be handled": "如何处理最接近特征的并列情况",
    "In addition to the closest feature in B, report its distance to A as an extra column": "除了报告B中距离A最近的一个特征外，还要在额外的一列中报告它与A的距离",
    "Add additional columns to report distance to upstream feature. Distance defintion": "添加额外的列以报告到上游特征的距离。距离定义",
    "Ignore features in B that are upstream of features in A": "忽略B中位于A上游的特征",
    "Ignore features in B that are downstream of features in A": "忽略B中位于A下游的特征",
    "Ignore features in B that overlap A": "忽略与A重叠的B中的特征",
    "How multiple databases are resolved": "如何解析多个数据库",
    "Force strandedness.": "强制单链性。",
    "Maximum distance between features allowed for features to be clustered": "允许特征聚类的最大距离",
    "Count how many intervals in this BED/VCF/GFF/BAM file (source)": "计算这个BED/VCF/GFF/BAM文件（来源）中的区间数量",
    "overlap the intervals in this BED file (target)": "在该 BED 文件（目标）中重叠区间",
    "Force strandedness": "强制链特异性",
    "Report the depth at each position in each B feature": "在每个B特征的每个位置上报告深度",
    "Report a histogram of coverage for each feature in B as well as a summary histogram for all features in B": "为B中的每个特征报告一个覆盖度直方图，以及B中所有特征的汇总直方图。",
    "Merge overlapping intervals before looking at overlap": "在查看重叠之前合并重叠区间",
    "Define -l and -r as a fraction of the feature’s length": "将 -l 和 -r 定义为特征长度的分数",
    "Define -l and -r based on strand": "根据链定义 -l 和 -r",
    "The BAM or BED file from which coverage should be computed": "要计算覆盖度的BAM或BED文件",
    "Output type": "输出类型",
    "Report regions with zero coverage": "报告零覆盖区域",
    "Scale the coverage by a constant factor": "将覆盖度按一个常数因子进行缩放",
    "Specify max depth": "指定最大深度",
    "Calculate coverage based on": "基于计算覆盖度",
    "Report the depth at each genome position with 1-based coordinates": "在每个基因组位置上报告深度，使用1-based坐标。",
    "Report the depth at each genome position with 0-based coordinatess": "在每个基因组位置报告深度，使用0-based坐标。",
    "Calculate coverage of 5’ positions": "计算5'端位置的覆盖度",
    "Calculate coverage of 3’ positions": "计算3'端位置的覆盖度",
    "Use the 'name' column in the BED file for the FASTA headers in the output FASTA file": "在输出的FASTA文件中，使用BED文件中的“name”列作为FASTA头。",
    "Report extract sequences in a tab-delimited format instead of in FASTA format": "将序列以制表符分隔的格式报告，而不是以FASTA格式报告",
    "Specifies which column(s) (1-based) should be used to group the input": "指定应该使用输入中的哪一列或哪些列（从1开始计数）来进行分组",
    "Specify the operation": "指定操作",
    "BED/VCF/GFF/BAM file": "BED/VCF/GFF/BAM 文件",
    "One or more BAM/BED/GFF/VCF file(s)": "一个或多个BAM/BED/GFF/VCF文件",
    "What should be written to the output file?": "应该写入输出文件的内容是什么？",
    "Minimum overlap required as a fraction of the BAM alignment": "BAM比对所需的最小重叠部分，以分数表示。",
    "Report only those alignments that **do not** overlap the BED file": "只报告那些不与BED文件重叠的比对",
    "Write the original A entry _once_ if _any_ overlaps found in B.": "如果B中发现任何重叠，则仅写入原始的A条目一次。",
    "For each entry in A, report the number of overlaps with B.": "对于A中的每个条目，报告与B重叠的数量。",
    "The 'basename' for the UCSC genome browser": "UCSC基因组浏览器的'basename'",
    "Organism name": "生物体名称",
    "The genome build": "基因组构建",
    "Work with": "使用",
    "Divide each input interval (either a chromosome or a BED interval) to fixed-sized windows": "将每个输入区间（可以是染色体或BED区间）划分为固定大小的窗口",
    "Specify Step size? i.e. how many base pairs to step before creating a new window": "指定步长大小？即在创建新窗口之前要跨越多少个碱基对",
    "Specify it": "指定它",
    "Divide each input interval (either a chromosome or a BED interval) to fixed number of windows": "将每个输入区间（可以是染色体或BED区间）划分为固定数量的窗口",
    "ID Naming Options": "ID命名选项",
    "File A (BAM/BED/VCF/GFF)": "文件A（BAM/BED/VCF/GFF）",
    "File B (BAM/BED/VCF/GFF)": "文件 B (BAM/BED/VCF/GFF)",
    "Require reciprocal overlap": "需要互惠重叠",
    "Specify the column(s) that should be summarized": "指定需要汇总的列",
    "Treat split/spliced BAM or BED12 entries as distinct BED intervals when computing coverage.": "将拆分/拼接的BAM或BED12条目视为独立的BED区间，在计算覆盖度时进行处理。",
    "Soft-mask (that is, convert to lower-case bases) the FASTA sequence": "将FASTA序列进行软掩蔽（即转换为小写碱基）",
    "Replace masking character": "替换掩码字符",
    "Sort the following BAM/BED/VCF/GFF file": "对以下BAM/BED/VCF/GFF文件进行排序",
    "Calculation based on strandedness?": "基于链特异性的计算？",
    "Maximum distance between features allowed for features to be merged": "允许特征合并的最大距离",
    "Sorted BED file": "排序后的BED文件",
    "Minimum mapping quality (MAPQ) allowed": "最小映射质量（MAPQ）允许值",
    "Include duplicate reads": "包含重复的读取",
    "Include failed-QC reads": "包含未通过质量控制的读取",
    "Only count proper pairs": "仅计算正确的配对",
    "BED files": "BED文件",
    "Add BED files": "添加 BED 文件",
    "Custom sample name": "自定义样本名称",
    "Invoke Ryan Layers's clustering algorithm": "调用Ryan Layers的聚类算法",
    "Report regions that are not covered by any of the files": "报告未被任何文件覆盖的区域",
    "Text to use for no-coverage value": "用于无覆盖值的文本",
    "Profile the sequence according to strand": "根据链对序列进行分析",
    "Print the extracted sequence": "打印提取的序列",
    "Report the number of times a user-defined sequence is observed": "报告观察到用户定义序列的次数",
    "Igore case when matching -pattern": "在匹配时忽略大小写 -pattern",
    "Specify the columns for the starts and ends of the features for which you’d like to compute the overlap/distance": "指定您想要计算重叠/距离的特征的起始和结束列",
    "The length of the intervals to generate": "生成间隔的长度",
    "The number of intervals to generate": "要生成的区间数",
    "Instead of a summary, report the relative distance for each interval in A": "而不是总结，报告A中每个间隔的相对距离",
    "The file is in BEDPE format": "该文件是BEDPE格式的",
    "Keep features in the input file on the same chromosome": "保持输入文件中同一染色体上的特征",
    "Choose an additional BED file": "选择一个额外的 BED 文件",
    "Choose File": "选择文件",
    "Choose chromosome first": "选择染色体首先",
    "Max. number of attempts to find a home for a shuffled interval in the presence of -incl or -excl": "最大尝试次数，在存在 -incl 或 -excl 参数的情况下，为一个打乱的区间寻找一个合适的位置",
    "Don’t allow shuffled intervals to overlap": "不允许打乱的区间重叠",
    "Allow the original the length of the original records to extebd beyond the length of the chromosome": "允许原始记录的长度超出染色体的长度",
    "Sort the following BED file": "对以下BED文件进行排序",
    "Sort by": "按排序",
    "Specify the tag to use": "指定要使用的标签",
    "Field from the annotation files to populate tags?": "注释文件中用于填充标签的字段？",
    "BedGraph files": "BedGraph 文件",
    "Add BedGraph files": "添加 BedGraph 文件",
    "BedGraph file": "BedGraph文件",
    "When using BAM input, write output as BED. The default is to write output in BAM when using a bam file": "当使用BAM输入时，将输出写为BED格式。默认情况下，使用bam文件时会以BAM格式输出",
    "Choose what you want to do": "选择您想要做什么",
    "Base pairs to add upstream and downstream": "上游和下游要添加的碱基对",
    "Base pairs added upstream (left) of each entry in A when searching for overlaps in B": "在B中搜索重叠时，每个条目A上游（左侧）添加的碱基对",
    "Base pairs added downstream (right) of each entry in A when searching for overlaps in B": "在B中搜索与A中每个条目重叠时，每个条目下游（右侧）添加的碱基对",
    "Write original A entry once if any overlaps found in B": "如果B中有任何重叠，则写入原始A条目的一次",
    "For each entry in A, report the number of hits in B": "对于A中的每个条目，在B中报告命中的次数。",
    "Only report those entries in A that have no overlaps with B": "只报告在A中那些与B没有重叠的条目",
    "@STD_BEDTOOLS_INPUT_LABEL@ file": "标准_BEDTOOLS输入标签文件",
    "@STD_BEDTOOLS_INPUT_LABEL@ files": "@STD_BEDTOOLS_INPUT_LABEL@ 文件",
    "Create IGV batch script for this @STD_BEDTOOLS_INPUT_LABEL@ file": "为这个 @STD_BEDTOOLS_INPUT_LABEL@ 文件创建 IGV 批量脚本",
    "Sort BAM file by": "对BAM文件进行排序",
    "Collapse the aligned reads prior to taking a snapshot?": "在进行快照之前，是否要折叠对齐的读取？",
    "Use the name field (column 4) for each image's filename.": "使用名称字段（第4列）作为每个图像的文件名。",
    "Number of flanking base pairs on the left and right of the image.": "图像左侧和右侧的相邻碱基对数。",
    "Select the type of image to be created.": "选择要创建的图像类型。",
    "Overlap with: will you select a @STD_BEDTOOLS_INPUT_LABEL@ file from your history or use a built-in GFF file?": "与：您将从历史记录中选择一个 @STD_BEDTOOLS_INPUT_LABEL@ 文件，还是使用内置的 GFF 文件？",
    "Select reference GFF": "选择参考 GFF",
    "Select a @STD_BEDTOOLS_INPUT_LABEL@ file": "选择一个 @STD_BEDTOOLS_INPUT_LABEL@ 文件",
    "Report the k closest hits": "报告k个最近的命中",
    "File A (on which coverage is calculated)": "文件 A（用于计算覆盖度的文件）",
    "Combined or separate output files": "合并或单独输出文件",
    "File B (for which coverage is calculated)": "文件B（用于计算覆盖度）",
    "File(s) B (for which coverage is calculated)": "文件(B)（用于计算覆盖度）",
    "Report the depth at each position in each A feature": "在每个A特征的每个位置上报告深度",
    "Report a histogram of coverage for each feature in A as well as a summary histogram for all features in A": "为A中的每个特征报告一个覆盖度直方图，以及A中所有特征的汇总直方图。",
    "Mean depth": "平均深度",
    "Require that the fraction overlap be reciprocal for A AND B.": "要求A和B的分数重叠是相互的。",
    "Require that the minimum fraction be satisfied for A OR B.": "要求 A 或 B 的最小分数必须满足。",
    "Input type": "输入类型",
    "Choose the source for the FASTA file": "选择FASTA文件的来源",
    "Use the 'name' column in the BED file and the coordinates for the FASTA headers in the output FASTA file": "在输出的FASTA文件中使用BED文件中的'name'列和FASTA头的坐标。",
    "File A to intersect with B": "文件A与B相交",
    "File B to intersect with A": "与文件A相交的文件B",
    "File(s) B to intersect with A": "与A相交的文件B",
    "Add data set names instead of indices": "添加数据集名称而不是索引",
    "Required overlap": "所需重叠",
    "Require that the fraction of overlap is fulfilled for A OR B.": "要求 A 或 B 的重叠部分的比例满足条件。",
    "Report only those alignments that **do not** overlap with file(s) B": "只报告那些与文件B（或多个文件）**不重叠**的比对结果",
    "When using BAM input, write output as BED instead of BAM.": "当使用BAM输入时，将输出写为BED而不是BAM。",
    "File A (BAM/@STD_BEDTOOLS_INPUT_LABEL@)": "文件 A (BAM/@STD_BEDTOOLS_INPUT_LABEL@)",
    "File B (BAM/@STD_BEDTOOLS_INPUT_LABEL@)": "文件 B（BAM/@STD_BEDTOOLS_INPUT_LABEL@）",
    "Specify the column(s) from the B file to map onto intervals in A": "指定从B文件中映射到A文件间隔的列",
    "Use full fasta header.": "使用完整的FASTA头信息。",
    "Sort the following BAM/@STD_BEDTOOLS_INPUT_LABEL@ file": "对以下BAM/@STD_BEDTOOLS_INPUT_LABEL@文件进行排序",
    "Sorted @STD_BEDTOOLS_INPUT_LABEL@ file": "排序后的 @STD_BEDTOOLS_INPUT_LABEL@ 文件",
    "Report empty regions": "报告空区域",
    "Ignore case when matching -pattern": "忽略匹配时的大小写 -pattern",
    "@STD_BEDTOOLS_INPUT_LABEL@/BAM file": "@STD_BEDTOOLS_INPUT_LABEL@/BAM文件",
    "Maximum overlap (as a fraction of the -i feature) with an -excl feature that is tolerated before searching for a new, randomized locus": "最大重叠（作为 -i 特征的一部分）与一个 -excl 特征容忍的重叠，才会在寻找一个新的、随机的位点之前进行搜索",
    "Sort the following @STD_BEDTOOLS_INPUT_LABEL@ file": "对以下 @STD_BEDTOOLS_INPUT_LABEL@ 文件进行排序",
    "When using BAM input, write output as BED. The default is to write output in BAM when using a BAM file": "当使用BAM输入时，将输出写为BED格式。默认情况下，使用BAM文件时会以BAM格式输出",
    "Output only unique elements?": "只输出唯一元素？",
    "Output only duplicate elements?": "只输出重复的元素？",
    "Sample BAM file": "样本BAM文件",
    "Limit variant calling to a set of regions?": "限制变异调用到一组区域？",
    "Limit analysis to targets listed in the BED-format FILE.": "将分析限制在BED格式文件中列出的目标上。",
    "Region Chromosome": "区域 染色体",
    "Region Start": "区域起始",
    "Region End": "区域结束",
    "Choose parameter selection level": "选择参数选择级别",
    "Do you want to provide additional inputs?": "您想提供额外的输入吗？",
    "Write out failed alleles file": "写出失败的等位基因文件",
    "Write out algorithm trace file": "写出算法追踪文件",
    "Limit analysis to samples listed (one per line) in the FILE": "将分析限制为文件中列出的样本（每行一个）",
    "Populations File": "种群文件",
    "Read a copy number map from the BED file FILE": "从BED文件读取拷贝数图",
    "Provide variants file": "提供变异文件",
    "Use variants reported in VCF file as input to the algorithm": "使用 VCF 文件中报告的变异作为算法的输入",
    "Only provide variant calls and genotype likelihoods for sites in VCF": "只提供 VCF 中位点的变异调用和基因型似然",
    "Only use variant alleles provided in this input VCF for the construction of complex or haplotype alleles": "仅使用此输入 VCF 中提供的变异等位基因来构建复杂或单倍型等位基因",
    "Report even loci which appear to be monomorphic, and report all considered alleles, even those which are not in called genotypes.": "报告即使是单态性的位点，并报告所有考虑的等位基因，包括那些未在被调用的基因型中的等位基因。",
    "Load read length-dependent allele observation biases from": "从加载读取长度依赖的等位基因观察偏差",
    "Upload per-sample estimates of contamination from": "上传每个样本的污染估计值",
    "Set reporting option?": "设置报告选项？",
    "Report sites if the probability that there is a polymorphism at the site is greater than": "如果位点上存在多态性的概率大于，则报告位点",
    "Set population model?": "设置种群模型？",
    "The expected mutation rate or pairwise nucleotide diversity among the population under analysis": "预期的突变率或分析中的种群中成对核苷酸多样性",
    "Set ploidy for the analysis": "为分析设置倍性",
    "Assume that samples result from pooled sequencing": "假设样本来自于混合测序",
    "Output all alleles which pass input filters, regardles of genotyping outcome or model": "输出所有通过输入过滤器的等位基因，无论基因分型结果或模型如何",
    "Use reference allele?": "使用参考等位基因？",
    "Include the reference allele in the analysis as if it is another sample from the same population": "将参考等位基因包含在分析中，就好像它是来自同一人群的另一个样本一样。",
    "Assign mapping quality of MQ (100) to the reference allele at each site and base quality of BQ (60)": "为每个位点的参考等位基因分配映射质量MQ（100）和碱基质量BQ（60）。",
    "Set allelic scope?": "设置等位基因范围？",
    "Ignore SNP alleles": "忽略SNP等位基因",
    "Ignore indels alleles": "忽略插入缺失等位基因",
    "Ignore multi-nucleotide polymorphisms, MNPs": "忽略多核苷酸多态性，MNPs",
    "Ignore complex events (composites of other classes).": "忽略复杂事件（其他类别的组合）。",
    "How many best SNP alleles to evaluate": "评估多少最佳SNP等位基因",
    "Allow haplotype calls with contiguous embedded matches of up to (nucleotides)": "允许具有最多（nucleotides）个连续嵌入匹配的单倍型调用",
    "When assembling observations across repeats, require the total repeat length at least this many bp": "在组装重复区域的观察结果时，要求总的重复长度至少为这么多碱基对",
    "To detect interrupted repeats, build across sequence until it has entropy > (bits per bp)": "为了检测中断的重复序列，构建序列直到它的熵大于（每碱基位的比特数）",
    "Exclude observations which do not fully span the dynamically-determined detection window": "排除那些没有完全跨越动态确定的检测窗口的观察结果",
    "Turn off left-alignment of indels?": "关闭插入缺失的左对齐？",
    "Set input filters?": "设置输入过滤器？",
    "Include duplicate-marked alignments in the analysis.": "在分析中包括标记为重复的比对。",
    "Exclude alignments from analysis if they have a mapping quality less than": "如果它们的比对质量小于指定值，则从分析中排除比对。",
    "Exclude alleles from analysis if their supporting base quality less than": "如果支持等位基因的碱基质量低于某个值，则将其排除在分析之外",
    "Consider any allele in which the sum of qualities of supporting observations is at least": "考虑任何等位基因，支持该等位基因的观察结果的质量之和至少为",
    "Consider any allele in which and the sum of mapping qualities of supporting reads is at least": "考虑任何等位基因，其中支持读取的映射质量之和至少为",
    "Perform mismatch filtering?": "执行不匹配过滤？",
    "Count mismatches toward -U (option below) if the base quality of the mismatch is >=": "将不匹配的计数计入 -U（下面的选项），如果该不匹配的碱基质量 >=",
    "Exclude reads with more than N mismatches where each mismatch has base quality >= Q (option above)": "排除具有超过N个不匹配的读取，其中每个不匹配的碱基质量 >= Q（上述选项）",
    "Exclude reads with more than N [0,1] fraction of mismatches where each mismatch has base quality >= Q (second option above)": "排除具有超过N[0,1]比例的不匹配的读取，其中每个不匹配的碱基质量 >= Q（上述第二个选项）",
    "Exclude reads with more than N base mismatches, ignoring gaps with quality >= Q (third option abobe)": "排除具有超过N个碱基错配的读取，忽略质量大于等于Q的间隙（第三个选项）",
    "Exclude reads with more than this number of separate gaps": "排除具有超过此数量单独间隙的读取",
    "Use stringent input base and mapping quality filters": "使用严格的输入基础和映射质量过滤器",
    "Require at least this fraction of observations supporting an alternate allele within a single individual in the in order to evaluate the position": "要求至少有这个比例的观察支持单个个体中的替代等位基因，以便评估位置",
    "Require at least this count of observations supporting an alternate allele within a single individual in order to evaluate the position": "要求至少有这么多观察支持单个个体中的替代等位基因，以便评估该位置",
    "Require at least this sum of quality of observations supporting an alternate allele within a single individual in order to evaluate the position": "要求至少有这么多质量的观察支持单个个体中的替代等位基因，以便评估该位置",
    "Require at least this count of observations supporting an alternate allele within the total population in order to use the allele in analysis": "要求至少有这么多的观察结果支持总人口中的替代等位基因，以便在分析中使用该等位基因",
    "Require at least this coverage to process a site": "要求至少这个覆盖度来处理一个位点",
    "Set population and mappability priors?": "设置人群和可映射性先验？",
    "No population priors": "没有种群先验",
    "Disable estimation of the probability of the combination arising under HWE given the allele frequency as estimated by observation frequency": "禁用根据观察频率估计的等位基因频率来估计在哈代-温伯格平衡下组合出现的概率",
    "Disable incorporation of prior expectations about observations": "禁用关于观察结果的先验期望的整合",
    "isable use of aggregate probability of observation balance between alleles as a component of the priors": "使用观察到的等位基因聚合概率平衡作为先验概率的一个组成部分",
    "Tweak genotype likelihoods?": "调整基因型似然？",
    "Limit estimated observation quality by capping base quality at": "将估计的观测质量限制在通过限制基础质量来设定的上限",
    "Generate genotype likelihoods using 'effective base depth' metric qual = 1-BaseQual * 1-MapQual": "使用“有效碱基深度”指标生成基因型可能性，质量值qual = 1 - BaseQual * 1 - MapQual",
    "An estimate of contamination to use for all samples.": "所有样本使用的所有样本污染估计值。",
    "Tweak algorithmic features?": "调整算法特性？",
    "Report genotypes using the maximum-likelihood estimate provided from genotype likelihoods.": "使用来自基因型似然的最大似然估计来报告基因型。",
    "Iterate no more than N times during genotyping step": "在基因分型步骤中，迭代次数不超过N次",
    "Integrate no deeper than the Nth best genotype by likelihood when genotyping": "在基因分型时，通过似然度整合不超过第N个最佳基因型",
    "Integrate all genotype combinations in our posterior space which include no more than N (1) samples with their Mth (3) best data likelihood": "将我们后验空间中的所有基因型组合进行整合，这些组合包括不超过N（1）个样本，并且具有它们的第M（3）好数据似然",
    "Skip sample genotypings for which the sample has no supporting reads": "跳过样本基因分型，如果样本没有支持的读取",
    "Do you want to to limit posterior integration": "您想要限制后验积分吗？",
    "Limit posterior integration to samples where the second-best genotype likelihood is no more than log(N) from the highest genotype likelihood for the sample.": "将后验积分限制在样本中，其中第二高的基因型似然不超过样本最高基因型似然的log(N)。",
    "Use mapping quality of alleles when calculating data likelihoods": "在计算数据似然时使用等位基因的比对质量",
    "Use a weighted sum of base qualities around an indel, scaled by the distance from the indel": "使用围绕插入或删除位点的碱基质量的加权和，根据与插入或删除位点的距离进行缩放。",
    "Incorporate non-independence of reads by scaling successive observations by this factor during data likelihood calculations": "在数据似然计算中，通过这个因子按比例调整连续观察，以纳入读取的非独立性",
    "Calculate the marginal probability of genotypes and report as GQ in each sample field in the VCF output": "计算基因型的边际概率并在VCF输出中每个样本字段报告为GQ",
    "Select BAM dataset to leftalign": "选择BAM数据集进行左对齐",
    "BAM dataset to re-align": "BAM数据集重新对齐",
    "Maximum number of iterations": "最大迭代次数",
    "Sequence changes (SNPs, MNPs, InDels)": "序列变化（SNPs, MNPs, InDels）",
    "Input format": "输入格式",
    "Compatible with GATK 1.x": "兼容GATK 1.x版本",
    "Create CSV report, useful for downstream analysis (-csvStats)": "创建CSV报告，适用于下游分析（-csvStats）",
    "Genome source": "基因组来源",
    "Regulation options": "调控选项",
    "Non-coding and regulatory annotation": "非编码和调控注释",
    "@SNPEFF_VERSION@ Genome Data": "@SNPEFF_VERSION@ 基因组数据",
    "Snpff Genome Version Name (e.g. GRCh38.86)": "Snpff 基因组版本名称（例如 GRCh38.86）",
    "Select genetic code for this sequence": "选择此序列的遗传密码",
    "Upstream / Downstream length": "上游/下游长度",
    "Set size for splice sites (donor and acceptor) in bases": "设置剪接位点（供体和受体）的大小，单位为碱基。",
    "spliceRegion Settings": "拼接区域设置",
    "Set size for splice site region within exons. Default: 3 bases": "设置内含子中剪接位点区域的大小。默认值：3个碱基",
    "Set minimum number of bases for splice site region within intron. Default: 3 bases": "设置内含子中剪接位点区域的最小碱基数目。默认值：3个碱基",
    "Set maximum number of bases for splice site region within intron. Default: 8 bases": "设置内含子中剪接位点区域的最大碱基数目。默认值：8个碱基",
    "Annotation options": "注释选项",
    "Use custom interval file for annotation": "使用自定义区间文件进行注释",
    "Only use the transcripts in this file": "仅使用此文件中的转录本",
    "Filter output": "过滤输出",
    "Filter out specific Effects": "过滤特定效应",
    "Filter output: do not report these Effects": "过滤输出：不要报告这些效应",
    "Chromosomal position": "染色体位置",
    "Text to prepend to chromosome name": "染色体名称前要添加的文本",
    "Produce Summary Stats": "产生摘要统计",
    "Suppress reporting usage statistics to server": "禁止向服务器报告使用统计信息",
    "Name for the database": "数据库名称",
    "Input annotations are in": "输入注释在",
    "Genbank dataset to build database from": "从Genbank数据集构建数据库",
    "Parse Genbank into Fasta": "解析 Genbank 文件为 Fasta 格式",
    "Remove sequence version label?": "移除序列版本标签？",
    "GFF dataset to build database from": "GFF 数据集构建数据库",
    "GTF dataset to build database from": "GTF 数据集用于构建数据库",
    "List entries matching the following expression": "列出匹配以下表达式的条目",
    "Do not output entries matching the following expression": "不要输出匹配以下表达式的条目",
    "Select the annotation database you want to download (e.g. GRCh38.86, mm10 etc.)": "选择您想要下载的注释数据库（例如 GRCh38.86，mm10 等）。",
    "Variant input file in VCF format": "变异输入文件，格式为VCF",
    "VCF File with ID field annotated (e.g. dbSNP.vcf)": "带有ID字段注释的VCF文件（例如 dbSNP.vcf）",
    "Fields to annotate": "要注释的字段",
    "Limit INFO annotation to these INFO IDs": "限制 INFO 注释到这些 INFO ID",
    "Case Control defined in": "病例对照定义在",
    "Case / Control column designation": "病例/对照列指定",
    "PLINK TFAM file": "PLINK TFAM 文件",
    "Fields to extract": "提取字段",
    "One effect per line": "每行一个效应",
    "multiple field separator": "多字段分隔符",
    "empty field text": "空字段文本",
    "Input variant list in VCF format": "输入变体列表，格式为VCF",
    "Type of filter expression": "过滤器表达式类型",
    "Filter criteria": "过滤标准",
    "Set value source": "设置值来源",
    "Invert filter": "反转筛选",
    "Filter mode": "过滤模式",
    "Value to add to FILTER field of non-selected variants": "非选定变异的FILTER字段要添加的值",
    "Value to remove from FILTER field of selected variants": "从选定变异的FILTER字段中移除的值",
    "Value to add to FILTER field of selected variants": "为选定变异的FILTER字段添加的值",
    "Intervals (BED file)": "区间（BED文件）",
    "Exclude Intervals": "排除区间",
    "Info fields to remove, e.g. EFF": "要移除的信息字段，例如 EFF",
    "Variant file (VCF)": "变异文件（VCF）",
    "Variant input file in VCF format to check": "变异输入文件以VCF格式进行检查",
    "Choose the source for the reference genome": "选择参考基因组的来源",
    "Select VCF dataset": "选择 VCF 数据集",
    "Exclude or include failed sites": "排除或包括失败的位点",
    "Data inputs": "数据输入",
    "Main input data type": "主要输入数据类型",
    "plink bed file": "PLINK 是一个广泛使用的遗传数据分析工具，它支持多种文件格式，其中 .bed 文件是 PLINK 的一种输入文件格式，用于存储基因型数据。在 PLINK 中，.bed 文件通常与 .bim 和 .fam 文件一起使用，这三个文件共同构成了 PLINK 的标准输入文件格式。  .bed 文件包含了个体的基因型数据，每个个体的基因型数据占据一行，每个位点的等位基因信息以空格分隔。这种格式便于 PLINK 读取和处理大规模的基因型数据。  如果你需要将数据转换为 PLINK 支持的 .bed 格式，可以使用 PLINK 提供的命令行工具或者在 Galaxy 生信云平台上找到相应的转换工具。",
    "VCF/BCF Input file": "VCF/BCF 输入文件",
    "Set phenotype": "设置表型",
    "Phenotype file": "表型文件",
    "Allow all phenotypes present in the phenotype file to be subject to the association tests you have requested": "允许表型文件中存在的所有表型接受您所请求的关联测试",
    "Input covariate file": "输入协变量文件",
    "Plink functions": "Plink 功能",
    "ID list functions": "ID 列表函数",
    "Keep file.": "保留文件。",
    "Keep-fam file": "保持-fam文件",
    "Remove file": "删除文件",
    "Remove-fam file": "移除-fam文件",
    "ID extraction functions": "ID提取功能",
    "Extract": "提取",
    "Chromosome(s)": "染色体",
    "Exclude Chromosome(s)": "排除染色体",
    "Allow extra chromosomes": "允许额外的染色体",
    "Autosome/unplaced exclusion": "常染色体/未定位排除",
    "SNP exclusive": "SNP独有",
    "Single or multiple variant-based range window?": "单个或多个基于变异的范围窗口？",
    "Specify range for variants": "指定变异的范围",
    "SNP variant id to include": "SNP变异体ID以包含",
    "SNP variant id to exclude": "排除的SNP变异ID",
    "Window": "窗口",
    "from-bp": "来自-bp",
    "to-bp": "到-bp",
    "Force intersect": "强制交集",
    "List of SNP variant ids to include": "包含的SNP变异ID列表",
    "List of SNP variant ids to exclude": "排除的SNP变异ID列表",
    "Arbitrary thinning": "任意细化",
    "Thin": "瘦弱",
    "Thin Count": "薄层计数",
    "BP space": "BP空间",
    "Thin Individual": "瘦个体",
    "Thin Individual count": "单个计数",
    "Missing Genotype Rates": "缺失基因型率",
    "Set Geno": "设置基因组",
    "Set Mind": "设定思维",
    "Allele Frequencies": "等位基因频率",
    "Minimum allele frequency": "最小等位基因频率",
    "Maximum allele frequency": "最大等位基因频率",
    "Minimum allele count": "最小等位基因计数",
    "Maximum allele count": "最大等位基因计数",
    "Hardy-Weinberg Equilibrium": "哈迪-温伯格平衡",
    "Test modifiers": "测试修饰符",
    "Filter on sex and/or founders": "根据性别和/或创始人进行筛选",
    "Sex select": "性别选择",
    "No sex settings": "无性别设置",
    "Nonfouders": "非着床者",
    "Merge plink tilesets": "合并 Plink 瓦片集",
    "Recode": "重新编码",
    "Flip DNA strand for SNPs": "翻转DNA链以进行SNP分析",
    "Update Variant Info: Template String": "更新变体信息：模板字符串",
    "Max allele length": "最大等位基因长度",
    "Update variant columns": "更新变异列",
    "New ID column number": "新ID列编号",
    "Old ID column": "旧ID列",
    "Skip": "跳过",
    "BP column number": "BP 列号",
    "Variant ID column": "变异ID列",
    "Set REF alleles": "设置参考等位基因",
    "Set all alleles specified in the file to REF": "将文件中指定的所有等位基因设置为参考基因型（REF）。",
    "Reference column number": "参考列号",
    "Variant id column in file": "文件中的变体ID列",
    "Set a2 alleles": "设置 a2 等位基因",
    "Set all alleles specified in the file to first ALT allele": "将文件中指定的所有等位基因设置为第一个备选等位基因（ALT）",
    "Alt column number": "替代列号",
    "Return allele frequency file": "返回等位基因频率文件",
    "Return  Hardy-Weinberg statistics file": "返回 哈迪-温伯格统计文件",
    "Missing Data": "缺失数据",
    "Inbreeding": "近交",
    "Sex imputation": "性别推断",
    "Mode select": "模式选择",
    "Female max proportion": "雌性最大比例",
    "Male min proportion": "最小比例",
    "Female max count": "雌性最大计数",
    "Male min count": "最小计数-男性",
    "Male mincountn": "男性 mincountn",
    "Variant pruning": "变异修剪",
    "Window size in bp": "窗口大小（以碱基对为单位）",
    "Step size (variant ct)": "步长（变异计数）",
    "Unphased Hardcall r^2 Threshold": "未分相的Hardcall r^2阈值",
    "Reusing an IBS/IBD calculation": "重用IBS/IBD计算",
    "Cluster flag modifiers": "聚类标志修饰符",
    "Dimension count": "维度计数",
    "mds-plot flag modifiers": "MDS-图 标志修饰符",
    "Perform 1df chi-square allelic test": "执行1df卡方等位基因检验",
    "Replications": "复制",
    "genedrop": "基因滴落",
    "Perm-count": "排列计数",
    "Fisher tests": "费舍尔检验",
    "Counts": "计数",
    "Report basic multiple testing corrections for the raw p-values": "对原始p值报告基本的多重检验校正",
    "Testing modifiers": "测试修饰符",
    "Perform logistic regression and return report": "执行逻辑回归并返回报告",
    "Set dominance model": "设置优势模型",
    "Hide covar": "隐藏协方差",
    "Sex as a covariate": "性别作为协变量",
    "Interaction": "相互作用",
    "Beta": "贝塔",
    "Intercept": "截距",
    "Lambda value": "Lambda值",
    "Scoring system which should be applied to all samples": "应用于所有样本的评分系统",
    "Allel column": "等位基因列",
    "Score column": "得分列",
    "Is a header line present in the scores file?": "分数文件中是否有标题行？",
    "How to combine the valid per-variant scores?": "如何组合每个变体的有效分数？",
    "min": "最小值",
    "max": "最大值",
    "PPC-gap": "PPC-gap",
    "Genome flag modifiers": "基因组标志修饰符",
    "Select sample file(s)": "选择样本文件",
    "Configure sequence error estimation for indels": "配置序列误差估计以处理插入和缺失",
    "Ploidy configuration": "倍性配置",
    "Select ploidy file": "选择倍性文件",
    "Call variants without ploidy assumption on this chromosome": "在该染色体上调用变体，不假设倍性",
    "Generate per-sample gVCFs?": "生成每个样本的gVCF文件？",
    "In the gVCFs, do NOT block-compress sites that fall in these regions": "在gVCF文件中，不要对落在这些区域的位点进行块压缩。",
    "Strelka run configuration": "Strelka 运行配置",
    "Set minMapq": "设置最小Mapq",
    "Select annotation mode(s)": "选择注释模式（们）",
    "Select output file(s)": "选择输出文件",
    "Select normal sample file": "选择正常样本文件",
    "Select tumor sample file": "选择肿瘤样本文件",
    "Generate bed file describing somatic callable regions of the genome": "生成描述基因组中体细胞可调用区域的bed文件",
    "Set depthFilterMultiple": "设置 depthFilterMultiple",
    "Set snvMaxFilteredBasecallFrac": "设置 snvMaxFilteredBasecallFrac",
    "Set snvMaxSpanningDeletionFrac": "设置 snvMaxSpanningDeletionFrac",
    "Set indelMaxWindowFilteredBasecallFrac": "设置indelMaxWindowFilteredBasecallFrac",
    "Set ssnvPrior": "设置 ssnvPrior",
    "Set sindelPrior": "设置 sindelPrior",
    "Set ssnvNoise": "设置 ssnvNoise",
    "Set sindelNoiseFactor": "设置 sindelNoiseFactor",
    "Set ssnvNoiseStrandBiasFrac": "设置 ssnvNoiseStrandBiasFrac",
    "Set minTier1Mapq": "设置 minTier1Mapq",
    "Set minTier2Mapq": "设置 minTier2Mapq",
    "Set ssnvQuality_LowerBound": "设置 ssnvQuality_LowerBound",
    "Set sindelQuality_LowerBound": "设置 sindelQuality_LowerBound",
    "Set ssnvContamTolerance": "设置 ssnvContamTolerance",
    "Set indelContamTolerance": "设置indelContamTolerance",
    "Generic options": "通用选项",
    "Set minimum fractional ALT support": "设置最小分数ALT支持",
    "Set minimum SV size": "设置最小结构变异（SV）大小",
    "Set maximum SV size": "设置最大结构变异（SV）大小",
    "Set minimum fraction of genotyped samples": "设置基因型样本的最小比例",
    "Filter sites for PASS?": "过滤通过的位点？",
    "SV calling options": "SV 调用选项",
    "Select filter mode": "选择过滤模式",
    "Set minimum coverage in tumor": "设置肿瘤中的最小覆盖度",
    "Set maximum fractional ALT support in control": "设置控制中最大分数ALT支持",
    "Set minimum median GQ for carriers and non-carriers": "设置携带者和非携带者的最小中位数基因型质量（GQ）",
    "Set maximum read-depth ratio of carrier vs. non-carrier for a deletion": "设置删除中携带者与非携带者的最大读取深度比率",
    "Set minimum read-depth ratio of carrier vs. non-carrier for a duplication": "设置重复区域携带者与非携带者之间的最小读取深度比率",
    "Select input files": "选择输入文件",
    "Set maximum chunk size to merge groups of BCF files": "设置合并BCF文件组的最大块大小",
    "Set minimum coverage": "设置最小覆盖度",
    "Merge Delly CNV files?": "合并 Delly CNV 文件？",
    "Filter sites for PRECISE?": "过滤位点以确保精确性？",
    "Overlap options": "重叠选项",
    "Set maximum breakpoint offset": "设置最大断点偏移量",
    "Set minimum reciprocal overlap": "设置最小互反重叠",
    "Select input file(s)": "选择输入文件",
    "Discovery options": "发现选项",
    "Set minimum paired-end mapping quality": "设置最小配对端映射质量",
    "Set minimum paired-end quality for translocation": "设置转录最小配对末端质量",
    "Set insert size cutoff": "设置插入大小截止值",
    "Genotyping options": "基因分型选项",
    "Select sequencing technology": "选择测序技术",
    "Set minimum mapping quality": "设置最小比对质量",
    "Consensus options": "共识选项",
    "Set maximum reads for consensus computation": "设置共识计算的最大读取数",
    "Set minimum flank size": "设置最小侧翼大小",
    "Set minimum flank quality": "设置最小侧翼质量",
    "Default Options": "默认选项",
    "Mask": "掩蔽",
    "Iupac Codes": "IUPAC代码",
    "Mark deletions": "标记删除",
    "Mark insertions": "标记插入",
    "Mark substitutions": "标记替换",
    "Mask file option": "掩码文件选项",
    "Mask with": "使用掩码",
    "Write a chain file for liftover": "为liftover编写一个链文件",
    "Set output FASTA ID from name of VCF": "将输出的FASTA ID设置为VCF文件名称",
    "Absent": "缺失",
    "Restrict to": "限制为",
    "Set minimum CNV size": "设置最小 CNV 大小",
    "Set maximum CNV size": "设置最大 CNV 大小",
    "Set probability germline": "设置概率为正常组织",
    "Set minimum site quality": "设置最小位点质量",
    "Set maximum population SD": "设定最大种群标准差",
    "Select mappability map file": "选择可映射性图文件",
    "Set baseline ploidy": "设置基准倍性",
    "CNV calling options": "CNV 检测选项",
    "Set minimum SD read-depth shift": "设置最小标准差读深度偏移",
    "Select delly SV file for breakpoint refinement": "选择delly SV文件以进行断点细化",
    "Use copy-number segmentation?": "使用拷贝数分段？",
    "Read-depth window options": "读取深度窗口选项",
    "Set window size": "设置窗口大小",
    "Set window offset": "设置窗口偏移量",
    "Select input BED file": "选择输入的 BED 文件",
    "Set minimum callable window fraction": "设置最小可调用窗口分数",
    "Use mappable bases for window size?": "使用可映射碱基作为窗口大小？",
    "GC fragment normalization options": "GC片段规范化选项",
    "Set scan window size": "设置扫描窗口大小",
    "Set uniqueness filter for scan windows": "设置扫描窗口的唯一性过滤器",
    "Select file with scanning regions": "选择包含扫描区域的文件",
    "Set count cutoff": "设置计数截止值",
    "Set threshold for excluding extreme GC fraction": "设置排除极端GC含量的阈值",
    "Skip scan window selection?": "跳过扫描窗口选择？",
    "VCF file(s) to merge": "要合并的 VCF 文件",
    "The maximum distance variants can be apart when being merged": "当合并时，变体可以相距的最大距离",
    "The minimum distance threshold a variant can have when using max_dist_linear": "在使用 max_dist_linear 时，变体可以具有的最小距离阈值",
    "Make max_dist this proportion of the length of each variant": "将max_dist设置为每个变体长度的这一比例",
    "The power to use in kd-tree distances (1 is Manhattan, 2 is Euclidean, etc.)": "在kd树距离计算中使用的幂次（1是曼哈顿距离，2是欧几里得距离等）",
    "The minimum sequence identity for two insertions to be merged": "两个插入序列合并所需的最小序列一致性",
    "The kmer size to use when computing Jaccard similarity of insertions": "在计算插入序列的Jaccard相似度时使用的k-mer大小",
    "The maximum length of duplication that can be converted to an insertion": "最大长度的重复可以转换为插入",
    "The minimum number of callsets a variant must be in to be output": "变体必须在的最小调用集数量才能被输出",
    "The minimum number of reads a variant needs to be in the specific callset": "变体在特定调用集中所需的最小读取数",
    "The minimum length a variant needs to be in the specific callset": "特定调用集中变体所需的最小长度",
    "Allow variants with different strands to be merged": "允许将不同链上的变异合并",
    "Allow variants with different types to be merged": "允许不同类型的变体合并",
    "Convert duplications to insertions for SV merging and then convert them back?": "将重复序列转换为插入序列以进行结构变异（SV）合并，然后再将它们转换回原来的形式。",
    "Mark calls in the original VCF files that have enough support to called specific": "Mark 在原始 VCF 文件中调用具有足够支持以特定方式调用的标记",
    "Run type normalization before merging": "在合并之前运行类型标准化",
    "Use edit distance for comparing insertion sequences instead of Jaccard": "使用编辑距离来比较插入序列，而不是使用杰卡德距离。",
    "Only run the preprocessing and not the actual merging or post-processing": "只运行预处理，而不是实际的合并或后处理。",
    "Only run the postprocessing and not the actual merging or pre-processing": "只运行后处理，而不是实际的合并或预处理",
    "Don't change variant IDs (should only be used if input IDs are unique across samples)": "不要改变变异ID（仅在输入ID在所有样本中唯一时使用）",
    "Use the end coordinate as the second coordinate instead of the variant length": "使用末端坐标作为第二个坐标，而不是变异长度",
    "Print the genotypes of the consensus variants in all of the samples they came from": "打印所有样本中共识变异的基因型",
    "Ignore merging info such as support vectors which is already present in the inputs": "忽略合并信息，如已在输入中存在的支持向量",
    "Require every group to have a centroid which is within the distance threshold of each variant": "要求每个组都有一个质心，该质心在每个变体的距离阈值内。",
    "Require every group to have each pair within in it be mergeable": "要求每个组内的每对元素都是可合并的。",
    "Allow variants in the same sample to be merged": "允许同一样本中的变体合并",
    "Convert all variants to INS/DEL/DUP/INV/TRA": "将所有变体转换为插入/删除/复制/倒置/转座",
    "Leave breakpoints as they are even if they are inconsistent": "保持断点不变，即使它们不一致",
    "Only output merged variants which include a variant from the first sample": "只输出合并的变体，这些变体包括来自第一个样本的变体",
    "Whether to normalize chromosome names": "是否标准化染色体名称",
    "A file containing chromosome name mappings": "一个包含染色体名称映射的文件",
    "The VCF file with variant calls/supporting reads": "带有变异调用/支持读取的 VCF 文件",
    "The BAM file containing the reads": "包含读取的BAM文件",
    "The number of bases to output before the variant in REF/ALT fields": "在 REF/ALT 字段中输出变体前的碱基数量",
    "The number of bases to output after the variant in REF/ALT fields": "在REF/ALT字段中，输出变体后的碱基数量",
    "The genome region on each side of the SV to align assembled reads to": "每个结构变异（SV）两侧的基因组区域，用于将组装后的读段对齐到该区域",
    "The min length allowed for a refined insertion sequence": "允许的精炼插入序列的最小长度",
    "The max distance a refined insertion call can be from its old position": "最大距离一个精炼的插入调用可以距离其旧位置",
    "The max length of variant which will be output": "输出变体的最大长度",
    "The max proportion by which a variant's length can change": "变异长度可以改变的最大比例",
    "Use racon twice for consensus finding": "使用racon两次进行共识序列寻找",
    "also try to refine deletion positions/lengths": "也尝试优化删除位置/长度",
    "output original VCF line for very long variants instead of ignoring them": "输出原始的 VCF 行，用于非常长的变异，而不是忽略它们",
    "Variant lists data": "变异列表数据",
    "Variant Frequency Threshold": "变异频率阈值",
    "Image Properties": "图像属性",
    "Plot output format": "绘图输出格式",
    "Cell Ratio": "细胞比例",
    "Color palette used for the gene annotations": "用于基因注释的颜色方案",
    "Perform Clustering?": "执行聚类？",
    "Number of clusters": "聚类数",
    "DCS Mutation File": "DCS突变文件",
    "DCS BAM File": "DCS BAM 文件",
    "Aligned Families File": "对齐家族文件",
    "SSCS BAM File": "单分子测序BAM文件",
    "BAM File of raw reads": "原始读取的BAM文件",
    "JSON File with DCS tag stats": "JSON 文件包含 DCS 标签统计信息",
    "JSON File with SSCS tag stats": "带有SSCS标签统计信息的JSON文件",
    "Tag count threshold": "标签计数阈值",
    "Phred quality score threshold": "Phred 质量分数阈值",
    "Trimming threshold": "修剪阈值",
    "Apply chimera correction?": "应用嵌合体校正？",
    "BAM input dataset": "BAM输入数据集",
    "Discordant reads input": "不匹配的读取输入",
    "Split alignments input": "拆分比对输入",
    "General options": "通用选项",
    "Show evidence for each call (-e)": "显示每个调用的证据（-e）",
    "Minimum weight across all samples for a call (-mw)": "最小权重 - 在所有样本中对于一个调用的最小权重（-mw）",
    "Trim threshold (-tt)": "修剪阈值（-tt）",
    "stat_tail_n": "统计尾部N个",
    "Paired-end options (-pe)": "配对末端选项（-pe）",
    "Read group-specific settings": "读取组特定的设置",
    "Read group to apply settings to": "应用设置的读取组",
    "Default settings for unconfigured read groups": "未配置的读取组的默认设置",
    "Split-read options (-sr)": "拆分读取选项（-sr）",
    "Sample-specific settings": "样本特定设置",
    "Sample to apply settings to": "要应用设置的样本",
    "Default settings for unconfigured samples": "未配置样本的默认设置",
    "Duplicate detection and removal": "重复检测和移除",
    "List of variants to filter": "变异列表筛选",
    "Types of variants to keep": "要保留的变异类型",
    "Quality-based filter options": "基于质量的过滤选项",
    "Coverage-based filter options": "基于覆盖度的过滤选项",
    "Allele frequency filter options": "等位基因频率筛选选项",
    "Strand bias filter options": "链偏性过滤选项",
    "Filter variants based on supporting strand bias?": "根据支持链偏性过滤变异？",
    "Multiple-testing corrected p-value threshold": "多重检验校正后的p值阈值",
    "Multiple testing correction method": "多重检验校正方法",
    "Use compound strand-bias filter?": "使用复合链偏性过滤器？",
    "Apply to indels?": "应用到插入/缺失？",
    "Action to be taken for variants that do not pass the filters defined above": "对于未通过上述定义的过滤器的变异，需要采取的行动",
    "Input reads in BAM format": "输入BAM格式的读取序列",
    "Call variants across": "在...中调用变体",
    "BED dataset with regions to examine": "BED数据集，包含要检查的区域",
    "Types of variants to call": "变异类型调用",
    "Variant calling parameters": "变异检测参数",
    "Coverage": "覆盖度",
    "Minimal coverage": "最小覆盖度",
    "Coverage cap": "覆盖度上限",
    "Use reads from anomalously mapped pairs": "使用来自异常映射对的读取",
    "Base-calling quality": "基序调用质量",
    "Minimum baseQ": "最小碱基质量",
    "Minimum baseQ for alternate bases": "备用碱基的最小碱基质量分数",
    "Base quality to use for alternate bases": "备用碱基使用的基础质量",
    "Overwrite original base qualities with this value": "用这个值覆盖原始的碱基质量",
    "Base alignment quality": "基本比对质量",
    "Consider base/indel alignment qualities during variant calling?": "在变体调用时考虑碱基/插入缺失对齐质量？",
    "Mapping quality": "映射质量",
    "Consider mapping quality during variant calling?": "在变体调用时考虑映射质量？",
    "Maximum mapping quality": "最大映射质量",
    "Source quality": "源数据质量",
    "Compute source quality and consider it during variant calling": "计算源质量并在变体调用时考虑它",
    "Replace non-match base qualities with this value": "将不匹配的碱基质量替换为这个值",
    "VCF input(s) with variants to ignore for source quality computation": "VCF输入（s），用于忽略源质量计算的变体",
    "Joint quality": "联合质量",
    "Minimum joinedQ": "最小连接Q",
    "Minimum joinedQ for alternate bases": "最小连接Q值用于替代碱基",
    "Overwrite joinedQs of alternate bases with this value": "用这个值覆盖交替碱基的joinedQs",
    "Variant filter parameters": "变异过滤参数",
    "Significance threshold for calls": "调用的显著性阈值",
    "Bonferroni correction factor for multiple testing": "多重检验的Bonferroni校正因子",
    "Apply default coverage and strand-bias filter?": "应用默认的覆盖度和链偏性过滤？",
    "Input Read File": "输入读取文件",
    "Coverage normalisation depth": "覆盖度标准化深度",
    "Use bwa aligner": "使用bwa比对器",
    "Select a primer scheme from your history or use one from a tool data table?": "从您的历史记录中选择一个引物方案，或者使用工具数据表中的一个？",
    "Primer Scheme": "引物方案",
    "Will you select a reference genome from your history or use a built-in reference?": "您将从历史记录中选择一个参考基因组，还是使用内置的参考基因组？",
    "Medaka model": "Medaka模型",
    "Sample file": "样本文件",
    "Sample name/id": "样本名称/ID",
    "Select the sequencing method of the input file": "选择输入文件的测序方法",
    "Input BAM file": "输入BAM文件",
    "Set general options": "设置通用选项",
    "Minimum Support": "最小支持度",
    "Maximum Number of Splits": "最大分裂数",
    "Maximum Distance": "最大距离",
    "Minimum Length": "最小长度",
    "Minimum Mapping Quality": "最小映射质量",
    "Number of reads to report": "报告的读数数量",
    "Minimum Seq Size": "最小序列大小",
    "Minimum ZMW": "最小ZMW",
    "Enable CS String": "启用CS字符串",
    "Clustering/phasing and genotyping options": "聚类/分型和基因型选项",
    "Cluster": "聚类",
    "Cluster Support": "聚类支持",
    "Allele Frequency Threshold": "等位基因频率阈值",
    "Minimum Homogenous Allele Frequency": "最小同质等位基因频率",
    "Minimum Heterogeneous Allele Frequency": "最小异质等位基因频率",
    "Report BND": "报告BND",
    "Don't report seq": "不要报告序列",
    "Igonore sd": "忽略 sd",
    "CCS Reads": "CCS读取",
    "Parameter Estimation Options": "参数估计选项",
    "Skip Parameter Estimation": "跳过参数估计",
    "Estimated Deletion Ratio": "估计的删除比率",
    "Estimated Insertion Ratio": "估计插入比率",
    "Maximum Differences Per Window": "每窗口最大差异",
    "Maximum Distance Between Alignment Events": "最大对齐事件间距离",
    "Choose run mode": "选择运行模式",
    "Normal file": "正常文件",
    "Minimum variant allele fraction": "最小变异等位基因分数",
    "Indicate whether to perform local realignment": "指示是否执行局部重新比对",
    "Minimum number of reads supporting the variant": "支持变体的最小读取数",
    "Minimum number of reads for determining strand bias": "确定链偏倚的最小读取数",
    "Minimum mapping quality for reads to be considered": "考虑的最小映射质量",
    "Maximum number of mismatches before a read is no longer considered (gaps are not counted as mismatches)": "在不再考虑一个读段之前，允许的最大不匹配数（不将间隙视为不匹配）",
    "Maximum number of bases considered from 5' end (default: 0, no trimming)": "从5'端考虑的最大碱基数量（默认：0，不进行修剪）",
    "Maximum number of extended based after indel to look for mismatches": "在插入或删除后寻找不匹配的最大扩展基数",
    "Maximum average read position for a variant to be considered.": "一个变体被考虑的最大平均读取位置。",
    "Minimum phred score for a base to be considered a good call": "一个碱基被认为是好的呼叫的最小Phred分数",
    "Minimum quality ratio [(good_quality_reads)/(bad_quality_reads+0.5)] (based on definition of a good call; see previous option)": "最小质量比率[（高质量读段）/（低质量读段+0.5）]（基于一个好的调用的定义；参见前一个选项）",
    "Minimum average mapping quality": "最小平均映射质量",
    "Maximum allowed variant allele fraction in the normal sample": "正常样本中允许的最大变异等位基因分数",
    "Select the sequencing method of the input file(s)": "选择输入文件的测序方法",
    "Advanced WGS settings": "高级全基因组测序设置",
    "Advanced WES settings": "高级WES设置",
    "Type of analysis": "分析类型",
    "Input Tumor BAM file": "输入肿瘤BAM文件",
    "Input Normal BAM file": "输入正常BAM文件",
    "Optional parameters": "可选参数",
    "Add Output Sam Program Record": "添加输出SAM程序记录",
    "Add Output Vcf Command Line": "添加输出 VCF 命令行",
    "Af Of Alleles Not In Resource": "Af 指的是等位基因频率（Allele Frequency），在资源中未包含的等位基因频率。",
    "Annotate With Num Discovered Alleles": "使用发现的等位基因数目进行注释",
    "Annotation groups": "注释组",
    "Annotations to exclude": "排除的注释",
    "Pedigree": "家谱",
    "Base Quality Score Threshold": "碱基质量分数阈值",
    "Minimum depth to be considered callable": "可调用的最小深度",
    "Disable Bam Index Caching": "禁用BAM索引缓存",
    "Disable Read Filter": "禁用读取过滤器",
    "Read Filter": "读取过滤器",
    "Disable Sequence Dictionary Validation": "禁用序列字典验证",
    "Downsampling Stride": "下采样步长",
    "Sites with depth higher than this value will be grouped": "深度高于该值的位点将被分组",
    "Skip sites with median mapping quality below this value": "跳过中位比对质量低于此值的位点",
    "Exclude bases below this quality from pileup": "排除质量低于此值的碱基，不参与堆积分析",
    "Founder Id": "创始人ID",
    "Gcs Max Retries": "Gcs 最大重试次数",
    "Genotype Germline Sites": "基因型胚系位点",
    "Genotype PoN Sites": "基因型非整倍体位点",
    "Alleles": "等位基因",
    "Germline Resource": "胚系资源",
    "Turn off read transformer that clips artifacts associated with end repair insertions near inverted tandem repeats": "关闭读取转换器，该转换器会剪切与末端修复插入物相关的人工制品，这些插入物位于倒置串联重复序列附近。",
    "Initial Tumor Lod": "初始肿瘤负荷",
    "Interval exclusion padding": "间隔排除填充",
    "Interval padding": "间隔填充",
    "Interval Merging Rule": "区间合并规则",
    "Interval Set Rule": "间隔集规则",
    "Lenient": "宽松的",
    "Max Population Af": "最大种群数量（Max Population Af）",
    "Max Reads Per Alignment Start": "每个对齐起始点的最大读取数",
    "Min Base Quality Score": "最小碱基质量分数",
    "Mitochondria mode sets emission and initial LODs to 0": "线粒体模式将发射和初始LODs设置为0",
    "Native Pair Hmm Use Double Precision": "原生配对HMM使用双精度",
    "Normal Lod": "正常 LOD",
    "Normal Sample": "正常样本",
    "Panel Of Normals": "正常人群组",
    "Phred-scaled PCR indel qual for overlapping fragments": "Phred-缩放的PCR插入缺失定性评分，用于重叠片段",
    "Phred-scaled PCR SNV qual for overlapping fragments": "Phred-缩放的PCR SNV质量得分，用于重叠片段",
    "Sites Only Vcf Output": "仅位点 VCF 输出",
    "Read Validation Stringency": "读取验证严格性",
    "Tumor Lod To Emit": "肿瘤 LOD 发射",
    "Advanced parameters": "高级参数",
    "Likelihood and read-based annotations will only take into consideration reads that overlap the variant or any base no further than this distance expressed in base pairs": "似然和基于读取的注释只会考虑与变体重叠或任何碱基不超过这个距离的读取，该距离以碱基对表示。",
    "Active Probability Threshold": "活性概率阈值",
    "Allow Non Unique Kmers In Ref": "允许在参考序列中使用非唯一的Kmers",
    "Assembly Region Padding": "组装区域填充",
    "Bam Writer Type": "Bam 写入器类型",
    "Disable adaptive pruning": "禁用自适应剪枝",
    "Disable Default Annotations": "禁用默认注释",
    "Disable default read filters": "禁用默认的读取过滤器",
    "Dont Increase Kmer Sizes For Cycles": "不要为循环增加Kmer大小",
    "Dont Use Soft Clipped Bases": "不要使用软剪切碱基",
    "Mode for emitting reference confidence scores": "发射参考置信度分数的模式",
    "Enable All Annotations": "启用所有注释",
    "Mark all regions active": "标记所有区域为活跃状态",
    "Force-call filtered alleles": "强制调用过滤掉的等位基因",
    "Upper bounds for reference confidence": "参考置信度的上限",
    "Independent mates": "独立配对",
    "Max Assembly Region Size": "最大组装区域大小",
    "Max Mnp Distance": "最大Mnp距离",
    "Max Num Haplotypes In Population": "最大单倍型数量在种群中",
    "Max Prob Propagation Distance": "最大概率传播距离",
    "Max Suspicious Reads Per Alignment Start": "最大可疑读段数每比对起始",
    "Maximum number of variants": "最大变异数",
    "Min Assembly Region Size": "最小组装区域大小",
    "Min Dangling Branch Length": "最小悬空分支长度",
    "Min Pruning": "最小剪枝",
    "Fractions to consider": "考虑的分数",
    "Num Pruning Samples": "样本修剪数量",
    "Pair Hmm Gap Continuation Penalty": "配对HMM间隙延续惩罚",
    "Pair Hmm Implementation": "配对隐马尔可夫模型实现",
    "Pcr Indel Model": "PCR插入缺失模型",
    "Phred Scaled Global Read Mismapping Rate": "Phred缩放全局读段错配率",
    "Pruning LOD threshold": "修剪LOD阈值",
    "Recover all dangling branches": "恢复所有悬挂分支",
    "Smith Waterman": "史密斯-沃特曼",
    "Use Filtered Reads For Annotations": "使用过滤后的读取序列进行注释",
    "Output parameters": "输出参数",
    "Assembly Region Out": "组装区域输出",
    "Bam Output": "Bam 输出",
    "Concat Options": "串联选项",
    "Naive concat": "简单的拼接",
    "Allow Overlaps": "允许重叠",
    "Remove duplicate": "删除重复",
    "Ligate": "连接",
    "Ligate mode": "连接模式",
    "Compact Ps": "紧凑的Ps",
    "Min Pq": "最小 Pq",
    "Select an annotated Coronavirus genome": "选择一个注释过的冠状病毒基因组",
    "Reads to realign": "重新对齐的读取",
    "Keep MC, MD, NM, and A flags?": "保留MC、MD、NM和A标志？",
    "How to handle base qualities of 2?": "如何处理基础质量为2的情况？",
    "Quality to assume for all bases with BQ2": "为所有碱基假设的质量为BQ2",
    "Overwrite exisitng values with newly computed ones": "用新计算的值覆盖现有的值",
    "Indel calculation approach": "插入缺失计算方法",
    "Indel quality to add": "插入缺失质量要添加",
    "Separate deletion quality": "分离删除质量",
    "Induce the graph from these PAF formatted alignments": "从这些PAF格式的比对中诱导出图谱",
    "The sequences used to generate the alignments": "用于生成比对的序列",
    "Subset the input alignments": "子集输入对齐",
    "Limit copies of a given input base": "限制给定输入碱基的副本数量",
    "Filter exact matches below this length": "过滤掉小于这个长度的完全匹配项",
    "Number of bp to use for transitive closure batch": "用于传递闭包批次的碱基对数",
    "odgi input file": "odgi 输入文件",
    "Width in pixels of output image": "输出图像的像素宽度",
    "Height in pixels of output image": "输出图像的像素高度",
    "Path display height": "路径显示高度",
    "Path x padding": "路径 x 填充",
    "Display a single path per row rather than packing them": "每行显示一条路径，而不是将它们打包在一起",
    "Show thin links of this relative width to connect path pieces": "显示这种相对宽度的细线以连接路径片段",
    "Construct the graph from this GFA input file": "从这个GFA输入文件构建图谱",
    "Topological sort the graph": "拓扑排序图",
    "Output additional GFA file": "输出额外的GFA文件",
    "Your input is in format": "你的输入格式是",
    "Input file in variant graph format (vg)": "输入文件为变体图格式（vg）",
    "Input file in Graphical Fragment Assembly (gfa)": "输入文件为图形化片段组装（GFA）格式",
    "Input file in JSON": "输入文件为JSON格式",
    "Input file in BAM": "输入文件为BAM格式",
    "Choose your output format": "选择您的输出格式",
    "Graph files": "图形文件",
    "A reference path to deconstruct against": "一个用于解构的参考路径",
    "All paths beginning with this value used as reference": "所有以该值开头的路径被用作参考",
    "Non-reference paths beginning with with this value get lumped together to same sample in VCF": "非参考路径，以这个值开始的路径，在VCF文件中会被归并到同一个样本中。",
    "Snarls files": "缠绕体文件",
    "Only consider traversals that correspond to paths in the grpah": "只考虑与图中路径相对应的遍历",
    "Input file in vg or xg format": "输入文件为vg或xg格式",
    "Snippy input zipped dirs": "Snippy 输入压缩目录",
    "Output selection": "输出选择",
    "Minumum proportion for variant evidence": "最小变异证据比例",
    "Minumum QUALITY in VCF column 6": "最小质量值在VCF文件的第6列",
    "Bam header @RG ID": "Bam 头部 @RG ID",
    "Extra BWA MEM options": "额外的BWA MEM选项",
    "Snippy core.full.aln file": "Snippy核心.full.aln文件",
    "Use custom replacement character": "使用自定义替换字符",
    "Replacement character": "替代字符",
    "Merge Options": "合并选项",
    "Force Samples": "强制样本",
    "Info Rules": "信息规则",
    "Merge": "合并",
    "Allow merge unindexed files": "允许合并未索引的文件",
    "Header Options": "标题选项",
    "Use Header": "使用标题",
    "Print Header": "打印标题",
    "VCF dataset to extract flanks": "VCF 数据集提取侧翼序列",
    "The length of the primer sequences on each side of the variant": "变异两侧引物序列的长度",
    "Consensus/variant calling Options": "共识/变异调用选项",
    "Calling method": "调用方法",
    "Constrain": "约束",
    "Insert missed": "插入遗漏",
    "Use prior knowledge of population allele frequencies": "使用种群等位基因频率的先验知识",
    "Prior": "先前",
    "Output also gVCF blocks of homozygous REF calls": "输出也包括了纯合参考位点的gVCF区块",
    "Pval Threshold": "P值阈值",
    "File format Options": "文件格式选项",
    "Select predefined ploidy": "选择预定义的倍性",
    "Ploidy file": "倍性文件",
    "Input/output Options": "输入/输出选项",
    "Single Sample Calling": "单样本变异检测",
    "Keep alts": "保留替代碱基",
    "Comma-separated list of FORMAT fields to output for each sample": "每个样本要输出的 FORMAT 字段的逗号分隔列表",
    "Keep masked reference alleles": "保留掩蔽的参考等位基因",
    "Skip variants": "跳过变异",
    "Output variant sites only": "输出变异位点",
    "When any REF allele does not match the reference genome base": "当任何参考等位基因与参考基因组碱基不匹配时",
    "Atomize": "原子化",
    "Left-align and normalize indels?": "左对齐并标准化插入缺失？",
    "Annotate modified records with INFO/STR indicating the original variant": "将修改后的记录用INFO/STR注释，以指示原始变体。",
    "Perform deduplication for the folowing types of variant records": "对以下类型的变体记录执行去重操作",
    "~multiallelics": "多等位基因",
    "split the following variant types": "分割以下变异类型",
    "merge the following variant types": "合并以下变异类型",
    "Strict Filter": "严格筛选",
    "Restrict all operations to": "限制所有操作到",
    "Other Options": "其他选项",
    "Site Window": "位点窗口",
    "Required References": "必需的参考文献",
    "GFF3 annotation file": "GFF3 注释文件",
    "CSQ Options": "CSQ选项",
    "maximum number of consequences to consider per site": "每个位点考虑的后果的最大数量",
    "run localized predictions considering only one VCF record at a time": "运行局部预测，每次只考虑一个VCF记录",
    "phase": "相位",
    "use this custom tag to store consequences rather than the default BCSQ tag": "使用此自定义标签来存储结果，而不是默认的BCSQ标签。",
    "Abbreviate protein-changing predictions": "缩写蛋白质变化预测",
    "Subset Options": "子集选项",
    "Sample relationship": "样本关系",
    "name of mother": "母亲的名字",
    "name of father": "父亲的名字",
    "name of child": "子节点名称",
    "name of sample A": "样本A的名称",
    "name of sample B": "样本 B 的名称",
    "Restrict analysis to": "限制分析至",
    "Query Sample": "查询样本",
    "Control Sample": "对照样本",
    "Plot results?": "绘制结果？",
    "Plot Threshold": "绘制阈值",
    "HMM Options": "HMM选项",
    "Estimated purity of the query sample": "查询样本的估计纯度",
    "Estimated purity of the control sample": "控制样本的估计纯度",
    "Adjust sample purity estimates based on data?": "根据数据调整样本纯度估计？",
    "Lower bound for adjusted estimate": "调整估计的下限",
    "Use BAF and LRR annotations to call CNVs?": "使用 BAF 和 LRR 注释来调用 CNVs？",
    "Baf Weight": "Baf 权重",
    "Query sample BAF deviation": "查询样本 BAF (等位基因频率) 偏差",
    "Control sample BAF deviation": "控制样本 BAF 偏差",
    "LRR Weight": "LRR权重",
    "Query sample LRR Deviation": "查询样本 LRR 偏差",
    "Control sample LRR Deviation": "对照样本 LRR 偏差",
    "LRR Smoothing Window": "LRR 平滑窗口",
    "Err Prob": "错误概率",
    "Same Prob": "同样的问题",
    "Xy Prob": "Xy 概率",
    "FASTA multiple sequence alignment": "FASTA 多序列比对",
    "Count all differences, not just ACTG": "计算所有差异，不仅仅是ACTG",
    "Keep case, don't uppercase all letters": "保持大小写，不要将所有字母大写",
    "Blank top left corner cell": "空白左上角单元格",
    "Output formats": "输出格式",
    "Output constant sites (suitable for use in BEAST)": "输出恒定位点（适用于BEAST使用）",
    "Only output columns containing exclusively ACTG": "仅输出包含仅包含ACTG的列",
    "Target regions (BED)": "目标区域（BED）",
    "Call CNVs using 1st sample as test": "调用 CNVs，使用第一个样本作为测试",
    "BAM": "BAM（二进制比对/映射格式）是一种用于存储生物信息学数据的文件格式，通常用于存储高通量测序数据的比对结果。",
    "Label": "标签",
    "Transition probability": "转移概率",
    "Limit analysis to regions in this BED dataset": "限制分析到此BED数据集中的区域",
    "Read coverage": "读取覆盖度",
    "Additional inputs": "额外输入",
    "Limit analysis to samples listed (one per line) in this dataset": "将分析限制为在此数据集中列出的样本（每行一个）",
    "Populations dataset": "种群数据集",
    "Read a copy number map from a BED dataset": "从BED数据集读取拷贝数图",
    "Provide variants dataset": "提供变异数据集",
    "Use variants reported in this VCF dataset as input to the algorithm": "使用此 VCF 数据集中报告的变异作为算法的输入",
    "Report even loci which appear to be monomorphic, and report all considered alleles, even those which are not in called genotypes": "报告即使是单态的位点，并报告所有考虑的等位基因，即使它们不在被调用的基因型中。",
    "Trim trailing reference matches": "修剪尾部的参考匹配",
    "Population model options": "种群模型选项",
    "Reference allele options": "参考等位基因选项",
    "Allelic scope options": "等位基因范围选项",
    "Ignore complex events (composites of other classes)": "忽略复合事件（其他类别的组合）",
    "Turn off left-alignment of indels": "关闭插入和缺失的左对齐",
    "Input filters": "输入过滤器",
    "Include duplicate-marked alignments in the analysis": "在分析中包含标记重复的比对",
    "Mismatch filters": "不匹配过滤器",
    "Exclude reads with more than N mismatches where each mismatch has base quality >= mismatch-base-quality-threshold (option above)": "排除具有超过N个不匹配的读取，其中每个不匹配的碱基质量 >= 不匹配碱基质量阈值（上述选项）",
    "Exclude reads with more than N [0,1] fraction of mismatches where each mismatch has base quality >= mismatch-base-quality-threshold (second option above)": "排除具有超过N [0,1]比例的不匹配的读取，其中每个不匹配的碱基质量大于或等于不匹配碱基质量阈值（上面的第二个选项）",
    "Exclude reads with more than N base mismatches, ignoring gaps with quality >= mismatch-base-quality-threshold (third option above)": "排除具有超过N个碱基不匹配的读段，忽略质量大于等于不匹配碱基质量阈值的间隙（上面的第三个选项）",
    "Population and mappability priors": "种群和可映射性先验",
    "Disable use of aggregate probability of observation balance between alleles as a component of the priors": "禁用将观察到的等位基因平衡的聚合概率作为先验概率的一个组成部分",
    "Genotype likelihood options": "基因型似然选项",
    "An estimate of contamination to use for all samples": "所有样本使用的污染估计值",
    "Algorithmic features": "算法特性",
    "Report genotypes using the maximum-likelihood estimate provided from genotype likelihoods": "使用来自基因型似然的最大似然估计报告基因型",
    "Limit posterior integration to samples where the second-best genotype likelihood is no more than log(N) from the highest genotype likelihood for the sample": "将后验积分限制在样本中，其中第二高的基因型似然不超过样本最高基因型似然的log(N)。",
    "Select alignment file in BAM format": "选择BAM格式的比对文件",
    "reference genome": "参考基因组",
    "aligned reads from normal sample": "正常样本的比对读段",
    "aligned reads from tumor sample": "肿瘤样本的比对读取",
    "Estimated purity (non-tumor content) of normal sample": "估计的纯度（非肿瘤内容）的正常样本",
    "Estimated purity (tumor content) of tumor sample": "估计的肿瘤样本纯度（肿瘤含量）",
    "Generate separate output datasets for SNP and indel calls?": "生成独立的输出数据集，用于SNP和插入/缺失（indel）调用？",
    "Settings for Variant Calling": "变异检测设置",
    "Read selection": "读取选择",
    "Try to correct for read-pair overlaps": "尝试校正配对读段的重叠",
    "Maximum number of reads per site": "每个位点的最大读取数",
    "P-value threshold for calling somatic variants and LOH events": "用于调用体细胞变异和LOH事件的P值阈值",
    "Settings for Posterior Variant Filtering": "后验变异过滤设置",
    "Minimum number of variant-supporting reads": "最小变异支持读取数",
    "Low coverage minimum number of variant-supporting reads": "低覆盖度最小变异支持读取数",
    "Minimum variant allele count threshold": "最小变体等位基因计数阈值",
    "Minimum variant allele frequency": "最小变异等位基因频率",
    "Minimum relative variant position in ref-supporting reads": "在参考支持的读取中，最小相对变异位置",
    "Minimum relative variant position in variant-supporting reads": "在变异支持读段中最小相对变异位置",
    "Minimum distance of variant site from 3'-end of ref-supporting reads": "变异位点与支持参考序列的读取3'端的最小距离",
    "Minimum distance of variant site from 3'-end of variant-supporting reads": "变异位点与支持变异的读取的3'端的最小距离",
    "Minimum length of ref-supporting reads": "参考支持的最小读长",
    "Minimum length of variant-supporting reads": "变异支持读段的最小长度",
    "Maximum relative read length difference": "最大相对读取长度差异",
    "Minimum fraction of variant reads from each strand": "每个链上变异读取的最小比例",
    "Minimum variant allele depth required to apply the --min-strandedness filter": "应用 --min-strandedness 过滤器所需的最小变异等位基因深度",
    "Minimum average base quality for the ref allele": "参考等位基因的最小平均碱基质量",
    "Minimum average base quality for the variant allele": "变体等位基因的最小平均碱基质量",
    "Maximum base quality difference between ref- and variant-supporting reads": "参考和变异支持读取之间的最大碱基质量差异",
    "Minimum average mapping quality of ref-supporting reads": "参考支持的读取的平均最小映射质量",
    "Minimum average mapping quality of variant-supporting reads": "变体支持性读取的平均最小映射质量",
    "Maximum mapping quality difference between ref- and variant-supporting reads": "最大映射质量差异，参考支持和变体支持的读取之间",
    "Maximum mismatch base quality sum of ref-supporting reads": "参考支持读段中最大错配碱基质量总和",
    "Maximum mismatch base quality sum of var-supporting reads": "最大不匹配碱基质量分数之和的变异支持读取",
    "Minimum difference between mismatch base quality sums of variant- and ref-supporting reads": "变体支持和参考支持的读取中不匹配碱基质量分数和的最小差异",
    "Maximum difference between mismatch base quality sums of variant- and ref-supporting reads": "变体支持和参考支持的读取中不匹配碱基质量分数之和的最大差异",
    "Reference Sequence File": "参考序列文件",
    "Minimum supporting reads, each side": "最小支持读取数，每侧",
    "Choose a bcf file to view": "选择一个bcf文件进行查看",
    "Choose the output format": "选择输出格式",
    "List of chromosome names for conversion": "染色体名称列表用于转换",
    "Use alternate INDEL-to-SNP mutation rate": "使用替代的插入缺失与单核苷酸多态性突变率",
    "Mutation rate for variant calling": "变异调用的突变率",
    "Retain all possible alternate alleles at variant sites": "在变异位点保留所有可能的替代等位基因",
    "Output Potential Variant Sites Only": "只输出潜在的变异位点",
    "SNP calling": "单核苷酸多态性(SNP)检测",
    "Suppress all individual genotype information": "抑制所有个体基因型信息",
    "Skip sites where the REF field is not A/C/G/T": "跳过参考字段不是A/C/G/T的位点",
    "Perform max-likelihood inference only": "仅执行最大似然推断",
    "Call per-sample genotypes at variant sites": "在变异位点调用每个样本的基因型",
    "VCF file to be normalised": "待规范化的 VCF 文件",
    "File containing list of intervals": "包含区间列表的文件",
    "Window size for local sorting of variants": "局部排序变异的窗口大小",
    "Restrict to regions": "限制到区域",
    "Restrict to regions by file": "通过文件限制区域",
    "Genomic Regions": "基因组区域",
    "Minimum number of reads needed to consider a REF/ALT": "最小需要的读取数量以考虑参考/替代位点",
    "Ploidy": "倍性",
    "Only write out positions with possible alternate alleles": "只写出可能有替代等位基因的位置",
    "Report counts by strand": "按链报告计数",
    "Show Advanced Options": "显示高级选项",
    "Choose the dtype to use for storing coverage information": "选择用于存储覆盖信息的数据类型",
    "Be extra safe": "格外小心",
    "Select Annotation database": "选择注释数据库",
    "Molecular Signatures Database (MSigDB)": "分子特征数据库（MSigDB）",
    "dbNSFP": "dbNSFP",
    "Annotate with": "注释",
    "DbNSFP": "DbNSFP",
    "Minimum read depth": "最小读取深度",
    "Minimum base quality to count for coverage": "用于覆盖度计算的最低碱基质量",
    "Minimum read mapping quality to count for coverage": "最小读取映射质量以计算覆盖度",
    "Minimum number of consecutive bases to report a segment": "报告一个区段所需的最小连续碱基数",
    "Max size before a new segment is made": "在创建新段之前的最大大小",
    "P-value threshold for significant copynumber change-point": "显著拷贝数变化点的 P 值阈值",
    "The normal/tumor input data ratio for copynumber adjustment": "正常/肿瘤输入数据比例用于拷贝数调整",
    "Pileup dataset": "堆叠数据集",
    "Analysis type": "分析类型",
    "Minimum supporting reads": "最小支持读数",
    "Minimum base quality at a position to count a read": "在某个位置上，计数一个读取的最小碱基质量",
    "Minimum variant allele frequency threshold": "最小变体等位基因频率阈值",
    "Minimum frequency to call homozygote": "最小频率以调用纯合子",
    "p-value threshold for calling variants": "调用变异的p值阈值",
    "Ignore variants with >90% support on one strand": "忽略在一个链上支持度超过90%的变异",
    "Samtools pileup dataset": "Samtools 堆积数据集",
    "Default p-value threshold for calling variants": "调用变异的默认p值阈值",
    "SnpEffect VCF report": "SnpEffect VCF报告",
    "Biomart Server (defaults to www.biomart.org, for an archive server enter the archive date)": "Biomart 服务器（默认为 www.biomart.org，对于存档服务器请输入存档日期）",
    "Oragnism": "有机体",
    "Filter SnpEffect": "过滤 SnpEffect",
    "Report the variant coding for each Ensembl Transcript at the variant position": "在变异位置报告每个Ensembl转录本的编码变异",
    "Report the variant coding only for Ensembl Transcripts with a CCDS ID": "仅报告具有CCDS ID的Ensembl转录本中的变异编码。",
    "Report the Amino_Acid_change from SnpEff": "报告 SnpEff 的氨基酸变化",
    "Ignore variants that are part of a Poly-A of at least this length": "忽略至少包含此长度的Poly-A部分的变异",
    "Report formats": "报告格式",
    "SnpEff generated VCF file with NON_SYNONYMOUS_CODING annotations": "SnpEff 生成了带有 NON_SYNONYMOUS_CODING 注释的 VCF 文件",
    "Ensembl all_pep.fa": "Ensembl 全蛋白序列.fa",
    "Preceeding AAs": "前面的氨基酸",
    "Following AAs": "接下来的氨基酸",
    "Select INFO annotation to parse": "选择INFO注释进行解析",
    "Enter fields to be extracted, listed either as indexes or names": "输入要提取的字段，可以是索引或名称列表",
    "Enable Output per transcript/allele consequences on a new line rather rather than as comma-separated fields on a single line": "在新行上启用每个转录本/等位基因后果的输出，而不是作为单个行上的逗号分隔字段。",
    "Enter prefix of INFO annotations to be created after splitting the CSQ string (optional)": "输入在拆分CSQ字符串后要创建的INFO注释的前缀（可选）",
    "Select transcripts to extract by type and/or consequence (optional)": "选择根据类型和/或结果提取转录本（可选）",
    "Allow undefined tags": "允许未定义的标签",
    "Input VCF": "输入 VCF",
    "Input Reference FASTA": "输入参考FASTA",
    "Alignment Inputs": "对齐输入",
    "Input BAM/CRAM": "输入BAM/CRAM",
    "Input BAM/CRAMs": "输入BAM/CRAM文件",
    "Genome Reference": "基因组参考",
    "Indel Calling": "插入缺失调用",
    "Perform INDEL calling": "进行插入缺失(INDEL)变异的检测",
    "Phred-scaled gap open sequencing error probability": "Phred-scaled 表示的间隔开放测序错误概率",
    "Phred-scaled gap extension sequencing error probability": "Phred-scaled 间隙延伸测序错误概率",
    "Coefficient for modeling homopolymer errors.": "模拟同聚物错误的系数。",
    "Skip INDEL calling if the average per-sample depth is above": "如果每个样本的平均深度超过，则跳过 INDEL 调用",
    "Minimum gapped reads for indel candidates": "最小间隔读取用于插入/缺失候选",
    "Minimum fraction of gapped reads": "最小间隙读取分数",
    "Apply --min-ireads and --gap-frac values on a per-sample basis": "在每个样本的基础上应用 --min-ireads 和 --gap-frac 值",
    "Platform for INDEL candidates": "插入缺失候选平台",
    "Platform to use for INDEL candidates": "用于INDEL候选的基因组平台",
    "Ambiguous indel reads": "模糊的插入或删除读取",
    "Indel bias": "插入缺失偏差",
    "Indel size": "插入缺失大小",
    "Input Filtering Options": "输入筛选选项",
    "Max reads per BAM": "最大BAM文件中读取数",
    "Do not skip anomalous read pairs in variant calling": "在变异检测中不要跳过异常的配对读取",
    "Skip reads with all of the FLAG bits set": "跳过所有FLAG位都设置的读取",
    "Skip reads with any of the FLAG bits set": "跳过设置了任何 FLAG 位的读取",
    "Skip reads with all of the FLAG bits unset": "跳过所有标志位未设置的读取",
    "Skip reads with any of the FLAG bits unset": "跳过未设置任何FLAG位的读取",
    "Quality Options": "质量选项",
    "per-Base Alignment Quality": "每个碱基的比对质量",
    "Select read groups to include or exclude": "选择包含或排除的读段组",
    "Include or Exclude these Read Groups": "包含或排除这些读取组",
    "gVCF blocks of homozygous REF calls": "gVCF块中的纯合参考调用",
    "No Update": "无需更新",
    "Trim Alt Alleles": "修剪替代等位基因",
    "Min Ac": "最小值",
    "Max Ac": "最大乙酰化水平",
    "Select Genotype": "选择基因型",
    "Select Types": "选择类型",
    "Exclude Types": "排除类型",
    "filter known or novel ID": "过滤已知或新的ID",
    "Min Alleles": "最小等位基因数",
    "Max Alleles": "最大等位基因数",
    "filter phased": "筛选相位",
    "Min Af": "最小亲和度",
    "Max Af": "最大亲和力",
    "filter on genotype uncalled": "过滤未确定的基因型",
    "filter on private": "私有过滤器",
    "Drop Genotypes": "删除基因型",
    "output header": "输出标题",
    "Compression Level": "压缩级别",
    "Invert Targets File": "反转目标文件",
    "Optional VCF/BCF Data to compare against": "可选的 VCF/BCF 数据用于比较",
    "Stats options": "统计选项",
    "First allele only": "仅考虑第一个等位基因",
    "Depth distribution": "深度分布",
    "Depth min": "深度最小值",
    "Depth max": "最大深度",
    "Depth bin size": "深度区间大小",
    "Choose a reference genome": "选择一个参考基因组",
    "Split by ID (ignored on multiple inputs)": "按ID分割（在多个输入上忽略）",
    "User Tstv": "用户 Tstv",
    "Set af-bins": "设置af-bins",
    "List of allele frequency bins": "等位基因频率区间列表",
    "File listing the allele frequency bins one per line": "列出等位基因频率区间的文件，每个区间占一行",
    "Allele frequency tag to use, by default estimated from AN,AC or GT": "等位基因频率标签用于使用，默认情况下从AN、ANAC或GT估计",
    "Verbose": "详细",
    "Create a plots pdf with this title": "创建一个带有这个标题的图表PDF",
    "Roh General Options": "Roh 通用选项",
    "AF TAG": "AF标签",
    "AF Default": "AF 默认",
    "Gts Only": "Gts 仅此",
    "Skip Indels": "跳过插入和缺失",
    "Genetic Map": "遗传图谱",
    "Rec Rate": "复发率",
    "Sliding Buffer Size": "滑动缓冲区大小",
    "Sliding Buffer Overlap": "滑动缓冲区重叠",
    "Ignore homo-ref": "忽略同源参照",
    "Include sites with no ALT allele": "包含没有ALT等位基因的位点",
    "Hw To Az": "如何将数据转换为Azure",
    "Az To Hw": "A到Hw",
    "Viterbi Training": "维特比训练",
    "limit output": "限制输出",
    "Header": "标题",
    "New Sample names": "新样本名称",
    "Samples file": "样本文件",
    "Samples text entry": "样本文本输入",
    "Query Options": "查询选项",
    "Format for Output": "输出格式",
    "Output is tab-delimited": "输出是制表符分隔的",
    "Allow Undefined Tags": "允许未定义的标签",
    "Print Header Line": "打印标题行",
    "Isec Options": "Isec 选项",
    "Nfiles": "文件数",
    "Complement": "互补",
    "Genotypes to compare against": "与基因型进行比较",
    "Target Sample": "目标样本",
    "All Sites": "所有站点",
    "Homs Only": "仅Homs",
    "Plot": "绘图",
    "Snpgap": "Snpgap",
    "Indelgap": "插入缺失间隙",
    "Mode FILTER annotation": "模式 过滤 注释",
    "Perform soft filtering": "执行软过滤",
    "Soft Filter": "软过滤器",
    "Set genotypes of failed samples": "设置失败样本的基因型",
    "convert from": "从转换",
    "Tabular Variant Data": "表格变异数据",
    "Genomic VCF (GATK HaplotypeCaller)": "基因组 VCF (GATK HaplotypeCaller)",
    "The .gen file": ".gen 文件",
    "The .samples file": ".samples 文件",
    "The .hap file": ".hap 文件",
    "The .legend file": ".传奇文件",
    "Keep duplicates": "保留重复项",
    "convert to": "转换为",
    "tag to take values for .gen file: GT,PL,GL,GP": "标签用于取 .gen 文件的值：GT,PL,GL,GP",
    "3N6 format": "3N6格式",
    "Add Annotations": "添加注释",
    "Annotations File": "注释文件",
    "Annotations VCF": "注释 VCF",
    "Header Lines File": "头文件行",
    "Header Lines": "标题行",
    "Mark Sites TAG": "标记位点标签",
    "Minimum required overlap of intersecting regions": "相交区域的最小重叠",
    "Set Id": "设置 ID",
    "Change Annotations": "更改注释",
    "Remove annotations": "删除注释",
    "Rename CHROM": "重命名 CHROM",
    "Rename annotations": "重命名注释",
    "VCF file": "VCF 文件",
    "Optional input files": "可选输入文件",
    "Bref3 or VCF file with phased genotypes": "Bref3或具有相位基因型的VCF文件",
    "PLINK map file with cM units": "PLINK 遗传图谱文件，单位为 cM",
    "Samples to exclude": "排除样本",
    "Markers to exclude": "排除标记",
    "Specify a chromosome interval": "指定一个染色体区间",
    "Effective population size": "有效种群大小",
    "Window length in cM": "重组距离窗口长度（cM）",
    "Window overlap in cM": "cM中的窗口重叠",
    "Allele mismatch probability for the hidden Markov model": "隐马尔可夫模型中的等位基因错配概率",
    "Output a log file": "输出一个日志文件",
    "Phasing parameters": "分相参数",
    "Max burnin iterations": "最大烧入迭代次数",
    "Phasing iterations": "分相迭代",
    "Model states for phasing": "模型状态用于相位分析",
    "Imputation parameters": "插补参数",
    "Impute ungenotyped markers": "填补未基因分型的标记",
    "Model states for imputation": "模型状态用于插补",
    "Minimum cM length of haplotype segments": "最小cM长度的单体型段",
    "Length in cM for detecting short IBS segments": "用于检测短IBS段的cM长度",
    "Max cM in a marker cluster": "最大cM值在一个标记簇中",
    "Include posterior allele probabilities": "包含后验等位基因概率",
    "Include posterior genotype probabilities": "包含后验基因型概率",
    "Model type": "模型类型",
    "Select specific regions to process": "选择特定区域进行处理",
    "Generate genomic VCF (gVCF) output": "生成基因组VCF (gVCF) 输出",
    "Binary Genotype File": "二进制基因型文件",
    "Close Relative Inference": "近亲推断",
    "Relationship Inference": "关系推断",
    "Duplicate Analysis": "重复分析",
    "Pairwise Relative Inference": "成对相对推断",
    "Kinship Inference": "亲缘关系推断",
    "IBD Segment Analysis": "IBD 片段分析",
    "IBS Summary Statistics": "IBS 汇总统计",
    "Homogeneous Population": "同质群体",
    "Inference Parameter": "推断参数",
    "Degrees of relatedness": "亲缘度",
    "Minimum IBD segments": "最小IBD片段",
    "Relationship Application": "关系应用",
    "Unrelated Option": "不相关选项",
    "Cluster Parameter": "聚类参数",
    "Reconstruct Pedigree": "重建家谱",
    "QC Report": "质量控制报告",
    "Set the N callrate": "设置N的调用率",
    "Set the M callrate": "设置 M 呼叫率",
    "Population Structure": "种群结构",
    "Add PCA as dimension reduction algorithm": "添加PCA作为降维算法",
    "Add MDS as dimension reduction algorithm": "添加MDS作为降维算法",
    "Structure Parameter": "结构参数",
    "Number of Principal Components to use": "要使用的主成分数量",
    "Projection N": "投影 N",
    "Disease Association": "疾病关联",
    "Transmission/Disequilibrium Test": "传递/不平衡检验",
    "Quantitative Trait Association": "数量性状关联",
    "Score Test between SNP and quantitative trait": "单核苷酸多态性（SNP）与数量性状之间的得分测试",
    "Association Model": "关联模型",
    "Trait names": "性状名称",
    "Covariate names": "协变量名称",
    "Association Parameter": "关联参数",
    "Normal Transformation": "正规变换",
    "maximum P values": "最大 P 值",
    "Genetic Risk Score": "遗传风险评分",
    "Predict Disease Risk": "预测疾病风险",
    "GRS Risk Model file": "GRS 风险模型文件",
    "Disease prevalence": "疾病流行率",
    "No Flip": "不翻转",
    "Optional Input": "可选输入",
    "Family File": "家族文件",
    "Map File": "映射文件",
    "Pair number of the Sex Chromosome.": "性染色体的配对数量。",
    "Optional Parameter": "可选参数",
    "R Code and Plots": "R代码和图表",
    "R Code and Plots in PNG format": "R代码和PNG格式的图表",
    "Output data in PLINK format": "输出数据为PLINK格式",
    "Output a Log?": "输出一个日志？",
    "Structure of your input data": "输入数据的结构",
    "Sequencing dataset(s) - one per sample": "测序数据集（们）- 每个样本一个",
    "Partial sequencing datasets for your sample": "部分测序数据集为您的样本",
    "Remove reads longer than": "移除大于指定长度的读取序列",
    "Remove reads shorter than": "移除短于",
    "Eliminate reads with a mean base quality score of less than": "消除平均碱基质量得分小于的读段",
    "Which function of PharmCAT should be run?": "应该运行PharmCAT的哪个功能？",
    "Input vcf file": "输入vcf文件",
    "Gene call TSV file from an outside tool": "来自外部工具的基因调用TSV文件",
    "Output reporter JSON report": "输出报告员JSON报告",
    "Output phenotyper JSON report": "输出表型分析JSON报告",
    "Input json file": "输入的json文件",
    "Optional TSV of outside caller information": "可选的外部调用者信息的制表符分隔值（TSV）文件",
    "JSON call data output from Phenotyper pharmCAT function": "Phenotyper pharmCAT 函数输出的 JSON 调用数据",
    "text to add to the report title": "报告标题中要添加的文本",
    "Bam file": "Bam 文件",
    "Select output format": "选择输出格式",
    "Output variant score histogram?": "输出变异分数直方图？",
    "Output zygosity histogram?": "输出合子型直方图？",
    "Output quality histogram?": "输出质量直方图？",
    "Select source of annotation data": "选择注释数据的来源",
    "Select annotation cache file": "选择注释缓存文件",
    "Select GFF or GTF annotation file from history": "从历史记录中选择 GFF 或 GTF 注释文件",
    "Use FASTA file as reference sequence (optional)": "使用FASTA文件作为参考序列（可选）",
    "Select reference sequence": "选择参考序列",
    "Output SIFT prediction": "输出SIFT预测",
    "Output PolyPhen prediction": "输出PolyPhen预测",
    "Retrieve transcript or gene with nearest TSS to each input variant": "检索每个输入变异最近的转录本或基因的转录起始位点（TSS）。",
    "Modify the distance up and/or downstream between a variant and a transcript for which VEP will assign the upstream_gene_variant or downstream_gene_variant consequences": "修改VEP分配上游基因变异或下游基因变异后果时，变异与转录本之间的上游和/或下游距离。",
    "Enter custom upstream and downstream distance": "输入自定义的上游和下游距离",
    "Enter custom upstream distance": "输入自定义上游距离",
    "Enter custom downstream distance": "输入自定义下游距离",
    "Report only regulatory regions that are found in the given cell type(s):": "只报告在给定的细胞类型中发现的调控区域：",
    "Consider only alternate alleles present in the genotypes of the specified individual(s):": "只考虑在指定个体（们）的基因型中出现的替代等位基因：",
    "Change name of the consequences INFO column in VCF output": "在VCF输出中更改后果INFO列的名称",
    "Enter custom INFO key": "输入自定义INFO键",
    "Type of consequence terms to output": "输出后果术语的类型",
    "Shift variants to the right": "将变体向右移动",
    "Report the distance each variant has been shifted": "报告每个变体被移动的距离",
    "Output option flags": "输出选项标志",
    "Identifiers": "标识符",
    "Load a file of chromosome synonyms": "加载一个染色体同义词文件",
    "Identifier flags": "标识符标志",
    "Co-located variants": "共位变异",
    "Co-located variants flags": "共位变异标志",
    "Filtering and QC options": "过滤和质量控制选项",
    "Filtering and QC flags": "过滤和质量控制标志",
    "VEP Plugins": "VEP 插件",
    "Enable Carol plugin": "启用Carol插件",
    "Enable Condel plugin": "启用Condel插件",
    "Enable ExACpLI plugin by selecting an ExACpLI_values text file": "通过选择一个 ExACpLI_values 文本文件来启用 ExACpLI 插件",
    "Enable LoFtool plugin by selecting a LoFtool_scores text file": "启用LoFtool插件，通过选择一个LoFtool_scores文本文件。",
    "Reference Genome": "参考基因组",
    "Reference source": "参考来源",
    "Galaxy Built-in Reference(s)": "Galaxy 内置参考数据",
    "Fasta or Genbank Reference(s)": "Fasta 或 Genbank 参考序列",
    "Run Mode": "运行模式",
    "Fastq Read Files": "Fastq 读取文件",
    "Detection Mode": "检测模式",
    "Analysis Name": "分析名称",
    "Predict Junctions": "预测接头",
    "Output Formats": "输出格式",
    "GenomeDiff (gd) Files": "基因组差异（gd）文件",
    "VCF input file": "VCF 输入文件",
    "Select FASTA file as reference sequence": "选择FASTA文件作为参考序列",
    "Select the source of annotation data if you want to use VEP": "如果您想使用VEP（Variant Effect Predictor，变异效应预测器），请选择注释数据的来源。",
    "Enter tumor sample ID (optional)": "输入肿瘤样本ID（可选）",
    "Enter normal sample ID (optional)": "输入正常样本ID（可选）",
    "Enter name of tumor genotype column (optional)": "输入肿瘤基因型列的名称（可选）",
    "Enter name of normal genotype column (optional)": "输入正常基因型列的名称（可选）",
    "Include MAF version header in output file": "在输出文件中包含 MAF 版本标题",
    "Allow also mismatched variant alleles when reporting co-located variants": "允许在报告共位变异时也包括不匹配的变异等位基因",
    "Enter minimum allele fraction to call a variant homozygous if GT is undefined in VCF": "输入在VCF中，如果GT未定义，则调用一个变体为纯合子的最小等位基因分数。",
    "Enter variant calling center to report in MAF": "进入变体呼叫中心以在MAF中报告",
    "Enter comma-delimited names of INFO fields to retain as extra columns in MAF": "输入以逗号分隔的INFO字段名称，以保留为MAF中的额外列",
    "Enter comma-delimited names of FORMAT fields to retain as extra columns in MAF": "输入以逗号分隔的FORMAT字段名称，以保留为MAF中的额外列",
    "Enter comma-delimited names of VEP annotations (within the VEP CSQ/ANN) to retain as extra columns in MAF": "输入逗号分隔的VEP注释（在VEP CSQ/ANN中）以保留为MAF中的额外列",
    "Plugin Options": "插件选项",
    "Target genotypes to change": "目标基因型改变",
    "New genotypes to set": "新设定的基因型",
    "FASTA Alignment": "FASTA 对齐",
    "Determine reference sequence": "确定参考序列",
    "Name of sequence that should be used as reference sequence:": "应使用的参考序列的名称：",
    "Treat ambiguous bases": "处理模糊碱基",
    "Exclude sequences": "排除序列",
    "Exclude sequences from text file": "排除文本文件中的序列",
    "Maximum number of mismatches": "最大不匹配数",
    "Mask sites": "掩蔽位点",
    "Mask sites at given positions (VCF file)": "在给定位置（VCF文件）遮蔽位点",
    "Window size to mask bases": "窗口大小以掩盖碱基",
    "Minimum of ambiguous characters within the window given above": "窗口内模糊字符的最小值",
    "Include positions without defined ALT allele": "包含没有定义ALT等位基因的位置",
    "Output VCF options": "输出 VCF 选项",
    "Start offset": "开始偏移量",
    "Include the reference in the genotype columns": "在基因型列中包含参考序列",
    "Output 8-column VCF": "输出8列VCF",
    "Use this sequence for the CHROM column in the VCF": "在VCF文件的CHROM列中使用这个序列",
    "Genome FASTA file": "基因组FASTA文件",
    "Minimum gap size between accessible sequence regions": "最小可访问序列区域之间的间隔大小",
    "Additional regions to exclude": "额外需要排除的区域",
    "input BED or interval file": "输入 BED 或间隔文件",
    " Regions of accessible sequence on chromosomes": "染色体上可访问序列的区域",
    "Average size of antitarget bins": "反目标区间的平均大小",
    "Minimum size of antitarget bins": "反目标箱的最小大小",
    "Input Sample BAM file/s": "输入样本BAM文件/s",
    "Input Potentially targeted genomic regions": "输入潜在目标基因组区域",
    "Build or reuse a copy number reference file (.cnn)?": "构建或重用一份拷贝数参考文件（.cnn）？",
    "CN segmentation calls cns file": "CN 分段调用 cns 文件",
    "additional SNP b_allele frequencies process": "额外的SNP b等位基因频率处理过程",
    "Add filters": "添加过滤器",
    "Interval BED file": "区间 BED 文件",
    "Get read depths by counting read midpoints within each bin": "通过计算每个区间内读取中点的数量来获取读取深度",
    "Minimum mapping quality score to count a read for coverage depth": "用于计算覆盖深度的最小映射质量分数",
    "Bin-Level log2 Ratios/Coverages cnr file": "二进制级别对数比率/覆盖度 cnr 文件",
    "Plot aesthetics": "绘图美学",
    "Sample Target coverage cnn file": "样本目标覆盖率CNN文件",
    "Sample Antitarget coverage cnn file": "样本反目标覆盖cnn文件",
    "Reference coverage cnn file": "参考覆盖度CNN文件",
    "Compare and use cluster-specific values present in the reference profile": "比较并使用参考档案中存在的特定于集群的值",
    "Sample ID for target/antitarget files": "目标/非目标文件的样本ID",
    "Skip GC correction": "跳过垃圾回收校正",
    "Skip edge-effect correction": "跳过边缘效应校正",
    " Skip RepeatMasker correction": "跳过RepeatMasker校正",
    "CN segmentation calls cns file/s": "CN 分段调用 cns 文件/s",
    "Build or reuse a copy number reference file cnn?": "构建或重用一份拷贝数参考文件 cnn？",
    "Bias corrections settings": "偏差校正设置",
    "Sample Target/antitarget Coverage cnn file": "样本目标/非目标覆盖率CNN文件",
    "additional process for SNP b_allele frequencies": "额外的SNP等位基因频率处理过程",
    "capture/target BED file": "捕获/目标 BED 文件",
    "Use gene models from this file to assign names to the target regions": "使用此文件中的基因模型为靶区域分配名称",
    "Reduce multi-accession bait labels to be short and consistent": "将多访问号的诱饵标签简化为简短且一致的格式",
    "Split large tiled intervals into smaller, consecutive targets": "将大的平铺间隔分割成更小的、连续的目标",
    "Average size of split target bins": "平均分割目标箱的大小",
    "Consensus": "共识",
    "Input BAM": "输入BAM文件",
    "Consensus-to-reference alignment": "共识序列到参考序列的比对",
    "Chain file": "链文件",
    "Reference data selection": "参考数据选择",
    "Genes GFF": "基因GFF",
    "Coding sequence containing ribosomal slippage site": "含有核糖体滑动位点的编码序列",
    "Output format options": "输出格式选项",
    "Write Summary Diagnosis?": "写总结诊断？",
    "Use 0-based Sequence Positions?": "使用0为基数的序列位置？",
    "Single 'normal' or 'normal vs tumor' analysis": "单样本“正常”或“正常与肿瘤”比较分析",
    "select normal BAM": "选择正常BAM",
    "select tumor BAM": "选择肿瘤BAM文件",
    "Additional parameters": "附加参数",
    "Set default task memory requirements": "设置默认任务内存需求",
    "Set maximum sequence region size": "设置最大序列区域大小",
    "Do you want to change default configuration settings?": "您想要更改默认配置设置吗？",
    "config file": "配置文件",
    "minCandidateVariantSize": "最小候选变异大小",
    "rnaMinCandidateVariantSize": "RNA最小候选变异大小",
    "minEdgeObservations": "最小边缘观测数",
    "graphNodeMaxEdgeCount": "图节点最大边数",
    "minCandidateSpanningCount": "最小候选跨越计数",
    "minScoredVariantSize": "最小得分变异大小",
    "minDiploidVariantScore": "最小二倍体变异分数",
    "minPassDiploidVariantScore": "最小通过二倍体变异分数",
    "minPassDiploidGTScore": "最小通过二倍体基因型得分",
    "minSomaticScore": "最小体细胞分数",
    "minPassSomaticScore": "最小体细胞得分",
    "enableRemoteReadRetrievalForInsertionsInGermlineCallingModes": "启用远程读取检索以在胚系调用模式中插入",
    "enableRemoteReadRetrievalForInsertionsInCancerCallingModes": "启用癌症呼叫模式中插入的远程读取检索",
    "useOverlapPairEvidence": "使用重叠配对证据",
    "output conf file": "输出配置文件",
    "Unfiltered structural variants": "未过滤的结构变异",
    "Unfiltered small indel candidates": "未过滤的小片段插入或缺失候选",
    "Score-filtered variants in diploid model": "在二倍体模型中的得分筛选变异",
    "CSV files": "CSV文件",
    "Phenopacket JSON files": "Phenopacket JSON文件",
    "Annotated vcf file": "注释过的vcf文件",
    "Dataset ID": "数据集ID",
    "CN segmentation calls cnn file": "CN 分段调用 cnn 文件",
    "Minimum propes": "最小支持度",
    "segmentation calls": "分割调用",
    "Min Number of covered probes": "覆盖探针的最小数量",
    "threshold": "阈值",
    "Assume inputs normalised to a male reference": "假设输入已标准化至男性参考",
    "Drop very-low-coverage bins": "删除极低覆盖度的bin",
    "Statistical settings": "统计设置",
    "Bin-Level log2 Ratios/Coverages cnn/cnr file": "Bin级别log2比率/覆盖度cnn/cnr文件",
    "Drop very-low-coverage bins before segmentation": "在分段之前删除非常低覆盖度的箱子",
    "Bin-Level log2 Ratios/Coverages cnr/cnn file": "二进制级别对数比值/覆盖度 cnr/cnn 文件",
    "MALE REFERENCE": "男性参考",
    "CNV VCF file": "CNV VCF文件",
    "Metadata file": "元数据文件",
    " Root sequence": "根序列",
    "Root tree using": "使用根树",
    "Use pairwise deletion of gaps/missing data": "使用成对缺失数据的间隙删除方法",
    "Change number of bootstrap replicas": "更改自举复制品的数量",
    "Major-alleles-only mode": "仅主要等位基因模式",
    "Input allele counts table": "输入等位基因计数表",
    "Also generate a report on data spread": "同时生成一个关于数据分布的报告",
    "Table containing major and minor alleles base per position": "包含每个位置的主要和次要等位基因碱基的表格",
    "Background sequence length": "背景序列长度",
    "File to process": "待处理文件",
    "AWK Program": "AWK 程序",
    "Datasets to concatenate": "要合并的数据集",
    "File to cut": "要切割的文件",
    "Operation": "操作",
    "Delimited by": "以分隔",
    "Cut by": "剪切",
    "List of Fields": "字段列表",
    "List of characters": "字符列表",
    "List of Bytes": "字节列表",
    "1st file": "第一个文件",
    "Column to use from 1st file": "从第一个文件中使用的列",
    "2nd File": "第二个文件",
    "Column to use from 2nd file": "从第二个文件中使用的列",
    "Output lines appearing in": "输出出现在",
    "First line is a header line": "第一行是标题行",
    "Value to put in unpaired (empty) fields": "填入未配对（空）字段的值",
    "Find pattern": "查找模式",
    "Replace with": "替换为",
    "Find-Pattern is a regular expression": "Find-Pattern 是一个正则表达式",
    "Replace all occurences of the pattern": "替换所有出现的模式",
    "Case-Insensitive search": "不区分大小写的搜索",
    "Find whole-words": "查找完整单词",
    "Ignore first line": "忽略第一行",
    "Find and Replace text in": "在文本中查找并替换",
    "in column": "在列",
    "Select lines from": "从...选择行",
    "that": "那个",
    "Type of regex": "正则表达式类型",
    "Match type": "匹配类型",
    "Show lines preceding the matched line": "显示匹配行之前的行",
    "Show lines trailing the matched line": "显示匹配行之后的行",
    "File to select": "选择文件",
    "Number of lines": "行数",
    "File to join": "要连接的文件",
    "add additional file": "添加额外的文件",
    "Common key column": "公共键列",
    "Column with values to preserve": "保留值的列",
    "Add header line to the output file": "在输出文件中添加标题行",
    "Input files contain a header line (as first line)": "输入文件包含一个标题行（作为第一行）",
    "Ignore duplicated keys": "忽略重复的键",
    " selection": "选择",
    "Characters to insert": "插入字符",
    "Specify the number of iterations by": "指定迭代次数",
    "How many times?": "多少次？",
    "Template file": "模板文件",
    "Replacement": "替换",
    "Replace with:": "替换为：",
    "SED Program": "流编辑程序",
    "Operation mode": "操作模式",
    "Sort Query": "排序查询",
    "Number of header lines": "标题行数",
    "Column selections": "列选择",
    "on column": "在列上",
    "in": "在",
    "Flavor": "风味",
    "Output unique values": "输出唯一值",
    "Tabular file that should be sorted": "应排序的表格文件",
    "File to scan for unique values": "扫描以查找唯一值的文件",
    "Do you want to group each unique group?": "您是否希望对每个唯一的组进行分组？",
    "Counting number of occurrences": "计数出现次数",
    "Only print duplicate lines": "只打印重复的行",
    "Only print unique lines": "只打印唯一的行",
    "Output all lines, and delimit each unique group": "输出所有行，并用分隔符隔开每个唯一组",
    "Ignore differences in case when comparing": "在比较时忽略大小写差异",
    "Avoid comparing the first N fields": "避免比较前N个字段",
    "Do you want to use a separator other than newline?": "您是否希望使用除换行符以外的分隔符？",
    "Attach the separator before instead of after": "在之前而不是之后附加分隔符",
    "Interpret the separator as a regular expression": "将分隔符解释为正则表达式",
    "Separator to use": "要使用的分隔符",
    "File to unfold": "展开文件",
    "Column to use for unfolding": "用于展开的列",
    "Values in column are delimited by": "列中的值由分隔",
    "Column only contains numeric values": "列只包含数值",
    "Column start": "列起始",
    "Column end": "列结束",
    "Input has a header line with column names?": "输入包含带有列名的标题行？",
    "The new column name": "新列名",
    "Avoid scientific notation in any newly computed columns": "在任何新计算的列中避免使用科学记数法",
    "Error handling": "错误处理",
    "Autodetect column types": "自动检测列类型",
    "Fail on references to non-existent columns": "失败在引用不存在的列上",
    "If an expression cannot be computed for a row": "如果一个表达式不能为一行计算",
    "Replacement value": "替换值",
    "First input file": "第一个输入文件",
    "Second input file": "第二个输入文件",
    "Generate HTML report": "生成HTML报告",
    "Choose report output format": "选择报告输出格式",
    "File in which you want to replace some values": "要替换其中一些值的文件",
    "Replace information file": "替换信息文件",
    "Which column should be replaced?": "哪一列应该被替换？",
    "Skip this many starting lines": "跳过这么多开始的行",
    "When an unknown value is encountered": "当遇到一个未知值时",
    "Do not perform replacement on lines starting with": "不要对以...开始的行进行替换",
    "text to add": "要添加的文本",
    "input file": "输入文件",
    "Options": "选项",
    "to Dataset": "到数据集",
    "input contains a header line?": "输入包含一个标题行吗？",
    "Prepend the colum": "在列前添加",
    "Select data": "选择数据",
    "Merge column": "合并列",
    "with column": "带有列",
    "Columns": "列",
    "Add column": "添加列",
    "Input Single or Multiple Tables": "输入单个或多个表格",
    "Type of table operation": "表格操作类型",
    "Duplicate Indices": "重复索引",
    "Filter criterion": "过滤标准",
    "Keep column/row if its observed": "保留观察到的列/行",
    "is": "是",
    "this value": "这个值",
    "Keep column/row if at least": "保留列/行，如果至少",
    "of its element are": "它的元素是",
    "Calculate": "计算",
    "Custom function on 'vec'": "自定义函数应用于 'vec'",
    "For each": "对于每个",
    "Variable IDs": "变量ID",
    "Unpivoted IDs": "非转置的ID",
    "Index": "索引",
    "Values": "值",
    "Aggregator Function": "聚合函数",
    "Column-Function Mapping": "列-函数映射",
    "Value Column": "值列",
    "Function": "功能",
    "Custom expression on 'table', along 'axis' (0 or 1)": "在“表格”上自定义表达式，沿“轴”（0或1）",
    "Operation to perform": "要执行的操作",
    "Transformation function": "转换函数",
    "Second operand value": "第二个操作数的值",
    "Custom expression on 'elem'": "自定义表达式在 'elem' 上",
    "Operate on elements": "操作元素",
    "Inclusive Selection": "包容性选择",
    "Tables": "表格",
    "Custom expression on 'tableN'": "自定义表达式在 'tableN' 上",
    "Output formatting options": "输出格式选项",
    "Output decimal precision": "输出小数精度",
    "Check": "检查",
    "Find Regex": "查找正则表达式",
    "Select cells from": "选择来自",
    "using column": "使用列",
    "2nd file": "第二个文件",
    "Does the input files contain a header line": "输入文件是否包含标题行",
    "Add an addional column with the calculated distance.": "添加一列，包含计算出的距离。",
    "Choose the mode of merging.": "选择合并模式。",
    "Choose the metrics of your distance": "选择您的距离度量",
    "Allowed distance between the two values that will trigger a merge": "允许两个值之间触发合并的距离",
    "Find and Replace": "查找和替换",
    "Add tables to an existing database": "将表格添加到现有数据库中",
    "Add tables to this Database": "将表添加到此数据库",
    "Database Table": "数据库表",
    "Tabular Dataset for Table": "表格数据集",
    "Filter Dataset Input": "过滤数据集输入",
    "Table Options": "表格选项",
    "Specify Name for Table": "指定表格名称",
    "Use first line as column names": "使用第一行作为列名",
    "Specify Column Names (comma-separated list)": "指定列名（用逗号分隔的列表）",
    "Only load the columns you have named into database": "只加载您已命名的列到数据库中",
    "Add an auto increment primary key column with this name": "添加一个名为该名称的自增主键列",
    "Table Index": "表索引",
    "This is a unique index": "这是一个唯一的索引",
    "Index on Columns": "索引列",
    "Modify the database": "修改数据库",
    "Database Manipulation SQL Statements": "数据库操作 SQL 语句",
    "SQL Query to modify the database": "SQL查询以修改数据库",
    "Save the sqlite database in your history": "将 SQLite 数据库保存到你的历史记录中。",
    "Additional Queries": "附加查询",
    "SQL Query": "SQL查询",
    "Columns on which to perform the replacement": "要进行替换的列",
    "Mapping table": "映射表",
    "Comment character": "注释字符",
    "SQLite Database": "SQLite数据库",
    "SQL query": "SQL查询",
    "Tabular Dataset to filter": "表格数据集过滤",
    "exclude lines starting with #": "排除以#开头的行",
    "JSON Input": "JSON 输入",
    "jq filter": "jq 过滤器",
    "Convert output to tabular": "将输出转换为表格",
    "Input should be molten data": "输入应该是熔化数据",
    "Input should have column headers - these will be the variable IDs that are summarized": "输入应包含列标题 - 这些将是被汇总的变量ID",
    "Parent GFF3 annotations": "父级 GFF3 注释",
    "Child GFF3 annotations to rebase against parent": "将子GFF3注释与父注释进行比对重新定位",
    "Interpro specific modifications": "Interpro 特定修饰",
    "Map protein translated results to original DNA data": "将翻译后的蛋白质结果映射回原始的DNA数据",
    "Tabular file": "表格文件",
    "Select Columns": "选择列",
    "Header name": "标题名称",
    "Keep named columns": "保留命名的列",
    "Characters to strip when doing name comparison in first column": "在第一列进行名称比较时需要去除的字符",
    "Identifier column": "标识符列",
    "What portion of": "什么部分",
    "Is covered by": "被覆盖",
    "Input File": "输入文件",
    "Search String": "搜索字符串",
    "Replace String": "替换字符串",
    "Multi-line": "多行",
    "Ignore Case": "忽略大小写",
    "Dot All": "点全部",
    "Replace Count(0=all)": "替换计数(0=全部)",
    "Select the first input tabular file": "选择第一个输入表格文件",
    "Select the second input tabular file": "选择第二个输入表格文件",
    "Choose the arithmetic operation:": "选择算术运算：",
    "Merge overlapping regions of": "合并重叠区域",
    "Output 3 column bed": "输出3列的bed文件",
    "Return": "返回",
    "that intersect": "相交",
    "for at least": "至少",
    "For every interval in": "对于每个间隔在",
    "Fetch closest feature(s) from": "获取最近的基因特征",
    "Located": "位于",
    "Region": "区域",
    "Location of the flanking region/s": "侧翼区域的位置",
    "Length of the flanking region(s)": "相邻区域的长度",
    "Subtract": "减去",
    "from": "从",
    "Restrict subtraction between 'begin column'": "限制在“开始列”之间的减法操作",
    "and 'end column'": "和“结束列”",
    "Ignore empty columns and whitespace at end of line when subtracting": "忽略空列和行尾空白字符时进行相减",
    "from query": "从查询",
    "ignore differences in case when comparing": "忽略比较时的大小写差异",
    "column only contains numeric values": "列仅包含数值",
    "Add this value": "添加这个值",
    "Iterate?": "迭代？",
    "What type of data do you wish to concatenate?": "您希望合并哪种类型的数据？",
    "Concatenate Datasets": "合并数据集",
    "Input paired collection to concatenate": "将配对的集合输入以进行连接",
    "What type of concatenation do you wish to perform?": "您希望执行哪种类型的连接？",
    "Input first collection": "输入第一个集合",
    "Input second collection": "输入第二个集合",
    "Input nested collection": "输入嵌套集合",
    "Include dataset names?": "包含数据集名称？",
    "Number of lines to skip at the beginning of each concatenation:": "每个连接开始时需要跳过的行数：",
    "Is there a header for the data's columns ?": "数据列有标题吗？",
    "File to number": "文件编号",
    "Line number increment at each line": "每行增加行号",
    "First line number": "第一行编号",
    "Line number format": "行号格式",
    "Number of characters for line numbers": "行号字符数",
    "VCF dataset to filter": "VCF 数据集过滤",
    "more filters": "更多过滤器",
    "Select the filter type": "选择过滤类型",
    "Specify filtering value": "指定过滤值",
    "Filter entire records, not just alleles": "过滤整个记录，而不仅仅是等位基因",
    "Tag vcf records as positively filtered with this tag, print all records": "标记 VCF 记录为通过此标签的正向过滤，并打印所有记录。",
    "Tag vcf records as negatively filtered with this tag, print all records": "将 VCF 记录标记为负向过滤，并打印所有记录。",
    "Append the existing filter tag, don't just replace it": "追加现有过滤标签，不要只是替换它",
    "Apply --tag-pass on a per-allele basis, adds or sets the corresponding INFO field tag": "对每个等位基因应用 --tag-pass，在相应的 INFO 字段中添加或设置对应的标签",
    "Inverts the filter, e.g. grep -v": "反转过滤器，例如 grep -v",
    "Use logical OR instead of AND to combine filters": "使用逻辑“或”而不是“与”来组合过滤器",
    "Specify a region on which to target the filtering": "指定一个区域以针对过滤",
    "Add INFO fields to": "添加INFO字段到",
    "using INFO fields from": "使用来自INFO字段的",
    "Select VCF dataset to convert": "选择 VCF 数据集进行转换",
    "Report data per sample": "按样本报告数据",
    "Fill empty fields with": "用填充空字段",
    "VCF dataset to left-align": "VCF数据集左对齐",
    "Use a window of this many bp when left-aligning": "使用这么多碱基对的窗口进行左对齐",
    "Select VCF Datasets": "选择 VCF 数据集",
    "Select type of calculation:": "选择计算类型：",
    "In this VCF dataset": "在这个 VCF 数据集中",
    "Select random sites from": "从随机选择位点",
    "base sampling probability per locus": "每个位点的基准采样概率",
    "use this random seed": "使用这个随机种子",
    "Advanced controls": "高级控制",
    "scale sampling likelihood by this INFO field": "按此INFO字段缩放采样可能性",
    "BED dataset or an interval to intersect with": "BED数据集或要相交的区间",
    "Select BED dataset": "选择 BED 数据集",
    "Enter interval string": "输入区间字符串",
    "Invert selection?": "反选？",
    "compare records up to this many bp away": "比较距离不超过这么多碱基对的记录",
    "What to report?": "报告什么？",
    "Retain MNPs as separate events": "保留多核苷酸多态性（MNPs）作为独立事件",
    "Tag records which are split apart of a complex allele with this flag.": "标记这个标志的记录，它们是复杂等位基因分裂开的部分。",
    "Do not manipulate records in which either the ALT or REF is longer than (bp)": "不要操作ALT或REF任一超过(bp)长度的记录。",
    "Maintain site and allele-level annotations when decomposing": "在分解时保留位点和等位基因级别的注释",
    "Maintain genotype-level annotations when decomposing": "在分解时保留基因型级别的注释",
    "Break or Create?": "断裂还是创建？",
    "The first VCF dataset": "第一个 VCF 数据集",
    "The second VCF dataset": "第二个 VCF 数据集",
    "Union or intersection": "并集或交集",
    "compare records up to this many bp away (window size)": "比较距离这么多碱基对以内的记录（窗口大小）",
    "output whole loci when one alternate allele matches": "当一个替代等位基因匹配时，输出整个位点",
    "Enter additional command line options described in the help section below": "输入在下面帮助部分描述的附加命令行选项",
    "VCF to be annotated": "待注释的VCF文件",
    "annotate the start and end as well as the interval itself?": "注释起始点、终止点以及间隔本身？",
    "annotate with an overlapping variant even it doesn't share the same ref and alt alleles.": "使用重叠的变异进行注释，即使它不共享相同的参考和备选等位基因。",
    "Input annotations": "输入注释",
    "Fields": "字段",
    "Op": "操作",
    "Select BED dataset to itersect with": "选择要与交集的 BED 数据集",
    "Intersecting VCF records will be prefixed with this TAG within the INFO field": "在INFO字段中，交集VCF记录将以此标签为前缀",
    "First dataset": "第一个数据集",
    "Second dataset": "第二个数据集",
    "Annotate genotypes in": "在中注释基因型",
    "using genotypes from": "使用基因型来自",
    "Mark genotypes added to the first dataset with this tag": "标记添加到第一个数据集中的基因型，使用这个标签",
    "Check to keep samples; Uncheck to remove": "勾选以保留样本；未勾选以删除",
    "SAM/BAM dataset": "SAM/BAM 数据集",
    "Select fasta reference": "选择fasta参考序列",
    "Counter configuration: [ococo16]": "计数器配置：[ococo16]",
    "Strategy for updates: [majority]": "更新策略：[多数]",
    "Save output(s)": "保存输出",
    "Minimum mapping quality score": "最小映射质量分数",
    "Initial nucleotides counter": "初始核苷酸计数器",
    "Minimum coverage required": "所需最小覆盖度",
    "Majority consensus threshold [0.51]": "多数一致性阈值 [0.51]",
    "Input variants from Naive Variants Detector": "从Naive Variants Detector输入变异",
    "Minor allele frequency threshold": "次要等位基因频率阈值",
    "Coverage threshold": "覆盖度阈值",
    "Do not filter sites or alleles": "不要过滤位点或等位基因",
    "Output stranded base counts": "输出有方向性的碱基计数",
    "Write header line": "写入标题行",
    "PRNG seed": "伪随机数生成器种子",
    "Complement intersection": "补集交集",
    "Datasets to Merge": "要合并的数据集",
    "Datasets to Compare": "数据集比较",
    "Dataset name": "数据集名称",
    "Comparison window": "比较窗口",
    "Ignore indels": "忽略插入和缺失",
    "Input": "输入",
    "Tag name": "标签名称",
    "Tag description": "标签描述",
    "Filter name": "过滤名称",
    "Filter condition": "过滤条件",
    "Input dataset": "输入数据集",
    "Remove alternate alleles if not found in the subset": "如果未在子集中找到，则删除替代等位基因",
    "Exclude rows not containing variants": "排除不包含变异的行",
    "Sample Sex file": "样本性别文件",
    "tags": "标签",
    "Phased": "相位化",
    "Major": "主要",
    "Trio File": "三联体文件",
    "Action": "动作",
    "TAGs to set": "要设置的标签",
    "VCF tag to determine the dosage from [PL,GL,GT]": "VCF标签用于确定[PL,GL,GT]中的剂量",
    "Tag conversion": "标签转换",
    "Drop the source tag": "删除源标签",
    "Threshold for GP to GT hard-call": "GP到GT硬调用的阈值",
    "Select how to split the spots": "选择如何分割斑点",
    "Dump only biological reads": "仅转储生物学读取",
    "Minimum spot ID": "最小斑点ID",
    "Maximum spot ID": "最大斑点ID",
    "Split spot by read pairs": "将测序点按读段对进行拆分",
    "filter by value": "按值过滤",
    "Filter by spot-groups": "按斑点组过滤",
    "Apply left and right clips": "应用左和右剪切",
    "Table name within cSRA object": "cSRA对象中的表名",
    "select output format": "选择输出格式",
    "select input type": "选择输入类型",
    "sra archive": "SRA档案",
    "SRR accession": "SRR登录号",
    "text file": "文本文件",
    "Prediction": "预测",
    "prediction method": "预测方法",
    "peptide lengths for prediction": "预测的肽段长度",
    "proteasome type": "蛋白酶体类型",
    "window_size": "窗口大小",
    "Peptide sequences": "肽序列",
    "Peptide Sequence Fasta": "肽段序列 Fasta",
    "Peptide Sequence Tabular": "肽段序列表格",
    "Select column with peptides": "选择包含肽段的列",
    "Select column with name": "选择具有名称的列",
    "Peptide Sequence": "肽段序列",
    "Download from": "从...下载",
    "Proteome Set": "蛋白质组集合",
    "Include isoform data": "包含异构体数据",
    "Proteome for body site": "蛋白质组对于身体部位",
    "Human Oral Microbiome Proteome": "人类口腔微生物组蛋白质组",
    "URL (http, ftp) of Fasta sequences": "Fasta序列的URL（http, ftp）",
    "Fasta source compression type": "Fasta源压缩类型",
    "Select source for IDs": "选择ID的来源",
    "Accession File": "访问文件",
    "ID List": "ID列表",
    "Molecule Type": "分子类型",
    "File Format": "文件格式",
    "Range": "范围",
    "How to handle download failures": "如何处理下载失败",
    "Common Organisms": "常见生物",
    "filter by reviewed status": "根据审核状态进行筛选",
    "Search ID values": "搜索 ID 值",
    "Dataset (tab separated) with ID column": "数据集（制表符分隔），含ID列",
    "Column with ID": "带有ID的列",
    "uniprot output format": "UniProt 输出格式",
    "Scientific name of the species": "物种的学名",
    "Data source to get data from": "数据来源，用于获取数据",
    "Number of records to return": "返回记录数",
    "Unipept application": "Unipept 应用程序",
    "Peptides input format": "肽段输入格式",
    "Peptide Input": "肽段输入",
    "Tabular Input Containing Peptide column": "包含肽段列的表格输入",
    "Peptide Fasta Input": "肽段Fasta输入",
    "mzIndetML Input": "mzIndetML 输入",
    "Match input peptides by": "匹配输入的肽段",
    "Show tryptic_match as:": "显示 tryptic_match 为：",
    "Choose outputs": "选择输出",
    "Exit with error on invalid peptides, otherwise ignore them": "在无效的肽段上退出并报错，否则忽略它们",
    "Search term": "搜索词",
    "Filter existing history": "过滤现有的历史记录",
    "Store results to history server": "将结果存储到历史服务器",
    "Filter by date": "按日期筛选",
    "Date type": "日期类型",
    "In past N days": "在过去的N天里",
    "Minimum date": "最小日期",
    "Maximum date": "最大日期",
    "Search Term": "搜索词",
    "Input method": "输入方法",
    "Citation table": "引用表",
    "Journal Title": "期刊标题",
    "Year": "年份",
    "Volume": "体积",
    "First Page": "第一页",
    "Author's Name": "作者姓名",
    "Citation key": "引用键",
    "From NCBI Database": "从NCBI数据库",
    "ELink command to execute": "ELink 命令执行",
    "To NCBI Database": "到NCBI数据库",
    "Input file with IDs": "输入带有ID的文件",
    "ID column": "ID 列",
    "Do you want to map IDs or retrieve data from UniProt": "您想要映射ID还是从UniProt检索数据",
    "Choose format of output file": "选择输出文件的格式",
    "List of Ensembl IDs": "Ensembl ID列表",
    "expand_3prime": "扩展3'端",
    "expand_5prime": "扩展5'端",
    "Ensembl ID": "Ensembl ID",
    "Output the sequences also in CIGAR format": "输出序列时也使用CIGAR格式",
    "NH format": "NH格式",
    "Select a query": "选择一个查询",
    "Input gene": "输入基因",
    "Input phenotype or set of phenotypes": "输入表型或一组表型",
    "Input gene or set of genes": "输入基因或一组基因",
    "Include genes without identified phenotypes?": "包含未鉴定表型的基因？",
    "Input pipeline": "输入管道",
    "Input ID": "输入ID",
    "Expand the search to include any connected features": "将搜索范围扩大以包括任何连接的特征",
    "Use free text search?": "使用自由文本搜索？",
    "Query": "查询",
    "Output Format": "输出格式",
    "Starting record to return (--retstart)": "开始返回记录（--retstart）",
    "Maximum number of records to return (--retmax)": "返回的最大记录数（--retmax）",
    "Enter Query IDs by...": "输入查询ID...",
    "ID File (Text)": "ID 文件（文本）",
    "ID File (XML)": "ID 文件（XML格式）",
    "ID File (JSON)": "ID文件（JSON格式）",
    "Paste ID List": "粘贴ID列表",
    "Target History": "目标历史",
    "Select an existing history": "选择一个现有的历史记录",
    "History File (JSON)": "历史文件（JSON格式）",
    "History File (XML)": "历史文件（XML）",
    "WebEnv History ID": "WebEnv 历史记录 ID",
    "Database Information Request Type": "数据库信息请求类型",
    "Ids for records to return": "要返回的记录的ID",
    "Taxonomy result to return": "返回分类学结果",
    "Fields to return": "返回的字段",
    "Identifiant of the run in EBI Metagenomics": "欧洲生物信息学研究所 (EBI) 宏基因组项目的运行标识符",
    "Type of information to download": "要下载的信息类型",
    "Sequences to download": "要下载的序列",
    "Quality control statistics to download": "下载质量控制统计信息",
    "Taxonomic analysis to download": "分类学分析下载",
    "Functional analysis to download": "功能分析以下载",
    "What would you like to do?": "您想要做什么？",
    "EGA Dataset Accession ID": "欧洲基因组档案（EGA）数据集访问编号",
    "EGA File Accession Identifier": "EGA文件访问编号",
    "Request a specific Genomic range?": "请求特定的基因组范围？",
    "Reference Sequence Name": "参考序列名称",
    "Start Position": "起始位置",
    "End Position": "结束位置",
    "Table with IDs to download": "要下载的ID表格",
    "Column containing the file IDs": "包含文件ID的列",
    "Request a specific Genomic range? (will be applied to ALL requested files)": "请求特定的基因组范围？（将应用于所有请求的文件）",
    "Output the log file?": "输出日志文件？",
    "Choose how to find genes to download": "选择查找要下载的基因的方法",
    "Species for gene symbol": "基因符号对应的物种",
    "Limit gene sequences and annotation report file to specified taxon": "将基因序列和注释报告文件限制为指定的分类群",
    "Length of flanking nucleotides": "侧翼核苷酸的长度",
    "Filters and Limit": "过滤和限制",
    "Filter protein and RNA sequences by RefSeq nucleotide and protein accessions": "过滤蛋白质和RNA序列，通过RefSeq核苷酸和蛋白质访问号",
    "Apply filter": "应用过滤器",
    "RefSeq nucleotide and protein accessions": "RefSeq核苷酸和蛋白质访问号",
    "Dataset with list of RefSeq nucleotide and protein accessions": "数据集包含RefSeq核苷酸和蛋白质访问号的列表",
    "Kingdom": "界",
    "Decompress FASTA": "解压缩FASTA",
    "Choose how to find genomes to download": "选择查找要下载的基因组的方法",
    "Exclude sub-species when a species-level taxon is specified": "当指定物种级分类单元时，排除亚种",
    "Limit to reference and representative (GCF_ and GCA_) assemblies": "限制为参考和代表性（GCF_ 和 GCA_）组装",
    "Assembly version(s)": "组装版本（s）",
    "Exclude atypical assemblies": "排除非典型组装",
    "Add search terms": "添加搜索词",
    "Only include genomes that have the specified text in the searchable fields": "只包含在可搜索字段中包含指定文本的基因组",
    "Database to query": "要查询的数据库",
    "Query keywords/BioProject ID": "查询关键词/BioProject ID",
    "Metagenomic data": "宏基因组数据",
    "Scientific name of the sample organism": "样本生物的学名",
    "Platform": "平台",
    "Library layout": "库布局",
    "Library preparation strategy": "文库制备策略",
    "Library selection": "文库选择",
    "Accession number": "访问号",
    "Sample size": "样本大小",
    "Publication date": "发布日期",
    "Level of search result details": "搜索结果详情级别",
    "Maximum number of entries": "最大条目数",
    "Additional output files": "附加输出文件",
    "Taxon ID": "分类群 ID",
    "Input latitude min (+north/-south):": "输入纬度最小值（+北/-南）：",
    "Input latitude max (+north/-south):": "输入纬度最大值（+北/-南）：",
    "Input longitude min (+east/-west):": "输入经度最小值（+东/-西）：",
    "Input longitude max (+east/-west):": "输入经度最大值（+东/西）：",
    "We have preconfigured some mode of operations for you. What mode do you want to use?": "我们已经为您预配置了一些操作模式。您想使用哪种模式？",
    "How do you want to select your data of interest ?": "您希望如何选取您感兴趣的数据？",
    "Input pressure min (db)": "输入压力最小值（分贝）",
    "Input pressure max (db)": "输入压力最大值（分贝）",
    "Input starting date": "输入开始日期",
    "Input ending date": "输入结束日期",
    "Input WMO number": "输入世界气象组织编号",
    "Input profile cycle number": "输入概要文件循环编号",
    "Which kind of dataset do you want ?": "您想要哪种类型的数据集？",
    "Choose which variable you want to study": "选择您想要研究的变量",
    "Choose which variables cannot be NaN and must return value": "选择哪些变量不能为NaN（非数字），并且必须返回值",
    "Which variables to add": "要添加哪些变量",
    "The key of the column for the latitude": "纬度列的键",
    "The key of the column for the longitude": "经度列的键",
    "A tolerance on the latitude/longitude for the request point": "请求点的纬度/经度容差",
    "The key of the column for the depth": "列深度的键",
    "The key of the column for the observation time": "观察时间的列键",
    "Verbose in the standard output of the tool": "在工具的标准输出中详细",
    "Filter metagenome assembled genomes (MAGs)": "过滤宏基因组组装的基因组（MAGs）",
    "Action to execute": "执行的操作",
    "Idempotent submissions: only add the elements that were not submitted before": "幂等提交：只添加之前未提交过的元素",
    "Source file": "源文件",
    "File to split": "要分割的文件",
    "Include header in splits?": "在拆分中包含头部？",
    "Select the file type to split": "选择要拆分的文件类型",
    "Tabular file to split": "制表符分隔的文件分割",
    "Number of header lines to transfer to new files": "要传输到新文件的标题行数",
    "Split by row or by a column?": "按行分割还是按列分割？",
    "Column to split on": "要分割的列",
    "Regex to match contents of id column": "正则表达式匹配id列的内容",
    "Pattern to replace match with": "替换匹配项的模式",
    "MGF file to split": "MGF 文件拆分",
    "FASTQ file to split": "将 FASTQ 文件拆分",
    "FASTA file to split": "FASTA 文件分割",
    "SD-file to split": "SD文件分割",
    "Text file to split": "文本文件分割",
    "Method to split files": "分割文件的方法",
    "Regex to match record separator": "匹配记录分隔符的正则表达式",
    "Split records before or after the separator?": "在分隔符之前或之后拆分行记录？",
    "Record length": "记录长度",
    "Collection of files to collapse into single dataset": "将文件集合成单一数据集",
    "Keep one header line": "保留一行标题",
    "Prepend File name": "在文件名前添加",
    "Where to add dataset name": "在哪里添加数据集名称",
    "input_file": "输入文件",
    "Extract single file": "提取单个文件",
    "Filepath": "文件路径",
    "Tabular files": "表格文件",
    "Number of header lines in each input file": "每个输入文件中标题行的数量",
    "Add column name to header": "添加列名到标题",
    "Fill character": "填充字符",
    "Additional datasets to create": "创建额外的数据集",
    "Cluster intervals of": "聚类间隔",
    "Max distance between intervals": "最大区间距离",
    "Min number of intervals per cluster": "最小区间数每簇",
    "Return type": "返回类型",
    "where minimal overlap is": "其中最小重叠是",
    "Compute coverage for": "计算覆盖度",
    "Complement regions of": "互补区域",
    "Genome-wide complement": "全基因组互补",
    "Choose Intervals": "选择区间",
    "Keep Region/Table Pairs with 0 Coverage": "保留覆盖度为0的区域/表格对",
    "Output per Region/Summary": "输出每个区域/摘要",
    "Choose Tables to Use": "选择要使用的表格",
    "Table": "表格",
    "Record ID column or value": "记录ID列或值",
    "Start column or value": "起始列或值",
    "End column or value": "结束列或值",
    "Type column or value": "输入列或值",
    "Score column or value": "得分列或值",
    "Source column or value": "源列或值",
    "Strand": "链",
    "Strand column": "链表列",
    "Strand value": "链值",
    "Qualifiers": "定语",
    "Name of the qualifier": "修饰符的名称",
    "Qualifier value column or raw text": "定语值列或原始文本",
    "A BED file with 12 columns": "一个包含12列的BED文件",
    "Source for Genomic Sequence Data": "基因组序列数据来源",
    "Select reference 2bit file": "选择参考 2bit 文件",
    "reference 2bit file": "参考 2bit 文件",
    "BED column conatining the genomic sequence": "BED列包含基因组序列",
    "BED Filtering Options": "BED过滤选项",
    "Restrict to features overlapping a comma-separated list of regions": "限制在与区域列表重叠的特征上",
    "Restrict Feature translation to these biotypes": "将特征翻译限制为这些生物类型",
    "Translation  Options": "翻译选项",
    "Feature translation": "特征翻译",
    "Minimum length of protein translation to report": "报告蛋白质翻译的最小长度",
    "Require translations to start with Methionine, trim other leading Amino Acids": "要求翻译以甲硫氨酸开始，修剪其他领先的氨基酸",
    "Digest enzyme": "消化酶",
    "Fasta ID Options": "Fasta ID 选项",
    "Genome reference name": "基因组参考名称",
    "fasta ID source, e.g. generic": "FASTA ID 来源，例如通用的",
    "fasta ID line separator character": "FASTA ID 行分隔符",
    "ID prefix for generated IDs": "生成ID的前缀",
    "Concatenate": "拼接",
    "Both datasets are in the same format?": "两个数据集的格式相同吗？",
    "Join": "连接",
    "With min overlap": "最小重叠",
    "BED file with added sequence column": "带有添加序列列的BED文件",
    "The SEQTYPE:STATUS to include in the fasta ID lines": "在fasta ID行中包含的SEQTYPE:STATUS",
    "Add the bed score field fasta ID line with this tag name": "在fasta ID行中添加带有此标签名称的bed分数字段。",
    "Filter out translations with stop codons before the splice site": "在剪接位点之前过滤掉含有终止密码子的翻译序列",
    "Stop codon filtering start position base pairs": "终止密码子过滤起始位置碱基对",
    "Stop codon filtering end position base pairs": "终止密码子过滤结束位置碱基对",
    "Trim translations to stop codons": "将翻译结果裁剪至终止密码子",
    "Maximum number of stop codons allowed in a translation to be reported": "允许在翻译中报告的最大终止密码子数量",
    "Minimum length of a translation to be reported": "报告的翻译最小长度",
    "reference bed file": "参考 bed 文件",
    "Extend the start position": "扩展起始位置",
    "Extend the end position": "扩展末端位置",
    "Glimmer Output File": "Glimmer 输出文件",
    "Input BED, GFF3 or GTF feature file": "输入 BED、GFF3 或 GTF 特征文件",
    "filters": "过滤器",
    "Filter by genome region": "按基因组区域过滤",
    "Only show transcripts overlapping coordinate range": "只显示与坐标范围重叠的转录本",
    "Discard all transcripts that are not fully contained within the given range": "丢弃所有未完全包含在给定范围内的转录本",
    "Filter out transcipts with large introns": "过滤掉具有大内含子的转录本",
    "Replace reference sequence names": "替换参考序列名称",
    "Transcript merging": "转录本合并",
    "Genome Reference Fasta": "基因组参考序列（Fasta格式）",
    "Feature File Output": "特征文件输出",
    "Ensembl GTF to GFF3 conversion": "Ensembl GTF 转 GFF3 转换",
    "full GFF attribute preservation (all attributes are shown)": "完整的GFF属性保留（显示所有属性）",
    "decode url encoded characters within attributes": "在属性内解码URL编码字符",
    "warn about duplicate transcript IDs and other potential problems with the given GFF/GTF records": "警告GFF/GTF记录中重复的转录本ID以及其他潜在问题",
    "Image to convert": "要转换的图像",
    "Transformations": "转换",
    "Reduce Color Palette": "减少颜色调色板",
    "Color Palette Size": "颜色调色板大小",
    "Resize (%)": "调整大小（百分比）",
    "Choose the source BIOM format": "选择源BIO生物观测矩阵（BIO）格式",
    "Process metadata associated with observations when converting": "在转换时处理与观测相关联的元数据",
    "Input BIOM table": "输入BIOM表格",
    "Choose the output type": "选择输出类型",
    "observation metadata to include": "观察元数据包括",
    "name to be given to the observation metadata column": "要赋予观测元数据列的名称",
    "Method for formatting the observation metadata": "观察元数据格式化方法",
    "Table Type": "表格类型",
    "BIOM format type": "BIOM格式类型",
    "Use collapsed observations": "使用折叠观测",
    "Use collapsed samples": "使用合并样本",
    "Add sample metadata from metadata mapping file": "添加来自元数据映射文件的样本元数据",
    "Add observation metadata from mapping file": "添加映射文件中的观察元数据",
    "Input idpDB": "输入 idpDB",
    "What proteomic entity do you want to group by?": "您想要按哪种蛋白质实体进行分组？",
    "Tab-delimited file": "制表符分隔的文件",
    "Title column(s)": "标题列（们）",
    "Sequence column": "序列列",
    "Convert these sequences": "将这些序列进行转换",
    "Tabular file to convert": "表格文件转换",
    "Quality column": "质量列",
    "Input GFA file": "输入 GFA 文件",
    "Thermo RAW file": "Thermo RAW 文件",
    "Include precursor scan number in MGF file": "在 MGF 文件中包含前体扫描编号",
    "Use zlib compression for the m/z ratios and intensities": "使用 zlib 压缩 m/z 比率和强度",
    "Use the peak picking provided by the native thermo library": "使用原生的Thermo库提供的峰值检测",
    "Output metadata": "输出元数据",
    "Ignore missing instrument properties": "忽略缺失的仪器属性",
    "Extract additional detector data": "提取额外的探测器数据",
    "Include reference and exception data": "包括参考和异常数据",
    "Select MS levels": "选择质谱级别",
    "Input unrefined MS data": "输入未精炼的质谱数据",
    "Choose intervals": "选择区间",
    "MAF Source": "MAF 来源",
    "Choose alignments": "选择比对",
    "Choose species": "选择物种",
    "Split blocks by species": "按物种拆分区块",
    "Collapse empty alignment columns": "折叠空白对齐列",
    "Wiggle file": "Wiggle文件",
    "Source for Input Data": "输入数据来源",
    "Lift Over To (FASTA file)": "提升至（FASTA文件）",
    "Full genome FASTA file": "全基因组FASTA文件",
    "Check if the reference allele is different from the alternate allele": "检查参考等位基因是否与替代等位基因不同",
    "GTF/GFF file": "GTF/GFF 文件",
    "Include failed liftovers": "包含失败的liftover",
    "Style of chromosome IDs": "染色体ID的风格",
    "Merge failed and converted entries into single file": "合并失败，并将条目转换为单个文件",
    "Gene BED File": "基因 BED 文件",
    "Split into Gapless MAF blocks": "拆分为无间隔的多重比对格式（MAF）块",
    "Add optional BAM Headers": "添加可选的BAM头部",
    "Insert size": "插入片段大小",
    "Insert size std. dev": "插入片段标准差",
    "Insert size std. dev foldchange": "插入大小标准差倍数变化",
    "Proteomics Identification files": "蛋白质组学鉴定文件",
    "Proteomics Spectrum files": "蛋白质组学谱图文件",
    "Proteomics Search Database Fasta": "蛋白质组学搜索数据库 Fasta",
    "A BED file with 12 columns, thickStart and thickEnd define protein coding region": "一个包含12列的BED文件，其中thickStart和thickEnd定义了蛋白质编码区域",
    "GTF annotated by gffCompare": "GTF 注释由 gffCompare 完成",
    "filter gffCompare class_codes to convert": "过滤 gffCompare 类别代码以进行转换",
    "Translated BED": "翻译后的 BED",
    "Identified Peptides": "鉴定的肽段",
    "PSM peptide column": "PSM 肽段列",
    "PSM protein name column": "PSM 蛋白质名称列",
    "PSM peptide offset column (optional)": "PSM 肽段偏移列（可选）",
    "Use #gffTags in output": "在输出中使用 #gffTags",
    "Gff file": "GFF文件",
    "Start coordinate": "起始坐标",
    "Distance to subtract from the start coordinate": "从起始坐标减去的距离",
    "Distance to add to the end coordinate": "添加到末端坐标的距离",
    "Resize window from": "调整窗口大小从",
    "Handle chromosome boundaries by": "处理染色体边界的方法",
    "Input mzML File": "输入mzML文件",
    "Input PSM Report File": "输入PSM报告文件",
    "Retain or Remove the MS/MS scans that were identified in PSM report": "保留或移除在PSM报告中被识别的MS/MS扫描。",
    "Add N random scans in addition to those in the list": "在列表中的扫描之外，添加N个随机扫描",
    "Select files": "选择文件",
    "Select output read type": "选择输出读类型",
    "Select pulse features in output BAM": "在输出的BAM文件中选择脉冲特征",
    "Store full, 16-bit IPD/PulseWidth data?": "存储完整的16位IPD/脉宽数据？",
    "Output BAMs in internal mode?": "输出BAM文件使用内部模式？",
    "Images": "图像",
    "# of images wide": "图片宽度数量",
    "AGAT tool selector": "AGAT 工具选择器",
    "Annotation file 1": "注释文件 1",
    "Annotation file 2": "注释文件2",
    "Type of feature to extract": "要提取的特征类型",
    "Extract mRNA sequences": "提取mRNA序列",
    "Extract the cDNA sequence": "提取cDNA序列",
    "Clean final stop codons": "清除最终的终止密码子",
    "Clean internal stop codons": "清除内部终止密码子",
    "Upstream nucleotides": "上游核苷酸",
    "Downstream nucleotides": "下游核苷酸",
    "Full": "完整",
    "Keep attributes": "保留属性",
    "Keep parental attributes": "保持父辈属性",
    "Plus strand only": "仅正链",
    "Protein": "蛋白质",
    "Remove ORF offset": "移除开放阅读框偏移",
    "Reverse complement the extracted sequence": "反转补码提取的序列",
    "Split": "拆分",
    "GTF version": "GTF版本",
    "Reference annotaiton": "参考注释",
    "Annotation to complement": "注释以补充",
    "Minimun CDS size": "最小编码序列大小",
    "Input interpro file": "输入InterPro文件",
    "Blast options": "Blast选项",
    "Input blast file": "输入BLAST文件",
    "Fasta file using for the blast": "用于BLAST的Fasta文件",
    "Maximum e-value to keep the annotation from the blast file": "最大e值，用于保留来自BLAST文件的注释",
    "Protein existence (PE)": "蛋白质存在（PE）",
    "Feature Type column": "特征类型列",
    "Feature Type": "特征类型",
    "Feature Identifier column": "特征标识符列",
    "Feature Identifier": "特征标识符",
    "Organism Name column": "生物体名称列",
    "Organism Name": "生物体名称",
    "BAM (or SAM) alignment file": "BAM（或SAM）比对文件",
    "Choose conversion within specific genomic region(s)": "选择特定基因组区域（s）内的转换",
    "Samtools: region in which pileup is generated": "Samtools：生成pileup的区域",
    "Only include reads overlapping this BED file": "仅包括与此 BED 文件重叠的读取",
    "Convert from": "从转换",
    "MS mzIdentML (mzid)": "MS mzIdentML（mzid）",
    "MS pepXML (pepxml)": "MS pepXML (pepxml)",
    "MS Identification": "MS鉴定",
    "Convert to": "转换为",
    "GTF File to convert": "GTF 文件转换",
    "Source Prefixes": "源前缀",
    "Source prefix": "源前缀",
    "Ignore groups without exons": "忽略没有外显子的组",
    "Skip hierarchy check": "跳过层级检查",
    "Skip all errors": "跳过所有错误",
    "Implied stop codon in after CDS": "隐含终止密码子位于编码序列之后",
    "Include gene and transcript version": "包含基因和转录本版本",
    "Output transcript information file": "输出转录本信息文件",
    "Input TwoBit file": "输入TwoBit文件",
    "Restrict this to just one sequence": "将此限制为仅一个序列",
    "Start at given position in sequence": "在序列中从给定位置开始",
    "End at given position in sequence": "在序列中给定的位置结束",
    "File containing list of the desired sequence names (Optional)": "包含所需序列名称列表的文件（可选）",
    "Convert sequence to all upper case?": "将序列转换为全部大写？",
    "Require proper mate-pairing?": "需要正确的配对吗？",
    "Read to output": "读取并输出",
    "Minimum insert size to output": "最小插入大小以输出",
    "Maximum insert size to output": "最大输出插入大小",
    "bigwig file to convert": "大wig文件转换",
    "Starting coordinate": "起始坐标",
    "End coordinate": "结束坐标",
    "Minimum ratio of bases that must remap.": "最小碱基重新映射比率。",
    "Chromid": "染色体",
    "Input ab1 file": "输入 ab1 文件",
    "Do you want trim ends according to quality scores ?": "您是否需要根据质量分数来修剪序列两端？",
    "Probability cutoff you want to use to trim": "您想要使用的用于修剪的概率截止值",
    "Minimum sequence length to allow the trim": "允许修剪的最小序列长度",
    "Phred offset for quality scores": "质量分数的Phred偏移量",
    "Select data to compress": "选择要压缩的数据",
    "input xml file to convert": "输入XML文件以转换",
    "input xsl convertion file": "输入 XSL 转换文件",
    "MGA Output": "MGA 输出",
    "GenBank file": "基因库文件",
    "Translation Table": "翻译表",
    "Combined GFF3/Fasta File": "合并的GFF3/Fasta文件",
    "Automatically generate any missing Gene features if CDS/RBS has none": "如果CDS/RBS没有基因特征，则自动生成任何缺失的基因特征。",
    "Automatically generate missing mRNA features for genes": "自动生成基因缺失的mRNA特征",
    "Qualifier to derive GFF ID from": "获取GFF ID的修饰符",
    "First image to compare": "第一幅比较图像",
    "Second image to compare": "第二张要比较的图像",
    "Pixel highlight style": "像素高亮风格",
    "Comparison metric": "比较指标",
    "PacBio Bam file": "PacBio Bam文件",
    "cool/hdf5 data to convert to hic binary": "将cool/hdf5格式数据转换为hic二进制格式",
    "Filter using the ID list from": "使用来自...的ID列表进行过滤",
    "Tabular file containing sequence identifiers": "包含序列标识符的表格文件",
    "Column(s) containing sequence identifiers": "包含序列标识符的列",
    "List of sequence identifiers (white space separated)": "序列标识符列表（以空格分隔）",
    "Output positive matches, negative matches, or both?": "输出正匹配、负匹配，还是两者都输出？",
    "Remove typical pair read name suffixes when matching identifiers?": "在匹配标识符时，是否移除典型的配对读取名称后缀？",
    "FASTA sequences": "FASTA序列",
    "Criteria for filtering on the headers": "过滤标题的标准",
    "List of IDs to extract sequences for": "要提取序列的ID列表",
    "Match IDs by": "匹配ID通过",
    "Regex search pattern for ID": "正则表达式搜索模式用于ID",
    "Regular expression pattern the header should match": "正则表达式模式，标题应匹配",
    "Criteria for filtering on the sequences": "序列筛选标准",
    "Minimum length": "最小长度",
    "Regular expression pattern the sequence should match": "正则表达式模式，序列应匹配",
    "Remove duplicate sequences": "移除重复序列",
    "Output discarded FASTA entries": "输出丢弃的FASTA条目",
    "file to rearrange": "需要重新排列的文件",
    "Specify the first few columns by name": "指定前几列的名称",
    "column": "列",
    "Discard unspecified columns?": "丢弃未指定的列？",
    "Sequence file": "序列文件",
    "Sub-sampling approach": "子采样方法",
    "N": "N",
    "Percentage": "百分比",
    "Interleaved paired reads": "交错配对读取",
    "Input XML data": "输入 XML 数据",
    "XPath Query": "XPath 查询",
    "Input file to be filtered": "输入文件将被过滤",
    "reference file to search": "参考文件搜索",
    "select columns to compare": "选择要比较的列",
    "column in input (defaults to last column)": "输入中的列（默认为最后一列）",
    "column in reference (defaults to last column)": "参考中的列（默认为最后一列）",
    "Ignore case when comparing": "忽略大小写进行比较",
    "reverse search: find the reference in the input": "反向搜索：在输入中查找参考",
    "Also search for reversed input string in the reference": "同时在参考序列中搜索输入字符串的反向序列",
    "Also search for the DNA reverse complementof of the input in the reference": "同时在参考序列中搜索输入序列的DNA反向互补序列",
    "Annotate found input entries with columns from reference": "为发现的输入条目用参考数据中的列进行注释。",
    "columns from reference to append to found input lines": "将参考文件中的列添加到找到的输入行中",
    "separator to place between annotations from different reference lines": "不同参考行注释之间的分隔符",
    "separator to place between annotation columns from the same reference line": "用于分隔同一参考行中注释列的分隔符",
    "GTF inputs for comparison": "GTF 输入用于比较",
    "Use reference annotation": "使用参考注释",
    "Choose the source for the reference annotation": "选择参考注释的来源",
    "Using reference annotation": "使用参考注释",
    "Reference annotation": "参考注释",
    "Sn correction": "Spike-in校正",
    "Sp correction": "Sp校正",
    "Strict match": "严格匹配",
    "Maximum range of variation for the free ends of terminal exons": "末端外显子自由端的最大变化范围",
    "Discard single-exon transcripts": "丢弃单外显子转录本",
    "Discart duplicate query transfrags": "丢弃重复的查询转录本片段",
    "Strict duplicate checking": "严格重复检查",
    "Disable close-exon merging": "禁用关闭外显子合并",
    "Use sequence data": "使用序列数据",
    "Choose the source for the reference sequence": "选择参考序列的来源",
    "Max distance for transcript grouping": "转录本聚类的最大距离",
    "Stats per reference contig/chromosome": "参考基因组/染色体的统计信息",
    "Generate TMAP and RefMap files for each input": "为每个输入生成 TMAP 和 RefMap 文件",
    "Combined GTF output parameters": "合并GTF输出参数",
    "Name prefix for consensus transcripts": "共识转录本的名称前缀",
    "Discard matching and 'contained' transfrags": "丢弃匹配和“包含”的转录片段",
    "Discard the 'contained' transfrags except intron-redundant transfrags starting with a different 5' exon": "丢弃“包含”的转录片段，除了以不同的5'外显子开始的内含子冗余转录片段。",
    "Discard the 'contained' transfrags also if ends stick out within the container's introns": "丢弃那些末端嵌入容器内含子中的“包含”转录片段",
    "Do NOT discard any redundant transfrag matching a reference": "不要丢弃任何与参考序列匹配的冗余转录片段",
    "Select the query sequence(s) in fasta": "选择fasta中的查询序列",
    "Reference sequence(s)": "参考序列",
    "Alignment method": "比对方法",
    "Score threshold for gapped alignment": "带间隔对齐的得分阈值",
    "Report alignment over a percentage of the maximum score attainable by each query": "报告每个查询所能达到的最大分数的百分比上的比对",
    "Report best N results per query (0 to report all)": "报告每个查询的最佳N个结果（0表示报告全部）",
    "Minimum intron length limit": "最小内含子长度限制",
    "Maximum intron length limit": "最大内含子长度限制",
    "Genome annotations in genbank format": "基因组注释在genbank格式中",
    "Funannotate database": "Funannotate 数据库",
    "Calculate dN/dS ratio on all orthologs": "计算所有同源基因的dN/dS比率",
    "P-value for FDR GO-enrichment": "FDR校正后的GO富集P值",
    "Cut-off for heatmap": "热图的截断值",
    "Number of Single-copy orthologs to use for ML": "用于ML的单拷贝直系同源基因数量",
    "Proteinortho POFF results": "Proteinortho POFF 结果",
    "Maximum Likelihood method": "最大似然法",
    "Number of boostrap replicates to run with RAxML": "RAxML 运行时的 bootstrap 复制次数",
    "FASTA sequence file": "FASTA 序列文件",
    "Check input file for potentially duplicated sequences": "检查输入文件中可能存在的重复序列",
    "Genome size": "基因组大小",
    "Unit counts format": "单位计数格式",
    "T_low": "低阈值",
    "T_high": "高温阈值",
    "Unit length": "单位长度",
    "Unit counts produced by WindowMasker mkcount": "WindowMasker mkcount 生成的单元计数",
    "Parse Seq-ids in FASTA input": "解析FASTA输入中的序列ID",
    "Use DUST to mask low complexity sequences?": "使用DUST来屏蔽低复杂性序列？",
    "DUST level": "DUST水平",
    "set_t_high": "设置高阈值",
    "set_t_low": "设置_t_low",
    "t_extend": "t_扩展",
    "t_low": "t_low",
    "t_high": "高温阈值",
    "Score threshold": "得分阈值",
    "Size of the sliding window": "滑动窗口的大小",
    "Load data in an existing GeneNoteBook database": "在现有的GeneNoteBook数据库中加载数据",
    "Genomes": "基因组",
    "Genome sequence": "基因组序列",
    "Public access": "公共访问",
    "Annotation name": "注释名称",
    "Protein naming method": "蛋白质命名方法",
    "Regex protein capture": "正则表达式蛋白质捕获",
    "Regex protein": "正则表达式蛋白质",
    "Protein id attribute": "蛋白质ID属性",
    "InterProScan results": "InterProScan 结果",
    "EggNOG-Mapper results": "EggNOG-Mapper 结果",
    "Hectar results": "Hectar 结果",
    "Import Blast or Diamond results": "导入Blast或Diamond结果",
    "Blast results (xml)": "Blast 结果（xml）",
    "Algorithm": "算法",
    "Matrix": "矩阵",
    "Diamond results (xml)": "钻石结果（xml）",
    "Expression data": "表达数据",
    "Expression data tabular file": "表达数据表格文件",
    "Expression replicas": "表达式复制品",
    "Name of a replica group": "副本组的名称",
    "Comma-separated list of columns of replicas (e.g. 1,2,3)": "逗号分隔的副本列列表（例如 1,2,3）",
    "Sequences to analyse": "待分析序列",
    "Lineage data source": "谱系数据来源",
    "Cached database with lineage": "缓存的谱系数据库",
    "Mode": "模式",
    "Generate miniprot output": "生成 miniprot 输出",
    "Use Augustus instead of Metaeuk": "使用Augustus而不是Metaeuk",
    "Augustus species model": "Augustus 物种模型",
    "Augustus model": "Augustus模型",
    "Optimization mode Augustus self-training": "优化模式 Augustus 自训练",
    "Auto-detect or select lineage?": "自动检测或选择谱系？",
    "auto-lineage group": "自动谱系组",
    "Lineage": "谱系",
    "Which outputs should be generated": "应该生成哪些输出",
    "E-value cutoff for BLAST searches.": "BLAST搜索的E值截止值。",
    "How many candidate regions to consider": "考虑多少候选区域",
    "Number of contiguous Ns to signify a break between contigs": "连续N的数量，用以表示序列片段之间的断裂",
    "Target sequence file": "目标序列文件",
    "GTF input": "GTF 输入",
    "Choose the analysis parameter for the gtf file": "选择gtf文件的分析参数",
    "List of files containing extrinsic evidence": "包含外部证据的文件列表",
    "Configuration file": "配置文件",
    "Print the transcript scores as a table to the specified file": "将转录本得分以表格形式打印到指定文件",
    "Ignore the phase transcripts while detecting clusters of overlapping transcripts": "忽略相位转录本，同时检测重叠转录本的聚类",
    "Filter out all single-exon genes": "过滤掉所有单外显子基因",
    "Protein sequences": "蛋白质序列",
    "Taxonomic domain": "分类域",
    "output_format": "输出格式",
    "Assembly to sort": "组装排序",
    "Base name to relabel contigs": "重新标记序列的基名称",
    "Shorter contigs are discarded": "较短的连续序列被丢弃",
    "Assembly to clean": "组装以清洁",
    "Percent identity of overlap": "重叠部分的一致性百分比",
    "Percent coverage of overlap": "重叠区域的百分比覆盖",
    "Minimum length of contig to keep": "保留的最小长度的连续序列",
    "Test every contig": "测试每个连续序列",
    "Assembly to annotate": "组装注释",
    "Check the genome sequence": "检查基因组序列",
    "Organism": "生物体",
    "Name of the species to annotate": "要注释的物种名称",
    "Isolate name": "分离物名称",
    "Strain name": "菌株名称",
    "Is it a fungus species?": "这是一种真菌物种吗？",
    "Ploidy of assembly": "组装的倍性",
    "Sequencing facility for NCBI tbl file": "NCBI tbl文件的测序设施",
    "Sequence accession number for NCBI tbl file": "NCBI tbl文件中的序列访问号",
    "Locus tag prefix": "基因座标签前缀",
    "Specify where gene numbering starts": "指定基因编号的起始位置",
    "Evidences": "证据",
    "RNA-seq mapped to genome to train Augustus/GeneMark-ET": "RNA-seq 映射到基因组以训练 Augustus/GeneMark-ET",
    "mRNA/ESTs to align to genome": "mRNA/ESTs与基因组比对",
    "Select protein evidences": "选择蛋白质证据",
    "Proteins to map to genome": "要映射到基因组的蛋白质",
    "Exonerate percent identity (for proteins)": "Exonerate 蛋白质序列相似度百分比",
    "Prefilter hits with (for proteins)": "预过滤命中（对于蛋白质）",
    "Busco": "Busco",
    "BUSCO models to align": "BUSCO模型对齐",
    "Initial Augustus species training set for BUSCO alignment": "初始的Augustus物种训练集用于BUSCO比对",
    "Minimum protein length": "最小蛋白长度",
    "Keep gene models without valid stops": "保留没有有效终止子的基因模型",
    "Repetitive gene model filtering": "重复基因模型过滤",
    "Ab-initio training parameters from a previous run": "从之前运行中获取的Ab-initio训练参数",
    "Augustus settings (advanced)": "Augustus 设置（高级）",
    "Augustus species training set": "Augustus 物种训练集",
    "Minimum number of models to train Augustus": "Augustus 训练的最小模型数量",
    "Run 'optimize_augustus.pl' to refine training (long runtime)": "运行 'optimize_augustus.pl' 来优化训练（运行时间较长）",
    "GeneMark settings (advanced)": "GeneMark 设置（高级）",
    "GeneMark license file": "GeneMark 许可证文件",
    "GeneMark mode": "基因标记模式",
    "Use pre-existing Genemark training file (e.g. gmhmm.mod)": "使用现有的Genemark训练文件（例如 gmhmm.mod）",
    "Softmasked length threshold for GeneMark": "基因标记的软掩蔽长度阈值",
    "Other annotations (advanced)": "其他注释（高级）",
    "StringTie GTF result": "StringTie GTF 结果",
    "MAKER2 GFF file": "MAKER2 GFF文件",
    "PASA generated gene models": "PASA生成的基因模型",
    "Weight for PASA generated gene models": "PASA生成的基因模型的权重",
    "Annotation pass-through to EVM": "注释传递到EVM",
    "Weight for annotation pass-through to EVM": "注释传递到EVM的权重",
    "EVM settings (advanced)": "EVM 设置（高级）",
    "Use repeats in EVM consensus model building": "使用EVM共识模型构建中的重复序列",
    "Split contigs into partitions for EVM processing?": "将序列分割成部分以进行EVM处理？",
    "Min length between genes to make a partition": "基因间最小长度以进行分区",
    "Custom ab-initio predictor and EVM weight": "自定义从头预测器和EVM权重",
    "Maximum length of FASTA headers": "FASTA头的最大长度",
    "Genome sequence is soft-masked": "基因组序列进行了软掩蔽",
    "Species name": "物种名称",
    "RNA-seq mapped to genome to train Augustus/GeneMark": "RNA-seq 映射到基因组以训练 Augustus/GeneMark",
    "GeneMark": "基因标记",
    "Fungal genome": "真菌基因组",
    "Augustus": "Augustus",
    "Use CRF training for Augustus": "使用CRF训练Augustus",
    "Number of optimization rounds used in optimize_augustus.pl": "在 optimize_augustus.pl 中使用的优化轮数",
    "Output ab initio predictions by AUGUSTUS": "输出由AUGUSTUS进行的从头预测结果",
    "CRF parameters": "条件随机场参数",
    "Advanced": "高级",
    "Filter out too short traingin gene predicted by GeneMark-EX": "过滤掉由GeneMark-EX预测的太短的训练基因",
    "Reference set to evaluate predictions": "参考集用于评估预测",
    "File with pseudogenes that will be excluded from accuracy evaluation": "包含将从准确性评估中排除的伪基因的文件",
    "Output alternative transcripts based on explicit evidence from hints": "根据来自提示的明确证据输出备选转录本",
    "Expert options": "专家选项",
    "List of splice site patterns for UTR prediction": "用于UTR预测的剪接位点模式列表",
    "Minimal contig length for GeneMark-EX": "GeneMark-EX的最小Contig长度",
    "Probablity for donor splice site pattern GC for gene prediction with GeneMark-EX": "供体剪接位点模式 GC 的概率，用于 GeneMark-EX 基因预测",
    "Maximum allowed size of intergenic regions in GeneMark-EX": "GeneMark-EX中允许的间隔区域最大大小",
    "Lambda parameter of the Poisson distribution": "泊松分布的Lambda参数",
    "Nucleotide Type": "核苷酸类型",
    "homozygosity beta": "纯合性 beta",
    "Enumerations": "枚举",
    "ILP solver": "整数线性规划求解器",
    "unpaired_weight": "未配对权重",
    "use_discordant": "使用不连续的",
    "Contigs to annotate": "待注释的连续序列",
    "Locus tag counter increment": "基因座标签计数器递增",
    "GFF version": "GFF版本",
    "Force GenBank/ENA/DDJB compliance": "强制遵守GenBank/ENA/DDJB规定",
    "Add 'gene' features for each 'CDS' feature (--addgenes)": "为每个'CDS'特征添加'基因'特征（--addgenes）",
    "Minimum contig size (--mincontiglen)": "最小拼接长度（--mincontiglen）",
    "Sequencing centre ID": "测序中心ID",
    "Genus name": "属名",
    "Plasmid name or identifier": "质粒名称或标识符",
    "Genetic code (transl_table)": "遗传密码（transl_table）",
    "Use genus-specific BLAST database": "使用特定属的BLAST数据库",
    "Optional FASTA file of trusted proteins to first annotate from": "可选的FASTA文件，用于首先注释可信蛋白质",
    "Improve gene predictions for highly fragmented genomes": "提高高度碎片化基因组的基因预测",
    "Fast mode": "快速模式",
    "Similarity e-value cut-off": "相似性 E 值截止",
    "Enable searching for ncRNAs with Infernal+Rfam (SLOW!)": "启用使用 Infernal+Rfam 搜索非编码 RNA（慢！）",
    "Don't run rRNA search with Barrnap": "不要使用Barrnap进行rRNA搜索",
    "Don't run tRNA search with Aragorn": "不要使用Aragorn进行tRNA搜索",
    "Additional outputs": "附加输出",
    "Individual gff files or a dataset collection": "单独的gff文件或数据集集合",
    "select gff inputs to Roary": "选择 GFF 输入到 Roary",
    "Dataset collection to submit to Roary": "提交给Roary的数据集集合",
    "minimum percentage identity for blastp": "BLASTP 最低百分比相似性",
    "percentage of isolates a gene must be in to be core": "核心基因必须存在于分离物的百分比",
    "Maximum number of clusters": "最大聚类数",
    "Don't split paralogs": "不要拆分旁系同源基因",
    "Translation table to use [1 or 4 or 11]": "要使用的转换表 [1 或 4 或 11]",
    "MCL inflation value": "MCL膨胀值",
    "Input File Type": "输入文件类型",
    "Read1": "读取1",
    "Read2": "读取2",
    "Reads (collection)": "读取（集合）",
    "Bam": "Bam",
    "Select advanced options": "选择高级选项",
    "Mapper": "映射器",
    "Variant caller": "变异检测器",
    "Min Depth": "最小深度",
    "Minimum allele frequency to call variants": "最小等位基因频率以调用变异",
    "Reporting Minimum allele frequency to call variants": "报告最小等位基因频率以调用变异",
    "Name for the HMM": "HMM的名称",
    "Pretend all inserts are length <= n": "假设所有插入的长度都小于或等于n",
    "Protein sequence to search with": "要搜索的蛋白质序列",
    "Sequence Database": "序列数据库",
    "window length for biased-composition modifier (MSV)": "偏置组成修饰（MSV）的窗口长度",
    "window length for biased-composition modifier (Vit)": "偏差组成修饰符（Vit）的窗口长度",
    "window length for biased-composition modifier (Fwd)": "偏向性组成修饰因子（正向）的窗口长度",
    "Sequence database to search against": "要搜索的序列数据库",
    "Select query sequence file": "选择查询序列文件",
    "Select profile database format": "选择配置文件数据库格式",
    "Select a compressed file with HMM and HAL file(s)": "选择包含HMM和HAL文件的压缩文件",
    "Select profile HMM file(s)": "选择HMM文件（一个或多个）",
    "Select KO list file": "选择KO列表文件",
    "Set E-value threshold": "设置E值阈值",
    "Set threshold scale": "设置阈值比例",
    "File with IDs": "带有ID的文件",
    "File has header?": "文件有标题吗？",
    "ID Type": "ID类型",
    "Output columns": "输出列",
    "Remove duplicates?": "移除重复项？",
    "Output Rscript?": "输出 R 脚本？",
    "Check for circularity flag generated by unicycler in contigs fasta headers?": "检查由Unicycler在contigs fasta头部生成的循环标志？",
    "Detect circular contigs (i.e. potential plasmids) with assembly overhangs?": "检测具有组装悬空端的循环片段（即潜在的质粒）？",
    "Provide debug information?": "提供调试信息？",
    "Minimum evalue threshold for replicon blastn": "复制子BLASTN的最小E值阈值",
    "Minimum evalue threshold for relaxase tblastn": "最小e值阈值用于relaxase tblastn",
    "Minimum evalue threshold for contig blastn": "最小e值阈值用于连通序列比对（blastn）",
    "Minimum evalue threshold for repetitve elements blastn": "重复元素BLASTN的最小e值阈值",
    "Minimum length of contigs to classify": "最小的拼接序列长度以进行分类",
    "Minimum sequence identity for replicons": "最小序列相似性用于复制子",
    "Minimum sequence identity for relaxases": "松弛酶的最小序列同一性",
    "Minimum sequence identity for contigs": "最小序列一致性用于拼接序列",
    "Minimum sequence identity for repetitive elements": "重复元素的最小序列相似性",
    "Minimum percentage coverage of replicon query by input assembly": "复制体查询在输入组装中的最小百分比覆盖率",
    "Minimum percentage coverage of relaxase query by input assembly": "relaxase查询在输入组装中的最小百分比覆盖率",
    "Minimum percentage coverage of assembly contig by the plasmid reference database to be considered": "组装片段由质粒参考数据库覆盖的最小百分比被认为是",
    "Minimum percentage coverage of contigs by repetitive elements": "重复元素对基因组序列的最小覆盖率",
    "Reference Database of complete plasmids": "完整质粒的参考数据库",
    "Custom MASH database of plasmids": "自定义的质粒 MASH 数据库",
    "Plasmid cluster metadata file": "质粒簇元数据文件",
    "FASTA file with plasmid replicons": "含有质粒复制子的FASTA文件",
    "FASTA of known repetitive elements": "已知重复元素的FASTA文件",
    "FASTA of plasmid relaxases": "质粒松弛酶的FASTA序列",
    "FASTA of known plasmid mate-pair proteins": "已知质粒接合对蛋白的FASTA序列",
    "FASTA of known plasmid oriT dna sequences": "已知质粒起始序列oriT的FASTA文件",
    "Treat each input sequence as an independant plasmid?": "将每个输入序列视为独立的质粒？",
    "Minimum overlap of fragments": "片段最小重叠",
    "Genome(s) input(s)": "基因组输入",
    "O antigen minimum %identity": "O抗原最小%一致性",
    "O antigen minimum %coverage": "O抗原最小%覆盖率",
    "H antigen minimum %identity": "H抗原最小%一致性",
    "H antigen minimum %coverage": "H抗原最小%覆盖率",
    "Enable E. coli species verification": "启用大肠杆菌物种验证",
    "Include BLAST allele alignment results tab-delim file in the outputs?": "在输出中包含BLAST等位基因比对结果的制表符分隔文件？",
    "Include log file in the run outputs?": "在运行输出中包含日志文件？",
    "Mash genome sketches (Optional)": "制作基因组草图（可选）",
    "Custom database of alleles (Optional)": "自定义等位基因数据库（可选）",
    "Input Genome(s)": "输入基因组（们）",
    "Results output format": "结果输出格式",
    "Use full cgMLST database for serovar prediction. About 10X slower with equivalent results to reduced centroid allele database.": "使用完整的cgMLST数据库进行血清型预测。与使用简化的质心等位基因数据库相比，速度大约慢10倍，但结果相当。",
    "Run Mash MinHash-based serovar prediction": "运行基于MinHash的Mash血清型预测",
    "Skip running cgMLST-based serovar prediction": "跳过基于cgMLST的血清型预测",
    "Basic QC of results": "基本质量控制的结果",
    "Results verbosity": "结果详细度",
    "Keep temporary analysis directory": "保持临时分析目录",
    "Logging verbosity": "日志详细度",
    "sequence": "序列",
    "Reference data": "参考数据",
    "Feature types": "特征类型",
    "Final overlap (nt)": "最终重叠（核苷酸数）",
    "Advanced options for protein coding gene prediction": "蛋白质编码基因预测的高级选项",
    "BLAST E-value Exponent": "BLAST E值指数",
    "Clipping factor": "剪切因子",
    "Fragment overlap": "片段重叠",
    "Fragment quality factor": "片段质量因子",
    "Start/stop range": "起始/终止范围",
    "Sequence file in GenBank,EMBL or FASTA format": "序列文件在GenBank、EMBL或FASTA格式",
    "GFF3 file": "GFF3 文件",
    "Taxonomic classification of input sequence": "输入序列的分类学分类",
    "Annotate with TIGRFam": "使用 TIGRFam 进行注释",
    "Motif based prediction of SM gene cluster regions": "基于基序的SM基因簇区域预测",
    "Full genome PFAM anotation": "全基因组PFAM注释",
    "PFAM anotation for only clusters": "PFAM注释，仅限聚类",
    "Run active site finder analysis": "运行活性位点发现分析",
    "Comparison against MIBiG database": "与MIBiG数据库的比较",
    "BLAST identified clusters against known clusters": "BLAST 识别出与已知聚类的簇",
    "KnowCluster BLAST analysis": "KnowCluster BLAST分析",
    "Subcluster BLAST analysis": "子聚类BLAST分析",
    "Run Pfam to Gene Ontology mapping module": "运行Pfam到基因本体（Gene Ontology）映射模块",
    "RREFinder precision mode": "RREFinder 精确模式",
    "Analysis of secondary metabolism gene families (smCOGs)": "次级代谢基因家族（smCOGs）的分析",
    "Lowest GC content to annotate TTA codons at": "最低的GC含量以注释TTA密码子",
    "Min length": "最小长度",
    "HMM detection strictness": "HMM检测严格度",
    "Number of clusters from ClusterBlast to display": "从ClusterBlast显示的聚类数量",
    "ClusterBlast minimum scaling factor": "ClusterBlast 最小缩放因子",
    "RRE cutoff": "RRE 截断值",
    "RRE minlength": "RRE 最小长度",
    "Sideloading": "侧载",
    "sideload json file": "侧载 JSON 文件",
    "Sideload Format": "边载格式",
    "Peaks file": "峰值文件",
    "Peaks file has header?": "峰值文件有头部吗？",
    "Annotation source": "注释来源",
    "Select a built-in GTF": "选择一个内置的GTF",
    "Select a history GTF": "选择一个历史 GTF",
    "Output PDF of plots?": "输出图表的PDF文件？",
    "Output RData file?": "输出 RData 文件？",
    "TSS upstream region": "转录起始位点上游区域",
    "TSS downstream region": "转录起始位点下游区域",
    "Add flanking gene information?": "添加侧翼基因信息？",
    "Flanking gene distance": "相邻基因距离",
    "Ignore upstream?": "忽略上游？",
    "Ignore downstream?": "忽略下游？",
    "Select sequences to align": "选择序列进行比对",
    "Apply Backbone": "应用骨架",
    "Island gap size": "岛间间隔大小",
    "Disable backbone": "禁用骨架",
    "Output Guide Tree": "输出引导树",
    "Output Backbone": "输出骨架",
    "MUMs": "MUMs",
    "Seed weight": "种子重量",
    "Match Input": "匹配输入",
    "Max gapped aligner length": "最大间隙比对长度",
    "input-guide-tree": "输入指南树",
    "Collinear inputs": "共线性输入",
    "Scoring scheme": "评分方案",
    "No weight scaling": "无权重缩放",
    "max-breakpoint-distance-scale": "最大断点距离比例尺",
    "conservation-distance-scale": "保守性-距离-尺度",
    "Skip refinement": "跳过细化",
    "Skip gapped alignment": "跳过有间隙的对齐",
    "BP dist estimate min score": "BP 距离估计最小分数",
    "Gap open": "Gap open",
    "Repeat penalty": "重复惩罚",
    "Gap extend": "Gap 扩展",
    "Weight": "权重",
    "Min scaled penalty": "最小缩放惩罚",
    "HMM p go homologous": "HMM p go 同源",
    "HMM p go unrelated": "HMM p go 不相关",
    "HMM identity": "HMM 一致性",
    "Seed family": "种子家族",
    "Solid seeds": "结实的种子",
    "Coding seeds": "编码种子",
    "No recursion": "不递归",
    "XMFA MSA": "XMFA 多序列比对",
    "Select sequences used in progressiveMauve alignment": "选择用于progressiveMauve比对的序列",
    "Window Size": "窗口大小",
    "Generate relative to Nth sequence in XMFA file": "生成相对于XMFA文件中第N个序列的相对值",
    "Minimum DNA %identity": "最小DNA%相似性",
    "Minimum DNA %coverage": "最小DNA%覆盖度",
    "Output novel alleles": "输出新等位基因",
    "Automatically set MLST scheme": "自动设置 MLST 方案",
    "Minimum score to match scheme": "匹配方案的最低分数",
    "Exclude these schemes": "排除这些方案",
    "PubMLST Scheme (list)": "PubMLST方案（列表）",
    "MLST Scheme (manual)": "MLST方案（手动）",
    "Output allele information": "输出等位基因信息",
    "Genbank file": "基因库文件",
    "Infer exon/mRNA subfeatures": "推断外显子/信使RNA子特征",
    "Sequence Ontology": "序列本体论",
    "Sequence Ontology File": "序列本体文件",
    "Sequence Ontology URL": "序列本体URL",
    "Error threshold for unflattener": "未平坦化器的错误阈值",
    "Gene Model": "基因模型",
    "Sequence Ontology type for landmark feature": "地标特征的序列本体类型",
    "Sequence": "序列",
    "Treat sequence as linear": "将序列视为线性",
    "Annotate only the best copy of each feature": "注释每个特征的最佳副本",
    "Use start/stop codons as in NCBI (default: learned start/stop codons)": "使用NCBI中的起始/终止密码子（默认：学习到的起始/终止密码子）",
    "Use the hmmer based method of Al Arab et al. 2016. This will consider the evalue, ncbicode, fragovl, fragfac": "使用基于HMMER的Al Arab等人2016年的方法。这将考虑evalue、ncbicode、fragovl、fragfac参数。",
    "Use the old start/stop prediction method of MITOS1": "使用MITOS1的旧起始/终止预测方法",
    "Advanced options for ncRNA gene prediction": "ncRNA基因预测的高级选项",
    "Run mitfi in glocal and local mode (default: local only)": "在全局和局部模式下运行mitfi（默认：仅局部）",
    "e-value to use for inferal fast mode": "用于推断的快速模式的e值",
    "Use infernals sensitive mode only": "使用Infernal的敏感模式",
    "Allow tRNA overlap of up to X nt for mitfi": "允许tRNA重叠最多X个核苷酸，用于mitfi",
    "Allow rRNA overlap of up to X nt for mitfi": "允许rRNA重叠最多X个核苷酸，用于mitfi",
    "Name prefix for this analysis": "此分析的名称前缀",
    "Paired-end reads": "成对末端读取",
    "Select a collection of paired-end reads": "选择一组配对末端读取",
    "Trim bases from 3 prime": "从3'端修剪碱基",
    "Query Sequence(s)": "查询序列",
    "Reference Sequence(s)": "参考序列",
    "Smallest read-count to consider": "最小读取计数以考虑",
    "Control .wig files": "控制 .wig 文件",
    "Number of samples": "样本数量",
    "Output histogram of the permutations": "输出排列的直方图",
    "Perform adaptive resampling": "执行自适应重采样",
    "Exclude rows with zero accross conditions": "排除所有条件下值为零的行",
    "Add pseudocounts at each site": "在每个位点添加伪计数",
    "Perform LOESS Correction": "执行LOESS校正",
    "Number of Burn-in samples": "烧入样本数",
    "Trimming interval for values": "修剪间隔的值",
    "Collection or paired reads": "集合或配对的读取",
    "Select a SNP profile from your history or use one from a tool data table?": "从您的历史记录中选择一个SNP（单核苷酸多态性）分析结果，或者使用工具数据表中的一个？",
    "SNP Profile": "SNP 概况",
    "Select a structural variant profile from your history or use one from a tool data table?": "从您的历史记录中选择一个结构变异档案，或者使用来自工具数据表的一个？",
    "Structural Variant Profile": "结构变异档案",
    "Transposable Element Reference": "转座元件参考",
    "Flank Length": "侧翼长度",
    "Minimum Reads": "最小读取数",
    "Minimum Reads (each strand)": "最小读取数（每个链）",
    "Minimum Mapped Length": "最小比对长度",
    "Include log in output": "在输出中包含日志",
    "Genome to annotate": "待注释的基因组",
    "Annotation to use for training": "用于训练的注释",
    "Genome Sequence": "基因组序列",
    "Don't report transcripts with in-frame stop codons": "不要报告含有帧内终止密码子的转录本",
    "Predict genes independently on each strand": "独立预测每个链上基因",
    "Predict the untranslated regions in addition to the coding sequence": "预测未翻译区域以及编码序列",
    "Trainingset": "训练集",
    "Model Organism": "模式生物",
    "Softmasking": "软掩蔽",
    "Predict genes on specific strands": "预测特定链上基因",
    "Use hints (extrinsic information)?": "使用提示（外部信息）？",
    "Select hints file from history": "从历史记录中选择提示文件",
    "Select extrinsic configuration file from history": "从历史记录中选择外部配置文件",
    "Specify prediction sequence range?": "指定预测序列范围？",
    "Starting position": "起始位置",
    "Ending position": "结束位置",
    "GFF formated output": "GFF格式输出",
    "Forward reads (R1)": "正向读取（R1）",
    "Reverse reads (R2)": "反向读取（R2）",
    "Paired collection": "配对集合",
    "Input contigs/genomes": "输入片段/基因组",
    "Resistance Genes": "抗性基因",
    "The threshold score for a sequence to be classified as a (almost) complete gene": "序列被分类为（几乎）完整基因的阈值分数",
    "The threshold score for a fragment to be classified as a positive. Expressed as score per amino acid": "片段被分类为阳性的阈值分数。以每个氨基酸的分数表示。",
    "Rescue short unassmebled plasmids": "拯救短的未组装的质粒",
    "The minimal length for a retrieved predicted ORF (nt)": "检索到的预测开放阅读框（核苷酸）的最小长度",
    "Use this flag if the whole sequence where a hit is detected should be retrieved": "如果检测到命中的整个序列都应该被检索，请使用这个标志。",
    "Do not perform ORF prediction": "不要执行开放阅读框（ORF）预测",
    "Use if no quality control should be performed on the metagenomic data": "如果不需要对宏基因组数据进行质量控制，则使用该选项",
    "Use if you want to skip the assembly and retrieval of contigs for metagenomic data": "如果您想跳过宏基因组数据的组装和检索片段，则使用此选项",
    "Use NCBI ORFfinder instead of prodigal for ORF prediction of genomes/contigs": "使用NCBI ORFfinder代替prodigal进行基因组/序列的开放阅读框预测。",
    "Store the translated sequences": "存储翻译后的序列",
    "GenBank GFF file": "GenBank GFF文件",
    "Input is whitespace-separated list of SNP IDs to be translated (should start with prefix ’rs’).": "输入是空格分隔的SNP ID列表，需要翻译（应以前缀'rs'开头）。",
    "Ordered query": "有序查询",
    "Significant only": "显著的只有",
    "Measure underrepresentation": "测量不足表示",
    "Evidence codes": "证据代码",
    "User threshold": "用户阈值",
    "Correction method": "校正方法",
    "Statistical domain scope": "统计域范围",
    "Custom background": "自定义背景",
    "Numeric IDs treated as": "数值ID被视为",
    "Data sources": "数据来源",
    "Gene Ontology": "基因本体论",
    "No electronic GO annotations": "没有电子基因本体（GO）注释",
    "biological pathways": "生物途径",
    "regulatory motifs in DNA": "DNA中的调控元件",
    "protein databases": "蛋白质数据库",
    "Human phenotype ontology": "人类表型本体论",
    "Export plot": "导出图表",
    "Source organism": "源生物体",
    "Target organism": "目标生物",
    "Maximum number of ortholog names per gene to show": "每个基因显示的同源基因名称的最大数量",
    "Target namespace": "目标命名空间",
    "Maximum number of results per initial identifier to show": "每个初始标识符显示的最大结果数",
    "Input file (Fasta, Genbank or EMBL file)": "输入文件（Fasta、Genbank或EMBL文件）",
    "Database to use - default is 'resfinder'": "使用的数据库 - 默认是 'resfinder'",
    "Suppress header": "隐藏标题",
    "Abricate output reports to summarize": "Abricate 输出报告汇总",
    "Blast XML": "Blast XML",
    "Maximum gap size before generating a new match_part": "最大间隔大小，在生成一个新的匹配部分之前",
    "Trim sides of blast hits": "修剪BLAST命中的两侧",
    "Are you running Maker for academic use, or did you purchase a license?": "你是为了学术用途运行Maker，还是购买了许可证？",
    "Organism type": "生物体类型",
    "Re-annotate using an existing Maker annotation": "使用现有的Maker注释重新注释",
    "Previous Maker annotation": "之前的Maker注释",
    "Re-use ESTs": "重用ESTs",
    "Re-use alternate organism ESTs": "重用其他生物的EST序列",
    "Re-use protein alignments": "重用蛋白质比对",
    "Re-use repeats": "重复使用",
    "Re-use gene models": "重用基因模型",
    "Re-use ab-initio predictions": "重用从头预测",
    "Passthrough anything else": "传递任何其他内容",
    "EST evidences (for best results provide at least one of these)": "EST证据（为了获得最佳结果，至少提供其中之一）",
    "Infer gene predictions directly from all ESTs": "从所有ESTs直接推断基因预测",
    "ESTs or assembled cDNA": "表达序列标签或组装的cDNA",
    "EST/cDNA from an alternate organism": "来自另一个生物体的EST/cDNA",
    "Aligned ESTs or cDNA": "对齐的ESTs或cDNA",
    "Aligned EST/cDNA from an alternate organism": "来自另一个生物体的对齐的EST/cDNA",
    "Protein evidences (for best results provide at least one of these)": "蛋白质证据（为了获得最佳结果，请至少提供其中之一）",
    "Infer gene predictions directly from all protein alignments": "从所有蛋白质比对中直接推断基因预测",
    "Aligned proteins": "对齐的蛋白质",
    "Ab-initio gene prediction": "从头基因预测",
    "SNAP model": "SNAP模型",
    "Prediction with Augustus": "使用Augustus进行预测",
    "Run ab-initio prediction programs on both masked and unmasked sequence regions": "在掩蔽和未掩蔽的序列区域上运行从头预测程序",
    "Repeat masking": "重复掩蔽",
    "Repeat library source": "重复库来源",
    "Select species name from a list?": "从列表中选择物种名称？",
    "Repeat source species": "重复源物种",
    "Use soft-masking rather than hard-masking in BLAST": "在BLAST中使用软掩蔽而不是硬掩蔽",
    "DFam library": "DFam库",
    "Transposable element sequences for RepeatRunner": "转座元件序列对于RepeatRunner",
    "Transposable element protein sequences for RepeatRunner": "转座元件蛋白序列，用于RepeatRunner",
    "Pre-identified repeat elements from an external GFF file": "从外部GFF文件中预先识别的重复元素",
    "Other predictions": "其他预测",
    "Predictions from an external GFF3 file": "来自外部GFF3文件的预测",
    "Annotated gene models from an external GFF3 file": "来自外部GFF3文件的注释基因模型",
    "Find tRNAs with tRNAscan": "使用tRNAscan寻找tRNA",
    "rRNA file to have Snoscan find snoRNAs": "rRNA文件用于Snoscan查找snoRNAs",
    "Fix nucleotides": "修复核苷酸",
    "Extra features to pass-through to final Maker generated GFF3 file": "传递给最终由Maker生成的GFF3文件的额外特性",
    "Amino acid used to replace non-standard amino acids in BLAST databases": "在BLAST数据库中用来替换非标准氨基酸的氨基酸",
    "Length for dividing up contigs into chunks": "将连续序列分成块的长度",
    "Skip genome contigs below this length": "跳过长度低于此值的基因组片段",
    "Flank for extending evidence clusters sent to gene predictors": "用于扩展发送给基因预测器的证据聚类的侧翼",
    "Report AED and QI statistics for all predictions as well as models": "报告所有预测以及模型的AED（平均编辑距离）和QI（质量指数）统计数据。",
    "Maximum Annotation Edit Distance allowed": "最大注释编辑距离允许",
    "Require at least this many amino acids in predicted proteins": "要求在预测的蛋白质中至少有这么多氨基酸",
    "Take extra steps to try and find alternative splicing": "采取额外步骤尝试寻找替代剪接",
    "Take extra steps to force the finding of a start and stop codons": "采取额外步骤强制查找起始和终止密码子",
    "Map names and attributes forward from old GFF3 genes": "将旧的GFF3基因的名称和属性映射到前面",
    "Concordance threshold to add unsupported gene prediction": "一致性阈值，用于添加不支持的基因预测",
    "length for the splitting of hits": "命中分裂的长度",
    "Limit use of ESTs in annotation to avoid fusion genes": "限制在注释中使用EST以避免融合基因",
    "Consider single exon EST evidence when generating annotations": "在生成注释时考虑单个外显子EST证据",
    "min length required for single exon ESTs if": "单外显子ESTs所需的最小长度为",
    "Select fasta dataset": "选择fasta数据集",
    "Select a species": "选择一个物种",
    "Optional Arguments": "可选参数",
    "Only look at this number of bases from start and end of fragment": "仅查看片段起始和结束处的这些碱基",
    "Minimum bit score": "最小比特分数",
    "Minimum alignment length": "最小比对长度",
    "Assume chromosome is not circularised": "假设染色体没有被环形化",
    "Differentially expressed genes file": "差异表达基因文件",
    "Gene lengths file": "基因长度文件",
    "Gene categories": "基因类别",
    "Select a genome to use": "选择一个要使用的基因组",
    "Select Gene ID format": "选择基因ID格式",
    "Select one or more categories": "选择一个或多个类别",
    "Gene category file": "基因类别文件",
    "Method Options": "方法选项",
    "Use Wallenius method": "使用瓦雷尼乌斯方法",
    "Use Hypergeometric method": "使用超几何分布方法",
    "Sampling number": "抽样数量",
    "Select a method for multiple hypothesis testing correction": "选择一种多重假设检验校正方法",
    "Count genes without any category?": "计算没有任何类别的基因数量？",
    "Output Top GO terms plot?": "输出顶级GO术语图？",
    "Produce diagnostic plots?": "生成诊断图？",
    "Extract the DE genes for the categories (GO/KEGG terms)?": "提取（GO/KEGG术语）的DE基因？",
    "Maker annotation to use for training": "用于训练的Maker注释",
    "Number of gene model to use for training": "用于训练的基因模型数量",
    "Maker annotation where to change ids": "Maker注释中更改ID的位置",
    "Prefix for ids": "ID前缀",
    "Justify numeric ids to this length": "将数字ID对齐到这个长度",
    "Ranked Genes": "排名基因",
    "Gene Sets": "基因集",
    "Minimum Size of Gene Set": "基因集的最小大小",
    "Maximal Size of Gene Set": "基因集的最大大小",
    "Number of Permutations": "排列数",
    "Output plots": "输出图表",
    "Plot top most significant pathways": "绘制最显著的通路",
    "Scientific name": "科学名称",
    "Choose source of gnm2tab file": "选择gnm2tab文件的来源",
    "gnm2tab file from spaln": "将 spaln 的 gnm2tab 文件转换",
    "NCBI Taxonomy SQLite database": "NCBI 分类法 SQLite 数据库",
    "Genome sequence to search (FASTA format)": "要搜索的基因组序列（FASTA格式）",
    "Query sequence(s) (protein or cDNA)": "查询序列（蛋白质或cDNA）",
    "Species to use for parameter setting": "用于参数设置的物种",
    "DNA query orientation": "DNA 查询方向",
    "Minimum space to induce Hirschberg's algorithm": "最小空间以诱导赫希伯格算法",
    "Limit 3' poly-As to this number of bases": "将3'端的多聚腺苷酸限制到这个碱基数量",
    "Report results even if the score is below the threshold": "即使得分低于阈值，也报告结果",
    "Gap-extension penalty": "间隙延伸惩罚",
    "Gap-open penalty": "间隙开放惩罚",
    "Band width for DP matrix scan": "DP矩阵扫描的带宽",
    "Stringency of splice site selection": "剪接位点选择的严格性",
    "Incline of long gap penalty": "长间隔惩罚的倾斜度",
    "Flex point where the incline of gap penalty changes": "Flex点，即间隙惩罚斜率变化的位置",
    "Use double affine gap penalty": "使用双仿射间隙惩罚",
    "Nucleotide match score": "核苷酸匹配得分",
    "Nucleotide mismatch score": "核苷酸错配分数",
    "Penalty for a premature termination codon": "提前终止密码子的惩罚",
    "Penalty for a frame shift error": "帧移错误惩罚",
    "Weight for splice site signal": "剪接位点信号的权重",
    "Weight for coding potential": "编码潜能权重",
    "Weight for branch point signal": "分支点信号的权重",
    "Minimum expected length of intron": "内含子的最小期望长度",
    "Weight for intron potential": "内含子潜能的权重",
    "Reset maximum expected gene size, suffix k or M is effective": "重置最大预期基因大小，后缀 k 或 M 有效",
    "Format for remote queries": "远程查询的格式",
    "Activate salvage mode": "激活恢复模式",
    "Full Gene Ontology File": "完整的基因本体文件",
    "GOSlim File": "GOSlim 文件",
    "Gene Product Annotation File": "基因产物注释文件",
    "Gene Ontology File": "基因本体论文件",
    "Study Set File": "研究集文件",
    "Population Set File (Optional)": "种群集文件（可选）",
    "Multiple Test Correction": "多重检验校正",
    "P-Value Cut-Off": "P值截止",
    "Output Graph Format": "输出图形格式",
    "Summarize Output": "总结输出",
    "Exclude Singletons": "排除单例",
    "Use All Relations": "使用所有关系",
    "Annotation to analyse": "注释分析",
    "Corresponding genome sequence": "相应的基因组序列",
    "FASTA file of DNA sequences": "DNA序列的FASTA文件",
    "Input sequences for which to predict localisation (protein FASTA format)": "为预测定位的输入序列（蛋白质FASTA格式）",
    "Organism type (N.B. all sequences in the above file must be of the same type)": "生物体类型（注意：上述文件中的所有序列必须为同一类型）",
    "Sets a cutoff value for reported results (e.g. 7.5)": "设置报告结果的截止值（例如7.5）",
    "Sets a cutoff value for the multiple localization flag (e.g. 4.5)": "设置多重定位标志的截止值（例如 4.5）",
    "FASTA file of protein sequences": "蛋白质序列的FASTA文件",
    "Which RXLR model?": "哪种 RXLR 模型？",
    "Truncate sequences to this many amino acids": "将序列截断为这么多氨基酸",
    "Function ids": "功能ID",
    "Input trait table": "输入性状表",
    "Input table delimiter": "输入表格分隔符",
    "Output table delimiter": "输出表格分隔符",
    "Tree to trait mapping": "树到性状映射",
    "Limit trees": "限制树",
    "Suppress bifurcating": "抑制分叉",
    "Convert to NEXUS format": "转换成NEXUS格式",
    "Convert character state to integers": "将字符状态转换为整数",
    "No minimum branch length": "无最小分支长度",
    "Supress tree filter": "抑制树过滤器",
    "Supress table filter": "抑制表格筛选器",
    "Add branch to root": "添加分支到根节点",
    "Observed table": "观察表",
    "Expected table": "预期表格",
    "Real counts instead of relative abundances?": "实际计数而非相对丰度？",
    "Compare observations": "比较观察结果",
    "Normalize?": "标准化？",
    "Limit to observed observations?": "限制在观察到的观测值？",
    "Limit to expected observations?": "限制在预期观测值内？",
    "Shuffle samples?": "是否打乱样本？",
    "Input biom file that is output of normalize_by_copy_number.py": "输入由 normalize_by_copy_number.py 输出的 biom 文件",
    "Normalize by function?": "按功能归一化？",
    "Normalize by OTU?": "按操作分类单元（OTU）标准化？",
    "Calculate accuracy metrics": "计算准确度指标",
    "Metadata category that describes the hierarchy": "描述层级结构的元数据类别",
    "Hierarchy Level": "层级级别",
    "Topology": "拓扑结构",
    "Search for tmRNA genes": "搜索tmRNA基因",
    "Search for tRNA genes": "搜索 tRNA 基因",
    "Search for Metazoan mitochondrial tRNA genes": "搜索多细胞动物线粒体tRNA基因",
    "Search for Mammalian mitochondrial tRNA genes": "搜索哺乳动物线粒体tRNA基因",
    "Search for tRNA genes with introns in anticodon loop": "搜索具有内含子的tRNA基因，这些内含子位于反密码子环中。",
    "Print out secondary structure": "打印出二级结构",
    "Convert output to GFF3": "将输出转换为GFF3格式",
    "Full gene model for GFF3 data": "GFF3数据的完整基因模型",
    "Select Organism": "选择生物体",
    "Select Mode": "选择模式",
    "Disable pseudogene checking": "禁用假基因检查",
    "Show primary and secondary structure components to Cove scores": "显示 Cove 分数的主结构和次级结构组分",
    "Show codons instead of tRNA anticodons": "显示密码子而不是 tRNA 反密码子",
    "GFF3 formatted Gene Calls": "GFF3格式的基因调用",
    "Energy parameters": "能量参数",
    "Fasta file with RNA-sequece": "RNA序列的Fasta文件",
    "RNA-sequence": "RNA序列",
    "Enable segment prediction functionality": "启用片段预测功能",
    "Minimum hairpin size": "最小发夹环大小",
    "Use custom 'segments.xml'-syntaxed file": "使用自定义的 'segments.xml' 语法文件",
    "Input DBN file": "输入 DBN 文件",
    "Regex to capture the targeted location in DBN file": "正则表达式用于捕获DBN文件中的靶向位置",
    "Use segment definition from history": "使用历史中的段定义",
    "Custom 'segments.xml'-syntaxed file": "自定义 'segments.xml' 语法文件",
    "Precision": "精确度",
    "Shuffle each sequence this many times and predict energy of shuffled sequence(s) (default: 0, 0 means disabled)": "将每个序列随机打乱这么多次，并预测打乱序列的能量（默认：0，0表示禁用）",
    "Optional sequences to scan for Segment(Loop)s (FASTA)": "可选序列扫描用于寻找片段（环）的序列（FASTA格式）",
    "Genomic reference FASTA file": "基因组参考FASTA文件",
    "BED file containing the sequence boxes": "包含序列框的BED文件",
    "Maximal distance between the boxes": "最大盒子间距",
    "Extend extracted sequences with this number of bases": "将提取的序列扩展这么多个碱基",
    "The resultes will be filtered based on overlap with annotations in this BED file": "结果将根据与此BED文件中注释的重叠进行过滤",
    "Energy value": "能量值",
    "Sequence of box1 (default = C-box: 'NRUGAUG')": "box1序列（默认=C-box:'NRUGAUG'）",
    "Sequence of box2 (default = D-box: 'CUGA')": "box2序列（默认=D-box:'CUGA'）",
    "Search in the forward direction of the reference sequence": "在参考序列的正向方向中搜索",
    "Search in the reverse complement of the reference sequence": "在参考序列的反向互补中搜索",
    "Whole genome alignment file": "全基因组比对文件",
    "Select the required output files": "选择所需的输出文件",
    "Iterations": "迭代次数",
    "Model conversion Options": "模型转换选项",
    "Outgroup": "外群",
    "Extensive search": "广泛搜索",
    "Really advanced options - change these if you really know what you are doing.": "高级选项 - 如果你确实知道自己在做什么，请修改这些选项。",
    "Tree builder": "树构建器",
    "Minimum SNPs": "最小单核苷酸多态性数目",
    "Filter percentage": "过滤百分比",
    "Minimum window size": "最小窗口大小",
    "Maximum window size": "最大窗口大小",
    "Remove identical sequences": "移除相同的序列",
    "p-Value": "P值",
    "Trimming ration": "修剪比例",
    "Sequencing type": "测序类型",
    "Which strands to search": "要搜索的链",
    "Databases to query": "要查询的数据库",
    "rRNA databases": "rRNA数据库",
    "Include aligned reads in FASTA/FASTQ format?": "包括对齐的FASTA/FASTQ格式的读取？",
    "Include rejected reads file?": "包含被拒绝的读取文件？",
    "Alignment report": "比对报告",
    "Number of output alignments": "输出对齐数",
    "Number of alignments to be reported": "要报告的比对数量",
    "Disable best alignments search": "禁用最佳对齐搜索",
    "Minimum Longest Increasing Subsequence (LIS)": "最小最长递增子序列（LIS）",
    "SW score for a match": "匹配的SW分数",
    "SW penalty for a mismatch": "史密斯-沃特曼（Smith-Waterman）算法中不匹配的罚分",
    "SW penalty for introducing a gap": "引入间隙的Smith-Waterman算法惩罚",
    "SW penalty for extending a gap": "Smith-Waterman 算法中，扩展一个间隙的惩罚",
    "E-value threshold": "E值阈值",
    "SW penalty for ambiguous letters (N's)": "SW 惩罚模糊字母（N 的）",
    "Choose your acquisition mode": "选择您的采集模式",
    "Annotate RData": "注释 RData",
    "Choose your input type method:": "选择您的输入类型方式：",
    "Positive annotate RData": "正向注释 RData",
    "Negative annotate RData": "负性注释 RData",
    "Extraction of CAMERA annotation [get.annot]": "CAMERA注释提取[get.annot]",
    "Retrieves peaks with no eviendence of adduct or isotope": "检索没有加合物或同位素证据的峰",
    "Exclude samples": "排除样本",
    "Samples to be excluded of peak counting to non-annotated peak selection.": "需要排除在非注释峰选择之外的样本。",
    "Database matching [create.reactionM]": "数据库匹配 [create.reactionM]",
    "Search on KEGG database or multiple organisms": "在KEGG数据库中搜索或针对多个生物体进行搜索",
    "Parts per million mass tolerance allowed in the mass search": "在质谱搜索中允许的质量容差（ppm）",
    "Probability calculations matrix export [export.class.table]": "概率计算矩阵导出 [export.class.table]",
    " Calculation of the probability to attribute a mass to a compound": "计算将质量归属于化合物的概率",
    "Logical, check if you want to generate a HTML ProbMetab report": "逻辑，检查是否要生成HTML ProbMetab报告",
    "Calculate the correlations and partial correlations and cross reference then with reactions [reac2cor]": "计算相关性、偏相关性，并将其与反应进行交叉引用 [reac2cor]",
    "Correlation option": "相关性选项",
    "Probability that the correlation is considered significant": "相关性被认为显著的概率",
    "Probability that the partial correlation is considered significant.": "部分相关性被认为显著的概率。",
    "Correlation intensity threshold": "相关性强度阈值",
    "Database file": "数据库文件",
    "Column names": "列名",
    "MS modes": "MS模式",
    "Retention time unit": "保留时间单位",
    "Peakforest security token": "Peakforest 安全令牌",
    "Input file - MZ(/RT) values": "输入文件 - MZ(/RT) 值",
    "Input column names": "输入列名",
    "MS mode": "MS模式",
    "M/Z precision": "质量/电荷比精度",
    "M/Z shift": "质量比/电荷数偏移",
    "M/Z tolerance unit": "质量/电荷容差单位",
    "Chromatographic columns": "色谱柱",
    "RTX": "RTX",
    "RTY": "RTY",
    "RTZ": "RTZ",
    "Precursor match": "前体匹配",
    "List of negative precursors": "负向前体列表",
    "List of positive precursors": "正向前体列表",
    "Multiple matches separator character": "多个匹配分隔符字符",
    "Input sequence file (.fasta format)": "输入序列文件（.fasta格式）",
    "Nucleosomes Concentration": "核小体浓度",
    "Temperature Scaling": "温度缩放",
    "Raw Binding": "原始结合",
    "Orthofinder starting point": "Orthofinder 起始点",
    "Select input fasta files": "选择输入的fasta文件",
    "Input contains nucleotide or amino acid sequences?": "输入包含核苷酸或氨基酸序列？",
    "Sequence search program": "序列搜索程序",
    "Do you want to get the blast results?": "您想要获取BLAST结果吗？",
    "Select the pre-computed blast files": "选择预先计算好的BLAST文件",
    "Select the fasta files": "选择fasta文件",
    "Select the SpeciesIds file": "选择 SpeciesIds 文件",
    "Select the SequencesIds file": "选择序列ID文件",
    "Orthofinder run mode": "Orthofinder 运行模式",
    "Method for gene tree inference": "基因树推断方法",
    "MSA program": "MSA程序",
    "Tree inference method": "树推断方法",
    "Inflation parameter": "膨胀参数",
    "Generate output about gene duplication events": "生成关于基因复制事件的输出",
    "Upload your FASTA file with RNA sequences:": "上传您的包含RNA序列的FASTA文件：",
    "Include kissing hairpins?": "包含接吻发夹？",
    "Include near-optimal local pseudoknots?": "包含近似最优的局部假结？",
    "Include global structure?": "包含全局结构？",
    "Input range coordinate basis": "输入范围坐标基础",
    "Model Ranges": "模型范围",
    "Model Range Start": "模型范围开始",
    "Model Range End": "模型范围结束",
    "Alignment Ranges": "对齐范围",
    "Alignment Range Start": "对齐范围起始",
    "Alignment Range End": "对齐范围结束",
    "Input/Output options": "输入/输出选项",
    "The amrfinderplus database": "amrfinderplus数据库",
    "Choose input type": "选择输入类型",
    "5' flanking sequence size added with sequence match": "5' 侧翼序列大小与序列匹配",
    "Organism options": "生物体选项",
    "Add organism specifications": "添加生物体规格",
    "Taxonomy group point mutation": "分类群组点突变",
    "Report genotypes at all locations screened for point mutations": "报告筛查所有点突变位置的基因型",
    "Report proteins common to a taxonomy group (need the plus and organism options)": "报告属于某个分类群的蛋白质（需要加号和生物选项）",
    "Translation table": "翻译表",
    "Add the plus genes to the report": "将正向基因添加到报告中",
    "Report all equally-scoring BLAST and HMM matches": "报告所有得分相同的BLAST和HMM匹配结果",
    "print hierarchy node (family)": "打印层级节点（家族）",
    "First column name": "第一列名称",
    "Genome annotation in genbank format": "基因组注释在genbank格式",
    "Genome annotation in gff format": "基因组注释在GFF格式",
    "NCBI submission template file": "NCBI提交模板文件",
    "Eggnog-mapper annotations file": "Eggnog-mapper 注释文件",
    "antiSMASH secondary metabolism results": "antiSMASH次级代谢结果",
    "InterProScan5 XML file": "InterProScan5 XML文件",
    "Phobius pre-computed results": "Phobius 预计算结果",
    "BUSCO models": "BUSCO模型",
    "Custom annotations": "自定义注释",
    "locus_tag from NCBI to rename GFF gene models with": "使用NCBI的locus_tag重命名GFF基因模型",
    "Gene/Product names fixed": "基因/产品名称固定",
    "Gene/Product names to remove": "要移除的基因/产品名称",
    "Sequence to align against model": "待比对序列与模型",
    "trim terminal tails of nonaligned residues from alignment": "从比对中修剪未对齐残基的末端尾部",
    "Input genome fasta": "输入基因组fasta文件",
    "Number of sequences to generate": "生成序列的数量",
    "show consensus as 'any' (X/N) unless >= this fraction": "显示一致性为'任何'（X/N），除非大于或等于这个分数",
    "show consensus as upper case if >= this fraction": "显示共识序列为大写字母，如果大于或等于此分数",
    "Expected length of profile": "期望的轮廓长度",
    "Emission profile options": "发射轮廓选项",
    "List of HMM names to extract": "提取的HMM名称列表",
    "Nucleotide sequence to be searched": "待搜索的核苷酸序列",
    "Search with a": "搜索使用一个",
    "Select one of the predefined pattern": "选择一个预定义的模式",
    "Search pattern": "搜索模式",
    "Search pattern on": "搜索模式打开",
    "Input Database": "输入数据库",
    "Input Query": "输入查询",
    "Database is split into subsequences of size:": "数据库被分割成大小为：",
    "E-value": "E值",
    "Percent identity of aligned sequences": "对齐序列的相似性百分比",
    "Print score distribution data for each query to a file": "为每个查询打印得分分布数据到一个文件",
    "Marker": "标记物",
    "Reference Database File (optional):": "参考数据库文件（可选）:",
    "Type of Sequences": "序列类型",
    "Paired Sequencing File(s)": "配对测序文件",
    "Forward Sequencing File(s)": "正向测序文件",
    "Reverse Sequencing File(s)": "反向测序文件",
    "Output header line in the report?": "在报告中输出标题行？",
    "Use Only Expert Curated Results": "仅使用专家筛选结果",
    "Print All Rows, including those without Annotation": "打印所有行，包括那些没有注释的行。",
    "Gene Name Column": "基因名称列",
    "Query Sequence": "查询序列",
    "Search Method": "搜索方法",
    "Custom or built-in HHM index": "自定义或内置的HHM索引",
    "Select ffindex": "选择 ffindex",
    "Custom or built-in cs219 index": "自定义或内置 cs219 索引",
    "E-value cutoff for inclusion in result alignment. (-e)": "结果比对中包含的E值截止值。(-e)",
    "Trainings Dataset": "训练数据集",
    "Set the depth of the ICM": "设置ICM的深度",
    "Set the period of the ICM": "设置ICM的周期",
    "Set the width of the ICM": "设置ICM的宽度",
    "Do not use any input strings with in-frame stop codons": "不要使用任何带有帧内终止密码子的输入字符串",
    "Specify start codons as": "指定起始密码子为",
    "Use Genbank translation table to specify stop codons": "使用Genbank翻译表来指定终止密码子",
    "Specify stop codons as a comma-separated list": "指定终止密码子，以逗号分隔的列表",
    "Interpolated context model (ICM)": "插值上下文模型（ICM）",
    "Set maximum overlap length": "设置最大重叠长度",
    "Set the minimum gene length to n nucleotides": "设置最小基因长度为n个核苷酸",
    "Set threshold score for calling as gene": "设定用于调用基因的阈值分数",
    "Set the GC percentage of the independent model, i.e., the model of intergenic sequence": "设置独立模型的GC百分比，即基因间序列的模型",
    "Assume linear rather than circular genome, i.e., no wraparound": "假设基因组是线性的，而不是环形的，即没有回绕。",
    "Don’t use the independent probability score column at all": "不要使用独立概率分数列",
    "Also score orfs that extend off the end of the sequence(s)": "同时对延伸到序列末端之外的开放阅读框进行评分",
    "Specify start codons as a comma-separated list": "指定起始密码子，以逗号分隔的列表",
    "Report the classic glimmer table output": "报告经典的glimmer表格输出",
    "Output a detailed gene prediction report as separate file": "输出一个详细的基因预测报告作为单独的文件",
    "Set maximum overlap length. Overlaps this short or shorter are ignored.": "设置最大重叠长度。比这个长度短或更短的重叠将被忽略。",
    "Set minimum gene length.": "设置最小基因长度。",
    "Set threshold score for calling as gene. If the in-frame score >= N, then the region is given a number and considered a potential gene.": "设置调用基因的阈值分数。如果帧内分数 >= N，则该区域将被赋予一个编号，并被视为潜在的基因。",
    "Input SnpEff annotated M.tuberculosis VCF(s)": "输入 SnpEff 注释的 M.tuberculosis VCF 文件",
    "TBProfiler Drug Resistance Report (Optional)": "TBProfiler 药物抗性报告（可选）",
    "Filter UPSTREAM, DOWNSTREAM and INTERGENIC variants": "过滤上游、下游和基因间区变异",
    "Specify COMBAT-TB-NeoDB URI": "指定 COMBAT-TB-NeoDB URI",
    "VCF file to be filter": "待过滤的 VCF 文件",
    "Filters to apply": "应用的过滤器",
    "Show options for the filters": "显示过滤器的选项",
    "Region filters to enable": "区域过滤器启用",
    "Window to mask around indels": "围绕插入/缺失的窗口进行掩蔽",
    "Minimum alternate allele percentage to accept": "最小替代等位基因百分比接受值",
    "Minimum read depth (coverage)": "最小读取深度（覆盖度）",
    "Source for spacer sequences": "间隔序列来源",
    "Spacer file": "间隔文件",
    "Sequence reads": "序列读取",
    "Query SITVIT": "查询 SITVIT",
    "Input is assembled sequence": "输入是组装后的序列",
    "Swift mode": "Swift模式",
    "Stringent filtering of reads": "严格过滤读取",
    "Reads are sorted to a reference genome": "读取已排序至参考基因组",
    "Generate plot from SITVIT report output": "生成 SITVIT 报告输出的图表",
    "Genomic DNA": "基因组DNA",
    "Custom library of repeats": "自定义重复序列库",
    "Cutoff score for masking repeats": "重复序列掩蔽的截止分数",
    "Output annotation of repeats in GFF format": "输出以GFF格式注释的重复序列",
    "Ignore stretches of Ns when computing statistics": "在计算统计数据时忽略N的连续序列",
    "Perform softmasking instead of hardmasking": "执行软掩蔽而不是硬掩蔽",
    "Only clip E coli insertion elements": "仅剪切大肠杆菌插入元件",
    "Clip IS elements before analysis": "在分析前剪接IS元素",
    "Skip bacterial insertion element check": "跳过细菌插入元件检查",
    "Only check for rodent specific repeats": "仅检查啮齿动物特有的重复序列",
    "Only check for primate specific repeats": "仅检查特定于灵长类动物的重复序列",
    "No low complexity masking": "无低复杂度掩蔽",
    "No interspersed repeat masking": "无间隔重复掩蔽",
    "No repeat-like-RNA masking": "无重复类RNA掩蔽",
    "Limit masking to (primate) Alu repeats": "限制掩蔽到（灵长类）Alu重复序列",
    "Limit masking to less diverged (younger) repeats": "将掩蔽限制在较少分歧（较年轻）的重复序列上",
    "Search speed vs sensitiviy trade-off": "搜索速度与敏感性权衡",
    "Maximum contiguous sequence searched": "最大连续序列搜索",
    "Select matrices for this GC%": "选择这个GC%的矩阵",
    "Calculate GC % for all sequences": "计算所有序列的GC百分比",
    "Skips cutting of repeats": "跳过重复序列的剪切",
    "Mask with X instead of N characters": "用X替换N字符",
    "Output alignments file": "输出比对文件",
    "Invert alignments in alignment file": "在对齐文件中反转对齐",
    "Output list of potentially polymorphic microsatellites": "输出潜在的多态性微卫星列表",
    "Motif file": "模式文件",
    "Will you select a reference genome from your history or use a installed genome?": "您将从历史记录中选择一个参考基因组，还是使用已安装的基因组？",
    "Output bed format": "输出 BED 格式",
    "Report all motifs even if overlap": "报告所有基序，即使它们重叠",
    "Do not look for motifs in RepeatMasked sequence (lower case sequence in FASTA files)": "不要在RepeatMasked序列（FASTA文件中的小写序列）中寻找基序",
    "Report motif positions based on the 5' end of the motif sequence": "根据基序序列的5'端报告基序位置",
    "Use homer1 instead of homer2": "使用 homer1 而不是 homer2",
    "Peak file": "峰值文件",
    "Do you want to compute CpG/GC content?": "您想要计算CpG/GC含量吗？",
    "GTF file": "GTF文件",
    "Annotation data file for HOMER": "HOMER的注释数据文件",
    "Will you select a reference genome from your history or use an installed genome?": "您将从历史记录中选择一个参考基因组，还是使用已安装的基因组？",
    "Preparsed FASTA": "预解析的FASTA",
    "Will you use a background model?": "您会使用背景模型吗？",
    "Background Peak file": "背景峰文件",
    "comma-separated motif lengths": "逗号分隔的基序长度",
    "Number of motifs to find": "要找到的基序数量",
    "Number of mismatches during global optimisation": "全局优化过程中的不匹配数目",
    "Don't search reverse strand for motifs": "不要在反向链上搜索基序",
    "Don't search for de novo motif enrichment": "不要搜索新发现的基序富集",
    "output RNA motif logos and compare to RNA motif database": "输出RNA基序标志，并与RNA基序数据库进行比较",
    "Known Motif Options/Visualization": "已知模式选项/可视化",
    "Check against motif collects": "检查针对基序集合",
    "Just visualize de novo motifs, don't check similarity with known motifs": "仅可视化去新（de novo）基序，不检查与已知基序的相似性",
    "Scale sequence logos by information content": "将序列标志按信息量进行缩放",
    "Don't search for de novo vs. known motif similarity": "不要搜索新发现的与已知的基序相似性",
    "known motifs to check against de novo motifs": "已知基序与新发现基序进行对比检查",
    "Don't search for known motif enrichment": "不要搜索已知的基序富集",
    "Known motifs to check for enrichment": "已知的用于检查富集的基序",
    "Omit humor": "忽略幽默",
    "Use weblogo/seqlogo/ghostscript to generate logos, default uses SVG now": "使用weblogo/seqlogo/ghostscript生成标志，现在默认使用SVG",
    "Sequence normalization options:": "序列标准化选项：",
    "Use hypergeometric for p-values, binomial is default": "使用超几何分布计算p值，默认是二项分布。",
    "Number of sequences to use for motif finding, default=max(50k, 2x input)": "用于发现基序的序列数量，默认值=max(50k, 2倍输入)",
    "local background size in bp for each side of regions": "每个区域各侧的局部背景大小（以碱基对为单位）",
    "Remove redundant sequences matching greater than # fraction, i.e. -redundant 0.5": "删除匹配度大于#分数的冗余序列，即 -redundant 0.5",
    "maximum percentage of N's in sequence to consider for motif finding": "在序列中用于发现基序的N的最大百分比",
    "motifs to mask before motif finding": "在寻找基序之前需要屏蔽的基序",
    "motifs to optimize or change length of": "要优化或改变长度的基序",
    "randomize target and background sequences labels": "随机化目标和背景序列标签",
    "use file for target and background - first argument is list of peak ids for targets": "使用文件作为目标和背景 - 第一个参数是目标的峰值ID列表",
    "Perform analysis of individual oligo enrichment": "进行单个寡核苷酸富集分析",
    "Number of randomizations to calculate empirical FDR for de novo discovery": "用于计算新发现经验FDR的随机化次数",
    "Which homer version do you want to use": "您想使用哪个版本的HOMER？",
    "length of lower-order oligos to normalize in background": "在背景中用于标准化的低阶寡核苷酸的长度",
    "Max normalization iterations": "最大归一化迭代次数",
    "weight sequences to neutral frequencies, i.e. 25%, 6.25%, etc.": "将序列的权重调整为中性频率，即25%，6.25%等。",
    "lower-order oligo normalization for oligo table, use if -nlen isn't working well": "低阶寡核苷酸标准化用于寡核苷酸表，如果 -nlen 不起作用时使用",
    "skip full masking after finding motifs, similar to original homer": "在找到基序后跳过完整掩蔽，类似于原始的Homer软件。",
    "stop looking for motifs when seed logp score gets above this number": "在种子 logp 分数超过这个数值时停止寻找基序",
    "time spent on local optimization default": "在局部优化上花费的时间，默认",
    "Format of input file": "输入文件的格式",
    "Primary identifier": "主要标识符",
    "Which feature(s) to use?": "使用哪些特征？",
    "Remove any .1, .2, etc. at end of accession numbers, i.e. AT1G01040.2": "移除访问号末尾的任何 .1、.2 等，例如 AT1G01040.2",
    "Remove 'transcript:' and '_T01' style ids": "移除 'transcript:' 和 '_T01' 风格的标识符",
    "distance upstream of TSS to start promoter annotation": "TSS上游距离以开始启动子注释",
    "distance upstream/downstream of TSS to end promoter annotation": "距离转录起始位点（TSS）上游/下游到终止启动子注释的距离",
    "distance upstream of TTS to start TTS annotation": "距离转录起始位点上游开始注释转录起始位点的距离",
    "distance upstream/downstream of TTS to end TTS annotation": "距离转录终止位点（TTS）上游/下游到结束TTS注释的距离",
    "Report file": "报告文件",
    "Analysis software version": "分析软件版本",
    "Reference database version": "参考数据库版本",
    "Report files": "报告文件",
    "Sequence file in GenBank, EMBL or FASTA format": "序列文件在GenBank、EMBL或FASTA格式",
    "Enable masking of regions with unknown nucleotides when finding ORFs": "在寻找开放阅读框时，启用对含有未知核苷酸区域的掩蔽",
    "Enable padding of gene sequences smaller than the CRF window length": "启用对小于CRF窗口长度的基因序列进行填充",
    "Minimum number of genes required for a cluster": "聚类所需的最小基因数",
    "Probability threshold for cluster detection": "聚类检测的概率阈值",
    "Post-processing method for gene cluster validation": "基因簇验证的后处理方法",
    "Number of genes from the contig edges to filter out": "从基因组末端过滤掉的基因数量",
    "Generate an antiSMASH v6 sideload JSON file": "生成一个 antiSMASH v6 侧载 JSON 文件",
    "Protein FASTA File": "蛋白质 FASTA 文件",
    "Type of the input sequences": "输入序列的类型",
    "InterProScan database": "InterProScan 数据库",
    "Applications to run": "运行应用程序",
    "Use applications with restricted license, only for non-commercial use?": "使用有限制许可的应用程序，仅限于非商业用途？",
    "Include pathway information": "包含通路信息",
    "Include Gene Ontology (GO) mappings": "包含基因本体（GO）映射",
    "Provide additional mappings": "提供额外的映射",
    "Contigs": "连续序列",
    "Remove table header": "删除表格标题",
    "CSV output": "CSV输出",
    "Required Inputs": "必需输入",
    "Select reference genome FASTA index (FA/FASTA file)": "选择参考基因组的FASTA索引文件（FA/FASTA文件）",
    "Select sample reads aligned to reference genome (BAM/SAM file)": "选择与参考基因组对齐的样本读段（BAM/SAM文件）",
    "Select reference genome TE annotation (GFF/GTF file)": "选择参考基因组转座元件注释（GFF/GTF文件）",
    "Select TE names": "选择转座元件名称",
    "Short-read sequencing fragment insert size": "短读长测序片段插入大小",
    "Maximum distance between reads for bedtools merge": "对于bedtools merge工具，最大读取间距",
    "Maximum TE target site duplication (TSD) length": "最大转座元件（TE）靶位点重复序列（TSD）长度",
    "Select output format as BED [GTF]": "选择输出格式为 BED [GTF]",
    "Genome fasta input": "基因组fasta输入",
    "Remove incomplete IS elements": "移除不完整的插入序列（IS）元素",
    "Keep the log file": "保留日志文件",
    "Database type": "数据库类型",
    "Genomic sequence (FASTA)": "基因组序列（FASTA）",
    "K-mer size for genome-wide indexing": "基因组范围索引的K-mer大小",
    "Number of bits per bin": "每个bin的比特数",
    "Sample k-mers at a rate 1/2**INT": "以 1/2 的 INT 次方的速率抽样 k-mers",
    "Minimum ORF length to index": "最小开放阅读框长度以进行索引",
    "Pre-indexed genomic database": "预索引的基因组数据库",
    "Protein sequence (FASTA)": "蛋白质序列（FASTA）",
    "Mapping": "映射",
    "No splicing": "无剪接",
    "Max k-mer occurences": "最大k-mer出现次数",
    "Log gap penalty weight": "对齐间隙惩罚权重",
    "Minimum number of syncmers in a chain": "链中同步子序列的最小数量",
    "Minimum chaining score": "最小链式得分",
    "K-mer size for second round of chaining": "第二轮链式处理的K-mer大小",
    "Max extension for second round of chaining": "第二轮链延长的最大延伸长度",
    "Minimum secondary-to-primary score ratio": "次级到初级得分比率最小值",
    "Max secondary alignments to consider": "考虑的最大次级比对数",
    "Gap open penalty": "间隔开放惩罚",
    "Intron open penalty": "内含子开放惩罚",
    "Weight of splice penalty; 0 to ignore splice signals": "剪接惩罚的权重；0表示忽略剪接信号",
    "Frameshift penalty": "移码惩罚",
    "Bonus score for alignment reaching query ends": "对齐达到查询端的额外得分",
    "Splicing model for the target genome": "目标基因组的剪接模型",
    "Prefix for IDs in GFF3 output": "GFF3输出中ID的前缀",
    "Print unmapped proteins": "打印未映射的蛋白质",
    "Outputs per query": "每个查询的输出",
    "Output residue alignment": "输出残基比对",
    "For each protein, only output alignments with a score higher than 'best_score' multiplied by this value": "对于每个蛋白质，只输出得分高于'best_score'乘以这个值的比对结果。",
    "Output if at least this fraction of query is aligned": "如果至少有这个比例的查询序列被比对上，则输出",
    "Output translated protein sequences": "输出翻译后的蛋白质序列",
    "Disable the cs tag": "禁用 cs 标签",
    "Query batch size": "查询批量大小",
    "Gene list": "基因列表",
    "Community detection method": "社区检测方法",
    "Global network null model": "全局网络零模型",
    "Global network null model iterations": "全局网络空模型迭代次数",
    "Global network model number of genes": "全局网络模型中基因的数量",
    "Local network null model": "局部网络零模型",
    "Local network null model iterations": "局部网络空模型迭代",
    "Network plot": "网络图",
    "Network plot width": "网络图宽度",
    "Network in SIF format": "SIF格式的网络",
    "Neighbor list": "邻居列表",
    "Module membership": "模块成员身份",
    "Node type": "节点类型",
    "Input parameters": "输入参数",
    "Choose a fasta or fastq file": "选择一个fasta或fastq文件",
    "PlasmidFinder database": "质粒指纹数据库",
    "Minimal identity": "最小一致性",
    "Output file selection": "输出文件选择",
    "Output files selection": "输出文件选择",
    "Genome sequence to mask": "基因组序列掩蔽",
    "Replicon file": "复制子文件",
    "Thorough local detection": "彻底的局部检测",
    "Default replicons topology": "默认的复制子拓扑结构",
    "Select a topology file from your history": "从您的历史记录中选择一个拓扑文件",
    "Search also for promoter and attI sites?": "搜索启动子和attI位点？",
    "Genbank output?": "GenBank输出？",
    "pdf output?": "PDF 输出？",
    "Advanced Parameters": "高级参数",
    "Attc options": "Attc 选项",
    "Threshold for clustering (in base)": "聚类阈值（以碱基为单位）",
    "Threshold to filter CALIN": "过滤CALIN的阈值",
    "Maximum value for attC size": "attC 大小的最大值",
    "Minimum value for attC size": "attC大小的最小值",
    "Keep palindromes with the highest evalue": "保留具有最高e值的回文序列",
    "Covariance Matrix": "协方差矩阵",
    "Protein options": "蛋白质选项",
    "Just look for attC sites": "只需寻找attC位点",
    "Use the union of the hits": "使用命中结果的并集",
    "Annotate cassettes given HMM profiles": "注释给定HMM轮廓的盒式磁带",
    "Remove log file": "删除日志文件",
    "Which Function should be run": "应该运行哪个功能",
    "Query nucletide sequences": "查询核苷酸序列",
    "Coding sequences file": "编码序列文件",
    "Non coding sequeces file": "非编码序列文件",
    "Start codon": "起始密码子",
    "Stop codons": "终止密码子",
    "Minimum ORF length": "最小开放阅读框长度",
    "Search for ORFs from the anti-sense strand": "搜索反义链上的开放阅读框（ORFs）",
    "Number of ORF candidates reported": "报告的开放阅读框(ORF)候选数",
    "Criteria to select the best ORF": "选择最佳开放阅读框（ORF）的标准",
    "Protein sequences FASTA file": "蛋白质序列FASTA文件",
    "Pfam-A HMM library": "Pfam-A HMM库",
    "Pfam-A HMM Stockholm file": "Pfam-A HMM Stockholm 文件",
    "Predict active site residues": "预测活性位点残基",
    "Active sites file": "活性位点文件",
    "Hmmscan evalue sequence cutoff": "Hmmscan evalue 序列截止值",
    "Hmmscan evalue domain cutoff": "Hmmscan e值域截止",
    "Hmmscan bit score sequence cutoff": "Hmmscan比特分数序列截止值",
    "Hmmscan bit score domain cutoff": "Hmmscan 位分数域截断",
    "Clan overlap": "家族重叠",
    "Show alignment": "显示比对",
    "Bacteriophage genome": "噬菌体基因组",
    "Using built-in pharokka DB": "使用内置的pharokka数据库",
    "User specified gene predictor": "用户指定的基因预测器",
    "Meta mode for metavirome input samples": "Meta模式用于宏病毒组输入样本",
    "E-value threshold for mmseqs2 PHROGs database search. Defaults to 1E-05.": "mmseqs2 PHROGs数据库搜索的E值阈值。默认为1E-05。",
    "Runs - terminase large subunit - re-orientation mode. Single genome input only and requires --terminase_strand and --terminase_start to be specified.": "运行 - 大亚基终止酶 - 重新定位模式。仅限单基因组输入，并且需要指定 --terminase_strand 和 --terminase_start。",
    "Strand of terminase large subunit.": "末端酶大亚基的链。",
    "Start coordinate of the terminase large subunit.": "终止酶大亚基的起始坐标。",
    "Create a Zip archive of the complete output for further investigation.": "创建一个包含完整输出的Zip归档文件以供进一步研究。",
    "How will you provide the input sets?": "您将如何提供输入集？",
    "List of gene identifiers for the background set": "背景集的基因标识符列表",
    "List of sequences in fasta format for the positive set": "正样本集的FASTA格式序列列表",
    "List of sequences in fasta format for the background set": "背景集的fasta格式序列列表",
    "Chop up background regions to match the size of target regions": "将背景区域切分成与目标区域大小相匹配的区域",
    "Search parameters": "搜索参数",
    "Motif lengths to search": "要搜索的基序长度",
    "Number of mismatches to allow during global optimisation": "在全局优化过程中允许的不匹配数",
    "Do not search reverse strand for motifs": "不要在反向链上搜索基序",
    "Do not search for de novo motif enrichment": "不要搜索新发现的基序富集",
    "Output RNA motif logos and compare to RNA motif database": "输出RNA基序标志，并与RNA基序数据库进行比较",
    "Known motif options": "已知的基序选项",
    "Check against motif collect": "检查与motif集合",
    "Search start position, offset from TSS": "搜索起始位置，从转录起始位点（TSS）的偏移量",
    "Search end position, offset from TSS": "搜索结束位置，从TSS（转录起始位点）偏移",
    "Use repeatmasked files": "使用带有重复掩蔽的文件",
    "Use binomial distribution to calculate p-values, instead of hypergeometric.": "使用二项分布计算p值，而不是超几何分布。",
    "Do not remove predetermined redundant promoters/sequences.": "不要移除预定的冗余启动子/序列。",
    "Motifs to mask before motif finding": "在进行基序发现之前需要屏蔽的基序",
    "Motifs to optimize or change the length of": "需要优化或改变长度的基序",
    "Randomize the labels of target and background sequences": "将目标序列和背景序列的标签随机化",
    "Calculate empirical FDR for de novo discovery by doing this many calculations": "计算经验FDR（错误发现率）用于新的发现，通过进行这么多次计算来完成。",
    "Remove sequences shorter than this (bp)": "删除短于这个长度的序列（碱基对）",
    "Remove sequences longer than this (bp)": "删除大于这个长度的序列（碱基对）",
    "Lower-order oligo normalization for oligo table": "低阶寡核苷酸标准化用于寡核苷酸表",
    "Skip full masking after finding motifs, similar to original homer": "在找到基序后跳过完整掩蔽，类似于原始的Homer软件  解释： 1. \"Skip full masking after finding motifs\" 翻译为 \"在找到基序后跳过完整掩蔽\"，其中 \"Skip full masking\" 表示跳过某个步骤，\"after finding motifs\" 表示在找到基序之后。 2. \"similar to original homer\" 翻译为 \"类似于原始的Homer软件\"，其中 \"similar to\" 表示类似于，\"original homer\" 表示原始的Homer软件，Homer是一个生物信息学软件，所以保持原文不翻译。",
    "Stop looking for motifs when seed logp score gets above this number.": "当种子对数几率分数超过这个数值时停止寻找基序。",
    "Time to spend on local optimization default": "在局部优化上花费的时间默认值",
    "Produce peak file of promoters": "生成启动子的峰值文件",
    "Save the summary in html files": "将摘要保存在 HTML 文件中",
    "Save the 4xn matrices for discovered motifs": "保存发现的基序的4×n矩阵",
    "Select transcriptomics data": "选择转录组学数据",
    "Transcriptomics data": "转录组学数据",
    "Gene ID format in transcriptomics data": "转录组数据中的基因ID格式",
    "Select proteomics data": "选择蛋白质组学数据",
    "Proteomics data": "蛋白质组学数据",
    "Gene ID format in proteomics data": "蛋白质组学数据中的基因ID格式",
    "Select metabolomics data": "选择代谢组学数据",
    "Metabolomics data": "代谢组学数据",
    "Metabolite ID format": "代谢物ID格式",
    "Supported organisms": "支持的生物",
    "Pathway databases": "通路数据库",
    "Combine p-values method": "合并 P 值方法",
    "P-values correction method": "P值校正方法",
    "Sequence file (nucleotides)": "序列文件（核苷酸）",
    "Look for ORFs or CDSs": "寻找开放阅读框（ORFs）或编码序列（CDSs）",
    "Minimum length ORF/CDS (in amino acids, e.g. 30 aa = 90 bp plus any stop codon)": "最小长度的开放阅读框/编码序列（以氨基酸计，例如30个氨基酸=90个碱基对加上任何终止密码子）",
    "Strand to search": "搜索链",
    "Minimum number of bases upstream of CDS for gap (--lookahead_min)": "CDS上游间隙的最小碱基数目（--lookahead_min）",
    "Maximum number of bases upstream of CDS for gap (--lookahead_max)": "CDS上游最大碱基数用于间隔（--lookahead_max）",
    "Automatically add RBSs to input GFF3": "自动向输入的GFF3文件添加RBS位点",
    "Only report best hits (--top_only)": "只报告最佳命中（--top_only）",
    "Feature Types": "特征类型",
    "Feature type": "特征类型",
    "Invert the filter": "反转过滤器",
    "Remove 'remark' features": "移除“备注”特征",
    "Remove 'region' features": "移除 'region' 特征",
    "Glimmer Output": "Glimmer 输出",
    "Genomic sequence": "基因组序列",
    "Available lineages": "可用谱系",
    "Batch size": "批量大小",
    "Post-processing": "后处理",
    "Edge threshold": "边缘阈值",
    "Peak threshold": "峰值阈值",
    "Minimum coding length": "最小编码长度",
    "Blastp Results": "Blastp 结果",
    "Separation Minimum": "最小分离",
    "Separation Maximum": "分离最大值",
    "Percent Identity Threshold": "相似度百分比阈值",
    "Filter for specific feature types": "过滤特定特征类型",
    "Remove description (use if blasting)": "移除描述（如果进行BLAST比对时使用）",
    "Naive orf calls": "天真的开放阅读框（ORF）调用",
    "Minimum distance in codons to start of lipobox": "lipobox起始处的最小密码子距离",
    "Maximum distance in codons to start of lipobox": "lipobox起始处的最大距离（以密码子为单位）",
    "Tag to update": "更新标签",
    "String prefix": "字符串前缀",
    "Number of leading zeros/padding": "前导零的数量/填充",
    "Force Updated Tags to initially match in addition to location checks.": "强制更新标签，除了位置检查外，还要使它们最初匹配。",
    "New ID": "新ID",
    "Sequence Component Selections": "序列组件选择",
    "Data Source": "数据来源",
    "Reverse + Complement": "反转 + 互补",
    "Custom Sequence": "自定义序列",
    "Minimum distance for SDs (bp)": "最小SDs距离（碱基对）",
    "Maximum distance for SDs (bp)": "最大SDs距离（碱基对）",
    "Minimum length of naively called ORFs in gaps (in AAs)": "在间隙中天真地称为ORFs的最小长度（以氨基酸为单位）",
    "Excessive overlap distance (non-divergent, bp)": "过度的重叠距离（非发散的，碱基对）",
    "Excessive overlap distance (divergent, bp)": "过度重叠距离（发散的，碱基对）",
    "Excessive gap distance (non-divergent, bp)": "过度间隔距离（非发散的，碱基对）",
    "Excessive gap distance (divergent, bp)": "过度间隔距离（发散的，碱基对）",
    "Report Format": "报告格式",
    "Extra sub-feature types to include (Comma-separated, Will always include gene)": "要包括的额外子特征类型（用逗号分隔，将始终包括基因）",
    "GAF Data": "基因注释格式数据",
    "GAF Columns": "GAF列",
    "Search sub-features for fields if gene does not define them": "在基因未定义它们的情况下，搜索字段的子特征",
    "Mapping Key": "映射键",
    "Select a database": "选择一个数据库",
    "Save filtered reads ?": "保存过滤后的读取序列？",
    "Maximum gap for blocks to be contigous, measured in multiples of the k-mer size": "最大间隔，用于块连续，以k-mer大小的倍数来测量",
    "Flanking region around a block to use for mapping": "用于映射的块周围的区域",
    "Minimum block size in bases": "最小区块大小（以碱基为单位）",
    "Minimum No. of kmers matching a read": "最小k-mer匹配读段数",
    "Minimum No. of kmers matching a read in 1st pass": "在第一次通过时，匹配读取的最小kmer数量",
    "Dont count kmers occuring more than this many times": "不要计算出现次数超过这个值的kmer",
    "Print ST every this number of reads": "打印ST，每个这个数量的读取",
    "k-mer size": "k-mer大小",
    "Genes which are not divisible by 3 are excluded": "不可被3整除的基因被排除在外",
    "Turn on debugging": "开启调试",
    "Input isoform file": "输入异构体文件",
    "Only compute a list of conserved HOGs": "仅计算保守的同源基因组（HOGs）列表",
    "OMAmer database": "OMAmer 数据库",
    "Taxonomic rank": "分类等级",
    "Target output": "目标输出",
    "Translation table to use": "使用翻译表",
    "Remove stop characters": "移除终止字符",
    "Convert first residue to Met": "将第一个残基转换为Met（甲硫氨酸）",
    "iLocus parsing options": "iLocus 解析选项",
    "Gene loci extension": "基因位点扩展",
    "Annotation mode": "注释模式",
    "Do not report intergenic iLoci": "不要报告基因间 iLoci",
    "Refine options": "细化选项",
    "Mode for handling of overlpping genes": "处理重叠基因的模式",
    "Use CDS rather than UTRs": "使用编码序列（CDS）而不是非编码区（UTRs）",
    "Minimum number of overlapping nucleotides": "最小重叠核苷酸数",
    "Comma-separated list of feature types to use": "使用逗号分隔的特征类型列表",
    "Create a parent for features that lack a parent feature": "为缺少父特征的特征创建一个父特征",
    "Correct erroneously labeled pseudogenes": "修正错误标记的伪基因",
    "Retain original feature IDs": "保留原始特征ID",
    "ID format for newly created locy": "新创建的locy的ID格式",
    "The bakta database": "巴克塔数据库",
    "Select genome in fasta format": "选择以fasta格式的基因组",
    "Minimum contig size": "最小连通子序列大小",
    "Optional organism options": "可选的生物体选项",
    "Specify genus name": "指定属名",
    "Specify species name": "指定物种名称",
    "Specify strain name": "指定菌株名称",
    "Specify plasmid name": "指定质粒名称",
    "Optional annotation": "可选注释",
    "Complete replicons": "完整复制子",
    "Prodigal file": "Prodigal 文件",
    "Keep original contig header (--keep-contig-headers)": "保持原始的基因组片段头信息（--keep-contig-headers）",
    "Replicon information table (tsv/csv)": "复制子信息表（tsv/csv）",
    "Force Genbank/ENA/DDJB compliance": "强制遵守Genbank/ENA/DDJB规范",
    "Protein fasta file": "蛋白质fasta文件",
    "Metagenome mode": "宏基因组模式",
    "Pre-annotated regions": "预注释区域",
    "Workflow option to skip steps": "工作流程选项以跳过步骤",
    "Select steps to skip": "选择要跳过的步骤",
    "Selection of the output files": "选择输出文件",
    "Input genome file": "输入基因组文件",
    "The mode of evaluation": "评估方式",
    "Specify the contigs to be evaluated": "指定要评估的基因组片段",
    "Choose the BUSCO database to be used": "选择要使用的 BUSCO 数据库",
    "Proteins in FASTA format": "FASTA格式的蛋白质",
    "Output the HMMer domain hits table?": "输出HMMer域命中表格？",
    "Infer gene feature for transcripts lacking it": "推断缺失基因特征的转录本",
    "Reset the source of each feature": "重置每个特征的来源",
    "GFF3 Annotations A": "GFF3 注释 A",
    "GFF3 Annotations B": "GFF3 注释 B",
    "Adjacency Window Size": "邻接窗口大小",
    "Only allow adjacencies for features on the same strand": "仅允许相同链上的特征之间存在邻接关系",
    "Genome fasta file": "基因组fasta文件",
    "Node name": "节点名称",
    "Scientific species name": "科学物种名称",
    "Specify input file": "指定输入文件",
    "Specify training file": "指定训练文件",
    "Specify output format": "指定输出格式",
    "Specify mode": "指定模式",
    "Specify a translation table to use": "指定要使用的转换表",
    "Closed ends": "封闭末端",
    "Force Prodigal to scan for motifs": "强制Prodigal扫描模式",
    "Treat runs of N as masked sequence": "将连续的N视为掩蔽序列",
    "Alignment GFF3 file": "比对GFF3文件",
    "Genes GFF3 file": "基因 GFF3 文件",
    "Weights for calculating integrity score": "计算完整性得分的权重",
    "Introns confirmed, or % expected CDS lenght for single-exon genes": "确认内含子，或单外显子基因的预期CDS长度百分比",
    "Exon coverage": "外显子覆盖度",
    "% expected 5’ UTR length": "预期的5'非翻译区长度",
    "% expected 3’ UTR lenght": "% 预期的3'非翻译区长度",
    "Expected feature lenghts for calculating integrity score": "计算完整性得分的预期特征长度",
    "Expected CDs length (in bp) (-c)": "预期的CDS长度（以碱基对为单位）(-c)",
    "Expected 5’ UTR length (-5)": "预期的5' UTR长度（-5）",
    "Expected 3’ UTR length (-3)": "预期的3'非翻译区长度（-3）",
    "Gene BED": "基因 BED",
    "Minimum length of peptide translation to report": "报告肽翻译的最小长度",
    "Only report the first unique occurrence of an alternate sequence": "只报告替代序列的第一个唯一出现情况",
    "Do not report any alternate sequences that match a provided canonical CDS": "不要报告任何与提供的规范CDS匹配的替代序列",
    "Create log file": "创建日志文件",
    "Source for Sequence Data": "序列数据来源",
    "Select FASTA nucleotide sequences": "选择FASTA核苷酸序列",
    "FASTA nucleotide sequences": "FASTA 核苷酸序列",
    "Taxonomy ID to list in output": "输出中列出的分类学ID",
    "Organelle to list in output": "在输出中列出的细胞器",
    "Division to list in output": "输出中要列出的分区",
    "Assembly to list in output": "组装到输出列表",
    "Species to list in output": "输出中要列出的物种",
    "Bicondon rates input": "Bicondon 输入速率",
    "Taxonomy ID of interest": "感兴趣的分类学ID",
    "Organelle of interest": "感兴趣的细胞器",
    "Find ORFs in:": "在以下内容中查找开放阅读框(ORFs)：",
    "Specify genetic code": "指定遗传密码",
    "Select outputs": "选择输出",
    "Select strand": "选择链",
    "Minimum length of ORFs": "开放阅读框的最小长度",
    "Maximum length of ORFs": "开放阅读框的最大长度",
    "Start codon(s) to use": "使用的起始密码子",
    "Stop codon(s) to use": "使用终止密码子",
    "Ignore case?": "忽略大小写？",
    "Output ORFs with Start but no Stop?": "输出具有起始密码子但没有终止密码子的开放阅读框？",
    "Output ORFs with Stop but no Start?": "输出带有终止密码子但没有起始密码子的开放阅读框？",
    "Output ORFs bound by Stop codons?": "输出由终止密码子界定的开放阅读框？",
    "Include stop codon in the results?": "在结果中包含终止密码子？",
    "Organism database": "生物体数据库",
    "Rank cutoff": "排名截止",
    "Subject coverage cutoff": "主题覆盖度截止值",
    "Only use orthologs": "仅使用直系同源基因",
    "Foreground": "前景",
    "Background": "背景",
    "Background file": "背景文件",
    "Species abbreviation": "物种缩写",
    "Statistical test method": "统计检验方法",
    "FDR": "FDR（False Discovery Rate）指的是错误发现率，是一种用于多重假设检验中控制第一类错误的统计方法。",
    "Cutoff": "截断值",
    "PDF output?": "PDF输出？",
    "Subsequence length: how much of the genome the Neural Network can see at once": "子序列长度：神经网络一次能看到基因组的多少部分",
    "Enable overlapping step after predictions": "启用预测后的重叠步骤",
    "Overlap offset: Smaller values may lead to better predictions but will take longer": "重叠偏移量：较小的值可能会带来更好的预测，但需要更长的时间。",
    "Overlap core length: Predicted subsequences will be cut to this length to increase prediction quality. Smaller values may lead to better predictions but will take longer": "重叠核心长度：预测的子序列将被剪切到这个长度以提高预测质量。较小的值可能会导致更好的预测，但需要更长的时间。",
    "ABRicate Report file": "ABRicate 报告文件",
    "AMRFinderPlus Report file": "AMRFinderPlus 报告文件",
    "AMRplusplus gene report": "AMRplusplus 基因报告",
    "ARIBA output": "ARIBA 输出",
    "c-SSTAR output": "c-SSTAR 输出",
    "DeepARG output": "DeepARG 输出",
    "fARGene HMM Search Result": "fARGene HMM搜索结果",
    "GROOT output": "GROOT 输出",
    "KmerResistance gene report": "Kmer抗性基因报告",
    "Mykrobe Report file": "Mykrobe 报告文件",
    "PointFinder results": "PointFinder结果",
    "ResFams results": "ResFams 结果",
    "ResFinder results": "ResFinder 结果",
    "RGI output": "RGI输出",
    "SraX detected ARGs": "SraX 检测到 ARGs",
    "SRST2 report": "SRST2报告",
    "StarAMR ResFinder output": "StarAMR ResFinder输出",
    "TBProfiler output": "TBProfiler输出",
    "Input counts data": "输入计数数据",
    "Type of analysis to perform": "要执行的分析类型",
    "Which sample in the input would you like to analyze?": "您想要分析输入中的哪个样本？",
    "Assign each sample to one of two groups": "将每个样本分配到两个组中的一个",
    "Counts threshold": "计数阈值",
    "Select RBPBench program mode": "选择 RBPBench 程序模式",
    "Genomic regions BED file": "基因组区域 BED 文件",
    "Select reference genome sequence (according to provided BED file)": "选择参考基因组序列（根据提供的 BED 文件）",
    "Select built-in genome sequence": "选择内置的基因组序列",
    "Select genome sequence in FASTA format from history": "从历史记录中选择FASTA格式的基因组序列",
    "Select RBP binding motif database": "选择 RBP 结合基序数据库",
    "Select RBPs for motif search": "选择用于基序搜索的RBPs（RNA结合蛋白）",
    "Select RBPs": "选择 RBPs",
    "Custom motif database ID": "自定义基序数据库ID",
    "Custom motif database info table file": "自定义基序数据库信息表文件",
    "Custom motif database MEME / DREME XML file": "自定义基序数据库 MEME / DREME XML 文件",
    "Custom motif database covariance models file": "自定义基序数据库共价模型文件",
    "Specify RBP IDs from custom motif database": "指定自定义基序数据库中的RBP ID",
    "Add user-supplied motifs": "添加用户提供的基序",
    "DREME or MEME motifs XML file": "DREME或MEME基序XML文件",
    "RBP ID": "RBP ID",
    "Covariance models file": "协方差模型文件",
    "Method ID": "方法ID",
    "Data ID": "数据ID",
    "Motif search settings": "Motif搜索设置",
    "Up- and downstream extension of genomic regions": "基因组区域的上游和下游扩展",
    "FIMO p-value threshold": "FIMO p值阈值",
    "BED score column used for p-value calculations": "用于 p 值计算的 BED 分数列",
    "Treat genomic regions in BED file as NOT strand-specific": "将BED文件中的基因组区域视为非链特异性",
    "Count each genomic region twice for RBP hit statistics": "在RBP命中统计中，每个基因组区域计算两次。",
    "Provide FIMO nucleotide frequencies file": "提供 FIMO 核苷酸频率文件",
    "HTML report options": "HTML报告选项",
    "GTF file to add genomic annotations to input regions": "GTF文件用于向输入区域添加基因组注释",
    "Transcript IDs file": "转录本ID文件",
    "List of transcript biotypes": "转录本生物型列表",
    "Upset plot minimum degree parameter": "Upset图最小度参数",
    "Upset plot maximum degree parameter": "Upset图最大度参数",
    "Upset plot minimum subset size parameter": "Upset图最小子集大小参数",
    "Set reference RBP ID for plotting motif distances": "设置参考 RBP ID 以绘制基序距离",
    "Motif co-occurrence minimum pair count": "Motif共现最小配对计数",
    "RBP co-occurrence minimum pair count": "RBP共现最小对数",
    "Store JS code for table sorting inside HTML?": "当然，你可以将JavaScript代码存储在HTML文件中以对表格进行排序。以下是如何实现的示例：  ```html <!DOCTYPE html> <html> <head> <style> table { border-collapse: collapse; width: 100%; } th, td { text-align: left; padding: 8px; } tr:nth-child(even){background-color: #f2f2f2;} </style> </head> <body>  <h2>可排序表格</h2>  <table id=\"myTable\"> <tr> <th onclick=\"sortTable(0)\">姓名</th> <th onclick=\"sortTable(1)\">年龄</th> <th onclick=\"sortTable(2)\">国家</th> </tr> <tr> <td>张三</td> <td>32</td> <td>中国</td> </tr> <tr> <td>李四</td> <td>25</td> <td>美国</td> </tr> <tr> <td>王五</td> <td>28</td> <td>英国</td> </tr> </table>  <script> function sortTable(n) { var table, rows, switching, i, x, y, shouldSwitch, dir, switchcount = 0; table = document.getElementById(\"myTable\"); switching = true; dir = \"asc\"; while (switching) { switching = false; rows = table.rows; for (i = 1; i < (rows.length - 1); i++) { shouldSwitch = false; x = rows[i].getElementsByTagName(\"TD\")[n]; y = rows[i + 1].getElementsByTagName(\"TD\")[n]; if (dir == \"asc\") { if (x.innerHTML.toLowerCase() > y.innerHTML.toLowerCase()) { shouldSwitch= true; break; } } else if (dir == \"desc\") { if (x.innerHTML.toLowerCase() < y.innerHTML.toLowerCase()) { shouldSwitch = true; break; } } } if (shouldSwitch) { rows[i].parentNode.insertBefore(rows[i + 1], rows[i]); switching = true; switchcount ++; } else { if (switchcount == 0 && dir == \"asc\") { dir = \"desc\"; switching = true; } } } } </script>  </body> </html> ```  在上面的示例中，我们有一个表格，其中包含三列：姓名、年龄和国家。每列的标题是一个可点击的按钮，当点击时，它将调用`sortTable`函数，该函数接受列索引作为参数。  `sortTable`函数通过遍历表格的行并比较要排序的列中的值来工作。如果值为升序，则函数会交换行以使值按升序排序。如果值为降序，则函数会交换行以使值按降序排序。  请注意，此示例使用内联CSS进行基本的表格样式设置。您可以根据自己的喜好修改样式。",
    "Specify plotly JS code source": "指定 Plotly JS 代码源",
    "Store plotly plotting code inside HTML?": "在HTML中存储Plotly绘图代码。",
    "Output HTML report?": "输出 HTML 报告？",
    "Plot RBP motifs?": "绘制 RBP 基序？",
    "Output filtered genomic regions BED + FASTA files": "输出过滤后的基因组区域 BED 和 FASTA 文件",
    "Output motif hits BED file": "输出基序命中的BED文件",
    "Output contingency table containing co-occurrence p-values": "输出包含共现p值的列联表",
    "Output genomic region annotations table file?": "输出基因组区域注释表文件？",
    "Select RBP for motif search": "选择RBP进行基序搜索",
    "Select reference genome sequence (according to the provided BED files)": "选择参考基因组序列（根据提供的BED文件）",
    "Data collection containing genomic regions BED files": "包含基因组区域的BED文件数据收集",
    "Provide batch processing table file": "提供批量处理表格文件",
    "Select reference genome sequence (according to BED file)": "选择参考基因组序列（根据BED文件）",
    "Nucleotide distribution plot settings": "核苷酸分布图设置",
    "Define zero position for plotting": "定义绘图的零点位置",
    "Up- and downstream extension of defined genomic positions": "上下游扩展定义的基因组位置",
    "Plot as PDF?": "绘制为PDF？",
    "Output genomic regions BED + FASTA files": "输出基因组区域的BED和FASTA文件",
    "Motif search results": "Motif搜索结果",
    "Output compared motif hits table file": "输出比较基序命中表格文件",
    "Output compared motif hits BED file": "输出比较的基序命中 BED 文件",
    "Output comparison statistics table file": "输出比较统计表文件",
    "RDATA file generated by riboWaltz tool": "由 riboWaltz 工具生成的 RDATA 文件",
    "Include read length distribution plots?": "包含读长分布图？",
    "Plot options": "绘图选项",
    "Confidence level": "置信水平",
    "How to handle multiple samples and replicates?": "如何处理多个样本和重复实验？",
    "How to organize and display multiple bar plots??": "如何组织和展示多个条形图？",
    "Inlcude plots about abundance reads extremities around start and stop codons?": "包括关于丰度读取在起始密码子和终止密码子周围的极端值的图表？",
    "5' UTR region flanking the start codon": "5'非翻译区区域环绕起始密码子",
    "CDS region flanking both the start and stop codon": "编码序列区域，位于起始和终止密码子两侧",
    "3' UTR region flanking the stop codon": "3'非翻译区区域，位于终止密码子的两侧",
    "Inlcude plots that show P-sites per region?": "包括显示每个区域的P位点的图表？",
    "Inlcude plots with Trinucleotide periodicity?": "包括三核苷酸周期性的图表？",
    "Region(s) of the apartment to be analyzed": "要分析的公寓区域",
    "Read length(s) to be included in the analysis": "包含在分析中的读取长度",
    "Read lengths ranging from": "读取长度范围从",
    "Read lengths ranging to": "读取长度范围到",
    "Inlcude metaplots?": "包括元图？",
    "Normalize the metaprofile(s) such that the area under the curve(s) is 1?": "将元轮廓（metaprofile）标准化，使得曲线下面积为1。",
    "Inlcude plots with codon usage?": "包含密码子使用情况的图表？",
    "normalize the 64 codon usage indexes for the corresponding codon frequencies in coding sequences?": "将64个密码子使用指数标准化为编码序列中相应密码子频率？",
    "Label the dots in the scatter plot?": "在散点图中标记点。",
    "how many dots in the scatter plot should be labeled?": "在散点图中应该标记多少个点？",
    "STAR Aligned.out.sam": "STAR 对齐后的输出文件 Aligned.out.sam",
    "STAR Chimeric.out.sam": "STAR 嵌合体输出 .sam",
    "File containing blacklisted ranges.": "包含黑名单范围的文件。",
    "File containing protein domains": "包含蛋白质域的文件",
    "File containing known fusions": "包含已知融合的文件",
    "File containing tag names for a fusion.": "包含融合标签名称的文件。",
    "Use whole-genome sequencing data": "使用全基因组测序数据",
    "whole-genome sequencing structural variant data": "全基因组测序结构变异数据",
    "Max genomic breakpoint distance": "最大基因组断点距离",
    "Arriba Options": "Arriba 选项",
    "Names of features in the GTF annotation file": "GTF注释文件中的特征名称",
    "Whether a strand-specific protocol was used for library preparation": "是否使用了特定于链的协议进行文库制备",
    "Comma/space-separated list of interesting contigs": "逗号/空格分隔的有趣片段列表",
    "Comma/space-separated list of viral contigs": "逗号/空格分隔的病毒基因组序列列表",
    "Disable filters": "禁用过滤器",
    "Max e-value threshold": "最大 E 值阈值",
    "Min supporting reads": "最小支持读段",
    "Max mismappers threshold": "最大错误映射器阈值",
    "Max homologs identity threshold": "最大同源物一致性阈值",
    "Homopolymer length": "同聚物长度",
    "Read-through distance": "读取距离",
    "Min anchor length": "最小锚点长度",
    "Many spliced events": "许多剪接事件",
    "Max kmer content": "最大k-mer含量",
    "Max mismatchrpvalue threahold": "最大错配rp值阈值",
    "Single-end fragment length": "单端片段长度",
    "Max reads": "最大读取数",
    "Quantile": "分位数",
    "Exonic fraction": "外显子分数",
    "top N viral contigs": "前N个病毒基因组片段",
    "Covered fraction": "覆盖分数",
    "Maximum length of internal tandem duplications": "最大内部串联重复长度",
    "Required fraction of supporting reads to report an internal tandem duplication": "必需的支持读取的分数以报告内部串联重复",
    "Required number of supporting reads to report an internal tandem duplication": "需要支持报告内部串联重复的读段数",
    "Use aligners duplicate marking": "使用比对工具的重复标记",
    "Fill all fusion.discarded.tsv columns": "填充所有 fusion.discarded.tsv 列",
    "Fill fusion transcript gaps from the assembly": "填充组装中的融合转录本间隙",
    "Output fusions.discarded.tsv": "输出融合基因.丢弃.tsv",
    "Output fusions.vcf": "输出融合.vcf",
    "Output fusion BAMs": "输出融合BAMs",
    "Generate visualization": "生成可视化",
    "Cufflinks assembled transcripts": "Cufflinks 组装转录本",
    "Cuffcompare tracking file": "Cuffcompare 追踪文件",
    "Sample Number": "样本编号",
    "A set of target references (annotation) in multi-FASTA format": "一组目标参考序列（注释）以多FASTA格式",
    "Alignments in the BAM or SAM format": "BAM或SAM格式的比对结果",
    "Use Additional Parameters?": "使用额外参数？",
    "Additional batch EM rounds": "额外的批EM轮次",
    "Additional online EM rounds": "额外的在线EM轮次",
    "Calculate the covariance between targets and outputs?": "计算目标和输出之间的协方差？",
    "Specifies the mean fragment length": "指定平均片段长度",
    "Specified the fragment length standard deviation": "指定片段长度标准差",
    "Use Reference Annotation": "使用参考注释",
    "Reference Annotation": "参考注释",
    "Use Sequence Data": "使用序列数据",
    "Minimum isoform fraction": "最小异构体分数",
    "SAM or BAM file of aligned RNA-Seq reads": "RNA-Seq 比对后的 SAM 或 BAM 文件",
    "Max Intron Length": "最大内含子长度",
    "Min Isoform Fraction": "最小异构体分数",
    "Pre MRNA Fraction": "前mRNA分数",
    "Count hits compatible with reference RNAs only": "计算仅与参考RNA兼容的命中数",
    "3prime overhang tolerance": "3'端突出容忍度",
    "Intronic overhang tolerance": "内含子突出容忍度",
    "Disable tiling of reference transcripts": "禁用参考转录本的平铺",
    "Perform Bias Correction": "执行偏差校正",
    "Reference sequence data": "参考序列数据",
    "Use multi-read correct": "使用多读校正",
    "Apply length correction": "应用长度校正",
    "Global model (for use in Trackster)": "全局模型（用于Trackster）",
    "Set advanced Cufflinks options": "设置高级Cufflinks选项",
    "Library prep used for input reads": "用于输入读取的文库制备",
    "Mask File": "掩码文件",
    "Inner mean distance": "内部平均距离",
    "Inner distance standard deviation": "内部距离标准差",
    "Max MLE iterations": "最大最大似然估计迭代次数",
    "Alpha value for the binomial test used during false positive spliced alignment filtration": "二项检验中用于假阳性剪接对齐过滤的alpha值",
    "percent read overhang taken as suspiciously small": "读取悬空部分的百分比被认为是可疑的小",
    "Maximum genomic length of a given bundle": "给定捆绑包的最大基因组长度",
    "Maximum number of fragments per locus": "每个位点的最大片段数",
    "Minimal allowed intron size": "最小允许的内含子大小",
    "Minimum average coverage required to attempt 3prime trimming.": "尝试进行3'端修剪所需的最小平均覆盖度。",
    "The fraction of average coverage below which to trim the 3prime end of an assembled transcript.": "在组装好的转录本的3'端修剪时，低于平均覆盖度的比例。",
    "Transcripts": "转录本",
    "Omit Tabular Datasets": "忽略表格数据集",
    "Generate SQLite": "生成 SQLite",
    "Library normalization method": "库标准化方法",
    "Dispersion estimation method": "离散度估计方法",
    "False Discovery Rate": "错误发现率",
    "Min Alignment Count": "最小比对计数",
    "Include Read Group Datasets": "包含读取组数据集",
    "Include Count Based output files": "包括基于计数的输出文件",
    "apply length correction": "应用长度校正",
    "Set Additional Parameters for single end reads? (not recommended for paired-end reads)": "为单端读取设置额外参数？（不推荐用于配对端读取）",
    "Average Fragment Length": "平均片段长度",
    "Fragment Length Standard Deviation": "片段长度标准差",
    "Set Advanced Cuffdiff parameters?": "设置高级Cuffdiff参数？",
    "Perform Time Series analysis": "进行时间序列分析",
    "Hits included in normalization": "包含在标准化中的命中",
    "Number of fragment generation samples": "片段生成样本数",
    "Number of fragment assignment samples per generation": "每一代片段分配样本数",
    "Minimal Replicates for isoform shift testing": "最小重复次数用于异构体转移测试",
    "Ignore reference transcripts that are not overlapped by any input transfrags": "忽略未被任何输入转录片段重叠的参考转录本",
    "Ignore input transcripts that are not overlapped by any reference transcripts": "忽略那些没有被任何参考转录本重叠的输入转录本",
    "discard (ignore) single-exon transcripts": "丢弃（忽略）单外显子转录本",
    "Max. Distance for assessing exon accuracy": "最大距离，用于评估外显子准确性",
    "Max.Distance for transcript grouping": "最大距离用于转录本分组",
    "discard intron-redundant transfrags sharing 5'": "丢弃共享5'端的内含子冗余转录片段",
    "Factor level": "因子水平",
    "Specify a factor level, typical values could be 'tumor', 'normal', 'treated' or 'control'": "指定一个因子水平，典型的值可以是“肿瘤”、“正常”、“治疗”或“对照”",
    "Counts file(s)": "计数文件",
    "Try to find at least this many factors of unwanted variation": "尝试找到至少这么多不需要变化的因素",
    "Try to find at most this many factors of unwanted variation": "尝试找到最多这个数量的不需要变化的因素",
    "Ignore genes with fewer than this many counts on average": "忽略平均计数少于这个数值的基因",
    "Files have header?": "文件有标题吗？",
    "Choice of Input data": "输入数据的选择",
    "Program used to generate TPMs": "用于生成TPMs的程序",
    "Gene mapping format": "基因映射格式",
    "GTF/GFF3 file with Transcript - Gene mapping": "带有转录本-基因映射的GTF/GFF3文件",
    "Tabular file with Transcript - Gene mapping": "带转录本-基因映射的表格文件",
    "Visualising the analysis results": "可视化分析结果",
    "Output RUVSeq normalized count tables": "输出 RUVSeq 标准化计数表",
    "Transcripts assembly": "转录本组装",
    "Alignment file": "比对文件",
    "Gene annotation file": "基因注释文件",
    "Select built-in genome": "选择内置基因组",
    "Using locally cached annotation": "使用本地缓存的注释",
    "Create gene-length file": "创建基因长度文件",
    "Does the input have read pairs?": "输入是否有配对的读段？",
    "Only allow fragments with both reads aligned": "只允许两个读取都对齐的片段",
    "Exclude chimeric fragments": "排除嵌合片段",
    "Check paired-end distance": "检查配对末端距离",
    "Minimum fragment/template length.": "最小片段/模板长度。",
    "Maximum fragment/template length.": "最大片段/模板长度。",
    "Read filtering options": "读取过滤选项",
    "Minimum mapping quality per read": "每个读取的最小映射质量",
    "Filter split alignments": "过滤分离的比对",
    "Only count primary alignments": "仅计数主要比对",
    "Ignore reads marked as duplicate": "忽略标记为重复的读取",
    "Allow reads to map to multiple features": "允许读取映射到多个特征",
    "Assign fractions to multi-mapping reads": "将分数分配给多映射的读取",
    "Assign fractions to multi-overlapping features": "为多重重叠的特征分配分数",
    "Assign fractions to both multi-mapping reads and multi-overlapping features": "将分数分配给多映射的读取和多重叠的特征",
    "Exon-exon junctions": "外显子-外显子连接处",
    "Reference sequence file": "参考序列文件",
    "Long reads": "长读长",
    "Count reads by read group": "计数读取，按读取组",
    "Largest overlap": "最大重叠",
    "Minimum bases of overlap": "最小重叠碱基数",
    "Minimum fraction (of read) overlapping a feature": "最小重叠特征的读段分数",
    "Minimum fraction (of feature) overlapping a read": "最小特征（片段）覆盖读段的比例",
    "Read 5' extension": "读取5'端延伸",
    "Read 3' extension": "读取3'端延伸",
    "Reduce read to single position": "将读取减少到单个位置",
    "Annotates the alignment file with 'XS:Z:'-tags to described per read or read-pair the corresponding assigned feature(s).": "使用 'XS:Z:' 标签注释比对文件，以描述每个读取或读取对所分配的特征。",
    "Receptor and Chain": "受体和链",
    "Quality Theshold": "质量阈值",
    "Report clonotypes with partial CDR3 mapping": "报告具有部分CDR3映射的克隆型",
    "use all alleles during alignemt (default = only major (*01) alleles)": "在比对时使用所有等位基因（默认 = 仅主要（*01）等位基因）",
    "Report clonotypes with no CDR3": "报告没有CDR3的克隆型",
    "Report clonotypes that have non-canonical CDR3": "报告具有非典型CDR3的克隆型。",
    "Report non-coding clonotypes (stop codon or frameshift)": "报告非编码克隆型（终止密码子或移码突变）",
    "Report mapping details for each read": "为每个读段报告映射细节",
    "Which Seurat method should be run?": "应该运行哪种Seurat方法？",
    "Counts file": "计数文件",
    "Minimum cells": "最小细胞数",
    "Minimum genes": "最小基因数",
    "Continue workflow after Normalization step?": "在标准化步骤后继续工作流程？",
    "Continue workflow after scaling step?": "继续在缩放步骤后的流程？",
    "Continue workflow after PCA step?": "在PCA步骤后继续工作流程？",
    "Continue workflow after TSNE and UMAP step?": "在TSNE和UMAP步骤后继续工作流程？",
    "RNA counts file": "RNA计数文件",
    "Protein counts file": "蛋白质计数文件",
    "Show code alongside outputs?": "显示代码及其输出？",
    "Include warnings in the output file (Yes) or pipe to stdout (No)": "在输出文件中包含警告信息（是）或将警告信息输出到标准输出（否）",
    "Display variable values used in code at the beginning of output file?": "在输出文件的开头显示代码中使用的变量值？",
    "Select input data": "选择输入数据",
    "Treatment sample file": "处理样本文件",
    "Control sample file": "对照样本文件",
    "Select file for gene annotations": "选择基因注释的文件",
    "Select file for transposable element annotations": "选择转座元件注释的文件",
    "Select library type": "选择库类型",
    "Are input files sorted by chromosome position?": "输入文件是否按染色体位置排序？",
    "Set TE counting mode": "设置转座元件计数模式",
    "Set read count cutoff": "设置读取计数截止值",
    "Set average length of fragment used for single-end sequencing": "设置用于单端测序的片段平均长度",
    "Set maximum number of iterations used to optimize multi-reads assignment": "设置用于优化多读分配的最大迭代次数",
    "Set FDR cutoff for significance": "设置FDR阈值以确定显著性",
    "Set fold-change ratio (absolute) cutoff for differential expression": "设置差异表达的绝对倍数变化比率截止值",
    "Select file with transcripts": "选择带有转录本的文件",
    "Select genetic code": "选择遗传密码",
    "LongOrfs options": "LongOrfs 选项",
    "Select gene-to-transcript identifier mapping file": "选择基因到转录本标识符映射文件",
    "Set minimum protein length": "设置最小蛋白质长度",
    "Activate strand-specificity?": "激活链特异性？",
    "Predict options": "预测选项",
    "Should likely coding regions be predicted?": "应该预测可能的编码区域吗？",
    "Select mode to retain long ORFs": "选择模式以保留长ORFs",
    "Set long ORFs length": "设置长开放阅读框的长度",
    "Select BlastP result file": "选择BlastP结果文件",
    "Select Pfam result file": "选择 Pfam 结果文件",
    "Retain only the single best ORF per transcript?": "只保留每个转录本中最佳的开放阅读框（ORF）？",
    "Start refinement that identifies potential start codons for 5' partial ORFs using a PWM?": "开始使用位置权重矩阵（PWM）进行细化，以识别5'部分开放阅读框（ORFs）的潜在起始密码子。",
    "Set top longest ORFs to train Markov Model": "设置最长的开放阅读框（ORF）用于训练马尔可夫模型",
    "Length of the UMI if present at the 5' end of your reads": "如果UMI（唯一分子标识符）存在于您读取序列的5'端，其长度为",
    "Input BED file of alignments": "输入对齐的BED文件",
    "Do you have an annotation in GTF format?": "您有GTF格式的注释文件吗？",
    "Annotations in GTF format": "GTF格式的注释",
    "Overlap fraction for merging mapped read positions to segments": "合并映射的读取位置到段的重叠分数",
    "Minimum length of the alignments as a fraction of longest": "对齐的最小长度作为最长长度的分数",
    "Overlap fraction for merging alignments/blocks": "合并对齐/区块的重叠分数",
    "Select the mode of merging": "选择合并模式",
    "Minimum locus size": "最小位点大小",
    "Blockbuster parameter distance": "大片参数距离",
    "Blockbuster parameter minClusterHeight": "Blockbuster 参数 minClusterHeight",
    "Blockbuster parameter minBlockHeight": "Blockbuster 参数 minBlockHeight",
    "Blockbuster parameter scale": "Blockbuster 参数 scale",
    "Did you use single or split reference for alignment?": "你使用的是单一参考还是分体参考进行比对？",
    "Second reference FASTA file": "第二个参考FASTA文件",
    "Input FASTA file": "输入FASTA文件",
    "Single or split reference?": "单个还是分割的参考？",
    "Aligner to use": "使用的比对工具",
    "Map reads to": "将读取映射到",
    "Seed length 1": "种子长度为1",
    "Seed length 2": "种子长度为2",
    "Minimum alignmnet score for 1st mapping iteration": "第一次映射迭代的最小比对得分",
    "Minimum alignmnet score for 2nd mapping iteration": "第二次映射迭代的最小比对得分",
    "Matching score for 1st mapping iteration": "第一次映射迭代的匹配得分",
    "Mismatch penalty for 1st mapping iteration": "第一次映射迭代的不匹配惩罚",
    "Matching score for 2nd mapping iteration": "第二次映射迭代的匹配得分",
    "Mismatch penalty for 2nd mapping iteration": "第二次映射迭代的不匹配惩罚",
    "Gap opening penalty for 1st mapping iteration": "第一次映射迭代的间隙开放惩罚",
    "Gap extension penalty for 1st mapping iteration": "第一次映射迭代的间隙延伸惩罚",
    "Gap opening penalty for 2nd mapping iteration": "第二次映射迭代的间隙开放惩罚",
    "Gap extension penalty for 2nd mapping iteration": "第二次映射迭代的间隙延伸惩罚",
    "Maximum number of allowed multi hits per read in 1st iteration": "在第一次迭代中，每个读取允许的最大多重命中数",
    "Maximum number of allowed multi hits per read in 2nd iteration": "在第二次迭代中，每个读取允许的最大多重命中数",
    "Minimum length for each fragment": "每个片段的最小长度",
    "Maximum number of bases allowed between the chimericsegments of a read": "一个读段中嵌合片段之间允许的最大碱基数量",
    "BED file of aligned segments": "对齐片段的BED文件",
    "Tabular file of merged alignments": "合并对齐的表格文件",
    "CRL locus fraction": "CRL位点分数",
    "EM threshold": "E-M 阈值",
    "Create CRLs": "创建证书吊销列表（CRLs）",
    "File containing CRLs information": "包含证书吊销列表信息的文件",
    "Have genomic information?": "有基因组信息吗？",
    "Genomic FASTA file": "基因组FASTA文件",
    "Select FASTA index": "选择FASTA索引",
    "TPM cut-off": "TPM 截断值",
    "Score cut-off": "得分截止",
    " Maximum number of bases allowed between the chimericsegments of a read": "允许在一个读段的嵌合体部分之间的最大碱基数量",
    "Hybridize chimeric loci?": "杂交嵌合位点？",
    "Enforce seed interaction?": "强制种子交互？",
    "Number of inter-molecular base pairs within the seed region": "种子区域内分子间碱基对的数量",
    "Minimal unpaired probability (per sequence) a seed region may have": "最小未配对概率（每序列）一个种子区域可能具有",
    "Compute accessibility profiles for interacting sequences?": "计算相互作用序列的可及性轮廓？",
    "Sliding window size for accessibility computation": "可访问性计算的滑动窗口大小",
    "IntaRNA temperature parameter in Celsius to setup the VRNA energy parameters": "IntaRNA 温度参数（摄氏度）用于设置 VRNA 能量参数",
    "Summarize interactions at loci level?": "总结位点水平的相互作用？",
    "Peptides": "肽段",
    "Genes to transcripts map": "基因到转录本的映射",
    "BLASTP: Peptides vs Uniprot.SwissProt": "BLASTP：肽段 vs Uniprot.SwissProt",
    "BLASTX: Transcripts vs Uniprot.SwissProt": "BLASTX：转录本 vs Uniprot.SwissProt",
    "HMMER hmmscan: Peptides vs PFAM": "HMMER hmmscan：肽段 vs PFAM",
    "TMHMM on Peptides": "TMHMM 在肽段上",
    "SignalP on Peptides": "在肽段上的SignalP",
    "Let Galaxy downloading the Trinotate Pre-generated Resource SQLite database": "让Galaxy下载Trinotate预生成的资源SQLite数据库",
    "Trinotate Pre-generated Resource SQLite database": "Trinotate 预生成资源 SQLite 数据库",
    "Report options": "报告选项",
    "Maximum E-value for reporting best blast hit and associated annotations.": "报告最佳BLAST命中及其相关注释的最大E值。",
    "PFAM cutoff": "PFAM 截止值",
    "Add the protein sequence data in the tab delimited report": "在制表符分隔的报告中添加蛋白质序列数据",
    "Add the transcript sequence data in the tab delimited report": "在制表符分隔的报告中添加转录本序列数据",
    "Keep the sqlite database file: Trinotate.sqlite": "保持 SQLite 数据库文件：Trinotate.sqlite",
    "alignment bam file": "比对的BAM文件",
    "chromosome length": "染色体长度",
    "coordinates of motif": "基序的坐标",
    "transcription factor": "转录因子",
    "cleavage/transposition bias": "剪切/转座偏差",
    "coordinates of ChIP-seq peaks": "ChIP-seq峰值坐标",
    "number of components": "组件数量",
    "background components": "背景组件",
    "fixed background component": "固定背景成分",
    "Include Read_Group/Attribute Datasets": "包含读取组/属性数据集",
    "Set Advanced Cuffnorm parameters?": "设置高级Cuffnorm参数？",
    "Select backend database (sqlite)": "选择后端数据库（sqlite）",
    "Plots": "绘图",
    "The width of the image": "图像的宽度",
    "The height of the image": "图像的高度",
    "Plot type": "绘图类型",
    "Add a smooth-fit regression line": "添加一条平滑拟合回归线",
    "First sample name for comparison": "比较的第一个样本名称",
    "Second sample name for comparison": "比较的第二个样本名称",
    "Cluster by": "聚类按照",
    "Display column labels?": "显示列标签？",
    "Display row labels?": "显示行标签？",
    "Draw border around plot?": "在图表周围绘制边框？",
    "Number of pre-defined clusters to attempt to find.": "尝试找到的预定义聚类数量。",
    "Max iterations": "最大迭代次数",
    "Use Count?": "使用计数？",
    "Gene ID": "基因ID",
    "Draw a 'summary' line with mean FPKM values for each condition?": "绘制一条包含每个条件的平均FPKM值的“摘要”线。",
    "Draw error bars?": "绘制误差线？",
    "Aligned SAM/BAM File": "对齐的SAM/BAM文件",
    "GFF File": "GFF 文件",
    "Stranded": "有方向的",
    "Minimum alignment quality": "最小比对质量",
    "ID Attribute": "ID 属性",
    "How to count nonunique or ambiguous mapping reads": "如何计算非唯一或模糊映射的读取",
    "Ignore seconday alignments (0x100 flag)": "忽略次要比对（0x100标志）",
    "Ignore supplementary alignments (0x800 flag)": "忽略补充对齐（0x800标志）",
    "Additional BAM Output": "额外的BAM输出",
    "Select salmon quantification mode:": "选择Salmon定量模式：",
    "Count file(s) collection": "计数文件（s）集合",
    "Select groups that correspond to this factor level": "选择与该因子水平相对应的组",
    "(Optional) provide a tabular file with additional batch factors to include in the model.": "（可选）提供一个表格文件，包含要包含在模型中的额外批次因子。",
    "GTF/GFF3 annotation file": "GTF/GFF3 注释文件",
    "Tabular file with Transcript-ID to Gene-ID mapping": "带有转录本ID到基因ID映射的表格文件",
    "Method for estimateSizeFactors": "估计大小因子的方法",
    "File with custom size factors": "自定义大小因子的文件",
    "Fit type": "拟合类型",
    "Turn off outliers replacement (only affects with >6 replicates)": "关闭异常值替换（仅影响具有>6个重复的情况）",
    "Turn off outliers filtering (only affects with >2 replicates)": "关闭异常值过滤（仅影响具有>2个重复的数据）",
    "Turn off independent filtering": "关闭独立过滤",
    "Perform pre-filtering": "执行预过滤",
    "Pre-filter value": "预过滤值",
    "Use beta priors": "使用贝塔先验",
    "Output selector": "输出选择器",
    "Alpha value for MA-plot": "MA图的 Alpha 值",
    "Input short mapped reads": "输入短映射读取",
    "Input long mapped reads": "输入长的映射读取",
    "Window around possibly erroneous splice sites": "可能错误的剪接位点周围的窗口",
    "Use a reference file to guide assembly?": "使用参考文件指导组装？",
    "Reference file": "参考文件",
    "Use a built-in GTF": "使用内置的GTF",
    "GTF/GFF3 dataset to guide assembly": "GTF/GFF3 数据集以指导组装",
    "Use Reference transcripts only?": "使用参考转录本吗？",
    "Output files for differential expression?": "差异表达的输出文件？",
    "Specify the average read length": "指定平均读取长度",
    "Cluster overlapping genes": "聚类重叠基因",
    "Prefix used for transcripts": "转录本使用的前缀",
    "Prefix for clustering": "聚类前缀",
    "Output coverage file?": "输出覆盖度文件？",
    "Output gene abundance estimation file?": "输出基因丰度估计文件？",
    "Do not assemble any transcripts on these reference sequence(s)": "不要在这些参考序列上组装任何转录本",
    "Name prefix for output transcripts": "输出转录本的名称前缀",
    "Minimum assembled transcript length": "最小组装转录本长度",
    "Minimum anchor length for junctions": "连接点的最小锚定长度",
    "Minimum junction coverage": "最小连接覆盖度",
    "Minimum bundle reads per bp coverage to consider for assembly": "每个碱基覆盖度考虑用于组装的最小捆绑读取数",
    "Gap between read mappings triggering a new bundle": "触发新捆绑的读映射间隔",
    "Fraction of bundle allowed to be covered by multi-hit reads": "允许由多次命中的读取覆盖的捆绑部分的比例",
    "Disable trimming of predicted transcripts based on coverage": "禁用基于覆盖度对预测转录本进行修剪",
    "Disable multi-mapping correction": "禁用多映射校正",
    "Input point-features dataset": "输入点特征数据集",
    "Reference annotation to include in the merging": "要合并的参考注释",
    "Minimum input transcript length to include in the merge": "合并中包含的最小输入转录本长度",
    "Minimum input transcript coverage to include in the merge": "最小输入转录本覆盖度以包含在合并中",
    "Minimum input transcript FPKM to include in the merge": "最小输入转录本FPKM以包含在合并中",
    "Minimum input transcript TPM to include in the merge": "合并时包含的最小输入转录本TPM值",
    "Gap between transcripts to merge together": "合并转录本之间的间隔",
    "Keep merged transcripts with retained introns": "保留带有保留内含子的合并转录本",
    "Paired or Single-end data?": "配对端或单端数据？",
    "Left/Forward strand reads": "正/前链读取",
    "Right/Reverse strand reads": "正向/反向链读取",
    "Strand specific data": "特定链数据",
    "Strand-specific Library Type": "特定链的库类型",
    "Single-end reads": "单端测序读取",
    "Abundance estimation method": "丰度估计方法",
    "Trinity assembly?": "Trinity 组装？",
    "Gene to transcript correspondence ('gene(tab)transcript' lines)": "基因到转录本对应关系（'gene(tab)transcript' 行）",
    "Are you pooling sequence datasets?": "您是否正在汇总序列数据集？",
    "FASTA/FASTQ dataset collection with R1/R2 pair": "FASTA/FASTQ 数据集集合，包含 R1/R2 对",
    "Run in silico normalization of reads": "运行在硅芯片上的读取归一化处理",
    "Minimum Contig Length": "最小连通体长度",
    "Use the genome guided mode?": "使用基因组引导模式？",
    "Coordinate-sorted BAM file": "坐标排序的BAM文件",
    "Maximum allowed intron length (also maximum fragment span on genome)": "最大允许的内含子长度（也是基因组上最大片段跨度）",
    "Minimum read coverage for identifying an expressed region of the genome": "识别基因组表达区域的最小读取覆盖度",
    "Minimum number of reads per partition": "每个分区的最小读取数",
    "Error-corrected or circular consensus (CCS) pac bio reads": "纠错或循环一致性（CCS）PacBio读取",
    "Minimum count for K-mers to be assembled": "K-mer组装的最小计数",
    "Use custom name for the input sequence files?": "使用自定义名称为输入序列文件？",
    "Label for the input sequences": "输入序列的标签",
    "Minimum read size": "最小读取大小",
    "Maximum read size": "最大读取大小",
    "Maximal genome mismatches": "最大基因组不匹配数",
    "Transposable Elements": "转座元件",
    "maximal TE mismatches": "最大转座元件不匹配",
    "miRNAs": "微小RNA",
    "snRNAs": "小核仁RNA",
    "rRNAs": "核糖体RNA",
    "tRNAs": "转运RNA",
    "siRNA": "小干扰RNA",
    "Lower bound for siRNA range": "siRNA范围的下限",
    "Higher bound of siRNA range": "siRNA范围的上限",
    "piRNA": "piRNA（Piwi-interacting RNA）是一种小RNA分子，与Piwi蛋白家族相互作用，主要参与基因沉默和转座元件的调控。",
    "lower bound of piRNA range": "piRNA 范围的下限",
    "higher bound of piRNA range": "piRNA 范围的上限",
    "PPPartners": "PPPartners",
    "Select an action": "选择一个操作",
    "Positive sequences FASTA file": "正序列 FASTA 文件",
    "Negative sequences FASTA file": "负序列 FASTA 文件",
    "Train a structure model": "训练一个结构模型",
    "Hyperparameter optimization settings": "超参数优化设置",
    "Select strategy": "选择策略",
    "Number of sequences for hyperparameter optimization": "用于超参数优化的序列数量",
    "Training options": "训练选项",
    "Disable 10-fold cross validation": "禁用10折交叉验证",
    "Disable motif generation": "禁用基序生成",
    "Minimum number of training sites demanded": "所需的最小训练位点数",
    "GraphProt model file": "GraphProt 模型文件",
    "Model parameter file": "模型参数文件",
    "Genomic BED file with coordinates from input sequences": "基因组BED文件，包含输入序列的坐标",
    "Prediction options": "预测选项",
    "Set GraphProt average profile peak score threshold for reporting peak regions": "设置 GraphProt 平均轮廓峰值得分的阈值，以报告峰值区域",
    "Smoothing parameter for calculating the average profile": "用于计算平均轮廓的平滑参数",
    "Maximum distance between two peak regions for merging": "两个峰值区域合并的最大距离",
    "Predict whole site instead of profile scores": "预测整个位点，而不是特征分数",
    "Output high-confidence (p50) peak regions": "输出高置信度（p50）的峰值区域",
    "File containing Heinz scores": "包含Heinz分数的文件",
    "Edge file": "边文件",
    "A node file with p-values": "一个包含 p 值的节点文件",
    "FDR value": "FDR值",
    "Choose your input type for BUM parameters": "选择您的BUM参数输入类型",
    "lambda": "拉姆达",
    "alpha": "α",
    "Output file of BUM model as input: lambda on the first line and alpha, the second": "BUM模型的输出文件作为输入：第一行是lambda，第二行是alpha",
    "Heinz output file": "海因茨输出文件",
    "Slamdunk BAM files": "Slamdunk BAM 文件",
    "Slamdunk Count TSV files": "Slamdunk 计数 TSV 文件",
    "Slamdunk VCF files": "Slamdunk VCF文件",
    "Read length": "读取长度",
    "Output T>C separated BAM files?": "输出 T>C 分离的 BAM 文件？",
    "FASTQ files": "FASTQ 文件",
    "Multimapper recovery": "多映射恢复",
    "Use separate 3' UTR bed to filter multimappers.": "使用独立的3'UTR bed文件来过滤多映射的reads。",
    "Bed to filter": "BED文件过滤",
    "Maximum number of alignments to report per read": "每个读取报告的最大比对数",
    "Use reference bed file to resolve multimappers.": "使用参考bed文件来解决多映射问题。",
    "Quantseq": "QuantSeq",
    "Number of bp to remove from the 5' end of all reads": "从所有读取的5'端移除的碱基对数",
    "Maximum number of As at the 3' end of a read": "在读取的3'端最多有A的数量",
    "Advanced settings.": "高级设置。",
    "Enable end-to-end alignments for mapping.": "启用端到端的比对。",
    "Minimum alignment identity": "最小比对一致性",
    "Minimum coverage to call variant": "最小覆盖度以调用变异",
    "Minimum variant fraction to call variant": "最小变体比例以调用变体",
    "Skip SNP step and provide custom variant file.": "跳过SNP步骤并提供自定义变异文件。",
    "T>C conversion threshold": "T>C 转换阈值",
    "Counting mode": "计数模式",
    "Genome annotation data": "基因组注释数据",
    "Keep the per-gene counts data?": "保留每个基因的计数数据？",
    "Name to use for the counts column": "用于计数列的名称",
    "Read selection for counting": "计数的读取选择",
    "Strandedness": "链特异性",
    "Multimapping reads": "多映射的读取",
    "Select the source of the quantification file": "选择定量文件的来源",
    "Name of the txID columns": "txID列的名称",
    "Name of the abundance column": "丰度列的名称",
    "Name of the counts column": "计数列的名称",
    "Name of the length column": "长度列的名称",
    "Is the gene id part of the counts file or will be obtained from an external file?": "基因ID是计数文件的一部分，还是将从外部文件获取？",
    "Name of the column containing the geneID": "基因ID所在列的名称",
    "Select a tx-to-gene table/GFF from your history or use a built-in file?": "选择一个tx-to-gene表格/GFF文件，可以从您的历史记录中选择，或者使用内置文件？",
    "Select an annotation version": "选择一个注释版本",
    "Will you provide a tx2gene or a GFF/GTF file?": "您将提供tx2gene文件还是GFF/GTF文件？",
    "Select your GFF file": "选择您的GFF文件",
    "Select your TranscriptID to GeneID table file": "选择您的转录本ID到基因ID的表格文件",
    "Summarization using the abundance (TPM) values?": "使用丰度（TPM）值进行总结？",
    "Choose data source": "选择数据来源",
    "Experiment matrix": "实验矩阵",
    "Gene expression matrix": "基因表达矩阵",
    "Column number containing time values": "包含时间值的列号",
    "Column number containing replicate coding": "包含重复编码的列号",
    "Output generated maSigPro input files?": "输出生成 maSigPro 输入文件？",
    "Time values": "时间值",
    "Specify a numerical time value": "指定一个数值时间值",
    "Counts file(s) at this time value": "在此时间值处计数文件",
    "Experimental groups": "实验组",
    "Specify the name of this experimental group": "指定这个实验组的名称",
    "Counts file(s) belonging to this experimental group": "计数文件（s）属于这个实验组",
    "Replicates": "重复样本",
    "Counts files that are replicates": "统计副本文件的数量",
    "Step 1: make.Design.Matrix - Defining the regression model": "步骤1：make.Design.Matrix - 定义回归模型",
    "Degree of regression fit polynome": "回归拟合多项式的度数",
    "Step 2: p.vector - Finding significant genes": "步骤2：p.vector - 寻找显著基因",
    "Q": "Q",
    "Minimum values": "最小值",
    "Step 3: T.fit - Finding significant differences": "第三步：T.fit - 寻找显著差异",
    "Step regression": "步骤回归",
    "nvar correction": "nvar校正",
    "alfa": "ALFA",
    "Step 4: get.siggenes - Obtaining lists of significant genes": "第四步：get.siggenes - 获取显著基因列表",
    "rsq": "rsq",
    "Variables": "变量",
    "Significant intercept": "显著截距",
    "Generate visualization PDF": "生成可视化PDF",
    "Step 5: see.genes - Visualization": "步骤5：see.genes - 可视化",
    "Cluster Data": "聚类数据",
    "Number of clusters for data partioning": "数据分区的聚类数量",
    "Add cluster information to summary file?": "将聚类信息添加到摘要文件？",
    "Cluster Method": "聚类方法",
    "Distance measure": "距离度量",
    "Agglomeration method": "聚合方法",
    "Color Mode": "颜色模式",
    "Show regression fit curves?": "显示回归拟合曲线？",
    "Draw lines?": "画线？",
    "Magnification for x labels": "X轴标签的放大倍数",
    "Add legend to plotting profiles?": "在绘图配置中添加图例？",
    "Trinity assembly": "Trinity 组装",
    "Abundance estimates": "丰度估计",
    "Cross sample normalization": "跨样本标准化",
    "Expression matrix": "表达矩阵",
    "Minimum expression level required across any sample": "在任何样本中所需的最小表达水平",
    "Isoform filtering method": "异构体过滤方法",
    "Minimum percent of dominant isoform expression": "主要异构体表达的最小百分比",
    "Sample description": "样本描述",
    "Differential analysis method": "差异分析方法",
    "edgeR dispersion value": "edgeR离散度值",
    "Number of bootstraps and permutation resampling": "引导重复次数和排列重采样",
    "Largest top genes size": "最大顶部基因大小",
    "Minimum count per replicate": "每个重复的最小计数",
    "Minimum number of replicates meeting minimum count per replicate": "每个重复中满足最小计数的最小重复数",
    "Name of a sample to which all other samples should be compared": "所有其他样本都应该与之比较的样本名称",
    "Pairs of sample comparisons to perform": "样本比较的配对",
    "Query sRNA in FASTA": "在FASTA中查询sRNA",
    "NCBI-nt database": "NCBI-nt 数据库",
    "Search Parameters": "搜索参数",
    "Choose taxon": "选择分类群",
    "Parameter Setup": "参数设置",
    "Maximum allowed E-Value": "最大允许的E值",
    "Minimum allowed identity [%]": "最小允许的一致性[%]",
    "Structure-based clustering": "基于结构的聚类",
    "Structure-based Clustering": "基于结构的聚类",
    "Structure-based filtering": "基于结构的筛选",
    "manual value for filtering": "手动过滤值",
    "Additional Settings": "附加设置",
    "Include upstream region": "包含上游区域",
    "Query Parameters": "查询参数",
    "Maximal number of unpaired bases between neighbored interacting bases": "最大未配对碱基数量（在相邻相互作用的碱基之间）",
    "Query regions to be considered for interaction prediction": "待考虑用于互作预测的查询区域",
    "Target Parameters": "目标参数",
    "Target regions to be considered for interaction prediction": "目标区域用于交互预测",
    "Seed Parameters": "种子参数",
    "Max. number of unpaired bases within the query's seed region": "查询种子区域中未配对碱基的最大数量",
    "Max. number of unpaired bases within the target's seed region": "目标种子区域中未配对碱基的最大数量",
    "Max. energy a seed region may have": "种子区域可能具有的最大能量",
    "Min. unpaired probability (per sequence)": "最小未配对概率（每序列）",
    "Interval(s) in the query to search for seeds": "在查询中搜索种子的区间",
    "Interval(s) in the target to search for seeds": "在目标中搜索种子的区间",
    "SHAPE Input Parameters": "SHAPE 输入参数",
    "SHAPE reactivity data to guide the query's accessibility computation": "SHAPE反应性数据用于指导查询的可接近性计算",
    "Prediction Parameters": "预测参数",
    "Prediction target": "预测目标",
    "Energy parameter": "能量参数",
    "Temperature [Celsius]": "温度 [摄氏度]",
    "Window-based computation: window width": "基于窗口的计算：窗口宽度",
    "Window-based computation: window overlap": "基于窗口的计算：窗口重叠",
    "Max. absolute energy of an interaction": "最大绝对能量交互",
    "Max. delta energy above mfe of an interaction": "最大能量差高于相互作用的最小自由能",
    "Interaction Parameters": "交互参数",
    "Average fragment length": "平均片段长度",
    "Estimated standard deviation of fragment length": "片段长度的估计标准差",
    "Collection or individual datasets": "集合或单独数据集",
    "Pseudoalignment uses UMIs": "伪比对使用UMIs（唯一分子标识符）",
    "Collection of reads": "读取集合",
    "Perform sequence based bias correction": "执行基于序列的偏差校正",
    "Number of bootstrap samples": "引导样本数",
    "Search for fusions": "搜索融合基因",
    "Single overhang": "单端突出",
    "Output pseudoalignments in BAM format": "输出伪比对结果为BAM格式",
    "Project pseudoalignments to genome": "将伪比对结果投影到基因组上",
    "Chromosome names and lengths": "染色体名称和长度",
    "Seed for the bootstrap sampling": "引导抽样的种子",
    "Count Files or Matrix?": "计数文件或矩阵？",
    "Factor": "因子",
    "Counts files": "计数文件",
    "Count Matrix": "计数矩阵",
    "Input factor information from file?": "从文件中输入因子信息？",
    "Factor File": "因子文件",
    "Factor Name": "因子名称",
    "Use Gene Annotations?": "使用基因注释？",
    "Gene Annotations": "基因注释",
    "Formula for linear model": "线性模型的公式",
    "Input contrasts manually or through a file": "手动输入对比组或通过文件输入",
    "Contrast": "对比",
    "Contrast of Interest": "兴趣对比",
    "Contrasts File": "对比文件",
    "Filter Low Counts": "过滤低计数",
    "Filter lowly expressed genes?": "过滤低表达基因？",
    "Filter on CPM or Count values?": "根据CPM或计数值进行过滤？",
    "Minimum CPM": "最小 CPM",
    "Minimum Samples": "最小样本数",
    "Minimum Count": "最小计数",
    "Filter on Total Count or per Sample Count values?": "根据总数计数还是每个样本的计数值进行过滤？",
    "Output Normalised Counts Table?": "输出标准化计数表？",
    "Minimum Log2 Fold Change": "最小对数2倍变化量",
    "P-Value Adjusted Threshold": "P值调整阈值",
    "P-Value Adjustment Method": "P值调整方法",
    "Normalisation Method": "标准化方法",
    "Use Robust Settings?": "使用稳健设置？",
    "Use Likelihood Ratio Test?": "使用似然比检验？",
    "Input Matrix": "输入矩阵",
    "Complete Barcodes": "完成条形码",
    "Plate Protocol": "板协议",
    "Barcode Format": "条形码格式",
    "Barcode Range: Start": "条形码范围：起始",
    "Barcode Range: End": "条形码范围：结束",
    "Batches utilizing this Range": "批次使用此范围",
    "Plate Format": "板式格式",
    "Plate Number": "板号",
    "Batches within this Plate Number": "这个板号内的批次",
    "RegEx to extract Plate, Batch, and Barcodes from headers": "正则表达式提取标题中的板、批次和条形码",
    "RegEx to replace Plate, Batch, and Barcodes from headers": "正则表达式替换头文件中的板、批次和条形码",
    "Sort Cells by Library Size": "按文库大小排序单元格",
    "Differential Expression Method": "差异表达方法",
    "Apply voom with sample quality weights?": "应用带有样本质量权重的voom方法？",
    "Prior count": "先验计数",
    "Input Contrast information from file?": "从文件中输入对比信息？",
    "Additional Plots": "附加图表",
    "Output Filtered Counts Table?": "输出过滤计数表？",
    "Output Library information file?": "输出库信息文件？",
    "Test significance relative to a fold-change threshold (TREAT)": "相对于折叠变化阈值测试显著性（TREAT）",
    "Number of genes to highlight in Volcano plot, Heatmap and Stripcharts": "在火山图、热图和条形图中要突出显示的基因数量",
    "GTF file created from DEXSeq-Count tool": "由DEXSeq-Count工具创建的GTF文件",
    "Specify a factor name": "指定一个因子名称",
    "Specify a factor level": "指定一个因子水平",
    "Count file for factor level 1": "计数文件用于因子水平1",
    "Count files for factor level 2": "计算因子水平2的文件数量",
    "Visualise the analysis results?": "可视化分析结果？",
    "Output rds file for plotDEXSeq?": "输出用于plotDEXSeq的RDS文件？",
    "All the genes under this FDR threshold will be shown in the html report": "所有在这个FDR阈值下的基因都会在HTML报告中显示",
    "Mode of operation": "操作模式",
    "Aggregate genes with exons?": "聚合具有外显子的基因？",
    "Input bam file": "输入BAM文件",
    "DEXSeq compatible GTF file": "DEXSeq兼容的GTF文件",
    "Is library paired end?": "是否为配对末端的文库？",
    "Is library strand specific?": "是否为库特异链特异性？",
    "Skip all reads with alignment quality lower than the given minimum value": "跳过所有比对质量低于给定最小值的读取",
    "Sorting order of alignments": "比对排序顺序",
    "DEXSeqResults object": "DEXSeq 结果对象",
    "Specify the primary factor name in the DEXSeqResults object": "在DEXSeqResults对象中指定主要因子名称",
    "Genes to plot": "要绘制的基因",
    "Gene identifier": "基因标识符",
    "List of gene IDs": "基因ID列表",
    "Display transcripts": "显示转录本",
    "Display transcript names": "显示转录本名称",
    "Plot count values from the individual samples": "绘制来自各个样本的计数值",
    "Plot exon usage": "绘制外显子使用情况",
    "Differential expression results": "差异表达结果",
    "p-value cutoff for FDR": "FDR的p值截止值",
    "min abs(log2(a/b)) fold change": "最小绝对值(log2(a/b))倍数变化",
    "Maximum differential expression genes per comparison": "每组比较中最大差异表达基因数",
    "Order columns by samples file": "按样本文件排序列",
    "Maximum genes in cluster": "聚类中最大基因数",
    "Run GO enrichment analysis": "运行GO富集分析",
    "Differential expression count matrices": "差异表达计数矩阵",
    "Extracted GO assignments file": "提取的GO分配文件",
    "Gene length file": "基因长度文件",
    "temperature [°C]": "温度 [摄氏度]",
    "Thermodynamic energy parameters": "热力学能量参数",
    "how to treat dangling end energies": "如何处理悬空末端能量",
    "Calculate structures subject to constraints": "计算受约束的结构",
    "No conversion": "无需转换",
    "No lonely pairs": "无孤对电子",
    "No GU pairing": "无GU配对",
    "No GU pairing at the ends": "两端不配对GU",
    "No stabilization for loops, hairpins etc.": "没有对循环、发夹等进行稳定化处理。",
    "Assume circular RNA structure": "假设循环RNA结构",
    "Don't generate PS files": "不生成PS文件",
    "Only consider hits with RNAz class probality P>cutoff": "只考虑RNAz类概率P>阈值的命中",
    "Print information for window or loci": "打印窗口或位点的信息",
    "Print a header explaining the fields of the output": "打印一个标题，解释输出字段",
    "Discordant alignment file of STAR)": "不一致的STAR比对文件",
    "Genomic region 1": "基因组区域1",
    "Genomic region 2": "基因组区域 2",
    "Output of Dr. Disco Detect": "Dr. Disco Detect的输出",
    "Blacklist Regions": "黑名单区域",
    "Blacklist Junctions": "黑名单连接点",
    "Minimum alignment length on each side of the junction.": "每个连接点两侧的最小比对长度。",
    "Discordant alignment file of STAR (processed with dr-disco fix)": "不一致的STAR比对文件（已用dr-disco修复处理）",
    "Minimal score to initiate pulling sub-graphs": "最小分数以启动拉取子图",
    "Discordant alignment file of STAR": "不匹配的STAR比对文件",
    "GTF file (Optional: for predicting frame shifts)": "GTF文件（可选：用于预测移码突变）",
    "Predict MEA structure": "预测MEA结构",
    "Dotplot": "点图",
    "Use default parameters": "使用默认参数",
    "Alpha value": "α值",
    "Beta value": "贝塔值",
    "Gamma value": "伽马值",
    "Delta value": "Δ值",
    "Structure": "结构",
    "Compare to a reference structure": "与参考结构进行比较",
    "Slide Rule": "滑尺规则",
    "Conflict Rule": "冲突规则",
    "Input model": "输入模型",
    "Input comparison": "输入比较",
    "Common parameters": "常见参数",
    "Window size, default 120": "窗口大小，默认为120",
    "Window step size, default 120": "窗口步长大小，默认为120",
    "Level of coarse graining": "粗粒化水平",
    "Input Alignment File": "输入对齐文件",
    "Number of sequences in the output alignment(s). (Default:6)": "输出对齐中序列的数量。（默认值：6）",
    "Number of output alignments (Default: 1)": "输出对齐数量（默认值：1）",
    "The resulting alignment(s) is (are) optimized for this value of mean pairwise identity (in percent, default: 80)": "生成的对齐（对齐结果）针对平均配对一致性（以百分比表示，默认值：80%）这个值进行了优化。",
    "Sequences from pairs with pairwise identity higher than X% are removed (default: 99, i.e. only almost identical sequences areremoved)": "序列对中相似度高于X%的序列被移除（默认：99，即只有几乎相同的序列被移除）",
    "By default the first sequence (=reference sequence) is always present in the output alignment(s). If you do not care having it removed, set this flag.": "默认情况下，第一个序列（=参考序列）始终包含在输出对齐中。如果你不介意将其删除，请设置此标志。",
    "Min size of block before slicing, default is window size 120": "切片前块的最小大小，默认为窗口大小120",
    "Maximum fraction of gaps, default 0.25": "最大间隙比例，默认为0.25",
    "Maximum fraction of masked letters in sequence, default 0.1": "序列中掩蔽字母的最大比例，默认为0.1",
    "Discard alignment windows with an overall mean pairwise identity smaller than X%. (Default: 50)": "丢弃整体平均配对比对相似度小于X%的比对窗口。（默认值：50）",
    "Minimum number of sequences in an alignment. Discard any windows with less than N sequences (Default:2)": "对齐中序列的最小数量。丢弃任何少于N个序列的窗口（默认值：2）",
    "Maximum number of sequences in an alignment. Discard any windows with less than N sequences (Default:6)": "在一个比对中的最大序列数。丢弃任何少于N个序列的窗口（默认值：6）",
    "Number of different subsets of sequences that is sampled if there are more sequences in the alignment than --max-seqs (Default: 1)": "如果比对中的序列数量超过 --max-seqs（默认值：1），则采样的不同序列子集的数量",
    "Minimum number of columns of an alignment slice. After removing sequences from the alignment, all-gap columns are removed. If the resulting alignment has fewer than N columns, the complete alignment is discarded.": "对齐切片的最小列数。在从对齐中移除序列后，所有间隙列将被移除。如果结果对齐的列数少于N，则整个对齐将被丢弃。",
    "If the number of sequences has to be reduced (see --max-seqs) a subset of sequences is chosen which is optimized for this value of mean pairwise identity. (In percent, default: 80)": "如果需要减少序列的数量（参见--max-seqs），则选择一个子集的序列，该子集针对平均配对同一性值进行优化。（以百分比表示，默认值：80）",
    "Scored strand": "得分链",
    "By default the first sequence is interpreted as reference sequence. This means, for example, that if the reference sequence is removed during filtering steps the complete alignment is discarded. Also, if there are too many sequences in the alignment, the reference sequence is never removed when choosing an appropriate subset. Having a reference sequence is crucial if you are doing screens of genomic regions. For some other applications it might not be necessary and in such cases you can change the default behaviour by setting this option.": "默认情况下，第一个序列被解释为参考序列。这意味着，例如，如果在过滤步骤中删除了参考序列，则整个比对被丢弃。此外，如果在比对中序列太多，在选择适当子集时，参考序列永远不会被删除。如果您正在筛选基因组区域，拥有参考序列至关重要。对于其他一些应用，可能没有必要，而在这种情况下，您可以通过设置此选项来更改默认行为。",
    "RNAz output to annotate": "RNAz输出用于注释",
    "Bed file with annotation": "带有注释的Bed文件",
    "Tabular output of DESeq2/edgeR/limma/DEXSeq": "DESeq2/edgeR/limma/DEXSeq的表格输出",
    "Input file type": "输入文件类型",
    "Reference annotation in GFF/GTF format": "参考注释在GFF/GTF格式中",
    "GFF feature type": "GFF 特征类型",
    "GFF feature identifier": "GFF 特征标识符",
    "GFF transcript identifier": "GFF 转录本标识符",
    "GFF attributes to include": "要包含的GFF属性",
    "Sequence input": "序列输入",
    "Free endgaps": "自由末端间隙",
    "Consistency transformation": "一致性转换",
    "Iterative refinement": "迭代细化",
    "Number of refinement iterations": "迭代次数",
    "Heuristic parameters for SPARSE": "SPARSE的启发式参数",
    "Standard output verbosity": "标准输出详细度",
    "Scoring parameters": "打分参数",
    "RNA folding parameters": "RNA 折叠参数",
    "Heuristic parameters": "启发式参数",
    "Constraint parameters": "约束参数",
    "Source for the genome": "基因组来源",
    "Use CCDS annotation (valid for Human Gencode 19 and Mouse Gencode M3)": "使用 CCDS 注释（适用于人类 Gencode 19 和小鼠 Gencode M3）",
    "Use Appris annotation (valid for Human Gencode 19 and Mouse Gencode M3)": "使用Appris注释（适用于人类Gencode 19和小鼠Gencode M3）",
    "Ribo-seq alignment file": "Ribo-seq 对齐文件",
    "Start_stops FAR": "起始终止位 FAR",
    "Annotation path": "注释路径",
    "RNA-seq alignment file": "RNA-seq比对文件",
    "Minimum read count": "最小读取数",
    "Known pairs": "已知配对",
    "Read count of sense strand": "正链读取计数",
    "Read count of antisense strand": "反义链的读取计数",
    "(Optional) Genome reference in FASTA format that corresponds to the reference genome or RNA database": "（可选）与参考基因组或RNA数据库相对应的FASTA格式的基因组参考序列",
    "Add 5'-offset to exon-type entry in GTF output file": "在GTF输出文件中，为外显子类型条目添加5'偏移量。",
    "Add 3'-offset to exon-type entry in GTF output file": "在GTF输出文件中，为外显子类型条目添加3'偏移量。",
    "(Optional) Custom parameters file": "（可选）自定义参数文件",
    "Input small RNA sequences file": "输入小RNA序列文件",
    "Target sequence database file": "目标序列数据库文件",
    "Prediction score cutoff value": "预测分数截断值",
    "Search reverse strand for targets?": "在反向链上搜索目标？",
    "Gene ID to search for": "要搜索的基因ID",
    "Select species": "选择物种",
    "Motif to search for": "要搜索的模式",
    "FusionGene Datasets": "融合基因数据集",
    "Dataset (RNA-Seq fusion gene detection experiment)": "数据集（RNA-Seq融合基因检测实验）",
    "Format of dataset": "数据集格式",
    "Corresponding gene-name annotation file (BED format)": "相应的基因名称注释文件（BED格式）",
    "Matching method: technique used to match fusion genes based on annotated gene sets": "匹配方法：基于注释基因集用于匹配融合基因的技术",
    "Consider fusion genes distinct when the breakpoints have different strands: (A<-,B<-) != (->A,B<-); default": "当融合基因的断点在不同链上时，视为不同的融合基因：(A<-,B<-) != (->A,B<-)；默认",
    "Consider fusion genes distinct when the donor and acceptor sites are swapped (A,B) != (B,A)": "当供体和受体位点互换时（A,B）不等于（B,A），考虑融合基因是不同的。",
    "Long gene size": "长基因大小",
    "Input sequence": "输入序列",
    "NCBI taxonomy id of input organism": "输入生物的NCBI分类学ID",
    "Only candidate sequences with a nSCI higher than this value are accepted.": "只有nSCI高于这个值的候选序列才会被接受。",
    "Evalue cutoff for cmsearch filtering": "Evalue 截止值用于 cmsearch 过滤",
    "Specify name of blast database to use.": "指定要使用的BLAST数据库的名称。",
    "Filter blast hits by coverage of at least 80%.": "过滤BLAST命中结果，要求覆盖度至少为80%。",
    "Only the best blast hit per taxonomic entry is considered.": "每个分类条目只考虑最佳BLAST命中。",
    "Toggles blast softmasking, meaning exclusion of low complexity (repetative) regions in lookup table.": "切换BLAST软掩蔽功能，意味着在查找表中排除低复杂度（重复）区域。",
    "Select mode of operation": "选择操作模式",
    "BAM file containing alignments": "包含比对结果的BAM文件",
    "Input blockgroups file": "输入块组文件",
    "My input files have no 'chr' for chromosome names": "我的输入文件中的染色体名称没有 'chr' 前缀",
    "Would you like to perform classification?": "您想要执行分类吗？",
    "Mode of classification": "分类模式",
    "Output of cmsearch tool": "cmsearch 工具的输出",
    "BED file containing clusters (output of BlockClust)": "包含簇的BED文件（BlockClust的输出）",
    "Pairwise similarities file": "配对相似性文件",
    "Samples description": "样本描述",
    "Samples": "样本",
    "Full sample name": "样本全名",
    "RData file": "R数据文件",
    "Method for partitioning genes into clusters": "基因聚类分组方法",
    "Cut tree based on this percent of max(height) of tree": "根据树的最大高度的百分比来剪枝",
    "Cut tree into K clusters": "将树切割成K个簇",
    "Define K clusters via k-means algorithm": "通过k-means算法定义K个聚类",
    "Column ordering in expression matrices": "表达矩阵中的列排序",
    "Arriba fusions.tsv": "Arriba融合.tsv",
    "STAR Aligned.out.bam": "STAR 对齐后的输出文件 Aligned.out.bam",
    "Visualization Options": "可视化选项",
    "Select reference": "选择参考",
    "Specify sample name": "指定样本名称",
    "Annotation in GTF format": "GTF格式的注释",
    "Separator between reference sequences' name and additional information to discard": "参考序列名称与要丢弃的额外信息之间的分隔符",
    "Positive integer value specifying the maximum number of indels allowed for each read.": "正整数值，指定每个读取允许的最大插入或缺失（indel）数量。",
    "Save p-site info RDATA file?": "保存p-site信息RDATA文件？",
    "Filtering Options": "过滤选项",
    "Perform duplicate filtering?": "执行重复过滤？",
    "Which reads should be considered duplicates?": "哪些读取应该被视为重复项？",
    "Which read to keep if duplicates disply different lengths?": "如果重复的读取显示不同的长度，应该保留哪个读取？",
    "Perform read length filtering": "执行读取长度过滤",
    "Only read lengths satisfying this threshold are kept": "只有满足这个阈值的读取长度才会保留",
    "Additional options for P-site offset computation": "P位点偏移计算的额外选项",
    "Min number of nucleotides that must flank the reference codon in both directions": "必须在参考密码子两侧的核苷酸的最小数量",
    "Use the translation initiation site as reference codon?": "使用翻译起始位点作为参考密码子？",
    "On which extrimity the correction step should be based on?": "校正步骤应该基于哪个肢体？",
    "Confidence level for generating occupancy metaprofiles for to a sub-range of read lengths": "置信水平，用于为读取长度的子范围生成占用元配置文件",
    "Options for codon and CDS coverage": "密码子和CDS覆盖度的选项",
    "Write number of P-Sites falling per region?": "写出每个区域中P位点的数量？",
    "Min number of overlapping positions between reads and codons to be considered": "最小重叠位置数，指的是在考虑读段和密码子之间重叠时的最小数量。",
    "Numer of nucleotides at the beginning of the coding sequences to be excluded": "编码序列开始处要排除的核苷酸数量",
    "Numer of nucleotides at the end of the coding sequences to be excluded": "编码序列末端要排除的核苷酸数量",
    "Only use exonic (UTR exons and CDS exons) reads, otherwise use all reads": "仅使用外显子（UTR外显子和CDS外显子）的读段，否则使用所有读段",
    "How should read-pairs that only have one end mapped be counted?": "应该如何统计只有一个端点映射的读对？",
    "Minimum number of fragments (default: 3)": "最小片段数（默认：3）",
    "Begin sampling from this percentile (default=5)": "从这个百分位数开始采样（默认=5）",
    "End sampling at this percentile (default=100)": "在此百分位数结束采样（默认=100）",
    "Sampling step size (default=5)": "采样步长（默认=5）",
    "Ignore transcripts with RPKM smaller than this number (default=0.01)": "忽略RPKM值小于这个数值的转录本（默认值=0.01）",
    "Chromosome size file (tab or space separated)": "染色体大小文件（制表符或空格分隔）",
    "Normalization": "标准化",
    "specified wigsum": "指定 wigsum",
    "Run each sample separately, or combine mutiple samples into one plot": "分别运行每个样本，或将多个样本合并为一个图表",
    "Input .bam file": "输入.bam文件",
    "Input .bam file(s)": "输入 .bam 文件（们）",
    "Minimum mRNA length (default: 100)": "最小mRNA长度（默认：100）",
    "Input bigwig file": "输入大wig文件",
    "Lower bound (bp, default=-250)": "下限（碱基对，默认=-250）",
    "Upper bound (bp, default=250)": "上限（碱基对，缺省值为250）",
    "Step size of histogram (bp, default=5)": "直方图的步长大小（碱基对，默认值为5）",
    "Minimum number of supporting reads to call a junction (default=1)": "支持一个连接点所需的最小读取数（默认=1）",
    "Sampling bounds and frequency": "采样界限和频率",
    "Lower Bound Sampling Frequency (bp, default=5)": "下限采样频率（碱基对，默认=5）",
    "Upper Bound Sampling Frequency (bp, default=100)": "上界采样频率（碱基对，默认=100）",
    "Sampling increment (default=5)": "采样增量（默认=5）",
    "Include N,X in NVC plot": "在NVC图中包含N、X",
    "Upper Limit of Plotted Duplicated Times (default=500)": "绘制重复次数的上限（默认=500）",
    "Read sequences in fasta or fastq format": "读取fasta或fastq格式的序列",
    "Reference genome seqeunce (fasta)": "参考基因组序列（fasta格式）",
    "Reference mRNA sequence (fasta)": "参考mRNA序列（fasta格式）",
    "Ignore Phred scores less than this amount (only applies to 'boxplot', default=1000)": "忽略小于这个数值的Phred分数（仅适用于'boxplot'，默认值为1000）",
    "Minimum coverage (default=10)": "最小覆盖度（默认=10）",
    "Sample size (default=100)": "样本大小（默认=100）",
    "Subtract background noise (default=No)": "减去背景噪声（默认=否）",
    "Genomic binding sites BED file": "基因组结合位点 BED 文件",
    "Genome reference FASTA file": "基因组参考FASTA文件",
    "Select structure prediction method": "选择结构预测方法",
    "Advanced preprocessing parameters": "高级预处理参数",
    "Skip the filtering step?": "跳过过滤步骤？",
    "Filtering: minimum score for binding site": "过滤：结合位点的最小分数",
    "Filtering: minimum binding site length": "过滤：最小结合位点长度",
    "Filtering: maximum binding site length": "过滤：最大结合位点长度",
    "Elongation: span for up- and downstream elongation of binding sites": "延伸：结合位点上游和下游延伸的范围",
    "Advanced training parameters": "高级训练参数",
    "Length of the motif that shall be found": "要找到的基序长度",
    "Initialize the model randomly?": "随机初始化模型？",
    "Greediness of Gibbs sampler": "吉布斯采样器的贪婪性",
    "Number of sequences to be held out in each iteration": "每次迭代中要保留的序列数量",
    "Termination threshold": "终止阈值",
    "Use only best structure for each sequence?": "仅对每个序列使用最佳结构？",
    "Output FASTA file and corresponding structures file?": "输出FASTA文件和相应的结构文件？",
    "Output logo files?": "输出 logo 文件？",
    "Output raw logo and model files?": "输出原始的Logo和模型文件？",
    "Select kallisto fusions file": "选择 kallisto 融合文件",
    "Output from kallisto quant --fusion": "kallisto quant --fusion 的输出",
    "Select Reference FASTA": "选择参考FASTA",
    "Reference transcriptome FASTA": "参考转录组FASTA",
    "Select a built-in FASTA": "选择一个内置的FASTA",
    "Select a history FASTA": "选择一个历史记录的FASTA文件",
    "Select Reference GTF": "选择参考基因组的GTF文件",
    "Reference GTF": "参考 GTF",
    "Maximum insert size of the paired-end library": "配对末端库的最大插入片段大小",
    "Number of mismatches allowed": "允许的不匹配数",
    "K-mer size used to create the kallisto index": "用于创建Kallisto索引的K-mer大小",
    "Ignore protein coding information in annotation": "忽略注释中的蛋白质编码信息",
    "Genome Version": "基因组版本",
    "Reference annotation in ENSEMBL GTF format": "参考注释在ENSEMBL GTF格式中",
    "Select analysis type": "选择分析类型",
    "Single sample analysis BED file": "单样本分析 BED 文件",
    "Run GO term enrichment analysis": "运行GO术语富集分析",
    "Run gene set enrichment analysis": "运行基因集富集分析",
    "Provide human Molecular Signatures Database (MSigDB) file": "提供人类分子特征数据库（MSigDB）文件",
    "Downsampling (N)": "下采样（N）",
    "Output raw data tables": "输出原始数据表格",
    "Multi sample analysis BED files": "多样本分析 BED 文件",
    "Output annotation summaries": "输出注释摘要",
    "Run motif analysis": "运行基序分析",
    "Multiple Alignment": "多重比对",
    "Break long alignment blocks": "打断长对齐块",
    "maxLength": "最大长度",
    "desiredLength": "期望长度",
    "process blocks separately": "独立处理区块",
    "Stop early": "提前停止",
    "Show only best non-overlapping hits": "仅显示最佳的不重叠的命中",
    "Show only best hit": "仅显示最佳命中",
    "Scoring parameters as comma separated string:'DELTA,OMEGA,omega,stop_penalty'": "评分参数以逗号分隔字符串：'DELTA,OMEGA,omega,stop_penalty'",
    "Create plots": "创建图表",
    "Create plots only for high scoring segments with p better than:": "只为得分高于阈值 p 的高得分片段创建图表：",
    "Which type of z-scores": "哪种类型的z分数",
    "Probability cutoff": "概率截止",
    "Use decision model for structural alignments": "使用结构比对的决策模型",
    "Never fall back to shuffling": "永远不要退回到洗牌",
    "Input: Fasta file": "输入：Fasta文件",
    "Calculation Mode": "计算模式",
    "Input from FASTA file": "来自FASTA文件的输入",
    "Sequence to fold (FASTA file)": "序列折叠（FASTA文件）",
    "Temperature [°C]": "温度 [摄氏度]",
    "How to treat dangling end energies": "如何处理悬挂端能量",
    "Layout algorithm": "布局算法",
    "Calculate Maximum Expected accuracy": "计算最大期望准确度",
    "MEA-Gamma Value": "MEA-Gamma值",
    "Calculate Partition Function": "计算配分函数",
    "Scaling factor": "缩放因子",
    "Do not produce postscript drawing of the mfe structure (reduces run-time)": "不生成mfe结构的postscript绘图（减少运行时间）",
    "Allow lonely base-pairs": "允许孤立的碱基对",
    "Allow GU pairing": "允许GU配对",
    "Allow GU pairing at the ends": "允许末端发生GU配对",
    "Allow stabilization for loops, hairpins etc.": "允许对环、发夹等进行稳定化处理。",
    "Convert Thymine to Uracil (T -> U)": "将胸腺嘧啶转换为尿嘧啶（T -> U）",
    "Take G Quadruplex formation into account": "考虑G四链体形成",
    "Model as circular RNA structure": "模拟成环状RNA结构",
    "Threshold for base pair probabilities": "碱基对概率的阈值",
    "Allow other pairs in addition to the usual AU,GC,and GU pairs.": "允许除了常规的AU、GC和GU配对外的其他配对。",
    "Scaling of Boltzman factors": "Boltzmann因子的缩放",
    "Structure constraints": "结构约束",
    "Set the maximum base pair span": "设置最大碱基对跨度",
    "Constraints": "约束条件",
    "Constraints file": "约束文件",
    "Use constraints for all alignment records": "使用约束条件对所有比对记录",
    "Remove non-canonical base pairs from he structure constraint": "从结构约束中移除非典型碱基对",
    "Enforce base pair given by round brackets () in structure constraint": "在结构约束中强制执行圆括号()中给定的碱基对",
    "Shape reactivity data": "形状反应性数据",
    "Shape file": "形状文件",
    "Slope m": "斜率 m",
    "shape reactivity data": "形态反应性数据",
    "Slope": "斜率",
    "Slope s": "斜率 s",
    "Sequence structure energy": "序列结构能量",
    "Naming Conventions": "命名约定",
    "Automatically generate an ID for each alignment.": "自动为每个比对生成一个ID。",
    "Prefix for automatically generated IDs (as used in output file names)": "自动生成的ID前缀（如在输出文件名中使用）",
    "The number of digits of the counter in automatically generated sequence IDs": "自动生成的序列ID中计数器的位数",
    "First number in automatically generated sequence IDs": "自动生成的序列ID中的第一个数字",
    "First set of peaks in GFF or BED format": "第一组峰值在GFF或BED格式中",
    "Second set of peaks in GFF or BED format": "第二个峰值集在GFF或BED格式中",
    "List of peaks to filter by": "峰值列表筛选",
    "Input BED format": "输入 BED 格式",
    "Column to use for ranking peaks": "用于排名峰值的列",
    "Output PNG plot of peaks": "输出峰值的PNG格式图表",
    "Use peaks without an overlapping match and set the value to 0": "使用没有重叠匹配的峰值，并将该值设置为0",
    "Method to use for merging peaks": "用于合并峰值的方法",
    "Initial value of mu": "μ的初始值",
    "Initial value of sigma": "σ的初始值",
    "Initial value of rho": "初始的rho值",
    "Initial value of the mixture parameters": "混合参数的初始值",
    "Fix mu to the starting point and do not let it vary": "将mu固定在起始点，并且不允许它变化",
    "Fix sigma to the starting point and do not let it vary": "将sigma固定在起始点，不允许其变化",
    "Allow signal points that are below the noise mean": "允许信号点低于噪声均值",
    "Set the IDR value for a group of multi summit peaks": "为一组多峰设置IDR值",
    "Allow negative values for scores": "允许分数为负值",
    "The random seed value": "随机种子值",
    "The maximum number of optimization iterations": "最大优化迭代次数",
    "The maximum change in parameter value changes for convergence": "参数值变化的最大变化量用于收敛",
    "Only return the merged peak list": "只返回合并后的峰值列表",
    "General Options": "通用选项",
    "Produce postscript output": "生成Postscript输出",
    "The number of digits of the counter in automatically generated alignment IDs": "自动生成的比对ID中计数器的位数",
    "First number in automatically generated alignment IDs": "自动生成的比对ID中的第一个数字",
    "Model Options": "模型选项",
    "Algorithm Options": "算法选项",
    "Calculate partition function": "计算配分函数",
    "Calculate homo-dimers as well as A and B monomers": "计算同源二聚体以及A和B单体",
    "Concentrations": "浓度",
    "A file with initial concentrations for the two sequences": "一个包含两个序列初始浓度的文件",
    "Concentrations file": "浓度文件",
    "G Quadruplex formation": "G 四链体形成",
    "Clustal/AlignmentFasta/Stockholm/MAF file": "Clustal/对齐的Fasta/斯德哥尔摩/MAF文件",
    "Gamma Value": "伽马值",
    "Most Informative Sequence": "最信息序列",
    "Number of random structures.": "随机结构的数量。",
    "Structure conservation index": "结构保守性指数",
    " Be verbose": "详细模式",
    "Colored secondary structure image": "彩色的二级结构图像",
    "Colored and annotated alignment image": "彩色并带有注释的比对图像",
    "Continuous alignment ID numbering": "连续的对齐ID编号",
    "Use consensus structures from Stockholm file (#=GF SS_cons) as constraint": "使用斯德哥尔摩文件中的共识结构（#=GF SS_cons）作为约束",
    "Shape File": "形状文件",
    "Intercept b": "截距 b",
    "Weight of the covariance term": "协方差项的权重",
    "Penalty for non-compatible sequences in the covariance term": "在协方差项中，不兼容序列的惩罚",
    "Score pairs with endgaps same as gap-gap pairs": "将具有末端间隙的对与间隙-间隙对的分数相同",
    "Use ribosum scoring matrix": "使用核糖体评分矩阵",
    "Select a reference transcriptome from your history or use a built-in index?": "从您的历史记录中选择一个参考转录组，或者使用内置索引？",
    "Select a reference transcriptome": "选择一个参考转录组",
    "Select the reference transcriptome": "选择参考转录组",
    "The size of the k-mer on which the index is built": "构建索引所使用的k-mer大小",
    "Relative orientation of reads within a pair": "配对中读取的相对方向",
    "File containing a mapping of transcripts to genes": "包含转录本到基因映射的文件",
    "Perform sequence-specific bias correction": "执行序列特异性偏差校正",
    "Perform fragment GC bias correction": "执行片段GC偏差校正",
    "Dump the equivalence class counts that were computed during quasi-mapping.": "导出在准映射过程中计算的等价类计数。",
    "The value by which to down-sample transcripts when representing the GC content": "用于表示GC含量时降低转录本采样率的值",
    "The value at which the fragment length PMF is down-sampled when evaluating GC fragment bias.": "在评估GC片段偏差时，片段长度概率质量函数（PMF）被下采样的值。",
    "Strict Intersect.": "严格交集。",
    "Calculate effective lengths": "计算有效长度",
    "Standard deviation": "标准差",
    "Maximal read mapping occurence": "最大读取映射发生次数",
    "Disable effective length correction": "禁用有效长度校正",
    "Use Variational Bayesian EM algorithm for optimization": "使用变分贝叶斯EM算法进行优化",
    "Discard orphaned reads as valid hits when performing lightweight-alignment": "在执行轻量级比对时，将孤立的读取作为有效的命中丢弃",
    "Use the un-smoothed approach to effective length correction": "使用未平滑的方法进行有效长度校正",
    "The maximum length of a fragment to consider when building the empirical fragment length distribution": "在构建经验片段长度分布时考虑的最大片段长度",
    "The key for aggregating transcripts during gene-level estimates": "在基因水平估计期间聚合转录本的关键是",
    "Disables strand-aware processing completely.": "完全禁用链特异性处理。",
    "Enforces strict library compatibility.": "强制执行严格的库兼容性。",
    "Allow paired-end reads from the same fragment to dovetail.": "允许来自同一片段的配对末端读取相互衔接。",
    "Number of fragment mappings to use when learning the sequene-specific bias model": "用于学习序列特异性偏差模型时使用的片段映射数量",
    "Number of fragments from unique alignments to sample when building the fragment length distribution": "在构建片段长度分布时，从唯一比对的片段中获取的片段数",
    "Number of Gibbs sampling rounds to perform.": "执行吉布斯抽样轮数的数量。",
    "Number of bootstrap samples to generate.": "生成的自举样本数量。",
    "Replicate": "复制",
    "Add replicate": "添加重复",
    "Set Advanced Cuffquant parameters?": "设置高级Cuffquant参数？",
    "advanced options": "高级选项",
    "Print out energy of each loop in the structure.": "打印出结构中每个环的能量。",
    "G-quadruplex formation prediction": "G-四链体形成预测",
    "Assume circular RNA": "假设是环状RNA",
    "Logarithmic energy functions for multi-loops": "对多环的对数能量函数",
    "calculate melting temperature": "计算熔解温度",
    "Concentration of the probe": "探针浓度",
    "Na concentration": "钠浓度",
    "Mg concentration": "镁浓度",
    "K concentration": "K 浓度",
    "Tris concentration": "Tris浓度",
    "Clustal file / Query sequences": "Clustal 文件 / 查询序列",
    "Clustal file / Target sequences": "Clustal 文件 / 目标序列",
    "maximal interaction length": "最大交互长度",
    "extension cost": "扩展成本",
    "scale all opening energy by a factor": "将所有开口能量乘以一个因子",
    "Calculate structures subject to constraints. Constraints have to be given in the input file": "计算受约束的结构。约束必须在输入文件中给出。",
    "Backtracking": "回溯",
    "Distance between target 3' ends of two consecutive duplexes": "两个连续双链体目标3'端之间的距离",
    "Minimum Energy Gain for a Duplex [kcal/mol]": "双链体的最小能量增益 [千卡/摩尔]",
    "Custom File": "自定义文件",
    "Number of backtrack of Boltzmann samples": "Boltzmann抽样的回溯次数",
    "[k]: Distance to first reference structure of neighborhood in which to backtrack": "[k]: 在回溯的邻域中，到第一个参考结构的距离",
    "[l]: Distance to second reference structure of neighborhood in which to backtrack": "[l]: 在回溯的邻域中，到第二个参考结构的距离",
    "Backtrack structures, don't calculate energy contributions only": "回溯结构，不只计算能量贡献",
    "Scaling factor for the partition function": "配分函数的缩放因子",
    "Maximum distance to first reference structure": "最大距离到第一个参考结构",
    "Maximum distance to second reference structure": "最大距离到第二个参考结构",
    "Comparison Option": "比较选项",
    "Print an alignment": "打印一个对齐结果",
    "weight of sequence compared to structure": "序列与结构的权重比较",
    "Allow free end-gaps": "允许自由端间隙",
    "DotBracket structure": "点括号结构",
    "Layout selection": "布局选择",
    "Output format selection": "输出格式选择",
    "Add annotation macros to postscript file": "向Postscript文件添加注释宏",
    "Add annotations to postscript file": "给postscript文件添加注释",
    "Select Filetype": "选择文件类型",
    "Query sequences": "查询序列",
    "Target sequences": "目标序列",
    "Maximal alignment length": "最大对齐长度",
    "Calculate the stem structure subject to constraints": "计算受约束的茎结构",
    "Maximal energy difference between the mfe and the desired suboptimal structure": "最大能量差，即在所需次优结构和最稳定折叠结构之间的能量差",
    "Backtracking based on the standard energy model": "基于标准能量模型的回溯",
    "Cost to add each nucleotide in a duplex": "在双链中添加每个核苷酸的成本",
    "Minimal right duplex energy": "最小右侧双链能量",
    "Minimal Loop energy": "最小环能量",
    "Minimal left duplex energy": "最小左侧双链能量",
    "Minimal duplex energy": "最小双链能量",
    "Minimal snoRNA stem length": "最小snoRNA茎长度",
    "Maximal snoRNA stem length": "最大snoRNA茎长度",
    "Minimal distance between the duplex end and the H/ACA box": "双链端与H/ACA框之间的最小距离",
    "Maximal distance between the duplex end and the H/ACA box": "双链末端与H/ACA盒之间的最大距离",
    "Minimal distance between the stem loop and the snoRNA sequence": "茎环结构与snoRNA序列之间的最小距离",
    "Maximal distance between the stem loop and the snoRNA sequence": "茎环与snoRNA序列之间的最大距离",
    "Minimal distance between the duplex and the snoRNA sequence": "双链体与snoRNA序列之间的最小距离",
    "Maximal distance between the duplex and the snoRNA sequence": "双链体与snoRNA序列之间的最大距离",
    "Minimal duplex stem energy": "最小双链茎能量",
    "Minimal total energy": "最小总能量",
    "Maximal snoRNA stem asymmetry": "最大snoRNA茎不对称性",
    "Minimal lower stem energy": "最小下干能量",
    "Move set options": "移动集选项",
    "Turn on shift moves.": "开启位移移动。",
    "Allow structures containing isolated base-pairs": "允许结构包含孤立的碱基对",
    "Simulation options": "模拟选项",
    "Number of trajectories to compute": "计算轨迹数",
    "Set maximum length of folding trajectory": "设置折叠轨迹的最大长度",
    "Simulate folding during transcription with a chain growth event taking place every rate timesteps.": "在转录过程中模拟折叠，每隔rate个时间步发生一次链增长事件。",
    "Start a folding during transcription simulation with an inital chain length of len.": "在转录模拟期间开始折叠，初始链长为len。",
    "compute first passage time": "计算首次通过时间",
    "Use the Metropolis rule for rate between two neighboring conformations": "使用 Metropolis 规则来计算两个邻近构象之间的速率",
    "Don’t print complete trajectory, but only local minimas encountered": "不要打印完整的轨迹，只打印遇到的局部最小值",
    "Cutoff energy": "截止能量",
    "Clustal alignment file": "Clustal 对齐文件",
    "Compute suboptimal structures with energy in a certain range of the optimum [kcal/mol]": "计算在最优能量值的一定范围内的次优结构[千卡/摩尔]",
    "How to treat dangling end energiesHow to treat dangling end energies for bases adjacent to helices in free ends and multi-loops": "如何处理末端能量，如何处理自由端和多环中靠近螺旋的碱基的末端能量",
    "Output will be sorted by free energy": "输出将根据自由能进行排序",
    "delta energy range [kcal/mol] for suboptimal structures": "次优结构的delta能量范围 [千卡/摩尔]",
    "Sort by energy": "按能量排序",
    "Primary and secondary stuctures file": "一级和二级结构文件",
    "Representation to calculate the distance / alignment option": "计算距离/比对选项的表示方法",
    "Use cost matrix by Bruce Shapiro": "使用布鲁斯·夏皮罗的成本矩阵",
    "base pair span": "碱基对跨度",
    "Activate Z−score computation. An optional argument may be supplied to set the threshold": "激活 Z 分数计算。可以提供可选参数来设置阈值",
    "Z-score": "Z分数",
    "Pseudoknots cutoff. Pseudoknots with smaller energy gains are rejected.": "伪结剪切值。能量增益较小的伪结被排除。",
    "Print suboptimal pseudoknots": "打印次优伪结",
    "Verbose mode, more output": "详细模式，更多输出",
    "cutoff probability for the reporting of the base pairs in the dot plot": "在点图(dot plot)中报告碱基对的截止概率",
    "Constraints for the secondary structure": "二级结构的约束",
    "Length of the unstructured region.": "无结构区域的长度。",
    "Specify which contributions to the probabilty of being unpaired are listed in the output": "指定输出中列出的未配对概率的贡献",
    "Maximal length of the region of interaction.": "相互作用区域的最大长度。",
    "Include the probability of unpaired regions in both RNAs": "在两个RNA中都包含未配对区域的概率",
    "Activate pairwise interaction mode": "激活成对交互模式",
    "Activate pairwise interaction mode, compare always with the first sequence": "激活成对交互模式，总是与第一个序列进行比较",
    "Extend the region of interaction in the target to some residues on the 5' side.": "将目标上的相互作用区域扩展到5'端的一些残基上。",
    "Extend the region of interaction in the target to some residues on the 3' side.": "将目标区域的相互作用扩展到3'端的一些残基上。",
    "Use scale*mfe as an estimate for the ensemble free energy": "使用 scale*mfe 作为整体自由能的估计",
    "Clustal file": "Clustal 文件",
    "Find so many sequences that fold close to that structure": "找到许多接近该结构的折叠序列",
    "Use these functions": "使用这些函数",
    "If partition function is selected this value determines the convergence criterium.": "如果选择了分区函数，这个值决定了收敛标准。",
    "minimal temperature [°C]": "最低温度 [°C]",
    "maximal temperature [°C]": "最高温度 [摄氏度]",
    "stepsize [°C]": "步长 [°C]",
    "data points to fit parabola to: 2*value+1": "数据点用于拟合抛物线：2*value+1",
    "Sort the secondary structures by energy": "根据能量对二级结构进行排序",
    "Calculate the density of states instead of the structure": "计算状态密度，而不是结构",
    "Calculate the Zuker suboptimals instead of the structures within a range.": "计算 Zuker 次优解，而不是范围内的结构。",
    "select algorithm": "选择算法",
    "Calculate this number of suboptimal structures chosen stochastically": "计算随机选择的次优结构的数量",
    "Range of energy for the suboptimal structures.": "次优结构的能量范围。",
    "select output restriction": "选择输出限制",
    "Print only those structures that have a smaller difference in energy than the mfe.": "只打印那些能量差小于最小自由能（mfe）的结构。",
    "Recalculate energies of structures using a logarithmic energy function for multi-loops": "使用对数能量函数重新计算多环结构的能量",
    "Remove non-canonical base pairs from the structure constraint": "从结构约束中移除非典型碱基对",
    "Sequence database": "序列数据库",
    "Subject covariance models": "主题协方差模型",
    "Covariance models": "协方差模型",
    "Covariance models file from the history.": "协方差模型文件来自历史记录。",
    "Turn on the glocal alignment algorithm": "打开全局比对算法",
    "Options controlling alignment algorithm": "控制比对算法的选项",
    "set RNG seed to 'n'": "将随机数生成器（RNG）种子设置为 'n'",
    "Use truncated alignment algorithm": "使用截断对齐算法",
    "Build sub CM for columns b/t HMM predicted start/end points": "构建子CM，用于HMM预测的起始/终止点之间的列。",
    "Options controlling speed and memory requirements": "控制速度和内存需求的选项",
    "HMM banding": "HMM 带状分析",
    "Tail loss probability for HMM bands": "HMM 带的尾部丢失概率",
    "Set the maximum allowable CM DP matrix size to 'x' megabytes": "将最大允许的CM DP矩阵大小设置为'x'兆字节",
    "Turn off the HMM band tightening strategy": "关闭HMM带宽压缩策略",
    "Set the maximum allowed value for tau during band tightening": "设置在带紧缩期间允许的最大tau值",
    "Use small memory divide and conquer algorithm": "使用小内存分而治之算法",
    "Read the aligment file that used to build the CM": "读取用于构建CM的对齐文件",
    "The aligment file that used to build the CM": "用于构建CM的比对文件",
    "Output the alignments as DNA sequence alignments": "输出DNA序列比对结果",
    "Annotate the output alignment with posterior probabilities": "使用后验概率注释输出的比对结果",
    "Only include match columns in the output alignment": "仅在输出对齐中包含匹配列",
    "Output the alignment in interleaved Stockholm format of a fixed width": "输出固定宽度的交错斯德哥尔摩格式的比对",
    "These options control how consensus columns are defined in an alignment": "这些选项控制了在比对中如何定义共识列",
    "Define the residue fraction threshold necessary to define a consensus (--symfrac)": "定义共识所需的残基分数阈值（--symfrac）",
    "Ignore the secondary structure annotation, if any, in your multiple alignment file (--noss)": "忽略多重比对文件中的二级结构注释（如果有的话）（--noss）",
    "Options controlling relative weights": "控制相对权重的选项",
    "Percent identity for clustering the alignment (--wid)": "聚类比对的相似度百分比（--wid）",
    "Controlling Filter P7 HMM construction": "控制过滤器P7 HMM构建",
    "For the filter p7 HMM, set minimum rel entropy/posn to": "对于过滤器 p7 HMM，设置最小相对熵/位置为",
    "Define the filter p7 HMM as the ML p7 HMM": "定义过滤器 p7 HMM 为最大似然 p7 HMM",
    "Number of sampled seqs to use for p7 local MSV calibration": "用于p7局部MSV校准的采样序列数",
    "Number of sampled seqs to use for p7 local Vit calibration": "用于 p7 本地 Vit 校准的采样序列数量",
    "Number of sampled seqs to use for p7 local Fwd calibration": "用于 p7 本地前向校准的采样序列数",
    "Number of sampled seqs to use for p7 glocal Fwd calibration": "用于p7 glocal Fwd校准的采样序列数",
    "Options controlling effective sequence number": "控制有效序列编号的选项",
    "Set the target mean match state relative entropy": "设置目标平均匹配状态相对熵",
    "Define the minimum allowed effective sequence number": "定义允许的最小有效序列编号",
    "Define the maximum allowed effective sequence number": "定义允许的最大有效序列号",
    "Set the target HMM mean match state relative entropy": "设置目标HMM平均匹配状态相对熵",
    "Set the effective sequence number for entropy weighting": "设置用于熵加权的有效序列编号",
    "Options for refining the input alignment": "输入比对的优化选项",
    "refinement mode": "细化模式",
    "Randam Seed": "随机种子",
    "Turn on the local alignment algorithm": "启用局部比对算法",
    "Turn off the truncated alignment algorithm": "关闭截断对齐算法",
    "Output the refined alignment file as it is used to build the covariance model": "输出经过精炼的比对文件，该文件用于构建共轭模型",
    "Output a summery file?": "输出一个摘要文件？",
    "Calibrate the covariance model": "校准协方差模型",
    "Total length of random sequences to search": "要搜索的随机序列的总长度",
    "Options controlling exponential tail fits": "控制指数尾部拟合的选项",
    " Fit the top 'n' hits/Mb in histogram for glocal modes": "将全局模式的前 'n' 命中/Mb 拟合到直方图中",
    " Fit the top 'n' hits/Mb in histogram for local modes": "将局部模式的直方图中的前'n'个命中/Mb拟合",
    "Set fraction of histogram tail to fit to exp tail": "设置直方图尾部拟合到指数尾部的比例",
    "Output extra files": "输出额外文件",
    "Optional output files": "可选输出文件",
    "Tail loss prob. for query dependent banding (QDB)": "查询依赖带状（QDB）的尾部丢失概率",
    "Turn off QDB during E-value calibration": "在 E 值校准期间关闭 QDB",
    "Turn off the null3 post hoc additional null model": "关闭null3事后附加的空模型",
    "use GC content of random null background model of CM": "使用 CM 的随机空模型的 GC 含量",
    "Use GC content distribution from file": "使用文件中的GC含量分布",
    "Subject covariance models <cmdb>": "主题协方差模型 <cmdb>",
    "Sequence database <seqfile>": "序列数据库 <seqfile>",
    "Auxillury files": "辅助文件",
    "Search space size in *Mb* for E-value calculations": "E值计算中搜索空间大小（以Mb为单位）",
    "Be verbose": "详细模式",
    "Inclusion thresholds": "包含阈值",
    "Use E-value": "使用 E 值",
    "Use bit score": "使用比特分数",
    "reporting thresholds": "报告阈值",
    "Options controlling model-specific reporting thresholds": "控制模型特定报告阈值的选项",
    "Use CM's GA gathering cutoffs as reporting thresholds": "使用CM的GA收集截止值作为报告阈值",
    "use CM's NC noise cutoffs as reporting thresholds": "使用CM的NC噪声截止值作为报告阈值",
    "use CM's TC trusted cutoffs as reporting thresholds": "使用CM的TC可信截止值作为报告阈值",
    "Options controlling acceleration heuristics": "控制加速启发式方法的选项",
    "P-value threshold for HMM stages": "HMM阶段的P值阈值",
    "Size of search space in Mb": "搜索空间大小（以Mb为单位）",
    "Skip truncated hit detection": "跳过截断的命中检测",
    "Allow full and truncated hits anywhere within sequences": "允许在序列的任何位置出现完整和截断的命中",
    "Turn off the null3 CM score corrections for biased composition": "关闭null3 CM分数校正以纠正偏倚组成",
    "Set the maximum allowable CM search DP matrix size to 'x' megabytes.": "将最大允许的CM搜索DP矩阵大小设置为'x'兆字节。",
    "Use the CYK algorithm, not Inside, to determine the final score of all hits": "使用CYK算法，而不是Inside算法，来确定所有命中的最终得分。",
    "Use the CYK algorithm to align hits": "使用CYK算法对命中结果进行比对",
    "Only search the bottom (Crick) strand of target sequences": "仅搜索目标序列的底部（克里克）链",
    "Only search the top (Watson) strand of target sequences": "仅搜索目标序列的顶部（沃森）链",
    "Calculate E-values as if the search space size is 'x' megabases (Mb)": "计算 E 值，假设搜索空间大小为 'x' 兆碱基（Mb）",
    "Save a multiple alignment of all significant hits": "保存所有显著命中的多重比对",
    "Omit the alignment section from the main input": "从主输入中省略对齐部分",
    "Include extra search pipeline statistics in the main output": "在主要输出中包含额外的搜索管道统计信息",
    "SNP": "单核苷酸多态性",
    "Windows size": "窗口大小",
    "RNA or DNA": "RNA 或 DNA",
    "Energy": "能量",
    "Min temperature [°C]": "最低温度 [摄氏度]",
    "Max temperature [°C]": "最大温度 [摄氏度]",
    "Temperature increment [°C]": "温度增量 [°C]",
    "Suffix": "后缀",
    "Salt concentration": "盐浓度",
    "Magnesium concentration": "镁浓度",
    "Compute relative entropy for all ppossible point mutations": "计算所有可能的点突变的相对熵",
    "Include dangling energies": "包含悬挂能量",
    "Polymer": "聚合物",
    "Zip": "压缩",
    "Log level": "日志级别",
    "List of SNP": "单核苷酸多态性列表",
    "Folding windows size": "折叠窗口大小",
    "Cut-off": "截断值",
    "Minimum length of the sequence interval": "序列区间的最小长度",
    "Span": "Span",
    "Length of the local structural element that we expect to have an effect": "预期产生影响的局部结构元素的长度",
    "Length of the interval over which the local structural changes are evaluated": "评估局部结构变化的区间长度",
    "p-value threshold to filter SNPs that are predicted using Mode 2": "p值阈值用于筛选使用模式2预测的SNPs",
    "p-value threshold to filter SNPs that are predicted using Mode 1": "p值阈值用于筛选使用模式1预测的SNPs",
    "Ensemble Euclidean distance": "集成欧几里得距离",
    "Boltzmann PreFactor": "玻尔兹曼预因子",
    "Other parameters": "其他参数",
    "Incorporate the read copy number": "整合读取拷贝数",
    "Please select one of the approaches": "请选择其中一种方法",
    "The maximum number": "最大数量",
    "Conversion": "转换",
    "Character from": "来自",
    "Character to": "字符到",
    "Required parameters": "必需参数",
    "BANDWIDTH": "带宽",
    "MINIMUM_READ_COUNT_PER_GROUP": "最小每组读数",
    "MINIMUM_READ_COUNT_PER_CLUSTER": "每个簇的最小读取数",
    "MINIMUM_READ_COUNT_FOR_KDE": "最小读取计数用于KDE",
    "MINIMUM_READ_COUNT_FOR_CLUSTER_INCLUSION": "最小读取计数以包含在聚类中",
    "MINIMUM_CLUSTER_SIZE": "最小聚类大小",
    "MINIMUM_CONVERSION_LOCATIONS_FOR_CLUSTER": "最小转换位置数以形成聚类",
    "MINIMUM_CONVERSION_COUNT_FOR_CLUSTER": "最小转换计数用于聚类",
    "MINIMUM_READ_LENGTH": "最小读取长度",
    "MAXIMUM_NUMBER_OF_NON_CONVERSION_MISMATCHES": "最大非转换错配数",
    "TopHat-Fusion fusion BAM file (used in TopHat-Fusion mapping)": "TopHat-Fusion 融合 BAM 文件（用于 TopHat-Fusion 映射）",
    "STAR Chimeric junction file (used in STAR mapping)": "STAR 嵌合连接文件（用于 STAR 映射）",
    "Gene annotation": "基因注释",
    "No-fix mode (useful for species with poor gene annotations)": "无固定模式（适用于基因注释较差的物种）",
    "Fasta file containing (exactly) one RBP sequence": "包含（确切地）一个RNA结合蛋白序列的Fasta文件",
    "BAM file of aligned RNA-Seq reads": "对齐的RNA-Seq读取的BAM文件",
    "Tabular file of potential fusions": "潜在融合的表格文件",
    "Num Fusion Reads": "融合读取数",
    "Num Fusion Pairs": "融合对数",
    "Num Fusion Reads + Pairs": "Num Fusion Reads + Pairs",
    "Fusion Read Mismatches": "融合读取错配",
    "Reads that map to more than this many places will be ignored.": "映射到超过这个数目的地方的序列将被忽略。",
    "Is your data from humans?": "您的数据来自人类吗？",
    "BED file containing read expressions": "包含读取表达量的BED文件",
    "minimum distance between two clusters": "两个簇之间的最小距离",
    "minimum height (readno) of a cluster": "最小高度（readno）的簇",
    "minimum height (readno) of a block": "最小高度（readno）的区块",
    "scale stddev for a single read": "为单个读取缩放标准差",
    "merge reads with almost similar means": "合并具有几乎相似均值的读取",
    "skip tags with expression smaller than this value": "跳过表达量小于这个值的标签",
    "Type of output": "输出类型",
    "Start with a structure other than the open chain": "以非开链结构开始",
    "Allow interrupted folding trajectories when the barrier is exceeded": "允许在超过障碍时中断折叠轨迹",
    "Barrier Heuristic": "障碍启发式",
    "How to treat conflict groups": "如何处理冲突组",
    "Number of basepairs that MorganHiggs forms its subpaths from": "MorganHiggs 从其子路径中形成的碱基对数",
    "Breadth of breadth first search": "广度优先搜索的广度",
    "Dangle value as in VienneRNA package": "VienneRNA包中的悬挂值",
    "Value of noLonelyPairs as in ViennaRNA": "在ViennaRNA中的noLonelyPairs的值",
    "Number of bases initially transcribed": "最初转录的碱基数量",
    "Number of bases transcribed per second": "每秒转录的碱基数量",
    "Max size of substructures considered for folding events during transcription": "在转录过程中考虑用于折叠事件的子结构的最大大小",
    "Structure constraint using RNA dotbracket notation with fuzzy block constraint.  (-Cstr)": "使用RNA点括号表示法进行结构约束，带有模糊块约束。(-Cstr)",
    "Objective target GC content in [0,1]. (-tGC)": "目标GC含量在[0,1]范围内。(-tGC)",
    "Number of sequences which shall be produced.  (-n)": "要生成的序列数量。（-n）",
    "Allowing GU base pairs.  (-GU)": "允许GU碱基对。(-GU)",
    "Provides a seed value for the used pseudo random number generator. (-s)": "为使用的伪随机数生成器提供一个种子值。（-s）",
    "Select the improving method.  h=hierarchical, s=score_based. (-ip)": "选择改进方法。h=层次化的，s=基于分数的。(-ip)",
    "Amount of maximal terrain resets, until the best solution is retuned as solution. (-r)": "最大地形重置次数，直到最佳解决方案被重新计算为解决方案。（-r）",
    "Delimits the convergence count criterion for a reset. (-CC)": "限定重置的收敛计数标准。(-CC)",
    "Delimits the amount of internal ants for termination convergence criterion for a reset. (-aTC)": "限定重置时终止收敛标准的内部蚂蚁数量。(-aTC)",
    "Switch to pseudoknot based prediction using pKiss. Check the pseudoknot parameter usage!!! (-p)": "切换到基于伪结的预测，使用 pKiss。检查伪结参数的使用！！！（-p）",
    "Enable optimized parameters for the usage of pseudo knots (Further parameter input ignored). (-pkPar)": "启用用于伪结的优化参数（忽略进一步的参数输入）。(-pkPar)",
    "Defining the pKiss folding strategy. (--strategy)": "定义 pKiss 折叠策略。(--strategy)",
    "Provides a maximum tGC value [0,1] for the case of uniform distribution sampling. The regular tGC value serves as minimum value. (-tGCmax)": "为均匀分布采样的情况提供了一个最大tGC值[0,1]。常规的tGC值作为最小值。(-tGCmax)",
    "Provides a tGC variance (sigma square) for the case of normal distribution sampling. The regular tGC value serves as expectation value (mu). (-tGCvar)": "为正态分布抽样的情况提供tGC方差（σ²）。常规的tGC值作为期望值（μ）。（-tGCvar）",
    "Provides a temperature for the folding algorithms. (-t)": "为折叠算法提供一个温度参数。(-t)",
    "Changes the energy parameterfile of RNAfold. If using this explicitly, please provide a suitable energy file delivered by RNAfold.  (-P)": "改变RNAfold的能量参数文件。如果明确使用此选项，请提供一个由RNAfold提供的合适的能量文件。(-P)",
    "Provide a path and an output file, e.g. \"/path/to/the/target_file\".  (-of)": "提供路径和输出文件，例如 \"/path/to/the/target_file\"。  (-of)",
    "Sequence constraint using RNA nucleotide alphabet {A,C,G,U} and wild-card \"N\".  (-Cseq)": "使用RNA核苷酸字母{A,C,G,U}和通配符\"N\"的序列约束。(-Cseq)",
    "Sets the level of allowed influence of sequence constraint on the structure constraint [0:no influence; 3:extensive influence]. (-l)": "设置序列约束对结构约束影响的允许程度[0:无影响; 3:广泛影响]。(-l)",
    "Defines a name which is used in the sequence output.  (--name)": "定义一个名称，该名称用于序列输出。（--name）",
    "Sets alpha, probability weight for terrain path influence. [0,1] (-a)": "设置 alpha，地形路径影响的概率权重。[0,1]（-a）",
    "Sets beta, probability weight for terrain pheromone influence. [0,1]  (-b)": "设置beta，地形信息素影响的概率权重。[0,1]  (-b)",
    "Pheromone evaporation rate.  (-er)": "信息素蒸发率。（-er）",
    "Structure constraint quality weighting factor. [0,1] (-Cstrw)": "结构约束质量加权因子。[0,1] (-Cstrw)",
    "GC content constraint quality weighting factor. [0,1] (-Cgcw)": "GC含量约束质量加权因子。[0,1] (-Cgcw)",
    "Sequence constraint quality weighting factor. [0,1] (-Cseqw)": "序列约束质量加权因子。[0,1] (-Cseqw)",
    "Displayes intermediate output. (-v)": "显示中间输出。(-v)",
    "Prints additional output to the headers of the produced sequences. (-ov)": "打印额外的输出到生成序列的头部。（-ov）",
    "Motif Descriptor File": "模式描述文件",
    "Sequence File": "序列文件",
    "Also search on complementary strands": "同时在互补链上进行搜索",
    "Skip overlapping matches": "跳过重叠的匹配",
    "Show Alignments": "显示比对",
    "File format of the reads (Default FASTQ)": "读取文件格式（默认为FASTQ）",
    "Fastq file": "Fastq 文件",
    "Reference genome/transcriptome": "参考基因组/转录组",
    "Number of nucleotides trimmed each round": "每轮修剪的核苷酸数量",
    "Minimum requirement of read length for mapping": "映射所需的最小读取长度",
    "Trimming end": "修剪末端",
    "Bowtie mapping flags (Default -v 0 -a --best --strata)": "Bowtie 映射标志（默认 -v 0 -a --best --strata）",
    "Trim n bases from high-quality (right) end of each read before alignment (-3)": "在比对前从每个读段的高质量（右侧）端修剪n个碱基（-3）",
    "Suppress all alignments for a read if more than n reportable alignments exist (-m), -1 for unlimited": "如果一个读取有超过n个可报告的比对，则抑制该读取的所有比对（-m），-1表示无限制",
    "Whether or not to make Bowtie guarantee that reported singleton alignments are 'best' in terms of stratum and in terms of the quality values at the mismatched positions (--best --strata)": "是否让Bowtie保证报告的单例比对是“最佳”的，无论是在层级上还是在不匹配位置的质量值上（--best --strata）",
    "Mapped file": "映射文件",
    "List of RNA ids to predict": "要预测的RNA ID列表",
    "Temperature (K)": "温度（K）",
    "Program for RNA structure prediction": "RNA结构预测程序",
    "RNA structure prediction type": "RNA结构预测类型",
    "Reactivity file": "反应性文件",
    "Slope used with structural restraints": "使用结构约束的斜率",
    "Intercept used with structural restraints": "使用结构约束的截距",
    "Threshold for high reactivities": "高反应性阈值",
    "Threshold for low reactivities": "低反应性阈值",
    "Incoorporate G-Quadruplex prediction if checked": "如果选中了，就结合 G-四链体预测",
    "RTSC file for (+) library": "(+)库的RTSC文件",
    "RTSC file for (-) library": "RTSC文件对于（-）库",
    "Nucleotide specificity": "核苷酸特异性",
    "Normalization is performed if checked": "如果选中，则执行标准化",
    "Threshold to cap the reactivities": "阈值用于限制反应活性",
    "Select tables to output": "选择要输出的表格",
    "Pool multiple read sets": "合并多个读取集",
    "Deep sequencing reads": "深度测序读取",
    "Remove reads with non-standard nucleotides": "移除含有非标准核苷酸的读段",
    "Convert RNA to DNA alphabet (to map against genome)": "将RNA转换为DNA字母表（以便与基因组进行比对）",
    "Clip 3' Adapter Sequence": "剪切3'端适配序列",
    "Sequence to clip": "序列剪辑",
    "Discard reads shorter than this length": "丢弃短于这个长度的读取",
    "Collapse reads and/or Map": "合并读取和/或映射",
    "Collapsed deep sequencing reads": "折叠深度测序读取",
    "Precursor sequences": "前体序列",
    "Mature miRNA sequences": "成熟miRNA序列",
    "Search in species": "在物种中搜索",
    "Star sequences": "星序列",
    "Sort reads by sample in PDF": "根据样本对PDF中的读取进行排序",
    "Include ID-flexible mapping": "包含ID灵活映射",
    "Skip file conversion": "跳过文件转换",
    "Skip mapping against precursor": "跳过针对前体的映射",
    "Consider the whole precursor as the 'mature sequence'": "将整个前体视为“成熟序列”",
    "Discard all read multimapper": "丢弃所有多次映射的读段",
    "Allowed mismatches": "允许的错配",
    "Mappings": "映射",
    "Mature miRNA sequences for this species": "这个物种的成熟miRNA序列",
    "Mature miRNA sequences for related species": "成熟miRNA序列相关物种",
    "Minimum read stack height": "最小读取栈高度",
    "Maximum precursors": "最大前体数",
    "Minimum miRNA score": "最小miRNA得分",
    "Disable randfold analysis": "禁用随机折叠分析",
    "Detailed fasta output": "详细的fasta输出",
    "Trinity fasta file": "Trinity 蛋白质序列文件",
    "RSEM Reference Source": "RSEM参考源",
    "Select RSEM reference": "选择 RSEM 参考",
    "RSEM reference": "RSEM参考",
    "RSEM Input file type": "RSEM 输入文件类型",
    "FASTQ type": "FASTQ 类型",
    "Library type": "文库类型",
    "Read 1 fastq file": "读取1个fastq文件",
    "Read 2 fastq file": "读取2个fastq文件",
    "fasta file": "FASTA 文件",
    "Read 1 fasta file": "读取1个fasta文件",
    "Read 2 fasta file": "读取2个fasta文件",
    "RSEM formatted SAM file": "RSEM格式的SAM文件",
    "Create bam results files": "创建BAM结果文件",
    "Reference transcript source": "参考转录本来源",
    "reference fasta file": "参考序列FASTA文件",
    "gtf": "基因转移格式（Gene Transfer Format，GTF）",
    "Map of gene ids to transcript (isoform) ids": "基因ID到转录本（异构体）ID的映射图",
    "reference name": "参考名称",
    "PolyA": "PolyA",
    "The length of the poly(A) tails to be added. (Default: 125)": "要添加的聚腺苷酸（poly(A)）尾的长度。（默认值：125）",
    "List of transcript IDs (one per line) that should should not have polyA tails added.": "应不添加聚腺苷酸尾部的转录本ID列表（每行一个）。",
    "Disable the conversion of 'N' characters to 'G' characters in the reference sequences": "禁用将参考序列中的'N'字符转换为'G'字符的功能",
    "Tabular count matrix": "表格计数矩阵",
    "Using RAVmodel": "使用RAVmodel模型",
    "Select a correlation coefficient": "选择一个相关系数",
    "Select whether to display the maximum value from dataset's PCs or avgLoadings": "选择是否显示数据集中主成分的最大值或平均载荷",
    "Output format of validated result": "验证结果的输出格式",
    "Normalize rows of datasets?": "标准化数据集的行？",
    "The number of top validated RAVs to check": "要检查的顶级验证RAVs的数量",
    "Reject length threshold": "拒绝长度阈值",
    "Length cutoff": "长度截止值",
    "e-value": "E值",
    "Include Sequences in GFF": "包含GFF中的序列",
    "Save rRNA hit seqs as FASTA": "保存rRNA命中序列为FASTA格式",
    "Tool module": "工具模块",
    "Annotation GTF file": "注释GTF文件",
    "RNA-seq FASTQ file(s)": "RNA测序FASTQ文件（们）",
    "Create BigWig file": "创建 BigWig 文件",
    "Scale BigWig to HPB": "将 BigWig 文件缩放到 HPB（高分辨率峰值）格式。",
    "Skip TopHat mapping": "跳过 TopHat 映射",
    "Skip TopHat fusion": "跳过 TopHat 融合",
    "Aligner": "比对器",
    "TopHat-Fusion file": "TopHat-Fusion 文件",
    "Paired-end aligner file": "配对末端比对文件",
    "STAR fusion file": "STAR融合文件",
    "MapSplice fusion file": "MapSplice 融合文件",
    "BWA fusion file": "BWA 融合文件",
    "segemehl fusion file": "Segemehl融合文件",
    "No fix mode": "无固定模式",
    "Low confidence": "低置信度",
    "Remove rRNA": "去除 rRNA",
    "Assemble file": "组装文件",
    "Detect alternative back-splicing": "检测替代性背剪接",
    "Characterize alternative splicing of cirRNAs": "研究环状RNA（circRNAs）的替代性剪接。",
    "Type of mapping": "映射类型",
    "No-fix mode": "无固定模式",
    "Calculate RPKM for cassete exons": "计算卡带外显子的RPKM值",
    "Sample annotation": "样本注释",
    "Pathways list": "通路列表",
    "Interactions data": "交互数据",
    "Filter P-value": "滤波 P 值",
    "VST": "VST",
    "Number of genes": "基因数量",
    "EPS": "EPS",
    "Correlation method": "相关性方法",
    "Correlation function": "相关函数",
    "Network type": "网络类型",
    "TOM type": "TOM类型",
    "Merge similar": "合并相似的",
    "Rank method": "排名方法",
    "Minimal number of genes per module": "模块中基因的最小数量",
    "Merging correlation threshold": "合并相关性阈值",
    "Cetrality measure to show in the plot": "中心度量展示在图中",
    "P-value for overrepresentation analysis": "过表达分析的P值",
    "Z-score transformation for GSEA analysis": "GSEA分析的Z分数转换",
    "Minimum size of gene sets for GSEA analysis": "GSEA分析中基因集的最小大小",
    "Maximum size of gene sets for GSEA analysis": "GSEA分析中基因集的最大大小",
    "Sample column name": "样本列名",
    "Experiment": "实验",
    "Name of the experiment": "实验名称",
    "Interaction summary file": "交互摘要文件",
    "Interaction summary file of a replicate": "重复样本的交互摘要文件",
    "Path to the genomic RBP crosslink or binding site locations (in BED format)": "基因组RBP交联或结合位点的位置（以BED格式）的路径",
    "How much context should be added at up- and downstream of each sequence": "每个序列的上游和下游应该添加多少上下文",
    "IntaRNA parameters file": "IntaRNA 参数文件",
    "Set 'off' if you want to disable structure, default 'on'": "如果你想禁用结构，设置为'off'，默认为'on'",
    "Time used for the optimization in seconds, default: 43200 (12h)": "优化所用时间（以秒为单位），默认值：43200（12小时）",
    "Filter the data for hybrids already detected by ChiRA": "过滤数据，筛选出已被ChiRA检测到的杂交体",
    "Input genome or sequence": "输入基因组或序列",
    "Scientific species name written separated by '_', such as: Homo_sapiens": "科学物种名称以'_'分隔书写，例如：Homo_sapiens",
    "Species tag to identify through experiment (i.e short 4 letters tag)": "物种标签，用于通过实验识别（即简短的4个字母标签）",
    "nBitscore": "n比特分数",
    "Repeats filter": "重复序列过滤器",
    "List of miRNA models to be searched": "要搜索的miRNA模型列表",
    "Choose the source for the database": "选择数据库的来源",
    "Pre-calculated datasets": "预计算数据集",
    "Input user Pre-calculated dataset": "输入用户 预计算数据集",
    "Select one miRNAture stage": "选择一个miRNA处理阶段",
    "Homology mode": "同源模式",
    "Select one or more pre-calculated set of blastn parameters, accordingly to the miRNAture publication.": "选择一个或多个预先计算的blastn参数集，根据miRNAture出版物。",
    "CSV file containing all RRIs that should be evaluated": "包含所有应评估的RRIs的CSV文件",
    "Model and feature files in a tar": "模型和特征文件在一个tar压缩文件中",
    "If you want to start from hand-curated feature files, use this for evaluating test set performance. Default: 'off'": "如果你想从手工整理的特征文件开始，使用这个选项来评估测试集的性能。默认值：'off'",
    "Occupied regions python object file containing a dictionary": "占用区域 Python 对象文件包含一个字典",
    "Tool function mode": "工具功能模式",
    "Quantification data source": "定量数据来源",
    "Average read length": "平均读取长度",
    "Fix StringTie annotation problem": "修复 StringTie 注释问题",
    "Analysis mode": "分析模式",
    "Annotation generated by StringTie merge": "由StringTie合并生成的注释",
    "Genome annotation": "基因组注释",
    "Transcriptome": "转录组",
    "Remove non-conventional chromosomes": "去除非传统染色体",
    "Paired samples between factors": "配对样本间的因子",
    "Generate count matrix files": "生成计数矩阵文件",
    "IsoformSwitchAnalyzeR R object": "IsoformSwitchAnalyzeR R对象",
    "Filter for consquences": "过滤结果",
    "Pre-filter parameters": "预过滤参数",
    "Gene expression cutoff": "基因表达截止值",
    "Isoform expresion cutoff": "异构体表达量截止值",
    "Remove single isoform genes": "移除单异构体基因",
    "DEXseq parameters": "DEXseq 参数",
    "Correct for confounding factors": "校正混杂因素",
    "Overwrite IF values": "覆盖IF值",
    "Reduce to switch genes": "减少到开关基因",
    "Reduce to genes with consequence potential": "减少到具有潜在影响的基因",
    "Novel isoform analysis parameters": "新异构体分析参数",
    "ORF identification method": "开放阅读框识别方法",
    "Maximal allowed premature termination codon-distance": "最大允许的过早终止密码子距离",
    "Sequence extraction parameters": "序列提取参数",
    "Remove short aminoacid sequences": "移除短的氨基酸序列",
    "Remove long aminoacid sequences": "删除长氨基酸序列",
    "Remove ORFs containint STOP codons": "移除包含终止密码子的开放阅读框",
    "Number of top switching features (genes/isoforms) to plot": "要绘制的顶级切换特征（基因/异构体）数量",
    "Full analysis advanced options": "完整分析高级选项",
    "Filter genes with functional consequences": "过滤具有功能后果的基因",
    "Sorting mode": "排序模式",
    "Number genes or isoform switches counts": "基因数量或异构体开关计数",
    "Summary as numbers of as fraction": "摘要为数字或分数",
    "Plot number/fraction of genes or isoforms": "绘制基因或同分异构体的数量或比例",
    "Simplify location": "简化位置",
    "Remove empty consequences": "删除空的后果",
    "Analysis opposite consequences in enrichment analysis": "在富集分析中分析相反的结果",
    "Gene name": "基因名称",
    "Single gene mode advanced options": "单个基因模式高级选项",
    "Rescale transcripts": "重新调整转录本",
    "Isoforms on minus strand should be inverted": "负链上的异构体应该被反转",
    "Add error bars": "添加误差棒",
    "Include prediction of coding potential information": "包括编码潜能信息的预测",
    "CPAT result file": "CPAT 结果文件",
    "Coding cutoff": "编码截止值",
    "CPC2 result file": "CPC2 结果文件",
    "Remove non-coding ORFs": "移除非编码开放阅读框",
    "Include Pfam information": "包含Pfam信息",
    "Include Pfam results (sequence analysis of protein domains)": "包含Pfam结果（蛋白质域的序列分析）",
    "Include SignalP results": "包含 SignalP 结果",
    "SignalP": "信号肽预测工具",
    "Minimum probability for calling a signal peptide": "信号肽预测的最小概率",
    "Include prediction of intrinsically disordered Regions (IDR) information": "包含内源性无序区域（IDR）信息的预测",
    "IUPred2A result file": "IUPred2A 结果文件",
    "Annotate binding sites": "注释结合位点",
    "Minimum IDBR binding size": "最小IDBR结合尺寸",
    "Minimum fraction of a predicted IDBR": "预测内部重复序列（IDBR）的最小分数",
    "NetSurfP-2 result file": "NetSurfP-2 结果文件",
    "Analyze switch consequences parameters": "分析交换后果参数",
    "Nucleotide length cutoff": "核苷酸长度截断",
    "Nucleotide length fraction cutoff": "核苷酸长度分数截止值",
    "Cutoff on Jaccard similarity between the overlap of two nucloetide sequences": "两个核苷酸序列重叠部分的Jaccard相似度的截断值",
    "Aminoacid lenght cutoff": "氨基酸长度截断",
    "Aminoacid length fraction cutoff": "氨基酸长度分数截止值",
    "Cutoff between the overlap of two aminoacid sequences": "两个氨基酸序列重叠的截止点",
    "Remove the comparison of isoforms where no consequences were found": "删除在未发现结果时的异构体比较",
    "Experiment design": "实验设计",
    "Input metadata file": "输入元数据文件",
    "Normalize data": "标准化数据",
    "NBins": "N个区间",
    "LWR": "LWR",
    "UPR": "UPR（未折叠蛋白反应）",
    "Trinity Assembly": "Trinity 组装",
    "Clan information": "家族信息",
    "The tabular result file from cmsearch": "来自cmsearch的表格结果文件",
    "Keep hits that only overlap with other hits that are not kept": "保留那些仅与未保留的命中区域重叠的命中区域",
    "Keep the intermediate sorted tblout file": "保留中间排序的tblout文件",
    "Provide trusted splice sites coordinates": "提供可信的剪接位点坐标",
    "Splice junction sites file": "剪接连接位点文件",
    "Subexon classifier score threshold": "子外显子分类器得分阈值",
    "Minimum retained intron converage": "最小保留内含子覆盖度",
    "Minimum transcript coverage": "最小转录覆盖度",
    "Library strand information": "库链信息",
    "Constrain cover iN  median exon depth (DP)": "限制覆盖度在中位数外显子深度（DP）内",
    "Use primary alignment to retain paralog genes": "使用主要比对以保留旁系同源基因",
    "Quantile for transcription start/end sites in subexon graph": "子外显子图中转录起始/终止位点的分位数",
    "GFF/GTF File": "GFF/GTF 文件",
    "Provide a reference sequence?": "提供参考序列？",
    "Fetch sequences for intervals in": "获取区间内的序列",
    "Interpret features when possible": "尽可能解释特征",
    "Select fasta header format": "选择fasta头格式",
    "Select fasta header field delimiter": "选择FASTA头字段分隔符",
    "Select a built-in GTF file or one from your history": "选择一个内置的GTF文件或者从你的历史记录中选择一个",
    "Select a GTF file": "选择一个 GTF 文件",
    "Analysis to perform": "要执行的分析",
    "Input equence file": "输入序列文件",
    "Does the sequence file have complete genomic sequences?": "该序列文件是否包含完整的基因组序列？",
    "Model": "模型",
    "Differentially Methylated Positions": "差异甲基化位点",
    "bindingType": "结合类型",
    "bindingRegionStart": "结合区域起始点",
    "bindingRegionEnd": "结合区域结束",
    "Additional Column of Score": "额外的得分列",
    "VCF dataset": "VCF 数据集",
    "How to treat individuals": "如何对待个体",
    "Column to convert": "列转换",
    "Format of input": "输入格式",
    "Column with genotype 1 count for group 1": "第一组基因型1的计数列",
    "Column with genotype 2 count for group 1": "第一组中基因型2的计数列",
    "Column with genotype 3 count for group 1": "第一组中基因型3的列计数",
    "Column with genotype 1 count for group 2": "第二组基因型1的计数列",
    "Column with genotype 2 count for group 2": "第二组中基因型2的计数列",
    "Column with genotype 3 count for group 2": "第2组中基因型3的列计数",
    "SNP Dataset": "SNP 数据集",
    "Group 1": "组 1",
    "Group 2": "组 2",
    "Database build": "数据库构建",
    "Disease term(s)": "疾病术语",
    "Sequence to plot": "序列绘图",
    "Value to plot": "要绘制的值",
    "Level": "水平",
    "Summarization mode": "摘要模式",
    "pgSnp dataset": "pgSnp 数据集",
    "Dataset has a column with the reference allele:": "数据集有一个包含参考等位基因的列：",
    "Column with reference allele": "参考等位基因的列",
    "Column with identifiers": "带有标识符的列",
    "Identifier type": "标识符类型",
    "Data to extract": "要提取的数据",
    "Interaction type": "相互作用类型",
    "Source file with aligned sequences": "带有对齐序列的源文件",
    "Substitution model": "替代模型",
    "Use empirical base frequencies in AA models": "在氨基酸模型中使用经验基础频率",
    "Random seed used for the parsimony inferences": "用于简约性推断的随机种子",
    "RAxML options to use": "RAxML选项使用",
    "Multiple boostrap specification": "多重自助法规范",
    "Number of alternative runs on distinct starting trees": "在不同起始树上的替代运行次数",
    "Bootstopping criteria": "停止引导的标准",
    "Column weight file": "列权重文件",
    "Secondary structure substitution model": "二级结构替代模型",
    "Random seed for non-parametric bootstrapping": "非参数自举的随机种子",
    "Random seed for rapid bootstrapping": "快速引导的随机种子",
    "MR cutoff threshold": "MR 截止阈值",
    "Number of Rate Categories for GTRCAT/GTRMIX": "GTRCAT/GTRMIX的速率类别数量",
    "Start ML optimization from a complete random starting tree": "从完全随机的起始树开始机器学习优化",
    "ML search convergence criterion": "机器学习搜索收敛标准",
    "Model optimization precision": "模型优化精度",
    "Exclude file": "排除文件",
    "Algorithm to execute": "要执行的算法",
    "ML tree searches under CAT model": "在CAT模型下进行最大似然树搜索",
    "Multifurcating constraint tree": "多岐分枝约束树",
    "Enable the ML-based evolutionary placement algorithm heuristics": "启用基于机器学习（ML）的进化定位算法启发式方法",
    "Initial rearrangement setting for the subsequent application of topological changes phase": "初始重排设置，用于后续应用拓扑变化阶段",
    "A posteriori bootstopping analysis": "后验引导分析",
    "Compute consensus tree": "计算共识树",
    "Print bootstrapped trees with branch lengths": "打印带有分支长度的引导树",
    "Specify a multi-state substitution model": "指定一个多状态替代模型",
    "Estimate individual per-partition branch lengths": "估计每个分区的分支长度",
    "Outgroup name": "外群名称",
    "Disable check for completely undetermined sequence in alignment": "禁用对齐中完全未确定序列的检查",
    "External AA (Protein) substitution model": "外部氨基酸（蛋白质）替代模型",
    "Assignment of models to alignment partitions for multiple models of substitution": "为多种替代模型分配模型到对齐分区",
    "Binary constraint tree": "二进制约束树",
    "Binary model parameter file": "二进制模型参数文件",
    "Secondary structure file": "二级结构文件",
    "Starting tree file": "起始树文件",
    "Use the median for the discrete approximation of the GAMMA model of rate heterogeneity": "使用中位数对GAMMA模型的速率异质性进行离散近似",
    "Save memory on large gappy alignments": "节省大型间隙对齐的内存",
    "Disable rate heterogeneity among sites model and use one without rate heterogeneity instead": "禁用不同位点之间的速率异质性模型，改用一个没有速率异质性的模型。",
    "Sliding window size for leave-one-out site-specific placement bias algorithm": "滑动窗口大小用于留一法位点特异性放置偏差算法",
    "Compute a randomized parsimony starting tree only": "计算一个随机化的简约性起始树",
    "Biochemical properties to use": "生化特性使用",
    "P-value threshold": "P值阈值",
    "Use site-level model fits to impute likely character states for each sequence": "使用站点级别的模型拟合来推断每个序列可能的特征状态",
    "Type of data": "数据类型",
    "Length of MCMC chain": "MCMC链的长度",
    "Number of samples to discard for burn-in": "烧入阶段要丢弃的样本数",
    "Number of steps to extract from chain sample": "从链样本中提取的步骤数",
    "Maximum number of parents allowed per node": "每个节点允许的最大父节点数",
    "Minimum number of ubstitutions per site to be included in the analysis": "每个位点包含在分析中的最少替换数",
    "Input FASTA or NEXUS file": "输入FASTA或NEXUS文件",
    "Alignment kind": "对齐类型",
    "Rate variation": "速率变异",
    "Rate classes": "速率类别",
    "Include support for multiple nucleotide substitutions": "包括对多个核苷酸替换的支持",
    "Include synonymous rate variation": "包含同义率变化",
    "Grid points": "网格点",
    "Concentration parameter of the Dirichlet prior": "狄利克雷先验的浓度参数",
    "Partitions": "分区",
    "Partition label": "分区标签",
    "Perform branch length re-optimization under the full codon model": "在完整密码子模型下执行分支长度的重新优化",
    "P-value": "P值",
    "Number of samples used to assess ancestral reconstruction uncertainty": "用于评估祖先重构不确定性的样本数量",
    "Compute profile likelihood confidence intervals for each variable site": "计算每个变量位点的轮廓似然置信区间",
    "Optimization precision for preliminary fits": "预拟合的优化精度",
    "Filter FASTA input?": "过滤FASTA输入？",
    "List of FASTA sequence IDs": "FASTA序列ID列表",
    "Pairwise Sequence Alignment Methods": "成对序列比对方法",
    "Pairwise Structual Alignment Methods": "成对结构比对方法",
    "Multiple Sequence Alignment Methods": "多序列比对方法",
    "Analysis Type": "分析类型",
    "Branches to use as the test set": "作为测试集使用的分支",
    "Branches to use as the reference set": "作为参考集使用的分支",
    "Multiple Sequence Alignment (MSA)": "多序列比对（MSA）",
    "NCBI datasource or other?": "NCBI 数据源或其他？",
    "Metadata from NCBI": "来自NCBI的元数据",
    "Metadata": "元数据",
    "GFF3 dataset": "GFF3 数据集",
    "Genome name": "基因组名称",
    "Gene features in JSON format generated by 'Get features by Ensembl ID' tool": "由“通过Ensembl ID获取特征”工具生成的JSON格式的基因特征",
    "Corresponding CDS datasets in FASTA format": "相应的CDS数据集在FASTA格式中",
    "Which transcripts to keep": "要保留哪些转录本",
    "Change the header line of the FASTA sequences to the following format": "将FASTA序列的标题行更改为以下格式",
    "Comma-separated list of region IDs (e.g. chromosomes or scaffolds) for which FASTA sequences should be filtered out": "逗号分隔的区域ID列表（例如染色体或支架），用于过滤FASTA序列",
    "Changes interleaved format": "更改交错格式",
    "Number of data sets": "数据集数量",
    "Data type": "数据类型",
    "Transition/transversion ratio": "转换/颠换比率",
    "Evolution model": "进化模型",
    "Proportion of invariant sites": "不变位点的比例",
    "Equilibrium frequencies": "平衡频率",
    "Number of categories for the discrete gamma model": "离散伽马模型的类别数目",
    "Parameter of the gamma model": "伽马模型的参数",
    "Tree topology search": "树拓扑搜索",
    "Optimise topology": "优化拓扑结构",
    "Statistical tests for branch support": "分支支持的统计检验",
    "Number of bootstrap replicates": "引导重复次数",
    "Number of seed used to initiate the random number generator": "用于初始化随机数生成器的种子数量",
    "Tree file": "树文件",
    "HyPhy SLAC Output": "HyPhy SLAC 输出",
    "HyPhy FEL Output": "HyPhy FEL 输出",
    "HyPhy MEME Output": "HyPhy MEME 输出",
    "HyPhy PRIME Output": "HyPhy PRIME 输出",
    "Alignment With Reference": "与参考序列比对",
    "Duplicates": "重复项",
    "GeneTree file": "基因树文件",
    "Species in order": "物种排序",
    "Select Gene node format": "选择基因节点格式",
    "GeneTree splitting algorithm": "基因树分裂算法",
    "In-group species list": "同组物种列表",
    "Out-group species list": "外群物种列表",
    "Find out gene gain/lose": "找出基因的获得/丢失",
    "Species file": "物种文件",
    "Input species tree format": "输入物种树格式",
    "Output GeneTree format": "输出基因树格式",
    "Pair types to keep": "保留的配对类型",
    "Allowed homology types for the GeneTree": "允许的基因树同源类型",
    "Get latest version": "获取最新版本",
    "NCBI taxdump": "NCBI taxdump",
    "(ETE3) Taxonomy Database": "(ETE3) 物种分类数据库",
    "Taxonomic levels": "分类水平",
    "Manual selection of ranks": "手动选择等级",
    "Fill unnamed ranks": "填充未命名的等级",
    "Prefer lower ranks": "优先选择较低的等级",
    "Include taxid in the table": "在表格中包含taxid",
    "Newick trees": "Newick树",
    "Verbosity level": "详细程度级别",
    "Reroot tree with outgroup (space separated list of leaf names)": "使用外群（以空格分隔的叶名列表）重新根植树",
    "Prune nodes from the tree (comma-separated list of leaf names)": "从树中剪除节点（以逗号分隔的叶节点名称列表）",
    "Prune: preserve length": "修剪：保留长度",
    "Unroots the tree": "去除树的根",
    "Sort branches": "排序分支",
    "Ladderize": "Ladderize",
    "Resolve polytomies": "解析多叉分支",
    "Standardize tree": "标准化树",
    "Use in TreeBest": "在TreeBest中使用",
    "hcluster output file": "hcluster 输出文件",
    "Minimum number of cluster elements": "最小聚类元素数",
    "Maximum number of cluster elements": "最大聚类元素数",
    "Species file in Newick format": "新克格式的物种文件",
    "CDS alignment in FASTA format": "CDS 序列对齐，FASTA 格式",
    "Skip PHYML": "跳过 PHYML",
    "Ignore the probability of gene evolution (NOT recommended)": "忽略基因进化的概率（不推荐）",
    "Apply constraint to PHYML": "对PHYML应用约束",
    "Discard species that do not appear at all": "丢弃那些根本没有出现的物种",
    "Constraining tree": "约束树",
    "Only build tree for genes from sequenced species": "仅为已测序物种的基因构建树",
    "Collapse alternative splicing forms": "折叠替代剪接形式",
    "Do not mask low-scoring segments": "不要遮盖低分片段",
    "Quality cut-off": "质量截止",
    "Number of rate categories for PHYML-HKY": "PHYML-HKY模型的速率类别数量",
    "tv/ts ratio (kappa)": "TV/TS比率（卡帕）",
    "Alpha parameter for Gamma distribution": "Gamma分布的Alpha参数",
    "Duplication probability": "重复概率",
    "Probability of a loss following a speciation": "物种形成后损失的概率",
    "Probability of a loss following a duplication": "复制后丢失的概率",
    "Probability of the presence of an inconsistent branch": "不一致分支存在的概率",
    "Gene tree": "基因树",
    "Protein alignments": "蛋白质比对",
    "Gene features": "基因特征",
    "Weight file": "权重文件",
    "Minimum edge weight": "最小边权重",
    "Only find single-linkage clusters": "只找到单链接聚类",
    "Minimum edge density between a join": "最小边密度在连接之间",
    "Breaking edge density": "断裂边缘密度",
    "Use once-fail-inactive-forever mode": "使用一次性失效永久禁用模式",
    "Weight resolution": "权重分辨率",
    "Category file": "类别文件",
    "Stringency level": "严格度水平",
    "Tabular data": "表格数据",
    "Reciprocal results": "互惠结果",
    "FASTA file with input sequences": "输入序列的FASTA文件",
    "Output insertion sequences?": "输出插入序列？",
    "Translate annotated genes based on GFF and gene list?": "根据GFF和基因列表翻译注释基因？",
    "Comma separated list of genes to translate.": "基因翻译的逗号分隔列表。",
    "GFF3 file containing custom gene map": "包含自定义基因图谱的GFF3文件",
    "Minimum length of nucleotide sequence to consider for alignment": "用于比对的核苷酸序列的最小长度",
    "Penalty for extending a gap.": "延伸一个间隙的惩罚。",
    "Penalty for opening of a gap.": "打开间隙的惩罚。",
    "Penalty for opening gaps at the beginning of a codon.": "在密码子开始处打开缺口的惩罚。",
    "Penalty for opening gaps in the body of a codon.": "在密码子主体中打开间隙的惩罚。",
    "Penalty for aligned nucleotides or aminoacids that differ in state during alignment": "在比对过程中，对于状态不同的已比对核苷酸或氨基酸的惩罚",
    "Score for encouraging aligned nucelotides or aminoacids with matching state.": "鼓励对齐的核苷酸或氨基酸与匹配状态的得分。",
    "Maximum length of insertions or deletions allowed to proceed with alignment.": "允许进行比对的插入或删除的最大长度。",
    "Seed length for nucleotide alignment.": "核苷酸序列比对的种子长度。",
    "Minimum number of seeds to search for during nucleotide alignment.": "在核苷酸比对过程中要搜索的最小种子数。",
    "Spacing between seeds during nucleotide alignment.": "核苷酸比对中种子之间的间距。",
    "Maximum number of mismatching nucleotides.": "最大不匹配核苷酸数",
    "Version of pangolin-data to use": "使用 pangolin-data 的版本",
    "Cached release of pangolin-data": "缓存的 pangolin-data 版本",
    "Download and use also latest UShER assignment cache?": "下载并使用最新的 UShER 分配缓存？",
    "Version of constellations to use": "使用星座的版本",
    "Cached constellations release": "缓存的星座释放",
    "Output multiple sequence alignment of input sequences": "输出输入序列的多序列比对",
    "Maximum proportion of Ns allowed": "允许的最大Ns比例",
    "Minimum query length allowed": "允许的最小查询长度",
    "Add expanded lineage column to output": "在输出中添加扩展的谱系列",
    "Include header line in output file": "在输出文件中包含标题行",
    "Version of database to use": "数据库使用的版本",
    "Cached nextclade database release": "缓存的 Nextclade 数据库版本",
    "Use advanced options": "使用高级选项",
    "Quality Control (QC) Config file": "质量控制（QC）配置文件",
    "Custom root sequence": "自定义根序列",
    "Custom reference tree": "自定义参考树",
    "Custom gene map": "自定义基因图谱",
    "Custom PCR primer sites": "定制PCR引物位点",
    "Configuration and data specific to a pathogen": "特定于病原体的配置和数据",
    "Include reference sequence in FASTA alignment output": "在FASTA比对输出中包含参考序列",
    "Input codon alignment": "输入密码子序列比对",
    "Keep deletions": "保留删除",
    "Multiple alignments": "多重比对",
    "Root node": "根节点",
    "Label for tree": "树的标签",
    "Input alignment": "输入比对",
    "Branch set": "分支集",
    "Label for set": "集合的标签",
    "Perform permutation significance tests": "执行排列显著性检验",
    "Significance value for site-tests": "位点测试的重要性值",
    "Significance value for FDR reporting": "FDR报告的重要性值",
    "Mutation-annotated tree object": "突变注释树对象",
    "matUtils mode": "matUtils 模式",
    "Select samples by explicitly naming them, one per line in a plain text file": "在纯文本文件中，通过显式命名每个样本，每行一个样本来选择样本",
    "Remove the selected samples": "删除选定的样本",
    "Select samples by membership in any of the indicated clade(s), comma delimited- e.g. -c clade1,clade2": "根据属于任何指定的类群（clade），用逗号分隔来选择样本 - 例如 -c 类群1,类群2",
    "Select samples by whether they contain any of the indicated mutation(s), comma delimited- e.g. -m mutation1,mutation2": "根据样本是否包含指定的突变（以逗号分隔），选择样本，例如：-m mutation1,mutation2",
    "Select samples by whether they have less than or equal to the maximum number of indicated equally parsimonious placements": "选择样本，根据它们是否具有小于或等于指示的等价简约放置的最大数量",
    "Select samples by whether they have less than or equal to the indicated maximum parsimony score (terminal branch length)": "根据样本是否具有小于或等于指定的最大简约性得分（末端分支长度）选择样本",
    "Select a specific sample and X context samples, formatted as 'sample_name:X'": "选择一个特定样本和X个上下文样本，格式为'样本名称:X'",
    "Remove samples which have branches of greater than the indicated length in their ancestry": "移除在其祖先中具有大于指定长度分支的样本",
    "Select two representative samples per clade": "选择每个类群的两个代表性样本",
    "Resolve all polytomies": "解决所有多分支问题",
    "Select the desired output": "选择所需的输出",
    "Select statistics and attribute information": "选择统计和属性信息",
    "File containing clade asssignments of samples": "包含样本类群分配的文件",
    "File mapping clades to their respective internal node identifiers": "文件映射类群到它们各自的内部节点标识符",
    "Minimum fraction of the lineage samples": "最小分支样本比例",
    "Remove current annotations": "移除当前注释",
    "Minimum allele frequency for finding the best clade root": "最小等位基因频率，用于寻找最佳支系根",
    "Minimum allele frequency for being masked": "最小等位基因频率，用于掩蔽",
    "Clip sample frequency": "剪切样本频率",
    "Metrics for sample placement certainty": "样本放置确定性的指标",
    "TSV file containing names of the samples to be renamed and their new names": "包含要重命名的样本名称及其新名称的TSV文件",
    "Use to automatically remove identifying information from the tree, including all sample names and private mutations": "用于自动从树中移除识别信息，包括所有样本名称和私有突变",
    "Mode of sorting new samples": "新样本排序模式",
    "Maximum number of equally parsimonious placements": "最大数量的等价简约放置",
    "Retain the branch lengths": "保留分支长度",
    "Collapse options": "折叠选项",
    "Collapse internal nodes of the input tree": "折叠输入树的内部节点",
    "Collapse internal nodes of the output tree": "折叠输出树的内部节点",
    "Output tree options": "输出树选项",
    "Save the new mutation-annotated tree object": "保存新的突变注释树对象",
    "Uncondensed final tree": "未压缩的最终树",
    "Generate multiple subtrees": "生成多个子树",
    "Generate a single subtree": "生成一个单独的子树",
    "Generate parsimony scores per node": "生成每个节点的简约性得分",
    "Multiple placements": "多重放置",
    "Combined reference and query alignments from TN93-Filter": "结合参考和查询比对结果，使用TN93筛选",
    "p-value to use": "使用 p 值",
    "HyPhy-Annotate branche labels in JSON format": "HyPhy-Annotate 分支标签为 JSON 格式",
    "Name of the gene or sequence being analyzed": "被分析的基因或序列的名称",
    "Default name for sequences that have no explicit label": "没有明确标签的序列的默认名称",
    "The sequence ID to highlight": "要高亮显示的序列ID",
    "HyPhy-FADE output dataset": "HyPhy-FADE输出数据集",
    "HyPhy-PRIME output dataset": "HyPhy-PRIME 输出数据集",
    "HyPhy-RELAX output dataset": "HyPhy-RELAX 输出数据集",
    "HyPhy-MEME output dataset": "HyPhy-MEME 输出数据集",
    "HyPhy-MEME-Full output dataset": "HyPhy-MEME-完整输出数据集",
    "HyPhy-BUSTED output dataset": "HyPhy-BUSTED输出数据集",
    "HyPhy-SLAC output dataset": "HyPhy-SLAC 输出数据集",
    "HyPhy-FEL output dataset": "HyPhy-FEL 输出数据集",
    "HyPhy-CFEL output dataset": "HyPhy-CFEL 输出数据集",
    "HyPhy-BGM output dataset": "HyPhy-BGM 输出数据集",
    "Summaries to merge": "摘要合并",
    "Annotations to merge": "要合并的注释",
    "Input function (gene family or pathway) abundance table": "输入函数（基因家族或途径）丰度表",
    "Argument specifying non-default reference files to use for sequence placement": "指定非默认参考文件以用于序列定位的参数",
    "Mapping table to use": "使用映射表",
    "Custom mapping table": "自定义映射表",
    "Newick tree with study sequences placed amongst reference sequences": "新克树，将研究序列放置在参考序列中",
    "Customized trait table": "定制性状表",
    "Seed to make output reproducible": "种子，用于生成可复现的输出",
    "Check input trait table before HSP": "在进行HSP（高得分对）之前，请检查输入的性状表。",
    "Table with predicted gene family copy numbers": "预测基因家族拷贝数的表格",
    "Table of predicted marker gene (16S or other) copy numbers": "预测的标记基因（16S或其他）拷贝数表",
    "Input table with gene family abundances": "输入包含基因家族丰度的表格",
    "Keep intermediate files": "保留中间文件",
    "Sequence placement options": "序列放置选项",
    "Hidden state prediction (HSP) options": "隐藏状态预测（HSP）选项",
    "Customized trait table(s)": "定制性状表（s）",
    "Metagenome prediction options": "宏基因组预测选项",
    "Infer pathways": "推断通路",
    "Functional database to use as reactions for inferring pathway abundances": "功能数据库，用作推断途径丰度的反应",
    "Default functional database": "默认功能数据库",
    "Keep intermediate files of": "保留中间文件",
    "Input prediction table": "输入预测表",
    "Number of shuffled replicates to create": "创建的洗牌重复次数",
    "Blastn Input Type": "Blastn 输入类型",
    "Blastn Results (Blast XML)": "Blastn 结果（Blast XML）",
    "Blastn Results": "Blastn 结果",
    "Number of results to return": "返回结果的数量",
    "Return Accession Numbers": "返回访问号码",
    "Automatically filter by phage Taxonomy IDs": "自动按噬菌体分类学ID过滤",
    "TaxIDs to filter out of results (Space separated)": "需要从结果中过滤掉的TaxIDs（以空格分隔）",
    "use dice method in percent similarity calculation": "在相似性计算中使用DICE方法",
    "Interval File": "区间文件",
    "Type of Output": "输出类型",
    "Cluster info": "集群信息",
    "Rename representative sequences with the  corresponding cluster name?": "将代表性序列重命名为相应的聚类名称？",
    "Representative sequences": "代表性序列",
    "Extract category distribution of each cluster?": "提取每个类别的聚类分布？",
    "Mapping file": "映射文件",
    "Sum sequence number for each category?": "为每个类别求和序列编号？",
    "DNA": "DNA",
    "Tree calculation/BOOTSTRAP options": "树计算/引导选项",
    "Output tree/distance forma": "输出树/距离格式",
    "Use Kimura correction": "使用金氏修正",
    "Ignore positions with gaps": "忽略带有间隙的位置",
    "Output alignment format": "输出比对格式",
    "Show residue numbers in clustal format output": "在Clustal格式输出中显示残基编号",
    "Output order": "输出顺序",
    "Output complete alignment (or specify part to output)": "输出完整比对（或指定要输出的部分）",
    "Start point": "起始点",
    "End point": "终点",
    "Select sequence file": "选择序列文件",
    "Select reference library type": "选择参考库类型",
    "Select reference library file": "选择参考库文件",
    "Select local cached library": "选择本地缓存的库",
    "Enable search stage?": "启用搜索阶段？",
    "Set number of candidate reference sequences retrieved from the k-mer based search": "设置从基于k-mer的搜索中检索到的候选参考序列的数量",
    "Set fraction of the search result that must share the same classification": "设置搜索结果中必须具有相同分类的比例",
    "Set k for the kmer based candidate search": "设置k值用于基于k-mer的候选搜索",
    "Set number of allowed mismatches within each kmer": "设置每个kmer内允许的不匹配数",
    "Skip fast family search?": "跳过快速家族搜索？",
    "Configures the candidate search to use absolute rather than length-relative scores for ordering the results?": "配置候选搜索，使用绝对分数而不是长度相对分数来排序结果？",
    "Set maximum number of search results to return for each query sequence": "为每个查询序列设置返回搜索结果的最大数量",
    "Bases remaining unaligned at the ends should be": "末端未对齐的碱基应该是",
    "Insertions wider than available columns should be": "插入的内容如果比可用的列宽，应该",
    "Enable turn check stage?": "启用转换检验阶段？",
    "Expert alignment parameters": "专家对齐参数",
    "Set minimum number of reference sequences used for each query": "设置每个查询使用的参考序列的最小数量",
    "Set maximum number of reference sequences used for each query": "设置每个查询使用的最大参考序列数",
    "Set minimum similarity reference sequences": "设置参考序列的最小相似性",
    "Set weight factor for the frequency at which a node was observed in the reference alignment": "设置参考比对中观察到节点频率的权重因子",
    "Set minimum number of reference sequences that must be found in order to attempt alignment": "设置至少需要找到多少个参考序列才能尝试进行比对",
    "Set minimum number of full length reference sequences that must be included in the selected reference set": "设置所选参考集必须包含的完整长度参考序列的最小数量",
    "Set minimum length a sequence is required to have to be considered full length": "设置序列必须具有的最小长度，以被视为完整长度。",
    "Set beginning of the gene within the reference alignment": "在参考比对中设置基因的起始位置",
    "Set end of the gene within the reference alignment": "在参考比对中设置基因的末端",
    "Set total of n sequences to cover each the beginning and the end of the gene within the alignment": "设置总共n个序列以覆盖对齐中基因的起始和结束部分",
    "Set score given for a match": "为匹配设置给定的分数",
    "Set score given for a mismatch": "为不匹配设置给定分数",
    "Set penalty subtracted from the score for opening a gap": "设置因打开一个间隙而从得分中减去的惩罚值",
    "Set penalty subtracted from the score for extending a gap": "设置扩展一个间隔时从得分中减去的惩罚值",
    "Set size of k for the reference search": "设置参考搜索的k值大小",
    "Set k-mer matches in the reference database to contain this number of mismatches": "将参考数据库中的k-mer匹配设置为包含这个数量的不匹配",
    "Use all k-mers occurring in the query sequence in the search?": "在搜索中使用查询序列中出现的所有k-mers？",
    "Use absolute match scores in the kmer search?": "在kmer搜索中使用绝对匹配分数？",
    "Exclude sequences sharing less than this value fractional identity with any of the alignment reference sequences from the output?": "排除与任何比对参考序列的相似性分数低于该值的序列，不将它们包含在输出中。",
    "Query sequence": "查询序列",
    "Reference sequence": "参考序列",
    "K-mer seed size": "K-mer种子大小",
    "Minimum similarity": "最小相似性",
    "Generate alignments file?": "生成对齐文件？",
    "Source sequence file": "源序列文件",
    "Specify a subset of labels to be used": "指定要使用的标签子集",
    "Sequence label": "序列标签",
    "Initial clustering options": "初始聚类选项",
    "Initial clustering algorithm": "初始聚类算法",
    "Maximal sequence shift": "最大序列偏移",
    "Sequence shift penalty": "序列位移惩罚",
    "Initial clustering order": "初始聚类顺序",
    "Seed for random number generation": "随机数生成的种子",
    "Substitiution matrix schema.": "替代矩阵架构。",
    "Set initial clustering threshold": "设置初始聚类阈值",
    "Initial clustering threshold": "初始聚类阈值",
    "HMM-clustering options": "HMM聚类选项",
    "How many initial clusters to use as cluster cores": "使用多少个初始聚类作为聚类核心",
    "The proporiton of the largest initial clusters to be used as cluster cores in subsquent clustering procedure.": "用于后续聚类过程中作为聚类核心的最大初始聚类的比例。",
    "Minimum size of an initial cluster needed for it to be used as cluster core in subsquent clustering procedure.": "初始聚类所需的最小簇大小，以便在后续的聚类过程中将其用作簇核心。",
    "The number of initial clusters to be used as cluster cores in subsquent clustering procedure.": "在后续聚类过程中要用作聚类核心的初始聚类数量。",
    "Select initial clusters on the basis of": "根据基础选择初始聚类",
    "Clustering rounds": "聚类轮次",
    "Relative/absolute scores": "相对/绝对分数",
    "Round": "舍入",
    "Assign threshold": "分配阈值",
    "Overlap threshold": "重叠阈值",
    "Merge threshold": "合并阈值",
    "Minimal number of conserved positions.": "保守位点的最小数量。",
    "Maximal proportion of gaps allowed on a conserved position": "允许在保守位置上的最大间隙比例",
    "Minimal information content allowed on a conserved position": "在保守位置上允许的最小信息量",
    "Maximum number of inner gaps": "最大内部间隔数",
    "Can MSA length be increased during extension step?": "在扩展步骤中，MSA（多序列比对）的长度可以增加吗？",
    "Min correlation": "最小相关性",
    "Output dotplot size": "输出点图大小",
    "Diffuse value": "扩散值",
    "Add grid to plot for multi-fasta data sets": "为多序列fasta数据集添加网格以绘图",
    "Generate image of detected events": "生成检测到的事件的图像",
    "Anchoring": "锚定",
    "Minimum Match Length": "最小匹配长度",
    "Direction": "方向",
    "Force 4 column output": "强制4列输出",
    "Match only the characters a, c, g, or t": "仅匹配字符 a、c、g 或 t",
    "Print length of query sequence in header of matches": "在匹配结果的标题中打印查询序列的长度",
    "Print first 53 characters of the matching substring": "打印匹配子字符串的前53个字符",
    "Position": "位置",
    "Suflink": "Suflink",
    "Child": "子节点",
    "Sparsify": "稀疏化",
    "kmer Table": "kmer表",
    "Do you want to output a 2-D dotplot of the input sequences?": "您是否想要输出输入序列的二维点图？",
    "Match File": "匹配文件",
    "Plot an ordered set of reference/query sequences?": "绘制一组有序的参考/查询序列？",
    "Layout": "布局",
    "Output files used to create the plot or just the plot?": "用于创建图表的输出文件，还是只是图表？",
    "Match file from nucmer": "从nucmer匹配文件",
    "Alignment Strategy": "对齐策略",
    "Minimum Identity": "最小相似性",
    "Minumum Legnth": "最小长度",
    "Overlaps": "重叠",
    "Minimum Alignment Uniqueness": "最小对齐唯一性",
    "Maximum Alignment Overlap": "最大对齐重叠",
    "Break Length": "断裂长度",
    "Minumum Cluster Length": "最小聚类长度",
    "Maximum Diagonal Difference": "最大对角线差",
    "No Extend": "不扩展",
    "Maximum Gap Distance": "最大间隙距离",
    "Minumum Alignment Length": "最小比对长度",
    "No Alignment Score Optimization": "无对齐分数优化",
    "Don't Simplify Alignments": "不要简化比对",
    "Banding": "条纹带",
    "Offsets": "偏移量",
    "Map genome to genome": "将基因组映射到基因组",
    "Max Chunk": "最大块",
    "Do you want to output a 2-D dotplot of the input sequences? (mummerplot)": "您是否希望输出输入序列的二维点图？(mummerplot)",
    "Output only the general report file?": "只输出通用报告文件？",
    "Match file from Nucmer": "匹配来自Nucmer的文件",
    "Identity": "同一性",
    "Minimum Alignment Length": "最小比对长度",
    "Annotate": "注释",
    "Sorting strategy for output": "输出排序策略",
    "Sequences to add to the alignment": "要添加到对齐中的序列",
    "What do you want to add to the alignment": "您想要向比对添加什么内容",
    "Preserve the original alignment": "保留原始比对",
    "Keep alignment length": "保持对齐长度",
    "Output a correspondance table of positions (--mapout)": "输出位置对应表（--mapout）",
    "Preserve the original order of sequences (--reorder)": "保留序列的原始顺序（--reorder）",
    "Sequences to align": "序列对齐",
    "MAFFT flavour": "MAFFT 版本",
    "Guide tree is built this number of times in the progressive stage.": "引导树在渐进阶段构建了这个次数。",
    "Use a fast tree-building method?": "使用快速树构建方法？",
    "Number of partitions in the PartTree algorithm": "PartTree算法中的分区数",
    "Do not make alignment larger than ... sequences": "不要使比对结果大于...序列",
    "Weighting factor for the consistency term calculated from pairwise alignments.": "从成对比对中计算出的一致性项的加权因子。",
    "Gap opening penalty at local pairwise alignment": "局部配对时的间隙开放惩罚",
    "Offset value at local pairwise alignment": "局部配对序列比对中的偏移值",
    "Gap extension penalty at local pairwise alignment.": "局部配对时的间隙延伸惩罚。",
    "Gap opening penalty to skip the alignment": "间隙开放惩罚以跳过比对",
    "Gap extension penalty to skip the alignment": "间隙延伸惩罚以跳过对齐",
    "Use FFT approximation in group-to-group alignment?": "在组到组比对中使用快速傅里叶变换（FFT）近似？",
    "Check alignment score in the iterative refinement stage?": "在迭代细化阶段检查比对分数？",
    "Gap extend penalty for group-to-group alignment": "组间比对的间隙延伸惩罚",
    "Gap opening penalty at group-to-group alignment.": "在组间比对中的间隙开放惩罚。",
    "Matrix selection": "矩阵选择",
    "Coefficient of the BLOSUM matrix": "BLOSUM矩阵的系数",
    "Coefficient of the JTT PAM matrix": "JTT PAM矩阵的系数",
    "Coefficient of the  transmembrane PAM matrix": "跨膜PAM矩阵的系数",
    "User-defined AA scoring matrix": "用户定义的氨基酸评分矩阵",
    "Incorporate the AA/nuc composition information into the scoring matrix?": "将氨基酸/核苷酸组成信息纳入评分矩阵？",
    "Reorder output?": "重新排序输出？",
    "Display alignment tree ?": "显示比对树？",
    "Start Position(s)": "起始位置",
    "End Position(s)": "结束位置",
    "Homology distance limit": "同源距离限制",
    "Compress": "压缩",
    "Input sequence file": "输入序列文件",
    "Length threshold for full sequences": "完整序列的长度阈值",
    "Alphabet for input sequences": "输入序列的字母表",
    "Add fragments without clusters": "添加无聚类的片段",
    "Add long sequences with too many ambiguous residues": "添加具有太多不确定残基的长序列",
    "Percent sequence similarity for clustering": "聚类时的序列相似性百分比",
    "Merge strategy": "合并策略",
    "Number of iterations to refine long alignments": "迭代次数以优化长序列比对",
    "Number of iterations to refine merged alignment": "合并对齐后进行细化的迭代次数",
    "Exclude clusters from final alignment": "排除最终比对中的簇",
    "Input clustal alignment": "输入clustal比对",
    "Number of sequences in the output alignment.": "输出对齐中序列的数量。",
    "Optimize for this percentage of mean pairwise identity": "优化以获得平均配对一致性的这个百分比",
    "Sequences with a higher percentage of pairwise identity will be removed.": "具有更高百分比的配对比对一致性的序列将被移除。",
    "The first sequence (=reference sequence) is always present in the output alignment.": "输出的比对中总是包含第一个序列（=参考序列）。",
    "FASTA input file": "FASTA 输入文件",
    "Specify the number of bootstrapping replicates to generate.": "指定要生成的自助法重复次数。",
    "Output log file": "输出日志文件",
    "Multi-FASTA file": "多序列FASTA文件",
    "Maximum number of iterations to run (-maxiters)": "最大迭代次数（-maxiters）",
    "Diagonal optimization (-diags)": "对角线优化（-diags）",
    "Target RNA Alignment": "目标RNA比对",
    "Reference RNA Alignment": "参考RNA比对",
    "Sequences to cluster/compare": "序列聚类/比较",
    "Cluster / Compare (i.e. call cd-hit[-est] / cd-hit[-est]-2d)?": "聚类/比较（即调用cd-hit[-est] / cd-hit[-est]-2d）？",
    "Other sequences to cluster/compare": "其他要聚类/比较的序列",
    "Sequence type?": "序列类型？",
    "Sequence identity threshold": "序列相似性阈值",
    "Word size": "单词大小",
    "Tolerance for redundance": "冗余容忍度",
    "Compare both strands?": "比较两个链？",
    "Advanced EST alignment options": "高级EST比对选项",
    "Masking letters": "掩蔽字母",
    "Match score": "匹配得分",
    "Mismatch score": "不匹配分数",
    "Gap opening score": "间隔开放分数",
    "Gap extension score": "间隙延伸分数",
    "Alignment band width": "对齐带宽",
    "Length of throw away sequences": "丢弃序列的长度",
    "Use local/global sequence identity": "使用局部/全局序列相似性",
    "Alignment coverage for the longer sequence": "较长序列的比对覆盖度",
    "Alignment coverage control for the longer sequence": "较长序列的比对覆盖度控制",
    "Alignment coverage for the shorter sequence": "较短序列的比对覆盖度",
    "Alignment coverage control for the shorter sequence": "较短序列的比对覆盖度控制",
    "Minimal alignment coverage control for the both sequences": "最小对齐覆盖度控制，用于两个序列",
    "Length difference cutoff": "长度差异截断值",
    "Length difference cutoff in residues": "残基中长度差异的截止值",
    "Advanced options for 2D local alignment": "2D局部比对的高级选项",
    "Length difference cutoff for other sequences": "其他序列的长度差异截止值",
    "Length difference cutoff in residues for other sequences": "残基长度差异阈值，用于其他序列",
    "Maximum unmatched percentage for the shorter sequence": "较短序列的最大不匹配百分比",
    "Maximum unmatched length": "最大不匹配长度",
    "Sequences are stored in RAM": "序列存储在RAM中",
    "Accurate but slow mode": "精确但慢的模式",
    "Sort clusters by size": "按大小排序聚类",
    "Sort FASTA/FASTQ by cluster size": "根据聚类大小对FASTA/FASTQ文件进行排序",
    "Print alignment overlap in .clstr file?": "在 .clstr 文件中打印对齐重叠？",
    "Length of description in .clstr file": "在 .clstr 文件中描述的长度",
    "Query sequences to mash against the references supplied below": "查询序列与下面提供的参考序列进行比较",
    "Reference or references to mash the query sequences on": "参考或参考序列用于对查询序列进行比对",
    "Identity threshold": "一致性阈值",
    "Minimum segment length": "最小段长度",
    "Sketch size - leave 0 for automatic setting based": "草图大小 - 留0以自动设置",
    "Dense sketching": "密集草图绘制",
    "Ignore the top % most-frequent kmer window": "忽略最频繁的kmer窗口的前%。",
    "Threshold for kmer complexity": "kmer复杂度的阈值",
    "Report predicted ANI values in [0, 100]": "报告预测的ANI值在[0, 100]范围内。",
    "Don't merge consecutive segment-level mappings": "不要合并连续的段级映射",
    "Use MashMap2 first pass filtering": "使用 MashMap2 进行首轮过滤",
    "What do you want to add to the alignment?": "您想要添加什么到比对中？",
    "Preserve the original alignment.": "保留原始比对。",
    "Output a correspondance table of position.": "输出一个位置对应表。",
    "Preserve the original order of sequences.": "保持序列的原始顺序。",
    "For multiple inputs generate": "对于多个输入生成",
    "Input batch": "输入批次",
    "Type of sequences": "序列类型",
    "Type of scoring matrix": "评分矩阵类型",
    "PAM value of the matrix": "矩阵的PAM值",
    "GuideTree-Generation": "GuideTree生成",
    "Guide tree is build this number of times in the progressive stage.": "引导树在渐进阶段构建了这个次数。",
    "Which distance for the fast tree-building method?": "哪种距离用于快速建树方法？",
    "Progressive alignment calculation": "逐步比对计算",
    "Keep alignment tree as output?": "保持对齐树作为输出？",
    "Create a 2-D dotplot of the input sequences?": "创建输入序列的二维点图。",
    "Order of Isolates": "分离菌株的顺序",
    "List of Isolates to show": "隔离列表展示",
    "Show the size of the spacers over the boxed diamonds?": "显示框内菱形上方间隔的大小？",
    "Skip the verification of the fasta file generated from the GFF file?": "跳过从GFF文件生成的fasta文件的验证？",
    "Score cutoff for pairing of the spacers": "间隔序列配对的得分阈值",
    "Gray out unique?": "灰色显示唯一？",
    "Gray out similar?": "灰化相似项？",
    "Change spacer colour?": "更改间隔颜色？",
    "Previous spacer.mcl.final": "上一个间隔序列.mcl.final",
    "Previous SVG": "上一个 SVG",
    "Specify Treated samples or Control": "指定处理样本或对照组",
    "Treated Sample Labels (or Indexes)": "处理样本标签（或索引）",
    "Control Sample Label": "对照样本标签",
    "Control Sample Labels (or Indexes)": "对照样本标签（或索引）",
    "Output normalized counts file": "输出标准化计数文件",
    "Output R files": "输出 R 文件",
    "Output Log file": "输出日志文件",
    "Method for normalization": "标准化方法",
    "Gene test FDR-adjusted Threshold": "基因测试 FDR 校正阈值",
    "Estimate variances from designated samples": "从指定样本估计方差",
    "Remove zero": "删除零",
    "Remove zero Threshold": "移除零阈值",
    "Gene Log-Fold Change Method.": "基因对数倍数变化方法。",
    "Control sgRNAs file": "控制sgRNAs文件",
    "CNV profile file": "拷贝数变异(CNV)轮廓文件",
    "Cell line column": "细胞系列",
    "Design matrix or sample labels": "设计矩阵或样本标签",
    "Design Matrix file": "设计矩阵文件",
    "Sample labels": "样本标签",
    "Control sample": "对照样本",
    "Number of genes for mean-variance modeling": "用于均值-方差建模的基因数量",
    "Number of permutations": "排列数",
    "Try to remove outliers": "尝试去除异常值",
    "Incorporate sgRNA efficiency": "整合 sgRNA 效率",
    "sgRNA efficiency file": "sgRNA效率文件",
    "sgRNA ID column": "sgRNA ID列",
    "sgRNA score column": "sgRNA评分列",
    "Update efficiency": "更新效率",
    "Gene Ranking file": "基因排名文件",
    "Pathway GMT file": "通路GMT文件",
    "Gene Ranking column number": "基因排名列号",
    "Name of the pathway to be tested": "要测试的通路名称",
    "Reverse the order of the gene": "反转基因的顺序",
    "Sort the pathways by p value": "按p值对通路进行排序",
    "Number of permutation number": "排列数的数量",
    "Reads Files or Count Table?": "读取文件或计数表？",
    "Sample reads": "样本读取",
    "sgRNA library file": "sgRNA库文件",
    "Specify sample labels": "指定样本标签",
    "Counts Table": "计数表",
    "Output Count Summary file": "输出计数汇总文件",
    "Output Normalized Counts file": "输出标准化计数文件",
    "Output Unmapped reads": "输出未映射的读取",
    "Pathway file for QC": "QC的通路文件",
    "5' Trim length": "5'端修剪长度",
    "Length of the sgRNA": "sgRNA长度",
    "Count sgRNAs with Ns": "统计含有N的sgRNAs",
    "Test running": "测试运行",
    "Single ranking file": "单个排名文件",
    "Method for testing pathway enrichment": "通路富集测试方法",
    "Gene Summary file column": "基因摘要文件列",
    "Alpha value for RRA pathway enrichment": "RRA通路富集的Alpha值",
    "Permutation number for GSEA": "GSEA的排列数",
    "Mykrobe results to parse.": "Mykrobe 结果解析。",
    "Optional Parameters": "可选参数",
    "Sequence quality options": "序列质量选项",
    "Average quality": "平均质量",
    "Gap opening": "间隙开放",
    "Gap extension": "间隙延伸",
    "Primer mismatch": "引物错配",
    "Donor mismatch": "供体不匹配",
    "Scoring matrix options": "评分矩阵选项",
    "Match scoring": "匹配得分",
    "Mismatch scoring": "不匹配评分",
    "Base only": "仅基于",
    "Scoring matrix type": "打分矩阵类型",
    "FASTQ files to use": "要使用的FASTQ文件",
    "Primer dimer": "引物二聚体",
    "Event filter": "事件过滤器",
    "Cut buffer": "切割缓冲区",
    "Promiscuous consensus": "普适性共识",
    "Alignment format output": "对齐格式输出",
    "Method used for plotting": "绘图使用的方法",
    "Plotting tool that computed coordinates": "绘图工具，计算坐标",
    "x coordinate": "x坐标",
    "y coordinate": "y坐标",
    "Color by": "按颜色分类",
    "Use the layers attribute?": "使用 layers 属性？",
    "Layers for x coordinate": "X坐标层",
    "Layers for y coordinate": "y坐标层",
    "Layers for color": "颜色层",
    "Plotting tool that computes coordinates": "绘图工具，计算坐标",
    "Plot data points with higher values on top of others?": "将数据点绘制成高值在其他值之上的图？",
    "Plot attributes": "绘图属性",
    "Keys for accessing variables": "访问变量的键",
    "Violin plot attributes": "小提琴图属性",
    "Display keys in multiple panels": "在多个面板中显示键",
    "Width of the figure": "图形的宽度",
    "Height of the figure": "图形的高度",
    "Label of the x axis": "x轴标签",
    "Rotation of xtick labels": "旋转 x 轴刻度标签",
    "Categorical annotation to plot with a different color map": "分类注释以不同颜色映射进行绘图",
    "Parameters for seaborn.clustermap": "seaborn.clustermap 的参数",
    "Linkage method to use for calculating clusters": "用于计算聚类的连锁方法",
    "Distance metric to use for the data": "用于数据的距离度量",
    "Calculate z-scores for the rows or the columns?": "计算行或列的z分数？",
    "Standardize a dimension?": "标准化一个维度？",
    "Cluster the columns?": "聚类列？",
    "Cluster the rows?": "聚类行？",
    "Plot highly variable genes or gene subset?": "绘制高变异性基因或基因子集？",
    "Number of top genes": "顶级基因数量",
    "Key for field in '.var' that stores gene symbols": "在 '.var' 文件中存储基因符号的字段的键",
    "Plot settings": "绘图设置",
    "Color for all of the elements, or seed for a gradient palette": "为所有元素设置颜色，或为渐变色板设置种子",
    "Colors to use for the different levels of the hue variable": "用于色调变量不同级别的颜色",
    "Proportion of the original saturation to draw colors at": "原始饱和度用于绘制颜色的比例",
    "List of comma-separated components": "逗号分隔的组件列表",
    "Number of PCs to show": "要显示的主成分个数",
    "Plotting layout": "绘图布局",
    "The embedding over which the density was calculated.": "密度计算所依据的嵌入。",
    "Name of the .obs covariate that contains the density estimates": ".obs 协变量的名称，其中包含密度估计",
    "Name of the condition used in tl.embedding_density": "在 tl.embedding_density 中使用的条件名称",
    "The category in the categorical observation annotation to be plotted": "要绘制的分类观察注释中的类别",
    "Dot size for background data points not in the group": "背景数据点不在组内的点的大小",
    "Dot size for foreground data points in the group": "组内前景数据点的点大小",
    "Plot the timeseries as heatmap?": "将时间序列绘制为热图？",
    "A path through nodes of the abstracted graph": "抽象图中节点的路径",
    "Key to plot": "关键绘图",
    "Number of data points to include in computation of running average": "计算移动平均值时包含的数据点数量",
    "Key of the grouping used to run PAGA": "用于运行 PAGA 的分组键",
    "Plot the node names on the nodes bar?": "在节点条上绘制节点名称？",
    "Show the colorbar?": "显示色条？",
    "Show the y ticks?": "显示 y 轴刻度？",
    "Shift and scale the running average to [0, 1] per gene?": "将每个基因的运行平均值进行平移和缩放至[0, 1]？",
    "Font size for gene names": "基因名称的字体大小",
    "Should the y-axis of each panels be shared?": "每个面板的 y 轴是否应该共享？",
    "Which genes to plot?": "要绘制哪些基因？",
    "List of genes to plot": "要绘制的基因列表",
    "Split the violins?": "分割小提琴？",
    "Method used for filtering": "过滤使用的方法",
    "Minimum number of counts required for a cell to pass filtering": "每个细胞通过过滤所需的最小计数数",
    "Maximum number of counts required for a cell to pass filtering": "一个细胞通过筛选所需的最大计数数",
    "Minimum number of genes expressed required for a cell to pass filtering": "细胞通过过滤所需的最小表达基因数",
    "Maximum number of genes expressed required for a cell to pass filtering": "细胞通过筛选所需的最大表达基因数",
    "Minimum number of counts required for a gene to pass filtering": "一个基因要通过过滤所需的最小计数数",
    "Maximum number of counts required for a gene to pass filtering": "一个基因通过过滤所需的最大计数数",
    "Minimum number of cells expressed required for a gene to pass filtering": "一个基因通过筛选所需的最小表达细胞数",
    "Maximum number of cells expressed required for a gene to pass filtering": "基因通过筛选所需的最大表达细胞数",
    "Key in adata.uns where the rank_genes_groups output is stored": "在adata.uns中输入rank_genes_groups输出存储的位置",
    "The key of the observations grouping to consider": "观察分组的关键要考虑",
    "Key that will contain new values": "将包含新值的键",
    "Minimum fraction of genes expressing the gene within the categories": "基因在类别中表达的最小基因比例",
    "Maximum fraction of genes expressing the gene outside the categories": "基因表达在类别之外的最大分数",
    "Minimum fold change": "最小倍数变化",
    "Flavor for computing normalized dispersion": "计算归一化离散度的风格",
    "Minimal mean cutoff": "最小均值截止",
    "Maximal mean cutoff": "最大均值截断",
    "Minimal normalized dispersion cutoff": "最小归一化离散度截止值",
    "Maximal normalized dispersion cutoff": "最大归一化离散度截断",
    "Number of highly-variable genes to keep": "保留高变异性基因的数量",
    "Number of bins for binning the mean gene expression": "平均基因表达的分箱数量",
    "Inplace subset to highly-variable genes?": "在地子集到高变异基因？",
    "Type of subsampling": "子采样类型",
    "Subsample to this 'fraction' of the number of observations": "将观测值的数量减采样到这个“分数”",
    "Subsample to this number of observations": "将观测值的数目降低到这个数目",
    "Random seed to change subsampling": "随机种子，用于改变子采样",
    "Target total counts per cell": "每个细胞的目标总计数",
    "Target total counts": "目标总数",
    "Sample the counts with replacement?": "是否替换抽样计数？",
    "Method used": "使用的方法",
    "Flavor for the clustering": "聚类风格",
    "Resolution": "分辨率",
    "Key under which to add the cluster labels": "将聚类标签添加到的键",
    "Interpret the adjacency matrix as directed graph?": "将邻接矩阵解释为有向图？",
    "Coarseness of the clusterin": "簇蛋白的粗糙度",
    "How many iterations of the Leiden clustering algorithm to perform.": "执行Leiden聚类算法的迭代次数。",
    "Number of dimensions of the representation": "表示的维度数",
    "Number of PCs to use": "使用主成分的数量",
    "Perplexity": "困惑度",
    "Early exaggeration": "早期夸大",
    "Learning rate": "学习率",
    "Random state": "随机状态",
    "Use the MulticoreTSNE package if possible?": "如果可能的话，请使用MulticoreTSNE软件包。",
    "Effective minimum distance between embedded points": "嵌入点之间的有效最小距离",
    "Effective scale of embedded points": "嵌入点的有效尺度",
    "Number of dimensions of the embedding": "嵌入的维度数量",
    "Number of iterations (epochs) of the optimization": "优化的迭代次数（轮次）",
    "Initial learning rate for the embedding optimization": "嵌入优化的初始学习率",
    "Weighting applied to negative samples in low dimensional embedding optimization": "在低维嵌入优化中应用于负样本的权重",
    "The number of negative edge/1-simplex samples to use per positive edge/1-simplex sample in optimizing the low dimensional embedding": "在优化低维嵌入时，每个正边/1-单纯形样本使用的负边/1-单纯形样本的数量",
    "How to initialize the low dimensional embedding": "如何初始化低维嵌入",
    "Precomputed coordinates for initialization": "预计算坐标用于初始化",
    "Sparse adjacency matrix of the graph": "稀疏邻接矩阵的图",
    "External key": "外部键",
    "Number of diffusion components to use": "使用扩散成分的数量",
    "Number of branchings to detect": "检测分支的数量",
    "Min group size": "最小组大小",
    "Allow Kendal tau shift?": "允许 Kendal tau 偏移？",
    "Key for categorical in the input": "输入中分类键",
    "Use RNA velocity to orient edges in the abstracted graph and estimate transitions?": "使用RNA速率在抽象图中定位边并估计转换？",
    "PAGA connectivity model": "PAGA 连接性模型",
    "The embedding over which the density will be calculated.": "密度计算所依据的嵌入。",
    "Key for categorical observation/cell annotation for which densities are calculated per category.": "用于按类别计算密度的分类观察/单元注释的键。",
    "Name of the .obs covariate that will be added with the density estimates.": "将密度估计添加的 .obs 协变量的名称。",
    "Method used for normalization": "用于标准化的方法",
    "Target sum": "目标和",
    "Exclude (very) highly expressed genes for the computation of the normalization factor (size factor) for each cell": "在计算每个细胞的标准化因子（大小因子）时，排除（非常）高表达的基因。",
    "Name of the field in 'adata.obs' where the normalization factor is stored": "在 'adata.obs' 中存储标准化因子的字段名称",
    "List of layers to normalize": "要进行归一化的层列表",
    "How to normalize layers?": "如何对层进行归一化？",
    "Number of genes to keep": "要保留的基因数量",
    "Mean threshold": "平均阈值",
    "CV threshold": "CV阈值",
    "Number of principal component": "主成分数量",
    "Denoised genes to return": "去噪后的基因返回",
    "Number of nearest neighbors on which to build kernel": "构建核函数时使用的最近邻居数",
    "Set decay rate of kernel tails": "设置核尾部的衰减率",
    "Maximum number of nearest neighbors with nonzero connection": "非零连接的最大最近邻数",
    "Power to which the diffusion operator is powered. This sets the level of diffusion": "扩散算子的幂次。这设置了扩散的水平",
    "Number of principal components to use for calculating neighborhoods": "用于计算邻域的主成分数量",
    "Which solver to use": "使用哪种求解器",
    "Method used for inspecting": "检查使用的方法",
    "Name of kind of values in X": "X中值的种类名称",
    "The kind of thing the variables are": "变量的类型",
    "Keys for boolean columns of '.var' which identify variables you could want to control for": "'.var'文件中布尔列的键，用于识别你可能想要控制的变量",
    "Proportions of top genes to cover": "顶尖基因覆盖比例",
    "The size of local neighborhood (in terms of number of neighboring data points) used for manifold approximation": "局部邻域的大小（就邻近数据点的数量而言）用于流形近似",
    "Indicated representation to use": "指示使用的表示",
    "Use a hard threshold to restrict the number of neighbors to n_neighbors?": "使用硬阈值限制邻居数量为 n_neighbors？",
    "Numpy random seed": "NumPy 随机种子",
    "Method for computing connectivities": "计算连接性的方法是",
    "Distance metric": "距离度量",
    "The list of gene names used for score calculation": "用于分数计算的基因名称列表",
    "Number of reference genes to be sampled": "要采样的参考基因数量",
    "Genes for sampling the reference set": "用于采样参考集的基因",
    "Name of the field to be added in '.obs'": "要添加到 '.obs' 的字段名称",
    "Format for the list of genes associated with S phase": "与S期相关基因的列表格式",
    "List of genes associated with S phase": "与S期相关的基因列表",
    "File with the list of genes associated with S phase": "与S期相关基因的列表文件",
    "Format for the list of genes associated with G2M phase": "与G2M阶段相关基因的列表格式",
    "List of genes associated with G2M phase": "与G2M期相关的基因列表",
    "File with the list of genes associated with G2M phase": "与G2M阶段相关基因的列表文件",
    "Subset of groups to which comparison shall be restricted": "要限制比较的组的子集",
    "Comparison": "比较",
    "Group identifier with respect to which compare": "与比较相关的组标识符",
    "The number of genes that appear in the returned tables": "返回表中出现的基因数量",
    "Method": "方法",
    "Algorithm to use in the optimization problem": "优化问题中使用的算法",
    "Dual (not primal) formulation?": "双重（非原始）公式？",
    "Should a constant (a.k.a. bias or intercept) be added to the decision function?": "是否应该在决策函数中添加一个常数（即偏置或截距）？",
    "Intercept scaling": "截距缩放",
    "Tolerance for stopping criteria": "停止准则的容忍度",
    "Inverse of regularization strength": "正则化强度的倒数",
    "Zero center?": "零中心？",
    "Maximum value": "最大值",
    "Keys for observation annotation on which to regress on": "用于回归分析的观察注释的键",
    "Extra annotated data matrix": "额外注释数据矩阵",
    "The subset of vars to be used when performing MNN correction": "在执行MNN校正时要使用的变量子集",
    "Batch key for the concatenate": "串联的批次键",
    "Separator to join the existing index names with the batch category": "分隔符，用于将现有的索引名称与批次类别连接起来",
    "Batch categories for the concatenate": "批量类别用于连接",
    "Number of mutual nearest neighbors": "互近邻数",
    "The bandwidth of the Gaussian smoothing kernel used to compute the correction vectors": "用于计算校正向量的高斯平滑核的带宽",
    "Should cosine normalization be performed on the input data prior to calculating distances between cells?": "在计算细胞间距离之前，是否应该对输入数据进行余弦归一化处理？",
    "Should cosine normalization be performed prior to computing corrected expression values?": "在计算校正表达值之前，是否应该进行余弦归一化？",
    "Number of dimensions to use for summarizing biological substructure within each batch": "在每个批次中用于总结生物子结构的维度数量",
    "Adjust variance of the correction vectors?": "调整校正向量的方差？",
    "compute the angle between each cell’s correction vector and the biological subspace of the reference batch?": "计算每个细胞的校正向量与参考批次的生物学子空间之间的角度。",
    "The order in which batches are to be corrected": "批次更正的顺序",
    "SVD mode": "奇异值分解模式",
    "Key to a categorical annotation from adata.obs that will be used for batch effect removal": "用于批次效应去除的adata.obs中分类注释的键",
    "Single or paired-end reads?": "单端或双端测序数据？",
    "Unmated reads files": "未配对的读取文件",
    "Type of single-cell protocol": "单细胞协议类型",
    "w1 adapters": "w1 适配器",
    "Transcript to gene map file": "转录本到基因映射文件",
    "Extra output files": "额外输出文件",
    "Whitelist file": "白名单文件",
    "Skip deduplication step": "跳过去重步骤",
    "Mito-RNA genes file": "线粒体RNA基因文件",
    "Ribosomal RNA file": "核糖体RNA文件",
    " Dump cell v transcripts count matrix in MTX format": "导出细胞v转录本计数矩阵为MTX格式",
    "Number of cells": "细胞数量",
    "Upper bound on expected number of cells": "期望细胞数量的上限",
    "Generate mean and variance for cell x gene matrix by boostrap": "通过bootstrap生成细胞×基因矩阵的均值和方差",
    "Minimum allowed score for a mapping": "映射的最低允许分数",
    "Fraction of cellular barcodes to keep": "保留的细胞条形码比例",
    "Maximum allowable limit to process the cell barcodes": "允许处理的细胞条形码的最大限制",
    "Minimum frequency for a barcode to be considered": "条形码被考虑的最小频率",
    "Convert From / To": "转换从/到",
    "MCMC parameters": "MCMC 参数",
    "Customize plots": "自定义图形",
    "Annotated data matrix": "注释数据矩阵",
    "What to inspect?": "要检查什么？",
    "What to inspect in uns?": "在uns中需要检查什么？",
    "Which annotation to inspect for the observations?": "要检查观察结果，应该查看哪种注释？",
    "Which annotation to inspect for the variables?": "要检查变量，应该查看哪种注释？",
    "hd5 format": "hd5格式",
    "Format to write the annotated data matrix": "注释数据矩阵的书写格式",
    "Loom file to be exported to a series of tabular files.": "Loom文件将被导出为一系列表格文件。",
    "Use less samples to get an estimation of the p-value.": "使用较少的样本来估计p值。",
    "hd5 format to be created": "创建 hd5 格式",
    "Format for the annotated data matrix": "注释数据矩阵的格式",
    "Is the data matrix to read sparse?": "数据矩阵是否为稀疏格式？",
    "Cleanup?": "清理？",
    "X_name": "X名称",
    "obs_names": "观察值名称",
    "var_names": "变量名",
    "Does the first column store the row names?": "第一列存储行名吗？",
    "Data matrix": "数据矩阵",
    "Use 10x Genomics formatted mtx": "使用10x Genomics格式的mtx",
    "Genes": "基因",
    "Features": "特征",
    "condensed count matrix from UMI tools": "从UMI工具中得到的压缩计数矩阵",
    "File for main layer of loom file.": "loom文件主层的文件。",
    "Add layers": "添加层",
    "Tsv of column data.": "制表符分隔的列数据。",
    "Tsv of row data.": "行数据的制表符分隔值（TSV）文件。",
    "Function to manipulate the object": "操作对象的函数",
    "Annotated data matrix to add": "注释数据矩阵以添加",
    "Join method": "连接方法",
    "Key to add the batch annotation to obs": "将批量注释添加到obs（观测数据）的关键是",
    "Key for observations or variables annotation": "观测值或变量注释的键",
    "Comma-separated list of new categories": "逗号分隔的新类别列表",
    "What to annotate?": "要注释什么？",
    "Table with new annotations": "带有新注释的表格",
    "What to filter?": "要过滤什么？",
    "Type of filtering?": "过滤类型？",
    "Key to filter": "关键筛选",
    "Type of value to filter": "要筛选的值的类型",
    "Value to keep": "保持的值",
    "Format for the filter by index": "按索引过滤的格式",
    "List of index to keep": "保留的索引列表",
    "File with the list of index to keep": "保留索引列表的文件",
    "Loom file": "Loom 文件",
    "Select data attribute to add to loom": "选择要添加到loom文件的数据属性",
    "Column file of same dimensions as existing file": "与现有文件尺寸相同的列文件",
    "Row file of same dimensions as existing file": "与现有文件具有相同维度的行文件",
    "Layer file(s) of same dimensions as existing file": "与现有文件尺寸相同的层文件",
    "Format for the input matrix": "输入矩阵的格式",
    "Count Data": "计数数据",
    "Genes List": "基因列表",
    "Barcodes List": "条形码列表",
    "Expected Number of Cells": "预期细胞数",
    "Upper Quantile": "上四分位数",
    "Lower Proportion": "较低比例",
    "Lower-bound Threshold": "下限阈值",
    "FDR Threshold": "FDR阈值",
    "Format for output matrices": "输出矩阵的格式",
    "Lower Bound": "下限",
    "Random Seed": "随机种子",
    "Expression matrix in sparse matrix format (.mtx)": "稀疏矩阵格式（.mtx）的表达矩阵",
    "Gene table": "基因表",
    "Barcode/cell table": "条形码/细胞表",
    "Should metadata file be added?": "是否需要添加元数据文件？",
    "Cell ID column": "细胞ID列",
    "SingleCellExperiment rdata object": "单细胞实验 R 数据对象",
    "UMI count lower bound": "UMI计数下限",
    "Number of iterations": "迭代次数",
    "FDR filter for removal of barcodes with no cells": "FDR过滤器用于移除没有细胞的条形码",
    "Should results be returned for barcodes with totals less than or equal to the value of 'lower'?": "是否应该返回条形码总数小于或等于'lower'值的结果？",
    "Ignore barcodes threshold": "忽略条形码阈值",
    "Retain barcodes threshold": "保留条形码阈值",
    "Should barcodes estimated to have no cells be removed from the output object?": "是否应该从输出对象中移除那些估计没有细胞的条形码？",
    "Input SingleCellLoomExperiment dataset": "输入 SingleCellLoomExperiment 数据集",
    "Feature (from annotation file) to colour t-SNE plot points by": "特征（来自注释文件）用于按颜色对 t-SNE 绘图点进行分类",
    "Feature (from annotation file) to size t-SNE plot points by": "特征（来自注释文件）按大小对 t-SNE 绘图点进行缩放",
    "Feature (from annotation file) to shape t-SNE plot points by": "特征（来自注释文件）用于按特征形状 t-SNE 绘图点",
    "Feature (from annotation file) to colour PCA plot points by": "特征（来自注释文件）用于按颜色对PCA绘图点进行分类",
    "Feature (from annotation file) to size PCA plot points by": "特征（来自注释文件）按大小对PCA绘图点进行缩放",
    "Feature (from annotation file) to shape PCA plot points by": "特征（来自注释文件）用于通过形状对PCA绘图点进行分类",
    "Plot on log scale": "在对数刻度上绘图",
    "Input SingleCellLoomExperiment dataset to normalize": "输入SingleCellLoomExperiment数据集以进行标准化",
    "Type of filter": "过滤类型",
    "Minimum library size (mapped reads) to filter cells on": "最小库大小（映射读数）以过滤细胞",
    "Maximum % of mitochondrial genes expressed per cell": "每个细胞表达的线粒体基因的最大百分比",
    "Remove cells that have less than the given number of expressed features": "删除表达特征数少于给定数量的细胞",
    "Number of reads mapped to a feature above which it to be deemed as expressed": "映射到一个特征的读段数量，超过这个数量就被认为是表达的",
    "Remove features that occur in less than the given number of cells": "移除出现在少于给定数量的细胞中的特征",
    "Expression matrix in tabular format": "表格格式的表达矩阵",
    "Format dataset describing the features in tabular format": "格式化数据集，以表格形式描述特征",
    "Dataset containing the list of the mitochondrial control genes": "包含线粒体控制基因列表的数据集",
    "Dataset containing the list of the ERCC controls": "包含ERCC对照列表的数据集",
    "Dataset containing the list of the cell controls": "包含细胞对照列表的数据集",
    "Input RaceID RDS": "输入 RaceID RDS",
    "Plot Clusters?": "绘制聚类图？",
    "Cluster Numbers to plot": "聚类编号以绘图",
    "Perform Subset Analysis?": "执行子集分析？",
    "Types Regex (removal)": "类型正则表达式（移除）",
    "Subset of Types: Regex (select)": "子类型：正则表达式（选择）",
    "Output FR map instead of tSNE": "输出FR图而不是t-SNE图",
    "Examine Genes of Interest": "检查感兴趣的基因",
    "Genes to Examine": "要检查的基因",
    "Cells to Examine": "细胞检查",
    "Impute Genes": "填充基因",
    "Title of plot": "图表标题",
    "Cluster Threshold": "聚类阈值",
    "Cluster List": "聚类列表",
    "Order Cells": "排序细胞",
    "Average Expression": "平均表达量",
    "Normalise Gene Expression": "标准化基因表达",
    "Lower-bound for Gene Expression": "基因表达的下限",
    "Upper-bound for Gene Expression": "基因表达的上限",
    "Regex to Select Sample Names For Each cell": "正则表达式用于选择每个单元格的样本名称",
    "Cluster columns": "聚类列",
    "Cluster rows": "聚类行",
    "Order Clusters by Heirarchal Clustering of Cluster Medoids.": "根据聚类中心的层次聚类对聚类进行排序。",
    "Differential Gene Testing": "差异基因测试",
    "P-value cutoff": "P值截止",
    "Apply B-H Correction": "应用贝努利-哈贝尔校正",
    "Log2-Fold Threshold": "对数2倍数阈值",
    "Log2 Mean Threshold": "对数2均值阈值",
    "Display Gene Names": "显示基因名称",
    "Min Transcripts": "最小转录本数",
    "Min Expression": "最小表达量",
    "Min Cells": "最小细胞数",
    "Count filtered features greater than or equal to 1": "计算过滤后大于或等于1的特征数量",
    "K-nearest-neighbours": "K最近邻",
    "CGenes": "C基因",
    "FGenes": "F基因",
    "Batch Regex": "批量正则表达式",
    "CCor": "CCor",
    "Batch Mode": "批量模式",
    "Perform Cell-cycle correction?": "执行细胞周期校正？",
    "List of Gene Sets": "基因集列表",
    "P-value Cutoff": "P值截止",
    "Quantification Fraction": "定量分数",
    "Number of components to use": "要使用的组件数量",
    "Derive Components from saturation criterion": "从饱和标准派生组件",
    "Type of Component Analysis": "组件分析类型",
    "Log-transform data prior to PCA or ICA": "在进行PCA（主成分分析）或ICA（独立成分分析）之前对数据进行对数变换。",
    "Clustering": "聚类",
    "Distance Metric": "距离度量",
    "Perform feature selection": "执行特征选择",
    "KNN": "KNN（K-最近邻）",
    "Saturation-based clustering?": "基于饱和度的聚类？",
    "Max number of clusters using Saturation-by-mean": "最大聚类数使用平均饱和度",
    "Sample random number of cells": "随机样本数量的细胞",
    "Number of booststrapping runs": "bootstrapping 运行次数",
    "Outliers": "异常值",
    "Minimum Transcripts": "最小转录本数",
    "Minimum Genes": "最小基因集",
    "Plot Final Clusters?": "绘制最终聚类？",
    "Outlier Probability Threshold": "异常值概率阈值",
    "Outlier Distance Quantile": "异常值距离分位数",
    "Number of trees to be built": "要构建的树的数量",
    "Tree Factor": "树因子",
    "tSNE, FR and UMAP": "tSNE、FR 和 UMAP",
    "N Neighbours": "N 邻居",
    "tSNE map initialised by classical multidimensional scaling": "tSNE 地图由经典多维尺度初始化",
    "Random Seed (tSNE)": "随机种子（tSNE）",
    "Random Seed (FR)": "随机种子（FR）",
    "Number Epochs (UMAP)": "UMAP 迭代次数",
    "Min Dist (UMAP)": "最小距离（UMAP）",
    "Extra Parameters": "额外参数",
    "Table Limit": "表格限制",
    "Plot Limit": "绘图限制",
    "Fold change": "倍数变化",
    "Input RDS": "输入 RDS",
    "Compute transcriptome entropy of each cell": "计算每个细胞的转录组熵。",
    "Neighbour Mode": "邻域模式",
    "F-R layout?": "F-R 布局？",
    "Compute Cell Projections for Randomized Background Distribution": "计算随机背景分布的细胞投影",
    "Shuffle Projections": "洗牌投影",
    "Use a fast approximate background model to infer links": "使用快速近似背景模型来推断链接",
    "Computing P-values for Link Significance": "计算链接显著性的P值",
    "Sensitivity": "灵敏度",
    "StemID2 Lineage Graph": "StemID2 谱系图",
    "Show Cells": "显示单元格",
    "Score Threshold": "得分阈值",
    "Transparency": "透明度",
    "Compute StemID2 score": "计算StemID2分数",
    "Number of Neighbours": "邻居数量",
    "Output Log?": "输出日志？",
    "StemID Branch Link Examine": "StemID 分支链接 检查",
    "Perform StemID?": "执行 StemID？",
    "Cluster Number": "簇数量",
    "Trajectory Path i, j, k": "轨迹路径 i, j, k",
    "Plot Z-score Transformed Projections": "绘制Z分数转换后的投影",
    "Number of DE Genes": "差异表达基因数量",
    "FateID Branch Link Examine": "FateID 分支链接检查",
    "Perform FateID?": "执行FateID？",
    "Cells from Clusters": "细胞来自聚类",
    "Min Number of Cells": "最小细胞数",
    "SOM Nodes": "SOM节点",
    "Smoothing parameter": "平滑参数",
    "Correlation threshold": "相关性阈值",
    "Min SOM": "最小自组织映射（SOM）",
    "Partitioning along the x-axis": "沿x轴划分",
    "Partitioning along the y-axis": "沿 y 轴划分",
    "Average position is given for each partition along the x-axis": "每个分区的平均位置在x轴上给出",
    "Perform Additional FateID Analysis with Self-Organised Map?": "使用自组织图进行额外的FateID分析？",
    "Genes to Inspect": "基因检查",
    "Gene List": "基因列表",
    "Types Regex (select)": "类型正则表达式（选择）",
    "scRNA Dataset": "单细胞RNA数据集",
    "Bulk RNA Dataset": "批量RNA数据集",
    "Purpose": "目的",
    "Methods to use": "使用方法",
    "Cell Types Label from scRNA dataset": "scRNA数据集中的细胞类型标签",
    "Samples Identifier from scRNA dataset": "单细胞RNA数据集中的样本标识符",
    "Phenotype factors": "表型因素",
    "Excluded phenotype factors": "排除的表型因素",
    "Show proportions of a disease factor?": "展示疾病因素的比例？",
    "scRNA Phenotype Cell Target": "单细胞RNA表型细胞靶标",
    "Bulk Phenotype Target": "批量表型目标",
    "Bulk Phenotype Target Threshold": "批量表型目标阈值",
    "scRNA Sample Disease Group": "单细胞RNA样本疾病组",
    "scRNA Sample Disease Group (Scale)": "单细胞RNA样本疾病组（规模）",
    "Scale all Y-axes to max limit": "将所有 Y 轴缩放到最大限制",
    "Cluster Types Label from scRNA dataset": "单细胞RNA数据集中的聚类类型标签",
    "Cluster Groups": "聚类组",
    "Cluster ID": "聚类ID",
    "Marker Gene Group Name": "标记基因组名称",
    "List of Gene Markers": "基因标记列表",
    "Assay Data": "实验数据",
    "Phenotype Data": "表型数据",
    "Meta Data": "元数据",
    "Label Description": "标签描述",
    "Experiment Data": "实验数据",
    "Experimenter Name": "实验者姓名",
    "Lab": "实验室",
    "Contact": "联系",
    "Abstract": "摘要",
    "Other Fields": "其他字段",
    "FieldName": "字段名",
    "ESet Dataset": "ESet 数据集",
    "Inspect": "检查",
    "New scRNA Group": "新单细胞RNA组",
    "Name of scRNA Dataset": "单细胞RNA数据集的名称",
    "Advanced scRNA Parameters": "高级单细胞RNA参数",
    "Bulk Datasets in scRNA Group": "单细胞RNA组中的批量数据集",
    "Name of Bulk Dataset": "批量数据集名称",
    "Advanced Bulk Parameters": "高级批量参数",
    "Method to use": "使用的方法",
    "Cluster heatmaps?": "聚类热图？",
    "Filter Summary Plots": "过滤摘要图",
    "Show only these cell types (blank for all)": "仅显示这些细胞类型（留空则显示全部）",
    "Show only these factors (blank for all)": "只显示这些因素（留空显示全部）",
    "FASTQ file containing cell barcode sequences": "包含细胞条形码序列的FASTQ文件",
    "Single or Paired-end data": "单端或配对端数据",
    "Forward reads FASTQ file": "正向读取 FASTQ 文件",
    "Reverse reads FASTQ file": "反向读取 FASTQ 文件",
    "Select FASTQ file": "选择 FASTQ 文件",
    "Number of bases to extract from barcode-containing FASTQ": "从包含条形码的FASTQ文件中提取的碱基数量",
    "Minimum MAPQ required to retain fragment": "最小MAPQ要求以保留片段",
    "Read tag storing cell barcode information": "读取存储细胞条形码信息的标签",
    "Regular expression used to extract cell barcode from read name": "用于从读取名称中提取细胞条形码的正则表达式",
    "Maximum distance between integration sites for the fragment to be retained": "片段保留所需整合位点间的最大距离",
    "Minimum distance between integration sites for the fragment to be retained": "片段保留的整合位点之间的最小距离",
    " Number of bases to shift Tn5 insertion position by on the forward strand": "Tn5插入位点在正链上移动的碱基数量",
    "Number of bases to shift Tn5 insertion position by on the reverse strand": "在反向链上通过Tn5插入位点偏移的碱基数量",
    "Take cell barcode into account when collapsing duplicate fragments": "在折叠重复片段时考虑细胞条形码",
    "ATAC fragments file": "ATAC片段文件",
    "Features file": "特征文件",
    "Normalize peak sizes?": "标准化峰值大小？",
    "Number of bases to extend both sides from the center of the peak": "峰值中心两侧延伸的碱基数量",
    "Select the chromosomes of the species you are considering": "选择您所考虑的物种的染色体",
    "Enter comma seperated list of chromosome ids (without chr prefix)": "输入以逗号分隔的染色体ID列表（不带chr前缀）",
    "Minimum counts": "最小计数",
    "Maximum counts": "最大计数",
    "Minimum  features": "最小特征数",
    "Maximum features": "最大特征数",
    "Min score": "最小分数",
    "Number of features": "特征数量",
    "Minimum score value": "最小分数值",
    "Log": "日志",
    "Key added": "密钥已添加",
    "Upstream": "上游",
    "Raw?": "原始数据？",
    "Method used for Clustering or Embedding": "用于聚类或嵌入的方法",
    "Compute PCA coordinates before the neighborhood graph": "在邻域图之前计算PCA坐标",
    "SVD solver to use": "奇异值分解（SVD）求解器的使用",
    "Number of principal component computed for PCA (and therefore neighbors, tsne and umap)": "用于PCA（以及因此的邻居、t-SNE和UMAP）计算的主成分数量",
    "Size of the local neighborhood (number of neighboring data points) used for manifold approximation": "局部邻域的大小（用于流形近似的邻近数据点的数量）",
    "Perplexity (number of nearest neighbors used in other manifold learning algorithms)": "困惑度（在其他流形学习算法中使用的最近邻数目）",
    "Kernel to use for computing connectivities": "用于计算连接性的核",
    "Metric that returns a distance": "返回距离的度量",
    "The effective minimum distance between embedded points": "嵌入点之间的有效最小距离",
    "The effective scale of embedded points": "嵌入点的有效尺度",
    "Use highly variable genes only": "仅使用高度变异基因",
    "The number of dimensions of the UMAP embedding": "UMAP 嵌入的维度数",
    "Use raw attribute of Anndata if present": "如果存在，使用Anndata的raw属性",
    "Subset of groups, to which comparison shall be restricted": "要限制比较的组的子集",
    "Compare each group with respect to this group": "将每个组与这个组进行比较",
    "The number of features that appear in the returned tables": "返回表中出现的特征数量",
    "Rank genes by the absolute value of the score, not by the score": "根据分数的绝对值对基因进行排序，而不是根据分数本身",
    "The key in adata.uns information is saved to": "adata.uns 中的信息键保存到",
    "p-value correction method": "P值校正方法",
    "Clustering method to use": "使用的聚类方法",
    "Range minimum": "范围最小值",
    "Range maximum": "范围最大值",
    "Maximum number of steps": "最大步数",
    "Variable name in obs": "观察值中的变量名",
    "Pipeline": "流程",
    "How to set sample name": "如何设置样本名称",
    "sample name": "样本名称",
    "BAM file including CB tag": "包含CB标签的BAM文件",
    "List of valid cell barcodes": "有效细胞条形码列表",
    "Metadata Table": "元数据表",
    "BAM files": "BAM文件",
    "One file per cell?": "每个细胞一个文件？",
    "Without UMI?": "没有UMI（Unique Molecular Identifier，独特的分子标识符）？",
    "UMI extension": "UMI扩展",
    "Bam file with sorted reads obtained from DropEst": "从DropEst获得的排序好的Bam文件",
    "R dump RDS file generated from DropEst": "R 转储 RDS 文件，该文件由 DropEst 生成",
    "verbosity level": "详细程度级别",
    "The name of the column in adata.obs that differentiates among experiments/batches.": "在adata.obs中用于区分实验/批次的列名。",
    "The name of the field in adata.obsm where the PCA table is stored. Defaults to 'X_pca', which is the default for sc.tl.pca().": "在adata.obsm中存储PCA表格的字段名称。默认为'X_pca'，这是sc.tl.pca()的默认值。",
    "he name of the field in adata.obsm where the adjusted PCA table will be stored after running this function.": "在运行此函数后，调整后的PCA表格将存储在adata.obsm中的字段名称。",
    "Use programme defaults": "使用程序默认设置",
    "Diversity clustering penalty parameter": "多样性聚类惩罚参数",
    "Ridge regression penalty parameter": "岭回归惩罚参数",
    "Number of clusters in model.": "模型中的聚类数量。",
    "Protection against overclustering small datasets with large ones.": "防止小数据集与大数据集过度聚类。",
    "What proportion of cells to update during clustering": "在聚类过程中更新细胞的比例",
    "Maximum number of rounds to run clustering at each round of Harmony": "在每个Harmony聚类轮次中运行的最大轮数",
    "Maximum number of rounds to run Harmony. One round of Harmony involves one clustering and one correction step.": "运行Harmony的最大轮数。一轮Harmony包括一次聚类和一次校正步骤。",
    "Convergence tolerance for clustering round of Harmony": "Harmony聚类轮次的收敛容差",
    "Convergence tolerance for Harmony.": "Harmony的收敛容差。",
    "Seed for random number generator.": "随机数生成器的种子。",
    "Cell metadata table": "单元格元数据表",
    "Gene metadata table": "基因元数据表",
    "Attribute used as annotation index": "用作注释索引的属性",
    "Output markers table in tsv format": "输出标记物表格为tsv格式",
    "Number of top genes to show per group/cluster": "每组/簇要显示的顶级基因数量",
    "The sample grouping/clustering to use.": "样本分组/聚类使用的方法。",
    "File with groupby. Overrides the groupby setting": "具有 groupby 的文件。覆盖 groupby 设置",
    "The key in adata.uns information is saved to. If included, the keyword 'GROUPBY' will be substituted with the value of the groupby setting.": "adata.uns信息中的关键字被保存到。如果包含，关键字'GROUPBY'将被替换为groupby设置的值。",
    "Method for testing differentially expressed genes": "差异表达基因检测方法",
    "Use raw attribute if present": "如果存在，请使用原始属性",
    "Rank by absolute value of the scores instead of the scores": "按分数的绝对值进行排序，而不是按分数排序",
    "Subset of groups/clusters to which comparisons should be restricted": "要限制比较的组/簇的子集",
    "If 'rest', compare to the union of the rest of the group/cluster. If a group identifier, compare to that group": "如果为“rest”，则与该组/群集的其余部分的并集进行比较。如果是一个组标识符，则与该组进行比较",
    "Use filtering defaults": "使用过滤默认值",
    "Minimum in-group fraction": "最小内群分数",
    "Maximum out-group fraction": "最大外群分数",
    "Layer to use for marker genes computation": "用于标记基因计算的层",
    "Compute the fraction of cells expressing the genes?": "计算表达基因的细胞比例",
    "Use tie correction for 'wilcoxon' scores. Used only for 'wilcoxon'.": "使用 'wilcoxon' 分数的系结校正。仅用于 'wilcoxon'。",
    "Output cluster in two column text format": "输出聚类结果为两列文本格式",
    "Clustering algorithm": "聚类算法",
    "Name of the slot that holds the KNN graph": "KNN 图的插槽名称",
    "Key from adata.layers whose value will be used to perform tests on. (Default: use .X)": "在adata.layers中用于执行测试的键的值。（默认：使用.X）",
    "Additional suffix to the name of the slot to save the calculated clustering": "为保存计算出的聚类结果的槽位名称添加额外后缀",
    "Resolution, high value for more and smaller clusters": "分辨率，更高的值会生成更多更小的聚类",
    "File with resolution, use with parameter iterator. Overrides the resolution setting": "文件包含分辨率，与迭代器参数一起使用。覆盖分辨率设置",
    "Restrict clustering to certain sample categories": "限制聚类到特定的样本类别",
    "Use weights from knn graph": "使用来自knn图的权重",
    "Seed for random number generator": "随机数生成器的种子",
    "Interpret the adjacency matrix as directed graph.": "将邻接矩阵解释为有向图。",
    "Output embeddings in tsv format": "输出嵌入到tsv格式",
    "Additional suffix to the name of the slot to save the embedding": "为保存嵌入的槽位名称添加额外的后缀",
    "The number of dimensions of the embedding": "嵌入的维度数量",
    "The effective spread of embedded points": "嵌入点的有效传播",
    "Initial learning rate": "初始学习率",
    "Weighting applied to negative samples": "对负样本应用的权重",
    "The ratio of negative to positive edge in optimisation": "在优化中，负边缘与正边缘的比例",
    "Method to initialise embedding, any key for adata.obsm or choose from the preset methods": "初始化嵌入的方法，可以是adata.obsm的任何键，也可以选择预设方法之一",
    "Number of iterations of optimisation": "优化迭代次数",
    "Seed for numpy random number generator": "numpy 随机数生成器的种子",
    "Use the indicated representation": "使用指定的表示",
    "The perplexity is related to the number of nearest neighbours, select a value between 5 and 50": "困惑度与最近邻的数量有关，选择一个介于5到50之间的值",
    "The perplexity is related to the number of nearest neighbours": "困惑度与最近邻的数量有关",
    "Controls the tightness within and between clusters": "控制簇内以及簇间的紧密程度",
    "Learning rate, should be between 100 and 1000": "学习率，应该在100到1000之间",
    "Use multicoreTSNE": "使用multicoreTSNE",
    "The number of jobs": "作业数",
    "The number of PCs to use": "使用的主成分个数",
    "Maximum number of neighbors used": "最大邻居数",
    "File with n_neighbours, use with parameter iterator. Overrides the n_neighbors setting": "文件包含 n_neighbours，与参数 iterator 一起使用。覆盖 n_neighbors 设置",
    "Use hard threshold to restrict neighbourhood size (otherwise use a Gaussian kernel to down weight distant neighbors)": "使用硬阈值来限制邻域大小（否则使用高斯核来降低远处邻居的权重）",
    "Method for calculating connectivity": "计算连通性的方法",
    "A known metric’s name": "一个已知指标的名称",
    "Number of PCs to produce": "生成的主成分个数",
    "Perform incremental PCA by chunks": "执行分块增量PCA",
    "Chunk size": "块大小",
    "Zero center data before scaling": "在缩放前将数据零中心化",
    "SVD solver": "奇异值分解求解器",
    "random seed for numpy random number generator": "numpy 随机数生成器的随机种子",
    "Export extra output": "导出额外输出",
    "Flavor of computing normalised dispersion": "计算规范化离散度的风味",
    "Min value for normalised mean expression (in log1p scale), ignored if flavor='seurat_v3'": "标准化后的平均表达值的最小值（以log1p尺度表示），如果flavor='seurat_v3'则忽略",
    "Max value for normalised mean expresssion (in log1p scale), Ignored if flavor='seurat_v3'": "在log1p尺度下，标准化均表达量的最大值，如果flavor='seurat_v3'则忽略。",
    "Min value for dispersion of expression, ignored if flavor='seurat_v3'": "表达分散的最小值，如果风味为'seurat_v3'则忽略",
    "Max value for dispersion of expresssion, ignored if flavor='seurat_v3'": "表达分散的最大值，如果风味='seurat_v3'则忽略",
    "Number of top variable genes to keep, mandatory if flavor='seurat_v3'": "要保留的顶部变量基因的数量，如果风味='seurat_v3'则是强制性的",
    "The fraction of the data (cells) used when estimating the variance in the loess model fit if flavor='seurat_v3'": "在使用loess模型拟合时，如果flavor='seurat_v3'，用于估计方差的数据处理（单元格）的比例",
    "Number of bins for binning the mean expression": "平均表达量分箱的数量",
    "Remove genes not marked as highly variable": "删除未标记为高变异性的基因",
    "Batch key": "批量密钥",
    "Parameters to select genes to keep": "选择保留基因的参数",
    "Name of parameter to filter on": "参数名称以进行过滤",
    "Min value": "最小值",
    "Max value": "最大值",
    "Categories to select genes to keep (unless negate is checked)": "选择保留基因的类别（除非选中了“取反”）",
    "Name of the categorical variable to filter on": "要过滤的分类变量名称",
    "Comma-separated list of categories": "逗号分隔的类别列表",
    "Apply as negative filter": "应用作为负向过滤器",
    "Subsets to select genes to keep": "选择要保留的基因的子集",
    "List of values to keep": "要保留的值列表",
    "Force recalculation of QC vars": "强制重新计算质量控制变量",
    "Name of the column in `anndata.var` that contains gene name": "`anndata.var`中包含基因名称的列名",
    "Parameters to select cells to keep": "选择保留的细胞的参数",
    "Categories to select cells to keep (unless negate is checked)": "选择要保留的细胞类别（除非勾选了否定）",
    "Subsets to select cells to keep": "选择子集以保留细胞",
    "Input object in hdf5 AnnData format": "输入对象为hdf5格式的AnnData格式",
    "Swap layer to X": "交换层到X",
    "Name of layer to swap to X": "要交换到的层的名称为 X",
    "Name of the new slot for X within layers": "X 在层内的新槽位名称",
    "Merge additional cell metadata": "合并额外的单元格元数据",
    "Cell metadata with headers": "带有标题的单元格元数据",
    "Copy AnnData to .raw": "复制 AnnData 到 .raw",
    "Change field names in AnnData observations": "在AnnData的观测数据中更改字段名称",
    "Original name": "原始名称",
    "New name": "新名称",
    "Keep original": "保持原样",
    "Make values in the field unique": "使字段中的值唯一",
    "Change field names in AnnData var": "在AnnData的var字段中更改字段名称",
    "Gene symbols field in AnnData": "在AnnData中的基因符号字段",
    "Flag genes that start with these names": "标记以这些名称开头的基因",
    "Starts with": "以...开始",
    "Var name": "变量名",
    "Field in var or obs to make unique": "在 var 或 obs 中的字段以使其唯一",
    "Copy adata.X to adata.raw": "将adata.X复制到adata.raw",
    "AnnData object .X with to copy to .raw": "AnnData 对象的 .X 需要复制到 .raw",
    "Copy .X to layers": "复制 .X 到层",
    "Source objects for .X and paired destination layers in the main AnnData object": "在主AnnData对象中，用于.X和配对目标层的源对象",
    "AnnData objects with .X to copy": "AnnData 对象，复制 .X",
    "Destination layer": "目标层",
    "Copy layers": "复制层",
    "Layers from which matrices will be copied": "要复制矩阵的层",
    "Key contains": "键包含",
    "AnnData objects with layers to copy": "带有要复制层的AnnData对象",
    "Copy observations (such as clusters)": "复制观察（例如簇）",
    "Keys from obs to copy": "从obs复制的密钥",
    "AnnData objects with obs to copy": "AnnData 对象，具有要复制的 obs",
    "Copy embeddings (such as UMAP, tSNE)": "复制嵌入（例如 UMAP、tSNE）",
    "Keys from embeddings to copy": "从嵌入中复制的键",
    "AnnData objects with embeddings to copy": "带有嵌入的AnnData对象以复制",
    "Copy uns": "复制未完成",
    "Keys from uns to copy": "来自uns的键复制",
    "AnnData objects with uns to copy": "AnnData 对象带有 uns 属性以复制",
    "Sanitise any null raw.varm objects if any": "如果存在任何空的 raw.varm 对象，请对其进行清理。",
    "Split on obs": "在观察值上拆分",
    "Obs key to split on": "观察键，用于分割",
    "Output main AnnData object": "输出主要的AnnData对象",
    "name of the embedding to plot": "要绘制的嵌入的名称",
    "color by attributes, comma separated texts": "按属性着色，用逗号分隔的文本",
    "Key for categorical in `.obs`": "在 `.obs` 中的分类键",
    "Field for gene symbols": "基因符号字段",
    "Use raw attributes": "使用原始属性",
    "Name of the AnnData object layer to plot": "要绘制的AnnData对象层的名称",
    "Where to look for neighbors connectivities.": "在哪里查找邻居连接性。",
    "Location of legend": "图例位置",
    "Legend font size": "图例字体大小",
    "Point size": "点大小",
    "Show edges?": "显示边缘？",
    "Width of edges": "边的宽度",
    "Color of edges": "边的颜色",
    "Plot in order?": "按顺序绘制？",
    "Components to plot": "组件以绘制",
    "Plot projection": "绘图投影",
    "Truncate to this value after scaling": "截断到这个值，缩放后",
    "Target number to normalise to": "目标数量以进行标准化",
    "Apply log transform?": "应用对数变换？",
    "Exclude highly expressed genes?": "排除高表达基因？",
    "Consider cells as highly expressed that have more counts than max_fraction of the original total counts in at least one cell.": "考虑那些在至少一个细胞中计数超过原始总计数最大分数的细胞为高度表达。",
    "Comma-separated list of layers to normalize. Set to 'all' to normalize all layers.": "用逗号分隔的需要进行标准化处理的层列表。设置为'all'以对所有层进行标准化处理。",
    "How to normalise layers": "如何对层进行归一化",
    "Name of the field in adata.obs where the normalization factor is stored. Default: don't store.": "在adata.obs中存储归一化因子的字段名称。默认：不存储。",
    "Image containing the all marker(s)": "包含所有标记的图像",
    "The numerical indices of the channel(s) for the nuclear markers": "核标记物的通道（或通道）的数值索引",
    "Resolution of the image in microns-per-pixel": "图像分辨率，以微米每像素表示",
    "Whether to np.squeeze the outputs before saving": "在保存之前是否对输出进行 np.squeeze 处理",
    "Compartment for segmentation prediction:": "用于分割预测的区域：",
    "Raw Cycle Images": "原始循环图像",
    "Raw Images": "原始图像",
    "Deep Field Profile Images": "深度场剖面图像",
    "Flat Field Profile Images": "平坦场校正剖面图像",
    "Flip X-axis": "翻转X轴",
    "Flip Y-axis": "翻转 Y 轴",
    "Maximum allowed per-tile corrective shift": "最大允许每瓦片校正偏移",
    "Upgrade to BF6-Compliant OME-TIFF Pyramid": "升级至BF6兼容的OME-TIFF金字塔",
    "Markers File (optional)": "标记文件（可选）",
    "Align Channel Number": "对齐通道编号",
    "Sigma": "西格玛",
    "Cyto Mask Channel": "细胞掩膜通道",
    "Flip output image horizontally": "水平翻转输出图像",
    "Flip output image vertically": "垂直翻转输出图像",
    "Write output as a single pyramidal TIFF": "将输出写为单个金字塔TIFF文件",
    "Registered TIFF": "注册的TIFF",
    "Down Sample Factor": "下采样因子",
    "Channel": "通道",
    "Buffer": "缓冲区",
    "Tissue": "组织",
    "UnMicst Tool": "UnMicst 工具",
    "Mean (-1 for model default)": "平均值（模型默认为-1）",
    "Standard Deviation (-1 for model default)": "标准差（-1 表示使用模型默认值）",
    "Channel to perform inference on": "在通道上执行推断",
    "Stack probability map outputs": "堆叠概率图输出",
    "Factor to scale by": "缩放因子",
    "Map percentile intensity to max when rescaling intensity values": "将重新缩放强度值时的百分位数强度映射到最大值",
    "Image File": "图像文件",
    "Contours Class Probabilities": "轮廓类概率",
    "Nuclei Class Probabilities": "细胞核类别概率",
    "Stack Probabilities": "堆叠概率",
    "Probability Maps Channel": "概率映射通道",
    "Crop Strategy": "裁剪策略",
    "TMA Mask File": "TMA掩膜文件",
    "Cyto Method": "细胞方法",
    "Nuclei Filter": "核筛选",
    "Nuclei Region": "核区域",
    "Pixel Threshold": "像素阈值",
    "Pixel Mask Channel": "像素掩码通道",
    "Segment Cytoplasm": "分段细胞质",
    "Save Mask": "保存掩码",
    "Save Figure": "保存图形",
    "Cyto Dilation": "细胞扩张",
    "logSigma": "对数σ",
    "Detect Puncta": "检测斑点",
    "Puncta Sigma": "斑点西格玛",
    "Puncta SD": "斑点标准差",
    "Primary Mask": "主掩码",
    "Additional Mask": "额外掩码",
    "Marker Channels": "标记通道",
    "Mask Metrics": "掩蔽指标",
    "Intensity Metrics": "强度指标",
    "Alpha": "阿尔法",
    "Sorted BAM file": "排序后的BAM文件",
    "Use a BED file of regions for inclusion": "使用一个包含区域的BED文件",
    "BED file of regions for inclusion": "包含区域的BED文件",
    "Use the strand column": "使用“链”列",
    "What do you want to do?": "你想要做什么？",
    "Provide a bigWig file containing mappability data": "提供一个包含可映射性数据的大波形文件。",
    "A bigWig file containing mappability data for filtering reads.": "一个包含用于过滤读取的可映射性数据的大Wig文件。",
    "Mappability threshold": "可映射性阈值",
    "Minimum mappable bases": "最小可映射碱基",
    "Merge per-Cytosine metrics": "合并每个胞嘧啶的度量",
    "Original top strand bounds (comma-separated, no spaces)": "原始顶链边界（逗号分隔，无空格）",
    "Original bottom strand bounds (comma-separated, no spaces)": "原始底部链边界（逗号分隔，无空格）",
    "Complementary to the original bottom strand bounds (comma-separated, no spaces)": "互补于原始的底部链界限（用逗号分隔，无空格）",
    "Output tab separated mbias metrics": "输出制表符分隔的 mbias 指标",
    "Region string in which to extract methylation (e.g. chr1:1-100)": "提取甲基化信息的区域字符串（例如：chr1:1-100）",
    "Keep duplicated alignments": "保留重复的比对",
    "Keep singletons": "保持单例",
    "Keep discordant alignmetns": "保留不一致的比对",
    "Minimum MAPQ threshold": "最小MAPQ阈值",
    "Minimum Phred threshold": "最小Phred阈值",
    "Minimum per-base depth": "最小每碱基深度",
    "Ignore certain alignments": "忽略某些比对",
    "Require alignment flags": "需要对齐标志",
    "Minimum depth for variant avoidance": "变异避免的最小深度",
    "Maximum tolerated variant fraction": "最大耐受变异分数",
    "Original top strand absolute bounds (comma-separated, no spaces)": "原始顶链绝对边界（逗号分隔，无空格）",
    "Original bottom strand absolute bounds (comma-separated, no spaces)": "原始底部链绝对界限（逗号分隔，无空格）",
    "Complementary to the original bottom strand absolute bounds (comma-separated, no spaces)": "相对于原始底部链的绝对界限（用逗号分隔，无空格）",
    "Context methylation metrics options": "上下文甲基化指标选项",
    "bedGraph generated MACS": "生成的MACS bedGraph",
    "Cutoff for peaks": "峰值截止",
    "Minimum length of peak": "峰值的最小长度",
    "Maximum gap between significant peaks": "显著峰值之间的最大间隔",
    "Maximum linking between significant peaks": "最大连接显著峰值",
    "Treatment bedGraph file": "处理 bedGraph 文件",
    "Control bedGraph file": "控制 bedGraph 文件",
    "Method to use while calculating a score in any bin by comparing treatment value and control value": "用于通过比较处理值和对照值来计算任何箱中得分的方法",
    "Set pseudocount": "设置伪计数",
    "BedGraph for Treatment experiment 1": "BedGraph 对于处理实验1",
    "BedGraph for Treatment experiment 2": "处理实验2的BedGraph",
    "BedGraph for Control experiment 1": "控制实验1的BedGraph",
    "Bedgraph for Control experiment 2": "控制实验2的Bedgraph",
    "Cutoff of log10 likelihood ratio cutoff": "对数似然比截断值",
    "Minimum length of differential region": "差异区域的最小长度",
    "Sequencing depth": "测序深度",
    "Sequence depth of condition 1 in million reads": "条件1的序列深度在百万个读段中",
    "Sequence depth of condition 2 in million reads": "条件2的序列深度（以百万个读段计）",
    "Maximum gap to merge nearby differential regions": "合并附近差异区域的最大间隔",
    "MACS score in bedGraph": "MACS 分数在 bedGraph 中",
    "Maximum gap between significant points in a peak": "峰值中显著点之间的最大间隔",
    "Perform cutoff analysis": "执行截断分析",
    "Include trackline into bedGraph output": "在bedGraph输出中包含trackline",
    "Are you pooling Treatment Files?": "您是在合并处理文件吗？",
    "ChIP-Seq Treatment File": "ChIP-Seq 处理文件",
    "Do you have a Control File?": "你有控制文件吗？",
    "Are you pooling Control Files?": "您正在汇总对照文件吗？",
    "ChIP-Seq Control File": "ChIP-Seq 控制文件",
    "Format of Input Files": "输入文件的格式",
    "Build Model": "构建模型",
    "Set lower mfold bound": "设置较低的mfold界限",
    "Set upper mfold bound": "设置mfold的上限",
    "Band width for picking regions to compute fragment size": "选择区域以计算片段大小的带宽",
    "Set extension size": "设置扩展大小",
    "Set shift size": "设置位移大小",
    "Peak detection based on": "基于的峰值检测",
    "p-value cutoff for peak detection": "峰值检测的p值截止",
    "Minimum FDR (q-value) cutoff for peak detection": "最小FDR（q值）阈值用于峰值检测",
    "Additional Outputs": "附加输出",
    "When set, scale the small sample up to the bigger sample": "当设置时，将小样本放大到大样本",
    "Use fixed background lambda as local lambda for every peak region": "使用固定的背景lambda作为每个峰值区域的局部lambda",
    "Save signal per million reads for fragment pileup profiles": "保存每个片段堆积剖面图的每百万个读段信号",
    "When set, use a custom scaling ratio of ChIP/control (e.g. calculated using NCIS) for linear scaling": "当设置时，使用自定义的 ChIP/对照（例如，使用 NCIS 计算）的缩放比例进行线性缩放",
    "The small nearby region in basepairs to calculate dynamic lambda": "在碱基对中计算动态lambda的附近小区域",
    "The large nearby region in basepairs to calculate dynamic lambda": "计算动态lambda的大邻近区域（以碱基对为单位）",
    "Composite broad regions": "复合宽区域",
    "Cutoff for broad region": "广泛区域的截止值",
    "Use a more sophisticated signal processing approach to find subpeak summits in each enriched peak region": "使用更复杂的信号处理方法来寻找每个富集峰区域的子峰顶点",
    "Sequencing alignment file": "测序比对文件",
    "Pvalue cutoff for binomial distribution test": "二项分布检验的P值截止值",
    "ChIP-seq alignment file": "ChIP-seq 对齐文件",
    "Select action to be performed": "选择要执行的操作",
    "Percentage of tags you want to keep": "您想要保留的标签百分比",
    "Number of tags you want to keep": "您想要保留的标签数量",
    "Random seed number": "随机种子数",
    "Candidate peak file in BED format": "候选峰值文件，采用BED格式。",
    "Scan window size on both side of the summit": "峰值两侧的扫描窗口大小",
    "Experiment Name": "实验名称",
    "Paired End Sequencing": "配对末端测序",
    "ChIP-Seq Tag File 1": "ChIP-Seq 标签文件 1",
    "ChIP-Seq Tag File 2": "ChIP-Seq 标签文件 2",
    "ChIP-Seq Control File 1": "ChIP-Seq 控制文件 1",
    "ChIP-Seq Control File 2": "ChIP-Seq 控制文件 2",
    "Best distance between Pair-End Tags": "最佳配对末端标签之间的距离",
    "ChIP-Seq Tag File": "ChIP-Seq 标签文件",
    "Effective genome size": "有效基因组大小",
    "Tag size": "标签大小",
    "Band width": "带宽",
    "Pvalue cutoff for peak detection": "峰值检测的P值截止值",
    "Select the regions with MFOLD high-confidence enrichment ratio against background to build model": "选择具有MFOLD高置信度富集比的区域来构建模型",
    "Parse xls files into into distinct interval files": "将 xls 文件解析成不同的区间文件",
    "Save shifted raw tag count at every bp into a wiggle file": "将每个碱基位置上的偏移原始标签计数保存到wiggle文件中",
    "Extend tag from its middle point to a wigextend size fragment.": "将标签从其中点延伸到一个wigextend大小的片段。",
    "Resolution for saving wiggle files": "保存wiggle文件的分辨率",
    "3 levels of regions around the peak region to calculate the maximum lambda as local lambda": "在峰值区域周围计算局部lambda的最大值，需要考虑3个层级的区域",
    "Arbitrary shift size in bp": "任意的偏移大小（以碱基对为单位）",
    "Diagnosis report": "诊断报告",
    "Min fold enrichment to consider": "最小倍数富集度以考虑",
    "Max fold enrichment to consider": "最大富集倍数考虑",
    "Fold enrichment step": "折叠富集步骤",
    "Perform the new peak detection method (futurefdr)": "执行新的峰值检测方法（futurefdr）",
    "Fix off-by-one errors in output files": "修正输出文件中的偏移错误",
    "Redundancy Threshold": "冗余阈值",
    "Fragment size": "片段大小",
    "Effective genome fraction": "有效基因组分数",
    "Gap size": "间隙大小",
    "Statistic threshold value": "统计阈值",
    "Select a pre-defined configuration file": "选择一个预定义的配置文件",
    "Length of DNA fragment": "DNA片段长度",
    "Step of sliding window": "滑动窗口步骤",
    "isStrandSensitiveMode": "是链特异性模式",
    "Minimum number of read counts at the peak": "峰值处的最小读数数量",
    "Number of peaks reported in top peak file": "在顶部峰值文件中报告的峰值数量",
    "Minimum score of normalized difference": "最小归一化差异分数",
    "Number of passes in the bootstrapping process": "在自助采样过程中的遍历次数",
    "Treatment File": "处理文件",
    "Treatment Files.": "治疗文件。",
    "Control File": "控制文件",
    "Control Files": "控制文件",
    "Do you have a BED file of genomic regions to exclude?": "您有要排除的基因组区域的BED文件吗？",
    "BED File": "BED文件",
    "Remove PCR duplicates": "移除PCR重复序列",
    "Comma-separated list of chromosomes to exclude": "要排除的染色体的逗号分隔列表",
    "Minimum MAPQ to keep an alignment.": "保持比对的最小MAPQ值。",
    "Keep sec alns with AS >= bestAS.": "保留与AS >= 最佳AS的次优序列比对。",
    "Keep unpaired alignments.": "保持未配对的比对结果。",
    "Keep unpaired alignments with a certain length.": "保留具有一定长度的未配对比对。",
    "Keep unpaired alns, lengths changed to paired average.": "保留未配对的比对，长度更改为配对平均值。",
    "ATAC Options": "ATAC选项",
    "Use ATAC-seq mode.": "使用ATAC-seq模式。",
    "Expand cut sites.": "扩展切割位点。",
    "Peakcalling Options": "峰值检测选项",
    "Maximum q-value.": "最大q值。",
    "Maximum p-value.": "最大 p 值。",
    "Minimum AUC for a peak.": "峰值的最小AUC（Area Under the Curve）。",
    "Minimum length of a peak.": "峰值的最小长度。",
    "Maximum distance between signif. sites.": "最大显著位点间距离",
    "Skip peak-calling.": "跳过峰值调用。",
    "Bedgraph-ish p/q Values": "Bedgraph 类型的 p/q 值",
    "Bedgraph-ish Pileups": "Bedgraph风格的堆积图",
    "Bed File": "Bed 文件",
    "Peak files": "峰值文件",
    "Read BAM files": "读取BAM文件",
    "Control BAM files": "控制BAM文件",
    "Score Column": "得分列",
    "Lower score is better?": "较低的分数更好吗？",
    "Summits": "峰值",
    "Output binding affinity matrix?": "输出结合亲和力矩阵？",
    "Output analysis info?": "输出分析信息？",
    "Experiment Libraries": "实验库",
    "Control Libraries": "控制库",
    "Pairwise Replicates": "配对重复样本",
    "Paired End": "配对末端",
    "Maximum Insert Size": "最大插入片段大小",
    "Sub-Features": "子特征",
    "Minimum cluster Expression Fraction": "最小聚类表达分数",
    "Minimum Block Overlap": "最小块重叠",
    "Minimum Block Expression": "最小块表达",
    "Size Factors": "大小因子",
    "Step Size": "步长大小",
    "Statistical Test": "统计检验",
    "Heterogeneous p-value Threshold": "异质性 P 值阈值",
    "Paired p-value Threshold": "配对p值阈值",
    "Mad Multiplier": "疯狂的乘法器",
    "Fold Change Threshold": "折叠变化阈值",
    "Adjusted p-value Threshold": "调整后的 p 值阈值",
    "ChIP alignment file(s)": "ChIP 对齐文件（们）",
    "Control alignment file(s)": "控制对齐文件（们）",
    "Window size in bp (default 300)": "窗口大小，单位为碱基对（默认为300）",
    "Minimum mapping quality (default 20)": "最小映射质量（默认为20）",
    "Output list of targets (default table)": "输出目标列表（默认表格）",
    "Print targets only with higher confidence": "仅打印置信度更高的目标",
    "Target BAM file": "目标BAM文件",
    "Genome reference file": "基因组参考文件",
    "Genomic chromosomes to learn HMM parameters": "基因组染色体学习隐马尔可夫模型参数",
    "Contigs to apply HMM": "将HMM应用于拼接序列",
    "Distance used to merge individual crosslink sites to binding regions": "用于将单个交联位点合并为结合区域的距离",
    "BAM file containing mapped reads from control experiment": "包含对照实验中映射读取的BAM文件",
    "Crosslink-associated (CL) motif options": "交联相关（CL）基序选项",
    "FIMO input motif score covariates file": "FIMO 输入模体得分协变量文件",
    "Max. motif ID to use": "最大使用的主题ID",
    "Define protein binding characteristics": "定义蛋白质结合特性",
    "Bandwidth for kernel density estimation used to access enrichment": "用于访问富集的核密度估计带宽",
    "Bandwidth for kernel density estimation used to estimate n for binomial distributions": "用于估计二项分布中n的核密度估计的带宽",
    "Initial value for binomial probability parameter of 'non-crosslink' state": "非交联状态的二项分布概率参数的初始值",
    "Initial value for binomial probability parameter of 'crosslink' state": "交叉连接状态的二项概率参数的初始值",
    "Choose n threshold for estimating crosslink state parameters": "选择n个阈值以估计交联状态参数",
    "Only sites with n >= ntp are used to learn binomial probability parameters": "只有 n >= ntp 的位点被用来学习二项概率参数",
    "Only sites with n >= ntp2 are used to learn probability of transition from state '2' to '2' or '3'": "仅使用 n >= ntp2 的位点来学习从状态 '2' 到 '2' 或 '3' 的转移概率",
    "Additional advanced parameters": "额外的高级参数",
    "Use higher precision to compute emission probabilities (long double)": "使用更高的精度来计算发射概率（长双精度）",
    "Use Viterbi instead of posterior decoding": "使用Viterbi算法而不是后验解码",
    "Maximum number of iterations within BRENT algorithm": "Brent算法中的最大迭代次数",
    "Maximum number of iterations within Baum-Welch algorithm": "Baum-Welch算法中的最大迭代次数",
    "Minimum shape k of 'non-enriched' gamma distribution": "最小形状 k 的“非富集”伽玛分布",
    "Maximum shape k of 'non-enriched' gamma distribution": "最大形状k的“非富集”伽马分布",
    "Minimum shape k of 'enriched' gamma distribution": "最小形状 k 的“富集”伽马分布",
    "Maximum shape k of 'enriched' gamma distribution": "最大形状k的“富集”伽玛分布",
    "Do not constrain 'non-enriched' shape parameter k": "不要限制“非富集”形状参数 k",
    "Max. k/N ratio (read start sites/N) used to learn truncation probabilities for 'non-crosslink' and 'crosslink' emission probabilities": "最大 k/N 比率（读取起始位点/N）用于学习“非交联”和“交联”发射概率的截断概率",
    "Min. transition probability from state '2' to '3'": "最小转移概率从状态'2'到'3'",
    "Minimum KDE value used for fitting left-truncated gamma distributions": "用于拟合左截断伽马分布的最小KDE值",
    "Length threshold for internal poly-X stretches to get excluded": "内部聚X序列排除的长度阈值",
    "Exclude intervals containing poly-A stretches from learning": "排除包含聚腺苷酸序列的区间以进行学习",
    "Exclude intervals containing poly-A stretches from analysis": "排除含有聚腺苷酸序列的区间，不进行分析。",
    "Exclude intervals containing poly-U stretches from learning": "排除包含多聚U序列的间隔，以便进行学习。",
    "Exclude intervals containing poly-U stretches from analysis": "排除包含多U序列的区间，不进行分析。",
    "Fit gamma shape k only for positions with min. covariate value": "仅对具有最小协变量值的位置拟合伽马形状 k",
    "Maximum number of truncations at one position used for learning": "用于学习的一个位置上允许的最大截断数",
    "Prior enrichment threshold": "先验富集阈值",
    "Additional output options": "附加输出选项",
    "Output learned parameter statistics file?": "输出学习到的参数统计文件？",
    "Choose input BED or BAM file": "选择输入的 BED 或 BAM 文件",
    "Choose covariate": "选择协变量",
    "Choose covariate BED file": "选择协变量 BED 文件",
    "Is your input file already sorted?": "你的输入文件已经排序了吗？",
    "Significance threshold for sites": "位点的显著性阈值",
    "Disable p-values correction for multiple hypothesis testing?": "禁用多重假设检验的 p 值校正？",
    "Indicates that this proportion of the lowest scores should be considered the background": "表明这个比例的最低分应该被认为是背景",
    "Indicates that input is raw reads and should be binned into bins of this size": "表示输入是原始读段，并且应该被分箱成这个大小的箱子",
    "Indicates that the covariates are raw reads and should be binned into bins of this size": "表明协变量是原始读段，应该被分箱到这个大小的箱子中",
    "Synonymous with -b x -i x for any x": "等同于 -b x -i x，对于任何 x",
    "Merge significant bins within certain distance?": "在一定距离内合并显著的bins？",
    "Merge significant bins within this distance": "在这个距离内合并显著的bin",
    "Print covariate values in output": "在输出中打印协变量值",
    "Fit only": "仅拟合",
    "Select distribution type": "选择分布类型",
    "Use component fitting method": "使用组件拟合方法",
    "Select the model file": "选择模型文件",
    "Preserve strand information": "保留链信息",
    "Normalise covariates?": "标准化协变量？",
    "Convert covariates to log scale?": "将协变量转换为对数尺度？",
    "GFF annotation file": "GFF 注释文件",
    "Is the GFF file unsorted?": "GFF 文件是否未排序？",
    "Sliding window settings": "滑动窗口设置",
    "Sliding window step size": "滑动窗口步长",
    "Output annotation BED file": "输出注释的BED文件",
    "CLIP-seq experiment BAM inputs": "CLIP-seq 实验 BAM 输入",
    "BAM files belonging to the CLIP-seq experiment": "属于CLIP-seq实验的BAM文件",
    "CLIP-seq control BAM inputs": "CLIP-seq 控制 BAM 输入",
    "BAM files belonging to the CLIP-seq control": "属于CLIP-seq对照的BAM文件",
    "Sliding windows BED file": "滑动窗口 BED 文件",
    "Crosslink site extraction settings": "交联位点提取设置",
    "Read mate to extract crosslink sites from": "读取配对序列以提取交联位点",
    "Specify crosslink site position on read": "指定读取上的交联位点位置",
    "Crosslink site offset": "交联位点偏移",
    "Use only primary positions of multimapping reads?": "仅使用多映射读取的主要位置？",
    "Maximum read interval length": "最大读取区间长度",
    "Specify chromosomes to extract crosslink sites from": "指定要从中提取交联位点的染色体",
    "BED file for filtering out BAM entries": "用于过滤BAM条目的BED文件",
    "Filtering mode for BED filtering": "BED过滤的过滤模式",
    "Overlap count settings": "重叠计数设置",
    "Should crosslink site counting be non-strand-specific?": "交联位点计数是否应该不区分链特异性？",
    "Windows annotation file": "Windows注释文件",
    "Count table file": "计数表文件",
    "Sample information table file": "样本信息表格文件",
    "DEWSeq min_sample parameter": "DEWSeq 最小样本参数",
    "DEWSeq min_count parameter": "DEWSeq 最小计数参数",
    "DEWSeq p_value_cutoff parameter": "DEWSeq p_value_cutoff 参数",
    "DEWSeq lfc_cutoff parameter": "DEWSeq lfc_cutoff 参数",
    "Use overlapping windows p-value correction?": "使用重叠窗口 p 值校正？",
    "Disable IHW?": "禁用IHW？",
    "Disable DEWSeq's decide_fit?": "禁用 DEWSeq 的 decide_fit？",
    "Use LRT instead of Wald test?": "使用似然比检验而不是Wald检验？",
    "DEWSeq dataset ID": "DEWSeq 数据集 ID",
    "Output significant regions BED file?": "输出显著区域的BED文件？",
    "Output significant windows + regions BED file?": "输出显著窗口+区域的BED文件？",
    "Output HTML report file?": "输出 HTML 报告文件？",
    "ChIP Dataset in BED format": "ChIP数据集，以BED格式",
    "Input Dataset in BED format": "输入数据集为 BED 格式",
    "Number of most highly enriched regions used to estimate peak size (Default: 200)": "用于估计峰值大小的最多高度富集区域数（默认值：200）",
    "Minimum cutoff for fold enrichment (Default: 2)": "富集倍数的最小截止值（默认值：2）",
    "Minimum cutoff for peak score (Default: 1)": "峰值分数的最小截止值（默认值：1）",
    "Manually set fragment size in bp": "手动设置片段大小（以碱基对为单位）",
    "Use empirical model estimate instead of gaussian": "使用经验模型估计而不是高斯模型",
    "Input is paired end and in BEDPE format": "输入是成对的末端，并且是BEDPE格式的。",
    "Output negative peaks in control sample": "输出对照样本中的负峰",
    "First BAM/CRAM file (e.g. treated sample)": "首个BAM/CRAM文件（例如，处理过的样本）",
    "Second BAM/CRAM file (e.g. control sample)": "第二个BAM/CRAM文件（例如对照样本）",
    "Bin size in bases": "基对数的箱大小",
    "Method to use for scaling the largest sample to the smallest": "用于将最大样本缩放到最小样本的方法",
    "Length in bases used to sample the genome and compute the size or scaling factors.": "用于采样基因组并计算大小或缩放因子的碱基长度。",
    "Number of samplings taken from the genome to compute the scaling factors": "从基因组中取样的次数，用于计算缩放因子",
    "Normalization method": "标准化方法",
    "How to compare the two files": "如何比较两个文件",
    "Coverage file format": "覆盖度文件格式",
    "Show advanced options": "显示高级选项",
    "regions that should be excluded for calculating the scaling factor": "应该排除用于计算缩放因子的区域",
    "BAM/CRAM file": "BAM/CRAM 文件",
    "Scaling/Normalization method": "缩放/标准化方法",
    "Regions that should be excluded for normalization": "应排除用于标准化的区域",
    "Determine nucleosome positions from MNase-seq data": "确定MNase-seq数据中的核小体位置",
    "Offset inside each alignment to use for the signal location.": "在每个比对中用于信号位置的偏移量。",
    "Only include reads originating from fragments from the forward or reverse strand.": "仅包含来自正向或反向链的片段的读取。",
    "Get the distribution of read/fragment lengths as a histogram": "获取读长/片段长度的分布，以直方图的形式展示。",
    "Labels for the samples (each BAM file) plotted": "样本（每个BAM文件）的标签绘制",
    "bin size, in bases": "二进制大小，以碱基为单位",
    "bin spacing, in bases": "bin间距，在碱基对上",
    "Write metrics to a tab-separated file?": "将指标写入一个制表符分隔的文件？",
    "Write a file containing the read/fragment length information?": "生成一个包含读长/片段长度信息的文件？",
    "Plot log frequencies": "绘制对数频率图",
    "Maximum fragment length": "最大片段长度",
    "Treatment bigwig file": "处理大wig文件",
    "bigWig file": "大波伊格文件",
    "Length, in bases, of the non-overlapping bins used for averaging the score over the regions length": "用于在区域长度上平均分数的非重叠箱的长度，以碱基为单位。",
    "Matrix file from the computeMatrix tool": "来自computeMatrix工具的矩阵文件",
    "List of samples to be used for sorting": "用于排序的样本列表",
    "Draw dashed lines in heatmap above all tick marks?": "在热图上方所有刻度标记上绘制虚线？",
    "Type of statistic that should be plotted in the summary image above the heatmap": "在热图上方绘制的摘要图像应使用的统计类型",
    "Type of summary plot": "摘要图类型",
    "Missing data color": "缺失数据颜色",
    "Colormap to use for each sample": "为每个样本使用的颜色映射",
    "Alpha channel (transparency)": "Alpha 通道（透明度）",
    "List of colors for each heatmap. Separate lists by spaces and the colors in the list by comas": "每个热图的颜色列表。用空格分隔列表，用逗号分隔列表中的颜色",
    "Minimum value for the Y-axis of the summary plot. Leave empty for automatic values": "摘要图的 Y 轴的最小值。留空以使用自动值",
    "Maximum value for Y-axis of the summary plot. Leave empty for automatic values": "摘要图Y轴的最大值。留空以使用自动值",
    "The x-axis label": "x轴标签",
    "The y-axis label for the top panel": "顶部面板的y轴标签",
    "Heatmap width in cm": "热图宽度（单位：厘米）",
    "Heatmap height in cm": "热图高度（以厘米为单位）",
    "What to show": "要展示什么",
    "Label for the region start": "区域起始标签",
    "Label for the region end": "区域结束的标签",
    "Reference point label": "参考点标签",
    "Labels for the samples (each bigwig) plotted": "样本标签（每个bigwig绘制的）",
    "Labels for the regions plotted in the heatmap": "热图中绘制区域的标签",
    "Make one plot per group of regions": "为每个区域组制作一个图表",
    "List of samples to be used for clustering": "用于聚类的样本列表",
    "Output of computeGCBias": "computeGCBias的输出",
    "Choose computation mode": "选择计算模式",
    "Bin size in bp": "Bin大小（以碱基对为单位）",
    "Region file in BED or INTERVAL format": "区域文件，格式为 BED 或 INTERVAL",
    "Save raw counts (coverages) to file": "将原始计数（覆盖度）保存到文件",
    "Save scaling factors": "保存缩放因子",
    "Matrix file from the multiBamSummary or multiBigwigSummary tools": "来自 multiBamSummary 或 multiBigwigSummary 工具的矩阵文件",
    "Save the matrix of PCA and eigenvalues underlying the plot.": "保存PCA矩阵和图表底层的特征值。",
    "Principal components to plot": "主成分绘图",
    "Number of rows to use": "要使用的行数",
    "log2 transform data": "对数转换数据",
    "Transpose Matrix?": "转置矩阵？",
    "Center Rows?": "居中行？",
    "Symbol colors": "符号颜色",
    "Custom markers": "自定义标记",
    "Select regions": "选择区域",
    "Regions to plot": "要绘制的区域",
    "computeMatrix has two main output options": "computeMatrix 有两个主要的输出选项",
    "Distance in bases to which all regions are going to be fit": "所有区域将被拟合的基底距离",
    "Set distance up- and downstream of the given regions": "设置给定区域上游和下游的距离",
    "Distance upstream of the region start position": "区域起始位置上游的距离",
    "Distance downstream of the region end position": "区域末端下游距离",
    "Number of bases within the gene body at the 5-prime end to exclude from scaling.": "在基因体的5'端排除用于缩放的碱基数量。",
    "Number of bases within the gene body at the 3-prime end to exclude from scaling.": "从基因体3'端排除的用于缩放的碱基数量。",
    "The reference point for the plotting": "绘图的参考点",
    "Discard any values after the region end": "丢弃区域结束之后的所有值",
    "Distance upstream of the start site of the regions defined in the region file": "在区域文件中定义的区域的起始位点上游的距离",
    "Distance downstream of the end site of the given regions": "给定区域的末端位点下游距离",
    "Length, in bases, of non-overlapping bins used for averaging the score over the regions length": "用于对区域长度上的平均分数进行计算的非重叠箱的长度，以碱基对为单位。",
    "Define the type of statistic that should be displayed.": "定义应该显示的统计类型。",
    "Convert missing values to 0?": "将缺失值转换为0？",
    "Minimum threshold": "最小阈值",
    "Maximum threshold": "最大阈值",
    "Number of sampling points to consider": "要考慮的採樣點數量",
    "Region size": "区域大小",
    "BED file containing genomic regions for which extra sampling is required because they are underrepresented in the genome": "包含基因组区域的BED文件，因为这些区域在基因组中代表性不足，所以需要额外采样。",
    "GC bias plot": "GC偏差图",
    "Region file in BED format": "区域文件，采用BED格式",
    "Save raw counts (scores) to file": "将原始计数（分数）保存到文件",
    "The input matrix was computed in scale-regions mode": "输入矩阵是在尺度区域模式下计算的",
    "Define the type of statistic that should be used for the profile.": "定义用于概要文件的统计类型。",
    "Plot height": "绘制高度",
    "Plot width": "绘图宽度",
    "List of colors to use for the plotted lines": "用于绘制线条的颜色列表",
    "Save the data underlying the average profile": "保存平均轮廓背后的数据",
    "Matrix file from the multiBamSummary tool": "来自multiBamSummary工具的矩阵文件",
    "Plotting type": "绘图类型",
    "Manually set X axis range?": "手动设置X轴范围？",
    "X axis minimum value": "X轴最小值",
    "X axis maximum value": "X轴最大值",
    "Manually set Y axis range?": "手动设置 Y 轴范围？",
    "Y axis minimum value": "Y轴最小值",
    "Y axis maximum value": "Y轴最大值",
    "Log transform": "对数转换",
    "Remove regions with very large counts": "移除计数非常大的区域",
    "Save the matrix of values underlying the heatmap": "保存热图底层的值矩阵",
    "Regions of interest": "感兴趣的区域",
    "Show coverage metrics options": "显示覆盖度指标选项",
    "Save per-threshold coverage metrics?": "保存每个阈值的覆盖度指标？",
    "Coverage Thresholds": "覆盖度阈值",
    "Coverage Threshold": "覆盖度阈值",
    "Save raw counts (coverages) to a file": "将原始计数（覆盖度）保存到文件中",
    "Show advanced output settings": "显示高级输出设置",
    "Save the bin counts": "保存区间计数",
    "Save quality metrics": "保存质量指标",
    "Jensen-Shannon distance reference sample": "詹森-香农距离参考样本",
    "Operation type": "操作类型",
    "New Region/Group Labels.": "新区域/组标签。",
    "New Sample Labels.": "新样品标签。",
    "Region groups to include.": "要包含的区域组。",
    "Samples to include.": "要包含的样本。",
    "Only include regions on this strand.": "仅包含此链上区域。",
    "Minimum value": "最小值",
    "Matrix files from computeMatrix": "来自computeMatrix的矩阵文件",
    "BED or GTF files of regions": "区域的 BED 或 GTF 文件",
    "Output in BEDPE format?": "输出为BEDPE格式？",
    "Amount to shift fragments": "片段偏移量",
    "Shift fragment ends as appropriate for ATAC-seq": "适当地调整 ATAC-seq 的片段末端",
    "Save the total number of reads seen and remaining after filtering to a text file?": "将过滤后剩余的总读数和过滤前看到的总读数保存到一个文本文件中。",
    "Save alignments NOT passing the filtering criteria?": "保存未通过过滤标准的比对结果？",
    "Optional attribute key": "可选属性键",
    "Labels for the features": "特征标签",
    "Alpha channel": "阿尔法通道",
    "List of colors to use for the plotted bars": "用于绘制条形图的颜色列表",
    "Create one plot per-sample": "为每个样本创建一个图表",
    "Allow variable scales": "允许变量比例尺",
    "Number of plots/row": "每行的图形数量",
    "Include BED12 exons": "包含 BED12 外显子",
    "Save percentages to a file": "将百分比保存到一个文件中",
    "Distance between bins": "两个区间之间的距离",
    "Input file(s)": "输入文件",
    "Scale factors (one per bigwig separated by ':')": "比例因子（每个 bigwig 一个，用 ':' 分隔）",
    "Build GEMINI query using": "使用构建 GEMINI 查询",
    "Type of report to generate": "生成报告的类型",
    "Tumor sample ID": "肿瘤样本ID",
    "Normal sample ID (for matched tumor/normal sample pair analyses only)": "正常样本ID（仅用于配对的肿瘤/正常样本对分析）",
    "Mutation status to be recorded for variants reported with this query": "变异状态需要记录在此查询报告的变体中",
    "Mutation status (custom value)": "突变状态（自定义值）",
    "SQL expression used to compute per-variant status": "用于计算每个变体状态的SQL表达式",
    "The query to be issued to the database": "要向数据库发出的查询",
    "Information to retrieve from the database": "从数据库中检索的信息",
    "Format of the pedigree file": "家谱文件的格式",
    "VCF dataset to be loaded in the GEMINI database": "要加载到GEMINI数据库中的VCF数据集",
    "The variants in this input are": "这个输入中的变体是",
    "This input comes with genotype calls for its samples": "这个输入包含了其样本的基因型调用",
    "Sample and family information in PED format": "PED格式的样本和家族信息",
    "Load the following optional content into the database": "将以下可选内容加载到数据库中",
    "Version of ensembl genes to use": "要使用的Ensembl基因版本",
    "Report only pathways with loss-of-function variants": "只报告带有功能丧失变异的通路",
    "The type of window analysis requested?": "请求的窗口分析类型？",
    "The operation that should be applied to the -t values": "应应用于-t值的操作",
    "Step size for sliding the window in bp": "窗口滑动的步长大小（以碱基对为单位）",
    "Dataset to use as the annotation source": "要用作注释来源的数据集",
    "Strict variant-identity matching of database and annotation records (VCF format only)": "严格匹配数据库和注释记录（仅限VCF格式）的变体身份",
    "Type of information to add to the database variants": "要添加到数据库变体的信息类型",
    "Annotation extraction recipe": "注释提取配方",
    "Elements to extract from the annotation source": "从注释源中提取的元素",
    "What type of data are you trying to extract?": "您想要提取哪种类型的数据？",
    "If multiple annotations are found for the same variant, store ...": "如果为同一变体发现多个注释，存储...",
    "Additional sample information file in PED+ format": "额外的样本信息文件，采用 PED+ 格式",
    "Set all values to None before loading": "在加载之前将所有值设置为无",
    "Make this an obligate filter that a variant has to pass to be considered": "将此设置为一个强制过滤器，变体必须通过该过滤器才能被考虑",
    "Minimum number of filters": "最小过滤器数量",
    "Output - included information": "输出 - 包含的信息",
    "Use the following types of variants for the burden calculation": "使用以下类型的变异进行负担计算",
    "Which type of burden calculation should be performed?": "应该进行哪种类型的负担计算？",
    "Selection of case and control samples": "案例和对照样本的选择",
    "Space separated list of cases for association testing": "用于关联测试的以空格分隔的病例列表",
    "Space separated list of controls for association testing": "用于关联测试的以空格分隔的对照列表",
    "The min. alt. allele frequency for a variant to be included": "最小等位基因频率，用于包含一个变体",
    "The max. alt. allele frequency for a variant to be included": "最大等位基因频率，用于包含一个变体",
    "Number of permutations to run for the C-alpha test": "C-alpha 测试运行的排列数",
    "Save the list of individual T-scores from all permutations as an extra dataset": "将所有排列的个体T分数列表保存为一个额外的数据集",
    "Select the type of statistics you are interested in": "选择您感兴趣的统计类型",
    "Calculate Ts/Tv statistics based on": "计算基于的Ts/Tv统计",
    "Compute the genotype counts table based on": "计算基于的基因型计数表",
    "Restrictions to apply to genotype values": "对基因型值应用的限制",
    "Additional constraints on the variants": "额外的变异约束",
    "One or both genes in fusion is in COSMIC cancer census": "融合基因中的一个或两个在COSMIC癌症普查中",
    "The min variant quality (VCF QUAL)": "最小变体质量（VCF QUAL）",
    "The supporting evidence types for the variant": "变异的支持证据类型",
    "Minimum number of expected homozygous SNPs": "预期的最小纯合SNP数量",
    "The minimum overall sequencing depth required for a SNP to be considered": "所需的最小整体测序深度，以便将单核苷酸多态性（SNP）考虑在内",
    "The minimum required sequencing depth underlying a given sample's genotype for a SNP to be considered": "对于一个给定样本的基因型，考虑一个SNP所需的最小测序深度",
    "Minimum run size in base pairs": "最小运行大小（以碱基对为单位）",
    "Maximum number of allowed hets in the run": "运行中允许的最大异质性数目",
    "Maximum number of allowed unknowns in the run": "运行中允许的最大未知数数量",
    "Comma separated list of samples to screen for ROHs": "逗号分隔的样本列表，用于筛选纯合区域（ROHs）",
    "The min combined depth for tumor + normal": "肿瘤和正常组织的最小组合深度",
    "The min somatic score (SSC)": "最小体细胞得分（SSC）",
    "The max frequency of the alternative allele in the normal sample": "正常样本中替代等位基因的最大频率",
    "The max count of the alternative allele in the normal sample": "正常样本中替代等位基因的最大计数",
    "The minimum depth allowed in the normal sample to believe somatic": "正常样本中允许的最小深度，以相信体细胞变异",
    "The minimum frequency of the alternative allele in the tumor sample": "肿瘤样本中替代等位基因的最小频率",
    "The minimum count of the alternative allele in the tumor sample": "肿瘤样本中替代等位基因的最小计数",
    "The minimum depth allowed in the tumor sample to believe somatic": "在肿瘤样本中允许的最小深度，以相信体细胞变异",
    "A specific chromosome on which to tag somatic mutations": "一个用于标记体细胞突变的特定染色体",
    "Which chromosome should the sex test be applied to?": "性染色体检测应该应用到哪个染色体上？",
    "Interaction data source": "交互数据源",
    "Report affected interaction partners of this particular gene": "报告这个特定基因的受影响的相互作用伙伴",
    "Report interaction partners up to (and including) this order": "报告相互作用伙伴，直到（包括）这个顺序",
    "Report format for interactions": "相互作用报告格式",
    "Your assumption about the inheritance pattern of the phenotype of interest": "你对感兴趣的表型遗传模式的假设",
    "Ignore sample phenotypes during variant identification": "在变异识别期间忽略样本表型",
    "Family-wise criteria for variant selection": "家族式标准用于变异选择",
    "List of families to restrict the analysis to (comma-separated)": "限制分析范围的家族列表（用逗号分隔）",
    "Specify additional criteria to exclude families on a per-variant basis": "为每个变体指定额外的标准以排除家族",
    "Per-variant DP threshold for including a family": "每个变体的DP阈值，用于包含一个家系",
    "per-variant GQ threshold for including a family": "每个变体包含家族的GQ阈值",
    "per-variant PL threshold for including a family": "每个变体包含一个家族的PL阈值",
    "On query": "在查询上",
    "Minimum length of region": "最小区域长度",
    "Protein sequence": "蛋白质序列",
    "Codon usage file": "密码子使用文件",
    "Output sequence file format": "输出序列文件格式",
    "Replace all": "替换所有",
    "with": "使用",
    "Minimum ORF length to report": "报告的最小开放阅读框长度",
    "An asterisk in the protein sequence indicates the position of a STOP codon": "蛋白质序列中的星号表示终止密码子的位置",
    "Output feature file format": "输出特征文件格式",
    "Sum codons over all sequences": "将所有序列的密码子求和",
    "Codon usage file 1": "密码子使用文件 1",
    "Codon usage file 2": "密码子使用文件 2",
    "Size of word (window) to count": "单词大小（窗口）计数",
    "Frame to inspect": "检查框架",
    "Minimum average observed to expected ratio": "最小观察到期望比率",
    "Minimum average percentage of G plus C": "G加C的最小平均百分比",
    "Score for each CG sequence found": "每个找到的CG序列的得分",
    "Start of region to delete": "要删除的区域的开始",
    "End of region to delete": "要删除区域的结束",
    "Step size (shift increment)": "步长（位移增量）",
    "DNA Concentration (nM)": "DNA浓度（纳摩尔）",
    "Salt concentration (mM)": "盐浓度（毫摩尔）",
    "Output the DeltaG, DeltaH and DeltaS values": "输出ΔG、ΔH和ΔS的值",
    "Temperature at which to calculate the DeltaG, DeltaH and DeltaS values": "计算DeltaG、DeltaH和DeltaS值的温度",
    "Sequence is RNA": "序列是RNA",
    "Include percent formamide, percent of mismatches allowed and product length": "包含甲酰胺百分比、允许的错配百分比和产物长度",
    "Formamide concentration (nM)": "甲酰胺浓度（纳摩尔）",
    "Percent mismatch to be used in calculations": "用于计算的百分比不匹配",
    "Product length to be used in calculations": "用于计算的产品长度",
    "Create a graph": "创建一个图形",
    "Name of the sequence": "序列名称",
    "Description of the sequence": "序列描述",
    "Append the name or description on to the end of the existing name or description": "将名称或描述附加到现有名称或描述的末尾",
    "Sequence 1": "序列 1",
    "Sequence 2": "序列 2",
    "Wordsize": "词长",
    "Report differences at the ends": "报告末端的差异",
    "Output report file format": "输出报告文件格式",
    "Sequence 1 output feature file format": "序列1输出特征文件格式",
    "Sequence 2 output feature file format": "序列2输出特征文件格式",
    "Enzyme/Reagent": "酶/试剂",
    "Trypsin will not normally cut after a K if it is followed by (e.g.) another K or a P. Specifying this shows those cuts, as well as the favoured ones": "胰蛋白酶通常不会在赖氨酸（K）后切割，如果它后面是（例如）另一个K或P。指定这一点会显示这些切割，以及更受青睐的切割。",
    "Used for partial digestion. Shows all cuts from favoured cut sites plus 1..3, 2..4, 3..5 etc but not (e.g.) 2..5. Overlaps are therefore fragments with exactly one potential cut site within it": "用于部分消化。显示所有来自优选切割位点的切割，加上1..3、2..4、3..5等，但不包括（例如）2..5。因此，重叠是具有恰好一个潜在切割位点的片段。",
    "As for overlap but fragments containing more than one potential cut site are included": "对于重叠，但包含一个以上潜在切割位点的片段则被包括在内",
    "Threshold": "阈值",
    "Display the overlapping matches (in red) as well as the minimal set of non-overlapping matches": "显示重叠匹配（红色显示）以及非重叠匹配的最小集合",
    "Draw a box around dotplot": "在点图周围画一个框。",
    "Regular expression pattern": "正则表达式模式",
    "Gap penalty": "间隔惩罚",
    "Minimum score threshold": "最小得分阈值",
    "Maximum separation between the start of repeat and the end of the inverted repeat": "重复序列起始点与倒置重复序列终点之间的最大分离距离",
    "Minimal distance between positively charged amino acids": "带正电荷的氨基酸之间的最小距离",
    "Threshold value to discriminate weak from potential PEST motifs": "用于区分弱PEST基序和潜在PEST基序的阈值",
    "Decide whether potential PEST motifs should be printed": "决定是否打印潜在的PEST序列模体",
    "Decide whether poor PEST motifs should be printed": "决定是否打印不良的PEST基序",
    "Decide whether invalid PEST motifs should be printed": "决定是否打印无效的PEST基序",
    "Decide whether PEST motifs should be mapped to sequence": "决定是否将 PEST 序列基序映射到序列上",
    "Maximum repeat size": "最大重复单元大小",
    "Threshold score": "阈值分数",
    "EST sequence(s)": "EST序列（s）",
    "Score for matching two bases": "匹配两个碱基的分数",
    "Cost for mismatching two bases": "两个碱基不匹配的代价",
    "Cost for deleting a single base in either sequence, excluding introns": "在任一序列中删除单个碱基的成本，不包括内含子",
    "Cost for an intron, independent of length": "内含子的代价，与长度无关",
    "Cost for an intron, independent of length and starting/ending on donor-acceptor sites": "内含子的成本，与长度无关，且起始于供体位点，结束于受体位点",
    "Exclude alignments with scores below this threshold score": "排除得分低于此阈值的比对",
    "Reverse the orientation of the EST sequence": "反转EST序列的方向",
    "Use donor and acceptor splice sites": "使用供体和受体剪接位点",
    "Comparison mode": "比较模式",
    "Only best comparisons": "仅最佳比较",
    "Shuffle": "洗牌",
    "Random number seed": "随机数种子",
    "Show the alignment": "显示对齐结果",
    "Alignment width": "对齐宽度",
    "Minimum repeat size": "最小重复大小",
    "Allow N as a mismatch": "允许N作为一个不匹配",
    "Allow uniform consensus": "允许统一共识",
    "Number of bases or residues before the feature to include in the extracted sequence": "在提取的序列中包含特征之前的基础或残基的数量",
    "Number of bases or residues after the feature to include in the extracted sequence": "在提取的序列中包含特征之后的基础数量或残基数量",
    "Feature source": "特征来源",
    "Feature sense": "特征 正向",
    "Maximum score": "最大分数",
    "Feature tags": "特征标签",
    "Tag values": "标签值",
    "Join features": "连接特征",
    "Put feature type in sequence name": "在序列名称中放入特征类型",
    "Specify one or more tag names that should be added to the output sequence Description text": "指定一个或多个标签名称，这些名称应添加到输出序列描述文本中",
    "Regions to extract": "提取区域",
    "Write each specified region as a separate sequence": "将每个指定的区域作为独立的序列写出",
    "Residue letters": "残基字母",
    "Stepping value": "步进值",
    "Averaging window": "平均窗口",
    "Number of mismatches": "不匹配数",
    "Search complementary strand": "搜索互补链",
    "Sequence associated qualifiers": "序列相关限定词",
    "Start of each sequence to be used": "每个序列的起始部分将被使用",
    "End of each sequence to be used": "每个序列要使用的末端",
    "Reverse (if DNA)": "反转（如果是DNA）",
    "Sequence is nucleotide": "序列是核苷酸",
    "Sequence is protein": "序列是蛋白质",
    "Make lower case": "将文本转换为小写",
    "Make upper case": "大写",
    "Outfile associated qualifiers": "输出文件相关修饰词",
    "Show accession number in the report": "在报告中显示访问号码",
    "Show description in the report": "在报告中显示描述",
    "Show the score in the report": "在报告中显示分数",
    "Show the full USA in the report": "在报告中显示完整的美国地图",
    "Maximum total hits to report": "最大报告的总命中数",
    "Maximum hits to report for one sequence": "对于一个序列报告的最大命中数",
    "Frame(s) to translate": "需要翻译的框架（或帧）",
    "Code to use": "使用代码",
    "idc": "IDC",
    "Minimum nucleotide size of ORF to report": "报告开放阅读框（ORF）的最小核苷酸大小",
    "Maximum nucleotide size of ORF to report": "报告的ORF最大核苷酸大小",
    "What to output": "输出什么",
    "All START codons to code for Methionine": "所有起始密码子编码蛋氨酸",
    "Circular sequence": "循环序列",
    "Find ORFs in the reverse complement": "在反向互补序列中查找开放阅读框（ORF）。",
    "Number of flanking nucleotides to output": "输出的侧翼核苷酸数量",
    "Mean value": "平均值",
    "Standard deviation value": "标准差值",
    "Minimum SD": "最小标准差",
    "Use the old (1987) weight data": "使用旧的（1987年）体重数据",
    "Alpha helix angle (degrees)": "α-螺旋角度（度）",
    "Step value for pH": "pH的步长值",
    "Number of N-termini": "N-末端数量",
    "Include charge at N and C terminus": "在N端和C端包含电荷",
    "Format output as an HTML table": "将输出格式化为 HTML 表格",
    "Display column headings": "显示列标题",
    "Display the USA of the sequence": "显示序列的USA",
    "Display 'name' column": "显示“名称”列",
    "Display 'accession' column": "显示“accession”列",
    "Display 'GI' column": "显示 'GI' 列",
    "Display 'version' column": "显示 '版本' 列",
    "Display 'type' column": "显示“类型”列",
    "Display 'length' column": "显示“长度”列",
    "Display 'percent GC content' column": "显示“GC含量百分比”列",
    "Display 'description' column": "显示“描述”列",
    "Shift increment": "位移增量",
    "Draw a ruler": "画一把尺子",
    "Type of blocks": "块类型",
    "Maximum number of groups": "最大组数",
    "Maximum number of labels": "最大标签数",
    "Colour of junctions between blocks": "区块间连接点的颜色",
    "Horizontal junctions between ticks": "水平连接节点",
    "Interval between ticks in the ruler": "标尺上刻度之间的间隔",
    "Vertical lines at the ruler's ticks": "刻度尺上的垂直线",
    "Height of text": "文本高度",
    "Length of text": "文本长度",
    "Width of left margin": "左边距宽度",
    "Height of ticks": "刻度高度",
    "Height of blocks": "区块高度",
    "Height of range ends": "范围结束高度",
    "Space between groups": "组间空格",
    "Space between text and ticks, blocks, and ranges": "文本与刻度、区块和范围之间的间距",
    "Feature to mask": "特征掩蔽",
    "Mask features by converting to lowercase": "将特征转换为小写以进行掩蔽",
    "Character to mask with": "掩蔽字符",
    "Regions to mask (Example 1-99)": "需要屏蔽的区域（示例 1-99）",
    "Mask by converting to lowercase": "掩码，通过转换为小写字母",
    "Character to use when masking": "用于掩蔽时使用的字符",
    "Number of alternative matches": "替代匹配的数量",
    "Gap length (extension) penalty": "间隔长度（延伸）惩罚",
    "Output alignment file format": "输出比对文件格式",
    "Prefer first sequence when mismatches occur": "优先选择第一个序列，当发生不匹配时",
    "Number of times to perform the mutation operations": "执行变异操作的次数",
    "Types of point mutations to perform": "执行点突变的类型",
    "Types of block mutations to perform": "要执行的块突变类型",
    "Types of codon mutations to perform": "要执行的密码子突变类型",
    "Do 'codon' and 'block' operations in frame": "在框架中进行“密码子”和“区块”操作",
    "Minimum size for a block mutation": "最小区块突变大小",
    "Maximum size for a block mutation": "最大区块突变大小",
    "Brief identity and similarity": "简短的身份和相似性",
    "Step size (shift)": "步长（位移）",
    "CpG score": "CpG得分",
    "Name of of the sequence": "序列名称",
    "Type of sequence": "序列类型",
    "The sequence itself": "序列本身",
    "Target operating system for end-of-line format": "目标操作系统的行结束格式",
    "Exclude list": "排除列表",
    "Number of the sequence to output": "输出序列的数量",
    "Display the octanol plot": "显示辛醇图",
    "Display the interface plot": "显示接口图",
    "Display the difference plot": "显示差异图",
    "Dataset in the format of the output produced by 'compseq' that is used to set the minimum frequencies of words in this analysis": "数据集采用“compseq”输出格式，用于设定本分析中单词的最小频率",
    "Size of window in which to count": "窗口大小，用于计数",
    "Ignore B and Z": "忽略B和Z",
    "Minimum length of palindrome": "最小回文长度",
    "Maximum length of palindrome": "最长回文长度",
    "Maximum gap between repeated regions": "重复区域之间最大间隔",
    "Report overlapping matches": "报告重叠的匹配",
    "Main sequence": "主序列",
    "Sequence to insert": "序列插入",
    "Position to insert after": "插入位置之后",
    "Report coiled coil regions": "报告螺旋卷曲区域",
    "Show coil frameshifts": "显示线圈移码",
    "Report non coiled coil regions": "报告非螺旋卷曲区域",
    "Window size for hydropathy averaging": "疏水性平均窗口大小",
    "Choose a plot type": "选择一个图表类型",
    "Residues to mark with squares": "标记为正方形的残基",
    "Residues to mark with diamonds": "标记为钻石的残基",
    "Residues to mark with octagons": "用八边形标记的残基",
    "Amphipathic?": "两亲性的？",
    "Steps, the number of residues plotted per turn is this value divided by the 'turns' value": "步骤，每个转角绘制的残基数是这个值除以“转角”的值",
    "Turns, the number of residues plotted per turn is the 'steps' value divided by this value": "每圈旋转，绘制的残基数是“步数”值除以这个值",
    "Plot the wheel": "绘制韦恩图",
    "Number of columns to average alignment quality over": "要平均对齐质量的列数",
    "Start codons": "起始密码子",
    "Draw a box around each dotplot": "为每个点图绘制一个框。",
    "Size of gap": "间隙大小",
    "The number of residues to be displayed on each line": "每行显示的残基数",
    "Residues before a space": "在空格前的残基",
    "Colour residues by their consensus value": "根据它们的一致性值对残基进行着色",
    "Colour to display identical residues": "显示相同残基的颜色",
    "Colour to display similar residues": "显示相似残基的颜色",
    "Colour to display other residues": "显示其他残基的颜色",
    "Colour residues by table oily, amide etc.": "按油性表、酰胺等对残基进行着色。",
    "Display the title": "显示标题",
    "Values to represent identical similar related": "表示相同、相似、相关的值",
    "Only match those which are identical in all sequences": "仅匹配在所有序列中都完全相同的部分",
    "Display prettyboxes": "显示漂亮的盒子",
    "Colour the background in the boxes": "在框中为背景上色",
    "Colour to be used for background": "用于背景的颜色",
    "Display the sequence names": "显示序列名称",
    "Margin size for the sequence name": "序列名称的边距大小",
    "Display the residue number": "显示残基编号",
    "Display the date and options used": "显示日期和使用的选项",
    "Display the consensus": "显示共识序列",
    "Allow collisions in calculating consensus": "在计算共识序列时允许发生冲突",
    "Use alternative collisions routine": "使用替代碰撞例程",
    "Print residue scores": "打印残基分数",
    "Set page to Portrait": "设置页面为纵向",
    "Add a ruler": "添加一个标尺",
    "Number translations": "数字翻译",
    "Number DNA sequence": "DNA序列数量",
    "Width of screen": "屏幕宽度",
    "Main sequences": "主序列",
    "Primer file": "引物文件",
    "Allowed percent mismatch": "允许的百分比不匹配",
    "Reverse the sequence": "反转序列",
    "Complement the sequence": "补全序列",
    "Use feature information": "使用特征信息",
    "Read one sequence and stop": "读取一个序列并停止",
    "Feature source to match": "特征来源匹配",
    "Feature type to match": "特征类型匹配",
    "Feature tags to match": "特征标签匹配",
    "Tag values to match": "标签值匹配",
    "Regions to annotate by marking": "标记注释区域",
    "Display the ID name of the sequence": "显示序列的ID名称",
    "Display the description of the sequence": "显示序列的描述",
    "Display the scale line": "显示比例尺",
    "Screen width": "屏幕宽度",
    "Collapse features": "合并特征",
    "Display forward sense features": "显示前向义项特征",
    "Display reverse sense features": "显示反向意义特征",
    "Display unknown sense features": "显示未知的语义特征",
    "Display the strand of the features": "显示特征的链",
    "Display the source of the features": "显示特征的来源",
    "Display the start and end position of the features": "显示特征的起始和终止位置",
    "Display the type of the features": "显示特征的类型",
    "Display the tags and values of the features": "显示特征的标签和值",
    "Display the tag values of the features": "显示特征的标签值",
    "Display only those tag/value pairs in a feature that match the specified tag and value": "只显示特征中与指定标签和值匹配的标签/值对",
    "Number of shuffles": "洗牌次数",
    "Minimum scoring weight value for the predicted cleavage site": "预测切割位点的最小得分权重值",
    "Specifies the sequence is prokaryotic and changes the default scoring data file": "指定序列为原核生物，并更改默认的评分数据文件",
    "Select only the 21 base probes that start with a purine (Pol III expression vectors)": "选择仅以嘌呤开头的21个碱基探针（Pol III表达载体）",
    "Select only those 23 base regions that start with AA": "选择以AA开头的23个碱基区域",
    "Select only those 23 base regions that end with TT": "选择以TT结尾的23个碱基区域",
    "Report more than those 23 base regions that have no repeat of 4 or more of any bases in a row": "报告那些连续4个或更多相同碱基重复的23个碱基区域之外的区域。",
    "Displays the whole 23 bases of the region with the first two bases in brackets": "显示整个23个碱基的区域，前两个碱基用括号括起来",
    "Count the beginning of a sequence as a possible ORF": "将序列的起始位置作为可能的开放阅读框（ORF）进行计数。",
    "Count the end of a sequence as a possible ORF": "将序列末端计为一个可能的ORF（开放阅读框）",
    "Displays only ORFs starting with an M": "仅显示以M开头的开放阅读框",
    "Display the translation of the DNA sequence in the 3 reverse frames": "显示DNA序列在3个反向框架中的翻译",
    "Minimum size of Open Reading Frames (ORFs) to display in the translations": "在翻译中显示开放阅读框（ORFs）的最小大小",
    "Regions to put in uppercase": "需要大写的区域",
    "Number the sequence at the beginning and the end of each line": "在每行的开头和结尾对序列进行编号",
    "Number of nucleotides displayed on each line": "每行显示的核苷酸数",
    "Line length of page": "页面行长",
    "Margin around sequence for numbering": "序列编号周围的边距",
    "Number from which you want the DNA sequence to be numbered": "您希望DNA序列从哪个数字开始编号",
    "Number of sequences to skip at start": "在开始处要跳过的序列数量",
    "Size to split at": "拆分大小",
    "Overlap between split sequences": "拆分序列之间的重叠",
    "Add overlap to size": "增加重叠到大小",
    "Large sequences": "大序列",
    "Sequences to match": "序列匹配",
    "Word length for initial matching": "初始匹配的词长",
    "Show common codon usage": "显示常见密码子使用情况",
    "Minimum value for a common codon": "一个常见密码子的最小值",
    "Pattern to search for": "搜索模式",
    "Do a case-sensitive search": "进行区分大小写的搜索",
    "Display accession column": "显示访问列",
    "Display name column": "显示名称列",
    "Display description column": "显示描述列",
    "Nucleic sequences": "核酸序列",
    "Regions to translate": "需要翻译的区域",
    "Remove all 'X' and '*' characters from the right end of the translation": "从翻译的右侧移除所有的'X'和'*'字符。",
    "Change all STOP codon positions from the '*' character to 'X'": "将所有终止密码子位置从'*'字符改为'X'。",
    "Define frame '-1' as using the set of codons starting with the last codon of the sequence": "将“-1”帧定义为使用以序列中最后一个密码子开始的密码子集合",
    "Minimum length that a poly-A (or poly-T) tail must have before it is removed": "一个聚腺苷酸（或聚胸苷酸）尾巴在被移除之前必须具有的最小长度",
    "Number of fewer mismatched non-A bases in a poly-A tail": "在聚腺苷酸尾部的较少不匹配非A碱基的数量",
    "Change the sequence to the forward sense when it is written out": "将序列书写为正向序列",
    "Mask poly-A by converting to lowercase": "将聚腺苷酸（poly-A）序列转换为小写字母以进行掩蔽。",
    "Inspect 5' end of the sequence for poly-T tails": "检查序列的5'端是否有多聚胸苷酸尾部",
    "Threshold of the percentage ambiguity": "歧义百分比阈值",
    "Trim all ambiguity codes": "修剪所有模糊代码",
    "In protein sequences, trim off not only X's, but also the *'s": "在蛋白质序列中，不仅需要去掉X，还需要去掉星号(*)。",
    "Trim at the start": "在开始处修剪",
    "Trim at the end": "在末端修剪",
    "Feature type you wish to allow. Feature 1": "您希望允许的特征类型。特征 1",
    "Feature type you wish to allow. Feature 2": "您希望允许的特征类型。特征 2",
    "Minimun range": "最小范围",
    "Maximum range": "最大范围",
    "Feature source 1": "特征来源 1",
    "Feature sense 1": "特征 正向 1",
    "Feature 1 minimum score": "特征1最小分数",
    "Feature1 maximum score": "特征1 最大分数",
    "Feature 1 tags": "特征 1 标签",
    "Feature 1 tag values": "特征1标签值",
    "Feature 2 source": "特征2来源",
    "Feature 2 sense": "特征 2 正向",
    "Feature 2 minimum score": "特征 2 最小分数",
    "Feature 2 maximum score": "特征 2 最大分数",
    "Feature 2 tags": "特征 2 标签",
    "Feature 2 tag values": "特征 2 标签值",
    "Allowed overlaps": "允许的重叠",
    "How to determine range": "如何确定范围",
    "Required sense": "必需的正向",
    "Required order of the two features": "两个特征的必需顺序",
    "Write out the two features themselves": "写出这两个特征本身",
    "New feature type": "新特性类型",
    "Vector file": "向量文件",
    "Max allowed percent mismatch": "最大允许的百分比不匹配",
    "Show only the best hits (minimize mismatches)": "只显示最佳匹配结果（最小化不匹配）",
    "The 5' sequence": "5'序列",
    "The 3' sequence": "3'序列",
    "Window size, in codons": "窗口大小，以密码子为单位",
    "Bases used": "使用的碱基",
    "Output feature 1 file format": "输出特征1文件格式",
    "Output feature 2 file format": "输出特征2文件格式",
    "BLAST results as XML": "BLAST 结果为 XML",
    "Standard columns": "标准列",
    "Extended columns": "扩展列",
    "Get taxids by": "通过...获取分类ID",
    "Taxon names": "分类群名称",
    "Taxon ids": "物种ID",
    "Type of identifier list": "标识符列表类型",
    "Sequence identifier(s)": "序列标识符",
    "Type of BLAST": "BLAST 类型",
    "Filter out low complexity regions (with DUST)": "过滤掉低复杂性区域（使用DUST）",
    "Percent identity cutoff": "相似度百分比截止值",
    "Perform ungapped alignment only?": "执行无间隙对齐吗？",
    "Multiple hits window size: use 0 to specify 1-hit algorithm, leave blank for default": "多重比对窗口大小：使用0指定1-hit算法，留空为默认值",
    "Compute locally optimal Smith-Waterman alignments": "计算局部最优的Smith-Waterman比对",
    "Masked FASTA file": "掩蔽的FASTA文件",
    "Used masking algorithm": "使用掩蔽算法",
    "Masking algorithm options to create the masked input": "掩蔽算法选项，用于创建掩蔽输入",
    "DUST window length": "DUST窗口长度",
    "DUST linker": "DUST连接器",
    "Molecule type of input": "输入分子类型",
    "Input FASTA files(s)": "输入FASTA文件（们）",
    "Title for BLAST database": "BLAST数据库标题",
    "Parse the sequence identifiers": "解析序列标识符",
    "Enable the creation of sequence hash values": "启用创建序列哈希值",
    "Optional ASN.1 file(s) containing masking data": "可选的ASN.1文件（们），包含掩蔽数据",
    "Taxonomy options": "分类学选项",
    "NCBI taxonomy ID": "NCBI 分类学 ID",
    "Input PSSM files(s)": "输入PSSM文件（们）",
    "Title for the profile database": "数据库的标题",
    "Minimum word score to add a word to the lookup table": "将单词添加到查找表中的最小单词得分",
    "Does your input file contain PSSM scores?": "您的输入文件是否包含PSSM分数？",
    "Cost to open a gap": "打开间隙的成本",
    "Cost to extend a gap": "延伸间隙的代价",
    "PSSM scale factor": "PSSM 缩放因子",
    "Exclude invalid domains?": "排除无效的域？",
    "Observation threshold": "观察阈值",
    "SEG window length": "SEG窗口长度",
    "SEG low cutoff": "SEG低截止值",
    "SEG high cutoff": "SEG 高截断值",
    "Genetic code used for translation of query in BLASTX mode": "用于BLASTX模式下查询翻译的遗传密码",
    "ignore translated sequences without an open reading frame of at least this length": "忽略没有至少这个长度的开放阅读框的已翻译序列",
    "query strands to search": "查询链以搜索",
    "Allow for frameshifts?": "允许移码突变？",
    "restrict hit culling to overlapping query ranges": "限制命中剔除到重叠的查询范围",
    "frame shift penalty": "移码惩罚",
    "Composition based statistics": "基于组成的统计",
    "Suppress reporting of identical self-hits between sequences": "抑制报告序列之间相同的自匹配",
    "Input query file in FASTA or FASTQ format": "输入查询文件为FASTA或FASTQ格式",
    "Will you select a reference database from your history or use a built-in index?": "您将从历史记录中选择一个参考数据库，还是使用内置索引？",
    "Select a reference database": "选择一个参考数据库",
    "Select the reference database": "选择参考数据库",
    "Restrict search taxonomically?": "是否按分类学限制搜索？",
    "Comma separated list of taxon ids": "逗号分隔的分类群ID列表",
    "Keep alignments within the given percentage range of the top alignment score for a quer": "保持比对结果在查询序列顶部比对得分的给定百分比范围内",
    "Sensitivity Mode": "敏感性模式",
    "Scoring matrix": "评分矩阵",
    "Masking algorithm": "掩蔽算法",
    "Method to filter?": "过滤方法？",
    "Maximum expected value to keep an alignment": "最大期望值以保留一个比对",
    "Minimum bit score to keep an alignment": "保持比对的最小比特分数",
    "Run multiple rounds of searches with increasing sensitivity": "进行多轮搜索，逐渐提高灵敏度",
    "Algorithm for seed search": "种子搜索算法",
    "Limit on the number of Smith Waterman extensions": "限制Smith-Waterman扩展的数量",
    "Minimum identity percentage to report an alignment": "报告比对的最小同一性百分比",
    "Minimum query cover percentage to report an alignment": "报告比对的最小查询覆盖百分比",
    "Minimum subject cover percentage to report an alignment": "报告比对的最小主体覆盖百分比",
    "Maximum number of HSPs": "最大HSPs数",
    "Output aligned/unaligned queries to separate file": "将对齐/未对齐的查询输出到单独的文件",
    "Set a complexity cutoff for indexed seeds": "设置索引种子的复杂度截止值",
    "Enable masking seeds based on frequency": "启用基于频率的种子掩蔽",
    "Softmask abundant motifs": "使用软掩码丰富的模式",
    "Input reference file in FASTA format": "输入参考文件为FASTA格式",
    "Add taxonomic data?": "添加分类学数据？",
    "Protein accession to taxid mapping file": "蛋白质访问号到分类ID映射文件",
    "Taxonomy nodes.dmp from NCBI": "来自NCBI的分类节点.dmp文件",
    "Taxonomy names.dmp from NCBI": "来自NCBI的分类学名称.dmp文件",
    "input file in DAA format": "输入文件为DAA格式",
    "only show alignments of forward strand": "只显示正链的比对结果",
    "Query file": "查询文件",
    "Query mate file (optional)": "查询配对文件（可选）",
    "Subject database/sequences": "主题数据库/序列",
    "Nucleotide blast database": "核苷酸BLAST数据库",
    "Nucleotide fasta subject file to use instead of a database": "核苷酸fasta主题文件，用于代替数据库",
    "General search": "通用搜索",
    "Minimum number of consecutive bases matching exactly": "最小连续匹配碱基数",
    "Penalty for a nucleotide mismatch": "核苷酸错配的惩罚",
    "Maximum allowed intron length": "最大允许内含子长度",
    "Query filtering": "查询过滤",
    "Use lower case filtering in subject sequences?": "在主题序列中使用小写字母过滤？",
    "Reject low quality sequences?": "是否排除低质量序列？",
    "Remove word seeds with high frequency in the searched database?": "移除在搜索数据库中高频出现的词种子？",
    "Words that appear more than this number of times in the database will be masked in the lookup table": "在数据库中出现次数超过这个数值的单词将在查找表中被屏蔽",
    "Number of words to skip after collecting one while creating a lookup table": "在创建查找表时，收集到一个单词后要跳过的单词数量",
    "Restrict database search": "限制数据库搜索",
    "Tabular file containing list of GIs to which to restrict database search": "包含要限制数据库搜索的基因标识符（GIs）列表的表格文件",
    "Tabular file containing list of GIs to restrict database search to everything except the specified GIs": "包含GI列表的表格文件，用于将数据库搜索限制为除指定GI之外的所有内容",
    "Tabular file containing list of SeqIDs to which to restrict database search": "包含要限制数据库搜索的序列ID列表的表格文件",
    "Tabular file containing list of SeqIDs to restrict database search to everything except the specified SeqIDs": "包含序列ID列表的表格文件，用于限制数据库搜索范围，排除指定的序列ID。",
    "Comma-separated list of taxonomy IDs to which to restrict database search": "逗号分隔的分类学ID列表，用于限制数据库搜索范围",
    "Tabular file containing list of taxonomy IDs to which to restrict database search": "包含限制数据库搜索的分类学ID列表的表格文件",
    "Comma-separated list of taxonomy IDs to restrict database search to everything except the specified taxonomy IDs": "逗号分隔的分类学ID列表，用于限制数据库搜索范围，排除指定的分类学ID",
    "Tabular file containing list of taxonomy IDs to restrict database search to everythin except the specified taxonomy IDs": "包含限制数据库搜索到除指定分类学ID之外的所有内容的分类学ID列表的表格文件",
    "Cutoff score for accepting alignments": "接受比对的截止分数",
    "Cutoff edit distance for accepting an alignment": "接受比对的截止编辑距离",
    "Search for spliced alignments?": "搜索剪接对齐？",
    "Type of the reference": "参考类型",
    "Report unaligned reads?": "报告未比对的读取？",
    "Output unaligned reads to a separate file?": "输出未比对的序列到一个单独的文件？",
    "Output format for unaligned reads": "未比对读取的输出格式",
    "Include MD tag in BAM output?": "在BAM输出中包含MD标签？",
    "Do not trim '.1', '/1', '.2', or '/2' at the end of read ids in BAM output for paired reads?": "不要在BAM输出中为成对的读取修剪读取ID末尾的'.1', '/1', '.2'或'/2'吗？",
    "Suppress discordant alignments for paired reads?": "抑制配对读取中的不一致比对？",
    "Genes/proteins from species A": "物种A的基因/蛋白质",
    "Genes/proteins from species B": "物种B的基因/蛋白质",
    "Molecule type of FASTA inputs": "FASTA 输入的分子类型",
    "Minimum percentage identity for BLAST matches": "BLAST匹配的最小百分比一致性",
    "Minimum percentage query coverage for BLAST matches": "BLAST匹配的最小百分比查询覆盖率",
    "Process input FASTA files to collapse identical sequences": "处理输入的FASTA文件，将相同的序列折叠起来",
    "BLAST results as tabular": "BLAST 结果以表格形式展示",
    "Column containing query ID (qseqid)": "包含查询序列ID（qseqid）的列",
    "Column containing match ID (sseqid)": "包含匹配ID（sseqid）的列",
    "Column containing containing descriptions (salltitles)": "包含描述的列（salltitles）",
    "Number of descriptions": "描述数量",
    "Select input": "选择输入",
    "FASTA input": "FASTA 输入",
    "Input is a": "输入是一个",
    "Installed genome": "已安装基因组",
    "Choose method to apply": "选择应用方法",
    "Multiplies a scaling factor to each entry of the expected matrix to take care of the proximity ligation": "将一个缩放因子乘以预期矩阵的每个条目，以考虑邻近连接。",
    "Computation per chromosome": "每个染色体的计算",
    "Chromosomes to include in the computation": "包含在计算中的染色体",
    "Hi-C matrix": "Hi-C 矩阵",
    "Use a self-trained classifier": "使用一个自训练的分类器",
    "Sam/Bam files to process (forward/reverse)": "Sam/Bam 文件处理（正向/反向）",
    "Lines to analyze for the QC report": "用于QC报告分析的行",
    "Reference point": "参考点",
    "Create bedgraph file with interactions": "创建一个包含相互作用的 bedgraph 文件。",
    "Short range distance": "短程距离",
    "Colorlist": "颜色列表",
    "Normalize matrices": "标准化矩阵",
    "Multiplicative value": "乘法值",
    "Set values below this threshold to zero": "将低于此阈值的值设置为零",
    "Protein peak file in bed format": "蛋白质峰值文件，采用bed格式",
    "Maximum number of loops": "最大循环次数",
    "Resolution of the matrix": "矩阵的解析度",
    "Add, remove, or keep the chr-prefix of the loop/TAD data": "添加、移除或保留循环/TAD数据中的chr前缀",
    "Hyperopt runs": "Hyperopt 运行",
    "Peak width": "峰宽",
    "P-value preselection": "P值预选",
    "Minimum interaction number": "最小交互数",
    "Obs/exp interaction threshold": "观测值/期望值交互作用阈值",
    "Maximal loop distance": "最大环距离",
    "Chromosomes to include": "要包含的染色体",
    "Method to compute the expected value": "计算期望值的方法",
    "Correlation method to use": "使用的相关性方法",
    "Use the log1p of the matrix values": "使用矩阵值的log1p",
    "List of chromosomes to be included in the correlation": "包含在相关性分析中的染色体列表",
    "chromosome (one per field)": "染色体（每字段一个）",
    "Eigenvector file": "特征向量文件",
    "Offset values": "偏移值",
    "Output matrix?": "输出矩阵？",
    "The aggregated data file created by chicAggregateStatistic": "由 chicAggregateStatistic 创建的聚合数据文件",
    "Statistical test": "统计测试",
    "Minimum read coverage": "最小读取覆盖度",
    "Minimum density": "最小密度",
    "Maximum region to consider": "最大考虑区域",
    "List of chromosomes to consider": "要分析的染色体列表",
    "DPI for image": "图像的每英寸点数（DPI）",
    "Interaction file order": "交互文件顺序",
    "Rejected H0 file order": "拒绝 H0 文件顺序",
    "Significant file order": "重要文件顺序",
    "Plot significant interactions": "绘制显著交互作用",
    "Upstream range": "上游范围",
    "Downstream range": "下游范围",
    "Background model": "背景模型",
    "Combination mode for the plots": "绘图组合模式",
    "Max p-value": "最大 P 值",
    "Min p-value": "最小P值",
    "Plot p-values as a colorbar": "将 P 值绘制为颜色条",
    "X-fold mean background": "X折均值背景",
    "Color list": "颜色列表",
    "Truncate zeros by setting them to one.": "通过将零设置为一来截断零。",
    "The interaction data file created by chicViewpoint": "由 chicViewpoint 创建的交互数据文件",
    "List of chromosomes or a BED file containg regions": "染色体列表或包含区域的BED文件",
    "The target data file created by chicSignificantInteractions": "由 chicSignificantInteractions 创建的目标数据文件",
    "P-value of interaction to be accepted as significant": "交互作用的P值被接受为显著的",
    "X-fold over mean background to be accepted.": "X倍背景均值被接受。",
    "Loose p-value": "宽松 p 值",
    "Minimum interactions": "最小相互作用",
    "Process matrices independently or combine always two (for differential analysis preparation)": "独立处理矩阵或始终组合两个（用于差异分析准备）",
    "Clip background distance": "剪切背景距离",
    "Reference points": "参考点",
    "Average contacts window size": "平均接触窗口大小",
    "Hi-C corrected matrices": "Hi-C校正矩阵",
    "Exclude diagonal counts": "排除对角线计数",
    "Generate plots per chromosome": "生成每个染色体的图表",
    "Max Depth": "最大深度",
    "Plot size": "绘图大小",
    "Chromosomes to exclude": "排除的染色体",
    "Bed file with domains coordinates": "包含域坐标的Bed文件",
    "Save data underlying the plots as BAM": "将图表背后的数据保存为BAM格式",
    "Output file format": "输出文件格式",
    "Eigenvectors to compute": "计算特征向量",
    "This option removes the masked bins before the PCA is computed": "这个选项在计算PCA之前移除掩蔽的bin",
    "Extra track type": "额外的轨道类型",
    "Correlate PCA with e.g. gene density or histon marks to flip sign": "将PCA与基因密度或组蛋白标记相关联，以翻转符号。",
    "Histon mark type": "组蛋白修饰类型",
    "Return internally used Pearson matrix": "返回内部使用的皮尔逊矩阵",
    "Return internally used observed / expected matrix": "返回内部使用的观察到的/预期的矩阵",
    "Loop file": "循环文件",
    "Validation data": "验证数据",
    "Validate loop or TAD data": "验证环或拓扑关联域（TAD）数据",
    "Used resolution": "使用分辨率",
    "Add, remove, or keep the chr-prefix of the validation data": "添加、删除或保留验证数据的chr前缀",
    "Loop files to merge": "循环文件以合并",
    "Lowest used resolution": "最低使用的分辨率",
    "Regions to average": "区域平均",
    "Define start value of range": "定义范围的起始值",
    "Consider strand direction": "考虑链方向",
    "Compute from scratch or use precomputed data": "从头计算或使用预计算数据",
    "Minimum window length (in bp) to be considered to the left and to the right of each Hi-C bin.": "每个Hi-C bin左右要考虑的最小窗口长度（以碱基对为单位）。",
    "Maximum window length (in bp) to be considered to the left and to the right of each Hi-C bin.": "每个Hi-C bin 左右考虑的最大窗口长度（以碱基对为单位）。",
    "Step size when moving from minDepth to maxDepth": "从最小深度移动到最大深度时的步长",
    "Precomputed TAD score file (bm)": "预计算的TAD得分文件（bm）",
    "Precomputed z-score matrix": "预计算的z分数矩阵",
    "Multiple Testing Corrections": "多重检验校正",
    "q-value": "q值",
    "Minimum threshold of the difference between the TAD-separation score of a putative boundary and the mean of the TAD-sep. score of surrounding bins.": "TAD分离分数的假定边界与周围箱的TAD分离分数的平均值之间的最小差异阈值。",
    "Minimum distance between boundaries (in bp).": "边界之间的最小距离（以碱基对为单位）。",
    "Number of bins to merge": "要合并的箱数",
    "Set to merge for using a running window of length --numBins. Usually not set.": "设置为使用长度为--numBins的运行窗口进行合并。通常不设置。",
    "Choose output matrix type": "选择输出矩阵类型",
    "Choose input file type": "选择输入文件类型",
    "To be stored resolutions": "待存储的分辨率",
    "Store applied correction factors in column count": "在列计数中存储应用的校正因子",
    "Enforce integer for count column": "强制计数列为整数",
    "Apply action": "应用操作",
    "Chromosomes to keep / remove / mask": "染色体保留/移除/掩蔽",
    "BED file with all regions to be considered": "包含所有要被考虑区域的BED文件",
    "Remove inter- or intra chromosomal regions": "去除染色体间或染色体内的区域",
    "Minimum distance between restriction sites": "限制性位点之间的最小距离",
    "Maximum library insert size defines different cut offs based on the maximum expected library size": "最大文库插入大小根据预期的最大文库大小定义不同的截止值",
    "Keep self circles": "保持自我循环",
    "Keep self ligation": "保持自环化",
    "Skip duplication check": "跳过重复检查",
    "Chromosome sizes for your genome": "你的基因组的染色体大小",
    "Save valid Hi-C reads in BAM file": "保存有效的Hi-C读取结果到BAM文件中",
    "Fasta file for the organism genome.": "该生物体基因组的Fasta文件。",
    "Restriction enzyme sequence": "限制酶序列",
    "Truncate zeros": "截断零",
    "Sparsity threshold": "稀疏性阈值",
    "Cluster file": "聚类文件",
    "Maximal distance": "最大距离",
    "Order samples by": "按样本排序",
    "Distance short range": "距离短程",
    "Distance long range": "长距离",
    "Running window": "运行窗口",
    "Raw fastq file from Nagano 2017.": "来自2017年长野的原始fastq文件。",
    "barcode file": "条形码文件",
    "SRR to Sample mapping file": "SRR到样本映射文件",
    "Buffer size": "缓冲区大小",
    "Cluster method:": "聚类方法：",
    "Number of hash functions": "哈希函数的数量",
    "Number of nearest neighbors": "最近邻数量",
    "The MinHash algorithm computes additional the exact euclidean distance.": "MinHash算法计算额外的确切欧几里得距离。",
    "Use different expected value computation": "使用不同的期望值计算方法",
    "Convert positive values to 1 and all negative to -1.": "将正值转换为1，所有负值转换为-1。",
    "Correlate PCA with e.g. gene density or histone marks to flip sign": "将PCA（主成分分析）与基因密度或组蛋白标记等进行相关性分析，以翻转符号。",
    "Apply dimension reduction:": "应用降维：",
    "Action to take apply on matrices:": "需要对矩阵采取的行动：",
    "Datatrack output type": "Datatrack 输出类型",
    "Chromosome size": "染色体大小",
    "Create only one target file": "创建一个目标文件",
    "Decimal places": "小数位数",
    "Target matrix to compute on": "目标矩阵计算",
    "Control matrix to compute on": "控制矩阵计算",
    "TAD domains file": "TAD 域文件",
    "Method to compute the differentail TAD expression": "计算不同TAD表达的方法",
    "Bed file with TAD domains coordinates": "带有TAD（拓扑关联结构域）坐标的Bed文件",
    "Domain files from hicFindTADs of different resolutions to include": "来自不同分辨率的 hicFindTADs 的领域文件以包含",
    "Minimum number of peaks": "最小峰值数目",
    "TAD seperation value": "TAD 分离值",
    "Percentage overlap": "百分比重叠",
    "Domain file or list of files containing tad boundaries": "包含TAD边界的域文件或文件列表",
    "Consider protein locations": "考虑蛋白质位置",
    "Protein file in bed format": "蛋白质文件，采用bed格式",
    "Leniency": "宽容度",
    "Configure ML model options": "配置机器学习模型选项",
    "Unselect border cases": "取消选择边缘案例",
    "Concatenate before resample": "在重新采样之前进行连接",
    "Use cleanlab": "使用Cleanlab",
    "Estimators per step": "每步估计器",
    "Distance": "距离",
    "Width": "宽度",
    "Show x labels": "显示x轴标签",
    "Set 'type' to 'lines'": "将 'type' 设置为 'lines'",
    "Border color": "边框颜色",
    "Global max rows": "全局最大行数",
    "Gene rows": "基因行",
    "Choose style of the plot.": "选择绘图的风格。",
    "Configure x-axis": "配置 x 轴",
    "Where to place the x-axis": "放置x轴的位置",
    "Input genotype data file": "输入基因型数据文件",
    "Discard loci?": "丢弃位点？",
    "Discard loci file": "丢弃位点文件",
    "SNP genotypes matrix data": "SNP基因型矩阵数据",
    "Only estimate F-stats (no selection)": "仅估计F统计量（无选择）",
    "Number of outputted iterations": "输出迭代次数",
    "Thinning interval size": "稀疏间隔大小",
    "Number of pilot runs": "试点运行次数",
    "Length of pilot runs": "引导运行的长度",
    "Additional burn-in length": "额外的预热长度",
    "Prior odds for the neutral model": "中性模型的先验概率",
    "Lower bound for uniform prior on Fis (dominant data)": "Fis（显性数据）的均匀先验的下限",
    "Higher bound for uniform prior on Fis (dominant data)": "Fis（显性数据）上的均匀先验的上限",
    "Threshold for the recessive genotype as a fraction of maximum band intensity": "隐性基因型作为最大条带强度分数的阈值",
    "Optional output file for pilot runs": "可选的引导运行输出文件",
    "Optional output file for allele frequencies": "可选的等位基因频率输出文件",
    "Output from previous Stacks pipeline steps (e.g. denovo_map or refmap)": "来自之前 Stacks 流水线步骤的输出（例如 denovo_map 或 refmap）",
    "Files containing reads to assemble": "包含待组装读段的文件",
    "White list of catalog IDs to include": "包含目录ID的白名单",
    "Minimum number of reads by locus": "每个位点的最小读取数",
    "Perform assembly with Velvet": "使用Velvet进行组装",
    "Minimum length for asssembled contigs": "组装后的连续序列的最小长度",
    "Output from previous Stacks pipeline steps (e.g. denovo_map, refmap, ustacks or pstacks)": "来自之前Stacks流程步骤的输出（例如denovo_map、refmap、ustacks或pstacks）",
    "Population map": "种群图",
    "Base catalog matching on genomic location, not sequence identity": "基于基因组位置匹配基础目录，而不是序列相似性",
    "Number of mismatches allowed between sample tags when building the catalog": "在构建目录时，允许样本标签之间存在的错误匹配数量",
    "Include tags in the catalog that match to more than one entry": "在目录中包括匹配到多个条目的标签",
    "Perform gapped alignments between stacks": "执行栈之间的间隔对齐",
    "Number of gaps allowed between stacks before merging": "允许在合并之前堆栈之间存在的最大间隙数",
    "Minimum length of aligned sequence in a gapped alignment": "在有间隙的比对中，比对序列的最小长度",
    "Do not trim low quality reads, just discard them": "不要修剪低质量的序列，直接丢弃它们",
    "Raw reads are circularized mate-pair data, first read will be reverse complemented": "原始读取是环形化配对数据，第一个读取将被反转互补",
    "Data does not contain an overhang nucleotide between barcode and seqeunce": "数据中在条形码和序列之间不包含突出的核苷酸",
    "Capture discarded reads to a file": "将丢弃的读取捕获到文件中",
    "Filtering options": "过滤选项",
    "Turn on filtering based on rare k-mers": "开启基于稀有k-mers的过滤",
    "Turn on filtering based on abundant k-mers": "启用基于丰富k-mers的过滤",
    "Advanced fitering options": "高级过滤选项",
    "Number of times a kmer must occur to be considered abundant": "一个kmer必须出现的次数，才能被认为是丰富的",
    "Number of rare kmers occuring in a row required to discard a read": "在一行中连续出现的罕见k-mer的数量，需要达到这个数量才会丢弃一个读取。",
    "Number of abundant kmers required to discard a read": "需要丢弃一个读段所需的丰富k-mers数量",
    "Normalization options": "标准化选项",
    "Normalize read depth according to k-mer coverage": "根据k-mer覆盖度标准化读取深度",
    "Characterizing K-mers options": "K-mers 特性选项",
    "Write kmers along with their frequency of occurrence and exit": "输出k-mer及其出现频率并退出",
    "Print k-mer frequency distribution and exit": "打印k-mer频率分布并退出",
    "Length of the single-end oligo sequence in dataset": "单端寡核苷酸序列在数据集中的长度",
    "Length of the paired-end oligo sequence in dataset": "配对末端寡核苷酸序列在数据集中的长度",
    "Do not trim off the random oligo sequence (if oligo is inline)": "不要剪掉随机寡核苷酸序列（如果寡核苷酸是内嵌的）",
    "Paired end options": "配对末端选项",
    "Variant calling options (for gstacks)": "变异检测选项（用于 gstacks）",
    "Assembly options": "组装选项",
    "Number of mismatches allowed between loci when processing a single individual": "在处理单个个体时，允许位点之间的不匹配数量",
    "Number of mismatches allowed between loci when building the catalog": "在构建目录时，允许位点之间存在的错误匹配数",
    "SNP model options": "SNP模型选项",
    "Paired-end options": "配对末端选项",
    "remove all but one set of read pairs of the same sample that have the same insert length": "删除同一样本中插入长度相同的所有读段对，只保留一组。",
    "Population filtering options": "种群过滤选项",
    "minimum percentage of individuals in a population required to process a locus for that population": "在处理一个位点时，所需的一个种群中个体的最小百分比",
    "minimum number of populations a locus must be present in to process a locus": "最小种群数量，一个位点必须存在于其中才能处理该位点",
    "Singles-end reads": "单端测序读取",
    "Number of enzymes": "酶的数量",
    "Enzyme": "酶",
    "Second enzyme": "第二个酶",
    "Library was generated using BestRAD, check for restriction enzyme on either read and potentially tranpose reads": "文库是使用BestRAD生成的，检查限制性内切酶在任一读取上，并可能转置读取。",
    "Disable checking if the RAD site is intact": "禁用检查RAD位点是否完整",
    "Specify a population map": "指定一个种群图",
    "Number of loci to process in a batch": "一批处理的位点数",
    "Data filtering options": "数据过滤选项",
    "Minimum percentage of individuals in a population required to process a locus for that population": "在人群中处理一个位点所需的最小个体百分比",
    "Minimum number of populations a locus must be present in to process a locus": "一个位点必须存在于最小数量的种群中才能处理该位点",
    "minimum percentage of individuals across populations required to process a locus": "在处理一个位点时，所需的最小个体百分比跨种群",
    "apply the above filters haplotype wise": "应用上述过滤器，以单倍型为单位",
    "Minimum minor allele frequency": "最小次等位基因频率",
    "Minimum minor allele count": "最小次等位基因计数",
    "Maximum observed heterozygosity required to process a nucleotide site at a locus.": "在位点上处理一个核苷酸位点所需的最大观察到的杂合性。",
    "Analyse all SNPs per locus": "分析每个位点的所有SNPs",
    "Specify a file containing markers to include in the export": "指定一个包含要导出标记的文件",
    "Specify a file containing markers to exclude from the export": "指定一个包含要从导出中排除的标记的文件",
    "Merging and Phasing": "合并和分相",
    "Provide the restriction enzyme used": "提供所使用的限制性内切酶",
    "Merge loci that were produced from the same restriction enzyme cutsite": "合并来自同一限制性内切酶切割位点的位点",
    "Fraction of samples possessing both loci to prune remaining samples from analysis": "拥有两个位点的样本比例，用于从分析中修剪剩余样本",
    "Locus Stats": "位点统计",
    "Calculate divergence from Hardy-Weinberg equilibrium for each locus": "计算每个位点的哈迪-温伯格平衡偏离度。",
    "Enable SNP and haplotype-based F statistics": "启用基于SNP和单倍型的F统计量",
    "apply a p-value correction to Fst values based on a Fisher's exact test": "对基于费舍尔精确检验的Fst值应用p值校正",
    "If data is reference aligned, exports will be ordered; only a single representative of each overlapping site.": "如果数据是参照比对的，导出的结果是有序的；每个重叠位点只有一个代表性的实例。",
    "Output locus consensus sequences in FASTA format": "输出基因座的一致序列，使用FASTA格式。",
    "Output the sequences of the two haplotypes of each (diploid) sample, for each locus, in FASTA format": "输出每个（二倍体）样本在每个位点上的两个单倍型序列，在FASTA格式中",
    "Output results in Variant Call Format (VCF)": "输出结果为变异调用格式（VCF）",
    "Output results in GenePop Format": "输出结果为GenePop格式",
    "Output results in Structure Format": "输出结果为结构化格式",
    "Output results in fineRADstructure/RADpainter format": "输出结果为fineRADstructure/RADpainter格式",
    "Output genotypes in PLINK format": "输出PLINK格式的基因型",
    "Output genotypes in Hybrid Zone Analysis using R (HZAR) format.": "输出基因型，使用R语言的杂交带分析（HZAR）格式。",
    "Output nucleotides that are fixed-within, and variant among populations in Phylip format for phylogenetic tree construction": "输出在Phylip格式中固定于种群内、种群间变异的核苷酸，用于构建系统发育树。",
    "Include variable sites in the phylip output encoded using IUPAC notation.": "在phylip输出中包含变量位，并使用IUPAC编码表示。",
    "Output SNPs in a format useable for the TreeMix program (Pickrell and Pritchard).": "输出可用于TreeMix程序（Pickrell和Pritchard）的SNP格式。",
    "Omit haplotype outputs": "忽略单倍型输出",
    "Output all haplotypes observed in each sample, for each locus, in FASTA format": "输出每个样本中每个位点观察到的所有单倍型，以FASTA格式。",
    "output locus positions in a GTF annotation file": "在GTF注释文件中输出位点位置",
    "Kernel smoothing": "核平滑",
    "Enable kernel-smoothed calculations": "启用核平滑计算",
    "Standard deviation of the kernel smoothing weight distribution": "核平滑权重分布的标准差",
    "Enable bootstrap resampling for all smoothed statistics": "启用所有平滑统计的引导重采样",
    "Enable boostrap resampling for smoothed SNP-based Pi and Fis calculations": "启用自助重采样以平滑基于SNP的Pi和Fis计算",
    "Enable boostrap resampling for smoothed Fst calculations based on pairwise population comparison of SNPs": "启用基于SNP的成对种群比较的平滑Fst计算的bootstrap重采样。",
    "Enable boostrap resampling for smoothed haplotype diveristy and gene diversity calculations based on haplotypes": "启用基于单倍型的平滑单倍型多样性和基因多样性计算的引导重采样",
    "Enable boostrap resampling for smoothed Phi_st calculations based on haplotypes.": "启用基于单倍型的平滑Phi_st计算的自助重采样。",
    "Number of bootstrap resamplings to calculate": "进行自举重采样以计算的次数",
    "Only bootstrap loci contained in this whitelist": "仅包含在此白名单中的引导定位点",
    "Log components of Fst/Phi_st calculations to a dataset": "将 Fst/Phi_st 计算的组成部分记录到一个数据集中",
    "Genetic map output options (population map must specify a genetic cross)": "遗传图谱输出选项（群体图谱必须指定一个遗传杂交）",
    "genetic map type to write": "遗传图谱类型写入",
    "mapping program format to write": "将映射程序格式写入",
    "Ignore paired-end reads": "忽略配对末端的序列读段",
    "K-mer length for the de Bruijn graph": "de Bruijn图的K-mer长度",
    "Maximum number of reads to use in the de Bruijn graph": "在德布鲁因图中使用的最大读取数",
    "Minimum coverage to consider a kmer": "最小覆盖度以考虑一个kmer",
    "save read alignments": "保存读段对齐",
    "Discard unpaired reads": "丢弃未配对的读段",
    "Remove read pairs of the same sample that have the same insert length": "删除同一样本中插入长度相同的配对序列",
    "Minimum PHRED-scaled mapping quality to consider a read": "最小PHRED缩放的映射质量，以考虑一个读取",
    "Maximum soft-clipping level": "最大软裁剪水平",
    "Maximum allowed sequencing insert length": "最大允许的测序插入长度",
    "Write a heaview output": "写一个重量级输出",
    "Edge coverage min": "边缘覆盖最小值",
    "Edge coverage max": "边缘覆盖最大",
    "Don't try to ignore dubious heterozygote genotypes during phasing": "不要在分相过程中忽略可疑的杂合子基因型",
    "Model to use to call variants and genotypes": "用于调用变异和基因型的模型",
    "Add log distribs output as dataset": "将对数分布输出作为数据集添加",
    "Verify haplotype of matching locus": "验证匹配位点的单倍型",
    "Number of mismatches allowed between sample loci when building the catalog": "在构建目录时，允许样本位点之间存在的错误匹配数量",
    "K-mer size for matching between between catalog loci": "匹配目录位点之间K-mer大小",
    "Report query loci that match more than one catalog locus": "报告查询位点，如果它们匹配了目录中的多个位点。",
    "Minimum depth of coverage required to create a stack": "创建堆叠所需的最小覆盖深度",
    "Maximum distance (in nucleotides) allowed between stacks": "最大允许的堆叠之间的距离（以核苷酸为单位）",
    "Maximum distance allowed to align secondary reads to primary stacks": "允许将次要读取与主要堆栈对齐的最大距离",
    "Retain unused reads": "保留未使用的读取",
    "Disable calling haplotypes from secondary reads": "禁用从次要读取中调用单倍型",
    "SNP Model Options (ustacks options)": "SNP模型选项（ustacks选项）",
    "Disable the algorithm that removes highly-repetitive stacks and nearby errors.": "禁用移除高度重复的栈和附近错误的算法。",
    "Enable the Deleveraging algorithm, used for resolving over merged tags": "启用去杠杆算法，用于解决过度合并的标签",
    "Highly-repetitive stacks threshold": "高度重复序列堆栈阈值",
    "Maximum number of stacks at a single de novo locus": "单个de novo位点上的最大堆叠数",
    "K-mer size for matching between alleles and loci (automatically calculated by default)": "匹配等位基因和位点之间的K-mer大小（默认情况下自动计算）",
    "Processing options": "处理选项",
    "Start identifier at": "在开始标识符处",
    "Specify a minimum stack depth required for individuals at a locus": "为一个位点上的个体指定所需的最小堆叠深度",
    "Specify a minimum minor allele frequency required before calculating Fst at a locus (between 0 and 0.5)": "指定在计算一个位点的Fst值之前所需的最小次等位基因频率（介于0和0.5之间）",
    "Correction type": "校正类型",
    "Filter loci with log likelihood values below this threshold": "过滤掉对数似然值低于此阈值的位点",
    "Restrict data analysis to only the first SNP per locus.": "限制数据分析，只分析每个位点的第一个SNP。",
    "Restrict data analysis to one random SNP per locus.": "将数据分析限制为每个位点一个随机SNP。",
    "Output haplotypes in Variant Call Format (VCF).": "输出单倍型到变异调用格式（VCF）。",
    "Output full sequence for each unique haplotype, from each sample locus in FASTA format, regardless of plausibility.": "对于每个样本位点中的每个独特单倍型，输出完整序列，以FASTA格式，不考虑其可能性。",
    "Output full sequence for each haplotype, from each sample locus in FASTA format, only for biologically plausible loci.": "为每个样本位点的每个单倍型输出完整序列，在FASTA格式下，仅针对生物学上合理的位点。",
    "Output genotypes in PHASE format": "输出基因型，使用PHASE格式",
    "Output genotypes in fastPHASE format": "输出基因型，使用fastPHASE格式",
    "Output genotypes in Beagle format": "输出基因型，使用Beagle格式",
    "Output haplotypes in Beagle format": "输出在Beagle格式中的单倍型",
    "Include all sequence as well as variable sites in the phylip output encoded using IUPAC notation.": "在phylip输出中包含所有序列以及使用IUPAC编码表示的可变位点。",
    "Output each nucleotide position (fixed or polymorphic) in all population members to a file": "将所有群体成员中的每个核苷酸位置（固定或多态性）输出到一个文件中。",
    "enable kernel-smoothed FIS, π, and FST calculations": "启用核平滑的FIS、π和FST计算",
    "Bootstrap resampling": "自助重采样",
    "Specify a file containing Whitelisted markers to include in the export": "指定一个包含要包含在导出中的白名单标记的文件",
    "Specify a file containing Blacklisted markers to be excluded from the export": "指定一个包含要排除在导出之外的黑名单标记的文件",
    "Batch ID to examine when exporting from the catalog": "批次ID，用于从目录导出时检查",
    "Single or Paired-end": "单端或配对端",
    "Barcode location": "条形码位置",
    "Forward FASTQ": "正向FASTQ",
    "Reverse FASTQ": "反向 FASTQ",
    "length of the single-end oligo sequence in data set": "数据集中单端寡核苷酸序列的长度",
    "length of the paired-end oligo sequence in data set": "数据集中配对末端寡核苷酸序列的长度",
    "Program which generated the stacks": "生成堆栈的程序",
    "Log generated by the program": "由程序生成的日志",
    "Output from ustacks or pstacks": "ustacks 或 pstacks 的输出",
    "Catalog files": "目录文件",
    "Samples stacks": "样本堆栈",
    "Don't verify haplotype of matching locus": "不要验证匹配位点的单倍型",
    "Cross type": "交叉类型",
    "Output file type": "输出文件类型",
    "Restriction enzyme used": "限制性内切酶使用",
    "Make automated corrections to the data": "对数据进行自动校正",
    "Minimum number of reads required at a stack to call a homozygous genotype": "在堆栈上调用纯合基因型所需的最小读取数",
    "Heterozygote minor allele minimum frequency": "杂合子次等位基因最小频率",
    "Heterozygote minor allele maximum frequency": "杂合子次等位基因最大频率",
    "Minimum number of progeny required to print a marker": "最小后代数量，需要打印标记",
    "Minimum stack depth required before exporting a locus in a particular individual": "在导出特定个体中某一位点之前所需的最小堆栈深度",
    "Path to file containing manual genotype corrections from a Stacks SQL database to incorporate into output.": "包含从 Stacks SQL 数据库手动更正基因型的路径的文件，以纳入输出。",
    "Select your usage": "选择您的使用方式",
    "Files containing parent sequences": "包含父序列的文件",
    "Files containing progeny sequences": "包含后代序列的文件",
    "Files containing an individual sample from a population": "包含来自种群中个体样本的文件",
    "Minimum number of identical raw reads required to create a stack": "创建一个栈所需的相同原始读取的最小数量",
    "Minimum number of identical, raw reads required to create a stack in 'progeny' individuals": "在“后代”个体中创建一个堆叠所需的相同、原始读取的最小数量",
    "Number of mismatches allowed when aligning secondary reads": "在比对次级读段时允许的不匹配数",
    "Remove, or break up, highly repetitive RAD-Tags in the ustacks program": "在ustacks程序中移除或打破高度重复的RAD标签",
    "Input short reads from an individual": "从个体中输入短序列读取",
    "Single-end or paired-end reads files": "单端或双端测序文件",
    "singles-end reads infile(s)": "单端测序读段输入文件",
    "paired-end reads infile(s) 1": "配对末端读取文件（1）",
    "paired-end reads infile(s) 2": "配对末端读取文件（2个）",
    "Set the size of the sliding window as a fraction of the read length, between 0 and 1 (default 0.15)": "将滑动窗口的大小设置为读取长度的分数，介于0和1之间（默认为0.15）",
    "Clean data, remove any read with an uncalled base": "清理数据，移除任何包含未确定碱基的读取序列。",
    "Discard reads with low quality scores": "丢弃质量分数低的读取",
    "Set the score limit. If the average score within the sliding window drops below this value, the read is discarded (default 10)": "设置得分限制。如果滑动窗口内的平均得分低于此值，则丢弃该读取（默认值为10）",
    "Rescue barcodes and RAD-Tags?": "救援条形码和RAD标签？",
    "Truncate final read length to this value": "将最终的读取长度截断到这个值",
    "Retain unmodified FASTQ headers in the output": "在输出中保留未修改的FASTQ头部",
    "Files containing parent alignments": "包含父序列比对的文件",
    "Files containing progeny alignments": "包含后代比对的文件",
    "Minimum depth of coverage": "最小覆盖深度",
    "Minimum depth of coverage in 'progeny' individuals": "后代个体中的最小覆盖深度",
    "SNP_Model_Options": "SNP模型选项",
    "Print distribution of mean log likelihoods for catalog loci.": "打印目录位点的平均对数似然分布。",
    "Proportion of loci in population that must be confounded relative to the catalog locus (between 0.0 and 1.0).": "在人群中，必须与目录位点混淆的位点比例（介于0.0和1.0之间）。",
    "Prune out non-biological haplotypes unlikely to occur in the population.": "修剪掉不太可能在人群中出现的非生物学单倍型。",
    "Only consider haplotypes for pruning if they occur in fewer than this value.": "仅在考虑剪枝的单倍型出现次数少于该值时考虑。",
    "Give a unique numeric ID to this sample": "为这个样本分配一个唯一的数字ID",
    "Alignments with more than this fraction of soft-clipped bases are discarded": "超过这个比例的软剪切碱基的比对将被丢弃",
    "Minimum required mapping quality": "最小映射质量",
    "Keep secondary alignments": "保留次要比对",
    "SNP Model Options (pstacks options)": "SNP模型选项（pstacks选项）",
    "Multiple iterations": "多次迭代",
    "Number of current iteration": "当前迭代次数",
    "Blacklist hit overlap": "黑名单命中重叠",
    "maximum radius": "最大半径",
    "maximum distance relations": "最大距离关系",
    "Deactivate caching of kernel value computation  (-no-cache)": "禁用内核值计算的缓存（-no-cache）",
    "eccess neighbour size factor": "额外邻居大小因子",
    "Use shared neighbourhood to weight center density (-usn)": "使用共享邻域来加权中心密度（-usn）",
    " flag to output clusters (-oc)": "输出聚类结果的标志（-oc）",
    "Maximal number of hash functions": "最大哈希函数数量",
    "Size of step for increasing hash functions": "增加哈希函数的步长大小",
    "Maxinum number of clusters": "最大聚类数",
    "gspan file": "gspan 文件",
    "Sequence and Structure input file in the prediction tool format": "预测工具格式中的序列和结构输入文件",
    "  Sequence Structure format of the input. Allowed formats: vrna-simple, vrna-mea": "输入的序列结构格式。允许的格式：vrna-simple，vrna-mea",
    "  Sequence Structure type from the input to use. Allowed types: MFE, MEA": "输入序列结构类型以使用。允许的类型：MFE（最小自由能），MEA（平均能量）",
    "Add stacking information to graphs": "将堆叠信息添加到图表中",
    "Add abstract structure graphs to the single shrep graph instances.": "为单个shrep图实例添加抽象结构图。",
    "Add for each 't #' a graph which contains no structure": "为每个 't #' 添加一个不包含任何结构的图形。",
    "Group size": "组大小",
    "SHAPE data": "SHAPE 数据",
    "Alignments file": "比对文件",
    "window size": "窗口大小",
    "window shift in percent": "窗口位移百分比",
    "minimum sequence length": "最小序列长度",
    "fast_cluster": "快速聚类",
    "fasta_data": "FASTA数据",
    "fast_cluster_sim": "快速聚类相似度",
    "data_map": "数据映射",
    "centers": "中心",
    "trees": "树",
    "tree_matrix": "树状矩阵",
    "Allow overlap in subtrees": "允许子树重叠",
    "Choose the type of parameters": "选择参数类型",
    "minimal probability": "最小概率",
    " maximal difference for sizes of matched arcs": "匹配弧线大小的最大差异",
    "Sequence contribution at structure match in percent": "序列在结构匹配中的百分比贡献",
    "maximal difference for alignment traces": "最大对齐跟踪差异",
    "Minimal length of a sequences for which RNAplfold is used": "RNAplfold使用序列的最小长度",
    "Structure weight": "结构权重",
    "Indel opening score": "插入缺失打开分数",
    "Indel score": "插入缺失分数",
    "Compute consensus dot plot by alifold": "使用alifold计算共识点图",
    "Maximum basepair span by RNAplfold (local folding); -1 for global folding": "RNAplfold（局部折叠）的最大碱基对跨度；-1表示全局折叠",
    "Window size for local folding": "局部折叠的窗口大小",
    "Add stacking information to graphs.": "在图表中添加堆叠信息。",
    "Max number of shreps that should be taken per window.": "每个窗口应该取的最大shreps数。",
    " Relative energy range, i.e. percentage (%) of MFE energy (RNAshapes)": "相对能量范围，即 MFE 能量的百分比（%）（RNAshapes）",
    " A list of window sizes to use": "使用窗口大小的列表",
    "The shift of the window, relative to the window size given inpercent.": "窗口相对于给定窗口大小的百分比的偏移。",
    "Ignore unstable structures (RNAshapes)": "忽略不稳定结构（RNAshapes）",
    "Hard partition": "硬分区",
    "Use CM score for cutoff": "使用CM分数作为截断值",
    "merge_cluster_ol": "合并聚类_ol",
    "merge_overlap": "合并重叠",
    "min_cluster_size": "最小聚类大小",
    "cm_min_bitscore": "最小比对得分",
    "cm_max_eval": "cm_max_eval",
    "cm_bitscore_sig": "Cm比特分数显著性",
    "results_top_num": "结果数量",
    "model_tree_stk": "模型树stk",
    "cmfinder_fa": "cmfinder_fa",
    "Use gap threshold": "使用间隙阈值",
    "Define the gap threshold to determine the conserved column (--g)": "定义间隔阈值以确定保守列（--g）",
    "conservation-metrics-tsv": "保守性度量-TSV",
    "Minimum cluster size": "最小聚类大小",
    "RNAz bed threshold": "RNAz bed 阈值",
    "Rscape bed threshold": "Rscape bed 阈值",
    "model-tree-stk": "模型树stk",
    "cm_max_E-val": "Cm最大E值",
    "CD-HIT output": "CD-HIT 输出",
    "merge_cluster_ovelap": "合并聚类重叠",
    "merge_fraction_overlap": "合并分数重叠",
    "minimum cluster size": "最小聚类大小",
    "Parameters affecting reference genome indexing": "影响参考基因组索引的参数",
    "seed size (default: 20)": "种子大小（默认值：20）",
    "hash table slack size (default: 0.3)": "哈希表松弛大小（默认值：0.3）",
    "index overflow factor (default: 40)": "索引溢出因子（默认值：40）",
    "Parameters applied to single-end samples": "应用于单端样本的参数",
    "edit distance (default: 8)": "编辑距离（默认值：8）",
    "confidence threshold (default: 2)": "置信度阈值（默认值：2）",
    "adaptive confdiff behaviour (default: 7)": "自适应置信区间差异行为（默认值：7）",
    "maximum seeds per read (default: 25)": "每个读取的最大种子数（默认值：25）",
    "maximum hits per seed (default: 250)": "最大种子命中数（默认：250）",
    "read clipping (default: from back and front)": "读取剪切（默认：从前和后）",
    "Parameters applied to paired-end samples": "应用于配对末端样本的参数",
    "minimum spacing to allow between paired ends (default: 100)": "配对末端之间允许的最小间距（默认值：100）",
    "maximum spacing to allow between paired ends (default: 10000)": "配对末端之间允许的最大间隔（默认值：10000）",
    "discard overlapping read pairs of type": "丢弃重叠的配对读取对类型",
    "Sort aligned reads in the output by coordinates": "根据坐标对输出中的比对读取进行排序",
    "Use = and X to indicate matches/mismatches in CIGAR strings explicitly instead of using M for both": "在CIGAR字符串中使用'='和'X'明确表示匹配和不匹配，而不是使用'M'表示两者。",
    "selectivity (default: 1)": "选择性（默认值：1）",
    "filter output (default: no filtering)": "过滤输出（默认：不过滤）",
    "datasets": "数据集",
    "choose mode": "选择模式",
    "input file format": "输入文件格式",
    "Alignment options for this sample": "此样本的比对选项",
    "edit distance": "编辑距离",
    "confidence threshold": "置信度阈值",
    "adaptive confdiff behaviour": "自适应置信区间差异行为",
    "maximum seeds per read": "每个读取的最大种子数",
    "maximum hits per seed": "最大种子命中数",
    "the paired-end fastq input is provided as": "成对的fastq输入提供为",
    "inputfile with the first set of reads of paired-end data": "输入文件包含配对末端数据的第一组读取",
    "inputfile with the second set of reads of paired-end data": "输入文件，包含配对末端数据的第二组读取",
    "paired input dataset collection": "配对输入数据集集合",
    "minimum spacing to allow between paired ends": "配对末端之间允许的最小间隔",
    "maximum spacing to allow between paired ends": "允许配对末端之间的最大间距",
    "Input data format": "输入数据格式",
    "BAM input dataset to sort": "BAM 输入数据集进行排序",
    "SAM input dataset to sort": "SAM 输入数据集进行排序",
    "Output format for the sorted data": "排序数据的输出格式",
    "Sort by read names instead of coordinates": "按读取名称排序，而不是坐标",
    "type of mapping analysis to perform": "要执行的映射分析类型",
    "data source to use": "数据源使用",
    "input file with variants to analyze": "输入包含要分析的变异的文件",
    "additional per-variant output file": "额外的每个变体输出文件",
    "mapping sample name": "映射样本名称",
    "name of the related parent sample": "相关父样本的名称",
    "name of the unrelated parent sample": "无关亲本样本的名称",
    "Infer alleles for missing parent": "推断缺失亲本的等位基因",
    "name of the contrast sample": "对比样本的名称",
    "BCF input file": "BCF 输入文件",
    "Aligned reads input dataset(s)": "对齐的读取输入数据集（s）",
    "BCF variant call dataset to extract coverage from": "BCF变异调用数据集，用于提取覆盖率",
    "group reads based on read group id only": "根据读取组ID仅对读取进行分组",
    "include low-coverage regions": "包含低覆盖区域",
    "maximal coverage allowed inside a low-coverage region (default: 0)": "在低覆盖区域允许的最大覆盖度（默认值：0）",
    "minimal deletion size (default: 100)": "最小删除大小（默认值：100）",
    "input dataset": "输入数据集",
    "installed genome": "已安装基因组",
    "output format": "输出格式",
    "Input chainfile": "输入链文件",
    "reverse the target and query sites of the initial chainfile": "将初始链文件中的目标位点和查询位点反转。",
    "Ambiguously mapping variants": "模糊映射的变异",
    "input dataset in BAM format": "输入BAM格式的数据集",
    "modify read-group information?": "修改读取组信息？",
    "modify comments in the input file?": "修改输入文件中的注释？",
    "comment line": "注释行",
    "rename read-group": "重命名读组",
    "old name": "旧名称",
    "new name": "新名称",
    "rename sequence": "重命名序列",
    "read-group ID (required)": "读取组 ID（必需）",
    "sample name (required)": "样本名称（必需）",
    "date (YYYY-MM-DD) the run was produced": "运行产生的日期（YYYY-MM-DD）",
    "name of sequencing center": "测序中心名称",
    "read-group library": "读取组库",
    "platform/technology used to produce the reads": "用于生成测序数据的平台/技术",
    "predicted median insert size": "预测的中位数插入片段大小",
    "platform unit; unique identifier": "平台单元；唯一标识符",
    "More options": "更多选项",
    "md5 sum verification of contigs/chromosomes": "md5 校验和验证 基因组片段/染色体",
    "average sample depth cap limit (default: 250)": "平均样本深度上限（默认：250）",
    "include information from pre-calculated vcf dataset": "包含预先计算的vcf数据集的信息",
    "independently generated vcf datset": "独立生成的vcf数据集",
    "keep all sites with alternate bases": "保留所有具有替代碱基的位点",
    "The VCF input with the variants to be reported": "待报告的变异的 VCF 输入",
    "Format to use for the report": "报告使用的格式",
    "Optional file with species-specific hyperlink formatting instructions": "可选文件，包含特定物种的超链接格式说明",
    "Sample-specific Filter": "样本特定过滤器",
    "sample": "样本",
    "genotype pattern(s) for the inclusion of variants": "包含变异的基因型模式",
    "depth of coverage for the sample at the variant site": "样本在变异位点的覆盖深度",
    "genotype quality for the variant in the sample": "样本中变异的基因型质量",
    "allelic fraction filter": "等位基因分数过滤器",
    "Region Filter": "区域过滤器",
    "Select the types of variants to include in the output": "选择要包含在输出中的变异类型",
    "CSV file": "CSV文件",
    "Choose the master variable type": "选择主变量类型",
    "Title of your plot": "你图表的标题",
    "Legend of X axis in your plot": "X轴图例",
    "Legend of Y axis in your plot": "你图中Y轴的图例",
    "Number of principal components": "主成分数量",
    "Source file First: a matrix (samples/observations in rows and variables/features in columns)": "源文件第一项：一个矩阵（样本/观测值在行中，变量/特征在列中）",
    "Source file Second: a table (samples/observations with response/class label)": "源文件第二列：一个表格（样本/观察结果与响应/类别标签）",
    "GFF Dataset to Filter": "GFF 数据集过滤",
    "Numerical columns": "数值列",
    "Response column (Y)": "响应列（Y）",
    "Predictor columns (X)": "预测器列（X）",
    "Select columns containing input variables": "选择包含输入变量的列",
    "Number of principal components to return": "返回的主成分数量",
    "Kernel function": "核函数",
    "sigma (inverse kernel width)": "sigma（逆核宽度）",
    "degree": "度",
    "scale": "缩放",
    "offset": "偏移量",
    "sigma": "标准差",
    "order": "排序",
    "Select a task:": "选择一个任务：",
    "Select columns containing X variables": "选择包含X个变量的列",
    "Select columns containing Y variables": "选择包含 Y 变量的列",
    "Number of canonical components to return": "返回的典型成分数量",
    "Type of Scaling for X variables": "X变量的缩放类型",
    "Type of Scaling for Y variables": "Y变量的缩放类型",
    "Report standardized scores?": "报告标准化分数？",
    "Dataset containing workflows": "包含工作流的数据集",
    "Dataset containing usage frequencies of tools": "包含工具使用频率的数据集",
    "Data parameters": "数据参数",
    "Cutoff date": "截止日期",
    "Maximum number of tools in a tool sequence": "工具序列中工具的最大数量",
    "Training parameters": "训练参数",
    "Maximum number of evaluations of different configurations of parameters": "参数配置的不同评估的最大次数",
    "Training batch size": "训练批次大小",
    "Test batch size": "测试批次大小",
    "Training log step": "训练日志步骤",
    "Test log step": "测试日志步骤",
    "Share of the test data": "测试数据的份额",
    "Neural network parameters": "神经网络参数",
    "Number of heads in Multihead attention network": "多头注意力网络中的头数",
    "Embedding dimensions": "嵌入维度",
    "Number of units for feed forward network": "前馈网络的单元数",
    "Amount of dropout between neural network layers. E.g. 0.2 means 20%": "神经网络层之间的dropout量。例如，0.2表示20%",
    "Feature column": "特征列",
    "alpha (significance level)": "α（显著性水平）",
    "Observation Classes": "观察类别",
    "Confidence bias for clustering": "聚类置信偏差",
    "Number of links towards closest neighbour with the same class": "与具有相同类别的最邻近节点的连接数",
    "Number of links towards denser neighbours with a different class": "与具有不同类别的更密集邻居之间的链接数量",
    "Number of nearest neighbours to limit the horizon of where to search for denser neighbours of a different class": "搜索不同类别的更密集邻居的视界限制的最近邻居数量",
    "Number of mutual nearest neighbours that define outlier instances.": "定义异常实例的互近邻数量。",
    "Use Advanced Parameters?": "使用高级参数？",
    "Convert data matrix to corr coeff matrix": "将数据矩阵转换为相关系数矩阵",
    "Convert data matrix to normalized matrix": "将数据矩阵转换为标准化矩阵",
    "Select most discriminative features": "选择最具区分性的特征",
    "Color map": "颜色图",
    "Min num instances per class": "最小实例数每类",
    "Max num instances per class": "每个类别的最大实例数",
    "Select input file in tabular or BED format": "选择表格格式或BED格式的输入文件",
    "Optional column to group": "可选列以分组",
    "Column with the value of interest": "感兴趣的列的值",
    "Provide a window type": "提供一个窗口类型",
    "beta": "β",
    "std": "标准偏差",
    "power": "功率",
    "width": "宽度",
    "Provide a statistical function": "提供一个统计函数",
    "center smoothed values": "中心平滑值",
    "Window length": "窗口长度",
    "Select IWTomicsData object with test results": "选择带有测试结果的IWTomicsData对象",
    "Select test IDs": "选择测试ID",
    "Select feature IDs": "选择特征ID",
    "Select scale for Interval-Wise Testing and plot": "选择区间智能测试的比例，并进行绘图",
    "Threshold on test scale": "测试尺度的阈值",
    "Test ID": "测试ID",
    "Feature ID": "特征ID",
    "Plot IWTomics test results": "绘制IWTomics测试结果",
    "Level of the test (alpha)": "测试水平（alpha）",
    "Select IWTomicsData object": "选择 IWTomicsData 对象",
    "Select region dataset IDs": "选择区域数据集ID",
    "Select regions for Interval-Wise Testing": "选择区域进行区间内测试",
    "Two-sample test": "双样本检验",
    "Region 1": "区域1",
    "Region 2": "区域 2",
    "Select features": "选择特征",
    "Test statistics": "测试统计",
    "Probabilities": "概率",
    "Probability": "概率",
    "Select region files": "选择区域文件",
    "Insert header file for regions (optional)": "插入区域的头文件（可选）",
    "Select feature files": "选择特征文件",
    "Insert header file for features (optional)": "插入特征头文件（可选）",
    "Start positions are:": "起始位置是：",
    "Smoothing": "平滑",
    "Bandwidth for smoothing": "平滑带宽",
    "Degree of local polynomial": "局部多项式的度数",
    "Degree of B-splines": "B-样条的度数",
    "Distance between nodes (approximate)": "节点间距离（近似）",
    "Plot data": "绘图数据",
    "Load blocks": "加载区块",
    "Advanced parsing": "高级解析",
    "Consider the first row as header of columns": "将第一行视为列的标题",
    "Column separator": "列分隔符",
    "Advanced analysis": "高级分析",
    "Number of component": "组件数量",
    "Scale the blocks": "缩放区块",
    "Analysis method": "分析方法",
    "label": "标签",
    "Load the design matrix (if superblock or supervised disabled)": "加载设计矩阵（如果超块或监督禁用）",
    "Use a superblock": "使用一个超级块",
    "Learning mode": "学习模式",
    "Use a block as response (supervised analysis)": "使用块作为响应（监督分析）",
    "Tau selection": "Tau选择",
    "Scheme function": "方案函数",
    "Advanced graphic": "高级图形",
    "Color the individual plot with a response variable": "用响应变量为个体图上色",
    "Display the names of the points (in biplots)": "在双标图上显示点的名称",
    "Component for the X-axis": "X轴组件",
    "Component for the Y-axis": "Y轴组件",
    "Visualise this block": "可视化这个区块",
    "Visualise this block for the Y-axis (in individual plot)": "为Y轴可视化这个区块（在单独的图中）",
    "Number of top variables": "顶级变量数量",
    "Select a statistical test method": "选择一个统计检验方法",
    "Coordinates of moving landmarks (tsv file)": "移动地标的坐标（tsv文件）",
    "Coordinates of fixed landmarks (tsv file)": "固定地标的坐标（tsv文件）",
    "Select the algorithm": "选择算法",
    "Maximum distance for a data point to be classified as an inlier.": "数据点被分类为内点的最大距离。",
    "Maximum number of iterations for random sample selection.": "随机样本选择的最大迭代次数。",
    "Coordinates of points to be transformed (tsv file)": "待转换点的坐标（tsv文件）",
    "Centering and Scaling": "中心化和缩放",
    "Select the first sample tabular file": "选择第一个样本表格文件",
    "Select the second sample tabular file": "选择第二个样本表格文件",
    "Choose the test sidedness:": "选择测试的单侧性：",
    "Choose the standard deviation equality status of the two populations:": "选择两个总体的标准差相等状态：",
    "tabular label input": "表格标签输入",
    "expect label:weight format": "期望标签：权重格式",
    "Inflation": "膨胀",
    "Transform input matrix values": "转换输入矩阵的值",
    "Discard loops in input": "丢弃输入中的循环",
    "Reweight loops": "重新加权循环",
    "Set loops to sum of other arcs weights": "将环的权重设置为其他弧线权重的总和",
    "Verbosity": "详细程度",
    "Pruning options": "修剪选项",
    "cutoff": "截断值",
    "selection number": "选择编号",
    "recover number": "恢复数量",
    "recover percentage": "恢复百分比",
    "Test data": "测试数据",
    "Select the dataset containing array to split": "选择包含要拆分的数组的数据集",
    "Does the dataset contain header?": "数据集包含标题吗？",
    "Select the splitting mode": "选择分割模式",
    "Test size:": "测试大小：",
    "Random seed number:": "随机种子编号：",
    "Shuffle strategy": "洗牌策略",
    "Select the dataset containing groups": "选择包含组的数据集",
    "Select the cv splitter:": "选择交叉验证分割器：",
    "n_repeats": "重复次数",
    "Target values": "目标值",
    "p": "p",
    "Maximum size of the training set": "训练集的最大大小",
    "test_fold": "测试折叠",
    "Select the dataset containing target values": "选择包含目标值的数据集",
    "n_groups": "组数",
    "Type the index of split to output": "输入要输出的拆分索引",
    "Select a scheme": "选择一个方案",
    "Test holdout": "测试保留",
    "Metrics for evaluation": "评估指标",
    "Validation holdout (recommend using the same method for both validation and test)": "验证保留（建议对验证和测试使用相同的方法）",
    "Save the fitted model": "保存拟合好的模型",
    "Exclude Header": "排除标题",
    "Select PCA Type": "选择PCA类型",
    "Number of components": "组件数量",
    "Whiten Components": "白化组分",
    "SVD Solver": "奇异值分解求解器",
    "Batch Size": "批量大小",
    "Kernel Type": "内核类型",
    "Degree of the polynomial": "多项式的度数",
    "Coef0": "系数0",
    "Eigen Solver": "特征值求解器",
    "Maximum Iterations": "最大迭代次数",
    "Choose the dataset containing pipeline/estimator object": "选择包含流水线/估计器对象的数据集",
    "Select invocation method": "选择调用方法",
    "Select input data type for prediction": "选择预测的输入数据类型",
    "Training samples dataset:": "训练样本数据集：",
    "Does the dataset contain header:": "数据集是否包含标题：",
    "Select a sparse matrix": "选择一个稀疏矩阵",
    "Dataset containing fasta genomic/protein sequences": "包含fasta格式的基因组/蛋白质序列的数据集",
    "Dataset containing reference genomic sequence": "包含参考基因组序列的数据集",
    "blacklist regioins": "黑名单区域",
    "Dataset containing sequence variations": "包含序列变异的数据集",
    "Encoding seqence length": "编码序列长度",
    "Predict the reference sequence?": "预测参考序列？",
    "Select a plotting type": "选择一个绘图类型",
    "Select the dataset containing values for plotting learning curve.": "选择包含用于绘制学习曲线的值的数据集。",
    "Whether to plot standard error bar?": "是否绘制标准误差条？",
    "The output format and library": "输出格式和库",
    "Select the dataset containing true labels.": "选择包含真实标签的数据集。",
    "Select the dataset containing predicted probabilities.": "选择包含预测概率的数据集。",
    "pos_label": "正类标签",
    "Report minimum number of positives": "报告最小阳性数",
    "drop_intermediate": "删除中间文件",
    "Select the dataset containing grid_scores from a fitted RFECV object.": "选择包含来自拟合好的RFECV对象的grid_scores数据集。",
    "Step IDs": "步骤ID",
    "Select the dataset containing fitted estimator/pipeline": "选择包含拟合估计器/管道的数据集",
    "Select the dataset containing feature names": "选择包含特征名称的数据集",
    "Threshold value": "阈值",
    "Custom plot title": "自定义图表标题",
    "Select the JSON dataset containing configuration for a neural network model": "选择包含神经网络模型配置的JSON数据集",
    "Select dataset containing true labels": "选择包含真实标签的数据集",
    "Select dataset containing predicted labels": "选择包含预测标签的数据集",
    "Choose plot color": "选择绘图颜色",
    "Classification or regression?": "分类还是回归？",
    "Loss function": "损失函数",
    "Number of classes": "类的数量",
    "Select the format of input data": "选择输入数据的格式",
    "Sparse vector (scipy.sparse.csr_matrix) file:": "稀疏向量（scipy.sparse.csr_matrix）文件：",
    "Data file with numeric values": "数值数据文件",
    "Clustering Algorithm": "聚类算法",
    "Damping factor": "阻尼因子",
    "Number of iterations at each convergence step": "每个收敛步骤的迭代次数",
    "Affinity": "亲和力",
    "Kernel bandwidth": "核带宽",
    "Discretize initial kernel locations": "离散化初始核位置",
    "Minimum number of seeds per bin": "每个bin的最小种子数",
    "Cluster all": "聚类全部",
    "Linkage": "连锁",
    "Choose the stacking ensemble type": "选择堆叠集成类型",
    "shuffle": "打乱",
    "Meta Estimator": "元估计器",
    "Base Estimator": "基底估计器",
    "Output parameters for searchCV?": "searchCV 的输出参数是什么？",
    "Validation holdout": "验证保留",
    "Validation holdout (recommend using the same splitting method as for test holdout)": "验证保留（建议使用与测试保留相同的分割方法）",
    "Metrics from scikit-learn": "来自scikit-learn的指标",
    "Classifier type": "分类器类型",
    "Nu control parameter": "Nu 控制参数",
    "Penalization norm": "惩罚范数",
    "Use the shrinking heuristic": "使用缩减启发式方法",
    "Multi-class strategy": "多类策略",
    "Calculate the intercept for this model": "计算此模型的截距",
    "Add synthetic feature to the instance vector": "在实例向量中添加合成特征",
    "Sample type:": "样本类型：",
    "Number of centers to generate": "生成中心的数量",
    "Standard deviation of the clusters": "簇的标准差",
    "Shuffle the samples": "打乱样本",
    "Number of informative features": "信息特征数",
    "Number of redundant features": "冗余特征的数量",
    "Number of duplicated features": "重复特征的数量",
    "Number of clusters per class": "每个类别的聚类数",
    "Fraction of samples with randomly exchanged class labels": "随机交换类别标签的样本比例",
    "Unit matrix coefficient": "单位矩阵系数",
    "Scale factor between inner and outer circle": "内外圆的比例因子",
    "Number of regression targets": "回归目标数量",
    "Bias of the true function": "真实函数的偏差",
    "Choose the dataset containing estimator/pipeline object": "选择包含估计器/管道对象的数据集",
    "Select an attribute retrival type": "选择一个属性检索类型",
    "Select a hyperparameter search algorithm": "选择一个超参数搜索算法",
    "Number of parameter settings that are sampled": "被采样的参数设置数量",
    "Number of parameter settings to sample in parallel": "并行采样的参数设置数量",
    "Advanced Options for SearchCV": "SearchCV的高级选项",
    "Whether to hold a portion of samples for test exclusively, nested CV?": "是否保留一部分样本专门用于测试，嵌套交叉验证？",
    "Select the outer cv splitter": "选择外部交叉验证拆分器",
    "Save best estimator?": "保存最佳估计器？",
    "Pre-processing step": "预处理步骤",
    "Choose the type of transformation:": "选择转换类型：",
    "Type in parameter settings for IRAPSCore if different from default:": "如果IRAPSCore的参数设置与默认值不同，请输入：",
    "P value threshold": "P值阈值",
    "fold change threshold": "折叠变化阈值",
    "reservation factor": "保留因子",
    "The z_score threshold to discretize target value": "z分数阈值用于离散化目标值",
    "Final Estimator": "最终估计器",
    "Choose the module that contains target estimator:": "选择包含目标估计器的模块：",
    "Choose estimator class:": "选择估计器类别：",
    "Choose the dataset containing the wrapped regressor": "选择包含包装回归器的数据集",
    "Choose the dataset containing transformer": "选择包含转换器的数据集",
    "Classifier or Regressor:": "分类器或回归器：",
    "Choose the dataset containing the wrapped estimator or pipeline": "选择包含包装估计器或管道的数据集",
    "Discrize target values using z_score": "使用z得分区分目标值",
    "Discretize target values using a fixed value instead": "使用固定值对目标值进行离散化",
    "Are the detecting values smaller than others?": "检测值是否比其他值小？",
    "Choose the dataset containing the custom estimator or pipeline": "选择包含自定义估计器或流水线的数据集",
    "Select the type of your input data:": "选择您的输入数据类型：",
    "Sum over features": "特征求和",
    "Neighbor selection method": "邻域选择方法",
    "Number of neighbors": "邻居数",
    "Metric": "指标",
    "Shrink threshold": "收缩阈值",
    "Select an ensemble method:": "选择一种集成方法：",
    "Boosting algorithm": "提升算法",
    "The minimum number of samples required to be at a leaf node": "最小样本数需要在叶节点",
    "Save the fitted selector?": "保存拟合好的选择器？",
    "Select a tabular file you want to train your preprocessor on its data:": "选择一个表格文件，您希望在该文件的数据上训练您的预处理器：",
    "Select a sparse representation you want to train your preprocessor on its data:": "选择一个稀疏表示，您希望在它的数据上训练您的预处理器：",
    "Save the preprocessor": "保存预处理器",
    "Metrics": "指标",
    "Number of digits for formatting output floating point values": "输出浮点数值格式化时使用的位数",
    " Drop suboptimal thresholds": "删除次优阈值",
    "Solver": "求解器",
    "Store covariance": "存储协方差",
    "Regularization coefficient": "正则化系数",
    "Store covariances": "存储协方差",
    "Select a linear model:": "选择一个线性模型：",
    "Use dual formulation": "使用对偶公式",
    "Optimization algorithm": "优化算法",
    "Intercept scaling factor": "截距缩放因子",
    "Multiclass option": "多类选项",
    "Number of folds used in cross validation": "交叉验证中使用的折数",
    "Average scores across all folds": "所有折数的平均分数",
    "Choose the dataset containing model/pipeline object": "选择包含模型/流水线对象的数据集",
    "Select a model validation function": "选择一个模型验证函数",
    "Invokes the passed method name of the passed estimator": "调用传递的估计器的传递方法名称",
    "train_sizes": "训练集大小",
    "n_permutations": "n_permutations",
    "Choose the dataset containing pipeline/estimator": "选择包含管道/估计器的数据集",
    "Is the estimator a deep learning model?": "估计器是一个深度学习模型吗？",
    "Choose a building mode": "选择一个构建模式",
    "Select the dataset containing model configurations (JSON)": "选择包含模型配置（JSON）的数据集",
    "Select a learning class": "选择一个学习类别",
    "Select a batch data generator": "选择一个批量数据生成器",
    "class_positive_factor": "正类权重因子",
    "prediction_steps": "预测步骤",
    "Do classification or regression?": "进行分类还是回归？",
    "The proportion of training data to set aside as validation set.": "训练数据中留出作为验证集的比例。",
    "Select keras model type": "选择 Keras 模型类型",
    "input_shape": "输入形状",
    "LAYER": "层",
    "Choose the type of layer": "选择图层类型",
    "Type the list of input layer indexes": "输入层索引列表的类型",
    "Type the list of output layer indexes": "输入输出层索引的列表",
    "Total number of classes": "总类别数",
    "DNA FASTA file": "DNA 序列文件",
    "Select a reference model": "选择一个参考模型",
    "Minimum contig length": "最小连通体长度",
    "Feature table": "特征表",
    "Parameter Set": "参数集",
    "Representation learning type": "表示学习类型",
    "Class labels": "类别标签",
    "Input image (label map)": "输入图像（标签图）",
    "Radius of the neighborhood": "邻域半径",
    "Background label": "背景标签",
    "Background color": "背景颜色",
    "Select a dataset": "选择一个数据集",
    "Vectors dimensionality": "向量维度",
    "Levels": "水平",
    "Model retraining iterations": "模型重新训练迭代次数",
    "Number of folds for cross-validation": "交叉验证的折数",
    "Enable feature selection": "启用特征选择",
    "Minimum number of selected features": "最小选择特征数",
    "Accuracy threshold": "准确度阈值",
    "Accuracy uncertainty percentage": "准确度不确定百分比",
    "Interval file with joined SNPs": "包含联合SNPs的区间文件",
    "Codon Sequence column": "密码子序列列",
    "SNP chromosome column": "SNP 染色体列",
    "SNP start column": "SNP起始列",
    "SNP end column": "SNP结束列",
    "SNP strand column": "SNP链位置列",
    "SNP observed column": "观察到的SNP列",
    "Specify input alignment file in PHYLIP, FASTA, NEXUS, CLUSTAL or MSF format.": "指定输入的序列比对文件，格式可以是PHYLIP、FASTA、NEXUS、CLUSTAL或MSF。",
    "Comma-separated list of taxons to output": "输出的物种列表，以逗号分隔",
    "Short alignments": "短序列比对",
    "Specify sequence type as either of DNA, AA, BIN, MORPH, CODON or NT2AA for DNA, amino-acid, binary, morphological, codon or DNA-to-AA-translated sequences": "指定序列类型为DNA、氨基酸、二进制、形态学、密码子或DNA转录为氨基酸序列。",
    "Specify a file containing starting tree for tree search": "指定一个包含树搜索起始树的文件",
    "Specify a random number seed to reproduce a previous run (leave blank to randomize)": "指定一个随机数种子以重现之前的运行（留空以随机化）",
    "Keep identical sequences in the alignment": "保持比对中的相同序列",
    "Likelihood epsilon for parameter estimate": "参数估计的似然误差界限",
    "Turn on safe numerical mode to avoid numerical underflow for large data sets with many sequences (typically in the order of thousands)": "开启安全数值模式以避免在包含大量序列（通常为数千个）的大型数据集时发生数值下溢。",
    "Time Tree Reconstruction": "时间树重建",
    "Source of date information": "日期信息来源",
    "File containing dates of tips or ancestral nodes": "包含末端或祖先节点日期的文件",
    "Tip dates as a real number or YYYY-MM-DD": "提示日期为实数或YYYY-MM-DD格式",
    "Root date as a real number or YYYY-MM-DD": "根日期作为实数或 YYYY-MM-DD",
    "Number of replicates to compute confidence interval": "重复次数以计算置信区间",
    "Standard-deviation for lognormal relaxed clock": "对数正态松弛时钟的标准差",
    "Exclude outgroup from time tree": "排除外群以构建时间树",
    "Z-score cutoff to remove outlier tips/nodes": "Z分数截断值，用于移除异常的末端/节点",
    "Extra options passing directly to LSD2": "额外选项直接传递给 LSD2",
    "Likelihood mapping analysis": "似然映射分析",
    "Specify the number of quartets to be randomly drawn": "指定要随机抽取的四分体数量",
    "Specify a NEXUS file containing taxon clusters (see below for example) for quartet mapping analysis.": "指定一个包含分类群簇的NEXUS文件（见下面的示例）进行四分体映射分析。",
    "Write quartet log-likelihoods into .lmap.quartetlh file (typically not needed).": "将四重态对数似然写入 .lmap.quartetlh 文件（通常不需要）。",
    "Modelling Parameters": "建模参数",
    "Automatic model selection": "自动模型选择",
    "Use Custom Model": "使用自定义模型",
    "Perform standard model selection like jModelTest (for DNA) and ProtTest (for protein)": "执行标准的模型选择，如DNA的jModelTest和蛋白质的ProtTest。",
    "Additional model specifications": "额外的模型规格",
    "Merge partitions": "合并分区",
    "Specify the percentage for the relaxed clustering algorithm (Lanfear et al., 2014)": "指定松弛聚类算法（Lanfear等人，2014年）的百分比",
    "Max number of partition pairs": "最大分区对数",
    "Specify the name of a program (raxml, phyml or mrbayes) to restrict to only those models supported by the specified program": "指定一个程序名称（raxml、phyml 或 mrbayes），以限制只支持指定程序的那些模型",
    "Specify either nuclear, mitochondrial, chloroplast or viral to restrict to those AA models designed for specified source.": "指定核基因组、线粒体、叶绿体或病毒，以限制到为特定来源设计的氨基酸模型。",
    "Specify a comma-separated list of frequency types for model selection": "指定用于模型选择的频率类型的逗号分隔列表",
    "Specify a comma-separated list of rate heterogeneity types for model selection": "指定一个用于模型选择的速率异质性类型的逗号分隔列表",
    "Specify minimum number of categories for FreeRate model": "指定 FreeRate 模型的最小类别数",
    "Specify maximum number of categories for FreeRate model": "指定 FreeRate 模型的最大类别数",
    "Specify either AIC, AICc or BIC for the optimality criterion to apply for new procedure": "指定 AIC、AICc 或 BIC 作为新程序应用的最优性标准。",
    "Turn on full tree search for each model considered, to obtain more accurate result": "对每个考虑的模型开启完整树搜索，以获得更准确的结果。",
    "Specify a comma-separated list of mixture models to additionally consider for model selection": "指定一个逗号分隔的混合模型列表，以供模型选择时额外考虑",
    "Specify a NEXUS model file to define new models.": "指定一个NEXUS模型文件来定义新模型。",
    "Find best codon/protein/DNA models": "寻找最佳密码子/蛋白质/DNA模型",
    "Specifying substitution models": "指定替代模型",
    "Turn on optimizing weights of mixture models": "开启优化混合模型的权重",
    "Rate heterogeneity": "速率异质性",
    "Specify the minimum Gamma shape parameter (default: estimate)": "指定最小Gamma形状参数（默认：估计）",
    "Perform the median approximation for Gamma rate heterogeneity instead of the default mean approximation (Yang, 1994)": "执行伽马速率异质性的中位数近似，而不是默认的均值近似（杨，1994年）",
    "Specify the proportion of invariable sites (default: estimate)": "指定不变位点的比例（默认：估计）",
    "Perform more thorough estimation for +I+G model parameters": "对+I+G模型参数进行更彻底的估计",
    "Write empirical Bayesian site rates to .rate file": "将经验贝叶斯位点速率写入到 .rate 文件",
    "Write maximum likelihood site rates to .mlrate file": "将最大似然位点速率写入 .mlrate 文件",
    "Partition model options": "分区模型选项",
    "Partition model": "分区模型",
    "Partition file": "分区文件",
    "Subsample partitions": "子采样分区",
    "Subsampling random seed": "随机抽样种子",
    "Site-specific frequency model options": "特定位点频率模型选项",
    "Specify a guide tree to infer site frequency profiles.": "指定一个引导树来推断位点频率特征。",
    "Specify a site frequency file, e.g": "指定一个位点频率文件，例如",
    "Switch to posterior maximum mode for obtaining site-specific profiles": "切换到后验最大模式以获取位点特异性剖面",
    "Tree Parameters": "树参数",
    "Tree search parameters": "树搜索参数",
    "Skip subsequent tree search (n = 0), useful when you only want to assess the phylogenetic information of the alignment. Otherwise specify number of iterations to stop (this option overrides --nstop criterion).": "跳过后续的树搜索（n = 0），当你只想评估对齐的系统发育信息时很有用。否则，指定停止的迭代次数（这个选项会覆盖 --nstop 标准）。",
    "Specify number of initial parsimony trees": "指定初始简约性树的数量",
    "Specify number of top initial parsimony trees to optimize with ML nearest neighbor interchange (NNI) search to initialize the candidate set": "指定使用最大似然最近邻互换（NNI）搜索来优化的顶级初始简约树的数量，以初始化候选集",
    "Specify number of trees in the candidate set to maintain during ML tree search": "在ML树搜索期间，指定要维护的候选集中树的数量",
    "Specify number of unsuccessful iterations to stop": "指定停止不成功迭代的次数",
    "Specify SPR radius for the initial parsimony tree search": "指定初始简约树搜索的SPR半径",
    "Specify perturbation strength (between 0 and 1) for randomized NNI": "为随机NNI指定扰动强度（介于0和1之间）",
    "Turn on more thorough and slower NNI search": "开启更彻底且更慢的近邻互作搜索",
    "Avoid computing ML pairwise distances and BIONJ tree.": "避免计算ML成对距离和BIONJ树。",
    "Specify a topological constraint tree file in NEWICK format": "指定一个NEWICK格式的拓扑约束树文件",
    "Fast search to resemble FastTree": "快速搜索以类似于FastTree的方式进行。",
    "Collapse near-zero branches into polytomy": "将接近零的分支合并为多叉分支",
    "Write locally optimal trees into .treels file": "将局部最优树写入 .treels 文件",
    "Compute tree likelihood without optimisation": "计算树似然而不进行优化",
    "Check if the tree lies on a phylogenetic terrace": "检查树是否位于系统发育梯田上",
    "Single branch tests": "单分支测试",
    "Specify number of replicates (>=1000) to perform SH-like approximate likelihood ratio test (SH-aLRT) (Guindon et al., 2010)": "指定执行SH-like近似似然比率检验（SH-aLRT）（Guindon等人，2010年）的重复次数（>=1000）",
    "Perform approximate Bayes test (Anisimova et al., 2011).": "执行近似贝叶斯检验（Anisimova等人，2011年）。",
    "Specify number of replicates (>=1000) to perform fast local bootstrap probability method (Adachi and Hasegawa, 1996).": "指定执行快速局部引导概率方法（Adachi和Hasegawa，1996年）的重复次数（>=1000）。",
    "Tree topology tests": "树拓扑测试",
    "Specify a file containing a set of trees": "指定一个包含一组树的文件",
    "Specify the number of RELL (Kishino et al., 1990) replicates (>=1000) to perform several tree topology tests for all trees passed via -z": "指定 RELL（Kishino 等人，1990）复制品的数量（>=1000），以执行通过 -z 传递的所有树的多个树拓扑结构测试",
    "Used together with --test to additionally perform the weighted-KH and weighted-SH tests.": "与--test一起使用，还可以额外执行加权KH和加权SH测试。",
    "Used together with --test to additionally perform the approximately unbiased (AU) test (Shimodaira, 2002)": "与--test一起使用，还可以额外执行近似无偏（AU）测试（Shimodaira, 2002）",
    "Constructing consensus tree": "构建共识树",
    "Compute consensus tree of the trees passed via -t": "通过 -t 计算传递的树的共识树",
    "Compute consensus network of the trees passed via -t": "计算通过 -t 参数传递的树的共识网络。",
    "Specify a minimum threshold (between 0 and 1) to keep branches in the consensus tree": "指定一个最小阈值（介于0和1之间）以保留在共识树中的分支",
    "Specify a burn-in, which is the number of beginning trees passed via -t to discard before consensus construction": "指定一个燃烧期，这是通过 -t 传递的起始树的数量，在构建共识之前丢弃",
    "Specify an input “target” tree file": "指定一个输入的“目标”树文件",
    "Specify name of a node in -sup target tree": "指定 -sup 目标树中的一个节点名称",
    "Computing Robinson-Foulds distance": "计算罗宾逊-福尔兹距离",
    "Compute all-to-all RF distances between all trees passed via -t": "计算通过-t参数传递的所有树之间的所有对所有RF距离",
    "Compute RF distances between -t trees and this set": "计算 -t 棵树与这个集合之间的 RF 距离",
    "Like -rf but trees can have unequal taxon sets": "像 -rf 一样，但是树可以有不同的分类群集合",
    "Generating random trees": "生成随机树",
    "Specify number of taxa": "指定分类群的数量",
    "Random tree creation": "随机树创建",
    "Minimum branch lengths of the random tree": "随机树的最小分支长度",
    "Median branch lengths of the random tree": "随机树的中位数分支长度",
    "Maximum branch lengths of the random tree": "随机树的最大分支长度",
    "Ancestral State Reconstruction": "祖先状态重建",
    "Perform ancestral state reconstruction by empirical Bayes": "通过经验贝叶斯进行祖先状态重建",
    "Minimum probability of ancestral state": "最小祖先状态概率",
    "Symmetry Tests": "对称性测试",
    "Perform three tests of symmetry": "执行三次对称性测试",
    "Do --symtest and remove bad partitions": "执行 --symtest 并移除不良分区",
    "Do --symtest and remove good partitions": "执行--symtest并移除良好的分区",
    "Type of test to use": "要使用的测试类型",
    "Keep NAs in the tests": "在测试中保留NA值",
    "Concordance Factor Analysis": "一致性因子分析",
    "Set of source trees for gene concordance factor": "基因一致性因子的源树集合",
    "Write discordant trees associated with gDF1": "将与 gDF1 相关的不一致树写入",
    "Number of quartets for site concordance factor": "四分位点位一致性因子的数量",
    "Write CF per tree/locus": "为每个树/位点编写CF",
    "Write sCF for all resampled quartets to .cf.quartet": "将所有重采样的四重奏的sCF写入.cf.quartet文件。",
    "Bootstrap Parameters": "引导程序参数",
    "Ultrafast bootstrap parameters": "超快速引导参数",
    "Specify number of bootstrap replicates (>=1000).": "指定bootstrap重复次数（大于等于1000）。",
    "Specify number of replicates for ultrafast jackknife (>=1000).": "指定超快速杰克刀法的重复次数（大于等于1000）。",
    "Subsampling proportion for jackknife.": "留一法的重采样比例。",
    "Turn on writing bootstrap trees to .ufboot file": "开启将引导树写入.ufboot文件",
    "Like -boot-trees but bootstrap trees written with branch lengths": "像 -boot-trees 一样，但是引导树的分支长度也被写入",
    "Specify maximum number of iterations to stop": "指定最大迭代次数以停止",
    "Specify minimum correlation coefficient for UFBoot convergence criterion": "指定UFBoot收敛准则的最小相关系数",
    "Specify iteration interval checking for UFBoot convergence": "指定UFBoot收敛的迭代间隔检查",
    "Specify a small epsilon to break tie in RELL evaluation for bootstrap trees": "为自助树的RELL评估指定一个小的epsilon值以打破平局",
    "Specify the resampling strategies for partitioned analysis": "为分区分析指定重新采样策略",
    "Perform an additional step to further optimize UFBoot trees by nearest neighbor interchange (NNI) based directly on bootstrap alignments": "执行一个额外的步骤，通过基于引导对齐的最邻近互操作（NNI）进一步优化UFBoot树",
    "Nonparametric bootstrap": "非参数自举法",
    "Replicates for bootstrap + ML tree + consensus tree": "重复抽样用于自助法 + ML树 + 共识树",
    "Replicates for jackknife + ML tree + consensus tree": "重采样用于刀切法 + 最大似然树 + 一致性树",
    "Replicates for bootstrap + consensus tree": "重复引导 + 共识树",
    "Replicates for bootstrap only": "用于自举的重复样本",
    "Transfer bootstrap expectation": "传递引导期望",
    "Miscellaneous options": "其他选项",
    "Specify a list of comma-separated integer numbers": "指定一个由逗号分隔的整数列表",
    "Sequences file": "序列文件",
    "tree file": "树文件",
    "Branch model ; for tree file editing in model 2 and 3, see paml manual (chap.3)": "分支模型；对于模型2和3中的树文件编辑，请参阅PAML手册（第3章）",
    "Set the level of details in the log file": "设置日志文件中的细节级别",
    "Tree analysis mode": "树分析模式",
    "Sequences format in the fasta file": "FASTA 文件中的序列格式",
    "Equilibrium codon frequencies in codon substitution model": "在密码子替代模型中的平衡密码子频率",
    "Specifies models concerning rate constancy or variation among lineages": "指定关于线系之间速率恒定或变化的模型",
    "Amino acid distances": "氨基酸距离",
    "Amino acide substitution rate matrix": "氨基酸替代率矩阵",
    "Icode : specifies the genetic code": "Icode：指定遗传密码",
    "Multiple genes": "多个基因",
    "Specifies wether kappa in  K80, F84, or HKY85 is fixed or estimated": "指定 K80、F84 或 HKY85 中的 kappa 是否固定或估计",
    "Initial or fixed value of kappa": "kappa的初始或固定值",
    "Fixed or estimated omega": "固定或估计的ω值",
    "Initial or fixed omega (according to your choice for fix_omega), for codons or codon-based AAs": "初始或固定的ω值（根据您的选择用于fix_omega），用于密码子或基于密码子的氨基酸",
    "Estimated or fixed gamma shape parameter": "估计或固定的伽玛形状参数",
    "Initial or fixed value of alpha (gamma shape parameter)": "alpha的初始值或固定值（gamma形状参数）",
    "Different alphas for genes": "不同基因的不同 alpha 值",
    "# of categories in dG of NSsites models": "在NSsites模型中dG的类别数量",
    "Independence or correlation of rates at adjacent sites": "相邻位点上速率的独立性或相关性",
    "Initial or fixed rho": "初始或固定的rho",
    "Estimates of the standard errors of estimated parameters.": "估计参数的标准误差的估计值。",
    "RateAncestor ; set 1 to force the program to do two additional analyses": "RateAncestor；设置1以强制程序进行两个额外的分析",
    "Value used in the difference approximation of derivatives": "在导数差分逼近中使用的值",
    "Remove sites with ambiguity data": "删除具有不确定性数据的位点",
    "branch length dealing": "分支长度处理",
    "Controls the iteration algorithm for estimating branch lengths": "控制估计分支长度的迭代算法",
    "Genotype data": "基因型数据",
    "Number of runs": "运行次数",
    "mainparams": "主要参数",
    "Number of populations assumed": "假设的种群数量",
    "Length of burnin period": "烧入期长度",
    "Number of MCMC reps after burnin": "MCMC 后燃烧期的重复次数",
    "Number of diploid individuals in data file": "数据文件中二倍体个体的数量",
    "Number of loci in data file": "数据文件中的位点数",
    "Ploidy of data": "数据的倍性",
    "Value given to missing genotype data": "给缺失基因型数据的值",
    "Store data for individuals in a single line": "将个体的数据存储在一行中",
    "Input file contains individual labels": "输入文件包含单独的标签",
    "Input file contains a user-defined population-of-origin for each individual": "输入文件包含每个个体的用户定义的起源种群",
    "Input file contains a location identifier": "输入文件包含一个位置标识符",
    "Input file contains phenotype information": "输入文件包含表型信息",
    "Number of additional columns of data before the genotype data start.": "在基因型数据开始之前的额外数据列数。",
    "Data file contains row of marker names": "数据文件包含标记名称的行",
    "Data file contains dominant markers (eg AFLPs) and a row to indicate which alleles are recessive": "数据文件包含显性标记（例如AFLPs），并有一行用于指示哪些等位基因是隐性的。",
    "Defines the code indicating that genotype data at a marker are unambiguous.": "定义一个代码，表示标记上的基因型数据是明确的。",
    "Data file contains row of map distances between loci": "数据文件包含位点间映射距离的行",
    "Data are in correct phase (relevant for linkage model only)": "数据处于正确的阶段（仅与连锁模型相关）",
    "The data for each individual contains a line indicating phase (linkage model)": "每个人的数据包含一行指示相位（连锁模型）的信息",
    "The phase info follows a Markov model.": "相位信息遵循马尔可夫模型。",
    "extraparams": "额外参数",
    "Use no admixture model": "使用无混合模型",
    "Use the linkage model model": "使用连锁模型模型",
    "Use prior population information to pre-assign individuals to clusters": "使用先前的群体信息预先将个体分配到簇中",
    "Input file contains a flag which says whether to use popinfo": "输入文件包含一个标志，指示是否使用 popinfo",
    "Prior prob that an individual is a migrant": "先验概率，即一个人是移民的概率",
    "Use location information to improve weak data": "使用位置信息来改善薄弱数据",
    "Allele frequencies are correlated among pops": "等位基因频率在种群间是相关的",
    "Assume same value of Fst for all subpopulations": "假设所有亚群的Fst值相同",
    "Infer ALPHA (the admixture parameter)": "推断ALPHA（混合参数）",
    "Dirichlet parameter for degree of admixture": "狄利克雷参数用于混合程度",
    "Individual alpha for each population": "每个种群的个体alpha",
    "Infer LAMBDA (the allele frequencies parameter)": "推断 LAMBDA（等位基因频率参数）",
    "Infer a separate lambda for each pop": "推断每个种群一个单独的lambda值",
    "Dirichlet parameter for allele frequencies": "等位基因频率的狄利克雷参数",
    "The Prior (Gamma distribution) mean of Fst for pops.": "先验（伽马分布）的Fst均值用于种群。",
    "The Prior (Gamma distribution) Standard Deviation of Fst for pops.": "先验（伽马分布）标准差用于种群间Fst值。",
    "Use a uniform prior for alpha; otherwise gamma prior": "使用均匀先验对 alpha；否则使用伽马先验",
    "Alpha has a gamma prior with mean A*B, and variance A*B^2.": "α 有一个均值为 A*B，方差为 A*B^2 的 γ（伽马）先验。",
    "Max value of alpha if uniform prior": "如果使用均匀先验，alpha的最大值",
    "Log10 of minimum allowed value of r under linkage model": "在连锁模型下，r的最小允许值的对数",
    "Log10 of maximum allowed value of r": "最大允许的r值的对数",
    "Standard deviation of log r in update": "更新中对数 r 的标准差",
    "Initial value of log10 r": "对数 r 的初始值",
    "For use when inferring whether an individual is an immigrant, or has an immigrant an cestor in the past GENSBACK generations.": "用于推断个体是否为移民，或者在过去GENSBACK代中是否有移民祖先。",
    "Only use individuals with POPFLAG=1 to update P.": "仅使用 POPFLAG=1 的个体来更新 P。",
    "Use POPDATA for location information": "使用POPDATA获取位置信息",
    "Initial value for r, the location prior": "r的初始值，位置先验",
    "Max allowed value for r": "r的最大允许值",
    "Print the 'net nucleotide distance' to screen during the run": "在运行过程中将“净核苷酸距离”打印到屏幕上",
    "Print current value(s) of lambda to screen": "打印lambda的当前值（们）到屏幕上",
    "Print summary of current population membership to screen": "将当前种群成员的摘要打印到屏幕上",
    "whether or not to print site by site results.": "是否打印每个位点的结果。",
    "Q-hat printed to a separate file.": "Q-hat 打印到一个单独的文件中。",
    "Frequency of printing update on the screen.": "屏幕上打印更新的频率。",
    "Print current likelihood to screen every rep": "在每次重复时将当前似然性打印到屏幕上",
    "Number of saves to file during run": "运行期间保存到文件的次数",
    "Print some of data file to screen to check that the data entry is correct.": "将部分数据文件打印到屏幕上，以检查数据输入是否正确。",
    "Collect data about the distribution of ancestry coefficients (Q) for each individual": "收集每个个体祖先系数（Q）分布的数据",
    "The distribution of Q values is stored as a histogram with this number of boxes.": "Q值的分布以直方图的形式存储，包含这个数量的箱子。",
    "The size of the displayed probability interval on Q (values between 0.0--1.0)": "在Q上显示的概率区间大小（值介于0.0到1.0之间）",
    "Estimate the probability of the Data under the model.": "估计数据在模型下的概率。",
    "Initial period of burnin with admixture model": "烧入期的初始阶段，使用混合模型",
    "SD of proposal for updating alpha": "更新alpha的提议的标准差",
    "Use given populations as the initial condition for population origins.": "使用给定的种群作为种群起源的初始条件。",
    "=use new random seed for each run": "=每次运行使用新的随机种子",
    "Seed value for random number generator": "随机数生成器的种子值",
    "seed value for random number generator": "随机数生成器的种子值",
    "Frequency of using Metropolis step to update Q under admixture model": "在混合模型下使用Metropolis步骤更新Q的频率",
    "Report hit rate if using METROFREQ": "如果使用METROFREQ，请报告命中率。",
    "Input to construct or reconstruct trees": "输入用于构建或重建树",
    "Samples to use for computing bootstrap": "用于计算自举的样本",
    "Input alignment type": "输入对齐类型",
    "Adjust for negative branch lengths.": "调整负分支长度。",
    "Input in FASTA format": "输入为FASTA格式",
    "Additional input FASTA for pairwise comparisons": "额外输入FASTA文件用于成对比较",
    "Only report distances below this threshold": "只报告低于此阈值的距离",
    "Strategy for ambiguous nucleotides": "模糊核苷酸的策略",
    "Maximum tolerated fraction of ambiguous characters": "最大容忍的不明确字符分数",
    "Only count pairs that overlap by at least N bases": "仅计算至少重叠N个碱基对的配对",
    "Sequence name component immediately preceding a copy number": "序列名称中紧接在拷贝数之前的组成部分",
    "Aligned reads": "对齐的读取",
    "FASTA file with compressed clusters": "带有压缩聚类的FASTA文件",
    "Pairwise distance threshold": "成对距离阈值",
    "Reference in FASTA format": "参考序列，FASTA格式",
    "Output additional fasta dataset with compressed clusters": "输出附加的压缩聚类fasta数据集",
    "Only retain this many clusters": "只保留这么多的聚类",
    "Distance threshold": "距离阈值",
    "Clustering rules": "聚类规则",
    "Only process pairs that overlap by at least N bases": "仅处理至少重叠N个碱基对的配对",
    "Mimimum cluster size to report in output": "输出中报告的最小聚类大小",
    "Aligned sequences file (FASTA or Phylip format)": "对齐序列文件（FASTA或Phylip格式）",
    "Allow spaces and other restricted characters (but not ') in sequence and quote names in the output tree.": "在输出树中，允许序列和引用名称中包含空格和其他限制字符（但不允许单引号）。",
    "Set starting tree(s)": "设置起始树",
    "PHYLIP file": "PHYLIP 文件",
    "Number of multiple alignments": "多重比对的数量",
    "Save log file output": "保存日志文件输出",
    "Protein or nucleotide alignment": "蛋白质或核苷酸序列比对",
    "Nucleotide evolution model": "核苷酸进化模型",
    "Protein evolution model": "蛋白质进化模型",
    "Use constant rates?": "使用恒定速率？",
    "Specify the number of rate categories of sites (default 20)": "指定位点速率类别的数量（默认为20）",
    "Use Gamma20": "使用 Gamma20",
    "Eliminates the support value computation. Support value computation is not recommended for wide alignments.": "消除支持值计算。对于宽对齐，不建议计算支持值。",
    "Turn off maximum-likelihood.": "关闭最大似然。",
    "Speed up the neighbor joining phase and reduce memory usage (recommended for >50,000 sequences)": "加快邻接连接阶段的速度并减少内存使用（推荐用于>50,000个序列）",
    "Use pseudocounts": "使用伪计数",
    "Optimize branch lengths without ML NNIs.": "优化分支长度，不使用最大似然的近邻插值法。",
    "Fasta File": "FASTA 文件",
    "Title for output Sequence Logo": "输出序列标志的标题",
    "Output format for image (or text report)": "图像（或文本报告）的输出格式",
    "Display Units": "显示单位",
    "Colour scheme for output Sequence Logo": "输出序列标志的配色方案",
    "Include entire sequence (default) or specify a subsequence range to use": "包括整个序列（默认）或指定一个子序列范围来使用",
    "Index (eg 1=first letter) of the start of the sequence range to include in the logo": "在徽标中包含序列范围的起始索引（例如1=第一个字母）",
    "Index (eg 75=75th letter) of the end of the sequence range to include in the logo": "序列范围结束位置的索引（例如75=第75个字母）以包含在标志中",
    "Output weblogo size": "输出 weblogo 大小",
    "Primary sequence file": "主要序列文件",
    "Control sequence file": "控制序列文件",
    "Minimum width to use for position specific priors": "用于位置特异性先验的最小宽度",
    "Maximum width to use for position specific priors": "用于位置特异性先验的最大宽度",
    "Alphabet": "字母表",
    "Use spaced triples instead of whole-word matches?": "使用空格分隔的三元组而不是整个单词匹配？",
    "Allow triples to start anywhere within a site?": "允许三元组在站点内的任何位置开始？",
    "Match as equal sequences of letters that appear together?": "匹配连续出现的相同字母序列？",
    "Consider both strands when calculating position specific priors for alphabets?": "在计算字母表的位置特异性先验时，考虑两个链吗？",
    "Set the lowest score value after scaling?": "设置缩放后最低分数值？",
    "Set the highest score value after scaling?": "设置缩放后的最高得分值？",
    "Choose the width with the biggest difference between minimum and maximum scores before scaling?": "选择在缩放前最小和最大分数之间差异最大的宽度？",
    "Output scores instead of priors?": "输出分数而不是先验概率？",
    "Output primary and control file names, scores and widths?": "输出主文件和对照文件的名称、分数和宽度？",
    "Report frequency of each score?": "报告每个分数的频率？",
    "I certify that I am not using this tool for commercial purposes.": "我保证我不将此工具用于商业目的。",
    "DREME or MEME output XML file": "DREME 或 MEME 输出 XML 文件",
    "Check reverse complement strand": "检查反向互补链",
    "Options configuration": "选项配置",
    "Output threshold for displaying search results": "显示搜索结果的输出阈值",
    "Apply output threshold to q-values?": "应用输出阈值到q值？",
    "Background model selection": "背景模型选择",
    "Background model file": "背景模型文件",
    "Report best match in case of overlapping matches on both strands?": "在两条链上都有重叠匹配的情况下，报告最佳匹配？",
    "Maximum number of scores that will be stored": "最大存储分数数量",
    "Specify single input motifs for scanning?": "指定用于扫描的单个输入基序？",
    "Supply input motif ID": "提供输入的基序ID",
    "Specify motif by id": "通过ID指定基序",
    "Pseudocount to add to counts in motif matrix": "在基序矩阵中添加的伪计数",
    "Disable q-value calculation?": "禁用q值计算？",
    "Check each sequence header for UCSC-style genomic coordinates?": "检查每个序列头是否为UCSC风格的基因组坐标？",
    "Use position-specific priors?": "使用位置特异性先验？",
    "Select file containing position-specific priors": "选择包含位置特异性先验的文件",
    "Alpha parameter for calculating position-specific priors": "计算位置特异性先验的Alpha参数",
    "Use binned distribution of priors?": "使用分箱的先验分布？",
    "Select dataset containing binned distribution of priors": "选择包含先验概率的分箱分布的数据集",
    "Output HTML file": "输出 HTML 文件",
    "Output XML file": "输出XML文件",
    "Output GFF file": "输出GFF文件",
    "Options Configuration": "选项配置",
    "Name of sequence set": "序列集名称",
    "Sequence Alphabet": "序列字母表",
    "Choice of prior": "先验选择",
    "strength of prior on model parameters": "模型参数的先验强度",
    "Dirichlet prior file": "狄利克雷先验文件",
    "EM starting points": "EM起始点",
    "Fuzziness of the mapping": "映射的模糊度",
    "Starting point from string": "起始点从字符串",
    "Check reverse complement": "检查反向互补",
    "Check for palindromes": "检查回文序列",
    "Number of different motifs to search": "要搜索的不同基序的数量",
    "E-value to stop looking for motifs": "E值用于停止寻找基序",
    "Expected motif distribution": "预期的基序分布",
    "Number of motif occurrences": "基序出现次数",
    "Search nsites number of occurrences": "搜索 nsites 数量的出现次数",
    "minsites": "最小位点数",
    "maxsites": "最大位点数",
    "Weight on the prior on nsites": "在nsites上的先验权重",
    "Motif width type": "Motif宽度类型",
    "Width of motif to search": "搜索的基序宽度",
    "Min width of motif to search": "搜索模式的最小宽度",
    "Max width of motif to search": "搜索的基序最大宽度",
    "Motif trim type": "基序修剪类型",
    "Gap cost": "间隔成本",
    "Space cost": "空间成本",
    "Do not penalize endgaps": "不要惩罚末端间隙",
    "Background Model": "背景模型",
    "Position-Specific Prior": "位置特异性先验",
    "Number of iterations of EM to run": "EM算法运行的迭代次数",
    "Convergence criterion": "收敛准则",
    "x-branching type": "X型分支",
    "Number of iterations of branching": "分支迭代次数",
    "Maximum number of heaps to use": "最大堆数量",
    "Positive FASTA sequence file": "正向FASTA序列文件",
    "Type of sequence alphabet": "序列字母类型",
    "Negative set definition": "负集定义",
    "Seed for shuffling positive sequences": "用于洗牌阳性序列的种子",
    "Negative FASTA sequence file": "负向FASTA序列文件",
    "stop if motif E-value > e": "如果基序的 E 值大于 e，则停止",
    "Minimum width of core motif": "核心基序的最小宽度",
    "Maximum width of core motif": "核心基序的最大宽度",
    "Stop if m motifs have been output": "如果输出了m个基序，则停止",
    "Stop if the specified time has elapsed": "如果指定的时间已经过去，则停止",
    "Number of REs to generalize": "REs泛化的数量",
    "Primary sequences": "主要序列",
    "Control sequences": "控制序列",
    "Sequence alphabet": "序列字母表",
    "Select the order of the Markov background model": "选择马尔可夫背景模型的阶数",
    "Limit of sequences to pass to MEME": "MEME传入序列的限制",
    "Should subsampling be random?": "子采样应该是随机的吗？",
    "Seed for the randomized selection of sequences": "序列随机选择的种子",
    "maximum size of a sequence before it is cut down to a centered section": "序列被切割到中心部分前的最大大小",
    "Primary threshold for clustering motifs": "聚类基序的主要阈值",
    "Secondary threshold for clustering motifs": "聚类基序的次级阈值",
    "E-value threshold for including motifs": "包括基序的E值阈值",
    "Search given strand only": "仅搜索给定链",
    "Pick cluster seed motifs based only on significance": "选择基于显著性的聚类种子基序",
    "What is the expected motif site distribution?": "什么是预期的基序位点分布？",
    "Minimum motif width": "最小基序宽度",
    "Maximum motif width": "最大基序宽度",
    "Maximum number of motifs to find": "最大找到的基序数",
    "Minimum number of sites per motif": "每个基序的最小位点数",
    "Maximum number of sites per motif": "每个基序的最大位点数",
    "Look for palindromes only": "寻找仅回文",
    "Stop DREME searching after reaching this E-value threshold": "在达到这个E值阈值后停止DREME搜索",
    "Stop DREME searching after finding this many motifs": "在找到这么多基序后停止DREME搜索",
    "First Name": "名字",
    "Last Name": "姓",
    "Apollo Organism Listing": "Apollo生物体列表",
    "Force the tracklist on in JBrowse": "在 JBrowse 中强制显示曲目列表",
    "Use the given name instead of generating one": "使用给定的名称，而不是生成一个新的。",
    "Disable CDS recalculation and instead use the one provided": "禁用CDS（编码序列）重新计算，并使用提供的CDS。",
    "Include fasta in the GFF3 file": "在GFF3文件中包含fasta",
    "Fail hard if any portion of the export fails": "如果导出的任何部分失败，则硬性失败。",
    "JBrowse HTML Output": "JBrowse HTML 输出",
    "Genus": "属",
    "Is Organism Public": "是公开生物体吗",
    "Grant access to a user group": "授予用户组访问权限",
    "Remove old data directory": "删除旧的数据目录",
    "Update genome sequences": "更新基因组序列",
    "Are you SURE you want to do this?": "你确定要这么做吗？",
    "Are you really, really SURE you want to do this?": "你真的，真的确定要这么做吗？",
    "Feature Type Filter": "特征类型过滤器",
    "Select action to perform": "选择要执行的操作",
    "HiFive fend file": "HiFive fend 文件",
    "HiFive HiC data file": "HiFive HiC 数据文件",
    "HiFive HiC project file": "HiFive HiC 项目文件",
    "Calculate and include trans interactions": "计算并包含转录互作",
    "Generate image": "生成图像",
    "Stopping coordinate": "停止坐标",
    "Maximum interaction distance": "最大作用距离",
    "HiFive fragment file": "HiFive 片段文件",
    "HiFive 5C data file": "HiFive 5C 数据文件",
    "HiFive 5C project file": "HiFive 5C 项目文件",
    "Filter based on entries in the info string": "根据信息字符串中的条目进行过滤",
    "Remove the genotype strings": "删除基因型字符串",
    "Mark all records as having passed filters": "标记所有记录为已通过过滤器",
    "Select target bedgraph file": "选择目标 bedgraph 文件",
    "Select control type": "选择控制类型",
    "Select control bedgraph file": "选择控制组 bedgraph 文件",
    "Set threshold": "设定阈值",
    "Normalize control to target data": "将对照组数据标准化至目标数据",
    "Will you select a reference genome from your history or use a cache FASTA?": "您将从历史记录中选择一个参考基因组，还是使用缓存的FASTA文件？",
    "Group1: fastq files": "Group1: 快速q文件",
    "Group2: fastq files": "组2：fastq文件",
    "Advanced options for BSMAP": "BSMAP 高级选项",
    "Seed size": "种子大小",
    "Set the mismatch rate or number?": "设置不匹配率或数量？",
    "Mismatch rate": "不匹配率",
    "Mismatch number": "不匹配数",
    "Maximum number of equal best hits to count": "最大等优命中数",
    "Restriction enzyme digestion sites for RRBS mode": "限制性内切酶切割位点，适用于RRBS模式",
    "Mapping to four strands?": "映射到四条链？",
    "Quality threshold in trimming": "修剪质量阈值",
    "Base quality": "碱基质量",
    "Maximum number of Ns in a read to filter out": "在读取中过滤掉的最大N的数量",
    "3' adapter sequence to trim": "3' 接头序列以修剪",
    "How to report repeat hits": "如何报告重复命中",
    "Print corresponding reference sequences?": "打印相应的参考序列？",
    "Report unmapped reads?": "报告未映射的读取？",
    "Minimal insert size allowed in paired-end mapping": "成对末端映射中允许的最小插入大小",
    "Maximal insert size allowed in paired-end mapping": "配对末端映射中允许的最大插入大小",
    "Advanced options for MCALL": "MCALL的高级选项",
    "Threshold for cytosine quality score": "胞嘧啶质量分数的阈值",
    "Threshold for the next base quality score": "下一个碱基质量分数的阈值",
    "Specify the quality score system": "指定质量评分系统",
    "Bases to trim end-repair sequences from +-/--": "从正负链的末端修复序列中修剪碱基",
    "Bases to trim end-repair sequences from ++/-+": "从 ++/-+ 的末端修复序列中修剪的碱基数量",
    "Count once or twice the overlap sequence of two pairs": "统计两个序列对的重叠序列一次或两次",
    "How to trim end-repair sequence for RRBS reads?": "如何为RRBS读取修剪末端修复序列？  RRBS（Reduced Representation Bisulfite Sequencing）是一种用于检测甲基化位点的高通量测序技术。在处理RRBS数据时，通常需要对测序读取的末端进行修剪，以去除可能影响后续分析的序列。  修剪末端修复序列的步骤通常包括以下几个方面：  1. **识别和去除接头序列**：在测序过程中，为了提高测序效率，通常会在DNA片段的两端添加特定的接头序列。这些序列在数据分析前需要被识别并去除。  2. **质量控制**：使用质量控制工具（如FastQC）对原始测序数据进行评估，以确定数据的质量。  3. **修剪低质量碱基**：使用修剪工具（如Trim Galore或Cutadapt）去除低质量的碱基，这些碱基通常位于读取的末端。  4. **去除可能的PCR重复**：PCR扩增过程中可能会产生重复序列，这些也需要被去除。  5. **修复序列**：如果有必要，可以使用序列修复工具（如End-Repair工具）来修复可能在测序过程中受损的末端序列。  6. **去除未甲基化的胞嘧啶**：在RRBS数据中，未甲基化的胞嘧啶可能会被错误地识别为甲基化，因此需要去除。  7. **数据格式化**：最后，将修剪后的数据格式化为适合后续分析的格式，如BAM或FASTQ。  在Galaxy平台上，可以使用一系列的工具来完成上述步骤，例如：  - **Cutadapt**：用于修剪接头序列和低质量碱基。 - **FastQC**：用于质量控制。 - **Trim Galore**：另一个修剪工具，特别适用于处理RRBS数据。 - **Bismark**：用于甲基化分析，可以在处理过程中自动进行一些修剪和修复工作。  具体的操作步骤和参数设置会根据你的数据和分析需求有所不同，但上述工具和步骤为处理RRBS数据提供了一个基本的框架。",
    "Minimal fragment size for properly mapped reads": "正确映射的读取的最小片段大小",
    "Minimal fragment size for multiply matched read": "最小片段大小，用于多次匹配的读取",
    "Generates CpG/A/C/T methylation?": "生成 CpG/A/C/T 甲基化？",
    "Advanced options for MCOMP": "MCOMP的高级选项",
    "Run the comparison or not": "运行比较与否",
    "Input comparison results": "输入比较结果",
    "Minimum depth for a site coverage": "位点覆盖的最小深度",
    "Minimum absolute credible methylation difference (CDIF)": "最小绝对可信甲基化差异（CDIF）",
    "Cutoff of Pvalue from Fisher Exact Test for DMC scan": "费舍尔精确检验的P值截止值，用于DMC扫描",
    "Cutoff of Pvalue from Fisher Exact Test for DMR scan": "费舍尔精确检验的DMR扫描P值截止值",
    "Minimal nominal methylation difference for DMC and DMR calling": "最小名义甲基化差异用于 DMC（差异甲基化胞嘧啶）和 DMR（差异甲基化区域）的调用",
    "Minimal credible methylation difference for DMC calling": "最小可信甲基化差异用于DMC调用",
    "Minimum number of DMCs in a DMR": "最小差异甲基化胞嘧啶数量在一个差异甲基化区域中",
    "Maximum distance between two consective DMCs for a DMR": "两个连续的DMC（DNA甲基化中心）之间的最大距离，对于一个DMR（差异甲基化区域）",
    "Select output files": "选择输出文件",
    "Red .IDAT files": "红色.I.DAT文件",
    "Green .IDAT files": "绿色.iDAT文件",
    "(Optional) Preprocessing Method": "（可选）预处理方法",
    "Phenotype Table": "表型表",
    "maxGap Size": "最大间隔大小",
    "Cutoff Size": "截止大小",
    "Number of Resamples": "重采样数量",
    "null Method": "空方法",
    "Phenotype Type": "表型类型",
    "qCutoff Size": "qCutoff 大小",
    "Variance Shrinkage": "方差收缩",
    "Genome Table": "基因组表格",
    "Input group 1": "输入组1",
    "Input group 2": "输入组 2",
    "BED file containing regions of interest": "包含感兴趣区域的BED文件",
    "The allowed nt distance between two CpGs within a DMR": "两个CpG之间在DMR（差异甲基化区域）中允许的核苷酸距离",
    "The minimum # of CpGs in a DMR": "DMR中CpG的最小数量",
    "Specify how many replicates must contain data for a certain CpG position in group 1": "指定组1中必须包含某个CpG位点数据的重复样本数。",
    "Specify how many replicates must contain data for a certain CpG position in group 2": "指定组2中必须包含某个CpG位点数据的重复样本数量。",
    "The minimum mean methylation difference for calling DMRs": "最小平均甲基化差异用于调用差异甲基化区域（DMRs）",
    "Stringency of the valley filter (0.0 - 1.0)": "谷值过滤器的严格度（0.0 - 1.0）",
    "Input files in BAM or SAM fomat": "输入BAM或SAM格式的文件",
    "Genome digest file": "基因组消化文件",
    "Will you select a reference genome from your history or use a genome which has a built-in index?": "您将从历史记录中选择一个参考基因组，还是使用具有内置索引的基因组？",
    "Regions in BED format": "BED格式的区域",
    "Chromatin mark": "染色质标记",
    "Mark name": "标记名称",
    "Reads in BAM format": "读取BAM格式的数据",
    "Number of states": "状态数",
    "Put the size of the bin in base pairs.": "将二进制文件的大小设置为碱基对。",
    "Select a file with the coordinates of the pairs (minimum 4 columns with chromosome and position for each mate).": "选择一个包含配对坐标的文件（至少4列，每对配对的染色体和位置）。",
    "Format of your input file": "输入文件的格式",
    "field number in the input file (starting from 1) of the chromosome of mate 1.": "输入文件中配对1的染色体的字段编号（从1开始）。",
    "field number in the input file (starting from 1) of the position of mate 1.": "输入文件中配对1的位置的字段编号（从1开始）。",
    "field number in the input file (starting from 1) of the chromosome of mate 2.": "输入文件中第二个配对染色体的字段编号（从1开始）。",
    "field number in the input file (starting from 1) of the position of mate 2.": "输入文件中配对2的位置的字段编号（从1开始）。",
    "Select the output of csort_tabix.": "选择 csort_tabix 的输出。",
    "field number in the input file (starting from 1) of the chromosome of mate 2 (same as in csort).": "输入文件中配对2的染色体的字段编号（从1开始，与csort相同）。",
    "field number in the input file (starting from 1) of the position of mate 2 (same as in csort).": "输入文件中配对2的位置的字段编号（从1开始，与csort相同）。",
    "name of genome assembly": "基因组组装名称",
    "Select the output of makebins.": "选择 makebins 的输出。",
    "Select the input cool file to balance.": "选择输入的cool文件进行平衡。",
    "Ignore bins from the contact matrix using the 'MAD-max' filter: bins whose log marginal sum is less than ``mad-max`` median absolute deviations below the median log marginal sum of all the bins in the same chromosome.": "使用“MAD-max”过滤器忽略接触矩阵中的bins：如果bins的对数边缘和小于同一染色体上所有bins的对数边缘和的中位数对数边缘和减去“mad-max”倍的中位数绝对偏差，则忽略这些bins。",
    "Ignore bins from the contact matrix whose marginal number of nonzeros is less than this number.": "忽略接触矩阵中边缘非零数小于此数值的箱子。",
    "Ignore bins from the contact matrix whose marginal count is less than this number.": "忽略接触矩阵中边缘计数小于此数字的箱子。",
    "Number of diagonals of the contact matrix to ignore, including the main diagonal. Examples: 0 ignores nothing, 1 ignores the main diagonal, 2 ignores diagonals (-1, 0, 1), etc.": "忽略接触矩阵对角线的数量，包括主对角线。示例：0 不忽略任何对角线，1 忽略主对角线，2 忽略对角线（-1, 0, 1）等。",
    "Distance in bp to ignore.": "忽略的距离（以碱基对为单位）。",
    "Threshold value of variance of the marginals for the algorithm to converge.": "算法收敛的边际方差阈值。",
    "Maximum number of iterations to perform if convergence is not achieved..": "如果未达到收敛，则执行的最大迭代次数。",
    "Interactions to consider to calculate weights.": "需要考虑的相互作用以计算权重。",
    "Path to a 3-column BED file containing genomic regions to mask out during the balancing procedure, e.g. sequence gaps or regions of poor mappability.": "在平衡程序期间，用于遮蔽基因组区域的3列BED文件的路径，例如序列间隙或映射性差的区域。",
    "Name of column to write to.": "要写入的列名称。",
    "Convergence policy: What to do with weights when balancing doesn't converge in max_iters.": "收敛策略：在最大迭代次数max_iters内平衡不收敛时，如何处理权重。",
    "Output of HiCUP to convert": "HiCUP的输出需要转换",
    "Provide digester file": "提供摘要文件",
    "Digester file: Output of HiCUP Digester": "Digester 文件：HiCUP Digester 的输出",
    "Use middle of fragments instead of 5' of reads": "使用片段的中间部分而不是读段的5'端",
    "Choose the type of parameter for this field": "选择此字段的参数类型",
    "Enter text that should be part of the computed value": "输入应成为计算值一部分的文本",
    "Enter integer that should be part of the computed value": "输入应该包含在计算值中的整数",
    "Enter float that should be part of the computed value": "输入应为计算值的一部分的浮点数",
    "Select type of input parameter to match": "选择输入参数的类型以匹配",
    "Select how to handle unmapped values": "选择如何处理未映射的值",
    "Use this value if the input parameter value was not found in mappings": "如果映射中未找到输入参数值，请使用此值",
    "Select type of parameter to output": "选择输出参数的类型",
    "Assembly mode": "组装模式",
    "What parental information are you using?": "你正在使用什么亲本信息？",
    "Haplotype 1 reads": "单倍型1读取",
    "Haplotype 2 reads": "单倍型 2 读取",
    "Haplotype 1 read list": "单倍型1读取列表",
    "Haplotype 2 read list": "单倍型 2 读取列表",
    "Lower bound of the binned k-mer's frequency": "二进制k-mer频率的下限",
    "Upper bound of the binned k-mer's frequency": "分箱k-mer频率的上限",
    "Yak counter k-mer length": "Yak计数器 k-mer长度",
    "Utilize homology information to correct trio-phasing errors": "利用同源信息来纠正三人组分相错误",
    "Bits for bloom filter": "布隆过滤器的位数",
    "Cleaning rounds": "清洁轮次",
    "Length of adapters to be removed": "要移除的适配器的长度",
    "Minimum contig bubble size": "最小连通泡大小",
    "Minimum unitig bubble size": "最小单元泡大小",
    "Tip unitigs": "提示单元基因组",
    "Maximum overlap drop ratio": "最大重叠下降比率",
    "Minimum overlap drop ratio": "最小重叠下降比率",
    "Skip post join contigs step": "跳过连接后接续步骤",
    "Ignore error corrected reads and overlaps": "忽略校正错误的读取和重叠部分",
    "Homozygous read coverage": "纯合子读取覆盖度",
    "Options for purging duplicates": "去除重复选项",
    "Purge level": "清除级别",
    "Similarity threshold for duplicate haplotigs": "重复单体型相似性阈值",
    "Minimum overlapped reads for duplicate haplotigs": "重复单倍体最小重叠读数",
    "Coverage upper bound": "覆盖度上限",
    "Assumtion of haplotype number": "假设单倍型数量",
    "Options for Hi-C-partition": "Hi-C 分区的选项",
    "Hi-C R1 reads": "Hi-C R1 读取",
    "Hi-C R2 reads": "Hi-C R2 读取",
    "Rounds of reweighting Hi-C links. Increasing this may improves phasing results but takes longer time": "Hi-C 链接的重新加权轮数。增加这个参数可能会改善分相结果，但需要更长的时间。",
    "Rounds of perturbation. Increasing this may improves phasing results but takes longer time": "扰动轮数。增加这个参数可能会改善相位结果，但需要更长的时间。",
    "Fraction to flip for perturbation. Increasing this may improves phasing results but takes longer time": "用于扰动的翻转比例。增加这个比例可能会提高相位分析的结果，但需要更长的时间。",
    "Detect misjoined unitigs of greater than or equal to specified size": "检测大于或等于指定大小的错误连接单元序列",
    "Options for ONT integration": "ONT集成选项",
    "Ultra-long (>10kb) reads": "超长（>10kb）读取",
    "error rate of ultra-long reads (default 0.2)": "超长读取的错误率（默认值为0.2）",
    "remove tip unitigs composed of less than (INT) reads for the UL assembly (default 6)": "删除由少于(INT)个读取组成的末端单元，用于UL组装（默认为6）",
    "Hifiasm k-mer length": "Hifiasm k-mer 长度",
    "Drop k-mers": "删除k-mers",
    "Maximum overlaps to consider": "最大重叠数",
    "Correction rounds": "校正轮次",
    "Minimum count threshold": "最小计数阈值",
    "Maximum k-mer ocurrence": "最大k-mer出现次数",
    "Estimated haploid genome size": "估计的单倍体基因组大小",
    "Options for hifiasm scaffolding": "hifiasm 支架构建选项",
    "Max scaffolding gap size": "最大支架间隙大小",
    "Output log file?": "输出日志文件？",
    "Output .bin files (used for development and debugging)?": "输出 .bin 文件（用于开发和调试）？",
    "Cluster datasets": "聚类数据集",
    "Long-read datasets": "长读长数据集",
    "Min bases aligned": "最小碱基对齐",
    "Min read length covered by alignments": "最小读取长度由比对覆盖",
    "Cluster multi-FASTA dataset": "聚类多FASTA数据集",
    "Input contigs are not circular": "输入的连续片段不是环形的",
    "Reconcile initial check options": "调和初始检查选项",
    "Max Mash distance": "最大 Mash 距离",
    "Max relative length factor": "最大相对长度因子",
    "Reconcile circularization options": "调和循环化选项",
    "Max number of pb to add to circularize": "最大添加的碱基对数以实现圆化",
    "Max percentage of a contig length to add to circularize": "将循环化时添加到一个连续序列长度的最大百分比",
    "Max number of pb to trim to circularize": "最大修剪碱基对数以实现循环化",
    "Max percentage of a contig length to trim to circularize": "将一个连续序列修剪到圆环化的最大百分比",
    "Reconcile final check": "最终检查调和",
    "Min global alignment percentage identity": "最小全局比对序列相似性百分比",
    "Minimum allowed pairwise 1kbp window identity": "最小允许的成对1kbp窗口一致性",
    "Multiple sequence alignment (MSA) options": "多序列比对（MSA）选项",
    "Look-ahead margin": "前瞻余量",
    "Long-read dataset": "长读长数据集",
    "Number of subsampled reads set to output": "设置输出的子采样读取数",
    "Provide an estimated genomoe size": "提供估计的基因组大小",
    "Estimated total genome size": "估计的基因组总大小",
    "Minimum subset read depth": "最小子集读取深度",
    "Haplotypes for Trio Binning Assembly": "三重拼合装配的单倍型",
    "Haplotype input reads": "单倍型输入读段",
    "Shot name to identify your haplotype": "拍摄名称以识别您的单倍型",
    "Technology": "技术",
    "Processing": "处理",
    "To restrict canu to only a specific stage, use": "要限制canu仅运行到特定的阶段，使用",
    "Estimated genome size (e.g. 8.0m, 15k, 2g)": "估计的基因组大小（例如：8.0m，15k，2g）",
    "Maximum raw overlap mismatch": "最大原始重叠不匹配",
    "Maximum corrected overlap mismatch": "最大校正重叠不匹配",
    "Minimum Input Coverage": "最小输入覆盖度",
    "Target coverage for corrected reads": "校正后的读段目标覆盖度",
    "Additonal options": "附加选项",
    "Stop the assembly if read coverage is too low to be useful": "如果读取覆盖度太低而无法使用，则停止组装。",
    "Contig Filters": "Contig 过滤器",
    "Minimum reads": "最小读取数",
    "Maximum single read span (fraction)": "最大单次读取跨度（分数）",
    "Low coverage span (fraction)": "低覆盖度跨度（分数）",
    "Low coverage depth": "低覆盖深度",
    "Reference ID": "参考ID",
    "Read pool": "读数池",
    "Sample ID": "样本ID",
    "Iteration to start with": "开始迭代",
    "Iteration to end with": "迭代结束",
    "kmer for baiting stringency": "kmer用于诱饵的严格性",
    "Sequencing platform": "测序平台",
    "Run MIRA in denovo mode": "运行 MIRA 在 denovo 模式",
    "Extend readpool to contain full read pairs, even if only one member was baited": "将读段池扩展以包含完整的读对，即使只有一个成员被用作诱饵",
    "Split reference at positions with more than 5N": "在超过5N的位置处拆分参考序列",
    "Trim data": "修剪数据",
    "Trim overhang": "修剪突出部分",
    "Number of allowed mismatches in mapping": "映射中允许的不匹配数",
    "Minimum average coverage of contigs to be retained": "保留的基因组片段的最小平均覆盖度",
    "Minimum length of contig to be retained as backbone": "保留为骨干的最小长度的连续序列",
    "Fasta file input": "Fasta 文件输入",
    "Function mode": "功能模式",
    "PAF input file": "PAF 输入文件",
    "Base-level coverage file": "基本覆盖度文件",
    "Cutoffs file": "截止值文件",
    "Minimum fraction of haploid/diploid/bad/repetitive bases in a sequence": "序列中单倍体/双倍体/不良/重复基的比例最小值",
    "Minimum max match score": "最小最大匹配得分",
    "Minimum matching bases for chaining": "最小匹配碱基对用于链接",
    "Maximum gap size for chaining": "最大间隙大小用于链式连接",
    "Rounds of chaining": "链式循环",
    "Maximum gap size for second round of chaining": "第二轮链式比对的最大间隔大小",
    "Minimum chaining score for a match": "匹配的最小链式得分",
    "Maximum extension for contig ends": "最大延伸的连续末端",
    "Generate log file": "生成日志文件",
    "Assembly FASTA file": "组装FASTA文件",
    "PBCSTAT options": "PBCSTAT 选项",
    "Minimum mapping length ratio": "最小映射长度比率",
    "Flanking space": "侧翼空间",
    "Use only primary alignments": "仅使用主要比对",
    "BAM input file": "BAM 输入文件",
    "NGSCSTAT options": "NGSCSTAT 选项",
    "Depths input file": "深度输入文件",
    "BED input file": "BED 输入文件",
    "Keep high coverage contigs in the primary contig set": "保留高覆盖率的连续序列在主要连续序列集中",
    "Do not add prefix to haplotigs": "不要给 haplotigs 添加前缀",
    "Minimum primary contig length": "最小主连通块长度",
    "Minimum ratio of remaining primary contig length to the original contig length": "剩余主序列长度与原始序列长度的最小比率",
    "Trim end sequences": "修剪末端序列",
    "Split contigs": "拆分连续序列",
    "Minimum gap size between duplications": "最小重复间隔",
    "FASTQ file(s)": "FASTQ文件（们）",
    "Additional read files": "额外的读取文件",
    "Guide tree": "引导树",
    "Input genome": "输入基因组",
    "Genome label": "基因组标签",
    "Evaluation mode": "评估模式",
    "K-mer counts database": "K-mer计数数据库",
    "Number of assemblies": "组装数量",
    "Genome assembly": "基因组组装",
    "First genome assembly": "第一次基因组组装",
    "Second genome assembly": "第二次基因组组装",
    "F1 k-mer counts": "F1 k-mer 计数",
    "Paternal hap-mer database": "父系单倍型数据库",
    "Maternal hap-mer database": "母体Hap-mer数据库",
    "Identification label": "鉴定标签",
    "Add column header on tabular outputs": "在表格输出中添加列标题",
    "Width of plot": "图形宽度",
    "Height of plot": "绘图高度",
    "Maximum limit for k-mer multiplicity (leave 0 for default)": "k-mer 多重性的最大限度（保留 0 为默认值）",
    "Maximum limit for k-mer count (leave 0 for default)": "k-mer计数的最大限制（留0表示使用默认值）",
    "Assembly only counts": "组装仅计数",
    "Solid k-mer cutoffs": "固体 k-mer 截止值",
    "Type of plot (default all)": "绘图类型（默认全部）",
    "Genome assembly to polish": "基因组组装以提高质量",
    "Type of data used for polishing": "用于润色的数据处理类型",
    "First set of short reads": "第一组短读序列",
    "Second set of short reads": "第二组短读序列",
    "Paired collection of short reads": "配对的短读段集合",
    "Pre-aligned reads": "预对齐的读取",
    "Include unpolished sequences in final output": "在最终输出中包含未润色的序列",
    "Input dataset in SAM or BAM format": "输入SAM或BAM格式的数据集",
    "Sort order:": "排序顺序：",
    "Minimum mapping quality:": "最小映射质量：",
    "How to filter:": "如何过滤：",
    "Filter list": "过滤列表",
    "Initial assembly file": "初始组装文件",
    "Bed alignment": "Bed 对齐",
    "Sequence graphs": "序列图",
    "Enzyme selection": "酶选择",
    "Preconfigured enzymes": "预配置的酶",
    "Restriction enzyme sequence(s)": "限制性内切酶序列",
    "Clean Assembly": "清洁组装",
    "Expected Genome Size": "预期基因组大小",
    "Config file options": "配置文件选项",
    "Config file": "配置文件",
    "Report graphical fragment assembly?": "报告图形片段组装？",
    "Report configuration file": "报告配置文件",
    "Report log file?": "报告日志文件？",
    "Reads options": "读取选项",
    "MinReadLength": "最小读取长度",
    "Desired coverage": "期望覆盖度",
    "Palindromic reads options": "回文读段选项",
    "Skip flagging palindromic reads?": "跳过标记回文读取？",
    "maxSkip for palindromic read detection": "最大跳过值，用于回文读段检测",
    "maxDrift for palindromic read detection": "最大漂移值，用于检测回文序列读取",
    "maxMarkerFrequency for palindromic read detection": "最大标记频率用于回文读检测",
    "alignedFractionThreshold for palindromic read detection": "对于回文读段检测的对齐分数阈值",
    "nearDiagonalFractionThreshold for palindromic read detection": "nearDiagonalFractionThreshold 用于检测回文读段的阈值",
    "deltaThreshold for palindromic read detection": "用于检测回文读段的delta阈值",
    "Kmers options": "Kmers选项",
    "Method to generate marker k-mers": "生成标记k-mers的方法",
    "Length of marker k-mers (In run length space)": "标记k-mer的长度（在运行长度空间中）",
    "Fraction k-mers used as a marker": "作为标记使用的k-mers分数",
    "Enrichment threshold for defining cutoff in Kmers.generationMethod that select based on enrichment": "富集阈值，用于在基于富集的 Kmers.generationMethod 中定义截止值",
    "The file containing the k-mers to be used as markers, one kmer per line. Only used if the Kmers.generationMethod is 'Read from file'": "包含用作标记的k-mers的文件，每行一个k-mer。仅当Kmers.generationMethod为'Read from file'时使用。",
    "MinHash options": "MinHash 选项",
    "Version of the LowHash algorithm to use": "要使用的LowHash算法版本",
    "The number of consecutive markers that define a MinHash/LowHash feature": "定义一个MinHash/LowHash特征的连续标记的数量",
    "Defines how low a hash has to be to be used with the LowHash algorithm": "定义哈希值需要多低才能与LowHash算法一起使用",
    "The number of MinHash/LowHash iterations": "MinHash/LowHash迭代次数",
    "If --MinHash.minHashIterationCount is 0, Min hash iteration is stopped when the average number of alignment candidates that each read is involved in reaches this value.": "如果 --MinHash.minHashIterationCount 是 0，当每个读参与的比对候选者的平均数量达到这个值时，MinHash 迭代就会停止。",
    "The minimum bucket size to be used by the LowHash algorithm": "LowHash算法使用的最小桶大小",
    "The maximum bucket size to be used by the LowHash algorithm": "LowHash算法使用的最大桶大小",
    "The minimum number of times a pair of reads must be found by the MinHash/LowHash algorithm in order to be considered a candidate alignment": "MinHash/LowHash算法认为配对序列是候选对齐的最小出现次数",
    "Skip the MinHash algorithm and mark all pairs of reads as alignment candidates with both orientation?": "跳过MinHash算法，并将所有成对的读取标记为具有两种方向的对齐候选。",
    "Align options": "对齐选项",
    "The maximum number of markers that an alignment is allowed to skip": "允许对齐跳过的最大标记数",
    "The maximum amount of marker drift that an alignment is allowed to tolerate": "对齐允许容忍的最大标记漂移量",
    "The maximum number of unaligned markers tolerated at the beginning and end of an alignment": "在比对的开始和结束处，可以容忍的未对齐标记的最大数量",
    "Marker frequency threshold": "标记频率阈值",
    "The minimum number of aligned markers for an alignment to be used": "对齐标记的最小数量，以便使用对齐",
    "The minimum fraction of aligned markers for an alignment to be used": "对齐标记的最小比例，用于使用对齐",
    "Match score for marker alignments": "标记对齐的匹配得分",
    "Mismatch score for marker alignments": "标记对齐的不匹配分数",
    "Gap score for marker alignments": "标记对齐的间隙得分",
    "Amount to extend the downsampled band": "下采样带的扩展量",
    "Maximum alignment band": "最大对齐带宽",
    "Suppress containment alignments?": "抑制包含对齐？",
    "Same channel read alignment options": "相同通道读取比对选项",
    "Supress delta threshold": "抑制delta阈值",
    "ReadGraph options": "读取图形选项",
    "The method used to create the read graph": "用于创建读取图的方法",
    "The maximum number of alignments to be kept for each read": "每个读取保留的最大比对数",
    "Max chimeric read distance": "最大嵌合读段距离",
    "Maximum distance (edges) for flagCrossStrandReadGraphEdges": "最大距离（边）以标记CrossStrandReadGraphEdges",
    "Maximum number of alignments to be kept for each contained read": "每个包含的读取保留的最大对齐数",
    "Maximum number of alignments to be kept in each direction (forward, backward) for each uncontained read": "每个未包含的读取在每个方向（正向，反向）中要保留的最大对齐数",
    "Percentile for marker Count": "标记物计数的百分位数",
    "Percentile for alignedFraction": "对齐分数的百分位数",
    "Percentile for maxSkip": "最大跳过的百分位数",
    "Percentile for maxDrift": "最大漂移的百分位数",
    "Percentile for maxTrim": "最大修剪的百分位数",
    "MarkerGraph options": "MarkerGraph选项",
    "Minimum coverage (number of supporting oriented reads) for a marker graph vertex to be created.": "最小覆盖度（支持定向读取的数量）以创建标记图顶点。",
    "Maximum coverage (number of supporting oriented reads) for a marker graph vertex": "标记图顶点的最大覆盖度（支持的定向读取数）",
    "Minimum coverage (number of supporting oriented reads) for each strand for a marker graph vertex": "每个标记图顶点的每个链的最小覆盖度（支持定向读取的数量）",
    "Low coverage threshold": "低覆盖度阈值",
    "High coverage threshold": "高覆盖度阈值",
    "Max distance": "最大距离",
    "Edge marker skip threshold": "边缘标记跳过阈值",
    "Number of prune iterations": "修剪迭代次数",
    "Simplify max length": "简化最大长度",
    "Cross edge coverage threshold": "交叉边覆盖阈值",
    "Refine threshold": "细化阈值",
    "Perform reverse transitive reduction?": "执行反向传递约简？",
    "Peak Finder options": "峰值查找选项",
    "Peak finding min area fraction": "峰值发现最小面积分数",
    "Peak finding area start index": "峰值查找区域起始索引",
    "Maximum average edge coverage for a cross edge of the assembly graph to be removed": "组装图中要移除的交叉边的最大平均边覆盖",
    "Marker graph edge length threshold for consensus": "标记图边长阈值用于共识",
    "Consensus caller for repeat counts": "重复次数的共识调用器",
    "Custom config file for consensus caller": "自定义配置文件用于共识调用器",
    "Write the reads that contributed to assembling each segment?": "写出贡献于组装每个片段的读取内容？",
    "Detangle options": "解缠选项",
    "Method used to detangle the assembly graph": "用于解开组装图的方法",
    "Minimum number of reads on detangle matrix diagonal elements required for detangling": "解缠矩阵对角元素所需的最小读数",
    "Maximum number of reads on detangle matrix off-diagonal elements allowed for detangling": "允许在解缠矩阵非对角线上的最大读取数，以便解缠",
    "Maximum ratio of total off-diagonal elements over diagonal element allowed for detangling": "允许的最大非对角元素与对角元素比率，用于解缠",
    "Iterative assembly options (experimental)": "迭代组装选项（实验性）",
    "Request iterative assembly? (experimental)": "请求迭代组装？（实验性）",
    "Number of iterations for iterative assembly (experimental)": "迭代组装的迭代次数（实验性）",
    "Psuedopath alignment match score for iterative assembly (experimental)": "伪路径对齐匹配得分，用于迭代组装（实验性）",
    "Psuedopath alignment mismatch score for iterative assembly (experimental)": "伪路径对齐不匹配分数，用于迭代组装（实验性）",
    "Psuedopath alignment gap score for iterative assembly (experimental)": "伪路径对齐间隙分数，用于迭代组装（实验性）",
    "Mismatch square factor for iterative assembly (experimental)": "迭代组装的不匹配正方形因子（实验性）",
    "Minimum psuedo-alignment score for iterative assembly (experimental)": "迭代组装的最小伪比对分数（实验性）",
    "Maximum number of read graph neighbors for iterative assembly (experimental)": "迭代组装中读取图邻接节点的最大数量（实验性）",
    "Number of read graph bridge removal iterations for iterative assembly (experimental)": "迭代组装中用于去除读取图桥的迭代次数（实验性）",
    "Maximum distance for reag graph bridge removal for iterative assembly (experimental)": "用于迭代组装的最大距离以移除reag图桥（实验性）",
    "Choose Lordec program": "选择 Lordec 程序",
    "Lordec correct advanced options": "Lordec 高级选项",
    "Number of target k-mers": "目标k-mers数量",
    "Maximum number of branches to explore": "最大分支探索数",
    "Long read FASTA/Q file": "长读长 FASTA/Q 文件",
    "Use the split option": "使用分割选项",
    "Select read file type": "选择读取文件类型",
    "Select file with combined reads": "选择包含合并后的读取文件",
    "Select file with forward reads": "选择包含正向读取的文件",
    "Select file with reverse reads": "选择包含反向读取的文件",
    "Select file with seed sequence": "选择带有种子序列的文件",
    "Use reference genome": "使用参考基因组",
    "Select reference file": "选择参考文件",
    "Detect heteroplasmy": "检测异质性",
    "Set minimum minor allele frequency": "设置最小次等位基因频率",
    "PCR-free": "无PCR",
    "Variance detection": "方差检测",
    "Read options": "读取选项",
    "Set read length": "设置读取长度",
    "Set total insert size of your paired end reads": "设置你的配对末端读取的总插入大小",
    "Assembly type": "组装类型",
    "Select chloroplast sequence": "选择叶绿体序列",
    "Extend seeds instead of finding matching reads": "扩展种子而不是寻找匹配的读取",
    "Set length of overlap between matching reads": "设置匹配读取之间的重叠长度",
    "Optional options": "可选选项",
    "Finetune your insert size automatically?": "自动微调您的插入大小吗？",
    "Take quality scores into account?": "考虑质量分数吗？",
    "Retain original IDs when saving assemled reads?": "在保存组装后的读取时保留原始ID？",
    "Select your input option": "选择您的输入选项",
    "Single-end file(s)": "单端文件",
    "Interleaved-paired-end file(s)": "交错配对端文件（们）",
    "Mate 1 input reads": "Mate 1 输入读取",
    "Mate 2 input reads": "第二对输入读取",
    "Basic assembly options": "基本组装选项",
    "minimum multiplicity for filtering (k_min+1)-mers": "过滤（k_min+1）-mers的最小多重性",
    "K-mer specification method": "K-mer 指定方法",
    "Comma-separated list of kmer size": "逗号分隔的kmer大小列表",
    "Minimum kmer size": "最小kmer大小",
    "Maximum kmer size": "最大kmer大小",
    "Increment of kmer size of each iteration": "每个迭代kmer大小的增量",
    "Advanced assembly options": "高级组装选项",
    "Do not add mercy kmers": "不要添加仁慈的k-mer",
    "Intensity of bubble merging (0-2), 0 to disable": "气泡合并强度（0-2），0表示禁用",
    "Merge complex bubbles of length": "合并长度的复杂气泡",
    "Strength of low depth pruning": "低深度剪枝强度",
    "Remove unitigs with avg kmer depth less than this value": "删除平均kmer深度小于该值的单元序列",
    "Disconnect unitigs if its depth is less than this ratio times the total depth of itself and its siblings": "如果单元的深度小于它本身和其兄弟单元总深度的这个比率，则断开单元。",
    "Remove unitigs if its depth is less than this ratio times the average depth of the neighborhoods": "如果单元图的深度小于邻域平均深度的这个比例，则删除单元图",
    "Number of rounds for graph cleanning": "图清理的轮数",
    "Disable local assembly": "禁用本地组装",
    "Use 1pass mode to build SdBG of k_min": "使用1pass模式构建k_min的SdBG。",
    "Minimum length of contigs to output": "输出的最小连续序列长度",
    "Return intermediate contigs?": "返回中间的连续序列？",
    "Minimum size of transcript": "转录本的最小大小",
    "Maximum number of isoforms": "最大异构体数量",
    "Maximum size of component": "组件的最大尺寸",
    "Input reads type, collection or single library": "输入读取类型，集合或单个库",
    "Trim reads": "修剪序列",
    "Assembler to use": "使用的组装器",
    "Contig name format": "Contig名称格式",
    "List of kmer sizes to use": "要使用的kmer大小列表",
    "Extra SPAdes options": "额外的SPAdes选项",
    "Disable post-assembly correction": "禁用组装后校正",
    "Keep bam files": "保留 bam 文件",
    "Minimum contig coverage": "最小连通覆盖度",
    "Include unpolished": "包括未打磨的",
    "Fragment correction": "片段校正",
    "Error threshold": "错误阈值",
    "No trimming": "不进行修剪",
    "Reads in FASTA or FASTQ format": "读取FASTA或FASTQ格式的序列",
    "Size of a kmer": "k-mer的大小",
    "Min abundance threshold for solid kmers (default: 2)": "固体kmers的最小丰度阈值（默认：2）",
    "Max abundance threshold for solid kmers": "固体k-mers的最大丰度阈值",
    "Expected HiFi reads error rate": "预期的HiFi读取错误率",
    "Number of polishing iterations": "抛光迭代次数",
    "Minimum overlap between reads": "最小读取重叠",
    "Keep haplotypes": "保持单倍型",
    "Enable scaffolding using graph": "启用使用图的脚手架",
    "Perform metagenomic assembly": "执行宏基因组组装",
    "Reduced contig assembly coverage": "减少的连续片段组装覆盖度",
    "Reduced coverage for initial disjointing assembly": "初始解离组装的覆盖度降低",
    "Remove all non-primary contigs from the assembly": "从组装中移除所有非主要的序列片段",
    "Generate a log file": "生成一个日志文件",
    "Assembly mode?": "组装模式？",
    "Contigs/scaffolds file": "Contigs/scaffolds 文件",
    "FASTQ/FASTA file #1": "FASTQ/FASTA 文件 #1",
    "FASTQ/FASTA file #2": "FASTQ/FASTA 文件 #2",
    "FASTQ/FASTA files": "FASTQ/FASTA 文件",
    "Contigs/scaffolds": "连续序列/支架序列",
    "Type of assembly": "组装类型",
    "Use a reference genome?": "使用参考基因组？",
    "Genomic feature positions in the reference genome": "参考基因组中的基因组特征位置",
    "Operon positions in the reference genome": "参考基因组中的操纵子位置",
    "Compute k-mer-based quality metrics?": "计算基于k-mer的质量指标？",
    "Size of k": "k的大小",
    "Generage Circos plot": "生成Circos图",
    "Estimated reference genome size (in bp) for computing NGx statistics": "估计参考基因组大小（以碱基对为单位），用于计算NGx统计数据",
    "Type of organism": "生物类型",
    "Maximum number of reference genomes (per each assembly) to download after searching in the SILVA databa": "在 SILVA 数据库中搜索后，要下载的参考基因组的最大数量（每个组装）",
    "Comma-separated list of reference genomes": "逗号分隔的参考基因组列表",
    "Reuse the alignments on the combined reference": "在组合参考上重用比对",
    "Lower threshold for a contig length (in bp)": "较小的基因组片段长度阈值（以碱基对为单位）",
    "Are assemblies scaffolds rather than contigs?": "组装是脚手架而不是连续序列吗？",
    "Is genome large (> 100 Mbp)?": "基因组是否较大（大于100 Mbp）？",
    "Tool for gene prediction": "基因预测工具",
    "Enables ribosomal RNA gene finding?": "启用核糖体RNA基因查找？",
    "Enables search for Universal Single-Copy Orthologs using BUSCO?": "使用BUSCO搜索通用单拷贝直系同源基因？",
    "Alignments": "比对",
    "Use all alignments as in QUAST v.1.*. to compute genome fraction, # genomic features, # operons metrics?": "使用QUAST v.1.*版本中的所有比对来计算基因组分数、基因组特征数量、操纵子数量指标？",
    "Minimum length of alignment": "对齐的最小长度",
    "How processing equally good alignments of a contig (probably repeats)?": "如何处理等同良好的连续片段（可能是重复序列）的比对？",
    "Score S for defining equally good alignments of a single contig": "得分 S 用于定义单个连续片段的同样好的对齐",
    "Fragmented reference genome": "片段化的参考基因组",
    "Simulate upper bound assembly": "模拟上限组装",
    "Minimal number of 'connecting reads' needed for joining upper bound contigs into a scaffold": "连接读取的最小数量，用于将上限的连续片段连接成支架。",
    "Minimal local misassembly size": "最小局部错误组装大小",
    "Comma-separated list of contig length thresholds (in bp)": "逗号分隔的连续片段长度阈值列表（以碱基对为单位）",
    "Break contigs at every misassembly event (including local ones) to compute NAx and NGAx statistics?": "在每个错误组装事件（包括局部错误组装）处打断连续序列，以计算NAx和NGAx统计数据？",
    "Lower threshold for the relocation size (gap or overlap size between left and right flanking sequence)": "重新定位大小的下限阈值（左右相邻序列之间的间隙或重叠大小）",
    "Max allowed scaffold gap length difference for detecting corresponding type of misassemblies": "允许的最大支架间隙长度差异，用于检测相应类型的错配",
    "Lower threshold for detecting partially unaligned contigs": "检测部分未对齐的连续片段的较低阈值",
    "Distinguish contigs with more than 50% unaligned bases as a separate group of contigs?": "将超过50%的碱基未比对的连续序列区分为一个单独的连续序列组？",
    "Fragment max indent": "片段最大缩进",
    "Report all metrics": "报告所有指标",
    "Report Nx, Lx, etc metrics for specific value of 'x'": "报告特定值 'x' 的 Nx、Lx 等指标",
    "Sequence Reads": "序列读取",
    "PAF file": "PAF 文件",
    "Preselection options": "预选选项",
    "Drop mappings having less than this number of matching bases": "删除匹配碱基数量少于该数字的映射",
    "During read filtering, ignore mappings with col10/col11 below this value": "在读取过滤过程中，忽略列10/列11低于此值的映射",
    "Drop mappings shorter than this number bp": "丢弃短于这个数目碱基对的映射",
    "Minimal coverage by other reads": "最小覆盖度由其他读取",
    "Overlapping options": "重叠选项",
    "Minimal overlap length [same as minSpan]": "最小重叠长度【与minSpan相同】",
    "Maximum overhang length": "最大突出长度",
    "Minimal ratio of mapping length to mapping+overhang length for a mapping considered a containment or an overlap": "映射长度与映射加延伸长度的最小比率，用于考虑映射是包含还是重叠",
    "Graph layout options": "图形布局选项",
    "Maximal gap differences between two reads in a mapping": "两个比对中读取之间的最大间隙差异",
    "Maximal probing distance for bubble popping": "最大探测距离用于气泡破裂",
    "A unitig is considered small if it is composed of less than this many reads": "如果一个单元由少于这个数量的读取组成，则认为它是小的",
    "Rounds of short overlap removal": "短重叠片段去除轮次",
    "Overlap drop ratio threshold after short unitig removal": "短单元去除后重叠下降比率阈值",
    "Paired-end library": "双端测序文库",
    "Mate-pair library": "配对末端库",
    "Long sequences": "长序列",
    "K-mer length (in bp)": "K-mer长度（以碱基对为单位）",
    "The length (in bp) of a single k-mer in a k-mer pair": "单个k-mer对中单个k-mer的长度（以碱基对为单位）",
    "Minimum base quality when trimming": "修剪时的最小碱基质量",
    "Minimum erosion k-mer coverage": "最小腐蚀k-mer覆盖度",
    "Minimum erosion k-mer coverage per strand": "最小侵蚀k-mer覆盖度每条链",
    "Maximum length of blunt contigs to trim": "最大长度的平端接合序列进行修剪",
    "Minimum mean k-mer coverage of a unitig": "单元IGS的最小均值k-mer覆盖度",
    "Maximum length of a bubble (in bp)": "最大气泡长度（以碱基对为单位）",
    "Assemble in strand-specific mode": "在特定的链模式下进行组装",
    "Minimum overlap of two unitigs (in bp)": "两个单元序列的最小重叠长度（以碱基对为单位）",
    "Minimum sequence identity of a bubble": "最小序列相似性泡沫",
    "Maximum number of branches of a bubble": "最大分支数",
    "Minimum alignment length of a read (in bp)": "读取的最小比对长度（以碱基对为单位）",
    "Minimum unitig length required for building contigs (in bp)": "构建连续序列所需的最小单元长度（以碱基对为单位）",
    "Minimum number of pairs required for building contigs": "构建连续序列所需的最小配对数",
    "Allowable error of a distance estimate (in bp)": "允许的距离估计误差（以碱基对为单位）",
    "Minimum contig size required for building scaffolds (in bp)": "构建支架所需的最小连续片段大小（以碱基对为单位）",
    "Minimum number of pairs required for building scaffolds": "构建支架所需的最小配对数",
    "Paired or Single end data?": "配对还是单端数据？",
    "Select unpaired reads": "选择未配对的读取",
    "Select long reads. If there are no long reads, leave this empty": "选择长读段。如果没有长读段，此处留空。",
    "Select Bridging mode": "选择桥接模式",
    "Exclude contigs from the FASTA file which are shorter than this length (bp)": "排除FASTA文件中长度小于该值（碱基对）的序列片段",
    "The expected number of linear (i.e. non-circular) sequences in the assembly": "在组装中预期的线性（即非圆形）序列的数量",
    "Unicycler will not use segments shorter than this as scaffolding anchors": "Unicycler不会使用比这个长度短的片段作为支架锚点",
    "SPAdes options": "SPAdes选项",
    "Lowest k-mer size for SPAdes assembly, expressed as a fraction of the read length": "SPAdes组装的最小k-mer大小，以读长的比例表示",
    "Highest k-mer size for SPAdes assembly, expressed as a fraction of the read length": "SPAdes组装中最大的k-mer大小，以读长的比例表示",
    "Exact k-mers to use for SPAdes assembly, comma-separated": "用于SPAdes组装的确切k-mers，用逗号分隔",
    "Number of k-mer steps to use in SPAdes assembly": "在SPAdes组装中使用的k-mer步数",
    "Filter out contigs lower than this fraction of the chromosomal depth": "过滤掉小于染色体深度这个分数的连续序列",
    "Only keep the largest connected component of the assembly graph": "只保留组装图中最大的连通组件",
    "Rotation options": "旋转选项",
    "Do not rotate completed replicons to start at a standard gene.": "不要将完成的复制子旋转到标准基因的起始位置。",
    "FASTA file of genes for start point of rotated replicons": "FASTA文件，包含用于旋转复制起点的基因",
    "The minimum required BLAST percent identity for a start gene search": "BLAST搜索起始基因所需的最小百分比一致性",
    "The minimum required BLAST percent coverage for a start gene search": "BLAST搜索起始基因所需的最小百分比覆盖度",
    "Graph cleaning options": "图清理选项",
    "Unbridged graph components smaller than this size will be removed from the final graph": "小于这个大小的未桥接图组件将从最终图中移除",
    "Graph dead ends smaller than this size will be removed from the final graph": "小于这个大小的图的末端将从最终图中移除",
    "Long read alignment parameters": "长读序列比对参数",
    "FASTA file of known contamination in long reads, e.g. lambda, phiXm or puc18 spike-ins.": "已知污染的长读序列FASTA文件，例如lambda、phiXm或puc18的尖峰插入。",
    "Comma-delimited string of alignment scores: match, mismatch, gap open, gap extend": "逗号分隔的对齐分数字符串：匹配、不匹配、间隔开放、间隔延伸",
    "Score threshold - alignments below this are considered poor": "得分阈值 - 低于此值的比对被认为是较差的",
    "Simple long-read bridging": "简单的长读桥接",
    "Outputs to keep": "保留输出",
    "Reuse checkpoint files from earlier runs": "重用早期运行的检查点文件",
    "Checkpoint file": "检查点文件",
    "Checkpoint": "检查点",
    "Minimum size of region in reference genome": "参考基因组中区域的最小大小",
    "Maximum gap in reference": "参考序列中的最大间隙",
    "FASTA/FASTQ file(s)": "FASTA/FASTQ 文件（们）",
    "Start k-mer size": "起始 k-mer 大小",
    "End k-mer size": "终止 k-mer 大小",
    "K-mer search step size": "K-mer搜索步长",
    "Input files": "输入文件",
    "Single or paired end reads": "单端或配对端读取",
    "Select the reads": "选择读取",
    "Select set of interleaved reads": "选择一组交错的读取",
    "K-mer optimisation function": "K-mer优化函数",
    "Coverage cutoff optimisation function": "覆盖度截断优化函数",
    "Other velvetg options": "其他 velvetg 选项",
    "Minimum coverage cutoff": "最小覆盖度截断值",
    "Maximum coverage cutoff": "最大覆盖度截止值",
    "Source for reference genome used for BAM alignments": "用于BAM比对的参考基因组来源",
    "Type automatically determined by pilon": "类型由 pilon 自动确定",
    "Variant calling mode": "变异检测模式",
    "Create changes file": "创建变更文件",
    "Paired end fragments": "配对末端片段",
    "Input BAM file (paired end fragments)": "输入BAM文件（配对端片段）",
    "Mate pairs": "配对端",
    "Input BAM file (mate pairs)": "输入BAM文件（配对序列）",
    "Unpaired reads": "未配对的读取",
    "Input BAM file (unpaired)": "输入BAM文件（未配对）",
    "VCF output": "VCF输出",
    "Output annotation tracks": "输出注释轨迹",
    "QE (not QP) in VCF": "QE（不是QP）在VCF",
    "Issues that pilon should try and fix": "Pilon 应该尝试修复的问题",
    "Organism is diploid": "生物体是二倍体",
    "Use duplicates": "使用重复项",
    "Use IUPAC codes in FASTA output": "在FASTA输出中使用IUPAC代码",
    "Use low quality reads": "使用低质量的读取",
    "List of targets to process (leave blank for all)": "目标列表以进行处理（留空则为全部）",
    "Verbose output (in tool log)": "详细输出（在工具日志中）",
    "Default base quality": "默认碱基质量",
    "Flanking bases to ignore": "忽略的两侧碱基",
    "Allowable gap margin": "允许的间隙边距",
    "Kmer size": "Kmer大小",
    "Minimum depth": "最小深度",
    "Mininum gap size": "最小间隙大小",
    "Disable 'stray read filtering'": "禁用“杂散读取过滤”",
    "Select input reads from history": "从历史记录中选择输入的读取数据",
    "Sequencing Technology": "测序技术",
    "Assembly Options": "组装选项",
    "Min read length": "最小读取长度",
    "Homopolymer-compressed kmer size": "同聚物压缩的kmer大小",
    "Max kmer frequency": "最大kmer频率",
    "Min read similarity": "最小读取相似度",
    "Min edge depth": "最小边深度",
    "Realignment": "重新比对",
    "Contained reads": "包含的读取",
    "Consensus Options": "共识选项",
    "Expected length of node": "节点的预期长度",
    "Insertion score": "插入分数",
    "Deletion score": "删除分数",
    "Tri-base match bonus": "三碱基匹配奖励",
    "Homopolymer merge score used in dp-call-cns mode": "在 dp-call-cns 模式中使用的同聚物合并分数",
    "POA Bandwidth (Wmin,Wmax,mat_rate)": "POA 带宽 (Wmin, Wmax, mat_rate)",
    "Window size in the middle of the first read for fast align remaining reads. If -W is negative, will disable fast align, but use the abs(-W) as Band align score cutoff": "在第一个读取的中间设置窗口大小，以快速对齐剩余读取。如果-W是负数，将禁用快速对齐，但使用绝对值(-W)作为带对齐得分截止值",
    "Min size of aligned size in window. Will default to -W * 0.5": "窗口中对齐大小的最小值。默认为 -W * 0.5",
    "Abort TriPOA": "终止 TriPOA",
    "Shuffle mode": "洗牌模式",
    "Realignment bandwidth": "重定位带宽",
    "Consensus mode": "共识模式",
    "Min count of bases to call a consensus base": "调用共识碱基的最小碱基计数",
    "Min frequency of non-gap bases to call a consensus base": "最小非间隙碱基频率以调用共识碱基",
    "Max number of reads in PO-MSA": "在PO-MSA中的最大读取数",
    "Type of library?": "文库类型？",
    "Single-read files": "单读文件",
    "Interleaved paired-read files": "交错配对读取文件",
    "Only build overlap graph?": "仅构建重叠图？",
    "Only simplify existing overlap graph?": "仅简化现有的重叠图？",
    "Minimum overlap length (bp) required to insert an edge between two reads during graph construction": "在图构建过程中，将两个读取之间插入边所需的最小重叠长度（碱基对）",
    "Print scaffolds?": "打印支架？",
    "Print unused reads?": "打印未使用的读取？",
    "Print GFA graph format?": "打印 GFA 图格式？",
    "Print GFA2 graph format?": "打印 GFA2 图形格式？",
    "Parameters for 1st assembly iteration": "第一次组装迭代的参数",
    "Parameters for 2nd assembly iteration": "第二次组装迭代的参数",
    "Parameters for 3rd assembly iteration": "第三次组装迭代的参数",
    "FASTQ RNA-seq file(s)": "FASTQ RNA-seq 文件（们）",
    "Set strand specificity": "设置链特异性",
    "Select optional output file(s)": "选择可选的输出文件",
    "Operation type selector": "操作类型选择器",
    "Count operations": "计数操作",
    "K-mer size selector": "K-mer大小选择器",
    "Tolerable collision rate": "可容忍碰撞率",
    "Filter operations": "过滤操作",
    "Input meryldb": "输入 meryldb",
    "Type of filtering": "过滤类型",
    "Number of times in the input": "输入中的次数",
    "Frequency": "频率",
    "Arithmetic operations": "算术运算",
    "Operand": "操作数",
    "Operations on sets of k-mers": "k-mer集合上的操作",
    "F1 reads": "F1 读取",
    "Paternal reads": "父系读取",
    "Maternal reads": "母体读取",
    "Assembly method": "组装方法",
    "Assembly quality grade": "组装质量等级",
    "Backbones/reference chromosomes?": "参考基因组骨架/参考染色体",
    "Backbone/reference sequences": "骨干/参考序列",
    "Sanger/Capillary reads?": "Sanger/毛细管测序读段？",
    "Sanger/Capillary reads file": "Sanger/毛细管测序读取文件",
    "454 reads?": "454测序读取？",
    "Roche 454 reads file": "罗氏454读取文件",
    "Solexa/Illumina reads?": "Solexa/Illumina 测序读取？",
    "Solexa/Illumina reads file": "Solexa/Illumina 测序读取文件",
    "Ion Torrent reads?": "Ion Torrent 测序数据？",
    "Ion Torrent reads file": "Ion Torrent 读取文件",
    "Edena overlap (.ovl) file (-e)": "Edena重叠文件（.ovl）（-e）",
    "Overlap cutoff (-m)": "重叠截止值（-m）",
    "Contextual cleaning of spurious edges (-cc)": "上下文清理错误边缘（-cc）",
    "Discard non usable nodes (-discardNonUsable)": "丢弃不可用的节点（-discardNonUsable）",
    "Minimum size of the contigs to output (-c)": "输出的最小基因组片段大小（-c）",
    "Minimum required coverage for the contigs (-minCoverage)": "最小必需的序列覆盖度（-minCoverage）",
    "Coverage cutoff for contigs ends (-trim)": "Contigs末端的覆盖度截止值（-trim）",
    "Maximum search distance for paired-end (forward-reverse) sampling (-sph)": "配对末端（正向-反向）采样的最大搜索距离（-sph）",
    "Maximum search distance for mate-pair (reverse-forward) sampling (-lph)": "配对末端（反向-正向）抽样的最大搜索距离（-lph）",
    "Unpaired input": "非配对输入",
    "Unpaired file": "未配对文件",
    "Paired-end (forward-reverse) file": "成对末端（正向-反向）文件",
    "Paired-end file 1": "配对端文件1",
    "Paired-end file 2": "配对末端文件2",
    "Mate-pair (reverse-forward) file": "配对序列（反向-正向）文件",
    "Mate-pair file 1": "配对文件 1",
    "Mate-pair file 2": "配对文件 2",
    "Minimum overlap size to compute (-M)": "最小重叠大小以计算（-M）",
    "3' end reads truncation (-t)": "3'端读取截断（-t）",
    "Contigs file": "Contigs 文件",
    "Paired-end Illumina library": "配对末端Illumina文库",
    "Maximum number of mismatches when aligning reads on contigs with Bowtie (-v)": "在Bowtie上对齐读取的片段时，允许的最大不匹配数（-v）",
    "If the number of times a read and its reverse complement appear in the library is equal to or more than this value, the pairing information from that read will be disregarded (-c)": "如果一个测序读取及其反向互补序列在库中出现的次数等于或超过这个值，那么该读取的配对信息将被忽略（-c）",
    "Minimum number of links between two contigs (-w)": "两个拼接片段之间最小链接数（-w）",
    "Minimum length of contigs to be used in scaffold assembly (-L)": "用于支架组装的最小长度的拼接序列（-L）",
    "h value (-h)": "h值(-h)",
    "Kind of reads (-p)": "读取类型（-p）",
    "Input FASTA file (read 1)": "输入FASTA文件（读1）",
    "Input FASTA file (read 2)": "输入FASTA文件（读2）",
    "Library insert size": "文库插入片段大小",
    "Minimum number of links (read pairs) to compute scaffold (-k)": "最小链接数（读对）以计算支架（-k）",
    "Error (%) allowed on mean distance (-e)": "允许在平均距离上的错误百分比（-e）",
    "Maximum link ratio between two best contig pairs (-a)": "两个最佳重叠群对之间的最大链接比率（-a）",
    "Minimum overlap required between contigs to merge adjacent contigs in a scaffold (-x)": "最小重叠要求是在将相邻的片段合并到一个脚手架中所需的（-x）",
    "FASTA file containing sequences to use as seeds exclusively (-s)": "包含用作种子的序列的FASTA文件（-s）",
    "Minimum depth of coverage allowed for contigs (-w)": "允许的最小覆盖深度（-w）",
    "Minimum number of overlapping bases with the seed/contig during overhang consensus build up (-m)": "在构建延伸一致序列时，与种子/片段重叠的最小碱基数目（-m）",
    "Minimum number of reads needed to call a base during an extension (-o)": "在扩展期间调用碱基所需的最小读取数（-o）",
    "Minimum base ratio used to accept a overhang consensus base (-r)": "用于接受悬空一致性碱基的最小碱基比率（-r）",
    "Input sequence as FASTA": "输入序列为FASTA格式",
    "Minimum number of repeats a CRISPER must have": "CRISPER必须具有的最小重复次数",
    "Minimum length of a CRISPER repeated region": "CRISPR重复区域的最小长度",
    "Maximum length of a CRISPER repeated region": "CRISPR重复区域的最大长度",
    "Minimum length of CRISPER non repeated region": "CRISPR 非重复区域的最小长度",
    "Maximum length of CRISPER non repeated region": "CRISPR 非重复区域的最大长度",
    "Length of search window to used to discover CRISPERs": "搜索窗口的长度，用于发现CRISPRs",
    "Additional output format": "附加输出格式",
    "Input histogram file": "输入直方图文件",
    "Ploidy for model to use": "模型使用的倍性",
    "K-mer length used to calculate k-mer spectra": "用于计算k-mer谱的k-mer长度",
    "Initial k-mer coverage estimate for model to use": "模型使用的初始 k-mer 覆盖度估计",
    "Maximum k-mer coverage threshold": "最大k-mer覆盖阈值",
    "Turn off yellow unique sequence line in plots": "关闭图表中黄色唯一序列线条",
    "Topological relationships between the homologous chromosomes": "同源染色体之间的拓扑关系",
    "Initial value for repetitiveness": "重复性的初始值",
    "Initial values for nucleotide heterozygosity rates": "核苷酸杂合性率的初始值",
    "Parameter for the exponent when fitting a transformed (x**transform_exp*y vs. x) k-mer histogram": "拟合一个变换后的（x**transform_exp*y vs. x）k-mer直方图时，指数的参数",
    "Create testing.tsv file with model parameters": "创建一个名为 testing.tsv 的文件，其中包含模型参数。",
    "Set true simulated parameters for testing mode": "设置测试模式下的真实模拟参数",
    "Turn on printing of iteration progress of nlsLM function": "开启nlsLM函数迭代进度的打印",
    "Number of optimization rounds": "优化轮数",
    "NGS FASTA": "高通量测序(NGS) FASTA",
    "BioNano CMAP": "BioNano CMAP",
    "Configuration mode": "配置模式",
    "Restriction enzyme": "限制性内切酶",
    "Conflict resolution file": "冲突解决文件",
    "Genome maps conflict filter": "基因组映射冲突过滤器",
    "Sequences conflict filter": "序列冲突过滤器",
    "Provide BisCot required files": "提供 BisCot 所需的文件",
    "Generate an output package in ZIP format": "生成一个ZIP格式的输出包",
    "Remove BioNano cut sites": "移除 BioNano 切割位点",
    "First set of reads": "第一组读取",
    "Second set of reads": "第二组读段",
    "Email": "电子邮件",
    "Minimal appropriate length": "最小适当长度",
    "Search for an exact species": "搜索确切的物种",
    "Input mode": "输入模式",
    "Pacbio Hifi reads": "Pacbio Hifi 读取",
    "Number of bits for bloom filter": "布隆过滤器的位数",
    "Assembled contigs/scaffolds": "组装的连续序列/支架",
    "Close-related mitogenome in fasta format": "密切相关的线粒体基因组在fasta格式中",
    "Close-related mitogenome in genbank format": "基因组库格式中的密切相关的线粒体基因组",
    "Blast percentage identity": "Blast 序列相似度百分比",
    "Estimated mtDNA size": "估计的线粒体DNA大小",
    "Circularization offset": "循环化偏移",
    "Optional outputs": "可选输出",
    "Length of minimizers used to find overlaps": "用于查找重叠的最小化子长度",
    "Length of sliding window from which minimizers are sampled": "采样最小值的滑动窗口长度",
    "Threshold for ignoring most frequent minimizers": "忽略最频繁最小子的阈值",
    "Number of times racon polishing is invoked": "racon 抛光被调用的次数",
    "output graphical fragment assembly file?": "输出图形化片段组装文件？",
    "Use validation reads": "使用验证读取",
    "Validation reads": "验证读取",
    "Read type": "读取类型",
    "Coverage validation window size": "覆盖度验证窗口大小",
    "Break sequences at regions at or above this coverage level": "在覆盖度达到或超过此水平的区域断开序列",
    "Break sequences at regions at or below this coverage level": "在覆盖度达到或低于此水平的区域断开序列",
    "Minimum break distance from contig ends": "最小断裂距离，从连续序列末端",
    "Break misassebly option": "断开错误组装选项",
    "Don't break sequences within GFF intervals": "不要在GFF区间内打断序列",
    "Minimum grouping confidence score": "最小分组置信度得分",
    "Minimum location confidence score": "最小位置置信分数",
    "Minimum orientation confidence score": "最小方向置信度得分",
    "Infer gap sizes": "推断间隔大小",
    "Minimum infered gap size": "最小推断间隔大小",
    "Maximum inferred gap size": "最大推断间隔大小",
    "Concatenate unplaced contigs and make 'chr0' (-C)": "将未定位的连续片段连接起来，并创建'chr0'（-C）",
    "List of query headers to leave unplaceds and exclude form 'chr0'": "查询头列表，用于保留未放置的并从'chr0'中排除",
    "Minimum merged alignment length": "最小合并比对长度",
    "Maximum merged alignment distance": "最大合并对齐距离",
    "Patching mode": "补丁模式",
    "Scaffold AGP files": "脚手架 AGP 文件",
    "Merging options": "合并选项",
    "List of query headers to leave unplaced": "查询头列表，保持未放置状态",
    "Minimum assembly sequence length": "最小组装序列长度",
    "Function for merging gap lengths": "合并间隔长度的函数",
    "HI-C options": "HI-C选项",
    "Hi-C alignments": "Hi-C 对齐",
    "Restriction enzymes/sites or 'DNase'": "限制性内切酶/位点或'DNase'",
    "Portion of the sequence termini to consider for links": "序列末端用于链接的部分",
    "Scaffold file": "脚手架文件",
    "Error correction": "错误校正",
    "Number of Racon error-correction rounds": "Racon纠错轮数",
    "Illumina reads": "Illumina 读取",
    "Number of Pilon error-correction rounds": "Pilon错误校正轮数",
    "Type of third generation reads": "第三代测序读段类型",
    "Minimum identity for filtering candidate sequences": "候选序列过滤的最小一致性",
    "Minimum matched length for filtering candidate sequences": "过滤候选序列的最小匹配长度",
    "Chunks for error correction": "错误校正的区块",
    "Gap-size diff check": "Gap-size差异检查",
    "Illumina FASTQ files": "Illumina 快速质量文件",
    "Draft genome assembly": "草图基因组组装",
    "BAM with illumina read alignments": "带有Illumina读取对齐的BAM文件",
    "BAM with ONT reads aligned": "BAM文件，包含已对齐的ONT读取数据",
    "Aproximate mean coverage of the short reads": "短读长的近似平均覆盖度",
    "Aproximate size of the genome": "基因组的大致大小",
    "Type of short reads": "短读序列类型",
    "Score for matching bases for short reads": "短读长匹配碱基的得分",
    "Score for mismatching bases for short reads": "短读长不匹配碱基的得分",
    "Gap penalty for short reads": "短读段的间隙惩罚",
    "Score for matching bases for long reads": "长读长匹配碱基的得分",
    "Score for mismatching bases for long reads": "长读长不匹配碱基的得分",
    "Gap penalty for long reads": "长读长间隔的惩罚",
    "Threshold for NED": "NED的阈值",
    "Threshold for mapping quality of reads": "读取映射质量的阈值",
    "Number of contigs to be processed in one batch": "一批要处理的基因组序列片段数量",
    "FASTA/FASTQ RNA-seq file(s)": "FASTA/FASTQ RNA序列文件",
    "Scaffolding options": "支架选项",
    "k-mer length": "k-mer长度",
    "Step": "步骤",
    "Maximum link ratio": "最大链接比率",
    "Number of links": "链接数量",
    "Bloom filter false positive rate": "布隆过滤器误报率",
    "Which function would you like to run?": "您想要运行哪个功能？",
    "Input contig sequences": "输入连续序列",
    "Alignment file of Hi-C reads to contigs": "Hi-C 读取与片段比对文件",
    "Input AGP file (for rescaffolding)": "输入 AGP 文件（用于重新组装）",
    "Resolutions": "分辨率",
    "Restriction enzyme used in Hi-C experiment": "Hi-C实验中使用的酶切酶",
    "Minimum contig length included for scaffolding": "用于支架化的最小Contig长度",
    "Minimum read mapping quality (for BAM input only)": "最小读取映射质量（仅限BAM输入）",
    "Skip assembly error correction step": "跳过组装错误校正步骤",
    "Skip assembly/scaffolding error check each round": "跳过每轮的组装/支架错误检查",
    "Contig fasta file": "Contig 指的是基因组测序中，由短序列拼接而成的较长序列。fasta 文件是一种文本格式，用于表示核酸或蛋白质序列。所以 \"Contig fasta file\" 指的是包含基因组测序中拼接得到的较长序列的 fasta 文件。",
    "Output fasta line length": "输出FASTA行长度",
    "Velvet Dataset": "Velvet 数据集",
    "Coverage cutoff": "覆盖度截断",
    "Remove nodes with coverage below": "移除覆盖度低于的节点",
    "Tracking of short read positions in assembly": "短读序列在组装中的位置跟踪",
    " Set minimum contig length": "设置最小片段长度",
    "minimum contig length": "最小连通体长度",
    "Expected Coverage of Unique Regions": "唯一区域的预期覆盖度",
    "Using Paired Reads": "使用配对读取",
    "Insert Length in Paired-End Read dataset (ignored when -1)": "配对末端读取数据集中的插入长度（当为-1时忽略）",
    "Velvet Advanced Options": "Velvet 高级选项",
    "Estimate of Standard Deviation of Paired-End Read dataset(ignored when -1)": "配对末端读取数据集的标准差估计（当为-1时忽略）",
    "Insert Length in 2nd Paired-End Short Read dataset (ignored when -1)": "在第二组配对端短读数据集中插入长度（当为-1时忽略）",
    "Estimate of Standard Deviation of 2nd Paired-End Read dataset (ignored when -1)": "估计第二对端读取数据集的标准偏差（当为-1时忽略）",
    "Insert Length in Long Paired-End Read dataset (ignored when -1)": "长配对末端测序数据集中的插入长度（当为-1时忽略）",
    "Maximum branch length (ignored when -1)": "最大分支长度（当为-1时忽略）",
    "Maximum max_divergence (between .0 and 1., ignored when -1.)": "最大差异度（在0到1之间，当为-1时忽略）",
    "Maximum gap count (ignored when -1)": "最大间隙计数（当为-1时忽略）",
    "Minimum read-pair count (ignored when -1)": "最小配对读段数（当为-1时忽略）",
    "Maximum coverage exclusion(ignored when -1.)": "最大覆盖排除（当为-1时忽略）。",
    "Minimum number of long reads required to merge contigs (ignored when -1)": "最小长读段数量以合并连续片段（当为-1时忽略）",
    "Use Scaffolding": "使用 Scaffolding",
    "Hash Length": "哈希长度",
    "Use strand specific transcriptome sequencing": "使用特定链的转录组测序",
    "Input Files": "输入文件",
    "Choose the input type": "选择输入类型",
    "read type": "读取类型",
    "Input sequences to assemble": "输入序列以组装",
    "specify overlap length cutoff > 15 (40)": "指定重叠长度截止值大于15（40）",
    "specify overlap percent identity cutoff N > 65 (90)": "指定重叠百分比一致性截断值 N > 65（90）",
    "File inputs": "文件输入",
    "Sequencing reads": "测序读段",
    "Kmer dump": "Kmer转储",
    "Meryl histogram": "梅丽尔直方图",
    "Meryl database": "梅丽尔数据库",
    "Homozygous": "纯合子",
    "Output summary table": "输出摘要表",
    "Output verbose summary": "输出详细摘要",
    "Output genome warnings": "输出基因组警告",
    "Input bigwig or bedgraph file": "输入大wig或bedgraph文件",
    "Pretext file": "前置文本文件",
    "Label for the graph": "图形的标签",
    "Annotated genbank file(s)": "注释过的GenBank文件（们）",
    "Gene type": "基因类型",
    "Single-end fastq": "单端fastq",
    "Forward fastq": "正向fastq",
    "Reverse fastq": "反向fastq",
    "Paired collection fastq": "配对的 fastq 集合",
    "Organelle type": "细胞器类型",
    "Maximum number of reads": "最大读取数",
    "Reduce reads for coverage": "减少用于覆盖度的读数",
    "Max rounds": "最大轮数",
    "Spades kmer settings": "Spades k-mer 设置",
    "Jump step": "跳跃步骤",
    "Mesh size": "网格大小",
    "Pre-grouped": "预分组的",
    "Max extending length": "最大延伸长度",
    "Fast": "快速",
    "Memory save": "内存保存",
    "Larger auto ws": "更大的自动窗口大小",
    "Long reads for polishing": "长读长用于润色",
    "Miniasm assembly to be polished": "需要进行润色的Miniasm组装",
    "Rounds": "轮次",
    "PacBio reads": "PacBio 读取",
    "Skip the initial polishing round": "跳过初始的打磨步骤",
    "Abs fragments contacts weighted": "Abs片段接触加权",
    "Fragments list": "片段列表",
    "info_contigs": "信息片段",
    "Reference genome source": "参考基因组来源",
    "Level (resolution) of the contact map": "接触图的水平（分辨率）",
    "Number of iterations to perform for each bin (row/column of the contact map)": "为接触图的每个bin（行/列）执行的迭代次数",
    "Number of standard deviations below the mean": "标准差低于均值的数量",
    "Number of neighbors to sample for potential mutations for each bin": "每个箱中潜在突变的采样邻居数",
    "Indicates genome is circular": "表示基因组是环状的",
    "Explode the genome prior to scaffolding": "在进行支架构建之前，将基因组分解。",
    "Only build multi-resolution contact maps (pyramids) and don't do any scaffolding": "仅构建多分辨率接触图（金字塔），不进行任何框架搭建。",
    "Only perform operations at the edge of the contigs": "仅在序列片段的边缘执行操作",
    "Number of Ns that you want to include during the polishing step": "在抛光步骤中您想要包含的N的数量",
    "Correction coverage": "校正覆盖度",
    "Assembly": "组装",
    "Assembly coverage": "组装覆盖度",
    "Polish contigs": "润色连续序列",
    "Reference CMAP file": "参考 CMAP 文件",
    "Query CMAP file": "查询 CMAP 文件",
    "XML configuration file": "XML 配置文件",
    "Bionano key file": "BioNano 关键文件",
    "Secondary map": "次级映射",
    "Two enzymes XML configuration file": "双酶XML配置文件",
    "Enable BLAT phase": "启用 BLAT 阶段",
    "Only confirmed positions": "仅确认的位置",
    "HiFi reads": "HiFi 读取",
    "Force read selection": "强制读取选择",
    "Lower 10% runtime kmer frequency threshold": "降低10%运行时间的kmer频率阈值",
    "Lower 5% runtime kmer frequency threshold": "5%运行时间kmer频率阈值下限",
    "Lower 3% runtime kmer frequency threshold": "降低3%运行时间的kmer频率阈值",
    "Enable read selection": "启用读取选择",
    "Overlap/Error correction": "重叠/误差校正",
    "rounds of correction": "校正轮次",
    "rounds of assembly cleaning": "组装清洁轮次",
    "Genome assembly file": "基因组组装文件",
    "Blobdir.tgz file": "Blobdir.tgz 文件",
    "BUSCO full table file": "BUSCO完整表格文件",
    "BLAST/Diamond hits": "BLAST/Diamond 命中",
    "BLAST/Diamond hits dataset": "BLAST/Diamond 命中数据集",
    "BLAST hits to taxa rule": "BLAST 命中到分类群的规则",
    "E-value cutoff": "E值截断",
    "Bitscore cutoff": "位分数截断",
    "Hits to a given taxon": "命中给定的分类群",
    "BLAST/Diamond file column order": "BLAST/Diamond 文件列顺序",
    "BED file of coverages per scaffold": "每个支架的覆盖度的BED文件",
    "BAM/SAM/CRAM read alignment file": "BAM/SAM/CRAM 读取比对文件",
    "FASTA sequence": "FASTA序列",
    "tRNAscan2-SE": "tRNAscan2-SE",
    "Prevent duplicated identifiers": "防止重复的标识符",
    "Blobtools v1 database": "Blobtools v1 数据库",
    "Identifier and sinonyms": "标识符和同义词",
    "Update plot": "更新图表",
    "Samtools Pileup": "Samtools 堆积",
    "Replace fields": "替换字段",
    "Blobdir file": "Blobdir文件",
    "Filter mode options": "过滤模式选项",
    "Parameter value": "参数值",
    "List of identifiers": "标识符列表",
    "JSON format list file": "JSON格式列表文件",
    "Assembly to be filtered": "待过滤的组装",
    "Filter FASTQ reads": "过滤 FASTQ 读取",
    "Reads to be filtered": "要过滤的读段",
    "Sumamry rank": "摘要排名",
    "Remove mode options": "移除模式选项",
    "Fields to remove": "要移除的字段",
    "Remove fields by ID": "根据ID移除字段",
    "Task": "任务",
    "Read cutoff": "读取截止值",
    "Provide estimated genome size or seed cutoff": "提供估计的基因组大小或种子截止值",
    "Seed depth": "种子深度",
    "Seed cutoff": "种子截止值",
    "Block size": "块大小",
    "OVL sort parameters": "OVL 排序参数",
    "Max depth of each overlap": "每个重叠的最大深度",
    "Max over hang length to filter": "最大悬挂长度筛选",
    "Minimap2 parameters": "Minimap2 参数",
    "Minimum match length": "最小匹配长度",
    "Re-align for every n reads": "对每n个读取进行重新比对",
    "Maximum over hang length for re-align": "最大重对齐悬挂长度",
    "Maximum over hang length for filtering contained reads": "过滤包含序列的读取时的最大悬挂长度",
    "Correction options": "校正选项",
    "NextGraph parameters": "NextGraph 参数",
    "Disable pre-filter chimeric reads.": "禁用预过滤嵌合读段。",
    "Retain potential chimeric edges.": "保留潜在的嵌合边缘。",
    "Delete complex bubble paths.": "删除复杂气泡路径。",
    "Output alternative contigs": "输出替代性连续序列",
    "Minimum short branch length": "最小短分支长度",
    "Minimum identity of alignmnents": "对齐的最小一致性",
    "Minimum test-to-best identity ratio": "最小测试到最佳一致性比率",
    "Maximum test-to-best identity ratio": "最大测试到最佳一致性比率",
    "Minimum test-to-best aligned length ratio": "最小测试到最佳对齐长度比率",
    "Maximum test-to-best score ratio of a low quality edge": "最差边缘的最大测试到最佳分数比率",
    "Minimum test-to-best aligned matches ratio": "最小测试到最佳比对匹配比率",
    "Minimum test-to-best depth ratio of an edge": "边的最小测试到最佳深度比率",
    "Minimum valid nodes of a read": "一个读取的有效最小节点数",
    "Minimum contained number to filter": "最小包含数过滤",
    "Minimum depth of an edge": "边的最小深度",
    "Depth of BFS to identify chimeric nodes": "广度优先搜索（BFS）的深度用于识别嵌合节点",
    "Maximum depth multiple of a node for BFS": "最大深度是节点的倍数，用于广度优先搜索（BFS）。",
    "Minimum depth multiple of a repeat node": "最小深度重复节点的倍数",
    "Maximum depth multiple of a node": "最大深度是节点的倍数",
    "Maximum length of a bubble": "最大泡沫长度",
    "Maximum length of a compound path": "复合路径的最大长度",
    "Maximum length of a z branch": "Z分支的最大长度",
    "Maximum length of a short branch": "短枝的最大长度",
    "Maximal length of a short loop": "短循环的最大长度",
    "Maximal over hang length of dovetails": "最大悬垂长度的榫头",
    "Fuzz length for trans-reduction": "转录本缩减中的模糊长度",
    "Input fasta file with at least two sequences": "输入至少包含两个序列的fasta文件",
    "Consensus method when differences are found between sequences": "当序列之间发现差异时的共识方法",
    "Take account of gaps?": "考虑间隙？",
    "Output consensus sequence AND original sequences in the FASTA file?": "输出一致序列以及FASTA文件中的原始序列？",
    "HiFi Reads Files": "HiFi 读取文件",
    "Oxford Nanopore Reads Files": "牛津纳米孔读取文件",
    "Algorithm parameters": "算法参数",
    "Do not perform Canu correction on HiFi reads": "不要对HiFi读取执行Canu校正",
    "Assign haplotypes from meryl databases": "从Meryl数据库分配单倍型",
    "h1": "h1",
    "h2": "h2",
    "Base k": "基数 k",
    "Max k": "最大 k",
    "Split bases": "拆分碱基",
    "Split reads": "拆分读取",
    "Min ont length": "最小本体长度",
    "Correct k mer size": "正确的 k 值大小",
    "Correct mer threshold": "正确的mer阈值",
    "Correct min read length": "正确的最小读取长度",
    " Correct min overlap length": "正确的最小重叠长度",
    "Correct hash bits": "正确的哈希位",
    "Seed min length": "种子最小长度",
    "Seed max length": "种子最大长度",
    "Align bandwidth": "对齐带宽",
    "Score fraction": "分数得分",
    "Min identity": "最小一致性",
    "End clipping": "末端剪切",
    "Incompatible cutoff": "不兼容的截止值",
    " Disable infer genes": "禁用推断基因",
    "Disable infer transcripts": "禁用推断转录本",
    "Aligned reads to reference genome": "将读取序列与参考基因组进行比对",
    "Run BUSCO": "运行 BUSCO",
    "Advaced options": "高级选项",
    "Strand-specific RNA-seq data": "链特异性 RNA-seq 数据",
    "Minimal alignment length to be used": "最小对齐长度使用",
    "Run with BLAT instead of GMAP": "使用BLAT而不是GMAP运行",
    "Meta Transcriptome": "宏转录组",
    "Lower threshold for x-assembled/covered/matched metrics.": "降低x-组装/覆盖/匹配指标的阈值。",
    "Upper threshold for x-assembled/covered/matched metrics.": "x-组装/覆盖/匹配指标的上限阈值。",
    "Prokararyotic organism(s)": "原核生物（单数或复数）",
    "Short report formats": "简短报告格式",
    "Ultra-long (>100kb) reads": "超长（>100kb）读取",
    "10X Genomics BAM file to convert to FASTQ": "10X Genomics BAM文件转换为FASTQ",
    "Read pairs to convert?": "要转换的配对读段？",
    "Only include read pairs mapping to a locus.": "仅包括映射到一个位点的读段对。",
    "Skip unpaired or duplicated reads instead of throwing an error?": "跳过未配对或重复的读取，而不是抛出错误？",
    "BAM file produced from": "由产生的BAM文件",
    "FASTQs or BAM": "FASTQ或BAM",
    "Cell barcodes file": "细胞条形码文件",
    "Reference genome FASTA": "参考基因组FASTA文件",
    "Paired reads or Paired collection": "配对的读取或配对集合",
    "Input Read 1": "输入读取1",
    "Input Read 2": "输入读取 2",
    "Exon annotation GFF3 file": "外显子注释GFF3文件",
    "Species gene id": "物种基因ID",
    "Barcode start Read 1": "条形码开始 读取1",
    "Barcode length Read 1": "条形码长度 读取 1",
    "Barcode start Read 2": "条形码开始 读取2",
    "Barcode length Read 2": "条形码长度 读取 2",
    "UMI start Read 2": "UMI起始读取2",
    "UMI length Read 2": "UMI长度 读取2",
    "Keep outliers?": "保留异常值？",
    "Output PDF with plots?": "输出带图形的PDF？",
    "Output QC metrics matrix?": "输出质量控制指标矩阵？",
    "Output HTML Report?": "输出 HTML 报告？",
    "FASTQ input only": "FASTQ 输入仅限",
    "Remove reads with N in barcode or UMI": "移除条形码或UMI中包含\"N\"的读段",
    "Remove reads with low quality": "移除低质量的读取",
    "Minimum read quality": "最小读取质量",
    "Maximum number of bases below minq": "最小质量分数以下的最大碱基数量",
    "Maximum mismatch allowed in barcode": "在条形码中允许的最大不匹配",
    "Maximum reads processed": "最大处理的读取数",
    "Minimum count to keep": "最小计数以保留",
    "Correct UMI sequence error": "校正UMI序列错误",
    "Perform strand-specific mapping": "执行特定于链的映射",
    "Remove low abundant genes": "去除低丰度基因",
    "Raw counts of expression data": "表达数据的原始计数",
    "Do you want to cluster cells ?": "您想要聚类细胞吗？",
    "Minimum size of each cluster": "每个聚类最小大小",
    "Input Type": "输入类型",
    "RNA-Seq FASTQ file, Barcode reads": "RNA-Seq FASTQ 文件，条形码读取",
    "RNA-Seq FASTQ file, HTO/CMO reads": "RNA-Seq FASTQ 文件，HTO/CMO 读取",
    "Collection of Pairs": "配对集合",
    "the CMO/HTO barcodes as well as their respective names": "CMO/HTO 条形码以及它们各自的名称",
    "Cell Barcode First Base": "细胞条形码首字母",
    "Cell Barcode Last Base": "细胞条形码 最后一个碱基",
    "UMI First Base": "UMI 首位碱基",
    "UMI Last Base": "UMI 最后碱基",
    "How many errors are allowed between two cell barcodes to collapse them onto one cell.": "两个细胞条形码之间允许有多少个错误才能将它们合并为一个细胞。",
    "How many errors are allowed between two umi within the same cell and TAG to collapse.": "在同一个细胞内，两个UMI（Unique Molecular Identifier，唯一分子标识符）之间允许有多少错误，以及用于折叠的标签（TAG）？",
    "Deactivate UMI correction": "停用UMI校正",
    "How many cells you expect in your run": "您期望在您的运行中有多少个细胞",
    "Whitelist of cell barcodes": "细胞条形码白名单",
    "Maximum Levenshtein distance allowed for CMO/HTO.": "允许的最大Levenshtein距离用于CMO/HTO。",
    "How many bases should be trimmed on CMO/HTO read before starting to map": "在开始映射之前，应该从CMO/HTO读取中修剪多少个碱基",
    "Activate sliding window alignement.": "激活滑动窗口对齐。",
    "Output in tsv format (in addition to the dense format)": "输出为tsv格式（除了密集格式之外）",
    "Select N reads to run on instead of all.": "选择N个读取运行，而不是全部。",
    "Write table of unknown TAGs to file.": "将未知的TAG写入文件。",
    "Top n unmapped TAGs to output.": "输出未映射的前n个标签。",
    "Input image": "输入图像",
    "Size of resulting (quadratic-shaped) slices": "生成的（呈二次方形状的）切片的大小",
    "Threshold to not be considered background": "不被考虑为背景的阈值",
    "Should the number of slices be limited?": "切片的数量是否应该受到限制？",
    "Maximum amount of slices; only needed if limit_slices is selected": "最大切片数；仅在选择了限制切片时需要",
    "Boolean that controls if a certain seed should be selected for the random number generator": "控制是否为随机数生成器选择某个特定种子的布尔值",
    "Seed chosen for the random number generator of the random choice of limited slices. Only required if cotrol_rng is selected": "用于随机选择有限切片的随机数生成器的种子。仅在选择了cotrol_rng时才需要",
    "Plot SNN on tSNE": "在tSNE上绘制SNN",
    "Reduction": "还原",
    "Dimensions": "维度",
    "Assay": "测定",
    "Distance matrix": "距离矩阵",
    "k": "k",
    "Compute SNN": "计算最近邻（SNN）",
    "Prune SNN": "修剪最近邻网络（SNN）",
    "NN method": "最近邻方法",
    "Annoy metric": "烦恼度量",
    "Graph name": "图名称",
    "NN Error bound": "神经网络误差界限",
    "Force recalc": "强制重新计算",
    "Mean function": "平均函数",
    "Dispersion function": "离散函数",
    "X-axis low cutoff": "X轴低截断",
    "X-axis high cutoff": "X轴高截止",
    "Y-axis low cutoff": "Y轴低截断",
    "Y-axis high cutoff": "Y轴高截止",
    "Selection method": "选择方法",
    "Normalisation method": "标准化方法",
    "Assay type": "检测类型",
    "Vars to regress": "回归变量",
    "Statistical model": "统计模型",
    "Use UMIs.": "使用UMIs（唯一分子标识符）。",
    "Skip centering": "跳过居中",
    "Skip scaling": "跳过缩放",
    "Scale maximum": "最大值缩放",
    "Minimum number of cells to block": "最小细胞数以阻止",
    "Check that data is normalized": "检查数据是否已标准化",
    "Faster execution": "更快的执行",
    "tSNE method": "tSNE 方法",
    "tSNE dimensions": "tSNE 维度",
    "Input expression data": "输入表达数据",
    "Choose the format of the input": "选择输入的格式",
    "Expression table": "表达量表格",
    "Cell Metadata": "单元格元数据",
    "Minimum cells to include features": "最小细胞数以包含特征",
    "Minimum features to include cells": "最小特征以包含细胞",
    "Genes to scale": "基因缩放",
    "Cells to scale": "细胞规模",
    "Principal components": "主成分",
    "Reverse PCA": "逆PCA",
    "LogFC logfc_threshold": "对数倍数变化 (LogFC) - logfc_threshold",
    "Max cells per ident": "最大细胞数每识别",
    "Min cells per gene": "每个基因的最小细胞数",
    "Min cells in one of the groups": "每个组中的最小细胞数",
    "Min Pct": "最小百分比",
    "Min diff Pct": "最小差异百分比",
    "Only positive markers": "仅正向标记",
    "Test to use": "测试使用",
    "Subsets used to filter cells": "用于筛选细胞的子集",
    "Cells to use": "使用单元格",
    "Marker genes file from Seurat": "Seurat中的标记基因文件",
    "Dimensionality reduction": "降维",
    "X axis dimension": "X轴维度",
    "Y axis dimension": "Y轴维度",
    "Cells list": "细胞列表",
    "Label size": "标签大小",
    "Label clusters": "标记聚类",
    "Repel labels": "排斥标签",
    "Group by": "分组",
    "Colors": "颜色",
    "Cell attribute for shape": "细胞形状属性",
    "Ident (clusters) order": "识别（聚类）顺序",
    "Figure width": "图形宽度",
    "Figure height": "图高",
    "Seurat object": "Seurat对象",
    "Modularity organization algorithm": "模块化组织算法",
    "Modularity function": "模块化函数",
    "Method for Leiden": "莱顿方法",
    "Graph Name": "图名称",
    "Random starts": "随机起始",
    "Group singletons": "分组孤立样本",
    "Parameter type": "参数类型",
    "Choose the format of the input values": "选择输入值的格式",
    "User input values": "用户输入值",
    "Starting value": "起始值",
    "Ending value": "结束值",
    "Layer to batch correct. By default corrects the contents of .X.": "层到批处理校正。默认情况下校正.X的内容。",
    "Key under which to add the computed results.": "将计算结果添加到的键。",
    "The subset of vars (list of str) to be used when performing MNN correction": "在执行MNN校正时使用的变量子集（字符串列表）",
    "Category or comma-separated list of categories": "分类或以逗号分隔的分类列表",
    "Number of mutual nearest neighbors.": "互近邻数",
    "Perform cosine normalization prior to computing corrected expression values?": "在计算校正后的表达值之前执行余弦归一化？",
    "Number of dimensions to use for summarizing biological substructure within each batch.": "用于总结每个批次中生物子结构的维度数。",
    "Compute the angle between each cell’s correction vector and the biological subspace of the reference batch?": "计算每个细胞的校正向量与参考批次的生物学子空间之间的角度。",
    "Name of the clustering": "聚类名称",
    "Additional suffix to the name of the slot to save the calculated trajectory": "为保存计算出的轨迹的槽位名称添加的额外后缀",
    "Use RNA velocity to orient edges in the abstracted graph and estimate transitions.": "使用RNA速度来确定抽象图中边的方向并估计转换。",
    "y axis scale transformation for observed transcriptome plot": "观察到的转录组图的y轴刻度转换",
    "y axis scale transformation for simulated transcriptome plot": "模拟转录组图的y轴刻度转换",
    "Use plot defaults": "使用绘图默认设置",
    "Figure size as 'width,height', e.g, '7,7'": "图形尺寸以“宽度，高度”格式，例如，“7,7”",
    "Figure dpi": "图象分辨率",
    "Figure font size": "图形字体大小",
    "Doublet score threshold.": "双细胞评分阈值。",
    "Remove predicted multiplets?": "移除预测的多重态？",
    "Where batches are present, the name of the column in adata.obs that differentiates among experiments/batches.": "在存在批次的情况下，adata.obs 中用于区分实验/批次的列的名称。",
    "The name of the column in adata.obs that differentiates among samples. This is a convenience for workflows, this is overriden if the batch key is set. It is an Scrubblet best practice to operate per samples rather than on the complete dataset (and it should use less memory).": "在adata.obs中用于区分样本的列名。这是工作流程的便利之处，如果设置了批次键，则会被覆盖。按照Scrubblet的最佳实践，应该按照样本操作而不是在完整数据集上操作（它应该使用更少的内存）。",
    "Number of doublets to simulate relative to the number of observed transcriptomes.": "相对于观察到的转录组数量，模拟的双体数量。",
    "Rate for sampling UMIs when creating synthetic doublets.": "在创建合成双细胞时采样UMIs的速率。",
    "Estimated doublet rate for the experiment.": "实验的估计双细胞率。",
    "Uncertainty in the expected doublet rate.": "预期双峰率的不确定性。",
    "A known metric’s name.": "一个已知的度量名称。",
    "Normalize the data such that each gene has a variance of 1?": "将数据标准化，使得每个基因的方差为1。",
    "Center the data such that each gene has a mean of 0": "将数据居中，使得每个基因的平均值为0",
    "Number of principal components.": "主成分个数。",
    "Use approximate nearest neighbor (annoy) method for the KNN classifier": "使用近似最近邻（ANNOY）方法进行KNN分类器",
    "Get doublet neighbor parents": "获取双细胞邻近的亲代细胞",
    "Number of neighbors.": "邻居数。",
    "Subset of groups/clusters to which comparisons shell be restricted.": "比较将被限制的组/簇的子集。",
    "The dimensionality reduction in .obsm to use for neighbour detection.": "用于邻域检测的.obsm中的降维。",
    "Use annoy’s approximate neighbour finding?": "使用 Annoy 的近似邻居查找？",
    "How many top neighbours to report for each batch": "每个批次要报告多少个最接近的邻居",
    "Trim the neighbours of each cell to these many top connectivities.": "将每个单元格的邻居修剪到这些最多的顶级连接数。",
    "The number of trees to construct in the annoy forest.": "在Annoy森林中构建树的数量。",
    "Use the faiss package to compute nearest neighbours if installed": "如果已安装，请使用 faiss 包计算最近邻。",
    "UMAP connectivity computation parameter": "UMAP 连接性计算参数",
    "UMAP connectivity computation parameter, how many nearest neighbors of each cell are assumed to be fully connected (and given a connectivity value of 1).": "UMAP 连接性计算参数，假设每个细胞有多少个最近邻被认为完全连接（并且给定一个连接值为1）。",
    "Comma separated list of additional covariates besides the batch variable such as adjustment variables or biological condition.": "除了批次变量之外，逗号分隔的附加协变量列表，如调整变量或生物学条件。",
    "Number of diffusion components to calculate": "计算扩散成分的数量",
    "Export embeddings as a tab-separated text table": "导出嵌入物作为制表符分隔的文本表格",
    "Additional suffix to the name of the slot to save the calculated diffusion map": "为保存计算出的扩散图的槽位名称添加的额外后缀",
    "The key in `.uns` that contains trajectory information": "`.uns` 中包含轨迹信息的键",
    "Layout functions": "布局函数",
    "Name of the embedding to plot": "要绘制的嵌入名称",
    "Do not draw edges with weights below this threshold": "不要绘制权重低于此阈值的边",
    "The index of the root node when choosing a tree layout": "选择树状布局时根节点的索引",
    "Restrict to the largest connected component": "限制到最大的连通组件",
    "Edges to be drawn in solid black": "以实心黑色绘制边",
    "Name of cell annotation or gene that is used to color the nodes": "用于对节点进行着色的细胞注释或基因名称",
    "Increase or decrease the size of the nodes": "增加或减少节点的大小",
    "Font size for the node labels": "节点标签的字体大小",
    "Increase or decrease the width of the edges": "增加或减少边缘的宽度",
    "Use .obsp[obsp] as adjacency": "使用 .obsp[obsp] 作为邻接矩阵",
    "Graph layout": "图形布局",
    "Root for tree layouts": "树状布局的根",
    "The fraction of total cells under which further splitting is skipped during recursive splitting of branches": "在分支递归拆分过程中，跳过进一步拆分的总细胞比例",
    "Name of attribute that defines clustering": "定义聚类的属性名称",
    "Name of the clustering that defines the root cell type": "定义根细胞类型的聚类名称",
    "Additional suffix to the name of the slot to save the calculated pseudo-time": "用于保存计算出的伪时间的槽位名称的额外后缀",
    "Allow Kendall Tau shift": "允许肯德尔τ偏移",
    "Expression matrix, genes as rows, cells as columns. Required input. Provide as TSV, CSV or RDS.": "表达矩阵，以基因为行，以细胞为列。必需的输入。请提供为TSV、CSV或RDS格式。",
    "Format of expression matrix": "表达矩阵的格式",
    "Per-cell annotation, optional. Row names must match the column names of the expression matrix. Provide as TSV, CSV or RDS.": "每个细胞的注释，可选。行名必须与表达矩阵的列名匹配。提供为TSV、CSV或RDS格式。",
    "Format of cell metadata": "单元格元数据的格式",
    "Per-gene annotation, optional. Row names must match the row names of the expression matrix. Provide as TSV, CSV or RDS.": "每个基因的注释，可选。行名必须与表达矩阵的行名匹配。请提供为TSV、CSV或RDS格式。",
    "Format of gene annotation": "基因注释的格式",
    "The column of reducedDimS(cds) to plot on the horizontal axis.": "在水平轴上绘制reducedDimS(cds)的列。",
    "The column of reducedDimS(cds) to plot on the vertical axis.": "在垂直轴上绘制 reducedDimS(cds) 列。",
    "The dimensionality reduction for plotting.": "降维用于绘图。",
    "The cell attribute (e.g. the column of pData(cds)) to map to each cell's color, or one of {cluster, partition, pseudotime}.": "单元格属性（例如 pData(cds) 的列）映射到每个单元格的颜色，或者是 {cluster, partition, pseudotime} 中的一个。",
    "A list of gene IDs/short names to plot.": "一个用于绘图的基因ID/简称列表。",
    "Determines how to transform expression values for plotting.": "确定如何转换表达值以进行绘图。",
    "The size of the point for each cell.": "每个单元格的点的大小。",
    "The alpha aesthetics for the original cell points, useful to highlight the learned principal graph.": "原始细胞点的alpha美学，有助于突出学习到的主图。",
    "If set, display the cell group names directly on the plot. Otherwise include a color legend on the side of the plot.": "如果设置，直接在图表上显示细胞组名称。否则，在图表的侧面包含一个颜色图例。",
    "When this option is set, skip displaying the trajectory graph inferred by learn_graph().": "当设置此选项时，跳过显示由 learn_graph() 推断的轨迹图。",
    "If set, and setting --color-cells-by to something other than cluster, label the cells of each cluster independently. Can result in duplicate labels being present in the manifold.": "如果设置，并设置 --color-cells-by 为除 cluster 之外的其它选项，则每个聚类的细胞独立标记。这可能导致在流形中出现重复的标签。",
    "If set, label the leaves of the principal graph.": "如果设置，为主图的叶节点添加标签。",
    "If set, label the roots of the principal graph.": "如果设置，为主图的根标记标签。",
    "If set, label the branch points of the principal graph.": "如果设置，标记主图的分支点。",
    "Input Object": "输入对象",
    "Reference cells source": "参考细胞来源",
    "Reference cells file": "参考细胞文件",
    "Number of reference cells": "参考细胞数量",
    "Filter fraction for expression": "表达式过滤分数",
    "Top-n-markers": "Top-n-标记物",
    "Number of genes to test per group": "每组要测试的基因数量",
    "Group cell by": "按组对单元格进行分组",
    "Use logistic regression to asses discriminatory power": "使用逻辑回归评估区分能力",
    "Save full markers": "保存全部标记",
    "Plot top markers": "绘制顶级标记",
    "What neighbor graph to use, principal graph recommended for trajectory analysis.": "使用哪种邻接图，建议在轨迹分析中使用主图。",
    "The reduction method to base the analysis on.": "分析基础的简化方法。",
    "Number of nearest neighbors used for building the kNN graph which is passed to knn2nb function during the Moran's I (Geary's C) test procedure.": "在 Moran's I（Geary's C）检验程序中，用于构建 kNN 图并传递给 knn2nb 函数的最近邻的数量。",
    "A character string specifying the method for detecting significant genes showing correlated expression along the principal graph embedded in the low dimensional space.": "一个字符字符串，指定了在低维空间中嵌入的主图中检测显示相关表达的显著基因的方法。",
    "A character string specifying the alternative hypothesis.": "一个指定备择假设的字符字符串。",
    "The dimensionality of the reduced space.": "降维后空间的维度。",
    "The algorithm to use for dimensionality reduction.": "用于降维的算法。",
    "The preprocessing method used on the data.": "数据所使用的预处理方法。",
    "Initial dimensionality reduction to perform.": "执行初始降维。",
    "Determines how to transform expression values prior to reducing dimensionality.": "确定在降维之前如何转换表达值。",
    "Manually subset the gene pool to these genes for dimensionality reduction.": "手动将基因池子集到这些基因以进行降维。",
    "A string model formula specifying effects to subtract from the data.": "一个字符串模型公式，用于指定要从数据中减去的效应。",
    "Amount to increase expression values before dimensionality reduction.": "在降维之前增加表达值的数量。",
    "When this option is NOT set, scale each gene before running trajectory reconstruction.": "当这个选项没有设置时，在运行轨迹重建之前对每个基因进行缩放。",
    "The dimensionality reduction to base the clustering on.": "降维以基于其进行聚类。",
    "Number of nearest neighbours used for Louvain clustering.": "用于Louvain聚类的最近邻数量。",
    "When this option is set, calculate the weight for each edge in the kNN graph.": "当设置此选项时，在kNN图中为每个边计算权重。",
    "The number of iterations for Louvain clustering.": "Louvain 聚类迭代次数。",
    "Resolution of clustering result, specifying the granularity of clusters. Not used by default and the standard igraph louvain clustering algorithm will be used.": "聚类结果的分辨率，指定聚类粒度。默认情况下不使用，将使用标准的igraph louvain聚类算法。",
    "The q-value threshold used to determine the partition of cells.": "用于确定细胞分区的q值阈值。",
    "The starting principal points. We learn a principal graph that passes through the middle of the data points and use it to represent the developmental process. Exclusive with the root cell option.": "起始主点。我们学习一个穿过数据点中间的主图，并用它来表示发展过程。与根细胞选项互斥。",
    "The starting cells. Each cell corresponds to a principal point and multiple cells can correspond to the same principal point. Exclusive with the root node option.": "起始细胞。每个细胞对应一个主成分点，多个细胞可以对应同一个主成分点。与根节点选项互斥。",
    "The cell phenotype (column in pdata) used to identify root principal nodes.": "用于识别根主节点的细胞表型（pdata中的列）。",
    "The value in the cell phenotype column used to extract root nodes.": "用于提取根节点的单元格中表型列的值。",
    "The dimensionality reduction that was used for clustering.": "用于聚类的降维。",
    "When this option is set, learn a single tree structure for all the partitions. If not set, use the partitions calculated when clustering and identify disjoint graphs in each.": "当设置此选项时，为所有分区学习单一树结构。如果未设置，则使用在聚类时计算的分区，并在每个分区中识别不相交的图。",
    "When this option is set, skip the additional run of loop closing after computing the principal graphs to identify potential loop structure in the data space.": "当设置此选项时，在计算主图以识别数据空间中的潜在循环结构后，跳过循环闭合的额外运行。",
    "The maximal ratio between the euclidean distance of two tip nodes in the spanning tree inferred from SimplePPT algorithm and that of the maximum distance between any connecting points on the spanning tree allowed to be connected during the loop closure procedure.": "在由SimplePPT算法推断出的生成树中，两个末端节点之间的欧几里得距离与在循环闭合过程中允许连接的生成树上任何连接点之间的最大距离之间的最大比率。",
    "The minimal ratio between the geodestic distance of two tip nodes in the spanning tree inferred from SimplePPT algorithm and that of the length of the diameter path on the spanning tree allowed to be connected during the loop closure procedure. (Both euclidean_distance_ratio and geodestic_distance_ratio need to be satisfied to introduce the edge for loop closure.)": "在从SimplePPT算法推断出的生成树中，两个端点节点的测地距离与在循环闭合过程中允许连接的直径路径长度的最小比率。（为了引入循环闭合的边，需要满足欧几里得距离比率和测地距离比率。）",
    "When this option is set, perform an additional run of graph pruning to remove smaller insignificant branches.": "当设置此选项时，执行额外的图形修剪运行以移除较小的不重要分支。",
    "The minimal length of the diameter path for a branch to be preserved during graph pruning procedure.": "在图剪枝过程中，为了保留分支，直径路径的最小长度。",
    "When this option is set, perform orthogonal projection for cells corresponding to the tip principal points.": "当设置此选项时，对相应的尖端主点细胞执行正交投影。",
    "Serialised sc3 SingleCellExperiment object with stored consensus kmeans clustering": "序列化的 sc3 单细胞实验对象，包含存储的共识 kmeans 聚类",
    "An integer or comma-separated list of integers": "一个整数或以逗号分隔的整数列表",
    "Biological analysis to perform": "要执行的生物分析",
    "Serialised sc3 SingleCellExperiment object with stored kmeans clustering": "序列化的 sc3 SingleCellExperiment 对象，存储了 kmeans 聚类",
    "Serialised sc3 SingleCellExperiment object": "序列化 sc3 单细胞实验对象",
    "Serialised sc3 SingleCellExperiment object with distances calculated": "序列化的sc3 SingleCellExperiment对象，已计算距离",
    "Serialised sc3 SingleCellExperiment object with transformed distances": "序列化 sc3 单细胞实验对象，包含转换后的距离",
    "Method to supply K": "供应K的方法",
    "Integer K as the number of clusters for kmeans": "整数 K 作为 kmeans 聚类的数量",
    "Text file containing estimated K": "包含估计的K值的文本文件",
    "Serialised SingleCellExperiment object normalised by scater": "序列化的SingleCellExperiment对象，由scater标准化",
    "Perform gene filtering?": "执行基因过滤？",
    "Minimum percent of dropouts": "最小百分比的丢失",
    "Maximum percent of dropouts": "最大丢失百分比",
    "Minimum number of eigenvectors used for kmeans clustering as a fraction of the total number of cells": "用于 k 均值聚类的最小特征向量数量，作为细胞总数的比例",
    "Maximum number of eigenvectors used for kmeans clustering as a fraction of the total number of cells": "用于kmeans聚类的最大特征向量数量，占总细胞数的比例",
    "Number of randomly selected training cells to be used for SVM prediction.": "用于SVM预测的随机选择训练细胞的数量。",
    "Text file with one integer per line defining indices of training cells that should be used for SVM training": "每行一个整数的文本文件，定义用于SVM训练的训练细胞的索引",
    "The number of cells below which SVM are not run": "SVM不运行的细胞数下限",
    "Seed of the random number generator": "随机数生成器的种子",
    "Number of random starts for kmeans": "kmeans的随机起始点数量",
    "Maximum number of iterations for kmeans": "k均值聚类的最大迭代次数",
    "Serialized SingleCellExperiment in RDS format": "序列化的 SingleCellExperiment 在 RDS 格式中",
    "Use pre-calculated size factors in the input object if available": "如果可用，在输入对象中使用预先计算的大小因子",
    "SingleCellExperiment object": "单细胞实验对象",
    "name of the variable that defines expression values in RDS object": "RDS对象中定义表达值的变量名称",
    "cell controls": "细胞控制",
    "lower detection limit for expression": "表达的下限检测",
    "Set spike-in as control features?": "将内参作为控制特征集？",
    "Input serialized SingleCellExperiment RDS object": "输入序列化的SingleCellExperiment RDS对象",
    "Name of the QC metric used to detected outlier": "用于检测异常值的QC指标名称",
    "Detect outlier from lower/higher/both ends of the distribution": "从分布的较低/较高/两端检测异常值",
    "Transform to log10 scale": "转换为对数尺度",
    "Threshold in number of MAD away from the median": "中位数偏差数的阈值",
    "Threshold in value difference from the median": "与中位数的值差异阈值",
    "Keys to regress": "回归的关键",
    "Assesment results": "评估结果",
    "Files containing names of rounds executed, produced by run-scaff": "由 run-scaff 产生的包含执行轮次名称的文件",
    "Choose the source of the slot for existing clustering": "选择现有聚类插槽的来源",
    "Specific round to asses": "特定轮次评估",
    "A file containing a single round to asses in the AnnData file.": "一个包含单个回合以在AnnData文件中评估的文件。",
    "Iterations for assesment": "评估迭代次数",
    "Use external cluster information": "使用外部集群信息",
    "Cluster table for assessment in tsv format": "聚类表格用于评估，以tsv格式呈现",
    "Attribute in input object that contains cluster information": "输入对象中包含聚类信息的属性",
    "Run clustering optimisation": "运行聚类优化",
    "Use input clustering to initialise optimisation": "使用输入聚类来初始化优化",
    "Resolution for initialising louvain clustering": "初始化Louvain聚类时的分辨率",
    "Stop condition": "停止条件",
    "Accuracy for convergence of the optimisation process": "优化过程收敛的准确度",
    "Underclustering boundary to use in the optimisation.": "在优化中使用的低聚类边界。",
    "Run only the optimisation and skip the initial assessment": "仅运行优化并跳过初始评估",
    "Produce parameter walk for asessment distribution": "生成参数步进以评估分布",
    "Number of iterations of sampling for the confusion matrix.": "混淆矩阵采样迭代次数。",
    "Input SCE object": "输入SCE对象",
    "Remove Matrix": "移除矩阵",
    "Cluster column": "聚类列",
    "Index SingleCellExperiment object": "索引 SingleCellExperiment 对象",
    "SingleCellExperiment object to project": "单细胞实验对象到项目",
    "Number of nearest neighbours": "最近邻数量",
    "Annotate cells of the projection dataset using labels of the reference?": "使用参考标签注释投影数据集的单元格？",
    "Scmap predictions file in text format": "Scmap预测文件，文本格式",
    "Should prediction scores be included?": "预测分数是否应该被包含？",
    "Scmap index object": "Scmap 索引对象",
    "Tool used (cell/cluster)": "使用的工具（单元/集群）",
    "Column name of similarity scores": "相似性得分的列名",
    "SCE object containing expression data": "包含表达数据的SCE对象",
    "Normalised counts slot": "标准化计数槽",
    "Input Seurat object": "输入Seurat对象",
    "Prediction Column": "预测列",
    "Correction Method": "校正方法",
    "Significance Threshold": "显著性阈值",
    "Reduction Parameter": "缩减参数",
    "Predictions Object": "预测对象",
    "Classifier object": "分类器对象",
    "Get scores": "获取分数",
    "Input Seurat Object": "输入 Seurat 对象",
    "Query Data Matrix": "查询数据矩阵",
    "Normalise Data": "标准化数据",
    "Scale Factor": "缩放因子",
    "Threshold Level": "阈值水平",
    "Max Iterations": "最大迭代次数",
    "Recompute Alignment": "重新计算对齐",
    "Reference Scaling": "参考尺度调整",
    "Model type to train": "训练模型类型",
    "Resample Method": "重采样方法",
    "Number of Iterations": "迭代次数",
    "Tune Length": "调长",
    "Performance Metric": "性能指标",
    "Pre-processing Method": "预处理方法",
    "Return Data": "返回数据",
    "Save Predictions": "保存预测结果",
    "Cells to Reclassify": "待重新分类的细胞",
    "Column name for assigned cell types": "指定细胞类型的列名",
    "Train/test split ratio": "训练/测试集分割比例",
    "Training matrix": "训练矩阵",
    "log-transform": "对数转换",
    "training labels file": "训练标签文件",
    "Number of pincipal components": "主成分数量",
    "CDS object": "编码序列对象",
    "Marker file path": "标记文件路径",
    "gene database": "基因数据库",
    "CDS gene id type": "编码序列基因ID类型",
    "Marker gene id type": "标记基因 ID 类型",
    "propogate markers": "传播标记",
    "Classifier gene ID type": "分类器基因ID类型",
    "Marker list object": "标记列表对象",
    "marker check file": "标记检查文件",
    "Name of summary column": "摘要列的名称",
    "Name of cell type column": "细胞类型列的名称",
    "Name of gene id column": "基因ID列的名称",
    "Input marker file": "输入标记文件",
    "p-value column name": "P值列名",
    "p-value threshold": "P值阈值",
    "groups column": "组列",
    "gene names": "基因名称",
    "number of outputs": "输出数量",
    "Min num of observations": "最小观察次数",
    "Max num of training samples": "最大训练样本数",
    "input CDS object": "输入CDS对象",
    "classifier object": "分类器对象",
    "node": "节点",
    "convert gene IDs": "转换基因ID",
    "cluster-extended classification": "扩展聚类分类",
    "label probability ratio": "标签概率比率",
    "Aggregated tables": "聚合表格",
    "Tool performance table": "工具性能表",
    "Include Semantic Similarity": "包括语义相似性",
    "Sort by aggregated score": "根据聚合分数排序",
    "Exclusions file": "排除文件",
    "cl terms mapping": "CL术语映射",
    "Ontology graph": "本体图谱",
    "True Labels": "真实标签",
    "Input AnnData file": "输入AnnData文件",
    "Obs Fields to Merge": "观测字段合并",
    "Groupby column": "按列分组",
    "Sample Key column": "样本关键列",
    "Layer": "层",
    "Decoupler pseudobulk Mode": "解耦器 伪集合模式",
    "Factor Fields": "因子字段",
    "Use Raw": "使用原始数据",
    "Minimum Cells": "最小细胞数",
    "Produce plots": "生成图表",
    "Produce AnnData with Pseudo-bulk": "生成具有伪批量数据的AnnData对象。",
    "Minimum Counts": "最小计数",
    "Minimum Total Counts": "最小总计数",
    "Enable Filtering by Expression": "启用按表达式过滤",
    "Plot Samples Figsize": "绘制样本图 大小",
    "Plot Filtering Figsize": "绘图过滤 图大小",
    "Input tables": "输入表格",
    "Reference metadata file": "参考元数据文件",
    "Exclusions": "排除",
    "Label - CL term mapping": "标签 - 细胞学（CL）术语映射",
    "Reference file barcode field": "参考文件 条形码字段",
    "Predicted file barcode field": "预测文件条形码字段",
    "Reference label column": "参考标签列",
    "Predicted file label column": "预测的文件标签列",
    "Table with tool stats": "工具统计表",
    "List of empirical distributions": "经验分布列表",
    "Input reference file": "输入参考文件",
    "Number of labels to sample": "标签采样数量",
    "Tables with predicted labels": "带有预测标签的表格",
    "Number of top labels": "顶级标签数",
    "scores": "得分",
    "Input SDRF tables": "输入SDRF表格",
    "Condensed SDRF": "压缩的SDRF",
    "Barcode column name": "条形码列名",
    "Cell label column name": "细胞标签列名",
    "Cell ontology column name": "细胞本体列名",
    "Wig/Bedgraph to convert": "Wig/Bedgraph 转换",
    "Chromosome length file": "染色体长度文件",
    "Converter settings to use": "使用转换器设置",
    "Items to bundle in r-tree": "打包到 r-tree 中的项目",
    "Data points bundled at lowest level": "数据点在最低级别捆绑",
    "Clip chromosome positions": "剪切染色体位置",
    "Do not use compression": "不要使用压缩",
    "Data files to include in the clustering": "要包含在聚类中的数据文件",
    "Include BGCs from MIBiG database?": "是否包含MIBiG数据库中的生物合成基因簇？",
    "Version from the MIBiG database": "来自MIBiG数据库的版本",
    "Adding extra string to BiG-SCAPE runs?": "在BiG-SCAPE运行中添加额外的字符串？",
    "Adding extra string to BiG-SCAPE runs name": "在BiG-SCAPE运行名称中添加额外的字符串",
    "Data file Pfam-A.hmm": "数据文件 Pfam-A.hmm",
    "Print more detailed information about each step": "打印有关每个步骤的更详细信息",
    "Extra log file as output": "额外的日志文件作为输出",
    "Include BGCs with lower cutoff distance": "包含具有较低截止距离的生物合成基因簇（BGCs）",
    "Specify when domains are considered to overlap": "指定何时认为域重叠",
    "Minimum size of a BGC (bp)": "生物合成基因簇（BGC）的最小大小（碱基对）",
    "mix all classes in the analysis": "在分析中混合所有类别",
    "No classified output based on the BGC product": "没有根据生物合成基因簇（BGC）产物进行分类的输出",
    "Excluded classes from classification in BiG-SCAPE": "在BiG-SCAPE中排除的分类类别",
    "Cutoffs": "截断值",
    "GCF cutoff value": "GCF 截断值",
    "Turn off cluster GCFs into GCCs": "关闭集群 GCFs 转换为 GCCs",
    "Change cutoff values for cluster GCF into GCC?": "更改聚类GCF（基因组完成度框架）到GCC（基因组完成度复合体）的截止值。",
    "GCC cutoff value": "GCC截断值",
    "Exclude hybrid predicted products": "排除杂交预测产物",
    "Alignment Mode": "对齐模式",
    "Change Anchorfile?": "更改锚点文件？",
    "Using a different Anchorfile instead of default file": "使用不同的锚点文件而不是默认文件",
    "Use hmmscan for the domain prediction": "使用hmmscan进行领域预测",
    "Use a domain list to include domains?": "使用一个域列表来包含域？",
    "Use a .txt file to include domains": "使用.txt文件来包含域",
    "Ordination method": "排序方法",
    "Minimum taxa abundance": "最小类群丰度",
    "Color the samples by": "将样本按颜色分类",
    "The taxonomic level to aggregate the OTUs": "OTUs的聚合分类水平",
    "Additional taxonomic level(s) to display": "显示额外的分类层级",
    "Weight the frequency by abundance": "按丰度加权频率",
    "Kraken output": "Kraken输出",
    "Display taxa even if they lack a read in any sample": "即使样本中没有读取到，也显示分类群",
    "Display a header line indicating sample IDs": "显示一个包含样本ID的标题行",
    "Select a Kraken database": "选择一个Kraken数据库",
    "How to name OTUs": "如何命名操作分类单元（OTUs）",
    "Sanitize Names": "净化名称",
    "Output Rank Name in (second to) last column": "在（倒数第二）列中输出排名名称",
    "Output taxonomy in last column": "在最后一列输出分类学信息",
    "Display intermediate ranks": "显示中间排名",
    "Cluster by taxonomic rank": "按分类等级聚类",
    "Summation of lower ranks into higher ranks": "将较低等级的求和到较高等级",
    "Output a newick tree": "输出一个新克树",
    "Select taxonomy file tabular formated": "选择分类学文件表格格式",
    "Type of input file (Centrifuge, CLARK, Generic, Kraken, LMAT)": "输入文件类型（Centrifuge，CLARK，通用，Kraken，LMAT）",
    "Format of the output files from a generic classifier": "通用分类器输出文件的格式",
    "Cached database whith taxa ID": "缓存的具有分类群ID的数据库",
    "Type of extra output to be generated (default on CSV)": "生成的额外输出类型（默认为CSV格式）",
    "Suppress saving the HTML output file": "禁止保存HTML输出文件",
    "Remove the log file": "删除日志文件",
    "Coarse tuning of algorithm parameters": "算法参数的粗调",
    "Number of first samples will be treated as negative controls": "首样本数量将被视为阴性对照",
    "Type of scoring to be applied": "要应用的评分类型",
    "Minimum score/confidence of the classification of a read to pass the quality filter": "分类的最小得分/置信度，以通过质量过滤器",
    "Minimum taxa to avoid collapsing one level into the parent": "避免将一个级别合并到父级所需的最小分类群数",
    "NCBI taxid code to exclude a taxon and all underneath": "NCBI 物种分类ID代码，用于排除一个分类群及其下的所有分类群",
    "NCBI taxid code to include a taxon and all underneath": "NCBI 物种识别码，用于包含一个分类群及其下所有分类群",
    "Avoid cross analysis": "避免交叉分析",
    " Fine tuning of algorithm parameters": "算法参数的微调",
    "Minimum score in control samples to pass the filter": "在对照样本中通过筛选的最低分数",
    "Add a summary of the analysis (default is Add)": "添加一个分析摘要（默认是添加）",
    "Remove counts directly assigned to the root level": "移除直接分配给根级别的计数",
    "Show the cellular organisms taxon": "显示细胞生物分类群",
    "Strain level instead of species as the resolution limit for the robust contamination removal algorithm; use with caution, this is an experimental feature": "菌株水平而非物种作为鲁棒污染去除算法的分辨率限制；请谨慎使用，这是一个实验性功能",
    "Deactivate parallel processing": "停用并行处理",
    "Contig sequences": "连续序列",
    "Bins": "区间",
    "Contigs-to-bin table": "Contigs-to-bin 表",
    "Name of binning prediction tool used to generate the table": "用于生成表格的分箱预测工具的名称",
    "Engine used for single copy gene identification": "用于单拷贝基因鉴定的引擎",
    "Predicted proteins": "预测蛋白质",
    "Score threshold until selection algorithm will keep selecting bins": "选择算法将持续选择箱子，直到达到分数阈值",
    "Penalty for duplicate single copy genes per bin (weight b)": "每个箱中重复的单拷贝基因的惩罚（权重 b）",
    "Penalty for megabins (weight c)": "对超大区间的惩罚（权重 c）",
    "Maximum number of iterations after reaching score threshold": "达到分数阈值后的最大迭代次数",
    "Output predicted proteins?": "输出预测蛋白质？",
    "Write evaluation of input bin sets?": "编写输入bin集的评估？",
    "Export bins as fasta files?": "导出箱作为fasta文件？",
    "Export unbinned contigs as fasta file?": "导出未分箱的连续片段为fasta文件？",
    "Write debug information to log file?": "将调试信息写入日志文件？",
    "Variable(s) which contain the desired grouping of samples": "包含所需样本分组的变量",
    "Threshold in percent": "百分比阈值",
    "Margin plots to show": "边距图展示",
    "The size of the values": "值的大小",
    "Relative width of the main and y margin plots": "主图和y边距图的相对宽度",
    "Relative widths of the main and x margin plots": "主图和X边距图的相对宽度",
    "Cached database with clade-specific marker genes": "缓存的具有特定类群标记基因的数据库",
    "Single- or Paired-end data?": "单端或双端数据？",
    "List of paired reads": "配对序列列表",
    "Mapping file (optional)": "映射文件（可选）",
    "Forward primer used to amplify DNA region (optional)": "用于扩增DNA区域的正向引物（可选）",
    "Reverse primer used to amplify DNA region (optional)": "用于扩增DNA区域的反向引物（可选）",
    "Clustering threshold for OTUs when using swarm clustering (optional)": "在使用swarm聚类确定操作分类单元（OTUs）的聚类阈值（可选）",
    "Clustering threshold for OTUs": "OTUs的聚类阈值",
    "Taxonomy aligner for taxonomic profiling of OTUs": "OTUs的分类学分析比对工具",
    "Confidence threshold for RDP (optional)": "RDP的置信度阈值（可选）",
    "Other Clustering Options": "其他聚类选项",
    "Minimum size of dereplicated raw reads (optional)": "去重后的原始读段的最小大小（可选）",
    "Chimera check": "嵌合体检查",
    "Skew in chimeric fragment abundance": "嵌合片段丰度的偏斜",
    "Other Taxonomy Options": "其他分类选项",
    "Amplicon type": "扩增子类型",
    "Tax group": "分类群",
    "Keep unclassified OTUs": "保留未分类的OTUs",
    "Minimum horizontal coverage of an OTU sequence against ref DB (optional)": "OTU序列与参考数据库比对的最小水平覆盖度（可选）",
    "Minimum fraction of reads with identical taxonomy (optional)": "最小读取具有相同分类学的比例（可选）",
    "Use LULU to merge OTUs based on their occurrence": "使用LULU根据它们的出现合并OTUs",
    "Build OTU phylogeny": "构建OTU（操作分类单元）的系统发育树",
    "Taxonomic groups to exclude, as a regular expression": "排除的分类群组，使用正则表达式",
    "SDM option file (optional)": "SDM选项文件（可选）",
    "Barcode (MID) sequences (optional)": "条形码（MID）序列（可选）",
    "Remove likely contaminant OTUs/ASVs based on alignment to host genome": "去除可能的污染OTUs/ASVs，基于与宿主基因组的比对",
    "FASTA reference genome": "FASTA参考基因组",
    "Use minimap2 instead of VSEARCH to map back reads to OTUs": "使用minimap2而不是VSEARCH将读取映射回OTUs",
    "Assembly type used to generate contig(s)": "用于生成连续序列的组装类型",
    "Reads file": "读取文件",
    "Abundance file": "丰度文件",
    "Reads file(s)": "读取文件",
    "Abundance file(s)": "丰度文件（们）",
    "Maximum Expectation-Maximization algorithm iteration number": "最大期望最大化算法迭代次数",
    "Probability threshold for EM final classification": "EM最终分类的概率阈值",
    "Generate visualization of the marker gene presence numbers": "生成标记基因存在数量的可视化图",
    "Output marker gene presence for bins table": "输出标记基因存在性表格",
    "Output marker genes for each bin as fasta": "输出每个bin的标记基因为fasta格式",
    "Output log": "输出日志",
    "Marker gene set": "标记基因集",
    "Select a draft genome for polishing": "选择一个草图基因组进行打磨",
    "Select aligned data to polish": "选择对齐数据进行润色",
    "Select a SAM/BAM file": "选择一个SAM/BAM文件",
    "Select forward SAM/BAM file": "选择正向 SAM/BAM 文件",
    "Select reverse SAM/BAM file": "选择反转SAM/BAM文件",
    "Single-end collection": "单端测序",
    "Minimal depth": "最小深度",
    "Minimal invalid fraction": "最小无效分数",
    "Number of mismatch/indels to ignore alignments": "忽略对齐中的不匹配/插入缺失的数量",
    "Minimal valid fraction": "最小有效分数",
    "Keep log file": "保持日志文件",
    "Keep per base information file": "保持每个碱基信息文件",
    "Alpha-diversity measure(s) to be included": "要包括的α多样性度量",
    "Calculate sample richness estimates": "计算样本丰富度估计",
    "What is the type of your input data": "您的输入数据类型是什么",
    "show ranks from root to": "显示从根节点到",
    "Provide a name for the basal rank": "为基本等级提供一个名称",
    "Combine data from multiple datasets?": "合并多个数据集的数据？",
    "Confidence threshold": "置信度阈值",
    "Restrict labels to standard rank assignments": "将标签限制为标准等级分配",
    "FASTA file(s) of assemblies": "装配的FASTA文件",
    "Turn on resistance genes screening?": "开启抗性基因筛选？",
    "Turn on screening of K loci?": "开启K位点筛选？",
    "Turn on screening of O loci?": "开启O位点筛选？",
    "Minimum alignment percent identity for main results": "主要结果的最小比对百分比一致性",
    "Minimum alignment percent coverage for main results": "主要结果的最小比对百分比覆盖度",
    "Minimum alignment percent identity for spurious results": "杂项结果的最小比对百分比一致性",
    "Minimum alignment percent coverage for spurious results": "杂项结果的最小比对百分比覆盖",
    "Print scientific names instead of just taxids": "打印学名而不是仅仅打印taxids",
    "Confidence": "置信度",
    "Minimum Base Quality": "最小碱基质量",
    "Minimum hit goups": "最小命中组",
    "Enable quick operation": "启用快速操作",
    "Split classified and unclassified outputs?": "分离分类和未分类的输出？",
    "Create Report": "创建报告",
    "Print a report with aggregrate counts/clade to file": "打印一个带有聚合计数/类群的报告到文件",
    "Format report output like Kraken 1's kraken-mpa-report": "将报告输出格式化为Kraken 1的kraken-mpa-report样式  Kraken是一款用于宏基因组序列分类的软件，而kraken-mpa-report是Kraken的一个报告工具，用于生成分类结果的报告。这个命令要求将报告的输出格式调整为Kraken 1版本的样式。",
    "Report counts for ALL taxa, even if counts are zero": "报告所有分类群的计数，即使计数为零",
    "Report minimizer data": "报告最小化数据",
    "samples": "样本",
    "Order ASVs by": "按顺序排列 ASVs",
    "Length filter method": "长度过滤方法",
    "Minimum sequence length": "最小序列长度",
    "Maximum sequence length": "最大序列长度",
    "plot sequence length distribution": "绘制序列长度分布图",
    "Processing mode": "处理模式",
    "Aggregate data": "聚合数据",
    "sample number": "样本编号",
    "Dada results for forward reads": "Dada 前向序列的结果",
    "Dada results for reverse reads": "Dada 结果用于反向读取",
    "Minimum length of the overlap": "最小重叠长度",
    "Concatenated rather than merge": "合并而不是串联",
    "Trim overhangs": "修剪突出部分",
    "sequences to be assigned": "待分配序列",
    "Select a reference dataset your history or use a built-in?": "选择一个参考数据集，可以使用您历史中的，或者使用内置的。",
    "Select reference data set": "选择参考数据集",
    "Reference data set": "参考数据集",
    "Names of the taxonomic levels in the data set": "数据集中的分类水平名称",
    "Minimum bootstrap confidence": "最小引导置信度",
    "Try reverse complement": "尝试反向互补",
    "Output bootstrap values": "输出引导值",
    "Add genus-species binomials to the taxonomic table": "在分类表中添加属-种二名法",
    "reporting options": "报告选项",
    "Number of matched species": "匹配物种数量",
    "Seed": "种子",
    "data sets": "数据集",
    "Dataset(s)": "数据集",
    "kmer size": "kmer大小",
    "width (nucleotides) of the moving window": "移动窗口的宽度（核苷酸数）",
    "step size (nucleotides)": "步长（核苷酸）",
    "number of bins to use for the histogram": "直方图使用的箱数",
    "Short read data": "短读数据",
    "Magnitude of number of bases to use for learning": "用于学习的碱基数量的大小",
    "Advanced Option": "高级选项",
    "Randomize samples": "随机化样本",
    "Maximum number of times to step through the selfconsistency loop": "最大自洽循环迭代次数",
    "Threshold at which unique sequences inferred to contain errors are corrected": "用于推断含有错误的独特序列被纠正的阈值",
    "Plotting Option": "绘图选项",
    "Plot observed error rates": "绘制观察到的误差率",
    "Plot output error rates": "绘制输出错误率",
    "Plot input error rates": "绘制输入错误率",
    "Plot expected error rates": "绘制预期错误率",
    "Process samples in batches": "批量处理样本",
    "Pool samples": "合并样本",
    "Error rates": "错误率",
    "sequence table": "序列表",
    "Separate filters and trimmers for reverse reads": "为反向读取分别设置过滤器和修剪器",
    "Discard reads matching the PhiX genome": "丢弃匹配PhiX基因组的读取",
    "Low complexity filter kmer threshold": "低复杂性过滤器 k-mer 阈值",
    "String present at the start of valid reads": "位于有效读取序列起始处的字符串",
    "Output statistics": "输出统计信息",
    "Contigs to search again": "Contigs 再次搜索",
    "Proteins to search against contigs": "用于搜索基因组序列的蛋白质",
    "Maximum e-value of individual match segment": "单个匹配片段的最大e值",
    "Minimum number of codons in predicted open reading frame": "预测开放阅读框架中的最小密码子数",
    "Maximum e-value of combined exon set": "组合外显子集的最大e值",
    "Minimum length ratio of combined exon set to target": "合并外显子集与目标长度比的最小比例",
    "Write the contig coords of the stop-to-stop fragment in which putative exon lies": "写出假定外显子所在的终止子到终止子片段的连续坐标",
    "Enable scanning for point mutations using the PointFinder database": "启用使用PointFinder数据库扫描点突变",
    "PointFinder organism (unvalidated)": "PointFinder 有机体（未经验证）",
    "Percent identity threshold for BLAST": "BLAST的相似性百分比阈值",
    "Percent length overlap of BLAST hit for ResFinder database": "BLAST命中ResFinder数据库的序列长度重叠百分比",
    "Percent length overlap of BLAST hit for PointFinder database": "BLAST命中PointFinder数据库的百分比长度重叠",
    "Percent length overlap of BLAST hit for PlasmidFinder database": "BLAST命中PlasmidFinder数据库的序列长度重叠百分比",
    "The lower bound for our genome size for the quality metrics": "我们基因组大小的质量指标的下限",
    "The upper bound for our genome size for the quality metrics": "我们基因组大小的质量指标的上限",
    "The minimum N50 value for the quality metrics": "质量指标的最小N50值",
    "The minimum contig length for the quality metrics": "最小连通体长度的质量指标",
    "The number of contigs, under the minimum contig length which is unacceptable for the quality metrics": "在最小拼接长度下，拼接片段的数量是不可接受的，这影响了质量指标。",
    "Report all BLAST results": "报告所有BLAST结果",
    "Exclude negative (non-resistant) results": "排除阴性（非耐药性）结果",
    "Exclude resistance phenotypes": "排除抗性表型",
    "MLST Scheme": "MLST方案",
    "Exclude certain AMR genes from the results": "排除结果中的某些抗性基因",
    "Exclude genes file": "排除基因文件",
    "Provide a custom list of complex mutations": "提供一个自定义的复杂突变列表",
    "Complex mutations file": "复杂突变文件",
    "Specify PlasmidFinder Database type": "指定质粒查找数据库类型",
    "Lastal arguments": "Lastal 参数",
    "The sequences are :": "序列是：",
    "Match/mismatch score matrix.": "匹配/不匹配得分矩阵。",
    "Score matrix file": "得分矩阵文件",
    "Specify lowercase-marking of repeats. (-R)": "指定重复区域的小写标记。(-R)",
    "Input sequences to uppercase.": "将输入序列转换为大写。",
    "Check for simple repeats.": "检查简单重复序列。",
    "E-value options": "E值选项",
    "Query letters per random alignment (-D)": "每随机比对的查询字母数（-D）",
    "Initial-match options": "初始匹配选项",
    "Maximum initial matches per query position": "每个查询位置的最大初始匹配数",
    "Minimum length for initial matches": "初始匹配的最小长度",
    "Maximum length for initial matches": "初始匹配的最大长度",
    "Use initial matches starting at every k-th position in each query": "在每个查询中，从每个k-th位置开始使用初始匹配",
    "Use minimum positions in sliding windows of W consecutive positions": "在W个连续位置的滑动窗口中使用最小位置",
    "Score matrix applies to forward strand of :": "得分矩阵适用于：正链",
    "Omit alignments whose query range lies in >= K others with > score": "忽略那些查询范围位于 >= K 其他区域的比对，这些比对的得分 > 分数",
    "Omit gapless alignments in >= C others with > score-per-length": "忽略无间隙对齐，当在C中大于等于时，在其他情况下大于每长度得分。",
    "Type of alignment": "对齐类型",
    "Maximum gapless alignments per query position": "每个查询位置的最大无间隔对齐",
    "Mask lowercase during extensions.": "在扩展期间掩蔽小写字母。",
    "Suppress repeats inside exact matches, offset by less or equal this distance. Use 0 to set this off": "在完全匹配内抑制重复，偏移量小于或等于这个距离。使用0来关闭这个功能。",
    "Genetic code.": "遗传密码。",
    "Gamma parameter for gamma-centroid and LAMA": "Gamma 参数用于 gamma-centroid 和 LAMA",
    "Reference(s) input files": "参考(或参考资料)输入文件",
    "Lastdb arguments": "Lastdb 参数",
    "User-defined alphabet.": "用户定义的字母表。",
    "Convert the input sequences to uppercase while reading them.": "在读取输入序列时，将它们转换为大写。",
    "Soft-mask lowercase letters.": "软掩蔽小写字母。",
    "Specify a seeding scheme.": "指定一个种子生成方案。",
    "Minimum limit on initial matches per query position": "每个查询位置的初始匹配的最小限制",
    "Child table type": "子表类型",
    "LASTal data file": "LASTal 数据文件",
    "Last-split options": "最后拆分选项",
    "RNA direction": "RNA方向",
    "Cis-splice probability per base": "顺式剪接概率每碱基",
    "Trans-splice probability per base": "转录拼接概率每碱基",
    "Mean of ln[intron length]": "内含子长度的自然对数的平均值",
    "Standard deviation of ln[intron length]": "自然对数[内含子长度]的标准差",
    "Maximum mismap probability": "最大错配概率",
    "Write original, not split, alignement.": "编写原始的、未分割的、对齐的序列。",
    "Queries input files": "查询输入文件",
    "Query input format": "查询输入格式",
    "Mismatch cost": "不匹配成本",
    "Last-train options": "末班车选项",
    "Query letters per random alignment": "每随机比对的查询字母数",
    "Maximum expected alignments per square giga": "最大期望对齐数每平方吉咖",
    "Query strand to use": "要使用的查询链",
    "Score matrix applies to forward strand of:": "得分矩阵适用于正链的：",
    "Type of alignment:": "对齐类型：",
    "Use initial matches starting at every STEP-th position in each query": "在每个查询中，从每个STEP-th位置开始使用初始匹配",
    "N/X is ambiguous in:": "N/X 在：中是模糊的",
    "MAF file to convert": "将 MAF 文件转换",
    "MAF-convert arguments": "MAF-convert 参数",
    "No header.": "无标题。",
    "Line length": "行长",
    "Protein alignments.": "蛋白质比对。",
    "Join neighboring alignments": "连接相邻的比对",
    "Include dictionary of sequence lengths in sam format": "在sam格式中包含序列长度的字典",
    "Get a sequence dictionary from DICTFILE": "从 DICTFILE 获取序列字典",
    "Specify read group information": "指定读取组信息",
    "Files from ASaiM for all samples (named after sample)": "来自ASaiM的所有样本文件（以样本命名）",
    "GO namespace": "GO命名空间",
    "File from HUMAnN2 after regrouping, renaming, joining, and renormalizing": "HUMAnN2 处理后的文件，经过重新分组、重命名、合并和重新标准化",
    "Select your FASTA file": "选择你的FASTA文件",
    "Output all pairwise alignments": "输出所有配对序列比对结果",
    "Reject if fraction of query seq. aligned lower than this value": "如果查询序列比对的比例低于这个值，则拒绝",
    "Detect chimeras": "检测嵌合体",
    "Select your reference database": "选择你的参考数据库",
    "Minimal abundance ratio of parent vs chimera": "父代与嵌合体的最小丰度比率",
    "'no' vote pseudo count": "'否'投票伪计数",
    "'no' vote weight": "'否' 投票权重",
    "Minimum number of differences in segment": "最小段差异数",
    "Minimum divergence from closest parent": "最小与最近亲本的分歧",
    "Select output formats": "选择输出格式",
    "Select your input FASTA file": "选择您的输入FASTA文件",
    "Choose sorting method to use before clustering": "选择聚类前使用的排序方法",
    "Indicate that input sequences are not presorted by length": "指示输入序列未按长度预先排序",
    "Do not ignore terminal gaps in MSA for consensus": "不要在MSA中忽略末端间隙以形成共识序列",
    "Minimum abundance": "最小丰度",
    "Maximum abundance": "最大丰度",
    "Select your query FASTA file": "选择您的查询FASTA文件",
    "Select your database FASTA file": "选择您的数据库FASTA文件",
    "Mask database sequences": "掩蔽数据库序列",
    "Output only hits with identity equal to the best": "只输出与最佳相似度相等的命中结果",
    "Reject if terminal gaps at alignment right end": "如果对齐的右端有末端间隙，则拒绝",
    "Reject if terminal gaps at alignment left end": "如果对齐的左端有末端间隙，则拒绝",
    "Reject if fraction of target seq. aligned lower than this value": "如果目标序列比对的比例低于这个值，则拒绝",
    "Reject if identity higher than this value": "如果同一性高于这个值，则拒绝",
    "Reject if query/target length ratio higher than this value": "如果查询/目标长度比高于这个值，则拒绝",
    "Reject if query/target abundance ratio higher than this value": "如果查询/目标丰度比高于这个值，则拒绝",
    "Reject if shorter/longer length ratio higher than this value": "如果短/长长度比高于这个值，则拒绝",
    "Reject if percent identity lower than this value, ignoring gaps": "如果同一性百分比低于这个值，则拒绝，忽略间隙",
    "Reject if query/target length ratio lower than this value": "如果查询/目标长度比低于此值，则拒绝",
    "Reject if shorter/longer length ratio lower than this value": "如果短/长长度比低于此值，则拒绝",
    "Reject if query/target abundance ratio lower than this value": "如果查询/目标丰度比低于此值，则拒绝",
    "Reject if query abundance larger than this value": "如果查询丰度大于这个值，则拒绝",
    "Reject if alignment length shorter than this value": "如果比对长度短于此值，则拒绝",
    "Reject if more substitutions than this value": "如果替换的数量超过这个值，则拒绝",
    "Reject if more substitutions or indels than this value": "如果替换或插入/删除的数量超过这个值，则拒绝",
    "Reject if more indels than this value": "如果插入或缺失的数量超过这个值，则拒绝",
    "Maximum number of hits to show": "最大显示命中数",
    "Reject if first n nucleotides do not match": "如果前n个核苷酸不匹配，则拒绝",
    "Reject if last n nucleotides do not match": "如果最后的n个核苷酸不匹配，则拒绝",
    "Reject if target abundance lower": "如果目标丰度较低，则拒绝",
    "Score for mismatch": "不匹配得分",
    "UCLUST-like output": "UCLUST 类似输出",
    "Show all, not just top hits": "显示全部，不仅仅是顶部命中",
    "Length of words for database index 3-15": "数据库索引的单词长度为3-15",
    "Write non-matching queries to the output file": "将不匹配的查询写入输出文件",
    "Sorting by": "排序依据",
    "Relabel with this prefix string after sorting": "在排序后用这个前缀字符串重新标记",
    "Add abundance annotation to output": "在输出中添加丰度注释",
    "Select queries from your history or use one from a tool data table?": "从您的历史记录中选择查询，或使用来自工具数据表的一个？",
    "Queries (Mash Sketch)": "查询（Mash草图）",
    "'Winner takes all' to remove redundancy in the result": "“胜者通吃”以消除结果中的冗余",
    "Minimum identity to report": "最低相似性报告",
    "Maximum p-value to report": "最大报告的p值",
    "prokaryote DNA sequences": "原核生物DNA序列",
    "MetaGenomic - Sequences are from multiple organisms": "宏基因组学 - 序列来自多个生物体",
    "output formats": "输出格式",
    "Input: Reads or Assemblies": "输出：读取或组装",
    "Minimum copies of each k-mer required to pass noise filter": "每个k-mer通过噪声过滤器所需的最小拷贝数",
    "Target coverage": "目标覆盖度",
    "Sketch individual Sequences": "草绘单个序列",
    "Sketch size": "草图大小",
    "Probability threshold for warning about low k-mer size": "警告关于低 k-mer 大小的概率阈值",
    "metagenomic fastq files": "宏基因组 fastq 文件",
    "filter": "过滤",
    "minlength": "最小长度",
    "minqualscore": "最小质量分数",
    "minorflength": "小片段长度",
    "minlongesttryppeplen": "最小最长尝试肽长度",
    "minreadcount": "最小读取计数",
    "maxreads": "最大读取数",
    "MetaGene Annotator": "MetaGene注释器",
    "metagenefile": "元基因文件",
    "minmetagenescore": "最小元基因得分",
    "select outputs": "选择输出",
    "metapeptide database": "代谢肽数据库",
    "minaaseqlength": "最小序列长度",
    "maxmetapeptides": "最大代谢肽段数",
    "fasta type": "FASTA格式",
    "missedcleavages": "未切割位点",
    "minpeptidelength": "最小肽段长度",
    "metapeptide databases": "代谢肽数据库",
    "metagenome assembled genomes (MAGs/bins)": "宏基因组组装基因组（MAGs/箱）",
    "Contigs fasta": "Contigs 是指在基因组测序过程中，通过序列比对和组装得到的较长的 DNA 片段。fasta 是一种文本格式，用于表示 DNA 或蛋白质序列。Contigs fasta 通常指的是以 fasta 格式存储的 Contigs 序列。",
    "Variable coverage": "变量覆盖度",
    "Max group size": "最大组大小",
    "Min partition size": "最小分区大小",
    "Output unassigned sequences": "输出未分配序列",
    "Require all sequences be properly paired?": "要求所有序列正确配对？",
    "Ignore all pairing information?": "忽略所有配对信息？",
    "Extra unpaired reads": "额外未配对的读取",
    "Optional k-mer countgraph": "可选的 k-mer 计数图",
    "Save the k-mer countgraph(s) in a file": "将k-mer计数图保存在文件中",
    "Save the k-mer countgraph to a file": "将 k-mer 计数图保存到文件中",
    "Verify read names match": "验证读取名称匹配",
    "Output classified and unclassified reads?": "输出分类和未分类的读取结果？",
    "Number of hits required for classification": "分类所需的命中数",
    "Print no Kraken output for unclassified sequences": "对于未分类的序列，不打印Kraken的输出",
    "classification.txt or ORF2LCA.txt from CAT contigs or CAT bins": "分类.txt 或者 ORF2LCA.txt 文件，来自 CAT 拼接的连续序列或 CAT 分箱的结果",
    "classification.names.txt output from CAT add_names": "分类名称.txt 文件，由 CAT add_names 输出",
    "contigs.fasta used for CAT contigs": "用于CAT的contigs.fasta文件",
    "Contig Sequences": "Contig序列",
    "Threshold for probability filtering": "概率过滤的阈值",
    "Diversity index to use": "多样性指数使用",
    "Margin for which the index is computed": "计算索引时的边界",
    "The logarithm base": "对数基数",
    "Subsample size for rarefying community": "稀有化群落的子样本大小",
    "Step size for sample sizes": "样本大小的步长",
    "X-axis label": "X轴标签",
    "Y-axis label": "Y轴标签",
    "Label rarefaction curves by rownames of X": "在X的行名上标记稀有度减少曲线",
    "Datasets to create": "创建数据集",
    "Candidate assemblies": "候选组装",
    "Candidate assembly file": "候选组装文件",
    "Name of the assembly": "组装的名称",
    "Type of input reads used for the assembly": "用于组装的输入读取类型",
    "Assembly input reads": "组装输入的读取",
    "Mate pair reads": "配对序列读取",
    "Assembly input first mate reads": "组装输入第一配对读取",
    "Assembly input second mate reads": "组装输入第二配对读取",
    "Mate paired read collections": "配对的配对读取集合",
    "Sliding window size when determining misassemblies": "滑动窗口大小在确定错误组装时",
    "Bowtie2 parameter to set the max number of alignments": "Bowtie2 参数设置最大比对数",
    "Minimum average coverage to run misassembly detection": "最小平均覆盖度以运行错误组装检测",
    "When binning by coverage, the new high = high + high * multiplier": "当按覆盖度进行分箱时，新高 = 高 + 高 * 乘数",
    "Minimum number of overlapping flagged miassemblies to mark region as suspicious": "标记区域为可疑的最小重叠标记的miassemblies数量",
    "Mark region as suspicious if multiple signatures occur within this window size": "如果在这个窗口大小内出现多个签名，则将该区域标记为可疑。",
    "Ignore contigs smaller than this length": "忽略小于这个长度的连续序列",
    "Ignore flagged regions within b bps from the ends of the contigs": "忽略从连续片段末端开始的 b bps 内的标记区域",
    "Bin sized used to find breakpoints": "用于查找断点的二进制大小",
    "Kmer length used for abundance estimation": "用于丰度估计的Kmer长度",
    "Assembly created per-contig coverage file": "每个连续序列创建覆盖度文件",
    "File containing ORFs": "包含开放阅读框（ORFs）的文件",
    "Specify the read type.": "指定读取类型。",
    "Single end read file(s)": "单端读取文件",
    "Forward paired-end read file": "正向配对端读取文件",
    "Reverse paired-end read file": "反向配对端读取文件",
    "Specify scheme to use. (Heidelberg is default)": "指定要使用的方案。（海德堡是默认的）",
    "Scheme Input": "方案输入",
    "Min k-mer freq/coverage": "最小k-mer频率/覆盖度",
    "Max k-mer freq/coverage": "最大 k-mer 频率/覆盖度",
    "Nonpareil algorithm?": "非等同算法？",
    "Factor for subsampling": "子采样因子",
    "Number of sub-samples to generate per point": "每个点生成的子样本数量",
    "Minimum overlapping percentage of the aligned region on the largest sequence": "最大序列上对齐区域的最小重叠百分比",
    "Maximum number of reads to use as query": "作为查询使用的最大读取数",
    "Report the sampled portions as a fraction of the library instead of the number of reads?": "将采样部分报告为库的一部分，而不是读取数的分数？",
    "Sampling": "抽样",
    "Minimum value of sampling portion": "采样部分的最小值",
    "Maximum value of sampling portion": "采样部分的最大值",
    "Interval between sampling portions": "采样部分之间的间隔",
    "Mating": "交配",
    "Do not use reverse-complement?": "不使用反向互补？",
    "Treat Ns as mismatches?": "将N视为不匹配？",
    "Similarity threshold": "相似性阈值",
    "Probability of taking a sequence into account as query for the construction of the curve": "作为曲线构建的查询，考虑序列的概率",
    "Random generator seed?": "随机数生成器种子？",
    "Input file (MetaPhlAN2 output)": "输入文件（MetaPhlAN2输出）",
    "Predicted taxon relative abundance tables generaed by MetaPhlAn": "由MetaPhlAn生成的预测物种相对丰度表",
    "Input file of microbial relative abundances": "输入微生物相对丰度的文件",
    "Format for the output image": "输出图像的格式",
    "Hierarchical clustering method": "层次聚类方法",
    "Distance function for samples": "样本距离函数",
    "Distance function for microbes": "微生物的距离函数",
    "Width of heatmap cells": "热图单元格的宽度",
    "Height of heatmap cells": "热图单元格的高度",
    "Minimum value to display": "最小显示值",
    "Maximum value to display": "最大显示值",
    "Taxonomic level to display": "显示的分类水平",
    "Percentile to be used for ordering the microbes": "用于对微生物排序的百分位数",
    "Number of most abundant microbes to display": "显示最丰富的微生物数量",
    "Height of the sample dendrogram": "样本树状图的高度",
    "Width of the sample dendrogram": "样本树状图的宽度",
    "Height of the colormap": "颜色图的高度",
    "Label font sizes": "标签字体大小",
    "Line width for the dendrograms": "树状图的线宽",
    "Percentile for which to scale data": "数据缩放所用的百分位数",
    "qFlag": "qFlag",
    "rel": "相对",
    "Percentile": "百分位数",
    "Log2 transform scaling": "对数2变换缩放",
    "The value to scale by.": "要乘以的数值。",
    "Choose dataset to convert": "选择要转换的数据集",
    "Select a taxonomy database": "选择一个分类数据库",
    "Read name": "读取名称",
    "Taxonomy ID field": "分类学ID字段",
    "Trait table": "性状表",
    "Gene Presence/Absence table from ROARY (default output)": "ROARY（默认输出）的基因存在/缺失表",
    "Table to analyze a subset of strains": "表格分析菌株的子集",
    "P-value cutoff(s) and Correction(s)": "P值截止值和校正",
    "P-value cutoff for one Trait": "单性状 P 值截止值",
    "P-value correction": "P值校正",
    "Permutations": "排列",
    "Maximal number of hits to report": "最大报告命中数",
    "Collapse correlated genes": "折叠相关基因",
    "Supply a custom tree (Newick format) for phylogenetic analyses instead instead of calculating it internally.": "提供自定义树（Newick格式）以进行系统发育分析，而不是在内部计算它。",
    "UPGMA tree": "UPGMA树",
    "Jellyfish command:": "水母命令：",
    "Input data file": "输入数据文件",
    "Length of mer": "mer的长度",
    "Initial hash size in Million (M)": "初始哈希大小（以百万计）",
    "Counter Length in bits": "计数器长度（以比特为单位）",
    "Count both strand and canonical representation": "计算正反两条链以及标准表示形式的数量",
    "Use bloom filters": "使用布隆过滤器",
    "Bloom counter file": "Bloom计数器文件",
    "False positive rate of bloom filter": "布隆过滤器的误报率",
    "Use bloom filter to count high-frequency mers": "使用布隆过滤器来计算高频子序列",
    "Count only k-mers in this file": "仅计算此文件中的k-mers",
    "Maximum number of repropbes": "最大重试次数",
    "Dump in text format": "文本格式转储",
    "Lower count": "较低计数",
    "upper count": "上计数",
    "Lower count value of histogram": "直方图的较低计数值",
    "Upper count value of histogram": "直方图的上限值",
    "Increment value for buckets": "增加桶的值",
    "Full histogram. Don't skip count 0": "完整直方图。不跳过计数0",
    "Use tab separator": "使用制表符分隔",
    "Don't output k-mer with count less than lower-count": "不要输出计数小于最小计数的k-mer",
    "Don't output k-mer with count greater than upper-count": "不要输出计数大于上限计数的k-mer",
    "Input(s)": "输入（s）",
    "Quality-controlled shotgun sequencing reads (metagenome (DNA reads) or metatranscriptome (RNA reads))": "质量控制的鸟枪测序读取（宏基因组（DNA读取）或宏转录组（RNA读取））",
    "Pre-computed mappings of reads to database sequences": "预先计算的将读取映射到数据库序列的映射",
    "Pre-computed (typically gene) abundance tables": "预计算的（通常是基因）丰度表",
    "Steps": "步骤",
    "Taxonomic profile file": "分类学档案文件",
    "Gene and pathway quantification": "基因和通路定量",
    "Use gap fill computation?": "使用间隙填充计算？",
    "Use minpath computation?": "使用最小路径计算？",
    "Database to use for pathway computations": "用于通路计算的数据库",
    "Use xipe computation?": "使用XIPE计算？",
    "Index of the gene in the sequence annotation": "序列注释中基因的索引",
    "id mapping file for alignments": "对齐的ID映射文件",
    "basename": "基名",
    "Format of the output files": "输出文件的格式",
    "Number of decimals to output": "输出小数位数",
    "Remove description in the output column?": "在输出列中删除描述？",
    "Remove stratification from output?": "从输出中去除分层？",
    "Intermediate output files": "中间输出文件",
    "Gene family or pathway table with metadata rows at the top": "带有元数据行顶部的基因家族或通路表",
    "Metadatum to test vs. community feature totals": "元数据与测试与社区功能总数",
    "End of metadata rows": "元数据行结束",
    "Metadatum type": "元数据类型",
    "FDR threshold": "FDR阈值",
    "HUMAnN table with optional metadata": "带有可选元数据的HUMAnN表格",
    "The name (header) of the last row containing metadata, if any": "最后一行包含元数据的名称（标题），如果有的话",
    "Feature ID of interest": "感兴趣的特征ID",
    "Manipulation of species contributions": "物种贡献的操纵",
    "Max taxon stratifications": "最大分类层级划分",
    "Collapse species to genera?": "将物种折叠为属？",
    "Do not include the 'unclassified' stratum?": "不要包含“未分类”的层级？",
    "Do not plot samples with zero sum for this feature?": "不要为该特征的零和样本绘图？",
    "Sample sorting methods": "样本排序方法",
    "Color space for taxa": "分类群的颜色空间",
    "Sample metadata": "样本元数据",
    "Metadata to highlight": "元数据以突出显示",
    "Number of most frequent metadata to keep": "保留最常见的元数据数量",
    "Color space for metadata levels": "元数据级别上的颜色空间",
    "Graphical tweaks": "图形调整",
    "Scaling options for total bar heights": "总条高缩放选项",
    "Minimum for y-axis": "y轴的最小值",
    "Don't plot y-axis grid lines?": "不绘制 y 轴网格线？",
    "Image height in inches": "图像高度（以英寸为单位）",
    "Image width in inches": "图像宽度（以英寸为单位）",
    "Name of y-axis abundance units": "y轴丰度单位名称",
    "Legend layout": "图例布局",
    "Number of legend columns": "图例列数",
    "Number of legend rows": "图例行数",
    "Ratio of legend to data axis height": "图例与数据轴高度的比例",
    "Format": "格式",
    "Gene/pathway table": "基因/通路表",
    "Gene/pathway/taxonomic table": "基因/通路/分类学表格",
    "Function to apply to select value to keep": "应用于选择要保留的值的函数",
    "How to sort the output": "如何对输出结果进行排序",
    "Gene families table": "基因家族表",
    "How to combine grouped features?": "如何合并分组特征？",
    "Grouping": "分组",
    "Mapping to use for the grouping": "用于分组的映射",
    "Mapping from features to groups?": "从特征映射到组？",
    "Custom mapping file for grouping": "自定义映射文件用于分组",
    "Is the groups file reversed?": "组文件是否已反转？",
    "Decimal places to round to after applying function": "应用函数后要舍入到的小数位数",
    "Include an 'UNGROUPED' group to capture features that did not belong to other groups?": "包含一个 '未分组' 分组以捕获不属于其他组的特征？",
    "Carry through protected features, such as 'UNMAPPED'?": "保留受保护的特性，例如 '未映射'？",
    "Type of feature renaming": "特征重命名类型",
    "Features to be renamed": "要重命名的特征",
    "Remove non-alphanumeric characters from names?": "从名称中移除非字母数字字符？",
    "Normalization scheme": "标准化方案",
    "Normalization level": "标准化水平",
    "Include the special features UNMAPPED, UNINTEGRATED, and UNGROUPED?": "包括特殊特性 UNMAPPED、UNINTEGRATED 和 UNGROUPED？",
    "Update '-RPK' in sample names to appropriate suffix?": "将样本名称中的'-RPK'更新为适当的后缀？",
    "Original DNA output table": "原始DNA输出表",
    "Original RNA output table": "原始 RNA 输出表格",
    "Choice of smoothing method": "平滑方法的选择",
    "Report log-transformed relative expression values?": "报告对数转换后的相对表达值？",
    "Base for log transformation": "对数转换的基数",
    "Stratified input table": "分层输入表",
    "Gene table to split": "基因表拆分",
    "Index of the gene in the taxonomy data": "分类数据中基因的索引",
    "Level of taxonomy for the output": "输出的分类学水平",
    "Merged gene families output for two or more samples": "合并两个或多个样本的基因家族输出",
    "Default mean non-zero gene abundance for inclusion": "默认非零基因丰度以包含",
    "Default non-zero number of genes for inclusion": "默认非零基因数量用于包含",
    "Low prevalence threshold": "低流行阈值",
    "High prevalence threshold": "高流行阈值",
    "Threshold number of samples having strain": "阈值样本数量具有菌株",
    "Limit output to species matching a particular pattern?": "限制输出以匹配特定模式的物种？",
    "Gene family or EC abundance file": "基因家族或EC丰度文件",
    "Pathway abundance file": "通路丰度文件",
    "Gene family to reaction mapping file": "基因家族与反应映射文件",
    "Reaction to pathway mapping file": "反应通路映射文件",
    "Remove the taxonomy from the output file?": "从输出文件中删除分类信息？",
    "Inputs": "输入",
    "Fasta/FastQ file(s) with microbiota reads": "微生物群落测序读取的Fasta/FastQ文件",
    "Single-end Fasta/FastQ file with microbiota reads": "单端Fasta/FastQ文件，包含微生物群落的读取数据",
    "Single-end Fasta/FastQ files with microbiota reads": "单端Fasta/FastQ文件，包含微生物群落的读取数据",
    "Forward paired-end Fasta/FastQ file with microbiota reads": "正向配对端的Fasta/FastQ文件，包含微生物群落的读取数据",
    "Reverse paired-end Fasta/FastQ file with microbiota reads": "反转成对的Fasta/FastQ文件，包含微生物群落的读取",
    "Minimum length of the reads to be considered when parsing the input file": "在解析输入文件时要考虑的读取的最小长度",
    "Presets options for BowTie2": "BowTie2的预设选项",
    "Minimum mapping quality value (MAPQ)": "最小映射质量值（MAPQ）",
    "Externally BowTie2-mapped SAM file": "外部 BowTie2 映射的 SAM 文件",
    "Intermediary mapping file of the microbiota generated by a previous MetaPhlAn run": "由之前的MetaPhlAn运行生成的微生物群落的中间映射文件",
    "Database with clade-specific marker genes": "具有特定类群标记基因的数据库",
    "Database with clade-specific marker genes from history": "历史中的特定类群标记基因数据库",
    "Metadata associated to the database with clade-specific marker genes from history": "与历史中具有特定类群标记基因的数据库相关联的元数据",
    "Analysis": "分析",
    "Clade for which to extract list of markers present": "提取标记物列表的目标类群",
    "The minimum percentage abundance for the clade": "该类群的最小百分比丰度",
    "Total number of reads in the original microbiota": "原始微生物群中读取的总数",
    "Threshold for calling a marker present": "标记物存在的判定阈值",
    "Minimum total nucleotide length for the markers in a clade for estimating the abundance without considering sub-clade abundances": "在不考虑子类群丰度的情况下，用于估计类群丰度的标记的最小总核苷酸长度",
    "Sam records for aligned reads with the longest subalignment length smaller than this threshold will be discarded.": "Sam记录对齐的读取，其中最长的子对齐长度小于此阈值的将被丢弃。",
    "Organisms to profile": "待分析的生物体",
    "Statistical approach for converting marker abundances into clade abundances": "将标记丰度转换为类群丰度的统计方法",
    "Quantile value for the robust average": "稳健平均的分位数值",
    "Percentage of markers with a non zero relative abundance for misidentify a species": "标记物非零相对丰度百分比用于误识别物种",
    "File containing a list of markers to ignore": "包含要忽略的标记列表的文件",
    "Deactivate the procedure of disambiguating the quasi-markers based on the marker abundance pattern found in the sample?": "停用基于样本中发现的标记丰度模式对准标记进行消歧义的过程？",
    "Sample ID key for this analysis": "此分析的样本ID键",
    "Sample ID for this analysis": "本次分析的样本ID",
    "Use a species as representative for species groups?": "使用一个物种作为物种组的代表？",
    "Old MetaPhlAn2 two columns output?": "旧版 MetaPhlAn2 的两列输出？",
    "Report the profiling using the CAMI output format?": "使用 CAMI 输出格式报告分析结果。",
    "Scale relative abundances to the number of reads mapping to known clades in order to estimate unknowness?": "将相对丰度缩放到已知类群的读取数，以估计未知性？",
    "Output for Krona?": "Krona的输出？",
    "Marker sequences": "标记序列",
    "Marker metadata": "标记元数据",
    "Customization": "定制化",
    "Sequences of the new markers": "新标记的序列",
    "Name of the new marker": "新标记的名称",
    "Length of the new marker": "新标记的长度",
    "Taxonomy of the genomes from which the new marker has been extracted": "提取新标记的基因组分类学",
    "Length of the genome": "基因组的长度",
    "GenBank assemblies id (GCA) of the genome": "基因组的GenBank组装编号（GCA）",
    "Kingdom: Name": "界：名称",
    "Kingdom: NCBI id": "界：NCBI编号",
    "Phylum: Name": "门：名称",
    "Phylum: NCBI id": "门：NCBI编号",
    "Class: Name": "类别：名称",
    "Class: NCBI id": "类别：NCBI 编号",
    "Order: Name": "排序：名称",
    "Order: NCBI id": "订单：NCBI 编号",
    "Family: Name": "家族：名称",
    "Family: NCBI id": "家族：NCBI 编号",
    "Genus: Name": "属：名称",
    "Genus: NCBI id": "属：NCBI 编号",
    "Species: Name": "物种：名称",
    "Species: NCBI id": "物种：NCBI编号",
    "Strain: Name": "菌株：名称",
    "List of markers to remove": "要移除的标记列表",
    "List of markers to keep": "标记物保留列表",
    "Fasta file containing contigs": "包含连续序列的Fasta文件",
    "Use a base coverage depth file?": "使用基础覆盖深度文件？",
    "Tabular depth matrix file having mean and variance of base coverage depth": "具有碱基覆盖深度的均值和方差的表格深度矩阵文件",
    "Base coverage depth file without variance": "基础覆盖深度文件，无变异",
    "Minimum size of a contig for binning": "用于分箱的最小片段大小",
    "Percentage of good contigs considered for binning decided by connection among contigs": "用于分箱的好的基因组片段百分比由基因组片段之间的连接决定",
    "Minimum score of an edge for binning": "边的最小得分用于分箱",
    "Maximum number of edges per node": "每个节点的最大边数",
    "TNF probability cutoff for building TNF graph": "TNF概率截止值，用于构建TNF图",
    "Turn off additional binning for lost or small contigs?": "关闭对丢失或小片段的额外分箱？",
    "Minimum mean coverage of a contig in each library for binning": "每个库中用于分箱的最小平均覆盖度的连通体",
    "Minimum total effective mean coverage of a contig for binning": "用于分箱的最小总有效平均覆盖度",
    "For exact reproducibility.": "为了精确重现。",
    "Minimum size of a bin as the output": "输出箱的最小大小",
    "Output only sequence labels as a list in a column without sequences?": "仅以一列的形式输出序列标签列表，而不包括序列本身？",
    "Save cluster memberships as a matrix format?": "将聚类成员保存为矩阵格式？",
    "Extra outputs": "额外输出",
    "Group the samples": "将样本分组",
    "Sort boxplots by": "按顺序对箱线图进行排序",
    "Flip the axes of the plot": "翻转图表的坐标轴",
    "Log10-scale the plot": "对数尺度绘制图表",
    "Add value to abundances": "为丰度添加值",
    "Add taxonomic strings to the output": "在输出中添加分类学字符串",
    "Group samples": "分组样本",
    "Facet the samples": "对样本进行分面",
    "Display sum of remaining taxa": "显示剩余分类群的总和",
    "Plot the values on the heatmap": "在热图上绘制值",
    "Size of the plotted values": "绘制值的大小",
    "Type of scale used for coloring abundances": "用于着色丰度的刻度类型",
    "Color missing values with the lowest color in the scale": "将缺失值用该尺度中最低的颜色表示",
    "Value to show across the groups": "在组间显示的值",
    "Sort heatmap by most abundant taxa": "根据最丰富的类群对热图进行排序",
    "Group to sort by": "按组排序",
    "Sample to sort by": "样本排序",
    "Normalize counts by a variable or a specific sample": "通过一个变量或特定样本来标准化计数",
    "Scale the abundances by a variable in the metadata": "根据元数据中的变量调整丰度",
    "Show functional information about the Genus-level OTUs": "显示关于属级操作分类单元（OTUs）的功能性信息",
    "Tabular dataset with functional information at Genus level": "具有属水平功能信息的表格数据集",
    "Function(s) to include": "要包含的功能",
    "OTU table": "操作分类单元(OTU)表",
    "OTU/ASV column has empty header": "OTU/ASV 列的标题为空",
    "ASV identifiers are the ASV sequences": "ASV标识符是ASV序列",
    "Guess metadata column types": "猜测元数据列类型",
    "Taxonomy table": "分类学表",
    "Phylogenetic tree": "系统发育树",
    "Remove singleton OTUs": "移除单例操作分类单元（OTUs）",
    "Output list data sets": "输出列表数据集",
    "Merge by exact matches between DNA reference sequences": "精确匹配DNA参考序列进行合并",
    "Variable that defines the sample groups": "定义样本组的变量",
    "Round decimals after merging": "合并后四舍五入小数",
    "Aggregate (sum) OTU's to a taxonomic level initially.": "将操作分类单元（OTU）的计数聚合（求和）到一个分类水平上。",
    "Group the samples based on a metadata variable.": "根据元数据变量对样本进行分组。",
    "Abundance threshold": "丰度阈值",
    "Auto-generate a figure caption": "自动生成图表标题",
    "Color sample points by": "根据...给样本点上色",
    "Shape sample points by": "根据...塑造样本点",
    "Frame the sample points with a polygon by": "通过多边形框选样本点",
    "Label Frame by": "通过标签框架",
    "Label sample points by": "标记样本点通过",
    "Make a trajectory between sample points by": "通过在样本点之间制作一条轨迹",
    "Make a trajectory between sample points by the sample_trajectory argument, but within individual groups.": "通过 sample_trajectory 参数在样本点之间创建轨迹，但在各个组内进行。",
    "Plot species points": "绘制物种点",
    "Number of the most extreme species labels to plot": "要绘制的最为极端的物种标签数量",
    "Taxonomic level by which to label the species points": "用于标记物种点的分类水平",
    "Size of the species text labels": "物种文本标签的大小",
    "Categorical variables to fit onto the ordination plot": "分类变量以适应于排序图",
    "Numerical variables to fit arrows onto the ordination plot.": "数值变量，用于将箭头拟合到排序图上。",
    "Significance threshold": "显著性阈值",
    "Repel all labels": "排斥所有标签",
    "Opacity of plotted points and colorframe": "绘制点的不透明度和颜色框架",
    "Output screeplot": "输出散点图",
    "Group the samples by": "将样本按分组",
    "Show standard deviation": "显示标准差",
    "Log10-transform the x-axis": "对x轴进行对数变换",
    "Color curves by": "按颜色分类曲线",
    "Facet curves by": "根据...绘制图表",
    "Character metadata variables": "字符元数据变量",
    "Numerical metadata variables": "数值元数据变量",
    "Discrete numerical metadata variables (integers)": "离散数值元数据变量（整数）",
    "Date metadata variables": "日期元数据变量",
    "Metadata variable": "元数据变量",
    "Metadata value(s)": "元数据值（们）",
    "Minimum number of reads": "最小读数",
    "Taxonomy list": "分类学列表",
    "Taxa to keep": "保留的分类群",
    "Time variable": "时间变量",
    "Create a facet for each taxon": "为每个分类群创建一个分面",
    "Group the data by": "按分组数据",
    "Abundance cutoff (percent)": "丰度截止（百分比）",
    "Frequency cutoff (percent)": "频率截止（百分比）",
    "Size of the plotted text.": "绘制文本的大小。",
    "Interleaved": "交错的",
    "Nanopore reads": "纳米孔测序读取",
    "Contigs/genomes": "Contigs/基因组",
    "Include log as output file.": "包括日志作为输出文件。",
    "Kraken report file": "Kraken报告文件",
    "Select a Kmer distribution": "选择一个Kmer分布",
    "Number of mismatches allowed when matching tag": "允许匹配标签时的不匹配数",
    "Produce Kraken-Style Bracken report": "生成Kraken风格的Bracken报告",
    "Add log file output": "添加日志文件输出",
    "Additional reads": "额外的读取",
    "Coverage calculation options": "覆盖度计算选项",
    "Method(s) for calculating coverage": "计算覆盖度的方法",
    "Genome definition": "基因组定义",
    "Exclude genomes from deshard": "排除基因组中的碎片化区域",
    "Dereplication / Genome clustering": "去冗余/基因组聚类",
    "Do genome dereplication via average nucleotide identity (ANI)?": "通过平均核苷酸身份（ANI）进行基因组去重？",
    "Overall ANI level to dereplicate at with FastANI": "FastANI 用于去重的总体 ANI 水平",
    "Minimum aligned fraction of two genomes for clustering": "两个基因组聚类的最小比对分数",
    "Length of fragment used in FastANI calculation (i.e. --fragLen)": "FastANI计算中使用的片段长度（即--fragLen）",
    "Scoring function for genome quality": "基因组质量评分函数",
    "Dereplication preclustering threshold": "去重预聚类阈值",
    "Method of calculating rough ANI for dereplication": "用于去重的粗略平均核苷酸一致性（ANI）计算方法",
    "checkM table for defining genome quality": "用于定义基因组质量的checkM表格",
    "dRep style genome info table for defining quality": "dRep 风格的基因组信息表，用于定义质量",
    "Minimum completeness": "最小完整性",
    "Maximum contamination": "最大污染",
    "Omit printing of genomes that have zero coverage?": "忽略打印覆盖度为零的基因组？",
    "Output a tabular files with dereplicated representatives and member lines?": "输出一个包含去重代表性序列和成员行的表格文件。",
    "Output dereplicated representative genomes?": "输出去重后的代表性基因组？",
    "Bin sequences": "二进序列",
    "Sequence table": "序列表",
    "Sample table": "样本表",
    "Mode to process BAM files": "处理BAM文件的模式",
    "Sorted bam files": "排序后的BAM文件",
    "Minimum end-to-end percent identity of qualifying reads": "合格读取的端到端最小百分比相似性",
    "Output the sparse matrix of contigs which paired reads span?": "输出成对的读取跨越的片段的稀疏矩阵。",
    "Remove variance from mean depth along the contig?": "去除沿连续序列的平均深度中的方差？",
    "Output a depth file per bam for each contig base?": "为每个BAM文件输出每个Contig碱基的深度文件？",
    "Minimum mapping quality necessary to count a read as mapped": "最小映射质量，用于将一个读取计数为已映射",
    "Weight per-base depth based on the MQ of the read": "基于读取的映射质量（MQ）的每碱基深度权重",
    "Include 1-readlength edges when calculating depth and variance?": "在计算深度和方差时，是否包括1个读长的边缘？",
    "Maximum length when calculating depth and variance": "计算深度和方差时的最大长度",
    "Options to control shredding contigs that are under-represented by the reads": "控制被读取不足的片段被切碎的选项",
    "Maximum length of the shreds": "片段的最大长度",
    "Depth to generate overlapping shreds": "生成重叠片段的深度",
    "Mimimum length of contig to include for mapping and shredding": "用于映射和切碎的最小长度的连续序列",
    "Minimum depth along the contig at which to break the contig": "在连续序列上断开连续序列的最小深度",
    "Contig annotations": "Contig注释",
    "Training": "训练",
    "Binning": "分箱",
    "Model to use": "使用模型",
    "Trained semi-supervised deep learning model": "训练有素的半监督深度学习模型",
    "Mode to train the models": "训练模型的模式",
    "Train data": "训练数据",
    "Split train data": "分割训练数据",
    "Cannot-link constraints": "无法链接约束",
    "A file containing metagenomic reads mapped to a DNA sequence": "一个包含已映射到DNA序列的宏基因组读取文件",
    "A file containing a DNA sequence.": "一个包含DNA序列的文件。",
    "Use full fasta header": "使用完整的fasta头信息",
    "Read Filtering": "读取过滤",
    "Minimum mapq score": "最小映射质量分数",
    "Maximum insert relative": "最大插入相对",
    "Minimum insert": "最小插入",
    "How should paired reads be handled?": "成对的读取应该如何处理？",
    "The location of a list of reads that should be retained regardless of pairing status": "保留列表中的位置，无论配对状态如何，这些读取都应该保留。",
    "Variant Calling": "变异检测",
    "Minimum SNP frequency": "最小SNP频率",
    "Gene Profiling": "基因分析",
    "Path to prodigal .fna genes file. If file ends in .gb or .gbk, will treat as a genbank file": "产蛋（Prodigal）.fna基因文件的路径。如果文件以.gb或.gbk结尾，将被视为基因库文件。",
    "Scaffold to bin": "将序列组装成二进制文件。",
    "Create output files on the mm level": "在mm级别上创建输出文件",
    "Skip mm profiling": "跳过 mm 特征分析",
    "Profile": "概要",
    "Database mode": "数据库模式",
    "Minimum scaffold reads": "最小支架读取数",
    "Minimum genome coverage": "最小基因组覆盖度",
    "Minimum SNP": "最小单核苷酸多态性",
    "Store everything": "存储一切",
    "Scaffolds to profile": "支架以分析概要",
    "Rarefied coverage": "稀有覆盖度",
    "Set Output Parameters": "设置输出参数",
    "Detailed mapping info": "详细映射信息",
    "Skip genome wide": "跳过全基因组范围",
    "Skip plot generation": "跳过绘图生成",
    "inStrain Profile IS folder": "inStrain 概要 IS 文件夹",
    "Variant Calling Options": "变体检测选项",
    " Minimum coverage to call an variant": "最小覆盖度以调用一个变体",
    "Minimum SNP frequency to confirm a SNV": "最小SNP频率以确认一个SNV",
    "Database Mode Parameters": "数据库模式参数",
    "Automatically determine which genomes are present in each Profile and only compare scaffolds from those genomes.": "自动确定每个Profile中存在的基因组，并仅比较这些基因组中的scaffolds。",
    "Minimum breadth_minCov required to count a genome present": "最小广度_minCov，要求计算一个基因组存在",
    "Location to a list of scaffolds to compare. You can also make this a .fasta file and it will load the scaffold names": "比较支架位置的列表。您也可以将其制作成.fasta文件，它将加载支架名称",
    "Run scaffolds belonging to this single genome only. Must provide an .stb file": "仅运行属于这个单一基因组的支架。必须提供一个.stb文件。",
    "Store coverage overlap on an mm level": "在毫米级别上存储覆盖重叠",
    "Store the locations of SNPs": "存储SNPs的位置",
    "Compare IS profiles against themself": "比较插入位点（IS）图谱与它们自身",
    "Dont create plots at the end of the run": "不要在运行结束时创建图表",
    "How many bp to compare simultaneously": "同时比较多少个碱基对",
    "Genome Clustering Options": "基因组聚类选项",
    "popANI threshold to cluster genomes at": "popANI聚类基因组的阈值",
    "Minimum percent_genome_compared for a genome comparison to count": "基因组比较计数的最小百分比基因组比较",
    "Algorithm used to cluster genomes": "用于聚类基因组的算法",
    "Data (or features) file": "数据（或特征）文件",
    "Interactions: Fixed effects": "交互作用：固定效应",
    "Random effects": "随机效应",
    "Minimum prevalence": "最小流行率",
    "Maximum significance": "最大显著性",
    "The normalization method to apply": "要应用的标准化方法",
    "The transform to apply": "要应用的转换",
    "The analysis method to apply": "要应用的分析方法",
    "Correction": "校正",
    "Apply z-score so continuous metadata are on  the same scale": "应用z分数，使连续的元数据在同一尺度上",
    "Set Plotting Output": "设置绘图输出",
    "Generate a heatmap for the significant  associations": "生成一个显著相关性的热图",
    "Heatmap plot first N": "热图绘制前N个",
    "Generate scatter plots for the significant associations": "生成显著关联的散点图",
    "Output data frame with residuals for each feature": "输出包含每个特征残差的数据框",
    "Input file containing DNA or protein sequences": "包含DNA或蛋白质序列的输入文件",
    "Select alignment mode": "选择比对模式",
    "Alignment match disjointScore": "对齐匹配 断开分数",
    "Alignment mis-match disjointScore": "对齐不匹配 - disjointScore",
    "Parameter Lambda for BLASTN statistics": "BLASTN 统计参数 Lambda",
    "Parameter K for BLASTN statistics": "BLASTN 统计参数 K",
    "Alignment type": "对齐类型",
    "Advanced options for performance": "高级性能选项",
    "Memory mode": "内存模式",
    "Maximum number of seed tables to use": "最大种子表使用数量",
    "Cache results for replicated queries?": "缓存复制查询的结果？",
    "Advanced options for filter": "高级过滤选项",
    "Minimum bit disjointScore": "最小位不相交分数",
    "Maximum expected disjointScore": "最大期望的不相交分数",
    "Maximum number of alignments per query": "每个查询的最大比对数",
    "Maximum number of (non-overlapping) alignments per reference": "每个参考序列的最大（不重叠）比对数",
    "Advanced options for LCA": "LCA的高级选项",
    "Top percent value for LCA algorithm": "LCA算法的置信度百分比值",
    "Minimum support value for LCA algorithm as a percent of assigned reads": "LCA算法的最小支持值为分配读取的百分比",
    "Minimum support value for LCA algorithm": "LCA算法的最小支持值",
    "Minimum percent identity used by the LCA algorithm": "最小百分比同一性，由LCA算法使用",
    "Use percent identity assignment filter?": "使用百分比一致性分配过滤器？",
    "Use the weighted LCA for taxonomic assignment?": "使用加权最近公共祖先（LCA）进行分类学分配？",
    "Reads have magnitudes (to be used in taxonomic or functional analysis)?": "读取具有大小（用于分类学或功能分析）？",
    "Advanced options for heuristics": "启发式算法的高级选项",
    "Maximum number of seed matches per offset per read frame": "每个偏移量每个读取框架的最大种子匹配数",
    "Maximum number of seed matches per read and reference": "每个读段和参考序列的最大种子匹配数",
    "Seed shift": "种子位移",
    "Advanced options for alignment": "高级比对选项",
    "Band width/2 for banded alignment": "带宽/2用于带状对齐",
    "Specify additional outputs?": "指定额外输出？",
    "Include unaligned queries in RMA output file?": "在RMA输出文件中包含未比对的查询？",
    "Output alignments?": "输出比对结果？",
    "Alignment format": "对齐格式",
    "Output all reads that have at least one significant alignment to the reference?": "输出所有至少有一个显著比对到参考序列的读段。",
    "Output all reads that do not have any significant alignment to the reference?": "输出所有与参考序列没有显著比对的读段。",
    "Output file of DIAMOND or MALT on input forward read file": "DIAMOND或MALT软件在输入正向读取文件时的输出文件",
    "Output file of DIAMOND or MALT on input reverse read file": "DIAMOND 或 MALT 的输出文件在输入反向读取文件时",
    "Length of name suffix used to distinguish read names": "用于区分读名字的名称后缀的长度",
    "Collection of paired read files": "成对的读取文件集合",
    "Files containing metadata to be included in the output files": "包含要包含在输出文件中的元数据的文件",
    "DAA file was created using paired reads?": "DAA文件是使用配对读取创建的吗？",
    "Blast mode": "Blast模式",
    "Blast file": "Blast 文件",
    "Map reads to KEGG KOs?": "将读取映射到KEGG KOs？",
    "Show taxonomic ranks?": "显示分类等级？",
    "Report only taxa that have an official rank?": "只报告具有官方等级的分类群？",
    "Report taxon ids rather than taxon names?": "报告分类群的ID而不是分类群的名称？",
    "Maximum number of KEGG assignments to report for a read": "为一个读取报告的最大KEGG分配数量",
    "Apply top percent filter in KEGG KO analysis?": "在KEGG KO分析中应用顶部百分比过滤器？",
    "Accession-to-Taxonomy mapping file": "访问号到分类学映射文件",
    "Synonyms-to-Taxonomy mapping file": "同义词到分类学映射文件",
    "Accession-to-KEGG mapping file": "访问号到KEGG映射文件",
    "Synonyms-to-KEGG mapping file": "同义词到KEGG映射文件",
    "First word in reference header is accession number?": "参考头文件中的第一个词是访问号码？",
    "List of accession tags": "访问标签列表",
    "Binned fasta files": "分箱的fasta文件",
    "Minimum size (Kbp) of refined bin": "最小大小（千碱基对）的精炼箱",
    "Coverage file": "覆盖度文件",
    "Composition file with sequences": "序列组成文件",
    "Maximum number of clusters for the Variational Gaussian Mixture Model (VGMM) algorithm": "变分高斯混合模型（VGMM）算法的最大聚类数",
    "Kmer length": "Kmer长度",
    "Sequence length threshold": "序列长度阈值",
    "Read length for coverage": "读取长度用于覆盖度",
    "Percentage of variance explained by the principal components for the combined data": "主成分对组合数据解释的方差百分比",
    "Seed for clustering": "聚类种子",
    "Maximum number of iterations for the Variational Bayes Gaussian Mixture Models (VBGMM)": "变分贝叶斯高斯混合模型（VBGMM）的最大迭代次数",
    "Skip normalization and only do log transorm of the coverage?": "跳过归一化，只对覆盖度进行对数转换？",
    "Eliminate the total coverage column from the coverage data matrix?": "从覆盖度数据矩阵中移除总覆盖度列？",
    "Write convergence information to files?": "将收敛信息写入文件？",
    "Output process log file?": "输出处理日志文件？",
    "Contigs BEDFile": "Contigs BED文件",
    "Type of assembly used to generate the contigs": "用于生成连续序列的组装类型",
    "Sorted BAM files": "排序后的BAM文件",
    "Fasta contigs file": "Fasta 序列片段文件",
    "Overlap size": "重叠大小",
    "Concatenate final part to last contig?": "将最终部分连接到上一个连续片段上？",
    "Output bed file with exact regions of the original contigs corresponding to the newly created contigs?": "输出与新创建的contigs相对应的原始contigs的确切区域的bed文件？",
    "Original contig file": "原始的连续序列文件",
    "CONCOCT clusters": "CONCOCT 聚类",
    "Clusters generated by CONCOCT": "由CONCOCT生成的聚类",
    "Input DIAMOND file": "输入DIAMOND文件",
    "Input is meganized?": "输入是被MEGAN化的？",
    "List class to count for named classification(s)?": "列出用于命名分类的计数类。",
    "Select class to count for named classification(s)": "选择要计数的命名分类",
    "Use summarized rather than assigned counts when listing class to count?": "在列出类别到计数时，使用汇总的而不是分配的计数？",
    "List read to class assignments for named classification(s)?": "列出用于命名分类的读取到类别分配的列表？",
    "Select read to class assignments for named classification(s)": "选择对命名分类的读段进行类别分配",
    "Report class names rather than class Id numbers?": "报告类名而不是类ID编号？",
    "Report class paths rather than class Id numbers?": "报告类路径而不是类ID数字？",
    "When reporting class paths for taxonomy, prefix single letter to indicate taxonomic rank?": "在报告分类路径时，为分类等级添加单个字母前缀？",
    "Only use major taxonomic ranks?": "只使用主要的分类等级？",
    "Restrict reporting to either bacterial reads or viral reads in taxonomic report": "在分类学报告中，将报告限制为细菌读段或病毒读段",
    "Don't report on reads that are unassigned?": "不报告未分配的读取？",
    "Output a MEGAN summary file?": "输出一个MEGAN摘要文件？",
    "Extract frame-shift corrected reads?": "提取移码校正后的读取序列？",
    "Specify classification to use?": "指定要使用的分类？",
    "Select classification to use": "选择要使用的分类",
    "Names (or ids) of classes from which to extract reads": "提取序列的类别（或ID）名称",
    "Ignore exceptions and continue processing?": "忽略异常并继续处理？",
    "Sequence selection options": "序列选择选项",
    "Profile set to use for the search": "用于搜索的配置文件集",
    "Domain E-value cutoff": "域 E 值截止",
    "Domain score cutoff": "域得分截断值",
    "Minimal number of domains": "最小域数",
    "Selection priority": "选择优先级",
    "HMMER search cutoff mode": "HMMER搜索截止模式",
    "E-value cutoff used in the HMMER search": "在HMMER搜索中使用的E值截止值",
    "The score cutoff used in the HMMER search": "HMMER搜索中使用的分数截止值",
    "Allow single domain": "允许单个域",
    "Allow reorder": "允许重新排序",
    "Check both DNA strains": "检查两个DNA菌株",
    "HMMER's heuristic filtering": "HMMER的启发式过滤",
    "Use nhmmer instead of hmmsearch": "使用 nhmmer 而不是 hmmsearch",
    "Preserve sequence headers in input file": "在输入文件中保留序列标题",
    "Save regions": "保存区域",
    "Only full-lenght regions": "仅全长区域",
    "Saves additional FASTA-files for full and partial ITS sequences longer than the specified cutoff": "保存额外的FASTA文件，用于超过指定截止长度的完整和部分ITS序列。",
    "Minimum length the ITS regions must be to be outputted in the concatenated file": "ITS区域在连接文件中输出的最小长度",
    "Truncates the FASTA output to only contain the actual ITS sequences found": "截断FASTA输出，只包含实际找到的ITS序列",
    "Reads table file": "读取表格文件",
    "Comparison group": "比较组",
    "Number of columns in group": "组内列数",
    "Number of Monte Carlo samples to use when estimating the underlying distributions": "在估计底层分布时使用的蒙特卡洛样本数量",
    "Select features to retain as the denominator for the Geometric Mean calculation": "选择保留为几何平均数计算的分母的特征",
    "Select the analysis to be performed": "选择要执行的分析",
    "Group number": "组号",
    "Select the plot type": "选择图表类型",
    "Select the method of calculating significance": "选择计算显著性的方法",
    "Benjamini-Hochberg fdr cutoff": "本杰明尼-霍奇伯格 FDR 截止值",
    "Effect size cutoff for plotting": "绘制效应大小的截止值",
    "x-label for the plot": "图的x轴标签",
    "y-label for the plot": "图表的 y 轴标签",
    "Name of the feature from the input data to be plotted": "要绘制的输入数据中的特征名称",
    "Calculate effect size for paired samples?": "计算配对样本的效应量大小？",
    "Plot a histogram of p-values for the first Dirichlet Monte Carlo instance?": "绘制第一个狄利克雷蒙特卡洛实例的p值直方图。",
    "File containing a phyloseq object": "包含phyloseq对象的文件",
    "Formula": "公式",
    "Select method to adjust p-values": "选择调整p值的方法",
    "Minimum taxa prevalence": "最小类群丰度",
    "Threshold for filtering samples based on library sizes": "基于库大小的样本过滤阈值",
    "Name of the group variable in the metadata": "元数据中组变量的名称",
    "Detect structural zeros based on group?": "基于组别检测结构性零值？",
    "Classify a taxon as a structural zero using its asymptotic lower bound?": "将一个分类群作为结构性零点使用其渐近下界进行分类？",
    "Iteration convergence tolerance for the E-M algorithm": "E-M算法的迭代收敛容差",
    "Maximum number of iterations for the E-M algorithm": "最大迭代次数用于期望最大化（E-M）算法",
    "Use a conservative variance estimator for the test statistic?": "使用保守的方差估计器来计算检验统计量？",
    "Level of significance": "显著性水平",
    "Perform global test?": "执行全局检验？",
    "Marker gene identification": "标记基因鉴定",
    "Bin assessment": "Bin 评估",
    "Bin placement in the genome tree and marker gene identification": "基因组树中的二进制放置和标记基因识别",
    "Bin lineage-specific marker set inference": "Bin 谱系特异性标记集推断",
    "Desired output": "期望输出",
    "Concatenated masked sequences": "串联掩蔽序列",
    "Sequences used to generate bins": "用于生成箱的序列",
    "Marker gene HMM info for each bin": "每个bin的标记基因HMM信息",
    "Marker gene bin stats": "标记基因箱统计",
    "Marker gene HMM hits to each bin": "标记基因 HMM 命中到每个 bin",
    "Nucleotide gene sequences for each bin": "每个bin的核苷酸基因序列",
    "Markers to exclude from marker sets": "排除标记集中的标记",
    "Coverage of each sequence": "每个序列的覆盖度",
    "Plot to generate": "生成图表",
    "x step size for calculating Nx": "计算Nx时的x步长",
    "Marker gene stats": "标记基因统计",
    "Marker gene bin extensive stats": "标记基因箱广泛统计",
    "Image type": "图像类型",
    "DPI of output image": "输出图像的分辨率",
    "Width of output image": "输出图像的宽度",
    "Height of output image": "输出图像的高度",
    "Concatenated tree": "串联树",
    "Reads files type for anaylsis": "读取文件类型进行分析",
    "Single end read file(s) for analysing (may be gzipped)": "用于分析的单端读取文件（可能是gzip压缩的）",
    "Paired end read files for analysing (may be gzipped)": "用于分析的成对末端读取文件（可能是gzip压缩的）",
    "Do you want to merge the data to get a single result": "您是否希望合并数据以获得单一结果",
    "Do you want to provide an MLST database of all allele sequences for the MLST scheme?": "您是否要提供包含所有等位基因序列的多位点序列分型（MLST）数据库？",
    "Fasta file of MLST alleles": "多重定位序列分型等位基因的Fasta文件",
    "ST definitions for MLST scheme": "MLST方案的ST定义",
    "Character(s) separating gene name from allele number in MLST database": "在MLST数据库中，分隔基因名称和等位基因编号的字符",
    "Maximum number of mismatches per read for MLST allele calling": "每个读取的最大不匹配数，用于 MLST 等位基因调用",
    "Minimum mean depth to flag as dubious allele call": "最小平均深度以标记为可疑等位基因调用",
    "Minimum edge depth to flag as dubious allele call": "最小边缘深度以标记为可疑等位基因调用",
    "Do you want to use a Gene database(s)?": "您想要使用基因数据库吗？",
    "Gene database(s)": "基因数据库（复数）",
    "Do you want reporting of gene typing?": "您需要进行基因分型报告吗？",
    "Maximum number of mismatches per read for gene detection and allele calling": "每个读取用于基因检测和等位基因调用时的最大不匹配数",
    "Minimum %coverage cutoff for gene reporting": "基因报告的最小%覆盖度截止值",
    "Maximum %divergence cutoff for gene reporting": "基因报告的最大%差异截止值",
    "Select all outputs you need": "选择你需要的所有输出",
    "Probability of sequencing error": "测序错误的概率",
    "% increase in score allowed to choose non-truncated allele": "允许选择非截断等位基因的分数百分比增加",
    "Stop mapping after this number of reads have been mapped": "在映射了这么多的读段后停止映射",
    "Read discarded from alignment": "从比对中丢弃的读取",
    "Samtools -q parameter (Minimum mapping quality)": "Samtools -q 参数（最小映射质量）",
    "Samtools -Q parameter (Minimum base quality)": "Samtools -Q 参数（最小碱基质量）",
    "SRST2 results files to compile": "SRST2 结果文件进行编译",
    "Input files to Kraken-biom: Kraken report output file(s)": "输入文件到Kraken-biom：Kraken报告输出文件",
    "Max": "最大值",
    "Min": "最小值",
    "Sample metadata file": "样本元数据文件",
    "Do you want to create an OTU IDs file": "您想要创建一个OTU ID文件吗？",
    "Type of sequencing data": "测序数据类型",
    "Sequenced reads": "测序读取",
    "Paired collection of sequenced reads": "成对的测序读取集合",
    "Limit number of reported matches to": "限制报告的匹配数量为",
    "Optional arguments": "可选参数",
    "Kmer Length": "Kmer长度",
    "Read kmer filtering threshold": "读取k-mer过滤阈值",
    "Coverage threshold for k-mer culling": "k-mer 筛选的覆盖度阈值",
    "Minimum k-mer proportion": "最小k-mer比例",
    "Fraction of best seeds to extend": "最佳种子延伸的比例",
    "Annotations file": "注释文件",
    "FASTA file to filter": "FASTA 文件过滤",
    "Space-separated list of fastas to keep": "以空格分隔的要保留的fasta列表",
    "Space-separated list of scaffolds to keep": "以空格分隔的要保留的支架列表",
    "rRNA file": "rRNA文件",
    "tRNA file": "tRNA文件",
    "Column from annotations to group as organism units": "将注释中的列作为有机体单位进行分组",
    "Give names of genes instead of counts in genome metabolism summary?": "在基因组代谢摘要中用基因名称代替计数？",
    "Number of genomes per product.html output": "每个product.html输出的基因组数量",
    "Minimum contig size to be used for gene prediction": "用于基因预测的最小基因组片段大小",
    "Select mode of prodigal to use for gene calling": "选择 Prodigal 用于基因调用的模式",
    "Translation table for prodigal to use for gene calling": "用于基因调用的prodigal翻译表",
    "Minimum bitScore of search to retain hits": "搜索结果保留命中的最小比对分数",
    "Minimum bitScore of reverse best hits to retain hits": "保留命中结果的反向最佳命中的最小比对分数",
    "Use dbcan2 suggested HMM cutoffs for KOfam annotation instead of KOfam recommended cutoffs?": "使用dbcan2建议的HMM截断值进行KOfam注释，而不是KOfam推荐的截断值？",
    "Specify additional fasta files against which to annotate?": "指定用于注释的额外fasta文件？",
    "Custom FASTA database name": "自定义FASTA数据库名称",
    "Custom database FASTA file": "自定义数据库FASTA文件",
    "Specify additional hmm files against which to annotate?": "指定额外的hmm文件以用于注释",
    "HMM file": "HMM文件",
    "Custom HMM database name": "自定义HMM数据库名称",
    "Custom database HMM file": "自定义数据库 HMM 文件",
    "Custom HMM cutoffs and descriptions file": "自定义HMM截断值和描述文件",
    "Annotate the input FASTA files against UniRef?": "将输入的FASTA文件注释到UniRef数据库中。",
    "Annotate the input FASTA files against VOGDB?": "注释输入的FASTA文件与VOGDB？",
    "Skip annotating with uniref and use kofam?": "跳过使用uniref进行注释，并使用kofam？",
    "Skip transcan": "跳过transcan",
    "Collection of annotation files": "注释文件集合",
    "Collection of genbank files": "基因库文件集合",
    "Collection of trna files": "tRNA文件集合",
    "Collection of rrna files": "收集 rRNA 文件",
    "Collection of scaffolds files": "收集支架文件集合",
    "Collection of genes.gff files": "基因集.gff文件的集合",
    "Collection of genes.fna files": "基因.fna文件集合",
    "Collection of genes.faa files": "基因集合的.faa文件",
    "File containing genes to filter to neighborhoods": "包含用于过滤到邻域的基因的文件",
    "File containing scaffolds to filter to neighborhoods": "包含要过滤到邻域的支架的文件",
    "Number of genes away from center to include in neighborhoods": "中心周围包含的基因数量",
    "Number of base pairs away from center to include in neighborhoods": "中心两侧包含的碱基对数",
    "Include intermediate (i.e. non-standard) ranks": "包含中间（即非标准）等级",
    "Taxonomy file(s)": "分类学文件（们）",
    "Specify type of input file(s)": "指定输入文件的类型",
    "Specify alpha diversity type": "指定alpha多样性类型",
    "Include intermediate ranks": "包含中间等级",
    "Use percentage of total reads for output.": "使用总读取数的百分比作为输出。",
    "Kraken report file(s)": "Kraken报告文件",
    "display headers in output": "在输出中显示标题",
    "display only combined read counts and percentages": "仅显示合并的读数计数和百分比",
    "Single or paired reads?": "单端或配对读取？",
    "FASTQ/A forward file": "FASTQ/A正向文件",
    "FASTQ/A reverse file": "FASTQ/A 反向文件",
    "Taxonomic ID(s) to match": "分类学ID（或多个ID）以匹配",
    "Maximum reads to save": "最大保存的读取数",
    "Invert output": "反转输出",
    "Output as FASTQ": "输出为FASTQ格式",
    "Include parents": "包含父节点",
    "Include children": "包括子节点",
    "Input file corresponding to MetaPhlAn output": "与MetaPhlAn输出对应的输入文件",
    "Input file corresponding to HUMAnN output": "输入文件对应于HUMAnN输出",
    "Type of characteristics in HUMAnN file": "HUMAnN 文件中的特征类型",
    "Use cached database or database from history": "使用缓存的数据库或历史中的数据库",
    "Using built-in mapseq DB": "使用内置的mapseq数据库",
    "Create OTU table": "创建OTU表",
    "Create taxon table for Krona": "创建适用于Krona的分类群表格",
    "Database file (FASTA format)": "数据库文件（FASTA格式）",
    "Taxonomy file": "分类学文件",
    "Database cluster": "数据库集群",
    "Create out of the MAPseq output a tab-separated output file?": "从MAPseq输出创建一个制表符分隔的输出文件？",
    "Top hits": "顶级命中",
    "Top OTUs": "顶级操作分类单元",
    "Minimum number of shared kmers": "最小共享k-mer数",
    "Number of ref. sequences": "参考序列数量",
    "OTU limit": "OTU 限制",
    "BIOM File": "BIOM 文件",
    "Sample Metadata Tabular File": "样本元数据表格文件",
    "Observation Metadata Tabular File": "观察元数据表格文件",
    "Comma-separated list of the metadata fields to split on semicolons": "逗号分隔的元数据字段列表，用分号分隔",
    "Comma-separated list of the metadata fields to split on semicolons and pipes (|)": "以逗号分隔的元数据字段列表，用分号和竖线（|）分隔",
    "Comma-separated list of the metadata fields to cast to integers": "以逗号分隔的要转换为整数的元数据字段列表",
    "Comma-separated list of the metadata fields to cast to floating point numbers": "逗号分隔的元数据字段列表，将其转换为浮点数",
    "Comma-separated list of the sample metadata field names": "逗号分隔的样本元数据字段名称列表",
    "Comma-separated list of the observation metadata field names": "逗号分隔的观测元数据字段名称列表",
    "Write the output file in JSON format (biom1)": "将输出文件以JSON格式写入（biom1）",
    "File with abundance values for community": "社区丰度值文件",
    "Group name column": "组名列",
    "Select Sample count columns": "选择样本计数列",
    "Input has a header line": "输入包含一个标题行",
    "Diversity index to compute": "多样性指数计算",
    "Select reads": "选择读取",
    "Filter out chimeric clusters": "过滤掉嵌合体簇",
    "Minimum length of common sequence shared between a chimeric read and each of its parents": "嵌合体读段与其每个亲本之间共有序列的最小长度",
    "Abundance cutoff": "丰度截止",
    "Abundance ratio between a parent read and a chimeric read": "亲本读取与嵌合体读取之间的丰度比率",
    "Dissimilarity control for chimeric filtering": "嵌合体过滤的不相似性控制",
    "Length of prefix to be used in the analysis": "在分析中使用的前缀长度",
    "Match length": "匹配长度",
    "Maximum number/percent of mismatches allowed": "允许的最大不匹配数/百分比",
    "Description length": "描述长度",
    "Input sequences file": "输入序列文件",
    "Do you want to add an extra column?": "您想要添加一个额外的列吗？",
    "Do you want to remove column containing the sequence definition?": "您是否要移除包含序列定义的列？",
    "filter attributes from tabular file": "从表格文件中筛选属性",
    "Category attribute": "类别属性",
    "How would you specify the category attribute key?": "你如何指定类别属性键？",
    "Regular expression pattern matched against the attributes of the sequence record. the value of this attribute is of the form : key:regular_pattern. The pattern is case sensitive.": "正则表达式模式与序列记录的属性进行匹配。该属性的值具有以下形式：key:regular_pattern。该模式是区分大小写的。",
    "Attribute used to categorize the sequence records": "用于对序列记录进行分类的属性",
    "Use a specific option": "使用特定选项",
    "Select your specific option": "选择您的特定选项",
    "Attribute to merge": "要合并的属性",
    "Sort in reverse order?": "是否需要按照相反的顺序进行排序？",
    "ecoPCR taxonomy database": "ecoPCR 物种分类数据库",
    "NCBI taxonomy dump repository": "NCBI 分类学转储库",
    "Do you want to create an ecoPCR database from sequence records results?": "您是否希望从序列记录结果创建一个ecoPCR数据库？",
    "Do you want to print sequences in upper case?": "您想要将序列打印为大写吗？",
    "Maximum numbers of differences between two variant sequences (default: 1)": "两个变异序列之间最大的差异数目（默认值：1）",
    "Specify an attribute containing sample definition": "指定一个包含样本定义的属性",
    "Threshold ratio between counts (rare/abundant counts) of two sequence records so that the less abundant one is a variant of the more abundant (default: 1, i.e. all less abundant sequences are variants)": "两个序列记录的计数（稀有/丰富计数）之间的阈值比例，使得较少的序列是较多的序列的一个变体（默认值：1，即所有较少的序列都是变体）",
    "Do you want to select only sequences with the head status in a least one sample?": "您是否只想选择在至少一个样本中具有头部状态的序列？",
    "Use specific option": "使用特定选项",
    "Add a merged attribute containing the list of sequence record ids merged within this group": "添加一个合并属性，包含在此组内合并的序列记录ID列表",
    "Dereplicate through a prefix matching": "通过前缀匹配去重",
    "Choose the sequence record selection option": "选择序列记录选择选项",
    "Regular expression pattern to be tested against the sequence itself. The pattern is case insensitive.": "用于测试序列本身的正则表达式模式。该模式不区分大小写。",
    "Regular expression pattern to be tested against the definition of the sequence record. The pattern is case sensitive.": "要针对序列记录的定义进行测试的正则表达式模式。该模式是区分大小写的。",
    "Regular expression pattern to be tested against the identifier of the sequence record. The pattern is case sensitive.": "用于测试序列记录标识符的正则表达式模式。该模式区分大小写。",
    "points to a text file containing the list of sequence record identifiers to be selected. The file format consists in a single identifier per line.": "指向一个包含要选定的序列记录标识符列表的文本文件。该文件格式由每行一个标识符组成。",
    "Selects sequence records having an attribute who is key.": "选择具有属性键的序列记录。",
    "Python boolean expression to be evaluated for each sequence record.": "Python布尔表达式，用于评估每个序列记录。",
    "lmax": "最大角动量量子数",
    "lmin": "最小值",
    "Invert the sequence record selection (option -v)": "反转序列记录选择（选项 -v）",
    "Add a new attribute seq_rank": "添加一个新属性 seq_rank",
    "Rename attribute": "重命名属性",
    "Delete attribute": "删除属性",
    "Create a new attribute": "创建一个新的属性",
    "key": "关键",
    "python expression": "Python表达式",
    "Use a tag list": "使用标签列表",
    "Set sequence record identifier with a value": "设置序列记录标识符的值",
    "Run a PYTHON_EXPRESSION on each selected sequence": "对每个选定的序列运行一个PYTHON_EXPRESSION",
    "Change the sequence itself with a value": "将序列本身更改为一个值",
    "Set sequence definition with a value computed": "设置序列定义为计算得出的值",
    "Clear all attributes associated to the sequence records": "清除与序列记录相关的所有属性",
    "Keep only attribute with key": "只保留具有指定键的属性",
    "if you want to specify a second key": "如果你想指定第二个键",
    "if you want to specify a third key": "如果您想指定第三个键",
    "if you want to specify a fourth key": "如果你想指定一个第四个键",
    "if you want to specify a fifth key": "如果你想指定第五个键",
    "Use the length option?": "使用长度选项？",
    "Add taxonomic annotation": "添加分类注释",
    "mcl file": "MCL 文件",
    "Force sequence record ids to be unique": "强制序列记录ID唯一",
    "Parameter file": "参数文件",
    "Read from file": "从文件读取",
    "Number of errors allowed for matching primers (default = 2)": "允许匹配引物的错误数量（默认 = 2）",
    "Do you want to generate a file with unidentified sequences?": "您是否想要生成一个包含未识别序列的文件？",
    "minimum score for keeping aligment": "保持比对的最低分数",
    "Upload all data files wich are raster files": "上传所有栅格文件格式的数据文件",
    "Upload all data files wich are data table": "上传所有数据文件，这些文件是数据表。",
    "Under what license are your data?": "您的数据在什么许可下？",
    "What type of file do you want for templates?": "您想要什么类型的文件作为模板？",
    "Upload all EAL templates": "上传所有 EAL 模板",
    "Title for your dataset.": "你的数据集标题。",
    "Temporal coverage": "时间覆盖范围",
    "Beginning date of the dataset in the format YYYY-MM-DD": "数据集开始日期的格式为 YYYY-MM-DD",
    "Ending date of the dataset in the format YYYY-MM-DD": "数据集结束日期的格式为 YYYY-MM-DD",
    "Do you have data table ?": "你有数据表吗？",
    "Give a short description of your data file. If there is multiple data files please separate your descriptions with a comma (,) in the same order of your data input": "请为您的数据文件提供一个简短的描述。如果您有多个数据文件，请按照数据输入的顺序，用逗号（，）分隔您的描述。",
    "What's the quotes of your data table ? Repeat this parameter as many time as the number of data tables you've input and in the same order": "您的数据表使用什么引号？请根据您输入的数据表数量重复此参数，并保持相同的顺序。",
    "What's the quotes of your data table ?": "你的数据表使用的是什么引号？",
    "Give the publicly accessible URL from which your data table can be downloaded. If more than one, data files please separate your URLs with a comma (,) in the same order of your data input. If wanting to include URLs for some but not all then use a - for those that don't have a URL.": "提供可以公开下载您的数据表的URL。如果不止一个数据文件，请用逗号（,）按您数据输入的顺序分隔您的URL。如果您想为某些数据包括URL，但不是全部，则对于没有URL的，请使用一个短划线（-）。",
    "Do you have raster data ?": "你有栅格数据吗？",
    "Upload all data files wich are spatial raster": "上传所有空间栅格数据文件",
    "Give the publicly accessible URL from which your raster data entity can be downloaded. If more than one, data files please separate your URLs with a comma (,) in the same order of your data input. If wanting to include URLs for some but not all then use a - for those that don't have a URL.": "请提供您的栅格数据实体可以从中下载的公开可访问的URL。如果不止一个数据文件，请用逗号（,）按照您数据输入的顺序分隔您的URL。如果想要为某些但不是全部包含URL，则在那些没有URL的用一个短横线（-）表示。",
    "Do you have vector data ?": "你有向量数据吗？",
    "Upload all data files wich are spatial vector": "上传所有空间向量数据文件",
    "Give the publicly accessible URL from which your vector data entity can be downloaded. If more than one, data files please separate your URLs with a comma (,) in the same order of your data input. If wanting to include URLs for some but not all then use a - for those that don't have a URL.": "请提供可以公开下载您的向量数据实体的URL。如果有多个数据文件，请用逗号（,）按照您的数据输入顺序分隔您的URL。如果您想要为某些数据包含URL，但不是全部，则在没有URL的数据处使用一个短横线（-）。",
    "Do you have other data entity ?": "你有其他的数据实体吗？",
    "Upload all data files wich are other entity": "上传所有属于其他实体的数据文件",
    "Give the publicly accessible URL from which your other data entity can be downloaded. If more than one, data files please separate your URLs with a comma (,) in the same order of your data input. If wanting to include URLs for some but not all then use a - for those that don't have a URL.": "提供可公开访问的URL，以便从该URL下载您的其他数据实体。如果有多个数据文件，请用逗号（,）隔开您的URLs，顺序与您的数据输入一致。如果只想为某些数据包含URL，而不是全部，则在没有URL的数据处使用一个短横线（-）。",
    "EML metadata file": "EML 元数据文件",
    "Upload table containing geographic coordinates": "上传包含地理坐标的表格",
    "What column of your table contains latitude coordinates in decimal degrees?": "您的表格中哪一列包含以十进制度表示的纬度坐标？",
    "What column of your table contains longitude coordinates in decimal degrees ?": "您的表格中哪一列包含以十进制度表示的经度坐标？",
    "What column of your table contains site names ?": "你的表格中哪一列包含地点名称？",
    "Write an empty template file ?": "编写一个空模板文件？",
    "Upload table containing all unique taxa in the data": "上传包含数据中所有唯一分类群的表格",
    "What column of your table listing all unique taxa ?": "你的表格中列出所有独特分类群的列是哪列？",
    "What type of taxa name are in your table ?": "您表格中的分类群名称是什么类型的？",
    "Select the taxonomic authority in wich the taxa's name will be search (you can select multiple authority). If you select several authhority, please select in order of decreasing in preference.": "选择用于搜索分类群名称的分类权威机构（可以选择多个权威机构）。如果您选择了多个权威机构，请按照偏好程度递减的顺序选择。",
    "Select your first taxonomic authority in wich the taxa's name will be search. (required)": "选择您要搜索的第一个分类权威机构。（必填）",
    "Select if you want a second taxonomic authority in wich the taxa's name will be search, else select 'None'.": "选择是否需要第二个分类学权威，以搜索该分类群的名称，否则选择“无”。",
    "Select if you want a third taxonomic authority in wich the taxa's name will be search, else select 'None'.": "选择是否需要第三个分类权威机构，在该机构中搜索分类群的名称，否则选择“无”。",
    "Upload all data files wich are vector files": "上传所有矢量文件格式的数据文件",
    "Upload EAL attributes templates (with if exist catvars and custom units files)": "上传 EAL 属性模板（如果存在 catvars 和自定义单位文件）",
    "Upload all data files wich are raster and/or vector and/or other entities files": "上传所有数据文件，包括栅格和/或矢量以及/或其他实体文件",
    "Per-sample variant calling data": "每个样本的变异调用数据",
    "Source of lineage definitions": "谱系定义的来源",
    "Collection of lineage definitions": "谱系定义的集合",
    "Source of genome feature annotations": "基因组特征注释的来源",
    "AF threshold for identifying variants per sample": "每个样本识别变异的等位基因频率阈值",
    "Input alignment fasta file": "输入对齐的fasta文件",
    "The reference sequence ...": "参考序列",
    "ID of reference sequence": "参考序列的ID",
    "Restrict genome positions to be plotted": "限制要绘制的基因组位置",
    "Restrict plot to specific regions/positions": "限制绘图到特定区域/位置",
    "Include position/region": "包含位置/区域",
    "Exclude specific regions/positions from plot": "排除特定区域/位置不显示在图中",
    "Exclude position/region": "排除位置/区域",
    "Exclude ambig positions?": "排除模糊位置？",
    "Customize rendering of samples and mutations": "自定义样本和突变的渲染",
    "Provide custom labels for sequences?": "为序列提供自定义标签？",
    "Labels mapping source": "标签映射源",
    "Name of the sequence ID column": "序列ID列的名称",
    "Name of the column with desired labels": "所需标签的列名",
    "Mutations colouring scheme": "突变着色方案",
    "Sequence ID of first recombinant parent sequence in alignment": "在比对中第一个重组亲本序列的序列ID",
    "Sequence ID of second recombinant parent sequence in alignment": "在比对中第二个重组亲本序列的序列ID",
    "Order of samples in the plot": "图中样本的顺序",
    "Invert sort order": "反转排序顺序",
    "Sort on": "根据上下文，\"Sort on\" 可以翻译为 \"根据...排序\" 或 \"排序依据\"。",
    "Sort (alphabetically) by base at position": "按位置处的碱基按字母顺序排序",
    "Invert resulting sort order": "反转结果的排序顺序",
    "Flip plot orientation?": "翻转绘图方向？",
    "Plot output file format": "绘图输出文件格式",
    "Customize plot dimensions": "自定义绘图尺寸",
    "Plot sizing options": "绘图大小选项",
    "Try to plot indels": "尝试绘制插入缺失（indels）图。",
    "Write out SNPs?": "写出SNPs？",
    "Multiple alignment of viral sequences": "病毒序列的多重比对",
    "What kind of primers would you like to design? (varvamp mode)": "您想要设计哪种引物？（varvamp模式）",
    "Limit the number of amplicons to report?": "限制报告的扩增子数量？",
    "Report only the n highest scoring amplicons": "仅报告得分最高的n个扩增子",
    "Minimal required overlap between tiled amplicons": "最小所需的平铺扩增子之间的重叠",
    "Maximum number of ambiguous nucleotides per qPCR probe to be tolerated": "每个qPCR探针可容忍的最大模糊核苷酸数量",
    "Top n qPCR amplicons to test": "前n个 qPCR 扩增子以测试",
    "Minimum free energy (kcal/mol/K) cutoff": "最小自由能（千卡/摩尔/开尔文）截止值",
    "Constraints on qPCR probes and amplicons": "qPCR探针和扩增子的约束",
    "Minimal qPCR probe melting temperature": "最小 qPCR 探针熔解温度",
    "Maximal qPCR probe melting temperature": "最大 qPCR 探针熔解温度",
    "Optimal qPCR probe melting temperature": "最优 qPCR 探针熔解温度",
    "Minimal qPCR probe size": "最小 qPCR 探针大小",
    "Maximal qPCR probe size": "最大 qPCR 探针大小",
    "Optimal qPCR probe size": "最优 qPCR 探针大小",
    "Minimal qPCR probe GC-content": "最小 qPCR 探针 GC 含量",
    "Maximal qPCR probe GC-content": "最大 qPCR 探针 GC 含量",
    "Optimal qPCR probe GC-content": "最优 qPCR 探针 GC 含量",
    "Minimal number of GCs among the 3'-terminal 5 bases of the probe": "探针3'端5个碱基中GC的最小数目",
    "Maximal number of GCs among the 3'-terminal 5 bases of the probe": "探针3'端5个碱基中GC含量的最大值",
    "Maximal melting temperature difference among qPCR primers": "qPCR引物间最大熔解温度差异",
    "Minimal melting temperature difference between qPCR probe and primers": "qPCR探针与引物之间的最低熔解温度差异",
    "Maximal melting temperature difference between qPCR probe and primers": "qPCR探针与引物之间的最大熔解温度差异",
    "Minimal distance of the qPCR probe from the primer on the same strand": "qPCR探针与同链上的引物的最小距离",
    "Maximal distance of the qPCR probe from the primer on the same strand": "qPCR探针与同一链上的引物的最大距离",
    "End Overlap": "末端重叠",
    "Minimal length of qPCR amplicons": "qPCR扩增子的最小长度",
    "Maximal length of qPCR amplicons": "qPCR扩增子的最大长度",
    "Minimal GC-content of qPCR amplicons": "qPCR扩增子的最小GC含量",
    "Maximal GC-content of qPCR amplicons": "qPCR扩增子的最大GC含量",
    "Cutoff for deletions allowed in aligned regions under qPCR amplicons": "在 qPCR 扩增子下的对齐区域中允许的缺失的截止值",
    "Minimum quality score threshold to count base": "最小质量分数阈值以计数碱基",
    "Minimum frequency threshold": "最小频率阈值",
    "Minimum indel frequency threshold": "最小插入缺失频率阈值",
    "Minimum depth to call consensus": "最小深度以调用共识",
    "How to represent positions with coverage less than the minimum depth threshold": "如何表示覆盖度低于最小深度阈值的位置",
    "Bam input": "Bam 输入",
    "Variants input": "变异输入",
    "Primer binding sites information": "引物结合位点信息",
    "Compute amplicon info from BED file": "从BED文件计算扩增子信息",
    "Primer to amplicon assignment info": "引物到扩增子分配信息",
    "Source of primer information": "引物信息来源",
    "BED file with primer sequences and positions": "带有引物序列和位置的BED文件",
    "Primer scheme name": "引物方案名称",
    "Filter reads based on amplicon info": "根据扩增子信息过滤读取",
    "Wiggling room for read ends relative to primer binding sites": "相对于引物结合位点的读取末端的摆动空间",
    "Include reads not ending in any primer binding sites?": "包含不以任何引物结合位点结束的读取？",
    "Minimum quality threshold for sliding window to pass": "滑动窗口通过的最小质量阈值",
    "Width of sliding window": "滑动窗口宽度",
    "Require a minimum length for reads to retain them after any trimming?": "要求在任何修剪后保留读取的最小长度？",
    "Minimum trimmed length threshold": "最小修剪长度阈值",
    "Input filtered variants tsv generated from filtervariants": "输入由 filtervariants 生成的过滤变体 tsv 文件",
    "Primer pair information file containing left and right primer names for the same amplicon": "引物对信息文件，包含相同扩增子的左右引物名称",
    "In VCF only output variants that PASS all filters": "在 VCF 文件中，只输出通过所有过滤器的变异。",
    "Coordinates of ORFs for optional variant effect prediction": "可选变异效应预测的ORF坐标",
    "Variant tsv files for each replicate": "每个重复的变异tsv文件",
    "Trim trailing As from the reference sequence": "从参考序列中修剪尾部的As（腺嘌呤）",
    "Sequencing technology": "测序技术",
    "Shortest match required to keep a read (in bp)": "最短匹配要求以保持读取（以碱基对为单位）",
    "Minimum length of match required to keep a read (as percentage of read length": "保持读取所需的匹配最小长度（占读取长度的百分比）",
    "Rename the reads 1,2,3,... (for paired reads, will also add /1 or /2 to the end of names)": "将读段重命名为1、2、3等（对于成对的读段，还会在名称的末尾添加/1或/2）",
    "Dataset with input variant calls": "输入变异调用的数据集",
    "Set sample name": "设置样本名称",
    "Name of the sample": "样本名称",
    "Depth cutoff for coverage estimate": "覆盖度估计的深度截止值",
    "Variants file": "变异文件",
    "Number of bootstraps": "引导次数",
    "Generate boxplot": "生成箱线图",
    "Aggregate demixed data or provide aggregated data file?": "聚合混合数据或提供聚合数据文件？",
    "Lineages abundances summary file(s)": "谱系丰度汇总文件",
    "Provide aggregated data": "提供聚合数据",
    "Report(s) to generate": "生成报告",
    "Provide a sample metadata file for plotting data over time?": "提供一个用于随时间绘制数据的样本元数据文件？",
    "Provide time(s) metadata file": "提供时间（s）元数据文件",
    "Choose interval": "选择区间",
    "Provide sample(s) metadata file": "提供样本元数据文件",
    "Introduction": "介绍",
    "Provide sample(s)/time(s) metadata file": "提供样本/时间元数据文件",
    "Aligned reads input to analyze": "对齐的读取输入以进行分析",
    "How would you like to provide amplicon and lineage information?": "您希望如何提供扩增子和谱系信息？",
    "Format of amplicon definitions": "扩增子定义的格式",
    "Amplicon insert scheme": "扩增子插入方案",
    "Primer scheme": "引物方案",
    "Primers to amplicons assignments": "引物到扩增子分配",
    "Minimum number of mutations required to co-occur on an amplicon": "在扩增子上共同发生的最小突变数量",
    "Produce combined per-amplicon lineage definition output": "产生每个扩增子谱系定义的组合输出",
    "Combined per-amplicon lineage definitions": "合并每个扩增子的谱系定义",
    "Produce output in these formats": "生成这些格式的输出",
    "Co-occurrence results generated by mutbamscan": "由mutbamscan生成的共现结果",
    "Available amplicon and lineage annotations": "可用的扩增子和谱系注释",
    "Combined cojac per-amplicon lineage definitions": "结合cojac每个扩增子谱系定义",
    "Output formatting": "输出格式化",
    "Type of output to produce": "产生的输出类型",
    "Results generated by mutbamscan": "由mutbamscan生成的结果",
    "Output table orientation": "输出表格方向",
    "A orthology-pairs / RBH file": "一个 同源对 / RBH 文件",
    "(optional) A second orthology-pairs / RBH file": "（可选）第二个同源对/RBH文件",
    "Select the input fasta files": "选择输入的fasta文件",
    "Query type": "查询类型",
    "A string of the protein/gene name/identifier that you want to search": "一个您想要搜索的蛋白质/基因名称/标识符的字符串",
    "A orthology-groups file": "一个同源群文件",
    "Enable regular expressions (perl)": "启用正则表达式（Perl）",
    "Add the filename to the gene/protein-name [--source]": "将文件名添加到基因/蛋白质名称上 [--source]",
    "Select the input fasta files (>2)": "选择输入的fasta文件（>2）",
    "Similarity comparision algorithm": "相似性比较算法",
    "Minimal reciprocal similarity in %": "最小互反相似度（%）",
    "Minimal algebraic connectivity": "最小代数连通性",
    "E-value threshold of the blast algorithm": "BLAST算法的E值阈值",
    "Minimal coverage of best blast alignments in %": "最佳BLAST比对的最小覆盖率（%）",
    "Minimal percent identity of best blast hits in %": "最佳BLAST命中的最小百分比相似性（以百分比表示）",
    "Apply selfblast, detects paralogs without orthologs": "应用自比对，检测无直系同源基因的旁系同源基因",
    "Report singleton genes without any hit": "报告没有命中的单例基因",
    "Stop clustering if a split would result in groups that do not span across all species of the inital connected component.": "如果一个分裂会导致群组不能跨越初始连通分量中的所有物种，则停止聚类。",
    "Use isoform information": "使用异构体信息",
    "Activate synteny feature (POFF)": "激活同线性特征（POFF）",
    "Number of reiterations for adjacencies heuristic, to determine duplicated regions": "邻接启发式重复区域确定的重新迭代次数",
    "Size of a maximum common substring (MCS) for adjacency matches": "最大公共子串（MCS）的尺寸用于邻接匹配",
    "Weight of adjacencies vs. sequence similarity": "邻接权重与序列相似性",
    "Select the GFF3 files matching the input fasta files": "选择与输入的fasta文件匹配的GFF3文件",
    "Spectral library": "光谱库",
    "Generate a spectral library": "生成一个光谱库",
    "Perform deep learning-based prediction of spectra, retention times and ion mobility values": "进行基于深度学习的光谱、保留时间和离子迁移率值预测。",
    "Library headers": "库头文件",
    "Use the input library 'as is'": "使用输入库“原样”",
    "Training library": "训练库",
    "Save raw empirical retention times in the spectral library": "在光谱库中保存原始的经验保留时间",
    "Reannotate the spectral library with protein information from the FASTA database": "使用FASTA数据库中的蛋白质信息重新注释谱图库",
    "FASTA database": "FASTA 数据库",
    "Fasta filter": "Fasta过滤器",
    "Perform in silico digest of the sequence database": "在计算机上对序列数据库进行消化",
    "Disable mass calibration": "禁用质量校准",
    "MS2 mass accuracy": "MS2 质量精度",
    "Mass accuracy during calibration phase": "校准阶段的质量精度",
    "MS1 mass accuracy": "MS1 质量精度",
    "Enable MBR": "启用 MBR",
    "Use the same settings for all runs during the second MBR pass": "在第二次MBR传递期间，对所有运行使用相同的设置",
    "Use a very heuristical protein inference algorithm": "使用一种非常启发式的蛋白质推断算法",
    "Protein inference": "蛋白质推断",
    "Neural network classifier": "神经网络分类器",
    "Quantification strategy": "定量策略",
    "Cross-run normalisation": "跨运行标准化",
    "Library generationm strategy": "文库制备策略",
    "Ion mobility window": "离子迁移窗口",
    "Disable IM-windowed search": "禁用即时消息窗口搜索",
    "Scan window radius": "扫描窗口半径",
    "IM extraction window": "互信息提取窗口",
    "IM extraction window factor": "IM提取窗口因子",
    "Precursor ion generation": "前体离子生成",
    "Cleavage specificity": "切割特异性",
    "Protease": "蛋白酶",
    "Cleavage pattern": "切割模式",
    "Maximum number of missed cleavages": "最大未切割数",
    "Enable protein N-term methionine excision": "启用蛋白质N端甲硫氨酸切除",
    "Minimum precursor length": "最小前体长度",
    "Maximum precursor length": "最大前体长度",
    "Minimum precursor m/z": "最小前体 m/z",
    "Maximum precursor m/z": "最大前体 m/z",
    "Minimum precursor charge": "最小前体电荷",
    "Maximum precursor charge": "最大前体电荷",
    "Minimum number of fragments per precursors": "每个前体的最小片段数",
    "Maximum fragments per precursor": "最大前体片段数",
    "Use one precursor per stripped sequence for training": "使用每个剥离序列的一个前体进行训练",
    "Interference removal": "干扰去除",
    "Modifications": "修改",
    "Fixed modification": "固定修饰",
    "Variable modification": "变量修饰",
    "Maximum number of variable modifications": "最大变量修饰数量",
    "Modification name": "修改名称",
    "Modification for PTM scoring and site localisation": "修饰用于PTM（翻译后修饰）评分和位点定位",
    "No cut after modification": "修改后不切割",
    "Disable automatic conversion of known modifications": "禁用已知修饰的自动转换",
    "Clear modifications": "清除修饰",
    "In silico modification": "在硅基修饰",
    "Only consider peptides bearing the modifications listed": "仅考虑带有列出的修饰的肽段",
    "Ignore modifications not supported by the deep learning predictor": "忽略深度学习预测器不支持的修饰",
    "Considers C carbamidomethylation": "考虑C端羰甲基化",
    "Considers Ox(M) methionine oxidation": "考虑Ox(M)甲硫氨酸氧化",
    "Mass calibration and Fragmentation": "质量校准和碎片化",
    "Disable selection of fragments for quantification": "禁用片段选择进行定量分析",
    "Number of top fragment ions for quantification": "用于定量的顶部片段离子数量",
    "Restrict fragments for quantification": "限制用于定量的片段",
    "Fragment filtering mass accuracy": "片段过滤质量精度",
    "Number of fragment ions for spectral library": "用于谱图库的片段离子数",
    "Decoy channel": "诱饵通道",
    "Disable the use of a decoy channel": "禁用使用诱饵通道",
    "Quantification": "定量分析",
    "Take advantage of the co-elution of isotopologues": "利用同位素分子的共洗脱优势",
    "Disable MaxLFQ for protein quantification": "禁用 MaxLFQ 用于蛋白质定量",
    "Minimum peak height": "最小峰高",
    "Disable RT-windowed search": "禁用 RT-windows 搜索",
    "Disable precursor stratification": "禁用前体分层",
    "Do not give preference for SwissProt proteins": "不要偏好SwissProt蛋白质",
    "Disable ion mobility prediction": "禁用离子迁移率预测",
    "Disable retention time prediction": "禁用保留时间预测",
    "Duplicate proteins": "重复蛋白质",
    "Disable approximate FDR estimation": "禁用近似FDR估计",
    "Force SwissProt sequences": "强制使用SwissProt序列",
    "Full UniMod modification database": "完整的UniMod修饰数据库",
    "Generate fragment exclusion information": "生成片段排除信息",
    "Disable RT-dependent mass calibration": "禁用依赖于保留时间的质荷比校准",
    "Independent mass accuracies": "独立质量精度",
    "Individual output reports": "单独输出报告",
    "Independent scan windows": "独立扫描窗口",
    "Do not extract chromatograms for heavy isotopologues": "不要为重同位素异构体提取色谱图",
    "Analyse all runs as regular runs": "分析所有运行作为常规运行",
    "Analyse all runs as Scanning SWATH runs": "分析所有运行作为扫描SWATH运行",
    "Add the organism identifier to the gene names": "在基因名称中添加生物体标识符",
    "Ignore errors when loading dia-PASEF data": "忽略加载dia-PASEF数据时的错误",
    "Do not produce the main report": "不要生成主报告",
    "Disable generation of the stats file": "禁用生成统计文件",
    "Compact report": "紧凑报告",
    "Output quantities matrices": "输出数量矩阵",
    "Channel q-value": "通道q值",
    "Translated q-value": "翻译后的q值",
    "Run-specific protein q-value filtering": "特定运行蛋白质q值过滤",
    "Add extra library information to the main output report": "在主输出报告中添加额外的库信息",
    "Extract and save chromatograms": "提取并保存色谱图",
    "Verbose level": "详细级别",
    "PepQuery Set": "PepQuery 集合",
    "Validation Task Type": "验证任务类型",
    "Identity the decoy version of the selected target protein": "识别所选目标蛋白的诱饵版本",
    "Input Data": "输入数据",
    "Peptides?": "肽段？",
    "Peptide Sequences (.txt)": "肽序列（.txt）",
    "Proteins?": "蛋白质？",
    "Protein Sequences (.txt)": "蛋白质序列（.txt）",
    "Protein Sequence": "蛋白质序列",
    "Protein Identifier": "蛋白质标识符",
    "DNA Sequence (at least 60 bp)": "DNA序列（至少60个碱基对）",
    "Frame(s) for DNA translation": "DNA翻译的框架",
    "Protein Reference Database from": "蛋白质参考数据库",
    "Protein Reference Database File": "蛋白质参考数据库文件",
    "Public protein sequence database": "公共蛋白质序列数据库",
    "MS/MS dataset to search": "MS/MS 数据集搜索",
    "Spectrum File": "谱图文件",
    "PepQuery Index": "PepQuery 索引",
    "PepQueryDB dataset": "PepQueryDB 数据集",
    "Public dataset": "公共数据集",
    "Report Spectrum Scan as": "报告谱扫描为",
    "MS/MS searching parameter set name": "MS/MS搜索参数集名称",
    "Fixed modification(s)": "固定修饰（固定修饰）",
    "Variable modification(s)": "变量修饰（修改）",
    "Max Modifications": "最大修饰数",
    "Use more stringent criterion for unrestricted modification searching": "使用更严格的标准进行无限制修饰搜索",
    "Consider amino acid substitution modifications?": "考虑氨基酸替代修饰？",
    "Digestion": "消化",
    "Max Missed Cleavages": "最大缺失切割数",
    "Mass spectrometer": "质谱仪",
    "Precursor Tolerance": "前体耐受性",
    "Precursor Unit": "前体单元",
    "PSM": "PSM（Post-Source-Decay）是一种质谱分析方法，用于鉴定和表征蛋白质和多肽。",
    "Fragmentation Method": "片段化方法",
    "Scoring Method": "评分方法",
    "Add extra score validation": "增加额外的分数验证",
    "Minimum Charge": "最小电荷",
    "Maximum Charge": "最大电荷",
    "Minimum Peaks": "最小峰值",
    "Isotope peak error range": "同位素峰误差范围",
    "Minimum Score": "最小分数",
    "Minimum length of peptide": "最小肽长",
    "Maximum length of peptide": "多肽的最大长度",
    "Number of Random Peptides": "随机肽段数量",
    "Use fast mode for searching": "使用快速模式进行搜索",
    "Input Options": "输入选项",
    "choose the type of your input files": "选择您的输入文件类型",
    "FASTA files": "FASTA文件",
    "identifier parse rule": "标识符解析规则",
    "description parse rule": "描述解析规则",
    "Search Options": "搜索选项",
    "Specify an experimental design template (if needed). For detailed instructions see the help text.": "指定一个实验设计模板（如果需要的话）。有关详细说明，请查看帮助文本。",
    "minimum peptide length": "最小肽段长度",
    "maximum peptide mass [Da]": "最大肽质量 [道尔顿]",
    "minimum unique peptides": "最小独特肽段数",
    "Calculate peak properties": "计算峰值属性",
    "Match between runs": "匹配运行之间的序列",
    "Match Time Window [min]": "匹配时间窗口 [分钟]",
    "Match Ion Mobility Window": "匹配离子迁移率窗口",
    "Alignment Time Window [min]": "对齐时间窗口 [分钟]",
    "Alignment Ion Mobility": "对齐 离子迁移率",
    "Match unidentified Features": "匹配未识别特征",
    "Include contaminants": "包含污染物",
    "Decoy mode": "诱饵模式",
    "PSM FDR": "PSM FDR",
    "Protein FDR": "蛋白质FDR",
    "Minimum peptide length for unspecific searches": "非特异性搜索的最小肽段长度",
    "Maximum peptide length for unspecific searches": "非特异性搜索中最大肽段长度",
    "Protein quantification": "蛋白质定量",
    "Peptides for quantification": "用于定量的肽段",
    "Use only unmodified peptides": "仅使用未修改的肽段",
    "Modifications used in protein quantification": "蛋白质定量中使用的修饰",
    "Discard unmodified counterpart peptides": "丢弃未修改的对应肽",
    "LFQ Options": "LFQ选项",
    "Separate LFQ in parameter Groups": "将LFQ（标签自由定量）在参数组中分离",
    "Stabilize large LFQ ratios": "稳定大型LFQ比率",
    "Require MS/MS for LFQ comparisons": "需要质谱/质谱（MS/MS）进行标签自由定量（LFQ）比较",
    "iBAQ (calculates absolute protein abundances by normalizing to copy number and not protein mass)": "iBAQ（通过将蛋白质丰度标准化为拷贝数而不是蛋白质质量来计算绝对蛋白质丰度）",
    "Logarithmic fit": "对数拟合",
    "Advanced site intensities": "高级位点强度",
    "Parameter Group": "参数组",
    "Infiles": "输入文件",
    "missed cleavages": "缺失切割",
    "fixed modifications": "固定修饰",
    "variable modifications": "可变修饰",
    "enzyme": "酶",
    "Digestion mode": "消化模式",
    "Quantitation Methods": "定量方法",
    "light labels": "轻标签",
    "medium labels": "中等标签",
    "heavy labels": "重标签",
    "LFQ minimum ratio count": "LFQ 最小比率计数",
    "LFQ minimum number of neighbours": "LFQ 最小邻居数",
    "LFQ average number of neighbours": "LFQ 平均邻居数",
    "Normalization type": "标准化类型",
    "Only create config files? (i.e. for testing)": "只创建配置文件？（例如，用于测试）",
    "Select the desired outputs.": "选择所需的输出。",
    "RAW Files": "原始文件",
    "mzXML Files": "mzXML 文件",
    "mzML Files": "mzML 文件",
    "mqpar.xml file with your search parameters. RAW file names must match the names displayed in galaxy. Their paths from the local machine are ignored. E.g. a file named 'test01.raw' in galaxy can either be named 'test01.raw' or 'D:\\path\\to\\test01.raw' in the mqpar.xml.": "mqpar.xml文件包含您的搜索参数。RAW文件名必须与Galaxy中显示的名称匹配。它们在本地机器上的路径将被忽略。例如，在Galaxy中名为'test01.raw'的文件，其在mqpar.xml中可以命名为'test01.raw'或者'D:\\path\\to\\test01.raw'。",
    "Input Source": "输入源",
    "MSstatsTMT (11 column format)": "MSstatsTMT（11列格式）",
    "evidence.txt - feature-level data": "证据.txt - 特征级数据",
    "proteinGroups.txt": "蛋白质群.txt",
    "annotation.txt": "注释.txt",
    "Select Protein ID in evidence.txt": "在evidence.txt中选择蛋白质ID",
    "MaxQtoMSstatsTMTFormat Options": "MaxQtoMSstatsTMTFormat 参数选项",
    "Remove proteins only identified by site": "移除仅通过位点鉴定的蛋白质",
    "Summary for multiple rows": "多行摘要",
    "OpenMS input": "OpenMS 输入",
    "OpenMStoMSstatsTMTFormat Options": "OpenMS 到 MSstats TMT 格式选项",
    "Summary for multiple PSMs": "多肽段匹配（PSMs）的总结",
    "PSM output": "PSM输出",
    "annotation": "注释",
    "Select Protein ID": "选择蛋白质ID",
    "PDtoMSstatsTMTFormat Options": "PDtoMSstatsTMTFormat 选项",
    "Remove shared peptides by information of # Proteins column in PSM sheet.": "根据PSM表格中的“蛋白质数量”列的信息，移除共享肽段。",
    "Summarize peptides into proteins": "将肽段汇总到蛋白质中",
    "Select method": "选择方法",
    "Global median normalization": "全局中位数标准化",
    "Reference channel based normalization": "参考通道基于标准化",
    "Remove ’Norm’ channels from protein level data.": "从蛋白质水平数据中移除“Norm”通道。",
    "Remove ’Empty’ channels from protein level data": "从蛋白质水平数据中移除“空”通道",
    "MBimpute": "MBimpute",
    "Maximum quantile for deciding censored missing value": "决定被审查缺失值的最大分位数",
    "Plot Output Options": "绘图输出选项",
    "Select protein IDs to draw plots": "选择蛋白质ID以绘制图表",
    "List of proteins": "蛋白质列表",
    "Width of the saved pdf file": "保存的PDF文件的宽度",
    "Height of the saved pdf file": "保存的PDF文件的高度",
    "Upper limit for y-axis in the log scale": "对数尺度下y轴的上限",
    "Lower limit for y-axis in the log scale": "对数尺度下y轴的下限",
    "Size of x-axis labeling": "x轴标签大小",
    "Size of y-axis labeling": "y轴标签大小",
    "Size of labels represented each condition at the top": "标签的大小代表顶部每种条件",
    "Angle of labels represented each condition at the top": "标签的角度表示顶部每个条件",
    "Size of legend above Profile plot": "在轮廓图上方的图例大小",
    "Size of dots in Profile plot": "轮廓图中点的大小",
    "Number of columns for legends at the top of plot": "图表顶部图例的列数",
    "Draw original profile plots without normalization": "绘制原始的轮廓图，不进行标准化处理",
    "Draw profile plots with protein summarization for each channel and MS run": "绘制每个通道和质谱运行的蛋白质汇总的概况图。",
    "Compare Groups": "比较组别",
    "Use comparison matrix?": "使用比较矩阵？",
    "Comparison Matrix": "比较矩阵",
    "Moderate t statistic": "中等t统计量",
    "Adjusted p value method for multiple comparison": "多重比较的校正P值方法",
    "Remove ’Norm’ channels from protein level data": "从蛋白质水平数据中移除'Norm'通道",
    "Comparison Plot Options": "比较图选项",
    "FDR cutoff for the adjusted p-values in heatmap and volcano plot": "在热图和火山图中用于调整后的p值的FDR截止值",
    "Involve fold change cutoff or not for volcano plot or heatmap.": "是否在火山图或热图分析中考虑对数变化阈值。",
    "For volcano plot or heatmap, logarithm transformation of adjusted p-valuewith base 2 or 10": "对于火山图或热图，将校正后的 p 值进行以 2 或 10 为底的对数转换。",
    "For all three plots, upper limit for y-axis.": "对于所有三个图表，y轴的上限。",
    "For all tree plots, lower limit for y-axis in the log scale": "对于所有树状图，对数尺度下y轴的下限",
    "For Volcano plot, the limit for x-axis": "对于火山图，x轴的限制",
    "Size of axes labels for Residual and QQ Plots": "残差图和QQ图的轴标签大小",
    "Size of dots in residual plots, QQPlots, volcano plot and comparison plot.": "残差图中的点大小，QQ图，火山图和比较图中的点大小。",
    "Size  of Protein Name label in the graph for Volcano Plot.": "火山图蛋白质名称标签的大小。",
    "Angle of x-axis labels represented each comparison at the bottom of graph incomparison plot.": "在比较图中，x轴标签代表每个比较，位于图形底部。",
    "Size of legend for color at the bottom of volcano plot.": "火山图底部颜色图例的大小。",
    "Display protein names in Volcano Plot.": "在火山图中显示蛋白质名称。",
    "Show colour key": "显示颜色键",
    "Number of proteins which will be presented in each heatmap.": "每个热图中将展示的蛋白质数量。",
    "Determines how to order proteins and comparisons. Hierarchical cluster analysis with Ward method(minimum variance) is performed.": "确定如何对蛋白质和比较进行排序。使用Ward方法（最小方差）进行层次聚类分析。",
    "Select comparisons to draw plots": "选择比较以绘制图表",
    "List of comparisons": "比较列表",
    "Select Outputs": "选择输出",
    "input source": "输入源",
    "MSstats 10-column input": "MSstats 10列输入",
    "proteinGroups.txt - protein-level data": "蛋白质组.txt - 蛋白质水平数据",
    "annotation file": "注释文件",
    "MaxQtoMSstatsFormat Options": "MaxQtoMSstatsFormat 选项",
    "Remove the peptides including 'M' sequence": "移除包含'M'序列的肽段",
    "Remove the peptides including Oxidized 'M' sequence": "移除包含氧化型“M”序列的肽段",
    "OpenMS input (e.g. output of MSstatsConverter)": "OpenMS 输入（例如 MSstatsConverter 的输出）",
    "If annotation is not yet complete in OpenMS, use annotation with Raw.file, Condition (the name of the condition is not allowed to start with a number or contain any special characters), BioReplicate, and Runinformation": "如果OpenMS中的注释尚未完成，请使用Raw.file、Condition（条件名称不允许以数字开头或包含任何特殊字符）、BioReplicate和Runinformation进行注释。",
    "OpenMStoMSstatsFormat Options": "OpenMS 转 MSstats 格式选项",
    "Remove the proteins which have only 1 peptide and charge": "删除只有1个肽段和电荷的蛋白质。",
    "OpenSWATH_input": "OpenSWATH输入",
    "OpenSWATHtoMSstatsFormat Options": "OpenSWATHtoMSstatsFormat 选项",
    "m_score cutoff": "m_score 截断值",
    "Skyline input": "Skyline 输入",
    "SkylinetoMSstatsFormat Options": "SkylinetoMSstatsFormat 选项",
    "Remove iRT": "移除 iRT",
    "Filter with Qvalue": "使用Q值过滤",
    "Cutoff for Detection QValue.": "检测Q值的截止值。",
    "Remove Oxidation M peptides": "移除氧化的M肽",
    "Remove proteins with 1 feature": "移除具有1个特征的蛋白质",
    "dataProcess Options": "数据处理选项",
    "logarithm transformation of intensities with base 2 or 10.": "对强度进行以2或10为底的对数转换。",
    "Normalization to remove systematic bias between MS runs": "去除质谱运行之间的系统偏差的标准化处理",
    "global standard peptide names": "全局标准肽段名称",
    "Feature Subset": "特征子集",
    "The number of top features for Feature Subset": "特征子集的顶级特征数量",
    "Remove features flagged with uninformative feature quality": "移除带有信息不足特征质量标记的特征",
    "Summary Method": "摘要方法",
    "Impute Missing Values": "填充缺失值",
    "Remove runs which have more than 50% missing values": "删除缺失值超过50%的运行。",
    "Account for heterogeneous variation among intensities from different features": "考虑不同特征之间强度的异质变异",
    "Censored intensity": "审查强度",
    "Maximum quantile for deciding censored missing values.": "决定截断缺失值的最大分位数。",
    "DataProcess Plot Options": "数据处理 绘图选项",
    "Select visualization outputs": "选择可视化输出",
    "Advanced visualization parameters": "高级可视化参数",
    "Feature name for Profile Plot": "特征名称用于轮廓图",
    "For all three plots, lower limit for y-axis in the log scale": "对于所有三个图表，对数刻度的 y 轴下限",
    "Scale for Condition Plot": "条件图的比例尺",
    "Interval for Condition Plot": "条件绘图区间",
    "Size of x-axis labeling for 'Run' in Profile Plot and QC Plot, and 'Condition' in Condition Plot": "在轮廓图和质量控制图中，“运行”的x轴标签大小，以及条件图中的“条件”标签大小。",
    "Size of labeling for feature names in normal QQPlots separately for each feature and size of labels represented each condition at the top of graph in Profile Plot and QC plot.": "在正常QQ图中，每个特征的标签大小以及在Profile图中每个条件的标签大小，以及在QC图中顶部表示每个条件的标签大小。",
    "Angle of labels represented each condition at the top of graph in Profile Plot and QC plot or x-axis labeling in Condition plot.": "标签角度表示图中每个条件的顶部在轮廓图和质量控制图中的表示，或条件图中的x轴标记。",
    "Size of  feature names in residual plots and feature legend (transition-level or peptide-level) above graph in Profile Plot.": "残差图中特征名称的大小和图上方轮廓图中特征图例（转换水平或肽水平）的大小。",
    "Size of dots in Condition plot": "条件图中点的大小",
    "Remove uninformative feature outlier in profile plots": "在轮廓图中移除无信息特征的异常值",
    "Comparison Visualization Options": "比较可视化选项",
    "Run tool in batch mode for -in": "批量运行工具，针对 -in 参数",
    "[idXML input only] Maximum allowed retention time deviation between identifications belonging to the same spectrum": "[idXML 输入专用] 属于同一谱图的鉴定之间允许的最大保留时间偏差",
    "[idXML input only] Maximum allowed precursor m/z deviation between identifications belonging to the same spectrum": "[idXML 输入专用] 属于同一谱图的鉴定之间允许的最大前体 m/z 偏差",
    "(only idXML) if set, mapping will be done based on exact matching of originating mzml file and spectrum_ref": "（仅限 idXML）如果设置，将基于原始 mzml 文件和 spectrum_ref 的精确匹配进行映射。",
    "Algorithm used for consensus scoring": "用于共识评分的算法",
    "Options for filtering peptide hits": "肽段命中的过滤选项",
    "The number of top hits in each ID run that are considered for consensus scoring ('0' for all hits)": "在每个ID运行中，用于共识评分的顶级命中数（'0'表示所有命中）",
    "For each peptide hit from an ID run, the fraction of other ID runs that must support that hit (otherwise it is removed)": "对于每个来自ID运行的肽段命中，必须有其他ID运行支持该命中的比例（否则将其移除）。",
    "Count empty ID runs": "计算空ID序列",
    "if set, keeps the original scores as user params": "如果设置，将原始分数作为用户参数保留",
    "PEPIons algorithm parameters": "PEPIons 算法参数",
    "Maximum difference between fragment masses (in Da) for fragments to be considered 'shared' between peptides": "片段质量之间最大差异（以道尔顿为单位），以便将片段视为肽段之间的“共享”",
    "The minimal number of 'shared' fragments (between two suggested peptides) that is necessary to evaluate the similarity based on shared peak count (SPC)": "最小共享片段数（两个建议的肽段之间）是评估基于共享峰计数（SPC）相似性所必需的",
    "PEPMatrix algorithm parameters": "PEPMatrix 算法参数",
    "Substitution matrix to use for alignment-based similarity scoring": "用于基于比对的相似性评分的替代矩阵",
    "Alignment gap penalty (the same value is used for gap opening and extension)": "对齐间隙惩罚（相同的值用于间隙开放和扩展）",
    "Overrides tool-specific checks": "覆盖特定工具的检查",
    "Enables the test mode (needed for internal use only)": "启用测试模式（仅供内部使用）",
    "Tabular file from metaQuantome stats or metaQuantome filter": "来自metaQuantome统计或metaQuantome过滤的表格文件",
    "Ontology": "本体论",
    "Make one plot for each of BiologicalProcess, CellularComponent, and MolecularFunction GO term categories.": "为生物过程（BiologicalProcess）、细胞组成（CellularComponent）和分子功能（MolecularFunction）这三个基因本体（GO）术语类别各制作一个图表。",
    "Taxonomic rank to restrict to in the plot": "在图表中限制的分类等级",
    "Mean intensity column name": "平均强度列名",
    "which distribution": "哪种分布",
    "Taxon id": "分类群ID",
    "Taxon name": "物种名称",
    "Functional term id": "功能术语ID",
    "Functional term name": "功能术语名称",
    "metaquantome expanded or filtered file": "MetaQuantome 扩展或过滤文件",
    "Name of the control sample group of control (used as denominator for fold change evaluation).": "用于作为对照组（作为倍数变化评估的分母）的对照样本组名称。",
    "Statistical test to perform": "要执行的统计测试",
    "Perform paired tests": "执行配对测试",
    "metaquantome expand file": "MetaQuantome 扩展文件",
    "min_peptides": "最小肽段数",
    "min_pep_nsamp": "最小肽段样本数",
    "min_children_non_leaf": "最小非叶子节点数",
    "min_child_nsamp": "最小子样本数",
    "qthreshold": "阈值",
    "Database Archive File": "数据库归档文件",
    "Intensity file": "强度文件",
    "Intensity file: peptide column name": "强度文件：肽段列名",
    "Databases to Download": "数据库下载",
    "Annotate seed orthologs": "注释种子直系同源基因",
    "MSI data": "微卫星不稳定性数据",
    "Centroided input": "中心化的输入",
    "Processed imzML file": "处理过的imzML文件",
    "Mass accuracy to which the m/z values will be binned": "m/z值将被分箱的质量精度",
    "Unit of the mass accuracy": "质量精度的单位",
    "Optional annotation of pixels with tabular files": "可选地使用表格文件对像素进行注释",
    "Pixel annotations tabular files": "像素注释表格文件",
    "Tabular files contain a header line": "表格文件包含一个标题行",
    "Way of combining multiple files": "合并多个文件的方法",
    "Number of empty pixels that should separate different datasets in x and y dimension": "在 x 和 y 维度上，应分隔不同数据集的空白像素数",
    "Datasetnames, x and y values to shift pixel coordinates before combining": "数据集名称，x 和 y 值，在组合前用于移动像素坐标",
    "Column with values for shift in x direction": "X方向位移的列",
    "Column with values for shift in y direction": "y方向位移的列",
    "Column with dataset names": "数据集名称列",
    "Set all file coordinates to 1/1 as origin": "将所有文件坐标设置为以1/1为原点",
    "Do the files have different m/z axis?": "这些文件的 m/z 轴是否不同？",
    "Output file type -- default: determined from file extension or content": "输出文件类型 -- 默认：根据文件扩展名或内容确定",
    "List of the MetaData parameters to be included in a feature set for precolato": "用于precolato的特征集应包含的元数据参数列表",
    "Combine PSMs from different search engines by merging on scan level": "将不同搜索引擎的PSM（Peptide Spectral Matches，肽段谱匹配）通过在扫描水平上合并来组合",
    "Manual override to skip the check if same settings for multiple search engines were applied": "手动覆盖，跳过检查是否对多个搜索引擎应用了相同的设置",
    "Naive merging of PSMs from different search engines: concatenate multiple search results instead of merging on scan level": "天真地合并来自不同搜索引擎的PSMs：将多个搜索结果串联起来，而不是在扫描级别上合并。",
    "Will instead of discarding all PSM not unanimously detected by all SE, impute missing values by their respective scores min/max observed": "将不会丢弃所有由所有搜索引擎（SE）未一致检测到的肽段谱匹配（PSM），而是通过它们各自的分数的最小值/最大值来估算缺失值。",
    "Will impute missing scores with the worst numerical limit (instead of min/max observed) of the respective score": "将缺失的分数用相应分数的最差数值限制（而不是最小/最大观测值）进行插补。",
    "Protein Database": "蛋白质数据库",
    "Input Peak Lists (mgf)": "输入峰值列表（mgf格式）",
    "Maximum Missed Cleavages": "最大未切割位点",
    "Precursor Options": "前体选项",
    "Precursor Ion Tolerance Units": "前体离子容差单位",
    "Percursor Ion Tolerance": "前体离子容忍度",
    "Fragment Tolerance (Daltons)": "片段容忍度（道尔顿）",
    "DB-Search Engines": "数据库搜索引擎",
    "Meta-protein generation": "蛋白质元组生成",
    "Iterative searching": "迭代搜索",
    "Fragmentation method": "片段化方法",
    "Peptide indexing (of FASTA database)": "肽段索引（FASTA数据库）",
    "FDR threshold for filtering": "FDR阈值用于过滤",
    "Sample file creation method": "样本文件创建方法",
    "File with group name headers": "带有组名标题的文件",
    "mztab or mztab2 data set": "mztab 或 mztab2 数据集",
    "Select desired tables": "选择所需的表格",
    "MZML file(s)": "MZML文件（们）",
    "OSW file": "OSW文件",
    "Maximum FDR query": "最大FDR查询",
    "Order of the polynomial to be fit in the kernel": "在核函数中拟合的多项式的阶数",
    "Number of data-points to consider in the kernel": "在核函数中考虑的数据点数量",
    "Global alignment": "全局比对",
    "Global alignment FDR": "全局比对FDR",
    "Global alignment Span": "全局比对 Span",
    "RSE distance": "RSE 距离",
    "SIM measure": "SIM 度量",
    "Gap penalty (first gap)": "间隔惩罚（第一个间隔）",
    "Gap penalty (subsequent gaps)": "间隔惩罚（后续间隔）",
    "cosAngleTresh": "余弦角度阈值",
    "Overlap alignment": "重叠比对",
    "dotProdThresh": "点积阈值",
    "Gap quantile": "间隙分位数",
    "Hard constrain": "硬约束",
    "Samples for gradient": "梯度样本",
    "Analyte FDR": "分析物错误发现率",
    "Unaligned FDR": "未比对的FDR",
    "Aligned FDR": "对齐的FDR",
    "Baseline method": "基线方法",
    "Integration method": "整合方法",
    "Fit EMG": "拟合肌电图（EMG）",
    "Recal intensity": "重新校准强度",
    "Fill missing": "填充缺失",
    "Smooth peak area": "平滑峰面积",
    "FASTG file": "FASTG文件",
    "Peptide Cleavage": "肽段裂解",
    "Minimum Protein Length in Amino Acids": "最小蛋白长度（氨基酸数）",
    "Minimum Peptide Length in Amino Acids": "最小肽长（氨基酸数）",
    "Create Diagnostic Plots": "创建诊断图",
    "MSGF+ tabular file": "MSGF+ 表格文件",
    "Database Name": "数据库名称",
    "FDR value for validation.": "验证的FDR值。",
    "Decoy protein header": "诱饵蛋白标题",
    "MzML Input file": "MzML 输入文件",
    "Digestion enzyme - currently only Trypsin is supported": "消化酶 - 目前仅支持胰蛋白酶",
    "MassAnalyze": "MassAnalyze",
    "Fragmentation error tolerance  (Da)": "片段化误差容忍度 (Da)",
    "Precursor error tolerance  (ppm or Da)": "前体离子误差容忍度（ppm或Da）",
    "Unit of precursor mass tolerance": "前体质量容差单位",
    "Variable modifications": "可变修饰",
    "Fixed modifications": "固定修饰",
    "Forbidden Resiudes": "禁止残基",
    "File to introduce customized algorithm parameters for advanced users (otional .novor file)": "为高级用户引入自定义算法参数的文件（可选的 .novor 文件）",
    "Input spectrum file": "输入谱图文件",
    "Protein/peptide identifications file": "蛋白质/肽段鉴定文件",
    "Tolerance of the peaks in the fragment spectrum": "片段谱中峰值的容忍度",
    "Unit of fragment mass tolerance": "片段质量容差单位",
    "Do not consider peaks below this value for matching fragment ions": "不考虑匹配片段离子的峰值低于此值",
    "List the amino acids to be searched for and their mass modifications, specified using UniMod (www.unimod.org) terms": "列出要搜索的氨基酸及其质量修饰，使用UniMod（www.unimod.org）术语进行指定",
    "List the types of neutral losses that you want to conside": "列出您想要考虑的中性损失类型",
    "How much to add to an amino acid to make it a decoy": "将氨基酸添加多少才能使其成为诱饵蛋白",
    "For handling the neutral loss from a decoy sequence": "为了处理来自诱饵序列的中性损失",
    "Do not consider PSMs with a charge state above this value": "不考虑电荷状态高于这个值的PSMs",
    "Restrict scoring to peptides with a length shorter than this value": "限制评分到长度短于该值的肽段",
    "Maximum number of permutations a sequence can have": "一个序列可能具有的最大排列数",
    "Minimum score a PSM needs to be considered for modeling": "用于建模的PSM所需的最低分数",
    "PSMs below this value will be discarded": "低于此值的PSMs将被丢弃",
    "The minimum number of PSMs you need for any charge state in order to build a model for it": "为了构建任何电荷状态下的模型，您所需的最小PSM数量",
    "Determines how Luciphor will run: 0 = calculate FLR then rerun scoring without decoys (two iterations), 1 = Report Decoys: calculate FLR but don't rescore PSMs, all decoy hits will be reported": "确定Luciphor的运行方式：0 = 计算FLR然后重新运行得分计算而不使用诱饵（两次迭代），1 = 报告诱饵：计算FLR但不要重新评分PSMs，将报告所有诱饵命中",
    "Set the retention time tolerance (for the mapping of identifications to spectra in case multiple search engines were used)": "设置保留时间容差（用于在多个搜索引擎使用的情况下，将鉴定结果映射到谱图）",
    "The Sirius executable": "天狼星可执行文件",
    "MzML input file(s) used for assay library generation": "用于生成分析库的 MzML 输入文件",
    "Run tool in batch mode for -in_id": "批量运行工具，针对 -in_id 参数",
    "FeatureXML input file(s) containing identification information": "特征XML输入文件（s），包含鉴定信息",
    "File type of output out (Assay library output file)": "输出文件类型（检测库输出文件）",
    "Fragment annotation method": "片段注释方法",
    "Mz tolerance for the resolution of identification ambiguity over multiple files": "m/z 容差，用于解决多个文件中鉴定歧义的分辨率",
    "RT tolerance in seconds for the resolution of identification ambiguity over multiple files": "RT公差（以秒为单位），用于解决多个文件中鉴定歧义的解析",
    "Filter compound based on total occurrence in analysed samples": "根据在分析样本中出现的总次数过滤化合物",
    "Filters annotations based on the explained intensity of the peaks in a spectrum": "根据质谱中峰值的解释强度过滤注释",
    "Decoys will be generated using the fragmentation tree re-rooting approach": "诱饵蛋白质将使用碎片树重新根植方法生成",
    "Uses different methods for decoy generation": "使用不同的方法进行诱饵生成",
    "Spectrum with the highest precursor intensity or a consensus spectrum is used for assay library construction (if no fragment annotation is used)": "使用具有最高前体强度的谱图或共识谱图进行检测库构建（如果不使用片段注释）",
    "Use exact mass for precursor and fragment annotations": "使用精确质量进行前体和片段注释",
    "Excludes precursor in ms2 from transition list": "从转换列表中排除ms2的前体",
    "Max m/z distance of the precursor entries of two spectra to be merged in [Da]": "两个谱图的前体条目合并时的最大 m/z 距离 [Da]",
    "Tolerance window (left and right) for precursor selection [seconds], for consensus spectrum generation (only available without fragment annotation)": "前体选择的容忍窗口（左侧和右侧）[秒]，用于共识谱生成（仅在没有片段注释的情况下可用）",
    "Use features without identification information": "使用不带识别信息的特征",
    "Minimal number of transitions": "最小转换数",
    "Maximal number of transitions": "最大转换数",
    "Threshold for cosine similarity of MS2 spectra from the same precursor used in consensus spectrum creation": "用于创建共识谱图的来自相同前体的MS2谱图的余弦相似度阈值",
    "Further transitions need at least x% of the maximum intensity (default 5%)": "进一步的转变需要至少最大强度的x%（默认为5%）",
    "deisotoping": "去同位素化",
    "Use Deisotoper (if no fragment annotation is used)": "使用Deisotoper（如果未使用片段注释）",
    "Tolerance used to match isotopic peaks": "用于匹配同位素峰的容差",
    "Unit of the fragment tolerance": "片段容忍度的单位",
    "The minimum charge considered": "考虑的最小电荷",
    "The maximum charge considered": "考虑的最大电荷",
    "The minimum number of isotopic peaks (at least 2) required for an isotopic cluste": "所需的同位素峰值的最小数量（至少2个）对于一个同位素簇",
    "The maximum number of isotopic peaks (at least 2) considered for an isotopic cluste": "考虑一个同位素簇时所考虑的最大同位素峰数量（至少2个）",
    "Only monoisotopic peaks of fragments with isotopic pattern are retained": "仅保留具有同位素模式的片段的单一同位素峰",
    "Annotate the charge to the peaks": "对峰值进行电荷注释",
    "Number of mass traces each feature has to have to be included": "每个特征必须包含的质量迹线数量",
    "Tolerance window for precursor selection (Feature selection in regard to the precursor)": "前体选择的容忍窗口（关于前体的特征选择）",
    "Unit of the precursor_mz_tolerance": "前体_mz容忍度的单位",
    "Tolerance window (left and right) for precursor selection [seconds]": "前体选择的容忍窗口（左侧和右侧）[秒]",
    "Number of iterations that should be performed to extract the C13 isotope pattern": "提取C13同位素模式应执行的迭代次数",
    "Uses the feature information from in_featureinfo to reduce the search space to MS2 associated with a feature": "使用来自 in_featureinfo 的特征信息，将搜索空间减少到与特征相关联的 MS2",
    "Use this flag if the masstrace information from a feature should be discarded and the isotope_pattern_iterations should be used instead": "如果需要丢弃来自特征的masstrace信息，并改用isotope_pattern_iterations，则使用此标志",
    "Number of cpu cores to use": "要使用的CPU核心数",
    "Ignore given molecular formula in internal .ms format, while processing": "忽略内部.ms格式中给定的分子公式，同时进行处理",
    "Suppress shell output": "抑制shell输出",
    "Maximum allowed mass deviation in ppm for decomposing masses [ppm]": "允许的最大质量偏差（ppm）用于分解质量 [ppm]",
    "Maximum allowed mass deviation in ppm for decomposing masses in MS2 [ppm].If not specified, the same value as for the MS1 is used": "在MS2中分解质量时允许的最大质量偏差（ppm）。如果没有指定，则使用与MS1相同的值。",
    "Time out in seconds per fragmentation tree computations": "每个片段树计算的超时时间（以秒为单位）",
    "Maximal computation time in seconds for a single compound": "单个化合物的最大计算时间（以秒为单位）",
    "Disable recalibration of input spectra": "禁用输入谱的重新校准",
    "Name of the configuration profile": "配置文件的名称",
    "Specify the neutral molecular formula of the measured compound to compute its tree or a list of candidate formulas the method should discriminate": "指定所测量化合物的中性分子公式以计算其树形图，或指定该方法应该区分的候选公式列表",
    "The iontype/adduct of the MS/MS data": "质谱/质谱数据的离子类型/加合物",
    "The number of formula candidates in the SIRIUS output": "SIRIUS输出中公式候选的数量",
    "Minimum number of candidates in the output for each ionization": "输出中每个电离态的候选基因最小数量",
    "Set the allowed elements for rare element detection": "设置稀有元素检测允许的元素",
    "Enforce elements for molecular formula determination": "强制元素用于分子公式的确定",
    "Disable isotope pattern score": "禁用同位素模式得分",
    "Disable molecular formula filte": "禁用分子公式过滤器",
    "the iontype/adduct of the MS/MS data": "质谱/质谱数据的离子类型/加成物",
    "Search formulas in the Union of the given databases db-name1,db-name2,db-name3": "在给定数据库 db-name1、db-name2、db-name3 的并集中搜索公式",
    "Number of molecular structure candidates in the output": "输出中分子结构候选物的数量",
    "Unit of the ambiguity_resolution_mz_tolerance": "歧义分辨率_mz公差单位",
    "Tolerance window for precursor selection (Annotation of precursor mz and intensity)": "前体选择的容忍窗口（前体mz和强度的注释）",
    "Unit of the precursor_mz_tolerance_annotation": "前体_mz容忍度注释的单位",
    "Minimal m/z of a fragment ion choosen as a transition": "作为过渡选择的片段离子的最小 m/z",
    "Maximal m/z of a fragment ion choosen as a transition": "选择作为转换的最大 m/z 的片段离子",
    "FeatureXML input with feature and adduct information": "FeatureXML 输入包含特征和加合物信息",
    "Output directory for SIRIUS project space": "SIRIUS项目空间的输出目录",
    "Use this flag in combination with the out_ms file to convert the input mzML and featureXML to a .ms file": "使用此标志与 out_ms 文件结合，将输入的 mzML 和 featureXML 转换为 .ms 文件",
    "Set to yes, if you have read and agreed to the MSFragger license terms": "如果您已阅读并同意MSFragger许可条款，请设置为“是”",
    "Maximum Java heap size (in MB)": "最大Java堆大小（以MB为单位）",
    "Input File with specta for MSFragge": "输入文件包含用于MSFragge的光谱数据",
    "Protein FASTA database file path": "蛋白质FASTA数据库文件路径",
    "Recalculate peptide to protein association using OpenMS": "使用 OpenMS 重新计算肽段到蛋白质的关联",
    "Search Tolerances": "搜索容差",
    "Lower precursor mass tolerance": "较低的前体质量容忍度",
    "Upper precursor mass tolerance": "上一代前体质量容差",
    "True precursor mass tolerance (window is +/- this value)": "真实前体质量容忍度（窗口值是正负这个值）",
    "Unit of precursor true tolerance": "前体真实容忍度的单位",
    "Fragment mass tolerance (window is +/- this value)": "片段质量容忍度（窗口值是 +/- 这个值）",
    "Isotope correction for MS/MS events triggered on isotopic peaks": "针对在同位素峰触发的MS/MS事件进行同位素校正",
    "In-Silico Digestion Parameters": "In-Silico Digestion 参数",
    "Name of the enzyme to be written to the pepXML file": "写入 pepXML 文件的酶名称",
    "Residues after which the enzyme cuts (specified as a string of amino acids)": "酶切割后残余的氨基酸序列（以氨基酸字符串的形式指定）",
    "Residues that the enzyme will not cut before": "酶不会在其之前的残基",
    "Number of enzyme termini (non-enzymatic (0), semi (1), fully (2)": "酶末端数量（非酶类（0），半酶类（1），全酶类（2））",
    "Allowed number of missed cleavages": "允许的未切割数",
    "Minimum length of peptides to be generated during in-silico digestion": "在计算机模拟消化过程中生成的肽段的最小长度",
    "Maximum length of peptides to be generated during in-silico digestion": "在计算机模拟消化过程中生成的肽段的最大长度",
    "Min mass of peptides to be generated (Da)": "生成肽段的最小质量（Da）",
    "Max mass of peptides to be generated (Da)": "生成肽的最大质量（道尔顿）",
    "Variable Modification Parameters": "变量修饰参数",
    "Specifies the trimming of a protein N-terminal methionine as a variable modification": "指定将蛋白质N末端甲硫氨酸的剪切作为可变修饰",
    "Masses for variable modifications": "可变修饰的质量",
    "Syntax Strings for variable modifications": "变量修饰的语法字符串",
    "Variable modifications in unimod syntax, is added to mass+syntax varmod list": "在Unimod语法中，变量修饰被添加到质量+语法的变量修饰列表中",
    "Enable common variable modifications (15.9949 M and 42.0106 [^)": "启用常见变量修改（15.9949 M 和 42.0106 [^",
    "Do not allow any one amino acid to be modified by multiple variable modifications": "不允许任何一个氨基酸被多个可变修饰所修改",
    "Maximum total number of variable modifications per peptide": "每个肽段最大可变修饰总数",
    "Maximum allowed number of modified variably modified peptides from each peptide sequence, (maximum of 65534)": "每个肽段序列允许的最大修改的可变修饰肽段数量，（最多65534）",
    "Spectrum Processing Parameters": "谱图处理参数",
    "Minimum number of peaks in experimental spectrum for matching": "实验谱图中用于匹配的最小峰值数量",
    "Pre-process experimental spectrum to only use top N peaks": "对实验谱图进行预处理，仅使用前N个峰值。",
    "Filters out all peaks in experimental spectrum less intense than this multiple of the base peak intensity": "过滤掉实验谱图中比基峰强度低这个倍数的所有峰",
    "Removes peaks in this m/z range prior to matching (minimum value)": "在匹配之前移除此 m/z 范围内的峰（最小值）",
    "Removes peaks in this m/z range prior to matching (maximum value)": "在匹配之前移除此 m/z 范围内的峰（最大值）",
    "Maximum charge state for theoretical fragments to match": "最大电荷状态，用于匹配理论片段",
    "Ignores precursor charge and uses charge state specified in precursor_charge range (parameters: spectrum:precursor_charge_min and spectrum:precursor_charge_max)": "忽略前体电荷，并使用在precursor_charge范围中指定的电荷状态（参数：spectrum:precursor_charge_min和spectrum:precursor_charge_max）",
    "Min charge of precursor charge range to conside": "前体电荷范围考虑的最小电荷",
    "Max charge of precursor charge range to conside": "前体电荷范围考虑的最大电荷",
    "Open Search Features": "开启搜索功能",
    "Track top N unmodified peptide results separately from main results internally for boosting features": "独立跟踪前N个未修改的肽段结果，以便在内部提升特征，与主要结果分开",
    "Ranks a zero-bin hit above all non-zero-bin hit if it has expectation less than this value": "如果期望值小于这个值，则将零桶命中排在所有非零桶命中之上",
    "Multiplies expect value of PSMs in the zero-bin during results ordering (set to less than 1 for boosting)": "在结果排序期间，将零位PSMs的期望值乘以（设置为小于1以增强）",
    "Inserts complementary ions corresponding to the top N most intense fragments in each experimental spectrum": "在每个实验谱图中插入与最强烈的N个片段相对应的互补离子",
    "Minimum number of matched peaks in PSM for inclusion in statistical modeling": "在统计建模中包含的PSM匹配峰值的最小数量",
    "Minimum number of matched peaks for PSM to be reported": "匹配峰值的最小数量，以报告PSM",
    "Reports top N PSMs per input spectrum": "报告每个输入谱图的前N个PSM（肽段匹配）",
    "Suppresses reporting of PSM if top hit has expectation greater than this threshold": "如果最佳命中的期望值大于这个阈值，则抑制报告PSM",
    "Include fragment ions mass-shifted by unknown modifications (recommended for open and mass offset searches) (0 for OFF, 1 for ON)": "包含未知修饰导致质量偏移的片段离子（推荐用于开放式和质量偏移搜索）（0表示关闭，1表示开启）",
    "Static Modification Parameters": "静态修饰参数",
    "Statically add mass in Da to C-terminal of peptide": "在肽的C端静态添加质量数（Da）",
    "Statically add mass in Da to N-terminal of peptide": "在肽的N端静态添加质量数（Da）",
    "Statically add mass in Da to C-terminal of protein": "在蛋白质的C端静态添加质量数（Da）",
    "Statically add mass in Da to N-terminal of protein": "在蛋白质的N端静态添加质量数（Da）",
    "Statically add mass to glycine": "静态地为甘氨酸添加质量",
    "Statically add mass to alanine": "静态地给丙氨酸添加质量",
    "Statically add mass to serine": "静态地给丝氨酸添加质量",
    "Statically add mass to proline": "静态地为脯氨酸添加质量",
    "Statically add mass to valine": "静态地向缬氨酸添加质量。",
    "Statically add mass to threonine": "静态地给苏氨酸添加质量",
    "Statically add mass to cysteine": "静态地向半胱氨酸添加质量",
    "Statically add mass to leucine": "静态地给亮氨酸添加质量",
    "Statically add mass to isoleucine": "静态地为异亮氨酸添加质量",
    "Statically add mass to asparagine": "静态地为天冬酰胺添加质量",
    "Statically add mass to aspartic_acid": "静态地给天冬氨酸增加质量",
    "Statically add mass to glutamine": "静态地为谷氨酰胺添加质量",
    "Statically add mass to lysine": "静态地给赖氨酸添加质量",
    "Statically add mass to glutamic_acid": "静态地为谷氨酸添加质量",
    "Statically add mass to methionine": "静态地为甲硫氨酸添加质量",
    "Statically add mass to histidine": "静态地向组氨酸添加质量",
    "Statically add mass to phenylalanine": "静态地为苯丙氨酸添加质量",
    "Statically add mass to arginine": "静态地给精氨酸添加质量",
    "Statically add mass to tyrosine": "静态地为酪氨酸添加质量",
    "Statically add mass to tryptophan": "静态地给色氨酸添加质量",
    "Fixed modifications in unimod syntax if specific mass is unknown": "如果特定质量未知，则使用Unimod语法固定修饰",
    "String that was appended (or prefixed - see 'decoy_string_position' flag below) to the accessions in the protein database to indicate decoy proteins": "在蛋白质数据库中，为了标记诱饵蛋白，会在访问号后面追加（或前缀，具体取决于下面的'decoy_string_position'标志）一个字符串。",
    "Is the 'decoy_string' prepended (prefix) or appended (suffix) to the protein accession": "“decoy_string”是添加到蛋白质访问号的前面（前缀）还是后面（后缀）？",
    "Action to take if NO peptide was assigned to a decoy protein (which indicates wrong database or decoy string): 'error' (exit with error, no output), 'warn' (exit with success, warning message), 'silent' (no action is taken, not even a warning)": "如果一个诱饵蛋白没有分配到任何肽段（这表明数据库或诱饵字符串有误）：'error'（退出并报错，没有输出），'warn'（成功退出，显示警告信息），'silent'（不采取任何行动，甚至不显示警告）",
    "If set, the protein sequences are stored as well": "如果设置，蛋白质序列也会被存储",
    "If set, the protein description is stored as well": "如果设置，蛋白质描述也会被存储",
    "If set, protein hits which are not referenced by any peptide are kept": "如果设置，未被任何肽段引用的蛋白质命中将被保留",
    "If peptide sequences cannot be matched to any protein: 1) raise an error; 2) warn (unmatched PepHits will miss target/decoy annotation with downstream problems); 3) remove the hit": "如果肽序列不能匹配到任何蛋白质：1) 提升一个错误；2) 发出警告（未匹配的PepHits将错过目标/诱饵注释，导致下游问题）；3) 删除该命中项",
    "Maximal number of ambiguous amino acids (AAAs) allowed when matching to a protein database with AAAs": "允许在与含有不明确氨基酸（AAAs）的蛋白质数据库匹配时，最多包含的不明确氨基酸（AAAs）的数量。",
    "Maximal number of mismatched (mm) amino acids allowed when matching to a protein database": "允许在与蛋白质数据库匹配时最多不匹配（mm）的氨基酸数量",
    "Treat the isobaric amino acids isoleucine ('I') and leucine ('L') as equivalent (indistinguishable)": "将同位素氨基酸异亮氨酸（'I'）和亮氨酸（'L'）视为等同的（不可区分的）",
    "Enzyme which determines valid cleavage sites -": "确定有效切割位点的酶",
    "Specificity of the enzyme": "酶的特异性",
    "Input file (MS-GF+ parameter '-s')": "输入文件（MS-GF+ 参数 '-s'）",
    "Protein sequence database (FASTA file; MS-GF+ parameter '-d')": "蛋白质序列数据库（FASTA文件；MS-GF+ 参数 '-d'）",
    "Precursor monoisotopic mass tolerance (MS-GF+ parameter '-t')": "前体单一同位素质量容差（MS-GF+ 参数 '-t'）",
    "Unit of precursor mass tolerance (MS-GF+ parameter '-t')": "前体质量容差单位（MS-GF+ 参数 '-t'）",
    "Range of allowed isotope peak errors (MS-GF+ parameter '-ti')": "允许的同位素峰误差范围（MS-GF+ 参数 '-ti'）",
    "Fragmentation method ('from_spectrum' relies on spectrum meta data and uses CID as fallback option; MS-GF+ parameter '-m')": "片段化方法（'from_spectrum' 依赖于谱图元数据，并使用 CID 作为备选方案；MS-GF+ 参数 '-m'）",
    "Instrument that generated the data ('low_res'/'high_res' refer to LCQ and LTQ instruments; MS-GF+ parameter '-inst')": "生成数据的仪器（'low_res'/'high_res' 指的是 LCQ 和 LTQ 仪器；MS-GF+ 参数 '-inst'）",
    "Enzyme used for digestion, or type of cleavage": "用于消化的酶，或切割类型",
    "Labeling or enrichment protocol used, if any (MS-GF+ parameter '-p')": "使用的标记或富集协议（如果有的话），在MS-GF+参数'-p'中指定",
    "Level of cleavage specificity required (MS-GF+ parameter '-ntt')": "所需的裂解特异性水平（MS-GF+ 参数 '-ntt'）",
    "Minimum precursor ion charge (only used for spectra without charge information; MS-GF+ parameter '-minCharge')": "最小前体离子电荷（仅用于没有电荷信息的谱图；MS-GF+参数'-minCharge'）",
    "Maximum precursor ion charge (only used for spectra without charge information; MS-GF+ parameter '-maxCharge')": "最大前体离子电荷（仅用于没有电荷信息的谱图；MS-GF+参数'-maxCharge'）",
    "Minimum peptide length to consider (MS-GF+ parameter '-minLength')": "考虑的最小肽段长度（MS-GF+ 参数 '-minLength'）",
    "Maximum peptide length to consider (MS-GF+ parameter '-maxLength')": "考虑的最大肽段长度（MS-GF+ 参数 '-maxLength'）",
    "Number of matches per spectrum to be reported (MS-GF+ parameter '-n')": "每个谱图要报告的匹配数（MS-GF+ 参数 '-n'）",
    "Output additional features (MS-GF+ parameter '-addFeatures')": "输出附加特征（MS-GF+ 参数 '-addFeatures'）",
    "Maximum number of modifications per peptide": "每个肽段的最大修饰数",
    "Maximum number of missed cleavages allowed for a peptide to be considered for scoring": "允许一个肽段被考虑用于打分的最大未切割数",
    "(Override the number of tasks to use on the threads; Default: (internally calculated based on inputs))": "（覆盖线程上使用的任务数；默认值：（根据输入内部计算））",
    "Fixed modifications, specified using Unimod (www.unimod.org) terms": "固定修饰，使用Unimod（www.unimod.org）术语指定",
    "Variable modifications, specified using Unimod (www.unimod.org) terms": "变量修饰，使用Unimod（www.unimod.org）术语指定",
    "Optional MSGF+ configuration file (passed as -conf <file> to MSGF+)": "可选的MSGF+配置文件（作为-conf <文件>传递给MSGF+）",
    "Create decoy proteins (reversed sequences) and append them to the database for the search (MS-GF+ parameter '-tda')": "创建诱饵蛋白（反向序列）并将它们添加到搜索数据库中（MS-GF+ 参数 '-tda'）",
    "Use the indirect conversion of MS-GF+ results to idXML via export to TSV": "通过将MS-GF+结果间接转换为idXML，使用导出到TSV的方式。",
    "Choose spectra": "选择谱图",
    "Load tabular file with pixel coordinates": "加载带有像素坐标的表格文件",
    "Column with x values": "X值列",
    "Column with y values": "列带有 y 值",
    "Separate plot per spectrum or overlaid plot with average spectra per annotation group": "每个谱图单独绘制一个图表，或者每个注释组的平均谱图叠加在一起绘制一个图表",
    "Select column with annotations": "选择带有注释的列",
    "Choose a colour scheme": "选择一个配色方案",
    "Colours for the plots": "图表的颜色",
    "Colours": "颜色",
    "Select a colourpalette": "选择一个颜色方案",
    "Plot complete m/z range": "绘制完整的 m/z 范围",
    "zoomed in m/z range": "放大 m/z 范围",
    "m/z value to subtract from m/z values in tabular file = lower m/z boundary": "m/z 值从表格文件中的 m/z 值中减去 = 较低的 m/z 边界",
    "m/z value to add to m/z values in tabular file = upper m/z boundary": "m/z 值添加到表格文件中的 m/z 值 = 上限 m/z 边界",
    "Load tabular file with m/z values": "加载带有 m/z 值的表格文件",
    "Column with mz values": "mz值的列",
    "Define min m/z and max m/z for the plotting window": "为绘图窗口定义最小 m/z 和最大 m/z",
    "lower m/z boundary for plotting window": "绘制窗口的较低 m/z 边界",
    "upper m/z boundary for plotting window": "绘制窗口的上m/z边界",
    "Add grid to plot": "添加网格到图表",
    "plusminus m/": "正负号 m/",
    "Contrast enhancement": "对比度增强",
    "Image smoothing": "图像平滑",
    "Colorscale": "色彩尺度",
    "Image Background": "图像背景",
    "Alpha power": "α功率",
    "Display colorkey in plot": "在图表中显示颜色键",
    "Linear normalization of image": "图像的线性归一化",
    "Export first valid m/z from tabular file as SVG": "将表格文件中的第一个有效 m/z 导出为 SVG",
    "Overlay image": "覆盖图像",
    "Position of legend": "图例位置",
    "Preprocessing": "预处理",
    "Preprocessing methods": "预处理方法",
    "Blocks": "区块",
    "Spar value": "Spar值",
    "Smoothing method": "平滑方法",
    "The standard deviation for the Gaussian kernel. Default = window/4": "高斯核的标准差。默认值 = 窗口/4",
    "The order of the smoothing filter, must be smaller than window size": "平滑滤波器的阶数，必须小于窗口大小",
    "Replace negative intensities with 0": "将负强度替换为0",
    "The coefficients for the moving average filter": "移动平均滤波器的系数",
    "tolerance": "公差",
    "The units to use for the tolerance.": "要使用的容差单位。",
    "Choose reference": "选择参考",
    "Tabular file with m/z features to use for alignment. Only the m/z values from the tabular file will be kept.": "用于对齐的m/z特征的表格文件。只有表格文件中的m/z值会被保留。",
    "quantile": "分位数",
    "span": "跨度",
    "Signal to noise ratio": "信噪比",
    "Number of blocks": "块数",
    "Peak picking method": "峰值挑选方法",
    "units": "单位",
    "Minimum frequency": "最小频率",
    "A reference to which the peaks are binned.": "对峰值进行分箱的参考。",
    "The tolerance to be used when matching the m/z features in the dataset to the reference. If this is NA, then automatically guess a resolution from the data.": "用于将数据集中的 m/z 特征与参考匹配时的容差。如果这个值为 NA，则自动从数据中猜测分辨率。",
    "Should the peak height or area under the curve be taken as the intensity value?": "应该将峰高或曲线下面积作为强度值？",
    "The width of a bin in m/z or ppm": "一个桶在 m/z 或者 ppm 中的宽度",
    "Unit for bin": "单元格单位",
    "Calculate sum or mean intensity for ions of the same bin": "计算同一箱体内离子的总强度或平均强度",
    "Replace NA with 0": "将NA替换为0",
    "Select m/z options": "选择 m/z 选项",
    "Minimum value for m/z": "最小 m/z 值",
    "Maximum value for m/z": "最大 m/z 值",
    "Tabular file with m/z features as reference for binning. Only the m/z values from the tabular file will be kept.": "用于分箱的m/z特征的表格文件作为参考。只有表格文件中的m/z值会被保留。",
    "Intensity transformations": "强度转换",
    "Annotation file that contains the pixel x and y coordinates, the batch identifier, and the condition annotation for each spectrum.": "包含每个光谱的像素 x 和 y 坐标、批次标识符以及条件注释的注释文件。",
    "X coordinates": "X坐标",
    "Y coordinates": "Y坐标",
    "Batch column": "批次列",
    "Condition column": "条件列",
    "Tabular file contains a header line": "表格文件包含一个标题行",
    "input profile data file": "输入数据文件",
    "Algorithm parameters section": "算法参数部分",
    "Minimal signal-to-noise ratio for a peak to be picked (0.0 disables SNT estimation!)": "峰值被选择的最小信噪比（0.0禁用信噪比估计！）",
    "The extension of a peak is stopped if the spacing between two subsequent data points exceeds 'spacing_difference_gap * min_spacing'": "峰值的延伸会在两个连续数据点之间的间距超过 'spacing_difference_gap * min_spacing' 时停止。",
    "Maximum allowed difference between points during peak extension, in multiples of the minimal difference between the peak apex and its two neighboring points": "在峰值扩展期间，允许的最大点间差异，以峰值顶点与其两个相邻点之间最小差异的倍数来表示",
    "Maximum number of missing points allowed when extending a peak to the left or to the right": "允许在向左或向右扩展峰值时缺失的最大点数",
    "List of MS levels for which the peak picking is applied": "应用峰值检测的质谱级别列表",
    "Add metadata for FWHM (as floatDataArray named 'FWHM' or 'FWHM_ppm', depending on param 'report_FWHM_unit') for each picked peak": "为每个选定的峰添加元数据，FWHM（作为名为'FWHM'或'FWHM_ppm'的浮点数据数组，取决于参数'report_FWHM_unit'）",
    "Unit of FWHM": "FWHM的单位",
    "maximal intensity considered for histogram construction": "用于直方图构建的最大强度",
    "parameter for 'max_intensity' estimation (if 'auto_mode' == 0): mean + 'auto_max_stdev_factor' * stdev": "参数用于“最大强度”估计（如果“自动模式”== 0）：均值 + 'auto_max_stdev_factor' * 标准差",
    "parameter for 'max_intensity' estimation (if 'auto_mode' == 1): auto_max_percentile th percentile": "参数用于“最大强度”估计（如果'auto_mode' == 1）：自动最大百分比 th 百分位",
    "method to use to determine maximal intensity: -1 --> use 'max_intensity'; 0 --> 'auto_max_stdev_factor' method (default); 1 --> 'auto_max_percentile' method": "用于确定最大强度的方法：-1 --> 使用 'max_intensity'；0 --> 'auto_max_stdev_factor' 方法（默认）；1 --> 'auto_max_percentile' 方法",
    "window length in Thomson": "Thomson中的窗口长度",
    "number of bins for intensity values": "强度值的箱数",
    "minimum number of elements required in a window (otherwise it is considered sparse)": "窗口所需的最小元素数量（否则被认为是稀疏的）",
    "noise value used for sparse windows": "用于稀疏窗口的噪声值",
    "Write out log messages in case of sparse windows or median in rightmost histogram bin": "在稀疏窗口或直方图最右端的箱中出现中位数时，写出日志消息",
    "Whether to load all data and process them in-memory or whether to process the data on the fly (lowmemory) without loading the whole file into memory first": "是否将所有数据加载到内存中进行处理，还是不先将整个文件加载到内存中就即时处理数据（低内存）",
    "Input file with RNA sequences and optionally charge numbers (mutually exclusive to 'in_seq')": "输入文件包含RNA序列，以及可选的电荷数（与'in_seq'互斥）",
    "List of RNA sequences (mutually exclusive to 'in')": "RNA序列列表（与'in'互斥）",
    "List of charge states; required if 'in_seq' is given": "电荷态列表；如果给出了'in_seq'，则必需",
    "Output format ('list': human-readable list, 'table': CSV-like table, 'mass_only': mass values only, 'mz_only': m/z values only, 'formula_only': sum formula only)": "输出格式（'list': 可读列表，'table': 类似CSV的表格，'mass_only': 仅质量值，'mz_only': 仅m/z值，'formula_only': 仅和式公式）",
    "Compute average (instead of monoisotopic) oligonucleotide masses": "计算平均（而非单一同位素）的寡核苷酸质量",
    "For what type of sequence/fragment the mass should be computed": "对于哪种类型的序列/片段应该计算质量",
    "Field separator for 'table' output format; by default, the 'tab' character is used": "用于“表格”输出格式的字段分隔符；默认情况下，使用“制表符”字符",
    "ConsensusXML input, generated by FeatureLinke": "ConsensusXML 输入，由 FeatureLink 产生",
    "Run tool in batch mode for -in_raw": "批量运行工具，针对 -in_raw 参数",
    "MzML input (after InternalCalibration, if available)": "MzML 输入（如果有内部校准，则在内部校准之后）",
    "Run tool in batch mode for -in_postFDR": "批量运行工具，针对参数 -in_postFDR",
    "FeatureXMLs after FDR filtering": "FDR过滤后的FeatureXMLs",
    "Proteins considered contaminants": "被认为是污染物的蛋白质",
    "Run tool in batch mode for -in_trafo": "批量模式下运行工具，针对 -in_trafo 参数",
    "trafoXMLs from MapAligners": "来自MapAligners的trafoXMLs",
    "If a Path is given, a MQEvidence txt-file will be created in this directory": "如果提供了路径，将在该目录下创建一个MQEvidence txt文件",
    "Fragment Mass Error settings": "片段质量误差设置",
    "Unit for mass tolerance": "质量容忍度单位",
    "m/z search window for matching peaks in two spectra": "m/z 搜索窗口，用于匹配两个谱图中的峰值",
    "MS2 ID Rate settings": "MS2 ID速率设置",
    "Forces the metric to run even if target/decoy annotation is missing (accepts all pep_ids as target hits)": "即使缺少目标/诱饵注释，也强制运行度量标准（接受所有pep_ids作为目标命中）",
    "input mzML file": "输入mzML文件",
    "fold change between XICs": "XICs之间的倍数变化",
    "RT tolerance in [s] for finding max peak (whole RT range around RT middle)": "RT容差（以秒为单位），用于寻找最大峰值（围绕RT中间的整个RT范围）",
    "m/z tolerance in [ppm] for finding a peak": "m/z 容差（以百万分之一为单位）用于寻找峰值",
    "Input file (OpenSwath results)": "输入文件（OpenSwath 结果）",
    "Assay library": "检测库",
    "Retention time transformation": "保留时间转换",
    "Number of decoy assays to select from the library for every true assay (0 for \"all\")": "用于为每个真实分析选择的诱饵分析数量（0表示“全部”）",
    "Number of transitions per feature to consider (highest intensities first; 0 for \"all\")": "每个特征要考虑的转换数（首先考虑最高强度；0表示“全部”）",
    "Parameters of the binomial GLM": "二项式广义线性模型（GLM）的参数",
    "Intercept term": "截距项",
    "Coefficient of retention time difference": "保留时间差异系数",
    "Coefficient of intensity distance": "强度距离系数",
    "Peptide multiplets with assigned sequence information": "具有指定序列信息的肽段多重态",
    "Optional input containing spectral peaks blacklisted during feature detection": "可选输入包含在特征检测期间被拉黑的光谱峰",
    "Parameters for the algorithm": "算法参数",
    "Labels used for labelling the samples": "用于标记样本的标签",
    "Maximum number of missed cleavages due to incomplete digestion": "最大未完全消化造成的切割缺失数",
    "Mass tolerance in Da for matching the mass shifts in the detected peptide multiplet to the theoretical mass shift pattern": "用于匹配检测到的肽多重态的质荷比偏移与理论质荷比偏移模式的质荷比容差（以道尔顿为单位）",
    "m/z tolerance in ppm for checking if dummy feature vicinity was blacklisted": "m/z 容差（ppm）用于检查虚拟特征邻近是否被加入黑名单",
    "Retention time tolerance in seconds for checking if dummy feature vicinity was blacklisted": "保留时间公差（以秒为单位），用于检查虚拟特征邻近是否被加入黑名单",
    "Isotopic labels that can be specified in section 'algorithm:labels'": "在“算法：标签”部分可以指定的同位素标记",
    "Label:13C(6)  |  C(-6) 13C(6)  |  unimod #188": "标记：13C(6)  |  C(-6)13C(6)  |  unimod #188",
    "Label:13C(6)15N(4)  |  C(-6) 13C(6) N(-4) 15N(4)  |  unimod #267": "标签：13C(6)15N(4)  |  C(-6) 13C(6) N(-4) 15N(4)  |  unimod #267",
    "Label:2H(4)  |  H(-4) 2H(4)  |  unimod #481": "标签：2H(4)  |  H(-4) 2H(4)  |  unimod #481",
    "Label:13C(6)15N(2)  |  C(-6) 13C(6) N(-2) 15N(2)  |  unimod #259": "标签：13C(6)15N(2)  |  C(-6) 13C(6) N(-2) 15N(2)  |  unimod #259",
    "Label:2H(3)  |  H(-3) 2H(3)  |  unimod #262": "标签：2H(3)  |  H(-3) 2H(3)  |  unimod #262",
    "Dimethyl  |  H(4) C(2)  |  unimod #36": "二甲基 | H(4) C(2) | unimod #36",
    "Dimethyl:2H(4)  |  2H(4) C(2)  |  unimod #199": "二甲基化：2H(4)  |  2H(4) C(2)  |  unimod #199",
    "Dimethyl:2H(4)13C(2)  |  2H(4) 13C(2)  |  unimod #510": "二甲基：2H(4)13C(2)  |  2H(4) 13C(2)  |  unimod #510",
    "Dimethyl:2H(6)13C(2)  |  H(-2) 2H(6) 13C(2)  |  unimod #330": "二甲基：2H(6)13C(2)  |  H(-2) 2H(6) 13C(2)  |  unimod #330",
    "ICPL  |  H(3) C(6) N O  |  unimod #365": "ICPL | H(3)C(6)N O | unimod #365",
    "ICPL:2H(4)  |  H(-1) 2H(4) C(6) N O  |  unimod #687": "ICPL:2H(4)  |  H(-1) 2H(4) C(6) N O  |  unimod #687  ICPL:2H(4)标记  |  H(-1) 2H(4) C(6) N O组成  |  unimod数据库 #687号",
    "ICPL:13C(6)  |  H(3) 13C(6) N O  |  unimod #364": "ICPL:13C(6)  |  H(3) 13C(6) N O  |  unimod #364  ICPL:13C(6)  |  H(3) 13C(6) N O  |  单一修饰物 #364",
    "ICPL:13C(6)2H(4)  |  H(-1) 2H(4) 13C(6) N O  |  unimod #866": "ICPL:13C(6)2H(4)  |  H(-1) 2H(4) 13C(6) N O  |  unimod #866  ICPL：13C（6）2H（4）  |  H（-1）2H（4）13C（6）N O  |  unimod #866",
    "MZSQLite DB": "MZSQLite 数据库",
    "Peptide Sequence List": "肽段序列列表",
    "Consecutive B-Ions": "连续的B型离子",
    "Consecutive Y-Ions": "连续Y离子",
    "Use Neutral Loss": "使用中性损失",
    "Use Internals": "使用内部",
    "Fragment Tolerance": "片段容差",
    "Input files separated by blank": "输入文件由空白分隔",
    "TraML transition file": "TraML 转换文件",
    "Whether to write short (one peptide per line) or long format (one transition per line)": "是否写短格式（每行一个肽段）或长格式（每行一个转换）",
    "If only the best scoring feature per peptide should be printed, give the variable name": "如果只需要打印每个肽段的最佳评分特征，请给出变量名。",
    "transition file": "转换文件",
    "Run tool in batch mode for -swath_files": "批量运行工具，用于 -swath_files",
    "Swath files that were used to extract the transitions": "用于提取转换的Swath文件",
    "Minimal distance to the edge to still consider a precursor, in Thomson (only in SWATH)": "最小距离到边缘仍然考虑前体的距离，以汤姆森为单位（仅在SWATH中）",
    "Report separate statistics for each of n RT slices of the map": "为地图的每个 n RT 切片报告单独的统计数据",
    "Show meta information about the whole experiment": "显示整个实验的元信息",
    "Shows data processing information": "显示数据处理信息",
    "Computes a summary statistics of intensities, qualities, and widths": "计算强度、质量、宽度的摘要统计信息",
    "the number of allowed missed cleavages": "允许的断裂数",
    "give your mass accuracy in pp": "给出您的质量精度，以百万分之一（ppm）为单位。",
    "minimum length of peptide": "最小肽长",
    "indicate 1 (peptide table only), 2 (statistics only) or (both peptide table + statistics)": "指明1（仅肽段表）、2（仅统计数据）或（肽段表+统计数据）",
    "The enzyme used for peptide digestion": "用于肽段消化的酶",
    "the motif for the restricted peptidome": "限制性肽组的基序",
    "If 'true' the optimal RT pair distance and deviation are estimated by fitting a gaussian distribution to the histogram of pair distance": "如果设置为'true'，则通过拟合成对距离的直方图的高斯分布来估计最优的RT对距离和偏差",
    "optimal pair distance in RT [sec] from light to heavy feature": "在RT（保留时间）中，从轻到重特征的最佳配对距离（秒）",
    "maximum allowed deviation below optimal retention time distance": "最大允许的偏离最佳保留时间距离",
    "maximum allowed deviation above optimal retention time distance": "最大允许的偏离最优保留时间距离",
    "optimal pair distances in m/z [Th] for features with charge +1 (adapted to +2, +3": "最优的成对距离在 m/z [Th] 对于带电荷 +1 的特征（适应到 +2，+3）",
    "maximum allowed deviation from optimal m/z distance": "最大允许的从最优 m/z 距离的偏差",
    "this option should be used if the features correspond mrm chromatograms (additionally the precursor is taken into account)": "如果特征对应于MRM色谱图（此外，还考虑了前体），则应使用此选项。",
    "Input file containing MS2 spectra": "包含MS2谱图的输入文件",
    "FASTA file or pro file": "FASTA 文件或 pro 文件",
    "Default X": "默认 X",
    "Set this to use the configuration given in 'default_config_file' exclusively, ignoring other parameters (apart from 'in', 'out', 'database', 'xtandem_executable') set via this adapte": "将此设置为仅使用在 'default_config_file' 中给定的配置，忽略通过此适配器设置的其他参数（除了 'in'、'out'、'database'、'xtandem_executable'）",
    "Precursor mass tolerance": "前体质量容差",
    "Fragment mass erro": "片段质量误差",
    "Parent monoisotopic mass error units": "父质量单同位素质量误差单位",
    "Fragment monoisotopic mass error units": "片段单同位素质量误差单位",
    "Maximum precursor charge ('0' to use X": "最大前体电荷（'0'表示使用X）",
    "By default, misassignment to the first and second isotopic 13C peak are also considered": "默认情况下，错误分配到第一个和第二个同位素13C峰也考虑在内",
    "Minimum fragment m/z": "最小片段 m/z",
    "Number of possible cleavage sites missed by the enzyme": "酶可能错过的切割位点数量",
    "Require only peptide end to have a valid cleavage site, not both": "只要求肽段的一端具有有效的裂解位点，而不是两端",
    "Which hits should be reported": "哪些命中应该被报告",
    "Maximal E-Value of a hit to be reported (only evaluated if 'output_result' is 'valid' or 'stochastic')": "要报告的命中的最大E值（仅在'output_result'为'valid'或'stochastic'时评估）",
    "Algorithm parameter subsection": "算法参数子节",
    "The size of the sliding window along the m/z axis": "沿 m/z 轴滑动窗口的大小",
    "The number of peaks that should be kept": "应保留的峰值数量",
    "Whether sliding window (one peak steps) or jumping window (window size steps) should be used": "是否应该使用滑动窗口（一个峰值步长）还是跳变窗口（窗口大小步长）",
    "transition file ('TraML','tsv','pqp')": "过渡文件（'TraML', 'tsv', 'pqp'）",
    "transition file ('TraML')": "过渡文件（'TraML'）",
    "additional nonlinear transition file ('TraML')": "额外的非线性转换文件（'TraML'）",
    "Optional, tab-separated file containing the SWATH windows for extraction: lower_offset upper_offset": "可选的，以制表符分隔的文件，包含用于提取的SWATH窗口：lower_offset upper_offset",
    "data is scanning SWATH data": "数据正在扫描SWATH数据",
    "Only extract RT around this value (-1 means extract over the whole range, a value of 600 means to extract around +/- 300 s of the expected elution)": "仅提取此值附近的RT（保留时间）（-1表示在整个范围内提取，600的值意味着在预期洗脱时间的前后300秒内提取）",
    "Extraction window in ion mobility dimension (in milliseconds)": "在离子迁移率维度中的提取窗口（以毫秒为单位）",
    "Extraction window in Thomson or ppm (see mz_extraction_window_unit)": "提取窗口在Thomson或ppm中（见mz_extraction_window_unit）",
    "Extraction window used in MS1 in Thomson or ppm (see mz_extraction_window_ms1_unit)": "在MS1中使用的提取窗口，单位为Thomson或ppm（参见mz_extraction_window_ms1_unit）",
    "Extraction window in ion mobility dimension for MS1 (in milliseconds)": "在离子迁移率维度中用于MS1的提取窗口（单位为毫秒）",
    "Parameters for the m/z and ion mobility calibration": "m/z和离子迁移率校准的参数",
    "Whether to use MS1 precursor data for the ion mobility calibration (default = false, uses MS2 / fragment ions for calibration)": "是否使用MS1前体数据进行离子迁移率校准（默认 = false，使用MS2 / 片段离子进行校准）",
    "Type of normalization function for IM calibration": "IM校准的归一化函数类型",
    "Debug file for Ion Mobility calibration": "Ion Mobility校准的调试文件",
    "Debug file for m/z calibration": "用于 m/z 校准的调试文件",
    "Library parameters section": "库参数部分",
    "How to interpret the provided retention time (the retention time column can either be interpreted to be in iRT, minutes or seconds)": "如何解释提供的保留时间（保留时间列可以解释为iRT、分钟或秒）",
    "Override an internal check that assures that all members of the same PeptideGroupLabel have the same PeptideSequence (this ensures that only different isotopic forms of the same peptide can be grouped together in the same label group)": "覆盖一个内部检查，该检查确保所有具有相同肽段组标签（PeptideGroupLabel）的成员具有相同的肽段序列（PeptideSequence）（这确保只有相同肽段的不同同位素形式可以在同一标签组中组合在一起）",
    "Force reading even if invalid modifications are encountered (OpenMS may not recognize the modification)": "即使遇到无效的修饰也强制读取（OpenMS可能无法识别修饰）",
    "Parameters for the RTNormalization for iRT petides": "iRT肽段的RTNormalization参数",
    "How to perform the alignment to the normalized RT space using anchor points": "如何使用锚点对标准化的RT空间进行比对",
    "Which outlier detection method to use (valid: 'iter_residual', 'iter_jackknife', 'ransac', 'none')": "使用哪种离群值检测方法（有效选项：'iter_residual'、'iter_jackknife'、'ransac'、'none'）",
    "Whether to use Chauvenet's criterion when using iterative methods": "是否在使用迭代方法时使用Chauvenet准则",
    "Maximum iterations for the RANSAC outlier detection algorithm": "RANSAC 外点检测算法的最大迭代次数",
    "Maximum threshold in RT dimension for the RANSAC outlier detection algorithm (in percent of the total gradient)": "RANSAC算法在RT维度中用于异常值检测的最大阈值（占总梯度的百分比）",
    "Sampling size of data points per iteration for the RANSAC outlier detection algorithm": "RANSAC 异常值检测算法每次迭代所需的数据点采样大小",
    "Whether the algorithms should try to choose the best peptides based on their peak shape for normalization": "算法是否应尝试根据峰形选择最佳的肽段以进行标准化",
    "The initial overall quality cutoff for a peak to be scored (range ca": "峰值得分的初始总体质量截止值（范围大约是",
    "The overall quality cutoff for a peak to go into the retention time estimation (range ca": "峰值进入保留时间估算的整体质量截止值（范围大约",
    "Number of RT bins to use to compute coverage": "用于计算覆盖度的RT区间数量",
    "Minimal number of peptides that are required for a bin to counted as 'covered'": "每个bin要被计算为“覆盖”所需的最小肽段数",
    "Minimal number of bins required to be covered": "最小需要覆盖的区间数",
    "Span parameter for lowess": "lowess的Span参数",
    "Number of nodes for b spline": "B样条的节点数",
    "Scoring parameters section": "评分参数部分",
    "Stop reporting after feature (ordered by quality; -1 means do not stop)": "在特征（按质量排序；-1表示不停止）后停止报告",
    "The normalized RT is expected to be between 0 and 1": "标准化的RT值预期在0到1之间",
    "Cutoff in m/z below which peaks should not be used for quantification any more": "m/z截止值，低于该值的峰不再用于定量",
    "Whether to write out all points of all features into the featureXML": "是否将所有特征的所有点写入到featureXML文件中",
    "For spectrum addition, either use simple concatenation or use peak resampling": "对于谱图添加，可以使用简单的串联，或者使用峰值重新采样。",
    "Add up spectra around the peak apex (needs to be a non-even integer)": "在峰顶周围累加谱图（需要是非偶数整数）",
    "If spectra are to be added, use this spacing to add them up": "如果需要添加光谱，请使用此间距将它们相加",
    "S/N threshold to consider identification transition (set to -1 to consider all)": "信噪比阈值，用于考虑识别转换（设置为-1以考虑全部）",
    "Peak area threshold to consider identification transition (set to -1 to consider all)": "考虑识别转换的峰面积阈值（设为-1以考虑全部）",
    "Scoring model to use": "使用的评分模型",
    "Extra drift time to extract for IM scoring (as a fraction": "用于IM评分的额外漂移时间（作为分数）",
    "Whether to error (true) or skip (false) if a transition in a transition group does not have a corresponding chromatogram": "是否在转换组中的一个转换没有相应的色谱图时出错（true）或跳过（false）",
    "Stop finding after feature (ordered by intensity; -1 means do not stop)": "在特征之后停止查找（按强度排序；-1表示不停止）",
    "Minimal peak width (s), discard all peaks below this value (-1 means no action)": "最小峰宽（秒），丢弃所有小于这个值的峰（-1表示不采取行动）",
    "Calculate the peak area and height either the smoothed or the raw chromatogram data": "计算平滑或原始色谱数据的峰面积和高度",
    "Remove background from peak signal using estimated noise levels": "使用估计的噪声水平从峰值信号中去除背景",
    "Tries to get better peak picking by looking at peak consistency of all picked peaks": "尝试通过查看所有选定峰值的峰值一致性来获得更好的峰值选择",
    "Use precursor chromatogram for peak picking (note that this may lead to precursor signal driving the peak picking)": "使用前体色谱图进行峰值选择（注意，这可能会导致前体信号驱动峰值选择）",
    "Use consensus peak boundaries when computing transition group picking (if false, compute independent peak boundaries for each transition)": "在计算转换组选择时使用共识峰边界（如果为假，则为每个转换计算独立的峰边界）",
    "Determines the maximal Z-Score (difference measured in standard deviations) that is considered too large for peak boundaries": "确定峰值边界所考虑的最大Z分数（以标准偏差为单位测量的差异）",
    "Only if compute_peak_quality is set, this parameter will not consider peaks below this quality threshold": "只有当设置了compute_peak_quality，这个参数才会不考虑低于这个质量阈值的峰值。",
    "For computing peak quality, how many extra seconds should be sample left and right of the actual peak": "在计算峰值质量时，实际峰值的左侧和右侧应该分别留出多少额外的秒数？",
    "Tries to compute a quality value for each peakgroup and detect outlier transitions": "尝试为每个峰值组计算一个质量值，并检测异常过渡",
    "Calculates various peak shape metrics": "计算各种峰形指标",
    "Compute mutual information metrics for individual transitions that can be used for OpenSWATH/IPF scoring": "计算可用于OpenSWATH/IPF评分的个体转换的互信息度量",
    "Method to use when selecting the best boundaries for peaks": "用于选择峰值最佳边界的方法",
    "The number of subsequent data points used for smoothing": "用于平滑的后续数据点数量",
    "Order of the polynomial that is fitted": "拟合的多项式阶数",
    "Gaussian width in seconds, estimated peak size": "高斯宽度（以秒为单位），估计的峰值大小",
    "Use Gaussian filter for smoothing (alternative is Savitzky-Golay filter)": "使用高斯滤波器进行平滑（另一种选择是Savitzky-Golay滤波器）",
    "Force a certain minimal peak_width on the data": "强制在数据上应用一定的最小峰宽",
    "Signal-to-noise threshold at which a peak will not be extended any more": "信号-噪声阈值，超过该阈值，峰值将不再扩展",
    "Write out log messages of the signal-to-noise estimator in case of sparse windows or median in rightmost histogram bin": "在稀疏窗口或直方图最右端的箱子中写入信噪估计器的日志消息",
    "Try to remove overlapping peaks during peak picking": "尝试在峰值选择期间移除重叠的峰值",
    "Which method to choose for chromatographic peak-picking (OpenSWATH legacy on raw data, corrected picking on smoothed chromatogram or Crawdad on smoothed chromatogram)": "选择哪种方法进行色谱峰值检测（在原始数据上使用OpenSWATH旧版，平滑色谱图上的校正检测，还是在平滑色谱图上使用Crawdad）",
    "The integration technique to use in integratePeak() and estimateBackground() which uses either the summed intensity, integration by Simpson's rule or trapezoidal integration": "在 integratePeak() 和 estimateBackground() 中使用的积分技术，可以采用总强度积分、Simpson规则积分或梯形积分。",
    "The baseline type to use in estimateBackground() based on the peak boundaries": "在 estimateBackground() 中使用的基线类型，基于峰值边界",
    "Fit the chromatogram/spectrum to the EMG peak model": "将色谱图/光谱拟合到EMG峰值模型",
    "DIA extraction window in Th or ppm": "DIA提取窗口在Th或ppm",
    "DIA extraction window unit": "DIA提取窗口单元",
    "Use centroided DIA data": "使用质心化的DIA数据",
    "DIA b/y series minimum intensity to conside": "DIA 系列的最小强度考虑",
    "DIA b/y series minimal difference in ppm to conside": "DIA b/y 系列最小差异以ppm为单位考虑",
    "DIA number of isotopes to conside": "DIA 考虑的同位素数量",
    "DIA number of charges to conside": "DIA 考虑的电荷数",
    "DIA maximal difference in ppm to count a peak at lower m/z when searching for evidence that a peak might not be monoisotopic": "DIA 最大差异（ppm）用于在搜索证据时计算较低 m/z 处的峰值，以确定该峰值可能不是单同位素峰。",
    "Maximum number of iterations using by Levenberg-Marquardt algorithm": "使用Levenberg-Marquardt算法的最大迭代次数",
    "Initialize parameters using method of moments estimators": "使用矩估计方法初始化参数",
    "Use the shape score (this score measures the similarity in shape of the transitions using a cross-correlation)": "使用形状得分（该得分使用交叉相关性来衡量转换的形状相似性）",
    "Use the coelution score (this score measures the similarity in coelution of the transitions using a cross-correlation)": "使用共洗脱分数（这个分数通过交叉相关性测量转换的共洗脱相似性）",
    "Use the retention time score (this score measure the difference in retention time)": "使用保留时间得分（这个得分衡量保留时间的差异）",
    "Use the library score": "使用库得分",
    "Use the intensity score": "使用强度分数",
    "Use the number of peaks score": "使用峰值数量得分",
    "Use the total XIC score": "使用总XIC得分",
    "Use the total MI score": "使用总互信息分数",
    "Use the SN (signal to noise) score": "使用信噪比（SN）分数",
    "Use the MI (mutual information) score": "使用MI（互信息）分数",
    "Use the DIA (SWATH) scores": "使用DIA（SWATH）评分",
    "Use the correlation scores with the MS1 elution profiles": "使用与MS1洗脱曲线相关的相关性分数",
    "Use the scores for SONAR scans (scanning swath)": "使用声纳扫描（扫描宽度）的分数",
    "Use the scores for Ion Mobility scans": "使用离子迁移率扫描的分数",
    "Use the full MS1 scan at the peak apex for scoring (ppm accuracy of precursor and isotopic pattern)": "使用峰顶处的完整MS1扫描进行评分（前体的ppm精度和同位素模式）",
    "Use the MS1 MI score": "使用MS1 MI分数",
    "Use UIS scores for peptidoform identification": "使用UIS分数进行肽形式鉴定",
    "Use MS2-level b/y ion-series scores for peptidoform identification": "使用MS2级别的b/y离子系列得分进行肽形式鉴定",
    "Use MS2-level isotope scores (pearson & manhattan) across product transitions (based on ID if annotated or averagine)": "使用MS2级别的同位素分数（皮尔逊和曼哈顿）跨产品转换（基于ID如果注释或平均素）",
    "RT normalization file (how to map the RTs of this run to the ones stored in the library)": "RT标准化文件（如何将此运行的RT映射到存储在库中的RT）",
    "Sort input SWATH files when matching to SWATH windows from swath_windows_file": "在将输入的 SWATH 文件与 swath_windows_file 中的 SWATH 窗口匹配时进行排序。",
    "Extract the precursor ion trace(s) and use for scoring if present": "提取前体离子迹线（s），如果存在，则用于评分。",
    "Enable additional scoring of identification assays using IPF (see online documentation)": "启用使用IPF（见在线文档）对鉴定分析进行额外评分",
    "File type of output out_chrom (Also output all computed chromatograms output in mzML (chrom.mzML) or sqMass (SQLite format))": "输出文件类型 out_chrom（还可以输出所有计算出的色谱图，输出为 mzML（chrom.mzML）或 sqMass（SQLite格式））",
    "Minimal distance to the upper edge of a Swath window to still consider a precursor, in Thomson": "最小距离到Swath窗口上边缘的Thomson值，仍然考虑作为前体。",
    "Output an XIC with a RT-window by this much large": "输出一个XIC（提取离子色谱图），其保留时间窗口比这个大这么多",
    "Unit for mz extraction": "mz提取的单位",
    "Unit of the MS1 m/z extraction window": "MS1 m/z提取窗口的单位",
    "Also perform precursor extraction using the same ion mobility window as for fragment ion extraction": "同时使用与片段离子提取相同的离子迁移窗口进行前体提取。",
    "Assume the input data is targeted / PRM-like data with potentially overlapping DIA windows": "假设输入数据是目标化/PRM（平行反应监测）类型的数据，可能存在重叠的DIA（数据独立采集）窗口。",
    "Extraction window used for iRT and m/z correction in Thomson or ppm (see irt_mz_extraction_window_unit)": "用于iRT和m/z校正的提取窗口，单位为Thomson或ppm（见irt_mz_extraction_window_unit）",
    "Ion mobility extraction window used for iRT (in 1/K0 or milliseconds)": "用于iRT的离子迁移窗口（以1/K0或毫秒为单位）",
    "Minimum r-squared of RT peptides regression": "最小R平方值的RT肽段回归",
    "Minimum relative amount of RT peptides to keep": "最小相对量的RT肽段保留",
    "The input files each contain one single SWATH (alternatively: all SWATH are in separate files)": "输入文件中每个文件包含一个单独的SWATH（或者：所有SWATH都在单独的文件中）",
    "Turn on elution model score (EMG fit to peak)": "开启洗脱模型得分（EMG拟合峰值）",
    "Whether to run OpenSWATH directly on the input data, cache data to disk first or to perform a datareduction step first": "是否直接在输入数据上运行OpenSWATH，先将数据缓存到磁盘上，或者首先执行数据简化步骤",
    "Use the retention time normalization peptide MS2 masses to perform a mass correction (linear, weighted by intensity linear or quadratic) of all spectra": "使用保留时间标准化的肽段MS2质量，对所有谱图进行质量校正（线性、按强度加权线性或二次方）",
    "Function used to extract the signal": "用于提取信号的函数",
    "The batch size of chromatograms to process (0 means to only have one batch, sensible values are around 250-1000)": "要处理的色谱图的批量大小（0表示只有一个批次，合理的值在250-1000左右）",
    "The number of MS1 isotopes used for extraction": "用于提取的MS1同位素数量",
    "List of qcml files to be merged": "要合并的 qcml 文件列表",
    "Use only when all given qcml files belong to one set, which will be held under the given name": "仅当所有给定的qcml文件属于同一组时使用，这些文件将被保存在给定的名称下",
    "Feature map/consensus map file": "特征图/共识图文件",
    "RT tolerance (in seconds) for the matching of peptide identifications and (consensus) features": "肽段鉴定和（共识）特征匹配的RT公差（以秒为单位）",
    "m/z tolerance (in ppm or Da) for the matching of peptide identifications and (consensus) features": "肽段鉴定和（共识）特征匹配的质量误差容限（以ppm或Da为单位）",
    "Unit of 'mz_tolerance'": "'mz_tolerance'的单位",
    "Source of m/z values for peptide identifications": "肽段鉴定的 m/z 值来源",
    "Additional options for featureXML input": "特征XML输入的额外选项",
    "Use the RT coordinates of the feature centroids for matching, instead of the RT ranges of the features/mass traces": "使用特征质心的保留时间（RT）坐标进行匹配，而不是特征/质谱迹的保留时间范围。",
    "Use the m/z coordinates of the feature centroids for matching, instead of the m/z ranges of the features/mass traces": "使用特征质心的 m/z 坐标进行匹配，而不是特征/质量迹线的 m/z 范围",
    "Additional options for consensusXML input": "共识XML输入的额外选项",
    "Match using RT and m/z of sub-features instead of consensus RT and m/z": "使用子特征的保留时间和质荷比进行匹配，而不是使用共识保留时间和质荷比",
    "Store the map index of the sub-feature in the peptide ID": "将子特征的映射索引存储在肽段ID中",
    "Additional options for mzML input": "mzML输入的额外选项",
    "MS run used to annotated unidentified spectra to features or consensus features": "用于将未鉴定的谱图注释为特征或共识特征的质谱运行",
    "For feature/consensus maps: Assign an ID independently of whether its charge state matches that of the (consensus) feature": "对于特征/共识图：独立于其电荷状态是否与（共识）特征匹配，分配一个ID",
    "Input mzML file list, containing the spectra": "输入包含谱图的mzML文件列表",
    "Run tool in batch mode for -id_in": "在批量模式下运行工具，用于 -id_in 参数",
    "Input idXML file list, containing the identifications": "输入包含鉴定结果的idXML文件列表",
    "If this flag is used, the precursor mass tolerances are estimated in ppm instead of Da": "如果使用此标志，则前体质量容差以ppm估算，而不是以道尔顿（Da）估算。",
    "If this flag is used, the fragment mass tolerances are estimated in ppm instead of Da": "如果使用了这个标志，片段质量容差将用ppm（百万分之一）而不是Da（道尔顿）来估算。",
    "Maximal fragment mass tolerance which is allowed for MS/MS spectra, used for the calculation of matching ions": "MS/MS 谱图允许的最大片段质量容差，用于计算匹配离子",
    "Number of bins that should be used to calculate the histograms for the fitting": "用于计算拟合直方图的箱数",
    "The RT of the first calibrant in the reference file": "参考文件中第一个校准物的保留时间",
    "The RT of the second calibrant in the reference file": "参考文件中第二个校准物的保留时间",
    "The RT of the first calibrant in the input file": "输入文件中第一个校准物的保留时间",
    "The RT of the second calibrant in the input file": "输入文件中第二个校准物的保留时间",
    "Input file (data annotated with identifications)": "输入文件（带有鉴定注释的数据）",
    "transition file ('TraML' or 'csv')": "过渡文件（'TraML' 或 'csv'）",
    "Stop after reaching intensity ratio": "在达到强度比率后停止",
    "Signal to noise window length": "信号噪声窗口长度",
    "Signal to noise bin count": "信噪比箱计数",
    "Input files to align (all must have the same file type)": "输入文件进行比对（所有文件必须具有相同的文件类型）",
    "Input file containing the experimental design": "输入文件包含实验设计",
    "Store the original retention times (before transformation) as meta data in the output?": "在输出中将原始保留时间（转换前）作为元数据存储？",
    "Options to define a reference file (use either 'file' or 'index', not both)": "定义参考文件的选项（使用 'file' 或 'index' 中的一个，不要同时使用）",
    "File to use as reference": "作为参考使用的文件",
    "Use one of the input files as reference ('1' for the first file, etc.)": "使用输入文件之一作为参考（'1' 表示第一个文件，依此类推）。",
    "Name of the score type to use for ranking and filtering (.oms input only)": "用于排名和筛选的分数类型名称（仅限.oms输入）",
    "Use only IDs above a score cut-off (parameter 'min_score') for alignment?": "使用高于分数截止值（参数 'min_score'）的ID进行比对？",
    "If 'score_cutoff' is 'true': Minimum score for an ID to be considered": "如果 'score_cutoff' 是 'true'：一个ID被认为是有效的最小分数",
    "Minimum number of runs (incl": "最小运行次数（包括",
    "Maximum realistic RT difference for a peptide (median per run vs": "肽段的最大真实保留时间差异（每个运行的中位数对比",
    "Should unassigned peptide identifications be used when computing an alignment of feature or consensus maps": "在计算特征或共识图的对齐时，是否应使用未分配的肽段鉴定结果",
    "When aligning feature or consensus maps, don't use the retention time of a peptide identification directly; instead, use the retention time of the centroid of the feature (apex of the elution profile) that the peptide was matched to": "在对特征或共识图进行比对时，不要直接使用肽段鉴定的保留时间；相反，使用肽段匹配到的特征（洗脱曲线顶点）的质心的保留时间。",
    "If IDs contain adducts, treat differently adducted variants of the same molecule as different": "如果ID包含加合物，则将同一分子的不同加合变体视为不同的",
    "Options to control the modeling of retention time transformations from data": "控制保留时间转换建模的选项",
    "Type of model": "模型类型",
    "Parameters for 'linear' model": "'线性'模型的参数",
    "Perform linear regression on 'y - x' vs": "对'y - x'进行线性回归",
    "Weight x values": "权重 x 值",
    "Weight y values": "权重 y 值",
    "Minimum x value": "最小 x 值",
    "Maximum x value": "最大 x 值",
    "Minimum y value": "最小 y 值",
    "Maximum y value": "最大 y 值",
    "Parameters for 'b_spline' model": "'b_spline'模型的参数",
    "Determines the amount of smoothing by setting the number of nodes for the B-spline": "通过设置B样条的节点数来确定平滑的程度",
    "Number of nodes for B-spline fitting": "B样条拟合的节点数",
    "Method to use for extrapolation beyond the original data range": "用于原始数据范围外外推的方法",
    "Boundary condition at B-spline endpoints: 0 (value zero), 1 (first derivative zero) or 2 (second derivative zero)": "B样条端点的边界条件：0（值为零），1（一阶导数为零）或2（二阶导数为零）",
    "Parameters for 'lowess' model": "'lowess'模型的参数",
    "Fraction of datapoints (f) to use for each local regression (determines the amount of smoothing)": "用于每个局部回归的数据点比例（f）（决定平滑程度）",
    "Number of robustifying iterations for lowess fitting": "lowess拟合的稳健化迭代次数",
    "Nonnegative parameter which may be used to save computations (recommended value is 0.01 of the range of the input": "非负参数，可用于节省计算（推荐值为输入范围的0.01）",
    "Method to use for interpolation between datapoints computed by lowess": "用于LOWESS计算的数据点之间插值的方法",
    "Method to use for extrapolation outside the data range": "用于数据范围外外推的方法",
    "Parameters for 'interpolated' model": "“插值”模型的参数",
    "Type of interpolation to apply": "要应用的插值类型",
    "Type of extrapolation to apply: two-point-linear: use the first and last data point to build a single linear model, four-point-linear: build two linear models on both ends using the first two / last two points, global-linear: use all points to build a single linear model": "外推类型应用：两点线性：使用第一个和最后一个数据点构建一个单一的线性模型，四点线性：在两端使用前两个/后两个点构建两个线性模型，全局线性：使用所有点构建一个单一的线性模型",
    "Isobaric Quantitation method used in the experiment": "实验中使用的同位素标记定量方法",
    "input raw/picked data file": "输入原始/挑选的数据文件",
    "Parameters for the channel extraction": "信道提取的参数",
    "Operate only on MSn scans where any of its precursors features a certain activation method": "仅对MSn扫描操作，其中其前体特征具有某种激活方法",
    "Allowed shift (left to right) in Th from the expected position": "允许从预期位置向左到右的Th偏移",
    "Minimum intensity of the precursor to be extracted": "前体提取的最小强度",
    "Flag if precursor with missing intensity value or missing precursor spectrum should be included or not": "如果前体缺失强度值或缺失前体谱图，是否应该包含标记",
    "Minimum intensity of the individual reporter ions to be extracted": "单个报告离子提取的最小强度",
    "Remove all reporter intensities if a single reporter is below the threshold given in 'min_reporter_intensity'": "如果任何一个报告者的强度低于在 'min_reporter_intensity' 中给定的阈值，则移除所有报告者的强度。",
    "Minimum fraction of the total intensity in the isolation window of the precursor spectrum attributable to the selected precurso": "前体谱图中隔离窗口内可归因于选定前体的总强度的最小分数",
    "Maximum allowed deviation (in ppm) between theoretical and observed isotopic peaks of the precursor peak in the isolation window to be counted as part of the precurso": "允许的偏差（以ppm为单位）在隔离窗口中的前体峰的理论同位素峰和观察到的同位素峰之间，被认为是前体的一部分",
    "If set to true the algorithm will try to compute the purity as a time weighted linear combination of the precursor scan and the following scan": "如果设置为true，算法将尝试将纯度计算为前体扫描和随后扫描的时间加权线性组合",
    "Algorithm parameters for iTRAQ 4-plex": "iTRAQ 4-plex算法参数",
    "Description for the content of the 114 channel": "114通道内容的描述",
    "Description for the content of the 115 channel": "115通道的内容描述",
    "Description for the content of the 116 channel": "116通道内容的描述",
    "Description for the content of the 117 channel": "117通道内容的描述",
    "Number of the reference channel (114-117)": "参考通道的数量（114-117）",
    "Correction matrix for isotope distributions (see documentation); use the following format: <-2Da>/<-1Da>/<+1Da>/<+2Da>;": "同位素分布的校正矩阵（参见文档）；使用以下格式：<-2Da>/<-1Da>/<+1Da>/<+2Da>；",
    "Algorithm parameters for iTRAQ 8-plex": "iTRAQ 8-plex算法参数",
    "Description for the content of the 113 channel": "113通道内容的描述",
    "Description for the content of the 118 channel": "118通道内容的描述",
    "Description for the content of the 119 channel": "119通道内容的描述",
    "Description for the content of the 121 channel": "121通道内容的描述",
    "Number of the reference channel (113-121)": "参考通道的数量（113-121）",
    "Parameters for the peptide quantification": "肽段定量的参数",
    "Enable isotope correction (highly recommended)": "启用同位素校正（强烈推荐）",
    "Enable normalization of channel intensities with respect to the reference channel": "启用通道强度相对于参考通道的标准化",
    "Algorithm parameters for TMT 10-plex": "TMT 10-plex 算法参数",
    "Description for the content of the 126 channel": "126通道内容的描述",
    "Description for the content of the 127N channel": "127N通道的内容描述",
    "Description for the content of the 127C channel": "127C通道的内容描述",
    "Description for the content of the 128N channel": "128N通道的内容描述",
    "Description for the content of the 128C channel": "128C通道的内容描述",
    "Description for the content of the 129N channel": "129N通道内容的描述",
    "Description for the content of the 129C channel": "129C通道内容的描述",
    "Description for the content of the 130N channel": "130N通道内容的描述",
    "Description for the content of the 130C channel": "130C通道的内容描述",
    "Description for the content of the 131 channel": "131通道内容的描述",
    "The reference channel (126, 127N, 127C, 128N, 128C, 129N, 129C, 130N, 130C, 131)": "参考通道（126, 127N, 127C, 128N, 128C, 129N, 129C, 130N, 130C, 131）",
    "Algorithm parameters for TMT 11-plex": "TMT 11-plex的算法参数",
    "Description for the content of the 131N channel": "131N通道的内容描述",
    "Description for the content of the 131C channel": "131C通道的内容描述",
    "The reference channel (126, 127N, 127C, 128N, 128C, 129N, 129C, 130N, 130C, 131N, 131C)": "参考通道（126, 127N, 127C, 128N, 128C, 129N, 129C, 130N, 130C, 131N, 131C）",
    "Algorithm parameters for TMT 16-plex": "TMT 16-plex 的算法参数",
    "Description for the content of the 132N channel": "132N通道内容的描述",
    "Description for the content of the 132C channel": "132C通道内容的描述",
    "Description for the content of the 133N channel": "133N通道的内容描述",
    "Description for the content of the 133C channel": "133C通道内容的描述",
    "Description for the content of the 134N channel": "134N通道内容描述",
    "Algorithm parameters for TMT 6-plex": "TMT 6-plex算法参数",
    "Description for the content of the 127 channel": "127通道内容的描述",
    "Description for the content of the 128 channel": "128通道内容的描述",
    "Description for the content of the 129 channel": "129通道内容的描述",
    "Description for the content of the 130 channel": "130通道内容的描述",
    "Number of the reference channel (126-131)": "参考通道的数量（126-131）",
    "Input mzML file containing the ER spectra": "输入包含ER谱图的mzML文件",
    "Pair-file in the format: m/z-light m/z-heavy charge rt": "配对文件的格式为：m/z-轻 m/z-重 电荷 rt",
    "Precursor mass tolerance which is used for the pair finding and the matching of the given pair m/z values to the features": "前体质量容差，用于成对发现和将给定的配对 m/z 值与特征匹配。",
    "Maximal deviation in RT dimension in seconds a feature can have when comparing to the RT values given in the pair file": "在与配对文件中给定的保留时间（RT）值进行比较时，特征在保留时间维度上的最大偏差（以秒为单位）。",
    "Maximal charge state features should be search fo": "最大电荷状态特征应该被搜索",
    "Intensity threshold, for the meaning see the documentation of the IsotopeWaveletFeatureFinder documentation": "强度阈值，具体含义请参见 IsotopeWaveletFeatureFinder 文档",
    "Max isotope of the isotope distribution to be considered": "考虑同位素分布中的最大同位素",
    "The range that is used to extend the isotope distribution with null intensity peaks in Th": "在Th中，用于扩展带有零强度峰的同位素分布的范围",
    "Mass traces": "质量迹线",
    "Minimal pearson correlation score": "最小皮尔逊相关系数得分",
    "Minimal peak nr to output pseudo spectra": "最小峰值数量以输出伪谱",
    "Maximal lag": "最大滞后",
    "Maximal difference of the apex in retention time": "保留时间中顶点的最大差异",
    "Maximal intensity to be added to a spectrum": "要添加到谱图中的最大强度",
    "Add a precursor mass": "添加一个前体质量",
    "MS1 mass traces": "MS1 质量迹线",
    "MS2 / SWATH mass traces": "MS2 / SWATH 质谱轨迹",
    "Assign unassigned MS2 fragments to all precursors (only for ms1_centrif)": "为所有前体分配未分配的MS2片段（仅适用于ms1_centrif）",
    "Minimal pearson correlation score to match elution profiles to each othe": "最小皮尔逊相关系数得分，用于匹配洗脱轮廓彼此之间的相似性",
    "Minimal number of ions to report a spectrum": "报告一个谱图所需的最小离子数",
    "Maximal difference of the apex in retention time (in seconds)": "保留时间（秒）中顶点的最大差异",
    "Swath lower isolation window": "Swath 下隔离窗口",
    "Swath upper isolation window": "Swath上隔离窗口",
    "Default Comet params file": "默认 Comet 参数文件",
    "Precursor monoisotopic mass tolerance (Comet parameter: peptide_mass_tolerance)": "前体单一同位素质量容差（Comet参数：peptide_mass_tolerance）",
    "Unit of precursor monoisotopic mass tolerance for parameter precursor_mass_tolerance (Comet parameter: peptide_mass_units)": "前体单一同位质量容差单位，参数为precursor_mass_tolerance（Comet参数：peptide_mass_units）",
    "This parameter controls whether the peptide_mass_tolerance takes into account possible isotope errors in the precursor mass measurement": "这个参数控制 peptide_mass_tolerance 是否考虑前体质量测量中可能的同位素误差",
    "This is half the bin size, which is used to segment the MS/MS spectrum": "这是用于将MS/MS谱图分段的一半的bin大小",
    "Offset of fragment bins": "片段箱的偏移量",
    "Comets theoretical_fragment_ions parameter: theoretical fragment ion peak representation, high-res: sum of intensities plus flanking bins, ion trap (low-res) ms/ms: sum of intensities of central M bin only": "Comets 理论片段离子参数：理论片段离子峰表示，高分辨率：强度总和加上两侧的峰，离子阱（低分辨率）ms/ms：仅中心M峰的强度总和",
    "Specify the termini where the cleavage rule has to match": "指定切割规则必须匹配的末端",
    "Minimum peptide length to conside": "最小考虑的肽段长度",
    "Maximum peptide length to conside": "最大考虑的肽段长度",
    "Number of peptide hits in output file": "输出文件中肽段命中数",
    "Precursor charge range to search (if spectrum is not annotated with a charge or if override_charge!=keep any known): 0:[num] == search all charges, 2:6 == from +2 to +6, 3:3 == +3": "搜索前体电荷范围（如果谱图没有注释电荷或如果override_charge!=keep any known）：0:[num] == 搜索所有电荷，2:6 == 从+2到+6，3:3 == +3",
    "_keep any known_: keep any precursor charge state (from input), _ignore known_: ignore known precursor charge state and use precursor_charge parameter, _ignore outside range_: ignore precursor charges outside precursor_charge range, _keep known search unknown_: keep any known precursor charge state": "保留已知的：保留任何前体电荷状态（来自输入），忽略已知的：忽略已知的前体电荷状态并使用precursor_charge参数，忽略范围外的：忽略超出precursor_charge范围的前体电荷，保留已知搜索未知的：保留任何已知的前体电荷状态",
    "MS level to analyze, valid are levels 2 (default) or 3": "要分析的质谱级别，有效的是级别2（默认）或3",
    "If not ALL, only searches spectra of the given method": "如果不是全部，只搜索给定方法的谱图",
    "Set maximum fragment charge state to analyze as long as still lower than precursor charge - 1": "将最大片段电荷状态设置为分析，只要它仍然低于前体电荷 - 1",
    "If set to true, also considers the peptide sequence w/o N-term methionine separately and applies appropriate N-term mods to it": "如果设置为真，还会单独考虑没有N端甲硫氨酸的肽序列，并对它应用适当的N端修饰。",
    "List of modification group indices": "修改组索引列表",
    "use A ions for PSM": "使用 A 离子进行 PSM",
    "use B ions for PSM": "使用B离子进行PSM（肽段匹配）",
    "use C ions for PSM": "使用C离子进行PSM（肽段匹配）",
    "use X ions for PSM": "使用X个离子进行PSM（肽段匹配）",
    "use Y ions for PSM": "使用 Y 离子进行 PSM（肽段匹配）",
    "use Z ions for PSM": "使用Z型离子进行PSM（肽段匹配）",
    "use neutral loss (NH3, H2O) ions from b/y for PSM": "使用中性丢失（NH3，H2O）离子从b/y中用于PSM",
    "Additional enzyme used for peptide digestion": "用于肽段消化的额外酶",
    "MH+ peptide mass range to analyze": "MH+ 肽段质量范围以进行分析",
    "set maximum precursor charge state to analyze (allowed max 9)": "设置分析的最大前体电荷状态（允许的最大值为9）",
    "One or more mass offsets to search (values subtracted from deconvoluted precursor mass)": "一个或多个质量偏移量用于搜索（从反卷积前体质量中减去的值）",
    "Required minimum number of peaks in spectrum to search (default 10)": "在谱图中搜索所需的最小峰值数量（默认为10）",
    "Minimum intensity value to read in": "读取的最小强度值",
    "no = no removal, yes = remove all peaks around precursor m/z, charge_reduced = remove all charge reduced precursor peaks (for ETD/ECD)": "no = 不移除，yes = 移除所有前体 m/z 附近的峰，charge_reduced = 移除所有电荷减少的前体峰（对于 ETD/ECD）",
    "one-sided tolerance for precursor removal in Thompson": "Thompson中的前体去除单侧公差",
    "for iTRAQ/TMT type data; will clear out all peaks in the specified m/z range, if not 0:0": "对于iTRAQ/TMT类型的数据；如果指定的m/z范围内不是0:0，则会清除该范围内的所有峰值。",
    "Set a maximum number of variable modifications per peptide": "为每个肽段设置可变修饰的最大数量",
    "If true, requires at least one variable modification per peptide": "如果为真，则每个肽段至少需要一种变量修饰",
    "input file in mzML format": "输入文件为mzML格式",
    "Algorithm section": "算法部分",
    "maximal amino acid frequency per decomposition": "最大氨基酸频率每分解",
    "if set to true only tryptic peptides are reported": "如果设置为 true，则仅报告胰蛋白酶肽段",
    "precursor mass tolerance": "前体质量容差",
    "fragment mass tolerance": "片段质量容差",
    "maximal number of pivot ions to be used": "最大使用的枢纽离子数",
    "maximal number of solutions of a subsegment that are kept": "保留子段解决方案的最大数量",
    "precision used to calculate the decompositions, this only affects cache usage!": "用于计算分解的精度，这只会影响缓存使用！",
    "minimal isotope intensity correlation of doubly charged ions to be used to score the single scored ions": "用于评分单个得分离子的双电荷离子的最小同位素强度相关性",
    "maximal m/z value used to calculate isotope distributions": "最大 m/z 值用于计算同位素分布",
    "minimal m/z value used to calculate the isotope distributions": "用于计算同位素分布的最小 m/z 值",
    "max isotope peak to be considered in the scoring": "在评分中考虑的最大同位素峰",
    "maximal m/z difference used to calculate the decompositions": "最大 m/z 差异用于计算分解",
    "max isotope used in the theoretical spectra to score": "在理论谱图中用于评分使用的最大同位素数",
    "maximal number of missed cleavages allowed per peptide": "每个肽段允许的最大未切割数",
    "maximal number of hits which are reported per spectrum": "每个谱图报告的最大命中数",
    "If set to true, the precursor charge will be estimated": "如果设置为真，将估计前体电荷",
    "how many sequences are kept after first rough scoring for better scoring": "在初次粗略评分后，有多少序列被保留以便进行更好的评分？",
    "fixed modifications, specified using UniMod (www.unimod.org) terms": "固定修饰，使用UniMod（www.unimod.org）术语指定",
    "variable modifications, specified using UniMod (www.unimod.org) terms": "变量修饰，使用UniMod（www.unimod.org）术语指定",
    "The predefined amino acid set that should be used, see doc of ResidueDB for possible residue sets": "应使用的预定义氨基酸集，请参阅ResidueDB文档以了解可能的残基集",
    "Input files separated by blanks (all must have the same type)": "输入文件由空格分隔（所有文件必须是相同类型）",
    "Output file type (default: determined from file extension)": "输出文件类型（默认：根据文件扩展名确定）",
    "Optional input file": "可选输入文件",
    "Store the original filename in each protein/peptide identification (meta value: 'file_origin') - idXML input/output only": "在每个蛋白质/肽段鉴定中存储原始文件名（元值：'file_origin'）- 仅限idXML输入/输出",
    "Merge idXML files derived from a pepXML and corresponding protXML file": "合并来自 pepXML 和相应 protXML 文件的 idXML 文件",
    "Merge all identified proteins by accession into one protein identification run but keep all the PSMs with updated links to potential new protein ID#s": "将所有通过访问号识别的蛋白质合并到一个蛋白质鉴定运行中，但保留所有与潜在新蛋白质ID#更新链接的PSM（肽段匹配）。",
    "Input spectra": "输入光谱",
    "Default spectral database": "默认的谱图数据库",
    "Error allowed for precursor ion mass": "前体离子质量允许的误差",
    "Error allowed for product ions": "允许的产物质子误差",
    "Unit of mass error (ppm or Da)": "质量误差单位（ppm或Da）",
    "Which results shall be reported: the top-three scoring ones or the best scoring one?": "应该报告哪些结果：得分最高的前三个还是得分最高的那一个？",
    "Positive or negative ionization mode?": "正离子模式或负离子模式？",
    "Indicates whether the output file format of the table should be LaTeX or TSV (default)": "指示表格的输出文件格式应该是LaTeX还是TSV（默认）",
    "Significance level of first dimension RT filte": "第一维度 RT 显著性水平过滤器",
    "Significance level of second dimension RT filte": "第二维度显著性水平 RT 过滤器",
    "Input qcml file": "输入 qcml 文件",
    "The name of the target runs or sets to be exported from": "目标名称的运行或设置将被导出",
    "The mapping of the exported table's headers to the according qp cvs": "导出表格的标题映射到相应的qp列",
    "Normalize via dividing by TIC ('to_TIC') per spectrum or normalize to max": "通过将每个谱图的总离子流（TIC）除以（'to_TIC'）进行归一化，或者归一化到最大值",
    "Input peak file": "输入峰值文件",
    "Finding calibrants in raw data uses this tolerance (for lock masses and ID's)": "在原始数据中寻找校准剂使用此容差（针对锁定质量及ID）",
    "Target MS levels to apply the transformation onto": "目标质谱级别，应用转换到",
    "RT window (one-sided": "RT窗口（单边）",
    "Chose one of two optional input files ('id_in' or 'lock_in') to define the calibration masses/function": "选择两个可选输入文件中的一个（'id_in' 或 'lock_in'）来定义校准质量/函数",
    "Identifications or features whose peptide ID's serve as calibration masses": "鉴定或特征，其肽段ID作为校准质量",
    "Input file containing reference m/z values (text file with each line as: m/z ms-level charge) which occur in all scans": "输入文件包含参考 m/z 值（文本文件，每行格式为：m/z 质谱级别 电荷）这些值出现在所有扫描中",
    "Require all lock masses to be monoisotopic": "要求所有锁定的质量为单同位素质量",
    "Require all lock masses to have at least the +1 isotope": "要求所有锁定的质量至少具有+1同位素",
    "Type of function to be fitted to the calibration points": "要拟合到校准点的函数类型",
    "Robust outlier removal using RANSAC": "鲁棒异常值去除使用RANSAC",
    "Apply RANSAC to calibration points to remove outliers before fitting a model": "对校准点应用RANSAC算法以去除异常值，然后再拟合模型。",
    "Threshold for accepting inliers (instrument precision (not accuracy!) as ppm^2 distance)": "接受内含物的阈值（仪器精度（不是准确性！）作为ppm^2距离）",
    "Minimum percentage (of available data) of inliers (<threshold away from model) to accept the model": "最小百分比（可用数据的百分比）的内点（距离模型小于阈值）以接受模型",
    "Maximal # iterations": "最大迭代次数",
    "Thresholds for accepting calibration success": "接受校准成功的阈值",
    "The median ppm error of calibrated masses must be smaller than this threshold": "校准质量的中位数ppm误差必须小于这个阈值",
    "The median absolute deviation of the ppm error of calibrated masses must be smaller than this threshold": "校准质量的ppm误差的中位数绝对偏差必须小于这个阈值",
    "input centroided mzML file": "输入质心化的mzML文件",
    "output file type -- default: determined from file extension or content": "输出文件类型 -- 默认值：根据文件扩展名或内容确定",
    "Intensity threshold below which peaks are regarded as noise": "强度阈值，低于该阈值的峰被视为噪声",
    "Minimum signal-to-noise a mass trace should have": "质量迹线应该具有的最小信噪比",
    "Expected chromatographic peak width (in seconds)": "预期色谱峰宽（以秒为单位）",
    "Allowed mass deviation (in ppm)": "允许的质量偏差（以ppm为单位）",
    "Enables dynamic re-estimation of m/z variance during mass trace collection stage": "启用在质量迹线收集阶段动态重新估计 m/z 变化的功能",
    "Method of quantification for mass traces": "质谱痕迹的定量方法",
    "Termination criterion for the extension of mass traces": "质谱迹线扩展的终止标准",
    "Mass trace extension in one direction cancels if this number of consecutive spectra with no detectable peaks is reached": "如果达到连续没有可检测到的峰的谱图数量，一个方向上的大量迹线扩展将被取消",
    "Minimum fraction of scans along the mass trace that must contain a peak": "沿质量迹线扫描的最小分数，必须包含一个峰值",
    "Minimum expected length of a mass trace (in seconds)": "质谱迹线（mass trace）的最小期望长度（以秒为单位）",
    "Maximum expected length of a mass trace (in seconds)": "质谱迹线的最大期望长度（以秒为单位）",
    "Enable filtering of unlikely peak widths": "启用对不太可能的峰宽进行过滤",
    "Minimum full-width-at-half-maximum of chromatographic peaks (in seconds)": "色谱峰半高宽的最小全宽（以秒为单位）",
    "Maximum full-width-at-half-maximum of chromatographic peaks (in seconds)": "色谱峰最大半高宽（以秒为单位）",
    "Apply post-filtering by signal-to-noise ratio after smoothing": "在平滑处理后，根据信噪比进行后过滤",
    "Enables/disables the chromatographic peak detection of mass traces": "启用/禁用质谱迹线的色谱峰检测",
    "Set this if you cannot control the filename of 'out'": "如果无法控制 'out' 文件名，请设置此选项",
    "The number of allowed missed cleavages": "允许的未切割数",
    "The type of digestion enzyme": "消化酶的类型",
    "Options for FASTA output files": "FASTA输出文件的选项",
    "Identifier to use for each peptide: copy from parent protein (parent); a consecutive number (number); parent ID + consecutive number (both)": "每个肽段使用的标识符：从父蛋白复制（parent）；连续编号（number）；父ID + 连续编号（both）",
    "Keep or remove the (possibly lengthy) FASTA header description": "保留或移除（可能很长的）FASTA头文件描述",
    "Name of the directory where the model files are kept": "模型文件保存的目录名称",
    "Find optimal precursor mass and charge values": "找到最优的前体质量值和电荷值",
    "Do not correct charge": "不要纠正电荷",
    "Do not correct the precursor m/z value that appears in the file": "不要校正文件中出现的前体 m/z 值",
    "Do not remove low quality spectra": "不要移除低质量的谱图",
    "The fragment tolerance (between 0 and 0.75 Da": "片段容差（介于0到0.75 Da之间）",
    "The precursor mass tolerance (between 0 and 5.0 Da": "前体质量容差（介于0和5.0 Da之间）",
    "Name of the model that should be used": "应使用的模型名称",
    "Enzyme used for digestion (default TRYPSIN)": "用于消化的酶（默认为胰蛋白酶）",
    "Returns peptide sequence of the specified length (only lengths 3-6 are allowed)": "返回指定长度的肽序列（只允许长度为3-6）",
    "Number of solutions to be computed": "要计算的解决方案数量",
    "Input idXML file containing the identifications": "输入包含鉴定信息的idXML文件",
    "Input sequence database in FASTA format": "输入序列数据库为FASTA格式",
    "Input file to transform (separated by blanks)": "要转换的输入文件（由空格分隔）",
    "Transformation to apply": "应用转换",
    "Invert transformation (approximatively) before applying it": "在应用转换之前（近似地）反转转换",
    "Store the original retention times (before transformation) as meta data in the output file": "在输出文件中将原始保留时间（转换前）作为元数据存储",
    "base name of DTA output files (RT, m/z and extension are appended)": "DTA输出文件的基本名称（RT、m/z和扩展名将被附加）",
    "m/z range of precursor peaks to extract": "前体峰提取的 m/z 范围",
    "retention time range of spectra to extract": "提取的光谱保留时间范围",
    "MS levels to extract": "提取的质谱级别",
    "minimal number of transitions": "最小转换数",
    "maximal number of transitions": "最大转换数",
    "allowed fragment types": "允许的片段类型",
    "allowed fragment charge states": "允许的片段电荷状态",
    "set this flag if specific neutral losses for detection fragment ions should be allowed": "如果应该允许特定的中性损失以检测片段离子，则设置此标志",
    "set this flag if unspecific neutral losses (H2O1, H3N1, C1H2N2, C1H2N1O1) for detection fragment ions should be allowed": "如果允许检测片段离子时发生非特异性中性损失（H2O1，H3N1，C1H2N2，C1H2N1O1），请设置此标志。",
    "MZ threshold in Thomson for precursor ion selection": "在Thomson单位中，用于前体离子选择的m/z阈值",
    "lower MZ limit for precursor ions": "前体离子的较低 m/z 限制",
    "upper MZ limit for precursor ions": "前体离子的上限质量数",
    "MZ threshold in Thomson for fragment ion annotation": "在Thomson单位中用于片段离子注释的m/z阈值",
    "lower MZ limit for fragment ions": "片段离子的较低 m/z 限制",
    "upper MZ limit for fragment ions": "片段离子的上限质量（m/z）",
    "Tab separated file containing the SWATH windows for exclusion of fragment ions falling into the precursor isolation window: lower_offset upper_offset \\newline 400 425 \\newline": "包含SWATH窗口的制表符分隔文件，用于排除落入前体隔离窗口的片段离子：lower_offset upper_offset \\newline 400 425 \\newline",
    "(Modified) Unimod XML file (http://www.unimod.org/xml/unimod.xml) describing residue modifiability": "（修改版）Unimod XML文件（http://www.unimod.org/xml/unimod.xml），描述残基可修饰性",
    "IPF: set this flag if identification transitions should be generated for IPF": "IPF：如果应该为IPF生成鉴定转换，则设置此标志",
    "IPF: maximum number of site-localization permutations": "IPF：最大定位排列数",
    "IPF: set this flag if MS2-level precursor ions for identification should not be allowed for extraction of the precursor signal from the fragment ion data (MS2-level)": "IPF：如果不允许从碎片离子数据（MS2级别）中提取前体信号，则应设置此标志。",
    "IPF: set this flag if specific neutral losses for identification fragment ions should not be allowed": "IPF：如果不允许特定的中性损失用于鉴定碎片离子，则设置此标志",
    "IPF: set this flag if unspecific neutral losses (H2O1, H3N1, C1H2N2, C1H2N1O1) for identification fragment ions should be allowed": "IPF：如果允许在鉴定碎片离子时发生非特异性中性丢失（H2O1，H3N1，C1H2N2，C1H2N1O1），请设置此标志。",
    "IPF: set this flag if identification transitions without precursor specificity": "IPF：如果鉴定转换没有前体特异性，则设置此标志",
    "Centroided MS1 data": "质心化MS1数据",
    "Protein sequence database": "蛋白质序列数据库",
    "Feature data annotated with identifications (IDMapper)": "特征数据用识别工具（IDMapper）进行了注释",
    "Tolerance in ppm": "容差（ppm）",
    "Rolerance window around feature rt for XIC extraction": "特征保留时间周围的容差窗口，用于XIC提取",
    "Intensity threshold to collect peaks in the MS1 spectrum": "在MS1谱图中收集峰值的强度阈值",
    "Correlation threshold for reporting a RIA": "报告相关性指数（RIA）的相关性阈值",
    "Minimum correlation to mono-isotopic peak for retaining a higher isotopic peak": "保留更高同位素峰的最小相关性",
    "Minimum R-squared of decomposition that must be achieved for a peptide to be reported": "分解的最小 R 方差必须达到，才能报告一个肽段",
    "Decomposition coefficients within +- this rate window will be combined": "在此速率窗口内分解系数将被合并",
    "Extension used for plots (png|svg|pdf)": "用于绘图的扩展名（png|svg|pdf）",
    "Output directory for the quality report": "质量报告的输出目录",
    "Which element (single letter code) is labeled": "哪个元素（单字母代码）被标记",
    "Include identifications not assigned to a feature in pattern detection": "在模式检测中包括未分配给特征的鉴定",
    "Use averagine peptides as model to perform pattern detection on unidentified peptides": "使用平均糖肽作为模型，对未知肽段进行模式检测。",
    "Whether purely natural peptides are reported in the quality report": "在质量报告中是否报告了纯自然肽",
    "Try to filter out mono-isotopic patterns to improve detection of low RIA patterns": "尝试过滤掉单同位素模式以提高低丰度相对离子丰度模式的检测",
    "Perform grouping": "执行分组",
    "Minimum difference in correlation between incorporation pattern and averagine pattern": "最小相关性差异，指的是在将蛋白质的氨基酸组成与平均蛋白质组成（Averagine）进行比较时，两者之间的相关性差异。",
    "The most intense peaks of the theoretical pattern contributing to at least this TIC fraction are taken into account": "考虑至少贡献了这个TIC分数的理论上最强烈的峰值",
    "Number of RIA bins for heat map generation": "热图生成的 RIA 箱数",
    "Fraction of observed/expected peaks": "观察到的/预期的峰值比例",
    "Minimum number of consecutive isotopic intensities needed": "所需的连续同位素强度的最小数量",
    "The minimum value of the score axis": "得分轴的最小值",
    "How RIAs are collected": "如何收集相对丰度指数（RIAs）",
    "Correlation threshold for reporting low RIA patterns": "报告低RIA模式的相关性阈值",
    "input file containing the database in FASTA format": "输入文件包含数据库的FASTA格式",
    "input file containing the identified peptides": "包含已识别肽段的输入文件",
    "Protein inference results for the identification runs that were used to annotate the input": "用于注释输入的鉴定运行的蛋白质推断结果",
    "input file containing the experimental design": "包含实验设计信息的输入文件",
    "Calculate protein abundance from this number of proteotypic peptides (most abundant first; '0' for all)": "计算蛋白质丰度，基于这些蛋白质特异性肽段的数量（首先最丰富的；'0'表示全部）",
    "Averaging method used to compute protein abundances from peptide abundances": "用于从肽段丰度计算蛋白质丰度的平均方法",
    "Include results for proteins with fewer proteotypic peptides than indicated by 'top' (no effect if 'top' is 0 or 1)": "包含具有少于“top”参数指示的蛋白质特异肽段数量的结果（如果“top”为0或1，则无效果）",
    "Distinguish between fraction and charge states of a peptide": "区分肽段的分数和电荷状态",
    "Pre-process identifications with greedy resolution of shared peptides based on the protein group probabilities": "使用基于蛋白质组概率的贪婪策略解决共享肽段问题，对鉴定结果进行预处理",
    "Add the log2 ratios of the abundance values to the output": "将丰度值的log2比率添加到输出中",
    "Add the log2 ratios for a triple SILAC experiment to the output": "为三重SILAC实验将log2比率添加到输出中",
    "Additional options for consensus maps (and identification results comprising multiple runs)": "共识图的额外选项（以及包含多次运行的鉴定结果）",
    "Scale peptide abundances so that medians of all samples are equal": "将肽段丰度进行缩放，使得所有样本的中位数相等。",
    "Use the same peptides for protein quantification across all samples": "在所有样本中使用相同的肽段进行蛋白质定量",
    "Character(s) used to separate fields; by default, the 'tab' character is used": "用于分隔字段的字符；默认情况下，使用制表符字符",
    "Method for quoting of strings: 'none' for no quoting, 'double' for quoting with doubling of embedded quotes": "字符串引号的方法：'none' 表示不使用引号，'double' 表示使用嵌入引号的加倍来进行引号",
    "If 'quoting' is 'none', used to replace occurrences of the separator in strings before writing": "如果“引用”设置为“无”，则在写入之前用于替换字符串中的分隔符出现的情况",
    "A map may contain multiple features with both identical (possibly modified": "一张地图可能包含多个具有相同（可能经过修改的）特征",
    "Minimal signal to noise ratio for a peak to be picked": "峰值被挑选的最小信噪比",
    "Percentage of the maximum height that the raw data points must exceed to be taken into account for the calculation of the centroid": "原始数据点必须超过最大高度的百分比，以便在计算质心时被考虑",
    "Approximate fwhm of the peaks": "峰值的近似全宽半高",
    "Flag if the average peak width shall be estimated": "标记是否估计平均峰宽",
    "Factor that calculates the minimal fwhm value from the peak_width": "计算峰宽的最小全宽半高（FWHM）值的因素",
    "Factor that calculates the maximal fwhm value from the peak_width": "计算峰宽最大全宽半高（FWHM）值的因素",
    "maximal number of iterations for the fitting step": "拟合步骤的最大迭代次数",
    "penalty term for the fitting of the position:If it differs too much from the initial one it can be penalized": "位置拟合的惩罚项：如果它与初始值相差太多，可以对其进行惩罚",
    "penalty term for the fitting of the left width:If the left width differs too much from the initial one during the fitting it can be penalized": "拟合左侧宽度的惩罚项：如果在拟合过程中左侧宽度与初始宽度相差太大，可以对其进行惩罚",
    "penalty term for the fitting of the right width:If the right width differs too much from the initial one during the fitting it can be penalized": "拟合合适宽度时的惩罚项：如果在拟合过程中合适的宽度与初始宽度相差太大，可以对其进行惩罚",
    "penalty term for the fitting of the intensity (only used in 2D Optimization):If it gets negative during the fitting it can be penalized": "用于强度拟合的惩罚项（仅在2D优化中使用）：如果在拟合过程中变为负值，可以对其进行惩罚",
    "mz tolerance for cluster construction": "聚类构建的质量误差容忍度",
    "maximal peak distance in mz in a cluste": "在簇中最大峰值距离（m/z）",
    "Minimal peak intensity": "最小峰值强度",
    "Minimal peak intensity for MS/MS peaks": "MS/MS 峰的最小峰强度",
    "minimal correlation of a peak and the raw signal": "峰值与原始信号的最小相关性",
    "noise level for the search of the peak endpoints": "搜索峰值端点时的噪声水平",
    "search radius for the search of the maximum in the signal after a maximum in the cwt was found": "在连续小波变换（CWT）中找到最大值后，搜索信号中最大值的搜索半径",
    "Spacing of the CWT": "小波变换（CWT）的间距",
    "If you want heavily overlapping peaks to be separated set this value to \"true\"": "如果您希望将高度重叠的峰分离，将此值设置为“true”",
    "If the symmetry of a peak is smaller than asym_thresholds it is assumed that it consists of more than one peak and the deconvolution procedure is started": "如果一个峰的对称性小于asym_thresholds，那么假定它由多个峰组成，并开始去卷积过程。",
    "1/left_width is the initial value for the left width of the peaks found in the deconvolution step": "1/left_width 是在去卷积步骤中找到的峰值左侧宽度的初始值",
    "1/right_width is the initial value for the right width of the peaks found in the deconvolution step": "1/right_width 是在去卷积步骤中找到的峰值右侧宽度的初始值",
    "Initial scaling of the cwt used in the separation of heavily overlapping peaks": "在分离重叠严重的峰时使用的连续小波变换（CWT）的初始缩放",
    "If the FWHM of a peak is higher than 'fwhm_thresholds' it is assumed that it consists of more than one peak and the deconvolution procedure is started": "如果峰的全宽半高（FWHM）高于'fwhm_thresholds'，则假定它由多个峰组成，并开始解卷积过程",
    "if the absolute error gets smaller than this value the fitting is stopped": "如果绝对误差小于这个值，拟合就会停止",
    "if the relative error gets smaller than this value the fitting is stopped": "如果相对误差小于这个值，拟合就会停止",
    "penalty term for the fitting of the peak position:If the position changes more than 0.5Da during the fitting it can be penalized as well as discrepancies of the peptide mass rule": "峰位拟合的惩罚项：如果峰位在拟合过程中变化超过0.5Da，可以对这种变化以及肽质量规则的不符之处进行惩罚。",
    "penalty term for the fitting of the intensity:If it gets negative during the fitting it can be penalized": "强度拟合的惩罚项：如果在拟合过程中变为负数，可以对其进行惩罚。",
    "penalty term for the fitting of the left width:If the left width gets too broad or negative during the fitting it can be penalized": "拟合左侧宽度的惩罚项：如果左侧宽度在拟合过程中变得太宽或为负，则可以对其进行惩罚",
    "penalty term for the fitting of the right width:If the right width gets too broad or negative during the fitting it can be penalized": "拟合合适宽度的惩罚项：如果在拟合过程中合适的宽度变得过大或为负，则可以对其进行惩罚。",
    "multiplier for stdev": "标准差乘数",
    "Write additional information about the picked peaks (maximal intensity, left and right area...) into the mzML-file": "将选定的峰（最大强度、左右区域等）的附加信息写入mzML文件中",
    "first input file": "第一个输入文件",
    "second input file": "第二个输入文件",
    "acceptable relative erro": "可接受的相对误差",
    "acceptable absolute difference": "可接受的绝对差异",
    "set verbose level:": "设置详细级别：",
    "tabulator width, used for calculation of column numbers": "制表符宽度，用于计算列数",
    "number of first column, used for calculation of column numbers": "第一列的数量，用于计算列号",
    "Lines containing one of these strings are skipped": "包含这些字符串之一的行将被跳过",
    "Lines where one file contains one string and the other file another string are skipped": "当一个文件包含一个字符串，另一个文件包含另一个字符串的行被跳过时",
    "Signal to noise value, each peak is required to be above this value (turn off by setting it to 0.0)": "信噪比值，每个峰值都需要高于这个值（将其设置为0.0以关闭）",
    "Expected peak width half width in Dalton - peaks will be extended until this half width is reached (even if the intensitity is increasing)": "预期峰宽半宽，以道尔顿为单位 - 峰将扩展到达到这个半宽（即使强度在增加）",
    "Difference between peaks in multiples of the minimal difference to continue": "峰值之间最小差异的倍数差异",
    "Bin count for the Signal to Noise estimation": "信噪比估计的分箱计数",
    "Nr of iterations to perform (how many times the peaks are re-centered)": "执行迭代的次数（峰值重新居中的次数）",
    "Window length for the Signal to Noise estimation": "信噪比估计的窗口长度",
    "Delete peaks where the spacing is larger than the peak width (should be set to true to avoid artefacts)": "删除峰值间隔大于峰值宽度的峰值（应设置为true以避免伪影）",
    "Only do MS1": "仅执行 MS1",
    "Delete meta data about peak width": "删除关于峰宽的元数据",
    "Feature decharging algorithm section": "特征去电荷算法部分",
    "Minimal possible charge": "最小可能电荷",
    "Maximal possible charge": "最大可能电荷",
    "Maximal range of charges for a single analyte": "单个分析物的最大电荷范围",
    "Try different values of charge for each feature according to the above settings ('heuristic' [does not test all charges, just the likely ones] or 'all' ), or leave feature charge untouched ('feature')": "根据上述设置（'启发式'[不测试所有电荷，只测试可能的电荷]或'所有'），为每个特征尝试不同的电荷值，或保持特征电荷不变（'特征'）",
    "Maximum allowed RT difference between any two features if their relation shall be determined": "如果需要确定两个特征之间的关系，则它们之间允许的最大RT差异",
    "Maximum allowed RT difference between between two co-features, after adduct shifts have been accounted for (if you do not have any adduct shifts, this value should be equal to 'retention_max_diff', otherwise it should be smaller!)": "最大允许的两个共特征之间的保留时间（RT）差异，在考虑了加成位移后（如果您没有任何加成位移，这个值应该等于'retention_max_diff'，否则应该更小！）",
    "Maximum allowed mass difference [in Th] for a single feature": "单个特征的最大允许质量差异（以道尔顿为单位）",
    "Adducts used to explain mass differences in format: 'Element:Charge(+/-):Probability[:RTShift[:Label]]'": "用于解释质量差异的加合物，格式为：'元素：电荷(+/-):概率[:保留时间偏移[:标签]]'",
    "Maximal number of neutral adducts(q=0) allowed": "允许的最大中性加合物数量（q=0）",
    "Maximum count of the least probable adduct (according to 'potential_adducts' param) within a charge variant": "在电荷变体中，根据“potential_adducts”参数，最不可能的加成物的最大计数",
    "Minimum overlap of the convex hull' RT intersection measured against the union from two features (if CHs are given)": "最小重叠的凸包' RT 交集与两个特征（如果给出 CH）的并集的度量",
    "Enable the intensity filter, which will only allow edges between two equally charged features if the intensity of the feature with less likely adducts is smaller than that of the other feature": "启用强度过滤器，该过滤器只允许两个电荷相同的特征之间的边缘，如果不太可能有加合物的特征的强度小于另一个特征的强度。",
    "Enable negative ionization mode": "启用负离子模式",
    "Label of map in output consensus file where all features are put by default": "输出共识文件中默认放置所有特征的地图标签",
    "Amount of debug information given during processing": "在处理过程中给出的调试信息量",
    "Number of parts to split into (takes precedence over 'size' if set)": "要分割成的部分数（如果设置，则优先于“大小”）",
    "Approximate upper limit for resulting file sizes (in 'unit')": "结果文件大小的近似上限（以'单位'表示）",
    "Unit for 'size' (base 1024)": "'大小'的单位（基数为1024）",
    "Remove chromatograms, keep only spectra": "移除色谱图，仅保留光谱",
    "Remove spectra, keep only chromatograms": "删除谱图，仅保留色谱图",
    "Input file to convert": "要转换的输入文件",
    "unique ID post-processing for output data": "输出数据的唯一ID后处理",
    "Export the TIC instead of the entire experiment in mzML/mzData/mzXML -> DTA2D conversions": "导出 TIC（总离子流图）而不是在 mzML/mzData/mzXML -> DTA2D 转换中导出整个实验。",
    "Use a more compact format when writing MGF (no zero-intensity peaks, limited number of decimal places)": "使用更紧凑的格式来写入MGF文件（没有零强度峰，限制小数位数）",
    "[mzXML output only] Make sure that MaxQuant can read the mzXML and set the msManufacturer to 'Thermo Scientific'": "[mzXML输出仅]确保MaxQuant能够读取mzXML，并将msManufacturer设置为'Thermo Scientific'",
    "[mzML output only] Make sure that TPP parsers can read the mzML and the precursor ion m/z in the file (otherwise it will be set to zero by the TPP)": "[mzML 输出仅] 确保 TPP 解析器能够读取 mzML 文件和文件中的前体离子 m/z（否则 TPP 会将其设置为零）",
    "[mzML output only] Assumes that the provided spectra represent data in SRM mode or targeted MS1 mode and converts them to chromatogram data": "[mzML 输出仅] 假设所提供的谱图代表 SRM 模式或目标化 MS1 模式的数据，并将其转换为色谱图数据",
    "[mzML output only] How to store ion mobility scans (none: no change in format; multiple_spectra: store each IM frame as multiple scans (one per drift time value); concatenated: store whole frame as single scan with IM values in a FloatDataArray": "[mzML输出仅]如何存储离子迁移率扫描（none:格式无变化；multiple_spectra:将每个IM帧存储为多个扫描（每个漂移时间值一个）；concatenated:将整个帧存储为单个扫描，IM值在FloatDataArray中）",
    "Append an index when writing mzML or mzXML files": "在写入mzML或mzXML文件时添加索引",
    "Use numpress compression to achieve optimally small file size using linear compression for m/z domain and slof for intensity and float data arrays (attention: may cause small loss of precision; only for mzML data)": "使用numpress压缩来实现文件大小的最优压缩，使用线性压缩用于m/z域，slof用于强度和浮点数据数组（注意：可能会导致小的精度损失；仅限mzML数据）",
    "Desired (absolute) m/z accuracy for lossy compression": "期望的（绝对）m/z精度用于有损压缩",
    "Whether to process the file on the fly without loading the whole file into memory first (only for conversions of mzXML/mzML to mzML)": "是否在不先将整个文件加载到内存中的情况下即时处理文件（仅适用于将mzXML/mzML转换为mzML）",
    "Disables vendor peak picking for raw files": "禁用原始文件的供应商峰值选择",
    "The number of peaks to keep": "保留峰值的数量",
    "Input file with peptide sequences and optionally charge numbers (mutually exclusive to 'in_seq')": "输入包含肽段序列的文件，以及可选的电荷数（与'in_seq'互斥）",
    "List of peptide sequences (mutually exclusive to 'in')": "肽序列列表（与“in”互斥）",
    "Output format ('list': human-readable list, 'table': CSV-like table, 'mass_only': mass values only, 'mz_only': m/z values only)": "输出格式 ('list': 可读列表, 'table': 类似CSV的表格, 'mass_only': 仅质量值, 'mz_only': 仅m/z值)",
    "Compute average (instead of monoisotopic) peptide masses": "计算平均肽质量（而不是单一同位素肽质量）",
    "Intensity threshold, peaks below this threshold are discarded": "强度阈值，低于此阈值的峰值将被丢弃",
    "File to use as reference (same file format as input files required)": "用作参考的文件（与输入文件相同的文件格式所需）",
    "The maximal number of peaks/features to be considered per map": "每个图考虑的最大峰值/特征数量",
    "Maximum of m/z deviation of corresponding elements in different maps": "不同地图中对应元素的 m/z 偏差的最大值",
    "Within each of the two maps, the pairs considered for pose clustering must be separated by at least this fraction of the total elution time interval": "在两个映射中的每一个中，考虑姿态聚类的配对必须至少被总洗脱时间间隔的这个分数分隔开",
    "Maximum number of elements considered in each map (selected by intensity)": "每个映射中考虑的元素的最大数量（按强度选择）",
    "The scaling of the retention time interval is being hashed into buckets of this size during pose clustering": "在姿态聚类期间，保留时间间隔的缩放被散列到这个大小的桶中",
    "The shift at the lower (respectively, higher) end of the retention time interval is being hashed into buckets of this size during pose clustering": "在保留时间间隔的较低（或较高）端的偏移量在姿势聚类期间被散列到这个大小的桶中",
    "Maximal shift which is considered during histogramming (in seconds)": "在直方图化过程中考虑的最大偏移量（以秒为单位）",
    "Maximal scaling which is considered during histogramming": "在直方图化过程中考虑的最大缩放",
    "[DEBUG] If non-empty, base filename where hash table buckets will be dumped to": "[调试] 如果非空，哈希表存储桶将被转储到的基础文件名",
    "[DEBUG] If non-empty, base filename where the individual hashed pairs will be dumped to (large!)": "[调试] 如果非空，将个别哈希对转储到的基础文件名（大！）",
    "Only link features whose distance to the second nearest neighbors (for both sides) is larger by 'second_nearest_gap' than the distance between the matched pair itself": "仅连接那些与第二个最近邻（对于两侧）的距离比匹配对本身之间的距离大 'second_nearest_gap' 的特征",
    "Never link features that are annotated with different peptides (features without ID's always match; only the best hit per peptide identification is considered)": "不要将用不同肽段注释的特征（没有ID的特征总是匹配的；只考虑每个肽段鉴定的最佳命中）进行链接。",
    "false [default]: pairing requires equal charge state (or at least one unknown charge '0'); true: Pairing irrespective of charge state": "false [默认值]：配对需要相同的电荷状态（或者至少有一个未知电荷'0'）；true：不考虑电荷状态进行配对",
    "true [default]: pairing requires equal adducts (or at least one without adduct annotation); true: Pairing irrespective of adducts": "默认情况下 [true]：配对需要相等的加成物（或者至少一个没有加成物注释）；true：不考虑加成物的配对",
    "Distance component based on RT differences": "基于保留时间差异的距离成分",
    "Never pair features with a larger RT distance (in seconds)": "不要将特征配对具有更大的保留时间距离（以秒为单位）",
    "Normalized RT differences ([0-1], relative to 'max_difference') are raised to this power (using 1 or 2 will be fast, everything else is REALLY slow)": "将标准化的相对丰度差异（范围[0-1]，相对于'max_difference'）提高到此幂（使用1或2会很快，其他任何值都会非常慢）",
    "Final RT distances are weighted by this facto": "最终的RT距离会根据这个因素进行加权",
    "Distance component based on m/z differences": "基于 m/z 差异的距离分量",
    "Never pair features with larger m/z distance (unit defined by 'unit')": "不要将特征与更大的 m/z 距离（单位由 'unit' 定义）配对",
    "Unit of the 'max_difference' paramete": "'max_difference' 参数的单位",
    "Normalized ([0-1], relative to 'max_difference') m/z differences are raised to this power (using 1 or 2 will be fast, everything else is REALLY slow)": "归一化（[0-1]，相对于'm/z差异最大值'）的m/z差异被提高到这个幂（使用1或2会很快，其他任何值都会非常慢）",
    "Final m/z distances are weighted by this facto": "最终的 m/z 距离会根据这个因素进行加权",
    "Distance component based on differences in relative intensity (usually relative to highest peak in the whole data set)": "基于相对强度差异的距离分量（通常相对于整个数据集中最高的峰）",
    "Differences in relative intensity ([0-1]) are raised to this power (using 1 or 2 will be fast, everything else is REALLY slow)": "相对强度（[0-1]）的差异被提升到这个幂（使用1或2会很快，其他任何值都会非常慢）",
    "Final intensity distances are weighted by this facto": "最终的强度距离通过这个因子加权",
    "Log-transform intensities": "对数转换强度",
    "Defines the qp cv accession of the qp to which the table/image is attached": "定义与表格/图像关联的qp的qp cv访问号",
    "Defines the cv accession of the attachment": "定义附件的cv访问号",
    "The file that defined the run under which the qp for the attachment is aggregated as mzML file": "定义运行的文件，该文件将附件的q\\_p聚合为mzML文件",
    "If no file for the run was given (or if the target qp is contained in a set), at least a name of the target run/set containing the the qp for the attachment has to be given": "如果未指定运行的文件（或如果目标qp包含在集合中），则至少需要指定包含附件的qp的目标运行/集合的名称。",
    "If a plot image is to be attached to a qp, this has to be specified here": "如果需要将绘图图像附加到qp（量子点），则必须在这里指定",
    "If a table is to be attached to a qp, this has to be specified here": "如果需要将表格附加到查询计划（query plan，简称qp）上，这需要在此指定。",
    "Input: identification results": "输出：鉴定结果",
    "(Currently unused) Input: experimental design": "输入：实验设计",
    "Output type: auto detected by file extension but can be overwritten here": "输出类型：根据文件扩展名自动检测，但可以在这里重写",
    "Additionally calculate the target-decoy FDR on protein-level based on the posteriors": "此外，根据后验概率，在蛋白质水平上计算目标-伪目标错误发现率（FDR）。",
    "Post-process inference output with greedy resolution of shared peptides based on the parent protein probabilities": "使用基于母蛋白质概率的贪婪解析共享肽段的方法对推断输出进行后处理",
    "Set PSMs with probability higher than this to this maximum probability": "将概率高于这个值的PSMs（肽段匹配）设置为这个最大概率。",
    "Parameters for the Algorithm section": "算法部分的参数",
    "Remove PSMs with probabilities less than this cutoff": "删除概率低于此截止值的PSMs",
    "Consider only top X PSMs per spectrum": "只考虑每个谱图的前X个PSM（Peptide-Spectrum Matches，肽段-谱图匹配）",
    "Epifany uses the best PSM per peptide for inference": "Epifany 使用每个肽段的最佳 PSM（Peptide-Spectrum Match）进行推断。",
    "(Experimental:) Update PSM probabilities with their posteriors under consideration of the protein probabilities": "（实验性：）在考虑蛋白质概率的情况下，更新PSM（肽段匹配）概率，使用它们的后验概率。",
    "(Experimental:) Uses the current protein scores as user-defined priors": "（实验性：）使用当前蛋白质得分作为用户定义的先验",
    "Annotates group probabilities for indistinguishable protein groups (indistinguishable by experimentally observed PSMs)": "为无法区分的蛋白质组注释组概率（通过实验观察到的PSMs无法区分）",
    "(Only consensusXML) Also use IDs without associated features for inference?": "（仅限共识XML）是否还使用没有相关特征的ID进行推断？",
    "Model parameters for the Bayesian network": "贝叶斯网络的模型参数",
    "Protein prior probability ('gamma' parameter)": "蛋白质先验概率（'gamma' 参数）",
    "Peptide emission probability ('alpha' parameter)": "肽段发射概率（'alpha' 参数）",
    "Spurious peptide identification probability ('beta' parameter)": "错误肽段鉴定概率（'beta' 参数）",
    "Peptide prior probability (experimental, should be covered by combinations of the other params)": "肽段先验概率（实验性的，应该由其他参数的组合覆盖）",
    "Regularize the number of proteins that produce a peptide together (experimental, should be activated when using higher p-norms)": "规范化同时产生一个肽段的蛋白质数量（实验性的，在使用更高的p-范数时应激活）",
    "Uses information from different peptidoforms also across runs (automatically activated if an experimental design is given!)": "使用不同肽形式的信息，也可以跨运行（如果给出了实验设计，则自动激活！）",
    "Settings for the loopy belief propagation algorithm": "循环信念传播算法的设置",
    "(Not used yet) How to pick the next message: priority = based on difference to last message (higher = more important)": "（尚未使用）如何选择下一条消息：优先级 = 基于与上一条消息的差异（差异越大 = 越重要）",
    "Initial threshold under which MSE difference a message is considered to be converged": "初始阈值，在此阈值下，均方误差差异被认为已经收敛",
    "Initial value for how strongly should messages be updated in each step": "每一步中消息更新的初始强度值",
    "(Usually auto-determined by estimated but you can set a hard limit here)": "（通常由估计自动确定，但您可以在这里设置一个硬性限制）",
    "P-norm used for marginalization of multidimensional factors": "用于多维因素边缘化的P-范数",
    "Settings for the parameter optimization": "参数优化的设置",
    "How important is target decoy AUC vs calibration of the posteriors": "目标-诱饵AUC与后验校准的重要性如何",
    "Use (D+1)/(T) instead of (D+1)/(T+D) for parameter estimation": "使用 (D+1)/(T) 而不是 (D+1)/(T+D) 进行参数估计",
    "Use a regularized FDR for proteins without unique peptides": "对于没有唯一肽段的蛋白质，使用规范化的FDR（假阳性率）。",
    "Use (D+1)/(T) instead of (D+1)/(T+D) for reporting protein FDRs": "使用 (D+1)/(T) 而不是 (D+1)/(T+D) 来报告蛋白质FDR值",
    "Use picked protein FDRs": "使用选定的蛋白质FDRs",
    "If using picked protein FDRs, which decoy string was used": "如果使用选定的蛋白质FDRs（假阳性率），使用了哪个诱饵字符串？",
    "If using picked protein FDRs, was the decoy string a prefix or suffix": "如果使用选定的蛋白质FDRs（假阳性率），那么诱饵字符串是前缀还是后缀？",
    "Set PSMs with probability lower than this to this minimum probability": "将概率低于此值的PSMs设置为此最小概率",
    "Input mzML file": "输入mzML文件",
    "Method of merging which should be used": "要使用的合并方法",
    "Algorithm section for merging spectra": "合并谱图的算法部分",
    "minimum m/z distance for two data points (profile data) or peaks (centroided data) to be considered distinct": "两个数据点（轮廓数据）或峰（质心化数据）被认为是不同的所需的最小 m/z 距离",
    "Unit in which the distance between two data points or peaks is given": "两个数据点或峰值之间的距离所表示的单位",
    "Sort blocks by <?> before merging them (useful for precursor order)": "在合并它们之前按 <?> 对块进行排序（对于前体顺序很有用）",
    "Spectrum type of the MS level to be averaged": "要平均的质谱级别的谱图类型",
    "Average spectra of this level": "这个级别的平均光谱",
    "FWHM of Gauss curve in seconds to be averaged ove": "高斯曲线的半高宽（FWHM）以秒为单位，需要被平均",
    "Intensity cutoff for Gaussian": "高斯强度截止值",
    "RT range to be averaged ove": "RT范围需要被平均",
    "Unit for RT range": "RT范围的单位",
    "Merge spectra of this level": "合并此级别的谱图",
    "Maximum number of scans to be summed up": "最大扫描次数总和",
    "Maximum RT size of the block in seconds (0.0 = no size restriction)": "块的最大RT大小（以秒为单位）（0.0 =无大小限制）",
    "Max RT distance of the precursor entries of two spectra to be merged in [s]": "两个谱图的前体条目合并的最大保留时间距离 [秒]",
    "input file with positive examples": "带有正例的输入文件",
    "input file with negative examples": "带有负样本的输入文件",
    "the penalty parameter of the svm": "支持向量机（SVM）的惩罚参数",
    "the type of the svm (NU_SVC or C_SVC)": "支持向量机（SVM）的类型（NU_SVC 或 C_SVC）",
    "the nu parameter [0..1] of the svm (for nu-SVR)": "支持向量机（SVM）的nu参数（取值范围为0到1，用于nu-SVR）",
    "the kernel type of the svm": "支持向量机（SVM）的核类型",
    "the degree parameter of the kernel function of the svm (POLY kernel)": "支持向量机（SVM）中多项式（POLY）核函数的度参数",
    "length of the POBK": "POBK的长度",
    "k_mer length of the POBK": "k_mer 长度的 POBK",
    "sigma of the POBK": "POBK的σ",
    "quantity of positive samples for training (randomly chosen if smaller than available quantity)": "用于训练的阳性样本数量（如果小于可用数量，则随机选择）",
    "if the input sets are redundant and the redundant peptides should occur more than once in the training set, this flag has to be set": "如果输入集是冗余的，并且冗余的肽段在训练集中应该出现不止一次，那么必须设置这个标志。",
    "if the step sizes should be interpreted additively (otherwise the actual value is multiplied with the step size to get the new value": "如果步长应该被解释为加性（否则实际值会与步长相乘以获得新值）",
    "Parameters for the grid search / cross validation:": "网格搜索/交叉验证的参数：",
    "Has to be set if the cv should be skipped and the model should just be trained with the specified parameters": "如果需要跳过cv（交叉验证），而仅使用指定的参数训练模型，则必须设置此参数",
    "number of runs for the CV": "交叉验证的运行次数",
    "number of CV partitions": "CV 分区的数量",
    "starting point of degree": "度数的起始点",
    "step size point of degree": "步长 点的度数",
    "stopping point of degree": "终止点的度数",
    "starting point of c": "C语言的起始点",
    "step size of c": "c的步长",
    "stopping point of c": "终止点的C",
    "starting point of nu": "起始点的核",
    "step size of nu": "nu的步长",
    "stopping point of nu": "终止点的核",
    "starting point of sigma": "σ的起始点",
    "step size of sigma": "σ的步长",
    "stopping point of sigma": "σ的停止点",
    "Decoy generation method": "诱饵生成方法",
    "decoy tag": "诱饵标签",
    "Whether to switch terminal K and R (to achieve different precursor mass)": "是否交换终端 K 和 R（以实现不同的前体质量）",
    "Minimum fraction of decoy / target peptides and proteins": "最小诱饵/目标肽段和蛋白质的比例",
    "Number of decoys the algorithm should generate (if unequal to 1, the algorithm will randomly select N peptides for decoy generation)": "算法应生成的诱饵数量（如果不等于1，则算法将随机选择N个肽用于诱饵生成）",
    "shuffle: maximum attempts to lower the amino acid sequence identity between target and decoy for the shuffle algorithm": "洗牌：为了降低洗牌算法中目标和诱饵之间的氨基酸序列相似性，最大尝试次数。",
    "shuffle: target-decoy amino acid sequence identity threshold for the shuffle algorithm": "洗牌：洗牌算法的目标-诱饵氨基酸序列相似性阈值",
    "shift: precursor ion MZ shift in Thomson for shift decoy method": "位移：用于位移诱饵方法的前体离子 MZ 位移，以 Thomson 为单位。",
    "shift: fragment ion MZ shift in Thomson for shift decoy method": "位移：片段离子 MZ 位移，以 Thomson 为单位，用于位移诱饵方法",
    "set this flag if decoys should not be appended to targets": "如果不需要将诱饵序列添加到目标序列中，请设置此标志。",
    "input file(s)": "输入文件（们）",
    "output file type": "输出文件类型",
    "If your idXML contains multiple runs, merge them beforehand": "如果您的idXML包含多个运行，请事先合并它们",
    "Additionally calculate the target-decoy FDR on protein-level after inference": "在推断后，在蛋白质水平上额外计算目标-诱饵FDR",
    "If true, adds a map_index MetaValue to the PeptideIDs to annotate the IDRun they came from": "如果为真，则向PeptideIDs添加一个map_index MetaValue以注释它们来自的IDRun",
    "Force merging of disagreeing runs": "强制合并不一致的运行",
    "Minimal number of peptides needed for a protein identification": "蛋白质鉴定所需的最小肽段数",
    "How to aggregate scores of peptides matching to the same protein?": "如何将匹配到同一蛋白质的肽段分数进行聚合？",
    "If this is true, different charge variants of the same peptide sequence count as individual evidences": "如果这是真的，相同肽序列的不同电荷变体将被视为独立的证据",
    "If this is true, different modification variants of the same peptide sequence count as individual evidences": "如果这是真的，那么同一肽段序列的不同修饰变体将被视为独立的证据。",
    "If this is true, shared peptides are used as evidences": "如果这是真的，共享肽段被用作证据",
    "If this is set, peptide counts won't be annotated at the proteins": "如果设置了这个选项，蛋白质上将不会注释肽段计数",
    "If this is true, calculates and annotates indistinguishable protein groups": "如果这是真的，计算并注释不可区分的蛋白质组",
    "If this is true, shared peptides will be associated to best proteins only": "如果这是真的，共享肽段将仅与最佳蛋白质相关联",
    "PQP to TraML: Should legacy TraML IDs be used?": "PQP 到 TraML：是否应该使用传统的 TraML ID？",
    "Output file type -- default: determined from file extension, ambiguous file extensions are interpreted as tsv": "输出文件类型 -- 默认：根据文件扩展名确定，不明确的文件扩展名被视为tsv",
    "Used to replace occurrences of the separator in strings before writing, if 'quoting' is 'none'": "用于在写入之前替换字符串中分隔符的出现，如果“引用”是“none”",
    "Suppresses output of identification data": "抑制识别数据的输出",
    "Options for featureXML input files": "featureXML输入文件的选项",
    "Set this flag to write only three attributes: RT, m/z, and intensity": "设置此标志以仅写入三个属性：保留时间（RT）、质荷比（m/z）和强度。",
    "Add columns for meta values which occur with a certain frequency (0-100%)": "添加出现特定频率（0-100%）的元值的列",
    "Options for idXML input files": "idXML 输入文件的选项",
    "Set this flag if you want only protein information from an idXML file": "如果需要从idXML文件中只获取蛋白质信息，请设置此标志",
    "Set this flag if you want only peptide information from an idXML file": "如果您只想从idXML文件中获取肽段信息，请设置此标志",
    "Set this flag if you want to also write indist": "如果你想要同时写入indist（不可区分的序列），就设置这个标志。",
    "If this flag is set the first_dim RT of the peptide hits will also be printed (if present)": "如果设置了这个标志，肽段命中的第一个维度RT（保留时间）也会被打印出来（如果存在）",
    "Add columns for meta values of PeptideID (=spectrum) entries which occur with a certain frequency (0-100%)": "为出现特定频率（0-100%）的PeptideID（=谱图）条目添加元值列。",
    "Add columns for meta values of PeptideHit (=PSM) entries which occur with a certain frequency (0-100%)": "为出现特定频率（0-100%）的PeptideHit（=PSM）条目添加元值列",
    "Add columns for meta values on protein level which occur with a certain frequency (0-100%)": "在蛋白质水平上添加元值的列，这些元值出现的频率为0-100%。",
    "Options for consensusXML input files": "共识XML输入文件的选项",
    "Sorting options can be combined": "排序选项可以组合使用",
    "Apply a stable sort by the covered maps, lexicographically": "应用一个稳定的排序，按照覆盖的映射进行字典序排序",
    "Apply a stable sort by decreasing size": "应用一个按照大小降序的稳定排序",
    "input peak or raw data file": "输入峰值或原始数据文件",
    "input file containing the external calibrant spectra (peak or raw data)": "输入文件包含外部校准物的光谱（峰值或原始数据）",
    "input file containing reference masses of the external calibrant spectra (one per line)": "包含外部校准物质谱图的参考质量的输入文件（每行一个）",
    "File containing TOF conversion constants": "包含TOF转换常数的文件",
    "set this flag, if you have peak data, not raw data (the picking parameters are accessible only from the INI file)": "如果存在峰值数据而非原始数据（仅可通过INI文件访问选择参数），请设置此标志。",
    "Algorithm section for peak picking": "峰值检测的算法部分",
    "Input file containing the transformation description": "输入文件包含转换描述",
    "Minimum value to transform": "最小值转换",
    "Maximum value to transform (if at or below 'min', select a suitable maximum based on the transformation description)": "最大值用于转换（如果等于或低于'min'，则根据转换描述选择一个合适的最大值）",
    "Step size between 'min' and 'max'": "步长在'最小值'和'最大值'之间",
    "Input file, in which the protein/peptide identifications must be tagged with 'file_origin'": "输入文件，其中蛋白质/肽段鉴定必须用'file_origin'进行标记",
    "Do not infer output filenames from spectra_data or file_origin but use the input filename with numeric suffixes": "不要从spectra_data或file_origin推断输出文件名，而是使用输入文件名加上数字后缀",
    "Split different identification runs into separate files": "将不同的鉴定运行拆分到不同的文件中",
    "file to validate": "待验证的文件",
    "schema to validate against": "用于验证的模式",
    "Input Training Spectra in mzML": "输入训练光谱文件，格式为mzML",
    "Input file with corresponding sequences in idXML": "输入包含idXML格式相应序列的文件",
    "Precursor charge state used for model training": "用于模型训练的前体电荷状态",
    "No models are trained but input training files for libSVM command line tools are produced": "没有模型被训练，但是为libSVM命令行工具生成了输入训练文件",
    "The number of intensity bins (for secondary type models)": "强度区间的数量（对于次级类型模型）",
    "The number of regions each spectrum is split to (for secondary type models)": "每个谱图被分割成的区域数量（对于次级类型模型）",
    "The maximum difference between theoretical and experimental parent mass to accept training spectrum": "最大理论值与实验值之间的差异，以接受训练谱的母体质量",
    "The maximum mass error for a peak to the expected mass of some ion type": "峰值与某种离子类型预期质量的最大质量误差",
    "Train simulator for b-ions": "b-离子训练模拟器",
    "Train simulator for y-ions": "y-离子训练模拟器",
    "Train simulator for a-ions": "α粒子训练模拟器",
    "Train simulator for c-ions": "碳离子训练模拟器",
    "Train simulator for x-ions": "x-离子训练模拟器",
    "Train simulator for z-ions": "Z离子训练模拟器",
    "Train simulator for neutral losses of H2O and NH3 for b-ions and y-ions": "中性损失H2O和NH3的b离子和y离子的训练模拟器",
    "Train simulator for doubly charged b-ions": "双电荷b-离子的训练模拟器",
    "Train simulator for double charged y-ions": "双电荷Y离子训练模拟器",
    "Parameters controlling SVM trainig behaviou": "控制支持向量机（SVM）训练行为的参数",
    "Type of the SVC: 0=C_SVC 1=NU_SVC": "支持向量分类器（SVC）的类型：0=C_SVC 1=NU_SVC",
    "Type of the SVR: 0=EPSILON_SVR 1=NU_SVR": "支持向量回归（SVR）的类型：0=Epsilon-SVR 1=Nu-SVR",
    "Apply scaling of feature values": "应用特征值的缩放",
    "Lower bound for scaling": "缩放的下限",
    "Upper bound for scaling": "缩放的上限",
    "n_fold cross validation is performed": "进行n折交叉验证",
    "Perform grid search": "执行网格搜索",
    "Additive step size (if false multiplicative)": "加性步长大小（如果为假，则为乘性）",
    "Parameters for svm - classification of missing/abundant": "支持向量机（SVM）参数 - 缺失/丰富分类",
    "Type of the kernel:  0=LINEAR 1=POLY 2=RBF 3=SIGMOID": "核函数类型：0=线性 1=多项式 2=径向基函数 3=sigmoid",
    "For POLY": "对于多态性",
    "For POLY/RBF/SIGMOID": "对于多项式/径向基函数/sigmoid",
    "Cost of constraint violation": "约束违反的代价",
    "For NU_SVC, ONE_CLASS and NU_SVR": "对于NU_SVC，ONE_CLASS和NU_SVR",
    "Use class balanced SVC training": "使用类别平衡的SVC训练",
    "starting point of gamma": "伽马的起始点",
    "step size point of gamma": "步长 伽马",
    "stopping point of gamma": "伽马的停止点",
    "Parameters for svm - regression of peak intensities": "支持向量机（SVM）参数 - 峰值强度的回归",
    "The epsilon for the loss function in epsilon-SVR": "在epsilon-SVR中用于损失函数的epsilon",
    "starting point of p": "起始点 p",
    "step size point of p": "步长点的 p",
    "stopping point of p": "停止点的 p",
    "input file in mzML format..": "输入文件为mzML格式。",
    "Mascot parameters used for searching": "用于搜索的Mascot参数",
    "Name of the sequence database": "序列数据库的名称",
    "Name of the search type for the query": "查询的搜索类型名称",
    "The enzyme descriptor to the enzyme used for digestion": "用于消化的酶的描述符",
    "Instrument definition which specifies the fragmentation rules": "仪器定义，它指定了碎片化规则",
    "Number of missed cleavages allowed for the enzyme": "允许酶发生错过切割的次数",
    "Tolerance of the precursor peaks": "前体峰的容忍度",
    "Units of the precursor mass tolerance": "前体质量容差单位",
    "Units of the fragment peaks tolerance": "片段峰值公差的单位",
    "Charge states to consider, given as a comma separated list of integers (only used for spectra without precursor charge information)": "考虑的电荷状态，以逗号分隔的整数列表给出（仅用于没有前体电荷信息的谱图）",
    "Taxonomy specification of the sequences": "序列的分类学规范",
    "List of fixed modifications, according to UniMod definitions": "固定修饰列表，根据UniMod定义",
    "Variable modifications given as UniMod definitions": "变量修饰符以UniMod定义给出",
    "Modifications with specificity groups that are used by Mascot and have to be treated specially": "具有特异性组的修饰，这些修饰被Mascot使用，并且需要特别处理",
    "Defines the mass type, either monoisotopic or average": "定义质量类型，可以是单一同位素质量或平均质量",
    "Number of hits which should be returned, if 0 AUTO mode is enabled": "返回命中数，如果为0，则启用自动模式",
    "Sometimes precursor charges are given for each spectrum but are wrong, setting this to 'true' does not write any charge information to the spectrum, the general charge information is however kept": "有时每个谱图都会给出前体电荷，但可能是错误的，将此设置为'true'不会将任何电荷信息写入谱图，但总体电荷信息仍然保留",
    "Set to true if mascot should generate the decoy database": "如果Mascot应该生成诱饵数据库，则设置为true",
    "Sets the title of the search": "设置搜索的标题",
    "Sets the username which is mentioned in the results file": "设置在结果文件中提到的用户名",
    "Sets the email which is mentioned in the results file": "设置在结果文件中提到的电子邮件",
    "Mascot server details": "Mascot 服务器详细信息",
    "Address of the host where Mascot listens": "Mascot监听的主机地址",
    "Port where the Mascot server listens, 80 should be a good guess": "Mascot服务器监听的端口，80应该是一个不错的选择",
    "Path on the server where Mascot server listens, 'mascot' should be a good guess": "服务器上 Mascot 服务器监听的路径，'mascot' 应该是一个不错的选择",
    "Timeout in seconds, after which the query is declared as failed.This is NOT the whole time the search takes, but the time in between two progress steps": "超时时间（以秒为单位），在此之后查询将被宣布为失败。这并不是搜索所需的整个时间，而是两个进度步骤之间的时间。",
    "Boundary for the MIME section": "MIME部分的边界",
    "Flag which enables the proxy usage for the HTTP requests, please specify at least 'proxy_host' and 'proxy_port'": "启用HTTP请求的代理使用的标记，请至少指定'proxy_host'和'proxy_port'",
    "Host where the proxy server runs on": "代理服务器运行的主机",
    "Port where the proxy server listens": "代理服务器监听的端口",
    "Login name for the proxy server, if needed": "如果需要，代理服务器的登录名",
    "Login password for the proxy server, if needed": "代理服务器的登录密码（如果需要）",
    "Flag which should be set 'true' if Mascot security is enabled; also set 'username' and 'password' then": "如果启用了Mascot安全性，则应设置为'true'的标志；然后还应设置'用户名'和'密码'。",
    "Name of the user if login is used (Mascot security must be enabled!)": "如果使用了登录（必须启用Mascot安全性！），则用户的名称",
    "Password of the user if login is used (Mascot security must be enabled!)": "如果使用登录（必须启用Mascot安全性！），则用户的密码",
    "Flag indicating whether you want to send requests to an HTTPS server or not (HTTP)": "标志，指示您是否要将请求发送到 HTTPS 服务器（HTTP）",
    "Adjustable export parameters (passed to Mascot's 'export_dat_2.pl' script)": "可调的导出参数（传递给Mascot的'export_dat_2.pl'脚本）",
    "For use with an external Mascot Percolator (via GenericWrapper): Run the Mascot search, but do not export the results": "用于外部Mascot Percolator（通过GenericWrapper）：运行Mascot搜索，但不要导出结果",
    "Number of spectra processed in one batch by Mascot (default 50000)": "由Mascot在一批中处理的谱图数量（默认为50000）",
    "Run tool in batch mode for -spectra_files": "批量运行工具，针对\"spectra_files\"参数。",
    "File names(s) of spectra to be searched": "要搜索的光谱文件名",
    "File type of output output_files (Output files. Make sure to specify one output file for each input file)": "输出文件类型输出文件（输出文件。确保为每个输入文件指定一个输出文件）",
    "Specify library file": "指定库文件",
    "The sequence database": "序列数据库",
    "Specify type of sequence database": "指定序列数据库的类型",
    "Only search a subset of the query spectra in the search file": "仅搜索查询文件中查询谱的子集",
    "m/z (in Th) tolerance within which candidate entries are compared to the query": "m/z（以Th单位）的容忍度，在此容忍度内将候选条目与查询进行比较",
    "Use isotopically averaged mass instead of monoisotopic mass": "使用同位素平均质量而不是单一同位素质量",
    "Search library spectra of all charge states": "搜索所有电荷状态下的库光谱",
    "Specify name of user-defined modifications file": "指定用户定义修饰文件的名称",
    "Input file (any xml file)": "输入文件（任何 XML 文件）",
    "Mapping file which is used to semantically validate the given XML file against this mapping file (see 'share/OpenMS/MAPPING' for templates)": "用于语义验证给定XML文件的映射文件（请参见'share/OpenMS/MAPPING'中的模板）",
    "Run tool in batch mode for -cv": "批量运行工具，参数为 -cv",
    "Controlled Vocabulary files containg the CV terms (if left empty, a set of default files are used)": "受控词汇文件包含CV术语（如果留空，则使用一组默认文件）",
    "searchable spectral library (MSP format)": "可搜索的光谱库（MSP格式）",
    "function for similarity comparison": "相似性比较函数",
    "Precursor (Parent Ion) Options": "前体（母离子）选项",
    "Width of precursor mass tolerance window": "前体质量容差窗口宽度",
    "Minimum precursor charge to be considered": "考虑的最小前体电荷",
    "Maximum precursor charge to be considered": "考虑的最大前体电荷",
    "Corrects for mono-isotopic peak misassignments": "纠正单同位素峰误分配",
    "Fragments (Product Ion) Options": "片段（产物离子）选项",
    "Fragment mass tolerance": "片段质量容差",
    "Reporting Options": "报告选项",
    "Maximum number of top scoring hits per spectrum that are reported": "每个谱图报告的得分最高的命中数的最大值",
    "All peaks of a query spectrum with intensities below <threshold> will be zeroed": "所有查询谱图中强度低于<阈值>的峰值将被置零。",
    "required minimum number of peaks for a query spectrum": "查询谱图所需的最小峰值数量",
    "Use only the top <number> of peaks": "仅使用前<number>个峰值",
    "Remove all peaks which are lower than 1/<number> of the highest peaks": "移除所有峰值低于最高峰值的1/<number>的峰。",
    "Modifications Options": "修改选项",
    "Fixed modifications, specified using UniMod (www.unimod.org) terms": "固定修饰，使用UniMod（www.unimod.org）术语指定",
    "Variable modifications, specified using UniMod (www.unimod.org) terms": "变量修饰，使用UniMod（www.unimod.org）术语指定",
    "Maximum number of residues carrying a variable modification per candidate peptide": "每个候选肽段可携带的可变修饰残基的最大数量",
    "Feature result file": "特征结果文件",
    "Expected result file": "预期结果文件",
    "Maximum allowed retention time deviation": "最大允许保留时间偏差",
    "Maximum allowed m/z deviation (divided by charge)": "最大允许的m/z偏差（除以电荷）",
    "This Parameter stands for the cost of opening a gap in the Alignment": "这个参数代表在比对中打开一个间隙的成本",
    "This Parameter controls the cost of extension a already open gap": "这个参数控制已经打开的间隙的延伸成本",
    "The Parameter defines the threshold which filtered spectra, these spectra are high potential candidate for deciding the interval of a sub-alignment": "参数定义了过滤光谱的阈值，这些光谱是决定子对齐间隔的高潜力候选者",
    "Defines the numbers of buckets": "定义桶的数量",
    "Defines the percent of numbers of match points which a selected from one bucket": "定义从一个桶中选择的匹配点数的百分比",
    "Defines the score of two spectra if they have no similarity to each othe": "定义两个谱图的分数，如果它们彼此没有相似性",
    "The score function is the core of an alignment": "得分函数是比对的核心",
    "Holds id, peptide, retention time etc": "包含id、肽段、保留时间等",
    " Separator between items. e.g. ,": "项目之间的分隔符。例如，逗号",
    "'true' or 'false' if true every item is enclosed": "如果为 'true'，则每个项目都被包围；如果为 'false'，则不是。",
    "spectra": "质谱图",
    "Optional idXML Input file(s) in the same order as mzML files - for Maracluster Cluster annotation": "可选的 idXML 输入文件（s），其顺序与 mzML 文件相同 - 用于 Maracluster 聚类注释",
    "log(p-value) cutoff, has to be < 0.0": "对数（p值）截止值，必须小于0.0",
    "minimum number of spectra in a cluster for consensus spectra": "在聚类中，用于共识谱图的最小谱图数量",
    "Set verbosity of output: 0=no processing info, 5=all": "设置输出的详细程度：0=不显示处理信息，5=显示所有信息",
    "Precursor monoisotopic mass tolerance": "前体单一同位素质量容差",
    "tolerance_mass_units 0=ppm, 1=Da": "容差质量单位 0=ppm，1=Da",
    "Should decoys be generated?": "是否需要生成诱饵序列？",
    "+/- tolerance for precursor mass": "前体质量的正负容忍度",
    "Unit of fragment m": "片段单位",
    "Annotation Options": "注释选项",
    "Annotations added to each PSM": "为每个PSM添加注释",
    "Peptide Options": "肽段选项",
    "Minimum size a peptide must have after digestion to be considered in the search": "肽段在消化后必须具有的最小尺寸，以被考虑在搜索中",
    "Maximum size a peptide must have after digestion to be considered in the search (0 = disabled)": "在搜索中考虑的肽段在消化后必须具有的最大大小（0 = 禁用）",
    "Number of missed cleavages": "缺失切割数",
    "If set, only peptides that contain this motif (provided as RegEx) will be considered": "如果设置，将只考虑包含此模式（以RegEx提供）的肽段",
    "Input feature map file (featureXML)": "输入特征映射文件（featureXML）",
    "file containing results of identification": "包含鉴定结果的文件",
    "number of precursors to be selected": "要选取的前体数量",
    "Input profile data": "输入数据概要",
    "The preprocessed db is loaded from file, not calculated": "预处理的数据库是从文件中加载的，而不是计算得出的",
    "The preprocessed db is stored": "预处理后的数据库被存储",
    "Simulate the whole LC-MS/MS run": "模拟整个液相色谱-串联质谱（LC-MS/MS）运行过程。",
    "db file": "数据库文件",
    "SVM Model for RTPredict": "支持向量机（SVM）模型用于 RTPredict",
    "SVM Model for PTPredict": "支持向量机（SVM）模型用于PTPredict",
    "the modifications i.e. Carboxymethyl (C)": "这些修饰，即羧甲基（C）",
    "Settings for the compound list creation and rescoring": "化合物列表创建和重新打分的设置",
    "Strategy for precursor ion selection": "前体离子选择策略",
    "Maximal number of iterations": "最大迭代次数",
    "Maximal number of precursors per rt bin": "每个保留时间区间内前体离子的最大数量",
    "Maximal number of precursors per iteration": "每次迭代的最大前体数",
    "Minimal peptide probability": "最小肽概率",
    "If true, precursors are selected sequentially with respect to their RT": "如果为真，前体将根据它们的保留时间（RT）顺序选择",
    "Minimal protein probability for a protein to be considered in the ILP": "最小蛋白质概率，用于考虑蛋白质是否包含在整数线性规划（ILP）中。",
    "Minimal protein probability for a protein to be considered identified": "蛋白质被认为鉴定出的最小蛋白质概率",
    "Minimal pt weight of a precurso": "最小前体点权重",
    "Minimal mz to be considered in protein based LP formulation": "在基于蛋白质的液相色谱（LP）公式中，要考虑的最小m/z值。",
    "Minimal predicted peptide probability of a precurso": "最小预测的前体肽概率",
    "Minimal rt weight of a precurso": "最小保留时间权重",
    "Use peptide rule instead of minimal protein id probability": "使用肽规则而不是最小蛋白质鉴定概率",
    "If use_peptide_rule is true, this parameter sets the minimal number of peptide ids for a protein id": "如果 use_peptide_rule 为真，此参数设置蛋白质 ID 的最小肽段 ID 数量",
    "If use_peptide_rule is true, this parameter sets the minimal probability for a peptide to be safely identified": "如果 use_peptide_rule 为真，此参数设置肽段被安全识别的最小概率",
    "combined ilp: weight for z_i": "组合整数线性规划：z_i的权重",
    "combined ilp: weight for x_j,s*int_j,s": "联合整数规划：x_j,s 的权重为 w_j,s*int_j,s",
    "combined ilp: weight for -x_j,s*w_j,s": "组合整数线性规划：权重为 -x_j,s*w_j,s",
    "flag if detectability * rt_weight shall be scaled to cover all [0,1]": "如果可检测性 * rt_weight 需要被缩放到覆盖所有 [0,1] 的范围，请标记",
    "Flag indicating if intensities shall be scaled to be in [0,1]": "标志，指示是否将强度缩放到[0,1]范围内",
    "The maximal number of precursors per feature": "每个特征的最大前体数",
    "Precursor mass tolerance which is used to query the peptide database for peptides": "前体质量容差，用于查询肽数据库中的肽段",
    "Precursor mass tolerance unit": "前体质量容差单位",
    "Path where the preprocessed database should be stored": "预处理数据库应存储的路径",
    "Path where the predicted rts of the preprocessed database should be stored": "预测的预处理数据库的返回点（rts）应存储的路径",
    "Number of peptides for that the pt and rt are parallelly predicted": "对于肽段，其保留时间和质子化时间是平行预测的数目",
    "Number of allowed missed cleavages": "允许的断裂缺失数量",
    "Taxonomy": "分类学",
    "Flag if peptide sequences should be stored": "如果需要存储肽序列，则标记",
    "Minimal RT in the experiment (in seconds)": "实验中最小的反应时间（以秒为单位）",
    "Maximal RT in the experiment (in seconds)": "实验中最大保留时间（单位：秒）",
    "Time between two consecutive spectra (in seconds)": "两个连续谱之间的时间（以秒为单位）",
    "mean of the gauss curve": "高斯曲线的均值",
    "std of the gauss curve": "高斯曲线的标准差",
    "LP solver type": "线性规划求解器类型",
    "Input file (see below for details)": "输入文件（详见下文）",
    "[idXML input only] Use the monoisotopic mass of the best peptide hit for the m/z position (default: use precursor m/z)": "[idXML 输入专用] 使用最佳肽段命中的单同位素质量来确定 m/z 位置（默认：使用前体 m/z）",
    "This is the name of the input file (RT prediction)": "这是输入文件（RT预测）的名称",
    "input file with positive examples (peptide separation prediction)": "输入包含正例的文件（肽段分离预测）",
    "input file with negative examples (peptide separation prediction)": "包含负样本的输入文件（肽段分离预测）",
    "the type of the svm (NU_SVR or EPSILON_SVR for RT prediction, automatically set": "支持向量机（SVM）的类型（对于保留时间预测，自动设置为NU_SVR或EPSILON_SVR）",
    "the epsilon parameter of the svm (for epsilon-SVR)": "支持向量机（SVM）的epsilon参数（用于epsilon-SVR）",
    "max standard deviation for a peptide to be included (if there are several ones for one peptide string)(median is taken)": "肽段被包含的最大标准差（如果一个肽段字符串有多个）（取中位数）",
    "the time (in seconds) of the gradient (only for RT prediction)": "梯度的时间（以秒为单位）（仅用于保留时间预测）",
    "if set the model will be built for first_dim_rt": "如果设置了模型，将针对 first_dim_rt 构建模型",
    "if the step sizes should be interpreted additively (otherwise the actual value is multiplied": "如果步长应该被解释为加性的（否则实际值将被乘以",
    "Set to enable Cross-Validation or set to true if the model should just be trained with 1 set of specified parameters": "设置启用交叉验证，或者设置为 true，如果模型应该只使用一组指定的参数进行训练",
    "number of runs for the CV (each run creates a new random partition of the data)": "交叉验证（CV）的运行次数（每次运行都会为数据创建一个新的随机分区）",
    "User specified seed list": "用户指定的种子列表",
    "Pseudo RT shift used when": "伪实时（RT）位移在使用时",
    "Settings for the calculation of a score indicating if a peak's intensity is significant in the local environment (between 0 and 1)": "计算得分的设置，该得分表示峰值强度在局部环境中是否显著（介于0到1之间）",
    "Number of bins per dimension (RT and m/z)": "每个维度（保留时间RT和质荷比m/z）的箱数",
    "Settings for the calculation of a score indicating if a peak is part of a mass trace (between 0 and 1)": "用于计算一个分数的设置，该分数指示峰值是否是质量迹线的一部分（介于0和1之间）",
    "Tolerated m/z deviation of peaks belonging to the same mass trace": "容忍属于同一质量迹线峰值的 m/z 偏差",
    "Number of spectra that have to show a similar peak mass in a mass trace": "需要在质量迹线中显示相似的峰质量的谱图数量",
    "Number of consecutive spectra where a high mass deviation or missing peak is acceptable": "允许连续谱图中出现高质量偏差或缺失峰的数量",
    "The maximum slope of mass trace intensities when extending from the highest peak": "当从最高峰延伸时，质量迹线强度的最大斜率",
    "Settings for the calculation of a score indicating if a peak is part of a isotopic pattern (between 0 and 1)": "用于计算一个分数的设置，该分数指示峰值是否是同位素模式的一部分（介于0和1之间）",
    "Lowest charge to search fo": "最低电荷搜索",
    "Highest charge to search fo": "最高电荷数搜索",
    "Tolerated m/z deviation from the theoretical isotopic pattern": "允许的 m/z 偏差来自理论同位素模式",
    "Isotopic peaks that contribute more than this percentage to the overall isotope pattern intensity must be present": "必须存在对整体同位素模式强度贡献超过此百分比的同位素峰。",
    "Isotopic peaks that contribute more than this percentage to the overall isotope pattern intensity can be missing": "贡献超过整体同位素模式强度这个百分比的同位素峰可能缺失",
    "Minimal percental improvement of isotope fit to allow leaving out an optional peak": "最小百分比改进同位素拟合以允许省略一个可选峰",
    "Window width in Dalton for precalculation of estimated isotope distributions": "道尔顿单位中用于预计同位素分布的预计算窗口宽度",
    "Rel": "相对的",
    "Settings that determine which peaks are considered a seed": "确定哪些峰值被认为是种子的设置",
    "Minimum seed score a peak has to reach to be used as seed": "最小种子分数，一个峰值必须达到这个分数才能被用作种子",
    "Settings for the model fitting": "模型拟合的设置",
    "Maximum number of iterations of the fit": "最大迭代次数",
    "Settings for the features (intensity, quality assessment, ...)": "特征设置（强度，质量评估，...）",
    "Feature score threshold for a feature to be reported": "特征得分报告阈值",
    "Minimum isotope fit of the feature before model fitting": "特征模型拟合前的最小同位素拟合",
    "Trace score threshold": "迹线分数阈值",
    "Minimum RT span in relation to extended area that has to remain after model fitting": "最小RT跨度与模型拟合后必须保留的扩展区域之间的关系",
    "Maximum RT span in relation to extended area that the model is allowed to have": "最大保留时间跨度与模型被允许拥有的扩展区域之间的关系",
    "Choose model used for RT profile fitting": "选择用于RT（保留时间）轮廓拟合的模型",
    "Maximum allowed intersection of features": "最大允许的特征交叉",
    "The mass type that is reported for features": "报告的特征的质量类型",
    "Settings for user-specified seeds": "用户指定种子的设置",
    "Allowed RT deviation of seeds from the user-specified seed position": "允许的种子位置与用户指定的种子位置的RT偏差",
    "Allowed m/z deviation of seeds from the user-specified seed position": "允许从用户指定的种子位置的种子质量偏差",
    "Overwrites 'seed:min_score' for user-specified seeds": "覆盖用户指定种子的“seed:min_score”",
    "Precursor mass tolerance (+/- around precursor m/z)": "前体质量容差（围绕前体m/z的正负偏差）",
    "Fragment mass tolerance (+/- around fragment m/z)": "片段质量容忍度（+/-围绕片段m/z）",
    "Maximum size a peptide may have after digestion to be considered in the search": "在搜索中考虑消化后多肽可能具有的最大大小",
    "RNPxl Options": "RNPxl 选项",
    "Oligonucleotide maximum length": "寡核苷酸最大长度",
    "Sequence to restrict the generation of oligonucleotide chains": "限制生成寡核苷酸链的序列",
    "format:  target nucleotide=empirical formula of nucleoside monophosphate": "格式：目标核苷酸=核苷酸单磷酸的经验公式",
    "Restrict which nucleotides can cooccur in a precursor adduct to be able to search both RNA and DNA (Formate": "限制哪些核苷酸可以在前体加合物中共存，以便能够搜索RNA和DNA（格式化",
    "format: source->target e.g. A->A, ..., U->U, U->X": "格式：源->目标 例如 A->A, ..., U->U, U->X",
    "format: 'U' if only U forms cross-links": "格式：'U'，如果只有U形式发生交叉链接",
    "format: [target nucleotide]:[formula] or [precursor adduct]->[fragment adduct formula];[name]:": "格式：[target nucleotide]:[formula] 或 [precursor adduct]->[fragment adduct formula];[name]:",
    "format: empirical formula e.g -H2O, ..., H2O+PO3": "格式：经验公式 例如 -H2O, ..., H2O+PO3",
    "Scoring algorithm used in prescoring (fast: total-loss, slow: all losses)": "在预评分中使用的评分算法（快速：总损失，慢速：所有损失）",
    "Generate decoy sequences and spectra": "生成诱饵序列和谱图",
    "Use this flag if the +152 adduct is expected": "如果预期有+152的加成物，请使用这个标志",
    "Use this flag to filter non-crosslinks by fractional mass": "使用此标志可以通过分数质量过滤非交叉链接",
    "Generate fragment shifts assuming full labeling of carbon": "生成片段偏移，假设碳元素完全标记",
    "Only search cross-links and ignore non-cross-linked peptides": "仅搜索交叉链接并忽略非交叉链接的肽段",
    "Filter precursor that can only correspond to non-crosslinks by mass": "过滤前体，只能对应于非交叉链接的质量",
    "Tolerance used to determine marker ions (Da)": "用于确定标记离子的容忍度（道尔顿）",
    "The size of the m/z window where the peaks are removed, +/- window_size": "m/z窗口的大小，在此窗口内峰值被移除，正负 window_size",
    "If the precursor has no charge set, the default charge is assumed": "如果前体没有设定电荷，将假定默认电荷",
    "Set to 1 if precursor ions of all possible charge states should be removed": "如果应该移除所有可能的电荷状态下的前体离子，则设置为1",
    "Whether NH3 loss peaks from the precursor should be removed": "是否应该移除前体的NH3损失峰",
    "Whether H2O loss peaks from the precursor should be removed": "是否应该从前体中移除H2O损失峰",
    "Reduce the intensities of the precursor and related ions by a given factor (set 'set_to_zero' to 0)": "将前体离子和相关离子的强度减少一个给定的因素（将'set_to_zero'设置为0）",
    "Factor which is used to reduce the intensities if 'reduce_by_factor' is selected": "如果选择了'reduce_by_factor'，则用于降低强度的因素",
    "Reduce the intensities of the precursor and related ions to zero": "将前体离子和相关离子的强度减少到零",
    "Method to read the file": "读取文件的方法",
    "Whether to actually load and decode the binary data (or whether to skip decoding the binary data)": "是否实际加载并解码二进制数据（或是否跳过解码二进制数据）",
    "Keep only peptide hits with variable modifications (as defined in the 'SearchParameters' section of the input file)": "只保留具有可变修饰的肽段命中（如输入文件中“SearchParameters”部分所定义的）",
    "Only peptides matching exactly one protein are kept": "仅保留完全匹配一种蛋白质的肽段",
    "Proteins not referenced by a peptide are retained in the IDs": "未被肽段引用的蛋白质保留在ID中",
    "Remove proteins according to the information in the user parameters": "根据用户参数中的信息移除蛋白质",
    "Peptides not referenced by any protein are deleted in the IDs": "未被任何蛋白质引用的肽在ID中被删除",
    "Filtering by precursor attributes (RT, m/z, charge, length)": "根据前体属性（保留时间，m/z，电荷，长度）进行过滤",
    "Retention time range to extract": "保留时间范围以提取",
    "Mass-to-charge range to extract": "提取的质量-电荷范围",
    "Keep only peptide hits with a sequence length in this range": "只保留序列长度在这个范围内的肽段命中",
    "Keep only peptide hits with charge states in this range": "只保留在这个电荷范围内的肽段命中",
    "Filtering by peptide/protein score": "通过肽段/蛋白质得分过滤",
    "The score which should be reached by a peptide hit to be kept": "肽段命中需要达到的分数才能被保留",
    "The score which should be reached by a protein hit to be kept": "蛋白质命中需要达到的分数以保留",
    "The score which should be reached by a protein group to be kept": "蛋白质组要保留需要达到的分数",
    "Filtering by whitelisting (only peptides/proteins from a given set can pass)": "通过白名单过滤（只有来自给定集合的肽段/蛋白质可以通过）",
    "Filename of a FASTA file containing protein sequences": "包含蛋白质序列的FASTA文件的文件名",
    "All peptides that do not reference at least one of the provided protein accession are removed": "所有不引用至少一个提供的蛋白质访问号的肽段都被移除",
    "Only peptides with the same sequence and modification assignment as any peptide in this file are kept": "仅保留与此文件中任何肽段具有相同序列和修饰分配的肽段",
    "Compare whitelisted peptides by sequence only": "比较仅按序列的白名单肽段",
    "Keep only peptides with sequences that contain (any of) the selected modification(s)": "只保留含有（任意）所选修饰的肽段序列",
    "Filtering by blacklisting (only peptides/proteins NOT present in a given set can pass)": "通过黑名单过滤（只有不在给定集中的肽段/蛋白质才能通过）",
    "All peptides that reference at least one of the provided protein accession are removed": "所有引用至少一个提供的蛋白质访问号码的肽段都被移除",
    "Peptides with the same sequence and modification assignment as any peptide in this file are filtered out": "与此文件中任何肽段具有相同序列和修饰分配的肽段被过滤掉",
    "Compare blacklisted peptides by sequence only": "比较仅按序列的黑名单肽",
    "Remove all peptides with sequences that contain (any of) the selected modification(s)": "移除所有序列中包含（任意）所选修饰的肽段",
    "Remove all peptides with (unmodified) sequences matched by the RegEx": "移除所有未修饰的序列与正则表达式匹配的肽段",
    "This filter option removes peptide hits which are not in the list of in silico peptides generated by the rules specified below": "这个过滤选项移除了不在由下面指定规则生成的in silico肽段列表中的肽段命中",
    "fasta protein sequence database": "FASTA 蛋白质序列数据库",
    "enzyme used for the digestion of the sample": "用于样本消化的酶",
    "Specificity of the filte": "过滤器的特异性",
    "range of allowed missed cleavages in the peptide sequences": "允许肽段序列中未切割的次数范围",
    "Allow methionine cleavage at the N-terminus of the protein": "允许在蛋白质的N-末端切割甲硫氨酸",
    "This filter option removes peptide hits which do not confirm with the allowed missed cleavages specified below": "这个过滤选项移除了不符合下面指定的允许的未切割肽键数量的肽段命中",
    "Filtering by RT predicted by 'RTPredict'": "根据“RTPredict”预测的RT进行过滤",
    "Retention time filtering by the p-value predicted by RTPredict": "保留时间通过 RTPredict 预测的 p 值进行过滤",
    "Retention time filtering by the p-value predicted by RTPredict for first dimension": "保留时间通过 RTPredict 预测的 p 值进行过滤，用于第一维",
    "Filtering by mass erro": "通过质量误差过滤",
    "Filtering by deviation to theoretical mass (disabled for negative values)": "根据与理论质量的偏差进行过滤（对于负值禁用）",
    "Absolute or relative erro": "绝对误差或相对误差",
    "Filtering best hits per spectrum (for peptides) or from proteins": "过滤每个谱图的最佳命中（对于肽段）或来自蛋白质的命中",
    "Keep only the 'n' best spectra": "只保留最好的'n'个光谱",
    "Keep only the 'n' highest scoring peptide hits per spectrum (for n > 0)": "只保留每个谱图中得分最高的'n'个肽段命中（对于n > 0）",
    "Keep only the 'n' highest scoring protein hits (for n > 0)": "只保留得分最高的'n'个蛋白质命中（对于n > 0）",
    "Keep only the highest scoring peptide hit": "只保留得分最高的肽段命中",
    "Peptide hit rank range to extracts": "肽段命中排名范围以提取",
    "Removes duplicated PSMs per spectrum and retains the one with the higher score": "去除每个谱图中重复的PSMs（蛋白质谱匹配），并保留得分较高的那个",
    "Expects a 3-tuple (=3 entries in the list)": "期望一个三元组（列表中有3个条目）",
    "Input file: spectra": "输入文件：质谱图",
    "Input file: sequence database": "输入文件：序列数据库",
    "Input file: pre-digested sequence database": "输入文件：预消化序列数据库",
    "Precursor (parent ion) options": "前体（母离子）选项",
    "Precursor mass tolerance (+/- around uncharged precursor mass)": "前体质量容忍度（+/- 未充电前体质量周围）",
    "Include MS2 spectra with unknown precursor charge - try to match them in any possible charge between 'min_charge' and 'max_charge', at the risk of a higher error rate": "包含未知前体电荷的MS2谱图 - 尝试在 'min_charge' 和 'max_charge' 之间的任何可能电荷中匹配它们，冒着错误率更高的风险",
    "Use average instead of monoisotopic precursor masses (appropriate for low-resolution instruments)": "使用平均质量代替单同位素前体质量（适用于低分辨率仪器）",
    "Consider possible salt adducts (see 'precursor:potential_adducts') when matching precursor masses": "在匹配前体质量时，考虑可能的盐加合物（参见'precursor:potential_adducts'）",
    "Adducts considered to explain mass differences": "考虑的加合物以解释质量差异",
    "Correct for mono-isotopic peak misassignments": "校正单同位素峰误分配",
    "Fragment (Product Ion) Options": "片段（产物离子）选项",
    "Fragment ions to include in theoretical spectra": "包含在理论谱中的片段离子",
    "Modification options": "修改选项",
    "Maximum number of residues carrying a variable modification per candidate oligonucleotide": "每个候选寡核苷酸携带可变修饰的最大残基数",
    "Attempt to resolve ambiguous modifications": "尝试解决模糊的修饰",
    "Oligonucleotide (digestion) options (ignored if 'digest' input is used)": "寡核苷酸（消化）选项（如果使用了'digest'输入，则忽略）",
    "Minimum size an oligonucleotide must have after digestion to be considered in the search": "在消化后，一个寡核苷酸必须具有的最小尺寸，以便在搜索中被考虑",
    "Maximum size an oligonucleotide must have after digestion to be considered in the search, leave at 0 for no limit": "在消化后，一个寡核苷酸必须具有的最大大小才能被考虑在搜索中，将其保留为0表示没有限制",
    "The enzyme used for RNA digestion": "用于RNA消化的酶",
    "Maximum number of top-scoring hits per spectrum that are reported ('0' for all hits)": "每个谱图报告的得分最高的命中数（'0'表示所有命中）",
    "False Discovery Rate options": "错误发现率选项",
    "String used as part of the accession to annotate decoy sequences": "用作注释诱饵序列的接入号的一部分",
    "Cut-off for FDR filtering; search hits with higher q-values will be removed": "FDR过滤的截止值；具有更高q值的搜索命中将被移除",
    "Do not score hits to decoy sequences and remove them when filtering": "不要对诱饵序列进行评分，并在过滤时删除它们",
    "Decharge the MS2 spectra for scoring": "为评分卸下MS2谱图",
    "Input file containing chromatograms (converted mzXML file)": "输入文件包含色谱图（已转换的mzXML文件）",
    "Precursor tolerance when mapping (in Th)": "映射时的前体容忍度（以千分之一为单位）",
    "Product tolerance when mapping (in Th)": "映射时的产品容忍度（以Th为单位）",
    "Allow to map multiple assays to chromatograms and duplicate these chromatograms in the output": "允许将多个检测方法映射到色谱图上，并在输出中复制这些色谱图。",
    "Treat remaining, unmapped chromatograms as an erro": "将剩余未映射的色谱图视为错误",
    "Fragment mass error in Dalton": "片段质量误差在道尔顿",
    "NCBI formatted FASTA files": "NCBI格式的FASTA文件",
    "Minimum precursor ion charge": "最小前体离子电荷",
    "Maximum precursor ion charge": "最大前体离子电荷",
    "number of missed cleavages allowed": "允许的未切割数",
    "maximum number of hits retained for one spectrum": "对于一个谱图保留的最大命中数",
    "the maximum e-value allowed in the hit list": "在命中列表中允许的最大 E 值",
    "The number of pseudocounts to add to each precursor mass bin": "向每个前体质量区间添加的伪计数数量",
    "the minimum number of m/z values a spectrum must have to be searched": "谱图必须具有的最小 m/z 值数量才能进行搜索",
    "scaling of precursor mass tolerance with charge (0 = none, 1= linear)": "前体质量容限与电荷的缩放（0=无，1=线性）",
    "product ion search type, with 0 = monoisotopic, 1 = average, 2 = monoisotopic N15, 3 = exact": "产物离子搜索类型，其中0=单同位素，1=平均，2=单同位素N15，3=精确",
    "precursor ion search type, with 0 = monoisotopic, 1 = average, 2 = monoisotopic N15, 3 = exact": "前体离子搜索类型，其中 0 = 单同位素，1 = 平均，2 = 单同位素 N15，3 = 精确",
    "threshold in Da above which the mass of a neutron should be added in an exact mass search": "在精确质量搜索中，超过Da阈值时，应添加一个中子的质量。",
    "minimum precursor charge to start considering multiply charged products": "最小前体电荷以开始考虑多电荷产物",
    "the fraction of peaks below the precursor used to determine if the spectrum is charge +1": "用于确定谱图是否为电荷+1的前体下方峰值的比例",
    "should charge +1 be determined algorithmically (1=yes)": "是否应该算法性地确定电荷+1（1=是）",
    "how should precursor charges be determined": "前体电荷应该如何确定",
    "set the maximum product charge to search": "设置要搜索的最大产物电荷",
    "minimum size of peptides for no-enzyme and semi-tryptic searches": "无酶切和半酶切搜索中肽段的最小尺寸",
    "maximum size of peptides for no-enzyme and semi-tryptic searches": "无酶和半胰蛋白酶搜索中肽段的最大大小",
    "comma delimited list of id numbers of ions to search": "要搜索的离子ID编号的逗号分隔列表",
    "number of product ions to search": "要搜索的产物离子数量",
    "should first forward": "应该首先前向",
    "should c terminus ions": "应该使用C末端离子",
    "comma delimited list of NCBI taxonomy ids to search (0 = all": "逗号分隔的NCBI分类学ID列表以进行搜索（0 = 全部）",
    "the minimum number of m/z matches a sequence library peptide must have for the hit to the peptide to be recorded": "序列库中的肽段必须至少匹配多少个 m/z 值，才能记录该肽段的命中结果。",
    "number of m/z values corresponding to the most intense peaks that must include one match to the theoretical peptide": "对应最高强度峰的m/z值的数量，必须包含一个与理论肽段匹配的值",
    "the maximum number of mass ladders to generate per database peptide": "每个数据库肽段生成的最大质量梯度数",
    "n-term methionine should not be cleaved": "N-末端的甲硫氨酸不应被切除",
    "evalue threshold to include a sequence in the iterative search, 0 = all": "在迭代搜索中包含序列的e值阈值，0 = 全部",
    "evalue threshold to replace a hit, 0 = only if bette": "evalue 阈值用于替换一个命中项，0 = 只有更好的",
    "evalue threshold to iteratively search a spectrum again, 0 = always": "evalue 阈值用于迭代搜索一个谱图，0 = 总是",
    "Number of spectra to submit in one chunk to OMSSA": "向OMSSA提交的一个块中的谱图数量",
    "Input raw data file": "输入原始数据文件",
    "Run tool in batch mode for -in_header": "批量运行工具，针对 -in_header 参数",
    "[for Waters data only] Read additional information from _HEADER.TXT": "[仅限沃特斯数据] 从 _HEADER.TXT 文件中读取额外信息",
    "Input config file stating where to find signal": "输入配置文件，指明在哪里找到信号",
    "# of scans up & down in RT from highest point for ppm estimation in result": "在结果中，从最高点向上和向下在保留时间（RT）中进行的扫描次数，用于估计每百万分之一（ppm）的误差。",
    "Parameters for automatic detection of injection RT peaks (no need to specify them in 'pos' input file)": "自动检测注射RT峰的参数（无需在'pos'输入文件中指定它们）",
    "Automatically detect injection peaks from TIC and quantify all m/z x RT combinations": "自动检测总离子流图（TIC）中的注射峰，并量化所有质荷比 x 保留时间组合。",
    "Expected full width at half-maximum of each raw RT peak in [s]": "每个原始保留时间峰值的预期半高宽（全宽半高）在 [秒]",
    "S/N threshold for a smoothed raw peak to pass peak picking": "信号/噪声阈值，用于平滑的原始峰值通过峰值选择",
    "Separator character for output CSV file": "输出CSV文件的分隔符",
    "Unit to be used for precursor mass tolerance": "用于前体质量容忍度的单位",
    "If this flag is set, the average mass is used in the precursor mass tolerance": "如果设置了这个标志，前体质量容差中将使用平均质量",
    "Unit to be used for fragment mass tolerance": "片段质量容差的单位",
    "FASTA protein database": "FASTA 蛋白质数据库",
    "The number of charge states that MyriMatch will handle during all stages of the program": "MyriMatch在程序的所有阶段将处理的电荷状态数量",
    "Noise peaks are filtered out by sorting the original peaks in descending order of intensity, and then picking peaks from that list until the cumulative ion current of the picked peaks divided by the total ion current (TIC) is greater than or equal to this paramete": "噪声峰通过将原始峰按强度降序排序，然后从该列表中挑选峰，直到所挑选峰的累积离子电流除以总离子电流（TIC）大于或等于此参数来过滤掉",
    "This parameter sets the maximum number of modified residues that may be in any candidate sequence": "这个参数设置了任何候选序列中可能存在的修饰残基的最大数量",
    "This parameter sets the maximum rank of peptide-spectrum-matches to report for each spectrum": "这个参数设置了每个谱图报告的肽谱匹配的最大排名",
    "This parameter allows the user to control the way peptides are generated from the protein database": "这个参数允许用户控制从蛋白质数据库生成肽段的方式",
    "By default, when generating peptides from the protein database, a peptide must start and end at a valid cleavage site": "默认情况下，在从蛋白质数据库生成肽段时，肽段必须在有效的裂解位点开始和结束",
    "By default, when generating peptides from the protein database, a peptide may contain any number of missed cleavages": "默认情况下，当从蛋白质数据库生成肽段时，一个肽段可能包含任意数量的未切割位点。",
    "When preprocessing the experimental spectra, any spectrum with a precursor mass that is less than the specified mass will be disqualified": "在预处理实验谱图时，任何前体质量小于指定质量的谱图都将被排除在外。",
    "When preprocessing the experimental spectra, any spectrum with a precursor mass that exceeds the specified mass will be disqualified": "在预处理实验谱图时，任何前体质量超过指定质量的谱图都将被排除在外",
    "When digesting proteins, any peptide which does not meet or exceed the specified length will be disqualified": "在消化蛋白质时，任何不符合或不满足指定长度的肽段将被取消资格。",
    "When digesting proteins, any peptide which exceeds this specified length will be disqualified": "在消化蛋白质时，任何超过这个指定长度的肽段将被排除在外",
    "When this parameter is set, then for each peptide bond, an internal calculation is done to estimate the basicity of the b and y fragment sequence": "当设置此参数时，对于每个肽键，将进行内部计算以估计b和y片段序列的碱度",
    "Before scoring any candidates, experimental spectra have their peaks stratified into the number of intensity classes specified by this paramete": "在对任何候选物进行评分之前，实验光谱的峰值会被划分为由该参数指定的强度等级数量。",
    "When stratifying peaks into a specified, fixed number of intensity classes, this parameter controls the size of each class relative to the class above it (where the peaks are more intense)": "当将峰值分层到指定的固定数量的强度类别时，此参数控制每个类别的大小相对于上面类别（其中峰值更强烈）的大小。",
    "This parameter defines a set of isotopes (0 being the instrument-called monoisotope) to try as the monoisotopic precursor m/z": "该参数定义了一组同位素（0表示仪器所称的单同位素）作为单同位素前体m/z进行尝试",
    "Optional set of filters as described in the MyriMatch documentation": "可选的过滤器集合，如 MyriMatch 文档中所述",
    "Ignore wrong parameter names or values": "忽略错误的参数名称或值",
    "C1 value of the normalization": "标准化的C1值",
    "C2 value of the normalization": "C2 标准化的值",
    "Threshold of the Bern et al. normalization": "Bern等人标准化的阈值",
    "Input protein sequences": "输入蛋白质序列",
    "Enzyme to use for digestion (select 'no cleavage' to skip digestion)": "用于消化的酶（选择'no cleavage'跳过消化）",
    "The cleavage model to use for digestion": "用于消化的切割模型",
    "Minimum peptide length after digestion (shorter ones will be discarded)": "消化后最小肽段长度（较短的将被丢弃）",
    "Model threshold for calling a cleavage": "切割位点判定模型阈值",
    "Maximum number of missed cleavages considered": "最大考虑的错切数",
    "Modelling of an RT or CE column": "RT或CE柱的建模",
    "Scale predicted RT's/MT's to given 'total_gradient_time'": "将预测的RT（保留时间）或MT（迁移时间）缩放到给定的'total_gradient_time'（总梯度时间）。",
    "The duration [s] of the gradient": "梯度持续时间 [秒]",
    "Time interval [s] between consecutive scans": "连续扫描之间的时间间隔 [秒]",
    "Start of RT Scan Window [s]": "实时扫描窗口开始时间 [秒]",
    "End of RT Scan Window [s]": "实时扫描窗口结束 [秒]",
    "Random component that simulates technical/biological variation": "模拟技术/生物变异的随机组分",
    "Standard deviation of shift in retention time [s] from predicted model (applied to every single feature independently)": "保留时间偏移的标准差 [s] 从预测模型（独立应用于每个特征）",
    "Global offset in retention time [s] from predicted model": "保留时间的全局偏移量 [秒] 来自预测模型",
    "Global scaling in retention time from predicted model": "保留时间从预测模型中的全局缩放",
    "Distortion of the elution profiles": "洗脱轮廓的畸变",
    "Width of the EGH elution shape": "EGH洗脱形状的宽度",
    "Width of the Exponential Gaussian Hybrid distribution shape of the elution profile": "高斯指数混合分布的宽度形状，用于洗脱轮廓",
    "Random component of the width (set to 0 to disable randomness)": "宽度的随机分量（设置为0以禁用随机性）",
    "Skewness of the EGH elution shape": "EGH洗脱形状的偏斜度",
    "Asymmetric component of the EGH": "EGH的非对称组分",
    "Random component of skewness (set to 0 to disable randomness)": "偏度的随机分量（设为0以禁用随机性）",
    "pH of buffe": "缓冲液的pH值",
    "Exponent Alpha used to calculate mobility": "用于计算迁移率的指数α",
    "Electroosmotic flow": "电渗流",
    "Length of capillary [cm] from injection site to MS": "毛细管长度（从进样点到质谱仪）[厘米]",
    "Total length of capillary [cm]": "毛细管总长度[厘米]",
    "Voltage applied to capillary": "电泳电压",
    "Modelling detectibility enabled": "模型检测已启用",
    "Minimum peptide detectability accepted": "最小肽检测接受度",
    "List of residues (as three letter code) that will be considered during ES ionization": "在ES电离过程中将考虑的残基列表（以三字母代码表示）",
    "List of charged ions that contribute to charge with weight of occurrence (their sum is scaled to 1 internally)": "带电离子列表，这些离子对电荷的贡献按出现权重排列（它们在内部被缩放到1）",
    "Maximal #combinations of charge impurities allowed (each generating one feature) per charge state": "每个电荷状态下允许的最大电荷杂质组合数（每种组合生成一个特征）",
    "Probability for the binomial distribution of the ESI charge states": "ESI电荷状态的二项分布概率",
    "List of probabilities for different charge states (starting at charge=1, 2, ...) during MALDI ionization (the list must sum up to 1.0)": "不同电荷状态下的概率列表（从电荷=1、2、...开始，在MALDI离子化过程中）（该列表的总和必须等于1.0）",
    "Lower m/z detector limit": "较低的 m/z 检测器限制",
    "Upper m/z detector limit": "上 m/z 检测器限制",
    "Enable RAW signal simulation": "启用原始信号模拟",
    "Peak Shape used around each isotope peak (be aware that the area under the curve is constant for both types, but the maximal height will differ (~ 2:3 = Lorentz:Gaussian) due to the wider base of the Lorentzian": "每个同位素峰周围使用的峰形（请注意，曲线下的面积对于两种类型是恒定的，但由于洛伦兹函数的基底更宽，最大高度会有所不同（~2:3=洛伦兹：高斯）",
    "Instrument resolution at 400 Th": "仪器在400 Th时的分辨率",
    "How does resolution change with increasing m/z": "分辨率如何随着 m/z 的增加而变化",
    "Baseline modeling for MALDI ionization": "基线建模用于MALDI电离",
    "Scale of baseline": "基线比例",
    "The baseline is modeled by an exponential probability density function (pdf) with f(x) = shape*e^(- shape*x)": "基线由指数概率密度函数(pdf)建模，其中 f(x) = shape*e^(- shape*x)",
    "Number of raw data points per FWHM of the peak": "每个峰值全宽半高（FWHM）的原始数据点数",
    "Contaminants file with sum formula and absolute RT interval": "污染物文件，包含总公式和绝对保留时间间隔",
    "Random components that simulate biological and technical variations of the simulated data": "模拟数据中模拟生物和技术变异的随机组分",
    "Shifts in mass to charge dimension of the simulated signals": "模拟信号的质量-电荷维度的变化",
    "Average systematic m/z error (in Da)": "平均系统性 m/z 误差（以道尔顿为单位）",
    "Standard deviation for m/z errors": "m/z误差的标准差",
    "Variations in intensity to model randomness in feature intensity": "强度变化以模拟特征强度中的随机性",
    "Constant scale factor of the feature intensity": "特征强度的恒定比例因子",
    "Standard deviation of peak intensity (relative to the scaled peak height)": "峰值强度的标准差（相对于缩放的峰值高度）",
    "Parameters modeling noise in mass spectrometry measurements": "参数模拟质谱测量中的噪声",
    "Parameters of Poisson and Exponential for shot noise modeling (set :rate OR :mean = 0 to disable)": "泊松和指数分布的参数用于射击噪声建模（设置:rate 或 :mean = 0以禁用）",
    "Poisson rate of shot noise per unit m/z (random peaks in m/z, where the number of peaks per unit m/z follows a Poisson distribution)": "泊松率的射击噪声每单位 m/z（随机峰值在 m/z，其中每单位 m/z 的峰值数量遵循泊松分布）",
    "Shot noise intensity mean (exponentially distributed with given mean)": "射击噪声强度均值（具有给定均值的指数分布）",
    "Parameters of Gaussian distribution for white noise modeling (set :mean AND :stddev = 0 to disable)": "高斯分布的参数用于白噪声建模（设置：均值 AND 标准差 = 0 以禁用）",
    "Mean value of white noise (Gaussian) being added to each *measured* signal intensity": "每个测量的信号强度中添加的白噪声（高斯）的平均值",
    "Standard deviation of white noise being added to each *measured* signal intensity": "每个测量信号强度添加的白噪声的标准差",
    "Parameters of Gaussian distribution for detector noise modeling (set :mean AND :stddev = 0 to disable)": "高斯分布的参数用于探测器噪声建模（设置：均值 AND 标准差 = 0 以禁用）",
    "Mean intensity value of the detector noise (Gaussian distribution)": "探测器噪声的平均强度值（高斯分布）",
    "Standard deviation of the detector noise (Gaussian distribution)": "探测器噪声的标准差（高斯分布）",
    "Create Tandem-MS scans?": "创建串联质谱扫描？",
    "Algorithm to generate the tandem-MS spectra": "生成串联质谱图的算法",
    "File containing the filenames of SVM Models for different charge variants": "包含不同电荷变体SVM模型文件名的文件",
    "Number of allowed MS/MS spectra in a retention time bin": "允许在一个保留时间窗口内的MS/MS谱数",
    "The minimal distance (in Th) between two peaks for concurrent selection for fragmentation": "两个峰值之间并发选择用于断裂的最小距离（以Th为单位）",
    "All peaks within a mass window (in Th) of a selected peak are also selected for fragmentation": "所有在选定峰值的质量窗口（Th）内的峰值也被选择进行碎裂。",
    "If true, overlapping or nearby peaks (within 'min_mz_peak_distance') are excluded for selection": "如果为真，那么在选定范围内（'min_mz_peak_distance'内）的重叠或邻近峰将被排除在选择之外",
    "Charges considered for MS2 fragmentation": "考虑对MS2碎片化进行充电",
    "If true dynamic exclusion is applied": "如果应用了真正的动态排除",
    "The time (in seconds) a feature is excluded": "特征被排除的时间（单位：秒）",
    "The maximal number of precursors in the inclusion list": "包含列表中前体的最大数量",
    "Minimal rt in seconds": "最小保留时间（秒）",
    "Maximal rt in seconds": "最大运行时间（以秒为单位）",
    "rt step size in seconds": "rt 步长（时间分辨率）以秒为单位",
    "rt window size in seconds": "实时窗口大小（秒）",
    "If true, the MS2 spectra for each peptide signal are included in the output (might be a lot)": "如果为真，每个肽段信号的MS2谱图将包含在输出中（可能会很多）",
    "Model to use for isotopic peaks ('none' means no isotopic peaks are added, 'coarse' adds isotopic peaks in unit mass distance, 'fine' uses the hyperfine isotopic generator to add accurate isotopic peaks": "用于同位素峰的模型（'none' 表示不添加同位素峰，'coarse' 在单位质量距离处添加同位素峰，'fine' 使用超精细同位素生成器添加精确的同位素峰）",
    "Defines the maximal isotopic peak which is added if 'isotope_model' is 'coarse'": "定义了当 'isotope_model' 为 'coarse' 时添加的最大同位素峰",
    "Defines the maximal isotopic probability to cover if 'isotope_model' is 'fine'": "定义了如果“同位素模型”是“fine”，则覆盖的最大同位素概率",
    "Adds the type of peaks as metainfo to the peaks, like y8+, [M-H2O+2H]++": "将峰值类型作为元信息添加到峰值中，例如 y8+，[M-H2O+2H]++",
    "Adds common losses to those ion expect to have them, only water and ammonia loss is considered": "添加常见的损失，这些损失是预期会有的，只考虑水和氨的损失",
    "Sort output by position": "按位置排序输出",
    "Adds peaks of the unfragmented precursor ion to the spectrum": "将未断裂的前体离子的峰添加到谱图中",
    "Adds precursor peaks with all charges in the given range": "添加给定范围内所有电荷的前体峰",
    "Add most abundant immonium ions (for Proline, Cystein, Iso/Leucine, Histidin, Phenylalanin, Tyrosine, Tryptophan)": "添加最丰富的亚氨离子（对于脯氨酸、半胱氨酸、异亮氨酸/亮氨酸、组氨酸、苯丙氨酸、酪氨酸、色氨酸）",
    "If set to true e.g. b1 ions are added": "如果设置为真，例如添加了 b1 离子",
    "Add peaks of y-ions to the spectrum": "在谱图中添加y离子的峰",
    "Add peaks of b-ions to the spectrum": "将 b-离子的峰添加到谱图中",
    "Add peaks of a-ions to the spectrum": "将α离子的峰添加到谱图中",
    "Add peaks of c-ions to the spectrum": "将c-ions的峰添加到谱图中",
    "Add peaks of  x-ions to the spectrum": "将 x-离子的峰添加到谱图中",
    "Add peaks of z-ions to the spectrum": "将z-离子的峰添加到谱图中",
    "Intensity of the y-ions": "y离子的强度",
    "Intensity of the b-ions": "b离子的强度",
    "Intensity of the a-ions": "α粒子的强度",
    "Intensity of the c-ions": "c离子的强度",
    "Intensity of the x-ions": "x-离子的强度",
    "Intensity of the z-ions": "z-离子的强度",
    "Intensity of loss ions, in relation to the intact ion intensity": "损失离子的强度，与完整离子强度的关系",
    "Intensity of the precursor peak": "前体峰的强度",
    "Intensity of the H2O loss peak of the precurso": "前体的H2O丢失峰的强度",
    "Intensity of the NH3 loss peak of the precurso": "NH3损失峰的强度",
    "If set to 1 isotope peaks of the product ion peaks are added": "如果设置为1，则将产物离子峰的同位素峰添加进去",
    "Defines the maximal isotopic peak which is added, add_isotopes must be set to 1": "定义添加的最大同位素峰，add_isotopes 必须设置为 1",
    "Type of Ionization (MALDI or ESI)": "离子化类型（MALDI 或 ESI）",
    "Select the labeling type you want for your experiment": "选择您实验所需的标记类型",
    "ICPL labeling on MS1 level of lysines and n-term (on protein or peptide level) with either two or three channels": "ICPL标记在赖氨酸和N端（在蛋白质或肽水平上）的MS1水平上，有两个或三个通道。",
    "Fixed retention time shift between labeled pairs": "固定标记对之间的保留时间偏移",
    "Enables protein-labeling": "启用蛋白质标记",
    "UniMod Id of the light channel ICPL label": "UniMod 编号 轻通道 ICPL 标记",
    "UniMod Id of the medium channel ICPL label": "UniMod 编号的中等通道 ICPL 标签",
    "UniMod Id of the heavy channel ICPL label": "UniMod 编号的重通道 ICPL 标记",
    "SILAC labeling on MS1 level with up to 3 channels and custom modifications": "SILAC标记在MS1水平上，支持多达3个通道和自定义修饰",
    "Fixed retention time shift between labeled peptides": "固定标记肽段之间的保留时间偏移",
    "Modifications for the medium SILAC channel": "中等SILAC通道的修改",
    "Modification of Lysine in the medium SILAC channel": "赖氨酸在中等丰度稳定同位素标记氨基酸培养（SILAC）通道中的修饰",
    "Modification of Arginine in the medium SILAC channel": "在中等丰度SILAC通道中的精氨酸修饰",
    "Modifications for the heavy SILAC channel": "重SILAC通道的修改",
    "Modification of Lysine in the heavy SILAC channel": "赖氨酸在重SILAC通道中的修饰",
    "Modification of Arginine in the heavy SILAC channel": "在重SILAC通道中精氨酸的修饰",
    "iTRAQ labeling on MS2 level with up to 4 (4plex) or 8 (8plex) channels": "iTRAQ标记在MS2水平上，有多达4个（4重）或8个（8重）通道。",
    "4plex or 8plex iTRAQ?": "四重或八重iTRAQ？",
    "Allowed shift (uniformly distributed - left to right) in Da from the expected position (of": "允许的位移（均匀分布 - 从左到右）在 Da 从预期位置（的",
    "Four-plex only: Each channel that was used in the experiment and its description (114-117) in format <channel>:<name": "四重检测专用：实验中使用且描述了每个通道（114-117），格式为<通道>:<名称>",
    "Eight-plex only: Each channel that was used in the experiment and its description (113-121) in format <channel>:<name": "八重标记仅使用：实验中使用的每个通道及其描述（113-121），格式为<通道>:<名称>",
    "override default values (see Documentation); use the following format: <channel>:<-2Da>/<-1Da>/<+1Da>/<+2Da> ;": "覆盖默认值（见文档）；使用以下格式：<通道>:<-2Da>/<-1Da>/<+1Da>/<+2Da>；",
    "Efficiency of labeling tyrosine ('Y') residues": "酪氨酸（'Y'）残基的标记效率",
    "18O labeling on MS1 level with 2 channels, requiring trypsin digestion": "在MS1水平上进行18O标记，需要2个通道，并要求进行胰蛋白酶消化。",
    "Describes the distribution of the labeled peptide over the different states (unlabeled, mono- and di-labeled)": "描述标记肽在不同状态（未标记、单标记和双标记）上的分布情况",
    "Parameters for generating the random aspects": "生成随机方面的参数",
    "Controls the 'biological' randomness of the generated data": "控制生成数据的“生物学”随机性",
    "Controls the 'technical' randomness of the generated data": "控制生成数据的“技术”随机性",
    "input raw data file": "输入原始数据文件",
    "Length of the structuring element (should be wider than maximal peak width - see documentation)": "结构元素的长度（应比最大峰宽宽 - 参见文档）",
    "Unit of 'struc_elem_length' paramete": "'struc_elem_length' 参数的单位",
    "The name of the morphological filter to be applied": "要应用的形态学滤波器的名称",
    "Input file containing RNA sequences": "包含RNA序列的输入文件",
    "Minimum length of a fragment": "片段的最小长度",
    "Maximum length of a fragment": "片段的最大长度",
    "Digestion enzyme (RNase)": "消化酶（核糖核酸酶）",
    "Report each unique sequence fragment only once": "仅报告每个独特的序列片段一次",
    "Input file contains cDNA sequences - replace 'T' with 'U')": "输入文件包含cDNA序列 - 将'T'替换为'U'",
    "Run tool in batch mode for -cv_files": "批量运行工具，针对 -cv_files 参数",
    "List of ontology files in OBO format": "OBO格式本体文件列表",
    "List of identifiers (one for each ontology file)": "每个本体文件的标识符列表",
    "Mapping file in CVMapping (XML) format": "映射文件在CVMapping（XML）格式中",
    "A list of CV identifiers which should be ignored": "一个应被忽略的CV标识符列表",
    "Use pixel annotation from tabular file for QC plots": "使用表格文件中的像素注释进行质量控制(QC)绘图",
    "ppm range": "PPM范围",
    "PCA with 2 components": "主成分分析（PCA）是一种统计方法，用于通过正交变换将一组可能相关的变量转换为一组线性不相关的变量，称为主成分。在这种情况下，PCA with 2 components 指的是使用两个主成分进行分析。",
    "Generate full QC report": "生成完整的质量控制报告",
    "Plot fold change of two m/z": "绘制两个m/z的倍数变化图",
    "M/z 1": "质荷比 1",
    "M/z 2": "质荷比 2",
    "Input FASTA file(s), each containing a database": "输入FASTA文件（们），每个文件都包含一个数据库",
    "String that is combined with the accession of the protein identifier to indicate a decoy protein": "与蛋白质标识符的访问号组合的字符串，用于表示诱饵蛋白",
    "Should the 'decoy_string' be prepended (prefix) or appended (suffix) to the protein accession?": "“decoy_string”应该添加到蛋白质访问号的前面（前缀）还是后面（后缀）？",
    "Write only decoy proteins to the output database instead of a combined database": "将仅伪蛋白质写入输出数据库，而不是组合数据库",
    "Method by which decoy sequences are generated from target sequences": "生成诱饵序列的方法",
    "Enzyme used for the digestion of the sample": "用于样本消化的酶",
    "Decoy parameters section": "诱饵参数部分",
    "Residues to not shuffle (keep at a constant position when shuffling)": "在洗牌时不进行洗牌的残基（保持在恒定位置）",
    "Whether to keep peptide N terminus constant when shuffling / reversing": "是否在洗牌/反转时保持肽段N端不变",
    "Whether to keep peptide C terminus constant when shuffling / reversing": "是否在洗牌/反转时保持肽段C末端不变",
    "Random number seed (use 'time' for system time)": "随机数种子（使用'time'代表系统时间）",
    "Mass offset in ppm": "质量偏移，在ppm（百万分之一）",
    "Slope (dependent on m/z)": "斜率（取决于 m/z）",
    "Power (dependent on m/z)": "功率（取决于 m/z）",
    "Identifications from searching a target-decoy database": "搜索目标-伪目标数据库的鉴定",
    "Perform FDR calculation on PSM level": "在PSM水平上执行FDR计算",
    "Perform FDR calculation on protein level": "在蛋白质水平上执行FDR计算",
    "FDR control": "FDR控制",
    "Filter PSMs based on q-value": "基于q值过滤PSMs",
    "Filter proteins based on q-value": "根据q值过滤蛋白质",
    "Cleanup references after FDR control": "在FDR控制后清理参考",
    "Remove proteins without PSMs (due to being decoy or below PSM FDR threshold)": "移除没有PSMs（由于是诱饵或低于PSM FDR阈值）的蛋白质",
    "Remove PSMs without proteins (due to being decoy or below protein FDR threshold)": "移除没有蛋白质的PSMs（由于是诱饵或低于蛋白质FDR阈值）",
    "Remove spectra without PSMs (due to being decoy or below protein FDR threshold)": "移除没有PSMs（由于是诱饵或低于蛋白质FDR阈值）的谱图",
    "Parameter section for the FDR calculation algorithm": "FDR计算算法的参数部分",
    "If 'true' strict FDRs will be calculated instead of q-values (the default)": "如果设置为“true”，将计算严格的FDR值，而不是q值（默认值）",
    "If 'true' not only the first hit, but all are used (peptides only)": "如果为“true”，则不仅使用第一个命中，而且使用所有命中（仅限肽段）",
    "If 'true' charge variants are treated separately (for peptides of combined target/decoy searches only)": "如果“真实”的电荷变体被单独处理（仅针对组合目标/诱饵搜索的肽段）",
    "If 'true' different search runs are treated separately (for peptides of combined target/decoy searches only)": "如果为'true'，则不同的搜索运行将被单独处理（仅针对组合目标/诱饵搜索的肽段）",
    "If 'true' decoy peptides will be written to output file, too": "如果“真实”的诱饵肽段也会被写入输出文件",
    "If 'true' decoy proteins will be written to output file, too": "如果“真实”的诱饵蛋白也会被写入输出文件",
    "If 'true' (D+1)/T instead of (D+1)/(T+D) is used as a formula": "如果使用 'true' (D+1)/T 而不是 (D+1)/(T+D) 作为公式",
    "Input file: LC-MS raw data": "输入文件：LC-MS 原始数据",
    "Input file: Peptide identifications derived directly from 'in'": "输入文件：直接从“in”派生的肽段鉴定",
    "Input file: 'External' peptide identifications": "输入文件：'External' 肽段鉴定",
    "Whether decoy peptides should be quantified (true) or skipped (false)": "是否需要对诱饵肽段进行定量（true）或跳过（false）",
    "Parameters for ion chromatogram extraction": "离子色谱图提取参数",
    "Nr of peptides used in each batch of chromatogram extraction": "每个色谱图提取批次中使用的肽段数量",
    "m/z window size for chromatogram extraction (unit: ppm if 1 or greater, else Da/Th)": "色谱图提取的 m/z 窗口大小（单位：ppm，如果为 1 或更大；否则为 Da/Th）",
    "Number of isotopes to include in each peptide assay": "每个肽段检测中包含的同位素数量",
    "Minimum probability for an isotope to be included in the assay for a peptide": "肽段同位素检测中包含的最小概率",
    "Quantile of the RT deviations between aligned internal and external IDs to use for scaling the RT extraction window": "对齐的内部和外部ID之间的RT偏差的分位数，用于缩放RT提取窗口",
    "RT window size (in sec.) for chromatogram extraction": "RT窗口大小（以秒为单位）用于色谱图提取",
    "Parameters for detecting features in extracted ion chromatograms": "在提取的离子色谱图中检测特征的参数",
    "Expected elution peak width in seconds, for smoothing (Gauss filter)": "预期洗脱峰宽度（秒），用于平滑（高斯滤波器）",
    "Minimum elution peak width": "最小洗脱峰宽",
    "Signal-to-noise threshold for OpenSWATH feature detection": "OpenSWATH特征检测的信噪比阈值",
    "RT tolerance (plus/minus) for mapping peptide IDs to features": "RT 容忍度（正负）用于将肽段 ID 映射到特征",
    "Parameters for scoring features using a support vector machine (SVM)": "使用支持向量机（SVM）评分特征的参数",
    "Number of observations to use for training ('0' for all)": "用于训练的观测值数量（'0'表示全部）",
    "By default, roughly the same number of positive and negative observations, with the same intensity distribution, are selected for training": "默认情况下，选择大致相同数量的正负观测值进行训练，且这些观测值具有相同的强度分布。",
    "SVM kernel": "SVM核函数",
    "Number of partitions for cross-validation (parameter optimization)": "交叉验证（参数优化）的分区数",
    "Values to try for the SVM parameter 'C' during parameter optimization": "在参数优化过程中尝试用于SVM参数'C'的值",
    "Values to try for the SVM parameter 'gamma' during parameter optimization (RBF kernel only)": "在参数优化期间尝试用于支持向量机（SVM）参数'gamma'的值（仅限径向基函数（RBF）核）",
    "Stopping criterion": "停止准则",
    "Size of the kernel cache (in MB)": "内核缓存大小（以MB为单位）",
    "Disable the shrinking heuristics": "禁用收缩启发式",
    "Names of OpenSWATH scores to use as predictors for the SVM (comma-separated list)": "用于SVM的OpenSWATH分数名称（逗号分隔列表）",
    "Minimum probability of correctness, as predicted by the SVM, required to retain a feature candidate": "由支持向量机（SVM）预测的保留特征候选所需的最小正确概率",
    "Parameters for fitting elution models to features": "拟合洗脱模型到特征的参数",
    "Type of elution model to fit to features": "要拟合到特征的洗脱模型类型",
    "Add zero-intensity points outside the feature range to constrain the model fit": "在特征范围外添加零强度点以约束模型拟合",
    "Suppress weighting of mass traces according to theoretical intensities when fitting elution models": "在拟合洗脱模型时，根据理论强度抑制质量迹线权重",
    "If fitting the elution model fails for a feature, set its intensity to zero instead of imputing a value from the initial intensity estimate": "如果拟合洗脱模型失败，为特征设置强度为零，而不是从初始强度估计中插值。",
    "Fit elution model to each individual mass trace": "拟合洗脱模型到每个单独的质量迹线",
    "Parameters for checking the validity of elution models (and rejecting them if necessary)": "检查洗脱模型有效性的参数（如有必要，可以拒绝它们）",
    "Lower bound for the area under the curve of a valid elution model": "有效洗脱模型曲线下面积的下限",
    "Time points corresponding to this fraction of the elution model height have to be within the data region used for model fitting": "对应于洗脱模型高度的这个时间点必须在用于模型拟合的数据区域内",
    "Upper limit for acceptable widths of elution models (Gaussian or EGH), expressed in terms of modified (median-based) z-scores": "可接受洗脱模型（高斯或EGH）宽度的上限，以修改后的（基于中位数的）z分数表示",
    "Upper limit for acceptable asymmetry of elution models (EGH only), expressed in terms of modified (median-based) z-scores": "可接受洗脱模型不对称性的上限（仅用于EGH），以修改后的（基于中位数的）z分数表示",
    "Parameters for fitting exp": "拟合指数函数的参数",
    "Maximum number of iterations for EMG fitting": "EMG拟合的最大迭代次数",
    "Alternative initial parameters for fitting through method of moments": "通过矩法拟合的替代初始参数",
    "Input file: Feature candidates from a previous run": "输入文件：来自之前运行的特征候选项",
    "The maximal charge state to be considered": "最大电荷状态要考虑",
    "The final threshold t' is build upon the formula: t' = av+t*sd, where t is the intensity_threshold, av the average intensity within the wavelet transformed signal and sd the standard deviation of the transform": "最终阈值 t' 是根据公式构建的：t' = av + t * sd，其中 t 是强度阈值，av 是小波变换信号内的平均强度，sd 是变换的标准差。",
    "Determines the intensity type returned for the identified features": "确定返回的已识别特征的强度类型",
    "Enables/disables a ppm test vs": "启用/禁用 ppm 测试与",
    "Must be true in case of high-resolution data": "在高分辨率数据的情况下必须为真",
    "Defines the minimum number of subsequent scans where a pattern must occur to be considered as a feature": "定义了必须连续出现模式的最小扫描次数，以将其视为特征",
    "Defines the maximum number of scans (w.r.t": "定义了扫描的最大数量（相对于",
    "Centroided mzML file": "质心化的mzML文件",
    "Common parameters for all other subsections": "所有其他子节的共同参数",
    "Mass Trace Detection parameters": "质量迹线检测参数",
    "Elution Profile Detection (to separate isobaric Mass Traces by elution time)": "洗脱轮廓检测（通过洗脱时间分离等位质量迹线）",
    "Enable splitting of isobaric mass traces by chromatographic peak detection": "启用通过色谱峰检测来分离等重质量迹线",
    "FeatureFinder parameters (assembling mass traces to charged features)": "特征发现参数（将质量迹线组装成带电特征）",
    "RT range where to look for coeluting mass traces": "保留时间范围，用于寻找共洗脱的质量迹线",
    "MZ range where to look for isotopic mass traces": "寻找同位素质量迹线时的m/z范围",
    "Lowest charge state to conside": "最低电荷状态考虑",
    "Highest charge state to conside": "考虑最高的电荷状态",
    "Set to true for a feature intensity summed up over all traces rather than using monoisotopic trace intensity alone": "设置为true，以便对所有迹线的特征强度进行求和，而不是仅使用单同位素迹线强度。",
    "Require sufficient overlap in RT while assembling mass traces": "在组装质谱迹线时需要有足够的RT（保留时间）重叠。",
    "Remove/score candidate assemblies based on isotope intensities": "根据同位素强度移除/评分候选装配",
    "Use the 13C isotope peak position (~1.003355 Da) as the expected shift in m/z for isotope mass traces (highly recommended for lipidomics!)": "使用13C同位素峰位置（约1.003355 Da）作为同位素质量迹线（用于同位素质量追踪）的预期m/z偏移值（在脂质组学中强烈推荐使用！）",
    "Use LOWESS intensities instead of raw intensities": "使用LOWESS强度而不是原始强度",
    "Augment each reported feature with the convex hull of the underlying mass traces (increases featureXML file size considerably)": "为每个报告的特征增加其底层质量迹线的凸包（显著增加featureXML文件大小）",
    "Remove unassembled traces (single traces)": "移除未组装的迹线（单个迹线）",
    "Use the m/z range of the assumed elements to detect isotope peaks": "使用假定元素的 m/z 范围来检测同位素峰",
    "Elements assumes to be present in the sample (this influences isotope detection)": "元素假定存在于样本中（这影响同位素检测）",
    "Input file: Metabolite identifications": "输入文件：代谢物鉴定",
    "Minimal distance of MRM features in seconds": "MRM 特征的最小距离（以秒为单位）",
    "Minimal number of peaks which are needed for a single feature": "单个特征所需的最小峰值数量",
    "Minimal S/N ratio a peak must have to be taken into account": "最小信噪比，一个峰值必须具有才能被考虑",
    "If set to true, for each feature a plot will be created, in the subdirectory 'debug'": "如果设置为真，对于每个特征，将在子目录'debug'中创建一个图表。",
    "If set to true, each trace, which is in this case a part of the MRM monitoring trace with signal is resampled, using the minimal distance of two data points in RT dimension": "如果设置为true，每个跟踪（在这种情况下是MRM监测跟踪的一部分，具有信号）将使用RT维度中两个数据点的最小距离进行重新采样。",
    "If set to true, debug messages are written, the output can be somewhat lengthy": "如果设置为真，将写入调试信息，输出可能会有些长",
    "LC-MS dataset in either centroid or profile mode": "液相色谱-质谱数据集，可以是质心模式或轮廓模式",
    "algorithmic parameters": "算法参数",
    "Range of charge states in the sample": "样品中电荷状态的范围",
    "Range of isotopes per peptide in the sample": "样本中每个肽段的同位素范围",
    "Typical retention time [s] over which a characteristic peptide elutes": "典型保留时间 [秒]，在这个时间内一个特征肽段被洗脱",
    "The algorithm searches for characteristic isotopic peak patterns, spectrum by spectrum": "该算法逐个谱图搜索特征的同位素峰模式",
    "Lower bound for the retention time [s]": "保留时间的下限 [秒]",
    "m/z tolerance for search of peak patterns": "m/z 容差用于搜索峰模式",
    "Unit of the 'mz_tolerance' paramete": "'mz_tolerance' 参数的单位",
    "Lower bound for the intensity of isotopic peaks": "同位素峰强度的下限",
    "Two peptides in a multiplet are expected to have the same isotopic pattern": "在多重态中的两个多肽预计会有相同的同位素模式",
    "The isotopic pattern of a peptide should resemble the averagine model at this m/z position": "肽段的同位素模式在该m/z位置应与平均素模型相似",
    "Let x denote this scaling factor, and p the averagine similarity paramete": "让 x 表示这个缩放因子，p 表示 averagine 相似性参数",
    "Type of MS1 spectra in input mzML file": "输入的mzML文件中MS1谱图的类型",
    "The type of averagine to use, currently RNA, DNA or peptide": "要使用的averagine类型，目前有RNA、DNA或肽。",
    "Is it likely that knock-outs are present": "是否有敲除（knock-outs）存在的可能性",
    "mass shifts for all possible labels": "所有可能标签的质量偏移",
    "input files separated by blanks": "输入文件由空白分隔",
    "For consensusXML input only: If set, the sub-features of the inputs are transferred to the output": "对于consensusXML输入仅：如果设置，输入的子特征将转移到输出",
    "Unit of m/z tolerance": "m/z 容差单位",
    "Number of partitions in m/z space": "m/z空间中的分区数",
    "Whether or not to internally warp feature RTs using LOWESS transformation before linking (reported RTs in results will always be the original RTs)": "是否在链接之前使用LOWESS转换内部变形特征RT（结果中报告的RT始终是原始RT）",
    "Width of RT tolerance window (sec)": "RT容忍窗口宽度（秒）",
    "m/z tolerance (in ppm or Da)": "m/z 容差（以 ppm 或 Da 表示）",
    "Maximum absolute log10 fold change between two compatible signals during compatibility graph construction": "在兼容性图构建期间，两个兼容信号之间的最大绝对对数10倍变化",
    "Only connected components containing compatible features from at least max(2, (warp_min_occur * number_of_input_maps)) input maps are considered for computing the warping function": "只有包含至少max(2, (warp_min_occur * number_of_input_maps))个输入图中兼容特征的连通组件才被考虑用于计算扭曲函数。",
    "Allow up to this many conflicts (features from the same map) per connected component to be used for alignment (-1 means allow any number of conflicts)": "允许每个连通组件使用多达这么多冲突（来自同一地图的特征）进行比对（-1表示允许任意数量的冲突）",
    "whether to disallow charge mismatches (Identical), allow to link charge zero": "是否不允许电荷不匹配（相同），允许连接电荷为零",
    "whether to only allow the same adduct for linking (Identical), also allow linking features with adduct-free ones, or disregard adducts (Any)": "是否只允许相同的加合物进行连接（相同），也允许加合物自由连接特征，或者忽略加合物（任何）",
    "LOWESS parameters for internal RT transformations (only relevant if 'warp:enabled' is set to 'true')": "LOWESS 参数用于内部 RT 转换（仅在将 'warp:enabled' 设置为 'true' 时相关）",
    "Never link features that are annotated with different peptides (only the best hit per peptide identification is taken into account)": "不要链接用不同肽段注释的特征（只考虑每个肽段鉴定的最佳命中）。",
    "How many partitions in m/z space should be used for the algorithm (more partitions means faster runtime and more memory efficient execution)": "算法在m/z空间中应该使用多少个分区（更多的分区意味着更快的运行时间和更高效的内存执行）",
    "If IDs are used: How many differences from matching IDs should be used to calculate a linking tolerance for unIDed features in an RT region": "如果使用ID：应该使用多少与匹配ID的差异来计算未ID特征在RT区域中的链接公差",
    "If IDs are used: What is the minimum score of an ID to assume a reliable match for tolerance calculation": "如果使用ID：假设一个可靠的匹配用于容差计算，一个ID的最低分数是多少",
    "If IDs are used: For the normalized distances, how high should the penalty for missing IDs be": "如果使用ID：对于标准化距离，缺失ID的惩罚应该有多高",
    "Feature input file, which contains the data to be tested against the truth file": "特征输入文件，其中包含要与真实文件进行测试的数据",
    "Truth feature file that defines what features should be found": "真实特征文件，定义了应该找到哪些特征",
    "Allowed tolerance of RT relative to average feature RT span": "允许的保留时间相对于特征保留时间平均值的公差范围",
    "Allowed absolute tolerance of RT (overwrites 'rt_tol' if set above zero)": "允许的保留时间（RT）绝对公差（如果设置为大于零，则覆盖'rt_tol'）",
    "Allowed tolerance in m/z (is divided by charge)": "允许的质量误差容限（以m/z计，除以电荷）",
    "Feature file containing seeds with abort reasons": "包含带有终止原因的种子特征文件",
    "Process multiple protein identification runs in the input separately, don't merge them": "独立处理输入中的多个蛋白质鉴定运行，不要合并它们",
    "Post-process Fido output with greedy resolution of shared peptides based on the protein probabilities": "使用基于蛋白质概率的贪婪策略解决共享肽段问题，对Fido输出进行后处理。",
    "Omit clean-up of peptide sequences (removal of non-letter characters, replacement of I with L)": "忽略肽段序列的清理（移除非字母字符，将I替换为L）",
    "Consider all PSMs of each peptide, instead of only the best one": "考虑每个肽段的所有PSMs（肽段-谱图匹配），而不是仅考虑最佳的一个。",
    "Perform inference on protein group level (instead of individual protein level)": "在蛋白质组水平（而不是单个蛋白质水平）上执行推断",
    "Binary logarithm of the max": "二进制对数的最大值",
    "Probability values for running Fido directly": "运行 Fido 的概率值",
    "Keep the group of proteins with estimated probability of zero, which is otherwise removed (it may be very large)": "保留估计概率为零的蛋白质组（否则将被移除，它可能非常大）",
    "Accuracy level of start parameters": "起始参数的准确度水平",
    "Like 'log2_states', but allows to set a separate limit for the precalculation": "类似于 'log2_states'，但允许为预计算设置单独的限制",
    "m/z range to extract (applies to ALL ms levels!)": "m/z 范围提取（适用于所有 MS 级别！）",
    "Intensity range to extract": "提取强度范围",
    "Sorts the output according to RT and m/z": "根据保留时间（RT）和质荷比（m/z）对输出进行排序",
    "Peak data options": "峰值数据选项",
    "Write peaks with S/N > 'sn' values only": "仅写出信噪比大于'sn'值的峰",
    "Remove MS(2) spectra with these precursor charges": "删除具有这些前体电荷的MS(2)谱图",
    "MSn (n>=2) precursor filtering according to their m/z value": "MSn（n>=2）前体筛选，根据它们的 m/z 值",
    "List of m/z values": "质荷比值列表",
    "Sorts the peaks according to m/z": "根据 m/z 对峰进行排序",
    "No conversion to space-saving real chromatograms": "不转换为节省空间的真实色谱图",
    "Removes chromatograms stored in a file": "移除存储在文件中的色谱图",
    "Removes spectra and chromatograms without peaks": "移除没有峰的谱图和色谱图",
    "Store base64 encoded m/z data using 32 or 64 bit precision": "使用32位或64位精度存储编码为base64的m/z数据",
    "Store base64 encoded intensity data using 32 or 64 bit precision": "使用32位或64位精度存储base64编码的强度数据",
    "Whether to add an index to the file when writing": "是否在写入文件时添加索引",
    "Whether to store data with zlib compression (lossless compression)": "是否使用 zlib 压缩（无损压缩）来存储数据",
    "Numpress compression for peak data": "Numpress 压缩用于峰值数据",
    "Apply MS Numpress compression algorithms in m/z or rt dimension (recommended: linear)": "在 m/z 或 rt 维度应用 MS Numpress 压缩算法（推荐：线性）",
    "Apply MS Numpress compression algorithms in intensity dimension (recommended: slof or pic)": "在强度维度应用MS Numpress压缩算法（推荐：slof或pic）",
    "Apply MS Numpress compression algorithms for the float data arrays (recommended: slof or pic)": "对浮点数据数组应用MS Numpress压缩算法（推荐：slof或pic）",
    "Remove spectra or select spectra (removing all others) with certain properties": "删除具有特定属性的谱图或选择具有特定属性的谱图（删除所有其他谱图）",
    "Remove zoom (enhanced resolution) scans": "移除缩放（增强分辨率）扫描",
    "Remove scans by scan mode": "移除按扫描模式的扫描",
    "Remove MSn scans where any of its precursors features a certain activation method": "移除MSn扫描，如果其前体特征具有某种特定的激活方法。",
    "Remove MSn scans with a collision energy in the given interval": "移除碰撞能量在给定区间内的MSn扫描",
    "Remove MSn scans whose isolation window width is in the given interval": "移除隔离窗口宽度在给定区间内的MSn扫描",
    "Select zoom (enhanced resolution) scans": "选择缩放（增强分辨率）扫描",
    "Selects scans by scan mode.": "选择扫描模式的扫描。",
    "Retain MSn scans where any of its precursors features a certain activation method": "保留MSn扫描，其中任何一个前体特征具有某种特定的激活方法。",
    "Select MSn scans with a collision energy in the given interval": "选择具有给定碰撞能量间隔的MSn扫描",
    "Select MSn scans whose isolation window width is in the given interval": "选择隔离窗口宽度在给定区间内的MSn扫描",
    "Retain MSn scans with a certain scan polarity": "保留具有特定扫描极性的MSn扫描",
    "Replaces in_charge with out_charge in all precursors": "在所有前体中，将 in_charge 替换为 out_charge",
    "Black or white listing of of MS2 spectra by spectral similarity": "MS2 谱图的黑白名单筛选，基于谱图相似性",
    "Input file containing MS2 spectra that should be retained or removed from the mzML file": "输入文件包含MS2谱图，这些谱图应该从mzML文件中保留或移除。",
    "Similarity threshold when matching MS2 spectra": "匹配MS2谱图时的相似性阈值",
    "Retention tolerance [s] when matching precursor positions": "保留前体位置匹配时的容忍时间 [秒]",
    "m/z tolerance [Th] when matching precursor positions": "m/z 容差 [Th] 在匹配前体位置时",
    "If ppm tolerance should be used": "如果应该使用ppm公差",
    "True: remove matched MS2": "True: 移除匹配的MS2",
    "Feature data options": "特征数据选项",
    "Overall quality range to extract [0:1]": "提取的整体质量范围 [0:1]",
    "Consensus feature data options": "共识特征数据选项",
    "Non-empty list of maps to be extracted from a consensus (indices are 0-based)": "非空的映射列表，需要从共识序列中提取（索引从0开始）",
    "Consensus features are kept only if they contain exactly one feature from each map (as given above in 'map')": "共识特征仅在它们包含每个映射（如上文所述的“map”）中的一个特征时才保留。",
    "Black or white listing of of MS2 spectra by consensus features": "MS2 谱图的共识特征进行黑名单或白名单管理",
    "Input file containing consensus features whose corresponding MS2 spectra should be removed from the mzML file": "输入文件包含需要从mzML文件中移除的MS2谱图对应的共识特征",
    "Maps used for black/white list filtering": "用于黑名单/白名单过滤的映射文件",
    "Retention tolerance [s] for precursor to consensus feature position": "保留时间容忍度 [秒]，用于前体到共识特征位置",
    "m/z tolerance [Th] for precursor to consensus feature position": "m/z 容差 [Th] 用于前体到共识特征位置",
    "Feature & Consensus data options": "特征和共识数据选项",
    "Charge range to extract": "提取电荷范围",
    "Size range to extract": "提取大小范围",
    "ID options": "ID选项",
    "Remove features with id clashes (different sequences mapped to one feature)": "移除具有ID冲突的特征（不同的序列映射到一个特征）",
    "in case of multiple peptide identifications, keep only the id with best score": "在多肽鉴定的情况下，只保留得分最高的鉴定结果。",
    "Keep only features containing whitelisted substrings": "只保留包含白名单子字符串的特征",
    "Comparison method used to determine if a feature is whitelisted": "用于确定一个特征是否被列入白名单的比较方法",
    "keep only features with white listed accessions": "只保留具有白名单访问权限的特征",
    "Remove features with annotations": "移除带有注释的特征",
    "Remove features without annotations": "移除没有注释的特征",
    "Remove unassigned peptide identifications": "删除未分配的肽段鉴定",
    "Input file containing MS2 identifications whose corresponding MS2 spectra should be removed from the mzML file": "输入文件包含MS2鉴定结果，其对应的MS2谱图应从mzML文件中移除",
    "Retention tolerance [s] for precursor to id position": "保留时间容忍度 [秒]，用于前体离子鉴定位置",
    "m/z tolerance [Th] for precursor to id position": "m/z 容差 [Th] 用于前体鉴定位置",
    "Allow for mismatching precursor positions (see 'id:blacklist')": "允许不匹配的前体位置（见'id:blacklist'）",
    "S/N algorithm section": "信噪比算法部分",
    "Computes a five-number statistics of intensities, qualities, and widths": "计算强度、质量以及宽度的五数统计量",
    "Show detailed listing of all spectra and chromatograms (peak files only)": "显示所有光谱和色谱图（仅限峰值文件）的详细列表",
    "Check for corrupt data in the file (peak files only)": "检查文件中的损坏数据（仅限峰值文件）",
    "Validate the file only (for mzML, mzData, mzXML, featureXML, idXML, consensusXML, pepXML)": "验证文件（仅限mzML、mzData、mzXML、featureXML、idXML、consensusXML、pepXML格式）",
    "Check whether a given mzML file contains valid indices (conforming to the indexedmzML standard)": "检查给定的mzML文件是否包含有效的索引（符合indexedmzML标准）",
    "File type of output out (Output file)": "输出文件的文件类型（输出文件）",
    "Store the original filename in each feature using meta value \"file_origin\" (for featureXML and consensusXML only)": "在每个特征中使用元值 \"file_origin\" 存储原始文件名（仅针对 featureXML 和 consensusXML）",
    "(ConsensusXML-only) Append quantitative information about features row-wise or column-wise": "（仅限ConsensusXML格式）按行或按列追加关于特征的定量信息",
    "Options for concatenating files in the retention time (RT) dimension": "在保留时间（RT）维度中连接文件的选项",
    "The amount of gap (in seconds) to insert between the RT ranges of different input files": "在不同输入文件的RT范围之间插入的间隔时长（以秒为单位）",
    "Options for raw data input/output (primarily for DTA files)": "原始数据输入/输出选项（主要用于DTA文件）",
    "Assign retention times automatically (integers starting at 1)": "自动分配保留时间（整数，从1开始）",
    "List of custom retention times that are assigned to the files": "分配给文件的自定义保留时间列表",
    "Try to guess the retention time of a file based on the filename": "尝试根据文件名猜测一个文件的保留时间",
    "If 1 or higher, this number is assigned to spectra as the MS level": "如果为1或更高，这个数字将作为质谱级别分配给谱图",
    "Input consensusXML file containing only consensusElements with \"peptide\" annotations": "输入包含仅带有“peptide”注释的consensusElements的consensusXML文件",
    "Run tool in batch mode for -in_mzml": "批量运行工具，针对 -in_mzml 参数。",
    "Original mzml files containing the ms2 spectra (aka peptide annotation)": "原始的mzml文件包含ms2谱图（即肽段注释）",
    "specificity of mgf output information": "mgf输出信息的特异性",
    "Number of most intense peptides to consider per consensus element; '-1' to consider all identifications": "每个共识元件考虑的最强肽段数；'-1' 表示考虑所有鉴定结果",
    "Bin size (Da) for fragment ions when merging ms2 scans": "片段离子合并ms2扫描时的箱大小（道尔顿）",
    "Options for exporting mgf file with merged spectra per consensusElement": "导出具有每个consensusElement合并光谱的mgf文件的选项",
    "Cosine similarity threshold for merged_spectra output": "合并光谱输出的余弦相似度阈值",
    "featureXML or consensusXML file": "特征XML或共识XML文件",
    "File type of output out_annotation (A copy of the input file, annotated with matching hits from the database.)": "输出文件类型 out_annotation（带有与数据库匹配命中的注释的输入文件副本。）",
    "This file contains the list of potential positive adducts that will be looked for in the database": "这个文件包含了将在数据库中查找的潜在正向加合物的列表",
    "This file contains the list of potential negative adducts that will be looked for in the database": "这个文件包含了将在数据库中查找的潜在负向加成物的列表",
    "Database files which contain the identifications": "包含鉴定结果的数据库文件",
    "Run tool in batch mode for -mapping": "批量运行工具以进行映射。",
    "Database input file(s), containing three tab-separated columns of mass, formula, identifie": "数据库输入文件，包含三列以制表符分隔的质量、分子式、标识符",
    "Run tool in batch mode for -struct": "批量运行工具以处理-struct参数",
    "Database input file(s), containing four tab-separated columns of identifier, name, SMILES, INCHI.The identifier should match with mapping file": "数据库输入文件，包含四列，以制表符分隔，分别为标识符、名称、SMILES、INCHI。标识符应与映射文件匹配。",
    "Tolerance allowed for accurate mass search": "允许的准确质量搜索的容差",
    "Positive or negative ionization mode": "正离子模式或负离子模式",
    "Computes a similarity score for each hit (only if the feature exhibits at least two isotopic mass traces)": "计算每个命中的相似度得分（仅当特征至少展示两个同位素质量迹线）",
    "Whether to filter AMS candidates mismatching available feature adduct annotation": "是否过滤不匹配可用特征加成注释的AMS候选物",
    "Keep features that did not yield any DB hit": "保留没有产生任何数据库命中的特征",
    "Use legacy (ProteinID/PeptideID based storage of metabolomics data) with mzTab-v1.0.0 as output format or novel Identification Data (ID) with mzTab-v2.0.0-M as output format (ID and its MzTab-M output is currently only support for featureXML files)": "使用旧版（基于ProteinID/PeptideID存储代谢组学数据）mzTab-v1.0.0作为输出格式，或使用新版鉴定数据（ID）mzTab-v2.0.0-M作为输出格式（ID及其MzTab-M输出目前仅支持featureXML文件）。",
    "[featureXML input only] Export column with available isotope trace intensities (opt_global_MTint)": "[仅 featureXML 输入] 导出具有可用同位素痕迹强度的列（opt_global_MTint）",
    "Input FASTA file, containing a database": "输入FASTA文件，包含数据库",
    "Input file containing identified peptides and proteins": "包含已识别肽段和蛋白质的输入文件",
    "Switch between white-/blacklisting": "切换白名单/黑名单",
    "Input experiment file, containing the wrongly sorted spectra": "输入实验文件，包含排序错误的谱图",
    "Number of spots in one column, until next row is spotted": "一列中的斑点数量，直到下一行被标记",
    "RT distance between two spots which is used to calculated pseudo RT": "两个斑点之间的保留时间（RT）距离，用于计算假定的保留时间（pseudo RT）。",
    "MS1 data centroid data": "MS1 数据质心数据",
    "Centroid window width": "质心窗口宽度",
    "Absolute isotope mass precision (Da)": "绝对同位素质量精度（道尔顿）",
    "Relative isotope mass precision": "相对同位素质量精度",
    "Minimal peak height": "最小峰高",
    "Minimal Centroid MS Signal Intensity": "最小质心质谱信号强度",
    "Precursor detection scan levels": "前体检测扫描水平",
    "MS1 max inter scan distance": "MS1 最大扫描间隔",
    "MS1 LC retention time resolution": "MS1 液相色谱保留时间分辨率",
    "FT peak detect MS1 intensity min threshold": "FT 峰值检测 MS1 强度最小阈值",
    "FT peak detect MS1 min nb peak members": "FT 峰值检测 MS1 最小峰值成员数",
    "Detectable isotope facto": "可检测同位素因子",
    "IntensityCV": "强度变异系数",
    "MS1 retention time tolerance (minutes)": "MS1 保留时间容差（分钟）",
    "MS1 m/z tolerance (ppm)": "MS1 m/z 容差（ppm）",
    "Activation of MS1 feature merging post processing": "激活MS1特征合并后处理",
    "Initial Apex Tr tolerance": "初始Apex Tr公差",
    "MS1 feature Tr merging tolerance": "MS1 特征 色谱峰 合并 公差",
    "Percentage of intensity variation between LC border peaks": "LC边界峰值之间的强度变化百分比",
    "PPM value for the m/z clustering of merging candidates": "PPM值用于合并候选物的m/z聚类",
    "start elution window (minutes)": "开始洗脱窗口（分钟）",
    "end elution window (minutes)": "结束洗脱窗口（分钟）",
    "MS1 feature mz range min": "MS1 特征 m/z 范围最小值",
    "MS1 feature mz range max": "MS1 特征 m/z 范围 最大值",
    "MS1 feature CHRG range min": "MS1 特征 CHRG 范围最小值",
    "MS1 feature CHRG range max": "MS1 特征 电荷范围 最大",
    "Analysis step to perform": "执行分析步骤",
    "Load tabular file with pixel coordinates and their classes": "加载带有像素坐标及其类别的表格文件",
    "Column with response (condition) values": "响应（条件）值的列",
    "Column with fold values - only neccessary for cvapply": "列有倍数值 - 仅对cvapply必要",
    "Select the method for classification": "选择分类方法",
    "Crossvalidation or analysis": "交叉验证或分析",
    "The number of PLS-DA components": "PLS-DA组分的数量",
    "The optimal number of PLS-DA components as indicated by cross-validations (minimum is 1)": "最优的偏最小二乘判别分析（PLS-DA）组分数量由交叉验证确定（最小值为1）",
    "Data scaling": "数据缩放",
    "Number of toplabels (m/z features) which should be written in tabular output": "在表格输出中应写入的顶级标签（m/z特征）的数量",
    "The number of OPLS-DA components": "OPLS-DA 组件的数量",
    "The optimal number of OPLS-DA components as indicated by cross-validations (minimum is 1)": "最优的OPLS-DA组分数由交叉验证确定（最少为1）",
    "Write out best r and s values": "写出最佳的 r 和 s 值",
    "The spatial neighborhood radius of nearby pixels to consider (r)": "考虑附近像素的空间邻域半径（r）",
    "The sparsity thresholding parameter by which to shrink the t-statistics (s).": "稀疏阈值参数，用于收缩t统计量（s）。",
    "The method to use to calculate the spatial smoothing kernels for the embedding. The 'gaussian' method refers to spatially-aware (SA) weights, and 'adaptive' refers to spatially-aware structurally-adaptive (SASA) weights": "用于计算嵌入的空间平滑核的方法。'gaussian'方法指的是空间感知（SA）权重，而'adaptive'指的是空间感知结构自适应（SASA）权重。",
    "Result from previous classification training": "先前分类训练的结果",
    "Which classification method was used": "使用了哪种分类方法",
    "Load annotations (optional, but allows accuracy calculations)": "加载注释（可选，但允许准确度计算）",
    "Load tabular file with pixel coordinates and the new response": "加载带有像素坐标和新响应的表格文件",
    "Column with new response values": "新响应值的列",
    "Results as .RData output": "结果以.RData格式输出",
    "Use numpress compression to achieve optimally small file size (attention: may cause small loss of precision; only for mzML data)": "使用numpress压缩以实现最优的文件大小（注意：可能会导致小幅度的精度损失；仅限mzML数据）",
    "Write full meta information into sqMass file (may require large amounts of memory)": "将完整的元信息写入sqMass文件（可能需要大量的内存）",
    "The batch size of the low memory conversion": "低内存转换的批量大小",
    "mProphet tsv output file: \"all_peakgroups.xls\"": "mProphet tsv 输出文件：“all_peakgroups.xls”",
    "input featureXML file": "输入 featureXML 文件",
    "FDR cutoff": "FDR截止值",
    "The search engine scores are split by charge if this flag is set": "如果设置了这个标志，搜索引擎得分将根据电荷进行分割",
    "If set only the top hits of every PeptideIdentification will be used": "如果只设置每个PeptideIdentification的顶部命中，将只使用这些顶部命中。",
    "If set errors will be written but ignored": "如果设置错误将被写入但忽略",
    "If set scores will be calculated as '1 - ErrorProbabilities' and can be interpreted as probabilities for correct identifications": "如果设定分数将被计算为'1 - 错误概率'，并且可以被解释为正确识别的概率",
    "Number of bins used for visualization": "用于可视化的箱数",
    "for 'Gumbel', the Gumbel distribution is used to plot incorrectly assigned sequences": "对于“Gumbel”，使用Gumbel分布来绘制错误分配的序列",
    "Bounds the number of iterations for the EM algorithm when convergence is slow": "限制当EM算法收敛速度慢时的迭代次数",
    "The negative logarithm of the convergence threshold for the likelihood increase": "似然增加的收敛阈值的负对数",
    "What to do with outliers:": "如何处理异常值：",
    "Only used, when top_hits_only set": "仅在设置 top_hits_only 时使用",
    "Input file or directory containing the data to convert": "输入包含要转换数据的文件或目录",
    "[pepXML, Sequest, Mascot, X": "[pepXML, Sequest, Mascot, X]",
    "[pepXML only] Experiment filename/path (extension will be removed) to match in the pepXML file ('base_name' attribute)": "[pepXML 仅限] 实验文件名/路径（将删除扩展名）以匹配 pepXML 文件中的（'base_name' 属性）",
    "[pepXML output only] Write output in the format of a PeptideProphet analysis result": "[pepXML 输出仅] 以 PeptideProphet 分析结果的格式写入输出",
    "[Percolator only] Which of the Percolator scores to report as 'the' score for a peptide hit": "[仅限Percolator] 报告肽段命中的“那个”分数时，应选择哪个Percolator分数",
    "[FASTA input + MzML output only] Parameters used to adjust simulation of the theoretical spectra": "[FASTA 输入 + MzML 输出仅] 用于调整理论谱模拟的参数",
    "Enzym used to digest the fasta proteins": "使用的酶来消化fasta蛋白质",
    "Number of allowed missed cleavages while digesting the fasta proteins": "在消化fasta蛋白质时允许的未切割数",
    "Minimum charge": "最小电荷",
    "Maximum charge": "最大电荷",
    "Manually set precursor charge": "手动设置前体电荷",
    "[Sequest only] Workaround to deal with .out files that contain": "[仅限Sequest] 处理包含的.out文件的解决方法",
    "[Mascot, pepXML, Percolator only] Regular expression used to extract the scan number or retention time": "[Mascot, pepXML, Percolator 仅适用] 用于提取扫描编号或保留时间的正则表达式",
    "[+mz_file only] Avoid overriding 'spectra_data' in protein identifications if 'mz_file' is given and 'spectrum_reference's are added/updated": "[+mz_file 仅] 如果提供了 'mz_file' 并且 'spectrum_reference's 被添加/更新，则避免覆盖蛋白质鉴定中的 'spectra_data'",
    "[+mz_file only] Avoid overriding 'spectrum_reference' in peptide identifications if 'mz_file' is given and a 'spectrum_reference' is already present": "[+mz_file 仅适用] 如果提供了 'mz_file' 并且已经存在 'spectrum_reference'，则避免覆盖肽段鉴定中的 'spectrum_reference'",
    "[+mz_file only] Annotate the identifications with ion matches from spectra in 'mz_file' using the given tolerance (in Da)": "[+mz_file 仅适用] 使用给定的容忍度（以 Da 为单位），根据 'mz_file' 中的谱图上的离子匹配来注释鉴定结果。",
    "[FASTA output only] Will concatenate the top peptide hits to one peptide sequence, rather than write a new peptide for each hit": "[仅限FASTA输出] 将顶级肽段命中结果合并为一个肽段序列，而不是为每个命中结果写入一个新的肽段",
    "[FASTA output only] Controls how many peptide hits will be exported": "[仅限FASTA输出] 控制将导出多少肽段命中",
    "Multiple output files can be selected": "可以选取多个输出文件",
    "Use file with m/z of interest to calculate their occurrence in each spectrum": "使用包含m/z值的文件来计算它们在每个谱图中的出现次数。",
    "For each spectrum the occurrence of the provided m/z values is counted": "对于每个谱图，提供的m/z值的出现次数被计算",
    "ppm range will be added in both directions to input m/z": "ppm范围将在输入m/z的两个方向上添加",
    "Pixel annotation can be used to summarize intensities per annotation group": "像素注释可以用来总结每个注释组的强度。",
    "Calculation for each m/z and all pixels of a annotation group": "为每个 m/z 和所有像素的注释组进行计算",
    "Select the tool for spatial clustering": "选择用于空间聚类的工具",
    "The number of principal components to calculate": "要计算的主成分数量",
    "The function used to calculate the singular value decomposition": "用于计算奇异值分解的函数",
    "Scaling of data before analysis": "数据分析前的缩放",
    "The number of clusters (k)": "聚类数（k）",
    "The method to use to calculate the spatial smoothing kernels for the embedding. The 'gaussian' method refers to spatially-aware (SA) clustering, and 'adaptive' refers to spatially-aware structurally-adaptive (SASA) clustering": "用于计算嵌入的空间平滑核的方法。'gaussian'方法指的是空间感知（SA）聚类，而'adaptive'指的是空间感知结构自适应（SASA）聚类。",
    "Number of toplabels (m/z) which should be written in tabular output": "在表格输出中应写入的顶级标签（m/z）的数量",
    "The initial number of clusters (k)": "初始聚类数（k）",
    "The sparsity thresholding parameter by which to shrink the t-statistics (s)": "稀疏阈值参数，用于收缩t统计量（s）",
    "Export first segmentation image as svg": "导出第一个分割图像为SVG格式",
    "set seed": "设置种子",
    "Select pixel filtering option": "选择像素过滤选项",
    "Tabular file with pixel coordinates": "带有像素坐标的表格文件",
    "Minimum value for x": "x的最小值",
    "Maximum value for x": "最大值 for x",
    "Minimum value for y": "y的最小值",
    "Maximum value for y": "最大值 for y",
    "Select m/z feature filtering option": "选择 m/z 特征过滤选项",
    "Tabular file with m/z features to keep": "带有要保留的 m/z 特征的表格文件",
    "Tabular file with m/z features to remove": "带有要移除的 m/z 特征的表格文件",
    "Window in which all m/z will be removed": "在该窗口中，所有m/z都将被移除",
    "Apply to protein scores instead of PSM scores": "应用于蛋白质分数而不是肽段匹配分数",
    "Name of the meta value to use as the new score": "要用作新分数的元值名称",
    "Orientation of the new score (are higher or lower values better?)": "新得分的方向（更高的值还是更低的值更好？）",
    "Name to use as the type of the new score (default: same as 'new_score')": "用作新得分类型的名称（默认：与“new_score”相同）",
    "Name to use for the meta value storing the old score (default: old score type)": "用于存储旧分数的元值名称（默认：旧分数类型）",
    "Input file containing the spectra": "输入包含谱图的文件",
    "Input file containing the linked mass peaks": "输入文件包含链接的质量峰",
    "Input file containing the protein database": "输入文件包含蛋白质数据库",
    "String that was appended (or prefixed - see 'prefix' flag below) to the accessions in the protein database to indicate decoy proteins": "在蛋白质数据库中，附加（或前缀，见下面的'prefix'标志）到访问号上的字符串，以表示诱饵蛋白",
    "Set to true, if the decoy_string is a prefix of accessions in the protein database": "如果 decoy_string 是蛋白质数据库中访问号的前缀，则设置为 true。",
    "Precursor filtering settings": "前体过滤设置",
    "Monoisotopic peak correction": "单同位素峰校正",
    "Fragment peak matching settings": "片段峰匹配设置",
    "Fragment mass tolerance for cross-link ions": "交联离子的片段质量容忍度",
    "Peptide modification settings": "肽段修饰设置",
    "Settings for digesting proteins into peptides": "蛋白质消化成肽段的设置",
    "Description of the cross-linker reagent": "交联试剂的描述",
    "Comma separated residues, that the first side of a bifunctional cross-linker can attach to": "逗号分隔的残基，双功能交联剂的一侧可以附着到这些残基上",
    "Comma separated residues, that the second side of a bifunctional cross-linker can attach to": "逗号分隔的残基，双功能交联剂的第二端可以附着在这些残基上",
    "Mass of the light cross-linker, linking two residues on one or two peptides": "轻交联剂的质量，连接一个或两个肽上的两个残基",
    "Mass of the isotopic shift between the light and heavy linkers": "轻质和重质连接物之间的同位素位移的质量",
    "Possible masses of the linker, when attached to only one peptide": "连接到仅一个肽段时，连接子的可能质量",
    "Name of the searched cross-link, used to resolve ambiguity of equal masses": "搜索的交联名称，用于解决等质量的歧义",
    "Additional algorithm settings": "额外的算法设置",
    "Number of top hits reported for each spectrum pai": "为每个谱图对报告的顶级命中数",
    "Set to true, if the input spectra should be deisotoped before any other processing steps": "如果输入的谱图在进行任何其他处理步骤之前应该进行去同位素处理，则设置为 true。",
    "Ion types to search for in MS/MS spectra": "在MS/MS谱图中搜索的离子类型",
    "Search for peaks of b-ions": "搜索 b-离子的峰值",
    "Search for peaks of y-ions": "搜索y离子的峰值",
    "Search for peaks of a-ions": "搜索α离子的峰值",
    "Search for peaks of x-ions": "搜索x-离子的峰值",
    "Search for peaks of c-ions": "搜索c离子的峰值",
    "Search for peaks of z-ions": "寻找z离子的峰值",
    "Search for neutral losses of H2O and H3N": "搜索中性损失的H2O和H3N",
    "Input file containing the decoy protein database": "输入文件包含伪蛋白质数据库",
    "Crosslink Identifications in either xquest.xml, idXML, or mzIdentML format (as produced by OpenPepXL)": "交叉链接鉴定结果可以是xquest.xml、idXML或mzIdentML格式（由OpenPepXL生成）",
    "Prefix of decoy protein ids": "诱饵蛋白质ID的前缀",
    "Filter for minimum precursor mass error (ppm) before FDR estimation": "在FDR估计之前，过滤最小前体质量误差（ppm）",
    "Filter for maximum precursor mass error (ppm) before FDR estimation": "在FDR估计之前过滤最大前体质量误差（ppm）",
    "Filter for delta score, 0 disables the filte": "过滤delta得分，0禁用该过滤",
    "Filter for minimum matched ions per peptide": "过滤每个肽段的最小匹配离子数",
    "Calculate statistics based only on unique IDs": "仅基于唯一ID计算统计信息",
    "Do not transform simple FDR to q-values": "不要将简单的FDR转换为q值",
    "Minimum score to be considered for FDR calculation": "用于FDR计算的最小分数",
    "Bin size for the cumulative histograms for score distributions": "累积直方图的分箱大小用于得分分布",
    "Run tool in batch mode for -in_decoy": "批量运行工具以处理 -in_decoy 参数",
    "Input decoy file(s) in case of separate searches": "输入诱饵文件（如果有单独搜索的话）",
    "Input file in OSW format": "输入文件为OSW格式",
    "Type of enzyme: no_enzyme,elastase,pepsin,proteinasek,thermolysin,chymotrypsin,lys-n,lys-c,arg-c,asp-n,glu-c,trypsin,trypsinp": "酶类型：无酶，弹性蛋白酶，胃蛋白酶，蛋白酶K，嗜热菌蛋白酶，胰蛋白酶，赖氨酸C末端特异性蛋白酶，赖氨酸N末端特异性蛋白酶，精氨酸C末端特异性蛋白酶，天冬氨酸C末端特异性蛋白酶，色氨酸N末端特异性蛋白酶，胰蛋白酶，改良胰蛋白酶",
    "Calculate peptide-level FDRs instead of PSM-level FDRs": "计算肽段级别的错误发现率（FDR），而不是肽段匹配（PSM）级别的错误发现率。",
    "Use the picked protein-level FDR to infer protein probabilities": "使用选择的蛋白质水平FDR（假发现率）来推断蛋白质概率",
    "OSW: the data level selected for scoring": "OSW：选定用于评分的数据级别",
    "Type of the peptide main score": "肽段主分数类型",
    "Use only generic": "仅使用通用的",
    "Only train an SVM on a subset of <x> PSMs, and use the resulting score vector to evaluate the other PSMs": "仅在小于x的PSM子集上训练SVM，并使用得到的得分向量来评估其他PSMs。",
    "Cpos, penalty for mistakes made on positive examples": "Cpos，对正例上犯的错误的惩罚",
    "Cneg, penalty for mistakes made on negative examples": "Cneg，对负样本上犯的错误的惩罚",
    "False discovery rate threshold for evaluating best cross validation result and the reported end result": "错误发现率阈值用于评估最佳交叉验证结果和报告的最终结果",
    "False discovery rate threshold to define positive examples in training": "在训练中定义阳性样本的假发现率阈值",
    "Number of nested cross-validation bins in the 3 splits": "在3个拆分中嵌套交叉验证的桶数",
    "Quicker execution by reduced internal cross-validation": "更快的执行通过减少内部交叉验证",
    "Read initial weights to the given file": "将初始权重读入给定的文件",
    "Use static model (requires init-weights parameter to be set)": "使用静态模型（需要设置init-weights参数）",
    "The most informative feature given as the feature name, can be negated to indicate that a lower value is bette": "给出的特征名称是最有信息量的特征，可以否定它以表示较低的值更好",
    "Use unit normalization [0-1] instead of standard deviation normalization": "使用单位标准化[0-1]而不是标准差标准化",
    "Measure performance on test set each iteration": "在每次迭代中测量测试集上的性能",
    "Override error check and do not fall back on default score vector in case of suspect score vecto": "覆盖错误检查，并且在怀疑得分向量的情况下不要退回到默认得分向量",
    "Setting seed of the random number generato": "设置随机数生成器的种子",
    "Include description of correct features": "包含正确特征的描述",
    "Retention time features calculated as in Klammer et al": "保留时间特征计算如 Klammer 等人所述",
    "Provide the fasta file as the argument to this flag, which will be used for protein grouping based on an in-silico digest (only valid if option -protein_level_fdrs is active)": "将fasta文件作为该标志的参数提供，该文件将用于基于in-silico消化的蛋白质分组（仅当激活选项-protein_level_fdrs时有效）",
    "Define the text pattern to identify the decoy proteins and/or PSMs, set this up if the label that identifies the decoys in the database is not the default (Only valid if option -protein_level_fdrs is active)": "定义用于识别诱饵蛋白和/或PSMs（肽段鉴定）的文本模式，如果用于在数据库中识别诱饵的标签不是默认的，则设置此选项（仅当激活-protein_level_fdrs选项时有效）。",
    "Use target-decoy competition to assign q-values and PEPs": "使用目标-诱饵竞争法来分配q值和蛋白质误差概率（PEPs）",
    "Enforce that, for each spectrum, at most one PSM is included in the positive set during each training iteration": "确保在每次训练迭代期间，对于每个谱图，最多只有一个PSM包含在正集中",
    "OSW/IPF: Assess transitions only for candidate peak groups until maximum posterior error probability": "OSW/IPF：仅对候选峰值组评估转换，直到最大后验误差概率",
    "OSW/IPF: Maximum isotope overlap to consider transitions in IPF": "OSW/IPF：在IPF中考虑转换的最大同位素重叠",
    "OSW/IPF: Minimum log signal-to-noise level to consider transitions in IPF": "OSW/IPF：在IPF中考虑转换的最小对数信噪比水平",
    "input file in mzData format": "输入文件为mzData格式",
    "output file type (for TOPPAS)": "输出文件类型（用于TOPPAS）",
    "the instrument that was used to measure the spectra": "用于测量光谱的仪器",
    "the precursor mass tolerance": "前体质量容忍度",
    "the peak mass tolerance": "峰值质量容差",
    "the taxonomy": "分类学",
    "the variable modifications i.e. Carboxymethyl (C)": "变量修饰，即羧甲基化（C）",
    "the different charge states": "不同的电荷状态",
    "the database to search in": "要搜索的数据库",
    "the number of hits to report": "要报告的命中数",
    "number of allowed missed cleavages": "允许的断裂数",
    "significance threshold": "显著性阈值",
    "peptide homology threshold": "肽段同源性阈值",
    "peptide ident threshold": "肽段鉴定阈值",
    "peptide rank": "肽段排名",
    "protein score": "蛋白质得分",
    "peptide score": "肽段分数",
    "peptide expected charge": "肽段预期电荷",
    "show_unassigned": "显示未分配的",
    "additional information which is added to every peptide identification as metavalue if set > 0": "如果设置大于0，则将附加信息作为元值添加到每个肽段鉴定中",
    "MIME boundary for mascot output format": "MIME边界用于Mascot输出格式",
    "mass type": "质量类型",
    "Maximum allowed mass tolerance per feature": "每个特征允许的最大质量容差",
    "Adducts used to explain mass differences in format: 'Elements:Charge(+/-/0):Probability[:RTShift[:Label]]'": "用于解释质量差异的加成物，格式为：'元素：电荷(+/-/0):概率[:保留时间偏移[:标签]]'",
    "Prune the considered adduct transitions by transition probabilities": "根据转换概率修剪所考虑的加成转换",
    "Limits allowed adduct compositions and changes between compositions in the underlying graph optimization problem by introducing a probability-based threshold: the minority bound sets the maximum count of the least probable adduct (according to 'potential_adducts' param) within a charge variant with maximum charge only containing the most likely adduct otherwise": "通过引入基于概率的阈值，限制底层图优化问题中允许的加成物组成以及组成之间的变化：少数派界限设置了在最大电荷下仅包含最可能的加成物的电荷变体中，根据“potential_adducts”参数，最不可能的加成物的最大计数。",
    "Input featureXML file containing the features of the MRM experiment spectra": "输入包含MRM实验光谱特征的特征XML文件",
    "Pair-file in the format: prec-m/z-light prec-m/z-heavy frag-m/z-light frag-m/z-heavy rt": "配对文件的格式为：前体m/z轻 前体m/z重 碎片m/z轻 碎片m/z重 rt",
    "Maximal deviation in RT dimension in seconds the two partners of a pair is allowed to have": "允许一对伙伴在保留时间（RT）维度上的最大偏差（以秒为单位）",
    "Input consensusXML with peptide intensities": "输入带有肽段强度的共识XML",
    "Experimental Design file": "实验设计文件",
    "Method used in the experiment(label free [LFQ], isobaric labeling [ISO]))": "实验中使用的方法（无标记[LFQ]，同位素标记[ISO]）",
    "Which column in the condition table should be used for MSstats 'BioReplicate'": "在条件表中，应该使用哪一列作为MSstats的'BioReplicate'（生物学重复）？",
    "Which column in the condition table should be used for MSstats 'Condition'": "在条件表中，应该使用哪一列作为 MSstats 的 'Condition' 列？",
    "Which column in the condition table should be used for MSstats 'Mixture'": "在条件表中，应该使用哪一列作为MSstats的'Mixture'？",
    "If set, IsotopeLabelType is 'H', else 'L'": "如果设置，IsotopeLabelType 是 'H'，否则是 'L'",
    "Run tool in batch mode for -reannotate_filenames": "批量运行工具，针对参数 -reannotate_filenames",
    "Overwrite MS file names in consensusXML": "在consensusXML中覆盖MS文件名",
    "How indistinguishable peptidoforms at different retention times should be treated": "在不同的保留时间下，应该如何处理无法区分的肽异构体",
    "Input files used to generate the mzTab file": "用于生成mzTab文件的输入文件",
    "Add optional columns which are not part of the mzTab standard": "添加可选列，这些列不是mzTab标准的一部分",
    "Does the first IdentificationRun in the file only represent (protein) inference results": "文件中的第一个IdentificationRun是否仅代表（蛋白质）推断结果",
    "Export all PSMs instead of only the best per spectrum": "导出所有鉴定的肽段匹配（PSMs），而不是每个谱图只导出最佳匹配。",
    "Use a gaussian filter width which has approximately the same width as your mass peaks (FWHM in m/z)": "使用一个高斯滤波器宽度，该宽度大约与您的质量峰（m/z中的FWHM）相同",
    "Gaussian width, depending on the m/z position": "高斯宽度，取决于 m/z 位置",
    "If true, instead of the gaussian_width value, the ppm_tolerance is used": "如果为真，则使用ppm_tolerance而不是gaussian_width的值",
    "true: Warn if no signal was found by the Gauss filter algorithm": "true: 如果高斯滤波器算法没有发现信号，则发出警告",
    "Order or the polynomial that is fitted": "拟合的多项式的阶数",
    "Protein Fasta database": "蛋白质 Fasta 数据库",
    "minCharge": "最小电荷",
    "maxCharge": "最大电荷",
    "defaultCharge": "默认电荷",
    "maxMissedCleavage": "最大未切割位点",
    "minMz": "最小m/z值",
    "maxMz": "最大m/z",
    "Normalized Collision Energy": "归一化碰撞能量",
    "Digestion Enzyme": "消化酶",
    "align between files": "在文件之间对齐",
    "Use sequence tags (de novo sequencing of short fragments) to filter out candidates before scoring": "使用序列标签（短片段的去重复测序）在评分前过滤候选基因",
    "Minimal length of sequence tags to use for filtering candidates": "用于筛选候选序列标签的最小长度",
    "Prosit output csv file": "Prosit输出的csv文件",
    "Minimal distance to the edge to still consider a precursor, in Thomson": "最小距离到边缘，仍然考虑前体，在汤姆森单位",
    "Extraction window in RT dimension (-1 means extract over the whole range)": "提取窗口在保留时间维度（-1表示提取整个范围）",
    "Extraction window in m/z dimension (in Thomson, to use ppm see -ppm flag)": "提取窗口在 m/z 维度（单位为Thomson，若要使用ppm，请查看-ppm标志）",
    "m/z extraction_window is in ppm": "m/z 提取窗口以ppm为单位",
    "Set this flag if the data is SWATH data": "如果数据是SWATH数据，请设置此标志",
    "Extract the MS1 transitions based on the precursor values in the TraML file (useful for extracting MS1 XIC)": "根据TraML文件中的前体值提取MS1转换（适用于提取MS1 XIC）",
    "Only for 'linear' model: Perform linear regression on 'y - x' vs": "仅适用于“线性”模型：对'y - x'进行线性回归",
    "input file containing the chromatograms": "包含色谱图的输入文件",
    "run in non-strict mode and allow some chromatograms to not be mapped": "在非严格模式下运行，允许一些色谱图未被映射",
    "[applies only if you have full MS2 spectra maps] Swath files that were used to extract the transitions": "[仅适用于您拥有完整的MS2谱图] 用于提取转换的Swath文件",
    "[applies only if you have full MS2 spectra maps] Minimal distance to the edge to still consider a precursor, in Thomson (only in SWATH)": "[仅适用于您拥有完整的MS2谱图时] 考虑前体的最小边缘距离，以Thomson为单位（仅在SWATH中）",
    "Only extract RT around this value (-1 means extract over the whole range, a value of 500 means to extract around +/- 500 s of the expected elution)": "仅提取围绕此值的RT（保留时间）（-1表示在整个范围内提取，500的值表示围绕预期洗脱时间的正负500秒内提取）",
    "Sampling rate for the interpolation of the model function": "模型函数插值的采样率",
    "Bounding box has range [minimim of data, maximum of data] enlarged by tolerance_stdev_bounding_box times the standard deviation of the data": "边界框的范围是数据的最小值到最大值，再乘以数据标准差的tolerance_stdev_bounding_box倍。",
    "Centroid position of the model": "模型的质心位置",
    "Variance of the model": "模型的方差",
    "Use the elution model (EMG) score (this score fits a gaussian model to the peak and checks the fit)": "使用洗脱模型（EMG）得分（该得分对峰值拟合高斯模型并检查拟合度）",
    "Input file (SWATH/DIA file)": "输入文件（SWATH/DIA 文件）",
    "Library: Chromatagram .ELIB or Spectrum .DLIB": "库：色谱图 .ELIB 或 光谱 .DLIB",
    "transition file with the RT peptides ('TraML' or 'csv')": "过渡文件包含RT肽段（'TraML'或'csv'）",
    "Parameters for the RTNormalization": "RTNormalization的参数",
    "Parameters for the peptide estimation (use -estimateBestPeptides to enable)": "肽段估计的参数（使用-estimateBestPeptides启用）",
    "Input file with MS/MS spectra": "输入包含MS/MS谱图的文件",
    "Identification input file which contains a search against a concatenated sequence database": "识别输入文件，该文件包含对连接的序列数据库的搜索",
    "Fragment mass tolerance for spectrum comparisons": "谱图比较中的片段质量容忍度",
    "Restrict scoring to peptides with a length no greater than this value ('0' for 'no restriction')": "将评分限制在长度不超过此值的肽段上（'0'表示“无限制”）",
    "Maximum number of permutations a sequence can have to be processed ('0' for 'no restriction')": "序列可以处理的最大排列数（'0'表示“无限制”）",
    "Score to use for unambiguous assignments, where all sites on a peptide are phosphorylated": "用于无歧义分配的分数，其中肽段上的所有位点都发生了磷酸化",
    "Input file (centroided data)": "输入文件（质心化数据）",
    "Use features for precursor mass correction": "使用特征进行前体质量校正",
    "Features used to correct precursor masses": "用于校正前体质量的特征",
    "The precursor mass tolerance": "前体质量容差",
    "Additional retention time tolerance added to feature boundaries": "在特征边界上增加额外的保留时间容差",
    "Maximum isotopic trace considered in matching a precursor to a feature": "在匹配前体到特征时考虑的最大同位素迹线",
    "Assume precursor charge to be correct": "假设前体电荷是正确的",
    "Make a copy of the precursor and MS2 (true) or discard the original (false)": "复制前体和MS2（真）或丢弃原始（假）",
    "Correct a precursor using all matching features (true) or only the nearest (false)": "校正前体使用所有匹配的特征（true）或仅使用最近的（false）",
    "Use nearest centroided MS1 peak for precursor mass correction": "使用最近的质心化MS1峰进行前体质量校正",
    "The precursor mass tolerance to find the closest MS1 peak": "前体质量容差，用于寻找最接近的MS1峰",
    "Use centroided MS1 peak with the highest intensity in a certrain mass range - for precursor mass correction": "在特定质量范围内使用最高强度的质心化MS1峰进行前体质量校正",
    "The precursor mass tolerance to find the highest intensity MS1 peak": "前体质量容差，用于寻找最高强度的MS1峰",
    "Input mzML file containing the spectra": "输入包含谱图的mzML文件",
    "Input featureXML file, containing features; if set, the MS/MS spectra precursor entries": "输入featureXML文件，包含特征；如果设置，MS/MS谱前体条目",
    "Maximal deviation in Th which is acceptable to be corrected;": "最大偏差在Th中是可以接受的，需要被纠正；",
    "Maximal charge that should be assumed for precursor peaks": "前体峰应假定的最大电荷",
    "Intensity threshold value for isotope wavelet feature finder, please look at the documentation of the class for details": "同位素小波特征查找的强度阈值，请查看该类的文档以获取详细信息。",
    "raw data input file (this is relevant if you want to look at MS1, MS2 and precursor peak information)": "原始数据输入文件（如果您想查看MS1、MS2和前体峰信息，这与此相关）",
    "unique name for the run that can be used in a figure label": "在图表标签中可以使用的运行的唯一名称",
    "name of the person creating this mzQC file": "创建此mzQC文件的人的姓名",
    "contact address (mail/e-mail or phone)": "联系方式（邮件/电子邮件或电话）",
    "description and comments about the mzQC file contents": "关于mzQC文件内容的描述和评论",
    "feature input file (this is relevant for most QC issues)": "特征输入文件（这与大多数质量控制问题相关）",
    "consensus input file (this is only used for charge state deconvoluted output": "共识输入文件（这只用于电荷状态去卷积输出）",
    "This flag should be set, if you work with a set of merged features": "如果使用一组合并的特征，应设置此标志",
    "Extra arguments to be passed to tide-index": "传递给tide-index的额外参数",
    "Extra arguments to be passed to tide-search": "传递给tide-search的额外参数",
    "Extra arguments to be passed to percolato": "传递给percolato的额外参数",
    "Precursor monoisotopic mass tolerance (Crux parameter: peptide_mass_tolerance)": "前体单一同位素质量容差（Crux参数：peptide_mass_tolerance）",
    "Unit of precursor mass tolerance (amu, m/z or ppm)": "前体质量容差单位（阿姆斯特朗，m/z 或 ppm）",
    "In the discretization of the m/z axes of the observed and theoretical spectra, this parameter specifies the location of the left edge of the first bin, relative to mass = 0": "在观察到的和理论的质谱的 m/z 轴的离散化中，这个参数指定了第一个箱子的左边缘相对于质量 = 0 的位置。",
    "Before calculation of the XCorr score, the m/z axes of the observed and theoretical spectra are discretized": "在计算XCorr分数之前，观察到的和理论的质谱的m/z轴被离散化",
    "List of positive, non-zero integers": "正整数列表",
    "Whether to run percolator after tide-search": "是否在tide-search之后运行percolator",
    "Full, partial or non specific digestion": "完全消化、部分消化或非特异性消化",
    "Number of possible cleavage sites missed by the enzyme, maximum value is 5; for enzyme search": "酶可能错过的切割位点数量，最大值为5；用于酶搜索",
    "Decoy generation method either by reversing the sequence or shuffling it": "诱饵生成方法，可以通过反转序列或打乱序列来实现",
    "Whether to keep N and C terminal in place or also shuffled / reversed": "是否保留N端和C端的位置不变，还是也进行打乱/反转",
    "Specifies C-terminal static and variable mass modifications on peptides": "指定肽段的C端静态和可变质量修饰",
    "Specifies N-terminal static and variable mass modifications on peptides": "指定肽段的N端静态和可变质量修饰",
    "Expression for static and variable mass modifications to include": "静态和可变质量修饰的表达式，以包含",
    "False discovery rate threshold used in selecting hyperparameters during internal cross-validation and for reporting the final results": "在内部交叉验证期间选择超参数时使用的假发现率阈值，以及用于报告最终结果的假发现率阈值",
    "Specify rules for in silico digestion of protein sequences": "为蛋白质序列的计算机模拟消化指定规则",
    "Specifies the prefix of the protein names that indicate a decoy": "指定蛋白质名称的前缀，以表示诱饵蛋白质",
    "Deisotope spectra before searching": "去同位素化谱图后再进行搜索",
    "Include decoys in the final reported dataset": "在最终报告的数据集中包含诱饵",
    "The normalization algorithm that is applied": "应用的标准化算法",
    "Only for 'robust_regression': the parameter is used to distinguish between non-outliers (ratio_threshold < intensity ratio < 1/ratio_threshold) and outliers": "仅针对“robust_regression”：该参数用于区分非异常值（ratio_threshold < 强度比率 < 1/ratio_threshold）和异常值",
    "Use only features with accessions (partially) matching this regular expression for computing the normalization factors": "仅使用与此正则表达式（部分）匹配的访问号特征来计算标准化因子",
    "Use only features with description (partially) matching this regular expression for computing the normalization factors": "仅使用与此正则表达式（部分）匹配的描述特征来计算归一化因子",
    "A list of cv accessions that should be removed": "应移除的cv访问列表",
    "The name of the target run or set that contains the requested quality paramete": "目标运行或集合的名称，其中包含所请求的质量参数",
    "The file from which the name of the target run that contains the requested quality parameter is taken": "该文件包含了包含所请求的质量参数的目标运行的名称",
    "The table containing the additional qp values in the columns": "包含额外q值的表格在列中",
    "The mapping of the table header to the according qp cvs, also in csv format": "表格标题映射到相应的qp cvs（以csv格式）",
    "Number of randomly chosen peptides": "随机选择的肽段数量",
    "Number of rand invocations before random draw (basically a seed)": "随机抽样前随机调用的次数（基本上是一个种子）",
    "If this flag is set the best n peptides are chosen": "如果设置了这个标志，将选择最佳的n个肽段",
    "Target attachment qp": "目标附着 qp",
    "Input database file": "输入数据库文件",
    "Input file(s) holding experimental data": "输入文件（们）包含实验数据",
    "Path to idXMLs or consensusXMLs files": "idXMLs或consensusXMLs文件的路径",
    "Text file containing the experimental design": "包含实验设计的文本文件",
    "Additional options for algorithm": "算法的额外选项",
    "Digestion enzyme": "消化酶",
    "Additional options for quantitative experimental design": "定量实验设计的其他选项",
    "Identifier for the experimental design": "实验设计标识符",
    "Identifier for the file name": "文件名的标识符",
    "Separator, which should be used to split a row into columns": "分隔符，用于将一行分割成多列",
    "Overwrite tool specific checks": "覆盖工具特定检查",
    "Tag File": "标签文件",
    "PepPuery Tag File": "PepPuery标签文件",
    "Tag File Index Type": "标签文件索引类型",
    "pFind .qry.res": "pFind 查询结果",
    "Unmodified?": "未修改的？",
    "Max Charge": "最大电荷",
    "Add spectrum_file column": "添加光谱文件列",
    "Input Raw MS File(s)": "输入原始质谱文件",
    "Output Type": "输出类型",
    "Decoy Prefix": "诱饵前缀",
    "Cleavage Agent": "切割剂",
    "Cleavage Specificity": "切割特异性",
    "Static Modifications": "静态修饰",
    "Amino Acid": "氨基酸",
    "Mass": "质量",
    "Dynamic Modifications": "动态修饰",
    "Amino Acid Motif": "氨基酸基序",
    "Maximum Dynamic Mods": "最大动态修饰",
    "M/Z Tolerance Options": "M/Z 容差选项",
    "Precursor Mass Type": "前体质量类型",
    "Monoisotopic Precursor m/z Tolerance": "单同位素前体 m/z 容差",
    "Monoisotopic Precursor m/z Tolerance Units": "单同位素前体 m/z 容差单位",
    "Average Precursor m/z Tolerance": "平均前体 m/z 容差",
    "Average Precursor m/z Tolerance Units": "平均前体 m/z 容差单位",
    "Fragment m/z Tolerance": "片段 m/z 容差",
    "Fragment m/z Tolerance Units": "片段 m/z 容差 单位",
    "Use Max Peaks": "使用最大峰值",
    "Override Fragmentation Rule (Ion Series)": "覆盖碎片化规则（Ion系列）",
    "Fragmentation Rule": "片段化规则",
    "Minimum Peptide Length": "最小肽段长度",
    "Maximum Peptide Length": "最大肽段长度",
    "Use Smart Plus 3 Model": "使用Smart Plus 3模型",
    "Number of Charge States": "电荷态数量",
    "Maximum Result Rank": "最大结果排名",
    "Monoisotope Adjustment Set": "单同位素调整集",
    "Input idpDB file": "输入 idpDB 文件",
    "Embed Gene Metadata?": "嵌入基因元数据？",
    "Do you have the raw data (spectra)?": "你有原始数据（谱图）吗？",
    "Input raw MS files": "输入原始质谱文件",
    "Embed Spectrum Sources?": "嵌入谱图源？",
    "Embed Spectrum Scan Times?": "嵌入谱扫描时间？",
    "Quantitation Method": "定量方法",
    "Input mzIdentML/pepXML file(s)": "输入mzIdentML/pepXML文件",
    "Max FDR": "最大假发现率",
    "Max Import FDR": "最大导入假发现率",
    "Max Result Rank": "最大结果排名",
    "Max Training Rank": "最大训练等级",
    "Rerank Matches": "重新排序匹配",
    "Gamma": "伽马",
    "Nu": "Nu",
    "True Positive Threshold": "真正阳性阈值",
    "Polynomial Degree": "多项式次数",
    "Minimum Partition Size": "最小分区大小",
    "SVM Type": "支持向量机类型",
    "Kernel": "核心",
    "Charge State Handling": "电荷状态处理",
    "Terminal Specificity Handling": "终端特异性处理",
    "Missed Cleavages Handling": "未切割位点处理",
    "Mass Error Handling": "质量误差处理",
    "Qonverter Method": "Qonverter 方法",
    "Custom Score Settings": "自定义分数设置",
    "Myrimatch (mvh) Score/Weight": "Myrimatch（匹配质量）得分/权重",
    "Myrimatch (mvh) Weight": "Myrimatch（匹配质量）权重",
    "Myrimatch (mvh) Normalization": "Myrimatch（mvh）标准化",
    "Non-Sequest (xcorr) Score/Weight": "非Sequest (xcorr) 分数/权重",
    "XCorr Weight": "XCorr 权重",
    "XCorr Normalization": "XCorr 归一化",
    "Sequest (xcorr) Score/Weight": "Sequest（xcorr）得分/权重",
    "Sequest (xcorr) Weight": "Sequest（xcorr）权重",
    "Sequest (xcorr) Normalization": "Sequest（xcorr）归一化",
    "Sequest (deltacn) Score/Weight": "Sequest（deltacn）得分/权重",
    "Sequest (deltacn) Weight": "Sequest（deltacn）权重",
    "Sequest (deltacn) Normalization": "Sequest（deltacn）标准化",
    "Mascot (score) Score/Weight": "Mascot（得分）得分/权重",
    "Mascot Score Weight": "Mascot 得分权重",
    "Mascot Score Normalization": "Mascot得分归一化",
    "X! Tandem (expect) Score/Weight": "X! Tandem（期望）得分/权重",
    "X! Tandem (expectation) Weight": "X! Tandem（期望）权重",
    "X! Tandem (expectation) Normalization": "X! 串联（期望）标准化",
    "X! Tandem (hyperscore) Score/Weight": "X! Tandem（超得分）得分/权重",
    "X! Tandem (hyperscore) Weight": "X! Tandem（超得分）权重",
    "X! Tandem (hyperscore)Normalization": "X! Tandem（超得分）标准化",
    "MS-GF+ (specevalue) Score/Weight": "MS-GF+（特异性评分）得分/权重",
    "MS-GF+ (specevalue) Weight": "MS-GF+（特异性评分）权重",
    "MS-GF+ (specevalue) Normalization": "MS-GF+（特异性值）标准化",
    "Comet (xcorr) Score/Weight": "COMET（交叉相关性）得分/权重",
    "Comet (xcorr) Weight": "COMET（交叉相关）权重",
    "Comet (xcorr) Normalization": "COMET（xcorr）归一化",
    "Comet (deltacn) Score/Weight": "彗星（deltacn）得分/权重",
    "Comet (deltacn) Weight": "Comet（deltacn）权重",
    "Comet (deltacn) Normalization": "COMET（deltaCn）标准化",
    "Comet (expect) Score/Weight": "COMET（期望）得分/权重",
    "Comet (expect) Weight": "COMET（期望）权重",
    "Comet (expect) Normalization": "Comet（期望）归一化",
    "Extra Search Engine Score/Weight": "额外搜索引擎得分/权重",
    "Normalization Method": "标准化方法",
    "Input idpDB(s)": "输入 idpDB(s)",
    "Max FDR Score": "最大FDR得分",
    "Filter at Gene Level": "基因水平的过滤",
    "Min Distinct Peptides per Gene Group": "每个基因组最小不同肽段数",
    "Min Filtered Spectra per Gene Group": "每个基因组的最小过滤谱图数",
    "Min Additional Peptides per Gene Group": "每组基因的最小额外肽段数",
    "Min Distinct Peptides per Protein Group": "每组蛋白的最小独特肽段数",
    "Min Filtered Spectra per Protein Group": "每个蛋白质组的最小筛选光谱数",
    "Min Additional Peptides per Protein Group": "每组蛋白质最小额外肽段数",
    "Min Filtered Spectra per Distinct Match": "每个不同匹配的最小筛选光谱",
    "Min Filtered Spectra per Distinct Peptide": "每个独特肽段的最小过滤谱图数量",
    "Max Protein Groups per Distinct Peptide": "每个独特肽段的最大蛋白质组数",
    "Assign sources to a hierarchy?": "将来源分配到一个层级结构中？",
    "Assign source files to groups": "将源文件分配给组",
    "Assign sample names to reporter ions?": "为报告离子分配样本名称？",
    "Assign sample names to reporter ions": "为报告离子分配样本名称",
    "MS Spectrum files": "质谱谱图文件",
    "Export format, either matrix, legacy_split, legacy_merged (mProphet/PyProphet) or score_plots format": "导出格式，可以是矩阵、legacy_split、legacy_merged（mProphet/PyProphet）或得分图格式",
    "Report aggregated transition-level quantification": "报告聚合的转录水平定量",
    "Maximum PEP to retain scored transitions for quantification (requires transition-level scoring)": "保留用于定量的得分转换的最大PEP（需要转换级别的评分）",
    "Should IPF results be reported if present? 'peptidoform': Report results on peptidoform-level, 'augmented': Augment OpenSWATH results with IPF scores, 'disable': Ignore IPF results": "如果存在IPF结果，是否应该报告？'peptidoform': 在肽形式级别报告结果，'augmented': 用IPF分数增强OpenSWATH结果，'disable': 忽略IPF结果",
    "IPF: Filter results to maximum run-specific peptidoform-level PEP": "IPF：将结果过滤为最大特定运行的肽形式水平PEP",
    "Filter results to maximum run-specific peak group-level q-value": "将结果过滤为最大运行特定峰值组级别q值",
    "Filter results to maximum global peptide-level q-value": "过滤结果以获得最大全局肽水平q值",
    "ilter results to maximum global protein-level q-value": "过滤结果以获取最大全局蛋白质水平q值",
    "Append peptide-level error-rate estimates if available": "如果可用，添加肽水平的错误率估计",
    "Append protein-level error-rate estimates if available": "如果可用，追加蛋白质水平的错误率估计",
    "Use swath2stats to export file for statsics": "使用 swath2stats 导出统计文件",
    "Study design tabular file": "研究设计表格文件",
    "Filter for fdr and number of replicates": "过滤FDR值和重复次数",
    "FFT. Ratio of false positives to true negatives, q-values from pyProphet stats output": "FFT. 假阳性与真阴性的比率，来自pyProphet统计输出的q值",
    "Option to set the number of magnitude for which the m_score threshold is decreased": "设置m_score阈值降低的幅度数量的选项",
    "FDR target.": "FDR目标。",
    "Number Replicates.": "重复次数。",
    "Filter for a maximum number of peptides per protein": "为每个蛋白质过滤最多肽段数",
    "Maximum number of peptides per protein.": "每个蛋白质的最大肽段数。",
    "Filter for a proteins that are supported by a minimum number of peptides": "筛选由最小数量的肽段支持的蛋白质",
    "Minimum number of peptides per protein": "每条蛋白质的最小肽段数",
    "Choose which module to run": "选择要运行的模块",
    "Choose the format for the MS/MS file": "选择MS/MS文件的格式",
    "RAW file(s)": "原始文件",
    "mzML file(s)": "mzML文件（们）",
    "Tolerance parameter": "容差参数",
    "moFF workflow": "moFF 工作流程",
    "retention time windows for XiC": "XiC的保留时间窗口",
    "retention time window for apex detection": "保留时间窗口用于峰值检测",
    "retention time window for the matched peak": "匹配峰的保留时间窗口",
    "Output the peptide summary?": "输出肽段摘要？",
    "Use only spectra of interest": "仅使用感兴趣的谱图",
    "Tabular file with coordinates": "带有坐标的表格文件",
    "File contains a header line": "文件包含一个标题行",
    "Spectra annotations": "光谱注释",
    "Use annotations from tabular file": "使用表格文件中的注释",
    "Column with pixel annotations": "带有像素注释的列",
    "Select a method": "选择一个方法",
    "Noise estimation function": "噪声估计函数",
    "Half window size (number of data points)": "半窗口大小（数据点数）",
    "Signal-to-noise-ratio": "信噪比",
    "Detect peaks on average mass spectra": "在平均质量谱图上检测峰值",
    "Minimal correlation": "最小相关性",
    "Size": "大小",
    "Warping methods": "变形方法",
    "Tolerance = abs(mz1 - mz2)/mz2": "容差 = abs(mz1 - mz2)/mz2",
    "Don't throw an error when less than 2 reference m/z were found in a spectrum": "当在谱图中发现少于2个参考 m/z 时，不要抛出错误。",
    "If TRUE the intensity values of MassSpectrum or MassPeaks objects with missing (NA) warping functions are set to zero": "如果为真，将缺失（NA）变形函数的MassSpectrum或MassPeaks对象的强度值设置为零",
    "Should empty spectra be removed": "是否移除空的谱图",
    "minFrequency": "最小频率",
    "Reference m/z values": "参考 m/z 值",
    "Column with m/z values": "m/z 值的列",
    "Spectrum wise alignment": "谱图智能对齐",
    "Bin creation rule": "分箱创建规则",
    "m/z filtering parameters are applied to all spectra or groups of spectra": "m/z 过滤参数应用于所有谱图或谱图组",
    "mergeWhitelists": "合并白名单",
    "m/z filtering parameters": "m/z 过滤参数",
    "Removal of all peaks which occur in less than minFrequency spectra": "移除在少于minFrequency个谱图中出现的峰",
    "Removal of all peaks which occur in less than minNumber spectra": "移除在少于minNumber个谱图中出现的所有的峰",
    "Either a 'LDA' or 'XGBoost' classifier is used for semi-supervised learning": "半监督学习中使用了“LDA”或“XGBoost”分类器",
    "XGBoost: Autotune hyperparameters": "XGBoost：自动调整超参数",
    "Apply PyProphet score weights file (osw format) instead of semi-supervised learning.": "应用PyProphet得分权重文件（osw格式）而不是半监督学习。",
    "The data level selected for scoring. 'ms1ms2' integrates both MS1- and MS2-level scores and can be used instead of 'ms2'-level results": "选择用于评分的数据级别。'ms1ms2'整合了MS1和MS2级别的分数，并且可以代替'ms2'级别的结果使用。",
    "Data fraction used for cross-validation of semi-supervised learning step": "用于半监督学习步骤交叉验证的数据比例",
    "Number of iterations for cross-validation of semi-supervised learning step": "半监督学习步骤交叉验证的迭代次数",
    "Initial FDR cutoff for best scoring targets": "最佳得分目标的初始FDR截止值",
    "Iteration FDR cutoff for best scoring targets": "迭代FDR截止值，用于最佳得分目标",
    "Number of iterations for semi-supervised learning step": "半监督学习步骤的迭代次数",
    "Main score to start semi-supervised-learning": "主要分数开始半监督学习",
    "Group identifier for calculation of statistics": "用于计算统计数据的组标识符",
    "Do parametric estimation of p-values": "进行参数化估计 p 值",
    "Compute positive false discovery rate (pFDR) instead of FDR": "计算正假发现率（pFDR）而不是假发现率（FDR）",
    "Use non-parametric estimation of p-values. Start value": "使用非参数估计的p值。起始值",
    "Use non-parametric estimation of p-values. End value": "使用非参数估计的p值。结束值",
    "Use non-parametric estimation of p-values. Steps value": "使用非参数估计的p值。步长值",
    "Either 'smoother' or 'bootstrap'; the method for automatically choosing tuning parameter in the estimation of pi_0, the proportion of true null hypotheses": "“平滑”或“自助法”；在估计pi_0（真实零假设的比例）时，自动选择调整参数的方法",
    "Number of degrees-of-freedom to use when estimating pi_0 with a smoother": "在使用平滑器估计pi_0时使用的自由度数量",
    "If True and pi0_method = 'smoother', pi0 will be estimated by applying a smoother to a scatterplot of log(pi0) estimates against the tuning parameter lambda": "如果为真并且pi0_method = 'smoother'，将通过将pi0的对数估计值与调整参数lambda的散点图应用平滑器来估计pi0",
    "If True, local FDR values >1 are set to 1": "如果为真，局部FDR值大于1的将被设置为1",
    "If True, local FDR values are non-decreasing with increasing p-values": "如果为真，局部FDR值随着p值的增加而增加",
    "Either a 'probit' or 'logit' transformation is applied to the p-values so that a local FDR estimate can be formed that does not involve edge effects of the [0,1] interval in which the p-values lie": "对p值应用“probit”或“logit”转换，以便形成一个不涉及p值所在的[0,1]区间边缘效应的局部FDR估计。",
    "Numeric value that is applied as a multiple of the smoothing bandwidth used in the density estimation": "在密度估计中使用的平滑带宽的倍数的数值",
    "Numeric value that is threshold for the tails of the empirical p-value distribution": "经验p值分布尾部的阈值数值",
    "Assess transitions only for candidate peak groups until maximum peak group rank": "仅对候选峰群进行转换评估，直到最大峰群排名",
    "Assess transitions only for candidate peak groups until maximum posterior error probability": "仅对候选峰群评估转换，直到最大后验误差概率",
    "Maximum isotope overlap to consider transitions in IPF": "最大同位素重叠，以考虑IPF中的跃迁",
    "Minimum log signal-to-noise level to consider transitions in IPF. Set -1 to disable this filter": "在IPF中考虑转换的最小对数信噪比水平。设置为-1以禁用此过滤器。",
    "Whether chromatogram probabilities for TRIC should be computed": "是否应该计算 TRIC 的色谱图概率",
    "Run in test mode with fixed seed": "在测试模式下运行，使用固定种子",
    "MS1 files with precursor spectra": "带有前体谱的MS1文件",
    "Spectral resolution at 400m/z": "在400 m/z处的光谱分辨率",
    "Centroided data?": "质心化数据？",
    "Instrument": "仪器",
    "MS level": "质谱级别",
    "Charge determination method": "电荷确定方法",
    "Min charge state": "最小电荷状态",
    "Max charge state": "最大电荷状态",
    "Sensitivity level": "敏感度水平",
    "Depth of combinatorial analysis": "组合分析的深度",
    "Max nr. of features for mz-window to solve": "最大特征数用于解决mz窗口问题",
    "Hardklör isotope distributions": "Hardklör 同位素分布",
    "Contaminant threshold in minutes": "污染阈值（以分钟为单位）",
    "Match tolerance": "匹配容差",
    "Gap tolerance": "间隙容忍度",
    "Max mass (Da)": "最大质量（道尔顿）",
    "Min mass (Da)": "最小质量（道尔顿）",
    "Mass accuracy (ppm)": "质量精度（ppm）",
    "MaxQuant output file msms.txt": "MaxQuant 输出文件 msms.txt",
    "MaxQuant output file evidence.txt": "MaxQuant 输出文件 evidence.txt",
    "Retention time alignment peptides": "保留时间对齐肽段",
    "RT alignment method": "实时对齐方法",
    "Template osw file": "模板 osw 文件",
    "Input files are from same run (deletes run information)": "输入文件来自同一运行（删除运行信息）",
    "Context to estimate protein-level FDR control": "估计蛋白质水平的FDR控制的上下文",
    "Subsample ratio used per input file (between 0 (0 %) and 1 (100 %))": "每个输入文件使用的子样本比例（介于0（0%）和1（100%）之间）",
    "identification file": "识别文件",
    "spectrum files": "谱图文件",
    "monoisotopic ppm tolerance": "单一同位素ppm容忍度",
    "isotopic distribution tolerance in ppm": "同位素分布容忍度（ppm）",
    "number of isotopes required to be observed": "观察到的同位素数量",
    "integrate peak areas (not recommended)": "整合峰值区域（不推荐）",
    "use only precursor charge state": "仅使用前体电荷状态",
    "match between runs": "匹配运行",
    "maximum MBR window in minutes": "最大MBR窗口（分钟）",
    "Use experimnetal design for normalization or protein fold-change analysis": "使用实验设计进行标准化或蛋白质倍数变化分析",
    "ExperimentalDesign.tsv": "实验设计.tsv",
    "normalize intensity results": "标准化强度结果",
    "Perform Bayesian protein fold-change analysis": "执行贝叶斯蛋白质折叠变化分析",
    "control condition for Bayesian protein fold-change analysis": "贝叶斯蛋白质丰度变化分析的对照条件",
    "fold-change cutoff": "折叠变化阈值",
    "use shared peptides for protein quantification": "使用共享肽段进行蛋白质定量",
    "require MS/MS ID in condition": "需要在条件中指定质谱/质谱鉴定",
    "number of markov-chain monte carlo iterations": "马尔可夫链蒙特卡洛迭代次数",
    "MGF/mzML file": "MGF/MzML 文件",
    "Isobaric reporter labeling type (MS2 level): iTRAQ or TMT": "等压报告标签类型（MS2水平）：iTRAQ 或 TMT",
    "You are using iTRAQ: 4Plex or 8Plex": "您正在使用iTRAQ：4Plex或8Plex",
    "Denominator reporter tag for ratio calculation (mention only base reporter integer value)": "比率计算的分母报告基因标签（仅提及基础报告基因的整数值）",
    "You are using TMT: 2Plex, 6Plex or 10Plex": "您正在使用TMT：2Plex、6Plex或10Plex",
    "Minimum number of reporter types required to consider a spectrum for quantitation. Default is 2": "用于定量的报告类型所需的最小数目。默认为2",
    "Spectrum area calculation method": "谱图面积计算方法",
    "Perform normalization": "执行标准化",
    "Reporter label maximum peak intensity threshold": "报告标签最大峰值强度阈值",
    "MS/MS (MS2) tolerance unit": "MS/MS (MS2) 容差单位",
    "Minimum MS/MS (MS2) tolerance range": "最小 MS/MS（MS2）容忍范围",
    "Maximum MS/MS (MS2) tolerance range": "最大 MS/MS（MS2）容忍范围",
    "Use default or from the history": "使用默认值或从历史记录中选择",
    "Select default purity correction type": "选择默认的纯度校正类型",
    "Select purity correction file from the history": "从历史记录中选择纯度校正文件",
    "Peptide List (without any header line)": "肽段列表（不带任何标题行）",
    "mz to sqlite (mzsqlite) file": "mz 到 SQLite (mzsqlite) 文件",
    "genomic mapping sqlite file": "基因组映射 SQLite 文件",
    "Match": "匹配",
    "Proteoforms": "蛋白质异构体",
    "Proteoform match criteria": "蛋白质形态匹配标准",
    "Integer range of error for PTM sites": "PTM位点的整数误差范围",
    "Proteoforms from Peptideshaker 2.x Archive": "来自Peptideshaker 2.x存档的蛋白质形态",
    "UniProt Accession list": "UniProt 访问号列表",
    "Ensembl identifier list": "Ensembl 标识符列表",
    "Variant Call Format Specification": "变异调用格式规范",
    "Chromosomes and base pairs": "染色体和碱基对",
    "SNP rsId list": "SNP 位点编号列表",
    "Simple list": "简单列表",
    "Peptide List with PTM types and sites": "带PTM类型和位点的肽段列表",
    "Proteoform match criteria. Only modified peptides.": "蛋白质形式匹配标准。仅限修饰的多肽。",
    "PTM position range": "PTM 位置范围",
    "Add Top Level Pathways in the search result.": "在搜索结果中添加顶级通路。",
    "Connection graphs": "连接图",
    "Database Processing Options": "数据库处理选项",
    "The decoy flag": "诱饵标志",
    "Location of the decoy flag: prefix or suffix": "诱饵标志的位置：前缀还是后缀",
    "Target decoy suffix": "目标-诱饵后缀",
    "Spectrum Matching Options": "谱图匹配选项",
    "Fixed Modifications": "固定修饰",
    "Variable Modifications": "变量修饰",
    "Enzymes": "酶",
    "Specificity": "特异性",
    "Precursor Ion Tolerance": "前体离子容忍度",
    "Fragment Tolerance Units": "片段容差单位",
    "Forward Ion": "正向离子",
    "Reverse Ion": "反向离子",
    "Minimum precursor isotope": "最小前体同位素",
    "Maximum precursor isotope": "最大前体同位素",
    "Search Engines Options": "搜索引擎选项",
    "X!Tandem Options": "X!Tandem 选项",
    "X!Tandem: Spectrum Import Related": "X!Tandem：与谱图导入相关",
    "X!Tandem: Dynamic Range": "X!Tandem：动态范围",
    "X!Tandem: Total Peaks": "X!Tandem：总峰值",
    "X!Tandem: Min Frag m/z": "X!Tandem：最小片段 m/z",
    "X!Tandem: Min Peaks": "X!Tandem：最小峰值",
    "X!Tandem: Noise Suppression": "X!Tandem：噪声抑制",
    "X!Tandem: Min Precursor Mass": "X!Tandem：最小前体质量",
    "X!Tandem: Parent monoisotopic mass isotope error": "X!Tandem：母离子单同位素质量同位素误差",
    "X!Tandem: Advanced Search Related": "X!Tandem：高级搜索相关",
    "X!Tandem: Quick Acetyl": "X!Tandem：快速乙酰化",
    "X!Tandem: Quick Pyrolidone": "X!Tandem：快速吡咯烷",
    "X!Tandem: Protein stP Bias": "X!Tandem：蛋白质stP偏差",
    "X!Tandem: PTM complexity": "X!Tandem：PTM复杂性",
    "X!Tandem: Output Related": "X!Tandem：输出相关",
    "X!Tandem: Maximum Valid Expectation Value": "X!Tandem：最大有效期望值",
    "X!Tandem: Output Proteins": "X!Tandem：输出蛋白质",
    "X!Tandem: Output Sequences": "X!Tandem：输出序列",
    "X!Tandem: Output Spectra": "X!Tandem：输出谱图",
    "X!Tandem: Output Histograms": "X!Tandem：输出直方图",
    "X!Tandem peptide model refinement": "X!Tandem 肽段模型优化",
    "X!Tandem: Maximum Valid Expectation Value, refinement": "X!Tandem：最大有效期望值，细化",
    "X!Tandem: Unanticipated cleavage, refinement": "X!Tandem：意外裂解，细化",
    "X!Tandem: semi-enzymatic cleavage, refinement": "X!Tandem：半酶解，细化",
    "X!Tandem: Use potential modifications for full refinement": "X!Tandem：在完整细化中使用潜在的修饰",
    "X!Tandem: Point mutations, refinement": "X!Tandem：点突变，细化",
    "X!Tandem: snAPs, refinement": "X!Tandem：snAPs，细化",
    "X!Tandem: Spectrum synthesis, refinement": "X!Tandem：谱图合成，精细化处理",
    "MyriMatch Options": "MyriMatch 选项",
    "MyriMatch: Minimum Peptide Length": "MyriMatch：最小肽段长度",
    "MyriMatch: Maximum Peptide Length": "MyriMatch：最大肽段长度",
    "MyriMatch: Minimum Precursor Mass": "MyriMatch：最小前体质量",
    "MyriMatch: Maximum Precursor Mass": "MyriMatch：最大前体质量",
    "MyriMatch: Maximum Number of Spectrum Matches": "MyriMatch：最大谱图匹配数",
    "MyriMatch: Max Variable PTMs per Peptide": "MyriMatch：每个肽段的最大可变修饰数量",
    "MyriMatch: Fragmentation Method": "MyriMatch：碎片化方法",
    "MyriMatch: Enzymatic Terminals": "MyriMatch：酶切末端",
    "MyriMatch: Use Smart Plus Three Option": "MyriMatch：使用Smart Plus Three选项",
    "MyriMatch: Compute Xcorr": "MyriMatch：计算Xcorr",
    "MyriMatch: TIC cutoff percentage": "MyriMatch：TIC截止百分比",
    "MyriMatch: Number of Intensity Classes": "MyriMatch：强度等级数量",
    "MyriMatch: Class Size Multiplier": "MyriMatch：类大小乘数",
    "MyriMatch: Number of Batches": "MyriMatch：批次数量",
    "MyriMatch: Maximum Peak Count": "MyriMatch：最大峰值计数",
    "MyriMatch: output format option": "MyriMatch：输出格式选项",
    "MSGF Options": "MSGF选项",
    "MSGF: Search Decoy Database": "MSGF：搜索诱饵数据库",
    "MSGF: MS/MS Detector": "MSGF: 质谱/质谱检测器",
    "MSGF: Fragmentation method": "MSGF：碎片化方法",
    "MSGF: Protocol": "MSGF：协议",
    "MSGF: Enzymatic Terminals": "MSGF：酶解末端",
    "MSGF: Minimum Peptide Length": "MSGF：最小肽段长度",
    "MSGF: Maximum Peptide Length": "MSGF：最大肽段长度",
    "MSGF: Max Variable PTMs per Peptide": "MSGF：每个肽段最大可变修饰数量",
    "MSGF: Number of Spectrum Matches": "MSGF：谱图匹配数",
    "MS-GF+ additional Output": "MS-GF+ 附加输出",
    "Number of tasks to use on the threads": "在线程上使用的任务数",
    "MSGF: Custom number of tasks to use on the threads": "MSGF：在线程上使用自定义数量的任务",
    "MS Amanda Options": "MS Amanda 选项",
    "MS Amanda: Generate Decoys": "MS阿曼达：生成诱饵",
    "MS Amanda: instrument": "MS阿曼达：仪器",
    "MS Amanda: Maximum Rank": "MS Amanda：最大秩",
    "MS Amanda: Use Monoisotopic Mass Values": "MS 阿曼达：使用单同位素质量值",
    "OMSSA Options": "OMSSA 选项",
    "OMSSA: Spectrum Related": "OMSSA：与谱图相关的",
    "OMSSA: Low Intensity Cutoff": "OMSSA：低强度截止",
    "OMSSA: High Intensity Cutoff": "OMSSA：高强度截止",
    "OMSSA: Intensity Increment": "OMSSA：强度增量",
    "OMSSA: Minimum Peak Count": "OMSSA：最小峰值计数",
    "OMSSA: Remove precursors": "OMSSA：移除前体",
    "OMSSA: Estimate charge": "OMSSA：估计电荷",
    "OMSSA: Estimate Plus One Charge": "OMSSA：估计加一个电荷",
    "OMSSA: Fraction of Peaks to estimate Charge 1": "OMSSA：用于估计电荷1的峰值比例",
    "OMSSA: Minimum Number of Precursors per Spectrum": "OMSSA：每个谱图的前体最小数量",
    "OMSSA: Scale Precursor Mass": "OMSSA：缩放前体质量",
    "OMSSA: Database Related": "OMSSA：与数据库相关的",
    "OMSSA: Map Sequences in Memory": "OMSSA：在内存中映射序列",
    "OMSSA: Cleave n-term Methionine": "OMSSA：切除N端甲硫氨酸",
    "OMSSA: Search Related": "OMSSA：搜索相关",
    "OMSSA: Neutron Mass": "OMSSA：中子质量",
    "OMSSA: Single Charge Window Width": "OMSSA：单电荷窗口宽度",
    "OMSSA: Double Charge Window Width": "OMSSA：双电荷窗口宽度",
    "OMSSA: Single Charge Window Peaks": "OMSSA：单电荷窗口峰值",
    "OMSSA: Double Charge Window Peaks": "OMSSA：双电荷窗口峰值",
    "OMSSA: Minimum Number of Annotated Peaks of Intense Ones": "OMSSA：最小注释峰值数量的强烈峰",
    "OMSSA: Minimum number of Annotated Peaks": "OMSSA：注释峰值的最小数量",
    "OMSSA: Maximum Number of m/z Ladders": "OMSSA：最大m/z梯子数",
    "OMSSA: Maximum Fragment Charge": "OMSSA：最大片段电荷",
    "OMSSA: Fragment Charge": "OMSSA：片段电荷",
    "OMSSA: Include First Forward Ion (b1) in Search": "OMSSA：在搜索中包含第一个前向离子（b1）",
    "OMSSA: Search Rewind": "OMSSA：搜索回溯",
    "OMSSA: Maximum Fragment per Series": "OMSSA：每个系列的最大片段数",
    "OMSSA: Use Correlation Correction": "OMSSA：使用相关性校正",
    "OMSSA: Consecutive Ion Probability": "OMSSA：连续离子概率",
    "OMSSA: Number of Hits per Spectrum per Charge": "OMSSA：每个谱图每个电荷的命中数",
    "OMSSA: Iterative Search Related": "OMSSA：迭代搜索相关",
    "OMSSA: Sequence e-value Cutoff": "OMSSA：序列 E 值截止",
    "OMSSA: Spectrum e-value Cutoff": "OMSSA：谱图 E 值截止",
    "OMSSA: Replace e-value cutoff": "OMSSA：替换e值截止值",
    "OMSSA: Semi-enzymatic Related": "OMSSA：半酶相关",
    "OMSSA: Minimum Peptide Length": "OMSSA：最小肽段长度",
    "OMSSA: Maximum Peptide Length": "OMSSA：最大肽段长度",
    "OMSSA: Output Related": "OMSSA：输出相关",
    "OMSSA: Maximal evalue Considered": "OMSSA：最大 E 值考虑",
    "OMSSA: Hitlist Length": "OMSSA: 命中列表长度",
    "OMSSA output format": "OMSSA 输出格式",
    "Comet Options": "COMET选项",
    "Comet: Spectrum Related": "Comet：与谱图相关的",
    "Comet: Minimum Number of Peaks per Spectrum": "Comet：每个谱图的最小峰值数量",
    "Comet: Minimum Peaks Intensity": "Comet：最小峰值强度",
    "Comet: Remove Precursor peaks": "Comet：移除前体峰",
    "Comet: Remove Precursor Tolerance": "Comet：移除前体容忍度",
    "Comet: Maximum Peaks Intensity": "Comet：最大峰值强度",
    "Comet: Search Related": "Comet：搜索相关",
    "Comet: Enzyme Type": "Comet：酶类型",
    "Comet: Isotope Correction": "Comet：同位素校正",
    "Comet: Minimum Precursor Mass": "Comet：最小前体质量",
    "Comet: Maximum Precursor Mass": "COMET：最大前体质量",
    "Comet: Maximum Fragment Charge": "Comet：最大片段电荷",
    "Comet: Remove Methionine": "Comet：去除甲硫氨酸",
    "Comet: Batch Size": "Comet：批量大小",
    "Comet: Maximum Number of PTMs": "Comet：最大PTMs数量",
    "Comet: Require Variable PTMs": "Comet：需要变量 PTMs",
    "Comet: Fragment Ions Related": "Comet：与片段离子相关的",
    "Comet: Correlation Score type": "Comet：相关性得分类型",
    "Comet: Fragment Bin Offset": "Comet：片段二进制偏移量",
    "Comet: Output Related": "Comet: 输出相关",
    "Comet: Maximum Number of Matches": "COMET：最大匹配数",
    "Comet: Output type": "Comet: 输出类型",
    "TIDE Options": "TIDE 选项",
    "Tide: Index Related": "Tide：索引相关",
    "TIDE: Minimum Peptide Length": "TIDE：最小肽段长度",
    "TIDE: Maximum Peptide Length": "TIDE：最大肽段长度",
    "TIDE: Minimum Precursor Mass": "TIDE：最小前体质量",
    "TIDE: Maximum Precursor Mass": "TIDE：最大前体质量",
    "TIDE: Monoisotopic": "TIDE: 单同位素",
    "TIDE: Clip Nterm Methionine": "TIDE：剪切N端甲硫氨酸",
    "TIDE: Maximum Number of PTMs": "TIDE：最大PTMs数量",
    "TIDE: Maximum Number of PTMs of each Type": "TIDE: 每种类型PTMs的最大数量",
    "TIDE: Enzyme/Digestion Type": "TIDE：酶/消化类型",
    "TIDE: Print Peptides": "TIDE：打印肽段",
    "TIDE: Decoy Format": "TIDE：诱饵格式",
    "TIDE: Keep Terminals": "TIDE：保留终端",
    "TIDE: Decoy Seed": "TIDE：诱饵种子",
    "TIDE: Remove Temp Folders": "TIDE：删除临时文件夹",
    "Tide: Search Related": "Tide：搜索相关",
    "TIDE: Compute Exact P-value": "TIDE：计算精确P值",
    "TIDE: Compute SP": "TIDE：计算SP",
    "TIDE: Minimum Spectrum m/z": "TIDE：最小谱图 m/z",
    "TIDE: Maximum Spectrum m/z": "TIDE：最大谱图 m/z",
    "TIDE: Minimum Spectrum Peaks": "TIDE：最小谱峰",
    "TIDE: Spectrum Charges": "TIDE：谱图电荷",
    "TIDE: Remove Precursor Peak": "TIDE：移除前体峰",
    "TIDE: Remove Precursor Tolerance": "TIDE: 移除前体容差",
    "TIDE: Use Flanking": "TIDE：使用侧翼",
    "TIDE: Neutral Losses": "TIDE：中性损失",
    "TIDE: mz Bin Width": "TIDE：m/z 箱宽",
    "TIDE: mz Bin Offset": "TIDE：mz 桶偏移量",
    "TIDE: Maximum Number of PSMs": "TIDE: 最大PSM数",
    "TIDE: Output Format": "TIDE：输出格式",
    "TIDE: Output Folder": "TIDE：输出文件夹",
    "TIDE: Progress Display Verbosity": "TIDE：进度显示详细度",
    "TIDE: Progress Indicator": "TIDE：进度指示器",
    "TIDE: Concat Target and Decoy": "TIDE：合并目标和诱饵",
    "TIDE: Store spectra": "TIDE：存储谱图",
    "DirectTag Options": "DirectTag 选项",
    "DirecTag tag length": "DirecTag 标签长度",
    "DirecTag maximum variable modifications per sequence": "DirecTag 每个序列最大变量修饰",
    "DirecTag number of charge states considered": "DirecTag 考虑的电荷状态数量",
    "DirecTag duplicate spectra per charge": "DirecTag 每个电荷重复的谱图",
    "DirecTag isotope mz tolerance": "DirecTag 同位素 m/z 容差",
    "DirecTag deisotoping mode": "DirecTag去异构体模式",
    "DirecTag number of intensity classses.": "DirecTag 强度类别的数量。",
    "DirecTag output suffix": "DirecTag 输出后缀",
    "DirecTag max peak count": "DirecTag 最大峰值计数",
    "DirecTag maximum tag count": "DirecTag 最大标签计数",
    "DirecTag TIC cutoff in percent": "DirecTag TIC 截止百分比",
    "DirecTag complement mz tolerance": "DirecTag 互补质量容忍度",
    "DirecTag precursor adjustment step": "DirecTag 前体调整步骤",
    "DirecTag minimum precursor adjustment": "DirecTag 最小前体调整",
    "DirecTag maximum precursor adjustment": "DirecTag 最大前体调整",
    "DirecTag intensity score weight": "DirecTag 强度得分权重",
    "DirecTag fidelity score weight": "DirecTag 保真度得分权重",
    "DirecTag complement_score_weight": "DirecTag 互补配对分数权重",
    "DirecTag adjust precursor": "DirecTag 调整前体",
    "DirecTag use charge state from M spectrum": "DirecTag 使用 M 谱图中的电荷状态",
    "MetaMorpheus Options": "MetaMorpheus 选项",
    "MetaMorpheus: Search related options": "MetaMorpheus：搜索相关选项",
    "MetaMorpheus search type": "MetaMorpheus 搜索类型",
    "MetaMorpheus number of partitions when indexing": "MetaMorpheus 在索引时的分区数量",
    "MetaMorpheus dissociation type": "MetaMorpheus 分解类型",
    "MetaMorpheus maximum modifications per peptide": "MetaMorpheus 每肽段最大修饰数",
    "MetaMorpheus initiator methionine behavior": "MetaMorpheus 发起者 甲硫氨酸 行为",
    "MetaMorpheus score cutoff": "MetaMorpheus 分数截止值",
    "MetaMorpheus use delta score": "MetaMorpheus 使用 delta 分数",
    "MetaMorpheus mass difference acceptor type": "MetaMorpheus 质量差异接受器类型",
    "MetaMorpheus minimum peptide length": "MetaMorpheus 最小肽段长度",
    "MetaMorpheus maximum peptide length": "MetaMorpheus 最大肽段长度",
    "MetaMorpheus: Output related options": "MetaMorpheus：输出相关选项",
    "MetaMorpheus write mzid": "MetaMorpheus 写入 mzid",
    "MetaMorpheus write pepxml": "MetaMorpheus 写入 pepXML",
    "MetaMorpheus: Deisotoing related options": "MetaMorpheus：去卷积相关选项",
    "MetaMorpheus use provided precursor info": "MetaMorpheus 使用提供的前体信息",
    "MetaMorpheus do precursor deconvolution": "MetaMorpheus 进行前体解卷积",
    "MetaMorpheus deconvolution intensity ratio": "MetaMorpheus 解卷积强度比率",
    "MetaMorpheus deoconvolution mass tolerance type": "MetaMorpheus去卷积质量容差类型",
    "MetaMorpheus: Protein grouping related options": "MetaMorpheus：与蛋白质分组相关的选项",
    "MetaMorpheus modified peptides are different": "MetaMorpheus修改的肽段是不同的",
    "MetaMorpheus exclude one hit wonders": "MetaMorpheus 排除一次性命中",
    "MetaMorpheus: Peak trimming related options": "MetaMorpheus：峰值修剪相关选项",
    "MetaMorpheus trim MS1 peaks": "MetaMorpheus 修剪 MS1 峰",
    "MetaMorpheus trim MSMS peaks": "MetaMorpheus 修剪 MSMS 峰",
    "MetaMorpheus number of peaks per window": "MetaMorpheus 每个窗口的峰值数量",
    "MetaMorpheus minimum allowed intensity ratio to base peak": "MetaMorpheus 最低允许的基峰强度比",
    "MetaMorpheus window width in Thompson": "MetaMorpheus 窗口宽度在 Thompson",
    "MetaMorpheus number of windows": "MetaMorpheus 窗口数量",
    "MetaMorpheus normalize peaks across all windows": "MetaMorpheus 标准化所有窗口的峰值",
    "MetaMorpheus: Silico digestion related options": "MetaMorpheus：计算机模拟消化相关选项",
    "MetaMorpheus fragmentation terminus": "MetaMorpheus 片段化末端",
    "MetaMorpheus maximum fragmentation size": "MetaMorpheus 最大碎片大小",
    "MetaMorpheus search target sequences": "MetaMorpheus 搜索目标序列",
    "MetaMorpheus decoy type": "MetaMorpheus 诱饵类型",
    "MetaMorpheus maximum modified isoforms": "MetaMorpheus 最大修饰异构体",
    "MetaMorpheus minimum variant depth": "MetaMorpheus 最小变体深度",
    "MetaMorpheus maximum hetrozygous variants": "MetaMorpheus 最大杂合变异",
    "MetaMorpheus: G-PTM search related options": "MetaMorpheus：G-PTM搜索相关选项",
    "MetaMorpheus run G-PTM": "MetaMorpheus 运行 G-PTM",
    "MetaMorpheus G-PTM categories to include in the G-PTM search": "MetaMorpheus G-PTM 搜索中要包含的类别",
    "Novor Options": "Novor选项",
    "Novor fragmentation method": "Novor 断裂方法",
    "Novor: mass analyzer": "Novor：质谱分析仪",
    "PepNovo+ Options": "PepNovo+ 选项",
    "PepNovo+ number of de novo solutions [0-2000], default is '10'.": "PepNovo+ 去卷积解决方案的数量 [0-2000]，默认为 '10'。",
    "PepNovo+ estimate precursor charge option.": "PepNovo+ 估计前体电荷选项。",
    "PepNovo+ correct precursor mass option.": "PepNovo+ 校正前体质量选项。",
    "PepNovo+ discard low quality spectra option.": "PepNovo+ 丢弃低质量谱图选项。",
    "PepNovo+ generate a BLAST query.": "PepNovo+ 生成一个 BLAST 查询。",
    "pNovo+ Options": "pNovo+ 选项",
    "pNovo+ number of peptides per spectrum, default is '10'.": "pNovo+ 每个谱图的肽段数，默认值为 '10'。",
    "pNovo+ minimum precursor mass, default is '300'.": "pNovo+ 最小前体质量，默认值为 '300'。",
    "pNovo+ maximum precursor mass, default is '5000'.": "pNovo+ 最大前体质量，默认值是 '5000'。",
    "pNovo+ activation type (HCD, CID or EDT)": "pNovo+ 激活类型（HCD、CID 或 EDT）",
    "Show/Hide advanced options": "显示/隐藏高级选项",
    "Spectrum Annotation Options": "谱图注释选项",
    "Percent of the possible annotations relative to the most intensive peak": "相对于最密集峰值的可能注释的百分比",
    "Fragment ion annotation accuracy": "片段离子注释准确性",
    "High resolution": "高分辨率",
    "Sequence Matching Options": "序列匹配选项",
    "The peptide to protein sequence matching type": "肽段到蛋白质序列的匹配类型",
    "The maximal share of Xs in a sequence": "序列中Xs的最大比例",
    "Tags should only be mapped to enzymatic peptides": "标签应仅映射到酶解肽段",
    "The maximum number of PTMs per peptide when mapping tags": "每个肽段在映射标签时的最大PTM（翻译后修饰）数量",
    "The minimum amino acid score when mapping tags": "映射标签时的最小氨基酸得分",
    "The minimum tag length when mapping tags": "映射标签时的最小标签长度",
    "Import filters": "导入过滤器",
    "Maximum Precursor Error": "最大前体误差",
    "Maximum Precursor Error Type": "最大前体误差类型",
    "Minimal number of missed cleavages to consider when importing identification files": "导入鉴定文件时考虑的最小未切割数",
    "Maximal number of missed cleavages to consider when importing identification files": "在导入鉴定文件时考虑的最大未断裂数",
    "Exclude Unknown PTMs": "排除未知的PTMs",
    "PTM Localizacion": "PTM 定位",
    "The PTM probabilistic score to use for PTM localization": "用于PTM定位的PTM概率分数",
    "Include Neutral Losses in A Score": "在 A 分数中包含中性损失",
    "The threshold to use for the PTM scores": "用于PTM分数的阈值",
    "The modification to peptide sequence matching type.": "对肽段序列匹配类型的修改。",
    "Align peptide ambiguously localized PTMs on confident sites": "在确定的位点上对模糊定位的磷酸化修饰（PTMs）的多肽进行比对",
    "Gene Annotation": "基因注释",
    "gene mappings will be used and saved along with the project (UniProt databases only)": "基因映射将被使用并保存在项目中（仅限UniProt数据库）",
    "Update gene mappings automatically from Ensembl (UniProt databases only)": "自动从Ensembl更新基因映射（仅限UniProt数据库）",
    "Protein Inference": "蛋白质推断",
    "Simplify protein groups": "简化蛋白质组",
    "Based on Uniprot evidence level": "基于Uniprot证据水平",
    "Based on the peptide confidence": "基于肽段的置信度",
    "Confidence below which a peptide is ignored": "忽略肽段的置信度阈值",
    "Based on Enzymaticity": "基于酶活性",
    "Based on Variant Mapping": "基于变体映射",
    "Account for modifications when mapping peptides to proteins": "在将肽段映射到蛋白质时考虑修饰",
    "Validation Levels": "验证级别",
    "FDR at the protein level": "蛋白质水平的FDR",
    "FDR at the peptide level": "肽水平上的FDR（假发现率）",
    "FDR at the PSM level": "FDR在PSM水平上",
    "Fraction Analysis": "分数分析",
    "Minimum confidence required for a protein in the fraction MW plot (in percentage)": "蛋白质在分子量分布图（MW plot）中的最小置信度要求（以百分比表示）",
    "Create a concatenated target/decoy database before running PeptideShaker": "在运行PeptideShaker之前，创建一个合并的目标/诱饵数据库。",
    "Target decoy suffix.": "目标-诱饵后缀。",
    "Compressed SearchGUI results": "压缩的 SearchGUI 结果",
    "Optional main parameters": "可选的主要参数",
    "Identification parameters": "识别参数",
    "Spectrum files": "谱图文件",
    "Exporting options": "导出选项",
    "Compress results into a single zip file": "将结果压缩到一个单独的zip文件中",
    "Exports PeptideShakers‘ .psdb file": "导出PeptideShakers的.psdb文件",
    "Also export reports out of the zip": "同时将报告导出到zip文件之外",
    "Also export mgf file/s out of the zip": "同时将mgf文件/们从zip中导出",
    "Creates a mzIdentML file": "创建一个mzIdentML文件",
    "Include the protein sequences in the mzIdentML file": "在mzIdentML文件中包含蛋白质序列",
    "Contact Information": "联系信息",
    "Contact first name.": "联系人名字",
    "Contact last name.": "联系人姓氏。",
    "Contact e-mail.": "联系电子邮件。",
    "Contact address.": "联系地址。",
    "Contact URL.": "联系 URL。",
    "Organization name.": "组织名称。",
    "Organization e-mail.": "组织电子邮件。",
    "Organization address.": "组织地址。",
    "Organization URL.": "组织URL。",
    "Follow-up analysis export options": "后续分析导出选项",
    "Spectra: Export spectrum files as MGF": "谱图：将谱图文件导出为 MGF 格式",
    "Proteins: Export accession numbers as TXT": "蛋白质：将访问号码导出为TXT格式",
    "Proteins: Export protein details/sequences in fasta format": "蛋白质：以fasta格式导出蛋白质详细信息/序列",
    "Proteins: Export proteoforms as TXT": "蛋白质：将蛋白质形式导出为TXT格式",
    "Label free quantification: Exports identification results in Progenesis LC-MS compatible format as TXT": "无标记定量：将识别结果导出为Progenesis LC-MS兼容格式的TXT文件。",
    "Targeted PTMs": "靶向性翻译后修饰",
    "Inclusion/Exclusion list: Exports a file for an inclusion list of validated hits": "包含/排除列表：导出一个验证命中的包含列表文件",
    "Peptide Filters": "肽段过滤器",
    "Protein Filters": "蛋白质过滤器",
    "Retention time window for the inclusion list export (in seconds)": "包含列表导出的保留时间窗口（单位：秒）",
    "Identification features reports to be generated": "生成识别特征报告",
    "Identification Parameters file": "识别参数文件",
    "Input Peak Lists": "输入峰值列表",
    "Search Engine Options": "搜索引擎选项",
    "SearchGUI Options": "SearchGUI选项",
    "How should PeptideShaker deal with duplicate spectra?": "PeptideShaker 应该如何处理重复的谱图？",
    "Add missing spectrum titles": "添加缺失的谱图标题",
    "The maximum mgf file size in MB before splitting the mgf": "在分割 mgf 文件之前的最大 mgf 文件大小（以MB为单位）",
    "The maximum number of spectra per mgf file when splitting": "在拆分时，每个mgf文件的最大谱图数量",
    "Gzip result files": "使用Gzip压缩结果文件",
    "Raw file": "原始文件",
    "Do you agree to the vendor licenses?": "您同意供应商许可证吗？",
    "Parsing and Quantification": "解析和定量",
    "Create Matrix file": "创建矩阵文件",
    "Labeling reagent": "标记试剂",
    "Unlabeled quant": "未标记的数量",
    "Creates a mgf file": "创建一个mgf文件",
    "Mass cutoff": "质量截止",
    "Precursor mass and charge refinement": "前体质量与电荷精细化",
    "Create chromatogram": "创建色谱图",
    "Chromatogram type": "色谱图类型",
    "Chromatogram order": "色谱图顺序",
    "Create metrics": "创建指标",
    "Proteomics Spectrum  files in mzXML format": "蛋白质组学谱图文件，格式为mzXML",
    "File name prefix": "文件名前缀",
    "instrument used": "使用的仪器",
    "Maximum mass error for two MS1 peaks": "两个MS1峰的最大质量误差",
    "Maximum mass error for two MS2 peaks": "两个MS2峰值的最大质量误差",
    "Advanced Precursor-fragment Grouping Settings": "高级前体-片段组设置",
    "RPmax": "RPmax",
    "RFmax": "RFmax",
    "CorrThreshold": "相关性阈值",
    "RTOverlap": "RTOverlap",
    "DeltaApex": "DeltaApex",
    "BoostComplementaryIon": "增强互补离子",
    "AdjustFragIntensity": "调整片段强度",
    "Advanced Signal Extraction Settings": "高级信号提取设置",
    "MinMSIntensity": "最小质谱强度",
    "MinMSMSIntensity": "最小MSMS强度",
    "MaxCurveRTRange": "最大曲线实时范围",
    "SN Threshold": "信号阈值",
    "MS2SN Threshold": "MS2SN 阈值",
    "StartRT": "开始运行时间",
    "EndRT": "EndRT",
    "MinMZ": "最小m/z值",
    "MinPrecursorMass": "最小前体质量",
    "MaxPrecursorMass": "最大前体质量",
    "IsoPattern": "IsoPattern",
    "MassDefectFilter": "质量缺陷过滤器",
    "MassDefectOffset": "质量缺陷偏移",
    "MinRTRange": "最小读取范围",
    "MaxNoPeakCluster": "最大峰值簇数",
    "MinNoPeakCluster": "最小无峰簇",
    "MinMS2NoPeakCluster": "最小MS2无峰簇",
    "RTtol": "RTtol",
    "NoPeakPerMin": "每分钟无峰值",
    "NoMissedScan": "无遗漏扫描",
    "Denoise": "去噪",
    "EstimateBG": "EstimateBG",
    "RemoveGroupedPeaks": "移除分组峰值",
    "MinFrag": "最小片段长度",
    "ExportPrecursorPeak": "导出前体峰",
    "ExportFragmentPeak": "导出片段峰值",
    "Output MGFs as a collection": "将MGF文件输出为一个集合",
    "Will you select a genome annotation from your history or use a built-in annotation?": "您是要从历史记录中选择一个基因组注释，还是使用内置注释？",
    "Select genome annotation": "选择基因组注释",
    "Annotate SNPs with rsid from dbSNP": "使用dbSNP注释SNPs的rsid",
    "Annotate somatic SNPs from COSMIC (human only)": "注释来自COSMIC的体细胞SNP（仅限人类）",
    "Exon Annotations": "外显子注释",
    "Protein Sequences": "蛋白质序列",
    "Protein Coding Sequences": "蛋白质编码序列",
    "Protein IDs": "蛋白质ID",
    "A dataframe containing dbSNP rsids": "一个包含dbSNP rsid的DataFrame",
    "A dataframe containing somatic SNPs from COSMIC (human only)": "一个包含来自COSMIC（仅限人类）的体细胞SNPs的数据框",
    "Transcript Expression Cutoff (RPKM)": "转录表达截止值（RPKM）",
    "Create a variant FASTA for short insertions and deletions": "创建一个用于短插入和删除的变体FASTA文件",
    "Create SQLite files for mapping proteins to genome and summarizing variant proteins": "创建用于将蛋白质映射到基因组并总结变异蛋白质的 SQLite 文件",
    "Create RData file of variant protein coding sequences": "创建变异蛋白质编码序列的RData文件",
    "Input Protein Abundance File": "输入蛋白质丰度文件",
    "Input RNA Abundance File": "输入 RNA 丰度文件",
    "Select data input type": "选择数据输入类型",
    "Data summarization method": "数据汇总方法",
    "Volcano plot with p-value or adjusted p-value": "火山图，使用 p 值或校正后的 p 值",
    "K-mean clustering: Number of clusters": "K均值聚类：聚类数量",
    "Hierarchical clustering: Number of clusters (from Heatmap)": "层次聚类：聚类数量（来自热图）",
    "Transformation method": "转换方法",
    "Polynomial order": "多项式阶数",
    "Weighted average": "加权平均",
    "Half window size": "半窗口大小",
    "Baseline removal method": "基线去除方法",
    "Intensity calibration method": "强度校准方法",
    "m/z range": "m/z 范围",
    "Start of m/z range, has to be inside m/z range": "m/z 范围的起始点，必须在 m/z 范围内",
    "End of m/z range, has to be inside m/z range": "m/z 范围的结束，必须在 m/z 范围之内",
    "Allow no matches": "允许无匹配",
    "Empty no matches": "空的没有匹配",
    "Remove empty spectra": "移除空的谱图",
    "Export processed imzML": "导出处理过的imzML文件",
    "Choose the source of the GTF file": "选择GTF文件的来源",
    "GTF file with the genome of interest": "感兴趣的基因组的GTF文件",
    "BED file with chromosomal coordinates of peptide": "带有肽段染色体坐标的BED文件",
    "Contig file": "Contig文件",
    "K-mer length": "K-mer长度",
    "Maximal amount of fractions to batch.": "最大批次处理的分数数量。",
    "Which datatype to nest?": "哪种数据类型需要嵌套？",
    "Batch pools": "批量池",
    "Filename part identifying batchpool": "文件名部分识别批次池",
    "List of files to batch": "文件批量列表",
    "Percolator input data": "Percolator 输入数据",
    "Target decoy method": "目标-伪目标方法",
    "Penalty for mistakes on positive examples": "正例错误惩罚",
    "Penalty for mistakes on negative examples": "负样本上的错误惩罚",
    "FDR threshold to define positive examples": "FDR阈值用于定义正例",
    "FDR threshold for evaluating best cross validation result": "FDR阈值用于评估最佳交叉验证结果",
    "Most informative feature given as feature name.": "最具信息量的特征以特征名称给出。",
    "Unit normalization instead of standard deviation": "单位标准化而非标准差",
    "Override error check?": "覆盖错误检查？",
    "Seed of random number generator": "随机数生成器的种子",
    "Skip all outputs except PSMs": "跳过所有输出，除了PSMs",
    "Search engine used": "搜索引擎使用",
    "Target MSGF+ results": "目标 MSGF+ 结果",
    "Decoy MSGF+ results": "诱饵 MSGF+ 结果",
    "Enzyme used": "使用的酶",
    "Max number of PSMs for a spectrum": "每个谱图的最大PSM数",
    "Calculate amino acid frequency features": "计算氨基酸频率特征",
    "Calculate feature for number of PTMs": "计算蛋白质翻译后修饰（PTMs）的数量特征",
    "Calculate feature based on N-linked glycosylation resulting from PNGaseF": "根据PNGaseF产生的N-糖基化计算特征",
    "Calculate mass difference to closest isotope instead of avg. mass": "计算与最接近的同位素的质量差异，而不是平均质量。",
    "Percolator output": "Percolator输出",
    "Multiple mzIdentML files?": "多个mzIdentML文件？",
    "mzIdentML from MSGF+": "mzIdentML 来自 MSGF+",
    "Which features to add to mzIdentML": "mzIdentML 是一种用于存储蛋白质组学质谱数据的文件格式，它包含了质谱实验中鉴定的蛋白质和肽段的信息。为了满足不断发展的科研需求，可以向 mzIdentML 格式添加以下特性：  1. **支持更多类型的质谱数据**：随着质谱技术的发展，新的数据类型不断出现，mzIdentML 可以扩展以支持这些新类型的数据。  2. **更详细的搜索参数**：记录更详细的质谱搜索参数，如酶切规则、修饰搜索等。  3. **数据质量指标**：包括更多的数据质量指标，如肽段和蛋白质鉴定的置信度评分。  4. **修饰信息**：更详细地记录肽段的修饰信息，包括修饰的位置和类型。  5. **支持定量数据**：扩展格式以支持蛋白质和肽段的定量信息。  6. **谱图信息**：包含原始谱图的元数据，如谱图的采集参数和处理信息。  7. **样本信息**：更全面地记录样本的描述信息，包括生物样本的详细信息。  8. **分析流程记录**：记录从原始数据到最终结果的整个分析流程，包括使用的工具和版本的信息。  9. **跨样本分析**：支持跨多个样本的比较分析，如蛋白质组学数据的批量处理和比较。  10. **数据共享和交换**：改进数据的可交换性和共享性，以便于科研合作和数据再利用。  11. **用户定义字段**：允许用户添加自定义的元数据字段，以适应特定的研究需求。  12. **数据安全和隐私**：考虑数据安全和隐私保护，如加密和访问控制。  13. **与其它标准兼容**：确保 mzIdentML 与其它相关标准（如 MIAPE、ISA-Tab）的兼容性。  14. **更好的软件支持**：提高现有软件对新特性的支持，以及开发新的工具来利用这些特性。  15. **社区反馈机制**：建立一个社区反馈机制，让研究人员可以提出改进建议和新的需求。  这些特性的添加需要社区的共同努力，包括标准制定者、软件开发者和最终用户的参与。通过不断的迭代和改进，mzIdentML 可以更好地服务于蛋白质组学研究。",
    "Skip percolator schema validation?": "跳过percolator模式验证？",
    "Peptide summary output from moFF or other peptide quantitation tools": "来自moFF或其他肽段定量工具的肽段汇总输出",
    "Metadata with sample names and condition": "带有样本名称和条件的元数据",
    "Only summarisiation": "仅摘要",
    "Type of MSGF+ analysis": "MSGF+ 分析类型",
    "Collection: Pairs of spectra (forward) and FASTA database (reverse)": "集合：正向的谱对（forward）和反向的FASTA数据库（reverse）",
    "Output TSV as well?": "输出 TSV 文件吗？",
    "Search with on-the-fly decoy database?": "使用即时诱饵数据库进行搜索？",
    "Precursor mass tolerance units": "前体质量容差单位",
    "Instrument type": "仪器类型",
    "Number of tolerable termini": "可容忍的末端数量",
    "Common Fixed Modifications": "常见固定修饰",
    "Common Variable Modifications": "常见变量修饰",
    "Custom Modifications": "自定义修改",
    "Formula or Mass": "公式或质量",
    "Amino Acid Specificity": "氨基酸特异性",
    "Variable or Fixed?": "变量还是固定？",
    "Positional Specificity": "位置特异性",
    "Minimum peptide length": "最小肽段长度",
    "Maximum peptide length": "最大肽段长度",
    "Maximum modifications allowed per peptide": "每个肽段允许的最大修饰数",
    "Fragmentation type": "片段化类型",
    "Protocol type": "协议类型",
    "Maximum matches per spectrum": "每个谱图的最大匹配数",
    "Calculate additional scoring features?": "计算额外的评分特征？",
    "Lower isotope error range": "较低的同位素误差范围",
    "Upper isotope error range": "上同位素误差范围",
    "Negative to positive instance ratio": "负正实例比率",
    "Order of k-mer shuffling": "k-mer 洗牌顺序",
    "Peptides with precursor information": "具有前体信息的肽段",
    "Peptides as text-based file": "肽段作为基于文本的文件",
    "input file with additional model parameters when using the OLIGO kernel": "在使用OLIGO核时，输入包含额外模型参数的文件",
    "input file with the used training dataset when using the OLIGO kernel": "在使用OLIGO核函数时，输入使用的培训数据集文件",
    "svm model in libsvm format (can be produced by RTModel)": "支持向量机模型在libsvm格式（可以由RTModel生成）",
    "The time (in seconds) of the gradient (peptide RT prediction)": "梯度（肽段保留时间预测）的时间（单位：秒）",
    "The maximum number of peptides considered at once (bigger number will lead to faster results but needs more memory)": "同时考虑的最大多肽数量（更大的数字将导致更快的结果，但需要更多的内存）",
    "Rewrites each peptideidentification's rt and mz from prediction and calculation (according to the best hit)": "将每个肽段鉴定的保留时间和质荷比（根据最佳命中）从预测和计算中重写",
    "svm model in libsvm format (can be produced by PTModel)": "libsvm格式的支持向量机模型（可由PTModel生成）",
    "the maximum number of peptides considered at once (bigger number will lead to faster results but needs more memory)": "一次考虑的最多肽段数（更大的数字将导致更快的结果，但需要更多的内存）",
    "input file in mzXML or mzData format": "输入文件为mzXML或mzData格式",
    "if this flag is set the InspectAdapter will read in mzXML, <br>write an Inspect input file and generate a trie database": "如果设置了这个标志，InspectAdapter 将读取 mzXML 文件，<br>写入 Inspect 输入文件并生成一个 trie 数据库",
    "if this flag is set the InspectAdapter will read in a Inspect results file <br>and write idXML": "如果设置了这个标志，InspectAdapter 将读取一个 Inspect 结果文件，并写入 idXML",
    "the directory in which Inspect is located": "Inspect 所在的目录",
    "a directory in which some temporary files can be stored": "一个可以存储一些临时文件的目录",
    "name(s) of database(s) to search in (FASTA and SwissProt supported)": "要搜索的数据库名称（支持FASTA和SwissProt）",
    "name(s) of databases(s) to search in (trie-format)": "要在trie格式中搜索的数据库名称（或多个数据库名称）",
    "name of the merged trie database": "合并后的trie数据库名称",
    "the instrument that was used to measure the spectra <br>(If set to QTOF, uses a QTOF-derived fragmentation model, <br>and does not attempt to correct the parent mass.)": "用于测量光谱的仪器 <br>（如果设置为 QTOF，则使用 QTOF 衍生的碎裂模型，<br>并且不尝试校正母体质量。）",
    "show a list of the available modifications": "显示可用修饰列表",
    "the colon-separated modifications; may be <br><name>,<type>,": "冒号分隔的修饰；可以是<名称>，<类型>，",
    "use monoisotopic masses for the modifications": "使用单同位素质量来修改",
    "name of an XML file with the modifications": "带有修改的 XML 文件的名称",
    "the enzyme used for digestion": "用于消化的酶",
    "name for the input file of Inspect (may only be used in a full run)": "Inspect输入文件的名称（仅在完整运行中使用）",
    "attempt to guess the precursor charge and mass, <br>and consider multiple charge states if feasible": "尝试猜测前体电荷和质量，如果可行，考虑多重电荷状态",
    "number of PTMs permitted in a single peptide": "单个肽段中允许的PTMs数量",
    "number of tags to generate": "生成标签的数量",
    "no temporary databases are used": "不使用临时数据库",
    "annotations with inferior p-value are ignored": "具有较低p值的注释被忽略",
    "Name of the contact": "联系人姓名",
    "Name of the contact institution": "联系机构的名称",
    "Some information about the contact": "关于联系人的一些信息",
    "perform a blind search (allowing arbitrary modification masses), <br>is preceeded by a normal search to gain a smaller database": "执行一个盲搜（允许任意修改质量），在进行一个常规搜索以获得更小的数据库之前。",
    "like blind but no prior search is performed to reduce the database size": "像BLAT一样，但是不执行预先搜索以减少数据库大小",
    "used for generating the minimized database": "用于生成最小化数据库",
    "name of the minimized trie database generated when using blind mode": "使用盲模式生成的最小化trie数据库的名称",
    "maximum modification size (in Da) to conside": "最大修饰尺寸（以道尔顿为单位）考虑",
    "Inclusion list input file in FASTA or featureXML format": "包含列表输入文件，格式为FASTA或featureXML",
    "Exclusion list input file in featureXML, idXML or FASTA format": "排除列表输入文件，支持featureXML、idXML或FASTA格式。",
    "RTModel file used for the rt prediction of peptides in FASTA files": "用于在FASTA文件中预测肽段保留时间的RT模型文件",
    "PTModel file used for the pt prediction of peptides in FASTA files (only needed for inclusion_strategy PreotinBased_LP)": "用于FASTA文件中肽段的pt预测的PTModel文件（仅当包含策略为PreotinBased_LP时需要）",
    "param_inclusion_charges": "参数_包含电荷",
    "List containing the charge states to be considered for the inclusion list compounds, space separated": "包含要考虑的包含列表化合物的电荷状态的列表，用空格分隔",
    "strategy to be used for selection": "用于选择的策略",
    "param_exclusion_charges": "参数_排除电荷",
    "List containing the charge states to be considered for the exclusion list compounds (for idXML and FASTA input), space separated": "包含要为排除列表化合物考虑的电荷状态的列表（用于idXML和FASTA输入），用空格分隔",
    "File containing the raw data (only needed for FeatureBased_LP)": "包含原始数据的文件（仅FeatureBased_LP需要）",
    "Number of missed cleavages used for protein digestion": "用于蛋白质消化的未切割数",
    "Create lists with units as seconds instead of minutes": "创建以秒为单位而不是分钟的列表",
    "Use relative RT window, which depends on RT of precurso": "使用相对的保留时间窗口，该窗口取决于前体的保留时间",
    "[for RT:use_relative == true] The relative factor X for the RT exclusion window,": "[对于 RT:use_relative == true] RT排除窗口的相对因子X，",
    "[for RT:use_relative == false] The absolute value X for the RT exclusion window in [sec],": "[对于 RT:use_relative == false] RT排除窗口的绝对值X，单位为[秒]，",
    "Two inclusion/exclusion windows are merged when they (almost) overlap in RT (see 'rt_tol') and are close in m/z by this tolerance": "当两个包含/排除窗口在保留时间（RT）上（几乎）重叠（参见'rt_tol'）并且通过此容差在m/z上接近时，它们会被合并。",
    "Unit of 'mz_tol'": "'mz_tol'的单位",
    "Maximal RT delta (in seconds) which would allow two windows in RT to overlap (which causes merging the windows)": "最大保留时间（RT）差异（以秒为单位），允许两个在保留时间上的窗口重叠（这会导致合并窗口）",
    "param_algorithm_ms_levels": "参数_算法_ms级别",
    "Unit of FWHM. Either absolute in the unit of input,": "FWHM的单位。可以是输入单位的绝对值，",
    "Identification input of combined forward decoy search (reindex with PeptideIndexer first)": "识别组合正向诱饵搜索的输入（首先使用PeptideIndexer重新索引）",
    "Identification input of forward run": "正向运行的识别输入",
    "Identification input of decoy run": "识别诱饵运行的输入",
    "Number of bins used for the fitting, if sparse datasets are used, this number should be smalle": "用于拟合的箱数，如果使用稀疏数据集，则此数字应更小",
    "This value is used if": "这个值被使用，如果",
    "param_target_nucleotides": "目标核苷酸参数",
    "param_mapping": "参数映射",
    "param_restrictions": "参数限制",
    "format: target nucleotide=min_count: e.g U=1 if at least one U must be in the generated sequence": "格式：目标核苷酸=最小计数：例如 U=1 如果生成的序列中至少需要有一个U",
    "param_modifications": "参数修改",
    "Lower peptide mass (Da) threshold": "降低肽质量（道尔顿）阈值",
    "Lower m/z (Th) threshold for precursor variant": "较低的m/z（Th）前体变体阈值",
    "Ini file for the OMSSA search engine": "OMSSA 搜索引擎的初始化文件",
    "Fasta file for search result annotation": "用于搜索结果注释的Fasta文件",
    "mz tolerance used to determine marker ions": "用于确定标记离子的质量偏差",
    "Do not recreate intermediate files to continue after unexpected crash": "不要重新创建中间文件以在意外崩溃后继续",
    "Tolerance in m/z dimension": "m/z 维度的容忍度",
    "Tolerance in RT dimension": "RT 维度中的容忍度",
    "param_concentrations": "参数浓度",
    "List of spiked concentrations": "尖峰浓度列表",
    "RT position of the feature": "特征的RT位置",
    "m/z position of the feature": "特征的 m/z 位置",
    "RT position of the standard": "标准位置的RT",
    "m/z position of the standard": "标准品的 m/z 位置",
    "Flag that activates the GNUplot output": "激活 GNUplot 输出的标志",
    "base file name (3 files with different extensions are created)": "基础文件名（创建3个具有不同扩展名的文件）",
    "Path to the percolator binary": "percolator二进制文件的路径",
    "Input target file": "输入目标文件",
    "Input decoy file": "输入伪序列文件",
    "Type of enzyme: no_enzyme,elastase,pepsin,proteinasek,thermolysin,chymotrypsin,lys-n,lys-c,arg-c,asp-n,glu-c,trypsin": "酶的类型：无酶，弹性蛋白酶，胃蛋白酶，蛋白酶K，碱性蛋白酶，胰蛋白酶，赖氨酸C，赖氨酸C，精氨酸C，天冬氨酸N，谷氨酸C，胰蛋白酶",
    "read xml-input format (pin) from standard input": "从标准输入读取 XML 输入格式（pin）",
    "Include decoys (PSMs, peptides and/or proteins) in the xml-output": "在xml输出中包含诱饵（PSMs、肽段和/或蛋白质）",
    "Fraction of the negative data set to be used as train set when only providing one negative set, remaining examples will be used as test set": "当仅提供一个负数据集时，用作训练集的负数据集的比例，剩余的示例将用作测试集。",
    "Input file given in the deprecated pin-xml format generated by": "输入文件使用了已弃用的 pin-xml 格式，该格式由",
    "Set verbosity of output: 0=no processing info, 5=all, default is 2": "设置输出的详细程度：0=无处理信息，5=全部信息，默认为2",
    "Do not remove redundant peptides, keep all PSMS and exclude peptide level probabilities": "不要移除冗余的肽段，保留所有的蛋白质谱匹配（PSMS），并排除肽段水平的概率。",
    "skip validation of input file against xml schema": "跳过对输入文件与XML模式的验证",
    "output protein level probabilities": "输出蛋白质水平概率",
    "Probability with which a present protein emits an associated peptide (to be used jointly with the -A option)": "当前蛋白质产生相关肽段的概率（与-A选项一起使用）",
    "Probability of the creation of a peptide from noise (to be used jointly with the -A option)": "从噪声中创建肽的概率（与-A选项一起使用）",
    "Prior probability of that a protein is present in the sample ( to be used with the -A option)": "样本中蛋白质存在先验概率（与-A选项一起使用）",
    "treat ties as if it were one protein (Only valid if option -A is active)": "将平局视为一个蛋白质（仅当激活选项-A时有效）",
    "use pi_0 value when calculating empirical q-values (no effect if option Q is activated) (Only valid if option -A is active)": "在使用pi_0值计算经验q值时（如果激活了Q选项，则此选项无效）（仅当激活了-A选项时有效）",
    "output empirical q-values and p-values (from target-decoy analysis) (Only valid if option -A is active)": "输出经验q值和p值（来自目标-诱饵分析）（仅当激活选项-A时有效）",
    "disactivates the grouping of proteins with similar connectivity, for example if proteins P1 and P2 have the same peptides matching both of them, P1 and P2 will not be grouped as one protein (Only valid if option -A is active)": "取消具有相似连接性的蛋白质分组，例如如果蛋白质P1和P2具有相同的肽段匹配，则P1和P2将不会被分组为一个蛋白质（仅当激活选项-A时有效）",
    "Proteins graph will not be separated in sub-graphs (Only valid if option -A is active)": "蛋白质图不会被分割成子图（仅当激活选项 -A 时有效）",
    "it does not prune peptides with a very low score (~0.0) which means that if a peptide with a very low score is matching two proteins, when we prune the peptide,it will be duplicated to generate two new protein groups (Only valid if option -A is active)": "它不会修剪得分非常低的肽段（~0.0），这意味着如果一个得分非常低的肽段与两个蛋白质相匹配，当我们修剪该肽段时，它将被复制以生成两个新的蛋白质组（仅当激活选项-A时有效）。",
    "Setting depth 0 or 1 or 2 from low depth to high depth(less computational time) of the grid search for the estimation Alpha,Beta and Gamma parameters for fido(Only valid if option -A is active)": "设置网格搜索估计 Alpha、Beta 和 Gamma 参数的深度为 0、1 或 2（从低深度到高深度，计算时间较少）（仅当激活选项 -A 时有效）",
    "Define the text pattern to identify the decoy proteins and/or PSMs, set this up if the label that identifies the decoys in the database is not the default (by default : random) (Only valid if option -A  is active)": "定义用于识别诱饵蛋白质和/或PSMs（肽段鉴定）的文本模式，如果用于在数据库中识别诱饵的标签不是默认值（默认为：random）（仅当激活选项-A时有效）",
    "Reduce the tree of proteins (removing low scored proteins) in order to estimate alpha,beta and gamma faster.(Only valid if option -A is active)": "为了更快地估计α、β和γ值，减少蛋白质的树（移除得分低的蛋白质）。（仅当激活选项-A时有效）",
    "Use target decoy competition to compute peptide probabilities.(recommended when using -A)": "使用目标-诱饵竞争来计算肽概率。（推荐在使用-A时使用）",
    "Q-value threshold that will be used in the computation of the MSE and ROC AUC score in the grid search (recommended 0.05 for normal size datasets and 0.1 for big size datasets).(Only valid if option -A is active)": "在网格搜索中用于计算MSE（均方误差）和ROC AUC（受试者工作特征曲线下面积）得分的Q值阈值（对于一般大小的数据集推荐使用0.05，对于大型数据集推荐使用0.1）。（仅当激活选项-A时有效）",
    "Proteins with a very low score (< 0.001) will be truncated (assigned 0.0 probability).(Only valid if option -A is active)": "得分非常低的蛋白质（< 0.001）将被截断（赋予 0.0 概率）。（仅当激活选项 -A 时有效）",
    "Uses protein group level inference, each cluster of proteins is either present or not, therefore when grouping proteins discard all possible combinations for each group.(Only valid if option -A is active and -N is inactive)": "使用蛋白质组水平推断，每个蛋白质簇要么存在要么不存在，因此在对蛋白质进行分组时，丢弃每个组的所有可能组合。（仅当激活选项-A且-N未激活时有效）",
    "FASTA Input": "FASTA 输入",
    "Include original entries in output database": "在输出数据库中包含原始条目",
    "Specify advanced decoy frequency options?": "指定高级诱饵频率选项？",
    "Decoy Frequency": "诱饵频率",
    "Additional Decoy Tags": "额外的诱饵标签",
    "Indexed mzML": "索引化 mzML",
    "MS Protein Search Database: UniProt Xml or Fasta": "MS蛋白质搜索数据库：UniProt Xml或Fasta",
    "FDR (Maximum False Discovery Rate percent)": "FDR（最大假阳性发现率百分比）",
    "Maximum Variable Modification Isoforms Per Peptide": "每个肽段的最大可变修饰异构体数",
    "Precursor Mass Tolerance Value": "前体质量容差值",
    "Precursor Mass Tolerance Units": "前体质量容差单位",
    "Set advanced options?": "设置高级选项？",
    "Use G-PTM with Uniprot Proteome Search Databases": "使用G-PTM与Uniprot蛋白质组搜索数据库",
    "Minimum Unknown Precursor Charge State": "最小未知前体电荷状态",
    "Maximum Unknown Precursor Charge State": "最大未知前体电荷状态",
    "Absolute MS/MS Intensity Threshold": "绝对 MS/MS 强度阈值",
    "Relative MS/MS Intensity Threshold": "相对MS/MS强度阈值",
    "Maximum Number of MS/MS Peaks": "最大MS/MS峰值数",
    "Assign Charge States": "分配电荷状态",
    "Deisotope": "去同位素",
    "Create Target-Decoy Database On The Fly": "创建目标-诱饵数据库（即时）",
    "Precursor Monoisotopic Peak Correction": "前体单一同位素峰校正",
    "Minimum Precursor Monoisotopic Peak Correction": "最小前体单同位素峰校正",
    "Maximum Precursor Monoisotopic Peak Correction": "最大前体单同位素峰校正",
    "Product Mass Type": "产品质量类型",
    "Product Mass Tolerance Value": "产品质量容忍值",
    "Product Mass Tolerance Units": "产品质量容忍单位",
    "Initiator Methionine Behavior": "起始甲硫氨酸行为",
    "Consider Modified Forms as Unique Peptides": "考虑将修饰形式视为独特的肽段",
    "Minimize Memory Usage": "最小化内存使用",
    "Peptides abundances": "肽段丰度",
    "Protein database": "蛋白质数据库",
    "Sample Groups": "样本组",
    "Reference sample groups": "参考样本组",
    "Peptides abundance scale": "肽段丰度尺度",
    "Sample-wise normalization": "样本标准化",
    "Hyperparameter mu": "超参数 mu",
    "Hyperparameter alpha": "超参数 alpha",
    "Minimum samples for peptide": "最小肽段样本数",
    "Minimum fraction of missing values in the group": "组内缺失值的最小比例",
    "Peptide cutoff weight": "肽段截断重量",
    "Use unique peptides only": "使用独特的肽段",
    "Allow early termination in EM calculation when noise is sufficiently small.": "在噪声足够小的情况下，允许在EM计算中提前终止。",
    "Perform Monte Carlo FDR simulation": "执行蒙特卡洛假发现率（FDR）模拟",
    "Output Protein Peptide loadings file": "输出蛋白质肽段载荷文件",
    "Instrument output": "仪器输出",
    "Binary data compression type": "二进制数据压缩类型",
    "Start retention time": "起始保留时间",
    "End retention time": "保留时间结束",
    "MS1 peak picking": "MS1 峰值挑选",
    "Centroidization mode": "质心化模式",
    "Number of scales": "尺度的数量",
    "Y threshold": "Y 阈值",
    "Centroid MZ tolerance": "质心 m/z 容差",
    "SNRS percentile": "SNRS百分位数",
    "Min SNR": "最小信噪比",
    "Refine MZ": "优化 m/z",
    "Sum intensities": "求和强度",
    "Specify padding parameters": "指定填充参数",
    "Speficy maximum padding points": "指定最大填充点",
    "Maximum padding points": "最大填充点",
    "Padding MZ tolerance": "填充 MZ 容差",
    "Spacing perc": "间距百分比",
    "MS2 peak picking": "MS2 峰值挑选",
    "Output parameter file": "输出参数文件",
    "Regex pattern to parse protein ID from .fasta header": "正则表达式模式，用于从.fasta文件头解析蛋白质ID",
    "Min missed cleavages": "最小未切割数",
    "Max missed cleavages": "最大未切割次数",
    "Max mass": "最大质量",
    "Min peptide length": "最小肽段长度",
    "Max peptide length": "最大肽段长度",
    "Isotopic mod": "同位素修饰",
    "Mass mode": "质量模式",
    "Fixed mods": "固定修饰",
    "Variable mods": "可变修饰",
    "Variable mod threshold": "变量修改阈值",
    "Peptide database": "肽段数据库",
    "Charge state determination parameters": "电荷状态确定参数",
    "Expected minimal charge": "预期最小电荷",
    "Expected maximum charge": "预期最大电荷",
    "Min intensity": "最小强度",
    "Delta min intensity": "最小强度差",
    "Number of random spectra": "随机谱图数量",
    "Lookup PPM": "查找 PPM",
    "N-terminal series": "N-末端序列",
    "C-terminal Series": "C端序列",
    "Andromeda": "安德洛墨达",
    "P min": "P 最小",
    "P max": "P 最大",
    "Matching ion tolerance PPM": "匹配离子容差 PPM",
    "Processing type": "处理类型",
    "Q-Value threshold": "Q值阈值",
    "Pep-Value threshold": "P值阈值",
    "Minimum increase between iterations": "最小迭代增量",
    "Sequest like": "类似于Sequest",
    "Parse protein ID regex pattern": "解析蛋白质ID正则表达式模式",
    "PSM file input mode": "PSM文件输入模式",
    "Scored PSM file": "得分的PSM文件",
    "Scored PSM files": "得分的PSM文件",
    "Groupe files": "分组文件",
    "Protein identifier regex pattern": "蛋白质标识符正则表达式模式",
    "Peptide": "肽段",
    "Q-Value method": "Q值方法",
    "Logistic regression type": "逻辑回归类型",
    "Select input FASTA dataset": "选择输入的FASTA数据集",
    "Fail job if invalid FASTA headers detected?": "如果检测到无效的FASTA头，则失败作业？",
    "Ensure that sequence is a valid amino acid sequence?": "确保序列是有效的氨基酸序列。",
    "Optional: add one letter codes for any non-standard amino acids that you are using.": "可选：为所使用的任何非标准氨基酸添加一个字母的代码。",
    "Only pass sequences with accession numbers?": "仅传递具有访问号码的序列？",
    "Minimum length for sequences to pass": "序列通过的最小长度",
    "Perform labeled quantification": "执行标记定量分析",
    "XIC extraction": "XIC提取",
    "Scan timewindow": "扫描时间窗口",
    "MzWindow_Da type": "MzWindow_Da 类型",
    "Window width": "窗口宽度",
    "Xic processing": "Xic处理",
    "Polynom order": "多项式阶数",
    "Window size type": "窗口大小类型",
    "Border padding amount": "边框填充量",
    "Border padding method": "边界填充方法",
    "Internal padding method": "内部填充方法",
    "Huge gap padding method": "巨大间隙填充方法",
    "Huge gap padding distance": "巨大的间隔填充距离",
    "Min peak distance": "最小峰值距离",
    "Min peak length": "最小峰值长度",
    "Max peak length": "最大峰值长度",
    "Noise quantile": "噪声分位数",
    "Top KPSMs": "顶级KPSMs",
    "Use baseline correction": "使用基线校正",
    "Lambda": "Lambda",
    "P": "P",
    "Quantified Peptides": "定量肽段",
    "Inferred Proteins": "推断蛋白质",
    "Protein assigned quant pep ions": "蛋白质定量肽离子分配",
    "Correlation Light Heavy Threshold": "相关性 轻 重 阈值",
    "Modification Filter": "修改过滤器",
    "Aggregate Global Modifications Parameters": "全局修饰聚合参数",
    "Aggregate Peptide Charge States Parameters": "聚合肽段电荷状态参数",
    "Specify Aggregate Peptide Charge States Parameters?": "指定聚合肽段的电荷状态参数？",
    "Aggregate Modified Peptides Parameters": "聚合修饰肽段参数",
    "Specify Aggregate Modified Peptides Parameters?": "指定聚合修改肽段参数？",
    "Aggregate To Protein Groups Parameters": "聚合至蛋白质组参数",
    "Output files containing all peakgroups from OpenSWATH (using either mProphet or pyProphet) or Peakview.": "输出包含所有OpenSWATH峰值组的文件（使用mProphet或pyProphet或Peakview）。",
    "Which input file format is used": "哪种输入文件格式被使用",
    "Which method to use for the clustering": "使用哪种方法进行聚类",
    "Which columns are written besides Intensity": "除了强度（Intensity）之外，还有哪些列被写入了",
    "How to re-align runs in retention time": "如何根据保留时间重新对齐运行",
    "Enable difference in RT for two aligned features": "启用两个对齐特征的RT差异",
    "Difference in RT (in seconds) for two aligned features": "两个对齐特征的RT（保留时间）差异（以秒为单位）",
    "Enable difference in RT for two isotopic channels": "启用两个同位素通道的保留时间差异",
    "Difference in RT (in seconds) for two isotopic channels.": "两个同位素通道的保留时间差（以秒为单位）。",
    "Enable fraction of runs": "启用运行的分数",
    "Fraction of runs": "运行次数的比例",
    "Choose FDR method": "选择FDR方法",
    "Target FDR": "目标假发现率",
    "Seeding score cutoff": "种子评分截断",
    "Extension score cutoff": "扩展分数截断值",
    "Further parameters": "其他参数",
    "Disable grouping of isotopic variants": "禁用同位素变体的分组",
    "Enable the filter by d score (this is mainly for speedup)": "启用按d分数过滤（这主要是为了加速）",
    "d score cut-off": "d分数截断",
    "Number of experiments with high confidence": "高置信度的实验数量",
    "Read full or minimal transition groups": "读取完整或最小的转换组",
    "Alignment score": "对齐分数",
    "Dataset groups": "数据集组",
    "r": "r",
    "weights method": "权重方法",
    "distance metric": "距离度量",
    "annealing": "退火",
    "init": "初始化",
    "p0": "p0",
    "iter.max": "迭代次数最大值",
    "Generate estimates results": "生成估计结果",
    "Generate probability and class results": "生成概率和类别结果",
    "Generate plots results": "生成图表结果",
    "Reference type": "参考类型",
    "m/z feature": "m/z 特征",
    "Reference name": "参考名称",
    "sort.by": "按顺序排序",
    "MGF Input Type": "MGF 输入类型",
    "MGF File": "MGF 文件",
    "MGF Collection": "MGF 集合",
    "Processing Control": "处理控制",
    "Size to split fasta for parallel processing": "用于并行处理的fasta文件分割大小",
    "MetaNovo Parameters": "MetaNovo 参数",
    "Enzyme Specificity": "酶特异性",
    "Enzyme Rule": "酶规则",
    "Number of enzymatic missed cleavages": "酶切未识别次数",
    "Spectrum Matching Parameters": "谱图匹配参数",
    "Precursor ion mass tolerance": "前体离子质量容差",
    "Precursor ion tolerance unit": "前体离子容忍单位",
    "Fragment ion mass tolerance": "片段离子质量容差",
    "Fragment ion tolerance unit": "片段离子容忍单位",
    "Fixed modifications as comma separated list": "固定修饰作为逗号分隔列表",
    "Variable modifications as comma separated list": "变量修饰作为逗号分隔列表",
    "Minimal charge to search for": "最小电荷以搜索",
    "Maximal charge to search for": "最大电荷量搜索",
    "Type of forward ion searched": "搜索的前向离子类型",
    "Type of rewind ion searched": "搜索的回卷离子类型",
    "Spectrum Annotation": "谱图注释",
    "The intensity threshold to consider for annotation": "用于注释的强度阈值",
    "If true the most accurate peak will be selected within the m/z tolerance.": "如果为真，则会在m/z容差内选择最准确的峰。",
    "Sequence Matching": "序列匹配",
    "sequence_index_type (deprecated)": "序列索引类型（已弃用）",
    "The maximal share of Xs in a sequence, 0.25 means 25% of X's": "序列中X的最大比例，0.25表示X的25%",
    "Import Filters": "导入过滤器",
    "The minimal peptide length to consider when importing identification files": "导入识别文件时要考虑的最小肽段长度",
    "The maximal peptide length to consider when importing identification files": "导入鉴定文件时考虑的最大肽段长度",
    "Maximal precursor ion deviation unit": "最大前体离子偏差单位",
    "Peptides presenting unrecognized PTMs will be excluded": "呈现未识别的PTMs的肽段将被排除",
    "PTM Localization": "PTM 定位",
    "The PTM probabilistic score to use for modification localization": "PTM概率分数，用于修饰定位",
    "Include neutral losses in spectrum annotation of the PTM score": "在PTM分数的谱图注释中包括中性损失",
    "The modification to peptide sequence matching type": "肽段序列匹配类型的修改",
    "Use and save gene mappings along with the project": "使用并保存基因映射，与项目一起保存",
    "Update gene mappings automatically from Ensembl": "自动从Ensembl更新基因映射",
    "Simplify protein groups based on the PeptideShaker target/decoy score": "根据PeptideShaker目标/迷惑分数简化蛋白质组",
    "Simplify protein groups based on the peptide enzymaticity": "简化基于肽酶解性的蛋白质组",
    "Simplify protein groups based on the UniProt protein evidence": "根据UniProt蛋白质证据简化蛋白质组",
    "Simplify protein groups based on the protein characterization": "简化基于蛋白质特性的蛋白质组",
    "FDR at the PSM level in percent": "FDR在PSM水平上的百分比",
    "FDR at the peptide level in percent": "肽水平上的FDR百分比",
    "FDR at the protein level in percent": "蛋白质水平上的FDR百分比",
    "Group PSMs by charge for scoring and validation": "将PSMs按电荷分组进行评分和验证",
    "Group peptides by modification status for scoring and validation": "将肽段根据修饰状态分组以进行评分和验证",
    "Merge small PSM and peptide groups for scoring and validation": "合并小的PSM和肽段组以进行评分和验证",
    "Minimum confidence required for a protein in the fraction MW plot": "在分子量（MW）分布图上，蛋白质所需的最小置信度",
    "DirecTag": "直接标签法",
    "TIC cutoff in percent": "TIC 截止百分比",
    "Max peak count": "最大峰值计数",
    "Number of intensity classes": "强度等级的数量",
    "Adjust precursor": "调整前体",
    "Minimum precursor adjustment": "最小前体调整",
    "Maximum precursor adjustment": "最大前体调整",
    "Precursor adjustment step": "前体调整步骤",
    "Number of charge states considered": "考虑的电荷状态数量",
    "Use charge state from M spectrum": "使用 M 谱图中的电荷状态",
    "Duplicate spectra per charge": "重复的电荷谱图",
    "Deisotoping mode": "去同位素模式",
    "Isotope mz tolerance": "同位素 m/z 容差",
    "Complement mz tolerance": "互补质量误差容忍度",
    "Tag length": "标签长度",
    "Maximum variable modifications per sequence": "每序列最大变量修饰",
    "Maximum tag count": "最大标签计数",
    "Intensity score weight": "强度得分权重",
    "Fidelity score weight": "保真度分数权重",
    "Complement score weight": "互补得分权重",
    "Choose input configuration options": "选择输入配置选项",
    "Sequence similarity search results in tabular format (-s)": "序列相似性搜索结果以表格格式展示（-s）",
    "Sequence similarity table": "序列相似性表",
    "Sequence similarity search result in tabular format (-s)": "序列相似性搜索结果以表格格式展示（-s）",
    "Field separator (-p)": "字段分隔符（-p）",
    "Header of the sequence similarity tables (-e)": "序列相似性表格的标题（-e）",
    "Blacklist Regexs (-b)": "黑名单正则表达式（-b）",
    "Capture replace pairs (-c)": "捕获替换对（-c）",
    "Filter regexs (-l)": "过滤正则表达式（-l）",
    "Non informative words regexs (-w)": "非信息性词汇正则表达式（-w）",
    "Description split regex (-r)": "描述分割正则表达式（-r）",
    "Center inverse word-information-content at quantile (-q)": "将反词信息内容的中心放在分位数(-q)",
    "Polishing capture replace pairs (-d)": "打磨捕获替换对（-d）",
    "Sequence family annotation": "序列家族注释",
    "Families of biological sequences (-f)": "生物序列家族（-f）",
    "Annotate non family query sequences (-a)": "注释非家族查询序列（-a）",
    "Sequence family file gene-id separator (-g)": "序列家族文件基因ID分隔符（-g）",
    "Sequence family file family - gene-id separator (-i)": "序列家族文件家族 - 基因ID分隔符（-i）",
    "Exclude not annotated query sequences (-x)": "排除未注释的查询序列（-x）",
    "Sequence pattern": "序列模式",
    "5' Restriction sites": "5' 限制性位点",
    "3' Restriction sites": "3' 限制性位点",
    "Phospho (STY)Sites.txt": "磷酸化位点（STY）列表.txt",
    "pattern for column 'Number of Phospho (STY)'": "列“磷酸化位点数（STY）”的模式",
    "pattern for first column of intensity values": "强度值第一列的模式",
    "Interval between the intensity column of samples": "样本强度列之间的间隔",
    "Phosphopeptide enrichment type": "磷酸肽富集类型",
    "Intensity merge function": "强度合并功能",
    "Localization Probability Cutoff": "定位概率截止值",
    "UniProtKB/SwissProt FASTA database": "UniProtKB/SwissProt FASTA数据库",
    "NetworKIN file": "NetworKIN 文件",
    "pSTY_Motifs file": "pSTY_Motifs 文件",
    "PSP_Kinase_Substrate_Dataset": "PSP_激酶_底物_数据集",
    "PSP_Regulatory_sites": "PSP_调控位点",
    "filter to limit PhosphoSitePlus records to indicated species": "过滤以限制PhosphoSitePlus记录到指定的物种",
    "Filtered phosphopeptide intensities (tabular)": "过滤后的磷酸肽强度（表格）",
    "ANOVA alpha cutoff level (tabular)": "方差分析（ANOVA）显著性水平阈值（表格形式）",
    "Database from mqppep_preproc (sqlite)": "来自 mqppep_preproc（sqlite）的数据库",
    "Intensity-column pattern": "强度列模式",
    "Imputation method": "填充方法",
    "Mean percentile for random values": "随机值的平均百分位数",
    "Percentile SD for random values": "随机值的百分位数标准差",
    "Sample-name extraction pattern": "样本名称提取模式",
    "Sample-group extraction pattern": "样本组提取模式",
    "Minimum number of values per sample-group": "每个样本组的最小值数量",
    "Filter sample-groups": "过滤样本组",
    "Minimum number of kinase-substrates for KSEA": "KSEA所需的最小激酶-底物数量",
    "KSEA threshold level": "KSEA 阈值水平",
    "Use abs(log2(fold-change)) for KSEA": "使用 abs(log2(fold-change)) 进行 KSEA 分析",
    "Minimum quality of substrates for KSEA": "KSEA所需的底物的最小质量",
    "Input QuPath annotation": "输入 QuPath 注释",
    "Spectrum file": "谱图文件",
    "Peptide file": "肽段文件",
    "Mass accuracy": "质量精度",
    "Bin delimiter": "二进制分隔符",
    "Target isotope": "目标同位素",
    "Compute clumps": "计算簇",
    "Custom isotope abundance matrix": "自定义同位素丰度矩阵",
    "Run benchmarking": "运行基准测试",
    "Nominal values": "名义值",
    "Spectra file": "光谱文件",
    "Keys to add": "添加的键",
    "Attribute Name": "属性名称",
    "quality Distance Over Span": "质量 距离 超过 跨度",
    "Select multiple test correction to apply": "选择应用多个测试校正",
    "Select output figure format": "选择输出图形格式",
    "width of figures": "图形宽度",
    "heigt of each figure": "每个图形的高度",
    " figure font size": "图形字体大小",
    "Matching type": "匹配类型",
    "Metadata Match": "元数据匹配",
    "Select color for lines": "选择线条的颜色",
    "alpha of subfig plots": "子图 alpha",
    "height of subfig plots": "子图高度",
    "width of subfig plots": "子图宽度",
    "plot as grid": "以网格形式绘图",
    "xaxis_title to add to output files": "x轴标题，添加到输出文件中",
    "X axis label tilt": "X轴标签倾斜",
    "add strip plot on abundance bar": "在丰度条形图上添加条形图",
    "list of identifiers": "标识符列表",
    "Harmonize metadata": "调和元数据",
    "Element selection": "元素选择",
    "Normalize intensities": "标准化强度",
    "Apply default filters": "应用默认过滤器",
    "Clean metadata": "清理元数据",
    "Filter relative intensity": "过滤相对强度",
    "Minimum intensity": "最小强度",
    "Maximum intensity": "最大强度",
    "Filter m/z range": "过滤 m/z 范围",
    "Minimum m/z": "最小 m/z",
    "Maximum m/z": "最大 m/z",
    "Require SMILES": "需要SMILES",
    "Require INCHI": "需要INCHING",
    "Derive precursor_mz from parent_mass": "从母体质量推导出前体_mz",
    "Estimate from adduct": "从加成物估算",
    "Reduce to top n peaks": "减少到顶部 n 个峰值",
    "Maximum number of peaks": "最大峰值数",
    "Scores object": "得分对象",
    "Metadata csv file": "元数据 CSV 文件",
    "specify column/metadata key": "指定列/元数据键",
    "Data matrix file": "数据矩阵文件",
    "Data Matrix file": "数据矩阵文件",
    "Sample Metadata file": "样本元数据文件",
    "Variable Metadata file": "变量元数据文件",
    "Conditions for N-ways anova": "N向方差分析的条件",
    "Enable interaction response p-values": "启用交互作用响应 P 值",
    "Decimal separator": "小数分隔符",
    "Variable metadata file": "变量元数据文件",
    "Sample classes": "样本类别",
    "Advanced computational parameters": "高级计算参数",
    "Classification method(s)": "分类方法",
    "Selection tier(s)": "选择层级（层）",
    "MZMine batch file": "MZMine 批量文件",
    "Spectra (mzml)": "质谱图 (mzml)",
    "Spectral libraries": "光谱库",
    "Local database": "本地数据库",
    "Peaklists (HDF5 file)": "峰值列表（HDF5文件）",
    "Filelist / Samplelist": "文件列表 / 样本列表",
    "Ppm error tolerance": "PPM错误容忍度",
    "Number of technical replicates for each sample": "每个样本的技术重复次数",
    "Minimum number of technical replicates a peak has to be present in": "峰值必须在技术重复中出现的最小数量",
    "Relative standard deviation threshold": "相对标准偏差阈值",
    "Select the MS data file type?": "选择质谱数据文件类型？",
    "Do you agree to the RawFileReader license terms?": "您同意 RawFileReader 许可协议的条款吗？",
    "*.raw files": "原始数据文件",
    "*.mzML files": "*.mzML 文件",
    "Function to calculate the noise from each scan": "计算每个扫描产生的噪声的函数",
    "Signal-to-noise ratio threshold": "信噪比阈值",
    "Filter specific windows or scan events?": "过滤特定的窗口或扫描事件？",
    "Include / Exclude scan event(s)": "包含/排除扫描事件",
    "Start m/z for scan event": "扫描事件的起始 m/z",
    "End m/z for scan event": "扫描事件的结束 m/z",
    "Scan type": "扫描类型",
    "Show options for multiple scans": "显示多重扫描的选项",
    "Minimum number of scans required for each m/z window or event": "每个 m/z 窗口或事件所需的最小扫描次数",
    "Minimum fraction (i.e. percentage) of scans a peak has to be present in.": "峰值必须出现在扫描中的最小分数（即百分比）。",
    "Skip SIM-Stitching?": "跳过SIM-拼接？",
    "Remove m/z range(s)?": "移除 m/z 范围（s）？",
    "Start m/z of removal range": "移除范围的起始 m/z",
    "End m/z of removal range": "移除范围的结束 m/z",
    "Relative intensity threshold used to remove ringing artifacts": "用于去除振铃伪影的相对强度阈值",
    "MetFrag result file (Output tabular file from MetFrag tool)": "MetFrag 结果文件（MetFrag 工具的输出表格文件）",
    "MetFrag Hits Limit": "MetFrag 命中限制",
    "Synonyms": "同义词",
    "ClassyFire": "ClassyFire",
    "Peak Intensity Matrix (HDF5 file)": "峰值强度矩阵（HDF5文件）",
    "Get...": "获取...",
    "Name average Peaklist": "名称 平均峰值列表",
    "Write Peaklists to tsv files (data collection)?": "将峰值列表写入tsv文件（数据收集）？",
    "Write Peaklists to tsv files?": "将峰值列表写入tsv文件？",
    "Content HDF5 file": "内容 HDF5 文件",
    "Peaklist(s) (HDF5 file)": "峰值列表（HDF5文件）",
    "Peak intensity matrix (HDF5 file)": "峰值强度矩阵（HDF5文件）",
    "Select format(s) tsv formatted peak instensity matrix": "选择格式（s）tsv格式化的峰值强度矩阵",
    "Should the rows or columns represent the samples?": "行或列应该代表样本吗？",
    "The Peak Matrix should contain ... values": "峰值矩阵应该包含...值",
    "Maximum percentage of missing values.": "最大缺失值百分比。",
    "Apply sample filter within each sample class.": "在每个样本类别内应用样本过滤器。",
    "Minimum fraction": "最小分数",
    "Label for the QC sample (RSD filter only)": "QC样本的标签（仅RSD过滤器）",
    "Label for the blank samples": "空白样品的标签",
    "Minimum fraction (percentage)": "最小分数（百分比）",
    "Remove blank samples (rows)": "删除空白样本（行）",
    "MSP file (output from Create MSP tool)": "MSP文件（Create MSP工具的输出）",
    "Select SIRIUS-CSI:FingerID Database": "选择 SIRIUS-CSI:FingerID 数据库",
    "Mass deviation of the fragment peaks in ppm": "碎片峰的质荷偏差（ppm）",
    "Maximum number of candidates in the output": "输出中候选项的最大数量",
    "Ion mode": "离子模式",
    "Analysis used": "分析使用",
    "Schema": "模式",
    "Choose how additional metadata columns are extracted": "选择如何提取额外的元数据列",
    "Minimum number of MS/MS peaks": "最小MS/MS峰值数量",
    "How to handle adducts": "如何处理加合物",
    "Select adducts": "选择加合物",
    "MSP file (Output from Create MSP tool)": "MSP 文件（创建 MSP 工具的输出）",
    "Choose Compound Database": "选择化合物数据库",
    "MetChem URL": "MetChem 网址",
    "Local database of compounds (CSV format)": "本地化合物数据库（CSV格式）",
    "Relative Mass Deviation for database search (ppm)": "数据库搜索的相对质量偏差（ppm）",
    "Fragment Peak Match Relative Mass Deviation (ppm)": "片段峰匹配相对质量偏差（ppm）",
    "Fragment Peak Match Absolute Mass Deviation (Da)": "片段峰值匹配绝对质量偏差（道尔顿）",
    "Polarity": "极性",
    "Suspect list": "嫌疑列表",
    "Use default list of suspect compounds?": "使用默认的可疑化合物列表？",
    "Suspect list file": "可疑列表文件",
    "Skip invalid or undefined adduct types?": "跳过无效或未定义的加成类型？",
    "Output the MetFrag command line call": "输出 MetFrag 命令行调用",
    "PreProcessing filters": "预处理过滤器",
    "filter non-connected compounds (e.g. salts)": "过滤非连接的化合物（例如盐类）",
    "filter compounds containing non-standard isotopes": "过滤包含非标准同位素的化合物",
    "Minimum Elements Filter": "最小元素过滤器",
    "Maximum Elements Filter": "最大元素过滤器",
    "Include substructures": "包含子结构",
    "Exclude substructures": "排除子结构",
    "Include elements": "包含元素",
    "Include elements (exclusive)": "包含元素（不包括）",
    "Exclude elements": "排除元素",
    "PostProcessing filters": "后处理过滤器",
    "Threshold for score after MetFrag search": "MetFrag搜索后得分的阈值",
    "Minimum percentage of explain peaks": "最小解释峰值百分比",
    "Query spectra input and filters": "查询谱输入和过滤器",
    "Library spectra input and filters": "库谱图输入和滤波器",
    "General arguments": "通用参数",
    "Filter on retention time match?": "过滤在保留时间匹配上？",
    "Retention time tolerance (seconds)": "保留时间容差（秒）",
    "Filter on matching precursors?": "过滤匹配的前体？",
    "Weighting for relative abundance": "相对丰度的加权",
    "Weighting for mz": "mz加权",
    "Update database with results?": "更新数据库以包含结果？",
    "Make a copy of the database?": "创建数据库的副本？",
    "singleFile": "单个文件",
    "Choose class of XCMS set object to perform calculation on?": "选择要在XCMS集对象上执行计算的类别？",
    "Choose class": "选择类别",
    "Ignore files?": "忽略文件？",
    "ignore files": "忽略文件",
    "xgroups": "xgroups",
    "xgroup ids": "xgroup ids",
    "Method of averaging purity": "平均纯度方法",
    "Has the raw retention time been tracked within the peaks?": "原始保留时间是否已在峰值内跟踪？",
    "Choose the source for the dataset": "选择数据集的来源",
    "Single *.mzML": "单个 *.mzML",
    "Data collection containing *.mzml files": "数据收集包含*.mzml文件",
    "Name of file to to calculate purity from": "用于计算纯度的文件名",
    "Library directory containing *.mzml files (dimspy input only) or path to an individual .mzML file": "包含 *.mzml 文件的库目录（仅 dimspy 输入）或指向单个 .mzML 文件的路径",
    "ppm": "每百万分率",
    "Remove rows where mz value is NA or NaN?": "删除mz值为NA或NaN的行？",
    "SIM-Stitch experiment?": "SIM-Stitch 实验？",
    "dimspy peak matrix text file usage?": "dimspy 峰值矩阵文本文件用法？",
    "File number": "文件编号",
    "Peaklist to merge": "峰值列表合并",
    "Create multiple merged peaklists?": "创建多个合并的峰值列表？",
    "Filelist": "文件列表",
    "msPurity purityA dataset": "msPurity purityA 数据集",
    "How to choose fragmentation spectra (if precursor was fragmented in >1 scans per XCMS group feature)": "如何选择碎片谱（如果前体在XCMS组特征中被碎片化超过1次扫描）",
    "Use additional metadata?": "使用额外的元数据？",
    "Metadata for each feature": "每个特征的元数据",
    "Metadata columns to use": "要使用的元数据列",
    "Metadata columns for MSP spectra name": "MSP 谱图名称的元数据列",
    "Select XCMS groups?": "选择 XCMS 组？",
    "XCMS peak group ids": "XCMS 峰值组标识",
    "Include intensity, relative abundanace or both in the MSP file": "在MSP文件中包含强度、相对丰度或两者",
    "MSP schema to use for files": "MSP 文件使用的模式",
    "Filter peaks that have been flagged in prior processing steps": "过滤在先前处理步骤中被标记的峰值",
    "Always include the following adduct descriptions in the MSP file": "在MSP文件中始终包含以下加合物描述",
    "Always include the following custom adducts": "总是包括以下自定义加合物",
    "Create MSP spectra for each adduct?": "为每种加合物创建MSP谱图？",
    "xcmsSet or CAMERA object": "xcmsSet 或 CAMERA 对象",
    "purityA object (frag4feature output)": "纯度A对象（frag4feature输出）",
    "Include EIC data?": "包括EIC数据？",
    "purityA object": "纯度A对象",
    "Average and filter fragmentation spectra for each XCMS feature": "为每个 XCMS 特征平均和过滤碎片谱",
    "Signal-to-noise threshold after averaging or summing": "信号-噪声阈值在平均或求和之后",
    "Relative abundance threshold after averaging or summing": "平均或求和后相对丰度阈值",
    "Minimum fraction (i.e. percentage) of (averaged) scans a fragment peak has to be present in.": "最小分数（即百分比）的（平均）扫描，片段峰必须存在于其中。",
    "Minimum number of (averaged) fragmentation scans for a fragmentation event (precursor).": "一个断裂事件（前体）所需的最小（平均）断裂扫描数。",
    "Sum intensities across (averaged) scans?": "在（平均化的）扫描中求和强度？",
    "Function to calculate the average intensity, m/z and SNR values across (averaged) scans after filtering.": "计算过滤后（平均化）扫描中平均强度、m/z和信噪比（SNR）值的函数。",
    "Remove peaks that do not meet the filtering criteria. Otherwise peaks will be flagged instead.": "移除不满足过滤标准的峰。否则，峰将被标记。",
    "purityA object with fragmentation linked to XCMS features": "纯度A 与 XCMS 特征相关联的片段化对象",
    "Miniumum precursor ion purity of the associated precursor for fragmentation spectra scan": "与碎片化谱扫描相关前体的最小前体离子纯度",
    "Peak instensity threshold": "峰值强度阈值",
    "Relative abundance threshold": "相对丰度阈值",
    "Signal-to-noise threshold": "信噪比阈值",
    "Function to calculate noise.": "计算噪声的函数。",
    "Perform filtering on all fragmentation spectra (not just those aligned to XCMS features)": "对所有碎片谱图进行过滤（不仅仅是那些与XCMS特征对齐的谱图）",
    "xcmsSet object": "xcmsSet 对象",
    "ppm error tolerance between precursor mz and XCMS feature mz": "前体mz与XCMS特征mz之间的ppm误差容限",
    "Precursor ion purity threshold": "前体离子纯度阈值",
    "Should the most intense precursor be used within the isolation window?": "是否应在隔离窗口内使用最强烈的前体？",
    "Was retention time correction used?": "使用了保留时间校正吗？",
    "For matching fragmentation to a feature, use the grouped feature range": "对于将片段匹配到特征，使用分组的特征范围",
    "*.mzML file": "*.mzML 文件",
    "Use most intense peak within isolation window for precursor?": "使用隔离窗口内最强烈的峰作为前体？",
    "Use nearest full scan to determine precursor?": "使用最近的完整扫描来确定前体？",
    "Interpolation PPM": "插值 PPM",
    "offsets": "偏移量",
    "Change biological sample flag parameters?": "更改生物样本标志参数？",
    "rsd_i_sample": "rsd_i_sample",
    "minfrac_sample": "最小样本比例",
    "rsd_rt_sample": "rsd_rt样本",
    "ithres_sample": "样本阈值",
    "Change blank flag parameters?": "更改空白标志参数？",
    "blank_class": "空白类",
    "rsd_i_blank": "rsd_i_blank",
    "minfrac_blank": "最小分数空白",
    "rsd_rt_blank": "rsd_rt_blank",
    "ithres_blank": "ithres_blank",
    "s2b": "s2b",
    "Remove peaks from xcmsSet object?": "从xcmsSet对象中移除峰值？",
    "minfrac_xcms": "minfrac_xcms",
    "mzwid": "mzwid",
    "bw": "带宽",
    "egauss_thr": "高斯阈值",
    "temp_save": "临时保存",
    "polarity": "极性",
    "grp_rm_ids": "grp_rm_ids",
    "xset_name": "x集名称",
    "Samplelist": "样本列表",
    "samplelist": "样本列表",
    "Variable metadata": "变量元数据",
    "Column containing the sample-class names (or treatment names)": "包含样本类别名称（或处理名称）的列",
    "Sample-class names (or patterns)": "样本类别名称（或模式）",
    "Exclude/include named (or matched) sample classes": "排除/包含命名的（或匹配的）样本类别",
    "Use 'wild card patterns' or 'regular expression patterns' to match sample-class names": "使用“通配符模式”或“正则表达式模式”来匹配样本类别名称",
    "Variable-range filters": "变量范围过滤器",
    "Imputation of missing values": "缺失值的填充",
    "Columns that specify order for samples": "指定样本顺序的列",
    "Columns that specify order for features": "指定特征排序的列",
    "Compute centers for classes (e.g., treatments)": "计算类别（例如，处理）的中心",
    "Dissimilarity to be used for clustering": "用于聚类的不相似性",
    "Number of sample clusters to identify": "要识别的样本聚类数量",
    "Number of variable clusters to identify": "要识别的变量聚类数量",
    "Method of correlation to be used": "要使用的关联方法",
    "Method of agglomeration to be used": "使用的聚合方法",
    "Color scale": "颜色尺度",
    "Variable standardization (for plotting only)": "变量标准化（仅限绘图）",
    "Size of labels": "标签大小",
    "MS data": "质谱数据",
    "MS/MS data (optional)": "串联质谱数据（可选）",
    "Structural formula existence check": "结构式存在性检查",
    "Valency sum parity": "价键和奇偶性",
    "Lower bound for valency sum - 2 * maximum valency for graphical formulas": "图形公式中价和的下界 - 2 * 最大价和",
    "Lower bound for valency sum - 2 * number of atoms + 2 for graphical connected formulas": "图形连接式中价和的下界 - 2 * 原子数 + 2",
    "Lower bound for valency sum - 2 * maximum valency for fragment ions": "价和的下限 - 2 * 片段离子的最大价和",
    "Lower bound for valency sum - 2 * number of atoms + 2 for fragment ions": "片段离子的价和下界 - 2 * 原子数 + 2",
    "Experimental molecular mass": "实验分子质量",
    "Type of ion measured": "测量的离子类型",
    "Accuracy of measurement of MS1": "MS1测量的准确性",
    "MS match value": "MS匹配值",
    "Allowed deviation for full acceptance": "完全接受的允许偏差",
    "Allowed deviation for total rejection": "允许的总拒绝偏差",
    "Threshold for the MS match value": "MS匹配值的阈值",
    "Threshold for the MS/MS match value": "MS/MS匹配值的阈值",
    "Threshold for the combined match value": "组合匹配值的阈值",
    "Sort generated formulas by": "按生成的公式排序",
    "Chemical element filtering": "化学元素过滤",
    "Used chemical elements": "使用化学元素",
    "Uses fuzzy formula for limits of element multiplicities": "使用模糊公式来限制元素的多重性",
    "Formulas must have at least one hetero atom": "公式必须至少有一个杂原子",
    "Apply Heuerding-Clerc filter": "应用Heuerding-Clerc滤波器",
    "Weighting of m/z for MS/MS match value": "质谱/质谱匹配值的 m/z 权重",
    "Intensity weighting for MS/MS match value": "MS/MS匹配值的强度加权",
    "Exponent": "指数",
    "Allow odd electron ions for explaining MS/MS peaks": "允许奇数电子离子以解释MS/MS峰",
    "Excess of double bond equivalent for ions": "离子的双键等价物过量",
    "Output options for main output": "主输出的输出选项",
    "Write explanations for MS/MS peaks to output": "为MS/MS峰值撰写解释以输出",
    "For analyzing MS/MS peaks write losses instead of fragments": "在分析MS/MS峰时，写入丢失而不是片段。",
    "Write intensities of MS/MS peaks to output": "将MS/MS峰的强度写入输出",
    "Write double bond equivalents to output": "将双键等效物写入输出",
    "Write calculated ion masses to output": "将计算出的离子质量写入输出",
    "Output match values in percent": "输出匹配值的百分比",
    "Strip calculated isotope distributions": "去除计算的同位素分布",
    "Additional output data sets": "附加输出数据集",
    "Write scaled MS peaks output": "写入缩放的MS峰值输出",
    "Write scaled MS/MS peaks output": "写入缩放后的MS/MS峰值输出",
    "Write MS/MS with only explained peaks": "撰写仅包含解释峰的MS/MS",
    "Zip file": "压缩文件",
    "Exclusion zone(s)": "排除区域（一个或多个）",
    "exclusion zones": "排除区域",
    "Left exclusion zone border": "左侧排除区域边界",
    "Right exclusion zone border": "右侧排除区域边界",
    "Graphical zone(s)": "图形区域",
    "grapical zones": "图形区域",
    "Left graphical zone border": "左侧图形区域边框",
    "Right graphical zone border": "正确的图形区域边界",
    "@INPUT_IMAGE_LABEL@": "输入图像标签",
    "Type of metadata": "元数据类型",
    "Computation method": "计算方法",
    "Statistics": "统计学",
    "Class column": "类别列",
    "Calculate the mean fold change": "计算平均倍数变化",
    "Logarithm": "对数",
    "Selected class": "选定类别",
    "Where should the class be placed for the mean fold change calculation?": "类应该放置在哪个位置以计算平均倍数变化？",
    "Metadata file containing your new IDs": "包含您新ID的元数据文件",
    "Which ID do you want to change?": "您想要更改哪个ID？",
    "Name of the column to consider as new ID": "要作为新ID考虑的列名",
    "Force unicity if not unique?": "强制唯一性，如果不唯一？",
    "Data matrix of preprocessed data": "预处理数据的数据矩阵",
    "Sample metadata matrix": "样本元数据矩阵",
    "Name of the column of the biological factor of interest (for PQN method)": "生物因素列的名称（用于PQN方法）",
    "Name of reference level for PQN normalization": "PQN标准化的参考水平名称",
    "Name of the column of the numerical variable for normalization (weight, osmolality, ...)": "用于归一化的数值变量的列名（权重，渗透压，...）",
    "Spectra representation": "光谱表示",
    "Fixed Factor of interest": "感兴趣的固定因子",
    "Repeated factor (time)": "重复因素（时间）",
    "Subject factor": "主题因素",
    "Degrees of freedom method for the F-tests": "自由度方法用于F检验",
    "Method for multiple testing correction": "多重检验校正方法",
    "Log transform of raw data": "原始数据的对数转换",
    "(Corrected) p-value significance threshold": "（校正后的）P值显著性阈值",
    "Perform diagnostic of the residuals": "执行残差的诊断",
    "Rounding the result's numerical columns": "将结果的数值列四舍五入",
    "Number of decimal places for rounding": "小数位数用于四舍五入",
    "Name of the sampleMetadata column containing the 1st factor for A-SCA": "样本元数据列的名称，包含 A-SCA 的第一个因子",
    "Name of the sampleMetadata column containing the 2nd factor for A-SCA": "样本元数据列的名称，包含 A-SCA 的第二个因素",
    "Scaling to apply to dataMatrix": "缩放以应用于数据矩阵",
    "Number of permutation to perform to compute factor significance": "执行排列的数量以计算因子的重要性",
    "Threshold for factor significance (permutation test)": "因子显著性阈值（置换检验）",
    "Table 1 file": "表1 文件",
    "Where are the samples in table 1?": "表1中的样本在哪里？",
    "Table 2 file": "表 2 文件",
    "Where are the samples in table 2?": "表2中的样本在哪里？",
    "Method to calculate the correlation coefficients": "计算相关系数的方法",
    "Significance test for the correlation coefficients": "相关系数的显著性检验",
    "Filter the correlation table": "过滤相关性表格",
    "Do you want to use only zero filter or combine it with the threshold filter?": "您想要仅使用零过滤器，还是将其与阈值过滤器结合使用？",
    "Reorder variables (using Hierarchical Cluster Analysis)": "重新排序变量（使用层次聚类分析）",
    "PDF output": "PDF输出",
    "Colored correlation table strategy": "彩色相关性表格策略",
    "Choose the type of classes": "选择类别类型",
    "Class size": "班级大小",
    "Vector with values for classes": "类别的向量值",
    "Would you use a file": "您会使用一个文件吗？",
    "File of masses (Variable Metadata)": "质量文件（变量元数据）",
    "Do you have a header": "你有标题吗？",
    "Column of masses": "质量列",
    "Mass to submit": "要提交的质量",
    "Mass-to-charge ratio": "质荷比",
    "Number of maximum entries returned by the query": "查询返回的最大条目数",
    "Molecular Species": "分子物种",
    "Select multiple adducts": "选择多个加合物",
    "Performing advanced annotation": "执行高级注释",
    "* Would you use a file?": "* 您会使用文件吗？",
    "FILE OF MASSES (format: tabular)": "质量文件（格式：表格）",
    "Do you have a header?": "你有标题吗？",
    "Would you use retention time (RT)?": "你会使用保留时间（RT）吗？",
    "Column of retention time (RT)": "保留时间（RT）的列",
    "Delta of retention time (RT) (in minutes)": "保留时间（RT）的差值（以分钟为单位）",
    "Column of masses (MZ)": "质量柱（MZ）",
    "retention time (RT) to submit": "保留时间 (RT) 提交",
    "Mass (MZ) to submit": "质量（MZ）提交",
    "Delta type": "德尔塔类型",
    "Delta of mass (MZ) (in Dalton)": "质量差异（MZ）（以道尔顿为单位）",
    "Delta of mass (MZ) (in ppm)": "质量误差（MZ）（以ppm为单位）",
    "* Use personal data bank?": "* 使用个人数据银行？",
    "BANK FILE": "银行文件",
    "Column of masses (MZ) in Bank file": "Bank文件中的质量（MZ）列",
    "Have you used the retention time (RT) for your input data?": "您是否已经使用了输入数据的保留时间（RT）？",
    "Column of retention time (RT) in Bank file": "保留时间（RT）列在Bank文件中",
    "* Would you more output?": "* 您需要更多的输出吗？",
    "File of masses (format: msp)": "质谱文件（格式：msp）",
    "Masses and intensities (entered manually)": "质量和强度（手动输入）",
    "Alkane Retention Index": "烷烃保留指数",
    "Retention Index Window": "保留指数窗口",
    "Maximum Hits": "最大命中数",
    "Number of significant decimal": "有效小数位数",
    "Maximum number of ions": "最大离子数",
    "JaccardDistance": "杰卡德距离",
    "s12GowerLegendreDistanceThreshold": "S12 高尔传奇距离阈值",
    "DotproductDistanceThreshold": "点积距离阈值",
    "HammingDistanceThreshold": "汉明距离阈值",
    "EuclideanDistanceThreshold": "欧几里得距离阈值",
    "Type of intensities": "强度类型",
    "Data matrix of FIDs": "FIDs的数据矩阵",
    "Group Delay Correction": "群延迟校正",
    "Display the FIDs after 1st order phase correction?": "显示一阶相位校正后的FIDs？",
    "Solvent Suppression": "溶剂抑制",
    "Solvent Suppression: Smoothing parameter": "溶剂抑制：平滑参数",
    "Display the FIDs after solvent suppression?": "显示溶剂抑制后的自由感应衰减（FIDs）？",
    "Apodization": "Apodization",
    "Apodization: method": "Apodization：方法",
    "Line broadening": "线宽展",
    "Phase": "相位",
    "Proportion of signal in the window": "窗口中信号的比例",
    "Display the FIDs after Apodization?": "显示经过平滑处理后的FIDs（自由感应衰减信号）？",
    "Fourier Transform": "傅里叶变换",
    "Display the Fourier transformed spectra?": "显示傅里叶变换后的光谱？",
    "Zero Order Phase Correction": "零阶相位校正",
    "Zero Order Phase Correction: method": "零阶相位校正：方法",
    "Zero Order Phase Correction: exclusion area(s)": "零阶相位校正：排除区域",
    "Exclusion_zone": "排除区域",
    "Excusion zone: left border": "排除区：左侧边界",
    "Excusion zone: right border": "排除区：右边界",
    "Display the spectra after the Zero Order Phase Correction?": "显示零阶相位校正后的光谱？",
    "Shift Referencing": "移位参考",
    "Shift Referencing: definition of the search zone": "位移参考：搜索区域的定义",
    "percentage of the ppm axis around the ppm value to look for the reference compound peak": "在ppm值周围用于查找参考化合物峰的ppm轴的百分比",
    "Search_zone": "搜索区域",
    "Search zone: left border": "搜索区域：左边界",
    "Search zone: right border": "搜索区域：右边界",
    "Shift Referencing: shiftHandling": "移位参考：shiftHandling",
    "Shift Referencing: the value of the reference peak in ppm": "位移参考：参考峰值在ppm中的值",
    "Display the spectra after Shift Referencing?": "显示进行位移参考后的光谱？",
    "Baseline Correction": "基线校正",
    "Baseline Correction: smoothing parameter": "基线校正：平滑参数",
    "Baseline Correction: asymmetry parameter": "基线校正：不对称性参数",
    "Baseline Correction: numerical precision for convergence when estimating the baseline": "基线校正：估计基线时收敛的数值精度",
    "Baseline Correction: exclusion area(s)": "基线校正：排除区域",
    "Display the spectra after Baseline Correction?": "显示基线校正后的光谱？",
    "Negative intensities to Zero": "负强度归零",
    "Set negative intensities to zero?": "将负强度设置为零？",
    "Display the final spectra?": "显示最终的光谱？",
    "Bruker FID file": "布鲁克 FID 文件",
    "Specify the line in the title file to recover the FID names (usually in pdata/1/title)": "指定标题文件中用于恢复 FID 名称的行（通常位于 pdata/1/title）",
    "Presence of subdirectories?": "子目录的存在？",
    "Use (sub)directories names as FID names?": "使用（子）目录名称作为FID名称？",
    "xset RData file": "xset RData 文件",
    "Group co-eluted peaks based on RT [groupFWHM]": "根据保留时间（RT）将共洗脱峰合并 [groupFWHM]",
    "Multiplier of the standard deviation": "标准差的乘数",
    "Percentage of FWHM width": "FWHM宽度的百分比",
    "Annotation general options": "注释通用选项",
    "General ppm error": "通用每百万误差",
    "General absolut error in m/z": "通用绝对误差在 m/z",
    "Annotate Isotopes [findIsotopes]": "注释同位素 [findIsotopes]",
    "Max. ion charge": "最大离子电荷",
    "Max. number of expected isotopes": "最大预期同位素数",
    "The percentage number of samples, which must satisfy the C12/C13 rule for isotope annotation": "样品中必须满足C12/C13规则以进行同位素注释的百分比编号",
    "Verifying grouping co-eluted peaks [groupCorr]": "验证共洗脱峰的分组 [groupCorr]",
    "groupCorr: correlation threshold (0..1)": "groupCorr：相关性阈值（0..1）",
    "groupCorr: Method selection for grouping peaks after correlation analysis into pseudospectra": "groupCorr：在将相关分析后的峰值分组到伪谱图中时，选择分组方法",
    "groupCorr: significant correlation threshold": "groupCorr：显著相关性阈值",
    "groupCorr: Use correlation inside samples for peak grouping": "groupCorr：在峰值分组内部使用相关性",
    "groupCorr: Use isotopic relationship for peak grouping": "groupCorr：使用同位素关系进行峰值分组",
    "groupCorr: Use correlation across samples for peak grouping": "groupCorr：使用样本间的相关性进行峰值分组",
    "Annotate Adducts [findAdducts]": "注释加成物 [findAdducts]",
    "Which polarity mode was used for measuring of the ms sample": "测量质谱样品时使用了哪种极性模式",
    "How much peaks will be calculated in every thread using the parallel mode": "在并行模式下，每个线程将计算多少个峰值",
    "Use a personal ruleset file": "使用个人规则集文件",
    "If no ruleset is provided, calculate ruleset with max. number n of [nM+x] clusterions": "如果未提供规则集，则使用最大数量为 n 的 [nM+x] 簇离子来计算规则集",
    "User defined ruleset": "用户定义的规则集",
    "for the output label": "对于输出标签",
    "Statistics and results export: [diffreport]": "统计和结果导出：[diff报告]",
    "Number of condition": "条件数量",
    "Number of the most significantly different analytes to create EICs for": "创建EICs的最具显著性差异分析物的数量",
    "Width (in seconds) of EICs produced": "EICs产生的宽度（以秒为单位）",
    "Intensity values to be used for the diffreport": "用于diffreport的强度值",
    "Numeric variable for the height of the eic and boxplots that are printed out": "用于打印出的 EIC 高度和箱线图的数值变量",
    "Numeric variable for the width of the eic and boxplots print out made": "用于打印出的箱线图和EIC（提取离子色谱图）宽度的数值变量",
    "Number of decimal places of title m/z values in the eic plot": "EIC图中标题m/z值的小数位数",
    "logical indicating whether the reports should be sorted by p-value": "逻辑值，指示报告是否应按p值排序",
    "Export the Diffreport files in": "导出Diffreport文件到",
    "Export the EIC and boxplots in": "导出EIC和箱线图",
    "Value by which the mz width of peaks should be expanded": "峰值mz宽度应扩展的值",
    "Value by which the RT width of peaks should be expanded": "峰值的保留时间宽度应该扩展的值",
    "Specifying a ppm by which the mz width of the peak region should be expanded": "指定一个ppm值，用于扩展峰区域的m/z宽度",
    "Method to use for retention time correction": "保留时间校正使用的方法",
    "Minimum required fraction of samples in which peaks for the peak group were identified": "峰值组中识别峰值所需的最小样本分数",
    "Maximal number of additional peaks for all samples to be assigned to a peak group for retention time correction": "保留时间校正中，所有样本分配给峰组的额外峰的最大数量",
    "Smooth method": "平滑方法",
    "Degree of smoothing for the loess fitting": "Loess拟合的平滑程度",
    "Family": "家族",
    "Bin size (in mz dimension) to be used for the profile matrix generation": "用于生成轮廓矩阵的mz维度的箱大小",
    "Index of the center sample in the experiment": "实验中心样本的索引",
    "Defining the responsiveness of warping": "定义弯曲的响应性",
    "Distance function to be used": "要使用的距离函数",
    "Penalty for gap enlargement": "间隔增大的惩罚",
    "Local weight applied to diagonal moves in the alignment": "对齐中对对角线移动应用的局部权重",
    "local weight for gap moves in the alignment": "在比对中的间隙移动的局部权重",
    "Whether a local alignment should be performed instead of the default global alignment": "是否应执行局部比对而不是默认的全局比对",
    "Penalty for initiating an alignment": "对齐起始的惩罚",
    "Method to use for grouping": "用于分组的方法",
    "Bandwidth": "带宽",
    "Minimum fraction of samples": "最小样本分数",
    "Minimum number of samples": "最小样本数",
    "Width of overlapping m/z slices": "m/z 切片重叠宽度",
    "Maximum number of groups to identify in a single m/z slice": "在单个 m/z 切片中要识别的最大组数",
    "Relative mz error for clustering/grouping in ppm": "聚类/分组时相对质量误差（mz）以ppm（百万分之一）为单位",
    "Absolute mz error for clustering/grouping": "聚类/分组的绝对 m/z 误差",
    "Factor by which mz values are multiplied before calculating the (euclician) distance between two peaks": "在计算两个峰之间的（欧几里得）距离之前，mz值乘以的因子",
    "Maximum tolerated distance for mz values": "最大容忍的mz值距离",
    "Maximum tolerated distance for RT values": "RT值的最大容忍距离",
    "Number of nearest Neighbours to check": "最近邻的数量",
    "Spectra Filters": "光谱滤波器",
    "Filter on Acquisition Numbers": "筛选采集编号",
    "Filter on Retention Time": "保留时间过滤",
    "Filter on Mz": "过滤在 Mz 上",
    "Extraction method for peaks detection": "峰值检测的提取方法",
    "Get a list of found chromatographic peaks": "获取一系列已发现的色谱峰",
    "List of regions-of-interest (ROI)": "感兴趣区域（ROI）列表",
    "Full width at half maximum of matched filtration gaussian model peak": "匹配滤波高斯模型峰值的半高宽",
    "Step size to use for profile generation": "用于生成概要文件的步长",
    "Method to be used for missing value imputation": "用于缺失值插补的方法",
    "The base value to which empty elements should be set": "应将空元素设置为基础值",
    "Number of non-empty neighboring element of an empty element that should be considered for linear interpolation.": "考虑对空元素进行线性插值时，应考虑的非空相邻元素的数量。",
    "Standard deviation (width) of matched filtration model peak": "匹配滤波模型峰值的标准差（宽度）",
    "Maximum number of peaks that are expected/will be identified per slice": "每个切片预期/将被识别的峰值的最大数量",
    "Signal to Noise ratio cutoff": "信噪比截止值",
    "Number of bins to be merged before filtration": "合并前过滤的箱数",
    "Minimum difference in m/z for peaks with overlapping Retention Times": "最小 m/z 差异，用于具有重叠保留时间的峰",
    "Signal to noise ratio cutoff": "信噪比截止值",
    "Maximal isotope charge": "最大同位素电荷",
    "Number of isotope peaks that should be predicted for each peak identified in the first centWave run": "在第一次centWave运行中，为每个已识别的峰预测的同位素峰的数量",
    "Is the mz range for the predicted isotope ROIs should be extended to increase detection of low intensity peaks": "是否应该扩展预测同位素感兴趣区域的m/z范围，以增加检测低强度峰的能力",
    "Polarity of the data": "数据的极性",
    "verbose Columns": "详细 输出列",
    "Scales of the Continuous Wavelet Transform (CWT)": "连续小波变换（CWT）的尺度",
    "Determine whether to include the nearby small peaks of major peaks": "确定是否包括主要峰附近的小峰",
    "Minimum required relative amplitude of the peak": "峰值所需的最小相对幅度",
    "File(s) from your history containing your chromatograms": "您历史记录中包含您的色谱图的文件",
    "Masses Column": "质量列",
    "Ionisation mode of your submitted data": "您提交数据的电离模式",
    "Do you want to modify input masses ?": "你想修改输入的质量吗？",
    "If you want to transform masses extracted from file with oxidation(s)": "如果你想要从文件中提取并转换氧化态的信息",
    "If you want to correct masses extracted from file with neutral loss": "如果你想校正从文件中提取的质量并进行中性丢失校正",
    "Do you want to perform a multiple queries on input masses ?": "您想要对输入的质量进行多个查询吗？",
    "Select the column containing category or class or sub class identifiants in your input file :": "选择输入文件中包含类别或类别或子类别标识符的列：",
    "Select a Lipid category for your query": "选择一个脂质类别进行查询",
    "Select a Lipid main class for your query": "选择一个脂质主类进行查询",
    "Select a Lipid subclass for your query": "选择一个脂质子类作为您的查询",
    "Type of cut of the masses": "质谱切割类型",
    "Variable metadata file (ideally from Univariate)": "变量元数据文件（理想情况下来自单变量分析）",
    "Factor of interest": "感兴趣的因素",
    "Univariate significance-test": "单变量显著性检验",
    "Levels of interest": "兴趣水平",
    "Level-name matching": "级别名称匹配",
    "How many features having extreme loadings should be labelled on cov-vs.-cor plot?": "在协方差与相关性图中，应该标记多少个具有极值载荷的特征？",
    "Minumum number of samples for OPLS-DA cross-validation.": "OPLS-DA交叉验证的最小样本数。",
    "Advanced (C-plots and customized p-value adjustment)": "高级（C-plots和定制化的p值调整）",
    "Data matrix positive": "数据矩阵正向",
    "Sample metadata positive": "样本元数据 阳性",
    "Variable metadata positive": "变量元数据 阳性",
    "Data matrix negative": "数据矩阵负值",
    "Sample metadata negative": "样本元数据 阴性",
    "Variable metadata negative": "变量元数据负数",
    "Deleting samples and/or variables according to Numerical values": "根据数值删除样本和/或变量",
    "Identify the parameter to filter": "识别过滤参数",
    "On file": "在文件上",
    "Name of the column to filter": "要筛选的列名",
    "Interval of values to remove": "要移除的值的区间",
    "Remove all values lower than": "删除所有低于的值",
    "Remove all values upper than": "删除所有大于的值",
    "Remove all values between": "移除所有值之间的",
    "And": "并且",
    "And upper than": "并且高于",
    "Deleting samples and/or variables according to Qualitative values": "根据定性值删除样本和/或变量",
    "Removing a level in factor": "移除因子中的一个水平",
    "Remove factor when": "移除因子时",
    "Coefficient of Variation": "变异系数",
    "Which type of CV calculation should be done": "哪种类型的CV（变异系数）计算应该被执行",
    "Use 'pool' samples as 'pool1' when computing the correlation with dilution?": "在计算稀释相关性时，使用“pool”样本作为“pool1”？",
    "Rdata from xcms and merged": "来自xcms的R数据和合并后的数据",
    "RT range option": "RT范围选项",
    "RTrange": "RT范围",
    "Use Personnal DataBase option": "使用个人数据库选项",
    " DB file": "数据库文件",
    "Use RI option": "使用 RI 选项",
    " RI file": "RI文件",
    "RT_Diff": "RT_Diff",
    "Min_Features": "最小特征数",
    "similarity_threshold": "相似性阈值",
    "min.class.fract": "最小类群分数",
    "min.class.size": "最小类大小",
    "Use RI as filter": "使用RI作为过滤器",
    "RIshift": "RI偏移",
    "Repeated mesurement label (Individual IDs, ...)": "重复测量标签（个体ID，...）",
    "Select number of levels1": "选择层级数1",
    "Level name (as in sampleMetadata)": "水平名称（如在sampleMetadata中）",
    "First level name (as in sampleMetadata)": "第一级名称（如在sampleMetadata中）",
    "Second level name (as in sampleMetadata)": "二级名称（如在sampleMetadata中）",
    "Add transformation to dataMatrix before withinVariation": "在 withinVariation 之前对 dataMatrix 添加转换",
    "Transformation": "转换",
    "Centering": "居中",
    "Make syntactically valid sample and variable names": "创建语法上有效的样本和变量名称",
    "Type of regression model": "回归模型类型",
    "File of masses (format: tabular)": "质量文件（格式：表格）",
    "Column of pcgroup": "列的pcgroup",
    "Do you have a column of intensity?": "你有强度列吗？",
    "Column of intensity": "强度列",
    "What kind of Ionization Search will be done?": "将进行哪种电离搜索？",
    "Score threshold to apply on returned MassBank results": "返回的MassBank结果应用的分数阈值",
    "Show advanced settings?": "显示高级设置？",
    "What kind of Instruments will be use?": "将使用哪种仪器？",
    "Unit of tolerance": "容差单位",
    "Tolerance of values of m/z of peaks (in unit)": "峰值的 m/z 值的容差（以单位表示）",
    "Tolerance of values of m/z of peaks (in ppm)": "峰值的m/z值的容差（以ppm为单位）",
    "Ignore peaks whose intensity is not larger than the value of cutoff": "忽略强度不大于截止值的峰值",
    "Which MassBank server start searching?": "哪个MassBank服务器开始搜索？",
    "Positive RData ion mode": "正离子模式下的RData",
    "Negative RData ion mode": "负 RData 离子模式",
    "Returned peaklist polarity mode": "返回峰值列表极性模式",
    "Retention time window in seconds": "保留时间窗口（以秒为单位）",
    "Matrix of matching rules": "匹配规则矩阵",
    "Keep only the metabolites which match a difference": "只保留那些差异匹配的代谢物",
    "Export options": "导出选项",
    "Convert retention time (seconds) into minutes": "将保留时间（秒）转换为分钟",
    "Number of decimal places for mass values reported in ions' identifiers.": "在离子标识符中报告的质量值的小数位数。",
    "Number of decimal places for retention time values reported in ions' identifiers.": "保留在离子标识符中报告的保留时间值的小数位数。",
    "Choose your input method": "选择您的输入方式",
    "Zip file (see the details for file upload in the help section below)": "压缩文件（有关文件上传的详细信息，请参见下面的帮助部分）",
    "Library directory name": "库目录名称",
    "Maximum deviation between centroids during band detection (in ppm)": "在频带检测期间，质心之间的最大偏差（以ppm为单位）",
    "Minimal maximum deviation between centroids during band detection (in Da)": "在带检测期间，质心之间的最小最大偏差（以道尔顿为单位）",
    "Accuracy of the mass spectrometer to be used during feature alignment (in ppm)": "在特征对齐期间使用的质谱仪的准确度（以ppm为单位）",
    "Minimal accuracy of the mass spectrometer to be used during feature alignment (in Da)": "在特征对齐期间使用的质谱仪的最小精确度（以道尔顿为单位）",
    " Minimum fraction of samples in which a peak should be detected in at least one class to be kept during feature alignment": "在特征对齐过程中，至少在一个类别中检测到峰值的样本的最小比例，以便保留",
    "Minimum fraction of centroids in the estimated injection window for a band to be built": "在估计的注入窗口中，要构建一个波段，所需的质心最小比例",
    "Minimum number of consecutive centroids for a band to be built": "最小连续质心数，用于构建一个条带",
    "First scan to be preprocessed": "首次扫描要进行预处理",
    "Last scan to be preprocessed": "最后一次扫描需要预处理",
    "Choose your inputs method": "选择您的输入方法",
    "Tsv file": "制表符分隔值文件",
    "Bucket width": "桶宽度",
    "Left Border": "左侧边界",
    "Right Border": "右边界",
    "Harmonization metadata": "标准化元数据",
    "Spectral library format": "光谱库格式",
    "Export peak comments": "导出峰值注释",
    "Export style": "导出样式",
    "Fingerprint similarity distance": "指纹相似度距离",
    "Similarity scores": "相似性分数",
    "Score Layer": "得分层",
    "Identifier key": "标识符键",
    "Network-graph format": "网络图格式",
    "Number of top edges to keep": "保留顶部边缘的数量",
    "Maximum number of links per node": "每个节点的最大链接数",
    "Score cutoff": "分数截断",
    "Link method": "链接方法",
    "Keep unconnected nodes": "保留未连接的节点",
    "Similarity metric": "相似性度量",
    "Keys to remove": "要删除的键",
    "Splitting method": "分割方法",
    "appearance separated time": "出现分离时间",
    "split plots by condition": "按条件分割图表",
    "share y axis": "共享 y 轴",
    "xticks text size": "X轴刻度文字大小",
    "yticks text size": "y轴刻度文字大小",
    "height of each stack": "每个堆栈的高度",
    "width of each stack": "每个堆栈的宽度",
    "inner number size": "内部数字大小",
    "max number carbons possible": "最大可能的碳原子数",
    "Query dataset": "查询数据集",
    "Reference dataset": "参考数据集",
    "Input mass spectral library with peak comments in RIKEN format.": "输入具有RIKEN格式注释的质谱库。",
    "only retain annotated peaks": "仅保留注释的峰值",
    "GC column type": "GC含量列类型",
    "Method for peak refinement": "峰值细化方法",
    "MaxPeakWidth": "最大峰宽",
    "minimal required intensity": "最小必需强度",
    "minimum number of data points per peak": "每个峰值所需的最小数据点数",
    "Definition of peak intensity": "峰值强度的定义",
    "rt expansion (seconds)": "运行时间扩展（秒）",
    "m/z expansion (m/z units)": "m/z 扩展（m/z 单位）",
    "ppm expansion (ppm units)": "ppm 扩展（ppm 单位）",
    "minimum proportion at midpoint": "最小中点比例",
    "msLevel for refinement": "msLevel 用于优化",
    "Upload data that has been corrected by Isocor (.tsv format)": "上传经过 Isocor 校正的数据（.tsv 格式）",
    "Upload template file containing plot metadata (.xlsx format)": "上传包含绘图元数据的模板文件（.xlsx格式）",
    "Plot format": "绘图格式",
    "Data to plot": "绘图数据",
    "Select static plots to generate": "选择生成静态图表",
    "Flag to generate barplots": "生成条形图的标志",
    "Flag to generate meaned barplots": "生成均值条形图的标志",
    "Flag to generate heatmap": "生成热图的标志",
    "Flag to generate clustermap": "生成聚类图的选项",
    "Select interactive plots to generate": "选择要生成的交互式图表",
    "Flag to unstack bars in barplots": "在条形图中取消堆叠条形的选项",
    "Flag to get debug information": "获取调试信息的标志",
    "Flag to add annotations on the maps": "在地图上添加注释的标志",
    "Input spectra data": "输入光谱数据",
    "Input extracted feature samples collection": "输入提取的特征样本集合",
    "Input corrected feature samples collection": "输入校正特征样本集合",
    "Metadata table": "元数据表",
    "RT table": "实时表格",
    "Intensity table": "强度表",
    "Input clustered features table": "输入聚类特征表",
    "Input template features table": "输入模板特征表",
    "Input data": "输入数据",
    "Table of known features": "已知特征表",
    "Tables merge direction": "表格合并方向",
    "Minimal occurence of feature": "特征的最小出现次数",
    "Match tolerance [ppm]": "匹配容差 [ppm]",
    "Clustered features": "聚类特征",
    "Minimal occurrence in samples": "样本中的最小出现次数",
    "Input mzML files.": "输入mzML文件。",
    "Compound list in CSV format.": "CSV格式的化合物列表。",
    "Infolist in CSV format.": "CSV格式的信息列表。",
    "Mode for RMassBank pipeline.": "RMassBank流水线的模式。",
    "Choose the source if you want to generate settings file or you will supply with your own settings file.": "选择来源，如果您想生成设置文件，或者您将提供自己的设置文件。",
    "Setting of general parameters.": "一般参数的设置。",
    "Deviation (in minutes) allowed the for retention time.": "允许的保留时间偏差（以分钟为单位）。",
    "Systematic retention time shift.": "系统保留时间偏移。",
    "Which MassBank record version to use; version 2 is advised.": "使用哪个MassBank记录版本；建议使用版本2。",
    "Include reanalyzed peaks?": "包含重新分析的峰值？",
    "Annotate the spectra files with (putative) molecular formulas for fragments?": "为谱图文件注释（假定的）片段的分子公式？",
    "Include sp tags?": "包含sp标签？",
    "Shifts of the starting points for RMassBank accession numbers.": "RMassBank 访问号码起始点的偏移。",
    "pH ([M+H]+: accession numbers 1-14)": "pH ([M+H]+: 接入编号 1-14)",
    "pM ([M]+: 17-30)": "pM（磷酸化修饰位点：第17到30位）",
    "pNa ([M+Na]+: 33-46)": "pNa ([M+Na]+: 33-46)",
    "mH ([M-H]-: 51-64)": "mH ([M-H]-: 51-64)",
    "mFA ([M+FA]-: 67-80)": "mFA（[M+FA]-: 67-80）",
    "Noise and recalibration settings": "噪声和校准设置",
    "Exclusion width of electronic noise peaks (from unmatched peaks, prior to reanalysis).": "电子噪声峰的排除宽度（来自不匹配的峰，在重新分析之前）。",
    "Recalibration settings:": "重新校准设置：",
    "Recalibrate MS1:": "重新校准MS1:",
    "Window width to look for MS1 peaks to recalibrate (in ppm integer values):": "寻找用于校准MS1峰的窗口宽度（以ppm整数值为单位）：",
    "Define the multiplicity filtering level, default is 2 (peak occurs at least twice, set this to 1 if you want to turn this option off, set this to anything > 2 if you want harder filtering": "定义多重性过滤水平，默认值为2（峰值至少出现两次，如果您想关闭此选项，请将其设置为1；如果您想要更严格的过滤，请将其设置为大于2的任何值）",
    "Data filtering settings": "数据过滤设置",
    "ppmHighMass": "ppm高质量",
    "ppmLowMass": "ppm低质量",
    "massRangeDivision": "质量范围划分",
    "ppmFine": "ppmFine",
    "prelimCut": "预处理剪切",
    "prelimCutRatio": "预修剪比例",
    "fineCut": "精细切割",
    "fineCutRatio": "精细切割比例",
    "specOkLimit": "specOkLimit",
    "dbeMinLimit": "dbe最小限制",
    "satelliteMzLimit": "卫星mz限制",
    "satelliteIntLimit": "卫星内限制",
    "Degine raw MS retrieval settings": "定义原始质谱检索设置",
    "mzCoarse": "m/z 粗略",
    "fillPrecursorScan (FALSE for 'good' mzML files which have all the info needed)": "填充前体扫描（对于包含所有所需信息的“良好”mzML文件，设置为FALSE）",
    "Select how to treat unknown compound masses.": "选择如何处理未知化合物的质量。",
    "Settings file": "设置文件",
    "The task to be permed [pred=prediction, cid=compound identification].": "要执行的任务是[pred=预测，cid=化合物鉴定]。",
    "The type of description.": "描述类型。",
    " The number of steps for the prediction.": "预测步骤的数量。",
    "Mass tolerance for metabolite identification.": "代谢物鉴定的质量容差。",
    "Input CSV file.": "输入CSV文件。",
    "Input spectra dataset": "输入光谱数据集",
    "Ordered conversions": "有序转换",
    "Available conversions": "可用转换",
    "Other conversions": "其他转换",
    "Save the log file": "保存日志文件",
    "Choose input format:": "选择输入格式：",
    "General parameters": "通用参数",
    "Sigma t": "Sigma t",
    "Experimental Design": "实验设计",
    "Experiment name (no spaces).": "实验名称（无空格）。",
    "Genus species from which samples are derived.": "样品来源的属和种。",
    "Type of sample (e.g., serum, leaf).": "样品类型（例如，血清，叶片）。",
    "Your or your PI's name.": "您或您导师的姓名。",
    "Choose platform:": "选择平台：",
    "Model of GC instrument.": "气相色谱仪型号。",
    "Model of MS instrument.": "质谱仪的模型。",
    "Column description.": "列描述。",
    "Temperature of inlet.": "进气口温度。",
    "Temperature of GC to MS transfer line.": "GC到MS传输线的温度。",
    "Type of mass spectrometer:": "质谱仪类型：",
    "Ion mode:": "离子模式：",
    "Ionization:": "电离：",
    "Scan range used for acquisition.": "扫描范围用于采集。",
    "Time for each full scan spectrum (e.g. 0.2 seconds).": "每个完整扫描谱图的时间（例如0.2秒）。",
    "Derivitization:": "衍生化：",
    "Number of levels of energy acquired - 1 typically.": "通常获得的能量级数减1。",
    "Model of LC instrument.": "液相色谱仪器型号。",
    "Solvent A composition.": "溶剂A的组成。",
    "Solvent B composition.": "溶剂B的组成。",
    "Collision energy of acquisition of MS data.": "质谱数据采集的碰撞能量。",
    "Collision energy of acquisition for MSe/idMSMS data (when applicable).": "MSe/idMSMS 数据采集时的碰撞能量（如适用）。",
    "Gas used for collisional dissociation.": "用于碰撞解离的气体。",
    "Cone voltage used for acquisition.": "用于采集的锥孔电压。",
    "number of levels of energy in XCMS object data - 1 typically.": "在XCMS对象数据中，能量水平的数量通常为-1。",
    "Adducts to filter by": "要过滤的加合物",
    "Queries spectra": "查询谱图",
    "Symmetric": "对称",
    "Reference spectra": "参考光谱",
    "Scores array type": "得分数组类型",
    "Model JSON file": "模型 JSON 文件",
    "Model NPY file": "模型 NPY 文件",
    "Algorithm Parameters": "算法参数",
    "intensity_power": "强度功率",
    "Maximum share of new peaks": "最大新峰值占比",
    "Apply RI filtering": "应用 RI 过滤",
    "Choose input files:": "选择输入文件：",
    "Input metadata table": "输入元数据表",
    "Input metadata": "输入元数据",
    "Transpose feature table": "转置特征表",
    "Batch correction mode": "批量校正模式",
    "mz_power": "m/z 功率",
    "Test": "测试",
    "demultiplexed_seqs: SampleData[SequencesWithQuality | PairedEndSequencesWithQuality | JoinedSequencesWithQuality]": "去重序列：样本数据[带有质量的序列 | 带有质量的配对端序列 | 连接的带有质量的序列]",
    "trim_length: Int": "修剪长度：整数",
    "Click here for additional options": "点击此处查看更多选项",
    "left_trim_len: Int % Range(0, None)": "left_trim_len: 整数 % 范围(0, 无限制)",
    "sample_stats: Bool": "样本统计信息：布尔值",
    "mean_error: Float": "平均误差：浮点数",
    "indel_prob: Float": "indel_prob：浮点数",
    "indel_max: Int": "插入缺失最大值：整数",
    "min_reads: Int": "最小读取数：整数",
    "min_size: Int": "最小尺寸：整数",
    "jobs_to_start: Int": "jobs_to_start: 整数",
    "hashed_feature_ids: Bool": "哈希特征ID：布尔值",
    "metadata: Metadata": "元数据：元数据",
    "Metadata Source": "元数据来源",
    "state_column: Str": "状态列：字符串",
    "individual_id_column: Str": "个体ID列：Str",
    "metric: Str": "指标：字符串",
    "table: FeatureTable[RelativeFrequency]": "表格：特征表[相对频率]",
    "baseline: Float": "基线：浮点数",
    "sequences: FeatureData[Sequence]": "序列：特征数据[序列]",
    "n_threads: Int % Range(1, None)": "线程数：整数 % 范围(1, 无限制)",
    "parttree: Bool": "parttree: 布尔值",
    "alignment: FeatureData[AlignedSequence]": "比对：特征数据[比对序列]",
    "addfragments: Bool": "添加片段：布尔值",
    "max_gap_frequency: Float % Range(0, 1, inclusive_end=True)": "最大间隙频率：浮点数 % 范围(0, 1, 包含端点=True)",
    "min_conservation: Float % Range(0, 1, inclusive_end=True)": "最小保守度：浮点数 % 范围(0, 1, 包含端点=True)",
    "table: FeatureTable[Frequency]": "表：特征表[频率]",
    "pseudocount: Int": "伪计数：整数",
    "table: FeatureTable[Composition]": "表格：特征表[组成]",
    "metadata: MetadataColumn[Categorical]": "元数据：分类元数据列",
    "Column Name": "列名",
    "filter_missing: Bool": "filter_missing: 布尔值",
    "formula: Str": "公式：字符串",
    "prv_cut: Float": "prv_cut：浮点数",
    "lib_cut: Int": "lib_cut：整数",
    "reference_levels: List[Str]": "参考水平：列表[字符串]",
    "element: Str": "元素：字符串",
    "neg_lb: Bool": "neg_lb: 布尔值",
    "tol: Float": "容差：浮点数",
    "max_iter: Int": "最大迭代次数：整数",
    "conserve: Bool": "保守：布尔值",
    "alpha: Float": "alpha：浮点数",
    "data: FeatureData[DifferentialAbundance]": "数据：特征数据[差异丰度]",
    "seqs: MultiplexedPairedEndBarcodeInSequence": "序列：多路复用成对端条形码在序列中",
    "forward_barcodes: MetadataColumn[Categorical]": "前向条形码：元数据列[分类]",
    "reverse_barcodes: MetadataColumn[Categorical]": "反向条形码：元数据列[分类]",
    "error_rate: Float % Range(0, 1, inclusive_end=True)": "错误率：浮点数 % 范围(0, 1, 包含结束值=True)",
    "batch_size: Int % Range(0, None)": "批量大小：整数 % 范围(0, 无限制)",
    "minimum_length: Int % Range(1, None)": "最小长度：整数 % 范围(1, 无限制)",
    "mixed_orientation: Bool": "混合方向：布尔值",
    "cores: Int % Range(1, None)": "核心数：整数 % 范围(1, 无限制)",
    "seqs: MultiplexedSingleEndBarcodeInSequence": "序列：多路复用单端条形码在序列中",
    "barcodes: MetadataColumn[Categorical]": "条形码：元数据列[分类]",
    "demultiplexed_sequences: SampleData[PairedEndSequencesWithQuality]": "去多重序列：样本数据[配对末端序列及质量]",
    "adapter_f: List[Str]": "适配器_f：列表[字符串]",
    "front_f: List[Str]": "前置参数列表：字符串列表",
    "anywhere_f: List[Str]": "任意位置_f：列表[字符串]",
    "adapter_r: List[Str]": "适配器_r：列表[字符串]",
    "front_r: List[Str]": "front_r: 列表[字符串]",
    "anywhere_r: List[Str]": "anywhere_r: 列表[字符串]",
    "indels: Bool": "插入缺失：布尔值",
    "times: Int % Range(1, None)": "次数：整数 % 范围(1, 无限制)",
    "overlap: Int % Range(1, None)": "重叠：整数 % 范围(1, 无限制)",
    "match_read_wildcards: Bool": "匹配读取通配符：布尔值",
    "match_adapter_wildcards: Bool": "匹配适配器通配符：布尔值",
    "discard_untrimmed: Bool": "丢弃未修剪的: 布尔值",
    "max_expected_errors: Float % Range(0, None)": "最大预期错误数：浮点数 % 范围(0, 无限制)",
    "max_n: Float % Range(0, None)": "最大值：浮点数 % 范围(0, 无限制)",
    "quality_cutoff_5end: Int % Range(0, None)": "质量截断_5端：整数 % 范围(0, 无限制)",
    "quality_cutoff_3end: Int % Range(0, None)": "质量截断_3端：整数 % 范围(0, 无限制)",
    "quality_base: Int % Range(0, None)": "质量碱基：整数 % 范围(0, 无限制)",
    "demultiplexed_sequences: SampleData[SequencesWithQuality]": "去多重序列：样本数据[带有质量的序列]",
    "adapter: List[Str]": "适配器：列表[字符串]",
    "front: List[Str]": "前端：列表[字符串]",
    "anywhere: List[Str]": "任何地方：列表[字符串]",
    "demultiplexed_seqs: SampleData[SequencesWithQuality]": "去多重序列：样本数据[带有质量的序列]",
    "front: Str": "前端：字符串",
    "adapter: Str": "适配器：字符串",
    "max_mismatch: Int": "最大不匹配数：整数",
    "trunc_len: Int": "截断长度：整数",
    "trim_left: Int": "左修剪: 整数",
    "max_ee: Float": "最大期望值：浮点数",
    "trunc_q: Int": "截断q：整数",
    "min_len: Int": "最小长度：整数",
    "max_len: Int": "最大长度：整数",
    "min_fold_parent_over_abundance: Float": "最小折叠变化倍数父丰度：浮点数",
    "allow_one_off: Bool": "允许单个不匹配：布尔值",
    "n_threads: Int": "n_threads: 整数",
    "n_reads_learn: Int": "n_reads_learn：整数",
    "demultiplexed_seqs: SampleData[PairedEndSequencesWithQuality]": "去多重序列：样本数据[配对端序列和质量]",
    "trunc_len_f: Int": "trunc_len_f: 整数",
    "trunc_len_r: Int": "trunc_len_r：整数",
    "trim_left_f: Int": "左修剪碱基数：整数",
    "trim_left_r: Int": "trim_left_r: 整数",
    "max_ee_f: Float": "最大期望误差因子：浮点数",
    "max_ee_r: Float": "最大期望值比率：浮点数",
    "min_overlap: Int % Range(4, None)": "最小重叠：整数 % 范围(4, 无限制)",
    "demultiplexed_seqs: SampleData[SequencesWithQuality | PairedEndSequencesWithQuality]": "去多重序列：样本数据[带有质量的序列 | 带有质量的成对末端序列]",
    "reference_seqs: FeatureData[Sequence]": "参考序列：特征数据[序列]",
    "deblur_stats: DeblurStats": "deblur_stats：去模糊统计",
    "seqs: EMPPairedEndSequences": "序列：EMP成对末端序列",
    "golay_error_correction: Bool": "高洛错误校正：布尔值",
    "rev_comp_barcodes: Bool": "反转互补条形码：布尔值",
    "rev_comp_mapping_barcodes: Bool": "rev_comp_mapping_barcodes：布尔值",
    "ignore_description_mismatch: Bool": "忽略描述不匹配：布尔值",
    "seqs: RawSequences | EMPSingleEndSequences | EMPPairedEndSequences": "序列：原始序列 | EMP单端序列 | EMP双端序列",
    "demux: SampleData[SequencesWithQuality¹ | PairedEndSequencesWithQuality² | JoinedSequencesWithQuality³]": "去重：样本数据[序列与质量¹ | 配对末端序列与质量² | 连接序列与质量³]",
    "where: Str": "位置：字符串",
    "exclude_ids: Bool": "排除ID：布尔值",
    "sequences: SampleData[PairedEndSequencesWithQuality]": "序列：样本数据[配对端序列和质量]",
    "fraction: Float % Range(0, 1, inclusive_start=False)": "分数：浮点数 % 范围(0, 1, 包含起始端=False)",
    "sequences: SampleData[SequencesWithQuality | PairedEndSequencesWithQuality]": "序列：样本数据[含质量的序列 | 含质量的配对端序列]",
    "data: SampleData[SequencesWithQuality | PairedEndSequencesWithQuality | JoinedSequencesWithQuality]": "数据：样本数据[带有质量的序列 | 带有质量的配对末端序列 | 连接的带有质量的序列]",
    "n: Int": "n：整数",
    "distance_matrix: DistanceMatrix": "距离矩阵：DistanceMatrix",
    "permutations: Int % Range(1, None)": "排列：整数 % 范围(1, 无限制)",
    "n_jobs: Int % Range(1, None)": "n_jobs: 整数 % 范围(1, 无限制)",
    "table: FeatureTable[Frequency | RelativeFrequency | PresenceAbsence]": "表格：特征表[频率 | 相对频率 | 出现/缺失]",
    "alpha_diversity: SampleData[AlphaDiversity]": "alpha多样性：样本数据[Alpha多样性]",
    "intersect_ids: Bool": "交集_IDs: 布尔值",
    "phylogeny: Phylogeny[Rooted]": "系统发育学：系统发育学[有根的]",
    "max_depth: Int % Range(1, None)": "最大深度：整数 % 范围(1, 无限制)",
    "min_depth: Int % Range(1, None)": "最小深度：整数 % 范围(1, 无限制)",
    "steps: Int % Range(2, None)": "步骤：整数 % 范围(2, 无限制)",
    "iterations: Int % Range(1, None)": "迭代次数：整数 % 范围(1, 无限制)",
    "pseudocount: Int % Range(1, None)": "伪计数：整数 % 范围(1, 无限制)",
    "metadata: MetadataColumn[Numeric]": "元数据：数值元数据列",
    "permutations: Int % Range(0, None)": "排列：整数 % 范围(0, 无限制)",
    "label1: Str": "标签1：字符串",
    "label2: Str": "label2: 字符串",
    "pairwise: Bool": "成对比较：布尔值",
    "permutations: Int": "排列：整数",
    "threads: Int % Range(1, None)": "线程数：整数 % 范围(1, 无限制)",
    "variance_adjusted: Bool": "方差校正：布尔值",
    "alpha: Float % Range(0, 1, inclusive_end=True)": "alpha：浮点数 % 范围(0, 1, 包含结束值=True)",
    "bypass_tips: Bool": "bypass_tips：布尔值",
    "sampling_depth: Int % Range(1, None)": "采样深度：整数 % 范围(1, 无限制)",
    "iterations: Int % Range(2, None)": "迭代次数：整数 % 范围(2, 无限制)",
    "with_replacement: Bool": "有放回：布尔值",
    "n_jobs_or_threads: Int % Range(1, None)": "n_jobs_or_threads：整数 % 范围(1, 无限制)",
    "dm1: DistanceMatrix": "dm1：距离矩阵",
    "dm2: DistanceMatrix": "dm2: 距离矩阵",
    "number_of_dimensions: Int % Range(1, None)": "维度数量：整数 % 范围(1, 无限制)",
    "pcoa: PCoAResults": "PCoA结果：PCoA分析结果",
    "features: FeatureTable[RelativeFrequency]": "特征：特征表[相对频率]",
    "reference: PCoAResults": "参考：PCoA结果",
    "other: PCoAResults": "其他：PCoA结果",
    "dimensions: Int % Range(1, None)": "维度：整数 % 范围(1, 无限制)",
    "number_of_dimensions: Int % Range(2, None)": "维度数量：整数 % 范围(2, 无限制)",
    "perplexity: Float % Range(1, None)": "困惑度：浮点数 % 范围(1, 无限制)",
    "n_iter: Int % Range(1, None)": "n_iter: 整数 % 范围(1, 无限制)",
    "learning_rate: Float % Range(10.0, None)": "学习率：浮点数 % 范围(10.0, 无限制)",
    "early_exaggeration: Float % Range(0, None)": "early_exaggeration：浮点数 % 范围(0, 无限制)",
    "random_state: Int": "随机状态：整数",
    "n_neighbors: Int % Range(1, None)": "邻居数：整数 % 范围(1, 无限制)",
    "min_dist: Float % Range(0, None)": "最小距离：浮点数 % 范围(0, 无限制)",
    "tables: List[FeatureTable[Frequency]]": "表格：特征频率表的列表",
    "phylogenies: List[Phylogeny[Rooted]]": "系统发育树：List[Phylogeny[有根]]",
    "weights: List[Float]": "权重：列表[浮点数]",
    "element: Float": "元素：浮点数",
    "table: FeatureTable[Frequency | RelativeFrequency]": "表：特征表[频率 | 相对频率]",
    "drop_undefined_samples: Bool": "删除未定义样本：布尔值",
    "biplot: PCoAResults % Properties('biplot')": "双标图：PCoA结果 % 属性('双标图')",
    "sample_metadata: Metadata": "样本元数据：元数据",
    "feature_metadata: Metadata": "特征元数据：元数据",
    "ignore_missing_samples: Bool": "忽略缺失样本：布尔值",
    "invert: Bool": "反转：布尔值",
    "number_of_features: Int % Range(1, None)": "特征数量：整数 % 范围(1, 无限制)",
    "custom_axes: List[Str]": "自定义轴：列表[字符串]",
    "ignore_pcoa_features: Bool": "忽略_PCOA_特征：布尔值",
    "reference_pcoa: PCoAResults": "参考主坐标分析：PCoA结果",
    "other_pcoa: PCoAResults": "其他PCoA：PCoA结果",
    "m2_stats: ProcrustesStatistics": "m2_stats：普氏方差统计",
    "query: FeatureData[Sequence]": "查询：特征数据[序列]",
    "reference_reads: FeatureData[Sequence]": "参考序列：特征数据[序列]",
    "maxaccepts: Int % Range(1, None)": "最大接受数：整数 % 范围(1, 无限制)",
    "perc_identity: Float % Range(0.0, 1.0, inclusive_end=True)": "序列相似性百分比：浮点数 % 范围(0.0, 1.0, 包含端点)",
    "query_cov: Float % Range(0.0, 1.0, inclusive_end=True)": "查询覆盖度：浮点数 % 范围(0.0, 1.0, 包含端点=True)",
    "evalue: Float": "e值：浮点数",
    "output_no_hits: Bool": "输出无命中结果：布尔值",
    "reference_taxonomy: FeatureData[Taxonomy]": "参考分类学：特征数据[分类学]",
    "min_consensus: Float % Range(0.5, 1.0, inclusive_start=False, inclusive_end=True)": "最小共识度：浮点数 % 范围(0.5, 1.0, 包括起始端False, 包括结束端True)",
    "unassignable_label: Str": "无法分配的标签：字符串",
    "search_exact: Bool": "搜索精确：布尔值",
    "top_hits_only: Bool": "仅显示最佳命中：布尔值",
    "maxhits: Int % Range(1, None)": "最大命中数：整数 % 范围(1, 无限制)",
    "maxrejects: Int % Range(1, None)": "最大拒绝数：整数 % 范围(1, 无限制)",
    "weak_id: Float % Range(0.0, 1.0, inclusive_end=True)": "weak_id: 浮点数 % 范围(0.0, 1.0, 包含端点=True)",
    "classifier: TaxonomicClassifier": "分类器：分类分类器",
    "reads_per_batch: Int % Range(1, None)": "reads_per_batch: 整数 % 范围(1, 无限制)",
    "confidence: Float % Range(0, 1, inclusive_end=True)": "置信度：浮点数 % 范围(0, 1, 包含端点=True)",
    "prefilter: Bool": "预过滤：布尔值",
    "sample_size: Int % Range(1, None)": "样本大小：整数 % 范围(1, 无限制)",
    "randseed: Int % Range(0, None)": "随机种子数：整数 % 范围(0, 无限制)",
    "reads: FeatureData[Sequence]": "序列特征数据：特征数据[序列]",
    "n_jobs: Int": "n_jobs: 整数",
    "pre_dispatch: Str": "预处理调度：字符串",
    "f_primer: Str": "f_primer：字符串",
    "r_primer: Str": "r_primer：字符串",
    "trim_right: Int": "trim_right: 整数",
    "identity: Float": "相似性：浮点数",
    "min_length: Int % Range(0, None)": "最小长度：整数 % 范围(0, 无限制)",
    "max_length: Int % Range(0, None)": "最大长度：整数 % 范围(0, 无限制)",
    "batch_size: Int % Range(1, None)": "批量大小：整数 % 范围(1, 无限制)",
    "search_results: FeatureData[BLAST6]": "搜索结果：特征数据[BLAST6]",
    "class_weight: FeatureTable[RelativeFrequency]": "类别权重：相对频率特征表",
    "classify__alpha: Float": "分类__alpha：浮点数",
    "classify__chunk_size: Int": "分类_块大小：整数",
    "classify__class_prior: Str": "分类__类先验：字符串",
    "classify__fit_prior: Bool": "分类_拟合先验：布尔值",
    "feat_ext__alternate_sign: Bool": "特征扩展_交替符号：布尔值",
    "feat_ext__analyzer: Str": "特征扩展分析器：字符串",
    "feat_ext__binary: Bool": "特征扩展二进制：布尔值",
    "feat_ext__decode_error: Str": "特征扩展_解码错误：字符串",
    "feat_ext__encoding: Str": "特征扩展编码：字符串",
    "feat_ext__input: Str": "特征扩展输入：字符串",
    "feat_ext__lowercase: Bool": "feat_ext__lowercase：布尔值",
    "feat_ext__n_features: Int": "特征扩展__n特征：整数",
    "feat_ext__ngram_range: Str": "特征扩展__ngram_range：字符串",
    "feat_ext__norm: Str": "特征扩展的标准化字符串",
    "feat_ext__preprocessor: Str": "特征扩展器：字符串",
    "feat_ext__stop_words: Str": "特征扩展_停止词：字符串",
    "feat_ext__strip_accents: Str": "特征扩展 - 去除重音符号：字符串",
    "feat_ext__token_pattern: Str": "特征扩展_标记模式：字符串",
    "feat_ext__tokenizer: Str": "特征扩展_分词器：字符串",
    "verbose: Bool": "详细模式：布尔值",
    "classifier_specification: Str": "分类器规范：Str",
    "min_fraction: Float % Range(0.0, 1.0, inclusive_start=False)": "最小比例：浮点数 % 范围(0.0, 1.0, 包含起始端点=False)",
    "max_fraction: Float % Range(0.0, 1.0, inclusive_end=True)": "最大比例：浮点数 % 范围(0.0, 1.0, 包含端点=True)",
    "min_frequency: Int": "最小频率：整数",
    "max_frequency: Int": "最大频率：整数",
    "min_samples: Int": "最小样本数：整数",
    "max_samples: Int": "最大样本数：整数",
    "filter_empty_samples: Bool": "过滤空样本：布尔值",
    "table: FeatureTable[Frequency¹ | RelativeFrequency² | PresenceAbsence³ | Composition⁴]": "表格：特征表[频率¹ | 相对频率² | 出现/缺失³ | 组成⁴]",
    "abundance: Float % Range(0, 1)": "丰度：浮点数 % 范围(0, 1)",
    "prevalence: Float % Range(0, 1)": "流行率：浮点数 % 范围(0, 1)",
    "min_features: Int": "最小特征数：整数",
    "max_features: Int": "最大特征数：整数",
    "filter_empty_features: Bool": "过滤空特征：布尔值",
    "data: FeatureData[Sequence¹ | AlignedSequence²]": "数据：特征数据[序列¹ | 对齐序列²]  1. 特征数据（FeatureData）：在生物信息学中，特征数据通常指的是包含生物特征信息的数据集，例如基因表达数据、蛋白质序列数据等。 2. 序列（Sequence）：在生物信息学中，序列通常指的是生物分子的序列，如DNA、RNA或蛋白质序列。 3. 对齐序列（AlignedSequence）：指的是通过序列比对得到的，具有相同或相似特征的序列集合。  在Galaxy平台上，FeatureData[Sequence | AlignedSequence] 通常用于表示处理序列数据的工具或分析流程，其中Sequence表示原始序列数据，而AlignedSequence表示经过比对处理的序列数据。",
    "sample_metadata: MetadataColumn[Categorical]": "样本元数据：分类元数据列",
    "feature_metadata: MetadataColumn[Categorical]": "特征元数据：分类元数据列",
    "normalize: Bool": "标准化：布尔值",
    "title: Str": "标题：Str",
    "tables: List[FeatureTable[Frequency]¹ | FeatureTable[RelativeFrequency]²]": "表格：特征表列表[频率¹ | 相对频率²]",
    "data: List[FeatureData[Sequence]]": "数据：列表[特征数据[序列]]",
    "data: List[FeatureData[Taxonomy]]": "数据：列表[特征数据[分类学]]",
    "strict: Bool": "严格：布尔值",
    "subsampling_depth: Int % Range(1, None)": "子采样深度：整数 % 范围(1, 无限制)",
    "data: FeatureData[Sequence | AlignedSequence]": "数据：特征数据[序列 | 对齐序列]",
    "representative_sequences: FeatureData[Sequence]": "代表性序列：FeatureData[序列]",
    "tree: Phylogeny[Rooted]": "树：系统发育[有根的]",
    "reference_database: SeppReferenceDatabase": "参考数据库：Sepp参考数据库",
    "alignment_subset_size: Int": "对齐子集大小：整数",
    "placement_subset_size: Int": "放置子集大小：整数",
    "threads: Int": "线程数：整数",
    "debug: Bool": "调试：布尔值",
    "input_table: FeatureTable[Frequency]": "输入表：特征表[频率]",
    "input_tree: Phylogeny[Rooted]": "输入树：系统发育树[有根的]",
    "pseudocount: Float": "伪计数：浮点数",
    "tree: Hierarchy": "树：层次结构",
    "ndim: Int": "维度：整数",
    "table: FeatureTable[Frequency | RelativeFrequency | Composition]": "表格：特征表[频率 | 相对频率 | 成分]",
    "gradient: MetadataColumn[Numeric]": "梯度：数值型元数据列",
    "weighted: Bool": "加权：布尔值",
    "table: FeatureTable[Frequency | Composition]": "表：特征表[频率 | 组成]",
    "differential: FeatureData[Differential]": "差异性：特征数据[差异性]",
    "top_k_var: Int": "top_k_var: 整数",
    "clades: List[Str]": "类群：列表[字符串]",
    "cv: Int % Range(1, None)": "交叉验证次数：整数 % 范围(1, 无限制)",
    "n_estimators: Int % Range(1, None)": "n_estimators：整数 % 范围(1, 无限制)",
    "parameter_tuning: Bool": "参数调优：布尔值",
    "importance_threshold: Float % Range(0, None, inclusive_start=False)": "重要性阈值：浮点数 % 范围(0, 无限制, 包含开始端点=False)",
    "feature_count: Int % Range(1, None)": "特征计数：整数 % 范围(1, 无限制)",
    "group_columns: Str": "分组列：字符串",
    "random_effects: Str": "随机效应：字符串",
    "lowess: Bool": "lowess：布尔值",
    "ci: Float % Range(0, 100)": "置信度：浮点数 % 范围(0, 100)",
    "group_by: Str": "分组依据：字符串",
    "control: Str": "控制：字符串",
    "test_size: Float % Range(0.0, 1.0)": "测试集大小：浮点数 % 范围(0.0, 1.0)",
    "step: Float % Range(0.0, 1.0, inclusive_start=False)": "步骤：浮点数 % 范围（0.0, 1.0, inclusive_start=False）",
    "optimize_feature_selection: Bool": "优化特征选择：布尔值",
    "stratify: Bool": "分层: 布尔值",
    "feature_count: Int % Range(0, None)": "特征计数：整数 % 范围(0, 无限制)",
    "state_1: Str": "状态_1：字符串",
    "state_2: Str": "状态2：字符串",
    "group_column: Str": "组列：字符串",
    "parametric: Bool": "参数化：布尔值",
    "importances: FeatureData[Importance]": "重要性：特征数据[重要性]",
    "default_group_column: Str": "默认组列：字符串",
    "default_metric: Str": "默认指标：字符串",
    "n_columns: Int": "列数：整数",
    "column_name_prefix: Str": "列名前缀：Str",
    "column_value_prefix: Str": "列值前缀：Str",
    "input: Metadata": "输出：元数据",
    "page_size: Int": "页面大小：整数",
    "mask_max_gap_frequency: Float % Range(0, 1, inclusive_end=True)": "掩码最大间隙频率：浮点数 % 范围(0, 1, 包括端点=True)  解释： - \"mask_max_gap_frequency\" 是一个参数，表示最大间隙频率的掩码，保持原文不变。 - \"Float\" 表示该参数的数据类型是浮点数。 - \"% Range\" 表示参数的取值范围。 - \"(0, 1, inclusive_end=True)\" 表示参数的取值范围是0到1（包括1），其中 \"inclusive_end=True\" 表示包括端点1。",
    "mask_min_conservation: Float % Range(0, 1, inclusive_end=True)": "掩蔽最小保守性：浮点数 % 范围(0, 1, 包含端点=True)",
    "fast: Bool": "快速：布尔值",
    "alrt: Int % Range(1000, None)": "alrt: 整数 % 范围(1000, 无限制)",
    "seed: Int": "种子：整数",
    "stop_iter: Int % Range(1, None)": "停止迭代次数：整数 % 范围(1, 无限制)",
    "perturb_nni_strength: Float % Range(0.01, 1.0)": "扰动NNI强度：浮点数 % 范围(0.01, 1.0)",
    "table: FeatureTable[Frequency¹ | RelativeFrequency² | PresenceAbsence³]": "表：特征表[频率¹ | 相对频率² | 出现/缺失³]",
    "tree: Phylogeny[Rooted | Unrooted]": "树：系统发育[有根 | 无根]",
    "tree: Phylogeny[Rooted¹ | Unrooted²]": "树：系统发育[有根¹ | 无根²]",
    "table: FeatureTable[Frequency | RelativeFrequency | PresenceAbsence | Composition]": "表格：特征表[频率 | 相对频率 | 出现/缺失 | 组成]",
    "n_cores: Int % Range(1, None)": "核心数：整数 % 范围(1, 无限制)",
    "n_cores_max: Int % Range(2, None)": "最大核心数：整数 % 范围(2, 无上限)",
    "n_runs: Int % Range(1, None)": "n_runs：整数 % 范围(1, 无限制)",
    "n_init_pars_trees: Int % Range(1, None)": "n_init_pars_trees：整数 % 范围(1, 无限制)",
    "n_top_init_trees: Int % Range(1, None)": "n_top_init_trees：整数 % 范围(1, 无限制)",
    "n_best_retain_trees: Int % Range(1, None)": "保留最佳树的数量：整数 % 范围(1, 无限制)",
    "spr_radius: Int % Range(1, None)": "spr_radius: 整数 % 范围(1, 无限制)",
    "allnni: Bool": "allnni：布尔值",
    "abayes: Bool": "abayes：布尔值",
    "lbp: Int % Range(1000, None)": "局部二值模式：整数 % 范围(1000, 无限制)",
    "safe: Bool": "安全：布尔值",
    "bootstrap_replicates: Int % Range(1000, None)": "bootstrap_replicates: 整数 % 范围(1000, 无限制)",
    "n_max_ufboot_iter: Int % Range(1, None)": "n_max_ufboot_iter：整数 % 范围(1, 无限制)",
    "n_ufboot_steps: Int % Range(1, None)": "n_ufboot_steps: 整数 % 范围(1, 无限制)",
    "min_cor_ufboot: Float % Range(0.51, 0.99)": "最小相关性_ufboot：浮点数 % 范围(0.51, 0.99)",
    "ep_break_ufboot: Float % Range(0.01, 0.99)": "ep_break_ufboot：浮点数 % 范围(0.01, 0.99)",
    "bnni: Bool": "bnni: 布尔值",
    "tree: Phylogeny[Unrooted]": "树：系统发育[无根]",
    "n_searches: Int % Range(1, None)": "n_searches: 整数 % 范围(1, 无限制)",
    "rapid_bootstrap_seed: Int": "快速引导种子：整数",
    "bootstrap_replicates: Int % Range(10, None)": "bootstrap_replicates：整数 % 范围(10, 无限制)",
    "trees: List[Phylogeny[Rooted | Unrooted]]": "树：列表[系统发育[有根 | 无根]]",
    "labels: List[Str]": "标签：列表[字符串]",
    "expected_features: FeatureTable[RelativeFrequency]": "期望特征：特征表[相对频率]",
    "observed_features: FeatureTable[RelativeFrequency]": "观察到的特征：特征表[相对频率]",
    "depth: Int": "深度：整数",
    "plot_tar: Bool": "plot_tar: 布尔值",
    "plot_tdr: Bool": "plot_tdr：布尔值",
    "plot_r_value: Bool": "plot_r_value：布尔值",
    "plot_r_squared: Bool": "绘制R平方值：布尔值",
    "plot_bray_curtis: Bool": "plot_bray_curtis: 布尔值",
    "plot_jaccard: Bool": "plot_jaccard：布尔值",
    "plot_observed_features: Bool": "绘制观测特征：布尔值",
    "plot_observed_features_ratio: Bool": "绘制观察到的特征比例：布尔值",
    "query_sequences: FeatureData[Sequence]": "查询序列：特征数据[序列]",
    "reference_sequences: FeatureData[Sequence]": "参考序列：特征数据[序列]",
    "show_alignments: Bool": "显示对齐：布尔值",
    "expected_taxa: FeatureData[Taxonomy]": "预期分类群：特征数据[分类学]",
    "observed_taxa: FeatureData[Taxonomy]": "观察到的类群：特征数据[分类学]",
    "feature_table: FeatureTable[RelativeFrequency]": "特征表：相对频率特征表",
    "require_exp_ids: Bool": "需要表达ID：布尔值",
    "require_obs_ids: Bool": "需要观察对象ID: 布尔值",
    "sample_id: Str": "样本ID：字符串",
    "perc_query_aligned: Float": "查询序列比对比例：浮点数",
    "left_justify: Bool % Choices(False)¹ | Bool²": "左对齐：布尔值 % 选择(False)¹ | 布尔值²",
    "demultiplexed_sequences: SampleData[SequencesWithQuality¹ | PairedEndSequencesWithQuality²]": "去多重序列：样本数据[带有质量的序列¹ | 带有质量的配对端序列²]",
    "database: Bowtie2Index": "数据库：Bowtie2索引",
    "ref_gap_open_penalty: Int % Range(1, None)": "参考间隙开放惩罚：整数 % 范围(1, 无限制)",
    "ref_gap_ext_penalty: Int % Range(1, None)": "ref_gap_ext_penalty: 整数 % 范围(1, 无限制)",
    "exclude_seqs: Bool": "排除序列：布尔值",
    "demux: SampleData[SequencesWithQuality | PairedEndSequencesWithQuality]¹ | SampleData[JoinedSequencesWithQuality]²": "去重：样本数据[带有质量的序列 | 带有质量的配对末端序列]¹ | 样本数据[合并的带有质量的序列]²",
    "min_quality: Int": "最小质量：整数",
    "quality_window: Int": "质量窗口：整数",
    "min_length_fraction: Float": "最小长度分数：浮点数",
    "max_ambiguous: Int": "最大模糊性：整数",
    "k: Int": "k: 整数",
    "predictions: SampleData[ClassifierPredictions]": "预测：样本数据[分类器预测]",
    "truth: MetadataColumn[Categorical]": "真实值：元数据列[分类]",
    "probabilities: SampleData[Probabilities]": "概率：样本数据[概率]",
    "vmin: Float": "最小值：浮点数",
    "vmax: Float": "最大值：浮点数",
    "importance: FeatureData[Importance]": "重要性：特征数据[重要性]",
    "importance_threshold: Float % Range(0, None)": "重要性阈值：浮点数 % 范围(0, 无限制)",
    "group_samples: Bool": "group_samples: 布尔值",
    "drop_all_unique: Bool": "drop_all_unique：布尔值",
    "sample_estimator: SampleEstimator[Classifier]": "样本估计器：样本估计器[分类器]",
    "sample_estimator: SampleEstimator[Regressor]": "样本估计器：样本估计器[回归器]",
    "predictions: SampleData[RegressorPredictions]": "预测：样本数据[回归预测]",
    "truth: MetadataColumn[Numeric]": "真实值：元数据列[数值型]",
    "table: FeatureTable[Frequency¹ | RelativeFrequency² | PresenceAbsence³ | Balance⁴ | PercentileNormalized⁵ | Design⁶]": "表：特征表[频率¹ | 相对频率² | 出现/缺失³ | 平衡⁴ | 百分位数标准化⁵ | 设计⁶]",
    "metadata: MetadataColumn[Numeric | Categorical]": "元数据：元数据列[数值 | 分类]",
    "sample_estimator: SampleEstimator[Classifier | Regressor]": "样本估计器：样本估计器[分类器 | 回归器]",
    "table: FeatureTable[Frequency | PresenceAbsence]": "表格：特征表[频率 | 有无]",
    "taxonomy: FeatureData[Taxonomy]": "分类学：特征数据[分类学]",
    "level_delimiter: Str": "级别分隔符：Str",
    "level: Int": "级别：整数",
    "include: Str": "包含：字符串",
    "exclude: Str": "排除：字符串",
    "query_delimiter: Str": "查询分隔符：字符串",
    "perc_identity: Float % Range(0, 1, inclusive_start=False, inclusive_end=True)": "百分比一致性：浮点数 % 范围(0, 1, 包含起始端=False, 包含结束端=True)",
    "threads: Int % Range(0, 256, inclusive_end=True)": "线程数：整数 % 范围(0, 256, 包含结束值=True)",
    "sequences: SampleData[Sequences] | SampleData[SequencesWithQuality] | SampleData[JoinedSequencesWithQuality]": "序列：样本数据[序列] | 样本数据[含质量分数的序列] | 样本数据[连接的含质量分数的序列]",
    "derep_prefix: Bool": "去重前缀：布尔值",
    "min_seq_length: Int % Range(1, None)": "最小序列长度：整数 % 范围(1, 无限制)",
    "min_unique_size: Int % Range(1, None)": "最小唯一大小：整数 % 范围(1, 无限制)",
    "truncqual: Int % Range(0, None)": "截断质量：整数 % 范围(0, 无限制)",
    "minlen: Int % Range(0, None)": "最小长度：整数 % 范围(0, 无限制)",
    "maxns: Int % Range(0, None)": "maxns: 整数 % 范围(0, 无限制)",
    "allowmergestagger: Bool": "允许合并交错：布尔值",
    "minovlen: Int % Range(0, None)": "最小重叠长度：整数 % 范围(0, 无限制)",
    "maxdiffs: Int % Range(0, None)": "最大差异数：整数 % 范围(0, 无限制)",
    "minmergelen: Int % Range(0, None)": "最小合并长度：整数 % 范围(0, 无限制)",
    "maxmergelen: Int % Range(0, None)": "最大合并长度：整数 % 范围(0, 无限制)",
    "maxee: Float % Range(0.0, None)": "最大值误差：浮点数 % 范围(0.0, 无限制)",
    "threads: Int % Range(0, 8, inclusive_end=True)": "线程数：整数 % 范围(0, 8, 包含结束值=True)",
    "dn: Float % Range(0.0, None)": "dn: 浮点数 % 范围(0.0, 无限制)",
    "mindiffs: Int % Range(1, None)": "mindiffs: 整数 % 范围(1, 无限制)",
    "mindiv: Float % Range(0.0, None)": "最小遗传距离：浮点数 % 范围(0.0, 无上限)",
    "minh: Float % Range(0.0, 1.0, inclusive_end=True)": "minh：浮点数 % 范围（0.0, 1.0, inclusive_end=True）",
    "xn: Float % Range(1.0, None, inclusive_start=False)": "xn: 浮点数 % 范围(1.0, 无限制, inclusive_start=False)",
    "input: The path to the artifact you want to export": "输出：您要导出的工件的路径",
    "The type of your input qza is:": "您输入的qza文件类型为：",
    "The current QIIME 2 format is:": "当前的QIIME 2格式是：",
    "To change the format, select the type indicated above:": "要更改格式，请选择上面指示的类型：",
    "QIIME 2 file format to convert to:": "QIIME 2 文件格式转换为：",
    "Type of data to import:": "要导入的数据类型：",
    "QIIME 2 file format to import from:": "QIIME 2 文件格式导入：",
    "Import idx1": "导入 idx1",
    "Import idx2": "导入 idx2",
    "Import ref3": "导入参考序列3",
    "Import ref4": "导入参考序列4",
    "Import rev1": "导入 rev1",
    "Import rev2": "导入 rev2",
    "Import forward": "导入正向",
    "Import reverse": "导入反向",
    "Import barcodes": "导入条形码",
    "Import sequences": "导入序列",
    "Import data_slices": "导入数据切片",
    "Select a mechanism": "选择一个机制",
    "Append an extension?": "添加一个扩展？",
    "Extension to append (e.g. '.fastq.gz')": "扩展名以追加（例如 '.fastq.gz'）",
    "Add Elements": "添加元素",
    "Import nutrition_facts": "导入营养事实",
    "Import left_rna_sequences": "导入左侧RNA序列",
    "Import right_rna_sequences": "导入右侧RNA序列",
    "Import left_dna_sequences": "导入左侧DNA序列",
    "Import right_dna_sequences": "导入右DNA序列",
    "Import forward_sequences": "导入前向序列",
    "Import reverse_sequences": "导入反向序列",
    "Import quality": "导入质量",
    "Import manifest": "导入清单",
    "Import metadata": "导入元数据",
    "Import version_info": "导入 version_info",
    "Import sklearn_pipeline": "导入 sklearn_pipeline",
    "Import alignment": "导入比对",
    "Import phylogeny": "导入系统发育树",
    "Import raxml_info": "导入 raxml_info",
    "Summarize taxonomic representation for": "总结分类学表示",
    "Draw phylogram for": "绘制系统发育树",
    "select font size": "选择字体大小",
    "maximum number of leaves": "最大叶子数",
    "First Column": "第一列",
    "Second Column": "第二列",
    "D value": "D值",
    "correction method": "校正方法",
    "for taxonomy dataset": "用于分类群数据集",
    "require the lowest rank to be at least": "要求最低排名至少为",
    "Show taxonomic representation for": "显示分类学表示",
    "GIs column": "基因信息列",
    "Name column": "名称列",
    "Find diagnostic hits in": "在中查找诊断命中",
    "Select column with sequence id": "选择带有序列标识的列",
    "select taxonomic ranks": "选择分类等级",
    "Ensembl GTF file": "Ensembl GTF文件",
    "Feature type for which to derive annotation": "要派生注释的特征类型",
    "Field to place first in output table": "输出表格中首先放置的字段",
    "Suppress header line in output?": "在输出中隐藏标题行？",
    "Comma-separated list of field names to extract from the GTF (default: use all fields)": "逗号分隔的字段名称列表，用于从GTF文件中提取（默认：使用所有字段）",
    "Append version to transcript identifiers?": "在转录本标识符后附加版本号？",
    "Column to be used to fill empty values in other fields": "用于填充其他字段空值的列",
    "Flag mitochondrial features?": "标记线粒体特征？",
    "Comma-separated list of possible mitochondrial chromosome names (case insensitive)": "逗号分隔的可能的线粒体染色体名称列表（不区分大小写）",
    "Comma-separated list of possible mitochondrial biotypes (case insensitive)": "逗号分隔的可能的线粒体生物类型列表（不区分大小写）",
    "Provide a cDNA file for extracting annotations and/ or possible filtering?": "提供一个cDNA文件以提取注释和/或可能的过滤？",
    "FASTA-format cDNA/ transcript file": "FASTA格式的cDNA/转录本文件",
    "Annotation field in GTF to match with sequences.": "在GTF文件中，用于与序列匹配的注释字段。",
    "Parse the FASTA headers for annotation info?": "解析FASTA头信息以获取注释信息？",
    "Filter the cDNA file to match the annotations?": "将cDNA文件过滤以匹配注释。",
    "Input MTX-format matrix?": "输入MTX格式矩阵？",
    "Matrix-market format matrix file, with cells by column (overrides --barcode-frequencies if supplied)": "矩阵市场格式矩阵文件，按列单元格（如果提供，则覆盖 --barcode-frequencies）",
    "For use with --mtx-matrix: force interpretation of matrix to assume cells are by row, rather than by column (default)": "用于与--mtx-matrix参数一起使用：强制解释矩阵，假设单元格是按行排列的，而不是按列（默认）",
    "A two-column tab-delimited file, with barcodes in the first column and frequencies in the second": "一个两列制表符分隔的文件，第一列是条形码，第二列是频率。",
    "Label to place in plot title": "图中标题位置的标签",
    "Number of bins used in barcode count frequency distribution": "在条形码计数频率分布中使用的箱数",
    "Above-baseline multiplier to calculate roryk threshold": "高于基线的乘数，用于计算roryk阈值",
    ".mtx-format matrix": ".mtx格式矩阵",
    "Tab-delimited genes file": "制表符分隔的基因文件",
    "Tab-delimited barcodes file": "制表符分隔的条形码文件",
    "Prefix to prepend to cell names / barcodes": "添加到单元格名称/条形码前的前缀",
    "SC-Atlas experiment accession": "SC-Atlas 实验访问号",
    "Choose the type of matrix to download": "选择要下载的矩阵类型",
    "Variables to regress out": "需要回归排除的变量",
    "Expression Set Dataset": "表达集数据集",
    "Concatenate other Expression Set objects?": "合并其他表达集对象？",
    "Additional Dataset": "附加数据集",
    "Subset the dataset?": "子集数据集？",
    "By": "通过",
    "Select N Samples": "选择N个样本",
    "Select N Genes": "选择N个基因",
    "List of Sample Labels, comma-delimited": "样品标签列表，用逗号分隔",
    "List of Gene Labels , comma-delimited": "基因标签列表，用逗号分隔",
    "List of Sample Indexes, ranges are colon-delimited, seperated by commas.": "样本索引列表，范围以冒号分隔，以逗号分隔。",
    "List of Gene Indexes, ranges are colon-delimited, seperated by commas.": "基因索引列表，范围用冒号分隔，用逗号隔开。",
    "Filter Samples by Condition": "过滤样品的条件",
    "Name of phenotype column": "表型列的名称",
    "List of values in this column to filter for, comma-delimited": "该列中要筛选的值的列表，用逗号分隔",
    "Filter Genes by Condition": "按条件过滤基因",
    // APPEND Text From XML-Help Auto Translated By Kimi
    "Must be of datatype \"fastqsanger\" or \"fasta\"": "必须为数据类型“fastqsanger”或“fasta”",
    "'F' means a read corresponds to a transcript. 'R' means a read corresponds to the reverse complemented counterpart of a transcript. With this option being used, every read alignment will have an XS attribute tag: '+' means a read belongs to a transcript on '+' strand of genome. '-' means a read belongs to a transcript on '-' strand of genome.": "'F'表示一个读对应于一个转录本。'R'表示一个读对应于一个转录本的反向互补序列。使用这个选项时，每个读对齐都会有一个XS属性标签：'+'表示一个读属于基因组'+'链上的一个转录本。'-'表示一个读属于基因组'-'链上的一个转录本。",
    "Must be of datatype \"fastqsanger\"or \"fasta\"": "必须为“fastqsanger”或“fasta”数据类型",
    "Must be of datatype \"fastqsanger\" or \"fasta\". --interleaved": "必须为数据类型“fastqsanger”或“fasta”。--interleaved",
    "Select this option for compatibility with MultiQC": "选择此选项以兼容MultiQC",
    "Output alignment summary to a file in addition to stderr.": "将对齐摘要输出到文件以及标准错误。",
    "default: 0": "默认值：0",
    "default: no limit": "默认：无限制",
    "Trim N bases from 5' (left) end of each read before alignment, default: 0": "在比对之前，从每个读取的5'（左侧）端修剪N个碱基，默认值：0",
    "Trim N bases from 3' (right) end of each read before alignment, default: 0": "在比对之前，从每个读段的3'（右侧）端修剪N个碱基，默认值：0",
    "See help below for more details": "详见下面的帮助以获取更多详细信息",
    "--solexa-quals; default: False": "--solexa-quals; 默认值：False",
    "Quality values are represented in the read input file as space-separated ASCII integers, e.g., 40 40 30 40..., rather than ASCII characters, e.g., II?I.... Integers are treated as being on the Phred quality scale unless --solexa-quals is also specified [default: False]": "质量值在读取输入文件中以空格分隔的ASCII整数值表示，例如，40 40 30 40...，而不是ASCII字符，例如，II?I....。除非同时指定了--solexa-quals [默认：False]，否则整数被视为在Phred质量标度上。",
    "Reads exceeding this ceiling are filtered out": "超过这个上限的读取被过滤掉",
    "Constant term for the above function": "上述函数的常数项",
    "Coefficient for the above function": "上述函数的系数",
    "When calculating a mismatch penalty, always consider the quality value at the mismatched position to be the highest possible, regardless of the actual value. I.e. input is treated as though all quality values are high. This is also the default behavior when the input doesn't specify quality values": "在计算不匹配惩罚时，始终考虑不匹配位置的质量值为最高可能值，而不考虑实际值。即输入被视为所有质量值都很高。当输入未指定质量值时，这也是默认行为。",
    "If --nofw is specified, HISAT2 will not attempt to align unpaired reads to the forward (Watson) reference strand. In paired-end mode, --nofw and --norc pertain to the fragments; i.e. specifying --nofw causes HISAT2 to explore only those paired-end configurations corresponding to fragments from the reverse-complement (Crick) strand": "如果指定了--nofw参数，HISAT2将不会尝试将未配对的读段对齐到正向（Watson）参考链。在成对末端模式下，--nofw和--norc参数与片段有关；即指定--nofw会导致HISAT2仅探索那些对应于来自反向互补（Crick）链的片段的成对末端配置。",
    "If --norc is specified, HISAT2 will not attempt to align unpaired reads against the reverse-complement (Crick) reference strand. In paired-end mode, --nofw and --norc pertain to the fragments; i.e. specifying --norc causes HISAT2 to explore only those paired-end configurations corresponding to fragments from the forward-complement (Watson) strand": "如果指定了--norc参数，HISAT2将不会尝试将未配对的读取序列与反向互补（克里克）参考链进行比对。在成对末端模式下，--nofw和--norc与片段有关；即指定--norc会导致HISAT2仅探索那些对应于来自正向互补（沃森）链的片段的成对末端配置。",
    "Sets the maximum mismatch penalty. A number less than or equal to MX and greater than or equal to MN is subtracted from the alignment score for each position where a read character aligns to a reference character, the characters do not match, and neither is an N. If --ignore-quals is specified, the number subtracted quals MX. Otherwise, the number subtracted is MN + floor( (MX-MN)(MIN(Q, 40.0)/40.0) ) where Q is the Phred quality value": "设置最大错配惩罚。对于每个位置，如果读取字符与参考字符对齐，字符不匹配，且两者均不是N，则从对齐分数中减去一个小于或等于MX且大于或等于MN的数字。如果指定了--ignore-quals，则减去的数字为quals MX。否则，减去的数字是MN + floor( (MX-MN)(MIN(Q, 40.0)/40.0) )，其中Q是Phred质量值。",
    "Sets the minimum mismatch penalty. A number less than or equal to MX and greater than or equal to MN is subtracted from the alignment score for each position where a read character aligns to a reference character, the characters do not match, and neither is an N. If --ignore-quals is specified, the number subtracted quals MX. Otherwise, the number subtracted is MN + floor( (MX-MN)(MIN(Q, 40.0)/40.0) ) where Q is the Phred quality value": "设置最小不匹配惩罚。对于每个位置，如果读取字符与参考字符对齐，字符不匹配，且两者都不是N，则从对齐分数中减去一个小于或等于MX且大于或等于MN的数字。如果指定了--ignore-quals，则减去的数字为MX。否则，减去的数字是MN + floor( (MX-MN)(MIN(Q, 40.0)/40.0) )，其中Q是Phred质量值。",
    "Sets the maximum (MX) penalty for soft-clipping per base. A number less than or equal to MX and greater than or equal to MN is subtracted from the alignment score for each position. The number subtracted is MN + floor( (MX-MN)(MIN(Q, 40.0)/40.0) ) where Q is the Phred quality value": "设置每个碱基软剪切的最大（MX）惩罚。从比对得分中减去一个小于或等于MX且大于或等于MN的数字，以惩罚每个位置。减去的数字是MN + floor( (MX-MN)(MIN(Q, 40.0)/40.0) )，其中Q是Phred质量值。",
    "Sets the minimum (MN) penalty for soft-clipping per base. A number less than or equal to MX and greater than or equal to MN is subtracted from the alignment score for each position. The number subtracted is MN + floor( (MX-MN)(MIN(Q, 40.0)/40.0) ) where Q is the Phred quality value": "设置每个碱基软裁剪的最小（MN）惩罚。从比对得分中减去一个小于或等于MX且大于或等于MN的数字，以计算每个位置的得分。减去的数字是MN + floor( (MX-MN)(MIN(Q, 40.0)/40.0) )，其中Q是Phred质量值。",
    "Sets penalty for positions where the read, reference, or both, contain an ambiguous character such as N": "设置对读取、参考序列或两者都包含模糊字符（如N）的位置的惩罚",
    "A read gap of length N gets a penalty of [open_penalty] + N * [extend_penalty]": "一个长度为N的读取间隙会得到一个惩罚，该惩罚为[open_penalty] + N * [extend_penalty]",
    "A reference gap of length N gets a penalty of [open_penalty] + N * [extend_penalty]": "一个长度为N的参考间隙将获得一个惩罚，该惩罚为 [open_penalty] + N * [extend_penalty]",
    "This is a function of read length": "这是一个读取长度的函数",
    "Alignments with shorter introns are preferred to those with longer ones": "较短内含子的比对更受青睐，而不是较长内含子的比对",
    "E.g. if -I 60 is specified and a paired-end alignment consists of two 20-bp alignments in the appropriate orientation with a 20-bp gap between them, that alignment is considered valid (as long as -X is also satisfied). A 19-bp gap would not be valid in that case. If trimming options -3 or -5 are also used, the -I constraint is applied with respect to the untrimmed mates. The larger the difference between -I and -X, the slower HISAT2 will run. This is because larger differences between -I and -X require that HISAT2 scan a larger window to determine if a concordant alignment exists. For typical fragment length ranges (200 to 400 nucleotides), HISAT2 is very efficient": "例如，如果指定了-I 60，并且配对端对齐由两个20-bp的对齐组成，这两个对齐在适当的方向上，并且它们之间有20-bp的间隙，那么这个对齐被认为是有效的（只要-X也满足）。如果间隙是19-bp，那么在这种情况下它将不被认为有效。如果还使用了修剪选项-3或-5，那么-I约束将应用于未修剪的配对。-I和-X之间的差异越大，HISAT2的运行速度就越慢。这是因为-I和-X之间的较大差异要求HISAT2扫描更大的窗口，以确定是否存在一致的对齐。对于典型的片段长度范围（200到400个核苷酸），HISAT2非常高效。",
    "E.g. if -X 100 is specified and a paired-end alignment consists of two 20-bp alignments in the proper orientation with a 60-bp gap between them, that alignment is considered valid (as long as -I is also satisfied). A 61-bp gap would not be valid in that case. If trimming options -3 or -5 are also used, the -X constraint is applied with respect to the untrimmed mates, not the trimmed mates. The larger the difference between -I and -X, the slower HISAT2 will run. This is because larger differences between -I and -X require that HISAT2 scan a larger window to determine if a concordant alignment exists. For typical fragment length ranges (200 to 400 nucleotides), HISAT2 is very efficient": "例如，如果指定了-X 100，并且配对端对齐由两个20-bp的端对齐组成，它们在正确的方向上，并且它们之间有60-bp的间隙，那么该端对齐被认为是有效的（只要-I也满足）。如果间隙为61-bp，则在该情况下将无效。如果还使用了修剪选项-3或-5，则-X约束适用于未修剪的配对端，而不是修剪后的配对端。-I和-X之间的差异越大，HISAT2的运行速度就越慢。这是因为-I和-X之间的较大差异要求HISAT2扫描更大的窗口，以确定是否存在一致的端对齐。对于典型的片段长度范围（200到400个核苷酸），HISAT2非常高效。",
    "Default: false": "默认值：false",
    "Search for at most K distinct, primary alignments for each read. Primary alignments mean alignments whose alignment score is equal or higher than any other alignments. The search terminates when it can't find more distinct valid alignments, or when it finds K, whichever happens first. The alignment score for a paired-end alignment equals the sum of the alignment scores of the individual mates. Each reported read or pair alignment beyond the first has the SAM 'secondary' bit (which equals 256) set in its FLAGS field. For reads that have more than K distinct, valid alignments, HISAT2 does not guarantee that the K alignments reported are the best possible in terms of alignment score. HISAT2 is not designed with large values for -k in mind, so when aligning reads to long repetitive genomes, a large K can be very, very slow. Default: 5 (HFM) or 10 (HGFM)": "为每个读寻找最多K个不同的、主要的比对。主要比对意味着比对分数等于或高于其他比对的比对。当它找不到更多的不同有效比对时，搜索就会终止，或者当它找到K个比对时，以先发生的情况为准。成对末端比对的比对分数等于各个配对的比对分数之和。每个报告的读或配对比对，除了第一个之外，在其FLAGS字段中都设置了SAM的'secondary'位（等于256）。对于有超过K个不同、有效比对的读，HISAT2不能保证报告的K个比对在比对分数方面是最好的可能比对。HISAT2没有考虑使用大的-k值进行设计，因此当将读与长重复基因组进行比对时，大的K值可能会非常非常慢。默认值：5（HFM）或10（HGFM）",
    "This triggers --un parameter for single reads and --un-conc for paired reads": "这触发了单端读取的 --un 参数和双端读取的 --un-conc 参数",
    "This triggers --al parameter for single reads and --al-conc for paired reads": "这触发了单端读取的 --al 参数和双端读取的 --al-conc 参数",
    "Set the read group ID to the value set here. This causes the SAM @RG header line to be printed, with this input value as the value associated with the ID: tag. It also causes the RG:Z: extra field to be attached to each SAM output record, with value set to the value set here.": "将读组ID设置为此处设置的值。这会导致SAM @RG头行被打印出来，并将此输入值作为与ID:标签关联的值。它还会导致RG:Z:额外字段附加到每个SAM输出记录上，并将值设置为此处设置的值。",
    "Add specified text (usually of the form TAG:VAL, e.g. SM:Pool1) as a field on the @RG header line. Note: in order for the @RG line to appear, Read Group ID must also be specified. This is because the ID tag is required by the SAM Spec. Specify --rg multiple times to set multiple fields. See the SAM Spec for details about what fields are legal.": "在@RG头行上添加指定的文本（通常为TAG:VAL格式，例如SM:Pool1）。注意：为了使@RG行出现，还必须指定读取组ID。这是因为SAM规范要求ID标签。通过多次指定--rg来设置多个字段。有关合法字段的详细信息，请参见SAM规范。",
    "Default=0": "默认=0",
    "see Help below for explanation of this option; default: False": "请参见下面的帮助以了解此选项的说明；默认值：False",
    "This should be a 6 column (or 7 with additional k=v pairs) tabular file of link data.": "这应该是一个包含链接数据的6列（或7列，如果包含额外的键值对k=v）的表格文件。",
    "Controls which end(s) of the link are processed for each bin. The start of the link is considered to be the end defined by the first link line, and the link end is defined by the second link line. For example, if -link_end=start then a bin will contain the sum (or number if -num is used) of links that start at that bin. If -link_end==end then links that end at that bin will be used. Finally, -link_end=both will make the bin value reflect the total outgoing and incoming links. (from the Circos Manual)": "控制每个箱处理链接的哪一端（或两端）。链接的起始端被认为是由第一条链接线定义的端点，链接的末端由第二条链接线定义。例如，如果-link_end=start，那么一个箱将包含从该箱起始的链接的总和（如果使用了-num，则为链接的数量）。如果-link_end==end，则使用在该箱结束的链接。最后，如果-link_end=both，则使箱的值反映总的出站和入站链接。（来自Circos手册）",
    "If the end of a link spans bins, each bin will contain a +1 contribution from the link. In other words, the sum of bin values may be larger than the number of links. (from the Circos Manual)": "如果一个链接的末端跨越了多个箱子，那么每个箱子将包含来自该链接的+1贡献。换句话说，箱子值的总和可能大于链接的数量。（来自Circos手册）",
    "This only works when you select 'stacked histograms' in the 'Output Style' section.": "这只有在您在“输出样式”部分选择“堆叠直方图”时才有效。",
    "Adjacent links are merged into bundles if their start/end coordinates are sufficiently close. Given two links L1 and L2, they are merged into a bundle if: chr( L1 ) == chr( L2 ) AND distance( start(L1), start(L2) ) <= max_gap AND distance( end(L1), end(L2) ) <= max_gap. If a link does not have any acceptable adjacent neighbouring links, it forms a single-link bundle.": "如果相邻链接的起始/终止坐标足够接近，则将它们合并为束。给定两个链接L1和L2，如果满足以下条件，则将它们合并为一个束：chr(L1) == chr(L2) AND distance(start(L1), start(L2)) <= max_gap AND distance(end(L1), end(L2)) <= max_gap。如果一个链接没有任何可接受的相邻邻近链接，则它形成一个单链接束。",
    "The minimum number of links required in a bundle for the bundle to be accepted": "一个捆绑包被接受所需的最小链接数",
    "Extent cutoff for a bundle. The extent of a bundle is the end-start value for its start and end. The minimum is taken, so the cutoff test is `min( end1-start1, end2-start2 ) >? cutoff`": "捆绑包的范围截止。捆绑包的范围是其起始和结束的结束-起始值。取最小值，因此截止测试是 `min(end1-start1, end2-start2) >? cutoff`",
    "Size cutoff for a bundle. The size of a bundle is the total size of merged link coordinates for the bundle. There is a separate size for the bundle start (composed of merged link ends that form the bundle start) and bundle end. The minimum is taken, so the cutoff test is `min( size(merged link for start), size(merged link for end) ) >? cutoff`": "捆绑包的大小限制。捆绑包的大小是捆绑包中合并链接坐标的总大小。捆绑包的起始部分（由形成捆绑包起始部分的合并链接末端组成）和结束部分有各自的大小。取两者中的最小值，因此大小限制测试为 `min( size(merged link for start), size(merged link for end) ) >? cutoff`",
    "This parameter filters bundles based on the bundle identity, which is defined as `identity = bundle_size / bundle_extent`. Identity of both ends of the bundle are evaluated independently and the minimum is taken `min( identity(start), identity(end) ) >? cutoff`": "这个参数基于捆绑包身份过滤捆绑包，捆绑包身份定义为 `identity = bundle_size / bundle_extent`。捆绑包两端的身份独立评估，并取最小值 `min( identity(start), identity(end) ) >? cutoff`",
    "In parentheses are the naming conventions for chromosomes used in each karyotype file. These are required for some plot parameters (e.g. filtering chromosomes or specifying rules). The karyotype will also be output along with your plot, for your review.": "在括号中是每个核型文件中使用的染色体命名约定。这些命名约定对于某些绘图参数是必需的（例如过滤染色体或指定规则）。核型图也将与您的绘图一起输出，供您查看。",
    "Warning: this also contains unplaced scaffolds present in the reference genome. You can restrict the regions to plot in 'Limit/Filter Chromosomes' below, or supply your own file.": "警告：这也包含了参考基因组中存在的未定位的scaffolds。你可以在下面的“限制/过滤染色体”中限制要绘制的区域，或者提供你自己的文件。",
    "Provide your own karyotype configuration. Should be 'chr - ID LABEL START END COLOR'": "提供您自己的核型配置。应该是'chr - ID 标签 START END 颜色'",
    "This needs to be a 2+ column tabular, e.g. from 'Compute sequence lengths', the first column should be chromosome and second should be length.": "这需要是一个两列以上的表格，例如从“计算序列长度”中，第一列应该是染色体，第二列应该是长度。",
    "For example: 'hs1;hs2;hs3' or 'chr1:1-1000;chr2:3000-9000' to plot only a subset of regions. The chromosome names must match those in your karyotype file. They will be displayed in this order as well, you will need to specify all chromosomes that you wish to plot. The numerical positions are in terms of chromosome units (whatever this is set to)": "例如：'hs1;hs2;hs3' 或 'chr1:1-1000;chr2:3000-9000' 仅绘制区域的子集。染色体名称必须与您的核型文件中的名称匹配。它们也将按照此顺序显示，您需要指定您希望绘制的所有染色体。数值位置是相对于染色体单位的（无论这设置为多少）",
    "Important: If you set this, you will need to set the 'Limit/Filter Chromosomes' option (above) as well": "重要：如果您设置了此选项，您还需要设置上面的“限制/过滤染色体”选项。",
    "Setting this to MB allows usage of numbers like 5u to represent 5MB, rather than having to specify 5000000 or 5e6": "将此设置为MB允许使用像5u这样的数字来表示5MB，而不是必须指定5000000或5e6",
    "as written in your karyotype file": "如您的核型文件中所写",
    "Rotate the plot such that the first ideogram is at the top center of the page.": "将图表旋转，使第一个染色体图位于页面的顶部中心。",
    "Chromosomes are plotted clockwise around the chromosome": "染色体以顺时针方向围绕染色体绘制",
    "When set to yes/true, labels will be perpendicular to the tangent of the circle at the location of the label. Otherwise, they will be parallel with the tangent of the circle": "当设置为yes/true时，标签将与圆上标签位置处的切线垂直。否则，它们将与圆的切线平行。",
    "Cytogenic bands can be defined inside the karyotype file": "细胞遗传带可以在核型文件中定义",
    "0 is solid, 1 is the least transparent, and 5 is the most transparent": "0 是不透明的，1 是透明度最低的，5 是透明度最高的",
    "Plot radius (in pixels), final plot dimensions will be 2*r by 2*r": "绘制半径（以像素为单位），最终绘制的尺寸将是2*r乘以2*r",
    "Only useful in a tiny subset of situations": "只在极少数情况下有用",
    "2D Data Plots allow you to graph all sorts of data visually such as scatter or line plots, heatmaps, or tiles representing gene locations. These are commonly used to plot sequencing density tracks or other datasets that have some value at different positions along a genome.": "二维数据图允许您以图形方式绘制各种数据，例如散点图或折线图、热图或代表基因位置的平铺图。这些通常用于绘制测序密度轨迹或具有基因组不同位置上某些值的其他数据集。",
    "Value must be between 1 (outer edge of plot) and 0 (center of the plot)": "值必须在1（图表的外边缘）和0（图表的中心）之间",
    "Columns must be (chrom, start, end, value)": "列必须为（染色体，起始位置，结束位置，值）",
    "Join histogram bins that do not touch (abut)": "合并不接触（相邻）的直方图箱",
    "Columns must be (chrom, start, end, values). Values should be comma-separated": "列必须是（染色体，起始位置，终止位置，值）。值应该是用逗号分隔的",
    "E.g. blues-5-seq-rev,reds-5-seq": "例如 blues-5-seq-rev,reds-5-seq",
    "If scale_log_base<1, the dynamic range of color mapping of small values will be increased. If scale_log_base>1, then dynamic range of large values will be increased.": "如果scale_log_base小于1，小值的颜色映射的动态范围将增加。如果scale_log_base大于1，则大值的动态范围将增加。",
    "Columns must be (chrom, start, end)": "列必须为（染色体，起始位置，结束位置）",
    "Column must be (chrom, start, end)": "列必须为（染色体，起始位置，结束位置）",
    "In pixels": "在像素中",
    "In chromosome units. This is essentially distance added to the gene, before laying it out. Increasing it will prevent genes from packing as well.": "在染色体单位中。这基本上是在布局基因之前添加到基因上的距离。增加它将防止基因打包得更好。",
    "Columns must be (chrom, start, end, text)": "列必须为（染色体，起始位置，结束位置，文本）",
    "From the manual: \"Each link line has five dimensions: outer padding, outer line length (drawn at new position), connecting line length (connects old to new position), inner line length (drawn at old position) and inner padding. If snuggling is not used, then distinction between line lengths is not made\".": "根据手册：\"每个链接线有五个维度：外部填充，外部线长（在新位置绘制），连接线长（连接旧位置到新位置），内部线长（在旧位置绘制）和内部填充。如果不使用紧凑布局，则不区分线长。\"",
    "The label's radial position may be slightly adjusted to reduce the number of layers of text": "标签的径向位置可能会稍微调整，以减少文本层数",
    "Shift label up to N times its height, in the angular direction": "将标签向上移动，最多为其高度的N倍，在角度方向上",
    "Sample possible positions every N pixels": "样本可能的位置每N个像素",
    "You can also short-circuit precise placement by setting snuggle_tolerance (absolute or relative to label's tangential size). The larger this value, the less precise the placement. (from Circos manual)": "您还可以通过设置snuggle_tolerance（相对于标签切向大小的绝对值或相对值）来短路精确放置。这个值越大，放置的精确度就越低。（来自Circos手册）",
    "If the labels have a link line, you choose to test whether link lines overlap with previous labels using snuggle_link_overlap_test (from Circos manual)": "如果标签有连接线，你选择使用 snuggle_link_overlap_test（来自Circos手册）来测试连接线是否与之前的标签重叠。",
    "The extent of acceptable overlap is set using snuggle_link_overlap_tolerance (from Circos manual)": "可接受的重叠范围是通过设置snuggle_link_overlap_tolerance来设定的（参考Circos手册）。",
    "The snuggle_refine parameter toggles an additional check for crossing of links of labels that are placed at similar radial positions. Label pairs whose link lines cross are swapped. This refine parameter is functionaly only if show_links=yes. (from Circos manual)": "snuggle_refine 参数切换了一个额外的检查，用于检查放置在相似径向位置的标签的链接是否交叉。交叉的链接线条的标签对将被交换。此细化参数仅在 show_links=yes 时才有效。（来自 Circos 手册）",
    "Columns must be (chrom, pos1, pos2). Connectors are articulated line segments that relate two positions on an ideogram between two radial positions.": "列必须是（染色体，位置1，位置2）。连接器是连接两个径向位置之间两个位置的连线段。",
    "When yes, the plot will face inwards; i.e. lowest values will be plotted to the outside": "当设置为“是”时，图表将向内展示；即最低值将被绘制在外部",
    "Link tracks show a relationship between two locations in your plot. They are often used for indicating sequence similarity between two regions of a genome, or some interaction between two groups.": "链接轨迹展示了你的剧情中两个位置之间的关系。它们通常用于指示基因组两个区域之间的序列相似性，或者两个群体之间的某种相互作用。",
    "This is the radial position of the termination of the link. Value must be between 1 (outer edge of plot) and 0 (center of the plot)": "这是链接终止处的径向位置。值必须在1（图表的外边缘）和0（图表的中心）之间。",
    "Select file with link data. Format: chr1 start1 end1 chr2 start2 end2": "选择包含链接数据的文件。格式：chr1 start1 end1 chr2 start2 end2",
    "Fine grained control for display behaviour of ribbons": "精细控制彩带显示行为",
    "The radial position of the third control point (in addition to the two positions defined by the link coordinates) used to draw the Bezier curve; if this parameter is 0 then straight lines will be used": "贝塞尔曲线绘制中第三个控制点（除了由链接坐标定义的两个位置之外）的径向位置；如果此参数为0，则将使用直线。",
    "Two additional Bezier control points can be set by using the crest parameter. When defined, points p3 and p4 are added. These points lie at the same angular position as the start and end link termini and have the radial position. In the crest=0 extreme, p3 and p4 are at the same position as p0,p1. In this case, crest has no effect. When crest=1, p3,p4 are at the radial position of p2, the control point set by bezier_radius": "通过使用crest参数，可以设置两个额外的贝塞尔控制点。当定义时，将添加点p3和p4。这些点位于与起始和结束链接末端相同的角度位置，并具有径向位置。在crest=0的极端情况下，p3和p4与p0、p1处于相同的位置。在这种情况下，crest没有效果。当crest=1时，p3、p4位于由bezier_radius设置的控制点p2的径向位置。",
    "The bezier_radius_purity adjusts the position of p2 for each link. The p2 control point will move along the line formed by the original p2 location and the intersection of p0-p1 and the bisecting radius. When purity = 1, p2' = p2. When purity = 0, p2' = midpoint(p0,p1)": "bezier_radius_purity 调整每个链接的 p2 的位置。p2 控制点将沿着由原始 p2 位置和 p0-p1 与平分半径的交点形成的线移动。当 purity = 1 时，p2' = p2。当 purity = 0 时，p2' = p0 和 p1 的中点。",
    "A set of parameters can be used to randomly adjust bezier_radius, bezier_radius_purity, and crest parameters to give the links a more disorganized, organic feel. By perturbing each link you can also show additional texture in the data among links which would have ordinarily overlapped. Each parameter's perturbation amount is defined as a pair of values - pmin,pmax. These are the minimum and maximum multipliers by which the value can be perturbed": "一组参数可以用来随机调整bezier_radius（贝塞尔半径）、bezier_radius_purity（贝塞尔半径纯度）和crest（顶点）参数，以赋予链接更无序、更有机的感觉。通过扰动每个链接，您还可以在数据中显示额外的纹理，这些纹理在通常重叠的链接中是看不到的。每个参数的扰动量被定义为一对值 - pmin,pmax。这些是值可以被扰动的最小和最大乘数。",
    "For example setting (min,max) = (0.5,1.2) will cause parameter to be randomly scaled between 50-120%": "例如，设置（最小值，最大值）=（0.5,1.2）将导致参数在50-120%之间随机缩放",
    "From the documentation: \"Local adjustment of scale is for cases when you want to zoom parts of your data domain without cropping. If you think of the ideogram as a rubber band, applying a local scale adjustment is analogous to locally stretching or compressing the rubber band. The effect is that you still see the entire rubber band, but the length scale across it is variable.\"": "从文档中：“本地缩放调整是用于当你想要在不裁剪的情况下放大数据域的某些部分。如果你将染色体视为一个橡皮筋，应用本地缩放调整相当于局部拉伸或压缩橡皮筋。其效果是你仍然可以看到整个橡皮筋，但上面的长度尺度是可变的。”",
    "in chromosome units": "在染色体单位中",
    "If your chromosome units is set to kilobases and you have enabled this option and enabled tick labels, they will include the size value, e.g. '500 kb'": "如果您的染色体单位设置为千碱基，并且您启用了此选项并启用了刻度标签，它们将包含大小值，例如“500 kb”",
    "Oftentimes this will not be viewable directly in Galaxy, you may need to download this file first.": "通常这在Galaxy中无法直接查看，你可能需要先下载这个文件。",
    "This contains all the files needed to make this plot. With the data contained within here, and a copy of Circos installed locally, you can rebuild the plot and make any manual changes that are not possible in Galaxy (e.g. colours not in the colour picker)": "这包含了制作此图表所需的所有文件。有了这里包含的数据，以及本地安装的Circos副本，您可以重建图表，并进行任何在Galaxy中不可能进行的手动更改（例如颜色选择器中没有的颜色）。",
    "These MUST have identical chromosomes and intervals, within their datasets. This tool is incredibly, intentionally naïve, and will crash if there are missing ranges in some bigwig files. This feels like a reasonable assumption to make (simplifies processing) at the expense of theoretical correctness, given that when you are stacking histograms they're probably coming from the same upstream pipeline with the same settings (i.e. step/width/etc.)": "这些必须在其数据集中具有相同的染色体和区间。这个工具非常、有意地天真，如果一些bigwig文件中缺少范围，它将会崩溃。这似乎是一个合理的假设（简化处理），以牺牲理论上的正确性为代价，因为当您堆叠直方图时，它们可能来自具有相同设置（即步长/宽度等）的相同上游管道。",
    "Input data is a tabular file in which the last column (including the column if there is only one) contains the target values.": "输入数据是一个表格文件，其中最后一列（如果只有一列则包括该列）包含目标值。",
    "Predicted data is a tabular file in which the last column (including the column if there is only one) contains the predicted values.": "预测数据是一个表格文件，其中最后一列（如果只有一列，则包括该列）包含预测值。",
    "Input data is a matrix (tabular) where each column is a feature and the last column contains the (true or original) class labels.": "输入数据是一个矩阵（表格），其中每一列是一个特征，最后一列包含（真实或原始的）类别标签。",
    "Predicted data is a matrix (tabular) where each column is a feature and the last column contains the predicted class labels.": "预测数据是一个矩阵（表格），其中每一列是一个特征，最后一列包含预测的类别标签。",
    "This file is a final model trained on training data.": "这个文件是在训练数据上训练出的最终模型。",
    "Dataset missing? See TIP below": "数据集丢失？请查看下面的提示",
    "If your data is not in a style similar to what is available from Galaxy (and the UCSC table browser), choose 'Basic'.": "如果您的数据格式与Galaxy（和UCSC表格浏览器）提供的格式不相似，请选择“基本”。",
    "These do not affect behavior, only suppress warning messages.": "这些不影响行为，只是抑制警告消息。",
    "The FASTA reference genome that reads were mapped against.": "被读取映射的FASTA参考基因组。",
    "Note that SVG might not display correctly on all Galaxy servers": "请注意，SVG在所有Galaxy服务器上可能无法正确显示",
    "The special value '-1' will first compute the average depth and the set the max depth to 2*average": "特殊值'-1'将首先计算平均深度，然后将最大深度设置为2倍的平均深度",
    "Default is to let the coverage plot exceed the y-axis limit": "默认情况下，覆盖度图会超过y轴的限制。",
    "A distance specified as a positive integer.": "一个指定为正整数的距离。",
    "See also --partition. Multiple read groups can be separated with commas.": "另见--partition参数。多个读组可以用逗号分隔。",
    "Partitions are modelled on the GATK logic as described here: https://gatk.broadinstitute.org/hc/en-us/articles/360051307491-DepthOfCoverage-BETA-#--partition-type": "分区是基于GATK逻辑建模的，如这里所述：https://gatk.broadinstitute.org/hc/en-us/articles/360051307491-DepthOfCoverage-BETA-#--partition-type",
    "Look for datasets with 'JBrowse ... Data Directory' in the name": "查找名称中带有“JBrowse ... 数据目录”的数据集",
    "Built-in references": "内置参考",
    "Organise your tracks into Categories for a nicer end-user experience. You can use #date# and it will be replaced with the current date in 'yyyy-mm-dd' format, which is very useful for repeatedly updating a JBrowse instance when member databases / underlying tool versions are updated.": "将您的轨道组织成类别，以提供更好的最终用户体验。您可以使用 #date#，它将被替换为当前日期的 'yyyy-mm-dd' 格式，这在更新成员数据库/底层工具版本时重复更新 JBrowse 实例非常有用。",
    "in GFF3. This is used  so we know where to map features. E.g. where results of which CDS Protein32 match up to. The query IDs in your blast results should MATCH some feature IDs in your GFF3 file. This is an optional field and is most useful if using JBrowse to display protein blast results on a DNA genome. blastn results don't need this, blastp results on a protein sequence don't need this.": "在GFF3格式中。这是用来确定特征映射位置的。例如，哪个CDS蛋白32的搜索结果与哪里匹配。你的BLAST搜索结果中的查询ID应该与你的GFF3文件中的一些特征ID匹配。这是一个可选字段，如果使用JBrowse在DNA基因组上显示蛋白质BLAST搜索结果时最有用。blastn搜索结果不需要这个，蛋白质序列上的blastp搜索结果也不需要这个。",
    "before a new match_part feature is created": "在创建新的 match_part 特性之前",
    "Match_parts have several options for the parent feature type, such as cDNA_match, match, translated_nucleotide_match, etc. Please select the appropriate one here. You can leave empty to try autodetection (only works with CanvasFeatures track type).": "Match_parts 有几种针对父特征类型的选项，比如 cDNA_match、match、translated_nucleotide_match 等。请在这里选择适当的选项。你可以留空以尝试自动检测（这只适用于 CanvasFeatures 轨迹类型）。",
    "Not recommended for deep coverage BAM files": "不建议用于深度覆盖的BAM文件",
    "Maximum size in bytes of BAM chunks that the browser will try to deal with. When this is exceeded, most tracks will display 'Too much data' message.": "浏览器尝试处理的最大BAM块大小（以字节为单位）。当超过这个大小时，大多数轨迹将显示“数据过多”的消息。",
    "instead of continuous colored band": "而不是连续的彩色带",
    "Only for XYPlots": "仅用于XYPlots",
    "MultiBigWig is a plugin to group multiple big-wig tracks in a more aesthetically pleasing format and to show data in a higher density display.": "MultiBigWig 是一个插件，用于将多个大wig轨道组合在一起，以更美观的格式展示，并且能够以更高的密度显示数据。",
    "A JBrowse plugin for viewing blast alignments. This plugin makes MANY assumptions about the format of your data, and may not work without those assumptions: 1) you use protein_match / match_part 2) your features have a Blast_qseq, Blast_mseq, and Blast_sseq attribute in their GFF3 representation.": "一个用于查看BLAST比对的JBrowse插件。这个插件对您的数据格式有很多假设，如果没有这些假设可能无法工作：1）您使用protein_match / match_part 2）您的特征在它们的GFF3表示中具有Blast_qseq、Blast_mseq和Blast_sseq属性。",
    "ComboTrackSelector is a plugin to allow the co-existence of the Hierarchical and Faceted Track selectors in JBrowse, built for/by the Arabidopsis Information Portal (Araport) project": "ComboTrackSelector 是一个插件，允许在 JBrowse 中同时使用分层和分面轨道选择器，为拟南芥信息门户（Araport）项目构建/提供。",
    "A JBrowse plugin for plotting GC Content and GC Skew. The plugin consists of a storeClass that automatically calculates the percentage of G/C bases in a region, a track type that derives from the Wiggle XY or density types, and a dialog box to adjust the sliding window size, window step size, and the calculation mode (content or skew).": "一个用于绘制GC含量和GC偏差的JBrowse插件。该插件包括一个storeClass，它自动计算区域中G/C碱基的百分比，一个派生自Wiggle XY或密度类型的track类型，以及一个对话框，用于调整滑动窗口大小、窗口步长和计算模式（内容或偏差）。",
    "The format is chr:start-end, for example chr10:10-500": "格式为 chr:起始-终止，例如 chr10:10-500",
    "If set, you may need to start by a spacer track": "如果设置，则可能需要以间隔轨道开始",
    "Resolution for the image in case the ouput is a raster graphics image (e.g png, jpg)": "如果输出是光栅图形图像（例如png、jpg），图像的分辨率",
    "This will overwrite what has been defined as a figure width.": "这将覆盖已定义的图形宽度。",
    "If not given, the figure height is computed based on the heights of the tracks. If given, the track height are proportionally scaled to match the desired figure height.": "如果未指定，则根据轨迹的高度计算图形的高度。如果指定了，则将轨迹高度按比例缩放以匹配所需的图形高度。",
    "By default, the x-axis is increasing. Use this option if you want to see all tracks with a decreasing x-axis.": "默认情况下，x轴是递增的。如果你想看到所有轨迹的x轴递减，请使用这个选项。",
    "Tracks can be of different filetypes. E.g BED, HiC-Matrix, BigWig or BedGraph. The order you add them represent the order they will be plotted from top to bottom.": "轨道可以是不同的文件类型。例如 BED、HiC-Matrix、BigWig 或 BedGraph。您添加它们的顺序代表了它们将从上到下绘制的顺序。",
    "Available color map names can be found here: https://matplotlib.org/examples/color/colormaps_reference.html": "可用的颜色映射名称可以在此处找到：https://matplotlib.org/examples/color/colormaps_reference.html",
    "If you leave it empty the height is calculated to have perfect squares rotated 45 degrees.": "如果您将其留空，则高度将计算为将正方形旋转45度的完美大小。",
    "If false, neighbour bins will be increase in size to hide the mask bins.": "如果为假，邻近的箱子大小会增加以隐藏掩码箱子。",
    "Will display triangles on top of the heatmap": "将在热图上方显示三角形",
    "if the values in the matrix need to be scaled the following parameter can be used": "如果矩阵中的值需要进行缩放，可以使用以下参数",
    "If you leave it empty the height is calculated to have perfect squares.": "如果留空，则高度会计算为完美平方。",
    "Will display squares on top of the heatmap": "将在热图上方显示方块",
    " Available color map names can be found here: https://matplotlib.org/examples/color/colormaps_reference.html": "可用的颜色映射名称可以在此处找到：https://matplotlib.org/examples/color/colormaps_reference.html",
    "Put labels on the left of the region if the region ends after the plotted region.": "如果区域在绘制区域之后结束，则在区域的左侧添加标签。",
    "Put labels in the margin if the region ends after the plotted region.": "如果区域在绘制区域之后结束，请将标签放在边距中。",
    "When set to False, the arrow indicating the orientation of the interval is plotted outside of the interval.": "当设置为False时，指示区间方向的箭头会绘制在区间之外。",
    "This size is also used to put space between the bed regions and their labels.": "这个大小也用于在床区域和它们的标签之间放置空间。",
    "the length of the arrow in bp (default is 4% of the plotted region)": "箭头的长度（以碱基对为单位，默认为绘制区域的4%）",
    "To be able to see small arcs when big arcs exists, you can set the upper y limit. The unit is bp. This corresponds to the longest arc you will see not truncated.": "为了在大弧存在时能看到小弧，您可以设置上限y值。单位是碱基对。这对应于您将看到的未被截断的最长弧。",
    "operation will be evaluated, it should contains file or file and second_file, for example `0.89 * file` `- file` `file - second_file` `log2((1 + file) / (1 + second_file))` or `max(file, second_file)`": "操作将被评估，它应该包含文件或文件和第二个文件，例如 `0.89 * file` `- file` `file - second_file` `log2((1 + file) / (1 + second_file))` 或 `max(file, second_file)`",
    "1 is opaque and 0 is totally transparent": "1 是完全不透明的，0 是完全透明的",
    "You can optionally define one or multiple parameters below.": "你可以在下面选择性地定义一个或多个参数。",
    "If not set and cannot be deduced from others it will be in the middle of the plotted area.": "如果未设置且不能从其他信息中推断出来，它将位于绘制区域的中间。",
    "If not set and cannot be deduced from others it will be the higher number that begins with 1, 2 or 5 followed by 0s that is less than half the plotted area.": "如果未设置且无法从其他信息中推断出来，它将是大于一半绘图区域的最小数，这个数以1、2或5开头，后面跟着0。",
    "checkout the choices for reference to see available species": "查看参考选项以了解可用的物种",
    "in pixels for image output or in characters for text output": "在像素中用于图像输出，或在字符中用于文本输出",
    "Supports multiple assembly graph formats: LastGraph (Velvet), FASTG (SPAdes), Trinity.fasta, ASQG and GFA.": "支持多种组装图形格式：LastGraph（Velvet）、FASTG（SPAdes）、Trinity.fasta、ASQG 和 GFA。",
    "If only height or width is set, the other will be determined automatically. If both are set, the image will be exactly that size. Default: 1000.": "如果只设置高度或宽度，另一个将自动确定。如果两者都设置，图像将恰好是那个大小。默认值：1000。",
    "If only height or width is set, the other will be determined automatically. If both are set, the image will be exactly that size. Default: not set.": "如果只设置了高度或宽度，另一个将自动确定。如果两者都设置了，图像将正好是那个大小。默认值：未设置。",
    "Node font size?": "节点字体大小？",
    "Node width for graph image?": "图形图像的节点宽度？",
    "Needs to be a string assembly name such as mm10 or hg38": "需要是一个字符串形式的组装名称，比如 mm10 或 hg38",
    "KEGG pathway ID with 5 digits and without the 3 letter KEGG species code.": "5位数字的KEGG通路ID，不包含3个字母的KEGG物种代码。",
    "A table with one-column with KEGG pathways to plot": "一张包含一列KEGG通路的表格以绘制图表",
    "Gene data is a broad concept including genes, transcripts, protein, enzymes and their expression, modifications and any measurable attributes.": "基因数据是一个广泛的概念，包括基因、转录本、蛋白质、酶及其表达、修饰和任何可测量的属性。",
    "It should be a table with first column being the gene ids and other being information (p-value, fold change, levels, etc) from one or several samples.": "它应该是一个表格，第一列是基因ID，其他列是来自一个或多个样本的信息（p值、倍数变化、水平等）。",
    "compound data is a broad concept including metabolites, drugs, their  measurements and attributes.": "复合数据是一个广泛的概念，包括代谢物、药物、它们的测量和属性。",
    "Gene or compound nodes will be sliced into multiple pieces corresponding to the number of states in the data": "基因或化合物节点将根据数据中状态的数量被切片成多块",
    "When let sample sizes of gene and compound be m and n, when m>n, extra columns of NA's (mapped to no color) will be added to make the sample size the same": "当基因和化合物的样本大小分别为 m 和 n 时，如果 m>n，则会添加额外的 NA 列（映射为无色）以使样本大小相同。",
    "If yes, node colors will be plotted in the same layer as the pathway graph.": "如果选择是，节点颜色将在同一层上绘制通路图。",
    "When NULL data are mapped, the gene or compound nodes in the pathway will berendered as actually mapped nodes, except with NA-valued color. When NULL data are not mapped, the nodes are rendered as unmapped nodes.": "当NULL数据被映射时，路径中的基因或化合物节点将被渲染为实际映射的节点，只是颜色值为NA。当NULL数据不被映射时，节点将被渲染为未映射的节点。",
    "If yes, edge/node type legend will be plotted in the same page.": "如果选择是，边/节点类型图例将在同一页面上绘制。",
    "Each split member nodes inherits all edges from the node group. This option also effects most metabolic pathways even without group nodes defined orginally. For these pathways, genes involved in the same reaction are grouped automatically when converting reactions to edges unless split.group is true.": "每个拆分成员节点继承了节点组的所有边。这个选项还影响大多数代谢途径，即使没有最初定义的组节点。对于这些途径，在将反应转换为边时，除非split.group为true，否则参与同一反应的基因会自动分组。",
    "Each expanded single-gene nodes inherits all edges from the original multiple-gene node. This option is not effective for most metabolic pathways where it conflits with converting reactions to edges.": "每个扩展的单基因节点继承原始多基因节点的所有边。此选项对于大多数代谢途径无效，因为它与将反应转换为边相冲突。",
    "Identical to LEfSe input file": "与LEfSe输入文件相同",
    "The levels must be in comma-separated form": "级别必须以逗号分隔的形式存在",
    "The clades must be in comma-separated form": "类群必须以逗号分隔的形式给出",
    "Colors can be either in RGB or HSV (using a semi-colon to separate values, surrounded with ()). The different colors must be in comma-separated form": "颜色可以是RGB或HSV格式（使用分号分隔值，用括号括起来）。不同的颜色必须用逗号分隔。",
    " The taxonomy is parsed, and the level is choosen in order to have at least the specified number of biomarkers": "分类学信息被解析，选择的层级是为了至少有指定数量的生物标记物",
    "-1 specifies that no names are present in the matrix": "-1 表示矩阵中没有名称",
    "The row numbers are indexed to 0 and comma separated": "行号从0开始，以逗号分隔",
    "The order is based on percentile specified by --sperc": "该顺序基于由--sperc指定的百分位数",
    "The order is based on percentile specified by --fperc": "排序是基于由 --fperc 指定的百分位数",
    "Tabular file (uses column one), FASTA, FASTQ or SFF file.": "表格文件（使用第一列）、FASTA、FASTQ或SFF文件。",
    "Two modes: 1) multiple names separated by comma, e.g. header1,header2, or 2) include `*` to initiate regular expression searches for every column name, for example, ^param_.* will retreive all the headers starting with param_. Note: no mix of the two is supported. Use default sanitizer: string.ascii_letters + string.digits + \" -=_.()/+*^,:?!\")": "两种模式：1）多个名称由逗号分隔，例如 header1,header2，或者 2）包含 `*` 以启动正则表达式搜索每个列名，例如，^param_.* 将检索所有以 param_ 开头的标题。注意：不支持这两种的混合使用。使用默认的消毒器：string.ascii_letters + string.digits + \" -=_.()/+*^,:?!\")",
    "e.g. mean_test_score": "例如，平均测试分数",
    "The annotation file is a tab-delimited file listing the graphical options for clades": "注释文件是一个制表符分隔的文件，列出了类群的图形选项。",
    "Input should have column headers with an identifier column followed by consencutive numerical columns used for clustering": "输入应具有列标题，其中包含一个标识符列，后跟用于聚类的连续数字列。",
    "Set to 42 as default": "设置为默认值42",
    "Choose input file from history. Format: count - Hierarchy (tab separated)": "选择历史记录中的输入文件。格式：计数 - 层级（制表符分隔）",
    "e.g. taxonomy file generated from Classify.otu?": "例如，由Classify.otu生成的分类学文件？",
    "Nucleotide FASTA file, e.g. contigs from genome assembly.": "核苷酸FASTA文件，例如来自基因组组装的片段。",
    "Will be printed only if Include Legend option is set to Yes": "如果“包含图例”选项设置为“是”，则将打印",
    "The lower bp limit on insert size for calculating the histogram.": "用于计算直方图的插入大小的下限碱基对数。",
    "The upper bp limit on insert size for calculating the histogram.": "用于计算直方图的插入大小上限基点数。",
    "Format of output image": "输出图像的格式",
    "An integer between 0 and 100, default 80. Larger values result in increased image quality and file size.": "一个介于0到100之间的整数，默认为80。数值越大，图像质量越高，文件大小也越大。",
    "Image resolution, a positive integer, default 1080. For non-square images this will be the resolution of the longest dimension.": "图像分辨率，一个正整数，默认为1080。对于非正方形图像，这将是最长维度的分辨率。",
    "Color map based on list available in the help section": "基于帮助部分列出的颜色映射表",
    "Each entry, except for '=all', corresponds to one output image. More information available in the help section.": "每个条目（除了'=all'）对应一个输出图像。更多信息可在帮助部分找到。",
    "Minimum map texels per image (along a single dimension), a positive integer, Output images over too small a range that violate this minimum will not be created.": "单张图像中最小映射纹理像素数（沿单一维度），一个正整数，违反此最小值范围的输出图像将不会被创建。",
    "Width in pixels of the sequence separation grid, a non-negative integer. Set to 0 to not overlay a grid.": "序列分隔网格的像素宽度，一个非负整数。设置为0则不覆盖网格。",
    "Colour of the sequence separation grid. Either, one of: 'black'(default), 'white', 'red', 'green', 'blue', 'yellow', 'cyan' or 'magenta'. Or, a sRGBA 32-bit hex code in RRGGBBAA format, e.g. 'ff00ff80' (magenta at half-occupancy).": "序列分离网格的颜色。可以是以下之一：'black'（默认）、'white'、'red'、'green'、'blue'、'yellow'、'cyan' 或 'magenta'。或者，可以是 RRGGBBAA 格式的 sRGBA 32 位十六进制代码，例如 'ff00ff80'（半占用的洋红色）。",
    "Print list of all individual suequeces names on the map, in order of appearance.": "打印地图上所有单独序列名称的列表，按照出现的顺序。",
    "Must be in gmt format (one pathway per line : Name, description, genes (one by column), tab separated)": "必须使用gmt格式（每行一条通路：名称、描述、基因（每列一个），用制表符分隔）",
    "If set the starting curve depends on the matrix of points with in a certain row order (first 'normal' then 'cancer' samples), otherwise the first principal component is used. See help for more informations": "如果设置起始曲线依赖于按特定行顺序排列的点矩阵（首先是“正常”样本，然后是“癌症”样本），否则将使用第一主成分。有关更多信息，请参见帮助。",
    "A two-column data frame, first column contains data labels, second column the levels of sample status : 1 = Healthy, 0 = Tumor (no header)": "一个两列的数据框，第一列包含数据标签，第二列包含样本状态的水平：1 = 健康，0 = 肿瘤（无标题）",
    "Use 'data' to use the minimum standard deviation of your data": "使用 'data' 来使用您的数据的最小标准偏差",
    "Use 'data' to use the minimum expression of your data": "使用 'data' 来使用你数据的最小表达量",
    "The sample names are listed in the columns of the VCF.": "样本名称列在VCF的列中。",
    "Provide the genes as a comma seperated line. This line can be empty and genes can be selected interactively using gene.iobio.": "以逗号分隔的线提供基因。这条线可以为空，并且可以使用gene.iobio交互式选择基因。",
    "Width of the plot. Increase for larger genomes.": "图形的宽度。对于较大的基因组，可以增加该值。",
    "Select 'no' to have no label box around feature labels": "选择 'no' 以去除特征标签周围的标签框",
    "Plot a specific region of the genome": "绘制基因组的特定区域",
    "start zoom": "开始缩放",
    "end zoom": "结束缩放",
    "Breaks up the plot into multiple lines": "将图表分成多行显示",
    "Common Features to be excluded from the plot": "排除在绘图之外的常见特征",
    "Feature(s) to exclude from plot (example: variation,misc_recomb). Separate by comma(s).": "要从图中排除的特征（例如：变异，重组记录）。用逗号分隔。",
    "Common Feature labels to exclude from the plot": "排除图中的常见特征标签",
    "If plotting multiple features, especially when using product names, and you want to ignore labeling certain types of features, input the feature(s) (example: variation,misc_recomb). For multiple feature labels to exclude, separate by commas.": "如果绘制多个特征，特别是使用产品名称时，您想要忽略标记某些类型的特征，请输入特征（例如：变异，重组杂合），对于要排除的多个特征标签，请用逗号分隔。",
    "Common product labels to ignore from the plot": "忽略图中常见的产品标签",
    "Product names to ignore as well as character patterns (example: hypothetical protein, JX). Separate by commas": "要忽略的产品名称以及字符模式（例如：假设蛋白，JX）。用逗号分隔",
    "Order of features to decide naming convention (example: locus_tag,product). Separate by commas.": "特征排序以决定命名约定（例如：locus_tag,product）。用逗号分隔。",
    "Some features will be displayed inside if there is sufficient space. If set to no, all labels will be placed in the box labels above the feature": "如果空间足够，一些特性将显示在内部。如果设置为否，所有标签将放置在特性上方的框标签中",
    "Feature name. Example: CDS. Case Sensitive.": "特征名称。示例：CDS。大小写敏感。",
    "Product Name. Example: holin. Case Sensitive.": "产品名称。示例：溶胞素。大小写敏感。",
    "Column headers are requirement. These will be the columns that are plotted.": "列标题是必需的。这些将是绘制的列。",
    "specific dataset input for reading": "特定数据集输入以读取",
    "Default horizontal: header names are interpreted as sample names": "默认水平：表头名称被解释为样本名称",
    "WARNING: please consider that using row names might shift the grouping columns": "警告：请注意，使用行名称可能会改变分组列",
    "default vertial: row names are interpreted as sample names": "默认垂直：行名被解释为样本名",
    "deside whether header names or row names are interpreted as sample names": "决定标题名称或行名称是否被解释为样本名称",
    "Select the samples for this group": "选择这个组的样本",
    "Use short names, avoid special characters and numbers at the beginning of the name (The names might be changed by the program to make them conform to processing in R)": "使用简短的名称，避免在名称开头使用特殊字符和数字（程序可能会更改名称以使其符合在R中的处理）",
    "WARNING: please use columns indices after removing the row name column if selected in second section!": "警告：如果在第二部分选择后删除了行名列，请使用列索引！",
    "The method used to compute the distance (dissimilarity) between both rows and columns. Default: Euclidean": "用于计算行和列之间距离（不相似性）的方法。默认值：欧几里得",
    "The method used to compute the hierarchical clustering. Default: Complete": "用于计算层次聚类的方法。默认值：完全连接",
    "Yes, if the starting curve depends on the matrix of points with in a certain row order (first 'normal' then 'cancer' samples), otherwise (No) the first principal component is used. See help section for more informations": "是的，如果起始曲线依赖于按特定行顺序排列的点矩阵（首先是“正常”样本，然后是“癌症”样本），否则（否）使用第一个主成分。有关更多信息，请参见帮助部分。",
    "tab delimited file with 2 columns: name and sequence.  For example: Illumina Small RNA RT Primer CAAGCAGAAGACGGCATACGA": "制表符分隔的文件，包含两列：名称和序列。例如：Illumina Small RNA RT Primer CAAGCAGAAGACGGCATACGA",
    "a file that specifies which submodules are to be executed (default=all) and also specifies the thresholds for the each submodules warning parameter": "一个指定要执行哪些子模块的文件（默认=全部），并且还指定了每个子模块的警告参数的阈值",
    "Use 0 to get the distribution of all quality values": "使用0来获取所有质量值的分布",
    "The seed used for the random number generator. Manually specifying a number here is useful for reproducing the same subsampling in different runs (e.g. read 1 and read 2)": "用于随机数生成器的种子。在这里手动指定一个数字对于在不同运行中重现相同的子采样非常有用（例如，读取1和读取2）。",
    "Use an integer > 1 to select a specific number of reads. Use a decimal (e.g. 0.5) to select a fraction of reads": "使用大于1的整数来选择特定数量的读取。使用小数（例如0.5）来选择读取的一部分。",
    "Use this if the number of reads you want to sample is small enough to fit into memory. If you're not sure, don't use this.": "如果你想要的采样的读段数量足够小，能够适应内存，请使用这个选项。如果你不确定，请不要使用这个选项。",
    "Effective with -f": "有效的参数为-f",
    "Effective with -M": "有效于 -M 参数",
    "FASTA, FASTQ, or SFF format.": "FASTA、FASTQ 或 SFF 格式。",
    "SAM or BAM format.": "SAM或BAM格式。",
    "FASTQ files.": "FASTQ 文件。",
    "FASTQ data in a paired collection": "成对集合中的FASTQ数据",
    "By default, sequences smaller than 60 bp are removed. No top threshold is defined": "默认情况下，小于60个碱基对的序列会被移除。没有定义上限阈值。",
    "By default, sequences smaller than 60 bp are removed.": "默认情况下，小于60个碱基对的序列将被移除。",
    "By default, no treatment based on a maximal length is made.": "默认情况下，没有基于最大长度的处理。",
    "By default, sequences with a mean score below 15 are removed.": "默认情况下，平均得分低于15的序列将被移除。",
    "By default, no treatment based on a minimum score is made.": "默认情况下，不会根据最小分数进行处理。",
    "By default, no treatment based on a maximum mean score is made.": "默认情况下，不会根据最大均值分数进行治疗。",
    "By default, sequences with more than 2% of N bases are removed.": "默认情况下，含有超过2%的N碱基的序列将被移除。",
    "By default, no treatment based on GC percentage is made.": "默认情况下，不根据GC百分比进行处理。",
    "By default, no treatment based on N number is made.": "默认情况下，不基于N数字进行治疗。",
    "By default, this treatment is not made. (-noniupac)": "默认情况下，不进行这种处理。(-noniupac)",
    "By default, no complexity filter is applied.": "默认情况下，不应用复杂性过滤器。",
    "The dust method uses the threshold as maximum allowed score and the entropy method as minimum allowed value.(-lc_threshold)": "尘土法使用阈值作为允许的最大分数，而熵法使用最小允许值。(-lc_threshold)",
    "By default, no length trimming is made": "默认情况下，不进行长度修剪",
    "By default, no position trimming is made": "默认情况下，不进行位置修剪",
    "The trim length is rounded towards the lower integer. By default, no position trimming is made": "修剪长度向下取整。默认情况下，不进行位置修剪",
    "By default, no tail trimming is made": "默认情况下，不进行尾部修剪",
    "By default, no poly-A/T tail trimming is made": "默认情况下，不进行多聚腺苷酸/胸腺嘧啶尾修剪",
    "By default, no 5'-end poly-A/T tail trimming is made": "默认情况下，不进行5'端的聚A/T尾修剪",
    "By default, no 3'-end poly-A/T tail trimming is made": "默认情况下，不进行3'端聚A/T尾修剪",
    "By default, no poly-N tail trimming is made": "默认情况下，不进行多聚N尾部修剪",
    "By default, no 5'-end poly-N tail trimming is made": "默认情况下，不做5'端的多N尾修剪",
    "By default, no 3'-end poly-N tail trimming is made": "默认情况下，不进行3'端多聚N尾修剪",
    "By default, a 3'-end trimming is made to remove ends with a minimum quality score over 5 bp below 20": "默认情况下，进行3'端修剪以去除质量分数低于20的末端，最小质量分数差为5。",
    "By default, no 5'-end quality trimming is made": "默认情况下，不进行5'端质量修剪",
    "By default, 3'-end trimming is made based on a score of 20": "默认情况下，3'端修剪是基于20分的分数进行的",
    "By default, min is used. (-trim_qual_type)": "默认情况下，使用的是min。（-trim_qual_type）",
    "By default, 'less than' is used. (-trim_qual_rule)": "默认情况下，使用的是“小于”。（-trim_qual_rule）",
    "To stop at the first base that fails the rule defined, use a window size of 1 (default value). (-trim_qual_window)": "要停在第一个不满足定义规则的碱基上，使用窗口大小为1（默认值）。(-trim_qual_window)",
    "To move the window over all quality scores without missing any, the step size sould be less or equal to the window size. The default value is 1. (-trim_qual_step)": "为了确保滑动窗口能够覆盖所有质量分数而不遗漏，步长应该小于或等于窗口大小。默认值是1。(-trim_qual_step)",
    "Optional, no reference is the default": "可选，没有参考是默认设置",
    "bases scoring below this value will trigger splitting": "得分低于这个值的碱基将触发拆分",
    "report all high quality segments above this length. Setting this option to '0' will cause the program to return a single longest run of high quality bases per read": "报告所有高于这个长度的高质量片段。将此选项设置为'0'将导致程序返回每个读取中最长的一段高质量碱基",
    "if set to 'DO NOT trigger splitting' the program will not count low quality bases that are within or adjacent to homonucleotide runs.  This will significantly reduce fragmentation of 454 data": "如果设置为'不要触发拆分'，程序将不会计算位于同聚物运行中或相邻的低质量碱基。这将显著减少454数据的碎片化。",
    "('0' = do not trim) The quality of Solexa reads drops towards the end. This option allows selecting the specified number of nucleotides from the beginning and then running the tool.": "（'0' = 不进行修剪）Solexa 测序读取的质量在末端会下降。这个选项允许从开始处选择指定数量的核苷酸，然后运行工具。",
    "No dataset? Read tip below": "没有数据集？请阅读下面的提示",
    "'0' = keep the whole thing": "'0' = 保留整个内容",
    "Stripping the description will truncate the fasta header to just the sequence ID. Otherwise the header description will be kept. This step is done before the 'How many characters to keep' option.": "剥离描述将把fasta头截断到只有序列ID。否则，头描述将被保留。这个步骤是在“要保留多少个字符”选项之前完成的。",
    "Use Absolute for fixed length reads (Illumina, SOLiD)<br>Use Percentage for variable length reads (Roche/454)": "使用绝对值用于固定长度的读取（Illumina, SOLiD）<br>使用百分比用于可变长度的读取（Roche/454）",
    "Values start at 0, increasing from the left": "值从0开始，从左侧递增",
    "Values start at 0, increasing from the right; use a negative value to remove everything to the right of the absolute value of the position": "值从0开始，从右侧递增；使用负值可以移除绝对值位置右侧的所有内容",
    "Typically 2 to take the ID (first word) and decription (rest) as two columns, or 1 to give a single column": "通常使用2来将ID（第一个词）和描述（其余部分）作为两列，或者使用1来给出一个单独的列。",
    "Requires groomed data: if your data does not appear here try using the FASTQ groomer": "需要整理过的数据：如果您的数据没有出现在这里，尝试使用FASTQ整理工具",
    "A maximum size less than 1 indicates no limit": "最大值小于1表示没有限制",
    "A maximum quality less than 1 indicates no limit": "最大质量小于1表示没有限制",
    "The above settings do not apply to these filters": "上述设置不适用于这些过滤器",
    "Values start at 0, increasing from the right": "值从右侧开始，从0递增",
    "Galaxy tools are designed to work with the Sanger Quality score format": "Galaxy工具旨在与Sanger质量评分格式一起工作",
    "An empty string will remove the adapter base": "一个空字符串将移除适配器基座",
    "Values are in Base-space and quality scores of maximal value will be used": "值以碱基空间表示，并将使用最大值的质量分数",
    "output of 'FASTQ Statistics' tool": "'FASTQ 统计'工具的输出",
    "Use 0 for single line out.": "使用0表示单行输出。",
    "Enable long reads up to this many bp long.": "启用长达这么多碱基对的长读长。",
    "Enable overrepresented sequence analysis.": "启用过度表示序列分析。",
    "One in (--overrepresentation_sampling) reads will be computed for overrepresentation analysis (1~10000), smaller is slower, default is 20.": "在（--overrepresentation_sampling）参数中，将计算过度代表性分析的一个读数（1~10000），数值越小，计算速度越慢，默认值为20。",
    "Quality filtering is enabled by default. If this option is specified, quality filtering is disabled.": "默认情况下启用质量过滤。如果指定了此选项，则禁用质量过滤。",
    "The quality value that a base is qualified. Default 15 means phred quality >=Q15 is qualified.": "一个碱基的质量值是合格的。默认值15表示Phred质量值>=Q15是合格的。",
    "How many percents of bases are allowed to be unqualified (0~100). Default 40 means 40%.": "允许有多少百分比的碱基不合格（0~100）。默认值40表示40%。",
    "If one read's number of N base is >n_base_limit, then this read/pair is discarded. Default is 5.": "如果一个读的N碱基数量大于n_base_limit，则丢弃该读/配对。默认值为5。",
    "Length filtering is enabled by default. If this option is specified, length filtering is disabled.": "长度过滤默认是启用的。如果指定了这个选项，长度过滤就会被禁用。",
    "Reads shorter than this value will be discarded. Default is 15.": "比这个值短的读取将被丢弃。默认值是15。",
    "Reads longer than this value will be discarded. Default is 0 and means no limitation.": "比这个值长的序列将被丢弃。默认值是0，意味着没有限制。",
    "The complexity is defined as the percentage of base that is different from its next base, default is No": "复杂性定义为与下一个碱基不同的碱基的百分比，默认为否。",
    "Threshold for low complexity filter (0~100). Default is 30, which means 30% complexity is required.": "低复杂性过滤器的阈值（0~100）。默认值是30，这意味着需要30%的复杂性。",
    "This feature is enabled for NextSeq/NovaSeq data by default. NextSeq/NovaSeq data is detected by the machine ID in the FASTQ records.": "这个特性默认情况下是为NextSeq/NovaSeq数据启用的。NextSeq/NovaSeq数据是通过FASTQ记录中的机器ID来检测的。",
    "Similar to polyG tail trimming. When polyG tail trimming and polyX tail trimming are both enabled, fastp will perform polyG trimming first, then perform polyX trimming. Disabled by default.": "类似于polyG尾部修剪。当polyG尾部修剪和polyX尾部修剪都启用时，fastp将首先执行polyG修剪，然后执行polyX修剪。默认情况下是禁用的。",
    "The minimum length to detect polyX in the read tail. 10 by default.": "在读取尾部检测polyX的最小长度。默认为10。",
    "Enable unique molecular identifer (UMI) preprocessing.": "启用独特的分子标识符（UMI）预处理。",
    "Specify the location of UMI, can be (index1/index2/read1/read2/per_index/per_read, default is none.": "指定UMI的位置，可以是（index1/index2/read1/read2/per_index/per_read，默认为无）。",
    "If the UMI is in read1/read2, its length should be provided.": "如果UMI在read1/read2中，应该提供其长度。",
    "If specified, an underline will be used to connect prefix and UMI (i.e. prefix=UMI, UMI=AATTCG, final=UMI_AATTCG). No prefix by default.": "如果指定了下划线，将使用下划线连接前缀和UMI（例如：前缀=UMI，UMI=AATTCG，最终结果=UMI_AATTCG）。默认情况下没有前缀。",
    "Enable per read cutting by quality in front (5'), default is disabled (WARNING: this will interfere deduplication for both PE/SE data).": "启用按质量对每个读取的前端（5'）进行切割，默认情况下是禁用的（警告：这将干扰成对/单端数据的去重）。",
    "Enable per read cutting by quality in tail (3'), default is disabled (WARNING: this will interfere deduplication for SE data).": "启用尾部（3'）按质量对每个读取进行切割（默认为禁用）（警告：这将干扰单端数据的去重）。",
    "The size of the sliding window for sliding window trimming, default is 4.": "滑动窗口修剪的滑动窗口大小，默认为4。",
    "The bases in the sliding window with mean quality below cutting_quality will be cut, default is Q20.": "滑动窗口中平均质量低于切割质量的碱基将被切割，默认值为Q20。",
    "Enable base correction in overlapped regions (only for PE data), default is disabled.": "在重叠区域启用基础校正（仅适用于成对末端数据），默认为禁用。",
    "fastp provides a QC report for the data Before and After filtering within a single HTML page, which enables comparison of the quality statistics changed by the preprocessing step directly": "fastp 提供了一个质量控制报告，该报告在单个 HTML 页面中展示了数据过滤前后的情况，这使得用户能够直接比较预处理步骤对质量统计数据的影响。",
    "The JSON report contains all the data visualized in the HTML report. The format of the JSON report is manually optimized to be easily readable by humans and is compatible with MultiQC": "JSON报告包含了HTML报告中可视化的所有数据。JSON报告的格式经过手动优化，易于人类阅读，并且与MultiQC兼容。",
    "Setting to '0' will return all sequences longer than the 'Minimal length'": "设置为'0'将返回所有比“最小长度”更长的序列",
    "See help section for an explanation of dataset collections": "查看帮助部分以了解数据集集合的说明",
    "You can use the default settings or set custom values for any of Trim Galore!'s parameters.": "你可以使用默认设置，或者为Trim Galore!的任何参数设置自定义值。",
    "For more information please see below.": "有关更多信息，请见下文。",
    "Instead of performing adapter-/quality trimming, this option will simply hard-trim sequences to N bp at the 5'-end. Once hard-trimming of files is complete, it will exit": "而不是执行接头/质量修剪，这个选项将简单地在5'端硬修剪序列到N个碱基。一旦文件的硬修剪完成，它将退出",
    "Instead of performing adapter-/quality trimming, this option will simply hard-trim sequences to N bp at the 3'-end. Once hard-trimming of files is complete, it will exit": "而不是进行接头/质量修剪，这个选项将简单地在3'端将序列硬修剪到N个碱基。一旦文件的硬修剪完成，它将退出。",
    "In this mode, reads are trimmed in a specific way that is currently used for the Mouse Epigenetic Clock": "在这种模式下，读取数据被以一种特定的方式进行修剪，目前这种方式被用于小鼠表观遗传时钟。",
    "This is a new, still experimental, trimming mode to identify and remove poly-A tails from sequences": "这是一种新的、仍在实验阶段的修剪模式，用于识别并移除序列中的多聚腺苷酸（poly-A）尾部。",
    "Please use the samtools sort tool to ensure a correct BAM input": "请使用samtools sort工具确保BAM输入正确",
    "This preserves uniqueness when merging with other files with the same headers. Note: filename must not contain a '.' character": "这在与其他具有相同标题的文件合并时保留了唯一性。注意：文件名不能包含一个“.”字符。",
    "Allow the presences of reads in read2 input that are not present in read1 input. This allows cell barcode filtering of read1s without considering read2s": "允许在read2输入中存在不在read1输入中的读取。这允许在不考虑read2s的情况下对read1s进行细胞条形码过滤。",
    "This only applies if your barcode file has two columns output from the umi_tools whitelist command": "这只适用于您的条形码文件是从umi_tools whitelist命令输出的两列数据。",
    "Remove reads where any UMI base quality score falls below this threshold": "删除任何UMI碱基质量评分低于此阈值的读段",
    "If a UMI base has a quality below this threshold, replace the base with 'N'": "如果UMI碱基的质量低于这个阈值，用'N'替换该碱基",
    "Note: Sickle will infer the quality type of the file from its datatype.  I.e., if the datatype is fastqsanger, then the quality type is sanger. The default is fastqsanger.": "注意：Sickle 会根据文件的数据类型推断其质量类型。即，如果数据类型是 fastqsanger，那么质量类型就是 sanger。默认值是 fastqsanger。",
    "This will output only one file with all the reads, where the reads that did not pass filter will be replaced with a single 'N', rather than discarded.": "这将只输出一个包含所有读取的文件，其中未通过筛选的读取将被一个单独的'N'替换，而不是被丢弃。",
    "Threshold for trimming based on average quality in a window (-q)": "基于窗口中平均质量的修剪阈值（-q）",
    "Threshold to keep a read based on length after trimming (-l)": "在修剪后基于长度保留读取的阈值（-l）",
    "Many published protocols rank CBs by the number of reads the CBs appear in. However you could also use the number of unique UMIs a CB is associated with. Note that this is still and approximation to the number of transcripts captured because the same UMI could be associated with two different transcripts and be counted as independent": "许多已发布的协议通过CBs（捕获的转录本）出现的读段数对它们进行排序。然而，您也可以使用与CB相关联的独特UMIs（唯一分子标识符）的数量。请注意，这仍然是对捕获的转录本数量的近似，因为相同的UMI可能与两个不同的转录本相关联，并被视为独立的。",
    "Will still output the plots if requested": "如果请求了，仍然会输出图表。",
    "which may be sequence errors from another CB": "这可能是来自另一个CB的序列错误",
    "The minimum required overlap length between two reads to provide a confident overlap.": "两个读取之间提供可靠重叠所需的最小重叠长度。",
    "Maximum overlap length expected in approximately 90% of read pairs. Overlaps longer than the maximum overlap parameter are still considered as good overlaps, but the mismatch density is calculated over the first max_overlap bases in the overlapped region rather than the entire overlap.": "在大约90%的读对中预期的最大重叠长度。超过最大重叠参数的重叠仍然被视为良好的重叠，但是在计算重叠区域的错配密度时，是针对重叠区域中前max_overlap个碱基而非整个重叠区域进行计算的。",
    "Maximum allowed ratio between the number of mismatched base pairs and the overlap length. Two reads will not be combined with a given overlap if that overlap results in a mismatched base density higher than this value.": "允许的最大不匹配碱基对数与重叠长度之比。如果某个重叠导致不匹配碱基密度高于该值，则不会将两个读取组合。",
    "FLASH uses the same parameters when trying each orientation. If a read pair can be combined in either orientation, the better-fitting one will be chosen using the same scoring algorithm that FLASH normally uses.": "FLASH 在尝试每种方向时使用相同的参数。如果一个读对可以在任一方向组合，将使用 FLASH 通常使用的相同评分算法选择更匹配的一个。",
    "FASTA dataset to get statistics.": "FASTA 数据集获取统计信息。",
    "This parameter is optional. If provided, it will be used for calculating the NG50 statistic.": "这个参数是可选的。如果提供了，它将被用来计算NG50统计量。",
    "FASTQ of input reads": "输入读段的FASTQ文件",
    "Keep only the best reads up to this many total bases": "保留最佳读取，直到达到这么多的总碱基",
    "Keep only this percentage of the best reads (measured by bases)": "只保留最佳读取的百分比（按碱基测量）",
    "Maximum length threshold": "最大长度阈值",
    "Minimum length threshold": "最小长度阈值",
    "Minimum mean quality threshold": "最小均值质量阈值",
    "Minimum window quality threshold": "最小窗口质量阈值",
    "Reference assembly in FASTA format": "参考组装序列的FASTA格式",
    "Reference Illumina reads in FASTQ format": "参考Illumina读取的FASTQ格式",
    "Weight given to the length score (default: 1)": "长度分数的权重（默认值：1）",
    "Weight given to the mean quality score (default: 1)": "赋予平均质量分数的权重（默认：1）",
    "Weight given to the window quality score (default: 1)": "赋予窗口质量分数的权重（默认：1）",
    "Trim non-k-mer-matching bases from start/end of reads": "从读取的开始/结束修剪非k-mer匹配的碱基",
    "split reads at this many (or more) consecutive non-k-mer-matching bases": "在连续的非k碱基匹配的碱基达到这么多（或更多）时，将读取分开",
    "size of sliding window used when measuring window quality (default: 250)": "滑动窗口大小，用于测量窗口质量（默认：250）",
    "Use this for hairpin barcoding. keep at 0 unless you know what you're doing": "用于发夹状条形码。除非您知道自己在做什么，否则保持为0。",
    "Should be of datatype \"fastq.gz\" or \"fasta\"": "应该是数据类型为 \"fastq.gz\" 或 \"fasta\"",
    "Should be of datatype \"fastq.gz\"or \"fasta\"": "应该是数据类型为“fastq.gz”或“fasta”",
    "Maximum allowed error rate (no. of errors divided by the length of the matching region).": "最大允许的错误率（错误数除以匹配区域的长度）。",
    "Do not allow indels in the alignments. That is, allow only mismatches. This option is currently only supported for anchored 5' adapters ('^ADAPTER') (default: both mismatches and indels are allowed).": "不允许对齐中的插入或缺失。也就是说，只允许不匹配。这个选项目前只支持锚定的5'适配器（'^ADAPTER'）（默认：允许不匹配和插入或缺失）。",
    "Try to remove adapters at most COUNT times. Useful when an adapter gets appended multiple times.": "尝试最多去除适配器 COUNT 次。当适配器被多次附加时很有用。",
    "Minimum overlap length. If the overlap between the adapter and the sequence is shorter than LENGTH, the read is not modified. This reduces the number of bases trimmed purely due to short random adapter matches.": "最小重叠长度。如果适配器与序列之间的重叠部分长度小于LENGTH，则不对读取序列进行修改。这减少了仅由于短的随机适配器匹配而被修剪的碱基数量。",
    "Interpret IUPAC wildcards in reads": "解释读取中的IUPAC通配符",
    "Interpret IUPAC wildcards in adapters.": "在适配器中解释IUPAC通配符。",
    "Check both the read and its reverse complement for adapter matches. If match is on reverse-complemented version, output that one. Default: check only read.": "检查读取及其反向互补序列以寻找适配器匹配。如果在反向互补版本上找到匹配，则输出该版本。默认：仅检查读取。",
    "Discard reads that contain the adapter instead of trimming them. Use the 'Minimum overlap length' option in order to avoid throwing away too many randomly matching reads!": "丢弃包含接头的读段，而不是修剪它们。使用“最小重叠长度”选项，以避免丢弃太多随机匹配的读段！",
    "Discard reads that do not contain the adapter.": "丢弃不包含接头的序列读取。",
    "Discard trimmed reads that are shorter than LENGTH.  Reads that are too short even before adapter removal are also discarded.": "丢弃长度小于LENGTH的修剪后的序列。在去除接头之前长度就不够的序列也会被丢弃。",
    "Discard trimmed reads that are longer than LENGTH.  Reads that are too long even before adapter removal are also discarded.": "丢弃比LENGTH更长的修剪后的读取序列。即使在去除接头之前就已经过长的读取序列也会被丢弃。",
    "Discard reads with more than this number of 'N' bases. A number between 0 and 1 is interpreted as a fraction of the read length.": "丢弃包含超过这个数量的'N'碱基的读段。介于0和1之间的数字被解释为读段长度的分数。",
    "Which of the reads in a paired-end read have to match the filtering criterion in order for the pair to be filtered. Default: any": "在配对端读取中，为了使配对被过滤，必须满足过滤标准的读取是哪一个。默认值：任何",
    "Discard reads whose expected number of errors (computed from quality values) exceeds this value.": "丢弃那些根据质量值计算出的预期错误数超过这个值的读段。",
    "As --max-expected-errors (see above), but divided by length to account for reads of varying length": "作为 --max-expected-errors（见上文），但是除以长度以适应不同长度的读取",
    "Discard reads that did not pass CASAVA filtering (header has :Y:).": "丢弃未通过CASAVA过滤的读段（头部包含:Y:）。",
    "Remove bases from each read (first read only if paired). If positive, remove bases from the beginning. If negative, remove bases from the end. This is applied *before* adapter trimming.": "从每个读取的序列中移除碱基（如果是配对的，则仅移除第一个读取的碱基）。如果为正数，则从序列的开始处移除碱基。如果为负数，则从序列的末端移除碱基。这会在 *适配器修剪* 之前应用。",
    " Trim low-quality bases from 5' and/or 3' ends of each read before adapter removal. Applied to both reads for paired-end data, unless a separate value for the second read is specified. If one value is given, only the 3' end is trimmed. If two comma-separated cutoffs are given, the 5' end is trimmed with the first cutoff, the 3' end with the second.": "在去除接头序列之前，从每个读段的5'端和/或3'端修剪低质量的碱基。对于成对末端数据，该操作应用于两个读段，除非为第二个读段指定了单独的值。如果给定一个值，则仅修剪3'端。如果给定两个用逗号分隔的截止值，则5'端使用第一个截止值进行修剪，3'端使用第二个截止值进行修剪。",
    "Experimental option for quality trimming of NextSeq data. This is necessary because that machine cannot distinguish between G and reaching the end of the fragment (it encodes G as ‘black’). This option works like regular quality trimming (where one would use -q 20 instead), except that the qualities of G bases are ignored.": "用于NextSeq数据质量修剪的实验选项。这是必要的，因为该机器无法区分G和片段末端（它将G编码为“黑色”）。此选项的工作方式类似于常规的质量修剪（其中可以使用-q 20代替），只是忽略了G碱基的质量。",
    "Trim N's on ends of reads.": "修剪读取序列两端的N碱基。",
    "Note, this trim poly-T 'heads' on R2": "注意，这个修剪poly-T '头部'在R2上",
    "Remove this suffix from read names if present.": "如果存在，请从读取名称中删除此后缀。",
    "If you want to remove a fixed number of bases from each read, use the –cut option instead.": "如果你想从每个读取中移除固定数量的碱基，请使用——cut选项。",
    "Shorten reads to this length. This modification is applied after adapter trimming.": "将读取缩短到这个长度。这个修改是在适配器修剪之后应用的。",
    "Search for TAG followed by a decimal number in the name of the read (description/comment field of the FASTA or FASTQ file). Replace the decimal number with the correct length of the trimmed read. For example, use --length-tag 'length=' to search for fields like 'length=123'.": "在读取名称（FASTA 或 FASTQ 文件的描述/注释字段）中搜索后跟小数点的 TAG，并将小数点后的数字替换为修剪后的读取长度。例如，使用 --length-tag 'length=' 来搜索像 'length=123' 这样的字段。",
    "This option can be used to rename both single-end and paired-end reads.": "这个选项可以用来重命名单端和双端测序的读段。",
    "Output of 'FASTX Statistics' tool": "'FASTX 统计'工具的输出",
    "Set position one nt downstream of 3'-end as crosslinked nucleotide. By default the crosslink site will be assumed to be one nt upstream of the 5'-end of the read.": "将3'端下游的一个核苷酸设置为交联核苷酸。默认情况下，交联位点将假定为读取的5'端上游的一个核苷酸。",
    "List of adapters adapter sequences which will be explicity searched against the library. It should be a tab-delimited file with 2 columns: name and sequence.": "适配器序列列表，这些序列将被明确地针对库进行搜索。它应该是一个制表符分隔的文件，有两列：名称和序列。",
    "Using this option will cause fastqc to crash and burn if you use it on really long reads, and your plots may end up a ridiculous size. You have been warned!": "使用此选项可能会导致fastqc在处理非常长的读取时崩溃，并且您的图表可能会变得非常庞大。您已经被警告了！",
    " As long as you set this to a value greater or equal to your longest read length then this will be the sequence length used to create your read groups.  This can be useful for making directly comaparable statistics from datasets with somewhat variable read lengths.": "只要您将此值设置为等于或大于最长的读取长度，那么这将用于创建您的读取组的序列长度。这对于制作直接可比较的统计数据从具有某些可变读取长度的数据集非常有用。",
    "Note: the Kmer test is disabled and needs to be enabled using a custom Submodule and limits file": "注意：Kmer测试已禁用，需要使用自定义子模块和限制文件来启用。",
    "Starting at 5'-end. X positions will be moved to a separate file, N positions will be kept.": "从5'端开始。X位置将移动到一个单独的文件中，N位置将保留。",
    "Remove all crosslinking events that combine less reads than this fraction of the maximum number of reads at this position.": "移除所有连接事件，这些事件组合的读取数少于该位置最大读取数的这个分数。",
    "where are the barcodes.": "条形码在哪里？",
    "If empty, upload or import a SAM/BAM dataset": "如果为空，上传或导入一个 SAM/BAM 数据集",
    "REMOVE_DUPLICATES; default=False": "移除重复项；默认=False",
    "ASSUME_SORTED; default=True": "假设已排序；默认为True",
    "BARCODE_FILE. Pre-defined list of Unique Molecular Identifiers that can be expected. Format: one column text file, one UMI per line. All UMIs MUST have the same length.": "条形码文件。预期中的独特分子标识符（UMI）的预定义列表。格式：单列文本文件，每行一个UMI。所有UMI的长度必须相同。",
    "SLOTS. The last position is considered by default (-1). See help below.": "插槽。默认情况下，最后一个位置被考虑（-1）。详见下面的帮助。",
    "MAX_NUMBER_OF_N. Above this value, reads are placed in a 'undefined' group. Default value is the MISMATCHES number.": "最大N的数目。超过这个值，读取被放入一个“未定义”的组。默认值是错配数。",
    "TSLOTS. Value for SLOTS is considered by default. See help below": "插槽数。默认情况下会考虑SLOTS的值。详见下面的帮助信息。",
    "You can provide multiple comments": "您可以提供多个注释",
    "DUPLICATE_SCORING_STRATEGY; default=SUM_OF_BASE_QUALITIES": "重复评分策略；默认=碱基质量之和",
    "READ_NAME_REGEX; Read names are parsed to extract three variables: tile/region, x coordinate and y coordinate. These values are used to estimate the rate of optical duplication in order to give a more accurate estimated library size. See help below for more info; default=[a-zA-Z0-9]+:[0-9]:([0-9]+):([0-9]+):([0-9]+).*.": "读取名称正则表达式；读取名称被解析以提取三个变量：平铺/区域、x坐标和y坐标。这些值被用来估计光学重复率，以给出更准确的估计库大小。有关更多信息，请参见下面的帮助；默认=[a-zA-Z0-9]+:[0-9]:([0-9]+):([0-9]+):([0-9]+).*。",
    "OPTICAL_DUPLICATE_PIXEL_DISTANCE; default=100": "光学重复像素距离；默认值=100",
    "Setting stringency to SILENT can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) do not otherwise need to be decoded.": "将严格性设置为SILENT可以在处理BAM文件时提高性能，当不需要解码可变长度数据（读取、质量、标签）时。",
    "where are the barcodes. If not using paired-end it does not matter what you specify here.": "条形码在哪里。如果不使用配对末端测序，这里指定什么并不重要。",
    "The 'merge all' mode produces one output FASTA for all input FASTA files. The individual mode generates one FASTA file for each set of input FASTAs. For example, if the tool is given 2 collections of 10 FASTAs, it will merge the collections pairwise to create an output collection of 10 FASTAs.": "\"合并所有\"模式会将所有输入的FASTA文件合并为一个输出FASTA文件。单独模式则为每组输入FASTA文件生成一个FASTA文件。例如，如果工具被赋予了2组共10个FASTAs，它将成对合并这些组以创建一个包含10个FASTAs的输出集合。",
    "Regular expression with 1 capture group; the capture group is the accession (which must be unique)": "正则表达式包含1个捕获组；捕获组是访问号（必须是唯一的）",
    "Mutliple sequence file in FASTA, ClustalW or PHYLIP format.": "多序列文件，支持FASTA、ClustalW或PHYLIP格式。",
    "Tables from the NCBI, these determine the start and stop codons": "来自NCBI的表格，这些表格决定了起始和终止密码子",
    "FASTA format, using same identifiers as your protein alignment": "FASTA格式，使用与您的蛋白质对齐相同的标识符",
    "(—k kmer_length genome_size alpha)": "（—k k-mer长度 genome_size 基因组大小 alpha）",
    "(—K kmer_length genome_size)": "（—K kmer长度 基因组大小）",
    "Nucleotide-space: Must have PHRED-scaled quality values with offset 33 (fastqsanger).": "核苷酸空间：必须具有偏移量为33的PHRED质量值（fastqsanger）。",
    "Default is space ' ' for Illumina reads": "默认情况下，Illumina读取的分隔符是空格 ' '",
    "Target specific sequence by header, optionally with coordinates: header[:start-end]": "根据头标指定特定序列，可选地包含坐标：头标[:起始-结束]",
    "Generates output on a subset list of headers or coordinates in 0-based bed format. It can be combined with --exclude-bed. Optional.": "生成基于0的bed格式的标头或坐标子集列表的输出。它可以与--exclude-bed结合使用。可选。",
    "Exclude a subset of headers or coordinates in 0-base bed format. It can be conmbined with --include-bed. Optional.": "排除0-base bed格式的一个头文件或坐标子集。它可以与--include-bed参数结合使用。可选。",
    "Set of instructions provided as an ordered list": "提供为有序列表的指令集",
    "Outputs selected sequences.": "输出选定的序列。",
    "In the graph space, an assembly is a collection of segments and edges/gaps between these segments. A path defines a potential walk through the segments and edges/gaps that corresponds to a hypothesis of the actual linear sequence.": "在图形空间中，一个组装是一系列片段以及这些片段之间的边/间隙的集合。一条路径定义了通过这些片段和边/间隙的潜在路径，它对应于实际线性序列的假设。",
    "Specify how to sort the sequences. Ascending/descending uses the sequence/path header.": "指定如何对序列进行排序。升序/降序使用序列/路径标题。",
    "Compress all the homopolymers in the input above the given threshold.": "压缩输入中超过给定阈值的所有同聚物。",
    "Generate a tabular file with the sequence sizes": "生成一个包含序列大小的表格文件",
    "Generates bed coordinates of given feature. Default: AGP": "生成给定特征的bed坐标。默认：AGP",
    "Estimated genome size. This parameter is optional, but required for NG* statistics.": "估计的基因组大小。这个参数是可选的，但对于NG*统计是必需的。",
    "It reports also the actual sequence": "它还报告了实际的序列",
    "Use commas for thousands separator in output?": "在输出中使用逗号作为千位分隔符？",
    "Generate output in tabular format": "生成以表格格式输出",
    "In the graph space an assembly is a collection of segment and edges/gaps between these segments. A path defines a potential walk through the segments and edges/gaps that corresponds to a hypothesis of the actual linear sequence": "在图形空间中，组装是一系列片段以及这些片段之间的边/间隙的集合。一条路径定义了通过这些片段和边/间隙的潜在路径，这对应于实际线性序列的假设。",
    "Integrate the AGP information into the assembly graph as paths. The scaffolding information is converted into paths through the graph": "将AGP信息整合到组装图中作为路径。将支架信息转换为图中的路径。",
    "This will run FilterUncorrectabledPEfastq and remove uncorrectable reads.": "这将运行 FilterUncorrectabledPEfastq 并移除无法校正的读取。",
    "(smaller 33, default: 23)": "（较小 33，默认值：23）",
    "the maximum number of correction within k-bp window (default: 4)": "在k-bp窗口内的最大校正数（默认值：4）",
    "the proportion of kmers that are used to estimate weak kmer count threshold, lower for more divergent genome (default: 0.95)": "用于估计弱kmer计数阈值的比例，对于更加多样化的基因组，这个比例更低（默认值：0.95）",
    "expected_number_of_kmers: does not affect the correctness of program but affect the memory usage (default: 100000000)": "预期的kmer数量：不会影响程序的正确性，但会影响内存使用（默认值：100000000）",
    "Nucleotide-space: Must have PHRED-scaled quality values. (-f and -r)": "核苷酸空间：必须具有PHRED比例的质量值。（-f 和 -r）",
    "If the computed p-value of a possible assembly exceeds the specified p-value then the paired-end read will not be assembled. Setting 1.0 disables the test. (--p-value)": "如果计算出的组装p值超过了指定的p值，则不会对配对末端读取进行组装。设置为1.0将禁用该测试。（--p-value）",
    "The minimum overlap may be set to 1 when the statistical test is used. However, further restricting the minimum overlap size to a proper value may reduce false-positive assembles. (--min-overlap)": "当使用统计测试时，最小重叠可以设置为1。然而，将最小重叠大小进一步限制为适当的值可以减少错误的组装。（--min-overlap）",
    "Setting this value to 0 disables the restriction and assembled sequences may be arbitrary long. (--max-assembly-length)": "将此值设置为0将禁用限制，组装序列可以任意长。(--max-assembly-length)",
    "Setting this value to 0 disables the restriction and assembled sequences may be arbitrary short. (--min-assembly-length)": "将此值设置为0将禁用限制，组装的序列可能会任意短。(--min-assembly-length)",
    "See option -q. (--min-trim-length)": "参见选项 -q。（--min-trim-length）",
    "If the quality scores of two consecutive bases are strictly less than the specified threshold, the rest of the read will be trimmed. (--quality-threshold)": "如果两个连续碱基的质量分数严格低于指定的阈值，则读取的其余部分将被修剪。（--quality-threshold）",
    "Setting this value to 0 will cause PEAR to discard all reads containing uncalled bases. The other extreme setting is 1 which causes PEAR to process all reads independent on the number of uncalled bases. (--max-uncalled-base)": "将此值设置为0将导致PEAR丢弃所有包含未调用碱基的读取。另一个极端设置是1，这将导致PEAR处理所有读取，而不考虑未调用碱基的数量。(--max-uncalled-base)",
    "If set to zero, capping is disabled. (--cap)": "如果设置为零，封顶功能将被禁用。(--cap)",
    "When  merging a base-pair that consists of two non-equal bases out of which none is degenerate, set the merged base to N and use the highest quality score of the two bases. (--nbase)": "当合并一个由两个不等的碱基对组成的碱基对时，其中没有一个是简并碱基，将合并后的碱基设置为N，并使用两个碱基中最高的质量分数。（--nbase）",
    "Includes quartiles of seq length, sum_gap, N50": "包括序列长度的四分位数、总间隙数和N50值。",
    "Only show warning messages": "只显示警告信息",
    "Possible values are 0 to 93 for Phred+33 encoded files": "可能的值是0到93，用于Phred+33编码的文件。",
    "The first two columns of each line in the file must correspond to values passed to --adapter1 and --adapter2": "文件中每一行的前两列必须对应于传递给 --adapter1 和 --adapter2 的值",
    "Ignored unless input files are single-end reads": "除非输入文件是单端读取，否则忽略",
    "The value is used directly if less than 1, the rate s set to 1 / value if the value is greater than 1": "如果值小于1，则直接使用该值；如果值大于1，则速率s设置为1除以该值",
    "If quality trimming is also enabled (--trimqualities), then stretches of mixed low-quality bases and/or Ns are trimmed": "如果同时启用了质量修剪（--trimqualities），那么混合低质量碱基和/或N的连续片段将被修剪",
    "If trimming of Ns is also enabled (--trimns), then stretches of mixed low-quality bases and Ns are trimmed": "如果同时启用了N的修剪（--trimns），那么混合低质量碱基和N的连续序列将被修剪",
    "See the Window based quality trimming section of the tool help below": "查看工具帮助中基于窗口的质量修剪部分",
    "Collapsed reads will not be quality trimmed when this option is enabled": "启用此选项时，合并后的读取将不会进行质量修剪",
    "Setting this option also sets --collapse": "设置此选项也会设置 --collapse",
    "Specify a pattern/motif sequence or a FASTA file with the motif of interest. Motifs could be EITHER plain sequence containing 'ACTGN' OR regular expression like 'A[TU]G(?:.{3})+?[TU](?:AG|AA|GA)' for ORFs": "指定一个模式/基序序列或包含感兴趣的基序的FASTA文件。基序可以是包含'ACTGN'的纯序列，或者是正则表达式，如用于开放阅读框（ORFs）的'A[TU]G(?:.{3})+?[TU](?:AG|AA|GA)'。",
    "Perl regular expressions are allowed": "Perl 正则表达式是允许的",
    "For automatic detection, it automatically detect by the first sequence. Default: auto": "对于自动检测，它会自动根据第一个序列进行检测。默认值：自动",
    "Default: 10000": "默认值：10000",
    "When using flag --circular, end position of matched subsequence that crossing genome sequence end would be greater than sequence length": "当使用标志 --circular 时，跨越基因组序列末端的匹配子序列的末端位置将大于序列长度。",
    "Do not use degenerate bases/residues in regular expression, you need convert them to regular expression, e.g., change 'N' or 'X'  to '.'": "不要在正则表达式中使用简并碱基/残基，你需要将它们转换为正则表达式，例如，将'N'或'X'更改为'.'",
    "For large genomes like human genome, using mapping/alignment tools would be faster": "对于像人类基因组这样的大型基因组，使用映射/比对工具会比较快。",
    "Faster, but muy miss motifs overlapping with others": "更快，但很多重叠的基序",
    "Use FM-index for much faster search of lots of sequence patterns. This option is not compatible with the --degenerate option": "使用 FM-index 可以更快地搜索许多序列模式。这个选项与 --degenerate 选项不兼容。",
    "Example: >gi|110645304|ref|NC_002516.2| Pseud...": "示例：>gi|110645304|ref|NC_002516.2| 伪...",
    "If selected, the tool with find the overlaps as if merging reads, use this overlap information to correct sequencing errors on both strands. However, the strands from the mates will not be merged but provided as two separate reads after error correction.": "如果选中，该工具将像合并读取一样找到重叠部分，使用此重叠信息来纠正两个链上的测序错误。然而，来自配对的链不会合并，而是在错误校正后作为两个单独的读取提供。",
    "Lower is more strict, -1 disables.": "更低意味着更严格，-1表示禁用。",
    "Higher is stricter. 0 will disable the filter; 1 will allow only perfect overlaps.": "更高则更严格。0将禁用过滤器；1将只允许完全重叠。",
    "Requires good coverage, 0 disables.": "需要良好的覆盖度，0表示禁用。",
    "If disabled, quality values are completely ignored, both for overlap detection and filtering. May be useful for data with inaccurate quality values.": "如果禁用，质量值将完全被忽略，无论是用于重叠检测还是过滤。对于具有不准确质量值的数据可能很有用。",
    "In the ratio mode, overlaps are decided based on the ratio of matching to mismatching bases. Flat mode scores overlaps based on the total number of mismatching bases only.": "在比率模式下，重叠部分是基于匹配与不匹配碱基的比率来决定的。平面模式则仅基于不匹配碱基的总数来评分重叠部分。",
    "Software name": "软件名称",
    "It should contain 'Stats for BAM file(s)'": "它应该包含“BAM文件的统计信息”",
    "It should contain 'This file was produced by bcftools stats'": "它应该包含“此文件由 bcftools stats 生成”",
    "It should contain 'reads; of these:'": "它应该包含“reads; 其中：”",
    "It should contain 'This is cutadapt' or 'You are running cutadapt'": "它应该包含“这是一个cutadapt”或“你正在运行cutadapt”",
    "It should be the json report from fastp containing 'report_title'": "它应该是来自fastp的json报告，包含'report_title'。",
    "It should contain 'Flexbar - flexible barcode and adapter removal'": "它应该包含“Flexbar - 灵活的条形码和接头去除”",
    "It should contain 'Min rest. site distance'": "它应该包含“最小休息位点距离”",
    "It should contain 'HISAT2 summary stats:'": "它应该包含“HISAT2 摘要统计：”",
    "It should contain '__too_low_aQual'": "它应该包含 '__too_low_aQual'",
    "It should contain '[quant] finding pseudoalignments for the reads'": "它应该包含'[quant] 为读取寻找伪对齐'",
    "It should be a tabular file containing '# This file is generated by MACS'": "它应该是一个包含“# This file is generated by MACS”的表格文件",
    "It should contain 'contigs:'": "它应该包含“contigs:”",
    "First result input should be genome_coverage (default from tool output). Run the Flatten Collection tool to keep all output in one report": "第一个结果输入应该是基因组覆盖度（默认来自工具输出）。运行Flatten Collection工具以将所有输出保存在一个报告中。",
    "It should contain 'Proper-paired reads map to different chrom:'": "它应该包含“正确配对的读段映射到不同的染色体：”",
    "It should contain 'Fraction of reads explained by'": "它应该包含“由...解释的读段比例”",
    "It should contain 'Partial Novel Splicing Junctions:'": "它应该包含“部分新剪接连接点：”",
    "It should contain 'Group Total_bases Tag_count Tags/Kb'": "它应该包含 'Group Total_bases Tag_count Tags/Kb'",
    "It should contain 'samblaster: Version'": "它应该包含“samblaster: 版本”",
    "It should contain 'This file was produced by samtools stats'": "它应该包含“此文件由samtools stats生成”",
    "It should contain 'in total (QC-passed reads + QC-failed reads)'": "它应该包含“总共（QC通过的读段数 + QC未通过的读段数）”",
    "It should contain '[bam_rmdup'": "它应该包含 '[bam_rmdup'",
    "It should contain '# slamdunk'": "它应该包含“# slamdunk”",
    "It should contain 'Minimal SW score based on E-value'": "它应该包含“基于E值的最小SW得分”",
    "It should contain 'TTrimmomatic'": "它应该包含'TTrimmomatic'",
    "Nice name used for the report section header": "报告部分标题的优美名称",
    "Introductory text to be printed under the section header": "在小节标题下打印的介绍性文本",
    "It is printed as page header": "它被打印为页眉",
    "It will be printed at the top of the report": "它将打印在报告的顶部",
    "To consume less disk space and resources to display. Used by default when there are 100 or more samples.": "为了减少显示时占用的磁盘空间和资源。默认情况下，当有100个或更多样本时使用。",
    "This will output a file for each plot containing the raw data used to generate the plot": "这将为每个图表输出一个文件，其中包含用于生成图表的原始数据。",
    "This is mostly useful for debugging purposes": "这主要用于调试目的",
    "Select between paired and single end data": "选择配对端和单端数据",
    "If this option is disabled, it won't fail if the quality encoding cannot be determined": "如果禁用此选项，如果无法确定质量编码，则不会失败",
    "By using this option no checks are made to determine if the read names/identifiers are unique (it will run faster and use less memory)": "通过使用此选项，不会进行检查以确定读取名称/标识符是否唯一（它将运行得更快并使用更少的内存）",
    "The non-internal 5’ and 3’ adapter types disallow internal occurrences of the adapter sequence. The adapter must always be at one of the ends of the read, but partial occurrences are also ok.": "非内部5'和3'的接头类型不允许接头序列在内部出现。接头必须始终位于读取的一端，但部分出现也是可以的。",
    "Allow 'N's as matches. Default: in the adapters but not in the reads": "允许'N's作为匹配项。默认情况下：在适配器中但不在读取中",
    "When trimming paired-end reads, the minimum/maximum lengths for R1 and R2 can be specified separately. If not provided, the same minimum length applies to both reads.": "当修剪成对的末端读取时，可以分别为R1和R2指定最小/最大长度。如果没有提供，则相同的最小长度适用于两个读取。",
    "To detect contamination from foreign organisms, a genome assembly in a fasta file.": "为了检测外来生物的污染，需要一个在fasta文件中的基因组组装。",
    "The appropriate tax-id for your genome assembly.  The appropriate tax-id for an organism can be retrieved from the NCBI Taxonomy website.": "您的基因组组装所需的适当物种编号（tax-id）。可以通过NCBI分类学网站检索到一个生物体的适当物种编号。",
    "Multiple tax-ids may be provided as a comma-separated list.": "多个分类ID可以作为逗号分隔的列表提供。",
    "Multiple gx-divisions may be selected.": "可以选择多个Galaxy工作区。",
    "from 'NCBI BLAST name' on taxon Info page": "从分类信息页面上的“NCBI BLAST名称”",
    "Select the FASTQ file containing the first set of paired-end reads.": "选择包含第一组配对端读取的FASTQ文件。",
    "Select the FASTQ file containing the second set of paired-end reads.": "选择包含第二组配对端读取的FASTQ文件。",
    "Enable this to merge overlapping reads from the provided paired-end FASTQ files.": "启用这个选项可以合并提供的配对端FASTQ文件中的重叠读取。",
    "first read discarded fastq filename": "第一个读取丢弃的fastq文件名",
    "second read discarded fastq filename": "第二个读取被丢弃的fastq文件名",
    "Input sequence is in phred+64 rather than phred+33 format, the output will still be phred+33": "输入序列使用的是phred+64格式而非phred+33格式，输出仍将为phred+33格式。",
    "Quality score cutoff for mismatches to be counted in overlap": "不匹配的重叠区域中考虑计数的质量分数截断值",
    "Minimum length of a trimmed or merged read to print it": "打印修剪或合并后的读取序列的最小长度",
    "Forward read primer/adapter sequence to trim as it would appear at the end of a read": "前向读取引物/适配器序列，修剪时它将出现在读取的末端",
    "Reverse read primer/adapter sequence to trim as it would appear at the end of a read": "将反向读取引物/适配器序列反转，以便在读取的末端进行修剪",
    "minimum overall base pair overlap with adapter sequence to trim": "与适配器序列最小整体碱基对重叠以进行修剪",
    "maximum fraction of good quality mismatching bases for primer/adapter overlap": "对于引物/接头重叠，优质不匹配碱基的最大比例",
    "minimum fraction of matching bases for primer/adapter overlap": "引物/接头重叠的匹配碱基的最小比例",
    "Adapter alignment minimum local alignment score cutoff [roughly (2*num_hits) - (num_gaps*gap_open) - (num_gaps*gap_close) - (gap_len*gap_extend) - (2*num_mismatches)]": "适配器比对的最小局部比对分数截断值【大致为 (2*num_hits) - (num_gaps*gap_open) - (num_gaps*gap_close) - (gap_len*gap_extend) - (2*num_mismatches)】",
    "read alignment maximum fraction gap cutoff": "读取比对最大间隙分数截止值",
    "Maximum quality score in output (phred 33), default = ']'": "输出中的最大质量分数（phred 33），默认值 = ']' 注：此处的 '[]' 应为参数，保持原文不做翻译。",
    "Use this if reads are different lengths": "如果读取长度不同，请使用此选项。",
    "Minimum overall base pair overlap to merge two reads": "最小整体碱基对重叠以合并两个读取",
    "Maximum fraction of good quality mismatching bases to overlap reads": "最大好质量不匹配碱基与重叠读取的比例",
    "Minimum fraction of matching bases to overlap reads": "匹配碱基的最小比例以重叠读取",
    "Specify dataset with forward reads": "指定包含正向读取的数据集",
    "Specify dataset with reverse reads": "指定包含反向读取的基因组数据集",
    "Specify dataset with single reads": "指定单端测序数据集",
    "Cut adapter and other illumina-specific sequences from the read": "从读取中剪切适配器和其他特定于Illumina的序列。",
    "Write sequences in the fasta format.": "将序列以fasta格式写入。",
    "See help below": "请看下面的帮助",
    "Bases at the start of the read with quality below the threshold will be removed": "读取开始处质量低于阈值的碱基将被移除",
    "Bases at the end of the read with quality below the threshold will be removed": "低于阈值的读取末端碱基将被移除",
    "The read length which is likely to allow the location of the read within the target sequence to be determined.": "读取长度，可能允许确定读取在目标序列中的位置。",
    "Set between zero and one - specifies the balance between preserving read length versus removal of incorrect bases; low values (<0.2) favours longer reads, high values (>0.8) favours read correctness.": "设置在0到1之间 - 指定在保留读取长度与去除错误碱基之间的平衡；低值（<0.2）有利于更长的读取，高值（>0.8）有利于读取的正确性。",
    "The phred+64 encoding works the same as the phred+33 encoding, except you add 64 to the phred score to determine the ascii code of the quality character. You will only find phred+64 encoding on older data, which was sequenced several years ago. FASTQC can be used in order to identify the encoding type.": "Phred+64编码与Phred+33编码的工作方式相同，只是你需要将Phred分数加上64，以确定质量字符的ASCII码。你只能在几年前测序的旧数据上找到Phred+64编码。可以使用FASTQC来识别编码类型。",
    "these are the messages written to stderr (eg. for use in MultiQC)": "这些是写入标准错误(stderr)的消息（例如，用于MultiQC）",
    "-I; This parameter is only used for paired reads. Only mean is required while sd, max, and min will be inferred. Examples: both \"250\" and \"250,25\" will work while \"250,,10\" will not. See below for details.": "-I; 这个参数仅用于配对读取。只需要指定平均值，标准差、最大值和最小值将被推断出来。示例：\"250\" 和 \"250,25\" 都是有效的，而 \"250,,10\" 则无效。详细信息请参见下文。",
    "Specify dataset with interleaved reads": "指定带有交错读取的数据集",
    "Sets -k, -w, -d, -r, -y, -c, -D, -W, -m, -S, -P, and -e options.": "设置 -k、-w、-d、-r、-y、-c、-D、-W、-m、-S、-P 和 -e 选项。",
    "Sets -A, -B, -O, -E, -L, and -U options.": "设置 -A、-B、-O、-E、-L 和 -U 选项。",
    "Sets -T, -h, -a, -C, -V, -Y, and -M options.": "设置 -T、-h、-a、-C、-V、-Y 和 -M 选项。",
    "-a; These alignments will be flagged as secondary alignments": "-a; 这些比对将被标记为次要比对",
    "-Y; By default, BWA-MEM uses soft clipping for the primary alignment and hard clipping for supplementary alignments": "-Y; 默认情况下，BWA-MEM 使用软剪切处理主要比对，而对补充比对使用硬剪切。",
    "-M; For Picard<1.96 compatibility": "-M; 为了兼容Picard<1.96版本",
    "Select between fastq and bam datasets and between paired and single end data": "选择fastq和bam数据集，以及成对和单端数据",
    "Specify BAM dataset with paired reads": "指定带有配对读取的BAM数据集",
    "Specify BAM dataset with single reads": "指定包含单次读取的BAM数据集",
    "aln -n; default=0.04": "aln -n; 默认值=0.04",
    "aln -o; default=1": "aln -o; 默认值=1",
    "aln -e; -1 disables long gaps and invokes k-difference mode; default=-1": "aln -e; -1 禁用长间隙并调用 k-差异模式；默认值为 -1",
    "aln -i; default=5": "aln -i; 默认值=5",
    "aln -d; default=10": "aln -d; 默认值=10",
    "aln -l; default=32": "aln -l; 默认值=32",
    "aln -k; default=2": "aln -k; 默认值=2",
    "aln -m; default=2000000": "aln -m; 默认=2000000",
    "aln -M; default=3": "aln -M; 默认=3",
    "aln -O; default=11": "aln -O; 默认值=11",
    "aln -E; default=4": "aln -E; 默认值=4",
    "aln -R; default=30": "aln -R; 默认值=30",
    "aln -q; default=0": "aln -q; 默认值=0",
    "aln -B; optional parameter": "aln -B; 可选参数",
    "aln -L; optional parameter": "aln -L；可选参数",
    "Size of best perfect match (-word_size)": "最佳完美匹配的大小（-word_size）",
    "no cutoff if 0": "如果为0，则无截止值",
    "in FASTA format": "在FASTA格式中",
    "If your genome of interest is not listed, contact your Galaxy admin": "如果您感兴趣的基因组没有列出，请与您的Galaxy管理员联系",
    "FASTQ file.": "FASTQ文件。",
    "Must have a fastqsanger datatype.": "必须具有fastqsanger数据类型。",
    "If desired, you can manually add read group information to the resulting BAM file. To do so, you MUST manually specify the entire string, such as '@RG\\tID:foo\\tSM:bar'": "如果需要，你可以手动向生成的BAM文件添加读取组信息。为此，你必须手动指定整个字符串，例如'@RG\\tID:foo\\tSM:bar'",
    "These are the sequences that you are aligning against TARGET": "这些是你正在比对到目标上的序列",
    "It is highly recommended to use lastz_32 instead of lastz if the reference genome size is greater than 2G": "强烈建议，如果参考基因组大小超过2G，使用lastz_32而不是lastz。",
    "Search both strands or choose plus or minus": "搜索两条链或选择正链或负链",
    "Computation is more efficient than it would be without this option, since only one of each mirror-image pair of alignment blocks is processed (the other, redundant one is skipped during processing, but re-created in the output). Also, the trivial self-alignment block along the main diagonal is omitted from the output. THIS OPTION CANNOT BE USED IF THE TARGET IS COMPRIZED OF MULTIPLE SEQUENCES": "计算比没有这个选项时更高效，因为只处理每个镜像对齐块中的一个（另一个冗余的在处理过程中被跳过，但在输出中重新创建）。此外，主对角线上的平凡自对齐块也从输出中省略。如果目标由多个序列组成，则不能使用此选项。",
    "Output consists of only one copy of each meaningful alignment block in a self-alignment. This option is only applicable when the ‑‑self option is used.": "输出仅包含自对齐中每个有意义的对齐块的一份副本。此选项仅在使用了‑‑self选项时适用。",
    "Any queries that exceed this limit are reported as a warning (to stderr), and no alignments are reported. This is useful for mapping reads to a reference genome, when some reads align to too many places in the reference.": "任何超过此限制的查询都会作为警告（到标准错误stderr）报告，并且不会报告任何比对。这对于将读取映射到参考基因组非常有用，因为有些读取在参考中比对到太多地方。",
    "For such a query, the first N HSPs found are passed along to downstream processing. Note that the HSPs reported are not the best N HSPs. They are simply the first N found; they very likely have a positional bias.": "对于这样的查询，找到的前N个HSP（高得分片段）会传递到下游处理。请注意，报告的HSPs并不是最好的N个HSPs。它们只是找到的前N个；它们很可能存在位置偏差。",
    "This is useful for mapping reads to a reference genome, when some reads align to too many places in the reference.": "这对于将读取映射到参考基因组非常有用，当一些读取在参考中对齐到太多位置时。",
    "This is a real number that corresponds to a depth of coverage threshold. For example, a value of 5.0 would cause termination once a query/strand has an average of five alignments for every base in the query. The numerator is the number of matches or substitutions (but not gaps); the denominator is the length of the query sequence. The purpose of this option is one of saving time. It is useful for automatically terminating the processing of queries with high repeat content, for which other methods of dealing with repetitive content fail. Moreover, back-end filtering options are not considered. In other words, matches are counted for any alignment that meets the scoring threshold, regardless of whether that alignment would be reported. The justification is that we are trying to abort the processing of queries that have too many bounding alignments in the DP matrix, and back-end filtering occurs later in the process.": "这是一个实数，对应于覆盖度阈值。例如，值为5.0将导致一旦查询/链在查询中的每个碱基上平均有五个比对就会终止。分子是匹配或替换的数量（但不是间隙）；分母是查询序列的长度。这个选项的目的是节省时间。它对于自动终止具有高重复内容的查询处理非常有用，对于这些查询，处理重复内容的其他方法失败了。此外，不考虑后端过滤选项。换句话说，对于满足评分阈值的任何比对，都会计算匹配项，无论是否会报告该比对。理由是我们试图终止在DP矩阵中有太多边界比对的查询的处理，而后端过滤在流程中稍后发生。",
    "Note that the alignments reported are not guaranteed to be the highest scoring alignments that would achieve the threshold. They are simply the first alignments found. In other words, the purpose of this option is one of saving time, not one of finding optimal alignments.": "请注意，报告的比对不一定是达到阈值的最高分比对。它们只是首先找到的比对。换句话说，此选项的目的是节省时间，而不是寻找最优比对。",
    "This option cannot be used in conjunction with ‑‑match or inference.": "这个选项不能与‑‑match或推理一起使用。",
    "Note that specifying ‑‑match changes the defaults for some of the other options (e.g. the scoring penalties for gaps, and various extension thresholds), as described in respective sections of LASTZ manual. The regular defaults are chosen for compatibility with BLASTZ, but since BLASTZ doesn't support ‑‑match, LASTZ infers that you are not expecting BLASTZ compatibility for this run, so it is free to use improved defaults. THIS OPTION CANNOT BE USED WITH --scores": "请注意，指定‑‑match会改变其他一些选项的默认值（例如，间隙的得分惩罚和各种扩展阈值），如LASTZ手册中相应部分所述。常规默认值是为与BLASTZ兼容而选择的，但由于BLASTZ不支持‑‑match，LASTZ推断您不希望在此运行中使用BLASTZ兼容性，因此可以使用改进的默认值。此选项不能与--scores一起使用",
    "These are specified as positive values; subtraction is implicitly assumed. Note that the first base in a gap incurs the sum of both penalties. This option is only valid if gapped extension is being performed, and cannot be used in conjunction with inference. These values specified on the command line override any corresponding values from a file provided with ‑‑scores.": "这些指定为正值；默认假定为减法。请注意，间隔中的第一个碱基会产生两个惩罚的总和。如果正在进行带间隔的扩展，此选项才有效，并且不能与推理一起使用。在命令行上指定的这些值将覆盖使用‑‑scores提供的文件中的任何相应值。",
    "Substitutions with N are scored as zero, instead of using the fill_score value from the scoring file (which is -100 by default).": "将 N 的替换得分记为零，而不是使用评分文件中的 fill_score 值（默认为 -100）。",
    "Substitutions with these characters are scored as zero, instead of using the fill_score value from the scoring file (which is -100 by default).": "带有这些字符的替换得分为零，而不是使用来自评分文件的fill_score值（默认为-100）。",
    "Select between 12of19 seed (19-bp word with matches in 12 specific positions: 1110100110010101111) or 14of22 seed (22-bp word with matches in 14 specific positions: 1110101100110010101111). 0 = mismatch, 1 = match": "在12of19种子（19个碱基的词，其中12个特定位置匹配：1110100110010101111）和14of22种子（22个碱基的词，其中14个特定位置匹配：1110101100110010101111）之间选择。0 = 不匹配，1 = 匹配",
    "Seeds require a N-bp word with matches in all positions": "种子需要一个N-bp的词，在所有位置上都有匹配项",
    "Seeds requires N-bp word with matches or transitions in all positions": "种子需要在所有位置上具有匹配或转换的N个碱基的词",
    "Use 1s, 0s, and Ts for seed discovery (e.g., 1110100110010101111 where 1 = match, 0 = mismatch, T = transition)": "使用1、0和T进行种子发现（例如，1110100110010101111，其中1=匹配，0=不匹配，T=转换）",
    "In each seed, specify how many match positions can be transition(s) instead": "在每个种子中，指定有多少个匹配位置可以是转换（transition）",
    "Allowing no more than this number of transversions. If not specified, any number of transversions is allowed (they are not limited)": "允许的转换数量不超过这个数值。如果没有指定，则允许任意数量的转换（不限制转换的数量）。",
    "Require at least this many exact matches": "至少需要这么多完全匹配",
    "This will take into account other papermeters in this section": "这将考虑到本节中的其他纸张测量器。",
    "Instead, pass the seeds along to the next specified stage.It is not recommended to use --nogfextend without also using --nogapped.": "相反，将种子传递给下一个指定的阶段。不建议在不使用 --nogapped 的情况下使用 --nogfextend。",
    "This is instead of using the x-drop method": "这是代替使用 x-drop 方法的",
    "Find HSPs using the mismatch extension method with the given length threshold and allowing specified number of mismatches": "使用给定的长度阈值和允许的不匹配数，通过不匹配扩展方法找到HSP（高得分片段）",
    "The dropoff setting determines the endpoints of each gap-free segment: the extension of each seed is stopped when its cumulative score drops off by more than the given threshold from the maximum seen so far.": "dropoff 设置决定了每个无间隔片段的端点：当每个种子的累积分数从迄今为止看到的最大值下降超过给定的阈值时，其扩展就会停止。",
    "HSPs scoring lower are discarded": "得分较低的高扫描质谱（HSPs）被丢弃",
    "HSPs scoring lower are discarded. The score threshold is chosen to limit the number of target sequence bases in HSPs to about this value (or possibly a little higher in case of ties, etc.).": "得分较低的高得分片段（HSPs）被丢弃。选择得分阈值是为了将HSPs中的目标序列碱基数量限制在这个值左右（或者在平局等情况下可能稍高一些）。",
    "HSPs scoring lower are discarded. The score threshold is chosen to limit the number of target sequence bases in HSPs to about this perentage value (or possibly a little higher in case of ties, etc.).": "得分较低的高得分片段（HSPs）被丢弃。选择得分阈值是为了将HSPs中目标序列碱基的数量限制在这个百分比值左右（或者在平局等情况下可能稍高一些）。",
    "Those that score just slightly above the HSP threshold are adjusted downward according to the entropy of their nucleotides, and any that then fall below the threshold are discarded.": "那些得分略高于HSP阈值的序列会根据它们核苷酸的熵值向下调整，然后那些低于阈值的序列将被丢弃。",
    "Reported HSPs are printed to stderr": "报告的HSPs打印到标准错误输出",
    "These are specified as positive values; subtraction from the score is implicitly assumed.": "这些被指定为正值；默认情况下假定从分数中减去。",
    "Extension of HSPs (or seeds, if gap-free extension is not performed), is performed after first reducing them to anchor points.": "HSPs（或种子，如果不执行无间隙扩展）的扩展是在首先将其简化为锚点之后执行的。",
    "This means that interpolation must also be skipped, since it is not allowed without gapped extension": "这意味着插值也必须跳过，因为没有间隔扩展是不允许的",
    "This restricts the endpoints of each local alignment by limiting the local region around each anchor in which extension is performed. The boundary of this region in the dynamic programming matrix is formed by the points where the cumulative score has dropped off by more than the given threshold from the maximum seen so far.": "这通过限制每个锚点周围局部区域的扩展来限制每个局部比对的端点。在动态规划矩阵中，这个区域的边界是由累积分数从迄今为止看到的最大值下降超过给定阈值的点形成的。",
    "This is highly recommended when either the target or query sequences are short reads (say, less than 100 bases), to prevent y-drop mismatch shadow.": "当目标或查询序列是短读长（比如少于100个碱基）时，强烈推荐这样做，以防止y-drop不匹配阴影。",
    "Alignments scoring lower than that value are discarded. When used along with the x-drop method for gap-free extension, this value is generally set at least as high as the HSP threshold. Setting it lower has no effect, since at worst the HSP itself would always qualify (both extension stages use the same scoring matrix).": "比该值低的比对得分将被丢弃。当与无间隔延伸的x-drop方法一起使用时，此值通常至少设置为HSP阈值一样高。将其设置得更低没有效果，因为最坏情况下HSP本身总是符合条件（两个延伸阶段使用相同的得分矩阵）。",
    "0 ≤ min ≤ max ≤ 100 percent. Identity is the percentage of aligned bases that are matches. Alignment blocks outside the given range are discarded.": "0 ≤ 最小值 ≤ 最大值 ≤ 100 percent。一致性是比对基基数的百分比，与给定范围匹配。在给定范围之外的比对块将被丢弃。",
    "0 ≤ min ≤ max ≤ 100 percent. Continuity is the percentage of alignment columns that are not gaps. Alignment blocks outside the given range are discarded.": "0 ≤ 最小值 ≤ 最大值 ≤ 100 percent。连续性是比对列中非空位的比例。超出给定范围的比对块将被丢弃。",
    " 0 ≤ min ≤ max ≤ 100 percent. Coverage is the percentage of the entire target or query sequence (whichever is shorter) that is included in the alignment block. Blocks outside the given range are discarded.": "0 ≤ 最小值 ≤ 最大值 ≤ 100 percent。覆盖率是指在整个目标或查询序列（两者中较短的一个）中包含在比对块中的百分比。超出给定范围的块将被丢弃。",
    "Requiring at least this number of matched bases, min > 0. Match count is the number of matched bases in the alignment.": "要求至少匹配这么多碱基，min > 0。匹配计数是比对中匹配碱基的数量。",
    "e.g., percentage of the query length.": "例如，查询长度的百分比。",
    "Allow no more than this number of mismatched bases. Mismatch count, or nmismatch, is the number of aligned bases in the alignment that are mismatches (substitutions).": "允许不匹配的碱基数不超过这个数字。不匹配计数，或nmismatch，是比对中不匹配（替换）的碱基数。",
    "Allow no more than this number of gaps. Gap count, or ngap, is the number of runs of gapped columns in the alignment (each run is counted as one gap).": "允许的间隙数量不超过这个数值。间隙计数，或ngap，是指在比对中存在间隙列的连续序列的数量（每个连续序列被计算为一个间隙）。",
    "Allow no more than this number of gaps. Gap column count, or cgap, is the number of gapped columns in the alignment (each column is counted as one gap).": "允许的间隙数量不得超过此数值。间隙列计数，或 cgap，是比对中间隙列的数量（每个列都计为一个间隙）。",
    "Note that using ‑‑self automatically enables this option.": "请注意，使用 --self 参数会自动启用此选项。",
    "This value is used as the threshold for both the gap-free and gapped extension sub-stages; see the discussion of interpolation for more details. This option is only valid if gapped extension is performed.": "这个值被用作无间隔和有间隔扩展子阶段的阈值；有关插值的更多详细信息，请参见讨论。如果执行有间隔扩展，则此选项仅有效。",
    "Lastz actually outputs SAM data but Galaxy converts it into BAM to save space. For alignments that don't reach the end of a query, ‑‑format=sam uses 'hard clipping', while ‑‑format=softsam uses 'soft clipping'. See the section on 'clipped alignment' in the SAM specification for an explanation of what this means. The options ‑‑format=sam- and ‑‑format=softsam- suppress the SAM header lines. This makes them suitable for concatenating output from multiple runs. If you need to specify readgroup information: use AddOrEplaceReadGroups from Picard package": "Lastz 实际上输出的是 SAM 格式的数据，但 Galaxy 为了节省空间将其转换为 BAM 格式。对于没有到达查询序列末端的比对，使用 ‑‑format=sam 参数会采用'硬剪切'，而使用 ‑‑format=softsam 参数会采用'软剪切'。有关这两者含义的解释，请参见 SAM 规范中的“剪切比对”部分。参数 ‑‑format=sam- 和 ‑‑format=softsam- 会抑制 SAM 头部行的输出。这使得它们适合于合并多次运行的输出。如果你需要指定读组信息：请使用 Picard 包中的 AddOrReplaceReadGroups。",
    "MAF is a multiple alignment format developed at UCSC": "MAF 是一种在 UCSC 开发的多序列比对格式",
    "The dotplot is only useful if query and target contain exactly one sequence each": "点图仅在查询和目标各自恰好包含一个序列时才有用",
    "Optional controf file. If nothing is selected, LASTZ_D uses default described in the manual": "可选的对照文件。如果没有选择任何内容，LASTZ_D 使用手册中描述的默认值",
    "Built-ins were indexed using default options": "内置索引使用默认选项进行了索引。",
    "if your genome of interest is not listed - contact Galaxy team": "如果您感兴趣的基因组没有列出 - 请联系Galaxy团队",
    "These settings are ignored when using a prebuilt index": "这些设置在使用预构建的索引时被忽略",
    "Allows you to set --packed, --bmax, --bmaxdivn, and --dcv": "允许您设置 --packed、--bmax、--bmaxdivn 和 --dcv 参数",
    "Packed representation saves memory but makes indexing 2-3 times slower": "打包表示节省内存，但会使索引速度变慢2-3倍",
    "-1 for not specified. Must be at least 1": "-1 表示未指定。必须至少为 1",
    "A larger period yields less memory overhead, but may make suffix sorting slower, especially if repeats are present": "较大的周期可以减少内存开销，但如果存在重复序列，可能会使后缀排序变慢",
    "Suffix sorting becomes quadratic-time in the worst case (with a very repetitive reference)": "后缀排序在最坏情况下（参考序列非常重复）会变成二次时间复杂度。",
    "Marking more rows makes reference-position lookups faster, but requires more memory to hold the annotations at runtime": "标记更多的行可以加快参考位置查找速度，但需要更多的内存来在运行时保存注释。",
    "ftab size is 4^(n+1) bytes": "ftab 大小为 4^(n+1) 字节",
    "Little is most appropriate for Intel- and AMD-based architecture": "Little最适合基于Intel和AMD架构的系统。",
    "Use -1 to use default": "使用 -1 以使用默认值",
    "Must have ASCII encoded quality scores": "必须有ASCII编码的质量分数",
    "For most mapping needs use Commonly used settings. If you want full control use Full parameter list": "对于大多数映射需求，请使用常用设置。如果您想要完全控制，请使用完整参数列表。",
    "-1 for off": "-1 表示关闭",
    "May be 0, 1, 2, or 3": "可能是0、1、2或3",
    "Minimum value is 5": "最小值是5",
    "Maq accepts quality values in the Phred quality scale, but internally rounds values to the nearest 10, with a maximum of 30. By default, bowtie also rounds this way": "Maq 在 Phred 质量评分标准下接受质量值，但在内部将值四舍五入到最近的 10，最大值为 30。默认情况下，Bowtie 也以这种方式进行四舍五入。",
    "Removes all strand bias. Only affects which alignments are reported by Bowtie. Runs slower with best option": "去除所有链偏。只影响Bowtie报告的比对。使用最佳选项运行速度较慢。",
    "Tryhard mode is much slower than regular mode": "尝试困难模式比常规模式慢得多",
    "-1 for no limit": "-1 表示无限制",
    "-1 for default": "-1 表示默认",
    "File format must match the Forward FASTQ file": "文件格式必须与正向FASTQ文件匹配",
    "Bowtie produces SAM with several lines of header information by default": "Bowtie 默认会生成带有多行头部信息的 SAM 文件",
    "Built-ins were indexed using default options. See `Indexes` section of help below. If you would like to perform self-mapping select `history` here, then choose your input file as reference.": "内置索引使用默认选项进行了索引。请参阅下面帮助的“索引”部分。如果您想执行自映射，请在这里选择“历史记录”，然后选择您的输入文件作为参考。",
    "Select genome from the list": "从列表中选择基因组",
    "You can upload a FASTA or FASTQ sequence to the history and use it as reference": "您可以将FASTA或FASTQ序列上传到历史记录中，并将其用作参考。",
    "Each profile comes with the preconfigured settings mentioned in parentheses. You can customize each profile further in the indexing, mapping and alignment options sections below. If you do not select a profile here, the tool will use the per-parameter defaults listed in the below sections unless you customize them.": "每个配置文件都带有在括号中提到的预配置设置。您可以在下面的索引、映射和比对选项部分进一步自定义每个配置文件。如果您在这里不选择一个配置文件，该工具将使用下面部分列出的每个参数的默认值，除非您自定义它们。",
    "Sets -f, -g, -F, -r, -n, -m, -X, -p, -N and --min-occ-floor options.": "设置 -f、-g、-F、-r、-n、-m、-X、-p、-N 和 --min-occ-floor 选项。",
    "default=5": "默认值=5",
    "The maximum apparent fragment length up to which paired-end reads are aligned together; at higher fragment lengths the mates will be aligned independent of each other; effective only for paired-end data and when spliced alignment mode is turned off; default=800": "最大明显的片段长度，在这个长度以内，双端测序的读段会被一起比对；在更高的片段长度下，配对的两个读段将独立比对；仅对双端数据有效，且仅在未开启剪接比对模式时有效；默认值=800",
    "default=0.0002": "默认值=0.0002",
    "Allows to specify an interval of k-mer occurrences with -U. For repeat-rich genomes, the automatic k-mer occurrence threshold determined by -f may be too large and makes alignment impractically slow. The new option protects against such cases. Enabled for asm* and map-hifi.": "允许使用-U指定k-mer出现次数的区间。对于重复序列丰富的基因组，由-f确定的自动k-mer出现次数阈值可能过大，导致比对速度不切实际地慢。新选项可以防止这种情况发生。asm*和map-hifi已启用此选项。",
    "Maximum occurence is the number of repetitive minimizers determined by '-f' or this value, whichever is higher.": "最大出现次数是由'-f'参数决定的重复最小子序列的数量，或者这个值，以较大者为准。",
    "Discard a query minimizer if its occurrence is higher than this fraction of query minimizers and than the reference occurrence threshold. It allows to avoid extremely long mapping time for pathologic reads with highly repeated k-mers not in the reference. Set to 0 to disable the new heuristic.": "如果一个查询最小化器的出现频率高于查询最小化器的这个比例，并且高于参考出现阈值，则丢弃该查询最小化器。这可以避免对于具有高度重复的k-mers的病理性读取，而这些k-mers不在参考中，导致非常长的映射时间。设置为0以禁用新的启发式方法。",
    "default=5000": "默认值=5000",
    "default=500": "默认=500",
    "default=3": "默认=3",
    "default=40": "默认值=40",
    "A heuristics that stops chaining early. Minimap2 uses dynamic programming for chaining. The time complexity is quadratic in the number of seeds. This option makes minimap2 exit the inner loop if it repeatedly sees seeds already on chains. Set to a large number to switch off this heurstics effectively. default=25": "一种提前终止链式计算的启发式方法。Minimap2 使用动态规划进行链式计算。时间复杂度与种子数的平方成正比。如果它反复看到已经在链上的种子，这个选项就会使 Minimap2 退出内循环。将其设置为一个较大的数字可以有效地关闭这种启发式方法。默认值=25",
    "A heuristics to avoid quadratic time complexity in the worst case. default=5000": "一种避免在最坏情况下出现二次时间复杂度的启发式方法。默认值=5000",
    "default=0.8": "默认值=0.8",
    "Keep an alignment if dropping it leaves an unaligned region on query longer than INT": "如果删除该对齐会使得查询序列上留下比INT还长的未对齐区域，则保留该对齐。",
    "Sets -A, -B, -O, -E, -z, -s, and spliced alignments options.": "设置 -A、-B、-O、-E、-z、-s 和拼接比对选项。",
    "Warning: Minimap2 cannot currently perform spliced alignments of read pairs. If you enable spliced alignment for paired-end data it will be treated as single-end!": "警告：Minimap2 当前无法对配对的读段进行剪接对齐。如果您为配对末端数据启用剪接对齐，它将被视为单端！",
    "Higher values cause slower spliced alignment. When in use, this option causes -r (in mapping options) to be set to the same value. default=200k": "较高的值会导致剪接对齐变慢。当使用此选项时，会导致在映射选项中将 -r 设置为相同的值。默认值=200k",
    "default=0": "默认=0",
    "default=n (don't match GT-AG)": "默认=n（不匹配GT-AG）",
    "Assume the next base to a GT donor site tends to be A/G (91% in human and 92% in mouse) and the preceding base to a AG acceptor tends to be C/T. This trend is evolutionarily conserved, all the way to S. cerevisiae (PMID:18688272). Specifying this option generally leads to higher junction accuracy by several percents, so it is applied by default with --splice. However, the SIRV control does not honor this trend (only ~60%) so this option reduces accuracy. If you are benchmarking minimap2 on SIRV data, please disable this option.": "假设GT供体位点的下一个碱基倾向于是A/G（在人类中为91%，在小鼠中为92%），而AG受体位点前面的碱基倾向于是C/T。这一趋势在进化上是保守的，一直延续到酿酒酵母（PMID:18688272）。指定此选项通常会通过几个百分点提高接头的准确性，因此默认情况下与--splice一起应用。然而，SIRV对照并不遵循这一趋势（只有约60%），所以这个选项会降低准确性。如果您正在对SIRV数据进行minimap2的基准测试，请禁用此选项。",
    "Gene annotations should be provided in BED12 (aka 12-column BED), intron positions in 5-column BED format.": "基因注释应以BED12（即12列的BED格式）提供，内含子位置应以5列的BED格式提供。",
    "Score bonus for a splice donor or acceptor found in annotation.": "在注释中发现剪接供体或受体时，给予分数奖励。",
    "default=2": "默认值=2",
    "Increase to improve the contiguity of alignments at the cost of poorer alignments in the middle. default=400": "增加以提高比对的连续性，但可能会牺牲中间的比对质量。默认值=400",
    "Decrease to find small inversions at the cost of performance and false positives. default=200": "减少以查找小的倒置，但会牺牲性能和增加误报。默认值=200",
    "default=80": "默认=80",
    "Sets -Q, -L, -R, -c, --cs and -K options.": "设置 -Q、-L、-R、-c、--cs 和 -K 选项。",
    "Useful for very long reads in SAM/BAM format": "适用于SAM/BAM格式中的非常长的读取",
    "default=500M": "默认=500M",
    "The cs tag is a more compact standalone representation of the MD tag, see help below.": "cs标签是MD标签的更紧凑的独立表示形式，详见下面的帮助。",
    "In PAF, the CIGAR is written to the ‘cg’ custom tag.": "在 PAF 中，CIGAR 被写入到自定义标签 'cg' 中。",
    "Select the '... with builtin gene-model' option to select from the list of available indexes that were built with splice junction information. Select the '... without builtin gene-model' option to select from the list of available indexes without annotated splice junctions, and provide your own splice junction annonations.": "选择“带有内置基因模型”选项可以从带有剪接位点信息的可用索引列表中进行选择。选择“不带有内置基因模型”选项可以从没有注释剪接位点的可用索引列表中进行选择，并提供您自己的剪接位点注释。",
    "Disable this if your R1 barcodes contain poly-T bases after the barcode sequence.": "如果你的R1条形码在条形码序列之后包含多聚T碱基，请禁用此选项。",
    "All has all UMIs with 1 mismatch distance to each other collapsed, Directional follows the 'directional' method given in UMI-tools, Exact collapses only exactly matching UMIs.": "所有具有1个不匹配距离的UMIs都被折叠在一起，Directional遵循UMI-tools中给出的'directional'方法，Exact仅折叠完全匹配的UMIs。",
    "Exact: only exact matches allowed; 1MM: only one match in whitelist with 1 mismatched base allowed. Allowed CBs have to have at least one read with exact match; 1MM_multi: multiple matches in whitelist with 1 mismatched base allowed, posterior probability calculation is used choose one of the matches; 1MM_multi_pseudocounts: same as 1MM_Multi, but pseudocounts of 1 are added to all whitelist barcodes.": "精确匹配：只允许完全匹配；1MM：白名单中只有一个匹配，允许有一个不匹配的碱基。允许的CBs至少有一个完全匹配的读取；1MM_multi：白名单中有多个匹配，允许有一个不匹配的碱基，使用后验概率计算选择匹配之一；1MM_multi_pseudocounts：与1MM_Multi相同，但向所有白名单条形码添加了1的伪计数。",
    "Unstranded has no strand information, Forward has the read strand the same as the original RNA molecule, Reverse has the read strand opposite to the original RNA molecule": "未标记方向的没有链信息，正向的读取链与原始RNA分子相同，反向的读取链与原始RNA分子相反。",
    "Input '-' to remove the 3rd column from the output.": "输入'-'来从输出中移除第三列。",
    "Can be used by MultiQC": "可以用MultiQC使用",
    "Select the '... with builtin gene-model' option to select from the list of available indexes that were built with splice junction information. Select the '... without builtin gene-model' option to select from the list of available indexes without annotated splice junctions, and, optionally, provide your own splice-junction annonations.": "选择“...带有内置基因模型”选项，从已构建的带有剪接位点信息的索引列表中进行选择。选择“...不带内置基因模型”选项，从不带注释剪接位点的可用索引列表中进行选择，并可选择性地提供您自己的剪接位点注释。",
    "To build an index with known splice junctions annotated, you will have to provide a GTF or GFF3 dataset that describes the gene models (the location of genes, transcripts and exons) known for the reference genome.": "要构建一个带有已知剪接连接的索引，您需要提供一个描述参考基因组已知基因模型（基因、转录本和外显子的位置）的GTF或GFF3数据集。",
    "For a study with multiple samples, multisample 2-pass mapping is the most sensitive approach. It involves two separate runs of STAR for each sample, where, in the second run of each sample, the splice junctions found in any sample in the first runs are treated as additional known junctions. If you plan to use the mapping results as input for STAR-Fusion it is recommended that you use at least single-sample 2-pass mapping of all reads.": "对于包含多个样本的研究，多样本两遍映射是最敏感的方法。它涉及对每个样本分别运行两次STAR，其中，在每个样本的第二次运行中，第一次运行中在任何样本中发现的剪接连接被视为额外的已知连接。如果您计划将映射结果用作STAR-Fusion的输入，则建议至少使用所有读取的单样本两遍映射。",
    "Choose if and how chimeric alignments should be reported. STAR-Fusion users should select the 'Junctions' option and use the resulting tabular dataset as input to STAR-Fusion. Everyone else: note that selecting 'WithinBAM' or 'WithinBAM Junctions' disables the --chimMultimapNmax setting in the algorithmic parameters section below (the tool will only consider uniquely mapped reads in the search for chimeric alignments). If you disable the reporting of chimeric alignments here, then all chimeric alignment settings in the algorithmic parameters section below will be ignored.": "选择是否以及如何报告嵌合对齐。STAR-Fusion用户应选择'Junctions'选项，并使用生成的表格数据集作为STAR-Fusion的输入。其他人请注意：选择'WithinBAM'或'WithinBAM Junctions'会在下面的算法参数部分禁用--chimMultimapNmax设置（该工具将只考虑唯一映射的读段来搜索嵌合对齐）。如果您在这里禁用嵌合对齐的报告，那么下面算法参数部分中的所有嵌合对齐设置将被忽略。",
    "Note on using the XS tag: If the XS tag is used, STAR will filter out alignments with undefined strand (i.e., those containing only non-canonical unannotated junctions). Using this tag is recommended if you plan to use the STAR results with STAR-Fusion. In addition, it is required for compatibility with Cufflinks if your sequences come from an unstranded library preparation.": "注意使用XS标签：如果使用了XS标签，STAR将会过滤掉具有未定义链（即，仅包含非规范未注释接点的）的比对。如果您计划将STAR的结果与STAR-Fusion一起使用，建议使用此标签。此外，如果您的序列来自非链特定文库制备，还需要此标签以兼容Cufflinks。",
    "Reads with at least this number of alignments will have no alignments output": "至少有这个数量的比对的读段将不会有任何比对输出",
    "Alignments with at least this number of mismatches will not be output": "至少有这么多不匹配的比对将不会被输出",
    "Alignments with a mismatch ratio of at least this value will not be output": "具有至少这个值的不匹配比例的比对将不会被输出",
    "Alignments must have scores higher than this value to be output": "对齐得分必须高于这个值才能输出",
    "Alignments must have (normalized) scores higher than this value to be output": "对齐必须具有高于该值的（标准化）分数才能输出",
    "Alignments must have the number of matched bases higher than this value to be output": "对齐结果必须有高于这个值的匹配碱基数量才能输出",
    "Alignments must have the (normalized) number of matched bases higher than this value to be output": "对齐必须具有（标准化的）匹配碱基数量高于此值才能输出",
    "A value of -1 (the default) results in all alignments (up to --outFilterMultimapNmax) being output": "值为-1（默认值）会导致输出所有比对（最多到--outFilterMultimapNmax）",
    "Default of 0 indicates no maximum length": "默认值为0表示没有最大长度",
    "For small numbers this will cause large number of chimeric alignments. A value of 12 is commonly used.": "对于小数值，这将导致大量的嵌合体比对。通常使用12这个值。",
    "A value of 1 prohibits multimapping main segments": "值为1禁止多映射主片段",
    "The default value of 1 only considers unique alignments. If you chose to report chimeric alignments alongside regular ones in the BAM output, this setting is ignored and only uniquely mapping chimeric reads get reported.": "默认值1只考虑唯一的比对。如果您选择在BAM输出中报告常规的嵌合比对，此设置将被忽略，并且只报告唯一映射的嵌合读段。",
    "The threshold below the best chimeric score that a multimapping chimera must have to be output. This is ignored unless --chimMultimapNmax is above 1": "输出多映射嵌合体必须具有的最佳嵌合分数的阈值。除非 --chimMultimapNmax 大于 1，否则忽略此参数。",
    "Higher values result in lower RAM requirements during the sorting step. The default value is 50. Tweak this if you are facing memory-related errors.": "较高的值会在排序步骤中降低内存需求。默认值是50。如果您遇到与内存相关的错误，请调整此值。",
    "Higher value can increase the runtime singificantly. This value should be set greater or equal to --outFilterMultimapNmax": "较高的值可能会显著增加运行时间。这个值应该设置为大于或等于--outFilterMultimapNmax",
    "Optional output of the module *Bismark Deduplicate*": "可选输出模块 *Bismark Deduplicate*",
    "Optional output of the module *Bismark Methylation Extractor*": "可选模块输出 *Bismark甲基化提取器*",
    "Optional output of the module *Bismark Mapping*": "模块 *Bismark Mapping* 的可选输出",
    "FASTQ or FASTA files.": "FASTQ或FASTA文件。",
    "You can use the default settings or set custom values for any of Bismark's parameters.": "你可以使用Bismark的默认设置，或者为Bismark的任何参数设置自定义值。",
    "Default is 0 and means 'no-limit'.": "默认值是0，表示“无限制”。",
    "This is a function of read length. For instance, specifying `L,0,-0.2` sets the minimum-score function `f` to `f(x) = 0 + -0.2 * x`, where `x` is the read length.": "这是读取长度的一个函数。例如，指定 `L,0,-0.2` 将最小分数函数 `f` 设置为 `f(x) = 0 + -0.2 * x`，其中 `x` 是读取长度。",
    "Write all reads which produce more than one valid alignment with the same number of lowest mismatches or other reads that fail to align uniquely.": "将所有产生一个以上有效对齐（具有相同最少不匹配数）的读段写入文件，或者将那些无法唯一对齐的读段写入文件。",
    "Use this option only if you are certain that your libraries were constructed following a PBAT protocol. If you don't know what PBAT-Seq is you should not specify this option.": "仅当您确信您的文库是按照PBAT协议构建的时，才使用此选项。如果您不知道PBAT-Seq是什么，那么您不应该指定此选项。",
    "This will produce eight strand-specific output files, or two output files in comprehensive mode.": "这将生成八个特定于链的输出文件，或在全面模式下生成两个输出文件。",
    "FASTQ datasets": "FASTQ 数据集",
    "FASTQ dataset": "FASTQ 数据集",
    "Run blastn with '-outputfmt 6' or choose 'Tabular (standard 12 columns)' in the Galaxy wrapper. For human data it is recommended to obtain it from here: https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB/": "使用'-outputfmt 6'运行blastn，或在Galaxy包装器中选择'Tabular (standard 12 columns)'。对于人类数据，建议从这里获取：https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB/",
    "You can use the default settings or set custom values for any STAR Fusion parameter.": "您可以使用STAR Fusion的默认设置，或者为任何STAR Fusion参数设置自定义值。",
    "Built-ins were indexed using default options. See `Indexes` section of help below": "内置选项已使用默认选项进行索引。请参阅下面帮助的 `索引` 部分",
    "Allow selecting among several preset parameter settings. Choosing between these will result in dramatic changes in runtime. See help below to understand effects of these presets.": "允许在几个预设的参数设置中进行选择。选择这些将导致运行时间发生显著变化。请参阅下面的帮助以了解这些预设的影响。",
    "See \"Input Options\" section of Help below for information": "查看下面帮助中的“输入选项”部分以获取信息",
    "-u/--qupto; for default behavior (no limit) leave this value very large": "-u/--qupto; 对于默认行为（无限制），请将此值留为非常大",
    "See \"Alignment Options\" section of Help below for information": "请参阅下面帮助中的“比对选项”部分以获取信息",
    "-L; Smaller values make alignment slower but more sensitive. Default=22": "-L；较小的值会使比对速度变慢，但灵敏度更高。默认值=22",
    "-i; Since it's best to use longer intervals for longer reads, this parameter sets the interval as a function of the read length, rather than a single one-size-fits-all number. For instance, specifying `-i S,1,2.5` sets the interval function `f` to `f(x) = 1 + 2.5 * sqrt(x)`, where x is the read length. If the function returns a result less than 1, it is rounded up to 1. Default=`S,1,1.15`": "-i；由于对于较长的读取来说，使用较长的间隔是更好的选择，因此此参数根据读取长度设置间隔，而不是一个一成不变的数字。例如，指定 `-i S,1,2.5` 将间隔函数 `f` 设置为 `f(x) = 1 + 2.5 * sqrt(x)`，其中 x 是读取长度。如果函数返回的结果小于 1，则向上取整到 1。默认值=`S,1,1.15`",
    "--n-ceil; For instance, specifying `L,0,0.15` sets the N-ceiling function `f` to `f(x) = 0 + 0.15 * x`, where x is the read length. Reads exceeding this ceiling are filtered out. Default=`L,0,0.15`": "--n-ceil; 例如，指定 `L,0,0.15` 将 N-上限函数 `f` 设置为 `f(x) = 0 + 0.15 * x`，其中 x 是读取长度。超过此上限的读取将被过滤掉。默认值=`L,0,0.15`",
    "In paired-end mode, `--nofw` and `--norc` pertain to the fragments; i.e. specifying `--nofw` causes `bowtie2` to explore only those paired-end configurations corresponding to fragments from the reverse-complement (Crick) strand. Default=False": "在配对末端模式下，`--nofw` 和 `--norc` 参数与片段有关；即指定 `--nofw` 会导致 `bowtie2` 仅探索对应于来自反向互补（Crick）链的片段的那些配对末端配置。默认=False",
    "--score-min; This is a function of read length. For instance, specifying `L,0,-0.6` sets the minimum-score function `f` to `f(x) = 0 + -0.6 * x`, where `x` is the read length. The default in `--end-to-end` mode is `L,-0.6,-0.6` and the default in `--local` mode is `G,20,8`": "--score-min；这是读取长度的函数。例如，指定 `L,0,-0.6` 将最小分数函数 `f` 设置为 `f(x) = 0 + -0.6 * x`，其中 `x` 是读取长度。在 `--end-to-end` 模式下的默认值是 `L,-0.6,-0.6`，而在 `--local` 模式下的默认值是 `G,20,8`",
    "See \"Scoring Options\" section of Help below for information": "查看下面的“帮助”中的“打分选项”部分以获取信息",
    "--ma;  In `--local` mode match bonus is added to the alignment score for each position where a read character aligns to a reference character and the characters match. Not used in `--end-to-end` mode; Default=2": "--ma; 在 `--local` 模式下，如果读取字符与参考字符对齐并且字符匹配，则将匹配奖金添加到对齐分数中。在 `--end-to-end` 模式下不使用；默认=2",
    "--mp; A number less than or equal to `MX` and greater than or equal to `MN` is subtracted from the alignment score for each position where a read character aligns to a reference character, the characters do not match, and neither is an `N`.  If `--ignore-quals` is specified, the number subtracted quals `MX`. Otherwise, the number subtracted is `MN + floor( (MX-MN)(MIN(Q, 40.0)/40.0) )` where Q is the Phred quality value; Default=6,2": "--mp; 对于每个读序列字符与参考序列字符对齐的位置，如果字符不匹配且都不是`N`，则从对齐得分中减去一个小于等于`MX`且大于等于`MN`的数字。如果指定了`--ignore-quals`，则减去的数字为`MX`。否则，减去的数字是`MN + floor( (MX-MN)(MIN(Q, 40.0)/40.0) )`，其中Q是Phred质量值；默认值=6,2",
    "--np; Default=1": "--np; 默认=1",
    "--rdg; this is the first component of --rdg flag - opening penalty; Default=5": "--rdg; 这是 --rdg 标志的第一个组成部分 - 开放性惩罚; 默认值=5",
    "--rdg; this is the second component of --rdg flag - extension penalty; Default=3": "--rdg；这是--rdg标志的第二个组成部分 - 扩展惩罚；默认值=3",
    "--rfg; this is the first component of --rfg flag - opening penalty; Default=5": "--rfg；这是--rfg标志的第一个组成部分 - 开放性惩罚；默认值=5",
    "--rfg; this is the second component of --rfg flag - extension penalty; Default=3": "--rfg；这是--rfg标志的第二个组成部分 - 扩展惩罚；默认值=3",
    "Make sure you understand implications of setting -k and -a. See \"Reporting Options\" section of Help below for information on -k and -a options": "确保您理解设置 -k 和 -a 的含义。有关 -k 和 -a 选项的信息，请参见下面帮助中的“报告选项”部分。",
    "-k; see detailed description of this option in the help section below. Note: Bowtie 2 is not designed with large values for `-k` in mind, and when aligning reads to long, repetitive genomes large `-k` can be very, very slow": "-k; 请参阅下面帮助部分中对此选项的详细描述。注意：Bowtie 2 并非针对大的 `-k` 值而设计，在对长序列、重复性高的基因组进行比对时，较大的 `-k` 值可能会非常非常慢。",
    "See \"Effort Options\" section of Help below for information": "请参见下方“帮助”中的“工作选项”部分以获取信息",
    "-D; A seed extension `fails` if it does not yield a new best or a new second-best alignment.  This limit is automatically adjusted up when -k or -a are specified. Default=15": "-D; 如果种子扩展没有产生新的最优或次优比对，则称其为“失败”。当指定-k或-a时，此限制会自动上调。默认值=15",
    "When `re-seeding`, Bowtie 2 simply chooses a new set of reads (same length, same number of mismatches allowed) at different offsets and searches for more alignments.  A read is considered to have repetitive seeds if the total number of seed hits divided by the number of seeds that aligned at least once is greater than 300.  Default=2": "当`重新播种`时，Bowtie 2 简单地选择一组新的读取（相同长度，允许相同数量的不匹配）在不同的偏移量处，并搜索更多的对齐。如果种子命中的总数除以至少对齐一次的种子数大于300，则认为读取具有重复种子。默认值=2",
    "See \"Other Options\" section of Help below for information": "查看下面的“帮助”中的“其他选项”部分以获取信息",
    "--seed; Default=0": "--种子；默认=0",
    "See \"Output Options\" section of Help below for information": "请参阅下面帮助中的“输出选项”部分以获取信息",
    "By default, the output from this Bowtie2 wrapper is a sorted BAM file.": "默认情况下，这个Bowtie2包装器的输出是一个排序后的BAM文件。",
    "--no-unal; Default=False": "--不取消标记；默认=False",
    "--omit-sec-seq; Default=False": "--忽略次级序列；默认=False",
    "Reorder guarantees that output SAM records are printed in an order corresponding to the order of the reads in the original input file, even when -p is set greater than 1.": "Reorder 保证输出的 SAM 记录按照原始输入文件中读取的顺序进行打印，即使 -p 设置为大于 1 的值。",
    "default: 5000": "默认：5000",
    "Report only if this value x score is larger than next best spliced alignment": "仅当此值 x 分数大于下一个最佳剪接对齐时报告",
    "default: none": "默认：无",
    "(0=automatic) (default:0)?": "（0=自动）（默认：0）？",
    "Useful for HiC data": "适用于HiC数据",
    "By default the MAPQ score of a supplementary alignment will be lowered to the primary alignment score.": "默认情况下，补充比对的MAPQ分数会降低到主要比对分数。",
    "The 'Not sorted' option can extend the run time of the tool significantly (cause it requires running on only a single thread).": "“未排序”选项可能会显著延长工具的运行时间（因为它需要仅在单个线程上运行）。",
    "Must have Sanger-scaled quality values with ASCII offset 33": "必须使用具有ASCII偏移量33的Sanger质量值",
    "You can use the default settings or set custom values for any of Tophat's parameters.": "你可以使用Tophat的默认设置，或者为Tophat的任何参数设置自定义值。",
    "--read-realign-edit-dist; Some of the reads spanning multiple exons may be mapped incorrectly as a contiguous alignment to the genome even though the correct alignment should be a spliced one - this can happen in the presence of processed pseudogenes that are rarely (if at all) transcribed or expressed. This option can direct TopHat to re-align reads for which the edit distance of an alignment obtained in a previous mapping step is above or equal to this option value. If you set this option to 0, TopHat will map every read in all the mapping steps (transcriptome if you provided gene annotations, genome, and finally splice variants detected by TopHat), reporting the best possible alignment found in any of these mapping steps. This may greatly increase the mapping accuracy at the expense of an increase in running time. The default value for this option is set such that TopHat will not try to realign reads already mapped in earlier steps.": "--read-realign-edit-dist; 一些跨越多个外显子的读段可能被错误地映射为连续对齐到基因组，即使正确的对齐应该是一个剪接对齐 - 这可能发生在加工过的假基因存在的情况下，这些假基因很少（如果确实有的话）被转录或表达。这个选项可以指示TopHat重新对齐那些在先前的映射步骤中获得的对齐编辑距离高于或等于这个选项值的读段。如果你将这个选项设置为0，TopHat将在所有映射步骤中映射每个读段（如果你提供了基因注释，包括转录组、基因组，以及最后由TopHat检测到的剪接变体），报告在这些映射步骤中找到的最佳可能对齐。这可能会大大提高映射精度，但代价是运行时间的增加。这个选项的默认值设置为TopHat不会尝试重新对齐在早期步骤中已经映射的读段。",
    "--read-edit-dist; Final read alignments having more than these many edit distance are discarded.": "--read-edit-dist; 最终的读段比对如果编辑距离超过这个数值则被丢弃。",
    "--library-type; TopHat will treat the reads as strand specific. Every read alignment will have an XS attribute tag. Consider supplying library type options below to select the correct RNA-seq protocol.": "--library-type; TopHat 会将读取视为特定于链的。每个读取比对将具有一个 XS 属性标签。考虑提供以下库类型选项以选择正确的 RNA-seq 协议。",
    "--read-mismatches; Final read alignments having more than these many mismatches are discarded.": "--读错误数；最终的读对齐如果超过这么多的错误数则被丢弃。",
    "--bowtie-n; TopHat uses \"-v\" in Bowtie for initial read mapping (the default), but with this option, \"-n\" is used instead. Read segments are always mapped using \"-v\" option.": "--bowtie-n; TopHat 使用 \"-v\" 在 Bowtie 中进行初始读取映射（默认设置），但使用此选项时，改用 \"-n\"。读取片段始终使用 \"-v\" 选项进行映射。",
    "-a/--min-anchor-length; TopHat will report junctions spanned by reads with at least this many bases on each side of the junction. Note that individual spliced alignments may span a junction with fewer than this many bases on one side. However, every junction involved in spliced alignments is supported by at least one read with this many bases on each side. This must be at least 3 and the default is 8.": "-a/--min-anchor-length; TopHat会报告那些跨越至少有这么多碱基的读段的连接点。请注意，个别的剪接对齐可能跨越一个连接点，但在这个连接点的一侧少于这么多碱基。然而，每个参与剪接对齐的连接点至少有一个读段在每侧有这么多碱基。这个值必须至少为3，默认值为8。",
    "-m/--splice-mismatches; The default is 0.": "-m/--splice-mismatches; 默认值为0。",
    "-i/--min-intron-length; TopHat will ignore donor/acceptor pairs closer than this many bases apart. The default is 70.": "-i/--min-intron-length; TopHat 会忽略距离小于这个数值的碱基对的供体/受体对。默认值是 70。",
    "-I/--max-intron-length; When searching for junctions ab initio, TopHat will ignore donor/acceptor pairs farther than this many bases apart, except when such a pair is supported by a split segment alignment of a long read. The default is 500000.": "-I/--max-intron-length; 在从头开始搜索接头时，TopHat会忽略距离超过这么多碱基对的供体/受体对，除非这样的一对由一个长读段的拆分对齐支持。默认值为500000。",
    "-g/--max-multihits; Instructs TopHat to allow up to this many alignments to the reference for a given read, and choose the alignments based on their alignment scores if there are more than this number. The default is 20 for read mapping. Unless you use --report-secondary-alignments, TopHat will report the alignments with the best alignment score. If there are more alignments with the same score than this number, TopHat will randomly report only this many alignments. In case of using --report-secondary-alignments, TopHat will try to report alignments up to this option value, and TopHat may randomly output some of the alignments with the same score to meet this number.": "-g/--max-multihits：指示TopHat允许对于给定的读段，最多与参考序列有这么多的比对，并在有超过这个数量的比对时，根据它们的比对分数选择比对。默认情况下，对于读段映射是20。除非你使用--report-secondary-alignments，TopHat会报告具有最佳比对分数的比对。如果有更多具有相同分数的比对超过这个数量，TopHat将随机报告这么多比对。在使用--report-secondary-alignments的情况下，TopHat将尝试报告多达这个选项值的比对，并且TopHat可能会随机输出一些具有相同分数的比对以达到这个数量。",
    "--min-segment-intron; The minimum intron length that may be found during split-segment search. The default is 50.": "--min-segment-intron; 在分割片段搜索期间可能找到的最小内含子长度。默认值为50。",
    "--max-segment-intron; The maximum intron length that may be found during split-segment search. The default is 500000.": "--max-segment-intron；在分割片段搜索期间可能发现的最大内含子长度。默认值为500000。",
    "--segment-mismatches;  Read segments are mapped independently, allowing up to this many mismatches in each segment alignment. The default is 2.": "--segment-mismatches; 读取片段独立映射，允许每个片段对齐中最多有这么多不匹配。默认值为2。",
    "--segment-length; Each read is cut up into segments, each at least this long. These segments are mapped independently. The default is 25.": "--片段长度；每个读取被切割成至少这么长的片段。这些片段独立映射。默认值是25。",
    "If checked, a BAM with the unmapped reads will be added to the history": "如果选中，未映射的读取将被添加到历史记录中的BAM文件",
    "Select 'Auto' to let TopHat decide when to enable coverage search (e.g. disable it for reads 75bp or longer). Select 'Yes' to enforce maximum sensitivity and to specify minimum and maximum intron length": "选择'Auto'让TopHat决定何时启用覆盖搜索（例如，对于75bp或更长的读取禁用它）。选择'Yes'以强制最大灵敏度，并指定最小和最大内含子长度。",
    "--min-coverage-intron; The minimum intron length that may be found during coverage search. The default is 50.": "--min-coverage-intron; 在覆盖度搜索期间可能发现的最小内含子长度。默认值为50。",
    "--max-coverage-intron; The maximum intron length that may be found during coverage search. The default is 20000.": "--max-coverage-intron；在覆盖度搜索期间可能发现的最大内含子长度。默认值是20000。",
    "--microexon-search; With this option, the pipeline will attempt to find alignments incident to microexons. Works only for reads 50bp or longer.": "--microexon-search; 使用此选项，该管道将尝试查找与微外显子相关的比对。仅适用于50bp或更长的读取。",
    "Reads can be aligned to potential fusion transcripts if the --fusion-search option is specified. The fusion alignments are reported in SAM format using custom fields XF and XP (see the output format) and some additional information about fusions will be reported (see fusions.out). Once mapping is done, you can run tophat-fusion-post to filter out fusion transcripts (see the TopHat-Fusion website for more details).": "如果指定了 --fusion-search 选项，读取可以与潜在的融合转录本对齐。融合对齐在 SAM 格式中使用自定义字段 XF 和 XP 报告（见输出格式），并且将报告有关融合的一些附加信息（见 fusions.out）。一旦完成映射，您可以运行 tophat-fusion-post 来过滤掉融合转录本（有关更多详细信息，请参见 TopHat-Fusion 网站）。",
    "--fusion-anchor-length; A 'supporting' read must map to both sides of a fusion by at least this many bases. The default is 20.": "--融合锚点长度；一个“支持”的读取必须至少通过这么多个碱基映射到融合的两侧。默认值是20。",
    "--fusion-min-dist; For intra-chromosomal fusions, TopHat-Fusion tries to find fusions separated by at least this distance. The default is 10000000.": "--fusion-min-dist; 对于染色体内的融合，TopHat-Fusion尝试寻找至少相隔这个距离的融合。默认值为10000000。",
    "--fusion-read-mismatches; Reads support fusions if they map across fusion with at most this many mismatches. The default is 2.": "--融合读取不匹配；如果读取映射在融合处并且最多有这么多不匹配，则支持融合。默认值为2。",
    "--fusion-multireads; Reads that map to more than this many places will be ignored. It may be possible that a fusion is supported by reads (or pairs) that map to multiple places. The default is 2.": "--fusion-multireads; 映射到超过这个数量的地方的读取将被忽略。有可能融合位点由映射到多个地方的读取（或配对）支持。默认值为2。",
    "--fusion-multipairs; Pairs that map to more than this many places will be ignored. The default is 2.": "--fusion-multipairs; 映射到多于这个数量的地方的配对将被忽略。默认值是2。",
    "Required if RG specified. Read group IDs may be modiﬁed when merging SAM ﬁles in order to handle collisions.": "如果指定了RG，则需要。在合并SAM文件以处理冲突时，可能会修改读组ID。",
    "Required if RG specified": "如果指定了 RG，则为必需的",
    "Required if RG specified. Valid values : CAPILLARY, LS454, ILLUMINA, SOLID, HELICOS, IONTORRENT and PACBIO": "如果指定了 RG，则必须。有效值：毛细管电泳, LS454, ILLUMINA, SOLID, HELICOS, IONTORRENT 和 PACBIO",
    "Required if RG specified. Use pool name where a pool is being sequenced": "如果指定了 RG，则需要。在进行测序的池中使用池名称",
    "ftab is 4^(n+1) bytes": "ftab 是 4^(n+1) 字节",
    "-1 for default MAQ-like alignment policy": "-1 表示默认的 MAQ 类似的比对策略",
    "Enter this OR Ratio of SNPs per base": "输入每个碱基的SNPs的比率",
    "Enter this OR SNP penalty": "输入这个或SNP惩罚",
    "Input file containing list of high freq. k-mers generated by meryl": "输入文件包含由meryl生成的高频k-mers列表",
    "k-mer size (no larger than 28).": "k-mer大小（不超过28）。",
    "Sets -f, -g, -F, -r, -n, -m, -X, -p, --sv-off and --min-occ-floor options.": "设置 -f、-g、-F、-r、-n、-m、-X、-p、--sv-off 和 --min-occ-floor 选项。",
    "SV aware k-mer search allows to find approximate mapping locations for a read": "SV感知k-mer搜索允许为一个读段找到近似的映射位置",
    "A heuristics that stops chaining early. Winnowmap uses dynamic programming for chaining. The time complexity is quadratic in the number of seeds. This option makes winnowmap exit the inner loop if it repeatedly sees seeds already on chains. Set to a large number to switch off this heurstics effectively. default=25": "一种在早期停止链化的启发式方法。Winnowmap 使用动态规划进行链化。时间复杂度与种子数的平方成正比。如果它反复看到已经在链上的种子，此选项使 Winnowmap 退出内循环。将其设置为一个较大的数字可以有效地关闭这种启发式方法。默认值=25",
    "Warning: Winnowmap cannot currently perform spliced alignments of read pairs. If you enable spliced alignment for paired-end data it will be treated as single-end!": "警告：Winnowmap当前无法执行配对读取的剪接对齐。如果您为配对末端数据启用剪接对齐，它将被视为单端！",
    "Assume the next base to a GT donor site tends to be A/G (91% in human and 92% in mouse) and the preceding base to a AG acceptor tends to be C/T. This trend is evolutionarily conserved, all the way to S. cerevisiae (PMID:18688272). Specifying this option generally leads to higher junction accuracy by several percents, so it is applied by default with --splice. However, the SIRV control does not honor this trend (only ~60%) so this option reduces accuracy. If you are benchmarking winnowmap on SIRV data, please disable this option.": "假设GT供体位点的下一个碱基倾向于是A/G（在人类中为91%，在小鼠中为92%），而AG受体位点前面的碱基倾向于是C/T。这种趋势在进化上是保守的，一直到酿酒酵母（PMID:18688272）。指定这个选项通常可以提高几个百分比的接头准确性，因此默认情况下与--splice一起应用。然而，SIRV对照并不遵循这一趋势（只有约60%），所以这个选项会降低准确性。如果你正在对SIRV数据进行winnowmap的基准测试，请禁用这个选项。",
    "For the benefit of certain tools that depend on this aligner, such as the TN-93 clustering tool, this dataset's sequence names will have non-alphanumeric characters replaced with underscores, and the sequences will be restricted to the set of IUPAC nucleotide characters.": "为了某些依赖此比对器的工具（例如TN-93聚类工具）的利益，该数据集的序列名称将把非字母数字字符替换为下划线，并且序列将被限制为IUPAC核苷酸字符集。",
    "You can use a predefined background cached on this Galaxy server or select a dataset from your history": "您可以使用这个Galaxy服务器上预定义的背景缓存，或者从您的历史记录中选择一个数据集。",
    "Sets -T, -h, -a, -C, -V, -Y, -M, and -K options.": "设置 -T、-h、-a、-C、-V、-Y、-M 和 -K 选项。",
    "Don't look for indels longer than this (lower is faster)": "不要寻找比这个更长的插入或缺失（越短越快）",
    "By default these are not sought, but may be found anyway": "默认情况下，这些不会被寻找，但无论如何都可能被找到",
    "Higher is faster and less sensitive": "更高的速度更快，但敏感度较低",
    "Higher is faster": "更高代表更快",
    "No value will map to opposite strands": "没有值会映射到相反的链上",
    "Select No for long-mate-pair libraries": "选择“否”以排除长片段配对的库",
    "Lower is faster": "更低的更快",
    "Varies dynamically; does not need to be specified": "动态变化；不需要指定",
    "BBMap is deterministic without this flag if using single-ended reads?": "BBMap 在使用单端测序数据时，如果不使用这个标志，它是确定性的吗？",
    "Zero value ignores": "零值忽略",
    "Faster but less accurate": "更快但准确度较低",
    "For example, 0.03 will ignore the most common 3% of kmers": "例如，0.03将忽略最常见的3%的k-mers",
    "Untrimmed bases will be soft-clipped in cigar strings": "未修剪的碱基将在CIGAR字符串中进行软裁剪",
    "Relevant only when outputting secondary alignments": "仅在输出次要比对时相关",
    "If input is paired, the number will be of pairs for which both reads are unmapped": "如果输入是配对的，那么这个数字将表示两个读段都未映射的配对数",
    "The 'Not sorted' option can significantly extend the run time of the tool (it runs using a single thread)": "“未排序”选项可能会显著延长工具的运行时间（它使用单个线程运行）",
    "Independent of approximate minimum alignment identity to look for": "独立于近似最小比对一致性以寻找",
    "Negative value ignores": "负值忽略",
    "You can upload a FASTA sequence to the history and use it as reference": "您可以将FASTA序列上传到历史记录中，并将其用作参考。",
    "PacBio reads in BAM or [gzipped] fasta or fastq format": "PacBio 读取 BAM 或 [gzipped] fasta 或 fastq 格式",
    "Sets -l, -N, --strip,  --split-by-sample,  --unmapped, --short-sa-cigar": "设置 -l, -N, --strip,  --split-by-sample,  --unmapped, --short-sa-cigar",
    "Map RNA-seq reads to a diploid genome": "将 RNA-seq 读取映射到二倍体基因组",
    "The homozygous and heterozygous alleles from this VCF will be inserted into the reference genome to create a transformed genome with two haplotypes": "这个 VCF 中的纯合子和杂合子等位基因将被插入到参考基因组中，以创建具有两个单倍型的转换基因组。",
    "Use this if downstream processing relies on reference coordinates": "如果下游处理依赖于参考坐标，请使用此选项",
    "You can upload a FASTA sequence to the history from which Picard will automatically generate dictionary using CreateSequenceDictionary command": "您可以将FASTA序列上传到历史记录中，Picard将自动使用CreateSequenceDictionary命令生成字典。",
    "This can be a bed or interval dataset": "这可以是一个bed或间隔数据集",
    "If empty, upload or import a SAM/BAM dataset.": "如果为空，请上传或导入一个SAM/BAM数据集。",
    "REFERENCE_SEQUENCE; You can upload a FASTA sequence to the history and use it as reference": "参考序列；您可以上传一个FASTA序列到历史记录中，并将其用作参考。",
    "You can provide multiple adaptor sequences": "您可以提供多个适配序列",
    "WINDOW_SIZE; default=100": "窗口大小；默认=100",
    "MINIMUM_GENOME_FRACTION; default=0.0005": "最小基因组比例；默认=0.0005",
    "If empty, upload or convert from BED": "如果为空，上传或从BED转换",
    "MINIMUM_MAPPING_QUALITY; default=20": "最小映射质量；默认值为20",
    "MINIMUM_BASE_QUALITY; default=20": "最小碱基质量；默认值=20",
    "COVERAGE_CAP; default=200": "覆盖度上限；默认值=200",
    "CLIP_OVERLAPPING_READS; default=true": "CLIP_OVERLAPPING_READS；默认值=true",
    "DEVIATIONS; This option is offered because insert size data typically includes enough anomalous values from chimeras and other artifacts to make the mean and SD grossly misleading regarding the real distribution. default=10.0": "偏差；提供此选项是因为插入大小数据通常包含来自嵌合体和其他工件的足够异常值，这使得均值和标准差对于真实分布极其误导。默认值=10.0",
    "HISTOGRAM_WIDTH; optional": "直方图宽度；可选",
    "MINIMUM_PCT; (Range: 0 to 1). default=0.05.": "最小百分比；（范围：0到1）。默认=0.05。",
    "See \"Obtaining gene annotations in refFlat format\" below for help": "参见下面的“获取refFlat格式的基因注释”以获得帮助",
    "RIBOSOMAL_INTERVALS; If not specified no bases will be identified as being ribosomal. The list of intervals can be geberated from BED or Interval datasets using Galaxy BedToIntervalList tool": "核糖体区间；如果未指定，则不会有碱基被识别为核糖体。区间列表可以使用Galaxy的BedToIntervalList工具从BED或Interval数据集中生成。",
    "STRAND_SPECIFICITY; For unpaired reads, use FIRST_READ_TRANSCRIPTION_STRAND if the reads are expected to be on the transcription strand.": "链特异性；对于未配对的读取，如果读取预计在转录链上，请使用FIRST_READ_TRANSCRIPTION_STRAND。",
    "MINIMUM_LENGTH; default=500": "最小长度；默认=500",
    "You can provide multiple sequences by clicking the button below": "您可以通过点击下面的按钮提供多个序列",
    "RRNA_FRAGMENT_PERCENTAGE; default=0.8": "rRNA片段百分比；默认值=0.8",
    "e.g NXN corresponds to 1 \"N\" on each side of \"X\"": "例如，NXN 对应于 \"X\" 两侧各有一个 \"N\"",
    "e.g. (for context size=1) AAA,AGG. Default prints all contexts": "例如（对于上下文大小=1）AAA,AGG。默认打印所有上下文",
    "COVERAGE_CAP; default=250": "覆盖度上限；默认=250",
    "If empty, upload or import a SAM or BAM dataset": "如果为空，请上传或导入一个SAM或BAM数据集",
    "PROBABILITY; specify 1 to keep all reads, 0.1 to keep 10% of the reads": "概率；指定1以保留所有读取，0.1以保留10%的读取",
    "RANDOM_SEED; default=1": "随机种子；默认=1",
    "MIN_IDENTICAL_BASES; In effect total_reads / 4^max_id_bases reads will be compared at a time, so lower numbers will produce more accurate results but consume exponentially more memory and CPU; default=5": "最小相同碱基数；实际上，每次将比较 total_reads / 4^max_id_bases 个读取，因此较小的数值将产生更准确的结果，但会消耗更多的内存和CPU；默认值=5",
    "MAX_DIFF_RATE; default=0.03": "最大差异率；默认值=0.03",
    "MIN_MEAN_QUALITY; Reads with lower average quality are filtered out and not considered in any calculations; default=20": "最小平均质量；平均质量低于此值的读段将被过滤掉，并且在任何计算中都不会考虑；默认值=20",
    "MAX_GROUP_RATIO; I.e. if the input contains 10m read pairs and MIN_IDENTICAL_BASES is set to 5, then the mean expected group size would be approximately 10 reads; default-500": "最大组比率；例如，如果输入包含1000万个读对，并且MIN_IDENTICAL_BASES设置为5，则预期的平均组大小大约为10个读；默认值-500",
    "Select between single end, paired end, and collections. See help below for full explanation of dataset types": "选择单端测序、双端测序和集合。详见下方帮助，以获取数据集类型的完整解释。",
    "FASTQ and FASTQ2; A collection of two datasets with forward and reverse reads. See help below on explanation of dataset collections": "FASTQ和FASTQ2；包含正向和反向读取的两个数据集的集合。有关数据集集合解释的帮助，请参见下面的说明。",
    "LIBRARY_NAME; Optional": "库名称；可选",
    "PLATFORM_UNIT; Optional": "平台单元；可选",
    "PLATFORM; Optional": "平台；可选",
    "SEQUENCING_CENTER; Optional": "测序中心；可选",
    "PREDICTED_INSERT_SIZE; Optional": "预测的插入片段大小；可选",
    "COMMENT; Optional": "注释；可选",
    "DESCRIPTION; Optional": "描述；可选",
    "RGDT; Optional; Format=YYYY-MM-DD (eg 1997-07-16)": "RGDT；可选；格式=YYYY-MM-DD（例如 1997-07-16）",
    "MIN_Q; An exception will be thrown if a quality is less than this value; default=0": "最小质量值；如果质量小于这个值，将抛出一个异常；默认=0",
    "MAX_Q; An exception will be thrown if a quality is greater than this value; default=93": "最大质量值；如果质量值大于这个值，将抛出一个异常；默认=93",
    "STRIP_UNPAIRED_MATE_NUMBER; default=false": "去除未配对的伙伴编号；默认=false",
    "ALLOW_AND_IGNORE_EMPTY_LINES; default=false": "允许并忽略空行；默认=false",
    "FILTER; see Help for deatiled info": "过滤；详见帮助以获取详细信息",
    "ADD_MATE_CIGAR; default=True": "添加配对CIGAR；默认=True",
    "ASSUME_SORTED; default=False": "假设排序；默认=False",
    "READ_NAME_REGEX; Read names are parsed to extract three variables: tile/region, x coordinate and y coordinate. These values are used to estimate the rate of optical duplication in order to give a more accurate estimated library size. See help below for more info; default='' (uses : separation)": "读取名称正则表达式；读取名称被解析以提取三个变量：平铺/区域，X坐标和Y坐标。这些值用于估计光学重复率，以给出更准确的估计库大小。有关更多信息，请参见下面的帮助；默认值为''（使用冒号分隔）。",
    "Barcode SAM tag. This tag can be utilized when you have data from an assay that includes Unique Molecular Indices. Typically 'RX'": "条形码 SAM 标签。当您拥有包含唯一分子索引的分析数据时，可以使用此标签。通常为 'RX'",
    "MINIMUM_DISTANCE; Set this number to -1 to use twice the first read's read length (or 100, whichever is smaller); default=-1": "最小距离；将此数字设置为-1以使用第一个读段长度的两倍（或100，以较小者为准）；默认值=-1",
    "SKIP_PAIRS_WITH_NO_MATE_CIGAR; default=True": "跳过没有配对伙伴的CIGAR；默认=True",
    "DUPLICATE_SCORING_STRATEGY; default=TOTAL_MAPPED_REFERENCE_LENGTH": "重复评分策略；默认=总映射参考长度",
    "ALIGNED_READS_ONLY; default=False": "仅对齐的读取；默认为False",
    "PF_READS_ONLY; default=False": "PF_READS_ONLY; 默认=False",
    "UNMAPPED_BAM; This dataset must be sorted in queryname order (use picard_SortSam to do this)": "未映射_BAM；此数据集必须按查询名称排序（使用picard_SortSam进行排序）",
    "You can provide multiple datasets": "您可以提供多个数据集",
    "CLIP_ADAPTERS; default=True": "剪接接头；默认=True",
    "IS_BISULFITE_SEQUENCE; default=False": "是否为双硫酸盐处理序列；默认值为False",
    "MAX_INSERTIONS_OR_DELETIONS; Alignments with more than this many insertions or deletions will be ignored. Set to -1 to allow any number of insertions or deletions. default=1": "最大插入或删除次数；超过这个次数的插入或删除将被忽略。设置为-1以允许任意数量的插入或删除。默认值=1",
    "You can provide multiple attributes": "您可以提供多个属性",
    "ATTRIBUTES_TO_RETAIN; example: XA": "保留属性；示例：XA",
    "ATTRIBUTES_TO_REMOVE; This overrides ATTRIBUTES_TO_RETAIN if they share common tags": "要移除的属性；如果它们有共同的标签，这会覆盖要保留的属性",
    "READ1_TRIM; default=0": "READ1_TRIM；默认值=0",
    "READ2_TRIM; default=0": "READ2_TRIM；默认值=0",
    "EXPECTED_ORIENTATIONS; multiple orinetations can be selected": "预期的方向；可以选择多个方向",
    "ALIGNER_PROPER_PAIR_FLAGS; default=False": "对齐器_正确配对标志；默认=False",
    "PRIMARY_ALIGNMENT_STRATEGY; see help below for more info; default=BestMapq": "主要比对策略；有关更多信息，请参见下面的帮助；默认值=BestMapq",
    "CLIP_OVERLAPPING_READS; default=True": "CLIP_OVERLAPPING_READS；默认=True",
    "INCLUDE_SECONDARY_ALIGNMENTS; default=True": "包含次级比对；默认=True",
    "MERGE_SEQUENCE_DICTIONARIES; default=False": "合并序列字典；默认=False",
    "If empty, upload or import a FASTA dataset": "如果为空，请上传或导入一个FASTA数据集",
    "LINE_LENGTH; default=100": "行长；默认=100",
    "TRUNCATE_SEQUENCE_NAMES_AT_WHITESPACE; default=False": "截断空格处的序列名称；默认=False",
    "INCLUDE_NO_CALLS; default=False": "包含无调用的位点；默认=False",
    "ALLOW_INCOMPLETE_DICT_CONCORDANCE; By default, this tool requires a corresponding contig in the new reference for each read contig; default=False": "允许不完整字典一致性；默认情况下，这个工具要求新参考中每个读取的连续片段都有一个对应的连续片段；默认值为=False",
    "ALLOW_CONTIG_LENGTH_DISCORDANCE; HIGHLY DANGEROUS! Only use if you know what you are doing; default=False": "允许片段长度不一致；极其危险！只有当你清楚自己在做什么时才使用；默认=假",
    "HEADER; If empty, upload or import a SAM/BAM dataset": "标题；如果为空，上传或导入一个SAM/BAM数据集",
    "RESTORE_ORIGINAL_QUALITIES; default=True": "恢复原始质量；默认=True",
    "MAX_RECORDS_TO_EXAMINE; default=10,000": "最大检查记录数；默认=10,000",
    "REMOVE_DUPLICATE_INFORMATION; Note that if this is true and REMOVE_ALIGNMENT_INFORMATION is set to False, the output may have the unusual but sometimes desirable trait of having unmapped reads that are marked as duplicates; default=True": "删除重复信息；请注意，如果此选项设置为true，并且删除比对信息（REMOVE_ALIGNMENT_INFORMATION）设置为False，则输出可能具有不寻常但有时可能有用的特征，即未比对的读段被标记为重复；默认值为True",
    "REMOVE_ALIGNMENT_INFORMATION; default=True": "移除比对信息；默认=True",
    "SANITIZE; WARNING: This option is potentially destructive. Reads discarded include (but are not limited to) paired reads with missing mates, duplicated records, records with mismatches in length of bases and qualities. This option can only be enabled if the output sort order is queryname and will always cause sorting to occur; default=False": "净化；警告：这个选项可能具有破坏性。被丢弃的读取包括（但不限于）缺少配对的读取、重复记录、碱基长度和质量长度不匹配的记录。这个选项只有在输出排序顺序是查询名称时才能启用，并且总是会导致排序发生；默认=False",
    "MAX_DISCARD_FRACTION; default=0.01": "最大丢弃分数；默认值=0.01",
    "SAMPLE_ALIAS; default=Null": "样本别名；默认=空",
    "LIBRARY_NAME; default=Null": "库名称；默认=空",
    "SORT_ORDER; Picard default=queryname; Galaxy default=coordinate": "排序顺序；Picard默认=queryname（查询名称）；Galaxy默认=coordinate（坐标）",
    "RE_REVERSE; default=True": "RE_REVERSE；默认=True",
    "INCLUDE_NON_PF_READS; PF means 'passes filtering'. Reads whose 'not passing quality controls' flag is set are non-PF reads; default=False": "包含非PF读取；PF代表“通过过滤”。设置了“未通过质量控制”标志的读取是非PF读取；默认=False",
    "CLIPPING_ATTRIBUTE; default=null": "截断属性；默认值为null",
    "CLIPPING_ACTION; default=null": "截断操作；默认值为null",
    "READ1_MAX_BASES_TO_WRITE; If there are fewer than this many bases left after trimming, all will be written. If this value is null then all bases left after trimming will be written; default=null (-1)": "READ1_MAX_BASES_TO_WRITE：如果修剪后剩余的碱基少于此数值，将全部写入。如果此值为null，则修剪后剩余的所有碱基都将被写入；默认值为null（-1）",
    "READ2_MAX_BASES_TO_WRITE; If there are fewer than this many bases left after trimming, all will be written. If this value is null then all bases left after trimming will be written; default=null (-1)": "READ2_MAX_BASES_TO_WRITE：如果修剪后剩余的碱基少于此数值，将全部写出。如果该值为null，则表示修剪后剩余的所有碱基都将被写出；默认值为null（-1）。",
    "INCLUDE_NON_PRIMARY_ALIGNMENTS; Support of non-primary alignments in SamToFastq is not comprehensive, so there may be exceptions if this is set to true and there are paired reads with non-primary alignments; default=False": "包含非主要比对；在SamToFastq中对非主要比对的支持不是全面的，因此如果设置为true并且存在具有非主要比对的配对读取，则可能会出现异常；默认=False",
    "SORT_ORDER; default=coordinate. Selecting Queryname will output SAM file, as Galaxy does not support BAM files that are not coordinate sorted.": "排序顺序；默认为坐标。选择查询名称将输出SAM文件，因为Galaxy不支持未按坐标排序的BAM文件。",
    "MAX_OUTPUT; default=100": "最大输出；默认=100",
    "IGNORE_WARNINGS; default=False": "忽略警告；默认=False",
    "VALIDATE_INDEX; default=True": "验证索引；默认=True",
    "IS_BISULFITE_SEQUENCED; If so, C->T is not counted as an error in computing the value of the NM tag; default=False": "是否是双硫仑处理的序列；如果是，那么在计算NM标签的值时，C->T不被视为错误；默认=False",
    "OUTPUT_PER_RG; default=False": "输出每个读取组的结果；默认为False",
    "INTERLEAVE; default=False": "交错；默认=False",
    "SORT_ORDER; default=coordinate": "排序顺序；默认=坐标",
    "You can upload sequences to the history from which Picard will automatically generate dictionary using CreateSequenceDictionary command": "你可以将序列上传到历史记录中，Picard将自动使用CreateSequenceDictionary命令生成字典。",
    "(-l) Requires headers in the input SAM or BAM, otherwise no alignments will be output": "（-l）需要输入的SAM或BAM文件中包含头部信息，否则将不会输出任何比对结果",
    "(-r) Requires headers in the input SAM or BAM, otherwise no alignments will be output": "（-r）需要在输入的SAM或BAM文件中包含标题，否则将不会输出任何比对结果。",
    "Select the opposite of the listed chromosomes": "选择所列染色体的相反染色体",
    "region should be presented in one of the following formats: `chr1', `chr2:1,000' and `chr3:1000-2,000'": "区域应该以以下格式之一呈现：`chr1`、`chr2:1,000` 和 `chr3:1000-2,000`",
    "hard clip (remove bases), unchekced = default soft-clipping": "硬剪切（移除碱基），未选中 = 默认软剪切",
    "mark reads QCFAIL at this length or shorter after clipping": "标记读取在剪切后在这个长度或更短时为 QCFAIL",
    "match region within this number of bases, default 5.": "在这段距离内匹配区域，默认为5个碱基。",
    "See \"Types of pileup datasets\" below for examples": "请参阅下面的“堆叠数据集类型”以获取示例",
    "something like this: ..,a..": "像这样的东西：..,a..",
    "something like this: IIIGIAI": "像这样的东西：IIIGIAI",
    "No variants with quality below this value will be reported": "不会报告质量低于该值的变异",
    "Pileup lines with coverage lower than this value will be skipped": "覆盖度低于该值的Pileup行将被跳过",
    "See \"Examples 1 and 2\" below for explanation": "请参见下面的“示例 1 和 2”以获得解释",
    "See \"Output format\" below for explanation": "请参见下面的“输出格式”以获取解释",
    "See \"Example 3\" below for explanation": "参见下面的“示例 3”以获得解释",
    "See \"Example 4\" below for explanation": "请参考下面的“示例 4”进行解释",
    "Do you want to retain original SAM fields? See example below.": "您是否希望保留原始的SAM字段？请参见下面的示例。",
    "Control the MERGE_SEQUENCE_DICTIONARIES flag for Picard MergeSamFiles. Default (true) correctly propagates read groups and other important metadata": "控制 Picard MergeSamFiles 的 MERGE_SEQUENCE_DICTIONARIES 标志。默认值（true）正确地传播了读取组和其他重要的元数据",
    "Makes the output easier to parse, but is space inefficient": "使输出更容易解析，但空间效率不高",
    "Greater than or equal to 2": "大于或等于2",
    "Format CHR:FROM-TO, e.g. 17:100-150. If used in conjunction with -l then considers the intersection of the two requests.": "格式为 CHR:FROM-TO，例如 17:100-150。如果与 -l 参数一起使用，则考虑两个请求的交集。",
    "Paste a list of read groups": "粘贴一个测序组列表",
    "Given a read with a phred-scaled probability q of being generated from the mapped position, the new mapping quality is about sqrt((INT-q)/INT)*INT. A zero value disables this functionality; if enabled, the recommended value for BWA is 50": "给定一个读取，其Phred-scaled概率q表示其可能从映射位置生成，新的映射质量大约是sqrt((INT-q)/INT)*INT。值为零将禁用此功能；如果启用，对于BWA推荐的值为50。",
    "Ignore existing BQ tags": "忽略现有的BQ标签",
    "Output all positions, including those with zero depth. (-a) Output absolutely all positions, including unused reference sequences (-aa). Note that when used in conjunction with a BED file the -a option may sometimes operate as if -aa was specified if the reference sequence has coverage outside of the region specified in the BED file.": "输出所有位置，包括那些深度为零的位置。（-a）输出绝对所有位置，包括未使用的参考序列（-aa）。请注意，当与BED文件一起使用时，-a选项有时可能像指定了-aa一样操作，如果参考序列在BED文件指定的区域外有覆盖范围。",
    "Output extra read fields and read tag values, e.g., NM and XS (must be comma seperated, e.g., NM,XS)": "输出额外的读取字段和读取标签值，例如，NM和XS（必须用逗号分隔，例如，NM,XS）",
    "BED datasets can be obtained using \"Get Data -> UCSC Main\" datasource": "BED 数据集可以通过 \"获取数据 -> UCSC 主站\" 数据源获得",
    "Click and type in the box above to see options. You can select multiple entries. If \"No options available\" is displayed, you need to re-detect metadata on the input dataset. See help section below.": "点击并输入上面的框以查看选项。您可以选择多个条目。如果显示“无可用选项”，则需要重新检测输入数据集的元数据。请参阅下面的帮助部分。",
    "Select chromosome/contig from the list. If \"No options available\" is displayed, you need to re-detect metadata on the input dataset. See help section below.": "从列表中选择染色体/基因组片段。如果显示“没有可用选项”，则需要重新检测输入数据集的元数据。请参阅下面的帮助部分。",
    "One or more space-separated region specifications to restrict output to only those alignments which overlap the specified region(s).": "一个或多个以空格分隔的区域规格说明，用于限制输出，只包括与指定区域（或多个区域）重叠的比对。",
    "Only output alignments overlapping the intervals in the input bed file.": "仅输出与输入的bed文件中的区间重叠的比对。",
    "Only output alignments in read group.": "仅输出读组中的比对结果。",
    "Output alignments in read groups listed in FILE.": "输出在文件中列出的读组的比对结果。",
    "Skip alignments with MAPQ smaller than INT.": "跳过 MAPQ 小于 INT 的比对。",
    "Only output alignments in library STR": "仅输出库STR中的比对结果",
    "Only output alignments with number of CIGAR bases consuming query sequence greater than or equal INT.": "仅输出CIGAR碱基消耗的查询序列数量大于或等于整数INT的比对结果。",
    "Only include reads with tag STR1 and associated value STR2. Write in the format STR1:STR2 (see help for more details).": "只包括带有标签STR1和关联值STR2的读取。按照STR1:STR2的格式写入（有关更多详细信息，请参见帮助）。",
    "Only output alignments with readnames that are listed in this file.": "只输出文件中列出的读名字的比对结果。",
    "The factor by which to downsample the input reads. A fraction of approx. 1/factor of the reads will be kept (default: 1 = no downsampling).": "输入序列的下采样因子。大约1/因子的序列会被保留（默认值：1 = 不进行下采样）。",
    "Sets the approx. target number of reads to subsample.": "设置大致的目标读数以进行子采样。",
    "Hint: To invert all of the filtering/subsampling logic configured above, choose 'Reads dropped during filtering and subsampling'.": "提示：要反转上述配置的所有过滤/子采样逻辑，请选择“在过滤和子采样期间删除的读取”。",
    "Reference data as fasta(.gz). Required for SAM input without @SQ headers and useful/required for writing CRAM output (see help).": "参考数据作为fasta(.gz)格式。在没有@SQ头的SAM输入中必需，并且对于写入CRAM输出很有用/必需（见帮助）。",
    "Header from this dataset will be used to replace header in the target dataset. May be either SAM or BAM dataset.": "此数据集的头部信息将被用来替换目标数据集的头部信息。可以是SAM或BAM数据集。",
    "Header from the source dataset will be inserted here. Should be a BAM dataset.": "源数据集的头部信息将在此插入。应该是一个BAM数据集。",
    "By default, samtools reheader will add itself to the chain of tools used to manipulate the target dataset, which will cause one or more @PG records to be inserted into the new header. If you want the output dataset to end up with the header of the source dataset exactly, you should activate this option.": "默认情况下，samtools reheader 会将自己添加到用于操作目标数据集的工具链中，这将导致一个或多个 @PG 记录被插入到新头文件中。如果你希望输出数据集的头文件与源数据集的头文件完全相同，你应该激活这个选项。",
    "If selected, report read statistics *outside* the regions in the regions file.": "如果选中，则在区域文件中报告区域*外*的读取统计信息。",
    "Produce additional tabular output listing the coverage at every site (omitting only zero-coverage positions) in the selected regions of the genome. Caution: Will generate a huge dataset for anything but small input genomes or restricted regions!": "生成额外的表格输出，列出所选基因组区域中每个位点的覆盖度（仅省略零覆盖度的位置）。注意：对于任何非小型输入基因组或受限区域，这将生成一个巨大的数据集！",
    "Affected plots: Coverage, Mapping Quality and Insert Size across reference, Mapped reads GC-content distribution; the value determines the resolution of the affected plots. Note: The lower the value, the higher the memory usage of the tool!": "受影响的图表：参考序列、映射质量以及插入大小的覆盖度，映射读取的GC含量分布；该值决定了受影响图表的分辨率。注意：该值越低，工具的内存使用量就越高！",
    "Affected plots: Coverage, Mapping Quality and Insert Size across reference; in across-reference plots, indicate chromosome boundaries with dotted lines and labels": "受影响的图表：参考基因组上的覆盖度、映射质量和插入片段大小；在参考基因组上的图表中，用虚线和标签标出染色体边界。",
    "Affected plot: Mapped reads GC-content distribution; include a precalculated GC-content distribution for the selected (Qualimap-supported) reference genome in the plot": "受影响的图形：映射读取的GC含量分布；在图形中包含一个预先计算的GC含量分布，用于所选的（Qualimap支持的）参考基因组。",
    "Affected plot: Homopolymer indels; sets the minimal number of consecutive bases that define a homopolymer": "受影响的绘图：同聚体插入缺失；设置定义同聚体的连续碱基的最小数量",
    "This is the name that will be used for this dataset throughout the Qualimap report. Default: Name of the raw data collection in the history": "这是在Qualimap报告中将用于此数据集的名称。默认值：历史记录中原始数据集的名称",
    "See help below for detailed description of each tool": "请参阅下面的帮助，以获取每个工具的详细描述。",
    "Select what to convert your BAM to": "选择将您的BAM文件转换为什么格式",
    "No duplicate checking is perfomed": "没有进行重复检查",
    "Use the same seed for reproducible results": "使用相同的种子以获得可复现的结果",
    "Do not remove duplicate marks": "不要移除重复标记",
    "Do not replace qualities with contect of OQ tag": "不要将OQ标签的内容替换为品质",
    "Selecting combined requires paired-end BAM input": "选择combined需要成对的BAM输入",
    "Requires paired-end BAM input": "需要成对的BAM输入",
    "Sets of aligned reads.": "对齐的读取集。",
    "Merge files in the specified region indicated by a string": "在指定区域合并文件，该区域由一个字符串指示。",
    "Use the lines of FILE as `@' headers to be copied to out.bam, replacing any header lines that would otherwise be copied from in1.bam. (FILE is actually in SAM format, though any alignment records it may contain are ignored.)": "使用文件中的行作为`@'标题复制到out.bam，替换从in1.bam复制的任何标题行。（文件实际上在SAM格式中，尽管它可能包含的任何对齐记录都被忽略。）",
    "When several input files contain @RG headers with the same ID, emit only one of them (namely, the header line from the first file we find that ID in) to the merged output file. Combining these similar headers is usually the right thing to do when the files being merged originated from the same file. Without -c, all @RG headers appear in the output file, with random suffices added to their IDs where necessary to differentiate them.": "当几个输入文件包含具有相同ID的@RG头时，在合并的输出文件中只发出其中的一个（即，我们找到该ID的第一个文件中的头行）。当合并的文件来自同一个文件时，通常合并这些类似的头是正确的做法。没有-c选项，所有的@RG头都会出现在输出文件中，必要时在它们的ID上添加随机后缀以区分它们。",
    "Similarly, for each @PG ID in the set of files to merge, use the @PG line of the first file we find that ID in rather than adding a suffix to differentiate similar IDs.": "同样地，对于要合并的文件集中的每个@PG ID，在找到该ID的第一个文件中使用@PG行，而不是添加后缀来区分相似的ID。",
    "Set of aligned reads.": "一组对齐的读取序列。",
    "These are used by markdup to select the best reads to keep.": "这些由markdup用于选择保留的最佳读取。",
    "Step value for coverage": "覆盖度的步长值",
    "Select between one single output or separate outputs for each statistics": "选择单个输出或为每个统计数据分别输出",
    "More info on the flags: https://samtools.github.io/hts-specs/SAMv1.pdf": "关于标志的更多信息：https://samtools.github.io/hts-specs/SAMv1.pdf",
    "Decreasing bin size increases memory requirement. default=2e4": "减少bin大小会增加内存需求。默认值=2e4",
    "default=8000": "默认值=8000",
    "No cutoff if left empty": "如果不设置，则没有截止值",
    "default=0.99": "默认值=0.99",
    "Required for GC-depth and mismatches-per-cycle calculation": "用于GC深度和每周期不匹配计算",
    "Suppress outputting IS rows where there are no insertions.": "抑制输出没有插入的IS行。",
    "Only count reads with mapping quality greater than this value": "只统计映射质量大于这个值的读取",
    " Do not include deletions (D) and ref skips (N) in bedcov computation": "不要在bedcov计算中包含删除（D）和参考跳过（N）",
    "Number of reference bases with coverage above and including this value will be displayed in a separate column.": "覆盖度高于或等于这个值的参考碱基数量将在一个单独的列中显示。",
    "Format is CHROMOSOME:FROM-TO": "格式是 染色体:起始-结束",
    "Compute depth at list of positions or regions in specified BED FILE.": "在指定的BED文件中，计算位置列表或区域的深度。",
    "Note that if the all option (-a) is used in conjunction with a BED file it may sometimes operate as if -aa was specified if the reference sequence has coverage outside of the region specified in the BED file.": "注意，如果将 all 选项（-a）与 BED 文件一起使用，有时如果参考序列在 BED 文件指定的区域外有覆盖，它可能会像指定了 -aa 一样操作。",
    "Use minimiser for clustering unaligned/unplaced reads.": "使用最小化器对未比对/未定位的读取进行聚类。",
    "Calculate the coverage for multiple bam files": "计算多个BAM文件的覆盖度",
    "Ignore reads shorter than INT base pairs (Default = 0)": "忽略短于INT个碱基对的读取（默认值 = 0）",
    "Minimum mapping quality for an alignment to be used (Default = 0)": "用于使用的比对的最小映射质量（默认 = 0）",
    "Minimum base quality for a base to be considered (Default = 0)": "考虑一个碱基的最小碱基质量（默认值 = 0）",
    "Show specified region. Format: chr:start-end.": "显示指定区域。格式：染色体：起始位置-结束位置。",
    "Show histogram instead of tabular output": "显示直方图而不是表格输出",
    "in bp": "在碱基对中",
    "both mapped, correct orientation, flag set in BAM": "都已映射，正确的方向，在BAM中设置标志",
    "e.g. 0x400, 0x2": "例如 0x400, 0x2",
    "up to the Nth nucleotide starting from the 5prime end. -1 means do not filter, 0 means check along the entire read, 10 would make filter reads that are unique over the first 10 nucleotides.": "从5'端开始，直到第N个核苷酸。-1表示不进行过滤，0表示检查整个读取序列，10将过滤掉在前10个核苷酸处唯一的读取序列。",
    "Use only for low-coverage samples": "仅用于低覆盖度样本",
    "Indels always counts as 1 regardless of length. Requires NM tag to be set.": "插入和缺失总是计算为1，不论其长度。需要设置NM标签。",
    "Remove reads flagged with 0x100": "移除带有0x100标志的读取",
    "Remove reads flagged with 0x200": "移除带有0x200标志的读取",
    "Remove reads flagged with 0x400": "去除带有0x400标志的读取",
    "Affects -excludebed and -includebed.": "影响 -excludebed 和 -includebed 参数。",
    "e.g. chrM, or _dup chromosomes": "例如，chrM 或 _dup 染色体",
    "fraction of length": "长度分数",
    "Default (3) is for a paired read mapped in a proper pair": "默认值（3）是用于成对的读取在正确的配对中映射",
    "Default (101M) is for 101 continuously matched bases": "默认（101M）是用于101个连续匹配的碱基",
    "You can use >, <, =, and ! (not) in your expression. E.g., to select pairs with insert size above 250 nt use \">=250\"": "您可以在表达式中使用 >、<、= 和 !（表示非）来选择。例如，要选取插入片段大小超过250个核苷酸的配对，可以使用 \">=250\"。",
    "Checked = Read IS Duplicate, Empty = Read is NOT Duplicate": "检查 = 读取是重复的，空 = 读取不是重复的",
    "Checked = Failed QC, Empty = Passed QC": "检查 = 未通过质量控制，空 = 通过质量控制",
    "Checked = is first mate, Empty = is NOT first mate": "选中 = 是大副，未选中 = 不是大副",
    "Checked = Mapped, Empty = NOT mapped": "已检查 = 已映射，空 = 未映射",
    "Checked = Mate IS mapped Empty = Mate is NOT mapped": "已检查 = 配对的IS已映射 为空 = 配对的IS未映射",
    "Checked = Mate IS on reverse strand, Empty = Mate is NOT on the reverse strand": "已检查 = 配对在反向链上，空 = 配对不在反向链上",
    "Checked = Read IS paired, Empty = Read is NOT paired": "已检查 = 读取是成对的，空 = 读取不成对",
    "Checked = Alignment IS primary, Empty = Alignment is NOT primary": "已检查 = 对齐是主要的，空 = 对齐不是主要的",
    "Checked = Read IS in proper pair, Empty = Read is NOT in the proper pair": "已检查 = 读取在正确的配对中，空 = 读取不在正确的配对中",
    "Checked = Read IS on the reverse strand, Empty = Read is NOT on the reverse strand": "已检查 = 读取位于反向链上，空 = 读取不位于反向链上",
    "Checked = Read IS second mate, Empty = Read is NOT second mate": "已检查 = 读取是第二配对，空 = 读取不是第二配对",
    "You can use >, <, =, and ! (not) in your expression. E.g., to select reads with mapping quality of at least 30 use \">=30\"": "你可以在表达式中使用 >、<、= 和 !（非）。例如，要选择映射质量至少为30的读取，可以使用 \">=30\"",
    "You can use >, <, =, and ! (not) in your expression. E.g., to select reads with mate (second end) mapping after position 1,000,000 use \">1000000\"": "您可以在表达式中使用 >, <, =, 和 !（非）。例如，要选择在位置1,000,000之后的配对（第二端）映射的读取，请使用\">1000000\"",
    "You can use = and ! (not) in your expression. E.g., to select reads with mates mapping to chrM use \"chr22\"": "你可以在表达式中使用“=”和“!”（表示非）来选择。例如，要选取与chrM配对的读段，可以使用“chr22”。",
    "You can use = and ! (not) in your expression.": "你可以在表达式中使用“=”和“!”（非）。",
    "You can use >, <, =, and ! (not) in your expression. E.g., to select reads mapping after position 5,000 use \">5000\"": "您可以在表达式中使用 >、<、= 和 !（非）。例如，要选择映射在5000位置之后的读取，可以使用 \">5000\"",
    "You can use ! (not) in your expression": "你可以在表达式中使用感叹号（!）表示否定",
    "You can use +, -, >, <, =, and ! (not).  Tag name and its value must be separated by \":\". E.g., to obtain reads with at least one mismatch use \"NM:>1\"": "您可以使用+, -, >, <, =, 和 !（非）。标签名称和它的值必须用\":\"分隔。例如，要获取至少有一个不匹配的读取，使用\"NM:>1\"",
    "Allows complex logical constructs. See Example 4 below.": "允许复杂的逻辑结构。见下面的例子4。",
    "This option can only be used with at least two conditions. Read help below (Example 4) to understand how it works.": "这个选项只能在至少有两个条件的情况下使用。请阅读下面的帮助（示例 4）以了解其工作原理。",
    "By default, BAQ scores are stored in a separate BQ tag, and the read base quality string is left untouched. When you enable this option, the BAQ scores will be used directly to modify the quality string instead.": "默认情况下，BAQ分数存储在单独的BQ标签中，而读取碱基质量字符串保持不变。当您启用此选项时，将直接使用BAQ分数来修改质量字符串。",
    "Replace bases in read sequences that match the reference base at that position with an equal sign": "将读取序列中与参考碱基在该位置相匹配的碱基替换为等号",
    "Higher values for this setting mean a stronger downgrade of the mapping quality of reads with excessive mismatches (50: recommended setting for reads aligned with BWA, 0: do not downgrade mapping qualities)": "这个设置的值越高，意味着对有过多不匹配的读取的映射质量的降低就越强（50：对于使用BWA对齐的读取推荐的设置，0：不要降低映射质量）",
    "BAM or CRAM format mapped reads": "BAM或CRAM格式的映射读取",
    "Skipping depth output for individual bases substantially decreases runtime": "跳过单个碱基的深度输出可以显著减少运行时间",
    "A BED file or window size can be provided to compute average depth by region": "可以提供BED文件或窗口大小，以计算每个区域的平均深度",
    "Average depth will be computed for non-overlapping regions of this size": "平均深度将计算为此大小的不重叠区域",
    "Average depth will be computed for regions specified in the BED file. If the BED file has a 4th column, it is used for the region name": "平均深度将针对BED文件中指定的区域进行计算。如果BED文件有一个第四列，则该列用于区域名称。",
    "Enter a chromosome name to only count depth for reads mapping to this chromosome": "输入一个染色体名称，仅对映射到该染色体的读取计算深度",
    "Exclude reads where any of these bits are set in the value of the read flags column": "排除在读取标志列的值中设置了这些位的读取",
    "Only include reads where any of these bits are set in the value of the read flags column": "只包含在读取标志列的值中设置了这些位的读取",
    "Fast mode doesn't look up internal cigar operations or correct mate overlaps. Disabling it is not recommended": "快速模式不会查找内部cigar操作或校正配对重叠。不建议禁用它",
    "Reads with a mapping quality lower than this value are ignored": "忽略映射质量低于该值的序列",
    "The number of bases covered by these thresholds will be reported. Multiple thresholds can be separated by commas": "这些阈值所覆盖的碱基数量将被报告。多个阈值可以用逗号分隔",
    "When computing depth of region, use median of depth rather than mean": "在计算区域深度时，使用深度的中位数而不是平均值",
    "only calculate depth for these comma-separated read groups IDs": "仅对这些用逗号分隔的读组ID计算深度",
    "Regions with this depth or lower will be included in this depth-defined region": "具有此深度或更低深度的区域将包含在此深度定义的区域内",
    "Name include in BED output for regions in this depth-defined region": "在深度定义区域的BED输出中包含名称",
    "Do not use read quality as part of checksum. (--no-quality)": "不要使用读取质量作为校验和的一部分。(--no-quality)",
    "Do not use read names as part of checksum. (--no-readnames)": "不要使用读取名称作为校验和的一部分。(--no-readnames)",
    "For example, to split on NM tag enter \"NM\"": "例如，要在NM标签上拆分，输入\"NM\"",
    "The SAMtools mpileup file": "SAMtools的mpileup文件",
    "Set to 1 to report only variants": "设置为1以仅报告变异",
    "The SAMtools mpileup file for normal sample": "正常样本的SAMtools mpileup文件",
    "The SAMtools mpileup file for tumor sample": "肿瘤样本的SAMtools mpileup文件",
    "See help below for explanation of each option": "请见下方的帮助，以了解每个选项的说明。",
    "Leave blank to skip": "留空以跳过",
    "See the help section below for information about this option": "请参阅下面的帮助部分，以获取有关此选项的信息",
    "Allows to choose between seeing the entire dataset with the header, header only, or data only.": "允许选择查看整个数据集及其头部信息、仅头部信息或仅数据。",
    "Any number of mismatches within this window will cause the read to be discarded": "在这个窗口内任何数量的不匹配都会导致读取被丢弃",
    "Default is to report solely the fraction of -i covered by each file.": "默认情况下，仅报告每个文件覆盖的-i部分的比例。",
    "Default is to report solely the fraction of the input file covered by each file.": "默认情况下，仅报告每个文件覆盖输入文件的比例。",
    "If Selected, bedToBam will convert blocked BED features (e.g., gene annotaions) into 'spliced' BAM alignments by creating an appropriate CIGAR string. (-bed12)": "如果选中，bedToBam 将把阻塞的 BED 特征（例如，基因注释）转换成 'spliced' BAM 对齐，通过创建适当的 CIGAR 字符串。（-bed12）",
    "This occurs when two features in B have exactly the same overlap with a feature in A.": "当B中的两个特征与A中的特征完全相同地重叠时，就会发生这种情况。",
    "The reported distance for overlapping features will be 0. (-d)": "报告重叠特征的距离将为0。(-d)",
    "Like -d, report the closest feature in B, and its distance to A as an extra column. However unlike -d, use negative distances to report upstream features. (-D)": "像-d参数一样，报告B中距离A最近的一个特征，并将其距离作为额外的一列。然而，与-d参数不同，使用负距离来报告上游的特征。（-D）",
    "This option requires -D and follows its orientation rules for determining what is 'upstream'. (-iu)": "这个选项需要 -D 并且遵循其方向规则来确定什么是 '上游'。(-iu)",
    "This option requires -D and follows its orientation rules for determining what is 'downstream'. (-id)": "这个选项需要 -D 并且遵循其方向规则来确定什么是 '下游'。(-id)",
    "That is, we want close, yet not touching features only. (-io)": "也就是说，我们希望特征彼此靠近，但又不接触。（-io）",
    "That is, only cluster features that are the same strand. By default, this is disabled.": "即，仅聚类具有相同链的特征。默认情况下，此功能是禁用的。",
    "Default is 0. That is, overlapping and/or book-ended features are clustered.": "默认值为0。即，重叠和/或两端对齐的特征被聚类。",
    "That is, only features in A are only counted towards coverage in B if they are the same strand. (-s)": "也就是说，只有在A中的特征，如果它们与B中的特征在同一链上，才会被计入B的覆盖度。（-s）",
    "Positions reported are one based.  Each position and depth follow the complete B feature. (-d)": "报告的位置是基于一的。每个位置和深度都遵循完整的B特征。（-d）",
    "Additonal columns after each feature in B: 1) depth 2) # bases at depth 3) size of B 4) % of B at depth. (-hist)": "在B文件中每个特征之后添加额外的列：1）深度 2）深度处的碱基数量 3）B的大小 4）深度处B的百分比。（-hist）",
    "E.g. if used on a 1000bp feature, -l 0.50, will add 500 bp “upstream”": "例如，如果在一个1000bp的特征上使用，-l 0.50，将会在“上游”增加500 bp",
    "For example. if used, -l 500 for a negative-stranded feature, it will add 500 bp to the end coordinate": "例如，如果使用，对于负链特征，-l 500 将在末端坐标上增加 500 个碱基对",
    "If set, regions without any coverage will also be reported.  (-bga)": "如果设置，没有覆盖的区域也将被报告。(-bga)",
    "Each bedGraph coverage value is multiplied by this factor before being reported. Useful for normalizing coverage by, e.g., reads per million (RPM). (-scale)": "每个 bedGraph 覆盖度值在报告前都会乘以这个因子。这在通过例如每百万个读段数（RPM）来标准化覆盖度时非常有用。（-scale）",
    "Combine all positions with a depth >= max into a single bin in the histogram. (-max)": "将所有深度大于等于最大值的位置合并为直方图中的单个bin。（-max）",
    "Instead of entire interval. (-5)": "而不是整个区间。(-5)",
    "Instead of entire interval. (-3)": "而不是整个区间。(-3)",
    "If the feature occupies the antisense strand, the sequence will be reverse complemented. (-s)": "如果特征占据反义链，序列将被反向互补。（-s）",
    "Columns may be comma-separated with each column must be explicitly listed. Or, ranges (e.g. 1-4) are also allowed. (-g)": "列可以是逗号分隔的，每个列必须明确列出。或者，也可以使用范围（例如 1-4）。（-g）",
    "Alignments are only retained if the overlap with the an interval in the BED file comprises at least this fraction of the BAM alignment's length.  For example, to require that the overlap affects 50% of the BAM alignment, use 0.50. (-f)": "如果与BED文件中的区间重叠的部分至少占BAM比对长度的这个比例，则仅保留比对。例如，要求重叠影响BAM比对的50%，请使用0.50。（-f）",
    "Just report the fact >=1 hit was found. (-u)": "只需报告发现>=1个命中的事实。（-u）",
    "Reports 0 for A entries that have no overlap with B. (-c)": "报告与B没有重叠的A条目的0。（-c）",
    "That is, only report hits in B that overlap A on the same strand. By default, overlaps are reported without respect to strand. (-s)": "即，只有在B中的命中与相同链上的A重叠时才报告。默认情况下，报告重叠时不考虑链。（-s）",
    "e.g. mouse, human (-org)": "例如：小鼠，人类（-组织）",
    "i.e. same number of nucleotide in each window": "即每个窗口中核苷酸的数量相同",
    "Used to create 'sliding' windows. Defaults to window size (non-sliding windows).": "用于创建“滑动”窗口。默认为窗口大小（非滑动窗口）。",
    "i.e. same number of windows, with varying window sizes": "即相同的窗口数量，窗口大小不同",
    "If set, the overlap between the BAM alignment and the BED interval must affect the above fraction of both the alignment and the BED interval. (-r)": "如果设置，BAM比对和BED区间之间的重叠必须影响比对和BED区间的上述分数。（-r）",
    "By default, hard-masking (that is, conversion to Ns) is performed. (-soft)": "默认情况下，执行硬掩蔽（即转换为Ns）。(-soft)",
    "That is, instead of masking with Ns, use another character. (-mc)": "也就是说，不是用Ns来掩蔽，而是使用另一个字符。（-mc）",
    "That is, overlapping and/or book-ended features are merged. (-d)": "即，重叠和/或两端封闭的特征被合并。（-d）",
    "Default counts non-duplicates only. (-D)": "默认只计算非重复项。（-D）",
    "Default counts pass-QC reads only (-F)": "默认计数通过质量控制的读取（-F）",
    "Default counts all alignments with MAPQ > -q argument, regardless of the BAM FLAG field. (-p)": "默认情况下，计算所有MAPQ大于-q参数的比对，而不考虑BAM标志位字段。（-p）",
    "If set, regions that are not overlapped by any file will also be reported. Requires a valid organism key for all input datasets": "如果设置，未被任何文件重叠的区域也将被报告。需要为所有输入数据集提供有效的生物体键",
    "Can be 0.0, N/A, - or any other value. (-filler)": "可以是0.0，N/A，-或任何其他值。（-filler）",
    "case-sensitive (-pattern)": "大小写敏感（-pattern）",
    "The columns must be listed in the following order: start1,end1,start2,end2": "列必须按照以下顺序列出：start1,end1,start2,end2",
    "Solely permute their location on the chromosome. By default, both the chromosome and position are randomly chosen. (-chrom)": "仅对其在染色体上的位置进行排列。默认情况下，染色体和位置都是随机选择的。(-chrom)",
    "Instead of choosing a position randomly among the entire genome (the default), first choose a chrom randomly, and then choose a random start coordinate on that chrom. This leads to features being ~uniformly distributed among the chroms, as opposed to features being distribute as a function of chrom size. (-chromFirst)": "而不是在基因组中随机选择一个位置（默认方式），而是首先随机选择一个染色体，然后在该染色体上随机选择一个起始坐标。这导致特征在染色体之间大致均匀分布，而不是根据染色体大小分布。（-chromFirst）",
    "If used, -l 500 for a negative-stranded feature, it will add 500 bp to the end coordinate": "如果使用，对于负链特征，-l 500 将在末端坐标上增加 500 个碱基对",
    "If set, regions without any coverage will also be reported. Requires a valid organism key for all input datasets": "如果设置，没有覆盖的区域也将被报告。需要为所有输入数据集提供有效的生物体键。",
    "Allows one to create assymetrical “windows”. Default is 1000bp. (-l)": "允许创建不对称的“窗口”。默认是1000个碱基对。（-l）",
    "Allows one to create assymetrical “windows”. Default is 1000bp. (-r)": "允许创建不对称的“窗口”。默认是1000个碱基对。(-r)",
    "In other words, just report the fact at least one overlap was found in B. (-u)": "换句话说，只需报告在B中至少发现了一个重叠的事实。（-u）",
    "Reports 0 for A entries that have no overlap with B (-c)": "报告与B没有重叠的A条目的0（-c）",
    "Default is to report solely the fraction of -i covered by each file": "默认情况下，只报告每个文件覆盖的-i部分的比例",
    "Default is to report solely the fraction of the input file covered by each file": "默认情况下，仅报告每个文件覆盖输入文件的比例",
    "If Selected, bedToBam will convert blocked BED features (e.g., gene annotaions) into 'spliced' BAM alignments by creating an appropriate CIGAR string": "如果选中，bedToBam 将把阻塞的 BED 特征（例如，基因注释）转换成 'spliced' BAM 对齐，通过创建一个合适的 CIGAR 字符串",
    "The type of BAM sorting you would like to apply to each image.": "您希望对每个图像应用的BAM排序类型。",
    "Default is to use chr:start-pos.": "默认使用 chr:start-pos。",
    "Built-in GFF files contain full annotation of a reference genome": "内置的GFF文件包含参考基因组的完整注释",
    "The reported distance for overlapping features will be 0": "报告的重叠特征之间的距离将为0",
    "Like -d, report the closest feature in B, and its distance to A as an extra column. However unlike -d, use negative distances to report upstream features": "像-d参数一样，报告B中距离A最近的一个特征，并将其距离作为额外的一列显示。然而，与-d参数不同的是，使用负距离来报告上游的特征。",
    "That is, we want close, yet not touching features only": "也就是说，我们只想要相互靠近但不接触的特征。",
    "BAM/@STD_BEDTOOLS_INPUT_LABEL@ format": "BAM/@STD_BEDTOOLS_INPUT_LABEL@ 格式",
    "Only report hits in B that overlap A on the same strand. By default, overlaps are reported without respect to strand": "仅报告在相同链上与A重叠的B中的命中。默认情况下，报告重叠时不考虑链。",
    "Positions reported are one based. Each position and depth follow the complete B feature": "报告的位置是一基于的。每个位置和深度都遵循完整的B特征",
    "Additional columns after each feature in A: 1) depth 2) # bases at depth 3) size of A 4) % of A at depth": "在A中的每个特征之后，增加额外的列：1）深度 2）在深度处的碱基数量 3）A的大小 4）在深度处A的百分比",
    "Report the mean depth of all positions in each A feature.": "报告每个A特征中所有位置的平均深度。",
    "if -f is 0.90 and -r is used, this requires that B overlap 90% of A and A _also_ overlaps 90% of B": "如果-f参数是0.90，并且使用了-r参数，这意味着B必须覆盖A的90%，同时A也必须覆盖B的90%。",
    "If -e is used with -f 0.90 and -F 0.10 this requires that either 90% of A is covered OR 10% of  B is covered. Without -e, both fractions would have to be satisfied": "如果同时使用 -e、-f 0.90 和 -F 0.10，这意味着要么 A 的 90% 被覆盖，要么 B 的 10% 被覆盖。如果不使用 -e，两个分数都需要满足。",
    "For example, if used, -l 500 for a negative-stranded feature, it will add 500 bp to the end coordinate": "例如，如果使用参数 `-l 500` 针对负链特征，它将在末端坐标上增加500个碱基对。",
    "If set, regions without any coverage will also be reported": "如果设置，没有覆盖的区域也将被报告",
    "Each bedGraph coverage value is multiplied by this factor before being reported. Useful for normalizing coverage by, e.g., reads per million (RPM).": "每个bedGraph覆盖度值在报告前都会乘以这个因子。这在通过例如每百万个读段数（RPM）来规范化覆盖度时非常有用。",
    "Combine all positions with a depth >= max into a single bin in the histogram": "将所有深度大于或等于最大值的位置合并为直方图中的单个箱子。",
    "Instead of entire interval": "而不是整个区间",
    "If the feature occupies the antisense strand, the sequence will be reverse complemented": "如果特征占据反义链，序列将被反转互补",
    "Columns may be comma-separated with each column must be explicitly listed. Or, ranges (e.g. 1-4) are also allowed.": "列可以是逗号分隔的，每个列都必须明确列出。或者，范围（例如 1-4）也是允许的。",
    "If enabled, then for -f 0.90 and -F 0.10 this requires that either 90% of A is covered OR 10% of B is covered, otherwise, both fractions would have to be satisfied.": "如果启用了，那么对于 -f 0.90 和 -F 0.10，这要求 A 的 90% 被覆盖或者 B 的 10% 被覆盖，否则，两个分数都需要满足。",
    "Just report the fact >=1 hit was found": "只需报告发现了一个或多个命中的事实",
    "Reports 0 for A entries that have no overlap with B": "报告与B没有重叠的A条目的0",
    "Only applies when used with -sorted option.": "仅在与-sorted选项一起使用时适用。",
    "That is, only report hits in B that overlap A on the same strand. By default, overlaps are reported without respect to strand.": "即，只报告在相同链上与A重叠的B中的命中。默认情况下，报告重叠时不考虑链。",
    "E.g. mouse, human": "例如：小鼠，人类",
    "I.e. same number of windows, with varying window sizes": "即相同的窗口数量，窗口大小不同。",
    "If set, the overlap between the BAM alignment and the BED interval must affect the above fraction of both the alignment and the BED interval": "如果设置，BAM比对和BED区间之间的重叠必须影响比对和BED区间的上述分数",
    "Multiple columns can be specified in a comma-delimited list": "多个列可以在逗号分隔的列表中指定",
    "By default, hard-masking (that is, conversion to Ns) is performed": "默认情况下，执行硬掩蔽（即转换为Ns）",
    "That is, instead of masking with Ns, use another character": "也就是说，不要使用Ns进行掩蔽，而是使用另一个字符。",
    "By default, only the word before the first space or tab is used": "默认情况下，只使用第一个空格或制表符前的单词",
    "That is, overlapping and/or book-ended features are merged": "即，重叠和/或首尾相连的特征被合并",
    "Default counts non-duplicates only": "默认只计算非重复项",
    "Default counts pass-QC reads only": "默认计数仅通过质量控制的读取",
    "Default counts all alignments with MAPQ > -q argument, regardless of the BAM FLAG field": "默认情况下，将计算所有MAPQ大于-q参数的比对，而不考虑BAM标志字段。",
    "Include regions that have zero coverage in all BedGraph datasets": "包含所有BedGraph数据集中覆盖度为零的区域",
    "Can be 0.0, N/A, - or any other value": "可以是0.0，N/A，- 或任何其他值",
    "Case-sensitive by default": "默认情况下区分大小写",
    "Solely permute their location on the chromosome. By default, both the chromosome and position are randomly chosen.": "仅对它们在染色体上的位置进行置换。默认情况下，染色体和位置都是随机选择的。",
    "Instead of choosing a position randomly among the entire genome (the default), first choose a chrom randomly, and then choose a random start coordinate on that chrom. This leads to features being ~uniformly distributed among the chroms, as opposed to features being distribute as a function of chrom size.": "而不是在基因组中随机选择一个位置（默认情况），首先随机选择一个染色体，然后在该染色体上随机选择一个起始坐标。这导致特征在染色体之间大致均匀分布，而不是根据染色体大小分布特征。",
    "Sort according to the chromosomes declared in a genome file": "根据基因组文件中声明的染色体进行排序",
    "Allows one to create assymetrical “windows”. Default is 1000bp.": "允许创建不对称的“窗口”。默认是1000个碱基对。",
    "In other words, just report the fact at least one overlap was found in B": "换句话说，只需报告在B中至少发现了一个重叠的事实。",
    "Sets --targets or --region options": "设置--targets或--region选项",
    "-t --targets": "-t --目标",
    "-r --region": "-区域",
    "Select how much control over the freebayes run you need": "选择您对 freebayes 运行所需的控制程度",
    "Sets --samples, --populations, --cnv-map, --trace, --failed-alleles, --varinat-input, --only-use-input-alleles, --haplotype-basis-alleles, --report-all-haplotype-alleles, --report-monomorphic options, --observation-bias, and --contamination-estimates": "设置 --samples, --populations, --cnv-map, --trace, --failed-alleles, --varinat-input, --only-use-input-alleles, --haplotype-basis-alleles, --report-all-haplotype-alleles, --report-monomorphic 选项，--observation-bias 和 --contamination-estimates",
    "--report-monomorphic": "-报告单态性",
    "--observation-bias; The format is [length] [alignment efficiency relative to reference] where the efficiency is 1 if there is no relative observation bias": "--观察偏差；格式为 [长度] [与参考相对比对效率]，如果没有相对观察偏差，效率为1",
    "--contamination-estimates; The format should be: sample p(read=R|genotype=AR) p(read=A|genotype=AA) Sample '*' can be used to set default contamination estimates.": "--污染估计；格式应该是：样本 p(读数=R|基因型=AR) p(读数=A|基因型=AA) 使用 '*' 作为样本可以设置默认的污染估计。",
    "Sets -P --pvar option": "设置 -P --pvar 选项",
    "Sets --theta, --ploidy, --pooled-discrete, and --pooled-continuous options": "设置 --theta、--ploidy、--pooled-discrete 和 --pooled-continuous 选项",
    "Sets --use-reference-allele and --reference-quality options": "设置 --use-reference-allele 和 --reference-quality 选项",
    "Sets -I, i, -X, -u, -n, --haplotype-length, --min-repeat-size, --min-repeat-entropy, and --no-partial-observations options": "设置 -I、i、-X、-u、-n、--haplotype-length（单倍型长度）、--min-repeat-size（最小重复大小）、--min-repeat-entropy（最小重复熵）和 --no-partial-observations（无部分观察）选项",
    "Sets -4, -m, -q, -R, -Y, -Q, -U, -z, -$, -e, -0, -F, -C, -3, -G, and -! options": "设置 -4、-m、-q、-R、-Y、-Q、-U、-z、-$、-e、-0、-F、-C、-3、-G 和 -! 选项",
    "Sets -Q, -U, -z, and $ options": "设置 -Q、-U、-z 和 $ 选项",
    "-$amp; --read-snp-limit N": "-$amp; --read-snp-limit N",
    "Sets -k, -w, -V, and -a options": "设置 -k、-w、-V 和 -a 选项",
    "Sets --base-quality-cap, --experimental-gls, and --prob-contamination options.": "设置 --base-quality-cap、--experimental-gls 和 --prob-contamination 选项。",
    "--experimental-gls; Incorporate partial observations. This is the default when contamination estimates are provided. Optimized for diploid samples.": "--experimental-gls；纳入部分观测值。当提供污染估计时，默认使用此选项。针对二倍体样本进行了优化。",
    "Sets --report-genotypes-likelihood-max, -B, --genotyping-max-banddepth, -W, -N, S, -j, -H, -D, -= options": "设置 --report-genotypes-likelihood-max, -B, --genotyping-max-banddepth, -W, -N, S, -j, -H, -D, -= 选项",
    "-S --genotype-variant-threshold": "-S --基因型变异阈值",
    "Iterate the left-realignment no more than this many times": "迭代左对齐不超过这么多次",
    "These are available for only a few genomes": "这些仅适用于少数基因组",
    "If this sequence uses non-standard genetic code, select one from these options": "如果这个序列使用非标准遗传密码，请从这些选项中选择一个",
    "Format is one transcript ID per line": "格式是每行一个转录本ID",
    "For E. coli K12, for example, you may want to use 'EcK12'. Note: Spaces are not allowed in the name and will get converted to underscores.": "对于大肠杆菌K12，例如，您可能想使用'EcK12'。注意：名称中不允许有空格，空格将被转换为下划线。",
    "Specify format for annotations you are using to create SnpEff database": "指定用于创建 SnpEff 数据库的注释格式",
    "This Genbank file will be used to generate snpEff database": "这个Genbank文件将被用来生成snpeff数据库",
    "This will generate an additional dataset containing all sequences from Genbank file in FASTA format": "这将生成一个额外的数据集，其中包含Genbank文件中所有序列的FASTA格式。",
    "Genbank sequences have version numbers such as B000564.2. This option removes them leaving only B000564": "Genbank序列具有版本号，例如B000564.2。此选项将删除它们，只留下B000564。",
    "This GFF file will be used to generate snpEff database": "这个GFF文件将用于生成snpEff数据库",
    "This GTF file will be used to generate snpEff database": "这个GTF文件将用于生成snpeff数据库",
    "Databases matching this expression will be listed. Here you can enter text or regular expression. For example, to show only mouse databases use 'Mouse'. Note that this parameter is case-sensitive.": "匹配此表达式的数据库将被列出。您可以在这里输入文本或正则表达式。例如，要仅显示鼠标数据库，请使用'Mouse'。请注意，此参数是区分大小写的。",
    "Databases matching this expression WILL NOT BE listed. Here you can enter text or regular expression. For excample, to avoid all ENSEMBL bundles enter 'ENSEMBL'. Note that this parameter is case-sensitive.": "匹配此表达式的数据库将不会被列出。您可以在这里输入文本或正则表达式。例如，为了避免列出所有ENSEMBL包，输入'ENSEMBL'。请注意，此参数是区分大小写的。",
    "The list of available databases can be obtained with 'SnpEff databases' tool": "可用数据库的列表可以通过使用'SnpEff databases'工具获得",
    "The ID field for a variant in input will be assigned from a matching variant in this file.": "输入文件中的变体的ID字段将从该文件中匹配的变体分配。",
    "List is a comma separated list of fields. When blank, all INFO fields are included": "列表是一个用逗号分隔的字段列表。当为空时，包括所有INFO字段",
    "Read more about TFAM at http://pngu.mgh.harvard.edu/~purcell/plink/data.shtml#tr": "在 http://pngu.mgh.harvard.edu/~purcell/plink/data.shtml#tr 上阅读更多关于 TFAM 的信息",
    "Name to append to the 'Cases' or 'Controls' tags": "要添加到“病例”或“对照组”标签的名称",
    "Separated by spaces. See help below for an explanation": "以空格分隔。有关解释，请参见下面的帮助。",
    "When variants have more than one effect, lists one effect per line, while all other parameters in the line are repeated across mutiple lines": "当变异具有多种影响时，每行列出一种影响，而该行中的所有其他参数在多行中重复。",
    "Separate multiple fields in one column with this character, e.g. a comma, rather than a column for each of the multiple values": "在一个列中用这个字符分隔多个字段，例如逗号，而不是为每个多个值分别设立一个列。",
    "Represent empty fields with this value, rather than leaving them blank": "使用这个值来表示空字段，而不是让它们空白",
    "Need help? See the tool help below for some examples.": "需要帮助？请参阅下面工具帮助中的一些示例。",
    "Select one or more datasets for construction of value sets. The datasets are supposed to specify one value per line. See also: the help section on Sets below.": "选择一个或多个用于构建值集的数据集。这些数据集应该每行指定一个值。另见：下面的帮助部分中的“集合”部分。",
    "Select variants that do not match the filter expression.": "选择不符合过滤表达式的变体。",
    "The value provided here will be added to the FILTER field of variants that are NOT selected by the expression above. Selected variants, in contrast, will have their FILTER field cleared and set to PASS.": "这里提供的值将被添加到未被上述表达式选中的变体的FILTER字段中。相比之下，被选中的变体将清除其FILTER字段，并将其设置为PASS。",
    "The FILTER field of variants that are selected by the expression above will be checked for existence of the specified value. When the value is found it will be removed, but other values will be preserved. If the removed value was the only value in a FILTER field, that field will be set to the '.' (missing) value.": "上述表达式所选择的变体的FILTER字段将检查是否存在指定的值。当找到该值时，将将其删除，但其他值将被保留。如果被删除的值是FILTER字段中唯一的值，则该字段将被设置为'.'（缺失）值。",
    "The value provided here will be added to the FILTER field of variants that are selected by the expression above. USE WITH CARE: The tool will NOT add a corresponding FILTER entry to the VCF header of the output, i.e. will produce output that violates the VCF format specification. To avoid issues with downstream tools it may be necessary to add such a header line using standard text processing tools before proceeding.": "这里提供的值将被添加到由上述表达式选择的变体的FILTER字段中。谨慎使用：该工具不会向输出的VCF头文件添加相应的FILTER条目，即会产生违反VCF格式规范的输出。为了避免与下游工具发生问题，可能需要在使用标准文本处理工具之前添加这样一个头文件行。",
    "Filter out (exclude) VCF entries that match any interval in the BED files": "过滤掉（排除）与BED文件中的任何区间匹配的VCF条目",
    "Read phenotype values from the 3rd column of the specified space- or tab-delimited file, instead of the .fam or .ped file.": "从指定的以空格或制表符分隔的文件的第3列中读取表型值，而不是从 .fam 或 .ped 文件中读取。",
    "Accepts one or more space/tab-delimited text files with sample IDs, and removes all unlisted samples from the current analysis;": "接受一个或多个以空格/制表符分隔的文本文件，其中包含样本ID，并从当前分析中移除所有未列出的样本；",
    "Accepts text files with family IDs in the first column, and keeps entire families.": "接受第一列包含家族ID的文本文件，并保留整个家族。",
    "Accepts one or more space/tab-delimited text files with sample IDs, and removes all listed samples from the current analysis": "接受一个或多个用空格或制表符分隔的文本文件，文件中包含样本ID，从当前分析中移除所有列出的样本。",
    "Acceptz text files with family IDs in the first column, and removes entire families.": "接受第一列包含家族ID的文本文件，并删除整个家族。",
    "Accepts one or more text file(s) with variant IDs , and removes all unlisted variants from the current analysis": "接受一个或多个包含变异ID的文本文件，并从当前分析中移除所有未列出的变异",
    "Input file is input in set range format.": "输入文件是集合范围格式的输入。",
    "Accepts one or more text file(s) with variant IDs , and removes all listed variants from the current analysis": "接受一个或多个包含变异ID的文本文件，并从当前分析中移除所有列出的变异。",
    "Excludes all variants not on the listed chromosome(s). Can be listed as single chromsome, a hyphenated range, or comma separated list.": "排除所有不在列出的染色体上的变异。可以作为单个染色体列出，也可以作为带连字符的范围，或者以逗号分隔的列表。",
    "Excludes all variants on the listed chromosome(s). Can be listed as single chromsome, a hyphenated range, or comma separated list.": "排除所有在列出的染色体上的变异。可以作为单个染色体列出，用连字符分隔的范围，或用逗号分隔的列表。",
    "Allows specified extra chromosomes/scaffolds not normally listed. ex. chr1_gl000191_random.": "允许指定一些通常不列出的额外染色体/支架。例如：chr1_gl000191_random。",
    "Variant ID. Excludes all variants on different chromosomes than the named variant, as well as those with smaller base-pair position values. If they are used together but the --from variant is after the --to variant, they are automatically swapped.": "变异ID。排除所有与指定变异不同染色体上的变异，以及那些具有较小碱基对位置值的变异。如果它们一起使用，但是--from变异在--to变异之后，它们将自动交换。",
    "Variant ID. Excludes all variants on different chromosomes than the named variant, as well as those with larger base-pair position values. If they are used together but the --from variant is after the --to variant, they are automatically swapped.": "变异ID。排除所有与命名变异不同的染色体上的变异，以及那些具有更大碱基对位置值的变异。如果它们一起使用，但--from变异在--to变异之后，它们会自动交换。",
    "Use this OR SNP variant to exclude": "使用这个或SNP变异来排除",
    "Use this OR SNP variant to include": "使用这个或SNP变体来包含",
    "All variants with physical position no more than half the specified kb distance (decimal permitted) from the named variant are loaded as well": "所有与指定的kb距离（允许小数）的一半以内的物理位置的变体也会被加载。",
    "These flags let you use physical positions to specify a variant range to load. Must also have specified a chromosome in above settings.": "这些标志允许您使用物理位置来指定要加载的变体范围。还必须在上述设置中指定了染色体。",
    "To reduce the potential for confusion, PLINK 2 normally errors out when multiple variant-inclusion filters (--extract[-intersect], --extract-col-cond, --from/--to, --from-bp/--to-bp, --snp, --snps) are specified, since it may not be obvious whether the intersection or union will be taken. --force-intersect allows the run to proceed; the set intersection will be taken.": "为了减少混淆的可能性，PLINK 2通常在指定多个变异包含过滤器（--extract[-intersect]，--extract-col-cond，--from/--to，--from-bp/--to-bp，--snp，--snps）时会报错，因为可能不清楚是取交集还是并集。--force-intersect允许运行继续；将取集合交集。",
    "Removes variants at random by retaining each variant with probability p": "随机移除变异，保留每个变异的概率为 p",
    "Removes variants at random until only n remain": "随机移除变体，直到只剩下n个",
    "Excludes one variant from each pair closer than the given bp count": "排除每对距离小于给定碱基对数的一个变异",
    "Removes samples at random by retaining each sample with probability p": "随机移除样本，每个样本保留的概率为p",
    "Removes samples at random until only n remain.": "随机移除样本，直到只剩下n个为止。",
    "filters out all variants with missing call rates exceeding the provided value (default 0.1) to be removed": "过滤掉所有缺失调用率超过提供值（默认为0.1）的变异体，以进行移除。",
    "filters out all samples with missing call rates exceeding the provided value (default 0.1) to be removed": "过滤掉所有缺失调用率超过提供值（默认为0.1）的样本以进行移除",
    "Filters out all variants with allele frequency below the provided threshold": "过滤掉所有等位基因频率低于提供阈值的变异",
    "Filters out all variants with allele frequency above the provided threshold": "过滤掉所有等位基因频率高于提供阈值的变体",
    "Filters out all variants with allele counts below the provided threshold": "过滤掉所有等位基因计数低于提供阈值的变异",
    "filters out all variants with allele counts above the provided threshold": "过滤掉所有等位基因计数超过提供阈值的变异",
    "Set Hardy-Weinberg equilibrium tests": "设置哈迪-温伯格平衡检验",
    "Filters out all variants which have Hardy-Weinberg equilibrium exact test p-value below the provided threshold. It is recommended setting a low threshold—serious genotyping errors often yield extreme p-values like 1e-501 which are detected by any reasonable configuration of this test, while genuine SNP-trait associations can be expected to deviate slightly from Hardy-Weinberg equilibrium (so it is dangerous to choose a threshold that filters out too many variants).": "过滤掉所有哈迪-温伯格平衡精确检验 p 值低于所提供阈值的变异。建议设置一个较低的阈值——严重的基因分型错误通常会得到像 1e-501 这样的极端 p 值，这些值会被这个测试的任何合理配置所检测到，而真正的 SNP-性状关联预计会稍微偏离哈迪-温伯格平衡（因此选择一个过滤掉太多变异的阈值是危险的）。",
    "Filter by phenotypes, experiment, and founder state.": "根据表型、实验和创始人状态进行过滤。",
    "How to deal with ambiguous sex phenotypes": "如何处理模糊的性别表型",
    "Include nonfounders in --freq[x] or --maf/--max-maf/--hwe calculations": "在 --freq[x] 或 --maf/--max-maf/--hwe 计算中包括非创始人",
    "Create a new text fileset, after applying sample/variant filters and other operations": "创建一个新的文本文件集，在应用样本/变体过滤器和其他操作之后",
    "Given a file containing a list of SNPs with A/C/G/T alleles, --flip swaps A↔T and C↔G.": "给定一个包含SNPs列表的文件，其中包含A/C/G/T等位基因，--flip参数会交换A↔T和C↔G。",
    "Replaces missing IDs. The parameter taken by these flags is a special template string, with a @ where the chromosome code should go, and a # where the base-pair position belongs.": "替换缺失的ID。这些标志采用的参数是一个特殊的模板字符串，其中@表示染色体代码的位置，#表示碱基对位置。",
    "Length threshold to rename alleles. Recommended default is 23": "重命名等位基因的长度阈值。建议的默认值是23",
    "By default, the new value is read from column 2 and the (old) variant ID from column 1, but you can adjust these positions with the second and third parameters.": "默认情况下，新值从第2列读取，而（旧的）变异ID从第1列读取，但您可以通过第二和第三个参数调整这些位置。",
    "Either a nonnegative integer, in which case it indicates the number of lines to skip at the top of the file, or a single nonnumeric character, which causes each line with that leading character to be skipped.": "可以是正整数，表示在文件顶部要跳过的行数，或者是一个非数字字符，这会导致跳过以该字符开头的每一行。",
    "These cannot be used with any other commands.": "这些不能与任何其他命令一起使用。",
    "Writes out writes autosomal Hardy-Weinberg equilibrium exact test statistics": "输出自交 Hardy-Weinberg平衡精确检验统计数据",
    "produces sample-based and variant-based missing data reports": "生成基于样本和基于变异的缺失数据报告",
    "Compute observed and expected homozygous/heterozygous genotype counts for each sample, and reports method-of-moments F coefficient estimates": "计算每个样本的观察到的和期望的纯合子/杂合子基因型计数，并报告矩法F系数估计值",
    "Since two-variant r2 only makes sense for biallelic variants, these collapse multiallelic variants down to most common allele vs. the rest.": "由于双变异r2仅适用于双等位基因变异，因此这些将多等位基因变异折叠为最常见的等位基因与其余等位基因。",
    "Use IBS values calculated via --genome to perform complete linkage clustering": "使用通过 --genome 参数计算的IBS值执行完全连锁聚类",
    "Return a Haploview-friendly multidimensional scaling report": "返回一个适合Haploview的多维尺度分析报告",
    "Request an adaptive or max(T) permutation test on the additive effect.": "请求对加性效应进行自适应或最大（T）排列检验。",
    "Cause offspring genotypes to be regenerated via gene-dropping in the permutation test.": "在排列测试中，通过基因丢弃使后代基因型重新生成。",
    "Cause the permutation test report to include permutation counts instead of frequencies.": "导致排列检验报告包含排列计数而不是频率。",
    "Report allele counts instead of frequencies": "报告等位基因计数而不是频率",
    "Remove covariate-specific lines from the main report": "从主报告中移除特定协变量的行",
    "Add genotype x covariate interactions to the model.": "在模型中添加基因型×协变量交互作用。",
    "Average is the default, but it can be changed to sum.": "平均值是默认设置，但可以更改为求和。",
    "Perform and return results of IBS/IBD computation": "执行并返回IBS/IBD计算的结果",
    "Minimum PI_HAT value": "最小 PI_HAT 值",
    "Maximum PI_HAT value": "最大 PI_HAT 值",
    "Minimum distance between informative pairs of SNPs used in the pairwise population concordance (PPC) inn kbs": "在成对人群一致性（PPC）中使用的有信息的SNP对之间的最小距离为kbs",
    "By default, indel error rates are estimated from a subsample of the input sequencing data for each sample. This step can be disabled so that the tool reverts to precomputed indel error rates reflecting an intermediate point between different sequencing assays. Alternatively, all data (from sufficiently large chromosomes) can be used to obtain the estimate, but this may greatly increase runtime.": "默认情况下，每个样本的插入/缺失错误率是通过对输入测序数据的子样本进行估计得到的。可以禁用此步骤，使工具恢复到使用预先计算的插入/缺失错误率，这反映了不同测序检测之间的中间点。或者，可以使用所有数据（来自足够大的染色体）来获得估计，但这可能会大大增加运行时间。",
    "Provide ploidy file in VCF. The VCF should include one sample column per input sample labeled with the same sample names found in the input BAM/CRAM RG header sections. Ploidy should be provided in records using the FORMAT/CN field, which are interpreted to span the range [POS+1, INFO/END]. Any CN value besides 1 (haploid regions) or 0 (regions expected to be absent) will be treated as 2.": "请提供包含等位基因组（ploidy）信息的VCF文件。该VCF文件应包含每个输入样本的一列样本，并使用与输入BAM/CRAM文件中的RG头部部分中找到的相同样本名称进行标记。等位基因组应使用FORMAT/CN字段在记录中提供，这些字段被解释为跨越范围[POS+1, INFO/END]。除了1（单倍体区域）或0（预期缺失的区域）之外的任何CN值都将被视为2。",
    "For the specified chromosome the sequencing data will be treated as a pooled sample: variants on it will be called with continuous frequencies and scored using a simple Poisson noise model. May be applied to the mitochondrial genome, for example.": "对于指定的染色体，测序数据将被视为混合样本：在其上的变异将以连续频率进行调用，并使用简单的泊松噪声模型进行评分。例如，这可能适用于线粒体基因组。",
    "If you have regions of special interest, for which you do not want site-information to be binned into blocks in the gVCF output, but would want to have calling statistics reported separately, you can specify those regions through a BED dataset here.": "如果您有特别感兴趣的区域，您不希望在gVCF输出中将位点信息分箱到区块中，而是希望单独报告调用统计信息，您可以通过这里指定一个BED数据集来指定这些区域。",
    "Don't use reads with MAPQ less than this value for variant calling.": "不要使用MAPQ值小于该值的reads进行变体调用。",
    "In bam or cram format.": "在BAM或CRAM格式中。",
    "If the depth filter is not skipped, all variants which occur at a depth greater than depthFilterMultiple*chromosome mean depth will be filtered out.": "如果深度过滤器没有被跳过，所有出现在深度大于 depthFilterMultiple*染色体平均深度的变异都将被过滤掉。",
    "Somatic SNV calls are filtered at sites where greater than this fraction of basecalls have been removed by the mismatch density filter in either sample.": "体细胞单核苷酸变异(SNV)调用在以下位置被过滤：在任一样本中，由于错配密度过滤器移除的碱基呼叫的比例大于这个分数。",
    "Somatic SNV calls are filtered at sites where greater than this fraction of overlapping reads contain deletions which span the SNV call site.": "体细胞SNV（单核苷酸变异）调用在重叠读取中超过此分数的读取包含跨越SNV调用位点的删除时被过滤。",
    "Somatic indel calls are filtered if greater than this fraction of basecalls in a window extending 50 bases to each side of an indel's call position have been removed by the mismatch density filter.": "体细胞indel调用如果在一个窗口中，这个窗口从indel的调用位置向两侧各延伸50个碱基，如果在该窗口中被错配密度过滤器移除的碱基调用比例大于这个分数，则会被过滤掉。",
    "Prior probability of a somatic snv or indel.": "体细胞SNV或indel的先验概率。",
    "Probability of an snv or indel noise allele NB: in the calling model a noise allele is shared in tumor and normal samples, but occurs at any frequency.": "一个snv（单核苷酸变异）或indel（插入或缺失）噪声等位基因的概率：在调用模型中，噪声等位基因在肿瘤和正常样本中共享，但可以出现在任何频率。",
    "Somatic indel noise factor.": "体细胞indel噪声因子。",
    "Fraction of snv noise attributed to strand-bias. It is not recommended to change this setting. However, if it is essential to turn the strand bias penalization off, the following is recommended: Assuming the current value of ssnvNoiseStrandBiasFrac is 0.5, (1) set ssnvNoiseStrandBiasFrac = 0 (2) divide the current ssnvNoise value by 2.": "单核苷酸变异（SNV）噪声中归因于链偏倚（strand-bias）的部分。不建议更改此设置。然而，如果必须关闭链偏倚惩罚，建议采取以下措施：假设当前的ssnvNoiseStrandBiasFrac值为0.5，（1）将ssnvNoiseStrandBiasFrac设置为0（2）将当前的ssnvNoise值除以2。",
    "Minimum MAPQ score for reads at tier1.": "在tier1中读取的最小MAPQ分数。",
    "Minimum MAPQ score for reads at tier2.": "在tier2中读取的最小MAPQ分数。",
    "Somatic quality score (QSS_NT, NT=ref) below which somatic SNVs are marked as filtered.": "体细胞质量分数（QSS_NT，NT=参考序列）低于此分数的体细胞单核苷酸变异（SNV）将被标记为已过滤。",
    "Somatic quality score (QSI_NT, NT=ref) below which somatic indels are marked as filtered.": "体细胞质量分数（QSI_NT，NT=参考）低于此值时，体细胞插入缺失将被标记为过滤。",
    "Tolerance of tumor contamination in the normal sample.": "正常样本中肿瘤污染的容忍度。",
    "median+s*MAD, deletions only (--mad-cutoff)": "中位数+s*MAD（平均绝对偏差），仅删除（--mad-cutoff）",
    "Replace regions with N": "用N替换区域",
    "Output variants in the form of IUPAC ambiguity codes": "输出变异形式为IUPAC歧义代码",
    "Instead of removing sequence, insert CHAR for deletions": "而不是删除序列，对于删除插入 CHAR",
    "Highlight insertions in uppercase (uc) or lowercase (lc), leaving the rest as is": "将插入部分以大写（uc）或小写（lc）突出显示，其余部分保持不变。",
    "Highlight substitutions in uppercase (uc) or lowercase (lc), leaving the rest as is": "将替换的字符以大写（uc）或小写（lc）突出显示，其余部分保持不变。",
    "Replace regions according to the next --mask-with option": "根据下一个 --mask-with 选项替换区域",
    "Replace with CHAR (skips overlapping variants); change to uppercase (uc) or lowercase (lc)": "替换为 CHAR（跳过重叠的变体）；改为大写（uc）或小写（lc）",
    "It allows to set positions with no supporting evidence to N (or any other character)": "它允许将没有支持证据的位置设置为N（或任何其他字符）",
    "(median + 3 * mad) (--mad-cutoff)": "（中位数 + 3 * 中位数绝对偏差）(--mad-cutoff)",
    "Setting both max_dist_linear and max_dist sets thresholds to minimum of max_dist and max_dist_linear * sv_length": "设置 max_dist_linear 和 max_dist 都会将阈值设置为 max_dist 和 max_dist_linear * sv_length 的最小值",
    "Requires reference genome": "需要参考基因组",
    "(to NCBI standards - without 'chr' - by default)": "（按照NCBI标准 - 默认不包含'chr'）",
    "Select at least two datasets (or a dataset collection) with variant lists (see the tool help below for format details). Datasets are expected to represent individual samples and dataset names will be used as sample identifiers.": "选择至少两个数据集（或一个数据集集合），包含变异列表（有关格式详情，请参阅工具帮助）。预期数据集代表个体样本，数据集名称将用作样本标识符。",
    "Only plot variants with an intrasample frequency above this threshold in at least one sample.": "仅绘制在一个样本中具有高于此阈值的内样本频率的变异。",
    "Width:Height ratio of individual heatmap cells": "单个热图单元格的宽度：高度比率",
    "VCF file with DCS mutations. See Help section below for a detailed explanation.": "包含DCS突变的VCF文件。有关详细说明，请参见下面的帮助部分。",
    "BAM file with aligned DCS reads.": "带有对齐的DCS读取的BAM文件。",
    "TABULAR file with aligned families.": "带有对齐家族的表格文件。",
    "BAM file with aligned SSCS reads.": "带有对齐的单分子共识序列（SSCS）读取的BAM文件。",
    "BAM file with aligned raw reads of selected tags.": "带有选定标签对齐的原始读取的BAM文件。",
    "JSON file generated by DCS mutations to tags/reads": "由DCS突变生成的JSON文件，用于标记/读取",
    "JSON file generated by DCS mutations to SSCS stats.": "由DCS突变生成的SSCS统计的JSON文件。",
    "Integer threshold for displaying mutations. Only mutations occuring in DCS of less than thresh tags are displayed. Default of 0 displays all.": "显示突变的整数阈值。只有在DCS中出现的标签数少于阈值的突变才会显示。默认值为0，显示所有突变。",
    "Integer threshold for Phred quality score. Only reads higher than this threshold are considered. Default = 20.": "Phred质量分数的整数阈值。只有高于此阈值的读段才会被考虑。默认值 = 20。",
    "Integer threshold for assigning mutations at start and end of reads to lower tier. Default 10.": "分配突变到读取的起始和终止位置的整数阈值。默认值为10。",
    "Count chimeric variants and correct the variant frequencies.": "计算嵌合变异并校正变异频率。",
    "Define paired-end options to be applied to one specific read group.": "定义成对末端选项，以便应用到一个特定的读组。",
    "All settings below will only be applied to reads belonging to the specified read group. The value provided here must correspond to one of the read group IDs defined in the main input and the discordant pairs datasets.": "所有下面的设置将只应用于属于指定读组的读段。这里提供的值必须与主输入和不一致对数据集中定义的一个读组ID相对应。",
    "Define split-reads options to be applied to one specific sample.": "定义拆分读取选项，以应用于一个特定的样本。",
    "All settings below will only be applied to reads of the specified sample. The value provided here must correspond to one of the read group SM values defined in the main input and the split-reads datasets.": "所有下面的设置将只应用于指定样本的读取。这里提供的值必须对应于主输入和拆分读取数据集中定义的一个读取组SM值。",
    "Duplicate reads can be detected and removed with the help of samblaster. If you have already marked duplicates in the input dataset with another tool, you can choose to skip duplicate detection and remove only pre-marked duplicates. Not handling duplicates is only recommended if you have already deduplicated your input with other tools.": "重复的读段可以通过samblaster工具检测并移除。如果你已经使用另一个工具在输入数据集中标记了重复项，你可以选择跳过重复检测并仅移除预先标记的重复项。只有在你已经使用其他工具对输入进行了去重，才推荐不处理重复项。",
    "You can specify a range of acceptable coverage values at variant sites by setting minimum and maximum coverage. Set either value to zero to make the range unbounded on the corresponding side, or disable filtering based on coverage by setting both values to zero.": "您可以通过设置最小和最大覆盖度来指定变体位点上可接受的覆盖度范围。将任一值设置为零可以使范围在相应侧无界，或者通过将两个值都设置为零来禁用基于覆盖度的过滤。",
    "The minimum coverage at a site required to keep variants": "在位点上保留变异所需的最小覆盖度",
    "The maximum coverage at a site allowed to keep variants": "允许保留变异位点的最大覆盖度",
    "You can specify a range of acceptable allele frequencies at variant sites by setting minimum and maximum AF values. Set either value to zero to make the range unbounded on the corresponding side, or disable filtering based on allele frequency by setting both values to zero.": "您可以通过设置最小和最大等位基因频率（AF）值来指定变异位点上可接受的等位基因频率范围。将任一值设置为零可以使相应侧的范围不受限制，或者通过将两个值都设置为零来禁用基于等位基因频率的过滤。",
    "To keep a variant its allele needs to be observed at this frequency at least.": "为了保留一个变体，它的等位基因至少需要在这个频率被观察到。",
    "To keep a variant its allele is allowed to be observed at this frequency at most.": "为了保留一个变体，其等位基因的观察频率最多被允许为这个频率。",
    "With compound filtering a variant is filtered only if it fails the strand-bias filter configured above AND has 85% of its supporting reads mapped to one reference genome strand. This guards against filtering based on statistically significant, but minor strand bias effects at high-coverage sites.": "通过复合过滤，只有在变体未能通过上述配置的链偏倚过滤器，并且其支持的读取中有85%映射到一个参考基因组链时，才会对其进行过滤。这可以防止基于高覆盖位点的统计学显著但轻微的链偏倚效应进行过滤。",
    "By default, indels are not filtered based on strand bias.": "默认情况下，插入/缺失(indels)不会根据链偏倚进行过滤。",
    "Note: To have indels included in the called variants you HAVE to preprocess your input data using lofreq indelqual to include indel qualities. In addition, it is highly recommended that you calculate and make use of indel alignment qualities. See the detailed tool help below.": "注意：为了在调用的变异中包含插入/缺失（indels），您必须使用lofreq indelqual预处理输入数据以包含插入/缺失质量。此外，强烈建议您计算并利用插入/缺失对齐质量。请参阅下面的详细工具帮助。",
    "Do not attempt variant calling at sites that are not covered by at least this number of reads (default: 1)": "不要在未被至少这么多读段覆盖的位点尝试变异调用（默认值：1）",
    "For efficiency, don not consider more than this number of reads at any site (default: 1,000,000)": "为了提高效率，不要在任何位点考虑超过这个数目的读取（默认：1,000,000）",
    "Applies to paired-end reads only. If set to true, reads from pairs that are flagged as non-proper pairs (SAM/BAM FLAG field 2) will be used in variant calling. The default is to ignore such reads.": "仅适用于配对末端读取。如果设置为true，则在变异调用中将使用标记为非适当配对（SAM/BAM FLAG字段2）的配对中的读取。默认情况下是忽略此类读取。",
    "For variant calling at any given site, do not consider reads for which the base at that site has a base quality less than this value (default: 6)": "对于任何给定位点的变异调用，不考虑在该位点的碱基质量低于此值的读取（默认值：6）",
    "For variant calling at any given site, do not consider reads that support a non-reference allele at the site if that base has a base quality less than this value (default: 6). Note: this setting will have no effect if the specified value is less than the general Minimum baseQ above.": "对于任何给定位点的变体调用，如果该位点支持非参考等位基因的读取其碱基质量小于该值（默认值：6），则不考虑该读取。注意：如果指定的值小于上面常规的最小碱基质量（Minimum baseQ），则此设置将不会产生任何效果。",
    "After filtering reads according to the Minimum baseQ settings above, you can choose to modify the base qualities of the non-reference bases in the surviving reads before performing further calculations.": "在根据上述的最小baseQ设置过滤读取后，你可以选择在进行进一步计算之前修改存活读取中非参考碱基的碱基质量。",
    "Choose here whether you want to incorporate base and/or indel alignment qualities into lofreq's joint quality model. If you have previously computed and stored (using lofreq alnqual) any of these quality scores into your input dataset, you can tell the tool to reuse them. Alternatively, the tool can calculate the necessary scores on the fly.": "在这里选择是否要将碱基和/或插入/缺失对齐质量纳入lofreq的联合质量模型中。如果您之前已经计算并存储了这些质量分数（使用lofreq alnqual），您可以告诉工具重用它们。或者，工具可以即时计算所需的分数。",
    "For variant calling at any given site, do not consider reads with a mapping quality (MAPQ) less than this value (default: 0 = do not filter on read mapping quality).": "对于任何给定位点的变体调用，不考虑映射质量（MAPQ）小于此值的读取（默认值：0 = 不根据读取映射质量进行过滤）。",
    "For the joint quality model at any site, cap the mapping quality of reads at this value (default: 255 = do not cap mapping qualities). Notes: 1) You cannot cap mapping qualities at less than the minimum MAPQ defined above. If you try, MAPQs will be capped at that minimum quality instead. 2) The special MAPQ value 255 is used by many tools to indicate undefined mapping quality, and lofreq call will ignore such reads during variant calling. Capping, however, will turn 255 into a regular MAPQ score. Thus, if you need to avoid using such reads, you should filter out MAPQ 255 reads from your input data with other tools before using lofreq call with MAPQ capping.": "对于任何位点的联合质量模型，将读取的映射质量限制在此值（默认值：255 = 不限制映射质量）。注意：1）您不能将映射质量限制在上述定义的最小MAPQ以下。如果您尝试这样做，MAPQ将被限制在该最小质量。2）特殊的MAPQ值255被许多工具用来表示未定义的映射质量，lofreq call在变异调用期间将忽略这样的读取。然而，限制将把255变成一个常规的MAPQ分数。因此，如果您需要避免使用这样的读取，您应该在使用具有MAPQ限制的lofreq call之前，使用其他工具从输入数据中过滤掉MAPQ 255读取。",
    "For the calculation of the source quality, replace all base qualities of non-match bases with this value (default: -1 = use original base qualities).": "在计算源质量时，将所有非匹配碱基的碱基质量替换为此值（默认值：-1 = 使用原始碱基质量）。",
    "Mismatches caused by known true variants in your samples should not lower the source quality estimate. If any read in your input has a base that is non-reference, but supports one of the known variants in the specified VCF datasets, that base will not be treated as a mismatch.": "由样本中已知的真实变异引起的不匹配不应降低源质量估计。如果您的输入中有任何读取的碱基是非参考的，但支持指定的VCF数据集中的已知变异之一，则该碱基将不被视为不匹配。",
    "At any site, do not use reads for variant calling, if their calculated joint quality at that site is lower than this value (default: 0 = do not filter based on joint quality)": "在任何位点，如果计算出的联合质量低于该值，则不要使用reads进行变体调用（默认值：0 = 不基于联合质量进行过滤）",
    "At any site, do not use reads for variant calling, if they support a non-reference allele at that site and their calculated joint quality at the site is lower than this value (default: 0 = do not filter based on joint quality). Note: this setting has no effect if the specified value is smaller than the general Minimum joinedQ set above.": "在任何位点，如果读取支持该位点的非参考等位基因，并且它们在该位点计算出的联合质量低于这个值，则不要使用这些读取进行变异调用（默认值：0 = 不根据联合质量进行过滤）。注意：如果指定的值小于上面设置的一般最小联合质量（Minimum joinedQ），则此设置将不会产生任何效果。",
    "After filtering according to the Minimum joinedQ settings above, set the joint quality values for all reads surviving filtering and supporting a non-reference allele to this value (default: 0 = use the original calculated joint quality).": "根据上面提到的最小联合质量（Minimum joinedQ）设置进行过滤后，为所有通过过滤并支持非参考等位基因的读取设置联合质量值为此值（默认值：0 = 使用原始计算的联合质量）。",
    "Set to zero to determine dynamically from actual number of variant tests performed. Dynamic detection will calculate separate correction factors for SNVs and indels": "设置为零以根据实际执行的变体测试数量动态确定。动态检测将为单核苷酸变异(SNVs)和小片段插入/缺失(indels)分别计算校正因子。",
    "Sample at most this number of reads per amplicon and strand. default=0 (use all reads)": "样本每个扩增子和链最多读取这个数量的读数。默认值=0（使用所有读数）",
    "Screening files must be stored in the 'primer_scheme_bedfiles' tool data table": "筛选文件必须存储在“primer_scheme_bedfiles”工具数据表中",
    "Model string to pass to medaka (see https://github.com/nanoporetech/medaka#models)": "将模型字符串传递给 medaka（参见 https://github.com/nanoporetech/medaka#models）",
    "Sample file in .BAM format.": "BAM格式的样本文件。",
    "Minimum number of reads that support a SV. [10]": "支持结构变异的最小读段数。[10]",
    "Maximum number of splits per read to be still taken into account. [7]": "每个读取要考虑的最大拆分数。[7]",
    "Maximum distance to group SV together. [1000]": "最大距离将结构变异（SV）分组在一起。[1000]",
    "Minimum length of SV to be reported. [30]": "要报告的SV的最小长度。[30]",
    "Minimum Mapping Quality. [20]": "最小映射质量。[20]",
    "Report up to N reads that support the SV in the vcf file. -1: report all. [0]": "在vcf文件中报告支持SV的最多N个读取。-1：报告全部。[0]",
    "Discard read if non of its segment is larger then this. [2000]": "如果读取的片段没有一段比这个长度大，则丢弃该读取。[2000]",
    "Discard SV that are not supported by at least x zmws. This applies only for PacBio recognizable reads. [0]": "丢弃不支持至少x个zmws的SV。这只适用于PacBio可识别的读取。[0]",
    "Enables the scan of CS string instead of Cigar and MD.  [false]": "启用扫描CS字符串而不是Cigar和MD。[false]",
    "Enables Sniffles to phase SVs that occur on the same reads [false]": "使Sniffles能够在同一个读取中对结构变异进行分相[false]",
    "Minimum number of reads supporting clustering of SV. [1]": "支持结构变异聚类的最小读取数。[1]",
    "Filters the SV calls based on the allele frequency. [0]": "根据等位基因频率过滤结构变异（SV）调用。[0]",
    "Minimum homogeneous threshold on allele frequency (0-1).  [0.8]": "最小等位基因频率的同质阈值（0-1）。[0.8]",
    "Minimum heterogeneous threshold on allele frequency (0-1).  [0.3]": "最小等位基因频率的异质性阈值（0-1）。[0.3]",
    "Report BND instead of Tra in vcf output.  [true]": "在vcf输出中报告BND而不是Tra。[真]",
    "Don't report sequences for indels in vcf output. (Beta version!)  [false]": "不要在vcf输出中报告插入/缺失序列。（测试版本！）[false]",
    "Ignores the sd based filtering.  [false]": "忽略基于sd的过滤。[false]",
    "Preset CCS Pacbio setting. (Beta)  [false]": "预设 CCS Pacbio 设置。（测试版）[否]",
    "Enables the scan if only very few reads are present.  [false]": "如果只有很少的读取存在，则启用扫描。[false]",
    "Estimated ratio of deletions per read (0-1).  [0.0458369]": "估计的每读删除比率（0-1）。[0.0458369]",
    "Estimated ratio of insertions per read (0-1).  [0.049379]": "估计每个读取的插入比例（0-1）。[0.049379]",
    "Maximum differences per 100bp. [50]": "每100个碱基对的最大差异数。[50]",
    "Maximum distance between alignment (indel) events. [4]": "最大对齐（插入/缺失）事件之间的距离。[4]",
    "If true, adds a PG tag to created SAM/BAM/CRAM files.": "如果为真，则在创建的SAM/BAM/CRAM文件中添加一个PG标签。",
    "If true, adds a command line header line to created VCF files.": "如果为真，会在创建的 VCF 文件中添加一个命令行标题行。",
    "Population allele fraction assigned to alleles not found in germline resource.  Please see docs/mutect/mutect2.pdf fora derivation of the default value.": "分配给在生殖系资源中未找到的等位基因的群体等位基因分数。有关默认值的推导，请参见docs/mutect/mutect2.pdf。",
    "If provided, we will annotate records with the number of alternate alleles that were discovered (but not necessarily genotyped) at a given site": "如果提供了，我们将会在记录中注释在给定位点发现的替代等位基因的数量（但不一定进行了基因分型）。",
    "One or more specific annotations to add to variant calls": "一个或多个要添加到变体调用中的特定注释",
    "One or more annotation groups to add to variant calls": "一个或多个要添加到变异调用中的注释组",
    "Specific annotations to exclude from variant calls": "排除变异调用中的特定注释",
    "Pedigree file for determining the population \"founders\". If a file is provided here, a pedigree-based annotation must be added above.": "用于确定种群“祖先”的家谱文件。如果在这里提供了一个文件，必须在上面添加基于家谱的注释。",
    "Base qualities below this threshold will be reduced to the minimum (6)": "低于此阈值的碱基质量将被降低到最小值（6）",
    "Does not affect genotyping": "不影响基因分型",
    "If true, don&apos;t cache bam indexes, this will reduce memory requirements but may harm performance if many intervals are specified.  Caching is automatically disabled if there are no intervals specified.": "如果为真，不缓存BAM索引，这将减少内存需求，但如果指定了很多区间，可能会损害性能。如果没有指定区间，缓存将自动禁用。",
    "Read filters to be disabled before analysis": "在分析之前要禁用的读取过滤器",
    "Read filters to be applied before analysis": "在分析之前要应用的读取过滤器",
    "If specified, do not check the sequence dictionaries from our inputs for compatibility. Use at your own risk!": "如果指定了，不要检查我们输入的序列字典的兼容性。请自行承担风险使用！",
    "Downsample a pool of reads starting within a range of one or more bases.": "从一组或多个碱基范围内开始的读取池中降采样。",
    "Samples representing the population &quot;founders&quot;": "代表“创始人”群体的样本",
    "If the GCS bucket channel errors out, how many times it will attempt to re-initiate the connection": "如果GCS存储桶通道出错，它将尝试重新建立连接多少次",
    "(EXPERIMENTAL) Call all apparent germline site even though they will ultimately be filtered.": "（实验性）即使最终会被过滤掉，也要调用所有明显的种系位点。",
    "Call sites in the PoN even though they will ultimately be filtered.": "在PoN中调用位点，尽管它们最终会被过滤掉。",
    "The set of alleles at which to genotype": "在哪些等位基因上进行基因分型",
    "Population vcf of germline sequencing containing allele fractions.": "包含等位基因分数的种系测序群体vcf。",
    "LOD threshold to consider pileup active.": "考虑成堆的LOD阈值。",
    "Amount of padding (in bp) to add to each interval you are excluding": "要排除的每个区间添加的填充量（以碱基对为单位）",
    "Amount of padding (in bp) to add to each interval you are including": "要在包含的每个区间中添加的填充量（以碱基对为单位）",
    "Interval merging rule for abutting intervals": "相邻区间合并规则",
    "Set merging approach to use for combining interval inputs": "设置用于合并区间输入的合并方法",
    "Lenient processing of VCF files": "宽松处理 VCF 文件",
    "Maximum population allele frequency in tumor-only mode.": "在肿瘤特有模式下，最大人群等位基因频率。",
    "Maximum number of reads to retain per alignment start position. Reads above this threshold will be downsampled. Set to 0 to disable.": "每个比对起始位置保留的最大读取数。超过此阈值的读取将被下采样。设置为0以禁用。",
    "Minimum base quality required to consider a base for calling": "用于调用的最小碱基质量要求",
    "use double precision in the native pairHmm. This is slower but matches the java implementation better": "在本地pairHmm中使用双精度。这会降低速度，但与Java实现的匹配度更高。",
    "LOD threshold for calling normal variant non-germline.": "用于调用正常变异非生殖细胞的LOD阈值。",
    "BAM sample name of normal.  May be URL-encoded as output by GetSampleName with -encode argument.": "正常样本的BAM名称。可能被URL编码，如使用-encode参数的GetSampleName输出。",
    "VCF file of sites observed in normal.": "正常组织中观察到的位点的VCF文件。",
    "If true, don't emit genotype fields when writing vcf file output.": "如果为真，在写入vcf文件输出时不发射基因型字段。",
    "Validation stringency for all SAM/BAM/CRAM/SRA files read by this program.  The default stringency value SILENT can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) do not otherwise need to be decoded.": "此程序读取的所有SAM/BAM/CRAM/SRA文件的验证严格度。默认的严格度值SILENT可以在处理BAM文件时提高性能，其中不需要对可变长度数据（读取、质量、标签）进行解码。",
    "If true, don&apos;t emit genotype fields when writing vcf file output.": "如果为真，在写入vcf文件输出时，不发射基因型字段。",
    "LOD threshold to emit tumor variant to VCF.": "LOD阈值，用于将肿瘤变异输出到VCF文件中。",
    "Number of overlapping bases around the variant.": "变异周围的重叠碱基数量。",
    "Minimum probability for a locus to be considered active.": "一个位点被认为是活跃的最小概率。",
    "Allow graphs that have non-unique kmers in the reference": "允许在参考中包含非唯一k-mers的图形",
    "Number of additional bases of context to include around each assembly region": "每个组装区域周围要包含的额外碱基对的数目",
    "Which haplotypes should be written to the BAM": "哪些单倍型应该写入BAM文件",
    "Disable the adaptive algorithm for pruning paths in the graph": "禁用图中修剪路径的自适应算法",
    "Disable all tool default annotations": "禁用所有工具默认注释",
    "WARNING: many tools will not function correctly without their default read filters on": "警告：许多工具如果没有启用它们的默认读取过滤器，将无法正常工作。",
    "Disable iterating over kmer sizes when graph cycles are detected": "当检测到图形循环时，禁用对kmer大小的迭代",
    "Do not analyze soft clipped bases in the reads": "不要分析读取中的软剪切碱基",
    "NOTE: This is a beta feature in Mutect2": "注意：这是Mutect2中的一个测试功能",
    "Use all possible annotations (not for the faint of heart)": "使用所有可能的注释（不适合胆小的人）",
    "If selected, all regions will be marked as active": "如果选中，所有区域将被标记为活跃",
    "Force-call filtered alleles included in the resource specified by --alleles": "在由 --alleles 指定的资源中包含的过滤等位基因被强制调用",
    "Exclusive upper bounds for reference confidence LOD bands (must be specified in increasing order)": "参考置信度LOD区间的独占上限（必须按递增顺序指定）",
    "Allow paired reads to independently support different haplotypes. Useful for validations with ill-designed synthetic data": "允许配对的读取独立支持不同的单倍型。对于使用设计不良的合成数据进行验证非常有用。",
    "Kmer size to use in the read threading assembler": "在读取线程组装器中使用的Kmer大小",
    "Maximum size of an assembly region": "组装区域的最大大小",
    "Two or more phased substitutions separated by this distance or less are merged into MNPs.": "两个或两个以上相距此距离或更短的相位替换被合并为多核苷酸多态性（MNPs）。",
    "Maximum number of haplotypes to consider for your population": "考虑您人群的最大单倍型数量",
    "Upper limit on how many bases away probability mass can be moved around when calculating the boundaries between active and inactive assembly regions": "在计算活性和非活性组装区域边界时，可以移动概率质量的上限是多少个碱基",
    "Maximum number of suspicious reads (mediocre mapping quality or too many substitutions) allowed in a downsampling stride.  Set to 0 to disable.": "在一个下采样步长中允许的可疑读段（平庸的映射质量或过多的替换）的最大数量。设置为0以禁用。",
    "Maximum number of variants in graph the adaptive pruner will allow": "自适应剪枝器允许图中变异的最大数量",
    "Minimum size of an assembly region": "组装区域的最小大小",
    "Minimum length of a dangling branch to attempt recovery": "尝试恢复悬挂分支的最小长度",
    "Minimum support to not prune paths in the graph": "在图中不修剪路径的最小支持度",
    "Lower bound of variant allele fractions to consider when calculating variant LOD": "在计算变异LOD时考虑的变异等位基因分数的下限",
    "Number of samples that must pass the minPruning threshold": "必须通过最小修剪阈值的样本数",
    "Flat gap continuation penalty for use in the Pair HMM": "在配对隐马尔可夫模型中使用的平坦间隙延续惩罚",
    "The PairHMM implementation to use for genotype likelihood calculations": "用于计算基因型可能性的PairHMM实现",
    "The PCR indel model to use": "要使用的PCR插入缺失模型",
    "The global assumed mismapping rate for reads": "全局假定的读取错误映射率",
    "Likelihood ratio threshold for adaptive pruning algorithm": "自适应剪枝算法的似然比阈值",
    "Which Smith-Waterman implementation to use, generally 'Fastest available' is the right choice": "选择使用哪种Smith-Waterman实现，通常“最快的可用选项”是正确的选择",
    "Use the contamination-filtered read maps for the purposes of annotating variants": "使用去污染的读图谱来注释变异",
    "Additional outputs for debugging purposes": "调试目的的额外输出",
    "Output the assembly region to this IGV formatted file": "将组装区域输出到这个IGV格式的文件中",
    "The assembled haplotypes and locally realigned reads will be written as BAM to this file if requested. This is intended to be used only for troubleshooting purposes, in specific areas where you want to better understand why the caller is making specific calls": "组装的单倍型和局部重新对齐的读取（reads）如果被请求，将作为BAM文件写入此文件。这仅用于故障排除目的，在您想要更好地理解调用者为何进行特定调用的特定区域。",
    "Ligate phased VCFs by matching phase at overlapping haplotypes": "将相位化的VCF文件通过匹配重叠的单倍型进行连接",
    "It allows finer control of --ligate behavior": "它允许更精细地控制--ligate行为",
    "Do not output PS tag at each site, only at the start of a new phase set block.": "不要在每个位点输出PS标签，只在新的阶段集块开始时输出。",
    "Break phase set if phasing quality is lower than <int>": "如果相位质量低于<int>，则设置断相位集",
    "Re-alignment resets the MC, MD, NM, and A flags by default. The values of these tags depend on the exact alignment. Use this option only if you know what you are doing.": "重新比对默认情况下会重置MC、MD、NM和A标志。这些标签的值取决于确切的比对。只有在你清楚自己在做什么的情况下才使用这个选项。",
    "In sequenced reads obtained with Illumina sequencing pipelines before version 1.8, base quality 2 is special in that it serves as a general indicator of low quality of the corresponding bases. For such reads, the tool can make an optimistic guess of the real quality of such bases by replacing base qualities of 2 with the median of all other base qualities observed in the read. Alternatively, you can provide a fixed replacement value. For recently obtained sequencing data, just keep BQ2 values unchanged (the default) since they have no special meaning.": "在Illumina测序流程1.8版本之前的测序读段中，碱基质量值为2是特殊的，因为它作为相应碱基质量低的通用指标。对于这样的读段，工具可以通过将质量值为2的碱基替换为读段中观察到的所有其他碱基质量的中位数，来对这些碱基的真实质量做出乐观的猜测。或者，您可以提供一个固定的替换值。对于最近获得的测序数据，只需保持BQ2值不变（默认设置），因为它们没有特殊含义。",
    "Should probably not be left at the default value": "可能不应该保留默认值",
    "Leave blank to use the same values for insertions and deletions": "留空以使用插入和删除相同的值",
    "Limit transitive closure to include no more than N copies of a given input base": "将传递闭包限制为包含不超过N个给定输入基底的副本",
    "This can smooth the graph locally and prevent the formation of complex local graph topologies from forming due to differential alignments.": "这可以平滑局部图形，并防止由于差异对齐而形成复杂的局部图形拓扑结构。",
    "Apply generalized topological sort to the graph and set node ids to order.": "对图应用广义拓扑排序，并将节点ID设置为顺序。",
    "comma-separated list accepted": "逗号分隔列表被接受",
    "from `vg snarls` to avoid recomputing": "从 `vg snarls` 避免重新计算",
    "Select all the snippy inputs for alignment": "选择所有用于比对的 Snippy 输入。",
    "Minimum mapping quality to allow": "最小映射质量允许",
    "Minimum coverage to call a snp": "最小覆盖度以调用SNP",
    "Use this @RG ID: in the BAM header": "在BAM头文件中使用此@RG ID：",
    "Extra BWA MEM options, eg. -x pacbio": "额外的BWA MEM选项，例如 -x pacbio",
    "When producing an output of the reference genome with variants instantiated, edit the header so that it is named after the input VCF": "当产生一个带有实例化变异的参考基因组输出时，编辑头部，使其以输入 VCF 的名称命名。",
    "Replace non-[AGTCN-] chars with this character, defaults to 'N' if none entered": "将非[AGTCN-]字符替换为此字符，默认情况下如果没有输入则使用'N'",
    "Resolve duplicate sample names": "解决重复的样本名称",
    "Rules for merging INFO fields (method is one of sum,avg,min,max,join) or \"-\" to turn off the default [DP:sum,DP4:sum]": "合并INFO字段的规则（方法之一为sum,avg,min,max,join）或\"-\"以关闭默认值[DP:sum,DP4:sum]",
    "Allow multiallelic records for: snps|indels|both|all|none|id, see man page for details [both]": "允许多等位基因记录用于：snps|indels|both|all|none|id，详细信息请参见手册页 [both]",
    "Requires the input files to have chromosomes in th same order and consistent with the order of sequences in the header": "要求输入文件中的染色体顺序与标题中序列的顺序一致且相同",
    "Use the provided header": "使用提供的标题",
    "Print only the merged header and exit": "仅打印合并后的标题并退出",
    "default = 20 bp": "默认值 = 20 碱基对",
    "One of: alleles, trio (see manual)": "其中之一：等位基因，三人组（见手册）",
    "Output also sites missed by mpileup but present in -T": "输出 mpileup 遗漏但 -T 中存在的位点。",
    "Expected substitution rate": "预期替代率",
    "The parameter value is the minimum per-sample depth required to include a site in the non-variant block": "参数值是将一个位点包含在非变异块中所需的每个样本的最小深度。",
    "Accept variant if P(ref|D)<FLOAT": "接受变异，如果 P(ref|D)<FLOAT",
    "Space/tab-delimited list of CHROM,FROM,TO,SEX,PLOIDY": "空格/制表符分隔的列表，包含 CHROM（染色体）、FROM（起始位置）、TO（终止位置）、SEX（性别）、PLOIDY（倍性）",
    "Group samples by population for single-sample calling (-G - is the only option implemented so far)": "将样本按种群分组以进行单样本呼叫（-G - 目前仅实现了此选项）",
    "Output all alternate alleles present in the alignments even if they do not appear in any of the genotypes": "在对齐中输出所有存在的替代等位基因，即使它们没有出现在任何基因型中",
    "Currently GQ and GP fields are supported": "当前支持GQ和GP字段",
    "Output sites where REF allele is N": "输出参考等位基因为N的位点",
    "Skip indels/SNP sites": "跳过插入/缺失位点/单核苷酸多态性位点",
    "Warnings about REF mismatches will be emitted to the standard error (stderr) stream, and it is recommended to check there for problems if you choose not to exit with an error immediately upon encountering a mismatch.": "关于REF不匹配的警告将被发送到标准错误（stderr）流，如果你选择在遇到不匹配时不立即退出并出现错误，建议检查那里以发现问题。",
    "Decompose complex variants (e.g. MNVs become consecutive SNVs)": "将复杂变体分解（例如，多核苷酸变异变为连续的单核苷酸变异）",
    "merged site is PASS only if all sites being merged PASS": "合并后的位点只有在所有被合并的位点都通过时才通过",
    "(-w, --site-win) Buffer for sorting lines which changed position during realignment": "（-w，--site-win）在重新对齐期间，用于排序改变位置的行的缓冲区",
    "From Ensembl:  ftp://ftp.ensembl.org/pub/current_gff3/": "来自Ensembl：ftp://ftp.ensembl.org/pub/current_gff3/",
    "-ncsq 16": "-ncsq 16",
    "--local-csq switch off haplotype-aware calling, run localized predictions considering only one VCF record at a time": "--local-csq 关闭单倍型感知调用，运行局部预测，一次只考虑一个 VCF 记录",
    "how to construct haplotypes and how to deal with unphased data": "如何构建单倍型以及如何处理未分相数据",
    "Abbreviate protein-changing predictions to max INT aminoacids": "将蛋白质变化预测缩写为最大整数个氨基酸",
    "BED file describing exons": "描述外显子的BED文件",
    "The name (as used in the input) of the query sample in the input. Can be ommitted if, and only if, there is only one sample in the input.": "查询样本在输入中的名称（如在输入中使用的）。如果输入中只有一个样本，则可以省略。",
    "The name (as used in the input) of an optional control sample to compare against. Note: The pairwise calling mode represents the real strength of the tool as it helps to reduce the number of false calls and also allows one to distinguish between normal and novel copy number variation.": "一个可选的对照样本名称（如输入中使用的），用于进行比较。注意：成对调用模式代表了该工具的真正优势，因为它有助于减少错误调用的数量，并且还允许区分正常和新颖的拷贝数变异。",
    "Plot aberrant chromosomes with quality at least 'float'": "绘制质量至少为'float'的异常染色体",
    "Estimate of the fractional contribution of the desired query tissue to the (possibly control-contaminated) query sample (default: 1 = no contamination with control tissue)": "估计所需查询组织对（可能被对照污染的）查询样本的分数贡献（默认值：1 = 没有对照组织的污染）",
    "Estimate of the fractional contribution of the desired control tissue to the (possibly query-contaminated) control sample (default: 1 = no contamination with query tissue)": "估计所需对照组织对（可能被查询污染的）对照样本的分数贡献（默认值：1 = 没有与查询组织污染）",
    "Instead of treating your specified estimates of the sample purities as fixed values, the tool can use them as starting values for an iterative optimization that tries to estimate the sample purities from the data. Note: With estimate optimization enabled the final estimates will be reported as cell fraction (CF) estimates in the summary report.": "而不是将您指定的样本纯度估计作为固定值处理，该工具可以将它们用作迭代优化的起始值，以尝试从数据中估计样本纯度。注意：启用估计优化后，最终估计值将在摘要报告中作为细胞分数（CF）估计报告。",
    "Constrains the final adjusted purity estimates not to be smaller than this value.": "限制最终调整的纯度估计值不得小于此值。",
    "Using LRR information in addition to BAF values is the default and helps in dealing with random noise in the data. However, the tool is also able to call CNVs from BAF values alone in case your input does not feature LRR information.": "使用LRR信息以及BAF值是默认设置，有助于处理数据中的随机噪声。然而，如果你的输入没有LRR信息，该工具也能够仅从BAF值中调用CNV。",
    "relative contribution from BAF": "相对贡献来自 BAF",
    "Expected BAF deviation in the query sample (default: 0.04)": "查询样本中预期的BAF偏差（默认值：0.04）",
    "Expected BAF deviation in the control sample (default: 0.04)": "预期在对照样本中的BAF偏差（默认值：0.04）",
    "Relative contribution from LRR. This option can have a big effect on the number of calls produced. With truly random noise (such as in simulated data), the value should be set high (1.0), but in the presence of systematic noise when LRR values are not informative, lower values result in cleaner calls (default: 0.2).": "相对LRR的贡献。这个选项对产生的调用数量有很大的影响。在真正随机的噪声（如模拟数据中）中，该值应设置为高（1.0），但在存在系统噪声时，当LRR值不具有信息性时，较低的值会产生更清晰的调用（默认值：0.2）。",
    "Expected LRR deviation in the query sample (default: 0.2)": "在查询样本中预期的LRR偏差（默认值：0.2）",
    "Expected LRR deviation in the control sample (default: 0.2)": "预期在对照样本中的LRR偏差（默认值：0.2）",
    "Window of LRR moving average smoothing (default: 10)": "LRR移动平均平滑窗口（默认：10）",
    "Uniform error probability": "均匀误差概率",
    "Provide a multiple sequence alignment containing sequences of equal length": "提供一个包含等长序列的多序列比对",
    "Output format to use if not counting constant sites": "如果不计算恒定位点，则使用的输出格式",
    "Output monomorphic (constant) as opposed to polymorphic (variable) sites": "输出单态（恒定）位点，而不是多态（可变）位点",
    "Ignore columns containing ambiguous characters and gaps": "忽略包含模糊字符和空白的列",
    "If checked, the tool will call CNVs in the first sample vs all the others. If unchecked, an all vs all CNV call will be performed": "如果选中，该工具将在第一个样本与其他所有样本之间调用CNV。如果未选中，将执行所有样本之间的CNV调用。",
    "Need to add more files? Use controls below.": "需要添加更多文件？请使用下面的控制项。",
    "Label to use in the output. If not given, the dataset name will be used instead": "在输出中使用的标签。如果没有给出，则将使用数据集名称代替",
    "Transition probability of the hidden Markov Chain from the normal copy number state to either a deletion or a duplication. The default value (0.0001) expects approximately 20 CNVs genome-wide": "隐藏马尔可夫链从正常拷贝数状态转换到删除或复制状态的转移概率。默认值（0.0001）预计全基因组大约有20个拷贝数变异（CNVs）",
    "Sets --min-coverage, --limit-coverage, and --skip-coverage": "设置 --min-coverage（最小覆盖度）、--limit-coverage（限制覆盖度）和 --skip-coverage（跳过覆盖度）",
    "Sets --samples, --populations, --cnv-map, --trace, --failed-alleles, --variant-input, --only-use-input-alleles, --haplotype-basis-alleles, --report-all-haplotype-alleles, --report-monomorphic options, --observation-bias, --contamination-estimates and --trim-complex-tail": "设置 --samples, --populations, --cnv-map, --trace, --failed-alleles, --variant-input, --only-use-input-alleles, --haplotype-basis-alleles, --report-all-haplotype-alleles, --report-monomorphic 选项，--observation-bias, --contamination-estimates 和 --trim-complex-tail",
    "By default FreeBayes will analyze all samples in its input BAM datasets": "默认情况下，FreeBayes会分析其输入BAM数据集中的所有样本",
    "Each line of this dataset should list a sample and a population which it is part of. The population-based bayesian inference model will then be partitioned on the basis of the populations": "这个数据集的每一行应该列出一个样本以及它所属的种群。然后，基于种群的贝叶斯推断模型将根据这些种群进行划分。",
    "The BED dataset should have the format: 'reference sequence, start, end, sample name, copy number' for each region in each sample which does not have the default copy number as set by --ploidy. If not specified, copy number is set to as specified by --ploidy": "BED数据集应该具有以下格式：'参考序列，起始位置，结束位置，样本名称，拷贝数'，用于每个样本中每个区域，这些区域的默认拷贝数不是由--ploidy参数设置的。如果没有指定，拷贝数将被设置为由--ploidy参数指定的值。",
    "The format is [length] [alignment efficiency relative to reference] where the efficiency is 1 if there is no relative observation bias": "格式为 [长度] [与参考相对的比对效率]，如果不存在相对观测偏差，则效率为1",
    "The format should be: sample p(read=R|genotype=AR) p(read=A|genotype=AA) Sample '*' can be used to set default contamination estimates": "格式应该是：样本 p(read=R|genotype=AR) p(read=A|genotype=AA) 使用 '*' 作为样本可以设置默认的污染估计",
    "Note that post-filtering is generally recommended over the use of this parameter": "请注意，通常建议使用后过滤而不是使用此参数。",
    "This serves as the single parameter to the Ewens Sampling Formula prior model": "这作为Ewens抽样公式先验模型的单一参数",
    "Model pooled samples using discrete genotypes across pools. When using this flag, set --ploidy to the number of alleles in each sample or use the --cnv-map to define per-sample ploidy": "使用离散基因型对池化样本进行建模。使用此标志时，将 --ploidy 设置为每个样本中的等位基因数目，或使用 --cnv-map 定义每样本的倍性图",
    "Alleles are ranked by the sum of supporting quality scores. Set to 0 to evaluate all": "等位基因根据支持质量分数的总和进行排名。设置为0以评估所有等位基因",
    "By default, FreeBayes uses all observations, dividing partial support across matching haplotypes when generating haplotypes": "默认情况下，FreeBayes使用所有观察值，在生成单倍型时，将部分支持分配给匹配的单倍型。",
    "--min-mapping-quality 30 --min-base-quality 20, --min-supporting-allele-qsum 0 --genotype-variant-threshold 0": "--min-mapping-quality 30 --min-base-quality 20, --min-supporting-allele-qsum 0 --genotype-variant-threshold 0",
    "default=~unbounded": "默认值=~无限制",
    "Equivalent to --pooled-discrete --hwe-priors-off and removal of Ewens Sampling Formula component of priors": "等同于--pooled-discrete --hwe-priors-off，并且移除了先验中Ewens抽样公式的部分",
    "Uses read placement probability, strand balance probability, and read position (5''-3'') probability": "使用读段定位概率、链平衡概率和读段位置（5''-3''）概率",
    "Sets --base-quality-cap, --experimental-gls, and --prob-contamination options": "设置 --base-quality-cap、--experimental-gls 和 --prob-contamination 选项",
    "Incorporate partial observations. This is the default when contamination estimates are provided. Optimized for diploid samples": "整合部分观测。当提供污染估计时，这是默认设置。针对二倍体样本进行了优化。",
    "By default, FreeBayes uses a minimum Base Quality in flanking sequence": "默认情况下，FreeBayes 使用周围序列中的最小碱基质量",
    "The fasta reference genome that variants should be called against.": "参考基因组的fasta文件，应该针对该基因组调用变异。",
    "The settings in this section control which mapped reads will be used for variant calling.": "本节中的设置控制哪些映射后的读取将用于变体调用。",
    "The minimum base quality (default: 13) at a given position required to use a read for calling variants at that site": "在给定位置，用于调用该位点变异所需的最小碱基质量（默认值：13）",
    "The minimum mapping quality (default: 0) required for a read to be considered in variant calling": "在变异调用中，考虑一个读段所需的最小映射质量（默认：0）",
    "Applies to paired-end reads only. If the reads of a pair overlap on the reference, then with this option (on by default), the bases for which both reads provide evidence will be counted only once in variant calling (of the two sequenced bases in the reads, the base with the better base quality will be used).": "仅适用于配对末端读取。如果一对读取在参考序列上重叠，那么使用此选项（默认开启），在变异调用中只会计算一次提供证据的碱基（在读取中测序的两个碱基中，将使用具有更好碱基质量的碱基）。",
    "Restrict the number of reads used for variant calling at a site to this maximum (default: 8000) for each sample. Helps protect against excessive memory usage (and slow tool runs) at sites of extraordinary high coverage.": "将每个样本在位点上用于变体调用的读取数限制为此最大值（默认值：8000）。有助于保护在异常高覆盖度的位点处防止内存使用过多（以及工具运行缓慢）。",
    "Minimum site coverage (default: 8) required in the normal and in the tumor sample to call a variant. This threshold gets applied after eliminating reads based on the read selection criteria above.": "在正常样本和肿瘤样本中调用一个变异所需的最小位点覆盖度（默认值：8）。在根据上述读取选择标准排除读取后，将应用此阈值。",
    "Minimum number (default: 2) of variant-supporting reads (after read selection) at a position required to make a call": "在某个位置上，为了进行变体调用，所需的最小变异支持读段数（默认值：2）（在读段选择之后）",
    "The p-value threshold (default: 0.99) used to determine if a variant should be called for either sample": "p值阈值（默认：0.99）用于确定是否应该为任一样本调用一个变体",
    "The p-value threshold (default: 0.05) used to determine if read count differences between the normal and the tumor sample justify classification of a variant as somatic or as an LOH event": "p值阈值（默认：0.05）用于确定正常样本和肿瘤样本之间的读取计数差异是否足以将一个变异分类为体细胞变异或作为LOH（杂合性缺失）事件。",
    "(default: 4)": "（默认值：4）",
    "This setting (default: 2) will be applied instead of the --min-var-count limit for sites with poor overall (see threshold below) coverage.": "这个设置（默认值：2）将代替--min-var-count限制，用于覆盖度整体较差（见下面的阈值）的位点。",
    "Combined depth (default: 10) of ref- and variant-supporting reads required at variant site to apply the (stricter) --min-var-count filter instead of --min-var-count-lc": "在变异位点处所需的参考和变异支持读段的组合深度（默认值：10），用于应用更严格的--min-var-count过滤器，而不是--min-var-count-lc过滤器。",
    "(default: 0.05)": "（默认值：0.05）",
    "The minimum average relative distance from either end of ref-supporting reads (default: 0.1) required for variant sites": "变异位点所需的参考支持的读取序列从任一端的最小平均相对距离（默认值：0.1）",
    "The minimum average relative distance from either end of variant-supporting reads (default: 0.1) required for variant sites": "变异支持的读取序列（默认值：0.1）从任一端到变异位点的最小平均相对距离",
    "The minimum average relative distance from the effective 3'end of ref-supporting reads (default: 0.1) required for variant sites. The effective 3'end is defined by the end of the alignment of the read to the reference (or, if the Illumina 1.5 compatibility setting is used, by the first base in 3'->5' direction with a base quality > 2).": "所需的参考支持读取的有效3'端与变异位点之间的最小平均相对距离（默认值：0.1）。有效3'端由读取与参考的对齐末端定义（或者，如果使用了Illumina 1.5兼容性设置，则由3'->5'方向上的第一个碱基质量大于2的碱基定义）。",
    "The minimum average relative distance from the effective 3'end of variant-supporting reads (default: 0.1) required for variant sites. The  effective 3'end is defined as above.": "对于变体位点，需要变体支持的读取的有效3'端的最小平均相对距离（默认值：0.1）。有效3'端的定义如上所述。",
    "The minimum average trimmed length (default: 90) required for reads supporting the reference allele": "最小平均修剪长度（默认值：90），用于支持参考等位基因的读取",
    "The minimum average trimmed length (default: 90) required for reads supporting the variant allele": "支持变异等位基因的读取所需的最小平均修剪长度（默认值：90）",
    "The maximum allowed difference (default: 0.25) in the average relative read length (ref - var) between reads supporting the reference and the variant allele": "允许的最大差异（默认值：0.25）是在支持参考等位基因和变异等位基因的读取之间平均相对读取长度（ref - var）的差异",
    "The minimum fraction (default: 0.01) of variant reads that are required to come from the forward and from the reverse strand": "最小分数（默认值：0.01）的变异读取需要来自正向和反向链。",
    "(default: 5)": "（默认值：5）",
    "The minimum average base quality (default: 15) required at the variant site for reads supporting the reference allele": "在变异位点上，支持参考等位基因的读取所需的最小平均碱基质量（默认值：15）",
    "The minimum average base quality (default: 15) required at the variant site for reads supporting the variant allele": "在变异位点上支持变异等位基因的读段所需的最小平均碱基质量（默认值：15）",
    "The maximum difference (default: 50) in the average base quality (ref - var) allowed between the variant site positions of reads supporting the reference and the variant allele": "最大差异（默认值：50）是允许支持参考和变体等位基因的读取中变体位点位置的平均碱基质量（参考 - 变体）之间的差异",
    "The minimum average mapping quality (default: 15) required for reads supporting the reference allele": "支持参考等位基因所需的最小平均比对质量（默认值：15）",
    "The minimum average mapping quality (default: 15) required for reads supporting the variant allele": "支持变异等位基因所需的最小平均比对质量（默认值：15）",
    "The maximum difference (default: 50) in the average mapping quality (ref - var) allowed between reads supporting the reference and the variant allele": "最大差异（默认值：50）是在支持参考等位基因和变异等位基因的读取之间允许的平均映射质量（ref - var）的差异",
    "The maximum mismatch base quality sum (default: 100) allowed for reads supporting the reference allele": "允许支持参考等位基因的读取的最大不匹配碱基质量总和（默认值：100）",
    "The maximum mismatch base quality sum (default: 100) allowed for reads supporting the variant allele": "允许支持变异等位基因的读取中最大不匹配碱基质量总和（默认：100）",
    "The minimum difference (default: 0) in the mismatch base quality sums (var - ref) required between reads supporting the variant and the reference allele": "最小差异（默认值：0）是在支持变体和参考等位基因的读取之间所需的不匹配碱基质量总和（变体 - 参考）之间的差异",
    "The maximum difference (default: 50) in the mismatch base quality sums (var - ref) allowed between reads supporting the variant and the reference allele": "最大差异（默认值：50）是在支持变体和参考等位基因的读取之间允许的不匹配碱基质量总和（变体 - 参考）",
    "FASTA file with the reference.": "带有参考序列的FASTA文件。",
    "SAM/BAM file to use as the input.": "用作输入的SAM/BAM文件。",
    "Smallest number of OEA (one-end-anchor) reads on each side to support an insertion.  In range [1..inf].  This is the minimum number of supporting reads (without mapped partners) on each side of an insertion breakpoint required to not be filtered.": "每个插入位点两侧支持插入的OEA（一端锚定）读取的最小数量。范围在[1..inf]之间。这是在不过滤的情况下，每个插入断点两侧所需的最小支持读取数量（不包括已映射的配对读取）。",
    "defaults to 0.15 (-i)": "默认为0.15（-i）",
    "default to 0.001 (-t)": "默认为0.001（-t）",
    "Forces -e the max-likelihood inference parameter. (-c)": "强制 -e 为最大似然推断参数。(-c)",
    "Including estimating the site allele frequency, testing Hardy-Weinberg equilibrium and testing associations with LRT. (-e)": "包括估计位点等位基因频率，测试哈迪-温伯格平衡以及使用似然比检验进行关联测试。(-e)",
    "0-based, closed. (BED style)": "0-based，闭合的。（BED格式）",
    "0-based, open. (BED style)": "0-based，开放区间。（BED格式）",
    "This affects the maximum recorded value for a position, e.g. uint8 would be 255 coverage, but will require the least amount of RAM": "这会影响一个位置记录的最大值，例如 uint8 将是 255 的覆盖度，但将需要最少的 RAM",
    "Minimum coverage threshold for copynumber segments": "拷贝数段的最小覆盖度阈值",
    "Minimum depth at a position to make a call": "在某个位置上进行调用所需的最小深度",
    "Minimum supporting reads at a position to make a call": "在某个位置上，为了做出判断，所需的最小支持读取数",
    "Separate sample names by comma; leave blank to use default sample names.": "用逗号分隔样本名称；留空以使用默认的样本名称。",
    "VCF output from SnpEffect or SnpSift": "SnpEffect或SnpSift的VCF输出",
    "feb2012 - would access ensembl server: feb2012.archive.ensembl.org": "2012年2月 - 将访问Ensembl服务器：feb2012.archive.ensembl.org",
    "Report coding changes for selected effects": "报告选定效应的编码变化",
    "Default is to report only the first Transcript that has a cDNA change": "默认情况下，只报告具有cDNA变化的第一个转录本",
    "Otherwise, report the first changed Animo Acid as: AposA (ref_AA pos_from_Nterminus alt_AA)": "否则，报告第一个变化的氨基酸为：AposA（参考氨基酸 位点从N端开始的编号 替代氨基酸）",
    "Leave blank to turn off poly A filtering": "留空以关闭poly A过滤",
    "An Ensembl all_pep.fa file corresponding to the genome build used for SnpEff (May be converted to tabular fasta format)": "一个对应于用于SnpEff的基因组构建的Ensembl all_pep.fa文件（可以转换为表格化fasta格式）",
    "The number of Amino Acids to include before the variant position (leave blank to include all)": "变异位点前包含的氨基酸数量（留空以包含全部）",
    "The number of Amino Acids to include after the variant position (leave blank to include all)": "变异位点后包含的氨基酸数量（留空以包含全部）",
    "Usually, annotations are located in the CSQ, ANN or EFF INFO field, though any arbitrary tag may be entered here.": "通常，注释位于CSQ、ANN或EFF INFO字段中，尽管这里可以输入任何任意标签。",
    "The default type of the new annotation is String but can be also set to Integer/Int or Float/Real by adding ':type' to the field name or index. Examples can be found in the tool help.": "新注释的默认类型是字符串，但也可以设置为整数（Integer/Int）或浮点数（Float/Real），方法是在字段名或索引后添加“:type”。工具帮助中可以找到示例。",
    "Argument has the form TR:CSQ. TR = transcript:   worst,primary,all (default: all). CSQ = consequence: any,missense,missense+,etc (default: any). Examples can be found in the tool help.": "参数具有形式 TR:CSQ。TR = 转录本：worst（最差）、primary（主要）、all（全部，默认值：all）。CSQ = 后果：any（任意）、missense（错义）、missense+（错义+）等（默认值：any）。示例可以在工具帮助中找到。",
    "Print \".\" for undefined tags": "打印“.”代表未定义的标签",
    "--per-sample-mF;  by default both options are applied to reads pooled from all samples": "--每个样本的最小自由能；默认情况下，这两个选项都应用于来自所有样本的池化读取。",
    "It is recommended to collect indel candidates from sequencing technologies that have low indel error rate such as ILLUMINA": "建议从测序技术中收集indel候选，这些技术具有较低的indel错误率，例如ILLUMINA",
    "What to do with ambiguous indel reads that do not span an entire short tandem repeat region: discard ambiguous reads from calling and do not increment high-quality AD depth counters (drop), exclude from calling but increment AD counters proportionally (incAD), exclude from calling and increment the first value of the AD counter (incAD0)": "对于不跨越整个短串联重复区域的模糊插入/删除读取，应该采取什么措施：丢弃模糊读取，不增加高质量AD深度计数器（drop），从调用中排除但按比例增加AD计数器（incAD），从调用中排除并增加AD计数器的第一个值（incAD0）",
    "Skews the indel scores up or down, trading recall (low false-negative) vs precision (low false-positive) [1.0]. In Bcftools 1.12 and earlier this parameter didn’t exist, but had an implied value of 1.0. If you are planning to do heavy filtering of variants, selecting the best quality ones only (favouring precision over recall), it is advisable to set this lower (such as 0.75) while higher depth samples or where you favour recall rates over precision may work better with a higher value such as 2.0": "倾斜indel分数，向上或向下，交换召回率（低假阴性）与精确度（低假阳性）[1.0]。在Bcftools 1.12及更早版本中，这个参数不存在，但有一个隐含的值1.0。如果你计划对变异进行大量过滤，只选择最佳质量的（偏爱精确度超过召回率），建议将其设置得更低（如0.75），而在更高的深度样本或你偏爱召回率超过精确度的情况下，使用更高的值（如2.0）可能会更好。",
    "Indel window size to use when assessing the quality of candidate indels. Note that although the window size approximately corresponds to the maximum indel size considered, it is not an exact threshold. Default: 110": "在评估候选插入缺失（indels）的质量时使用的插入缺失窗口大小。请注意，尽管窗口大小大致对应于考虑的最大插入缺失大小，但它并不是一个确切的阈值。默认值：110",
    "Only warn about unknown subset samples": "仅对未知的子集样本发出警告",
    "Do not (re)calculate INFO fields for the subset (currently INFO/AC and INFO/AN)": "不要（重新）计算子集的INFO字段（当前为INFO/AC和INFO/AN）",
    "Trim alternate alleles not seen in the subset": "修剪在子集中未看到的交替等位基因",
    "(-c --min-ac) Minimum count for non-reference (nref), 1st alternate (alt1), least frequent (minor), most frequent (major) or sum of all but most frequent (nonmajor) alleles": "（-c --min-ac）非参考（nref）、第一替代（alt1）、最不频繁（minor）、最频繁（major）或除最频繁外所有等位基因的计数最小值",
    "(-C --max-ac) Maximum count for non-reference (nref), 1st alternate (alt1), least frequent (minor), most frequent (major) or sum of all but most frequent (nonmajor) alleles": "(-C --max-ac) 非参考（nref）、第一替代（alt1）、最不常见（minor）、最常见（major）或除最常见外所有等位基因（nonmajor）的最大计数",
    "(-q) Minimum frequency for non-reference (nref), 1st alternate (alt1), least frequent (minor), most frequent (major) or sum of all but most frequent (nonmajor) alleles": "(-q) 非参考（nref）、第一替代（alt1）、最不常见（minor）、最常见（major）或除最常见外所有等位基因（nonmajor）的最小频率",
    "(-Q) Maximum frequency for non-reference (nref), 1st alternate (alt1), least frequent (minor), most frequent (major) or sum of all but most frequent (nonmajor) alleles": "(-Q) 最大频率用于非参考（nref）、第一替代（alt1）、最不常见（minor）、最常见（major）或除最常见外所有等位基因的总和（nonmajor）",
    "Drop individual genotype information (after subsetting if -s option set)": "删除个体基因型信息（如果设置了 -s 选项，则在子集化后）",
    "Compression level: 0 uncompressed, 1 best speed, 9 best compression": "压缩级别：0 未压缩，1 最快速度，9 最佳压缩",
    "Inverts the query/filtering applied by Targets File": "反转目标文件应用的查询/过滤",
    "When this second dataset is also specified, separate stats for intersection and the complements are generated": "当这个第二个数据集也被指定时，将为交集和补集生成单独的统计信息。",
    "Include only first allele at multiallelic sites": "仅包含多等位基因位点上的第一个等位基因",
    "Collect stats for sites with ID separately (known vs novel)": "为具有ID的位点分别收集统计信息（已知 vs 新颖）",
    "Collect Ts/Tv stats for any tag using the given binning: TAG[:min:max:binsize]": "收集任何标签在给定的分箱下Ts/Tv（同义和非同义替换率）的统计信息：TAG[:最小值：最大值：分箱大小]",
    "e.g. 0.1,0.5,1": "例如 0.1,0.5,1",
    "Produce verbose per-site and per-sample output": "生成详细的每个位点和每个样本的输出",
    "Use the specified INFO tag TAG as an allele frequency estimate instead of the defaul AC and AN tags. Sites which do not have TAG will be skipped.": "使用指定的INFO标签TAG作为等位基因频率估计，而不是默认的AC和AN标签。没有TAG的位点将被跳过。",
    "If AF is not known, use this allele frequency": "如果等位基因频率AF未知，使用这个等位基因频率",
    "Use GTs, ignore PLs, use <float> for PL of unseen genotypes. Safe value to use is 30 to account for GT errors.": "使用GTs（基因型），忽略PLs（等位基因可能性），对于未见过的基因型使用<float>作为PL。一个安全的值是30，以考虑GT错误。",
    "Skip indels as their genotypes are enriched for errors": "忽略插入缺失，因为它们的基因型富含错误",
    "Genetic map in IMPUTE2 format, single file or mask, where string \"{CHROM}\" is replaced with chromosome name": "遗传图谱，以IMPUTE2格式，单个文件或掩码，其中字符串\"{CHROM}\"被染色体名称替换",
    "Constant recombination rate per bp": "恒定的每碱基重组率",
    "skip hom-ref genotypes (0/0)": "跳过纯合野生型基因型（0/0）",
    "Ignored by default": "默认忽略",
    "P(AZ|HW) transition probability from HW (Hardy-Weinberg) to AZ (autozygous) state": "从哈迪-温伯格（Hardy-Weinberg）到自交（autozygous）状态的转移概率P(AZ|HW)",
    "P(HW|AZ) transition probability from AZ to HW state": "从AZ状态到HW状态的P(HW|AZ)转移概率",
    "Perform Viterbi training to estimate transition probabilities": "执行Viterbi训练以估计转移概率",
    "New header": "新标题",
    "Fields in your Format are separated by a TAB character: \\t": "你格式中的字段由制表符字符分隔：\\t",
    "Output positions present in this many (=), this many or more (+), this many or fewer (-), the exact (~) files": "输出位置存在于这么多（=）、这么多或更多（+）、这么多或更少（-）、确切的（~）文件中",
    "Output positions present only in the first file but missing in the others": "输出仅在第一个文件中存在而在其他文件中缺失的位置",
    "(-g) Checks sample identity or wehn omitted a multi-sample cross-check is performed": "(-g) 检查样本身份，或者当省略时，执行多样本交叉检查",
    "Target sample in the -g file (used only for plotting)": "在-g文件中的目标样本（仅用于绘图）",
    "Output comparison for all sites": "所有位点的输出比较",
    "Homozygous genotypes only (useful for low coverage data)": "仅纯合子基因型（适用于低覆盖度数据）",
    "Query sample (by default the first sample is checked)": "查询样本（默认情况下检查第一个样本）",
    "Prefix name for plot": "绘图前缀名称",
    "(-g) Filter SNPs within <int> base pairs of an indel": "（-g）在插入或删除序列的<int>个碱基对范围内过滤SNPs",
    "(-G) Filter clusters of indels separated by <int> or fewer base pairs allowing only one to pass": "（-G）筛选由少于或等于<int>个碱基对分隔的插入/缺失（indels）簇，只允许一个通过",
    "Reference sequence and samples are required": "参考序列和样本是必需的",
    "Keep all multiallelic variants": "保留所有多等位基因变异",
    "Use 3N+6 column format instead of the old 3N+5 column format": "使用3N+6列格式代替旧的3N+5列格式",
    "lines which should be appended to the VCF header": "应该添加到 VCF 头部的行",
    "add INFO/TAG flag to sites which are (\"+\") or are not (\"-\") listed in the annotations file": "在注释文件中列出的位点添加INFO/TAG标志（\"+\"）或未列出的位点（\"-\"）。",
    "Minimum overlap required as a fraction of the variant in the annotation -a file (ANN), in the target VCF file (:VCF), or both for reciprocal overlap (ANN:VCF). By default overlaps of arbitrary length are sufficient.": "在注释文件（-a）中，在目标 VCF 文件（:VCF）中，或两者的互惠重叠（ANN:VCF）中，所需的最小重叠作为变体的分数。默认情况下，任意长度的重叠就足够了。",
    "rename chromosomes according to the map in file, with old_name new_name pairs separated by whitespaces, each on a separate line.": "根据文件中的映射表重命名染色体，使用空格分隔的旧名称和新名称对，每行一个。",
    "TYPE/old\\tnew, where TYPE is one of FILTER,INFO,FORMAT": "类型/旧的\\t新的，其中类型是 FILTER、INFO、FORMAT 中的一个",
    "It specifies a VCF file containing genotypes for the study samples. Each VCF record must contain a GT (genotype) format field": "它指定了一个包含研究样本基因型的VCF文件。每个VCF记录必须包含一个GT（基因型）格式字段",
    "Each genotype must have two phased, non-missing alleles. If a VCF file is specified, the phased allele separator must be used '|'": "每个基因型必须有两个相位已知且非缺失的等位基因。如果指定了 VCF 文件，则必须使用相位等位基因分隔符 '|'。",
    "Beagle uses linear interpolation to estimate genetic positions between map positions. If no genetic map is specified, Beagle assumes a constant recombination rate of 1 cM per Mb": "Beagle 使用线性插值来估算基因位置之间的遗传位置。如果没有指定遗传图谱，Beagle 假设重组率为每Mb 1 cM。",
    "It specifies a file containing samples (one sample identifier per line) to be excluded from the analysis": "它指定了一个包含要排除在分析之外的样本的文件（每行一个样本标识符）",
    "It specifies a file containing markers (one marker per line) to be excluded from the analysis. Each line of the file can be either an identifier from a VCF record’s ID field or a genomic coordinate in the format: CHROM:POS": "它指定了一个包含要排除在分析之外的标记的文件（每行一个标记）。该文件的每一行可以是VCF记录的ID字段中的一个标识符，或者是以以下格式的基因组坐标：CHROM:POS",
    "Input format: [chrom]:[start]-[end]. The entire chromosome, the beginning, or the end may be specified by chrom=[chrom], chrom=[chrom]:-[end], and chrom=[chrom]:[start]-, respectively": "输入格式：[染色体]：[起始]-[结束]。整个染色体、起始或结束可以通过 chrom=[染色体], chrom=[染色体]:-[结束], 和 chrom=[染色体]:[起始]- 分别指定",
    "The default value is suitable for a large, outbred population. It is needed to specify an appropriate effective populations size if you are imputing ungenotyped markers in a small or inbred population": "默认值适用于大型、非近交的种群。如果您在小型或近交种群中对未进行基因分型的标记进行插补，需要指定一个适当的有效种群大小。",
    "The window parameter must be at least 1.1 times as large as the overlap parameter. The window parameter controls the amount of memory required for the analysis": "窗口参数必须至少是重叠参数的1.1倍。窗口参数控制分析所需的内存量。",
    "It specifies the cM length of overlap between adjacent sliding windows": "它指定了相邻滑动窗口之间重叠的cM长度",
    "If no err parameter is specified, the err parameter will be set equal 𝜃/(2(𝜃 + 𝐻)) where 𝜃 = 1/(0.5 + ln 𝐻) and 𝐻 is the number of haplotypes": "如果未指定err参数，则err参数将被设置为等于𝜃/(2(𝜃 + 𝐻))，其中𝜃 = 1/(0.5 + ln 𝐻)，𝐻是单倍型的数量",
    "A random seed is a number used to initialize a pseudorandom number generator": "随机种子是一个用于初始化伪随机数生成器的数字",
    "It is the maximum number of burnin iterations used to estimate an initial haplotype frequency model for inferring genotype phase": "这是用于估计初始单倍型频率模型的最大烧入迭代次数，用于推断基因型相位",
    "It is the number of iterations used to estimate genotype phase. Increasing this parameter will trade increased computation time for increased phasing accuracy": "这是用于估计基因型相位的迭代次数。增加此参数将增加计算时间以换取更高的相位准确性",
    "It is the number of model states used to estimate genotype phase": "它是用于估计基因型相位的模型状态数",
    "It specifies whether markers that are present in the reference panel but absent in that target will be imputed. This option has no effect if no reference panel is specified": "它指定了在参考组中存在但在目标中缺失的标记是否将被估算。如果没有指定参考组，则此选项无效。",
    "It is the number of model states used to impute ungenotyped markers": "这是用于插值未基因分型标记的模型状态数",
    "It is the minimum cM length of haplotype segments that will be incorporated in the HMM state space for a target haplotype.": "它是将被纳入目标单倍型隐马尔可夫模型状态空间的单倍型片段的最小cM长度。",
    "It is the length in cM of the step used for detecting short IBS segments": "这是用于检测短IBS（Identity By State，同源性）片段的步长，单位为cM（厘米）。",
    "It specifies the maximum cM distance between individual markers that are combined into an aggregate marker when imputing ungenotyped markers": "它指定了在插补未基因分型标记时，将个别标记组合成聚合标记的最大cM距离",
    "It specifies whether AP1 and AP2 (allele probability) fields will be included in the output VCF file when imputing ungenotyped markers": "它指定在对未进行基因分型标记进行插补时，输出的VCF文件中是否包含AP1和AP2（等位基因概率）字段。",
    "It specifies whether a GP (genotype probability) format field will be included in the output VCF file when imputing ungenotyped markers. Genotype probabilities are calculated from allele probabilities assuming Hardy-Weinberg Equilibrium. Consequently, the alleles in the genotype with highest genotype probability may occasionally be different than the genotype obtained by taking the allele with highest probability on each haplotype, which is the genotype reported in the GT format field": "它指定在对未进行基因分型标记进行插补时，是否在输出的VCF文件中包含一个GP（基因型概率）格式字段。基因型概率是从假设哈迪-温伯格平衡的等位基因概率中计算得出的。因此，具有最高基因型概率的基因型中的等位基因有时会与通过取每个单倍体上最高概率的等位基因获得的基因型不同，后者是在GT格式字段中报告的基因型。",
    "Built-in references were created using default options.": "内置参考序列是使用默认选项创建的。",
    "If your genome of interest is not listed, contact the Galaxy team.": "如果您感兴趣的基因组没有列出，请与Galaxy团队联系。",
    "A reference genome in FASTA format": "一个FASTA格式的参考基因组",
    "An aligned reads file in BAM format. The reads must be aligned to the reference genome": "一个对齐的读取文件，格式为BAM。读取必须与参考基因组对齐。",
    "Type of model to use for variant calling": "用于变异检测的模型类型",
    "Restrict the analysis to specific regions. A space-separated list of chromosome regions to process. Individual elements can be region literals, such as chr20:10-20 or paths to BED files.": "将分析限制在特定区域。一个以空格分隔的染色体区域列表，用于处理。单个元素可以是区域字面量，例如 chr20:10-20，或者是指向 BED 文件的路径。",
    "This option refers to contigs present in the reference genome. These arguments accept space-separated lists, so all of the follow examples are valid arguments: 'chr20:10,000,000-11,000,000', 'chr20 chr21' and 'chr20'": "这个选项指的是参考基因组中存在的连续序列。这些参数接受以空格分隔的列表，因此以下所有示例都是有效的参数：'chr20:10,000,000-11,000,000'，'chr20 chr21' 和 'chr20'",
    "The BED should the store genomic regions of interest": "BED 应该存储感兴趣的基因组区域",
    "The key difference between a regular VCF and a gVCF is that the gVCF has records for all sites, whether there is a variant call there or not. The goal is to have every site represented in the file in order to do joint analysis of a cohort in subsequent steps": "常规VCF和gVCF之间的关键区别在于，gVCF为所有位点都有记录，无论那里是否有变体调用。目标是让文件中的每个位点都有表示，以便在后续步骤中对队列进行联合分析。",
    "Generated by PLINK": "由PLINK生成",
    "Implements the fastest and integrated relationship inference.": "实现最快和集成的关系推断。",
    "Implements the fastest (and accurate) algorithm to identify duplicates or MZ twins": "实现了最快（且准确）的算法来识别重复项或同位素双胎。",
    "Estimates pair-wise kinship coefficients": "估计成对亲权系数",
    "IBD segment analysis determines all IBD (IBD1 and IBD2) segments shared between relatives": "IBD段分析确定亲属之间共享的所有IBD（IBD1和IBD2）段",
    "Counts and average of IBS": "IBS的计数和平均值",
    "Estimates pair-wise kinship coefficients assuming a homogeneous population.": "估计同质种群中个体间的亲缘关系系数。",
    "Filters relative pairs based on kinship coefficients.": "根据亲缘关系系数过滤相对配对。",
    "specifies the minimum length of IDB segments that are considered towards the relationship inference": "指定在关系推断中考虑的IDB片段的最小长度",
    "Extract a list of unrelated individuals.": "提取一份无关个体的列表。",
    "Clusters relatives into families by generating an updateid file.": "将相似的序列聚类成家族，通过生成一个updateid文件。",
    "Reconstructs pedigrees using SNP data": "使用SNP数据重建家谱",
    "allow the specification of the number of PCs with a default value of 10.": "允许指定主成分的数量，默认值为10。",
    "Includes the first N samples of a subset.": "包含子集中的前N个样本。",
    "implements the well-known Transmission/Disequilibrium Test for family data that consist of parent-affected child trios.": "实现了著名的传输/不平衡检验，用于由父母-受影响的孩子三口之家组成的家庭数据。",
    "Only association results with P value less than 5E-8 are printed out, including both cis- and trans- effect associations.": "只有P值小于5E-8的关联结果被打印出来，包括顺式和反式效应关联。",
    "Specifies the trait names to be analyzed in the association analysis.": "指定要在关联分析中分析的性状名称。",
    "Specifies the covariate names to be adjusted in the association analysis": "指定在关联分析中要调整的协变量名称",
    "Carries out inverse normal transformation for quantitative traits prior to association analysis.": "在关联分析之前对定量性状进行逆正态转换。",
    "specifies the maximum P values to print out in the output files.": "指定在输出文件中打印出的最大P值。",
    "predicts disease risks for each individual according to the GRS risk model": "根据GRS风险模型预测每个个体的疾病风险",
    "If strands of genotype data are already consistent with model, enable this.": "如果基因型数据的链已经与模型一致，启用此选项。",
    "specifies the pair number of the sex chromosome, which should be userful for non-human species. The default sex chromosome is 23.": "指定性染色体的对数，这对于非人类物种应该很有用。默认的性染色体是23。",
    "Generates R code first and then calls R program to make plots in a PDF file.": "生成 R 代码，然后调用 R 程序在 PDF 文件中制作图表。",
    "Generates R code first and then calls R program to make plots in a PNG file for certain applications.": "生成R代码，然后调用R程序为某些应用生成PNG文件中的图表。",
    "Multiple datasets selected here will get combined into a single output for a single assumed sample. Select a nested list to have its inner lists interpreted as data from one sample each and to obtain one output per inner list.": "在这里选择的多个数据集将被合并成一个单一的输出，用于一个假定的样本。选择一个嵌套列表，将其内部列表解释为每个样本的数据，并为每个内部列表获得一个输出。",
    "remove reads greater than this number of base pairs": "删除大于这个碱基对数的读段",
    "remove reads less than this number of base pairs": "删除少于这个碱基对数的读取",
    "Set to 0 to skip the quality check.": "设置为0以跳过质量检查。",
    "Must be formatted in modified official VCF format as detailed here: https://github.com/PharmGKB/PharmCAT/wiki/VCF-Requirements": "必须按照这里详细说明的修改后的官方 VCF 格式进行格式化：https://github.com/PharmGKB/PharmCAT/wiki/VCF-Requirements",
    "JSON output of the NamedAlleleCaller pharmcat function": "NamedAlleleCaller 药物分类函数的 JSON 输出",
    "Bam files produced by BBMap are recommended": "推荐使用BBMap生成的Bam文件",
    "See help section below": "查看下面的帮助部分",
    "Ensembl strongly recommends to only use annotation cache files with a version number matching the VEP version. You can disable the corresponding filtering of available cache files at your own risk. This does not apply if you select a GFF/GTF annotation source.": "Ensembl强烈建议只使用与VEP版本号相匹配的注释缓存文件。你可以自行承担风险，禁用对可用缓存文件的相应过滤。如果你选择GFF/GTF注释源，这则不适用。",
    "If the annotation data of interest is not listed, have a look at all available cache files regardless of their version number or contact your Galaxy admin.": "如果感兴趣的注释数据没有列出，请查看所有可用的缓存文件，无论它们的版本号如何，或者联系您的Galaxy管理员。",
    "If the annotation data of interest is not listed, contact your Galaxy admin.": "如果感兴趣的注释数据没有列出，请与您的Galaxy管理员联系。",
    "Choosing this option requires that you also supply a Fasta file as reference genome.": "选择此选项需要您同时提供一个Fasta文件作为参考基因组。",
    "Must be specified if using a GFF/GTF annotation source or if fetching HGVS or SPDI notations.": "如果使用GFF/GTF注释源，或者获取HGVS或SPDI表示法时，必须指定。",
    "SIFT predicts whether an amino acid substitution affects protein function based on sequence homology and the physical properties of amino acids.": "SIFT 预测氨基酸替换是否影响蛋白质功能，基于序列同源性和氨基酸的物理性质。",
    "PolyPhen is a tool which predicts possible impact of an amino acid substitution on the structure and function of a human protein using straightforward physical and comparative considerations. VEP uses the humVar score by default.": "PolyPhen是一个工具，它使用直接的物理和比较考虑来预测氨基酸替换对人类蛋白质结构和功能的可能影响。VEP默认使用humVar分数。",
    "Retrieve the transcript or gene with the nearest protein-coding transcription start site (TSS) to each input variant. Note that the nearest TSS may not belong to a transcript that overlaps the input variant, and more than one may be reported in the case where two are equidistant from the input coordinates.": "检索每个输入变异点最近的编码蛋白质的转录起始位点（TSS）。请注意，最近的TSS可能不属于与输入变异重叠的转录本，如果两个与输入坐标等距离，则可能会报告多个。",
    "Default: 5000. Upstream is 5' and downstream 3' direction. (--distance [bp_distance(,downstream_distance_if_different)])": "默认值：5000。上游方向为5'，下游方向为3'。(--distance [碱基对距离(,如果不同则为下游距离)])",
    "Recommended for advanced users only. Can be a single cell type or a comma-separated list. The functional type in each cell type is reported under CELL_TYPE in the output. (--cell_type)": "仅推荐高级用户使用。可以是单个细胞类型或逗号分隔的列表。每个细胞类型中的功能类型在输出中的CELL_TYPE下报告。（--cell_type）",
    "Enter the individual's IDs as comma-separated list. Individual variant combinations homozygous for the given reference allele will not be reported. Each individual and variant combination is given on a separate line of output. Only works with VCF files containing individual genotype data. (--individual [ind list])": "输入个体的ID，以逗号分隔的列表。对于给定的参考等位基因纯合的个体变异组合将不会被报告。每个个体和变异组合在输出中占据单独的一行。仅适用于包含个体基因型数据的VCF文件。（--individual [ind list]）",
    "Default: CSQ. Use ANN for compatibility with other tools such as snpEff (--vcf_info_field [CSQ|ANN|(other)])": "默认值：CSQ。使用ANN以兼容其他工具，如snpeff（--vcf_info_field [CSQ|ANN|(其他)]）",
    "The default used for terms is Sequence Ontology.": "默认使用的术语是序列本体论。",
    "File should be tab-delimited with the primary identifier in column 1 and the synonym in column 2. Synonyms allow different chromosome identifiers to be used in the input file and any annotation source. (--synonyms [file])": "文件应该是制表符分隔的，第一列是主要标识符，第二列是同义词。同义词允许在输入文件和任何注释源中使用不同的染色体标识符。（--synonyms [文件]）",
    "This plugin calculates the Combined Annotation scoRing toOL (CAROL) score for a missense mutation and adds it to VEP's output. Both the SIFT and PolyPhen-2 scores are required for the calculation, so these two have to be activated in the output options section.": "这个插件计算了联合注释评分工具（CAROL）的分数，用于一个错义突变，并将该分数添加到VEP的输出中。计算需要SIFT和PolyPhen-2两个分数，因此这两个分数需要在输出选项部分激活。",
    "This plugin calculates the Consensus Deleteriousness (Condel) score for a missense mutation and adds it to VEP's output. Both the SIFT and PolyPhen-2 scores are required for the calculation, so these two have to be activated in the output options section.": "这个插件计算错义突变的共识有害性（Condel）得分，并将该得分添加到VEP的输出中。计算需要SIFT和PolyPhen-2得分，因此这两个得分必须在输出选项部分激活。",
    "This plugin calculates the probabililty of a gene being loss-of-function intolerant (pLI) and adds it to VEP's output. The closer pLI is to 1, the more likely the gene is loss-of-function (LoF) intolerant. An ExACpLI_values.txt file is required for usage and it is recommended to use the one provided in the VEP_plugins GitHub repository: https://github.com/Ensembl/VEP_plugins": "这个插件计算一个基因失去功能不耐受（loss-of-function intolerant，简称 pLI）的概率，并将该概率添加到变异效应预测器（VEP）的输出中。pLI 越接近 1，该基因越有可能对失去功能（LoF）不耐受。使用该插件需要一个 ExACpLI_values.txt 文件，建议使用 VEP_plugins GitHub 仓库中提供的文件：https://github.com/Ensembl/VEP_plugins",
    "This plugin calculates LoFtool scores and adds them to VEP's output. LoFtool provides a rank of genic intolerance and consequent susceptibility to disease based on the ratio of Loss-of-function (LoF) to synonymous mutations for each gene in 60,706 individuals from ExAC, adjusting for the gene de novo mutation rate and evolutionary protein conservation. The lower the LoFtool gene score percentile the most intolerant is the gene to functional variation. A LoFtool_scores.txt file is required for usage and it is recommended to use the one provided in the VEP_plugins GitHub repository: https://github.com/Ensembl/VEP_plugins": "这个插件计算LoFtool分数，并将它们添加到VEP的输出中。LoFtool提供了基于ExAC中60,706个个体每个基因的无义突变（LoF）与同义突变比率的基因不耐受性的排名，同时考虑基因新突变率和进化蛋白保守性。LoFtool基因分数百分位数越低，该基因对功能变异的不耐受性越高。使用时需要一个LoFtool_scores.txt文件，建议使用VEP_plugins GitHub存储库中提供的文件：https://github.com/Ensembl/VEP_plugins",
    "Detect, annotate, or compare variants.": "检测、注释或比较变异。",
    "**Polymorphism mode**: Detect variants with frequencies between 0% and 100% if a mixture model is well-supported by the read alignment evidence. Use to analyze a mixed population of genomes evolved from a common ancestor. **Consensus mode**: Detect variants present in 100% of the sample. Use when re-sequencing a clonal haploid genome. This mode is the default.": "**多态性模式**：如果混合模型得到读取对齐证据的良好支持，则检测频率在0%到100%之间的变体。用于分析从共同祖先演化而来的混合基因组群体。**共识模式**：检测在样本100%中存在的变体。用于重测序克隆单倍体基因组。此模式为默认模式。",
    "Human-readable name of the analysis run for output (DEFAULT=none).": "输出分析运行的人可读名称（默认=无）。",
    "Predict new sequence junctions (default).  --no-junction-prediction is supplied if 'No' is selected.  Otherwise, there is no flag.": "预测新序列连接点（默认）。如果选择“否”，则提供--no-junction-prediction参数。否则，没有标志。",
    "Files as produced by breseq": "由breseq生成的文件",
    "vcf2maf can utilize Ensembl's VEP to select a single effect per variant. VEP can only be used if SIFT is available for the selected genome assembly. Ensembl strongly recommends to only use annotation cache files with a version number matching the VEP version. You can disable the corresponding filtering of available cache files at your own risk.": "vcf2maf 可以使用 Ensembl 的 VEP 来为每个变体选择一个单一的效应。只有当为选定的基因组组装提供了 SIFT 时，才能使用 VEP。Ensembl 强烈建议只使用与 VEP 版本号相匹配的注释缓存文件。你可以自行承担风险禁用可用缓存文件的相应过滤。",
    "Used to fill the Tumor_Sample_Barcode column of the output MAF with the tumor sample ID.": "用于填充输出的MAF文件中的Tumor_Sample_Barcode列，使用肿瘤样本ID。",
    "Used to fill the Matched_Norm_Sample_Barcode column of the output MAF with the normal sample ID.": "用于填充输出的MAF文件中的Matched_Norm_Sample_Barcode列，以填入正常样本ID。",
    "VCFs from variant callers like VarScan use hardcoded sample IDs TUMOR/NORMAL to name genotype columns. Use this parameter to have vcf2maf correctly locate these columns to parse genotypes, while still printing proper sample IDs in the output MAF.": "变异调用软件如VarScan使用硬编码样本ID TUMOR/NORMAL来命名基因型列。使用此参数，vcf2maf可以正确定位这些列以解析基因型，同时在输出的MAF中仍然打印正确的样本ID。",
    "The MAF version header is a comment that indicates the MAF version and is written into the first row of the ouput file. This is required by some downstream tools. The row containing column headers follows next.": "MAF版本头部是一个注释，它指示了MAF的版本，并写入到输出文件的第一行。这是一些下游工具所必需的。紧接着是包含列标题的行。",
    "Default value is 0.7": "默认值是0.7",
    "Must contain a series of sequences with different names and the same length. Both N and - are treated as missing information.": "必须包含一系列具有不同名称且长度相同的序列。N和-都被视为缺失信息。",
    "Which sequence from the FASTA file should be used as the reference sequence.": "在FASTA文件中，应该使用哪个序列作为参考序列。",
    "Must be present in the FASTA file.": "必须在FASTA文件中存在。",
    "If 1: Treat ambiguous bases as N, ambiguous bases (N, R, V etc.) are treated as N (no call). If 2: Resolve ambiguous characters, if the character represents two bases and one is the reference base, convert it to the non-reference base. Otherwise convert it to N. Default: 0: Don't treat ambiguous bases": "如果选择1：将模糊的碱基视为N，模糊的碱基（N、R、V等）被视为N（未检测到）。如果选择2：解析模糊的字符，如果字符代表两个碱基，其中一个是参考碱基，将其转换为非参考碱基。否则，将其转换为N。默认：0：不处理模糊碱基",
    "Exclude sequences named in file which has one sequence name per line.": "排除文件中命名的序列，每个序列名称占一行。",
    "Exclude sequences with more than N mismatches with the reference sequence. If -windowSize is used, sequences are masked accordingly before the mismatches are counted. Default: 0": "排除与参考序列有超过N个不匹配的序列。如果使用了-windowSize参数，则在计数不匹配之前相应地对序列进行掩蔽。默认值：0",
    "Exclude variants in positions recommended for masking in file. Typically https://github.com/W-L/ProblematicSites_SARS-CoV2/raw/master/problematic_sites_sarsCov2.vcf": "排除文件中建议遮蔽的位置的变异。通常为 https://github.com/W-L/ProblematicSites_SARS-CoV2/raw/master/problematic_sites_sarsCov2.vcf",
    "Mask any base for which there are at least -minAmbigWindow bases in a window of +-N bases around the base. Masking approach adapted from https://github.com/roblanf/sarscov2phylo/blob/master/scripts/mask_seq.py Use -windowSize=7 for same results. Default: 0": "对于周围有至少-minAmbigWindow个碱基的窗口中，对每个碱基进行掩蔽。掩蔽方法改编自 https://github.com/roblanf/sarscov2phylo/blob/master/scripts/mask_seq.py 使用-windowSize=7 可以获得相同的结果。默认值：0",
    "When -windowSize is provided, mask any base for which there are at least this many N, ambiguous or gap characters within the window. Default: 2": "当提供-windowSize参数时，将掩蔽窗口内至少有这么多N、模糊或间隔字符的任何碱基。默认值：2",
    "Include base positions with no alternate alleles observed, but at least one N (missing base/no-call). Default: false": "包括没有观察到替代等位基因的碱基位置，但至少有一个N（缺失碱基/未调用）。默认值：false",
    "Ignore alternate alleles observed fewer than N times. Default: 0": "忽略观察到少于N次的替代等位基因。默认值：0",
    "Ignore alternate alleles observed in less than F of non-N bases. Default: 0.0": "忽略在非N碱基中观察到的频率小于F的替代等位基因。默认值：0.0",
    "Add N bases to each position, e.g. for trimmed alignments. Default: 0": "在每个位置上添加N个碱基，例如用于修剪后的比对。默认值：0",
    "Default: omitted as redundant (false)": "默认：作为冗余省略（false）",
    "VCF without the sample genotype columns. Default: false": "不包含样本基因型列的VCF。默认值：false",
    "Default: name of the reference sequence.": "默认值：参考序列的名称。",
    "Reference genome fasta file": "参考基因组fasta文件",
    "Regions separated by less than this distance will be joined together": "距离小于这个值的区域将会被合并在一起",
    "e.g. all possible exons for the reference genome. Format: BED, interval list, etc.": "例如，参考基因组的所有可能的外显子。格式：BED、区间列表等。",
    "TAS-on-target coverage cnn file is also accepted": "TAS-on-target 覆盖率 cnn 文件也被接受",
    "Otherwise inferred from file names": "否则从文件名推断",
    "Sample coverages as raw probes cnr or segments cns": "样本覆盖度作为原始探针 cnr 或片段 cns",
    "Use the output of the CNVkit fix": "使用 CNVkit fix 的输出",
    "Format: UCSC refFlat.txt or ensFlat.txt file, BED, interval list, GFF, or similar": "格式：UCSC refFlat.txt 或 ensFlat.txt 文件，BED，区间列表，GFF 或类似格式",
    "Fasta file containing the sample's consensus sequence (majority, with indels)": "包含样本共识序列的Fasta文件（多数，带有插入和缺失）",
    "Input BAM file with sample's sequencing reads, aligned against the reference": "输入BAM文件，包含样本的测序读取，与参考序列比对",
    "To find insertions and deletions, the tool needs information how the consensus aligns to the reference (lift-over). You can provide a .chain file describing how the consensus maps to the reference, otherwise mafft will be used to align the consensus to the reference.": "为了找到插入和删除，该工具需要了解共识序列与参考序列（lift-over）的比对方式。您可以提供一个.chain文件来描述共识序列如何映射到参考序列，否则将使用mafft将共识序列与参考序列进行比对。",
    "Chain file describing how the consensus is aligned to the reference (e.g. ouput of `bcftools consensus --chain …`).": "描述共识序列如何与参考序列对齐的链文件（例如 `bcftools consensus --chain …` 的输出）。",
    "Select built-in genome files to base reported positions and annotations on the SARS-CoV-2 reference sequence NC_045512.2. If you have mapped to a different reference, select custom genome files and provide the reference sequence and genomic feature annotations for it in fasta and gff format, repsectively.": "选择内置的基因组文件以基于SARS-CoV-2参考序列NC_045512.2报告位置和注释。如果您已映射到不同的参考序列，请选择自定义基因组文件，并分别为其提供fasta格式的参考序列和gff格式的基因组特征注释。",
    "Fasta input containing the reference sequence (used during alignment) to compare against": "Fasta 输入包含参考序列（在比对期间使用）以进行比较",
    "GFF input listing genes positions on the reference sequence": "GFF 输入列出了参考序列上的基因位置",
    "ID of the full CDS comprising the ribosomal slippage site as it appears in the GFF input. In the GFF this CDS should consist of 2 entries with the same CDS ID due to the partial overlap caused by the ribosomal shift at translation time. Hint: To use the tool with genomes that do not have a ribosomal slippage site, enter an ID that is not present in the GFF.": "在GFF输入中，包含核糖体滑动位点的完整CDS的ID。在GFF中，由于翻译时核糖体的移位导致部分重叠，这个CDS应该由两个具有相同CDS ID的条目组成。提示：如果要使用该工具处理没有核糖体滑动位点的基因组，请输入一个在GFF中不存在的ID。",
    "If checked writes english summary diagnosis.": "如果选中，则写入英文摘要诊断。",
    "Use 0-based instead of 1-based genome positions": "使用基于0的基因组位置而不是基于1的位置",
    "Select between a single normal BAM file or a pair of normal/tumor BAM files": "选择单个正常BAM文件或一对正常/肿瘤BAM文件",
    "Select the files you wish to send to Manta (normal sample, it must be in BAM format).": "选择您希望发送到Manta（正常样本，必须是BAM格式）的文件。",
    "Select the files you wish to send to Manta (tumor sample, it must be in BAM format).": "选择您要发送到Manta（肿瘤样本，必须是BAM格式）的文件。",
    "The maximum memory size to assign to tasks": "分配给任务的最大内存大小",
    "The maximum sequence region size (in megabases) scanned by each task during SV Locus graph generation. (default: 12)": "在结构变异位点图生成期间，每个任务扫描的最大序列区域大小（以兆碱基为单位）。（默认值：12）",
    "Run discovery and candidate reporting for all SVs/indels at or above this size.": "运行发现和候选报告，针对所有大小达到或超过此大小的SVs/indels。",
    "Separate option (to provide different default) used for runs in RNA-mode.": "独立的选项（用于提供不同的默认值）用于RNA模式下的运行。",
    "Remove all edges from the graph unless they're supported by this many 'observations'.": "除非这些边由这么多的“观察”支持，否则从图中删除所有边。",
    "If both nodes of an edge have an edge count higher than this, then skip evaluation of the edge.": "如果一个边的两个节点的边数都高于这个值，则跳过该边的评估。",
    "Run discovery and candidate reporting for all SVs/indels with at least this many spanning support observations.": "运行发现和候选报告，对于所有具有至少这么多跨越支持观察的SVs/indels。",
    "After candidate identification, only score and report SVs/indels at or above this size.": "在候选者识别后，只对等于或大于此大小的SVs/indels进行评分和报告。",
    "Minimum VCF 'QUAL' score for a variant to be included in the diploid vcf.": "在二倍体vcf中包含一个变异所需的最小VCF 'QUAL'分数。",
    "VCF 'QUAL' score below which a variant is marked as filtered in the diploid vcf.": "在二倍体 VCF 文件中，将低于某个特定 'QUAL' 分数的变异标记为过滤掉的。",
    "Minimum genotype quality score below which single samples are filtered for a variant in the diploid vcf.": "在二倍体 VCF 中，对于单个样本的变体，过滤掉最小基因型质量分数低于该值的样本。",
    "Somatic quality scores below this level are not included in the somatic vcf.": "体细胞质量评分低于这个水平的不包括在体细胞变异文件(vcf)中。",
    "Somatic quality scores below this level are filtered in the somatic vcf.": "在体细胞变异的vcf文件中，低于此水平的体细胞质量得分会被过滤掉。",
    "Remote read retrieval is used ot improve the assembly of putative insertions by retrieving any mate reads in remote locations with poor mapping quality. This feature can be enabled/disabled separately for germline and cancer calling below.": "远程读取检索用于通过检索具有较差映射质量的远程位置中的任何配对读取来改善假定插入的组装。此功能可以分别针对生殖系和癌症调用启用/禁用。",
    "Here 'CancerCallingModes' includes tumor-normal subtraction and tumor-only calling. 'GermlineCallingModes' includes all other calling modes.": "在这里，“CancerCallingModes”包括肿瘤-正常组织减法和仅肿瘤调用。“GermlineCallingModes”包括所有其他调用模式。",
    "Set if an overlapping read pair will be considered as evidence. Set this value <= 0 to skip overlapping read pairs.": "设置是否将重叠的读段对视为证据。将此值设置为<= 0 可跳过重叠的读段对。",
    "Show configuration file on history": "在历史记录中显示配置文件",
    "All unscored structural variant candidates": "所有未评分的结构变异候选",
    "Subset of the unscored candidates, containing only small indel variants": "未评分候选子集，仅包含小的插入缺失变异",
    "Show filtered variants in a diploid (only normal) model. In the case of a tumor/normal subtraction, the scores in this file *do not* reflect any information from the tumor sample": "在二倍体（仅正常）模型中显示过滤后的变异。在肿瘤/正常减法的情况下，此文件中的分数*不*反映来自肿瘤样本的任何信息",
    "The output genomic variations VCF file of bcftools, snpeff, snpsift": "bcftools、SnpEff 和 SnpSift 的输出基因组变异 VCF 文件",
    "Give an ID for the created genomicVariations dataset": "为创建的 genomicVariations 数据集指定一个 ID",
    "Select the reference genome used the annotate the data to create the genomicVariations dataset examples for reference genomes are hs37, hg37 and hg38": "选择用于注释数据以创建基因组变异数据集的参考基因组，参考基因组的示例包括hs37、hg37和hg38。",
    "The output of the CNVkit fix": "CNVkit fix的输出",
    "The output of the CNVkit segment": "CNVkit 分段的输出",
    "Minimum number of covered probes to label a gene": "标记一个基因所需的最小覆盖探针数",
    "Min Number of covered probes to report a gain/loss": "覆盖探针的最小数量以报告增益/缺失",
    "otherwise male samples would have -1 chrX": "否则，男性样本的X染色体数量将为-1。",
    "To avoid false-positive deletions in poor-quality tumor samples": "为了避免在质量较差的肿瘤样本中出现误删",
    "Copy number cnn or copy ratio cnr files": "拷贝数CNN或拷贝比CNR文件",
    "Assume inputs were normalized to a male reference": "假设输入已经标准化到男性参考标准",
    "Minor allele sequences must be labeled \"_minor\" or \"_test\" e.g. \">sample1_minor\"": "次要等位基因序列必须标记为\"_minor\"或\"_test\"，例如\">sample1_minor\"",
    "Must have the same length as input sequences": "必须与输入序列的长度相同",
    "Defaults to \"Complete deletion\"": "默认为“完全删除”",
    "0 to deactivate": "0 以停用",
    "All sequences will be treated as major allele sequences. Useful to generate background tree": "所有序列都将被视为主要等位基因序列。这有助于生成背景树。",
    "must be tabular and follow the Variant Annotator tool output format": "必须是表格形式，并遵循变异注释工具输出格式",
    "e.g. 16569 for mitochondrial variants": "例如，16569用于线粒体变异",
    "These will be kept/discarded (depending on 'operation'). <BR /> Examples: 1,3,4 or 2-5": "这些将被保留/丢弃（取决于'操作'）。<BR /> 示例：1,3,4 或 2-5",
    "Use if first line contains column headers. It will not be sorted.": "如果第一行包含列标题，则使用。它将不会被排序。",
    "Sort and Join key column values regardless of upper/lower case letters.": "忽略大小写字母，对键列的值进行排序和连接。",
    "Use simple text, or a valid regular expression (without backslashes // )": "使用简单文本，或一个有效的正则表达式（不使用反斜杠 // ）",
    "Use simple text, or $& (dollar-ampersand) and $1 $2 $3 to refer to matched text. See examples below.": "使用简单的文本，或者使用 $&（美元-和号）和 $1 $2 $3 来引用匹配的文本。请参阅下面的示例。",
    "see help section for details.": "查看帮助部分了解详细信息。",
    "ignore partial matches (e.g. 'apple' will not match 'snapple')": "忽略部分匹配（例如 'apple' 不会匹配 'snapple'）",
    "Select this option if the first line contains column headers. Text in the line will not be replaced.": "如果第一行包含列标题，请选择此选项。该行中的文本不会被替换。",
    "See below for more details": "详见下文",
    "leave it at zero unless you know what you're doing. (-B)": "保持零值，除非你清楚你在做什么。（-B）",
    "leave it at zero unless you know what you're doing. (-A)": "除非您知道自己在做什么，否则请将其保持为零。(-A)",
    "These will be kept/discarded depending on 'operation'. (--lines)": "这些将根据'operation'保留/丢弃。(--lines)",
    "Usually gene-ID or other common value": "通常为基因ID或其他常见值",
    "If not set, duplicated keys in the same file will cause an error.": "如果未设置，同一文件中的重复键将导致错误。",
    "Specify the characters that will be inserted X times in every line": "在每一行中指定将插入X次的字符",
    "For every line, the specified characters will be written once. That means X is the line-number from the given file.": "对于每一行，将写入指定的字符一次。这意味着X是从给定文件中的行号。",
    "Use simple text, or & (ampersand) and \\\\1 \\\\2 \\\\3 to refer to matched text. See examples below.": "使用简单的文本，或者使用 & （和号）和 \\1 \\2 \\3 来引用匹配的文本。请参阅下面的示例。",
    "Same as 'sed -n', leave at 'normal' unless you know what you're doing.": "与'sed -n'相同，除非你知道你在做什么，否则保持为'normal'。",
    "These will be ignored during sort.": "这些在排序时将被忽略。",
    "Print only unique values, based on sorted key columns. See help section for details. (--unique)": "仅打印基于排序的键列的唯一值。有关详细信息，请参见帮助部分。（--unique）",
    "Sort and Join key column values regardless of upper/lower case letters. (-i)": "根据字母的大小写排序并合并关键列的值。(-i)",
    "Make sure you have sorted this file": "确保您已对此文件进行排序",
    "Prefix lines by the number of occurrences. (-c)": "在行前加上出现次数。（-c）",
    "Use zero to start from the first field. (-f)": "使用零作为起始字段（-f）。",
    "These will be kept (depending on 'operation'). (--lines)": "这些将被保留（取决于'operation'参数）。（--lines）",
    "Unique on specific column range": "特定列范围内的唯一值",
    "Select Yes to be able to specify names for new columns and have them added to the header line. If you select No, the first line will be treated as a regular line: If it is empty or starts with a # character it will be skipped, otherwise the tool will attempt to compute the specified expression on it.": "选择“是”可以为新列指定名称并将它们添加到标题行中。如果您选择“否”，则第一行将被视为常规行：如果它是空的或以“#”字符开头，它将被跳过；否则，工具将尝试计算它上面的指定表达式。",
    "If yes, use fully expanded decimal representation when writing new columns with floating point values. To prevent scientific notation in just specific new columns, you can use numpy's format_float_positional function in the corresponding expression.": "如果需要，在写入带有浮点值的新列时使用完全展开的小数表示。为了防止在特定新列中使用科学记数法，您可以在相应的表达式中使用 numpy 的 format_float_positional 函数。",
    "By default, try to use the column types that Galaxy has recorded for the input. This simplifies expressions, but can occasionally cause problems on its own. If disabled all column values are assumed to be strings and you will have to handle conversions to different types explicitly in the expression.": "默认情况下，尝试使用Galaxy为输入记录的列类型。这简化了表达式，但有时也可能导致问题。如果禁用，所有列值都假定为字符串，您将不得不在表达式中显式处理转换为不同类型。",
    "If any expression references a column number that does not exist when that expression gets computed, the tool run will fail. Uncheck to have such a situation handled as a case of a non-computable expression as configured below.": "如果任何表达式引用了一个在计算该表达式时不存在的列号，那么工具运行将失败。取消选中可以将这种情况处理为无法计算表达式的情况，如下面配置的那样。",
    "Pick from suggestions or enter your own.": "从建议中选择或输入您自己的内容。",
    "Generates an HTML report to visualize the differences": "生成一个HTML报告以可视化差异",
    "Displays the differences in the report using the selected format. Either line by line or side by side.": "使用选定的格式显示报告中的不同之处。可以逐行显示或并排显示。",
    "The entries of a specific column are replaced by the information given by the next input file.": "特定列的条目被下一个输入文件给出的信息替换。",
    "This file contains in the first column the entries that should be replaced by the values of the second column.": "这个文件的第一列包含了应该被第二列的值替换的条目。",
    "select a column header to be added in the first line": "选择一个要在第一行添加的列标题",
    "Dataset missing? See TIP below.": "数据集缺失？请参阅下面的提示。",
    "Need to add more columns? Use controls below.": "需要添加更多列？请使用下方的控件。",
    "Operations on single tables are better tailored towards more general use-cases. For multiple tables usage, all tables should ideally be of the same dimensions for simple operations, although complex operations are also supported for unequally sized tables (see Help section).": "单表操作更适合更一般的用例。对于多表使用，所有表理想情况下应该具有相同的维度以进行简单操作，尽管对于尺寸不同的表也支持复杂操作（见帮助部分）。",
    "Keep duplicates when specifying ranges (e.g. if unset, @COPEN@1:3,2:4@CCLOSE@ will yield @COPEN@1,2,3,4@CCLOSE@ instead of @COPEN@1,2,3,2,3,4@CCLOSE@)": "在指定范围时保留重复项（例如，如果未设置，@COPEN@1:3,2:4@CCLOSE@ 将产生 @COPEN@1,2,3,4@CCLOSE@ 而不是 @COPEN@1,2,3,2,3,4@CCLOSE@）",
    "See Example #2 for an example of using this mode.": "参见示例 #2 以了解如何使用此模式。",
    "The parameter name is @COPEN@vec@CCLOSE@, referring to a vector of a specific row or column being operated on. Numpy and Pandas DataFrame operators are supported. e.g. @COPEN@np.sum(vec) + np.median(vec)@CCLOSE@, and @COPEN@vec.sum() + vec.median()@CCLOSE@ are equivalent operations.": "参数名为 @COPEN@vec@CCLOSE@，指的是正在操作的特定行或列的向量。支持 Numpy 和 Pandas DataFrame 运算符。例如，@COPEN@np.sum(vec) + np.median(vec)@CCLOSE@ 和 @COPEN@vec.sum() + vec.median()@CCLOSE@ 是等效的操作。",
    "See Examples 5, 7, and 8 for usage": "参见示例5、7和8了解使用方法",
    "Comma-delimited list of column names to use as identifiers": "逗号分隔的用作标识符的列名列表",
    "Comma-delimited list of column names to un-pivot. Leave blank to use all.": "逗号分隔的要取消行列转换的列名列表。留空则使用全部列。",
    "Name of the column to use as new index": "用作新索引的列名称",
    "Name of the column to use to generate the columns of the new table from. Cannot be used in conjunction with the Column-Function Mapping parameter.": "用于生成新表格列的列名称。不能与“列-函数映射”参数一起使用。",
    "Names of the columns to use for populating the cells of the new table. Leave blank to use all.": "用于填充新表格单元格的列名称。留空则使用所有列。",
    "The function used to aggregate the values for each unique index. Cannot be used with the Column-Function Mapping parameter below. By default this value is np.mean. Leave blank for default behaviour. See Example 10 for more information on its usage.": "用于聚合每个唯一索引的值的函数。不能与下面的列-函数映射参数一起使用。默认情况下，这个值是 np.mean。留空表示默认行为。有关其用法的更多信息，请参见示例 10。",
    "Either a numeric index, or a value column name.": "可以是一个数值索引，或者一个值列的名称。",
    "The function used to aggregate the values for each unique index. By default this is np.mean. See Example 11 for more information on its usage.": "用于聚合每个唯一索引的值的函数。默认情况下这是 np.mean。有关其用法的更多信息，请参见示例 11。",
    "The parameter name is @COPEN@table@CCLOSE@ and @COPEN@axis@CCLOSE@, referring to the table being acted on and the column (@COPEN@0@CCLOSE@) or row (@COPEN@1@CCLOSE@) to perform the operation on. Numpy, math, Pandas DataFrame operators, and inline @COPEN@if else@CCLOSE@ are supported (e.g. @COPEN@np.log(table) - table.mean(0) / table.std(1)@CCLOSE@). See Example #5 in the Help section.": "参数名为 @COPEN@table@CCLOSE@ 和 @COPEN@axis@CCLOSE@，指的是正在操作的表格以及要执行操作的列（@COPEN@0@CCLOSE@）或行（@COPEN@1@CCLOSE@）。支持 Numpy、math、Pandas DataFrame 运算符以及内联 @COPEN@if else@CCLOSE@（例如 @COPEN@np.log(table) - table.mean(0) / table.std(1)@CCLOSE@）。请参阅帮助部分的示例 #5。",
    "If you want to reuse the current value of each element as part of the  replacement value, you can refer to it using the special @COPEN@{elem}@CCLOSE@ placeholder (see Example #9 in the Help section).": "如果你想要在替换值中重用每个元素的当前值，你可以使用特殊的@COPEN@{elem}@CCLOSE@占位符来引用它（参见帮助部分的示例#9）。",
    "Example: to transform values to their square root, select @COPEN@Square Root@CCLOSE@ here.": "示例：要将值转换为其平方根，在这里选择@COPEN@平方根@CCLOSE@。",
    "The selected operation will be applied to the table elements as the first operand. The value of the second operand can be provided in the next box. Example: to raise all values to the power of 5, select @COPEN@Power@CCLOSE@ here and type @COPEN@5@CCLOSE@ in the box below.": "所选操作将作为第一个操作数应用于表格元素。第二个操作数的值可以在下一个框中提供。示例：要将所有值提高到5的幂，在这里选择@COPEN@Power@CCLOSE@，然后在下面的框中输入@COPEN@5@CCLOSE@。",
    "The parameter name is @COPEN@elem@CCLOSE@, referring to the element being acted on. Most operators and @COPEN@if@CCLOSE@ @COPEN@else@CCLOSE@ statements are supported. See Examples #3 and #4 in the Help section.": "参数名为 @COPEN@elem@CCLOSE@，指的是正在操作的元素。大多数运算符和 @COPEN@if@CCLOSE@ @COPEN@else@CCLOSE@ 语句都是支持的。请参见帮助部分的示例 #3 和 #4。",
    "Only selected elements will be manipulated. Other elements will retain their original value.": "只有选定的元素会被操作。其他元素将保留其原始值。",
    "If enabled, all elements that match ANY row or column given above will be acted on. Otherwise, only the elements that match BOTH a row or column given above will be acted on.": "如果启用，所有与上述任一行或列匹配的元素都将被操作。否则，只有同时与上述任一行或列匹配的元素才会被操作。",
    "The parameter name is @COPEN@tableN@CCLOSE@, where @COPEN@N@CCLOSE@ refers to a specific table. e.g. @COPEN@table1 + table3 - table2@CCLOSE@, will add the first and third input tables and then subtract the second. Most operators and @COPEN@if@CCLOSE@ @COPEN@else@CCLOSE@ statements are supported. See Example #6 in the Help section for further examples.": "参数名为 @COPEN@tableN@CCLOSE@，其中 @COPEN@N@CCLOSE@ 指的是一个特定的表格。例如 @COPEN@table1 + table3 - table2@CCLOSE@，将会先添加第一个和第三个输入表格，然后再减去第二个。大多数运算符和 @COPEN@if@CCLOSE@ @COPEN@else@CCLOSE@ 语句都是支持的。更多例子请参见帮助部分的示例 #6。",
    "here you can enter text or regular expression (for syntax check lower part of this frame)": "在这里，您可以输入文本或正则表达式（用于语法检查，请参阅此框架的下半部分）",
    "The file needs to be sorted by this column, ascending.": "该文件需要根据这一列进行升序排序。",
    "ppm is useful for very small differences": "ppm 对于非常小的差异很有用",
    "Make sure your added table names are not already in this database": "确保你添加的表名在这个数据库中尚未存在",
    "The names will be quoted if they are not valid SQLite column names.": "如果名称不是有效的SQLite列名，它们将被引用。",
    "Only creates this additional column when a name is entered. (This can not be the same name as any of the other columns in this table.)": "仅在输入名称时创建此额外列。（此名称不能与此表中的其他任何列同名。）",
    "SQLite to tabular tool can run additional queries on this database": "SQLite 转表格工具可以在该数据库上运行额外的查询",
    "Must contain 2 tab-separated columns": "必须包含2个以制表符分隔的列",
    "Lines starting with this character will be directly printed to the output file": "以该字符开头的行将直接打印到输出文件中",
    "@tsv can be used normally, but this will automatically mark the output as tabular": "@tsv 可以正常使用，但这将自动将输出标记为表格",
    "When enabled revert the tool's action and drop all columns that are NOT selected above.": "当启用时，撤销工具的操作并删除所有未在上方选中的列。",
    "Removes characters from the left of the first column only.": "仅从第一列左侧移除字符。",
    "This column will be made left-most.": "此列将被放置在最左侧。",
    "(see figure below)": "（见下图）",
    "(bp)": "碱基对",
    "Use positive values to offset co-ordinates in the direction of transcription and negative values to offset in the opposite direction.": "使用正值来在转录方向上偏移坐标，使用负值来向相反方向偏移。",
    "Use non-negative value for length": "使用非负值作为长度",
    "Specifying columns for restricting subtraction is available only for tabular formatted datasets": "只有表格格式的数据集才能指定用于限制减法的列",
    "did the calumn have numeric values": "该列是否包含数值",
    "Depending on the type of input selected the concatenation options will differ": "根据所选输入类型的不同，连接选项会有所不同",
    "All inputed datasets will be concatenated tail-to-head.": "所有输入的数据集将被尾对头连接。",
    "The first collection contains the datasets that will be written first in the concatenated file": "第一个集合包含将首先写入合并文件中的数据集",
    "The second collection contains  the datasets that will be written last in the concatenated file": "第二个集合包含将在合并文件中最后写入的数据集",
    "The Nested collection which items you want to concatenate.": "要连接的嵌套集合中的项。",
    "If 'Yes' is selected '#name of dataset' will be added when concatenating.": "如果选择“是”，在连接时将添加“#数据集名称”。",
    "This paremeter exists so as to not concatenate comments or headers contained at the start of the files.": "这个参数存在是为了不连接文件开头包含的注释或标题。",
    "See explanation of filtering options below": "查看下面的过滤选项说明",
    "Regions should be specified as chr:start-end, e.g X:1000-2000": "区域应指定为chr:start-end，例如 X:1000-2000",
    "First VCF dataset": "第一个 VCF 数据集",
    "Second VCF dataset": "第二个 VCF 数据集",
    "-g option": "-g 选项",
    "-n option": "-n 选项",
    "default = 150 bp": "默认值 = 150 bp",
    "-r, --rate": "-r, --rate",
    "-p, --random-seed; use this to make results reproducible": "-p, --random-seed; 使用此选项可确保结果可重现",
    "Allows you to specify options that are not listed above": "允许您指定上面未列出的选项",
    "-s, --scale-by; this only works for Float info fields such as AF": "-s, --scale-by; 这仅适用于像 AF 这样的浮点信息字段",
    "use chr:start-end format": "使用染色体：起始-结束格式",
    "-v, --invert. Print entries that DO NOT intersect.": "-v, --invert. 打印不相交的条目。",
    "--only-variants option": "--仅变异选项",
    "--use-mnps option": "--使用-mnps选项",
    "--tag-parsed option": "--tag-parsed 选项",
    "--max-length option": "最大长度选项",
    "Note that in many cases, such as multisample VCFs, these won't be valid post-decomposition.  For biallelic loci in single-sample VCFs, they should be usable with caution. (--keep-info)": "请注意，在许多情况下，例如多样本VCFs，这些在分解后可能不是有效的。对于单样本VCFs中的双等位基因位点，它们应该谨慎使用。(--keep-info)",
    "Similar caution should be used for this as for --keep-info. (--keep-geno)": "对于这个选项，应该像对待--keep-info一样小心使用。(--keep-geno)",
    "The second dataset will be instersected with the first": "第二个数据集将与第一个数据集进行交集操作。",
    "This tools needs to access reference genomes sequence specified by this option.": "这个工具需要访问通过此选项指定的参考基因组序列。",
    "-i or -u": "-i 或 -u",
    "-v, --invert": "-v, --invert",
    "-w, --window-size": "-w, --窗口大小",
    "-l, --loci": "-l, --位点",
    "such as -m, -t, -V, -M, and -T": "例如 -m、-t、-V、-M 和 -T",
    "Default is to require exact match between variants.": "默认情况下，要求变体之间完全匹配。",
    "ID and FILTER are special fields that pull the ID and FILTER columns from the VCF": "ID 和 FILTER 是特殊字段，它们从 VCF 文件中提取 ID 和 FILTER 列。",
    "In most cases, we will have a single annotation entry for each entry (variant) in the query VCF, in which case the self op is the best choice. However, it is possible that there will be multiple annotations from a single annotation file--in this case, the op determines how the many values are reduced. In nearly all cases, if you are annotating with a VCF, use self": "在大多数情况下，我们将为查询 VCF 中的每个条目（变体）有一个单独的注释条目，在这种情况下，选择 self 操作是最好的选择。然而，有可能来自单个注释文件会有多个注释——在这种情况下，op 确定如何将多个值进行简化。在几乎所有情况下，如果你正在使用 VCF 进行注释，请使用 self。",
    "(1-based) column number from which to pull the value for the op": "（基于1的）用于提取操作值的列号",
    "Field in the BAM to pull value from": "从BAM文件中提取值的字段",
    "Op to perform on the field. Note: DP2 field always will result in DP2 op": "对字段执行的操作。注意：DP2字段始终会导致DP2操作",
    "--key option": "--关键选项",
    "Annotation tag": "注释标签",
    "Determine which output file(s) to keep": "确定要保留的输出文件",
    "skip alignments with mapping quality smaller than INT [1]": "跳过映射质量小于INT的比对[1]",
    "skip bases with base quality smaller than INT [13]": "跳过碱基质量小于INT的碱基[13]",
    "initial counter value for nucleotides from ref [0]": "参考序列中核苷酸的初始计数值为[0]",
    "minimum coverage required for update [2]": "更新所需的最小覆盖度[2]",
    "in percent": "在百分比中",
    "in reads (per strand)": "在读取中（每个链）",
    "If checked, output positions present in the first file but missing from the other files": "如果选中，则输出在第一个文件中存在但在其他文件中缺失的位置",
    "In repetitive sequences, the same indel can be called at different positions. Consider records this far apart as matching (be it a SNP or an indel).": "在重复序列中，相同的插入/缺失（indel）可以在不同的位置被识别。考虑将相距这么远的记录视为匹配的（无论是单核苷酸多态性SNP还是插入/缺失indel）。",
    "Exclude sites containing indels from genotype comparison": "排除含有插入/缺失位点的基因型比较",
    "Set to 0|0": "设置为0|0",
    "Set to major allele": "设置为主等位基因",
    "List of trios, one per line": "三元组列表，每行一个",
    "Filter by sequence length. Will dump only reads longer or equal to this value.": "根据序列长度过滤。仅输出长度大于或等于该值的序列。",
    "This option will only be used when there are multiple reads per spot (for example paired-end).": "这个选项只有在每个位点有多个读取（例如配对末端）时才会使用。",
    "Will not be used if --split-3 is selected.": "如果选择了--split-3，则不会使用。",
    "Compression will greatly reduce the amount of space occupied by downloaded data. Downstream applications such as a short-read mappers will accept compressed data as input. Consider this example: an uncoimpressed 400 Mb fastq datasets compresses to 100 Mb or 80 Mb by gzip or bzip2, respectively.": "压缩将大大减少下载数据所占用的空间。下游应用，如短读映射器，将接受压缩数据作为输入。考虑这个例子：一个未压缩的400Mb fastq数据集通过gzip或bzip2分别压缩到100Mb或80Mb。",
    "Minimum spot id to be dumped.": "要导出的最小斑点ID。",
    "Maximum spot id to be dumped.": "最大斑点ID将被转储。",
    "Split spots into individual reads.": "将斑点分割成独立的读取。",
    "For SRA of noisy long-reads put SEQUENCE": "对于嘈杂的长读序列读取档案（SRA），输入序列（SEQUENCE）。",
    "In vast majority of cases you want to download data in bam format. It is more compact and is accepted by all downstream tools.": "在绝大多数情况下，您希望以bam格式下载数据。它更加紧凑，并且被所有下游工具接受。",
    "Must start with SRR, e.g. SRR925743": "必须以SRR开头，例如SRR925743",
    "window_size should be less than the sequence length, and less than 8 for Karplus-Schulz method": "窗口大小应小于序列长度，并且对于Karplus-Schulz方法应小于8",
    "Select database source. cRAP acts as a database for common MS contaminants. UniProtKB is a cross species collection of functional protein databases": "选择数据库来源。cRAP 作为常见质谱污染的数据库。UniProtKB 是一个跨物种的功能性蛋白质数据库集合。",
    "select species for protein database": "选择蛋白质数据库的物种",
    "Specify a NCBI taxon id to override species selection": "指定一个NCBI分类ID以覆盖物种选择",
    "UniProtKB/TrEMBL (unreviewed)is a large, automatically annotated database- may contain redundant sequences, but there is a higher chance peptides will be identified. UniProtKB/Swiss-Prot (reviewed) is a smaller, manually annotated database- less of a chance peptides will be identified but less sequence redundancy": "UniProtKB/TrEMBL（未经审核）是一个大型的、自动注释的数据库，可能包含冗余序列，但肽段被鉴定的几率更高。UniProtKB/Swiss-Prot（经过审核）是一个较小的、手动注释的数据库，肽段被鉴定的几率较小，但序列冗余较少。",
    "several different forms of a given protein are incorporated into database": "将给定蛋白质的几种不同形式纳入数据库",
    "Newline/Comma separated list of IDs": "用换行符或逗号分隔的ID列表",
    "Region to subset accession. Start and end position separated by ':', '..', or '.'. Only for single accession": "区域以子集访问号。起始和结束位置由 ':', '..', 或 '.' 分隔。仅适用于单个访问号",
    "Enter one or more IDs (separated by commas) from http://www.uniprot.org/proteomes/": "输入一个或多个来自 http://www.uniprot.org/proteomes/ 的ID（用逗号分隔）",
    "Genus species format, eg : Canis lupus": "属种格式，例如：Canis lupus",
    "Any combination of gbif, bison, inat, ebird, antweb, ala, idigbio, obis, ecoengine and/or vertnet": "任何gbif、bison、inat、ebird、antweb、ala、idigbio、obis、ecoengine和/或vertnet的组合",
    "This is passed across all sources. If you have an empty output you should try with a records value smaller.": "这会在所有来源中传递。如果你的输出为空，你应该尝试使用一个更小的记录值。",
    "YYYY, YYYY/MM or YYYY/MM/DD": "年份，年份/月份或年份/月份/日期",
    "Columns must be in a specific order, see help documentation": "列必须按照特定顺序排列，请参阅帮助文档",
    "E.g. proc natl acad sci u s a": "例如，美国国家科学院院刊",
    "Used to match input results to NCBI's output": "用于将输入结果与NCBI的输出进行匹配",
    "One ID in each line.": "每行一个ID。",
    "Expand each sequence downstream of the sequence by this many basepairs. Only available when using genomic sequence type.": "将每个序列在其序列下游扩展这么多碱基对。仅在使用基因组序列类型时可用。",
    "Type of sequence. Defaults to genomic where applicable, i.e. not translations. cDNA refers to the spliced transcript sequence with UTR; CDS refers to the spliced transcript sequence without UTR": "序列类型。默认为基因组，即非翻译序列。cDNA指带有UTR的剪接转录序列；CDS指不带UTR的剪接转录序列。",
    "The format of a NH (New Hampshire) request": "NH（新罕布什尔州）请求的格式",
    "Enter a single MGI gene ID or gene symbol": "输入一个 MGI 基因 ID 或基因符号",
    "Enter a txt file with the Gene MGI ID or gene symbol": "输入一个包含基因MGI ID或基因符号的文本文件",
    "Enter a single MP/HP term ID or a list dividing each ID with a comma (without spaces)": "输入一个MP/HP术语ID，或者输入一个ID列表，每个ID之间用逗号分隔（没有空格）",
    "Enter a txt file with the MP/HP terms": "输入一个包含MP/HP术语的文本文件",
    "Enter a single MGI gene ID (or gene symbol) or a list dividing each ID with a comma (without spaces)": "请输入一个MGI基因ID（或基因符号），或者输入一个ID列表，每个ID之间用逗号分隔（无空格）。",
    "Enter a txt file with the genes MGI IDs or symbols": "输入一个包含MGI基因ID或基因符号的文本文件",
    "Enter an IMPReSS parameter ID or a list of IDs dividing each ID with a comma (without spaces)": "输入一个IMPReSS参数ID或一系列ID，每个ID之间用逗号分隔（无空格）",
    "Enter a txt file with an IMPReSS parameter ID or a list of IDs": "输入一个包含IMPReSS参数ID的文本文件，或包含多个ID的列表",
    "Choose if include in the output table also those genes that have no registred phenotypes. By default they are excluded.": "选择是否在输出表中包含那些没有注册表型特征的基因。默认情况下，它们是被排除的。",
    "Enter a IMPReSS pipeline ID": "输入一个 IMPReSS 流程 ID",
    "Enter a txt file with an IMPReSS pipeline ID": "输入一个带有IMPReSS流水线ID的txt文件",
    "Enter a top level phenotype category ID": "输入一个顶层表型类别 ID",
    "Enter a txt file with a top level phenotype category ID": "输入一个包含顶级表型类别ID的txt文件",
    "Max 1000 IDs": "最多1000个ID",
    "For example, if the object is a gene, its transcripts, translations and exons will be returned as well": "例如，如果对象是基因，其转录本、翻译产物和外显子也将被返回",
    "If not, the ENA data warehouse is used": "如果无法获取，将使用ENA数据仓库。",
    "Files output by ELink or ESearch are acceptable.  Query IDs in an ELink result are ignored.": "由ELink或ESearch生成的文件是可接受的。在ELink结果中的查询ID将被忽略。",
    "A Text file containing one ID per line": "一个文本文件，每行包含一个ID",
    "ESearch or ELink Result XML file": "ESearch或ELink结果XML文件",
    "ESearch or ELink Result JSON file": "ESearch 或 ELink 结果 JSON 文件",
    "IDs will be saved to a supplied or created history.": "ID 将被保存到提供或创建的历史记录中。",
    "A JSON file containing the WebEnv ID referencing the history on the NCBI history server to which to add the IDs": "一个包含 WebEnv ID 的 JSON 文件，该 ID 引用了 NCBI 历史服务器上要添加 ID 的历史记录",
    "An XML file containing the WebEnv ID referencing the history on the NCBI history server to which to add the IDs": "一个包含WebEnv ID的XML文件，引用了NCBI历史服务器上要添加ID的历史记录",
    "List databases or describe a database.": "列出数据库或描述一个数据库。",
    "Other than Taxon and Project. It can also be range of ids (e.g. ERA000010-ERA000020)": "除了分类单元和项目之外，它还可以是一系列编号（例如 ERA000010-ERA000020）。",
    "Identifier starting with 'EGAD'. For example: EGAD00001003338": "标识符以'EGAD'开头。例如：EGAD00001003338",
    "Identifier starting with 'EGAF'. For example: EGAF00001753735": "标识符以\"EGAF\"开头。例如：EGAF00001753735",
    "For example 'chr1', '1', or 'chrX'. If unspecified, all data is returned.": "例如 'chr1'、'1' 或 'chrX'。如果未指定，则返回所有数据。",
    "0-based, inclusive. Only used if a reference sequence name was specified": "基于0的，包括在内。只有在指定了参考序列名称时才使用。",
    "0-based, exclusive. Only used if a reference sequence name was specified": "0-based，排他性。仅在指定了参考序列名称时使用。",
    "A tabular file where one column contains the set of file IDs. This will output a collection. Please select files that are all the same format (e.g. all BAM or all VCF).": "一个表格文件，其中一列包含文件ID集合。这将输出一个集合。请选择所有相同格式的文件（例如全部为BAM或全部为VCF）。",
    "File Identifiers starting with 'EGAF'. For example: EGAF00001753735": "文件标识符以“EGAF”开头。例如：EGAF00001753735",
    "NCBI taxid, common or scientific name": "NCBI 分类号，常见名称或科学名称",
    "any rank, only available for WP accessions": "任何等级，仅适用于WP访问码",
    "WP accessions only": "WP 访问号仅限",
    "Comma separated": "逗号分隔",
    "Prokaryotic: Accessions starting with WP_. Data report has a different format and the rna, cds, 3/5' UTR and gene-product report are not suported.": "原核生物：以WP_开头的访问号。数据报告有不同的格式，而RNA、CDS、3/5' UTR和基因产物报告不被支持。",
    "By default FASTA files are provided zipped (fasta.gz) if this is checked the data will be decompressed": "默认情况下，FASTA文件是以压缩格式提供的（fasta.gz），如果选中此选项，数据将被解压缩",
    "Searchable fields are species and infraspecies, assembly name and submitter": "可搜索字段包括物种和亚种、组装名称以及提交者",
    "Sequence Read Archive (SRA) data, is the largest publicly available repository of high throughput sequencing data. The European Nucleotide Archive (ENA) provides a comprehensive record of nucleotide sequencing information.": "序列读取档案（SRA）数据是最大的公开可用的高通量测序数据存储库。欧洲核苷酸档案（ENA）提供了一个全面的核苷酸测序信息记录。",
    "Multiple keywords should be separated by spaces. Example: colorectal cancer": "多个关键词应该用空格分隔。示例：结直肠癌",
    "Enable this option if you are interested in genetic material recovered directly from environmental or clinical samples. Default: disabled": "如果对从环境或临床样本中直接恢复的遗传物质感兴趣，请启用此选项。默认：禁用",
    "The scientific name of a species that is set by binomial nomenclature entails two parts: generic name (or genus name) and specific name": "双名法所规定的一个物种的科学名称包含两个部分：属名（或通用名）和种名。",
    "Sequencing platform used for generating the reads": "用于生成读取的测序平台",
    "Paired-end reads improve the ability to identify the relative positions of various reads in the genome, making it much more effective than single-end reading in resolving structural rearrangements such as gene insertions, deletions, or inversions. Note: Only available for the Illumina platform": "成对末端测序提高了识别基因组中各种测序读段相对位置的能力，使其在解析结构重排（如基因插入、缺失或倒位）方面比单端测序有效得多。注意：仅适用于Illumina平台。",
    "Sequencing technique intended for the library": "用于文库的测序技术",
    "Method used to select and/or enrich the material being sequenced": "用于选择和/或富集待测序材料的方法",
    "Relevant study/experiment/sample/run accession number": "相关研究/实验/样本/运行访问号",
    "Size of the sample rounded to the nearest megabase": "样本大小四舍五入到最近的兆碱基",
    "Publication date of the run in the format dd-mm-yyyy. If a date range is desired, enter the start date, followed by end date, separated by a colon ':'. Example: 01-01-2010:31-12-2010": "运行的发布日期格式为 dd-mm-yyyy。如果需要指定日期范围，请先输入开始日期，然后输入结束日期，两者之间用冒号 ':' 分隔。示例：01-01-2010:31-12-2010",
    "Default: 2": "默认值：2",
    "Note: If the maximum number set is large, querying the SRA database will take significantly longer due to API limits": "注意：如果设置的最大数值较大，由于API限制，查询SRA数据库将需要更长的时间。",
    "Genus species format, eg : Scomber scombrus": "属种格式，例如：鲭鱼属鲭鱼",
    "If you want to enter multiple ones just separate them with a comma (example : 6902746, 6902755)": "如果你想输入多个，请用逗号隔开（示例：6902746，6902755）",
    "if you want to enter multiple ones just separate them with a comma (example : 3,12)": "如果你想输入多个，请用逗号隔开（示例：3,12）",
    "At this time, BGC parameters are only available in expert user mode.": "目前，BGC参数仅在专家用户模式下可用。",
    "NOTE: this feature is based on a beta parameter of the CLI that checks the remote ENA repository for entries using the alias. This remote detection can have False positives, i.e assumes that it's present in the repository but it's not": "注意：这个功能基于CLI的一个beta参数，该参数检查使用别名的远程ENA存储库中的条目。这种远程检测可能会出现误报，即假定它存在于存储库中，但实际上并不存在。",
    "Include the first line (the assumed header line) to all split files.": "在所有分割的文件中包含第一行（假定的标题行）。",
    "If before, the separator will appear at the start of each record; if after, at the end": "如果设置为before，则分隔符将出现在每个记录的开始位置；如果设置为after，则分隔符将出现在每个记录的结束位置。",
    "The number of lines after which each record ends": "每条记录结束后的行数",
    "Combine first line of each file as the header for the final dataset. Useful when same header line is found in all files.": "将每个文件的第一行合并为最终数据集的标题。当在所有文件中都找到相同的标题行时非常有用。",
    "The column that will be used to join the input datasets": "用于连接输入数据集的列",
    "If this is set to 0, a header line will be added containing column names as follows: the identifier column will be named #KEY and the other columns are named by the input dataset names/columns. If you have one or more header lines in your input, set this to the number of header lines.": "如果设置为0，将添加一个包含列名的标题行，如下所示：标识符列将命名为#KEY，其他列以输入数据集的名称/列命名。如果您的输入中有一个或多个标题行，请将其设置为标题行的数量。",
    "Disable if you want column headers to only be composed of the input file names, for example, if you want headers like file1 and not file1_column1, see Help section below. Default: Yes": "如果你想要列标题只由输入文件名组成，例如，你想要像file1这样的标题而不是file1_column1，禁用此选项。请参阅下面的帮助部分。默认值：是",
    "a placeholder for empty cells": "一个用于空单元格的占位符",
    "of the first dataset (see figure below)": "第一个数据集的（见下图）",
    "Selecting no tables will result in using all tables.": "选择不使用表格将导致使用所有表格。",
    "Other columns can be turned into qualifiers with arbitrary keys and values": "其他列可以转换为具有任意键和值的限定词",
    "thickStart and thickEnd define protein coding region, blocks define exon regions": "thickStart 和 thickEnd 定义蛋白质编码区域，blocks 定义外显子区域",
    "unspliced genomic sequence from chromStart to chromEnd (Extract Genomic DNA)": "未剪接的基因组序列，从chromStart到chromEnd（提取基因组DNA）",
    "For 20 column BED from Ensembl REST server": "来自Ensembl REST服务器的20列BED文件",
    "Split the protein into peptides according to enzyme digestion": "将蛋白质根据酶消化分裂成肽段",
    "By default, the input bed dataset metadata will be used.": "默认情况下，将使用输入的bed数据集元数据。",
    "Any Compomics application such as PeptideShaker, requires a source for non reference proteins of 'generic' e.g.: generic|pep1|peptide description": "任何Compomics应用程序，如PeptideShaker，都需要一个非参考蛋白质的来源，这些蛋白质是“通用”的，例如：generic|pep1|肽段描述",
    "defaults to the pipe character, Ensembl FASTA files usually use a space character": "默认使用管道字符，Ensembl FASTA文件通常使用空格字符",
    "Can be used to distinguish samples": "可以用来区分样本",
    "If unchecked, Second dataset will be forced into the format of First dataset": "如果未勾选，第二个数据集将被强制转换为第一个数据集的格式",
    "Output from 'Extract Genomic DNA' run on tophat junctions.bed": "来自 '提取基因组DNA' 在 tophat junctions.bed 上的运行输出",
    "By default, the database metadata will be used.": "默认情况下，将使用数据库元数据。",
    "For example:  pep:splice": "例如：pep:splice",
    "For example:  with the tag name 'depth' and bed score 12:   depth:12": "例如：带有标签名 'depth' 和 bed 分数 12： depth:12",
    "Do not reject translation is stop_codons are within base pairs of the BED start position for positive strand": "不拒绝翻译，如果终止密码子在正链的BED起始位置的碱基对之内",
    "Do not reject translation is stop_codons are within base pairs of the BED end position for negative strand": "不要在负链上，如果终止密码子位于BED末端位置的碱基对范围内，则拒绝翻译",
    "e.g. tophat junctions.bed run without GTF option or no-novel-junctions": "例如，使用没有GTF选项或no-novel-junctions选项运行tophat junctions.bed",
    "e.g. tophat junctions.bed run with GTF option and no-novel-junctions": "例如，使用GTF选项和no-novel-junctions参数运行tophat junctions.bed",
    "The number of base pairs to extend the start of the exon before the junction position": "在连接点位置之前，外显子起始端需要延伸的碱基对数",
    "The number of base pairs to extend the end of the exon after the junction position": "在连接位点之后，外显子末端延伸的碱基对数",
    "If set, discard transcripts having an intron larger": "如果设置，则丢弃具有较大内含子的转录本",
    "NOTE: Required for fasta outputs": "注意：fasta输出所需",
    "Number of colors to use, should be in powers of 2 (1, 2, 4, 8, 16, 32, 64, ...), otherwise space is wasted in the palette.": "要使用的颜色数量，应该是2的幂（1, 2, 4, 8, 16, 32, 64, ...），否则在调色板中会浪费空间。",
    "Leave empty to ignore": "留空以忽略",
    "Only protein- and gene-centric entities are currently supported.": "目前仅支持蛋白质和基因中心实体。",
    "Multi-select list - hold the appropriate key while clicking to select multiple columns": "多选列表 - 按住适当的键同时点击以选择多个列",
    "Applies only to the first column taken from the title string ('0' = keep the whole thing), useful when your sequence identifiers are all the same length.": "只应用于从标题字符串中取出的第一列（'0' = 保持整个内容），当您的序列标识符长度都相同时非常有用。",
    "If false, it stops the conversion if instrument properties are missing": "如果为假，它会在仪器属性缺失时停止转换",
    "UV/PDA etc": "紫外/可见光分光光度计等",
    "(MS1, MS2, etc) included in the output, should be a comma-separated list of integers ( 1,2,3 ) and/or intervals ( 1-3 ), open-end intervals ( 1- ) are allowed": "输出中包含的（MS1、MS2等）应该是一个由逗号分隔的整数列表（1,2,3）和/或区间（1-3），允许使用开区间（1-）。",
    "Select species to be included in the final alignment": "选择最终比对中要包含的物种",
    "Not usually applicable. See help below for more information.": "通常不适用。有关更多信息，请参见下面的帮助。",
    "The FASTA file must be on the same build (dbkey) as the LiftOver chain file": "FASTA文件必须与LiftOver链文件具有相同的构建（dbkey）",
    "If a coordinate can not be lift over, do you want to include it in the output (it is still being marked 'fail')": "如果一个坐标无法lift over，您是否希望将其包含在输出中（它仍然被标记为“失败”）",
    "BED format file must have at least 3 columns (chrom, start, end) and no more than 12 columns": "BED格式文件必须至少有3列（染色体，起始位置，结束位置），且列数不超过12列。",
    "When set to Yes, blocks are divided around gaps appearing in any species. This will prevent gaps occurring in the interior of the sequence for an aligning species from overwriting a nucleotide found for the same position in a lower-scoring block.": "当设置为“是”时，会在任何物种中出现的间隙周围划分区块。这将防止对齐物种序列内部的间隙覆盖在同一位置的低分块中找到的核苷酸。",
    "Average insert size of pair-end sequencing (bp)": "成对末端测序的平均插入大小（碱基对）",
    "Stanadard deviation of insert size": "插入片段大小的标准差",
    "A mapped pair is considered as 'proper pair' if both ends mapped to different strand and the distance between them is less then '-t' * stdev from the mean": "一个映射对被认为是“正确对”，如果两个端点映射到不同的链上，并且它们之间的距离小于“-t”乘以标准差的平均值。",
    "These can provide sequences and length for proteins if not already present in the mzIdentML input": "这些可以为蛋白质提供序列和长度，如果它们在mzIdentML输入中尚未存在。",
    "No selection implies no filtering": "没有选择意味着没有过滤",
    "mapping Protein IDs from a Protein Search fasta to a reference genome": "将蛋白质搜索fasta中的蛋白质ID映射到参考基因组上",
    "Such as a  PSM (Peptide Spectral Match) report from a Proteomics Search Application": "例如，来自蛋白质组学搜索应用的PSM（肽段谱匹配）报告",
    "Contains the peptide amino acid sequence. Defaults to first column": "包含肽段氨基酸序列，默认为第一列",
    "The name in this column must match the name column in the Translate BED input. Defaults to second column.": "此列中的名称必须与Translate BED输入中的名称列匹配。默认为第二列。",
    "and use the peptide as the display name for the entry": "并使用肽段作为条目的显示名称",
    "Input data is 0-based or 1-based": "输入数据是0基（从0开始计数）还是1基（从1开始计数）",
    "The midpoint is computed as (start + end) // 2": "中点计算为（开始 + 结束）// 2",
    "Expanding the region may result in crossing chromosome start and end coordinate boundaries.": "扩展区域可能会导致跨越染色体起始和结束坐标边界。",
    "Should be from the same movie.": "应该来自同一部电影。",
    "Instead of downsampled, 8-bit encoding.": "而不是下采样，8位编码。",
    "Currently this indicates that non-sequencing ZMWs should be included in the output scraps BAM file, if applicable.": "目前，这表示如果适用，非测序的零模波导孔（ZMWs）应该包含在输出的片段BAM文件中。",
    "As AGAT is a toolkit, it contains a lot of tools. If any of them is missing, please contact the server admin.": "作为AGAT工具箱，它包含许多工具。如果缺少任何一个，请与服务器管理员联系。",
    "Input GTF/GFF file": "输入 GTF/GFF 文件",
    "Define the feature you want to extract the sequence from.": "定义您想要从中提取序列的特征。",
    "This extract the mrna sequence (i.e transcribed sequence (devoid of introns, but containing untranslated exons)).": "这个提取mRNA序列（即转录序列（不含内含子，但包含未翻译的外显子））。",
    "This extract the cdna sequence (i.e reverse complement of the mRNA: transcribed sequence (devoid of introns, but containing untranslated exons, then reverse complemented).": "这提取了cDNA序列（即mRNA的反向互补序列：转录序列（不含内含子，但包含未翻译的外显子，然后进行反向互补）。",
    "This option allows removing the translation of the final stop codons that is represented by the '*' character. This character can be disturbing for many programs (e.g interproscan)": "这个选项允许移除由'*'字符表示的终止密码子的翻译。这个字符可能会干扰许多程序（例如 interproscan）",
    "The Clean Internal Stop option allows replacing the translation of the stop codons present among the sequence that is represented by the '*' character by . This character can be disturbing for many programs (e.g interproscan)": "“清除内部终止密码子”选项允许将序列中由'*'字符表示的终止密码子的翻译替换为。这个字符可能会干扰许多程序（例如interproscan）。",
    "It will take that number of nucleotide in more at the 5 extremity.": "它将在5'端多取一定数量的核苷酸。",
    "It will take that number of downstream nucleotides.": "它将采用该数目的下游核苷酸。",
    "This option allows dealing with feature that may span over several locations like CDS or exon, in order to extract the full sequence from the start extremity of the first chunck to the end extremity of the last chunk. The use of that option with '--type exon' will extract the pre-mRNA sequence (i.e with introns). Use of that option on CDS will give the pre-mRNA without the untraslated regions (UTRs).": "这个选项允许处理可能跨越多个位置的特征，如CDS或外显子，以便从第一个区块的起始端提取到最后一个区块的末端的完整序列。使用'--type exon'选项将提取前mRNA序列（即包含内含子）。在CDS上使用该选项将得到没有未翻译区域（UTRs）的前mRNA。",
    "The value of the attribute tags will be extracted from the feature type specified by the option --type and stored in the FASTA header.": "属性标签的值将从通过选项--type指定的特征类型中提取，并存储在FASTA标题中。",
    "By default, only features that span several locations (e.g. CDS and utr can span over several exons) are merged together. In order to merge other type of features (e.g. exon) you must activate this parameter.": "默认情况下，只有跨越多个位置的特征（例如，编码序列（CDS）和非翻译区（utr）可以跨越多个外显子）才会被合并在一起。为了合并其他类型的特征（例如，外显子），你必须激活这个参数。",
    "By default the extrated feature sequences from a minus strand is reverse complemented. Activating this option you will always get sequence from plus strand (not reverse complemented).": "默认情况下，从负链提取的特征序列会被反向互补。激活此选项，您将始终获得正链上的序列（不进行反向互补）。",
    "It will extract the sequence in amino acids.": "它将提取氨基酸序列。",
    "CDS can start with a phase different from 0 when a gene model is fragmented. When asking for protein translation this (start) offset is trimmed out automatically. But when you extract CDS dna sequences, this (start) offset is not removed by default. To remove it activate this option. If --upstream or --downstream option are used too, the (start) offset is trimmed first, then is added the piece of sequence asked for.": "当基因模型是片段化的，编码序列（CDS）可以从非0的相位开始。当请求蛋白质翻译时，这个（开始）偏移量会自动被剔除。但是，当你提取CDS DNA序列时，默认情况下这个（开始）偏移量不会被移除。要移除它，请激活这个选项。如果同时使用了--upstream或--downstream选项，那么首先会剔除（开始）偏移量，然后再加上所请求的序列片段。",
    "By default the extrated feature sequences from a minus strand is reverse complemented. Consequently, for minus strand features that option will extract the sequences from plus strand from left to right.": "默认情况下，从负链提取的特征序列会被反转补码。因此，对于负链特征，该选项将从左到右提取正链上的序列。",
    "By default, all features that span several locations (e.g. CDs and UTR can span over several exons) are merge together to shape the biological feature (e.g. several CDS chuncks are merged to create the CDS in its whole). If you wish to extract all the chuncks independently activate this option.": "默认情况下，所有跨越多个位置的特征（例如CDS和UTR可以跨越多个外显子）都会被合并在一起以形成生物学特征（例如，将多个CDS块合并以创建完整的CDS）。如果您希望独立提取所有块，请激活此选项。",
    "Reference GTF/GFF file": "参考 GTF/GFF 文件",
    "Annotation file you would like to use to complement the reference annotation.": "您想要用来补充参考注释的注释文件。",
    "Option to keep the non-overlping gene only if the CDS size (in nucleotide) is over the minimum size defined. Default = 0 that means all of them are kept.": "选项：仅保留不重叠的基因，如果编码序列（CDS）的大小（以核苷酸计）超过定义的最小大小时。默认值 = 0，这意味着保留所有基因。",
    "Gene names and products/descriptions will be fished from this file.": "基因名称和产物/描述将从这个文件中提取出来。",
    "You can decide until which protein existence level you want to consider to lift the finctional information": "您可以决定要考虑哪种蛋白质存在水平，以提升功能信息",
    "Optional, will override column selection with static text value": "可选，将用静态文本值覆盖列选择",
    "e.g. chrX or chr:start-end": "例如：chrX 或 chr:起始位置-结束位置",
    "Advanced options for gtfToGenePred.": "gtfToGenePred的高级选项。",
    "Only process entries where the source name has the specified prefixes": "仅处理源名称具有指定前缀的条目",
    "Ignore groups that do not have exons, otherwise they will cause an error.": "忽略没有外显子的组，否则它们会导致错误。",
    "Only check column validity, not heirarchy, may result in invalid output.": "仅检查列的有效性，而不是层级结构，可能会导致无效的输出。",
    "Skip groups with errors rather than aborting. Useful for getting information about as many errors as possible.": "跳过出错的组，而不是中止。这对于获取尽可能多的错误信息非常有用。",
    "Assume there is an implied stop codon after CDS.": "假设在编码序列（CDS）之后有一个隐含的终止密码子。",
    "If gene_version and/or transcript_version attributes exist, include the version in the corresponding identifiers.": "如果存在gene_version和/或transcript_version属性，则在相应的标识符中包含版本。",
    "Outputs a file with information about each transcript.": "输出一个包含每个转录本信息的文件。",
    "Optional": "可选的",
    "Zero-based, optional": "零基，可选",
    "Non-inclusive, optional": "非包容性的，可选的",
    "It must be in the format seqSpec[:start-end], e.g. chr1 or chr1:0-189 where coordinates are half-open zero-based, i.e. [start,end)": "它必须在格式 seqSpec[:start-end] 中，例如 chr1 或 chr1:0-189，其中坐标是半开零基的，即 [start,end)",
    "Required if filtering by insert size for single-end Reqd 1 (below).": "如果需要根据插入大小进行过滤，则对于单端数据是必需的。Reqd 1（下面）。",
    "Will not filter out single-end Read 1 unless proper mate-pairing is required (above).": "不会过滤掉单端的Read 1，除非需要正确的配对（如上所述）。",
    "Leave blank to extract all sequences": "留空以提取所有序列",
    "The style of chromosome IDs": "染色体ID的风格",
    "A xml file corresponding to a xsd schema you want to convert in another.": "一个与您想要转换为另一个的 xsd 架构相对应的 xml 文件。",
    "A xsl file describing the mapping between a first xsd specification to another.": "一个描述第一个XSD规范到另一个映射的XSL文件。",
    "Output a fasta.gz or fastq.gz": "输出一个fasta.gz或fastq.gz文件",
    "You can use both spaces and new lines to separate your identifiers.": "您可以使用空格和换行符来分隔您的标识符。",
    "Will remove suffixes including Illumina /1 and /2, Roche 454 .f and .r, and assorted Sanger names like .p* and .q*": "将移除包括Illumina的/1和/2、Roche 454的.f和.r，以及各种Sanger命名如.p*和.q*的后缀。",
    "Default: >ID will use search pattern >([^| ]+) to input ID; Use custom regex to change": "默认：>ID 将使用搜索模式 >([^| ]+) 来输入 ID；使用自定义正则表达式进行更改",
    "Search pattern must contain %s where the ID will be substituted. Use this for Uniprot Acc: >.+?\\|(.+?)\\|.*$": "搜索模式必须包含%s，其中ID将被替换。对于Uniprot Acc，请使用：>.+?\\|(.+?)\\|.*$",
    "Columns not explicitly specified above for rearrangement can be appended after the last specified column in their original order (the default) or be discarded from the output.": "未明确指定用于重新排列的列可以在其原始顺序中附加在最后指定的列之后（默认），或从输出中丢弃。",
    "At least 2, e.g. 5 will take every 5th sequence (taking 20% of the sequences)": "至少2，例如5将取每5个序列（取序列的20%）",
    "Between 0 and 100, e.g. 20% will take every 5th sequence": "在0到100之间，例如20%将取每5个序列",
    "Number of unique sequences to pick (between 1 and number itotal n input file)": "要挑选的唯一序列数量（介于1和输入文件中的总数之间）",
    "This mode keeps paired reads together (e.g. take every 5th read pair)": "这种模式保持配对的读取在一起（例如，每隔5个读取对取一对）",
    "e.g. a peptide fasta converted to tabular": "例如，将一个肽段的fasta文件转换为表格形式。",
    "e.g. a protein fasta converted to tabular": "例如，将蛋白质的FASTA格式转换为表格格式",
    "defaults to ;": "默认为分号；",
    "defaults to ,": "默认为，",
    "Requires an annotation file in GFF3 or GTF format": "需要一个GFF3或GTF格式的注释文件",
    "Consider only the reference transcripts that overlap any of the input transfrags": "只考虑与输入转录片段（transfrags）有重叠的参考转录本。",
    "Consider only the input transcripts that overlap any of the reference transcripts. Warning: this will discard all 'novel' loci!": "只考虑与任何参考转录本重叠的输入转录本。警告：这将丢弃所有“新”位点！",
    "Make the accuracy estimation at transcript level much more stringtent by only allowing a limited variation of the outer coordinates of the terminal exons. Transcript matching takes into account the -e range for terminal exons; code '=' is only assigned if transcript ends are within that range, otherwiscode '~' is assigned for intron chain match or single-exon": "在转录本水平上进行准确性估计时，通过仅允许末端外显子的外部坐标有限变化，使估计更加严格。转录本匹配考虑了末端外显子的 -e 范围；如果转录本末端在该范围内，则仅分配代码 '='，否则对于内含子链匹配或单外显子，分配代码 '~'。",
    "If -S and also --strict-match is given, exact matching of all exon boundaries is required": "如果同时给出 -S 和 --strict-match 参数，则需要精确匹配所有外显子边界",
    "Discard duplicate query transfrags (i.e. same intron chain) within a single sample (disable annotation mode for a single file); this option is automatically enabled when multiple query files are provided": "丢弃单个样本中重复的查询转录本片段（即相同的内含子链），在单个样本中禁用注释模式（为单个文件禁用）；当提供多个查询文件时，此选项会自动启用。",
    "When -D is enabled (or multiple query files are provided), perform a more strict duplicate checking: only discard matching (same intron chain) query transcripts from the same sample if their boundaries are fully contained within (or same with) matching transcripts if --strict-match is also given, exact match of all exons is required": "当启用-D（或提供了多个查询文件）时，执行更严格的重复检查：只有在其边界完全包含在（或与）匹配的转录本内（如果同时给出--strict-match，则相同）时，才从同一样本中丢弃匹配的（相同内含子链的）查询转录本，如果给出--strict-match，则需要所有外显子的精确匹配。",
    "Default: merge exons separated by 'introns' shorter than 5 bases": "默认：合并由短于5个碱基的“内含子”分隔的外显子",
    "Use sequence data for some optional classification functions, including the addition of the p_id attribute required by Cuffdiff.": "使用序列数据进行一些可选的分类功能，包括添加 Cuffdiff 所需的 p_id 属性。",
    "Optional. Repeats must be soft-masked (lower case) in order to be able to classify transfrags as repeats": "可选。为了能够将转录片段分类为重复序列，重复序列必须进行软掩蔽（小写）。",
    "Maximum distance (range) for grouping transcript start sites. Default: 100": "最大距离（范围）用于分组转录起始位点。默认值：100",
    "Show summary and accuracy data separately for each reference sequence in the transcript accuracy data set": "在转录准确度数据集中，为每个参考序列分别显示摘要和准确度数据。",
    "TMAP are tabular files that store the information regarding the best match for each prediction in the reference. RefMap files are tabular files which store the information regarding the best match for each reference transcript, among all possible prediction models. More information in the help section": "TMAP是表格文件，它们存储有关参考中每个预测的最佳匹配的信息。RefMap文件是表格文件，它们存储有关所有可能的预测模型中每个参考转录本的最佳匹配的信息。更多信息在帮助部分。",
    "I.e. collapse intron-redundant transfrags across all query files": "即在所有查询文件中折叠冗余的内含子片段。",
    "Like -C but does not discard intron-redundant transfrags if they start with a different 5' exon": "类似于-C选项，但如果内含子冗余的转录片段以不同的5'外显子开始，则不会丢弃它们。",
    "Like -C but also discard contained transfrags if transfrag ends stick out within the container's introns": "类似于-C选项，但如果转录本片段（transfrag）的末端突出在容器的内含子内部，也会丢弃这些被包含的转录本片段。",
    "For -C/-A/-X": "对于 -C/-A/-X",
    "If your genome of interest is not listed, contact the Galaxy server administrators": "如果您感兴趣的基因组没有列出，请与Galaxy服务器管理员联系",
    "This is not a hard limit - it only affects size of introns which are sought during heuristic alignment.": "这不是一个硬性限制 - 它只影响在启发式比对期间寻找的内含子的大小。",
    "Outputs from 'Funannotate predict annotation' tool": "来自“Funannotate 预测注释”工具的输出",
    "TSV format, if available": "TSV格式，如果可用",
    "Use this genome size to compute the unit length": "使用这个基因组大小来计算单位长度",
    "The format the unit counts data should be generated": "单位计数数据应该生成的格式",
    "Save only units that appears at least this many times": "仅保存至少出现这么多次的单元",
    "Set the maximum count value for units": "设置单元的最大计数值",
    "Unit length used to mask the genome": "用于遮蔽基因组的单元长度",
    "Score threshold for subwindows": "子窗口的得分阈值",
    "Score for units with unit count greater than T_high": "对于单位计数大于T_high的单元的得分",
    "Score for units with unit count less than T_low": "单位计数小于T_low的单元的得分",
    "Override the t_extend value in the unit counts file": "覆盖单位计数文件中的 t_extend 值",
    "Override the t_low value in the unit counts file": "覆盖单位计数文件中的 t_low 值",
    "Override the t_high value in the unit counts file": "覆盖单位计数文件中的 t_high 值",
    "Override the score threshold value in the unit counts file": "覆盖单位计数文件中的分数阈值",
    "The default is unit_size + 4": "默认值是 unit_size + 4",
    "Leave empty to start a new database from scratch": "保持为空以从头开始创建一个新的数据库",
    "Reference genome name": "参考基因组名称",
    "Regular expression to capture groups in mRNA name to use in 'Regex protein'": "正则表达式用于捕获mRNA名称中的组，以便在“正则蛋白”中使用",
    "Replacement string for the protein name using capturing groups defined in 'Regex protein capture'": "使用在'Regex protein capture'中定义的捕获组来替换蛋白质名称的字符串",
    "Attribute containing the protein uniquename. It is searched at the mRNA level, and if not found at CDS level.": "包含蛋白质唯一名称的属性。首先在mRNA水平上进行搜索，如果未找到，则在CDS水平上进行搜索。",
    "e.g. blastx, blastp": "例如：blastx，blastp",
    "e.g. BLOSUM90, BLOSUM80, PAM100": "例如 BLOSUM90、BLOSUM80、PAM100",
    "e.g. Non-reundant protein sequences (nr)": "例如，非冗余蛋白质序列（nr）",
    "One line per mRNA, first column is mRNA ID, each other columns is an experimental condition. TPM values preferably.": "每条mRNA一行，第一列是mRNA ID，其他列是实验条件。最好使用TPM值。",
    "Comma-separated list of column positions, which are part of the same replica group. Columns not mentionned in any replica group will be considered to be a single replica experiment.": "逗号分隔的列位置列表，这些列属于同一个重复组。在任何重复组中未提及的列将被视为单个重复实验。",
    "Can be an assembled genome or transcriptome (DNA), or protein sequences from an annotated gene set.": "可以是组装过的基因组或转录组（DNA），或者是来自注释基因集的蛋白质序列。",
    "Adds considerably to run time, but can improve results for some non-model organisms": "会增加运行时间，但可以改善一些非模式生物的结果",
    "Let BUSCO decide the best lineage automatically, or select from known lineage": "让BUSCO自动决定最好的谱系，或从已知的谱系中选择",
    "Taxonomic group to run with auto-lineage.": "使用自动谱系运行的分类群。",
    "Find the recommanded parameter at config/default.cfg": "在 config/default.cfg 中查找推荐的参数",
    "Default is to stop at N50 value": "默认是在N50值处停止",
    "The assembly should be soft-masked (with RepeatMasker for example)": "组装应该进行软掩蔽（例如使用RepeatMasker）",
    "Disable at your own risk if you want to ignore problems in the genome sequence reported by Funannotate": "禁用此选项有风险，如果您想忽略Funannotate报告的基因组序列中的问题，请自行承担风险。",
    "e.g. Genus species": "例如 属 种",
    "If relevant (e.g. Af293)": "如果相关（例如 Af293）",
    "If relevant (e.g. FGSCA4)": "如果相关（例如 FGSCA4）",
    "Will prefix all the gene names": "将为所有基因名称添加前缀",
    "BUSCO will be used to perform initial training of ab initio predictors (e.g. Augustus).": "BUSCO 将被用于对从头预测器（例如 Augustus）进行初始训练。",
    "Used when BUSCO runs Augustus internally.": "当BUSCO内部运行Augustus时使用。",
    "'overlap' drops gene models that are more than 90% contained within a repeat region; 'blast' compares the amino acid sequences to a small database of known transposons": "“重叠”会丢弃那些超过90%包含在重复区域内的基因模型；“blast”会将氨基酸序列与已知转座子的小型数据库进行比较。",
    "If specified, will over-rule any other training presets based on sepcies selection.": "如果指定，将基于物种选择覆盖任何其他训练预设。",
    "Select a species from the list": "从列表中选择一个物种",
    "GeneMark is not a free software, to use it download and unzip a license from http://topaz.gatech.edu/GeneMark/license_download.cgi (ES/ET/EP version). GeneMark needs to be installed manually by Galaxy administrators, it might not be available on this server.": "GeneMark 不是免费软件，要使用它，需要从 http://topaz.gatech.edu/GeneMark/license_download.cgi 下载并解压缩一个许可证（ES/ET/EP 版本）。GeneMark 需要由 Galaxy 管理员手动安装，可能在这个服务器上不可用。",
    "GeneMark will skip prediction on repeat regions shorter than this value": "GeneMark 将跳过短于这个值的重复区域进行预测",
    "Parse results directly to EVM": "将结果直接解析到EVM",
    "Not recommended for fungal genomes that have high gene density. You might want to turn this option on for larger genomes or those that have a high repeat content.": "不建议用于具有高基因密度的真菌基因组。您可能希望为较大的基因组或具有高重复内容的基因组打开此选项。",
    "Splits big contigs in smaller overlaping chunks to reduce memory usage and parallelize": "将大的连续序列分割成较小的重叠片段，以减少内存使用量并实现并行化处理。",
    "e.g. augustus:2 pasa:10": "例如：augustus:2 pasa:10",
    "The NCBI max FASTA header length is 16. Increase if you don't submit to NCBI.": "NCBI 最大 FASTA 头长度是 16。如果你不向 NCBI 提交，可以增加。",
    "The assembly should preferably be soft-masked (with RepeatMasker for example)": "组装最好使用软掩蔽（例如使用RepeatMasker）",
    "Using Sp_1, if no species is assigned": "使用 Sp_1，如果没有指定物种",
    "GeneMark-EX option, run algorithm with branch point model (most useful for fungal genomes)": "GeneMark-EX选项，运行具有分支点模型的算法（对真菌基因组最有用）",
    "Alternate training method (Conditional Random Field)": "交替训练方法（条件随机场）",
    "keep CRF parameters even if they are no better than HMM parameters": "保持条件随机场（CRF）参数，即使它们并不比隐马尔可夫模型（HMM）参数更好。",
    "using evaluation scripts from GaTech": "使用来自GaTech的评估脚本",
    "Only affects UTR training example generation, not gene prediction by AUGUSTUS (default: GTAG": "只影响UTR训练示例的生成，不影响AUGUSTUS的基因预测（默认值：GTAG）",
    "If not set, the value is automatically determined by GeneMark-EX": "如果未设置，则由GeneMark-EX自动确定该值",
    "for downsampling of training gene structures according to their number of introns distribution": "用于根据内含子数量分布对训练基因结构进行下采样",
    "The beta value for for homozygosity detection (Leave blank for default: 0.009)": "同质性检测的beta值（留空使用默认值：0.009）",
    "The number of enumerations (Leave blank for default: 1)": "枚举的数量（留空默认为：1）",
    "FASTA format": "FASTA格式",
    "Equivalent to --addgenes --mincontiglen 200 --centre Prokka (or other centre specified below) (--compliant)": "等同于--addgenes --mincontiglen 200 --centre Prokka（或下面指定的其他中心）(--compliant)",
    "NCBI needs 200": "NCBI需要200",
    "May be used to aid annotation, see --usegenus below": "可以用来辅助注释，参见下面的--usegenus",
    "Will use the BLAST database for the genus specified above, if installed": "将使用上面指定的属的BLAST数据库，如果已安装",
    "Will set --meta option for Prodigal": "将为Prodigal设置--meta选项",
    "Skip CDS /product searching": "跳过CDS/产品搜索",
    "Select between individual gff files or a collection of gff files": "选择单个gff文件或gff文件集合",
    "Select the files you wish to send to Roary, must be in gff3 format with the sequence data at the end of the file.": "选择您希望发送到Roary的文件，这些文件必须以gff3格式存在，并且文件末尾要有序列数据。",
    "A dataset list collection of gff3 files to send to Roary for analysis.": "一个包含gff3文件的列表集合，用于发送给Roary进行分析。",
    "Sets the minimum percentage identity for protein matches": "设置蛋白质匹配的最小百分比相似性",
    "The total percentage of the isolates that must have the gene for it to be considered a core gene.": "必须拥有该基因的分离菌株的总百分比，才能将其视为核心基因。",
    "Summary statistics, core gene alignments and Gene Presence/Absence tables are always produced.": "汇总统计、核心基因序列比对和基因存在/缺失表总是被生成。",
    "The maximum number of clusters to assign proteins": "要分配蛋白质的最大聚类数",
    "Click yes so that paralogs aren't split.": "点击“是”，以便同源基因不会被拆分。",
    "DNA -> RNA translation table to use. Roary is designed for prokaryotes and so tables 1, 4 or 11 will work, others could be problematic.": "DNA转录成RNA时使用的密码子表。Roary是为原核生物设计的，因此使用1号、4号或11号密码子表是可行的，其他密码子表可能会出现问题。",
    "This is an advanced setting, change only if you know what you are doing and then at your own risk!": "这是一个高级设置，只有在你清楚自己在做什么并且愿意承担风险的情况下才进行更改！",
    "First read file (default: None)": "第一个读取文件（默认值：无）",
    "Second read file (default: None)": "第二个读取文件（默认：无）",
    "Warning!!!: The BAM files must have been created using the ensembl version of the genome.": "警告！！！：BAM文件必须使用基因组的Ensembl版本创建。",
    "Mapping tools to use (default: bwa)": "使用映射工具（默认：bwa）",
    "Variant calling tool to use": "使用变体检测工具",
    "Minimum depth required to call variant. Bases with depth below this cutoff will be marked as missing (default: 10)": "调用变异所需的最小深度。低于此截止值的碱基将被标记为缺失（默认值：10）",
    " Minimum allele frequency to call variants (default: 0.1)": "调用变异的最小等位基因频率（默认：0.1）",
    "Please note this file should have a length less than 100K.": "请注意，这个文件的长度应该小于100K。",
    "Nucleotide sequences are not accepted.": "不接受核苷酸序列。",
    "Compressed archives are available from KofamKOALA web service (https://www.genome.jp/tools/kofamkoala/).": "压缩的档案可以从 KofamKOALA 网络服务（https://www.genome.jp/tools/kofamkoala/）获得。",
    "The score thresholds will be multiplied by this value.": "得分阈值将乘以这个值。",
    "Only shown in history if selected here and generated by the specific run.": "只有在此处选中并且由特定运行生成时，才会在历史记录中显示。",
    "A tabular file with the first column containing one of the supported types of identifier, see Help below.": "一个表格文件，第一列包含支持的标识符类型之一，请参见下面的帮助。",
    "If this option is set to Yes, the tool will assume that the input file has a column header in the first row and the identifers commence on the second line. Default: No": "如果将此选项设置为Yes，工具将假定输入文件的第一行包含列标题，并且标识符从第二行开始。默认值：No",
    "Select the organism the identifiers are from": "选择这些标识符所来自的生物体",
    "Select the type of IDs in your input file": "选择输入文件中ID的类型",
    "Choose the columns you want in the output table. Note that selecting some columns such as GO or KEGG could make the table very large as some genes may be associated with many terms. Default: ENSEMBL, ENTREZID, SYMBOL, GENENAME": "选择您希望在输出表中出现的列。请注意，选择某些列，如GO或KEGG，可能会使表变得非常大，因为某些基因可能与许多术语相关联。默认值：ENSEMBL，ENTREZID，SYMBOL，GENENAME",
    "If this option is set to Yes, only the first occurrence of each input Gene ID will be kept. Default: No": "如果将此选项设置为“是”，则仅保留每个输入基因ID的第一个出现。默认值：否。",
    "If this option is set to Yes, the Rscript used to annotate the IDs will be provided as a text file in the output. Default: No": "如果将此选项设置为Yes，用于注释ID的Rscript将作为文本文件提供在输出中。默认值：No",
    "FASTA file with contig(s)": "带有连续序列的FASTA文件",
    "MASH sketch of the reference plasmids database": "MASH 草图的参考质粒数据库",
    "Companion MASH database of reference database": "参考数据库的配套 MASH 数据库",
    "MOB-cluster plasmid cluster formatted metadata file matched to the reference plasmid db": "MOB-聚类 质粒聚类格式化元数据文件与参考质粒数据库匹配",
    "FASTA of plasmid replicons": "质粒复制子的FASTA文件",
    "FASTA or FASTQ file(s)": "FASTA 或 FASTQ 文件（们）",
    "Optionally provide custom MASH genome sketch to help with species identification (otherwise default RefSeq sketch is used)": "可选地提供自定义的MASH基因组草图以帮助物种鉴定（否则使用默认的RefSeq草图）",
    "Optionally provide custom database of alleles in JSON format": "可选地提供自定义的等位基因数据库，格式为JSON",
    "a single sequence in fasta formated sequence": "一个以fasta格式编写的单一序列",
    "contact the administrator of this Galaxy instance if you miss reference data": "如果缺少参考数据，请联系这个Galaxy实例的管理员",
    "Feature types that should be predicted by MITOS (--noprot,--notrna,--norrna)": "应该由MITOS预测的特征类型（--noprot，--notrna，--norrna）",
    "Maximum number of nucleotides by which genes of different types may overlap": "不同类型基因可能重叠的最大核苷酸数",
    "Negation of the exponent of the E-value threshold used by BLAST, i.e. a value X gives an E-value of 10^(-X)": "BLAST使用的E值阈值的指数的否定，即一个值X给出一个10^(-X)的E值",
    "Minimum allowed quality in % of the maximum quality value per reading frame": "每个读取框架中允许的最小质量百分比",
    "Maximum allowed overlap of proteins in percent of the smaller feature": "蛋白质在较小特征的百分比中允许的最大重叠",
    "Clip overlapping proteins with the same name that differ by less than the specified factor": "剪切重叠的蛋白质，如果它们的名称相同并且差异小于指定的因子。",
    "Maximum allowed overlap of proteins in the query (in percent of the shorter query range) for two hits to be counted as fragments of the same gene": "查询中蛋白质的最大允许重叠（以较短查询范围的百分比表示），以便将两个命中计为同一基因的片段",
    "Maximum factor by which fragments of the same protein may differ in their quality": "同一蛋白质的片段在其质量上可能存在的最大差异因子",
    "Number of aminoacids searched for start and stop codon of proteins": "搜索蛋白质起始和终止密码子的氨基酸数量",
    "Specify GFF3 file to extract features from": "指定GFF3文件以提取特征",
    "Source of DNA": "DNA来源",
    "Annotate clusters using TIGRFam profiles. TIGRFAMs is a collection of manually curated protein families focusing primarily on prokaryotic sequences": "使用TIGRFam剖面标注聚类。TIGRFAM是一组手动策划的蛋白质家族，主要关注原核生物序列。",
    "Improved prediction of gene cluster borders for fungal BGCs (CASSIS)": "改进的真菌生物合成基因簇边界预测（CASSIS）",
    "Each gene product encoded in the detected BGCs is analyzed against the PFAM database. Hits are annotated in the final Genbank/EMBL files. Also, selecting this option normally increases the runtime": "在检测到的生物合成基因簇（BGCs）中，每个基因产物都会与PFAM数据库进行比对。命中结果会在最终的Genbank/EMBL文件中进行注释。此外，选择此选项通常会增加运行时间。",
    "Run a cluster-limited HMMer analysis": "运行一个集群限制的HMMer分析",
    "Active sites of several highly conserved biosynthetic enzymes are detected and variations of the active sites are reported": "检测到几个高度保守的生物合成酶的活性位点，并报告了活性位点的变异。",
    "Run a comparison against the MIBiG database": "运行与MIBiG数据库的比较",
    "Compare identified clusters against a database of antiSMASH-predicted clusters.": "将识别出的聚类与antiSMASH预测的聚类数据库进行比较。",
    "Compare identified clusters against known gene clusters from the MIBiG database. MIBiG is a hand curated data collection of biosynthetic gene clusters, which have been experimentally characterized": "将识别出的基因簇与MIBiG数据库中已知的基因簇进行比较。MIBiG是一个手工整理的生物合成基因簇数据集，这些基因簇已经过实验性特征化。",
    "The identified clusters are searched against a database containing operons involved in the biosynthesis of common secondary metabolite building blocks (e.g. the biosynthesis of non-proteinogenic amino acids)": "已识别的聚类针对包含参与生物合成常见次级代谢产物构建块（例如非蛋白质氨基酸的生物合成）的操纵子的数据库进行搜索。",
    "Run RREFinder precision mode on all RiPP gene clusters. Many ribosomally synthesized and posttranslationally modified peptide classes (RiPPs) are reliant on a domain called the RiPP recognition element (RRE). The RRE binds specifically to a precursor peptide and directs the posttranslational modification enzymes to their substrates": "在所有RiPP基因簇上运行RREFinder精确模式。许多核糖体合成并经过翻译后修饰的肽类（RiPPs）依赖于一个称为RiPP识别元件（RRE）的域。RRE专门与前体肽结合，并将翻译后修饰酶引导到它们的底物上。",
    "It attempts to allocate each gene in the detected gene clusters to a secondary metabolism-specific gene family using profile hidden Markov models specific for the conserved sequence region characteristic of this family. In other words, each gene of the cluster is compared to a database of clusters of orthologous groups of proteins involved in secondary metabolism": "它尝试使用特定于该家族的保守序列区域的特征的配置隐藏马尔可夫模型，将检测到的基因簇中的每个基因分配给次级代谢特定的基因家族。换句话说，每个基因簇的基因都与参与次级代谢的同源蛋白组的簇数据库进行比较。",
    "High-GC containing bacterial sequences contain the rare Leu-codon “TTA” as a mean for post-transcriptional regulation by limiting/controlling the amount of TTA-tNRA in the cell. This type of regulation is commonly found in secondary metabolite BGCs. This feature will annotate such TTA codons in the identified BGCs. Default: 0.65": "高GC含量的细菌序列包含稀有的亮氨酸密码子“TTA”，作为通过限制/控制细胞中TTA-tNRA的数量进行转录后调控的手段。这种类型的调控通常在次级代谢产物生物合成基因簇（BGCs）中发现。这个特性将在已识别的BGCs中注释这些TTA密码子。默认值：0.65",
    "Only process sequences larger than this value. Default: 1000": "仅处理大于该值的序列。默认值：1000",
    "Defines which level of strictness to use for HMM-based cluster detection. Default: relaxed": "定义了用于基于HMM的聚类检测的严格程度。默认值：宽松",
    "Default: 10": "默认值：10",
    "A minimum scaling factor for the query BGC in ClusterBlast results. Default: 0": "在ClusterBlast结果中，查询细菌基因组聚类（BGC）的最小缩放因子。默认值：0",
    "Bitscore cutoff for RRE pHMM detection. Default: 25.0": "RRE pHMM检测的比特分数截断值。默认：25.0",
    "Minimum amino acid length of RRE domains. Default: 50": "RRE结构域的最小氨基酸长度。默认值：50",
    "Sideloading allows you to load external datasets into AntiSMASH": "侧载允许您将外部数据集加载到AntiSMASH中",
    "Sideload json file to add to your report": "将 JSON 文件侧载到你的报告中",
    "A peaks file in BED format.": "一个以BED格式的峰值文件。",
    "If this option is set to Yes, the tool will assume that the peak file has column headers in the first row. Default: No": "如果将此选项设置为“是”，则工具将假定峰值文件在第一行具有列标题。默认值：否。",
    "Select a GTF to use for annotation source.": "选择一个GTF文件作为注释来源。",
    "If the GTF file for your transcriptome of interest is not listed, contact your Galaxy administrator": "如果您感兴趣的转录组的GTF文件没有列出，请联系您的Galaxy管理员",
    "Default: Yes": "默认：是",
    "Output all the data used by R to construct the tables and plots, can be loaded into R. Default: No": "输出R用于构建表格和图形的所有数据，可以加载到R中。默认值：否。",
    "User can define TSS (transcription start site) region, by default TSS is defined from -3kb to +3kb.": "用户可以定义转录起始位点（TSS）区域，默认情况下，TSS 定义为从 -3kb 到 +3kb。",
    "If specified all genes within the flanking gene distance are reported for each peak. Default: No.": "如果指定，则报告每个峰值内相邻基因距离内的所有基因。默认值：否。",
    "If flanking gene info is turned on the flanking distance can be specified. Default: 5000.": "如果开启了旁侧基因信息，则可以指定旁侧距离。默认值：5000。",
    "If Yes only annotate gene at the 3’ of the peak. Default: No.": "如果选择“是”，则只在峰值的3'端注释基因。默认值：否。",
    "If Yes only annotate gene at the 5’ of the peak. Default: No.": "如果选择“是”，则仅注释峰值5'端的基因。默认值：否。",
    "in fasta format": "在FASTA格式中",
    "Read an existing sequence alignment in XMFA format and apply backbone statistics to it (--apply-backbone)": "读取一个已存在的XMFA格式的序列比对，并对它应用骨架统计学（--apply-backbone）",
    "Alignment gaps above this size in nucleotides are considered to be islands": "在核苷酸中，超过这个大小的对齐间隙被认为是孤立的",
    "Disable backbone detection": "禁用骨架检测",
    "Write out the guide tree used for alignment to a file": "将用于比对的指导树写入文件",
    "Write out the backbone to a file": "将骨架写入文件",
    "Find MUMs only, do not attempt to determine locally collinear blocks (LCBs)": "只寻找最小唯一匹配（MUMs），不要尝试确定局部共线性块（LCBs）",
    "Use the specified seed weight for calculating initial anchors": "使用指定的种子重量来计算初始锚点",
    "Use specified match file instead of searching for matches": "使用指定的匹配文件，而不是搜索匹配项",
    "Maximum number of base pairs to attempt aligning with the gapped aligner": "与带间隙对齐器尝试对齐的最大碱基对数",
    "A phylogenetic guide tree in Newick format that describes the order in which sequences will be aligned": "一个描述序列将如何进行比对的新克格式的系统发育指导树",
    "Assume that input sequences are collinear--they have no rearrangements": "假设输入序列是共线性的——它们没有重排",
    "Selects the anchoring score function": "选择锚定分数函数",
    "Don't scale LCB weights by conservation distance and breakpoint distance": "不要根据保守距离和断点距离缩放LCB权重",
    "Set the maximum weight scaling by breakpoint distance": "设置最大权重缩放，以断点距离为准",
    "Scale conservation distances by this amount": "将尺度保守距离除以这个数值",
    "Do not perform iterative refinement": "不要执行迭代细化",
    "Do not perform gapped alignment": "不要进行带间隔的对齐",
    "Minimum LCB score for estimating pairwise breakpoint distance": "估计成对断点距离的最小 LCB 分数",
    "Sets whether the repeat scores go negative or go to zero for highly repetitive sequences": "设置重复分数是否为负值，或者对于高度重复的序列是否变为零",
    "Gap extend penalty": "间隙延伸惩罚",
    "Minimum pairwise LCB score": "最小配对LCB分数",
    "Minimum breakpoint penalty after scaling the penalty by expected divergence": "最小断点惩罚，在通过预期的发散对惩罚进行缩放之后",
    "Probability of transitioning from the unrelated to the homologous state": "从无关状态转变为同源状态的概率",
    "Probability of transitioning from the homologous to the unrelated state": "从同源状态转换到无关状态的概率",
    "Expected level of sequence identity among pairs of sequences": "序列对之间预期的序列同一性水平",
    "Use a family of spaced seeds to improve sensitivity": "使用一系列间隔的种子来提高灵敏度",
    "Use solid seeds. Do not permit substitutions in anchor matches": "使用实心种子。在锚定匹配中不允许替换",
    "Use coding pattern seeds. Useful to generate matches coding regions with 3rd codon position degeneracy": "使用编码模式种子。有助于生成与第三个密码子位置简并性相匹配的编码区域",
    "Disable recursive anchor search": "禁用递归锚点搜索",
    "Should be provided in the same order as in original progressiveMauve run": "应与原始progressiveMauve运行中的顺序相同",
    "maximum length of a match_part feature. Lower number = larger GFF output": "匹配部分特征的最大长度。数字越小 = 更大的GFF输出",
    "progressiveMauve (un)helpfully strips original sequence IDs and replaces with integer numbers. This option lets you generate the alignment relative to e.g. the second genome if you wished to display against that base sequence": "progressiveMauve（不）有帮助地剥离原始序列ID，并用整数替换。这个选项允许您相对于例如第二个基因组生成比对，如果您希望针对该基础序列进行显示。",
    "Minimum DNA %identity of full allelle to consider 'similar' (default 95, must be between 0-100)": "最小DNA%身份的完整等位基因以考虑“相似”（默认为95，必须在0-100之间）",
    "Minimum DNA %coverage to report partial allele at all (default 10, must be between 0-100)": "最小DNA%覆盖率以报告所有部分等位基因（默认为10，必须在0-100之间）",
    "Minumum score out of 100 to match a scheme": "匹配方案的最低分（满分100分）",
    "Ignore these schemes from auto detection (comma sep. list) (default 'ecoli_2,abaumannii')": "忽略这些自动检测方案（逗号分隔列表）（默认'ecoli_2,abaumannii'）",
    "Select a MLST scheme": "选择一个多位点序列分型方案",
    "Enter a MLST scheme to use (see output of 'MLST List' tool for a list of available schemes)": "输入一个 MLST 方案以使用（查看 'MLST List' 工具的输出以获取可用方案的列表）",
    "Selecting 'Yes' will provide the alleles for all MLST schemes": "选择\"Yes\"将为所有MLST方案提供等位基因",
    "E.g. chromosome, region, contig": "例如染色体、区域、连续序列",
    "A single sequence in fasta formated sequence": "一个以fasta格式表示的单序列",
    "Contact the administrator of this Galaxy instance if you miss reference data": "如果您缺少参考数据，请联系此Galaxy实例的管理员",
    "Select between paired and paired-collection data": "选择配对数据和配对集合数据",
    "trim this many bases from the low-quality end of each read": "从每个读取的低质量端修剪这么多碱基",
    "Specify any number of query sequences in fasta format as input.": "指定任意数量的以fasta格式输入的查询序列。",
    "Specify any number of reference sequences in fasta format as input.": "指定任意数量的fasta格式的参考序列作为输入。",
    "Helps remove possible genomic position bias.": "有助于去除可能的基因组位置偏差。",
    "A three-column tabular file with SNP profile definitions": "一个包含SNP（单核苷酸多态性）特征定义的三列表格文件",
    "A two-column tabular file with structural variant profile definitions": "一个具有结构变异档案定义的两列表格文件",
    "Length of flanking region to extract.": "要提取的侧翼区域的长度。",
    "Minimum read number for including a specific flanking sequence.": "包含特定侧翼序列所需的最小读数。",
    "Minimum read number for each strand for including a specific flanking sequence.": "每个链包含特定侧翼序列的最小读取数。",
    "Minimum length of mapping for a read to be used in determining flanking sequences. Higher values are more robust to spurious mapping. Lower values will recover more reads.": "用于确定侧翼序列的读取的映射最小长度。更高的值对错误映射更为稳健。较低的值将恢复更多读取。",
    "Minimum quality value across extracted flanking sequence.": "提取的侧翼序列中最小质量值。",
    "Otherwise, intron-spanning stop codons could occur.": "否则，内含子跨越的终止密码子可能会出现。",
    "This allows overlapping genes on opposite strands.": "这允许在相反链上存在重叠的基因。",
    "This currently works only for human, galdieria, toxoplasma and caenorhabditis.": "目前这只能用于人类、Galdieria、弓形虫和秀丽隐杆线虫。",
    "Archive created with the 'Train Augustus' tool": "使用“Train Augustus”工具创建的档案",
    "Choose a specialised trainingset.": "选择一个专门的训练集。",
    "If this option is enabled, lowercase letters are considered as repeated regions.": "如果启用此选项，小写字母被视为重复区域。",
    "Gene Model to predict, for more information please refer to the help.": "基因模型预测，更多信息请参考帮助。",
    "A file containing hints in gff format (--hintsfile)": "一个包含gff格式提示的文件（--hintsfile）",
    "A .cfg file listing hint sources and their boni and mali (--extrinsicCfgFile)": "一个列出提示来源及其优点和缺点的 .cfg 文件（--extrinsicCfgFile）",
    "The beginning of the search range (--predictionStart)": "搜索范围的起始点（--predictionStart）",
    "The end of the search range (--predictionEnd); must be greater than starting position": "搜索范围的结束位置（--predictionEnd）必须大于起始位置",
    "Standard output is GTF.": "标准输出是GTF。",
    "Select 'paired end' reads or 'sequence' for genomes/contigs": "选择“配对末端”读取或“序列”用于基因组/基因组片段",
    "The file of forward reads in FASTQ format": "正向读取文件在FASTQ格式中",
    "The file of reverse reads in FASTQ format": "反向读取文件在FASTQ格式中",
    "Useful if you plan to redo the analysis using a different model and want to skip the preprocessing steps": "如果您计划使用不同的模型重新进行分析，并希望跳过预处理步骤，这将非常有用。",
    "When input gene lists are ranked this option may be used to get GSEA style p-values.": "当输入基因列表是按排名排序时，可以使用此选项来获取GSEA风格的p值。",
    "Whether all or only statistically significant results should be returned.": "是否应返回所有结果，还是只返回统计上显著的结果。",
    "Include evidence codes to the results. Note that this can decrease performance and make the query slower. In addition, a column ’intersection’ is created that contains the gene id-s that intersect between the query and term.": "在结果中包含证据代码。请注意，这可能会降低性能并使查询变慢。此外，会创建一个名为“交集”的列，其中包含查询和术语之间相交的基因ID。",
    "Custom p-value threshold, results with a larger p-value are excluded.": "自定义 P 值阈值，排除 P 值较大的结果。",
    "The algorithm used for multiple testing correction.": "用于多重检验校正的算法。",
    "Effective genomic landscape for statistical testing.": "有效的基因组景观用于统计测试。",
    "The prefix for fully numeric IDs. Check the “Resources” section below for the complete list of supported namespaces. E.g., ENTREZGENE_ACC, WIKIGENE_ACC.": "用于完全数字ID的前缀。请查看下面的“资源”部分以获取支持的命名空间的完整列表。例如，ENTREZGENE_ACC，WIKIGENE_ACC。",
    "Exclude electronic GO annotations (IEA evidence code).": "排除电子基因本体（GO）注释（IEA证据代码）。",
    "An interactive Plotly graph will be created alongside the tabular output. The plot is very similar to the one shown in the g:GOSt web tool.": "一个交互式的Plotly图表将与表格输出一起创建。该图表与g:GOSt网络工具中显示的图表非常相似。",
    "The prefix for fully numeric IDs. Check the “Resources” section below for the complete list of supported namespaces (including numeric). E.g., ENTREZGENE_ACC, WIKIGENE_ACC.": "完全数字ID的前缀。请查看下面的“资源”部分，以获取支持的命名空间的完整列表（包括数字）。例如，ENTREZGENE_ACC，WIKIGENE_ACC。",
    "Shows all by default.": "默认显示所有。",
    "E.g., ENSG, ENTREZGENE, UNIPROT_GN, REACTOME, WIKIGENE. Check the “Resources” section below for the complete list of supported namespaces.": "例如，ENSG、ENTREZGENE、UNIPROT_GN、REACTOME、WIKIGENE。请查看下面的“资源”部分，以获取支持的命名空间的完整列表。",
    "To screen for antibiotic resistant genes, can be a fasta file, a genbank file or an EMBL file.": "为了筛选抗生素抗性基因，可以是fasta文件、genbank文件或EMBL文件。",
    "Option to switch to other AMR/other database": "切换到其他抗微生物耐药性（AMR）数据库/其他数据库的选项",
    "Suppress output file's column headings": "隐藏输出文件的列标题",
    "The MAKER authors specifically allowed to make it available on Galaxy. MAKER is free for academic use, but commercial Galaxy users of MAKER still need a license, which can be obtained at http://weatherby.genetics.utah.edu/cgi-bin/registration/maker_license.cgi": "MAKER 的作者特别允许在 Galaxy 上使用它。MAKER 可以免费用于学术用途，但是商业 Galaxy 用户使用 MAKER 仍然需要一个许可证，该许可证可以在 http://weatherby.genetics.utah.edu/cgi-bin/registration/maker_license.cgi 上获得。",
    "Maker will blindly trust EST alignments to create gene models. Use this only before training ab-initio predictors.": "Maker 将盲目信任 EST 比对来创建基因模型。仅在训练 ab-initio 预测器之前使用此选项。",
    "Maker will blindly trust protein alignments to create gene models. Use this only before training ab-initio predictors.": "Maker 会完全信任蛋白质比对来创建基因模型。仅在训练 ab-initio 预测器之前使用此选项。",
    "From mutiple organisms": "来自多种生物",
    "Leave empty to disable gene prediction by SNAP": "留空以禁用SNAP基因预测",
    "Predictors will look for genes in both masked or unmasked sequence regions (instead of only unmasked). Only useful when you believe that transposons might have been integrated into a real gene structure (rare).": "预测器将在已掩蔽或未掩蔽的序列区域中寻找基因（而不是仅在未掩蔽的区域中）。只有当你认为转座子可能已经整合到一个真实的基因结构中时（罕见），这才有用。",
    "To use RepBase, choose 'Custom library of repeats' and select a fasta version of this non-free database.": "要使用RepBase，请选择“自定义重复序列库”并选择这个非免费数据库的fasta版本。",
    "Source species (or clade name) used to select repeats from DFam": "源物种（或类群名称），用于从DFam中选择重复序列",
    "i.e. seg and dust filtering": "即分段和除尘过滤",
    "The full DFam library can be downloaded from https://www.dfam.org/releases/current/families/Dfam.h5.gz": "完整的DFam库可以从 https://www.dfam.org/releases/current/families/Dfam.h5.gz 下载。",
    "annotation pass-through": "注释传递",
    "This will replace non-IUPAC characters with 'N's.": "这将用'N'替换非IUPAC字符。",
    "Increases/decreases memory usage": "增加/减少内存使用",
    "Under 10kb are often useless": "小于10kb的序列通常没有用处",
    "Will try to output gene isoforms when detected instead of a single consensus isoform.Use this if you have good EST evidences allowing to detect isoforms.": "将尝试输出检测到的基因异构体，而不是单一的共识异构体。如果您有良好的EST证据允许检测异构体，请使用此选项。",
    "Only cannonical gene structures will be reported, but it can lead to biologically incorrect seqences.": "仅报告规范的基因结构，但这可能导致生物学上不正确的序列。",
    "Expected max intron size for evidence alignments": "预期最大内含子大小用于证据比对",
    "Take this number of bases from the start and end of a fragment and compare to the database. This is an experimental feature, it hasn't performed as expected and may be removed at a later point. (default: None)": "从片段的起始和结束处取这个数量的碱基与数据库进行比较。这是一个实验性功能，它的表现没有达到预期，可能会在以后的版本中移除。（默认值：无）",
    "Internally blastn is used and this allows you to specify the minimum bit score for a hit to be considered, since blast will throw up a lot of small hits (default: 100)": "内部使用blastn，这允许您指定一个命中被认为是有效的最小比特分数，因为blast会生成很多小的命中（默认值：100）",
    "Only consider blast alignment lengths above this value. Remember that there can be some very short fragments between rrns, so you'll need to know the approximate minimum fragment size (in bases) before increasing this value too high. (default: 100)": "只考虑比这个值大的BLAST比对长度。请记住，rrn之间可能有一些非常短的片段，因此您需要知道大致的最小片段大小（以碱基为单位），然后再将此值提高得太高。（默认值：100）",
    "(default: False)": "（默认值：False）",
    "A tabular file with Gene IDs in the first column, and True or False in the second column. True means a gene is differentially expressed. See Help section for details.": "一个表格文件，第一列包含基因ID，第二列包含“True”或“False”。“True”表示一个基因是差异表达的。详细信息请见帮助部分。",
    "You can calculate the gene lengths using featureCounts or the Gene length and GC content tool.": "您可以使用featureCounts或基因长度和GC含量工具来计算基因长度。",
    "You can obtain a mapping of genes to categories (for some genomes only) or you can provide your own category file.": "您可以获得基因到类别的映射（仅限于某些基因组），或者您可以提供您自己的类别文件。",
    "Supported Gene IDs to automatically fetch categories should either be Entrez, Ensembl, or gene symbols.": "支持的基因ID应为自动获取类别的Entrez、Ensembl或基因符号。",
    "By default, goseq tests all three major Gene Ontology branches; Cellular Component, Biological Process and Molecular Function. However, it is possible to limit testing to any combination and/or to also use KEGG pathways.": "默认情况下，goseq测试所有三个主要的基因本体分支；细胞组分、生物过程和分子功能。然而，可以限制测试到任何组合，并且/或者也使用KEGG通路。",
    "See help for details": "查看帮助以获取详细信息",
    "Does not use gene length information. See help for details": "不使用基因长度信息。详见帮助了解详细信息。",
    "Number of random samples to be calculated when sampling is used. Set to 0 to not do sampling. Larger values take a long time": "当使用抽样时，要计算的随机样本数量。设置为0则不进行抽样。值越大，所需时间越长",
    "For example, a large number of genes may have no GO term annotated. If this option is set to No, those genes will be ignored in the calculation of p-values. If this option is set to Yes, then these genes will count towards the total number of genes outside the category being tested. This was the default behaviour for version 1.15.1 and earlier": "例如，许多基因可能没有GO术语注释。如果此选项设置为否，则在计算p值时将忽略这些基因。如果此选项设置为是，则这些基因将计入正在测试的类别之外的基因总数。这是1.15.1及更早版本的默认行为。",
    "Output a PDF plot of the Top 10 over-represented GO terms": "输出一个PDF格式的图表，展示10个过度表达的GO术语。",
    "This will produce the length bias (PWF) plot. If both sampling and wallenius methods are selected, it will also produce a plot comparing their p-values. These plots may help you compare the different p-value estimation methods that goseq can use": "这将生成长度偏差（PWF）图。如果同时选择了采样和Wallenius方法，它还将生成一个比较它们p值的图。这些图可能有助于您比较goseq可以使用的不同p值估计方法。",
    "Output all the data used by R to construct the tables and plots, can be loaded into R": "输出 R 用来构建表格和图形的所有数据，可以加载到 R 中",
    "A tabular file with gene symbols in the first column, and a ranked statistic (e.g. t-statistic or log fold-change) in the second column": "一个表格文件，第一列包含基因符号，第二列包含排名统计量（例如 t 统计量或对数倍数变化）",
    "If this option is set to Yes, the tool will assume that the ranked genes file has a column header in the first row and the identifers commence on the second line. Default: Yes": "如果将此选项设置为“是”，则工具将假定排名基因文件在第一行具有列标题，并且标识符从第二行开始。默认值：是",
    "A tabular file in GMT file or an RData file containing a list of gene sets, see below for more information": "一个包含基因集列表的GMT文件或RData文件，有关更多信息，请参见下文。",
    "Minimal size of a gene set to test. All pathways below the threshold are excluded. Default: 1": "基因集测试的最小大小。所有低于阈值的通路都被排除。默认值：1",
    "Maximal size of a gene set to test. All pathways above the threshold are excluded. Default: 500": "要测试的基因集的最大大小。超过阈值的所有通路都被排除在外。默认值：500",
    "Number of permutations to do. Minimial possible nominal p-value is about 1/nperm. Default: 1000": "要做的排列数。最小可能的名义p值大约是1/nperm。默认值：1000",
    "Output a PDF file containing plots for top pathways by P value significance. Default: No": "输出一个包含按P值显著性排序的顶级通路的PDF文件。默认值：否。",
    "If Output plots is selected the number of top pathways to plot can be specified. Default: 10": "如果选择了输出图表，则可以指定要绘制的顶级通路数量。默认值：10",
    "Output all the data used by R in the fgsea analysis, can be loaded into R. Default: No": "输出 R 在 fgsea 分析中使用的所有数据，可以加载到 R 中。默认值：否。",
    "The gnm2tab file should be from the table directory of spaln v. @TOOL_VERSION@": "gnm2tab文件应该来自spaln v. @TOOL_VERSION@ 的表格目录",
    "Choose a species table (e.g. cynosemi) from which to read parameters to optimise spaln": "选择一个物种表（例如 cynosemi），从中读取参数以优化 spaln。",
    "Determines how to treat orientation of query sequence when searching": "确定在搜索时如何处理查询序列的方向",
    "Default is 16M (16x1024x1024 bytes)": "默认值是16M（16乘以1024乘以1024字节）",
    "poly-A/poly-T trimming is only done if -S (orientation) option is 0 or 3": "如果 -S（方向）选项是 0 或 3，则仅执行 poly-A/poly-T 修剪",
    "Which dinucleotide pairs to accept at the ends of an intron": "哪种二核苷酸对在内含子的末端被接受",
    "Use the double affine gap rathr than single affine gap penalty calculation": "使用双仿射间隙惩罚而不是单仿射间隙惩罚计算",
    "More sensitive but less economic than the default": "比默认设置更敏感，但经济性较差",
    "Examine all positively scored blocks. Considerably slow.": "检查所有得分为正的区块。速度相当慢。",
    "Full Gene Ontology file in OBO or OWL format (see @ONTOLOGY_URL@)": "完整的基因本体文件，格式为OBO或OWL（详见@ONTOLOGY_URL@）",
    "Slim Gene Ontology file in OBO or OWL format (see @ONTOLOGY_URL@)": "瘦化基因本体文件，格式为OBO或OWL（见@ONTOLOGY_URL@）",
    "Tabular file containing annotations from gene products to GO terms (in GAF or BLAST2GO format, or a simple two-column table)": "包含基因产物到GO术语注释的表格文件（GAF或BLAST2GO格式，或简单的两列表格）",
    "Gene Ontology file in OBO or OWL format (see @ONTOLOGY_URL@": "基因本体论文件，格式为 OBO 或 OWL（参见 @ONTOLOGY_URL@）",
    "File containing the gene products corresponding to the study set (one per line)": "包含与研究集相对应的基因产物的文件（每行一个）",
    "File containing the gene products corresponding to the population set (one per line). If no file is submitted, the population set will be the set of all gene products listed in the annotation file.": "包含对应于种群集的基因产物的文件（每行一个）。如果没有提交文件，种群集将是注释文件中列出的所有基因产物的集合。",
    "The multiple test correction method to use (Benjamini-Hochberg is recommended).": "要使用的多重检验校正方法（推荐使用Benjamini-Hochberg）。",
    "The corrected p-value (or q-value) cut-off to apply for the graph output.": "校正后的 p 值（或 q 值）截止值，用于图形输出。",
    "The format of the output graphs (png, svg, or tabular for importing into cytoscape).": "输出图形的格式（png、svg 或表格形式，用于导入到 Cytoscape 中）。",
    "Whether to produce a summarized list of GO terms or the full list of those that are statistically significant": "是否生成一个GO术语的汇总列表，或那些具有统计学意义的完整列表",
    "Whether to exclude GO terms that are annotated to a single gene product in the study set": "是否排除在研究集中仅注释给单个基因产物的GO术语",
    "Whether to infer annotations through 'part_of' and other non-hierarchical relationships, or only through 'is_a' relations": "是否通过“part_of”和其他非层次关系推断注释，还是仅通过“is_a”关系推断注释",
    "Leave blank or use zero for no cutoff.": "留空或使用零表示没有截止值。",
    "Use zero for no truncation, maximum value 6000": "使用零表示不截断，最大值为6000",
    "Biom file, which should be the output of the normalize_by_copy_number command.": "Biom文件，应该是normalize_by_copy_number命令的输出。",
    "Limit to specified function ids (e.g. K00500). If limiting to multiple ids then separate them by commas with no spaces between them.": "限制到指定的函数ID（例如 K00500）。如果限制到多个ID，则用逗号隔开，ID之间不要有空格。",
    "Input trait table in QIIME OTU table format": "输入特征表，以QIIME OTU表格式",
    "Input tree in Newick format": "输入以Newick格式的树",
    "The character delimiting fields in the input trait table": "输入性状表中字段分隔符",
    "The character delimiting fields in the output trait table": "在输出性状表中用于分隔字段的字符",
    "An optional two-column, tab-delimited text file mapping identifiers in the tree (column 1) to identifiers in the trait table (column 2). If supplied, the identifiers in the trait table will be converted to match the identifiers in the tree. (This mapping does not need to be supplied if the tree and trait table already use a common set of identifiers.)": "一个可选的两列制表符分隔的文本文件，用于将树中的标识符（第一列）映射到性状表中的标识符（第二列）。如果提供了该文件，性状表中的标识符将被转换以匹配树中的标识符。（如果树和性状表已经使用一组共同的标识符，则不需要提供此映射。）",
    "Prune the reference tree to contain only those tips that are within the given OTU table": "修剪参考树，只保留在给定OTU表中的那些末端节点。",
    "If set, don’t ensure that tree is fully bifurcating": "如果设置，不要确保树是完全二分的",
    "Convert tree to NEXUS format, including a translate block mapping tip names to numbers": "将树转换为NEXUS格式，包括一个将末端名称映射到数字的转换块。",
    "Convert the values for each character state to integers": "将每个字符状态的值转换为整数",
    "If set, don’t ensure all branches have at least a small but non-zero branch length": "如果设置，不要确保所有分支至少有一个小但非零的分支长度",
    "If set, don’t filter out tree tips that aren’t listed in the trait table": "如果设置，不要过滤出在性状表中未列出的树尖",
    "If set, don’t filter out trait table entries that aren’t listed in the tree": "如果设置，不要过滤出在树中未列出的特征表条目",
    "Add a short branch to the root node (this is required by some phylogeny programs). The length of the branch is determined by the min_branch_length option": "在根节点添加一个短分支（这在一些系统发育程序中是必需的）。分支的长度由min_branch_length选项确定。",
    "The observed table of either relative abundance or real counts": "观察到的表，可以是相对丰度或实际计数",
    "The expected table of either relative abundance or real counts": "预期的表格可以是相对丰度或实际计数",
    "Round numbers (instead of taking ceil() which is used for RA) before calculating TP,FP,FN,TN": "将数字四舍五入（而不是使用天花板函数 ceil()，该函数用于 RA），然后计算真正例（TP）、假正例（FP）、假负例（FN）和真负例（TN）。",
    "Calculate accuracy values by comparing between observations (instead of between samples).": "通过比较观测值（而不是样本之间）来计算准确度值。",
    "Convert both expected and observed tables to relative abundances (instead of observations).": "将期望和观察到的表格都转换为相对丰度（而不是观察值）。",
    "Ignore observations that are not in the observed table.": "忽略不在观测表中的观察结果。",
    "Ignore observations that are not in the expected table.": "忽略不在预期表中的观察结果。",
    "Shuffle samples ids randomly before measuring accuracy.": "在测量准确性之前，随机打乱样本编号。",
    "Normalizes the predicted functional abundances by dividing each abundance by the sum of functional abundances in the sample. Total sum of abundances for each sample will equal 1": "将预测的功能丰度通过将每个丰度除以样本中功能丰度的总和来标准化。每个样本的丰度总和将等于1",
    "Normalizes the predicted functional abundances by dividing each abundance by the sum of OTUs in the sample. Note: total sum of abundances for each sample will NOT equal 1": "通过将样本中的每个OTU的丰度除以样本中OTUs的总和来标准化预测的功能性丰度。注意：每个样本的丰度总和将不等于1",
    "requires that per-genome accuracy metrics were calculated during genome prediction (predict_traits tool) (e.g. there are NSTI values in the genome biom file metadata)": "要求在基因组预测期间计算每个基因组的准确性指标（predict_traits工具）（例如，在基因组生物文件元数据中有NSTI值）",
    "Biom file that is output from the PICRISt predict tool": "由PICRISt predict工具输出的Biom文件",
    "a value of 1 is the highest level, and any higher value nears the leaves of the hierarchy. For instance, if the hierarchy contains 4 levels, specifying 3 would collapse at one level above being fully specified": "值为1是最高级别，任何接近层次结构末端的值都更高。例如，如果层次结构包含4个级别，指定3将折叠在完全指定的上一层级别。",
    "-i switch will be ignored. Composite Metazoan mitochondrial genetic code used. (-mt)": "-i 开关将被忽略。使用复合多细胞动物线粒体遗传密码。（-mt）",
    "-i switch will be ignored. Mammalian mitochondrial genetic code used. (-mtmam)": "-i 开关将被忽略。使用哺乳动物线粒体遗传密码。(-mtmam)",
    "With a maximum length of 3000 bases. (-i).": "最大长度为3000个碱基。(-i)。",
    "The 'fasta'-headers should contain the genomic position being used to find overlapping reads in the BAM file": "'fasta'文件头应包含用于在BAM文件中查找重叠读取的基因组位置",
    "Do not change this value unless you're using customized software in the pipeline - default: '>.*?(chr[^:]):([0-9]+)-([0-9]+)'": "不要更改这个值，除非你在流水线中使用了定制的软件 - 默认：'>.*?(chr[^:]):([0-9]+)-([0-9]+)'",
    "Minimal difference for binary split - the smaller this value the slower. if this value equals 0, the difference is set to infinity (default: 0.05)": "二元分裂的最小差异 - 这个值越小，速度越慢。如果这个值等于0，差异被设置为无穷大（默认值：0.05）",
    "Use sequences from a FASTA file instead of the XML file that contains the segments. In XML files you can explicitly link one Segment(Loop) to one particular sequence instead of doing n*n comparisons (default: None)": "使用来自FASTA文件的序列，而不是包含片段的XML文件。在XML文件中，您可以显式地将一个片段（循环）链接到一个特定的序列，而不是进行n*n次比较（默认：无）。",
    "This file should have been created with 'find-boxes'": "这个文件应该用 'find-boxes' 创建",
    "(default=250bp)": "（默认值=250bp）",
    "(default: 10bp)": "（默认值：10bp）",
    "Entries with transitions with energy smaller than energy (< e) or without transitions will be put into DBN_OUTPUT_FILE_LARGER_OR_EQUAL and those larger or equal (>= e) to DBN_OUTPUT_FILE_SMALLER": "具有能量小于能量（< e）的跃迁或没有跃迁的条目将被放入DBN_OUTPUT_FILE_LARGER_OR_EQUAL，而那些更大或等于（>= e）的将被放入DBN_OUTPUT_FILE_SMALLER。",
    "Sequence encoding can be found at the following url: https://en.wikipedia.org/wiki/FASTA_format#Sequence_representations": "序列编码可以在以下网址找到：https://en.wikipedia.org/wiki/FASTA_format#Sequence_representations",
    "Whole genome alignment file in fasta format": "全基因组比对文件，使用fasta格式",
    "Default selections are the Final Tree in Newick format, the Recombination Predictions in gff3 format and the Summary of SNP Distribution": "默认选项是新克格式的最终树、gff3格式的重组预测和SNP分布摘要。",
    "Maximum No. of iterations, default is 5": "最大迭代次数，默认为5",
    "Criteria to use to know when to halt iterations": "停止迭代的条件",
    "Outgroup name for rerooting. A list of comma separated names can be used if they form a clade.": "重根的外群名称。如果它们形成一个类群，可以使用逗号分隔的名称列表。",
    "Default: False": "默认值：False",
    "Application to use for tree building, default RAxML": "用于构建树的应用程序，默认为 RAxML",
    "Min SNPs to identify a recombination block. Default: 3": "最小SNPs数量以识别一个重组块。默认值：3",
    "Filter out taxa with more than this percentage of gaps. Default: 25": "过滤掉具有超过此百分比间隙的分类群。默认值：25",
    "Minimum window size. Default: 100": "最小窗口大小。默认值：100",
    "Maximum window size. Default: 10000": "最大窗口大小。默认值：10000",
    "p-Value for detecting recombinations. Default: 0.05": "重组检测的p值。默认值：0.05",
    "Ratio of log probabilities used to trim recombinations. Default: 1": "用于修剪重组的对数概率比率。默认值：1",
    "Public rRNA databases provided with SortMeRNA have been indexed. On the contrary, personal databases must be indexed each time SortMeRNA is launched. Please be patient, this may take some time depending on the size of the given database.": "SortMeRNA 提供的公共 rRNA 数据库已经被索引过了。相反，个人数据库每次启动 SortMeRNA 时都必须进行索引。请耐心等待，这可能需要一些时间，具体取决于给定数据库的大小。",
    "Your databases will be indexed first, which may take up to several minutes.": "你的数据库将首先被索引，这可能需要几分钟时间。",
    "Report all alignments can be time consuming; this option is not suggested for high similarity rRNA databases.": "报告所有比对可能会消耗很多时间；对于高相似性rRNA数据库，不建议使用此选项。",
    "The 'best' alignment is the highest scoring alignment out of All alignments of a read, and the read can potentially be aligned (reaching E-value threshold) to multiple reference sequences. By default the program searches for best alignments i.e. performs an exhaustive search over all references. Using '-no-best' will make the program to search just the first N alignments.": "“最佳”比对是指一个读取的所有比对中得分最高的比对，并且读取可能（达到E值阈值）被比对到多个参考序列上。默认情况下，程序搜索最佳比对，即对所有参考进行穷举搜索。使用“-no-best”将使程序只搜索前N个比对。",
    "It is computed using seeds, which are k-mers common to the read and the reference sequence. Sorted sequences of such seeds are used to filter the candidate references prior performing the Smith-Waterman alignment.": "它使用种子来计算，这些种子是读取和参考序列共有的k-mers。使用这些种子的排序序列来过滤候选参考，然后再执行Smith-Waterman比对。",
    "Defines the 'statistical significance' of a local alignment. Exponentially correllates with the Minimal Alignment score. Higher E-values (100, 1000, ...) cause more reads to pass the alignment threshold.": "定义了局部比对的“统计显著性”。与最小比对分数呈指数相关。较高的E值（100，1000，...）会导致更多的读取通过比对阈值。",
    "Scored as --mismatch": "评分为 --mismatch",
    "Output file from annotate step": "注释步骤的输出文件",
    "output file from annotate step": "注释步骤的输出文件",
    " [allowMiss] (ionAnnot function) Logical, annotate also the peaks as single charged molecules [M+/-H].": "[允许缺失] (ionAnnot 函数) 逻辑值，将峰也注释为单电荷分子 [M+/-H]。",
    "[toexclude]": "[排除]",
    "Search on all KEGG organisms or multiple organisms (id1,id2,id3,...).By default,the value is KEGG which means searching on all KEGG organism. The list of KEGG IDs are available at http://rest.kegg.jp/list/organism": "在所有KEGG生物体或多个生物体（id1,id2,id3,...）上进行搜索。默认值是KEGG，意味着在所有KEGG生物体上进行搜索。KEGG ID的列表可在 http://rest.kegg.jp/list/organism 获取。",
    "[ppm.tol]": "[ppm.tol]",
    "[prob] Default is 'count'. See the tool help for more details.": "[问题] 默认是“计数”。有关更多详细信息，请查看工具帮助。",
    "[html] This parameter uses the raw data to plot EICs and may be time consuming.": "这个参数使用原始数据绘制EICs（电离层不规则体），可能会很耗时。",
    "[opt] cor for correlation, and pcor for partial correlation.": "[选项] cor 表示相关性，pcor 表示偏相关性。",
    "[corprob]": "[保守性概率]",
    "[pcorprob]": "[相关概率]",
    "[corths]": "[相关性热图]",
    "Decimal: '.', missing: NA, mode: character and numerical, sep: tabular. Retention time values must be in seconds.": "小数点：'.'，缺失值：NA，模式：字符和数值，分隔符：表格。保留时间值必须以秒为单位。",
    "The list of column names of your database in-house file, as a coma separated list of key/value pairs.": "数据库内部文件的列名列表，作为以逗号分隔的键/值对列表。",
    "Values used for the file database MS modes, as a coma separated list of key/value pairs.": "用于文件数据库MS模式的值，作为键/值对的逗号分隔列表。",
    "If you do not have yet a Peakforest token, go to Peakforest website and request one from your account.": "如果您还没有Peakforest令牌，请前往Peakforest网站并从您的账户中请求一个。",
    "Decimal: '.', missing: NA, mode: character and numerical, sep: tabular. RT values must be in seconds.": "小数点：'.'，缺失值：NA，模式：字符和数值，分隔符：表格。RT值必须以秒为单位。",
    "Input file column names, as a coma separated list of key/value pairs.": "输入文件列名称，以逗号分隔的键/值对列表。",
    "The retention time tolerance X parameter (in seconds).": "保留时间公差 X 参数（以秒为单位）。",
    "The retention time tolerance Y parameter (no unit).": "保留时间容差 Y 参数（无单位）。",
    "The retention time tolerance used when precursor matching is enabled.": "当启用前体匹配时使用的保留时间容差。",
    "Nucleosomes concentration: the default value = 0.1": "核小体浓度：默认值为0.1",
    "Temperature scaling: the default value = 1": "温度缩放：默认值 = 1",
    "Raw Binding: When 'No' (default), then the output will be the default average occupancy probabilities. When 'yes', then the output will be  the raw nucleosome binding log-ratio per basepair": "原始结合：当为'No'（默认值）时，输出将是默认的平均占有率概率。当为'yes'时，输出将是每个碱基对的原始核小体结合对数比率。",
    "OrthoFinder can be run in 2 steps. Choose 'From fasta files' to run OrthoFinder from scratch or 'From blast results' if you have all the blast results from a previous OrthoFinder run.": "OrthoFinder 可以分两步运行。选择“从fasta文件”从头开始运行OrthoFinder，或者如果你有之前OrthoFinder运行的所有blast结果，选择“从blast结果”。",
    "One fasta file per species; species and sequence names in the results will remain the same than in the input files.": "每个物种一个fasta文件；结果中的物种和序列名称将与输入文件中的保持不变。",
    "Used to re-run OrthoFinder from pre-computed blast results": "用于从预先计算的BLAST结果重新运行OrthoFinder。",
    "blastX_Y.txt files from the blast output files of a previous OrthoFinder run.": "来自先前OrthoFinder运行的BLAST输出文件的blastX_Y.txt文件。",
    "= SpeciesX.fa files from the blast output files of a previous OrthoFinder run.": "= 从上一次OrthoFinder运行的blast输出文件中获取SpeciesX.fa文件。",
    "SpeciesIDs.txt file from the blast output files of a previous OrthoFinder run.": "来自先前OrthoFinder运行的blast输出文件的SpeciesIDs.txt文件。",
    "SequencesIDs.txt file from the blast output files of a previous OrthoFinder run.": "来自先前OrthoFinder运行的BLAST输出文件的SequencesIDs.txt文件。",
    "Modify inflation parameter for MCL. Not recommended.": "修改MCL的膨胀参数。不推荐。",
    "The FASTA file must contain a comment line starting with > followed by the sequence. It can also contain several consecutive sequences and DotKnot will be executed for each sequence in the file.": "FASTA文件必须包含以>开头的注释行，后面跟着序列。它还可以包含几个连续的序列，DotKnot将针对文件中的每个序列执行。",
    "Kissing hairpins are complex and biologically relevant types of pseudoknots. Inclusion of kissing hairpins will lead to increased run time, yet produce more meaningful results.": "接吻发夹是复杂且生物学上相关的伪结类型。包含接吻发夹将导致运行时间增加，但会产生更有意义的结果。",
    "Shows top five near-optimal local pseudoknots in terms of two criteria: estimated free energy to length ratio and lowest estimated free energy. This can help to identify promising pseudoknot foldings and may compensate for the limitations of the energy parameters.": "显示基于两个标准的前五个近似最优局部假结：估计自由能与长度比和最低估计自由能。这有助于识别有希望的假结折叠，并可能弥补能量参数的局限性。",
    "Shows predicted global structure in addition to predicted pseudoknots.": "显示预测的全局结构以及预测的伪结。",
    "Choose the type of files (nucleotide and/or protein to be analyzed)": "选择要分析的文件类型（核苷酸和/或蛋白质）",
    "Choose a taxonomic group to point mutation screening": "选择一个分类群进行点突变筛查",
    "Minimum proportion of identical amino acids in alignment. -1 means use a curated threshold if it exists and 0.9 otherwise (default: -1)": "在比对中相同氨基酸的最小比例。-1 表示如果存在一个经过校准的阈值则使用该阈值，否则使用 0.9（默认值：-1）",
    "Minimum coverage of the reference protein (0 to 1, default: 0.5)": "参考蛋白的最小覆盖度（0到1，默认：0.5）",
    "NCBI genetic code for translated BLAST (default: bacterial table 11)": "NCBI 遗传密码表，用于翻译 BLAST（默认：细菌表 11）",
    "Text to be added as the first column 'name' to all rows of the report": "要添加到报告所有行的第一列 'name' 的文本",
    "Output from 'Funannotate predict annotation' tool": "来自“Funannotate 预测注释”工具的输出",
    "Create it on https://submit.ncbi.nlm.nih.gov/genbank/template/submission/ (or leave empty to use a default one, not suitable for submission at NCBI)": "在 https://submit.ncbi.nlm.nih.gov/genbank/template/submission/ 创建它（或者留空以使用默认模板，该模板不适用于向NCBI提交）",
    "'annotations' output from 'eggNOG Mapper' tool": "'eggNOG Mapper' 工具输出的 '注释'",
    "Genbank output from 'Antismash' tool": "来自'Antismash'工具的Genbank输出",
    "XML output from InterProScan": "InterProScan的XML输出",
    "3 column tsv file": "三列制表符分隔的文件",
    "TSV: GeneID Name Product": "TSV: 基因ID 名称 产品",
    "TSV: Gene Product": "TSV: 基因产物",
    "Sequence should be in FASTA format": "序列应该以FASTA格式提供",
    "This can be in FASTA or FASTQ format.": "这可以是FASTA或FASTQ格式。",
    "For example GGATCC for BamHI.": "例如，BamHI的限制性内切酶识别位点为GGATCC。",
    "default: 10000": "默认值：10000",
    "Value should be < 10. default: 1e-3": "值应该小于10，默认值：1e-3",
    "Value should be in [0.0,100.00]": "值应该在 [0.0,100.00] 范围内",
    "Single sequence or multiple sequence alignment (MSA) in FASTA format, or HMM in hhm3 format.": "单个序列或多序列比对（MSA）以FASTA格式，或HMM以hhm3格式。",
    "Select a search method. See help below for more information.": "选择一种搜索方法。有关更多信息，请参见下面的帮助。",
    "Built-ins have been indexed using ffindex": "内置的已经使用ffindex进行了索引。",
    "A set of known genes in FASTA format.": "一组已知基因的FASTA格式文件。",
    "The depth is the maximum number of positions in the context window that will be used to determine the probability of the predicted position.": "深度是上下文窗口中用于确定预测位置概率的最大位置数。",
    "The period is the number of different submodels for different positions in the text in a cyclic pattern. E.g., if the period is 3, the first submodel will determine positions 1, 4, 7, ...": "周期是指文本中不同位置的子模型数量，以循环模式出现。例如，如果周期为3，则第一个子模型将确定位置1、4、7等。",
    "The width includes the predicted position.": "宽度包括预测位置。",
    "Overlaps this short or shorter are ignored.": "重叠部分如果短于或等于这个长度，则会被忽略。",
    "This does not include the bases in the stop codon.": "这并不包括终止密码子中的碱基。",
    "If the in-frame score >= N, then the region is given a number and considered a potential gene.": "如果内框得分 >= N，则该区域被赋予一个编号，并被视为潜在的基因。",
    "If 0.0 specified, the GC percentage will be counted from the input file.": "如果指定为0.0，则GC百分比将从输入文件中计算。",
    "Using this option will produce more short gene predictions.": "使用此选项将产生更多短基因预测。",
    "For people with their own deployment of COMBAT-TB-NeoDB": "对于拥有自己的COMBAT-TB-NeoDB部署的人来说",
    "Reads of sample(s) being spoligotyped": "样本的读取正在进行 spoligo 打字",
    "FASTA format file containing customer spacer library": "FASTA格式文件，包含客户间隔序列库",
    "Query SITVIT service for report on spoligotype": "查询SITVIT服务以获取有关spoligotype的报告",
    "Input is either a complete genomic sequence or assembled contigs from an isolate": "输入可以是来自分离株的完整基因组序列或组装的连续序列",
    "Scaffolds are sometimes joined with stretches of 25 or more Ns. This option ignores them when calculating repeat statistics": "当计算重复序列统计信息时，此选项忽略了由25个或更多N碱基组成的片段连接的支架。",
    "Output repetitive regions as lowercase, non-repetitive regions as uppercase (instead of replacing repetitive regions with 'N's)": "输出重复区域为小写，非重复区域为大写（而不是用'N's替换重复区域）",
    "Normally RepeatMasker will report on IS element, with this option selected it will clip them before analysis": "通常RepeatMasker会报告插入序列（IS）元素，选择此选项后，它会在分析之前剪切它们。",
    "If this option is select a check for rodent specific repeats is done instead of a full RepeatMasker run": "如果选择这个选项，将执行针对啮齿动物特异性重复序列的检查，而不是完整的RepeatMasker运行。",
    "If this option is select a check for primate specific repeats is done instead of a full RepeatMasker run": "如果选择此选项，将执行针对灵长类特异性重复序列的检查，而不是完整的RepeatMasker运行。",
    "Skip masking of simple tandem repeats and low complexity regions.": "跳过简单串联重复和低复杂性区域的掩蔽。",
    "Only mask simple repeats, skip masking of interspersed repeats.": "仅掩蔽简单重复序列，跳过散布重复序列的掩蔽。",
    "Skip masking of small pol III transcribed RNA (these are masked by default because they resemble SINEs)": "跳过对小型RNA聚合酶III转录的RNA的掩蔽（默认情况下这些会被掩蔽，因为它们类似于SINEs）",
    "Maximum length of sequencing that is search without fragmenting": "最大不断裂的搜索测序长度",
    "Valid values are a percentage or -1 to choose the default": "有效值是百分比或-1以选择默认值",
    "By default RepeatMasker skips calculating GC % for small sequences": "默认情况下，RepeatMasker 会跳过为小序列计算 GC%",
    "Show alignments in the orientation of the repeat sequence, not the query sequence": "显示与重复序列方向一致的比对，而不是查询序列方向",
    "By default, HOMER will remove a motif that overlaps itself, useful for palindromes. To report all, specify this option.": "默认情况下，HOMER会移除一个与其自身重叠的基序，这对于回文结构很有用。若要报告所有情况，请指定此选项。",
    "Must be of datatype \"gtf\" or \"gff\" or \"gff3\"": "必须为数据类型“gtf”或“gff”或“gff3”",
    "This file can be generated from gtf with the tool 'Converts GTF to Annotations file for Homer": "这个文件可以使用工具 'Converts GTF to Annotations file for Homer' 从 gtf 文件生成",
    "values greater 12 may cause the program to run out of memory - in these cases decrease the number of sequences analyzed (-N), or try analyzing shorter sequence regions (i.e. -size 100)": "值大于12可能会导致程序运行内存不足 - 在这些情况下，减少分析的序列数量（-N），或者尝试分析更短的序列区域（即 -size 100）",
    "automatically sets -norevopp": "自动设置 -norevopp",
    "0 means no local background.": "0表示没有局部背景。",
    "Maximum expected motif instance per bp in random sequence": "在随机序列中，每个碱基对上预期的最大基序实例数",
    "-removeAccVer; Default=False": "移除版本号；默认=否",
    "-removeEnsemblVer; Default=False": "移除Ensembl版本号；默认=否",
    "-annTSSstartOffset; Default=-1000": "-annTSSstartOffset; 默认=-1000",
    "-annTSSendOffset; Default=100": "-annTSSendOffset; 默认=100",
    "-annTTSstartOffset; Default=-100": "-annTTSstartOffset; 默认值=-100",
    "-annTTSendOffset; Default=1000": "-annTTSendOffset; 默认=1000",
    "Select your program": "选择你的程序",
    "The corresponding tools must be installed manually by the administrator of this Galaxy instance": "相应的工具必须由这个Galaxy实例的管理员手动安装",
    "Select your programm.": "选择你的程序。",
    "Option that provides mappings from matches to pathway information, which is based on the matched manually curated InterPro entries.": "提供从匹配到通路信息的映射选项，这基于匹配的手动策划的InterPro条目。",
    "Look up of corresponding Gene Ontology annotation. Implies -iprlookup option.": "查找相应的基因本体注释。意味着使用-iprlookup选项。",
    "Provide mappings from matched member database signatures to the InterPro entries that they are integrated into": "提供从匹配的成员数据库签名到它们集成的InterPro条目的映射。",
    "Please select a output format (JSON output can be visualised on https://www.ebi.ac.uk/interpro/result/InterProScan/).": "请选择一个输出格式（JSON格式可以在 https://www.ebi.ac.uk/interpro/result/InterProScan/ 上可视化）。",
    "One or more input files in FASTA, FASTQ, Genbank, EMBL, Clustal, Stockholm or GFF format.": "一个或多个输入文件，格式可以是FASTA、FASTq、Genbank、EMBL、Clustal、Stockholm或GFF。",
    "Removes the table header from the output (ie. FILE SBT flaA ...)": "删除输出中的表格标题（例如：文件 SBT flaA ...）",
    "Prints the output in comma-separated value format instead of tab-separated value format.": "将输出打印为逗号分隔值格式，而不是制表符分隔值格式。",
    "Single column text file": "单列文本文件",
    "Fasta sequence to analyse for IS": "用于分析插入序列（IS）的FASTA序列",
    "Remove partial IS elements which include IS element with length > 400 or single copy IS element without perfect TIR": "移除部分插入序列元件，包括长度大于400的插入序列元件或没有完美靶向重复序列的单一拷贝插入序列元件。",
    "Build an index from FASTA or use a pre-indexed database": "从FASTA文件构建索引或使用预索引的数据库",
    "Genomic contigs / scaffolds to be aligned against in FASTA format": "要进行比对的基因组片段/脚手架，以FASTA格式提供。",
    "Miniprot splits the genome into non-overlapping bins of 2^8 bp in size": "Miniprot 将基因组分割成大小为 2^8 碱基对的不重叠的箱子",
    "Increasing this option reduces peak memory but decreases sensitivity": "增加此选项可以减少峰值内存使用，但会降低敏感性",
    "A pre-indexed database built by miniprot": "由miniprot构建的预索引数据库",
    "Protein sequences to be aligned in FASTA format": "要进行比对的蛋白质序列以FASTA格式提供",
    "No splicing (apply -G1000 -J1000 -e1000)": "不进行剪接（应用 -G1000 -J1000 -e1000）",
    "A k-long gap costs open_penalty+k*extension_penalty": "一个长度为k的间隙成本为open_penalty+k*extension_penalty",
    "2=mammal, 1=general, 0=none": "2=哺乳动物，1=一般，0=无",
    "The number of outputs will be the minimum of this and the max secondary alignments option": "输出数量将是这个和最大次级比对选项的最小值",
    "Only for GFF output": "仅用于GFF输出",
    "Skipping frameshift": "跳过移码",
    "A newline-delimited text file that contains a list of genes of interest": "一个包含感兴趣基因列表的以换行符分隔的文本文件",
    "The corrected p-value (or q-value) cut-off to apply for the graph output. Default: 0.05": "用于图形输出的校正p值（或q值）截止值。默认值：0.05",
    "Community detection methods include edge betweenness score (EBC), leading eigenvector method (LEC), Louvain method (LOUV), or Leiden method (LEID). Default: EBC": "社区检测方法包括边介数评分（EBC）、主导特征向量方法（LEC）、Louvain方法（LOUV）或Leiden方法（LEID）。默认：EBC",
    "Perform global network null model analysis. The global network null model calculates the empirical p-value as the number of times (over a set of iterations) the size of the largest connected component (the giant component) in the network coming from the same number of randomly selected genes equals or exceeds the size of the largest connected component in the observed network. Default: Yes": "执行全局网络空模型分析。全局网络空模型计算经验p值，作为在一系列迭代中，来自相同数量随机选择基因的网络中最大连通分量（即巨型组件）的大小等于或超过观察到的网络中最大连通分量的大小的次数。默认值：是",
    "Global network null model iterations. Recommended: 1000": "全局网络空模型迭代次数。推荐：1000",
    "Global network null model number of genes.": "全局网络零模型基因数量。",
    "Perform local network null model analysis. The local network null model evaluates the deviation of modularity in the observed network from modularity distribution in the random network. Default: Yes": "执行局部网络空模型分析。局部网络空模型评估观察到的网络中模块度与随机网络中模块度分布的偏差。默认值：是",
    "Local network null model iterations. Recommended: 1000": "局部网络零模型迭代。推荐：1000",
    "Output the annotated graph of netboxr output. Default: Yes": "输出netboxr的注释图。默认值：是",
    "Plot width. Recommended: 9": "绘图宽度。推荐值：9",
    "Output the network in SIF format. Default: Yes": "输出SIF格式的网络。默认：是",
    "Output a table containing the information of all neighbor nodes. Default: Yes": "输出一个包含所有邻居节点信息的表格。默认值：是",
    "Output a table containing the identified pathway module numbers. Default: Yes": "输出一个包含已识别的通路模块编号的表格。默认值：是",
    "Output a table containing the information of node types. Default: Yes": "输出一个包含节点类型信息的表格。默认值：是",
    "File to be analyzed": "待分析的文件",
    " Fraction of matching sequence on the total target sequence(default: 0.6)": "匹配序列在总目标序列中的比例（默认值：0.6）",
    " Fraction of shared nucleotide on the match (default: 0.95)": "匹配上共享的核苷酸比例（默认值：0.95）",
    "Genomic contigs / scaffolds in FASTA format": "基因组的连续序列/脚手架，以FASTA格式",
    "Replicon can be entire chromosome, contif, PCR fragments...": "复制子可以是整个染色体、连续片段、PCR片段等。",
    "This option allows a more sensitive search. I will be slower (dependant on the number of hits) if integrons are found, but will be as fast if nothing is detected and will not increase the false positive rate.": "这个选项允许进行更敏感的搜索。如果发现整合子，搜索速度会变慢（取决于命中数），但如果未检测到任何内容，则速度会很快，并且不会增加误报率。",
    "Set the default topology for replicons, linear, circular (deault: no topology)": "设置复制子的默认拓扑结构，线性的，圆形的（默认：无拓扑结构）",
    "Generate a GenBank file with the sequence annotated with the same annotations than .integrons file.": "生成一个带有与.integrons文件相同注释的序列的GenBank文件。",
    "For each complete integron, a simple graphic of the region is depicted (in pdf format)": "对于每个完整的整合子，该区域的简单图形被描绘出来（以pdf格式）。",
    "By default, to cluster an array of attC sites and an integron integrase, they must be less than 4 kb apart. You can here change this value.": "默认情况下，为了聚类一个attC位点数组和一个整合子整合酶，它们之间的距离必须小于4 kb。您可以在这里更改这个值。",
    "Keep 'CALIN' only if attC sites number >= calin-threshold": "如果attC位点数量大于或等于calin-threshold，则仅保留'CALIN'",
    "For a given hit, if the palindromic version is found, don't remove the one with highest evalue": "对于给定的命中，如果找到了回文版本，不要移除具有最高e值的那个。",
    "When enabled, it does not annotate CDS and does not find integrase.": "当启用时，它不会注释CDS，也不会发现整合酶。",
    "Instead of taking intersection of hits from Phage_int profile (Tyr recombinases) and integron_integrase profile, use the union of the hits": "而不是取Phage_int（Tyr重组酶）和integron_integrase（整合子整合酶）两个profile的命中结果的交集，使用它们的并集。",
    "It is recommended to use short and unique sequence identifiers": "建议使用简短且唯一的序列标识符",
    "Reference genome sequences in FASTA format": "参考基因组序列在FASTA格式中",
    "Coding sequence (must be CDS without UTR, i.e. from start coden to stop coden) in FASTA format": "编码序列（必须是不包含非翻译区的CDS，即从起始密码子到终止密码子）以FASTA格式",
    "Noncoding sequences in FASTA format": "非编码序列在FASTA格式中",
    "Minimum ORF length in nucleotides": "最小开放阅读框架长度（核苷酸）",
    "RNAs may have dozens of putative ORFs, in most cases, the real ORF is ranked (by size) in the top several": "RNA可能包含数十个推定的开放阅读框（ORF），在大多数情况下，真正的开放阅读框（ORF）按大小排名前几位。",
    "Pfam-A HMMs in an HMM library searchable with the hmmscan program.": "Pfam-A HMM 在一个可以被 hmmscan 程序搜索的 HMM 库中。",
    "Stockholm format is a multiple sequence alignment format used by Pfam, Rfam and Dfam, to disseminate protein, RNA and DNA sequence alignments.": "斯德哥尔摩格式是一种多序列比对格式，由Pfam、Rfam和Dfam使用，用于传播蛋白质、RNA和DNA序列比对。",
    "Predict active site residues for Pfam-A matches": "预测Pfam-A匹配的活性位点残基",
    "This file is required for predicting the active site residues.": "这个文件是用于预测活性位点残基所必需的。",
    "Specify hmmscan evalue sequence cutoff for Pfam-A searches (default Pfam defined)": "指定 hmmscan evalue 序列截止值用于 Pfam-A 搜索（默认为 Pfam 定义的值）",
    "Specify hmmscan evalue domain cutoff for Pfam-A searches (default Pfam defined)": "为Pfam-A搜索指定hmmscan的e值域截止（默认为Pfam定义）",
    "Specify hmmscan bit score sequence cutoff for Pfam-A searches (default Pfam defined)": "指定 hmmscan 位分数序列截止值，用于 Pfam-A 搜索（默认为 Pfam 定义）",
    "Specify hmmscan bit score domain cutoff for Pfam-A searches (default Pfam defined)": "指定 hmmscan 比特分数域截止值，用于 Pfam-A 数据库搜索（默认为 Pfam 定义的值）",
    "Show overlapping hits within clan member families (applies to Pfam-A families only)": "显示家族成员中重叠的命中（仅适用于Pfam-A家族）",
    "Show the HMM-sequence alignment for each match": "显示每个匹配的HMM-序列比对",
    "Please upload a genome file of a bacteriophage in fasta format.": "请上传一个以fasta格式的噬菌体基因组文件。",
    "HOMER accepts a broad range of different types of gene identifiers such as NCBI Entrez Gene IDs, NCBI Unigene IDs, NCBI Refseq IDs, Ensembl Gene IDs, Gene Symbols (e.g. 'Nfkb1\\'), popular affymetrix probe IDs (MOE430, U133plus, U95, U75A)": "HOMER接受多种不同类型的基因标识符，如NCBI Entrez基因ID、NCBI Unigene ID、NCBI Refseq ID、Ensembl基因ID、基因符号（例如'Nfkb1'）、流行的Affymetrix探针ID（MOE430、U133plus、U95、U75A）",
    "Might be useful if background you provide is a full genome, transcriptome or so.": "如果提供的背景是完整的基因组、转录组等，可能会很有用。",
    "If multiple, use a comma-separated format, e.g. '8,10,12'. Too high numbers may cause the program to run out of memory. In such cases decrease the number of sequences analyzed (-N).": "如果多个，使用逗号分隔的格式，例如 '8,10,12'。数值过高可能会导致程序运行内存不足。在这种情况下，减少分析的序列数量（-N）。",
    "Automatically disables reverse strand search": "自动禁用反向链搜索",
    "Use this if -nlen does not work well.": "如果-nlen参数效果不佳，请使用此方法。",
    "Maximum expected motif instance per bp in a random sequence": "在随机序列中，每碱基最大期望基序实例数",
    "Can be used with Homer findMotifsGenome tool.": "可以使用Homer的findMotifsGenome工具。",
    "Braker uses GeneMark, which is not a free software, to use it download and unzip a license from http://topaz.gatech.edu/GeneMark/license_download.cgi (ES/ET/EP version). GeneMark needs to be installed manually by Galaxy administrators.": "Braker 使用了 GeneMark，这不是一个免费软件，要使用它需要从 http://topaz.gatech.edu/GeneMark/license_download.cgi 下载并解压缩一个许可证（ES/ET/EP 版本）。GeneMark 需要由 Galaxy 管理员手动安装。",
    "Experimental, requires RNASeq data (bam) and a softmasked genome": "实验性的，需要RNA测序数据（bam格式）和一个软掩蔽的基因组。",
    "String specifying the returned gene ID format.": "指定返回的基因ID格式的字符串。",
    "String specifying the returned gene ID format": "字符串，指定返回的基因ID格式",
    "String specifying the returned metabolite ID format.": "指定返回的代谢物ID格式的字符串。",
    "Available pathway databases": "可用的通路数据库",
    "It specifies the method to combine multiple p-values": "它指定了组合多个p值的方法",
    "Multiple testing corrections adjust p-values derived from multiple statistical tests to correct for occurrence of false positives": "多重检验校正调整来自多重统计检验的p值，以纠正假阳性的发生。",
    "Use the forward only option if your sequence directionality is known (e.g. from poly-A tails, or strand specific RNA sequencing).": "如果您的序列方向性已知（例如，来自聚腺苷酸尾巴，或特定链的RNA测序），请使用仅正向选项。",
    "Choose the model to use for the annotation": "选择用于注释的模型",
    "It may be necessary to reduce it if the GPU runs out of memory": "如果GPU内存不足，可能需要减少它",
    "This determines the number of bases averaged during the sliding window approach": "这决定了在滑动窗口方法中平均的碱基数量",
    "This threshold specifies the genic score which defines the start / end boundaries of each candidate region": "这个阈值指定了基因分数，它定义了每个候选区域的起始/结束边界",
    "This threshold specifies the minimum peak genic score required to accept the candidate region": "这个阈值指定了接受候选区域所需的最小峰基因分数",
    "Minimum number of amino acids a CDS pair must be separated by to detect an intron. Set to a negative number to instead allow CDS to overlap by up to that many AAs.": "检测内含子时，CDS对之间必须相隔的最小氨基酸数量。如果设置为负数，则允许CDS最多重叠该数值的氨基酸。",
    "Maximum number of bases a CDS pair may be separated by to detect an intron (eg, distance from end of 1 cds to the start of another).": "CDS对之间检测内含子时可能分离的最大碱基数量（例如，从一个CDS的末端到另一个CDS的起始端的距离）。",
    "Minimum Identity Percentage for HSP to be considered for intron comparison (0.00 to 1.00)": "用于内含子比较的HSP的最小一致性百分比（0.00到1.00）",
    "Use 'nice_cds' if your features are coming from Apollo, however this will fail on non-Apollo data": "如果您的特征来自Apollo，请使用'nice_cds'，但这对于非Apollo数据将会失败。",
    "Which tag is used to store gene numbers": "哪个标签用于存储基因编号",
    "A string to use as a prefix for the numbering. Will be used as XXXXXXNNN where XXXXXX is the string and NNN is a numerical identifier. Using \"display_id\" has special meaning, it will use the genome's name/accession number": "一个用作编号前缀的字符串。将用作 XXXXXXNNN，其中 XXXXXX 是该字符串，NNN 是一个数字标识符。使用 \"display_id\" 具有特殊含义，它将使用基因组的名称/访问号码。",
    "If tag is not present, only location and type checks will be used to infer renumber": "如果标签不存在，则仅使用位置和类型检查来推断重新编号",
    "New ID for the sequence to uniquely identify it from the previous build of the sequence. E.g. append Miro could become Miro.2 or Miro.v2": "为序列分配一个新的ID，以将其与先前构建的序列区分开。例如，附加Miro可以变成Miro.2或Miro.v2。",
    "Inclusive range, 1-indexed genome. (I.e. specifying 1-2000 will include base number 1)": "包含范围，1-索引的基因组。（即指定1-2000将包含碱基编号1）",
    "Inclusive range. (I.e. specifying 1-2000 will include base number 2000)": "包含性范围。（即指定1-2000将包括基数2000）",
    "Enter the sequence, e.g. 'ACTG'. No FASTA definition line.": "输入序列，例如 'ACTG'。不要使用FASTA定义行。",
    "Optional GAF Data Table. This usually comes from the CPT CACAO GAF Data Export tool. The [GAF] Columns below are ONLY available when this data is populated.": "可选的基因关联文件（GAF）数据表。这通常来自CPT CACAO GAF数据导出工具。下面的[GAF]列只有在这些数据被填充时才可用。",
    "Specify dataset with single read": "指定单端测序数据集",
    "Default:4": "默认值：4",
    "Default:10": "默认值：10",
    "Default:150": "默认值：150",
    "Default:5": "默认值：5",
    "(Default:500)": "（默认：500）",
    "(Default:11)": "（默认值：11）",
    "(Default:No)": "（默认值：否）",
    "A semi-colon separated file, listing all isoforms of each genes, with one gene per line.": "一个以分号分隔的文件，列出了每个基因的所有异构体，每行一个基因。",
    "Can be used to obtain a set of genes on which to train": "可以用来获得一组用于训练的基因集",
    "NCBI taxid corresponding to the input proteome": "NCBI taxid 对应于输入的蛋白质组",
    "When parsing interval loci, use the following delta to extend gene loci and include potential regulatory regions": "在解析区间位点时，使用以下增量来扩展基因位点并包括潜在的调控区域",
    "Refine mode allows for a more nuanced handling of overlapping genes.": "细化模式允许对重叠基因进行更细致的处理。",
    "The minimum number of nucleotides two genes must overlap to be grouped in the same iLocus": "两个基因必须重叠的最小核苷酸数，才能被分组到同一个iLocus中。",
    "Select the featured used to annotate intervals on a genome (e.g. gene, intron, exon).": "选择用于注释基因组上区间的特征（例如：基因、内含子、外显子）。",
    "If a feature of type $CT exists without a parent, create a parent for this feature with type $PT; for example, mRNA:gene will create a gene feature  as a parent for any top-level mRNA feature.": "如果存在一个类型为$CT的特征而没有父特征，则为这个特征创建一个类型为$PT的父特征；例如，mRNA:gene将为任何顶级mRNA特征创建一个基因特征作为其父特征。",
    "Retain original feature IDs from input files; conflicts will arise if input contains duplicated ID values": "保留输入文件中的原始特征ID；如果输入包含重复的ID值，将会产生冲突。",
    "Provide a format string to override the default ID format for newly created loci; default is \"locus%lu\" (locus1, locus2, etc) for loci and \"iLocus%lu\" (iLocus1, iLocus2, etc) for interval loci; note the format string should include a single %lu specifier to be filled in with a long unsigned integer value.": "提供一个格式字符串以覆盖新创建的位点的默认ID格式；位点的默认格式是\"locus%lu\"（例如：locus1, locus2等），间隔位点的默认格式是\"iLocus%lu\"（例如：iLocus1, iLocus2等）；请注意格式字符串中应包含一个单独的%lu指定符，用于填充一个长无符号整数值。",
    "Minimum contig size (default = 1; 200 in compliant mode) (--min-contig-length)": "最小连体大小（默认 = 1；合规模式下为 200）(--min-contig-length)",
    "ex. Escherichia": "大肠杆菌",
    "ex. 'coli O157:H7'": "例：'大肠杆菌 O157:H7'",
    "ex. Sakai": "例如：Sakai",
    "ex. pOSAK1": "例如 pOSAK1",
    "All sequences are complete replicons (chromosome/plasmid[s])": "所有序列都是完整的复制子（染色体/质粒[们]）",
    "Prodigal training file for CDS prediction": "用于编码序列预测的Prodigal训练文件",
    "Default is the bacterial table 11": "默认是细菌表11",
    "Fasta file of trusted protein sequences for CDS annotation": "用于CDS注释的可信蛋白序列的Fasta文件",
    "Run in metagenome mode. This only affects CDS prediction": "在宏基因组模式下运行。这只会影响编码序列（CDS）预测",
    "Regions only, no functional annotations.": "仅区域，无功能注释。",
    "If you want to use hmmsearch, select BUSCO mode. Otherwise, select lite mode": "如果您想使用hmmsearch，请选择BUSCO模式。否则，请选择lite模式。",
    "e.g. chr1 chr2 chr3. If not specified, all contigs will be evaluated": "例如：chr1 chr2 chr3。如果没有指定，将评估所有片段。",
    "For transcript features lacking an explicitly declared gene feature as a parent, create this feature on-they-fly": "对于缺少明确声明的基因特征作为父特征的转录特征，在运行时创建这个特征。",
    "Reset the source of each feature to the given value. Source keywords can be a program (e.g. RepeatMasker) or an organization (like TAIR)": "将每个特征的来源重置为给定的值。来源关键词可以是一个程序（例如 RepeatMasker）或一个组织（如 TAIR）",
    "e.g. Caenorhabditis": "例如，秀丽隐杆线虫",
    "Separated by underscore e.g. Caenorhabditis_elegans": "用下划线分隔，例如 Caenorhabditis_elegans",
    "Read and use the specified training file (-t)": "读取并使用指定的训练文件（-t）",
    "Closed ends. Do not allow partial genes at edges of sequence (-c)": "封闭末端。不允许序列边缘处有部分基因（-c）",
    "Do not use the Shine-Dalgarno RBS finder and instead force Prodigal to scan for motifs (-n)": "不要使用Shine-Dalgarno RBS查找器，而是强制Prodigal扫描模式（-n）",
    "Treat runs of N as masked sequence; don't build genes across them (-m)": "将连续的N视为掩蔽序列；不要在它们之间构建基因（-m）",
    "The percentage of introns confirmed by an alignment gap; for single-exon gene predictions lacking introns, it represents the ratio of the observed CDS length to the expected CDS length.": "通过比对间隙确认的内含子百分比；对于缺乏内含子的单外显子基因预测，它代表观察到的编码序列（CDS）长度与预期的CDS长度的比率。",
    "Overall percentage of individual exons spanned by at least one sequencing read.": "由至少一个测序读取覆盖的个体外显子的总体百分比。",
    "The ratio of the observed 5’ UTR length to the expected 5’ UTR length.": "观察到的5'UTR长度与预期的5'UTR长度之比。",
    "The ratio of the observed 3’ UTR length to the expected 3’ UTR length.": "观察到的3'UTR长度与预期的3'UTR长度之比。",
    "9606 is Homo sapiens (human)": "9606 是指智人（人类）的生物分类学编号。",
    "genomic is default": "基因组是默认的",
    "custom is default": "自定义是默认的",
    "hg38 is default": "hg38 是默认设置",
    "Homo sapiens is default": "智人是默认的",
    "Formatted like CoCoPUTs.": "格式化为CoCoPUTs。",
    "ORFs will be detected in this sequence": "在该序列中将检测到开放阅读框（ORFs）。",
    "DNA, RNA, and Peptide options will produce FASTA outputs": "DNA、RNA 和肽段选项将生成FASTA输出",
    "Identify ORFs on which strand": "鉴定在哪个链上的开放阅读框（ORFs）",
    "No ORFs below this value will be reported. All ORFs will be reported if this parameter is not set. Default is 30": "不会报告低于这个值的开放阅读框（ORFs）。如果未设置此参数，则会报告所有开放阅读框。默认值是30。",
    "This value doesn't take in account the STOP codon. No ORFs over this value will be reported. Default is 1.000.000.000.": "这个值没有考虑终止密码子。超过这个值的开放阅读框（ORF）将不会被报告。默认值是1,000,000,000。",
    "A comma-separated list without spaces. Only ATCG and comma are allowed": "一个由逗号分隔的列表，没有空格。只允许包含ATCG和逗号。",
    "A commma separated list without spaces. Only ATCG and comma are allowed": "一个由逗号分隔的列表，没有空格。只允许使用ATCG和逗号。",
    "Ignore case and find ORFs in lower case sequences too. NO = do not ignore (use upper case only). YES = ignore": "忽略大小写，并在小写序列中也查找开放阅读框（ORF）。NO = 不忽略（仅使用大写）。YES = 忽略",
    "Output ORFs with a start codon but lacking an inframe stop codon. NO = do not output. YES = do output": "输出带有起始密码子但缺少一个同义终止密码子的开放阅读框。NO = 不输出。YES = 输出。",
    "Output ORFs with an inframe stop codon lacking an inframe start codon. NO = do not output. YES = do output": "输出带有非框内终止密码子但缺少非框内起始密码子的开放阅读框（ORF）。NO = 不输出。YES = 输出。",
    "Output ORFs defined as regions between stop codons (regions free of stop codon). This will set --partial-3 and --partial-5 true": "输出由终止密码子（无终止密码子的区域）之间的区域定义的开放阅读框（ORFs）。这将设置 --partial-3 和 --partial-5 为 true",
    "Output ORFs with an inframe stop codon lacking an inframe start codon. NO = do not include. YES = include": "输出带有非框内终止密码子但缺少非框内起始密码子的开放阅读框（ORF）。NO = 不包括。YES = 包括。",
    "Input data file matching the input type.": "输入数据文件与输入类型匹配。",
    "Input data type, default fasta:pro": "输入数据类型，默认为fasta:pro",
    "KOBAS organism database, available at http://kobas.cbi.pku.edu.cn/download.php": "KOBAS 物种数据库，可在 http://kobas.cbi.pku.edu.cn/download.php 获取",
    "Expect threshold for BLAST, default 1e-5": "BLAST的期望阈值，默认为1e-5",
    "Rank cutoff for valid hits from BLAST result, default 5": "BLAST结果中有效命中的排名截止值，默认为5",
    "subject coverage cutoff for BLAST, default 0": "BLAST的主题覆盖度截止值，默认为0",
    "Whether only use orthologs for cross-species annotation or not, default NO (if only use orthologs, please provide the species abbreviation of your input)": "是否仅使用同源基因进行跨物种注释，默认为否（如果仅使用同源基因，请提供您输入的物种缩写）",
    "Foreground file, the output of KOBAS Annotate": "前景文件，KOBAS 注释的输出",
    "Select your desired databases: (Note: the Corrected P-Values will be affected by the number of selected databases)": "选择您想要的数据库：（注意：所选数据库的数量将影响校正后的 P 值）",
    "Optional background file, the output of annotate (3 or 4-letter file name is not allowed), or species abbreviation (for example: hsa for Homo sapiens, mmu for Mus musculus, dme for Drosophila melanogaster, ath for Arabidopsis thaliana, sce for Saccharomyces cerevisiae and eco for Escherichia coli K-12 MG1655), default same species as annotate": "可选的背景文件，annotate的输出（不允许使用3或4个字母的文件名），或物种缩写（例如：hsa代表智人，mmu代表小鼠，dme代表果蝇，ath代表拟南芥，sce代表酿酒酵母，eco代表大肠杆菌K-12 MG1655），默认与annotate相同的物种",
    "For example: ko for KEGG Orthology, hsa for Homo sapiens, mmu for Mus musculus, dme for Drosophila melanogaster, ath for Arabidopsis thaliana, sce for Saccharomyces cerevisiae and eco for Escherichia coli K-12 MG1655": "例如：ko 代表 KEGG 正交性，hsa 代表智人，mmu 代表小鼠，dme 代表果蝇，ath 代表拟南芥，sce 代表酿酒酵母，eco 代表大肠杆菌 K-12 MG1655",
    "False discovery rate (FDR) correction method": "错误发现率（FDR）校正方法",
    "Terms with less than cutoff number of genes are not used for statistical tests, default 5": "基因数量少于截止值的术语不用于统计测试，默认为5",
    "If you do not want to keep the default value, please enter the new value. Default values are 21384 for fungi, 106920 for land plant, and 213840 for vertebrates and invertebrates": "如果您不想保留默认值，请输入新值。真菌的默认值为21384，陆生植物的默认值为106920，脊椎动物和无脊椎动物的默认值为213840。",
    "This step combines predictions made on each subsequences, to improve quality near start and end of subsequences.": "这一步将对每个子序列所做的预测进行组合，以提高子序列起始和结束附近的质量。",
    "If you do not want to keep the default value, please enter the new value. Default values are 10692 for fungi, 53460 for land plant, and 106920 for vertebrates and invertebrates.": "如果您不想保留默认值，请输入新值。真菌的默认值为10692，陆生植物的默认值为53460，脊椎动物和无脊椎动物的默认值为106920。",
    "If you do not want to keep the default value, please enter the new value. Default values are 16038 for fungi, 80190 for land plant, and 160380 for vertebrates and invertebrates.": "如果您不想保留默认值，请输入新值。真菌的默认值为16038，陆生植物的默认值为80190，脊椎动物和无脊椎动物的默认值为160380。",
    "The 'Comparison' mode currently works for exactly two groups of samples. Enter comma-separated group names - one for each sample. The group names will be assigned to the samples in the order of the sample columns of the counts data.": "“比较”模式目前仅适用于恰好两组样本。输入以逗号分隔的组名 - 每个样本一个。组名将按照计数数据的样本列顺序分配给样本。",
    "Genomic regions (e.g. RBP binding sites) in BED format (>= 6-columns) for RBP binding motif search": "基因组区域（例如RBP结合位点）以BED格式（至少6列）用于RBP结合位点模式搜索",
    "Set ID / name for provided custom motif database (default: custom_db_id)": "为提供的自定义基序数据库设置ID/名称（默认：custom_db_id）",
    "Provide custom motif database info table file containing RBP ID -> motif ID -> motif type assignments. The motif IDs must correspond to the provided custom MEME / DREME XML and/or covariance models file(s).": "提供自定义的基序数据库信息表文件，该文件包含RBP ID（RNA结合蛋白ID）到基序ID，再到基序类型的分配。基序ID必须与提供的自定义MEME/DREME XML文件和/或共轭模型文件相对应。",
    "Provide custom motif database MEME / DREME XML file containing sequence motifs": "提供自定义的MEME/DREME XML文件，其中包含序列基序",
    "Provide custom motif database covariance models file containing structure motif(s) (i.e., covariance model(s))": "提供包含结构基序（即，协方差模型）的自定义基序数据库协方差模型文件",
    "Provide list of RBP IDs found in custom motif database to use for motif search. This can be a single ID, as well as several (separated by spaces, as in the example). To search using all RBPs in the custom database, simply type ALL.": "提供在自定义基序数据库中找到的RBP ID列表，以用于基序搜索。这可以是单个ID，也可以是多个（由空格分隔，如示例所示）。要使用自定义数据库中的所有RBP进行搜索，只需键入ALL。",
    "DREME or MEME output XML file containing sequence motif(s)": "DREME或MEME输出的XML文件包含序列基序",
    "RBP ID (RBP name) for provided sequence motif(s). Make sure to provide user RBP IDs and motif IDs that are different from selected database RBP and motif IDs.": "提供的序列基序的RBP ID（RBP名称）。请确保提供的RBP ID和基序ID与选定的数据库RBP和基序ID不同。",
    "Covariance models file containing structure motif(s)": "包含结构基序的协方差模型文件",
    "RBP ID (RBP name) for provided structure motif(s). Make sure to provide user RBP IDs and motif IDs that are different from selected database RBP and motif IDs.": "RBP ID（RBP名称）用于提供的结构基序。请确保提供的RBP ID和基序ID与选定的数据库中的RBP和基序ID不同。",
    "Method ID which can be used to describe the peak calling method (e.g. clipper_idr). This ID (together with data ID and set RBP ID(s)) defines which search results get compared in RBPBench's comparison mode (see Help below for more details).": "方法ID，可以用来描述峰值检测方法（例如clipper_idr）。这个ID（与数据ID和一组RBP ID一起）定义了在RBPBench比较模式中要比较哪些搜索结果（更多详细信息请参见下面的帮助）。",
    "Data ID which can be used to describe from which cell type and/or CLIP-seq protocol the data originates (e.g. k562_eclip or pum2_k562_eclip). This ID (together with method ID and set RBP ID(s)) defines which search results get compared in RBPBench's comparison mode (see Help below for more details).": "数据ID，可用于描述数据来自哪种细胞类型和/或CLIP-seq协议（例如k562_eclip或pum2_k562_eclip）。此ID（与方法ID和RBP ID（s）一起）定义了在RBPBench的比较模式中比较哪些搜索结果（有关更多详细信息，请参见下面的帮助）。",
    "Up- and downstream extension of genomic regions in nucleotides (nt). E.g. set to \"30\" to extend 30 nt on both sides, or \"20,10\" for different up- and downstream extension (default: 0)": "在核苷酸（nt）中，基因组区域的上游和下游扩展。例如，设置为“30”以在两侧各扩展30个核苷酸，或“20,10”以进行不同的上游和下游扩展（默认值：0）。",
    "FIMO p-value threshold (FIMO option: --thresh) for reporting motif hits (default: 0.001)": "FIMO 显著性阈值（FIMO 选项：--thresh）用于报告基序命中（默认值：0.001）",
    "Score column of genomic regions BED file used for p-value calculations. BED score can be e.g. log2 fold change or -log10 p-value of the region (default: 5)": "用于计算 p 值的基因组区域 BED 文件中的分数列。BED 分数可以是例如对数比值变化或区域的 -log10 p 值（默认值：5）",
    "Set if genomic regions in BED file are NOT strand-specific, i.e., to look for motifs on both strands of the provided regions. Note that the two strands of a region will still be counted as one region (change with option below) (default: False)": "如果BED文件中的基因组区域不是特定的链，即在提供的区域的两个链上寻找基序，则设置此选项。请注意，一个区域的两个链仍将被计算为一个区域（可以通过下面的选项更改）（默认值：False）",
    "Count each genomic region twice for RBP hit statistics when non-strand-specific option above is enabled (default: False)": "在启用上述非链特异性选项时，对RBP命中统计的每个基因组区域计数两次（默认：False）",
    "Provide FIMO nucleotide frequencies (FIMO option: --bifile) file. By default, an internal frequencies file optimized for human transcripts is used": "提供 FIMO 核苷酸频率文件（FIMO 选项：--bifile）。默认情况下，使用针对人类转录本优化的内部频率文件。",
    "Provide GTF file with genomic annotations to add to HTML report plots (e.g. from GENCODE or Ensembl). By default, the most prominent transcripts will be extracted and used for functional annotation. Alternatively, provide a list of expressed transcripts via --tr-list option (together with --gtf containing the transcripts). Note that currently only features on standard chromosomes (1,2,..,X,Y,MT) are used for annotation": "提供带有基因组注释的GTF文件以添加到HTML报告图表中（例如来自GENCODE或Ensembl）。默认情况下，将提取最突出的转录本并用于功能注释。或者，通过--tr-list选项提供表达转录本的列表（与包含转录本的--gtf一起）。请注意，目前仅使用标准染色体（1,2,..,X,Y,MT）上的特征进行注释。",
    "Supply file with transcript IDs (one ID per row) to define which transcripts to use from GTF file for adding functional annotations to HTML report plots": "提供包含转录本ID的文件（每行一个ID），以定义要从GTF文件中使用哪些转录本来为HTML报告图表添加功能注释。",
    "List of transcript biotypes to consider from GTF file. By default an internal selection of transcript biotypes is used (in addition to intron, CDS, UTR, intergenic). Provide a list of IDs separated by spaces. Note that provided biotype IDs need to be in GTF file!": "从GTF文件中考虑的转录本生物类型列表。默认情况下，除了内含子、编码序列(CDS)、非翻译区(UTR)、基因间区之外，还会使用内部选择的转录本生物类型。请提供由空格分隔的ID列表。请注意，提供的生物类型ID必须在GTF文件中存在！",
    "Upset plot minimum degree parameter for HTML report upset plot. This defines the minimum number of RBPs for a combination to be included (default: 2)": "Upset图最小度参数，用于HTML报告中的Upset图。这定义了一个组合要被包含的最小RBPs数量（默认值：2）",
    "Upset plot maximum degree parameter for HTML report upset plot. By default no maximum degree is set. Useful together with minimum degree to look at specific degrees (e.g. only 2, or between 2 and 3) (default: None)": "Upset图最大度参数，用于HTML报告中的Upset图。默认情况下，不设置最大度。与最小度一起使用时，可以查看特定度数（例如只有2，或在2和3之间）（默认值：无）",
    "Upset plot minimum subset size parameter for HTML report upset plot. This defines the minimum number of hits for a specific RBP combination to be included (default: 5)": "HTML报告中upset图的最小子集大小参数。这定义了一个特定的RBP组合被包含的最小命中数（默认值：5）",
    "Set reference RBP ID to plot motif distances relative to motifs from this RBP (--set-rbp-id). Motif plot will be centered on best scoring motif of the RBP for each region. Note that set RBP ID needs to be one of the above selected RBP IDs!": "将参考RBP ID设置为相对于此RBP的基序绘制基序距离（--set-rbp-id）。基序图将以每个区域的最佳得分基序为中心。请注意，设置的RBP ID需要是上述选定的RBP ID之一！",
    "Set range of motif distance plot. I.e., centered on the set RBP (--set-rbp-id) motifs, motifs within minus and plus --motif-distance-plot-range will be plotted (default: 60)": "设置基序距离图的范围。即，以设定的RBP（--set-rbp-id）基序为中心，将绘制距离减去和加上--motif-distance-plot-range的基序（默认值：60）",
    "Minimum count of co-occurrences of a motif with set RBP ID (--set-rbp-id) motif to be reported and plotted (default: 10)": "报告和绘制具有设置RBP ID（--set-rbp-id）的基序的最小共现计数（默认：10）",
    "Minimum amount of co-occurrences of motifs for an RBP ID compared to set RBP ID (--set-rbp-id) motifs to be reported and plotted (default: 10)": "对于一个RBP ID与设定的RBP ID (--set-rbp-id)的基序共现的最小数量，以便报告和绘制（默认值：10）",
    "Store JavaScript code for table sorting inside output HTML files. By default code is stored locally in extra file located in HTML output folder.": "将表格排序的 JavaScript 代码存储在输出的 HTML 文件中。默认情况下，代码会存储在 HTML 输出文件夹中的一个额外的本地文件里。",
    "1: Store plotly JavaScript code locally inside HTML output folder. 2: Add hyperlink to report HTML file (internet connection required). 3: Store code inside report HTML file (default: 1)": "1: 将 plotly JavaScript 代码本地存储在 HTML 输出文件夹中。2: 添加到报告 HTML 文件的超链接（需要互联网连接）。3: 将代码存储在报告 HTML 文件中（默认：1）",
    "Store plotly plotting code inside HTML. By default code is stored in separate HTML files in HTML report output folder.": "将 Plotly 绘图代码存储在 HTML 中。默认情况下，代码存储在 HTML 报告输出文件夹中的单独 HTML 文件中。",
    "Generate an HTML report containing RBP co-occurrence + combination + distance statistics and plots (default: True)": "生成一个包含RBP共现+组合+距离统计和图表的HTML报告（默认：True）",
    "Visualize selected RBP motifs, by outputting sequence logos and motif hit statistics into a separate HTML file (default: False)": "可视化选定的RBP基序，通过将序列标志和基序命中统计输出到一个单独的HTML文件中（默认：False）",
    "Output filtered genomic regions BED/FASTA file used for motif search. Filtered means that the actual regions used for motif search can differ from the input genomic regions, e.g. through default filtering by chromsome ID (only regions with valid IDs), removal of duplicated regions, or through optional extension of the regions": "输出用于基序搜索的过滤后的基因组区域 BED/FASTA 文件。过滤意味着实际用于基序搜索的区域可能与输入的基因组区域不同，例如，通过默认的染色体 ID 过滤（只有具有有效 ID 的区域），删除重复区域，或通过可选的区域扩展。",
    "Output motif hits BED file containing motif hits in provided genomic regions for selected RBPs": "输出基序命中的BED文件，包含所提供基因组区域中选定的RBPs（RNA结合蛋白）的基序命中。",
    "Output contingency table containing co-occurrence p-values (Fisher's exact test) between each RBP pair (see manual for more information)": "输出包含共现p值（费舍尔精确检验）的列联表，用于分析每对RBP（RNA结合蛋白）之间的关系（有关更多信息，请参见手册）。",
    "Output genomic region annotations table file containing assigned annotations for each BED input region. Note that a GTF file has to be provided (default: False)": "输出包含每个BED输入区域分配注释的基因组区域注释表文件。请注意，必须提供GTF文件（默认：False）",
    "Method ID which can be used to describe the peak calling method (e.g. clipper_idr). This ID (together with data ID and set RBP ID) defines which search results get compared in RBPBench's comparison mode (see Help below for more details).": "方法ID，可以用来描述峰值调用方法（例如clipper_idr）。这个ID（连同数据ID和集合RBP ID）定义了在RBPBench的比较模式中比较哪些搜索结果（更多详细信息请参见下面的帮助）。",
    "Data ID which can be used to describe from which cell type and/or CLIP-seq protocol the data originates (e.g. k562_eclip or pum2_k562_eclip). This ID (together with method ID set RBP ID) defines which search results get compared in RBPBench's comparison mode (see Help below for more details).": "数据ID，可用于描述数据来源于哪种细胞类型和/或CLIP-seq协议（例如k562_eclip或pum2_k562_eclip）。此ID（与方法ID设置RBP ID）定义了在RBPBench的比较模式中要比较的搜索结果（有关更多详细信息，请参见下面的帮助）。",
    "Score column of genomic regions BED files used for p-value calculations. BED score can be e.g. log2 fold change or -log10 p-value of the region (default: 5)": "用于 p 值计算的基因组区域 BED 文件的得分列。BED 得分可以是例如区域的 log2 倍数变化或 -log10 p 值（默认值：5）",
    "Set if genomic regions in BED files are NOT strand-specific, i.e., to look for motifs on both strands of the provided regions. Note that the two strands of a region will still be counted as one region (change with option below) (default: False)": "设置如果BED文件中的基因组区域不是特定于链的，即在提供的区域的两个链上寻找基序。请注意，一个区域的两个链仍然将被计算为一个区域（可以通过下面的选项更改）（默认值：False）",
    "Output motif hits BED file containing motif hits for all input datasets": "输出包含所有输入数据集的基序命中的BED文件",
    "Data collection containing genomic regions BED files to be processed. Note that dataset names inside collection must correspond to names given in the batch processing table file below": "包含基因组区域BED文件的数据集，将被处理。请注意，在集合内部的数据集名称必须与下面批量处理表格文件中给出的名称相对应。",
    "Provide batch processing table file with one row for each batch job. Each row contains the tab-delimited information: RBP ID (RBP name), method ID, data ID, dataset name. The dataset name must be present in the supplied data collection of BED files": "提供批量处理表格文件，每个批次作业一行。每行包含制表符分隔的信息：RBP ID（RBP名称）、方法ID、数据ID、数据集名称。数据集名称必须存在于提供的BED文件数据集中",
    "Genomic regions (e.g. RBP binding sites) in BED format. Zero position for plotting can be defined in options": "基因组区域（例如RBP结合位点）以BED格式。绘图时零点位置可以在选项中定义。",
    "Define which position of genomic sites to use as zero position for plotting. 1: upstream end position, 2: center position, 3: downstream end position (default: 1)": "定义用于绘图的基因组位点的哪个位置作为零点位置。1：上游端点位置，2：中心位置，3：下游端点位置（默认：1）",
    "Up- and downstream extension of defined genomic positions in nucleotides (nt) to include in plotting (default: 10)": "在核苷酸（nt）中，定义的基因组位置的上游和下游扩展以包含在绘图中（默认值：10）",
    "Plot nucleotide distribution as PDF (default: PNG)": "绘制核苷酸分布图，以PDF格式（默认：PNG）",
    "Output genomic regions BED/FASTA file used for plotting": "输出用于绘图的基因组区域 BED/FASTA 文件",
    "Supply motif search results table files for comparison. These are the hit statistics table files output by single or batch motif search jobs. Both RBP and motif hit statistics table files are needed, and can be from any single or batch search job.": "提供用于比较的motif搜索结果表文件。这些是由单个或批量motif搜索作业输出的命中统计表文件。需要RBP和motif命中统计表文件，可以来自任何单个或批量搜索作业。",
    "Output compared motif hits table file showing motif hits and which data or method IDs contain them": "输出比较基序命中表格文件，显示基序命中以及包含它们的数据或方法ID",
    "Output compared motif hits table file showing motif hits and which data or method IDs contain them in BED format": "输出比较基序命中表格文件，显示基序命中情况以及包含它们的数据或方法ID，采用BED格式。",
    "Output comparison statistics table file containing the statistics found in the comparison HTML report": "输出包含比较HTML报告中发现的统计信息的比较统计表文件",
    "If checked and if multisamples is set to average or sum, normalization is performed before combining the signal from multiple samples": "如果选中，并且multisamples设置为average（平均）或sum（求和），则在将多个样本的信号组合之前执行标准化处理。",
    " This parameter is considered only if there exatcly two input samples": "如果只有两个输入样本，则只考虑此参数",
    "This can be the known fusions if that input has a third column with a name": "如果输入包含一个带有名称的第三列，这可以是已知的融合。",
    "These coordinates serve to increase sensitivity towards weakly expressed fusions and to eliminate fusions with low evidence.": "这些坐标有助于提高对表达较弱融合蛋白的敏感性，并排除证据不足的融合蛋白。",
    "determines how far a genomic breakpoint may be away from a transcriptomic breakpoint to consider it as a related event.": "确定基因组断裂点与转录组断裂点之间的最大距离，以将其视为相关事件。",
    "The multi-FASTA file can also be a fasta file.": "多FASTA文件也可以是一个fasta文件。",
    "The set of aligned reads.": "对齐的读取集。",
    "0 is default. Can improve accuracy at the cost of time.": "0是默认值。可以以时间的代价提高准确性。",
    "This calculation requires slightly more time and memory.": "这个计算需要稍微多一点的时间和内存。",
    "200 is default. While the empirical distribution is estimated from paired-end reads on-the-fly, this value paramaterizes the prior distribution.": "200是默认值。虽然经验分布是从成对末端读取实时估计的，但这个值参数化了先验分布。",
    "60 is default. While the empirical distribution is estimated from paired-end reads on-the-fly, this value paramaterizes the prior distribution.": "60是默认值。虽然经验分布是从成对末端读取实时估计的，但这个值参数化了先验分布。",
    "Requires an annotation file in GFF3 or GTF format.": "需要一个GFF3或GTF格式的注释文件。",
    "Discard isoforms with abundance below this value": "丢弃丰度低于此值的异构体",
    "ignore alignments with gaps longer than this": "忽略间隔超过此长度的比对",
    "suppress transcripts below this abundance level": "抑制低于此丰度水平的转录本",
    "suppress intra-intronic transcripts below this level": "抑制低于此水平的内含子内转录本",
    "Gene annotation dataset in GTF or GFF3 format.": "基因注释数据集，格式为GTF或GFF3。",
    "With this option, Cufflinks counts only those fragments compatible with some reference transcript towards the number of mapped hits used in the FPKM denominator. This option can only be used in combination with --GTF.": "使用这个选项，Cufflinks 只计算与某些参考转录本兼容的片段，以用于 FPKM 分母中的映射命中数。这个选项只能与 --GTF 组合使用。",
    "The number of bp allowed to overhang the 3prime end of a reference transcript when determining if an assembled transcript should be merged with it (ie, the assembled transcript is not novel). The default is 600 bp.": "在确定组装的转录本是否应该与参考转录本合并时（即，组装的转录本不是新的），允许超出参考转录本3'端的碱基对数。默认值是600个碱基对。",
    "The number of bp allowed to enter the intron of a reference transcript when determining if an assembled transcript should be merged with it (ie, the assembled transcript is not novel). The default is 50 bp.": "在确定组装的转录本是否应该与参考转录本合并时（即，组装的转录本不是新的），允许进入参考转录本内含子的碱基对数（bp）。默认值是50个碱基对。",
    "This option disables tiling of the reference transcripts with faux reads. Use this if you only want to use sequencing reads in assembly but do not want to output assembled transcripts that lay within reference transcripts. All reference transcripts in the input annotation will also be included in the output.": "这个选项禁用了使用假读段对参考转录本进行平铺。如果您只想在组装中使用测序读段，但不想输出位于参考转录本内的组装转录本，请使用此选项。输入注释中的所有参考转录本也将包含在输出中。",
    "Bias detection and correction can significantly improve accuracy of transcript abundance estimates.": "偏差检测和校正可以显著提高转录本丰度估计的准确性。",
    "Tells Cufflinks to do an initial estimation procedure to more accurately weight reads mapping to multiple locations in the genome.": "告诉Cufflinks进行初步估计程序，以更准确地对映射到基因组多个位置的读取进行加权。",
    "Mode of length normalization to transcript FPKM.": "转录本FPKM长度标准化的模式。",
    "Ignore all alignment within transcripts in this file": "忽略此文件中所有转录本内的比对",
    "This is the expected (mean) inner distance between mate pairs. For, example, for paired end runs with fragments selected at 300bp,where each end is 50bp, you should set it as 200. The default is 45bp.": "这是预期的（平均）配对末端之间的内部距离。例如，对于在300bp处选择片段的配对末端运行，其中每个末端是50bp，您应该将其设置为200。默认值是45bp。",
    "The standard deviation for the distribution on inner distances between mate pairs. The default is 20bp.": "内配对间距离分布的标准差，默认值为20bp。",
    "Sets the number of iterations allowed during maximum likelihood estimation of abundances. Default: 5000": "设置在丰度的最大似然估计中允许的迭代次数。默认值：5000",
    "Default: 0.001": "默认值：0.001",
    "Spliced reads with less than this percent of their length on each side of the junction are considered suspicious and are candidates for filtering prior to assembly. Default: 0.09.": "在连接点两侧，拼接的读取长度小于这个百分比的被认为是可疑的，并且在组装之前是过滤的候选对象。默认值：0.09。",
    "The number of bp allowed to enter the intron of a transcript when determining if a read or another transcript is mappable to/compatible with it. The default is 8 bp based on the default bowtie/TopHat parameters.": "在确定一个读段或另一个转录本是否可以映射到/兼容于转录本内含子时，允许进入内含子的碱基对数。默认值是8个碱基对，基于bowtie/TopHat软件的默认参数。",
    "Default: 3,500,000bp": "默认值：3,500,000个碱基对",
    "Sets the maximum number of fragments a locus may have before being skipped. Skipped loci are listed in skipped.gtf. Default: 1,000,000": "设置一个位点在被跳过之前可能拥有的最大片段数。被跳过的位点会列在skipped.gtf文件中。默认值：1,000,000",
    "Default: 50bp": "默认值：50bp",
    "Default: 0.1": "默认值：0.1",
    "A transcript GFF3 or GTF file produced by cufflinks, cuffcompare, or other source.": "由cufflinks、cuffcompare或其他来源生成的转录本GFF3或GTF文件。",
    "Discard the tabular output.": "丢弃表格输出。",
    "Generate a SQLite database for use with cummeRbund.": "生成一个SQLite数据库，用于与cummRbund一起使用。",
    "If using only one sample per condition, you must use 'blind.'": "如果每个条件只使用一个样本，你必须使用'blind'。",
    "The allowed false discovery rate.": "允许的假发现率。",
    "The minimum number of alignments in a locus for needed to conduct significance testing on changes in that locus observed between samples.": "在位点中进行该位点样本间变化显著性检验所需的最小比对数。",
    "Read group datasets provide information on replicates.": "读取组数据集提供了关于复制品的信息。",
    "Cuffdiff estimates the number of fragments that originated from each transcript, primary transcript, and gene in each sample. Primary transcript and gene counts are computed by summing the counts of transcripts in each primary transcript group or gene group.": "Cuffdiff 估计每个样本中每个转录本、主要转录本和基因起源的片段数量。主要转录本和基因计数是通过将每个主要转录本组或基因组中的转录本计数相加得出的。",
    "mode of length normalization to transcript fpkm.": "转录本FPKM长度标准化的方式。",
    "Instructs Cuffdiff to analyze the provided samples as a time series, rather than testing for differences between all pairs of samples. Samples should be provided in increasing time order at the command line (e.g first time point SAM, second timepoint SAM, etc.)": "指导Cuffdiff将提供的样本作为时间序列进行分析，而不是测试所有样本对之间的差异。样本应在命令行中按时间顺序递增提供（例如，第一个时间点的样本SAM，第二个时间点的样本SAM等）。",
    "Maximum iterations allowed for Maximal Likelyhood Estimation calculations": "最大似然估计计算允许的最大迭代次数",
    "All Hits: With this option, Cufflinks counts all fragments, including those not compatible with any reference transcript, towards the number of mapped fragments used in the FPKM denominator. Compatible Hits: With this option, Cufflinks counts only those fragments compatible with some reference transcript towards the number of mapped fragments used in the FPKM denominator. Using this mode is generally recommended in Cuffdiff to reduce certain types of bias caused by differential amounts of ribosomal reads which can create the impression of falsely differentially expressed genes. It is active by default.": "所有命中：使用此选项，Cufflinks计算所有片段，包括与任何参考转录本不兼容的片段，用于FPKM分母中的已映射片段数。兼容命中：使用此选项，Cufflinks只计算与某些参考转录本兼容的片段，用于FPKM分母中的已映射片段数。通常建议在Cuffdiff中使用此模式，以减少由核糖体读取量的差异引起的某些类型的偏差，这可能会造成假性差异表达基因的印象。默认情况下它是激活的。",
    "Sets the maximum number of fragments a locus may have before being skipped. Skipped loci are listed in skipped.gtf. Default: 500,000": "设置一个位点在被跳过之前可能拥有的最大片段数。被跳过的位点会列在skipped.gtf文件中。默认值：500,000",
    "Cuffdiff will make this many draws from each transcript's predicted negative binomial random numbder generator. Each draw is a number of fragments that will be probabilistically assigned to the transcripts in the transcriptome. Used to estimate the variance-covariance matrix on assigned fragment counts. Default: 100.": "Cuffdiff 将从每个转录本的预测负二项分布随机数生成器中进行这么多次抽取。每次抽取是将要概率性地分配给转录组中转录本的片段数。用于估计分配片段计数的方差-协方差矩阵。默认值：100。",
    "For each fragment drawn from a transcript, Cuffdiff will assign it this many times (probabilistically), thus estimating the assignment uncertainty for each transcript. Used to estimate the variance-covariance matrix on assigned fragment counts. Default: 50.": "对于从转录本中抽取的每个片段，Cuffdiff会以这种方式（概率性地）分配给它这么多次，从而估计每个转录本的分配不确定性。用于估计分配片段计数的方差-协方差矩阵。默认值：50。",
    "Cuffdiff won't test genes for differential regulation unless the conditions in question have at least this many replicates. Default: 3.": "Cuffdiff 不会在条件中测试基因的差异化调控，除非所讨论的条件至少有这么多的重复。默认值：3。",
    "consider only the reference transcripts that overlap any of the input transfrags (Sn correction)": "只考虑与输入的转录片段（transfrags）有重叠的参考转录本（Sn校正）。",
    "consider only the input transcripts that overlap any of the reference transcripts (Sp correction). Warning: this will discard all 'novel' loci!": "只考虑与参考转录本重叠的输入转录本（Sp校正）。警告：这将丢弃所有“新”位点！",
    "max. distance (range) allowed from free ends of terminal exons of reference transcripts when assessing exon accuracy. Default: 100": "最大距离（范围）允许从参考转录本末端外显子的自由端评估外显子准确性时。默认值：100",
    "max. distance (range) for grouping transcript start sites. Default: 100": "最大距离（范围）用于对转录起始位点进行分组。默认值：100",
    "Discard intron-redundant transfrags if they share the 5' end (if they differ only at the 3' end)": "丢弃内含子冗余的转录片段，如果它们共享5'端（如果它们只在3'端不同）",
    "Only letters, numbers and underscores will be retained in this field": "在这个字段中，只保留字母、数字和下划线",
    "If this option is set to Yes, the tool will assume that the count files have column headers in the first row. Default: Yes": "如果将此选项设置为Yes，工具将假定计数文件在第一行具有列标题。默认值：Yes",
    "output an additional PDF files": "输出额外的PDF文件",
    "If this option is set to Yes, the tool will generate RUVseq normalized count files. Default: No": "如果将此选项设置为Yes，工具将生成RUVseq标准化计数文件。默认值：No",
    "Stringtie or Cufflinks output": "Stringtie 或 Cufflinks 输出",
    "The input alignment file(s) where the gene expression has to be counted. The file can have a SAM or BAM format; but ALL files must be in the same format. Unless you are using a Gene annotation file from the History, these files must have the database/genome attribute already specified e.g. hg38, not the default: ?": "输入的比对文件（或文件们），需要计算基因表达的地方。文件可以是SAM或BAM格式；但所有文件必须使用相同的格式。除非你使用的是来自历史记录的基因注释文件，这些文件必须已经指定了数据库/基因组属性，例如hg38，而不是默认的：？",
    "Indicate if the data is stranded and if strand-specific read counting should be performed. Strand setting must be the same as the strand settings used to produce the mapped BAM input(s)": "请指明数据是否是链特异性的，以及是否应该执行链特异性的读数计数。链设置必须与用于生成映射的BAM输入的链设置相同。",
    "Built-in gene annotations for genomes hg38, hg19, mm10 and mm9 are included in featureCounts": "内置基因注释对于基因组hg38、hg19、mm10和mm9已包含在featureCounts中",
    "If the annotation file you require is not listed here, please contact the Galaxy administrator.": "如果您需要的注释文件在此未列出，请与Galaxy管理员联系。",
    "The program assumes that the provided annotation file is in GFF/GTF format. Make sure that the gene annotation file corresponds to the same reference genome as used for the alignment.": "该程序假定提供的注释文件是GFF/GTF格式的。请确保基因注释文件与用于比对的参考基因组相对应。",
    "The output format will be tabular, select the preferred columns here": "输出格式将是表格形式的，在这里选择您喜欢的列",
    "Creates a tabular file that contains the effective (nucleotides used for counting reads) length of the feature; might be useful for estimating FPKM/RPKM": "创建一个表格文件，其中包含特征的有效（用于计数读取的核苷酸）长度；可能对估计FPKM/RPKM有用",
    "Were the bam files generated by aligning the output of a paired-end sequencing experiment? If yes, the tool can consider 2 reads = 1 read pair as 1 entity to count. Alternatively, you can opt to consider treating the read pairs as 2 individual reads to count seperately.": "这些BAM文件是否由配对末端测序实验的输出对齐生成？如果是，该工具可以将2个读取视为1个读取对作为1个实体进行计数。或者，您可以选择将读取对视为2个单独的读取分别进行计数。",
    "If specified, only fragments that have both ends successfully aligned will be considered for summarization. This option is only applicable for paired-end reads.": "如果指定，只有两端都成功比对的片段才会被考虑进行汇总。这个选项只适用于成对的读取。",
    "If specified, the chimeric fragments (those fragments that have their two ends aligned to different chromosomes) will NOT be included for summarization. This option is only applicable for paired-end read data.": "如果指定了，嵌合片段（那些两端分别比对到不同染色体的片段）将不会被包括在汇总中。这个选项只适用于成对读取数据。",
    "If specified, paired-end distance will be checked when assigning fragments to meta-features or features. This option is only applicable when -p (Count fragments instead of reads) is specified. The distance thresholds can be specified using -d and -D (minimum and maximum fragment/template length) options.": "如果指定了，当将片段分配给元特征或特征时，将检查配对末端距离。此选项仅在指定了-p（计数片段而不是读取）时适用。可以使用-d和-D（最小和最大片段/模板长度）选项指定距离阈值。",
    "The minimum mapping quality score a read must satisfy in order to be counted. For paired-end reads, at least one end should satisfy this criteria. 0 by default.": "为了使一个读段被计数，它必须满足的最小映射质量分数。对于成对末端读段，至少有一个末端应该满足这个标准。默认值为0。",
    "Split alignments are alignments with CIGAR string containing 'N', e.g. exon spanning reads in RNASeq.": "拆分比对是指CIGAR字符串中包含'N'的比对，例如RNASeq中的外显子跨越读取。",
    "If specified, only primary alignments will be counted. Primary and secondary alignments are identified using bit 0x100 in theFlag field of SAM/BAM files. All primary alignments in a dataset will be counted regardless of whether they are from multi-mapping reads or not ('-M' is ignored).": "如果指定，仅计数主对齐。主对齐和次对齐是通过使用SAM/BAM文件Flag字段中的位0x100来识别的。无论它们是否来自多映射读取，都将计数数据集中的所有主对齐（'-M'被忽略）。",
    "If specified, reads that were marked as duplicates will be ignored. Bit Ox400 in the FLAG field of a SAM/BAM file is used for identifying duplicate reads. In paired end data, the entire read pair will be ignored if at least one end is found to be a duplicate read.": "如果指定了，被标记为重复的读取将被忽略。在SAM/BAM文件的FLAG字段中，使用位Ox400来识别重复读取。在成对末端数据中，如果至少一端被判定为重复读取，则整个读取对将被忽略。",
    "Setting -O, -M and --fraction": "设置 -O, -M 和 --fraction",
    "If specified, a fractional count 1/x will be generated for each multi-mapping read, where x is the number of alignments (indicated by 'NH' tag) reported for the read.": "如果指定了，对于每个多比对的读取，将生成一个分数计数1/x，其中x是为该读取报告的比对数量（由'NH'标签指示）。",
    "If specified, a fractional count 1/y will be generated for each multi-overlapping feature, where y is the number of features overlapping with the read.": "如果指定了，对于每个多重重叠的特征，将生成一个分数计数1/y，其中y是与读段重叠的特征数量。",
    "If specified, a fractional count 1/(x*y) will be generated, where x is the number of alignments (indicated by 'NH' tag) and y the number of overlapping features.": "如果指定了，将生成一个分数计数1/(x*y)，其中x是比对数（由'NH'标签指示），y是重叠特征数。",
    "Junctions are identified from those exon-spanning reads (containing ‘N’ in CIGAR string) in input data. The output result includes names of primary and secondary genes that overlap at least one of the two splice sites of a junction.": "接头是从输入数据中跨越外显子的读取（包含CIGAR字符串中的'N'）中识别出来的。输出结果包括至少在一个接头的两个剪接位点之一上重叠的主要和次要基因的名称。",
    "The FASTA-format file that contains the reference sequences used in read mapping can be used to improve read counting for junctions": "包含用于读取映射的参考序列的FASTA格式文件可用于改善接头的读取计数",
    "If specified, long reads such as Nanopore and PacBio reads will be counted. Long read counting can only run in one thread and only reads (not read-pairs) can be counted.": "如果指定了，像Nanopore和PacBio这样的长读序列将被计数。长读计数只能在一个线程中运行，并且只能计数读序列（而不是读序列对）。",
    "If specified, reads are counted for each read group separately. The 'RG' tag must be present in the input BAM/SAM alignment files.": "如果指定了，每个读组的读数将分别进行计数。输入的BAM/SAM比对文件中必须存在'RG'标签。",
    "If specified, reads (or fragments) will be assigned to the target that has the largest number of overlapping bases": "如果指定了，读取（或片段）将被分配给重叠碱基数量最多的目标。",
    "Specify the minimum required number of overlapping bases between a read (or a fragment) and a feature. 1 by default. If a negative value is provided, the read will be extended from both ends.": "指定一个读段（或片段）与特征之间所需的最小重叠碱基数量。默认为1。如果提供了一个负数，则读段将从两端延伸。",
    "Specify the minimum required fraction of overlapping bases between a read (or a fragment) and a feature. Value should be within range [0,1]. 0 by default. Number of overlapping bases is counted from both reads if paired end. Both this option and '--minOverlap' need to be satisfied for read assignment.": "指定一个读（或片段）与特征之间重叠碱基的最小所需比例。该值应在范围[0,1]内。默认值为0。如果为成对末端，则从两个读计数重叠碱基。为了进行读分配，需要同时满足此选项和 '--minOverlap'。",
    "Specify the minimum required fraction of bases included in a feature overlapping bases between a read (or a read-pair). Value should be within range [0,1]. 0 by default.": "指定在特征重叠碱基中包含的最小碱基比例，该比例是针对一个读取（或一对读取）的。值应在[0,1]范围内，默认为0。",
    "Reads are extended upstream by ... bases from their 5' end": "读取的上游从它们的5'端延伸了...个碱基。",
    "Reads are extended upstream by ... bases from their 3' end": "读取的序列在其3'端上游延伸了...个碱基。",
    "The read is reduced to its 5' most base or 3'most base. Read summarization is then performed based on the single base the the read is reduced to.": "读取被缩减为其5'端最基础或3'端最基础。然后根据读取被缩减到的单个基础进行读取总结。",
    "Threshold for average quality of mutations and N-regions of CDR3 (only relevant for fastq input)": "CDR3突变和N区域的平均质量阈值（仅适用于fastq输入）",
    "The should be a TAB-separated count matrix with gene identifers in the first column and a header row": "输出应该是一个以制表符分隔的计数矩阵，第一列包含基因标识符，并有一行标题行。",
    "Include genes with detected expression in at least this many cells": "包含在至少这么多细胞中检测到表达的基因",
    "Include cells where at least this many genes are detected": "包含至少检测到这么多基因的细胞",
    "For paired-end, estimated from the input alignment file. For single-end, ignored by default.": "对于成对末端，从输入的比对文件中估计。对于单末端，通常被忽略。",
    "Result files for gene TE analysis and sigDiff gene TE will be created if more than one dataset is applied.": "如果应用了多个数据集，将创建基因转座元件（TE）分析和显著差异基因转座元件（sigDiff gene TE）的结果文件。",
    "gene_id<tab>trans_id<return>": "基因ID<tab>转录本ID<return>",
    "Only analyse top strand?": "仅分析正链？",
    "(TransDecoder.Predict)": "(TransDecoder.Predict)",
    "In dynamic mode: set range according to 1% FDR in a random sequence of same GC content.": "在动态模式中：根据相同GC含量的随机序列中的1% FDR设定范围。",
    "Retain all ORFs found that are equal or longer than these many nucleotides even if no other evidence marks it as coding.": "保留所有发现的开放阅读框（ORFs），如果它们等于或长于这些核苷酸，即使没有其他证据表明它是编码的。",
    "Any ORF with a blast match will be retained in the final output. (tabular outfmt6 file)": "任何具有BLAST匹配的开放阅读框（ORF）都将保留在最终输出中。（制表符分隔的outfmt6文件）",
    "Domain table output file from running hmmscan to search Pfam. Any ORF with a pfam domain hit will be retained in the final output. (tabular domtblout file)": "使用hmmscan搜索Pfam的域表输出文件。任何具有pfam域命中的ORF都将保留在最终输出中。（制表符分隔的domtblout文件）",
    "Prioritized by homology than ORF length.": "优先考虑同源性而非开放阅读框长度。",
    "The first (10*value) elements are selected for removing redundancies. Then number of longst ORFs of this value are selected from the non-redundant set.": "首先选择前（10*value）个元素来去除冗余。然后从非冗余集合中选择这个值的最长ORF的数量。",
    "Matching read positions with greater than this fraction overlap are merged into a segment": "将具有大于该分数重叠的读取位置匹配并合并为一个片段",
    "Consider the alignments of a that read have at least longer than this value times the longest alignment for that read": "考虑这个对齐，使得该读取的对齐长度至少比该读取的最长对齐长度长这个值倍。",
    "Minimum fraction of BED entries/blockbuster blocks that must overlap inorder to merge": "最小重叠比例，即BED条目或blockbuster块必须重叠的最小比例，以便进行合并。",
    "Minimum number of alignments to be present to output the merged a locus": "最小对齐数，以输出合并后的基因座",
    "Minimum distance between two clusters": "两个簇之间的最小距离",
    "Reference fasta file": "参考序列fasta文件",
    "Second reference fasta file.": "第二个参考序列的fasta文件。",
    "Input fasta file": "输入fasta文件",
    "Use single if you have all the transcripts in single fasta file. Use split, if you split the reference into two such that each chimeric read arm corresponds to one of them": "如果您所有的转录本都在一个单独的fasta文件中，请使用single。如果您将参考序列分割成两个，使得每个嵌合读段的臂对应于它们中的一个，请使用split。",
    "Seed length for 1st mapping iteration. bwa-mem parameter -k": "第一条映射迭代的种子长度。bwa-mem 参数 -k",
    "Seed length for 2nd mapping iteration. bwa-mem parameter -k": "第二次映射迭代的种子长度。bwa-mem 参数 -k",
    "bwa-mem parameter '-T'": "BWA-MEM 参数 '-T'",
    "Minimu length of the read segment that needs to be mapped. clan_search parameter '-l'": "读取片段需要映射的最小长度。clan_search参数'-l'",
    "Minimum fraction of reads of a locus that must overlap with all CRL loci inorder to merge itinto that CRL.": "一个位点的最小读段比例，这些读段必须与所有CRL位点重叠，以便将其合并到该CRL中。",
    "Minimum number of reads a locus should have in order to participate in CRL creation. Always set this value relative to your sequencing depth. Setting this to lower leads CRLs of random multimappings Also consider setting the --crl_share option along with this": "一个位点为了参与CRL（核心参考区间）的创建，应该具有的最小读取数。总是将这个值相对于您的测序深度来设置。将此值设置得较低会导致随机多映射的CRL。同时考虑与此选项一起设置--crl_share选项。",
    "The maximum difference of transcripts expression between two consecutive iterations of EM algorithm to converge.": "EM算法连续两次迭代之间转录本表达量的最大差异以收敛。",
    "Create CRLs and qunatify them or use the loci further without creating the CRLs": "创建CRLs（共表达基因网络）并量化它们，或者不创建CRLs而直接使用这些位点。",
    "Selecet Yes if you have an annotation file and provide corresponding genomic fasta file": "如果您有一个注释文件，并提供相应的基因组fasta文件，请选择“是”。",
    "Transcripts with less than this percentile TPMs will be discarded in the final output. [0-1.0]": "在最终输出中，表达量低于该百分位数的TPM值的转录本将被丢弃。[0-1.0]",
    "Hybrids with less than this score will be discarded in the final output. [0-2.0]": "得分低于这个分数的杂交体将在最终输出中被丢弃。[0-2.0]",
    "Turning this option on increases the run time of the tool significantly.": "开启此选项将显著增加工具的运行时间。",
    "IntaRNA --seedBP parameter": "IntaRNA --种子碱基对参数",
    "IntaRNA --seedMinPu parameter": "IntaRNA --最小种子惩罚参数",
    "IntaRNA --accW parameter": "IntaRNA --accW 参数",
    "Trinity output: Assembled Transcripts": "Trinity输出：组装后的转录本",
    "Transdecoder output: transdecoder_pep": "Transdecoder 输出：transdecoder_pep",
    "Genes to transcripts map output. Containing correspondance between gene ids and transcript ids based on the name of transcripts assembled by Trinity": "基因到转录本的映射输出。包含基于Trinity组装的转录本名称的基因ID和转录本ID之间的对应关系。",
    "[optional] NCBI BLAST+ blastp output. Output format: Tabular (standard 12 columns)": "[可选] NCBI BLAST+ 程序的 blastp 输出。输出格式：表格（标准12列）",
    "[optional] NCBI BLAST+ blastx output. Output format: Tabular (standard 12 columns)": "[可选] NCBI BLAST+ 程序的 blastx 输出。输出格式：表格（标准12列）",
    "[optional] HMMER hmmscan output: Table of per-domain hits. Output format: Table of per-domain hits (--domtblout)": "[可选] HMMER hmmscan输出：每个域的命中表格。输出格式：每个域的命中表格（--domtblout）",
    "[optional]": "[可选]",
    "https://data.broadinstitute.org/Trinity/__deprecated_trinotate_resources/Trinotate_v3_RESOURCES/Trinotate_v3.sqlite.gz": "https://data.broadinstitute.org/Trinity/__deprecated_trinotate_resources/Trinotate_v3_RESOURCES/Trinotate_v3.sqlite.gz",
    "Can be used for visualization purpose": "可用于可视化目的",
    "The version of genome against which the reads were aligned.": "比对的基因组版本。",
    "Genome sequences in FASTA format": "基因组序列在FASTA格式",
    "e.g. CTCF": "例如 CTCF",
    "Read group/attribute datasets provide information on replicates. One of both is provided, depending on the output format.": "读取组/属性数据集提供有关重复的信息。根据输出格式，提供其中一个或两个。",
    "By default, Cuffnorm reports expression levels in the 'simple-table' tab-delimted text files. The program also reports information about your samples and about the genes, transcripts, TSS groups, and CDS groups as tab delimited text files. Note that these files have a different format than the files used by Cuffdiff. However, you can direct Cuffnorm to report its output in the same format as used by Cuffdiff if you wish": "默认情况下，Cuffnorm在以制表符分隔的文本文件中报告表达水平。该程序还以制表符分隔的文本文件形式报告有关您的样本以及基因、转录本、TSS组和CDS组的信息。请注意，这些文件的格式与Cuffdiff使用的文件格式不同。然而，如果您愿意，您可以指示Cuffnorm以与Cuffdiff使用的格式相同的方式报告其输出。",
    "You can produce this file using RUVSeq or svaseq.": "您可以使用RUVSeq或svaseq生成此文件。",
    "Method for estimation: either 'ratio', 'poscounts', or 'iterate'. 'ratio' uses the standard median ratio method introduced in DESeq. The size factor is the median ratio of the sample over a 'pseudosample': for each gene, the geometric mean of all samples. 'poscounts' and 'iterate' offer alternative estimators, which can be used even when all genes contain a sample with a zero (a problem for the default method, as the geometric mean becomes zero, and the ratio undefined). The 'poscounts' estimator deals with a gene with some zeros, by calculating a modified geometric mean by taking the n-th root of the product of the non-zero counts. This evolved out of use cases with Paul McMurdie's phyloseq package for metagenomic samples. The 'iterate' estimator iterates between estimating the dispersion with a design of ~1, and finding a size factor vector by numerically optimizing the likelihood of the ~1 model.": "估计方法：可以是'ratio'、'poscounts'或'iterate'。'ratio'使用DESeq中引入的标准中位数比率方法。大小因子是样本与'伪样本'的中位数比率：对于每个基因，所有样本的几何平均值。'poscounts'和'iterate'提供了替代的估计器，即使所有基因都包含一个样本的零值（默认方法的问题，因为几何平均值变为零，比率未定义）时也可以使用。'poscounts'估计器处理一些零值的基因，通过计算修改后的几何平均值，取非零计数的n次根。这是从Paul McMurdie的phyloseq包用于宏基因组样本的用例中演变而来的。'iterate'估计器在估计具有~1设计的离散度和通过数值优化~1模型的可能性来寻找大小因子向量之间进行迭代。",
    "The input must be a 2-column file: col1 should have the input file names. Col2 should contain your custom size factors.": "输入必须是一个两列的文件：第一列应包含输入文件名，第二列应包含您自定义的大小因子。",
    "When there are more than 6 replicates for a given sample, the DESeq2 will automatically replace counts with large Cook’s distance with the trimmed mean over all samples, scaled up by the size factor or normalization factor for that sample": "当给定样本的重复次数超过6次时，DESeq2会自动将具有较大Cook距离的计数替换为所有样本的修剪均值，该均值会根据该样本的大小因子或标准化因子进行缩放。",
    "When there are more than 2 replicates for a given sample, the DESeq2 will automatically filter genes which contain a Cook’s distance above a cutoff": "当给定样本的重复次数超过2次时，DESeq2会自动过滤掉包含超过截止值的Cook距离的基因。",
    " DESeq2 performs independent filtering by default using the mean of normalized counts as a filter statistic": "DESeq2 默认使用规范化计数的均值作为过滤统计量进行独立过滤",
    "While it is not necessary to pre-filter low count genes before running the DESeq2 functions, there are two reasons which make pre-filtering useful: by removing rows in which there are very few reads, we reduce the required memory, and we increase the speed. It can also improve visualizations, as features with no information for differential expression are not plotted.": "虽然在运行DESeq2函数之前没有必要预先过滤低计数基因，但有两个原因使得预先过滤很有用：通过移除读数非常少的行，我们减少了所需的内存，并提高了速度。它还可以改善可视化，因为不用于差异表达的特征不会被绘制出来。",
    "Keep only rows that have at least N reads total.": "只保留至少有N个读段的行。",
    "Whether or not to put a zero-mean normal prior on the non-intercept coefficients": "是否对非截距系数施加零均值正态先验",
    "Default value is 0.1. This option is only meaninful when plots are generated": "默认值是0.1。这个选项只有在生成图表时才有意义。",
    "Input BAM/SAM/CRAM file containing the short reads you want to assemble into transcripts": "输入包含您想要组装成转录本的短读序列的BAM/SAM/CRAM文件",
    "Input BAM/SAM/CRAM file containing the long reads you want to assemble into transcripts": "输入包含您想要组装成转录本的长读序列的BAM/SAM/CRAM文件",
    "This option allows to define window around possibly erroneous splice sites from long reads to look out for correct splice sites. Default: 25": "这个选项允许定义一个窗口，用于在长读长的可能错误的剪接位点周围寻找正确的剪接位点。默认值：25",
    "Select 'Forward (FR)' if your reads are from a forward-stranded library, 'Reverse (RF)' if your reads are from a reverse-stranded library, or 'Unstranded' if your reads are not from a stranded library. See Help section below for more information. Default: Unstranded": "如果读取来自正链库，请选择“正向（FR）”，如果读取来自反链库，请选择“反向（RF）”，如果读取不是来自链特异性库，请选择“非链特异性”。有关更多信息，请参见下面的帮助部分。默认：非链特异性",
    "Use the reference annotation file (in GTF or GFF3 format) to guide the assembly process. The output will include expressed reference transcripts as well as any novel transcripts that are assembled. This option is required by option -e (Use Reference transcripts only), see below.": "使用参考注释文件（GTF或GFF3格式）来指导组装过程。输出将包括表达的参考转录本以及组装的任何新转录本。此选项由选项-e（仅使用参考转录本）所需，见下文。",
    "Limit the processing of read alignments to only estimate and output the assembled transcripts matching the reference transcripts given with the -G option. With this option, read bundles with no reference transcripts (novel transcripts) will be entirely skipped, which may provide a considerable speed boost when the given set of reference transcripts is limited to a set of target genes, for example. Default: No": "限制对读取对齐的处理，仅估计并输出与使用 -G 选项给定的参考转录本匹配的组装转录本。使用此选项时，没有参考转录本（新转录本）的读取束将完全跳过，当给定的参考转录本集限制为一组目标基因时，这可能会提供相当大的速度提升。默认值：否。",
    "Select to output additional files that can be used with Ballgown or DESeq2/edgeR. See Help section below for more information": "选择输出附加文件，这些文件可以与Ballgown或DESeq2/edgeR一起使用。有关更多信息，请参见下面的帮助部分。",
    "Default: 75": "默认值：75",
    "Choose whether to cluster genes with different gene IDs that overlap. Transcripts containing the geneID prefix will be ignored. Default: No": "选择是否对具有不同基因ID的重叠基因进行聚类。包含特定基因ID前缀的转录本将被忽略。默认值：否。",
    "If a different prefix was used for geneIDs assigned by StringTie than the default, specify it here. Only letters and numbers will be retained in this field. Default: MSTRG": "如果StringTie分配的基因ID使用了不同的前缀，请在此指定。此字段中将仅保留字母和数字。默认值：MSTRG",
    "If clustering, what prefix to use for geneIDs assigned by this script. Only letters and numbers will be retained in this field. Default: prepG": "如果进行聚类，那么由这个脚本分配的基因ID应该使用什么前缀？在这个字段中，将只保留字母和数字。默认值：prepG",
    "If StringTie is run with this option (requires -G), it returns a file with all the transcripts in the reference annotation that are fully covered, end to end, by reads. The output format is a GTF file as described below. Each line of the GTF corresponds to a gene or transcript in the reference annotation.": "如果StringTie使用这个选项运行（需要-G），它将返回一个包含所有在参考注释中完全被读取覆盖的转录本的文件。输出格式是下面描述的GTF文件。GTF的每一行对应参考注释中的一个基因或转录本。",
    "If selected, gene abundances will be reported in a tab-delimited file, see below for more information": "如果选中，基因丰度将在制表符分隔的文件中报告，有关更多信息，请参见下文",
    "Ignore all read alignments (and thus do not attempt to perform transcript assembly) on the specified reference sequences. This parameter can be a single reference sequence name (e.g. chrM) or a comma-delimited list of sequence names (e.g. chrM,chrX,chrY). This can speed up StringTie especially in the case of excluding the mitochondrial genome, whose genes may have very high coverage in some cases, even though they may be of no interest for a particular RNA-Seq analysis. The reference sequence names are case sensitive, they must match identically the names of chromosomes/contigs of the target genome against which the RNA-Seq reads were aligned in the first place.": "忽略所有指定参考序列上的读取对齐（因此不尝试执行转录本组装）。此参数可以是单个参考序列名称（例如 chrM），或用逗号分隔的序列名称列表（例如 chrM,chrX,chrY）。特别是在排除线粒体基因组的情况下，这可以加快 StringTie 的速度，因为线粒体基因组中的基因在某些情况下可能具有非常高的覆盖率，尽管它们可能对特定的 RNA-Seq 分析没有兴趣。参考序列名称是区分大小写的，它们必须与目标基因组的染色体/片段的名称完全匹配，RNA-Seq 读取最初就是与该基因组对齐的。",
    "This prefix will be added to the name of the transcripts that are output. Only letters and numbers will be retained in this field. Default: STRG": "这个前缀将被添加到输出的转录本名称中。在这个字段中，只会保留字母和数字。默认值：STRG",
    "Sets the minimum isoform abundance of the predicted transcripts as a fraction of the most abundant transcript assembled at a given locus. Lower abundance transcripts are often artifacts of incompletely spliced precursors of processed transcripts. Default: 0.05": "设置预测转录本的最小异构体丰度，作为在给定位点组装的最丰富的转录本的分数。丰度较低的转录本通常是未完全剪接的处理转录本前体的人工产物。默认值：0.05",
    "Sets the minimum length allowed for the predicted transcripts. Default: 200": "设置预测转录本的最小长度。默认值：200",
    "Junctions that don't have spliced reads that align across them with at least this amount of bases on both sides are filtered out. Default: 10": "不包含至少有这么多碱基对在两侧与之对齐的剪接读段的连接点会被过滤掉。默认值：10",
    "There should be at least this many spliced reads that align across a junction (i.e. junction coverage). This number can be fractional, since some reads align in more than one place. A read that aligns in n places will contribute 1/n to the junction coverage. Default: 1": "至少应该有这么多经过剪接的比对到连接点的读段（即连接点覆盖度）。这个数字可以是小数，因为有些读段在多个位置比对。一个在n个位置比对的读段将对连接点覆盖度贡献1/n。默认值：1",
    "Sets the minimum read coverage allowed for the predicted transcripts. A transcript with a lower coverage than this value is not shown in the output. Default: 2": "设置预测转录本所允许的最小读取覆盖度。覆盖度低于此值的转录本将不会在输出中显示。默认值：2",
    "Minimum locus gap separation value. Reads that are mapped closer than this distance are merged together in the same processing bundle. Default: 50 (bp)": "最小基因座间隙分离值。比这个距离更近的读取被合并在同一个处理束中。默认值：50（碱基对）",
    "Sets the maximum fraction of muliple-location-mapped reads that are allowed to be present at a given locus. Default: 0.95": "设置在给定位点上允许存在的多重位置映射的读取的最大比例。默认值：0.95",
    "This parameter disables trimming at the ends of the assembled transcripts. By default StringTie adjusts the predicted transcript's start and/or stop coordinates based on sudden drops in coverage of the assembled transcript. Default: No": "这个参数用于禁用组装转录本两端的修剪。默认情况下，StringTie会根据组装转录本覆盖度的突然下降来调整预测转录本的起始和/或终止坐标。默认值：否。",
    "Default: No": "默认：否",
    "Loads a list of point-features from a text feature file to guide the transcriptome assembly. Accepted point features are transcription start sites (TSS) and polyadenylation sites (CPAS). There are four tab-delimited columns in the feature file. The first three define the location of the point feature on the cromosome (sequence name, coordinate and strand), and the last is the type of the feature (TSS or CPAS).": "从文本特征文件加载点特征列表以指导转录组组装。接受的点特征是转录起始位点（TSS）和聚腺苷酸位点（CPAS）。特征文件中有四个以制表符分隔的列。前三个定义了染色体上点特征的位置（序列名称、坐标和链），最后一个是特征的类型（TSS或CPAS）。",
    "In GTF or GFF3 format": "在GTF或GFF3格式中",
    "By default these are not kept unless there is strong evidence for them": "默认情况下，除非有充分的证据支持，否则这些不会被保留。",
    "de novo assembly of RNA-Seq data": "RNA-Seq数据的去重复组装",
    "Bowtie -X parameter": "Bowtie -X 参数",
    "If the transcripts were not assembled by trinity, additional information is needed": "如果转录本不是由Trinity组装的，那么就需要额外的信息。",
    "Can be lists of pair dataset collection": "可以是成对数据集的列表",
    "Elements of collection will NOT be merged": "集合中的元素将不会被合并",
    "Pair dataset collection. The paired datasets will NOT be merged": "配对数据集集合。配对的数据集将不会被合并",
    "Defaults to max. read coverage of 50.": "默认最大读取覆盖度为50。",
    "All contigs shorter than this will be discarded": "所有比这个长度短的连续序列都将被丢弃",
    "If you already mapped the reads to the genome, Trinity can use this information": "如果你已经将读取映射到基因组，Trinity可以使用这些信息",
    "Experimental feature! Long reads must be in the same orientation as short reads if they are strand specific": "实验特性！如果长读长是特定链的，则它们必须与短读长在同一方向上",
    "Also set a name with --fastq_n": "同时设置一个名称，使用 --fastq_n",
    "Positive sequences (== RBP binding sites) FASTA file for model training": "正序列（即RBP结合位点）FASTA文件，用于模型训练",
    "Negative sequences FASTA file for model training": "用于模型训练的负序列 FASTA 文件",
    "Train a structure model (default: train a sequence model)": "训练一个结构模型（默认：训练一个序列模型）",
    "Hyperparameter optimization set size (taken away from both positive and negative input sequences) (default: 500)": "超参数优化集大小（从正负输入序列中各取走的大小）（默认值：500）",
    "Positive (== RBP binding sites) sequences FASTA file for hyperparameter optimization": "正向（== RBP 结合位点）序列的 FASTA 文件用于超参数优化",
    "Negative sequences FASTA file for hyperparameter optimization": "负序列FASTA文件用于超参数优化",
    "Disable 10-fold cross validation step. As a result, no generalization performance results (.cv_results) are output. On the other hand, run time is reduced considerably (default: false)": "禁用10折交叉验证步骤。因此，不会输出泛化性能结果（.cv_results）。另一方面，运行时间会大大减少（默认：false）",
    "Disable motif generation step, therefore no _motif and _motif.png files are output (default: false)": "禁用模式生成步骤，因此不输出_motif和_motif.png文件（默认：false）",
    "Minimum number of training sites demanded (for both negatives and positives). In general, try to get more training sites if possible (>> 1000), before lowering this number (default: 750)": "要求的最小训练位点数量（对于负样本和正样本都是）。一般来说，如果可能的话，尽量获得更多的训练位点（>> 1000），然后再降低这个数字（默认：750）",
    "FASTA file with sequences for which to predict binding profiles or whole site scores": "用于预测结合剖面或完整位点分数的FASTA序列文件",
    "GraphProt model to use for predictions": "用于预测的GraphProt模型",
    "Parameter file containing model parameters": "包含模型参数的参数文件",
    "BED file specifying the genomic regions of the input sequences, to also output peak regions with their genomic coordinates (default: false)": "指定输入序列的基因组区域的BED文件，以输出峰值区域及其基因组坐标（默认：否）",
    "Regions with peak score higher or equal the given value are reported (default: 0)": "得分高于或等于给定值的区域将被报告（默认值：0）",
    "Defines the average profile up- and downstream extension to produce the average profile. The mean over small sequence windows (window_length = set_value*2 + 1) is used to get the average profile position-wise scores. A value of 0 means no additional smoothing (== original profile scores), while 10 is applies fairly strong smoothing (default: 5)": "定义平均轮廓的上游和下游扩展，以产生平均轮廓。使用小序列窗口（窗口长度 = 设置值*2 + 1）的均值来获取平均轮廓位置的得分。值0表示没有额外的平滑（==原始轮廓得分），而10则应用相当强的平滑（默认：5）",
    "By default all non-overlapping regions will be reported. E.g. a distance of 1 means that two regions above the set threshold score will be merged if there is 1 nucleotide that separates the two regions": "默认情况下，所有不重叠的区域都会被报告。例如，距离为1意味着如果有两个区域之间的1个核苷酸将这两个区域分隔开，那么这两个高于设定阈值分数的区域将被合并。",
    "Run a whole site prediction instead of calculating profiles (default: false)": "运行一个全站点预测，而不是计算配置文件（默认：false）",
    "Output filtered peak regions BED file or predictions file (if whole site scores prediction enabled), using the median positive training site score (stored in .params file) for filtering (default: false)": "输出过滤后的峰值区域 BED 文件或预测文件（如果启用了整个位点分数预测），使用存储在 .params 文件中的正训练位点分数的中位数进行过滤（默认：false）。",
    "The input file should only contain p-values, one per line.": "输入文件应仅包含p值，每行一个。",
    "The file should contain two columns: the node identifier (e.g. gene) and the corresponding Heinz score": "该文件应包含两列：节点标识符（例如基因）和相应的Heinz得分",
    "In this file, two nodes define one edge, one edge per line": "在该文件中，每行定义一条边，由两个节点组成",
    "The file should contain two columns: the node identifier (e.g. gene) and the corresponding p-value": "该文件应包含两列：节点标识符（例如基因）和相应的p值",
    "False Discovery Rate (FDR), e.g. 0.0001": "错误发现率（FDR），例如 0.0001",
    "Output file of the tool \"Identify optimal scoring subnetwork\"": "工具“识别最优得分子网络”的输出文件",
    "Minimum base quality for T>C conversions (default: 27).": "T>C转换的最小碱基质量（默认：27）。",
    "Maximum read length (before trimming).": "最大读取长度（修剪前）。",
    "If this option is set to Yes, the Alleyoop read-separator module will be run to output BAM files of separated T>C reads from non T>C reads. Default: No": "如果将此选项设置为Yes，将运行Alleyoop读分隔模块，以输出将T>C读与非T>C读分开的BAM文件。默认值：No",
    "The maximum number of alignments is used to track multimapping read alignments. The more are allowed, the more sensitive the multimapper filtering will be, but also the longer the runtime will be. 100 was used in previous publications.": "最大对齐数用于追踪多映射读对齐。允许的越多，多映射器过滤就越敏感，但运行时间也越长。在以前的出版物中使用了100。",
    "Enable multimapper recovery, requires -n to be set to a value > 1 to have impact.": "启用多映射恢复，需要将-n设置为大于1的值才能产生影响。",
    "For Lexogen's Quantseq kit and previous SLAM-seq papers a clipping of 12 bp from the 5' end is recommended.": "对于Lexogen的Quantseq试剂盒和之前SLAM-seq论文，建议从5'端剪切12个碱基对。",
    "The maximum number of allowed As at the 3' end of a read. All A-stretches that exceed this threshold are clipped because they are likely part of the poly-A tail.": "在一个读取的3'端允许的最大A碱基数量。所有超过这个阈值的A碱基序列将被剪切，因为它们可能是多聚腺苷酸尾部的一部分。",
    "Enable end-to-end alignments for mapping in slamdunk with --endtoend": "在 slamdunk 中使用 --endtoend 参数启用端到端比对进行映射",
    "Minimum mapping quality to consider alignments (default: 2).": "考虑比对的最小映射质量（默认值：2）。",
    "Minimum alignment-identity to consider alignments (default: 0.95).": "最小比对一致性以考虑比对（默认值：0.95）。",
    "Maximum number of mismatches to consider alignments. Negative numbers deactivate filter (default: -1).": "考虑比对的最大不匹配数。负数将禁用该过滤器（默认值：-1）。",
    "Minimum coverage to call variant (default: 10).": "调用变异的最小覆盖度（默认值：10）。",
    "Minimum variant fraction to call variant (default: 0.8).": "调用变异的最小变异分数（默认值：0.8）。",
    "Number of T>C conversions required to count a read as a T>C read (default: 1).": "需要T>C转换的数量，以将一个读取计数为T>C读取（默认值：1）。",
    "You will usually want to choose 'Count fragments' for paired-end data. For single-end data, choose 'Count reads'. See tool help below.": "您通常需要为成对末端数据选择“计数片段”。对于单末端数据，选择“计数读取”。详见下面的工具帮助。",
    "The resulting dataset can, for example, serve as input to QualiMap Counts QC for further assessment.": "生成的数据集可以作为QualiMap Counts QC的输入，用于进一步评估。",
    "Consider using the name of the analyzed sample here. Default: Name of the mapped reads input dataset in the history": "考虑在这里使用分析样本的名称。默认值：历史记录中映射读取输入数据集的名称",
    "If the build of your interest is not listed contact your Galaxy admin": "如果您感兴趣的构建没有列出，请联系您的Galaxy管理员",
    "Choose if you want to provide seperate count files (e.g. from HTSeq-count or feature-seq) and define your experiment design matrix here, or if you have maSigPro edesign and data input files already.": "选择是否要提供单独的计数文件（例如来自HTSeq-count或feature-seq），并在这里定义您的实验设计矩阵，或者如果您已经有了maSigPro的edesign和数据输入文件。",
    "Matrix describing experimental design. Rows must be arrays and columns experiment descriptors": "描述实验设计的矩阵。行必须是数组，列是实验描述符。",
    "Matrix containing normalized gene expression data. Genes must be in rows and arrays in columns": "包含标准化基因表达数据的矩阵。基因必须在行中，而数组在列中。",
    "Column number in edesign containing time values. Default is first column": "在edesign中包含时间值的列号。默认是第一列",
    "Column number in edesign containing coding for replicate arrays. Default is second column": "在edesign中包含重复数组编码的列号。默认是第二列",
    "Choose if you want to output the generated edesign and data files for direct use in maSigPro as history elements.": "选择是否要输出生成的edesign和数据文件，以便在maSigPro中直接作为历史元素使用。",
    "Only numbers will be allowed": "只允许数字",
    "Use a single name without spaces or special characters": "使用一个没有空格或特殊字符的单一名称",
    "‘make.design.matrix’ creates the design matrix of dummies for fitting time series micorarray gene expression experiments.": "‘make.design.matrix’ 创建了用于拟合时间序列微阵列基因表达实验的虚拟设计矩阵。",
    "The degree of the regression fit polynome. ‘degree’ = 1 returns linear regression, ‘degree’ = 2 returns quadratic regression, etc": "回归拟合多项式的度数。'degree' = 1 返回线性回归，'degree' = 2 返回二次回归，依此类推。",
    "‘p.vector’ performs a regression fit for each gene taking all variables present in the model given by a regression matrix and returns a list of FDR corrected significant genes": "'p.vector' 对模型中给定的回归矩阵中存在的所有变量执行每个基因的回归拟合，并返回一个经过FDR校正的显著基因列表",
    "Significance level": "显著性水平",
    "Genes with less than this number of true numerical values will be excluded from the analysis. Minimum value to estimate the model is (degree+1)xGroups+1. Default is 6.": "基因中真实数值的数量少于这个数值的将被排除在分析之外。估计模型的最小值是（次数+1）x组数+1。默认值是6。",
    "‘T.fit’ selects the best regression model for each gene using stepwise regression. In the maSigPro approach ‘p.vector’ and ‘T.fit’ are subsequent steps, meaning that significant genes are first selected on the basis of a general model and then the significant variables for each gene are found by step-wise regression.": "'T.fit' 使用逐步回归为每个基因选择最佳回归模型。在 maSigPro 方法中，'p.vector' 和 'T.fit' 是连续的步骤，意味着首先根据通用模型选择显著基因，然后通过逐步回归找到每个基因的显著变量。",
    "The step regression can be ‘backward’ or ‘forward’ indicating whether the step procedure starts from the model with all or none variables. With the ‘two.ways.backward’ or ‘two.ways.forward’ options the variables are both allowed to get in and out. At each step the p-value of each variable is computed and variables get in/out the model when this p-value is lower or higher than given threshold alfa.": "逐步回归可以是“向后”或“向前”，这表明逐步过程是从包含所有变量的模型开始，还是从不包含任何变量的模型开始。使用“two.ways.backward”或“two.ways.forward”选项时，变量既可以进入也可以离开模型。在每一步中，计算每个变量的p值，当这个p值低于或高于给定的阈值alfa时，变量会进入或离开模型。",
    "When nvar.correction is TRUE the given significance level is corrected by the number of variables in the model.": "当 nvar.correction 为 TRUE 时，给定的显著性水平会根据模型中的变量数量进行校正。",
    "Significance level used for variable selection in the stepwise regression": "在逐步回归中用于变量选择的显著性水平",
    "This function creates lists of significant genes for a set of variables whose significance value has been computed with the ‘T.fit’ function.": "这个函数为一组变量创建显著基因的列表，这些变量的显著性值是使用‘T.fit’函数计算的。",
    "cut-off level at the R-squared value for the stepwise regression fit. Only genes with R-squared more than rsq are selected": "用于逐步回归拟合的 R 平方值的截断水平。只有 R 平方值大于 rsq 的基因被选择",
    "Variables for which to extract significant genes. ‘all’: generates one single matrix or gene list with all significant genes. ‘each’: generates as many significant genes extractions as variables in the general regression model. Each extraction contains the significant genes for that variable. ‘groups’: generates a significant genes extraction for each experimental group. The difference between ‘each’ and ‘groups’ is that in the first case the variables of the same group (e.g. ‘TreatmentA’ and ‘time*TreatmentA’) will be extracted separately and in t he second case jointly.": "提取显著基因的变量。‘all’：生成一个包含所有显著基因的单一矩阵或基因列表。‘each’：根据一般回归模型中的变量生成相应数量的显著基因提取。每个提取包含该变量的显著基因。‘groups’：为每个实验组生成一个显著基因提取。‘each’和‘groups’之间的差异在于，在第一种情况下，同一组的变量（例如‘TreatmentA’和‘time*TreatmentA’）将分别提取，而在第二种情况下将联合提取。",
    "The argument ‘significant.intercept’ modulates the treatment for intercept coefficients to apply for selecting significant genes when ‘vars’ equals ‘groups’. There are three possible values: ‘none’, no significant intercept (differences) are considered for significant gene selection, ‘dummy’, includes genes with significant intercept differences between control and experimental groups, and ‘all’ when both significant intercept coefficient for the control group and significant intercept differences are considered for selecting significant genes.": "参数 'significant.intercept' 调节处理截距系数的方法，以应用于当 'vars' 等于 'groups' 时选择显著基因。有三个可能的值：'none'，不考虑显著截距（差异）进行显著基因选择，'dummy'，包括控制组和实验组之间显著截距差异的基因，以及 'all'，当同时考虑控制组的显著截距系数和显著截距差异时，用于选择显著基因。",
    "Choose if you want to generate a PDF file containing the visualizations": "选择是否生成包含可视化的PDF文件",
    "This function provides visualisation tools for gene expression values in a time course experiment. The function first calls the heatmap function for a general overview of experiment results. Next a partioning of the data is generated using a clustering method. The results of the clustering are visualized both as gene expression profiles extended along all arrays in the experiment, as provided by the plot.profiles function, and as summary expression profiles for comparison among experimental groups.": "这个功能提供了用于时间序列实验中基因表达值的可视化工具。首先，该功能调用热图函数以获得实验结果的总体概览。接下来，使用聚类方法生成数据的分区。聚类结果通过plot.profiles函数提供的所有阵列中的基因表达轮廓进行可视化，并作为实验组之间比较的摘要表达轮廓。",
    "Data clustering can be done on the basis of either the original expression values, the regression coefficients, or the t.scores. In case ‘data’ is a ‘get.siggenes’ object, this is given by providing the element names of the list ‘c(sig.profiles,coefficients,t.score)’ of their list position (1,2 or 3).": "数据聚类可以基于原始表达值、回归系数或t分数进行。如果'data'是一个'get.siggenes'对象，可以通过提供列表'c(sig.profiles,coefficients,t.score)'的元素名称及其列表位置（1、2或3）来指定。",
    "Adds columns with the cluster assignment for each gene.": "为每个基因添加包含聚类分配的列。",
    "clustering method for data partioning. Currently ‘hclust’, ‘kmeans’ and ‘Mclust’ are supported": "数据分区的聚类方法。目前支持‘hclust’、‘kmeans’和‘Mclust’",
    "Distance measurement function when ‘cluster.method’ is ‘hclust’. Default uses correlation.": "当‘cluster.method’为‘hclust’时的距离测量函数。默认使用相关性。",
    "The agglomeration method to be used when ‘cluster.method’ is ‘hclust’.": "当‘cluster.method’为‘hclust’时使用的聚合方法。",
    "Maximum number of iterations when ‘cluster.method’ is ‘kmeans’": "当 'cluster.method' 是 'kmeans' 时，迭代的最大次数",
    "Color scale for plotting profiles. Can be either ‘rainbow’ or ‘gray’": "绘制轮廓图的颜色刻度。可以是“彩虹”或“灰度”",
    "Indicating whether regression fit curves must be plotted": "指示是否必须绘制回归拟合曲线",
    "Indicating whether a line must be drawn joining plotted data points for each group": "指出是否需要为每个组绘制连接数据点的线",
    "Graphical parameter maginfication to be used for x labels in plotting functions": "用于绘图函数中x标签的图形参数放大",
    "Indicating whether legend must be added when plotting profiles": "指示在绘制轮廓图时是否必须添加图例",
    "output(s) from 'Align reads and estimate abundance' tool": "来自“比对读取并估计丰度”工具的输出",
    "Only needed for gene level estimates": "仅用于基因水平的估计",
    "Matrix of normalized expression values produced by 'Build expression matrix for a de novo assembly of RNA-Seq data by Trinity' tool": "由'Trinity'工具生成的去新生转录组RNA-Seq数据构建表达矩阵的标准化表达值矩阵",
    "De novo assembly of RNA-Seq data": "RNA-Seq数据的去重复组装",
    "output of Build expression matrix - matrix of raw read counts (not normalized!)": "构建表达矩阵 - 原始读取计数的矩阵（未归一化！）的输出",
    "file describing samples and replicates": "描述样本和重复的文件",
    "Read edgeR manual to guide your value choice": "阅读edgeR手册以指导您的值选择",
    "Only those rows of matrix meeting requirement will be tested": "只有满足要求的矩阵行将被测试",
    "default is doing all pairwise-comparisons among samples": "默认是进行样本之间的所有成对比较",
    "A 2-column tabular with lists of pairs of samples to compare": "一个包含样本对比较列表的两列表格",
    "... to be considered in interactions within the query": "... 在查询中的交互中需要考虑",
    "In the format 'from1-to1,from2-to2,..' assuming indexing starts with 1": "在格式“from1-to1,from2-to2,...”中，假设索引从1开始",
    "... to be considered in interactions within the target": "... 在目标内的交互中需要考虑",
    "in format 'from1-to1,from2-to2,...' (Note, only for single query)": "格式为 'from1-to1,from2-to2,...'（注意，仅适用于单个查询）",
    "Two column text file with sequence positions (1-based) and normalized reactivity values. Missing values may be left out, or assigned a negative score": "两列文本文件，包含序列位置（1 为起始）和标准化的反应性值。缺失值可以省略，或者分配一个负分数。",
    "Sets what to optimize for": "设置要优化的目标",
    "Temperature scaling of ViennaRNA energy parameters": "维也纳RNA能量参数的温度缩放",
    "If set, large sequences are decomposed in overlapping windows of the given size to reduce the memory consumption of IntaRNA.": "如果设置，大序列将被分解成给定大小的重叠窗口，以减少IntaRNA的内存消耗。",
    "If window width is set, this specifies the overlap of the windows. Note, it has to exceed the maximal interaction length of both query and target.": "如果设置了窗口宽度，则这指定了窗口的重叠部分。注意，它必须超过查询和目标之间最大相互作用长度。",
    "Only interactions with E ≤ maxE are reported": "只报告与E ≤ maxE的相互作用",
    "suboptimal output: only interactions with E ≤ (minE+deltaE) are reported": "次优输出：仅报告满足 E ≤ (minE+deltaE) 的相互作用",
    "Written in a format similar to RNAplfold unpaired probability output.": "以类似于RNAplfold未配对概率输出的格式编写。",
    "Illumina typically produces reads of 180-200bp": "Illumina通常产生180-200个碱基对的读长",
    "It allows to learn parameters for a model of sequences specific bias and corrects the abundances accordlingly": "它允许学习序列特异性偏差的模型参数，并相应地校正丰度。",
    "Running with bootstraps is mandatory if you want to perform differential expression analysis of isoforms with Sleuth.Default: 0": "使用引导是强制性的，如果您想使用Sleuth进行异构体的差异表达分析。默认值：0",
    "It generates the required files for Pizzly. This option does normal quantification, but additionally looks for reads that do not pseudoalign because they are potentially from fusion genes.": "它为Pizzly生成所需的文件。这个选项执行常规的定量分析，但额外地寻找那些因为可能来自融合基因而没有伪对齐的读取。",
    "Include reads where unobserved rest of fragment is predicted to lie outside a transcript": "包括那些未观察到的片段其余部分预计位于转录本之外的读取",
    "GTF file for transcriptome information": "转录组信息的GTF文件",
    "Default: 42": "默认值：42",
    "You can choose to input either separate count files (one per sample) or a single count matrix": "您可以选择输入单独的计数文件（每个样本一个）或单个计数矩阵",
    "Name of experiment factor of interest (e.g. Genotype). One factor must be entered and there must be two or more groups per factor. Optional additional factors (e.g. Batch) can be entered using the Insert Factor button below, see Help section for more information. NOTE: Please only use letters, numbers or underscores, and the first character of each factor must be a letter": "感兴趣的实验因素名称（例如：基因型）。必须输入一个因素，并且每个因素必须有两个或更多的组。可以使用下面的“插入因素”按钮输入可选的附加因素（例如：批次），有关更多信息，请参见帮助部分。注意：请仅使用字母、数字或下划线，并且每个因素的第一个字符必须是字母。",
    "Name of group that the counts files belong to (e.g. WT or Mut). NOTE: Please only use letters, numbers or underscores (case sensitive), and the first character of each group must be a letter": "计数文件所属组的名称（例如 WT 或 Mut）。注意：请仅使用字母、数字或下划线（区分大小写），并且每个组的第一个字符必须是字母",
    "You can choose to input the factor and group information for the samples from a file or manually enter below. NOTE: Please only use letters, numbers or underscores (case sensitive), and the first character of each sample, factor and group must be a letter": "你可以选择从文件中输入样本的因子和组信息，或者在下方手动输入。注意：请只使用字母、数字或下划线（区分大小写），并且每个样本、因子和组的第一个字符必须是字母。",
    "Name of experiment factor of interest (e.g. Genotype). One factor must be entered and there must be two or more groups per factor. Additional factors (e.g. Batch) can be entered using the Insert Factor button below, see Help section below. NOTE: Please only use letters, numbers or underscores, and the first character of each factor must be a letter": "感兴趣的实验因素名称（例如：基因型）。必须输入一个因素，并且每个因素必须有两个或更多的组。可以使用下面的“插入因素”按钮输入其他因素（例如：批次），请参阅下面的帮助部分。注意：请只使用字母、数字或下划线，并且每个因素的第一个字符必须是字母",
    "Enter the group names for the samples separated with commas e.g. WT,WT,WT,Mut,Mut,Mut. The order of the names must match the order of the samples in the columns of the count matrix. NOTE: Please only use letters, numbers or underscores (case sensitive), and the first character of each group must be a letter": "输入样品的组名，组名之间用逗号分隔，例如 WT,WT,WT,Mut,Mut,Mut。组名的顺序必须与计数矩阵列中样品的顺序相匹配。注意：请仅使用字母、数字或下划线（大小写敏感），并且每个组的第一个字符必须是字母。",
    "If you provide an annotation file, annotations will be added to the table(s) of differential expression results to provide descriptions for each gene. See Help section below.": "如果您提供了注释文件，注释将被添加到差异表达结果的表格中，以提供每个基因的描述。请参阅下面的帮助部分。",
    "An optional formula for the EdgeR linear model, this will override the use of the fields in factors as a simple sum. The formula can only use elements available in the factors file. This needs to be exactly as EdgeR expect the formula, ie. `~ 0 + factor_A + factor_B:factor_C`. See EdgeR documentation for more details.": "一个可选的EdgeR线性模型公式，这将覆盖使用因子中字段的简单求和。该公式只能使用因子文件中可用的元素。这需要完全按照EdgeR期望的公式格式，即`~ 0 + factor_A + factor_B:factor_C`。有关更多详细信息，请参见EdgeR文档。",
    "Names of two groups to compare separated by a hyphen e.g. Mut-WT. If the order is Mut-WT the fold changes in the results will be up/down in Mut relative to WT. If you have more than one contrast enter each separately using the Insert Contrast button below. For differences between contrasts use e.g. (MT.t1-MT.t0)-(WT.t1-WT.t0). For more info, see Chapter 8 in the limma User's guide: https://www.bioconductor.org/packages/release/bioc/vignettes/limma/inst/doc/usersguide.pdf or https://bioconductor.org/packages/release/bioc/vignettes/edgeR/inst/doc/edgeRUsersGuide.pdf page 36 for nested comparisons.": "两个要比较的组的名称由短划线分隔，例如 Mut-WT。如果顺序是 Mut-WT，则结果中的倍数变化将是相对于 WT 的 Mut 上下变化。如果您有多个对比，请使用下面的“插入对比”按钮分别输入每个对比。对于对比之间的区别，请使用例如 (MT.t1-MT.t0)-(WT.t1-WT.t0)。有关更多信息，请参见 limma 用户指南的第 8 章：https://www.bioconductor.org/packages/release/bioc/vignettes/limma/inst/doc/usersguide.pdf 或 https://bioconductor.org/packages/release/bioc/vignettes/edgeR/inst/doc/edgeRUsersGuide.pdf 第 36 页的嵌套比较。",
    "Setting this file will ignore any manually added contrasts above, make sure to remove any contrast fields above pressing the trash bin icon, or the tool will fail. First line of the file must be a header, below that each separate contrast should be on a line. Contrast formulas need to be based on ther factors data and potentially the formula provided. See EdgeR documentation on contrasts for more details.": "设置此文件将忽略上面手动添加的所有对比，确保在按下垃圾箱图标之前删除上面的任何对比字段，否则工具将失败。文件的第一行必须是标题，在标题下面，每个单独的对比应该在一行上。对比公式需要基于因素数据，并且可能基于提供的公式。有关对比的更多详细信息，请参见EdgeR文档。",
    "Treat genes with very low expression as unexpressed and filter out. See the Filter Low Counts section below for more information. Default: No": "将表达量非常低的基因视为未表达，并将其过滤掉。有关更多信息，请参见下面的“过滤低计数”部分。默认值：否。",
    "It is slightly better to base the filtering on count-per-million (CPM) rather than the raw count values so as to avoid favoring genes expressed in samples sequenced to a higher depth.": "基于每百万计数（CPM）进行过滤，而不是基于原始计数值，这稍微好一些，以避免偏袒测序深度更高的样本中表达的基因。",
    "Treat genes with CPM below this value as unexpressed and filter out. See the Filter Low Counts section below for more information.": "将CPM值低于该值的基因视为未表达，并过滤掉。有关更多信息，请参见下面的“过滤低计数”部分。",
    "Filter out all genes that do not meet the Minimum CPM in at least this many samples. See the Filter Low Counts section below for more information.": "过滤掉所有在至少这么多样本中未达到最小CPM（每百万计数数）的基因。有关更多信息，请参见下面的“过滤低计数”部分。",
    "Filter out all genes that do not meet this minimum count. You can choose below to apply this filter to the total count for all samples or specify the number of samples under Minimum Samples. See the Filter Low Counts section below for more information.": "滤除所有不满足此最小计数的基因。您可以选择将此过滤器应用于所有样本的总计数，或指定在“最小样本数”下的样本数。有关更多信息，请参见下面的“过滤低计数”部分。",
    "Filter out all genes that do not meet the Minimum Count in at least this many samples. See the Filter Low Counts section below for more information.": "过滤掉所有在至少这么多样本中未达到最小计数的基因。有关更多信息，请参见下面的“过滤低计数”部分。",
    "Output a file containing the normalised counts, these are in log2 counts per million (logCPM). Default: No": "输出一个包含标准化计数的文件，这些计数是以每百万的对数计数（logCPM）表示的。默认值：否。",
    "If this option is set to Yes, the Rscript used will be provided as a text file in the output. Default: No": "如果将此选项设置为“是”，则使用的R脚本将作为文本文件提供在输出中。默认值：否。",
    "Output all the data used by R to construct the plots and tables, can be loaded into R. A link to the RData file will be provided in the HTML report. Default: No": "输出 R 用于构建图表和表格的所有数据，可以加载到 R 中。HTML 报告中将提供 RData 文件的链接。默认值：否。",
    "Genes above this threshold and below the p-value threshold are considered significant and highlighted in the MD plot. Default: 0.": "超过这个阈值且低于p值阈值的基因被认为是显著的，并在MD图中突出显示。默认值：0。",
    "Genes below this threshold are considered significant and highlighted in the MD plot. If either BH(1995) or BY(2001) are selected then this value is a false-discovery-rate control. If Holm(1979) is selected then this is an adjusted p-value for family-wise error rate. Default: 0.05.": "低于此阈值的基因被认为是显著的，并在MD图中突出显示。如果选择了BH(1995)或BY(2001)，则此值为假发现率控制。如果选择了Holm(1979)，则这是家族错误率的校正p值。默认值：0.05。",
    "Default: BH": "默认值：BH",
    "Default: TMM": "默认：TMM",
    "Using robust settings is usually recommended to protect against outlier genes. Default: Yes": "使用健壮的设置通常建议用来保护免受异常基因的影响。默认值：是",
    "Use likelihood ratio test instead of quasi-likelihood F-test. Default: No": "使用似然比检验代替准似然F检验。默认值：否。",
    "e.g. Batches 1 and 4 use barcodes 1-100 in the Barcodes file, and Batches 2 and 3 use barcodes 101-200 in the Barcodes file; specify '1' and '100' as Range values, and '1,4' as Batch values, and in the next format specify '101' and '200' as Range values and '2,3' as Batch values": "例如，批次1和4使用条形码文件中的1-100号条形码，批次2和3使用条形码文件中的101-200号条形码；指定'1'和'100'作为范围值，'1,4'作为批次值，然后在下一个格式中指定'101'和'200'作为范围值，'2,3'作为批次值。",
    "e.g. Plate 1 encompasses Batches 1-4, and Plate 2 encompasses Batches 5-8; specify '1' as a Plate value, and '1,2,3,4' as Batch values, and in the next format specify '2' as a Plate value and '5,6,7,8' as Batch values": "例如，板1包含批次1-4，板2包含批次5-8；指定'1'作为板值，'1,2,3,4'作为批次值，然后在下一个格式中指定'2'作为板值，'5,6,7,8'作为批次值。",
    "Select the limma-voom or limma-trend method. See Help section below for more information. Default: limma-voom": "选择limma-voom或limma-trend方法。有关更多信息，请参见下面的帮助部分。默认值：limma-voom",
    "Apply weights if outliers are present (voomWithQualityWeights). Default: False.": "如果存在异常值，则应用权重（voomWithQualityWeights）。默认值：False。",
    "Average count to be added to each observation to avoid taking log of zero. Default: 3.": "为了避免对零取对数，在每个观测值上要添加的平均计数。默认值：3。",
    "Name of experiment factor of interest (e.g. Genotype). One factor must be entered and there must be two or more groups per factor. Optional additional factors (e.g. Batch) can be entered using the Insert Factor button below, see Help section for more information. NOTE: Please only use letters, numbers or underscores.": "实验感兴趣的因素名称（例如：基因型）。必须输入一个因素，并且每个因素必须有两个或更多的组。可以使用下面的“插入因素”按钮输入可选的附加因素（例如：批次），有关更多信息，请参见帮助部分。注意：请仅使用字母、数字或下划线。",
    "Name of group that the counts files belong to (e.g. WT or Mut). NOTE: Please only use letters, numbers or underscores (case sensitive).": "计数文件所属的组名（例如 WT 或 Mut）。注意：请仅使用字母、数字或下划线（区分大小写）。",
    "You can choose to input the factor and group information for the samples from a file or manually enter below. NOTE: Please only use letters, numbers or underscores (case sensitive), the group names MUST not contain hyphens.": "您可以选择通过文件输入样本的因子和组信息，或在下方手动输入。注意：请仅使用字母、数字或下划线（区分大小写），组名中不能包含连字符。",
    "Name of experiment factor of interest (e.g. Genotype). One factor must be entered and there must be two or more groups per factor. Additional factors (e.g. Batch) can be entered using the Insert Factor button below, see Help section below. NOTE: Please only use letters, numbers or underscores.": "感兴趣的实验因素名称（例如：基因型）。必须输入一个因素，并且每个因素必须有两个或更多的组。可以使用下面的“插入因素”按钮输入其他因素（例如：批次），请参阅下面的帮助部分。注意：请只使用字母、数字或下划线。",
    "Enter the group names for the samples separated with commas e.g. WT,WT,WT,Mut,Mut,Mut. The order of the names must match the order of the samples in the columns of the count matrix. NOTE: Please only use letters, numbers or underscores (case sensitive), the group names MUST not contain hyphens.": "输入以逗号分隔的样本组名，例如 WT,WT,WT,Mut,Mut,Mut。组名的顺序必须与计数矩阵列中样本的顺序相匹配。注意：请仅使用字母、数字或下划线（区分大小写），组名中绝不能包含连字符。",
    "If you provide an annotation file, annotations will be added to the table(s) of differential expression results to provide descriptions for each gene, and used to label the top genes in the Volcano plot. Interactive Glimma Volcano and MD plots will also be generated. See Help section below.": "如果您提供了注释文件，注释将被添加到差异表达结果的表格中，以提供每个基因的描述，并用于标记火山图中的顶部基因。还将生成交互式的 Glimma 火山图和 MD 图。请参阅下面的帮助部分。",
    "You can choose to input the contrast information for the samples from a file or manually enter below. NOTE: Please only use letters, numbers or underscores (case sensitive), the group names MUST not contain hyphens. Use a hyphen to separate the groups you want to compare, as shown in the Help section below.": "你可以选择从文件中输入样本的对比信息，或者在下方手动输入。注意：请只使用字母、数字或下划线（区分大小写），组名中不能包含连字符。使用连字符来分隔你想要比较的组，如下方的帮助部分所示。",
    "Names of two groups to compare separated by a hyphen e.g. Mut-WT. If the order is Mut-WT the fold changes in the results will be up/down in Mut relative to WT. If you have more than one contrast enter each separately using the Insert Contrast button below. For differences between contrasts use e.g. (Mut1-Ctrl1)-(Mut2-Ctrl2). For more info, see Chapter 8 in the limma User's guide: https://www.bioconductor.org/packages/release/bioc/vignettes/limma/inst/doc/usersguide.pdf": "两个要比较的组的名称由一个连字符分隔，例如 Mut-WT。如果顺序是 Mut-WT，则结果中的倍数变化将是相对于 WT 的 Mut 中的上升/下降。如果您有多个对比，请使用下面的“插入对比”按钮单独输入每个对比。对于对比之间的差异，请使用例如 (Mut1-Ctrl1)-(Mut2-Ctrl2)。有关更多信息，请参见 limma 用户指南的第 8 章：https://www.bioconductor.org/packages/release/bioc/vignettes/limma/inst/doc/usersguide.pdf",
    "Select additional plots to output in the report": "在报告中选择要输出的其他图表",
    "Output a file containing the raw filtered counts if Filter Low Counts is selected. Default: No": "如果选择了“过滤低计数”选项，则输出包含原始过滤计数的文件。默认值：否。",
    "Output a tabular file showing the library sizes, normalisation factors and effective library sizes for the samples. Default: No": "输出一个表格文件，显示样本的库大小、标准化因子和有效库大小。默认值：否。",
    "If you want to apply a cut-off on a fold change the TREAT function can be used, see Help section below. Default: No": "如果你想对fold change应用一个阈值，可以使用TREAT函数，详见下面的帮助部分。默认值：不应用",
    "The top DE genes will be highlighted in the Volcano plot for each contrast and can be output in heatmap and stripchart PDFs (max 100). Default: 10.": "在火山图中，每个对比中的顶部差异表达（DE）基因将被突出显示，并且可以输出为热图和条形图PDF（最多100个）。默认值：10。",
    "Output an additional HTML file.": "输出一个额外的HTML文件。",
    "Can be used to generate plots for individual genes with plotDEXSeq": "可以用来生成单个基因的图表，使用plotDEXSeq绘图。",
    "Indicates whether two or more genes sharing an exon should be merged into an 'aggregate gene'. If 'no', the exons that can not be assiged to a single gene are ignored.": "指示是否应将共享一个外显子的两个或多个基因合并为一个“聚合基因”。如果为“否”，则忽略不能分配给单个基因的外显子。",
    "Created by prepare mode": "由准备模式创建",
    "If you generated your alignments using tophat, they are by default position sorted. Ignored for single-end data": "如果您使用tophat生成了比对结果，它们默认是按位置排序的。对于单端数据，这一点会被忽略。",
    "A DEXSeqResults object in RDS format. This can be output from the DEXSeq tool": "一个RDS格式的DEXSeq结果对象。这可以是DEXSeq工具的输出结果。",
    "Select to input a single gene ID or a list of IDs. Default: single gene ID": "选择输入单个基因ID或基因ID列表。默认：单个基因ID",
    "Gene identifier to visualize": "基因标识符可视化",
    "This should be a single tabular column with one gene per row and no header.": "这应该是一个单一的表格列，每行一个基因，没有标题。",
    "If Yes, the transcripts are displayed in the plot. Default: No": "如果选择是，转录本会在图表中显示。默认值：否。",
    "If Yes, the names of the transcripts are shown. Default: No": "如果选择是，则显示转录本的名称。默认值：否。",
    "If yes, provides a plot of the counts normalized by the size factors. Default: No": "如果需要，提供按大小因子标准化后的计数的绘图。默认：否。",
    "If yes, the samples gene expression effects are averaged out, leaving only exon usage coefficients. Default: No": "如果选择是，则样本的基因表达效应会被平均化，只留下外显子使用系数。默认值：否。",
    "Raw counts matrix produced by 'Build expression matrix for a de novo assembly of RNA-Seq data by Trinity' tool": "由“通过Trinity为RNA-Seq数据的去新组装构建表达矩阵”工具生成的原始计数矩阵",
    "File describing samples and replicates": "描述样本和重复的文件",
    "Generated by 'Differential expression analysis using a Trinity assembly' tool": "由“使用Trinity组装的差分表达分析”工具生成",
    "Default: 2 (meaning 2^(2) or 4-fold": "默认值：2（意味着2的2次方或4倍）",
    "Extract only up to the top number of DE features within each pairwise comparison. This is useful when you have massive numbers of DE features but still want to make useful heatmaps and other plots with more manageable numbers of data points.": "在每一对比较中只提取顶部数量的DE特征。这在你有大量DE特征但仍然想要制作有用的热图和其他图表时非常有用，但数据点的数量更易于管理。",
    "Instead of clustering samples or replicates hierarchically based on gene expression patterns, order columns according to order in the --samples file.": "而不是基于基因表达模式对样本或重复进行层次聚类，根据 --samples 文件中的顺序排列列。",
    "If more than 10000, heatmaps are not generated, since too time consuming": "如果超过10000个，热图就不会生成了，因为太耗时了。",
    "To examine GO enrichment, you must first run Trinotate and then extract all GO assignments for each gene feature, with the Trinotate script extract_GO_assignments_from_Trinotate_xls.pl": "要进行GO富集分析，您首先需要运行Trinotate，然后使用Trinotate脚本extract_GO_assignments_from_Trinotate_xls.pl提取每个基因特征的所有GO分配。",
    "Generated by 'Differential expression analysis using a Trinity assembly' tool. If not, be careful that the file names are identical to the file names of differential expression results, with extension '.count_matrix' instead of '.DE_results'.": "由“使用Trinity组装进行差异表达分析”工具生成。如果不是，请确保文件名与差异表达结果的文件名完全相同，扩展名为“.count_matrix”，而不是“.DE_results”。",
    "Generated by the Trinotate script extract_GO_assignments_from_Trinotate_xls.pl. Must have 2 columns: feature_id GO:000001,GO:00002,...": "由Trinotate脚本extract_GO_assignments_from_Trinotate_xls.pl生成。必须有2列：feature_id GO:000001,GO:00002,...",
    "Must have 2 columns: feature_id length": "必须有两列：feature_id 长度",
    "Do not convert thymine to uracile (T -> U). (--noconv)": "不要将胸腺嘧啶转换为尿嘧啶（T -> U）。(--noconv)",
    "Don't allow lonely pairs. (--noLP)": "不允许孤立配对。(--noLP)",
    "Don't allow pairing of G and U. (--noGU)": "不允许G和U配对。(--noGU)",
    "Don't allow pairing of G and U at the ends of helices. (--noClosingGU)": "不允许在螺旋的末端配对G和U。(--noClosingGU)",
    "List of regions known to be false positives (https://github.com/yhoogstrate/dr-disco/tree/master/share/)": "已知的假阳性区域列表（https://github.com/yhoogstrate/dr-disco/tree/master/share/）",
    "List of junctions know to be false positives (https://github.com/yhoogstrate/dr-disco/tree/master/share/)": "已知假阳性连接点列表（https://github.com/yhoogstrate/dr-disco/tree/master/share/）",
    "May need to be set to smaller values for read lengths smaller than 75bp.": "可能需要为小于75bp的读取长度设置较小的值。",
    "Larger numbers boost performance but result in suboptimal results. Values of 8 and below are considered to be very safe (this corresponds to initiate merging already from clusters that consist of 2 or 3 reads).": "较大的数字可以提高性能，但会导致次优的结果。8及以下的值被认为是非常安全的（这对应于从包含2个或3个读取的簇开始合并）。",
    "This GTF file requires the following attributes: gene_name, transcript_id and transcript_version": "这个GTF文件需要包含以下属性：gene_name（基因名称）、transcript_id（转录本ID）和transcript_version（转录本版本）。",
    "Dotplot file (RNA base pair probabilities)": "点图文件（RNA碱基对概率）",
    "The default parameters assign a base pair weight of 0.5 and penalize long base pairs.": "默认参数将碱基对权重分配为0.5，并对长碱基对进行惩罚。",
    "Slope of base pair distance penalty. (--alpha)": "碱基对距离惩罚的斜率。（--alpha）",
    "Turning point of base pair distance penalty. (--beta)": "碱基对距离惩罚的转折点。(--beta)",
    "Base pair weight factor. (--gamma)": "碱基对权重因子。（--gamma）",
    "Minimum penalty factor for base pairs. (--delta)": "碱基对的最小惩罚因子。（--delta）",
    "(Predicted) RNA secondary structure for comparison to a reference structure in dot-bracket format": "（预测的）RNA二级结构，用于与点-括号格式的参考结构进行比较",
    "Whether to compare the predicted (or specified) structure to a reference structure.": "是否将预测（或指定）的结构与参考结构进行比较。",
    "Reference structure in dot-bracket format": "点-括号格式的参考结构",
    "Use slide rule. (--no-slide-rule)": "使用滑尺规则。（--no-slide-rule）",
    "Use onflict rule. (--no-conflict-rule)": "使用冲突规则。（--no-conflict-rule）",
    "This is the 3rd column in GFF file. Only rows which have the matched feature type in the GTF annotation file will be included. `exon' by default.": "这是GFF文件中的第3列。只有那些在GTF注释文件中具有匹配特征类型的行才会被包含。默认为`exon`。",
    "GFF attribute to be used as feature identifier. The value of this attribute should match the first column of DESeq2 output (default: gene_id)": "GFF属性用作特征标识符。此属性的值应与DESeq2输出的第一列匹配（默认：gene_id）",
    "GFF attribute to be used as transcript identifier. This options is only used for DEXSeq output annotation. Exon numbers are counted for each transcript separately (default: transcript_id)": "GFF属性用作转录本标识符。此选项仅用于DEXSeq输出注释。每个转录本分别计算外显子编号（默认：transcript_id）",
    "Comma separated list of attributes from GFF file to include in output. These attributes should associate with your chosen GFF feature type.": "在输出中包含来自GFF文件的以逗号分隔的属性列表。这些属性应该与您选择的GFF特征类型相关联。",
    "Sequence input in pure fasta format": "纯FASTA格式的序列输入",
    "Sequence input in fasta format with locarna-specific extensions": "以fasta格式输入序列，并带有Locarna特定的扩展",
    "Note that local alignment mode usually requires to turn off the 'max-diff' heuristic (maximal difference for alignment traces).": "请注意，局部比对模式通常需要关闭“max-diff”启发式方法（比对迹线的最大差异）。",
    "Specify whether gaps at the ends (all, 5', or 3' ends) of the sequences should be penalized or allowed for free.": "指定序列的末端（所有末端、5'末端或3'末端）的间隙是否应该受到惩罚或允许免费。",
    "--iterations num": "--迭代次数 num",
    "The GTF file should contain: 1) coding and non-coding genes, 2) a 'transcript_id' and a 'gene_id' field for each 'exon' and 'CDS' row.": "GTF文件应包含：1) 编码和非编码基因，2) 每个'exon'和'CDS'行的'transcript_id'和'gene_id'字段。",
    "If yes, only exons/transcripts with the CCDS tag will be used as CCDS exons/transcripts, otherwise all exons/transcripts with a CDS region are going to be annotated as CCDS.": "如果选择是，那么只有带有CCDS标签的外显子/转录本将被用作CCDS外显子/转录本，否则所有带有编码区（CDS）的外显子/转录本都将被注释为CCDS。",
    "If yes, only exons/transcripts with the appris tag will be used, using only 1 transcript per appris gene (the appris_principal transcript or other appris transcript). If a gene does not have appris transcript, all the annotated transcript structures are used.": "如果选择是，只有带有appris标签的外显子/转录本将被使用，每个appris基因只使用1个转录本（appris_principal转录本或其他appris转录本）。如果一个基因没有appris转录本，则使用所有注释的转录本结构。",
    "Ribo-seq alignment file in BAM format.": "Ribo-seq 对齐文件，以 BAM 格式。",
    "Please run 'ribotaper part 1' to generate the table.": "请运行'ribotaper 第一部分'以生成表格。",
    "Please run 'ribotaper part 1' to generate the archive.": "请运行'ribotaper 第一部分'以生成归档文件。",
    "RNA-seq alignment file in BAM format.": "RNA-seq比对文件，格式为BAM。",
    "Read lengths, comma-separated values, which are used for P-site calculation. Example, 26,28,29. Please run 'ribotaper part 2' to deterimine appropriate values.": "读取长度，用逗号分隔的值，这些值用于P位点计算。例如，26,28,29。请运行'ribotaper part 2'以确定适当的值。",
    "Cutoffs, comma-separated values, which are used for P-sites calculation. Example, 9,12,12. Please run 'ribotaper part 2' to deterimine appropriate values.": "截断值，用逗号分隔的数值，用于P位点计算。例如，9,12,12。请运行'ribotaper part 2'以确定适当的值。",
    "The bam or sam file containing alignment of the read data.": "包含读取数据比对的bam或sam文件。",
    "You can use the default settings or set custom values for any of pipmir's parameters.": "您可以使用pipmir的默认设置，或者为它的任何参数设置自定义值。",
    "Minimum size of a precursor sequence (Default: 50)": "前体序列的最小大小（默认值：50）",
    "Maximum size of a precursor sequence (Default: 500)": "前体序列的最大大小（默认值：500）",
    "The step size used to identifiy the precursor from the minimum to the maximum possible size of precursor (Default: 2)": "用于从最小到最大可能大小的前体识别前体的步长大小（默认值：2）",
    "Minimum read count for a mature to be considered expressed (Default: 10)": "成熟转录本被认为表达的最小读取计数（默认值：10）",
    "The gene ids should be in agreement in the files of annotation, known pairs and read count.": "基因ID在注释文件、已知配对和读数计数文件中应该一致。",
    "A known pair of cis-natural antisense transcripts": "一对已知的顺式天然反义转录本",
    "Aligned small RNA-Seq reads must be single end and should not be fragmented in the library preparation": "对齐的小RNA-Seq测序读取必须是单端的，并且在文库制备中不应被断裂。",
    "By selecting this file, sequences will be provided in the corresponding column in the output file": "选择此文件后，序列将在输出文件的相应列中提供",
    "Exon-type entries are often used to measure expression. Because of the small size of the fragments it may be desired to add an offset to allow less stringent read counting.": "外显子类型的条目通常用于测量表达。由于片段的小尺寸，可能需要添加一个偏移量以允许不那么严格的读取计数。",
    "FASTA-format": "FASTA格式",
    "Predicted targets are printed out if they are equal to or lower than the cutoff score specified. DEFAULT = 4": "如果预测的目标等于或低于指定的截止分数，则会打印出来。默认值 = 4",
    "Use this option if the database is genomic DNA.": "如果数据库是基因组DNA，请使用这个选项。",
    "Output format for small RNA-target pairs (Default = classic).": "小RNA-靶标对的输出格式（默认 = 经典）。",
    "Try e.g. cxcl2": "尝试例如 cxcl2",
    "Choose a single motif, e.g. ATTTA": "选择一个单一的模式，例如 ATTTA",
    "Make use of persistent gene annotations! Gene annotations should only be different if different reference genome builds were used": "使用持久的基因注释！只有在使用了不同的参考基因组构建时，基因注释才应该不同",
    "You can use the default settings or set custom values for any FuMa parameter": "您可以使用默认设置或为任何FuMa参数设置自定义值",
    "Overlap is the most sensitive but also more sensitive for long gene artefacts; subset is the recommended technique and EGM is conservative": "重叠是最为敏感的方法，但对长基因伪影也更为敏感；子集是推荐的技术，而EGM则更为保守。",
    "Only a limited number of file formats support this feature": "只有有限数量的文件格式支持此功能",
    "This settings is not recommended when fusion genes detected in DNA-Seq are used": "这种设置不适用于在DNA-Seq中检测到的融合基因",
    "Gene-name based matching is more sensitive to long genes. This is the gene size used to mark fusion genes spanning a 'long gene' as reported the output. Use 0 to disable this feature": "基于基因名称的匹配对长基因更为敏感。这是用于标记跨越“长基因”的融合基因的基因大小，如输出中所报告的。使用0来禁用此功能。",
    "Blast databases, e.g. nt, refseq_genomic,..": "Blast 数据库，例如 nt, refseq_genomic 等",
    "Output of abundance_estimates_to_matrix tool": "丰度估计转矩阵工具的输出",
    "e.g. CondA_replicate1": "例如 CondA_重复1",
    "e.g. ConditionA": "例如：条件A",
    "RData file produced by 'Extract and cluster differentially expressed transcripts from a Trinity assembly' tool": "由“从Trinity组装中提取和聚类差异表达转录本”工具生成的RData文件",
    "Give a percent value": "给出一个百分比值",
    "This does not leverage the already hierarchically clustered genes, and instead uses a least-sum-of-squares method to define exactly k gene clusters.": "这不会利用已经分层聚类的基因，而是使用最小二乘法来精确定义k个基因簇。",
    "riboWaltz only works for read alignments based on transcript coordinates": "riboWaltz 仅适用于基于转录本坐标的读段比对。",
    " All characters before the first occurrence of the specified separator are kept": "在第一次出现的指定分隔符之前的所有字符都被保留",
    "Useful for advanced plotting using riboWaltz-plot tool": "适用于使用riboWaltz-plot工具进行高级绘图",
    "This parameter is considered only if one of 5' or 3' end extrimity was chosen": "如果选择了5'端或3'端的外端性，则仅考虑此参数",
    "If not checked, the second to last codon is used instead": "如果未勾选，则使用倒数第二个密码子代替",
    "If not checked, number of read foot prints per region trturned": "如果未勾选，每个区域返回的读取足迹数量",
    "Minimum mRNA length in bp, mRNA that are shorter than this value will be skipped (--minimum_length).": "最小mRNA长度，以碱基对(bp)为单位，小于该值的mRNA将被跳过(--minimum_length)。",
    "Used for plotting histogram (--lower-bound)": "用于绘制直方图（--lower-bound）",
    "Used for plotting histogram (--upper-bound)": "用于绘制直方图（--upper-bound）",
    "Minimum number of reads mapped to a transcript (--minCov).": "转录本上映射的最小读数（--minCov）。",
    "Number of equal-spaced nucleotide positions picked from mRNA. Note: if this number is larger than the length of mRNA (L), it will be halved until is's smaller than L. (--sample-size).": "从mRNA中选取等间距核苷酸位置的数量。注意：如果这个数字大于mRNA的长度(L)，它将被减半，直到它小于L。(--sample-size)。",
    "Subtract background noise (estimated from intronic reads). Only use this option if there are substantial intronic reads (--subtract-background).": "减去背景噪声（从内含子读取中估计）。只有在有大量内含子读取时才使用此选项（--subtract-background）。",
    "Genomic BED file containing protein binding sites": "基因组 BED 文件包含蛋白质结合位点",
    "Genomic FASTA file for extracting sequences defined in BED file": "用于从BED文件定义的序列中提取序列的基因组FASTA文件",
    "By default model is initialized with Baum-Welch optimized sequence motif": "默认情况下，模型使用Baum-Welch优化的序列基序进行初始化",
    "Model parameters are sampled from among the top f configurations (default: 10); set f to 0 in order to include all possible configurations": "模型参数是从排名靠前的f个配置中抽样的（默认值：10）；将f设置为0以包含所有可能的配置",
    "The iterative algorithm is terminated if the given reduction in sequence structure log-likelihood is not reached for any of the 3 last measurements (default: 10)": "迭代算法在给定的序列结构对数似然减少量未达到任何3个最后测量值时终止（默认值：10）",
    "Train only using best structure for each sequence (default: use all structures)": "仅使用每个序列的最佳结构进行训练（默认：使用所有结构）",
    "Set to output FASTA file and corresponding structures file for the extracted genomic regions": "设置输出FASTA文件和相应的结构文件，用于提取的基因组区域",
    "Set to output logo_global.png, logo_hairpin.png, and logo_best_sequences.png files": "设置以输出logo_global.png、logo_hairpin.png和logo_best_sequences.png文件",
    "Set to output final_model.xml, logo_best_sequences.txt, logo_global.txt, logo_hairpin.txt files": "设置输出文件为final_model.xml、logo_best_sequences.txt、logo_global.txt、logo_hairpin.txt。",
    "Make sure the FASTA corresponds to the same file used with kallisto quant --fusion": "确保FASTA文件与kallisto quant --fusion使用的文件相同。",
    "If your transcriptome of interest is not listed, contact your Galaxy administrator": "如果您感兴趣的转录组没有列出，请联系您的Galaxy管理员",
    "kallisto will estimate this from the reads, default: 400": "Kallisto 将从读取数据中估计这个值，默认值：400",
    "default: 2": "默认值：2",
    "default: 31": "默认：31",
    "Warning: this will probably lead to an increase in the number of false positives reported": "警告：这可能会导致报告的假阳性数量增加",
    "Genomic BED file used for single sample analysis": "用于单样本分析的基因组BED文件",
    "Run GO term enrichment analysis (supported genome versions: hg19, hg38, mm9, mm10, dm3)": "运行GO术语富集分析（支持的基因组版本：hg19、hg38、mm9、mm10、dm3）",
    "This database file is needed for gene set enrichment analysis (supported genome versions: hg19, hg38, mm9, mm10, dm3). For non-human species, the human MSigDB will be automatically converted.": "这个数据库文件是用于基因集富集分析的（支持的基因组版本：hg19、hg38、mm9、mm10、dm3）。对于非人类物种，人类MSigDB将自动转换。",
    "Randomly sample down query regions to (N). To activate sampling a positive integer value smaller than the total number of query regions should be given. Default value is 0 (i.e. no downsampling applied)": "随机将查询区域下采样到(N)。要激活下采样，应提供一个正整数值，该值小于查询区域的总数。默认值为0（即不应用下采样）。",
    "Output single sample analysis raw data tables that are used for plots/tables as text files": "输出用于图表/表格的单个样本分析原始数据表，以文本文件形式。",
    "Genomic BED files used for multi sample analysis. NOTE that the dataset name inside the Galaxy history is used as identifier for each set, resulting in plots with the Galaxy dataset names corresponding to their respective input BED files.": "用于多样本分析的基因组BED文件。请注意，Galaxy历史中的数据集名称用作每组的标识符，从而在图表中使用Galaxy数据集名称对应于各自的输入BED文件。",
    "Output annotation summaries from overlap operations": "输出重叠操作的注释摘要",
    "Run motif analysis for each input dataset and each transcript region": "为每个输入数据集和每个转录区域运行基序分析",
    "Alignment needs to be formatted in ClustalW or MAF format": "序列比对需要格式化为ClustalW或MAF格式",
    "If your alignments contain blocks of long genomic regions it is usually not reasonable to score these long regions as a whole.": "如果您的比对包含长基因组区域的块，通常将这些长区域作为一个整体进行评分是不合理的。",
    "Break all blocks longer than that.": "打破所有超过该长度的区块。",
    "Try to create blocks of this size.": "尝试创建这个大小的块。",
    "If enables RNAcode is called separately for each block. This might be a reasonable strategy if RNAcode fails for some blocks.": "如果启用RNAcode，则会为每个区块单独调用。如果RNAcode对某些区块失败，这可能是一个合理的策略。",
    "Show only regions that have a p-value below the given number. By default all hits are shown.": "只显示具有低于给定数值的p值的区域。默认情况下，显示所有命中。",
    "Number of random alignments that are sampled to calculate the p-value. RNAcode estimates the significance of a coding prediction by sampling a given number of random alignments. Default is 100 which gives reasonably stable p-values that are useful for assessing the relevance of a prediction.": "随机比对的数量，用于采样以计算p值。RNAcode通过采样给定数量的随机比对来估计编码预测的显著性。默认值为100，这可以给出足够稳定的p值，用于评估预测的相关性。",
    "Setting this option stops the sampling process as soon as it is clear that the best hit will not fall below the given p-value cutoff. For example, assume a p-value cutoff of 0.05 (see --cutoff) and a sample size of 1000 is given (see --num-samples). As soon as 50 random samples score better than the original alignment, the process is stopped and all hits in the original alignment are reported as p>0.05 (or by convention as 1.0 in gtf and tabular output).": "设置此选项会在明确最佳命中不会低于给定的p值截止值时立即停止抽样过程。例如，假设给定的p值截止值为0.05（见--cutoff）并且给定了1000的样本大小（见--num-samples）。一旦有50个随机样本的得分优于原始对齐，该过程就会停止，并在原始对齐中报告所有命中为p>0.05（或按照惯例，在gtf和表格输出中为1.0）。",
    "By default all positive scoring segments are shown in the output if they fall below the given p-value cutoff. If two hits overlap (different frame or different strand) and --best_region is given only the hit with the highest score is shown. Strong coding regions often lead to statistically significant signals also in other frames. These hits are suppressed by this option and only the correct reading frame is reported.": "默认情况下，如果所有正得分片段的p值低于给定的阈值，则它们都会显示在输出中。如果两个命中重叠（不同的框架或不同的链），并且给出了--best_region参数，则只显示得分最高的命中。强编码区域通常也会导致其他框架中的统计学显著信号。这些命中通过此选项被抑制，并且只报告正确的读取框架。",
    "This options shows only the one single best hit for each alignment.": "这个选项只显示每个比对的单一最佳命中。",
    "See the appendix of the Paper for an explanation for the meaning of these parameters. Default: '-10.0,-4.0,-2.0,-8.0'": "请参阅论文的附录部分，了解这些参数的含义。默认值：'-10.0,-4.0,-2.0,-8.0'",
    "The generated plots are resolution independent vector graphics that can be included in any graphics software. For each high scoring segment below a given cutoff (see --eps-cutoff) a file named hss-N.eps is created (N is the running number of the high scoring segment)": "生成的图形是分辨率独立的矢量图形，可以包含在任何图形软件中。对于每个低于给定截止值（见 --eps-cutoff）的高得分片段，都会创建一个名为 hss-N.eps 的文件（N 是高得分片段的运行编号）。",
    "For larger numbers of high scoring segments output to a collection can be slow. Choose zipped or single page eps in this case.": "对于输出大量高得分片段的情况，输出到集合可能会很慢。在这种情况下，选择压缩或单页eps。",
    "-1.0 to deactivate": "-1.0 表示禁用",
    "(--mode) For a description of the different modes see below.": "(--mode) 关于不同模式的描述，请参见下文。",
    "In the calculation of the pf use scale*mfe as an estimate for the ensemble free energy (used to avoid overflows). The default is 1.07, useful values are 1.0 to 1.2. Occasionally needed for long sequences.": "在计算自由能（pf）时，使用scale*mfe作为整个自由能的估计（用于避免溢出）。默认值是1.07，有用的值在1.0到1.2之间。对于长序列，偶尔需要使用。",
    "Allow pairing of G and U at the ends of helices. --noClosingGU": "允许在螺旋的末端配对G和U。--noClosingGU",
    " Include special tabulated stabilizing energies for tri-, tetra- and hexaloop hairpins. Mostly for testing. (--noTetra)": "包含特殊的三环、四环和六环发夹结构的稳定能量表格。主要用于测试。（--noTetra）",
    "Avoids confusion with DNA sequences (--noconv)": "避免与DNA序列混淆（--noconv）",
    "By setting the threshold the base pair probabilities that are included in the output can be varied. By default only those exceeding 1e−5 in probability will be shown as squares in the dot plot. Changing the threshold to any other value allows for increase or decrease of data.": "通过设置阈值，可以改变包含在输出中的碱基对概率。默认情况下，只有概率超过1e-5的碱基对会在点图中以正方形显示。将阈值更改为任何其他值，可以增加或减少数据。",
    "Its argument is a comma separated list of additionally allowed pairs. If the first character is '-' then AB will imply that AB and BA are allowed pairs. e.g. RNAfold -nsp -GA  will allow GA and AG pairs. Nonstandard pairs are given 0 stacking energy.": "它的参数是一个逗号分隔的额外允许配对列表。如果第一个字符是'-'，则AB将意味着AB和BA是允许的配对。例如，RNAfold -nsp -GA 将允许GA和AG配对。非标准配对的堆叠能量为0。",
    " The argument provided with this option enables to scale the thermodynamic temperature used in the Boltzmann factors independently from the temperature used to scale the individual energy contributions of the loop types.": "这个选项提供的参数允许独立于用于缩放环类型个体能量贡献的温度来缩放用于Boltzmann因子中的热力学温度。",
    "Usually, constraints provided from input file are only applied to a single sequence alignment. Therefore, RNAalifold will stop its computation and quit after the first input alignment was processed. Using this switch, RNAfold processes all sequence alignments in the input and applies the same provided constraints to each of them.": "通常，从输入文件提供的约束只应用于单个序列比对。因此，RNAalifold将在处理完第一个输入比对后停止计算并退出。使用此开关，RNAfold将处理所有输入的序列比对，并将相同的提供的约束应用于它们每一个。",
    "Specify stabilizing effect of ligand binding to a particular sequence/structure motif. Some ligands binding to RNAs require and/or induce particular sequence and structure motifs. For instance they bind to an interior loop, or small hairpin loop. If for such cases a binding free energy is known, the binding and therefore stabilizing effect of the ligand can be included in the folding recursions. Interior loop motifs are specified as concatenations of 5’ and 3’ motif, separated by an ’&’ character. Energy contributions must be specified in kcal/mol.": "指定配体结合对特定序列/结构基序的稳定效应。一些结合到RNA的配体需要和/或诱导特定的序列和结构基序。例如，它们结合到一个内部环或小发夹环。如果对于这种情况已知结合自由能，则配体的结合（从而稳定）效应可以包含在折叠递归中。内部环基序被指定为5'和3'基序的串联，由一个'&'字符分隔。能量贡献必须以kcal/mol指定。",
    "If this flag is active, RNAfold ignores any IDs retrieved from the input and automatically generates an ID for each sequence.": "如果这个标志被激活，RNAfold将忽略从输入中检索到的任何ID，并自动为每个序列生成一个ID。",
    "If this parameter is set, each sequence will be prefixed with the provided string. Hence, the output files will obey the following naming scheme: 'prefix_xxxx_ss.ps' (secondary structure plot), 'prefix_xxxx_dp.ps' (dot−plot), 'prefix_xxxx_aln.ps' (annotated sequence), etc. where xxxx is the sequence number beginning with the second sequence in the input. Use this setting in conjunction with the −−continuous−ids flag to assign IDs beginning with the first input sequence.": "如果设置了此参数，每个序列将用提供的字符串作为前缀。因此，输出文件将遵循以下命名方案：'prefix_xxxx_ss.ps'（二级结构图），'prefix_xxxx_dp.ps'（点图），'prefix_xxxx_aln.ps'（注释序列）等，其中xxxx是序列编号，从输入中的第二个序列开始。使用此设置与——continuous-ids标志结合使用，以从第一个输入序列开始分配ID。",
    "When sequences IDs are automatically generated, they receive an increasing number, starting with 1. This number will always be left−padded by leading zeros, such that the number takes up a certain width. Using this parameter, the width can be specified to the users need. We allow numbers in the range [1:18].": "当序列ID自动生成时，它们会接收一个递增的数字，从1开始。这个数字将始终用前导零左对齐，以使数字占据一定的宽度。使用此参数，可以根据用户的需求指定宽度。我们允许的数字范围是[1:18]。",
    "When sequence IDs are automatically generated, they receive an increasing number, usually starting with 1. Using this parameter, the first number can be specified to the users requirements. Note: negative numbers are not allowed. Note: Setting this parameter implies continuous sequence IDs, i.e. it activates the −−continuous−ids flag..": "当序列ID自动生成时，它们会接收一个递增的数字，通常从1开始。使用此参数，可以根据用户的要求指定第一个数字。注意：不允许使用负数。注意：设置此参数意味着连续的序列ID，即它激活了——continuous-ids标志。",
    "This option is ignored if the input file is GFF": "如果输入文件是GFF格式，则忽略此选项",
    "A group of peaks with the same chr/start/stop but different summits": "一组具有相同染色体/起始/终止位置但不同峰顶的峰群",
    "Avoids confusion with DNA sequences": "避免与DNA序列混淆",
    "If this flag is active, RNAalifold ignores any IDs retrieved from the input and automatically generates an ID for each alignment.": "如果这个标志被激活，RNAalifold将忽略从输入中检索到的任何ID，并为每个比对自动生成一个ID。",
    "If this parameter is set, each alignment will be prefixed with the provided string. Hence, the output files will obey the following naming scheme: 'prefix_xxxx_ss.ps' (secondary structure plot), 'prefix_xxxx_dp.ps' (dot−plot), 'prefix_xxxx_aln.ps' (annotated alignment), etc. where xxxx is the alignment number beginning with the second alignment in the input. Use this setting in conjunction with the −−continuous−ids flag to assign IDs beginning with the first input alignment.": "如果设置了此参数，每个比对前都会加上提供的字符串。因此，输出文件将遵循以下命名方案：'prefix_xxxx_ss.ps'（二级结构图），'prefix_xxxx_dp.ps'（点图），'prefix_xxxx_aln.ps'（注释比对）等，其中xxxx是比对编号，从输入中的第二个比对开始。使用此设置与——continuous-ids标志结合使用，以从第一个输入比对开始分配ID。",
    "When alignments IDs are automatically generated, they receive an increasing number, starting with 1. This number will always be left−padded by leading zeros, such that the number takes up a certain width. Using this parameter, the width can be specified to the users need. We allow numbers in the range [1:18].": "当比对ID自动生成时，它们会接收一个递增的数字，从1开始。这个数字将始终用前导零进行左对齐，以便该数字占据一定的宽度。使用此参数，用户可以根据需要指定宽度。我们允许的数字范围是[1:18]。",
    "When alignment IDs are automatically generated, they receive an increasing number, usually starting with 1. Using this parameter, the first number can be specified to the users requirements. Note: negative numbers are not allowed. Note: Setting this parameter implies continuous alignment IDs, i.e. it activates the −−continuous−ids flag..": "当比对ID自动生成时，它们会接收一个递增的数字，通常从1开始。使用此参数，可以根据用户的要求指定第一个数字。注意：不允许使用负数。注意：设置此参数意味着连续的比对ID，即它激活了——continuous-ids标志。",
    "Calculate the partition function and base pairing probability matrix in addition to the mfe structure. Default is calculation of mfe structure only.": "计算配分函数和碱基配对概率矩阵，以及最小自由能（mfe）结构。默认情况下仅计算mfe结构。",
    "In addition to everything listed under the -a option, read in initial monomer concentrations and compute the expected equilibrium concentrations of the 5 possible species (AB, AA, BB, A, B).": "除了在 -a 选项下列出的所有内容外，还要读取初始单体浓度，并计算 5 种可能物种（AB、AA、BB、A、B）的预期平衡浓度。",
    "The table consits of arbitrary many lines with just two numbers (the concentration of sequence A and B). This option will automatically toggle the −c (and thus -a and --all_pf) options": "表由任意多行组成，每行只有两个数字（序列A和B的浓度）。此选项将自动切换-c（因此-a和--all_pf）选项",
    "take into account G Quadruplex formation": "考虑G四链体形成",
    "Calculate an MEA (maximum expected accuracy) structure. The parameter gamma tunes the importance of correctly predicted pairs versus unpaired bases. Thus, for small values of gamma the MEA structure will contain only pairs with very high probability. where the expected accuracy is computed from the pair probabilities: each base pair (i,j) gets a score 2*gamma*p_ij and the score of an unpaired base is given by the probability of not forming a pair.": "计算一个MEA（最大期望准确度）结构。参数gamma调节正确预测的配对与未配对碱基的重要性。因此，对于小的gamma值，MEA结构将只包含具有非常高概率的配对。其中，期望准确度是从配对概率计算得出的：每个碱基对（i,j）获得一个分数2*gamma*p_ij，而未配对碱基的分数由不形成配对的概率给出。",
    "Output 'most informative sequence' instead of simple consensu": "输出“最具信息量的序列”而不是简单的共识序列",
    "Compute a certain number of random structures with a probability dependend on the partition function. Print out the energies and probabilities of the backtraced structures.": "计算一定数量的随机结构，这些结构的概率取决于配分函数。打印出回溯结构的能量和概率。",
    "Take into account G Quadruplex formation (-g)": "考虑G四链体形成（-g）",
    "Compute the structure conservation index (SCI) for the MFE consensus structure of the alignment": "计算对齐的最小自由能(MFE)一致结构的结构保守指数(SCI)。",
    "Produce a colored version of the consensus strcture plot": "生成一个共识结构图的彩色版本",
    "Produce a colored and structure annotated alignment": "生成一个带有颜色和结构注释的对齐结果",
    "Use continuous alignment ID numbering when no alignment ID can be retrieved from input data.": "当无法从输入数据中检索到对齐ID时，使用连续的对齐ID编号。",
    "Usually, constraints provided from input file are only applied to a single sequence alignment. Therefore, RNAalifold will stop its computation and quit after the first input alignment was processed. Using this switch, RNAalifold processes all sequence alignments in the input and applies the same provided constraints to each of them.": "通常，从输入文件提供的约束只应用于单个序列比对。因此，RNAalifold将在处理完第一个输入比对后停止计算并退出。使用此开关，RNAalifold将处理所有输入的序列比对，并将相同的提供的约束应用于它们。",
    "Stockholm formatted alignment files have the possibility to store a secondary structure string in one of if ('#=GC') column annotation meta tags. The corresponding tag name is usually 'SS_cons', a consensus secondary structure. Activating this flag allows one to use this consensus secondary structure from the input file as structure constraint. Note: Activating this flag implies −−constraint.": "斯德哥尔摩格式的比对文件有可能在一个'#=GC'列注释元标签中存储一个二级结构字符串。相应的标签名称通常是'SS_cons'，一个共识二级结构。激活这个标志允许使用输入文件中的这个共识二级结构作为结构约束。注意：激活这个标志意味着——约束。",
    "Currently, the only data conversion method available is that of to Deigan et al 2009. This method is the default. The slope 'm' and the intercept 'b' can be set to a non−default value if necessary. Otherwise m=1.8 and b=−0.6 as stated in the paper mentioned before.": "目前，可用的唯一数据转换方法是Deigan等人2009年的方法。此方法是默认方法。如果需要，可以将斜率'm'和截距'b'设置为非默认值。否则，如前文提到的论文所述，m=1.8，b=-0.6。",
    "If your transcriptome of interest is not listed, contact your Galaxy admin": "如果您感兴趣的转录组没有列出，请联系您的Galaxy管理员",
    "There is a tradeoff here between the distinctiveness of the k-mers and their robustness to errors. The shorter the k-mers, the more robust they will be to errors in the reads, but the longer the k-mers, the more distinct they will be. We generally recommend using a k-mer size of at least 20.": "在这里，k-mer的独特性与它们对错误鲁棒性之间存在权衡。k-mer越短，它们对读取中的错误就越鲁棒，但k-mer越长，它们就越独特。我们通常建议使用至少20的k-mer大小。",
    "Calculates the aggregated gene-level abundance estimations. This file should be eiher a GTF file or tab-delimited format where each line contains the name of a transcript and the gene to which it belongs separated by a tab.": "计算聚合的基因水平丰度估计。该文件应该是一个GTF文件或制表符分隔格式，其中每行包含一个转录本的名称和它所属的基因，两者之间用制表符分隔。",
    "Larger values will reduce memory usage, but may decrease the fidelity of bias modeling results.": "较大的值将减少内存使用，但可能会降低偏差建模结果的保真度。",
    "Larger values speed up effective length correction, but may decrease the fidelity of bias modeling results.": "较大的值可以加快有效长度校正的速度，但可能会降低偏差建模结果的准确性。",
    "When this flag is set, if the intersection of the quasi-mappings for the left and right is empty, then all mappings for the left and all mappings for the right read are reported as orphaned quasi-mappings.": "当设置此标志时，如果左侧和右侧准映射的交集为空，则报告左侧的所有准映射和右侧的所有准映射读取为孤立的准映射。",
    "If single end reads are being used for quantification, or there are an insufficient number of uniquely mapping reads when performing paired-end quantification to estimate the empirical fragment length distribution, then use this value to calculate effective lengths.": "如果使用单端测序数据进行定量分析，或者在执行双端定量分析时，唯一映射的读段数量不足，无法估计经验片段长度分布，则使用此值计算有效长度。",
    "The standard deviation used in the fragment length distribution for single-end quantification or when an empirical distribution cannot be learned.": "在单端定量时使用的片段长度分布的标准差，或者当无法学习到经验分布时使用的标准差。",
    "Reads mapping to more than this many places won't be considered.": "读取映射到超过这个数量的地方不会被考虑。",
    "Disables effective length correction when computing the probability that a fragment was generated from a transcript. If this flag is passed in, the fragment length distribution is not taken into account when computing this probability.": "在计算一个片段是否来自转录本时，禁用有效长度校正。如果传入了这个标志，计算这个概率时就不会考虑片段长度分布。",
    "Use Variational Bayesian EM algorithm rather than the traditional EM angorithm for optimization": "使用变分贝叶斯EM算法而不是传统的EM算法进行优化",
    "This option will discard orphaned fragments. This only has an effect on paired-end input, but enabling this option will discard, rather than count, any reads where only one of the paired fragments maps to a transcript.": "这个选项将丢弃孤立的片段。这只对成对末端输入有效，但启用这个选项将丢弃，而不是计数，任何只有一个配对片段映射到转录本的读取。",
    "This traditional approach works by convolving the FLD with the characteristic function over each transcript.": "这种传统方法通过将FLD与每个转录本上的特征函数进行卷积来工作。",
    "This is mutually exclusive with Gibbs": "这与Gibbs是互斥的",
    "A transcript annotation (GFF3 or GTF) file produced by cufflinks, cuffcompare, or other source.": "由cufflinks、cuffcompare或其他来源生成的转录本注释（GFF3或GTF）文件。",
    "Sets the maximum number of fragments a locus may have before being skipped. Default: 500,000": "设置一个位点在被跳过之前可能拥有的最大片段数。默认值：500,000",
    "The default (500) is very short and meant for testing purposes only.": "默认值（500）非常短，仅用于测试目的。",
    "stop when a stop-structure is reached": "在达到终止结构时停止",
    "Use the Metropolis rule for rate between two neighboring conformations, i.e. k=min{1,exp(−dE/RT)}. By default Kinfold uses the symmetric Kawasaki rule k=exp(−dE/2RT).": "使用 Metropolis 规则来计算两个相邻构象之间的速率，即 k=min{1,exp(−dE/RT)}。默认情况下，Kinfold 使用对称的 Kawasaki 规则 k=exp(−dE/2RT)。",
    "Print only those parts of the trajectory that stays below energy.": "只打印轨迹中能量低于指定值的部分。",
    "Default is calculation of mfe structure only.": "默认情况下只计算最小自由能结构。",
    "Allow pairing of G and U at the ends of helices.": "允许在螺旋的末端配对G和U。",
    " Include special tabulated stabilizing energies for tri-, tetra- and hexaloop hairpins. Mostly for testing.": "包括特殊的表格化稳定能量，用于三环、四环和六环发夹结构。主要用于测试。",
    "Maximal distance between two paired bases. (-L)": "两个配对碱基之间的最大距离。(-L)",
    "An optional argument may be supplied to set the threshold": "可以提供一个可选参数来设置阈值",
    "-R: This parameter could be negative to search for exact solutions, but in that case the program might not terminate. If you need this functionality contact your admin or run the program locally.": "-R: 这个参数可以是负数，用于搜索精确解，但在那种情况下程序可能无法终止。如果你需要这个功能，请联系你的管理员或在本地运行程序。",
    "... global with respect to the query model and local with respect to the target database.": "全局查询模型与目标数据库局部。",
    "if 0: one-time arbitrary seed": "如果0：一次性任意种子",
    "This is the amount of probability mass within the HMM posterior probabilities that is considered negligible.": "这是在隐马尔可夫模型（HMM）后验概率中被认为可以忽略的概率质量的大小。",
    "Cmalign will attempt to iteratively tighten the HMM bands it uses to constrain the alignment by raising the tau parameter and recalculating the bands until the total matrix size needed falls below 'x' megabytes or the maximum allowable tau value (0.05 by default, but changeable with --maxtau) is reached.": "Cmalign将尝试通过提高tau参数并重新计算带宽来迭代地缩小用于约束对齐的HMM带宽，直到所需的总矩阵大小降至'x'兆字节以下，或者达到最大允许的tau值（默认为0.05，但可以通过——maxtau更改）。",
    "Allows CM alignment within practical memory limits, reducing the memory required for alignment. WARNING: It can only be used when --cyk,--noprob,--nonbanded,--notrunc are all enabled": "允许在实际内存限制内进行CM比对，减少比对所需的内存。警告：只有当--cyk、--noprob、--nonbanded、--notrunc都被启用时，才能使用。",
    "The alignment from the file is held fixed. This allows you to align sequences to a model with cmalign and view them in the context of an existing trusted multiple alignment.": "文件中的比对是固定的。这允许您使用cmalign将序列与模型比对，并在现有可信的多重比对的上下文中查看它们。",
    "... instead of RNA ones.": "... 而不是 RNA 的。",
    "Do not include any insertions relative to the consensus model.": "不要包含与共识模型相对的任何插入。",
    "will only work for alignments of up to 100,000 sequences or a total of 100,000,000 aligned nucleotides": "只适用于最多100,000个序列的比对，或者总共100,000,000个比对核苷酸。",
    "Set the target mean match state relative entropy for the filter p7 HMM": "设置过滤器 p7 HMM 的目标平均匹配状态相对熵",
    "Use a mzimum likelihood p7 HMM built from the CM as the filter HMM": "使用从CM构建的最大似然p7 HMM作为过滤HMM",
    "... which allows the alignment to span two or more subsequences if necessary": "...这允许比对跨越两个或更多子序列（如果必要的话）",
    "A CM file must be calibrated with cmcalibrate before it can be used in cmsearch or cmscan. cmcalibrate is very slow. It takes a couple of hours to calibrate a single average sized CM on a single CPU": "一个CM文件在使用cmsearch或cmscan之前必须使用cmcalibrate进行校准。cmcalibrate非常慢。在单个CPU上校准一个平均大小的CM需要几个小时。  解析： - CM文件：指的是一个模型文件，这里不翻译。 - cmcalibrate：一个软件工具，用于校准CM文件，保持原文。 - cmsearch和cmscan：两个软件工具，用于搜索和扫描序列数据库，保持原文。 - CPU：一个硬件设备，保持原文。",
    "Set random seq length to search in Mb (megabases)": "设置随机序列搜索长度为兆碱基（Mb）",
    "If the seed is nonzero, stochas-tic simulations will be reproducible. If 0, the random number generator is seeded arbitrarily": "如果种子非零，则随机模拟将是可重现的。如果为0，则随机数生成器将任意地设置种子。",
    "The beta parameter is the amount of probability mass excluded during band calculation, higher values of beta give greater speedups but sacrifice more accuracy than lower values.": "beta参数是在带计算期间排除的概率质量的量，较高的beta值可以提供更大的加速，但比较低的值牺牲更多的准确性。",
    "This will slow down calibration": "这将减慢校准",
    "This is not recommended unless you plan on using the same option to cmsearch and/or cmscan": "这并不推荐，除非你计划在cmsearch和/或cmscan中使用相同的选项。",
    "Use the background null model of the CM to generate the random sequences, instead of the more realistic HMM. Unless the CM was built using the --null option to cmbuild, the background null model will be 25% each A, C, G and U": "使用CM的背景零模型生成随机序列，而不是更现实的HMM。除非CM是使用--null选项构建的，否则背景零模型将是A、C、G和U各占25%。",
    "Generate the random sequences using the nucleotide distribution from the sequence file": "使用序列文件中的核苷酸分布生成随机序列",
    "A tar file contains the four auxillury files suffixed .i1{fimp}. These files are generated after pressing the cm files using cmpress": "一个tar文件包含了四个辅助文件，它们的后缀为.i1{fimp}。这些文件是在使用cmpress处理cm文件后生成的。",
    "Without the use of this option, the search space size changes for each query sequence": "如果不使用这个选项，每个查询序列的搜索空间大小都会发生变化",
    "report extra information; mainly useful for debugging": "报告额外信息；主要用于调试",
    "Inclusion thresholds are stricter than reporting thresholds. Inclusion thresholds control which hits are considered to be reliable enough to be included in an output alignment or in a possible subsequent search round, or marked as significant (”!”) as opposed to questionable (”?”) in hit output.": "包含阈值比报告阈值更严格。包含阈值控制哪些命中被认为是足够可靠的，以便包含在输出对齐中，或在可能的后续搜索轮次中，或在命中输出中被标记为显著（\"!\"）而不是可疑（\"?\"）。",
    "of <= X as the hit inclusion threshold.": "将 <= X 作为命中包含阈值。",
    "of >= X as the hit inclusion threshold.": "将 >= X 作为命中包含的阈值。",
    "Reporting thresholds control which hits are reported in output files": "报告阈值控制输出文件中报告哪些命中",
    "of <= X as the hit reporting threshold. The default is 10.0, meaning that on average, about 10 false positives will be reported per query, so you can see the top of the noise and decide for yourself if it’s really noise.": "将 <= X 作为命中报告阈值。默认值为 10.0，意味着每个查询平均会报告大约 10 个假阳性，因此您可以看到噪声的顶部，并自行决定它是否真的是噪声。",
    "of >= X as the hit reporting threshold.": "将 >= X 作为命中报告阈值。",
    "Curated CM databases may define specific bit score thresholds for each CM, superseding any thresholding based on statistical significance alone.": "策划的CM数据库可能为每个CM定义特定的比特分数阈值，取代仅基于统计显著性的任何阈值设定。",
    "GA thresholds are generally considered to be the reliable curated thresholds defining family membership": "GA阈值通常被认为是定义家族成员身份的可靠且经过精心策划的阈值",
    "NC thresholds are generally considered to be the score of the highest-scoring known false positive.": "NC阈值通常被认为是最高得分的已知假阳性的分数。",
    "TC thresholds are generally considered to be the score of the lowest-scoring known true positive that is above all known false positives.": "TC阈值通常被认为是所有已知假阳性之上得分最低的已知真阳性的分数。",
    "These options are, in order from least strict (slowest but most sensitive) to most strict (fastest but least sensitive)": "这些选项按照严格程度从低到高（即从最不严格但最敏感到最严格但最不敏感）的顺序排列",
    "This correction is not used during the HMM filter stages.": "这个校正不在使用HMM过滤器阶段时使用。",
    "By default, the Durbin/Holmes optimal accuracy algorithm is used, which finds the alignment that maximizes the expected accuracy of all aligned residues.": "默认情况下，使用Durbin/Holmes最优准确性算法，该算法找到最大化所有对齐残基预期准确性的对齐方式。",
    "in the sequence database": "在序列数据库中",
    "... those satisfying inclusion thresholds": "... 那些满足包含阈值的",
    "This can greatly reduce the output volume": "这可以大大减少输出量",
    "They include filter survival statistics for truncated hit detection and number of envelopes discarded due to matrix size overflows.": "它们包括截断命中检测的生存统计信息以及由于矩阵大小溢出而丢弃的信封数量。",
    "remuRNA accepts only a single tag in each call.": "remuRNA 每次调用只接受一个标签。",
    "Compute average distance over windows of size winsize.": "计算窗口大小为winsize的窗口上的平均距离。",
    "default, dangling energies will be added for the bases adjacent to a helix on both sides in any case": "默认情况下，任何情况下，对于两侧都与螺旋相邻的碱基，将添加悬挂能量。",
    "The list of SNPs to be tested have to be provided in separate lines": "要测试的SNPs列表必须在单独的行中提供",
    "Length of flanking sequence on either side of SNP considered for folding. please make sure that the input sequence is at least twice the size of chosen flanking. This restriction is necessary to keep the size of parameter tables for the p-value calculations manageable. In case the input sequence is less than twice the size of chosen flanking, the RNAsnp takes the nts up to the start and end position of the given sequence from the SNP position and perform the analysis.": "考虑用于折叠的SNP两侧的侧翼序列长度。请确保输入序列至少是所选侧翼大小的两倍。这个限制是必要的，以保持用于p值计算的参数表的大小可管理。如果输入序列小于所选侧翼大小的两倍，RNAsnp将从SNP位置开始和结束位置取到给定序列的核苷酸，并执行分析。",
    "Cut-off for the base pair probabilities.": "碱基对概率的截止值。",
    "The structural difference between wild-type and mutant is computed for all sequence intervals with the selected minimum length": "野生型与突变体之间的结构差异是针对所有具有所选最小长度的序列间隔计算的",
    "Average the pair probabilities over windows of given size": "将给定大小窗口内的配对概率求平均",
    "Set the maximum allowed separation of a base pair to span. i.e. no pairs (i,j) with j-i > L will be allowed.": "设置允许的碱基对最大分离距离。即不允许(j-i > L)的配对。",
    "size of the flanking region on either side of SNP that includes the local window returned by Mode 2. This subsequence is then passed to Mode 1 for re-computation": "SNP两侧包含局部窗口的邻近区域的大小，该局部窗口由模式2返回。然后，这个子序列被传递给模式1进行重新计算。",
    "compute ensemble Euclidean distance between the distribution of structures between two sequences": "计算两个序列之间结构分布的集合欧几里得距离",
    "Multiply the bolztmann factor with a prefactor alpha": "将玻尔兹曼因子乘以一个前因子alpha",
    "Sequence input in fasta format, optionally with locarna-specific extensions": "序列输入为fasta格式，可选地包含Locarna特定的扩展",
    "The sam file containing alignment of the read data.": "包含读取数据比对的sam文件。",
    "If reads were collapse before alignment and you want to incorporate the read copy number, please select Yes, otherwise select No": "如果在读段比对前进行了合并，并且您希望纳入读段的拷贝数，请选择“是”，否则请选择“否”。",
    "The maximum number of reads to extend beyond the positive signal in each direction (default 0) the cluster is defined": "每个方向上扩展超过正信号的最大读取数（默认为0），定义了聚类",
    "Please specify characters": "请指定字符",
    "Character representing the modified ribonucleotide (default 'T')": "表示修饰核苷酸的字符（默认为'T'）",
    "Character representing what the modified ribonucleotide is read as by rTranscriptase (default 'C')": "字符表示修饰核苷酸被r转录酶读取为（默认为'C'）",
    "You can use the default settings or set custom values for any of paralyzer's parameters.": "您可以使用默认设置或为paralyzer的任何参数设置自定义值。",
    "Size of bandwidth for KDE calculation (default 3)": "核密度估计（KDE）计算带宽大小（默认为3）",
    "Minimum number of reads required to call a group (default 5)": "调用一个组所需的最小读取数（默认为5）",
    "Minimum number of reads required to call a cluster (default 2)": "调用一个聚类所需的最小读取数（默认为2）",
    "Minimum read depth at a location to make a KDE estimate (default 3)": "在某个位置进行KDE估计的最小读取深度（默认为3）",
    "Minimum read depth for a location to be included within a cluster (default 1)": "一个位置被包含在聚类内的最小读取深度（默认为1）",
    "Minimum length required for a cluster to be reported (default 11)": "报告一个聚类所需的最小长度（默认为11）",
    "Minimum number of separate locations to have a reported conversion for a cluster to be reported (default 2)": "一个簇要被报告，至少需要有的独立位置数量（默认为2）",
    "Minimum number of conversion events within a region to report a cluster (default 2)": "在区域内报告一个簇的最小转换事件数（默认为2）",
    "Minimum length of mapped read to be included in the analysis (default 20)": "分析中包含的映射读段的最小长度（默认为20）",
    "Maximum number of non-conversion mismatches of a mapped read to be included in the analysis (default 1)": "映射读取中包含在分析中的非转换错配的最大数目（默认为1）",
    "Fusions with at least this many supporting reads will be reported.": "至少有这么多支持读取的融合将被报告。",
    "Fusions with at least this many supporting pairs will be reported.": "至少有这么多支持配对的融合将被报告。",
    "The sum of supporting reads and pairs is at least this number for a fusion to be reported.": "支持的读段和配对的总数至少为这个数值，才会报告融合事件。",
    "Reads support fusions if they map across fusion with at most this many mismatches.": "如果读取跨越融合区域并且最多有这么多不匹配，则支持融合。",
    "A single sequence in FASTA format": "一个FASTA格式的序列",
    "0 means all is transcribed (--transcribed)": "0 表示全部转录了 (--transcribed)",
    "0= all are considered. (--windowsize)": "0=全部都考虑。（--windowsize）",
    "This file contains the description of the motif for which to search": "这个文件包含了要搜索的基序的描述",
    "This file specifies the sequence in which the motif will be searched": "这个文件指定了搜索motif的顺序",
    "-c : Search both strands of the supplied sequence": "-c：搜索提供的序列的两个链",
    "-s : This is a workaround to avoid a problem in the DNABANK, overlapping matches will be ignored": "-s：这是一个避免DNABANK中问题的临时解决方案，重叠的匹配将被忽略",
    "Display full alignments to pattern": "显示与模式的完整对齐",
    "Reads in fastq or FASTA format": "读取fastq或FASTA格式的序列",
    "Must be a 3 letters/digits code": "必须是一个3个字母/数字的代码",
    "Remove all entries that have a sequence that contains letters other than a,c,g,t,u,n,A,C,G,T,U,N. (-j)": "删除所有序列中含有除 a,c,g,t,u,n,A,C,G,T,U,N 之外的字母的条目。(-j)",
    "Adapter Sequence can only contain a,c,g,t,u,n,A,C,G,T,U,N": "适配器序列只能包含a,c,g,t,u,n,A,C,G,T,U,N",
    "Set to 0 to keep all reads. (-l)": "设置为0以保留所有读取。(-l)",
    "Reads in fasta format. (-r)": "以fasta格式读取序列。（-r）",
    "miRNA precursor sequences from miRBase. (-p)": "来自miRBase的miRNA前体序列。（-p）",
    "Mature miRNA sequences from miRBase. (-m)": "来自miRBase的成熟miRNA序列。(-m)",
    "If not searching in a specific species all species in your files will be analyzed. (-t)": "如果不在特定物种中搜索，您文件中的所有物种都将被分析。（-t）",
    "From miRBase in fasta format (optional) (-s)": "从 miRBase 的 fasta 格式文件中（可选）(-s)",
    "Also considers precursor-mature mappings that have different ids, eg let7c would be allowed to map to pre-let7a. (-k)": "同时考虑具有不同ID的前体-成熟体映射，例如，允许let7c映射到pre-let7a。（-k）",
    "Number of nucleotides upstream of the mature sequence to consider. (-e)": "成熟序列上游考虑的核苷酸数量。(-e)",
    "Number of nucleotides downstream of the mature sequence to consider. (-f)": "成熟序列下游要考虑的核苷酸数量。（-f）",
    "Number of allowed mismatches when mapping reads to precursors. (-g)": "映射读取到前体时允许的不匹配数。（-g）",
    "Genome contigs in fasta format. The identifiers should be unique.": "基因组的连续片段，以fasta格式。这些标识符应该是唯一的。",
    "Reads mapped against genome. Mappings should be in ARF format.": "比对到基因组的序列。比对结果应该是ARF格式。",
    "miRBase miRNA sequences in fasta format. These should be the known mature sequences for the species being analyzed.": "miRBase miRNA序列以fasta格式。这些应该是被分析物种的已知成熟序列。",
    "miRBase miRNA precursor sequences in fasta format. These should be the known precursor sequences for the species being analyzed.": "miRBase miRNA前体序列，以fasta格式。这些应该是被分析物种的已知前体序列。",
    "Maximum number of precursors to analyze when automatic excision gearing is used. If set to -1 all precursors will be analyzed. (-g).": "当使用自动切除齿轮时，要分析的前体最大数量。如果设置为-1，则分析所有前体。（-g）。",
    "Minimum score cut-off for predicted novel miRNAs to be displayed in the overview table. (-b)": "在概览表中显示预测新miRNA的最小分数截止值。（-b）",
    "Output fasta files of precursors, mature and star strand for both novel and known miRNAs": "输出前体、成熟和星形链的fasta文件，用于已知和新发现的miRNAs",
    "Select from a list of pre-indexed references. If you don't see anything consult the wrapper's documentation on how to create or download a reference": "从预索引参考列表中选择。如果您没有看到任何内容，请查阅包装器的文档，了解如何创建或下载参考",
    "In addition to the transcript-coordinate-based BAM file output, also output a BAM file with the read alignments in genomic coordinates": "除了输出基于转录本坐标的BAM文件外，还输出一个包含基因组坐标中读段比对的BAM文件。",
    "The files should contain the sequences of transcripts.": "文件应包含转录本的序列。",
    "The file should contain the sequence of an entire genome.": "该文件应包含整个基因组的序列。",
    "extract transcript reference sequences using the gene annotations specified in this GTF": "使用此GTF中指定的基因注释提取转录本参考序列",
    "Bowite uses the automatic N to G conversion to to align against all positions in the reference.": "Bowtie 使用自动的 N 到 G 转换来对齐参考序列中的所有位置。",
    "Select model from the list": "从列表中选择模型",
    "With Principal Component (PC), the maximum correlation coefficient from top 8 PCs for each avgLoading will be selected as an output. If you choose Average Loading, the Average Loading with the maximum correlation coefficient with each Principal Component will be in the output.": "使用主成分（PC），将为每个平均载荷选择前8个主成分中的最大相关系数作为输出。如果您选择平均载荷，那么将输出与每个主成分具有最大相关系数的平均载荷。",
    "max will output the matrix containing only the maximum coefficient. To get the coefficient of all 8 PCs, set this argument to all.": "max 将输出只包含最大系数的矩阵。要获取所有 8 个主成分的系数，将此参数设置为 all。",
    "Fasta file to look for rRNA's in": "Fasta 文件，用于查找 rRNA",
    "Proportional length threshold to reject prediction": "拒绝预测的比例长度阈值",
    "Proportional length threshold to tag a find as pseudo": "标记为伪影的比例长度阈值",
    "Similarity e-value cutoff": "相似性E值截断",
    "Include the fasta sequences in the GFF output file": "在GFF输出文件中包含fasta序列",
    "Produce an output file of the rRNA sequences found in FASTA format.": "生成一个包含FASTA格式的rRNA序列的输出文件。",
    "CIRCexplorer2 contains 5 modules. Each module functions as an independent component owning its distinctive duty. Meanwhile, they inteact with each other, and different circular RNA analysis pipelines are derived from different combinations of several modules.": "CIRCexplorer2包含5个模块。每个模块作为一个独立的组件，拥有其独特的职责。同时，它们相互交互，不同的环状RNA分析流程来源于几个模块的不同组合。",
    "Only single-read RNA-seq is supported. It is recommended to convert paired-end RNA-seq to single-read RNA-seq before alignment": "仅支持单端RNA-seq。建议在比对之前将双端RNA-seq转换为单端RNA-seq。",
    " It will not consider strand information of read alignment": "它不会考虑读取对齐的链信息",
    "Expression levels will be scaled to hits per billion-mapped-bases (HPB)": "表达水平将被缩放到每十亿映射碱基的命中数（HPB）",
    "CIRexplorer2 aligns reads onto genome and transcriptome using TopHat2 to reduce false positive reads aligned in the TopHat-Fusion alignment step": "CIRexplorer2 通过使用 TopHat2 将读取与基因组和转录组对齐，以减少在 TopHat-Fusion 对齐步骤中对齐的假阳性读取。",
    "It is useful for poly(A)+ RNA-seq": "这对于poly(A)+ RNA-seq非常有用",
    "CIRCexplorer2 parse could accept results derived from TopHat-Fusion, STAR, MapSplice, BWA and segemehl.": "CIRCexplorer2 解析可以接受来自 TopHat-Fusion、STAR、MapSplice、BWA 和 segemehl 的结果。",
    "The required input file is accepted_hits.bam": "所需的输入文件是 accepted_hits.bam",
    "Parse paired-end alignment file. If this is set, then -f is set automatically.": "解析配对端对齐文件。如果设置了这个选项，那么-f选项会自动设置。",
    "The required input file is Chimeric.out.junction": "所需的输入文件是 Chimeric.out.junction。",
    "The required input file is fusions_raw.txt": "所需的输入文件是 fusions_raw.txt",
    "The required input file is the output sam file": "所需的输入文件是输出的sam文件",
    "The required input file is splicesites.bed": "所需的输入文件是 splicesites.bed",
    "When enabled, realignment step of fusion junction reads will be skipped. It is useful for species with poor gene annotations, but the accuracy of circular RNA prediction would decrease.": "当启用时，融合连接处的比对步骤将被跳过。这对于基因注释不完整的物种很有用，但环状RNA预测的准确性会降低。",
    "By default, it extracts fusion junction reads exactly matching the boundaries of exons of the same isoform by default. If you set the --low-confidence, it will also extract fusion junction reads matching the boundaries of exons of the different isofoms of the same gene": "默认情况下，它会提取与同一异构体的外显子边界完全匹配的融合连接读取。如果您设置了--low-confidence参数，它还会提取与同一基因的不同异构体的外显子边界匹配的融合连接读取。",
    "Ignore rRNA during assembling (only for human hg19)": "在组装过程中忽略rRNA（仅适用于人类hg19）",
    "Generated by the module CIRCexplorer2 assemble": "由模块 CIRCexplorer2 组装生成",
    "When set, it will characterize the alternative back-splicing of circular RNAs": "当设置时，它将表征环状RNA的替代性反向剪接",
    "If enabled, it will characterize the alternative splicing of circular RNAs, including cassette exons, retained introns, A5SS and A3SS": "如果启用，它将表征环状RNA的替代剪接，包括盒式外显子、保留内含子、A5SS和A3SS。",
    "In this mode, you should also offer the path of TopHat mapping directory for p(A)-/p(A)+ RNA-seq via -m/-n option": "在这种模式下，您还应该通过 -m/-n 选项提供 TopHat 映射目录的路径，用于 p(A)-/p(A)+ RNA-seq。",
    "If enabled, realignment step of fusion junction reads will be skipped. It is useful for species with poor gene annotations, but the accuracy of circular RNA prediction would decrease.": "如果启用，融合连接读取的重新比对步骤将被跳过。这对于基因注释较差的物种很有用，但环状RNA预测的准确性会降低。",
    "If enabled, RPKM of cassette exons would be calculated": "如果启用，将计算盒式外显子的RPKM值",
    "It allows to build a more complete object and generate richer reports about the expression data. Sample annotation can be supplied in a data.frame that specifies a class for each sample. Classes can represent different conditions, phenotypes, cell lines, time points, etc.": "它允许构建一个更完整的对象，并生成关于表达数据的更丰富的报告。样本注释可以以data.frame的形式提供，该data.frame指定了每个样本的类别。类别可以代表不同的条件、表型、细胞系、时间点等。",
    "CEMiTool can determine which biological functions are associated with the modules by performing an over representation analysis (ORA). To do this you must provide a pathway list in the form of GMT file. CEMiTool will then analyze how these pathways are represented in the modules.": "CEMiTool 可以通过执行过度表达分析（ORA）来确定与模块相关联的生物学功能。为此，您必须提供一个以 GMT 文件形式的通路列表。然后，CEMiTool 将分析这些通路在模块中的表示情况。",
    "Interaction data, such as protein-protein interactions can be added in order to generate annotated module graphs. Interaction files contains two columns for interacting pairs of genes": "交互数据，例如蛋白质-蛋白质相互作用，可以添加以生成注释模块图。交互文件包含两列，用于表示相互作用的基因对。",
    "If enabled, it will filter expression data": "如果启用，它将过滤表达数据",
    "P-value threshold for filtering": "过滤的P值阈值",
    "If enabled, it apply Variance Stabilizing Transform before filtering genes": "如果启用，它在过滤基因之前应用方差稳定变换",
    "Number of genes left after filtering": "过滤后剩余的基因数量",
    "A value for accepted R-squared interval between subsequent beta values": "一个用于接受的随后beta值之间R平方区间的值",
    "Correlation coefficient to be computed": "要计算的相关系数",
    "Correlation function to be used": "要使用的关联函数",
    "Indicates if network type should be computed as 'signed' or 'unsigned'": "指示是否应将网络类型计算为“有符号”或“无符号”",
    "Indicates if TOM type should be computed as 'signed' or 'unsigned'": "指示TOM类型是否应计算为“有符号”或“无符号”",
    "If enabled, merge similar modules": "如果启用，合并相似的模块",
    "Indicate how to rank genes": "指示如何对基因进行排序",
    "Module merging correlation threshold for eigengene similarity": "模块合并相关性阈值用于特征基因相似性",
    "If enabled, it applies z-score transformation": "如果启用，它应用z分数转换",
    "This is only required if an annotation file is provided": "如果提供了注释文件，则只需要这个",
    "Only letters, numbers and underscores will be retained in this field. If more than one experiment is provided, then the tool generates a mixed model by combining datasets.": "在这个字段中，只保留字母、数字和下划线。如果提供了多个实验，则该工具通过组合数据集生成一个混合模型。",
    "Input sequence to be processed by miRNAture": "输入序列由miRNAture处理",
    "Control of normalized bitscore (nbitscore) threshold to filter Rfam candidates": "控制标准化比对分数（nbitscore）阈值以筛选 Rfam 候选序列",
    "Repetition cutoff": "重复截止",
    "Select a pre-calculated dataset to run miRNAture (i.e. https://doi.org/10.5281/zenodo.7180160)": "选择一个预先计算的数据集来运行miRNAture（即https://doi.org/10.5281/zenodo.7180160）",
    "Please submit the pre-calculated data to run miRNAture (i.e https://doi.org/10.5281/zenodo.7180160)": "请提交预先计算的数据以运行miRNAture（即 https://doi.org/10.5281/zenodo.7180160）",
    "Query sequences to search in target genome": "查询序列以在目标基因组中搜索",
    "The first step of a IsoformSwitchAnalyzeR workflow is to import and integrate the isoform quantification with its basic annotation. Once you have all the relevant data imported into R (IsoformSwitchAnalyzeR will also help you with that), the workflow for identification and analysis of isoform switches with functional consequences can be divided into two parts.": "IsoformSwitchAnalyzeR工作流程的第一步是导入并整合异构体定量及其基本注释。一旦你将所有相关数据导入到R中（IsoformSwitchAnalyzeR也会帮助你完成这项工作），识别和分析具有功能后果的异构体开关的工作流程可以分成两部分。",
    "IsoformSwitchAnalyzeR has different functions for importing data from different sources.": "IsoformSwitchAnalyzeR 具有从不同来源导入数据的不同功能。",
    "Must be the number of base pairs sequenced. e.g. if the data quantified is 75 bp paired ends the the user should supply readLength=75": "必须指定测序得到的碱基对数。例如，如果数据是75个碱基对的配对末端，用户应该提供readLength=75",
    "This option will automatically try and correct some of the annoation problems created when doing transcript assembly (unassigned transcripts and merged genes)": "这个选项将自动尝试纠正在进行转录本组装时产生的一些注释问题（未分配的转录本和合并的基因）。",
    "The merged GTF is used to recalculate expression estimates using the merged, novel transcripts.": "合并后的GTF用于使用合并后的新转录本重新计算表达估计。",
    "It is used to integrate the coding sequence (CDS) regions from in the GTF file as the ORF regions used by IsoformSwitchAnalyzeR.": "它用于将GTF文件中的编码序列（CDS）区域整合为IsoformSwitchAnalyzeR使用的开放阅读框（ORF）区域。",
    "Please note this different from a fasta file with the sequences of the entire genome.": "请注意，这与包含整个基因组序列的fasta文件不同。",
    "These regions are typically used to annotate regions that cannot be associated to a specific region.": "这些区域通常用于注释那些不能与特定区域关联的区域。",
    "Samples from different factors belong to the same individual (e.g. samples from same patient from health and cancerous tissues or different parts from the same plant)": "来自不同因素的样本属于同一个个体（例如，来自同一个病人的健康和癌组织样本，或者来自同一种植物的不同部分的样本）",
    "If IsoformSwitchAnalyzeR is used for fixing Stringtie annotation problem, it can generate count files for analyzing differential expression with DESeq2 (when selecting collection) or CEMiTool (when secting the expression matrix format).": "如果IsoformSwitchAnalyzeR用于解决Stringtie注释问题，它可以生成用于分析差异表达的计数文件，与DESeq2（当选择集合时）或CEMiTool（当选择表达矩阵格式时）一起使用。",
    "It is generated when running the analysis part 1.": "它是在运行分析第一部分时生成的。",
    "Filter for genes with functional consequences. The output will then be the number of significant genes and isoforms originating from genes with predicted consequences": "筛选具有功能性后果的基因。输出将是具有预测后果的基因产生的显著基因和同工型的数量。",
    "SwitchAnalyzeR will remove genes/isoforms with the aim of allowing faster processing time as well as more trustworthy results.": "SwitchAnalyzeR 旨在移除基因/同工型，以便加快处理时间并得到更可靠的结果。",
    "The expression cutoff (most likely in TPM/RPKM/FPKM) which the average expression in BOTH condisions must be higher than.": "表达量截止值（很可能是TPM/RPKM/FPKM）在两种条件下平均表达量必须高于该值。",
    "The expression cutoff (most likely in RPKM/FPKM) which isoforms must be expressed more than, in at least one conditions of a comparison. Default is 0 (which removes completely unused isoforms).": "表达阈值（很可能以RPKM/FPKM为单位），在比较的至少一个条件下，异构体的表达量必须超过这个阈值。默认值为0（这将完全移除未被使用的异构体）。",
    "The cutoff on isoform usage (measured as Isoform Fraction) which isoforms must be used more than in at least one conditions of a comparison": "在比较中，至少在一个条件下使用（以异构体分数度量）的异构体必须超过的截止值",
    "Only keep genes containing more than one isoform (in any comparison, after the other filters have been applied)": "只保留含有一个以上异构体的基因（在任何比较中，在其他过滤器应用之后）",
    "DEXSeq is used to test isoforms (isoform resolution) for differential isoform usage.": "DEXSeq 用于测试异构体（异构体分辨率）以进行差异异构体使用。",
    "A logic indicating whether IsoformSwitchAnalyzeR to use limma to correct for any confounding effects (e.g. batch effects) as indicated in the design matrix (as additional columns (apart from the two default columns))": "一个逻辑值，指示IsoformSwitchAnalyzeR是否使用limma来校正设计矩阵中所示的任何混杂效应（例如批次效应）（作为额外的列（除了两个默认列））",
    "It indicates whether to overwrite the IF and dIF stored in the switchAnalyzeRlist with the corrected IF and dIF values - if no confounding effects are present in the design matrix this will not change anything": "它指示是否用校正后的IF和dIF值覆盖存储在switchAnalyzeRlist中的IF和dIF - 如果设计矩阵中没有混杂效应，这将不会改变任何内容",
    "Reduced to the genes which contains at least one isoform significantly differential used (as indicated by the alpha and dIFcutoff parameters": "减少到至少包含一个显著差异表达的同工型的基因（由alpha和dIFcutoff参数指示）",
    "This argument is a more strict version of reduceToSwitchingGenes as it not only requires that at least one isoform is significantly differential used (as indicated by the alpha and dIFcutoff parameters) but also that there is an isoform with the opposite effect size (e.g. used less if the first isoform is used more). The minimum effect size of the opposing isoform usage is also controlled by dIFcutoff. The existence of such an opposing isoform means a switch pair can be formed": "这个参数是reduceToSwitchingGenes的一个更严格的版本，因为它不仅要求至少一个异构体的使用有显著差异（由alpha和dIFcutoff参数指示），而且还要求存在一个具有相反效应大小的异构体（例如，如果第一个异构体的使用增加，则使用减少）。相反异构体使用的最小效应大小也由dIFcutoff控制。这样的相反异构体的存在意味着可以形成一对开关对。",
    "For the subset of isoforms not already annotated with ORFs this function predicts the most likely Open Reading Frame (ORF) and the NMD sensitivity. This function is made to help annotate isoforms if you have performed (guided) de-novo isoform reconstruction (isoform deconvolution).": "对于尚未注释有开放阅读框（ORF）的异构体子集，此功能预测最有可能的开放阅读框（ORF）和NMD（无义介导的mRNA降解）敏感性。这个功能是为了帮助注释异构体，如果您执行了（引导的）去新异构体重建（异构体解码）。",
    "The minimum size (in nucleotides) an ORF must be to be considered (and reported). Default is 100 nucleotides, which around 97.5% of Gencode coding isoforms in both human and mouse have.": "开放阅读框（ORF）必须达到的最小大小（以核苷酸为单位）才能被考虑（并报告）。默认值是100个核苷酸，这大约是人和小鼠基因组中97.5%的Gencode编码异构体的大小。",
    "More information in the help section": "帮助部分的更多信息",
    "The minimum distance (number of nucleotides) from the STOP codon to the final exon-exon junction. If the distance from the STOP to the final exon-exon junction is larger than this the isoform to be marked as NMD-sensitive.": "从终止密码子到最终外显子-外显子连接点的最小距离（核苷酸数）。如果从终止密码子到最终外显子-外显子连接点的距离大于这个值，那么该异构体将被标记为NMD敏感性。",
    "switchAnalyzeR will extracts the nucleotide (NT) sequence of transcripts by extracting and concatenating the sequences of a reference genome corresponding to the genomic coordinates of the isoforms.": "switchAnalyzeR 将通过提取和连接参考基因组中与异构体的基因组坐标相对应的序列，提取转录本的核苷酸（NT）序列。",
    "This option exist to allows for easier usage of the Pfam and SignalP web servers which both currently have restrictions on allowed sequence lengths. If enabled AA sequences are filtered to be > 5 AA. This will only affect the sequences written to the FASTA file not the sequences added to the switchAnalyzeRlist": "这个选项的存在是为了更方便地使用Pfam和SignalP网络服务器，这两个服务器目前对允许的序列长度有限制。如果启用，氨基酸序列将被过滤为大于5个氨基酸。这只会影响写入FASTA文件的序列，而不会影响添加到switchAnalyzeRlist中的序列。",
    "A logical indicating whether to removesequences based on their length. This option exist to allows for easier usage of the Pfam and SignalP web servers which both currently have restrictions on allowed sequence lengths. If enabled AA sequences are filtered to be smaller 1000 AA. This will only affect the sequences written to the fasta file (if writeToFile=TRUE) not the sequences added to the switchAnalyzeRlist.": "一个逻辑值，指示是否根据序列长度来移除序列。这个选项的存在是为了更方便地使用Pfam和SignalP网络服务器，这两个服务器目前都对允许的序列长度有限制。如果启用，氨基酸序列将被过滤为小于1000个氨基酸。这只会影响写入到fasta文件中的序列（如果writeToFile=TRUE），而不会影响添加到switchAnalyzeRlist中的序列。",
    "ORFs containing stop codons, defined as * when the ORF nucleotide sequences is translated to the amino acid sequence, should be A) removed from the ORF annotation in the switchAnalyzeRlist and B) removed from the sequences added to the switchAnalyzeRlist and/or written to FASTA files. This is only necessary if you are analyzing quantified known annotated data where you supplied a GTF file to the import function": "包含终止密码子的开放阅读框（ORF），当ORF核苷酸序列翻译成氨基酸序列时，被定义为“*”。应该A)从switchAnalyzeRlist中的ORF注释中移除，B)从添加到switchAnalyzeRlist的序列中移除和/或写入FASTA文件。这只在你分析已知的定量注释数据时才有必要，即你向导入功能提供了一个GTF文件。",
    "It is generated when running the analysis part 2.": "这是在运行分析的第二部分时生成的。",
    "This selector allows so specify if you want to analyze a specific gene or the (top) switching genes/isoforms": "这个选择器允许您指定是否要分析特定的基因或（最顶部的）切换基因/异构体。",
    "This parameters allows to specify the number of top genes/isoforms to plot": "这个参数允许指定要绘制的顶部基因/异构体的数量",
    "The output will then be the number of significant genes and isoforms originating from genes with predicted consequences": "输出将是具有预测后果的基因来源的显著基因和异构体的数量",
    "A logic indicating whether to the top n features are sorted by decreasing significance (increasing q-values) (if enabled) or decreasing switch size (absolute dIF, which are still significant as defined by alpha) (if disabled). The dIF values for genes are considered as the total change within the gene calculated as sum(abs(dIF)) for each gene": "一个逻辑指示，是否将前n个特征按显著性递减（如果启用，则为递增的q值）或开关大小递减（绝对dIF，这些dIF仍然按照alpha定义为显著）进行排序（如果禁用）。基因的dIF值被视为基因内部的总变化，计算为每个基因的sum(abs(dIF))。",
    "This parameter indicates whether it is the number of genes (if enabled) or isoform switches (if disabled) which primary result in gain/loss that are counted": "这个参数指示计数的是基因数量（如果启用）还是异构体转换（如果禁用），这些是导致增益/丢失的主要结果",
    "The consequences/splicing events should be summarized calculated as numbers (if disabled) or as a fraction of the total number of switches/genes": "后果/剪接事件应该总结计算为数字（如果禁用）或作为开关/基因总数的分数",
    "Plot the number/fraction of genes with (if enabled) or isoforms (if disabled) involved with isoform switches with functional consequences (both filtered via alpha and dIFcutoff)": "绘制具有（如果启用）或涉及（如果禁用）功能后果的异构体转换的基因数/分数（两者都通过alpha和dIFcutoff过滤）的图表",
    "Simplify the switches involved in changes in subcellular localizations (due the the hundreds of possible combinations)": "简化涉及亚细胞定位变化的开关（由于可能的组合有数百种）",
    "Remove consequenses analyzed but where no differences was found (those showing zero in the plot)": "移除那些在分析中被考虑但未发现差异的结果（即在图中显示为零的那些）。",
    "Reverse the analysis meaning if 'Domain gains' are analyze will case the analysis to be performed on 'Domain loss'. The main effect is for the visual appearance of plot which will be mirrored (around the 0.5 fraction)": "如果分析“域增益”，则反转分析的含义将导致执行“域丢失”的分析。主要效果是用于图形的视觉外观，该外观将被镜像（围绕0.5分数）",
    "Either the gene_id or the gene name of the gene to plot": "要绘制的基因的基因ID或基因名称",
    "The cutoff used for the minimum contribution to gene expression (in at least one condition) for an isoforms must have to be plotted (measured as Isoform Fraction (IF) values)": "用于基因表达（至少在一个条件下）的最小贡献的截断值（以等位基因分数（IF）值衡量）必须被绘制出来。",
    "All the isoforms should be resealed to the square root of their original sizes. This feature is implemented because introns usually are much larger than exons making it difficult to see structural changes. This is very useful for structural visualization but the scaling might distort actual intron and exon sizes": "所有异构体的大小应该重新调整为其原始大小的平方根。这个功能之所以实现，是因为内含子通常比外显子大得多，这使得观察结构变化变得困难。这对于结构可视化非常有用，但是缩放可能会扭曲实际的内含子和外显子的大小。",
    "Isoforms on minus strand should be inverted so they are visualized as going from left to right instead of right to left": "负链上的异构体应该反转，以便它们被可视化为从左到右而不是从右到左。",
    "Error bars should be added to the expression plots to show uncertainty in estimates": "在表达量图中应添加误差条以显示估计中的不确定性",
    "Integrate in the analysis de output from CPAT or CPC2.": "在分析中整合来自CPAT或CPC2的输出。",
    "Use default parameters and the nucleotide fasta file (_nt.fasta). If the webserver was used, download the tab-delimited result file (available at the bottom of the result page). If a stand-alone version was used, just supply the path to the result file": "使用默认参数和核苷酸fasta文件(_nt.fasta)。如果使用了网页服务器，请下载制表符分隔的结果文件（可在结果页面底部找到）。如果使用了独立版本，只需提供结果文件的路径。",
    "cutoff used by CPAT for distinguishing between coding and non-coding transcripts. The cutoff is dependent on species analyzed. IsoformSwitchAnalyzerR developers suggest that the optimal cutoff for overlapping coding and noncoding isoforms are 0.725 for human and 0.721 for mouse. However the suggested cutoffs from the CPAT develpers derived by comparing known genes to random non-coding regions of the genome is 0.364 for human and 0.44 for mouse": "CPAT用于区分编码和非编码转录本的截止值。该截止值取决于所分析的物种。IsoformSwitchAnalyzerR开发者建议，对于人类，重叠的编码和非编码异构体的最佳截止值为0.725，而对于小鼠为0.721。然而，CPAT开发者建议的截止值是通过将已知基因与基因组的随机非编码区域进行比较得出的，对于人类为0.364，对于小鼠为0.44。",
    "Use default parameters and if required select the most similar species. If the webserver (batch submission) was used, download the tab-delimited result file (via the “Download the result” button). If a stand-alone version was just just supply the path to the result file": "使用默认参数，如果需要，选择最相似的物种。如果使用了网络服务器（批量提交），请下载制表符分隔的结果文件（通过“下载结果”按钮）。如果只是使用了独立版本，只需提供结果文件的路径。",
    "Remove ORF information from the isoforms which the CPC2 analysis classifies as non-coding. This can be particular useful if the isoform (and ORF) was predicted de-novo but is not recommended if ORFs was imported from a GTF file": "从CPC2分析被分类为非编码的异构体中移除开放阅读框（ORF）信息。如果异构体（和ORF）是从头预测的，这可能特别有用，但如果ORF是从GTF文件导入的，则不推荐这样做。",
    "Numeric indicating the cutoff used by CPC2 for distinguishing between coding and non-coding transcripts. The cutoff appears to be species independent.": "数值表示CPC2用于区分编码和非编码转录本的截止值。该截止值似乎是物种独立的。",
    "Pfam is a database of protein families that includes their annotations and multiple sequence alignments generated using hidden Markov models.": "Pfam 是一个蛋白质家族数据库，包含了它们的注释和使用隐马尔可夫模型生成的多序列比对。",
    "Use default parameters and the amino acid fasta file (_AA.fasta). If the webserver is used you need to copy/paste the result part of the mail you receive into an empty plain text document (notepad, sublimetext, TextEdit or similar (not Word)) and save that to a plain text (txt) file. The path to that file should be supplied. If a stand-alone version was used, just supply the path to the result file": "使用默认参数和氨基酸fasta文件(_AA.fasta)。如果使用的是网络服务器，你需要将收到的邮件中的“结果”部分复制/粘贴到一个空的纯文本文档中（记事本、sublimetext、TextEdit或类似的（不是Word）），并将该文档保存为纯文本（txt）文件。应提供该文件的路径。如果使用的是独立版本，只需提供结果文件的路径。",
    "Integration of the result of SignalP (external sequence analysis of signal peptides)": "整合SignalP（信号肽的外部序列分析）的结果",
    "Use the amino acid fasta file (_AA.fasta). If using the webserver SignalP should be run with the parameter 'Short output (no figures)' under 'Output format' and one should select the appropriate 'Organism group'. When using a stand-alone version SignalP should be run with the '-f summary' option. If using the webserver the results can be downloaded using the 'Downloads' button in the top-right corner where the user should select 'Prediction summary' and supply the path to the resulting file to the 'pathToSignalPresultFile' argument. If a stand-alone version was just supply the path to the summary result file.": "使用氨基酸fasta文件（_AA.fasta）。如果使用网络服务器，SignalP应该用参数'Short output (no figures)'（简短输出（无图形））在'Output format'（输出格式）下运行，并选择适当的'Organism group'（生物群体）。当使用独立版本时，SignalP应该用'-f summary'选项运行。如果使用网络服务器，可以通过右上角的'Downloads'（下载）按钮下载结果，用户应选择'Prediction summary'（预测摘要）并提供结果文件的路径到'pathToSignalPresultFile'参数。如果只是使用独立版本，则只需提供摘要结果文件的路径。",
    "Integrate in the analysis de output from IUPred2A or NetSurfP-2": "在分析中整合来自IUPred2A或NetSurfP-2的输出结果",
    "Can be gziped. If multiple result files were created (multiple web-server runs) just supply all of them.": "可以被gzip压缩。如果创建了多个结果文件（多次网络服务器运行），只需提供它们所有。",
    "Integrate the ANCHOR2 prediction of Intrinsically Disordered Binding Regions (IDBRs)": "整合ANCHOR2对内在无序结合区域（IDBRs）的预测",
    "How long a stretch of binding site the region part of the Intrinsically Disordered Binding Regions (IDBR)": "Intrinsically Disordered Binding Regions (IDBR) 中的结合位点区域有多长？",
    "Minimum fraction of a predicted IDBR must also be within a IDR before the IDR is considered as a an IDR with a binding region": "预测的内在无序区域（IDBR）中至少有一部分必须位于固有无序区域（IDR）内，才能将该IDR视为具有结合区域的IDR。",
    "The length difference (in nucleotides) a comparison must be larger than for reporting differences": "比较必须大于报告差异的核苷酸长度差异",
    "The cutoff in length difference, measured as a fraction of the length of the downregulated isoform, a comparison must be larger than for reporting differences. For example does 0.05 mean the upregulated isoform must be 5% longer/shorter before it is reported.": "长度差异的截止值，以下调表达异构体长度的分数来衡量，比较值必须大于报告差异的值。例如，0.05表示上调表达的异构体必须比下调表达的异构体长/短5%才能被报告。",
    "If the measured JCsim is smaller than this cutoff the sequences are considered different and reported as such": "如果测量到的JCsim小于这个截止值，那么序列就被认为是不同的，并会作为不同被报告出来。",
    "Length difference (in AA) a comparison must be larger than for reporting differences when evaluating ’ORF_seq_similarity’, primarily implemented to avoid differences in very short AA sequences being classified as different": "氨基酸长度差异（以氨基酸为单位）比较时必须大于报告差异的值，以评估'ORF_seq_similarity'，主要是为了避免将非常短的氨基酸序列的差异分类为不同。",
    "Cutoff of length difference of the protein domain or IDR. The difference is measured as a fraction of the longest region, a comparison must be larger than before reporting it": "蛋白质域或无序区域长度差异的截止值。该差异以最长区域的分数来衡量，在报告之前，比较必须大于这个值。",
    "If you have multiple experimental conditions, you should use propably the complex design mode. In the help section you can find more information.": "如果您有多个实验条件，您应该使用复杂的设计模式。在帮助部分，您可以找到更多信息。",
    "You can find more details about the format of the design table in the help section.": "您可以在帮助部分找到有关设计表格式的更多详细信息。",
    "If this is set to false, bootstraps will not be read and transformation of the data will not be done. This should only be set to false if one desires to do a quick check of the raw data.": "如果此选项设置为false，则不会读取引导程序，也不会对数据进行转换。只有在希望快速检查原始数据时，才应将其设置为false。",
    "The number of bins that the data should be split for the sliding window shrinkage using the mean-variance curve.": "数据在滑动窗口收缩时，应将数据分割为多少个箱，以使用均值-方差曲线。",
    "The lower range of variances within each bin that should be included for the shrinkage procedure.": "用于收缩过程的每个箱中应包括的方差下限范围。",
    "The upper range of variances within each bin that should be included for the shrinkage procedure.": "用于收缩过程的每个箱中应包含的方差上限范围。",
    "Trinity Assembled fasta file": "Trinity 组装好的fasta文件",
    "Each row in the file contains an RNA clan and its corresponding family members": "文件中的每一行包含一个RNA家族及其相应的家族成员。",
    "If this option is not set, all hits with higher scoring overlap will be removed.": "如果这个选项没有设置，所有具有更高得分重叠的命中都将被移除。",
    "PsiCLASS has been tuned to run on alignments generated with the tools HISAT and STAR": "PsiCLASS 已经调整为可以运行在 HISAT 和 STAR 工具生成的比对上。",
    "It is possible to supply an set of trusted introns, for instance generated by RNASTAR or extracted from the GENCODE gene annotations": "可以提供一组可信内含子，例如由RNASTAR生成或从GENCODE基因注释中提取的内含子。",
    "High confidence collapsed splice junction file": "高置信度的合并剪接连接文件",
    "Only use the subexons with classifier score minor or equal than the given number": "仅使用分类器得分小于或等于给定数值的子外显子",
    "Minimum average number of supported read for retained introns": "保留内含子支持的最小平均读数",
    "Minimum average coverage depth of a transcript to be reported": "转录本被报告的最小平均覆盖深度",
    "Stranded data shows advantages over non-stranded RNA-Seq data such as higher assembly and differential expression accuracy": "链特异性数据比非链特异性RNA-Seq数据具有更高的组装和差异表达准确性的优势",
    "-1 for infinite": "-1 表示无限",
    "Default: use unique alignments": "默认：使用唯一对齐",
    "Write out all SAM alignment records into an output BAM file, annotating each line with its assignment to a feature or a special counter (as an optional field with tag ‘XF’).": "将所有SAM比对记录写入输出BAM文件，并在每行上注释其分配给特征或特殊计数器（作为带有标签‘XF’的可选字段）。",
    "Required for SAM input without @SQ headers, not necessary otherwise and ignored for BAM input.": "对于没有@SQ头文件的SAM输入是必需的，否则不必要，并且在BAM输入中被忽略。",
    "Applicable only when input dataset format is in the gff family": "仅适用于输入数据集格式为gff系列时",
    "Choose history if you don't see the correct GTF": "如果看不到正确的GTF，请选择历史记录。",
    "Select the GTF from a list of pre-installed files": "从预安装文件列表中选择 GTF",
    "Make sure that the GTF corresponds to the same genome as the FASTA": "确保GTF文件与FASTA文件对应的基因组是一致的。",
    "Specifying the criteria to associate peaks with annotation e.g. startSite start position of the feature strand": "指定将峰值与注释相关联的标准，例如特征链的起始位点起始位置",
    "e.g. to obtain peaks within 5kb upstream and up to 3kb downstream of TSS within the gene body, set bindingType = startSite and bindingRegion = c(-5000, 3000)": "例如，要在基因体内获得位于转录起始位点（TSS）上游5kb以内和下游最多3kb的峰值区域，可以设置bindingType为startSite，bindingRegion为c(-5000, 3000)。",
    "e.g. to obtain peaks with nearest bi-directional promoters within 5kb upstream and 3kb downstream of TSS, set bindingType = nearestBiDirectionalPromoters and bindingRegion = c(-5000, 3000)": "例如，要获得在TSS上游5kb和下游3kb内有最近双向启动子的峰值，设置bindingType = nearestBiDirectionalPromoters和bindingRegion = c(-5000, 3000)",
    "Position of Additional Column of the Score": "分数的附加列的位置",
    "Name of sequence (from the chromosome column in the dataset) to plot.  If left blank, the first sequence in the dataset will be plotted.": "要绘制的序列名称（来自数据集中的染色体列）。如果留空，将绘制数据集中的第一个序列。",
    "Level of Hilbert curve.  The resulting image will have 2<sup>level</sup> by 2<sup>level</sup> pixels.": "希尔伯特曲线的级别。生成的图像将具有 2^级别 × 2^级别的像素。",
    "Method used to determine a value for a point in the plot which covers multiple values in the input.": "用于确定覆盖输入中多个值的绘图中点的值的方法。",
    "At least four aligned genomes are needed for RAxML.": "至少需要四个对齐的基因组来进行RAxML分析。",
    "The required minimal settings are the input file and the substitution model. To specify extra options select the 'Full option list'": "所需的最低设置是输入文件和替代模型。要指定额外选项，请选择“完整选项列表”",
    "In combination with the '-b' option, this will invoke a multiple boostrap analysis": "结合使用“-b”选项，这将调用多重自助分析",
    "Bootstopping will only work in combination with '-x' or '-b'": "Bootstopping 只有在与 '-x' 或 '-b' 结合使用时才会有效",
    "Specifying a value turns on bootstrapping": "指定一个值将开启自助采样",
    "Specifying a value turns on rapid bootstrapping. CAUTION: unlike in version 7.0.4 RAxML will conduct rapid BS replicates under the model of rate heterogeneity you specified via '-m' and not by default under CAT": "指定一个值将打开快速引导支持树。警告：与7.0.4版本不同，RAxML将在您通过“-m”指定的速率异质性模型下进行快速引导支持树复制，而不是默认在CAT下进行。",
    "Cutoff threshold for the MR-based bootstopping criteria": "基于MR的引导停止标准的最大阈值",
    "This will break off ML searches if the relative Robinson-Foulds distance between the trees obtained from two consecutive lazy SPR cycles is smaller or equal to 1%. Usage recommended for very large datasets in terms of taxa. On trees with more than 500 taxa this will yield execution time improvements of approximately 50% while yielding only slightly worse trees": "如果从两个连续的懒惰SPR循环获得的树之间的相对Robinson-Foulds距离小于或等于1%，则会中断ML搜索。建议在物种数量非常大的数据集上使用。在物种数量超过500的树上，这将使执行时间提高大约50%，同时只会导致树的性能略有下降。",
    "Set model optimization precision in log likelihood units for final optimization of tree topology": "在树拓扑的最终优化中设置模型优化精度（以对数似然单位为单位）",
    "Should contain a specification of alignment positions you wish to exclude": "应该包含您希望排除的比对位置的说明",
    "ML tree searches under CAT model for very large trees without switching to GAMMA in the end (saves memory) and no thorough optimization under GAMMA": "在CAT模型下对非常大的树进行最大似然树搜索，最后不切换到GAMMA（节省内存），并且在GAMMA下不进行彻底的优化",
    "This tree does not need to be comprehensive, i.e. does not have to contain all taxa": "这棵树不必全面，即不必包含所有分类群",
    "By specifying a threshold value (fraction of insertion branches to be evaluated using slow insertions under ML)": "通过指定一个阈值（使用最大似然法评估插入分支的比例）",
    "The bootstraps will run a bit longer, because model parameters will be optimized at the end of each run under GAMMA or GAMMA+P-Invar respectively": "Bootstrap 运行的时间会更长，因为在每次运行结束时，模型参数将在 GAMMA 或 GAMMA+P-Invar 下分别进行优化。",
    "Only has effect with a partition file. A weighted average of the branch lengths is computed by using the respective partition lengths": "仅在有分区文件时有效。通过使用相应的分区长度来计算分支长度的加权平均值",
    "E.g. Mouse or Mouse,Rat. No spaces between taxon names are allowed": "例如，小鼠或小鼠，大鼠。不允许在分类单元名称之间留空格",
    "The program will not exit with an error message when '-O' is specified.": "程序在指定 '-O' 参数时，不会以错误信息退出。",
    "This file must contain 420 entries, the first 400 being the AA substitution rates (this must be a symmetric matrix) and the last 20 are the empirical base frequencies": "这个文件必须包含420个条目，前400个是氨基酸替换率（这必须是一个对称矩阵），最后的20个是经验基础频率。",
    "For the syntax of this file please consult the RaXML manual": "关于这个文件的语法，请参考RaXML手册",
    "This parameter file can be generated with RAxML using the '-f e' tree evaluation option": "这个参数文件可以使用 RAxML 工具，并通过 '-f e' 树评估选项生成。",
    "The file can contain '.' for alignment columns that do not form part of a stem and characters '()<>[]{}' to define stem regions and pseudoknots": "该文件可以包含'.'来表示不构成茎部的对齐列，并且可以使用字符'()<>[]{}'来定义茎部区域和伪结",
    "In Newick format": "在 Newick 格式中",
    "Try to save memory by using SEV-based implementation for gap columns on large gappy alignments. This will only work for DNA and/or PROTEIN data": "尝试通过在大型间隙对齐上使用基于SEV的实现来节省内存。这只适用于DNA和/或蛋白质数据",
    "Only works if you specify the CAT model of rate heterogeneity": "只有在您指定速率异质性的CAT模型时才有效",
    "Only effective when used in combination with '-f S'": "只有在与'-f S'组合使用时才有效",
    "The program will exit after computation of the starting tree": "程序将在起始树计算完成后退出",
    "Allow synonymous rates to vary from site to site": "允许同义位点的速率在不同位点之间变化",
    "MSA to be used in HyPhy analysis": "用于HyPhy分析的MSA",
    "If not given it will be downloaded": "如果没有提供，它将被下载",
    "Tabular file with metadata with the correct columns (see below)": "带有正确列的元数据表格文件（见下文）",
    "Genome name without whitespaces or special characters": "基因组名称不能包含空格或特殊字符",
    "Each FASTA header line should start with a transcript id": "每个FASTA头行应该以转录本ID开始",
    "As required by TreeBest, part of the GeneSeqToFamily workflow, only TranscriptId_species is acceptable format by Aequatus visualisation": "正如TreeBest所要求的，作为GeneSeqToFamily工作流程的一部分，只有TranscriptId_species格式才能被Aequatus可视化所接受。",
    "Region IDs are in the `seqid` column for GFF3 and in the `seq_region_name` field in JSON. This is typically used to filter out chromosomes with a non-standard genetic code, like mitochondria, to be analysed separately": "区域ID在GFF3的`seqid`列中，在JSON中的`seq_region_name`字段中。这通常用于筛选出具有非标准遗传代码的染色体，如线粒体，以便单独分析。",
    "phylip format": "PHYLIP格式",
    "Must be a positive integer, 'e' if you want PhyML to estimate it": "必须是一个正整数，如果你想让PhyML估计它，可以使用'e'",
    "Can be a fixed value in the [0,1] range or 'e' to get the maximum likelihood estimate, 0 to ignore this parameter": "可以是一个在[0,1]范围内的固定值，或者是'e'以获得最大似然估计，0表示忽略此参数",
    "Empirical: frequencies are estimated by counting the occurences in the alignment. ML/Model: frequencies are estimated using ML for nucleotides or defined by the proteic substitution model.": "经验：频率是通过计算比对中的发生次数来估计的。最大似然/模型：频率是使用核苷酸的最大似然估计或由蛋白质替代模型定义的。",
    "1 means no gamma model": "1表示不使用伽马模型",
    "'e' if you want PhyML to estimate it": "如果你希望PhyML估计它，请选择'e'",
    "Use aLRT or aBayes to save computing time": "使用aLRT或aBayes来节省计算时间",
    "Must be a positive integer": "必须是一个正整数",
    "0 to random seed": "0 到随机种子",
    "newick format": "新克格式",
    "GeneTree in nhx format, where nodes are in form of geneid_species": "GeneTree 以 nhx 格式，其中节点以 geneid_species 的形式呈现",
    "Comma-separated species list": "逗号分隔的物种列表",
    "GeneTree in nhx format": "NHX格式的基因树",
    "Select Gene node format from one of the option": "从选项中选择基因节点格式",
    "Comma-separated species list without whitespaces or special characters": "逗号分隔的物种列表，不包含空格或特殊字符",
    "Species Tree in nhx format": "物种树（Species Tree）在NHX格式中",
    "See table below": "见下表",
    "If the GeneTree contains any other homology type, then the output will be empty": "如果GeneTree包含任何其他同源类型，则输出将为空",
    "Get the latest version of the taxdump from the NCBI website": "从NCBI网站获取最新的taxdump版本",
    "NCBI taxonomy database dump in tar.gz format": "NCBI 分类数据库转储，tar.gz 格式",
    "List with one species/taxid per line": "每行一个物种/分类ID的列表",
    "The sqlite formatted Taxonomy used by ETE3 (which is derived from NCBI taxonomy)": "由 ETE3 使用的 sqlite 格式分类学（源自 NCBI 分类学）",
    "Taxonomic levels to include in the output": "输出中要包含的分类水平",
    "Fill unnamed ranks with super/sub ranks (see -l)": "用上级/下级等级填充未命名的等级（见 -l）",
    "Take the next available lower rank (default: higher)": "取下一个可用的较低等级（默认：较高）",
    "For the case when the input consists of taxon names": "对于输入包含分类单元名称的情况",
    "0=totally quite, 1=errors only, 2=warning+errors, 3=info+warnings+errors 4=debug (-v)": "0=完全静默，1=仅错误，2=警告+错误，3=信息+警告+错误 4=调试（-v）",
    "Root the tree using the provided outgroup. If several names are provided, the first common ancestor grouping all of them will be selected as outgroup": "使用提供的外类群对树进行根定位。如果提供了多个名称，将选择将它们全部分组的第一个共同祖先作为外类群。",
    "Remove all nodes in the tree except the ones provided (empty does not prune)": "删除树中除提供的节点之外的所有节点（空的不会剪枝）",
    "Branch lengths of the removed nodes are added to the kept branches, thus preserving original tree length": "被移除节点的分支长度被加到保留的分支上，从而保留了原始树的长度",
    "Sort branches according to node names": "根据节点名称对分支进行排序",
    "Sort branches by partition size": "按分区大小排序分支",
    "Converts polytomies into bifurcations": "将多岐分支转换为二岐分支",
    "Standardize tree topology by expanding polytomies and single child nodes": "将树状拓扑标准化，通过扩展多叉和单子节点",
    "List with one species per line": "每行列出一个物种",
    "Select yes if specie tree to be used in TreeBest": "如果需要在TreeBest中使用物种树，请选择是",
    "Tab-separated format: cluster_id [other optional columns] cluster-size cluster-members": "制表符分隔格式：聚类ID [其他可选列] 聚类大小 聚类成员",
    "Optional (-C)": "可选（-C）",
    "Gene tree in Newick format, e.g. generated from 'TreeBeST best' or 'Get gene tree by Ensembl ID' tool": "新克格式的基因树，例如由“TreeBeST best”或“通过Ensembl ID获取基因树”工具生成",
    "Protein alignments in fasta_aln format generated by T-Coffee": "由 T-Coffee 生成的 fasta_aln 格式的蛋白质比对",
    "Gene features in SQLite format generated by 'GSTF preparation' tool": "由“GSTF准备”工具生成的SQLite格式基因特征",
    "3-columns format: id-1 id-2 integer-weight": "三列格式：id-1 id-2 整数权重",
    "Bypass h-cluster": "绕过 h-cluster",
    "BLAST 12 column tabular format data": "BLAST 12列制表格式数据",
    "returns only reciprocal results": "只返回互惠结果",
    "Outputs stripped insertions relative to reference as CSV": "输出相对于参考的剥离插入，以CSV格式",
    " If a sequence is shorter than that, alignment will not be attempted and a warning will be emitted. When adjusting this parameter, note that alignment of short sequences can be unreliable.": "如果序列短于该长度，将不会尝试进行比对，并且会发出警告。在调整此参数时，请注意短序列的比对可能是不可靠的。",
    "If zero, all gaps regardless of length incur the same penalty.": "如果设置为零，则不论长度如何，所有间隙都会产生相同的惩罚。",
    "A higher penalty results in fewer gaps and more mismatches. Should be less than the penalty value of opening a gap in frame to avoid gaps in genes.": "更高的惩罚结果会导致更少的间隙和更多的不匹配。应该小于在框架中打开间隙的惩罚值，以避免基因中的间隙。",
    "Should be greater than the penalty of opening a and less than penalty of opening a gap out of frame, to avoid gaps in genes, but favor gaps that align with codons.": "应该大于打开a的惩罚，小于在框架外打开缺口的惩罚，以避免在基因中产生缺口，但要倾向于与密码子对齐的缺口。",
    "Should be greater than the penalty for opening gaps in-frame to favor gaps that align with codons.": "应该大于在框架内打开间隙的惩罚，以利于与密码子对齐的间隙。",
    "Note that this is redundantly parameterized with score match.": "请注意，这与得分匹配有冗余参数化。",
    "Note that this is redundantly parameterized with mismatch penalty.": "请注意，这与不匹配惩罚参数重复。",
    "Alignments with long indels are slow to compute and require substantial memory in the current implementation. Alignment of sequences with indels that are longer than this value will not be attempted and a warning will be emitted.": "具有较长插入或删除（indels）的比对计算缓慢，并且需要在当前实现中使用大量内存。将不会尝试比对插入或删除长度超过此值的序列，并会发出警告。",
    "Seeds should be long enough to be unique, but short enough to match with high probability.": "种子应该足够长以确保唯一性，但又应该足够短以匹配高概率。",
    "Relevant for short sequences. In long sequences, the number of seeds is determined by nucleotide seed spacing. This should be a positive integer.": "适用于短序列。在长序列中，种子的数量由核苷酸种子间隔决定。这应该是一个正整数。",
    "Maximum number of mismatching nucleotides allowed for a seed to be considered a match.": "允许种子被认为是匹配的最大不匹配核苷酸数量。",
    "The analysis engine to use for lineage assignment. Note: In this version, UShER mode is implemented through the usher-sampled algorithm of UShER. If you prefer to run the original usher algorithm, please use version 4.1.3 of the tool.": "用于谱系分配的分析引擎。注意：在此版本中，UShER模式是通过UShER的usher-sampled算法实现的。如果您更喜欢运行原始的usher算法，请使用工具的4.1.3版本。",
    "Get the latest UShER assignment cache from the pangolin-assignment online repository and use it to speed up UShER lineage assignment. Note: Downloading the cached assignments will only pay off for large numbers of input samples.": "从 pangolin-assignment 在线仓库获取最新的 UShER 分配缓存，并使用它来加速 UShER 谱系分配。注意：下载缓存的分配只有在输入样本数量很大时才会有益。",
    "Please note: pangoLEARN mode is considered deprecated and the underlying machine learning model has been removed from recent versions of pangolin-data. If you really still want to use this mode, you are restricted to the compatible historic pangolin-data versions cached on this Galaxy server.": "请注意：pangoLEARN模式被认为是已弃用的，并且该模式下的基础机器学习模型已从pangolin-data的近期版本中移除。如果您仍然想要使用这个模式，您将被限制在与此Galaxy服务器上缓存的兼容的历史版本pangolin-data上。",
    "Maximum proportion of Ns allowed for pangolin to attempt assignment": "允许的最大Ns比例，以便尝试分配给穿山甲（pangolin）",
    "Minimum query length allowed for pangolin to attempt assignment. Please note that in the current implementation this parameter is used to calculate an alternate value for the 'Maximum proportion of Ns allowed' parameter as 1-(minlen/reflen). The smaller of the two will be used.": "Pangolin尝试分配时允许的最小查询长度。请注意，在当前实现中，此参数用于计算“允许的N的最大比例”参数的替代值，计算公式为1-(minlen/reflen)。将使用两个值中较小的一个。",
    "Optional expanded lineage information as defined in the alias.json file in pangolin-data can be appended as an additional column to the output.": "可选的扩展谱系信息，如在pangolin-data中的alias.json文件中定义的，可以作为额外的列附加到输出中。",
    "Output reports and optionally tree": "输出报告，可选择性地输出树状图",
    "QC config json file containing custom QC configuration": "QC配置文件，包含自定义的QC配置的JSON文件",
    "Text file containing custom root sequence": "包含自定义根序列的文本文件",
    "Auspice JSON v2 file containing custom reference tree": "包含自定义参考树的Auspice JSON v2文件",
    "JSON file containing custom gene map. Gene map (sometimes also called 'gene annotations') is used to resolve aminoacid changes in genes": "包含自定义基因图谱的JSON文件。基因图谱（有时也称为“基因注释”）用于解析基因中的氨基酸变化",
    "CSV file containing a list of custom PCR primer sites. These are used to report mutations in these sites": "包含自定义PCR引物位点列表的CSV文件。这些位点用于报告这些位点上的突变。",
    "For more info on the virus properties JSON file, see the Nextclade documentation.": "有关病毒属性JSON文件的更多信息，请参见Nextclade文档。",
    "The FASTA alignment is an optional output of nextclade. Select this option to include the reference sequence in that alignment file": "FASTA比对是一个可选的输出，由nextclade生成。选择这个选项可以在那个比对文件中包含参考序列。",
    "If this is checked, deletions will be left in the output file": "如果选中此选项，删除的内容将保留在输出文件中",
    "Load a mutation annotated tree file, in protocol-buffers format (protobuf3).": "加载一个以协议缓冲区格式（protobuf3）注释的突变树文件。",
    "Remove the selected samples instead of keeping them in the output files": "移除选定的样本，而不是将它们保留在输出文件中",
    " Toggle to automatically select two representative samples per clade currently included in the tree, pruning all other samples from the tree. Applies after other selection steps": "切换以自动选择当前包含在树中的每个分支的两个代表性样本，从树中剪除所有其他样本。在其他选择步骤之后应用",
    "Resolve all polytomies by assigning branch length 0 relationships arbitrarily. Applied after selection; prevents recondensing of the MAT": "通过将所有多分叉点的分支长度设置为0来解决多分叉问题，关系任意分配。应用在选取之后；防止MAT重新凝聚。",
    "An algorithm automatically locates and annotates clade root nodes.": "一个算法自动定位并注释类群根节点。",
    "Internal node names are not maintained when saving and loading from a .pb file. It is not guaranteed that internal node names will correspond directly between two .pb files, so use the direct assignment method with caution.": "内部节点名称在保存和从.pb文件加载时不会保留。两个.pb文件之间内部节点名称的对应关系不能保证是直接的，因此在使用直接赋值方法时要谨慎。",
    "Minimum fraction of the lineage samples that should be desecendants of the assigned clade root. Used only with --clade-names": "在分配的类群根的后代中，谱系样本的最小比例。仅与 --clade-names 一起使用。",
    "Use to remove current annotations before applying new annotations.": "用于在应用新注释之前移除当前注释。",
    "Minimum allele frequency in input samples for finding the best clade root. Used only with 'remove current annotation'. Used only with --clade-names": "在输入样本中寻找最佳类群根的最小等位基因频率。仅在“移除当前注释”时使用。仅与--clade-names一起使用。",
    "Minimum allele frequency in input samples that should be masked for finding the best clade root. Used only with --clade-names": "在输入样本中，用于寻找最佳类群根的最小等位基因频率。仅与 --clade-names 一起使用",
    "Maximum proportion of samples in a branch that are exemplars from --clade-names to consider when sorting candidate clade root nodes.": "在对候选类群根节点进行排序时，考虑来自--clade-names的样本中作为典型样本的最大比例。",
    "Sample names to restrict. Use to perform masking": "样本名称以限制。用于执行掩蔽",
    "Input VCF file.": "输入 VCF 文件。",
    "Input tree file in Newick format.": "输入树文件，使用Newick格式。",
    "Maximum number of equally parsimonious placements allowed per sample beyond which the sample is ignored.": "每个样本允许的等价简约放置的最大数量，超过此数量的样本将被忽略。",
    "Retain the branch lengths from the input tree in out newick files instead of using number of mutations for the branch lengths.": "在新的新克（Newick）格式文件中保留输入树的分支长度，而不是使用突变次数作为分支长度。",
    "Collapse internal nodes of the input tree with no mutations and condense identical sequences in polytomies into a single node and the save the tree to file condensed-tree.nh in outdir.": "将输入树中没有突变的内部节点折叠起来，并将多叉点中相同序列压缩成单个节点，并将树保存到outdir目录下的condensed-tree.nh文件中。",
    "Collapse internal nodes of the output tree with no mutations before the saving the tree to file final-tree.nh in outdir.": "在将输出树保存到文件 final-tree.nh 中之前，折叠输出树中在 outdir 中没有突变的内部节点。",
    "Save output mutation-annotated tree object to the file new_global_assignments.pb.": "将输出的突变注释树对象保存到文件 new_global_assignments.pb 中。",
    "Write the final tree in uncondensed format and save to file uncondensed-final-tree.nh in outdir.": "将最终的树以非简化格式写入，并保存到outdir目录下的uncondensed-final-tree.nh文件中。",
    "Write minimum set of subtrees covering the newly added samples of size equal to this value.": "编写一组最小的子树，以覆盖新增样本，且子树的大小等于此值。",
    "Similar to write-subtrees-size but produces a single subtree with all newly added samples along with random samples up to the value specified by this argument.": "类似于write-subtrees-size，但是生成一个包含所有新添加样本的单个子树，以及随机样本，直到达到此参数指定的值。",
    "Write the parsimony scores for adding new samples at each existing node in the tree without modifying the tree in a file names parsimony-scores.tsv in outdir.": "在不修改树的情况下，为树中每个现有节点添加新样本的简约性得分，并将其写入名为 parsimony-scores.tsv 的文件，该文件位于 outdir 目录下。",
    "Create a new tree up to this limit for each possibility of parsimony-optimal placement.": "为每个简约最优放置的可能性创建一个新的树，直到这个限制。",
    "A table with at least two columns linking function ids to descriptions for each function": "一个至少包含两列的表格，将功能ID与每个功能描述相链接",
    "The full reference tree containing both study sequences (i.e. ASVs or OTUs) and reference sequences.": "包含研究序列（例如ASV或OTU）和参考序列的完整参考树。",
    "Describes directly observed traits (e.g. sequenced genomes) in tab-delimited format.": "描述以制表符分隔格式直接观察到的特征（例如测序基因组）。",
    "is necessary for the emp_prob method": "对于emp_prob方法来说是必要的",
    "This table is generated by the tool for Hidden state prediction (HSP)": "这个表格是由隐状态预测（HSP）工具生成的",
    "Unstratified or stratified output of PICRUSt2 Metagenome prediction tool": "PICRUSt2 宏基因组预测工具的非分层或分层输出",
    "Intermediate output files will be deleted by default": "默认情况下，中间输出文件将被删除",
    "with gene families as columns and genomes as rows": "以基因家族为列，基因组为行",
    "An input 2 column map table linking function ids to descriptions for each function": "一个输入的2列映射表，将功能ID链接到每个功能描述",
    "Set this for reproducible shufflings": "设置此选项以实现可重复的随机打乱",
    "The dice method alters the total similarity calculation to reflect the length of both sequences. The default for this option is true.": "Dice方法改变了总相似性计算，以反映两个序列的长度。这个选项的默认值是true。",
    "The mapping file is a tabular file with 2 columns. First column contains the sequence names and the second one the corresponding category (--input_mapping)": "映射文件是一个包含两列的表格文件。第一列包含序列名称，第二列包含相应的类别（--input_mapping）。",
    "The alternative is the sum of size for sequences in each category (if the size information is available in sequence name, --number_sum)": "替代方案是每个类别中序列大小的总和（如果序列名称中包含大小信息，--number_sum）",
    "Specify the output format. Default: phylip": "指定输出格式。默认：phylip",
    "For small divergence (say inferior 10%) this option makes no difference. For greater divergence, this option corrects for the fact that observed distances underestimate actual evolutionary distances.": "对于小的分歧（比如说小于10%），这个选项没有区别。对于更大的分歧，这个选项纠正了观察到的距离低估实际进化距离的事实。",
    "Sequence range to write": "序列范围以写入",
    "FASTA or ARB file format (--db)": "FASTA 或 ARB 文件格式 (--db)",
    "For each candidate, the MSA based similarity is calculated and the search result based on these numbers. A value for n one or two orders larger than --search-max-result is usually quite sufficient. (--search-kmer-candidates)": "对于每个候选序列，基于MSA（多序列比对）的相似性被计算出来，并且基于这些数值进行搜索结果的排序。通常，一个比--search-max-result（搜索最大结果数）大一到两个数量级的n值就足够了。（--search-kmer-candidates）",
    "Using the default parameters --search-max-result=10 and --lca-quorum=0.7, this means that the deepest classification shared by 7 out of the top 10 search results is chosen for the query sequence. (--lca-quorum)": "使用默认参数 --search-max-result=10 和 --lca-quorum=0.7，这意味着查询序列选择由前10个搜索结果中7个共享的最深分类作为其分类。(--lca-quorum)",
    "See --fs-kmer-len. (--search-kmer-len)": "见 --fs-kmer-len。（--search-kmer-len）",
    "See --fs-kmer-mm.  (--search-kmer-mm)": "见 --fs-kmer-mm。（--search-kmer-mm）",
    "This option configures the same behavior for the search stage. See --fs-kmer-no-fast. (--search-no-fast)": "这个选项为搜索阶段配置了相同的行为。参见 --fs-kmer-no-fast。(--search-no-fast)",
    "See --fs-kmer-norel. ( --search-kmer-norel)": "参见 --fs-kmer-norel。（--search-kmer-norel）",
    "Sequences not oriented in accordance with the reference database will be reverse complemented as needed. If all is specified, sequences will also be tested for only reversal or only complemented (this should only be necessary if your data was mishandled).": "与参考数据库不一致的序列将根据需要进行反向互补。如果指定了“all”，则序列还将被测试仅反转或仅互补（这只有在您的数据被误处理时才有必要）。",
    "Required to have with the query sequence. This affects the range between --fs-min and --fs-max(--fs-msc)": "需要与查询序列一起使用。这会影响 --fs-min 和 --fs-max(--fs-msc) 之间的范围。",
    "Use 0 to disable weighting. This feature prefers the more common placement for bases with inconsistent alignment in the reference database. (--fs-weight)": "使用0来禁用权重。这个特性更倾向于在参考数据库中不一致对齐的碱基的更常见位置。（--fs-weight）",
    "If fewer sequences than indicated here are found, the respective query sequence will be discarded. (--fs-req)": "如果找到的序列少于这里指定的数量，则相应的查询序列将被丢弃。(--fs-req)",
    "The search will proceed regardless of other settings until this setting has been satisfied. If it cannot be satisfied by any sequence in the reference database, the query sequence will be discarded. This setting exists to ensure that the entire length of the query sequence will be covered in the presence of partial sequences contained within your reference database. Note: If you are working with sequences other than 16S, you need to adjust this value or the value of --fs-full-len accordingly. In particular when working with short reference sequences, this setting may prevent any acceptable reference sequences from being found, leading to no sequences being aligned. (--fs-req-full)": "搜索将继续进行，不管其他设置如何，直到满足这个设置。如果参考数据库中的任何序列都无法满足它，查询序列将被丢弃。这个设置的存在是为了确保在参考数据库中包含的部分序列存在的情况下，查询序列的整个长度都能被覆盖。注意：如果您正在使用16S以外的序列工作，您需要相应地调整这个值或--fs-full-len的值。特别是在处理短的参考序列时，这个设置可能会阻止任何可接受的参考序列被找到，导致没有序列被比对。（--fs-req-full）",
    "See --fs-cover-gene. (--gene-start)": "见 --fs-cover-gene。(--gene-start)",
    "See --fs-cover-gene. (--gene-end)": "查看 --fs-cover-gene 参数。（--gene-end 参数）",
    "Similar to --fs-req-full.  This option is more precise than --fs-req-full, but requires that the column numbers for the range in which the full gene is expected be specified via --gene-start and --gene-end. (--fs-cover-gene)": "类似于--fs-req-full。这个选项比--fs-req-full更精确，但需要通过--gene-start和--gene-end指定期望中完整基因所在的列范围的列号。（--fs-cover-gene）",
    "For SSU rRNA sequences, the default of 10 is a good value. For different sequence types, different values may perform better. For 5S, for example, 6 has shown to be more effective. (--fs-kmer-len)": "对于SSU rRNA序列，10是一个不错的默认值。对于不同类型的序列，不同的值可能会表现得更好。例如，对于5S序列，6被证明更有效。（--fs-kmer-len）",
    "By default, only k-mers starting with an A are used for extra performance. (--fs-kmer-no-fast)": "默认情况下，只有以A开头的k-mers被用于提高额外性能。（--fs-kmer-no-fast）",
    "Absolute (number of shared k-mers) rather than relative (number or shared k-mers divided by length of reference sequence) (--fs-kmer-norel)": "绝对（共享k-mer的数量）而非相对（共享k-mer的数量除以参考序列的长度）的（--fs-kmer-norel）",
    "Only relevant for FASTA output. (--min-idty)": "仅与FASTA输出相关。（--min-idty）",
    "Query sequence file in fasta format": "查询序列文件，使用fasta格式",
    "Reference sequence file in fasta format": "参考序列文件，格式为fasta",
    "Use 32 for larger sequences (above 5mbps) and 16 or smaller otherwise": "对于较大的序列（超过5mbp），使用32；否则使用16或更小的值。",
    "The minimum length a fragment must achieve to be reported": "片段必须达到的最小长度才能被报告",
    "Percentage of similarity threshold (calculated as the attained score divided by the maximum possible score)": "相似性阈值百分比（计算为获得的分数除以可能的最高分数）",
    "File with sequences to cluster in fasta format. See -i, --input in the manual for details.": "用于聚类的序列文件，格式为fasta。有关详细信息，请参见手册中的-i、--input。",
    "Set Automatic to use all labels present in the data or choose a subset of labels to be used. See -l, --labels in the manual for details.": "设置“自动”以使用数据中存在的所有标签，或选择一个子集的标签使用。有关详细信息，请参见手册中的 -l, --labels。",
    "Select which initial clustering algorithm to use.  See --use clinkage and --use greedy in the manual for details.": "选择要使用的初始聚类算法。有关详细信息，请参见手册中的--use clinkage和--use greedy。",
    "Maximal number of positions sequences are allowed to shift for during initial clustering. See -x, --max shift in the manual for details.": "在初始聚类过程中，序列允许的最大偏移位置数。有关详细信息，请参见手册中的 -x，--max shift。",
    "Score penalty added to to each alignment score during initial clustering. This penalty is added for every amino acid aligned towards a (trailing) gap. This value should typically be non-positive (With a positive value, sequences benefit from containing more gaps). See -p, --gap penalty in the manual for details.": "在初始聚类过程中，每个比对得分将加上一个分数惩罚。对于每个与（尾部）间隙对齐的氨基酸，都会加上这个惩罚。这个值通常应该是非正数（正值时，序列会因包含更多间隙而受益）。有关详细信息，请参见手册中的 -p、--gap penalty。",
    "Select the order of sequences during initial clustering. See -R, --order in the manual for details.": "在初始聚类过程中选择序列的顺序。有关详细信息，请参见手册中的 -R, --order。",
    "Set a seed value for the pseudorandom sequence order. See -S, --seed in the manual for details.": "为伪随机序列顺序设置一个种子值。有关详细信息，请参见手册中的-S，--seed。",
    "Select a substitution matrix to be used to score alignments during glreedy clustering. See -m, --matrix in the manual for details.": "选择一个替代矩阵，用于在glreedy聚类过程中对齐序分数。有关详细信息，请参见手册中的-m，--matrix选项。",
    "Minimal alignment score needed for a sequence to join a cluster during initial clustering. Can be either user defined or set automatically based on mean sequence length. See -g, --greedy threshold in the manual for details.": "在初始聚类过程中，序列加入聚类所需的最小比对分数。可以由用户定义，也可以根据平均序列长度自动设置。有关详细信息，请参见手册中的 -g，--greedy 阈值。",
    "Minimal alignment score needed for a sequence to join a cluster during initial clustering.": "在初始聚类过程中，序列加入聚类所需的最小比对得分。",
    "After initial clusering, some of the largest clusters are selected as cluster cores for subsequent clustering procedure. The number of cluster cores can be determined either automatically or manually as top x percent of largest clusters, all clusters satisfying size threshold or exact number of clusters. See -a, --part threshold, -s, --size threshold and -c, --count threshold in the manual for details.": "在初始聚类之后，一些最大的聚类被选为后续聚类过程的聚类核心。聚类核心的数量可以自动或手动确定，作为最大聚类的前x百分比，满足大小阈值的所有聚类，或确切的聚类数量。有关详细信息，请参见手册中的-a（自动）、--part threshold（部分阈值）、-s（大小阈值）和-c（计数阈值）。",
    "See -a, --part threshold in the manual for details.": "参见手册中的 -a、--part 阈值选项以获取详细信息。",
    "See -s, --size threshold in the manual for details.": "请参考手册中的 -s 或 --size 阈值选项以获取详细信息。",
    "See -c, --count threshold in the manual for details": "详见手册中的-c，--count阈值选项",
    "Initial clusters to be used as cluster cores will be selected on the basis of eiter sum of all sequence counts across all labels or unique sequence count.  See -U, --unique in the manual for details.": "初始聚类将根据所有标签中所有序列计数的总和或唯一序列计数来选择作为聚类核心。有关详细信息，请参见手册中的-U，--unique。",
    "Set the number of clustering rounds and score thresholds used. Automatic mode means 3 rounds and score thresholds defined based on mean sequence length.": "设置使用的聚类轮数和分数阈值。自动模式意味着3轮，分数阈值基于平均序列长度定义。",
    "All score thresholds in all clustering rounds can be interpreted either as relative values (per HMM match-state) or absolute values. See -e, --relative thresholds in the manual for details.": "所有聚类轮次中的所有分数阈值可以被解释为相对值（每个HMM匹配状态）或绝对值。详见手册中的-e、--relative thresholds选项。",
    "Minimal score needed for a sequence to be assigned to a cluster. See -n, --assign thresholds in the manual for details.": "序列被分配到一个聚类所需的最小分数。有关详细信息，请参见手册中-n，--assign阈值的说明。",
    "Minimal score needed for two clusters to be considered overlapping. This affects cluster merging step heuristic speedup. If this is set to 0.0, full cluster merging routine will be performed, which is the most precise but the slowest. It is suggested to perform full cluster merging routine at least in the last round. See -v, --overlap thresholds in the manual for details.": "两个聚类被认为是重叠的所需的最小分数。这影响聚类合并步骤的启发式加速。如果将其设置为0.0，将执行完整的聚类合并程序，这是最精确的，但也是速度最慢的。建议至少在最后一轮执行完整的聚类合并程序。有关详细信息，请参见手册中的-v，--overlap阈值。",
    "Minimal score needed for two clusters to be merged. See -r, --merge thresholds in the manual for details": "两个聚类合并所需的最小分数。有关详细信息，请参见手册中关于-r、--merge阈值的说明。",
    " Minimal number of match conserved MSA positions maintained in each cluster throughout the computation. See -h, --min_conserved_positions in the manual for details.": "在每次计算过程中，每个聚类中保持的保守MSA（多序列比对）位置的最小数量。有关详细信息，请参见手册中的-h，--min_conserved_positions选项。",
    "Maximal proportion of gaps in HMM match states. Any multiple sequence alignment column containing more gaps will not be considered a match state. See -y, --max gap proportion in the manual for details.": "在HMM匹配状态中最大间隙比例。任何包含更多间隙的多序列比对列将不被考虑为匹配状态。有关详细信息，请参见手册中的-y，--max gap proportion。",
    "Minimal information content (In terms of Shannon information theory) of HMM match states. Any multiple sequence alignment column having lower information content will not be considered a match state. Minimum: 0.0 (any MSA column composition), maximum: 4.32 (MSA column containing the same amino acid on each line). See -k, --min ic in the manual for details.": "HMM匹配状态的最小信息量（根据香农信息论）。任何具有较低信息量的多序列比对列将不被视为匹配状态。最小值：0.0（任何MSA列组成），最大值：4.32（每行都包含相同氨基酸的MSA列）。有关详细信息，请参见手册中的-k、--min ic。",
    "Maximal multiple sequence alignment length for every cluster. Can be either user defined or specify automatically based on mean sequence length. See -j, --max aln length in the manual for details.": "每个聚类的最长多重序列比对长度。可以由用户定义，或者根据平均序列长度自动指定。有关详细信息，请参见手册中的 -j, --max aln length。",
    "The maximal number of multiple sequence alignment columns for every cluster.": "每个聚类中最大多重序列比对列数。",
    "Maximum number of inner gaps in any line of any cluster's multiple sequence alignment. See -u, --max inner gaps in the manual for details.": "任何聚类中多序列比对的任何一行中内间隙的最大数量。有关详细信息，请参见手册中的 -u，--max inner gaps。",
    "By default, only cluster merging can increase cluster's multiple sequence alignment length. Setting this option to 'Yes' will allow also sequence insertions to icrease the MSA length. See -q, --extension increase length in the manual for details.": "默认情况下，只有聚类合并可以增加聚类的多序列比对长度。将此选项设置为'Yes'也将允许序列插入来增加MSA长度。有关详细信息，请参见手册中的-q、--extension增加长度。",
    "Minimal correlation of label profiles within a cluster. -1 means no correlation limit. See -C, --min correlation in the manual for details": "在聚类内标签轮廓的最小相关性。-1表示没有相关性限制。有关详细信息，请参见手册中的-C、--min相关性。",
    "Use around 1000 for chromosome-sized sequences and around 2000 for complete genomes": "对于染色体大小的序列使用大约1000，对于完整基因组使用大约2000",
    "Use 32 as default, and 16 in case no similarities are found": "使用32作为默认值，如果没有找到相似性，则使用16",
    "Level of the heuristic subsampling employed. Change to 1 or 2 if no similarity is found": "使用启发式子采样的水平。如果没有找到相似性，请更改为1或2",
    "Do not use grid if your multi-fasta contains more than a hundred sequences (approximately)": "如果您的多序列fasta文件包含超过一百个序列（大约），请避免使用网格。",
    "If enabled, a png image will be generated showing the detected blocks and classified by color": "如果启用，将生成一个png图像，显示检测到的区块，并按颜色进行分类。",
    "FastA or multi-FastA": "FastA 或多FastA",
    "Choose a match anchoring strategy": "选择一个匹配锚定策略",
    "Minimum length of a match. (-l)": "匹配的最小长度。（-l）",
    "Direction of complement sequence to use": "要使用的互补序列方向",
    "force 4 column output format regardless of the number of reference sequence inputs. (-F)": "强制输出格式为4列，无论输入的参考序列数量如何。(-F)",
    "Report the query position of a reverse complement match relative to the forward strand of the query sequence. (-c)": "报告相对于查询序列正向链的反向互补配对的查询位置。（-c）",
    "Sampled suffix positions. (-k)": "采样后缀位置。(-k)",
    "Use suffix links in the index and during search? (-suflink)": "在索引中和搜索期间使用后缀链接？（-suflink）",
    "Use child table in the index and during search? (-child)": "在索引和搜索期间使用子表？（-child）",
    "Sparsify the MEM-finding algorithm even more, performing jumps of skip*k [auto (l-10)/k]. (-skip)": "进一步稀疏化MEM查找算法，执行跳跃的步长为skip*k [自动 (l-10)/k]。(-skip)",
    "Use kmer table containing sa-intervals (speeds up searching first k characters) in the index and during search. (-kmer)": "使用包含sa-区间（加速搜索前k个字符）的k-mer表，在索引和搜索期间。（-kmer）",
    "Layout a .delta multiplot in an intelligible fashion. (--layout)": "以易于理解的方式布局一个.delta多图。（--layout）",
    "Set the minimum alignment identity. (-i)": "设置最小比对一致性。(-i)",
    "Set the minimum alignment length. (-l)": "设置最小比对长度。(-l)",
    " Maps each position of each query/reference to its best hit in the reference/query, allowing for reference/query overlaps.": "将每个查询/参考序列的每个位置映射到其在参考/查询中的最佳匹配位置，允许参考/查询重叠。",
    "Set the minimum alignment uniqueness, i.e. percent of the alignment matching to unique reference AND query sequence. (-u)": "设置最小比对唯一性，即比对匹配到唯一参考和查询序列的百分比。（-u）",
    "Set the maximum alignment overlap for overlap options as a percent of the alignment length. (-o)": "将重叠选项的最大对齐重叠设置为对齐长度的百分比。(-o)",
    "Set the distance an alignment extension will attempt to extend poor scoring regions before giving up. (-b)": "设置在放弃之前，对齐扩展将尝试延伸低分区域的距离。（-b）",
    "Sets the minimum length of a cluster of matches. (-c)": "设置匹配簇的最小长度。（-c）",
    "Set the maximum diagonal difference between two adjacent anchors in a cluster. (-D)": "设置一个簇中两个相邻锚点之间的最大对角线差异。(-D)",
    "Set the maximum diagonal difference between two adjacent anchors in a cluster as a differential fraction of the gap length. (-d)": "将两个相邻锚点在聚类中的最大对角线差异设置为间隔长度的差分分数。(-d)",
    "Do not perform cluster extension step. (--noextend)": "不要执行聚类扩展步骤。(--noextend)",
    "Choose a direction of Query Sequence to Use": "选择要使用的查询序列的方向",
    "Set the maximum gap between two adjacent matches in a cluster. (-g)": "设置一个簇中两个相邻匹配之间的最大间隔。(-g)",
    "Set the minimum length of a single exact match. (-l)": "设置单个精确匹配的最小长度。（-l）",
    "Minimum length of an alignment, after clustering and extension. (-L)": "对齐的最小长度，在聚类和扩展之后。(-L)",
    "No alignment score optimization, i.e. if an alignment extension reaches the end of a sequence, it will not backtrack to optimize the alignment score and instead terminate the alignment at the end of the sequence. (--nooptimize)": "不进行比对得分优化，即如果比对扩展到达序列的末端，它不会回溯以优化比对得分，而是在序列的末端终止比对。(--nooptimize)",
    "Don't simplify alignments by removing shadowed clusters. Use this option when aligning a sequence to itself to look for repeats. (--nosimplify)": "不要通过移除被遮蔽的聚类来简化比对。当将序列与其自身进行比对以查找重复序列时，请使用此选项。（--nosimplify）",
    "Enforce absolute banding of dynamic programming matrix based on diagdiff parameter. (--banded)": "强制根据diagdiff参数对动态规划矩阵进行绝对带状处理。（--banded）",
    "Force the use of large offsets. (--large)": "强制使用大偏移量。(--large)",
    "For long query sequences. (-G)": "对于较长的查询序列。(-G)",
    "Stop adding sequence for a thread if more than MAX already. (-M)": "如果一个线程的序列数量超过MAX，则停止添加序列。(-M)",
    "Select no to output all output files": "选择“否”以输出所有输出文件",
    "Merges overlapping alignments regardless of match dir or frame and does not display any idenitity information. (-b)": "合并重叠的比对，不考虑匹配方向或框架，并且不显示任何相似性信息。（-b）",
    "Set minimum percent identity to display (-I)": "设置显示的最小百分比一致性（-I）",
    "Set minimum alignment length to display (-L)": "设置最小比对长度以显示（-L）",
    "Annotate maximal alignments between two sequences, i.e. overlaps between reference and query sequences (-o)": "对两个序列之间的最大比对进行注释，即参考序列和查询序列之间的重叠（-o）。",
    "Amino acid or nucleotide sequences in FASTA format.": "氨基酸或核苷酸序列在FASTA格式中。",
    "Amino acid or nucleotide sequences in aligned FASTA format.": "对齐的FASTA格式中的氨基酸或核苷酸序列。",
    "Keep the given alignment unchanged (--add .)If not, the aligned letters in the seed alignment are preserved but gaps are not necessarily preserved (--seed)": "保持给定的对齐不变（--add .）如果不这样，种子对齐中的对齐字母被保留，但间隔不一定被保留（--seed）",
    "The alignment length is unchanged.  Insertions at the additional sequences are deleted": "对齐长度保持不变。在附加序列中的插入被删除。",
    "Output a correspondence table of positions, sequence.map, between before and after the calculation.  The --mapout option automatically turns on the --keeplength option, to keep the numbering of sites in the reference alignment": "输出一个位置对应表，sequence.map，计算前后之间的对应关系。--mapout选项会自动打开--keeplength选项，以保持参考比对中位点的编号",
    "Run mafft with pre-defined input parameters. Specification of these parameters can be found in the help section.": "使用预定义的输入参数运行mafft。这些参数的具体说明可以在帮助部分找到。",
    "Distance method must be chosen regarding your data": "距离方法必须根据你的数据来选择",
    "Valid with 6mer distance": "有效，6mer距离",
    "Recommended for a large number (> ~10,000) of sequences are input": "推荐用于输入大量（> ~10,000）序列。",
    "Valid when either of --globalpair, --localpair, --genafpair, --fastapair or --blastpair is selected.": "当选择了--globalpair、--localpair、--genafpair、--fastapair或--blastpair中的任意一个时有效。",
    "-2.00 default value": "-2.00 默认值",
    "0.1 default value": "0.1 默认值",
    "-0.1 default value": "-0.1 默认值",
    "-6.00 default value": "-6.00 默认值",
    "0 default value": "0 默认值",
    "1000 for maximum quality": "1000 代表最大质量",
    "Offset value, which works like gap extension penalty, for group-to-group alignment. For E-INS-i, 0 is recommended to allow large gaps": "偏移值，它像间隙延伸惩罚一样，用于组对组比对。对于E-INS-i，建议使用0以允许较大的间隙。",
    "1.53 default value": "1.53 默认值",
    "Usefull only for amino acids": "仅对氨基酸有用",
    "The format of matrixfile is the same to that of BLAST. Ignored when nucleotide sequences are input.": "矩阵文件的格式与BLAST的格式相同。在输入核苷酸序列时忽略。",
    "Either FASTA or ClustalW": "任一FASTA或ClustalW",
    "Single FASTA reference sequence to be aligned": "单个FASTA参考序列需要对齐",
    "Multi-FASTA of seqeunces to be aligned with the reference": "与参考序列对齐的多FASTA序列",
    "The 1-indexed start position of the gene of interest in the reference sequence (For multi-segmented sequences, input each start site separated by a comma ex: 12562,12591)": "参考序列中感兴趣的基因的1-索引起始位置（对于多段序列，输入每个起始位点，用逗号分隔，例如：12562,12591）",
    "The 1-indexed end position of the gene of interest in the reference sequence (For multi-segmented sequences, input each end site separated by a comma ex: 12592,13905)": "参考序列中感兴趣基因的1-索引结束位置（对于多段序列，输入每个结束位点，用逗号隔开，例如：12592,13905）",
    "Sequences that are more than an empirically determined distance from the reference are discarded before the final alignment": "与参考序列在经验确定的距离之外的序列在最终比对之前被丢弃",
    "Compress identical sequences": "压缩相同的序列",
    "Default: 0.7": "默认值：0.7",
    "Set to 2 for large datasets where refinement (iterations 3+) is too slow.": "对于大型数据集，如果细化（第3次及以上迭代）太慢，则设置为2。",
    "Speed up the algorithm, especially for closely related sequences, at the expense of some reduction in accuracy": "加速算法，特别是对于密切相关的序列，以牺牲一些准确性为代价",
    "For nucleotides the -est variant of cd-hit is called": "对于核苷酸，cd-hit的-est变体被称为",
    "Global sequence identity: number of identical alignment positions divided by the full length of the shorter sequence": "全局序列一致性：相同比对位置的数量除以较短序列的全长",
    "NX, to mask out both 'N' and 'X'": "NX，用于遮盖'N'和'X'",
    "Default 2 (1 for T-U and N-N)": "默认值是2（T-U和N-N为1）",
    "Local sequence identity: number of identical amino alignment positions divided by the length of the alignment. Note: in local mode one of -aL, aS, or A must be > 0!": "局部序列同一性：相同氨基酸对齐位置的数量除以对齐的长度。注意：在局部模式下，-aL、aS或A中的一个必须大于0！",
    "Fraction of the longer sequence that must be covered": "较长序列必须覆盖的部分的分数",
    "Maximum number of residues uncovered residues of the longer sequence": "最长序列未覆盖残基的最大数量",
    "Fraction of the shorter sequence that must be covered": "较短序列必须覆盖的部分的比例",
    "Maximum number of residues uncovered residues of the shorter sequence": "较短序列未覆盖残基的最大数量",
    "Minimum number of residues of both sequences that must be covered": "两个序列必须覆盖的最小残基数",
    "If set to 0.9, the shorter sequences need to be at least 90% length of the representative of the cluster": "如果设置为0.9，那么较短序列的长度至少需要是该聚类代表性序列长度的90%。",
    "If set to 60, the length difference between the shorter sequences and the representative of the cluster can not be bigger than 60": "如果设置为60，则较短序列与聚类代表之间的长度差异不能超过60",
    "By default, seqs in db1 >= seqs in db2 in a same cluster if set to 0.9, seqs in db1 may just >= 90% seqs in db2. Only effective in local alignment mode": "默认情况下，如果设置为0.9，在同一簇中，数据库1中的序列数 >= 数据库2中的序列数。只有在局部比对模式下才有效。",
    "by default, seqs in db1 >= seqs in db2 in a same cluster if set to 60, seqs in db2 may 60aa longer than seqs in db1. Only effective in local alignment mode": "默认情况下，如果设置为60，在同一个聚类中db1中的序列数量 >= db2中的序列数量。如果设置为60，db2中的序列可能比db1中的序列长60个氨基酸。只有在局部比对模式下有效。",
    "If set to 0.1, the unmatched region (excluding leading and tailing gaps) must not be more than 10% of the sequence": "如果设置为0.1，则不匹配区域（不包括前导和尾部间隙）不能超过序列的10%",
    "If set to 10, the unmatched region (excluding leading and tailing gaps) must not be more than 10 bases": "如果设置为10，则不匹配区域（不包括前导和尾部间隙）不能超过10个碱基",
    "If false: sequence are stored on hard drive - use for huge data sets": "如果错误：序列存储在硬盘上 - 用于大型数据集",
    "By cd-hit's default algorithm, a sequence is clustered to the first cluster that meet the threshold (fast cluster). If set to true, the program will cluster it into the most similar cluster that meet the threshold (accurate but slow mode)": "根据cd-hit的默认算法，一个序列会被聚类到第一个满足阈值的聚类中（快速聚类）。如果设置为true，程序将会将其聚类到最相似的满足阈值的聚类中（准确但慢模式）。",
    "When disabled, clusters are sorted by decreasing length; if enabled, clusters are sorted by decreasing size": "当禁用时，聚类按照长度递减排序；如果启用，聚类按照大小递减排序",
    "When enabled, output sequences are sorted by decreasing cluster size": "当启用时，输出序列将根据聚类大小递减排序",
    "If set to 0, it takes the fasta defline and stops at first space": "如果设置为0，它将采用fasta定义行，并在第一个空格处停止",
    "Choose one or more reference sequences to mash the query sequences against.": "选择一个或多个参考序列，以便将查询序列与之进行Mash比对。",
    "By default, it is set to 85, implying mappings with 85 or more identity should be reported. For example, it can be set to 80to account for more noisy long-read datasets or 95 for mapping human genome assembly to human reference.": "默认情况下，它设置为85，意味着应该报告具有85%或更高一致性的映射。例如，它可以设置为80以适应更嘈杂的长读长数据集，或者设置为95用于将人类基因组组装映射到人类参考基因组。",
    "Default is 5,000 bp. Sequences below this length are ignored. Mashmap provides guarantees on reporting local alignments of length twice this value.": "默认值是5,000个碱基对(bp)。比这个长度短的序列会被忽略。Mashmap保证报告长度为这个值两倍的局部比对。",
    "This parameter sets the seed density of the winnowing scheme, gauranteeing that the minhash will be calculated from a sample of sketchSize k-mers for each segment. It is set automatically based on --pi but can be manually set as well.": "这个参数设置了 winnowing 方案的种子密度，保证 minhash 将针对每个段的草图大小 k-mers 的样本进行计算。它根据 --pi 自动设置，但也可以手动设置。",
    "This flag will increase the seed density substantially, resulting in a density of roughly 0.02 * (1 + (1 - pi) / .05) where pi is the perc_identity threshold. This leads to longer runtimes and higher RAM usage, but significantly more accurate estimates of ANI.": "这个标志将大幅增加种子密度，导致密度大约为0.02 * (1 + (1 - perc_identity阈值) / .05)，其中pi是百分比身份识别阈值。这会导致运行时间更长，RAM使用量更高，但能显著提高ANI的估计精度。",
    "Mashmap implements a plane-sweep based algorithm to perform the alignment filtering. Similar to delta-filter in nucmer, different filtering options are provided that are suitable for long read or assembly mapping. Option -f map is suitable for reporting the best mappings for long reads, whereas -f one-to-one is suitable for reporting orthologous mappings among all computed assembly to genome mappings.": "Mashmap 实现了一种基于平面扫描的算法来执行比对过滤。类似于 nucmer 中的 delta-filter，提供了不同的过滤选项，适用于长读长或组装映射。选项 -f map 适用于报告长读长的最佳比对，而 -f one-to-one 适用于报告所有计算出的组装到基因组映射中的同源比对。",
    "instead of [0,1]": "而不是 [0,1]",
    "Don't use the hypergeometric filtering and instead use the MashMap2 first pass filtering.": "不要使用超几何分布过滤，而是首先使用MashMap2进行过滤。",
    "Keep the given alignment unchanged .If not, the aligned letters in the seed alignment are preserved but gaps are not necessarily preserved.": "保持给定的对齐不变。如果不是这样，在种子对齐中的对齐字母会被保留，但间隔不一定会被保留。",
    "Output a correspondence table of positions, sequence.map, between before and after the calculation.  The mapout option automatically turns on the keeplength option, to keep the numbering of sites in the reference alignment": "输出一个位置对应表，sequence.map，在计算之前和之后之间的对应关系。mapout选项会自动打开keeplength选项，以保持参考比对中位点的编号",
    "All you have is a single dataset with the sequences to align? You can skip this help text and continue with the default setting. For multiple input datasets, the first mode will launch separate MAFFT jobs for all sequences from the first, second, ..., n-th dataset/element from each input batch, respectively, resulting in n separate MSAs. The second mode will concatenate all input sequences from all inputs for a single run of MAFFT and will generate a single MSA.": "如果你只有一个要进行比对的序列数据集，你可以忽略这段帮助文本并继续使用默认设置。对于多个输入数据集，第一种模式将为每个输入批次中的第一个、第二个、...、第n个数据集/元素的所有序列分别启动单独的MAFFT作业，从而生成n个独立的MSA（多序列比对）。第二种模式将把所有输入序列从所有输入中串联起来，进行一次MAFFT运行，并生成一个单一的MSA。",
    "Amino acid or nucleotide sequences in FASTA format. Add Dataset for concatenation of every additional dataset with each file of the first upload panel": "氨基酸或核苷酸序列，采用FASTA格式。添加数据集以将每个额外的数据集与第一个上传面板中的每个文件进行连接。",
    "The tool can try to detect the type of the input sequences, but you likely want to declare it explicitly. Doing so will also give you control over the scoring matrix used for the alignment, while autodetection will result in the Kimura PAM200 and the BLOSUM62 matrix being used for nucleic acids and protein alignments, respectively.": "该工具可以尝试检测输入序列的类型，但你最好明确声明它。这样做还可以让你控制用于比对的评分矩阵，而自动检测将分别使用Kimura PAM200和BLOSUM62矩阵用于核酸和蛋白质比对。",
    "See the tool help below for details about the available options.": "请参阅下面的工具帮助，了解可用选项的详细信息。",
    "The expected format of the matrix is the same as that used by BLAST.": "矩阵的预期格式与BLAST使用的格式相同。",
    "Run mafft with pre-defined input parameters. Specification of these parameters can be found in the help section. With 'Auto', the tool automatically selects an appropriate strategy from L-INS-i, FFT-NS-i and FFT-NS-2, according to data size from few to many respectively. Default setting: FFT-NS-2.": "使用预定义的输入参数运行mafft。这些参数的具体说明可以在帮助部分找到。选择'Auto'时，工具会根据数据量的大小，自动从L-INS-i、FFT-NS-i和FFT-NS-2中选择一个合适的策略。默认设置：FFT-NS-2。",
    "Parttree is recommended for a large number (> ~10,000) of sequences as input": "Parttree 建议用于输入大量序列（> ~10,000）的情况",
    "Default order is input order.": "默认顺序是输入顺序。",
    "Select delta format if a plot is needed. Jbrowse is a good choice to view cram and bam tracks": "如果需要绘图，请选择delta格式。Jbrowse是查看cram和bam轨迹的好选择",
    "Layout a .delta multiplot in an intelligible fashion.": "以易于理解的方式布局一个.delta多图。",
    "Set the distance an alignment extension will attempt to extend poor scoring regions before giving up.": "设置在放弃之前，对齐扩展将尝试延伸低分区域的距离。",
    "Sets the minimum length of a cluster of matches.": "设置匹配簇的最小长度。",
    "Set the maximum diagonal difference between two adjacent anchors in a cluster.": "设置一个簇中两个相邻锚点之间最大对角线差异。",
    "Set the maximum diagonal difference between two adjacent anchors in a cluster as a differential fraction of the gap length.": "将一个簇中两个相邻锚点之间的最大对角线差异设置为间隔长度的差分分数。",
    "Do not perform cluster extension step.": "不要执行聚类扩展步骤。",
    "Set the maximum gap between two adjacent matches in a cluster.": "设置一个聚类中两个相邻匹配之间的最大间隔。",
    "Set the minimum length of a single exact match.": "设置单个精确匹配的最小长度。",
    "Minimum length of an alignment, after clustering and extension.": "聚类和扩展后对齐的最小长度。",
    "Don't simplify alignments by removing shadowed clusters. Use this option when aligning a sequence to itself to look for repeats.": "不要通过移除阴影聚类来简化比对。当将序列与自身比对以查找重复序列时，请使用此选项。",
    "gff (GFF3) files are valid": "gff（GFF3）文件是有效的",
    "Determine order of isolates in figure (-s)": "确定图中分离物的顺序（-s）",
    "Generate the figure with the subset of isolates listed in the file": "生成包含文件中列出的子集菌株的图表",
    "We recommend to use this option only for experimental analysis of the data": "我们建议您仅在对数据进行实验性分析时使用此选项",
    "The verification is mainly based on the length of the spacer sequences. If a spacer is 1.5 time shorter or longer than the average spacer size of the dataset, a warning is raised and the script stops (Verification is ran by default).": "验证主要基于间隔序列的长度。如果一个间隔比数据集中平均间隔大小短1.5倍或长1.5倍，则会发出警告并停止脚本（默认情况下运行验证）。",
    "The unique spacers will be grayed": "唯一间隔将被置灰",
    "The conserved spacers will be grayed": "保守的间隔区将被置灰",
    "Use this option to keep the same color attributed to the spacer during a previous analysis": "使用此选项可以在先前分析中为间隔区域保持相同的颜色属性",
    "Input a previous .final file you want to retain the colour from": "输入一个你想要保留颜色的先前的 .final 文件",
    "Input a previous .svg file you want to retain the colour from": "输入一个您想要保留颜色的以前的.svg文件",
    "A tab-separated file of read counts. See Help below for format": "一个制表符分隔的读数计数文件。有关格式的详细信息，请参见下面的帮助。",
    "You can choose to either specify the treated samples or the control": "您可以选择指定处理过的样本或对照组",
    "If sample label is provided, the labels must match the labels in the first line of the count table, separated by comma (,); for example, HL60.final,KBM7.final. For sample index, 0,2 means the 1st and 3rd samples are treatment experiments. See Help below for a detailed description.": "如果提供了样本标签，这些标签必须与计数表第一行中的标签相匹配，用逗号（，）分隔；例如，HL60.final,KBM7.final。对于样本索引，0,2 表示第1个和第3个样本是处理实验。有关详细说明，请参见下面的帮助。",
    "Specify the label for the control sample. For every other sample label, the module will treat it as a treatment condition and compare with control sample (usually day 0 or plasmid)": "指定对照样本的标签。对于其他每个样本标签，该模块会将其视为处理条件，并与对照样本进行比较（通常是第0天或质粒）。",
    "If sample label is provided, the labels must match the labels in the first line of the count table, separated by comma (,). Default is all the samples not specified in treatment experiments. See Help below for a detailed description.": "如果提供了样本标签，则这些标签必须与计数表第一行中的标签相匹配，用逗号（，）分隔。默认情况下是所有未在处理实验中指定的样本。有关详细描述，请参见下面的帮助。",
    "Generate PDF of the plots. Default: No": "生成图表的PDF。默认值：否。",
    "Output the .R and .Rnw files to generate the PDF report. The gene summary file will also be output as it is required to regenerate the report. Default: No": "输出.R和.Rnw文件以生成PDF报告。基因摘要文件也将被输出，因为它是重新生成报告所需的。默认值：否",
    "This file includes the logging information during the execution. Default: No": "此文件包含执行期间的日志信息。默认值：否。",
    "If control is specified, the size factor will be estimated using control sgRNAs specified in --control-sgrna option. Default: Median": "如果指定了控制组，将使用在 `--control-sgrna` 选项中指定的对照sgRNA来估算大小因子。默认值：中位数",
    "FDR threshold for gene test. Default: 0.25": "基因测试的FDR阈值。默认值：0.25",
    "Method for sgRNA-level p-value adjustment, including False Discovery Rate (FDR), Holm's method (Holm), or Pounds's method (Pounds). Default: FDR": "sgRNA水平p值校正方法，包括假发现率（FDR）、Holm方法（Holm）或Pounds方法（Pounds）。默认值：FDR",
    "To use, specify sample label or sample index for estimating variances, separated by comma (,). See -t/--treatment-id option for specifying samples.": "使用时，需要指定样本标签或样本索引来估计方差，用逗号（，）分隔。请参见-t/--treatment-id选项以指定样本。",
    "Remove sgRNAs whose mean value is zero in Control, Treatment, Both control/treatment, or Any control/treatment sample. Default: Both (remove those sgRNAs that are zero in both control and treatment samples)": "移除在对照组、处理组、对照/处理组或任何对照/处理组样本中平均值为零的sgRNA。默认值：Both（移除在对照组和处理组样本中均为零的sgRNA）",
    "sgRNA normalized count threshold to be considered removed in the --remove-zero option. Default: 0": "sgRNA标准化计数阈值，用于在--remove-zero选项中考虑是否去除。默认值：0",
    "Method to calculate gene log fold changes (LFC) from sgRNA LFCs. Available methods include the median/mean of all sgRNAs (median/mean), or the median/mean sgRNAs that are ranked in front of the alpha cutoff in RRA (alphamedian/alphamean), or the sgRNA that has the second strongest LFC (secondbest). In the alphamedian/alphamean case, the number of sgRNAs correspond to the goodsgrna column in the output, and the gene LFC will be set to 0 if no sgRNA is in front of the alpha cutoff. Default: Median. (new since v0.5.5)": "从sgRNA的对数倍数变化（LFC）计算基因对数倍数变化的方法。可用的方法包括所有sgRNA的中位数/平均值（median/mean），或在RRA中排名在alpha截止值前面的sgRNA的中位数/平均值（alphamedian/alphamean），或具有第二强LFC的sgRNA（secondbest）。在alphamedian/alphamean的情况下，sgRNA的数量对应于输出中的goodsgrna列，如果没有任何sgRNA在alpha截止值前面，则基因LFC将被设置为0。默认值：中位数。（自v0.5.5起新增）",
    "A list of control sgRNAs for normalization and for generating the null distribution of RRA": "用于标准化和生成RRA的空分布的对照sgRNA列表",
    "A tab-delimited file containing the CNV status for each gene. See Help below for more information and format.": "一个以制表符分隔的文件，包含每个基因的拷贝数变异状态。有关更多信息和格式，请参见下面的帮助。",
    "Name of the column from the CNV profile file to use. See Help below for more information": "要从CNV（拷贝数变异）分析文件中使用的列的名称。有关更多信息，请参见下面的帮助。",
    "Provide a tab-separated count table. Each line in the table should include sgRNA name (1st column), target gene (2nd column) and read counts in each sample. See Help below for more information": "提供一个制表符分隔的计数表。表中的每一行应包括sgRNA名称（第一列）、目标基因（第二列）以及每个样本中的读数计数。有关更多信息，请参见下面的帮助。",
    "You can choose to either provide a design matrix or specify the samples": "您可以选择提供设计矩阵或指定样本",
    "Provide a design matrix, either a file name or a quoted string of the design matrix. For example, 1,1;1,0. The row of the design matrix must match the order of the samples in the count table (if --include-samples is not specified), or the order of the samples by the --include-samples option": "提供一个设计矩阵，可以是文件名或者设计矩阵的字符串引用。例如，1,1;1,0。设计矩阵的行必须与计数表中的样本顺序（如果未指定--include-samples）或通过--include-samples选项指定的样本顺序相匹配。",
    "Specify the sample labels if the design matrix is not given by file in the --design-matrix option. Sample labels are separated by comma (,) and must match the labels in the count table": "如果设计矩阵不是通过 --design-matrix 选项提供的文件给出的，请指定样本标签。样本标签由逗号 (,) 分隔，并且必须与计数表中的标签匹配。",
    "Specify the control sample label (usually day 0 or plasmid). For every other sample label, the MLE module will treat it as a single condition and generate an corresponding design matrix": "指定对照样本标签（通常是第0天或质粒）。对于其他每个样本标签，MLE模块会将其视为一个单独的条件，并生成相应的设计矩阵。",
    "Specify the labels of the variables (i.e., beta), if the design matrix is not given by file in the --design-matrix option. Should be separated by commas (,), and the number of labels must equal to the number of columns of design matrix), including baseline labels. Default: bata_0,beta_1,beta_2,....": "指定变量（例如，beta）的标签，如果设计矩阵不是通过--design-matrix选项提供的文件给出。标签应由逗号（,）分隔，标签的数量必须等于设计矩阵的列数，包括基线标签。默认值：bata_0,beta_1,beta_2,....",
    "A matrix of copy number variation data across cell lines to normalize CNV-biased BetaScores": "一个跨细胞系的拷贝数变异数据矩阵，用于标准化受拷贝数变异（CNV）偏差影响的BetaScores。",
    "Default: 1000": "默认值：1000",
    "The rounds for permutation. The permutation time is (# genes) * x for x rounds of permutation. Default: 2": "排列的轮数。排列时间是（基因数）* x，其中x为排列的轮数。默认值：2",
    "Turning this option on will slow the algorithm": "开启此选项将减慢算法的速度",
    "Optionally sgRNA efficiency information can be incorporated into the analysis. See the MAGeCK website here for more information: https://sourceforge.net/p/mageck/wiki/Home/#tutorial-3-include-the-sgrna-efficiency-into-mle-calculation": "可选地，sgRNA效率信息可以被纳入分析中。有关更多信息，请参见MAGeCK网站：https://sourceforge.net/p/mageck/wiki/Home/#tutorial-3-include-the-sgrna-efficiency-into-mle-calculation",
    "A file of sgRNA efficiency prediction from the SSC program. The efficiency prediction will be used as an initial guess of the probability an sgRNA is efficient. Must contain at least two columns, one containing sgRNA ID, the other containing sgRNA efficiency prediction.": "来自SSC程序的sgRNA效率预测文件。效率预测将用作sgRNA效率的初始猜测。必须至少包含两列，一列包含sgRNA ID，另一列包含sgRNA效率预测。",
    "The sgRNA ID column in sgRNA efficiency prediction file (specified by the --sgrna-efficiency option). Default is 1 (the first column)": "sgRNA效率预测文件中的sgRNA ID列（由--sgrna-efficiency选项指定）。默认为1（第一列）",
    "The sgRNA efficiency prediction column in sgRNA efficiency prediction file (specified by the --sgrna-efficiency option). Default is 2 (the second column).": "在sgRNA效率预测文件（由--sgrna-efficiency选项指定）中，sgRNA效率预测列。默认是第2列（第二列）。",
    "Iteratively update sgRNA efficiency during EM iteration": "在EM迭代过程中迭代更新sgRNA效率",
    "This file includes the logging information during the execution. For count command, it will list some basic statistics of the dataset at the end, including the number of reads, the number of reads mapped to the library, the number of zero-count sgRNAs, etc. Default: No": "此文件包括执行期间的日志信息。对于 count 命令，它将在最后列出数据集的一些基本统计信息，包括读取数量、映射到库的读取数量、零计数 sgRNA 的数量等。默认值：否。",
    "A gene ranking file such as that generated by mageck test. The first column of the rank file must contain gene identifiers, of the same type used in the GMT file": "一个基因排名文件，比如由 mageck test 生成的文件。排名文件的第一列必须包含基因标识符，这些标识符与 GMT 文件中使用的类型相同。",
    "The pathway file in GMT format. See Help below for more information": "GMT格式的通路文件。有关更多信息，请参见下面的帮助。",
    "The column number containing gene scores in the gene ranking file. If you just want to use the ranking of the gene (located at the 1st column), use 0. The column number starts from 0. Default: 0": "基因排名文件中包含基因得分的列号。如果您只想使用基因的排名（位于第1列），请使用0。列号从0开始。默认值：0",
    " If not found, will test all pathways": "如果未找到，将测试所有路径",
    "You can choose to input either separate files of reads (one per sample) or a single count table": "您可以选择输入单独的读取文件（每个样本一个）或单个计数表",
    "The input reads must be in FASTQ, FASTQ.GZ or BAM format and all files must be in the same format.": "输入的序列文件必须是FASTQ、FASTQ.GZ或BAM格式，并且所有文件必须使用相同的格式。",
    "A library file must be provided with three columns containing the sgRNA ID, sequence, and gene it is targeting, see Help below for more information.": "必须提供一个包含三列的库文件，分别包含sgRNA ID、序列和它所针对的基因，有关更多信息，请参见下面的帮助。",
    "By default, the input filenames will be used as the sample labels. Optionally you can specify different sample labels to use which must be separated by comma (,). Must be equal to the number of samples provided in --fastq option.": "默认情况下，输入文件名将用作样本标签。您可以选择使用不同的样本标签，这些标签必须用逗号（,）分隔。必须与在--fastq选项中提供的样本数量相等。",
    "Alternatively, a tab-separated file of read counts can be used as input. See Help below for format": "或者，可以使用制表符分隔的读取计数文件作为输入。有关格式，请参见下面的帮助。",
    "Optionally, a library file can be provided with three columns containing the sgRNA ID, sequence, and gene it is targeting, see Help below for more information.": "可选地，可以提供一个包含三个列的库文件：sgRNA ID、序列和它所针对的基因，有关更多信息，请参见下面的帮助。",
    "Output summary statistics of the fastq files. Default: No": "输出fastq文件的摘要统计信息。默认值：否。",
    "Save nmapped reads to file. Default: No": "将未映射的读取保存到文件。默认：否。",
    "Output the .R and .Rnw files used to generate the plots in the PDF report. The median-normalized read counts file will also be output as it is required to regenerate the plots. Default: No": "输出用于生成PDF报告中图表的.R和.Rnw文件。默认情况下：不输出。同时，还会输出经过中位数标准化的读取计数文件，因为该文件是重新生成图表所需的。",
    "This file includes the logging information, it will list some basic statistics of the dataset at the end. Default: No": "此文件包含日志信息，它将在最后列出数据集的一些基本统计信息。默认值：否。",
    "The pathway file used for QC, in GMT format. By default it will use the GMT file provided by MAGeCK": "用于QC的通路文件，在GMT格式中。默认情况下，它将使用由MAGeCK提供的GMT文件。",
    "Length of trimming the 5' of the reads. Default: 0": "读取5'端修剪的长度。默认值：0",
    "Methods include: None (no normalization), Median (median normalization), Total (normalization by total read counts), Control (normalization by control sgRNAs specified by the --control-sgrna option). Default: Median": "方法包括：无（不进行标准化）、中位数（中位数标准化）、总数（按总读数标准化）、对照（按由--control-sgrna选项指定的对照sgRNA标准化）。默认：中位数",
    "A file of control sgRNA IDs for normalization and for generating the null distribution of RRA": "用于标准化和生成RRA的空分布的对照sgRNA ID文件",
    "The program will automatically determine the sgRNA length from the library file, so only use this if you turn on the --unmapped-to-file option. Default: autodetected": "程序将自动从库文件中确定sgRNA的长度，因此只有在您打开--unmapped-to-file选项时才使用此选项。默认值：自动检测",
    "By default, sgRNAs containing Ns will be discarded": "默认情况下，包含N的sgRNA将被丢弃",
    "Reverse complement the sequences in library for read mapping": "为读取映射反向互补文库中的序列",
    "If this option is on, MAGeCK will only process the first 1M records for each file": "如果这个选项开启，MAGeCK将只处理每个文件的前100万个记录",
    "The gene ranking file generated by the gene test step. Only one enrichment comparison will be performed.": "由基因测试步骤生成的基因排名文件。将仅执行一次富集比较。",
    "The provided file is a (single) gene ranking file, either positive or negative selection. Only one enrichment comparison will be performed. Default: No": "提供的文件是一个（单个）基因排名文件，可以是正向或负向选择。将仅执行一次富集比较。默认值：无",
    "Column number or label in gene summary file for gene ranking; can be either an integer of column number, or a string of column label. Default: 2 (the 3rd column)": "基因摘要文件中用于基因排名的列号或标签；可以是列号的整数，也可以是列标签的字符串。默认值：2（第三列）",
    "Column number or label in gene summary file for gene ranking; can be either an integer of column number, or a string of column label. This option is used to determine the column for positive selections and is disabled if --single-ranking is specified. Default: 8 (the 9th column)": "基因摘要文件中用于基因排名的列号或标签；可以是列号的整数，也可以是列标签的字符串。此选项用于确定正向选择的列，如果指定了--single-ranking则禁用。默认值：8（第9列）",
    "The default alpha value for RRA pathway enrichment. Default: 0.25": "RRA 通路富集的默认 alpha 值。默认值：0.25",
    "Mykrobe predict results in JSON format.": "Mykrobe 预测结果以 JSON 格式呈现。",
    "Links each lineage name to its human-readable counterpart.": "将每个谱系名称链接到其可读的对应名称。",
    "If the average quality of the reads fall below value of average_quality then sequence is filtered. Default is 0": "如果测序的平均质量低于average_quality设定的值，则序列会被过滤掉。默认值为0。",
    "If one of nucleotides has quality below min_quality, then the sequence is filtered. Default is 20": "如果某个核苷酸的质量低于min_quality，则该序列会被过滤掉。默认值为20。",
    "The opening gap score": "开放间隙分数",
    "The gap extension score": "间隙延伸分数",
    "Decide how many mismatches are allowed during primer matching of the reads, that groups reads by experiments. When primer_mismatch = 0 no mismatches are allowed, which can increase number of unasssigned read": "在引物匹配读取时，决定允许有多少个不匹配，这可以通过实验对读取进行分组。当 primer_mismatch = 0 时，不允许有任何不匹配，这可能会增加未分配读取的数量。",
    "How many events of length 1 (mismatches, deletions and insertions of length 1) are allowed when aligning toward the donor template. This parameter is only used when donor template is specified. The higher the parameter the less strict will be algorithm accepting read as HDR. Set to 0 if only perfect alignments to the donor template marked as HDR, unadvised due to error rate of the sequencers": "在对供体模板进行比对时，允许有多少个长度为1的事件（不匹配、长度为1的缺失和插入）。当指定供体模板时，仅使用此参数。参数越高，算法接受作为HDR的读取的严格性就越低。如果仅将完美比对到标记为HDR的供体模板，则将其设置为0，不建议这样做，因为测序仪的误差率。",
    "Normally you want to use both FASTQ files. But in some special cases, you may want to use only the forward file, or only the reverse file": "通常您需要使用两个FASTQ文件。但在某些特殊情况下，您可能只想使用正向文件，或者只使用反向文件。",
    "Value specifying buffer for PRIMER DIMER detection. For a given read it will be recognized as PRIMER DIMER when alignment will introduce gap of size bigger than: length of amplicon - (lengths of PRIMERS + PRIMER_DIMER value)": "用于指定引物二聚体检测的缓冲值。对于给定的读取，当对齐引入的间隙大小大于：扩增子长度 - （引物长度 + 引物二聚体值）时，它将被识别为引物二聚体。",
    "Whether detection of offtarget reads, should be enabled": "是否检测非目标读取，应该被启用",
    "The number of bases by which extend expected cut sites (specified as UPPER case letters in the amplicon) in 5' and 3' directions": "预期切割位点（在扩增子中指定为大写字母）在5'和3'方向上扩展的碱基数量",
    "Whether rules of amplicanConsensus should be promiscuous. When promiscuous, we allow indels that have no confirmation on the other strand": "是否应该使amplicanConsensus的规则具有不专一性。当不专一时，我们允许插入或缺失在另一条链上没有确认的情况下发生。",
    "Whether it should write alignments results to separate files": "是否将比对结果写入到单独的文件中",
    "Select the output files.": "选择输出文件。",
    "Index or key from either '.obs' or '.var'": "索引或键从 '.obs' 或 '.var' 中任选其一",
    "Color points by single variable in `.obs` or `.var`": "在 `.obs` 或 `.var` 中按单个变量为点着色",
    "Use the 'layers' attribute of 'adata' if present": "如果存在，请使用 'adata' 的 'layers' 属性",
    "For continuous annotations used as color parameter": "对于用作颜色参数的连续注释",
    "One or a list of comma-separated index or key from '.var_names' or fields of '.obs'": "一个或多个由逗号分隔的索引或键，来自 '.var_names' 或者 '.obs' 字段",
    "Also when 'groupby is not provided": "同时，当未提供 'groupby' 时",
    "Defaults to 'groupby' if 'rotation' is 'None', otherwise, no label is shown.": "如果 'rotation' 是 'None'，默认为 'groupby'，否则，不显示标签。",
    "Currently, only a single key is supported.": "目前，只支持单个键。",
    "More details in https://docs.scipy.org/doc/scipy/reference/generated/scipy.cluster.hierarchy.linkage.html": "更多详细信息请查看 https://docs.scipy.org/doc/scipy/reference/generated/scipy.cluster.hierarchy.linkage.html",
    "See scipy.spatial.distance.pdist documentation for more options https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.distance.pdist.html": "请参阅scipy.spatial.distance.pdist文档以获取更多选项 https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.distance.pdist.html",
    "Z scores are: z = (x - mean)/std, so values in each row (column) will get the mean of the row (column) subtracted, then divided by the standard deviation of the row (column). This ensures that each row (column) has mean of 0 and variance of 1.": "Z分数是：z = (x - 平均值)/标准差，因此每一行（列）的值将减去该行（列）的平均值，然后除以该行（列）的标准差。这确保了每一行（列）的平均值为0，方差为1。",
    "It means for each row or column, subtract the minimum and divide each by its maximum.": "这意味着对于每一行或每一列，先减去最小值，然后每个数除以其最大值。",
    "Fill it if you do not want to use '.var_names'.": "如果你不想使用 '.var_names'，请填写此处。",
    "See https://seaborn.pydata.org/tutorial/color_palettes.html for more details.": "请查看 https://seaborn.pydata.org/tutorial/color_palettes.html 以获取更多详细信息。",
    "Large patches often look better with slightly desaturated colors, but set this to 1 if you want the plot colors to perfectly match the input color spec.": "大区域通常使用稍微去饱和的颜色看起来更好，但如果你想让绘图颜色完全匹配输入的颜色规格，可以将此设置为1。",
    "1, 2, 3 means first, second and third principal components": "1、2、3 表示第一、第二和第三主成分",
    "This embedded representation should be found in adata.obsm['X_[basis]']": "这种嵌入表示应该在adata.obsm['X_[basis]']中找到",
    "Alternatively, pass _groupby_.": "或者，传递 _groupby_。",
    "Alternatively, pass _key_.": "或者，传递 _key_。",
    "For example, 'G1' in the cell cycle 'phase' covariate. If multiple categories want to be plotted use a comma separated list": "例如，在细胞周期的“阶段”协变量中使用'G1'。如果需要绘制多个类别，请使用逗号分隔的列表。",
    "Each node is represented by its indice (within .categories) for the groups that have been used to run PAGA. Comma-separated": "每个节点由其索引（在 .categories 中）表示，这些组别被用来运行 PAGA，以逗号分隔。",
    "Keys for obs, comma-separated": "观察值的键，用逗号分隔",
    "If not set, use uns/page/groups": "如果未设置，则使用uns/page/groups",
    "If not, annotations have no effect.": "如果没有，注释就没有效果。",
    "If not, each panel has its own y-axis range": "如果不这样，每个面板都有自己的 y 轴范围",
    "A list of comma-separated names": "逗号分隔的名称列表",
    "Normalization is done with respect to each bin. If just a single gene falls into a bin, the normalized dispersion is artificially set to 1": "归一化是针对每个bin进行的。如果只有一个基因落入一个bin中，归一化后的离散度会被人为地设置为1。",
    "Otherwise it merely indicates highly variable genes.": "否则，它仅仅表明基因变异性很高。",
    "If a cell has more than ‘counts_per_cell’, it will be downsampled to this number. Resulting counts can be specified on a per cell basis by passing an array.": "如果一个细胞的计数超过'counts_per_cell'，它将被下采样到这个数字。可以通过传递一个数组来为每个细胞指定结果计数。",
    "If the count matrix has more than total_counts it will be downsampled to have this number.": "如果计数矩阵的计数总数超过total_counts，它将被下采样以具有这个数目。",
    "Higher resolution means finding more and smaller clusters, which defaults to 1.0. See “Time as a resolution parameter” in Lambiotte et al, 2009": "更高的分辨率意味着找到更多和更小的聚类，默认值为1.0。参见Lambiotte等人2009年的“时间作为分辨率参数”。",
    "Higher values lead to more clusters": "更高的值会导致更多的簇",
    "Positive values above 2 define the total number of iterations to perform, -1 has the algorithm run until it reaches its optimal clustering.": "正值大于2定义要执行的迭代次数，-1使算法运行直到达到其最优聚类。",
    "The perplexity is related to the number of nearest neighbors that is used in other manifold learning algorithms. Larger datasets usually require a larger perplexity. Consider selecting a value between 5 and 50. The choice is not extremely critical since t-SNE is quite insensitive to this parameter.": "困惑度与在其他流形学习算法中使用的最近邻数有关。较大的数据集通常需要较大的困惑度。考虑选择5到50之间的值。选择这个参数并不非常关键，因为t-SNE对这一参数相当不敏感。",
    "Controls how tight natural clusters in the original space are in the embedded space and how much space will be between them. For larger values, the space between natural clusters will be larger in the embedded space. Again, the choice of this parameter is not very critical. If the cost function increases during initial optimization, the early exaggeration factor or the learning rate might be too high.": "控制原始空间中自然聚类在嵌入空间中的紧密程度，以及它们之间的空间大小。对于较大的值，嵌入空间中自然聚类之间的空间将会更大。同样，这个参数的选择并不是非常关键。如果成本函数在初始优化期间增加，早期夸大因子或学习率可能太高。",
    "The learning rate can be a critical parameter. It should be between 100 and 1000. If the cost function increases during initial optimization, the early exaggeration factor or the learning rate might be too high. If the cost function gets stuck in a bad local minimum increasing the learning rate helps sometimes.": "学习率可能是一个关键参数。它应该在100到1000之间。如果成本函数在初始优化期间增加，早期夸张因子或学习率可能太高。如果成本函数陷入一个不良的局部最小值，有时增加学习率会有帮助。",
    "Change this to use different intial states for the optimization": "将此更改为使用不同的初始状态进行优化",
    "Smaller values will result in a more clustered/clumped embedding where nearby points on the manifold are drawn closer together, while larger values will result on a more even dispersal of points. The value should be set relative to the 'spread' value, which determines the scale at which embedded points will be spread out. The default of in the 'umap-learn' package is 0.1.": "较小的值将导致更聚集/聚类的嵌入，其中流形上附近的点被拉得更近，而较大的值将导致点的更均匀分散。该值应相对于“spread”值设置，该值决定了嵌入点将展开的尺度。在“umap-learn”软件包中的默认值为0.1。",
    "In combination with 'min_dist' this determines how clustered/clumped the embedded points are.": "结合“min_dist”参数，这决定了嵌入点的聚集/聚集程度。",
    "Called 'n_epochs' in the original UMAP.": "在原始的UMAP中称为'n_epochs'。",
    "Values higher than one will result in greater weight being given to negative samples.": "值高于1将导致负样本被赋予更大的权重。",
    "Called 'init' in the original UMAP": "在原始的UMAP中调用了'init'",
    "It should be a valid 2d observation (e.g. paga)": "它应该是一个有效的二维观测（例如paga）",
    "If not set, it uses the unstructured annotation (uns) / neighbors / connectivities": "如果未设置，则使用非结构化注释（uns）/邻居/连接性",
    "If not set, it appends 'layout'": "如果未设置，则追加“布局”",
    "During recursive splitting of branches ('dpt groups') for 'n_branchings' > 1, do not consider groups that contain less than 'min_group_size' data points. If a float, 'min_group_size' refers to a fraction of the total number of data points.": "在递归地对分支（'dpt groups'）进行拆分（'n_branchings' > 1）时，不考虑包含少于'min_group_size'数据点的组。如果'min_group_size'是一个浮点数，则它指的是数据点总数的分数。",
    "If a very small branch is detected upon splitting, shift away from maximum correlation in Kendall tau criterion of Haghverdi et al (2016) to stabilize the splitting.": "如果在分裂时检测到一个非常小的分支，那么在Haghverdi等人（2016年）的Kendall tau标准中，从最大相关性偏离以稳定分裂。",
    "You can pass your predefined groups by choosing any categorical annotation of observations ('adata.obs').": "您可以通过选择观察值（'adata.obs'）的任何分类注释来传递您预定义的组。",
    "Requires that 'adata.uns' contains a directed single-cell graph with key '['velocyto_transitions']'. This feature might be subject to change in the future.": "要求 'adata.uns' 包含一个有向的单细胞图，其键为 '['velocyto_transitions']'。这个特性在未来可能会有所改变。",
    "If not provided, after normalization, each observation (cell) has a total count equal to the median of the total counts (cells) before normalization.": "如果未提供，在标准化后，每个观察值（单元格）的总计数等于标准化前总计数（单元格）的中位数。",
    " A gene is considered highly expressed, if it has more than max_fraction of the total counts in at least one cell. The not-excluded genes will sum up to target_sum": "一个基因被认为是高度表达的，如果在至少一个细胞中，它的计数占总计数的比例超过max_fraction。不被排除的基因将累加到target_sum",
    "'All' will normalize all layers. The list should be comma-separated.": "\"All\"将对所有层进行归一化。列表应该是逗号分隔的。",
    "Selecting all genes may require a large amount of memory": "选择所有基因可能需要大量的内存",
    "If not set, alpha decaying kernel is not used": "如果未设置，则不使用alpha衰减核",
    "If not set, will be set to 3 * knn": "如果未设置，则将设置为3 * knn",
    "If ‘-1’, this parameter is selected according to the Procrustes disparity of the diffused data.": "如果为‘-1’，则根据扩散数据的Procrustes差异选择此参数。",
    "For extremely large datasets, using n_pca less than 20 allows neighborhoods to be calculated in roughly log(n_samples) time. If not set, no PCA is performed.": "对于非常大的数据集，使用小于20的n_pca可以在大约log(n_samples)的时间内计算邻域。如果未设置，则不执行PCA。",
    "Keys separated by a comma": "用逗号分隔的键",
    " Values (integers) are considered 1-indexed, '50' finds cumulative proportion to the 50th most expressed genes. Values separated by a comma. If empty don't calculate": "值（整数）被视为1-索引的，“50”查找累积比例到表达量第50高的基因。值由逗号分隔。如果为空，则不计算",
    "Larger values result in more global views of the manifold, while smaller values result in more local data being preserved. In general values should be in the range 2 to 100. If 'knn' is 'True', number of nearest neighbors to be searched. If 'knn' is 'False', a Gaussian kernel width is set to the distance of the 'n_neighbors' neighbor.": "较大的值会导致流形的全局视图更多，而较小的值则会导致保留更多的局部数据。一般来说，值应该在2到100的范围内。如果'knn'是'True'，则搜索的最近邻的数量。如果'knn'是'False'，则将高斯核宽度设置为'n_neighbors'邻居的距离。",
    "If not set, the representation is chosen automatically: for n_vars below 50, X is used, otherwise X_pca (uns) is used. If X_pca is not present, it's computed with default parameter": "如果没有设置，表示法将自动选择：对于 n_vars 小于 50 的情况，使用 X；否则使用 X_pca（不监督）；如果 X_pca 不存在，则使用默认参数计算它。",
    "If true, it considers a knn graph. Otherwise, it uses a Gaussian Kernel to assign low weights to neighbors more distant than the 'n_neighbors' nearest neighbor.": "如果为真，它考虑一个k最近邻(knn)图。否则，它使用高斯核为比最近邻(n_neighbors)更远的邻居分配较低的权重。",
    "Genes separated by a comma": "基因用逗号分隔",
    "If 'len(gene_list)' is not too low, you can set 'ctrl_size=len(gene_list)'.": "如果'len(gene_list)'的值不是太低，你可以设置'ctrl_size=len(gene_list)'。",
    "Default is all genes. Genes separated by a comma": "默认是所有基因。基因以逗号分隔。",
    "One gene per line": "每行一个基因",
    "e.g. ['g1', 'g2', 'g3']. If not passed, a ranking will be generated for all groups.": "例如：['g1', 'g2', 'g3']。如果没有传递，将为所有组生成排名。",
    "For small datasets, ‘liblinear’ is a good choice, whereas ‘sag’ and ‘saga’ are faster for large ones. For multiclass problems, only ‘newton-cg’, ‘sag’, ‘saga’ and ‘lbfgs’ handle multinomial loss; ‘liblinear’ is limited to one-versus-rest schemes. ‘newton-cg’, ‘lbfgs’ and ‘sag’ only handle L2 penalty, whereas ‘liblinear’ and ‘saga’ handle L1 penalty.": "对于小数据集，'liblinear' 是一个不错的选择，而 'sag' 和 'saga' 对于较大的数据集更快。对于多类问题，只有 'newton-cg'、'sag'、'saga' 和 'lbfgs' 处理多项式损失；'liblinear' 仅限于一对一的方案。'newton-cg'、'lbfgs' 和 'sag' 仅处理 L2 惩罚，而 'liblinear' 和 'saga' 处理 L1 惩罚。",
    "Prefer primal when n_samples > n_features": "当样本数大于特征数时，优先选择Primal算法。",
    "x becomes [x, self.intercept_scaling], i.e. a 'synthetic' feature with constant value equal to intercept_scaling is appended to the instance vector. The intercept becomes intercept_scaling * synthetic_feature_weight.": "x 变成 [x, self.intercept_scaling]，即向实例向量追加一个常数值等于 intercept_scaling 的“合成”特征。截距变成了 intercept_scaling * synthetic_feature_weight。",
    "It must be a positive float. Like in support vector machines, smaller values specify stronger regularization.": "它必须是一个正浮点数。就像在支持向量机中，较小的值指定了更强的正则化。",
    "If not, it omits zero-centering variables, which allows to handle sparse input efficiently.": "如果不是，它省略了零中心变量，这允许有效地处理稀疏输入。",
    "Clip (truncate) to this value after scaling. If not set, it does not clip.": "在缩放后剪切（截断）到这个值。如果未设置，则不进行剪切。",
    "They should have same number of variables.": "它们应该具有相同数量的变量。",
    "List of comma-separated key from '.var_names'. If not set, all vars are used": "以逗号分隔的来自'.var_names'的键列表。如果没有设置，则使用所有变量",
    "Leave it empty to keep existing indices": "保持现有索引，请将其留空",
    "List of comma-separated key": "逗号分隔的关键字列表",
    "If not set, biological components will not be removed from the correction vectors.": "如果未设置，生物组成部分将不会被从校正向量中移除。",
    "This step takes most computing time.": "这一步占用了大部分计算时间。",
    "List of comma-separated key. If not set, datas are corrected sequentially": "逗号分隔的键列表。如果未设置，则数据将按顺序进行校正。",
    "CB+UMI raw sequence file(s)": "CB+UMI 原始序列文件（们）",
    "Read-sequence file(s)": "读取序列文件（或文件们）",
    "In cases where single-cell protocol supports variable length cellbarcodes, alevin adds nucleotide padding to make the lengths uniform. Furthermore, the padding scheme ensures that there are no collisions added in the process.": "在支持可变长度细胞条形码的单细胞协议的情况下，alevin添加核苷酸填充以使长度统一。此外，填充方案确保在此过程中不会添加任何碰撞。",
    "Tsv with no header, containing two columns mapping each transcript present in the reference to the corresponding gene (the first column is a transcript and the second is the corresponding gene).": "无标题的tsv文件，包含两列，将参考中的每个转录本映射到相应的基因（第一列是转录本，第二列是相应的基因）。",
    "Explicitly specify whitelist CP for cell detection and CB sequence correction. If not specified, putative CBs generated.": "明确指定用于细胞检测和CB序列校正的白名单CP。如果没有指定，将生成假定的CBs。",
    "Causes pipeline to only perform CB correction, then maps the read-sequences to the transcriptome generating the interim data-structure of CB-EqClass-UMI-count. Used in parallel with --dumpBarcodeEq or --dumpBfh for the purposes of obtaining raw information or debugging.": "使管道仅执行CB校正，然后将读序列映射到转录组，生成CB-EqClass-UMI-count的临时数据结构。与--dumpBarcodeEq或--dumpBfh并行使用，用于获取原始信息或调试。",
    "Single column tsv of mitochondrial genes which are to be used as a feature for CB whitelising naive Bayes classification.": "用于特征提取的单个列制表符分隔的线粒体基因，这些基因将用于朴素贝叶斯分类的CB白名单。",
    "Single column tsv of ribosomal genes which are to be used as a feature for CB whitelising naive Bayes classification.": "用于CB白名单朴素贝叶斯分类的单列tsv格式的核糖体基因特征。",
    "Converts the default binary format of alevin for gene-count matrix into a human readable mtx (matrix market exchange) sparse format.": "将alevin的基因计数矩阵的默认二进制格式转换为人类可读的mtx（矩阵市场交换）稀疏格式。",
    "Explicitly specify the number of cells.": "明确指定细胞数量。",
    "define a close upper bound on expected number of cells.": "定义一个关于预期细胞数量的紧密上限。",
    "Performs certain number of bootstrap and generate the mean and variance of the count matrix": "执行一定数量的自助法，并生成计数矩阵的均值和方差。",
    "This value controls the minimum allowed score for a mapping to be considered valid. It matters only when --validateMappings has been passed to Salmon. The maximum possible score for a fragment is ms = read_len * ma (or ms = (left_read_len + right_read_len) * ma for paired-end reads). The argument to --minScoreFraction determines what fraction of the maximum score s a mapping must achieve to be potentially retained. For a minimum score fraction of f, only mappings with a score less than (f * s) will be kept. Mappings with lower scores will be considered as low-quality, and will be discarded.": "这个值控制了映射被认为是有效的最小允许分数。它只在将--validateMappings传递给Salmon时才重要。片段的最大可能分数是ms = read_len * ma（对于成对末端读取，ms = (left_read_len + right_read_len) * ma）。--minScoreFraction参数决定了映射必须达到的最大分数s的分数，才能被潜在地保留。对于最小分数分数f，只有分数小于(f * s)的映射将被保留。分数更低的映射将被视为低质量，并被丢弃。",
    "Use 1 to quantify all": "使用1来量化所有",
    "Default: 100000": "默认值：100000",
    "All subsequent tsv must be the same dimensions as this file. When converted back to tsv using hd5 export, this will be labeled as 'mainmatrix.tsv'": "所有后续的tsv文件必须与此文件具有相同的维度。当使用hd5导出并转换回tsv时，这将被标记为'mainmatrix.tsv'",
    "Adds layers of same dimension to the loom file. When converted to tsv using hd5 export, these layers will retain their names.": "将具有相同维度的层添加到loom文件中。当使用hd5 export转换为tsv时，这些层将保留它们的名称。",
    "First row is column attributes, subsequent are values.": "第一行是列属性，随后的是值。",
    "First row is row attributes, subsequent are values.": "第一行是行属性，随后的是值。",
    "Annotation key in obs or var": "在 obs 或 var 中的注释键",
    "It should be the same number as the old categories": "它应该是与旧类别相同的数字",
    "The new table should have the same number of rows and same order than obs or var. The key names should be in the header (1st line)": "新表格应该具有与obs或var相同数量的行和相同的顺序。键名应该在标题（第一行）中。",
    "Indexes separated by a comma": "用逗号分隔的索引",
    "One index per line": "每行一个索引",
    "The first column must contain gene names, and the first row must contain barcode identifiers.": "第一列必须包含基因名称，第一行必须包含条形码标识符。",
    "The filtering uses intelligent methods to generate output 10X matrices as would be otherwise generated by CellRanger. The ranking option serves to illustrate the number of detected barcodes in the sample.": "过滤使用智能方法生成输出10X矩阵，就像CellRanger生成的矩阵一样。排名选项用于说明样本中检测到的条形码数量。",
    "The DefaultDrops method calls cells according to the number of UMIs associated with each barcode as implemented by CellRanger. The EmptyDrops method identifies empty droplets using a smart ambient background model.": "DefaultDrops 方法根据每个条形码关联的 UMI 数量来调用细胞，这是由 CellRanger 实现的。EmptyDrops 方法使用一个智能环境背景模型来识别空液滴。",
    "The expected number of cells in the sample.": "样本中预期的细胞数量。",
    "The quantile of the top expected barcodes to consider.": "考虑顶部预期条形码的分位数。",
    "The fraction of molecules of the upper quantile that must be exceeded for a cell barcode to be detected.": "分子上四分位数的分数必须超过这个分数，才能检测到细胞条形码。",
    "The lower-bound threshold of the total UMI count at which barcodes beneath this are assumed to be empty droplets.": "在总UMI计数的下限阈值，低于此阈值的条形码被假定为空白液滴。",
    "False Discovery Rate threshold at which droplets with significant deviations from the ambient profile are detected at this threshold. If set to 0, the threshold is ignored.": "假发现率阈值，在该阈值下检测到与环境轮廓有显著偏差的液滴。如果设置为0，则忽略该阈值。",
    "Lower bound on the total UMI count, at or below which all barcodes are assumed to correspond to empty droplets.": "UMI总数的下限，在此下限或以下，所有条形码都被假定为对应于空液滴。",
    "Provide metadata file e.g. containing cell types": "提供元数据文件，例如包含细胞类型",
    "Cell ID column. Values must match those provided in the expression matrix.": "细胞ID列。值必须与表达矩阵中提供的值相匹配。",
    "e.g. Output from DropletUtils Read10x.": "例如，来自DropletUtils Read10x的输出。",
    "A numeric scalar specifying the lower bound on the total UMI count, at or below which all barcodes are assumed to correspond to empty droplets.": "一个数值标量，指定总UMI计数的下限，在此下限或以下的所有条形码都假定对应于空液滴。",
    "An integer scalar specifying the number of iterations to use for the Monte Carlo p-value calculations.": "一个整型标量，用于指定蒙特卡洛 p 值计算中使用的迭代次数。",
    "A numeric scalar specifying the lower bound on the total UMI count, at or below which barcodes will be ignored. (default: none)": "一个数值型标量，指定总UMI计数的下限，低于或等于此值的条形码将被忽略。（默认值：无）",
    "A numeric scalar specifying the threshold for the total UMI count above which all barcodes are assumed to contain cells. (default: none)": "一个数值标量，指定所有条形码假定包含细胞的总UMI计数的阈值。（默认值：无）",
    "The values represented in this options should be categorical": "这个选项中表示的值应该是分类的",
    "The values represented in this options should be numerical and not categorical": "此选项中表示的值应该是数值型而不是分类型",
    "Recommended for large datasets": "推荐用于大型数据集",
    "Raising this number will raise the stringency and may lower the number of included cells": "提高这个数字将提高严格性，可能会减少包含的细胞数量",
    "Cells that exceed this value will be filtered out": "超过这个值的细胞将被过滤掉",
    "Raising this number will raise the stringency and may lower the number of expressed genes": "提高这个数字将提高严格性，可能会降低表达基因的数量",
    "Features that exceed this value will be filtered out": "超过这个值的特征将被过滤掉",
    "The first column of all files is assumed to be feature names and the first row is assumed to be sample names": "所有文件的第一列假定为特征名称，第一行假定为样本名称。",
    "The number of rows (samples) must equal the number of columns in the expression matrix": "表达矩阵中的行数（样本数）必须等于列数",
    "Cell (sample) names used to identify cell controls (for example, blank wells or bulk controls)": "用于识别细胞对照（例如，空白孔或大量对照）的细胞（样本）名称",
    "Requires the RDS output from the cluster analysis": "需要来自聚类分析的RDS输出",
    "Generates tSNE and F-R plots": "生成 tSNE 和 F-R 图",
    "Leave blank to plot all clusters": "留空以绘制所有聚类",
    "Regular expression to be used to remove portions of the cell names, in order to indicate cell type. e.g. if your cells are labelled as 'cdk5_1, cdk5_2, glow_1, glow_2', then a regex of '_\\\\d$' would trim off the ends and set the cell types to 'cdk5, cdk5, glow, glow'.": "用于删除单元格名称部分的正则表达式，以指示细胞类型。例如，如果您的细胞标记为'cdk5_1, cdk5_2, glow_1, glow_2'，那么正则表达式'_\\\\d$'将删除末尾并设置细胞类型为'cdk5, cdk5, glow, glow'。",
    "Regular expression to be used to select for cell types of interest. e.g. if we are only interested in 'cdk5' positive cells, then we would simply put 'cdk5' here.": "用于选择感兴趣的细胞类型的正则表达式。例如，如果我们只对'cdk5'阳性细胞感兴趣，那么我们可以在这里简单地输入'cdk5'。",
    "Performs a Fruchterman-Rheingold projection instead of tSNE": "执行Fruchterman-Rheingold投影而不是tSNE",
    "Leave blank to examine all": "留空以检查全部",
    "Only works if k-nearest neighbours was used in the clustering": "只有在聚类中使用了k最近邻算法时才有效",
    "If blank, the list of genes are used": "如果为空，则使用基因列表",
    "Include only clusters with cells greater than this": "只包含细胞数量大于这个值的聚类",
    "List of cluster numbers to include. Leave blank to include all": "要包含的聚类编号列表。留空以包含全部",
    "Order heatmap by cell names, not by cluster": "按单元格名称排序热图，而不是按聚类排序",
    "Show only average expression for each cluster": "只显示每个簇的平均表达",
    "Normalise gene expression to 1 to depict genes on the same scale": "将基因表达标准化为1，以在同一尺度上描绘基因",
    "Cutoff for displaying differentially expressed genes": "显示差异表达基因的截止值",
    "Display genes with a Benjamini-Hochberg corrected false discovery rate lower than the above P-value threshold": "显示具有低于上述P值阈值的Benjamini-Hochberg校正假发现率的基因",
    "Differentially expressed genes are displayed only for log2 fold-changes higher than this": "差异表达基因仅显示log2倍数变化高于此值的",
    "Differentially expressed genes are displayed only for log2 mean expression greater than this": "差异表达基因仅显示对数2均表达大于这个值的",
    "The minimum total transcripts required. Cells with less than mintotal transcripts are filtered out.": "所需的最小总转录本数。小于mintotal转录本数的细胞被过滤掉。",
    "The minimum required transcript counts of a gene in the minimum number of cells (below)": "一个基因在最小数量的细胞中（下面）所需的最小转录本计数",
    "The minumum number of cells for gene expression to be counted": "基因表达计数的最小细胞数",
    "By default features are counted if they are above zero, but RaceID adds 0.1 to all counts after normalisation to create a non-zero dataset.": "默认情况下，如果特征计数大于零，就会进行计数，但是RaceID在标准化后将所有计数增加0.1，以创建一个非零数据集。",
    "Number of nearest neighbors used to infer corresponding cell types in different batches": "用于推断不同批次中相应细胞类型的最近邻数目",
    "Filter out genes with correlated expression for cell type inference": "过滤掉具有相关表达的基因，用于细胞类型推断",
    "Explicitly filter out genes for cell type inference": "显式过滤基因以进行细胞类型推断",
    "List of regexes to capture experimental batches for batch effect correction": "用于捕获实验批次以进行批次效应校正的正则表达式列表",
    "Correlation coefficient used as a threshold for determining correlated genes": "用作确定相关基因的阈值的相关系数",
    "Method to regress out batch effects": "去除批次效应的方法",
    "P-value cutoff for determining enriched components": "确定富集组分的P值截止值",
    "Upper and lower fraction of gene loadings use for determining enriched components": "用于确定富集组分的基因载荷的上下限分数",
    "If left blank, the maximum number of components are used": "如果留空，则使用最大数量的成分",
    "If ICA is selected, ensure that the number of components value above is sufficiently high": "如果选择了ICA（独立成分分析），请确保上面提到的组件数量值足够高。",
    "Requires the RDS output from the normalisation stage": "需要正规化阶段的RDS输出",
    "Number of nearest neighbours for imputing gene expression": "用于插补基因表达的最近邻数目",
    "Determine number of clusters on saturation point of the mean within-cluster dispersion as a function of the cluster number.": "确定在簇数函数的簇内离散度平均值饱和点上的簇数。",
    "Max number of clusters for the derivation of the cluster number by the saturation of mean within-cluster-dispersion.": "用于通过簇内离散度饱和来推导簇数的最大簇数。",
    "Number of random sample of cells used for the inference of cluster number and Jaccard similarity": "用于推断聚类数量和杰卡德相似度的随机细胞样本数",
    "minimal transcript count of a gene in a clusters to be tested for being an outlier gene": "在聚类中，一个基因的最小转录计数，以测试其是否为异常基因。",
    "Minimum number of outlier genes required for being an outlier cell": "异常细胞所需的最小异常基因数",
    "Reclassification of cell types using out-of-bag analysis is performed based on the final clusters after outlier identification. If 'FALSE', then the cluster partition prior to outlier identification is used for reclassification.": "基于异常值识别后的最终聚类，执行基于袋外分析的细胞类型重新分类。如果为'FALSE'，则在异常值识别之前的聚类划分用于重新分类。",
    "Probability threshold for the above specified minimum number of genes to be an outlier cell. This probability is computed from a negative binomial background model of expression in a cluster": "用于指定最小基因数的异常细胞的概率阈值。这个概率是从聚类中表达的负二项背景模型计算得出的",
    "Outlier cells are merged to outlier clusters if their distance smaller than the outdistquant-quantile of the distance distribution of pairs of cells in the orginal clusters after outlier removal": "异常细胞如果它们与原始聚类中细胞对的距离分布的outdistquant-分位数的距离小于该分位数，则会合并到异常聚类中。",
    "Number of trees based on the number of cells multiplied by this factor. Effective only if the number of trees parameter is set to 0": "基于细胞数量乘以这个因子得到的树的数量。仅当树的数量参数设置为0时有效。",
    "Perplexity of the t-SNE map": "t-SNE 地图的困惑度",
    "Number of nearest neighbours used for the inference of the Fruchterman-Rheingold layout": "用于推断Fruchterman-Rheingold布局的最近邻数量",
    "Number of nearest neighbours used for the UMAP connectivity": "用于UMAP连接性的最近邻数量",
    "Top N genes to print per cluster": "每个聚类打印的前N个基因",
    "Top N genes to plot. Must be less than or equal to the Table Limit": "要绘制的前N个基因。必须小于或等于表格限制",
    "P-value cutoff for the inference of differential gene expression": "差异基因表达推断的P值截止值",
    "This is the output RDS file given by the Cluster stage": "这是由聚类阶段生成的输出RDS文件",
    "Number of nearest neighbours to consider.": "考虑的最近邻居数量。",
    "Clusters to be included in the StemID analysis must have more than this number of cells": "要包含在StemID分析中的簇必须具有超过此数量的细胞",
    "Assign a cell of a given cluster to the cluster with the smallest average distance of the knn nearest neighbours within this cluster.": "将给定簇的一个单元分配给该簇内k最近邻的平均距离最小的簇。",
    "Use Fruchterman-Rheigold layout instead of t-SNE": "使用 Fruchterman-Rheingold 布局而不是 t-SNE",
    "Number of randomizations of cell positions for which to compute projections of cells on inter-cluster links": "计算细胞在簇间链接上的投影时，细胞位置的随机化次数",
    "Cutoff for link significance. This threshold is applied for the calculation of link scores reflecting how uniformly a link is occupied by cells.": "链接显著性阈值。这个阈值用于计算链接得分，反映了细胞如何均匀地占据链接。",
    "All cells on the most highly significant link are and the link itself are disregard to test significance of the remaining links with a binomial p-value": "所有位于最显著链接上的单元格以及链接本身都被忽略，以测试其余链接的显著性，使用二项式p值。",
    "Projections of cells are shown in the plot": "细胞的投影在图中显示",
    "Score threshold for links to be shown in the graph": "图中显示链接的分数阈值",
    "Level of transparency of t-SNE map": "t-SNE 地图的透明度水平",
    "Number of higher order neighbours to be included for the determination of links": "用于确定链接的高阶邻居数量",
    "Score threshold for links to be included in the calculation": "用于计算的链接包含分数阈值",
    "Requires the RDS output from the trajectory analysis": "需要轨迹分析的RDS输出",
    "StemID derives cell lineage trees and predicts multipotent cell identities": "StemID 推导细胞谱系树并预测多能细胞身份",
    "A path of three clusters starting at i, passing through j, and ending at k.": "一个从 i 开始，经过 j，以 k 结束的包含三个聚类的路径。",
    "Number of differentially expressed genes to output per cluster": "每个聚类输出的差异表达基因数量",
    "FateID infers cell fate bias in multipotent progenitor cells": "FateID 推断多能祖细胞中的细胞命运偏差",
    "Vector of valid cluster numbers ordered along the trajectory": "沿轨迹排列的有效簇编号向量",
    "Minimum expression required for at least minnumber cells": "至少需要minnumber个细胞的最小表达量",
    "Minimum number of cells in which a gene needs to be expressed at least a level of minexpr.": "一个基因至少需要在多少个细胞中表达，表达水平至少为 minexpr。",
    "Number of nodes of the self-organizing map.": "自组织映射的节点数。",
    "Pseudo-temporal expression profiles are derived by a local regression of expression values across the ordered cells using the function 'loess' from the package 'stats'. This is the parameter, which controls the degree of smoothing. Larger values return smoother profiles.": "伪时间表达谱是通过使用来自'stats'包的'loess'函数对有序细胞中表达值进行局部回归得到的。这是控制平滑程度的参数。较大的值会返回更平滑的谱。",
    "The z-score of the average normalized pseudo-temporal expression profiles within each node of the self-organizing map is computed, and the correlation of these z-scores between neighbouring nodes is computed. If the correlation is greater than 'corthr', neighbouring nodes are merged": "在自组织图的每个节点内，计算平均标准化伪时间表达式轮廓的z分数，并计算相邻节点之间这些z分数的相关性。如果相关性大于'corthr'，则合并相邻节点。",
    "Nodes of the self-organizing map with less than this number of transcripts are discarded": "自组织映射中转录本数量少于这个数字的节点将被丢弃",
    "Regex to select types across cell names": "正则表达式选取单元格名称中的类型",
    "List of phenotypes factors to be used in the linear regression. Please make sure that each factor has more than one unique value. Names correspond to column names in the bulk RNA dataset phenotype table. If blank, then treat all bulk phenotype columns as factors.": "在线性回归中要使用的表型因素列表。请确保每个因素具有多个不同的值。名称对应于大量RNA数据集表型表中的列名。如果为空，则将所有大量表型列视为因素。",
    "List of phenotype factors to always exclude in the analysis": "在分析中始终排除的表型因素列表",
    "The name of a target scRNA cell type to select in the phenotype comparison.": "在表型比较中选择的目标单细胞RNA细胞类型的名称。",
    "MUST exist in the bulk RNA datasets phenotype factors as above.": "必须存在于大量RNA数据集中的表型因素如上所述。",
    "The (%) threshold at which the phenotype target manifests. Leave at -99 to select all.": "表型目标显现的（%）阈值。留空为-99以选择全部。",
    "Name for target disease group, ideally a value from the scRNA phenotype factor data": "目标疾病组的名称，理想情况下是从单细胞RNA表型因子数据中的一个值。",
    "Used to accentutate certain features in the plots. Increase this number to reduce the effect.": "用于强调图表中的某些特征。增加这个数字可以减少效果。",
    "Leave blank to autoscale each plot.": "留空以自动缩放每个图表。",
    "Insert cell cluster groups based on a previous clustering.": "根据之前的聚类结果插入细胞群组。",
    "e.g. C1 or Cluster1, etc.": "例如 C1 或者 Cluster1 等。",
    "Name of the list of gene markers used to describe the marker list supplied below.": "用于描述下面提供的标记列表的基因标记列表的名称。",
    "A single column of marker genes": "单一列的标记基因",
    "F rows of features and S columns of samples. A Tab delimited file, with a header line and the first column has row names.": "F行特征和S列样本。一个制表符分隔的文件，带有标题行，第一列有行名。",
    "S rows of samples, and V columns of covariates (e.g. sex, age)": "S 行样本，V 列协变量（例如性别、年龄）",
    "A string detailing which chip was used": "一个详细说明使用了哪个芯片的字符串",
    "Metadata should correspond directly to the columns of the Phenotype Data": "元数据应直接对应于表型数据的列",
    "Inspect an aspect of the ESet dataset": "检查ESet数据集的一个方面",
    "Cell type proportion comparisons are performed between bulk datasets in each scRNA group. A second summary is performed comparing all cell type proportions across all groups.": "在每个单细胞RNA组的大量数据集中执行细胞类型比例比较。进行第二次总结，比较所有组中所有细胞类型的比例。",
    "Choose bulk RNA datasets": "选择批量RNA数据集",
    "Name of column in phenotype data containing factor values. If column does not exist, it is a new factor that is applied to all samples in the dataset. Plots will be coloured by these factors.": "在表型数据中包含因子值的列的名称。如果该列不存在，则是一个新因子，该因子将应用于数据集中的所有样本。图表将根据这些因子着色。",
    "One to compare across all": "全部比较的一个",
    "Samples, Cells and Datasets can all be clustered by similarity.": "样本、细胞和数据集都可以根据相似性进行聚类。",
    "Comma-delimited list. Cell types given in the above scRNA datasets are still used for deconvolution (bulk reads are still assigned to discrete cell types), but we merely select the ones we want to show.": "逗号分隔的列表。在上述单细胞RNA数据集中给出的细胞类型仍然用于反卷积（大量读取仍然分配给离散的细胞类型），但我们只选择我们想要显示的那些。",
    "Comma-delimited list. Factors must exist in those inferred from the above bulk datasets.": "逗号分隔的列表。因子必须存在于从上述大量数据集推断出的因子中。",
    "[^:]* matches all characters up to the first colon": "[^:]* 匹配第一个冒号前的所有字符",
    "Allows filtering of implausible fragments that likely result from incorrect mapping positions.": "允许过滤掉可能是由于不正确的映射位置而产生的不可信片段。",
    "Setting this flag means that fragments with the same coordinates can be identified provided they originate from a different cell barcode.": "设置此标志意味着，只要片段来自不同的细胞条形码，就可以识别具有相同坐标的片段。",
    "Peaks BED file or MACS2 narrowPeak file": "峰值 BED 文件或 MACS2 narrowPeak 文件",
    "All resulting peaks will have the same length": "所有生成的峰都将具有相同的长度",
    "Minimum number of features expressed required for a cell to pass filtering": "细胞通过筛选所需表达的特征最小数量",
    "Maximum number of features expressed required for a cell to pass filtering": "一个细胞通过筛选所需的最大特征表达数量",
    "Minimum threshold variability score to retain features": "保留特征的最小阈值变异性得分",
    "Default value is None, if specify it will select a the top most variable features. If this parameter is larger than the total number of feature, it filters based on the min_score argument.": "默认值是None，如果指定它将选择变化最大的特征。如果这个参数大于特征的总数，它将根据min_score参数进行过滤。",
    "Log mode": "对数模式",
    "(gtf_file)": "(gtf_file)",
    "(key_added)": "（密钥已添加）",
    "(upstream)": "（上游）",
    "(feature_type)": "特征类型",
    "(annotation)": "注释",
    "(raw)": "（原始数据）",
    "(pp_pca)": "(主成分分析)",
    "(svd_solver)": "奇异值分解求解器",
    "(nb_pcs)": "主成分个数",
    "(n_neighbors)": "（邻居数）",
    "(perplexity)": "困惑度",
    "(method)": "方法",
    "(metric)": "（度量）",
    "(min_dist)": "（最小距离）",
    "(spread)": "（传播）",
    "(use_highly_variable)": "（使用高度变异）",
    "(n_components)": "（组件数）",
    "(groupby)": "分组",
    "(use_raw)": "（使用原始数据）",
    "(groups)": "（组）",
    "(reference)": "（参考）",
    "(n_features)": "(n_特征数)",
    "(rankby_abs)": "(按绝对值排名)",
    "(corr_method)": "(相关性方法)",
    "(n_cluster)": "（n_cluster）",
    "(range_min)": "（范围最小值）",
    "(range_max)": "（范围最大值）",
    "(max_steps)": "（最大步数）",
    "(num_clusters)": "（聚类数）",
    "This name will appear as prefix of each cell barcode.": "这个名字将作为每个单元格条形码的前缀出现。",
    "Can be Cell ranger output or STAR solo output": "可以是Cell Ranger输出或STAR solo输出",
    "Can be STAR solo barcodes output": "可以是STAR独立条形码输出",
    "Table containing metadata of the various samples (csv formatted rows are samples and cols are entries)": "包含各种样本元数据的表格（以逗号分隔的格式，行是样本，列是条目）",
    "A BAM file for each cell. At least two required.": "每个细胞一个BAM文件。至少需要两个。",
    "If this flag is used every bamfile passed is interpreted as an independent cell, otherwise multiple files are interpreted as batch of different cells to be analyzed together. Important: cells reads should not be distributed over multiple bamfiles is not supported!! (default: off)": "如果使用此标志，则传入的每个 bamfile 被视为独立的细胞，否则多个文件被视为不同细胞的批次，需要一起分析。重要提示：不支持将细胞读取分布到多个 bamfiles 中！！（默认：关闭）",
    "If this flag is used the data is assumed UMI-less and reads are counted instead of molecules (default: off)": "如果使用此标志，则假定数据没有UMI，并且计数读取而不是分子（默认：关闭）",
    "In case UMI is too short to guarantee uniqueness (without information from the ampping) set this parameter to chr, Gene ro [N]bp If set to chr the mapping position (binned to 10Gb intervals) will be appended to UB (ideal for InDrops+dropEst). If set to Gene then the GX tag will be appended to the UB tag. If set to [N]bp the first N bases of the sequence will be used to extend UB (ideal for STRT). (Default: no)": "如果UMI太短而无法保证唯一性（没有来自扩增的信息），请将此参数设置为chr、Gene或[N]bp。如果设置为chr，映射位置（分箱到10Gb区间）将附加到UB（适用于InDrops+dropEst）。如果设置为Gene，则GX标签将附加到UB标签。如果设置为[N]bp，则将使用序列的前N个碱基来扩展UB（适用于STRT）。（默认值：无）",
    "theta=0 does not encourage any diversity. Larger values of theta result in more diverse clusters.": "theta=0 不鼓励任何多样性。theta 的较大值会导致更多样化的聚类。",
    "Lambda must be strictly positive. Smaller values result in more aggressive correction.": "Lambda 必须严格为正。较小的值会导致更激进的校正。",
    "Width of soft kmeans clusters. Sigma scales the distance from a cell to cluster centroids. Larger values of sigma result in cells assigned to more clusters. Smaller values of sigma make soft kmeans cluster approach hard clustering.": "软k均值聚类宽度。Sigma 缩放了一个细胞到聚类中心的距离。Sigma 的较大值会导致细胞被分配到更多的聚类中。Sigma 的较小值会使软k均值聚类接近硬聚类。",
    "nclust=1 equivalent to simple linear regression.": "nclust=1 相当于简单线性回归。",
    "'tau is the expected number of cells per cluster.": "tau 是每个簇中预期的细胞数量。",
    "Between 0 to 1, default 0.05. Larger values may be faster but less accurate.": "介于0到1之间，默认为0.05。较大的值可能会更快，但准确度较低。",
    "Set to -Inf to never stop early.": "设置为-Inf以永不提前停止。",
    "Requires a header row and index column that matches the barcode/cell table": "需要一个包含标题行和索引列的表格，该索引列应与条形码/细胞表相匹配。",
    "Requires a header row and index column that matches the gene table": "需要一个标题行和与基因表匹配的索引列",
    "If Gene ID is selected the index will point to the gene identifiers (first columns expected) and the gene symbols will be left in a field called 'gene_symbols'.": "如果选择了基因ID，索引将指向基因标识符（预期的第一列），而基因符号将保留在名为'gene_symbols'的字段中。",
    "Uses adata.raw, usually what was available before processing. If the layer option is set, this will be ignored (raw will not be used).": "使用adata.raw，通常是在处理之前可用的数据。如果设置了层选项，这将被忽略（不会使用raw）。",
    "Comma separated list of groups existing within the groupby field in obs. Currently it fails if a single group is given, provide at least two.": "在 obs 中的 groupby 字段内存在的组的逗号分隔列表。如果只给定了一个组，它就会失败，至少需要提供两个。",
    "Post-test filtering to only keep genes expressed in at least this fraction of cells in the test group.": "在测试组中，只保留至少在该比例的细胞中表达的基因的后测试过滤。",
    "Post-test filtering to only keep genes expressed in at most this fraction of cells in the reference group.": "在参考组中，只保留在最多这个比例的细胞中表达的基因的后测试过滤。",
    "Post-test filtering to only keep genes with at least this fold change of expression relative to the reference group.": "测试后过滤，只保留与参考组相比表达量至少有此倍数变化的基因。",
    "The method prefers matrices/layers that are logged. Leave empty to use the default .X matrix. It will override the use of raw if set.": "该方法偏好使用对数化的矩阵/层。留空以使用默认的.X矩阵。如果设置，它将覆盖使用原始数据的选项。",
    "If included, the keyword 'METHOD' will be substituted with the value of the method setting and 'RESOLUTION' with the value of that field.": "如果包含关键字 'METHOD'，它将被替换为方法设置的值，'RESOLUTION' 将被替换为该字段的值。",
    "Use the indicated representation. 'X' (for the content of .X, usuaally normalised expression values) or any key for .obsm (e.g. X_pca for PCA) is valid. If not set, the representation is chosen automatically: For .n_vars less than 50, .X is used, otherwise ‘X_pca’ is used. If ‘X_pca’ is not present, it’s computed with default parameters.": "使用指定的表示方法。'X'（用于.X的内容，通常是标准化的表达值）或.obsm中的任何键（例如X_pca用于PCA）都是有效的。如果未设置，则自动选择表示方法：如果.n_vars小于50，则使用.X，否则使用‘X_pca’。如果‘X_pca’不存在，则使用默认参数计算。",
    "If included, PERPLEXITY will be substituted with the value of the perplexity setting.": "如果包含，PERPLEXITY将被替换为困惑度设置的值。",
    "For use with the parameter iterator. Overrides the persplexity option above": "用于参数迭代器。覆盖上述的perplexity选项",
    "Depending on the setup and version, the availability of the needed library might vary and hence fail.": "根据设置和版本，所需的库的可用性可能会有所不同，因此可能会失败。",
    "If not specified, the neighbors data is stored in .uns[‘neighbors’], distances and connectivities are stored in .obsp[‘distances’] and .obsp[‘connectivities’] respectively. If specified, the neighbors data is added to .uns[key_added], distances are stored in .obsp[key_added+’_distances’] and connectivities in .obsp[key_added+’_connectivities’].": "如果未指定，则邻居数据存储在 .uns['neighbors'] 中，距离和连通性分别存储在 .obsp['distances'] 和 .obsp['connectivities'] 中。如果指定了，则邻居数据添加到 .uns[key_added] 中，距离存储在 .obsp[key_added+'_distances'] 中，连通性存储在 .obsp[key_added+'_connectivities'] 中。",
    "'X' (for the content of .X, usually normalised expression values) or any key for .obsm (e.g. X_pca for PCA) is valid. If not set, the representation is chosen automatically: For .n_vars less than 50, .X is used, otherwise ‘X_pca’ is used. If ‘X_pca’ is not present, it’s computed with default parameters.": "'X'（用于.X的内容，通常是标准化后的表达值）或.obsm中的任何键（例如X_pca用于PCA）都是有效的。如果未设置，则自动选择表示方式：对于.n_vars小于50的情况，使用.X；否则使用'X_pca'。如果'X_pca'不存在，则使用默认参数计算它。",
    "When set, inplace subset to highly-variable genes, otherwise only flag highly-variable genes.": "当设置时，将高度可变基因的子集内嵌入，否则仅标记高度可变基因。",
    "If specified, highly-variable genes are selected within each batch separately and merged. This simple process avoids the selection of batch-specific genes and acts as a lightweight batch correction method. For all flavors, genes are first sorted by how many batches they are a HVG. For dispersion-based flavors ties are broken by normalized dispersion. If flavor = 'seurat_v3', ties are broken by the median (across batches) rank based on within-batch normalized variance.": "如果指定了，每个批次内分别选择高变异基因，并将它们合并。这个简单的过程避免了选择批次特异性基因，并作为一种轻量级的批次校正方法。对于所有变体，首先根据它们是高变异基因的批次数量对基因进行排序。对于基于离散度的变体，如果出现并列，则通过归一化离散度来打破平局。如果风味='seurat_v3'，则根据批次内的归一化方差，通过批次间的中位数（跨批次）排名来打破平局。",
    "for example n_genes or n_counts": "例如 n_genes 或 n_counts",
    "Genes with value below min will be discarded.": "值低于最小值的基因将被丢弃。",
    "Genes with value above max will be discarded.": "值超过最大值的基因将被丢弃。",
    "If enabled, specified categories will be removed rather than retained.": "如果启用，将移除指定的类别，而不是保留。",
    "A one-column headerless text file is required": "需要一个没有列标题的单列文本文件",
    "If set, it will recalculate pcts and other existing QC vars, overwriting existing ones.": "如果设置，它将重新计算百分比和其他现有的QC变量，覆盖现有的变量。",
    "Used for flagging mitochondria genes (starting with 'MT-'). Leave empty if gene table already has a boolean column called 'mito' that flags MT genes": "用于标记线粒体基因（以'MT-'开头）。如果基因表已经有一个名为'mito'的布尔列来标记MT基因，则留空",
    "Cells with value below min will be discarded.": "值低于最小值的细胞将被丢弃。",
    "Cells with value above max will be discarded.": "值超过最大值的单元格将被丢弃。",
    "Cells with these values in this categorical variable will be kept.": "具有这些值的分类变量中的细胞将被保留。",
    "This layer name needs to exist within ad.layers or this will fail.": "这个层的名称需要在ad.layers中存在，否则将会失败。",
    "Leave empty and the old X will be lost.": "保持空白，旧的X将会丢失。",
    "A tabular file with headers, where the first column contains cell barcodes. Will be merged via a left join, so not all cells in the obs need to be in the metadata. Currently duplicated column headers will be ignored and the originals in the AnnData will be kept.": "一个带有标题的表格文件，其中第一列包含细胞条形码。将通过左连接进行合并，因此obs中的所有单元格不一定要在元数据中。当前，重复的列标题将被忽略，并将保留AnnData中的原始标题。",
    "If activated, it will do 'adata.raw = adata'": "如果激活，它将执行 'adata.raw = adata'",
    "Name in observations that you want to change": "要更改的观测值中的名称",
    "New name in observations that you want to change to": "您想要更改为的观测值中的新名称",
    "If activated, it will also keep the original column": "如果激活，它还将保留原始列",
    "If activated, it will make the values in the column unique by appending '_dnum' on each repeated value.": "如果激活，它将通过在每个重复的值后面添加'_dnum'来使列中的值唯一。",
    "Name in var that you want to change": "在变量中输入你想要更改的名称",
    "New name in var that you want to change": "在变量var中您想要更改的新名称",
    "Field inside var.params where the gene symbols are, normally 'index' or 'gene_symbols'": "在 var.params 中包含基因符号的字段，通常是 'index' 或 'gene_symbols'",
    "Text that you expect the genes to be flagged to start with, such as 'MT-' for mito genes": "预期基因名称以特定文本开始的标志，例如用于线粒体基因的'MT-'",
    "Name of the column in var.names where this boolean flag is stored, for example 'mito' for mitochondrial genes.": "在var.names中存储此布尔标志的列的名称，例如用于线粒体基因的'mito'。",
    "to calculate percentage of the flagged genes in that number of top genes. Used by sc.pp.calculate_qc_metrics (integer).": "计算在这些顶部基因中标记基因的百分比。由sc.pp.calculate_qc_metrics使用（整数）。",
    "Field inside var or obs to be made unique by appending a suffix (useful for gene symbols in var). A new field will be added with the '_u' suffix. It happens after all the above operations.": "在变量或观测中的字段通过添加后缀使其唯一（对于变量中的基因符号很有用）。将添加一个带有'_u'后缀的新字段。它发生在所有上述操作之后。",
    "Copies adata (subset to matching obs) from this AnnData object into the main input as .raw. Make sure to use an AnnData object containing all .obs in the main input.": "将此AnnData对象中的adata（子集以匹配obs）复制到主输入中作为.raw。确保使用一个包含主输入中所有.obs的AnnData对象。",
    "make sure to use AnnData objects that are compatible in terms of genes, cells and expression data.": "确保使用的AnnData对象在基因、细胞和表达数据方面是兼容的。",
    "Extracts .X from these AnnData objects and merges them into the main input as layers. Make sure to use AnnData objects that are compatible in terms of genes, cells and expression data.": "从这些AnnData对象中提取.X，并将其作为层合并到主输入中。确保使用的AnnData对象在基因、细胞和表达数据方面是兼容的。",
    "will copy all layers in the given AnnData object to the main AnnData object. Make sure to use AnnData objects that are compatible in terms of genes, cells and expression data.": "将给定的AnnData对象中的所有层复制到主AnnData对象中。确保使用的AnnData对象在基因、细胞和表达数据方面是兼容的。",
    "Keys to be copied need to contain the text set here.": "需要复制的键需要包含此处设置的文本。",
    "Extracts layers from these AnnData objects and merges them into the main input. Make sure to use AnnData objects that are compatible in terms of genes, cells and expression data.": "从这些AnnData对象中提取层，并将其合并到主输入中。确保使用的AnnData对象在基因、细胞和表达数据方面是兼容的。",
    "will copy all obs keys in the given AnnData object to the main AnnData object. Make sure to use AnnData objects that are compatible in terms of genes, cells and expression data. You can use this to copy clusters. Uns elements with the same name will be also transferred.": "将复制给定的AnnData对象中的所有obs键到主AnnData对象。请确保使用的AnnData对象在基因、细胞和表达数据方面是兼容的。您可以使用此功能来复制聚类。具有相同名称的Uns元素也将被转移。",
    "Extracts obs (such as clusters) from these AnnData objects and merges them into the main input. Make sure to use AnnData objects that are compatible in terms of genes, cells and expression data.": "从这些AnnData对象中提取obs（例如聚类）并将其合并到主输入中。确保使用的AnnData对象在基因、细胞和表达数据方面是兼容的。",
    "will copy all embedding keys in the given AnnData object to the main AnnData object. Make sure to use AnnData objects that are compatible in terms of genes, cells and expression data. You can use this to copy tSNE, UMAP, etc.": "将复制给定的AnnData对象中的所有嵌入键到主AnnData对象中。确保使用的AnnData对象在基因、细胞和表达数据方面是兼容的。您可以使用此功能来复制tSNE、UMAP等。",
    "Extracts embeddings (tSNE, UMAP) from these AnnData objects and merges them into the main input. Make sure to use AnnData objects that are compatible in terms of genes, cells and expression data.": "从这些AnnData对象中提取嵌入（tSNE，UMAP）并将它们合并到主输入中。确保使用的AnnData对象在基因、细胞和表达数据方面是兼容的。",
    "will copy all uns keys in the given AnnData object to the main AnnData object. Make sure to use AnnData objects that are compatible in terms of genes, cells and expression data. You can use this to copy rank_genes_groups for instance.": "将复制给定的AnnData对象中的所有uns键到主AnnData对象中。确保使用的AnnData对象在基因、细胞和表达数据方面是兼容的。例如，你可以使用这个来复制rank_genes_groups。",
    "Extracts uns (such as ranked_genes_groups) from these AnnData objects and merges them into the main input. Make sure to use AnnData objects that are compatible in terms of genes, cells and expression data.": "从这些AnnData对象中提取uns（例如ranked_genes_groups），并将它们合并到主输入中。确保使用的AnnData对象在基因、细胞和表达数据方面是兼容的。",
    "This might be relevant for interfacing with newer versions of AnnData, that might complain if .raw includes a varm null object.": "这可能与与较新版本的AnnData接口有关，如果.raw包含一个varm空对象，它可能会报错。",
    "Split the AnnData object into multiple AnnData objects based on the values of a given obs key. This is useful for example to split a dataset based on a cluster annotation.": "根据给定的obs键的值将AnnData对象分割成多个AnnData对象。例如，这在基于聚类注释分割数据集时非常有用。",
    "The obs key to split on. For example, if you want to split on cluster annotation, you can use the key 'louvain'.": "obs键用于分割。例如，如果你想根据聚类注释进行分割，可以使用键'louvain'。",
    "If checked, the main AnnData object will be outputted as well.": "如果选中，主AnnData对象也将被输出。",
    "You can pass your predefined groups by choosing any categorical annotation of observations.": "您可以通过选择观察的任何分类注释来传递您预定义的组。",
    "The field used in the AnnData object to store gene symbols, leave blank for the default (index), else a common value is 'gene_symbols'": "在AnnData对象中用于存储基因符号的字段，留空表示默认值（索引），否则一个常见的值是'gene_symbols'",
    "Using this option means that scanpy will expect to find the raw data object. The job will fail if the raw data object is not present within the supplied AnnData file.": "使用此选项意味着Scanpy预期在原始数据对象中找到。如果提供的AnnData文件中不存在原始数据对象，则作业将失败。",
    "By default adata.raw.X is plotted. If use_raw=False is set, then adata.X is plotted. If layer is set to a valid layer name, then the layer is plotted. layer takes precedence over use_raw.": "默认情况下，绘制adata.raw.X。如果设置use_raw=False，则绘制adata.X。如果将layer设置为有效的层名称，则绘制该层。layer优先于use_raw。",
    "If not specified, this looks in .obsp[‘connectivities’] for connectivities (default storage place for pp.neighbors). If specified, this looks .obsp[.uns[neighbors_key][‘connectivities_key’]] for connectivities.": "如果未指定，这会在.obsp['connectivities']中查找连接性（pp.neighbors的默认存储位置）。如果指定了，这会在.obsp[.uns[neighbors_key]['connectivities_key']]中查找连接性。",
    "Automatically determined if not specified.": "如果未指定，则自动确定。",
    "For continuous annotations used as 'color parameter, plot data points with higher values on top of others.": "对于用作“颜色参数”的连续注释，将具有较高值的数据点绘制在其他数据点的上方。",
    "Comma-separated list, e.g. '1,2', '2,3'. To plot all available components use 'all'.": "逗号分隔的列表，例如 '1,2', '2,3'。要绘制所有可用的组件，请使用 'all'。",
    "Aimed counts per cell after normalisation.": "每个细胞标准化后的计数目标。",
    "If enabled, will apply a log transformation following normalisation.": "如果启用，将在规范化后应用对数转换。",
    "Exclude (very) highly expressed genes for the computation of the normalization factor (size factor) for each cell. A gene is considered highly expressed, if it has more than max_fraction of the total counts in at least one cell. The not-excluded genes will sum up to the number specified by --normalize-to.": "在计算每个细胞的标准化因子（大小因子）时，排除（非常）高表达的基因。如果一个基因在至少一个细胞中占总计数的比例超过max_fraction，则认为该基因表达水平高。未排除的基因将累加到由--normalize-to指定的数值。",
    "If None, after normalization, for each layer in layers each cell has a total count equal to the median of the counts_per_cell before normalization of the layer. If 'after', for each layer in layers each cell has a total count equal to the value of --normalize-to. If 'X', for each layer in layers each cell has a total count equal to the median of total counts for observations (cells) of adata.X before normalization.": "如果选择“None”，在标准化后，对于layers中的每一层，每个细胞的总计数等于该层在标准化前的counts_per_cell的中位数。如果选择'after'，在标准化后，对于layers中的每一层，每个细胞的总计数等于--normalize-to的值。如果选择'X'，在标准化后，对于layers中的每一层，每个细胞的总计数等于adata.X在标准化前的观察值（细胞）的总计数的中位数。",
    "No quotes, separated by comma. e.g. 0, 1.": "不要使用引号，用逗号分隔。例如：0, 1。",
    "In micros": "在微观层面",
    "Space separated list of additional metrics to be calculated for every mask.": "为每个掩模计算的额外指标的空格分隔列表。",
    "Space separated list of additional metrics to be calculated for every marker separately.": "为每个标记单独计算的额外指标的空格分隔列表。",
    "If a BED file is specified, then this option will cause the strand column (column 6) to be utilized, if present. Thus, if a region has a '+' in this column, then only metrics from the top strand will be output.": "如果指定了一个 BED 文件，那么这个选项将导致使用第六列（即链表列）中的数据，如果存在的话。因此，如果一个区域在该列中有一个 '+'，则只有来自正链的指标会被输出。",
    "Given a bigWig of Bismap mappability data (using the -M option), MethylDackel can also filter out reads with a mappability score which is too low to support methylation calling.": "给定一个使用 Bismap 可映射性数据的 bigWig 文件（使用 -M 选项），MethylDackel 还可以过滤掉可映射性分数太低而无法支持甲基化调用的读取。",
    "If a bigWig file is provided, this sets the threshold mappability value above which a base is considered mappable (default 0.01).": "如果提供了一个bigWig文件，这将设置一个阈值可映射性值，超过该值的碱基被认为是可映射的（默认值为0.01）。",
    "If a bigWig file is provided, this sets the number of mappable bases needed for a read to be considered mappable (default 15).": "如果提供了一个bigWig文件，这将设置一个读段被认为是可映射的所需最小可映射碱基数量（默认为15）。",
    "Merge per-Cytosine metrics from CpG and CHG contexts into per-CPG or per-CHG metrics": "将CpG和CHG上下文中每个胞嘧啶的指标合并成每个CpG或每个CHG的指标。",
    "Inclusion bounds for methylation calls from reads/pairs origination from the original top strand. Suggested values can be obtained from the MBias program. Each integer represents a 1-based position on a read. For example --OT A,B,C,D translates to, 'Include calls at positions from A through B on read #1 and C through D on read #2'. If a 0 is used a any position then that is translated to mean start/end of the alignment, as appropriate. For example, --OT 5,0,0,0 would include all but the first 4 bases on read #1. Users are strongly advised to consult a methylation bias plot, for example by using the MBias program.": "来自原始顶链的甲基化调用的包含界限。建议值可以从MBias程序获得。每个整数表示一个基于1的读取位置。例如，--OT A,B,C,D 翻译为，“在读取#1的A到B的位置和读取#2的C到D的位置包含调用”。如果0在任何位置使用，则翻译为适当的对齐的开始/结束。例如，--OT 5,0,0,0 将包含读取#1上除了前4个碱基之外的所有内容。强烈建议用户咨询甲基化偏差图，例如使用MBias程序。",
    "As with --OT, but for the original bottom, complementary to the original top, and complementary to the original bottom strands, respectively.": "与--OT参数类似，但是分别针对原始的下链、与原始上链互补的序列以及与原始下链互补的序列。",
    "It can be imported into R or another program for manual plotting and analysis. Note that coordinates are 1-based.": "它可以导入到R或另一个程序中进行手动绘图和分析。请注意，坐标是基于1的。",
    "By default, any alignment marked as a duplicate is ignored. This option causes them to be incorporated.": "默认情况下，任何标记为重复的比对都会被忽略。这个选项会将它们纳入考虑。",
    "By default, if only one read in a pair aligns (a singleton) then it's ignored.": "默认情况下，如果配对的两个读段中只有一个能够比对上（单配体），则忽略它。",
    "By default, paired-end alignments with the properly-paired bit unset in the FLAG field are ignored. Note that the definition of concordant and discordant is based on your aligner settings.": "默认情况下，配对末端比对中在FLAG字段中未设置正确配对位的比对会被忽略。请注意，符合和不符合的定义是基于您的比对器设置的。",
    "Minimum MAPQ threshold to include an alignment (default 10)": "最小MAPQ阈值以包含一个比对（默认为10）",
    "Minimum Phred threshold to include a base (default 5). This must be >0.": "包含碱基的最小Phred阈值（默认为5）。这必须大于0。",
    "Minimum per-base depth for reporting output. If you use --mergeContext, then this applies to the merged CpG/CHG (default 1).": "报告输出的最小每碱基深度。如果您使用--mergeContext，则这适用于合并的CpG/CHG（默认值为1）。",
    "By deault, any alignment marked as secondary (bit 0x100), failing QC (bit 0x200), a PCR/optical duplicate (0x400) or supplemental (0x800) is ignored. This equates to a value of 0xF00 or 3840 in decimal. If you would like to change that, you can specify a new value here.": "默认情况下，任何被标记为次要（位0x100）、未通过QC（位0x200）、PCR/光学重复（0x400）或补充（0x800）的比对都将被忽略。这相当于十进制中的0xF00或3840的值。如果你想要更改这个设置，你可以在这里指定一个新的值。",
    "Require each alignment to have all bits in this value present, or else the alignment is ignored. This is equivalent to the -f option in samtools. The default is 0, which includes all alignments.": "要求每个比对必须具备此值中的所有位，否则该比对将被忽略。这相当于samtools中的-f选项。默认值为0，包括所有比对。",
    "The minimum depth on the strand opposite of a C required to flag the position as possibly being a SNP. The default is 0 (ignore possible SNPs). Setting this to a value above 0 will cause the position to be ignored if a given fraction of non-G bases are present on the opposite strand and the overall coverage there is at least what is specified here.": "在C的对立链上标记位置可能为SNP所需的最小深度。默认值为0（忽略可能的SNP）。如果将此值设置为大于0，则在相反链上存在的非G碱基的比例给定，并且那里的总覆盖度至少为此处指定的值时，该位置将被忽略。",
    "If the depth on the strand opposite of a C is at least that specified by 'Minimum depth for variant avoidance', and the fraction of non-G bases exceeds this value, then methylation extraction is skipped for this position.": "如果C碱基对面链的测序深度至少达到“避免变异的最小深度”所指定的值，并且非G碱基的比例超过这个值，则该位置的甲基化提取将被跳过。",
    "Like --OT, but always exclude INT bases from a given end from inclusion,regardless of the length of an alignment. This is useful in cases where reads may have already been trimmed to different lengths, but still none-the-less contain a certain length bias at one or more ends.": "与--OT参数类似，但是总是从给定端排除INT个碱基不包含在内，不考虑比对的长度。这在读取可能已经被修剪到不同长度的情况下非常有用，但仍然在一端或多端包含某种长度偏差。",
    "As with --nOT, but for the original bottom complementary to the original top, and complementary to the original bottom strands, respectively.": "与--nOT类似，但对于原始的底部互补于原始的顶部，以及原始底部的互补链，分别如此。",
    "Cutoff depends on which method you used for score track. If the file contains pvalue scores from MACS2, score 5 means pvalue 1e-5. Default=5 (--cutoff)": "阈值取决于您用于得分轨迹的方法。如果文件包含来自MACS2的p值得分，得分5表示p值1e-5。默认值=5（--cutoff）",
    " Cutoff for linking regions/low abundance regions depending on which method you used for score track. If the file contains qvalue scores from MACS2, score 1 means qvalue 0.1, and score 0.3 means qvalue 0.5. Default=1 (--cutoff-link)": "链接区域/低丰度区域的截止值，取决于你用于得分轨迹的方法。如果文件包含来自MACS2的qvalue分数，得分1表示qvalue 0.1，得分0.3表示qvalue 0.5。默认值=1 (--cutoff-link)",
    "It is recommended to set it as d value. Default=200 (--min-length)": "建议将其设置为d值。默认值=200（--min-length）",
    "It is recommended to set it to tag size. Default=30 (--lvl1-max-gap)": "建议将其设置为标签大小。默认值=30 (--lvl1-max-gap)",
    "It is recommended to set it to 4 times of d value. Default=800 (--lvl2-max-gap)": "建议将其设置为d值的4倍。默认值=800 (--lvl2-max-gap)",
    "Available choices are: ppois, qpois, subtract, logFE, logLR, and slogLR. They represent Poisson Pvalue (-log10(pvalue) form) using control as lambda and treatment as observation, q-value through a BH process for poisson pvalues, subtraction from treatment, linear scale fold enrichment, log10 fold enrichment(need to set pseudocount), log10 likelihood between ChIP-enriched model and open chromatin model(need to set pseudocount), symmetric log10 likelihood between two ChIP-enrichment models, or maximum value between the two tracks. Default=ppois (--method).": "可用选项包括：ppois、qpois、subtract、logFE、logLR 和 slogLR。它们分别代表使用对照组作为lambda，处理组作为观测值的泊松P值（以 -log10(pvalue) 形式），通过BH过程计算泊松P值的q值，处理组的减法，线性尺度的富集倍数，对数尺度的富集倍数（需要设置伪计数），ChIP富集模型与开放染色质模型之间的对数似然比（需要设置伪计数），两个ChIP富集模型之间的对称对数似然比，或两个轨迹之间的最大值。默认值=ppois（--method）。",
    "The count will be applied after normalization of sequencing depth. Default=0.0 (no pseudocount is applied).": "计数将在测序深度标准化后应用。默认值=0.0（不应用伪计数）。",
    "Default=1.0 correspoinding to likelihood ratio of 1,000 (--cutoff)": "默认值=1.0，对应于1,000的似然比率，（--cutoff参数）",
    " Try bigger value to remove small regions. Default=200 (--min-len)": "尝试使用更大的值来移除较小的区域。默认值=200（--min-len）",
    "Sequencing depths will be used to calculate scaling factor for each sample, to down-scale larger sample to the level of smaller one. For example, while comparing condition 1 (10 Million reads) and condition 2 (20 million reads), use --depth1 10 --depth2 20, then pileup value in bedGraph for condition 2 will be divided by 2.": "测序深度将用于计算每个样本的缩放因子，以将较大的样本缩小到较小样本的水平。例如，在比较条件1（1000万读数）和条件2（2000万读数）时，使用 --depth1 10 --depth2 20，然后条件2的bedGraph中的pileup值将除以2。",
    "Default=1 (--depth1)": "默认值=1 （--depth1）",
    "Default=1 (--depth2)": "默认值=1 (--depth2)",
    "Consider a wider gap for broad marks. Maximum gap should be smaller than minimum length. Default=100 (--max-gap).": "考虑为宽标记使用更大的间隔。最大间隔应该小于最小长度。默认值=100 (--max-gap)。",
    "Cutoff depends on which method you used for score track. If the file contains pvalue scores from MACS2, score 5 means pvalue 1e-5. Default=5.0 (--cutoff)": "截止值取决于您用于得分轨迹的方法。如果文件包含来自MACS2的p值得分，得分5表示p值1e-5。默认值=5.0 (--cutoff)",
    "better to set it as d value. Default=200 (--min-length)": "最好将其设置为d值。默认值=200（--min-length）",
    "better to set it as tag size Default=30 (--max-gap)": "最好将其设置为标签大小默认值30（--max-gap）",
    "While set, bdgpeakcall will analyze number or total length of peaks that can be called by different cutoff then output a summary table to help user decide a better cutoff. Note, --min-len and --max-gap may affect the results. Default=False (--cutoff-analysis)": "当设置时，bdgpeakcall将分析可以通过不同截止值调用的峰的数量或总长度，然后输出一个摘要表以帮助用户决定一个更好的截止值。注意，--min-len和--max-gap可能会影响结果。默认为False (--cutoff-analysis)",
    "required by UCSC Genome Browser (--no-trackline)": "由UCSC基因组浏览器要求（--no-trackline）",
    "For more information, see Help section below": "有关更多信息，请参见下面的帮助部分",
    "For Paired-end BAM (BAMPE) the 'Build model step' will be ignored and the real fragments will be used for each template defined by leftmost and rightmost mapping positions. Default: Single-end BAM": "对于成对末端的BAM文件（BAMPE），“构建模型步骤”将被忽略，并且将使用由最左侧和最右侧的映射位置定义的每个模板的真实片段。默认值：单端BAM",
    "Select the lower region within MFOLD range of high confidence enrichment ratio against background to build model. Fold-enrichment in regions must be higher than lower limit (--mfold). Default: 5": "在MFOLD范围内选择高置信度富集比对背景的较低区域来构建模型。区域内的折叠富集必须高于下限（--mfold）。默认值：5",
    "Select the upper region within MFOLD range of high confidence enrichment ratio against background to build model. Fold-enrichment in regions must be lower than the upper limit (--mfold). Default: 50": "选择在MFOLD范围内具有高置信度富集比的上部区域来构建模型。区域中的折叠富集必须低于上限（--mfold）。默认值：50",
    " You can set this parameter as the medium fragment size expected from sonication or size selection. Default: 300": "您可以将此参数设置为从声波断裂或尺寸选择中预期得到的中等片段大小。默认值：300",
    "The arbitrary extension size in bp. When nomodel is true, MACS will use this value as fragment size to extend each read towards 3-prime; end, then pile them up. It is exactly twice the number of obsolete SHIFTSIZE. In previous language, each read is moved 5-prime-to-3-prime direction to middle of fragment by 0.5 d, then extended to both direction with 0.5 d. This is equivalent to say each read is extended towards 5-prime-to-3-prime into a d size fragment. --extsize (this option) and --shift (the option below) can be combined when necessary. See --shift option below. Default: 200.": "任意扩展大小，单位为碱基对。当nomodel为真时，MACS将使用此值作为片段大小，将每个读取向3'端延伸；然后堆叠它们。这恰好是过时的SHIFTSIZE的两倍。在之前的语言中，每个读取通过0.5 d向5'至3'方向移动到片段的中间，然后向两个方向延伸0.5 d。这相当于说每个读取向5'至3'方向延伸到一个d大小的片段。--extsize（此选项）和--shift（下面的选项）在必要时可以组合使用。请参阅下面的--shift选项。默认值：200。",
    "(NOT the legacy --shiftsize option!) The arbitrary shift in bp. Use discretion while setting it other than default value. When NOMODEL is set, MACS will use this value to move cutting ends (5-prime) towards 5-prime-to-3-prime  direction then apply EXTSIZE to extend them to fragments. When this value is negative, ends will be moved toward 3-prime-to-5-prime  direction. Recommended to keep it as default 0 for ChIP-Seq datasets, or -1 * 0.5 of --extsize (option above) together with --extsize option for detecting enriched cutting loci such as certain DNAseI-Seq datasets. Note, you can't set values other than 0 if format is paired-end data (BAMPE). Default: 0": "（不是过时的--shiftsize选项！）任意的碱基对偏移量。在设置默认值以外的值时请谨慎。当设置了NOMODEL时，MACS将使用这个值将切割末端（5'端）向5'到3'方向移动，然后应用EXTSIZE将它们扩展成片段。当这个值为负时，末端将向3'到5'方向移动。建议对于ChIP-Seq数据集保持默认值0，或者对于检测富集切割位点（例如某些DNAseI-Seq数据集）与--extsize选项一起使用-1乘以--extsize（上面选项）的0.5。注意，如果格式是双端数据（BAMPE），你不能设置0以外的值。默认值：0",
    "default uses q-value": "默认使用q值",
    "Default: not set": "默认：未设置",
    "The q-value (minimum FDR) cutoff to call significant regions. Default is 0.05. For broad marks, you can try 0.05 as cutoff. Q-values are calculated from p-values using Benjamini-Hochberg procedure": "q值（最小FDR）的截止值用于调用显著区域。默认值为0.05。对于宽标记，您可以尝试使用0.05作为截止值。q值是使用Benjamini-Hochberg程序从p值计算得出的。",
    "PDF is only created when the model is built": "PDF 仅在构建模型时创建",
    "By default, the bigger dataset will be scaled down towards the smaller dataset, which will lead to smaller p/qvalues and more specific results. Keep in mind that scaling down will bring down background noise more. Default: No": "默认情况下，较大的数据集将被缩小到较小数据集的大小，这将导致更小的p/q值和更具体的结果。请注意，缩小会更多地降低背景噪声。默认值：否。",
    "up to 9X more time consuming. Default: No": "最多耗时9倍。默认值：否。",
    "Requires 'Scores in bedGraph files (--bdg)' output to be selected. Default: No": "需要选择 'bedGraph 文件中的分数 (--bdg)' 输出。默认值：否。",
    "Default: ignore": "默认：忽略",
    "This is used to capture the bias near the peak summit region. Invalid if there is no control data. If you set this to 0, MACS will skip slocal lambda calculation. *Note* that MACS will always perform a d-size local lambda calculation. The final local bias should be the maximum of the lambda value from d, slocal, and llocal size windows. (--slocal). Default: 1000": "这用于捕获峰值最高区域附近的偏差。如果没有对照数据，则无效。如果将其设置为0，则MACS将跳过slocal lambda计算。*注意*MACS将始终执行d-size local lambda计算。最终的局部偏差应该是d、slocal和llocal大小窗口中lambda值的最大值。(--slocal)。默认值：1000",
    "This is used to capture the surround bias. If you set this to 0, MACS will skip llocal lambda calculation. *Note* that MACS will always perform a d-size local lambda calculation. The final local bias should be the maximum of the lambda value from d, slocal, and llocal size windows. Default: 10000": "这用于捕获周围偏差。如果将其设置为0，MACS将跳过局部lambda计算。*注意*，MACS将始终执行d-size局部lambda计算。最终的局部偏差应该是来自d、slocal和llocal大小窗口的lambda值的最大值。默认值：10000",
    "by putting nearby highly enriched regions into a broad region with loose cutoff": "通过将附近的高富集区域放入一个具有宽松截止值的广泛区域中",
    "value is either p-value or q-value as specified above (--broad-cutoff)": "值可以是如上所述的p值或q值（--broad-cutoff）",
    "Default=1e-5 (--pvalue)": "默认值=1e-5（--pvalue）",
    "If multiple files are given, then they will all be read and combined. Note that pair-end data is not supposed to work with this command. (-i)": "如果给出了多个文件，那么这些文件将全部被读取并合并。请注意，配对末端数据不应与此命令一起使用。（-i）",
    "Percentage of tags you want to keep. Input 80.0 for 80%.(--percentage)": "您希望保留的标签百分比。输入80.0表示80%。(--percentage)",
    "Note that the number of tags in output is approximate as the number specified here (--number)": "请注意，输出中的标签数量是近似的，因为这里指定的数量（--number）",
    "Set the random seed while down sampling data. Must be a positive integer in order to be effective. (--seed)": "在对数据进行下采样时设置随机种子。必须是一个正整数才能有效。(--seed)",
    "default: 5.0 (--cutoff)": "默认值：5.0 (--cutoff)",
    "default: 200bp (--window-size)": "默认：200bp（--window-size）",
    "default: 2.7e+9": "默认值：2.7e+9",
    "default: 1e-5": "默认值：1e-5",
    "Use value less than 0 for default (modeled d)": "使用小于0的值作为默认值（模拟的d）",
    "up to 9X more time consuming": "高达9倍的时间消耗",
    "The default method only consider the peak location, 1k, 5k, and 10k regions in the control data; whereas the new future method also consider the 5k, 10k regions in treatment data to calculate local bias.": "默认方法只考虑对照数据中的峰值位置、1k、5k和10k区域；而新的方法还考虑了处理数据中的5k、10k区域来计算局部偏差。",
    "SICER creates non-standard output files, this option will fix these coordinates": "SICER创建的输出文件是非标准的，这个选项将修正这些坐标",
    "The number of copies of identical reads allowed in a library": "允许在库中存在的相同读取副本的数量",
    "Resolution of SICER algorithm. For histone modifications, one can use 200 bp": "SICER算法的分辨率。对于组蛋白修饰，可以使用200 bp",
    "for determination of the amount of shift from the beginning of a read to the center of the DNA fragment represented by the read. FRAGMENT_SIZE=150 means the shift is 75.": "用于确定从读取的开始位置到由读取表示的DNA片段中心的偏移量。FRAGMENT_SIZE=150意味着偏移是75。",
    "Effective Genome as fraction of the genome size. It depends on read length.": "有效基因组占基因组大小的比例。它取决于读取长度。",
    "Needs to be multiples of window size. Namely if the window size is 200, the gap size should be 0, 200, 400, 600, ...": "需要是窗口大小的倍数。也就是说，如果窗口大小是200，那么间隔大小应该是0, 200, 400, 600, ...",
    "FDR (with control) or E-value (without control)": "FDR（有控制）或E值（无控制）",
    "transcription factor binding default: 300; histone modifications default: 500": "转录因子结合默认值：300；组蛋白修饰默认值：500",
    "transcription factor binding default: 10; histone modifications default: 50": "转录因子结合默认值：10；组蛋白修饰默认值：50",
    "This file should be a name-sorted BAM file. 'samtools sort' can do this with the 'name (-n)' parameter.": "这个文件应该是一个按名称排序的BAM文件。使用'samtools sort'命令和'name (-n)'参数可以实现这一点。",
    "These files should be a name-sorted BAM file. 'samtools sort' can do this with the 'name (-n)' parameter.": "这些文件应该是一个按名称排序的BAM文件。使用'samtools sort'命令和'name (-n)'参数可以实现这一点。",
    "Input BED file of genomic regions to exclude.": "输入要排除的基因组区域的BED文件。",
    "In this process, it analyzes reads/fragments based on their alignments, in three separate groups (proper pairs, discordant pairs, and singletons), and removes those identified as duplicates from further analysis. One novel feature is that this evaluation takes into account reads/fragments with multiple alignments.": "在这个过程中，它根据它们的比对情况，将读段/片段分析为三个独立的组（正确的配对、不一致的配对和单体型），并从进一步分析中移除那些被识别为重复的读段/片段。一个新颖的特性是，这种评估考虑了具有多个比对的读段/片段。",
    "All alignments to the given list of chromosomes (reference sequences) are excluded from peak-calling. More details can be found in the tool description.": "所有对给定染色体列表（参考序列）的比对在峰值调用中被排除。更多细节可以在工具描述中找到。",
    "All alignments with MAPQ less than the given value are eliminated. This is equivalent to filtering with samtools view -q. This option should not be used if the SAM/BAM lists multiple alignments for some reads/fragments. Instead, filtering should be accomplished via -s. (def. 0)": "所有MAPQ小于给定值的比对都被消除。这相当于使用samtools view -q进行过滤。如果SAM/BAM列出了某些reads/fragments的多个比对，则不应使用此选项。相反，应通过-s选项完成过滤。（默认值为0）",
    "Genrich considers all secondary alignments of multimapping reads, but, by default, it keeps only the alignments whose scores are equal to the best score for the read/fragment. Setting a value such as -s 20 causes Genrich also to keep secondary alignments whose scores are within 20 of the best. (def. 0)": "Genrich会考虑多映射读取的所有次要比对，但是，默认情况下，它只保留与读取/片段的最佳得分相等的比对。设置一个值，例如-s 20，会导致Genrich也保留得分在最佳得分20以内的次要比对。（默认值为0）",
    "Unpaired alignments are kept, just as they appear in the SAM/BAM. (def. false)": "未配对的比对被保留，就像它们在SAM/BAM中显示的那样。（默认值：false）",
    "Unpaired alignments are kept, with their lengths changed to the given value (from their 5' ends). (def. not defined)": "未配对的比对被保留，它们的长度被更改为给定的值（从它们的5'端开始）。（默认值：未定义）",
    "Unpaired alignments are kept, with their lengths changed to the average length of fragments inferred from properly paired alignments (excluding those aligning to skipped chromosomes [-e]). (def. not defined)": "未配对的比对被保留，它们的长度被更改为从正确配对的比对中推断出片段的平均长度（不包括那些比对到跳过的染色体的片段[-e]）。（默认值：未定义）",
    "Use ATAC-seq mode (def. false)": "使用 ATAC-seq 模式（默认为 false）",
    "Expand cut sites to x bp (def. 100)": "扩展切割位点至x个碱基对（默认值为100）",
    "Maximum q-value (FDR-adjusted p-value). These parameters establish the statistical threshold below which a base is considered significantly enriched in the experimental sample(s) vs. the control/background. The significance value is automatically converted to a -log10 scale by Genrich. (def. 0.05)": "最大q值（FDR校正后的p值）。这些参数设定了统计阈值，低于该阈值的碱基在实验样本与对照/背景中被认为显著富集。显著性值会自动被Genrich转换为-log10尺度。（默认值为0.05）",
    "When -p is selected, q-values are not calculated (reported as -1). (def. turned off)": "当选择-p时，不计算q值（报告为-1）。（默认关闭）",
    "Minimum AUC for a peak. (def. 20.0)": "峰值的最小AUC值。（默认值20.0）",
    "With this option, any potential peak whose length is below the specified value is discarded, regardless of its significance. The default of 0 means that no peaks are eliminated on this basis. (def. 0)": "使用此选项，任何长度低于指定值的潜在峰值都将被丢弃，无论其显著性如何。默认值为0，表示基于此标准不消除任何峰值。（默认值为0）",
    "This parameter sets the maximum distance between sites that achieve significance in order for them to be linked together into the same potential peak. (def. 100)": "这个参数设置在将位点链接到同一个潜在峰值时，位点之间达到显著性的最大距离。（默认值：100）",
    "This is a convenience option for those who are unsure of the peak-calling parameters but do not want to run the full analysis multiple times. Genrich interprets the alignment files (including identifying PCR duplicates) and produces intermediate log files, but does not perform the peak-calling step.": "这是一个方便的选项，适用于那些不确定峰值调用参数但又不想多次运行完整分析的人。Genrich解释对齐文件（包括识别PCR重复项）并生成中间日志文件，但不会执行峰值调用步骤。",
    "Output bedgraph-ish file for p/q values.": "输出用于 p/q 值的类似 bedgraph 文件。",
    "Output bedgraph-ish file for pileups and p-values.": "输出用于堆积和P值的bedgraph格式文件。",
    "Output BED file for reads/fragments/intervals.": "输出用于读取/片段/区间的BED文件。",
    "Name for the Group that the peak and BAM files belong to e.g. Resistant/Responsive (two Groups must be specified for DiffBind). NOTE: Please only use letters, numbers or underscores.": "峰值和BAM文件所属的组的名称，例如：耐药/敏感（对于DiffBind必须指定两个组）。注意：请仅使用字母、数字或下划线。",
    "Result of your Peak calling experiment": "你的峰值调用实验结果",
    "Specify the Read BAM files used in the Peak calling. The input order of the BAM files for the samples MUST match the input order of the peaks files.": "指定用于峰值呼叫的读取BAM文件。BAM文件的输入顺序对于样品必须与峰值文件的输入顺序相匹配。",
    "If specifying a control BAM file, all samples are required to specify one, see Help section below. The input order of the BAM files for the samples MUST match the input order of the peaks files.": "如果指定了一个对照BAM文件，所有样本都需要指定一个，详见下面的帮助部分。BAM文件的输入顺序必须与峰值文件的输入顺序相匹配。",
    "Column in peak files that contains peak scores. Default: 8 (narrowPeak)": "在峰值文件中包含峰值分数的列。默认值：8（narrowPeak）",
    "DiffBind by default assumes that a higher score indicates a better peak, for example narrowPeaks -log10pvalue. If this is not the case, for example if the score is a p-value or FDR, set this option to Yes. Default: No": "DiffBind默认假设更高的分数表示更好的峰，例如narrowPeaks -log10pvalue。如果情况不是这样，例如分数是p值或FDR，将此选项设置为是。默认值：否。",
    "Extend peaks Nbp up- and downstream of the summit. For punctate peaks it is advisable to extend (e.g. 250bp), see the DiffBind User Guide": "将峰值在峰顶的上游和下游扩展Nbp。对于点状峰，建议扩展（例如250bp），请参阅DiffBind用户指南。",
    "Significance threshold; all sites with FDR less than or equal to this value will be included in the output. A value of 1 will output all binding sites. Default: 0.05": "显著性阈值；所有FDR小于或等于此值的位点将包含在输出中。值为1将输出所有结合位点。默认值：0.05",
    "output an additional PDF file": "输出一个额外的PDF文件",
    "Output a table of the binding scores": "输出一个结合分数的表格",
    "Output all the data used by R to construct the plots and tables, can be loaded into R. Default: No": "输出 R 用来构建图表和表格的所有数据，可以加载到 R 中。默认值：否。",
    "If this option is set to Yes, information from the dba.count and dba.analyze commmands will be output in a text file. Default: No": "如果将此选项设置为Yes，dba.count和dba.analyze命令的信息将输出到一个文本文件中。默认值：No",
    "These modes work differently.": "这些模式的工作原理不同。",
    "Minimum fraction of a block in a cluster for further consideration.": "在聚类中进一步考虑一个块的最小分数。",
    "Minimum fraction of the width of blocks for merging.": "合并区块的最小宽度比例。",
    "Minimum fraction of expression of blocks for merging.": "合并区块所需的表达量最小比例。",
    "Size factors have to be seperated by SPACE": "大小因子必须由空格分隔",
    "Size factors have to be separated by SPACE": "大小因子必须用空格分隔",
    "Restricts intervals accordingly in list output": "限制列表输出中的区间",
    "Genomic chromosomes to learn HMM parameters, e.g. 'chr1;chr2;chr3'. Contigs have to be in the same order as in BAM file. Useful to reduce runtime and memory consumption. Default: all contigs from reference file are used (useful when applying to transcript-wise alignments or poor data).": "用于学习隐马尔可夫模型参数的基因组染色体，例如 'chr1;chr2;chr3'。片段必须与BAM文件中的顺序相同。这有助于减少运行时间和内存消耗。默认值：使用参考文件中的所有片段（适用于转录本对齐或数据不足的情况）。",
    "Contigs to apply HMM, e.g. 'chr1;chr2;chr3;'. Contigs have to be in the same order as in BAM file.": "要应用HMM的连续序列，例如'chr1;chr2;chr3;'。连续序列的顺序必须与BAM文件中的顺序相同。",
    "Mapped reads in BAM format from a control experiment, e.g. eCLIP input": "来自对照实验的映射读取，以BAM格式，例如eCLIP输入",
    "Max. motif ID to use (Default: only covariates with motif ID 1 are used)": "最大使用的主题ID（默认值：仅使用主题ID为1的协变量）",
    "NOTE: Increasing the bandwidth increases runtime and memory consumption": "注意：增加带宽会增加运行时间和内存消耗",
    "For proteins that rather slide along the RNA or show long crosslink clusters increase -bwn, e.g. to 100 (should be LE 4*bw)": "对于沿RNA滑动或显示长交联簇的蛋白质，增加-bwn参数，例如增加到100（应小于等于4*bw）。",
    "Either automatically choose n threshold (-ntp, -ntp2) to estimate parameters linked to crosslink states based on expected read start count at crosslink sites, or manually set values": "自动选择n个阈值（-ntp，-ntp2）以基于交联位点处预期的读取起始计数来估计与交联状态相关的参数，或手动设置值",
    "Useful for data with low truncation rates at crosslink sites or in general high fraction of non-coinciding read starts": "适用于交联位点或一般高比例非一致性起始点的低截断率数据",
    "Useful in cases of extreme outliers, e.g. extreme high read start counts whose emission probabilities are close to zero and which would be discarded in default setting (along with warning messages). Note: increases memory consumption. Use in combination with '-iv' (default: double)": "在极端异常值的情况下非常有用，例如极端高的读段起始计数，其发射概率接近零，这些在默认设置中会被丢弃（同时伴随着警告消息）。注意：会增加内存消耗。与'-iv'组合使用（默认：双精度）。",
    "When incorporating input signal, do not constrain 'non-enriched' shape parameter k LE 'enriched' gamma parameter k": "在整合输入信号时，不要限制“非富集”形状参数 k 小于等于“富集”伽马参数 k",
    "NOTE: high ratios might originate from mapping artifacts that can disturb parameter learning": "注意：高比率可能源于映射工件，这些工件可能会干扰参数学习。",
    "Helpful for poor data, where no clear distinction between 'enriched' and 'non-enriched' is possible": "适用于数据质量较差的情况，其中无法明确区分“富集”和“非富集”",
    "Default: corresponding to singleton read start.": "默认：对应于单例读取起始点。",
    "NOTE: for sites with counts above threshold the whole covered regions will be ignored for learning!": "注意：对于计数超过阈值的位点，整个覆盖区域在学习时将被忽略！",
    "A KDE threshold corresponding to -pet read start counts at one position will be used for initial classification of 'non-enriched' and 'enriched' site": "一个KDE阈值将被用于将一个位置处的-pet读段起始计数用于初始分类为“非富集”和“富集”位点。",
    "We correct by default using B & H.": "我们默认使用B和H进行校正。",
    "Default is 1 (merge adjacent)": "默认值为1（合并相邻的）",
    "If not set all the peaks will be associated to the positive strand. (-x)": "如果没有设置，所有的峰值都将与正链相关联。(-x)",
    "Provide a genomic annotation file in GFF3 format": "提供一份GFF3格式的基因组注释文件",
    "Check if GFF file is unsorted (default: GFF file is assumed to be sorted)": "检查GFF文件是否未排序（默认：假定GFF文件已排序）",
    "Set the sliding window size in nucleotides. If unsure, try 75-100 (default: 50)": "设置滑动窗口大小，单位为核苷酸。如果不确定，可以尝试使用75-100（默认值：50）",
    "Set the sliding window step size (default: 20)": "设置滑动窗口的步长大小（默认：20）",
    "Output annotation BED file used for creating sliding windows": "输出用于创建滑动窗口的注释BED文件",
    "Select BAM file belonging to the CLIP-seq experiment. NOTE that order determines replicate numbering in output tables": "选择属于CLIP-seq实验的BAM文件。注意，顺序决定了输出表中重复编号的顺序。",
    "Select BAM file belonging to the CLIP-seq control. NOTE that order determines replicate numbering in output tables": "选择属于CLIP-seq对照的BAM文件。请注意，顺序决定了输出表格中的重复编号。",
    "Provide a genomic regions BED file for calculating crosslink site overlap counts. Typically this is the sliding windows BED file created with htseq-clip's 'Create sliding windows' procedure": "提供一个基因组区域的BED文件，用于计算交联位点的重叠计数。通常这是使用htseq-clip的“创建滑动窗口”过程创建的滑动窗口BED文件。",
    "Provide a dataset ID (e.g., RNA-binding protein name) used in the generated data table (default: Rbp)": "提供一个在生成的数据表中使用的数据集ID（例如，RNA结合蛋白名称，默认：Rbp）",
    "Select the read mate (1, 2) to extract crosslink sites from. For single-end CLIP-seq data, select 1 (default: 1)": "选择用于提取交联位点的读取配对（1，2）。对于单端CLIP-seq数据，选择1（默认值：1）",
    "Specify crosslink site position in the read, i.e., the genomic position to be extrated (default: middle position)": "指定交联位点在读取中的位置，即要提取的基因组位置（默认：中间位置）",
    "Number of nucleotides to offset for crosslink sites. Can be positive (upstream direction) or negative (downstream direction) (default: 0)": "交联位点的核苷酸偏移数量。可以是正数（上游方向）或负数（下游方向）（默认值：0）",
    "Minimum alignment quality for filtering input BAM files. BAM entries greater than set quality will be filtered out (default: 10)": "输入BAM文件的最小比对质量用于过滤。大于设定质量的BAM条目将被过滤掉（默认值：10）",
    "Check if only primary positions of multimapping reads should be kept": "检查是否只保留多映射读取的主位置",
    "Minimum read length for filtering input BAM files (default: 0)": "过滤输入BAM文件的最小读取长度（默认：0）",
    "Maximum read length for filtering input BAM files (default: 500)": "过滤输入BAM文件时的最大读取长度（默认值：500）",
    "Maximum read interval length for filtering input BAM files (default: 10000)": "过滤输入BAM文件时的最大读取区间长度（默认值：10000）",
    "Extract crosslink sites only from chromosomes given in this file (format: one chromsome ID per file)": "从文件中指定的染色体提取交联位点（格式：每个文件一个染色体ID）",
    "Provide BED file to filter BAM entries based on their overlap with genomic regions inside the provided BED file": "提供BED文件以过滤BAM条目，这些BAM条目与提供的BED文件内的基因组区域重叠。",
    "Specify mode of filtering out BAM entries, with respect to the genomic regions inside the provided BED file)": "指定过滤BAM条目的模式，关于在提供的BED文件内基因组区域的方面",
    "Check if crosslink site position should be counted for overlapping features on both strands": "检查是否应计算在两个链上重叠特征的交联位点位置",
    "Windows annotation table file (output of htseq-clip 'Create sliding windows' Galaxy wrapper)": "Windows 注释表文件（htseq-clip '创建滑动窗口' Galaxy 包装器的输出）",
    "CLIP-seq samples count table file (output of htseq-clip 'Create count table' Galaxy wrapper)": "CLIP-seq样品计数表文件（htseq-clip '创建计数表' Galaxy包装器的输出）",
    "CLIP-seq sample information table file (output of htseq-clip 'Create count table' Galaxy wrapper)": "CLIP-seq样本信息表文件（htseq-clip '创建计数表' Galaxy包装器的输出）",
    "Keep only windows where at least min_sample samples have a crosslink site count > min_count (including control samples) (default: 2)": "只保留至少有min_sample个样本的交联位点计数大于min_count（包括对照样本）的窗口（默认值：2）",
    "Minimum crosslink site count per window per sample (default: 2)": "每个样本每个窗口的最小交叉链接位点计数（默认值：2）",
    "Adjusted p-value threshold for a window to be reported as significant window (default: 0.1)": "调整显著窗口报告的p值阈值（默认值：0.1）",
    "Log2 fold change threshold for a window to be reported as significant window (default: 1.0)": "窗口被认为是显著窗口的对数比变化阈值（默认值：1.0）",
    "Choose yes to adjust p-values for overlapping windows, using Bonferroni family-wise error rate correction on overlapping sliding windows": "选择“是”以调整重叠窗口的p值，使用Bonferroni家族错误率校正方法对重叠的滑动窗口进行校正",
    "Choose yes to disable independent hypothesis weighting (IHW) for multiple testing correction. By default, IHW is used for multiple tesing correction instead of the default method Benjamini Hochberg (BH). The authors recommend using IHW instead of BH for FDR correction": "选择“是”以禁用多重检验校正的独立假设加权（IHW）。默认情况下，多重检验校正使用IHW而不是默认方法Benjamini Hochberg（BH）。作者建议使用IHW而不是BH进行FDR校正。",
    "Choose yes if DEWSeq should not decide on the type of dispersion estimation fit to be used. By default, DEWSeq decides on the dispersion estimation fit type (local or parametric). If this option is enabled, parametric fit will be used instead. By default, DEWSeq fits data using both parametric and local fit types and chooses the best fit of the two (see documentation for details). Typically, this should give better results, but keep in mind that this will also increase the total run time": "如果DEWSeq不应该决定使用的离散度估计拟合类型，请选择“是”。默认情况下，DEWSeq会决定离散度估计拟合类型（局部或参数化）。如果启用此选项，则将使用参数化拟合。默认情况下，DEWSeq使用参数化和局部拟合类型对数据进行拟合，并选择两者中的最佳拟合（有关详细信息，请参阅文档）。通常情况下，这应该会给出更好的结果，但请注意，这也会增加总运行时间。",
    "Choose yes to use likelihood ratio test (LRT) instead of Wald test (see documentation for details). By default, DEWSeq uses Wald test. The authors note that LRT is more accurate than Wald test, but one should keep in mind that LRT is a stringent test in comparison to Wald. So if the RNA-binding protein of interest is a very active binder, it can make sense to enable LRT, otherwise it should be used with caution as one may end up with little or no significant windows at all": "选择“是”以使用似然比检验（LRT）代替Wald检验（详见文档）。默认情况下，DEWSeq使用Wald检验。作者指出，LRT比Wald检验更准确，但需要注意的是，与Wald检验相比，LRT是一个更为严格的检验。因此，如果感兴趣的RNA结合蛋白是一个非常活跃的结合因子，启用LRT是有意义的；否则，应谨慎使用，因为最终可能没有或只有很少显著的窗口。",
    "DEWSeq dataset ID for output HTML report (default: RBP)": "DEWSeq 数据集 ID 用于输出 HTML 报告（默认：RBP）",
    "Output significant regions BED file. Regions can be made up of single significant windows or merged adjacent significant windows (column 7: mean adjusted p-value of region, column 8: mean log2 fold change of region)": "输出显著区域的BED文件。区域可以由单个显著窗口组成，或者合并相邻的显著窗口（第7列：区域的平均校正p值，第8列：区域的平均log2倍数变化）",
    "Output significant windows + regions BED file. This outputs both significant regions and if the region is made up of several windows the single significant windows as well (corresponds to output_bed_file in the DEWSeq R markdown script)": "输出显著窗口+区域的BED文件。这同时输出了显著区域以及如果该区域由几个窗口组成，则输出单个显著窗口（对应于DEWSeq R markdown脚本中的output_bed_file）。",
    "Output HTML report file (corresponds to output_file in the DEWSeq R markdown script)": "输出 HTML 报告文件（对应于 DEWSeq R markdown 脚本中的 output_file）",
    "If not set, it will be estimated from data": "如果未设置，它将根据数据进行估算",
    "The genome will be divided into bins of the specified size. For each bin, the overlapping number of fragments (or reads) will be reported. If only half a fragment overlaps then this fraction will be reported.": "基因组将被划分为指定大小的bin。对于每个bin，将报告重叠的片段（或读取）数量。如果只有半个片段重叠，则将报告这个分数。",
    "The default is fine. Only change it if you know what you are doing.": "默认设置就可以了。除非你知道自己在做什么，否则不要改变它。",
    "The default is to output the log2ratio between the two samples. The reciprocal ratio returns the negative of the inverse of the ratio if the ratio is less than 0. The resulting values are interpreted as negative fold changes. To output the scaled values from the first or second BAM file, select 'first' or 'second'.": "默认情况下，输出两个样本之间的log2比率。倒数比率在比率小于0时返回比率的倒数的负值。这些结果值被解释为负倍数变化。要从第一个或第二个BAM文件输出缩放值，请选择'first'或'second'。",
    "Sometimes it makes sense to exclude certain regions when calculating the scaling factor. For example, if you know of copy number variations between samples then you may want to exclude these. Another typical example is the difference in chromosome X copies between males and females in many species. Example inputs are chrX,chrY,chr3 or chr10:12220-128932": "有时在计算缩放因子时排除某些区域是有意义的。例如，如果您知道样本之间的拷贝数变异，则可能想要排除这些区域。另一个典型的例子是许多物种中雄性和雌性之间X染色体拷贝数的差异。示例输入是chrX、chrY、chr3或chr10:12220-128932。",
    "The genome will be divided into bins of the specified size. For each bin, the overlaping number of fragments (or reads)  will be reported. If only half a fragment overlaps, this fraction will be reported.": "基因组将被划分为指定大小的bin。对于每个bin，将报告重叠的片段（或读取）数量。如果只有半个片段重叠，将报告这个分数。",
    "A list of chromosome names separated by spaces containing those chromosomes that should be excluded during normalization. This is useful when considering samples with unequal coverage across chromosomes, like male and female samples. Example: chrX chrM": "在标准化过程中需要排除的染色体名称列表，这些染色体名称由空格分隔，包含那些应该被排除的染色体。这在考虑染色体上覆盖度不均的样本时非常有用，比如男性和女性样本。示例：chrX chrM",
    "Only the 3 nucleotides at the center of each fragment are counted. The fragment ends are defined by the two mate reads. *NOTE*: Requires paired-end data. By default, only fragments between 130 and 200 bases will be used, though this can be changed with the --minFragmentLength and --maxFragmentLength options.": "只有每个片段中心的3个核苷酸被计数。片段的末端由两个配对的读取定义。*注意*：需要成对末端数据。默认情况下，只有长度在130到200个碱基之间的片段会被使用，尽管这可以通过--minFragmentLength和--maxFragmentLength选项进行更改。",
    "Uses this offset inside of each read as the signal. This is useful in cases like RiboSeq or GROseq, where only the 12th, 15th or 1st base aligned should be used to denote where the signal is (rather than the span of the whole alignment). This can be paired with the --filterRNAstrand option. Note that negative values indicate offsets from the end of each read. A value of 1 indicates the first base of the alignment (taking alignment orientation into account). Likewise, a value of -1 is the last base of the alignment. An offset of 0 is not permitted. If two values (separated by spaces) are specified, then they will be used to specify a range of positions. Note that specifying something like --Offset 5 -1 will result in the 5th through last position being used, which is equivalent to trimming 4 bases from the 5-prime end of alignments.": "使用每个读取中的这个偏移量作为信号。这在 RiboSeq 或 GROseq 等情况下非常有用，因为只有第 12 个、第 15 个或第 1 个碱基对齐应该用来表示信号的位置（而不是整个比对的跨度）。这可以与 --filterRNAstrand 选项配对使用。请注意，负值表示从每个读取的末尾开始的偏移量。值 1 表示比对的第一个碱基（考虑比对的方向）。同样，值 -1 是比对的最后一个碱基。不允许偏移量为 0。如果指定了两个值（由空格分隔），则它们将被用来指定一系列位置。请注意，指定像 --Offset 5 -1 这样的值将导致使用从第 5 个到最后一个位置，这等同于从比对的 5' 端修剪 4 个碱基。",
    "By default (the no option), all reads are processed, regardless of the strand they originated from. For RNAseq, it can be useful to separately create bigWig files for the forward or reverse strands. Note that this tools assumes that a dUTP-based method was used, so fragments will be assigned to the reverse strand if the second read in a pair is reverse complemented.": "默认情况下（没有选项），所有读取都会被处理，无论它们来自哪个链。对于RNAseq，分别创建正向或负向链的bigWig文件可能会很有用。请注意，此工具假定使用了基于dUTP的方法，因此如果一对中的第二个读取是反向互补的，则片段将被分配给负向链。",
    "The default is to use the file name of the sample. The sample labels should be separated by spaces and quoted if a label itself contains a space E.g. label-1 \"label 2\"": "默认情况下，使用样本的文件名作为标签。样本标签应该用空格分隔，如果标签本身包含空格，则应该用引号括起来。例如：label-1 \"label 2\"",
    "Length in bases of the window used to sample the genome. (--binSize)": "用于采样基因组的窗口长度（以碱基为单位）。(--binSize)",
    "To reduce the computation time, not every possible genomic bin is sampled. This option allows you to set the distance between bins actually sampled from. Larger numbers are sufficient for high coverage samples, while smaller values are useful for lower coverage samples. Note that if you specify a value that results in too few (<1000) reads sampled, the value will be decreased. (--distanceBetweenBins)": "为了减少计算时间，并不是每个可能的基因组箱都被采样。这个选项允许您设置实际采样的箱之间的距离。较大的数值适用于高覆盖度样本，而较小的值适用于低覆盖度样本。请注意，如果您指定的值导致采样的读数太少（<1000），该值将被减小。(--distanceBetweenBins)",
    "Write the read/fragment metrics to a tab-separated file, which is primarily useful for more automated downstream processing.": "将读取/片段指标写入制表符分隔的文件，这主要适用于更自动化的下游处理。",
    "Write the read/fragment length distribution underlying the (optional) plot to a file. This can then be processed in R or other downstream applications.": "将（可选的）绘图背后的读长/片段长度分布写入文件。然后，这些数据可以在R或其他下游应用程序中进行处理。",
    "Plot the frequencies on the log10 scale": "在对数10的尺度上绘制频率",
    "Maximum fragment length included in the histogram. A value of 0 (the default) denotes twice the mean fragment length": "直方图中包含的最大片段长度。值为0（默认值）表示平均片段长度的两倍。",
    "The default is to output the log2ratio between the two samples. The reciprocal ratio returns the negative of the inverse of the ratio if the ratio is less than 0. The resulting values are interpreted as negative fold changes. To output the scaled values from the first or second bigWig file, select 'first' or 'second'.": "默认情况下，输出两个样本之间的log2比率。倒数比率在比率小于0时返回比率的倒数的负值。生成的值被解释为负倍数变化。要从第一个或第二个bigWig文件输出缩放值，选择'first'或'second'。",
    "Size of the bins in bases for the output of the bigwig/bedgraph file.": "输出大wig/bedgraph文件的bin大小（以碱基对为单位）。",
    "List of sample numbers (order as in matrix), which are used by --sortUsing for sorting. If no value is set, it uses all samples. Example: 1 3 (space separated!)": "样本编号列表（矩阵中的顺序），由--sortUsing用于排序。如果没有设置值，它将使用所有样本。示例：1 3（以空格分隔！）",
    "If 'Represent missing data as zero' is not set, such cases will be colored in black by default. By using this parameter a different color can be set. A value between 0 and 1 will be used for a gray scale (black is 0). Also color names can be used, see a list here: http://packages.python.org/ete2/reference/reference_svgcolors.html. Alternatively colors can be specified using the #rrggbb notation.": "如果“将缺失数据表示为零”未设置，那么这些情况将默认为黑色。通过使用此参数可以设置不同的颜色。0到1之间的值将用于灰度（黑色为0）。也可以使用颜色名称，请参阅此处的列表：http://packages.python.org/ete2/reference/reference_svgcolors.html。或者可以使用#rrggbb符号表示颜色。",
    "A different colormap can be applied to each column of the heatmap. If you select fewer colormaps than you have samples in your matrix, then the select colormap(s) will be repeated (e.g., if you select one colormap then it will be applied to all samples).": "热图中的每一列可以应用不同的颜色图。如果您选择的颜色图少于矩阵中的样本数，那么所选的颜色图将被重复使用（例如，如果您选择一种颜色图，则它将应用于所有样本）。",
    "The alpha channel (transparency) to use for the heatmaps. The default is 1.0 and values must be between 0 and 1. A value of 0.0 would be fully transparent.": "热图使用的透明度通道（alpha channel）。默认值为1.0，值必须在0和1之间。值为0.0时将完全透明。",
    "The color of the heatmaps can be specified as a list of colors separated by comas with not space in between. For example: white,blue white,green will set a color map from white to blue for the first heatmap and for white to green for the next heatmap.": "热图的颜色可以通过逗号分隔的颜色列表来指定，列表中的颜色之间没有空格。例如：white,blue white,green 将为第一个热图设置从白色到蓝色的颜色映射，而为下一个热图设置从白色到绿色的映射。",
    "The minimum value is 1 and the maximum is 100.": "最小值是1，最大值是100。",
    "The minimum value is 3 and the maximum is 100.": "最小值是3，最大值是100。",
    "The default is to include a summary or profile plot on top of the heatmap and a heatmap colorbar.": "默认情况下，会在热图上方包含一个摘要或轮廓图，以及一个热图颜色条。",
    "Only for scale-regions mode. Label shown in the plot for the start of the region. Default is TSS (transcription start site), but could be changed to anything, e.g. \"peak start\".": "仅用于规模区域模式。在图中显示区域起始处的标签。默认为TSS（转录起始位点），但可以更改为任何内容，例如“峰值起始”。",
    "Only for scale-regions mode. Label shown in the plot for the region end. Default is TES (transcription end site).": "仅用于尺度区域模式。在图中显示的区域结束标签。默认为TES（转录结束位点）。",
    "Label shown in the plot for the reference-point. Default is the same as the reference point selected (e.g. TSS), but could be anything, e.g. \"peak start\" etc.": "在图中显示的参考点标签。默认与所选的参考点相同（例如 TSS），但可以是任何内容，例如“峰值起点”等。",
    "If more than one region is being plotted a list of labels separated by space is required. If a label itself contains a space, then quotes are needed. For example, label_1 \"label 2\"": "如果正在绘制多个区域，则需要一个由空格分隔的标签列表。如果标签本身包含空格，则需要使用引号。例如，label_1 \"label 2\"",
    "The default is to make one plot per bigWig file, i.e., all samples next to each other. Choosing this option will make one plot per group of regions.": "默认情况下，每个bigWig文件会生成一个图表，即所有样本并排显示。选择此选项将为每组区域生成一个图表。",
    "List of sample numbers (order as in matrix), which are used by --kmeans or --hclust for clustering. If no value is set, it uses all samples. Example: 1 3 (space separated!)": "样本编号列表（矩阵中的顺序），由--kmeans或--hclust用于聚类。如果没有设置值，它将使用所有样本。示例：1 3（以空格分隔！）",
    "This should be same file that was used for computeGCbias.": "这应该是用于计算GC偏差的相同文件。",
    "In the bins mode, the coverage is computed for equally sized bins. \\nIn BED file mode, a list of genomic regions in BED or INTERVAL format has to be given. For each region in the BED file, the number of overlapping reads from each BAM file is counted.": "在bins模式下，覆盖度是为等大小的bins计算的。 在BED文件模式下，必须给出一个以BED或INTERVAL格式列出的基因组区域列表。对于BED文件中的每个区域，将计算每个BAM文件中重叠的读取数。",
    "Coverage is computed for the number of reads that overlap such regions.": "覆盖度是计算重叠这些区域的读取数。",
    "Scaling factors calculated as in DESeq2 and made directly compatible with bamCoverage.": "缩放因子计算方式与DESeq2相同，并且与bamCoverage直接兼容。",
    "The principal components to plot. If specified, you must provide two different integers, greater than zero, separated by a space. An example (and the default) is '1 2'.": "要绘制的主成分。如果指定，您必须提供两个不同的整数，大于零，由空格分隔。一个例子（也是默认值）是'1 2'。",
    "Use only the top N most variable rows in the original matrix. Specifying 0 will result in all rows being used. If the matrix is to be transposed, rows with 0 variance are always excluded, even if a values of 0 is specified. The default is 1000.": "只使用原始矩阵中变异性最高的N行。指定0将使用所有行。如果矩阵要转置，即使指定了0的值，也会排除方差为0的行。默认值是1000。",
    "log2 transform the datapoints prior to computing the PCA. Note that 0.01 is added to all values to prevent 0 values from becoming -infinity. Using this option with input that contains negative values will result in an error.": "在计算PCA之前，对数据点进行log2转换。请注意，所有值都加上了0.01以防止0值变成负无穷大。如果使用包含负值的输入使用此选项将导致错误。",
    "Perform the PCA on the transpose of the matrix, (i.e., with samples as rows and features/genes as columns). This then matches what is typically done in R for RNAseq data.": "对矩阵进行转置后进行主成分分析（PCA），即以样本为行，特征/基因为列。这样就与R中通常对RNA测序数据进行的操作相匹配。",
    "When specified, each row (bin, gene, etc.) in the matrix is centered at 0 before the PCA is computed. This is useful only if you have a strong bin/gene/etc. correlation and the resulting principal component has samples stacked vertically. This option is not applicable if the PCA is performed on the transposed matrix.": "当指定时，在计算PCA之前，矩阵中的每行（bin、基因等）都以0为中心。这只有在您有强烈的bin/基因等之间的相关性，并且结果的主成分将样本垂直堆叠时才有用。如果PCA是在转置矩阵上执行的，则此选项不适用。",
    "A list of colors for the symbols. Color names and html hex string (e.g., #eeff22) are accepted. The color names should be space separated. For example, --colors 'red blue green'. If not specified, the symbols will be given automatic colors.": "符号的颜色列表。可以接受颜色名称和html十六进制字符串（例如，#eeff22）。颜色名称应该是以空格分隔的。例如，--colors 'red blue green'。如果未指定，则符号将被赋予自动颜色。",
    "A list of markers for the symbols. (e.g., '<','>','o') are accepted. The marker values should be space separated. For example, 's' 'o' 's' 'o'. If not specified, the symbols will be given automatic shapes.": "标记符号的列表。（例如，'<','>','o'）是可以接受的。标记值应该是空格分隔的。例如，'s' 'o' 's' 'o'。如果没有指定，符号将被赋予自动形状。",
    "File, in BED format, containing the regions to plot.": "包含要绘制区域的文件，格式为BED。",
    "In the scale-regions mode, all regions in the BED file are stretched or shrunken to the same length (in bases) that is indicated by the user. Reference-point refers to a position within the BED regions (start or end of each region). In the reference-point mode only those genomic positions before (upstream) and/or after (downstream) the reference point will be considered.": "在尺度区域模式下，BED文件中的所有区域都将被拉伸或缩小到用户指定的相同长度（以碱基对为单位）。参考点指的是BED区域内部的一个位置（每个区域的起始点或结束点）。在参考点模式下，只有参考点之前（上游）和/或之后（下游）的基因组位置会被考虑。",
    "If the regions are genes, this would be the distance upstream of the transcription start site.": "如果这些区域是基因，这将是转录起始位点上游的距离。",
    "If the regions are genes, this would be the distance downstream of the transcription end site.": "如果这些区域是基因，这将是转录终止位点下游的距离。",
    "By default, each region is scaled to a given length (see the --regionBodyLength option). In some cases it is useful to look at unscaled signals around region boundaries, so this setting specifies the number of unscaled bases on the 5-prime end of each boundary.": "默认情况下，每个区域都被缩放到给定的长度（参见--regionBodyLength选项）。在某些情况下，查看区域边界周围未缩放的信号是有用的，因此此设置指定了每个边界5'端未缩放的碱基数量。",
    "As with --unscaled5prime, but for the 3-prime end.": "与--unscaled5prime相同，但用于3'端。",
    "This is useful to visualize the region end when not using the scale-regions mode and when the reference-point is set to the TSS. (--nanAfterEnd)": "这在不使用缩放区域模式并且参考点设置为转录起始位点(TSS)时，对于可视化区域末端非常有用。(--nanAfterEnd)",
    "If the regions are genes, this would be the distance upstream of the transcription start site. (--beforeRegionStartLength)": "如果这些区域是基因，这将是转录起始位点上游的距离。(--beforeRegionStartLength)",
    "If the regions are genes, this would be the distance downstream of the transcription end site. (--afterRegionStartLength)": "如果这些区域是基因，这将是转录结束位点下游的距离。(--afterRegionStartLength)",
    "The value is computed for each bin. (--averageTypeBins)": "该值是为每个区间计算的。(--averageTypeBins)",
    "If set to 'yes', missing values (NAs) are converted to 0. If you want to use clustering with plotHeatmap or plotProfile, set this to 'yes'. The default is to ignore missing values, which will be depicted as black areas once a heatmap is created. (--missingDataAsZero)": "如果设置为'yes'，缺失值（NAs）将被转换为0。如果您想使用plotHeatmap或plotProfile进行聚类，请将其设置为'yes'。默认情况下，忽略缺失值，一旦创建了热图，这些值将被描绘为黑色区域。（--missingDataAsZero）",
    "Any region containing a value that is equal or less than this numeric value will be skipped. This is useful to skip, for example, genes where the read count is zero for any of the bins which could be the result of unmappable areas and can bias the overall results. (--minThreshold)": "任何包含等于或小于这个数值的区域都将被跳过。这在跳过例如基因的读数计数为零的任何bin中非常有用，这可能是不可映射区域的结果，并且可能会偏倚整体结果。（--minThreshold）",
    "Any region containing a value that is equal or higher that this numeric value will be skipped. The max threshold is useful to skip those few regions with very high read counts (e.g. major satellites) that may bias the average values. (--maxThreshold)": "任何包含等于或高于此数值的区域都将被跳过。最大阈值对于跳过那些具有非常高的读数计数（例如主要的卫星）的少数区域非常有用，这些区域可能会对平均值产生偏差。（--maxThreshold）",
    "If set, all values are multiplied by this number. (--scale)": "如果设置，所有值都将乘以这个数字。（--scale）",
    "To plot the reads per GC over a region, the size of the region is required (see below for more details about the method). By default, the bin size is set to 300 bases, which is close to the standard fragment size of many sequencing applications. However, if the depth of sequencing is low, a larger bin size will be required, otherwise many bins will not overlap with any read. (--regionSize)": "要绘制一个区域内每GC含量的读段数，需要指定区域的大小（有关该方法的更多详细信息，请参见下文）。默认情况下，将箱大小设置为300个碱基，这接近许多测序应用的标准片段大小。然而，如果测序深度较低，则需要更大的箱大小，否则许多箱将不会与任何读段重叠。（--regionSize）",
    "If given, a diagnostic image summarizing the GC bias found on the sample will be created. (--plotFileFormat)": "如果提供了，将创建一个诊断图像，总结在样本中发现的GC偏差。（--plotFileFormat）",
    "In the bins mode, the correlation is computed using equally sized bins. In the BED file mode, a list of genomic regions in BED format has to be given. For each region in the BED file, the number of overlapping reads is counted in each of the BigWig files. Then the correlation is computed.": "在分箱模式下，使用等大小的箱体来计算相关性。在BED文件模式下，需要提供一个以BED格式列出的基因组区域列表。对于BED文件中的每个区域，在每个BigWig文件中计算重叠的读取数量。然后计算相关性。",
    "Length in bases for a window used to sample the genome. (--binSize)": "用于采样基因组的窗口长度（以碱基为单位）。（--binSize）",
    "Correlation is computed for the number of reads that overlap such regions.": "计算重叠这些区域的读取数的相关性。",
    "Label shown in the plot for the start of the region. Default is TSS (transcription start site), but could be changed to anything, e.g. \"peak start\".": "在区域起始处的图中显示的标签。默认为TSS（转录起始位点），但可以更改为任何内容，例如“峰值起始”。",
    "Label shown in the plot for the region end. Default is TES (transcription end site).": "在图中显示的区域结束标签。默认为TES（转录结束位点）。",
    "Height in cm. The default for the plot height is 5 centimeters. The minimum value is 3 cm.": "身高，以厘米为单位。绘图高度的默认值为5厘米。最小值为3厘米。",
    "Width in cm. The default value is 11 centimeters. The minimum value is 1 cm.": "宽度以厘米为单位。默认值为11厘米。最小值为1厘米。",
    "The \"lines\" option will plot the profile line based on the average type selected. The \"fill\" option fills the region between zero and the profile curve. The filled in color is semi-transparent. The \"add standard error\" option colors the region between the profile and the standard error of the data. As in the case of fill, a semi-transparent color is used. The option \"overlapped_lines\" plots each region's value, one on top of the other.": "\"lines\"选项将根据所选的平均类型绘制轮廓线。\"fill\"选项填充从零到轮廓曲线之间的区域。填充的颜色是半透明的。\"add standard error\"选项将轮廓和数据的标准误差之间的区域着色。与填充的情况一样，使用半透明颜色。\"overlapped_lines\"选项将每个区域的值重叠绘制，一个接一个。",
    "The default is to use the file name of the sample. The sample labels should be separated by spaces and quoted if a label itself contains a space, e.g., label-1 \"label 2\"": "默认情况下，使用样本的文件名。样本标签应该用空格分隔，如果标签本身包含空格，则需要用引号括起来，例如，label-1 \"label 2\"",
    "Color names and html hex strings (e.g. #eeff22) are accepted. The color names should be separated by spaces. (--colors red blue green)": "颜色名称和HTML十六进制字符串（例如 #eeff22）都是可以接受的。颜色名称应该用空格分隔。（--colors red blue green）",
    "This option will create an additional output file.": "这个选项会创建一个额外的输出文件。",
    "X axis minimum value. Not specifying this scales these such that the full range of dots is displayed.": "X轴的最小值。不指定此值将使点的完整范围显示出来。",
    "X axis maximum value. Not specifying this scales these such that the full range of dots is displayed.": "X轴最大值。不指定此值将使点的完整范围显示。",
    "Y axis minimum value. Not specifying this scales these such that the full range of dots is displayed.": "Y轴最小值。不指定此值会将这些值缩放，以便显示点的完整范围。",
    "Y axis maximum value. Not specifying this scales these such that the full range of dots is displayed.": "Y轴最大值。不指定此值，则会将这些值缩放，以便显示点的完整范围。",
    "Plot the natural log of the scatter plot after adding 1. Note that this is ONLY for plotting, the correlation is unaffected.": "绘制散点图的自然对数，加1后进行绘图。请注意，这只是用于绘图，相关性不受影响。",
    "If set, bins with very large counts are removed. Bins with abnormally high reads counts artificially increase pearson correlation; that's why, by default, plotCorrelation tries to remove outliers using the median absolute deviation (MAD) method applying a threshold of 200 to only consider extremely large deviations from the median. ENCODE blacklist page (https://sites. google.com/site/anshulkundaje/projects/blacklists) contains useful information about regions with unusually high counts.": "如果设置，将移除计数非常大的区间。计数异常高的区间会人为地增加皮尔逊相关性；因此，默认情况下，plotCorrelation尝试使用绝对中位偏差（MAD）方法移除异常值，应用阈值200仅考虑与中位数的极大偏差。ENCODE黑名单页面（https://sites.google.com/site/anshulkundaje/projects/blacklists）包含有关计数异常高的区域的有用信息。",
    "Limits the coverage analysis to the regions specified in these files. This overrides --numberOfSamples. It is inadvisable to combine this with saving the raw counts.": "将覆盖度分析限制在这些文件中指定的区域。这将覆盖 --numberOfSamples 参数。不建议将其与保存原始计数结合使用。",
    "Number of samples taken from the genome to compute the scaling factors.": "从基因组中取样的数量，用于计算缩放因子。",
    "Length in bases for a window used to sample the genome.": "用于采样基因组的窗口长度（以碱基对为单位）。",
    "The file will have one row per input BAM file and columns containing a variety of QC metrics. For the full description of all metrics and their interpretation, please see our online documentation: http://deeptools.readthedocs.io/en/latest/content/feature/plotFingerprint_QC_metrics.html. (--outQualityMetrics)": "该文件将针对每个输入的BAM文件有一行数据，并且列中包含各种QC指标。有关所有指标的完整描述及其解释，请参见我们的在线文档：http://deeptools.readthedocs.io/en/latest/content/feature/plotFingerprint_QC_metrics.html。(--outQualityMetrics)",
    "If output metrics should be saved, then use this sample as the reference for computing the Jensen-Shannon distance and CHANCE metrics of all other samples. Normally this is an input sample. If you don't specify this, then these metrics will not be computed. This is only applicable if quality metrics are being saved. (--JSDsample)": "如果需要保存输出指标，则使用这个样本作为计算所有其他样本的Jensen-Shannon距离和CHANCE指标的参考。通常这是一个输入样本。如果您不指定这个，那么这些指标将不会被计算。这只适用于正在保存质量指标的情况。（--JSDsample）",
    "If blank, the labels are left unchanged, otherwise they're changed to be those provided here. Multiple labels should be separated by spaces. If you provide labels, you must provide exactly one label for each group of regions.": "如果留空，标签将保持不变，否则它们将更改为此处提供的标签。多个标签应由空格分隔。如果您提供标签，则必须为每组区域提供完全一个标签。",
    "If blank, the labels are left unchanged, otherwise they're changed to be those provided here. Multiple labels should be separated by spaces. If you provide labels, you must provide exactly one label for each sample.": "如果留空，标签将保持不变，否则它们将被更改为这里提供的标签。多个标签应该用空格分隔。如果您提供了标签，您必须为每个样本提供一个标签。",
    "If blank, all region groups are output. Alternatively, one or more groups listed in the output of the 'info' command can be used (separate groups by a space and enclose in quotes if needed)": "如果留空，则输出所有区域组。或者，可以使用“info”命令输出中列出的一个或多个组（如果需要，用空格分隔组并在周围加上引号）",
    "If blank, all samples are output. Alternatively, one or more samples listed in the output of the 'info' command can be used (separate sample names by a space and enclose in quotes if needed)": "如果为空，将输出所有样本。或者，可以使用'info'命令输出中列出的一个或多个样本（如果需要，用空格分隔样本名称并用引号括起来）",
    "Only regions on the desired strand will be included in the output. If you desire to instead remove regions on only one strand (or the . strand), separately run this on the other strand and then use the rbind operation.": "只有目标链上的区域会被包含在输出中。如果你希望去除一个链上的区域（或点链），则需要在另一个链上单独运行此操作，然后使用rbind操作。",
    "Exclude any row in the matrix with a value below this. By default there is no minimum value.": "排除矩阵中任何低于此值的行。默认情况下没有最小值。",
    "Exclude any row in the matrix with a value above this. By default there is no maximum value.": "排除矩阵中任何一列的值超过这个值。默认情况下没有最大值。",
    "Instead of producing BAM files, write output in BEDPE format (as defined by MACS2). Note that only reads/fragments passing filtering criterion are written in BEDPE format.": "不生成BAM文件，而是将输出以BEDPE格式（如MACS2所定义）写入。请注意，只有通过过滤标准的读段/片段才会以BEDPE格式写入。",
    "Shift the left and right end of a fragment. A positive value shift an end to the right (on the + strand) and a negative value shifts a fragment to the left. Either 2 or 4 integers can be provided. For example, '2 -3' will shift the left-most fragment end two bases to the right and the right-most end 3 bases to the left. If 4 integers are provided, then the first and last two refer to fragments whose read 1 is on the left or right, respectively. Consequently, it is possible to take strand into consideration for strand-specific protocols. Note that only properly paired reads are considered.": "将片段的左右两端进行移动。正值将一端向右移动（在正链上），负值将片段向左移动。可以提供2个或4个整数。例如，'2 -3'将最左边的片段端点向右移动两个碱基，最右边的端点向左移动3个碱基。如果提供了4个整数，则前两个和后两个分别指代其读1在左侧或右侧的片段。因此，对于特定链的协议，可以考虑链。请注意，只有正确配对的读段才被考虑。",
    "Instead of deriving the feature label from the feature column, use the value of the given attribute key. For example, the gene_biotype. Note that 'None' is used for BED files or entries where the attributeKey is not found.": "而不是从特征列派生特征标签，而是使用给定属性键的值。例如，gene_biotype。请注意，对于BED文件或未找到attributeKey的条目，使用'None'。",
    "Height in cm. The default for the plot height is 20 centimeters. The minimum value is 3 cm.": "身高以厘米为单位。绘图高度的默认值是20厘米。最小值是3厘米。",
    "Width in cm. The default value is 20 centimeters. The minimum value is 1 cm.": "宽度以厘米为单位。默认值为20厘米。最小值为1厘米。",
    "For BED files, this sets the label for all entries in each file. For GTF files this is ignored.": "对于BED文件，这将为每个文件中的所有条目设置标签。对于GTF文件，这将被忽略。",
    "The alpha channel value (between 0 and 1). A value of 0 is transparent. Default: 0.9": "Alpha通道的值（介于0和1之间）。值为0表示完全透明。默认值：0.9",
    "Plot per-sample, rather than per-feature.": "按样本绘图，而不是按特征。",
    "By default, the y-axis always goes from 0 to 100. If this option is selected, then the maximum value will vary with the dataset.": "默认情况下，y轴的范围始终是从0到100。如果选择了这个选项，那么最大值将根据数据集变化。",
    "Only for BED12 files, include columns 10-12 rather than just 2 and 3.": "仅针对 BED12 文件，包括第 10 到 12 列，而不仅仅是第 2 和第 3 列。",
    "To reduce the computation time, not every possible genomic bin is sampled. This option allows you to set the distance between bins actually sampled from. Larger numbers are sufficient for high coverage samples, while smaller values are useful for lower coverage samples. Note that if you specify a value that results in too few (<1000) reads sampled, the value will be decreased.": "为了减少计算时间，并不是每个可能的基因组箱都被采样。这个选项允许您设置实际采样的箱之间的距离。较大的数字适用于高覆盖度样本，而较小的值适用于低覆盖度样本。请注意，如果您指定的值导致采样的读数太少（<1000），该值将被减小。",
    "Input files in BAM or SAM format.": "输入BAM或SAM格式的文件。",
    "'Somatic', 'Germline', 'LOH', 'Wildtype' and 'None' are fixed values that are treated explicitly by cBioportal. In particular, 'Somatic' and 'Germline' are supported by the cBioportal user interface in the Mutations tab, while 'LOH', 'Wildtype' and 'None' will not be loaded. Any other values will cause the variants to be loaded into cBioPortal and will be displayed as text in the Mutations tab.": "\"体细胞\"、\"生殖系\"、\"LOH\"、\"野生型\"和\"无\"是由cBioportal明确处理的固定值。特别是，\"体细胞\"和\"生殖系\"在cBioportal用户界面的突变标签页中得到支持，而\"LOH\"、\"野生型\"和\"无\"将不会被加载。任何其他值将导致变异被加载到cBioPortal中，并在突变标签页中作为文本显示。",
    "Enter a valid SQL result column expression to compute the mutation status from columns of the variants table in the GEMINI database. As one example, the expression ifnull(nullif(ifnull(nullif(ifnull(nullif(somatic_status, 3), 'LOH'), 2), 'Somatic'), 1), 'Germline') assumes that you have a column somatic_status added to the variants table of your database, and will record 'Germline', 'Somatic', or 'LOH' for variants with a value of 1, 2, or 3 in that column, respectively.": "在GEMINI数据库的变异表中，输入一个有效的SQL结果列表达式来计算突变状态。例如，表达式ifnull(nullif(ifnull(nullif(ifnull(nullif(somatic_status, 3), 'LOH'), 2), 'Somatic'), 1), 'Germline')假定你已经在数据库的变异表中添加了一个名为somatic_status的列，并且会为该列中值为1、2或3的变异分别记录'Germline'、'Somatic'或'LOH'。",
    "Formulate your query using SQL syntax.": "使用 SQL 语法制定你的查询。",
    "Only build 37 (aka hg19) of the human genome is supported.": "仅支持人类基因组的第37版（又称hg19）。",
    "GEMINI can parse and use annotations generated with either snpEff (both 'EFF'- and 'ANN'-style annotations are supported) or VEP. You can also load unannotated variants, but most of GEMINI's functionality will not be available or not be very useful without annotations.": "GEMINI 可以解析并使用由 snpEff（支持 'EFF'- 和 'ANN'- 风格的注释）或 VEP 生成的注释。您也可以加载未注释的变体，但如果没有注释，GEMINI 的大多数功能将不可用或不太有用。",
    "This is usually the case, but some published datasets, like some 1000G VCFs, are missing genotype information.": "这通常是情况，但一些已发布的数据集，比如一些1000G VCFs，缺少基因型信息。",
    "The pedigree dataset is optional, but several GEMINI tools require the relationship between samples (i.e., the family structure) and/or the sample phenotype to be defined. The PED format is a simple tabular format (see the tool help below for details). If you choose to not provide sample information now, but later find that you need it for your analysis, you can also add it to an existing GEMINI database by using the GEMINI amend tool.": "这个谱系数据集是可选的，但是一些GEMINI工具需要定义样本之间的关系（即家族结构）和/或样本的表型。PED格式是一个简单的表格格式（详见下面工具的帮助信息）。如果你现在选择不提供样本信息，但后来发现你的分析需要它，你也可以使用GEMINI修正工具将它添加到现有的GEMINI数据库中。",
    "The preselected defaults should be ok for most use cases (feel free to enable CADD scores for non-commercial use). If you are not interested in certain annotations, you can speed up database creation and decrease the resulting database size slightly by not loading them into the database. Note: GERP and CADD scores are optional parts of the annotation source and can only be loaded if available.": "预选的默认设置应该适用于大多数用例（如果非商业用途，可以自由启用CADD分数）。如果您对某些注释不感兴趣，可以通过不将它们加载到数据库中来加快数据库创建速度，并略微减小生成的数据库大小。注意：GERP和CADD分数是注释源的可选部分，只有在可用时才能加载。",
    "Supported versions: 66 to 71. Use versions that match the VEP/snpEff versions of the annotated vcf for correctness. For e.g VEP v2.6 and snpEff v3.1 use Ensembl 68 version of the genomes. (-v)": "支持的版本：66到71。为了确保正确性，请使用与注释的vcf文件的VEP/snpEff版本相匹配的版本。例如，如果使用VEP v2.6和snpEff v3.1，则应使用Ensembl基因组的68版本。（-v）",
    "Specify 0 here for sliding by one full window size": "在这里指定0，表示滑动一个完整的窗口大小",
    "The tool can use the information from a BED or VCF dataset to annotate the database variants.": "该工具可以使用来自BED或VCF数据集的信息来注释数据库中的变体。",
    "The default is to consider VCF-formatted annotations only if a variant in the GEMINI database and a record in the annotation source describe the exact same nucleotide change at the same position in the genome. You can disable this option to make use of any annotation that overlaps with the position of a database variant. This setting is ignored for annotation sources in BED format, for which matching is always based on overlapping positions only.": "默认情况下，只有在GEMINI数据库中的变体和注释源中的记录描述了基因组中相同位置的完全相同的核苷酸变化时，才会考虑VCF格式的注释。您可以禁用此选项以利用与数据库变体位置重叠的任何注释。对于以BED格式的注释源，此设置将被忽略，因为匹配始终仅基于重叠位置。",
    "For an annotation source in BED format, specify the number of the column from which the annotations should be read. For a VCF source, name an INFO field element.": "对于以BED格式的注释源，指定从哪一列读取注释。对于VCF源，命名一个INFO字段元素。",
    "Your selection will determine the data type used to store the new annotations in the database.": "您的选择将决定用于在数据库中存储新注释的数据类型。",
    "Note: If indicated (in parentheses) an option is only applicable to annotations of a specific type.": "注意：如果指明（在括号内），一个选项只适用于特定类型的注释。",
    "Makes it possible to delete values. Otherwise the old ones are kept": "使得可以删除值。否则旧的值会被保留",
    "By default, a variant has to pass a minimum number of genotype filters (set below) to get reported. By making a filter required, you ensure that variants that fail this one filter are always excluded. Required filters that a variant passes do not count towards its number of passed (regular) filters": "默认情况下，一个变体必须通过最少数量的基因型过滤器（下面设置）才能被报告。通过使一个过滤器成为必需的，你确保了未通过这个过滤器的变体总是被排除。必需的过滤器，变体通过它们并不计入其通过的（常规）过滤器的数量。",
    "Specify how case and control samples for the C-alpha association test should be selected": "指定如何为C-alpha关联测试选择病例和对照样本",
    "Restricting the calculation to coding/noncoding regions will only produce meaningful results with preannotated variants. If you haven't annotated your variants with SnpEff or VEP before loading them into GEMINI, select All SNPs.": "将计算限制在编码/非编码区域只有在使用预先注释的变体时才会产生有意义的结果。如果您在将变体加载到GEMINI之前没有使用SnpEff或VEP对它们进行注释，请选择所有SNP。",
    "If you select All variants the genotype counts will be produced using --summarize with the wildcard query \"select * from variants\".": "如果您选择“所有变体”，将使用通配符查询“select * from variants”和--summarize参数来生成基因型计数。",
    "Enter valid constraints for the WHERE clause of a GEMINI query here. You could use, for example: chrom = 'chr1' or impact_severity = 'HIGH', to include only high-impact variants on chromosome 1 in the counts table.": "在此处输入GEMINI查询的WHERE子句的有效约束。例如，您可以使用：chrom = 'chr1' 或 impact_severity = 'HIGH'，以便在计数表中仅包括染色体1上的高影响变体。",
    "default: -1 (not set) (--min_qual)": "默认：-1（未设置）(--min_qual)",
    "default: 25": "默认值：25",
    "default: 20": "默认值：20",
    "default: 100000": "默认值：100000",
    "default: 1": "默认值：1",
    "default: 3": "默认：3",
    "e.g. S120,S450": "例如 S120, S450",
    "default: -1 (not set) (--min-depth)": "默认：-1（未设置）(--min-depth)",
    "default: -1 (not set) (--min-qual)": "默认：-1（未设置）(--min-qual)",
    "default: -1 (not set) (--min-somatic-score)": "默认：-1（未设置）(--最小体细胞得分)",
    "default: -1 (not set) (--max-norm-alt-freq)": "默认值：-1（未设置）(--max-norm-alt-freq)",
    "default: -1 (not set) (--max-norm-alt-count)": "默认值：-1（未设置）(--max-norm-alt-count)",
    "default: -1 (not set) (--min-norm-depth)": "默认：-1（未设置）(--最小-标准化-深度)",
    "default: -1 (not set) (--min-tumor-alt-freq)": "默认：-1（未设置）(--min-tumor-alt-freq)",
    "default: -1 (not set) (--min-tumor-alt-count)": "默认值：-1（未设置）(--最小肿瘤等位基因计数)",
    "default: -1 (not set) (--min-tumor-depth)": "默认值：-1（未设置）(--最小肿瘤深度)",
    "e.g. chrom12 (--chrom)": "例如：染色体12（--染色体）",
    "This tool requires a catalogue of known protein-protein interactions. Such interaction data, obtained from the Human Protein Reference Database (HPRD), is part of GEMINI's own annotation data, but you can choose to provide your own interactions data instead.": "这个工具需要一个已知蛋白质-蛋白质相互作用的目录。这种相互作用数据，来自人类蛋白质参考数据库（HPRD），是GEMINI自己的注释数据的一部分，但你可以选择提供你自己的相互作用数据。",
    "You can provide interaction data as a simple text file with one interaction of the form geneA|geneB (e.g., ZFPM2|GATA4) per line.": "您可以提供一个简单的文本文件，其中包含以基因A|基因B（例如，ZFPM2|GATA4）形式的每个相互作用数据，每行一个。",
    "By default, the tool finds all genes affected by loss-of-function variants in your input, then, for every such gene, reports its interaction partners if they are also affected by any variant. If you specify the name of a gene of interest (e.g. PTPN22) here, you get the affected interaction partners of only this particular gene reported, irrespective of whether your gene of interest itself is affected by any variant or not.": "默认情况下，该工具会找到输入数据中所有受功能丧失变异影响的基因，然后，对于每个这样的基因，如果它们也受到任何变异的影响，报告它们的相互作用伙伴。如果您在这里指定一个感兴趣的基因名称（例如 PTPN22），无论您感兴趣的基因本身是否受到任何变异的影响，您将只得到这个特定基因的受影响的相互作用伙伴的报告。",
    "A value of 1, for example, means: report only affected direct interaction partners. A value of 0 restricts the report to just variants in the query gene itself.": "例如，值为1表示：只报告受影响的直接相互作用伙伴。值为0将报告限制为仅查询基因本身的变体。",
    "When turned on, the identification of compound heterozygous variant pairs gets based on the family tree only, i.e., the tool looks for heterozygous allele pairs in any kid that weren't occuring together in the parents (see the tool help below for the exact criteria used to detect compound heterozygosity).": "当启用时，复合杂合变异对的鉴定仅基于家谱进行，即，该工具在任何孩子中寻找未在父母中共同出现的杂合等位基因对（有关用于检测复合杂合性的确切标准，请参见下面的工具帮助）。",
    "When turned on, runs the tool with --max-priority 3 instead of the default value 1. This leads to inclusion of more ambiguous cases of compound heterozygosity.": "当开启时，使用 --max-priority 3 而不是默认值 1 来运行工具。这将导致包含更多复合杂合性的模糊情况。",
    "When turned on, the identification of candidate variants gets based on the observed inheritance pattern only. The default is to report candidates only if there is evidence for them being phenotypically relevant, i.e., if they are observed in an affected sample.": "当启用时，候选变异的鉴定仅基于观察到的遗传模式。默认情况下，只有在有证据表明它们具有表型相关性时才报告候选变异，即，如果它们在受影响的样本中被观察到。",
    "Leave empty for an analysis including all families": "留空以包含所有家族的分析",
    "All samples from a family must have a sequencing depth of at least this value at a given variant site in order for the family to be included in the analysis of this particular variant. default: 0 (do not apply this filter)": "所有来自一个家系的样本必须在给定的变异位点上至少有这个值的测序深度，才能将该家系包含在对这个特定变异的分析中。默认值：0（不应用此过滤器）",
    "The genotypes of all samples from a family must be called with at least this quality at a given variant site in order for the family to be included in the analysis of this particular variant. default: 0 (do not apply this filter)": "所有样本的基因型在某个变异位点上必须至少被以这个质量评分调用，才能将这个家族包含在这个特定变异的分析中。默认值：0（不应用此过滤器）",
    "The genotypes at a given variant site of all samples from a family must not have a higher (phred-scaled) likelihood to be wrong than this value in order for the family to be included in the analysis of this particular variant. default: -1 (do not apply this filter); if used the GEMINI documentation suggests 10 as a reasonable value": "在分析某个特定变异时，来自一个家族的所有样本在该变异位点的基因型不能有比这个值更高的（phred-scaled）错误可能性，以便将该家族包含在该特定变异的分析中。默认值：-1（不应用此过滤器）；如果使用，GEMINI文档建议10作为一个合理的值。",
    "The tool reports key information about the inheritance pattern detection for each candidate variant found. It can precede each such row with additional columns, listing information about the variant taken from the variants table of the GEMINI database. Here, you can control which subset of the variants table columns should be added to the output.": "该工具报告了每个候选变异体发现的遗传模式检测的关键信息。它可以在每一行之前增加额外的列，列出来自GEMINI数据库变异体表的有关变异体的信息。在这里，你可以控制哪些子集的变异体表列应该被添加到输出中。",
    "In FASTA format": "在FASTA格式中",
    "Checktrans assumes that all ORFs end in a STOP codon. Forcing the sequence to end with an asterisk, if there is not one there already, makes checktrans treat the end as a potential ORF. If an asterisk is added, it is not included in the reported count of STOPs": "Checktrans 假定所有开放阅读框（ORF）都以终止密码子结束。如果序列中没有星号（*），则强制序列以星号结束，这使得 checktrans 将末端视为潜在的 ORF。如果添加了星号，它将不包括在报告的终止密码子计数中。",
    "In their paper, GOR mention that if you know something about the secondary structure content of the protein you are analyzing, you can do better in prediction. 'idc' is an index into a set of arrays, dharr[] and dsarr[], which provide 'decision constants' (dch, dcs), which are offsets that are applied to the weights for the helix and sheet (extend) terms. So, idc=0 says don't use the decision constant offsets, and idc=1 to 6 indicates that various combinations of dch,dcs offsets should be used": "在他们的论文中，GOR提到，如果你了解你正在分析的蛋白质的二级结构内容，你可以在预测中做得更好。'idc'是一组数组的索引，dharr[]和dsarr[]，它们提供“决策常数”（dch，dcs），这些是应用于螺旋和层（扩展）项权重的偏移量。所以，idc=0表示不要使用决策常数偏移，而idc=1到6表示应使用各种dch，dcs偏移的组合。",
    "Enter a number <1.0 or >1.0 to decrease or increase the size, respectively": "输入一个小于1.0或大于1.0的数字，分别减小或增大大小",
    "This is the region left to the groups where the names of the groups are displayed. Enter a number <1.0 or >1.0 to decrease or increase the size, respectively": "这是显示组名的组左侧的区域。输入小于1.0或大于1.0的数字，可以分别减小或增加该区域的大小。",
    "These are only done if the sequence is nucleic": "这些仅在序列是核酸时才进行",
    "If you want to count frequencies in a 40 aa stretch you should enter 40 here": "如果你想在一个40个氨基酸的区间内计算频率，你应该在这里输入40",
    "The amino acid code B represents Asparagine or Aspartic acid and the code Z represents Glutamine or Glutamic acid. These are not commonly used codes and you may wish not to count words containing them, just noting them in the count of 'Other' words": "氨基酸代码 B 代表天冬酰胺或天冬氨酸，代码 Z 代表谷氨酰胺或谷氨酸。这些不是常用的代码，你可能不希望计算包含它们的单词，只是将它们记录在“其他”单词的计数中。",
    "If this is true then the residues ACFGILMVWY are marked as squares and all other residues are unmarked. This overrides any other markup that you may have specified": "如果这是真的，那么残基 ACFGILMVWY 被标记为方块，所有其他残基则没有标记。这会覆盖您可能已指定的任何其他标记",
    "If this is left blank, then the sequence case is left alone. A set of regions is specified by a set of pairs of positions. The positions are integers. They are separated by any non-alpha-numeric character. Three examples of region specifications are: 1) 24-45, 56-78 2) 1:45, 67=99;765..888 3) 1,5,8,10,23,45,57,99": "如果这里留空，那么序列的大小写就不会改变。一组区域由一组位置对指定。位置是整数。它们由任何非字母数字字符分隔。区域规范的三个示例是：1）24-45，56-78 2）1:45，67=99；765..888 3）1,5,8,10,23,45,57,99",
    "comma separated": "逗号分隔",
    "Plain text file with one ID per line, optionally with space separated range, strand, and algorithm.": "每行一个ID的纯文本文件，可选地包含以空格分隔的范围、链和算法。",
    "Comma or new line separated list": "逗号或新行分隔的列表",
    "Use zero for no cutoff": "使用零作为没有截止值",
    "Only relevant for dc-megablast, and otherwise ignored. Default window size changes with substitution matrix and BLAST type. Entering a non-negative integer will override the default.": "仅对dc-megablast有效，否则将被忽略。默认窗口大小会根据替代矩阵和BLAST类型而变化。输入一个非负整数将覆盖默认值。",
    "See help text for default parameter values for each BLAST type.": "查看每种BLAST类型的默认参数值的帮助文本。",
    "Free text to describe the options used to create the masking files": "自由文本描述用于创建掩蔽文件的选项",
    "How close masked intervals should be to get merged together": "掩蔽区间需要多接近才能合并在一起",
    "One or more FASTA files": "一个或多个FASTA文件",
    "This is the database name shown in BLAST search output": "这是在BLAST搜索输出中显示的数据库名称",
    "This is only advised if your FASTA file follows the NCBI naming conventions using pipe '|' symbols": "这只有在您的FASTA文件遵循NCBI命名约定，使用管道符'|'的情况下才建议使用。",
    "These hash values can then be used to quickly determine if a given sequence data exists in this BLAST database.": "这些哈希值随后可以用来快速确定给定的序列数据是否存在于此BLAST数据库中。",
    "As produced by NCBI masking applications (e.g. dustmasker, segmasker, windowmasker)": "由NCBI掩蔽应用程序产生（例如：dustmasker、segmasker、windowmasker）",
    "Integer >=0, e.g. 9606 for Homo sapiens": "整数 >=0，例如 9606 代表智人（Homo sapiens）",
    "One or NCBI PSSM ASN.1 format scoremat files (often named *.smp)": "一个或NCBI PSSM ASN.1格式的得分矩阵文件（通常命名为*.smp）",
    "Exclude domains that do not pass validation test": "排除未通过验证测试的域",
    "Exclude domains with with maximum number of independent observations below this threshold": "排除具有独立观察值最大数量低于此阈值的域",
    "(blastp/blastx)": "(blastp/blastx)",
    "By default this feature is disabled for sequences of length below 30, set to 20 for sequences of length below 100, and set to 40 otherwise. Setting this option to 1 will disable this feature": "默认情况下，对于长度小于30的序列，此功能是禁用的；对于长度小于100的序列，设置为20；否则设置为40。将此选项设置为1将禁用此功能。",
    "This feature is designed for long query DNA sequences that may span several genes. In these cases, the default of reporting the 25 best overall hits could cause hits to a lower scoring gene to be overshadowed. But just increasing the number of alignments reported will bloat the output size and reduce performance. Using this feature along with -k 25 (default), a hit will only be deleted if at least 50% of its query range is spanned by at least 25 higher or equal scoring hits. Using this feature along with --top 10, a hit will only be deleted if its score is more than 10% lower than that of a higher scoring hit over at least 50% of its query range. The percentage is configurable using --range-cover. Note that this feature is currently only available in frameshift alignment mode": "这个特性是为可能跨越几个基因的长查询DNA序列设计的。在这些情况下，报告25个最佳整体命中的默认设置可能会导致对评分较低的基因的命中被掩盖。但是，仅仅增加报告的比对数量将使输出大小膨胀并降低性能。使用这个特性以及-k 25（默认值），只有在至少有25个更高或相等评分的命中覆盖了其查询范围的至少50%时，命中才会被删除。使用这个特性以及--top 10，只有在其评分比覆盖其查询范围至少50%的更高评分命中的评分低10%以上时，命中才会被删除。可以使用--range-cover配置百分比。请注意，这个特性目前只在移码对齐模式下可用。",
    "Values around 15 are reasonable for this parameter. Enabling this feature will have the aligner tolerate missing bases in DNA sequences and is most recommended for long, error-prone sequences like MinION reads. In the pairwise output format, frameshifts will be indicated by \\ and / for a shift by +1 and -1 nucleotide in the direction of translation respectively.": "对于这个参数，大约15的值是合理的。启用此功能将使比对器能够容忍DNA序列中缺失的碱基，并且最推荐用于像MinION读取这样的长且容易出错的序列。在成对输出格式中，移码将以\\和/表示，分别表示在翻译方向上移码+1和-1个核苷酸。",
    "Compositionally biased sequences often cause false positive matches, which are effectively filtered by this algorithm in a way similar to the composition based statistics used by BLAST": "组成偏差序列经常导致假阳性匹配，该算法以类似于BLAST使用的基于组成的统计学方法有效地过滤了这些匹配。",
    "The FASTA sequence identifiers as well as the sequences of query and target need to be identical for a hit to be deleted": "FASTA序列标识符以及查询和目标序列需要完全相同，才能删除命中结果",
    "If your database of interest is not listed, contact your Galaxy admin": "如果您感兴趣的数据库没有列出，请与您的Galaxy管理员联系",
    "Any taxonomic rank can be used, and only reference sequences matching one of the specified taxon ids will be searched against.": "可以使用任何分类等级，并且只有与指定的分类单元ID匹配的参考序列才会被搜索。",
    "Choose one of the sensitivity modes. The default mode is mainly designed for short read alignment, i.e. finding significant matches of >50 bits on 30-40aa fragments. The sensitive mode is a lot more sensitive than the default and generally recommended for aligning longer sequences. The more sensitive mode provides even more sensitivity. More sensitivity may increase computation time.": "选择一个敏感性模式。默认模式主要设计用于短读段比对，即在30-40个氨基酸片段上找到大于50位的显著匹配。敏感模式比默认模式敏感得多，通常推荐用于比对较长序列。更高敏感性模式提供更高的敏感性。更高的敏感性可能会增加计算时间。",
    "In parentheses are the supported values for (gap open)/(gap extend). In brackets are default gap penalties": "在括号内是支持的（打开间隙）/（延伸间隙）的值。在方括号内是默认的间隙惩罚值",
    "Leave empty for default (see scoring matrix)": "留空以使用默认值（参见评分矩阵）",
    "DIAMOND by default applies the tantan repeat masking algorithm to the query and target sequences as described in (Frith, 2011). This masking procedure increases the specificity of alignments and serves to filter out spurious hits. Note that when using --comp-based-stats (2,3,4), tantan masking is disabled by default.": "DIAMOND 默认对查询和目标序列应用了 tantan 重复掩蔽算法，如 (Frith, 2011) 所述。这种掩蔽程序提高了比对的特异性，并有助于过滤掉假阳性命中。请注意，当使用 --comp-based-stats (2,3,4) 时，tantan 掩蔽默认是禁用的。",
    "he query dataset will first be searched at a lower sensitivity setting, only searching those query sequences at the target sensitivity that fail to produce a significant alignment at a lower sensitivity.": "查询数据集将首先在较低的灵敏度设置下进行搜索，只有在较低灵敏度下未能产生显著比对的查询序列才会在目标灵敏度下进行搜索。",
    "Double-indexed is the main algorithm of the program, designed for large input files but less efficient for small query files. Query-indexed and improves performance for small query files. This mode will be automatically triggered based on the input. Contiguous-seed mode and further improves performance for small query files. The modes differ slightly in their sensitivity, so results are not guaranteed to be 100% identical for different settings of this option.": "双索引是程序的主要算法，设计用于大型输入文件，但对小型查询文件的效率较低。查询索引和提高小型查询文件的性能。这种模式将根据输入自动触发。连续种子模式，进一步提高小型查询文件的性能。这些模式在灵敏度上略有不同，因此不同设置下的结果不能保证100%相同。",
    "Target sequences will be ranked according to their ungapped extension scores at seed hits, and gapped extensions will only be computed for the best N targets for each query. Note that this option increases memory use.": "目标序列将根据它们在种子命中处的未间隙延伸分数进行排名，并且仅针对每个查询的前N个目标计算有间隙的延伸。请注意，此选项会增加内存使用。",
    "Report only alignments above the given percentage of sequence identity": "只报告序列相似性高于给定百分比的比对结果",
    "Report only alignments above the given percentage of query cover": "只报告查询覆盖度高于给定百分比的比对",
    "Report only alignments above the given percentage of subject cover": "只报告覆盖目标序列百分比以上的比对结果",
    "The maximum number of HSPs (High-Scoring Segment Pairs) per target sequence to report for each query. The default policy is to report only the highest-scoring HSP for each target, while disregarding alternative, lower-scoring HSPs that are contained in the same target.": "对于每个查询，报告目标序列中HSPs（高得分片段对）的最大数量。默认策略是只为每个目标报告最高得分的HSP，同时忽略同一目标中包含的替代的、得分较低的HSPs。",
    "This option is incompatible with --sed-cut": "这个选项与--sed-cut不兼容",
    "Enable or disable motif masking": "启用或禁用基序掩蔽",
    "Needs to be supplied in order to provide taxonomy features of the aligner": "需要提供以提供比对器的分类学特征",
    "Path to mapping file that maps NCBI protein accession numbers to taxon ids (gzip compressed). This parameter is optional and needs to be supplied in order to provide taxonomy features. A custom file following the same format may be supplied here. Note that the first line of this file is assumed to contain headings and will be ignored": "映射文件的路径，该文件将NCBI蛋白质访问号码映射到分类单元ID（gzip压缩）。此参数是可选的，需要提供以便提供分类学特征。可以在此提供遵循相同格式的自定义文件。请注意，此文件的第一行假定包含标题，将被忽略。",
    "This parameter is optional and needs to be supplied in order to provide taxonomy features": "这个参数是可选的，需要提供以提供分类学特征",
    "Fasta or fastqsanger, optionally gzipped": "Fasta 或 Fastqsanger 格式，可选 gzip 压缩",
    "Available only for database searches": "仅适用于数据库搜索",
    "Zero value is unlimited": "零值表示无限制",
    "Select No to output all reads to the same file": "选择“否”将所有读取输出到同一个文件",
    "Ignored if no query mate": "忽略，如果没有查询伴侣",
    "FASTA file, one sequence per gene/protein.": "FASTA文件，每个基因/蛋白质一个序列。",
    "Default is 70%, use 0 for no filtering.": "默认是70%，使用0表示不过滤。",
    "Default is 50%, use 0 for no filtering.": "默认是50%，使用0表示不过滤。",
    "i.e. First make the input non-redundant": "即首先使输入非冗余",
    "This is column 1 in standard BLAST tabular output": "这是标准BLAST表格输出的第一列",
    "This is column 2 in standard BLAST tabular output": "这是标准BLAST表格输出中的第二列",
    "This is column 25 in the default extended BLAST tabular output": "这是默认的扩展BLAST表格输出中的第25列",
    "FASTA file with one or more sequences to add to the database": "FASTA文件，包含一个或多个序列，用于添加到数据库中",
    "This tool supports only the cooler file format": "这个工具仅支持cooler文件格式",
    "Please use the special BAM datatype: qname_input_sorted.bam and use for 'bowtie2' the '--reorder' option to create a BAM file.": "请使用特殊的BAM数据类型：qname_input_sorted.bam，并使用'bowtie2'的'--reorder'选项来创建一个BAM文件。",
    "Number of lines to consider for the qc test run.": "用于qc测试运行考虑的行数。",
    "The format is chr:referencePoint or chr:regionStart-regionEnd.": "格式为 chr:参考点 或 chr:区域起始-区域结束。",
    "Distance which should be considered as short range. Default 2MB.": "应视为短程的距离。默认为2MB。",
    "Colorlist for the boxplots.": "箱线图的颜色列表。",
    "The protein file to validate the detected loops.": "验证检测到的环的蛋白质文件。",
    "The maximum number of loops that should be used for optimization computation.": "用于优化计算的最大循环次数。",
    "The matrix resolution of the Hi-C interaction matrix.": "Hi-C互作矩阵的分辨率。",
    "The number of iterations hyperopt uses to search for a result": "超参数优化工具hyperopt用来搜索结果的迭代次数",
    "The width of the peak region in bins. The square around the peak will include (2 * peakWidth)^2 bins.": "峰值区域的宽度，以bin为单位。围绕峰值的正方形将包括(2 * peakWidth)^2个bin。",
    "The window size for the neighborhood region the peak is located in. All values from this region (exclude the values from the peak region) are tested against the peak region for significant difference. The square will have the size of (2 * windowSize)^2 bins": "峰值所在邻域区域的窗口大小。测试该区域（排除峰值区域的值）的所有值与峰值区域是否存在显著差异。正方形的大小将是（2 * windowSize）^2个bin。",
    "Only candidates with p-values less the given threshold will be considered as candidates. For each genomic distance a negative binomial distribution is fitted and for each pixel a p-value given by the cumulative density function is given. This does NOT influence the p-value for the neighborhood testing.": "只有p值小于给定阈值的候选者才会被视为候选者。对于每个基因组距离，拟合一个负二项分布，并且对于每个像素，给出了由累积密度函数给出的p值。这并不影响邻域测试的p值。",
    "The minimum number of interactions a detected peaks needs to have to be considered.": "检测到的峰值需要至少具有的相互作用数量才能被考虑。",
    "The minimum number of obs/exp interactions a detected peaks needs to have to be considered": "检测到的峰值需要考虑的最小 obs/exp 交互作用次数",
    "Rejection level for the statistical test for H0. H0 is peak region and background have the same distribution.": "统计检验的拒绝水平，用于假设H0。H0假设峰值区域和背景具有相同的分布。",
    "Maximum genomic distance of a loop, usually loops are within a distance of ~2MB.": "环的最大基因组距离，通常环的距离在大约2MB以内。",
    "Chromosomes to include in the analysis. If not set, all chromosomes are included.": "包含在分析中的染色体。如果没有设置，则包含所有染色体。",
    "The format is chr:start-end.": "格式是 染色体起始-终止。",
    "If set, then the log1p of the matrix values is used. This parameter has no effect for Spearman correlations but changes the output of Pearson correlation and, for the scatter plot, if set, the visualization of the values is easier.": "如果设置，则使用矩阵值的log1p。此参数对Spearman相关性没有影响，但会改变Pearson相关性的输出，并且对于散点图，如果设置，值的可视化更容易。",
    "A PCA vector as a bedgraph file with no header. In case of several matrices with different conditions, ie. control treatment, the PCA of control can be used. Note that only one PCA can be provided.": "一个没有标题的PCA向量作为bedgraph文件。如果存在具有不同条件的多个矩阵，例如对照组和治疗组，可以使用对照组的PCA。请注意，只能提供一个PCA。",
    "The number of to be used quantiles.": "要使用的分位数的数量。",
    "The number of outliers to be removed.": "要移除的异常值的数量。",
    "Accept all samples to significance level alpha.": "接受所有样本到显著性水平α。",
    "Remove all samples with a lower read coverage as this value.": "移除所有读段覆盖度低于该值的样本。",
    "Remove all samples with a lower density as this value.": "删除所有密度低于该值的样本。",
    "To compute the density, consider only this genomic distance around the diagonal.": "为了计算密度，只考虑对角线周围的这段基因组距离。",
    "Please separate the chromosomes by space": "请用空格分隔染色体",
    "Change the default resolution of the plot.": "更改绘图的默认分辨率。",
    "Select if the significant regions should be highlighted in the plot. If not set, the significant file data is used only to update the p-values.": "选择是否在图中高亮显著区域。如果没有设置，显著文件数据仅用于更新p值。",
    "Defines the region upstream of a reference point which should be considered in the analysis.": "定义了在分析中应考虑的参考点上游的区域。",
    "The background file computed by chicViewpointBackgroundModel": "由 chicViewpointBackgroundModel 计算的背景文件",
    "Gene name that should be plotted.": "应绘制的基因名称。",
    "Maximum value of the plotted p-value.": "绘制的P值的最大值。",
    "Minimum value of the plotted p-value.": "绘制的P值的最小值。",
    "Plot x-fold region for the mean background.": "绘制平均背景的x折区域。",
    "Colorlist for the viewpoint lines. For all available colors please check matplotlib documentation: https://matplotlib.org/stable/gallery/color/named_colors.html": "视点线的颜色列表。有关所有可用颜色，请查看matplotlib文档：https://matplotlib.org/stable/gallery/color/named_colors.html",
    "Sets all p-values which are equal to zero to one. This has the effect that the associated positions are not disturbing the presenation of small p-values.": "将所有等于零的p值设置为一。这样做的效果是，相关位置不会干扰小p值的展示。",
    "BED file which stores a list of regions that are used to aggregate the data for the differntial test.": "BED文件，它存储了用于为差异性测试聚合数据的区域列表。",
    "Filter x-fold over background. Used to merge neighboring bins with a broader peak but less significant interactions to one peak with high significance. Used only for pValue option.": "过滤x倍背景。用于将相邻的bin合并为一个具有高显著性的峰，但这些bin的相互作用较不明显，且峰较宽。仅用于pValue选项。",
    "loose p-value threshold value to filter target regions in a first round. Used to merge neighboring bins with a broader peak but less significant interactions to one peak with high significance. Used only for pValue option.": "宽松的p值阈值，用于第一轮筛选目标区域。用于将相邻的具有更广泛峰值但相关性较低的bin合并为一个具有高相关性的峰。仅在pValue选项中使用。",
    "Fixate score of background model starting at distance x. E.g. all values greater 500kb are set to the value of the 500kb bin.": "固定背景模型的得分从距离x开始。例如，所有大于500kb的值都设置为500kb区间的值。",
    "Sets all p-values which are equal to zero to one. This has the effect that the associated positions are not part of the significance decision.": "将所有等于零的p值设置为一。这样做的效果是，相关位置不参与显著性决策。",
    "Bed file contains all reference points which should be used to create viewpoints.": "Bed文件包含所有应使用以创建视点的参考点。",
    "Average the contacts of n bins via a sliding window approach.": "通过滑动窗口方法平均计算n个区间的接触次数。",
    "If set, diagonal counts are not included": "如果设置，对角线计数不包括在内",
    "If more than one Hi-C matrix is given for each chromosome a new plot is made. Otherewise, a single plot with one line per chromosome is created.": "如果每个染色体给出了多个Hi-C矩阵，则会生成一个新的图表。否则，会创建一个单一的图表，每个染色体用一条线表示。",
    "Maximum distance from diagonal to use. In other words, distances up to maxDeph are computed.": "最大距离用于对角线。换句话说，计算的距离最多为maxDeph。",
    "Width and height of the plot (in inches). Default is 6 * number of cols, 4 * number of rows. The maximum number of rows is 4. Example: 6 5.": "绘图的宽度和高度（以英寸为单位）。默认值是6乘以列数，4乘以行数。行数的最大值是4。示例：6 5。",
    "This is useful for example to exclude the Y chromosome or to reduce the analysis to only a few chromosomes.": "这对于例如排除Y染色体或将分析仅限于少数几个染色体非常有用。",
    "Bed file with domains coordinates: instead of evaluating the distance vs. Hi-C counts for intra chromosomal counts, compute it for intra-domains.": "带有域坐标的Bed文件：不要评估染色体内计数的距离与Hi-C计数，而是计算域内的计数。",
    "A bam file containing all data underlying the plots is saved on this file.": "一个包含所有图表底层数据的BAM文件保存在此文件中。",
    "Define the eigenvectors that should be computed as a list. For example 1 2 5 to compute the first, second and fifth eigenvetor.": "定义要计算的特征向量作为列表。例如 1 2 5 来计算第一个、第二个和第五个特征向量。",
    "The loop file from hicDetectLoops. To use files from other sources, please follow 'chr start end chr start end' format.": "来自hicDetectLoops的循环文件。要使用其他来源的文件，请遵循'chr start end chr start end'格式。",
    "The protein peak file or a cool file containing e.g. HiChIP information": "蛋白质峰值文件或包含例如HiChIP信息的cool文件",
    "The to be binned resolution.": "要进行分箱的分辨率。",
    "The loop files from hicDetectLoops. To use files from other sources, please follow \\'chr start end chr start end\\' format and remove any header.": "来自hicDetectLoops的循环文件。若要使用其他来源的文件，请遵循'chr start end chr start end'格式，并删除任何标题。",
    "The lowest resolution of all loop files, i.e. 5kb, 10kb and 25kb, please use 25000.": "所有环状文件的最低分辨率，即5kb、10kb和25kb，请使用25000。",
    "BED file which stores a list of regions that are summed and averaged.": "BED文件存储了一组区域的列表，这些区域被求和并求平均。",
    "If the region contains start and end coordinates, define if the start, center (start + (end-start) / 2) or end bin should be used as start for range.": "如果区域包含起始和结束坐标，定义是使用起始、中心（起始 + （结束-起始）/ 2）还是结束的bin作为范围的起始。",
    "This parameter specifies if the strand information is taken into account for the aggregation. It has the effect that the contacts of a reverse strand region are inverted e.g. [1,2,3] becomes [3,2,1].": "这个参数指定在聚合时是否考虑链信息。它的效果是反转链区域的接触，例如 [1,2,3] 变成 [3,2,1]。",
    "This number should be at least 3 times as large as the bin size of the Hi-C matrix.": "这个数字应该至少是Hi-C矩阵的箱大小的3倍。",
    "This number should around 6-10 times as large as the bin size of the Hi-C matrix.": "这个数字应该大约是Hi-C矩阵的箱大小的6-10倍。",
    "The step size grows exponentially as maxDeph + (step * int(x)**1.5) for x in [0, 1, ...] until it reaches maxDepth. For example, selecting step=10,000, minDepth=20,000 and maxDepth=150,000 will compute TAD-scores for window sizes: 20,000, 30,000, 40,000, 70,000 and 100,000": "步长会随着maxDeph + (step * int(x)**1.5)呈指数增长，其中x取值范围为[0, 1, ...]，直到达到maxDepth。例如，选择step=10,000, minDepth=20,000 和 maxDepth=150,000 将计算窗口大小为：20,000, 30,000, 40,000, 70,000 和 100,000 的TAD分数。",
    "The delta value reduces spurious boundaries that are shallow, which usually occur at the center of large TADs when the TAD-sep. score is flat. Higher delta threshold values produce more conservative boundary estimations. By default, multiple delta thresholds are saved for the following delta values: 0.001, 0.01, 0.03, 0.05, 0.1. Other single or multiple values can be given.": "delta值减少了浅层的假边界，这些边界通常出现在大型TADs的中心，当TAD-sep.分数平坦时。较高的delta阈值值会产生更保守的边界估计。默认情况下，为以下delta值保存了多个delta阈值：0.001、0.01、0.03、0.05、0.1。也可以给出其他单个或多个值。",
    "This parameter can be used to reduce spurious boundaries caused by noise.": "这个参数可以用来减少由噪声引起的错误边界。",
    "Resolutions can be specified seperated with a space, e.g. '10000 20000 50000'": "分辨率可以通过空格分隔来指定，例如 '10000 20000 50000'",
    "Option only for cool input files": "选项仅适用于冷却输入文件",
    "The selected option will be applied on the selected chromosomes.": "所选选项将应用于选定的染色体。",
    "Should contain the regions which should be kept, removed or masked.": "应该包含需要保留、移除或掩蔽的区域。",
    "Restriction sites that are closer that this distance are merged into one. This option only applies if --restrictionCutFile is given.": "如果限制性位点之间的距离小于这个距离，则它们将合并为一个位点。这个选项仅在给出 --restrictionCutFile 参数时适用。",
    "*This is not the average fragment size* but the higher end of the fragment size distribution (obtained using for example Fragment Analyzer) which usually is between 800 to 1500 bp. If this value if not known use the default of 1000. The insert value is used to decide if two mates belong to the same fragment (by checking if they are within this max insert size) and to decide if a mate is too far away from the nearest restriction site.": "*这不是平均片段大小*，而是片段大小分布的高端（例如使用Fragment Analyzer获得），通常在800到1500 bp之间。如果这个值未知，使用默认值1000。插入值用于决定两个配对是否属于同一片段（通过检查它们是否在这个最大插入大小内）以及决定一个配对是否离最近的限制位点太远。",
    "If used, the restriction cut places (if given) are used to only consider reads that are in the vicinity of the resctriction sites. Otherwise all reads in the interval are considered. Use multiple ones to create a mcool file.": "如果使用，限制性切割位点（如果给出）将被用来仅考虑位于限制性位点附近的读取。否则，将考虑间隔内的所有读取。使用多个位点来创建一个mcool文件。",
    "If set, outward facing reads without any restriction fragment (self circles) are kept. They will be counted and shown in the QC plots.": "如果设置，保留没有限制片段（自身循环）的向外读取。它们将被计数并在QC图中显示。",
    "If set, inward facing reads less than 1000 bp apart and having a restriction site in between are kept. Although this reads do not contribute to any distant contact, they are useful to account for bias in the data.": "如果设置，保留相距小于1000个碱基对且之间有限制性位点的内向读取。尽管这些读取不会对任何远距离接触做出贡献，但它们对于解释数据中的偏差是有用的。",
    "Identification of duplicated read pairs is memory consuming. Thus, in case of memory errors this check can be skipped.": "重复读对的识别会消耗大量内存。因此，如果出现内存错误，可以跳过此检查。",
    "File with the chromosome sizes for your genome. A tab-delimited two column layout 'chr_name size' is expected Usually the sizes can be determined from the SAM/BAM input files, however, for cHi-C or scHi-C it can be that at the start or end no data is present. Please consider that this option causes that only reads are considered which are on the listed chromosomes. Use this option to guarantee fixed sizes. An example file is available via UCSC: http://hgdownload.soe.ucsc.edu/goldenPath/dm3/bigZips/dm3.chrom.sizes": "包含您基因组染色体大小的文件。期望的布局是制表符分隔的两列 'chr_name size'。通常，大小可以从SAM/BAM输入文件中确定，然而，对于cHi-C或scHi-C，可能在开始或结束时没有数据。请注意，此选项仅考虑列出的染色体上的读数。使用此选项以保证固定大小。可以通过UCSC获取示例文件：http://hgdownload.soe.ucsc.edu/goldenPath/dm3/bigZips/dm3.chrom.sizes",
    "A bam file containing all valid Hi-C reads can be created using this option. This bam file could be useful to inspect the distribution of valid Hi-C reads pairs or for other downstream analyses, but is not used by any HiCExplorer tool. Computation will be significantly longer if this option is set.": "使用此选项可以创建一个包含所有有效Hi-C读取的bam文件。这个bam文件对于检查有效Hi-C读取对的分布或用于其他下游分析可能很有用，但不会被任何HiCExplorer工具使用。如果设置了此选项，计算时间将显著变长。",
    "The organism genome fasta file to compute the restriction enzyme position.": "用于计算限制性内切酶位置的生物体基因组fasta文件。",
    "Search pattern. For example, for HindIII this pattern is \\\"AAGCTT\\\". Both, forward and reverse strand are searched for a match. The pattern is a regexp and can contain regexp specif syntax (see https://docs.python.org/2/library/re.html). For example the pattern CG..GC will find all occurrence of CG followed by any two bases and then GC.": "搜索模式。例如，对于 HindIII，这个模式是 \"AAGCTT\"。正向和反向链都会搜索匹配。该模式是一个正则表达式，可以包含正则表达式特定的语法（见 https://docs.python.org/2/library/re.html）。例如，模式 CG..GC 将找到所有 CG 后面跟着任意两个碱基，然后是 GC 的出现。",
    "Bed file contains all reference points which should be used to build the background model.": "Bed文件包含了所有应该用来构建背景模型的参考点。",
    "Set this parameter if you want truncate zeros to fight over dispersion.": "如果您想要截断零以对抗离散度，请设置此参数。",
    "Viewpoints with a sparsity less than given are considered of bad quality. If multiple matrices are given, the viewpoint is removed as soon as it is of bad quality in at least one matrix.": "视点的稀疏度小于给定值时，被认为是质量差的。如果给出了多个矩阵，那么只要视点在至少一个矩阵中质量差，就会将其移除。",
    "Cluster file created by scHicCluster, scHicClusterCompartments, scHicClusterMinHash or scHicClusterSVL": "由scHicCluster、scHicClusterCompartments、scHicClusterMinHash或scHicClusterSVL创建的聚类文件",
    "Maximal distance to consider for plotting": "最大绘图考虑距离",
    "Distance for the short range to compute the ratio of sum (short range interactions) / sum (long range interactions)": "短程距离用于计算总和比率（短程相互作用）/总和（长程相互作用）",
    "Distance for the long range to compute the ratio of sum (short range interactions) / sum (long range interactions)": "长程距离用于计算总和（短程相互作用）/总和（长程相互作用）的比率",
    "Number of bins to merge to achieve the desired resolution. For example: To merge a 10kb matrix to 1MB, 10kb * 100 = 1MB i.e. the value needs to be 100.": "为了达到所需的分辨率，需要合并的箱数。例如：要将一个10kb的矩阵合并到1MB，10kb * 100 = 1MB，即该值需要是100。",
    "Set to merge for using a running window of length --numBins. Must be an odd number.": "设置为使用长度为--numBins的运行窗口进行合并。必须是奇数。",
    "Use GSE94489_README.txt file": "使用 GSE94489_README.txt 文件",
    "The mappings from SRR number to sample id as given in the barcode file.": "从条形码文件中给出的SRR编号到样本ID的映射。",
    "Number of lines to buffer in memory, if full, write the data to disk.": "在内存中缓冲的行数，如果满了，将数据写入磁盘。",
    "How many clusters should be computed by kmeans or spectral clustering": "kmeans 或谱聚类应该计算多少个聚类",
    "How many hash functions the minHash algorithm uses.": "MinHash算法使用了多少个哈希函数。",
    "How many nearest neighbors should be computed for the k-nn graph?": "应该为 k-nn 图计算多少个最近邻居？",
    "Chromosome size as tabular file. One chromosome per line": "染色体大小作为表格文件。每行一个染色体",
    "Based on this gene name the output is extracted.": "根据这个基因名称，提取输出结果。",
    "Number of decimal places for float values.": "浮点数值的小数位数。",
    "The TADs domain file computed by hicFindTADs.": "由 hicFindTADs 计算得到的 TADs 域文件。",
    "H0 is considered as \\'two regions are identical.\\' i.e. all regions with a test result of smaller p-value are rejected and considered as differential.": "零假设被认为是“两个区域是相同的”。即所有具有较小p值的测试结果的区域都被拒绝并被认为是差异的。",
    "Font size for the plot": "图表的字体大小",
    "In order to be able to better assess the relationship between TADs, the associated protein file (e.g. CTCF for mammals) can be included. The protein file is required in broadpeak forma": "为了能够更好地评估TADs之间的关系，可以包含相关的蛋白质文件（例如哺乳动物的CTCF）。蛋白质文件需要以broadpeak格式提供。",
    "Optional parameter to adjust the number of protein peaks when adapting the resolution to the domain files. At least minimumNumberOfPeaks of unique peaks must be in a bin to considered. Otherwise the bin is treated like it has no peaks": "可选参数，用于在将分辨率适应到域文件时调整蛋白质峰的数量。一个bin中至少必须有minimumNumberOfPeaks个独特的峰才能被考虑。否则，该bin被视为没有峰。",
    "Determine a value by how much the boundaries of two TADs must at least differ to consider them as two separate TADs": "确定一个值，通过这个值可以判断两个拓扑关联结构域（TADs）的边界至少相差多少才能将它们视为两个独立的TADs。",
    "For the relationship determination, a percentage is required from which area coverage the TADs are related to each other. For example, a relationship should be entered from 5 percent area coverage": "对于关系确定，需要一个百分比，以确定TADs（拓扑相关域）彼此之间的关系。例如，当区域覆盖率达到5%时，应建立关系。",
    "Consider only protein peaks with at least the threshold value": "只考虑至少达到阈值的蛋白质峰",
    "Leniency for protein quality check. Widens peaks of protein file by leniency*resolution": "蛋白质质量检查的宽容度。通过宽容度*分辨率加宽蛋白质文件的峰值。",
    "Set whether genes at the border of the matrix up to set distance will not be used for training and testing": "设置是否不使用矩阵边缘上到设定距离的基因进行训练和测试",
    "Whether features build from matrix list are concatenated and resampled together or resampled separatly per matrix. Not important for random undersampling, but alter for other resampling methods and check if performance increases.": "是否将从矩阵列表构建的特征进行连接和重新采样，还是每个矩阵单独重新采样。对于随机欠采样不重要，但对其他重新采样方法可能会有影响，并检查性能是否提高。",
    "Use Confident Learning with the cleanlab library": "使用 cleanlab 库中的 Confident Learning",
    "How many estimators are added in each training step for the classifier": "在每个训练步骤中，分类器增加了多少估计器",
    "max distance between TADs to be used in calculation": "在计算中使用的TADs之间的最大距离",
    "non-numerical float values in matrix will be replaced by this value": "矩阵中非数值型浮点数将被替换为此值",
    "Tracks can be of different filetypes. E.g BED, HiC-Matrix, BigWig or BedGraph": "轨道可以是不同的文件类型。例如 BED、HiC-Matrix、BigWig 或 BedGraph。",
    "if type is set as lines, then the TAD score lines are drawn instead of the matrix set to lines if a heatmap representing the matrix is not wanted": "如果类型设置为lines，则绘制TAD分数线条，而不是将矩阵设置为lines，如果不想用热图表示矩阵的话",
    "must be space/tab delimitted plain text file": "必须是空格或制表符分隔的纯文本文件",
    "Optional input file containing list of loci to discard": "可选输入文件，包含要丢弃的位点列表",
    "Use SNP genotypes matrix": "使用SNP基因型矩阵",
    "Nlength of an interval between two draws in MCMC": "MCMC中两次抽取之间的间隔长度",
    "only R2 reads": "仅 R2 读取",
    "If not selected, the tool will only produce of collection of fasta files (one per locus) containing reads ready to assemble.": "如果未选择，该工具将只产生一组fasta文件（每个位点一个），包含准备组装的读取。",
    "If set, the catalog will be built from samples listed in this file": "如果设置，目录将从该文件中列出的样本构建",
    "(default: 80% of the k-mer length).": "（默认值：k-mer长度的80%）。",
    "(default: 80% of the k-mers in a read)": "（默认值：读取中 k-mers 的 80%）",
    "Paired end data or reverse reads. If a paired list is provided only the reverse reads are used in tsv2bam. Leave selection empty if you analyse single end data.": "配对末端数据或反向读取。如果提供了配对列表，只有在tsv2bam中使用反向读取。如果你分析单端数据，请将选择留空。",
    "If set, matching will be done only for samples listed in this file": "如果设置，匹配将只针对此文件中列出的样本进行",
    "select single/paired for single end data or to select advanced paired end options, --unpaired: treat reverse reads as if they were forward reads; --ignore-pe-reads: ignore paired-end reads even if present in the input": "选择单端/配对用于单端数据，或者选择高级配对端选项，--unpaired：将反向读取视为正向读取；--ignore-pe-reads：即使输入中存在配对端读取，也忽略它们",
    "used in ustacks": "用于ustacks",
    "used in cstacks; suggested: set to -M": "在cstacks中使用；建议：设置为-M",
    "(for populations; default: 0)": "（对于种群；默认值：0）",
    "(for populations; default: 1)": "（针对种群；默认值：1）",
    "provide the restriction enzyme used": "提供所使用的限制性内切酶",
    "provide the second restriction enzyme used": "提供第二个限制性内切酶",
    "select input file type": "选择输入文件类型",
    "Format is 'SAMPLE1 \\t POP1 \\n SAMPLE2 ...'.": "格式为 '样本1 \\t 群体1 \\n 样本2 ...'。",
    "(default: 10,000 in de novo mode; in reference mode, one chromosome per batch). Increase to speed analysis, uses more memory, decrease to save memory)": "（默认值：在de novo模式下为10,000；在参考模式下，每个批次一个染色体）。增加可以加快分析速度，但会使用更多内存；减少可以节省内存。",
    "unshared SNPs will be pruned to reduce haplotype-wise missing data.": "未共享的SNPs将被修剪以减少单倍型缺失数据。",
    "specify a minimum minor allele frequency required to process a nucleotide site at a locus": "指定一个最小次等位基因频率，以处理一个位点上的核苷酸位点",
    "specify a minimum minor allele count required to process a nucleotide site at a locus": "指定一个最小次要等位基因计数，以处理一个位点上的核苷酸位点",
    "required if generating genomic output": "如果生成基因组输出，则必需",
    "(requires reference-aligned data)": "（需要参考序列对齐的数据）",
    "for merging adjacent loci": "用于合并相邻位点",
    "required p-value to keep an Fst measurement (0.05 by default).": "保留Fst测量所需的p值（默认为0.05）。",
    "The options -k and --smooth-fstats implicitly enable --fstats. Note that, in order to set non-default values for --fst-correction or --p-value-cutoff the option --fstats still needs to be enabled in the tool form.": "选项 -k 和 --smooth-fstats 隐式地启用了 --fstats。请注意，为了在工具表单中设置 --fst-correction 或 --p-value-cutoff 的非默认值，仍然需要启用 --fstats 选项。",
    "distance over which to average values (default 150Kb)": "平均值计算的距离（默认为150Kb）",
    "works only if a population map specifying only two groupings: 'parent' and 'progeny'": "仅在指定仅包含两个分组的种群图时有效：'parent' 和 'progeny'",
    "needs to be selected to trigger the additional Genotyping markers output": "需要选择以触发额外的基因分型标记输出",
    "ignore paired-end reads even if present in the input": "忽略配对末端读取，即使在输入中存在",
    "heavy BAM files": "大型BAM文件",
    "implies --rm-unpaired-reads": "意味着 --删除未配对的读取",
    "in fraction of read length": "在读取长度的分数中",
    "range of edge coverage thresholds to iterate over when building the graph of allele cooccurrences for SNP phasing (--phasing-cooccurrences-thr-range)": "在构建SNP分型图时，用于迭代的边覆盖度阈值范围（--phasing-cooccurrences-thr-range）",
    "(default 1; suggested: set to ustacks -M)": "（默认值1；建议：设置为ustacks -M）",
    "default: automatic": "默认：自动",
    "Single end data or forward reads. If a paired list is provided only the forward reads are used in ustacks": "单端数据或前向读取。如果提供了配对列表，只有在ustacks中使用前向读取。",
    "(default: M + 2)": "（默认值：M + 2）",
    "in standard deviation units": "在标准差单位内",
    "If you are combining multiple ustacks runs at the cstacks stage, use this option to avoid having different samples with the same identifier.": "如果在cstacks阶段合并多个ustacks运行，请使用此选项以避免具有相同标识符的不同样本。",
    "specify a correction to be applied to Fst values: 'p_value', 'bonferroni_win', or 'bonferroni_gen'": "指定要应用于Fst值的校正类型：'p_value'、'bonferroni_win' 或 'bonferroni_gen'",
    "required p-value to keep an Fst measurement (0.05 by default). Also used as base for Bonferroni correction": "保留Fst测量所需的p值（默认为0.05）。也用作Bonferroni校正的基础",
    "Standard deviation of the kernel smoothing weight distribution (sigma, default 150Kb)": "核平滑权重分布的标准差（sigma，默认为150Kb）",
    "Only useful if you analyse data that was processed outside galaxy": "只有在分析在Galaxy之外处理的数据时才有用",
    "output from a previous Stacks pipeline steps e.g. denovo_map, refmap or cstacks": "来自之前 Stacks 流水线步骤的输出，例如 denovo_map、refmap 或 cstacks",
    "output from previous Stacks pipeline steps e.g. denovo_map, refmap or ustacks/pstacks": "来自之前 Stacks 流水线步骤的输出，例如 denovo_map、refmap 或 ustacks/pstacks",
    "Output map file type to write": "输出映射文件类型以写入",
    "Only needed for Genomic output format": "仅适用于基因组输出格式",
    "below this minor allele frequency a stack is called a homozygote, above it (but below --max_het_seqs) it is called unknown": "在这个次要等位基因频率以下，一个栈被称为纯合子，高于它（但低于 --max_het_seqs）则被称为未知。",
    "minimum frequency of minor allele to call a heterozygote": "次等位基因的最小频率，以判断杂合子",
    "Files containing parent sequences from a mapping cross (only R1 reads). Dataset names will be used as sample name (no space allowed).": "包含来自映射交叉的父母序列的文件（仅限R1读取）。数据集名称将用作样本名称（不允许有空格）。",
    "files containing progeny sequences from a mapping cross (only R1 reads). Dataset names will be used as sample name (no space allowed).": "包含来自映射杂交后代序列的文件（仅限R1读取）。数据集名称将用作样本名称（不允许有空格）。",
    "files containing an individual sample from a population (only R1 reads). Dataset names will be used as sample name (no space allowed).": "包含来自种群的个体样本的文件（仅R1读取）。数据集名称将用作样本名称（不允许有空格）。",
    "default is [-M]+2": "默认是 [-M]+2",
    "input files": "输入文件",
    "Files must have this syntax : name_R1_001.fastq": "文件必须遵循这种语法：name_R1_001.fastq",
    "Files must have this syntax : name_R2_001.fastq": "文件必须具有这种语法：name_R2_001.fastq",
    "Dataset names will be used as sample name (no space allowed).": "数据集名称将用作样本名称（不允许有空格）。",
    "specify the minimum depth of coverage to report a stack in pstacks": "指定在 pstacks 中报告一个 stack 的最小覆盖深度。",
    "specify the minimum depth of coverage to report a stack in pstacks for 'progeny' individuals": "为 'progeny' 个体在 pstacks 中报告一个 stack 时指定最小覆盖深度",
    "for single iteration- NO, for multiple-YES": "对于单次迭代 - 否，对于多次迭代 - 是",
    "to minimize memory usage": "为了最小化内存使用",
    "by default true": "默认为真",
    "-knn num": "-knn num",
    "-nhf num": "-nhf num",
    "The number of hash functions is increased by this value after each iteration.": "每次迭代后，哈希函数的数量增加此值。",
    "Default is 10000": "默认值是10000",
    "stokholm format": "斯德哥尔摩格式",
    "fasta format": "FASTA格式",
    "similarity sclores": "相似性得分",
    "data map": "数据映射",
    "must be the same as for previous step": "必须与前一步相同",
    "text format": "文本格式",
    "otherwise ignore subtree if it contains overlapping sequences": "否则如果子树包含重叠序列，则忽略该子树",
    "comma separated integers": "逗号分隔的整数",
    "by default 30": "默认情况下为30",
    "by default 10000": "默认情况下为10000",
    "otherwise use E-value": "否则使用E值",
    "Tabular tsv file of the computed conservation metrics from align cluster step": "对齐聚类步骤计算出的保守性指标的表格tsv文件",
    "Clusters with fewer seqeunces are excluded": "序列较少的聚类被排除在外",
    "Minimum RNAz SVM RNA-class probablity for bed annotation as RNAz hit": "RNAz SVM RNA类别概率的最小值，用于bed注释作为RNAz命中",
    "Minimum number of significant covarying basepairs by Rscape for bed annotation as Rscape hit": "Rscape用于bed注释的显著共变碱基对的最小数量为Rscape命中",
    "Exclude spurious alignment structs with poor SCI less than 0.01": "排除具有小于0.01的SCI值的可疑对齐结构",
    "Output additional prediction metrics": "输出额外的预测指标",
    "FASTA.zip from pre-processing step": "预处理步骤中的FASTA.zip文件",
    "Tabular cmsearch results of the candidate clusters from the cmsearch step": "来自cmsearch步骤的候选簇的表格cmsearch结果",
    "model.tree.stk files from pgma_graphclust candidate clustering step": "pgma_graphclust候选聚类步骤生成的model.tree.stk文件",
    "Whether to do hard partitioning (no overlap) or soft mode(cluster elements may overlap)": "是否进行硬分区（无重叠）或软模式（聚类元素可能重叠）",
    "Optional CD-HIT pre-clustering output to be combined into the final clustering output": "可选的CD-HIT预聚类输出，将被合并到最终的聚类输出中",
    "Overlapping ratio criteria to merge overlapping clusters or keep separate clusters (soft partitioning)": "重叠比例标准，用于合并重叠的聚类或保持独立的聚类（软分区）",
    "Overlapping ratio criteria to merge overlapping sequence fractions from same input sequence": "重叠比率标准，用于合并来自同一输入序列的重叠序列片段",
    "Minimum number of elements that can form a cluster. Higher values discard small clusters and may produce larger merged clusters": "最小元素数量，可以形成聚类。较高的值会丢弃小聚类，并可能产生更大的合并聚类。",
    "The fasta reference genome that SNAP should align reads against.": "SNAP 应该用来比对读取的fasta参考基因组。",
    "Length of the seeds used in the reference genome hash table (SNAP index option -s).": "在参考基因组哈希表中使用的种子长度（SNAP索引选项 -s）。",
    "Corresponds to the -h option of SNAP index.": "对应于 SNAP 索引的 -h 选项。",
    "Factor (between 1 and 1000) that controls the size of the index build overflow space. For certain genomes you may have to increase this value if you are getting a corresponding error from the tool.": "控制索引构建溢出空间大小的因子（介于1和1000之间）。对于某些基因组，如果您从工具中获得相应的错误，可能需要增加此值。",
    "The global alignment parameters in this section will be used for samples for which you do not provide their own sample-specific settings.": "本节中的全局比对参数将用于那些您没有提供自己特定样本设置的样本。",
    "These parameters will affect the alignments for any single-end sample for which you do not provide sample-specific settings.": "这些参数将影响任何单端样本的比对，对于这些样本，如果您没有提供特定于样本的设置。",
    "maximum edit distance allowed per read or pair (SNAP option -d); higher values allow more divergent alignments to be found, but increase the rate of misalignments.": "每个读取或配对允许的最大编辑距离（SNAP选项-d）；更高的值允许找到更多不同的对齐，但会增加错配的比率。",
    "Confidence threshold (SNAP option -c); the minimum edit distance difference between two alternate alignments required to reject the poorer alignment as suboptimal; higher values increase the rate of ambiguously aligned reads.": "置信度阈值（SNAP选项-c）；两个交替比对之间的最小编辑距离差异，用于拒绝较差的比对作为次优比对；较高的值会增加不确定比对的读取率。",
    "Specifies how many seeds of a read may be ignored (based on the maximum hits value above) before the confidence threshold above gets increased by one for that read (SNAP option -a); helps fine-tuning alignment accuracy in repetitive regions of the genome.": "指定在增加某个read的置信度阈值（基于上述最大命中数）之前，可以忽略该read的多少个种子（SNAP选项-a）；有助于在基因组的重复区域中微调对齐精度。",
    "Number of seeds to use per read (SNAP option -n) when trying to match it to the reference genome; higher numbers will increase the rate of aligned reads and reduce the rate of misalignments, but will reduce performance.": "每个读取（SNAP选项-n）在尝试将其与参考基因组匹配时使用的种子数；较高的数字将增加对齐读取的比率并减少错配的比率，但会降低性能。",
    "Maximum hits to consider per seed (SNAP option -h); don't use a seed region in the alignment process if it matches more than maxhits regions in the reference genome. Higher values reduce the rate of misalignments, but reduce performance.": "每个种子考虑的最大命中数（SNAP选项-h）；如果种子区域在参考基因组中匹配的区域超过maxhits，则在比对过程中不要使用该种子区域。较高的值可以降低错配率，但会降低性能。",
    "Specifies from which end of a read low-quality bases should be clipped (SNAP option -Cxx)": "指定从哪个端点剪切低质量碱基（SNAP选项 -Cxx）",
    "These parameters will affect the alignments for any paired-end sample for which you do not provide sample-specific settings.": "这些参数将影响任何配对端样本的比对，对于您没有提供特定样本设置的样本。",
    "Corresponds to the first value of the SNAP option -s.": "对应于SNAP选项-s的第一个值。",
    "Corresponds to the second value of the SNAP option -s.": "对应于SNAP选项的第二个值 -s。",
    "Consider overlapping mate pairs of the given orientation type(s) anomalous and discard them; allowed values: RF, FR, FF, RR; multiple types may be specified as a comma-separated list and ALL can be used as a shortcut for discarding all overlapping mate pairs; leave blank to retain all overlapping pairs.": "考虑给定方向类型的重叠配偶对为异常并丢弃它们；允许的值：RF, FR, FF, RR；可以指定多个类型作为逗号分隔的列表，ALL可以用作丢弃所有重叠配偶对的快捷方式；留空以保留所有重叠对。",
    "No matter how many input datasets you specify below and what there formats are, this tool will produce a single output file with the aligned reads from all samples. In this section you can configure some aspects of what the output should look like. Unless you have a really special usecase, you can (and probably should) just go with the default settings though.": "无论您在下面指定了多少输入数据集以及它们的格式是什么，这个工具都会生成一个包含所有样本对齐读取的单一输出文件。在这一节中，您可以配置一些输出应该是什么样子的方面。除非您有一个非常特殊的用例，否则您可以（并且可能应该）只使用默认设置。",
    "Turn off if you want to retain the read order of the input file(s) (mimodd snap option --no-sort).": "如果你想要保留输入文件（mimodd snap选项--no-sort）的读取顺序，请关闭。",
    "Warning: Downstream tools may still rely on the classic M notation! Turn this on at your own risk (mimodd snap option -X).": "警告：下游工具可能仍然依赖于经典的M表示法！自行承担风险打开此选项（mimodd snap选项 -X）。",
    "randomly choose 1/selectivity of the reads to score (SNAP option -S). The default of 1 indicates that all reads should be worked with.": "随机选择1/评分的读取选择性（SNAP选项-S）。默认值1表示应该处理所有读取。",
    "filter output (SNAP option -F) to retain only specific classes of reads.": "过滤输出（SNAP选项 -F），以保留特定类别的读取。",
    "Reads obtained from single-end sequencing runs should be aligned in 'single' mode, paired-end reads in 'paired' mode. **WARNING**: if the read input file is in SAM/BAM format, the current version of this tool will **not** verify the mode and may produce erroneous alignments with wrong settings!": "来自单端测序运行的读取应该以“single”模式对齐，配对端读取以“paired”模式对齐。**警告**：如果读取输入文件是SAM/BAM格式的，这个工具的当前版本将**不**验证模式，可能会产生错误的对齐结果，如果设置错误的话！",
    "Any options you specify here will overwrite the global alignment settings defined for all single-end samples above.": "您在这里指定的任何选项都将覆盖上面为所有单端样本定义的全局比对设置。",
    "Any options you specify here will overwrite the global alignment settings defined for all paired-end samples above.": "您在这里指定的任何选项都将覆盖上面为所有配对端样本定义的全局比对设置。",
    "A less common option, but necessary, e.g., if you want to re-align coordinate sorted paired-end reads with the MiModD Read Alignment Tool or other NGS mapping tools.": "一个不太常见的选项，但有时是必要的，例如，如果你想使用MiModD读取对齐工具或其他下一代测序（NGS）映射工具重新对齐坐标排序的配对末端读取。",
    "Select Simple Variant Density (SVD) Mapping to map mutations based on linked inheritance in near isogenic populations, Variant Allele Frequency (VAF) Mapping for bulk segregant analysis. Select Reprocess for rapidly replotting the result of a previous VAF analysis.": "选择简单变异密度（SVD）映射以基于近等基因群体中链接遗传来映射突变，选择变异等位基因频率（VAF）映射进行大量分离分析。选择重新处理以快速重新绘制先前VAF分析的结果。",
    "You can either choose to produce a tabular per-variant report, which is useful for fast replotting with different plot settings, or a VCF-like CloudMap-compatibility output that can be used as input for the older CloudMap EMS Variant Density Mapping tool as an alternative plotting tool.": "你可以选择生成每个变体的表格报告，这有助于使用不同的绘图设置快速重新绘图，或者生成类似VCF的CloudMap兼容性输出，该输出可以用作较旧的CloudMap EMS变体密度映射工具的输入，作为替代绘图工具。",
    "the sample to perform mutation mapping for": "样本进行突变映射",
    "the sample that provides variants present in your original mutant strain or in an ancestor (like the pre-mutagenesis strain); leave blank if not available": "提供原始突变菌株中存在的变体或祖先（如突变前的菌株）的样本；如果不可用，请留空。",
    "the sample that provides variants present in the unrelated mapping strain (or in an ancestor of it) used in the mapping cross; leave blank if not available": "提供在无关的作图菌株（或其祖先）中存在的变异的样本；如果不可用，请留空",
    "if variant data for either the related or the unrelated parent strain is not available, the tool can try to infer the alleles present in that parent from the allele spectrum found in the mapping sample. Use with caution on carefully filtered variant lists only!": "如果相关或无关的亲本菌株的变异数据不可用，该工具可以尝试从映射样本中发现的等位基因谱推断该亲本中存在的等位基因。请仅在仔细过滤的变异列表上谨慎使用！",
    "You can either choose to produce a tabular per-variant report, which is useful for fast replotting with different plot settings, or a VCF-like CloudMap-compatibility output that can be used as input for the older CloudMap Hawaiian Variant Mapping tool as an alternative plotting tool.": "你可以选择生成一个每个变体的表格报告，这对于使用不同的绘图设置进行快速重新绘图非常有用，或者选择一个类似VCF的CloudMap兼容输出，该输出可以用作旧版CloudMap夏威夷变体映射工具的输入，作为替代绘图工具。",
    "the sample that provides contrasting allele composition around the causal variant": "提供因果变异周围对比等位基因组成的样本",
    "You can choose to produce a tabular per-variant report, which is useful for fast replotting with different plot settings.": "你可以选择生成一个每个变体的表格报告，这对于使用不同的绘图设置快速重新绘图非常有用。",
    "Your choice will update the interface to display further choices appropriate for your type of input data.": "您的选择将更新界面以显示适合您输入数据类型的进一步选择。",
    "Use the Variant Calling tool to generate input for this tool.": "使用变异检测工具生成此工具的输入。",
    "Use the MiModD Variant Calling Tool to generate this file.": "使用 MiModD 变异检测工具生成此文件。",
    "If selected, reads from different read groups will be treated strictly separate. If turned off, read groups with identical sample names are used together for identifying uncovered regions, but are still treated separately for the prediction of deletions.": "如果选中，来自不同读组的读取将被严格分开处理。如果关闭，具有相同样本名称的读组将一起用于识别未覆盖区域，但在预测缺失时仍然分开处理。",
    "If selected, regions that fulfill the coverage criteria below, but are not statistically significant deletions, will be included in the output.": "如果选定，满足下面覆盖标准但不是统计学上显著的缺失的区域将被包含在输出中。",
    "The maximal coverage at a site allowed to consider it as part of a low-coverage region": "在考虑一个位点作为低覆盖区域的一部分时，允许的最大覆盖度",
    "A low-coverage region must consist of at least this number of consecutive bases below the maximal coverage to consider it in further analyses.": "一个低覆盖区域必须包含至少这个数量的连续碱基低于最大覆盖度，才能在进一步分析中考虑。",
    "the VCF file to rebase": "待转换为标准碱基表示的 VCF 文件",
    "the UCSC chain file to calculate new coordinates from": "UCSC链文件用于计算新的坐标",
    "swap the genome versions specified in the chain file, i.e., assume the coordinates in the input file are based on the chain file target genome version and should be mapped to the source genome version": "交换链文件中指定的基因组版本，即假设输入文件中的坐标基于链文件目标基因组版本，并应将其映射到源基因组版本。",
    "How to treat variants that map to more than one position in the target coordinate system": "如何处理映射到目标坐标系统中多个位置的变异",
    "the file to reheader.": "要重新添加头部的文件。",
    "Replace mode will ignore ALL existing read group information in the input file and use ONLY template information, Update mode will COPY existing input file information and UPDATE it with template information; choose No, ... to leave read-group information alone.": "替换模式将忽略输入文件中的所有现有读组信息，并且只使用模板信息，更新模式将复制现有输入文件信息并用模板信息进行更新；选择“否，...”以保留读组信息不变。",
    "Warning: changing read-group IDs may increase job runtime substantially.": "警告：更改读取组ID可能会大幅度增加作业运行时间。",
    "as it appears in the current input file header": "如当前输入文件头部所示",
    "Warning: changing sequence names may increase job runtime substantially.": "警告：更改序列名称可能会显著增加作业运行时间。",
    "Select at least one dataset to call variants on. If you select several datasets or a dataset collection, this tool will perform joint variant calling on all of them and produce a single, possibly multisample, output dataset.": "选择至少一个数据集来调用变异。如果您选择了几个数据集或一个数据集集合，这个工具将对所有数据集执行联合变异调用，并产生一个单一的、可能是多样本的输出数据集。",
    "If selected, this option ensures that only the read group id (but not the sample name) is considered in grouping reads in the input file(s). If turned off, read groups with identical sample names are automatically pooled and analyzed together even if they come from different NGS runs.": "如果选择了这个选项，它确保只有读取组ID（而不是样本名称）在将输入文件中的读取进行分组时被考虑。如果关闭，即使来自不同的NGS运行，具有相同样本名称的读取组也会自动被汇集并一起分析。",
    "leave turned on to avoid accidental variant calling against a wrong reference genome version (see the tool help below).": "保持开启以避免意外地针对错误版本的参考基因组进行变异调用（请参阅下面的工具帮助）。",
    "only relevant for very large sample numbers and/or very high sample coverage; increase to use more of the data, decrease to save memory": "仅适用于非常大的样本数量和/或非常高的样本覆盖率；增加以使用更多数据，减少以节省内存",
    "Use the MiModD Variant Calling tool to generate the input for this tool.": "使用 MiModD 变异检测工具来生成此工具的输入。",
    "If selected, the VCF output will include ALL sites for which non-reference bases have been observed, i.e., even those not considered allelic sites by the variant caller.": "如果选中，VCF输出将包括观察到非参考碱基的所有位点，即，甚至包括变异调用器未考虑为等位基因位点的那些位点。",
    "If you declare the species your input data comes from, variant reports in html format can have hyperlinks to species-specific databases and genome browsers added. If you have used the MiModD Variant Annotation tool to generate the input dataset, the species information will already be recorded in the dataset, but you can overwrite it if you wish.": "如果您声明了您的输入数据来自的物种，那么以HTML格式的变异报告可以添加指向物种特定数据库和基因组浏览器的超链接。如果您已经使用MiModD变异注释工具生成了输入数据集，物种信息将已经记录在数据集中，但如果您愿意，可以覆盖它。",
    "If the tool has no built-in support for your species, i.e., it does not know how to generate hyperlinks for it, you can provide a custom recipe here.": "如果该工具没有为您的物种提供内置支持，即它不知道如何为其生成超链接，您可以在这里提供一个自定义配方。",
    "name of a sample as it appears in the VCF input file and that indicates the sample that this filter should be applied to.": "在 VCF 输入文件中出现的样本名称，并指示应将此过滤器应用于哪个样本。",
    "keep only variants for which the genotype of the sample matches the specified pattern; format: x/x where x = 0 is wildtype and x = 1 is mutant. Multiple genotypes can be specified as a comma-separated list.": "只保留样本的基因型与指定模式匹配的变异；格式：x/x，其中x=0表示野生型，x=1表示突变型。多个基因型可以作为逗号分隔的列表指定。",
    "keep only variants with at least this sample-specific coverage at the variant site": "仅保留至少在变异位点具有此特定样本覆盖度的变异",
    "keep only variants for which the genotype prediction for the sample has at least this quality": "仅保留样本的基因型预测质量至少达到此值的变异。",
    "expected format: [allele number]:[minimal fraction]:[maximal fraction]; keep only variants for which the fraction of sample-specific reads supporting a given allele number is between minimal and maximal fraction; if allele number is omitted, the filter operates on the most frequent non-reference allele instead": "预期格式：[等位基因编号]:[最小分数]:[最大分数]；只保留样本特定读取支持给定等位基因编号的比例在最小和最大分数之间的变异；如果省略了等位基因编号，过滤器将操作最频繁的非参考等位基因",
    "Filter variant sites by their position in the genome. If multiple Region Filters are specified, all variants that fall in ONE of the regions are reported.": "根据它们在基因组中的位置过滤变异位点。如果指定了多个区域过滤器，则报告落在任一区域中的所有变异。",
    "Filter output by sample name; only the sample-specific columns with their sample name matching any of the comma separated filters will be retained in the output.": "根据样本名称过滤输出；只有与逗号分隔的过滤器中的任何样本名称匹配的样本特定列才会保留在输出中。",
    "See syntax below": "见下面的语法",
    "See TIP below": "请见下面的提示",
    "To return all, enter 0": "请输入0以返回所有结果",
    "Enter an integer value greater than 0": "输入一个大于0的整数值",
    "Can be used to center and/or scale variables": "可以用来居中和/或缩放变量",
    "Selecting 'Yes' will rescale scores (and coefficients) to produce scores of unit variance": "选择\"Yes\"将重新调整分数（和系数）以产生单位方差的分数",
    "Please provide Galaxy workflows as a tabular file.": "请提供以表格文件形式的Galaxy工作流。",
    "Please provide tools usage frequencies as a tabular file.": "请提供工具使用频率的表格文件。",
    "Provide a date (in the past) in yyyy-mm-dd format. The earliest date from which usage of tools will be extracted. For example, 2017-12-01 specifies that the usage of tools from this date until the data extraction date is extracted. The usage of tools before this date is not considered.": "提供一个日期（在过去的日期）以 yyyy-mm-dd 格式。这是提取工具使用情况的最早日期。例如，2017-12-01 指定从这个日期到数据提取日期的工具使用情况将被提取。不考虑这个日期之前的工具使用情况。",
    "Provide an integer between 1 and 25. A workflow is divided into unique paths and this number specifies the maximum number of tools a path can have. Paths longer than this number are ignored and are not included in the deep learning training.": "提供一个介于1到25之间的整数。一个工作流被划分为独特的路径，此数字指定一个路径可以拥有的最大工具数量。超过这个数量的路径将被忽略，并且不会包含在深度学习训练中。",
    "Provide an integer. Different combinations of parameters are sampled and optimized to find the best one. This number specifies the number of different configurations sampled and tested.": "提供一个整数。不同的参数组合被采样和优化以找到最佳配置。这个数字指定了被采样和测试的不同配置的数量。",
    "Provide a real number between 0.0 and 1.0. This set of data is used to look through the prediction accuracy on unseen data after neural network training on an optimised configuration of parameters.": "提供一个介于 0.0 到 1.0 之间的实数。这组数据用于在神经网络针对一组优化的参数配置训练完成后，评估模型在未见过的数据上的预测准确性。",
    "Please select at least one column": "请选择至少一列",
    "An feature-observation matrix, with features as rows and observations as columns (e.g. Genes vs Cells)": "一个特征-观测矩阵，其中特征作为行，观测作为列（例如基因对细胞）",
    "A two-column file with observations in the first column, and an integer representing their assigned class in the second column (e.g. 'YFPCD24X3w_20 2')": "一个包含两列的文件，第一列是观察值（例如 'YFPCD24X3w_20'），第二列是一个整数，表示它们分配的类别（例如 '2'）。",
    "For example if you have a chromosome column you probably want to group each chromosome before you apply any function.": "例如，如果您有一个染色体列，您可能希望在应用任何函数之前先对每个染色体进行分组。",
    "For more information please see https://en.wikipedia.org/wiki/Window_function": "有关更多信息，请参见 https://en.wikipedia.org/wiki/窗口函数",
    "By default the labels are set to the right edge of the window. Here you can change that to the center.": "默认情况下，标签设置在窗口的右边缘。在这里，您可以将其更改为中心。",
    "File created by 'IWTomics Test and Plot'.": "由“iwtomics测试和绘图”创建的文件。",
    "Maximum interval length for the p-value adjustment. If 0 the maximum possible scale is used.": "p值校正的最大区间长度。如果为0，则使用可能的最大尺度。",
    "File created by 'IWTomics Load Smooth and Plot'.": "由'IWTomics 加载平滑并绘图'创建的文件。",
    "IDs of the region datasets to be tested.": "要测试的区域数据集的ID。",
    "IDs of the features to be tested.": "要测试的特征的ID。",
    "Probabilities corresponding to the quantiles in test statistics.": "对应于检验统计量中分位数的概率。",
    "Tabular file. Column 1: names of the region files; Column 2: unique identifiers of the region datasets (spaces and special characters are not allowed); Column 3: name of the region datasets to be used in the output.": "表格文件。第1列：区域文件的名称；第2列：区域数据集的唯一标识符（不允许使用空格和特殊字符）；第3列：在输出中使用的区域数据集的名称。",
    "Tabular file. Column 1: names of the feature files; Column 2: unique identifiers of the features (spaces and special characters are not allowed); Column 3: name of the features to be used in the output.": "表格文件。第1列：特征文件的名称；第2列：特征的唯一标识符（不允许有空格和特殊字符）；第3列：在输出中使用的特征名称。",
    "Type of smoothing to be applied to the feature curves.": "要应用于特征曲线的平滑类型。",
    "TSV file containing a matrix with: (i) quantitative values only (decimal should be separated by '.'), (ii) the samples in lines (should be labelled in the 1rst column) and (iii) variables in columns (should have a header).": "包含矩阵的TSV文件，其中：(i) 仅包含定量值（小数点应以“.”分隔），(ii) 样本在行中（应在第1列中标记），以及(iii) 变量在列中（应有标题）。",
    "By default, on tabulated files with a header.": "默认情况下，在带有标题的表格文件上。",
    "Used for both blocks and color files.": "用于块和颜色文件。",
    "Character used to separate the column (for all blocks, connection and color files).": "用于分隔所有块（连接和颜色文件）中列的字符。",
    "By default, the analysis: is a Regularised Generalised Canonical Correlation Analysis, scales the blocks, uses a superblock with a factorial scheme function, a tau equals to one and two components for each block.": "默认情况下，分析是一种正则化的广义典型相关分析，对块进行缩放，使用具有因子方案函数的超块，τ等于1，并且每个块有两个组分。",
    "The number of component to use in the analysis for each block (should not be greater than the minimum number of variable among the blocks).": "在每个区块的分析中要使用的组件数量（不应大于区块中变量的最小数量）。",
    "A data centering step is always performed. If activated, each block is normalised and divided by the square root of its number of variables.": "总是执行一个数据中心化步骤。如果激活，每个块会被标准化并除以其变量数量的平方根。",
    "TSV file without header and without row names. This file describes the connections between the blocks. It should contain 1 (if two blocks are related) or 0 values otherwise. The columns are separated by tabulations. It is a symmetric matrix with the same dimension as the number of blocks.": "没有标题和行名的制表符分隔值（TSV）文件。该文件描述了区块之间的连接。如果两个区块相关，则应包含1（值），否则包含0值。列由制表符分隔。这是一个与区块数量相同维度的对称矩阵。",
    "A block defined as the concatenation of all the other blocks. The space spanned by global components is viewed as a compromise space that integrated all the modalities and facilitates the visualization of the results and their interpretation. If disabled, all blocks are assumed to be connected or a connection file could be used.": "一个区块被定义为所有其他区块的串联。由全局成分所跨越的空间被视为一个折衷的空间，它整合了所有模态并促进了结果的可视化和解释。如果禁用，所有区块都被假定为连接的，或者可以使用连接文件。",
    "@BLOCK_RULES@ By default, the first block is selected.": "默认情况下，选择第一个块。",
    "For RGCCA, a regularization parameter for each block (i.e., tau) [default: 1]. Tau varies from 0 (maximizing the correlation) to 1 (maximizing the covariance). For SGCCA, tau is automatically set to 1 and a shrinkage parameter can be defined instead for automatic variable selection, varying from the square root of the variable number (the fewest selected variables) to 1 (all the variables are included).": "对于RGCCA，每个块（即，tau）都有一个正则化参数[默认值：1]。Tau的值从0（最大化相关性）变化到1（最大化协方差）。对于SGCCA，tau自动设置为1，可以定义一个收缩参数以实现自动变量选择，该参数的值从变量数量的平方根（选择的变量最少）变化到1（包括所有变量）。",
    "Link (i.e. scheme) function for covariance maximization is calculated with: the identity function (horst scheme), the absolute values (centroid scheme), the squared values (factorial scheme). Only, the horst scheme penalizes structural negative correlation. The factorial scheme discriminates more strongly the blocks than the centroid one.": "用于协方差最大化的链接（即方案）函数通过以下方式计算：恒等函数（horst方案）、绝对值（质心方案）、平方值（因子方案）。只有horst方案会惩罚结构性负相关。因子方案比质心方案更强烈地区分块。",
    "By default, the x-axis and y-axis are respectively the first and the second components, the number of top variables is 100 and a superblock is used.": "默认情况下，x轴和y轴分别是第一和第二个主成分，顶部变量的数量是100，并且使用了超块。",
    "A TSV file containing either: (i) an only column with a qualitative or a quantitative variable; (ii) multiple columns corresponding to a disjunctive table.": "一个包含以下内容的制表符分隔值（TSV）文件：（i）只有一列，包含定性或定量变量；（ii）多列，对应于一个析取表。",
    "The component used in the X-axis for biplots and the only component used for top variable plot. @COMP_RULES@": "在双标图（biplots）中用于X轴的组件，以及用于顶部变量图的唯一组件。@COMP_RULES@",
    "The component used in the Y-axis for biplots. @COMP_RULES@": "在双标图（biplots）中用于 Y 轴的组件。@COMP_RULES@",
    "Block used in the X-axis for individual plot and the only block used for corcircle and top variable plots. @BLOCK_RULES@": "在个体图表中用于X轴的区块，以及用于corcircle和顶部变量图表的唯一区块。@BLOCK_RULES@",
    "0 corresponds to the superblock (or the last block loaded), @BLOCK_RULES@ By default, the superblock is selected.": "0 对应于超级块（或最后加载的块），@BLOCK_RULES@ 默认情况下，选择超级块。",
    "tabular file containing the observations": "包含观察结果的表格文件",
    "The correlation matrix can only be used if there are no constant variables": "相关性矩阵只有在没有常量变量的情况下才能使用",
    "use tab file to convert the output to labels": "使用制表符文件将输出转换为标签",
    "accept extended SIF (label:weight fields)": "接受扩展的SIF（标签：权重字段）",
    "This value is the main handle for affecting cluster granularity. It is usually chosen somewhere in the range [1.2-5.0]. -I 5.0 will tend to result in fine-grained clusterings, and -I 1.2 will tend to result in very coarse grained clusterings. Your mileage will vary depending on the characteristics of your data. That is why it is a good idea to test the quality and coherency of your clusterings using clm dist and clm info. This will most likely reveal that certain values of -I are simply not right for your data.": "这个值是影响聚类粒度的主要控制参数。它通常选择在范围[1.2-5.0]中。-I 5.0将倾向于产生细粒度的聚类，而-I 1.2将倾向于产生非常粗粒度的聚类。你的结果会根据你的数据特性而有所不同。这就是为什么使用clm dist和clm info测试你的聚类质量和一致性是一个好主意。这最有可能揭示某些-I值对你的数据来说根本是不正确的。",
    "See https://micans.org/mcl/man/mcxio.html#trans for the transformation syntax": "参见 https://micans.org/mcl/man/mcxio.html#trans 以了解转换语法",
    "Remove any loops that are present in the input. Bear in mind that loops will still be modified in all cases where the loop weight is not maximal among the list of edge weights for a given node.": "移除输入中存在的任何环。请注意，在给定节点的边权重列表中，如果环的权重不是最大的，环仍然会被修改。",
    "As the final step of loop computation (i.e. after initialization and shadowing) all loop weights are multiplied by the provided value.": "作为循环计算的最后一步（即在初始化和遮蔽之后），所有循环权重都乘以提供的值。",
    "inverse cutoff pruning value. Entries smaller than cutoff are removed": "逆截断剪枝值。小于截断值的条目将被移除",
    "pruning selection value. maximum number of entries (if applicable)": "剪枝选择值。最大条目数（如果适用）",
    "Pruning recover number. Revover the largest discarded entries during pruning, if number of entries less then -R": "修剪恢复数量。在修剪过程中，如果条目数量少于-R，则恢复最大被丢弃的条目。",
    "Pruning recover percentage. Revover the largest discarded entries during pruning, if sum of remaining entries is less than -pct/100": "修剪恢复百分比。在修剪过程中，如果剩余条目的总和小于-pct/100，则恢复被丢弃的最大条目。",
    "This tool only supports to split one array at each tool run. If X, y are in separate files, the splitting task could be done by invoking this tool twice in which this input dataset is swapped while all other parameters are kept the same.": "这个工具每次运行只支持分割一个数组。如果X、y在不同的文件中，可以通过两次调用这个工具来完成分割任务，其中输入数据集被交换，而所有其他参数保持不变。",
    "A float number, 0.0 - 1.0, represents the proportion of the dataset to be included in the test split.": "一个浮点数，范围从0.0到1.0，表示数据集中将要包含在测试分割中的比例。",
    "Number of times cross-validator needs to be repeated.": "交叉验证需要重复的次数。",
    "Integer. Size of the test sets.": "整数。测试集的大小。",
    "Number of re-shuffling and splitting iterations.": "重新洗牌和分割迭代的次数。",
    "Maximum size for a single training set.": "单个训练集的最大大小。",
    "List, e.g., [0, 1, -1, 1], represents two test sets, [X[0]] and [X[1], X[3]], X[2] is excluded from any test set due to '-1'.": "列表，例如，[0, 1, -1, 1]，代表两个测试集，[X[0]] 和 [X[1], X[3]]，由于'-1'，X[2] 被排除在任何测试集之外。",
    "Number of groups (p) to leave out in the test split.": "在测试分割中要留出的组数（p）。",
    "Split index starts from 1 to  total = n_splits (x n_repeats). (nth_split)": "分割索引从1开始，总共等于 n_splits (x n_repeats)。（第 nth_split 个分割）",
    "For security reason, deep learning models will be saved into two datasets, model skeleton and weights.": "出于安全原因，深度学习模型将被保存到两个数据集中，模型骨架和权重。",
    "If present, exclude the header row from the dataset": "如果存在，从数据集中排除标题行",
    "Choose which flavour of PCA to use": "选择使用哪种主成分分析（PCA）的风格",
    "Number of components to keep. If not set, all components are kept": "要保留的成分数量。如果没有设置，则保留所有成分",
    "Setting this option will reduce the redundancy and correlations between the features": "设置此选项将减少特征之间的冗余和相关性",
    "Method to perform the singular value decomposition": "执行奇异值分解的方法",
    "Convergence tolerance for arpack. If 0, optimal value will be chosen by arpack": "ARPACK的收敛容差。如果为0，ARPACK将选择最优值",
    "The number of samples to use for each batch": "每批要使用的样本数量",
    "Kernel coefficient for rbf, poly and sigmoid kernels. Ignored by other kernels": "径向基函数（rbf）、多项式（poly）和sigmoid核的核系数。其他核类型会忽略此参数。",
    "Degree for poly kernels. Ignored by other kernels": "多项式核的度数。其他核会忽略此参数",
    "Independent term in poly and sigmoid kernels. Ignored by other kernels": "独立项在多项式和sigmoid核函数中。其他核函数忽略它",
    "Maximum number of iterations for arpack": "ARPACK 的最大迭代次数",
    "Sequences will be one-hot encoded to arrays.": "序列将被独热编码成数组。",
    "fasta": "FASTA",
    "A pre-loaded list of blacklisted intervals.Refer to `selene` for details.": "一个预先加载的黑名单区间列表。详情请参考`selene`。",
    "vcf": "变体调用格式（Variant Call Format，VCF）",
    "A stretch of sequence surrounding the variation position on the reference genome.": "参考基因组上变异位置周围的一段序列。",
    "If False, predict on the variant sequence.": "如果为False，则在变异序列上进行预测。",
    "This dataset should be the output of tool model_validation->learning_curve.": "这个数据集应该是工具 model_validation->learning_curve 的输出。",
    "Optional. If change is desired.": "可选。如果需要更改。",
    "y_true. Headered. For 2-class, single column contains both class labels (e.g. True and False). For multi-label, each column, hot-encoded, corresponds to one label.": "y_true。带标题的。对于二分类，单列包含两个类别标签（例如True和False）。对于多标签，每列热编码对应一个标签。",
    "y_preds. Headered. For 2-class, sinle column or the first column contains scores for the positive label. For multi-label, each column corresponds to one label.": "y_preds. 有标题。对于二分类，单个列或第一列包含正标签的分数。对于多标签，每列对应一个标签。",
    "The label of positive class. If not specified, it will be 1 by default.": "正类标签。如果未指定，默认为1。",
    "For mulitple label binary classifications, whose number of true postives is less than the threhold will be ignored.": "对于多重标签二分类，其真正阳性数量少于阈值的情况将被忽略。",
    "Whether to drop some suboptimal thresholds which would not appear on a plotted ROC curve.": "是否删除一些在绘制的ROC曲线上不会出现的次优阈值。",
    "Headered. Single column. Could be Output from `estimator_attributes->grid_scores_`.": "带标题的。单列。可能是来自 `estimator_attributes->grid_scores_` 的输出。",
    "List, containing hover labels for each grid_score. For example: `list(range(10)) + [15, 20]`.": "包含每个grid_score的悬停标签的列表。例如：`list(range(10)) + [15, 20]`。",
    "Make sure the headers (first row) are feature names.": "确保表头（第一行）是特征名称。",
    "Features with importance below the threshold value will be ignored.": "特征重要性低于阈值的特征将被忽略。",
    "Optional.": "可选的。",
    "The number of boosting stages to perform": "执行提升阶段的数量",
    "maximum depth of the individual regression estimators": "个体回归估计器的最大深度",
    "The following clustering algorithms support sparse matrix operations: ''Birch'', ''DBSCAN'', ''KMeans'', ''Mini BatchK Means'', and ''Spectral Clustering''. If your data is in tabular format, please use other clustering algorithms.": "以下聚类算法支持稀疏矩阵操作：“Birch”、“DBSCAN”、“KMeans”、“Mini BatchK Means”和“Spectral Clustering”。如果您的数据是表格格式，请使用其他聚类算法。",
    "Damping factor between 0.5 and 1.": "阻尼因子在0.5到1之间。",
    "Number of iterations with no change in the number of estimated clusters that stops the convergence.": "没有估计聚类数变化的迭代次数，用于停止收敛。",
    "If False, the affinity matrix is modified inplace by the algorithm, for memory efficiency.": "如果为False，亲和矩阵将由算法直接修改，以提高内存效率。",
    "Affinity to use; euclidean uses the negative squared euclidean distance between points.": "亲和力使用；欧几里得使用点之间的负平方欧几里得距离。",
    "Bandwidth used in the RBF kernel. If not given, it will be computed using a heuristic based on the median of all pairwise distances.": "在径向基函数（RBF）核中使用的带宽。如果没有给出，它将根据所有成对距离的中位数使用启发式方法计算。",
    "If true, initial kernel locations are the bins grid whose coarseness corresponds to the bandwidth, speeding up the algorithm.": "如果为真，初始核位置是与带宽相对应的网格粗糙度的箱格，这将加快算法的速度。",
    "To speed up the algorithm, accept only those bins with at least min_bin_freq points as seeds.": "为了加速算法，只接受至少有min_bin_freq个点的箱子作为种子。",
    "If true, all points (including orphans) are clustered. If false, orphans are given cluster label -1.": "如果为真，则所有点（包括孤立点）都被聚类。如果为假，则孤立点被赋予聚类标签-1。",
    "Metric used to compute the linkage. If linkage is ''ward'', only ''euclidean'' is accepted.": "用于计算链接的度量。如果链接是'ward'，则只接受'euclidean'。",
    "If ‘hard’, uses predicted class labels for majority rule voting. Else if ‘soft’, predicts the class label based on the argmax of the sums of the predicted probabilities, which is recommended for an ensemble of well-calibrated classifiers.": "如果设置为‘hard’，则使用预测的类别标签进行多数投票。否则，如果设置为‘soft’，则根据预测概率之和的argmax来预测类别标签，这对于一个校准良好的分类器集成来说是推荐的。",
    "Optional. Tunable parameters could be obtained through `estimator_attributes` tool.": "可选。可通过 `estimator_attributes` 工具获得可调参数。",
    "Evaluation scores will be output by default.": "评估分数将默认输出。",
    "Controls the number of support vectors. Should be in the interval (0, 1].": "控制支持向量的数量。应该在区间(0, 1]内。",
    "The maximum number of iterations to be run.": "最大迭代次数。",
    "Specifies the loss function. ''squared_hinge'' is the square of the hinge loss.": "指定损失函数。''squared_hinge'' 是合页损失的平方。",
    "Select the algorithm to either solve the dual or primal optimization problem. Prefer dual=False when n_samples > n_features.": "选择算法来解决对偶或原始优化问题。当样本数大于特征数时，优先选择dual=False。",
    "Determines the multi-class strategy if y contains more than two classes.": "如果y包含两个以上的类别，则确定多类策略。",
    "If set to false, data is expected to be already centered.": "如果设置为 false，则预期数据已经被中心化。",
    "Each class is composed of a number of gaussian clusters each located around the vertices of a hypercube in a subspace of dimension n_informative. For each cluster, informative features are drawn independently from N(0, 1) and then randomly linearly combined within each cluster in order to add covariance. The clusters are then placed on the vertices of the hypercube.": "每个类别由多个高斯聚类组成，每个聚类都位于一个超立方体的顶点上，该超立方体位于一个维度为 n_informative 的子空间中。对于每个聚类，信息特征独立地从 N(0, 1) 中抽取，然后在每个聚类内随机线性组合，以增加协方差。然后，这些聚类被放置在超立方体的顶点上。",
    "These features are generated as random linear combinations of the informative features.": "这些特征是作为信息特征的随机线性组合生成的。",
    "These are drawn randomly from the informative and the redundant features.": "这些是从有信息特征和冗余特征中随机抽取的。",
    "The number of classes (or labels) of the classification problem.": "分类问题中的类别（或标签）数量。",
    "The covariance matrix will be this value times the unit matrix. This dataset only produces symmetric normal distributions.": "协方差矩阵将是这个值乘以单位矩阵。这个数据集只产生对称的正态分布。",
    " Floating point number less than 1.": "小于1的浮点数。",
    "the number of features used to build the linear model used to generate the output": "用于生成输出的线性模型所使用的特征数量",
    "The dimension of the y output vector associated with a sample. By default, the output is a scalar.": "与样本相关联的y输出向量的维度。默认情况下，输出是一个标量。",
    "The bias term in the underlying linear model.": "线性模型中的偏差项。",
    "dict, optional Dict of arguments passed to :class:Optimizer. For example, {'base_estimator': 'RF'} would use a Random Forest surrogate instead of the default Gaussian Process.": "字典，可选的参数字典，传递给:class:Optimizer。例如，{'base_estimator': 'RF'} 将使用随机森林替代默认的高斯过程。",
    "Maximum parallel equals n_points times cv jobs.": "最大并行等于 n_points 乘以 cv 作业。",
    "For a non-deep learning model, save will output fitted best_estimator_ or a list of cv_results_ from each outer split if in nested CV mode. For a deep learning model, by checking the boolean option below the model input, the outputs are two parts, model skeleton and weights. Save Deep learning model for nested CV is not supported.": "对于非深度学习模型，保存操作将输出拟合后的best_estimator_或在嵌套交叉验证模式下每个外部分割的cv_results_列表。对于深度学习模型，通过检查模型输入下方的布尔选项，输出分为两部分：模型框架和权重。不支持为嵌套交叉验证保存深度学习模型。",
    "Default(=blank): n_iter=1000, responsive_thres=-1, resistant_thres=0, random_state=None. No double quotes": "默认（=空白）：n_iter=1000，responsive_thres=-1，resistant_thres=0，random_state=None。不使用双引号",
    "Float. default=0.001": "浮点数，默认值为0.001",
    "Float. default=0.1": "浮点数，默认值=0.1",
    "Float. default=0.7": "浮点数，默认值为0.7",
    "Float. default=-1": "浮点数，默认值为-1",
    "Optional. default: None.": "可选。默认值：无。",
    "If True, return the pairwise distance matrix, else return the componentwise L1 pairwise-distances.": "如果为真，返回成对距离矩阵，否则返回成对的L1逐元素距离。",
    "Range of parameter space to use by default for :meth ''radius_neighbors'' queries.": "默认情况下，用于:meth \"radius_neighbors\"查询的参数空间范围。",
    "The metric to use when calculating distance between instances in a feature array.": "在特征数组中计算实例之间距离时使用的度量标准。",
    "Floating point number for shrinking centroids to remove features.": "用于缩小质心以移除特征的浮点数。",
    "Used when updating the weights after each boosting iteration.": "在每次提升迭代后更新权重时使用。",
    "The alpha-quantile of the huber loss function and the quantile loss function": "Huber损失函数的α分位数和分位数损失函数",
    "Saves the preprocessor after fitting to the data. The preprocessor can then be passed to other tools and used in later operations.": "将拟合数据后的预处理器保存起来。然后，可以将预处理器传递给其他工具，并在后续操作中使用。",
    "If false, returns the number of correctly classified samples. Otherwise, returns the fraction of correctly classified samples.": "如果为假，返回正确分类的样本数量。否则，返回正确分类样本的比例。",
    "If false, returns the number of misclassifications. Otherwise, returns the fraction of misclassifications.": "如果为假，返回误分类的数量。否则，返回误分类的分数。",
    "This is useful in order to create lighter ROC curves.": "这有助于创建更轻量的ROC曲线。",
    "Log loss is undefined for p=0 or p=1, so probabilities are clipped to max(eps, min(1 - eps, p)).": "对数损失在 p=0 或 p=1 时未定义，因此概率被限制为 max(eps, min(1 - eps, p))。",
    "If true, returns the mean loss per sample. Otherwise, returns the sum of the per-sample losses.": "如果为真，则返回每个样本的平均损失。否则，返回每个样本损失的总和。",
    "Number of components for dimensionality reduction. ( always less than  n_classes - 1 )": "降维的组件数量。（总是小于 n_classes - 1）",
    "Compute class covariance matrix.": "计算类协方差矩阵。",
    "Covariance estimate regularizer.": "协方差估计正则化器。",
    "Compute class covariance matrixes.": "计算类协方差矩阵。",
    "Useful only if solver is liblinear.": "仅当求解器是liblinear时才有用。",
    "Works only for lbfgs solver.": "仅适用于lbfgs求解器。",
    "A grid of Cs values are chosen in a logarithmic scale between 1e-4 and 1e4. Like in support vector machines, smaller values specify stronger regularization.": "在对数刻度上选择一系列Cs值，介于1e-4和1e4之间。与支持向量机一样，较小的值指定更强的正则化。",
    "If not set, the default cross-validation generator (Stratified K-Folds) is used.": "如果没有设置，默认使用交叉验证生成器（分层K折）。",
    "Whether to include train scores.": "是否包含训练分数。",
    "Controls the number of jobs that get dispatched during parallel execution": "控制并行执行期间分派的作业数量",
    "Relative or absolute numbers of training examples that will be used to generate the learning curve. Supports 1) tuple, to be evaled by np.linspace, e.g. (0.1, 1.0, 5); 2) array-like, e.g. [0.1  , 0.325, 0.55 , 0.775, 1.]": "用于生成学习曲线的训练样本的相对或绝对数量。支持1）元组，由 np.linspace 计算，例如 (0.1, 1.0, 5)；2）数组类型，例如 [0.1, 0.325, 0.55, 0.775, 1.]",
    "Whether to apply incremental learning to speed up fitting of the estimator if supported": "是否应用增量学习来加速拟合估计器（如果支持）",
    "Whether to shuffle training data before taking prefixes": "是否在取前缀之前对训练数据进行洗牌",
    "Number of times to permute y": "对 y 进行排列的次数",
    "For binary classification only. If int, like 5, will convert to class_weight {0: 1, 1: 5}. If float, 0.2, corresponds to class_weight {0: 1/0.2, 1: 1}": "对于二元分类。如果为整数，如5，将转换为类别权重 {0: 1, 1: 5}。如果为小数，如0.2，对应于类别权重 {0: 1/0.2, 1: 1}",
    "Prediction steps. Optional. If None, it equals number of samples divided by `batch_size`.": "预测步骤。可选。如果没有指定，则等于样本数除以`batch_size`。",
    "Integer or blank for None. Warning: when random seed is set to an integer, training will be running in single thread mode, which may cause slowness.": "整数或留空表示无。警告：当随机种子设置为整数时，训练将在单线程模式下运行，这可能会导致速度变慢。",
    "Will be ignored if `validation_data` is set explicitly, such as in `Deep learning training and evaluation` tool.": "如果明确设置了`validation_data`，例如在`深度学习训练和评估`工具中，将被忽略。",
    "Tuple of integers or None. Required for the first layer.": "整数元组或None。对于第一层是必需的。",
    "List of integers. Find index number at the left top corner of layer configuration block.": "整数列表。找到层配置块左上角的索引号。",
    "If your model of interest is not listed, contact the Galaxy team": "如果您感兴趣的模型未列出，请联系Galaxy团队",
    "Do not predict contigs shorter than this value. Default: 750": "不要预测短于这个值的基因组片段。默认值：750",
    "Dataset containing the features of samples": "包含样本特征的数据集",
    "The tool can either only create a latent representation of the data or create a latent representation of the data and cross validate a classifier using that encoding.": "该工具可以仅创建数据的潜在表示，或者创建数据的潜在表示并使用该编码进行交叉验证分类器。",
    "The type of representation learning": "表示学习类型",
    "Dataset containing the class labels corresponding to the features": "包含与特征相对应的类别标签的数据集",
    "The type of representation learning. `Train on input` trains the classifier on the input features without representation learning": "表示学习类型。`Train on input` 在输入特征上训练分类器，而不进行表示学习",
    "Defines the neighborhood (in pixels) where labels are considered to be adjacent.": "定义了在像素中被认为是相邻的标签的邻域。",
    "Input dataset with features on columns and observations on rows. The first column must contain the observation IDs, while the last column must contain classes. The header line is also required.": "输入数据集，特征在列上，观测在行上。第一列必须包含观测ID，最后一列必须包含类别。还需要包含标题行。",
    "Size of hypervectors is usually 10,000 in vector-symbolic architectures. However, lower values could work with small datasets in terms of number of features and observations. Please note that you may require to increase this number in case of datasets with a huge number of features.": "在向量符号架构中，超向量的大小通常是10,000。然而，在特征和观测数量较少的小型数据集上，较低的值也可以工作。请注意，如果数据集具有大量特征，您可能需要增加此数字。",
    "Number of level vectors. You may consider to look at the distribution of your data in order to choose the most appropriate value.": "水平向量的数量。您可以考虑查看您的数据分布，以选择最合适的值。",
    "Maximum number of retraining iterations. Class hypervectors are retrained to minimize errors caused by noise.": "最大重训练迭代次数。类别超向量被重新训练以最小化由噪声引起的错误。",
    "This tool makes use of k-folds cross-validation to evaluate the accuracy of the hyperdimensional model. Make sure to choose a good number of folds for validating the classification model. Please note that higher number of folds could significantly increase the running time.": "这个工具使用k折交叉验证来评估超维模型的准确性。请确保选择一个合适的折数来验证分类模型。请注意，较高的折数可能会显著增加运行时间。",
    "If selected, this will extract a set of features with the better discriminative power among classes. The feature selection algorithm is defined as a backward variable selection method.": "如果选中，这将提取一组在类别之间具有更好区分能力的特征。特征选择算法被定义为一种向后变量选择方法。",
    "Tool will stop removing features if its number will reach this value.": "工具将在达到此数值时停止移除特征。",
    "Stop the execution if the best accuracy reached for a group of features is lower than this value.": "如果一组特征达到的最佳准确度低于此值，则停止执行。",
    "Consider non optimal solutions if model accuracy is greater than the best accuracy minus this percentage.": "考虑非最优解，如果模型准确度大于最佳准确度减去这个百分比。",
    "The interval metadata for this file should be set for the codon positions.": "该文件的区间元数据应设置为密码子位置。",
    "IQ-TREE will fail if given too many threads on input data that consists of short alignments, this option will force the use of a single thread.": "IQ-TREE在处理由短序列组成的输入数据时，如果给定了太多线程，将会失败，这个选项将强制使用单个线程。",
    "By default: IQ-TREE will remove them during the analysis and add them in the end.": "默认情况下：IQ-TREE在分析过程中会移除它们，并在最后添加它们。",
    "This mode is automatically turned on when having more than 2000 sequences.": "当序列数量超过2000时，这种模式会自动开启。",
    "If you specify -lmap ALL, all unique quartets will be drawn, instead.": "如果您指定 -lmap ALL，那么将绘制所有唯一的四重奏，而不是其他选项。",
    "See http://www.iqtree.org/doc/Substitution-Models": "参见 http://www.iqtree.org/doc/Substitution-Models",
    "This is similar to --rcluster-percent option of PartitionFinder. For example, giving the value 10 only the top 10% partition schemes are considered to save computations.": "这类似于PartitionFinder的--rcluster-percent选项。例如，给定值10，只考虑前10%的分区方案以节省计算。",
    "Alternatively, one can specify a comma-separated list of base models. For example, -mset WAG,LG,JTT will restrict model selection to WAG, LG, and JTT instead of all 18 AA models to save computations.": "另外，可以指定一个由逗号分隔的基线模型列表。例如，-mset WAG,LG,JTT 将限制模型选择为 WAG、LG 和 JTT，而不是所有 18 个 AA 模型，以节省计算。",
    "For example, -madd LG4M,LG4X to additionally include these two protein mixture models.": "例如，-madd LG4M,LG4X 来额外包含这两个蛋白质混合模型。",
    "Note that for models like LG+C20+F+G this mode is automatically turned on, but not for LG+C20+G.": "请注意，对于像LG+C20+F+G这样的模型，这种模式会自动开启，但对于LG+C20+G则不会。",
    "Use a negative value for complement": "使用负值表示补码",
    "the .sitefreq file obtained from -ft run. This will save memory used for the first phase of the analysis.": "从 -ft 运行获得的 .sitefreq 文件。这将节省分析第一阶段使用的内存。",
    "Default: posterior mean.": "默认值：后验均值。",
    "The constraint tree can be a multifurcating tree and need not to include all taxa.": "约束树可以是一个多分支树，并且不必包括所有分类群。",
    "If number of replicates is set to 0 (-alrt 0), then the parametric aLRT test (Anisimova and Gascuel 2006) is performed, instead of SH-aLRT.": "如果重复次数设置为0（-alrt 0），那么将执行参数化aLRT测试（Anisimova和Gascuel 2006），而不是SH-aLRT。",
    "IQ-TREE will compute the log-likelihoods of all trees.": "IQ-TREE 将计算所有树的对数似然。",
    "The tests include bootstrap proportion (BP), KH test (Kishino and Hasegawa, 1989), SH test (Shimodaira and Hasegawa, 1999) and expected likelihood weights (ELW) (Strimmer and Rambaut, 2002).": "这些测试包括自助法比例（BP）、KH检验（Kishino和Hasegawa，1989年）、SH检验（Shimodaira和Hasegawa，1999年）以及期望似然权重（ELW）（Strimmer和Rambaut，2002年）。",
    "Note that you have to specify the number of replicates for the AU test via -test.": "请注意，您必须通过-test参数指定AU测试的重复次数。",
    "Resulting consensus tree is written to .contree file.": "生成的一致性树被写入到.contree文件中。",
    "Resulting consensus network is written to .nex file.": "生成的共识网络写入到 .nex 文件中。",
    "This is useful e.g. when summarizing trees from MrBayes analysis.": "这在例如从MrBayes分析中总结树时非常有用。",
    "That means, support values are first extracted from the trees passed via -t, and then mapped onto the target tree. Resulting tree with assigned support values is written to .suptree file. This option is useful to map and compare support values from different approaches onto a single tree.": "这意味着，首先从通过-t参数传递的树中提取支持值，然后将这些支持值映射到目标树上。具有分配支持值的结果树被写入到.suptree文件中。这个选项对于将不同方法的支持值映射并比较到一个单独的树上非常有用。",
    "The corresponding node of .suptree will then be assigned with IDs of trees where this node appears. Special option --suptag ALL will assign such IDs for all nodes of the target tree.": "相应的.suptree节点将被赋予在该节点出现的树的ID。特殊选项--suptag ALL将为目标树的所有节点分配这样的ID。",
    "IQ-TREE will create a random tree under Yule-Harding model with specified number of taxa": "IQ-TREE将根据Yule-Harding模型创建一个具有指定数量的分类群的随机树",
    "Default: 0.5": "默认值：0.5",
    "By default, IQ-TREE resamples alignment sites within partitions. With --sampling GENE IQ-TREE will resample partitions. With --sampling GENESITE IQ-TREE will resample partitions and then resample sites within resampled partitions (Gadagkar et al., 2005).": "默认情况下，IQ-TREE会在分区内部对序列比对位点进行重采样。使用--sampling GENE参数，IQ-TREE会重采样分区。使用--sampling GENESITE参数，IQ-TREE会先重采样分区，然后在重采样的分区内重采样位点（Gadagkar等人，2005年）。",
    "This option is recommended in the presence of severe model violations. It increases computing time by 2-fold but reduces the risk of overestimating branch supports due to severe model violations. Introduced in IQ-TREE 1.6.": "这个选项建议在模型严重违反的情况下使用。它将计算时间增加了2倍，但减少了由于模型严重违反而导致过度估计分支支持的风险。在IQ-TREE 1.6中引入。",
    "This will perform both bootstrap and analysis on original alignment and provide a consensus tree.": "这将对原始比对执行引导和分析，并提供一个共识树。",
    "The number of entries should be equal to the number of states in the model (e.g. 4 for DNA and 20 for protein). IQ-TREE will then add a number of constant sites accordingly. For example, -fconst 10,20,15,40 will add 10 constant sites of all A, 20 constant sites of all C, 15 constant sites of all G and 40 constant sites of all T into the alignment.": "条目数量应等于模型中的州数（例如，DNA为4，蛋白质为20）。IQ-TREE将相应地添加一定数量的恒定位点。例如，-fconst 10,20,15,40 将向比对中添加10个全为A的恒定位点，20个全为C的恒定位点，15个全为G的恒定位点和40个全为T的恒定位点。",
    "The fasta file with the sequences to be analyzed": "待分析序列的fasta文件",
    "Tree file in Newick format": "新克格式的树文件",
    "Used in combination with option G in the sequence data file, for combined analysis of data from multiple genes or multiple site partitions": "在序列数据文件中与选项G结合使用，用于对多个基因或多个位点分区的数据进行联合分析",
    "kappa refers to the transition/tranversion rate ratio": "kappa指的是转换/颠换率比率",
    "0: constant rate. fix_alpha !=1 and alpha !=0 are not compatible with NSsites !=0": "0：恒定速率。fix_alpha !=1 和 alpha !=0 与 NSsites !=0 不兼容",
    "fix_rho=1 and rho=0 : independent rates": "fix_rho=1 和 rho=0：独立速率",
    "Warning : choosing 'yes' may remove a lot (possibly all) of data.": "警告：选择“是”可能会删除大量（可能全部）数据。",
    "Note that the runs are sequential. Please launch separate runs if it's too long": "请注意，这些运行是连续的。如果太长，请分别启动运行。",
    "or [K]": "或者 [K]",
    "or [N]": "或者 [N]",
    "or [L]": "或 [L]",
    " E.g., for diploid data, this would mean that the two alleles for each locus are in consecutive order in the same row, rather than being arranged in the same column, in two consecutive rows": "例如，对于二倍体数据，这意味着每个位点的两个等位基因会按照连续的顺序在同一行中排列，而不是在两行连续的列中排列。",
    "For use with polyploids when RECESSIVEALLELES=1/True. Must not match MISSING or any allele value in the data.": "用于多倍体时，当RECESSIVEALLELES=1/True。不能匹配数据中的MISSING或任何等位基因值。",
    "(0/False=admixture model, 1/True=no-admix)": "(0/False=混合模型，1/True=无混合)",
    "[mainparams] when USEPOPINFO is 1/True": "[主要参数] 当 USEPOPINFO 为 1/True 时",
    "(used only when USEPOPINFO==1/True). This should be small, eg 0.01 or 0.1.": "（仅当USEPOPINFO==1/True时使用）。这个值应该很小，例如0.01或0.1。",
    "this is the initial value if INFERALPHA is 1/True.": "如果 INFERALPHA 是 1/True，这是初始值。",
    "(only if INFERLAMBDA=1/True).": "（仅当 INFERLAMBDA=1/True 时）。",
    "(only if UNIFPRIORALPHA==0/False)": "（仅当 UNIFPRIORALPHA==0/False 时）",
    "eg, if GENSBACK==2, it tests for immigrant ancestry back to grandparents.": "例如，如果 GENSBACK==2，它测试的是追溯到祖父母的移民祖先。",
    "This is to enable use of a reference set of individuals for clustering additional 'test' individuals.": "这是为了启用使用一组参考个体来对额外的“测试”个体进行聚类。",
    "(Linkage model only) This is a large file!": "（仅限连锁模型）这是一个大文件！",
    "Turn this on before using STRAT.": "在使用STRAT之前，请先开启这个选项。",
    "Set automatically if this is 0/False.": "如果这个值为0/False，则自动设置。",
    "This is used when choosing the best number of subpopulations.": "这用于选择最佳的亚群数量。",
    "[only relevant for linkage model] see Documentation": "[仅与连锁模型相关] 请参阅文档",
    "(Need POPDATA==1). It is assumed that the PopData in the input file are between 1 and k where k is less or equal MAXPOPS.": "（需要POPDATA==1）。假设输入文件中的PopData在1到k之间，其中k小于或等于MAXPOPS。",
    "(must set RANDOMIZE=0)": "（必须设置 RANDOMIZE=0）",
    "(ie use the metr. move every i steps).  If this is set to 0, it is never used. (Proposal for each q^(i) sampled from prior. The goal is to improve mixing for small alpha.)": "（即使用metropolis准则。每i步移动一次）。如果这个设置为0，则从未使用过。（对于从先验中采样的每个q^(i)提出的建议。目标是改善小alpha值下的混合效果。）",
    "Specifies which sequence evolution method to use when computing distance estimates from multiple alignments, using Kimura's model by default.": "指定在从多个比对计算距离估计时使用哪种序列进化方法，默认使用Kimura模型。",
    "The output tree will be annotated with bootstrap values": "输出的树将带有自举值注释",
    "Sequences which lie within this distance will be clustered": "位于此距离内的序列将会被聚类",
    "\"All\": Each sequence within the cluster must be within the specified distance of every other sequence in the cluster. \"Any\": Each sequence within the cluster must be within the specified distance of at least one other sequence.": "\"全部\"：聚类内每个序列必须与聚类中其他序列保持指定距离。\"任意\"：聚类内每个序列必须与至少一个其他序列保持指定距离。",
    "This tool is only available for fasta input only; FastTree will not be able to read these trees back in.": "这个工具只适用于fasta输入；FastTree将无法读取这些树。",
    "The tree produced by the full alignment to be used as a starting point for each bootstrapping replicate. This will save considerable computation time.": "由完整对齐生成的树将用作每个自举复制的起始点。这将节省大量的计算时间。",
    "Specify if the aligned sequences are nucleotide or protein sequences": "指定对齐的序列是核苷酸序列还是蛋白质序列",
    "After optimizing the tree under the CAT approximation, rescale the lengths to optimize the Gamma20 likelihood.": "在CAT近似下优化树后，重新调整长度以优化Gamma20似然。",
    "Recommended for highly gapped/fragmentary sequences.": "推荐用于高度间隙/片段化的序列。",
    "What the height of each logo element depicts - eg bits of entropy (default)": "每个Logo元素的高度所表示的含义 - 例如熵的位数（默认）",
    "Note that some of these only make sense for protein sequences!": "请注意，这些中的一些仅对蛋白质序列有意义！",
    "WARNING: Specifying indexes outside the sequence lengths will cause unpredictable but bad consequences!": "警告：指定超出序列长度的索引将导致不可预测但不良的后果！",
    "Recommended for protein": "推荐用于蛋白质",
    "Select 'No' to only consider triples starting at the start of the site": "选择“否”以仅考虑从位点开始的三元组",
    "Select 'No' to choose the width with the biggest maximum score (before scaling)": "选择'No'以选择具有最大最大分数（在缩放之前）的宽度",
    "DREME or MEME output XML file containing found motifs": "DREME 或 MEME 输出的 XML 文件，包含找到的基序",
    "Search for motifs also on reverse complement strand": "在反转互补链上也搜索基序",
    "Only output results with a p-value less than the given threshold. To apply set threshold to q-values rather than p-values, set --qv-thresh.": "只输出 p 值小于给定阈值的结果。要将阈值应用于 q 值而非 p 值，设置 --qv-thresh。",
    "Default: apply set threshold on p-values": "默认：对P值应用设定阈值",
    "File must be in Markov background model format (see MEME suite tool fasta-get-markov for details)": "文件必须采用马尔可夫背景模型格式（详见MEME套件工具fasta-get-markov获取详细信息）",
    "If matches on both strands at a given position satisfy the output threshold, only report the match for the strand with the higher score. If the scores are tied, the matching strand is chosen at random (default: report both matches).": "如果给定位置上两个链上都有匹配，并且满足输出阈值，则只报告得分更高的链上的匹配。如果得分相同，则随机选择匹配的链（默认：报告两个匹配）。",
    "Keeping a complete list of scores may exceed available memory. Once the number of stored scores reaches the maximum allowed, the least significant 50% of scores will be dropped. In this case, the list of reported motifs may be incomplete and the q-value calculation will be approximate.": "保持一个完整的分数列表可能会超出可用内存。一旦存储的分数数量达到最大允许值，最不重要的50%的分数将被丢弃。在这种情况下，报告的基序列表可能是不完整的，q值计算将是近似的。",
    "A pseudocount to be added to each count in the motif matrix, after first multiplying by the corresponding background frequency": "在模式矩阵中，每个计数乘以相应的背景频率后，要添加的伪计数",
    "The q-value calculation is that of Benjamini and Hochberg (1995) (default: calculate q-value for each p-value)": "q值计算是Benjamini和Hochberg（1995年）的方法（默认：为每个p值计算q值）",
    " If genomic coordinates are found they will be used as the coordinates in the output.": "如果发现基因组坐标，它们将被用作输出中的坐标。",
    "File should be in MEME PSP format or wiggle format (can be generated by MEME suite tool create-priors)": "文件应该是MEME PSP格式或wiggle格式（可以通过MEME套件工具create-priors生成）",
    "Represents the fraction of all transcription factor binding sites that are binding sites for the TF of interest (must be between 0 and 1)": "表示所有转录因子结合位点中，对感兴趣转录因子（TF）的结合位点所占的比例（必须在0到1之间）",
    "This file can be generated using the MEME suite tool create-priors": "这个文件可以使用MEME套件工具create-priors生成",
    "FIMO HTML output file": "FIMO HTML输出文件",
    "FIMO XML output file": "FIMO XML输出文件",
    "FIMO GFF output file": "FIMO GFF 输出文件",
    "positive (primary) sequence file": "正向（主要）序列文件",
    "(default: 1)": "（默认值：1）",
    "negative (control) sequence file (default: positive sequences are shuffled to create the negative set)": "负对照序列文件（默认：正对照序列被打乱以创建负集）",
    "stop if motif E-value > given threshold (default: 0.05)": "如果基序的E值大于给定的阈值（默认：0.05）则停止",
    "minimum width of core motif (default: 3)": "核心基序的最小宽度（默认：3）",
    "maximum width of core motif (default: 8)": "核心基序的最大宽度（默认：8）",
    "stop if m motifs have been output (default: only stop at E-value threshold)": "如果输出了m个基序（默认：只在E值阈值时停止）",
    "stop if the specified time has elapsed (default: only stop at E-value threshold)": "如果在指定的时间过去后停止（默认：只在E值阈值时停止）",
    "number of REs to generalize (default: 100) (Hint: Increasing n will make the motif search more thoroughly at some cost in speed)": "REs泛化的数量（默认：100）（提示：增加n将使基序搜索更加彻底，但会以牺牲速度为代价）",
    "Nucleotide sequences must have equal length": "核苷酸序列必须长度相等",
    "If no selection, positive sequences in the input are shuffled to create the negative set": "如果没有选择，输入中的正序列将被随机打乱以创建负集。",
    "Select 'No' if your input sequences are sorted in order of confidence (best to worst)": "如果输入序列是按照置信度（从最好到最差）排序的，请选择“否”。",
    "Zero value indicates the sequences should not be cut down": "零值表示序列不应被截断",
    "Zero value results in 2*primary threshold": "零值结果导致2倍主要阈值",
    "The fasta is available separately in the cDNA sequence file, but some tools might prefer it included in the GFF3 file.": "fasta文件在cDNA序列文件中单独可用，但某些工具可能更倾向于将其包含在GFF3文件中。",
    "By default it will just print a warning message, this causes any error to be fatal.": "默认情况下，它只会打印一条警告消息，这会导致任何错误都是致命的。",
    "Select this if you want Apollo to reload the genome sequences when updating an existing organism. BEWARE! You will lose annotation data for all sequences that are not present anymore in the new jbrowse dataset.": "如果希望在更新现有生物体时让Apollo重新加载基因组序列，请选择此选项。注意！您将丢失新jbrowse数据集中不再存在的所有序列的注释数据。",
    "It will PERMANENTLY delete all of the features on this organism.": "它将永久删除这个生物体上的所有特征。",
    "There's NO coming back from this.": "没有回头路可走。",
    "A value of zero will include all interaction distances that fall between the start and stop coordinates.": "值为零将包括所有落在起始和终止坐标之间的相互作用距离。",
    "Filter out SNPs with qualities lower than selected value": "过滤掉质量低于选定值的SNPs",
    "This option takes three values: the info string tag, the cutoff value and whether to filter out those records with less than (lt) or greater than (gt) this value.  For example: DP 10 lt": "这个选项需要三个值：信息字符串标签、截止值以及是否过滤掉那些记录，这些记录的值小于（lt）或大于（gt）这个截止值。例如：DP 10 lt",
    "Target data bedgraph file in UCSC bedgraph format that omits regions containing 0 signal.": "目标数据床图文件采用UCSC床图格式，该格式省略了包含0信号的区域。",
    "Control (IgG) data bedgraph file to generate an empirical threshold for peak calling.": "控制（IgG）数据 bedgraph 文件，用于生成用于峰值检测的经验阈值。",
    "A numeric threshold n between 0 and 1 returns the top n fraction of peaks based on total signal within peaks.": "一个介于0和1之间的数值阈值n，根据峰值内的总信号量返回前n分数的峰值。",
    "'norm' denotes normalization of control to target data, 'non' skips this behavior. 'norm' is recommended unless experimental and control data are already rigorously normalized to each other (e.g. via spike-in).": "'norm' 表示将对照数据标准化至目标数据，'non' 跳过这种行为。除非实验数据和对照数据已经通过严格标准化（例如，通过添加内标）进行了标准化，否则推荐使用 'norm'。",
    "'relaxed' uses a total signal threshold between the knee and peak of the total signal curve, and corresponds to the 'relaxed' mode described in the text, whereas 'stringent' uses the peak of the curve, and corresponds to 'stringent' mode.": "“relaxed”使用总信号曲线的拐点和峰值之间的总信号阈值，并对应于文本中描述的“relaxed”模式，而“stringent”使用曲线的峰值，并对应于“stringent”模式。",
    "Cached FASTA": "缓存的FASTA",
    "The seed size for the HASH table. BSMAP implements a seeding algorithm by indexing reference for all possible k-mers, i.e. seeds. As for the seed size, i.e. the length of k-mers, 16 is suggested for the WGBS mode, and 12 is suggested for the RRBS mode. Min=8, max=16.": "HASH表的种子大小。BSMAP通过为所有可能的k-mers（即种子）索引参考实现一种播种算法。就种子大小而言，即k-mers的长度，建议WGBS模式使用16，建议RRBS模式使用12。最小值为8，最大值为16。",
    "When `Do not set` selected, BSMAP will allow a default suggested 8% mismatch rate. Otherwise, a customized mismatch can be controlled by specifying a mismatch rate or a mismatch number.": "当选择“不设置”时，BSMAP将允许一个默认建议的8%的错配率。否则，可以通过指定一个错配率或一个错配数来控制自定义的错配。",
    "The mismatch rate w.r.t to the read length": "与读取长度相关的不匹配率",
    "The maximum number of mismatches allowed on a read": "允许在一个读取上的最大不匹配数",
    "Maximum number of equal best hits to count. When multiple mapping occurs for a read, it should control the number of records to report. Default: 1000.": "最大相同最佳命中数。当一个读取有多于一个的映射时，它应该控制报告的记录数。默认值：1000。",
    "For RRBS data, this option activates RRBS mapping mode and set restriction enzyme digestion sites. Digestion position marked by '-', example: -D C-CGG for MspI digestion.  default: none (WGBS mode).": "对于RRBS数据，这个选项激活RRBS映射模式并设置限制性内切酶切割位点。消化位置用'-'标记，例如：-D C-CGG用于MspI消化。默认值：无（WGBS模式）。",
    "Seed for random number generation used in selecting multiple hits. Other seed values generate pseudo random number based on read index number, to allow reproducible mapping results. Default=0 (get seed from system clock, mapping results not resproducible).": "用于选择多个命中时随机数生成的种子。其他种子值基于读取索引号生成伪随机数，以允许可重现的映射结果。默认值=0（从系统时钟获取种子，映射结果不可重现）。",
    "Yes: map SE or PE reads to all 4 strands, i.e. ++, +-, -+, --; No: only map to 2 forward strands, i.e. BSW(++) and BSC(-+). For example, for a traditional library construction, two forward strands, ++ and -+, are sufficient for alignments. However, a Pico library construction needs all four-strand mappings.": "是：将单端(SE)或双端(PE)读取映射到所有4个链上，即 ++, +-, -+, --; 否：仅映射到2个正向链上，即 BSW(++) 和 BSC(-+)。例如，对于传统的文库构建，两个正向链 ++ 和 -+ 就足以进行比对。然而，Pico 文库构建需要所有四个链的映射。",
    "The quality threshold to trim read bases. To obtain an accurate mapping, low-quality bases should be skipped beforehand. Min=0, max=40. Default=0 (no trim).": "用于修剪读取碱基的质量阈值。为了获得准确的映射，应该事先跳过低质量的碱基。最小值为0，最大值为40。默认值为0（不修剪）。",
    "Base quality for sequencing reads, Illumina or Sanger.": "测序读取的碱基质量，Illumina或Sanger。",
    "To filter out low-quality reads containing >n Ns. Default=5.": "为了过滤掉含有多于n个N的低质量读取。默认值=5。",
    "To trim 3' adapter sequence. Default: none (no trim).": "为了去除3'端的接头序列。默认值：无（不去除）。",
    "0=none(unique hit/pair); 1=random one; 2=all(slow). When input reads coverage is high, it is suggested to report only unique hits (r=0). For a low-depth library, a random one from multiple mappings (r=1), or all multiple mappings (r=2) can be specified to increase read coverage, yet be cautious about bias caused by ambiguous mappings.": "0=无（唯一命中/配对）；1=随机一个；2=全部（慢）。当输入读取的覆盖度很高时，建议只报告唯一命中（r=0）。对于低深度文库，可以指定从多个映射中随机选择一个（r=1），或者指定所有多个映射（r=2）以增加读取覆盖度，但要注意由不明确的映射引起的偏差。",
    "Yes: print corresponding reference sequences in mapping records, a `RS:` tag will be added in record attributes; No: do not print reference sequences.": "是：在映射记录中打印相应的参考序列，记录属性中将添加一个`RS:`标签；否：不打印参考序列。",
    "Yes: print unmapped reads; No: do not print unmapped reads.": "是：打印未比对的序列；否：不打印未比对的序列。",
    "For paired-end reads, the minimal insert size allowed in two end mapping. Default=28.": "对于成对末端测序，两个末端映射时允许的最小插入片段大小。默认值=28。",
    "For paired-end reads, the maximal insert size allowed in two end mapping. Default=1000.": "对于成对末端测序，两个末端映射时允许的最大插入片段大小。默认值=1000。",
    "Threshold for cytosine quality score. Discard the base if threshold is not reached. Default=20.": "胞嘧啶质量分数的阈值。如果未达到阈值，则丢弃该碱基。默认值=20。",
    "Threshold for the next base quality score. Possible values: -1 makes the program not to check if next base matches reference; any positive integer or zero makes the program to check if next base matches reference and reaches this score threshold; default=3, i.e., better than 'B' or '#'.": "下一个碱基的质量分数阈值。可能的值：-1 使程序不检查下一个碱基是否与参考匹配；任何正整数或零使程序检查下一个碱基是否与参考匹配并达到此分数阈值；默认值为3，即比 'B' 或 '#' 更好。",
    "Sanger, Solexa, or Illumina. See wiki FASTQ_format for details. Default: auto-detection.": "Sanger、Solexa 或 Illumina。详细信息请参见维基百科的 FASTQ_format 页面。默认：自动检测。",
    "To trim end-repair sequence from begin of +-/-- reads from Pair End WGBS Sequencing. 0: no trim; n (positive integer): trim n bases from begin of +-/-- reads. Default: trim 3 bases.": "从成对末端WGBS测序的+-/--读取的开始处修剪末端修复序列。0：不修剪；n（正整数）：从+-/--读取的开始处修剪n个碱基。默认：修剪3个碱基。",
    "To trim end-repair sequence from end of ++/-+ reads from Pair End WGBS Sequencing. 0: no trim; n (positive integer): trim n bases from end of ++/-+ reads. Default: trim 3 bases.": "从成对末端WGBS测序的++/-+读段末端修剪末端修复序列。0：不进行修剪；n（正整数）：从++/-+读段末端修剪n个碱基。默认：修剪3个碱基。",
    "Two ends of paired-end reads may be overlapped in mapping. The overlap sequencce will be counted once or twice for cytosine methylation measurements. Default: once.": "成对序列读取的两端在映射时可能会重叠。重叠的序列将被计算一次或两次以测量胞嘧啶甲基化。默认值：一次。",
    "To trim end-repair sequence for RRBS reads. 0: no trim; 2: trim the last CG at exactly end of ++/-+ reads and trim the first CG at exactly begin of +-/-- reads like the WGBS situation. Default=2.": "为了修剪RRBS读取的末端修复序列。0：不修剪；2：修剪++/-+读取的末端精确的最后一个CG，并修剪+-/--读取的起始精确的第一个CG，类似于WGBS的情况。默认值=2。",
    "To retain properly mapped and large enough fragment sizes. The 9th field in the BAM file is the fragment size of the mapping, and non-properly-paired read has 0 at the 9th field. This option is set to require properly paired and large enough fragment size. Default=0 for all records.": "为了保留正确映射且足够大的片段大小。BAM文件中的第9个字段是映射的片段大小，非正确配对的读取在第9个字段处为0。此选项设置为要求正确配对且片段大小足够大。默认值=所有记录均为0。",
    "Same as --minFragSize but this option is only applicable to reads with flag 0x100 set as 1, i.e., reads multiply mapped. Default=0 for all records.": "与--minFragSize相同，但此选项仅适用于将标志位0x100设置为1的读取，即多重映射的读取。默认值为所有记录为0。",
    "To generate methylation for CpG, or CpA/CpC/CpT. Default=CpG.": "为了生成CpG的甲基化，或者CpA/CpC/CpT的甲基化。默认=CpG。",
    "Yes: compare; No: do not compare, using the comparison result by `-c`": "是：比较；否：不比较，使用 `-c` 的比较结果",
    "Previously generated comparison file from history": "之前从历史记录生成的比较文件",
    "If a site has depth less than `d`, this site is ignored for statistical tests. This option affects much of nominal ratios but none of credible ratios. This option may be reset during later DMC/DMR rescan to filter sites with depth less than `d`. Default=3.": "如果一个位点的深度小于 `d`，则在统计测试中忽略该位点。这个选项影响了许多名义比率，但没有影响可信比率。在后续的 DMC/DMR 重新扫描期间，这个选项可能会被重置，以过滤深度小于 `d` 的位点。默认值=3。",
    "If absolute value of CDIF for a site less than filterCredibleDif, this site is ignored for regional calculation. Use a small value, such as 0.01, to filter all sites with no difference; use 0.20 (for example) to select DMCs. Any negative number means no filter. Default=-10.": "如果某个位点的CDIF绝对值小于filterCredibleDif，则在区域计算中忽略该位点。使用较小的值，例如0.01，可以过滤掉没有差异的所有位点；使用0.20（例如）可以选择DMC。任何负数都表示不使用过滤器。默认值=-10。",
    "Cutoff of P value from Fisher Exact Test for DMC scan. Default=`0.05`.": "费舍尔精确检验的P值截止值，用于DMC扫描。默认值为`0.05`。",
    "Cutoff of P value from Fisher Exact Test for DMR scan. Default=`0.05`.": "费舍尔精确检验用于差异甲基化区域扫描的P值截断值。默认值=`0.05`。",
    "Minimal nominal methylation difference for DMC and DMR. Default=`0.3333`.": "最小名义甲基化差异用于DMC（差异甲基化胞嘧啶）和DMR（差异甲基化区域）。默认值=`0.3333`。",
    "Minimal CDIF for DMC calling. Default=`0.2`.": "最小CDIF用于DMC调用。默认值=`0.2`。",
    "Minimum number of DMCs in a DMR. Default=3.": "在DMR中DMC的最小数量。默认值=3。",
    "Maximum distance between two consective DMCs for a DMR. Default=300.": "两个连续的DMC（DNA甲基化中心）之间的最大距离，用于定义一个DMR（DNA甲基化区域）。默认值=300。",
    "Red .IDAT files extension is followed by the unmethylated signal intensity read in the red channel.": "红色.Illumina IDAT文件扩展名后跟的是红色通道读取的未甲基化信号强度。",
    "Green .IDAT files extension is followed by the methylated signal intensity read in the green channel.": "绿色.IDAT文件扩展名后面跟随的是绿色通道中读取的甲基化信号强度。",
    "Mapping Ilumina methylation array data to the genome with or without additional preprocessing methods.": "将Illumina甲基化芯片数据映射到基因组上，可以采用预处理方法，也可以不采用预处理方法。",
    "Phenotype Table must include the following information: sampleID, phenotype and paird or unpaired samples column.": "表型表必须包含以下信息：样本ID、表型以及配对或非配对样本列。",
    "If cluster is not provided this maximum location gap will be used to define cluster.": "如果未提供聚类，则将使用此最大位置间隔来定义聚类。",
    "A numeric value. Values of the estimate of the genomic profile above the cutoff or below the negative of the cutoff will be used as candidate regions. It is possible to give two separate values (upper and lower bounds). If one value is given, the lower bound is minus the value.": "一个数值。如果基因组轮廓估计值高于截止值或低于截止值的负数，则将其用作候选区域。可以给出两个独立的值（上界和下界）。如果给出一个值，则下界是该值的负数。",
    "An integer denoting the number of resamples to use when computing null distributions. This defaults to 0. If permutations is supplied that defines the number of permutations/bootstraps and B is ignored.": "一个整数，表示在计算空分布时使用的重采样次数。默认为0。如果提供了排列，则定义排列/自举的数量，B将被忽略。",
    "Method used to generate null candidate regions (defaults to ‘permutation’). Note that for cases with more than one covariate the permutation approach is not generally recommended.": "用于生成零假设候选区域的方法（默认为‘permutation’）。请注意，对于具有多个协变量的情况，通常不推荐使用排列方法。",
    "Table of compared samples and their characteristics, may be categorical (e.g. cancer vs. normal) or continuous (e.g. blood pressure).": "比较样本及其特征的表格，可以是分类的（例如癌症与正常）或连续的（例如血压）。",
    "DMPs with an FDR q-value greater than this will not be returned.": "具有大于此FDR q值的DMP将不会被返回。",
    "Enable variance shrinkage is recommended when sample sizes are small.": "当样本量较小时，推荐启用方差收缩。",
    "Reference Sequence e.g. wgEncodeHaibMethyl450Gm12878SitesRep1.": "参考序列，例如wgEncodeHaibMethyl450Gm12878SitesRep1。",
    "Set of sorted bedgraph/bed input profiles of group 1": "一组排序后的bedgraph/bed输入文件集合",
    "Set of sorted bedgraph/bed input profiles of group 2": "第二组排序后的bedgraph/bed输入文件集",
    "Regions that should be checked for differential methylation.": "应该检查的区域以确定甲基化的差异。",
    "-M, --maxdist": "-M, --最大距离",
    "-m, --mincpgs": "-m, --最小胞嘧啶-磷酸-鸟嘌呤位点数",
    "to estimate missing ones": "估计缺失值",
    "-d, --minMethDiff": "-d, --minMethDiff",
    "-v, --valley": "-v, --valley",
    "Name of the genome to be digested.": "要被消化的基因组名称。",
    "Specify the genome digest file (created by hicup_digester)": "指定基因组消化文件（由 hicup_digester 创建）",
    "BED file with the genomic regions of interest. These regions will be automatically partitioned into smaller, consecutive bins. Only the first three fields in the file matter. If the region lengths are not multiples of the given binsize a new bed file will be produced where each coordinate is a multiple of binsize.": "感兴趣的基因组区域的BED文件。这些区域将自动划分为更小的连续箱。文件中只有前三个字段重要。如果区域长度不是给定箱大小的倍数，将产生一个新的BED文件，其中每个坐标都是箱大小的倍数。",
    "Size of a bin in base pairs. Each given region will be partitioned into bins of this size.": "每个给定区域将被划分为这个大小的基对数的箱子。",
    "Chromatin segmentation may be performed across one or more datasets (e.g. cell-types or conditions) each of which consisting of a set of chromatin marks (e.g. H3K4me3). If multiple datasets are used, they must share the same set of chromatin marks.": "染色质分割可以在一个或多个数据集（例如细胞类型或条件）上执行，每个数据集都包含一组染色质标记（例如H3K4me3）。如果使用多个数据集，它们必须共享相同的一组染色质标记。",
    "BAM file containing the read alignments for the corresponding chromatin mark. If the same mark name is used multiple times, the associated reads will be treated as replicates and collapsed into one experiment.": "包含相应染色质标记的读取比对的BAM文件。如果相同的标记名称被多次使用，相关读取将被视为重复项，并将合并为一个实验。",
    "Minimum mapping quality for the reads (see the bam format specification for the mapq field). Only reads with the mapq field above or equal to the specified value will be considered.": "对于读取的最小映射质量（参见 bam 格式规范中的 mapq 字段）。只有 mapq 字段高于或等于指定值的读取才会被考虑。",
    "Set this option to TRUE or FALSE to activate or deactivate the paired-end mode. Only read pairs where both ends are mapped will be considered and assigned to the bin where the midpoint of the read pair is located. If this flag is set, the `shift` option will be ignored.": "将此选项设置为TRUE或FALSE以激活或停用配对端模式。只有两端都映射的读对才会被考虑并分配到读对中点所在的bin中。如果设置了此标志，则将忽略`shift`选项。",
    "Shift the reads in the 5' direction by a fixed number of base pairs. The read will be assigned to the bin where the shifted 5' end of the read is located. This option will be ignored in paired-end mode.": "将读取的序列在5'方向上移动固定数量的碱基对。读取将被分配到移动后的5'端所在的bin中。在成对模式下将忽略此选项。",
    "Number of chromatin states.": "染色质状态的数量。",
    "Maximum number of iterations during training.": "训练过程中的最大迭代次数。",
    "(Optional) Annotation tracks might be overlayed with the segmentation results.": "（可选）注释轨迹可能与分割结果重叠。",
    "BED-file containing the annotation. For example, gene annotation or CpG-islands.": "包含注释的BED文件。例如，基因注释或CpG岛注释。",
    "Max between distance and diags will be used.": "最大距离和对角线之间的值将被使用。",
    "'store_final': Store the final result, regardless of whether the iterations converge to the specified tolerance; 'store_nan': Store a vector of NaN values to indicate that the matrix failed to converge; 'discard': Store nothing and exit gracefully; 'error': Abort with non-zero exit status.": "'store_final': 存储最终结果，无论迭代是否收敛到指定的容差；'store_nan': 存储一个NaN值的向量，以表示矩阵未能收敛；'discard': 不存储任何内容并优雅退出；'error': 以非零退出状态中止。",
    "With digester file you have correct fragment id and can use option -usemid": "使用digester文件，您将拥有正确的片段ID，可以使用选项-usemid。",
    "A value of 0 disables the bloom filter": "值为0会禁用布隆过滤器",
    "Pop contig graph bubbles smaller than this value": "Pop contig 图表中小于这个值的气泡",
    "Pop unitig graph bubbles smaller than this value": "弹出小于此值的单元图泡沫",
    "Keep only tip unitigs with a number of reads greater than or equal to this value": "只保留读取数大于或等于这个值的末端单元序列",
    "This option is used with -r. Given a node N in the assembly graph, let max(N) be the length of the largest overlap of N. Hifiasm iteratively drops overlaps of N if their length/max(N) are below a threshold controlled by -x. Hifiasm applies -r rounds of short overlap removal with an increasing threshold between -x and -y": "这个选项与 -r 一起使用。给定一个节点 N 在组装图中，设 max(N) 为 N 的最大重叠长度。Hifiasm 会迭代地减少 N 的重叠，如果它们的长度/max(N) 低于由 -x 控制的阈值。Hifiasm 应用 -r 轮短重叠移除，阈值在 -x 和 -y 之间逐渐增加。",
    "This option is used with -r. Given a node N in the assembly graph, let max(N) be the length of the largest overlap of N. Hifiasm iteratively drops overlaps of N if their length/max(N) are over a threshold controlled by -y. Hifiasm applies -r rounds of short overlap removal with an increasing threshold between -x and -y": "这个选项与-r一起使用。给定一个节点N在组装图中，让max(N)成为N的最大重叠长度。Hifiasm会迭代地删除N的重叠，如果它们的长宽比/max(N)超过由-y控制的阈值。Hifiasm应用-r轮短重叠移除，阈值在-x和-y之间递增。",
    "May improve N50": "可能提高N50",
    "Ignore  error corrected reads and overlaps saved in prefix.*.bin files.  Apart from assembly graphs, hifiasm also outputs three binary files  that  save  alloverlap information during assembly step.  With these files, hifiasm can avoid the time-consuming all-to-all overlap calculation step, and  do  the  assembly directly  and  quickly.  This might be helpful when users want to get an optimized assembly by multiple rounds of experiments with different parameters.": "忽略错误校正的读取和保存在prefix.*.bin文件中的重叠。除了组装图，hifiasm还输出三个二进制文件，这些文件在组装步骤期间保存所有重叠信息。有了这些文件，hifiasm可以避免耗时的全对全重叠计算步骤，并直接快速地进行组装。这在用户希望通过多轮不同参数的实验获得优化的组装时可能会很有帮助。",
    "If not set, this will be determined automatically": "如果没有设置，这将自动确定",
    "A haplotype is defined as the combination of alleles for different polymorphisms that occur on the same chromosome.": "单倍型是指在同一条染色体上发生的不同多态性的等位基因组合。",
    "A value of 0 disables this filter": "值为0禁用此过滤器",
    "K-mers that occur more than this value multiplied by the coverage will be discarded": "出现次数超过这个值乘以覆盖度的K-mers将被丢弃",
    "The software selects the larger of this value and the k-mer count multiplied by coverage": "该软件选择这个值和k-mer计数乘以覆盖度中的较大者",
    "When analyzing the k-mer spectrum, ignore counts below this value": "在分析k-mer谱时，忽略低于此值的计数",
    "Employ k-mers occurring less than INT times to rescue repetitive overlaps": "使用出现次数少于INT次的k-mers来挽救重复的重叠区域",
    "Estimated haploid genome size used for inferring read coverage. If not provided, this parameter will be infered by hifism. Common suffices are required, for example, 100m or 3g": "用于推断读取覆盖度的估计单倍体基因组大小。如果没有提供，此参数将由hifism推断。需要使用常见的后缀，例如，100m或3g",
    "Set the max gap size of scaffolds (default is 3,000,000)": "设置支架的最大间隙大小（默认为3,000,000）",
    "Clustered contigs (multiple FASTA files)": "聚类拼接序列（多个FASTA文件）",
    "Long reads (FASTQ format) used to generate the assemblies": "用于生成组装的长读序列（FASTQ格式）",
    "Reads with less than this many bases aligned (default = 1000) will be ignored.": "比这个碱基数量少的序列（默认 = 1000）将被忽略。",
    "Reads with less than this percentage of their length covered by alignments (default = 90.0) will be ignored.": "比对覆盖度低于此百分比的序列将被忽略（默认 = 90.0）。",
    "The input should be an independent cluster generated by the *trycycler cluster* tool": "输入应该是由*trycycler cluster*工具生成的独立簇",
    "Use this option if your input contigs are not circular. It will disable the circularisation-correction steps in Trycycler reconcile.": "如果输入的序列不是环状的，请使用此选项。它将在 Trycycler 协调中禁用环状化校正步骤。",
    "If any of the sequences have a pairwise Mash distance of more than this (default = 0.02), then the contigs will fail the initial check.": "如果任何序列的配对Mash距离超过这个值（默认=0.02），那么拼接序列将无法通过初始检查。",
    "If any of the sequences have a pairwise relative length factor of more than this, then the contigs will fail the initial check. For example, if set to 1.1 (the default), then no contig can be more than 10% longer than any other.": "如果任何序列之间的相对长度因子超过这个值，那么在初始检查中，这些连续片段将会失败。例如，如果设置为1.1（默认值），那么没有任何一个连续片段可以比其他连续片段长10%以上。",
    "If they are set to 1000, then Trycycler will be willing to add up to 1000 bp to circularise it. Any contig which requires more than 1000 bp added to circularise will cause Trycycler reconcile to fail.": "如果它们被设置为1000，那么Trycycler将愿意添加多达1000个碱基对以使其闭合成环。任何需要添加超过1000个碱基对才能闭合成环的连续序列都会导致Trycycler协调失败。",
    "If they are set to 5, then Trycycler will be willing to add up to 5% of a contig's length to circularise it. Any contig which requires more than 5% of its length added to circularise will cause Trycycler reconcile to fail.": "如果它们设置为5，那么Trycycler将愿意增加一个contig长度的5%来使其闭合。任何需要增加超过其长度5%才能闭合的contig都会导致Trycycler的reconcile失败。",
    "If they are set to 50000, then Trycycler will be willing to remove up to 5000 bp to circularise it. Any contig which requires more than 5000 bp removed to circularise will cause Trycycler reconcile to fail.": "如果它们被设置为50000，那么Trycycler愿意移除最多5000个碱基对以使其圆化。任何需要移除超过5000个碱基对才能圆化的片段都会导致Trycycler协调失败。",
    "If they are set to 10, then Trycycler will be willing to remove up to 10% of a contig's length to circularise it. Any contig which requires more than 10% of its length removed to circularise will cause Trycycler reconcile to fail.": "如果它们被设置为10，那么Trycycler将愿意移除一个连续片段长度的10%来使其闭合成环。任何需要移除超过其长度10%才能闭合成环的连续片段都将导致Trycycler的调和失败。",
    "If any of the sequences have a pairwise global alignment percent identity of less than this (default = 98), then the contigs will fail the final check.": "如果任何序列的配对全局比对的相似性百分比低于这个值（默认值为98），那么连续序列将无法通过最终检查。",
    "If any of the sequences have lower pairwise window identity, it is discarted.": "如果任何序列具有较低的成对窗口一致性，则将其丢弃。",
    "The k-mer size used for sequence partitioning (default = 32)": "用于序列分割的k-mer大小（默认 = 32）",
    "The step size used for sequence partitioning (default = 1000).": "用于序列分割的步长大小（默认 = 1000）。",
    "The look-ahead margin used for sequence partitioning (default = 10000).": "用于序列分割的前瞻边距（默认值 = 10000）。",
    "The number of subsamples reads to make maximally-independent read subsets. The default is 12, a good number for most cases.": "生成最大独立读段子集的子样本读取数。默认值为12，对大多数情况来说都是一个不错的选择。",
    "If you do not provide this, Trycycler subsample will run miniasm to quickly assemble your read set to get a size. This value is used to calculate read depths and does not need to be exact, e.g. 10% error is fine.": "如果您没有提供这个参数，Trycycler subsample 将会运行 miniasm 以快速地组装您的读取集来获取一个大小。这个值被用来计算读取深度，并且不需要非常精确，例如，10%的误差是可以接受的。",
    "An estimate of the isolate total genome size.": "一个对分离菌株总基因组大小的估计。",
    "This is the minimum allowed read depth and also controls the subsampled depths. If is not provided, Trycycle decides what read depth each subset will be.": "这是允许的最小读取深度，同时也控制着子采样深度。如果未提供，则Trycycle决定每个子集的读取深度将是多少。",
    "Canu has support for using parental short-read sequencing to classify and bin": "Canu支持使用父代短读序列来分类和分箱",
    "The defaults are 0.300 for PacBio reads and 0.500 for Nanopore reads.": "默认值对于PacBio读取是0.300，对于Nanopore读取是0.500。",
    "The allowed difference in an overlap between two corrected reads. Assemblies of low coverage or data with biological differences will benefit from a slight increase in this. Defaults are 0.045 for PacBio reads and 0.144 for Nanopore reads.": "两个校正后的读取之间允许的重叠差异。低覆盖度的组装或具有生物学差异的数据将从这个值的轻微增加中受益。PacBio读取的默认值为0.045，Nanopore读取的默认值为0.144。",
    "Additional specifications provided in a canu spec file.": "在Canu规范文件中提供的额外规格。",
    "Coverage is checked whene when input sequences are initially loaded into the sequence store, when corrected reads are generated, and when read ends are trimmed off.": "覆盖度在输入序列最初加载到序列存储时、生成校正后的读取时以及修剪读取端点时进行检查。",
    "If a FASTA sequence is given it will be used  as bait (using the --quick option). If a MAF alignment is given the reference is extracted from the alignment (using the --maf option)": "如果给定了一个FASTA序列，它将作为诱饵使用（使用--quick选项）。如果给定了一个MAF比对，参考序列将从比对中提取出来（使用--maf选项）。",
    "If resuming, use the same as in previous iteration/initial MIRA assembly": "如果继续进行，使用与上一次迭代/初始MIRA组装相同的设置。",
    "Read pairs need to be interleaved for full functionality of the '-pair' option below": "需要将配对的读段交错放置，以便在下面的'-pair'选项中实现完整功能",
    "If resuming, the sampleID needs to be identical to that of the previous iteration / MIRA assembly": "如果恢复运行，样本ID需要与上一次迭代/MIRA组装时使用的样本ID相同。",
    "Defaults to 0 when using '--quick' reference": "默认情况下，当使用 '--quick' 引用时，设置为 0",
    "Default=startiteration, i.e. if not specified otherwise stop after 1 iteration": "默认值为startiteration，即如果没有特别指定，则在1次迭代后停止",
    "Relies on /1 and /2 header convention for read pairs": "依赖于读段对的 /1 和 /2 头部约定",
    "It is recommended to trim beforehand and feed MITObim with pre trimmed data": "建议事先进行修剪，然后使用预先修剪过的数据输入MITObim。",
    " up- and downstream of reference. That is don't extend the bait": "在参考序列的上游和下游。也就是说，不要扩展诱饵序列。",
    "Only for illumina data (default: 15% of avg. read length)": "仅适用于Illumina数据（默认值：平均读取长度的15%）",
    "Fasta file to trim, and from which to remove Ns and fake cut sites.": "需要修剪的Fasta文件，以及从中移除Ns和假切割位点。",
    "This file is generated with purge_dups by using the 'Calculate coverage cutoff, base-level read depth and create read depth histogram' option": "这个文件是使用 purge_dups 工具的 'Calculate coverage cutoff, base-level read depth and create read depth histogram' 选项生成的",
    "This parameter is set for a suspect haplotigs. If 80% (default value) of a scaffold is high coverage (defined by the sixth column of the cutoff file), it's a repetitive contig. If 80% is low coverage (defined in the first column of the cutoff file), it's a junk contig. If 80% is above diploid coverage(defined in the fourth column of the cutoff file), it's a diploid, otherwise it's a suspect haplotig": "这个参数是为疑似单倍型设置的。如果一个支架的80%（默认值）是高覆盖率（由截止文件的第六列定义），那么它就是重复的连续序列。如果80%是低覆盖率（在截止文件的第一列中定义），那么它就是垃圾连续序列。如果80%高于二倍体覆盖率（在截止文件的第四列中定义），那么它就是二倍体，否则它就是疑似单倍型。",
    "If its alignment score is larger than this parameter and max match score larger than the 'mininum max match score' (-b), it is marked as a repeat; if the alignment score is larger than this parameter and max match score no larger than the 'mininum max match score', it is marked as a haplotig. Otherwise it is left as a candidate primary contig. If after purging, the complete genes reported by BUSCO are too low, you can try to increase this parameter": "如果其比对得分大于此参数，并且最大匹配得分大于“最小最大匹配得分”（-b），则将其标记为重复序列；如果比对得分大于此参数，并且最大匹配得分不大于“最小最大匹配得分”，则将其标记为单体型；否则，将其保留为候选主片段。如果清除后，BUSCO报告的完整基因太少，可以尝试增加此参数。",
    "If its alignment score is larger than the 'minimum align score' (-a) and max match score larger than this parameter, it is marked as a repeat; if the alignment score is larger than the 'minimum align' and max match score no larger than this parameter, it is marked as a haplotig. Otherwise it is left as a candidate primary contig.": "如果它的比对分数大于“最小比对分数”（-a）并且最大匹配分数大于这个参数，它就被标记为重复序列；如果比对分数大于“最小比对”并且最大匹配分数不大于这个参数，它就被标记为单倍型；否则，它就被保留为候选主片段。",
    "In the first round, it will asset chains consistent alignments within this parameter value": "在第一轮中，它将评估在这个参数值内一致的对齐链",
    "In the second round, it will asset chains consistent alignments within this parameter value": "在第二轮中，它将评估在这个参数值内一致的对齐链",
    "This parameter control the overlap size. You should decrease its value to allow more overlaps": "这个参数控制重叠大小。您应该减小它的值以允许更多的重叠。",
    "If the chained alignment is within this value to the contig ends, it will extended to the ends": "如果链式比对在到连续序列末端的这个值范围内，它将延伸到末端",
    "The sequence will be cleaved in those position in which the nucleotides is an 'N' or an 'n'.": "该序列将在核苷酸为'N'或'n'的位置被切割。",
    "Generated by the 'purge_dups' function mode.": "由 'purge_dups' 函数模式生成。",
    "Only remove sequences at end of halplotigs. If you also want to remove the duplications in the middle, set to false, however that may delete false positive duplications.": "仅移除halplotigs末端的序列。如果您还想移除中间的重复，可以将其设置为false，但这样可能会删除错误的重复。",
    "The taxonomic relationship between input genomes. If genomes are from multiple individuals of the same species, select 'Within-species'": "输入基因组之间的分类学关系。如果基因组来自同一物种的多个个体，则选择“物种内”",
    "Phylogenetic tree in Newick format. Required by Cactus to achieve linear scaling with number of input genomes": "新克格式的系统发育树。Cactus 需要它来实现输入基因组数量的线性扩展。",
    "Pangenomes from Minigraph-Cactus depend on a predetermined reference genome. Specify one of the Input Genomes as the reference genome. This must match the label used in 'Genome Label'.": "Minigraph-Cactus 生成的泛基因组依赖于一个预先确定的参考基因组。请指定一个输入基因组作为参考基因组。这必须与 'Genome Label' 中使用的标签相匹配。",
    "NO SPACES. Must match a label in the guide tree.": "无空格。必须与指南树中的标签匹配。",
    "Merqury allows two operational modes: default and trio mode.": "Merqury允许两种操作模式：默认模式和三重模式。",
    "Select the k-mer counts of the read set.": "选择读取集的 k-mer 计数。",
    "Select the number of assemblies.": "选择组装的数量。",
    "Select the assembled genome (FASTA).": "选择组装好的基因组（FASTA格式）。",
    "Select the first assembled genome (FASTA).": "选择第一个组装后的基因组（FASTA格式）。",
    "Select the second assembled genome (FASTA).": "选择第二个组装后的基因组（FASTA格式）。",
    "Select the k-mer counts of the read set (meryldb ).": "选择读取集（meryldb）的k-mer计数。",
    "K-mer counts of the paternal haplotype (meryldb).": "父系单倍型的K-mer计数（meryldb）。",
    "K-mer counts of the maternal haplotype (meryldb).": "母系单倍型的K-mer计数（meryldb）。",
    "Assign an identification label.": "分配一个识别标签。",
    "default: (x where y=peak) * 2.1": "默认：(x 在 y=峰值处) * 2.1",
    "default: (y where y=peak) * 1.1": "默认：(y 在 y=峰值处) * 1.1",
    ".only.hist file": ".only.hist 文件",
    "cutoff.txt file": "截止值文件",
    "Sort the genome by length or name of the scaffolds": "根据支架的长度或名称对基因组进行排序",
    " Filter mapped read by minimum MAPQ. Useful to remove spurious contacts from multimapping reads": "过滤最小MAPQ的映射读取。有助于从多映射读取中移除错误接触",
    "Subset the output excluding a list of scaffolds": "子集输出，排除一系列支架",
    "Comma separated list of sequence names to use as a filter": "逗号分隔的序列名称列表，用作过滤器",
    "Headers must not contain ':'.": "标题中不能包含冒号。",
    "To start scaffolding with SALSA, reads need to be mapped to the assembly. BWA or BOWTIE2 are recommended. SALSA requires a bed file as the input. The alignment bam file can be converted using the bamToBed command from the Bedtools package.": "要开始使用SALSA进行支架搭建，需要将读取映射到组装上。推荐使用BWA或BOWTIE2。SALSA需要一个bed文件作为输入。可以使用Bedtools包中的bamToBed命令将对齐的bam文件转换。",
    "Minimum contig length to scaffold": "最小连通体长度以构建框架图",
    "An assembly graph can be optionally provided to guide the scaffolding, potentially reducing the scaffolding errors": "一个组装图可以被选择性地提供以指导支架化，可能减少支架化错误",
    "Hi-C experiments can use different restriction enzymes. The enzyme frequency in contigs is used to normalize the Hi-C interaction frequency. Note that you need to specify the actual sequence of the cutting site for a restriction enzyme and not the enzyme name. You can also specify DNASE as an enzyme if you use an enzyme-free prep, e.g. Omin-C.": "Hi-C实验可以使用不同的限制性内切酶。在进行Hi-C互作频率的标准化时，需要使用到这些内切酶在基因组上的切割位点的频率。请注意，您需要指定限制性内切酶切割位点的实际序列，而不是酶的名称。如果您使用的是无酶预处理，例如Omin-C，您也可以将DNASE指定为酶。",
    "Restriction enzyme sequence. If multiple were used, include all as a comma separated list without spaces (ex. 'GATC,AAGCTT').": "限制酶序列。如果使用了多个，请将它们作为逗号分隔的列表包含在内，不要加空格（例如'GATC,AAGCTT'）。",
    "SALSA will scaffold through sequential iterations. The default number of iterations is 3. Increasing the number of iterations will potentially increase the number of joins, however it could also introduce additional misjoins": "SALSA 将通过连续迭代进行支架搭建。默认迭代次数为 3。增加迭代次数可能会增加连接的数量，但也可能引入额外的错误连接。",
    "Set this option to 'yes' if you want to find misassemblies in input assembly": "如果希望在输入的组装中找到错误组装，请将此选项设置为'是'",
    "Expected Genome size of the assembled genome. If not set, Salsa will estimate genome size.": "组装基因组的预期基因组大小。如果未设置，Salsa将估计基因组大小。",
    "If you use a config file and specify any conflicting values in the other options, the values specified in the other options will take precedence.": "如果您使用配置文件并在其他选项中指定任何冲突的值，则在其他选项中指定的值将优先。",
    "The graphical fragment assembly file describes the assembly and how contigs in the assembly relate to one another in the assembly graph": "图形片段组装文件描述了组装过程以及组装图中片段之间的关系",
    "The configuration file contains all of the parameters used by shasta based on the options selected.": "配置文件包含了根据所选选项由shasta使用的所有参数。",
    "Log file contains basic information about the run": "日志文件包含有关运行的基本信息",
    "Read length cutoff. Shorter reads are discarded": "读取长度截止。较短的读取被丢弃",
    "Reduce coverage to desired value. If not zero, specifies desired coverage (number of bases). The read length cutoff specified via --Reads.minReadLength is increased to reduce coverage to the specified value. Power of 10 multipliers can be used, for example 120Gb to request 120 Gb of coverage": "将覆盖度减少到所需的值。如果不为零，则指定所需的覆盖度（碱基数量）。通过 --Reads.minReadLength 参数指定的读取长度截止值将增加，以将覆盖度减少到指定的值。可以使用10的幂乘数，例如 120Gb 以请求 120 Gb 的覆盖度。",
    "Oxford Nanopore reads should be flagged for better results.": "牛津纳米孔测序读取应该被标记以获得更好的结果。",
    "Set to 0 to let --MinHash.alignmentCandidatesPerRead control the number of iterations": "设置为0，以让--MinHash.alignmentCandidatesPerRead控制迭代次数",
    "If --MinHash.minHashIterationCount is not 0, this is not used": "如果--MinHash.minHashIterationCount不为0，则不使用此参数",
    "This should only be used for experimentation on very small runs because it is very time consuming": "这应该只用于在非常小的运行上进行实验，因为它非常耗时。",
    "The alignment method to be used to create the read graph and the marker graph.": "要用于创建读取图和标记图的比对方法。",
    "Markers more frequent than this value in either of two oriented reads being aligned are discarded and not used to compute the alignment.": "如果两个定向读取中任意一个中的标记比这个值更频繁，则这些标记将被丢弃，并且不用于计算对齐。",
    "Only used for 'SeqAn' and 'Banded SeqAn' alignment methods": "仅用于“SeqAn”和“Banded SeqAn”比对方法",
    "Only used for 'Banded SeqAn' alignment method": "仅用于“Banded SeqAn”比对方法",
    "Containment alignments are alignments in which one read is entirely contained in another read, except possibly for up to maxTrim markers at the beggining and end": "嵌套比对是指一个读取完全包含在另一个读取中，除了可能在开头和结尾有最多maxTrim个标记之外。",
    "If not zero, alignments between reads from the same nanopore channel and close in time are suppressed. The 'read' meta data fields from the FASTA or FASTQ header are checked. If their difference, in absolute value, is less than the value of this option, the alignment is suppressed. This can help avoid assembly artifact. This check is only done if the two reads have identical meta data fields 'runid', 'sampleid', and 'ch'. If any of these metadata fields are missing this check is suppressed and this option has no effect.": "如果非零，来自同一纳米孔通道且时间接近的读取之间的比对会被抑制。将检查FASTA或FASTQ头中的'read'元数据字段。如果它们的差值（绝对值）小于此选项的值，则抑制比对。这有助于避免组装工件。只有在两个读取具有相同的元数据字段'runid'、'sampleid'和'ch'时，才会执行此检查。如果缺少这些元数据字段中的任何一个，则抑制此检查，此选项无效。",
    "Used for chimeric read detection": "用于嵌合体读段检测",
    "Set this to zero to entirely suppress flagCrossStrandReadGraphEdges": "将此设置为零以完全抑制flagCrossStrandReadGraphEdges",
    "Only used when creation method for the read graph is Experimental version 1": "仅在创建读图的方法为实验版本1时使用",
    "Only used when creation method for the read graph is Experimental version 2": "仅在创建读取图的方法为实验版本2时使用",
    "Specifying 0 causes a suitable value of this parameter to be selected automatically": "指定0会导致自动选择此参数的一个合适的值",
    "Used during approximate transitive reduction. Marker graph edges with coverage lower than this value are always marked as removed regardless of reachability": "在近似传递约减期间使用。覆盖度低于此值的标记图边始终被标记为已删除，而不考虑其可达性。",
    "Used during approximate transitive reduction. Marker graph edges with coverage higher than this value are never marked as removed regardless of reachability": "在近似传递约减期间使用。覆盖度高于此值的标记图边永远不会被标记为移除，无论其可到达性如何",
    "Used during approximate transitive reduction": "在近似传递归约期间使用",
    "Maximum lengths (in markers) used at each iteration of simplifyMarkerGraph. Ex.) 10,100,1000": "在简化标记图（simplifyMarkerGraph）的每次迭代中使用的最长长度（以标记为单位）。例如：10,100,1000",
    "Experimental. If this is not zero, assembly graph cross-edges with average edge coverage less than this value are removed, together with the corresponding marker graph edges. A cross edge is defined as an edge v0=>v1 with out-degree(v0)>1, in-degree(v1)>1": "实验性的。如果这个值不为零，那么平均边缘覆盖度小于这个值的组装图交叉边将被移除，同时相应的标记图边缘也会被移除。一个交叉边被定义为一个边 v0=>v1，其中 out-degree(v0)>1，in-degree(v1)>1。",
    "Length threshold, in markers, for the marker graph refinement step, or 0 to turn off refinement step": "标记图细化步骤的长度阈值（以标记为单位），或设置为0以关闭细化步骤",
    "Used in the automatic selection of --MarkerGraph.minCoverage when --MarkerGraph.minCoverage is set to 0": "当--MarkerGraph.minCoverage设置为0时，用于自动选择--MarkerGraph.minCoverage的值",
    "Controls assembly of long marker graph edges": "控制长标记图边的组装",
    "Controlled by Assembly.detangle.* options (experimental)": "由 Assembly.detangle.* 选项控制（实验性）",
    "PacBio reads files": "PacBio 读取文件",
    "Trim all weak regions and split the reads on inner weak regions": "修剪所有低质量区域并在内部低质量区域处拆分读取序列",
    "Currently only paired-end reads are supported": "目前仅支持成对末端读取",
    "(Combined reads)": "（合并的读取）",
    "(Forward reads)": "（前向读取序列）",
    "(Reverse reads)": "（反向读取序列）",
    "(Seed Input)": "（种子输入）",
    "If a reference is available, you can select the fasta file.": "如果参考序列可用，您可以选择fasta文件。",
    "The assembly will still be de novo, but references of the same genus can be used as a guide to resolve duplicated regions in the plant mitochondria or the inverted repeat in the chloroplast. References from different genus haven't beeen tested yet.": "该组装仍然是从头开始的，但是可以使用同属的参考作为指南，以解决植物线粒体中的重复区域或叶绿体中的反向重复。尚未测试不同属的参考。",
    "If you want to detect heteroplasmy, first assemble the genome without this option. Then give the resulting sequence as a reference and as a seed input.": "如果你想检测异质性，首先在不使用这个选项的情况下组装基因组。然后，将得到的序列作为参考序列和种子输入。",
    "If you have a PCR-free library select yes": "如果您有无需PCR的文库，请选择“是”",
    "It will create a vcf file with all the variances compared to the given reference genome.": "它将创建一个与给定参考基因组相比所有变异的vcf文件。",
    "(Read Length)": "（读取长度）",
    "It doesn't have to be accurate but should be close enough. (Insert size)": "它不需要非常精确，但应该足够接近。（插入大小）",
    "Chloroplast assembly (chloro), mitochondrial assembly (mito) and mitochondrial assembly in plants (mito_plant).": "叶绿体组装（chloro）、线粒体组装（mito）和植物中线粒体组装（mito_plant）。",
    "Chloroplast needs to be assembled before mitochondria. (Chloroplast sequence)": "叶绿体需要在线粒体之前组装。（叶绿体序列）",
    "Only use this when your seed originates from the same sample and there are no possible mismatches. (Extend seed directly)": "仅当您的种子来自相同样本且不存在可能的不匹配时，请使用此选项。（直接扩展种子）",
    "If reads are shorter then 90 bp or you have low coverage data, this value should be decreased down to 23. For reads longer then 101 bp, this value can be increased, but this is not necessary. (kmer)": "如果读取长度小于90个碱基对，或者你有低覆盖度数据，这个值应该减小到23。对于大于101个碱基对的读取，这个值可以增加，但这并不是必须的。（kmer）",
    "(Insert size auto)": "（插入大小自动）",
    "Only use this when reads have low quality, like with the 300 bp reads of Illumina. (Use Quality Scores)": "仅在读取质量较低时使用此选项，例如Illumina的300 bp读取。（使用质量分数）",
    "All the reads used for the assembly will be stored in seperate files. (Save assembled reads)": "所有用于组装的读取将存储在单独的文件中。（保存组装后的读取）",
    "Accepts fasta (.fasta, .fa) and fastq (.fastq) formats as well as gzip (.gz) and gzip2 (.bz2) files": "接受fasta (.fasta, .fa) 和 fastq (.fastq) 格式，以及gzip (.gz) 和 gzip2 (.bz2) 文件。",
    "Selecting individual mode will generate one assembly dataset for each fastq pair-end dataset. Selecting the merge option will produce one assembly dataset for all input fastq pair-end datasets": "选择个体模式将为每个fastq成对数据集生成一个组装数据集。选择合并选项将为所有输入的fastq成对数据集生成一个组装数据集。",
    "(kmin+1)-mer with multiplicity lower than d (default 2, specified by --min-count option) will be discarded. You should be cautious to set d less than 2, which will lead to a much larger and noisy graph. We recommend using the default value 2 for metagenomics assembly. If you want to use MEGAHIT to do generic assemblies, please change this value according to the sequencing depth. (recommend --min-count 3 for >40x).": "（kmin+1）-mer，其多重性低于d（默认为2，由--min-count选项指定）将被丢弃。你应该小心设置d小于2，这将导致一个更大且嘈杂的图。我们建议在宏基因组组装中使用默认值2。如果你想使用MEGAHIT进行通用组装，请根据测序深度更改此值。（推荐对于>40x的测序深度使用--min-count 3）。",
    "all must be odd, in the range 15-255, and with increments <= 28": "所有值必须是奇数，在15到255的范围内，并且增量 <= 28",
    "Must be odd number. For ultra complex metagenomics data such as soil, a larger kmin, say 27, is recommended to reduce the complexity of the de Bruijn graph. Quality trimming is also recommended. For high-depth generic data, large --k-min (25 to 31) is recommended. Smaller --k-step, say 10, is more friendly to low-coverage datasets.": "必须是奇数。对于超复杂的宏基因组数据，如土壤，建议使用较大的kmin，比如27，以降低德布鲁因图的复杂性。还建议进行质量修剪。对于高深度的通用数据，推荐使用较大的--k-min（25到31）。较小的--k-step，比如10，对低覆盖度数据集更友好。",
    "must be odd number": "必须是奇数",
    "must be even number": "必须是偶数",
    "Mercy kmers are specially designed for metagenomics assembly to recover low coverage sequences. For generic dataset >= 30x, MEGAHIT may generate better results with no mercy kmers.": "Mercy kmers 是为宏基因组组装特别设计的，用于恢复低覆盖序列。对于通用数据集 >= 30x，MEGAHIT 可能在没有 mercy kmers 的情况下生成更好的结果。",
    "Select 'paired end' for a single library or 'collection' for a paired end collection": "选择“成对末端”用于单个库，或选择“集合”用于成对末端集合",
    "Use Trimmomatic to remove common adaptors first (default: OFF)": "使用Trimmomatic去除常见的接头序列（默认：关闭）",
    "Which assembler would you like shovill to use, default is Spades": "您希望Shovill使用哪种组装器，默认是Spades。",
    "Format of contig FASTA IDs in 'printf' style (default: 'contig%05d')": "在 'printf' 风格的 'contig' FASTA ID 格式（默认：'contig%05d'）",
    "Sub-sample --R1/--R2 to this depth. Disable with --depth 0 (default: 100)": "子样本 --R1/--R2 到这个深度。使用 --depth 0 禁用（默认值：100）",
    "An estimate of the final genome size, it will autodetect if this is blank. (default: '')": "一个对最终基因组大小的估计，如果这个留空，它将自动检测。（默认值：''）",
    "List of K-mer sizes to use in SPAdes. Blank is AUTO. default: ''": "在SPAdes中使用的K-mer大小列表。空白为AUTO。默认值：''",
    "eg. --plasmid --sc ... (default: '')": "例如 --质粒 --sc ... （默认值：''）",
    "Disable post assembly correction with pilon (default: ON)": "禁用 Pilon 组装后的校正（默认：开启）",
    "Keep bam files only if post-assembly correction is enable": "仅在启用组装后校正时保留 bam 文件",
    "Minimum length of contig to be output. 0 is AUTO (default: 0)": "输出的最小连续序列长度。0是自动选择（默认：0）",
    "Minimum coverage to call part of a contig. 0 is AUTO (default: 2)": "调用一个连续片段的最小覆盖度。0是自动（默认值：2）",
    "Return the Shovill log file as part of the output. Default is on": "将Shovill日志文件作为输出的一部分返回。默认为开启。",
    "input file in FASTA/FASTQ format (can be compressed with gzip) containing sequences used for correction": "输入文件为FASTA/FASTQ格式（可以使用gzip压缩），包含用于校正的序列",
    "input file in MHAP/PAF/SAM format (can be compressed with gzip) containing overlaps between sequences and target sequences": "输入文件为MHAP/PAF/SAM格式（可以使用gzip压缩），包含序列与目标序列之间的重叠信息",
    "Input file in FASTA/FASTQ format (can be compressed with gzip) containing sequences which will be corrected": "输入文件为FASTA/FASTQ格式（可以用gzip压缩），包含将被校正的序列",
    "Output unpolished target sequences": "输出未修饰的目标序列",
    "Perform fragment correction instead of contig polishing. Note: overlaps file should contain dual/self overlaps": "执行片段校正而不是拼接序列润色。注意：重叠区域文件应包含双端/自身重叠区域",
    "Size of window on which POA is performed": "POA执行的窗口大小",
    "Threshold for average base quality of windows used in POA": "用于POA中窗口平均碱基质量的阈值",
    "Maximum allowed error rate used for filtering overlaps": "用于过滤重叠的最大允许误差率",
    "Disables consensus trimming at window ends": "禁用窗口末端的一致性修剪",
    "Score for matching bases. Defalt: 3": "匹配碱基的得分。默认值：3",
    "Score for mismatching bases. Default: -5": "不匹配碱基的得分。默认值：-5",
    "Default: 0.01": "默认值：0.01",
    "Polishing is performed as the final assembly stage. By default, Flye runs one polishing iteration. Additional iterations might correct a small number of extra errors (due to improvements on how reads may align to the corrected assembly). If the parameter is set to 0, the polishing is not performed": "打磨是在最终组装阶段进行的。默认情况下，Flye运行一次打磨迭代。额外的迭代可能会修正一小部分额外的错误（由于改进了读取对校正组装的对齐方式）。如果将参数设置为0，则不执行打磨。",
    "This sets a minimum overlap length for two reads to be considered overlapping. By default it is chosen automatically based on the read length distribution (reads N90) and does not require manual setting. Typical value is 3k-5k (and down to 1k for datasets with shorter read length). Intuitively, we want to set this parameter as high as possible, so the repeat graph is less tangled. However, higher values might lead to assembly gaps. In some rare cases it makes sense to manually increase minimum overlap for assemblies of big genomes with long reads and high coverage.": "这为两个读段被认为是重叠的设置了最小重叠长度。默认情况下，它是基于读段长度分布（读段N90）自动选择的，不需要手动设置。典型的值是3k-5k（对于读段长度较短的数据集可以低至1k）。直观地，我们希望尽可能地将此参数设置得高一些，这样重复图谱就不会太纠结。然而，较高的值可能会导致组装间隙。在一些罕见的情况下，对于具有长读段和高覆盖率的大型基因组的组装，手动增加最小重叠长度是有意义的。",
    "By default, Flye collapses graph structures caused by alternative haplotypes (bubbles, superbubbles, roundabouts) to produce longer consensus contigs. This option retains the alternative paths on the graph, producing less contigouos, but more detailed assembly.": "默认情况下，Flye会折叠由替代单倍型（泡沫、超泡沫、环岛）引起的图结构，以产生更长的共识序列。这个选项保留了图上的替代路径，生成的序列较少，但组装更详细。",
    "Starting from the version 2.9 Flye does not perform scaffolding by default, which guarantees that all assembled sequences do not have any gaps": "从2.9版本开始，Flye默认不执行scaffolding，这保证了所有组装的序列都没有间隙。",
    "It is designed for highly non-uniform coverage and is sensitive to underrepresented sequence at low coverage (as low as 2x). In some examples of simple metagenomes, we observed that the normal mode assembled more contigious bacterial consensus sequence, while the metagenome mode was slightly more fragmented, but revealed strain mixtures": "它设计用于高度非均匀覆盖，并对低覆盖度（低至2x）下代表性不足的序列敏感。在一些简单的宏基因组示例中，我们观察到普通模式组装了更多的连续细菌共识序列，而宏基因组模式稍微更加碎片化，但揭示了菌株混合。  专业术语解释： 1. highly non-uniform coverage - 高度非均匀覆盖 2. underrepresented sequence - 代表性不足的序列 3. low coverage - 低覆盖度 4. contigious bacterial consensus sequence - 连续细菌共识序列 5. strain mixtures - 菌株混合",
    "Typically, assemblies of large genomes at high coverage require a hundreds of RAM. For high coverage assemblies, you can reduce memory usage by using only a subset of longest reads for initial contig extension stage (usually, the memory bottleneck)": "通常，对大型基因组进行高覆盖度组装需要数百GB的RAM。对于高覆盖度的组装，您可以通过仅使用最长的读段的一个子集来进行初始的连通体扩展阶段（通常是内存瓶颈），从而减少内存使用。  角色设定：  你是一位资深的生物信息学专家，对于生物信息学的专业术语和软件使用具有非常深刻的理解。 你非常擅长在Galaxy生信云平台上使用各种分析工具，并且非常擅长在Galaxy上使用各种分析工具。  角色偏好：  你非常擅于将生物信息学专业术语翻译成中文专业术语。  特别要注意的点：  你要从所需翻译的内容中区分哪些是生物信息学软件或者软件包的参数，遇到参数时需要保持原文不做翻译。 括符中以横线'-'或下划线'_'开头的内容，通通不翻译。  输出格式：  接收一行文本，输出一行翻译后的文本。  输入： > Tool: \"Bowtie 2\" aligner with default settings for paired-end reads. > Description: Aligns paired-end reads to a reference genome using the Bowtie 2 aligner. The Bowtie 2 aligner is designed to be very fast and to handle large genomes.  输出： > 工具：\"Bowtie 2\" 比对器，使用成对末端读取的默认设置。 > 描述：使用 Bowtie 2 比对器将成对末端读取比对到参考基因组。Bowtie 2 比对器旨在非常快速并能处理大型基因组。",
    "This parameter specifies the target coverage of the longest reads. For a typical assembly, 30x is enough to produce good initial contigs. Regardless of this parameter, all reads will be used at the later pipeline stages.": "这个参数指定了最长读取序列的目标覆盖度。对于典型的组装，30倍的覆盖度足以产生良好的初始连续序列。无论这个参数如何，所有读取序列都会在后期的流水线阶段被使用。",
    "For example, 5m or 2.6g. No longer required as input. However, it must be used in conjunction with --asm-coverage option.": "例如，5m 或 2.6g。不再作为输入要求。但是，它必须与 --asm-coverage 选项一起使用。",
    "Useful to know if contigs have been generated all samples together (co-assembly) or on each sample individually (individual assembly)": "有助于了解是否所有样本一起生成了连续序列（共组装），还是在每个样本单独生成（单独组装）。",
    "Many metrics can't be evaluated without a reference. If this is omitted, QUAST will only report the metrics that can be evaluated without a reference.": "许多指标没有参考数据就无法评估。如果省略了参考数据，QUAST将只报告那些可以在没有参考数据的情况下评估的指标。",
    "Gene coordinates for the reference genome": "参考基因组的基因坐标",
    "Operon coordinates for the reference genome": "参考基因组的操纵子坐标",
    "It is recommended for large genomes. This may significantly increase memory and time consumption on large genomes": "对于大型基因组，建议使用此选项。这可能会显著增加大型基因组的内存和时间消耗。",
    "Plot Circos version of Icarus contig alignment viewer": "绘制 Icarus 片段对齐查看器的 Circos 版本",
    "If no reference genomes is provided, the tool will try to identify genome content of the metagenome. By default, it will align contigs to SILVA 16S rRNA database, i.e. FASTA file containing small subunit ribosomal RNA sequences to identify genomes. The reference genomes for the chosen genomes are downloaded from the NCBI database. After that, Quast is run on all of them. Reference genomes with low genome fraction (less than 10%) are removed. The usual MetaQUAST analysis continues with the remaining references.": "如果未提供参考基因组，该工具将尝试识别宏基因组的基因组内容。默认情况下，它将对齐到SILVA 16S rRNA数据库的片段，即包含小亚基核糖体RNA序列的FASTA文件，以识别基因组。所选基因组的参考基因组从NCBI数据库下载。之后，在所有基因组上运行Quast。具有低基因组分数（少于10%）的参考基因组将被移除。通常的MetaQUAST分析继续使用剩余的参考基因组。",
    "MetaQUAST will search for these references in the NCBI database and will download the found ones": "MetaQUAST将在NCBI数据库中搜索这些参考序列，并下载找到的序列",
    "Reuse the alignments on the combined reference in the subsequent runs per separate references. That is, the alignment procedure is performed only once (for all assemblies against the combined reference) and does NOT executed for each subgroups of contigs against the corresponding separate reference genomes. In each separate reference run, all precomputed assembly alignments for other references are simply ignored": "在后续针对各个独立参考基因组的运行中重用组合参考基因组上的比对。也就是说，比对过程只执行一次（针对所有组装与组合参考基因组），并不会针对每个小组的连续序列与相应的独立参考基因组执行。在每个独立的参考基因组运行中，所有针对其他参考基因组的预先计算的组装比对都被简单地忽略。",
    "Shorter contigs won't be taken into account": "较短的基因组片段将不被考虑",
    "QUAST will add split versions of assemblies to the comparison. Assemblies are split by continuous fragments of N's of length >= 10. If broken version is equal to the original assembly (i.e. nothing was split) it is not included in the comparison.": "QUAST 会将组装结果的拆分版本添加到比较中。组装结果通过长度大于等于10的连续N片段进行拆分。如果拆分后的版本与原始组装结果相同（即没有进行拆分），则不将其包含在比较中。",
    "Use optimal parameters for evaluation of large genomes. Affects speed and accuracy. In particular, imposes --eukaryote --min-contig 3000 --min-alignment 500 --extensive-mis-size 7000 (can be overridden manually with the corresponding options). In addition, this mode tries to identify misassemblies caused by transposable elements and exclude them from the number of misassemblies.": "使用适用于大型基因组评估的最优参数。影响速度和准确性。特别是，强制使用--eukaryote --min-contig 3000 --min-alignment 500 --extensive-mis-size 7000（可以使用相应的选项手动覆盖）。此外，此模式尝试识别由转座元件引起的错配，并将其从错配数量中排除。",
    "By default, we assume that the genome is prokaryotic, and Barrnap uses the bacterial database for rRNA prediction. If the genome is eukaryotic (fungal), use --eukaryote (--fungus) option to force Barrnap to work with the eukaryotic (fungal) database.": "默认情况下，我们假设基因组是原核生物的，Barrnap 使用细菌数据库进行 rRNA 预测。如果基因组是真核生物（真菌）的，请使用 --eukaryote (--fungus) 选项强制 Barrnap 使用真核生物（真菌）数据库。",
    "By default, we assume that the genome is prokaryotic, and BUSCO uses the bacterial database of orthologs. If the genome is eukaryotic (fungal), use --eukaryote (--fungus) option to force BUSCO to work with the eukaryotic (fungal) database.": "默认情况下，我们假设基因组是原核生物的，BUSCO使用细菌的直系同源数据库。如果基因组是真核生物（真菌）的，请使用--eukaryote（--fungus）选项强制BUSCO使用真核生物（真菌）数据库。",
    "By default, QUAST v.2.0 and higher filters out ambiguous and redundant alignments, keeping only one alignment per contig (or one set of non-overlapping or slightly overlapping alignments)": "默认情况下，QUAST v.2.0 及更高版本会过滤掉不明确的和冗余的比对，只保留每个基因组序列（或一组不重叠或略有重叠的比对）。",
    "Alignments shorter than this value will be filtered. Note that all alignments shorter than 65 bp will be filtered regardless of this threshold.": "比这个值短的比对将被过滤。请注意，所有比对长度小于65个碱基对（bp）的，无论这个阈值如何，都将被过滤。",
    "All alignments are sorted by decreasing LEN × IDY% value. All alignments with LEN × IDY% less than S × best(LEN × IDY%) are discarded.": "所有比对按照递减的“长度×一致性百分比”值进行排序。所有“长度×一致性百分比”小于S×最佳（长度×一致性百分比）的比对都被丢弃。",
    "Reference genome is fragmented (e.g. a scaffold reference). QUAST will try to detect misassemblies caused by the fragmentation and mark them fake (will be excluded from misassemblies). Note: QUAST will not detect misassemblies caused by the linear representation of circular genome": "参考基因组是片段化的（例如，一个支架参考）。QUAST将尝试检测由片段化引起的错误组装，并将其标记为假的（将从错误组装中排除）。注意：QUAST不会检测由圆形基因组的线性表示引起的错误组装。",
    "Simulate upper bound assembly based on the reference genome and a given set reads (mate-pairs or long reads, such as Pacbio SMRT/Oxford Nanopore, are REQUIRED). This assembly is added to the comparison and could be useful for estimating the upper bounds of completeness and contiguity that theoretically can be reached by assembly software from this particular set of reads. The concept is based on the fact that the reference genome cannot be completely reconstructed from raw reads due to long genomic repeats and low covered regions.": "模拟基于参考基因组和给定的一组读取（必须提供配对末端或长读取，例如Pacbio SMRT/Oxford Nanopore）。这个组装被添加到比较中，并且可能有助于估计从这组特定的读取理论上可以由组装软件达到的完整性和连续性的上限。这个概念基于这样一个事实：由于长基因组重复和低覆盖区域，参考基因组不能从原始读取中完全重建。",
    "This is important for a realistic estimation of genome assembly fragmentation due to long repeats. The default values is 2 for mate-pairs and 1 for long reads (PacBio or Nanopore libraries)": "这对于因长重复序列导致的基因组组装碎片化的真实估计很重要。对于配对端，其默认值为2；对于长读长（PacBio或Nanopore库），其默认值为1。",
    "Lower threshold for the local misassembly size. Shorter inconsistencies are considered as (long) indels. The default value is 200 bp. Note that the threshold should be equal to or lower than minimal extensive misassembly size, which is 1000 bp by default": "局部错误组装大小的下限阈值。较短的不一致性被视为（长）插入/缺失。默认值是200个碱基对。请注意，该阈值应等于或低于默认的最小广泛错误组装大小，后者为1000个碱基对。",
    "Used in # contigs ≥ x and total length (≥ x) metrics": "用于# 序列数 ≥ x 和总长度 (≥ x) 指标",
    "By default, QUAST breaks contigs only at extensive misassemblies (not local ones).": "默认情况下，QUAST仅在广泛的错误组装处（而非局部错误组装处）断裂连续序列。",
    "Shorter relocations are considered as local misassemblies. It does not affect other types of extensive misassemblies (inversions and translocations). The default value is 1000 bp. Note that the threshold should be greater than maximum indel length which is 85 bp.": "较短的重定位被视为局部错误组装。它不会影响其他类型的广泛错误组装（倒置和易位）。默认值为1000个碱基对。请注意，阈值应大于最大插入/缺失长度，即85个碱基对。",
    "Longer inconsistencies are considered as relocations and thus, counted as extensive misassemblies. The default value is 10000 bp. Note that the threshold make sense only if it is greater than extensive misassembly size": "较长的不一致性被视为重定位，因此，被计为广泛的错误组装。默认值是10000碱基对。请注意，只有当阈值大于广泛的错误组装大小时，该阈值才有意义。",
    "Mark translocation as fake if both alignments are located no further than N bases from the ends of the reference fragments. The value should be less than extensive misassembly size.Default value is 50. Note: requires --fragmented option": "如果两个比对都位于参考片段末端N个碱基以内，则将易位标记为假的。该值应小于广泛的错误组装大小。默认值是50。注意：需要--fragmented选项。",
    "Keep all quality metrics in the main report. Usually, all not-relevant metrics are not included in the report, e.g., reference-based metrics in the no-reference mode. Also, if metric values are undefined for all input assemblies, the metric is removed from the report": "在主报告中保留所有质量指标。通常，报告中不包括所有不相关的指标，例如，在无参考模式下不包括基于参考的指标。此外，如果所有输入组装的指标值未定义，则该指标将从报告中删除。",
    "Value of 'x' for Nx, Lx, NGx, NGAx, etc metrics reported in addition to N50, L50, NG50, NGA50, etc": "在报告的 Nx、Lx、NGx、NGAx 等指标中，'x' 的值除了 N50、L50、NG50、NGA50 等之外",
    "Pairwise mapping format": "成对映射格式",
    "column 10 in PAF": "PAF文件中的第10列",
    "Ignored mappings are still used for read overlaps": "忽略的映射仍然用于读取重叠",
    "This option also affects the second round of read filtering and minimal overlap length": "这个选项也会影响第二轮的读取过滤和最小重叠长度",
    "In the first round of filtering, miniasm finds the longest region covered by this number or more reads. In the second round, it in addition requires each remaining base to be covered by this many bases at least minSpan/2 from the ends of other reads": "在第一轮过滤中，miniasm 找出由这个数目或更多读段覆盖的最长区域。在第二轮中，它还要求每个剩余的碱基至少被其他读段末端至少 minSpan/2 的距离所覆盖。",
    "An overhang is an unmapped region that should be mapped given a true overlap or true containment. If the overhang is too long, the mapping is considered an internal match and will be ignored": "一个悬挂端是一个未映射的区域，如果存在真实的重叠或真实的包含关系，它应该被映射。如果悬挂端太长，映射将被视为内部匹配并被忽略。",
    "This option has a similar role to MaxHang, except that it controls the ratio, not length": "这个选项与MaxHang有类似的作用，但它控制的是比率，而不是长度。",
    "This parameter is only used for transitive reduction": "这个参数仅用于传递简化",
    "Bubbles longer than this value will not be popped": "比这个值更长的气泡将不会被弹出",
    "Miniasm may try to remove small unitigs at various steps": "Miniasm 可能会在不同步骤尝试移除小单元基因",
    "(lib)": "（库）",
    "The mate-pair libraries are used only for scaffolding and do not contribute towards the consensus sequence (mp)": "配对文库仅用于支架搭建，不贡献于共识序列（mp）。",
    "Long sequence libraries are used only for rescaffolding and do not contribute towards the consensus sequence": "长序列库仅用于重新组装，不贡献于共识序列",
    "The length of a k-mer (when -K is not set) or the span of a k-mer pair (when K is set)": "k-mer的长度（当未设置-K时）或k-mer对的跨度（当设置了K时）",
    "Trim bases from the ends of reads whose quality is less than this value": "从质量值小于该值的读段末端修剪碱基",
    "Mask all bases of reads whose quality is less than this value as 'N'": "将质量值小于这个值的所有碱基的读取掩蔽为'N'",
    "Erode bases at the ends of blunt contigs with coverage less than this value. Defaults to round(sqrt(median))": "在覆盖度小于这个值的平端连续片段的末端侵蚀碱基。默认值为中位数的平方根向上取整。",
    "Erode bases at the ends of blunt contigs with coverage less than this value on either strand. Defaults to 1 if sqrt(median) > 2 else 0": "在覆盖度小于此值的平末端重叠群的末端侵蚀碱基，对于任一链，默认值为如果中位数的平方根大于2则为1，否则为0。",
    "Defaults to the value of k": "默认值为 k",
    "Defaults to sqrt(median)": "默认为中位数的平方根",
    "abyss-pe has two bubble popping stages. The default limits are 3*k bp for ABYSS and 10000 bp for PopBubbles": "abyss-pe有两个泡泡破裂阶段。ABYSS的默认限制是3*k bp，PopBubbles的默认限制是10000 bp。",
    "Requires that all libraries are strand-specific RNA-Seq libraries. Assumes that the first read in a read pair is reveresed with respect to the transcripts sequenced": "要求所有库都是特异性的RNA-Seq库。假设配对读取中的第一个读取与被测序的转录本相反。",
    "Defaults to 0 (interpreted as k - 1) if mate-pair libraries are provided or if k<=50, otherwise 50": "默认为0（解释为k-1），如果提供了配对文库或者如果k<=50，否则为50",
    "The seed length should be at least twice the value of k. If more sequence is assembled than the expected genome size, try increasing this value": "种子长度至少应该是k值的两倍。如果组装的序列比预期的基因组大小多，尝试增加这个值。",
    "It can be an interval. Defaults to 100-5000": "它可以是一个区间。默认为100-5000",
    "It can be an interval. Defaults to 15-20": "它可以是一个区间。默认为15-20",
    "Specify dataset with unpaired reads": "指定包含未配对读取的数据集",
    "Unicycler uses SPAdes to construct assembly graphs. You can modify some of the SPAdes settings here. Use this ONLY if you know what you are doing!": "Unicycler 使用 SPAdes 构建组装图。你可以在这里修改一些 SPAdes 的设置。仅在你知道自己在做什么的情况下使用！",
    "It is done if does not result in graph dead ends": "如果不会生成图中的死路，那么就算完成",
    "These options control the rotation of completed circular sequence near the end of the Unicycler pipeline. Use this ONLY if you know what you are doing!": "这些选项控制 Unicycler 流水线末端完成的圆形序列的旋转。仅在您知道自己在做什么时使用！",
    "Unicycler uses TBLASTN to search for dnaA or repA alleles in each completed replicon. If one is found, the sequence is rotated and/or flipped so that it begins with that gene encoded on the forward strand. This provides consistently oriented assemblies and reduces the risk that a gene will be split across the start and end of the sequence.": "Unicycler 使用 TBLASTN 在每个已完成的复制子中搜索 dnaA 或 repA 等位基因。如果找到了一个，序列将被旋转和/或翻转，以便它以正链上编码的该基因开始。这提供了一致的定向组装，并降低了基因会被分割在序列的起始和结束处的风险。",
    "These options control the removal of small leftover sequences after bridging is complete.": "这些选项控制在桥接完成后移除小的残留序列。",
    "These options control the alignment of long reads to the assembly graph.": "这些选项控制长读段与组装图的比对。",
    "default = set automatically": "默认 = 自动设置",
    "Level of file retention. Default: 1": "文件保留级别。默认值：1",
    "To run read error correction, reads should be in FASTQ format.": "要运行读取错误校正，读取应为FASTQ格式。",
    "If --sc is set the default values are 21,33,55. For multicell datasets K values are automatically selected using maximum read length.": "如果设置了--sc，默认值是21,33,55。对于多细胞数据集，K值会根据最大读取长度自动选择。",
    "Odd integer, Lower limit of k-mer size range to search for optimum value": "奇数整数，k-mer大小范围的下限，用于寻找最优值",
    "Odd integer, Upper limit of k-mer size range to search for optimum value": "奇数整数，k-mer大小范围的上限，用于搜索最优值",
    "Even integer, the k-mer value step size when searching the range": "即使整数，搜索范围时k-mer值步长的尺寸",
    "Specify dataset with the reads": "指定包含读取数据集",
    "Specify dataset with interleaved paired reads": "指定包含交错配对读取的数据集",
    "Include verbose velvet output in log file": "在日志文件中包含详细的Velvet输出",
    "See help below for possibilities!": "请看下面的帮助以了解可能性！",
    "Add any other required velvetg options from the advanced set": "添加任何其他所需的velvetg选项，来自高级设置",
    "The minimum coverage cutoff to consider in the optimisation": "在优化中考虑的最小覆盖度截止值",
    "The maximum coverage cutoff to consider expressed as a fraction of the calculated expected coverage.": "最大覆盖度截止值，表示为计算出的预期覆盖度的分数。",
    "Sets up heuristics for variant calling, as opposed to assembly improvement; equivalent to '--vcf --fix all,breaks'.": "设置变体检测的启发式规则，而不是组装改进；等同于 '--vcf --fix all,breaks'。",
    "If specified, a file listing changes in the <output>.fasta will be generated.": "如果指定，将生成一个列出<output>.fasta中变化的文件。",
    "BAM file consisting of fragment paired-end alignments.": "由片段配对末端对齐组成的BAM文件。",
    "BAM file consisting of jump (mate pair) paired-end alignments.": "由跳跃（配对）配对末端对齐组成的BAM文件。",
    "BAM file consisting of unpaired alignments.": "由未配对的比对组成的BAM文件。",
    "If specified, a vcf file will be generated (even if 'Variant calling mode' is off)": "如果指定，将生成一个vcf文件（即使“变体调用模式”关闭）",
    "Write many track files (*.bed, *.wig) suitable for viewing in a genome browser.": "编写多个适用于基因组浏览器查看的轨迹文件（*.bed, *.wig）。",
    "Input FASTA elements larger than this will be processed in smaller pieces not to exceed this size.": "输入的FASTA元素如果大于这个大小，将会被分割成不超过这个大小的较小片段进行处理。",
    "If specified the VCF will contain a QE (quality-weighted evidence) field rather than the default QP (quality-weighted percentage of evidence) field.": "如果指定了，VCF文件将包含一个QE（质量加权证据）字段，而不是默认的QP（质量加权证据百分比）字段。",
    "Sample is from diploid organism; will eventually affect calling of heterozygous SNPs": "样本来自二倍体生物体；最终将影响杂合单核苷酸多态性的调用",
    "Use reads marked as duplicates in the input BAMs": "在输入的BAM文件中使用标记为重复的读取",
    "Output IUPAC ambiguous base codes in the output FASTA file when appropriate": "在适当的情况，在输出的FASTA文件中输出IUPAC模糊碱基代码",
    "Use reads which failed sequencer quality filtering": "使用未通过测序仪质量过滤的读取",
    "Only process the specified target(s).  Targets are comma-separated, and each target is a fasta element name optionally followed by a base range.": "仅处理指定的目标。目标以逗号分隔，每个目标是一个fasta元素名称，后面可跟一个基底范围。",
    "Assumes bases are of this quality if quals are no present in input BAMs": "假设碱基的质量为这个值，如果输入的BAM文件中没有质量分数（quals）的话",
    "This many bases at each end of the good reads will be ignored.": "在好的序列的每个末端，将忽略这么多碱基。",
    "Closed gaps must be within this number of bases of true size to be closed": "封闭的间隙必须在真实大小的这个基数范围内才能被封闭",
    "Kmer size used by internal assembler": "内部组装器使用的kmer大小",
    "Minimum depth of coverage required for variants to be called. See complete documentation below.": "要求调用变体所需的最小覆盖深度。请参阅下面的完整文档。",
    "Minimum size for unclosed gaps": "未闭合间隙的最小尺寸",
    "Minimum alignment mapping quality for a read to count in pileups": "在成堆比对中，用于计数的最低比对映射质量",
    "Minimum base quality to consider for pileups": "用于成堆的最小碱基质量",
    "See documentation below": "参见下面的文档",
    "Select one or more input fastq or fasta files from your history. To select multiple files, use ctrl + click": "从您的历史记录中选择一个或多个输入的fastq或fasta文件。要选择多个文件，请使用Ctrl + 单击",
    "Sequencing technology used to generate reads": "用于生成读取的测序技术",
    "Estimated genome size. k/m/g suffix is allowed - eg a 4500000bp ecoli genome can be written as 4.5m. For a human genome, use 3.2g": "估计的基因组大小。允许使用k/m/g后缀 - 例如，一个4500000碱基对的大肠杆菌基因组可以写成4.5m。对于人类基因组，请使用3.2g。",
    "(-X) [float] Choose the best [float] depth from input reads. ie if the estimated genome size is 5m, setting this value to 50.0 would select the best 2.5mb worth of reads.": "（-X）[浮点数] 从输入的读取数据中选择最佳[浮点数]深度。例如，如果估计的基因组大小为5M，将此值设置为50.0将选择最佳2.5Mb的读取数据。",
    "(-L) [int] Choose the longest subread and drop reads shorter than [int]": "(-L) [整数] 选择最长的子读段，并丢弃短于[整数]的读段",
    "(-k) [int] Kmer size, 0 ≤ k ≤ 23": "(-k) [整数] Kmer大小，0 ≤ k ≤ 23",
    "(-p) [int] Homopolymer-compressed kmer size, 0 ≤ p ≤ 23": "（-p）[整数] 同聚体压缩的kmer大小，0 ≤ p ≤ 23",
    "(-K) [float] Filter high frequency kmers where frequency > [float]": "(-K) [浮点数] 过滤高频kmers，其中频率 > [浮点数]",
    "(-s) [float] Min similarity between reads to label as related, calculated by kmer matched length / aligned length": "（-s）[浮点数] 将读取标记为相关的最小相似性，通过 kmer 匹配长度 / 对齐长度 计算",
    "(-e) [int] Min read depth of a valid edge": "(-e) [整数] 有效边的最小读取深度",
    "(-R) Enable realignment mode": "（-R）启用重新对齐模式",
    "(-A) Keep contained reads during alignment": "（-A）在比对过程中保留包含的读取",
    "mat_rate = matched_bases/total_bases": "匹配率 = 匹配碱基数/总碱基数",
    "Abort TriPOA when any read cannot be fast aligned, then try POA": "在任何读取无法快速对齐时终止TriPOA，然后尝试POA",
    "set to 0 to disable": "设置为0以禁用",
    "Increase this to reduce N50 and mis-assemblies": "增加这个值可以减少N50值和错误组装。",
    "By default rnaSPAdes uses 2 k-mer sizes, which are automatically detected using read length (approximately one third and half of the maximal read length). We recommend not to change this parameter because smaller k-mer sizes typically result in multiple chimeric (misassembled) transcripts.": "默认情况下，rnaSPAdes 使用 2 个 k-mer 大小，这些大小是使用读长自动检测的（大约是最大读长的三分之一和一半）。我们建议不要更改此参数，因为较小的 k-mer 大小通常会导致多个嵌合体（错误组装的）转录本。",
    "rnaSPAdes supports strand-specific RNA-Seq datasets. Use 'RF' when first read in pair corresponds to reverse gene strand (antisense data, e.g. obtained via dUTP protocol) and 'FR' otherwise.  If the dataset is single-end use 'FR' option in case when reads correspond to gene strand and 'RF' otherwise. Note: strand-specificity is not related and should not be confused with FR and RF orientation of paired reads. RNA-Seq paired-end reads typically have forward-reverse orientation, which is assumed by default and no additional options are needed": "rnaSPAdes支持特定于链的RNA-Seq数据集。当配对中的第一个读对应于反向基因链（反义数据，例如通过dUTP协议获得）时，请使用'RF'，否则使用'FR'。如果数据集是单端的，在读取对应于基因链的情况下使用'FR'选项，否则使用'RF'。注意：链特异性与配对读取的FR和RF方向无关，也不应该混淆。RNA-Seq配对末端读取通常具有正向-反向方向，这是默认假定的，不需要额外的选项。",
    "Select a type of operation": "选择一种操作类型",
    "Select an operation to be executed": "选择一个要执行的操作",
    "Select your reads in FASTA/FASTQ format.": "选择您在FASTA/FASTQ格式中的序列读取。",
    "For a human genome, the best k-mer size is k=21 for both haploid (3.1G) or diploid (6.2G)..": "对于人类基因组，无论是单倍体（3.1G）还是双倍体（6.2G），最佳的k-mer大小都是k=21。",
    "Haploid genome size or diploid genome size, depending on what we evaluate. In bp. Only required if the k-mer size is not provided.": "单倍体基因组大小或二倍体基因组大小，取决于我们评估的内容。以碱基对为单位。只有在未提供k-mer大小的情况下才需要。",
    "Tolerable collision rate. By default is 0.001.": "可容忍的碰撞率。默认值为0.001。",
    "Select a meryldb dataset": "选择一个Meryldb数据集",
    "Return k-mers that occur N times in the input.": "返回在输入中出现N次的k-mers。",
    "Return k-mers that at specific frequency (e.g. frequency = 0.9998 returns top 0.02% most frequent)": "返回具有特定频率的k-mers（例如，频率=0.9998返回最频繁的0.02%的k-mers）",
    "Select F1 reads used for generating the assembly": "选择用于生成组装的F1读取",
    "Select the paternal reads used for generating the assembly": "选择用于生成组装的父本读取",
    "Select the maternal reads used for generating the assembly": "选择用于生成组装的母本读取",
    "For a human genome, the best k-mer size is k=21 for both haploid (3.1G) or diploid (6.2G).": "对于人类基因组，无论是单倍体（3.1G）还是双倍体（6.2G），最佳的k-mer大小都是k=21。",
    "Mapping mode requires backbone/reference sequence(s)": "映射模式需要骨干/参考序列",
    "Required for mapping, optional for de novo assembly.": "映射必需，去新组装可选。",
    "FASTQ format": "FASTQ格式",
    "Specify here the Edena “.ovl” file obtained from the overlapping step": "在这里指定从重叠步骤获得的 Edena “.ovl” 文件",
    "The overlap cutoff is by default set to half of the reads length L (see the log output by the overlapping step to identify it). It is however still worth trying to increase this setting since it can greatly simplify highly connected overlaps graphs, and thus speed up the assembly. If one step during the assembly hangs, increasing the overlap cutoff is the first thing to do.": "重叠截断默认设置为读取长度L的一半（可以通过重叠步骤的日志输出来识别它）。然而，尝试增加此设置仍然值得，因为它可以大大简化高度连接的重叠图，从而加快组装速度。如果组装过程中的一个步骤出现挂起，增加重叠截断是首先要做的事情。",
    "Contextual cleaning is a procedure that efficiently identifies and removes false positive edges, improving thus the assembly. This procedure can be seen as a dynamic overlap cutoff on the overlaps graph. It is possible however for this step to be slow on ultra-high covered sequencing data. In such cases, try to increase the overlap cutoff value, or to simply disable this option.": "上下文清洁是一个有效识别并移除假阳性边缘的过程，从而改善组装。这个过程可以被视为重叠图上动态的重叠截止值。然而，在超高覆盖的测序数据上，这一步可能会很慢。在这种情况下，尝试增加重叠截止值，或者简单地禁用这个选项。",
    "This procedure discards orphan nodes smaller than 1.5*readLength.": "这个程序丢弃小于1.5*readLength的孤立节点。",
    "If not specified, this value is set to 1.5*readLength.": "如果没有指定，这个值将被设置为1.5*读取长度。",
    "If not specified, this value is automatically determined from the nodes coverage distribution. This estimation however supposes a uniform coverage. It could be worth overriding this parameter in some cases, i.e. with transcriptome data, or a mix of PCR product assemblies.": "如果未指定，该值将自动从节点覆盖度分布中确定。然而，这种估算假设了均匀覆盖度。在某些情况下，可能值得覆盖此参数，例如使用转录组数据或PCR产物组装的混合体。",
    "Contig interruptions are caused either because of a non-resolved ambiguity, or because of a lack of overlapping reads. In the latter case, the contig end may be inaccurate. This option will trim a few bases from these ends until a minimum coverage is reached. By default, this value is set to 4. To disable contigs ends trimming, set this value to 1.": "片段中断可能是由于未解决的歧义或缺乏重叠的读取造成的。在后一种情况下，片段末端可能不准确。此选项将从这些末端修剪一些碱基，直到达到最小覆盖度。默认情况下，此值设置为4。要禁用片段末端修剪，请将此值设置为1。",
    "Edena samples the overlaps graph to accurately determine the paired distance distribution. This parameter specifies the maximum distance that is searched during this sampling. This value has to be set to at least 2X the expected size of the longest paired-end library.": "Edena 通过抽样重叠图来准确确定配对距离分布。此参数指定了在抽样过程中搜索的最大距离。这个值必须至少设置为最长配对末端库预期大小的 2 倍。",
    "Edena samples the overlaps graph to accurately determine the paired distance distribution. This parameter specifies the maximum distance that is searched during this sampling. This value has to be set to at least 2X the expected size of the longest mate-pair library.": "Edena 样本通过抽样重叠图来准确确定配对距离分布。这个参数指定了在抽样期间搜索的最大距离。这个值必须至少设置为最长配对库预期大小的 2 倍。",
    "FASTA or FASTQ format": "FASTA或FASTQ格式",
    "If not specified, this value is set to half of the reads length. When the sequencing coverage is sufficient, you can increase this value which will reduce the computational time. Edena will compute the overlaps whose sizes range from this value to the reads length.": "如果不指定，这个值将设置为读取长度的一半。当测序覆盖度足够时，你可以增加这个值，这将减少计算时间。Edena将计算大小在这个值到读取长度范围内的重叠。",
    "Use this option to truncate the 3’end of the reads to the specified length. You may consider reads truncation since it can significantly improve the assembly. Since Edena computes exact overlaps, only error free reads can take part to the assembly. Since errors are likely to occur at the 3’ ends, shortening the reads by some nucleotides may increase the number of errors-free reads in the dataset, and thus increase the assembly performance.": "使用这个选项可以截断读取序列的3'端到指定的长度。您可能需要考虑截断读取，因为它可以显著提高组装效果。由于Edena计算的是精确的重叠区域，只有无误的读取序列才能参与组装。由于错误很可能发生在3'端，通过减少一些核苷酸的长度可以增加数据集中无误读取序列的数量，从而提高组装性能。",
    "Insert size for the library (-d)": "文库的插入大小（-d）",
    "High coverage contigs (above mean_coverage + h x std_coverage) are not considered in the scaffold assembly mainly to exclude reads from repetitive regions": "高覆盖度的连续序列（高于平均覆盖度加上h倍的标准覆盖度）在支架组装中不考虑，主要是为了排除来自重复区域的读取。",
    "Optional, specify only if different from read set": "可选，仅当与读取集不同时指定",
    "-minNR, default 3": "最小非冗余数，默认为3",
    "-minRL, default 19": "最小读取长度，默认值为19",
    "-maxRL, default 38": "-最大读取长度，默认为38",
    "or spacer region, -minSL, default 19": "或间隔区域，-minSL，默认值为19",
    "or spacer region, -maxSL, default 48": "或间隔区，-maxSL，默认48",
    "or spacer region": "或间隔区",
    "Please select a output format.": "请选择一个输出格式。",
    "This file is a two column tabular file for example generated with the histo function of Jellyfish.": "这个文件是一个两列制表格式的文件，例如使用Jellyfish的histo函数生成的。",
    "This parameter is optional": "这个参数是可选的",
    "K-mers with coverage greater than max_kmercov are ignored by the model. This parameter is optional": "覆盖度大于max_kmercov的k-mers会被模型忽略。此参数是可选的。",
    "Flag for topology for model to use. Topology refers to the similarities between distinct homologues. Each triploid topology consists of two nucleotide heterozygosity forms (e.g. aab and abc), while each tetraploid, pentaploid, and hexaploid topology consists of three, four, and five heterozygosity forms respectively.": "为模型使用的拓扑结构标记。拓扑结构指的是不同同源体之间的相似性。每个三倍体拓扑结构由两种核苷酸杂合形式组成（例如aab和abc），而每个四倍体、五倍体和六倍体拓扑结构分别由三种、四种和五种杂合形式组成。",
    "Nucleotide hetegozygosity rates should be float values (separated by commas if necessary)": "核苷酸杂合率应该是浮点数值（如果需要，可以用逗号分隔）",
    "Default: 1": "默认值：1",
    "Input NGS FASTA": "输入 NGS FASTA",
    "Input BioNano CMAP": "输入 BioNano CMAP",
    "It defines the parameters used in each step of hybrid scaffold": "它定义了杂交支架中每个步骤所使用的参数",
    "Define single enzyme for in-silico FASTA to CMAP digestion. Avalible enzymes: BspQI, BbvCI, BsmI, BsrDI, BssSI, DLE1.": "定义单个酶用于在硅片上将FASTA文件转化为CMAP格式的消化。可用的酶有：BspQI、BbvCI、BsmI、BsrDI、BssSI、DLE1。",
    "Depending on the complexity of the genome of interest, the values of certain parameters could be adjusted accordingly.": "根据所关注基因组的复杂性，某些参数的值可以相应地进行调整。",
    "Input a conflict resolution file indicating which NGS and BioNano conflicting contigs to be cut [optional]": "输入一个冲突解决文件，指明要切割的NGS和BioNano冲突的contigs [可选]",
    "Conflict filter level genome maps [required if not using -M option]": "冲突过滤级别基因组图谱 [如果不使用 -M 选项则必需]",
    "Conflict filter level for sequences [required if not using -M option]": "序列冲突过滤级别【如果未使用-M选项，则为必需】",
    "Generate a collection with the files contained in the hybrid_scaffolds folder": "生成一个包含在 hybrid_scaffolds 文件夹中的文件的集合。",
    "The hybrid scaffold output package (.zip) can be imported into Access for visualization": "混合支架输出包（.zip）可以导入到Access中进行可视化",
    "This option removes the spurious BioNano cut sites that are inserted into gaps in some assemblies, replacing them with Ns.": "这个选项移除了在某些组装中插入到间隙里的假的BioNano切割位点，将它们替换为Ns。",
    "This is usually the forward reads in your experiment.": "这通常是您实验中的正向读取。",
    "This is usually the reverse reads in your experiment.": "这通常是你实验中的反向读取。",
    "Species name in binomial nomenclature": "二名法中的物种名称",
    "Required for NCBI database query": "需要用于NCBI数据库查询",
    "The reads are mapped to close-related species (to pull out mito-reads and exclude possible NUMTS) and assembled by using Hifiasm": "读取映射到近缘物种（以提取线粒体读取并排除可能的核糖体DNA序列）并使用Hifiasm进行组装。",
    "For small genomes, use -f0 to disable the initial bloom filter which takes 16GB memory at the beginning. For genomes much larger than human, applying -f38 or even -f39 is preferred to save memory on k-mer counting.": "对于小型基因组，使用-f0来禁用初始的布隆过滤器，该过滤器在开始时会占用16GB的内存。对于比人类基因组大得多的基因组，应用-f38甚至-f39以节省k-mer计数上的内存是首选。",
    "Assembled contigs/scaffolds in FASTA format will be used to find the mitogenome": "组装的连续序列/支架以FASTA格式用于寻找线粒体基因组",
    "Close-related mitogenome is fasta format": "密切相关的线粒体基因组是fasta格式",
    "Close-related mitogenome is genbank format": "密切相关的线粒体基因组是GenBank格式",
    "Organism genetic code following NCBI table (for mitogenome annotation)": "生物体遗传密码遵循NCBI表格（用于线粒体基因组注释）",
    "Percentage of query in the blast match with close-related mito. By default it is going to chose any contig which has 50% or more of its length in the blast match.": "在与密切相关的线粒体的BLAST匹配中查询的百分比。默认情况下，它将选择任何在其BLAST匹配中有50%或更多长度的片段。",
    "Size to consider when checking for circularization": "在检查环化时要考虑的大小",
    "Offset from start and finish to consider when looking for circularization": "寻找环化时考虑起始和结束的偏移量",
    "This option creates a zip file with all the files and folders generated by MitoHiFi.": "这个选项创建一个包含MitoHiFi生成的所有文件和文件夹的zip文件。",
    "Without validation, the module will break at any point of reference discordance as defined by the 'correction options'. With validation, RagTag maps reads to the query assembly and verifies putative break points if they are near regions of exceptionally low or high coverage. The reads used for validation should come from the same genotype as the query assembly to ensure that coverage abnormalities don't arise from true biological variation": "如果没有验证，该模块将在由“校正选项”定义的任何参考不一致点处中断。有了验证，RagTag将读取映射到查询组装上，并在假定的断点附近是异常低或高覆盖区域时验证它们。用于验证的读取应来自与查询组装相同的基因型，以确保覆盖异常不是由真正的生物学变异引起的。",
    "This parameter specifies the window around the putative misassembly break point that RagTag examines for exceptionally low or high read coverage. The larger this window size, the more likely it is to find an unrelated coverage abnormality": "这个参数指定了RagTag检查假定的组装错误断裂点周围的窗口，以寻找异常低或高的读取覆盖度。这个窗口大小越大，越有可能发现一个无关的覆盖度异常。",
    "Breaks will not be made within -b bp of query sequence termini": "断裂不会在查询序列末端的-b个碱基范围内产生",
    "One can also direct RagTag to only break misassemblies between (--inter, query maps to >1 reference sequence) or within (--intra, query maps discordantly to 1 reference sequence) reference sequences": "用户还可以直接让RagTag只打断（--inter，查询映射到>1个参考序列）或（--intra，查询不和谐地映射到1个参考序列）参考序列之间的错误组装。",
    "If one has annotations associated with the query assembly, provide them with the --gff option to ensure that the query assembly is never broken within annotation intervals.": "如果查询组装与注释相关联，请使用--gff选项提供它们，以确保查询组装永远不会在注释间隔内被打断。",
    "The grouping confidence score is the number of base pairs a contig covered in its assigned reference chromosome divided by the total number of covered base pairs in the entire reference genome": "分组置信度分数是重叠群在其分配的参考染色体上覆盖的碱基对数除以整个参考基因组中覆盖的碱基对总数。",
    "To create a metric associated with contig ordering confidence, Ragtag define a location confidence. First, the smallest and largest alignment positions, with respect to the reference, between a contig and its assigned reference chromosome are found. The location confidence is then calculated as the number of covered base pairs in this range divided by the total number of base pairs in the range": "为了创建一个与连续序列排序置信度相关的指标，Ragtag定义了一个位置置信度。首先，找到关于参考序列，一个连续序列及其分配的参考染色体之间的最小和最大比对位置。然后，位置置信度被计算为在这个范围内覆盖的碱基对数除以该范围内总碱基对数。",
    "To calculate the orientation confidence, each base pair in each alignment between a contig and its assigned reference chromosome casts a vote for the orientation of its alignment. The orientation confidence is the number of votes for the assigned orientation of the contig divided by the total number of votes": "为了计算方向置信度，每个一致性中每个碱基对在一致性序列和其分配的参考染色体之间的比对中为其比对的方向投一票。方向置信度是分配给一致性序列方向的投票数除以总投票数。",
    "When disabled, all gaps are 100 bp (-r)": "当禁用时，所有间隙都是100个碱基对（-r）",
    "After merging, alignments less than -s bp long will be removed": "合并后，小于 -s bp 长度的比对将被移除",
    "Maximum merged alignment distance from sequence terminus as fraction of the sequence length. Alignments must be within -i bp of a target sequence terminus or gap to be considered for patchin": "序列末端与合并对齐距离的最大比例。对齐必须在目标序列末端或间隙的-i个碱基对以内才能被考虑进行修补。",
    "At least two AGP files are required.": "至少需要两个AGP文件。",
    "Assembly sequences shorter than -l will also be left unplaced.": "比 -l 参数短的序列组装也将不被放置。",
    "The edges in the merging graph represent scaffolding adjacencies. If an AGP file supports a particular adjacency, its weight is added to the edge weight. Any edges with a weight lower than the minimum edge weigth will be removed from the graph": "在合并图中，边代表支架相邻关系。如果一个AGP文件支持特定的相邻关系，其权重将被添加到边的权重中。任何权重低于最小边权重的边将从图中移除。",
    "Scaffold gaps can differ between input AGP files. For example, a Hi-C derived AGP file might place 100 bp gaps between sequences while a reference-guided AGP file might infer gap sizes based on a reference genome. Use this parameter to specify how gap sizes should be computed from the supporting AGP files (--gap-func)": "支架间隙在输入的AGP文件中可能不同。例如，Hi-C衍生的AGP文件可能在序列之间放置100个碱基对的间隙，而参考引导的AGP文件可能会根据参考基因组推断间隙大小。使用此参数来指定应如何从支持的AGP文件计算间隙大小（--gap-func）",
    "Sorted by read name": "按读取名称排序",
    "List of restrction enzimes/sites or 'DNase', separated by comma. E.g. GATC,GACC": "限制性内切酶/位点列表或'DNase'，用逗号分隔。例如 GATC,GACC",
    "Although multiple rounds of racon can increase the quality of an assembly there are indications that it also fragments the assembly and may decrease quality by removing structural variants and SNPs. Published assembly workflows differ in the number of rounds but rarely apply more than 4 rounds of racon": "尽管多轮racon可以提高组装质量，但有迹象表明它也会使组装变得碎片化，并通过移除结构变异和SNPs来降低质量。已发布的组装工作流程在轮数上有所不同，但很少应用超过4轮的racon。",
    "Pilon can utilize Illumina short reads mapped to the draft assembly to improve the local accuracy of the sequence by correcting sequence errors, fixing misassemblies, and filling gaps": "Pilon 可以利用 Illumina 短读段映射到草图组装上，通过纠正序列错误、修复错误组装和填补间隙来提高序列的局部准确性。",
    "Split candidates into # of chunks to separately correct errors": "将候选项分割成单独纠正错误的#个块",
    "Input file name containing the alignments of short reads against the draft (must have CIGAR information)": "输入包含短读段与草图比对的文件名（必须包含CIGAR信息）",
    "Input file name containing the alignments of long reads against the draft (must have CIGAR information). Optional (only Short reads polishing will be performed if this argument is not given)": "输入文件名，包含长读段与草图的比对（必须有CIGAR信息）。可选（如果未给出此参数，则仅执行短读段打磨）",
    "A number can be followed by units k/m/g; e.g. 10m, 2.3g.": "一个数字后面可以跟随单位 k/m/g；例如 10m, 2.3g。",
    "Default value is 5": "默认值是5",
    "Default value is -4": "默认值是-4",
    "Default value is -8 (must be negative)": "默认值是-8（必须是负数）",
    "Default value is 3": "默认值是3",
    "Default value is -5": "默认值是 -5",
    "Default value is -4 (must be negative)": "默认值是-4（必须是负数）",
    "Threshold for Normalised Edit Distance of long arms allowed in a window (in percentage). Higher number means more arms allowed which may slow down the execution. Default value is 20": "允许在窗口中长臂标准化编辑距离的阈值（以百分比表示）。数值越高，允许的臂数越多，这可能会减慢执行速度。默认值为20。",
    "The reads with mapping quality below this threshold will not be taken into consideration. Default value is 2": "映射质量低于此阈值的读取将不被考虑。默认值为2。",
    "Lower value means less memory usage but slower speed. By default, all the contigs in the draft": "较低的值意味着内存使用较少，但速度较慢。默认情况下，草图中的所有连续片段",
    "Sequences to scaffold": "序列到支架",
    "Long sequence reads for scaffolding": "长序列读取用于支架构建",
    "Distance between k-mer pairs. Multiple distances are separated by comma, e.g. 500,1000,2000,3000": "k-mer对之间的距离。多个距离用逗号分隔，例如500,1000,2000,3000",
    "Step of sliding window when extracting k-mer pairs from long reads. Multiple steps are separated by comma, e.g. 10,5": "从长读段中提取k-mer对时滑动窗口的步长。多个步长用逗号分隔，例如 10,5",
    "Maximum link ratio between two best contig pairs. Higher values lead to least accurate scaffolding": "两个最佳重叠群对之间的最大链接比率。较高的值会导致支架构建的准确性最低。",
    "Error (%) allowed on -d distance": "允许在 -d 距离上的误差（%）",
    "Minimum number of links (k-mer pairs) to compute scaffold": "最小链接数（k-mer对）以计算支架",
    "Offset position for extracting k-mer pairs": "提取k-mer对的偏移位置",
    "Increase to prevent memory allocation errors": "增加以防止内存分配错误",
    "Minimum contig length to consider for scaffolding": "用于支架化的最小连续序列长度",
    "You can specify a AGP format file to ask YaHS to do scaffolding with the scaffolds in the AGP file as the start point": "您可以指定一个 AGP 格式文件，要求 YaHS 以 AGP 文件中的支架作为起点进行支架搭建。",
    "Comma separated, ascending list of range of resolutions with no spaces. Ex. 50000,100000,200000,500000,1000000,2000000,5000000. By default and the upper limit is automatically adjusted with the genome size": "逗号分隔的升序分辨率范围列表，没有空格。例如：50000,100000,200000,500000,1000000,2000000,5000000。默认情况下，上限会根据基因组大小自动调整。",
    "Automatically active if AGP output is set": "如果AGP输出被设置，则自动激活",
    "Will also prevent any break.agp output files": "也会阻止生成任何 break.agp 输出文件",
    "Prepared by velveth.": "由velveth准备。",
    "Generates Graph2 dataset": "生成 Graph2 数据集",
    "minimum contig length exported to contigs.fa file (default: hash length * 2).": "输出到contigs.fa文件的最小序列长度（默认值：哈希长度乘以2）。",
    "minimum contig length exported to contigs.fa file (default: hash length * 2)": "导出到contigs.fa文件的最小片段长度（默认值：哈希长度 * 2）",
    "Expected distance between two paired end reads": "两个配对末端读取之间的预期距离",
    "Estimate of standard deviation of Paired-End Read dataset (default: 10% of corresponding length)": "配对末端读取数据集的标准偏差估计（默认：相应长度的10%）",
    "Expected distance between two paired end reads in the second short-read dataset": "第二个短读数据集中两个配对末端读取之间的预期距离",
    "Estimate of standard deviation of 2nd Paired-End Read dataset (default: 10% of corresponding length)": "第二对端读取数据集的标准差估计（默认值：相应长度的10%）",
    "Expected distance between two long paired-end reads": "两个长配对末端读取之间的预期距离",
    "Estimate of standard deviation of Long Paired-End Read dataset (default: 10% of corresponding length)": "长配对末端读取数据集的标准差估计（默认值：相应长度的10%）",
    "maximum length in base pair of bubble (default: 100)": "泡沫最大长度（以碱基对为单位，默认值：100）",
    "maximum divergence rate between two branches in a bubble (default: .2)": "两个分支在泡沫中的最大发散率（默认值：.2）",
    "maximum number of gaps allowed in the alignment of the two branches of a bubble (default: 3)": "在两个分支的泡沫对齐中允许的最大间隙数（默认值：3）",
    "minimum number of paired end connections to justify the scaffolding of two long contigs (default: 10)": "最小配对末端连接数以证明两个长片段的支架化（默认值：10）",
    "Exclude data that has coverage more than this maximum coverage value": "排除覆盖度高于此最大覆盖度值的数据",
    "minimum number of long reads required to merge contigs (default: 2)": "长读长序列合并重叠群所需的最小数量（默认值：2）",
    "Scaffold contigs that it cannot quite be connected (This results in sequences of Ns in the contigs)": "将无法完全连接的支架连接起来（这会导致连接子序列中出现N的序列）",
    "k-mer length in base pairs of the words being hashed.": "哈希所用的碱基对数的k-mer长度。",
    "If you are using a strand specific transcriptome sequencing protocol, you may wish to use this option for better results.": "如果您使用的是特定于链的转录组测序协议，您可能希望使用此选项以获得更好的结果。",
    "-o option. must be > 15. 40 by default": "-o 选项。必须大于15。默认为40。",
    "-p option. must be > 65. 90 by default": "-p 选项。必须大于 65。默认为 90。",
    "For more control, create your own Kmer dump using Jellyfish. See Smudgeplot on GitHub for more details: https://github.com/KamilSJaron/smudgeplot": "为了获得更多的控制权，请使用Jellyfish创建您自己的Kmer转储。有关更多详细信息，请查看GitHub上的Smudgeplot：https://github.com/KamilSJaron/smudgeplot",
    "Sequencing reads corresponding to your genome. Don't worry about read pairing as it is not used in Kmer-counting. If selecting multiple datasets, please do not mix datatypes!": "对应于您基因组的测序读取。不用担心读取配对，因为在Kmer计数中不使用它。如果选择多个数据集，请不要混合数据类型！",
    "The size of k-mers should be large enough allowing the k-mer to map uniquely to the genome": "k-mer的大小应该足够大，以便k-mer能够唯一地映射到基因组上",
    "Upload your own Kmer dump file created with the Jellyfish or KMC tool. This enables control over kmer-counting parameters.": "上传您自己使用Jellyfish或KMC工具创建的Kmer转储文件。这可以控制kmer计数参数。",
    "Assume no heterozygosity in the genome - plotting a paralog structure.": "假设基因组中没有杂合性 - 绘制同源异形结构。",
    "Sequence names in the Pretext file must match sequence names in the bedgraph data; although relative sort order is unimportant.": "Pretext 文件中的序列名称必须与 bedgraph 数据中的序列名称相匹配；尽管相对排序顺序不重要。",
    "Genbank files with annotated regions to extract. Multiple files can be selected.": "带有注释区域的Genbank文件以提取。可以选择多个文件。",
    "Maximum number of reads to be used. An input VALUE of 0 will be treated as infinity (INF). Default: 1.5E7 (plant chloroplast, plant ribosomal, fungus mitochondrial, fungus nuclear) 7.5E7 (Plant mitochondrial) 3E8 (Animal mitochondrial)": "最大使用读段数。输入值为0将被视为无穷大（INF）。默认值：1.5E7（植物叶绿体，植物核糖体，真菌线粒体，真菌核）7.5E7（植物线粒体）3E8（动物线粒体）",
    "Maximum number of reads to be used according to target-hitting base coverage. If the estimated target-hitting base coverage is too high and over this VALUE, GetOrganelle automatically reduce the number of reads to generate a final assembly with base coverage close to this VALUE. This design could greatly save computational resources in many situations. A mean base coverage over 500 is extremely sufficient for most cases. This VALUE must be larger than 10. An input VALUE of 0 will be treated as infinity (INF). Default: 500.": "根据目标击中基础覆盖度使用的最大读取数。如果估计的目标击中基础覆盖度过高并且超过这个值，GetOrganelle会自动减少读取数以生成基础覆盖度接近这个值的最终组装。这种设计可以在许多情况下大大节省计算资源。平均基础覆盖度超过500对于大多数情况来说已经非常足够了。这个值必须大于10。输入值为0将被视为无穷大（INF）。默认值：500。",
    "Word size for pre-grouping and extending process. This script would try to guess (auto-estimate) a proper word size using an empirical function based on average read length, reads quality, target genome coverage, and other variables that might influence the extending process. Default: estimated automatically": "预分组和扩展过程中的单词大小。该脚本将尝试使用基于平均读取长度、读取质量、目标基因组覆盖度以及其他可能影响扩展过程的变量的实证函数来自动估计（自动估算）合适的单词大小。默认值：自动估算",
    "Maximum number of extending rounds (suggested: >=2). Default: 15 (-F embplant_pt), 30 (-F embplant_mt/other_pt), 10 (-F embplant_nr/animal_mt/fungus_mt/fungus_nr), inf (-P 0).": "最大扩展轮数（建议：>=2）。默认值：15（-F embplant_pt），30（-F embplant_mt/other_pt），10（-F embplant_nr/animal_mt/fungus_mt/fungus_nr），无穷大（-P 0）。",
    "Use the same format as in SPAdes. Illegal kmer values would be automatically discarded by GetOrganelle. Default: 21,55,85,115": "使用与SPAdes相同的格式。不合法的kmer值将被GetOrganelle自动丢弃。默认值：21,55,85,115",
    "The length of step for checking words in reads during extending process (integer >= 1). When you have reads of high quality, the larger the number is, the faster the extension will be, the more risk of missing reads in low coverage area. Choose 1 to choose the slowest but safest extension strategy. Default: 3": "在扩展过程中检查读取中的单词的步长（整数 >= 1）。当您有高质量的读取时，数字越大，扩展速度越快，低覆盖区域遗漏读取的风险越大。选择 1 以选择最慢但最安全的扩展策略。默认值：3",
    "The length of step for building words from seeds during extending process (integer >= 1). When you have reads of high quality, the larger the number is, the faster the extension will be, the more risk of missing reads in low coverage area. Another usage of this mesh size is to choose a larger mesh size coupled with a smaller word size, which makes smaller word size feasible when memory is limited. Choose 1 to choose the slowest but safest extension strategy. Default: 2": "在扩展过程中从种子构建单词的步长（整数 >= 1）。当您有高质量的读取时，数字越大，扩展速度越快，低覆盖区域遗漏读取的风险越大。这种网格大小的另一种用途是选择较大的网格大小，配合较小的单词大小，这在内存有限时使较小的单词大小成为可能。选择 1 以选择最慢但最安全的扩展策略。默认值：2",
    "The maximum number (integer) of high-covered reads to be pre-grouped before extending process. pre_grouping is suggested when the whole genome coverage is shallow but the organ genome coverage is deep. The default value is 2E5. For personal computer with 8G memory, we suggest no more than 3E5. A larger number (ex. 6E5) would run faster but exhaust memory in the first few minutes. Choose 0 to disable this process.": "最大数目（整数）的高覆盖度的reads在扩展过程之前要被预先分组。当整个基因组覆盖度浅但器官基因组覆盖度深时，建议使用pre_grouping。默认值是2E5。对于有8G内存的个人电脑，我们建议不超过3E5。一个更大的数目（例如6E5）会运行得更快，但会在前几分钟耗尽内存。选择0来禁用这个过程。",
    "Maximum extending length(s) derived from the seed(s). A single value could be a non-negative number. An input VALUE of 0 will be treated as infinity (INF) and is the default. This is designed for properly stopping the extending from getting too long and saving computational resources. However, empirically, a maximum extending length value larger than 6000 would not be helpful for saving computational resources. This value would not be precise in controlling output size, especially when pre-group (followed by '-P') is turn on. A sequence of a closely related species would be preferred for estimating a better maximum extending length value. If you are using limited loci, e.g. rbcL gene as the seed for assembling the whole plastome (with extending length ca. 75000 >> 6000), you should set maximum extending length to inf. Default: inf.": "从种子（seed）得出的最大延伸长度。单个值可以是非负数。输入值0将被视为无穷大（INF），并且是默认值。这是为了适当地停止延伸，以避免延伸过长并节省计算资源。然而，经验上，最大延伸长度值大于6000对于节省计算资源并没有帮助。这个值在控制输出大小时不会特别精确，特别是当预分组（由'-P'后跟）被打开时。使用密切相关物种的序列来估计更好的最大延伸长度值是首选。如果您正在使用有限的位点，例如rbcL基因作为组装整个叶绿体基因组的种子（延伸长度约为75000 >> 6000），则应将最大延伸长度设置为inf。默认值：inf。",
    "='-R 10 -t 4 -J 5 -M 7 --max-n-words 3E7 --larger-auto-ws --disentangle-time-limit 360' This option is suggested for homogeneously and highly covered data (very fine data). You can overwrite the value of a specific option listed above by adding that option along with the '--fast' flag. You could try GetOrganelle with this option for a list of samples and run a second time without this option for the rest with incomplete results.": "='-R 10 -t 4 -J 5 -M 7 --max-n-words 3E7 --larger-auto-ws --disentangle-time-limit 360' 这个选项适用于均匀且高度覆盖的数据（非常精细的数据）。你可以通过添加带有 '--fast' 标志的特定选项来覆盖上面列出的特定选项的值。你可以尝试使用这个选项和 GetOrganelle 运行一次样本列表，然后不使用这个选项再次运行剩余的结果不完整的样本。  角色设定：  你是一位资深的生物信息学专家，对于生物信息学的专业术语和软件使用具有非常深刻的理解。 你非常擅长在Galaxy生信云平台上使用各种分析工具，并且非常擅长在Galaxy上使用各种分析工具。  角色偏好：  你非常擅于将生物信息学专业术语翻译成中文专业术语。  特别要注意的点：  你要从所需翻译的内容中区分哪些是生物信息学软件或者软件包的参数，遇到参数时需要保持原文不做翻译。 括符中以横线'-'或下划线'_'开头的内容，通通不翻译。  输出格式：  接收一行文本，输出一行翻译后的文本。",
    "='--out-per-round -P 0 --remove-duplicates 0' You can overwrite the value of a specific option listed above by adding that option along with the '--memory-save' flag. A larger '-R' value is suggested when '--memory-save' is chosen.": "='--out-per-round -P 0 --remove-duplicates 0' 你可以通过添加带有 '--memory-save' 标志的特定选项来覆盖上面列出的某个选项的值。当选择 '--memory-save' 时，建议使用较大的 '-R' 值。",
    "By using this flag, the empirical function for estimating W would tend to produce a relative larger W, which would speed up the matching in extending, reduce the memory cost in extending, but increase the risk of broken final graph. Suggested when the data is good with high and homogenous coverage.": "通过使用这个标志，用于估计W的经验函数将倾向于产生一个相对较大的W，这将加快扩展时的匹配速度，减少扩展时的内存消耗，但增加最终图断裂的风险。建议在数据质量好、覆盖度高且均匀时使用。",
    "Number of full Racon polishing rounds. Default: 2": "完整的Racon抛光轮数。默认值：2",
    "Use this flag for PacBio reads to make Minipolish use the map-pb Minimap2 preset. Default: assumes Nanopore reads and uses the map-ont preset": "使用此标志处理PacBio读取，以使Minipolish使用map-pb Minimap2预设。默认情况下：假定为Nanopore读取，并使用map-ont预设。",
    "Appropriate if the input GFA does not have 'a' lines. Default: do the initial polishing round": "如果输入的GFA没有'a'行，则适用。默认：执行初始打磨轮次",
    "Increasing level by one means a threefold smaller resolution but also a threefold faster computation time.": "增加一个级别意味着分辨率减少三倍，但计算时间也减少三倍。",
    "A high number of cycles has diminishing returns but there is a necessary minimum for assembly convergence.": "高周期数会带来递减的回报，但组装收敛需要一个必要的最小值。",
    "Coverage below which fragments should be filtered out prior to binning.": "在分箱之前，应该过滤掉覆盖度低于该值的片段。",
    "Input read files (FASTQ or FASTA). To select more than one file or collection from your history, use the 'ctrl' key": "输入读取文件（FASTQ或FASTA）。要从历史记录中选择多个文件或集合，请使用'ctrl'键。",
    "Estimated size of genome (bp)": "估计的基因组大小（碱基对）",
    "Minimum length for input reads": "输入序列的最小长度",
    "Number of reads to correct in terms of genome coverage. For a 4Gb genome, setting correction coverage = 10 will correct the longest 40Gb worth of reads from the input fastq.": "在基因组覆盖度方面的校正读取数。对于一个4Gb的基因组，设置校正覆盖度=10将从输入的fastq文件中校正最长40Gb的读取。",
    "Number of reads to use in genome assembly in terms of genome coverage": "用于基因组组装的测序深度",
    "Polish contigs as final step after briding": "在桥接后的最后步骤中完成对片段的修饰",
    "Warning: only change these if you really know what you are doing": "警告：只有在你确实知道自己在做什么的情况下才更改这些设置",
    "This file describes the positions of enzymatic labelling sites on the reference genome": "这个文件描述了参考基因组上酶标记位点的位置",
    "This file describes the positions of enzymatic labelling sites on the contigs": "这个文件描述了酶标记位点在连续片段上的位置",
    "This file describes the alignments of contig labels on the anchor": "这个文件描述了锚点上连续标签的对齐情况",
    "Contigs file (FASTA) that was scaffolded by the Bionano scaffold": "由Bionano进行支架化的拼接序列文件（FASTA格式）",
    "Bionano key file giving the correspondance between maps and contigs": "Bionano 关键文件，提供映射和 contigs（基因组片段）之间的对应关系",
    "This argument is used to provide the final XMAP file containing the mappings of labels of both enzymes. This argument is useful (and recommended) to ensure that no mapping has been missed inside one of the individual XMAP file": "这个参数用于提供最终的XMAP文件，其中包含两个酶的标签映射。这个参数很有用（并且推荐使用），以确保在个别XMAP文件中没有遗漏任何映射。",
    "enables the sequence similarity scaffolding. In a first phase, BiSCoT will search similarities between contigs based on label mappings. If this parameter is set, BiSCoT will search for sequence similarity to close gaps created by the first step": "启用序列相似性骨架化。在第一阶段，BiSCoT将基于标签映射搜索片段之间的相似性。如果设置了这个参数，BiSCoT将搜索序列相似性以关闭第一步创建的间隙。",
    "To be retained, an alignment position in --xmap-1 or --xmap-2 has to be present in --xmap-2enz": "为了保留，--xmap-1 或 --xmap-2 中的对齐位置必须出现在 --xmap-2enz 中",
    "Log file can be helpful for debuggin purposes": "日志文件对于调试目的可能很有帮助",
    "Input reads for assembly": "组装输入序列",
    "Drop reads according to configurable runtime kmer frequency thresholds": "根据可配置的运行时kmer频率阈值丢弃读段",
    "If enabled, hifiasm_meta will estimate the total number of read overlaps. If the estimation seems acceptable, no read will be dropped; otherwise, reads will be dropped from the most redundant ones until the criteria are satisfied. This can only be enabled if forced read selection is disabled.": "如果启用了hifiasm_meta，将估计读取重叠的总数。如果估计看起来可以接受，则不会丢弃任何读取；否则，将从最冗余的读取开始丢弃读取，直到满足条件为止。这只能在禁用强制读取选择时启用。",
    "Select a BlobToolKit module": "选择一个 BlobToolKit 模块",
    "Optional metadata dataset": "可选元数据集",
    "Add ranks to metadata for a taxid": "为某个taxid的元数据添加排名",
    "This file should be generated by Blobtool create": "这个文件应该由Blobtool create生成",
    "Tabular BLAST/Diamond output file": "表格化BLAST/Diamond输出文件",
    "Rule to use when assigning BLAST hits to taxa": "分配BLAST命中结果到分类群时使用的规则",
    "The smaller the E-value, the better the match. Any hits with an evalue weaker than the value specified will be excluded": "E值越小，匹配度越好。任何E值比指定值更弱的命中都将被排除",
    "The higher the bit-score, the better the sequence similarity. Any hits with an bitscore lower the value specified will be excluded": "比特分数越高，序列相似性越好。任何比特分数低于指定值的命中都将被排除。",
    "By default the 10 highest scoring hits to a given taxon will be used when applying the --taxrule": "默认情况下，当应用 --taxrule 参数时，将使用给定分类群的10个最高分命中结果。",
    "tRNAscan-SE is employed for identifying and annotating tRNA genes in genomes": "tRNAscan-SE 被用于在基因组中识别和注释 tRNA 基因",
    "Prevent fields in files with duplicate identifiers being loaded as array fields": "防止具有重复标识符的文件中的字段作为数组字段加载",
    "This file should have been generated with the previous Blobtools tool version": "这个文件应该使用之前的Blobtools工具版本生成",
    "TSV file containing current identifiers and synonyms": "包含当前标识符和同义词的制表符分隔值（TSV）文件",
    "Flag to use new taxrule as default category": "使用新分类规则作为默认类别的标志",
    "Key/value pairs to pass to samtools pileup": "传递给samtools pileup的键/值对",
    "Allow existing fields to be overwritten": "允许覆盖现有字段",
    "This file should be generated by the moudule create": "这个文件应该由模块创建生成",
    "String of type param=value. Individual param=value pairs can be specified to filter based on Variable or Category fields": "参数类型为param=value的字符串。可以指定个别的param=value对，根据变量或类别字段进行过滤。",
    "Space separated list of identifiers": "以空格分隔的标识符列表",
    "JSON format list file as generated by BlobtoolKit Viewer": "由 BlobtoolKit Viewer 生成的 JSON 格式列表文件",
    "FASTA format assembly file to be filtered": "FASTA格式的组装文件需要被过滤",
    "FASTQ format read file to be filtered": "FASTQ格式的读取文件需要被过滤",
    "Taxonomic level for summary": "摘要的分类水平",
    "Invert the filter (exclude matching records)": "反转过滤器（排除匹配记录）",
    "You can use raw or corrected reads.": "您可以使用原始或校正的读取数据。",
    "filter reads with length smaller than read_cutoff": "过滤长度小于read_cutoff的读取",
    "It is requried to perform some simple statistics (such as length distribution, total amount of data and sequencing depth) on the input data": "需要对输入数据执行一些简单的统计分析（例如长度分布、数据总量和测序深度）。",
    "Estimated genome size, suffix K/M/G recognized, used to calculate seed_cutoff/seed_cutfiles/blocksize and average depth, it can be omitted when manually setting seed_cutoff. Spaces are not allowed.": "估计的基因组大小，识别后缀 K/M/G，用于计算种子截止值/种子截止文件/区块大小和平均深度，当手动设置种子截止值时可以省略。不允许有空格。",
    "Expected seed depth, used to calculate seed_cutoff, co-use with genome_size, you can try to set it 30-45 to get a better assembly result.": "预期的种子深度，用于计算种子截止值，与基因组大小共同使用，您可以尝试将其设置为30-45以获得更好的组装结果。",
    "Minimum seed length. Set it to 0 for calculating it automatically.": "最小种子长度。将其设置为0以自动计算。",
    "Block size for parallel running, split non-seed reads into small files, the maximum size of each file is blocksize.": "并行运行的区块大小，将非种子读取分割成小文件，每个文件的最大大小是区块大小。",
    "This value should be equal or smaller than the average sequencing depth.": "这个值应该等于或小于平均测序深度。",
    "Split the corrected seed with un-corrected regions": "将校正后的种子与未校正区域分开",
    "0.5-1 times faster mode with a little lower accuracy.": "0.5-1倍速度更快的模式，准确度略有降低。",
    "For highly heterozygous genomes, it will increase assembly size.": "对于高度杂合的基因组，它将增加组装大小。",
    "By default it is disabled (value = 0).": "默认情况下它是禁用的（值 = 0）。",
    "/!\\ If gaps are at the same place in all the sequences don't check 'no', you'll face an error": "/!\\ 如果在所有序列中间隙出现在相同位置，不要选择'no'，否则你会面临错误",
    "File(s) with transcripts in FASTA format.": "包含转录本的FASTA格式文件。",
    "File with reference genome containing all chromosomes/scaffolds in FASTA forma.": "包含所有染色体/支架的参考基因组文件，以FASTA格式呈现。",
    "Genome annotation file. We recommend to use files downloaded from GENCODE or Ensembl.": "基因组注释文件。我们建议使用从GENCODE或Ensembl下载的文件。",
    "Use this option if your GTF file already contains genes records, otherwise gffutils will fix it. Note that gffutils may work for quite a long time": "如果您的GTF文件已经包含基因记录，请使用此选项，否则gffutils将进行修正。请注意，gffutils可能需要运行相当长的时间。",
    "Is option if your GTF file already contains transcripts records, otherwise gffutils will fix it.": "如果选项是您的GTF文件已经包含转录本记录，否则gffutils将修复它。",
    "File with read alignments to the reference genome": "参考基因组的读取比对文件",
    "BUSCO allows to detect core genes in the assembled transcripts": "BUSCO 允许在组装后的转录本中检测核心基因",
    "Select a lineage for using BUSCO": "选择一个用于使用BUSCO的谱系",
    "Set if transcripts were assembled using strand-specific RNA-Seq data in order to benefit from knowing whether the transcript originated from the + or - strand": "如果使用特定于链的RNA-Seq数据组装转录本，以便于了解转录本是来自+链还是-链，请设置此选项。",
    "Default value is 50": "默认值是50",
    "BALT is especially useful for aligning long sequences and gapped mapping, which cannot be performed properly by other fast sequence mappers designed for short reads.": "BALT特别适用于对长序列和带有间隔的映射进行比对，这是其他为短读序列设计的快速序列比对工具无法正确执行的。",
    "Run quality asessment for meta-transcriptome assemblies": "运行元转录组组装的质量评估",
    "Use this option if the genome is prokaryotic": "如果基因组是原核生物的，请使用这个选项",
    "The format is chr:start-end, for example \"chr1:14300-125000\".": "格式为chr:start-end，例如“chr1:14300-125000”。",
    "Select the format of the input sample": "选择输入样本的格式",
    "File of cell barcodes. Should contain at least two columns, where the first column has the cell id and the second column contains the barcode sequence.": "细胞条形码文件。应至少包含两列，其中第一列包含细胞ID，第二列包含条形码序列。",
    "If your genome of interest is not listed, contact your Galaxy administrator": "如果您感兴趣的基因组没有列出，请与您的Galaxy管理员联系",
    "Read 1 should contain the transcripts in fastq.gz format": "读取1应该包含fastq.gz格式的转录本",
    "Read 2 should contain UMI and barcodes in fastq.gz format": "Read 2 应该包含 UMI 和条形码，在 fastq.gz 格式中。",
    "Optional file of cell barcodes. If not provied the barcodes will be detected from the reads. Should contain at least two columns, where the first column has the cell id and the second column contains the barcode sequence.": "可选的细胞条形码文件。如果没有提供，条形码将从读取中检测。该文件应至少包含两列，其中第一列有细胞ID，第二列包含条形码序列。",
    "Current supported source is ENSEMBL": "当前支持的来源是 ENSEMBL",
    "This must be in biomaRt ENSEMBL listDatasets() format e.g. hsapiens_gene_ensembl. See the biomaRt user guide here: https://www.bioconductor.org/packages/release/bioc/vignettes/biomaRt/inst/doc/biomaRt.html": "这必须使用biomaRt ENSEMBL listDatasets()的格式，例如hsapiens_gene_ensembl。请参阅biomaRt用户指南：https://www.bioconductor.org/packages/release/bioc/vignettes/biomaRt/inst/doc/biomaRt.html",
    "Barcode start position in Read 1. Positions are 0-indexed so the first base is considered base 0, -1 indicates no barcode. Default: -1": "Read 1 中的条形码起始位置。位置是 0-索引的，因此第一个碱基被认为是碱基 0，-1 表示没有条形码。默认值：-1",
    "Barcode length in Read 1, 0 if no barcode present. Default: 0": "读取1中的条形码长度，如果没有条形码则为0。默认值：0",
    "Barcode start position in Read 2. Positions are 0-indexed so the first base is considered base 0, -1 indicates no barcode.  Default: 6": "在Read 2中的条形码起始位置。位置是0索引的，因此第一个碱基被认为是碱基0，-1表示没有条形码。默认值：6",
    "Barcode length in Read 2, 0 if no barcode present. Default: 8": "Read 2 中的条形码长度，如果没有条形码则为 0。默认值：8",
    "UMI start position in Read 2. Positions are 0-indexed so the first base is considered base 0, -1 indicates no UMI.  Default: 0": "Read 2 中 UMI 的起始位置。位置是 0 索引的，因此第一个碱基被认为是碱基 0，-1 表示没有 UMI。默认值：0",
    "UMI length in Read 2, 0 if no UMI present. Default: 6": "在读取2中的UMI长度，如果没有UMI则为0。默认值：6",
    "If this option is set to Yes, outlier cells will not be removed from the gene count matrix. Default: No": "如果将此选项设置为Yes，则异常值单元格将不会被从基因计数矩阵中移除。默认值：No",
    "If this option is set to Yes, a PDF containing QC plots will be output. Default: Yes": "如果将此选项设置为“是”，则会输出包含质量控制（QC）图表的PDF文件。默认值：是",
    "If this option is set a matrix of QC metrics will be output. Default: No": "如果设置了这个选项，将输出一个QC指标矩阵。默认值：否。",
    "Only valid if FASTQs are input. If this option is set to Yes, a HTML report containing QC metrics will be output. Default: No": "只有在输入文件为FASTQ格式时才有效。如果将此选项设置为Yes，则会输出包含质量控制指标的HTML报告。默认值：No",
    "Default: 20": "默认值：20",
    "Maximum reads processed if detecting barcodes. Default: 1,000,000": "在检测条形码时处理的最大读取数。默认值：1,000,000",
    "Minimum count to keep if detecting barcodes. Barcode will be discarded if it has lower count. This should be set according to --max_reads. Default: 10": "检测条形码时保留的最小计数。如果计数较低，则会丢弃条形码。此值应根据 --max_reads 设置。默认值：10",
    "0 means no correction, 1 means simple correction and merge UMI with distance 1. Default: 1": "0表示不进行校正，1表示进行简单校正并将距离为1的UMI合并。默认值：1",
    "Low abundant is defined as only one copy of one UMI for this gene. Default: No": "低丰度定义为该基因只有一个UMI的副本。默认值：否。",
    "A tsv file that must have an header line": "一个必须有标题行的tsv文件",
    "Perform scaling method on metacell, see Details": "对元细胞进行缩放方法处理，详见详细信息。",
    "first column is the sequence and second column is the name": "第一列是序列，第二列是名称",
    "Comma-separated list of genes to use for building SNN.": "用于构建自组织神经网络（SNN）的基因列表，以逗号分隔。",
    "Text file with one gene per line to use for building SNN. Overrides Features.": "每行一个基因的文本文件，用于构建最近邻网络（SNN），这会覆盖特征（Features）。",
    "Plot SNN graph on tSNE coordinates": "在 tSNE 坐标上绘制 SNN 图",
    "Reduction to use as input for building the SNN": "作为构建SNN输入的简化",
    "Dimensions of reduction to use as input. A comma-separated list of the dimensions to use in construction of the SNN graph (e.g. To use the first 5 PCs, pass 1,2,3,4,5).": "作为输入使用的降维尺寸。用于构建SNN图的尺寸的逗号分隔列表（例如，要使用前5个主成分，传入1,2,3,4,5）。",
    "Assay to use in construction of SNN": "在构建 SNN 时使用的测定方法",
    "Boolean value of whether the provided matrix is a distance matrix; note, for objects of class dist, this parameter will be set automatically.": "布尔值，用于指示提供的矩阵是否为距离矩阵；注意，对于类dist的对象，此参数将自动设置。",
    "Defines k for the k-nearest neighbor algorithm": "为 k-最近邻算法定义 k",
    "Also compute the shared nearest neighbor graph": "同时计算共享最近邻图",
    "Sets the cutoff for acceptable Jaccard index when computing the neighborhood overlap for the SNN construction. Any edges with values less than or equal to this will be set to 0 and removed from the SNN graph. Essentially sets the strigency of pruning (0 --- no pruning, 1 --- prune everything).": "设置计算SNN（最近邻）构建时邻域重叠的Jaccard指数的可接受截止值。任何小于或等于该值的边将被设置为0并从SNN图中移除。本质上，它设置了修剪的严格程度（0 --- 不修剪，1 --- 修剪所有）。",
    "Method for nearest neighbor finding. Options include: rann (default), annoy": "最近邻查找方法。选项包括：rann（默认），annoy",
    "Distance metric for annoy. Options include: euclidean (default), cosine, manhattan, and hamming": "对于 annoy 的距离度量。选项包括：欧几里得（默认），余弦，曼哈顿和汉明。",
    "Name of graph to use for the clustering algorithm.": "用于聚类算法的图名称。",
    "Error bound when performing nearest neighbor seach using RANN; default of 0.0 implies exact nearest neighbor search": "在使用RANN进行最近邻搜索时的错误界限；默认的0.0意味着精确的最近邻搜索",
    "Force recalculation of SNN": "强制重新计算最近邻（SNN）",
    "Number of features to return.": "返回的特征数量。",
    "Function to compute x-axis value (average expression). Default is to take the mean of the detected (i.e. non-zero) values.": "计算x轴值（平均表达量）的函数。默认情况下，是取检测到的（即非零的）值的平均值。",
    "Function to compute y-axis value (dispersion). Default is to take the standard deviation of all values.": "计算 y 轴值（离散度）的函数。默认是取所有值的标准差。",
    "Bottom cutoff on x-axis (mean) for identifying variable genes.": "x轴（均值）上的下限截断，用于识别可变基因。",
    "Top cutoff on x-axis (mean) for identifying variable genes.": "X轴（均值）上用于识别可变基因的顶部截止值。",
    "Bottom cutoff on y-axis (dispersion) for identifying variable genes.": "在y轴（离散度）上识别可变基因的下限截止值。",
    "Top cutoff on y-axis (dispersion) for identifying variable genes.": "在 y 轴（离散度）上用于识别可变基因的顶部截止值。",
    "How to choose top variable features. Choose one of: 'vst', 'mvp', disp.": "如何选择最重要的变量特征。选择以下之一：'vst'、'mvp'、'disp'。",
    "Method for normalization. Default is log-normalization (LogNormalize). Can be 'CLR' or 'RC' additionally.": "标准化方法。默认是日志标准化（LogNormalize）。还可以是'CLR'或'RC'。",
    "Type of assay to normalize for (default is RNA), but can be changed for multimodal analyses.": "要进行标准化的检测类型（默认是RNA），但可以更改以进行多模态分析。",
    "Sets the scale factor for cell-level normalization": "设置细胞水平标准化的比例因子",
    "Comma-separated list of variables to regress out (previously latent.vars in RegressOut). For example, nCount_RNA, or percent.mito.": "以逗号分隔的变量列表，用于回归分析（之前在RegressOut中称为latent.vars）。例如，nCount_RNA或percent.mito。",
    "Use a linear model or generalized linear model (poisson, negative binomial) for the regression.": "使用线性模型或广义线性模型（泊松，负二项式）进行回归。",
    "Regress on UMI count data. Default is FALSE for linear modeling, but automatically set to TRUE if model.use is 'negbinom' or 'poisson'.": "对UMI计数数据进行回归。默认为FALSE，用于线性建模，但如果model.use是'negbinom'或'poisson'，则自动设置为TRUE。",
    "By default data is centered, with this option you can skip centering.": "默认情况下数据是中心化的，使用此选项可以跳过中心化。",
    "By default data is scaled, with this option you can skip scaling.": "默认情况下，数据会被缩放，使用此选项可以跳过缩放。",
    "Max value to return for scaled data. The default is 10. Setting this can help reduce the effects of genes that are only expressed in a very small number of cells. If regressing out latent variables and using a non-linear model, the default is 50.": "返回缩放数据的最大值。默认值是10。设置这个值可以帮助减少只在非常少数细胞中表达的基因的影响。如果回归出潜在变量并使用非线性模型，默认值是50。",
    "Default size for number of genes to scale at in a single computation. Increasing block.size may speed up calculations but at an additional memory cost. Defaults to 1000 if not specified.": "单个计算中用于缩放基因数量的默认大小。增加block.size可能会加快计算速度，但会增加内存成本。如果未指定，默认为1000。",
    "If object contains fewer than this number of cells, don't block for scaling calculations. Defaults to 1000.": "如果对象包含的细胞少于这个数字，不要阻止进行缩放计算。默认值为1000。",
    "Check to see if data has been normalized, if not, output a warning (TRUE by default). Data can be normalised by Seurat normalise module.": "检查数据是否已标准化，如果没有，则输出警告（默认为TRUE）。数据可以通过Seurat的normalize模块进行标准化。",
    "If TRUE, uses the Barnes-hut implementation, which runs faster, but is less flexible. TRUE by default.": "如果为真（TRUE），则使用Barnes-hut实现，其运行速度更快，但灵活性较低。默认为真（TRUE）。",
    "Select the method to use to compute the tSNE. Available methods are: Rtsne, Flt-SNE": "选择用于计算tSNE的方法。可用的方法有：Rtsne, Flt-SNE",
    "The dimensional space of the resulting tSNE embedding (default is 2). For example, set to 3 for a 3d tSNE": "生成的tSNE嵌入的维数空间（默认为2）。例如，设置为3以进行3D tSNE。",
    "An integer above 0.": "一个大于0的整数。",
    "10X MTX or tabular": "10X MTX 或表格",
    "Tab-delimted expression table": "制表符分隔的表达量表",
    "Raw expression quantification as a sparse matrix in Matrix Market format, where each column is a gene and each row is a barcode/cell.": "原始表达量量化为矩阵市场格式的稀疏矩阵，其中每一列是一个基因，每一行是一个条形码/细胞。",
    "Matrix market column file for genes": "基因的矩阵市场列文件",
    "Matrix market row file for genes": "基因矩阵市场行文件",
    "File with metdata for the cells, first column should be cell identifiers as used in the cells 10x file.": "包含细胞元数据的文件，第一列应该是细胞标识符，如在细胞10x文件中使用的那样。",
    "Include features detected in at least this many cells. Will subset the counts matrix as well. To reintroduce excluded features, create a new object with a lower cutoff.": "包括至少在这么多细胞中检测到的特征。也会对计数矩阵进行子集化处理。要重新引入被排除的特征，可以创建一个具有较低截止值的新对象。",
    "Include cells where at least this many features are detected.": "包含至少检测到这么多特征的细胞。",
    "File with gene names to scale/center. Default is all genes in object@data.": "用于缩放/居中的基因名称文件。默认情况下是对象@data中的所有基因。",
    "File with cell names to scale/center. Default is all cells in object@data.": "用于缩放/居中细胞名称的文件。默认情况下是对象@data中的所有细胞。",
    "Total Number of PCs to compute and store (20 by default). Less PCs might be faster, but will explain less variance.": "要计算和存储的主成分总数（默认为20）。较少的主成分可能会更快，但解释的方差会较少。",
    "Run PCA on reverse matrix (gene x cell; FALSE by default means cell x gene).": "在基因 x 细胞的转置矩阵上运行PCA（默认为FALSE，表示细胞 x 基因）。",
    "Limit testing to genes which show, on average, at least X-fold difference (log-scale) between the two groups of cells. Default is 0.25 Increasing logfc.threshold speeds up the function, but can miss weaker signals.": "将测试限制在平均至少显示两组细胞之间X倍差异（对数尺度）的基因上。默认值为0.25。增加logfc.threshold可以加快函数速度，但可能会错过较弱的信号。",
    "Down sample each identity class to a max number. Default is no downsampling. Not activated by default (set to Inf).": "将每个同一性类别的样本数量减少到最大数量。默认情况下不进行减少采样。默认值是无穷大（不激活）。",
    "Minimum number of cells expressing the gene in at least one of the two groups, currently only used for poisson and negative binomial tests.": "在两个组中至少有一个组中表达该基因的细胞的最小数量，目前仅用于泊松和负二项检验。",
    "Only test genes that are detected in a minimum fraction of min.pct cells in either of the two populations. Meant to speed up the function by not testing genes that are very infrequently expressed. Default is 0.1.": "仅测试在两个群体中的任一个中被检测到的最小比例为min.pct的细胞的基因。这意味着通过不测试非常不频繁表达的基因来加快函数的速度。默认值为0.1。",
    "Only test genes that show a minimum difference in the fraction of detection between the two groups. Set to -Inf by default.": "只测试在两组之间检测分数的最小差异中显示出来的基因。默认设置为负无穷大。",
    "Only return positive markers (FALSE by default).": "仅返回正向标记（默认为FALSE）。",
    "for example nCount_RNA, nFeature_RNA, the name of a gene, PC1, a column name in object@meta.data, etc. Any argument that can be retreived using FetchData.": "例如 nCount_RNA、nFeature_RNA、一个基因的名称、PC1、object@meta.data 中的一列名称等。任何可以使用 FetchData 检索的参数。",
    "Comma-separated list of cell names to use as a subset.": "以逗号分隔的单元格名称列表，用作子集。",
    "Which dimensionality reduction to use. Default is \"pca\", can also be \"tsne\", or \"ica\", assuming these are precomputed": "要使用哪种降维方法。默认是 \"pca\"，也可以是 \"tsne\" 或 \"ica\"，假设这些是预先计算好的",
    "Dimension for x-axis": "x轴的维度",
    "Dimension for y-axis": "y轴的维度",
    "File to be used to derive a vector of cells to plot (default is all cells)": "用于派生要绘制的细胞向量的文件（默认为所有细胞）",
    "Adjust point size for plotting": "调整绘图的点大小",
    "Sets size of labels": "设置标签的大小",
    "Group (color) cells in different ways": "以不同方式（颜色）对细胞进行分组",
    "Comma-separated list of colors, each color corresponds to an identity class. By default, ggplot assigns colors.": "逗号分隔的颜色列表，每种颜色对应一个身份类别。默认情况下，ggplot分配颜色。",
    "If NULL, all points are circles (default). You can specify any cell attribute (that can be pulled with FetchData) allowing for both different colors and different shapes on cells.": "如果为NULL，默认所有点都是圆形。您可以指定任何单元格属性（可以使用FetchData提取的属性），从而允许单元格上有不同的颜色和不同的形状。",
    "Comma-separated string specifying the order of plotting for the idents (clusters). This can be useful for crowded plots if points of interest are being buried. Provide either a full list of valid idents or a subset to be plotted last (on top).": "逗号分隔的字符串，用于指定idents（簇）的绘图顺序。如果绘图拥挤，感兴趣的点被遮盖，这可能会很有用。提供有效idents的完整列表，或要绘制在最后（顶部）的子集。",
    "Width of png in pixels.": "PNG图片宽度（以像素为单位）。",
    "Height of png in pixels.": "PNG图片的像素高度。",
    "RDS file from Seurat read10x module.": "Seurat 软件包的 read10x 模块读取的 RDS 文件。",
    "Value of the resolution parameter, use a value above (below) 1.0 if you want to obtain a larger (smaller) number of communities. Defaults to 0.8.": "分辨率参数的值，如果您想要获得更多（更少）的社区，则使用大于（小于）1.0的值。默认值为0.8。",
    "Method for leiden  (defaults to matrix which is fast for small datasets). Select iGraph to avoid casting large data to a dense matrix.": "用于leiden的方法（默认为矩阵，这对小型数据集很快）。选择iGraph以避免将大型数据转换为密集矩阵。",
    "Name of graph to use for the clustering algorith.": "用于聚类算法的图名称。",
    "Number of random starts, 10 by default.": "随机启动次数，默认为10。",
    "Group singletons into nearest cluster. If FALSE, assign all singletons to a 'singleton' group.": "将单个元素分组到最近的簇中。如果为FALSE，则将所有单个元素分配到一个名为'singleton'的组中。",
    "This is a value that will be prefixed to output file names": "这是一个值，它将被添加到输出文件名的前面",
    "step increase values or list of all the parameter values": "步骤增加值或所有参数值的列表",
    "List of all comma separated values for the parameter": "参数的所有逗号分隔值列表",
    "Initial value for the parameter": "参数的初始值",
    "Increment to use from first to last value": "从第一个值到最后一个值使用的增量",
    "Final value for the parameter. The final value is not included if increments of step don't match the last value.": "参数的最终值。如果步长的增量与最后一个值不匹配，则不包括最终值。",
    "By default a new layer will be created called 'mnn', 'mnn_{layer}' or 'mnn_{layer}_{key_added}' where those parameters were specified. A value of 'X' causes batch-corrected values to overwrite the original content of .X.": "默认情况下，将创建一个名为'mnn'、'mnn_{layer}'或'mnn_{layer}_{key_added}'的新层，其中指定了这些参数。值为'X'会导致批量校正的值覆盖.X原始内容。",
    "e.g. 'highly_variable'": "例如 '高度可变'",
    "e.g. 'True'": "例如 'True'",
    "The bandwidth of the Gaussian smoothing kernel used to compute the correction vectors.": "用于计算校正向量的高斯平滑核的带宽。",
    "Note this step takes most computing time.": "注意这一步占据了大部分的计算时间。",
    "'svd' computes SVD using a non-randomized SVD-via-ID algorithm, while 'rsvd' uses a randomized version. 'irlb' performs truncated SVD by implicitly restarted Lanczos bidiagonalization (forked from https://github.com/airysen/irlbpy).": "'svd' 使用非随机化的 SVD-via-ID 算法计算奇异值分解（SVD），而 'rsvd' 使用随机化版本。'irlb' 通过隐式重启的 Lanczos 双向量分解（forked from https://github.com/airysen/irlbpy）执行截断奇异值分解（truncated SVD）。",
    " Requires that adata.uns contains a directed single-cell graph with key velocity_graph. This feature might be subject to change in the future.": "要求adata.uns包含一个具有关键velocity_graph的有向单细胞图。这个特性在未来可能会有所改变。",
    "Threshold for calling a transcriptome a doublet. If not set, this is set automatically by looking for the minimum between the two modes of the doublet_scores_sim_ histogram. It is best  practice to check the threshold visually using the doublet_scores_sim_ histogram and/or based on co-localization of predicted doublets in a 2-D embedding.": "调用转录组为双体的阈值。如果未设置，则通过查找doublet_scores_sim_直方图的两个模式之间的最小值来自动设置。最好的做法是使用doublet_scores_sim_直方图和/或基于预测双体在2-D嵌入中的共定位来直观检查阈值。",
    "By default, the output object is annotated but not filtered according to the scrublet status.": "默认情况下，输出对象会被注释，但不会根据scrublet状态进行过滤。",
    "If 1.0, each doublet is created by simply adding the UMI counts from two randomly sampled observed transcriptomes. For values less than 1, the UMI counts are added and then randomly sampled at the specified rate.": "如果为1.0，则每个双体是通过简单地将两个随机抽样的观测转录组的UMI计数相加而创建的。对于小于1的值，UMI计数相加后再按照指定的比率随机抽样。",
    "klearn.decomposition.TruncatedSVD will be used for dimensionality reduction, if --no-mean-center is set. Use this flag to disable that behaviour.": "klearn.decomposition.TruncatedSVD 将用于降维，如果设置了 --no-mean-center。使用此标志以禁用该行为。",
    "Whether to use :func:~scanpy.pp.log1p to log-transform the data prior to PCA.": "是否在进行PCA之前使用：func：~scanpy.pp.log1p对数据进行对数转换。",
    "sklearn.decomposition.PCA will be used for dimensionality reduction.": "将使用sklearn.decomposition.PCA进行降维。",
    "Used to embed the transcriptomes prior to k-nearest-neighbor graph construction.": "用于在构建k-最近邻图之前嵌入转录组。",
    "Return (in .uns) the parent transcriptomes that generated the doublet neighbors of each observed transcriptome. This information can be used to infer the cell states that generated a given doublet state.": "返回（在.uns文件中）生成每个观察到的转录组的双重邻近区域的父转录组。这些信息可以用来推断生成给定双重状态的细胞状态。",
    "Used to construct the KNN graph of observed transcriptomes and simulated doublets. If not set, this is automatically set to np.round(0.5 * np.sqrt(n_obs)).": "用于构建观察到的转录组和模拟双体的KNN图。如果未设置，则自动设置为np.round(0.5 * np.sqrt(n_obs))。",
    "This results in a quicker run time for large datasets while also potentially increasing the degree of batch correction.": "这可以在大型数据集上实现更快的运行时间，同时还可能增加批次校正的程度。",
    "Total number of neighbours will be this number times the number of batches.": "邻居的总数将是这个数字乘以批次的数量。",
    "May help with population independence and improve the tidiness of clustering. The lower the value the more independent the individual populations, at the cost of more conserved batch effect. If not set, sets the parameter value automatically to 10 times the total number of neighbours for each cell. Set to 0 to skip.": "可能有助于提高种群独立性并改善聚类整洁度。该值越低，各个种群越独立，但代价是批次效应更加保守。如果未设置，则自动将参数值设置为每个细胞的邻居总数的10倍。设置为0以跳过。",
    "More trees give higher precision when querying, at the cost of increased run time and resource intensity.": "更多的树在查询时可以提供更高的精确度，但代价是运行时间和资源消耗的增加。",
    "If approx=False and the metric is 'euclidean' use the faiss package to compute nearest neighbours if installed. This improves performance at a minor cost to numerical precision as faiss operates on float32.": "如果 approx=False 且度量为 'euclidean'，则在安装 faiss 包的情况下使用 faiss 包来计算最近邻。这在数值精度略有损失的情况下提高了性能，因为 faiss 操作使用 float32。",
    "Float between 0 and 1, controlling the blend between a connectivity matrix formed exclusively from mutual nearest neighbour pairs (0) and a union of all observed neighbour relationships with the mutual pairs emphasised (1).": "介于0和1之间的浮点数，用于控制从仅由相互最近邻对形成的连通性矩阵（0）到强调相互对的所观察到的所有邻居关系的联合（1）之间的混合。",
    "This parameter refers to the design matrix X in Equation 2.1 in [Johnson07] and to the mod argument in the original combat function in the sva R package. Note that not including covariates may introduce bias or lead to the removal of biological signal in unbalanced designs.": "这个参数指的是[Johnson07]中方程2.1的设计矩阵X，以及sva R包中原始combat函数的mod参数。请注意，不包括协变量可能会引入偏差或导致在不平衡设计中去除生物信号。",
    "By default a new layer will be created called 'combat', 'combat_{layer}' or 'combat_{layer}_{key_added}' where those parameters were specified. A value of 'X' causes batch-corrected values to overwrite the original content of .X.": "默认情况下，将创建一个名为'combat'、'combat_{layer}'或'combat_{layer}_{key_added}'的新层，其中指定了这些参数。值'X'会导致批量校正的值覆盖.X原始内容。",
    "Must be a key of `.obsm` without the prefix 'X_', e.g. 'umap'.": "必须是 `.obsm` 的键，且没有前缀 'X_'，例如 'umap'。",
    "Set to 0 to include all edges.": "设置为0以包含所有边。",
    "You can’t specify both obsp and neighbors-key at the same time.": "您不能同时指定obsp和neighbors-key。",
    "Ignored when `Number of branchings to detect` is 0": "当“检测到的分支数”为0时，将忽略此选项",
    "If a very small branch is detected upon splitting, shift away from maximum correlation in Kendall tau criterion of [Haghverdi16] to stabilize the splitting": "如果在分裂时检测到一个非常小的分支，根据[Haghverdi16]中的Kendall tau准则，从最大相关性转变为稳定分裂",
    "For adding more than one gene, write them as a comma separated list, with no spaces in the middle, like `crp1,crp2,cycX`.": "为了添加多个基因，将它们以逗号分隔的列表形式书写，中间不要留空格，例如 `crp1,crp2,cycX`。",
    "Input file with monocle3 object cds3 in RDS": "输入包含 monocle3 对象 cds3 的 RDS 文件",
    "(Optional) Set of cells to be considered as reference for marker significance test. Accelerates the marker significance test at some cost in sensitivity.": "（可选）将被考虑作为标记显著性测试的参考细胞集。这可以在敏感性上有所牺牲，以加速标记显著性测试。",
    "File with cells to be used as reference cells.": "用作参考细胞的细胞文件。",
    "Number of cells to be used as reference cells.": "作为参考细胞使用要使用的细胞数量。",
    "Filters the markers test result by this fraction of expression": "根据表达量的这个比例过滤标记测试结果",
    "The number of top marker genes to report in plots and in top markers list": "在图表和标记基因列表中报告的顶级标记基因的数量",
    "how many genes of the top ranked specific genes by Jenson-Shannon, to do the more expensive regression test on": "在詹森-香农（Jenson-Shannon）排名中，有多少个特定基因需要进行更昂贵的回归测试？",
    "Cell groups, choose from 'cluster', 'partition', or any categorical variable in `colData(cds)`": "细胞组，从 'cluster'、'partition' 或 `colData(cds)` 中的任何分类变量中选择",
    "whether to assess the discriminative power of each marker through logistic regression. Can be slow, consider disabling to speed up top_markers().": "是否通过逻辑回归评估每个标记的区分能力。可能会很慢，考虑禁用以加快top_markers()的速度。",
    "whether to generate the full markers list as well.": "是否同时生成完整的标记列表。",
    "whether to generate the top markers plot.": "是否生成顶级标记图。",
    "A boolean variable which defines whether to perform gene filtering before SC3 clustering.": "一个布尔变量，用于定义在SC3聚类之前是否执行基因过滤。",
    "An integer value. Genes with percent of dropouts smaller than this value are filtered out before clustering.": "一个整数值。在聚类之前，会筛选掉掉包百分比小于该值的基因。",
    "An integer value. Genes with percent of dropouts larger than this value are filtered out before clustering.": "一个整数值。在聚类之前，将掉包率大于该值的基因过滤掉。",
    "SC3 is a stochastic method, so setting the rand_seed to a fixed values can be used for reproducibility purposes.": "SC3 是一种随机方法，因此将 rand_seed 设置为固定值可以用于可重复性目的。",
    "When unspecified, default to 1000 for up to 2000 cells and 50 for more than 2000 cells.": "如果未指定，默认情况下，对于不超过2000个细胞，使用1000；对于超过2000个细胞，使用50。",
    "When unspecified, default to 1e+9.": "当未指定时，默认为1e+9。",
    "File name in which a serialized R SingleCellExperiment object where object matrix found": "包含序列化 R SingleCellExperiment 对象的文件名，其中找到了对象矩阵",
    "singleCellExperiment object containing expression values and experimental information. Must have been appropriately prepared.": "单细胞实验对象包含表达值和实验信息。必须已经适当准备。",
    "character, indicating slot of the assays of the object that should be used to define expression. Valid options are counts as default. tpm, fpkm and logcounts, or anything else in the object added manually by the user.": "字符，表示对象测定的槽位，用于定义表达。有效选项默认为计数。TPM、FPKM 和 logcounts，或用户手动添加到对象中的任何其他内容。",
    "file (one cell per line) to be used to derive a vector of cell (sample) names used to identify cell controls (for example, blank wells or bulk controls).": "文件（每行一个单元格）用于派生用于识别细胞控制（例如，空白孔或批量控制）的细胞（样本）名称向量。",
    "Each element is treated as a number of top genes to compute the percentage of library size occupied by the most highly expressed genes in each cell.": "每个元素被视为计算每个细胞中最高度表达基因所占库大小百分比的顶级基因数量。",
    "A numeric scalar to be passed to 'nexprs', specifying the lower detection limit for expression.": "一个要传递给'nexprs'的数值标量，用于指定表达式的下限检测限。",
    "A logical scalar indicating whether existing spike-in sets in 'object' should be automatically added to 'feature_controls'": "一个逻辑标量，指示是否应自动将“对象”中现有的spike-in集添加到“feature_controls”中",
    "A serialized SingleCellExperiment RDS object with pre-calculated QC metrics": "一个序列化的SingleCellExperiment RDS对象，其中包含预先计算的QC（质量控制）指标。",
    "character scalar, name of a numeric QC metric available in the input SingleCellExperiment object.": "字符标量，输入的SingleCellExperiment对象中可用的数值质量控制(QC)指标的名称。",
    "character scalar, choice indicate whether outliers should be looked for at both tails (default: both) or only at the lower end (lower) or the higher end (higher).": "字符标量，选择指示是否应在两个尾部查找异常值（默认值：两个尾部）或仅在较低端（lower）或较高端（higher）。",
    "logical, should the values of the metric be transformed to the log10 scale before computing median-absolute-deviation for outlier detection": "逻辑，是否应在计算用于异常值检测的中位绝对偏差之前将度量的值转换为对数10尺度",
    "scalar, number of median-absolute-deviations away from median required for a value to be called an outlier.": "标量，一个值要被判定为异常值，需要与中位数偏离的中位数绝对偏差的数量。",
    "numeric scalar indicating the minimum difference from the median to consider as an outlier. The outlier threshold is defined from the larger of nmads MADs and min.diff, to avoid calling many outliers when the MAD is very small. If NA, it is ignored.": "数值标量，表示与中位数的最小差异，以将其视为异常值。异常值阈值定义为 nmads 个 MADs 和 min.diff 中较大的一个，以避免在 MAD 非常小的时候调用许多异常值。如果为 NA，则忽略。",
    "Comma separated keys for regressing out; they need to exist in the observations part of the AnnData object.": "用逗号分隔需要回归掉的键；它们需要存在于AnnData对象的观测部分中。",
    "A collection of results of SCCAF Assesment for the same or different rounds of the Run SCCAF optimisation process.": "一组相同或不同轮次的运行SCCAF优化过程的SCCAF评估结果。",
    "Round files collection from Run SCCAF.": "从运行SCCAF中收集圆文件。",
    "either a round file or an explicit name of a round.": "要么是一个圆形文件，要么是一个明确的圆形名称。",
    "Write the slot of the AnnData object that you want assessed.": "写下你想要评估的AnnData对象的槽位。",
    "For use with sccaf_distribute_assesment": "用于与 sccaf_distribute_assesment 一起使用",
    "The number of times that the assesment is internally repeated to achieve a result.": "评估内部重复的次数，以获得结果。",
    "If the provided AnnData/Loom file does not include the clustering, or if you want to use an external clustering assigment.": "如果提供的AnnData/Loom文件不包括聚类，或者如果您想使用外部的聚类分配。",
    "If you are not using an external clustering, then you must specify the slot/index in the AnnData or Loom file where the clustering to be used for assessment (and potentially optimisation starting point) is saved.": "如果您没有使用外部聚类，则必须指定在AnnData或Loom文件中用于评估（以及可能的优化起点）的聚类存储的槽/索引。",
    "By default the tool only runs an assesment of the clustering quality. To further optimise the clustering, enable this option.": "默认情况下，该工具仅运行对聚类质量的评估。要进一步优化聚类，请启用此选项。",
    "This option uses the previously specified clustering (either external file provided or internal index/slot specified) to serve as starting point for the optimisation process.": "这个选项使用之前指定的聚类（无论是外部文件提供的还是内部索引/槽指定的）作为优化过程的起点。",
    "Perform an initial clustering and use this as an starting point.": "进行初步聚类，并将其作为起点。",
    "How should the optimisation algorithm be stopped.": "优化算法应该如何停止。",
    "The SCCAF optimisation process will converge once this accuracy is achieved.": "SCCAF优化过程将在达到此准确性后收敛。",
    "The slot inside the ann data object with the desired underclustering (low resolution) to be used as exit condition.": "在ann数据对象中，具有所需欠聚类（低分辨率）的槽位，将用作退出条件。",
    "If you are running the optimisation, you can choose skip the initial assessment to save time.": "如果您正在运行优化，您可以选择跳过初始评估以节省时间。",
    "Will split rounds in files so that multiple assess processes can be distributed down the line": "将分轮次的文件分割，以便多评估进程可以沿流水线分布",
    "By default is 3, higher numbers mean more stable results (number of rounds) but also longer execution times.": "默认值为3，数值越高意味着结果更稳定（轮数），但同时执行时间也会更长。",
    "SCE object to be pre-processed": "待预处理的SCE对象",
    "File with serialized SingleCellExperiment object as produced by 'scmap select features'": "由 'scmap select features' 生成的包含序列化 SingleCellExperiment 对象的文件",
    "Number of chunks into which the expr matrix is split": "表达矩阵被分割成的块数",
    "Number of clusters per group for k-means clustering": "每组的聚类数目，用于k均值聚类",
    "ID of the training dataset": "训练数据集的ID",
    "Should the expression data be removed after classifier is trained?": "在分类器训练完成后，是否应该删除表达数据？",
    "scmap-cell contains k-means step which makes it stochastic, i.e. running it multiple times will provide slightly different results. A fixed random seed ensures reproducibility.": "scmap-cell包含了k-means步骤，这使得它具有随机性，即多次运行会得到略有不同的结果。一个固定的随机种子可以确保可重复性。",
    "Column name in the 'colData' slot of the SingleCellExperiment object containing the cell classification information.": "在SingleCellExperiment对象的'colData'插槽中包含细胞分类信息的列名。",
    "File with serialized SingleCellExperiment object as produced by 'scmap select features' and 'scmap index clusters', onto which another dataset will be projected.": "由 'scmap select features' 和 'scmap index clusters' 生成的序列化 SingleCellExperiment 对象的文件，另一个数据集将被投影到该文件上。",
    "File with serialized SingleCellExperiment object to project onto the index'": "包含序列化SingleCellExperiment对象的文件，用于投影到索引上。",
    "A positive integer specifying the number of nearest neighbours to find": "一个正整数，指定要查找的最近邻的数量",
    "If cell cluster annotation is available for the reference datasets, in addition to finding top 10 nearest neighbours scmap-cell also allows to annotate cells of the projection dataset using labels of the reference.": "如果参考数据集有细胞群注释，除了找到scmap-cell的前10个最近邻之外，scmap-cell还允许使用参考数据集的标签来注释投影数据集的细胞。",
    "Threshold on similarity (or probability for SVM and RF).": "相似性阈值（对于SVM和RF是概率）。",
    "File with serialized SingleCellExperiment object": "包含序列化SingleCellExperiment对象的文件",
    "Number of features to be selected": "要选取的特征数量",
    "Path to the predictions file in text format": "预测文件的文本格式路径",
    "Boolean indicating whether similarity scores should be included in the final output": "布尔值，指示是否应在最终输出中包含相似性得分",
    "Scmap index object in rdata format": "Scmap 索引对象以 rdata 格式",
    "What tool was used - scmap-cell or scmap-cluster?": "使用了哪个工具 - scmap-cell 还是 scmap-cluster？",
    "Name of column that contains distances between clusters/cells": "包含簇/细胞之间距离的列名称",
    "File with serialized SingleCellExperiment object that contains expression matrix": "包含表达矩阵的序列化SingleCellExperiment对象的文件",
    "Slot containing normalised counts in input SCE object": "包含输入SCE对象中标准化计数的槽",
    "Input Seurat object in .rds format": "输入Seurat对象，格式为.rds",
    "Name of the metadata column that contains training labels": "包含训练标签的元数据列的名称",
    "Method for mulpiple testing correction": "多重检验校正方法",
    "Significance threshold for principal components explaining class identity": "解释类别身份的主成分的显著性阈值",
    "Name of reduction in Seurat objet to be used to determine the feature space. Default: \"pca\"": "在Seurat对象中用于确定特征空间的降维方法名称。默认值：\"pca\"",
    "Path to the classifier object used to obtain predictions": "用于获得预测的分类器对象的路径",
    "Boolean: should the prediction scores be included? default: FALSE": "布尔值：是否包含预测分数？默认值：FALSE",
    "Path to the input prediction matrix in .rds format": "输入预测矩阵的路径，在.rds格式中",
    "Should the predicted expression data be normalised? Default: False": "预测的表达数据是否需要标准化？默认值：False",
    "What normalisation method should be applied to predicted data? Default: LogNormalize": "预测数据应该应用哪种标准化方法？默认：LogNormalize",
    "What scale factor should be applied? Note: for CPM normalisation, 1e6 is selected by default": "应该应用什么比例因子？注意：对于CPM标准化，默认选择1e6",
    "Classification threshold value": "分类阈值",
    "Maximum number of rounds to run Harmony. One round of Harmony involves one clustering and one correction step": "Harmony运行的最大轮数。一轮Harmony包括一次聚类和一次校正步骤",
    "Recompute alignment? Useful if scPredict() has already been run. Default: True": "重新计算对齐？如果已经运行了scPredict()，这将非常有用。默认值：True",
    "Scale new dataset based on means and stdevs from reference dataset before alignment. Default: True": "在对齐之前，根据参考数据集的均值和标准差对新数据集进行缩放。默认值：True",
    "Input SCE object in .rds format": "输入RDS格式的SCE对象",
    "Model type used for training. Must be one of the models supported by Caret package. Default: svmRadial": "用于训练的模型类型。必须是 Caret 包支持的模型之一。默认值：svmRadial",
    "Resampling method used for model fit evaluation": "用于模型拟合评估的重采样方法",
    "Number of resampling iterations. Default: 5": "重采样迭代次数。默认值：5",
    "An integer denoting the amount of granularity in the tuning parameter grid": "一个整数，表示调优参数网格中的粒度大小",
    "Performance metric to be used to select best model": "用于选择最佳模型的性能指标",
    "A string vector that defines a pre-processing of the predictor data. Enter values as comma-separated string. Current possibilities are 'BoxCox', 'YeoJohnson', 'expoTrans', 'center', 'scale', 'range', 'knnImpute', 'bagImpute', 'medianImpute', 'pca', 'ica' and 'spatialSign'. The default is 'center' and 'scale'.": "一个字符串向量，用于定义预测器数据的预处理。输入值时以逗号分隔字符串。当前可能性包括'BoxCox'、'YeoJohnson'、'expoTrans'、'center'、'scale'、'range'、'knnImpute'、'bagImpute'、'medianImpute'、'pca'、'ica'和'spatialSign'。默认值为'center'和'scale'。",
    "If TRUE, training data is returned within scPred object. Default: False": "如果为真，训练数据将包含在scPred对象中。默认值：假",
    "Specifies the set of hold-out predictions for each resample that should be returned. Values can be either 'all', 'final' or 'none'.": "指定应返回每个重采样的保留预测集。值可以是'all'、'final'或'none'。",
    "Cell types to reclassify using a different model": "使用不同模型重新分类的细胞类型",
    "What proportion of the data set (0 to 1) should be used for training": "数据集（0到1）中用于训练的比例应该是多少",
    "Training matrix obtained from train/test split step": "从训练/测试分割步骤中获得的训练矩阵",
    "Should log-transform be performed on training data?": "应该对训练数据进行对数转换吗？",
    "Path to training labels in text format": "训练标签文本格式的路径",
    "CDS object with expression data for training": "CDS对象带有用于训练的表达数据",
    "File with marker genes specifying cell types. See https://cole-trapnell-lab.github.io/garnett/docs/#constructing-a-marker-file for specification of the file format": "包含标记基因以指定细胞类型的文件。有关文件格式的详细信息，请参见 https://cole-trapnell-lab.github.io/garnett/docs/#constructing-a-marker-file",
    "argument for Bioconductor AnnotationDb-class package used for converting gene IDs": "用于将基因ID转换的Bioconductor AnnotationDb类软件包的参数",
    "Format of the gene IDs in your CDS object": "您CDS对象中基因ID的格式",
    "Format of the gene IDs in your marker file": "标记文件中基因ID的格式",
    "Should markers from child nodes of a cell type be used in finding representatives of the parent type?": "在寻找父类型代表时，是否应该使用来自细胞类型子节点的标记？",
    "The type of gene ID that will be used in the classifier. If possible for your organism, this should be 'ENSEMBL', which is the default.": "在分类器中将使用的基因ID类型。如果可能的话，对于您的生物体，这应该是'ENSEMBL'，这是默认值。",
    "Serialised object containing markers list produced by garnett_transform_markers": "由garnett_transform_markers生成的包含标记列表的序列化对象",
    "Path to the file with marker gene assessment done by garnett": "由Garnett完成的标记基因评估文件的路径",
    "Marker gene assessment column name": "标记基因评估列名",
    "Gene id column name in marker assessment file": "标记评估文件中基因ID列的名称",
    "Path to the SCXA-style marker gene file in .txt format": "SCXA风格的标记基因文件路径，在.txt格式",
    "Column name of marker p-values": "标记物 P 值的列名",
    "Cut-off p-value for marker genes": "标记基因的截止P值",
    "Column name of cell groups (i.e. cluster IDs or cell types) in marker file": "标记文件中单元格群组（即聚类ID或细胞类型）的列名",
    "Column containing gene names in marker file": "标记文件中包含基因名称的列",
    "Number of outgroups to compare against": "与外群比较的数量",
    "The minimum number of representative cells per cell type required to include the cell type in the predictive model": "每个细胞类型在预测模型中包含所需的最小代表性细胞数量",
    "The maximum number of representative cells per cell type to be included in the model training. Decreasing this number increases speed, but may hurt performance of the model.": "模型训练中每种细胞类型包含的代表性细胞的最大数量。降低此数量可以提高速度，但可能会影响模型的性能。",
    "CDS object with predicted labels": "带预测标签的CDS对象",
    "Classifier object in rdata format": "R数据格式中的分类器对象",
    "Path to the object of class garnett_classifier, which is either trained via garnett_train_classifier.R or obtained elsewhere": "类garnett_classifier的对象的路径，该对象是通过garnett_train_classifier.R训练的，或者从其他地方获得的",
    "In case a hierarchical marker tree was used to train the classifier, specify which node features should be shown. Default is 'root'. For other nodes, use the corresponding parent cell type name": "如果使用层次化标记树来训练分类器，请指定应显示哪些节点特征。默认为'root'。对于其他节点，请使用相应的父细胞类型名称。",
    "Boolean that indicates whether the gene IDs should be converted into SYMBOL notation": "一个布尔值，指示是否将基因ID转换为SYMBOL表示法",
    "Boolean, tells Garnett whether to create a second set of assignments that expands classifications to cells in the same cluster": "布尔值，告诉Garnett是否创建第二组分配，将分类扩展到同一簇中的单元",
    "Numeric value greater than 1. This is the minimum odds ratio between the probability of the most likely cell type to the second most likely cell type to allow assignment. Default is 1.5. Higher values are more conservative.": "数值大于1。这是最可能的细胞类型与第二可能的细胞类型之间允许分配的概率的最小比值。默认值为1.5。更高的值更保守。",
    "Aggregated prediction tables from multiple tools": "聚合多个工具的预测表格",
    "Table with tools performance metrics": "工具性能指标表",
    "Should semantic similarity be included in calculation of combined score?": "是否应将语义相似性包含在综合评分的计算中？",
    "Should the cells be ordered by aggregated score?": "细胞是否应该根据聚合分数排序？",
    "Yaml file with trivial terms and unlabelled cells": "包含简单术语和未标记单元格的YAML文件",
    "Ontology graph to run similairty calculations": "本体图用于运行相似性计算",
    "Path to the true labels tsv file in case tool performance is evaluated. Expected columns: cell_id, true_label, ontology_term": "如果评估工具性能，则为真实标签tsv文件的路径。预期列：cell_id, true_label, ontology_term",
    "Fields in adata.obs to merge, comma separated (optional). They will be available as field1_field2_field3 in the AnnData Obs dataframe. You can have multiple groups to merge, separated by colon (:).": "在adata.obs中要合并的字段，用逗号分隔（可选）。它们将作为field1_field2_field3在AnnData Obs数据框中可用。您可以有多组要合并的，用冒号（:）分隔。",
    "The column in adata.obs that defines the groups. Merged columns in the above field are available here.": "在adata.obs中定义组的列。上述字段中合并的列在这里可用。",
    "The column in adata.obs that defines the samples. Merged columns in the above field are available here.": "在adata.obs中定义样本的列。上述字段中合并的列在这里可用。",
    "The name of the layer of the AnnData object to use. It needs to be present in the AnnData object.": "要使用的AnnData对象层的名称。它需要在AnnData对象中存在。",
    "Determines how counts are aggregated across cells with the specificied groups: sum, mean or median.": "确定如何将具有特定组的细胞中的计数进行聚合：求和、平均值或中位数。",
    "Fields in adata.obs to use as factors, comma separated (optional). For EdgeR make sure that the first field is the main contrast field desired and the rest of the fields are the covariates desired. Decoupler produces two fields in the intermediate AnnData (which can be added here if desired for covariates): psbulk_n_cells and psbulk_counts.": "在adata.obs中用作因子的字段，用逗号分隔（可选）。对于EdgeR，请确保第一个字段是所需的主要对比字段，其余字段是所需的协变量字段。Decoupler在中间AnnData中产生两个字段（如果需要用作协变量，可以在这里添加）：psbulk_n_cells和psbulk_counts。",
    "X and Y sizes in points separated by a space": "X 和 Y 大小以点为单位，用空格分隔",
    "Classification tables obtained from multiple classisifiers": "从多个分类器获得的分类表",
    "Metadata file with reference labels": "带有参考标签的元数据文件",
    "Config file with exclusions": "配置文件，包含排除项",
    "Ontology graph to run semantis similarity computations": "本体图用于运行语义相似性计算",
    "Mapping between cell labels and ontology terms": "细胞标签与本体术语之间的映射",
    "Name of barcode field in reference metadata file": "参考元数据文件中条形码字段的名称",
    "Name of barcode field in predictions file": "预测文件中条形码字段的名称",
    "Label column in reference file": "参考文件中的标签列",
    "Name of label field in predictions file": "预测文件中标签字段的名称",
    "Path to the table of tool statistics from get_tool_performance_table.R": "工具性能统计表的路径，来自 get_tool_performance_table.R",
    "Path to the list of empirical distributions in .rds format": "经验分布列表的路径，格式为.rds",
    "Text file with reference cell labels": "参考细胞标签的文本文件",
    "YML file with excluded/unlabelled terms": "YML 文件，包含排除/未标记的术语",
    "Number of simulations to generate empirical CDF": "生成经验累积分布函数（CDF）的模拟次数",
    "Number of labels to sample for each permutation": "每个排列要采样的标签数量",
    "Standardised tables obtained from multiple classifiers": "从多个分类器获得的标准表格",
    "Number of top labels to select": "选择顶级标签的数量",
    "Are scores included into the tool outpus?": "分数是否包含在工具输出中？",
    "Are the supplied SDRFs of condensed format?": "提供的SDRFs是简化格式的吗？",
    "Name of the barcode field in provided SDRF files": "在提供的SDRF文件中条形码字段的名称",
    "Name of the cell label field name in provided SDRF files": "提供的SDRF文件中细胞标签字段名称",
    "Name of the cell ontology terms column in SDRF files (must be identical across all files)": "SDRF文件中细胞本体术语列的名称（必须在所有文件中保持一致）",
    "Was the Wig/bedgraph input constructed using a genome from your history, or a built-in genome?": "Wig/bedgraph 输入是使用您历史记录中的基因组构建的，还是使用内置的基因组？",
    "A file of sequence lengths for the history reference is required to make a bigwig. Compute sequence length tool can make these from fasta files": "要制作bigwig文件，需要一个历史参考的序列长度文件。计算序列长度的工具可以从fasta文件生成这些文件。",
    "Default settings should usually be used.": "默认设置通常应该被使用。",
    "Issue warning messages rather than dying if wig file contains items off end of chromosome.": "如果wig文件包含染色体末端之外的项目，则发出警告消息，而不是停止运行。",
    "Add your .gbk files here. Do not wonder, in the filename 'region. will be added to ensure that every file will be included!": "将您的.gbk文件放在这里。不要担心，在文件名中'region.'将被添加以确保每个文件都被包含在内！",
    "Select yes and select which version of the database you want to use": "选择“是”，并选择您想要使用的数据库版本",
    "Select which version of the MIBiG databse you want to use.": "选择您想要使用的MIBiG数据库的版本。",
    "Select yes if you want to add an extra string to the outputs": "如果您想在输出中添加额外的字符串，请选择“是”",
    "By default the BiG-SCAPE runs are named as (YYYY-MM-DD_HH-MM-SS_[extra]) where extra means the mode and if activated (hybrids).": "默认情况下，BiG-SCAPE的运行以(YYYY-MM-DD_HH-MM-SS_[额外])命名，其中额外表示模式，如果激活了(杂交)。",
    "Add Pfam-A.hmm.gz file here, please. Look at the help section where you can download it!": "请在这里添加Pfam-A.hmm.gz文件。请查看帮助部分，您可以在那里下载它！",
    "Getting more information about the analysis if toggled is true.": "如果设置为真，则获取有关分析的更多信息。",
    "When using you will receive a log file for the printed output of the toll. Can be useful when using the option above!": "当使用你将收到一个日志文件，用于打印工具的输出。在使用上面的选项时可能会很有用！",
    "With this option you can include BGCs who don't have a distance lower than the cutoff distance specified": "使用此选项，您可以包含那些距离不小于指定截止距离的BGC（生物合成基因簇）。",
    "When using this option you can specify at which percentage domains are considered to overlap. The domain with the best score is kept. The default value is 0.1": "使用此选项时，您可以指定在哪个百分比下认为域重叠。保留得分最高的域。默认值为0.1",
    "Minimum size of a BGC to be included in the analysis. The Default value is 0 base pairs. This also includes the sum of all loci in a multi-record GenBank file.": "在分析中包含的生物合成基因团簇（BGC）的最小大小。默认值是0个碱基对。这也包括了多记录GenBank文件中所有位点的总和。",
    "BiG-SCAPE separates the analysis according to the BGC product by default. If used BiG-SCAPE will mix all classes and analyse them.": "BiG-SCAPE 默认会根据生物合成基因簇（BGC）的产物将分析分开。如果使用 BiG-SCAPE，它将混合所有类别并进行分析。",
    "By default, BiG-SCAPE classifies the output based on the BGC product. If toggled it will deactivate it. Note: when (--mix) is not activated, BiG-SCAPE will not create any network file!": "默认情况下，BiG-SCAPE会根据生物合成基因簇（BGC）的产物对输出进行分类。如果切换了该选项，它将停用该功能。注意：当未激活（--mix）选项时，BiG-SCAPE将不会创建任何网络文件！",
    "You can exclude any of these classes to not be classified. Multiple banned classes are possible.": "您可以排除这些类别中的任何一个，以避免被分类。可以禁止多个类别。",
    "Generate networks using multiple raw distance cutoff values. The default value here is 0.3.": "使用多个原始距离截止值生成网络。这里的默认值是0.3。",
    "By default, BiG-SCAPE will perform a second layer of clustering to group GCFs into GCCs. Toggle to deactivate this.": "默认情况下，BiG-SCAPE会执行第二层聚类，将基因簇家族（GCFs）聚类成基因簇复合体（GCCs）。切换此选项可以停用这个功能。",
    "Select yes if you want to change cutoffs for cluster families into clans.": "如果要选择是否要更改聚类家族到氏族的截止值，请选择“是”。",
    "This value is for finding GCFs which will be used for clan calling. The default value is 0.3.": "这个值用于寻找用于族群命名的公共子序列（GCFs）。默认值是0.3。",
    "This value is for clustering families into clans. The default value is 0.7. Every pair of GCFs linked with a distance of this value or less will be taken into account!": "这个值用于将基因家族聚类成超家族。默认值是0.7。与这个值或更小距离连接的每对基因家族簇（GCFs）都将被考虑在内！",
    "By default, BGCs with hybrid predicted products from the PKS/NRPS Hybrids and Others classes will be included in each subclass. Since the same cluster can appear in different classes you can turn this off here.": "默认情况下，每个子类中将包含来自PKS/NRPS杂合体和其他类别的混合预测产物的生物合成基因簇（BGCs）。由于同一个簇可能出现在不同的类别中，您可以在这里关闭此功能。",
    "Here you can choose between 3 Alignment Mode which is used in comparing each pair of gene clusters. For more information look into the help section!": "在这里，你可以选择3种比对模式中的一种，用于比较每对基因簇。更多信息请查看帮助部分！",
    "Select yes if you want to use an Anchorfile. BiG-SCAPE has a default file which is always used and only these domains are counted in the result. Look in the help section to see what are the default domains!": "如果需要使用Anchor文件，请选择“是”。BiG-SCAPE有一个默认文件，该文件始终被使用，并且只有在结果中计算这些域。请查看帮助部分，了解默认域是什么！",
    "Use a custom Anchorfile (in .txt format) to give certain domains a special weight in the DSS index. This file is important because otherwise certain domains (given from Pfam IDs) will not be listed in the results!": "使用自定义的Anchor文件（.txt格式），为DSS索引中的某些域赋予特殊权重。这个文件很重要，因为否则某些域（由Pfam ID给出）将不会出现在结果中！",
    "Even if BiG-SCAPE finds processed domtables files we can force the domain prediction with hmmscan. Toggle to force hmmscan!": "即使BiG-SCAPE找到了处理过的domtables文件，我们也可以强制使用hmmscan进行域预测。切换以强制执行hmmscan！",
    "Select yes if you want to use a list to filter the input to certain domains. Look in the help section how this list should look like": "如果您想要使用列表来筛选输入到特定域，请选择“是”。请查看帮助部分，了解此列表应该如何设置。",
    "Upload a text file where the inputs are filtered based on the Pfam IDs/domains given in this file!": "上传一个文本文件，其中输入基于该文件中给定的Pfam ID/域进行过滤！",
    "Utilized only if a distance matrix is required by the Ordination method selected above": "只有在所选的排序方法需要距离矩阵时才使用",
    "per sample": "每个样本",
    "Select the same database used to classify reads": "选择用于分类读取的相同数据库",
    "Replace special chars (\\t| |\\||\\.;) with underscore (_)": "将特殊字符（\\t| |\\||\\.;）替换为下划线(_)",
    "Combines rows under the selected taxon and reports only selected rank.": "将选定分类群下的行合并，并仅报告选定的等级。",
    "Trees are pruned at specified rank when clustering": "在聚类时，树在指定的等级上进行剪枝",
    "string like 'TYP:csv,TID:1,LEN:3,SCO:6,UNC:0' where valid file Types are csv/tsv/ssv, and the rest of fields indicate the number of column used (starting in 1) for the TaxIDs assigned,the LENgth of the read, the SCOre given to the assignment": "字符串格式为 'TYP:csv,TID:1,LEN:3,SCO:6,UNC:0'，其中有效的文件类型为 csv/tsv/ssv，其余字段表示用于分配 TaxIDs 的列数（从 1 开始），读取的长度，分配的得分。",
    "Default : CSV": "默认：CSV",
    "remove html output": "移除HTML输出",
    "Add control samples (default is 0)": "添加对照样本（默认为0）",
    "Scoring algorithm, depending of the input files, see the help section of the tools": "评分算法，根据输入文件的不同，请参阅工具的帮助部分",
    "All pass by default (--minscore)": "默认情况下全部通过（--minscore）",
    "If not specified a value will be automatically assigned (--mintaxa)": "如果没有指定一个值，将会自动分配一个值（--mintaxa）",
    "Default: no exclude": "默认：不排除",
    "Default: all included": "默认：包含全部",
    "Default: no": "默认值：否",
    "Minimum score/confidence of the classification of a read in control samples to pass the quality filter; it defaults to minscore": "在对照样本中，一个读段的分类得分/置信度通过质量过滤器的最小值；它默认为minscore",
    "If not specified a value will be automatically assigned": "如果没有指定值，将自动分配一个值",
    "Select to 'add' summary samples to other samples, or to 'only' show summary samples or to 'avoid' summaries at all": "选择“添加”摘要样本到其他样本中，或者“仅”显示摘要样本，或者“避免”所有摘要",
    "Tabular with two columns: contig-IDs and bin-IDs. Fasta_to_Contigs2Bin can be used to  Converts genome bins in fasta format to Contigs-to-bin table": "表格包含两列：contig-IDs和bin-IDs。Fasta_to_Contigs2Bin可以用来将fasta格式的基因组bins转换为Contigs-to-bin表格。",
    "The file should be prodigal fasta format: >contigID_geneNo": "文件应该是prodigal fasta格式：>contigID_geneNo",
    "Only change if you know what you are doing": "仅在您知道自己在做什么的情况下更改",
    "Can also be done by sample, just provide the name of the variable containing unique sample ID's": "也可以按样本进行操作，只需提供包含唯一样本ID的变量名称",
    "Threshold for defining abundant (a.k.a. core) OTU's for each group": "定义每个组中丰富（即核心）操作分类单元（OTU）的阈值",
    "Margin plots display the cumulative read abundances of all OTU's per group sharing either x or y axis with the main plot": "边距图显示了所有OTU在与主图共享x轴或y轴的每个组中的累积读取丰度。",
    "As indicated in the margin plots on top of the bars. Set to 0 to disable.": "如条形图上方的边缘图所示。设置为0以禁用。",
    "Needed to demultiplex the FASTQ files using sdm. If the FASTQ are already demultiplexed, this can be omitted.": "需要使用sdm对FASTQ文件进行去多重化。如果FASTQ文件已经被去多重化了，这个步骤可以省略。",
    "E.g. 16S primer fwd": "例如 16S 引物 前向",
    "E.g. 16S primer rev": "例如 16S 引物反向",
    "E.g. 4:1,4:2,3:3 . See https://lotus2.earlham.ac.uk/images/Derep_options.pdf for how to specify this parameter. If not specified, LotuS2 will select an appropriate default for the chosen clustering algorithm.": "例如：4:1,4:2,3:3。请参见 https://lotus2.earlham.ac.uk/images/Derep_options.pdf 了解如何指定此参数。如果未指定，LotuS2将为所选的聚类算法选择一个适当的默认值。",
    "Includes unclassified OTUs (no Phylum assignment) in OTU and taxa abundance matrix calculations": "包括未分类的OTUs（没有门的分配）在OTU和分类群丰度矩阵计算中",
    "OTUs matching this regular expression will be assigned as unknown, and by default removed": "匹配这个正则表达式的OTUs将被指定为未知，并且默认情况下会被移除。",
    "FASTQ file with barcodes (in the processed mi/hiSeq format), if provided by the sequencer": "带有条形码的 FASTQ 文件（如果测序仪提供了经过处理的 mi/hiSeq 格式）",
    "Useful for low-bacterial biomass samples to remove possible host genome contaminations": "适用于低细菌生物量的样本，以去除可能的宿主基因组污染",
    "Fasta sequence to be cleaned using short-reads data": "使用短读序列数据清洗Fasta序列",
    "Choose number of aligned sam/bam files. Need aligned file with all possible locations in aligner option": "选择对齐的sam/bam文件的数量。需要一个包含比对器选项中所有可能位置的对齐文件。",
    "Specify dataset with only one SAM/BAM file": "指定仅包含一个SAM/BAM文件的数据集",
    "Specify the forward SAM/BAM files": "指定正向的 SAM/BAM 文件",
    "Specify the reverse SAM/BAM files": "指定反向的SAM/BAM文件",
    "Specify a list of single-end dataset": "指定一个单端数据集的列表",
    "A base must occur at least this many times in the pileup to be considered valid [default: 5]": "在堆叠中，一个碱基必须至少出现这么多次才被认为是有效的 [默认值：5]",
    "A base must make up less than this fraction of the read depth to be considered invalid [default: 0.2]": "一个碱基必须占测序深度的少于这个比例才能被认为是无效的【默认值：0.2】",
    "Ignore alignments with more than this many mismatches and indels [default: 10]": "忽略具有超过这个数量的不匹配和插入/删除的比对 [默认值：10]",
    "A base must make up at least this fraction of the read depth to be considered valid [default: 0.5": "一个碱基必须至少占到读取深度的这个分数才能被认为是有效的 [默认值：0.5]",
    "Chao1 and ACE": "Chao1 和 ACE",
    "before calculating alpha diversity and/or richnes": "在计算alpha多样性和/或丰富度之前",
    "Choose between Galaxy Taxonomy and generic table format (e.g. from MetaPhlAn or mothur)": "选择Galaxy分类学和通用表格格式（例如来自MetaPhlAn或mothur）之间的格式",
    "Select a taxonomy dataset": "选择一个分类学数据集",
    "-d; Show all level from root up to this point": "-d; 显示从根节点到这一点的所有层级",
    "Select a MetaPhlAn dataset": "选择一个MetaPhlAn数据集",
    "-n; Otherwise it will simply be called \"Root\"": "-n; 否则它将简单地被称为“根”",
    "-c; Combine data from each dataset, rather than creating separate datasets within the chart": "-c; 将每个数据集的数据合并，而不是在图表中创建单独的数据集",
    "Select taxonomy classification produced by kraken": "选择由Kraken生成的分类学分类",
    "A floating point number between 0 and 1; default=0": "一个介于0和1之间的浮点数；默认值=0",
    "you may provide multiple assemblies for screening": "您可以提供多个基因组组装用于筛选",
    "Screen for capsule synthesis loci (K-loci) using the 'Kaptive' tool": "使用“Kaptive”工具筛选胶囊合成位点（K-位点）",
    "Screen for lipopolysaccharide loci (O-loci) using the 'Kaptive' tool": "使用“Kaptive”工具筛选脂多糖位点（O-位点）",
    "Must be a value between 0 and 100": "必须是一个介于0到100之间的值",
    "Confidence score threshold. Must be in [0, 1]": "置信度得分阈值。必须在[0, 1]范围内",
    "Minimum base quality used in classification (only effective with FASTQ input)": "用于分类的最小碱基质量（仅在FASTQ输入时有效）",
    "Number of overlapping k-mers sharing the same minimizer needed to make a call": "需要重叠的k-mers共享相同的最小值以进行调用的数量",
    "Quick operation (use first hit)": "快速操作（使用首次命中）",
    "Sets --unclassified-out and --classified-out": "设置 --未分类输出 和 --已分类输出",
    "Report minimizer and distinct minimizer count information in addition to normal Kraken report": "在常规Kraken报告的基础上，报告最小化器和不同最小化器计数信息  Kraken是一种用于识别和分类宏基因组学数据中的序列的生物信息学工具。最小化器是一种用于减少Kraken数据库大小的技术，它通过选择代表性序列来减少数据库中的序列数量，从而减少计算资源的使用。在Kraken报告中，除了常规的分类信息外，还可以报告最小化器和不同最小化器的数量，以提供更多关于数据库结构和分类精度的信息。",
    "Joint processing processes all reads at once in a single job creating a single output (two in the case of paired data). Batch processes the samples in separate jobs and creates separate output for each": "联合处理会在一个单一的作业中一次性处理所有读取，创建一个单一的输出（对于配对数据则是两个）。批量处理则会在单独的作业中处理样本，并为每个样本创建单独的输出。",
    "Create a single plot for all data sets (default) or a separate plot for each data set": "为所有数据集创建一个单一的图表（默认）或为每个数据集创建一个单独的图表",
    "number of records to sample from the fastq file": "从fastq文件中采样的记录数",
    "despite the parameter name the sequences don't need to be dereplicated": "尽管参数名称如此，但序列无需去重。",
    "The forward and reverse-complemented reverse read are concatenated rather than merged, with a 10 Ns spacer inserted between them": "正向和反向互补的反向读取是串联而不是合并的，在它们之间插入了10个N的间隔符。",
    "Overhangs are when the reverse read extends past the start of the forward read, and vice-versa, as can happen when reads are longer than the amplicon and read into the other-direction primer region": "过挂是指反向读取延伸到正向读取的起始点之后，反之亦然，当读取比扩增子更长并读取到另一个方向的引物区域时，就会发生这种情况。",
    "If a reference data set of interest is not listed, contact the Galaxy administrators": "如果感兴趣的参考数据集没有列出，请联系Galaxy管理员",
    "comma separated list": "逗号分隔列表",
    "for assigning a taxonomic level": "用于指定分类水平",
    "the reverse-complement of each sequence will be used for classification if it is a better match to the reference sequences than the forward sequence": "每个序列的反向互补序列将用于分类，如果它与参考序列的匹配度比正向序列更好。",
    "Seed for the random number renerator. Set it in order to reproduce exactly the same results.": "随机数生成器的种子。为了精确地重现相同的结果，请设置它。",
    "kmer: also known as oligonucleotides words": "kmer：也被称为寡核苷酸词",
    "If not specified (default) the whole sequence is used": "如果未指定（默认），则使用整个序列",
    "between each moving window tested": "在每个测试的移动窗口之间",
    "forward or reverse reads should be processed separately": "正向或反向读取应该分别处理",
    "Pick samples at random, otherwise samples are read in the provided order until enough reads are obtained (default).": "随机选择样本，否则将按照提供的顺序读取样本，直到获得足够的读取次数（默认）。",
    "If convergence was not reached in MAX_CONSIST steps, the estimated error rates in the last step are returned.": "如果MAX_CONSIST步内没有达到收敛，则返回最后一步中的估计错误率。",
    "For reasons of convergence, and because it is more conservative, it is recommended to set this value to 0, which means that all reads are counted and contribute to estimating the error rates.": "由于收敛的原因，以及这种方法更为保守，建议将此值设置为0，这意味着所有读取都被计数并有助于估计错误率。",
    "process samples jointly (default) or in independent jobs (see also below)": "联合处理样本（默认）或独立处理样本（另见下文）",
    "only applies to paired end data": "只适用于配对末端数据",
    "see below": "见下文",
    "Create extra table with the number of reads pre and post filtering": "创建一个额外的表格，列出过滤前后的读段数。",
    "The fragment coordinates in square brackets refer to the original fragment in which the exon was found.": "方括号中的片段坐标指的是外显子被发现的原始片段。",
    "Select one of the unvalidated PointFinder organisms to use for scanning for point mutations. These have not been fully validated as compatible with staramr so caution is recommended when interpreting results.": "选择一个未经验证的PointFinder生物体用于扫描点突变。这些尚未完全验证为与staramr兼容，因此在解释结果时建议谨慎。",
    "Report all BLAST results (includes overlapping hits, mainly for debugging)": "报告所有BLAST结果（包括重叠的命中，主要用于调试）",
    "Pass a file containing a list of genes to exclude from the ResFinder/PointFinder results": "传递一个包含要从ResFinder/PointFinder结果中排除的基因列表的文件",
    "Pass a file containing a list of complex mutations to group (if present) in the PointFinder results": "将包含复杂突变列表的文件传递给PointFinder结果进行分组（如果存在）",
    "Infinity if m=0": "无穷大，如果 m=0",
    "Use 0 to turn this off": "使用0来关闭这个功能",
    "This means that, when we compare these sequences to some other sequences using lastal, lowercase letters will be excluded from initial matches. This will apply to lowercase letters in both sets of sequences.": "这意味着，当我们使用lastal比较这些序列与其他序列时，小写字母将被排除在初始匹配之外。这将适用于两组序列中的小写字母。",
    "Omit any header lines from the output": "从输出中省略任何标题行",
    "Join neighboring alignments if they are co-linear and separated by at most N letters.": "如果相邻的比对是共线性的，并且它们之间最多相隔N个字母，则将它们连接起来。",
    "Example: 'ID:1 PL:ILLUMINA SM:mysample'": "示例：'ID:1 PL:ILLUMINA SM:mysample'",
    "This option overrides all other accept/reject options including identity. (--acceptall)": "这个选项会覆盖所有其他接受/拒绝选项，包括身份。（--acceptall）",
    "Allowed values: 3-15 (--wordlength)": "允许的值范围：3-15 (--wordlength)",
    "Zero to use random data source.": "零表示使用随机数据源。",
    "If this option is not enabled, every matching strain from the same species of the reference database is reported in the result.": "如果这个选项没有被启用，那么参考数据库中同一物种的每个匹配菌株都会在结果中被报告。",
    "If specified, sketching will conclude if this coverage is reached before the end of the input file (estimated by average k-mer multiplicity)": "如果指定了，草图将在达到此覆盖度之前结束，而不管输入文件是否结束（通过平均k-mer多重性估计）。",
    "If specified, will be used for p-value calculation instead of an estimated size from k-mer content": "如果指定了，将用于计算p值，而不是使用从k-mer内容估计的大小",
    "Sketch individual sequences, rather than whole files, e.g. for multi-fastas of single-chromosome genomes or pair-wise gene comparisons": "为单个序列绘制草图，而不是整个文件，例如用于单染色体基因组的多fasta或成对基因比较。",
    "Each sketch will have at most this many non-redundant min-hashes": "每个草图最多有这么多非冗余的最小哈希值",
    "min AA length of a metapeptide": "最小氨基酸长度的代谢肽",
    "min base-call phred score across any NT in a metapeptide": "在任何NT（核苷酸）中的最小碱基呼叫Phred分数，用于一个metapeptide（元肽段）。",
    "min length of ORF-portion": "ORF部分的最小长度",
    "minimum length of the longest tryptic peptide": "最长的胰蛋白酶肽段的最小长度",
    "minimum read count": "最小读取计数",
    "stop early if we hit this many reads": "如果达到这么多的读取数，就提前停止",
    "MetaGene Annotator output file. Records must be in same linear order as reads in fastqfiles": "MetaGene注释器输出文件。记录必须与fastq文件中的读取顺序相同",
    "minimum MetaGene score": "最小MetaGene分数",
    "Can be generated with sixgill_build": "可以用sixgill_build生成",
    "min AA sequence length": "最小氨基酸序列长度",
    "maximum number of metapeptides to write": "最大写入的代谢肽段数",
    "Only trim when a sequence has high enough coverage; median abundance > 20": "仅当序列具有足够高的覆盖度时才进行修剪；中位数丰度 > 20",
    "Trim at k-mers below this abundance": "在丰度低于这个值的k-mers处进行修剪",
    "No more than this many number of sequences will be stored in each output": "每个输出中将存储的序列数量不会超过这个数目",
    "The minimum partition size worth keeping (--min-partition-size/-m)": "最小值得保留的分区大小（--min-partition-size/-m）",
    "The tool will fail if given improperly paired reads and this option is selected.": "如果选择了这个选项，并且给定了配对不正确的读取，工具将会失败。",
    "By default this tool process reads in a pair-aware manner. This option disables that behavior.": "默认情况下，这个工具以配对感知的方式处理读取。这个选项禁用了这种行为。",
    "If all but one of your sequence files are interleaved paired end reads you can include one unpaired file to be processed last without regard to pairing.": "如果您的所有序列文件中只有一个不是成对的交错读取，则可以将一个未配对的文件包含在最后处理，而无需考虑配对。",
    "The inputs file(s) will be processed using the kmer counts in the specified k-mer countgraph file as a starting point.": "输入文件（们）将使用指定的k-mer计数图文件中的k-mer计数作为起点进行处理。",
    "FASTA or FASTQ datasets": "FASTA或FASTQ数据集",
    "FASTA or FASTQ dataset": "FASTA 或 FASTQ 数据集",
    "Quick mode: rather than searching all k-mers in a sequence, stop classification after a specified number of database hit": "快速模式：不是在序列中搜索所有k-mers，而是在达到指定数量的数据库命中后停止分类。",
    "min-hits will allow you to require multiple hits before declaring a sequence classified, which can be especially useful with custom databases when testing to see if sequences either do or do not belong to a particular genome; default=1": "最小命中数（min-hits）允许您在将序列分类之前要求多个命中，这在使用自定义数据库进行测试以查看序列是否属于特定基因组时特别有用；默认值=1",
    "The classication must be made with --only_official names": "分类必须使用 --only_official 名称进行。",
    "Required if summarising a contig2classification.names output derived from CAT contigs": "如果从CAT contigs派生出contig2classification.names输出，则需要此选项",
    "Leave empty to subsample by community size for the smallest sample.": "留空以根据最小样本的群落大小进行子采样。",
    "Output file of MetaPhlAN with all taxonomic levels (from kingdom to strains, --metaphlan2_output)": "MetaPhlAN输出文件包含所有分类水平（从界到菌株，--metaphlan2_output）",
    "default = 10": "默认值 = 10",
    "default = 200": "默认值 = 200",
    "FastQ format is recommended for kmer algorithm and Fasta for the alignment": "建议使用FastQ格式进行kmer算法，使用Fasta格式进行比对。",
    "Subsample iteratively applying this factor to the number of reads, resulting in logarithmic subsampling. Use 0 to fall back to linear sampling, control": "迭代地将这个因子应用于读取数，从而实现对数下采样。使用0则退回到线性采样，控制",
    "If it is not a multiple of the number of threads, it is rounded to the next (upper) multiple": "如果它不是线程数的倍数，则向上取最近的倍数",
    "The similarity is evaluated for the aligned region only": "相似性仅针对对齐区域进行评估",
    "This is useful for single stranded sequences data": "这对于单链序列数据很有用",
    "By default, Ns (unknown nucleotides) match any nucleotide": "默认情况下，Ns（未知核苷酸）匹配任何核苷酸",
    "Reducing this option will increase sensitivity while increasing running time": "减少这个选项将提高敏感性，同时增加运行时间",
    "You can increase kmer size to increase sensitivity": "您可以增加kmer大小以提高灵敏度",
    "Higher values reduce accuracy but increase speed. If set, it overides the maximum number of reads to use as query": "较高的值会降低准确性但提高速度。如果设置，它将覆盖作为查询使用的最大读取数。",
    "This file is typically obtained with merge_metaphlan_tables": "这个文件通常是通过merge_metaphlan_tables获得的",
    "Automatically set, this option should not be necessary unless for very large heatmaps": "自动设置，除非是非常大的热图，否则此选项不应必要",
    "Values around 0.001 are also reasonable": "值在0.001左右也是合理的",
    "Default is maximum value present, can be set e.g. to 100 to display the full scale": "默认值是存在的最高值，可以设置，例如设置为100以显示全量程",
    "BIOM format": "生物观测矩阵（BIOM）格式",
    "Flag to either calculate the proper percentile using R's step-wise quantile function or approximate function.": "标志位，用于计算使用R语言的逐步百分位数函数或近似函数的正确百分位数。",
    "Cutoff for the relative difference from one median difference from the reference to the next": "相对差异的截止值，该差异是从一个中位数差异到下一个中位数差异的参考差异",
    "Manual Percentile": "手动百分位数",
    "(default=1000)": "（默认值=1000）",
    "Select column containing read names": "选择包含读名字的列",
    "Select column containing taxonomy ID": "选择包含分类学ID的列",
    "SCOARY will not report genes with higher p-values than this (Default=1.0=All). Provide a single value (applied to all) or exactly as many values as correction criteria and in corresponding order (e.g., 0.05 0.05 for two traits).": "SCOARY 不会报告 p 值高于这个值的基因（默认=1.0=全部）。提供一个单一的值（应用于全部）或提供与校正标准数量相同且顺序一致的值（例如，对于两个性状，提供 0.05 0.05）。",
    "Apply the p-value corrections to the p-value cutoffs you have entered (Default = Individual p-value).": "应用 p 值校正到您输入的 p 值截止值（默认 = 单独 p 值）。",
    "Perform N number of permutations of the significant results post-analysis. (Default = 0 = None)": "在分析后对显著结果执行N次排列。（默认值 = 0 = 无）",
    "SCOARY will only report the top max hits results per trait. (Default = 0 = All)": "SCOARY 只会报告每个特征的前 max hits 个结果。（默认值 = 0 = 全部）",
    "Collapse correlated genes (genes that have identical distribution patterns in the sample) into merged units. (Default=false)": "将相关基因（在样本中具有相同分布模式的基因）合并为合并单元。（默认值=false）",
    "Calculate UPGMA tree to a newick file.": "计算 UPGMA 树到一个 Newick 文件。",
    "Don't output k-mer with count less than this value": "不要输出计数小于这个值的k-mer",
    "Don't output k-mer with count greater than this value": "不输出计数大于该值的k-mer",
    "Paired-end Fasta/FastQ files should be merged first": "配对端的Fasta/FastQ文件应该首先合并",
    "Give ID not full name": "给出ID而不是全名",
    "Automatic if not provided": "如果未提供，则自动",
    "Others will be collapsed": "其他将被折叠",
    "Taxa are always proportional to height": "类群总是与高度成比例",
    "The format is explained in the help": "格式在帮助中解释",
    "Mapping from features to groups": "从特征映射到组",
    "More features can be renamed with advanced mode": "更多功能可以在高级模式下重命名",
    "The mapping file must be tabular format file with two tab-separated columns. The first column must contain the value you want to modify and the second contain the new value": "映射文件必须是具有两列的表格格式文件，第一列必须包含您想要修改的值，第二列包含新值。",
    "If input is from picrust metagenome_contributions.py": "如果输入来自 picrust metagenome_contributions.py",
    "Only genes with prevalence higher than the threshold are allowed": "只有流行度高于阈值的基因才被允许",
    "Only genes with prevalence lower than the threshold are allowed": "只有流行度低于阈值的基因才被允许",
    "e.g. 'Streptococcus'": "例如 '链球菌'",
    "Applied only with FASTA files": "仅适用于FASTA文件",
    "BowTie2 needs to be used first to map microbiota reads": "BowTie2需要首先用来映射微生物群落的读取数据",
    "File needs to be generated with MetaPhlAn versions >3.0": "需要生成的文件必须使用 MetaPhlAn 版本大于 3.0",
    "Markers are also extracted for subclades": "标记也用于提取亚群",
    "It is used for normalizing the length-normalized counts with the microbiota size as well. No normalization applied if the value is not specified": "它用于使用微生物群大小对长度标准化计数进行标准化。如果未指定值，则不应用标准化。",
    "One marker per line": "每行一个标记",
    "It is generally recommended to keep the disambiguation procedure in order to minimize false positives": "通常建议保持消歧程序，以最小化误报",
    "1 marker per line": "每行一个标记",
    "Generated by the Calculate contig depths for MetaBAT2 tool": "由“计算MetaBAT2的contig深度”工具生成",
    "Generated by a tool - not the Calculate contig depths for MetaBAT2 tool": "由工具生成 - 非MetaBAT2计算连体深度的工具",
    "The greater, the more sensitive": "越大，越敏感",
    "The greater, the more specific": "越大，越具体",
    "Use it to skip the preparation step (0:auto)": "使用它来跳过准备步骤（0：自动）",
    "Sum of depth over minCV": "最小变异系数覆盖度的总和",
    "Zero value will use random seed": "零值将使用随机种子",
    "by a discrete variable in the metadata": "通过元数据中的离散变量",
    "in order to keep abundances of 0 in the calculation of medians": "为了在计算中位数时保留0的丰度",
    "By a categorical variable in the metadata": "在元数据中通过一个分类变量",
    "By a categorical variable in the metadata.": "通过元数据中的分类变量。",
    "Add an additional row at the bottom displaying the sum of all remaining taxa that are not part of the top tax_show most abundant taxa.": "在底部添加一行，显示所有未被包含在最多丰度分类群展示中的剩余分类群的总和。",
    "All values below this value are given the same color.": "所有低于这个值的值都被赋予相同的颜色。",
    "All values above this value are given the same color.": "所有高于这个值的值都被赋予相同的颜色。",
    "Produces a 2-column grid plot, showing known functional information about the Genus-level OTUs next to the heatmap. When using this feature, make sure that either tax_aggregate or tax_add is set to Genus and that Genus is the lowest level in either.": "生成一个两列的网格图，显示关于属级操作分类单元（OTUs）的已知功能信息，并将其显示在热图旁边。使用此功能时，请确保将tax_aggregate或tax_add设置为属（Genus），并且属是两者中最低的分类级别。",
    "See help": "查看帮助",
    "By default ampvis2 expects a column named ASV or OTU containing the ASV or OTU identifiers. By checking this a column with an empty header will be used (as produced by dada2).": "ampvis2 默认期望有一个名为 ASV 或 OTU 的列，其中包含 ASV 或 OTU 标识符。勾选此选项将使用一个带有空标题的列（由 dada2 生成）。",
    "By checking this the identifiers will be renamed to ASV1, ASV2, etc and the sequences will be stored in the ampvis2 object.": "勾选此选项后，标识符将被重命名为 ASV1、ASV2 等，序列将存储在 ampvis2 对象中。",
    "Needed by most downstream tools. Select if the inputs contain taxonomic / metadata information.": "大多数下游工具都需要。如果输入包含分类学/元数据信息，请选择。",
    "The full DNA sequences will then be used as the new names in the output.": "完整的DNA序列将用作输出中的新名称。",
    "If the read counts are integers, any decimals present after merging will be rounded either up or down. Make sure this makes sense if the read counts have been normalised, as it may result in 0's, 1's, and 2's everywhere.": "如果读取计数是整数，合并后出现的任何小数将向上或向下四舍五入。如果读取计数已经过标准化处理，请确保这在逻辑上是有意义的，因为这可能导致到处都是0、1和2。",
    "OTU's that have not been assigned at the chosen level will be removed with a message.": "未被分配到所选水平的OTU将被移除，并伴随一条消息。",
    "Variable needs to be categorical/discrete": "变量需要是分类/离散的",
    "Only relevant if for group_by a value is selected": "仅在选择了group_by的值时才相关",
    "Remove low abundant OTU's across all samples below this threshold in percent. Setting this to 0 may drastically increase computation time.": "移除所有样本中低于此百分比阈值的低丰度OTU。将此设置为0可能会大幅增加计算时间。",
    "Note that PCoA is not performed by the vegan package, but the pcoa function from the APE package.": "请注意，主坐标分析（PCoA）不是由vegan包执行的，而是由APE包中的pcoa函数执行的。",
    "Based on the used arguments. The caption includes a description of how the result has been generated as well as references for the methods used.": "根据所使用的参数，标题包含了对结果生成方式的描述以及所使用的方法的引用。",
    "Split by the variable defined by sample_color_by": "根据由 sample_color_by 定义的变量进行拆分",
    "Ordered by the sum of the numerical values of the x,y coordinates. Only makes sense with PCA/RDA).": "按 x,y 坐标数值之和排序。仅与 PCA/RDA 有意义。",
    "The lengths of the arrows are scaled by significance.": "箭头的长度按显著性进行了缩放。",
    "For displaying the fitting results": "用于展示拟合结果",
    "To prevent cluttering of the plot": "为了预防图表混乱",
    "0: invisible, 1: opaque.": "0: 透明，1: 不透明。",
    "plot of variances against the number of the principal component": "方差与主成分数量的图表",
    "Often most variation is observed among the most abundant OTU's, log10-transforming the x-axis will highlight this better.": "通常在丰度最高的操作分类单元（OTU）之间观察到的变异最大，对x轴进行log10变换可以更好地突出这一点。",
    "Lower is prettier but takes more time to generate.": "低分辨率的看起来更漂亮，但生成时间更长。",
    "Only relevant if for facet_by a value is selected": "仅在为facet_by选择一个值时才相关",
    "Per sample. Samples below this value will be removed initially": "每个样本。低于这个值的样本最初将被移除",
    "This is done initially, but after filtering based on the minreads value, if set.": "这是最初完成的，但在基于minreads值进行过滤后，如果设置了该值。",
    "Generated with ampvis2: load": "使用ampvis2生成：加载",
    "Selected taxa will be removed instead of being the only ones kept in the data.": "选定的分类群将被移除，而不是作为数据中唯一保留的分类群。",
    "Must be directly compatible with as_date and preferably of the form yyyy-mm-dd or %Y-%m-%d": "必须与as_date直接兼容，最好采用yyyy-mm-dd或%Y-%m-%d的形式",
    "OTU's below this abundance are excluded from the analysis": "低于这个丰度的OTU被排除在分析之外",
    "OTU's within the top cut_f of the reads are considered a core OTU": "在读取的顶部cut_f中，OTU被认为是核心OTU",
    "Level to push all reads to": "将所有读取推送到的级别",
    "Threshold for the minimum number of reads kraken must assign to a classification for that classification to be considered in the final abundance estimation.": "Kraken必须分配给一个分类的最小读段数量的阈值，以便在最终丰度估计中考虑该分类。",
    "Ignore genomes whose name appears in this newline-separated file when combining shards.": "在合并片段时，忽略那些名称出现在这个用换行符分隔的文件中的基因组。",
    "When this is run, dereplication occurs transparently through the Galah method.": "当运行此操作时，去重操作会通过Galah方法透明地进行。",
    "Require at least this dashing-derived ANI for preclustering and to avoid FastANI on distant lineages within preclusters.": "要求至少使用这个基于dashing的ANI进行预聚类，并避免在预聚类内的不同谱系上使用FastANI。",
    "It is used both for filtering and to rank genomes during clustering": "它既用于过滤，也用于在聚类过程中对基因组进行排序",
    "Genomes with lower completeness percentage will be ignored": "基因组完整性百分比较低的将被忽略",
    "Genomes with higher contamination percentage will be ignored": "基因组中污染百分比较高的将被忽略",
    "Produced by the dada2 removeBimeraDenovo tool": "由dada2的removeBimeraDenovo工具生成",
    "Produced by the dada2 assignTaxonomyAndAddSpecies tool": "由dada2 assignTaxonomyAndAddSpecies工具生成",
    "Rows are sample names, Columns are properties": "行是样本名称，列是属性",
    "Zero value disables": "零值禁用",
    "Ignored when including 1-readlength edges when calculating depth and variance": "在计算深度和方差时，忽略包含1个读取长度的边缘",
    "In addition to the training data": "除了训练数据",
    "Sorted Bam file": "排序的Bam文件",
    "Instead of using the fasta ID (space in header before space), use the full header. Needed for some mapping tools (including bbMap)": "而不是使用fasta ID（标题前面的空格），使用完整的标题。这对于某些映射工具（包括bbMap）是必需的。",
    " Minimum percent identity of read pairs to consensus to use the reads. Must be >, not >=": "用于共识的读对的最小百分比一致性。必须大于，而不是大于等于",
    "Minimum mapq score of EITHER read in a pair to use that pair. Must be >, not >=": "对于一对中的任何一个读段，使用该对的最小映射质量分数。必须大于，而不是大于等于。",
    "Multiplier to determine maximum insert size between two reads - default is to use 3x median insert size. Must be >, not >=": "乘数用于确定两个读取之间的最大插入片段大小，默认情况下是使用3倍的中位数插入片段大小。必须大于，而不是大于等于。",
    "Minimum insert size between two reads - default is 50 bp. If two reads are 50bp each and overlap completely, their insert will be 50. Must be >, not >=": "两个读取之间的最小插入片段大小 - 默认为50个碱基对。如果两个读取各自为50个碱基对，并且完全重叠，则它们的插入片段大小将为50。必须大于，而不是大于等于。",
    "For example long reads or merged reads. This can be a .fastq file or text file with list of read names (will assume file is compressed if ends in .gz": "例如长读段或合并读段。这可以是一个.fastq文件或包含读段名称列表的文本文件（如果文件以.gz结尾，则假定文件是压缩的）。",
    " Minimum coverage to call a variant": "最小覆盖度以调用一个变体",
    "Minimum SNP frequency to confirm a SNV (both this AND the FDR snp count cutoff must be true to call a SNP).": "确认SNV（单核苷酸变异）的最小SNP频率（必须同时满足这个条件和FDR snp计数截止值才能调用一个SNP）。",
    "SNP false discovery rate- based on simulation data with a 0.1 percent error rate (Q30)": "SNP假发现率 - 基于0.1%错误率（Q30）的模拟数据",
    "EXPERIMENTAL; the name of the gene must be in the gene qualifier": "实验性的；基因名称必须在基因限定符中",
    "This can be a file with each line listing a scaffold and a bin name, tab-seperated. This can also be a space-seperated list of .fasta files, with one genome per .fasta file. If nothing is provided, all scaffolds will be treated as belonging to the same genome": "这可以是一个文件，每行列出一个scaffold和一个bin名称，用制表符分隔。这也可以是一个由空格分隔的.fasta文件列表，每个.fasta文件包含一个基因组。如果未提供任何内容，所有scaffold将被视为属于同一个基因组。",
    "Dont perform analysis on an mm level; saves RAM and time; impacts plots and raw_data": "不要在毫米级别上进行分析；节省内存和时间；影响图表和原始数据",
    "Set a number of parameters to values appropriate for mapping to a large fasta file.": "设置一些参数，使其适用于映射到大型fasta文件的值。",
    "Minimum number of reads mapping to a scaffold to proceed with profiling it": "用于构建概要文件的映射到支架的最小读取数",
    "Minimum number of reads mapping to a genome to proceed with profiling it. MUST profile .stb if this is set": "最小映射到基因组的读数数量以进行分析。如果设置了这个参数，则必须对.stb文件进行分析。",
    "Absolute minimum number of reads connecting two SNPs to calculate LD between them.": "计算两个SNP之间连锁不平衡的绝对最小读取数。",
    "Store intermediate dictionaries in the pickle file; will result in significantly more RAM and disk usage": "将中间字典存储在pickle文件中；这将导致显著增加的RAM和磁盘使用量",
    "File containing a list of scaffolds to profile- if provided will ONLY profile those scaffolds": "包含要分析的支架列表的文件 - 如果提供，则仅分析这些支架",
    "When calculating nucleotide diversity, also calculate a rarefied version with this much coverage": "在计算核苷酸多样性时，还要计算一个具有如此多覆盖度的稀释版本",
    "Break scaffolds into windows of this length when profiling": "在进行分析时，将支架分成这个长度的窗口",
    "Make a detailed read report indicating deatils about each individual mapped read": "制作一个详细的读取报告，报告中包含每个单独映射读取的详细信息。",
    "Do not generate tables that consider groups of scaffolds belonging to genomes": "不要生成考虑属于基因组的支架组的表格",
    "Do not make plots": "不生成图表",
    " The Zip files for the IS profiles outputs you want to compare": "您想要比较的 IS 轮廓输出的 Zip 文件",
    "Both this AND the FDR snp count cutoff must be true to call a SNP.": "要调用一个SNP，这两个条件以及FDR snp计数截止值都必须为真。",
    "All profiles must have run Profile with the same .stb": "所有配置文件必须使用相同的.stb文件运行。",
    "higher will use more RAM and run more quickly": "更高的值会使用更多的RAM，并且运行得更快",
    "Must provide .stb file to do so": "必须提供 .stb 文件才能这样做",
    "if below the popANI will be set to 0": "如果低于popANI，将被设置为0",
    "The fixed effects for the model, comma-delimited for multiple effects": "模型的固定效应，用于多个效应时用逗号分隔",
    "The random effects for the model,  comma-delimited for multiple effects": "模型的随机效应，多个效应用逗号分隔",
    "The minimum abundance for each feature": "每个特征的最小丰度",
    "The minimum percent of samples for which a feature is detected at minimum abundance": "特征被检测到的最小样本百分比，其最小丰度",
    "The q-value threshold for significance": "显著性的q值阈值",
    "The correction method for computing  the q-value": "计算q值的校正方法",
    "In heatmap, plot top N features with significant associations": "在热图中，绘制与显著关联的前N个特征",
    "Zero value uses all tables": "零值使用所有表格",
    "MALT reports up to this many best scoring matches for each hit reference sequence": "MALT 报告每个命中的参考序列最多这么多得分最高的匹配",
    "For each read, matches are used for taxonomic placement whose bit disjointScore is within this percentage of the best disjointScore": "对于每个读取，使用匹配项进行分类定位，其位不相交分数是最佳不相交分数的这个百分比内。",
    "Zero value ignores this option": "零值忽略此选项",
    "Overrides the above parameter": "覆盖上述参数",
    "This read file should be the one used by DIAMOND or MALT to generate the SAM file below": "这个读取文件应该是 DIAMOND 或 MALT 用来生成下面 SAM 文件的那个文件",
    "Use 0 if read and mate have the same name": "如果读取和配对具有相同的名称，则使用0",
    "Set to true for NCBI-nr downloaded Sep 2016 or later": "设置为 true 以适用于 2016 年 9 月或之后下载的 NCBI-nr",
    "Specify a space-separated list of tags (e.g., 'gb|' 'ref|')": "指定一个以空格分隔的标签列表（例如，'gb|' 'ref|'）",
    "Bins smaller than this will be eliminated": "小于这个值的箱子将被消除",
    "Table where each row correspond to a contig, and each column correspond to a sample. The values are the average coverage for this contig in that sample": "表格中每一行对应一个连续序列，每一列对应一个样本。这些值是该连续序列在该样本中的平均覆盖度。",
    "It is named the composition file since it is used to calculate the kmer composition (the genomic signature) of each contig.": "它被称为组成文件，因为它用于计算每个连续序列的kmer组成（基因组特征）。",
    "Contigs shorter than this value will not be included": "比这个值短的连续序列将不会被包含",
    "By default, the coverage is normalized for samples, then normalized for contigs and finally log transformed. By setting this flag you skip the normalization and only do log transorm of the   coverage.": "默认情况下，覆盖度首先针对样本进行标准化，然后针对连续序列进行标准化，最后进行对数转换。通过设置这个标志，您将跳过标准化，只对覆盖度进行对数转换。",
    "By default, total coverage is included, independently of coverage normalization but previous to log transformation. Use this tag to escape this behaviour.": "默认情况下，总覆盖度被包括在内，与覆盖度标准化无关，但在对数转换之前。使用此标签可避免这种行为。",
    "The file should have four columns representing: 'Contig ID, Start Position, End Position and SubContig ID' respectively. The Subcontig ID must contain the pattern 'concoct_part_[0-9]*' while the contigs which are not cutup cannot contain this pattern. This file can be generated by the CONCOCT: Cut up contigs tool.": "该文件应有四列，分别表示：'Contig ID（片段ID）、Start Position（起始位置）、End Position（结束位置）和 SubContig ID（子片段ID）'。SubContig ID 必须包含模式 'concoct_part_[0-9]*'，而未被切割的 contigs 不能包含此模式。此文件可由 CONCOCT: Cut up contigs（切割片段）工具生成。",
    "Information used to process the BAM files": "用于处理BAM文件的信息",
    "BAM file with mappings to the original contigs, after sorting": "BAM文件，排序后映射到原始基因组片段",
    "BAM files with mappings to the original contigs, after sorting": "BAM文件在排序后映射到原始的连续片段",
    "Zero value produces non-overlapping parts": "零值产生不重叠的部分",
    "Can be used as input to the SAMTools bedcov tool": "可以用作SAMTools bedcov工具的输入",
    "Implies the input file was produced by the MEGAN: DAA Meganizer tool": "意味着输入文件是由MEGAN：DAA Meganizer工具生成的",
    "Contains all classifications, but no reads or alignments": "包含所有分类，但没有读取或比对",
    "Set to Yes only if blast mode of input is BlastX": "仅当输入的blast模式为BlastX时，设置为Yes",
    "Leave blank to extract all classes": "留空以提取所有类别",
    "Can be used to restrict the search to only a few organism groups types to save time, if one or more of the origins are not relevant to the dataset under study. Default: all": "可以用来限制搜索，只搜索少数几个生物群组类型以节省时间，如果一个或多个起源与正在研究的数据集不相关。默认值：全部",
    "Domain E-value cutoff for a sequence to be included in the output. Default: 1e-5": "序列被包含在输出中的域E值截止值。默认值：1e-5",
    "Domain score cutoff for a sequence to be included in the output. Default:0": "序列被包含在输出中所需的域得分截止值。默认值：0",
    "The minimal number of domains that must match a sequence before it is included. Default:2": "必须匹配序列的最小域数，才能将其包含在内。默认值：2",
    "Selects what will be of highest priority when determining the origin of the sequence. Default:score": "选择在确定序列起源时优先考虑的因素。默认值：得分",
    "Default: Score cutoff": "默认：分数截断",
    "High numbers may slow down the process": "高数值可能会减慢处理过程",
    "Low numbers may slow down the process": "低数值可能会减慢进程",
    "Allow inclusion of sequences that only find a single domain, given that they meet the given E-value and score thresholds. Default: 0": "允许包含仅找到一个域的序列，只要它们满足给定的E值和分数阈值。默认值：0",
    "Allows profiles to be in the wrong order on extracted sequences": "允许在提取的序列上，概要文件的顺序错误",
    "Checks both DNA strands against the database, creating reverse complements": "检查两个DNA链与数据库，创建反向互补序列",
    "Leave this setting off for higher precision": "将此设置关闭以获得更高的精度",
    "Preserve sequence headers in input file instead of printing out ITSx headers. Default: No": "在输入文件中保留序列标题，而不是打印ITSx标题。默认值：否。",
    "A comma separated list of regions to output separate FASTA files": "输出单独FASTA文件的区域列表，以逗号分隔",
    "If enabled, output is limited to full-length regions. Default: no": "如果启用，输出将限制为全长区域。默认值：否。",
    "Default: 0 (disabled)": "默认值：0（禁用）",
    "Default: 0": "默认值：0",
    "Default: yes": "默认：是",
    "Rows contain genes and columns contain sequencing read counts": "行包含基因，列包含测序读取计数",
    "Specify a group label and the number of sequencing read count columns associated with it": "指定一个组标签以及与之关联的测序读数列的数量",
    "Specify a group number and the number of columns from the Reads table file associated with the group": "指定一个组号以及与组相关联的读取表文件中的列数",
    "Expresses how the microbial absolute abundances for each taxon depend on the variables in the metadata": "表达微生物每种分类单元的绝对丰度如何依赖于元数据中的变量",
    "Taxa with prevalences less than this will be excluded": "流行率低于此值的类群将被排除",
    "Samples with library sizes less than this will be excluded": "样本库大小小于这个值的将被排除",
    "Group should be discrete, specifying group is required for detecting structural zeros and performing global test": "组别应该是离散的，指定组别是为了检测结构性零值和执行全局检验所必需的",
    "Recommended if the sample size is small and/or the number of differentially abundant taxa is believed to be large": "如果样本量较小和/或认为差异丰富的类群数量较大，则推荐使用",
    "Output of the CheckM analyze tool": "CheckM 分析工具的输出",
    "Generated by the coverage command": "由 coverage 命令生成",
    "Optional output of the CheckM tree or lineage_wf tools": "可选的CheckM树或lineage_wf工具的输出",
    "Output of the CheckM qa tool or optional output of the lineage_wf or taxonomy_wf tools": "CheckM qa工具的输出，或可选的lineage_wf或taxonomy_wf工具的输出",
    "Output of the CheckM tree tool": "CheckM 树工具的输出",
    "Important only if all the input read sets belong to a single sample": "仅在所有输入的读取集属于单个样本时才重要",
    "This is the file that tells you the ST number that is assigned to known combinations of alleles. Column 1 is the ST, and subsequent columns are the loci that make up the scheme.": "这是一个告诉你已知等位基因组合被分配的序列类型（ST）编号的文件。第一列是序列类型（ST），随后的列是构成方案的位点。",
    "E.g.'-', as in arcc-1": "例如'-'，就像在arcc-1中",
    "Fasta file/s for gene databases": "基因数据库的FASTA文件/们",
    "Leave empty to map all": "留空以映射全部",
    "if either of its ends has unaligned overlap with the reference that is longer than this value": "如果它的任一端与参考序列有未对齐的重叠，并且这个重叠的长度大于这个值",
    "Any new results from this run will also be incorporated": "这次运行的任何新结果也将被纳入",
    "Assigned reads will be recorded only if they are at or below max rank. Default: O": "分配的读段只有在它们的等级等于或低于最大等级时才会被记录。默认值：O",
    "Reads assigned at and below min rank will be recorded as being assigned to the min rank level. Default: S": "读取分配在最小等级及以下的将被记录为分配到最小等级水平。默认值：S",
    "This should be in TSV or CSV (Tabular) format. The first column should be the Sample ID (e.g. 0-kraken_report, 1-kraken_report, etc.). This is the same name as the input files. If no metadata is given, basic metadata is added to support importing the BIOM table into sites like phinch (http://phinch.org/index.html). Example for metadata files: http://qiime.org/documentation/ file_formats.html#mapping-file-overview": "这应该是以制表符分隔的值（TSV）或逗号分隔的值（CSV）的表格格式。第一列应该是样本ID（例如0-kraken_report，1-kraken_report等）。这与输入文件的名称相同。如果没有给出元数据，则会添加基本的元数据以支持将Biom表格导入到像phinch（http://phinch.org/index.html）这样的网站。元数据文件的示例：http://qiime.org/documentation/ file_formats.html#mapping-file-overview",
    "Create a file containing just the (NCBI) OTU IDs for use with a service such as phyloT (http://phylot.biobyte.de/) to generate a phylogenetic tree for use in downstream analysis such as UniFrac, iTol (itol.embl.de), or PhyloToAST (phylotoast.org)": "创建一个仅包含（NCBI）OTU ID的文件，以便与phyloT（http://phylot.biobyte.de/）等服务一起使用，生成用于下游分析的系统发育树，例如UniFrac、iTol（itol.embl.de）或PhyloToAST（phylotoast.org）",
    "Set the output format of the BIOM table. Default is HDF5": "设置BIOM表格的输出格式。默认格式是HDF5。",
    "Select a multisample fasta dataset with reference sequences to base classification on.": "选择一个带有参考序列的多样本fasta数据集，以进行基于参考序列的分类。",
    "Specify the sequenced reads dataset.": "指定序列化读取数据集。",
    "Specify the sequenced reads dataset with forward reads.": "指定带有正向读取的测序数据集。",
    "Specify the sequenced reads dataset with reverse reads.": "指定带有反向读取的测序读取数据集。",
    "Select a collection with forward and reverse reads.": "选择一个包含正向和反向读取的集合。",
    "Determines the maximum number of candidate matches sorted by score that will be reported. Set to zero to get all candidate matches reported.": "确定将报告的最大候选匹配数，按得分排序。设置为零以报告所有候选匹配。",
    "Generate k-mers of this length from the reference sequences and the sequenced reads. Note: smaller k-mer sizes come at the cost of decreased specificity and k-mer sizes below 21 have an increased risk of contaminating sequences getting analyzed. Only decrease the default (21) if you know your sample is pure (i.e., sequenced reads represent viral reads only), or if you have increased --threshold sufficiently.": "从参考序列和测序读取中生成这个长度的k-mers。注意：较小的k-mer大小会降低特异性，且k-mer大小低于21会增加污染序列被分析的风险。只有在你知道你的样本是纯净的（即，测序读取仅代表病毒读取）时，或者你已经充分增加了--threshold参数，才应降低默认值（21）。",
    "Sequenced reads that don't have at least this proportion of their k-mers matching k-mers generated from the reference sequences will not be considered in the analysis; default: 0.2": "序列读取中，如果它们的k-mer与参考序列生成的k-mer匹配的比例至少没有达到这个比例，则在分析中不考虑；默认值：0.2",
    "Minimum coverage by sequenced reads for a reference k-mer to be kept during culling; default: 5": "在剔除过程中，被测序读取覆盖的参考k-mer的最小覆盖度；默认值：5",
    "Minimum proportion of matching kmers required for queries; default: 0.1": "查询所需的匹配k-mer的最小比例；默认值：0.1",
    "Of the queries still considered after applying the --min_kmer_prop filter above, the tool will calculate and report scores only for this fraction. Lowering this value leads to shorter runtime because fewer scores will have to be calculated, but also to fewer results getting reported. CAVEAT: this version of the tool will round down the result of applying this parameter so it is possible to end up with zero queries to be considered further. Change only if runtime is an issue!": "在应用上述的--min_kmer_prop过滤器后，该工具将只为这个比例的查询计算并报告分数。降低这个值可以缩短运行时间，因为需要计算的分数更少，但同时报告的结果也会减少。警告：这个版本的工具会将应用此参数的结果向下取整，因此可能会最终没有查询被进一步考虑。只有在运行时间是问题时才改变！",
    "Produced by the DRAM annotate tool": "由DRAM注释工具生成",
    "Optional, leave blank to ignore": "可选的，留空以忽略",
    "Produced by the DRAM annotate tool (optional, leave blank to ignore)": "由DRAM注释工具生成（可选，留空以忽略）",
    "Single mode requires genomes which are high quality with low contamination and long contigs (average length >3 Kbp)": "单模式需要高质量、低污染和长片段（平均长度>3 Kbp）的基因组。",
    "Increases run time and memory requirements": "增加运行时间和内存需求",
    "Decreases run time": "减少运行时间",
    "Decreases memory requirements": "减少内存需求",
    "Optional, set to 0 to ignore, but must provide distance away in genes, bp or both": "可选，设置为0以忽略，但必须提供基因中的距离，基对或两者兼备",
    "Optional, set to 0 to ignore, but must provide distance away in bp, genes or both": "可选，设置为0以忽略，但必须提供距离（以碱基对、基因或两者）",
    "The files can be generated by Kraken, Bracken or Krona": "这些文件可以由Kraken、Bracken或Krona生成",
    "The file can be created by Bracken": "该文件可以通过Bracken创建",
    "Include non-standard levels. Default only outputs standard levels [D,P,C,O,F,G,S].": "包含非标准水平。默认情况下，只输出标准水平 [D,P,C,O,F,G,S]。",
    "Outputs percentages of reads instead of total reads.": "输出读取的百分比，而不是总读取数。",
    "FASTQ or FASTQ input reads (may be gzipped)": "FASTQ或FASTQ输入读取（可能是gzip压缩的）",
    "FASTQ or FASTQ input reads (for paired reads, may be gzipped)": "FASTQ或FASTQ输入读取（对于配对读取，可能是gzip压缩的）",
    "FASTQ or FASTA read pair collection (may be gzipped)": "FASTQ或FASTA读段集合（可能是gzip压缩的）",
    "Results (classification) file from Kraken/KrakenUniq/Kraken2": "Kraken/KrakenUniq/Kraken2的分类结果文件",
    "Report file from Kraken/KrakenUniq/Kraken2": "来自Kraken/KrakenUniq/Kraken2的报告文件",
    "Space-delimited list of taxonomic IDs for which to extract matching reads": "以空格分隔的分类学ID列表，用于提取匹配的读取",
    "Maximum number of reads to save for each ID": "每个ID保存的最大读取数",
    "Instead of finding reads that match given taxonomic IDs, find all reads that DO NOT match given IDs": "不是寻找匹配给定分类学ID的读取，而是寻找所有不匹配给定ID的读取。",
    "Write output as FASTQ instead of the default FASTA": "将输出格式写为FASTQ，而不是默认的FASTA",
    "Include reads classified at parent levels of the specified tax IDs": "包括在指定的分类ID的父级分类中被分类的读取",
    "Include reads classified more specifically than the specified tax IDs": "包含被分类为比指定的税号更具体的读段",
    "The MetaPhlAn output file contains relative abundance of clades at different taxonomic levels": "MetaPhlAn输出文件包含不同分类水平的类群相对丰度",
    "The HUMAnN output file contains relative abundance of gene families or pathways with corresponding taxonomic stratification": "HUMAnN输出文件包含基因家族或途径的相对丰度，并对应于分类学分层",
    "Creates a tab-separated OTU table (including taxonomy classification) that can be used to create BIOM files": "创建一个制表符分隔的操作单元(OTU)表格（包括分类学分类），该表格可用于创建Biom文件。",
    "Generates a reads per taxon file suitable for the use with Krona": "生成一个适合用于Krona的每类群读取文件",
    "The OTU table produced for the taxonomies found in the reference databases that was used with MAPseq": "MAPseq 用于参考数据库中发现的分类群生成的 OTU 表",
    "Generates an output file suitable for the use with Krona": "生成一个适合用于Krona的输出文件",
    "Number of reference sequences to include in alignment phase": "在比对阶段包含的参考序列数量",
    "Number of internal reference otus to include in alignment phase": "在对齐阶段包含的内部参考操作单元数",
    "Minimum score cutoff to consider for a classification, should be reduced when searching very small sequences, i.e.: primer search": "用于分类的最低分数截止值，当搜索非常小的序列时，例如引物搜索，应该降低。",
    "Minimum number of shared kmers to consider hit in second phase kmer search": "在第二阶段的kmer搜索中，考虑命中的最小共享kmer数量",
    "Minimum number of shared kmers to consider hit in alignment phase": "在比对阶段考虑命中的最小共享k-mer数量",
    "The `confidences` format outputs confidence values for each of the taxonomic levels.": "`confidences`格式输出每个分类水平的置信值。",
    "Rows are samples; columns are species/phyla/community classifier": "行代表样本；列代表物种/门/群落分类器",
    "Species, phylum, etc": "物种，门等",
    "Leave blank for all": "留空表示全部",
    "For joined Paired-end reads choose Single.": "对于连接的配对末端读取，选择“单个”。",
    "-a; Tool Author recommend default of 2, but this would require the chimera itself to need 2 copies": "-a; 工具作者推荐默认值为2，但这将要求Chimera本身需要2个副本",
    "-m; specifies whether the lengths of two reads should be exactly the same to be considered as duplicates.": "-m; 指定两个读取的长度是否应该完全相同才能被视为重复。",
    "-d; 0 means truncate at the first whitespace character": "-d; 0 表示在第一个空白字符处截断",
    "Use this option if you want to add an extra column at the end of the table for the sequence itself": "如果您想要在表格末尾为序列本身添加一个额外的列，请使用此选项。",
    "Use this option if you want to remove column containing the sequence definition in the output tab file": "如果您希望在输出的制表符分隔的表格文件中移除包含序列定义的列，请使用此选项。",
    "This key must be encoded in your FASTA/FASTQ headers as key=value. See help (below) for more information.": "这个键必须以key=value的形式编码在您的FASTA/FASTQ头信息中。有关更多信息，请参见帮助（下方）。",
    "database used for the in silico PCR. The database must be in the ecoPCR format (for example output of obiconvert)": "用于在 silico PCR 的数据库。该数据库必须为 ecoPCR 格式（例如 obiconvert 的输出）",
    "Use this option if you want to generate an ecoPCR database output file": "如果您想要生成一个ecoPCR数据库输出文件，请使用这个选项",
    "The attribute keys defined for each sequence record can be used in the expression as variable names. An extra variable named sequenceefers to the sequence record itself.": "为每个序列记录定义的属性键可以在表达式中用作变量名。一个名为sequence的额外变量指的是序列记录本身。",
    "Keeps sequence records whose sequence length is equal or shorter than lmax": "保留序列长度等于或小于lmax的序列记录",
    "Keeps sequence records whose sequence length is equal or longer than lmin": "保留序列长度等于或大于lmin的序列记录",
    "Use this option if you want to add a new attribute named seq_rank to the sequence record indicating its entry number in the sequence file": "如果您想要在序列记录中添加一个名为seq_rank的新属性，以指示其在序列文件中的条目编号，请使用此选项。",
    "Use this option if you want to delete attribute named ATTRIBUTE_NAME.When this attribute is missing, the sequence record is skipped and the next one is examined": "如果希望删除名为ATTRIBUTE_NAME的属性，请使用此选项。当缺少此属性时，将跳过序列记录并检查下一个记录。",
    "file containing identifiers of sequences to select": "包含要选定序列标识符的文件",
    "Use this option if you want to set sequence record identifier with a value computed from PYTHON_EXPRESSION": "如果您想要根据PYTHON_EXPRESSION计算的值来设置序列记录标识符，请使用此选项。",
    "Use this option if you want to run a PYTHON_EXPRESSION on each selected sequence": "如果您希望对每个选定的序列运行PYTHON_EXPRESSION，请使用此选项",
    "Use this option if you want to change the sequence itself with a value computed from PYTHON_EXPRESSION": "如果你想要通过计算PYTHON_EXPRESSION的值来改变序列本身，请使用这个选项。",
    "Use this option if you want to set sequence definition with a value computed from PYTHON_EXPRESSION": "如果您想要通过PYTHON_EXPRESSION计算得到的值来设置序列定义，请使用此选项",
    "Use this option if you want to clear all attributes associated to the sequence records": "如果您想要清除与序列记录关联的所有属性，请使用此选项",
    "Use this option if you want to add attribute with seq_length as a key and sequence length as a value": "如果您想要添加一个以 seq_length 为键，序列长度为值的属性，请使用这个选项。",
    "Use this option if you want to add taxonomic annotation at taxonomic rank RANK_NAME": "如果您想要在分类等级 RANK_NAME 添加分类注释，请使用这个选项",
    "use this option if you want to add a new attribute containing the number of the cluster the sequence record was assigned to, as indicated in file MCLFILE": "如果希望在新属性中添加序列记录被分配到的聚类编号，并且该编号如文件 MCLFILE 中所示，则使用此选项。",
    "Use this option if you want to force sequence record ids to be unique": "如果需要强制序列记录ID唯一，请使用此选项",
    "extrafile describing the DNA tags and primers sequences used for each sample": "额外文件描述了每个样本所使用的DNA标签和引物序列",
    "file of Illumina pair-end reads assembled by illuminapairedend": "由Illumina pair-end软件组装的Illumina成对末端读取文件",
    "file of 3p (1:) Illumina pair-end reads to assemble in sanger fastq nucleic format (standard fastq)": "3端（1:）Illumina成对末端读取文件，用于在Sanger FASTQ核酸格式（标准FASTQ）中组装",
    "file of 5p (2:) Illumina pair-end reads to assemble in sanger fastq nucleic format (standard fastq)": "5'端（2:）Illumina成对末端读取文件，用于在Sanger FASTQ核酸格式（标准FASTQ）中进行组装。",
    "Input a data collection.": "输入一个数据集合。",
    "You can upload dataset collection or upload templates one by one (especially if there are multiple format).": "您可以上传数据集集合或逐个上传模板（特别是如果有多种格式）。",
    "Insert only metadata using EML standard": "仅使用EML标准插入元数据",
    "Input a data collection": "输入一个数据集",
    "Lineagespot detects lineage evidence based on lineage mutation patterns defined in tool-specific definition files. The Galaxy tool also has experimental support for reading lineage definitions from cached constellations data (see tool help below).": "Lineagespot 根据工具特定的定义文件中定义的谱系突变模式检测谱系证据。Galaxy 工具还具有从缓存的星座数据中读取谱系定义的实验性支持（请参阅下面的工具帮助）。",
    "Select built-in genome file to use built-in annotations for the SARS-CoV-2 reference sequence NC_045512.2.": "选择内置的基因组文件，以便为SARS-CoV-2参考序列NC_045512.2使用内置注释。",
    "GFF3 input listing gene positions on the reference sequence": "GFF3输入列出参考序列上的基因位置",
    "Variants in the input VCFs with an allele frequency less than the threshold will be ignored as noise in the lineage analysis.": "在谱系分析中，输入的 VCF 文件中等位基因频率低于阈值的变异将被视为噪声并被忽略。",
    "Please note that you will HAVE to specify the reference explicitly via its ID if you choose the 'recombinant mutations colouring' scheme and in this case the reference can NOT be one of the recombinant parents.": "请注意，如果您选择“重组突变着色”方案，您将必须通过其ID明确指定参考序列，而在这种情况下，参考序列不能是重组亲本之一。",
    "A range (closed, inclusive; one-indexed) or specific position to include in the output, e.g. '100-150' or '100'": "一个范围（闭区间，包含端点；从1开始计数）或特定位置，以包含在输出中，例如'100-150'或'100'",
    "A range (closed, inclusive; one-indexed) or specific position to exclude from the output, e.g. '100-150' or '100'; considered after any '--include-positions'.": "一个范围（闭区间，包含；从1开始计数；例如'100-150'或'100'；在任何'--include-positions'之后考虑）或特定位置以从输出中排除。",
    "Exclude positions with an ambiguous base in any sequences; considered after '--include-positions'.": "排除任何序列中具有模糊碱基的位置；在'--include-positions'之后考虑。",
    "This expects a CSV dataset with two columns named 'name' and 'label' where the values in the 'name' column must have matches for all sequence IDs (including the reference) in the alignment input.": "这期望一个CSV数据集，其中包含两列，分别命名为'name'和'label'，其中'name'列中的值必须与比对输入中的所有序列ID（包括参考序列）匹配。",
    "Select a CSV dataset with an arbitrary number of named columns here and specify below which column contains the sequence IDs to be mapped and which one the labels to be used instead. The dataset must define mappings for all sequence IDs including the reference.": "选择一个CSV数据集，该数据集具有任意数量的命名列，并在下面指定哪个列包含要映射的序列ID，以及哪个列包含要使用的标签。数据集必须为所有序列ID（包括参考）定义映射。",
    "Mutations can be coloured by base change or, alternatively, by their presence/absence in any of two parent sequences in the alignment (recombinant mode).": "突变可以按照碱基变化进行上色，或者，也可以根据它们在比对中的两个亲本序列中的存在/缺失（重组模式）来进行上色。",
    "If selected sort sequences from highest to lowest number of mutations instead of from lowest to highest.": "如果选定，则将序列按照突变数量从高到低排序，而不是从低到高排序。",
    "Flip the orientation of the plot so sequences are below the reference rather than above it.": "将图表的方向翻转，使序列位于参考序列下方而不是上方。",
    "Set to 0 to keep default figure width.": "设置为0以保持默认图形宽度。",
    "Set to 0 to keep default figure height.": "设置为0以保持默认图形高度。",
    "This is newly introduced functionality for which you may encounter bugs still.": "这是新引入的功能，您可能会遇到一些错误。",
    "Produces extra CSV output with lists of SNPs per sequence in the alignment if enabled.": "如果启用，会在对齐的每个序列中生成包含SNP列表的额外CSV输出。",
    "default: 100": "默认：100",
    "To enforce specificity of detection, varVAMP will refuse to work if you set this value higher than for the amplicon primers above, and you may actually want to set it slightly lower than that value.": "为了确保检测的特异性，如果将此值设置为高于上述扩增引物的值，varVAMP将拒绝工作，实际上您可能希望将其设置为略低于该值。",
    "test the top n qPCR amplicons for secondary structures at the minimal primer temperature. (default: 50)": "测试前n个qPCR扩增子在最小引物温度下的二级结构。（默认值：50）",
    "Minimum free energy (kcal/mol/K) cutoff at the lowest primer melting temperature. (default: -3.": "最低引物熔解温度的最小自由能（千卡/摩尔/开尔文）截止值。（默认值：-3）",
    "With varVAMP default settings, the probe melting temperatures are 7°C higher than those of amplicon primers.": "使用varVAMP的默认设置，探针熔解温度比扩增子引物高7°C。",
    "Maximal overlap in bases between the ends of the qPCR probe and the primer on the opposite strand": "qPCR探针末端与相对链上引物之间的最大碱基重叠",
    "qPCR amplicon candidates are not considered further if they span regions of the alignment that contain deletions as long as this cutoff in bases, or longer.": "如果qPCR扩增子候选区域跨越比这个碱基长度阈值更长的缺失区域，则不再考虑这些候选区域。",
    "Aligned reads, to trim primers and quality": "对齐的读取，修剪引物和质量",
    "Same meaning as the \"Minimum frequency threshold\" above, but applied to indels.": "与上面“最小频率阈值”相同的含义，但应用于插入/缺失（indels）。",
    "BAM dataset, preprocessed with ivar trim, to remove reads from": "BAM数据集，使用ivar trim预处理，以去除读取",
    "This dataset will be scanned for variants that affect primer binding sites and needs to be in tabular format with affected chromosome names in the first, and positions in the second column. If there is a header line, the name of the second column should be POS.": "这个数据集将被扫描以查找影响引物结合位点的变体，并且需要以表格格式呈现，第一列是受影响的染色体名称，第二列是位置。如果存在标题行，则第二列的名称应该是 POS。",
    "The same six-column BED dataset that served as input to ivar trim": "相同的六列BED数据集，作为ivar trim的输入",
    "For suitable primer binding site datasets amplicon info can be computed directly (see tool help below). For others you will need to provide an extra amplicon info dataset.": "对于合适的引物结合位点数据集，扩增子信息可以直接计算（请参阅下面的工具帮助）。对于其他数据集，您需要提供额外的扩增子信息数据集。",
    "This input should consist of one line per amplicon with the tab-separated names of all primers used to generate that amplicon.": "输入应包含每行一个扩增子，使用制表符分隔所有用于生成该扩增子的引物名称。",
    "Select primer scheme bed file from a list": "从列表中选择引物方案的bed文件",
    "When you select Yes, reads that are not fully contained in any amplicon will be dropped before primer trimming. This option is currently marked as [Experimental] in ivar, but nevertheless recommended here. Info on amplicons can be computed from suitable primer BED files (see tool help below) or provided by the user.": "当您选择“是”时，不在任何扩增子中完全包含的读取将在引物修剪之前被丢弃。这个选项在ivar中当前被标记为[实验性]，但在这里仍然推荐使用。扩增子的信息可以从合适的引物BED文件中计算得出（见下面的工具帮助），或者由用户提供。",
    "Reads that occur at the specified offset positions relative to primer positions (as annotated in the primer information dataset) will also be trimmed (default: 0)": "在引物位置（如引物信息数据集中注释的）的指定偏移位置上出现的读取也将被修剪（默认值：0）",
    "The default automatic setting will determine the threshold as 50% of the mean length of the first 1000 raw input reads, which may or may not be adequate for your data. You can always make the outcome of filtering more predictable by specifying the threshold explicitly. You can also opt to retain reads independently of their trimmed length, but typically this only makes sense if you are performing additional read filtering with additional tools in your analysis pipeline.": "默认的自动设置将阈值确定为第一个1000个原始输入读取的平均长度的50%，这可能或可能不适用于您的数据。您可以通过明确指定阈值来使过滤结果更加可预测。您也可以选择独立于其修剪长度保留读取，但通常这只有在您在分析流程中使用其他工具进行额外的读取过滤时才有意义。",
    "Reads with a trimmed length shorter than this value will be discarded.": "读长修剪后短于该值的序列将会被丢弃。",
    "Provide a GFF3 dataset with annotated ORFs to have the rightmost 5 columns of the tabular output populated with mutation information at the residue level. This has no effect on the VCF output.": "提供一个带有注释的ORF的GFF3数据集，以便在表格输出的最右侧5列中填充氨基酸水平上的突变信息。这对VCF输出没有影响。",
    "Provide a GFF3 dataset with annotated ORFs to have the rightmost 5 columns of the tabular output populated with mutation information at the residue level.": "提供一个带有注释的ORF的GFF3数据集，以便表格输出的最右边5列填满在残基水平上的突变信息。",
    "Remove all As at the end of the reference sequence to ensure that the reference has no poly-A tail (see Note in the general help section below)": "移除参考序列末尾的所有As以确保参考序列没有聚腺苷酸尾部（参见下方一般帮助部分的注释）",
    "After primer trimming and alignment to reference genome.": "在引物修剪和与参考基因组比对后。",
    "Note that the reference should match the fasta file used for alignment.": "请注意，参考序列应与用于比对的fasta文件相匹配。",
    "This can be a VCF dataset, or the tabular calls output of freayja call or ivar variants.": "这可以是VCF数据集，或者是freayja call或ivar variants的表格调用输出。",
    "Select autodetect to have the dataset or collection element name used as the sample name, or, for a single input dataset, provide an explicit sample name.": "选择“自动检测”以使用数据集或集合元素名称作为样本名称，或者对于单个输入数据集，提供一个明确的样本名称。",
    "In the result file the coverage value will provide the 10x coverage estimate (percent of sites with 10 or greater reads- 10 is the default but can be modfied in this field.": "在结果文件中，覆盖度值将提供10倍覆盖度估计（具有10个或更多读取位点的百分比 - 10是默认值，但可以在该字段中修改）。",
    "One file per one sample.": "每个样本一个文件。",
    "See info box below to learn more about the form of meta-file. Accepts only CSV (.csv) files": "请查看下面的信息框，以了解更多关于元文件格式的信息。只接受CSV（.csv）文件。",
    "Used for pdf format.": "用于PDF格式。",
    "Used for interactive dashboard.": "用于交互式仪表板。",
    "Used for pdf non-interactive plot.": "用于PDF非交互式绘图。",
    "BED dataset providing the positions of the amplicon inserts (i.e. excluding the primer sequences) on the reference sequence.": "提供扩增子插入位点（即不包括引物序列）在参考序列上位置的BED数据集。",
    "The tool will attempt to autodetect the relationship between primers and amplicons based on the primer names.": "该工具将尝试根据引物名称自动检测引物与扩增子之间的关系。",
    "This input should have one line of tab-separated primer names (as they appear in the primer scheme) per amplicon.": "这个输入应该有一个每行以制表符分隔的引物名称（就像它们在引物方案中出现的那样）的扩增子。",
    "The tool can generate a YAML-formatted report of expected co-occurrences of mutations per amplicon and lineage. This output combines amplicon scheme and lineage definitions and can be used for rerunning the tool with identical configuration in 'Combined' mode. It is also a recommended input to the cojac pubmut tool so leave this option checked if you are intending to use this downstream tool.": "该工具可以生成一个YAML格式的报告，报告中包含了每个扩增子和谱系中预期的突变共现情况。此输出结合了扩增子方案和谱系定义，并且可以使用相同的配置在“组合”模式下重新运行该工具。它也是cojac pubmut工具的推荐输入，因此，如果您打算使用此下游工具，请确保此选项已勾选。",
    "YAML-formatted report of expected co-occurrences of mutations per amplicon and lineage generated by a previous run of the tool.": "由工具先前运行生成的每个扩增子和谱系中预期突变共现的YAML格式报告。",
    "Please note that only yaml- and json-formatted results can be processed with the downstream pubmut and tabmut tools.": "请注意，只有yaml格式和json格式的结果才能被下游的pubmut和tabmut工具处理。",
    "The tool can work with json- or yaml-formatted output of cojac mutbamscan.": "该工具可以处理 cojac mutbamscan 的 json 或 yaml 格式输出。",
    "For best readability of its report the tool requires a per-lineage amplicon info dataset (in yaml format) produced by the mutbamscan tool and the original lineage definitions. If all you have is the per-lineage amplicon info, you can use it to obtain a nearly identical report, but cojac-internal lineage identifiers won't be translated into standard lineage names. With no available annotations a limited report can still be generated, but this is not recommended.": "为了获得最佳可读性的报告，该工具需要一个由mutbamscan工具产生的每株系扩增子信息数据集（以yaml格式），以及原始的株系定义。如果您拥有的只是每株系扩增子信息，您可以使用它来获得几乎相同的报告，但是cojac内部株系标识符将不会被翻译成标准株系名称。如果没有可用的注释，仍然可以生成一个有限的报告，但这是不推荐的。",
    "If you provide a second file, then difference is calculated (GRAPH - second GRAPH)": "如果您提供了第二个文件，那么将计算差异（图形 - 第二个图形）",
    "For each group a fasta file is generated containing all proteins/genes of that group.": "对于每个组，都会生成一个包含该组所有蛋白质/基因的fasta文件。",
    "If not: the string is escaped (e.g. | -> \\|) [-E]": "如果不使用：字符串会被转义（例如 | -> \\|）[-E]",
    "The input fasta files. At least 2 are needed!": "输入的fasta文件。至少需要2个！",
    "In the first step of proteinortho an all-versus-all reciprocal best hit graph is build from the input files (using this algorithm).": "在Proteinortho的第一步中，会根据输入文件（使用此算法）构建一个全对全的互惠最佳命中图。",
    "This and --evalue are main parameters for the generation of the reciprocal best hit graph. 1 = only the best reciprocal hits are reported, 0 = all possible reciprocal blast matches (within the E-value cutoff) are reported.": "这个和--evalue是生成互惠最佳命中图的主要参数。1=只报告最佳互惠命中，0=报告所有可能的互惠BLAST匹配（在E值截止范围内）。",
    "This is the main parameter for the clustering step. Choose larger values than more splits are done, resulting in more and smaller clusters. A value of 0 corresponds to no clustering.": "这是聚类步骤的主要参数。选择比分裂次数更多的较大值，将产生更多且更小的聚类。值为0表示不进行聚类。",
    "Larger values results in more false positives (connections between proteins).": "较大的值会导致更多的假阳性（蛋白质之间的连接）。",
    "Overrules the -conn threshold.": "覆盖 -conn 阈值。",
    "The reciprocal best hit graph is built using isoform information (isoforms are treated equivalent). For ncbi : simply add the additional files to the input (file names need to match). For Uniprot : the isoforms need to contain the word isoform and the corresponding identifier. For trinity simply use the trinity output format.": "互为最佳命中图是使用异构体信息构建的（异构体被视为等同）。对于NCBI：只需将额外的文件添加到输入中（文件名需要匹配）。对于Uniprot：异构体需要包含单词“isoform”和相应的标识符。对于Trinity：只需使用Trinity输出格式。",
    "To enhance the prediction accuracy, the relative order of genes (synteny) can be used as an additional feature for the discrimination of orthologs. For more details see doi:10.1371/journal.pone.0105015.": "为了提高预测的准确性，基因（共线性）的相对顺序可以作为区分同源基因的额外特征。更多详细信息请参见doi:10.1371/journal.pone.0105015。",
    "alpha[FF-adj score] + (1−alpha)[BLAST score]": "α[FF-adj 分数] + (1−α)[BLAST 分数]",
    "The GFF3 files need matching names with the input fasta files. If you provide mybacteria123.faa or mybacteria123.fasta ... then you need to provide mybacteria123.gff here accordingly. The attributes column (#9) must contain the attribute Name=GENE IDENTIFIER where GENE IDENTIFIER corresponds to the respective (protein) identifier in the FASTA input. For example see https://gitlab.com/paulklemm_PHD/proteinortho/-/blob/master/test/C.gff": "GFF3文件需要与输入的fasta文件具有匹配的名称。如果您提供了mybacteria123.faa或mybacteria123.fasta等文件，那么您需要相应地在这里提供mybacteria123.gff。属性列（第9列）必须包含属性Name=GENE IDENTIFIER，其中GENE IDENTIFIER对应于FASTA输入中的相应（蛋白质）标识符。例如，请参阅https://gitlab.com/paulklemm_PHD/proteinortho/-/blob/master/test/C.gff",
    "Specify a run to be analysed": "指定一个要分析的运行",
    "Instructs DIA-NN to generate a spectral library": "指导DIA-NN生成一个谱图库",
    "Instructs DIA-NN to perform deep learning-based prediction of spectra, retention times and ion mobility values": "指导DIA-NN执行基于深度学习的谱图、保留时间和离子迁移率值的预测",
    "Specify a spectral library": "指定一个光谱库",
    "Specifies column names in the spectral library to be used, in the order described in Spectral library formats [name 1],[name 2],.... Use '*' instead of the column name if DIA-NN already recognizes its name": "指定在光谱库中使用的列名，按照光谱库格式中描述的顺序 [name 1]、[name 2]，... 如果 DIA-NN 已经识别出列名，则可以使用 '*' 代替列名",
    "The input library will be used 'as is' without discarding fragments that might be harmful for the analysis; use with caution": "输入的库将“原样”使用，不会丢弃可能对分析有害的片段；请谨慎使用",
    "Specifies a 'training library' for the legacy predictor": "指定一个用于旧版预测器的“训练库”",
    "Instructs DIA-NN to save raw empirical retention times in the spectral library being generated, instead of saving RTs aligned to a particular scale": "指导DIA-NN在生成的谱库中保存原始的经验保留时间，而不是保存与特定标度对齐的保留时间。",
    "Reannotate the spectral library with protein information from the FASTA database, using the specified digest specificity": "使用来自FASTA数据库的蛋白质信息重新注释谱图库，并使用指定的消化特异性。",
    "Specify a sequence database in FASTA format": "指定一个FASTA格式的序列数据库",
    "Only consider peptides matching the stripped sequences specified in the text file provided, when processing a sequence database": "仅在处理序列数据库时，考虑与提供的文本文件中指定的剥离序列匹配的肽段",
    "Instructs DIA-NN to perform an in silico digest of the sequence database": "指导DIA-NN对序列数据库进行in silico消化",
    "Disables mass calibration": "禁用质量校准",
    "Sets the MS2 mass accuracy to N ppm": "设置MS2 质量精度为 N ppm",
    "Sets the mass accuracy used during the calibration phase of the search to N ppm": "设置在搜索校准阶段使用的质谱精度为N ppm",
    "Sets the MS1 mass accuracy to N ppm": "设置MS1 质量精度为 N ppm",
    "Enables MBR": "启用MBR",
    "When using the 'Unrelated runs' option in combination with MBR, the same settings will be used to process all runs during the second MBR pass": "在使用“不相关的运行”选项与MBR（多区域对齐）组合时，将在第二次MBR通过期间使用相同的设置来处理所有运行。",
    "Instructs DIA-NN to use a very heuristical protein inference algorithm (similar to the one used by FragPipe and many other software tools), wherein DIA-NN aims to make sure that no protein is present simultaneously in multiple protein groups. This mode (i) is recommended for method optimisation and benchmarks, (ii) might be convenient for gene set enrichment analysis and related kinds of downstream processing. However the alternative protein inference strategy of DIA-NN is more reliable for differential expression analyses (this is one of the advantages of DIA-NN). Equivalent to the 'Heuristic protein inference' GUI setting.": "指导DIA-NN使用一种非常启发式的蛋白质推断算法（类似于FragPipe和许多其他软件工具所使用的算法），其中DIA-NN旨在确保没有蛋白质同时存在于多个蛋白质组中。这种模式（i）推荐用于方法优化和基准测试，（ii）可能适用于基因集富集分析和相关类型的下游处理。然而，DIA-NN的另一种蛋白质推断策略对于差异表达分析更为可靠（这是DIA-NN的优势之一）。等同于“启发式蛋白质推断”GUI设置。",
    "Controls the protein inference mode. If protein inference (that is protein grouping) is disabled, protein groups from the spectral library will be used instead": "控制蛋白质推断模式。如果禁用蛋白质推断（即蛋白质分组），则将使用来自谱库的蛋白质组",
    "Activates the neural network classifier. Default is single-pass mode. Double-pass mode is the best in most cases but is about twice slower than single-pass": "激活神经网络分类器。默认是单次传递模式。双次传递模式在大多数情况下是最好的，但比单次传递模式慢大约两倍。",
    "Quantification algorithms. --no-ifs-removal/Any LC (high precision): Turns off interference subtraction from fragment ion chromatograms. --peak-center/Robust LC (high accuracy): Instructs DIA-NN to integrate chromatographic peaks only in the vicinity of the apex. Robust LC (high precision): combination of --peak-center and --no-ifs-removal. --peak-height/Peak-height Instructs DIA-NN to use the apex height of the peak for quantification - equivalent to the 'Peak height' quantification mode.": "定量算法。--no-ifs-removal/任何LC（高精度）：关闭从片段离子色谱图中减去干扰的功能。--peak-center/鲁棒LC（高准确度）：指示DIA-NN仅在峰顶附近积分色谱峰。鲁棒LC（高精度）：--peak-center和--no-ifs-removal的组合。--peak-height/峰高 指示DIA-NN使用峰的顶点高度进行定量 - 相当于“峰高”定量模式。",
    "Normalisation strategy. --global-norm/Global: Instructs DIA-NN to use simple global normalisation instead of RT-dependent normalisation. RT-dependent: Instructs DIA-NN to use RT-dependent normalisation. --sig-norm/RT and signal-dep. (experimental): Instructs DIA-NN to use RT and signal dependent normalisation (experimental). --no-norm/Off: disables cross-run normalisation. Normalised quantities reported along with the raw quantities. Default:  RT-dependent": "标准化策略。--global-norm/全局：指示DIA-NN使用简单的全局标准化，而不是依赖于保留时间（RT）的标准化。依赖于RT：指示DIA-NN使用依赖于保留时间的标准化。--sig-norm/RT和信号依赖（实验性）：指示DIA-NN使用依赖于保留时间和信号的标准化（实验性）。--no-norm/关闭：禁用跨运行的标准化。报告标准化量和原始量。默认：依赖于RT",
    "Enables an intelligent algorithm which determines how to extract spectra, when creating a spectral library from DIA data. This is highly recommended and should almost always be enabled": "启用一种智能算法，该算法确定如何从DIA数据中提取光谱，当从DIA数据创建光谱库时。这是非常推荐的，并且几乎应该始终启用。",
    "Disables IM-windowed search": "禁用IM窗口搜索",
    "Sets the scan window radius to a specific value. Ideally, should be approximately equal to the average number of data points per peak": "将扫描窗口半径设置为一个特定值。理想情况下，应大致等于每个峰值的数据点平均数",
    "Fixes IM extraction window to the specific value": "将IM提取窗口固定为特定值",
    "Controls the minimum size of the IM extraction window. Default is 2.0": "控制IM提取窗口的最小尺寸，默认为2.0。",
    "Specifies cleavage specificity for the in silico digest. Cleavage sites (pairs of amino acids) are listed separated by commas, '*' indicates any amino acid, and '!' indicates that the respective site will not be cleaved. Examples: K*,R*,!*P - canonical tryptic specificity, leave blank to disable digest": "指定了在计算机模拟消化中的切割特异性。切割位点（氨基酸对）以逗号分隔，'*' 表示任意氨基酸，'!' 表示相应的位点不会被切割。示例：K*,R*,!*P - 典型的胰蛋白酶特异性，留空以禁用消化",
    "Cleavage sites (pairs of amino acids) are listed separated by commas, '*' indicates any amino acid, and '!' indicates that the respective site will not be cleaved. For example: K*,R*,!*P for canonical tryptic specificity.": "切割位点（氨基酸对）以逗号分隔，'*'表示任何氨基酸，'!'表示相应的位点不会被切割。例如：K*,R*,!*P 表示典型的胰蛋白酶特异性。",
    "Sets the maximum number of missed cleavages": "设置最大允许的酶切位点错切数",
    "Enables protein N-term methionine excision as variable modification for the in silico digest": "允许将蛋白质N端甲硫氨酸切除作为可变修饰用于计算机模拟消化",
    "Sets the minimum precursor length for the in silico library generation or library-free search": "设置在silico库生成或无库搜索中前体的最小长度",
    "Sets the maximum precursor length for the in silico library generation or library-free search": "设置在 silico 库生成或无库搜索中的最大前体长度",
    "Sets the minimum precursor m/z for the in silico library generation or library-free search": "设置在 silico 库生成或无库搜索中的最小前体 m/z",
    "Sets the maximum precursor m/z for the in silico library generation or library-free search": "设置生成in silico库或无库搜索时的最大前体m/z值",
    "Sets the minimum precursor charge for the in silico library generation or library-free search": "设置在 silico 库生成或无库搜索中的最小前体电荷",
    "Sets the maximum precursor charge for the in silico library generation or library-free search": "设置生成in silico库或无库搜索时的最大前体电荷",
    "Specifies the minimum number of fragments per precursors in the spectral library being saved": "指定在保存的谱库中每个前体的最小片段数",
    "Specifies the maximum number of fragments per precursor in the spectral library being saved": "指定在保存的谱库中每个前体的最大片段数",
    "Only use one (best) precursor per stripped sequence for the training of the neural network classifier": "仅使用一个（最佳）前体序列来训练神经网络分类器",
    "Specify 0 to disable the removal of interfering precursors": "指定0以禁用干扰前体的移除",
    "Adds the modification name to the list of recognised names and specifies the modification as fixed [name],[mass],[sites],[optional: 'label']": "将修饰名称添加到已识别名称列表中，并指定修饰为固定修饰 [name],[mass],[sites],[可选：'label']",
    "Adds the modification name to the list of recognised names and specifies the modification as variable [name],[mass],[sites],[optional: 'label']. [sites] can contain a list of amino acids and 'n' which codes for the N-terminus of the peptide. '*n' indicates protein N-terminus. Similar to --mod can be followed by 'label' Examples: UniMod:21,79.966331,STY - phosphorylation, UniMod:1,42.010565,*n - N-terminal protein acetylation.": "将修饰名称添加到已识别名称的列表中，并指定修饰为可变型[name],[mass],[sites],[可选：'label']。[sites]可以包含氨基酸列表和'n'，后者表示肽的N-末端。'*n'表示蛋白质N-末端。类似于--mod，后面可以跟'label'。示例：UniMod:21,79.966331,STY - 磷酸化，UniMod:1,42.010565,*n - N端蛋白质乙酰化。",
    "Sets the maximum number of variable modifications": "设置可变修饰的最大数量",
    "Declares a modification name [name],[mass],[optional: 'label']. Examples: UniMod:5,43.005814, SILAC-Lys8,8.014199,label": "声明一个修饰名称[name]，[mass]，可选：'label'。示例：UniMod:5,43.005814,SILAC-Lys8,8.014199,label",
    "Apply PTM scoring and site localisation for a particular modification": "应用特定修饰的PTM评分和位点定位",
    "Discard peptides generated via in silico cuts after residues bearing a particular modification. Example: SILAC-Lys8": "丢弃通过特定修饰残基后进行的in silico切割生成的肽段。例如：SILAC-Lys8",
    "Disables the automatic conversion of known modifications to the UniMod format names": "禁用将已知修饰自动转换为UniMod格式名称",
    "Makes DIA-NN 'forget' all built-in modification (PTM) names": "使DIA-NN“忘记”所有内置的修饰（PTM）名称",
    "Applies a modification, previously declared using --fixed-mod, to a spectral library": "将之前使用 --fixed-mod 声明的修饰应用于谱库",
    "Only consider peptides bearing the modifications listed with --monitor-mod": "只考虑带有--monitor-mod参数列出的修饰的肽段。",
    "Instructs DIA-NN to ignore modifications that are not supported by the deep learning predictor, when performing the prediction": "指示 DIA-NN 在执行预测时忽略深度学习预测器不支持的修饰",
    "The selection of fragments for quantification based on the quality assessment of the respective extracted chromatograms will be disabled": "基于各自提取的色谱图的质量评估来选择片段进行定量的功能将被禁用",
    "Sets the number of top fragment ions among which the fragments that will be used for quantification are chosen. Default value is 6": "设置用于定量的片段离子中选择的顶部片段离子的数量。默认值是6。",
    "Some fragments will not be used for quantification, based on the value in the ExcludeFromAssay spectral library column": "一些片段将不会用于定量，这基于“排除自实验的光谱库列”中的值。",
    "Sets the fragment filtering mass accuracy (in ppm) when reading .sptxt/.msp libraries": "设置在读取.sptxt/.msp库时的片段筛选质量精度（以ppm为单位）",
    "Fragment ions beyond this number will only be included in the spectral library being created (from DIA data) if they have high-quality chromatograms. Default value is 6": "超过此数字的片段离子只有在它们具有高质量色谱图的情况下才会被包含在正在创建的光谱库中（来自DIA数据）。默认值是6。",
    "Specifies the decoy channel masses (same syntax as for --channels)": "指定诱饵通道的质量（与 --channels 参数的语法相同）",
    "Disables the use of a decoy channel for channel q-value calculation": "禁用用于通道q值计算的诱饵通道",
    "Instructs DIA-NN to take advantage of the co-elution of isotopologues, when identifying and quantifying precursors": "指导DIA-NN利用同位素分子的共洗脱特性，在鉴定和定量前体时",
    "Disables MaxLFQ for protein quantification": "禁用MaxLFQ用于蛋白质定量",
    "Sets the minimum peak height to consider": "设置要考虑的最小峰值高度",
    "Disables RT-windowed search": "禁用 RT-windowed 搜索",
    "Disables precursor stratification based on the modification status. Stratification works in combination with monitor-mod to ensure that no matter how few modified peptides are reported, the FDR specifically across modified peptides is well controlled.": "禁用基于修饰状态的前体分层。分层与monitor-mod结合使用，以确保无论报告的修饰肽段有多少，修饰肽段的FDR都得到了很好的控制。",
    "Instruct DIA-NN not to give preference for SwissProt proteins when inferring protein groups": "指示 DIA-NN 在推断蛋白质组时不要优先考虑 SwissProt 蛋白质",
    "When using the deep learning predictor, prediction of ion mobilities will not be performed": "在使用深度学习预测器时，不会进行离子迁移率的预测",
    "When using the deep learning predictor, prediction of retention times will not be performed": "在使用深度学习预测器时，不会进行保留时间的预测",
    "Instructs DIA-NN not to skip entries in the sequence database with duplicate IDs (while by default if several entries have the same protein ID, all but the first entry will be skipped)": "指示 DIA-NN 在序列数据库中不要跳过具有重复 ID 的条目（默认情况下，如果几个条目具有相同的蛋白质 ID，则除了第一个条目之外的所有条目都将被跳过）",
    "Approximate FDR estimation for confident peptides based on parametric modelling will be disabled": "基于参数化建模的可信肽段的近似FDR估计将被禁用",
    "Only consider SwissProt sequences when processing a sequence database": "在处理序列数据库时，只考虑SwissProt序列",
    "Loads the complete UniMod modification database and disables the automatic conversion of modification names to the UniMod format": "加载完整的UniMod修饰数据库，并禁用将修饰名称自动转换为UniMod格式的功能",
    "Annotates the library with fragment exclusion information, based on the runs being analysed (fragments least affected by interferences are selected for quantification, why the rest are excluded)": "为文库注释片段排除信息，基于正在分析的运行（选择最少受干扰影响的片段进行定量，其余的被排除）",
    "Disables RT-dependent mass calibration": "禁用依赖于保留时间的质荷比校准",
    "Mass accuracies, if set to automatic, will be determined independently for different runs": "如果设置为自动，则不同运行的精确质量将独立确定",
    "A separate output report will be created for each run": "每次运行将创建一个单独的输出报告",
    "Scan window, if set to automatic, will be determined independently for different runs": "扫描窗口，如果设置为自动，则会独立地为不同的运行确定",
    "All runs will be analysed as if they were not Scanning SWATH runs": "所有运行都将被分析，就好像它们不是扫描SWATH运行一样。",
    "All runs will be analysed as if they were Scanning SWATH runs": "所有运行都将被分析，就好像它们是扫描SWATH运行一样。",
    "Instructs DIA-NN to add the organism identifier to the gene names - useful for distinguishing genes from different species, when analysing mixed samples. Works with UniProt sequence databases.": "指示DIA-NN将生物体标识符添加到基因名称中 - 在分析混合样本时，这对于区分不同物种的基因非常有用。与UniProt序列数据库兼容。",
    "DIA-NN will ignore errors when loading dia-PASEF data": "DIA-NN 在加载 dia-PASEF 数据时会忽略错误",
    "Disables the generation of the stats file": "禁用生成统计文件",
    "Instructs DIA-NN to provide less information in the main report": "指示DIA-NN在主报告中提供较少的信息",
    "Outputs quantities matrices": "输出数量矩阵",
    "Sets the 'channel q-value' used to filter the output matrices": "设置用于过滤输出矩阵的“通道q值”",
    "Sets the q-value used to filter the output matrices": "设置用于过滤输出矩阵的q值",
    "Sets the 'translated q-value' used to filter the output matrices": "设置用于过滤输出矩阵的“翻译后的q值”",
    "Run-specific protein q-value filtering will be used when saving protein matrices. The ability to filter based on run-specific protein q-values, which allows to generate highly reliable data, is one of the advantages of DIA-NN.": "在保存蛋白质矩阵时，将使用特定于运行的蛋白质q值过滤。基于特定于运行的蛋白质q值进行过滤的能力，可以生成高度可靠的数据，这是DIA-NN的优势之一。",
    "Adds extra library information on the precursor and its fragments to the main output report": "在主输出报告中为前体及其片段添加额外的库信息",
    "Instructs DIA-NN to extract and save chromatograms in the vicinity of the detected elution apex, for all PSMs matching the stripped sequences provided, for all runs in the experiment; at least N scans in the vicinity of the apex will be extracted": "指导DIA-NN提取并保存检测到的洗脱峰顶附近的色谱图，适用于实验中所有匹配剥离序列的PSMs；至少提取峰顶附近N个扫描。",
    "Sets the level of detail of the log": "设置日志的详细程度",
    "Peptide sequence(s) which you want to search": "要搜索的肽段序列",
    "Protein fasta file containing proteins which you want to search.": "蛋白质fasta文件包含您想要搜索的蛋白质。",
    "Protein sequence which you want to search": "您要搜索的蛋白质序列",
    "Protein ID from the selected Protein Reference Database. E.g. sp|P07205|PGK2_HUMAN from swissprot:human": "从选定的蛋白质参考数据库中获取的蛋白质ID。例如，来自swissprot:human的sp|P07205|PGK2_HUMAN",
    "DNA sequence which you want to search": "您要搜索的DNA序列",
    "The frame(s) to translate DNA sequence to protein. Selecting nothing (default) keeps the longest frame": "将DNA序列翻译成蛋白质的框架（或多个框架）。不选择任何选项（默认）将保留最长的框架",
    "an input sequence that matches a reference will be ignored.": "匹配参考序列的输入序列将被忽略。",
    "Spectrum file used for identification, formats: MGF,mzML,mzXML,Thermo RAW": "用于鉴定的谱图文件，格式：MGF、mzML、mzXML、Thermo RAW",
    "Default: index": "默认：索引",
    "Max number of variable modifications  Default: 3": "最大变量修饰数量  默认值：3",
    "TRUE: score(UMS)>=score(targetPSM); FALSE: score(UMS)>score(targetPSM)": "真：UMS的分数大于等于目标PSM的分数；假：UMS的分数大于目标PSM的分数",
    "Whether or not to consider aa substitution modifications when perform modification filtering.": "是否在执行修饰筛选时考虑氨基酸替代修饰。",
    "Enzyme used for protein digestion.  Default: Trypsin": "用于蛋白质消化的酶。默认：胰蛋白酶",
    "The max missed cleavages": "最大未切割次数",
    "The error window on experimental peptide mass values. This parameter is usually set according to the mass spectrometer which was used to generate the MS/MS data.  Default: 10": "实验肽质量值的错误窗口。这个参数通常根据用于生成MS/MS数据的质谱仪进行设置。默认值：10",
    "The unit of precursor ion m/z tolerance. Default: ppm": "前体离子 m/z 容差单位。默认值：ppm",
    "Error window for MS/MS fragment ion mass values in Da unit.  Default: 0.6 Da": "MS/MS 片段离子质量误差窗口，单位为道尔顿。默认值：0.6 Da",
    "Default: CID/HCD": "默认：CID/HCD",
    "Default: HyperScore": "默认值：超得分",
    "use two scoring algorithms for peptide identification": "使用两种评分算法进行肽段鉴定",
    "The minimum charge to consider if the charge state is not available.  Default: 2": "如果电荷状态不可用，则考虑的最小电荷。默认值：2",
    "The maximum charge to consider if the charge state is not available.  Default: 3": "如果电荷状态不可用，则考虑的最大电荷。默认值：3",
    "Min peaks in spectrum. Default: 10": "在谱图中的最小峰值数。默认值：10",
    "Minimum score to consider for peptide searching.  Default: 12": "用于肽段搜索的最低分数。默认值：12",
    "The maximum length of peptide to consider.  Default: 7": "考虑的肽段最大长度。默认值：7",
    "The maximum length of peptide to consider.  Default: 45": "考虑的肽段最大长度。默认值：45",
    "The number of random peptides.  Default: 1000": "随机肽段的数量。默认值：1000",
    "In fast mode, only one better match from reference peptide-based competitive filtering steps will be returned. A peptide identified or not is not affected by this setting. For most applications, fast mode will speed up the analysis.": "在快速模式下，只会从基于参考肽的竞争性过滤步骤返回一个更好的匹配。肽段是否被鉴定不受此设置影响。对于大多数应用，快速模式将加快分析速度。",
    "Specify one or more FASTA databases.": "指定一个或多个FASTA数据库。",
    "Modify parse rules if needed.": "如果需要，修改解析规则。",
    "Peptides shorter than this value will not be reported nor be considered during protein identification and quantification short peptides are usually not unique in the protein database and therefore not statistically informative.": "肽段长度小于这个值将不会被报告，也不会在蛋白质鉴定和定量中被考虑，短肽段通常在蛋白质数据库中不是唯一的，因此统计上没有信息。",
    "Peptides that are heavier than this mass will be discarded in the Andromeda search.": "在Andromeda搜索中，比这个质量重的肽段将被丢弃。",
    "The minimum number of unique peptides a protein group should have to be considered as identified and reported in the final table.": "蛋白质组应该至少有多少个独特的肽段才能被认为是鉴定出来的，并在最终的表格中报告。",
    "If checked, several quantities characterizing peaks and isotopes patterns are calculated. This may lead to a substantial increase in computation time.": "如果选中，将计算几个表征峰和同位素模式的数量。这可能会导致计算时间大幅增加。",
    "Identifications are transferred to non-sequenced or non-identified MS features in other LC-MS runs.": "鉴定结果被转移到其他LC-MS运行中未测序或未鉴定的质谱特征上。",
    "The time window that is used in 'Match between runs' for the transfer of identifications.": "在“跨运行匹配”中用于传递鉴定结果的时间窗口。",
    "The ion mobility window that is used in 'Match between runs' for the transfer of identifications.": "在“跨运行匹配”中用于传递鉴定的离子迁移窗口。",
    "The time window that is used in retention time alignment to search for the best alignment function.": "在保留时间对齐中使用的窗口时间，用于搜索最佳对齐函数。",
    "The ion mobility window that is used in ion mobility alignment to search for the best alignment function.": "在离子迁移率对齐中用于搜索最佳对齐函数的离子迁移率窗口。",
    "Also unidentified peptide features will be matched between runs. Their expression profiles over multiple samples are written into the table called 'matchedFeatures.txt'.": "此外，未鉴定的肽段特征将在不同的运行之间进行匹配。它们在多个样本上的表达谱被写入一个名为'matchedFeatures.txt'的表格中。",
    "‘Yes’ means that proteins from a contaminant list are automatically added to the list of proteins for in silico digestion (in the fasta file)": "“是”表示污染列表中的蛋白质会自动添加到蛋白质的虚拟消化列表中（在fasta文件中）。",
    "The decoy sequences can be generated by reversing or randomizing the original protein sequences. Reversing is recommended.": "诱饵序列可以通过反转或随机化原始蛋白质序列来生成。建议使用反转方法。",
    "Specify the desired false discovery rate (FDR) at the peptide spectrum match (PSM) level. It is determined by the target-decoy approach. A value of 0.01 stands for 1% FDR.": "指定肽谱匹配（PSM）水平上所需的错误发现率（FDR）。这是通过目标-伪目标方法确定的。0.01的值代表1% FDR。",
    "Specify the desired false discovery rate (FDR) at the protein level. It is determined by the target-decoy approach. A value of 0.01 stands for 1% FDR.": "指定蛋白质水平上期望的假发现率（FDR）。它由目标-伪目标方法确定。0.01的值代表1% FDR。",
    "During unspecific searches peptides that have fewer amino acids than this value will be discarded. This applies also to semi-specific (e.g. semi tryptic) searches": "在非特异性搜索中，少于该值的氨基酸的肽将被丢弃。这也适用于半特异性搜索（例如半胰蛋白酶搜索）。",
    "During unspecific searches peptides that have more amino acids than this value will be discarded. This applies also to semi-specific (e.g. semi tryptic) searches": "在非特异性搜索中，氨基酸数量超过此值的肽段将被丢弃。这也适用于半特异性（例如半胰蛋白酶）搜索。",
    "Specify how the protein ratios will be calculated. When ‘All’ is selected the quantification is done on all peptides. With ‘Unique’, only the peptides unique for that specific protein group are used for quantification. The ‘unique + razor’ mode calculates protein ratios from the unique and razor peptides. Razor peptides are non-unique peptides assigned to the protein group with the most other peptides (Occam’s razor principle).": "指定如何计算蛋白质比例。当选择“所有”时，对所有肽段进行定量。选择“唯一”时，只有特定蛋白质组独有的肽段用于定量。“唯一 + 剃刀刀片”模式从唯一肽段和剃刀刀片肽段计算蛋白质比例。剃刀刀片肽段是非唯一肽段，分配给具有最多其他肽段的蛋白质组（奥卡姆剃刀原理）。",
    "Indicate here which kind of peptides should be used for protein quantification. If ‘No’, all peptides are used irrespective of their modification state. If ‘Yes’, all unmodified peptides are used plus those peptides that have only the modifications specified in the field below.": "在这里指明应该使用哪种肽用于蛋白质定量。如果选择“否”，则无论它们的修饰状态如何，都会使用所有肽。如果选择“是”，则使用所有未修饰的肽以及那些在下方字段中指定的仅含有修饰的肽。",
    "Peptides containing only these modifications will be used for the quantification of proteins in addition to unmodified peptides. This includes ratio and intensity calculations including iBAQ": "仅包含这些修饰的肽段将用于蛋白质的定量，除了未修饰的肽段。这包括包括iBAQ在内的比例和强度计算。",
    "If selected, also unmodified peptides will be discarded for which a modified counterpart has been identified with a modification that is not specified in the field above.": "如果选中，对于已经识别出具有未在上述字段中指定的修饰的修饰对应物的未修饰肽段，也将被丢弃。",
    "The MaxLFQ algorithm will be applied independently to samples in different parameter groups.": "MaxLFQ算法将独立应用于不同参数组中的样本。",
    "Large protein ratios will get an admixture of the total protein intensity ratio as described in the MaxLFQ paper": "较大的蛋白质比例将得到总蛋白质强度比例的混合，如MaxLFQ论文中所述。",
    "Requires for each pari-wise peptide intensity comparison that at least one of two peptides has been identified by MS/MS": "要求对于每对肽段强度比较，至少有一个肽段通过MS/MS被鉴定出来",
    "Only select infiles matching the filetype specified in the input options.": "仅选择与输入选项中指定的文件类型匹配的输入文件。",
    "The number of missed cleavages that are maximally tolerated in the in-silico digestion of the protien sequences.": "在蛋白质序列的in-silico消化中，可以容忍的最大未切割数。",
    "Select zero or more fixed modifications. They will always be attached to any occurence of the respective amino acid.": "选择零个或多个固定修饰。它们将始终附加到相应氨基酸的任何出现上。",
    "Select zero or more variable modifications. Do not specify label modifications here, neither ms1 level labels, like SILAC, nor isobaric labels.": "选择零个或多个变量修饰。不要在这里指定标签修饰，也不要指定ms1水平标签，如SILAC，或同位素标签。",
    "Select zero or more enzymes. The enzymes used for generating the in silico peptides for the Andromeda search.": "选择零个或多个酶。用于为Andromeda搜索生成in silico肽段的酶。",
    "'Unspecific means that no cleavage rule is applied but instead all peptides up to a maximal length are considered": "“非特异性”意味着不应用任何切割规则，而是考虑所有最大长度的肽段。",
    "Select a method if needed.": "如果需要，选择一个方法。",
    "Select zero or more light modifications.": "选择零个或多个轻修饰。",
    "Select zero modifications if you have two labels. Select a medium modification if you have three labels.": "如果你有两个标签，选择零修饰。如果你有三个标签，选择中等修饰。",
    "Select zero or more heavy modifications.": "选择零个或多个重修饰。",
    "Minimum number of peptides that has to be available in pair-wise comparisons between two samples for a protein.": "在两个样本之间的成对比较中，蛋白质必须可用的最小肽段数量。",
    "Defines the network to normalize the samples in the fast LFQ mode.": "定义在快速LFQ模式下用于标准化样本的网络。",
    "Specify one or more Thermo RAW files": "指定一个或多个Thermo RAW文件",
    "Specify one or more mzXML files": "指定一个或多个mzXML文件",
    "Specify one or more mzML files": "指定一个或多个mzML文件",
    "Modify parse rules if needed": "如有必要，修改解析规则",
    "It needs to matching protein group ID. If proteinGroups=NULL, use 'Proteins' column in 'evidence.txt'": "它需要匹配蛋白质组ID。如果proteinGroups=NULL，则使用'evidence.txt'中的'Proteins'列。",
    "Data frame which contains column Run, Fraction, TechRepMixture, Channel, Condition, BioReplicate, Mixture.": "数据框，包含列 Run（运行）、Fraction（分数）、TechRepMixture（技术重复混合物）、Channel（通道）、Condition（条件）、BioReplicate（生物重复）和 Mixture（混合物）。",
    "Yes will remove proteins with ’+’ in ’Only.identified.by.site’ column from proteinGroups.txt, which was identified only by a modification site. No is the default.": "是的，将从proteinGroups.txt文件中移除在'Only.identified.by.site'列中带有'+'的蛋白质，这些蛋白质仅通过一个修饰位点被鉴定。默认设置为否。",
    "When there are multiple measurements for certain feature in certain run, select the feature with the largest summation or maximal value.": "当在某个运行中某个特征有多个测量值时，选择具有最大总和或最大值的特征。",
    "Data frame contains column Run, Fraction, TechRepMixture, Mixture, Channel, BioReplicate, Condition.": "数据框包含列 Run（运行）、Fraction（分数）、TechRepMixture（技术重复混合）、Mixture（混合）、Channel（通道）、BioReplicate（生物重复）、Condition（条件）。",
    "Global median normalization on peptide level data (equalizing the medians across all the channels and MS runs). Default is Yes. It will be performed before protein-level summarization.": "全局中位数标准化对肽水平数据（使所有通道和质谱运行的中位数相等）。默认为是。它将在蛋白质水平汇总之前执行。",
    "Reference channel based normalization between MS runs on protein level data. Yes (default) needs at least one reference channel in each MS run, annotated by ’Norm’ in condition column. It will be performed after protein-level summarization. No will not perform this normalization step. If data only has one run, then use No": "在蛋白质水平数据上，基于参考通道对质谱(MS)运行之间的标准化。是（默认）需要每个MS运行中至少有一个参考通道，在条件列中用'Norm'进行注释。它将在蛋白质水平汇总后执行。不将不执行此标准化步骤。如果数据只有一个运行，则使用不。",
    "Only for 'method = msstats'. Yes (default) imputes missing values by accelerated failure time (AFT) model. No uses minimum value to impute the missing value for each peptide precursor ion.": "仅适用于'method = msstats'。是（默认）通过加速失效时间（AFT）模型来插补缺失值。否使用每个肽前体离子的最小值来插补缺失值。",
    "We assume missing values are censored. maxQuantileforCensored is maximum quantile for deciding censored missing value, for instance, 0.999. Default is empty": "我们假设缺失值是被审查的。maxQuantileforCensored 是决定审查缺失值的最大分位数，例如，0.999。默认为空。",
    "No (Default) for Profile Plot and QC Plot uses the upper limit as rounded off maximum of log2(intensities) after normalization + 3.": "没有（默认）对于轮廓图和质量控制图，使用对数2（强度）规范化后的上限四舍五入值加3。",
    "No (Default) for Profile Plot and QCPlot uses 0.": "无（默认）用于轮廓图和QC图，使用0。",
    "No (default) uses ordinary t statistic": "不（默认）使用普通的t统计量",
    "Level of significance for comparison plot. 100(1-sig)% confidence interval will be drawn.": "比较图中的显著性水平。将绘制100(1-sig)%的置信区间。",
    "Empty (default) means no fold change cutoff is applied for significance analysis. Specific value means specific fold change cutoff is applied": "空（默认）表示在显著性分析中不应用折叠变化截止值。特定值表示应用特定的折叠变化截止值。",
    "Empty (default) for volcano plot/heatmap use maximum of -log2 (adjusted p-value) or -log10 (adjusted p-value), for comparison plot uses maximum of log-fold change + CI. Alternatively, insert specific value of y-axis limit.": "空（默认）用于火山图/热图时使用 -log2（校正后的 p 值）或 -log10（校正后的 p 值）的最大值，用于比较图时使用对数倍数变化 + 置信区间的最大值。或者，插入 y 轴限制的具体值。",
    "Empty (default) for volcano plot/heatmap use minimum of -log2 (adjusted p-value) or -log10 (adjusted p-value), for comparison plot uses minimum of log-fold change - CI. Alternatively, insert specific value of y-axis limit.": "空（默认）用于火山图/热图使用最小值的-log2（校正后的p值）或-log10（校正后的p值），对于比较图使用最小值的log2倍数变化 - CI。或者，插入特定的y轴限制值。",
    "Empty (default) for use maximum for absolute value of log-fold change or 3 as default if maximum for absolute value of log-fold change is less than 3. Alternatively, insert specific value of y-axis limit.": "空（默认）以使用对数倍数变化绝对值的最大值，或者如果对数倍数变化绝对值的最大值小于3，则默认为3。或者，插入y轴限制的具体值。",
    "Yes (default) means protein names, which are significant, are displayed next to the points. No means no protein names are displayed.": "是（默认）意味着显著的蛋白质名称会显示在点旁边。否意味着不显示蛋白质名称。",
    "It needs to match protein group ID. If not selected use Proteins in 'evidence.txt'": "它需要匹配蛋白质组ID。如果没有选择，则使用'evidence.txt'中的蛋白质。",
    "Columns: Raw.file, Condition (the name of the condition is not allowed to start with a number or contain any special characters.), BioReplicate, Run, IsotopeLabelType information": "列：原始文件，条件（条件名称不允许以数字开头或包含任何特殊字符）、生物重复、运行、同位素标记类型信息",
    "Yes (default) will remove the proteins or peptides which are labeld ’iRT’ in ’StandardType’ column. No will keep them.": "是（默认）将删除在“StandardType”列中标记为'iRT'的蛋白质或肽段。否将保留它们。",
    "Yes (default) will filter out the intensities that have greater than qvalue_cutoff in Detection QValue column. Those intensities will be replaced with zero and will be considered as censored missing values for imputation purpose.": "是（默认）将过滤掉在检测Q值列中大于qvalue_cutoff的强度。这些强度将被替换为零，并在插补目的中被视为被审查的缺失值。",
    "Yes will remove the peptides including ’oxidation (M)’ in modification.": "是的，将移除包括“氧化（M）”在内的修饰的肽段。",
    "Yes will remove the proteins which have only 1 peptide and charge.": "是的，将移除只有1个肽段和电荷的蛋白质。",
    "Intensities for original intensity between 0 and 1 will be replaced with zero value after normalization.": "原始强度在0到1之间的强度值，在标准化后将被替换为零值。",
    "Peptide names should be double-quoted and separated by commas": "肽段名称应该用双引号括起来，并用逗号分隔",
    "Yes: inserts 'NA' or '0' (depending on censored intensity), No: uses the values assigned by cutoff value for censoring": "是：插入'NA'或'0'（取决于被审查的强度），否：使用截止值分配的值进行审查",
    "Yes: assumes equal variance among intensities from features. No: means that we cannot assume equal variance among intensities from features, then we will account for heterogeneous variation from different features": "是：假设特征之间的强度具有相等的方差。否：意味着我们不能假设特征之间的强度具有相等的方差，然后我们将考虑不同特征的异质变化。",
    "If you don't want to apply the threshold of noise intensity in your data, remove the value (empty field)": "如果您不想在您的数据中应用噪声强度的阈值，请移除该值（空字段）。",
    "Transition means printing feature legend intransition-level; Peptide means printing feature legend in peptide-level; NA means no feature legend printing.": "Transition 表示在转换级别打印特征图例；Peptide 表示在肽级别打印特征图例；NA 表示不打印任何特征图例。",
    "Empty (default) for Profile Plot and QC Plot uses the upper limit as rounded off maximum of log2(intensities) after normalization + 3; for Condition Plot maximum of log ratio + SD or CI. Alternatively, insert specific value of y-axis limit.": "空（默认）对于轮廓图和质量控制图使用对数2（强度）标准化后的上限四舍五入的最大值加3；对于条件图，使用对数比率加标准差或置信区间的最大值。或者，插入特定的y轴限制值。",
    "Empty (default) for Profile Plot and QCPlot uses 0; for Condition Plot is minimum of log ratio - SD or CI. Alternatively, insert specific value of lower y-axis limit.": "空（默认）的用于轮廓图和质量控制图使用0；对于条件图是最小对数比率 - 标准差或置信区间。或者，插入特定的下y轴限制值。",
    " No (Default) means each conditional level is not scaled  at  x-axis according to its actual value (equal space at x-axis). Yes means each conditional level is scaled at x-axis according to its actual value (unequal space at x-axis).": "“否”（默认）意味着每个条件级别在x轴上不按其实际值进行缩放（x轴上等距）。“是”意味着每个条件级别在x轴上按其实际值进行缩放（x轴上不等距）。",
    "CI (default) uses confidence interval with 0.95 significant level for the width of error bar. SD uses standard deviation for the width of error bar.": "CI（默认）使用0.95的显著水平来计算误差条的宽度，表示置信区间。SD使用标准差来计算误差条的宽度。",
    "It only works after  when feature subset high Quality was used in dataProcess options. Yes allows to remove 1) the features are flagged in the column, feature_quality=Uninformative which are features with bad quality, 2) outliers that are flagged in the column, is_outlier=TRUE in profile plots. No (default) shows all features and intensities in profile plots.": "它只有在数据处理选项中使用了高质量特征子集时才有效。是允许移除1)在列中标记的特征，feature_quality=Uninformative，这些是质量差的特征，2)在轮廓图中标记为is_outlier=TRUE的异常值。否（默认）在轮廓图中显示所有特征和强度。",
    " select consensusxml,featurexml,idxml data sets(s)": "选择 consensusxml、featurexml、idxml 数据集（s）",
    "* PEPMatrix: Scoring based on posterior error probabilities (PEPs) and peptide sequence similarities (scored by a substitution matrix). Requires PEPs as scores.. * PEPIons: Scoring based on posterior error probabilities (PEPs) and fragment ion similarities ('shared peak count'). Requires PEPs as scores.. * best: For each peptide ID, use the best score of any search engine as the consensus score. Requires the same score type in all ID runs.. * worst: For each peptide ID, use the worst score of any search engine as the consensus score. Requires the same score type in all ID runs.. * average:  For each peptide ID, use the average score of all search engines as the consensus. Requires the same score type in all ID runs.. * ranks: Calculates a consensus score based on the ranks of peptide IDs in the results of different search engines. The final score is in the range (0, 1], with 1 being the best score. No requirements about score types": "* PEPMatrix：基于后验误差概率（PEPs）和肽段序列相似性（由替代矩阵评分）。需要PEPs作为分数。* PEPIons：基于后验误差概率（PEPs）和片段离子相似性（“共享峰计数”）。需要PEPs作为分数。* best：对于每个肽段ID，使用任何搜索引擎的最佳分数作为共识分数。需要所有ID运行中相同的分数类型。* worst：对于每个肽段ID，使用任何搜索引擎的最差分数作为共识分数。需要所有ID运行中相同的分数类型。* average：对于每个肽段ID，使用所有搜索引擎的平均分数作为共识。需要所有ID运行中相同的分数类型。* ranks：根据不同搜索引擎结果中肽段ID的排名计算共识分数。最终分数在（0,1]范围内，其中1是最佳分数。对分数类型没有要求",
    "(i.e. those containing no peptide hit for the current spectrum) when calculating 'min_support'?": "（即那些对于当前谱图不包含任何肽段命中）在计算'min_support'时？",
    "possibilities are taken from the input dataset.": "可能性来自输入数据集。",
    "Mean intensity column name for desired experimental condition": "所需实验条件的平均强度列名称",
    "must be created by 'metaQuantome: download'": "必须由 'metaQuantome: 下载' 创建",
    "The column name within the intensity file that corresponds to the peptide sequences": "强度文件中对应于肽段序列的列名",
    "Input file as imzML (composite upload), Analyze7.5 (composite upload) or Cardinal 'MSImageSet'  or 'MSImagingExperiment' saved as RData (regular upload)": "输入文件为 imzML（复合上传），Analyze7.5（复合上传）或 Cardinal 'MSImageSet' 或 'MSImagingExperiment' 保存为 RData（常规上传）",
    "Choose Yes if peak detection has already been done.": "如果峰值检测已经完成，请选择“是”。",
    "Choose no if your input is an Analyze7.5 or continuous imzML file": "如果您的输入是Analyze7.5或连续的imzML文件，请选择“否”。",
    "This should be set to the native accuracy of the mass spectrometer, if known": "如果已知，这应该设置为质谱仪的固有精度",
    "either m/z or ppm": "要么 m/z 或者 ppm",
    "Same number and order of files as input files. First column x values, second column y values, further columns with annotations": "与输入文件相同数量和顺序的文件。第一列是 x 值，第二列是 y 值，其他列包含注释。",
    "Tabular file with three columns: 1 for the filename, 1 for the x-coordinate shift and 1 for the y-coordinate shift. Pixels with the same coordinates after shifting will be deleted.": "具有三列的表格文件：1列为文件名，1列为x坐标偏移，1列为y坐标偏移。偏移后具有相同坐标的像素将被删除。",
    "Yes: all file coordinates are shifted in order to have at least one pixel with x = 1 and one with y = 1; then coordinates shifts are added. No: the coordinate shifts are added to the current coordinates of the file": "是的：所有文件坐标都会发生偏移，以便至少有一个像素的x坐标为1，另一个像素的y坐标为1；然后加上坐标偏移量。否：坐标偏移量会加到文件当前的坐标上。",
    "If all m/z values of all files are exactly the same choose 'No'. If not sure choose 'Yes'.": "如果所有文件的所有m/z值完全相同，请选择“否”。如果不确定，请选择“是”。",
    " select idxml,mzid data sets(s)": "选择 idxml 和 mzid 数据集（s）",
    " (space separated list, in order to allow for spaces in list items surround them by single quotes)": "（用空格分隔的列表，为了允许列表项中有空格，将它们用单引号包围起来）",
    "Only valid together with -multiple_search_engines flag": "仅与-multiple_search_engines标志一起有效",
    "Select FASTA database from history": "从历史记录中选择FASTA数据库",
    "Select appropriate MGF dataset(s) from history": "从历史记录中选择适当的 MGF 数据集",
    "Allow peptides to contain up to this many missed enzyme cleavage sites.": "允许肽段包含最多这个数量的未被酶切割位点。",
    "Select based on instrument used, as different machines provide different quality of spectra. ppm is a standard for most precursor ions": "根据使用的仪器进行选择，因为不同的机器提供不同质量的谱图。ppm 对于大多数前体离子来说是一个标准",
    "Provide error value for precursor ion, based on instrument used. 10 ppm recommended for Orbitrap instrument": "根据使用的仪器，为前体离子提供误差值。对于Orbitrap仪器，推荐使用10 ppm。",
    "Provide error value for fragment ions, based on instrument used": "根据使用的仪器提供片段离子的错误值",
    "aka. protein grouping": "别名：蛋白质分组",
    "aka. two-step searching": "也称为两步搜索",
    "Semi-tryptic cleavage": "半胨酶解",
    "for the MS instrument": "对于质谱仪",
    "Provide a name for the experimental condition.": "为实验条件提供一个名称。",
    "Give a comma-separated list of the column names that correspond to samples within the experimental condition.": "提供一个逗号分隔的列名列表，这些列名对应于实验条件中的样本。",
    "Ex: moFF tool output (headers: peptide X737NS X737WS)": "例如：moFF工具输出（标题：肽段 X737NS X737WS）",
    "Specify the column indices of the file's header": "指定文件头的列索引",
    "Sample file(s) in .MZML format.": "样本文件(.s)在.MZML格式中。",
    "Sample file in .OSW format.": "OSW格式的样本文件。",
    "Numeric value between 0 and 1. It is used to filter features from osw file which have SCORE_MS2.QVALUE less than itself.": "数值值介于0和1之间。它用于从osw文件中过滤具有 SCORE_MS2.QVALUE 小于自身的特征。",
    "Numeric value between 0 and 1. Features should have m-score lower than this value for participation in LOESS fit.": "数值介于0和1之间。特征的m分数应低于此值才能参与LOESS拟合。",
    "Span value for LOESS fit. For targeted proteomics 0.1 could be used.": "LOESS拟合的跨度值。对于靶向蛋白质组学，可以使用0.1。",
    "Defines how much distance in the unit of rse remains a noBeef zone.": "定义了在rse单位中，无牛肉区域保持多少距离。",
    "Penalty for introducing first gap in alignment. This value is multiplied by base gap-penalty.": "在比对中引入第一个间隙的惩罚。这个值乘以基础间隙惩罚。",
    "Penalty for introducing subsequent gaps in alignment. This value is multiplied by base gap-penalty.": "在比对中引入后续空位的惩罚。该值乘以基础空位惩罚。",
    "In SIM measure = dotProductMasked mode, angular similarity should be higher than cosAngleThresh otherwise similarity is forced to zero.": "在SIM度量 = dotProductMasked模式下，角度相似性应高于cosAngleThresh，否则相似性将被强制为零。",
    "An input for alignment with free end-gaps. No: Global alignment Yes: Overlap alignment": "一个用于带有自由端间隙的对齐输入。否：全局对齐 是：重叠对齐",
    "In SIM measure = dotProductMasked mode, values in similarity matrix higher than dotProdThresh quantile are checked for angular similarity.": "在SIM模式下，相似性矩阵中高于dotProdThresh分位数的值会检查角度相似性。",
    "Must be between 0 and 1. This is used to calculate base gap-penalty from similarity distribution.": "必须在0到1之间。这用于根据相似性分布计算基准间隙惩罚。",
    "No: Indices farther from noBeef distance are filled with distance from linear fit line.": "没有：距离noBeef距离更远的索引用线性拟合线上的距离填充。",
    "Modulates penalization of masked indices.": "调节对掩蔽索引的惩罚。",
    "Defines the upper limit of FDR on a precursor to be considered for multipeptide.": "定义了FDR的上限，以考虑前体是否适用于多肽。",
    "Must be between 0 and maxFdrQuery. Features below unalignedFDR are considered for quantification even without the RT alignment.": "必须在0和maxFdrQuery之间。即使没有RT对齐，未对齐FDR以下的特征也会被考虑进行定量。",
    "Must be between unalignedFDR and 1. Features below alignedFDRare considered for quantification after the alignment.": "必须在未比对FDR和1之间。在比对后，低于比对FDR的特征将被考虑用于量化。",
    "Method to estimate the background of a peak contained in XICs.": "估计XICs中峰值背景的方法。",
    "Method to compute the area of a peak contained in XICs.": "计算XICs中峰面积的方法。",
    "Enable/disable exponentially modified gaussian peak model fitting.": "启用/禁用高斯峰模型的指数修改拟合。",
    "Recalculate intensity for all analytes.": "重新计算所有分析物的强度。",
    "Calculate intensity for analytes for which features are not found.": "计算未找到特征的分析物的强度。",
    "No: Raw chromatograms will be used for quantification. Yes: Smoothed chromatograms will be used for quantification.": "不：将使用原始色谱图进行定量。是：将使用平滑色谱图进行定量。",
    " select mzml data sets(s)": "选择mzml数据集（s）",
    "e.g. (Oritrap CID-Trap, CID-FT, HCD-FT; QTof CID-TOF)": "例如 (Oritrap CID-Trap, CID-FT, HCD-FT; QTof CID-TOF)",
    " select txt data sets(s)": "选择文本数据集（s）",
    " select idxml data sets(s)": "选择 idXML 数据集（s）",
    "e.g. 'Carbamidomethyl (C)'": "例如 'Carbamidomethyl (C)'",
    "The residue field is case sensitive. For example: lower case 'sty' implies that the neutral loss can only occur if the specified modification is present. Syntax: NL = <RESDIUES> -<NEUTRAL_LOSS_MOLECULAR_FORMULA> <MASS_LOST> (space separated list, in order to allow for spaces in list items surround them by single quotes)": "残基字段是区分大小写的。例如：小写的 'sty' 表示只有在指定的修饰存在时，中性丢失才能发生。语法：NL = <RESDIUES> -<NEUTRAL_LOSS_MOLECULAR_FORMULA> <MASS_LOST> （空格分隔的列表，为了允许列表项中有空格，请用单引号将它们包围起来）",
    "The syntax for this is identical to that of the normal neutral losses given above except that the residue is always 'X'. Syntax: DECOY_NL = X -<NEUTRAL_LOSS_MOLECULAR_FORMULA> <MASS_LOST> (space separated list, in order to allow for spaces in list items surround them by single quotes)": "这个语法与上面给出的正常中性丢失的语法相同，不同之处在于残基总是'X'。语法：DECOY_NL = X -<NEUTRAL_LOSS_MOLECULAR_FORMULA> <MASS_LOST>（空格分隔的列表，如果列表项中有空白，则用单引号将它们包围起来）",
    "Provide a full or relative path, or make sure it can be found in your PATH environment select txt data sets(s)": "提供完整或相对路径，或确保它可以在您的PATH环境变量中找到，选择txt数据集",
    "(e.g. AccurateMassSearch) select featurexml data sets(s)": "（例如 AccurateMassSearch）选择 featureXML 数据集（们）",
    "This option does only work in combination with the fragment annotation via Sirius": "这个选项仅与通过Sirius进行片段注释结合使用时才有效",
    "Basis for the method is the fragmentation-tree re-rooting approach ('original'). This approach can be extended by using 'resolve_overlap', which will resolve overlapping fragments of the highest intensity fragments chosen, by adding -CH2 mass to the overlapping fragments. 'Add_shift' will add a -CH2 mass shift to the target fragments and use them as additional decoys if fragmentation-tree re-rooting failed. 'Both' combines the extended methods (resolve_overlap, add_shift)": "该方法的基础是断裂树重新根化方法（'original'）。通过使用'resolve_overlap'可以扩展该方法，它将通过向重叠片段添加-CH2质量来解决所选最高强度片段的重叠片段。'Add_shift'将向目标片段添加一个-CH2质量偏移，并在断裂树重新根化失败时将它们用作额外的诱饵。'Both'结合了扩展方法（resolve_overlap，add_shift）。",
    "To use this parameter, setting the feature_only flag is necessary": "要使用这个参数，需要设置feature_only标志",
    "If no peak is found (C13 distance) the function will abort. Be careful with noisy data - since this can lead to wrong isotope patterns": "如果未找到峰（C13距离），该函数将中止。请注意噪声数据，因为这可能导致错误的同位素模式。",
    "If not specified SIRIUS uses all available cores": "如果未指定，SIRIUS会使用所有可用的核心",
    "0 for an infinite amount of time": "0表示无限的时间",
    "Omit this option if you want to consider all possible molecular formulas": "如果您希望考虑所有可能的分子公式，则忽略此选项",
    "Example: [M+H]+, . [M-H]-, [M+Cl]-, [M+Na]+, [M]+. You can also provide a . comma separated list of adducts": "示例：[M+H]+，。[M-H]-，[M+Cl]-，[M+Na]+，[M]+。您也可以提供一个用逗号分隔的加合物列表",
    "Set to force output of results for each possible ionization, even if not part of highest ranked results. -1 omits parameter in Sirius": "设置为强制输出每个可能的离子化结果，即使它们不是排名最高的结果的一部分。-1 在 Sirius 中省略了参数",
    "Write SBrClBSe to allow the elements S,Br,Cl,B and Se": "将 SBrClBSe 写入以允许元素 S、Br、Cl、B 和 Se",
    "Write CHNOPSCl to allow the elements C, H, N, O, P, S and Cl. Add numbers in brackets to restrict the minimal and maximal allowed occurrence of these elements: CHNOP[5]S[8]Cl[1-2]. When one number is given then it is interpreted as upper bound. Default is CHNOP": "将 CHNOPSCl 写入以允许元素 C、H、N、O、P、S 和 Cl。在括号中添加数字以限制这些元素的最小和最大允许出现次数：CHNOP[5]S[8]Cl[1-2]。如果给出一个数字，则将其解释为上限。默认为 CHNOP。",
    "When filtering is enabled, molecular formulas are excluded if their theoretical isotope pattern does not match the theoretical one, even if their MS/MS pattern has high score": "当启用过滤时，如果分子公式的理论同位素模式与理论模式不匹配，则会排除它们，即使它们的MS/MS模式得分很高",
    "Example: [M+H]+, [M-H]-, [M+Cl]-, [M+Na]+, [M]+. You can also provide a comma separated list of adducts": "示例：[M+H]+，[M-H]-，[M+Cl]-，[M+Na]+，[M]+。您也可以提供一个用逗号分隔的加合物列表",
    "If no database is given all possible molecular formulas will be respected (no database is used). Example: possible DBs: ALL,BIO,PUBCHEM,MESH,HMDB,KNAPSACK,CHEBI,PUBMED,KEGG,HSDB,MACONDA,METACYC,GNPS,ZINCBIO,UNDP,YMDB,PLANTCYC,NORMAN,ADDITIONAL,PUBCHEMANNOTATIONBIO,PUBCHEMANNOTATIONDRUG,PUBCHEMANNOTATIONSAFETYANDTOXIC,PUBCHEMANNOTATIONFOOD,KEGGMINE,ECOCYCMINE,YMDBMINE": "如果没有指定数据库，则将尊重所有可能的分子公式（不使用数据库）。示例：可能的数据库包括：ALL,BIO,PUBCHEM,MESH,HMDB,KNAPSACK,CHEBI,PUBMED,KEGG,HSDB,MACONDA,METACYC,GNPS,ZINCBIO,UNDP,YMDB,PLANTCYC,NORMAN,ADDITIONAL,PUBCHEMANNOTATIONBIO,PUBCHEMANNOTATIONDRUG,PUBCHEMANNOTATIONSAFETYANDTOXIC,PUBCHEMANNOTATIONFOOD,KEGGMINE,ECOCYCMINE,YMDBMINE",
    " select featurexml data sets(s)": "选择 feature.xml 数据集（们）",
    "Without further SIRIUS processing": "无需进一步的SIRIUS处理",
    " select mzml,mzxml data sets(s)": "选择 mzml、mzxml 数据集（s）",
    " select fasta data sets(s)": "选择 FASTA 数据集（一个或多个）",
    "Annotates target-decoy information": "注释目标-诱饵信息",
    "Used for tie breaker of results (in spectrally ambiguous cases) and zero bin boosting in open searches (0 disables these features). This option is STRONGLY recommended for open searches": "用于光谱歧义情况下结果的平局破解，以及开放搜索中的零二进制提升（0禁用这些功能）。对于开放搜索，强烈推荐此选项",
    "Should be set to 0 (disabled) for open search or 0/1/2 for correction of narrow window searches. Shifts the precursor mass window to multiples of this value multiplied by the mass of C13-C12": "应设置为0（禁用）以进行开放搜索，或设置为0/1/2以校正狭窄窗口搜索。将前体质量窗口移动到此值的倍数乘以C13-C12的质量差。",
    "If a greater number than the maximum is generated, only the unmodified peptide is considered": "如果生成的数字大于最大值，则只考虑未修改的肽链",
    "Useful for iTRAQ/TMT experiments (i.e. 0.0 150.0)": "适用于iTRAQ/TMT实验（即 0.0 150.0）",
    "If specified, also spectrum:override_charge must be set)": "如果指定了，还必须设置spectrum:override_charge（参数）",
    "Should be set to a number greater than search:output_report_topN if zero bin boosting is desired": "应设置为大于search:output_report_topN的数字，如果需要零二进制提升的话",
    "Useful for recovery of modified peptides near C-terminus in open search. 0 disables this option": "适用于在开放搜索中恢复C末端附近的修饰肽段。0禁用此选项",
    "MSFragger recommends a minimum of 4 for narrow window searching and 6 for open searches": "MSFragger 建议在窄窗口搜索中至少使用 4，而在开放搜索中至少使用 6。",
    "e.g. Carbamidomethylation (C). When multiple different masses are given for one aminoacid this parameter (unimod) will have priority (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如，羰甲基化（C）。当为一个氨基酸给出多个不同的质量时，这个参数（unimod）将具有优先权（用空格分隔的列表，为了允许列表项中有空格，将它们用单引号包围）",
    "If empty (default), it's determined automatically (checking for common terms, both as prefix and suffix)": "如果为空（默认），则会自动确定（检查常见的术语，作为前缀和后缀）",
    "(ignored if decoy_string is empty)": "（如果decoy_string为空，则忽略）",
    "AAAs are 'B', 'J', 'Z' and 'X'": "AAAs 是 'B'、'J'、'Z' 和 'X'",
    "The required runtime is exponential in the number of mm's; apply with care. MM's are allowed in addition to AAA's": "所需的运行时间与mm的数量呈指数关系；请谨慎使用。除了AAA之外，还允许使用MM。",
    "Also occurrences of 'J' will be treated as 'I' thus avoiding ambiguous matching": "同时，'J'的出现将被视为'I'，从而避免了模糊匹配",
    "e.g. trypsin cleaves after lysine (K) or arginine (R), but not before proline (P). Default: deduce from input": "例如，胰蛋白酶在赖氨酸（K）或精氨酸（R）后切割，但不在脯氨酸（P）前切割。默认值：从输入中推断",
    "Default: deduce from input..   'full': both internal cleavage sites must match..   'semi': one of two internal cleavage sites must match..   'none': allow all peptide hits no matter their context (enzyme is irrelevant)": "默认：从输入中推断。'full'：两个内部切割位点都必须匹配。'semi'：两个内部切割位点中的一个必须匹配。'none'：允许所有肽段命中，不管它们的上下文如何（酶是无关紧要的）",
    " select mgf,ms2,mzml,mzxml data sets(s)": "选择 mgf、ms2、mzml、mzxml 数据集（s）",
    "Non-existing relative filenames are looked up via 'OpenMS.ini:id_db_dir' select fasta data sets(s)": "不存在的相对文件名通过 'OpenMS.ini:id_db_dir' 查找 fasta 数据集",
    "Takes into account the error introduced by choosing a non-monoisotopic peak for fragmentation. Combined with 'precursor_mass_tolerance'/'precursor_error_units', this determines the actual precursor mass tolerance. E.g. for experimental mass 'exp' and calculated mass 'calc', '-precursor_mass_tolerance 20 -precursor_error_units ppm -isotope_error_range -1,2' tests '|exp - calc - n * 1.00335 Da| < 20 ppm' for n = -1, 0, 1, 2": "考虑选择非单同位素峰进行碎裂时引入的误差。结合 'precursor_mass_tolerance'/'precursor_error_units'，这决定了实际的前体质量容差。例如，对于实验质量 'exp' 和计算质量 'calc'，'-precursor_mass_tolerance 20 -precursor_error_units ppm -isotope_error_range -1,2' 测试 '|exp - calc - n * 1.00335 Da| < 20 ppm'，其中 n = -1, 0, 1, 2",
    "Note: MS-GF+ does not support blocking rules. (MS-GF+ parameter '-e')": "注意：MS-GF+ 不支持屏蔽规则。（MS-GF+ 参数 '-e'）",
    "This is required by Percolator and hence by default enabled": "这是由Percolator所需的，因此默认启用",
    "If this value is large, the search may take very long": "如果这个值很大，搜索可能需要很长时间",
    "(default: -1 meaning unlimited)": "（默认：-1 表示无限制）",
    "More tasks than threads will reduce the memory requirements of the search, but will be slower (how much depends on the inputs).. 1 <= tasks <= numThreads: will create one task per thread, which is the original behavior.. tasks = 0: use default calculation - minimum of: (threads*3) and (numSpectra/250).. tasks < 0: multiply number of threads by abs(tasks) to determine number of tasks (i.e., -2 means \"2 * numThreads\" tasks).. One task per thread will use the most memory, but will usually finish the fastest.. 2-3 tasks per thread will use comparably less memory, but may cause the search to take 1.5 to 2 times as long": "比线程数更多的任务将减少搜索的内存需求，但会更慢（具体取决于输入）。1 <= tasks <= numThreads：将创建每个线程一个任务，这是原始的行为。tasks = 0：使用默认计算 - 最小值：（线程数*3）和（numSpectra/250）。tasks < 0：将线程数乘以abs(tasks)以确定任务数（即，-2表示“2 * numThreads”任务）。每个线程一个任务将使用最多的内存，但通常会完成得最快。每个线程2-3个任务将使用相对较少的内存，但可能会导致搜索时间延长1.5到2倍。",
    "e.g. 'Carbamidomethyl (C)' or 'Oxidation (M)'": "例如 'Carbamidomethyl (C)' 或 'Oxidation (M)'",
    "See documentation for examples. Parameters of the adapter take precedence. Use conf file only for settings not available here (for example, any fixed/var modifications, in the conf file will be ignored, since they are provided via -mod flag) select txt data sets(s)": "查看文档以获取示例。适配器的参数优先。仅在此处无法使用的设置（例如，任何固定/可变修改，在conf文件中将被忽略，因为它们是通过-mod标志提供的）时使用conf文件。选择txt数据集",
    "This allows the calculation of FDRs, but should only be used if the database does not already contain decoys": "这允许计算FDRs（假阳性率），但仅在数据库尚未包含诱饵时使用",
    "Try this only if the default conversion takes too long or uses too much memory": "如果默认的转换耗时过长或占用内存过多，请尝试这个方法",
    "Two or three columns: x values, y values, optionally annotations": "两列或三列：x 值，y 值，可选注释",
    "Numbers of colours should be the same as number of components": "颜色的数量应该与组件的数量相同",
    "m/z range to add on either side of the given m/z to create a window in which the mean of all intensities will be computed": "在给定的 m/z 两侧添加的 m/z 范围，以创建一个窗口，在该窗口中将计算所有强度的平均值",
    "The 'histogram' equalization method flatterns the distribution of intensities. The hotspot 'suppression' method uses thresholding to reduce the intensities of hotspots": "“直方图”均衡化方法使强度分布趋于平坦。热点“抑制”方法使用阈值处理来减少热点的强度。",
    "The 'gaussian' smoothing method smooths images with a simple gaussian kernel. The 'adaptive' method uses bilateral filtering to preserve edges": "“高斯”平滑方法使用简单的高斯核平滑图像。“自适应”方法使用双边滤波来保留边缘。",
    "Opacity scaling factor (1 is linear)": "不透明度缩放因子（1 表示线性）",
    "Numbers of colours should be the same as number of m/z in tabular file": "颜色的数量应该与表格文件中 m/z 的数量相同",
    "Smoothing parameter for the spline smoothing applied to the spectrum in order to decide the cutoffs for throwing away false noise spikes that might occur inside peaks": "用于对谱进行样条平滑的平滑参数，以便决定截断值，以丢弃可能在峰内出现的假噪声尖峰",
    "Savitzky golay smoothing can introduce negative intensity values it is recommended to replace them with zero": "Savitzky-Golay 平滑可能会引入负强度值，建议将它们替换为零。",
    "The tolerance to be used when matching the peaks in the unaligned spectra to the reference spectrum. If this is NA, then automatically guess a tolerance from the data.": "在将未对齐的光谱中的峰与参考光谱匹配时使用的容差。如果这是NA，则自动从数据中猜测容差。",
    "The top quantile of reference points (peaks detected via local maxima) to use from the reference spectrum.": "参考谱中使用局部最大值检测到的参考点（峰值）的顶部分位数。",
    "The smoothing parameter for the local polynomial regression used to determine the warping function.": "用于确定变形函数的局部多项式回归的平滑参数。",
    "The minimal signal to noise ratio for peaks to be considered as a valid peak.": "峰值被认为是有效峰值的最小信噪比。",
    "Number of blocks in which to divide mass spectrum to calculate noise": "将质谱分成多少个区块以计算噪声",
    "Window width for seeking local maxima": "寻找局部最大值的窗口宽度",
    "Peaks that differ less than this value will be aligned together": "峰值差异小于这个值的将被对齐在一起",
    "Peaks that occur in the dataset in lesser proportion than this will be dropped (0.01 --> filtering for 1% of spectra)": "在数据集中出现比例小于这个值的峰将被删除（0.01 --> 筛选1%的光谱）",
    "Tabular file with m/z features to extract from input file": "输入文件中要提取的 m/z 特征的表格文件",
    "Width must be greater than range of m/z values divided by number of m/z features": "宽度必须大于 m/z 值范围除以 m/z 特征数",
    "Binning can introduce NAs, should they be replaced with 0": "分箱可能会引入NA值，是否应该将它们替换为0",
    "logarithm base 2 (log2) or squareroot (sqrt)": "以2为底的对数（log2）或平方根（sqrt）",
    "0 values are set to NA before log2 transformation, after transformation they can be set back to 0": "在对数转换前，0值被设置为NA，转换后它们可以被重新设置为0。",
    "Annotation tabular file that contains the batch identifier for each spectrum in one column.": "包含每个谱图的批次标识符的注释表格文件。",
    "Column with x coordinates of pixels.": "像素的x坐标列。",
    "Column with y ccordinates of pixels.": "像素的 y 坐标列。",
    "The column that contains the batch identifier for each spectrum.": "包含每个谱图批次标识符的列。",
    "The column that contains the condition annotation for each spectrum. Typically these are the groups you want to compare. If not applicable, the batch column can be selected again as this information is only used for the QC plot.": "包含每个谱图条件注释的列。通常这些是您想要比较的组。如果不适用，可以选择批次列，因为此信息仅用于质量控制图。",
    "'min_spacing' is the smaller of the two spacings from the peak apex to its two neighboring points. '0' to disable the constraint. Not applicable to chromatograms": "'min_spacing' 是从峰顶到其两个相邻点的两个间距中较小的一个。'0' 用于禁用此约束。不适用于色谱图",
    "If this difference is exceeded a missing point is assumed (see parameter 'missing'). A higher value implies a less stringent peak definition, since individual signals within the peak are allowed to be further apart. '0' to disable the constraint. Not applicable to chromatograms": "如果这个差异被超过，则假定有一个缺失点（见参数'missing'）。一个更高的值意味着一个不太严格的峰定义，因为峰内的个别信号被允许更远地分开。'0'用于禁用该约束。不适用于色谱图",
    "A missing data point occurs if the spacing between two subsequent data points exceeds 'spacing_difference * min_spacing'. 'min_spacing' is the smaller of the two spacings from the peak apex to its two neighboring points. Not applicable to chromatograms": "如果两个连续数据点之间的间距超过“spacing_difference * min_spacing”，则会出现一个缺失的数据点。“min_spacing”是从峰顶到其两个邻近点的两个间距中较小的一个。不适用于色谱图",
    "If empty, auto mode is enabled, all peaks which aren't picked yet will get picked. Other scans are copied to the output without changes (space separated list, in order to allow for spaces in list items surround them by single quotes)": "如果为空，将启用自动模式，所有尚未选择的峰将被选择。其他扫描将被复制到输出中，且不做任何更改（以空格分隔的列表，如果列表项中包含空格，请用单引号将它们包围起来）。",
    "Either absolute in the unit of input, e.g. 'm/z' for spectra, or relative as ppm (only sensible for spectra, not chromatograms)": "可以是输入单位中的绝对值，例如质谱图中的“m/z”，或者是相对值，如ppm（仅适用于质谱图，不适用于色谱图）",
    "By default, it will be calculated automatically (see auto_mode). Only provide this parameter if you know what you are doing (and change 'auto_mode' to '-1')! All intensities EQUAL/ABOVE 'max_intensity' will be added to the LAST histogram bin. If you choose 'max_intensity' too small, the noise estimate might be too small as well.  If chosen too big, the bins become quite large (which you could counter by increasing 'bin_count', which increases runtime). In general, the Median-S/N estimator is more robust to a manual max_intensity than the MeanIterative-S/N": "默认情况下，它将自动计算（见auto_mode）。只有在你知道自己在做什么时才提供此参数（并将'auto_mode'更改为'-1'）！所有强度等于/高于'max_intensity'的值将被添加到直方图的最后一个箱子中。如果你选择的'max_intensity'太小，噪声估计可能也太小。如果选择得太大，箱子会变得相当大（你可以通过增加'bin_count'来抵消，这会增加运行时间）。一般来说，中值信噪比估计器比均值迭代信噪比估计器对手动max_intensity更为健壮。",
    " select consensusxml data sets(s)": "选择共识XML数据集（s）",
    " select trafoxml data sets(s)": "选择 trafoxml 数据集（s）",
    "If the directory does not exist, it will be created as well": "如果目录不存在，它也会被创建",
    "'auto' uses information from FeatureXML": "\"自动\" 使用来自 FeatureXML 的信息",
    " select traml data sets(s)": "选择traml数据集（s）",
    "Needed for generation of dummy features select mzml data sets(s)": "需要生成虚拟特征选择mzml数据集（s）",
    "[...] specifies the labels for a single sample. For example. [][Lys8,Arg10] ... SILAC. [][Lys4,Arg6][Lys8,Arg10] ... triple-SILAC. [Dimethyl0][Dimethyl6] ... Dimethyl. [Dimethyl0][Dimethyl4][Dimethyl8] ... triple Dimethyl. [ICPL0][ICPL4][ICPL6][ICPL10] ... ICPL": "[...] 指定了单个样本的标签。例如，[][Lys8,Arg10] ... SILAC。[][Lys4,Arg6][Lys8,Arg10] ... 三重SILAC。[Dimethyl0][Dimethyl6] ... 二甲基化。[Dimethyl0][Dimethyl4][Dimethyl8] ... 三重二甲基化。[ICPL0][ICPL4][ICPL6][ICPL10] ... ICPL",
    "(Only relevant if enzymatic cutting site coincides with labelling site. For example, Arg/Lys in the case of trypsin digestion and SILAC labelling.)": "（仅在酶切位点与标记位点重合时相关。例如，在胰蛋白酶消化和SILAC标记的情况下，精氨酸/赖氨酸。）",
    "If present, SWATH specific scoring will be applied select mzml data sets(s)": "如果存在，SWATH特定的评分将被应用选择mzml数据集（s）",
    " select consensusxml,featurexml data sets(s)": "选择 consensusxml 和 featurexml 数据集（s）",
    "Note that this works only datasets with a significant amount of pairs! If 'false' the parameters 'rt_pair_dist', 'rt_dev_low' and 'rt_dev_high' define the optimal distance": "请注意，这只适用于具有大量配对数据集！如果为'false'，则参数'rt_pair_dist'、'rt_dev_low'和'rt_dev_high'定义了最佳距离",
    "by division through charge) (space separated list, in order to allow for spaces in list items surround them by single quotes)": "通过电荷除法（用空格分隔的列表，如果列表项中有空格，则用单引号将它们包围）",
    "Non-existing relative file-names are looked up via'OpenMS.ini:id_db_dir' select fasta data sets(s)": "不存在的相对文件名通过'OpenMS.ini:id_db_dir'查找fasta数据集",
    "Tandem configuration file. All parameters of this adapter take precedence over the file - use it for parameters not available here. A template file can be found at 'OpenMS/share/CHEMISTRY/XTandem_default_config.xml' select xml data sets(s)": "串联配置文件。此适配器的所有参数优先于文件中的参数 - 用于此处不可用的参数。模板文件可以在'OpenMS/share/CHEMISTRY/XTandem_default_config.xml'找到，选择XML数据集。",
    "Tandem default)": "串联默认值",
    "Set this flag to disable": "将此标志设置为禁用",
    "All, valid ones (passing the E-Value threshold), or stochastic (failing the threshold)": "所有，有效的（通过E值阈值的），或者随机的（未通过阈值的）",
    " select mzml,mzxml,sqmass data sets(s)": "选择 mzml、mzxml、sqmass 数据集（s）",
    " select pqp,tabular,traml data sets(s)": "选择 pqp、tabular、traml 数据集（s）",
    "Note that the first line is a header and will be skipped select txt data sets(s)": "请注意，第一行是标题，将被跳过，选择 txt 数据集（s）",
    "This is the full window size, e.g. a value of 10 milliseconds would extract 5 milliseconds on either side": "这是完整的窗口大小，例如，10毫秒的值将提取任一侧的5毫秒",
    "Only turn this off if you know what you are doing": "只有在你清楚自己在做什么时才关闭此选项",
    "This specifies how the RT alignment is performed and how outlier detection is applied. Outlier detection can be done iteratively (by default) which removes one outlier per iteration or using the RANSAC algorithm": "这指定了如何执行RT比对以及如何应用异常值检测。异常值检测可以迭代地进行（默认情况下），每次迭代移除一个异常值，或者使用RANSAC算法进行。",
    "'linear': perform linear regression (for few anchor points). 'interpolated': Interpolate between anchor points (for few, noise-free anchor points). 'lowess' Use local regression (for many, noisy anchor points). 'b_spline' use b splines for smoothing": "'线性': 执行线性回归（适用于少数锚点）。'插值': 在锚点之间进行插值（适用于少数、无噪声的锚点）。'lowess' 使用局部回归（适用于多数、有噪声的锚点）。'b_spline' 使用b样条曲线进行平滑。",
    "Iterative methods remove one outlier at a time. Jackknife approach optimizes for maximum r-squared improvement while 'iter_residual' removes the datapoint with the largest residual error (removal by residual is computationally cheaper, use this with lots of peptides)": "迭代方法一次移除一个异常值。Jackknife方法优化以获得最大的R平方改进，而'iter_residual'移除具有最大残差误差的数据点（通过残差移除在计算上更便宜，当有很多肽时使用此方法）。",
    "This should be used if the algorithm removes too many datapoints but it may lead to true outliers being retained": "如果算法删除了太多数据点，可以使用此选项，但可能会导致真正的异常值被保留。",
    "Default is set to 3% which is around +/- 4 minutes on a 120 gradient": "默认设置为3%，这在120分钟的梯度上大约是正负4分钟。",
    "Use this option you do not expect all your peptides to be detected in a sample and too many 'bad' peptides enter the outlier removal step (e.g. due to them being endogenous peptides or using a less curated list of peptides)": "使用此选项，如果您不期望样本中检测到所有肽段，并且太多的“不良”肽段进入异常值去除步骤（例如，由于它们是内源性肽段或使用较少策划的肽段列表）。",
    "-2 to 2)": "-2到2",
    "0 to 10)": "0到10",
    "This option should be used to ensure that there is a complete coverage of the RT space (this should detect cases where only a part of the RT gradient is actually covered by normalization peptides)": "这个选项应该被用来确保完全覆盖RT空间（这应该能够检测到只有RT梯度的一部分实际上被标准化肽覆盖的情况）",
    "If your normalized RT has a different range, pass this here (e.g. it goes from 0 to 100, set this value to 100)": "如果标准化后的相对丰度（RT）有不同的范围，请在这里指定（例如，如果范围是从0到100，请将此值设置为100）。",
    "e.g. 0.25 means 25% extra on each side)": "例如，0.25表示每侧额外增加25%",
    "The 'original' method is only provided for historical purposes, please use the 'exact' method and set parameters using the PeakIntegrator: settings. The same original or smoothed chromatogram specified by peak_integration will be used for background estimation": "“原始”方法仅出于历史原因提供，请使用“精确”方法，并使用PeakIntegrator：设置来设置参数。将使用由peak_integration指定的相同原始或平滑色谱图进行背景估计。",
    "Tries to use the consensus (median) peak border if the variation within the picked peaks is too large": "尝试使用共识（中位数）峰值边界，如果选定峰值内的变异过大",
    "If the Z-Score is above this value, the median is used for peak boundaries (default value 1.0)": "如果Z-Score高于这个值，将使用中位数作为峰值边界（默认值为1.0）",
    "The resulting score is centered around zero and values above 0 are generally good and below -1 or -2 are usually bad": "结果得分以零为中心，大于0的值通常是好的，而小于-1或-2的值通常是不好的。",
    "(e.g., tailing) that can be used for downstream QC/QA": "（例如，尾部）可用于下游质量控制/质量保证",
    "This number has to be uneven. If it is not, 1 will be added": "这个数字必须是奇数。如果不是，将加上1",
    "(e.g. extend the peak at least by this amount on both sides) in seconds. -1 turns this feature off": "（例如，在两侧至少扩展峰值这么多秒）-1关闭此功能",
    "Note that setting this too high (e.g. 1.0) can lead to peaks whose flanks are not fully captured": "请注意，如果将此设置得过高（例如1.0），可能会导致峰值的两侧未能完全捕获",
    "A rectangular baseline shape is computed based either on the minimal intensity of the peak boundaries, the maximum intensity or the average intensity (base_to_base)": "基于峰值边界的最小强度、最大强度或平均强度（基线到基线）计算一个矩形基线形状",
    "If turned off, will not use fragment ion spectra for scoring": "如果关闭，将不会使用片段离子谱进行评分",
    "If set, tr_irt may be omitted select trafoxml data sets(s)": "如果设置，tr_irt 可以省略选择 trafoxml 数据集（s）",
    "(e.g. to visually inspect a larger area of the chromatogram)": "（例如，为了直观地检查色谱图的更大区域）",
    "Will only attempt to extract each assay from the *best* matching DIA window (instead of all matching windows)": "将仅尝试从每个检测中提取与DIA窗口（而不是所有匹配的窗口）最佳匹配的数据",
    "If you choose cache, make sure to also set tempDirectory": "如果您选择缓存，请确保同时设置临时目录",
    " select qcml data sets(s)": "选择 QCML 数据集（s）",
    " select consensusxml,featurexml,mzq data sets(s)": "选择consensusxml、featurexml、mzq数据集（s）",
    "Tolerance is understood as 'plus or minus x', so the matching range increases by twice the given value": "容差被理解为“正负x”，因此匹配范围增加了给定值的两倍",
    "If 'precursor', the precursor-m/z from the idXML is used. If 'peptide',. masses are computed from the sequences of peptide hits; in this case, an identification matches if any of its hits matches.. ('peptide' should be used together with 'feature:use_centroid_mz' to avoid false-positive matches.)": "如果选择“precursor”，则使用idXML中的前体-m/z。如果选择“peptide”，则根据肽段命中序列计算质量；在这种情况下，如果其任何命中匹配，则认为鉴定匹配（应与“feature:use_centroid_mz”一起使用“peptide”，以避免误报匹配）。",
    "(If you choose 'peptide' as 'mz_reference', you should usually set this flag to avoid false-positive matches.)": "（如果您选择'peptide'作为'mz_reference'，您通常应该设置这个标志以避免误报匹配。）",
    "A consensus feature matches if any of its sub-features matches": "一个共识特征匹配，如果它的任何一个子特征匹配",
    "Please note that this value needs to be set. The default value -1.0 is not allowed": "请注意，这个值需要设置。默认值-1.0是不允许的。",
    " select consensusxml,featurexml,mzml data sets(s)": "选择 consensusxml、featurexml、mzml 数据集（s）",
    " select csv,traml data sets(s)": "选择 CSV、Traml 数据集（s）",
    " select consensusxml,featurexml,idxml,sqlite data sets(s)": "选择 consensusxml、featurexml、idxml、sqlite 数据集（s）",
    " select tabular data sets(s)": "选择表格数据集",
    "If '0', no explicit reference is set - the algorithm will select a reference": "如果为'0'，则不设置显式参考 - 算法将选择一个参考",
    "If left empty, a score type is picked automatically": "如果留空，将自动选择一个分数类型",
    "Unless you have very few runs or identifications, increase this value to focus on more informative peptides": "除非你的运行次数或鉴定结果非常少，否则请增加这个值，以便更专注于提供更多信息的肽段。",
    "reference, if any) in which a peptide must occur to be used for the alignment.. Unless you have very few runs or identifications, increase this value to focus on more informative peptides": "参考（如果有的话），一个肽段必须在其中出现才能用于比对。除非你的运行或鉴定非常少，否则增加这个值以专注于更多信息丰富的肽段。",
    "reference). Peptides with higher shifts (outliers) are not used to compute the alignment.. If 0, no limit (disable filter); if > 1, the final value in seconds; if <= 1, taken as a fraction of the range of the reference RT scale": "参考)。具有更高偏移量的肽段（异常值）不用于计算对齐。如果为0，则没有限制（禁用过滤器）；如果大于1，则为最终值，单位为秒；如果小于或等于1，则作为参考保留时间（RT）范围的分数。",
    "If 'false', only peptide IDs assigned to features will be used": "如果为“false”，则仅使用分配给特征的肽段ID",
    "If different identifications are matched to one feature, only the peptide closest to the centroid in RT is used.. Precludes 'use_unassigned_peptides'": "如果不同的鉴定结果匹配到一个特征上，只有最接近保留时间（RT）中心的肽段被使用。这排除了使用“use_unassigned_peptides”选项。",
    "'y + x', instead of on 'y' vs. 'x'": "“y + x”，而不是“y”与“x”",
    "The number is chosen so that the spline approximates a low-pass filter with this cutoff wavelength. The wavelength is given in the same units as the data; a higher value means more smoothing. '0' sets the number of nodes to twice the number of input points": "所选数值是为了使样条曲线近似于具有此截止波长的低通滤波器。波长以与数据相同的单位给出；较高的值意味着更多的平滑。'0'将节点数设置为输入点数的两倍。",
    "Overrides 'wavelength' if set (to two or greater). A lower value means more smoothing": "如果设置了'wavelength'（为2或更大），则覆盖它。较低的值意味着更平滑",
    "'linear': Linear extrapolation using the slope of the B-spline at the corresponding endpoint. 'b_spline': Use the B-spline (as for interpolation). 'constant': Use the constant value of the B-spline at the corresponding endpoint. 'global_linear': Use a linear fit through the data (which will most probably introduce discontinuities at the ends of the data range)": "线性：使用相应端点处的B样条斜率进行线性外推。B样条：使用B样条（如插值）。常数：使用相应端点处的B样条的常数值。全局线性：使用通过数据的线性拟合（这可能会在数据范围的末端引入不连续性）",
    "Choosing this parameter in the range .2 to .8 usually results in a good fit": "选择这个参数在0.2到0.8的范围内通常能得到一个良好的拟合",
    "e.g. for data ranging from 1000 seconds to 2000 seconds, it could be set to 10). Setting a negative value will automatically do this": "例如，对于范围在1000秒到2000秒之间的数据，可以将其设置为10。设置一个负值将自动执行此操作。",
    "'linear': Linear interpolation. 'cspline': Use the cubic spline for interpolation. 'akima': Use an akima spline for interpolation": "'linear': 线性插值。'cspline': 使用三次样条进行插值。'akima': 使用Akima样条进行插值。",
    "'two-point-linear': Uses a line through the first and last point to extrapolate. 'four-point-linear': Uses a line through the first and second point to extrapolate in front and and a line through the last and second-to-last point in the end. 'global-linear': Uses a linear regression to fit a line through all data points and use it for interpolation": "'two-point-linear': 使用通过第一和最后一点的线条进行外推。'four-point-linear': 使用通过第一和第二点的线条进行前方外推，以及通过最后和倒数第二点的线条在末端进行外推。'global-linear': 使用线性回归拟合通过所有数据点的线条，并用它进行插值。",
    "Note that global-linear may not be continuous at the border": "请注意全局线性在边界处可能不连续",
    "Setting to \"auto\" uses HCD and HCID spectra. Set to empty string if you want to disable filtering": "设置为“auto”使用HCD和HCID谱图。如果你想禁用过滤，就设置为空字符串",
    "MS/MS scans having a precursor with a lower intensity will not be considered for quantitation": "具有较低强度前体离子的MS/MS扫描将不被考虑进行定量分析",
    "If set to false, only the precursor scan will be used": "如果设置为false，将只使用前体扫描",
    "e.g. '0/0.3/4/0', '0.1/0.3/3/0.2' (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如 '0/0.3/4/0', '0.1/0.3/3/0.2'（用空格分隔的列表，如果列表项中包含空格，则用单引号将其包围）",
    "Please note that 120 is not valid": "请注意，120是无效的。",
    "Note that you need to provide a correct isotope correction matrix otherwise the tool will fail or produce invalid results": "请注意，您需要提供一个正确的同位素校正矩阵，否则该工具将失败或产生无效结果。",
    "The normalization is done by using the Median of Ratios (every channel / Reference). Also the ratio of medians (from any channel and reference) is provided as control measure!": "标准化是通过使用比率中位数（每个通道/参考）来完成的。同时，还提供了中位数的比率（来自任何通道和参考）作为控制度量！",
    "(e.g. by how many spectra the peak may be shifted at most). This parameter will depend on your chromatographic setup but a number between 1 and 3 is usually sensible": "（例如，峰值可能移动的光谱数）。此参数将取决于您的色谱设置，但通常在1到3之间的数字是合理的。",
    "This is a hard parameter, all profiles further away will not be considered at all": "这是一个硬参数，所有更远的配置文件将完全不被考虑",
    "All parameters of this take precedence. A template file can be generated using 'comet.exe -p' select txt data sets(s)": "所有参数将优先使用。可以使用 'comet.exe -p' 选择 txt 数据集来生成模板文件。",
    "See also precursor_error_units to set the unit": "另见precursor_error_units以设置单位",
    "Use -8/-4/0/4/8 only for SILAC": "仅将-8/-4/0/4/8用于SILAC（稳定同位素标记的氨基酸细胞培养）",
    "Thus, the value should be a bit higher than for other search engines, since the bin might not be centered around the peak apex (see 'fragment_bin_offset').CAUTION: Low tolerances have heavy impact on RAM usage (since Comet uses a lot of bins in this case). Consider using use_sparse_matrix and/or spectrum_batch_size": "因此，这个值应该比其他搜索引擎的值稍微高一些，因为bin可能不会围绕峰值顶点（参见'fragment_bin_offset'）居中。警告：低公差对RAM使用有重大影响（因为Comet在这种情况下使用了很多bin）。考虑使用use_sparse_matrix和/或spectrum_batch_size。",
    "Recommended by Comet: low-res: 0.4, high-res: 0.0": "Comet推荐：低分辨率：0.4，高分辨率：0.0",
    "It has no effect if enzyme is unspecific cleavage": "如果酶是非特异性切割，则它没有效果",
    "For unknown charge states, search as singly charged if there is no signal above the precursor m/z or use the precursor_charge range": "对于未知的电荷状态，如果没有前体 m/z 以上的信号，则将其作为单电荷进行搜索，或者使用前体电荷范围。",
    "(Allowed max 5)": "（允许的最大值是5）",
    "Indices correspond to the binary modification index used by comet to group individually searched lists of variable modifications.. Note: if set, both variable_modifications and binary_modifications need to have the same number of entries as the N-th entry corresponds to the N-th variable_modification.. if left empty (default), all entries are internally set to 0 generating all permutations of modified and unmodified residues.. For a detailed explanation please see the parameter description in the Comet help (space separated list, in order to allow for spaces in list items surround them by single quotes)": "索引对应于 Comet 用于将单独搜索的可变修饰列表分组的二进制修饰索引。注意：如果设置，variable_modifications 和 binary_modifications 都需要具有与 N-th entry 对应的 N-th variable_modification 相同数量的条目。如果留空（默认），所有条目内部都设置为 0，生成所有修饰和未修饰残基的排列。有关详细说明，请参见 Comet 帮助中的参数描述（空格分隔的列表，为了允许列表项中有空格，请用单引号将它们包围）。",
    "number of spectra to search at a time; use 0 to search the entire scan range in one batch": "同时搜索的谱图数量；使用0表示在一个批次中搜索整个扫描范围",
    "Has to include 0.0 if you want the default mass to be searched (space separated list, in order to allow for spaces in list items surround them by single quotes)": "如果需要搜索默认的质量，则必须包含0.0（用空格分隔的列表，如果列表项中包含空格，则用单引号将它们包围起来）",
    "phosphate_loss = remove the HPO3 (-80) and H3PO4 (-98) precursor phosphate neutral loss peaks. See also remove_precursor_tolerance": "磷酸盐丢失 = 移除HPO3（-80）和H3PO4（-98）前体磷酸盐中性丢失峰。另见remove_precursor_tolerance",
    "e.g. from the precursor peaks of the ETD spectrum.. The input is believed otherwise": "例如，来自ETD谱图的前体峰。输入被认为否则。",
    " select idxml,sqlite data sets(s)": "选择 idxml, sqlite 数据集（s）",
    "IDs from 'in' are added to this file, but only if the (modified) peptide sequences are not present yet (considering only best hits per spectrum) select idxml data sets(s)": "从“in”文件中添加ID，但仅当（修改后的）肽段序列尚未存在时（仅考虑每个谱图的最佳命中），选择idxml数据集",
    "Exactly two input files are expected in this case. Not compatible with 'add_to'": "在这种情况下，预计会有正好两个输入文件。与'add_to'不兼容",
    "Not compatible with 'add_to'": "不兼容 'add_to'",
    "If empty, from all will be exported (space separated list, in order to allow for spaces in list items surround them by single quotes)": "如果为空，将导出所有内容（以空格分隔的列表，如果列表项中包含空格，请用单引号将它们包围起来）",
    "The first row is considered containing the headers as for the exported the table. The second row is considered the according qp cv accessions of the qp to be exported select csv data sets(s)": "第一行被认为包含表头，就像导出的表格一样。第二行被认为包含要导出的qp的相应qp cv访问号，选择csv数据集。",
    "intensity of one ('to_one') per spectrum": "每个谱图的强度（'to_one'）",
    "Does not affect calibrant collection (space separated list, in order to allow for spaces in list items surround them by single quotes)": "不影响校准剂收集（以空格分隔的列表，如果列表项中有空格，请用单引号将它们包围起来）",
    "i.e. left->center, or center->right) around an MS scan in which calibrants are collected to build a model. Set to -1 to use ALL calibrants for all scans, i.e. a global model": "即左->中心，或中心->右，在其中收集校准剂以构建模型的质谱扫描周围。设置为-1以使用所有扫描的所有校准剂，即全局模型。",
    " select featurexml,idxml data sets(s)": "选择 featurexml、idxml 数据集(s)",
    " select csv data sets(s)": "选择 CSV 数据集（s）",
    "i.e. not the iso1, iso2 etc ('charge' column is used to determine the spacing). Peaks which are not mono-isotopic are not used": "即不是iso1、iso2等（使用“电荷”列来确定间距）。非单一组分的峰不使用",
    "Peaks without isotope pattern are not used": "没有同位素模式的峰不被使用",
    "For LC data 'area' is recommended, 'median' for direct injection data. 'max_height' simply uses the most intense peak in the trace": "对于液相色谱（LC）数据，推荐使用“面积”（'area'），对于直接进样数据，则推荐使用“中位数”（'median'）。“最大高度”（'max_height'）则是简单地使用色谱图中最强烈的峰值。",
    "In 'outlier' mode, trace extension cancels if a predefined number of consecutive outliers are found (see trace_termination_outliers parameter). In 'sample_rate' mode, trace extension in both directions stops if ratio of found peaks versus visited spectra falls below the 'min_sample_rate' threshold": "在“异常值”模式下，如果发现连续的异常值数量达到预设值（参见trace_termination_outliers参数），则取消迹线扩展。在“采样率”模式下，如果找到的峰值与访问的光谱之比低于“min_sample_rate”阈值，则在两个方向上都停止迹线扩展。",
    "Set to a negative value to disable maximal length check during mass trace detection": "设置为负值以在质量迹线检测期间禁用最大长度检查",
    "The fixed setting filters out mass traces outside the [min_fwhm, max_fwhm] interval (set parameters accordingly!). The auto setting filters with the 5 and 95% quantiles of the peak width distribution": "固定设置会过滤掉质量痕迹在[min_fwhm, max_fwhm]区间之外的部分（请相应地设置参数！）。自动设置会使用峰宽分布的5%和95%分位数进行过滤。",
    "Ignored if parameter width_filtering is off or auto": "如果参数width_filtering关闭或设置为自动，则忽略",
    "e.g., in TOPPAS": "例如，在TOPPAS中",
    "Keeping it can increase resulting FASTA file significantly": "保留它可以显著增加生成的FASTA文件的大小",
    "Set to -1.0 to use model's default setting)": "设置为-1.0以使用模型的默认设置)",
    "Leave empty for using the same DB as used for the input idXML (this might fail). Non-existing relative filenames are looked up via 'OpenMS.ini:id_db_dir' select fasta data sets(s)": "留空以使用与输入idXML相同的数据库（这可能会失败）。不存在的相对文件名将通过'OpenMS.ini:id_db_dir'查找fasta数据集",
    " select consensusxml,featurexml,idxml,mzml data sets(s)": "选择 consensusxml、featurexml、idxml、mzml 数据集（s）",
    "This option is ignored for MS level 1": "这个选项对于质谱级别1被忽略",
    " select mrm,pqp,tabular,traml data sets(s)": "选择 mrm、pqp、tabular、traml 数据集（s）",
    " select xml data sets(s)": "选择 XML 数据集（s）",
    "Note: Requires setting 'unimod_file'": "注意：需要设置 'unimod_file'",
    "(i.e. across whole precursor isolation window instead of precursor MZ) should be generated": "（即在整个前体隔离窗口中而不是前体m/z）应该生成",
    "If featureXML from reference file is used it should be disabled (set to -1) as no mono-isotopic peak is expected to be present": "如果使用了参考文件中的featureXML，应该将其禁用（设置为-1），因为预计不会有单同位素峰出现。",
    "Positive values filter all RIAs passing the correlation threshold but that also show a better correlation to an averagine peptide. Disabled for values <= -1": "正值过滤所有通过相关性阈值的相对丰度指数分析（RIAs），但同时也显示出与平均蛋白质肽段更好的相关性。对于值 <= -1 的情况，该功能被禁用。",
    "Values smaller than zero usually only make sense if the observed peak fraction is set to 0": "值小于零通常只有在观察到的峰值分数设置为0时才有意义。",
    "Disable and take correlation_threshold value for negative values": "禁用并为负值取correlation_threshold的值",
    "(e.g. from ProteinProphet via IDFileConverter or Fido via FidoAdapter).. Information about indistinguishable proteins will be used for protein quantification select idxml data sets(s)": "（例如通过ProteinProphet使用IDFileConverter或通过Fido使用FidoAdapter）... 用于蛋白质定量的不可区分蛋白质的信息将用于选择idxml数据集（s）",
    "For peptides, abundances will be reported separately for each fraction and charge;. for proteins, abundances will be computed based only on the most prevalent charge observed of each peptide (over all fractions).. By default, abundances are summed over all charge states": "对于肽段，每个分数和电荷的丰度将分别报告；对于蛋白质，丰度将基于观察到的每个肽段的最常见电荷（在所有分数中）计算。默认情况下，丰度会将所有电荷状态求和。",
    "(Only works with an idXML file given as protein_groups parameter)": "（仅适用于作为蛋白质组参数提供的idXML文件）",
    "Format: log_2(x_0/x_0) <sep> log_2(x_1/x_0) <sep> log_2(x_2/x_0)": "格式：log_2(x_0/x_0) <sep> log_2(x_1/x_0) <sep> log_2(x_2/x_0)",
    "Only applicable to consensus maps of exactly three sub-maps. Format: log_2(heavy/light) <sep> log_2(heavy/middle) <sep> log_2(middle/light)": "仅适用于恰好有三个子图的共识图。格式：log_2（重/轻）<sep> log_2（重/中）<sep> log_2（中/轻）",
    "With 'top 0',all peptides that occur in every sample are considered.. Otherwise ('top N'), the N peptides that occur in the most samples (independently of each other) are selected,. breaking ties by total abundance (there is no guarantee that the best co-ocurring peptides are chosen!)": "使用“top 0”，所有在每个样本中都出现的肽段都会被考虑。否则（'top N'），选择在最多样本中出现的N个肽段（彼此独立），通过总丰度来打破平局（不能保证选择的最佳共现肽段！）。",
    "'escape' for quoting with backslash-escaping of embedded quotes": "使用反斜杠转义的引号进行引用",
    "i.e. not stripped) sequence and charge state. The feature with the 'highest intensity' is very likely the most reliable one. When switched on, the filter removes the sequence annotation from the lower intensity features, thereby resolving the multiplicity. Only the most reliable features for each (possibly modified i.e. not stripped) sequence maintain annotated with this peptide sequence": "即未剥离的序列和电荷状态。具有“最高强度”的特征很可能是最可靠的一个。当打开时，过滤器会移除低强度特征的序列注释，从而解决多重性问题。每个（可能经过修饰，即未剥离）序列的最可靠特征仅使用此肽序列进行注释。",
    "If it is 1 the centroid position corresponds to the position of the highest intensity": "如果它是1，则质心位置对应于最高强度的位置",
    "Attention: when this flag is set, the peak_width is ignored": "注意：当设置了这个标志时，peak_width 被忽略",
    "All peaks with width smaller than fwhm_bound_factor * peak_width are discarded": "所有宽度小于 fwhm_bound_factor * peak_width 的峰值都被丢弃",
    "All peaks with width greater than fwhm_upper_bound_factor * peak_width are discarded": "所有宽度大于 fwhm_upper_bound_factor * peak_width 的峰值都被丢弃",
    "If a peak has a lower correlation it is skipped": "如果一个峰值的相关性较低，则会被跳过",
    "Note that the accuracy of the picked peak's centroid position depends in the Raw data spacing, i.e., 50% of raw peak distance at most": "请注意，选择的峰值的质心位置的准确性取决于原始数据的间距，即最多为原始峰值距离的50%。",
    "The initial value is used for charge 1, for higher charges it is adapted to scaling/charge": "初始值用于电荷1，对于更高的电荷，它被调整以适应缩放/电荷",
    "By default, it will be calculated automatically (see auto_mode). Only provide this parameter if you know what you are doing (and change 'auto_mode' to '-1')! All intensities EQUAL/ABOVE 'max_intensity' will not be added to the histogram. If you choose 'max_intensity' too small, the noise estimate might be too small as well. If chosen too big, the bins become quite large (which you could counter by increasing 'bin_count', which increases runtime)": "默认情况下，它将自动计算（见auto_mode）。只有在你知道自己在做什么时才提供此参数（并将'auto_mode'更改为'-1'）！所有强度等于或高于'max_intensity'的值将不会被添加到直方图中。如果你选择的'max_intensity'太小，噪声估计可能也太小。如果选择得太大，箱子会变得相当大（你可以通过增加'bin_count'来抵消这种情况，这会增加运行时间）",
    "Attention: this can blow up files, since seven arrays are stored per spectrum!": "注意：这可能会使文件体积增大，因为每个谱图存储了七个数组！",
    "Only one of 'ratio' or 'absdiff' has to be satisfied.  Use \"absdiff\" to deal with cases like \"zero vs. epsilon\"": "只有 'ratio' 或 'absdiff' 之一需要满足。使用 \"absdiff\" 来处理像 \"零与epsilon\" 这样的案例。",
    "Only one of 'ratio' or 'absdiff' has to be satisfied.": "只需要满足“ratio”或“absdiff”中的一个。",
    "0 = very quiet mode (absolutely no output). 1 = quiet mode (no output unless differences detected). 2 = default (include summary at end). 3 = continue after errors.": "0 = 非常安静的模式（绝对没有输出）。1 = 安静的模式（除非检测到差异，否则没有输出）。2 = 默认（在最后包括摘要）。3 = 在错误后继续。",
    "Input is given as list of colon separated tuples, e.g. String1:String2 String3:String4 (space separated list, in order to allow for spaces in list items surround them by single quotes)": "输入以冒号分隔的元组列表形式给出，例如 String1:String2 String3:String4（为了允许列表项中有空格，可以用单引号将它们包围起来）",
    "In conjunction with check_width_internally it will also be used to remove peaks whose spacing is larger than this value": "结合check_width_internally，它还将用于移除间距大于此值的峰值。",
    "The higher this value is set, the further apart peaks are allowed to be to still extend a peak. E.g. if the value is set to 1.5 and in a current peak the minimal spacing between peaks is 10 mDa, then only peaks at most 15 mDa apart will be added to the peak": "这个值设置得越高，峰值被允许的距离就越远，仍然可以扩展一个峰。例如，如果该值设置为1.5，并且在当前峰中峰值之间的最小间距为10毫道尔顿（mDa），那么最多只有相距15毫道尔顿的峰会被添加到峰中。",
    "i.e. observing q1=[5,6,7] implies span=3. Setting this to 1 will only find adduct variants of the same charge": "即观察到 q1=[5,6,7] 意味着跨度为3。将其设置为1将只找到相同电荷的加合变体。",
    "i.e. the number of '+' or '-' indicate the charge, e.g. 'Ca:++:0.5' indicates +2. Probabilites have to be in (0,1]. RTShift param is optional and indicates the expected RT shift caused by this adduct, e.g. '(2)H4H-4:0:1:-3' indicates a 4 deuterium label, which causes early elution by 3 seconds. As a fifth parameter you can add a label which is tagged on every feature which has this adduct. This also determines the map number in the consensus file (space separated list, in order to allow for spaces in list items surround them by single quotes)": "即 '+' 或 '-' 的数量表示电荷，例如 'Ca:++:0.5' 表示 +2。概率必须在 (0,1] 范围内。RTShift 参数是可选的，表示由此加成物引起的预期 RT 偏移，例如 '(2)H4H-4:0:1:-3' 表示一个 4 氘标记，导致提前 3 秒洗脱。作为第五个参数，您可以添加一个标签，该标签标记具有此加成物的每个特征。这也决定了共识文件中的映射编号（用空格分隔的列表，为了允许列表项中有空格，请将它们用单引号包围）。",
    "Add them in the 'potential_adducts' section!": "将它们添加到“潜在加合物”部分！",
    "E.g. setting this to 2 will not allow an adduct composition of '1(H+),3(Na+)' if Na+ is the least probable adduct": "例如，将此设置为2将不允许如果Na+是最不可能的加成物，则加成物组成为'1(H+),3(Na+)'",
    "It is not used for features of different charge": "它不用于不同电荷的特征",
    " select consensusxml,dta,dta2d,edta,featurexml,kroenik,mgf,ms2,mzdata,mzml,mzxml,peplist,sqlite,tabular,thermo.raw data sets(s)": "选择 consensusxml、dta、dta2d、edta、featurexml、kroenik、mgf、ms2、mzdata、mzml、mzxml、peplist、sqlite、tabular、thermo.raw 数据集（s）",
    "Note: that not all conversion paths work or make sense": "注意：并非所有转换路径都有效或有意义",
    "'none' keeps current IDs even if invalid.. 'ensure' keeps current IDs but reassigns invalid ones.. 'reassign' assigns new unique IDs": "'none' 保持当前的ID，即使它们是无效的。'ensure' 保留当前的ID，但重新分配无效的ID。'reassign' 分配新的唯一的ID。",
    "Some external tools might rely on it": "一些外部工具可能会依赖它",
    "(e.g. use 0.0001 for a mass accuracy of 0.2 ppm at 500 m/z, default uses -1.0 for maximal accuracy)": "（例如，使用0.0001表示在500 m/z时的质荷比精度为0.2 ppm，如果使用默认值-1.0，则表示最大精度）",
    "Note: this flag will prevent conversion from spectra to chromatograms": "注意：这个标志将阻止从光谱转换为色谱图",
    " select featurexml,mzml data sets(s)": "选择 featureXML、mzML 数据集（s）",
    "To use all, set to '-1'": "要使用全部，设置为 '-1'",
    "This condition applies to the pairs considered in hashing": "这个条件适用于在哈希中考虑的配对",
    "(i.e., max - min).": "（即，最大值 - 最小值）。",
    "Use this to reduce the running time and to disregard weak signals during alignment.  For using all points, set this to -1": "使用此选项可以减少运行时间，并在比对过程中忽略弱信号。若要使用所有点，请将此选项设置为-1。",
    "A good choice for this would be a bit smaller than the error you would expect from repeated runs": "一个好的选择是比重复运行时预期的错误略小一些",
    "A good choice for this would be about the time between consecutive MS scans": "一个好的选择是连续质谱扫描之间的时间",
    "This applies for both directions": "这适用于两个方向",
    "The minimal scaling is the reciprocal of this": "最小缩放是这个的倒数",
    "A serial number for each invocation will be appended automatically": "每个调用将自动附加一个序列号",
    "If disabled, d = |int_f2 - int_f1| / int_max. If enabled, d = |log(int_f2 + 1) - log(int_f1 + 1)| / log(int_max + 1))": "如果禁用，d = |int_f2 - int_f1| / int_max。如果启用，d = |log(int_f2 + 1) - log(int_f1 + 1)| / log(int_max + 1)|",
    "The file is only used to extract the run name from the file name select mzml data sets(s)": "该文件仅用于从文件名中提取运行名称，选择mzml数据集（s）",
    " select png data sets(s)": "选择 PNG 数据集（s）",
    " select consensusxml,idxml data sets(s)": "选择 consensusxml 和 idxml 数据集（s）",
    "Also adds the resolved ambiguity groups to output": "同时将解析后的歧义组添加到输出中",
    "0 considers all": "0 考虑所有",
    "Discard the rest (true) or keepe.g. for quantification/reporting?": "丢弃其余部分（true）或保留，例如用于定量/报告？",
    "Negative values enable grid search for this param": "负值将启用此参数的网格搜索",
    "Usually much smaller than emission from proteins. Negative values enable grid search for this param": "通常比蛋白质的辐射要小得多。负值可以为这个参数启用网格搜索",
    "fifo = first in first out. subtree = message passing follows a random spanning tree in each iteration": "先进先出（FIFO）是一种数据结构，其中第一个进入的元素将是第一个出来的元素。子树是指在每次迭代中，消息传递遵循随机生成的生成树。",
    "0 = new message overwrites old completely (no dampening; only recommended for trees),0.5 = equal contribution of old and new message (stay below that),In-between it will be a convex combination of both. Prevents oscillations but hinders convergence": "0 = 新消息完全覆盖旧消息（无衰减；仅推荐用于树），0.5 = 旧消息和新消息的等量贡献（保持在该值以下），介于两者之间将是两者的凸组合。防止振荡但阻碍收敛",
    "If not all messages converge, how many iterations should be done at max per connected component?": "如果并非所有消息都收敛，那么对于每个连通分量，最多应该进行多少次迭代？",
    "1 == sum-product inference (all configurations vote equally) (default),<= 0 == infinity = max-product inference (only best configurations propagate)The higher the value the more important high probability configurations get": "1 == 乘积-和推理（所有配置的投票权重相同）（默认），<= 0 == 无穷大 = 最大乘积推理（只有最佳配置传播）。值越高，高概率配置的重要性就越大。",
    "0 = maximize calibration only, 1 = maximize AUC only, between = convex combination": "0 = 仅最大化校准，1 = 仅最大化AUC，介于两者之间 = 凸组合",
    "Leave blank for auto-detection": "留空以自动检测",
    "Ignored during auto-detection": "在自动检测期间被忽略",
    "Closer data points or peaks will be merged": "更接近的数据点或峰值将被合并",
    "All other spectra remain unchanged": "所有其他谱图保持不变",
    "The Gaussian RT profile decreases from 1 at its apex to 0 at infinity. Spectra for which the intensity of the Gaussian drops below the cutoff do not contribute to the average": "高斯RT轮廓从其顶点的1递减到无穷大的0。高斯强度低于截止值的谱图不会对平均值产生贡献",
    "i.e. +/-(RT range)/2 from each spectrum": "即每个谱图的正负（RT范围）/2",
    "All spectra with other MS levels remain untouched (space separated list, in order to allow for spaces in list items surround them by single quotes)": "所有其他质谱级别的谱图保持不变（以空格分隔的列表，如果列表项中包含空格，则用单引号将它们包围起来）",
    "Otherwise performs inference separately per run": "否则，分别对每次运行进行推断。",
    "Use at your own risk": "自行承担风险使用",
    "If set to zero, unmatched proteins get a score of -Infinity. If bigger than zero, proteins with less peptides are filtered and evidences removed from the PSMs. PSMs that do not reference any proteins anymore are removed but the spectrum info is kept": "如果设置为零，未匹配的蛋白质将获得一个负无穷大的分数。如果大于零，肽段较少的蛋白质将被过滤掉，并且证据将从PSMs中移除。不再引用任何蛋白质的PSMs将被移除，但保留谱图信息。",
    "Note: shared_peptides are not deleted and potentially resolved in postprocessing as well": "注意：共享肽段不会被删除，也可能在后处理中得到解决。",
    "(i.e. become potentially quantifiable razor peptides)": "（即成为潜在可量化的剃刀肽）",
    "See http://www.openms.de/current_doxygen/html/UTILS_TargetedFileConverter.html for format of OpenSWATH transition TSV file or SpectraST MRM file select mrm,pqp,tabular,traml data sets(s)": "请参考 http://www.openms.de/current_doxygen/html/UTILS_TargetedFileConverter.html 以获取 OpenSWATH 转换 TSV 文件或 SpectraST MRM 文件的格式，选择 mrm,pqp,tabular,traml 数据集。",
    "Note: not all conversion paths work or make sense": "注意：并非所有的转换路径都是可行的或者有意义的",
    "Set to -1 to omit meta values (default)": "设置为-1以省略元值（默认）",
    "group information from an idXML file": "从 idXML 文件中获取组信息",
    "The precedence is: sort_by_size, sort_by_maps, sorting_method": "优先级是：sort_by_size, sort_by_maps, sorting_method",
    "(i.e., the number of elements)": "（即，元素的数量）",
    " select tabular,txt data sets(s)": "选择表格、文本数据集（s）",
    "These can be either two or three constants. per set, depending on the conversion type. Either one set for all calibrant spectra . (tab separated), or one for each spectrum.. For a detailed description, please have a look at the doxygen documentation.(one set, tab separated, per line) select tabular data sets(s)": "这些可以是两个或三个常数。每组，取决于转换类型。要么为所有校准光谱设置一个组（用制表符分隔），要么为每个光谱设置一个组。有关详细描述，请查看doxygen文档。（每行一组，用制表符分隔）选择表格数据集",
    " select consensusxml,featurexml,idxml,mzdata,mzid,mzml,mzxml,pepxml,traml,xml data sets(s)": "选择共识XML、特征XML、ID XML、mzData、mzID、mzML、mzXML、pepXML、TraML、XML数据集（s）",
    "If no schema is given, the file is validated against the latest schema of the file type select xml data sets(s)": "如果未指定模式，则文件将根据所选文件类型的最新模式进行验证，选择XML数据集（s）",
    "All parameter names are chosen as in the libSVM library. Please refer to libSVM documentation for explanation": "所有参数名称都选择为libSVM库中的名称。请参考libSVM文档以获取解释。",
    "(Trypsin is default, None would be best for peptide input or unspecific digestion, for more please refer to your mascot server)": "（胰蛋白酶是默认的，对于肽段输入或非特异性消化，\"None\"可能是最佳选择，更多信息请参考您的Mascot服务器）",
    "Note: Some server require that a proper email is provided": "注意：有些服务器要求提供有效的电子邮件地址",
    "e.g. 'mascot-server' or '127.0.0.1'": "例如 'mascot-server' 或 '127.0.0.1'",
    "Some Mascot servers freeze during this (unstable network etc) and idle forever, the connection is killed. Set this to 0 to disable timeout!": "一些Mascot服务器在这种情况下会冻结（例如网络不稳定等），并且会永远空闲，连接会被终止。将此设置为0以禁用超时！",
    "Requires OpenSSL to be installed (see openssl.org)": "需要安装OpenSSL（参见openssl.org）",
    "Generally only parameters that control which hits to export are safe to adjust/add. Many settings that govern what types of information to include are required by OpenMS and cannot be changed. Note that setting 'query_master' to 1 may lead to incorrect protein references for peptides": "通常只有控制导出哪些命中的参数才是安全的调整/添加。许多控制包含哪些类型的信息的设置是由OpenMS所需的，不能更改。请注意，将'query_master'设置为1可能会导致肽段的蛋白质引用不正确。",
    "The output file produced by MascotAdapterOnline will contain only the Mascot search number": "MascotAdapterOnline 生成的输出文件将只包含 Mascot 搜索编号",
    " select dta,mgf,msp,mzdata,mzml,mzxml data sets(s)": "选择 dta、mgf、msp、mzdata、mzml、mzxml 数据集（s）",
    " select splib data sets(s)": "选择 splib 数据集（s）",
    "Monoisotopic mass is assumed": "单同位素质量被假定",
    "i.e., ignore specified charge state (if any) of the query spectrum": "即，忽略查询谱图（如果有的话）的特定电荷状态",
    "Default is \"spectrast.usermods\" select txt data sets(s)": "默认是“spectrast.usermods”选择文本数据集（s）",
    " select mzdata,mzid,mzml,traml,xml data sets(s)": "选择 mzdata、mzid、mzml、traml、xml 数据集（s）",
    " select obo data sets(s)": "选择 OBO 数据集（s）",
    " select msp data sets(s)": "选择msp数据集（s）",
    "(E.g.: 1 = prec. may be misassigned to first isotopic peak) (space separated list, in order to allow for spaces in list items surround them by single quotes)": "（例如：1 = 精确度可能被错误地分配给第一个同位素峰） （用空格分隔的列表，为了允许列表项中有空白，在它们周围用单引号括起来）",
    "Most are especially useful when the query spectra are raw": "大多数在查询谱图是原始数据时特别有用",
    "Default equals all peaks which are lower than 0.001 of the maximum intensity peak": "默认等于所有峰值低于最大强度峰值的0.001",
    "e.g. 'Oxidation (M)'": "例如 '氧化（M）'",
    "A gap means that one spectrum can not be aligned directly to another spectrum in the Map. This happens, when the similarity of both spectra a too low or even not present. Imagine it as a insert or delete of the spectrum in the map (similar to sequence alignment). The gap is necessary for aligning, if we open a gap there is a possibility that an another spectrum can be correct aligned with a higher score as before without gap. But to open a gap is a negative event and needs to carry a punishment, so a gap should only be opened if the benefits outweigh the downsides. The Parameter is to giving as a positive number, the implementation convert it to a negative number": "一个间隔表示一个谱图不能直接与地图中的另一个谱图对齐。当两个谱图的相似性太低或甚至不存在时，就会发生这种情况。可以想象成在地图中插入或删除了一个谱图（类似于序列比对）。间隔对于对齐是必要的，如果我们打开一个间隔，就有可能使另一个谱图能够以比没有间隔时更高的分数正确对齐。但是打开一个间隔是一个负面事件，需要受到惩罚，因此只有在好处大于坏处时才应该打开间隔。参数是一个正数，实现时将其转换为负数。",
    "The idea behind the affine gapcost lies under the assumption, that it is better to get a long distance of connected gaps than to have a structure of gaps interspersed with matches (gap match gap match etc.). Therefore the punishment for the extension of a gap generally should be lower than the normal gapcost. If the result of the alignment shows high compression, it is a good idea to lower either the affine gapcost or gap opening cost": "仿射间隙成本背后的思想基于这样的假设：与具有间隙与匹配交错（间隙匹配间隙匹配等）的结构相比，获得较长距离的连续间隙更好。因此，对间隙延伸的惩罚通常应该低于正常的间隙成本。如果比对的结果显示高压缩，降低仿射间隙成本或间隙开放成本是一个好主意。",
    "Only those pair of spectra are selected, which has a score higher or same of the threshold": "只有那些对的谱图被选中，其得分高于或等于阈值。",
    "It is a quantize of the interval of those points, which defines the main alignment (match points). These points have to filtered, to reduce the amount of points for the calculating a smoother spline curve": "它是那些点的间隔的一个量化，它定义了主要的对齐（匹配点）。这些点需要被过滤，以减少计算更平滑的样条曲线所需的点的数量。",
    "The high score pairs are previously selected. The reduction of match points helps to get a smoother spline curve": "高分对是预先选定的。匹配点的减少有助于获得更平滑的样条曲线。",
    "The success of an alignment depends mostly of the elected score function. The score function return the similarity of two spectra. The score influence defines later the way of possible traceback. There are multiple spectra similarity scores available": "一个比对的成功主要取决于所选的得分函数。得分函数返回两个谱图的相似度。得分影响定义了后续可能的回溯方式。有多种可用的谱图相似度得分。",
    "e.g. '$peptide$,$run$": "例如 '$peptide$,$run$",
    " select mzdata,mzxml data sets(s)": "选择 mzdata 和 mzxml 数据集（s）",
    " select mgf,mzml data sets(s)": "选择 mgf、mzml 数据集（s）",
    "Default: -10.0": "默认值：-10.0",
    "This is done for each feature separately, so that the feature's maximal intensity in a spectrum is set to 1": "这是针对每个特征单独进行的，以便将特征在谱图中的最大强度设置为1。",
    " select featurexml,idxml,mzml data sets(s)": "选择 featurexml、idxml 和 mzml 数据集（s）",
    "It is assumed that the file type is idXML. Alternatively you can provide a .txt file having a sequence and the corresponding rt per line..  select idxml,txt data sets(s)": "假设文件类型为idXML。或者，您可以提供一个.txt文件，每行包含一个序列和相应的保留时间。选择idxml、txt数据集。",
    "to C_SVC for separation prediction).": "（用于C_SVC分离预测）。",
    "with the step size to get the new value": "以步长大小来获得新值",
    "The higher this value, the more local the intensity significance score is.. This parameter should be decreased, if the algorithm is used on small regions of a map": "这个值越高，强度显著性得分就越局部化。如果算法用于地图的小区域，则应降低此参数。",
    "It should be larger than the m/z resolution of the instrument.. This value must be smaller than that 1/charge_high!": "它应该比仪器的m/z分辨率大。这个值必须小于1/charge_high！",
    "This parameter should be well below 'min_spectra'!": "这个参数应该远低于'min_spectra'!",
    "This parameter is important to separate overlapping elution peaks.. It should be increased if feature elution profiles fluctuate a lot": "这个参数对于分离重叠的洗脱峰非常重要。如果特征洗脱轮廓波动很大，则应该增加它。",
    "abundance of the light carbon. Modify if labeled": "轻碳的丰度。如果标记了，则修改",
    "abundance of the light nitrogen. Modify if labeled": "轻氮的丰度。如果标记了，则修改",
    "The seed score is the geometric mean of intensity score, mass trace score and isotope pattern score.. If your features show a large deviation from the averagene isotope distribution or from an gaussian elution profile, lower this score": "种子得分是强度得分、质量迹线得分和同位素模式得分的几何平均值。如果您的特征显示出与平均同位素分布或高斯洗脱曲线有较大偏差，请降低这个得分。",
    "The feature score is the geometric mean of the average relative deviation and the correlation between the model and the observed peaks": "特征分数是模型与观测峰值之间的相关性与平均相对偏差的几何平均值",
    "Traces below this threshold are removed after the model fitting.. This parameter is important for features that overlap in m/z dimension": "在此阈值以下的迹线在模型拟合后被移除。这个参数对于在m/z维度上重叠的特征非常重要。",
    "If set to symmetric a gauss shape is used, in case of asymmetric an EGH shape is used": "如果设置为对称，则使用高斯形状；在不对称的情况下，使用EGH形状。",
    "'maximum' returns the m/z value of the highest mass trace.. 'average' returns the intensity-weighted average m/z value of all contained peaks.. 'monoisotopic' returns the monoisotopic m/z value derived from the fitted isotope model": "'maximum' 返回最高质量迹线中的 m/z 值。'average' 返回所有包含峰的强度加权平均 m/z 值。'monoisotopic' 返回从拟合的同位素模型中派生的单同位素 m/z 值。",
    "The cutoff is typically a bit lower in this case": "在这种情况下，截止值通常稍低一些",
    "0 = disable search for RNA variants": "0 = 禁用搜索RNA变体",
    "(disabled for empty sequence)": "（用于空序列时禁用）",
    "e.g. A=C10H14N5O7P, ..., U=C10H14N5O7P, X=C9H13N2O8PS  where X represents e.g. tU .  or e.g. Y=C10H14N5O7PS where Y represents tG (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如 A=C10H14N5O7P, ..., U=C10H14N5O7P, X=C9H13N2O8PS 其中 X 代表例如 tU。或者例如 Y=C10H14N5O7PS 其中 Y 代表 tG（以空格分隔的列表，为了允许列表项中有空格，将它们用单引号包围）",
    "e.g.: AU CG) (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如：AU CG（用空格分隔的列表，如果列表项中有空格则用单引号包围它们）",
    "'CATG' if C, A, G, and T form cross-links": "如果C、A、G和T形成交叉链接，则为'CATG'",
    "e.g., 'U:C9H10N2O5;U-H3PO4' or 'U:U-H2O->C9H11N2O8P1;U-H2O', (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如，'U:C9H10N2O5;U-H3PO4' 或 'U:U-H2O->C9H11N2O8P1;U-H2O'，（以空格分隔的列表，如果列表项中包含空格，则用单引号将它们包围起来）",
    "(e.g. completely labeled U13)": "（例如，完全标记的U13）",
    " select consensusxml,dta,dta2d,edta,featurexml,kroenik,mgf,ms2,mzdata,mzml,mzxml,peplist,tabular data sets(s)": "选择 consensusxml、dta、dta2d、edta、featurexml、kroenik、mgf、ms2、mzdata、mzml、mzxml、peplist、表格数据集（s）",
    "Remember that isoforms count as different proteins!": "请记住，异构体被视为不同的蛋白质！",
    "Usually used in combination with 'delete_unreferenced_peptide_hits'": "通常与'delete_unreferenced_peptide_hits'一起使用",
    "Usually used in combination with 'score:prot' or 'thresh:prot'": "通常与'score:prot'或'thresh:prot'结合使用",
    "All proteins are filtered based on their singleton scores irrespective of grouping. Use in combination with 'delete_unreferenced_peptide_hits' to remove affected peptides": "所有蛋白质都根据它们的singleton分数进行过滤，而不考虑分组。可以与'delete_unreferenced_peptide_hits'结合使用，以去除受影响的肽段。",
    "Performs group level score filtering (including groups of single proteins). Use in combination with 'delete_unreferenced_peptide_hits' to remove affected peptides": "执行组水平的得分筛选（包括单个蛋白质的组）。与'delete_unreferenced_peptide_hits'结合使用，以去除受影响的肽段",
    "All peptides that are not referencing a protein in this file are removed.. All proteins whose accessions are not present in this file are removed select fasta data sets(s)": "所有不引用此文件中蛋白质的肽段都被移除。所有在该文件中没有访问号的蛋白质都被移除，选择fasta数据集（s）。",
    "Only proteins of the provided list are retained (space separated list, in order to allow for spaces in list items surround them by single quotes)": "仅保留提供列表中的蛋白质（以空格分隔的列表，如果列表项中包含空格，则用单引号将它们括起来）。",
    "Use with 'whitelist:ignore_modifications' to only compare by sequence..  select idxml data sets(s)": "使用 'whitelist:ignore_modifications' 仅按序列进行比较。选择 idxml 数据集（s）",
    "All peptides that are referencing a protein in this file are removed.. All proteins whose accessions are present in this file are removed select fasta data sets(s)": "所有引用此文件中蛋白质的肽段都被移除。所有在该文件中存在访问号的蛋白质都被移除，选择的FASTA数据集（或多个数据集）",
    "Only proteins not in the provided list are retained (space separated list, in order to allow for spaces in list items surround them by single quotes)": "只保留不在提供列表中的蛋白质（用空格分隔的列表，在列表项中包含空格时，用单引号将它们包围起来）",
    "Use with 'blacklist:ignore_modifications' to only compare by sequence..  select idxml data sets(s)": "使用 'blacklist:ignore_modifications' 仅通过序列进行比较。选择 idxml 数据集（s）",
    "e.g. [BJXZ] removes ambiguous peptides": "例如，[BJXZ] 移除模糊的肽段",
    "By default missed cleavages are ignored": "默认情况下，未切割的位点会被忽略",
    "For example: 0:1 -> peptides with two or more missed cleavages will be removed,. 0:0 -> peptides with any missed cleavages will be removed": "例如：0:1 -> 移除两个或更多未切割的肽段，0:0 -> 移除任何未切割的肽段",
    "(i.e., PeptideIdentifications) (for n > 0). A spectrum is considered better if it has a higher scoring peptide hit than the other spectrum": "（即，PeptideIdentifications）（对于n > 0）。如果一个谱图有一个比其他谱图得分更高的肽段命中，则认为该谱图更好。",
    "Similar to n_peptide_hits=1, but if there are ties between two or more highest scoring hits, none are kept": "类似于 n_peptide_hits=1，但如果两个或多个最高得分的命中之间存在平局，则都不保留",
    "i.e. <name> 'lt|eq|gt|ne' <value>; the first is the name of meta value, followed by the comparison operator (equal, less, greater, not equal) and the value to compare to. All comparisons are done after converting the given value to the corresponding data value type of the meta value (for lists, this simply compares length, not content!)! (space separated list, in order to allow for spaces in list items surround them by single quotes)": "即 <名称> 'lt|eq|gt|ne' <值>；第一个是元值的名称，后面是比较运算符（等于、小于、大于、不等于）和要比较的值。所有比较都是在将给定值转换为元值对应的数据类型之后进行的（对于列表，这仅仅比较长度，而不是内容！）（以空格分隔的列表，为了允许列表项中有空格，用单引号将它们包围起来）",
    "Required unless 'digest' is set select fasta data sets(s)": "除非设置了'digest'，否则需要选择一个或多个fasta数据集",
    "Can be used instead of 'database'. Sets all 'oligo:...' parameters select sqlite data sets(s)": "可以用作“数据库”的替代。设置所有'oligo:...'参数选择SQLite数据集",
    "Format: 'Element:Charge(+/-)', i.e. the number of '+' or '-' indicates the charge, e.g. 'Ca:++' indicates +2. Only used if 'precursor:use_adducts' is set (space separated list, in order to allow for spaces in list items surround them by single quotes)": "格式：'元素：电荷(+/-)'，即 '+' 或 '-' 的数量表示电荷，例如 'Ca:++' 表示 +2。仅当设置 'precursor:use_adducts' 时使用（以空格分隔的列表，为允许列表项中有空格，用单引号将它们包围）",
    "E.g.: 1 = precursor may be misassigned to the first isotopic peak. Ignored if 'use_avg_mass' is set (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如：1 = 前体可能被错误地分配给第一个同位素峰。如果设置了'use_avg_mass'，则忽略（以空格分隔的列表，为了允许列表项中有空格，请用单引号将它们包围起来）",
    "(e.g. 'mA?' for 'mA'/'Am') based on a-B ions.. This incurs a performance cost because two modifications have to be considered for each case.. Requires a-B ions to be enabled in parameter 'fragment:ions'": "（例如，基于a-B离子的'mA?'表示'mA'或'Am'）。这会产生性能成本，因为必须考虑每种情况的两种修饰。需要在参数'fragment:ions'中启用a-B离子。",
    "(e.g. 'DECOY_'). Leave empty to skip the FDR/q-value calculation": "（例如 'DECOY_'）。留空以跳过FDR/q值计算",
    "The .psq filename should be given, e.g. 'SwissProt.fasta.psq'. If the filename does not end in '.psq' (e.g., SwissProt.fasta) the psq suffix will be added automatically. Non-existing relative file-names are looked up via'OpenMS.ini:id_db_dir' select fasta data sets(s)": ".psq 文件名应该被给出，例如 'SwissProt.fasta.psq'。如果文件名不以 '.psq' 结尾（例如，SwissProt.fasta），psq 后缀将会自动添加。不存在的相对文件名将通过 'OpenMS.ini:id_db_dir' 查找 fasta 数据集。",
    "Note: even when set to 1 OMSSA may report multiple hits with different charge states": "注意：即使设置为1，OMSSA也可能报告具有不同电荷状态的多个命中",
    "If you set this parameter too small (e.g., he=1), this will effectively introduce FDR filtering. Thus, allowing a less stringent FDR during post-processing will nevertheless return the (better) FDR introduced here, since mediocre hits are not even reported": "如果你将这个参数设置得太小（例如，he=1），这将有效地引入FDR过滤。因此，在后处理中允许不太严格的FDR，但仍然会返回这里引入的（更好的）FDR，因为平庸的命中甚至没有被报告。",
    "(1=believe the input file,2=use the specified range)": "（1=相信输入文件，2=使用指定的范围）",
    "(e.g. b1) product ions be searched (1 = no, 0 = yes)": "（例如 b1）是否搜索产物离子（1 = 不搜索，0 = 搜索）",
    "(e.g. y1) be searched (1 = no, 0 = yes)": "（例如 y1）是否搜索（1 = 不搜索，0 = 搜索）",
    "This is the default)": "这是默认值",
    "Chunks with more than 30k spectra will likely cause memory allocation issues with 32bit OMSSA versions (which is usually the case on Windows). To disable chunking (i.e. submit all spectra in one big chunk), set it to '0'": "具有超过30k个谱图的块可能会导致32位OMSSA版本（通常在Windows上是这种情况）的内存分配问题。要禁用分块（即在一个大块中提交所有谱图），请将其设置为'0'。",
    "Provide one for each raw input file select txt data sets(s)": "为每个原始输入文件选择一个txt数据集。",
    " select edta data sets(s)": "选择 EDTA 数据集（s）",
    "Gaussian smoothing filter with this width is applied to TIC": "高斯平滑滤波器，宽度为这个值，应用于总离子流色谱图（TIC）",
    "Higher thesholds will result in less peaks": "更高的阈值将导致较少的峰值",
    "For more details, see http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=MS&termId=MS:1001045&termName=cleavage%20agent%20name": "更多详细信息，请查看 http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=MS&termId=MS:1001045&termName=裂解试剂名称",
    "Setting this parameter to 0 or 1 will reduce that requirement, so that neither terminus or only one terminus of the peptide must match one of the cleavage rules specified in the CleavageRules parameter. This parameter is useful to turn a tryptic digest into a semi-tryptic digest": "将此参数设置为0或1将减少该要求，以便多肽的任一末端或仅有一个末端必须与在CleavageRules参数中指定的裂解规则之一匹配。此参数对于将胰蛋白酶消化转变为半胰蛋白酶消化非常有用。",
    "A missed cleavage is a site within the peptide that matches one of the cleavage rules (refer to CleavageRules). Settings this parameter to some other number will stop generating peptides from a sequence if it contains more than the specified number of missed cleavages": "一个未切割位点是肽链中与切割规则（参考CleavageRules）之一匹配的位点。将此参数设置为其他数字将阻止从包含超过指定数量未切割位点的序列中生成肽链。",
    "The precursors protons are distributed to those ions based on that calculation, with the more basic sequence generally getting more of the protons": "前体质子根据该计算分配给那些离子，通常更基本的序列会获得更多的质子。",
    "To disable this technique, set the value to '0'": "要禁用此技术，请将值设置为'0'",
    "Use with maximum caution!": "请谨慎使用！",
    "'Trained' is based on a log likelihood model (see DOI:10.1021/pr060507u)": "“训练”基于对数似然模型（参见DOI:10.1021/pr060507u）",
    "Higher values increase the number of cleavages. -2 will give no cleavages, +4 almost full cleavage": "更高的值会增加切割次数。-2将不进行切割，+4则几乎完全切割",
    "All possible resulting peptides will be created": "所有可能生成的肽段都将被创建",
    "If 'true', for CE this means that 'CE:lenght_d', 'CE:length_total', 'CE:voltage' have no influence": "如果为“true”，则对于CE（毛细管电泳），这意味着“CE:lenght_d”、“CE:length_total”和“CE:voltage”没有影响",
    "Good presets are 0 for a perfect elution profile, 1 for a slightly distorted elution profile etc... For trapping instruments (e.g. Orbitrap) distortion should be >4": "良好的预设值为0表示完美的洗脱曲线，1表示略微失真的洗脱曲线等。对于捕集仪器（例如 Orbitrap），失真度应大于4。",
    "i.e. the sigma^2 parameter, which is computed using 'value' + rnd_cauchy('variance')": "即 sigma^2 参数，该参数是使用 'value' + rnd_cauchy('variance') 计算得出的",
    "This does not correspond directly to the width in [s]": "这并不直接对应于宽度在 [s]",
    "i.e. scale parameter for the lorentzian variation of the variance (Note: The scale parameter has to be >= 0)": "即洛伦兹方差变化的尺度参数（注意：尺度参数必须大于等于0）",
    "i.e. the tau parameter, which is computed using 'value' + rnd_cauchy('variance')": "即 tau 参数，它是使用 'value' + rnd_cauchy('variance') 计算得出的",
    "Higher absolute(!) values lead to more skewness (negative values cause fronting, positive values cause tailing). Tau parameter of the EGH, i.e. time constant of the exponential decay of the Exponential Gaussian Hybrid distribution shape of the elution profile": "更高的绝对值（！）会导致更多的偏斜（负值导致前沿，正值导致尾部）。EGH的Tau参数，即指数高斯混合分布洗脱峰形状的指数衰减时间常数。",
    "i.e. scale parameter for the lorentzian variation of the time constant (Note: The scale parameter has to be > 0)": "即洛伦兹变化的时间常数的尺度参数（注意：尺度参数必须大于0）",
    "This can serve as a filter to remove peptides which ionize badly, thus reducing peptide count": "这可以作为一个过滤器，用于移除离子化效果不好的肽段，从而减少肽段计数",
    "Peptides with a lower score will be removed": "得分较低的肽段将被移除",
    "The N-term is always assumed to carry a charge. This parameter will be ignored during MALDI ionization": "N-端总是假定带有电荷。在MALDI电离过程中将忽略此参数",
    "e.g. ['H:1'] or ['H:0.7' 'Na:0.3'], ['H:4' 'Na:1'] (which internally translates to ['H:0.8' 'Na:0.2']) (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如 ['H:1'] 或 ['H:0.7' 'Na:0.3']，['H:4' 'Na:1']（这在内部转换为 ['H:0.8' 'Na:0.2']）（以空格分隔的列表，为了允许列表项中有空格，将它们用单引号包围）",
    "E.g. assuming charge=3 and this parameter is 2, then we could choose to allow '3H+, 2H+Na+' features (given a certain 'charge_impurity' constraints), but no '3H+, 2H+Na+, 3Na+'": "例如，假设电荷=3，这个参数是2，那么我们可以选择允许'3H+, 2H+Na+'特征（在一定的'charge_impurity'约束条件下），但不允许'3H+, 2H+Na+, 3Na+'",
    "(select 'false' if you only need feature-maps)": "（如果只需要特征图，请选择'false'）",
    "QTOFs usually show 'constant' behavior, FTs have linear degradation, and on Orbitraps the resolution decreases with square root of mass": "QTOF（四极杆-飞行时间质谱仪）通常表现出“恒定”的行为，FT（傅立叶变换质谱仪）有线性退化，而在Orbitrap（轨道阱质谱仪）上，分辨率随质量的平方根而降低。",
    "Set to 0 to disable simulation of baseline": "设置为0以禁用基线模拟",
    "See 'share/OpenMS/SIMULATION/contaminants.txt' for details": "详见'share/OpenMS/SIMULATION/contaminants.txt'",
    "Set to 0 to disable simulation of m/z errors": "设置为0以禁用模拟m/z误差",
    "Set to 1.0 to get the real intensity values provided in the FASTA file": "设置为1.0以获取FASTA文件中提供的实测强度值",
    "Set to 0 to get simple rescaled intensities": "设置为0以获得简单的重新缩放强度",
    "Set this to 0 to disable simulation of shot noise": "将此设置为0以禁用散粒噪声的模拟",
    "No new peaks are generated; only intensity of existing ones is changed": "没有生成新的峰值；只有现有峰值的强度发生了变化",
    "If enabled, ALL possible m/z positions (up to sampling frequency of detector) will receive an intensity increase/decrease according to the specified Gaussian intensity distribution (similar to a noisy baseline)": "如果启用，所有可能的m/z位置（高达探测器的采样频率）将根据指定的高斯强度分布增加/减少强度（类似于嘈杂的基线）。",
    "0 - fixed intensities, 1 - SVC prediction (abundant/missing), 2 - SVR prediction of peak intensity .": "0 - 固定强度，1 - SVC预测（丰富/缺失），2 - SVR预测峰值强度。",
    "Also used to define the m/z width of an exclusion window (distance +/- from m/z of precursor). If you set this lower than the isotopic envelope of a peptide, you might get multiple fragment spectra pointing to the same precursor": "也用于定义排除窗口的m/z宽度（与前体m/z的距离 +/-）。如果您将其设置得低于肽的同位素包络，则可能会得到指向同一前体的多个碎片谱图。",
    "They will have a meta value 'MSE_DebugSpectrum' attached, so they can be filtered out. Native MS_E spectra will have 'MSE_Spectrum' instead": "它们将附加一个元值'MSE_DebugSpectrum'，因此它们可以被过滤掉。本地的MS_E谱图将有'MSE_Spectrum'代替。",
    "Note that adding isotopic peaks is very slow": "请注意，添加同位素峰非常慢",
    "If set to 0.0 only the retention times, computed by the RT model step are used": "如果设置为0.0，则仅使用由保留时间（RT）模型步骤计算出的保留时间",
    "(select 'false' if you only need peptide-labeling)": "（如果只需要肽段标记，请选择“false”）",
    "If set to 0.0 only the retention times computed by the RT model step are used": "如果设置为0.0，则仅使用由保留时间(RT)模型步骤计算的保留时间",
    "If you want to use only 2 channels, just leave the Labels as they are and provide only 2 input files": "如果您只想使用2个通道，请保持标签不变，并且只提供2个输入文件。",
    "If left empty, two channelSILAC is assumed": "如果留空，则假定为双通道SILAC",
    "If left empty, two-channel SILAC is assumed": "如果留空，则假定为双通道SILAC",
    "e.g. 114.1, 115.1)": "例如 114.1, 115.1",
    "e.g. \"114:myref\",\"115:liver\" (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如：“114:myref”，“115:liver”（用空格分隔的列表，如果列表项中有空格，则用单引号将它们包围起来）",
    "e.g. \"113:myref\",\"115:liver\",\"118:lung\" (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如：“113:myref”、“115:liver”、“118:lung”（用空格分隔的列表，如果列表项中有空格，则用单引号将它们包围起来）",
    "e.g. '114:0/0.3/4/0' , '116:0.1/0.3/3/0.2'  (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如 '114:0/0.3/4/0'，'116:0.1/0.3/3/0.2'（以空格分隔的列表，如果列表项中包含空格，则用单引号将其包围）",
    "e.g. '113:0/0.3/4/0' , '116:0.1/0.3/3/0.2'  (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如 '113:0/0.3/4/0'，'116:0.1/0.3/3/0.2'（以空格分隔的列表，如果列表项中包含空格，则用单引号将其包围）",
    "0=off, 1=full labeling": "0=关闭，1=完整标记",
    "(e.g. noise) in the simulated data. The generation is separated into two parts, the technical part, like noise in the raw signal, and the biological part, like systematic deviations in the predicted retention times": "（例如噪声）在模拟数据中。生成被分为两个部分，技术部分，如原始信号中的噪声，以及生物学部分，如预测保留时间中的系统偏差。",
    "(e.g. systematic effects like deviations in RT). If set to 'random' each experiment will look different. If set to 'reproducible' each experiment will have the same outcome (given that the input data is the same)": "（例如系统效应，如响应时间的偏差）。如果设置为'random'，每个实验看起来都会不同。如果设置为'reproducible'，给定输入数据相同的情况下，每个实验都会有相同的结果。",
    "(e.g. noise in the raw signal). If set to 'random' each experiment will look different. If set to 'reproducible' each experiment will have the same outcome (given that the input data is the same)": "（例如原始信号中的噪声）。如果设置为'random'，则每个实验看起来都会不同。如果设置为'reproducible'，则每个实验（在输入数据相同的情况下）将具有相同的结果。",
    "If you are unsure, use the default": "如果你不确定，使用默认值",
    "Will be added in both directions to input calibrant m/z": "将在两个方向上添加到输入的校准物 m/z",
    "No: does not generate all plots but only the most informatives": "不：不生成所有图表，只生成最有信息量的图表",
    "First m/z": "首先 m/z",
    "Second m/z": "第二个 m/z",
    "Will be added in both directions to input calibrant m/z and intensities will be averaged in this range.": "将在两个方向上添加到输入的校准物 m/z，并且在此范围内平均化强度。",
    "Optional title for fold change plot.": "折叠变化图的可选标题。",
    "It is recommended to include a contaminant database as well select fasta data sets(s)": "建议包括一个污染数据库，同时选择一个或多个fasta数据集。",
    "RNA sequences may contain modification codes, which will be handled correctly if this is set to 'RNA'": "RNA序列可能包含修饰代码，如果将其设置为'RNA'，则将正确处理这些修饰代码",
    "Note that all sequences are shuffled using the same random seed, ensuring that identical sequences produce the same shuffled decoy sequences. Shuffled sequences that produce highly similar output sequences are shuffled again (see shuffle_sequence_identity_threshold)": "请注意，所有序列都使用相同的随机种子进行洗牌，确保相同的序列产生相同的洗牌诱饵序列。如果产生高度相似的输出序列的洗牌序列将再次进行洗牌（见shuffle_sequence_identity_threshold）。",
    "Only applicable if parameter 'type' is 'protein'": "仅当参数'type'为'protein'时适用",
    "Separate by comma, e.g. use 'K,P,R' here": "用逗号分隔，例如，这里使用 'K,P,R'",
    "If the sequence identity is above this threshold, shuffling is repeated. In case of repeated failure, individual amino acids are 'mutated' to produce a different amino acid sequence": "如果序列相似性高于这个阈值，就会重复打乱。如果重复失败，就会对单个氨基酸进行“突变”以产生不同的氨基酸序列。",
    "Scans with other levels remain unchanged (space separated list, in order to allow for spaces in list items surround them by single quotes)": "扫描其他级别保持不变（以空格分隔的列表，为了允许列表项中有空格，请用单引号将它们包围起来）",
    "(e.g., 0.05 = 5% FDR, disabled for 1)": "（例如，0.05 = 5% FDR，1时禁用）",
    "Caution: if remove_psms_without_proteins is false, protein level filtering does not propagate": "警告：如果 remove_psms_without_proteins 是 false，蛋白质水平的过滤不会传播",
    "The q-value is set to the closest target score": "q值被设置为最接近的目标分数",
    "(e.g. from aligned runs) select idxml data sets(s)": "（例如，来自对齐的运行）选择 idxml 数据集（s）",
    "Smaller values decrease memory usage but increase runtime": "较小的值会减少内存使用，但会增加运行时间",
    "If set, this parameter takes precedence over 'extract:n_isotopes'": "如果设置，此参数将优先于'extract:n_isotopes'",
    "If set, this parameter takes precedence over 'extract:rt_quantile'": "如果设置此参数，则它将优先于 'extract:rt_quantile'",
    "Also determines the RT extration window, unless set explicitly via 'extract:rt_window'": "还决定了保留时间（RT）提取窗口，除非通过 'extract:rt_window' 明确设置",
    "Absolute value in seconds if 1 or greater, else relative to 'peak_width'": "如果为1或更大，则以秒为单位的绝对值，否则相对于'peak_width'",
    "Absolute value in seconds if 1 or greater, else relative to the RT span of the feature": "如果为1或更大，则以秒为单位的绝对值；否则相对于特征的保留时间（RT）跨度",
    "This aims to reduce biases, but also reduces the amount of training data. Set this flag to skip this procedure and consider all available observations (subject to 'svm:samples')": "这旨在减少偏差，但也会减少训练数据的数量。设置此标志以跳过此过程，并考虑所有可用的观察结果（受 'svm:samples' 参数限制）",
    "A value 'x' is used as 'C = 2^x' (space separated list, in order to allow for spaces in list items surround them by single quotes)": "一个值 'x' 被用作 'C = 2^x'（空格分隔的列表，为了允许列表项中有空格，用单引号将它们包围起来）",
    "A value 'x' is used as 'gamma = 2^x' (space separated list, in order to allow for spaces in list items surround them by single quotes)": "一个值 'x' 被用作 'gamma = 2^x'（用空格分隔的列表，为了允许列表项中有空格，在它们周围加上单引号）",
    "This parameter sets the weight given to these points during model fitting; '0' to disable": "这个参数设置了在模型拟合期间给予这些点的权重；'0' 用于禁用",
    "'0' to disable. Not applied to individual mass traces (parameter 'each_trace')": "'0' 用于禁用。不适用于单独的质量迹线（参数 'each_trace'）",
    "mod. Gaussians to mass traces": "模态高斯到质量迹线",
    "If set, only feature classification and elution model fitting are carried out, if enabled. Many parameters are ignored select featurexml data sets(s)": "如果设置，仅进行特征分类和洗脱模型拟合，如果启用。许多参数被忽略选择featurexml数据集（s）",
    "If you set intensity_threshold=-1, t' will be zero.. As the 'optimal' value for this parameter is highly data dependent, we would recommend to start with -1, which will also extract features with very low signal-to-noise ratio. Subsequently, one might increase the threshold to find an optimized trade-off between false positives and true positives. Depending on the dynamic range of your spectra, suitable value ranges include: -1, [0:10], and if your data features even very high intensity values, t can also adopt values up to around 30. Please note that this parameter is not of an integer type, s.t. you can also use t:=0.1, e.g": "如果您将intensity_threshold设置为-1，t'将为零。由于此参数的“最优”值高度依赖于数据，我们建议从-1开始，这也会提取信噪比非常低的特征。随后，您可能需要提高阈值以找到假阳性和真阳性之间的最佳权衡。根据您的光谱动态范围，合适的值范围包括：-1，[0:10]，如果您的数据特征甚至非常高的强度值，t也可以采用高达30左右的值。请注意，此参数不是整数类型，因此您也可以使用t:=0.1，例如",
    "'ref' (default) returns the sum of the intensities of each isotopic peak within an isotope pattern. 'trans' refers to the intensity of the monoisotopic peak within the wavelet transform. 'corrected' refers also to the transformed intensity with an attempt to remove the effects of the convolution. While the latter ones might be preferable for qualitative analyses, 'ref' might be the best option to obtain quantitative results. Please note that intensity values might be spoiled (in particular for the option 'ref'), as soon as patterns overlap (see also the explanations given in the class documentation of FeatureFinderAlgorihtmIsotopeWavelet)": "'ref'（默认）返回每个同位素峰值的强度之和。'trans'指的是小波变换中单同位素峰的强度。'corrected'也指的是经过变换的强度，并尝试去除卷积效应。虽然后者可能更适合定性分析，但'ref'可能是获得定量结果的最佳选择。请注意，强度值可能会受到影响（特别是对于'ref'选项），一旦模式重叠（另见FeatureFinderAlgorithmIsotopeWavelet类文档中的解释）。",
    "the averagine model, i.e. potential peptide masses are checked for plausibility. In addition, a heuristic correcting potential mass shifts induced by the wavelet is applied": "阿韦拉金模型，即潜在肽质量被检查其合理性。此外，还应用了一种启发式方法，用于纠正由小波引起的潜在质量偏移。",
    "i.e. for spectra featuring large m/z-gaps (present in FTICR and Orbitrap data, e.g.). Please check a single MS scan out of your recording, if you are unsure": "即对于具有较大m/z间隙的谱图（存在于FTICR和Orbitrap数据中，例如）。如果您不确定，请检查您的记录中的单个MS扫描",
    "rt_votes_cutoff) where an expected pattern is missing. There is usually no reason to change the default value": "rt_votes_cutoff) 其中缺少一个预期的模式。通常没有理由改变默认值",
    "Disable for direct injection": "禁用直接注入",
    "Disable for direct injection data": "禁用直接注入数据",
    "SVM isotope models for metabolites were trained with either 2% or 5% RMS error. For peptides, an averagine cosine scoring is used. Select the appropriate noise model according to the quality of measurement or MS device": "稳定同位素标记的代谢物支持向量机（SVM）模型使用2%或5%的均方根误差（RMS error）进行训练。对于肽段，使用平均蛋白质余弦打分。根据测量质量或质谱设备的选择，选择适当的噪声模型。",
    "Disable for general metabolites (as described in Kenar et al. 2014, MCP.)": "禁用一般代谢物（如Kenar等人在2014年MCP中所述）。",
    "A expected m/z range is computed from the isotopes of the assumed elements. If enabled, this ignores 'mz_scoring_13C'": "一个预期的m/z范围是根据假定元素的同位素计算得出的。如果启用了这个选项，它将忽略'mz_scoring_13C'参数。",
    "Set to zero if the MRM-traces contains mostly signals, and no noise": "如果MRM迹线主要包含信号而没有噪声，则设置为零。",
    "If the sample is unlabelled (i.e. you want to detect only single peptide features) please leave this parameter empty. [...] specifies the labels for a single sample. For example. [][Lys8,Arg10] ... SILAC. [][Lys4,Arg6][Lys8,Arg10] ... triple-SILAC. [Dimethyl0][Dimethyl6] ... Dimethyl. [Dimethyl0][Dimethyl4][Dimethyl8] ... triple Dimethyl. [ICPL0][ICPL4][ICPL6][ICPL10] ... ICPL": "如果样本未标记（即您只想检测单个肽段特征），请将此参数留空。 [...] 指定单个样本的标记。例如。 [][Lys8,Arg10] ... SILAC。 [][Lys4,Arg6][Lys8,Arg10] ... 三重SILAC。 [Dimethyl0][Dimethyl6] ... 二甲基化。 [Dimethyl0][Dimethyl4][Dimethyl8] ... 三重二甲基化。 [ICPL0][ICPL4][ICPL6][ICPL10] ... ICPL",
    "i.e. min charge : max charge": "即最小电荷：最大电荷",
    "For example 3:6, if isotopic peptide patterns in the sample consist of either three, four, five or six isotopic peaks.": "例如 3:6，如果样本中的同位素肽模式包含三个、四个、五个或六个同位素峰。",
    "(This is not an upper bound. Peptides that elute for longer will be reported.)": "（这不是一个上限。洗脱时间更长的肽段也将被报告。）",
    "For some low-intensity peptides, an important peak might be missing in one spectrum but be present in one of the neighbouring ones. The algorithm takes a bundle of neighbouring spectra with width rt_band into account. For example with rt_band = 0, all characteristic isotopic peaks have to be present in one and the same spectrum. As rt_band increases, the sensitivity of the algorithm but also the likelihood of false detections increases": "对于一些低强度的肽段，一个重要的峰可能在一个谱图中缺失，但在相邻的谱图中存在。该算法考虑了一组具有宽度rt_band的相邻谱图。例如，当rt_band = 0时，所有特征的同位素峰必须出现在同一个谱图中。随着rt_band的增加，算法的灵敏度提高了，但同时也增加了误检的可能性。",
    "(Any peptides seen for a shorter time period are not reported.)": "（任何在更短时间周期内观察到的肽段都不会被报告。）",
    "This parameter is a lower bound on their similarity": "这个参数是它们相似性的下界",
    "This parameter is a lower bound on similarity between measured isotopic pattern and the averagine model": "这个参数是测量到的同位素模式与averagine模型之间相似性的下限",
    "For the detection of single peptides, the averagine parameter p is replaced by p' = p + x(1-p), i.e. x = 0 -> p' = p and x = 1 -> p' = 1. (For knock_out = true, peptide doublets and singlets are detected simultaneously. For singlets, the peptide similarity filter is irreleavant. In order to compensate for this 'missing filter', the averagine parameter p is replaced by the more restrictive p' when searching for singlets.)": "对于单个肽段的检测，averagine参数p被替换为p' = p + x(1-p)，即x = 0 -> p' = p，x = 1 -> p' = 1。（对于knock_out = true，同时检测肽段二聚体和单体。对于单体，肽段相似性过滤器是无关紧要的。为了补偿这个“缺失的过滤器”，在搜索单体时，averagine参数p被更严格的p'替换。）",
    "'automatic' determines the spectrum type directly from the input mzML file": "“自动”直接根据输入的mzML文件确定谱图类型",
    "(Supported for doublex, triplex and quadruplex experiments only.)": "（仅支持双倍体、三倍体和四倍体实验。）",
    "Two signals from different maps will not be connected by an edge in the compatibility graph if absolute log fold change exceeds this limit (they might still end up in the same connected component, however). Note: this does not limit fold changes in the linking stage, only during RT alignment, where we try to find high-quality alignment anchor points. Setting this to a value < 0 disables the FC check": "如果绝对对数倍数变化超过此限制，则不同图中的两个信号不会在兼容性图中通过边连接（它们仍可能最终在同一个连通分量中）。注意：这不会限制链接阶段的倍数变化，仅在RT对齐期间，我们尝试找到高质量的对齐锚点。将此设置为小于0的值将禁用FC检查。",
    "(i.e., unknown charge state) with every charge state, or disregard charges (Any)": "（即，未知的电荷状态）使用每种电荷状态，或者忽略电荷（任何）",
    "RT regions will be extended until that number is reached": "RT区域将被扩展，直到达到该数字",
    "Check your current score type!": "检查你当前的分数类型！",
    "0 = no bias, 1 = IDs inside the max tolerances always preferred (even if much further away)": "0 = 无偏差，1 = 总是优先选择在最大容忍度内的ID（即使更远）",
    "Merging results in loss of descriptive information of the single protein identification runs": "合并结果会导致单个蛋白质鉴定运行的描述性信息丢失",
    "This will lead to higher probabilities for (bigger) protein groups": "这将导致（更大的）蛋白质组的概率更高",
    "number of connected states in a subgraph. For a value N, subgraphs that are bigger than 2^N will be split up, sacrificing accuracy for runtime. '0' uses the default (18)": "子图中连通状态的数量。对于一个值N，比2^N大的子图将会被分割，以牺牲准确性来换取运行时间。'0'使用默认值（18）",
    "i.e. without parameter estimation (in which case other settings, except 'log2_states', are ignored)": "即，不进行参数估计（在这种情况下，除了'log2_states'之外，其他设置都被忽略）",
    "There is a trade-off between accuracy and runtime. Empty uses the default ('best')": "精度和运行时间之间存在权衡。Empty使用默认值（'best'）",
    "All spectra without precursor are kept! (space separated list, in order to allow for spaces in list items surround them by single quotes)": "所有没有前体的谱图都被保留！（用空格分隔的列表，为了允许列表项中有空格，将它们用单引号包围）",
    "Do not use this flag in conjunction with 'mz', unless you want to actually remove peaks in spectra (see 'mz'). RT filtering is covered by 'rt' and compatible with this flag": "不要将此标志与'mz'一起使用，除非你确实想要在谱图中移除峰（见'mz'）。RT过滤由'rt'覆盖，并且与此标志兼容。",
    "If a precursor window covers ANY of these values, the corresponding MS/MS spectrum will be kept (space separated list, in order to allow for spaces in list items surround them by single quotes)": "如果前体窗口覆盖了这些值中的任何一个，则相应的MS/MS谱图将被保留（以空格分隔的列表，为了允许列表项中有空格，请用单引号将它们包围起来）",
    "e.g. from SRM scans": "例如，来自SRM扫描",
    "Matching tolerances are taken from 'spectra:blackorwhitelist:similarity_threshold|rt|mz' options..  select mzml data sets(s)": "匹配容差取自 'spectra:blackorwhitelist:similarity_threshold|rt|mz' 选项。选择 mzml 数据集（s）",
    "Otherwise Da are used": "否则使用Da",
    "False: retain matched MS2 spectra. Other levels are kept": "错误：保留匹配的MS2谱图。其他级别被保留",
    "Matching tolerances are taken from 'consensus:blackorwhitelist:rt' and 'consensus:blackorwhitelist:mz' options.. If consensus:blackorwhitelist:maps is specified, only these will be used..  select consensusxml data sets(s)": "匹配公差来自 'consensus:blackorwhitelist:rt' 和 'consensus:blackorwhitelist:mz' 选项。如果指定了 consensus:blackorwhitelist:maps，则只使用这些。选择 consensusxml 数据集(们)",
    "i.e. <name> 'lt|eq|gt' <value>; the first is the name of meta value, followed by the comparison operator (equal, less or greater) and the value to compare to. All comparisons are done after converting the given value to the corresponding data value type of the meta value (for lists, this simply compares length, not content!)! (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如 <名称> 'lt|eq|gt' <值>；第一个是元值的名称，后面是用于比较的运算符（等于、小于或大于）以及要比较的值。所有比较都是在将给定的值转换为元值对应的数据类型之后进行的（对于列表，这仅仅是比较长度，而不是内容！）（以空格分隔的列表，如果列表项中有空格，请用单引号将它们包围起来）",
    "The Priority of the id-flags is: remove_annotated_features / remove_unannotated_features -> remove_clashes -> keep_best_score_id -> sequences_whitelist  / accessions_whitelist": "id-flags的优先级是：remove_annotated_features/去除注释特征 -> remove_unannotated_features/去除未注释特征 -> remove_clashes/去除冲突 -> keep_best_score_id/保留最佳得分id -> sequences_whitelist/序列白名单 / accessions_whitelist/访问号白名单",
    "e.g. features containing LYSNLVER or the modification (Oxidation). To control comparison method used for whitelisting, see 'id:sequence_comparison_method' (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如，包含 LYSNLVER 或者修饰（氧化）的特征。要控制用于白名单比较的方法，请查看 'id:sequence_comparison_method'（用空格分隔的列表，为了允许列表项中有空格，用单引号将它们包围起来）。",
    "e.g. sp|P02662|CASA1_BOVIN (space separated list, in order to allow for spaces in list items surround them by single quotes)": "例如 sp|P02662|CASA1_BOVIN（用空格分隔的列表，如果列表项中有空格，则用单引号将它们包围起来）",
    "Matching tolerances are taken from 'id:rt' and 'id:mz' options.. This tool will require all IDs to be matched to an MS2 spectrum, and quit with error otherwise. Use 'id:blacklist_imperfect' to allow for mismatches select idxml data sets(s)": "匹配容差来自'id:rt'和'id:mz'选项。该工具将要求所有ID与MS2谱图匹配，并在未匹配的情况下报错退出。使用'id:blacklist_imperfect'以允许不匹配，选择idxml数据集(们)。",
    " select consensusxml,dta,dta2d,fasta,featurexml,idxml,mgf,mzdata,mzid,mzml,mzxml,pepxml,pqp,sqmass,trafoxml data sets(s)": "选择 consensusxml、dta、dta2d、fasta、featurexml、idxml、mgf、mzdata、mzid、mzml、mzxml、pepxml、pqp、sqmass、trafoxml 数据集（s）",
    " select consensusxml,dta,dta2d,fasta,featurexml,mgf,mzdata,mzml,mzxml,traml data sets(s)": "选择consensusxml、dta、dta2d、fasta、featurexml、mgf、mzdata、mzml、mzxml、traml数据集（单数或复数）",
    "- 'append_rows' is usually used when the inputs come from the same MS run (e.g. caused by manual splitting or multiple algorithms on the same file). - 'append_cols' when you want to combine consensusXMLs from e.g. different fractions to be summarized in ProteinQuantifier or jointly exported with MzTabExporter": "- 'append_rows' 通常用于输入来自同一 MS 运行的情况（例如，由手动拆分或同一文件上的多个算法引起）。- 'append_cols' 当您想要结合来自不同组分的 consensusXMLs，以便在 ProteinQuantifier 中汇总或与 MzTabExporter 一起导出时使用。",
    "The RT ranges of inputs are adjusted so they don't overlap in the merged file (traML input not supported)": "输入的RT范围被调整，以确保它们在合并后的文件中不会重叠（不支持traML输入）。",
    "RT concatenation is enabled if a value > 0 is set": "RT串联功能如果设置了一个大于0的值则被启用。",
    "The number of given retention times must be equal to the number of input files (space separated list, in order to allow for spaces in list items surround them by single quotes)": "给定的保留时间数量必须等于输入文件的数量（以空格分隔的列表，如果列表项中包含空格，则用单引号将其包围）",
    "This option is useful for merging DTA files, where filenames should contain the string 'rt' directly followed by a floating point number, e.g. 'my_spectrum_rt2795.15.dta'": "这个选项对于合并 DTA 文件非常有用，其中文件名应该包含字符串 'rt' 直接后跟一个浮点数，例如 'my_spectrum_rt2795.15.dta'",
    "This option is useful for DTA files which do not contain MS level information": "这个选项对于不包含MS级别信息的DTA文件非常有用",
    "Must be in order that the consensusXML file maps the original mzML files select mzml data sets(s)": "必须按照一致性XML文件映射原始mzML文件的顺序选择mzML数据集",
    "Edit the list if you wish to exclude/include adducts. By default CHEMISTRY/PositiveAdducts.tsv in OpenMS/share is used select tabular data sets(s)": "如果你希望排除/包括加合物，可以编辑列表。默认情况下，OpenMS/share中的CHEMISTRY/PositiveAdducts.tsv用于选择表格数据集。",
    "Edit the list if you wish to exclude/include adducts. By default CHEMISTRY/NegativeAdducts.tsv in OpenMS/share is used select tabular data sets(s)": "如果您需要排除/包括加合物，请编辑列表。默认情况下，OpenMS/share中的CHEMISTRY/NegativeAdducts.tsv用于选择表格数据集。",
    "If 'mass' is 0, it is re-computed from the molecular sum formula. By default CHEMISTRY/HMDBMappingFile.tsv in OpenMS/share is used! If empty, the default will be used select tabular data sets(s)": "如果“质量”为0，它将根据分子分子式重新计算。默认情况下，在OpenMS/share中使用的是CHEMISTRY/HMDBMappingFile.tsv！如果为空，则将使用默认值选择表格数据集。",
    "SMILES and INCHI are reported in the output, but not used otherwise. By default CHEMISTRY/HMDB2StructMapping.tsv in OpenMS/share is used! If empty, the default will be used select tabular data sets(s)": "SMILES和INCHI在输出中被报告，但除此之外没有被使用。默认情况下，在OpenMS/share中使用CHEMISTRY/HMDB2StructMapping.tsv！如果为空，将使用默认值选择表格数据集。",
    "If 'auto' is used, the first feature of the input map must contain the meta-value 'scan_polarity'. If its missing, the tool will exit with error": "如果使用“auto”选项，输入文件的第一个特征必须包含元数据值“scan_polarity”。如果缺少该值，工具将报错退出。",
    "Three or four columns: x values, y values, response values, optionally fold values": "三列或四列：x 值、y 值、响应值，可选地折叠值",
    "This is the condition (pixel group) which will be classified": "这是将被分类的条件（像素组）",
    "Each fold must contain pixels of all response groups and is used for cross validation": "每个折点必须包含所有响应组的像素，并且用于交叉验证",
    "For cvapply multiple values are allowed (e.g. 1,2,3 or 2:5). Mininum is 1.": "对于cvapply，允许多个值（例如1,2,3或2:5）。最小值是1。",
    "Run cvApply first to optain optimal number of PLS-DA components": "首先运行cvApply以获得最佳的PLS-DA组件数量",
    "For cvapply multiple values are allowed (e.g. 1,2,3 or 2:5). Minimum is 1.": "对于cvapply，允许多个值（例如1,2,3或2:5）。最小值是1。",
    "Run cvApply first to optain optimal number of OPLS-DA components": "首先运行cvApply以获得最佳的OPLS-DA组件数量",
    "Can be used to generate automatic classification workflow": "可以用来生成自动分类工作流",
    "For cvapply multiple values are allowed (e.g. 0,1,2,3 or 2:5)": "对于cvapply，允许多个值（例如0,1,2,3或2:5）",
    "For cvapply multiple values are allowed (e.g. 0,1,2 or 2:5)": "对于cvapply，允许多个值（例如 0,1,2 或 2:5）",
    "Can be used to generate a classification prediction on new data": "可以用来对新数据生成分类预测",
    " select mzml,sqmass data sets(s)": "选择 mzml、sqmass 数据集（s）",
    "(e.g. to remove all features with a an m_score above 0.05 use 0.05 here)": "（例如，要移除所有m_score高于0.05的特征，在这里使用0.05）",
    "Thus, for each charge state a new model will be computed": "因此，对于每个电荷状态，将计算一个新的模型",
    "Useful for pipelines with many datasets where only a few are bad, but the pipeline should run through": "适用于包含许多数据集的流程，其中只有少数数据集有问题，但流程仍应运行完毕。",
    "Only needed if each iteration step of the EM-Algorithm will be visualized": "如果EM算法的每个迭代步骤都需要可视化，则需要这个参数",
    "For 'Gauss', the Gauss distribution is used": "对于“高斯”，使用了高斯分布",
    "- ignore_iqr_outliers: ignore outliers outside of 3*IQR from Q1/Q3 for fitting. - set_iqr_to_closest_valid: set IQR-based outliers to the last valid value for fitting. - ignore_extreme_percentiles: ignore everything outside 99th and 1st percentile (also removes equal values like potential censored max values in XTandem). - none: do nothing": "- 忽略四分位数极差异常值：在拟合时忽略超出Q1/Q3 3倍四分位数极差范围的异常值。 - 将四分位数极差异常值设置为最接近的有效值：在拟合时将基于四分位数极差的异常值设置为最接近的有效值。 - 忽略极值百分位数：忽略99%和1%百分位数以外的所有值（也会移除相等的值，如XTandem中潜在的被截断的最大值）。 - 无：不采取任何措施。",
    "Additionally, target/decoy information should be available. The score_type must be q-value from an previous False Discovery Rate run": "此外，目标/诱饵信息应该是可用的。得分类型必须是来自先前假发现率运行的q值。",
    "This may be:. - a single file in OpenMS database format (.oms),. - a single file in a multi-purpose XML format (.idXML, .mzid, .pepXML, .protXML),. - a single file in a search engine-specific format (Mascot: .mascotXML, OMSSA: .omssaXML, X! Tandem: .xml, Percolator: .psms, xQuest: .xquest.xml),. - a single file in fasta format (can only be used to generate a theoretical mzML),. - a single text file (tab separated) with one line for all peptide sequences matching a spectrum (top N hits),. - for Sequest results, a directory containing .out files..  select fasta,idxml,mascotxml,mzid,pepxml,protxml,psms,sqlite,tabular,xquest.xml data sets(s)": "这可能是：- 一个OpenMS数据库格式的单个文件（.oms），- 一个多用途XML格式的单个文件（.idXML、.mzid、.pepXML、.protXML），- 一个特定搜索引擎格式的单个文件（Mascot：.mascotXML，OMSSA：.omssaXML，X! Tandem：.xml，Percolator：.psms，xQuest：.xquest.xml），- 一个fasta格式的单个文件（只能用于生成理论的mzML），- 一个包含所有肽序列匹配一个谱图（前N个命中）的肽序列的单个文本文件（制表符分隔），- 对于Sequest结果，包含.out文件的目录。选择fasta、idxml、mascotxml、mzid、pepxml、protxml、psms、sqlite、tabular、xquest.xml数据集。",
    "Tandem, mzid, Percolator only] Retention times and native spectrum ids (spectrum_references) will be looked up in this file select mzdata,mzml,mzxml data sets(s)": "[串联，mzid，Percolator 仅] 保留时间和原始谱图 ID（spectrum_references）将在这些文件中查找 mzdata、mzml、mzxml 数据集",
    "Only necessary if different from 'mz_file'": "只有在与'mz_file'不同的情况下才需要",
    "By default a 'raw' pepXML is produced that contains only search engine results": "默认情况下，生成的是一个“原始”的pepXML文件，其中只包含搜索引擎的结果",
    "(default: 0, meaning max_charge + 1 will be used as precursor charge)": "（默认值：0，意味着将使用max_charge + 1作为前体电荷）",
    "e.g. \"+1\" in references column,. but do not list extra references in subsequent lines (try -debug 3 or 4)": "例如，在参考文献列中使用 \"+1\"，但不要在随后的行中列出额外的参考文献（尝试使用 -debug 3 或 4）",
    "See documentation for details": "详见文档了解详细信息",
    "Use only if you are sure it is absolutely the same 'mz_file' as used for identification": "只有在确定使用的是与鉴定时完全相同的'mz_file'时才使用",
    "This will take quite some time": "这将需要相当长的时间",
    "A value of 0 or less exports all hits": "值小于或等于0将导出所有命中结果",
    "The m/z window is used to search for peaks, if intensity > 0 found in the window the m/z is considered present, if all intensities are 0 the m/z is considered not present": "m/z窗口用于搜索峰值，如果在窗口中找到强度> 0的峰，则认为m/z存在；如果所有强度都为0，则认为m/z不存在。",
    "This step will only work if pixel annotations are provided": "这一步只有在提供了像素注释的情况下才能工作",
    "Multiple values are allowed (e.g. 1,2,3 or 2:5)": "允许多个值（例如 1,2,3 或 2:5）",
    "As s increases, fewer m/z features (m/z values) will be used in the spatial segmentation, and only the informative m/z features will be retained. Multiple values are allowed (e.g. 1,2,3 or 2:5)": "随着s的增加，在空间分割中将使用较少的m/z特征（m/z值），并且只保留有信息量的m/z特征。允许多个值（例如1,2,3或2:5）",
    "Use same value to reproduce previous results": "使用相同的值来复现之前的结果",
    "Tabular file with two columns: x values and y values": "具有两列的表格文件：x值和y值",
    "This value is the half window size, it will be added and substracted from the given input value": "这个值是半窗口大小，它将从给定的输入值中加和减",
    "Otherwise it is a suffix": "否则它是一个后缀",
    "Matches candidates for possible monoisotopic precursor peaks for experimental mass m and given numbers n at masses (m - n * (C13-C12)). These should be ordered from more extreme to less extreme corrections. Numbers later in the list will be preferred in case of ambiguities (space separated list, in order to allow for spaces in list items surround them by single quotes)": "为实验质量 m 和给定数字 n 在质量 (m - n * (C13-C12)) 处寻找可能的单同位素前体峰的候选项。这些应该从更极端到不太极端的校正顺序排列。如果存在歧义，列表中后出现的数字将被优先选择（用空格分隔的列表，为了允许列表项中有空格，请用单引号将它们包围）。",
    "(e.g. DSS or BS3)": "（例如 DSS 或 BS3）",
    "If set to auto the spectra will be deisotoped, if the fragment mass tolerance is < 0.1 Da or < 100 ppm (0.1 Da at a mass of 1000)": "如果设置为自动，谱图将进行去同位素处理，如果片段质量容差小于0.1 Da或小于100 ppm（在1000的质量下为0.1 Da）",
    "Decoys can also be included in the normal database file instead (or additionally) select fasta data sets(s)": "诱饵序列也可以包含在常规数据库文件中（或者额外地选择fasta数据集）",
    " select idxml,mzid,xquest.xml data sets(s)": "选择 idxml、mzid、xquest.xml 数据集（s）",
    "The correspondig target protein id should be retrievable by deleting this prefix": "相应的目标蛋白质ID应该可以通过删除这个前缀来检索",
    "Values outside of the tolerance window of the original search will effectively disable this filter": "原始搜索的公差窗口外的值将有效地禁用此过滤器",
    "Minimum delta score required, hits are rejected if larger or equal. The delta score is a ratio of the score of a hit and the score of the next best hit to the same spectrum, so the value range is between 0 and 1 with 1.0 meaning the scores are equal and 0.5 meaning the next best score is half as high as the current one": "所需的最小delta分数，如果大于或等于该分数，命中将被拒绝。Delta分数是命中分数与同一谱图的下一个最佳命中分数的比率，因此其值范围在0到1之间，1.0表示分数相等，0.5表示下一个最佳分数是当前分数的一半。",
    "For a set of IDs from equal candidates (same pair of peptides, modifications and cross-linked positions), only the highest scoring hit will be considered. By default the score distribution will be estimated using all 1st ranked candidates": "对于一组来自相同候选者（相同的肽对、修饰和交联位置）的ID，只有得分最高的命中会被考虑。默认情况下，将使用所有排名第一的候选者来估计得分分布。",
    "A number lower than the lowest score will effectively disable this filter": "一个低于最低分数的数值将有效地禁用此过滤器",
    "Should be about the same size as the smallest expected difference between scores. Smaller numbers will make XFDR more robust, but much slower. Negative numbers are not allowed. Should only be changed if the range of the main score changes or another score than the OpenPepXL score is used": "应与预期分数之间最小差异的大小大致相同。较小的数字会使XFDR更稳健，但速度会慢得多。不允许使用负数。只有在主要分数的范围发生变化或使用OpenPepXL分数以外的分数时，才应更改此值。",
    " select osw data sets(s)": "选择OSW数据集（s）",
    "Use the -fasta option and -decoy_pattern to set the Fasta file and decoy pattern": "使用 -fasta 选项和 -decoy_pattern 设置 Fasta 文件和诱饵模式",
    "(i.e. not search engine specific) features. Generating search engine specific features for common search engines by PSMFeatureExtractor will typically boost the identification rate significantly": "（即不是搜索引擎特定的）特征。通过PSMFeatureExtractor为常见搜索引擎生成特定于搜索引擎的特征通常会显著提高识别率。",
    "Recommended when analyzing huge numbers (>1 million) of PSMs. When set to 0, all PSMs are used for training as normal": "当分析大量（>100万）的PSMs时推荐使用。当设置为0时，所有PSMs将像平常一样用于训练。",
    "Set by cross validation if not specified": "如果未指定，则通过交叉验证设置",
    "Set to testFDR if 0": "设置为 testFDR 如果为 0",
    "Only available if -doc is set": "仅在设置了-doc参数时可用",
    "If the user only provides one PSM per spectrum, this filter will have no effect": "如果用户只为每个谱图提供一个PSM（Peptide Spectrum Match，肽段谱图匹配），这个过滤器将不会有任何效果。",
    "Set -1 to disable this filter": "设置为-1以禁用此过滤器",
    "Note: In mode 'mascot_out' a Mascot results file (.mascotXML) is read select mascotxml,mzdata data sets(s)": "注意：在'mascot_out'模式下，读取一个Mascot结果文件（.mascotXML），选择mascotxml，mzdata数据集",
    "Defines a symmetric tolerance window around the feature. When looking at possible feature pairs, the allowed feature-wise errors are combined for consideration of possible adduct shifts. For ppm tolerances, each window is based on the respective observed feature mz (instead of putative experimental mzs causing the observed one)!": "定义特征周围的对称容忍窗口。在查看可能的特征对时，允许的特征误差会结合考虑可能的加成位移。对于ppm容忍度，每个窗口基于相应的观察到的特征m/z（而不是导致观察到的一个的假定实验m/z）！",
    "i.e. the number of '+' or '-' indicate the charge ('0' if neutral adduct), e.g. 'Ca:++:0.5' indicates +2. Probabilites have to be in (0,1]. The optional RTShift param indicates the expected RT shift caused by this adduct, e.g. '(2)H4H-4:0:1:-3' indicates a 4 deuterium label, which causes early elution by 3 seconds. As fifth parameter you can add a label for every feature with this adduct. This also determines the map number in the consensus file. Adduct element losses are written in the form 'H-2'. All provided adducts need to have the same charge sign or be neutral! Mixing of adducts with different charge directions is only allowed as neutral complexes. For example, 'H-1Na:0:0.05' can be used to model Sodium gains (with balancing deprotonation) in negative mode (space separated list, in order to allow for spaces in list items surround them by single quotes)": "即“+”或“-”的数量表示电荷（如果是中性加和物则为'0'），例如'Ca:++:0.5'表示+2。概率必须在(0,1]范围内。可选的RTShift参数表示由此加和物引起的预期RT偏移，例如'(2)H4H-4:0:1:-3'表示一个4氘标记，导致提前3秒洗脱。作为第五个参数，你可以为具有此加和物的每个特征添加一个标签。这也决定了共识文件中的地图编号。加和物元素损失以'H-2'的形式书写。所有提供的加和物必须具有相同的电荷符号或为中性！只允许将不同电荷方向的加和物混合为中性复合物。例如，'H-1Na:0:0.05'可以用来模拟负模式下的钠增益（通过平衡去质子化）（空格分隔的列表，为了允许列表项中的空格，用单引号将它们包围）。",
    "E.g., for 'charge_max' 4 and 'max_minority_bound' 2 with most probable adduct being H+ and least probable adduct being Na+, this will allow adduct compositions of '2(H+),2(Na+)' but not of '1(H+),3(Na+)'. Further, adduct compositions/changes less likely than '2(H+),2(Na+)' will be discarded as well": "例如，对于“charge_max”为4，“max_minority_bound”为2，最可能的加合物是H+，最不可能的加合物是Na+，这将允许加合物组成为'2(H+),2(Na+)'，但不允许'1(H+),3(Na+)'。此外，比'2(H+),2(Na+)'更不可能的加合物组成/变化也将被丢弃。",
    "This is usually necessary for LFQ experiments and therefore defaults to 'max'. In case of TMT/iTRAQ, MSstatsTMT does the aggregation itself later and the parameter always resets to manual (i.e. is unused)": "这通常对于LFQ实验是必要的，因此默认为'max'。在TMT/iTRAQ的情况下，MSstatsTMT会在稍后自己进行聚合，并且参数总是重置为手动（即不使用）。",
    " select consensusxml,featurexml,idxml,mzid data sets(s)": "选择 consensusxml、featurexml、idxml、mzid 数据集（s）",
    "If so, read peptide information only from second to last runs": "如果这样，请只从第二个到最后一个运行中读取肽段信息。",
    "The higher the value, the wider the peak and therefore the wider the gaussian": "值越高，峰就越宽，因此高斯分布就越宽。",
    "The gaussian is calculated in each step anew, so this is much slower": "高斯在每一步都重新计算，因此这要慢得多",
    "retention-time alignment of peptides should be enabled when quantifying samples": "在定量样品时，应启用肽段的保留时间对齐。",
    "This will make the search faster, but can impact the sensitivity positively or negatively, depending on the dataset": "这将使搜索更快，但可能会根据数据集的不同，对灵敏度产生正面或负面的影响。",
    "Longer tags will make the search faster but much less sensitive. Ignored if 'algorithm:use_sequence_tags' is false": "更长的标签将使搜索更快，但灵敏度会大大降低。如果'algorithm:use_sequence_tags'为false，则忽略此设置。",
    "This is the full window size, e.g. a value of 1000 seconds would extract 500 seconds on either side": "这是完整的窗口大小，例如，1000秒的值将提取中心点前后各500秒",
    "This is the full window size, e.g. 100 ppm would extract 50 ppm on either side": "这是完整的窗口大小，例如100 ppm会在两侧各提取50 ppm",
    "If present, SWATH specific scoring will be used select mzml data sets(s)": "如果存在，SWATH特定的评分将被用于选择mzml数据集",
    "For this to work, the TraML input file needs to contain normalized RT values": "为了使其工作，TraML输入文件需要包含标准化的保留时间（RT）值",
    "retention-time alignment of peptides is generally not needed when when building a library from narrow-window spectrums": "保留时间对齐通常在构建来自窄窗口光谱的库时不需要。",
    "RT normalization and outlier detection can be done iteratively (by default) which removes one outlier per iteration or using the RANSAC algorithm": "RT 标准化和异常值检测可以迭代进行（默认情况下），每次迭代移除一个异常值，或者使用 RANSAC 算法。",
    "(Note: If a peptide is not phosphorylated at all, its score is set to '-1'.)": "（注：如果一个肽段完全没有磷酸化，则其得分设置为'-1'。）",
    "Used to determine matching to feature mass traces": "用于确定与特征质量迹线匹配",
    "Only evaluated if copies are created (feature:keep_original)": "只有在创建副本时才会评估（特性：keep_original）",
    "(Disable method by setting value to 0.0)": "（通过将值设置为0.0来禁用方法）",
    "Suggested value 1/max. expected charge. (Disable method by setting value to 0.0)": "建议的值是1除以最大预期电荷。（将值设置为0.0以禁用方法）",
    "will be matched to the feature m/z values if possible select featurexml data sets(s)": "如果可能，将匹配到特征 m/z 值，请选择 featureXML 数据集（s）",
    "this value should be set to the instruments selection window": "这个值应该设置到仪器选择窗口",
    "Your identifications will be exported in an easy-to-read format select idxml data sets(s)": "您的鉴定结果将以易于阅读的格式导出，请选择idxml数据集(们)",
    "Use the consensusXML output form the DeCharger) select consensusxml data sets(s)": "使用DeCharger的consensusXML输出，选择consensusxml数据集。",
    "(i.e., mz-bin-offset = 0.xx means the left edge of the first bin will be located at +0.xx Da)": "（即，mz-bin-offset = 0.xx 表示第一个bin的左边缘将位于 +0.xx Da）",
    "This parameter specifies the size of each bin. The exact formula for computing the discretized m/z value is floor((x/mz-bin-width) + 1.0 - mz-bin-offset), where x is the observed m/z value. For low resolution ion trap ms/ms data 1.0005079 and for high resolution ms/ms 0.02 is recommended": "这个参数指定了每个bin的大小。计算离散化m/z值的确切公式是 floor((x/mz-bin-width) + 1.0 - mz-bin-offset)，其中x是观测到的m/z值。对于低分辨率离子阱ms/ms数据，推荐使用1.0005079，而对于高分辨率ms/ms，推荐使用0.02。",
    "Specify a comma-separated list of C-terminal modification sequences of the form: X+21.9819 Default = <empty>": "指定一个以逗号分隔的C端修饰序列列表，格式为：X+21.9819 默认值 = <空>",
    "Specify a comma-separated list of N-terminal modification sequences of the form: 1E-18.0106,C-17.0265 Default = <empty>": "指定一个以逗号分隔的N端修饰序列列表，形式为：1E-18.0106,C-17.0265 默认值 = <空>",
    "Specify a comma-separated list of modification sequences of the form: C+57.02146,2M+15.9949,1STY+79.966331,... Default = C+57.02146": "指定一个以逗号分隔的修饰序列列表，形式为：C+57.02146,2M+15.9949,1STY+79.966331,... 默认值 = C+57.02146",
    "Overrides the enzyme option. Two lists of residues are given enclosed in square brackets or curly braces and separated by a |. The first list contains residues required/prohibited before the cleavage site and the second list is residues after the cleavage site.": "覆盖酶选项。给出两个残基列表，它们被方括号或花括号包围，并由 '|' 分隔。第一个列表包含在切割位点之前所需的/禁止的残基，第二个列表是切割位点后的残基。",
    "'robust_regression' scales each map by a fator computed from the ratios of non-differential background features (as determined by the ratio_threshold parameter), 'quantile' performs quantile normalization, 'median' scales all maps to the same median intensity, 'median_shift' shifts the median instead of scaling (WARNING: if you have regular, log-normal MS data, 'median_shift' is probably the wrong choice. Use only if you know what you're doing!)": "'robust_regression' 通过由非差异背景特征（由 ratio_threshold 参数确定）的比率计算出的因子来缩放每个图，'quantile' 执行分位数标准化，'median' 将所有图缩放到相同的中位强度，'median_shift' 改变中位数而不是缩放（警告：如果您有规则的，对数正态的 MS 数据，'median_shift' 可能是错误的选择。仅在您知道自己在做什么时使用！）",
    "Useful, e.g., if you have known house keeping proteins in your samples. When this parameter is empty or the regular expression matches the empty string, all features are used (even those without an ID). No effect if quantile normalization is used": "有用，例如，如果您的样本中已知有管家蛋白。当此参数为空或正则表达式匹配空字符串时，将使用所有特征（即使没有ID的特征）。如果使用了分位数标准化，则没有效果。",
    "If empty, the usual suspects will be removed! (space separated list, in order to allow for spaces in list items surround them by single quotes)": "如果为空，通常会删除一些可疑对象！（用空格分隔的列表，如果列表项中包含空格，请用单引号将它们括起来）",
    "This overrides the name parameter! select mzml data sets(s)": "这会覆盖名称参数！选择mzml数据集",
    "First row is considered containing the header. The target run or set names/ids are indicated by column \"raw data file\", so each row after the header will contain the values of qps for that run. (csv without \"!) select csv data sets(s)": "第一行被认为包含标题。目标运行或集名称/ID由“原始数据文件”列指示，因此标题后的每行将包含该运行的qps值。（不带“!”的csv选择csv数据集）",
    "The first row is considered containing the headers as in the table. The second row is considered the according qp cv accessions. (csv without \"!) select csv data sets(s)": "第一行被视为包含表头。第二行被视为相应的qp cv访问号。（csv格式，不包含\"!) 选择csv数据集",
    "Ignored if 'in' is given": "如果给出了 'in'，则忽略",
    "See documentation for specific format requirements select txt data sets(s)": "查看文档以获取特定格式要求，选择文本数据集（s）",
    "Peptide sequence file containing peptides which you want to search (no column header, 1 peptide per line).": "包含您想要搜索的肽段序列的文件（无列标题，每行一个肽段）。",
    "Peptide sequence which you want to search": "要搜索的肽段序列",
    "Spectrum file used for identification, mgf format": "用于鉴定的谱图文件，mgf格式",
    "pFind uses the spectrum title": "pFind 使用谱图标题",
    "pFind can be used to generate a tags file for no-enzyme immunopeptidomics search": "pFind 可以用来为无酶免疫肽组学搜索生成一个标签文件。",
    "Max number of variable modifications": "最大变量修饰数",
    "Validation with unrestricted modification searching": "无限制修饰搜索验证",
    "the error window on experimental peptide mass values. This parameter is usually set according to the mass spectrometer which was used to generate the MS/MS data.": "实验肽质量值的误差窗口。这个参数通常根据用于生成MS/MS数据的质谱仪进行设置。",
    "The unit of precursor ion m/z tolerance": "前体离子 m/z 容差单位",
    "Error window for MS/MS fragment ion mass values in Da unit.": "质谱/质谱碎片离子质量值的误差窗口，以道尔顿（Da）为单位。",
    "Enzyme used for protein digestion": "用于蛋白质消化的酶",
    "The maximum charge to consider if the charge state is not available": "如果电荷状态不可用，则考虑的最大电荷",
    "The minimum charge to consider if the charge state is not available": "如果电荷状态不可用，则考虑的最小电荷",
    "Min peaks in spectrum": "谱中最小峰值数",
    "Minimum score to consider for peptide searching": "用于肽段搜索的最低分数",
    "The maximum length of peptide to consider.": "考虑的多肽最大长度。",
    "The number of random peptides": "随机肽段的数量",
    "Add a spectrum_file column before the spectrum_title column.  Useful for combining pepquery outputs.": "在spectrum_title列之前添加一个spectrum_file列。这对于合并pepquery输出非常有用。",
    "The file format to use for results of the search.": "搜索结果使用的文件格式。",
    "The FASTA protein database to search against. Only amino-acid type FASTA is supported.": "要搜索的FASTA蛋白质数据库。只支持氨基酸类型的FASTA。",
    "If the protein database contains accessions prefixed with this string, those proteins will be considered decoys. If there are no decoy protein, this prefix will be used to generate an automatic decoy for each target protein. Set this to an empty string to disable automatic decoys.": "如果蛋白质数据库包含以该字符串为前缀的访问号，这些蛋白质将被视为诱饵。如果没有诱饵蛋白质，将使用此前缀为每个目标蛋白质自动生成一个诱饵。将其设置为空字符串以禁用自动诱饵。",
    "The cleavage agent to generate in silico peptides from the protein database.": "用于从蛋白质数据库生成计算机模拟肽段的切割剂。",
    "Controls which peptide termini must match the cleavage rules.": "控制哪个肽末端必须符合裂解规则。",
    "Candidate peptides with more than this number of uncleaved cleavage sites will be skipped.": "具有超过此数量未切割位点的候选肽将被跳过。",
    "Modifications that are always applied to an amino acid. (-StaticMods)": "始终应用于氨基酸的修饰。(-StaticMods)",
    "Modifications that can either be present or absent on an amino acid. (-DynamicMods)": "氨基酸上可能存在或不存在的修饰。(-动态修饰)",
    "Candidate peptides with more than this number of dynamic modifications will be skipped.": "具有超过此数量动态修饰的候选肽段将被跳过。",
    "For data from Thermo instruments, using the 'auto' setting on a will automatically choose monoisotopic or average mass values, for other instruments monoisotopic or average should be chosen.": "对于来自Thermo仪器的数据，使用“auto”设置将自动选择单同位素或平均质量值，对于其他仪器，应选择单同位素或平均值。",
    "This tolerance will be used for spectra that are determined to have monoisotopic precursors. MonoisotopeAdjustmentSet is applied to these.": "这个容差将用于被确定具有单同位素前体的谱图。MonoisotopeAdjustmentSet 应用于这些谱图。",
    "This tolerance will be used for spectra that are determined to have average mass precursors. MonoisotopeAdjustmentSet is not applied to these.": "这个容差将用于被确定为具有平均质量前体的谱图。MonoisotopeAdjustmentSet不适用于这些。",
    "This tolerance will be used for all spectra, but MonoisotopeAdjustmentSet will never be applied.": "这个容差将用于所有谱图，但MonoisotopeAdjustmentSet永远不会被应用。",
    "This tolerance will be used for all spectra, and MonoisotopeAdjustmentSet is likewise applied to all spectra.": "这个容差将用于所有谱图，并且MonoisotopeAdjustmentSet同样应用于所有谱图。",
    "This tolerance is used for matching to fragment ions.": "这个容差用于匹配到片段离子。",
    "Filter out all but the specified number of peaks, keep empty to use all peaks.": "过滤掉所有但指定数量的峰值，保持空白以使用所有峰值。",
    "If the input file provides mass analyzer information (mzML, mzXML, mz5), MyriMatch can automatically choose the ion series to search on a scan-by-scan basis.": "如果输入文件提供了质谱分析器信息（mzML、mzXML、mz5），MyriMatch可以自动地逐个扫描选择要搜索的离子系列。",
    "Specify as a comma-separated list of a, b, c, x, y, z, or z* (z+1), e.g. 'b,y,z'": "指定为逗号分隔的列表，包含 a、b、c、x、y、z 或 z*（z+1），例如 'b,y,z'",
    "Candidate peptides shorter than this will not be compared against any spectrum.": "候选肽段如果短于这个长度，将不会与任何谱图进行比较。",
    "Candidate peptides longer than this will not be compared against any spectrum.": "候选肽段如果长于这个长度，将不会与任何谱图进行比较。",
    "For +3 and higher precursors, the fragment ions predicted depend on the way this parameter is set. When this parameter is true, then for each peptide bond, an internal calculation is done to estimate the basicity of the b and y fragment sequence. When this parameter is false, however, ALL possible charge distributions for the fragment ions are generated for every peptide bond.": "对于+3和更高电荷的前体离子，预测的碎片离子取决于此参数的设置方式。当此参数为真时，对于每个肽键，会进行内部计算以估计b和y片段序列的碱基性。然而，当此参数为假时，对于每个肽键，会为所有可能的碎片离子电荷分布生成。  1. 将句子拆分成若干部分，以便逐步翻译。 2. 识别并保留专业术语和参数，如\"+3\", \"precursors\", \"fragment ions\", \"parameter\", \"peptide bond\", \"b and y fragment sequence\", \"charge distributions\"。 3. 对剩余部分进行翻译，注意保持原文的逻辑和语法结构。 4. 将翻译后的各部分组合成完整的句子。",
    "For spectra with an ambiguous multiple charge state, it will be treated as any charge in the range: [+2,NumChargeStates].": "对于具有不明确多重电荷状态的谱图，它将被视为在范围：[+2,NumChargeStates]内的任何电荷。",
    "The maximum rank of a search result (results with the same score occupy the same rank).": "搜索结果的最高排名（具有相同分数的结果占据相同的排名）。",
    "For monoisotopic precursors where the monoisotope may have been incorrectly assigned to a nearby isotope, this range of adjustments will be considered. Instead of trying a wide precursor tolerance for a spectrum, this tries multiple tight tolerances.": "对于单同位素前体，如果单同位素可能被错误地分配给了附近的同位素，将考虑这种调整范围。这不是尝试一个宽的前体容差来分析一个谱图，而是尝试多个紧密的容差。",
    "Allows gene-centric analysis in IDPicker. The gene metadata embedding only works for human/mouse proteins from a RefSeq database. For best results, run idpQonvert with RefSeq no matter what database was used for the search: idpQonvert always remap the peptides anyway. Non-human/mouse proteins and any unmappable human/mouse proteins will get gene ids like \"Unmapped_<protein accession>\". This option, run by itself, is very fast.": "允许在IDPicker中进行基因中心分析。基因元数据嵌入仅适用于来自RefSeq数据库的人类/小鼠蛋白。为了获得最佳结果，无论用于搜索的是哪个数据库，都应运行idpQonvert：idpQonvert总是会重新映射肽段。非人类/小鼠蛋白和任何无法映射的人类/小鼠蛋白将获得类似\"Unmapped_<protein accession>\"的基因ID。这个选项单独运行时非常快。",
    "More embed options are available if the raw spectra are available.": "如果原始光谱可用，将提供更多的嵌入选项。",
    "Allows visualizing peptide-spectrum-matches without downloading the raw data. Embedding spectra will greatly increase the size of the database, even though only spectra that passed the import FDR filter will be included. This option can take a LONG time to run.": "允许在不下载原始数据的情况下可视化肽段-谱图匹配。嵌入谱图将大大增加数据库的大小，即使只有通过导入FDR过滤器的谱图才会被包含。这个选项可能需要很长时间才能运行。",
    "If the pepXML/mzIdentML file did not contain scan time (retention time) information, this will look up that information in the raw data. This option, run by itself, will take some time (it has to open every raw file).": "如果 pepXML/mzIdentML 文件没有包含扫描时间（保留时间）信息，这将从原始数据中查找该信息。单独运行此选项将需要一些时间（它必须打开每个原始文件）。",
    "Enables quantitation methods other than spectral counting. For isobaric isotope labelling quantitation (iTRAQ/TMT) or intensity-based label-free quantitation (XIC), select the appropriate QuantitationMethod here. You have to keep iTRAQ/TMT and label-free data separate since you can only specify a single QuantitationMethod for the entire assembly. Like embedding spectrum sources, this option can take a LONG time to run, although not quite as long.": "启用非光谱计数的定量方法。对于同位素标记定量（iTRAQ/TMT）或基于强度的无标记定量（XIC），请在此处选择适当的QuantitationMethod。您必须保持iTRAQ/TMT和无标记数据分开，因为您只能为整个装配指定单个QuantitationMethod。像嵌入光谱源一样，这个选项可能需要很长时间才能运行，尽管不会那么长。",
    "The FASTA protein database to map peptides against. Only peptides from target proteins will be mapped; decoy peptides will be stored as-is in the idpDB.": "用于映射肽段的FASTA蛋白质数据库。只有来自目标蛋白质的肽段会被映射；诱饵肽段将原样存储在idpDB中。",
    "The prefix to look for on protein accessions to determine whether it is a target or decoy.": "用于确定蛋白质访问号是目标还是诱饵的前缀。",
    "PSMs with a greater FDR than this will be excluded from the filtered data set, but remain in the file.": "具有大于该值的FDR的PSM将从过滤后的数据集中排除，但仍保留在文件中。",
    "PSMs with a greater FDR than this will be completely excluded from the idpDB.": "具有大于这个FDR的PSM将完全从idpDB中排除。",
    "For scores in the PSI-MS controlled vocabulary, this field must match the name for that score; for other scores, the UserParam name or pepXML name will suffice.": "对于PSI-MS控制词汇中的成绩，该字段必须与该成绩的名称匹配；对于其他成绩，UserParam名称或pepXML名称就足够了。",
    "Negative weights indicate that a lower score is better (e.g. expectation value, p-value).": "负权重表示较低的分数更好（例如期望值，p值）。",
    "Peptide-spectrum-matches (PSMs) with an FDR score (interpolated Q-value) higher than this will be excluded from the filtered data set.": "肽谱匹配（PSMs）具有高于此值的FDR得分（插值Q值）将从过滤后的数据集中排除。",
    "Apply filters at the gene level (i.e. 'min distinct peptides per gene group' instead of 'min distinct peptides per protein group')": "在基因水平上应用过滤器（即“每个基因组的最小独特肽段数”，而不是“每个蛋白质组的最小独特肽段数”）",
    "Gene groups with fewer than this number of peptides will be excluded from the filtered data set.": "基因组中肽段数量少于这个数字的将从过滤后的数据集中排除。",
    "Gene groups with fewer than this number of spectra will be excluded from the filtered data set.": "基因组中，如果谱图数量少于这个数值，将从过滤后的数据集中排除。",
    "Gene groups that are not necessary to explain the presence of at least this many extra peptides will be from the filtered data set. A value of 1 means that each gene group must explain at least 1 peptide that other gene groups do not explain.": "不需要解释至少存在这么多额外肽段的基因组将来自过滤后的数据集。值为1意味着每个基因组必须至少解释1个其他基因组不解释的肽段。",
    "Protein groups with fewer than this number of peptides will be excluded from the filtered data set.": "蛋白质组中肽段数量少于该数值的将从过滤后的数据集中排除。",
    "Protein groups with fewer than this number of spectra will be excluded from the filtered data set.": "蛋白质组中，如果谱图数量少于这个数值，将被排除在过滤后的数据集之外。",
    "Protein groups that are not necessary to explain the presence of at least this many extra peptides will be from the filtered data set. A value of 1 means that each protein group must explain at least 1 peptide that other protein groups do not explain.": "蛋白质组，如果不需要解释至少这么多额外的肽段的存在，将从过滤后的数据集中删除。值为1意味着每个蛋白质组必须至少解释1个其他蛋白质组不能解释的肽段。",
    "Distinct matches with fewer than this number of spectra will be excluded from the filtered data set.": "具有少于这个数字的谱图的独特匹配将从过滤后的数据集中排除。",
    "Distinct peptides with fewer than this number of spectra will be excluded from the filtered data set.": "具有少于这个数量的谱图的独特肽段将从过滤后的数据集中排除。",
    "Peptides that map to more than this number of protein groups will be excluded from the filtered data set. Highly ambiguous peptides are not very useful for quantitation.": "映射到超过这个数量的蛋白质组的肽段将从过滤后的数据集中排除。高度模糊的肽段对于定量不是很有用。",
    "A tab-delimited file that organizes source files (e.g. individual runs in a fractionated experiment) into groups. See below for more details.": "一个制表符分隔的文件，它将源文件（例如，在分馏实验中的单独运行）组织成组。有关更多详细信息，请参见下文。",
    "A tab-delimited file that gives sample names to isobaric reporter ion channels (i.e. iTRAQ, TMT) across a given source group. See below for more details.": "一个制表符分隔的文件，用于为等压报告离子通道（例如iTRAQ、TMT）提供样本名称（即等压标签），跨越给定的源组。有关更多详细信息，请参见下文。",
    "This file needs to be in OSW format (--in)": "这个文件需要是OSW格式（--in）",
    "Needs to have columns with Filename, Condition, BioReplicate, Run": "需要有包含文件名、条件、生物学重复和运行的列。",
    "As an approximation, the q-values of multiple runs are averaged and supplied as argument FFT. Numeric from 0 to 1.": "作为近似，多次运行的q值被平均，并作为参数FFT提供。数值范围从0到1。",
    "An m_score cutoff achieving and FDR smaller fdr_target will be selected. Calculated as FDR = decoys*FFT/targets": "一个达到并小于fdr_target的m_score截止值将被选择。计算公式为FDR = decoys*FFT/targets",
    "Number of measurements within at least one condition that have to pass the mscore threshold for this transition.": "至少在一个条件下，必须通过此转换的mscore阈值的测量次数。",
    "Maximum number of highest intense peptides to filter the data on.": "最高强度肽段的最大数量，用于筛选数据。",
    "Number of minimal number of peptide IDs associated with a protein ID in order to be kept in the dataset.": "为了保留在数据集中，与蛋白质ID相关联的最小数量的肽ID的数量。",
    "Specify the tolerance parameter in ppm.": "指定ppm（百万分之一）的容差参数。",
    "select the moFF workflow": "选择 moFF 工作流",
    "Specify rt window for xic in minutes.": "指定XIC的保留时间窗口，单位为分钟。",
    "Specify rt window for the peak in minutes.": "指定峰值的保留时间窗口，单位为分钟。",
    "Specify the retention time window for the matched peak in minutes.": "指定匹配峰保留时间窗口，单位为分钟。",
    "Specify the retention time window for the peak in minutes.": "指定峰值的保留时间窗口，以分钟为单位。",
    "Input file as imzML (composite upload) or tabular peaklist (regular upload)": "输入文件为imzML格式（复合上传）或表格形式的峰列表（常规上传）",
    "Choose Yes if peak detection has already been done. Peak detection cannot be run again on centroided data": "如果峰值检测已经完成，请选择“是”。在质心化数据上不能再运行峰值检测。",
    "This option only works for imzML files": "这个选项只适用于imzML文件",
    "Annotations can be used for group wise peak detection or filtering": "注释可用于组内峰值检测或筛选",
    "Tabular file with three columns: x values, y values and pixel annotations": "具有三列的表格文件：x值、y值和像素注释",
    "The resulting window reaches from mass[currentIndex-halfWindowSize] to mass[currentIndex+halfWindowSize] (window size is 2*halfWindowSize+1). The best size differs depending on the selected smoothing method.": "生成的窗口从 mass[currentIndex-halfWindowSize] 到 mass[currentIndex+halfWindowSize]（窗口大小为 2*halfWindowSize+1）。最佳大小取决于所选的平滑方法。",
    "Spectra with same annotation are averaged before peak detection, no imzML output": "具有相同注释的光谱在峰值检测之前进行平均，不输出imzML",
    "Minimal correlation between the peak pattern generated by the model and the experimental peaks in the MassPeaks object to be recognized as isotopic pattern.": "模型生成的峰模式与MassPeaks对象中的实验峰之间的最小相关性，被认为是同位素模式。",
    "Maximal relative deviation of a peak position (m/z) to be considered as identical: abs(((mass[i]+distance)-mass[i+1])/mass[i]) smaller than 'tolerance'. For 100ppm use 0.0001": "峰位（m/z）的最大相对偏差被认为是相同的：abs(((mass[i]+distance)-mass[i+1])/mass[i]) 小于 'tolerance'。对于100ppm使用0.0001",
    "Distance between two consecutive peaks in an isotopic pattern. 1.00235 is average distance for polypeptides. Multiple values can be used to find multiple charged pattern e.g. 1, 0.5 ,0.33": "在同位素模式中两个连续峰之间的距离。1.00235 是多肽的平均距离。可以使用多个值来查找多重带电模式，例如 1, 0.5, 0.33",
    "Size (length) of isotopic pattern, longer patterns are prefered over shorter ones, min size is 2, a range can be used.": "同位素模式的大小（长度），较长的模式比短的模式更受青睐，最小尺寸为2，可以使用范围。",
    "Maximal relative deviation of a peak position (m/z) to be considered as identical. For 50ppm use 0.00005 or 5e-5": "峰位（m/z）的最大相对偏差，以被视为相同。对于50ppm，使用0.00005或5e-5",
    "If given, samples will be aligned to reference, use internal calibrants to perform m/z calibration": "如果提供，样本将与参考序列对齐，使用内部校准物进行 m/z 校准",
    "Removal of all peaks which occur in less than minFrequency spectra to generate the reference m/z": "移除在少于minFrequency个谱图中出现的峰，以生成参考m/z",
    "Internal binning is omitted to avoid interaction between spectra": "内部分箱被省略以避免光谱之间的相互作用",
    "After the alignment the peak positions (mass) are very similar but not identical. The binning is needed to make similar peak mass values identical.": "对齐后，峰位（质量）非常相似但不完全相同。需要进行分箱，以使相似的峰质量值相同。",
    "strict - creates bins never containing two or more peaks of the same sample. relaxed - allows multiple peaks of the same sample in one bin.": "严格 - 创建的bin永远不包含同一样本的两个或更多峰。宽松 - 允许一个bin中存在同一样本的多个峰。",
    "By default a single group is assumed and filtering will be done based on all pixels. To filter groups of spectra, an annotation annotation file has to be specified above": "默认情况下，假设为单一组，并将基于所有像素进行过滤。要过滤光谱组，必须在上面指定注释注释文件。",
    "Yes means that peaks that survive the filtering in one annotation group are also kept in other groups regardless if the filtering criteria are met in these groups": "是的，这意味着在一个注释组中通过过滤的峰值也会保留在其他组中，而不管这些组中是否满足过滤标准。",
    "minFrequency: Removal of all peaks which occur in less than minFrequency spectra (relative treshold). minNumber: Removal of all peaks which ocur in lass than minNumber spectra (absolute threshold). If both are set the higher value from relative and absolute threshold is taken.": "最小频率：移除在少于最小频率（相对阈值）的光谱中出现的峰。最小数量：移除在少于最小数量（绝对阈值）的光谱中出现的峰。如果两者都设置，则采用相对和绝对阈值中的较高值。",
    "( --tric_chromprob / --no-tric_chromprob)": "( --tric_chromprob / --no-tric_chromprob)",
    "Ranges from 0 (min, fast) to 3 (max, slow)": "范围从0（最小，快速）到3（最大，慢速）",
    "Max number of overlapping features in any mz window. Keep as low as necessary for speed": "在任何mz窗口中重叠特征的最大数量。为了速度，尽可能保持低值。",
    "Peptides persisting for longer than this value are excluded": "肽段如果持续时间超过这个值则被排除",
    "Minimum consecutive scans peptide must be found in": "最小连续扫描次数，必须在其中找到肽段",
    "Maximum amount of scans a peptide can skip to be considered in results": "一个肽段在结果中被考虑时可以跳过的扫描的最大数量",
    "Maximum mass of peptide isotope distribution": "肽同位素分布的最大质量",
    "Minimum mass of peptide isotope distribution": "肽同位素分布的最小质量",
    "Peptides in adjacent scans must differ by less than this value to be called the same.": "相邻扫描中的肽段必须相差小于这个值才能被认为是相同的。",
    "tabular file containing peptides for the alignment, e.g. iRTs or ciRTs": "包含用于比对的肽段的表格文件，例如 iRTs 或 ciRTs。",
    "This file needs to be in OSW format, an SQLite variant.": "这个文件需要是OSW格式的，SQLite的一个变种。",
    "This file needs to be in OSW format.": "这个文件需要是OSW格式的。",
    "MetaMorpheus,Morpheus,PeptideShaker PSM Report,MaxQuant": "MetaMorpheus、Morpheus、PeptideShaker PSM 报告、MaxQuant",
    "Leave blank to detemine emperically from data.": "留空以从数据中经验性地确定。",
    "Reporter label maximum peak intensity threshold to consider the tag for quantitation. Default is 0. It depends on signal to noise ratio (S/N)": "报告标签最大峰值强度阈值以考虑标签进行定量。默认值为0。这取决于信号与噪声比（S/N）",
    "A proteoform defines a specific state of a protein. It is composed by the protein UniProt accession, isoform and set of post translational modifications. The input file contains one line for each proteoform. Each PTM is specified using a modification identifier and a site, separated by ':'(semicolon). For example: '00046:133'. The identifier is a 5 digit id from the PSI-MOD Protein Modification Onthology [6].": "一个蛋白质形态定义了一个特定状态的蛋白质。它由蛋白质UniProt访问号、异构体和一组翻译后修饰组成。输入文件包含每个蛋白质形态的一行。每个PTM（翻译后修饰）使用修饰标识符和位点指定，由冒号 ':'（分号）分隔。例如：'00046:133'。该标识符是PSI-MOD蛋白质修饰本体[6]中的一个5位数字id。",
    "Plus minus positions for the same PTM site": "同一PTM位点的正负位置",
    "File with a one gene name in each line. Genes follow the HUGO gene nomenclature[3].": "每行包含一个基因名称的文件。基因遵循HUGO基因命名法[3]。",
    "File with a one Uniprot Accession [4] in each line.": "每行包含一个Uniprot访问号[4]的文件。",
    "File with a one Ensembl identifier [5] in each line.": "每行包含一个Ensembl标识符的文件。",
    "The input follows the Variant Call Format Specification[2] v4.3. It also allows the possibility to specify only the first 4 columns in the data section of the file: CHROM, POS, ID, REF.": "输入遵循变体调用格式规范[2] v4.3。它还允许在文件的数据部分中仅指定前4列：CHROM（染色体）、POS（位置）、ID（标识符）、REF（参考序列）。",
    "Genetic variants can also be represented using the chromosome and the base pair numbers. The input should be sorted by chromosome number and then by base pair.": "遗传变异也可以用染色体和碱基对数来表示。输入应先按染色体编号排序，然后按碱基对排序。",
    "The file contains one rsid identifier as defined in dbSNP[1] on each row. The list must be ordered by chromosome and base pair (bp). The list must not have duplicates. All rsids must appear in the human assembly GRCh37.p13.": "该文件每行包含一个在dbSNP[1]中定义的rsid标识符。列表必须按染色体和碱基对（bp）排序。列表中不能有重复项。所有的rsid都必须出现在人类基因组组装GRCh37.p13中。",
    "File with a one peptide sequence in each line.": "每行包含一个肽段序列的文件。",
    "Each line of the file corresponds to a single peptide with post-translational modifications.": "文件中的每一行对应一个带有翻译后修饰的肽段。",
    "Integer number margin error for sites of PTMs. Only for modified peptides.": "整数数量边距错误，针对PTMs（翻译后修饰）位点。仅针对修饰肽段。",
    "Generates a zipped file with connection graphs as an additional output when executing the pathway search and analysis. The graph can use genes, proteins or proteoforms as vertices.": "在执行通路搜索和分析时，生成一个包含连接图的压缩文件作为额外输出。该图可以使用基因、蛋白质或蛋白质异构体作为顶点。",
    "Occurs in known places on peptide sequence. Hold the appropriate key while clicking to select multiple items": "出现在肽序列的已知位置。按住相应的键，同时单击以选择多个项目",
    "Can occur anywhere on the peptide sequence; adds additional error to search score. Hold the appropriate key while clicking to select multiple items": "可以出现在肽段序列的任何位置；为搜索得分增加额外的错误。在点击选择多个项目时，按住适当的键。",
    "Which enzyme was used for protein digest in experiment? In most cases, trypsin is used": "在实验中使用了哪种酶进行蛋白质消化？在大多数情况下，使用的是胰蛋白酶。",
    "Lowest searched charge value for fragment ions": "片段离子的最低搜索电荷值",
    "Highest searched charge value for fragment ions": "片段离子的最高搜索电荷值",
    "Searched fragment ion type. Select a, b or c based on collisions induced in experiment": "搜索片段离子类型。根据实验中诱导的碰撞，选择 a、b 或 c",
    "Searched fragment ion type. Select x, y, or z based on collisions induced in experiment": "搜索片段离子类型。根据实验中诱导的碰撞，选择x、y或z",
    "Sets the dynamic range for scoring spectra": "设置用于评分谱图的动态范围",
    "Maximum number of peaks to be used from a spectrum": "从谱图中使用的最大峰值数",
    "Fragment mass peaks with m/z less than this value will be discarded": "小于这个值的m/z的片段质量峰将被丢弃",
    "Minimum number of peaks required for a spectrum to be considered": "一个谱图被考虑所需的最小峰值数量",
    "Use noise suppression": "使用噪声抑制",
    "Minimum mass of 1+ mass of parent ion to be considered": "考虑的最小质量为1+母离子的质量",
    "when activated, the parent ion mass tolerance is expanded by opening up multiple tolerance windows centered on the first and second 13C isotope peaks for a peptide.": "当激活时，父离子质量容差通过打开多个容差窗口来扩大，这些窗口以肽的前两个13C同位素峰为中心。",
    "Protein N-terminal modification detection": "蛋白质N端修饰检测",
    "Peptide N-terminus cyclization detection": "肽N端环化检测",
    "Interpretation of peptide phosphorylation models": "肽段磷酸化模型的解释",
    "maximum number of variable modification alternatives that will be tested for a particular peptide": "对于特定肽段，将测试的可变修饰替代物的最大数量",
    "Determines which results are written to the output file at the end of a modelling session": "确定在建模会话结束时哪些结果被写入输出文件",
    "Highest value for recorded peptides": "记录肽段的最高值",
    "Controls output of protein sequences": "控制蛋白质序列的输出",
    "Controls output of sequence information": "控制序列信息的输出",
    "Controls output of spectrum information": "控制输出的谱信息",
    "Controls output of statistical information about an spectrum-to-sequence assignment": "控制输出关于谱图到序列分配的统计信息",
    "Highest value for recorded peptides during refinement": "在细化过程中记录的肽段的最高值",
    "Allow for unanticipated cleavage during refinement": "允许在细化过程中发生未预料到的剪切",
    "Search for semi-tryptic peptides during refinement": "在细化过程中搜索半胰蛋白酶肽",
    "Controls the use of refinement modifications in all refinement modules.": "控制所有细化模块中细化修饰的使用。",
    "Allow for point mutations during refinement": "在细化过程中允许点突变",
    "Search for known single amino acid polymorphisms during refinement": "在细化过程中搜索已知的单氨基酸多态性",
    "Use spectrum synthesis scoring": "使用谱图合成评分",
    "Minimum length for a peptide to be considered": "考虑肽段的最小长度",
    "Maximum length for a peptide to be considered": "考虑一个肽段的最大长度",
    "Minimum precursor mass of parent ion to be considered": "考虑的母离子前体质量的最小值",
    "Maximum precursor mass of parent ion to be considered": "考虑母离子的最大前体质量",
    "Set the value for the maximum number of spectrum matches": "设置最大谱图匹配数的值",
    "Set the number of PTMS allowed per peptide": "设置每个肽段允许的磷酸化修饰(PTM)数量",
    "Choose the fragmentation method used (CID: b,y) or (ETD: c, z*)": "选择使用的碎片化方法（CID: b,y）或（ETD: c, z*）",
    "Select the number of enzymatic terminals": "选择酶切末端的数量",
    "Defines what algorithms are used to generate a set of theoretical fragment ions": "定义用于生成一组理论片段离子的算法",
    "a Sequest-like cross correlation score can be calculated for the top ranking hits": "一个类似Sequest的交叉相关分数可以为排名靠前的命中计算",
    "Cumulative ion current of picked peaks divided by TIC >= this value for peaks to be retained (0.0 - 1.0)": "选定峰值的累积离子电流除以总离子电流（TIC）大于或等于此值时保留峰值（0.0 - 1.0）",
    "Experimental spectra have their peaks stratified into this number of intensity classed": "实验谱图的峰值被划分为这个数量的强度等级",
    "Has to do with previous option, this parameter controls the size of each class relative to the class above": "与前一个选项有关，这个参数控制每个类别相对于上面类别的大小",
    "The number of batches per node to strive for when usinge the MPI-based parallelization features": "在使用基于MPI的并行化特性时，每个节点要努力达到的批次数",
    "If yes then a decoy database will be generated and searched. Assumed input database contains no decoys": "如果选择是，则会生成并搜索一个诱饵数据库。假定输入的数据库不包含诱饵。",
    "Identifier of the instrument to generate MS/MS spectra (used to determine the scoring model)": "仪器标识符，用于生成MS/MS谱图（用于确定评分模型）",
    "Fragmentation method identifier (used to determine the scoring model)": "片段化方法标识符（用于确定评分模型）",
    "Protocol identifier. Protocols are used to enable scoring parameters for enriched and/or labeled samples": "协议标识符。协议用于为富集和/或标记的样本启用评分参数",
    "Searches will take much longer if selecting a value other than 2": "如果选择除2以外的值，搜索将需要更长的时间",
    "Number of peptide matches per spectrum to report": "每个谱图报告的肽段匹配数",
    "Additional features to export": "额外的导出功能",
    "Manually set the number of tasks to create for the search. More tasks than threads will reduce the memory requirements of the search, but will be slower (how much depends on the inputs). Check the documentation carefully.": "手动设置要为搜索创建的任务数。任务数多于线程数将减少搜索的内存需求，但速度会变慢（具体程度取决于输入）。请仔细查看文档。",
    "generate decoys": "生成诱饵序列",
    "MS Amanda instrument id option. Available ion types are listed here.": "MS Amanda 仪器 ID 选项。可用的离子类型在此列出。",
    "MS Amanda maximum rank": "MS Amanda 最大等级",
    "MS Amanda use monoisotopic mass values": "MS 阿曼达使用单同位素质量值",
    "Low intensity cutoff as a fraction of max peak": "最大峰值的低强度截止值作为分数",
    "High intensity cutoff as a fraction of max peak": "高强度截止作为最大峰值的分数",
    "Intensity increment": "强度增量",
    "The minimum number of m/z values a spectrum must have to be searched": "谱图必须具有的最小 m/z 值数量才能进行搜索",
    "Eliminate charge reduced precursors in spectra": "在谱图中消除电荷减少的前体",
    "estimate precursor charge": "估计前体电荷",
    "Allow OMSSA to estimate plus one charge algorithmically": "允许OMSSA算法估计额外的一个电荷",
    "fraction of peaks to estimate charge 1": "用于估计电荷1的峰值比例",
    "Minimum number of precursors per spectrum": "每个谱图的前体分子最小数量",
    "scale precursor mass": "前体质量缩放",
    "Use memory mapped sequence libraries": "使用内存映射的序列库",
    "Allow for N-terminal methionine cleavage": "允许N端甲硫氨酸的剪切",
    "Mass after which OMSSA should consider neutron exact mass": "OMSSA 在考虑中子精确质量时的质量",
    "Single charge window width in Da (integer)": "单电荷窗口宽度（以道尔顿为单位，整数）",
    "OMSSA double charge window width in Da (integer)": "OMSSA 双电荷窗口宽度，以道尔顿（整数）为单位",
    "Minimum number of peaks in single charge window (integer)": "单个电荷窗口中的最小峰值数量（整数）",
    "Minimum number of peaks in double charge window (integer)": "双电荷窗口中峰值的最小数量（整数）",
    "Minimum number of annotated peaks among the most intense ones": "在最强烈的峰值中注释的峰值的最小数量",
    "Minimum number of annotated peaks": "最小注释峰值数量",
    "The maximum number of mass ladders to generate per database peptide": "每个数据库肽段生成的最大质量梯度数",
    "Maximum fragment charge": "最大片段电荷",
    "OMSSA fragment charge option": "OMSSA 片段电荷选项",
    "Allow OMSSA to include first forward ion (b1) in search": "允许OMSSA在搜索中包含第一个前向离子（b1）",
    "Allow search rewind (C-terminal) ions": "允许搜索回溯（C端）离子",
    "Max number of fragments ions ions in each series being searched": "每个系列中搜索的片段离子的最大数量",
    "Allow the use correlation correction score": "允许使用相关性校正分数",
    "Probability of consecutive ion (used in correlation correction)": "连续离子的概率（用于相关性校正）",
    "number of hits per spectrum per charge": "每个谱图每个电荷的命中数",
    "The maximum e-value allowed to consider a sequence in the iterative search(0.0 means all)": "在迭代搜索中考虑序列的最大e值（0.0表示全部）",
    "The maximum e-value allowed to consider a spectrum in the iterative search(0.0 means all)": "在迭代搜索中考虑一个谱图所允许的最大e值（0.0表示全部）",
    "The maximum e-value allowed to replace a hit in the iterative search (0.0 means all)": "在迭代搜索中允许的最大e值以替换命中结果（0.0表示全部）",
    "Minimum length of peptides for no-enzyme and semi-tryptic searches": "无酶切和半酶切搜索的肽段最小长度",
    "Maximum length of peptides for no-enzyme and semi-tryptic searches (0: none)": "无酶和半胰蛋白酶搜索中肽段的最大长度（0：无限制）",
    "The maximum e-value considered": "考虑的最大 e 值",
    "OMSSA hitlist length, 0 means all": "OMSSA 命中列表长度，0 表示全部",
    "The minimum number of peaks per spectrum": "每个谱图的最小峰值数量",
    "The minimum intensity for input peaks to be considered": "输入峰值需考虑的最小强度",
    "Select for precursor m/z signal removal": "选择前体 m/z 信号去除",
    "Intended for iTRAQ/TMT type data where one might want to remove the reporter ion signals, lower m/z range": "适用于iTRAQ/TMT类型的数据，其中可能需要去除报告离子信号，降低m/z范围。",
    "Intended for iTRAQ/TMT type data where one might want to remove the reporter ion signals, upper m/z range": "适用于iTRAQ/TMT类型的数据，其中可能需要去除报告离子信号，上m/z范围",
    "Specifies the number of enzyme termini a peptide must have": "指定一个肽段必须拥有的酶切末端数量",
    "Controls whether the peptide_mass_tolerance takes into account possible isotope errors in the precursor mass measurement": "控制peptide_mass_tolerance是否考虑前体质量测量中可能的同位素误差",
    "The minimum precursor mass considered": "考虑的最小前体质量",
    "The maximum precursor mass considered": "考虑的最大前体质量",
    "Sets the maximum fragment charge (fill value between 1 and 5)": "设置最大片段电荷（填充值介于1到5之间）",
    "Specifies whether the N-terminal methionine is cleaved prior to matching": "指定在匹配之前是否剪切N端的甲硫氨酸",
    "0 means load and search all spectra at once, otherwise spectra are loaded and searched in batches of the number specified": "0表示一次性加载并搜索所有谱图，否则以指定数量的批次加载和搜索谱图",
    "The maximum number of ptms per peptide": "每个肽段的最大修饰数",
    "Require at least one variable PTM per peptide": "每个肽段至少需要一个修饰变量（PTM）。",
    "Specifies how theoretical fragment ion peaks are represented (on the sum in the bin alone or using flanking bins too)": "指定理论片段离子峰如何表示（仅在箱中求和，还是也使用相邻箱）",
    "Controls how each fragment bin is defined in terms of where each bin starts": "控制每个片段箱是如何定义的，即每个箱从哪里开始",
    "The maximum number of peptide matches per spectrum": "每个谱图匹配的肽段最大数量",
    "Output type: PepXML, SQT, TXT or Percolator": "输出类型：PepXML、SQT、TXT 或 Percolator",
    "Set the minimum length of peptide to be considered": "设置考虑的肽段的最小长度",
    "Set the maximum length of peptide to be considered": "设置要考慮的肽段的最大長度",
    "Set the minimum precursor mass to be considered": "设置要考虑的最小前体质量",
    "Set the maximum precursor mass to be considered": "设置要处理的最大前体质量",
    "If true, the precursor mass is monoisotopic": "如果为真，前体质量是单同位素的",
    "If true, the Nterm Methionine will be clipped": "如果为真，N端甲硫氨酸将被剪切",
    "Set the maximum number of PTMs on peptide to be considered": "设置要考虑的肽段上最大PTM（翻译后修饰）数量",
    "Set the maximum number of PTMs of each type to be considered": "设置每种类型的PTM（翻译后修饰）考虑的最大数量",
    "Either both ends (full-digest) or at least one end (partial-digest) of a peptide must conform to enzyme specificity rules": "肽段的两端（完全消化）或至少一端（部分消化）必须符合酶的特异性规则",
    "If true, the peptides will be printed in the output": "如果为真，肽段将在输出中打印",
    "Select the format for generating the decoy sequences": "选择生成诱饵序列的格式",
    "Select to keep the terminal amino acids when creating decoys": "选择在创建诱饵时保留末端氨基酸",
    "Set the decoy seed": "设置诱饵种子",
    "If true, the temp folders are removed when the search is done.": "如果为真，搜索完成后将删除临时文件夹。",
    "If true, the exact p-values are calculated": "如果为真，将计算确切的P值",
    "If true, the SP-score is calculated": "如果为真，则计算SP分数",
    "Set the minimum spectrum m/z value for a spectrum to be considered": "设置一个谱图被考虑的最小质荷比（m/z）值",
    "Set the maximum spectrum m/z value for a spectrum to be considered": "设置最大谱图 m/z 值，以考虑谱图",
    "Set the minimum amount of peaks in a spectrum for it to be considered": "设置一个谱图中被认为是有效的最小峰值数量",
    "Select what precursor charges should be taken into account for matching": "选择在匹配时应考虑的前体电荷",
    "If true, the peak that corresponds to the precursor mass is excluded": "如果为真，则排除对应于前体质量的峰值",
    "Choose the threshold for precursor mass searching (for precursor peak removal)": "选择前体质量搜索的阈值（用于前体峰值去除）",
    "Includes two flanking peaks on either side of each b- and y-ion to compute the XCorr": "在每个 b- 和 y- 离子的两侧各包含两个邻近峰，以计算 XCorr",
    "Includes fragment peaks with neutral losses to perform the matching": "包括具有中性丢失的片段峰以执行匹配",
    "Choose bin size to analyze the fragmentation spectrum": "选择一个区间大小来分析片段化谱",
    "Choose bin offset to analyze the fragmentation spectrum": "选择bin偏移量以分析片段化谱",
    "Set the maximum number of PSMs to be considered": "设置要考虑的最大PSM（肽段匹配）数量",
    "Tide output folder (relative to the Tide working folder). Not to be changed if the .par file will be used by SearchGUI in Galaxy": "Tide输出文件夹（相对于Tide工作文件夹）。如果.par文件将由Galaxy中的SearchGUI使用，则不要更改。",
    "Select the display verbosity level to report the search progress": "选择显示详细程度级别以报告搜索进度",
    "Choose the progress indicator frequency  (in number of fragmentation spectra processed)": "选择进度指示器频率（处理的片段化谱图数量）",
    "If true, the target results are concatenated with the decoy results": "如果为真，目标结果将与诱饵结果连接起来",
    "Tide file name in with to store the binary spectra. Not to be changed if the .par file will be used by SearchGUI in Galaxy": "Tide文件名，用于存储二进制谱图。如果.par文件将由Galaxy中的SearchGUI使用，则不要更改。",
    "Identifier of the instrument to generate MS/MS spectra": "生成MS/MS谱图的仪器标识符",
    "The intensity percentile to consider for annotation, e.g. 0.75 means that the 25% most intense peaks will be annotated, default is 0.75.": "用于注释的强度百分位数，例如 0.75 表示将注释最高强度的 25% 的峰，默认值为 0.75。",
    " The m/z tolerance to annotate peaks, default is equal to the search settings MS2 tolerance.": "用于注释峰的 m/z 容差，默认等于搜索设置中的 MS2 容差。",
    "The most accurate peak will be selected within the m/z tolerance if chosen": "如果在m/z容差内选择了最准确的峰值",
    "0.25 means 25% of Xs": "0.25表示X的25%",
    "The higher precursor mass error allowed in order to filter out outliers": "允许的更高的前体质量误差，以便过滤掉异常值",
    "Default is no filter": "默认不进行过滤",
    "If true, peptides presenting unrecognized PTMs will be excluded": "如果为真，将排除展示未识别的PTMs的肽段",
    "Automatic mode will be used if not set": "如果未设置，则使用自动模式",
    "This should only be enabled for UniProt databaases": "这应该仅对UniProt数据库启用",
    "In percent (default 1% FDR: '1')": "在百分比（默认1% FDR：'1'）",
    "default 95%: '95.0'": "默认 95%：'95.0'",
    "Selecting this option will help PeptideShaker calculate FDR values": "选择此选项将帮助PeptideShaker计算FDR值",
    "SearchGUI Results from History": "SearchGUI 结果来自历史记录",
    "This optional parameter file will be used instead of the default one": "这个可选的参数文件将被用来替代默认的文件",
    "This optional fasta file will be used instead of the one included in the SearchGUI file (if it exists). If the SearchGUI file does not include the fasta file, this parameter is must be considered as mandatory.": "这个可选的fasta文件将被用来替代SearchGUI文件中包含的那个（如果存在的话）。如果SearchGUI文件没有包含fasta文件，那么这个参数就必须被考虑为强制性的。",
    "Optional spectrum files will be used together with spectra included in the SearchGUI output": "可选的谱图文件将与SearchGUI输出中包含的谱图一起使用",
    "All different outputs PeptideShaker may produce": "PeptideShaker 可能产生的所有不同输出",
    ".cui index files are exported as a different collection too": ".cui 索引文件也被导出为不同的集合",
    "Create a Galaxy workflow to save these values": "创建一个 Galaxy 工作流来保存这些值",
    "Commonly: spectra of Non-Validated PSMs": "通常：非验证的肽段匹配的谱图",
    "Commonly: Main Accession of Validated Protein Groups": "通常：验证蛋白质组的主要接入号",
    "Commonly: Validated PSMs of Validated Peptides of Validated Proteins": "通常：验证蛋白质的验证肽段的验证PSMs",
    "Commonly: Thermo": "通常：Thermo",
    "For postprocessing with PeptideShaker the sequences must have UniProt style headers and contain decoy sequences (see help).": "对于使用PeptideShaker进行后处理，序列必须具有UniProt风格的标题，并且包含诱饵序列（见帮助）。",
    "Select appropriate MGF/MZML dataset(s) from history. Remember to choose only .mzML files when using MetaMorpheus.": "从历史记录中选择适当的MGF/MZML数据集。使用MetaMorpheus时，请确保只选择.mzML文件。",
    "Unless you suspect some input files to be genuine duplicates then rename spectra is the safest option": "除非你怀疑某些输入文件是真正的重复，那么重命名谱图是最安全的选择",
    "Choose a smaller value if you are running on a machine with limited memory": "如果您在具有有限内存的机器上运行，请选择一个较小的值",
    "Select the raw file from history": "从历史记录中选择原始文件",
    "This tool uses a proprietary vendor library; to run it you must agree to the vendor license. Read it at https://github.com/kevinkovalchik/RawTools/blob/master/RawFileReaderLicense": "这个工具使用了专有的供应商库；要运行它，您必须同意供应商许可证。您可以在 https://github.com/kevinkovalchik/RawTools/blob/master/RawFileReaderLicense 上阅读它。",
    "Parses raw file meta and scan data and creates a tab-delimited text file.": "解析原始文件元数据和扫描数据，并创建一个制表符分隔的文本文件。",
    "Also quantifies reporter ions and write results to output matrix.": "还量化报告离子，并将结果写入输出矩阵。",
    "Reagents used to label peptides.": "用于标记肽段的试剂。",
    "Calculate areas of precursor peaks and writes them to the matrix file as ParentPeakArea column.": "计算前体峰的面积，并将它们写入矩阵文件作为ParentPeakArea列。",
    "Creates a standard MGF file.": "创建一个标准的MGF文件。",
    "Specify a mass cutoff to be applied when generating MGF files. May be of use if removal of reporter ions is desired prior to searching of MS2 spectra.": "指定在生成MGF文件时应用的质量截止值。如果希望在搜索MS2谱图之前去除报告离子，可能会有用。",
    "Refine precursor charge and monoisotopic mass assignments. Highly recommended if monoisotopic precursor selection was turned off in the instrument method (or peptide match on a QE instrument).": "精炼前体电荷和单同位素质量分配。如果仪器方法中关闭了单同位素前体选择（或在QE仪器上进行了肽段匹配），强烈推荐使用此功能。",
    "Creates a text file of chromatographic data for investigation or plotting.": "创建一个用于调查或绘图的色谱数据文本文件。",
    "MS Level": "质谱级别",
    "MS Order": "MS 顺序",
    "Creates a text file containing general metrics about the Ms run.": "创建一个文本文件，包含有关 Ms 运行的一般指标。",
    "Names inputs: prefix_rep#.mzXML Leave blank to use History names of input": "命名输入：前缀_rep#.mzXML 如果想使用历史名称作为输入，请留空",
    "Sets default parameters": "设置默认参数",
    "Usually do not need to be changed": "通常不需要更改",
    "Output detailed information about detected MS1 precursor and MS2 unfragmented precursor signals": "输出关于检测到的MS1前体和MS2未碎裂前体信号的详细信息",
    "Output detailed information about detected MS2 signals": "输出检测到的MS2信号的详细信息",
    "See `Annotations` section of help below": "查看下面帮助中的“注释”部分",
    "The genome annotation must have been prepared with dbSNP.": "基因组注释必须使用dbSNP进行了准备。",
    "The genome annotation must have been prepared with COSMIC.": "基因组注释必须使用COSMIC准备。",
    "A dataframe of exon annotations in an RData file": "一个包含外显子注释的 RData 文件中的 dataframe",
    "A dataframe containing protein ids and protein sequences in an RData file": "一个包含蛋白质ID和蛋白质序列的RData文件的数据框",
    "A dataframe cotaining coding sequences for each protein in an RData file": "一个包含每个蛋白质的编码序列的数据框在RData文件中",
    "A dataframe cotaining IDs for each protein in an RData file": "一个包含每个蛋白质ID的数据框在一个RData文件中",
    "If non-zero, if a transcript does not meet this expression cutoff (based on RPKM) then it will not be included in the output database.": "如果非零，如果一个转录本没有达到这个基于RPKM的表达量截止值，那么它将不会被包含在输出数据库中。",
    "The PSM2SAM tool needs this to map variant proteins to genomic locations": "PSM2SAM 工具需要这个功能，以便将变异蛋白质映射到基因组位置。",
    "Protein abundance input file": "蛋白质丰度输入文件",
    "Transcript abundance input file": "转录本丰度输入文件",
    "If the input files already have fold-change values, select Log fold-change. Else, select abundances and the tool will perform the fold-change analysis": "如果输入文件已经有倍数变化值，选择“对数倍数变化”。否则，选择丰度，工具将执行倍数变化分析。",
    "Please check the format of the experiment design file": "请检查实验设计文件的格式",
    "Perform T-Test on selecting Mean; Wilcoxon Ranksum Test on selecting Median": "执行T检验以选择均值；执行Wilcoxon秩和检验以选择中位数。",
    "Input file as imzML (composite upload). The file must be in profile mode, not centroided.": "输入文件为imzML格式（组合上传）。文件必须处于轮廓模式，而不是质心化模式。",
    "This method smoothes the intensity values of a MassSpectrum object.": "这种方法平滑了质谱对象的强度值。",
    "Controls the order of the filter, should be smaller than the resulting window.": "控制滤波器的阶数，应小于结果窗口的大小。",
    "Indicates if the average should be equal weight or if it should have weights depending on the distance from the center as calculated as 1/2^abs(-halfWindowSize:halfWindowSize) with the sum of all weigths normalized to 1": "表明平均值应该是等权重的，还是应该有权重，权重取决于与中心的距离，计算方式为1/2^abs(-halfWindowSize:halfWindowSize)，所有权重的总和归一化到1",
    "Number of data points, the resulting window reaches from mass[currentIndex-halfWindowSize] to mass[currentIndex+halfWindowSize] (window size is 2*halfWindowSize+1).": "数据点的数量，结果窗口从 mass[currentIndex-halfWindowSize] 延伸到 mass[currentIndex+halfWindowSize]（窗口大小为 2*halfWindowSize+1）。",
    "Corresponds to half window size: The resulting window reaches from mass[cur_index-iterations] to mass[cur_index+iterations]": "对应于半窗口大小：结果窗口从 mass[cur_index-iterations] 到 mass[cur_index+iterations]",
    "Number of data points, the resulting window reaches from mass[currentIndex-halfWindowSize] to mass[currentIndex+halfWindowSize]": "数据点的数量，所得窗口从质量[currentIndex-halfWindowSize]延伸到质量[currentIndex+halfWindowSize]",
    "Intensity normalization": "强度标准化",
    "Instead of the whole m/z range, a specified m/z range can be used to calculate the scaling factor.": "而不是整个 m/z 范围，可以使用指定的 m/z 范围来计算缩放因子。",
    "Scaling factor is calculated on the mass range and applied to the whole spectrum.": "缩放因子是根据质量范围计算的，并应用于整个光谱。",
    "abs(mz1 - mz2)/mz2, maximal relative deviation of a peak position (m/z) to be considered as identical. For 50ppm use 0.00005 or 5e-5": "mz1和mz2之间的绝对差值除以mz2，这是考虑峰位（m/z）相同允许的最大相对偏差。对于50ppm，使用0.00005或5e-5",
    "Number of data points, the resulting window reaches from mass[currentIndex-halfWindowSize] to mass[currentIndex+halfWindowSize] (window size is 2*halfWindowSize+1). The best size differs depending on the selected smoothing method.": "数据点的数量，所得窗口从 mass[currentIndex-halfWindowSize] 到 mass[currentIndex+halfWindowSize]（窗口大小为 2*halfWindowSize+1）。最佳大小取决于所选的平滑方法。",
    "otherwise continuous imzML will be exported": "否则，连续的imzML将被导出",
    "Ex: k99.contigs.fa": "例如：k99.contigs.fa",
    "Usually indicated within the filename (when derived from MEGAHIT)": "通常在文件名中指示（当来自MEGAHIT时）",
    "No value means no maximum": "没有值意味着没有最大值",
    "Pools containing batches (optional, not using this will result in one pool)": "包含批次的池（可选，不使用这个将导致一个池）",
    "Specify part of the input filenames that are unique for each pool, e.g set_A. Do not use 3 consecutive underscores.": "为每个池指定输入文件名中唯一的一部分，例如 set_A。不要使用3个连续的下划线。",
    "Set by cross validation if 0.": "如果为0，则由交叉验证设置。",
    "Can be negated to indicate lower value=better, e.g -featureName": "可以被否定以表示较低的值更好，例如-featureName",
    "and no fallback on default score vector in case of suspect score vector": "并且在怀疑得分向量的情况下不要回退到默认得分向量",
    "This option will deactivate the quantifiation of data.": "这个选项将停用数据的定量分析。",
    "MSGF+ uses XXX_ as an accession prefix to indicate a decoy hit": "MSGF+ 使用 XXX_ 作为访问号前缀来表示诱饵命中",
    "Error tolerance for matching peptide mass to precursor ion mass": "匹配肽质量与前体离子质量时的错误容忍度",
    "Daltons are common for low-res instruments, ppm for high-res instruments": "道尔顿常用于低分辨率仪器，ppm常用于高分辨率仪器",
    "The instrument type that generated the MS/MS spectra is used to determine the scoring model": "生成MS/MS谱图的仪器类型用于确定打分模型",
    "Enzyme used to digest proteins in sample preparation; trypsin is the most commonly used enzyme": "用于样品制备中消化蛋白质的酶；胰蛋白酶是最常用的酶",
    "Semi-specific requires more time than fully specific; non-specific requires much more.": "半特异性比完全特异性需要更多的时间；非特异性需要的时间则更多。",
    "Specify modifications with custom parameters": "使用自定义参数指定修饰",
    "If this mod has an entry there in Unimod, this name should match its name there": "如果这个修饰在Unimod中有条目，那么这个名字应该与那里的名字相匹配",
    "Minimum precursor charge to consider if charges are not specified in the spectrum file": "如果谱图中没有指定电荷，则考虑的最小前体电荷",
    "Maximum precursor charge to consider if charges are not specified in the spectrum file": "如果谱图中没有指定电荷数，则考虑的最大前体电荷",
    "Minimum peptide length to consider": "最小考虑的肽段长度",
    "Maximum peptide length to consider": "考虑的最大肽段长度",
    "Protocols are used to enable scoring parameters for enriched and/or labeled samples": "协议用于为富集和/或标记的样本启用评分参数",
    "If true, several extra derivative scores are calculated for each match": "如果为真，将为每个匹配计算几个额外的派生分数",
    "Takes into account of the error introduced by chooosing a non-monoisotopic peak for fragmentation (-ti)": "考虑了由于选择非单同位素峰进行碎裂（-ti）引入的误差",
    "Relative size ratio for the randomly permuted negative instances w.r.t. the positive instances.": "相对于正样本，随机排列的负样本的相对大小比例。",
    "Order of the k-mer for the random shuffling procedure.": "随机洗牌程序中k-mer的顺序。",
    "This should be in FASTA format.": "这应该是FASTA格式。",
    "(-in_id)": "(-in_id)",
    "(-in_text)": "(-输入文本)",
    "(-in_oligo_params)": "(-寡核苷酸参数)",
    "(-in_oligo_trainset)": "(-in_oligo_trainset)",
    "(-svm_model)": "(-svm模型)",
    "(-total_gradient_time)": "（总梯度时间）",
    "(-max_number_of_peptides)": "（最大肽段数）",
    "(-force)": "(-强制)",
    "(-rewrite_peptideidentification_rtmz)": "（重写_肽段鉴定_rtmz）",
    "(-in)": "（输入）",
    "(-max_number_of_peptides) <br>": "（最大肽段数）",
    "(-in) <br>Note: In mode 'inspect_out' an Inspect results file is read": "（输入）<br>注意：在“inspect_out”模式下，将读取一个Inspect结果文件",
    "(-inspect_in)": "(-inspect_in)",
    "(-inspect_out)": "(-inspect_out)",
    "(-inspect_directory)": "（-检查目录）",
    "(-temp_data_directory)": "（-临时数据目录）",
    "(-dbs)": "（数据库）",
    "(-trie_dbs)": "(-trie_dbs)",
    "(-new_db)": "（新数据库）",
    "(-instrument)": "（-仪器）",
    "(-precursor_mass_tolerance)": "（前体质量容忍度）",
    "(-peak_mass_tolerance)": "（峰值质量容差）",
    "(-list_modifications)": "-列出修改",
    "(-modifications) e.g.: Deamidation,opt or <br><composition>,<residues>,<type>,<name>, e.g.: H2C2O,KCS,opt,Acetyl or <br><mass>,<residues>,<type>,<name>, e.g.: 42.0367,KCS,opt,Acetyl or <br>Valid values for type are \"fix\" and \"opt\" (default) <br>If you want terminal PTMs, write \"cterm\" or \"nterm\" instead of residues": "（修饰）例如：去酰胺化，可选或<br>&lt;组成&gt;,&lt;残基&gt;,&lt;类型&gt;,&lt;名称&gt;,例如：H2C2O,KCS,可选，乙酰化或<br>&lt;质量&gt;,&lt;残基&gt;,&lt;类型&gt;,&lt;名称&gt;,例如：42.0367,KCS,可选，乙酰化或<br>类型的有效值是“固定”和“可选”（默认）<br>如果你想指定末端PTMs，用“cterm”或“nterm”代替残基",
    "(-use_monoisotopic_mod_mass)": "-使用单同位素修饰质量",
    "(-modifications_xml_file)": "(-修改_xml文件)",
    "(-cleavage)": "（-切割）",
    "(-inspect_input)": "（检查输入）",
    "(-multicharge)": "（多电荷）",
    "(-max_modifications_pp)": "（最大修饰次数每肽段）",
    "(-tag_count)": "（-标签计数）",
    "(-no_tmp_dbs)": "-无临时数据库",
    "(-p_value)": "(-p_value)",
    "(-contact_name)": "-联系人姓名",
    "(-contact_institution)": "-联系机构",
    "(-contact_info)": "（联系信息）",
    "(-blind_only)": "（仅盲模式）",
    "(-p_value_blind)": "(-p_value_blind)",
    "(-snd_db)": "(-snd_db)",
    "(-max_ptm_size)": "（最大后缀匹配大小）",
    "(-include)": "（包含）",
    "(-exclude)": "（排除）",
    "(-rt_model)": "（-rt_model）",
    "(-pt_model)": "（-pt_model）",
    "(-inclusion_charges)": "（包含电荷）",
    "(-inclusion_strategy)": "（包含策略）",
    "(-exclusion_charges)": "（排除电荷）",
    "(-raw_data)": "（原始数据）",
    "(-missed_cleavages) <br>": "（-missed_cleavages）",
    "(-unit)": "（单位）",
    "(-use_relative)": "（使用相对）",
    "(-window_relative) e.g. the window is calculated as [rt - rt*X, rt + rt*X]": "（窗口相对）例如，窗口计算为 [rt - rt*X, rt + rt*X]",
    "(-window_absolute) e.g. the window is calculated as [rt - X, rt + X]": "（窗口绝对值）例如，窗口被计算为 [rt - X, rt + X]",
    "(-mz_tol) Unit of this is defined in 'mz_tol_unit'": "（-mz_tol）此参数的单位在'mz_tol_unit'中定义",
    "(-mz_tol_unit)": "(-mz_tol_unit)",
    "(-rt_tol) Two inclusion/exclusion windows are merged when they (almost) overlap in RT and are close in m/z by this tolerance (see 'mz_tol'). Unit of this param is [seconds]": "(-rt_tol) 当两个包含/排除窗口在保留时间（RT）上（几乎）重叠，并且通过这个容差在m/z上接近时，它们会被合并（参见'mz_tol'）。此参数的单位是[秒]",
    "(-ms2_spectra_per_rt_bin)": "（-ms2_spectra_per_rt_bin）",
    "(-exclude_overlapping_peaks)": "（排除重叠峰）",
    "(-use_dynamic_exclusion)": "（使用动态排除）",
    "(-exclusion_time)": "（排除时间）",
    "(-max_list_size)": "（最大列表大小）",
    "(-min_rt)": "（最小保留时间）",
    "(-max_rt)": "（最大运行时间）",
    "(-rt_step_size)": "（-rt_step_size）",
    "(-rt_window_size)": "（-rt_window_size）",
    "(-min_protein_id_probability)": "（最小蛋白ID概率）",
    "(-min_pt_weight)": "(-最小点权重)",
    "(-min_mz)": "(-最小_mz)",
    "(-max_mz)": "（最大_mz）",
    "(-use_peptide_rule)": "（使用肽段规则）",
    "(-min_peptide_ids)": "（最小肽段标识）",
    "(-min_peptide_probability)": "（最小肽概率）",
    "(-no_intensity_normalization) This is done for each feature separately, so that the feature's maximal intensity in a spectrum is set to 1": "（不进行强度标准化）这是对每个特征单独进行的，以便将特征在谱图中的最大强度设置为1。",
    "(-max_number_precursors_per_feature)": "（最大前体数每特征）",
    "(-signal_to_noise)": "（信噪比）",
    "(-ms_levels) Other scans are copied to the output without changes": "(-ms_levels) 其他扫描被复制到输出中，不做任何改变",
    "(-report_FWHM)": "-报告_FWHM",
    "(-win_len)": "-窗口长度",
    "(-bin_count)": "(-二进制计数)",
    "(-min_required_elements)": "（最小必需元素数）",
    "(-write_log_messages)": "（写入日志消息）",
    "(-spacing_difference_gap) 'min_spacing' is the smaller of the two spacings from the peak apex to its two neighboring points. '0' to disable the constraint. Not applicable to chromatograms": "（间距差异间隔）'min_spacing' 是从峰顶到其两个相邻点的两个间距中较小的一个。'0' 用于禁用该约束。不适用于色谱图",
    "(-spacing_difference) If this difference is exceeded a missing point is assumed (see parameter 'missing'). A higher value implies a less stringent peak definition, since individual signals within the peak are allowed to be further apart. '0' to disable the constraint. Not applicable to chromatograms": "（间距差异）如果这个差异超过设定值，就会假定有一个缺失点（参见参数'missing'）。较高的值意味着峰值定义不那么严格，因为峰内个别信号被允许相距更远。'0'用于禁用该约束。不适用于色谱图。",
    "(-missing) A missing data point occurs if the spacing between two subsequent data points exceeds 'spacing_difference * min_spacing'. 'min_spacing' is the smaller of the two spacings from the peak apex to its two neighboring points. Not applicable to chromatograms": "（-missing）如果两个连续数据点之间的间距超过“spacing_difference * min_spacing”，则会发生缺失数据点。“min_spacing”是从峰顶到其两个相邻点的两个间距中较小的一个。不适用于色谱图。",
    "(-max_intensity) By default, it will be calculated automatically (see auto_mode). Only provide this parameter if you know what you are doing (and change 'auto_mode' to '-1')! All intensities EQUAL/ABOVE 'max_intensity' will be added to the LAST histogram bin. If you choose 'max_intensity' too small, the noise estimate might be too small as well.  If chosen too big, the bins become quite large (which you could counter by increasing 'bin_count', which increases runtime). In general, the Median-S/N estimator is more robust to a manual max_intensity than the MeanIterative-S/N": "（-最大强度）默认情况下，它将自动计算（见auto_mode）。只有在你知道自己在做什么时才提供此参数（并将'auto_mode'更改为'-1'）！所有等于/高于'max_intensity'的强度将被添加到直方图的最后一个bin中。如果你选择的'max_intensity'太小，噪声估计可能也太小。如果选择得太大，bin会变得相当大（你可以通过增加'bin_count'来抵消这种情况，这会增加运行时间）。一般来说，中值信噪比估计器比平均迭代信噪比估计器对手动max_intensity更为稳健。",
    "(-auto_max_stdev_factor)": "(-自动最大标准差因子)",
    "(-auto_max_percentile)": "(-自动最大百分位数)",
    "(-auto_mode)": "(-自动模式)",
    "(-noise_for_empty_window)": "(-空窗口的噪声)",
    "(-fwd_in)": "（-fwd_in）",
    "(-rev_in)": "(-rev_in)",
    "(-number_of_bins)": "（-bins数量）",
    "(-in_mzML)": "(-输入_mzML)",
    "(-length)": "（长度）",
    "(-target_nucleotides) e.g. A=C10H14N5O7P, ..., U=C10H14N5O7P, X=C9H13N2O8PS  where X represents e.g. tU  <br> or e.g. Y=C10H14N5O7PS where Y represents tG": "（目标核苷酸）例如 A=C10H14N5O7P, ..., U=C10H14N5O7P, X=C9H13N2O8PS 其中 X 代表例如 tU 或者例如 Y=C10H14N5O7PS 其中 Y 代表 tG",
    "(-mapping)": "（映射）",
    "(-restrictions)": "（-限制）",
    "(-modifications)": "（修改）",
    "(-peptide_mass_threshold)": "肽段质量阈值",
    "(-precursor_variant_mz_threshold)": "（前体变体_mz阈值）",
    "(-CysteineAdduct)": "（-半胱氨酸加合物）",
    "(-in_OMSSA_ini)": "(-in_OMSSA_ini)",
    "(-in_fasta)": "（输入为FASTA格式）",
    "(-marker_ions_tolerance)": "（-标记离子容忍度）",
    "(-continue)": "（继续）",
    "(-mz_tolerance)": "（-mz_容忍度）",
    "(-rt_tolerance)": "(-rt容忍度)",
    "(-concentrations)": "（浓度）",
    "(-feature_rt)": "（特征_实时）",
    "(-feature_mz)": "(-特征_mz)",
    "(-standard_rt)": "（标准实时）",
    "(-standard_mz)": "（标准_mz）",
    "(-write_gnuplot_output)": "(-写入_gnuplot输出)",
    "(-out_gp)": "输出基因组路径",
    "(-percolator_executable)": "(-percolator_可执行文件)",
    "(-in_target)": "（非目标内）",
    "(-in_decoy)": "（非诱饵）",
    "(-enzyme)": "（-酶）",
    "(-e)": "-e",
    "(-Z) Only available if -X is used": "(-Z) 仅在使用了-X参数时可用",
    "(-p) Set by cross validation if not specified": "（-p）如果未指定，则通过交叉验证设置",
    "(-n) Set by cross validation if not specified": "（-n）如果未指定，则通过交叉验证设置",
    "(-F) Set by cross validation if 0. Default is 0.01": "（-F）如果为0，则通过交叉验证设置。默认值为0.01。",
    "(-t) Default is 0.01": "（-t）默认值为0.01",
    "(-i)": "(-i)",
    "(-x)": "（-x）",
    "(-f) Set to 0.6 by default": "（-f）默认设置为0.6",
    "(-k) e.g. sqt2pin with the -k option": "（-k）例如，使用带有 -k 选项的 sqt2pin",
    "(-v)": "（-v）",
    "(-u)": "(-u)",
    "(-S) Default value is 1": "(-S) 默认值是 1",
    "(-s)": "(-s)",
    "(-a) Set by grid search if not specified": "(-a) 如果没有指定，则通过网格搜索设置",
    "(-b) Set by grid search if not specified": "（-b）如果没有指定，则通过网格搜索设置",
    "(-G) Set by grid search if not specified": "（-G）如果不指定，则通过网格搜索设置",
    "(-g)": "(-g)",
    "(-q)": "(-q)",
    "(-d) Default value is 0": "(-d) 默认值是 0",
    "(-fido-truncation)": "(-fido-截断)",
    "select input database from history in FASTA format": "从历史记录中选择输入数据库为FASTA格式",
    "(default) keep checked in all standard searches": "（默认）在所有标准搜索中保持选中",
    "\"Decoy Prefix\" will name the prefix of each decoy sequence, \"Frequency\" will specify how many decoy sequences are created": "\"诱饵前缀\"将命名每个诱饵序列的前缀，\"频率\"将指定创建多少个诱饵序列",
    "Provides additional controls": "提供额外的控制",
    "to disable set to: -1": "要禁用，请设置为：-1",
    "Set to No if Search Database ia a fasta that already includes decoys.": "如果搜索数据库是一个已经包含诱饵的fasta文件，则设置为“否”。",
    "Required if the Peptide abundances input does not have Protein IDs in the second column": "如果肽段丰度输入在第二列中没有蛋白质ID，则需要",
    "Hyperparameter mu (default: 0.1)": "超参数 mu（默认值：0.1）",
    "Hyperparameter weight of prior probability (default: 0.1)": "先验概率的超参数权重（默认值：0.1）",
    "Minimum number of samples peptides needed to be quantified in": "最小数量的样本肽需要被量化在",
    "Impute missing values if missing fraction is larger than the threshold.": "如果缺失比例大于阈值，则填充缺失值。",
    "Peptides weighted lower than the cutoff will be excluded.": "低于截止值的肽段将被排除。",
    "Please specify the .mzML file you want to convert": "请指定您想要转换的 .mzML 文件",
    "You can optionally specify a minimum scan time. MS scans measured earlier will not be copied.": "您可以选择指定最小扫描时间。测量时间早于该时间的质谱扫描将不会被复制。",
    "You can optionally specify a maximum scan time. MS scans measured at a later scan time will not be copied.": "您可以选择性地指定最大扫描时间。在稍后的扫描时间测量的MS扫描将不会被复制。",
    "Please specify your protein sequences in the .fasta format. If you want to search for contaminants you have to include the sequences in your file. You do not have to supply decoy sequences, those are generated for you.": "请将您的蛋白质序列以.fasta格式指定。如果您想要搜索污染物，您需要将序列包含在您的文件中。您不需要提供诱饵序列，这些将为您自动生成。",
    "Fasta headers do often contain additional information in addition to your protein identifier, by specifying a regex pattern the tool can extract the protein IDs. If you fasta headers are already cleaned you can leave this field empty.": "Fasta 头信息通常除了蛋白质标识符外还包含其他信息，通过指定一个正则表达式模式，工具可以提取蛋白质 ID。如果你的 fasta 头信息已经被清理，你可以将这个字段留空。",
    "Select a protease the tool can use when performing in silico digestion.": "选择一个蛋白酶，该工具可以在进行计算机模拟消化时使用。",
    "Select the minimum amount of miss cleavages in a peptide sequence.": "选择肽序列中缺失切割的最小数量。",
    "Select the maximum amount of miss cleavages in a peptide sequence.": "选择一个肽序列中允许的最大错切次数。",
    "Select the maximum mass of peptides included in the data base.": "选择数据库中包含的多肽的最大质量。",
    "Select the minimum amino acid  length of peptides included in the data base.": "选择数据库中包含的肽段的最小氨基酸长度。",
    "Select isotopic amino acid modifications. These modifications change the natural abundance of isotopes of single elements. Each peptide sequence is duplicated and included in the data base at natural isotopic abundances and using the modified isotopic abundances.": "选择同位素氨基酸修饰。这些修饰改变了单一元素同位素的自然丰度。每个肽段序列都被复制，并以自然同位素丰度和使用修改后的同位素丰度包含在数据库中。",
    "Specify how the peptide masses should be calculated.": "指定如何计算肽段质量。",
    "Select fixed amino acid modifications. These Modifications are applied whenever possible.": "选择固定的氨基酸修饰。这些修饰在可能的情况下应用。",
    "Select variable amino acid modifications. Whenever a this modification can be applied the peptide sequence is duplicated and included in the data base in a modified and unmodiefied version.": "选择可变的氨基酸修饰。每当可以应用这种修饰时，肽序列会被复制并在数据库中以修饰和未修饰的版本包含。",
    "Select the maximum amount of variable amino acid modifications in one sequence. This parameter is needed to circumvent a combinatoric explosion.": "选择一个序列中可变氨基酸修饰的最大数量。这个参数需要用来避免组合爆炸。",
    "Specify mass spectrometry data you want to analyze.": "指定您想要分析的质谱数据。",
    "Specify the peptide data base.": "指定肽段数据库。",
    "Specify the minimum peptide ion charge state to consider.": "指定要考虑的最小肽离子电荷状态。",
    "Specify the maximum peptide ion charge state to consider.": "指定要考虑的最大肽离子电荷状态。",
    "Specify the width around the picked precursor ion used in charge state determination.": "指定用于电荷状态确定的选定前体离子周围的宽度。",
    "Specify minimum relative intensity within an putative isotopic envelope to be included when performing charge state determination.": "指定在执行电荷状态确定时，包含在假定的同位素包络内所需的最小相对强度。",
    "Specify minimum relative intensity to the prior peak when iterating through peak lists while performing charge state determination.": "在进行电荷状态确定时，遍历峰值列表时，指定相对于前一个峰值的最小相对强度。",
    "When performing charge state determination we compared measured envelopes to the distribution of randomly generated spectra. This parameter tunes the size of the generated data set.": "在执行电荷状态确定时，我们将测量的包络与随机生成的谱图的分布进行比较。此参数调整生成数据集的大小。",
    "Specify the width of the search space when retrieving in silico generated peptide ions from the peptide data base. The width is calculated relative to the precursor mz in ppm.": "指定从肽段数据库检索计算生成的肽段离子时搜索空间的宽度。该宽度是相对于前体m/z值，以ppm（百万分之一）计算的。",
    "Specify the types of N-terminal charged ion fragments included in the generation of in silico fragment spectra.": "指定在生成in silico片段谱时包括的N端带电离子片段类型。",
    "Specify the types of C-terminal charged ion fragments included in the generation of in silico fragment spectra.": "指定在生成in silico片段谱时包含的C末端带电离子片段的类型。",
    "Specify the minimum N of the top N most abundand peaks within a 100 Da window used to compute the Andromeda score.": "指定在100 Da窗口内用于计算Andromeda得分的前N个最丰富的峰值的最小N值。",
    "Specify the maximum N of the top N most abundand peaks within a 100 Da window used to compute the Andromeda score.": "指定在100 Da窗口内用于计算Andromeda分数的前N个最丰富的峰值的最大N值。",
    "Specify the minimum m/z difference between a measured and an in silico generated peak during peak matching. The width is calculated relative to the m/z of the measured peak in ppm.": "指定在峰匹配过程中测量值和计算生成的峰之间的最小m/z差异。宽度是相对于测量峰的m/z以ppm计算的。",
    "Specify list of peptide spectrum matches to be scored.": "指定要评分的肽谱匹配列表。",
    "Specify how PSM thresholds should be estimated using semi supervised machine learning techniques (recommended) or if fixed score values should be used instead.": "指定如何使用半监督机器学习技术（推荐）估计PSM阈值，或者是否应该使用固定的分数值。",
    "Specify peptide identifications.": "指定肽段鉴定。",
    "Specify list of peptide identifications.": "指定肽段鉴定的列表。",
    "If checked, protein inference is carried out using peptide information from all files simultaneously.": "如果选中，将使用所有文件中的肽段信息同时进行蛋白质推断。",
    "Specify how protein groups are created. For details please refer to the description below.": "指定如何创建蛋白质组。有关详细信息，请参阅下面的描述。",
    "Specify how peptides are used to infer protein groups. For details please refer to the description below.": "指定如何使用肽段来推断蛋白质组。有关详细信息，请参阅下面的描述。",
    "Specify if and how q-value calculation should be carried out.": "指定是否以及如何进行q值计算。",
    "Checks that sequence only contains the 20 essential amino acids (and optional non-standard AAs), and checks that is not DNA or RNA": "检查序列是否仅包含20种必需氨基酸（以及可选的非标准氨基酸），并检查它不是DNA或RNA。",
    "Anything that is not an upper case letter [A-Z] will be ignored.": "任何不是大写字母[A-Z]的字符都将被忽略。",
    "If checked, a quantification of peptides which contain an isotopic modification is performed. Additionally, for every identified peptide we check if the mass spectrometry run contains a peak at the m/z of the labeled/unlabeled counterpart and quantify it.": "如果选中，将对含有同位素修饰的肽段进行定量。此外，对于每个已鉴定的肽段，我们都会检查质谱运行中是否存在标记/未标记对应物的m/z处的峰，并对其进行定量。",
    "Specify the length of the scan time window used for XIC creation.": "指定用于创建提取离子色谱图（XIC）的扫描时间窗口的长度。",
    "Specify the MzWindow Da Type": "指定 MzWindow Da 类型",
    "Specify the m/z window set centered on the target m/z used for XIC creation": "指定用于创建XIC的以目标m/z为中心的m/z窗口集",
    "Specify Quantified Peptides.": "指定定量肽段。",
    "Specify Inferred Proteins.": "指定推断蛋白质。",
    "Specify list of .quantAndProt files": "指定.quantAndProt文件列表",
    "Note that the MST options will perform a local, MST guided alignment while the other options will use a reference-guided alignment. The global option will also move peaks which are below the selected FDR threshold. (--method)": "请注意，MST选项将执行局部的、由MST引导的比对，而其他选项将使用参考引导的比对。全局选项还将移动低于所选FDR阈值的峰值。（--method）",
    "Maximal difference in RT for two aligned features": "两个对齐特征的最大RT差异",
    "Maximal difference in RT (in seconds) for two isotopic channels in the same run": "在同一运行中两个同位素通道的最大RT（保留时间）差异（以秒为单位）",
    "Do not write peakgroup if selected in less than this fraction of runs": "如果所选的peakgroup在少于这个比例的运行中被选中，则不写入peakgroup",
    "Refers to the desired FDR on assay level. (--target_fdr)": "指的是在检测水平上期望的FDR值。(--target_fdr)",
    "A fixed m-score cutoff which does not take into account the number of runs (use target_fdr instead). (--fdr_cutoff)": "一个固定的m分数截断值，不考虑运行次数（使用target_fdr代替）。(--fdr_cutoff)",
    "During the extension phase of the algorithm, peakgroups of this quality will still be considered for alignment (in FDR) - it is possible to give a range in the format lower,higher+stepsize,stepsize - e.g. 0,0.31,0.01  (-1 will set it to fdr_cutoff) (--max_fdr_quality)": "在算法的扩展阶段，这个质量的峰值组仍将被考虑进行比对（在FDR中）-可以给出一个范围，格式为下限，上限+步长，步长-例如0,0.31,0.01（-1将将其设置为fdr_cutoff）(--max_fdr_quality)",
    "Disable grouping of isotopic variants by peptide_group_label, thus disabling matching of isotopic variants of the same peptide across channels. If turned off, each isotopic channel will be matched independently of the other. If enabled, the more certain identification will be used to infer the location of the peak in the other channel. (--disable_isotopic_grouping)": "禁用通过peptide_group_label对同位素变体进行分组，从而禁用跨通道的相同肽段的同位素变体的匹配。如果关闭，每个同位素通道将独立匹配。如果启用，将使用更确定的鉴定来推断另一个通道中峰的位置。(--disable_isotopic_grouping)",
    "Quality cutoff to still consider a feature for alignment using the d_score: everything below this d-score is discarded (this is mainly for speedup). (--dscore_cutoff)": "用于对齐的特征的质量截止值，使用d_score：低于此d分数的所有内容都将被丢弃（这主要是为了加速）。（--dscore_cutoff）",
    "Number of experiments in which the peptide needs to be identified with high confidence (e.g. above fdr_curoff). (--nr_high_conf_exp)": "在肽段需要高置信度鉴定的实验数量（例如，高于fdr_curoff）。(--nr_high_conf_exp)",
    "Minimal score needed for a feature to be considered for alignment between runs (e.g. score needed to be considered an 'anchor point' for pairwise alignment). (--alignment_score)": "为了使一个特征被考虑用于运行之间的比对（例如，需要的分数被认为是成对比对的“锚点”），所需的最小分数。（--alignment_score）",
    "Pixels from different groups will be segmented separately. For the validity of downstream statistical analysis, it is important that each distinct observational unit (e.g., tissue sample) is assigned to a unique group": "不同组的像素将分别进行分割。为了确保下游统计分析的有效性，重要的是将每个独特的观察单位（例如，组织样本）分配给一个独特的组",
    "The spatial neighborhood radius of nearby pixels to consider. Only a single value is allowed": "考虑附近像素的空间邻域半径。只允许一个单一值",
    "The maximum number of segments (clusters). The final number of segments may differ. Only a single value is allowed.": "最大段数（聚类）。最终的段数可能会有所不同。只允许一个单一值。",
    "The method to use to calculate the spatial smoothing weights. The 'gaussian' method refers to spatially-aware (SA) weights, and 'adaptive' refers to spatially-aware structurally-adaptive (SASA) weights": "用于计算空间平滑权重的方法。'gaussian'方法指的是空间感知（SA）权重，而'adaptive'指的是空间感知结构自适应（SASA）权重。",
    "The type of distance metric to use when calculating neighboring pixels based on r. The options are ‘radial’, ‘manhattan’, ‘minkowski’, and ‘chebyshev’ (the default).": "当根据 r 计算邻近像素时使用的距离度量类型。选项有‘径向’、‘曼哈顿’、‘闵可夫斯基’和‘切比雪夫’（默认）。",
    "Should simulated annealing be used during the optimization process to improve parameter estimates?": "在优化过程中是否应该使用模拟退火来改进参数估计？",
    "Should the parameter estimates be initialized using k-means (’kmeans’) or Gaussian mixture model (’gmm’)?": "参数估计应该使用 k-means ('kmeans') 初始化，还是使用高斯混合模型 ('gmm') 初始化？",
    "A regularization parameter applied to estimated posterior class probabilities to avoid singularities. Must be positive for successful gradient descent optimization.Changing this value (within reason) should have only minimal impact on values of parameter estimates, but may greatly affect the algorithm’s speed and stability.": "一个正则化参数应用于估计的后验类概率以避免奇异性。必须为正数才能成功进行梯度下降优化。改变这个值（在合理范围内）应该只对参数估计值有最小的影响，但可能会极大地影响算法的速度和稳定性。",
    "The maximum number of EM-algorithm iterations.": "最大EM算法迭代次数。",
    "The tolerance convergence criterion for the EM-algorithm. Corresponds to the change in log-likelihood.": "EM算法的容忍收敛标准。对应于对数似然的变化。",
    "Region of interest (selected spectra) or m/z feature.": "感兴趣区域（选定的光谱）或 m/z 特征。",
    "The m/z closest to this m/z value will be used as a reference.": "最接近这个 m/z 值的 m/z 将被用作参考。",
    "Name has to match exactly on of the annotation names in the annotation file": "名称必须完全匹配注释文件中注释名称之一",
    "The number of top-ranked colocalized features to return in the tabular file.": "在表格文件中返回的排名靠前的共定位特征的数量。",
    "The colocalization measure used to rank colocalized features. Possible options include Pearson’s correlation and Manders’ colocalization coefficients.": "用于排名共定位特征的共定位度量。可能的选项包括皮尔逊相关系数和曼德斯共定位系数。",
    "Submit either a single file, or a collection of files.": "提交一个单独的文件，或者一组文件。",
    "If more than one enzyme was used, please provide the missed cleavages for every enzyme in the same order, with a comma separated list, e.g. \"2, 1\".": "如果使用了多个酶，请按照相同的顺序为每种酶提供缺失的切割次数，用逗号分隔的列表表示，例如“2, 1”。",
    "Using percentiles, 0.75 means that the 25% most intense peaks will be annotated.": "使用百分位数，0.75意味着将注释最强烈的25%的峰。",
    "UniProt databases only": "UniProt数据库仅限",
    "Files in which to find sequence similarity search results in tabular format (SSST). Use e.g. Blast or Diamond to produce them. Required columns are: 'qacc sacc stitle' (Blast) or 'qseqid sseqid stitle' (Diamond).": "在其中查找序列相似性搜索结果的表格格式（SSST）的文件。使用例如Blast或Diamond来生成它们。必需的列是：'qacc sacc stitle'（Blast）或'qseqid sseqid stitle'（Diamond）。",
    "File in which to find sequence similarity search results in tabular format (SSST). Use e.g. Blast or Diamond to produce them. Required columns are: 'qacc sacc stitle' (Blast) or 'qseqid sseqid stitle' (Diamond).": "在SSST（序列相似性搜索结果的表格格式）中查找序列相似性搜索结果的文件。可以使用例如Blast或Diamond来生成它们。所需的列是：'qacc sacc stitle'（Blast）或'qseqid sseqid stitle'（Diamond）。",
    "Field-Separator of the (-s) sequence similarity table. The default value is the 'TAB' character. Set to 'default' to use the hard coded default": "（-s）序列相似性表的字段分隔符。默认值是'TAB'字符。设置为'default'以使用硬编码的默认值",
    "Header of the (-s) sequence similarity table. Separated by space (' ') the names of the in order of appearance in the respective table. Required and default columns are 'qacc sacc stitle'. Set to 'default' to use the hard coded default": "(-s) 序列相似性表的标题。按在相应表中出现的顺序用空格（' '）分隔的名称。必需和默认列是 'qacc sacc stitle'。设置为 'default' 以使用硬编码的默认值。",
    "A file with regular expressions, one per line. Any match to any of these regular expressions causes sequence similarity search result descriptions ('stitle' in Blast terminology) to be discarded from the prot-scriber annotation process. Set to 'default' to use the hard coded default": "一个包含正则表达式，每行一个正则表达式的文件。任何与这些正则表达式匹配的内容都会导致序列相似性搜索结果描述（在Blast术语中称为'stitle'）从prot-scriber注释过程中被丢弃。设置为'default'以使用硬编码的默认值。",
    "A file with pairs of lines. Within each pair the first line is a regular expressions defining one or more capture groups. The second line of a pair is the string used to replace the match in the regular expression with. Set to 'default' to use the hard coded default": "一个包含成对行的文件。在每一对中，第一行是一个定义了一个或多个捕获组的正则表达式。第二行是用于在正则表达式中替换匹配项的字符串。如果设置为'default'，则使用硬编码的默认值。",
    "A file with regular expressions, one per line. Any match to any of these regular expressions causes the matched sub-string to be deleted, i.e. filtered out. Set to 'default' to use the hard coded default": "一个包含正则表达式，每行一个的文件。这些正则表达式中的任何一个匹配都会导致匹配的子字符串被删除，即被过滤掉。设置为'default'以使用硬编码的默认值。",
    "A file in which regular expressions (regexs) are stored, one per line. These regexs are used to recognize non-informative words, which will only receive a minimum score in the prot-scriber process that generates human readable description.": "一个存储正则表达式（regexs）的文件，每行一个。这些正则表达式用于识别非信息性词汇，在prot-scriber过程中，这些词汇只会获得最低分数，该过程用于生成人类可读的描述。",
    "A regular expression to be used to split descriptions (`stitle` in Blast terminology) into words. Default is '([~_\\-/|\\;,':.\\s]+)'.": "一个用于将描述（在Blast术语中为`stitle`）分割成单词的正则表达式。默认为'([~_\\-/|\\;,':.\\s]+)'。",
    "The quantile (percentile) to be subtracted from calculated inverse word information content to center these values. Value between 0 and 1.": "从计算出的逆词信息内容中减去的分位数（百分位数），以使这些值居中。值介于0和1之间。",
    "A file with pairs of lines. Defines pairs of regex / replace pairs for post polishing of annotation results. Set to 'none' or provide an empty file to supress polishing.": "一个包含成对行的文件。定义了用于注释结果后处理的正则表达式/替换对。设置为'none'或提供一个空文件以抑制抛光。",
    "A file in which families of biological sequences are stored, one family per line. Each line must have format 'fam_name TAB gene1,gene2,gene3'. Make sure no gene appears in more than one family.": "一个存储生物序列家族的文件，每个家族占一行。每行必须有格式 '家族名 TAB gene1,gene2,gene3'。确保没有基因出现在多个家族中。",
    "Set this to true to also annotate sequences are not member of a sequence family.": "将此设置为true，以对不属于序列家族的序列进行注释。",
    "A regular expression used to split the list of gene_identifiers in the argument --seq-families (-f) gene families file. Default is '(\\s*,\\s*|\\s+)'.": "一个用于在参数 --seq-families (-f) 基因家族文件中分割 gene_identifiers 列表的正则表达式。默认是 '(\\s*,\\s*|\\s+)'。",
    "A string used as separator in the argument --seq-families (-f) gene families file. This string separates the gene_family_identifier (name) from the gene_identifier list that family comprises. Default is 'TAB'.": "在参数 --seq-families (-f) 基因家族文件中使用的分隔符字符串。该字符串用于分隔基因家族标识符（名称）和该家族包含的基因标识符列表。默认值为 'TAB'。",
    "Use this option to exclude results from the output table that could not be annotated.": "使用此选项可排除无法注释的结果出现在输出表中。",
    "Sequence of the restrict": "限制序列",
    "Tabular 'Phospho (STY)Sites.txt' produced by MaxQuant": "由MaxQuant生成的表格“Phospho (STY)Sites.txt”",
    "PERL-compatible regular expression matching header of column having number of 'Phospho (STY)'": "使用Perl兼容的正则表达式匹配包含“Phospho (STY)”数量的列的标题",
    "PERL-compatible regular expression matching column header having first sample intensity": "Perl兼容的正则表达式匹配具有第一个样本强度的列标题",
    "E.g., 1 if subsequent column is next sample; 2 if next sample is two columns away, etc.": "例如，1表示相邻列是下一个样本；2表示下一个样本在两列之外，依此类推。",
    "Were samples enriched for pS and pT, or were they enriched for pY instead?": "样品是否富集了磷酸化的丝氨酸（pS）和苏氨酸（pT），还是富集了磷酸化的酪氨酸（pY）？",
    "When a peptide is multiply phosphorylated, how should intensities be merged? [default: sum]": "当一个肽段被多重磷酸化时，强度应该如何合并？[默认：求和]",
    "See help below for an explanation.": "请参阅下面的帮助以获得解释。",
    "Sequence database; supply the same FASTA file as you supplied to by MaxQuant": "序列数据库；提供与您提供给MaxQuant的相同FASTA文件",
    "NetworKIN file; see help section below": "NetworKIN 文件；请参阅下面的帮助部分",
    "pS/pT/pY phosphorylation site motifs; see help section below": "pS/pT/pY 磷酸化位点模式；请参阅下面的帮助部分",
    "'Kinase-substrate dataset'; see help section below": "激酶-底物数据集；请参阅下面的帮助部分",
    "'Regulatory sites'; see help section below": "“调控位点”；请参阅下面的帮助部分",
    "(field may be empty) [default: human].  If you supply this parameter, use the species indentifier seen as a suffix in UniProtKB": "（字段可能为空）[默认：人类]。如果您提供此参数，请使用在UniProtKB中作为后缀看到的物种标识符。",
    "'preproc_tab' dataset produced by 'MaxQuant Phosphopeptide Preprocessing' tool": "由“MaxQuant Phosphopeptide Preprocessing”工具生成的“preproc_tab”数据集",
    "ANOVA alpha cutoff values for significance testing: tabular data having one column and no header": "方差分析显著性检验的显著性水平截断值：表格数据具有一列且无标题",
    "'preproc_sqlite' dataset produced by 'MaxQuant Phosphopeptide Preprocessing' tool": "由“MaxQuant 磷酸化肽段预处理”工具生成的“preproc_sqlite”数据集",
    "Pattern matching columns that have peptide intensity data (PERL-compatible regular expression matching column label)": "匹配包含肽段强度数据的列（使用符合PERL兼容正则表达式的列标签进行匹配）",
    "Impute missing values by (1) using median for each sample-group; (2) using median across all samples; (3) using mean across all samples; or (4) using randomly generated values having same SD as across all samples (with mean specified by 'Mean percentile for random values')": "通过以下方法填充缺失值：(1) 对每个样本组使用中位数；(2) 跨所有样本使用中位数；(3) 跨所有样本使用均值；或 (4) 使用具有与所有样本相同标准差的随机生成值（由 '随机值的均值百分位数' 指定均值）。",
    "Percentile center of random values; range [1,99]": "随机值的百分位数中心；范围[1,99]",
    "Standard deviation adjustment-factor for random values; real number.  (1.0 means SD of random values equal to the SD for the entire data set.)": "随机值的标准差调整因子；实数。（1.0 表示随机值的标准差等于整个数据集的标准差。）",
    "Pattern extracting sample-names from names of columns of 'Filtered phosphopeptide intensities' that have peptide intensity data (PERL-compatible regular expression)": "从“筛选后的磷酸肽强度”列名中提取样本名称的模式（兼容PERL的正则表达式）",
    "Pattern extracting sample-group from the extracted sample-names (PERL-compatible regular expression)": "从提取的样本名中提取样本组（Perl兼容的正则表达式）",
    "Only consider as comparable those intensities having at least this number of values in each sample-group (range [0,∞])": "只考虑在每个样本组中至少有此数量值的强度作为可比的（范围[0,∞]）",
    "What filter should be applied to sample-group names? (1) 'none', no filter; (2) 'include', match is required; (3) 'exclude', match is forbidden.": "应该对样本组名称应用什么过滤器？(1) 'none'，不应用过滤器；(2) 'include'，需要匹配；(3) 'exclude'，禁止匹配。",
    "Minimum number of substrates to consider any kinase for KSEA (range [1,∞])": "最小底物数量以考虑任何激酶进行KSEA（范围[1,∞]）",
    "Maximum FDR to be used to score a kinase enrichment as significant; see warning against setting this too low in help text below.": "最大FDR（假发现率）用于评分激酶富集为显著；请参见下面的帮助文本中的警告，不要将此设置得太低。",
    "Should log2(fold-change) be used for KSEA? (Checking this may alter (possibly reduce) the number of hits.)": "KSEA中是否应该使用log2(fold-change)？（检查此选项可能会改变（可能减少）命中数量。）",
    "Minimum 'quality' of substrates to be considered for KSEA (range [0,∞]); higher numbers reduce the number of substrates considered - see help text below.": "用于KSEA（k-mer显著性富集分析）考虑的底物的最小'质量'值（范围[0,∞]）；较高的数值会减少考虑的底物数量 - 详见下面的帮助文本。",
    "Collection containing GeoJSON files": "包含GeoJSON文件的集合",
    "Psm file": "Psm 文件",
    "The maximum mass difference between theoretical mass and experimental mass of a peptide": "肽段的理论质量与实验质量之间的最大质量差异",
    "For metagenomic data, the delimiter that separates the bin ID from the protein ID (default: \"_\"). Use \"-\" to ignore bins ID entirely.": "对于宏基因组数据，分隔bin ID和蛋白质ID的分隔符（默认为：\"_\"）。使用\"-\"则完全忽略bin ID。",
    "To compute clumpiness of carbon assimilation. Only use when samples are labeled tosaturation. Estimation of clumpiness takes much additional time.": "计算碳同化率的团簇性。仅在样品被标记为饱和时使用。团簇性的估计需要额外的时间。",
    "If not given the built in matrix will be used": "如果没有给出，将使用内置的矩阵",
    "A table containing ms_run and their nominal value (1, 5, or 10)": "一个包含ms_run及其名义值（1、5或10）的表格",
    "Mass spectral library file to add key.": "质谱谱库文件添加键。",
    "Name of the attribute which will be assigned to all spectra records in the MSP.": "在MSP中将被分配给所有谱图记录的属性名称。",
    "Value of the attribute which will be assigned to all spectra records in the MSP.": "在MSP中将被分配给所有谱图记录的属性值。",
    "Default value is -0.3.": "默认值是-0.3。",
    "Please enter at max 1 method": "请输入最多1个方法",
    "Please enter at max 1 format": "请输入最多1种格式",
    "Default value is 7.": "默认值是7。",
    "Default value is 12.": "默认值是12。",
    "Type of matching to perform.": "要执行的匹配类型。",
    "Metadata field to use for matching.": "用于匹配的元数据字段。",
    "Tolerance for matching the selected metadata field - only pairs within the tolerance will be contained in the output.": "匹配选定元数据字段的容差 - 只有容差内的配对才会包含在输出中。",
    "Metadata field to use for matching": "用于匹配的元数据字段",
    "Default value is 1.": "默认值是1。",
    "Default value is 3.": "默认值是3。",
    "Default value is false.": "默认值是false。",
    "Default value is 70.": "默认值是70。",
    "Default value is 5.5": "默认值是5.5",
    "Mass spectral library file.": "质谱谱库文件。",
    "Include/Exclude the spectra matching the identifiers in the file. The default is include.": "包含/排除文件中标识符匹配的谱图。默认为包含。",
    "file containing list of identifiers in tsv or csv.": "包含标识符列表的文件，格式为制表符分隔值（tsv）或逗号分隔值（csv）。",
    "Apply harmonization process on the metadata keys (e.g. parse numeric values, derive precursor mass).": "对元数据键应用协调处理过程（例如解析数值，推导前体质量）。",
    "Generate formula candidates that just contain the elements selected by the users. Check 'IsTmsMeoxDerivative' if you want to annotate EI-MS spectra.": "生成只包含用户选定元素的公式候选项。如果您想要注释EI-MS谱图，请勾选'IsTmsMeoxDerivative'。",
    "Mass spectra file to be filtered.": "待过滤的质谱文件。",
    "Normalize intensities of peaks (and losses) to unit height.": "将峰（和损失）的强度标准化为单位高度。",
    "Collection of filters that are considered default and that do no require any (factory) arguments.": "一组被认为默认的过滤器，这些过滤器不需要任何（工厂）参数。",
    "Apply all adding and cleaning filters if possible, so that the spectra have canonical metadata.": "如果可能的话，应用所有添加和清理过滤器，以便光谱具有规范的元数据。",
    "Keep only peaks within set relative intensity range (keep if maximum >= intensity >= minimum).": "只保留在设定的相对强度范围内的峰（如果最大值 >= 强度 >= 最小值，则保留）。",
    "Lower bound for intensity filter.": "强度过滤器的下限。",
    "Upper bound for intensity filter.": "强度过滤器的上限。",
    "Keep only peaks between set m/z range (keep if maximum >= m/z >= minimum).": "只保留设定的m/z范围内的峰（如果最大值 >= m/z >= 最小值）。",
    "Lower bound for m/z filter.": "m/z 过滤器的下限。",
    "Upper bound for m/z filter.": "m/z 过滤器的上限。",
    "Remove spectra that does not contain SMILES.": "删除不包含SMILES的谱图。",
    "Remove spectra that does not contain INCHI.": "删除不包含INCHI的谱图。",
    "Derives the precursor_mz from the parent mass and adduct or charge.": "根据母体质量以及加成离子或电荷计算前体_mz。",
    "Lowest intensity peaks will be removed when it has more peaks than desired.": "当峰值数量超过期望值时，最低强度的峰将被移除。",
    "Maximum number of peaks. Remove peaks if more peaks are found.": "最大峰值数。如果发现更多峰值，则移除峰值。",
    "matchms Scores json file.": "matchms 分数 json 文件。",
    "csv file containing the metadata.": "包含元数据的CSV文件。",
    "Name of the user specified column to merge the data on.": "用户指定的用于合并数据的列名。",
    "Matrix of numeric data with headers.": "带有标题的数值数据矩阵。",
    "Tabular file with the data metadata : one sample per line and at least two columns : ids and one condition": "带有数据元数据的表格文件：每行一个样本，至少两列：id和至少一个条件",
    "Tabular file with information about your tested variables. Only used to aggregate generated information.": "包含有关您测试变量信息的表格文件。仅用于汇总生成的信息。",
    "Perform the anova tests on column/row; for W4M 3-tables format, use 'row'.": "对列/行执行方差分析测试；对于W4M 3表格式，使用'行'。",
    "The column name of the condition. ex: hour or treatment": "条件的列名。例如：小时或治疗",
    "Used if more than 1 conditon. The anova will produse p-value according to the interaction between your condition (ex: condition1:conditions2, condition1:conditions3, condition2:conditions3 and condition1:condition2:conditions3)": "如果存在多个条件，可以使用此选项。方差分析（ANOVA）将根据条件之间的交互作用（例如：condition1:conditions2, condition1:conditions3, condition2:conditions3 和 condition1:condition2:conditions3）产生p值。",
    "Method used to apply a correction on the pvalue because of the number of test": "用于根据测试数量对p值进行校正的方法",
    "max adjusted p.value accepted": "最大接受的校正P值",
    "Intersection: all condition p-value must be under the threshold. Union: at least condition p-value must be under the threshold.": "交集：所有条件的p值必须低于阈值。并集：至少有一个条件的p值必须低于阈值。",
    "variable x sample, decimal: '.', missing: NA, mode: numerical, sep: tabular": "变量 x 样本，小数点：'.'，缺失值：NA，模式：数值，分隔符：制表符",
    "sample x metadata, decimal: '.', missing: NA, mode: character and numerical, sep: tabular": "样本 x 元数据，小数点：'.'，缺失值：NA，模式：字符和数值，分隔符：表格",
    "variable x metadata, decimal: '.', missing: NA, mode: character and numerical, sep: tabular": "变量 x 元数据，小数点：'.'，缺失值：NA，模式：字符和数值，分隔符：制表符",
    "Column of sampleMetadata containing 2 types of strings (e.g., 'case' and 'control')": "样本元数据列包含2种类型的字符串（例如，'case' 和 'control'）",
    "Must be between 0 and 1": "必须在0和1之间",
    "Select an integer (e.g., 123) if you want to obtain exactly the same signatures when re-running the algorithm; 0 means that no seed is selected": "选择一个整数（例如，123），如果你希望在重新运行算法时获得完全相同的签名；0表示没有选择种子",
    "XML batch file (ideally created with version @TOOL_VERSION@)": "XML批量文件（最好使用版本@TOOL_VERSION@创建）",
    "For 'Custom database search'": "对于“自定义数据库搜索”",
    "Only provide a filelist if you like to exclude Peaklists, update the metadata (e.g. classLabel), or if you have not provided a filelist for  Process Scans or Replicate Filter.": "如果您想要排除峰值列表，更新元数据（例如 classLabel），或者如果您还没有为“处理扫描”或“复制过滤器”提供文件列表，则只提供文件列表。",
    "Maximum tolerated m/z deviation across samples in parts per million (ppm).": "最大容忍的毫焦耳偏差在样品间的百万分率（ppm）。",
    "The size of each block of peaks to perform clustering on.": "每个要执行聚类的峰值块的大小。",
    "Peaklists generated by Process Scans (SIM-Stitch).": "由处理扫描（SIM-Stitch）生成的峰值列表。",
    "Only provide a filelist if you like to exclude Peaklists, update the metadata (e.g. classLabel), or if you have not provided a filelist for Process Scans.": "如果您想要排除峰值列表、更新元数据（例如 classLabel）或者您还没有为处理扫描提供文件列表，则只需提供一个文件列表。",
    "Maximum tolerated m/z deviation across technical replicates in parts per million (ppm).": "在百万分之一（ppm）的单位中，技术重复之间的最大容忍m/z偏差。",
    "Maximum tolerated relative standard deviation (RSD) of the peak intensities across technical replicates. Leave empty to skip this filter step.": "最大可容忍的峰强度在技术重复中的相对标准偏差（RSD）。留空以跳过此过滤步骤。",
    "*.raw files are read using the RawFileReader reading tool (Copyright © 2016 by Thermo Fisher Scientific, Inc. All rights reserved). To run this tool and process .raw files you must agree to the RawFileReader license terms. Read it at https://github.com/computational-metabolomics/dimspy-galaxy/blob/master/tools/dimspy/RawFileReaderLicense.md. See generic help section of this tool for more details.": "*.raw 文件使用 RawFileReader 读取工具（版权所有 © 2016 Thermo Fisher Scientific, Inc. 保留所有权利）进行读取。要运行此工具并处理 .raw 文件，您必须同意 RawFileReader 许可条款。在 https://github.com/computational-metabolomics/dimspy-galaxy/blob/master/tools/dimspy/RawFileReaderLicense.md 上阅读它。有关更多详细信息，请参见此工具的通用帮助部分。",
    "Maximum tolerated m/z deviation in consecutive scans in parts per million.": "在连续扫描中最大可容忍的 m/z 偏差，以百万分之一为单位。",
    "Select '0' to skip this step.": "选择 '0' 跳过这一步。",
    "Select '0' to skip this step. Maximum tolerated relative standard deviation (RSD) of the peak intensities across scans.": "选择'0'跳过这一步。峰强度在扫描过程中的最大容忍相对标准偏差（RSD）。",
    "When set to 'yes' it will skip the processing step where (SIM) windows are 'stitched' or 'joined' together. Set this option to 'yes' if you like to proces individual scan/SIM windows (events/ranges) without 'stitching' them.": "当设置为'yes'时，它将跳过（SIM）窗口被“拼接”或“连接”在一起的处理步骤。如果您喜欢处理单个扫描/SIM窗口（事件/范围）而不进行“拼接”，请将此选项设置为'yes'。",
    "Select '0' to skip this filter.": "选择'0'跳过此过滤器。",
    "Limit of candidate hits returned per compound": "每个化合物返回的候选命中数限制",
    "Fetch synonyms for each candidate from PubChem": "从PubChem获取每个候选词的同义词",
    "Fetch compound classes for each candidate using ClassyFire": "使用ClassyFire为每个候选化合物获取化合物类别。",
    "Minimum fraction (i.e. percentage) of samples a peak has to be present": "峰值必须存在的样本的最小比例（即百分比）",
    "Leave empty to skip this step": "留空以跳过此步骤",
    "Peaks that have a larger RSD than the QC peaks are removed (Leave empty to skip this step).": "具有比QC峰更大的RSD的峰被移除（留空以跳过此步骤）。",
    "The percentage of samples (with a intensity value) that must beat the blank samples (peak by peak basis)": "样品（具有强度值）的百分比必须高于空白样品（按峰值计算）",
    "Select the function that should be used to calculate the peak intensity threshold (blanks only)": "选择用于计算峰值强度阈值的函数（仅空白）",
    "Schema used for the MSP file (auto will try to determine the schema automatically)": "用于MSP文件的模式（自动将尝试自动确定模式）",
    "The SIRIUS-CSI:Fingerid output can have additional metadata columns added; these can be either extracted from all MSP parameters or from the 'Name' and 'RECORD_TITLE' MSP parameters. Additionally, columns can be added from the 'Name' or 'RECORD_TITLE' parameters by splitting on | and : e.g. 'MZ:100.2 | RT:20 | xcms_grp_id:1' would create MZ,RT and xcms_grp_id columns": "SIRIUS-CSI:Fingerid的输出可以添加额外的元数据列；这些列可以来自所有MSP参数，或者来自'Name'和'RECORD_TITLE'的MSP参数。此外，可以通过在'|'和':'上拆分来从'Name'或'RECORD_TITLE'参数添加列。例如，'MZ:100.2 | RT:20 | xcms_grp_id:1'将创建MZ、RT和xcms_grp_id列。",
    "Choose whether to include a suspect list": "选择是否包含一个可疑列表",
    "A value in ppm that defines the deviation of theoretical masses in the database vs. the measured masses": "在ppm值中定义了数据库中理论质量与测量质量偏差的值",
    "Relative mass deviation in ppm of theoretical fragment peaks vs. measured fragment peaks": "理论碎片峰与测量碎片峰的相对质量偏差（ppm）",
    "Absolute mass deviation in Dalton of theoretical fragment peaks vs. measured fragment peaks": "理论片段峰与测量片段峰的绝对质量偏差（以道尔顿为单位）",
    "The polarity used for the mode of acquisition": "用于采集模式的极性",
    "The schema used for the MSP file (auto will try automatically determine the schema)": "用于MSP文件的模式（自动将尝试自动确定模式）",
    "The MetFrag output can have additional meta data columns added, these can be either extracted from the MSP parameters or from the 'Name' and 'RECORD_TITLE' MSP parameter. Additionally, columns can be added from the 'Name' or 'RECORD_TITLE' parameter by splitting on | and : e.g. 'MZ:100.2 | RT:20 | xcms_grp_id:1' would create MZ,RT and xcms_grp_id columns": "MetFrag的输出可以添加额外的元数据列，这些列可以来自MSP参数，也可以来自'Name'和'RECORD_TITLE'的MSP参数。此外，可以通过在'|'和':'上拆分来从'Name'或'RECORD_TITLE'参数添加列，例如'MZ:100.2 | RT:20 | xcms_grp_id:1'将创建MZ、RT和xcms_grp_id列。",
    "Either provide a file containing a list of suspect compounds or a default file of an aggregated list of in silico predicted MS/MS spectra of natural products from the Universal Natural Products Database (http://pkuxxj.pku.edu.cn/UNPD/index.php). The list is an aggregated version of the github repository https://github.com/oolonek/ISDB/tree/master/Data/dbs.": "要么提供一个包含疑似化合物列表的文件，或者提供一个默认文件，该文件是来自通用天然产物数据库（http://pkuxxj.pku.edu.cn/UNPD/index.php）的天然产物的MS/MS谱的in silico预测的聚合列表。该列表是github仓库https://github.com/oolonek/ISDB/tree/master/Data/dbs的聚合版本。",
    "File containing a list of suspects inchikeys": "包含嫌疑人 inchikeys 列表的文件",
    "If no adduct type is provided within the MSP file or if the adduct type is not usable with MetFrag, set to 'yes' if these spectra should be skipped or 'no' if the default of [M+H]+ for pos data or [M-H]- for neg data should be used": "如果MSP文件中没有提供加合物类型，或者加合物类型不适用于MetFrag，那么设置为'yes'表示应该跳过这些谱图，或者设置为'no'表示应该使用默认的[M+H]+对于正数据或[M-H]-对于负数据。",
    "This will output all the parameters used for each MetFrag command line call (including the fragment peaks)": "这将输出每个MetFrag命令行调用使用的所有参数（包括碎片峰）。",
    "Filter by minimum of contained elements. Ex: N2O3 include compounds with at least 2 nitrogens and 3 oxygens": "根据所含元素的最小数量进行过滤。例如：N2O3 包括至少含有2个氮元素和3个氧元素的化合物。",
    "Filter by maximum of contained elements. Ex: N5O7 filter out compounds with at maximum 5 nitrogens and 7 oxygens": "根据所含元素的最大数量进行过滤。例如：N5O7 过滤掉含氮最多为5个和氧最多为7个的化合物。",
    "Filter by presence of defined sub-structures. Ex: c1ccccc1 include compounds containing benzene": "根据特定子结构的存在进行过滤。例如：c1ccccc1 包含含有苯环的化合物",
    "Filter by absence of defined sub-structures. Ex: [OX2H] filter out compounds containing hydroxyl groups": "根据缺少定义的子结构进行过滤。例如：[OX2H] 过滤掉含有羟基的化合物",
    "Filter by presence of defined elements (other elements are allowed). Ex: 'N,O' include compounds containing nitrogen and oxygen": "根据定义的元素存在性进行过滤（允许其他元素存在）。例如：'N,O' 包括含有氮和氧的化合物。",
    "Filter by presence of defined elements (no other elements are allowed). Ex: 'N,O' include compounds only composed of nitrogen and oxygen": "根据特定元素的存在进行过滤（不允许有其他元素）。例如：'N,O' 仅包括由氮和氧组成的化合物",
    "Filter by absence of defined sub-structures. Ex: 'Cl,Br' filter out compounds including bromine or chlorine": "根据缺少定义的子结构进行过滤。例如：'Cl,Br' 过滤掉包含溴或氯的化合物。",
    "Query SQLite database - in the standard XCMS msPurity workflow - the output of msPurity.createDatabase should be used here. However any SQLite database following the schema of as https://bioconductor.org/packages/release/bioc/vignettes/msPurity/inst/doc/msPurity-spectral-database-vignette.html can be used as input": "查询 SQLite 数据库 - 在标准的 XCMS msPurity 工作流程中 - 这里应使用 msPurity.createDatabase 的输出。然而，任何遵循 https://bioconductor.org/packages/release/bioc/vignettes/msPurity/inst/doc/msPurity-spectral-database-vignette.html 模式的 SQLite 数据库都可以用作输入",
    "Library SQLite database - in the standard XCMS msPurity workflow - a default database of MassBank, HMDB, LipidBlast and GNPS is used. However any SQLite database following the schema of https://bioconductor.org/packages/release/bioc/vignettes/msPurity/inst/doc/msPurity-spectral-database-vignette.html can be used as input": "库 SQLite 数据库 - 在标准的 XCMS msPurity 工作流程中 - 使用 MassBank、HMDB、LipidBlast 和 GNPS 的默认数据库。然而，任何遵循 https://bioconductor.org/packages/release/bioc/vignettes/msPurity/inst/doc/msPurity-spectral-database-vignette.html 模式的 SQLite 数据库都可以用作输入。",
    "Retention time tolerance in seconds to match precursors": "保留时间公差（以秒为单位）以匹配前体",
    "If True, spectra will be filtered by similarity of precursors based on the library and query ppm defined tolerance": "如果为真，将根据库和查询ppm定义的容忍度，通过前体的相似性来过滤谱图",
    "Relative abundance weight for spectra (default to 0.5 as determined by massbank for ESI data)": "谱图的相对丰度权重（默认为0.5，这是MassBank为ESI数据确定的值）",
    "mz weight for spectra (default to 2 as determined by massbank for ESI data)": "谱图的 m/z 权重（默认为 2，这是 MassBank 为 ESI 数据确定的）",
    "A copy will be made of the input SQLite target database and the results will be added to this copy. When False, the input SQLite database will be updated with the matching results. Use False if you want to reduce storage space being used.": "将对输入的SQLite目标数据库进行复制，并将结果添加到此副本中。如果设置为False，则会使用匹配结果更新输入的SQLite数据库。如果您想减少使用的存储空间，请使用False。",
    "xcmsSet object saved as an RData file": "xcmsSet 对象保存为 RData 文件",
    "If just a single file for purity is to be calculated (rather than the grouped XCMS peaks). Uses the index of the files in xcmsSet object. If zero this is ignored": "如果只需要计算单个纯度文件（而不是分组的XCMS峰），则使用xcmsSet对象中文件的索引。如果为零，则忽略此选项。",
    "Comma separated string of classes (found in the phenoData slot of xcmsSet object i.e. xset@phenoData). An average will across all the choosen classes.": "逗号分隔的类别字符串（位于xcmsSet对象的phenoData槽中，即xset@phenoData）。将计算所有选定类别的平均值。",
    "Comma separated string of filenames to ignore (do not include the directory) eg. blank1.mzML,blank2.mzML,sample4.mzML": "逗号分隔的文件名字符串，用于指定要忽略的文件（不要包含目录），例如：blank1.mzML,blank2.mzML,sample4.mzML",
    "Comma separated string of xcms groups to perform prediction on eg. 1,2,3,4": "逗号分隔的xcms组字符串，用于执行预测，例如1,2,3,4",
    "Area and average used for the purity predictions. FWHM='Full width half maximum', FW='Full width'": "用于纯度预测的区域和平均值。FWHM='半高宽'，FW='全宽'",
    "Only applicable when using retention time correction with obiwarp. When obiwarp is used the retention time correction tracking is slightly different to the other correction methods. To ensure correct tracking with obiwarp an additional tool (track_rt_raw) should be performed prior to any retention time correction.": "仅在使用obiwarp进行保留时间校正时适用。当使用obiwarp时，保留时间校正跟踪与其他校正方法略有不同。为了确保与obiwarp的正确跟踪，在进行任何保留时间校正之前，应先执行一个额外的工具（track_rt_raw）。",
    "tsv or tabular file with one column containing the mz values (column header should be either mz)": "包含一列质谱值（mz）的tsv或表格文件（列标题应为mz）",
    "ppm tolerance to compare between mz values": "ppm 容差用于比较 m/z 值之间的差异",
    "Choose the file number from the dimspy matrix to use to calculate the precursor ion purity metric (order based on column order). This file will then be looked for in the library folder to calculate the metric.": "从dimspy矩阵中选择文件编号，用于计算前体离子纯度指标（基于列顺序）。然后，将在库文件夹中查找此文件以计算指标。",
    "Select a minimum of two hdf5 files": "选择至少两个HDF5文件",
    "Based on a 'multilist' column in the filelist text file, divide the peaklists into different lists e.g. [pl1 pl2], [pl3, pl4] can be converted to 2 new lists: [pl1, pl3] [pl2, pl4]": "基于文件列表文本文件中的“multilist”列，将峰值列表划分为不同的列表，例如 [pl1 pl2], [pl3, pl4] 可以转换为 2 个新列表：[pl1, pl3] [pl2, pl4]",
    "A table containing metadata for each experimental sample. See help section below for more details.": "一个包含每个实验样本元数据的表格。有关更多详细信息，请参见下面的帮助部分。",
    "RData file containing the purityA object following frag4feature and/or averageFragmentation": "包含遵循 frag4feature 和/或 averageFragmentation 的 purityA 对象的 RData 文件",
    "'Average all' will use the averaged MS/MS spectra of the XCMS grouped feature, ignoring inter and intra file relationships, 'Average' (intra)' will use the intra file (within file) averaged MS/MS spectra of the XCMS grouped feature, 'Average (inter)' will use the inter file (across file) MS/MS averaged spectra, 'All scans' will export all matching MS/MS spectra to XCMS grouped features, 'Max intensity' will choose the most MS/MS spectra with the most intense precursor ion for each XCMS grouped feature": "“平均所有”将使用XCMS分组特征的平均MS/MS谱图，忽略文件间和文件内的关系，“平均（文件内）”将使用XCMS分组特征的文件内（文件内）平均MS/MS谱图，“平均（文件间）”将使用文件间（跨文件）MS/MS平均谱图，“所有扫描”将导出所有匹配的MS/MS谱图到XCMS分组特征，“最大强度”将为每个XCMS分组特征选择最强烈的前体离子的MS/MS谱图。",
    "Metadata for each grouped XCMS feature, e.g. could be the adduct calculated from CAMERA": "每个分组的XCMS特征的元数据，例如可以是CAMERA计算的加合物。",
    "Comma separated string of column names where the corresponding values in the metadata will be used": "逗号分隔的列名字符串，其中元数据中相应的值将被使用",
    "Comma separated string of column names where the corresponding values in the metadata will be used for MSP spectra name": "逗号分隔的列名字符串，其中元数据中的相应值将用于MSP光谱名称",
    "if set to no, all XCMS group features will be used": "如果设置为否，将使用所有XCMS组特征",
    "Comma separated string of XCMS group ids to export MSP spectra for. If blank all XCMS peak groups will be used": "逗号分隔的 XCMS 组 ID 字符串，用于导出 MSP 谱图。如果为空，则将使用所有 XCMS 峰组。",
    "Additional adducts to include, can be useful for downstream processing": "额外的加成物包括，对于下游处理可能有用",
    "Additional custom adducts to include, should be comma separated": "要包括的额外自定义加合物，应该用逗号分隔",
    "Useful if the MSP file will be used for further annotation": "如果MSP文件将用于进一步注释，则非常有用",
    "Either xcmsSet object saved as 'xset' or CAMERA 'xa' object in an RData file. Please specify which to use below": "要么将 xcmsSet 对象保存为 'xset'，要么在 RData 文件中使用 CAMERA 的 'xa' 对象。请在下面指定使用哪一个。",
    "purityA object saved as 'pa' in a RData file (output from frag4feature)": "纯净度A对象被保存为RData文件中的'pa'（frag4feature的输出）",
    "The Extracted Ion Chromatogram can be calculated for each peak and stored in the database. Note, this will take considerable time for large datasets and the resulting SQLite database can be large": "提取的离子色谱图可以为每个峰计算并存储在数据库中。请注意，这对于大型数据集将需要相当长的时间，并且生成的SQLite数据库可能会很大。",
    "purityA object saved as 'pa' in a RData file (output from frag4feature tool)": "纯度A对象保存为RData文件中的'pa'（frag4feature工具的输出）",
    "Maximum tolerated m/z deviation in parts per million.": "最大允许的m/z偏差，以百万分之一为单位。",
    "This is ignored for intensities when intensities are summed.": "这在强度被求和时会被忽略。",
    "purityA object saved as 'pa' in a RData file where fragmentation has been linked to XCMS features(output from frag4feature tool)": "在RData文件中保存为'pa'的对象，其中将片段化与XCMS特征相关联（来自frag4feature工具的输出）的纯度A对象。",
    "grouped xcmsSet object saved as 'xset' in an RData file": "分组的 xcmsSet 对象保存为 RData 文件中的 'xset'",
    "purityA object generated from msPurity_purityA. Contains details of fragmentation spectra and precursor ion purity results (output from purityA tool)": "由msPurity_purityA生成的purityA对象。包含碎片化谱图和前体离子纯度结果的详细信息（来自purityA工具的输出）。",
    "Fragmentation will be ignored if the precursor mz value is not within the ppm error tolerance to the XCMS feature mz": "如果前体 m/z 值不在 XCMS 特征 m/z 的 ppm 误差容限范围内，则将忽略碎裂。",
    "Fragmentation will be ignore if the precursor ion purity is less than the threshold (further filtering on the precursor ion purity can be done at the averaging stage if required).": "如果前体离子纯度低于阈值，则将忽略碎片化（如果需要，可以在平均阶段进一步过滤前体离子纯度）。",
    "If TRUE the most intense precursor will be used. If FALSE the precursor closest to the center of the isolation window will be used": "如果为真，将使用最强烈的前体。如果为假，将使用最接近隔离窗口中心的前体。",
    "If retention time correction has been used in XCMS set this to yes": "如果在XCMS中使用了保留时间校正，则将其设置为“是”",
    "For special cases where the MS2 files have no MS1 data or if the MS1 data in the MS2 file is unreliable": "对于MS2文件中没有MS1数据或MS2文件中的MS1数据不可靠的特殊情况",
    "If yes, this will ignore the recorded precursor within the mzML file and use use the most intense peak within the isolation window to calculate the precursor ion purity": "如果选择是，这将忽略mzML文件中记录的前体，并使用隔离窗口内最强烈的峰来计算前体离子纯度",
    "If TRUE, this will use the neareset full scan to the fragmentation scan to determine what the m/z value is of the precursor": "如果为真，这将使用最接近碎片化扫描的完整扫描来确定前体的 m/z 值是什么",
    "Set the ppm tolerance for the precursor ion purity interpolation. i.e. the ppm tolerence between the precursor ion found in the neighbouring scans. The closest match within the window will be used for the interpolation": "设置前体离子纯度插值的ppm容忍度。即在邻近扫描中找到的前体离子之间的ppm容忍度。窗口内最接近的匹配将用于插值。",
    "The path to the xcmsSet object saved as an RData file": "xcmsSet对象保存为RData文件的路径",
    "Relative Standard Deviation threshold for the sample classes": "样本类别的相对标准偏差阈值",
    "minimum fraction of files for features needed for the sample classes": "最小文件比例，用于样本类别所需的特征",
    "Relative standard Deviation threshold for the retention time of the sample classes": "样本类别保留时间的相对标准偏差阈值",
    "Intensity threshold for the sample": "样本的强度阈值",
    "A string representing the class that will be used for the blank": "一个字符串，表示将用于空白的类",
    "RSD threshold for the blank": "空白的RSD阈值",
    "minimum fraction of files for features needed for the blank": "空白所需的特征文件的最小比例",
    "RSD threshold for the retention time of the blank": "空白保留时间的RSD阈值",
    "Intensity threshold for the blank": "空白的强度阈值",
    "fold change (sample/blank) needed for sample peak to be allowed. e.g. if s2b set to 10 and the recorded sample 'intensity' value was 100 and blank was 10. 1000/10 = 100, so sample has fold change higher than the threshold and the peak is not considered a blank": "样品与空白的倍数变化（fold change）需要满足样品峰值被允许的条件。例如，如果s2b设置为10，记录的样品“强度”值为100，空白为10，则1000/10=100，因此样品的倍数变化高于阈值，峰值不被视为空白。",
    "minfrac for xcms  grouping": "minfrac 是 xcms 分组的参数",
    "mzwid for xcms  grouping": "mzwid 是 xcms 分组的参数",
    "bw for xcms  grouping": "bw for xcms 分组",
    "Threshold for filtering out non gaussian shaped peaks. Note this only works if the 'verbose columns' and 'fit gauss' was used with xcms": "用于过滤掉非高斯形状峰的阈值。请注意，这只有在使用xcms时同时使用了'verbose columns'和'fit gauss'才会有效。",
    "Assign True if files for each step saved (for testing purposes)": "为每个步骤保存的文件分配真值（用于测试目的）",
    "polarity (just used for naming purpose when files are saved)": "极性（仅用于保存文件时的命名目的）",
    "comma seperated list of grouped_xcms peak ids to remove (corresponds to the row from xcms::group output) e.g '1,20,30,56'": "逗号分隔的grouped_xcms峰值ID列表，以删除（对应于xcms::group输出的行），例如'1,20,30,56'",
    "Name of the xcmsSet object within the RData file": "RData文件中xcmsSet对象的名称",
    "A samplelist can be provided to find an appriopiate blank class (requires a column 'blank' where 'yes' indicates the class should be used as the blank)": "一个样本列表可以提供以找到合适的空白类别（需要一个列名为'blank'，其中'yes'表示该类别应被用作空白）",
    "Choose data-matrix file (tab-separated values with sample names in first row and feature names in first column).": "选择数据矩阵文件（以制表符分隔的值，第一行包含样本名称，第一列包含特征名称）。",
    "Choose sample-metadata file (tab-separated values with one row per sample, sample name in first column).": "选择样本元数据文件（用制表符分隔的值，每个样本一行，样本名称在第一列）。",
    "Choose variable-metadata file (tab-separated values with one row per feature, feature name in first column).": "选择变量元数据文件（以制表符分隔的值，每个特征一行，特征名称在第一列）。",
    "Name the column in 'Sample metadata' that has the values to be referenced by 'Sample-class names' and 'Compute centers for classes'. [default: 'class']": "在“样本元数据”中指定一个列名，该列的值将被“样本类别名称”和“计算类别中心”所引用。[默认: 'class']",
    "List of names (or patterns to match names) of sample classes to include or exclude.  List should be comma-separated with no stray space characters. (Leave this empty to match no names.) [default: empty]": "包含或排除样本类别名称（或匹配名称的模式）的列表。列表应该是逗号分隔的，没有多余的空格字符。（保持空白以匹配没有名称。）[默认值：空]",
    "Indicate meaning of preceding list: either to identify classes to exclude from output or to identify classes to include in output. [default: 'filter-out']": "指示前一个列表的含义：要么用于识别要从输出中排除的类别，要么用于识别要包含在输出中的类别。[默认值：'filter-out']",
    "See '<i>Wild-card patterns to match class names</i>' and '<i>Regular-expression patterns to match sample-class names</i>' sections below. [default: 'wild-card patterns']": "请参见下文的“<i>通配符模式以匹配类名</i>”和“<i>正则表达式模式以匹配样本类名</i>”部分。[默认值：'通配符模式']",
    "List of filters, each specifying the range of permitted values in a column of 'Variable metadata' (specified as 'column:min:max'), as described in '<i>Variable-range filters</i>' section below.  List should be comma-separated with no stray space characters.  (Leave this empty for no filtering.) [default: empty]": "列出过滤器，每个过滤器指定“变量元数据”一列中允许值的范围（格式为'列：最小值：最大值'），如下面的'<i>变量范围过滤器</i>'部分所述。列表应该是逗号分隔的，没有多余的空格字符。（不进行过滤则留空。）[默认值：空]",
    "Choose transformation.  In all cases, negative intensities become missing values.  See '<i>Data transformation and imputation</i>' section below. [default: 'none']": "选择转换。在所有情况下，负强度值将变成缺失值。请参阅下面的'<i>数据转换和插补</i>'部分。[默认值：'none']",
    "Choose imputation for missing values.  See '<i>Data transformation and imputation</i>' section below. [default: 'zero']": "选择缺失值的插补方法。 详见下面的'<i>数据转换和插补</i>'部分。[默认值: '零']",
    "List of sample-metadata column names for sorting samples.  List should be comma-separated with no stray space characters.  (This is ignored when 'Compute centers for classes' is set to either 'centroid' or 'median'.) [default: 'sampleMetadata']": "用于排序样本的样本元数据列名称列表。列表应该是逗号分隔的，没有多余的空格字符。（当“计算类别的中心”设置为'centroid'或'median'时，这将被忽略。）[默认值：'sampleMetadata']",
    "List of feature-metadata column names for sorting features.  List should be comma-separated with no stray space characters. [default: 'variableMetadata']": "用于排序特征的特征元数据列名称列表。列表应该是逗号分隔的，没有多余的空格字符。[默认值: 'variableMetadata']",
    "[default: 'none']": "[默认：'无']",
    "If correlation is selected, the pearson method will be used by default unless a specific method is selected in the advanced parameters below": "如果选择了相关性，将默认使用皮尔逊方法，除非在下面的高级参数中选择了特定的方法",
    "Standardization is performed after the clustering for display only (may enhance contrast) and does not modify cluster computation nor intensities in the output files": "聚类后进行标准化，仅用于显示（可能增强对比度），并不修改聚类计算或输出文件中的强度。",
    "Allow only molecular formulas for that at least one structural formula exists or specify advanced restictions, note that the default of genform is 'no existence check'": "只允许至少存在一种结构式的分子公式，或者指定高级限制，请注意，genform的默认设置是“不进行存在性检查”",
    "Even for graphical formulas": "即使是图形化的公式",
    "(default: mass of MS basepeak)": "（默认值：质谱基峰的质量）",
    "In parts per million": "每百万分之",
    "Select output type of the MS match value": "选择MS匹配值的输出类型",
    "of MS/MS peak in ppm": "质谱/质谱峰值的百万分率",
    "Choose if you want to exclude particular zone(s)": "选择是否要排除特定的区域",
    "Maximal allowed shift for spectra alignment. Default value is 0.01 ppm": "允许的最大谱图对齐偏移量。默认值是0.01 ppm。",
    "Choose if you want to display particular zone(s)": "选择是否要显示特定的区域（们）",
    "@INPUT_IMAGE_HELP@ from multiple findChromPeaks": "来自多个findChromPeaks的@INPUT_IMAGE_HELP@",
    "Sample metadata or variable metadata": "样本元数据或变量元数据",
    "Select the first method if you don't want to take into account any class of samples": "选择第一种方法，如果您不想考虑任何类别的样本",
    "Name of the column in Sample metadata": "样本元数据中的列名",
    "Choose if you want the mean fold change to be converted into a log mean fold change (log2 or log10)": "选择是否将平均倍数变化转换为对数平均倍数变化（以2为底或以10为底）",
    "Name of the class to test versus all the remaining samples.": "要测试的类别名称与所有剩余样本相对比。",
    "Exact name; must be in your metadata file": "确切名称；必须在您的元数据文件中",
    "Default method is total intensity": "默认方法是总强度",
    "Select 'None' for no representation,'Overlay' to overlay all spectra on a unique chart and 'One per individual' to generate an individual chart for each observation": "选择“无”表示不展示，选择“叠加”将所有谱图叠加在一张独特的图表上，选择“每个个体一个”为每个观察生成一个单独的图表。",
    "Name of sample metadata column corresponding to the fixed factor (use none if only time factor": "样本元数据列的名称，对应于固定因子（如果只有时间因子，则使用none）",
    "Name of the column of the sample metadata table corresponding to the repeated factor": "样本元数据表中对应重复因子的列名",
    "Name of the column of the sample metadata table corresponding to the subject factor": "样本元数据表中对应于受试者因素的列名",
    "Transformation of raw data": "原始数据的转换",
    " Used to assess the quality of models considering distribution of residuals": "用于评估模型质量，考虑残差分布",
    "Should the numerical values generated by this tool be rounded to a chosen number of decimal places?": "这个工具生成的数值是否应该四舍五入到选定的小数位数？",
    "Positive interger": "正整数",
    "The two input tables must have the same sample IDs": "两个输入表格必须具有相同的样本ID",
    "The zero filter removes variables which have all their correlation coefficients equal to 0. The threshold filter removes variables which have all their correlation coefficients, in absolute value, strictly below a threshold.": "零过滤器移除所有相关系数都等于0的变量。阈值过滤器移除所有相关系数的绝对值都严格低于一个阈值的变量。",
    "To determine whether a colored correlation table is plotted.": "确定是否绘制彩色相关性表格。",
    "Standard corresponds to a scale with a smooth gradient between three colors: red, white and green (continuous case). Customized creates classes for the correlation coefficients - the scale has discrete values.": "标准对应于三种颜色之间平滑渐变的刻度：红色、白色和绿色（连续情况）。自定义为相关系数创建类别 - 刻度具有离散值。",
    "Regular means the classes have the same size. Irregular means it is possible to choose any intervals.": "“Regular”意味着类别具有相同的大小。“不规则”意味着可以选择任意间隔。",
    "The vector must be of the following form: (value1,value2,value3,..). The values must be between -1 and 1 not included. For example: (-0.8,-0.5,-0.4,0,0.4,0.5,0.8).": "向量必须具有以下形式：(value1,value2,value3,..)。这些值必须在-1和1之间，不包括-1和1。例如：(-0.8,-0.5,-0.4,0,0.4,0.5,0.8)。",
    "if 'NO' is selected then one or more mass(es) must be entered manually": "如果选择“否”，则必须手动输入一个或多个质量值。",
    "Generally variable metadata file. Please note : HMDB allows maximum 700 query masses per request": "通常变量元数据文件。请注意：HMDB允许每个请求最多查询700个质量数。",
    "if 'YES' is selected then enter your number of header lines": "如果选择“是”，则输入你的标题行数",
    "number of lines not containing masses": "不包含质量的行数",
    "For a masses list, writes : m1 m2 m3. Your values must be separated by spaces. You should use dot (.) like decimal separator": "对于质量列表，写作：m1 m2 m3。你的值必须用空格分隔。你应该使用点（.）作为小数分隔符。",
    "Default value is 0.05 for HMDB (mass-to-charge ratio)": "默认值为0.05，用于HMDB（质荷比）",
    "Default value is 10 for each queried mz / max is 50": "默认值是每个查询的mz为10/最大为50",
    "Getting more information from HMDB (Inchi, logP, ...) but increase time of tool execution": "从HMDB获取更多信息（Inchi，logP等），但增加了工具执行的时间。",
    "Generally variable metadata file": "通常变量元数据文件",
    "Do you have retention times in your bank? In your analyzed file? Would you use it?": "您有保留时间吗？在您的分析文件中？您会使用它吗？",
    "Default value is 0.1min": "默认值是0.1分钟",
    "Do you have retention times in your bank? Would you use it?": "您有保留时间吗？您会使用它吗？",
    "For a rt list, writes : rt1 rt2 rt3. Your values ​​must be separated by spaces. You should use dot (.) like decimal separator": "对于一个保留时间列表，写出：rt1 rt2 rt3。你的值必须由空格分隔。你应该使用点（.）作为小数分隔符。",
    "For a masses list, writes : m1 m2 m3. Your values ​​must be separated by spaces. You should use dot (.) like decimal separator": "对于质量列表，写入：m1 m2 m3。您的值必须由空格分隔。您应该使用点（.）作为小数分隔符。",
    "Default value is 0.005 Da": "默认值是0.005 Da",
    "Default value is 5 ppm": "默认值是5 ppm",
    "if 'NO' is selected an internal file will be used! It doesn't contain a retention time (RT)": "如果选择“否”，则将使用一个内部文件！该文件不包含保留时间（RT）",
    "Do you have retention times in your bank? In your analyzed data? Would you use it?": "你有保留时间在你的库中吗？在你的分析数据中？你会使用它吗？",
    "if 'YES' is selected then 2 more files will be generate (JSON + variableMetadata and bank full column)": "如果选择“是”，则会生成两个更多的文件（JSON + variableMetadata 和银行全列）。",
    "If 'NO' is selected then one or more mass(es) must be entered manually.": "如果选择“否”，则必须手动输入一个或多个质量值。",
    ".msp output file from metaMS.runGC function, or any msp file.": "来自metaMS.runGC函数的.msp输出文件，或任何.msp文件。",
    "For a list of masses + intensities write : mz1 int1 mz2 int2 mzx intx...": "对于一系列质量+强度，写入：mz1 int1 mz2 int2 mzx intx...",
    "5%-phenyl-95%-dimethylpolysiloxane (VAR5), 35%-phenyl-65%-dimethylpolysiloxane (MDN35).": "5%-苯基-95%-二甲基聚硅氧烷(VAR5)，35%-苯基-65%-二甲基聚硅氧烷(MDN35)。",
    "If neither an alkane RIs for VAR5 nor MDN35 is available in your setup, please select 'none' in the input field above!": "如果在你的设置中既没有烷烃RIs用于VAR5，也没有MDN35，请在上方的输入字段中选择“无”！",
    "This value is for the library search used only. A larger window size will increase the number of matches. At the same time the identification becomes less reliable due to false matching spectra without RI consensus.": "这个值仅用于库搜索。窗口尺寸越大，匹配的数量就会增加。同时，由于没有RI共识的假匹配谱图，鉴定的可靠性会降低。",
    "Maximum number of hits per queried spectra, default=all (0).": "每个查询谱最多命中数，默认为全部（0）。",
    "Number of significant decimals of your m/z.": "你的 m/z 的有效小数位数。",
    "Number of m/z per spectra you want to keep for the queries, default 0 = all of them.": "您希望为查询保留每个谱图的 m/z 数量，默认值为 0 表示保留所有 m/z。",
    "Number of matches, divided by the sum of matches and mismatches.": "匹配数除以匹配数和不匹配数之和。",
    "The distance measure S12GowLeg = sqrt(1 - s12) is derived from the S12 coefficient of Gower & Legendre defined as s12 = a / sqrt((a + b)(a + c)).": "距离度量 S12GowLeg = sqrt(1 - s12) 是由 Gower 和 Legendre 定义的 S12 系数派生出来的，S12 系数定义为 s12 = a / sqrt((a + b)(a + c))。",
    "The Dotproduct distance is summing the multiplied intensities over all matching peaks within both spectra. Here, to satisfy the conditions of a metric I) non-negativity, II) identity of indiscernibles, III) symmetry and IV) subadditivity / triangle inequality, we use 1-Dotproduct. Both spectra are normalised prior to the spectral vector norm in that way, that the absolute value of the squared intensities is equal to 1.": "点积距离是对两个光谱中所有匹配峰值的乘积强度进行求和。在这里，为了满足度量条件 I) 非负性，II) 不可分辨身份的一致性，III) 对称性，以及 IV) 次可加性 / 三角不等式，我们使用 1-点积。在进行光谱向量范数计算之前，两个光谱都被标准化，使得平方强度的绝对值等于 1。",
    "In information theory, the Hamming distance between two strings of equal length is the number of positions for which the corresponding symbols are different. Put another way, it measures the minimum number of substitutions required to change one into the other, or the number of errors that transformed one string into the other.": "在信息论中，两个等长字符串之间的汉明距离是对应符号不同的位数。换句话说，它衡量了将一个字符串转换为另一个字符串所需的最小替换次数，或者将一个字符串转换为另一个字符串的错误数量。",
    "The Euclid is the square root of the sum of the squared differences over all matching peaks.": "欧几里得距离是所有匹配峰值的平方差之和的平方根。",
    "Would you work with relative or absolute intensities? Example: relative = percentage, absolute = untouched. Relative is preferred": "你会使用相对强度还是绝对强度？例如：相对 = 百分比，绝对 = 未处理。相对强度是首选。",
    "Select 'YES' to display the spectra or 'NO' to not display them": "选择'YES'来显示谱图或者'NO'不显示它们",
    "Default value is 1e6": "默认值是1e6",
    "Default method is Decreasing Exponential signal. See details below": "默认方法是递减指数信号。详见下文。",
    "Default value is 1": "默认值是1",
    "Default value is 0": "默认值是0",
    "Default value is 0.5": "默认值是0.5",
    "Default value is 0.3": "默认值是0.3",
    "Method used to select the angles to rotate the spectra": "用于选择旋转光谱的角度的方法",
    "Definition of the search zone": "搜索区域的定义",
    "Default value is 0.02": "默认值是0.02",
    "How to deal with shifts between spectra on their left and right sides": "如何处理光谱左右两侧的偏移",
    "Smoothing parameter, generally 1e5 – 1e8. Default value is 100000": "平滑参数，通常为1e5到1e8。默认值为100000",
    "Asymmetry parameter. Default value is 0.05": "不对称参数。默认值是0.05",
    "Numerical precision for convergence when estimating the baseline. Default value is 1e-8": "在估计基线时，收敛的数值精度。默认值是1e-8",
    "If YES, sets negative intensities to zero": "如果为是，将负强度设置为零",
    "Default value is line 1": "默认值是第1行",
    "Select 'FALSE' when there is no subdirectories, 'TRUE' if there are subdirectories": "当没有子目录时选择'FALSE'，如果有子目录则选择'TRUE'",
    "Select 'TRUE' to use the subdirectories names as the FID names (instead of looking in the title file)": "选择 'TRUE' 以使用子目录名称作为 FID 名称（而不是在标题文件中查找）",
    "output file from another function xcms (xcmsSet, group, retcor, fillpeaks etc.)": "来自另一个函数 xcms (xcmsSet, group, retcor, fillpeaks 等) 的输出文件",
    "output file from another function xcms (fillPeaks)": "来自另一个函数 xcms (fillPeaks) 的输出文件",
    "[sigma]": "[西格玛]",
    "[perfwhm]": "[性能监控]",
    "[ppm]": "[每百万分之一]",
    "[mzabs]": "[质量绝对值]",
    "[maxcharge]": "[maxcharge]",
    "[maxiso]": "[maxiso]",
    "[minfrac]": "最小分数",
    "[quick] If TRUE, use only groupFWHM and findIsotopes functions. Else if FALSE, use also groupCorr and findAdducts": "[快速] 如果为 TRUE，则仅使用 groupFWHM 和 findIsotopes 函数。否则，如果为 FALSE，则还使用 groupCorr 和 findAdducts",
    "[cor_eic_th]": "[cor_eic_th]",
    "[graphMethod]": "[图方法]",
    "[pval]": "P值",
    "[calcCiS]": "[计算CiS]",
    "[calcIso]": "[计算同位素]",
    "[calcCaS]": "[计算CaS]",
    "[max_peaks]": "最大峰值数",
    "[multiplier]": "[乘数]",
    "[rules]": "[规则]",
    "[eicmax]": "[eicmax]",
    "[eicwidth]": "[电离电流宽度]",
    "[value]": "[值]",
    "[height]": "[高度]",
    "[width]": "宽度",
    "[mzdec]": "质谱分辨率",
    "[sortpval]": "[sortpval]",
    "@INPUT_IMAGE_HELP@ from groupChromPeaks": "从groupChromPeaks的@INPUT_IMAGE_HELP@",
    "Each peak is expanded in mz direction by ‘expandMz *’ their original mz width. A value of ‘0’ means no expansion, a value of ‘1’ grows each peak by 1 * the mz width of the peak resulting in peakswith twice their original size in mz direction (expansion by half mz width to both sides).": "每个峰值在mz方向上通过“expandMz *”它们原始的mz宽度进行扩展。值为“0”表示不扩展，值为“1”则使每个峰值增加1 * 峰值的mz宽度，从而在mz方向上使峰值的大小增加一倍（在两侧各扩展半个mz宽度）。",
    "Each peak is expanded in RT direction by ‘expandRt *’ their original RT width. A value of ‘0’ means no expansion, a value of ‘1’ grows each peak by 1 * the RT width of the peak resulting in peakswith twice their original size in RT direction (expansion by half RT width to both sides).": "每个峰值在保留时间（RT）方向上通过“expandRt *”其原始保留时间宽度进行扩展。值为“0”表示不扩展，值为“1”则使每个峰值增加1 * 峰值的保留时间宽度，从而使峰值在保留时间方向上的大小是原始大小的两倍（保留时间宽度的一半扩展到两侧）。",
    "For peaks with an mz width smaller than ‘mean(c(mzmin, mzmax)) * ppm / 1e6’, the ‘mzmin’ will be replaced by ‘mean(c(mzmin, mzmax)) - (mean(c(mzmin, mzmax)) * ppm / 2 / 1e6)’ and ‘mzmax’ by ‘mean(c(mzmin, mzmax)) + (mean(c(mzmin, mzmax)) * ppm / 2 / 1e6)’. This is applied before eventually expanding the mz width using the ‘expandMz’ parameter.": "对于质荷比宽度小于“mean(c(mzmin, mzmax)) * ppm / 1e6”的峰，‘mzmin’将被替换为‘mean(c(mzmin, mzmax)) - (mean(c(mzmin, mzmax)) * ppm / 2 / 1e6)’，‘mzmax’将被替换为‘mean(c(mzmin, mzmax)) + (mean(c(mzmin, mzmax)) * ppm / 2 / 1e6)’。这将在最终使用‘expandMz’参数扩展质荷比宽度之前应用。",
    "@INPUT_IMAGE_HELP@ from: findChromPeaks, groupChromPeaks": "@输入图像帮助@ 来自：findChromPeaks, groupChromPeaks",
    "See the help section below": "请看下面的帮助部分",
    "(previously missing)": "（之前缺失）",
    "For a data set with 6 samples, ‘extraPeaks = 1’ uses all peak groups with a total peak count lower or equal to ‘6 + 1’. The total peak count is the total number of peaks being assigned to a peak group and considers also multiple peaks within a sample being assigned to the group. (previously extra)": "对于一个包含6个样本的数据集，'extraPeaks = 1' 使用所有峰值组，其总峰值计数低于或等于'6 + 1'。总峰值计数是分配给峰值组的峰值总数，并考虑一个样本中的多个峰值也被分配到该组。（之前是额外的）",
    "if gaussian fitting is by least-squares with no outlier removal, and if symmetric a re descending M estimator is used with Tukey's biweight function, allowing outlier removal": "如果高斯拟合是通过最小二乘法进行的，并且没有进行异常值剔除，那么如果使用Tukey双权重函数的对称下降M估计量进行异常值剔除",
    "See ‘step’ parameter in ‘profile-matrix’ documentation for more details. (previously profStep)": "请参阅“profile-matrix”文档中的“step”参数以获取更多详细信息。（之前是profStep）",
    "It defaults to ‘floor(median(1:length(fileNames(object))))’": "它默认为‘floor(median(1:length(fileNames(object))))’",
    "with ‘response = 0’ giving linear warping on start and end points and ‘response = 100’ warping using all bijective anchors.": "使用“response = 0”在起点和终点进行线性变形，而“response = 100”则使用所有双射锚点进行变形。",
    "@INPUT_IMAGE_HELP@ from: findChromPeaks, groupChromPeaks or adjustRtime": "来自：findChromPeaks、groupChromPeaks 或 adjustRtime",
    "bandwidth (standard deviation or half width at half maximum) of gaussian smoothing kernel to apply to the peak density chromatogram": "高斯平滑核的带宽（标准偏差或半高宽）应用于峰密度色谱图",
    "in at least one sample group in which the peaks have to be present to be considered as a peak group (feature)": "在至少一个样本组中，峰值必须存在才能被视为峰值组（特征）",
    "in at least one sample group in which the peaks have to be detected to be considered a peak group (feature)": "在至少一个样本组中，必须检测到峰值才能被认为是峰值组（特征）",
    "to use for creating peak density chromatograms and grouping peaks across samples (previously mzdiff)": "用于创建峰值密度色谱图和跨样本分组峰值（之前是mzdiff）",
    "must contain at least one column with the sample id and one column with the sample class": "必须至少包含一列样本ID和一列样本类别",
    "Must contain at least one column with the sample id and one column with the sample class": "必须包含至少一列样本ID和一列样本类别",
    "@INPUT_IMAGE_HELP@ from MSnbase readMSData": "从MSnbase的readMSData获取的@INPUT_IMAGE_HELP@",
    "min,max": "最小值，最大值",
    "Generates a tabular file with all the chromatographic peaks obtained.": "生成一个包含所有色谱峰的表格文件。",
    "Only used to calculate the actual sigma": "仅用于计算实际的σ",
    "The peak detection algorithm creates extracted ion base peak chromatograms (EIBPC) on a fixed step size. (Previously step)": "峰值检测算法在固定步长上创建提取离子基峰色谱图（EIBPC）。（之前是步长）",
    "(previously profmethod)": "（之前是profmethod）",
    "The default for the ‘baseValue’ is half of the smallest value in ‘x’ (‘NA’s being removed).": "默认情况下，‘baseValue’ 是 ‘x’ 中最小值的一半（移除 'NA' 值）。",
    "Leave it to empty to calculate it using fwhm by default at fwhm/2.3548": "保持为空，以默认使用fwhm的一半（即fwhm/2.3548）来计算。",
    "defining the signal to noise cutoff to be used in the chromatographic peak detection step": "定义色谱峰检测步骤中使用的信噪比截止值",
    "(i.e. the number of neighboring bins that will be joined to the slice in which filtration and peak detection will be performed)": "（即在执行过滤和峰值检测的切片中将与之合并的相邻箱子的数量）",
    "By default: 0.8-binSize*steps": "默认值：0.8-binSize*steps",
    "used in the second centWave run to identify peaks for predicted isotope ROIs.": "在第二次centWave运行中用于识别预测的同位素感兴趣区域（ROIs）的峰值。",
    "Isotopes will be defined for charges ‘1:maxCharge’.": "同位素将为电荷 '1:maxCharge' 定义。",
    "whether additional peak meta data columns should be returned": "是否应该返回额外的峰值元数据列",
    "Scales are linked to the width of the peaks that are to be detected.": "尺度与要检测的峰的宽度相关联。",
    "Ratio to the maximum of CWT coefficients": "与CWT系数最大值的比率",
    "Single file mode for the following formats: mzxml, mzml, mzdata and netcdf. Zip file mode for the following formats: zip. See the help section below.": "支持以下格式的单文件模式：mzxml、mzml、mzdata 和 netcdf。支持以下格式的压缩文件模式：zip。请参阅下面的帮助部分。",
    "A tabular/csv file is needed and can contains a header line.": "需要一个表格/CSV文件，它可能包含一个标题行。",
    "number of lines not containing values": "不包含值的行数",
    "select the column in your input file.": "选择输入文件中的列。",
    "Please be aware !!! This service of LipidMaps uses Neutral mass -  If you need to transform your masses for the query, please select the ionisation mode (positive or negative) of your instrument- This operation is really pertinent only if yours masses corresponding to [M+H]+ or [M-H]- species": "请注意！！！LipidMaps的这项服务使用的是中性质量 - 如果您需要转换您的查询质量，请选择您仪器的电离模式（正或负）- 这个操作仅在您的质量对应于[M+H]+或[M-H]-物种时才是真正相关的",
    "Posibility to apply oxidation(s) and/or neutral loss(es).": "可能性应用氧化和/或中性损失。",
    "Select one or more boxes": "选择一个或多个框",
    "Choose 'no' to use the following classification on all mz or 'yes' to search each mz on one given cat/class/subclass.": "选择“no”对所有mz进行以下分类，或者选择“yes”对给定的每个mz进行搜索，以确定其类别/子类别。",
    "Example : 3 for Glycerophospholipids category, 310 for Glycerophosphates main class or 32001 for a subclass": "示例：甘油磷脂类别为3，甘油磷酸盐主类为310，或甘油磷酸盐亚类为32001",
    "Number of significant decimal of your mz.": "你的mz的显著小数位数。",
    "Example 801.1789 is truncate as 801.17 or arround as 801.18 with 2 significant decimal.": "示例 801.1789 被截断为 801.17 或者四舍五入为 801.18，保留两位有效小数。",
    "Default value is 0.5 for LM (mass-to-charge ratio)": "默认值为0.5，用于LM（质荷比）",
    "variables ✖ samples": "变量 × 样本",
    "sample metadata, one row per sample": "样本元数据，每个样本一行",
    "variable metadata, one row per variable": "变量元数据，每个变量一行",
    "REQUIRED - The name of the column of sampleMetadata corresponding to the qualitative variable used to define the contrasts.  Except when the 'Univariate Significance-test' is set to 'none', this also must be a portion of the column names in the variableMetadata file.": "必需 - 对照组元数据中用于定义对比的定性变量的列名。除非将“单变量显著性测试”设置为“无”，这还必须是变量元数据文件中列名的一部分。",
    "Either 'none' or the name of the statistical test that was run by the 'Univariate' tool to produce the variableMetadata file.": "“无”或者由“单变量”工具运行的统计检验的名称，以产生变量元数据文件。",
    "Comma-separated level-names (or comma-separated regular expressions to match level-names) to consider in analysis; must match at least two levels; levels must be non-numeric; may include wild cards or regular expressions.  Note that extra space characters will affect results - when 'a,b' is correct, 'a, b' is not equivalent and likely will fail or give different results.": "在分析中考虑的用逗号分隔的级别名称（或用逗号分隔的正则表达式以匹配级别名称）；必须至少匹配两个级别；级别不能是数字；可以包括通配符或正则表达式。请注意，额外的空格字符会影响结果 - 当 'a,b' 是正确的时候，'a, b' 不是等效的，并且可能会失败或给出不同的结果。",
    "How to specify level-names generically. (See help below for details on using wild cards or regular expressions.)": "如何通用地指定级别名称。（有关使用通配符或正则表达式的详细信息，请参见下面的帮助。）",
    "Specify the number of features at each of the loading-extremes that should be labelled (with the name of the feature) on the covariance-vs.-correlation plot; specify 'ALL' to label all features or '0' to label no features; this choice has no effect on the OPLS-DA loadings plot.": "指定在协方差对相关性图中，每个载荷极值处应该标记（以特征名称）的特征数量；指定'ALL'以标记所有特征或'0'以不标记任何特征；此选择对OPLS-DA载荷图没有影响。",
    "What is the minimum number of samples to be used by the ropls package for cross-validation of OPLS-DA predictions?  This should be not more than half the number of your samples.": "ropls包用于OPLS-DA预测交叉验证的最小样本数是多少？这个数目不应超过你样本数的一半。",
    "Choose 'Do not include ...' to hides further choices.": "选择“不包括...”以隐藏更多选项。",
    "Positive ionization-mode: variables ✖ samples": "正离子模式：变量 × 样本",
    "Positive ionization-mode: sample metadata, one row per sample": "正离子模式：样本元数据，每个样本一行",
    "Positive ionization-mode: variable metadata, one row per variable": "正离子模式：可变元数据，每个变量一行",
    "Negative ionization-mode: variables ✖ samples": "负离子化模式：变量×样本",
    "Negative ionization-mode: sample metadata, one row per sample": "负电离模式：样本元数据，每个样本一行",
    "Negative ionization-mode: variable metadata, one row per variable": "负离子模式：变量元数据，每个变量一行",
    "if comparing pool and sample CVs, corresponds to the max ratio tolerated (basically between 1.0 and 1.25) ; else corresponds to the max pool CV tolerated (basically 0.3)": "如果比较池和样本的变异系数（CV），对应于可容忍的最大比率（基本上在1.0到1.25之间）；否则对应于可容忍的最大池变异系数（基本上为0.3）。",
    "Rdata file containing a xset object": "包含 xset 对象的 Rdata 文件",
    "Choose the settings used for finding peaks": "选择用于寻找峰值的设置",
    "RT range to process in minutes, for example 5,25": "RT范围以分钟为单位进行处理，例如5,25",
    "A database file as needed for DB option in runGC": "一个数据库文件，用于 runGC 中 DB 选项所需的文件",
    "A file with two column (rt and RI) with rt in minutes as needed for RI option in runGC": "一个包含两列(rt 和 RI)的文件，其中rt以分钟为单位，这是运行GC所需的RI选项。",
    "The allowed RT shift in minutes between same molecule in different sample": "允许的同一分子在不同样本之间的保留时间（RT）偏移（以分钟为单位）",
    "The minimum number of ion in a mass spectra to consider it a molecule": "在质谱中，考虑其为分子的最小离子数",
    "The minimum similarity allowed between peaks mass spectra to be considered as equal": "允许的峰值质谱之间最小相似度，以被视为相等",
    "The fraction of samples in which a pseudospectrum is present before it is regarded as an unknown": "在将伪谱视为未知之前，样本中存在伪谱的比例",
    "The absolute number of samples in which a pseudospectrum is present before it is regarded as an unknown": "在将其视为未知之前，伪谱存在的样本的绝对数量",
    "RI shift accepted": "RI位移已被接受",
    "Indicate the column name of the sample table to be used as repeated mesurement factor": "指定样本表中要作为重复测量因子的列名",
    "Indicate the column name of the sample table to be used as factor": "指明要用作因子的样本表中的列名",
    "Indicate the column name of the sample table to be used as first factor": "指明要用作第一因子的样本表中的列名",
    "Indicate the column name of the sample table to be used as second factor": "指出要作为第二个因素使用的样本表的列名",
    "must contain at least the three following columns: 'batch' + 'injectionOrder' + 'sampleType'": "必须包含至少以下三个列：“批次” + “注射顺序” + “样本类型”",
    "To select between linear or non-linear (lowess or loess) methods to be used in Van der Kloet algorithm ; when using loess, you can choose to use pools or samples to model batch effect.": "为了在Van der Kloet算法中选择线性或非线性（lowess或loess）方法；当使用loess时，你可以选择使用池或样本来模拟批次效应。",
    "smoothing parameter; must be > 0": "平滑参数；必须大于0",
    "should only be put at 'show' if you have good understanding of the span parameter and are willing to adjust it": "只有在您对span参数有良好的理解并愿意调整它的情况下，才应将其放在'show'上",
    "applied to lowess and loess regression": "应用于 LOWESS 和洛伊斯回归",
    "if 'No' is selected, the value of intensity for each mz is set to 10 (MassBank spectrum search require mz-pcgroup-intensity data - please use massbank peak search if needed)": "如果选择“否”，则每个mz的强度值将设置为10（MassBank 谱图搜索需要mz-pcgroup-intensity数据 - 如有需要，请使用massbank峰值搜索）",
    "Default value is 0.5.": "默认值是0.5。",
    "if 'NO' is selected then defaults values are: instruments = all ; max = 10 ; unit = unit ; tolerance = 0.3 ; cutoff = 50 ; server = JP": "如果选择“否”，则默认值是：仪器 = 全部；最大值 = 10；单位 = 单位；容差 = 0.3；截止值 = 50；服务器 = JP",
    "'0' means unspecified and then all results are obtained.": "'0' 表示未指定，然后获取所有结果。",
    "Default value is 0.3 unit.": "默认值是0.3单位。",
    "Default value is 50 ppm.": "默认值是50 ppm。",
    "Ignore peaks whose intensity is not larger than the value of cutoff.": "忽略强度不大于截止值的峰值。",
    "output file from CAMERA.annotate using a positive polarity mode": "使用正极性模式的CAMERA.annotate生成的输出文件",
    "output file from CAMERA.annotate using a positive negative mode": "使用正负模式从CAMERA.annotate生成的输出文件",
    "[pos] As first step it searches for pseudospectra from the positive and the negative sample within a retention time window": "作为第一步，它在保留时间窗口内搜索来自阳性样本和阴性样本的伪谱。",
    "[ruleset] Matrix of matching rules. By default, the matrix (1,1) would create the M+H/M-H rule, since the first rule of xsa.pos@ruleset and xsa.neg@ruleset is M+H respectively M-H. Only rules with identical charge can be combined!": "[规则集] 匹配规则的矩阵。默认情况下，矩阵（1,1）将创建 M+H/M-H 规则，因为 xsa.pos@ruleset 和 xsa.neg@ruleset 的第一条规则分别是 M+H 和 M-H。只有具有相同电荷的规则才能组合在一起！",
    "Convert the columns rtmed, rtmin and rtmax into minutes": "将rtmed、rtmin和rtmax列转换为分钟",
    "A minimum of 4 decimal places is recommended. Useful to avoid duplicates within identifiers": "建议至少使用4位小数。这有助于避免在标识符中出现重复",
    "Useful to avoid duplicates within identifiers": "有助于避免标识符内的重复",
    "The name of your directory containing all your data": "包含你所有数据的目录的名称",
    "[dmz] shloud be at most 0.002 for high resolution": "[dmz] 应最多为 0.002，以获得高分辨率",
    "[ppmGroup] Should be inferior to the ppm parameter above.": "[ppmGroup] 应低于上面的ppm参数。",
    "[dmzGroup] shloud be at most 0.002 for high resolution": "[dmzGroup] 应最多为 0.002，以获得高分辨率",
    "[fracGroup]": "[分组]",
    "[imputation]": "[插补]",
    "[bandCoverage] Must be between 0 and 1": "[bandCoverage] 必须在0到1之间",
    "[sizeMin] If set to 'none', the half of the estimated injection window will be used": "[sizeMin] 如果设置为 'none'，则将使用估计的注射窗口的一半",
    "[scanMin]": "[扫描最小值]",
    "[scanMax] Set to 'none' to preprocess up to the last acquired scan": "[scanMax] 设置为 'none' 可以预处理到最后获得的扫描",
    "Default value is 0.04 ppm": "默认值是0.04 ppm",
    "Default value is 10 ppm": "默认值是10 ppm",
    "Default value is 0.5 ppm": "默认值是0.5 ppm",
    "Number of the reference spectrum. Default value is 0: automatic determination of the reference spectrum": "参考谱图的数量。默认值是0：自动确定参考谱图",
    "Size of a single small segment after division of spectra. Default value is 64": "单个小段在光谱分割后的尺寸。默认值为64",
    "Threshold to remove all peaks with intensity lower this value. Default value is 1": "移除所有峰值强度低于此阈值的所有峰。默认值是1",
    "Mass spectral library file to convert.": "要转换的质谱库文件。",
    "Apply metadata harmonization, parsing certain keys required by additional filters or initializing other attributes. The default is True.": "应用元数据协调，解析额外过滤器所需的某些键或初始化其他属性。默认值为True。",
    "Output format to convert the spectral library into.": "输出格式，用于将光谱库转换成。",
    "Retain peak comments from individual peaks after the respective mz and intensity pairs when selected.": "在选定后，保留每个峰值对应的 m/z 和强度对后面的峰值注释。",
    "Converts the keys to a specific style - see [1] for details.. One of ['matchms', 'massbank', 'nist', 'riken', 'gnps']. Default is 'matchms'.": "将键转换为特定风格 - 详情见[1]。其中之一为['matchms', 'massbank', 'nist', 'riken', 'gnps']。默认为'matchms'。",
    "Distance measure to use to compute between the molecular fingerprints - see [2] for details regarding the used fingerprint.": "用于计算分子指纹之间距离的度量方法 - 有关使用的指纹的详细信息，请参见[2]。",
    "matchms similarity-scores file.": "matchms 相似性分数 文件。",
    "Metric of the score layer to create a network from.": "用于创建网络的得分层的度量标准。",
    "Unique metadata identifier of each spectrum from which scores are computed.": "每个计算分数的谱图的唯一元数据标识符。",
    "Output format of the similarity network.": "相似性网络的输出格式。",
    "The number of highest-score edges to keep between a pair of nodes. Depending on 'Link method' parameter the top N edges will be selected either from disjoined sets of individual specta or joined subset of scores between the given pair of spectra. This value has to be higher than the 'Maximum number of links'.": "保留每对节点之间最高分的边的数量。根据“链接方法”参数，将从单个谱的不相交集合中或给定的一对谱之间的分数的联合子集中选择前N个边。这个值必须高于“最大链接数”。",
    "Maximum number of links to add per node.": "每个节点要添加的链接的最大数量。",
    "Minimum similarity score value to link two spectra.": "将两个谱图连接的最小相似性分数值。",
    "Method for computing top N edges for each node. If selected 'Single', top N nodes will be selected from disjoined sets of highest scores for nodes of a given pair (i.e., if a given edge belongs to top N scores of at least one node, it is kept). If selected 'Mutual', top N nodes will be selected from a joined subset of a given spectra pair (i.e., if and only if a given edge belongs to top N scores of both nodes, it is kept).": "计算每个节点的前N条边的方法。如果选择'Single'，则将从前N个节点中选择得分最高的不相交集合中的节点（即，如果给定边至少属于一个节点的前N个得分，它将被保留）。如果选择'Mutual'，则将从前N个节点中选择给定谱对的联合子集中的节点（即，只有当给定边同时属于两个节点的前N个得分时，它才被保留）。",
    "If set to No all unconnected nodes will be removed.": "如果设置为否，则所有未连接的节点将被移除。",
    "Similarity metric to use for score computation.": "用于得分计算的相似性度量。",
    "Name of the attribute which will be removed from all spectra records in the MSP.": "在MSP中，将从所有谱图记录中移除的属性名称。",
    "Spectral library (.msp) to split into smaller chunks. Requires keyword 'compound_name', 'compoundname' or 'name' to identify individual spectra using the one-per-file method.": "将谱图库（.msp）分割成更小的块。需要使用关键字'compound_name'、'compoundname'或'name'来使用每个文件单独的方法识别单个谱图。",
    "Method on how to split the library.": "如何分割库的方法。",
    "Number of spectra per output chunks.": "每个输出块的谱图数量。",
    "Number of chunks into which to split the library.": "将库分成的块数。",
    "Default value is 18.": "默认值是18。",
    "Default value is 4.6.": "默认值是4.6。",
    "Default value is 3.0.": "默认值是3.0。",
    "Default value is 13.": "默认值是13。",
    "Retention time units in the query file.": "查询文件中的保留时间单位。",
    "Retention time units in the reference file.": "参考文件中的保留时间单位。",
    "Computation method to use.": "使用的计算方法。",
    "Type of gas chromatographic column for which to set the retention index value": "为设定保留指数值的气相色谱柱类型",
    "@INPUT_IMAGE_HELP@ from: findChromPeaks": "@输入图像帮助@ 来自：findChromPeaks",
    "maximum peak width in seconds.  See CleanPeaks manual (linked in help below).": "最大峰宽（以秒为单位）。请参阅CleanPeaks手册（下面有帮助链接）。",
    "minimal required intensity for a peak to be retained.  Defaults to 0.0.": "峰值保留所需的最小强度。默认值为0.0。",
    "number of data points (per chromatographic peak) that have to be >= threshold. Defaults to 1.": "每个色谱峰必须有大于或等于阈值的数据点数。默认为1。",
    "See the FilterIntensity manual (linked in help below). 'intb' requires centWave processed data": "请参阅 FilterIntensity 手册（下面提供的链接中有帮助信息）。'intb' 需要经过 centWave 处理的数据",
    "seconds of expansion of retention time window (on both sides) to check for overlapping peaks. Defaults to 2.": "保留时间窗口扩展秒数（两侧各扩展）以检查重叠峰。默认为2。",
    "'m/z units' of expansion of m/z window (on both sides) to check for overlapping peaks. Defaults to 0.": "'m/z单位'是扩展m/z窗口（在两侧）以检查重叠峰的单位。默认值为0。",
    "'relative 'm/z units' (in parts per million) of expansion of m/z window (on both sides) to check for overlapping peaks. Defaults to 10.": "“相对'm/z单位'（以百万分之一为单位）的m/z窗口扩展（在两侧）以检查重叠峰。默认值为10。”",
    "See MergeNeighboringPeaks manual (linked in help below).  Minimum proportion midway between peaks relative to the smaller peak's 'maxo' (maximal intensity at peak apex).  Defaults to 0.75.": "参见MergeNeighboringPeaks手册（下面有链接帮助）。在峰值之间中点处的最小比例相对于较小峰值的'maxo'（峰顶处的最大强度）。默认为0.75。",
    "the MS level on which refinement should be performed. Defaults to 1.": "在质谱级别上进行细化。默认为1。",
    "Mass spectrometry profile data.": "质谱分析剖面数据。",
    "Mass spectrometry sample-wise features table.": "质谱样品特征表。",
    "Mass spectrometry files containing feature samples.": "质谱文件包含特征样本。",
    "Mass spectrometry file containing corrected feature samples.": "质谱文件包含校正的特征样本。",
    "Peak metadata table from the align features step.": "从“比对特征”步骤生成的峰值元数据表。",
    "Table with retention times for features (rows) across samples (columns).": "特征（行）在样本（列）中的保留时间表。",
    "Table with intensities for features (rows) across samples (columns).": "特征（行）在样本（列）上的强度表。",
    "Mass spectrometry clustered features table.": "质谱聚类特征表。",
    "Template features table.": "模板特征表。",
    "List of tables containing features.": "包含特征的表格列表。",
    "Sample data in .mzml format acquired in profile mode.": "以轮廓模式采集的.mzml格式的样本数据。",
    "Peak metadata table from the align features step": "从“对齐特征”步骤生成的峰值元数据表",
    "A data table containing the known metabolite ions and previously found features. The table must contain these 18 columns: chemical_formula (optional), HMDB_ID (optional), KEGG_compound_ID (optional), neutral.mass (optional), ion.type (the ion form - optional), m.z (either theoretical or mean observed m/z value of previously found features), Number_profiles_processed (the total number of processed samples to build this database), Percent_found (the percentage of historically processed samples in which the feature appeared), mz_min (minimum observed m/z value), mz_max (maximum observed m/z value), RT_mean (mean observed retention time), RT_sd (standard deviation of observed retention time), RT_min (minimum observed retention time), RT_max (maximum observed retention time), int_mean.log. (mean observed log intensity), int_sd.log. (standard deviation of observed log intensity), int_min.log. (minimum observed log intensity), int_max.log. (maximum observed log intensity).": "一个包含已知代谢物离子和先前发现的特征的数据表。该表必须包含这18列：化学式（可选）, HMDB_ID（可选）, KEGG_化合物ID（可选）, 中性质量（可选）, 离子类型（离子形式 - 可选）, m.z（先前发现的特征的理论或平均观察到的m/z值）, 处理的轮廓数（构建此数据库所用的已处理样本的总数）, 发现百分比（历史上处理的样本中出现该特征的百分比）, mz_min（观察到的最小m/z值）, mz_max（观察到的最大m/z值）, RT_mean（观察到的平均保留时间）, RT_sd（观察到的保留时间的标准差）, RT_min（观察到的最小保留时间）, RT_max（观察到的最大保留时间）, int_mean.log.（观察到的平均对数强度）, int_sd.log.（观察到的对数强度的标准差）, int_min.log.（观察到的最小对数强度）, int_max.log.（观察到的最大对数强度）。",
    "Choose between merging feature to known_table and known_table to features.": "选择将特征合并到已知表格中，或者将已知表格合并到特征中。",
    "The minimum number of occurrences of a historically unseen (unknown) feature to add this feature into the database of known features.": "将一个历史上未见过（未知）的特征添加到已知特征数据库中的最小出现次数。",
    "The ppm tolerance to match identified features to known metabolites/features.": "匹配已识别特征与已知代谢物/特征的ppm容忍度。",
    "Table containing clustered features.": "包含聚类特征的表格。",
    "List of tables containing clustered features.": "包含聚类特征的表格列表。",
    "A feature has to show up in at least this number of profiles to be included in the final result.": "一个特征必须至少出现在这么多的配置文件中，才能被包含在最终结果中。",
    "Preserve a log with details about the annotation process.": "保留一个包含注释过程详细信息的日志。",
    "Severity of log messages  present in the log file.": "日志文件中日志消息的严重程度。",
    "Retention time similarity (optional). A recommended starting point is half the value of your average chromatographic peak width at half max (seconds)).": "保留时间相似性（可选）。推荐的起始点是您色谱峰半高宽平均值的一半（秒）。",
    "Retention time similarity. A recommended starting point is half the value of your average chromatographic peak width at half max (seconds)).": "保留时间相似性。推荐的起始点是您色谱峰半高宽平均值的一半（秒）。",
    "Query mass spectra to match against references.": "查询质谱数据以匹配参考数据。",
    "Reference mass spectra to match against as library.": "参考质谱图谱以匹配作为库。",
    "Matrix type for storing scores objects. Sparse type more memory-efficient and better for large arrays. Note that whatever is selected the output might still be sprase array if scores have too much 0-entries.": "用于存储分数对象的矩阵类型。稀疏类型更节省内存，更适合大型数组。请注意，无论选择什么，如果分数有太多的0条目，输出可能仍然是稀疏数组。",
    "Model JSON file to use for Spec2Vec similarity computing.": "用于Spec2Vec相似性计算的模型JSON文件。",
    "Model NPY file to use for Spec2Vec similarity computing.": "用于Spec2Vec相似性计算的模型NPY文件。",
    "Spectrum vectors are a weighted sum of the word vectors. The given word intensities will be raised to the given power. The default is 0, which means that no weighing will be done.": "谱向量是词向量的加权和。给定的词强度将被提升到给定的幂。默认值为0，这意味着不会进行加权。",
    "Maximum allowed share of the peaks that are new to the model in relation to the whole peak corpus.": "模型中新出现的峰值与整个峰值语料库相比所允许的最大共享比例。",
    "Peaks will be considered a match when less than tolerance apart.": "峰值在小于容差范围内时将被视为匹配。",
    "Swap sample names with feature names as column headers (to fit recetox-aplcms outputs).": "将样本名称与特征名称互换作为列标题（以适应recetox-aplcms输出）。",
    "'multiple batches' takes into account inter- and intrabatch intensity drift; 'single batch' relies only on the injection order of the samples and requires no batch information [2]": "“多批次”考虑了批次间和批次内的强度漂移；“单批次”仅依赖于样本的注射顺序，并且不需要批次信息[2]",
    "Peaks will be considered a match when less than tolerance apart. Absolute m/z value, not in ppm.": "峰值在小于容差范围内时将被视为匹配。使用绝对m/z值，而不是ppm值。",
    "The power to raise mz to in the cosine function.": "在余弦函数中将 m/z 提升到的幂。",
    "The power to raise intensity to in the cosine function.": "在余弦函数中提高强度的幂。",
    "Name of the column of the sample metadata table corresponding to the qualitative or quantitative variable": "样本元数据表中对应定性或定量变量的列名",
    "[required]  The demultiplexed sequences to be denoised.": "[必需] 需要去噪的去多重序列。",
    "[required]  Sequence trim length, specify -1 to disable trimming.": "[必需] 序列修剪长度，指定 -1 以禁用修剪。",
    "[default: 0]  Sequence trimming from the 5' end. A value of 0 will disable this trim.": "[默认值：0]  从5'端修剪序列。值为0将禁用此修剪。",
    "[default: No]  If true, gather stats per sample.": "[默认：否] 如果为真，将为每个样本收集统计信息。",
    "[default: 0.005]  The mean per nucleotide error, used for original sequence estimate.": "[默认：0.005] 每个核苷酸的平均错误，用于原始序列估计。",
    "[default: 0.01]  Insertion/deletion (indel) probability (same for N indels).": "[默认值：0.01] 插入/删除（indel）概率（对N个插入/删除相同）。",
    "[default: 3]  Maximum number of insertion/deletions.": "[默认：3] 最大插入/删除次数。",
    "[default: 10]  Retain only features appearing at least min_reads times across all samples in the resulting feature table.": "[默认值：10] 仅保留在所有样本中至少出现min_reads次的特征，以生成特征表。",
    "[default: 2]  In each sample, discard all features with an abundance less than min_size.": "[默认：2] 在每个样本中，丢弃所有丰度小于min_size的特征。",
    "[default: 1]  Number of jobs to start (if to run in parallel).": "[默认: 1]  要启动的作业数量（如果要并行运行）。",
    "[default: Yes]  If true, hash the feature IDs.": "[默认：是] 如果为真，哈希特征ID。",
    "[required]  Sample metadata file containing individual_id_column.": "[必需] 包含个体ID列的样本元数据文件。",
    "[required]  Metadata column containing state (time) variable information.": "[必需] 包含状态（时间）变量信息的元数据列。",
    "[required]  Metadata column containing IDs for individual subjects.": "[必需] 包含个体主题ID的元数据列。",
    "[required]  Numerical metadata or artifact column to test.": "[必需] 用于测试的数值元数据或工件列。",
    "[optional]  Feature table to optionally use for computing first differences.": "[可选] 用于计算首次差异的可选特征表。",
    "[optional]  A value listed in the state_column metadata column against which all other states should be compared. Toggles calculation of static differences instead of first differences (which are calculated if no value is given for baseline). If a \"baseline\" value is provided, sample differences at each state are compared against the baseline state, instead of the previous state. Must be a value listed in the state_column.": "[可选] 在 state_column 元数据列中列出的一个值，应将所有其他状态与之进行比较。切换计算静态差异而不是首次差异（如果没有为基线提供值，则计算首次差异）。如果提供了“基线”值，则将每个状态的样本差异与基线状态进行比较，而不是与前一个状态进行比较。必须是在 state_column 中列出的值。",
    "[required]  The sequences to be aligned.": "[必需] 需要对齐的序列。",
    "[default: 1]  The number of threads. (Use `auto` to automatically use all available cores)": "[默认值：1] 线程数。（使用`auto`自动使用所有可用的核）",
    "[default: No]  This flag is required if the number of sequences being aligned are larger than 1000000. Disabled by default": "[默认：否] 如果正在比对的序列数量大于100万，则需要此标志。默认情况下为禁用。",
    "[required]  The alignment to which sequences should be added.": "[必需] 需要添加序列的比对。",
    "[required]  The sequences to be added.": "[必需] 要添加的序列。",
    "[default: No]  Optimize for the addition of short sequence fragments (for example, primer or amplicon sequences). If not set, default sequence addition is used.": "[默认值：否] 优化添加短序列片段（例如引物或扩增子序列）。如果未设置，则使用默认序列添加。",
    "[required]  The alignment to be masked.": "[必需] 需要被掩蔽的比对。",
    "[default: 1.0]  The maximum relative frequency of gap characters in a column for the column to be retained. This relative frequency must be a number between 0.0 and 1.0 (inclusive), where 0.0 retains only those columns without gap characters, and 1.0 retains all columns regardless of gap character frequency.": "[默认：1.0] 保留列中间隔字符的最大相对频率。这个相对频率必须是一个介于0.0和1.0（包括）之间的数字，其中0.0只保留没有间隔字符的列，而1.0则不管间隔字符的频率如何都保留所有列。",
    "[default: 0.4]  The minimum relative frequency of at least one non-gap character in a column for that column to be retained. This relative frequency must be a number between 0.0 and 1.0 (inclusive). For example, if a value of 0.4 is provided, a column will only be retained if it contains at least one character that is present in at least 40% of the sequences.": "[默认值：0.4] 一个列中至少一个非空字符的最小相对频率，以便保留该列。这个相对频率必须是一个介于0.0到1.0之间的数字（包括0.0和1.0）。例如，如果提供了0.4的值，只有当一个列至少包含一个在至少40%的序列中出现的字符时，该列才会被保留。",
    "[required]  The feature table to which pseudocounts should be added.": "[必需] 需要添加伪计数的特征表。",
    "[default: 1]  The value to add to all counts in the feature table.": "[默认: 1] 在特征表中，要添加到所有计数的值。",
    "[required]  The feature table to be used for ANCOM computation.": "[必需] 用于ANCOM计算的特征表。",
    "[required]  The categorical sample metadata column to test for differential abundance across.": "[必需] 用于测试差异丰度的分类样本元数据列。",
    "[default: No]  If True, samples with missing metadata values will be filtered from the table prior to analysis. If False, an error will be raised if there are any missing metadata values.": "[默认值：否] 如果为真，将在分析之前从表中过滤掉具有缺失元数据值的样本。如果为假，如果有任何缺失的元数据值，将引发错误。",
    "[required]  The feature table to be used for ANCOM-BC computation.": "[必需] 用于ANCOM-BC计算的特征表。",
    "[required]  The sample metadata.": "[必需] 样本元数据。",
    "[required]  How the microbial absolute abundances for each taxon depend on the variables within the `metadata`.": "[必需] 微生物各分类单元的绝对丰度如何依赖于 `metadata` 中的变量。",
    "[default: 0.1]  A numerical fraction between 0-1. Taxa with prevalences less than this value will be excluded from the analysis.": "[默认值：0.1] 一个介于0到1之间的数值分数。低于此值的物种丰度将被排除在分析之外。",
    "[default: 0]  A numerical threshold for filtering samples based on library sizes. Samples with library sizes less than this value will be excluded from the analysis.": "[默认：0] 一个用于基于库大小过滤样本的数值阈值。小于此值的库大小的样本将被排除在分析之外。",
    "[optional]  Define the reference level(s) to be used for categorical columns found in the `formula`. These categorical factors are dummy coded relative to the reference(s) provided. The syntax is as follows: \"column_name::column_value\"": "[可选] 定义在 `formula` 中发现的分类列所使用的参考水平。这些分类因子相对于提供的参考（或多个参考）进行虚拟编码。语法如下：\"列名::列值\"",
    "[required]": "[必需]",
    "[default: No]  Whether to classify a taxon as a structural zero using its asymptotic lower bound.": "[默认值：否] 是否使用渐近下界将一个分类群分类为结构性零。",
    "[default: 1e-05]  The iteration convergence tolerance for the E-M algorithm.": "[默认值：1e-05] 期望最大化（E-M）算法的迭代收敛容差。",
    "[default: 100]  The maximum number of iterations for the E-M algorithm.": "[默认值：100] E-M算法的最大迭代次数。",
    "[default: No]  Whether to use a conservative variance estimator for the test statistic. It is recommended if the sample size is small and/or the number of differentially abundant taxa is believed to be large.": "[默认值：否] 是否使用保守的方差估计器来计算检验统计量。如果样本量较小和/或认为差异丰富的分类群数量很大，建议使用。",
    "[default: 0.05]  Level of significance.": "[默认：0.05] 显著性水平。",
    "[required]  The ANCOM-BC output to be tabulated.": "[必需] 需要被制表的ANCOM-BC输出。",
    "[required]  The paired-end sequences to be demultiplexed.": "[必需] 需要去多重性的成对序列。",
    "[required]  The sample metadata column listing the per-sample barcodes for the forward reads.": "[必需] 列出每个样本正向读取的条形码的样本元数据列。",
    "[optional]  The sample metadata column listing the per-sample barcodes for the reverse reads.": "[可选] 列出每个样本反向读取的条形码的样本元数据列。",
    "[default: 0.1]  The level of error tolerance, specified as the maximum allowable error rate.": "[默认值：0.1] 错误容忍度，指定为最大允许的错误率。",
    "[default: 0]  The number of samples cutadapt demultiplexes concurrently. Demultiplexing in smaller batches will yield the same result with marginal speed loss, and may solve \"too many files\" errors related to sample quantity. Set to \"0\" to process all samples at once.": "[默认值：0] 同时去多路复用的样品数量。将去多路复用分成更小的批次进行，将得到相同的结果，但速度略有损失，并且可能解决与样品数量相关的“文件太多”的错误。设置为“0”以一次性处理所有样品。",
    "[default: 1]  Discard reads shorter than specified value. Note, the cutadapt default of 0 has been overridden, because that value produces empty sequence records.": "[默认值：1]  丢弃短于指定值的读取。请注意，cutadapt默认值0已被覆盖，因为该值会产生空序列记录。",
    "[default: No]  Handle demultiplexing of mixed orientation reads (i.e. when forward and reverse reads coexist in the same file).": "[默认：否] 处理混合方向的读取（即正向和反向读取共存于同一文件中）。",
    "[default: 1]  Number of CPU cores to use.": "[默认值：1] 要使用的CPU核心数。",
    "[required]  The single-end sequences to be demultiplexed.": "[必需] 需要进行去复用的单端序列。",
    "[required]  The sample metadata column listing the per-sample barcodes.": "[必需] 列出每个样本条形码的样本元数据列。",
    "[default: 0.1]  The level of error tolerance, specified as the maximum allowable error rate. The default value specified by cutadapt is 0.1 (=10%), which is greater than `demux emp-*`, which is 0.0 (=0%).": "[默认值：0.1] 错误容忍度，指定为最大允许的错误率。cutadapt指定的默认值为0.1（=10%），这比`demux emp-*`的0.0（=0%）要大。",
    "[required]  The paired-end sequences to be trimmed.": "[必需] 需要修剪的配对端序列。",
    "[optional]  Sequence of an adapter ligated to the 3' end. The adapter and any subsequent bases are trimmed. If a `$` is appended, the adapter is only found if it is at the end of the read. Search in forward read. If your sequence of interest is \"framed\" by a 5' and a 3' adapter, use this parameter to define a \"linked\" primer - see https://cutadapt.readthedocs.io for complete details.": "[可选] 连接到3'端的接头序列。将修剪接头及之后的所有碱基。如果附加了`$`，则仅当接头位于读取的末端时才查找它。在正向读取中搜索。如果您感兴趣的序列由5'和3'接头“框架”，请使用此参数定义“链接”的引物 - 有关完整详细信息，请参见 https://cutadapt.readthedocs.io。",
    "[optional]  Sequence of an adapter ligated to the 5' end. The adapter and any preceding bases are trimmed. Partial matches at the 5' end are allowed. If a `^` character is prepended, the adapter is only found if it is at the beginning of the read. Search in forward read.": "[可选] 与5'端连接的接头序列。将修剪接头及其前面的任何碱基。允许5'端的部分匹配。如果前面加上了`^`字符，则只有在接头位于读取的开头时才查找接头。在正向读取中搜索。",
    "[optional]  Sequence of an adapter that may be ligated to the 5' or 3' end. Both types of matches as described under `adapter` and `front` are allowed. If the first base of the read is part of the match, the behavior is as with `front`, otherwise as with `adapter`. This option is mostly for rescuing failed library preparations - do not use if you know which end your adapter was ligated to. Search in forward read.": "[可选] 可能被连接到5'或3'端的接头序列。允许使用在`adapter`和`front`下描述的两种匹配类型。如果读取的第一个碱基是匹配的一部分，则行为与`front`相同，否则与`adapter`相同。这个选项主要用于挽救失败的文库制备 - 如果你知道接头连接到了哪个端，请不要使用。在正向读取中搜索。",
    "[optional]  Sequence of an adapter ligated to the 3' end. The adapter and any subsequent bases are trimmed. If a `$` is appended, the adapter is only found if it is at the end of the read. Search in reverse read. If your sequence of interest is \"framed\" by a 5' and a 3' adapter, use this parameter to define a \"linked\" primer - see https://cutadapt.readthedocs.io for complete details.": "[可选] 连接到3'端的接头序列。接头及其后的任何碱基都将被修剪。如果附加了一个`$`，则只有在接头位于读取的末端时才会发现它。在反向读取中搜索。如果您感兴趣的序列由5'和3'接头“框架”，请使用此参数定义“链接”引物 - 有关完整详细信息，请参见 https://cutadapt.readthedocs.io。",
    "[optional]  Sequence of an adapter ligated to the 5' end. The adapter and any preceding bases are trimmed. Partial matches at the 5' end are allowed. If a `^` character is prepended, the adapter is only found if it is at the beginning of the read. Search in reverse read.": "[可选] 连接到5'端的接头序列。将修剪接头及其前面的任何碱基。允许5'端的部分匹配。如果前面加上`^`字符，则只有在接头位于读取的开头时才查找接头。在反向读取中搜索。",
    "[optional]  Sequence of an adapter that may be ligated to the 5' or 3' end. Both types of matches as described under `adapter` and `front` are allowed. If the first base of the read is part of the match, the behavior is as with `front`, otherwise as with `adapter`. This option is mostly for rescuing failed library preparations - do not use if you know which end your adapter was ligated to. Search in reverse read.": "[可选] 可能被连接到5'或3'端的接头序列。允许使用在`adapter`和`front`下描述的两种匹配类型。如果读取的第一个碱基是匹配的一部分，则行为与`front`相同，否则与`adapter`相同。此选项主要用于挽救失败的文库制备 - 如果你知道接头连接到了哪个端，请不要使用。在反向读取中搜索。",
    "[default: 0.1]  Maximum allowed error rate.": "[默认：0.1] 允许的最大错误率。",
    "[default: Yes]  Allow insertions or deletions of bases when matching adapters.": "[默认值：是] 在匹配适配器时允许插入或删除碱基。",
    "[default: 1]  Remove multiple occurrences of an adapter if it is repeated, up to `times` times.": "[默认值：1] 如果适配器重复出现，则移除适配器的多次出现，最多移除`times`次。",
    "[default: 3]  Require at least `overlap` bases of overlap between read and adapter for an adapter to be found.": "[默认：3] 要求在读取和适配器之间至少有`overlap`个碱基的重叠，才能找到适配器。",
    "[default: No]  Interpret IUPAC wildcards (e.g., N) in reads.": "[默认：否] 在读取中解释IUPAC通配符（例如，N）。",
    "[default: Yes]  Interpret IUPAC wildcards (e.g., N) in adapters.": "[默认：是] 解释适配器中的IUPAC通配符（例如，N）。",
    "[default: No]  Discard reads in which no adapter was found.": "[默认值：否]  丢弃未发现接头的读段。",
    "[optional]  Discard reads that exceed maximum expected erroneous nucleotides.": "[可选] 丢弃超过预期最大错误核苷酸数的读取。",
    "[optional]  Discard reads with more than COUNT N bases. If COUNT_or_FRACTION is a number between 0 and 1, it is interpreted as a fraction of the read length.": "[可选] 丢弃具有超过 COUNT N 个碱基的读取。如果 COUNT_or_FRACTION 是一个介于 0 和 1 之间的数字，它被解释为读取长度的分数。",
    "[default: 0]  Trim nucleotides with Phred score quality lower than threshold from 5 prime end.": "[默认值：0]  从5'端修剪Phred评分质量低于阈值的核苷酸。",
    "[default: 0]  Trim nucleotides with Phred score quality lower than threshold from 3 prime end.": "[默认：0]  从3'端修剪Phred得分质量低于阈值的核苷酸。",
    "[default: 33]  How the Phred score is encoded (33 or 64).": "[默认：33] Phred分数是如何编码的（33或64）。",
    "[required]  The single-end sequences to be trimmed.": "[必需] 需要进行修剪的单端序列。",
    "[optional]  Sequence of an adapter ligated to the 3' end. The adapter and any subsequent bases are trimmed. If a `$` is appended, the adapter is only found if it is at the end of the read. If your sequence of interest is \"framed\" by a 5' and a 3' adapter, use this parameter to define a \"linked\" primer - see https://cutadapt.readthedocs.io for complete details.": "[可选] 连接到3'端的接头序列。接头和随后的任何碱基都会被修剪掉。如果附加了一个`$`，只有在接头位于读取的末端时才会找到它。如果你感兴趣的序列由5'和3'接头“框架”，请使用此参数定义一个“链接”的引物 - 详见 https://cutadapt.readthedocs.io 获取完整详细信息。",
    "[optional]  Sequence of an adapter ligated to the 5' end. The adapter and any preceding bases are trimmed. Partial matches at the 5' end are allowed. If a `^` character is prepended, the adapter is only found if it is at the beginning of the read.": "[可选] 连接到5'端的接头序列。将修剪接头及其前面的任何碱基。允许5'端的部分匹配。如果前面加上`^`字符，则只有在接头位于读取的开头时才找到它。",
    "[optional]  Sequence of an adapter that may be ligated to the 5' or 3' end. Both types of matches as described under `adapter` and `front` are allowed. If the first base of the read is part of the match, the behavior is as with `front`, otherwise as with `adapter`. This option is mostly for rescuing failed library preparations - do not use if you know which end your adapter was ligated to.": "[可选] 可能被连接到5'或3'端的接头序列。允许使用在`adapter`和`front`下描述的两种匹配类型。如果读取的第一个碱基是匹配的一部分，则行为与`front`相同，否则与`adapter`相同。这个选项主要用于挽救失败的文库制备——如果你知道你的接头连接到了哪个端，请不要使用。",
    "[required]  The single-end demultiplexed PacBio CCS sequences to be denoised.": "[必需] 需要去噪的单端去复用的PacBio CCS序列。",
    "[required]  Sequence of an adapter ligated to the 5' end. The adapter and any preceding bases are trimmed. Can contain IUPAC ambiguous nucleotide codes. Note, primer direction is 5' to 3'. Primers are removed before trim and filter step. Reads that do not contain the primer are discarded. Each read is re-oriented if the reverse complement of the read is a better match to the provided primer sequence. This is recommended for PacBio CCS reads, which come in a random mix of forward and reverse-complement orientations.": "[必需] 连接到5'端的接头序列。将修剪接头及其前面的任何碱基。可以包含IUPAC模糊核苷酸代码。注意，引物方向是5'到3'。在修剪和过滤步骤之前移除引物。不包含引物的读取被丢弃。如果读取的反向互补是与提供的引物序列更好的匹配，则重新定向每个读取。这对于以正向和反向互补方向随机混合出现的PacBio CCS读取是推荐的。",
    "[required]  Sequence of an adapter ligated to the 3' end. The adapter and any preceding bases are trimmed. Can contain IUPAC ambiguous nucleotide codes. Note, primer direction is 5' to 3'. Primers are removed before trim and filter step. Reads that do not contain the primer are discarded.": "[必需] 连接到3'端的接头序列。将修剪掉接头及其前面的任何碱基。可以包含IUPAC模糊核苷酸代码。注意，引物方向是从5'到3'。在修剪和过滤步骤之前去除引物。不包含引物的读段将被丢弃。",
    "[default: 2]  The number of mismatches to tolerate when matching reads to primer sequences - see http://benjjneb.github.io/dada2/ for complete details.": "[默认值：2] 在将读取与引物序列匹配时可以容忍的不匹配数量 - 有关完整详细信息，请参见 http://benjjneb.github.io/dada2/。",
    "[default: No]  Allow insertions or deletions of bases when matching adapters. Note that primer matching can be significantly slower, currently about 4x slower": "[默认值：否] 允许在匹配适配器时插入或删除碱基。请注意，引物匹配可能会显著变慢，目前大约慢4倍。",
    "[default: 0]  Position at which sequences should be truncated due to decrease in quality. This truncates the 3' end of the of the input sequences, which will be the bases that were sequenced in the last cycles. Reads that are shorter than this value will be discarded. If 0 is provided, no truncation or length filtering will be performed. Note: Since Pacbio CCS sequences were normally with very high quality scores, there is no need to truncate the Pacbio CCS sequences.": "[默认值：0] 由于质量下降而应截断序列的位置。这将截断输入序列的3'端，即最后循环测序的碱基。短于该值的读段将被丢弃。如果提供0，则不执行截断或长度过滤。注意：由于Pacbio CCS序列通常具有非常高的质量分数，因此无需截断Pacbio CCS序列。",
    "[default: 0]  Position at which sequences should be trimmed due to low quality. This trims the 5' end of the of the input sequences, which will be the bases that were sequenced in the first cycles.": "[默认：0] 由于质量低，应该修剪序列的位置。这会修剪输入序列的5'端，这些将是序列的第一个周期中测序的碱基。",
    "[default: 2.0]  Reads with number of expected errors higher than this value will be discarded.": "[默认值：2.0] 预期错误数高于该值的读取将被丢弃。",
    "[default: 2]  Reads are truncated at the first instance of a quality score less than or equal to this value. If the resulting read is then shorter than `trunc_len`, it is discarded.": "[默认值：2] 当读取的质量分数小于或等于这个值时，读取会被截断。如果截断后的读取长度小于 `trunc_len`，则该读取会被丢弃。",
    "[default: 20]  Remove reads with length less than minLen. minLen is enforced after trimming and truncation. For 16S Pacbio CCS, suggest 1000.": "[默认值：20] 移除长度小于minLen的读段。在修剪和截断后强制执行minLen。对于16S Pacbio CCS，建议使用1000。",
    "[default: 0]  Remove reads prior to trimming or truncation which are longer than this value. If 0 is provided no reads will be removed based on length. For 16S Pacbio CCS, suggest 1600.": "[默认：0] 在进行修剪或截断之前移除长度超过此值的读取。如果提供0，则不会基于长度移除任何读取。对于16S Pacbio CCS，建议使用1600。",
    "[default: 3.5]  The minimum abundance of potential parents of a sequence being tested as chimeric, expressed as a fold-change versus the abundance of the sequence being tested. Values should be greater than or equal to 1 (i.e. parents should be more abundant than the sequence being tested). Suggest 3.5. This parameter has no effect if chimera_method is \"none\".": "[默认值：3.5] 被测试为嵌合体的序列的潜在亲本的最小丰度，表示为与被测试序列丰度的倍数变化。值应大于或等于1（即亲本应比被测试的序列更丰富）。建议使用3.5。如果chimera_method是“none”，则此参数无效。",
    "[default: No]  Bimeras that are one-off from exact are also identified if the `allow_one_off` argument is True. If True, a sequence will be identified as bimera if it is one mismatch or indel away from an exact bimera.": "[默认值：否] 如果 `allow_one_off` 参数为 True，则还会识别与精确双峰值只有一个差异的双峰值。如果为 True，则如果序列与精确双峰值有一个不匹配或插入/删除差异，则会将其识别为双峰值。",
    "[default: 1]  The number of threads to use for multithreaded processing. If 0 is provided, all available cores will be used.": "[默认：1] 用于多线程处理的线程数。如果提供0，则将使用所有可用的核心。",
    "[default: 1000000]  The number of reads to use when training the error model. Smaller numbers will result in a shorter run time but a less reliable error model.": "[默认：1000000] 在训练错误模型时使用的读取数。较小的数字将导致运行时间缩短，但错误模型的可靠性较低。",
    "[default: Yes]  If true, the feature ids in the resulting table will be presented as hashes of the sequences defining each feature. The hash will always be the same for the same sequence so this allows feature tables to be merged across runs of this method. You should only merge tables if the exact same parameters are used for each run.": "[默认: 是] 如果为真，结果表中的特征ID将作为定义每个特征的序列的哈希值呈现。对于相同的序列，哈希值将始终相同，因此这允许特征表在这种方法的运行之间合并。只有在每次运行使用完全相同的参数时，您才应该合并表。",
    "[required]  The paired-end demultiplexed sequences to be denoised.": "[必需] 需要去噪的配对端去复用序列。",
    "[required]  Position at which forward read sequences should be truncated due to decrease in quality. This truncates the 3' end of the of the input sequences, which will be the bases that were sequenced in the last cycles. Reads that are shorter than this value will be discarded. After this parameter is applied there must still be at least a 12 nucleotide overlap between the forward and reverse reads. If 0 is provided, no truncation or length filtering will be performed": "[必需] 由于质量下降而应截断前向读取序列的位置。这将截断输入序列的3'端，这些序列是在最后几个循环中测序的碱基。短于该值的读取将被丢弃。在此参数应用后，前向和反向读取之间仍必须至少有12个核苷酸的重叠。如果提供0，则不执行截断或长度过滤。",
    "[required]  Position at which reverse read sequences should be truncated due to decrease in quality. This truncates the 3' end of the of the input sequences, which will be the bases that were sequenced in the last cycles. Reads that are shorter than this value will be discarded. After this parameter is applied there must still be at least a 12 nucleotide overlap between the forward and reverse reads. If 0 is provided, no truncation or length filtering will be performed": "[必需] 由于质量下降，反向读取序列应在哪个位置截断。这将截断输入序列的3'端，即最后循环测序的碱基。短于该值的读取将被丢弃。在此参数应用后，正向和反向读取之间仍必须至少有12个核苷酸的重叠。如果提供0，则不执行截断或长度过滤。",
    "[default: 0]  Position at which forward read sequences should be trimmed due to low quality. This trims the 5' end of the input sequences, which will be the bases that were sequenced in the first cycles.": "[默认：0] 由于质量较低，应该修剪正向读取序列的位置。这将修剪输入序列的5'端，这些将是首先进行测序的碱基。",
    "[default: 0]  Position at which reverse read sequences should be trimmed due to low quality. This trims the 5' end of the input sequences, which will be the bases that were sequenced in the first cycles.": "[默认：0] 由于质量较低，应该在哪个位置修剪反向读取序列。这将修剪输入序列的5'端，这些将是首先进行测序的碱基。",
    "[default: 2.0]  Forward reads with number of expected errors higher than this value will be discarded.": "[默认值：2.0] 预期错误数高于该值的正向读取将被丢弃。",
    "[default: 2.0]  Reverse reads with number of expected errors higher than this value will be discarded.": "[默认值：2.0] 预期错误数高于该值的反向读取将被丢弃。",
    "[default: 2]  Reads are truncated at the first instance of a quality score less than or equal to this value. If the resulting read is then shorter than `trunc_len_f` or `trunc_len_r` (depending on the direction of the read) it is discarded.": "[默认值：2] 当读取的质量分数小于或等于这个值时，读取会被截断。如果截断后的读取长度小于 `trunc_len_f` 或 `trunc_len_r`（取决于读取的方向），则该读取会被丢弃。",
    "[default: 12]  The minimum length of the overlap required for merging the forward and reverse reads.": "[默认：12] 合并正向和反向读取所需的最小重叠长度。",
    "[default: 1.0]  The minimum abundance of potential parents of a sequence being tested as chimeric, expressed as a fold-change versus the abundance of the sequence being tested. Values should be greater than or equal to 1 (i.e. parents should be more abundant than the sequence being tested). This parameter has no effect if chimera_method is \"none\".": "[默认值：1.0] 正在测试的嵌合序列潜在亲本的最小丰度，表示为与正在测试序列的丰度的倍数变化。值应大于或等于1（即亲本应比正在测试的序列更丰富）。如果chimera_method是“none”，则此参数无效。",
    "[default: No]  Bimeras that are one-off from exact are also identified if the `allow_one_off` argument is TrueIf True, a sequence will be identified as bimera if it is one mismatch or indel away from an exact bimera.": "[默认值：否] 如果 `allow_one_off` 参数为 True，则也会识别出与精确双聚体只有一个差异的双聚体。如果为 True，则如果序列与精确双聚体有一个错配或插入/删除差异，它将被识别为双聚体。",
    "[required]  The single-end demultiplexed pyrosequencing sequences (e.g. 454, IonTorrent) to be denoised.": "[必需] 需要去噪的单端去复用焦磷酸测序序列（例如 454, IonTorrent）。",
    "[required]  Position at which sequences should be truncated due to decrease in quality. This truncates the 3' end of the of the input sequences, which will be the bases that were sequenced in the last cycles. Reads that are shorter than this value will be discarded. If 0 is provided, no truncation or length filtering will be performed": "[必需] 由于质量下降而应截断序列的位置。这将截断输入序列的3'端，即最后几个循环测序的碱基。短于该值的读段将被丢弃。如果提供0，则不执行截断或长度过滤。",
    "[default: 0]  Remove reads prior to trimming or truncation which are longer than this value. If 0 is provided no reads will be removed based on length.": "[默认值：0] 在进行修剪或截断之前移除比这个值还长的reads。如果提供0，则不会基于长度移除任何reads。",
    "[default: 250000]  The number of reads to use when training the error model. Smaller numbers will result in a shorter run time but a less reliable error model.": "[默认值：250000] 在训练错误模型时使用的读段数。较小的数字将导致运行时间更短，但错误模型的可靠性较低。",
    "[required]  The single-end demultiplexed sequences to be denoised.": "[必需] 需要去噪的单端去复用序列。",
    "[default: No]  Bimeras that are one-off from exact are also identified if the `allow_one_off` argument is True.If True, a sequence will be identified as bimera if it is one mismatch or indel away from an exact bimera.": "[默认值：否] 如果 `allow_one_off` 参数为 True，则也会识别与精确双嵌合体相差一个的双嵌合体。如果为 True，则如果序列与精确双嵌合体相差一个错配或插入/删除，则会将其识别为双嵌合体。",
    "[required]  Positive filtering database. Keep all sequences aligning to these sequences.": "[必需] 正向过滤数据库。保留所有与这些序列对齐的序列。",
    "[required]  Summary statistics of the Deblur process.": "[必需] Deblur 过程的汇总统计信息。",
    "[required]  The sample metadata column containing the per-sample barcodes.": "[必需] 包含每个样本条形码的样本元数据列。",
    "[default: Yes]  Perform 12nt Golay error correction on the barcode reads.": "[默认：是] 对条形码读取进行12nt高莱误差校正。",
    "[default: No]  If provided, the barcode sequence reads will be reverse complemented prior to demultiplexing.": "[默认值：否] 如果提供，条形码序列的读取将在去复用之前进行反向互补。",
    "[default: No]  If provided, the barcode sequences in the sample metadata will be reverse complemented prior to demultiplexing.": "[默认：否] 如果提供了，样本元数据中的条形码序列将在去复用之前进行反向互补。",
    "[default: No]  If enabled, ignore mismatches in sequence record description fields.": "[默认：否] 如果启用，忽略序列记录描述字段中的不匹配。",
    "[required]  The demultiplexed data from which samples should be filtered.": "[必需] 需要过滤的去复用数据样本。",
    "[required]  Sample metadata indicating which sample ids to filter. The optional `where` parameter may be used to filter ids based on specified conditions in the metadata. The optional `exclude_ids` parameter may be used to exclude the ids specified in the metadata from the filter.": "[必需] 样本元数据，指明要过滤的样本ID。可选的 `where` 参数可用于根据元数据中指定的条件过滤ID。可选的 `exclude_ids` 参数可用于从过滤中排除元数据中指定的ID。",
    "[optional]  Optional SQLite WHERE clause specifying sample metadata criteria that must be met to be included in the filtered data. If not provided, all samples in `metadata` that are also in the demultiplexed data will be retained.": "[可选] 可选的 SQLite WHERE 子句，用于指定样本元数据标准，只有满足这些标准的样本才会被包含在过滤后的数据中。如果没有提供，所有在 `metadata` 中的样本，并且也在解复用数据中的样本将被保留。",
    "[default: No]  Defaults to False. If True, the samples selected by the `metadata` and optional `where` parameter will be excluded from the filtered data.": "[默认值：否] 默认为False。如果设置为True，则通过`metadata`和可选的`where`参数选择的样本将从过滤后的数据中排除。",
    "[required]  The demultiplexed sequences to be subsampled.": "[必需] 需要进行子采样的去多重序列。",
    "[required]  The fraction of sequences to retain in subsample.": "[必需] 要保留在子样本中的序列分数。",
    "[required]  The demultiplexed sequences to be summarized.": "[必需] 需要汇总的去多路复用序列。",
    "[default: 10000]  The number of sequences that should be selected at random for quality score plots. The quality plots will present the average positional qualities across all of the sequences selected. If input sequences are paired end, plots will be generated for both forward and reverse reads for the same `n` sequences.": "[默认值：10000] 应该随机选择用于质量得分图的序列数量。质量图将展示所有选定序列的平均位置质量。如果输入序列是成对的，将为相同的`n`序列生成正向和反向读取的质量图。",
    "[required]  Matrix of distances between pairs of samples.": "[必需] 样本对之间的距离矩阵。",
    "[required]  Sample metadata containing formula terms.": "[必需] 包含公式术语的样本元数据。",
    "[required]  Model formula containing only independent terms contained in the sample metadata. These can be continuous variables or factors, and they can have interactions as in a typical R formula. E.g., the formula \"treatment+block\" would test whether the input distance matrix partitions based on \"treatment\" and \"block\" sample metadata. The formula \"treatment*block\" would test both of those effects as well as their interaction. Enclose formulae in quotes to avoid unpleasant surprises.": "[必需] 包含仅在样本元数据中包含的独立项的模型公式。这些可以是连续变量或因子，并且它们可以像典型的R公式一样具有交互作用。例如，公式“treatment+block”将测试输入的距离矩阵是否基于“treatment”和“block”样本元数据进行分区。公式“treatment*block”将测试这两种效应以及它们的交互作用。将公式用引号括起来以避免不愉快的意外。",
    "[default: 999]  The number of permutations to be run when computing p-values.": "[默认：999] 在计算p值时要运行的排列数。",
    "[default: 1]  Number of parallel processes to run.": "[默认值：1] 要运行的并行进程数。",
    "[required]  The feature table containing the samples for which alpha diversity should be computed.": "[必需] 包含用于计算α多样性的样本的特征表。",
    "[required]  Vector of alpha diversity values by sample.": "[必需] 每个样本的alpha多样性值向量。",
    "[default: No]  If supplied, IDs that are not found in both the alpha diversity vector and metadata will be discarded before calculating the correlation. Default behavior is to error on any mismatched IDs.": "[默认值：否] 如果提供了参数，那么在计算相关性之前，不在alpha多样性向量和元数据中找到的ID将被丢弃。默认的行为是在任何不匹配的ID上出现错误。",
    "[required]  Phylogenetic tree containing tip identifiers that correspond to the feature identifiers in the table. This tree can contain tip ids that are not present in the table, but all feature ids in the table must be present in this tree.": "[必需] 包含与表中特征标识符相对应的末端标识符的系统发育树。这棵树可以包含表中未出现的末端ID，但表中的所有特征ID都必须出现在这棵树中。",
    "[required]  Feature table to compute rarefaction curves from.": "[必需] 用于计算稀释曲线的特征表。",
    "[required]  The maximum rarefaction depth. Must be greater than min_depth.": "[必需] 最大稀释深度。必须大于 min_depth。",
    "[optional]  Optional phylogeny for phylogenetic metrics.": "[可选] 用于系统发育指标的可选系统发育树。",
    "[optional]  The metrics to be measured. By default computes observed_features, shannon, and if phylogeny is provided, faith_pd.": "[可选] 要测量的指标。默认情况下计算 observed_features、shannon，如果提供了系统发育信息，还会计算 faith_pd。",
    "[optional]  The sample metadata.": "[可选] 样本元数据。",
    "[default: 1]  The minimum rarefaction depth.": "[默认：1] 最小的稀释深度。",
    "[default: 10]  The number of rarefaction depths to include between min_depth and max_depth.": "[默认值：10] 在最小深度和最大深度之间包含的稀释深度的数量。",
    "[default: 10]  The number of rarefied feature tables to compute at each step.": "[默认：10] 每一步计算的稀释特征表的数量。",
    "[required]  The feature table containing the samples over which beta diversity should be computed.": "[必需] 包含用于计算β多样性的样本的特征表。",
    "[default: 1]  A pseudocount to handle zeros for compositional metrics.  This is ignored for other metrics.": "[默认值：1] 一个用于处理组成性指标零值的伪计数。对于其他指标，这被忽略。",
    "[default: 1]  The number of concurrent jobs to use in performing this calculation. May not exceed the number of available physical cores. If n_jobs = 'auto', one job will be launched for each identified CPU core on the host.": "[默认: 1] 在执行此计算时使用的并发作业数。不得超过可用物理核心数。如果 n_jobs = 'auto'，则在主机上为每个识别到的 CPU 核心启动一个作业。",
    "[required]  Numeric metadata column from which to compute pairwise Euclidean distances": "[必需] 用于计算成对欧几里得距离的数值元数据列",
    "[default: 999]  The number of permutations to be run when computing p-values. Supplying a value of zero will disable permutation testing and p-values will not be calculated (this results in *much* quicker execution time if p-values are not desired).": "[默认：999] 在计算p值时要运行的排列数量。如果提供零值，将禁用排列测试，并且不会计算p值（如果不需要p值，这将导致执行时间*大大*加快）。",
    "[default: No]  If supplied, IDs that are not found in both distance matrices will be discarded before applying the Mantel test. Default behavior is to error on any mismatched IDs.": "[默认：否] 如果提供了，那些在两个距离矩阵中都没有找到的ID将在应用Mantel检验之前被丢弃。默认行为是在任何不匹配的ID上出错。",
    "[default: 'Metadata']  Label for `distance_matrix` in the output visualization.": "[默认: '元数据']  在输出可视化中 `distance_matrix` 的标签。",
    "[default: 'Distance Matrix']  Label for `metadata_distance_matrix` in the output visualization.": "[默认: '距离矩阵']  输出可视化中 `metadata_distance_matrix` 的标签。",
    "[required]  Categorical sample metadata column.": "[必需] 分类样本元数据列。",
    "[default: No]  Perform pairwise tests between all pairs of groups in addition to the test across all groups. This can be very slow if there are a lot of groups in the metadata column.": "[默认值：否] 除了跨所有组的测试外，还对元数据列中的所有组对执行配对测试。如果元数据列中有很多组，这可能会非常慢。",
    "[default: 1]  The number of CPU threads to use in performing this calculation. May not exceed the number of available physical cores. If threads = 'auto', one thread will be created for each identified CPU core on the host.": "[默认: 1] 在执行此计算时使用的CPU线程数。不得超过可用物理核心数。如果线程 = 'auto'，则会在主机上为每个识别出的CPU核心创建一个线程。",
    "[default: No]  Perform variance adjustment based on Chang et al. BMC Bioinformatics 2011. Weights distances based on the proportion of the relative abundance represented between the samples at a given node under evaluation.": "[默认：否] 根据Chang等人2011年在BMC Bioinformatics上的研究，执行基于方差的调整。根据在给定节点下评估的样本之间相对丰度所代表的比例，对距离进行加权。",
    "[optional]  This parameter is only used when the choice of metric is generalized_unifrac. The value of alpha controls importance of sample proportions. 1.0 is weighted normalized UniFrac. 0.0 is close to unweighted UniFrac, but only if the sample proportions are dichotomized.": "[可选] 这个参数仅在度量选择为generalized_unifrac时使用。alpha值控制样本比例的重要性。1.0是加权的标准化UniFrac。0.0接近无权UniFrac，但只有在样本比例二值化的情况下。",
    "[default: No]  In a bifurcating tree, the tips make up about 50% of the nodes in a tree. By ignoring them, specificity can be traded for reduced compute time. This has the effect of collapsing the phylogeny, and is analogous (in concept) to moving from 99% to 97% OTUs": "[默认：否] 在一个二叉树中，末端节点大约占树中节点的50%。通过忽略它们，可以牺牲特异性以换取减少的计算时间。这会产生折叠系统发育的效果，并且（在概念上）类似于从99%的OTUs移动到97%的OTUs。",
    "[required]  Feature table upon which to perform beta diversity rarefaction analyses.": "[必需] 用于执行贝塔多样性稀有化分析的特征表。",
    "[required]  The sample metadata used for the Emperor jackknifed PCoA plot.": "[必需] 用于 Emperor Jackknifed PCoA 绘图的样本元数据。",
    "[required]  The total frequency that each sample should be rarefied to prior to computing the diversity metric.": "[必需] 在计算多样性指标之前，每个样本应该被稀释到的总频率。",
    "[optional]  Phylogenetic tree containing tip identifiers that correspond to the feature identifiers in the table. This tree can contain tip ids that are not present in the table, but all feature ids in the table must be present in this tree. [required for phylogenetic metrics]": "[可选] 包含与表中特征标识符相对应的末端标识符的系统发育树。这棵树可以包含不出现在表中的末端ID，但表中的所有特征ID都必须出现在这棵树中。[用于系统发育指标]",
    "[default: 10]  Number of times to rarefy the feature table at a given sampling depth.": "[默认：10] 在给定的采样深度下对特征表进行稀释的次数。",
    "[required]  The feature table containing the samples over which diversity metrics should be computed.": "[必需] 包含样本的特征表，用于计算多样性指标。",
    "[required]  The total frequency that each sample should be rarefied to prior to computing diversity metrics.": "[必需] 在计算多样性指标之前，每个样本应被稀释到的总频率。",
    "[required]  The sample metadata to use in the emperor plots.": "[必需] 在皇帝（Emperor）图中使用的样本元数据。",
    "[default: No]  Rarefy with replacement by sampling from the multinomial distribution instead of rarefying without replacement.": "[默认值：否] 通过从多项式分布中进行有放回抽样进行稀释，而不是进行无放回稀释。",
    "[default: 1]  [beta methods only] - The number of concurrent jobs to use in performing this calculation. May not exceed the number of available physical cores. If n_jobs = 'auto', one job will be launched for each identified CPU core on the host.": "[默认值：1]  [仅限beta方法] - 在执行此计算时使用的并发作业数。不得超过可用的物理核心数。如果n_jobs = 'auto'，则在主机上识别的每个CPU核心上启动一个作业。",
    "[default: 1]  [beta/beta-phylogenetic methods only] - The number of concurrent jobs or CPU threads to use in performing this calculation. Individual methods will create jobs/threads as implemented in q2-diversity-lib dependencies. May not exceed the number of available physical cores. If n_jobs_or_threads = 'auto', one thread/job will be created for each identified CPU core on the host.": "[默认值：1] [仅限beta/基于系统发育的方法] - 在执行此计算时使用的并发作业数或CPU线程数。各个方法将根据q2-diversity-lib依赖项中的实现创建作业/线程。不得超过可用物理核心的数量。如果n_jobs_or_threads = 'auto'，则将为主机上识别的每个CPU核心创建一个线程/作业。",
    "[required]  Distance matrix to filter by sample.": "[必需] 根据样本筛选的距离矩阵。",
    "[required]  Sample metadata used with `where` parameter when selecting samples to retain, or with `exclude_ids` when selecting samples to discard.": "[必需] 用于与 `where` 参数一起选择保留样本时使用的样本元数据，或与 `exclude_ids` 一起选择丢弃样本时使用。",
    "[optional]  SQLite WHERE clause specifying sample metadata criteria that must be met to be included in the filtered distance matrix. If not provided, all samples in `metadata` that are also in the input distance matrix will be retained.": "[可选] SQLite WHERE 子句，指定样本元数据标准，只有符合这些标准的样本才会被包含在过滤后的距离矩阵中。如果没有提供，那么在输入距离矩阵中也存在的 `metadata` 中的所有样本都将被保留。",
    "[default: No]  If `True`, the samples selected by `metadata` or `where` parameters will be excluded from the filtered distance matrix instead of being retained.": "[默认值：否] 如果设置为 `True`，则通过 `metadata` 或 `where` 参数选择的样本将从过滤后的距离矩阵中排除，而不是保留。",
    "[default: 'Distance Matrix 1']  Label for `dm1` in the output visualization.": "[默认：'距离矩阵 1'] 在输出可视化中的 `dm1` 标签。",
    "[default: 'Distance Matrix 2']  Label for `dm2` in the output visualization.": "[默认: '距离矩阵 2'] 在输出可视化中的 `dm2` 标签。",
    "[required]  The distance matrix on which PCoA should be computed.": "[必需] 应该计算PCoA的距离矩阵。",
    "[optional]  Dimensions to reduce the distance matrix to. This number determines how many eigenvectors and eigenvalues are returned,and influences the choice of algorithm used to compute them. By default, uses the default eigendecomposition method, SciPy's eigh, which computes all eigenvectors and eigenvalues in an exact manner. For very large matrices, this is expected to be slow. If a value is specified for this parameter, then the fast, heuristic eigendecomposition algorithm fsvd is used, which only computes and returns the number of dimensions specified, but suffers some degree of accuracy loss, the magnitude of which varies across different datasets.": "[可选] 降低距离矩阵的维度。此数字决定了返回多少个特征向量和特征值，并影响用于计算它们算法的选择。默认情况下，使用默认的特征分解方法，即SciPy的eigh，它以精确的方式计算所有的特征向量和特征值。对于非常大的矩阵，预计这将是缓慢的。如果为这个参数指定了一个值，那么就会使用快速的启发式特征分解算法fsvd，它只计算并返回指定数量的维度，但会有一定的精度损失，其大小因不同的数据集而异。",
    "[required]  The PCoA where the features will be projected onto.": "[必需] 将特征投影到的PCoA（主坐标分析）。",
    "[required]  Variables to project onto the PCoA matrix": "[必需] 用于投影到主坐标分析（PCoA）矩阵的变量",
    "[required]  The ordination matrix to which data is fitted to.": "[必需] 拟合数据所使用的排序矩阵。",
    "[required]  The ordination matrix that's fitted to the reference ordination.": "[必需] 与参考排序拟合的排序矩阵。",
    "[default: 5]  The number of dimensions to use when fitting the two matrices": "[默认：5] 拟合两个矩阵时使用的维度数量",
    "[default: 999]  The number of permutations to be run when computing p-values. Supplying a value of `disable` will disable permutation testing and p-values will not be calculated (this results in *much* quicker execution time if p-values are not desired).": "[默认: 999] 计算 p 值时运行的排列数。提供 `disable` 值将禁用排列测试，并且不会计算 p 值（如果不需要 p 值，这将导致执行时间 *大大* 缩短）。",
    "[required]  The distance matrix on which t-SNE should be computed.": "[必需] 应该计算t-SNE的距离矩阵。",
    "[default: 2]  Dimensions to reduce the distance matrix to.": "[默认：2] 降低距离矩阵的维度。",
    "[default: 25.0]  Provide the balance between local and global structure. Low values concentrate on local structure. Large values sacrifice local details for a broader global embedding. The default value is 25 to achieve better results for small microbiome datasets.": "[默认值：25.0] 提供局部和全局结构之间的平衡。较低的值集中于局部结构。较大的值为了更广泛的全局嵌入而牺牲局部细节。默认值是25，以获得更好的小型微生物组数据集的结果。",
    "[default: 1000]": "[默认值：1000]",
    "[default: 200.0]  Controls how much the weights are adjusted at each update.": "[默认值：200.0] 控制每次更新时权重的调整程度。",
    "[default: 12.0]  Affects the tightness of the shown clusters. Larger values increase the distance between natural clusters in the embedded space.": "[默认值：12.0] 影响显示的聚类紧密度。较大的值会增加嵌入空间中自然聚类之间的距离。",
    "[optional]  Seed used by random number generator.": "[可选] 随机数生成器使用的种子。",
    "[required]  The distance matrix on which UMAP should be computed.": "[必需] 需要计算UMAP的距离矩阵。",
    "[default: 15]  Provide the balance between local and global structure. Low values prioritize the preservation of local structures. Large values sacrifice local details for a broader global embedding.": "[默认值：15] 提供局部和全局结构之间的平衡。低值优先保留局部结构。大值为了更广泛的全局嵌入牺牲局部细节。",
    "[default: 0.4]  Controls the cluster size. Low values cause clumpier clusters. Higher values preserve a broad topological structure. To get less overlapping data points the default value is set to 0.4. For more details visit: https://umap-learn.readthedocs.io/en/latest/parameters.html": "[默认值：0.4] 控制聚类大小。较低的值会导致更紧密的聚类。较高的值保留更广泛的拓扑结构。为了获得更少重叠的数据点，默认值设置为0.4。有关更多详细信息，请访问：https://umap-learn.readthedocs.io/en/latest/parameters.html",
    "[required]  The feature table containing the samples for which a selected metric should be computed.": "[必需] 包含用于计算所选指标的样本的特征表。",
    "[default: 1]  A pseudocount to handle zeros for compositional metrics. This is ignored for non-compositional metrics.": "[默认值：1] 一个用于处理组成性指标中零值的伪计数。对于非组成性指标，这将被忽略。",
    "[required]  The feature tables containing the samples over which beta diversity should be computed.": "[必需] 包含用于计算beta多样性的样本的特征表。",
    "[required]  Phylogenetic trees containing tip identifiers that correspond to the feature identifiers in the table. This tree can contain tip ids that are not present in the table, but all feature ids in the table must be present in this tree.": "[必需] 包含与表中特征标识符相对应的末端标识符的系统发育树。这棵树可以包含表中未出现的末端ID，但表中的所有特征ID都必须出现在这棵树中。",
    "[optional]  The weight applied to each tree/table pair. This tuple is expected to be in index order with tables and phylogenies. Default is to weight each tree/table pair evenly.": "[可选] 应用于每对树/表的权重。这个元组预期以索引顺序与表和系统发育树相对应。默认是对每对树/表均匀地加权。",
    "[required]  The feature table containing the samples for which Bray-Curtis dissimilarity should be computed.": "[必需] 包含要计算Bray-Curtis差异性的样本的特征表。",
    "[required]  The feature table containing the samples for which Faith's phylogenetic diversity should be computed. Table values will be converted to presence/absence.": "[必需] 包含用于计算Faith的系统发育多样性的样本的特征表。表中的值将被转换为出现/缺失。",
    "[required]  The feature table containing the samples for which Jaccard distance should be computed.": "[必需] 包含用于计算Jaccard距离的样本的特征表。",
    "[required]  The feature table containing the samples for which the number of observed features should be calculated. Table values will be converted to presence/absence.": "[必需] 包含要计算观察到的特征数量的样本的特征表。表中的值将被转换为存在/缺失。",
    "[required]  The feature table containing the samples for which Pielou's evenness should be computed.": "[必需] 包含用于计算Pielou均匀度的样本的特征表。",
    "[default: No]  Samples with fewer than two observed features produce undefined (NaN) values. If true, these samples are dropped from the output vector.": "[默认值：否] 观察到的特征少于两个的样本会产生未定义（NaN）值。如果设置为真，这些样本将从输出向量中删除。",
    "[required]  The feature table containing the samples for which Shannon's Entropy should be computed.": "[必需] 包含要计算香农熵的样本的特征表。",
    "[default: No]  Samples with no observed features produce undefined (NaN) values. If true, these samples are dropped from the output vector.": "[默认值：否] 没有观察到特征的样本会产生未定义（NaN）值。如果设置为真，这些样本将从输出向量中删除。",
    "[required]  The feature table containing the samples for which Unweighted Unifrac should be computed.": "[必需] 包含要计算无权重Unifrac的样本的特征表。",
    "[required]  The feature table containing the samples for which Weighted Unifrac should be computed.": "[必需] 包含用于计算加权Unifrac的样本的特征表。",
    "[required]  The principal coordinates matrix to be plotted.": "[必需] 要绘制的主成分坐标矩阵。",
    "[required]  The sample metadata": "[必需] 样本元数据",
    "[optional]  The feature metadata (useful to manipulate the arrows in the plot).": "[可选] 特征元数据（用于在图中操作箭头非常有用）。",
    "[default: No]  This will suppress the error raised when the coordinates matrix contains samples that are not present in the metadata. Samples without metadata are included by setting all metadata values to: \"This sample has no metadata\". This flag is only applied if at least one sample is present in both the coordinates matrix and the metadata.": "[默认：否] 这将抑制坐标矩阵包含元数据中不存在的样本时引发的错误。没有元数据的样本通过将所有元数据值设置为：“此样本没有元数据”来包含。只有在坐标矩阵和元数据中至少存在一个样本时，才应用此标志。",
    "[default: No]  If specified, the point and arrow coordinates will be swapped.": "[默认值：否] 如果指定，点和箭头坐标将被交换。",
    "[default: 5]  The number of most important features (arrows) to display in the ordination. “Importance” is calculated for each feature based on the vector’s magnitude (euclidean distance from origin).": "[默认：5] 在排序图中显示的最重要的特征（箭头）的数量。“重要性”是根据每个特征向量的幅度（从原点的欧几里得距离）计算的。",
    "[optional]  Numeric sample metadata columns that should be included as axes in the Emperor plot.": "[可选] 在 Emperor 绘图中作为轴包含的数值样本元数据列。",
    "[default: No]  Biplot arrows cannot be visualized using this method. If you want to visualize biplot arrows use the `biplot` method. Enabling this setting will ignore any PCoA features that are present, otherwise, if PCoA features are detected an error will be raised.": "[默认：否] 生物坐标图箭头无法使用此方法进行可视化。如果您想可视化生物坐标图箭头，请使用 `biplot` 方法。启用此设置将忽略任何存在的PCoA特征，否则，如果检测到PCoA特征，将引发错误。",
    "[required]  The reference ordination matrix to be plotted.": "[必需] 需要绘制的参考排序矩阵。",
    "[required]  The \"other\" ordination matrix to be plotted (the one that was fitted to the reference).": "[必需] 要绘制的“其他”排序矩阵（即拟合到参考矩阵的那个）。",
    "[optional]  The M^2 value of the procrustes analysis & its associated p value.": "[可选] Procrustes分析的M^2值及其相关的p值。",
    "[required]  Query sequences.": "[必需] 查询序列。",
    "[required]  Reference sequences.": "[必需] 参考序列。",
    "[default: 10]  Maximum number of hits to keep for each query. BLAST will choose the first N hits in the reference database that exceed perc_identity similarity to query. NOTE: the database is not sorted by similarity to query, so these are the first N hits that pass the threshold, not necessarily the top N hits.": "[默认值：10] 每个查询要保留的命中数的最大值。BLAST将选择参考数据库中第一个N个超过查询的百分比相似度的命中。注意：数据库不是根据与查询的相似度排序的，因此这些是第一个通过阈值的N个命中，不一定是前N个命中。",
    "[default: 0.8]  Reject match if percent identity to query is lower.": "[默认值：0.8] 如果与查询的相似性百分比较低，则拒绝匹配。",
    "[default: 0.8]  Reject match if query alignment coverage per high-scoring pair is lower. Note: this uses blastn's qcov_hsp_perc parameter, and may not behave identically to the query_cov parameter used by classify-consensus-vsearch.": "[默认值：0.8] 如果查询比对覆盖率对于高得分配对较低，则拒绝匹配。注意：这使用blastn的qcov_hsp_perc参数，并且可能不会与classify-consensus-vsearch使用的query_cov参数的行为完全相同。",
    "[default: 0.001]  BLAST expectation value (E) threshold for saving hits.": "[默认值：0.001] 保存命中结果的BLAST期望值（E）阈值。",
    "[default: Yes]  Report both matching and non-matching queries. WARNING: always use the default setting for this option unless if you know what you are doing! If you set this option to False, your sequences and feature table will need to be filtered to exclude unclassified sequences, otherwise you may run into errors downstream from missing feature IDs. Set to FALSE to mirror default BLAST search.": "[默认：是] 报告匹配和不匹配的查询。警告：除非你知道自己在做什么，否则总是使用此选项的默认设置！如果你将此选项设置为False，你的序列和特征表将需要被过滤以排除未分类的序列，否则你可能会在缺少特征ID的情况下出现下游错误。设置为FALSE以镜像默认的BLAST搜索。",
    "[required]  reference taxonomy labels.": "[必需] 参考分类标签。",
    "[default: 0.51]  Minimum fraction of assignments must match top hit to be accepted as consensus assignment.": "[默认：0.51] 最小的分配比例必须与最佳命中匹配才能被接受为共识分配。",
    "[default: 'Unassigned']  Annotation given to sequences without any hits.": "[默认: '未分配']  给没有匹配到序列的注释。",
    "[required]  Query Sequences.": "[必需] 查询序列。",
    "[required]  Reference taxonomy labels.": "[必需] 参考分类标签。",
    "[default: 10]  Maximum number of hits to keep for each query. Set to \"all\" to keep all hits > perc_identity similarity. Note that if strand=both, maxaccepts will keep N hits for each direction (if searches in the opposite direction yield results that exceed the minimum perc_identity). In those cases use maxhits to control the total number of hits returned. This option works in pair with maxrejects. The search process sorts target sequences by decreasing number of k-mers they have in common with the query sequence, using that information as a proxy for sequence similarity. After pairwise alignments, if the first target sequence passes the acceptation criteria, it is accepted as best hit and the search process stops for that query. If maxaccepts is set to a higher value, more hits are accepted. If maxaccepts and maxrejects are both set to \"all\", the complete database is searched.": "[默认值：10] 每个查询要保留的最大命中数。设置为“all”以保留所有具有大于perc_identity相似度的命中。请注意，如果strand=both，maxaccepts将为每个方向保留N个命中（如果搜索相反方向的结果超过了最小perc_identity）。在这些情况下，请使用maxhits来控制返回的总命中数。此选项与maxrejects配对使用。搜索过程根据目标序列与查询序列共有的k-mer数量递减排序，使用该信息作为序列相似度的代理。在成对比对之后，如果第一个目标序列通过了接受标准，则将其接受为最佳命中，并且对于该查询，搜索过程停止。如果将maxaccepts设置为更高的值，则接受更多的命中。如果maxaccepts和maxrejects都设置为“all”，则搜索整个数据库。",
    "[default: 0.8]  Reject match if query alignment coverage per high-scoring pair is lower.": "[默认值：0.8] 如果查询序列在高得分配对中的比对覆盖度较低，则拒绝匹配。",
    "[default: No]  Search for exact full-length matches to the query sequences. Only 100% exact matches are reported and this command is much faster than the default. If True, the perc_identity, query_cov, maxaccepts, and maxrejects settings are ignored. Note: query and reference reads must be trimmed to the exact same DNA locus (e.g., primer site) because only exact matches will be reported.": "[默认值：否]  搜索与查询序列完全匹配的完整序列。只有100%完全匹配的序列会被报告，并且这个命令比默认设置快得多。如果设置为真，perc_identity、query_cov、maxaccepts 和 maxrejects 的设置将被忽略。注意：查询和参考序列必须被修剪到完全相同的DNA位点（例如，引物位点），因为只有完全匹配的序列会被报告。",
    "[default: No]  Only the top hits between the query and reference sequence sets are reported. For each query, the top hit is the one presenting the highest percentage of identity. Multiple equally scored top hits will be used for consensus taxonomic assignment if maxaccepts is greater than 1.": "[默认值：否] 仅报告查询和参考序列集之间的顶级命中。对于每个查询，顶级命中是呈现最高相似性百分比的那个。如果maxaccepts大于1，则多个得分相同的顶级命中将被用于共识分类学分配。",
    "[default: 'all']  Maximum number of hits to show once the search is terminated.": "[默认：'all'] 搜索终止后一次显示的最大命中数。",
    "[required]  Maximum number of hits to show once the search is terminated.": "[必需] 搜索结束时一次显示的最大命中数。",
    "[default: 'all']  Maximum number of non-matching target sequences to consider before stopping the search. This option works in pair with maxaccepts (see maxaccepts description for details).": "[默认值：'all'] 在停止搜索之前考虑的不匹配目标序列的最大数量。此选项与maxaccepts配对使用（详见maxaccepts描述）。",
    "[required]  Maximum number of non-matching target sequences to consider before stopping the search. This option works in pair with maxaccepts (see maxaccepts description for details).": "[必需] 在停止搜索之前考虑的不匹配目标序列的最大数量。此选项与 maxaccepts（见 maxaccepts 描述以获取详细信息）配对使用。",
    "[default: Yes]  Report both matching and non-matching queries. WARNING: always use the default setting for this option unless if you know what you are doing! If you set this option to False, your sequences and feature table will need to be filtered to exclude unclassified sequences, otherwise you may run into errors downstream from missing feature IDs.": "[默认：是] 报告匹配和不匹配的查询。警告：除非你知道你在做什么，否则总是使用此选项的默认设置！如果你将此选项设置为False，你的序列和特征表将需要被过滤以排除未分类的序列，否则你可能会在缺少特征ID的情况下在下游遇到错误。",
    "[default: 0.0]  Show hits with percentage of identity of at least N, without terminating the search. A normal search stops as soon as enough hits are found (as defined by maxaccepts, maxrejects, and perc_identity). As weak_id reports weak hits that are not deduced from maxaccepts, high perc_identity values can be used, hence preserving both speed and sensitivity. Logically, weak_id must be smaller than the value indicated by perc_identity, otherwise this option will be ignored.": "[默认值：0.0] 显示至少有N%的相似性百分比的命中，但不终止搜索。正常的搜索在找到足够的命中时就会停止（由maxaccepts、maxrejects和perc_identity定义）。由于weak_id报告的不是由maxaccepts推导出来的弱命中，因此可以使用较高的perc_identity值，从而保持速度和灵敏度。逻辑上，weak_id必须小于由perc_identity指示的值，否则此选项将被忽略。",
    "[default: 1]  Number of threads to use for job parallelization.": "[默认值：1] 用于作业并行化的工作线程数。",
    "[required]  Pre-trained sklearn taxonomic classifier for classifying the reads.": "[必需] 用于对读段进行分类的预训练sklearn分类器。",
    "[default: 0.5]  Percent sequence similarity to use for PREFILTER. Reject match if percent identity to query is lower. Set to a lower value to perform a rough pre-filter. This parameter is ignored if `prefilter` is disabled.": "[默认：0.5]  用于PREFILTER的序列相似性百分比。如果与查询的百分比一致性较低，则拒绝匹配。设置较低的值以执行粗略的预过滤。如果禁用了`prefilter`，则忽略此参数。",
    "[default: 0.8]  Query coverage threshold to use for PREFILTER. Reject match if query alignment coverage per high-scoring pair is lower. Set to a lower value to perform a rough pre-filter. This parameter is ignored if `prefilter` is disabled.": "[默认值：0.8]  用于PREFILTER的查询覆盖度阈值。如果高得分对的查询比对覆盖度较低，则拒绝匹配。将此值设置为较低值以执行粗略的预过滤。如果禁用了`prefilter`，则忽略此参数。",
    "[default: 'all']": "[默认值：'all']",
    "[default: 'auto']  Number of reads to process in each batch for sklearn classification. If \"auto\", this parameter is autoscaled to min(number of query sequences / threads, 20000).": "[默认: 'auto']  每次处理用于sklearn分类的读取数量。如果为\"auto\"，则此参数自动缩放到最小值（查询序列数/线程数，20000）。",
    "[required]  Number of reads to process in each batch for sklearn classification. If \"auto\", this parameter is autoscaled to min(number of query sequences / threads, 20000).": "[必需] 对于sklearn分类，每个批次要处理的读取数。如果为\"auto\"，则此参数自动缩放到最小值（查询序列数/线程数，20000）。",
    "[default: 0.7]  Confidence threshold for limiting taxonomic depth. Set to \"disable\" to disable confidence calculation, or 0 to calculate confidence but not apply it to limit the taxonomic depth of the assignments.": "[默认值：0.7] 限制分类深度的置信度阈值。设置为“disable”以禁用置信度计算，或设置为0以计算置信度但不应用它来限制分配的分类深度。",
    "[default: Yes]  Toggle positive filter of query sequences on or off.": "[默认: 是] 切换查询序列的正向过滤开关。",
    "[default: 1000]  Randomly extract the given number of sequences from the reference database to use for prefiltering. This parameter is ignored if `prefilter` is disabled.": "[默认值：1000] 从参考数据库中随机提取指定数量的序列用于预筛选。如果禁用了`prefilter`，则忽略此参数。",
    "[default: 0]  Use integer as a seed for the pseudo-random generator used during prefiltering. A given seed always produces the same output, which is useful for replicability. Set to 0 to use a pseudo-random seed. This parameter is ignored if `prefilter` is disabled.": "[默认: 0] 使用整数作为预筛选过程中使用的伪随机数生成器的种子。给定的种子总是产生相同的输出，这对于可复制性非常有用。设置为0以使用伪随机种子。如果`prefilter`被禁用，则忽略此参数。",
    "[required]  The feature data to be classified.": "[必需] 需要分类的特征数据。",
    "[required]  The taxonomic classifier for classifying the reads.": "[必需] 用于对读段进行分类的分类器。",
    "[default: 'auto']  Number of reads to process in each batch. If \"auto\", this parameter is autoscaled to min( number of query sequences / n_jobs, 20000).": "[默认: 'auto'] 每个批次处理的读取数。如果为“auto”，则此参数自动缩放到最小值（查询序列数 / n_jobs, 20000）。",
    "[required]  Number of reads to process in each batch. If \"auto\", this parameter is autoscaled to min( number of query sequences / n_jobs, 20000).": "[必需] 每个批次要处理的读数。如果为“auto”，则此参数自动缩放到最小值（查询序列数 / n_jobs, 20000）。",
    "[default: 1]  The maximum number of concurrently worker processes. If -1 all CPUs are used. If 1 is given, no parallel computing code is used at all, which is useful for debugging. For n_jobs below -1, (n_cpus + 1 + n_jobs) are used. Thus for n_jobs = -2, all CPUs but one are used.": "[默认值：1] 同时进行工作进程的最大数量。如果为-1，则使用所有CPU。如果给出1，则根本不使用并行计算代码，这在调试时非常有用。对于n_jobs小于-1的情况，使用(n_cpus + 1 + n_jobs)。因此，对于n_jobs = -2，使用除了一个之外的所有CPU。",
    "[default: '2*n_jobs']  \"all\" or expression, as in \"3*n_jobs\". The number of batches (of tasks) to be pre-dispatched.": "[默认: '2*n_jobs']  “all”或表达式，如“3*n_jobs”。要预先分派的批次（任务）的数量。",
    "[required]  forward primer sequence (5' -> 3').": "[必需] 正向引物序列（5' -> 3'）。",
    "[required]  reverse primer sequence (5' -> 3'). Do not use reverse-complemented primer sequence.": "[必需] 反向引物序列（5' -> 3'）。不要使用反向互补的引物序列。",
    "[default: 0]  trim_right nucleotides are removed from the 3' end if trim_right is positive. Applied before trunc_len and trim_left.": "[默认值：0] 如果trim_right为正，则从3'端移除相应数量的核苷酸。在应用trunc_len和trim_left之前执行。",
    "[default: 0]  read is cut to trunc_len if trunc_len is positive. Applied after trim_right but before trim_left.": "[默认：0] 如果截断长度trunc_len是正数，则将读取长度截断为trunc_len。在trim_right之后，trim_left之前应用。",
    "[default: 0]  trim_left nucleotides are removed from the 5' end if trim_left is positive. Applied after trim_right and trunc_len.": "[默认：0] 如果trim_left为正，则从5'端移除相应数量的核苷酸。该操作在trim_right和trunc_len之后应用。",
    "[default: 0.8]  minimum combined primer match identity threshold.": "[默认值：0.8] 最低的引物匹配一致性阈值。",
    "[default: 50]  Minimum amplicon length. Shorter amplicons are discarded. Applied after trimming and truncation, so be aware that trimming may impact sequence retention. Set to zero to disable min length filtering.": "[默认：50] 最小扩增子长度。较短的扩增子会被丢弃。应用在修剪和截断之后，因此请注意修剪可能会影响序列保留。设置为零以禁用最小长度过滤。",
    "[default: 0]  Maximum amplicon length. Longer amplicons are discarded. Applied before trimming and truncation, so plan accordingly. Set to zero (default) to disable max length filtering.": "[默认：0] 最大扩增子长度。较长的扩增子将被丢弃。在修剪和截断之前应用，因此请相应规划。设置为零（默认）以禁用最大长度过滤。",
    "[default: 1]  Number of seperate processes to run.": "[默认：1]  要运行的独立进程数。",
    "[default: 'auto']  Number of sequences to process in a batch. The `auto` option is calculated from the number of sequences and number of jobs specified.": "[默认: 'auto']  批量处理序列的数量。`auto`选项是根据指定的序列数量和作业数量计算得出的。",
    "[required]  Number of sequences to process in a batch. The `auto` option is calculated from the number of sequences and number of jobs specified.": "[必需] 一批要处理的序列数量。`auto`选项是根据指定的序列数量和作业数量计算得出的。",
    "[required]  Search results in BLAST6 output format": "[必需] 在BLAST6输出格式中搜索结果",
    "[default: 'Unassigned']  Annotation given when no consensus is found.": "[默认：'未分配'] 当没有找到共识时给出的注释。",
    "[default: 0.001]": "[默认值：0.001]",
    "[default: 20000]": "[默认：20000]",
    "[default: 'null']": "[默认: 'null']",
    "[default: No]": "[默认：否]",
    "[default: 'char_wb']": "[默认: 'char_wb']",
    "[default: 'strict']": "[默认：'严格']",
    "[default: 'utf-8']": "[默认: 'utf-8']",
    "[default: 'content']": "[默认：'内容']",
    "[default: Yes]": "[默认：是]",
    "[default: 8192]": "[默认值：8192]",
    "[default: '[7, 7]']": "[默认：'[7, 7]']",
    "[default: 'l2']": "[默认: 'l2']",
    "[default: '(?u)\\\\b\\\\w\\\\w+\\\\b']": "[默认: '(?u)\\\\b\\\\w\\\\w+\\\\b']",
    "[required]  The feature table to use in core features calculations.": "[必需] 在核心特征计算中使用的feature表。",
    "[default: 0.5]  The minimum fraction of samples that a feature must be observed in for that feature to be considered a core feature.": "[默认：0.5] 一个特征必须在样本中被观察到的最小分数，以便该特征被认为是一个核心特征。",
    "[default: 1.0]  The maximum fraction of samples that a feature must be observed in for that feature to be considered a core feature.": "[默认：1.0] 一个特征必须在样本中被观察到的最大分数，以便该特征被认为是一个核心特征。",
    "[default: 11]  The number of steps to take between `min_fraction` and `max_fraction` for core features calculations. This parameter has no effect if `min_fraction` and `max_fraction` are the same value.": "[默认值：11] 在计算核心特征时，`min_fraction` 和 `max_fraction` 之间的步数。如果 `min_fraction` 和 `max_fraction` 是相同的值，则此参数无效。",
    "[required]  The feature table from which features should be filtered.": "[必需] 应从中筛选特征的特征表。",
    "[default: 0]  The minimum total frequency that a feature must have to be retained.": "[默认：0] 一个特征必须保持的最小总频率。",
    "[optional]  The maximum total frequency that a feature can have to be retained. If no value is provided this will default to infinity (i.e., no maximum frequency filter will be applied).": "[可选] 一个特征可以保留的最大总频率。如果没有提供值，这将默认为无穷大（即，不应用最大频率过滤器）。",
    "[default: 0]  The minimum number of samples that a feature must be observed in to be retained.": "[默认：0] 一个特征必须被观察到的最小样本数，以便保留。",
    "[optional]  The maximum number of samples that a feature can be observed in to be retained. If no value is provided this will default to infinity (i.e., no maximum sample filter will be applied).": "[可选] 一个特征可以被观察到的最大样本数以保留。如果没有提供值，这将默认为无穷大（即，不应用最大样本过滤器）。",
    "[optional]  Feature metadata used with `where` parameter when selecting features to retain, or with `exclude_ids` when selecting features to discard.": "[可选] 特征元数据，用于与 `where` 参数一起选择要保留的特征，或与 `exclude_ids` 一起选择要丢弃的特征。",
    "[optional]  SQLite WHERE clause specifying feature metadata criteria that must be met to be included in the filtered feature table. If not provided, all features in `metadata` that are also in the feature table will be retained.": "[可选] SQLite WHERE子句，用于指定特征元数据标准，只有满足这些标准的特征才会被包含在过滤后的特征表中。如果没有提供，那么在`metadata`中存在且也在特征表中的特征都将被保留。",
    "[default: No]  If true, the features selected by `metadata` or `where` parameters will be excluded from the filtered table instead of being retained.": "[默认值：否] 如果为真，通过 `metadata` 或 `where` 参数选择的特征将从过滤表中排除，而不是保留。",
    "[default: Yes]  If true, drop any samples where none of the retained features are present.": "[默认: 是] 如果为真，丢弃任何保留的特征都不存在的样本。",
    "[required]  The minimum relative abundance for a feature to be retained.": "[必需] 一个特征被保留的最小相对丰度。",
    "[required]  The minimum portion of samples that a feature must have a relative abundance of at least `abundance` to be retained.": "[必需] 一个特征必须在至少“abundance”相对丰度的样本中保留的最小样本比例。",
    "[required]  The feature table from which samples should be filtered.": "[必需] 需要过滤样本的特征表。",
    "[default: 0]  The minimum total frequency that a sample must have to be retained.": "[默认：0] 一个样本必须保留的最小总频率。",
    "[optional]  The maximum total frequency that a sample can have to be retained. If no value is provided this will default to infinity (i.e., no maximum frequency filter will be applied).": "[可选] 一个样本可以保留的最大总频率。如果没有提供值，这将默认为无穷大（即，不应用最大频率过滤器）。",
    "[default: 0]  The minimum number of features that a sample must have to be retained.": "[默认：0] 一个样本必须具有的最小特征数量才能被保留。",
    "[optional]  The maximum number of features that a sample can have to be retained. If no value is provided this will default to infinity (i.e., no maximum feature filter will be applied).": "[可选] 一个样本可以保留的特征的最大数量。如果没有提供值，这将默认为无穷大（即，不应用最大特征过滤器）。",
    "[optional]  Sample metadata used with `where` parameter when selecting samples to retain, or with `exclude_ids` when selecting samples to discard.": "[可选] 用于 `where` 参数选择保留样本时，或用于 `exclude_ids` 选择丢弃样本时的样本元数据。",
    "[optional]  SQLite WHERE clause specifying sample metadata criteria that must be met to be included in the filtered feature table. If not provided, all samples in `metadata` that are also in the feature table will be retained.": "[可选] SQLite WHERE 子句，指定样本元数据标准，只有满足这些标准才会被包含在过滤后的特征表中。如果没有提供，则保留 `metadata` 中所有也在特征表中的样本。",
    "[default: No]  If true, the samples selected by `metadata` or `where` parameters will be excluded from the filtered table instead of being retained.": "[默认值：否] 如果为真，由 `metadata` 或 `where` 参数选择的样本将从过滤后的表中排除，而不是保留。",
    "[default: Yes]  If true, features which are not present in any retained samples are dropped.": "[默认值：是] 如果为真，不在任何保留样本中出现的特征将被删除。",
    "[required]  The sequences from which features should be filtered.": "[必需] 应从哪些序列中过滤特征。",
    "[optional]  Table containing feature ids used for id-based filtering.": "[可选] 包含用于基于ID过滤的特征ID的表格。",
    "[optional]  Feature metadata used for id-based filtering, with `where` parameter when selecting features to retain, or with `exclude_ids` when selecting features to discard.": "[可选] 用于基于ID过滤的特征元数据，在保留特征时使用`where`参数，或在选择要丢弃的特征时使用`exclude_ids`。",
    "[optional]  SQLite WHERE clause specifying feature metadata criteria that must be met to be included in the filtered feature table. If not provided, all features in `metadata` that are also in the sequences will be retained.": "[可选] SQLite WHERE 子句，用于指定特征元数据标准，只有满足这些标准的特征才会被包含在过滤后的特征表中。如果没有提供，那么所有在`metadata`中并且在序列中的特征都将被保留。",
    "[default: No]  If true, the features selected by the `metadata` (with or without the `where` parameter) or `table` parameter will be excluded from the filtered sequences instead of being retained.": "[默认值：否] 如果为真，通过 `metadata`（无论是否带有 `where` 参数）或 `table` 参数选择的特征将从过滤后的序列中排除，而不是保留。",
    "[required]  The table to group samples or features on.": "[必需] 用于对样本或特征进行分组的表格。",
    "[required]  A column defining the groups. Each unique value will become a new ID for the table on the given `axis`.": "[必需] 定义组的列。每个唯一值将成为给定 `axis` 上表格的新 ID。",
    "[required]  The feature table to visualize.": "[必需] 要可视化的特征表。",
    "[optional]  Annotate the sample IDs with these sample metadata values. When metadata is present and `cluster`='feature', samples will be sorted by the metadata values.": "[可选] 使用这些样本元数据值来注释样本ID。当存在元数据且 `cluster`='feature' 时，样本将根据元数据值进行排序。",
    "[optional]  Annotate the feature IDs with these feature metadata values. When metadata is present and `cluster`='sample', features will be sorted by the metadata values.": "[可选] 使用这些特征元数据值注释特征ID。当存在元数据且 `cluster`='sample' 时，特征将根据元数据值进行排序。",
    "[default: Yes]  Normalize the feature table by adding a psuedocount of 1 and then taking the log10 of the table.": "[默认：是] 通过添加一个假计数1，然后取表的log10来对特征表进行标准化。",
    "[optional]  Optional custom plot title.": "[可选] 可选的自定义图表标题。",
    "[required]  The collection of feature tables to be merged.": "[必需] 要合并的特征表集合。",
    "[default: 'error_on_overlapping_sample']  Method for handling overlapping ids.": "[默认: 'error_on_overlapping_sample']  处理重叠id的方法。",
    "[required]  The collection of feature sequences to be merged.": "[必需] 要合并的特征序列集合。",
    "[required]  The collection of feature taxonomies to be merged.": "[必需] 要合并的特征分类集合。",
    "[required]  The feature table to be converted into presence/absence abundances.": "[必需] 要转换为存在/不存在丰度特征表。",
    "[required]  The feature table to be rarefied.": "[必需] 需要进行稀释处理的特征表。",
    "[required]  The total frequency that each sample should be rarefied to. Samples where the sum of frequencies is less than the sampling depth will be not be included in the resulting table.": "[必需] 每个样本应被稀释到的总频率。频率总和小于抽样深度的样本将不会被包含在结果表中。",
    "[required]  The feature table to be converted into relative frequencies.": "[必需] 要被转换为相对频率的特征表。",
    "[required]  The table to be renamed": "[必需] 要重命名的表",
    "[required]  A metadata column defining the new ids. Each original id must map to a new unique id. If strict mode is used, then every id in the original table must have a new id.": "[必需] 一个元数据列，定义新的标识符。每个原始标识符必须映射到一个新的唯一标识符。如果使用严格模式，则原始表中的每个标识符都必须有一个新标识符。",
    "[default: No]  Whether the naming needs to be strict (each id in the table must have a new id). Otherwise, only the ids described in `metadata` will be renamed and the others will keep their original id names.": "[默认值：否] 是否需要严格的命名（表中的每个id都必须有一个新的id）。否则，只有在 `metadata` 中描述的id将被重命名，其他的id将保留其原始的id名称。",
    "[required]  The feature table to be sampled.": "[必需] 要采样的特征表。",
    "[required]  The total number of samples or features to be randomly sampled. Samples or features that are reduced to a zero sum will not be included in the resulting table.": "[必需] 要随机抽样的样本或特征的总数。被减少到零和的样本或特征将不会被包含在结果表中。",
    "[required]  The feature table to be summarized.": "[必需] 要汇总的特征表。",
    "[required]  The feature sequences to be tabulated.": "[必需] 需要制表的特征序列。",
    "[required]  The feature table to be transposed.": "[必需] 需要转置的特征表。",
    "[required]  The sequences used for a 'sepp' run to produce the 'tree'.": "[必需] 用于运行 'sepp' 以产生 '树' 的序列。",
    "[required]  The tree resulting from inserting fragments into a reference phylogeny, i.e. the output of function 'sepp'": "[必需] 将片段插入参考系统发育树中得到的树，即函数 'sepp' 的输出",
    "[required]  Reference taxonomic table that maps every OTU-ID into a taxonomic lineage string.": "[必需] 参考分类表，将每个OTU-ID映射为分类谱系字符串。",
    "[required]  A feature-table which needs to filtered down to those fragments that are contained in the tree, e.g. result of a Deblur or DADA2 run.": "[必需] 一个特征表，需要被过滤到那些包含在树中的片段，例如 Deblur 或 DADA2 运行的结果。",
    "[required]  The sequences to insert into the reference tree.": "[必需] 要插入到参考树中的序列。",
    "[required]  The reference database to insert the representative sequences into.": "[必需] 要将代表性序列插入的参考数据库。",
    "[default: 1000]  Each placement subset is further broken into subsets of at most these many sequences and a separate HMM is trained on each subset.": "[默认：1000] 每个放置子集进一步被分解为最多这些序列的子集，并且对每个子集训练一个单独的隐马尔可夫模型（HMM）。",
    "[default: 5000]  The tree is divided into subsets such that each subset includes at most these many subsets. The placement step places the fragment on only one subset, determined based on alignment scores. Further reading: https://github.com/smirarab/sepp/blob/master/tutorial/sepp-tutorial.md#sample-datasets-default-parameters.": "[默认值：5000] 树被划分为子集，以便每个子集最多包含这些数量的子集。放置步骤将片段仅放置在一个子集上，这是基于对齐分数决定的。进一步阅读：https://github.com/smirarab/sepp/blob/master/tutorial/sepp-tutorial.md#sample-datasets-default-parameters",
    "[default: 1]  The number of threads to use.": "[默认：1] 使用的线程数。",
    "[default: No]  Collect additional run information to STDOUT for debugging. Temporary directories will not be removed if run fails.": "[默认：否] 收集额外的运行信息到标准输出（STDOUT）以便于调试。如果运行失败，临时目录将不会被移除。",
    "[required]  The input table of counts.": "[必需] 输入计数表。",
    "[required]  The input tree with potential missing ids.": "[必需] 输入的树可能包含缺失的ID。",
    "[required]  The feature table containing the samples in which the columns will be clustered.": "[必需] 包含将要进行聚类的样本的特征表。",
    "[default: 0.5]  The value to add to zero counts in the feature table.": "[默认值：0.5] 要添加到特征表中零计数的值。",
    "[required]  The feature table that will be plotted as a heatmap. This table is assumed to have strictly positive values.": "[必需] 将作为热图绘制的特征表。假定此表中的值全为正数。",
    "[required]  A hierarchy of feature identifiers where each tipcorresponds to the feature identifiers in the table. This tree can contain tip ids that are not present in the table, but all feature ids in the table must be present in this tree.": "[必需] 一个特征标识符的层级结构，其中每个末端对应于表中的特征标识符。这棵树可以包含表中不存在的末端ID，但表中的所有特征ID都必须出现在这棵树中。",
    "[required]  Categorical metadata column to group the samples.": "[必需] 用于对样本进行分组的分类元数据列。",
    "[default: 0.5]  The pseudocount to add to avoid division by zero.": "[默认值：0.5] 为了避免除以零，要添加的伪计数。",
    "[default: 10]  Number of dimensions to highlight.": "[默认值：10] 要突出显示的维度数量。",
    "[required]  Contains gradient values to sort the features and samples.": "[必需] 包含用于对特征和样本进行排序的梯度值。",
    "[default: Yes]  Specifies if abundance or presence/absence information should be used to perform the clustering.": "[默认值: 是] 指定是否应该使用丰度或存在/不存在信息来执行聚类。",
    "[required]  The feature table containing the samples in which the ilr transform will be performed.": "[必需] 包含将在其中执行ilr转换的样本的特征表。",
    "[required]  A hierarchy of feature identifiers that defines the partitions of features.  Each tip in the hierarchycorresponds to the feature identifiers in the table. This tree can contain tip ids that are not present in the table, but all feature ids in the table must be present in this tree.  This assumes that all of the internal nodes in the tree have labels.": "[必需] 一个特征标识符的层级结构，它定义了特征的分区。层级结构中的每个末端对应于表中的特征标识符。这个树可以包含表中不存在的末端ID，但是表中的所有特征ID都必须在此树中存在。这假设树中的所有内部节点都有标签。",
    "[required]  A rooted phylogeny of feature identifiers that defines the partitions of features.  Each tip in the hierarchycorresponds to the feature identifiers in the table. This tree can contain tip ids that are not present in the table, but all feature ids in the table must be present in this tree.  This assumes that all of the internal nodes in the tree have labels. This tree may contain polytomic nodes (i.e., nodes with more than two children), in which case they will be bifurcated.": "[必需] 一个有根的系统发育树，定义了特征标识符的分区。层次结构中的每个末端对应于表中的特征标识符。这棵树可以包含表中不存在的末端ID，但表中的所有特征ID都必须存在于这棵树中。这假设树中的所有内部节点都有标签。这棵树可能包含多岐节点（即，具有多于两个子节点的节点），在这种情况下，它们将被二分。",
    "[required]  The differential abundance results in which will be ilr transformed.": "[必需] 差异丰度结果将进行ilr转换。",
    "[default: 10]  The top k most variable balances.": "[默认：10] 最多变的k个平衡。",
    "[optional]  The names of clades to focus on (overrides top-k-var).": "[可选] 要重点关注的类群名称（覆盖 top-k-var）。",
    "[required]  R-style formula specifying the model. All terms must be present in the sample metadata or metadata-transformable artifacts and can be continuous or categorical metadata columns. Formulae will be in the format \"a ~ b + c\", where \"a\" is the metric (dependent variable) and \"b\" and \"c\" are independent covariates. Use \"+\" to add a variable; \"+ a:b\" to add an interaction between variables a and b; \"*\" to include a variable and all interactions; and \"-\" to subtract a particular term (e.g., an interaction term). See https://patsy.readthedocs.io/en/latest/formulas.html for full documentation of valid formula operators. Always enclose formulae in quotes to avoid unpleasant surprises.": "[必需] 指定模型的 R 风格公式。所有项必须在样本元数据或可转换为元数据的工件中存在，并且可以是连续的或分类的元数据列。公式的格式为 \"a ~ b + c\"，其中 \"a\" 是度量（因变量），\"b\" 和 \"c\" 是独立协变量。使用 \"+\" 添加变量；\"+ a:b\" 添加变量 a 和 b 之间的交互作用；\"*\" 包括一个变量和所有交互作用；\"-\" 减去一个特定项（例如，一个交互项）。有关有效公式运算符的完整文档，请参见 https://patsy.readthedocs.io/en/latest/formulas.html。始终用引号括住公式以避免不愉快的意外。",
    "[required]  Feature table containing all features that should be used for target prediction.": "[必需] 包含所有应用于目标预测的特征的特诊表。",
    "[required]  Metadata containing collection time (state) values for each sample. Must contain exclusively numeric values.": "[必需] 包含每个样本的采集时间（状态）值的元数据。必须只包含数值。",
    "[optional]  Metadata column containing IDs for individual subjects.": "[可选] 包含个体受试者ID的元数据列。",
    "[default: 5]  Number of k-fold cross-validations to perform.": "[默认值：5] 执行 k 折交叉验证的次数。",
    "[default: 1]  Number of jobs to run in parallel.": "[默认：1]  要并行运行的工作数量。",
    "[default: 100]  Number of trees to grow for estimation. More trees will improve predictive accuracy up to a threshold level, but will also increase time and memory requirements. This parameter only affects ensemble estimators, such as Random Forest, AdaBoost, ExtraTrees, and GradientBoosting.": "[默认值：100] 用于估算的树的数量。更多的树将提高预测精度，直到达到一个阈值水平，但也会增加时间和内存需求。这个参数只影响集成估计器，如随机森林、AdaBoost、ExtraTrees和梯度提升。",
    "[default: No]  Automatically tune hyperparameters using random grid search.": "[默认：否] 使用随机网格搜索自动调整超参数。",
    "[optional]  Filter feature table to exclude any features with an importance score less than this threshold. Set to \"q1\", \"q2\", or \"q3\" to select the first, second, or third quartile of values. Set to \"None\" to disable this filter.": "[可选] 过滤特征表以排除任何重要性得分低于此阈值的特征。设置为 \"q1\"、\"q2\" 或 \"q3\" 以选择值的第一个、第二个或第三个四分位数。设置为 \"None\" 以禁用此过滤器。",
    "[required]  Filter feature table to exclude any features with an importance score less than this threshold. Set to \"q1\", \"q2\", or \"q3\" to select the first, second, or third quartile of values. Set to \"None\" to disable this filter.": "[必需] 过滤特征表以排除任何重要性得分低于此阈值的特征。设置为 \"q1\"、\"q2\" 或 \"q3\" 以选择值的第一个、第二个或第三个四分位数。设置为 \"None\" 以禁用此过滤器。",
    "[default: 100]  Filter feature table to include top N most important features. Set to \"all\" to include all features.": "[默认值：100] 过滤特征表以包含最重要的N个特征。设置为“all”以包含所有特征。",
    "[optional]  A value listed in the state_column metadata column against which all other states should be compared. Toggles calculation of static distances instead of first distances (which are calculated if no value is given for baseline). If a \"baseline\" value is provided, sample distances at each state are compared against the baseline state, instead of the previous state. Must be a value listed in the state_column.": "[可选] 在 state_column 元数据列中列出的一个值，所有其他状态都应与之进行比较。切换计算静态距离而不是首次距离（如果没有为基线提供值，则计算首次距离）。如果提供了“基线”值，则将每个状态的样本距离与基线状态进行比较，而不是与前一个状态进行比较。必须是 state_column 中列出的值。",
    "[optional]  Feature table containing metric.": "[可选] 包含度量的特诊表。",
    "[optional]  Dependent variable column name. Must be a column name located in the metadata or feature table files.": "[可选] 因变量列名。必须是元数据或特征表文件中的列名。",
    "[optional]  Comma-separated list (without spaces) of metadata columns to use as independent covariates used to determine mean structure of \"metric\".": "[可选] 逗号分隔的元数据列列表（不含空格），用作独立协变量以确定“指标”的平均结构。",
    "[optional]  Comma-separated list (without spaces) of metadata columns to use as independent covariates used to determine the variance and covariance structure (random effects) of \"metric\". To add a random slope, the same value passed to \"state_column\" should be passed here. A random intercept for each individual is set by default and does not need to be passed here.": "[可选] 以逗号分隔（不含空格）的元数据列列表，用作独立的协变量，用于确定“指标”的方差和协方差结构（随机效应）。要添加一个随机斜率，应该在这里传递与“state_column”相同的值。默认情况下为每个个体设置了随机截距，不需要在这里传递。",
    "[default: No]  Estimate locally weighted scatterplot smoothing. Note that this will eliminate confidence interval plotting.": "[默认值：否] 估计局部加权散点平滑。请注意，这将消除置信区间绘图。",
    "[default: 95]  Size of the confidence interval for the regression estimate.": "[默认：95] 回归估计置信区间的大小。",
    "[optional]  R-style formula to use for model specification. A formula must be used if the \"metric\" parameter is None. Note that the metric and group columns specified in the formula will override metric and group columns that are passed separately as parameters to this method. Formulae will be in the format \"a ~ b + c\", where \"a\" is the metric (dependent variable) and \"b\" and \"c\" are independent covariates. Use \"+\" to add a variable; \"+ a:b\" to add an interaction between variables a and b; \"*\" to include a variable and all interactions; and \"-\" to subtract a particular term (e.g., an interaction term). See https://patsy.readthedocs.io/en/latest/formulas.html for full documentation of valid formula operators. Always enclose formulae in quotes to avoid unpleasant surprises.": "[可选] 用于模型规范的 R 风格公式。如果“metric”参数为 None，则必须使用公式。请注意，公式中指定的度量和组列将覆盖作为参数单独传递给此方法的度量和组列。公式的格式为“a ~ b + c”，其中“a”是度量（因变量），“b”和“c”是独立协变量。使用“+”添加变量；“+ a:b”添加变量 a 和 b 之间的交互；“*”包括变量及其所有交互；“-”减去特定项（例如，交互项）。有关有效公式运算符的完整文档，请参见 https://patsy.readthedocs.io/en/latest/formulas.html。始终用引号括住公式以避免不愉快的意外。",
    "[required]  Numeric metadata column containing sampling time (state) data to use as prediction target.": "[必需] 包含采样时间（状态）数据的数值型元数据列，用作预测目标。",
    "[required]  Categorical metadata column to use for plotting and significance testing between main treatment groups.": "[必需] 用于绘图和主处理组之间显著性测试的分类元数据列。",
    "[required]  Value of group_by to use as control group. The regression model will be trained using only control group data, and the maturity scores of other groups consequently will be assessed relative to this group.": "[必需] 用作对照组的group_by的值。回归模型将仅使用对照组数据进行训练，其他组的成熟度得分将相对于该组进行评估。",
    "[optional]  Optional metadata column containing IDs for individual subjects. Adds individual subject (spaghetti) vectors to volatility charts if a column name is provided.": "[可选] 包含个体主题ID的可选元数据列。如果提供了列名称，则将个体主题（意大利面）向量添加到波动性图表中。",
    "[default: 0.5]  Fraction of input samples to exclude from training set and use for classifier testing.": "[默认值：0.5] 从训练集中排除并用于分类器测试的输入样本的比例。",
    "[default: 0.05]  If optimize_feature_selection is True, step is the percentage of features to remove at each iteration.": "[默认值：0.05] 如果 optimize_feature_selection 是 True，则 step 是每轮迭代中要移除的特征的百分比。",
    "[default: No]  Automatically optimize input feature selection using recursive feature elimination.": "[默认：否] 自动使用递归特征消除来优化输入特征选择。",
    "[default: No]  Evenly stratify training and test data among metadata categories. If True, all values in column must match at least two samples.": "[默认值：否] 在元数据类别中均匀地分层训练和测试数据。如果为真，则列中的所有值至少必须匹配两个样本。",
    "[default: 50]  Filter feature table to include top N most important features. Set to zero to include all features.": "[默认：50] 过滤特征表以包含最重要的N个特征。设置为零以包含所有特征。",
    "[required]  Feature table to use for microbial interdependence test.": "[必需] 用于微生物相互依赖性测试的特征表。",
    "[required]  Metadata column containing state (e.g., Time) across which samples are paired.": "[必需] 包含状态（例如，时间）的元数据列，用于配对样本。",
    "[required]  Baseline state column value.": "[必需] 基线状态列值。",
    "[required]  State column value to pair with baseline.": "[必需] 指定要与基线配对的列值。",
    "[required]  Metadata column containing subject IDs to use for pairing samples. WARNING: if replicates exist for an individual ID at either state_1 or state_2, that subject will be dropped and reported in standard output by default. Set replicate_handling=\"random\" to instead randomly select one member.": "[必需] 包含用于配对样本的主体ID的元数据列。警告：如果在状态1或状态2中存在个体ID的重复样本，则默认情况下该主体将被丢弃，并通过标准输出报告。设置replicate_handling=\"random\"以随机选择一个成员。",
    "[optional]  Feature table to optionally use for paired comparisons.": "[可选] 用于成对比较的可选特征表。",
    "[optional]  Metadata column on which to separate groups for comparison": "[可选] 用于将组进行比较的元数据列",
    "[default: No]  Perform parametric (ANOVA and t-tests) or non-parametric (Kruskal-Wallis, Wilcoxon, and Mann-Whitney U tests) statistical tests.": "[默认值：否] 执行参数化（方差分析和t检验）或非参数化（Kruskal-Wallis、Wilcoxon和Mann-Whitney U检验）的统计测试。",
    "[required]  Metadata column on which to separate groups for comparison": "[必需] 用于将组进行比较的元数据列",
    "[required]  Feature table containing features found in importances.": "[必需] 包含在重要性中找到的特征的特征表。",
    "[required]  Feature importance scores.": "[必需] 特征重要性得分。",
    "[optional]  The default metadata column on which to separate groups for comparison (all categorical metadata columns will be available in the visualization).": "[可选] 用于比较组别分隔的默认元数据列（所有分类元数据列都将在可视化中可用）。",
    "[optional]  Feature table containing metrics.": "[可选] 包含度量指标的特征表。",
    "[optional]  Numeric metadata or artifact column to test by default (all numeric metadata columns will be available in the visualization).": "[可选] 默认情况下要测试的数字元数据或工件列（所有数字元数据列都将在可视化中可用）。",
    "[required]  Numeric metadata column to compute pairwise Euclidean distances from": "[必需] 用于计算成对欧几里得距离的数值元数据列",
    "[required]  Categorical metadata column to shuffle.": "[必需] 要打乱的分类元数据列。",
    "[default: 3]  The number of shuffled metadata columns to create.": "[默认：3] 要创建的打乱的元数据列的数量。",
    "[default: 'shuffled.grouping.']  Prefix to use in naming the shuffled metadata columns.": "[默认: 'shuffled.grouping.']  在命名打乱的元数据列时使用的前缀。",
    "[default: 'fake.group.']  Prefix to use in naming the values in the shuffled metadata columns.": "[默认: 'fake.group.']  在重命名洗牌元数据列的值时使用的前缀。",
    "[required]  The metadata to tabulate.": "[必需] 要制表的元数据。",
    "[default: 100]  The maximum number of Metadata records to display per page": "[默认值：100] 每页显示的元数据记录的最大数量",
    "[required]  The sequences to be used for creating a fasttree based rooted phylogenetic tree.": "[必需] 用于创建基于fasttree的有根系统发育树的序列。",
    "[default: 1]  The number of threads. (Use `auto` to automatically use all available cores) This value is used when aligning the sequences and creating the tree with fasttree.": "[默认值：1] 线程数。（使用 `auto` 可以自动使用所有可用的核心）在对序列进行比对和使用 fasttree 创建树时使用这个值。",
    "[default: 1.0]  The maximum relative frequency of gap characters in a column for the column to be retained. This relative frequency must be a number between 0.0 and 1.0 (inclusive), where 0.0 retains only those columns without gap characters, and 1.0 retains all columns  regardless of gap character frequency. This value is used when masking the aligned sequences.": "[默认值：1.0] 保留一列中间隔字符的最大相对频率。这个相对频率必须是一个介于0.0到1.0之间的数字（包括0.0和1.0），其中0.0只保留那些没有间隔字符的列，而1.0则无论间隔字符频率如何都保留所有列。当掩蔽对齐序列时使用这个值。",
    "[default: 0.4]  The minimum relative frequency of at least one non-gap character in a column for that column to be retained. This relative frequency must be a number between 0.0 and 1.0 (inclusive). For example, if a value of  0.4 is provided, a column will only be retained  if it contains at least one character that is present in at least 40% of the sequences. This value is used when masking the aligned sequences.": "[默认值：0.4] 至少一个非间隔字符在列中的最小相对频率，以便保留该列。这个相对频率必须是一个介于0.0和1.0（包括）之间的数字。例如，如果提供了0.4的值，只有当一列至少包含一个在至少40%的序列中出现的字符时，该列才会被保留。在掩蔽对齐序列时使用这个值。",
    "[default: No]  This flag is required if the number of sequences being aligned are larger than 1000000. Disabled by default.": "[默认值：否] 如果比对的序列数量大于100万，则需要此标志。默认情况下禁用。",
    "[required]  The sequences to be used for creating a iqtree based rooted phylogenetic tree.": "[必需] 用于创建基于IQ-TREE的有根系统发育树的序列。",
    "[default: 1]  The number of threads. (Use 0 to automatically use all available cores This value is used when aligning the sequences and creating the tree with iqtree.": "[默认：1] 线程数。（使用0自动使用所有可用的核心，此值用于对序列进行比对和使用iqtree创建树时。）",
    "[default: No]  Fast search to resemble FastTree.": "[默认：否] 快速搜索以类似于FastTree。",
    "[optional]  Single branch test method. Number of bootstrap replicates to perform an SH-like approximate likelihood ratio test (SH-aLRT). Minimum of 1000 replicates is required.": "[可选] 单分支测试方法。执行类似于自举近似似然比率测试（SH-aLRT）的自举复制次数。最少需要1000次复制。",
    "[optional]  Random number seed for the iqtree parsimony starting tree. This allows you to reproduce tree results. If not supplied then one will be randomly chosen.": "[可选] IQ-TREE软件在使用简约性方法构建起始树时所使用的随机数种子。这允许您重现树的结果。如果未提供，则会随机选择一个。",
    "[optional]  Number of unsuccessful iterations to stop. If not set, program defaults will be used. See IQ-TREE manual for details.": "[可选]  停止不成功的迭代次数。如果没有设置，程序将使用默认值。详细信息请参阅IQ-TREE手册。",
    "[optional]  Perturbation strength for randomized NNI. If not set, program defaults will be used. See IQ-TREE manual for details.": "[可选] 随机NNI的扰动强度。如果未设置，则使用程序默认值。详情请参见IQ-TREE手册。",
    "[default: 1]  The number of threads. (Use `all` to automatically use all available cores. This value is used when aligning the sequences and creating the tree with iqtree.": "[默认值：1] 线程数。（使用`all`可以自动使用所有可用的核心。当使用iqtree对序列进行比对和创建树时，会使用这个值。）",
    "[default: No]  This flag is required if the number of sequences being aligned are larger than 1000000. Disabled by default. NOTE: if using this option, it is recomended that only the CAT-based substitution models of RAxML be considered for this pipeline.": "[默认：否] 如果正在比对的序列数量大于100万，则需要此标志。默认情况下处于禁用状态。注意：如果使用此选项，建议仅考虑RAxML的基于CAT的替代模型用于此管道。",
    "[optional]  Random number seed for the parsimony starting tree. This allows you to reproduce tree results. If not supplied then one will be randomly chosen.": "[可选] 用于启发式起始树的随机数种子。这允许您重现树的结果。如果没有提供，则将随机选择一个。",
    "[required]  Aligned sequences to be used for phylogenetic reconstruction.": "[必需] 用于系统发育重建的对齐序列。",
    "[default: 1]  The number of threads. Using more than one thread runs the non-deterministic variant of `FastTree` (`FastTreeMP`), and may result in a different tree than single-threading. See http://www.microbesonline.org/fasttree/#OpenMP for details. (Use `auto` to automatically use all available cores)": "[默认值：1] 线程数。使用超过一个线程将运行非确定性变体的`FastTree`（`FastTreeMP`），并且可能会得到与单线程不同的树。详细信息请参见 http://www.microbesonline.org/fasttree/#OpenMP。（使用`auto`可以自动使用所有可用的核心）",
    "[required]  Feature table that features should be filtered from.": "[必需] 特征表，应从中过滤特征。",
    "[required]  Tree where tip identifiers are the feature identifiers that should be retained in the table.": "[必需] 树，其中末端标识符是应在表中保留的特征标识符。",
    "[required]  Tree that should be filtered": "[必需] 应该被过滤的树",
    "[optional]  Feature table which contains the identifier that should be retained in the tree": "[可选] 包含应保留在树中的标识符的特征表",
    "[optional]  Feature metadata to use with the 'where' statement or to select tips to be retained. Metadata objects could also include FeatureData[Sequence] data types, if, forinstance, you want to filter to match represenative sequencces.": "[可选] 用于 'where' 语句或选择要保留的末端的特征元数据。元数据对象还可以包括 FeatureData[Sequence] 数据类型，例如，如果您想过滤以匹配代表性序列。",
    "[optional]  Random number seed. If not set, program defaults will be used. See IQ-TREE manual for details.": "[可选] 随机数种子。如果未设置，程序将使用默认值。有关详细信息，请参见IQ-TREE手册。",
    "[default: 1]  The number of cores to use for parallel processing. Use `auto` to let IQ-TREE automatically determine the optimal number of cores to use.": "[默认：1] 用于并行处理的核心数。使用 `auto` 让 IQ-TREE 自动确定使用的最佳核心数。",
    "[optional]  Limits the maximum number of cores to be used when 'n_cores' is set to 'auto'.": "[可选] 当 'n_cores' 设置为 'auto' 时，限制使用的最大核心数。",
    "[default: 1]  Number of indepedent runs. Multiple  independent runs (e.g. 10) can outperform a single run in terms of likelihood maximisation.": "[默认：1] 独立运行的次数。多次独立运行（例如10次）在似然最大化方面可以优于单次运行。",
    "[optional]  Number of initial parsimony trees. If not set, program defaults will be used. See IQ-TREE manual for details.": "[可选] 初始简约树的数量。如果未设置，程序将使用默认值。详细信息请参阅IQ-TREE手册。",
    "[optional]  Number of top initial trees. If not set, program defaults will be used. See IQ-TREE manual for details.": "[可选] 最初树的数量。如果没有设置，程序将使用默认值。详细信息请参阅IQ-TREE手册。",
    "[optional]  Number of best trees retained during search. If not set, program defaults will be used. See IQ-TREE manual for details.": "[可选] 在搜索过程中保留的最佳树的数量。如果没有设置，程序将使用默认值。详见IQ-TREE手册以获取详细信息。",
    "[optional]  Fix number of iterations to stop. If not set, program defaults will be used. See IQ-TREE manual for details.": "[可选] 固定停止迭代的次数。如果未设置，则会使用程序默认值。详细信息请参见IQ-TREE手册。",
    "[optional]  Radius for parsimony SPR search. If not set, program defaults will be used. See IQ-TREE manual for details.": "[可选] 用于简约性SPR搜索的半径。如果未设置，则会使用程序默认值。有关详细信息，请参阅IQ-TREE手册。",
    "[default: No]  Perform more thorough NNI search.": "[默认值：否] 执行更彻底的最近邻插值搜索。",
    "[optional]  Single branch test method. Number of bootstrap replicates to perform an SH-like approximate likelihood ratio test (SH-aLRT). Minimum of 1000 replicates is required. Can be used with other 'single branch test methods'. Values reported in the order of: alrt, lbp, abayes.": "[可选] 单分支测试方法。执行类似SH的近似似然比率测试（SH-aLRT）的bootstrap重复次数。最少需要1000次重复。可以与其他“单分支测试方法”一起使用。报告的值的顺序为：alrt、lbp、abayes。",
    "[default: No]  Single branch test method. Approximate Bayes test. Can be used with other 'single branch test methods'. Values reported in the order of: alrt, lbp, abayes.": "[默认：否]  单分支测试方法。近似贝叶斯测试。可以与其他“单分支测试方法”一起使用。报告的值的顺序为：alrt，lbp，abayes。",
    "[optional]  Single branch test method. Number of bootstrap replicates to perform a fast local bootstrap probability method. Minimum of 1000 replicates is required. Can be used with other 'single branch test methods'. Values reported in the order of: alrt, lbp, abayes.": "[可选] 单分支测试方法。执行快速局部引导概率方法的引导重复次数。至少需要1000次重复。可以与其他“单分支测试方法”一起使用。报告的值的顺序为：alrt、lbp、abayes。",
    "[default: No]  Safe likelihood kernel to avoid numerical underflow.": "[默认：否] 安全似然核以避免数值下溢。",
    "[default: 1000]  The number of bootstrap searches to perform. Minimum of 1000 recomended.": "[默认：1000] 要执行的自举搜索次数。建议最少1000次。",
    "[optional]  Maximum number of iterations. If not set, program defaults will be used. See IQ-TREE manual for details.": "[可选] 最大迭代次数。如果未设置，则使用程序默认值。详见IQ-TREE手册以获取详细信息。",
    "[optional]  Number of iterations for UFBoot stopping rule. If not set, program defaults will be used.See IQ-TREE manual for details.": "[可选] UFBoot停止规则的迭代次数。如果未设置，则会使用程序默认值。详细信息请参见IQ-TREE手册。",
    "[optional]  Minimum correlation coefficient. If not set, program defaults will be used.See IQ-TREE manual for details.": "[可选] 最小相关系数。如果没有设置，程序将使用默认值。详情请参见IQ-TREE手册。",
    "[optional]  Epsilon value to break tie. If not set, program defaults will be used. See IQ-TREE manual for details.": "[可选] 用于打破平局的Epsilon值。如果没有设置，程序将使用默认值。详情请参阅IQ-TREE手册。",
    "[default: No]  Single branch test method. Performs an approximate Bayes test. Can be used with other 'single branch test methods' and ultrafast bootstrap. Values reported in the order of: alrt, lbp, abayes, ufboot.": "[默认：否] 单分支测试方法。执行一个近似贝叶斯测试。可以与其他“单分支测试方法”和超快速引导一起使用。报告的值的顺序为：alrt，lbp，abayes，ufboot。",
    "[optional]  Single branch test method. Number of bootstrap replicates to perform a fast local bootstrap probability method. Minimum of 1000 replicates is required. Can be used with other 'single branch test methods'. Values reported in the order of: alrt, lbp, abayes, ufboot.": "[可选] 单分支测试方法。执行快速局部引导概率方法的引导重复次数。至少需要1000次重复。可以与其他“单分支测试方法”一起使用。报告的值的顺序为：alrt，lbp，abayes，ufboot。",
    "[default: No]  Optimize UFBoot trees by NNI on bootstrap alignment. This option reduces the risk of overestimating branch supports with UFBoot due to severe model violations.": "[默认值：否] 在自助对齐上通过最近邻插值（NNI）优化UFBoot树。这个选项降低了由于严重模型违规而导致UFBoot过高估计分支支持的风险。",
    "[required]  The phylogenetic tree to be rooted.": "[必需] 要进行根定位的系统发育树。",
    "[default: 1]  The number of independent maximum likelihood searches to perform. The single best scoring tree is returned.": "[默认值：1] 要执行的独立最大似然搜索的数量。返回单个得分最高的树。",
    "[default: 1]  The number of threads to use for multithreaded processing. Using more than one thread will enable the PTHREADS version of RAxML.": "[默认值：1] 用于多线程处理的线程数。使用超过一个线程将启用RAxML的PTHREADS版本。",
    "[optional]  Specify a random seed for rapid bootstrapping. This allows you to reproduce rapid bootstrap results. If not supplied then one will be randomly chosen.": "[可选] 指定一个随机种子以快速引导。这允许你重现快速引导的结果。如果没有提供，则将随机选择一个。",
    "[default: 100]  The number of bootstrap searches to perform.": "[默认：100] 要执行的自举搜索次数。",
    "[required]  Phylogenetic trees to compare with Robinson-Foulds. Rooting information and branch lengths are ignored by this metric.": "[必需] 用于与Robinson-Foulds比较的系统发育树。这个指标忽略了根信息和分支长度。",
    "[optional]  Labels to use for the tree names in the distance matrix. If ommited, labels will be \"tree_n\" where \"n\" ranges from 1..N. The number of labels must match the number of trees.": "[可选] 用于距离矩阵中树名称的标签。如果省略，标签将是“tree_n”，其中“n”的范围是从1到N。标签的数量必须与树的数量相匹配。",
    "[required]  Reference sequences used to build bowtie2 index.": "[必需] 用于构建bowtie2索引的参考序列。",
    "[default: 1]  Number of threads to launch": "[默认值：1]  启动的线程数",
    "[required]  Expected feature compositions": "[必需] 预期的特征组成",
    "[required]  Observed feature compositions": "[必需] 观察到的特征组成",
    "[default: 7]  Maximum depth of semicolon-delimited taxonomic ranks to test (e.g., 1 = root, 7 = species for the greengenes reference sequence database).": "[默认值：7] 测试分号分隔的分类等级的最大深度（例如，1 = 根，7 = 物种，针对greengenes参考序列数据库）。",
    "[default: Yes]  Plot taxon accuracy rate (TAR) on score plot. TAR is the number of true positive features divided by the total number of observed features (TAR = true positives / (true positives + false positives)).": "[默认：是] 在得分图上绘制分类群准确率（TAR）。TAR 是真正例特征的数量除以观察到的特征总数（TAR = 真正例 / （真正例 + 假正例））。",
    "[default: Yes]  Plot taxon detection rate (TDR) on score plot. TDR is the number of true positive features divided by the total number of expected features (TDR = true positives / (true positives + false negatives)).": "[默认：是] 在得分图上绘制分类群检测率（TDR）。TDR 是真正例特征数除以预期特征总数的比例（TDR = 真正例 /（真正例 + 假负例））。",
    "[default: No]  Plot expected vs. observed linear regression r value on score plot.": "[默认：否] 在得分图上绘制预期与观察到的线性回归 r 值。",
    "[default: Yes]  Plot expected vs. observed linear regression r-squared value on score plot.": "[默认：是] 在得分图上绘制预期与观察到的线性回归 R 平方值。",
    "[default: No]  Plot expected vs. observed Bray-Curtis dissimilarity scores on score plot.": "[默认：否] 在得分图上绘制预期与观察到的Bray-Curtis差异性得分。",
    "[default: No]  Plot expected vs. observed Jaccard distances scores on score plot.": "[默认：否] 在得分图上绘制预期与观察到的Jaccard距离得分。",
    "[default: No]  Plot observed features count on score plot.": "[默认值：否] 在得分图上绘制观测到的特征计数。",
    "[default: Yes]  Plot ratio of observed:expected features on score plot.": "[默认: 是] 在得分图上绘制观察到的特征与期望特征的比率。",
    "[optional]  Optional sample metadata that maps observed_features sample IDs to expected_features sample IDs.": "[可选] 可选的样本元数据，将 observed_features 样本 ID 映射到 expected_features 样本 ID。",
    "[required]  Sequences to test for exclusion": "[必需] 用于测试排除的序列",
    "[required]  Reference sequences to align against feature sequences": "[必需] 用于与特征序列比对的参考序列",
    "[default: No]  Option to plot pairwise alignments of query sequences and their top hits.": "[默认值：否] 绘制查询序列及其顶级命中的配对比对的选项。",
    "[required]  Expected taxonomic assignments": "[必需] 预期的分类学分配",
    "[required]  Observed taxonomic assignments": "[必需] 观察到的分类学分配",
    "[required]  Maximum depth of semicolon-delimited taxonomic ranks to test (e.g., 1 = root, 7 = species for the greengenes reference sequence database).": "[必需] 测试分号分隔的分类等级的最大深度（例如，1 = 根，7 = 物种，针对greengenes参考序列数据库）。",
    "[optional]  Optional feature table containing relative frequency of each feature, used to weight accuracy scores by frequency. Must contain all features found in expected and/or observed taxa. Features found in the table but not the expected/observed taxa will be dropped prior to analysis.": "[可选] 包含每个特征的相对频率的可选特征表，用于按频率加权准确度分数。必须包含在预期和/或观察到的分类群中发现的所有特征。在表中但在预期/观察到的分类群中未找到的特征将在分析之前被删除。",
    "[default: Yes]  Require that all features found in observed taxa must be found in expected taxa or raise error.": "[默认: 是] 要求在观察到的分类群中发现的所有特征都必须在预期的分类群中找到，否则会报错。",
    "[default: Yes]  Require that all features found in expected taxa must be found in observed taxa or raise error.": "[默认值：是] 要求所有在预期分类群中找到的特征也必须在观察到的分类群中找到，否则就报错。",
    "[optional]  Optional sample ID to use for extracting frequency data from feature table, and for labeling accuracy results. If no sample_id is provided, feature frequencies are derived from the sum of all samples present in the feature table.": "[可选] 用于从特征表中提取频率数据以及标记准确性结果的可选样本ID。如果没有提供sample_id，则特征频率是从特征表中存在的所有样本的总和中派生出来的。",
    "[default: 'blast']  Alignment method to use for matching feature sequences against reference sequences": "[默认：'blast']  用于将特征序列与参考序列匹配的比对方法",
    "[default: 0.97]  Reject match if percent identity to reference is lower. Must be in range [0.0, 1.0]": "[默认：0.97] 如果与参考的相似性百分比较低，则拒绝匹配。必须在范围[0.0, 1.0]内。",
    "[optional]  BLAST expectation (E) value threshold for saving hits. Reject if E value is higher than threshold. This threshold is disabled by default.": "[可选]  保存命中的BLAST期望值（E）阈值。如果E值高于阈值，则拒绝。默认情况下，此阈值被禁用。",
    "[default: 0.97]  Percent of query sequence that must align to reference in order to be accepted as a hit.": "[默认值：0.97] 为了被接受为一个命中，查询序列必须与参考序列对齐的百分比。",
    "[default: 1]  Number of jobs to execute. Only applies to vsearch method.": "[默认值：1] 要执行的作业数。仅适用于vsearch方法。",
    "[default: No]  Reject match if the pairwise alignment begins with gaps": "[默认值：否] 如果配对比对以空缺开始，则拒绝匹配",
    "[required]  The sequences to be trimmed.": "[必需] 需要修剪的序列。",
    "[required]  Bowtie2 indexed database.": "[必需] Bowtie2索引数据库。",
    "[default: 1]  Number of alignment threads to launch.": "[默认：1]  启动的比对线程数。",
    "[default: 5]  Reference gap open penalty.": "[默认：5] 参考间隙开放惩罚。",
    "[default: 3]  Reference gap extend penalty.": "[默认：3] 参考间隙延伸惩罚。",
    "[default: Yes]  Exclude sequences that align to reference. Set this option to False to exclude sequences that do not align to the reference database.": "[默认值：是] 排除与参考序列对齐的序列。将此选项设置为False以排除不与参考数据库对齐的序列。",
    "[required]  The demultiplexed sequence data to be quality filtered.": "[必需] 需要进行质量过滤的去多重序列数据。",
    "[default: 4]  The minimum acceptable PHRED score. All PHRED scores less that this value are considered to be low PHRED scores.": "[默认：4] 可接受的最低PHRED分数。所有低于这个值的PHRED分数都被认为是低PHRED分数。",
    "[default: 3]  The maximum number of low PHRED scores that can be observed in direct succession before truncating a sequence read.": "[默认：3] 在截断序列读取之前，可以直接连续观察到的低PHRED分数的最大数量。",
    "[default: 0.75]  The minimum length that a sequence read can be following truncation and still be retained. This length should be provided as a fraction of the input sequence length.": "[默认：0.75] 序列读取在截断后可以保留的最小长度。这个长度应该作为输入序列长度的分数提供。",
    "[default: 0]  The maximum number of ambiguous (i.e., N) base calls. This is applied after trimming sequences based on `min_length_fraction`.": "[默认值：0] 模糊（即，N）碱基调用的最大数量。这在根据 `min_length_fraction` 修剪序列后应用。",
    "[required]  Categorical metadata column to use as prediction target.": "[必需] 用作预测目标的分类元数据列。",
    "[default: 0.2]  Fraction of input samples to exclude from training set and use for classifier testing.": "[默认值：0.2] 从训练集中排除并用于分类器测试的输入样本的比例。",
    "[required]  a distance matrix": "[必需] 一个距离矩阵",
    "[default: 1]  Number of nearest neighbors": "[默认值：1] 最近邻的数量",
    "[required]  Predicted values to plot on x axis. Should be predictions of categorical data produced by a sample classifier.": "[必需] 预测值用于绘制 x 轴。应该是由样本分类器产生的分类数据的预测。",
    "[required]  Metadata column (true values) to plot on y axis.": "[必需] 用于在y轴上绘制的元数据列（真实值）。",
    "[optional]  Predicted class probabilities for each input sample.": "[可选] 每个输入样本预测的类别概率。",
    "[default: 'auto']  The minimum value to use for anchoring the colormap. If \"auto\", vmin is set to the minimum value in the data.": "[默认: 'auto'] 用于锚定颜色图的最小值。如果为\"auto\"，则将vmin设置为数据中的最小值。",
    "[required]  The minimum value to use for anchoring the colormap. If \"auto\", vmin is set to the minimum value in the data.": "[必需] 用于锚定颜色图的最小值。如果为“auto”，则将vmin设置为数据中的最小值。",
    "[default: 'auto']  The maximum value to use for anchoring the colormap. If \"auto\", vmax is set to the maximum value in the data.": "[默认值: 'auto'] 用于锚定颜色图的最大值。如果为\"auto\"，则将vmax设置为数据中的最大值。",
    "[required]  The maximum value to use for anchoring the colormap. If \"auto\", vmax is set to the maximum value in the data.": "[必需] 用于锚定色图的最大值。如果为“auto”，则将vmax设置为数据中的最大值。",
    "[required]  Numeric metadata column to use as prediction target.": "[必需] 用作预测目标的数值元数据列。",
    "[required]  Feature importances.": "[必需] 特征重要性。",
    "[optional]  Sample metadata column to use for sample labeling or grouping.": "[可选] 用于样本标记或分组的样本元数据列。",
    "[optional]  Feature metadata (e.g., taxonomy) to use for labeling features in the heatmap.": "[可选] 用于热图中标记特征的功能元数据（例如，分类学信息）。",
    "[default: 0]  Filter feature table to exclude any features with an importance score less than this threshold. Set to zero to include all features.": "[默认：0] 过滤特征表以排除任何重要性得分低于此阈值的特征。设置为零以包含所有特征。",
    "[default: No]  Group samples by sample metadata.": "[默认值：否]  根据样本元数据对样本进行分组。",
    "[required]  Metadata file to convert to feature table.": "[必需] 要转换为特征表的元数据文件。",
    "[optional]  Feature table containing all features that should be used for target prediction.": "[可选] 包含所有用于目标预测的特征的表格。",
    "[default: No]  If True, columns that contain a unique value for every ID will be dropped.": "[默认值：否] 如果为真，包含每个ID唯一值的列将被删除。",
    "[required]  Sample classifier trained with fit_classifier.": "[必需] 使用 fit_classifier 训练的样本分类器。",
    "[required]  Sample regressor trained with fit_regressor.": "[必需] 使用 fit_regressor 训练的样本回归器。",
    "[required]  Predicted values to plot on y axis. Must be predictions of numeric data produced by a sample regressor.": "[必需] 要在 y 轴上绘制的预测值。必须是样本回归器产生的数值数据的预测。",
    "[required]  Metadata column (true values) to plot on x axis.": "[必需] 用于在x轴上绘制的元数据列（真实值）。",
    "[default: Yes]  Evenly stratify training and test data among metadata categories. If True, all values in column must match at least two samples.": "[默认值：是] 在元数据类别中均匀地分层训练和测试数据。如果为真，那么列中的所有值至少必须匹配两个样本。",
    "[required]  Sample estimator trained with fit_classifier or fit_regressor.": "[必需] 使用fit_classifier或fit_regressor训练的样本估计器。",
    "[required]  Feature table to visualize at various taxonomic levels.": "[必需] 用于在不同分类水平上可视化的特征表。",
    "[optional]  Taxonomic annotations for features in the provided feature table. All features in the feature table must have a corresponding taxonomic annotation. Taxonomic annotations that are not present in the feature table will be ignored. If no taxonomy is provided, the feature IDs will be used as labels.": "[可选] 提供的特征表中特征的分类注释。特征表中的所有特征都必须有一个相应的分类注释。不在特征表中出现的分类注释将被忽略。如果没有提供分类信息，特征ID将被用作标签。",
    "[optional]  Attempt to parse hierarchical taxonomic information from feature IDs by separating levels with this character. This parameter is ignored if a taxonomy is provided as input.": "[可选] 尝试通过使用此字符分隔级别来解析特征ID中的层次分类信息。如果输入提供了分类，此参数将被忽略。",
    "[required]  Feature table to be collapsed.": "[必需] 需要合并的特征表。",
    "[required]  Taxonomic annotations for features in the provided feature table. All features in the feature table must have a corresponding taxonomic annotation. Taxonomic annotations that are not present in the feature table will be ignored.": "[必需] 提供的特征表中特征的分类注释。特征表中的所有特征都必须有相应的分类注释。不在特征表中的分类注释将被忽略。",
    "[required]  The taxonomic level at which the features should be collapsed. All ouput features will have exactly this many levels of taxonomic annotation.": "[必需] 特征应该合并的分类级别。所有输出特征将具有完全相同数量的分类注释级别。",
    "[required]  Feature sequences to be filtered.": "[必需] 待过滤的特征序列。",
    "[required]  Taxonomic annotations for features in the provided feature sequences. All features in the feature sequences must have a corresponding taxonomic annotation. Taxonomic annotations for features that are not present in the feature sequences will be ignored.": "[必需] 提供的特征序列中特征的分类注释。特征序列中的所有特征都必须有一个相应的分类注释。不在特征序列中的特征的分类注释将被忽略。",
    "[optional]  One or more search terms that indicate which taxa should be included in the resulting sequences. If providing more than one term, terms should be delimited by the query-delimiter character. By default, all taxa will be included.": "[可选] 一个或多个搜索词，用于指示结果序列中应包含哪些分类群。如果提供多个词，应使用查询分隔符字符分隔。默认情况下，将包含所有分类群。",
    "[optional]  One or more search terms that indicate which taxa should be excluded from the resulting sequences. If providing more than one term, terms should be delimited by the query-delimiter character. By default, no taxa will be excluded.": "[可选] 一个或多个搜索词，用于指示应从结果序列中排除哪些分类群。如果提供多个词，词之间应以查询分隔符字符分隔。默认情况下，不会排除任何分类群。",
    "[default: ',']  The string used to delimit multiple search terms provided to include or exclude. This parameter should only need to be modified if the default delimiter (a comma) is used in the provided taxonomic annotations.": "[默认：','] 用于分隔提供给包含或排除的多个搜索词的字符串。如果提供的分类注释中使用了默认分隔符（逗号），则可能需要修改此参数。",
    "[required]  Feature table to be filtered.": "[必需] 要被过滤的特征表。",
    "[required]  Taxonomic annotations for features in the provided feature table. All features in the feature table must have a corresponding taxonomic annotation. Taxonomic annotations for features that are not present in the feature table will be ignored.": "[必需] 提供的特征表中特征的分类注释。特征表中的所有特征都必须有相应的分类注释。不在特征表中的特征的分类注释将被忽略。",
    "[optional]  One or more search terms that indicate which taxa should be included in the resulting table. If providing more than one term, terms should be delimited by the query-delimiter character. By default, all taxa will be included.": "[可选] 一个或多个搜索词，表明哪些分类群应该包含在结果表中。如果提供多个词，词之间应该用查询分隔符字符分隔。默认情况下，将包含所有分类群。",
    "[optional]  One or more search terms that indicate which taxa should be excluded from the resulting table. If providing more than one term, terms should be delimited by the query-delimiter character. By default, no taxa will be excluded.": "[可选] 一个或多个搜索词，用于指示哪些分类群应该从结果表中排除。如果提供多个词，词之间应该用查询分隔符字符分隔。默认情况下，不会排除任何分类群。",
    "[required]  The sequences corresponding to the features in table.": "[必需] 与表中特征相对应的序列。",
    "[required]  The feature table to be clustered.": "[必需] 要进行聚类的特征表。",
    "[required]  The sequences to use as cluster centroids.": "[必需] 用作聚类中心的序列。",
    "[required]  The percent identity at which clustering should be performed. This parameter maps to vsearch's --id parameter.": "[必需] 聚类操作应执行的相似度百分比。此参数映射到vsearch的--id参数。",
    "[default: 1]  The number of threads to use for computation. Passing 0 will launch one thread per CPU core.": "[默认：1] 用于计算的线程数。传递0将为每个CPU核心启动一个线程。",
    "[required]  The sequences to be dereplicated.": "[必需] 需要去重的序列。",
    "[default: No]  Merge sequences with identical prefixes. If a sequence is identical to the prefix of two or more longer sequences, it is clustered with the shortest of them. If they are equally long, it is clustered with the most abundant.": "[默认值：否] 合并具有相同前缀的序列。如果一个序列与两个或更多更长序列的前缀相同，它将与最短的序列聚类在一起。如果它们长度相同，则与最丰富的序列聚类在一起。",
    "[default: 1]  Discard sequences shorter than this integer.": "[默认：1]  丢弃短于这个整数的序列。",
    "[default: 1]  Discard sequences with a post-dereplication abundance value smaller than integer.": "[默认值：1]  丢弃去重后丰度值小于整数的序列。",
    "[required]  Fastq sequences": "[必需] 快速q序列",
    "[default: 1]  The number of threads used for computation.": "[默认：1] 用于计算的线程数。",
    "[required]  The demultiplexed paired-end sequences to be merged.": "[必需] 需要合并的去多重化的配对末端序列。",
    "[optional]  Truncate sequences at the first base with the specified quality score value or lower.": "[可选] 在具有指定质量得分值或更低的第一个碱基处截断序列。",
    "[default: 1]  Sequences shorter than minlen after truncation are discarded.": "[默认: 1] 截断后短于minlen的序列将被丢弃。",
    "[optional]  Sequences with more than maxns N characters are discarded.": "[可选] 含有超过maxns N个字符的序列将被丢弃。",
    "[default: No]  Allow merging of staggered read pairs.": "[默认: 否]  允许合并错位的配对读取。",
    "[default: 10]  Minimum length of the area of overlap between reads during merging.": "[默认值：10] 在合并过程中，读取重叠区域的最小长度。",
    "[default: 10]  Maximum number of mismatches in the area of overlap during merging.": "[默认：10] 在合并过程中重叠区域中不匹配的最大数量。",
    "[optional]  Minimum length of the merged read to be retained.": "[可选] 保留合并后的读取的最小长度。",
    "[optional]  Maximum length of the merged read to be retained.": "[可选] 要保留的合并后的读取的最大长度。",
    "[optional]  Maximum number of expected errors in the merged read to be retained.": "[可选] 保留合并后的读取中预期错误的最大数量。",
    "[default: 1]  The number of threads to use for computation. Does not scale much past 4 threads.": "[默认：1] 用于计算的线程数。超过4个线程后扩展性不高。",
    "[required]  The feature sequences to be chimera-checked.": "[必需] 需要进行嵌合体检查的特征序列。",
    "[required]  Feature table (used for computing total feature abundances).": "[必需] 特征表（用于计算总特征丰度）。",
    "[default: 1.4]  No vote pseudo-count, corresponding to the parameter n in the chimera scoring function.": "[默认值：1.4] 没有投票的伪计数，对应于嵌合体评分函数中的参数n。",
    "[default: 3]  Minimum number of differences per segment.": "[默认：3] 每个片段的差异最小数量。",
    "[default: 0.8]  Minimum divergence from closest parent.": "[默认：0.8]  与最近亲本的最小差异。",
    "[default: 0.28]  Minimum score (h). Increasing this value tends to reduce the number of false positives and to decrease sensitivity.": "[默认值：0.28] 最小得分（h）。增加此值倾向于减少假阳性数量并降低敏感性。",
    "[default: 8.0]  No vote weight, corresponding to the parameter beta in the scoring function.": "[默认: 8.0] 无投票权重，对应于评分函数中的参数beta。",
    "[required]  The non-chimeric reference sequences.": "[必需] 非嵌合的参考序列。",
    "Filename to import the data as. Must match regex: .+(?‹!\\.rev)\\.1\\.bt2l?": "要导入数据的文件名。必须匹配正则表达式：.+(?‹!\\.rev)\\.1\\.bt2l?",
    "This data should be formatted as a Bowtie2IndexFileFormat. See the documentation below for more information.": "该数据应格式化为Bowtie2索引文件格式。有关更多信息，请参见下面的文档。",
    "Filename to import the data as. Must match regex: .+(?‹!\\.rev)\\.2\\.bt2l?": "要作为导入数据的文件名。必须匹配正则表达式：.+(?‹!\\.rev)\\.2\\.bt2l?",
    "Filename to import the data as. Must match regex: .+\\.3\\.bt2l?": "要导入数据的文件名。必须匹配正则表达式：.+\\.3\\.bt2l?",
    "Filename to import the data as. Must match regex: .+\\.4\\.bt2l?": "要导入数据的文件名。必须匹配正则表达式：.+\\.4\\.bt2l?",
    "Filename to import the data as. Must match regex: .+\\.rev\\.1\\.bt2l?": "要导入数据的文件名。必须匹配正则表达式：.+\\.rev\\.1\\.bt2l?",
    "Filename to import the data as. Must match regex: .+\\.rev\\.2\\.bt2l?": "要导入数据的文件名。必须匹配正则表达式：.+\\.rev\\.2\\.bt2l?",
    "This data should be formatted as a DeblurStatsFmt. See the documentation below for more information.": "这些数据应该格式化为DeblurStatsFmt格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a LSMatFormat. See the documentation below for more information.": "这些数据应该格式化为LSMatFormat。有关更多信息，请参见下面的文档。",
    "Filename to import the data as. You shouldn't need to change this unless something is wrong.": "要导入数据的文件名。除非有问题，否则您不需要更改此设置。",
    "This data should be formatted as a FastqGzFormat. See the documentation below for more information.": "这些数据应该格式化为FastqGz格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a ErrorCorrectionDetailsFmt. See the documentation below for more information.": "这些数据应该被格式化为 ErrorCorrectionDetailsFmt。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a AlignedProteinFASTAFormat. See the documentation below for more information.": "这些数据应该格式化为对齐蛋白FASTA格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a AlignedRNAFASTAFormat. See the documentation below for more information.": "这些数据应该格式化为对齐的RNAFASTA格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a MixedCaseAlignedRNAFASTAFormat. See the documentation below for more information.": "这些数据应该被格式化为混合大小写对齐的RNA FASTA格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a AlignedDNAFASTAFormat. See the documentation below for more information.": "这些数据应该格式化为对齐的DNA FASTA格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a MixedCaseAlignedDNAFASTAFormat. See the documentation below for more information.": "这些数据应该格式化为混合大小写对齐的DNA FASTA格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a BLAST6Format. See the documentation below for more information.": "这些数据应该格式化为BLAST6格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a DecontamScoreFormat. See the documentation below for more information.": "这些数据应该格式化为DecontamScoreFormat格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a FrictionlessCSVFileFormat. See the documentation below for more information. Elements must match regex: .+\\.csv": "这些数据应该格式化为FrictionlessCSVFileFormat。有关更多信息，请参见下面的文档。元素必须匹配正则表达式：.+\\.csv",
    "This is needed if your element identifiers lack one.": "如果元素标识符缺少一个，则需要这样做。",
    "Filename to import the data as. Must match regex: .+\\.csv": "要导入数据的文件名。必须匹配正则表达式：.+\\.csv",
    "This data should be formatted as a FrictionlessCSVFileFormat. See the documentation below for more information.": "这些数据应该格式化为FrictionlessCSV文件格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a DataPackageSchemaFileFormat. See the documentation below for more information.": "这些数据应该格式化为 DataPackageSchemaFileFormat。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a DifferentialFormat. See the documentation below for more information.": "这些数据应该格式化为差异格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a ImportanceFormat. See the documentation below for more information.": "这些数据应该被格式化为重要性格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a RNAFASTAFormat. See the documentation below for more information.": "这些数据应该格式化为RNAFASTA格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a DNAFASTAFormat. See the documentation below for more information.": "这些数据应该格式化为DNAFASTA格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a ProteinFASTAFormat. See the documentation below for more information.": "这些数据应该格式化为蛋白质FASTA格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a MixedCaseRNAFASTAFormat. See the documentation below for more information.": "这些数据应该格式化为混合大小写 RNAFASTA 格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a MixedCaseDNAFASTAFormat. See the documentation below for more information.": "这些数据应该被格式化为混合大小写DNA的FASTA格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a BIOMV210Format. See the documentation below for more information.": "这些数据应该格式化为BIOMV210格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a HeaderlessTSVTaxonomyFormat. See the documentation below for more information.": "这些数据应该格式化为无头TSV分类格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a TSVTaxonomyFormat. See the documentation below for more information.": "这些数据应该格式化为TSV分类体系格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a BIOMV100Format. See the documentation below for more information.": "这些数据应该格式化为BIO-MV100格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a NewickFormat. See the documentation below for more information.": "这些数据应该格式化为Newick格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a ImmutableMetadataFormat. See the documentation below for more information.": "这些数据应该格式化为 ImmutableMetadataFormat。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a QualFormat. See the documentation below for more information.": "这些数据应该格式化为QualFormat。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a OrdinationFormat. See the documentation below for more information.": "这些数据应该被格式化为一个“排序格式”。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a PlacementsFormat. See the documentation below for more information.": "这些数据应该格式化为“PlacementsFormat”。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a ProcrustesStatisticsFmt. See the documentation below for more information.": "这些数据应该被格式化为ProcrustesStatisticsFmt格式。更多信息请参见下面的文档。",
    "This data should be formatted as a QualityFilterStatsFmt. See the documentation below for more information.": "这些数据应该格式化为QualityFilterStatsFmt。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a AlphaDiversityFormat. See the documentation below for more information.": "这些数据应该格式化为 AlphaDiversityFormat。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a ArtificialGroupingFormat. See the documentation below for more information.": "这些数据应该格式化为“人工分组格式”。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a BooleanSeriesFormat. See the documentation below for more information.": "这些数据应该被格式化为BooleanSeriesFormat。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a PredictionsFormat. See the documentation below for more information.": "这些数据应该格式化为PredictionsFormat。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a DADA2StatsFormat. See the documentation below for more information.": "这些数据应该被格式化为 DADA2StatsFormat。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a FirstDifferencesFormat. See the documentation below for more information.": "这些数据应该格式化为FirstDifferencesFormat。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a FastqGzFormat. See the documentation below for more information. Elements must match regex: .+_.+_R[12]_001\\.fastq\\.gz": "这些数据应该格式化为FastqGz格式。有关更多信息，请参见下面的文档。元素必须匹配正则表达式：.+_.+_R[12]_001\\.fastq\\.gz",
    "Filename to import the data as. Must match regex: .+_.+_R[12]_001\\.fastq\\.gz": "要导入数据的文件名。必须匹配正则表达式：.+_.+_R[12]_001\\.fastq\\.gz",
    "This data should be formatted as a FastqGzFormat. See the documentation below for more information. Elements must match regex: .+_.+_L[0-9][0-9][0-9]_R[12]_001\\.fastq\\.gz": "这些数据应该格式化为FastqGzFormat。有关更多信息，请参见下面的文档。元素必须匹配正则表达式：.+_.+_L[0-9][0-9][0-9]_R[12]_001\\.fastq\\.gz",
    "Filename to import the data as. Must match regex: .+_.+_L[0-9][0-9][0-9]_R[12]_001\\.fastq\\.gz": "要导入数据的文件名。必须匹配正则表达式：.+_.+_L[0-9][0-9][0-9]_R[12]_001\\.fastq\\.gz",
    "This data should be formatted as a FastqGzFormat. See the documentation below for more information. Elements must match regex: .+\\.fastq\\.gz": "这些数据应该格式化为FastqGz格式。有关更多信息，请参见下面的文档。元素必须匹配正则表达式：.+\\.fastq\\.gz",
    "Filename to import the data as. Must match regex: .+\\.fastq\\.gz": "要作为数据导入的文件名。必须匹配正则表达式：.+\\.fastq\\.gz",
    "This data should be formatted as a SingleEndFastqManifestPhred33. See the documentation below for more information.": "这些数据应该格式化为SingleEndFastqManifestPhred33。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a SingleEndFastqManifestPhred33V2. See the documentation below for more information.": "这些数据应该格式化为SingleEndFastqManifestPhred33V2。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a SingleEndFastqManifestPhred64. See the documentation below for more information.": "这些数据应该格式化为 SingleEndFastqManifestPhred64。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a SingleEndFastqManifestPhred64V2. See the documentation below for more information.": "这些数据应该格式化为SingleEndFastqManifestPhred64V2。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a FastqManifestFormat. See the documentation below for more information.": "这些数据应该格式化为FastqManifestFormat。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a YamlFormat. See the documentation below for more information.": "这些数据应该格式化为 YamlFormat。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a PairedEndFastqManifestPhred33. See the documentation below for more information.": "这些数据应该被格式化为PairedEndFastqManifestPhred33。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a PairedEndFastqManifestPhred33V2. See the documentation below for more information.": "这些数据应该格式化为 PairedEndFastqManifestPhred33V2。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a PairedEndFastqManifestPhred64. See the documentation below for more information.": "这些数据应该被格式化为 PairedEndFastqManifestPhred64。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a PairedEndFastqManifestPhred64V2. See the documentation below for more information.": "这些数据应该被格式化为 PairedEndFastqManifestPhred64V2。有关更多信息，请查看下面的文档。",
    "This data should be formatted as a ProbabilitiesFormat. See the documentation below for more information.": "这些数据应该格式化为概率格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a QIIME1DemuxFormat. See the documentation below for more information.": "这些数据应该被格式化为 QIIME1DemuxFormat。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a JSONFormat. See the documentation below for more information.": "这些数据应该被格式化为JSON格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a PickleFormat. See the documentation below for more information.": "这些数据应该被格式化为Pickle格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a RAxMLinfoFormat. See the documentation below for more information.": "这些数据应该格式化为RAxMLinfo格式。有关更多信息，请参见下面的文档。",
    "This data should be formatted as a UchimeStatsFmt. See the documentation below for more information.": "这些数据应该格式化为 UchimeStatsFmt。有关更多信息，请参见下面的文档。",
    "Choosing to show entire tree may produce very large PDF file disabling your viewer": "选择显示整个树可能会产生非常大的PDF文件，导致您的查看器无法正常工作",
    "set to 0 to show all": "设置为0以显示所有",
    "select column containing GI numbers": "选择包含GI编号的列",
    "select column containing identifiers you want to include into output": "选择要包含在输出中的标识符所在的列",
    "For transcript feature type only: where the GTF contains transcript versions, should these be appended to transcript identifiers? Useful when generating transcript/gene mappings for use with transcriptomes": "对于转录本特征类型：如果GTF文件包含转录本版本信息，是否应该将这些版本信息添加到转录本标识符上？这在生成用于转录组的转录本/基因映射时非常有用。",
    "Only when output fields are defined, useful when you need to guarantee a value, for example a gene ID for a transcript/gene mapping.": "只有在定义了输出字段时，这才有用处，例如当你需要保证一个值，比如转录本/基因映射的基因ID时。",
    "For some applications, e.g. transcriptome mappers, its useful to match a cDNAs file to an annotation list (e.g. transcript-to-gene mapping)": "对于某些应用，例如转录组映射器，将cDNA文件与注释列表（例如转录本到基因的映射）匹配是有用的。",
    "e.g. to find gene IDs for transcripts not present in the GTF. May only work for Ensembl GTFs.": "例如，为了找到GTF中不存在的转录本的基因ID。可能只适用于Ensembl GTFs。",
    "Barcode frequencies will be calculated for you. The alternative is to provide barcode frequencies directly": "条形码频率将为您计算。另一种方法是直接提供条形码频率",
    "This is useful when multiple matrices from different libraries with overlapping barcodes will be merged": "这在将来自不同库的多个矩阵合并时非常有用，这些矩阵具有重叠的条形码。",
    "EBI Single Cell Atlas accession for the experiment that you want to retrieve.": "EBI单细胞图谱实验的访问编号，用于检索您想要获取的实验。",
    "Raw filtered counts or (non-filtered) TPMs": "原始过滤计数或（未过滤的）每百万转录本数",
    "Use comma to separate multiple variables. Not supplying variables will simply pass the input data as output unchanged.": "使用逗号分隔多个变量。不提供变量将简单地将输入数据不变地作为输出传递。",
    "Phenotype data must match between objects, and objects will be concatenated in the order given below.": "表型数据必须在对象之间匹配，并且对象将按照下面给出的顺序连接。",
    "If multiple objects are concatenated as in the above section, the resulting object will be subsetted.": "如果像上节中那样将多个对象连接起来，生成的对象将被子集化。",
    "e.g. random subsampling, index ranges and indices, specific labels, phenotype conditions": "例如，随机抽样、索引范围和索引、特定标签、表型条件",
    "e.g. '10' will select 10 random samples.": "例如，'10' 将随机选择10个样本。",
    "e.g. '123' will select 123 random genes.": "例如，'123' 将选择 123 个随机基因。",
    "e.g. 'Control1,ALPOL56,SampleX' would select just those 3 samples.": "例如，“Control1,ALPOL56,SampleX”将只选择这三个样本。",
    "e.g. 'GeneA,GeneX,Gene123' would select just those 3 genes.": "例如，'GeneA,GeneX,Gene123' 将只选择这3个基因。",
    "e.g. '5:3,57:60,27' would yield '5,4,3,57,58,59,60,27'": "例如，'5:3,57:60,27' 将会产生 '5,4,3,57,58,59,60,27'",
    "e.g. '15:18,26,27,3:1' would yield '15,16,17,18,26,27,3,2,1'": "例如，'15:18,26,27,3:1' 将产生 '15,16,17,18,26,27,3,2,1'",
    "e.g. 'gender' or 'control' etc": "例如 '性别' 或 '对照组' 等",
    "e.g. 'female,unknown' selects only samples with values in the above phenotype column of 'female' and 'unknown'.": "例如，“女性，未知”只选择上述表型列中值为“女性”和“未知”的样本。",
    "e.g. 'housekeeping' or 'marker' etc": "例如“管家”或“标记”等",
    "e.g. '' selects only samples with values in the above phenotype column of 'female' and 'unknown'.": "例如，\"\" 只选择上述表型列中值为 'female' 和 'unknown' 的样本。",
});
